
<html lang="en"     class="pb-page"  data-request-id="977e1b36-f7d8-483a-805c-d2aec108ff5e"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.6b00358;wgroup:string:ACHS website Group;issue:issue:10.1021/jmcmar.2016.59.issue-12;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Uncoupling the Structure–Activity Relationships of β2 Adrenergic Receptor Ligands from Membrane Binding" /></meta><meta name="dc.Creator" content="Callum J.  Dickson" /></meta><meta name="dc.Creator" content="Viktor  Hornak" /></meta><meta name="dc.Creator" content="Camilo  Velez-Vega" /></meta><meta name="dc.Creator" content="Daniel J. J.  McKay" /></meta><meta name="dc.Creator" content="John  Reilly" /></meta><meta name="dc.Creator" content="David A.  Sandham" /></meta><meta name="dc.Creator" content="Duncan  Shaw" /></meta><meta name="dc.Creator" content="Robin A.  Fairhurst" /></meta><meta name="dc.Creator" content="Steven J.  Charlton" /></meta><meta name="dc.Creator" content="David A.  Sykes" /></meta><meta name="dc.Creator" content="Robert A.  Pearlstein" /></meta><meta name="dc.Creator" content="Jose S.  Duca" /></meta><meta name="dc.Description" content="Ligand binding to membrane proteins may be significantly influenced by the interaction of ligands with the membrane. In particular, the microscopic ligand concentration within the membrane surface ..." /></meta><meta name="Description" content="Ligand binding to membrane proteins may be significantly influenced by the interaction of ligands with the membrane. In particular, the microscopic ligand concentration within the membrane surface ..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="June 7, 2016" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.6b00358" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2016 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.6b00358" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.6b00358" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.6b00358" /></link>
        
    
    

<title>Uncoupling the Structure–Activity Relationships of β2 Adrenergic Receptor Ligands from Membrane Binding | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.6b00358" /></meta><meta property="og:title" content="Uncoupling the Structure–Activity Relationships of β2 Adrenergic Receptor Ligands from Membrane Binding" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-12/acs.jmedchem.6b00358/20160617/images/large/jm-2016-003586_0007.jpeg" /></meta><meta property="og:description" content="Ligand binding to membrane proteins may be significantly influenced by the interaction of ligands with the membrane. In particular, the microscopic ligand concentration within the membrane surface solvation layer may exceed that in bulk solvent, resulting in overestimation of the intrinsic protein–ligand binding contribution to the apparent/measured affinity. Using published binding data for a set of small molecules with the β2 adrenergic receptor, we demonstrate that deconvolution of membrane and protein binding contributions allows for improved structure–activity relationship analysis and structure-based drug design. Molecular dynamics simulations of ligand bound membrane protein complexes were used to validate binding poses, allowing analysis of key interactions and binding site solvation to develop structure–activity relationships of β2 ligand binding. The resulting relationships are consistent with intrinsic binding affinity (corrected for membrane interaction). The successful structure-based design of ligands targeting membrane proteins may require an assessment of membrane affinity to uncouple protein binding from membrane interactions." /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta><meta name="twitter:description" content="Ligand binding to membrane proteins may be significantly influenced by the interaction of ligands with the membrane. In particular, the microscopic ligand concentration within the membrane surface solvation layer may exceed that in bulk solvent, resulting in overestimation of the intrinsic protein–ligand binding contribution to the apparent/measured affinity. Using published binding data for a set of small molecules with the β2 adrenergic receptor, we demonstrate that deconvolution of membrane and protein binding contributions allows for improved structure–activity relationship analysis and structure-based drug design. Molecular dynamics simulations of ligand bound membrane protein complexes were used to validate binding poses, allowing analysis of key interactions and binding site solvation to develop structure–activity relationships of β2 ligand binding. The resulting relationships are consistent with intrinsic binding affinity (corrected for membrane interaction). The successful structure-based design of ligands targeting membrane proteins may require an assessment of membrane affinity to uncouple protein binding from membrane interactions." /></meta><meta name="twitter:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-12/acs.jmedchem.6b00358/20160617/images/large/jm-2016-003586_0007.jpeg" /></meta><meta name="twitter:title" content="Uncoupling the Structure–Activity Relationships of β2 Adrenergic Receptor Ligands from Membrane Binding" /></meta>
        
            <meta property="og:type" content="Article" /></meta>
        

        
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.6b00358"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/ke1q~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-6ee29527b3d8ff00a106.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.6b00358">
            
        





































    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-12/achre4.2021.54.issue-12/20210615/achre4.2021.54.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-6/amrcda.2021.2.issue-6/20210625/amrcda.2021.2.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-6/aabmcb.2021.4.issue-6/20210621/aabmcb.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-6/aaembp.2021.3.issue-6/20210622/aaembp.2021.3.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-6/aaemcq.2021.4.issue-6/20210628/aaemcq.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-25/aamick.2021.13.issue-25/20210630/aamick.2021.13.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-6/aanmf6.2021.4.issue-6/20210625/aanmf6.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-6/aapmcd.2021.3.issue-6/20210611/aapmcd.2021.3.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-6/abseba.2021.7.issue-6/20210614/abseba.2021.7.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-12/accacs.2021.11.issue-12/20210618/accacs.2021.11.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-6/acscii.2021.7.issue-6/20210623/acscii.2021.7.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-6/acbcct.2021.16.issue-6/20210618/acbcct.2021.16.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-3/achsc5.2021.28.issue-3/20210524/achsc5.2021.28.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-12/acncdm.2021.12.issue-12/20210616/acncdm.2021.12.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-6/aesccq.2021.5.issue-6/20210617/aesccq.2021.5.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-6/aelccp.2021.6.issue-6/20210611/aelccp.2021.6.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-6/aeecco.2021.1.issue-6/20210611/aeecco.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-6/aewcaa.2021.1.issue-6/20210611/aewcaa.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-5/afsthl.2021.1.issue-5/20210618/afsthl.2021.1.issue-5.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-6/aidcbc.2021.7.issue-6/20210611/aidcbc.2021.7.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-6/amlccd.2021.10.issue-6/20210615/amlccd.2021.10.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-6/amlcef.2021.3.issue-6/20210607/amlcef.2021.3.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-6/amclct.2021.12.issue-6/20210610/amclct.2021.12.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-6/ancac3.2021.15.issue-6/20210622/ancac3.2021.15.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-25/acsodf.2021.6.issue-25/20210629/acsodf.2021.6.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-6/apchd5.2021.8.issue-6/20210616/apchd5.2021.8.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-6/ascefj.2021.6.issue-6/20210625/ascefj.2021.6.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-25/ascecg.2021.9.issue-25/20210628/ascecg.2021.9.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-6/asbcd6.2021.10.issue-6/20210618/asbcd6.2021.10.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-25/ancham.2021.93.issue-25/20210629/ancham.2021.93.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-25/bichaw.2021.60.issue-25/20210629/bichaw.2021.60.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-6/bcches.2021.32.issue-6/20210616/bcches.2021.32.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-6/bomaf6.2021.22.issue-6/20210614/bomaf6.2021.22.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-24/cgeabj.2021.99.issue-24/20210628-01/cgeabj.2021.99.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-6/crtoec.2021.34.issue-6/20210621/crtoec.2021.34.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-12/chreay.2021.121.issue-12/20210623/chreay.2021.121.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-12/cmatex.2021.33.issue-12/20210622/cmatex.2021.33.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-6/cgdefu.2021.21.issue-6/20210602/cgdefu.2021.21.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-12/enfuem.2021.35.issue-12/20210617/enfuem.2021.35.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-12/esthag.2021.55.issue-12/20210615/esthag.2021.55.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-6/estlcu.2021.8.issue-6/20210608/estlcu.2021.8.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-25/iecred.2021.60.issue-25/20210630/iecred.2021.60.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-12/inocaj.2021.60.issue-12/20210621/inocaj.2021.60.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-6/jaaucr.2021.1.issue-6/20210628/jaaucr.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-25/jafcau.2021.69.issue-25/20210630/jafcau.2021.69.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-6/jceaax.2021.66.issue-6/20210610/jceaax.2021.66.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-6/jceda8.2021.98.issue-6/20210608/jceda8.2021.98.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-6/jcisd8.2021.61.issue-6/20210628/jcisd8.2021.61.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-6/jctcce.2021.17.issue-6/20210608/jctcce.2021.17.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/jmcmar.2021.64.issue-12/20210624/jmcmar.2021.64.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-6/jnprdf.2021.84.issue-6/20210625/jnprdf.2021.84.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-12/joceah.2021.86.issue-12/20210618/joceah.2021.86.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-24/jpcafh.2021.125.issue-24/20210624/jpcafh.2021.125.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-24/jpcbfk.2021.125.issue-24/20210624/jpcbfk.2021.125.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-24/jpccck.2021.125.issue-24/20210624/jpccck.2021.125.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-24/jpclcd.2021.12.issue-24/20210624/jpclcd.2021.12.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-6/jprobs.2021.20.issue-6/20210604/jprobs.2021.20.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-25/jacsat.2021.143.issue-25/20210630/jacsat.2021.143.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-6/jamsef.2021.32.issue-6/20210602/jamsef.2021.32.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-25/langd5.2021.37.issue-25/20210629/langd5.2021.37.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-12/mamobx.2021.54.issue-12/20210622/mamobx.2021.54.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-6/mpohbp.2021.18.issue-6/20210607/mpohbp.2021.18.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-12/nalefd.2021.21.issue-12/20210623/nalefd.2021.21.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-12/orlef7.2021.23.issue-12/20210618/orlef7.2021.23.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-6/oprdfk.2021.25.issue-6/20210618/oprdfk.2021.25.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-12/orgnd7.2021.40.issue-12/20210628/orgnd7.2021.40.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.6b00358&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.6b00358&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.6b00358&amp;href=/doi/10.1021/acs.jmedchem.6b00358" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2016</span><span class="cit-fg-volume">, 59</span><span class="cit-fg-issue">, 12</span><span class="cit-fg-pageRange">, 5780-5789</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/59/12" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.6b00324" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.6b00382" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Uncoupling the Structure–Activity Relationships of β2 Adrenergic Receptor Ligands from Membrane Binding</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Callum+J.++Dickson">Callum J. Dickson</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Viktor++Hornak">Viktor Hornak</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Camilo++Velez-Vega">Camilo Velez-Vega</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Daniel+J.+J.++McKay">Daniel J. J. McKay</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=John++Reilly">John Reilly</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=David+A.++Sandham">David A. Sandham</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Duncan++Shaw">Duncan Shaw</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Robin+A.++Fairhurst">Robin A. Fairhurst</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Steven+J.++Charlton">Steven J. Charlton</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=David+A.++Sykes">David A. Sykes</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Robert+A.++Pearlstein">Robert A. Pearlstein</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jose+S.++Duca">Jose S. Duca</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">†</span> <span class="aff-text">Computer-Aided Drug Discovery, Global Discovery Chemistry, Novartis Institutes for BioMedical Research, 100 Technology Square, Cambridge, Massachusetts 02139, United States</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">‡</span> <span class="aff-text">Global Discovery Chemistry, Novartis Institutes for BioMedical Research, 100 Technology Square, Cambridge, Massachusetts 02139, United States</span></div><div class="aff-info" id="aff3"><span class="aff-symbol">§</span> <span class="aff-text">Global Discovery Chemistry, Novartis Institutes for BioMedical Research, Novartis Pharma AG, Werk Klybeck, Postfach, CH-4002 Basel, Switzerland</span></div><div class="aff-info" id="aff4"><span class="aff-symbol">∥</span> <span class="aff-text">Cell Signalling Research Group, School of Life Sciences, University of Nottingham, Queen’s Medical Centre, Nottingham NG7 2UH, U.K.</span></div><div class="corresp-info"><strong>*</strong>For C.J.D.: phone, +1-617-871-4958; E-mail, <a href="/cdn-cgi/l/email-protection#1b787a77776e76357f7278706874755b75746d7a696f726835787476"><span class="__cf_email__" data-cfemail="cdaeaca1a1b8a0e3a9a4aea6bea2a38da3a2bbacbfb9a4bee3aea2a0">[email protected]</span></a></div><div class="corresp-info"><strong>*</strong>For V.H.: phone, +1-617-871-8016; E-mail, <a href="/cdn-cgi/l/email-protection#cabca3a1bea5b8e4a2a5b8a4aba18aa4a5bcabb8bea3b9e4a9a5a7"><span class="__cf_email__" data-cfemail="cfb9a6a4bba0bde1a7a0bda1aea48fa1a0b9aebdbba6bce1aca0a2">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.6b00358&amp;href=/doi/10.1021%2Facs.jmedchem.6b00358" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2016</span></span><span class="cit-volume">, 59</span><span class="cit-issue">, 12</span><span class="cit-pageRange">, 5780–5789</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">May 30, 2016</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>8 March 2016</li><li><span class="item_label"><b>Published</b> online</span>7 June 2016</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 23 June 2016</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.6b00358" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.6b00358</a></div><div class="article_header-article-copyright"><strong>Copyright © 2016 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D5780%26pageCount%3D10%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DCallum%2BJ.%2BDickson%252C%2BViktor%2BHornak%252C%2BCamilo%2BVelez-Vega%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D59%26issueNum%3D12%26contentID%3Dacs.jmedchem.6b00358%26title%3DUncoupling%2Bthe%2BStructure%25E2%2580%2593Activity%2BRelationships%2Bof%2B%25CE%25B22%2BAdrenergic%2BReceptor%2BLigands%2Bfrom%2BMembrane%2BBinding%26numPages%3D10%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D5789%26publicationDate%3DJune%2B2016">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.6b00358"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1880</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">12</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.6b00358" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Uncoupling the Structure–Activity Relationships of β2 Adrenergic Receptor Ligands from Membrane Binding&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Callum&quot;,&quot;last_name&quot;:&quot;J. Dickson&quot;},{&quot;first_name&quot;:&quot;Viktor&quot;,&quot;last_name&quot;:&quot;Hornak&quot;},{&quot;first_name&quot;:&quot;Camilo&quot;,&quot;last_name&quot;:&quot;Velez-Vega&quot;},{&quot;first_name&quot;:&quot;Daniel&quot;,&quot;last_name&quot;:&quot;J. J. McKay&quot;},{&quot;first_name&quot;:&quot;John&quot;,&quot;last_name&quot;:&quot;Reilly&quot;},{&quot;first_name&quot;:&quot;David&quot;,&quot;last_name&quot;:&quot;A. Sandham&quot;},{&quot;first_name&quot;:&quot;Duncan&quot;,&quot;last_name&quot;:&quot;Shaw&quot;},{&quot;first_name&quot;:&quot;Robin&quot;,&quot;last_name&quot;:&quot;A. Fairhurst&quot;},{&quot;first_name&quot;:&quot;Steven&quot;,&quot;last_name&quot;:&quot;J. Charlton&quot;},{&quot;first_name&quot;:&quot;David&quot;,&quot;last_name&quot;:&quot;A. Sykes&quot;},{&quot;first_name&quot;:&quot;Robert&quot;,&quot;last_name&quot;:&quot;A. Pearlstein&quot;},{&quot;first_name&quot;:&quot;Jose&quot;,&quot;last_name&quot;:&quot;S. Duca&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2016&quot;,&quot;month&quot;:&quot;06&quot;,&quot;day&quot;:&quot;07&quot;,&quot;issue&quot;:&quot;12&quot;,&quot;volume&quot;:&quot;59&quot;,&quot;pages&quot;:&quot;5780-5789&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.6b00358&quot;},&quot;abstract&quot;:&quot;Ligand binding to membrane proteins may be significantly influenced by the interaction of ligands with the membrane. In particular, the microscopic ligand concentration within the membrane surface solvation layer may exceed that in bulk solvent, resulting in overestimation of the intrinsic protein–ligand binding contribution to the apparent/measured affinity. Using published binding data for a set of small molecules with the β2 adrenergic receptor, we demonstrate that deconvolution of membrane and protein binding contributions allows for improved structure–activity relationship analysis and structure-based drug design. Molecular dynamics simulations of ligand bound membrane protein complexes were used to validate binding poses, allowing analysis of key interactions and binding site solvation to develop structure–activity relationships of β2 ligand binding. The resulting relationships are consistent with intrinsic binding affinity (corrected for membrane interaction). The successful structure-based design of l&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b00358&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b00358" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b00358&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b00358" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b00358&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b00358" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.6b00358&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b00358&amp;href=/doi/10.1021/acs.jmedchem.6b00358" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.6b00358" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.6b00358" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (4 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.6b00358&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.6b00358%26sid%3Dliteratum%253Aachs%26pmid%3D27239696%26genre%3Darticle%26aulast%3DDickson%26date%3D2016%26atitle%3DUncoupling%2Bthe%2BStructure%25E2%2580%2593Activity%2BRelationships%2Bof%2B%25CE%25B22%2BAdrenergic%2BReceptor%2BLigands%2Bfrom%2BMembrane%2BBinding%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D59%26issue%3D12%26spage%3D5780%26epage%3D5789%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=291691" title="Membranes">Membranes</a>,</li><li><a href="/action/doSearch?ConceptID=291352" title="Screening assays">Screening assays</a>,</li><li><a href="/action/doSearch?ConceptID=292186" title="Receptors">Receptors</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/59/12" title="Go to Volume 59, Issue 12"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-12/jmcmar.2016.59.issue-12/20160623/jmcmar.2016.59.issue-12.largecover.jpg" alt="Go to Volume 59, Issue 12"/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-12/acs.jmedchem.6b00358/20160617/images/medium/jm-2016-003586_0007.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-12/acs.jmedchem.6b00358/20160617/images/large/jm-2016-003586_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00358&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Ligand binding to membrane proteins may be significantly influenced by the interaction of ligands with the membrane. In particular, the microscopic ligand concentration within the membrane surface solvation layer may exceed that in bulk solvent, resulting in overestimation of the intrinsic protein–ligand binding contribution to the apparent/measured affinity. Using published binding data for a set of small molecules with the β2 adrenergic receptor, we demonstrate that deconvolution of membrane and protein binding contributions allows for improved structure–activity relationship analysis and structure-based drug design. Molecular dynamics simulations of ligand bound membrane protein complexes were used to validate binding poses, allowing analysis of key interactions and binding site solvation to develop structure–activity relationships of β2 ligand binding. The resulting relationships are consistent with intrinsic binding affinity (corrected for membrane interaction). The successful structure-based design of ligands targeting membrane proteins may require an assessment of membrane affinity to uncouple protein binding from membrane interactions.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_62343" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_62343" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Integral membrane proteins, and G protein-coupled receptors (GPCRs) in particular, participate in a wide array of cellular functions and thereby constitute a large class of pharmacologically important targets.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Early drug discovery is highly dependent on measured structure–activity relationships (SAR), the interpretation of which is guided in many cases by ligand- (e.g., pharmacophore, QSAR) and structure-based models. It is implicitly assumed in all cases that measured/apparent and true activities are approximately equivalent. However, the validity of this assumption can vary on a case-by-case basis, depending on both the nature of the target and the experimental methods. Membrane protein binding is typically measured using membrane preparations due to difficulties in isolating membrane embedded proteins from their natural environment. It is apparent that membrane proteins are “solvated” by a combination of water and phospholipids and other species partitioned in the phospholipid bilayer (e.g., cholesterol). Therefore, it is reasonable to conclude that ligand binding is governed by three-way interactions with water, protein, and membrane.</div><div class="NLM_p">Because of this increased complexity, it is likely that the pharmacology of drug binding to membrane embedded proteins, such as the β2-adrenergic receptor GPCR (β2-AR), is more intricate than drug interaction with a soluble target. This may complicate the interpretation of binding data for membrane proteins and thus SAR built on such data. It has been proposed that the binding of drugs targeting membrane receptors may be influenced by a number of membrane related phenomena. A “microkinetic” model<a onclick="showRef(event, 'ref2 ref3 ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref2 ref3 ref4 ref5">(2-5)</a> suggests that the accumulation of drug inside or in the vicinity of the membrane compartment acts as a depot, increasing local drug concentration around the target. Slow release from the membrane depot may account for a longer duration of action.<a onclick="showRef(event, 'ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref6 ref7">(6, 7)</a> The “reduction in dimensionality” effect<a onclick="showRef(event, 'ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref8 ref9">(8, 9)</a> may also need to be considered, whereby ligands move from 3D diffusion in solution to 2D lateral diffusion in the membrane, possibly resulting in an increased drug–receptor collision rate. The extent of this effect would depend on factors such as the drug surface density and 2D lateral diffusion coefficient. Alternatively, an “exosite” model<a onclick="showRef(event, 'ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref10 ref11">(10, 11)</a> accounts for the extended duration of action of long-acting β2-adrenoceptor agonists by proposing that the side chain of the agonist anchors into the membrane, or some other exosite, allowing the activating warhead to flip in and out of the binding site. Lastly, a “rebinding” model<a onclick="showRef(event, 'ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref12 ref13">(12, 13)</a> proposes that dissociated drug is able to quickly reassociate with the same or other nearby receptor, delaying escape from the cell surface.</div><div class="NLM_p">While no single model necessarily describes all membrane protein–ligand binding cases, we adopt the microkinetic model in this work. Considerable evidence suggests that certain molecules partition or adsorb in or on bilayers, thereby enhancing their copy number proximal to membrane protein binding sites and promoting slow return back to the central compartment (where clearance occurs). Although the exact mechanism underlying the longer duration of action of certain β2 ligands remains controversial,<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> the microkinetic model is consistent with computational simulations performed by Szczuka et al., suggesting that it adequately explains long-lasting β2-AR stimulation due to ligand enrichment in the membrane<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> and the identification of a tissue binding site for β2 agonists distinct from the β2-AR via radio-ligand binding.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> Conversely, the exosite model does not account for the similarly long duration of action of the β2 agonists formoterol and salmeterol despite the shorter nonpolar side chain of formoterol.<a onclick="showRef(event, 'ref17 ref18 ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref17 ref18 ref19 ref20">(17-20)</a></div><div class="NLM_p">In addition to affecting the pharmacokinetics of a ligand, drug–membrane interactions may also obscure the true intrinsic binding affinity of ligands for a membrane embedded target making experimental measurements of membrane receptor binding difficult to interpret. Typical amphiphilic drug molecules may absorb onto the membrane surface, causing an enhancement in local concentration around the receptor and observed on-rate to the target. The molecules with high membrane affinity show higher observed target binding (lower <i>K</i><sub>d</sub>) than is really occurring; the measured receptor occupancy, and thus <i>K</i><sub>d</sub>, is driven by a high local concentration rather than the assumed bulk concentration of ligand.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> High membrane association can therefore obscure low intrinsic affinity for the protein. This effect also skews SAR to favoring more lipophilic ligands as an apparent way to increase affinity. In such cases, only by uncoupling intrinsic protein binding from membrane association is it possible to build a meaningful SAR for membrane protein binding.</div><div class="NLM_p">Recently, Sykes et al.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> investigated the influence of a membrane compartment on membrane receptor binding by separately determining the membrane affinity and β2-AR binding for a number of β2 agonists and antagonists, using the immobilized artificial membrane (IAM) HPLC method,<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> together with radioligand competition assays on membrane preparations, with results suggesting that the apparent measured <i>K</i><sub>d</sub> is indeed influenced by ligand–membrane interactions. Cationic amphiphilic drugs, such as those considered in the present work, have been found via surface plasmon resonance to bind to the surface of liposomes, with the binding affinity dependent on the drug structure and lipid type.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> The assumption used in this work is that such membrane association is reasonably well captured by the IAM experiment. It has previously been found that IAM chromatography accurately predicts the promiscuity of PET imaging drugs to bind nonspecifically to membranes.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> Once <i>K</i><sub>IAM</sub> is determined, the intrinsic receptor binding (<i>K</i><sub>d</sub><sup>int</sup>) may be calculated by subtracting <i>K</i><sub>IAM</sub> from the observed target <i>K</i><sub>d</sub> (<i>K</i><sub>d</sub><sup>obs</sup>).<a onclick="showRef(event, 'ref9 ref24 ref25'); return false;" href="javascript:void(0);" class="ref ref9 ref24 ref25">(9, 24, 25)</a> To facilitate the understanding of the microkinetic model in this work, we make a distinction between membrane association and membrane partitioning. While association with the membrane helps to increase the ligand concentration around the membrane and the protein target, membrane partitioning effectively removes the ligand from the solution phase and makes it unavailable for binding (excluding the possibility of binding from the membrane compartment). The essence of the binding model is captured in <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-12/acs.jmedchem.6b00358/20160617/images/medium/jm-2016-003586_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-12/acs.jmedchem.6b00358/20160617/images/large/jm-2016-003586_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Membrane receptor binding model. Observed binding (<i>K</i><sub>d</sub><sup>obs</sup>) results from both the affinity of the ligand to associate with the membrane (approximated by measured <i>K</i><sub>IAM</sub>) and “true”/intrinsic affinity for the protein (<i>K</i><sub>d</sub><sup>int</sup>).  On the basis of the microkinetic model, the ligand association with the membrane may create higher “local” concentration in the target compartment (dark-blue region), <i>L</i><sub>assoc</sub>, which can result in higher receptor occupancy than would have been expected from the bulk ligand concentration <i>L</i><sub>bulk</sub>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-12/acs.jmedchem.6b00358/20160617/images/large/jm-2016-003586_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00358&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In this work, we have used all-atom molecular dynamics (MD) simulations of the β2-AR embedded in a fully solvated phospholipid bilayer model, together with water structure analysis, to study the intrinsic binding contributions (<i>K</i><sub>d</sub><sup>int</sup> versus <i>K</i><sub>d</sub><sup>obs</sup>) in the β2 ligand data set of Sykes et al. The modeled binding modes were used in conjunction with solvation analysis (from our in-house WATMD software<a onclick="showRef(event, 'ref26 ref27'); return false;" href="javascript:void(0);" class="ref ref26 ref27">(26, 27)</a>) to explore the SAR against observed versus intrinsic affinity (<i>K</i><sub>d</sub><sup>int</sup>, derived from <i>K</i><sub>d</sub><sup>obs</sup> and <i>K</i><sub>IAM</sub>). We demonstrate that correction of observed receptor binding affinity for membrane association is essential for deriving meaningful SAR needed for ligand- or structure-based lead optimization.</div><div class="NLM_p">Our study is based on the binding and kinetic data of a number of β2 agonists and antagonists for the inactive/G protein-uncoupled form of β2-AR reported by Sykes et al.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> In that work, association and dissociation rates of the ligands were measured using a kinetic competition radioligand binding method on Chinese hamster ovary cell membrane preparations expressing β2-AR. Membrane affinity (<i>K</i><sub>IAM</sub>) was estimated using an immobilized artificial membrane (IAM) HPLC column,<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> which comprises phosphatidylcholine analogues immobilized on silica as a chromatography phase to mimic a cell membrane. To account for enhanced ligand concentration around the β2-AR receptor due to membrane interactions, kinetic association rates were corrected as <i>k</i><sub>on</sub><sup>corr</sup> = <i>k</i><sub>on</sub>/<i>K</i><sub>IAM</sub>.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> This allowed calculation of the “true” or intrinsic binding affinity via <i>K</i><sub>d</sub><sup>int</sup> = <i>k</i><sub>off</sub>/<i>k</i><sub>on</sub><sup>corr</sup>, which is equivalent to a simpler expression:<span class="NLM_disp-formula" id="eq1"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-12/acs.jmedchem.6b00358/20160617/images/jm-2016-003586_m001.gif" alt="" /></img><span class="labelSpan">(1)</span></span></div><div class="NLM_p">Taking the logarithmic form then yields <i>K</i><sub>d</sub><sup>int</sup> as the observed binding affinity minus the membrane affinity, p<i>K</i><sub>d</sub><sup>int</sup> = p<i>K</i><sub>d</sub><sup>obs</sup> – log <i>K</i><sub>IAM</sub>. Interestingly, the correction for membrane binding is of the same form as lipophilic efficiency (LiPE = pIC<sub>50</sub> – log <i>P</i>).<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a></div><div class="NLM_p">For the discussion of how membrane interactions affect functional versus binding assays, we refer to the Hill–Langmuir equation describing receptor occupancy ρ as a function of ligand concentration [L] and ligand affinity to the receptor <i>K</i><sub>d</sub>:<span class="NLM_disp-formula" id="eq2"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-12/acs.jmedchem.6b00358/20160617/images/jm-2016-003586_m002.gif" alt="" /></img><span class="labelSpan">(2)</span></span></div><div class="NLM_p">Note that according to <a class="ref internalNav" href="#eq2" aria-label="eq 2">eq 2</a>, receptor occupancy depends on the free ligand concentration [L] and binding constant <i>K</i><sub>d</sub>. Lacking accurate knowledge of [L] at the binding site, there is no way to distinguish the relative contributions of free concentration and <i>K</i><sub>d</sub> to occupancy solely on the basis of functional assay data.</div><div class="NLM_p">All ligand structures used in this study, along with <i>K</i><sub>d</sub><sup>obs</sup>, <i>K</i><sub>d</sub><sup>int</sup>, and <i>K</i><sub>IAM</sub>, are shown in <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>. Most compounds are racemic, but were modeled with the same stereochemistry (<i>R</i> for salmeterol series and <i>S</i> for cyanopindolol series) of the canonical β-hydroxy-amine motif within each group of analogues. Therefore, the measured <i>K</i><sub>d</sub> can shift slightly (about 2-fold as a consequence of the inactive enantiomer) but likely in the same direction within a series. The <i>cis</i>-stereoisomer was assumed for nadolol at the tetralin ring.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-12/acs.jmedchem.6b00358/20160617/images/medium/jm-2016-003586_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-12/acs.jmedchem.6b00358/20160617/images/large/jm-2016-003586_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Structures, <i>K</i><sub>d</sub><sup>obs</sup>, corrected <i>K</i><sub>d</sub><sup>int</sup>, and <i>K</i><sub>IAM</sub> values for structurally related groups of β2-AR ligands: salmeterol series (agonists isoprenaline, salbutamol, salmeterol), atenolol series (antagonists atenolol, metoprolol, bisoprolol), and cyanopindolol series (antagonists cyanopindolol, CGP12177, nadolol, propranolol, ICI-118,551). Data (<i>K</i><sub>d</sub><sup>obs</sup>, <i>K</i><sub>IAM</sub>) from Sykes et al.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> *, racemic mixture; #, racemic mixture of the 1-diastereoisomer; ∧, mixture of four diastereoisomers with a cis-relationship in the tetralin ring. A full list of binding and kinetic data is available in the Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.6b00358/suppl_file/jm6b00358_si_001.pdf" class="ext-link">Table S1</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-12/acs.jmedchem.6b00358/20160617/images/large/jm-2016-003586_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00358&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">To demonstrate the effect of <i>K</i><sub>IAM</sub> on intrinsic <i>K</i><sub>d</sub><sup>int</sup>, let us consider the following illustrative example. The observed binding affinity of salmeterol to β2-AR is <i>K</i><sub>d</sub><sup>obs</sup> = 0.3 nM. However, this ligand has high membrane association (log <i>K</i><sub>IAM</sub> = 3.5), creating a high local concentration around the receptor and an apparent enhanced binding affinity observed experimentally under the assumption of uniform bulk concentration. To correct for this effect, the intrinsic binding <i>K</i><sub>d</sub><sup>int</sup> may be derived from <a class="ref internalNav" href="#eq1" aria-label="eq 1">eq 1</a> as <i>K</i><sub>d</sub><sup>int</sup> = 0.3 × 10<sup>–9</sup> × 10<sup>3.5</sup> = 1100 nM. High membrane association of salmeterol therefore obscures its much lower intrinsic binding <i>K</i><sub>d</sub><sup>int</sup>.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_76742" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_76742" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The primary objective of our study was to assess whether direct protein–receptor binding is best reflected in the calculated <i>K</i><sub>d</sub><sup>int</sup> or experimental <i>K</i><sub>d</sub><sup>obs</sup> values based on observed interactions in our MD simulations and WATMD analysis results. Groups of structurally related ligands were selected based on the published binding data.</div><div class="NLM_p">All ligands modeled display conserved interactions between the β-hydroxy-amine motif on the ligand tails and β2-AR Asp113 and Asn312 residues at the front of the binding pocket, a common anchor shared by many β2 antagonists, inverse agonists, and agonists.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> The majority of residues forming the remainder of the narrow binding pocket are hydrophobic, with the exception of certain polar residues at the rear: Ser203, Ser204, Ser207, and Tyr199 on H5, Asn293 on H6, and Thr118 on H3.</div><div class="NLM_p">The first example examines structurally related β2 agonists isoprenaline, salbutamol, and the long-acting salmeterol, for which membrane and receptor binding data are available<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> (see <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>). The observed 700-fold higher potency of the β2 agonist salmeterol in comparison to the structurally related agonists isoprenaline and salbutamol (<i>K</i><sub>d</sub><sup>obs</sup>) is found to be solely due to membrane binding; salmeterol has far higher membrane affinity, and upon correcting for this, all three agonists have comparable <i>K</i><sub>d</sub><sup>int</sup> values for the inactive form of β2-AR (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>). The SAR of salmeterol is necessarily weighted toward higher membrane association to a greater extent than the other compounds, which can mislead SAR interpretation in the absence of this knowledge.</div><div class="NLM_p">Our structural analysis suggests that the similar intrinsic activities likely result from the catechol (isoprenaline) and saligenin (salbutamol and salmeterol) groups of the agonists forming similar hydrogen bonding networks to the serine residue Ser203 and to Thr118 via a bridging water molecule at the rear of the binding pocket (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>) in addition to conserved anchor interactions to Asp113 and Asn312. A docked pose of salmeterol is able to create an additional hydrogen bond with β2-AR (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>, circled in red), however this does not appear to translate into higher intrinsic binding affinity, possibly due to the entropic penalty encountered on binding this highly flexible ligand. In this first example, traditional analysis would point to salmeterol as having the strongest binding to β2-AR (<i>K</i><sub>d</sub><sup>obs</sup> = 0.3 nM), however, the membrane correction puts all three ligands at roughly equal binding affinity (<i>K</i><sub>d</sub><sup>int</sup> = 740–1400 nM), which is more consistent from the point of view of very similar ligand–receptor interactions in the binding pocket. Using <i>K</i><sub>d</sub><sup>int</sup> as a primary binding affinity indicator changes our perception of what ligand features are important for binding.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-12/acs.jmedchem.6b00358/20160617/images/medium/jm-2016-003586_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-12/acs.jmedchem.6b00358/20160617/images/large/jm-2016-003586_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Salmeterol series. The modeled binding modes of agonist isoprenaline, salbutamol, and salmeterol. All three make similar hydrogen bonds to hydrophilic residues at the rear of the binding pocket, either directly or via a bridging water. Salmeterol is able to form an additional hydrogen bond to Phe193 (circled in red for clarity). High occupancy solvation sites are represented by spheres whose size correlates with the magnitude of their occupancy with respect to bulk water. Only those with occupancy higher than 2× bulk are shown. In turn, the coloring of each water site correlates with its relative hydrogen:oxygen occupancy. Namely, each sphere is colored in red–white–blue spectrum, with intense red corresponding to oxygen-only occupancy, deep-blue to hydrogen only occupancy, and white to a 2:1 hydrogen:oxygen (i.e., water-like) occupancy. Additional information concerning visualization of WATMD structures can be found in ref <a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">26</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-12/acs.jmedchem.6b00358/20160617/images/large/jm-2016-003586_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00358&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The three β2-AR antagonist analogues atenolol, metoprolol, and bisoprolol contain an aromatic ring with a β-hydroxy-amine tail, differing solely in their headgroup substituents; atenolol contains an amide chain, metoprolol an ether, and bisoprolol a diether (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>). The observed binding affinity of atenolol, metoprolol, and bisoprolol are virtually equivalent, with all three having a measured <i>K</i><sub>d</sub><sup>obs</sup> in the 100–300 nM range. However, after correcting for membrane binding, atenolol becomes the most potent, having an intrinsic <i>K</i><sub>d</sub><sup>int</sup> which is roughly 10-fold improved over either metoprolol or bisoprolol. Upon examination of the modeled binding modes (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>), this separation in <i>K</i><sub>d</sub><sup>int</sup> values appears sensible, as atenolol is able to form an interaction network between the amide side chain and Ser207 and Thr118, in addition to the standard anchor interactions to Asp113 and Asn312. Conversely, metoprolol finds only a single additional hydrogen bond between the ether chain and Tyr199, and bisoprolol displays a single additional hydrogen bond between the diether and Ser207. Furthermore, the longer, flexible ether chains of metoprolol and bisoprolol may potentially lose greater conformational entropy upon binding in comparison to the shorter amide side chain of atenolol. In this example, we observe the reverse scenario compared to the salmeterol series. While we start with roughly equipotent observed binding affinities, membrane correction singles out atenolol as the tightest binder to β2-AR. Again, corrected <i>K</i><sub>d</sub><sup>int</sup> values change how we interpret the SAR in this series of analogues.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-12/acs.jmedchem.6b00358/20160617/images/medium/jm-2016-003586_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-12/acs.jmedchem.6b00358/20160617/images/large/jm-2016-003586_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Atenolol series. The modeled binding modes of atenolol, metoprolol, and bisoprolol. Atenolol becomes the most potent in the series after correction for membrane binding and is observed to form a greater number of hydrogen bonds with the protein than metoprolol or bisoprolol.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-12/acs.jmedchem.6b00358/20160617/images/large/jm-2016-003586_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00358&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The cyanopindolol β2 antagonist series consists of cyanopindolol, CGP12177, nadolol, propranolol, and ICI-118,551 (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>). The observed binding affinity of cyanopindolol is initially 700-fold more potent than the structurally similar ligands CGP12177, nadolol, propranolol, and ICI-118,551. Nadolol exhibits the lowest potency of the series, with CGP12177, propranolol, and ICI-118,551 being roughly equipotent. After correction for membrane binding, cyanopindolol remains the most potent, yet the ranking of the remainder of the series shifts as follows: CGP12177 and nadolol display higher binding affinity than the hydrophobic core bearing ligands of propranolol and ICI-118,551.</div><div class="NLM_p">The stronger intrinsic binding propensity of cyanopindolol, CGP12177, and nadolol over that of propranolol and ICI-118,551 is revealed on analysis of their binding modes to β2-AR (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>): the former ligands can make additional hydrogen bonds to the protein, which the hydrophobic cores of propranolol and ICI-118,551 cannot achieve. Cyanopindolol and CGP12177 are able to form stable hydrogen bonds to Ser203, while the hydroxyl groups of nadolol make hydrogen bonds to Ser204 and high occupancy water. The ∼30-fold higher binding affinity of cyanopindolol in comparison to the structurally similar CGP12177 cannot be explained from direct receptor–ligand interactions; however, WATMD analysis reveals that CGP12177 remains partially solvated in the binding pocket, while cyanopindolol does not. The water next to the cyclic urea of CGP12177 forms ∼3 hydrogen bonds (i.e., less than the average 3.6 bonds in bulk water), suggesting H-bond/enthalpic depletion; furthermore, it is trapped at the rear of the binding pocket so it should pay a high entropic penalty. The cyano group of cyanopindolol efficiently fills this space in the binding pocket by displacing this energetically unfavorable water, resulting in higher binding affinity. A similar result was reported by Pearlstein et al.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> in a WaterMap study of p38α kinase, in which trapped water was displaced by the naphthyl group of one inhibitor but not the phenyl of another. Solvation with occupancy higher than bulk is also seen in this region of the binding pocket in the nadolol, propranolol, and ICI-118,551 results. In the cyanopindolol example, reranking of ligands based on <i>K</i><sub>d</sub><sup>int</sup> is more subtle than in the previous two examples, yet still significant, especially for propranolol and ICI-118,551, which started as more potent binders based on their observed affinities (<i>K</i><sub>d</sub><sup>obs</sup>) but ended up as the weakest binders upon uncoupling their intrinsic binding affinity (<i>K</i><sub>d</sub><sup>int</sup>) from membrane interactions.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-12/acs.jmedchem.6b00358/20160617/images/medium/jm-2016-003586_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-12/acs.jmedchem.6b00358/20160617/images/large/jm-2016-003586_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Cyanopindolol series. The modeled binding modes of cyanopindolol, CGP12177, nadolol, propranolol, and ICI-118,551 with WATMD results. The first panel shows the WATMD results from simulation of the apo β2-AR protein (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2RH1">2RH1</a> with antagonist carazolol removed), with cyanopindolol binding mode shown for reference. After correction for membrane binding, CGP12177, and nadolol become more potent than propranolol or ICI-118,551, which is consistent with the greater number of hydrogen bonds the former two compounds make with the protein. Cyanopindolol remains the most potent in the series, which correlates with efficient replacement of unfavorable water at the rear of the binding pocket.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-12/acs.jmedchem.6b00358/20160617/images/large/jm-2016-003586_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00358&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">The binding kinetics of these β2 antagonists may be explained by how ligand binding perturbs the water structure inside the protein pocket. Upon correction of the kinetic data for membrane binding, all five ligands have very similar <i>k</i><sub>on</sub> values of 0.33–3.11 × 10<sup>6</sup> M<sup>–1</sup> min<sup>–1</sup> (Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.6b00358/suppl_file/jm6b00358_si_001.pdf" class="ext-link">Table S1</a>).<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> The higher <i>K</i><sub>d</sub><sup>int</sup> of cyanopindolol is due to far slower <i>k</i><sub>off</sub> than the remainder of the series (cyanopindolol <i>k</i><sub>off</sub> 0.002 min<sup>–1</sup>, CGP12177 0.05 min<sup>–1</sup>, nadolol 0.14 min<sup>–1</sup>, propranolol 0.46 min<sup>–1</sup>, and ICI-118,551 0.21 min<sup>–1</sup>). This is possibly due to the efficient replacement of solvation by the cyano group from the unbound protein upon ligand binding, a premise supported by the fact that a WATMD analysis of the unbound protein reveals high water occupancy at the rear of the binding pocket precisely at the position of the cyano group (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>). The relation between solvation and association and dissociation rates has been previously studied<a onclick="showRef(event, 'ref30 ref31 ref32'); return false;" href="javascript:void(0);" class="ref ref30 ref31 ref32">(30-32)</a> and is an active area of research in our group.</div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_65076" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_65076" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Modeling of receptor–ligand interactions, combined with binding pocket solvation analysis, may adequately guide SAR interpretation in cases where direct protein–ligand interactions are known to predominate in measured protein–ligand binding affinity. However, the effects of membrane binding on <i>K</i><sub>d</sub><sup>obs</sup> cannot be directly measured in vitro due to the practical limitations of expressing properly folded membrane proteins isolated from the phospholipid environment, thereby preventing direct measurements on membrane free protein. Although variable levels of ligand–membrane binding across a series of receptor ligands can confound the interpretation of experimentally determined SAR, indirect methods for deconvoluting <i>K</i><sub>d</sub><sup>obs</sup> and <i>K</i><sub>mem</sub>, including <i>K</i><sub>IAM</sub> combined with appropriate structure-based modeling, can be used to a first approximation. Although this work is concerned with β2-AR binding, membrane interactions are expected to affect binding to all integral membrane protein binding sites. Exploring the SAR and corresponding binding modes for larger chemical series with <i>K</i><sub>IAM</sub> and <i>K</i><sub>d</sub><sup>obs</sup> measurements, for either the β2-AR or other membrane protein, would further our understanding on the need to uncouple receptor and membrane binding.</div><div class="NLM_p">In this work, we performed a qualitative analysis of interactions and solvation of several series of ligands with the β2-AR receptor. We compared interaction patterns to their experimental binding affinities (<i>K</i><sub>d</sub><sup>obs</sup>), as well as to membrane corrected intrinsic binding affinities (<i>K</i><sub>d</sub><sup>int</sup>), concluding that receptor–ligand interactions are more consistent with the intrinsic binding affinities <i>K</i><sub>d</sub><sup>int</sup> rather than observed values (<i>K</i><sub>d</sub><sup>obs</sup>). Prospective quantitative predictions of intrinsic binding affinities remain challenging even with state-of-the-art energy scoring functions.<a onclick="showRef(event, 'ref33 ref34'); return false;" href="javascript:void(0);" class="ref ref33 ref34">(33, 34)</a> More rigorous methods based on free energy calculations<a onclick="showRef(event, 'ref35 ref36'); return false;" href="javascript:void(0);" class="ref ref35 ref36">(35, 36)</a> have experienced a recent resurgence and have shown progress toward a more reliable quantitative prediction of free energies of binding.<a onclick="showRef(event, 'ref37 ref38 ref39'); return false;" href="javascript:void(0);" class="ref ref37 ref38 ref39">(37-39)</a></div><div class="NLM_p">Drug discovery targeted at GPCRs is predominantly driven by functional, rather than binding assays, and as such membrane binding is an important consideration in the interpretation of SAR based on functional data. Functional readouts are downstream from binding and receptor activation and therefore more closely tied to therapeutic effects in vivo. Functional response, which is directly related to receptor occupancy, is fully determined by intrinsic receptor binding (<i>K</i><sub>d</sub><sup>int</sup>) when the ligand concentration is known (<a class="ref internalNav" href="#eq2" aria-label="eq 2">eq 2</a>). However, ligand–membrane interactions, which are implicit to the microkinetic model, obscure the concentration of the ligand in the target compartment. The ligand concentration can be enriched at the receptor binding site due to ligand–membrane interactions, resulting in a greater response than would have been expected based on the applied “bulk” ligand concentration in the functional assay. In such cases, the desired receptor occupancy is achieved via weaker than expected <i>K</i><sub>d</sub><sup>int</sup> (i.e., <i>K</i><sub>d</sub><sup>int</sup> > <i>K</i><sub>d</sub><sup>obs</sup>). We can imagine a hypothetical example in which two ligands, A and B, elicit an equivalent functional response and therefore are assumed to have the same intrinsic receptor occupancy. Let us further assume that, due to membrane association, ligand A has 100-fold higher local concentration in the target compartment than ligand B. Ligand A can now achieve the same receptor occupancy with 100-times weaker intrinsic affinity <i>K</i><sub>d</sub><sup>int</sup>. Therefore, ligand A achieves the same receptor occupancy due to its higher concentration at the expense of its weaker intrinsic binding (<i>K</i><sub>d</sub><sup>int</sup>). The interpretation of the SAR for ligands A and B would therefore be very different once we understand that the same magnitude of functional response can result from significantly different intrinsic affinities (<i>K</i><sub>d</sub><sup>int</sup>).</div><div class="NLM_p">In this work, we have adopted the membrane receptor binding model as used by Sykes et al.,<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> whereby ligands with higher levels of membrane affinity accumulate at the membrane/water interface. Receptor on-rate is enhanced by enrichment of the ligand concentration within the target compartment (appearing as more potent <i>K</i><sub>d</sub><sup>obs</sup>). It is assumed that ligand enrichment at the membrane surface translates to increased concentration in the target compartment (dark-blue zone in <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>) and remains available for binding to the receptor. Furthermore, it is assumed that <i>K</i><sub>IAM</sub> is a reasonable measure of this membrane association. By the same token, membrane interactions can work to lower the free ligand concentration in the target compartment, depending on the nature of the ligand: highly lipophilic compounds may become trapped within the membrane for extended periods of time, while highly polar molecules may accumulate in the solution bulk phase. Concentration lowering effects are not captured by our current model (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>), which assumes that the ligand concentration is always enhanced in the target compartment. As a future extension of this work, we plan to develop a more complete kinetic model of β2-AR binding that incorporates both receptor–ligand and membrane–ligand binding kinetics (see <a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>). We expect that the optimal <i>K</i><sub>IAM</sub> is neither too low (i.e., reflecting low or no enrichment) nor too high (i.e., reflecting high membrane partitioning, where the membrane acts as a ligand sink). The effect of target compartment enrichment may be overestimated for ligands with higher <i>K</i><sub>IAM</sub> values used in our study (e.g., salmeterol), resulting in overcorrection of the intrinsic binding affinity (<i>K</i><sub>d</sub><sup>int</sup>). A solution to this problem necessitates consideration of both membrane–ligand and protein–ligand binding kinetics (<a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>). While this scheme is more realistic, it is also more complex, requiring experimental determination of all kinetic rate constants depicted in the model.</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-12/acs.jmedchem.6b00358/20160617/images/medium/jm-2016-003586_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-12/acs.jmedchem.6b00358/20160617/images/large/jm-2016-003586_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. This model accounts for both increased free ligand concentration in the target compartment (dark-blue zone) relative to bulk solvent (light-blue zone) and ligand partitioned into the membrane (i.e., not available for binding from solution phase but possibly available for binding from the membrane phase) (gray zone). While this scheme is more realistic, it is also more complex, and experimental determination of <i>K</i><sub>assoc</sub>, <i>k</i><sub>1</sub>, k<sub>–1</sub>, <i>K</i><sub>part</sub>, <i>k</i><sub>2</sub>, and <i>k</i><sub>–2</sub> may prove challenging. Additional terms (<i>K</i><sub>d</sub><sup>mem,int</sup>, <i>k</i><sub>3</sub>, <i>k</i><sub>–3</sub>) are needed to account for binding at membrane-exposed protein binding sites via lateral diffusion of partitioned compounds and to account for compounds that simply permeate through the membrane exiting into the intracellular compartment (<i>k</i><sub>4</sub>, <i>k</i><sub>–4</sub>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-12/acs.jmedchem.6b00358/20160617/images/large/jm-2016-003586_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00358&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Kinetics characterization is critically important for translating in vitro data to the cellular setting in vivo,<a onclick="showRef(event, 'ref40 ref41 ref42'); return false;" href="javascript:void(0);" class="ref ref40 ref41 ref42">(40-42)</a> and for this reason, we have been strongly advocating the use of a structure–kinetics paradigm in drug discovery for some time. Several studies reported by Pearlstein et al. point to the critical role of water in governing structure–kinetics relationships.<a onclick="showRef(event, 'ref30 ref31 ref32'); return false;" href="javascript:void(0);" class="ref ref30 ref31 ref32">(30-32)</a> We have used these principles to predict the structure–kinetics relationships of the β2-AR agonists and antagonists based on the overlay between the docked compounds and the solvation structure of apo β2-AR calculated using WATMD. The results suggest that compounds with experimentally determined slow dissociation from the receptor (i.e., slow <i>k</i><sub>off</sub>) cause efficient transfer of enthalpically unfavorable solvation from the binding site to bulk solvent (e.g., cyanopindolol compared with CGP12177).</div><div class="NLM_p">Some GPCRs, including CXCR4, rhodopsin,<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> and cannabinoid receptors,<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> are believed to contain membrane-exposed binding sites that are only accessible from within the bilayer (i.e., the bilayer serves as the target compartment). In such cases, high membrane partitioning (i.e., fast <i>k</i><sub>2</sub> and slow <i>k</i><sub>–2</sub>) should translate to faster receptor on-rates, and conversely, fast <i>k</i><sub>–2</sub> should translate to slower on-rates. On-rates may be further enhanced by more rapid rates of 2D lateral diffusion compared with diffusion in 3D volumes.<a onclick="showRef(event, 'ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref8 ref9">(8, 9)</a> Provisions for binding to membrane-exposed binding sites could be added to extend our model to other GPCRs (<a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>). However, binding from the membrane compartment is clearly not the case for the β2-AR, whose transmembrane pocket entrance is extracellular, and whose endogenous ligand, epinephrine, is polar.<a onclick="showRef(event, 'ref44 ref45'); return false;" href="javascript:void(0);" class="ref ref44 ref45">(44, 45)</a></div><div class="NLM_p last">While membrane association and partitioning can be pharmacologically beneficial, the trapping of amphiphilic compounds within membranes may also translate to high volumes of distribution in pharmacokinetic studies, which is typically associated with undesirably long lifetimes in the body and toxicity (e.g., phospholipidosis,<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> off-target effects<a onclick="showRef(event, 'ref47 ref48'); return false;" href="javascript:void(0);" class="ref ref47 ref48">(47, 48)</a>). The degree of drug–membrane interaction as determined by IAM chromatography has been found to correlate well with phospholipidosis risk.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> Cationic amphiphilic compounds are known to sequester in membranes, and some select phospholipogenics cause liver toxicity in preclinical species such as rats<a onclick="showRef(event, 'ref50 ref51 ref52 ref53'); return false;" href="javascript:void(0);" class="ref ref50 ref51 ref52 ref53">(50-53)</a> and dogs<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> at higher dose levels.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i4">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_46700" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_46700" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Our study highlights the importance of deconvoluting membrane– and protein–ligand binding contributions to measured ligand binding data (<i>K</i><sub>d</sub><sup>obs</sup>) for integral membrane proteins. Membrane affinity corrections can result in significant differences in the rank ordering of measured activity and SAR and greatly impact the direction of medicinal chemistry efforts. We have tested one such correction strategy consisting of using <i>K</i><sub>d</sub><sup>obs</sup> together with measured membrane association (<i>K</i><sub>IAM</sub>) to estimate the intrinsic binding affinity (<i>K</i><sub>d</sub><sup>int</sup>) by cross-checking the <i>K</i><sub>d</sub><sup>int</sup> against qualitative binding information derived from an in silico binding modes and solvation analysis.</div><div class="NLM_p last">The microkinetic model has important implications for lead selection and optimization as well as pharmacokinetic studies. In particular, potent <i>K</i><sub>d</sub><sup>obs</sup> resulting from a combination of weaker <i>K</i><sub>d</sub><sup>int</sup> and high membrane association may translate to higher dosing requirements due to the on-rate driven nature of receptor occupancy in such cases. In other words, weak intrinsic affinity <i>K</i><sub>d</sub><sup>int</sup> has to be compensated by higher concentration around the receptor to achieve the required receptor occupancy necessary for the functional response. Furthermore, rational lead optimization depends on understanding all of the different contributions to the observed in vitro potency and in vivo efficacy.</div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i5">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_08158" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_08158" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The experimental binding data used in the present work was measured using the inactive form of the β2-AR, thus all systems used inactive β2-AR crystal structure from PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2RH1">2RH1</a> in the OPM orientation.<a onclick="showRef(event, 'ref55 ref56'); return false;" href="javascript:void(0);" class="ref ref55 ref56">(55, 56)</a> All nonprotein molecules were deleted and only crystal waters inside the protein retained. The T4L loop required for crystallization purposes was deleted and replaced with the artificial loop from β2-AR PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GBR">4GBR</a>, residues 230–238.<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> The resulting structure then underwent the Structure Preparation and Protonate 3D steps in MOE.<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> All protonation states of acidic/basic residues were then manually verified for consistency of the hydrogen-bonding network. His296 was δ protonated and His93, His172, His178, and His269 were in ε protonation state. To charge neutralize the system, solvent exposed, noninteracting residues Lys97, Lys227, and Lys232 on the protein surface were modeled in the neutral state. These residues were chosen to ensure that the centroid of overall charge distribution of the protein system was minimally perturbed.</div><div class="NLM_p">Initial solvating waters within the protein were placed using 3D-RISM,<a onclick="showRef(event, 'ref59 ref60'); return false;" href="javascript:void(0);" class="ref ref59 ref60">(59, 60)</a> complementing retained crystal waters. The protein was inserted into a solvated lipid bilayer using the Charmm GUI.<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a> A membrane consisting of 272 POPC lipids total was built around the protein (136 per leaflet), and the system was further solvated with a 20 Å water layer above and below the protein. This number of lipid molecules was required to ensure that there will be at least three lipid layers between the protein and the edge of the periodic box. The final PDB of the membrane protein system was converted to AMBER format using the charmmlipid2amber script.<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a></div><div class="NLM_p">MOE was used for initial placement of atenolol, bisoprolol, metoprolol, cyanopindolol, nadolol, propranolol, and CGP12177 in the model protein structure based on the crystal structure of β2-AR bound to carazolol (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2RH1">2RH1</a><a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a>). Isoprenaline and salbutamol were built using the structure of adrenaline from 4LDO<a onclick="showRef(event, 'ref63'); return false;" href="javascript:void(0);" class="ref ref63">(63)</a> as a starting point. After building the ligands in a structurally intuitive manner, using the conserved interactions between the ligand β-hydroxy-amine motif and β2-AR Asp113 and Asn312 residues at the front of the binding pocket as an anchor, the coordinates of the ligand molecule were minimized with MOE. Salmeterol was docked into the β2-AR binding pocket using Glide SP from Schrödinger,<a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(64)</a> and the top scoring pose was selected. The coordinates of the ligand ICI-118,551 were obtained from the X-ray structure 3NY8.<a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(65)</a></div><div class="NLM_p">All systems underwent molecular dynamics simulation with an identical protocol using AMBER 14<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a> and PMEMD CUDA<a onclick="showRef(event, 'ref66 ref67 ref68'); return false;" href="javascript:void(0);" class="ref ref66 ref67 ref68">(66-68)</a> on a GPU cluster. The protein was modeled with the ff12SB protein force field,<a onclick="showRef(event, 'ref69'); return false;" href="javascript:void(0);" class="ref ref69">(69)</a> ligands with the <a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="7b0b1a09163b3d091408080f">[email protected]</a> force field<a onclick="showRef(event, 'ref70'); return false;" href="javascript:void(0);" class="ref ref70">(70)</a> and conformationally averaged AM1-BCC charges,<a onclick="showRef(event, 'ref26 ref71 ref72 ref73'); return false;" href="javascript:void(0);" class="ref ref26 ref71 ref72 ref73">(26, 71-73)</a> and lipids were modeled using the AMBER Lipid14 force field<a onclick="showRef(event, 'ref74 ref75 ref76'); return false;" href="javascript:void(0);" class="ref ref74 ref75 ref76">(74-76)</a> and water using the TIP3P model.<a onclick="showRef(event, 'ref77'); return false;" href="javascript:void(0);" class="ref ref77">(77)</a></div><div class="NLM_p">First, the system was energy minimized for 10000 steps, of which the first 5000 steps used the steepest descent method and the remaining steps used the conjugate gradient method.<a onclick="showRef(event, 'ref78'); return false;" href="javascript:void(0);" class="ref ref78">(78)</a> The system was then heated from 0 to 100 K using Langevin dynamics<a onclick="showRef(event, 'ref79'); return false;" href="javascript:void(0);" class="ref ref79">(79)</a> within a 5 ps constant volume run, with restraints on the protein, ligand, and lipids (force constant 10 kcal/mol/Å<sup>2</sup>). The volume was then allowed to change freely and the temperature increased to 303 K with a Langevin collision frequency of γ = 1 ps<sup>–1</sup>, and anisotropic Berendsen control of the pressure<a onclick="showRef(event, 'ref80'); return false;" href="javascript:void(0);" class="ref ref80">(80)</a> around 1 atm was applied by coupling the periodic box with a time constant of 2 ps for 100 ps. The same restraint of 10 kcal/mol/Å<sup>2</sup> was maintained on the protein, ligand and lipid molecules.</div><div class="NLM_p">The pressure relaxation time was then reduced to 1 ps, restraints were removed on lipids only and the system left to equilibrate for 5 ns in the constant pressure and constant temperature ensemble (NPT). Three-dimensional periodic boundary conditions with the usual minimum image convention were employed. Bonds involving hydrogen were constrained using the SHAKE algorithm,<a onclick="showRef(event, 'ref81'); return false;" href="javascript:void(0);" class="ref ref81">(81)</a> allowing a 2 fs time-step. Particle mesh Ewald was used to treat all electrostatic interactions<a onclick="showRef(event, 'ref82'); return false;" href="javascript:void(0);" class="ref ref82">(82)</a> beyond a cutoff of 10 Å. A long-range analytical dispersion correction was applied to energy and pressure. The system was then run for 5 ns with restraints on the ligand and protein backbone atoms followed by a further 5 ns NPT with restraints on the ligand and protein C-α atoms only. Finally, all restraints were removed and a 10 ns NPT production run performed. Trajectory coordinates were recorded every 10 ps.</div><div class="NLM_p">To collect trajectory data to allow solvation analysis using our in-house program WATMD, simulations were extended for a further 10 ns in NPT, with coordinates recorded every 0.25 ps to properly sample water positions. A total of five repeat WATMD simulations (10 ns each) were performed to validate the convergence of the solvation structure. The ligands remained stable in all simulations (see Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.6b00358/suppl_file/jm6b00358_si_001.pdf" class="ext-link">Figure S1</a> for root-mean-square deviation plots).</div><div class="NLM_p">All structural analyses of the resulting trajectories were performed with CPPTRAJ.<a onclick="showRef(event, 'ref83'); return false;" href="javascript:void(0);" class="ref ref83">(83)</a> Representative binding modes were obtained by calculating time averaged structures from the full production runs using CPPTRAJ. The average structures take flexibility into account and may therefore result in seemingly structurally distorted depictions. All modeling figures were generated with PYMOL,<a onclick="showRef(event, 'ref84'); return false;" href="javascript:void(0);" class="ref ref84">(84)</a> depicted hydrogen bonds using the default PYMOL definition of 3.6 Å distance cutoff and 63° angle cutoff.</div><div class="NLM_p">A number of methodologies have been developed which attempt to quantify the crucial role that water plays in molecular recognition.<a onclick="showRef(event, 'ref59 ref85 ref86 ref87 ref88 ref89 ref90 ref91'); return false;" href="javascript:void(0);" class="ref ref59 ref85 ref86 ref87 ref88 ref89 ref90 ref91">(59, 85-91)</a> In this work, our in-house tool WATMD<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> was used on each of the 10 ns trajectories with 0.25 ps frame rate to study solvation of the binding site. WATMD calculates the time-averaged distribution of atomic solvation, which essentially describes the occupancy of water at discrete positions in the binding pocket. Key advantages of WATMD compared to other implementations are (1) the protein conformation is not fixed during the MD simulations, and (2) water occupancy in the vicinity of the protein is calculated relative to water occupancy in bulk water. For a more detailed description of the methodology, see ref <a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">26</a>.</div><div class="NLM_p last">By studying the number of hydrogen bonds made by each high occupancy water site identified via WATMD (for this work, only those sites with occupancy higher than 2× bulk were considered), it is possible to qualitatively classify each of these sites as either enthalpically favorable or enthalpically unfavorable with respect to bulk water. Specifically, for TIP3P water simulations at physiological conditions, we consider that those solvation sites that make less than ∼3.6 hydrogen bonds are unfavorable, while those that make 3.6 or more hydrogen bonds are favorable. The value of 3.6 corresponds to the average number of hydrogen bonds of each water molecule in pure solvent, computed in silico at 1 atm and 300 K. Note that this rough categorization is only useful when a solvation site is enthalpically unfavorable, given that the entropy of confined waters is almost always unfavorable relative to bulk, so it is likely that the free energy of that particular water site is also unfavorable relative to bulk. Conversely, if a solvation site appears to be enthalpically favorable, nothing can be concluded about its free energy in the absence of an entropy estimate. WATMD has recently been extended to allow quantification of both local and global solvation enthalpies and entropies (and thus free energies),<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> which should improve the level of understanding of solvation for future work.</div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i6"><a href="/doi/suppl/10.1021/acs.jmedchem.6b00358">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_55161" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_55161" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.6b00358" class="ext-link">10.1021/acs.jmedchem.6b00358</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Full list of binding and kinetic data for all compounds used in this study, ligand RMSD plots from the molecular dynamics trajectories (<a href="/doi/suppl/10.1021/acs.jmedchem.6b00358/suppl_file/jm6b00358_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.6b00358/suppl_file/jm6b00358_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b00358/suppl_file/jm6b00358_si_001.pdf">jm6b00358_si_001.pdf (657.66 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b00358/suppl_file/jm6b00358_si_002.csv">jm6b00358_si_002.csv (0.53 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.6b00358" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_71258" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_71258" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Callum J. Dickson</span> - <span class="hlFld-Affiliation affiliation">Computer-Aided
Drug Discovery, Global Discovery Chemistry, Novartis Institutes for BioMedical Research, 100 Technology Square, Cambridge, Massachusetts 02139, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#6b080a07071e06450f0208001804052b05041d0a191f021845080406"><span class="__cf_email__" data-cfemail="ef8c8e83839a82c18b868c849c8081af8180998e9d9b869cc18c8082">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Viktor Hornak</span> - <span class="hlFld-Affiliation affiliation">Computer-Aided
Drug Discovery, Global Discovery Chemistry, Novartis Institutes for BioMedical Research, 100 Technology Square, Cambridge, Massachusetts 02139, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#92e4fbf9e6fde0bcfafde0fcf3f9d2fcfde4f3e0e6fbe1bcf1fdff"><span class="__cf_email__" data-cfemail="fb8d92908f9489d5939489959a90bb95948d9a898f9288d5989496">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Camilo Velez-Vega</span> - <span class="hlFld-Affiliation affiliation">Computer-Aided
Drug Discovery, Global Discovery Chemistry, Novartis Institutes for BioMedical Research, 100 Technology Square, Cambridge, Massachusetts 02139, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Daniel J. J. McKay</span> - <span class="hlFld-Affiliation affiliation">Computer-Aided
Drug Discovery, Global Discovery Chemistry, Novartis Institutes for BioMedical Research, 100 Technology Square, Cambridge, Massachusetts 02139, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">John Reilly</span> - <span class="hlFld-Affiliation affiliation">Global
Discovery Chemistry, Novartis Institutes
for BioMedical Research, 100 Technology Square, Cambridge, Massachusetts 02139, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">David A. Sandham</span> - <span class="hlFld-Affiliation affiliation">Global
Discovery Chemistry, Novartis Institutes
for BioMedical Research, 100 Technology Square, Cambridge, Massachusetts 02139, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Duncan Shaw</span> - <span class="hlFld-Affiliation affiliation">Global
Discovery Chemistry, Novartis Institutes
for BioMedical Research, 100 Technology Square, Cambridge, Massachusetts 02139, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Robin A. Fairhurst</span> - <span class="hlFld-Affiliation affiliation">Global
Discovery Chemistry, Novartis Institutes
for BioMedical Research, Novartis Pharma AG, Werk Klybeck, Postfach, CH-4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Steven J. Charlton</span> - <span class="hlFld-Affiliation affiliation">Cell
Signalling Research Group, School of Life Sciences, University of Nottingham, Queen’s Medical Centre, Nottingham NG7 2UH, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">David A. Sykes</span> - <span class="hlFld-Affiliation affiliation">Cell
Signalling Research Group, School of Life Sciences, University of Nottingham, Queen’s Medical Centre, Nottingham NG7 2UH, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Robert A. Pearlstein</span> - <span class="hlFld-Affiliation affiliation">Computer-Aided
Drug Discovery, Global Discovery Chemistry, Novartis Institutes for BioMedical Research, 100 Technology Square, Cambridge, Massachusetts 02139, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jose S. Duca</span> - <span class="hlFld-Affiliation affiliation">Computer-Aided
Drug Discovery, Global Discovery Chemistry, Novartis Institutes for BioMedical Research, 100 Technology Square, Cambridge, Massachusetts 02139, United States</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><p>The authors declare no competing financial interest.<br /></br></p></li></ul></div><div class="ack" id="ACK-d7e1596-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i7">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_81001" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_81001" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">C.J.D. thanks the Novartis Institutes for BioMedical Research Education Office for a presidential postdoctoral fellowship.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i8" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i8"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i9" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i9"> Abbreviations Used</h2><tr><td class="NLM_term">β2-AR, β2</td><td class="NLM_def"><p class="first last">β2 adrenergic receptor</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationships</p></td></tr><tr><td class="NLM_term">IAM</td><td class="NLM_def"><p class="first last">immobilized artificial membrane</p></td></tr><tr><td class="NLM_term">MD</td><td class="NLM_def"><p class="first last">molecular dynamics</p></td></tr><tr><td class="NLM_term">GPCR</td><td class="NLM_def"><p class="first last">G protein-coupled receptor</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i10">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_13042" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_13042" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 91 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Pierce, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Premont, R. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lefkowitz, R. J.</span><span> </span><span class="NLM_article-title">Seven-transmembrane receptors</span> <span class="citation_source-journal">Nat. Rev. Mol. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">639</span><span class="NLM_x">–</span> <span class="NLM_lpage">650</span><span class="refDoi"> DOI: 10.1038/nrm908</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1038%2Fnrm908" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=12209124" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BD38Xmslansr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2002&pages=639-650&author=K.+L.+Pierceauthor=R.+T.+Premontauthor=R.+J.+Lefkowitz&title=Seven-transmembrane+receptors&doi=10.1038%2Fnrm908"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Seven-transmembrane receptors</span></div><div class="casAuthors">Pierce, Kristen L.; Premont, Richard T.; Lefkowitz, Robert J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">639-650</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Seven-transmembrane receptors, which constitute the largest, most ubiquitous, and most versatile family of membrane receptors, are also the most common target of therapeutic drugs.  Recent findings indicate that the classical models of G-protein coupling and activation of second-messenger-generating enzymes do not fully explain their remarkably diverse biol. actions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5tA1lRPo14bVg90H21EOLACvtfcHk0lgene4w0UvWYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xmslansr4%253D&md5=35c79407b9a30df8eb018e31529ada81</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrm908&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm908%26sid%3Dliteratum%253Aachs%26aulast%3DPierce%26aufirst%3DK.%2BL.%26aulast%3DPremont%26aufirst%3DR.%2BT.%26aulast%3DLefkowitz%26aufirst%3DR.%2BJ.%26atitle%3DSeven-transmembrane%2520receptors%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2002%26volume%3D3%26spage%3D639%26epage%3D650%26doi%3D10.1038%2Fnrm908" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Anderson, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Linden, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rabe, K.</span><span> </span><span class="NLM_article-title">Why are long-acting beta-adrenoceptor agonists long-acting?</span> <span class="citation_source-journal">Eur. Respir. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">569</span><span class="NLM_x">–</span> <span class="NLM_lpage">578</span><span class="refDoi"> DOI: 10.1183/09031936.94.07030569</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1183%2F09031936.94.07030569" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=7912202" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADyaK2cXkvFyrurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=1994&pages=569-578&author=G.+Andersonauthor=A.+Lindenauthor=K.+Rabe&title=Why+are+long-acting+beta-adrenoceptor+agonists+long-acting%3F&doi=10.1183%2F09031936.94.07030569"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Why are long-acting beta-adrenoceptor agonists long-acting?</span></div><div class="casAuthors">Anderson, G. P.; Linden, A.; Rabe, K. F.</div><div class="citationInfo"><span class="NLM_cas:title">European Respiratory Journal</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">569-78</span>CODEN:
                <span class="NLM_cas:coden">ERJOEI</span>;
        ISSN:<span class="NLM_cas:issn">0903-1936</span>.
    </div><div class="casAbstract">A review with 60 refs.  The extended duration of bronchodilation due to formoterol and salmeterol greatly exceeds that of short acting beta2-adrenoceptor agonists, such as salbutamol or terbutaline.  This extended duration and their capacity to "reassert" airway smooth muscle relaxation in vitro despite repeated washing has prompted considerable debate on the underlying mechanism(s).  The comparative pharmacol., and mol. modeling of these drugs and of the beta2-adrenoceptor and its ligand binding core have cast doubt on the exosite/exoceptor model previously proposed to explain the behavior of salmeterol.  The authors present evidence supporting a unifying hypothesis that the duration of action both of formoterol and salmeterol is detd. principally by their physicochem. interactions with membrane lipid bilayers (plasmalemma diffusion microkinetic model), rather than putative distinct exosite/exoceptor binding sites in or near the beta2-adrenoceptor.  This model provides a clearer understanding of the pharmacol. profile of these drugs (rate of onset, duration, "reassertion", interaction with hydrophilic and hydrophobic beta2-adrenoceptor antagonists), and explains why in human airway smooth muscle in vitro a true relaxation-concn. response may not exist for salmeterol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbiLdcPisP0LVg90H21EOLACvtfcHk0lgene4w0UvWYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXkvFyrurc%253D&md5=e3235f7cb3d120c886336564010db96e</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1183%2F09031936.94.07030569&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1183%252F09031936.94.07030569%26sid%3Dliteratum%253Aachs%26aulast%3DAnderson%26aufirst%3DG.%26aulast%3DLinden%26aufirst%3DA.%26aulast%3DRabe%26aufirst%3DK.%26atitle%3DWhy%2520are%2520long-acting%2520beta-adrenoceptor%2520agonists%2520long-acting%253F%26jtitle%3DEur.%2520Respir.%2520J.%26date%3D1994%26volume%3D7%26spage%3D569%26epage%3D578%26doi%3D10.1183%2F09031936.94.07030569" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Sargent, D. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwyzer, R.</span><span> </span><span class="NLM_article-title">Membrane lipid phase as catalyst for peptide-receptor interactions</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1986</span><span class="NLM_x">, </span> <span class="NLM_volume">83</span><span class="NLM_x">, </span> <span class="NLM_fpage">5774</span><span class="NLM_x">–</span> <span class="NLM_lpage">5778</span><span class="refDoi"> DOI: 10.1073/pnas.83.16.5774</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1073%2Fpnas.83.16.5774" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=83&publication_year=1986&pages=5774-5778&author=D.+F.+Sargentauthor=R.+Schwyzer&title=Membrane+lipid+phase+as+catalyst+for+peptide-receptor+interactions&doi=10.1073%2Fpnas.83.16.5774"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1073%2Fpnas.83.16.5774&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.83.16.5774%26sid%3Dliteratum%253Aachs%26aulast%3DSargent%26aufirst%3DD.%2BF.%26aulast%3DSchwyzer%26aufirst%3DR.%26atitle%3DMembrane%2520lipid%2520phase%2520as%2520catalyst%2520for%2520peptide-receptor%2520interactions%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1986%26volume%3D83%26spage%3D5774%26epage%3D5778%26doi%3D10.1073%2Fpnas.83.16.5774" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Vauquelin, G.</span><span> </span><span class="NLM_article-title">On the ‘micro’-pharmacodynamic and pharmacokinetic mechanisms that contribute to long-lasting drug action</span> <span class="citation_source-journal">Expert Opin. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">1085</span><span class="NLM_x">–</span> <span class="NLM_lpage">1098</span><span class="refDoi"> DOI: 10.1517/17460441.2015.1067196</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1517%2F17460441.2015.1067196" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=1085-1098&author=G.+Vauquelin&title=On+the+%E2%80%98micro%E2%80%99-pharmacodynamic+and+pharmacokinetic+mechanisms+that+contribute+to+long-lasting+drug+action&doi=10.1517%2F17460441.2015.1067196"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1517%2F17460441.2015.1067196&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F17460441.2015.1067196%26sid%3Dliteratum%253Aachs%26aulast%3DVauquelin%26aufirst%3DG.%26atitle%3DOn%2520the%2520%25E2%2580%2598micro%25E2%2580%2599-pharmacodynamic%2520and%2520pharmacokinetic%2520mechanisms%2520that%2520contribute%2520to%2520long-lasting%2520drug%2520action%26jtitle%3DExpert%2520Opin.%2520Drug%2520Discovery%26date%3D2015%26volume%3D10%26spage%3D1085%26epage%3D1098%26doi%3D10.1517%2F17460441.2015.1067196" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Sykes, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parry, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reilly, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wright, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fairhurst, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Charlton, S. J.</span><span> </span><span class="NLM_article-title">Observed drug-receptor association rates are governed by membrane affinity: the importance of establishing “micro-pharmacokinetic/pharmacodynamic relationships” at the β2-adrenoceptor</span> <span class="citation_source-journal">Mol. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">85</span><span class="NLM_x">, </span> <span class="NLM_fpage">608</span><span class="NLM_x">–</span> <span class="NLM_lpage">617</span><span class="refDoi"> DOI: 10.1124/mol.113.090209</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1124%2Fmol.113.090209" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=24476583" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmsVyntbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=85&publication_year=2014&pages=608-617&author=D.+A.+Sykesauthor=C.+Parryauthor=J.+Reillyauthor=P.+Wrightauthor=R.+A.+Fairhurstauthor=S.+J.+Charlton&title=Observed+drug-receptor+association+rates+are+governed+by+membrane+affinity%3A+the+importance+of+establishing+%E2%80%9Cmicro-pharmacokinetic%2Fpharmacodynamic+relationships%E2%80%9D+at+the+%CE%B22-adrenoceptor&doi=10.1124%2Fmol.113.090209"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Observed drug-receptor association rates are governed by membrane affinity: the importance of establishing "micro-pharmacokinetic/pharmacodynamic relationships" at the β2-adrenoceptor</span></div><div class="casAuthors">Sykes, David A.; Parry, Cheryl; Reilly, John; Wright, Penny; Fairhurst, Robin A.; Charlton, Steven J.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">85</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">608-617, 10 pp.</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">1521-0111</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Current pharmacol. models for detg. affinity and kinetics of drugs for membrane receptors assume the interacting mols. are homogeneously distributed in the bulk aq. phase.  The phospholipid membrane can, however, provide a second compartment into which drugs can partition, particularly lipophilic/basic compds.  In this study we measured the phospholipid affinity and receptor binding kinetics of several clin. relevant β2-adrenoceptor agonists and antagonists and demonstrated that the degree of phospholipid interaction directly affects the obsd. kinetic assocn. rate (kon) and dissocn. const. (kd), but not the dissocn. rate (koff) from the target, by concg. drug in the local environment around the receptor.  When the local drug concn. was accounted for, the kon was comparable across the cohort and the cor. kd was directly related to the koff.  In conclusion, we propose a new approach to detg. the pharmacol. of drugs for membrane targets that accounts for differences in local drug concn. brought about by direct affinity for phospholipids, establishing "micro-pharmacokinetic/pharmacodynamic relationships" for drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYPaYJeo0NAbVg90H21EOLACvtfcHk0lhdJb-cn6GTQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmsVyntbg%253D&md5=b8038cbc7d9409ec8358f2836bd4df1f</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1124%2Fmol.113.090209&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.113.090209%26sid%3Dliteratum%253Aachs%26aulast%3DSykes%26aufirst%3DD.%2BA.%26aulast%3DParry%26aufirst%3DC.%26aulast%3DReilly%26aufirst%3DJ.%26aulast%3DWright%26aufirst%3DP.%26aulast%3DFairhurst%26aufirst%3DR.%2BA.%26aulast%3DCharlton%26aufirst%3DS.%2BJ.%26atitle%3DObserved%2520drug-receptor%2520association%2520rates%2520are%2520governed%2520by%2520membrane%2520affinity%253A%2520the%2520importance%2520of%2520establishing%2520%25E2%2580%259Cmicro-pharmacokinetic%252Fpharmacodynamic%2520relationships%25E2%2580%259D%2520at%2520the%2520%25CE%25B22-adrenoceptor%26jtitle%3DMol.%2520Pharmacol.%26date%3D2014%26volume%3D85%26spage%3D608%26epage%3D617%26doi%3D10.1124%2Fmol.113.090209" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Alikhani, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beer, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bentley, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bruce, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cuenoud, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fairhurst, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gedeck, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haberthuer, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hayden, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janus, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jordan, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lewis, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smithies, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wissler, E.</span><span> </span><span class="NLM_article-title">Long-chain formoterol analogues: an investigation into the effect of increasing amino-substituent chain length on the β2-adrenoceptor activity</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">4705</span><span class="NLM_x">–</span> <span class="NLM_lpage">4710</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2004.06.086</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1016%2Fj.bmcl.2004.06.086" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=15324892" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmvV2ht7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2004&pages=4705-4710&author=V.+Alikhaniauthor=D.+Beerauthor=D.+Bentleyauthor=I.+Bruceauthor=B.+M.+Cuenoudauthor=R.+A.+Fairhurstauthor=P.+Gedeckauthor=S.+Haberthuerauthor=C.+Haydenauthor=D.+Janusauthor=L.+Jordanauthor=C.+Lewisauthor=K.+Smithiesauthor=E.+Wissler&title=Long-chain+formoterol+analogues%3A+an+investigation+into+the+effect+of+increasing+amino-substituent+chain+length+on+the+%CE%B22-adrenoceptor+activity&doi=10.1016%2Fj.bmcl.2004.06.086"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Long-chain formoterol analogues: an investigation into the effect of increasing amino-substituent chain length on the β2-adrenoceptor activity</span></div><div class="casAuthors">Alikhani, Vahid; Beer, David; Bentley, David; Bruce, Ian; Cuenoud, Bernard M.; Fairhurst, Robin A.; Gedeck, Peter; Haberthuer, Sandra; Hayden, Claire; Janus, Diana; Jordan, Lynne; Lewis, Christine; Smithies, Kirsty; Wissler, Elke</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">4705-4710</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The synthesis of a series of long-chain formoterol analogs in which the terminal ether residue of the β-phenethylamino-substituent has been extended beyond the Me ether residue present in the parent compd. are described.  Evaluation of these analogs as β2-adrenoceptor agonists was used to provide an insight into the factors controlling the magnitude and duration of receptor activation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp69yYgHdjw2bVg90H21EOLACvtfcHk0lhdJb-cn6GTQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmvV2ht7o%253D&md5=55b9db6af59617a1e5dfbf2c80cb53bd</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2004.06.086&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2004.06.086%26sid%3Dliteratum%253Aachs%26aulast%3DAlikhani%26aufirst%3DV.%26aulast%3DBeer%26aufirst%3DD.%26aulast%3DBentley%26aufirst%3DD.%26aulast%3DBruce%26aufirst%3DI.%26aulast%3DCuenoud%26aufirst%3DB.%2BM.%26aulast%3DFairhurst%26aufirst%3DR.%2BA.%26aulast%3DGedeck%26aufirst%3DP.%26aulast%3DHaberthuer%26aufirst%3DS.%26aulast%3DHayden%26aufirst%3DC.%26aulast%3DJanus%26aufirst%3DD.%26aulast%3DJordan%26aufirst%3DL.%26aulast%3DLewis%26aufirst%3DC.%26aulast%3DSmithies%26aufirst%3DK.%26aulast%3DWissler%26aufirst%3DE.%26atitle%3DLong-chain%2520formoterol%2520analogues%253A%2520an%2520investigation%2520into%2520the%2520effect%2520of%2520increasing%2520amino-substituent%2520chain%2520length%2520on%2520the%2520%25CE%25B22-adrenoceptor%2520activity%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2004%26volume%3D14%26spage%3D4705%26epage%3D4710%26doi%3D10.1016%2Fj.bmcl.2004.06.086" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Stocks, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alcaraz, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bailey, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonnert, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cadogan, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Christie, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dixon, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Connolly, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cook, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fisher, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flaherty, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Humphries, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ingall, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jordan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lawson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mullen, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nicholls, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paine, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pairaudeau, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, A.</span><span> </span><span class="NLM_article-title">Discovery of AZD3199, an inhaled ultralong acting β2 receptor agonist with rapid onset of action</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">416</span><span class="NLM_x">–</span> <span class="NLM_lpage">421</span><span class="refDoi"> DOI: 10.1021/ml4005232</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml4005232" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=416-421&author=M.+J.+Stocksauthor=L.+Alcarazauthor=A.+Baileyauthor=R.+Bonnertauthor=E.+Cadoganauthor=J.+Christieauthor=J.+Dixonauthor=S.+Connollyauthor=A.+Cookauthor=A.+Fisherauthor=A.+Flahertyauthor=A.+Humphriesauthor=A.+Ingallauthor=S.+Jordanauthor=M.+Lawsonauthor=A.+Mullenauthor=D.+Nichollsauthor=S.+Paineauthor=G.+Pairaudeauauthor=A.+Young&title=Discovery+of+AZD3199%2C+an+inhaled+ultralong+acting+%CE%B22+receptor+agonist+with+rapid+onset+of+action&doi=10.1021%2Fml4005232"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Fml4005232&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml4005232%26sid%3Dliteratum%253Aachs%26aulast%3DStocks%26aufirst%3DM.%2BJ.%26aulast%3DAlcaraz%26aufirst%3DL.%26aulast%3DBailey%26aufirst%3DA.%26aulast%3DBonnert%26aufirst%3DR.%26aulast%3DCadogan%26aufirst%3DE.%26aulast%3DChristie%26aufirst%3DJ.%26aulast%3DDixon%26aufirst%3DJ.%26aulast%3DConnolly%26aufirst%3DS.%26aulast%3DCook%26aufirst%3DA.%26aulast%3DFisher%26aufirst%3DA.%26aulast%3DFlaherty%26aufirst%3DA.%26aulast%3DHumphries%26aufirst%3DA.%26aulast%3DIngall%26aufirst%3DA.%26aulast%3DJordan%26aufirst%3DS.%26aulast%3DLawson%26aufirst%3DM.%26aulast%3DMullen%26aufirst%3DA.%26aulast%3DNicholls%26aufirst%3DD.%26aulast%3DPaine%26aufirst%3DS.%26aulast%3DPairaudeau%26aufirst%3DG.%26aulast%3DYoung%26aufirst%3DA.%26atitle%3DDiscovery%2520of%2520AZD3199%252C%2520an%2520inhaled%2520ultralong%2520acting%2520%25CE%25B22%2520receptor%2520agonist%2520with%2520rapid%2520onset%2520of%2520action%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D5%26spage%3D416%26epage%3D421%26doi%3D10.1021%2Fml4005232" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">McCloskey, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poo, M. M.</span><span> </span><span class="NLM_article-title">Rates of membrane-associated reactions: reduction of dimensionality revisited</span> <span class="citation_source-journal">J. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1986</span><span class="NLM_x">, </span> <span class="NLM_volume">102</span><span class="NLM_x">, </span> <span class="NLM_fpage">88</span><span class="NLM_x">–</span> <span class="NLM_lpage">96</span><span class="refDoi"> DOI: 10.1083/jcb.102.1.88</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1083%2Fjcb.102.1.88" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=1986&pages=88-96&author=M.+A.+McCloskeyauthor=M.+M.+Poo&title=Rates+of+membrane-associated+reactions%3A+reduction+of+dimensionality+revisited&doi=10.1083%2Fjcb.102.1.88"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1083%2Fjcb.102.1.88&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1083%252Fjcb.102.1.88%26sid%3Dliteratum%253Aachs%26aulast%3DMcCloskey%26aufirst%3DM.%2BA.%26aulast%3DPoo%26aufirst%3DM.%2BM.%26atitle%3DRates%2520of%2520membrane-associated%2520reactions%253A%2520reduction%2520of%2520dimensionality%2520revisited%26jtitle%3DJ.%2520Cell%2520Biol.%26date%3D1986%26volume%3D102%26spage%3D88%26epage%3D96%26doi%3D10.1083%2Fjcb.102.1.88" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Vauquelin, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Packeu, A.</span><span> </span><span class="NLM_article-title">Ligands, their receptors and··· plasma membranes</span> <span class="citation_source-journal">Mol. Cell. Endocrinol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">311</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">10</span><span class="refDoi"> DOI: 10.1016/j.mce.2009.07.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1016%2Fj.mce.2009.07.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=19647036" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtV2ksL%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=311&publication_year=2009&pages=1-10&author=G.+Vauquelinauthor=A.+Packeu&title=Ligands%2C+their+receptors+and%C2%B7%C2%B7%C2%B7+plasma+membranes&doi=10.1016%2Fj.mce.2009.07.022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Ligands, their receptors and ... plasma membranes</span></div><div class="casAuthors">Vauquelin, G.; Packeu, A.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Endocrinology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">311</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">1-10</span>CODEN:
                <span class="NLM_cas:coden">MCEND6</span>;
        ISSN:<span class="NLM_cas:issn">0303-7207</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ireland Ltd.</span>)
        </div><div class="casAbstract">A review.  Ligand-receptor interactions are customarily described by equations that apply to solutes.  Yet, most receptors are present in cell membranes so that sufficiently lipophilic ligands could reach the receptor by a 2-dimensional approach within the membrane.  As summarized in this review, this may affect the ligand-receptor interaction in many ways.  Biophysicists have calcd. that, compared to a 3-dimensional approach from the liq. phase, such an approach could alter the time the ligands need to find a receptor.  Biochemists have found that ligand incorporation into lipid bilayers modifies their conformation.  This, along with the depth at which the ligands reside in the bilayer, will affect the probability of successful receptor interaction.  Novel mechanisms have also been introduced, including "exosite" binding and ligand translocation between the receptor α-helical transmembrane domains.  Pharmacologists have focused attention at ligand concns. in the membrane, their adsorption and release rates, and the effects thereof on ligand potency and residence time at the receptor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGquzgYx7fkdlbVg90H21EOLACvtfcHk0lhMzKamymHZrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtV2ksL%252FO&md5=e2ffcdac2bb4cb2a905df184a20e31b5</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.mce.2009.07.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.mce.2009.07.022%26sid%3Dliteratum%253Aachs%26aulast%3DVauquelin%26aufirst%3DG.%26aulast%3DPackeu%26aufirst%3DA.%26atitle%3DLigands%252C%2520their%2520receptors%2520and%25C2%25B7%25C2%25B7%25C2%25B7%2520plasma%2520membranes%26jtitle%3DMol.%2520Cell.%2520Endocrinol.%26date%3D2009%26volume%3D311%26spage%3D1%26epage%3D10%26doi%3D10.1016%2Fj.mce.2009.07.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Johnson, M.</span><span> </span><span class="NLM_article-title">Salmeterol: a novel drug for the treatment of asthma</span> <span class="citation_source-journal">Agents Actions Suppl.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1991</span><span class="NLM_x">, </span> <span class="NLM_volume">34</span><span class="NLM_x">, </span> <span class="NLM_fpage">79</span><span class="NLM_x">–</span> <span class="NLM_lpage">95</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1686530" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADyaK38XltFamsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=1991&pages=79-95&author=M.+Johnson&title=Salmeterol%3A+a+novel+drug+for+the+treatment+of+asthma"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Salmeterol:  a novel drug for the treatment of asthma</span></div><div class="casAuthors">Johnson, M.</div><div class="citationInfo"><span class="NLM_cas:title">Agents and Actions Supplements</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">New Drugs Asthma Ther.</span>),
    <span class="NLM_cas:pages">79-95</span>CODEN:
                <span class="NLM_cas:coden">AASUDJ</span>;
        ISSN:<span class="NLM_cas:issn">0379-0363</span>.
    </div><div class="casAbstract">Salmeterol (I) is the first of a new generation of rationally-designed, potent and selective β2-adrenoceptor agonists.  It has a unique pharmacol. profile that arises from a novel mechanism of action, whereby the mol. is long-acting by persisting at exo-receptor binding sites in the cell membrane.  I induces sustained bronchodilatation and has significant anti-inflammatory activity.  This combination of properties suggests that salmeterol may represent an important new advance in the treatment of bronchial asthma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpT2ecouZFxS7Vg90H21EOLACvtfcHk0lhMzKamymHZrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38XltFamsg%253D%253D&md5=9d55110aa68898b4cc92a8e46da9b224</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DJohnson%26aufirst%3DM.%26atitle%3DSalmeterol%253A%2520a%2520novel%2520drug%2520for%2520the%2520treatment%2520of%2520asthma%26jtitle%3DAgents%2520Actions%2520Suppl.%26date%3D1991%26volume%3D34%26spage%3D79%26epage%3D95" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Green, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spasoff, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coleman, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liggett, S. B.</span><span> </span><span class="NLM_article-title">Sustained activation of a G protein-coupled receptor via “anchored” agonist binding: molecular localization of the salmeterol exosite within the 2-adrenergic receptor</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">271</span><span class="NLM_x">, </span> <span class="NLM_fpage">24029</span><span class="NLM_x">–</span> <span class="NLM_lpage">24035</span><span class="refDoi"> DOI: 10.1074/jbc.271.39.24029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1074%2Fjbc.271.39.24029" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=271&publication_year=1996&pages=24029-24035&author=S.+A.+Greenauthor=A.+P.+Spasoffauthor=R.+A.+Colemanauthor=M.+Johnsonauthor=S.+B.+Liggett&title=Sustained+activation+of+a+G+protein-coupled+receptor+via+%E2%80%9Canchored%E2%80%9D+agonist+binding%3A+molecular+localization+of+the+salmeterol+exosite+within+the+2-adrenergic+receptor&doi=10.1074%2Fjbc.271.39.24029"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1074%2Fjbc.271.39.24029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.271.39.24029%26sid%3Dliteratum%253Aachs%26aulast%3DGreen%26aufirst%3DS.%2BA.%26aulast%3DSpasoff%26aufirst%3DA.%2BP.%26aulast%3DColeman%26aufirst%3DR.%2BA.%26aulast%3DJohnson%26aufirst%3DM.%26aulast%3DLiggett%26aufirst%3DS.%2BB.%26atitle%3DSustained%2520activation%2520of%2520a%2520G%2520protein-coupled%2520receptor%2520via%2520%25E2%2580%259Canchored%25E2%2580%259D%2520agonist%2520binding%253A%2520molecular%2520localization%2520of%2520the%2520salmeterol%2520exosite%2520within%2520the%25202-adrenergic%2520receptor%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1996%26volume%3D271%26spage%3D24029%26epage%3D24035%26doi%3D10.1074%2Fjbc.271.39.24029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Vauquelin, G.</span><span> </span><span class="NLM_article-title">Rebinding: or why drugs may act longer in vivo than expected from their in vitro target residence time</span> <span class="citation_source-journal">Expert Opin. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">927</span><span class="NLM_x">–</span> <span class="NLM_lpage">941</span><span class="refDoi"> DOI: 10.1517/17460441.2010.512037</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1517%2F17460441.2010.512037" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2010&pages=927-941&author=G.+Vauquelin&title=Rebinding%3A+or+why+drugs+may+act+longer+in+vivo+than+expected+from+their+in+vitro+target+residence+time&doi=10.1517%2F17460441.2010.512037"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1517%2F17460441.2010.512037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F17460441.2010.512037%26sid%3Dliteratum%253Aachs%26aulast%3DVauquelin%26aufirst%3DG.%26atitle%3DRebinding%253A%2520or%2520why%2520drugs%2520may%2520act%2520longer%2520in%2520vivo%2520than%2520expected%2520from%2520their%2520in%2520vitro%2520target%2520residence%2520time%26jtitle%3DExpert%2520Opin.%2520Drug%2520Discovery%26date%3D2010%26volume%3D5%26spage%3D927%26epage%3D941%26doi%3D10.1517%2F17460441.2010.512037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Vauquelin, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Charlton, S. J.</span><span> </span><span class="NLM_article-title">Long-lasting target binding and rebinding as mechanisms to prolong in vivo drug action</span> <span class="citation_source-journal">Br. J. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">161</span><span class="NLM_x">, </span> <span class="NLM_fpage">488</span><span class="NLM_x">–</span> <span class="NLM_lpage">508</span><span class="refDoi"> DOI: 10.1111/j.1476-5381.2010.00936.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1111%2Fj.1476-5381.2010.00936.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=20880390" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1Kju7zE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=161&publication_year=2010&pages=488-508&author=G.+Vauquelinauthor=S.+J.+Charlton&title=Long-lasting+target+binding+and+rebinding+as+mechanisms+to+prolong+in+vivo+drug+action&doi=10.1111%2Fj.1476-5381.2010.00936.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Long-lasting target binding and rebinding as mechanisms to prolong in vivo drug action</span></div><div class="casAuthors">Vauquelin, Georges; Charlton, Steven J.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">161</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">488-508</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">1476-5381</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  An increasing no. of examples in the literature suggest that the in vivo duration of drug action not only depends on macroscopic pharmacokinetic properties like plasma half-life and the time needed to equilibrate between the plasma and the effect compartments, but is also influenced by long-lasting target binding and rebinding.  The present review combines information from different research areas and simulations to explore the nature of these mechanisms and the conditions in which they are most prevalent.  Simulations reveal that these latter phenomena become esp. influential when there is no longer sufficient free drug around to maintain high levels of receptor occupancy.  There is not always a direct link between slow dissocn. and long-lasting in vivo target protection, as the rate of free drug elimination from the effect compartment is also a key influencing factor.  Local phenomena that hinder the diffusion of free drug mols. away from their target may allow them to consecutively bind to the same target and/or targets nearby (denoted as 'rebinding') even when their concn. in the bulk phase has already dropped to insignificant levels.  The micro-anat. properties of many effect compartments are likely to intensify this phenomenon.  By mimicking the complexity of tissues, intact cells offer the opportunity to investigate both mechanisms under the same, physiol. relevant conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqzSGNmKeaTzrVg90H21EOLACvtfcHk0liyq5KMd4Kkbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht1Kju7zE&md5=bd91e09a0fd79a2de6bf9ef0cfd3443e</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1111%2Fj.1476-5381.2010.00936.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1476-5381.2010.00936.x%26sid%3Dliteratum%253Aachs%26aulast%3DVauquelin%26aufirst%3DG.%26aulast%3DCharlton%26aufirst%3DS.%2BJ.%26atitle%3DLong-lasting%2520target%2520binding%2520and%2520rebinding%2520as%2520mechanisms%2520to%2520prolong%2520in%2520vivo%2520drug%2520action%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2010%26volume%3D161%26spage%3D488%26epage%3D508%26doi%3D10.1111%2Fj.1476-5381.2010.00936.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Coleman, R. A.</span><span> </span><span class="NLM_article-title">On the mechanism of the persistent action of salmeterol: what is the current position?</span> <span class="citation_source-journal">Br. J. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">158</span><span class="NLM_x">, </span> <span class="NLM_fpage">180</span><span class="NLM_x">–</span> <span class="NLM_lpage">182</span><span class="refDoi"> DOI: 10.1111/j.1476-5381.2009.00370.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1111%2Fj.1476-5381.2009.00370.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=19719780" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsVKmu7fL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=158&publication_year=2009&pages=180-182&author=R.+A.+Coleman&title=On+the+mechanism+of+the+persistent+action+of+salmeterol%3A+what+is+the+current+position%3F&doi=10.1111%2Fj.1476-5381.2009.00370.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">On the mechanism of the persistent action of salmeterol: what is the current position?</span></div><div class="casAuthors">Coleman, R. A.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">158</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">180-182</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">1476-5381</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  The mechanism of the long duration of action of salmeterol at β2-adrenoceptors has long been a matter of debate, and is still unresolved.  Szczuka and colleagues have both summarized the position to date and suggested a new mechanistic contender, receptor rebinding.  Despite this, they still do not come to any clear conclusion.  Much of the literature data that they have drawn upon appears contradictory, and math. models are inevitably flawed by the questionable validity of key values applied to them.  Although the issue will undoubtedly eventually be resolved, it will probably require investigators to apply carefully designed studies on simple exptl. systems such as isolated membranes or cultured cells.  Only then should studies be extended to more complex systems such as isolated prepns. of airways smooth muscle, where tissue bulk inevitably presents a complicating factor, particularly where relatively lipophilic compds. are concerned.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojZUYZ-ov-JrVg90H21EOLACvtfcHk0liyq5KMd4Kkbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsVKmu7fL&md5=f7c10b7848968bc91725fd8939ffb6c2</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1111%2Fj.1476-5381.2009.00370.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1476-5381.2009.00370.x%26sid%3Dliteratum%253Aachs%26aulast%3DColeman%26aufirst%3DR.%2BA.%26atitle%3DOn%2520the%2520mechanism%2520of%2520the%2520persistent%2520action%2520of%2520salmeterol%253A%2520what%2520is%2520the%2520current%2520position%253F%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2009%26volume%3D158%26spage%3D180%26epage%3D182%26doi%3D10.1111%2Fj.1476-5381.2009.00370.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Szczuka, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wennerberg, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Packeu, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vauquelin, G.</span><span> </span><span class="NLM_article-title">Molecular mechanisms for the persistent bronchodilatory effect of the β2-adrenoceptor agonist salmeterol</span> <span class="citation_source-journal">Br. J. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">158</span><span class="NLM_x">, </span> <span class="NLM_fpage">183</span><span class="NLM_x">–</span> <span class="NLM_lpage">194</span><span class="refDoi"> DOI: 10.1111/j.1476-5381.2009.00296.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1111%2Fj.1476-5381.2009.00296.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=19594756" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsVKlsr7M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=158&publication_year=2009&pages=183-194&author=A.+Szczukaauthor=M.+Wennerbergauthor=A.+Packeuauthor=G.+Vauquelin&title=Molecular+mechanisms+for+the+persistent+bronchodilatory+effect+of+the+%CE%B22-adrenoceptor+agonist+salmeterol&doi=10.1111%2Fj.1476-5381.2009.00296.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular mechanisms for the persistent bronchodilatory effect of the β2-adrenoceptor agonist salmeterol</span></div><div class="casAuthors">Szczuka, A.; Wennerberg, M.; Packeu, A.; Vauquelin, G.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">158</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">183-194</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">1476-5381</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">β2-adrenoceptor agonists are effective bronchodilators.  In vitro studies demonstrated long-lasting airway smooth muscle relaxation by salmeterol after washout, the quick disappearance of this effect in presence of antagonists and its recovery after antagonist removal.  Current explanations invoke salmeterol accumulation in the membrane ('diffusion microkinetic' model) or the existence of salmeterol-binding 'exosites'.  An alternative model based on 'rebinding' of a dissocd. ligand to the receptor mols. also produces an apparent decrease in the ligand's dissocn. rate in the absence of competing ligands.  Computer-assisted simulations were performed to follow the receptor-occupation by a salmeterol-like ligand and a competing ligand as a function of time.  The aptness of the models to describe the above in vitro findings was evaluated.  The 'diffusion microkinetic' model is sufficient to explain a long-lasting β2-adrenoceptor stimulation and reassertion as long as the membrane harbors a high concn. of the agonist.  At lower concn., 'rebinding' and, in second place, 'exosite' binding are likely to become operational.  The 'rebinding' and 'exosite' binding mechanisms take place at a sub-cellular/mol. scale.  Pending their demonstration by expts. on appropriate, simple models such as intact cells or membranes thereof, these mechanisms remain hypothetical in the case of salmeterol.  Airway smooth muscle contraction could also be governed by addnl. mechanisms that are particular to this macroscopic approach.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEC9rfvp5YbbVg90H21EOLACvtfcHk0liyq5KMd4Kkbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsVKlsr7M&md5=0515669d4f02ba8bd14f1bca84da4275</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1111%2Fj.1476-5381.2009.00296.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1476-5381.2009.00296.x%26sid%3Dliteratum%253Aachs%26aulast%3DSzczuka%26aufirst%3DA.%26aulast%3DWennerberg%26aufirst%3DM.%26aulast%3DPackeu%26aufirst%3DA.%26aulast%3DVauquelin%26aufirst%3DG.%26atitle%3DMolecular%2520mechanisms%2520for%2520the%2520persistent%2520bronchodilatory%2520effect%2520of%2520the%2520%25CE%25B22-adrenoceptor%2520agonist%2520salmeterol%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2009%26volume%3D158%26spage%3D183%26epage%3D194%26doi%3D10.1111%2Fj.1476-5381.2009.00296.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Slack, R. J.</span><span> </span><span class="NLM_article-title">Evidence for a non-β2-adrenoceptor binding site in human lung tissue for a subset of β2-adrenoceptor agonists</span> <span class="citation_source-journal">Pharmacol. Pharm.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">30</span><span class="NLM_x">–</span> <span class="NLM_lpage">36</span><span class="refDoi"> DOI: 10.4236/pp.2014.51006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.4236%2Fpp.2014.51006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlejt7vJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=30-36&author=R.+J.+Slack&title=Evidence+for+a+non-%CE%B22-adrenoceptor+binding+site+in+human+lung+tissue+for+a+subset+of+%CE%B22-adrenoceptor+agonists&doi=10.4236%2Fpp.2014.51006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Evidence for a non-β2-adrenoceptor binding site in human lung tissue for a subset of β2-adrenoceptor agonists</span></div><div class="casAuthors">Slack, Robert J.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacology & Pharmacy</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">30-36</span>CODEN:
                <span class="NLM_cas:coden">PPHHBN</span>;
        ISSN:<span class="NLM_cas:issn">2157-9431</span>.
    
            (<span class="NLM_cas:orgname">Scientific Research Publishing, Inc.</span>)
        </div><div class="casAbstract">The aim of this study was to compare the binding profile of a range of β2-adrenoceptor (β2-AR) agonists and antagonists in human lung tissue.  Radioligand satn. and competition binding expts. were performed by filtration with a β2-AR antagonist ([3H]propranolol) or agonist ([3H]vilanterol) radioligand and membrane fragments generated from lung parenchyma in the presence of 100 μM guanosine 5'-[β,γ-imido]triphosphate (Gpp(NH)p).  In membranes prepd. from human lung parenchyma, carmoterol, formoterol, ICI118551, propranolol and salbutamol resulted in inhibition of [3H]vilanterol binding to levels that were significantly different from indacaterol, salmeterol and vilanterol (ANOVA, Bonferroni post-test, P < 0.001 except formoterol vs indacaterol where P < 0.01).  Indacaterol and salmeterol resulted in inhibition of [3H]vilanterol binding to levels that were not significantly different from vilanterol (ANOVA, Bonferroni post-test, P > 0.05).  Indacaterol, salmeterol and vilanterol resulted in full inhibition of [3H]propranolol binding to levels not significantly different from ICI118551 (ANOVA, Bonferroni post-test, P > 0.05).  Indacaterol, salmeterol and vilanterol bind to an addnl. site in human lung parenchyma membranes that is distinct from the β2-AR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGryywv3XAzCzLVg90H21EOLACvtfcHk0lgIXblYHmbdhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlejt7vJ&md5=0e090c02447be8d16ff0742fe1ad16f7</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.4236%2Fpp.2014.51006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4236%252Fpp.2014.51006%26sid%3Dliteratum%253Aachs%26aulast%3DSlack%26aufirst%3DR.%2BJ.%26atitle%3DEvidence%2520for%2520a%2520non-%25CE%25B22-adrenoceptor%2520binding%2520site%2520in%2520human%2520lung%2520tissue%2520for%2520a%2520subset%2520of%2520%25CE%25B22-adrenoceptor%2520agonists%26jtitle%3DPharmacol.%2520Pharm.%26date%3D2014%26volume%3D5%26spage%3D30%26epage%3D36%26doi%3D10.4236%2Fpp.2014.51006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Ochsner, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jaekel, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mutz, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anderson, G. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">John, E.</span><span> </span><span class="NLM_article-title">Comparative biophysical analysis of the interaction of bronchodilating β2-adrenoceptor agonists with lipid membranes</span> <span class="citation_source-journal">Eur. J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">34</span><span class="NLM_x">, </span> <span class="NLM_fpage">451</span><span class="NLM_x">–</span> <span class="NLM_lpage">462</span><span class="refDoi"> DOI: 10.1016/S0223-5234(99)80095-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1016%2FS0223-5234%2899%2980095-6" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=1999&pages=451-462&author=M.+Ochsnerauthor=K.+Jaekelauthor=M.+Mutzauthor=G.+P.+Andersonauthor=E.+John&title=Comparative+biophysical+analysis+of+the+interaction+of+bronchodilating+%CE%B22-adrenoceptor+agonists+with+lipid+membranes&doi=10.1016%2FS0223-5234%2899%2980095-6"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2FS0223-5234%2899%2980095-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0223-5234%252899%252980095-6%26sid%3Dliteratum%253Aachs%26aulast%3DOchsner%26aufirst%3DM.%26aulast%3DJaekel%26aufirst%3DK.%26aulast%3DMutz%26aufirst%3DM.%26aulast%3DAnderson%26aufirst%3DG.%2BP.%26aulast%3DJohn%26aufirst%3DE.%26atitle%3DComparative%2520biophysical%2520analysis%2520of%2520the%2520interaction%2520of%2520bronchodilating%2520%25CE%25B22-adrenoceptor%2520agonists%2520with%2520lipid%2520membranes%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D1999%26volume%3D34%26spage%3D451%26epage%3D462%26doi%3D10.1016%2FS0223-5234%2899%2980095-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Bergendal, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lindén, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skoogh, B. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerspacher, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anderson, G. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Löfdahl, C. G.</span><span> </span><span class="NLM_article-title">Extent of salmeterol-mediated reassertion of relaxation in guinea-pig trachea pretreated with aliphatic side chain structural analogues</span> <span class="citation_source-journal">Br. J. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">117</span><span class="NLM_x">, </span> <span class="NLM_fpage">1009</span><span class="NLM_x">–</span> <span class="NLM_lpage">1015</span><span class="refDoi"> DOI: 10.1111/j.1476-5381.1996.tb16690.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1111%2Fj.1476-5381.1996.tb16690.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=8882590" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADyaK28XitVentLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=117&publication_year=1996&pages=1009-1015&author=A.+Bergendalauthor=A.+Lind%C3%A9nauthor=B.+E.+Skooghauthor=M.+Gerspacherauthor=G.+P.+Andersonauthor=C.+G.+L%C3%B6fdahl&title=Extent+of+salmeterol-mediated+reassertion+of+relaxation+in+guinea-pig+trachea+pretreated+with+aliphatic+side+chain+structural+analogues&doi=10.1111%2Fj.1476-5381.1996.tb16690.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Extent of salmeterol-mediated reassertion of relaxation in guinea pig trachea pretreated with aliphatic side chain structural analogs</span></div><div class="casAuthors">Bergendal, A.; Linden, A.; Skoogh, B.-E.; Gerspacher, M.; Anderson, G. P.; Loefdahl, C.-G.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">117</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1009-15</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">0007-1188</span>.
    
            (<span class="NLM_cas:orgname">Stockton</span>)
        </div><div class="casAbstract">Salmeterol is a potent, selective and long acting β2-adrenoceptor agonist.  In vitro, salmeterol exerts 'reassertion' relaxation of airways smooth muscle.  Reassertion relaxation refers to the capacity of salmeterol to cause repeated functional antagonism of induced contraction when airway smooth muscle is intermittently exposed to, then washed free from, β-adrenoceptor antagonists such as sotalol.  The mechanism(s) underlying reassertion relaxation are unknown but may relate to high affinity binding of the long aliph. side chain of salmeterol to an accessory site, distinct from the agonist recognition site, in or near the β2-adrenoceptor (exosite binding hypothesis).  To test the exosite hypothesis, three pure analogs of salmeterol, each exactly preserving the mol. structure of the aliph. side chain but with zero or low efficacy at the β2-adrenoceptor were synthesized.  The effect of pre-incubating guinea-pig tracheal smooth muscle with these analogs on salmeterol-induced reassertion relaxation was detd.  Computer Assisted Mol. Modeling of these mols. revealed that each of them exactly preserved the low energy linear conformation of the aliph. side chain of salmeterol.  Measurement of lipophilicity (octanol:water partition coeff.; log P) and direct partition into synthetic membranes (membrane partition coeff.; Kpmem) showed that all compds. had high affinity for lipids and membranes.  In particular the biophys. properties of CGP 59162 (log P 1.89, Kpmem 16500) were very similar to salmeterol (log P 1.73, Kpmem 16800).  Two of the analogs, CGP 54103 and D 2543 (1 μM), which are structural mimics of the sie chain of salmeterol, differing slightly in their length, did not prevent either the initial relaxation induced by salmeterol (0.1 μM) or the reassertion relaxation; however, it was not possible to det. whether either of these mols. occupied the β2-adrenoceptor.  The third analog, CGP 59162, which has the substituents on the active saligenin head group of salmeterol in transposed positions, itself exerted a weak β2-adrenoceptor-mediated relaxation antagonized by ICI 118551 (β2-selective antagonist) but not CGP 20712 (β1-selective antagonist) and, at higher concns. CGP 59162 caused reassertion relaxation suggesting that it may occupy and activate the β2-adrenoceptor in a manner analogous to salmeterol.  CGP 59162, at concns. up to ten fold molar excess, did not prevent or reduce salmeterol-induced reassertion relaxation.  In conclusion these data are not consistent with the existence of a distinct 'exosite' recognizing the aliph. side chain of salmeterol mediating reassertion.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIwfxNXAL4k7Vg90H21EOLACvtfcHk0lgIXblYHmbdhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XitVentLg%253D&md5=84fc71792ca22dcb18f6b0fda3b4c7c2</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1111%2Fj.1476-5381.1996.tb16690.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1476-5381.1996.tb16690.x%26sid%3Dliteratum%253Aachs%26aulast%3DBergendal%26aufirst%3DA.%26aulast%3DLind%25C3%25A9n%26aufirst%3DA.%26aulast%3DSkoogh%26aufirst%3DB.%2BE.%26aulast%3DGerspacher%26aufirst%3DM.%26aulast%3DAnderson%26aufirst%3DG.%2BP.%26aulast%3DL%25C3%25B6fdahl%26aufirst%3DC.%2BG.%26atitle%3DExtent%2520of%2520salmeterol-mediated%2520reassertion%2520of%2520relaxation%2520in%2520guinea-pig%2520trachea%2520pretreated%2520with%2520aliphatic%2520side%2520chain%2520structural%2520analogues%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D1996%26volume%3D117%26spage%3D1009%26epage%3D1015%26doi%3D10.1111%2Fj.1476-5381.1996.tb16690.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Jeppsson, A. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Löfdahl, C. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waldeck, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Widmark, E.</span><span> </span><span class="NLM_article-title">On the predictive value of experiments in vitro in the evaluation of the effect duration of bronchodilator drugs for local administration</span> <span class="citation_source-journal">Pulm. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1989</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">81</span><span class="NLM_x">–</span> <span class="NLM_lpage">85</span><span class="refDoi"> DOI: 10.1016/0952-0600(89)90028-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1016%2F0952-0600%2889%2990028-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=2577691" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A280%3ADyaK383hs1ClsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=1989&pages=81-85&author=A.+B.+Jeppssonauthor=C.+G.+L%C3%B6fdahlauthor=B.+Waldeckauthor=E.+Widmark&title=On+the+predictive+value+of+experiments+in+vitro+in+the+evaluation+of+the+effect+duration+of+bronchodilator+drugs+for+local+administration&doi=10.1016%2F0952-0600%2889%2990028-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">On the predictive value of experiments in vitro in the evaluation of the effect duration of bronchodilator drugs for local administration</span></div><div class="casAuthors">Jeppsson A B; Lofdahl C G; Waldeck B; Widmark E</div><div class="citationInfo"><span class="NLM_cas:title">Pulmonary pharmacology</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">81-5</span>
        ISSN:<span class="NLM_cas:issn">0952-0600</span>.
    </div><div class="casAbstract">Six bronchodilating beta-adrenoceptor agonists, clinically documented with respect to the duration of action after inhalation, were included in this study in vitro on the guinea-pig trachea.  Relaxation of carbachol contracted trachea strip preparations and inhibition of contraction of a vagus nerve-tracheal tube preparation were measured.  The relaxing effects of salbutamol and fenoterol (both with relatively short duration in man) were rapid in onset and easily reversed by washing in a drug-free medium.  The relaxation by salmeterol (long duration) and D2343 (intermediate duration) developed more slowly, resisted washing but was reversed by propranolol.  Formoterol (long duration) and salmefamol (intermediate duration) showed properties between these two extremes.  All test compounds inhibited the vagally-induced contractions of tracheal concentration dependently.  The EC50 values for the hydrophilic compounds salbutamol and fenoterol were higher with intra- as compared with extratracheal administration.  For the more lipophilic compounds formoterol, salmefamol, salmeterol and D2343, this difference was less pronounced.  A high lipophilicity and a retention by the tissue in vitro of a beta-adrenoceptor agonist may be factors contributing to a long effect duration after inhalation but a further selection has to be made in vivo since metabolic and circulatory effects may influence the effect kinetics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR6jE8Lji0hK_EeosoOnpOLfW6udTcc2ebF4BJUzztQHbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK383hs1ClsQ%253D%253D&md5=8a4d18187e98c34daf7f2735b922c727</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2F0952-0600%2889%2990028-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0952-0600%252889%252990028-8%26sid%3Dliteratum%253Aachs%26aulast%3DJeppsson%26aufirst%3DA.%2BB.%26aulast%3DL%25C3%25B6fdahl%26aufirst%3DC.%2BG.%26aulast%3DWaldeck%26aufirst%3DB.%26aulast%3DWidmark%26aufirst%3DE.%26atitle%3DOn%2520the%2520predictive%2520value%2520of%2520experiments%2520in%2520vitro%2520in%2520the%2520evaluation%2520of%2520the%2520effect%2520duration%2520of%2520bronchodilator%2520drugs%2520for%2520local%2520administration%26jtitle%3DPulm.%2520Pharmacol.%26date%3D1989%26volume%3D2%26spage%3D81%26epage%3D85%26doi%3D10.1016%2F0952-0600%2889%2990028-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Löfdahl, C.-G.</span><span> </span><span class="NLM_article-title">Basic pharmacology of new long-acting sympathomimetics</span> <span class="citation_source-journal">Lung</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1990</span><span class="NLM_x">, </span> <span class="NLM_volume">168</span><span class="NLM_x">, </span> <span class="NLM_fpage">18</span><span class="NLM_x">–</span> <span class="NLM_lpage">21</span><span class="refDoi"> DOI: 10.1007/BF02718108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1007%2FBF02718108" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=168&publication_year=1990&pages=18-21&author=C.-G.+L%C3%B6fdahl&title=Basic+pharmacology+of+new+long-acting+sympathomimetics&doi=10.1007%2FBF02718108"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1007%2FBF02718108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF02718108%26sid%3Dliteratum%253Aachs%26aulast%3DL%25C3%25B6fdahl%26aufirst%3DC.-G.%26atitle%3DBasic%2520pharmacology%2520of%2520new%2520long-acting%2520sympathomimetics%26jtitle%3DLung%26date%3D1990%26volume%3D168%26spage%3D18%26epage%3D21%26doi%3D10.1007%2FBF02718108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Valko, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Du, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bevan, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reynolds, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abraham, M. H.</span><span> </span><span class="NLM_article-title">Rapid-gradient HPLC method for measuring drug interactions with immobilized artificial membrane: Comparison with other lipophilicity measures</span> <span class="citation_source-journal">J. Pharm. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">89</span><span class="NLM_x">, </span> <span class="NLM_fpage">1085</span><span class="NLM_x">–</span> <span class="NLM_lpage">1096</span><span class="refDoi"> DOI: 10.1002/1520-6017(200008)89:8<1085::AID-JPS13>3.0.CO;2-N</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1002%2F1520-6017%28200008%2989%3A8%3C1085%3A%3AAID-JPS13%3E3.0.CO%3B2-N" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=89&publication_year=2000&pages=1085-1096&author=K.+Valkoauthor=C.+M.+Duauthor=C.+D.+Bevanauthor=D.+P.+Reynoldsauthor=M.+H.+Abraham&title=Rapid-gradient+HPLC+method+for+measuring+drug+interactions+with+immobilized+artificial+membrane%3A+Comparison+with+other+lipophilicity+measures&doi=10.1002%2F1520-6017%28200008%2989%3A8%3C1085%3A%3AAID-JPS13%3E3.0.CO%3B2-N"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1002%2F1520-6017%28200008%2989%3A8%3C1085%3A%3AAID-JPS13%3E3.0.CO%3B2-N&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F1520-6017%2528200008%252989%253A8%253C1085%253A%253AAID-JPS13%253E3.0.CO%253B2-N%26sid%3Dliteratum%253Aachs%26aulast%3DValko%26aufirst%3DK.%26aulast%3DDu%26aufirst%3DC.%2BM.%26aulast%3DBevan%26aufirst%3DC.%2BD.%26aulast%3DReynolds%26aufirst%3DD.%2BP.%26aulast%3DAbraham%26aufirst%3DM.%2BH.%26atitle%3DRapid-gradient%2520HPLC%2520method%2520for%2520measuring%2520drug%2520interactions%2520with%2520immobilized%2520artificial%2520membrane%253A%2520Comparison%2520with%2520other%2520lipophilicity%2520measures%26jtitle%3DJ.%2520Pharm.%2520Sci.%26date%3D2000%26volume%3D89%26spage%3D1085%26epage%3D1096%26doi%3D10.1002%2F1520-6017%28200008%2989%3A8%3C1085%3A%3AAID-JPS13%3E3.0.CO%3B2-N" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Abdiche, Y. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Myszka, D. G.</span><span> </span><span class="NLM_article-title">Probing the mechanism of drug/lipid membrane interactions using Biacore</span> <span class="citation_source-journal">Anal. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">328</span><span class="NLM_x">, </span> <span class="NLM_fpage">233</span><span class="NLM_x">–</span> <span class="NLM_lpage">243</span><span class="refDoi"> DOI: 10.1016/j.ab.2004.01.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1016%2Fj.ab.2004.01.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=15113702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjsVWmurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=328&publication_year=2004&pages=233-243&author=Y.+N.+Abdicheauthor=D.+G.+Myszka&title=Probing+the+mechanism+of+drug%2Flipid+membrane+interactions+using+Biacore&doi=10.1016%2Fj.ab.2004.01.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Probing the mechanism of drug/lipid membrane interactions using Biacore</span></div><div class="casAuthors">Abdiche, Yasmina N.; Myszka, David G.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">328</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">233-243</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science</span>)
        </div><div class="casAbstract">Assay conditions were established to screen a panel of drugs for binding to liposome surfaces using a surface plasmon resonance (SPR) biosensor.  Drugs were found to bind negligibly or reversibly or were retained on the liposome surface.  Cationic amphiphilic drugs fell into the last class and correlated with drugs that induce phospholipidosis in vivo.  To a first approxn., a single-site model yielded apparent binding affinities that adequately described a drug's dose-dependent binding to liposome surfaces.  Affinities ranged at least 1000-fold within the drug panel.  A liposome's drug-binding capacity and affinity depended on both the lipid headgroup and the drug's structure.  Although a drug's charge state generally dominated whether or not it remained bound to the liposome, subtle structural differences between members of certain drug families led to them having widely differing binding affinities.  A comparison between the dissocn. of drugs from liposome surfaces by Biacore and the lipid retention measurements detd. by a parallel artificial membrane permeability assay was drawn.  The results from this study demonstrate the potential of using SPR-based assays to characterize drug/liposome-binding interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBtD87YZF42bVg90H21EOLACvtfcHk0liXMwfd53z71Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjsVWmurc%253D&md5=e57a50d7112633499a19ea91b78b1ca1</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.ab.2004.01.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ab.2004.01.018%26sid%3Dliteratum%253Aachs%26aulast%3DAbdiche%26aufirst%3DY.%2BN.%26aulast%3DMyszka%26aufirst%3DD.%2BG.%26atitle%3DProbing%2520the%2520mechanism%2520of%2520drug%252Flipid%2520membrane%2520interactions%2520using%2520Biacore%26jtitle%3DAnal.%2520Biochem.%26date%3D2004%26volume%3D328%26spage%3D233%26epage%3D243%26doi%3D10.1016%2Fj.ab.2004.01.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Jiang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reilly, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Everatt, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Briard, E.</span><span> </span><span class="NLM_article-title">A rapid vesicle electrokinetic chromatography method for the in vitro prediction of non-specific binding for potential PET ligands</span> <span class="citation_source-journal">J. Pharm. Biomed. Anal.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">722</span><span class="NLM_x">–</span> <span class="NLM_lpage">729</span><span class="refDoi"> DOI: 10.1016/j.jpba.2010.11.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1016%2Fj.jpba.2010.11.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=21126842" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsF2mtrrO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=722-729&author=Z.+Jiangauthor=J.+Reillyauthor=B.+Everattauthor=E.+Briard&title=A+rapid+vesicle+electrokinetic+chromatography+method+for+the+in+vitro+prediction+of+non-specific+binding+for+potential+PET+ligands&doi=10.1016%2Fj.jpba.2010.11.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">A rapid vesicle electrokinetic chromatography method for the in vitro prediction of non-specific binding for potential PET ligands</span></div><div class="casAuthors">Jiang, Zhengjin; Reilly, John; Everatt, Brian; Briard, Emmanuelle</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmaceutical and Biomedical Analysis</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">722-729</span>CODEN:
                <span class="NLM_cas:coden">JPBADA</span>;
        ISSN:<span class="NLM_cas:issn">0731-7085</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">High non-specific binding (NSB) is one of the most common reasons for candidate failure in potential positron emission tomog. (PET) radiotracer development.  It is of interest to develop high throughput in vitro methods for predicting non-specific binding prior to radiolabeling, which would help guide radiotracer candidate selection and assist decision making in new radiotracer discovery.  We evaluated several electrokinetic chromatog. (EKC) systems to help identify PET ligands with low non-specific binding characteristics by mimicking the ligand-brain tissue interaction.  The measured retention factors of tracers in clin. use or terminated candidates within AOT vesicle EKC systems were compared with literature in vitro or in vivo NSB data.  We conclude that there is a statistical correlation between the chromatog. retention parameters of tested drugs and their NSB.  The AOT vesicle EKC method can provide NSB in vitro trend anal. for a large no. of drug candidates early in the novel radiotracer discovery process with minimal resources.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphoegUxlpyHrVg90H21EOLACvtfcHk0liXMwfd53z71Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsF2mtrrO&md5=dd95312fc52995ed2233f2d9ccf7012a</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.jpba.2010.11.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jpba.2010.11.004%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DZ.%26aulast%3DReilly%26aufirst%3DJ.%26aulast%3DEveratt%26aufirst%3DB.%26aulast%3DBriard%26aufirst%3DE.%26atitle%3DA%2520rapid%2520vesicle%2520electrokinetic%2520chromatography%2520method%2520for%2520the%2520in%2520vitro%2520prediction%2520of%2520non-specific%2520binding%2520for%2520potential%2520PET%2520ligands%26jtitle%3DJ.%2520Pharm.%2520Biomed.%2520Anal.%26date%3D2011%26volume%3D54%26spage%3D722%26epage%3D729%26doi%3D10.1016%2Fj.jpba.2010.11.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Mason, R. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rhodes, D. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herbette, L. G.</span><span> </span><span class="NLM_article-title">Reevaluating equilibrium and kinetic binding parameters for lipophilic drugs based on a structural model for drug interactions with biological membranes</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1991</span><span class="NLM_x">, </span> <span class="NLM_volume">34</span><span class="NLM_x">, </span> <span class="NLM_fpage">869</span><span class="NLM_x">–</span> <span class="NLM_lpage">877</span><span class="refDoi"> DOI: 10.1021/jm00107a001</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00107a001" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=1991&pages=869-877&author=R.+P.+Masonauthor=D.+G.+Rhodesauthor=L.+G.+Herbette&title=Reevaluating+equilibrium+and+kinetic+binding+parameters+for+lipophilic+drugs+based+on+a+structural+model+for+drug+interactions+with+biological+membranes&doi=10.1021%2Fjm00107a001"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fjm00107a001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00107a001%26sid%3Dliteratum%253Aachs%26aulast%3DMason%26aufirst%3DR.%2BP.%26aulast%3DRhodes%26aufirst%3DD.%2BG.%26aulast%3DHerbette%26aufirst%3DL.%2BG.%26atitle%3DReevaluating%2520equilibrium%2520and%2520kinetic%2520binding%2520parameters%2520for%2520lipophilic%2520drugs%2520based%2520on%2520a%2520structural%2520model%2520for%2520drug%2520interactions%2520with%2520biological%2520membranes%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1991%26volume%3D34%26spage%3D869%26epage%3D877%26doi%3D10.1021%2Fjm00107a001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Castanho, M. A. R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fernandes, M. X.</span><span> </span><span class="NLM_article-title">Lipid membrane-induced optimization for ligand–receptor docking: recent tools and insights for the “membrane catalysis” model</span> <span class="citation_source-journal">Eur. Biophys. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">35</span><span class="NLM_x">, </span> <span class="NLM_fpage">92</span><span class="NLM_x">–</span> <span class="NLM_lpage">103</span><span class="refDoi"> DOI: 10.1007/s00249-005-0007-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1007%2Fs00249-005-0007-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=16217647" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BD28XpsFKh" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2006&pages=92-103&author=M.+A.+R.+B.+Castanhoauthor=M.+X.+Fernandes&title=Lipid+membrane-induced+optimization+for+ligand%E2%80%93receptor+docking%3A+recent+tools+and+insights+for+the+%E2%80%9Cmembrane+catalysis%E2%80%9D+model&doi=10.1007%2Fs00249-005-0007-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Lipid membrane-induced optimization for ligand-receptor docking: recent tools and insights for the "membrane catalysis" model</span></div><div class="casAuthors">Castanho, Miguel A. R. B.; Fernandes, Miguel X.</div><div class="citationInfo"><span class="NLM_cas:title">European Biophysics Journal</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">92-103</span>CODEN:
                <span class="NLM_cas:coden">EBJOE8</span>;
        ISSN:<span class="NLM_cas:issn">0175-7571</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  Cells in living organisms are regulated by chem. and phys. stimuli from their environment.  Often, ligands interact with membrane receptors to trigger responses and Sargent and Schwyzer conceived a model to describe this process, "membrane catalysis".  There is a notion that the phys. organization of membranes can control the response of cells by speeding up reactions.  The authors revisit the "membrane catalysis" model in the light of recent tech., methodol. and theor. advances and how they can be exploited to highlight the details of membrane mediated ligand-receptor interactions.  The authors examine the possible effects that ligand concn. causes in the membrane catalysis and focus the authors' attention in techniques used to det. the partition const.  The hypothetical diffusional advantage assocd. with membrane catalysis is discussed and the applicability of existing models is assessed.  The role of in-depth location and orientation of ligands is explored emphasizing the contribution of new anal. methods and spectroscopic techniques.  Results suggest that membranes can optimize the interaction between ligands and receptors through several different effects but the relative contribution of each must be carefully investigated.  The authors certainly hope that the conjugation of the methodol. and tech. advances here reported will revive the interest in the membrane catalysis model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-spWQhjU7D7Vg90H21EOLACvtfcHk0lhy_xWnFLtdHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XpsFKh&md5=bf2a385e80f2d655cae22a1aad370f7a</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1007%2Fs00249-005-0007-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00249-005-0007-9%26sid%3Dliteratum%253Aachs%26aulast%3DCastanho%26aufirst%3DM.%2BA.%2BR.%2BB.%26aulast%3DFernandes%26aufirst%3DM.%2BX.%26atitle%3DLipid%2520membrane-induced%2520optimization%2520for%2520ligand%25E2%2580%2593receptor%2520docking%253A%2520recent%2520tools%2520and%2520insights%2520for%2520the%2520%25E2%2580%259Cmembrane%2520catalysis%25E2%2580%259D%2520model%26jtitle%3DEur.%2520Biophys.%2520J.%26date%3D2006%26volume%3D35%26spage%3D92%26epage%3D103%26doi%3D10.1007%2Fs00249-005-0007-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Velez-Vega, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McKay, D. J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aravamuthan, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pearlstein, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duca, J. S.</span><span> </span><span class="NLM_article-title">Time-averaged distributions of solute and solvent motions: exploring proton wires of GFP and PfM2DH</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">3344</span><span class="NLM_x">–</span> <span class="NLM_lpage">3361</span><span class="refDoi"> DOI: 10.1021/ci500571h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci500571h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFGktbbO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=3344-3361&author=C.+Velez-Vegaauthor=D.+J.+J.+McKayauthor=V.+Aravamuthanauthor=R.+Pearlsteinauthor=J.+S.+Duca&title=Time-averaged+distributions+of+solute+and+solvent+motions%3A+exploring+proton+wires+of+GFP+and+PfM2DH&doi=10.1021%2Fci500571h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Time-Averaged Distributions of Solute and Solvent Motions: Exploring Proton Wires of GFP and PfM2DH</span></div><div class="casAuthors">Velez-Vega, Camilo; McKay, Daniel J. J.; Aravamuthan, Vibhas; Pearlstein, Robert; Duca, Jose S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3344-3361</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Proton translocation pathways of selected variants of the green fluorescent protein (GFP) and Pseudomonas fluorescens mannitol 2-dehydrogenase (PfM2DH) were studied via an explicit solvent mol. dynamics-based anal. protocol that allows for direct quant. relation between a crystal structure and its time-averaged solute-solvent structure obtained from simulation.  The authors' study of GFP is in good agreement with previous research suggesting that the proton released from the chromophore upon photoexcitation can diffuse through an extended internal hydrogen bonding network that allows for the proton to exit to bulk or be recaptured by the anionic chromophore.  Conversely for PfM2DH, the authors identified the most probable ionization states of key residues along the proton escape channel from the catalytic site to bulk solvent, wherein the solute and high-d. solvent crystal structures of binary and ternary complexes were properly reproduced.  Furthermore, the authors proposed a plausible mechanism for this proton translocation process that is consistent with the state-dependent structural shifts obsd. in the authors' anal.  The time-averaged structures generated from the authors' analyses facilitate validation of MD simulation results and provide a comprehensive profile of the dynamic all-occupancy solvation network within and around a flexible solute, from which detailed hydrogen-bonding networks can be inferred.  In this way, potential drawbacks arising from the elucidation of these networks by examn. of static crystal structures or via alternate rigid-protein solvation anal. procedures can be overcome.  Complementary studies aimed at the effective use of the authors' methodol. for alternate implementations (e.g., ligand design) are currently underway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpub7nL0UsqGrVg90H21EOLACvtfcHk0lhy_xWnFLtdHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFGktbbO&md5=d34a5696e577371ff4db5630923dedb0</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fci500571h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci500571h%26sid%3Dliteratum%253Aachs%26aulast%3DVelez-Vega%26aufirst%3DC.%26aulast%3DMcKay%26aufirst%3DD.%2BJ.%2BJ.%26aulast%3DAravamuthan%26aufirst%3DV.%26aulast%3DPearlstein%26aufirst%3DR.%26aulast%3DDuca%26aufirst%3DJ.%2BS.%26atitle%3DTime-averaged%2520distributions%2520of%2520solute%2520and%2520solvent%2520motions%253A%2520exploring%2520proton%2520wires%2520of%2520GFP%2520and%2520PfM2DH%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D3344%26epage%3D3361%26doi%3D10.1021%2Fci500571h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Velez-Vega, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McKay, D. J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kurtzman, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aravamuthan, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pearlstein, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duca, J. S.</span><span> </span><span class="NLM_article-title">Estimation of solvation entropy and enthalpy via analysis of water oxygen–hydrogen correlations</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">5090</span><span class="NLM_x">–</span> <span class="NLM_lpage">5102</span><span class="refDoi"> DOI: 10.1021/acs.jctc.5b00439</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jctc.5b00439" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1Wqs77F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=5090-5102&author=C.+Velez-Vegaauthor=D.+J.+J.+McKayauthor=T.+Kurtzmanauthor=V.+Aravamuthanauthor=R.+A.+Pearlsteinauthor=J.+S.+Duca&title=Estimation+of+solvation+entropy+and+enthalpy+via+analysis+of+water+oxygen%E2%80%93hydrogen+correlations&doi=10.1021%2Facs.jctc.5b00439"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Estimation of Solvation Entropy and Enthalpy via Analysis of Water Oxygen-Hydrogen Correlations</span></div><div class="casAuthors">Velez-Vega, Camilo; McKay, Daniel J. J.; Kurtzman, Tom; Aravamuthan, Vibhas; Pearlstein, Robert A.; Duca, Jose S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">5090-5102</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A statistical-mech. framework for estn. of solvation entropies and enthalpies is proposed, which is based on the anal. of water as a mixt. of correlated water oxygens and water hydrogens.  Entropic contributions of increasing order are cast in terms of a Mutual Information Expansion that is evaluated to pairwise interactions.  In turn, the enthalpy is computed directly from a distance-based hydrogen bonding energy algorithm.  The resulting expressions are employed for grid-based analyses of Mol. Dynamics simulations.  In this first assessment of the methodol., we obtained global ests. of the excess entropy and enthalpy of water that are in good agreement with expt. and examd. the method's ability to enable detailed elucidation of solvation thermodn. structures, which can provide valuable knowledge toward mol. design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcC8EPpRstJbVg90H21EOLACvtfcHk0lhy_xWnFLtdHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1Wqs77F&md5=a53e1cb93683f72291d013f77c181d2a</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Facs.jctc.5b00439&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jctc.5b00439%26sid%3Dliteratum%253Aachs%26aulast%3DVelez-Vega%26aufirst%3DC.%26aulast%3DMcKay%26aufirst%3DD.%2BJ.%2BJ.%26aulast%3DKurtzman%26aufirst%3DT.%26aulast%3DAravamuthan%26aufirst%3DV.%26aulast%3DPearlstein%26aufirst%3DR.%2BA.%26aulast%3DDuca%26aufirst%3DJ.%2BS.%26atitle%3DEstimation%2520of%2520solvation%2520entropy%2520and%2520enthalpy%2520via%2520analysis%2520of%2520water%2520oxygen%25E2%2580%2593hydrogen%2520correlations%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2015%26volume%3D11%26spage%3D5090%26epage%3D5102%26doi%3D10.1021%2Facs.jctc.5b00439" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Leeson, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Springthorpe, B.</span><span> </span><span class="NLM_article-title">The influence of drug-like concepts on decision-making in medicinal chemistry</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">881</span><span class="NLM_x">–</span> <span class="NLM_lpage">890</span><span class="refDoi"> DOI: 10.1038/nrd2445</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1038%2Fnrd2445" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=17971784" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BD2sXht1els7rL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2007&pages=881-890&author=P.+D.+Leesonauthor=B.+Springthorpe&title=The+influence+of+drug-like+concepts+on+decision-making+in+medicinal+chemistry&doi=10.1038%2Fnrd2445"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">The influence of drug-like concepts on decision-making in medicinal chemistry</span></div><div class="casAuthors">Leeson, Paul D.; Springthorpe, Brian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">881-890</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Despite the wide acceptance of drug-like principles such as the 'rule of five', this anal. of mols. currently being synthesized in leading pharmaceutical companies reveals that their phys. properties differ significantly from those of recently discovered oral drugs.  The marked increase in lipophilicity in particular could increase the likelihood of attrition in drug development.  The application of guidelines linked to the concept of drug-likeness, such as the 'rule of five', has gained wide acceptance as an approach to reduce attrition in drug discovery and development.  However, despite this acceptance, anal. of recent trends reveals that the phys. properties of mols. that are currently being synthesized in leading drug discovery companies differ significantly from those of recently discovered oral drugs and compds. in clin. development.  The consequences of the marked increase in lipophilicity - the most important drug-like phys. property - include a greater likelihood of lack of selectivity and attrition in drug development.  Tackling the threat of compd.-related toxicol. attrition needs to move to the mainstream of medicinal chem. decision-making.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTUeonLD6iebVg90H21EOLACvtfcHk0ljbxA7Gj9i7zQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXht1els7rL&md5=5665a9e2dc266a1ee096e20625757ef7</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1038%2Fnrd2445&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2445%26sid%3Dliteratum%253Aachs%26aulast%3DLeeson%26aufirst%3DP.%2BD.%26aulast%3DSpringthorpe%26aufirst%3DB.%26atitle%3DThe%2520influence%2520of%2520drug-like%2520concepts%2520on%2520decision-making%2520in%2520medicinal%2520chemistry%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2007%26volume%3D6%26spage%3D881%26epage%3D890%26doi%3D10.1038%2Fnrd2445" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Katritch, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reynolds, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cherezov, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hanson, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, C. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yeager, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abagyan, R.</span><span> </span><span class="NLM_article-title">Analysis of full and partial agonists binding to β(2)-adrenergic receptor suggests a role of transmembrane helix V in agonist-specific conformational changes</span> <span class="citation_source-journal">J. Mol. Recognit.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">307</span><span class="NLM_x">–</span> <span class="NLM_lpage">318</span><span class="refDoi"> DOI: 10.1002/jmr.949</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1002%2Fjmr.949" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=19353579" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BD1MXnslGis7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2009&pages=307-318&author=V.+Katritchauthor=K.+A.+Reynoldsauthor=V.+Cherezovauthor=M.+A.+Hansonauthor=C.+B.+Rothauthor=M.+Yeagerauthor=R.+Abagyan&title=Analysis+of+full+and+partial+agonists+binding+to+%CE%B2%282%29-adrenergic+receptor+suggests+a+role+of+transmembrane+helix+V+in+agonist-specific+conformational+changes&doi=10.1002%2Fjmr.949"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Analysis of full and partial agonists binding to β2-adrenergic receptor suggests a role of transmembrane helix V in agonist-specific conformational changes</span></div><div class="casAuthors">Katritch, Vsevolod; Reynolds, Kimberly A.; Cherezov, Vadim; Hanson, Michael A.; Roth, Christopher B.; Yeager, Mark; Abagyan, Ruben</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Recognition</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">307-318</span>CODEN:
                <span class="NLM_cas:coden">JMORE4</span>;
        ISSN:<span class="NLM_cas:issn">0952-3499</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons Ltd.</span>)
        </div><div class="casAbstract">The 2.4 Å crystal structure of the β2-adrenergic receptor (β2AR) in complex with the high-affinity inverse agonist (-)-carazolol provides a detailed structural framework for the anal. of ligand recognition by adrenergic receptors.  Insights into agonist binding and the corresponding conformational changes triggering G-protein coupled receptor (GPCR) activation mechanism are of special interest.  While the carazolol pocket captured in the β2AR crystal structure accommodates (-)-isoproterenol and other agonists without steric clashes, a finite movement of the flexible extracellular part of TM-V helix (TM-Ve) obtained by receptor optimization in the presence of docked ligand can further improve the calcd. binding affinities for agonist compds.  Tilting of TM-Ve towards the receptor axis provides a more complete description of polar receptor-ligand interactions for full and partial agonists, by enabling optimal engagement of agonists with two exptl. identified anchor sites, formed by Asp 113/Asn 312 and Ser 203/Ser 204/Ser 207 side chains.  Further, receptor models incorporating a flexible TM-V backbone allow reliable prediction of binding affinities for a set of diverse ligands, suggesting potential utility of this approach to design of effective and subtype-specific agonists for adrenergic receptors.  Systematic differences in capacity of partial, full and inverse agonists to induce TM-V helix tilt in the β2AR model suggest potential role of TM-V as a conformational "rheostat" involved in the whole spectrum of β2AR responses to small mol. signals.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjjBCjcVJWTrVg90H21EOLACvtfcHk0ljbxA7Gj9i7zQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXnslGis7k%253D&md5=01e2c841b200697382e551b3cf104451</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1002%2Fjmr.949&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjmr.949%26sid%3Dliteratum%253Aachs%26aulast%3DKatritch%26aufirst%3DV.%26aulast%3DReynolds%26aufirst%3DK.%2BA.%26aulast%3DCherezov%26aufirst%3DV.%26aulast%3DHanson%26aufirst%3DM.%2BA.%26aulast%3DRoth%26aufirst%3DC.%2BB.%26aulast%3DYeager%26aufirst%3DM.%26aulast%3DAbagyan%26aufirst%3DR.%26atitle%3DAnalysis%2520of%2520full%2520and%2520partial%2520agonists%2520binding%2520to%2520%25CE%25B2%25282%2529-adrenergic%2520receptor%2520suggests%2520a%2520role%2520of%2520transmembrane%2520helix%2520V%2520in%2520agonist-specific%2520conformational%2520changes%26jtitle%3DJ.%2520Mol.%2520Recognit.%26date%3D2009%26volume%3D22%26spage%3D307%26epage%3D318%26doi%3D10.1002%2Fjmr.949" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Pearlstein, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sherman, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abel, R.</span><span> </span><span class="NLM_article-title">Contributions of water transfer energy to protein-ligand association and dissociation barriers: WaterMap analysis of a series of p38α MAP kinase inhibitors</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">81</span><span class="NLM_x">, </span> <span class="NLM_fpage">1509</span><span class="NLM_x">–</span> <span class="NLM_lpage">1526</span><span class="refDoi"> DOI: 10.1002/prot.24276</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1002%2Fprot.24276" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=23468227" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVaktbrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=2013&pages=1509-1526&author=R.+A.+Pearlsteinauthor=W.+Shermanauthor=R.+Abel&title=Contributions+of+water+transfer+energy+to+protein-ligand+association+and+dissociation+barriers%3A+WaterMap+analysis+of+a+series+of+p38%CE%B1+MAP+kinase+inhibitors&doi=10.1002%2Fprot.24276"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Contributions of water transfer energy to protein-ligand association and dissociation barriers: Watermap analysis of a series of p38α MAP kinase inhibitors</span></div><div class="casAuthors">Pearlstein, Robert A.; Sherman, Woody; Abel, Robert</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Bioinformatics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">81</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1509-1526</span>CODEN:
                <span class="NLM_cas:coden">PSFBAF</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">In our previous work, we proposed that desolvation and resolvation of the binding sites of proteins can serve as the slowest steps during ligand assocn. and dissocn., resp., and tested this hypothesis on two protein-ligand systems with known binding kinetics behavior.  In the present work, we test this hypothesis on another kinetically-detd. protein-ligand system-that of p38α and eight Type II BIRB 796 inhibitor analogs.  The kon values among the inhibitor analogs are narrowly distributed (104 ≤ kon ≤ 105 M-1s-1), suggesting a common rate-detg. step, whereas the koff values are widely distributed (10-1 ≤ koff ≤ 10-6 s-1), suggesting a spectrum of rate-detg. steps.  We calcd. the solvation properties of the DFG-out protein conformation using an explicit solvent mol. dynamics simulation and thermodn. anal. method implemented in WaterMap to predict the enthalpic and entropic costs of water transfer to and from bulk solvent incurred upon assocn. and dissocn. of each inhibitor.  The results suggest that the rate-detg. step for assocn. consists of the transfer of a common set of enthalpically favorable solvating water mols. from the binding site to bulk solvent.  The rate-detg. step for inhibitor dissocn. consists of the transfer of water from bulk solvent to specific binding site positions that are unfavorably solvated in the apo protein, and evacuated during ligand assocn.  Different sets of unfavorable solvation are evacuated by each ligand, and the obsd. dissocn. barriers are qual. consistent with the calcd. solvation free energies of those sets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4gR0ZeehncLVg90H21EOLACvtfcHk0ljbxA7Gj9i7zQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVaktbrK&md5=45e9cac74fbe0e13b0469b5ce9df622c</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1002%2Fprot.24276&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.24276%26sid%3Dliteratum%253Aachs%26aulast%3DPearlstein%26aufirst%3DR.%2BA.%26aulast%3DSherman%26aufirst%3DW.%26aulast%3DAbel%26aufirst%3DR.%26atitle%3DContributions%2520of%2520water%2520transfer%2520energy%2520to%2520protein-ligand%2520association%2520and%2520dissociation%2520barriers%253A%2520WaterMap%2520analysis%2520of%2520a%2520series%2520of%2520p38%25CE%25B1%2520MAP%2520kinase%2520inhibitors%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2013%26volume%3D81%26spage%3D1509%26epage%3D1526%26doi%3D10.1002%2Fprot.24276" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Pearlstein, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, Q.-Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yowe, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levell, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dale, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaushik, V. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Daniels, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hanrahan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sherman, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abel, R.</span><span> </span><span class="NLM_article-title">New hypotheses about the structure–function of proprotein convertase subtilisin/kexin type 9: analysis of the epidermal growth factor-like repeat A docking site using WaterMap</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">78</span><span class="NLM_x">, </span> <span class="NLM_fpage">2571</span><span class="NLM_x">–</span> <span class="NLM_lpage">2586</span><span class="refDoi"> DOI: 10.1002/prot.22767</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1002%2Fprot.22767" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=2010&pages=2571-2586&author=R.+A.+Pearlsteinauthor=Q.-Y.+Huauthor=J.+Zhouauthor=D.+Yoweauthor=J.+Levellauthor=B.+Daleauthor=V.+K.+Kaushikauthor=D.+Danielsauthor=S.+Hanrahanauthor=W.+Shermanauthor=R.+Abel&title=New+hypotheses+about+the+structure%E2%80%93function+of+proprotein+convertase+subtilisin%2Fkexin+type+9%3A+analysis+of+the+epidermal+growth+factor-like+repeat+A+docking+site+using+WaterMap&doi=10.1002%2Fprot.22767"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1002%2Fprot.22767&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.22767%26sid%3Dliteratum%253Aachs%26aulast%3DPearlstein%26aufirst%3DR.%2BA.%26aulast%3DHu%26aufirst%3DQ.-Y.%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DYowe%26aufirst%3DD.%26aulast%3DLevell%26aufirst%3DJ.%26aulast%3DDale%26aufirst%3DB.%26aulast%3DKaushik%26aufirst%3DV.%2BK.%26aulast%3DDaniels%26aufirst%3DD.%26aulast%3DHanrahan%26aufirst%3DS.%26aulast%3DSherman%26aufirst%3DW.%26aulast%3DAbel%26aufirst%3DR.%26atitle%3DNew%2520hypotheses%2520about%2520the%2520structure%25E2%2580%2593function%2520of%2520proprotein%2520convertase%2520subtilisin%252Fkexin%2520type%25209%253A%2520analysis%2520of%2520the%2520epidermal%2520growth%2520factor-like%2520repeat%2520A%2520docking%2520site%2520using%2520WaterMap%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2010%26volume%3D78%26spage%3D2571%26epage%3D2586%26doi%3D10.1002%2Fprot.22767" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Tran, Q.-T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Farid, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Erdemli, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pearlstein, R.</span><span> </span><span class="NLM_article-title">The translocation kinetics of antibiotics through porin OmpC: insights from structure-based solvation mapping using WaterMap</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">81</span><span class="NLM_x">, </span> <span class="NLM_fpage">291</span><span class="NLM_x">–</span> <span class="NLM_lpage">299</span><span class="refDoi"> DOI: 10.1002/prot.24185</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1002%2Fprot.24185" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=2013&pages=291-299&author=Q.-T.+Tranauthor=S.+Williamsauthor=R.+Faridauthor=G.+Erdemliauthor=R.+Pearlstein&title=The+translocation+kinetics+of+antibiotics+through+porin+OmpC%3A+insights+from+structure-based+solvation+mapping+using+WaterMap&doi=10.1002%2Fprot.24185"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1002%2Fprot.24185&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.24185%26sid%3Dliteratum%253Aachs%26aulast%3DTran%26aufirst%3DQ.-T.%26aulast%3DWilliams%26aufirst%3DS.%26aulast%3DFarid%26aufirst%3DR.%26aulast%3DErdemli%26aufirst%3DG.%26aulast%3DPearlstein%26aufirst%3DR.%26atitle%3DThe%2520translocation%2520kinetics%2520of%2520antibiotics%2520through%2520porin%2520OmpC%253A%2520insights%2520from%2520structure-based%2520solvation%2520mapping%2520using%2520WaterMap%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2013%26volume%3D81%26spage%3D291%26epage%3D299%26doi%3D10.1002%2Fprot.24185" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Warren, G. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Andrews, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Capelli, A.-M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clarke, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">LaLonde, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lambert, M. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lindvall, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nevins, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Semus, S. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senger, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tedesco, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wall, I. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woolven, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peishoff, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Head, M. S.</span><span> </span><span class="NLM_article-title">A Critical Assessment of Docking Programs and Scoring Functions</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">5912</span><span class="NLM_x">–</span> <span class="NLM_lpage">5931</span><span class="refDoi"> DOI: 10.1021/jm050362n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm050362n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BD2MXnslOrtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=5912-5931&author=G.+L.+Warrenauthor=C.+W.+Andrewsauthor=A.-M.+Capelliauthor=B.+Clarkeauthor=J.+LaLondeauthor=M.+H.+Lambertauthor=M.+Lindvallauthor=N.+Nevinsauthor=S.+F.+Semusauthor=S.+Sengerauthor=G.+Tedescoauthor=I.+D.+Wallauthor=J.+M.+Woolvenauthor=C.+E.+Peishoffauthor=M.+S.+Head&title=A+Critical+Assessment+of+Docking+Programs+and+Scoring+Functions&doi=10.1021%2Fjm050362n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">A Critical Assessment of Docking Programs and Scoring Functions</span></div><div class="casAuthors">Warren, Gregory L.; Andrews, C. Webster; Capelli, Anna-Maria; Clarke, Brian; LaLonde, Judith; Lambert, Millard H.; Lindvall, Mika; Nevins, Neysa; Semus, Simon F.; Senger, Stefan; Tedesco, Giovanna; Wall, Ian D.; Woolven, James M.; Peishoff, Catherine E.; Head, Martha S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">5912-5931</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Docking is a computational technique that samples conformations of small mols. in protein binding sites; scoring functions are used to assess which of these conformations best complements the protein binding site.  An evaluation of 10 docking programs and 37 scoring functions was conducted against eight proteins of seven protein types for three tasks: binding mode prediction, virtual screening for lead identification, and rank-ordering by affinity for lead optimization.  All of the docking programs were able to generate ligand conformations similar to crystallog. detd. protein/ligand complex structures for at least one of the targets.  However, scoring functions were less successful at distinguishing the crystallog. conformation from the set of docked poses.  Docking programs identified active compds. from a pharmaceutically relevant pool of decoy compds.; however, no single program performed well for all of the targets.  For prediction of compd. affinity, none of the docking programs or scoring functions made a useful prediction of ligand binding affinity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfJn8_WHSjurVg90H21EOLACvtfcHk0lg43CQBsVaX1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXnslOrtrY%253D&md5=463ab40c3d23e67b62df408fc5e7f08d</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fjm050362n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm050362n%26sid%3Dliteratum%253Aachs%26aulast%3DWarren%26aufirst%3DG.%2BL.%26aulast%3DAndrews%26aufirst%3DC.%2BW.%26aulast%3DCapelli%26aufirst%3DA.-M.%26aulast%3DClarke%26aufirst%3DB.%26aulast%3DLaLonde%26aufirst%3DJ.%26aulast%3DLambert%26aufirst%3DM.%2BH.%26aulast%3DLindvall%26aufirst%3DM.%26aulast%3DNevins%26aufirst%3DN.%26aulast%3DSemus%26aufirst%3DS.%2BF.%26aulast%3DSenger%26aufirst%3DS.%26aulast%3DTedesco%26aufirst%3DG.%26aulast%3DWall%26aufirst%3DI.%2BD.%26aulast%3DWoolven%26aufirst%3DJ.%2BM.%26aulast%3DPeishoff%26aufirst%3DC.%2BE.%26aulast%3DHead%26aufirst%3DM.%2BS.%26atitle%3DA%2520Critical%2520Assessment%2520of%2520Docking%2520Programs%2520and%2520Scoring%2520Functions%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D5912%26epage%3D5931%26doi%3D10.1021%2Fjm050362n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Plewczynski, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Łaźniewski, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Augustyniak, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ginalski, K.</span><span> </span><span class="NLM_article-title">Can we trust docking results? Evaluation of seven commonly used programs on PDBbind database</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x">, </span> <span class="NLM_fpage">742</span><span class="NLM_x">–</span> <span class="NLM_lpage">755</span><span class="refDoi"> DOI: 10.1002/jcc.21643</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1002%2Fjcc.21643" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=20812323" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BC3MXosVemuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2011&pages=742-755&author=D.+Plewczynskiauthor=M.+%C5%81a%C5%BAniewskiauthor=R.+Augustyniakauthor=K.+Ginalski&title=Can+we+trust+docking+results%3F+Evaluation+of+seven+commonly+used+programs+on+PDBbind+database&doi=10.1002%2Fjcc.21643"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Can we trust docking results? Evaluation of seven commonly used programs on PDBbind database</span></div><div class="casAuthors">Plewczynski, Dariusz; Lazniewski, Michal; Augustyniak, Rafal; Ginalski, Krzysztof</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">742-755</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  Docking is one of the most commonly used techniques in drug design.  It is used for both identifying correct poses of a ligand in the binding site of a protein as well as for the estn. of the strength of protein-ligand interaction.  Because millions of compds. must be screened, before a suitable target for biol. testing can be identified, all calcns. should be done in a reasonable time frame.  Thus, all programs currently in use exploit empirically based algorithms, avoiding systematic search of the conformational space.  Similarly, the scoring is done using simple equations, which makes it possible to speed up the entire process.  Therefore, docking results have to be verified by subsequent in vitro studies.  The purpose of our work was to evaluate seven popular docking programs (Surflex, LigandFit, Glide, GOLD, FlexX, eHiTS, and AutoDock) on the extensive dataset composed of 1300 protein-ligands complexes from PDBbind 2007 database, where exptl. measured binding affinity values were also available.  We compared independently the ability of proper posing [according to Root mean square deviation (or Root mean square distance) of predicted conformations vs. the corresponding native one] and scoring (by calcg. the correlation between docking score and ligand binding strength).  To our knowledge, it is the first large-scale docking evaluation that covers both aspects of docking programs, i.e., predicting ligand conformation and calcg. the strength of its binding.  More than 1000 protein-ligand pairs cover a wide range of different protein families and inhibitor classes.  Our results clearly showed that the ligand binding conformation could be identified in most cases by using the existing software, yet we still obsd. the lack of universal scoring function for all types of mols. and protein families.  © 2010 Wiley Periodicals, Inc.  J Comput Chem, 2011.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrrK3tNTWoTdrVg90H21EOLACvtfcHk0lg43CQBsVaX1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXosVemuw%253D%253D&md5=61745292dd287a8139df98c6a2b72d40</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1002%2Fjcc.21643&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.21643%26sid%3Dliteratum%253Aachs%26aulast%3DPlewczynski%26aufirst%3DD.%26aulast%3D%25C5%2581a%25C5%25BAniewski%26aufirst%3DM.%26aulast%3DAugustyniak%26aufirst%3DR.%26aulast%3DGinalski%26aufirst%3DK.%26atitle%3DCan%2520we%2520trust%2520docking%2520results%253F%2520Evaluation%2520of%2520seven%2520commonly%2520used%2520programs%2520on%2520PDBbind%2520database%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2011%26volume%3D32%26spage%3D742%26epage%3D755%26doi%3D10.1002%2Fjcc.21643" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Kollman, P.</span><span> </span><span class="NLM_article-title">Free energy calculations: Applications to chemical and biochemical phenomena</span> <span class="citation_source-journal">Chem. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">93</span><span class="NLM_x">, </span> <span class="NLM_fpage">2395</span><span class="NLM_x">–</span> <span class="NLM_lpage">2417</span><span class="refDoi"> DOI: 10.1021/cr00023a004</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr00023a004" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADyaK3sXmt1Sktr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=1993&pages=2395-2417&author=P.+Kollman&title=Free+energy+calculations%3A+Applications+to+chemical+and+biochemical+phenomena&doi=10.1021%2Fcr00023a004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Free energy calculations: Applications to chemical and biochemical phenomena</span></div><div class="casAuthors">Kollman, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2395-417</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    </div><div class="casAbstract">A review with 252 refs. about applications of free energy calcns. employing mol. dynamics or Monte Carlo methods to a variety of chem. and biochem. phenomena.  The focus is on applications of such calcns. to mol. solvation, mol. assocn., macromol. stability, and enzyme catalysis.  The mols. discussed range from monovalent ions and small mols. to proteins and nucleic acids.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrN1w4SyFDqCLVg90H21EOLACvtfcHk0liF_kGZGqXcow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXmt1Sktr0%253D&md5=f12326b24734ea092996cf22efb6ebd6</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fcr00023a004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr00023a004%26sid%3Dliteratum%253Aachs%26aulast%3DKollman%26aufirst%3DP.%26atitle%3DFree%2520energy%2520calculations%253A%2520Applications%2520to%2520chemical%2520and%2520biochemical%2520phenomena%26jtitle%3DChem.%2520Rev.%26date%3D1993%26volume%3D93%26spage%3D2395%26epage%3D2417%26doi%3D10.1021%2Fcr00023a004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Gilson, M. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Given, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bush, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCammon, J. A.</span><span> </span><span class="NLM_article-title">The statistical-thermodynamic basis for computation of binding affinities: a critical review</span> <span class="citation_source-journal">Biophys. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">72</span><span class="NLM_x">, </span> <span class="NLM_fpage">1047</span><span class="NLM_x">–</span> <span class="NLM_lpage">1069</span><span class="refDoi"> DOI: 10.1016/S0006-3495(97)78756-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1016%2FS0006-3495%2897%2978756-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=9138555" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADyaK2sXhsVeisbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=1997&pages=1047-1069&author=M.+K.+Gilsonauthor=J.+A.+Givenauthor=B.+L.+Bushauthor=J.+A.+McCammon&title=The+statistical-thermodynamic+basis+for+computation+of+binding+affinities%3A+a+critical+review&doi=10.1016%2FS0006-3495%2897%2978756-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">The statistical-thermodynamic basis for computation of binding affinities: a critical review</span></div><div class="casAuthors">Gilson, Michael K.; Given, James A.; Bush, Bruce L.; Mccammon, J. Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Biophysical Journal</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1047-1069</span>CODEN:
                <span class="NLM_cas:coden">BIOJAU</span>;
        ISSN:<span class="NLM_cas:issn">0006-3495</span>.
    
            (<span class="NLM_cas:orgname">Biophysical Society</span>)
        </div><div class="casAbstract">A review with many refs.  Although the statistical thermodn. of noncovalent binding has been considered in a no. of theor. papers, few methods of computing binding affinities are derived explicitly from this underlying theory.  This has contributed to uncertainty and controversy in certain areas.  This article therefore reviews and extends the connections of some important computational methods with the underlying statistical thermodn.  A derivation of the std. free energy of binding forms the basis of this review.  This derivation should be useful in formulating novel computational methods for predicting binding affinities.  It also permits several important points to be established.  For example, it is found that the double annihilation method of computing binding energy does not yield the std. free energy of binding, but can be modified to yield this quantity.  The derivation also makes it possible to define clearly the changes in translational, rotational, configurational, and solvent entropy upon binding.  It is argued that mol. mass has a negligible effect upon the std. free energy of binding for biomol. systems, and that the cratic entropy defined by Gurney is not a useful concept.  In addn., the use of continuum models of the solvent in binding calcns. is reviewed, and a formalism is presented for incorporating a limited no. of solvent mols. explicitly.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtZwKHjQ8gW7Vg90H21EOLACvtfcHk0liF_kGZGqXcow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXhsVeisbo%253D&md5=2bf97b219b9d15fccab6cb50dc7686aa</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2FS0006-3495%2897%2978756-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0006-3495%252897%252978756-3%26sid%3Dliteratum%253Aachs%26aulast%3DGilson%26aufirst%3DM.%2BK.%26aulast%3DGiven%26aufirst%3DJ.%2BA.%26aulast%3DBush%26aufirst%3DB.%2BL.%26aulast%3DMcCammon%26aufirst%3DJ.%2BA.%26atitle%3DThe%2520statistical-thermodynamic%2520basis%2520for%2520computation%2520of%2520binding%2520affinities%253A%2520a%2520critical%2520review%26jtitle%3DBiophys.%2520J.%26date%3D1997%26volume%3D72%26spage%3D1047%26epage%3D1069%26doi%3D10.1016%2FS0006-3495%2897%2978756-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">Mobley, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Graves, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chodera, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McReynolds, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dill, K. A.</span><span> </span><span class="NLM_article-title">Predicting Absolute Ligand Binding Free Energies to a Simple Model Site</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">371</span><span class="NLM_x">, </span> <span class="NLM_fpage">1118</span><span class="NLM_x">–</span> <span class="NLM_lpage">1134</span><span class="refDoi"> DOI: 10.1016/j.jmb.2007.06.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1016%2Fj.jmb.2007.06.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=17599350" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BD2sXot12rt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=371&publication_year=2007&pages=1118-1134&author=D.+L.+Mobleyauthor=A.+P.+Gravesauthor=J.+D.+Choderaauthor=A.+C.+McReynoldsauthor=B.+K.+Shoichetauthor=K.+A.+Dill&title=Predicting+Absolute+Ligand+Binding+Free+Energies+to+a+Simple+Model+Site&doi=10.1016%2Fj.jmb.2007.06.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Predicting Absolute Ligand Binding Free Energies to a Simple Model Site</span></div><div class="casAuthors">Mobley, David L.; Graves, Alan P.; Chodera, John D.; McReynolds, Andrea C.; Shoichet, Brian K.; Dill, Ken A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">371</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1118-1134</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A central challenge in structure-based ligand design is the accurate prediction of binding free energies.  Here the authors apply alchem. free energy calcns. in explicit solvent to predict ligand binding in a model cavity in T4 lysozyme.  Even in this simple site, there are challenges.  The authors made systematic improvements, beginning with single poses from docking, then including multiple poses, addnl. protein conformational changes, and using an improved charge model.  Computed abs. binding free energies had an RMS error of 1.9 kcal/mol relative to previously detd. exptl. values.  In blind prospective tests, the methods correctly discriminated between several true ligands and decoys in a set of putative binders identified by docking.  In these prospective tests, the RMS error in predicted binding free energies relative to those subsequently detd. exptl. was only 0.6 kcal/mol.  X-ray crystal structures of the new ligands bound in the cavity corresponded closely to predictions from the free energy calcns., but sometimes differed from those predicted by docking.  Finally, the authors examd. the impact of holding the protein rigid, as in docking, with a view to learning how approxns. made in docking affect accuracy and how they may be improved.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqD1-EYEODQ47Vg90H21EOLACvtfcHk0liF_kGZGqXcow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXot12rt7o%253D&md5=2480139daa5a0857c59e821d2ecbce18</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2007.06.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2007.06.002%26sid%3Dliteratum%253Aachs%26aulast%3DMobley%26aufirst%3DD.%2BL.%26aulast%3DGraves%26aufirst%3DA.%2BP.%26aulast%3DChodera%26aufirst%3DJ.%2BD.%26aulast%3DMcReynolds%26aufirst%3DA.%2BC.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DDill%26aufirst%3DK.%2BA.%26atitle%3DPredicting%2520Absolute%2520Ligand%2520Binding%2520Free%2520Energies%2520to%2520a%2520Simple%2520Model%2520Site%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2007%26volume%3D371%26spage%3D1118%26epage%3D1134%26doi%3D10.1016%2Fj.jmb.2007.06.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Chodera, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mobley, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shirts, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dixon, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Branson, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pande, V. S.</span><span> </span><span class="NLM_article-title">Alchemical free energy methods for drug discovery: progress and challenges</span> <span class="citation_source-journal">Curr. Opin. Struct. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">150</span><span class="NLM_x">–</span> <span class="NLM_lpage">160</span><span class="refDoi"> DOI: 10.1016/j.sbi.2011.01.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1016%2Fj.sbi.2011.01.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=21349700" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BC3MXkt1Kisrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2011&pages=150-160&author=J.+D.+Choderaauthor=D.+L.+Mobleyauthor=M.+R.+Shirtsauthor=R.+W.+Dixonauthor=K.+Bransonauthor=V.+S.+Pande&title=Alchemical+free+energy+methods+for+drug+discovery%3A+progress+and+challenges&doi=10.1016%2Fj.sbi.2011.01.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Alchemical free energy methods for drug discovery: Progress and challenges</span></div><div class="casAuthors">Chodera, John D.; Mobley, David L.; Shirts, Michael R.; Dixon, Richard W.; Branson, Kim; Pande, Vijay S.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">150-160</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Improved rational drug design methods are needed to lower the cost and increase the success rate of drug discovery and development.  Alchem. binding free energy calcns., one potential tool for rational design, have progressed rapidly over the past decade, but still fall short of providing robust tools for pharmaceutical engineering.  Recent studies, esp. on model receptor systems, have clarified many of the challenges that must be overcome for robust predictions of binding affinity to be useful in rational design.  In this review, inspired by a recent joint academic/industry meeting organized by the authors, we discuss these challenges and suggest a no. of promising approaches for overcoming them.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobUBHsGUCIWLVg90H21EOLACvtfcHk0liF_kGZGqXcow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXkt1Kisrk%253D&md5=fa17c0982d921cd1c32cd095fd34420e</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.sbi.2011.01.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.sbi.2011.01.011%26sid%3Dliteratum%253Aachs%26aulast%3DChodera%26aufirst%3DJ.%2BD.%26aulast%3DMobley%26aufirst%3DD.%2BL.%26aulast%3DShirts%26aufirst%3DM.%2BR.%26aulast%3DDixon%26aufirst%3DR.%2BW.%26aulast%3DBranson%26aufirst%3DK.%26aulast%3DPande%26aufirst%3DV.%2BS.%26atitle%3DAlchemical%2520free%2520energy%2520methods%2520for%2520drug%2520discovery%253A%2520progress%2520and%2520challenges%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2011%26volume%3D21%26spage%3D150%26epage%3D160%26doi%3D10.1016%2Fj.sbi.2011.01.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pierce, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krilov, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lupyan, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robinson, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dahlgren, M. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greenwood, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Romero, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Masse, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knight, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steinbrecher, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beuming, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Damm, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harder, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sherman, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brewer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wester, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murcko, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Farid, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mobley, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berne, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friesner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abel, R.</span><span> </span><span class="NLM_article-title">Accurate and Reliable Prediction of Relative Ligand Binding Potency in Prospective Drug Discovery by Way of a Modern Free-Energy Calculation Protocol and Force Field</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">137</span><span class="NLM_x">, </span> <span class="NLM_fpage">2695</span><span class="NLM_x">–</span> <span class="NLM_lpage">2703</span><span class="refDoi"> DOI: 10.1021/ja512751q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja512751q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsF2iuro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=137&publication_year=2015&pages=2695-2703&author=L.+Wangauthor=Y.+Wuauthor=Y.+Dengauthor=B.+Kimauthor=L.+Pierceauthor=G.+Krilovauthor=D.+Lupyanauthor=S.+Robinsonauthor=M.+K.+Dahlgrenauthor=J.+Greenwoodauthor=D.+L.+Romeroauthor=C.+Masseauthor=J.+L.+Knightauthor=T.+Steinbrecherauthor=T.+Beumingauthor=W.+Dammauthor=E.+Harderauthor=W.+Shermanauthor=M.+Brewerauthor=R.+Westerauthor=M.+Murckoauthor=L.+Fryeauthor=R.+Faridauthor=T.+Linauthor=D.+L.+Mobleyauthor=W.+L.+Jorgensenauthor=B.+J.+Berneauthor=R.+A.+Friesnerauthor=R.+Abel&title=Accurate+and+Reliable+Prediction+of+Relative+Ligand+Binding+Potency+in+Prospective+Drug+Discovery+by+Way+of+a+Modern+Free-Energy+Calculation+Protocol+and+Force+Field&doi=10.1021%2Fja512751q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Accurate and Reliable Prediction of Relative Ligand Binding Potency in Prospective Drug Discovery by Way of a Modern Free-Energy Calculation Protocol and Force Field</span></div><div class="casAuthors">Wang, Lingle; Wu, Yujie; Deng, Yuqing; Kim, Byungchan; Pierce, Levi; Krilov, Goran; Lupyan, Dmitry; Robinson, Shaughnessy; Dahlgren, Markus K.; Greenwood, Jeremy; Romero, Donna L.; Masse, Craig; Knight, Jennifer L.; Steinbrecher, Thomas; Beuming, Thijs; Damm, Wolfgang; Harder, Ed; Sherman, Woody; Brewer, Mark; Wester, Ron; Murcko, Mark; Frye, Leah; Farid, Ramy; Lin, Teng; Mobley, David L.; Jorgensen, William L.; Berne, Bruce J.; Friesner, Richard A.; Abel, Robert</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">137</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2695-2703</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Designing tight-binding ligands is a primary objective of small-mol. drug discovery.  Over the past few decades, free-energy calcns. have benefited from improved force fields and sampling algorithms, as well as the advent of low-cost parallel computing.  However, it has proven to be challenging to reliably achieve the level of accuracy that would be needed to guide lead optimization (∼5× in binding affinity) for a wide range of ligands and protein targets.  Not surprisingly, widespread com. application of free-energy simulations has been limited due to the lack of large-scale validation coupled with the tech. challenges traditionally assocd. with running these types of calcns.  Here, we report an approach that achieves an unprecedented level of accuracy across a broad range of target classes and ligands, with retrospective results encompassing 200 ligands and a wide variety of chem. perturbations, many of which involve significant changes in ligand chem. structures.  In addn., we have applied the method in prospective drug discovery projects and found a significant improvement in the quality of the compds. synthesized that have been predicted to be potent.  Compds. predicted to be potent by this approach have a substantial redn. in false positives relative to compds. synthesized on the basis of other computational or medicinal chem. approaches.  Furthermore, the results are consistent with those obtained from our retrospective studies, demonstrating the robustness and broad range of applicability of this approach, which can be used to drive decisions in lead optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGor9UtXEyLPpLVg90H21EOLACvtfcHk0lhECqoLU5aHVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsF2iuro%253D&md5=37a4f4a6c085f47ed531342643b6c33b</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fja512751q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja512751q%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DDeng%26aufirst%3DY.%26aulast%3DKim%26aufirst%3DB.%26aulast%3DPierce%26aufirst%3DL.%26aulast%3DKrilov%26aufirst%3DG.%26aulast%3DLupyan%26aufirst%3DD.%26aulast%3DRobinson%26aufirst%3DS.%26aulast%3DDahlgren%26aufirst%3DM.%2BK.%26aulast%3DGreenwood%26aufirst%3DJ.%26aulast%3DRomero%26aufirst%3DD.%2BL.%26aulast%3DMasse%26aufirst%3DC.%26aulast%3DKnight%26aufirst%3DJ.%2BL.%26aulast%3DSteinbrecher%26aufirst%3DT.%26aulast%3DBeuming%26aufirst%3DT.%26aulast%3DDamm%26aufirst%3DW.%26aulast%3DHarder%26aufirst%3DE.%26aulast%3DSherman%26aufirst%3DW.%26aulast%3DBrewer%26aufirst%3DM.%26aulast%3DWester%26aufirst%3DR.%26aulast%3DMurcko%26aufirst%3DM.%26aulast%3DFrye%26aufirst%3DL.%26aulast%3DFarid%26aufirst%3DR.%26aulast%3DLin%26aufirst%3DT.%26aulast%3DMobley%26aufirst%3DD.%2BL.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DBerne%26aufirst%3DB.%2BJ.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DAbel%26aufirst%3DR.%26atitle%3DAccurate%2520and%2520Reliable%2520Prediction%2520of%2520Relative%2520Ligand%2520Binding%2520Potency%2520in%2520Prospective%2520Drug%2520Discovery%2520by%2520Way%2520of%2520a%2520Modern%2520Free-Energy%2520Calculation%2520Protocol%2520and%2520Force%2520Field%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2015%26volume%3D137%26spage%3D2695%26epage%3D2703%26doi%3D10.1021%2Fja512751q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">Guo, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heitman, L. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ijzerman, A. P.</span><span> </span><span class="NLM_article-title">The role of target binding kinetics in drug discovery</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">1793</span><span class="NLM_x">–</span> <span class="NLM_lpage">1796</span><span class="refDoi"> DOI: 10.1002/cmdc.201500310</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1002%2Fcmdc.201500310" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=26506404" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlyjurfF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=1793-1796&author=D.+Guoauthor=L.+H.+Heitmanauthor=A.+P.+Ijzerman&title=The+role+of+target+binding+kinetics+in+drug+discovery&doi=10.1002%2Fcmdc.201500310"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">The Role of Target Binding Kinetics in Drug Discovery</span></div><div class="casAuthors">Guo, Dong; Heitman, Laura H.; IJzerman, Adriaan P.</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1793-1796</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Traditionally structure-activity/affinity relationships (SAR) have dominated research in medicinal chem.  However, structure-kinetics relationships (SKR) can be very informative too.  In this viewpoint we explore the mol. determinants of binding kinetics and discuss challenges for future binding kinetics studies.  A scheme for future kinetics-directed drug design and discovery is also proposed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoiX7v30ryLZLVg90H21EOLACvtfcHk0li2vFNm29clfw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlyjurfF&md5=8442ba4d4ec014762ed470f7f6131d09</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201500310&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201500310%26sid%3Dliteratum%253Aachs%26aulast%3DGuo%26aufirst%3DD.%26aulast%3DHeitman%26aufirst%3DL.%2BH.%26aulast%3DIjzerman%26aufirst%3DA.%2BP.%26atitle%3DThe%2520role%2520of%2520target%2520binding%2520kinetics%2520in%2520drug%2520discovery%26jtitle%3DChemMedChem%26date%3D2015%26volume%3D10%26spage%3D1793%26epage%3D1796%26doi%3D10.1002%2Fcmdc.201500310" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group">Guo, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hillger, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ijzerman, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heitman, L. H.</span><span> </span><span class="NLM_article-title">Drug-target residence time—a case for G protein-coupled receptors</span> <span class="citation_source-journal">Med. Res. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">34</span><span class="NLM_x">, </span> <span class="NLM_fpage">856</span><span class="NLM_x">–</span> <span class="NLM_lpage">892</span><span class="refDoi"> DOI: 10.1002/med.21307</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1002%2Fmed.21307" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=24549504" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BC2cXptVagtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2014&pages=856-892&author=D.+Guoauthor=J.+M.+Hillgerauthor=A.+P.+Ijzermanauthor=L.+H.+Heitman&title=Drug-target+residence+time%E2%80%94a+case+for+G+protein-coupled+receptors&doi=10.1002%2Fmed.21307"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Drug-Target Residence Time-A Case for G Protein-Coupled Receptors</span></div><div class="casAuthors">Guo, Dong; Hillger, Julia M.; Ijzerman, Adriaan P.; Heitman, Laura H.</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">856-892</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  A vast no. of marketed drugs act on G protein-coupled receptors (GPCRs), the most successful category of drug targets to date.  These drugs usually possess high target affinity and selectivity, and such combined features have been the driving force in the early phases of drug discovery.  However, attrition has also been high.  Many investigational new drugs eventually fail in clin. trials due to a demonstrated lack of efficacy.  A retrospective assessment of successfully launched drugs revealed that their beneficial effects in patients may be attributed to their long drug-target residence times (RTs).  Likewise, for some other GPCR drugs short RT could be beneficial to reduce the potential for on-target side effects.  Hence, the compds.' kinetics behavior might in fact be the guiding principle to obtain a desired and durable effect in vivo.  We therefore propose that drug-target RT should be taken into account as an addnl. parameter in the lead selection and optimization process.  This should ultimately lead to an increased no. of candidate drugs moving to the preclin. development phase and on to the market.  This review contains examples of the kinetics behavior of GPCR ligands with improved in vivo efficacy and summarizes methods for assessing drug-target RT.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpN5vhWI93LMbVg90H21EOLACvtfcHk0li2vFNm29clfw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXptVagtbs%253D&md5=af48cdedf3c60ecb608d5c9ad03a0bbe</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1002%2Fmed.21307&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21307%26sid%3Dliteratum%253Aachs%26aulast%3DGuo%26aufirst%3DD.%26aulast%3DHillger%26aufirst%3DJ.%2BM.%26aulast%3DIjzerman%26aufirst%3DA.%2BP.%26aulast%3DHeitman%26aufirst%3DL.%2BH.%26atitle%3DDrug-target%2520residence%2520time%25E2%2580%2594a%2520case%2520for%2520G%2520protein-coupled%2520receptors%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2014%26volume%3D34%26spage%3D856%26epage%3D892%26doi%3D10.1002%2Fmed.21307" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pompliano, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meek, T. D.</span><span> </span><span class="NLM_article-title">Drug-target residence time and its implications for lead optimization</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">730</span><span class="NLM_x">–</span> <span class="NLM_lpage">739</span><span class="refDoi"> DOI: 10.1038/nrd2082</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1038%2Fnrd2082" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=16888652" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BD28XptVCltro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2006&pages=730-739&author=R.+A.+Copelandauthor=D.+L.+Pomplianoauthor=T.+D.+Meek&title=Drug-target+residence+time+and+its+implications+for+lead+optimization&doi=10.1038%2Fnrd2082"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Drug-target residence time and its implications for lead optimization</span></div><div class="casAuthors">Copeland, Robert A.; Pompliano, David L.; Meek, Thomas D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">730-739</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Much of drug discovery today is predicated on the concept of selective targeting of particular bioactive macromols. by low-mol.-mass drugs.  The binding of drugs to their macromol. targets is therefore seen as paramount for pharmacol. activity.  In vitro assessment of drug-target interactions is classically quantified in terms of binding parameters such as IC50 or Kd.  This article presents an alternative perspective on drug optimization in terms of drug-target binary complex residence time, as quantified by the dissociative half-life of the drug-target binary complex.  We describe the potential advantages of long residence time in terms of duration of pharmacol. effect and target selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGry6SJrMlrrTbVg90H21EOLACvtfcHk0li2vFNm29clfw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XptVCltro%253D&md5=60ede2301584b10ac4e8fa18e1e6d107</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1038%2Fnrd2082&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2082%26sid%3Dliteratum%253Aachs%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DPompliano%26aufirst%3DD.%2BL.%26aulast%3DMeek%26aufirst%3DT.%2BD.%26atitle%3DDrug-target%2520residence%2520time%2520and%2520its%2520implications%2520for%2520lead%2520optimization%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2006%26volume%3D5%26spage%3D730%26epage%3D739%26doi%3D10.1038%2Fnrd2082" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group">Tian, H.; Sakmar, T. P.; Huber, T.</span><span> </span><span class="NLM_article-title">Site-specific labeling of genetically encoded azido groups for multicolor, single-molecule fluorescence imaging of GPCRs</span>. In  <span class="citation_source-book">Methods in Cell Biology</span>; <span class="NLM_contrib-group">Conn, P. M.</span>, Ed.; <span class="NLM_publisher-name">Academic Press</span>: <span class="NLM_publisher-loc">Cambridge, MA</span>,<span class="NLM_x"> </span><span class="NLM_year">2013</span><span class="NLM_x">; </span>Vol.  <span class="NLM_volume">117</span>, Chapter 15, pp  <span class="NLM_fpage">267</span><span class="NLM_x">–</span> <span class="NLM_lpage">303</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1016%2FB978-0-12-408143-7.00015-3" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&pages=267-303&author=H.+Tian&author=T.+P.+Sakmar&author=T.+Huberauthor=P.+M.+Conn&title=Methods+in+Cell+Biology"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2FB978-0-12-408143-7.00015-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1016%252FB978-0-12-408143-7.00015-3%26sid%3Dliteratum%253Aachs%26aulast%3DTian%26aufirst%3DH.%26atitle%3DSite-specific%2520labeling%2520of%2520genetically%2520encoded%2520azido%2520groups%2520for%2520multicolor%252C%2520single-molecule%2520fluorescence%2520imaging%2520of%2520GPCRs%26btitle%3DMethods%2520in%2520Cell%2520Biology%26aulast%3DConn%26aufirst%3DP.%2BM.%26pub%3DAcademic%2520Press%26date%3D2013%26volume%3D117%26spage%3D267%26epage%3D303" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group">Hurst, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmeisser, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reggio, P. H.</span><span> </span><span class="NLM_article-title">Endogenous lipid activated G protein-coupled receptors: emerging structural features from crystallography and molecular dynamics simulations</span> <span class="citation_source-journal">Chem. Phys. Lipids</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">169</span><span class="NLM_x">, </span> <span class="NLM_fpage">46</span><span class="NLM_x">–</span> <span class="NLM_lpage">56</span><span class="refDoi"> DOI: 10.1016/j.chemphyslip.2013.01.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1016%2Fj.chemphyslip.2013.01.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=23485612" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BC3sXltVGht7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=169&publication_year=2013&pages=46-56&author=D.+P.+Hurstauthor=M.+Schmeisserauthor=P.+H.+Reggio&title=Endogenous+lipid+activated+G+protein-coupled+receptors%3A+emerging+structural+features+from+crystallography+and+molecular+dynamics+simulations&doi=10.1016%2Fj.chemphyslip.2013.01.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Endogenous lipid activated G protein-coupled receptors: Emerging structural features from crystallography and molecular dynamics simulations</span></div><div class="casAuthors">Hurst, Dow P.; Schmeisser, Marianne; Reggio, Patricia H.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry and Physics of Lipids</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">169</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">46-56</span>CODEN:
                <span class="NLM_cas:coden">CPLIA4</span>;
        ISSN:<span class="NLM_cas:issn">0009-3084</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Class A G-protein coupled receptors (GPCRs) are thought to have a common topol. that includes seven transmembrane alpha helixes (TMHs) that are arranged to form a closed bundle.  This bundle forms the ligand binding pocket into which ligands are commonly thought to enter via the extracellular milieu.  This ligand approach direction makes sense for GPCRs that have small pos. charged ligands, such as the beta-2-adrenergic or the dopamine D2 receptor.  However, there is a growing sub-group of Class A GPCRs that bind lipid-derived endogenous ligands, such as the cannabinoid CB1 and CB2 receptors (with endogenous ligands, N-arachidonoylethanolamine (anandamide) and sn-2-arachidonylglycerol (2-AG)) and the S1P1-5 receptors (with endogenous ligand, sphingosine-1-phosphate).  Even the widely studied Class A GPCR, rhodopsin, binds a highly lipophillic chromophore (11-cis-retinal).  For these receptors, ligand approach from the extracellular milieu has seemed unlikely given that the ligands of these receptors readily partition into lipid or are actually synthesized in the lipid bilayer.  The recent X-ray-crystal structure of the sub-type 1 sphingosine-1-phosphate receptor (S1P1) provides important information on the key structural variations that may be the hallmarks for a Class A GPCR that binds lipid-derived ligands.  These include an extracellular domain that is closed off to the extracellular milieu and the existence of an opening between transmembrane helixes that may serve as a portal for ligand entry via the lipid bilayer.  This review examines structural aspects that the cannabinoid receptors may share with the S1P1 receptor based upon sequence homol.  This review also examines exptl. and simulation results that suggest ligand entry via a lipid portal is quite likely for this emerging sub-group.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRGcleyJ2OO7Vg90H21EOLACvtfcHk0lgt0SVpY1YL-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXltVGht7c%253D&md5=65882b737337687be6527251a9cad9d1</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2Fj.chemphyslip.2013.01.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chemphyslip.2013.01.009%26sid%3Dliteratum%253Aachs%26aulast%3DHurst%26aufirst%3DD.%2BP.%26aulast%3DSchmeisser%26aufirst%3DM.%26aulast%3DReggio%26aufirst%3DP.%2BH.%26atitle%3DEndogenous%2520lipid%2520activated%2520G%2520protein-coupled%2520receptors%253A%2520emerging%2520structural%2520features%2520from%2520crystallography%2520and%2520molecular%2520dynamics%2520simulations%26jtitle%3DChem.%2520Phys.%2520Lipids%26date%3D2013%26volume%3D169%26spage%3D46%26epage%3D56%26doi%3D10.1016%2Fj.chemphyslip.2013.01.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group">Wang, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duan, Y.</span><span> </span><span class="NLM_article-title">Ligand entry and exit pathways in the β2-adrenergic receptor</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">392</span><span class="NLM_x">, </span> <span class="NLM_fpage">1102</span><span class="NLM_x">–</span> <span class="NLM_lpage">1115</span><span class="refDoi"> DOI: 10.1016/j.jmb.2009.07.093</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1016%2Fj.jmb.2009.07.093" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=392&publication_year=2009&pages=1102-1115&author=T.+Wangauthor=Y.+Duan&title=Ligand+entry+and+exit+pathways+in+the+%CE%B22-adrenergic+receptor&doi=10.1016%2Fj.jmb.2009.07.093"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2009.07.093&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2009.07.093%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DT.%26aulast%3DDuan%26aufirst%3DY.%26atitle%3DLigand%2520entry%2520and%2520exit%2520pathways%2520in%2520the%2520%25CE%25B22-adrenergic%2520receptor%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2009%26volume%3D392%26spage%3D1102%26epage%3D1115%26doi%3D10.1016%2Fj.jmb.2009.07.093" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group">Seddon, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casey, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Law, R. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gee, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Templer, R. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ces, O.</span><span> </span><span class="NLM_article-title">Drug interactions with lipid membranes</span> <span class="citation_source-journal">Chem. Soc. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">38</span><span class="NLM_x">, </span> <span class="NLM_fpage">2509</span><span class="NLM_x">–</span> <span class="NLM_lpage">2519</span><span class="refDoi"> DOI: 10.1039/b813853m</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1039%2Fb813853m" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=19690732" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVSlsLbL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2009&pages=2509-2519&author=A.+M.+Seddonauthor=D.+Caseyauthor=R.+V.+Lawauthor=A.+Geeauthor=R.+H.+Templerauthor=O.+Ces&title=Drug+interactions+with+lipid+membranes&doi=10.1039%2Fb813853m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Drug interactions with lipid membranes</span></div><div class="casAuthors">Seddon, Annela M.; Casey, Duncan; Law, Robert V.; Gee, Antony; Templer, Richard H.; Ces, Oscar</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Society Reviews</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2509-2519</span>CODEN:
                <span class="NLM_cas:coden">CSRVBR</span>;
        ISSN:<span class="NLM_cas:issn">0306-0012</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  The field of drug-membrane interactions is one that spans a wide range of scientific disciplines, from synthetic chem., through biophysics to pharmacol.  Cell membranes are complex dynamic systems whose structures can be affected by drug mols. and in turn can affect the pharmacol. properties of the drugs being administered.  In this tutorial review we aim to provide a guide for those new to the area of drug-membrane interactions and present an introduction to areas of this topic which need to be considered.  We address the lipid compn. and structure of the cell membrane and comment on the phys. forces present in the membrane which may impact on drug interactions.  We outline methods by which drugs may cross or bind to this membrane, including the well understood passive and active transport pathways.  We present a range of techniques which may be used to study the interactions of drugs with membranes both in vitro and in vivo and discuss the advantages and disadvantages of these techniques and highlight new methods being developed to further this field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyed_OqOLrjrVg90H21EOLACvtfcHk0lgt0SVpY1YL-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVSlsLbL&md5=81e81a98d8de9c9a889ea378f927411b</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1039%2Fb813853m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fb813853m%26sid%3Dliteratum%253Aachs%26aulast%3DSeddon%26aufirst%3DA.%2BM.%26aulast%3DCasey%26aufirst%3DD.%26aulast%3DLaw%26aufirst%3DR.%2BV.%26aulast%3DGee%26aufirst%3DA.%26aulast%3DTempler%26aufirst%3DR.%2BH.%26aulast%3DCes%26aufirst%3DO.%26atitle%3DDrug%2520interactions%2520with%2520lipid%2520membranes%26jtitle%3DChem.%2520Soc.%2520Rev.%26date%3D2009%26volume%3D38%26spage%3D2509%26epage%3D2519%26doi%3D10.1039%2Fb813853m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group">Chen, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.-B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.-Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ren, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, C.-M.</span><span> </span><span class="NLM_article-title">Green tea polyphenol epigallocatechin-3-gallate (EGCG) induced intermolecular cross-linking of membrane proteins</span> <span class="citation_source-journal">Arch. Biochem. Biophys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">507</span><span class="NLM_x">, </span> <span class="NLM_fpage">343</span><span class="NLM_x">–</span> <span class="NLM_lpage">349</span><span class="refDoi"> DOI: 10.1016/j.abb.2010.12.033</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1016%2Fj.abb.2010.12.033" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=21211509" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BC3MXit12jt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=507&publication_year=2011&pages=343-349&author=R.+Chenauthor=J.-B.+Wangauthor=X.-Q.+Zhangauthor=J.+Renauthor=C.-M.+Zeng&title=Green+tea+polyphenol+epigallocatechin-3-gallate+%28EGCG%29+induced+intermolecular+cross-linking+of+membrane+proteins&doi=10.1016%2Fj.abb.2010.12.033"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Green tea polyphenol epigallocatechin-3-gallate (EGCG) induced intermolecular cross-linking of membrane proteins</span></div><div class="casAuthors">Chen, Rong; Wang, Jian-Bo; Zhang, Xian-Qing; Ren, Jing; Zeng, Cheng-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Archives of Biochemistry and Biophysics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">507</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">343-349</span>CODEN:
                <span class="NLM_cas:coden">ABBIA4</span>;
        ISSN:<span class="NLM_cas:issn">0003-9861</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Increasing evidence has demonstrated that EGCG possesses prooxidant potential in biol. systems, including modifying proteins, breaking DNA strands and inducing the generation of reactive oxygen species.  In the present study, the prooxidant effect of EGCG on erythrocyte membranes was investigated.  SDS-PAGE and NBT-staining assay were utilized to detect the catechol-protein adducts that generated upon treating the membranes with EGCG.  The results indicated that EGCG was able to bind covalently to sulfhydryl groups of membrane proteins, leading to the formation of protein aggregates with intermol. crosslinking.  We suggested that the catechol-quinone originated from the oxidn. of EGCG acted as a cross-linker on which peptide chains were combined through thiol-S-alkylation at the C2- and C6-sites of the gallyl ring.  EGC showed similar effects as EGCG on the ghost membranes, whereas ECG and EC did not, suggesting that a structure with a gallyl moiety is a prerequisite for a catechin to induce the aggregation of membrane proteins and to deplete membrane sulfhydryls.  EDTA and ascorbic acid inhibited the EGCG-induced aggregation of membrane proteins by blocking the formation of catechol-quinone.  The information of the present study may provide a fresh insight into the prooxidant effect and cytotoxicity of tea catechins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphG6UeJYYteLVg90H21EOLACvtfcHk0lgt0SVpY1YL-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXit12jt74%253D&md5=eeb20fbc69ad825c696971d8f93ed143</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.abb.2010.12.033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.abb.2010.12.033%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DR.%26aulast%3DWang%26aufirst%3DJ.-B.%26aulast%3DZhang%26aufirst%3DX.-Q.%26aulast%3DRen%26aufirst%3DJ.%26aulast%3DZeng%26aufirst%3DC.-M.%26atitle%3DGreen%2520tea%2520polyphenol%2520epigallocatechin-3-gallate%2520%2528EGCG%2529%2520induced%2520intermolecular%2520cross-linking%2520of%2520membrane%2520proteins%26jtitle%3DArch.%2520Biochem.%2520Biophys.%26date%3D2011%26volume%3D507%26spage%3D343%26epage%3D349%26doi%3D10.1016%2Fj.abb.2010.12.033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group">Ingólfsson, H. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thakur, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herold, K. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hobart, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramsey, N. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Periole, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Jong, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zwama, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yilmaz, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hall, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maretzky, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hemmings, H. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blobel, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marrink, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koçer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sack, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Andersen, O. S.</span><span> </span><span class="NLM_article-title">Phytochemicals perturb membranes and promiscuously alter protein function</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">1788</span><span class="NLM_x">–</span> <span class="NLM_lpage">1798</span><span class="refDoi"> DOI: 10.1021/cb500086e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb500086e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BC2cXptlyqt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=1788-1798&author=H.+I.+Ing%C3%B3lfssonauthor=P.+Thakurauthor=K.+F.+Heroldauthor=E.+A.+Hobartauthor=N.+B.+Ramseyauthor=X.+Perioleauthor=D.+H.+de+Jongauthor=M.+Zwamaauthor=D.+Yilmazauthor=K.+Hallauthor=T.+Maretzkyauthor=H.+C.+Hemmingsauthor=C.+Blobelauthor=S.+J.+Marrinkauthor=A.+Ko%C3%A7erauthor=J.+T.+Sackauthor=O.+S.+Andersen&title=Phytochemicals+perturb+membranes+and+promiscuously+alter+protein+function&doi=10.1021%2Fcb500086e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Phytochemicals Perturb Membranes and Promiscuously Alter Protein Function</span></div><div class="casAuthors">Ingolfsson, Helgi I.; Thakur, Pratima; Herold, Karl F.; Hobart, E. Ashley; Ramsey, Nicole B.; Periole, Xavier; de Jong, Djurre H.; Zwama, Martijn; Yilmaz, Duygu; Hall, Katherine; Maretzky, Thorsten; Hemmings, Hugh C.; Blobel, Carl; Marrink, Siewert J.; Kocer, Armagan; Sack, Jon T.; Andersen, Olaf S.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1788-1798</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A wide variety of phytochems. are consumed for their perceived health benefits.  Many of these phytochems. have been found to alter numerous cell functions, but the mechanisms underlying their biol. activity tend to be poorly understood.  Phenolic phytochems. are particularly promiscuous modifiers of membrane protein function, suggesting that some of their actions may be due to a common, membrane bilayer-mediated mechanism.  To test whether bilayer perturbation may underlie this diversity of actions, we examd. five bioactive phenols reported to have medicinal value: capsaicin from chili peppers, curcumin from turmeric, EGCG from green tea, genistein from soybeans, and resveratrol from grapes.  We find that each of these widely consumed phytochems. alters lipid bilayer properties and the function of diverse membrane proteins.  Mol. dynamics simulations show that these phytochems. modify bilayer properties by localizing to the bilayer/soln. interface.  Bilayer-modifying propensity was verified using a gramicidin-based assay, and indiscriminate modulation of membrane protein function was demonstrated using four proteins: membrane-anchored metalloproteases, mechanosensitive ion channels, and voltage-dependent potassium and sodium channels.  Each protein exhibited similar responses to multiple phytochems., consistent with a common, bilayer-mediated mechanism.  Our results suggest that many effects of amphiphilic phytochems. are due to cell membrane perturbations, rather than specific protein binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrL0NNb8_Q6qrVg90H21EOLACvtfcHk0li_5XMfPxkHvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXptlyqt7s%253D&md5=b61876239e926b652adea2e130627bc3</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Fcb500086e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb500086e%26sid%3Dliteratum%253Aachs%26aulast%3DIng%25C3%25B3lfsson%26aufirst%3DH.%2BI.%26aulast%3DThakur%26aufirst%3DP.%26aulast%3DHerold%26aufirst%3DK.%2BF.%26aulast%3DHobart%26aufirst%3DE.%2BA.%26aulast%3DRamsey%26aufirst%3DN.%2BB.%26aulast%3DPeriole%26aufirst%3DX.%26aulast%3Dde%2BJong%26aufirst%3DD.%2BH.%26aulast%3DZwama%26aufirst%3DM.%26aulast%3DYilmaz%26aufirst%3DD.%26aulast%3DHall%26aufirst%3DK.%26aulast%3DMaretzky%26aufirst%3DT.%26aulast%3DHemmings%26aufirst%3DH.%2BC.%26aulast%3DBlobel%26aufirst%3DC.%26aulast%3DMarrink%26aufirst%3DS.%2BJ.%26aulast%3DKo%25C3%25A7er%26aufirst%3DA.%26aulast%3DSack%26aufirst%3DJ.%2BT.%26aulast%3DAndersen%26aufirst%3DO.%2BS.%26atitle%3DPhytochemicals%2520perturb%2520membranes%2520and%2520promiscuously%2520alter%2520protein%2520function%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2014%26volume%3D9%26spage%3D1788%26epage%3D1798%26doi%3D10.1021%2Fcb500086e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group">Jiang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reilly, J.</span><span> </span><span class="NLM_article-title">Chromatography approaches for early screening of the phospholipidosis-inducing potential of pharmaceuticals</span> <span class="citation_source-journal">J. Pharm. Biomed. Anal.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">61</span><span class="NLM_x">, </span> <span class="NLM_fpage">184</span><span class="NLM_x">–</span> <span class="NLM_lpage">190</span><span class="refDoi"> DOI: 10.1016/j.jpba.2011.11.033</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1016%2Fj.jpba.2011.11.033" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=22200505" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1aiu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2012&pages=184-190&author=Z.+Jiangauthor=J.+Reilly&title=Chromatography+approaches+for+early+screening+of+the+phospholipidosis-inducing+potential+of+pharmaceuticals&doi=10.1016%2Fj.jpba.2011.11.033"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Chromatography approaches for early screening of the phospholipidosis-inducing potential of pharmaceuticals</span></div><div class="casAuthors">Jiang, Zhengjin; Reilly, John</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmaceutical and Biomedical Analysis</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">184-190</span>CODEN:
                <span class="NLM_cas:coden">JPBADA</span>;
        ISSN:<span class="NLM_cas:issn">0731-7085</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Drug-induced phospholipidosis (PLD) is an excessive accumulation of polar phospholipids within cells as a result of medical treatment.  Even though a direct link between in vitro drug-induced PLD and toxicity in humans has not yet been firmly established, the development of PLD during preclin. testing in animals is a recognized problem in the pharmaceutical industry and can delay or abort the development process.  Therefore, it is of interest to investigate the potential PLD risk of candidates at an early stage of the drug discovery process.  In this work, a high throughput physicochem. approach, which is based on measuring the retention factors of the test compd. within several different sepn. systems, was developed for screening phospholipidosis-inducing potential (PLIP) of pharmaceuticals.  The measured retention factors of 36 drugs were compared with literature data on PLIP risk from three different sources.  It is clearly shown that there is a statistical correlation between the chromatog. retention parameters of tested drugs and their PLIP risk.  In conclusion, the retention factor (log k AOT) obsd. on a docusate sodium salt (AOT) surfactant vesicle electrokinetic chromatog. (EKC) system and the logarithm of the vol. of distribution (log V d) calcd. from immobilized artificial membrane chromatog. at pH 7.4 (CHI IAM7.4) and HSA binding value (% HSA) can provide primary profile prediction for a large no. of drug candidates early in the drug discovery process with minimal resources.  The observations are that the higher the value of both log k AOT and log V d, the higher the PLIP risk, and we would recommend this dual approach.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoutkXEtBQfHLVg90H21EOLACvtfcHk0li_5XMfPxkHvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1aiu78%253D&md5=828cf8ec32952b5b01734816d69fafc4</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1016%2Fj.jpba.2011.11.033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jpba.2011.11.033%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DZ.%26aulast%3DReilly%26aufirst%3DJ.%26atitle%3DChromatography%2520approaches%2520for%2520early%2520screening%2520of%2520the%2520phospholipidosis-inducing%2520potential%2520of%2520pharmaceuticals%26jtitle%3DJ.%2520Pharm.%2520Biomed.%2520Anal.%26date%3D2012%26volume%3D61%26spage%3D184%26epage%3D190%26doi%3D10.1016%2Fj.jpba.2011.11.033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group">Reasor, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hastings, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ulrich, R. G.</span><span> </span><span class="NLM_article-title">Drug-induced phospholipidosis: issues and future directions</span> <span class="citation_source-journal">Expert Opin. Drug Saf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">567</span><span class="NLM_x">–</span> <span class="NLM_lpage">583</span><span class="refDoi"> DOI: 10.1517/14740338.5.4.567</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1517%2F14740338.5.4.567" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=16774494" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BD28XmslSksL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2006&pages=567-583&author=M.+J.+Reasorauthor=K.+L.+Hastingsauthor=R.+G.+Ulrich&title=Drug-induced+phospholipidosis%3A+issues+and+future+directions&doi=10.1517%2F14740338.5.4.567"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Drug-induced phospholipidosis: issues and future directions</span></div><div class="casAuthors">Reasor, Mark J.; Hastings, Kenneth L.; Ulrich, Roger G.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Safety</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">567-583</span>CODEN:
                <span class="NLM_cas:coden">EODSA9</span>;
        ISSN:<span class="NLM_cas:issn">1474-0338</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Numerous drugs contg. a cationic amphiphilic structure are capable of inducing phospholipidosis in cells under conditions of in vivo administration or ex vivo incubation.  The principal characteristics of this condition include the reversible accumulation of polar phospholipids in assocn. with the development of unicentric or multicentric lamellated bodies within cells.  There is an abundance of data providing an understanding of potential mechanisms for the induction of phospholipidosis; however, the process is likely to be complex and may differ from one drug to another.  The functional consequences of the presence of this condition on cellular or tissue function are not well understood.  The general consensus is that the condition is an adaptive response rather than a toxicol. manifestation; however, addnl. studies to examine this question are needed.  Until this issue is resolved, concerns about phospholipidosis will continue to exist at regulatory agencies.  Procedures for the screening of potential phospholipogenic candidate compds. are available.  In contrast, a clear need exists for the identification of valid biomarkers to assess the development of phospholipidosis in preclin. and clin. studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwYtmd7pqUebVg90H21EOLACvtfcHk0li_5XMfPxkHvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XmslSksL8%253D&md5=a2801d85bc4f1730f69272eaea3ce17b</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1517%2F14740338.5.4.567&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F14740338.5.4.567%26sid%3Dliteratum%253Aachs%26aulast%3DReasor%26aufirst%3DM.%2BJ.%26aulast%3DHastings%26aufirst%3DK.%2BL.%26aulast%3DUlrich%26aufirst%3DR.%2BG.%26atitle%3DDrug-induced%2520phospholipidosis%253A%2520issues%2520and%2520future%2520directions%26jtitle%3DExpert%2520Opin.%2520Drug%2520Saf.%26date%3D2006%26volume%3D5%26spage%3D567%26epage%3D583%26doi%3D10.1517%2F14740338.5.4.567" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group">Reasor, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCloud, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beard, T. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ebert, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kacew, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gardner, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aldern, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hostetler, K. Y.</span><span> </span><span class="NLM_article-title">Comparative evaluation of amiodarone-induced phospholipidosis and drug accumulation in Fischer-344 and Sprague-Dawley rats</span> <span class="citation_source-journal">Toxicology</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">106</span><span class="NLM_x">, </span> <span class="NLM_fpage">139</span><span class="NLM_x">–</span> <span class="NLM_lpage">147</span><span class="refDoi"> DOI: 10.1016/0300-483X(95)03175-F</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1016%2F0300-483X%2895%2903175-F" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=8571385" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADyaK2MXpvV2nsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=1996&pages=139-147&author=M.+J.+Reasorauthor=C.+M.+McCloudauthor=T.+L.+Beardauthor=D.+C.+Ebertauthor=S.+Kacewauthor=M.+F.+Gardnerauthor=K.+A.+Aldernauthor=K.+Y.+Hostetler&title=Comparative+evaluation+of+amiodarone-induced+phospholipidosis+and+drug+accumulation+in+Fischer-344+and+Sprague-Dawley+rats&doi=10.1016%2F0300-483X%2895%2903175-F"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Comparative evaluation of amiodarone-induced phospholipidosis and drug accumulation in Fischer-344 and Sprague-Dawley rats</span></div><div class="casAuthors">Reasor, Mark J.; McCloud, Christy M.; Beard, Terry L.; Ebert, Debora C.; Kacew, Sam; Gardner, Michael F.; Aldern, Kathy A.; Hostetler, Karl Y.</div><div class="citationInfo"><span class="NLM_cas:title">Toxicology</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">139-47</span>CODEN:
                <span class="NLM_cas:coden">TXCYAC</span>;
        ISSN:<span class="NLM_cas:issn">0300-483X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Amiodarone (AD) and its major metabolite, desethylamiodarone (desethylAD), are both phospholipogenic.  The present study was undertaken to evaluate the comparative susceptibilities of male Fischer-344 and Sprague-Dawley rats to AD-induced phospholipidosis in alveolar macrophages (AMs), liver and kidney tissue and the concomitant accumulation of AD and desethylAD in these cells, tissues and plasma.  Rats were administered AD (100 mg/kg/day, p.o.) for 1 wk.  Plasma concns. of AD and desethylAD were approx. 4- and 12-fold higher, resp., in Fischer-344s compared to Sprague-Dawleys 24 h after the last dose.  AD and desethylAD levels in AMs were approx. 12- and 25-fold higher, resp., in Fischer-344s than Sprague-Dawleys.  In the liver and kidney, levels of both compds. were also significantly higher in Fischer-344s than Sprague-Dawleys.  Ultrastructural features indicative of phospholipidosis were not obsd. consistently in any tissue except AMs from treated Fischer-344s.  AM total phospholipid increased nearly 5-fold in Fischer-344s, while Sprague-Dawleys showed no increase over control.  AMs from both strains incubated with 10 μM AD or desethylAD in vitro were not significantly different in their accumulation of the compds.  When incubated with AD or desethylAD, the lysosomal phospholipases A1 partially purified from AMs of both strains were equally sensitive to inhibition as measured by the drug concn. giving 50% inhibition in activity (IC50).  The results of this study indicate that at the same administered dose, AD and desethylAD, accumulate to higher tissue levels and are more phospholipogenic in male Fischer-344 rats than in male Sprague-Dawley rats.  The basis for the high susceptibility of Fischer-344 rats to AM-induced phospholipidosis is unknown at present but appears not to be related to biochem. or cellular features of the AMs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSyPj0Leb8jrVg90H21EOLACvtfcHk0ljpS8Ohq_EpKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXpvV2nsb8%253D&md5=16395ca5e1a13917d6085b81ec7fec59</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1016%2F0300-483X%2895%2903175-F&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0300-483X%252895%252903175-F%26sid%3Dliteratum%253Aachs%26aulast%3DReasor%26aufirst%3DM.%2BJ.%26aulast%3DMcCloud%26aufirst%3DC.%2BM.%26aulast%3DBeard%26aufirst%3DT.%2BL.%26aulast%3DEbert%26aufirst%3DD.%2BC.%26aulast%3DKacew%26aufirst%3DS.%26aulast%3DGardner%26aufirst%3DM.%2BF.%26aulast%3DAldern%26aufirst%3DK.%2BA.%26aulast%3DHostetler%26aufirst%3DK.%2BY.%26atitle%3DComparative%2520evaluation%2520of%2520amiodarone-induced%2520phospholipidosis%2520and%2520drug%2520accumulation%2520in%2520Fischer-344%2520and%2520Sprague-Dawley%2520rats%26jtitle%3DToxicology%26date%3D1996%26volume%3D106%26spage%3D139%26epage%3D147%26doi%3D10.1016%2F0300-483X%2895%2903175-F" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group">Williard, J.</span><span> </span><span class="NLM_article-title">FDA phospholipidosis working group preliminary results and developing opinions</span>, Presented at the 27th Annual Meeting of the Society of Toxicologic Pathology; <span class="NLM_publisher-loc">San Francisco, CA, USA</span>,<span class="NLM_x"> </span><span class="NLM_year">2008</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2008&author=J.+Williard&title=FDA+phospholipidosis+working+group+preliminary+results+and+developing+opinions"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dconf-proc%26sid%3Dliteratum%253Aachs%26aulast%3DWilliard%26aufirst%3DJ.%26atitle%3DFDA%2520phospholipidosis%2520working%2520group%2520preliminary%2520results%2520and%2520developing%2520opinions%26jtitle%3D27th%2520Annual%2520Meeting%2520of%2520the%2520Society%2520of%2520Toxicologic%2520Pathology%26date%3D2008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group">Bernstein, P. R.; Ciaccio, P.; Morelli, J.</span><span> </span><span class="NLM_article-title">Drug-induced phospholipidosis</span>. In  <span class="citation_source-book">Annual Reports in Medicinal Chemistry</span>; <span class="NLM_contrib-group">John, E. M.</span>, Ed.; <span class="NLM_publisher-name">Academic Press</span>: <span class="NLM_publisher-loc">Cambridge, MA</span>,<span class="NLM_x"> </span><span class="NLM_year">2011</span><span class="NLM_x">; </span>Vol.  <span class="NLM_volume">46</span>, Chapter 25, pp  <span class="NLM_fpage">419</span><span class="NLM_x">–</span> <span class="NLM_lpage">430</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1016%2FB978-0-12-386009-5.00001-1" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2011&pages=419-430&author=P.+R.+Bernstein&author=P.+Ciaccio&author=J.+Morelliauthor=E.+M.+John&title=Annual+Reports+in+Medicinal+Chemistry"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1016%2FB978-0-12-386009-5.00001-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1016%252FB978-0-12-386009-5.00001-1%26sid%3Dliteratum%253Aachs%26aulast%3DBernstein%26aufirst%3DP.%2BR.%26atitle%3DDrug-induced%2520phospholipidosis%26btitle%3DAnnual%2520Reports%2520in%2520Medicinal%2520Chemistry%26aulast%3DJohn%26aufirst%3DE.%2BM.%26pub%3DAcademic%2520Press%26date%3D2011%26volume%3D46%26spage%3D419%26epage%3D430" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group">Barone, L. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boyer, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Damewood, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fikes, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciaccio, P. J.</span><span> </span><span class="NLM_article-title">Phospholipogenic pharmaceuticals are associated with a higher incidence of histological findings than nonphospholipogenic pharmaceuticals in preclinical toxicology studies</span> <span class="citation_source-journal">J. Toxicol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">2012</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">7</span><span class="refDoi"> DOI: 10.1155/2012/308594</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1155%2F2012%2F308594" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2012&publication_year=2012&pages=1-7&author=L.+R.+Baroneauthor=S.+Boyerauthor=J.+R.+Damewoodauthor=J.+Fikesauthor=P.+J.+Ciaccio&title=Phospholipogenic+pharmaceuticals+are+associated+with+a+higher+incidence+of+histological+findings+than+nonphospholipogenic+pharmaceuticals+in+preclinical+toxicology+studies&doi=10.1155%2F2012%2F308594"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1155%2F2012%2F308594&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1155%252F2012%252F308594%26sid%3Dliteratum%253Aachs%26aulast%3DBarone%26aufirst%3DL.%2BR.%26aulast%3DBoyer%26aufirst%3DS.%26aulast%3DDamewood%26aufirst%3DJ.%2BR.%26aulast%3DFikes%26aufirst%3DJ.%26aulast%3DCiaccio%26aufirst%3DP.%2BJ.%26atitle%3DPhospholipogenic%2520pharmaceuticals%2520are%2520associated%2520with%2520a%2520higher%2520incidence%2520of%2520histological%2520findings%2520than%2520nonphospholipogenic%2520pharmaceuticals%2520in%2520preclinical%2520toxicology%2520studies%26jtitle%3DJ.%2520Toxicol.%26date%3D2012%26volume%3D2012%26spage%3D1%26epage%3D7%26doi%3D10.1155%2F2012%2F308594" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group">Cherezov, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosenbaum, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hanson, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rasmussen, S. G. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thian, F. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobilka, T. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choi, H.-J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhn, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weis, W. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobilka, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stevens, R. C.</span><span> </span><span class="NLM_article-title">High-resolution crystal structure of an engineered human β2-adrenergic G protein–coupled receptor</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">318</span><span class="NLM_x">, </span> <span class="NLM_fpage">1258</span><span class="NLM_x">–</span> <span class="NLM_lpage">1265</span><span class="refDoi"> DOI: 10.1126/science.1150577</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1126%2Fscience.1150577" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=17962520" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlGmur7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=318&publication_year=2007&pages=1258-1265&author=V.+Cherezovauthor=D.+M.+Rosenbaumauthor=M.+A.+Hansonauthor=S.+G.+F.+Rasmussenauthor=F.+S.+Thianauthor=T.+S.+Kobilkaauthor=H.-J.+Choiauthor=P.+Kuhnauthor=W.+I.+Weisauthor=B.+K.+Kobilkaauthor=R.+C.+Stevens&title=High-resolution+crystal+structure+of+an+engineered+human+%CE%B22-adrenergic+G+protein%E2%80%93coupled+receptor&doi=10.1126%2Fscience.1150577"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">High-Resolution Crystal Structure of an Engineered Human β2-Adrenergic G Protein-Coupled Receptor</span></div><div class="casAuthors">Cherezov, Vadim; Rosenbaum, Daniel M.; Hanson, Michael A.; Rasmussen, Soren G. F.; Thian, Foon Sun; Kobilka, Tong Sun; Choi, Hee-Jung; Kuhn, Peter; Weis, William I.; Kobilka, Brian K.; Stevens, Raymond C.; Takeda, S.; Kadowaki, S.; Haga, T.; Takaesu, H.; Mitaku, S.; Fredriksson, R.; Lagerstrom, M. C.; Lundin, L. G.; Schioth, H. B.; Pierce, K. L.; Premont, R. T.; Lefkowitz, R. J.; Lefkowitz, R. J.; Shenoy, S. K.; Rosenbaum, D. M.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">318</span>
        (<span class="NLM_cas:issue">5854</span>),
    <span class="NLM_cas:pages">1258-1265</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Second extracellular loop, which in the β2-adrenergic receptor contains an unusual pair of disulfide bonds and an extra helix.  This loop and the absence Heterotrimeric guanine nucleotide-binding protein (G protein)-coupled receptors constitute the largest family of eukaryotic signal transduction proteins that communicate across the membrane.  We report the crystal structure of a human β2-adrenergic receptor-T4 lysozyme fusion protein bound to the partial inverse agonist carazolol at 2.4 angstrom resoln.  The structure provides a high-resoln. view of a human G protein-coupled receptor bound to a diffusible ligand.  Ligand-binding site accessibility is enabled by the second extracellular loop, which is held out of the binding cavity by a pair of closely spaced disulfide bridges and a short helical segment within the loop.  Cholesterol, a necessary component for crystn., mediates an intriguing parallel assocn. of receptor mols. in the crystal lattice.  Although the location of carazolol in the β2-adrenergic receptor is very similar to that of retinal in rhodopsin, structural differences in the ligand-binding site and other regions highlight the challenges in using rhodopsin as a template model for this large receptor family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpC-8wZ2k2GHLVg90H21EOLACvtfcHk0ljpS8Ohq_EpKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlGmur7I&md5=12c5bacb8464a4b243fe9341192b5b3b</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1126%2Fscience.1150577&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1150577%26sid%3Dliteratum%253Aachs%26aulast%3DCherezov%26aufirst%3DV.%26aulast%3DRosenbaum%26aufirst%3DD.%2BM.%26aulast%3DHanson%26aufirst%3DM.%2BA.%26aulast%3DRasmussen%26aufirst%3DS.%2BG.%2BF.%26aulast%3DThian%26aufirst%3DF.%2BS.%26aulast%3DKobilka%26aufirst%3DT.%2BS.%26aulast%3DChoi%26aufirst%3DH.-J.%26aulast%3DKuhn%26aufirst%3DP.%26aulast%3DWeis%26aufirst%3DW.%2BI.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26atitle%3DHigh-resolution%2520crystal%2520structure%2520of%2520an%2520engineered%2520human%2520%25CE%25B22-adrenergic%2520G%2520protein%25E2%2580%2593coupled%2520receptor%26jtitle%3DScience%26date%3D2007%26volume%3D318%26spage%3D1258%26epage%3D1265%26doi%3D10.1126%2Fscience.1150577" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group">Lomize, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lomize, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pogozheva, I. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mosberg, H. I.</span><span> </span><span class="NLM_article-title">OPM: orientations of proteins in membranes database</span> <span class="citation_source-journal">Bioinformatics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">623</span><span class="NLM_x">–</span> <span class="NLM_lpage">625</span><span class="refDoi"> DOI: 10.1093/bioinformatics/btk023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1093%2Fbioinformatics%2Fbtk023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=16397007" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BD28XhvVeksro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2006&pages=623-625&author=M.+A.+Lomizeauthor=A.+L.+Lomizeauthor=I.+D.+Pogozhevaauthor=H.+I.+Mosberg&title=OPM%3A+orientations+of+proteins+in+membranes+database&doi=10.1093%2Fbioinformatics%2Fbtk023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">OPM: Orientations of Proteins in Membranes database</span></div><div class="casAuthors">Lomize, Mikhail A.; Lomize, Andrei L.; Pogozheva, Irina D.; Mosberg, Henry I.</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">623-625</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The Orientations of Proteins in Membranes (OPM) database provides a collection of transmembrane, monotopic, and peripheral proteins from the Protein Data Bank whose spatial arrangements in the lipid bilayer have been calcd. theor. and compared with exptl. data.  The database allows anal., sorting, and searching of membrane proteins based on their structural classification, species, destination membrane, nos. of transmembrane segments and subunits, nos. of secondary structures, and the calcd. hydrophobic thickness or tilt angle with respect to the bilayer normal.  All coordinate files with the calcd. membrane boundaries are available for downloading.  The Internet site for the database is: http://opm.phar.umich.edu.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1WjpHiq7e8LVg90H21EOLACvtfcHk0lidML8t9ahMHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhvVeksro%253D&md5=8a9590a32e912509a53ca0bfc7ede4ad</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtk023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtk023%26sid%3Dliteratum%253Aachs%26aulast%3DLomize%26aufirst%3DM.%2BA.%26aulast%3DLomize%26aufirst%3DA.%2BL.%26aulast%3DPogozheva%26aufirst%3DI.%2BD.%26aulast%3DMosberg%26aufirst%3DH.%2BI.%26atitle%3DOPM%253A%2520orientations%2520of%2520proteins%2520in%2520membranes%2520database%26jtitle%3DBioinformatics%26date%3D2006%26volume%3D22%26spage%3D623%26epage%3D625%26doi%3D10.1093%2Fbioinformatics%2Fbtk023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group">Zou, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weis, W. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobilka, B. K.</span><span> </span><span class="NLM_article-title">N-terminal T4 lysozyme fusion facilitates crystallization of a G protein coupled receptor</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">e46039</span><span class="refDoi"> DOI: 10.1371/journal.pone.0046039</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1371%2Fjournal.pone.0046039" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=23056231" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFaksr%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=e46039&author=Y.+Zouauthor=W.+I.+Weisauthor=B.+K.+Kobilka&title=N-terminal+T4+lysozyme+fusion+facilitates+crystallization+of+a+G+protein+coupled+receptor&doi=10.1371%2Fjournal.pone.0046039"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">N-Terminal T4 lysozyme fusion facilitates crystallization of a G protein coupled receptor</span></div><div class="casAuthors">Zou, Yaozhong; Weis, William I.; Kobilka, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">e46039</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">A highly crystallizable phage T4 lysozyme (T4L) was fused to the N-terminus of the β2 adrenergic receptor (β2AR), a G-protein coupled receptor (GPCR) for catecholamines.  It was demonstrated that the N-terminal fused T4L was sufficiently rigid relative to the receptor to facilitate crystallogenesis without thermostabilizing mutations or the use of a stabilizing antibody, G protein, or protein fused to the 3rd intracellular loop.  This approach adds to the protein engineering strategies that enable crystallog. studies of GPCRs alone or in complex with a signaling partner.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruYBtPrpjek7Vg90H21EOLACvtfcHk0lidML8t9ahMHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFaksr%252FM&md5=058b6ea877cef9d7f86cf0894c965f23</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0046039&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0046039%26sid%3Dliteratum%253Aachs%26aulast%3DZou%26aufirst%3DY.%26aulast%3DWeis%26aufirst%3DW.%2BI.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26atitle%3DN-terminal%2520T4%2520lysozyme%2520fusion%2520facilitates%2520crystallization%2520of%2520a%2520G%2520protein%2520coupled%2520receptor%26jtitle%3DPLoS%2520One%26date%3D2012%26volume%3D7%26spage%3De46039%26doi%3D10.1371%2Fjournal.pone.0046039" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span>Molecular Operating Environment (MOE); <span class="NLM_publisher-name">Chemical Computing Group Inc.</span>, <span class="NLM_publisher-loc">1010 Sherbooke St. West, Suite #910, Montreal, QC, Canada, H3A 2R7</span>,<span class="NLM_x"> </span><span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Molecular+Operating+Environment+%28MOE%29%3B+Chemical+Computing+Group+Inc.%2C+1010+Sherbooke+St.+West%2C+Suite+%23910%2C+Montreal%2C+QC%2C+Canada%2C+H3A+2R7%2C+2015."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Operating%2520Environment%2520%2528MOE%2529%26pub%3DChemical%2520Computing%2520Group%2520Inc%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group">Sindhikara, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirata, F.</span><span> </span><span class="NLM_article-title">Analysis of biomolecular solvation sites by 3D-RISM theory</span> <span class="citation_source-journal">J. Phys. Chem. B</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">117</span><span class="NLM_x">, </span> <span class="NLM_fpage">6718</span><span class="NLM_x">–</span> <span class="NLM_lpage">6723</span><span class="refDoi"> DOI: 10.1021/jp4046116</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp4046116" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BC3sXns1Gms74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=117&publication_year=2013&pages=6718-6723&author=D.+J.+Sindhikaraauthor=F.+Hirata&title=Analysis+of+biomolecular+solvation+sites+by+3D-RISM+theory&doi=10.1021%2Fjp4046116"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Analysis of Biomolecular Solvation Sites by 3D-RISM Theory</span></div><div class="casAuthors">Sindhikara, Daniel J.; Hirata, Fumio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">117</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">6718-6723</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1520-5207</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We derive, implement, and apply equil. solvation site anal. for biomols.  Our method utilizes 3D-RISM calcns. to quickly obtain equil. solvent distributions without either necessity of simulation or limits of solvent sampling.  Our anal. of these distributions exts. highest likelihood poses of solvent as well as localized entropies, enthalpies, and solvation free energies.  We demonstrate our method on a structure of HIV-1 protease where excellent structural and thermodn. data are available for comparison.  Our results, obtained within minutes, show systematic agreement with available exptl. data.  Further, our results are in good agreement with established simulation-based solvent anal. methods.  This method can be used not only for visual anal. of active site solvation but also for virtual screening methods and exptl. refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6CAgZkVvDJLVg90H21EOLACvtfcHk0lidML8t9ahMHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXns1Gms74%253D&md5=f709823750c912702cee0653b9170662</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1021%2Fjp4046116&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp4046116%26sid%3Dliteratum%253Aachs%26aulast%3DSindhikara%26aufirst%3DD.%2BJ.%26aulast%3DHirata%26aufirst%3DF.%26atitle%3DAnalysis%2520of%2520biomolecular%2520solvation%2520sites%2520by%25203D-RISM%2520theory%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2013%26volume%3D117%26spage%3D6718%26epage%3D6723%26doi%3D10.1021%2Fjp4046116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group">Sindhikara, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoshida, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirata, F.</span><span> </span><span class="NLM_article-title">Placevent: an algorithm for prediction of explicit solvent atom distribution—application to HIV-1 protease and F-ATP synthase</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x">, </span> <span class="NLM_fpage">1536</span><span class="NLM_x">–</span> <span class="NLM_lpage">1543</span><span class="refDoi"> DOI: 10.1002/jcc.22984</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1002%2Fjcc.22984" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=22522665" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BC38XlslyktLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2012&pages=1536-1543&author=D.+J.+Sindhikaraauthor=N.+Yoshidaauthor=F.+Hirata&title=Placevent%3A+an+algorithm+for+prediction+of+explicit+solvent+atom+distribution%E2%80%94application+to+HIV-1+protease+and+F-ATP+synthase&doi=10.1002%2Fjcc.22984"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Placevent: An algorithm for prediction of explicit solvent atom distribution - Application to HIV-1 protease and F-ATP synthase</span></div><div class="casAuthors">Sindhikara, Daniel J.; Yoshida, Norio; Hirata, Fumio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">1536-1543</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">We have created a simple algorithm for automatically predicting the explicit solvent atom distribution of biomols.  The explicit distribution is coerced from the three-dimensional (3D) continuous distribution resulting from a 3D ref. interaction site model (3D-RISM) calcn.  This procedure predicts optimal location of solvent mols. and ions given a rigid biomol. structure and the solvent compn.  We show examples of predicting water mols. near the KNI-272 bound form of HIV-1 protease and predicting both sodium ions and water mols. near the rotor ring of F-ATP synthase.  Our results give excellent agreement with exptl. structure with an av. prediction error of 0.39-0.65 Å.  Further, unlike exptl. methods, this method does not suffer from the partial occupancy limit.  Our method can be performed directly on 3D-RISM output within minutes.  It is extremely useful for examg. multiple specific solvent-solute interactions, as a convenient method for generating initial solvent structures for mol. dynamics calcns., and may assist in refinement of exptl. structures.  © 2012 Wiley Periodicals, Inc.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm61L3EIlF97Vg90H21EOLACvtfcHk0lgvnor0uak7CQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XlslyktLk%253D&md5=f80d4c1a0b2c5d87695481674da31814</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1002%2Fjcc.22984&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.22984%26sid%3Dliteratum%253Aachs%26aulast%3DSindhikara%26aufirst%3DD.%2BJ.%26aulast%3DYoshida%26aufirst%3DN.%26aulast%3DHirata%26aufirst%3DF.%26atitle%3DPlacevent%253A%2520an%2520algorithm%2520for%2520prediction%2520of%2520explicit%2520solvent%2520atom%2520distribution%25E2%2580%2594application%2520to%2520HIV-1%2520protease%2520and%2520F-ATP%2520synthase%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2012%26volume%3D33%26spage%3D1536%26epage%3D1543%26doi%3D10.1002%2Fjcc.22984" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group">Jo, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lim, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klauda, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Im, W.</span><span> </span><span class="NLM_article-title">CHARMM-GUI membrane builder for mixed bilayers and its application to yeast membranes</span> <span class="citation_source-journal">Biophys. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">97</span><span class="NLM_x">, </span> <span class="NLM_fpage">50</span><span class="NLM_x">–</span> <span class="NLM_lpage">58</span><span class="refDoi"> DOI: 10.1016/j.bpj.2009.04.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1016%2Fj.bpj.2009.04.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=19580743" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BD1MXpsFOmt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=97&publication_year=2009&pages=50-58&author=S.+Joauthor=J.+B.+Limauthor=J.+B.+Klaudaauthor=W.+Im&title=CHARMM-GUI+membrane+builder+for+mixed+bilayers+and+its+application+to+yeast+membranes&doi=10.1016%2Fj.bpj.2009.04.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">CHARMM-GUI Membrane Builder for mixed bilayers and its application to yeast membranes</span></div><div class="casAuthors">Jo, Sunhwan; Lim, Joseph B.; Klauda, Jeffery B.; Im, Wonpil</div><div class="citationInfo"><span class="NLM_cas:title">Biophysical Journal</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">97</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">50-58</span>CODEN:
                <span class="NLM_cas:coden">BIOJAU</span>;
        ISSN:<span class="NLM_cas:issn">0006-3495</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The CHARMM-GUI Membrane Builder (http://www.charmm-gui.org/input/membrane), an intuitive, straightforward, web-based graphical user interface, was expanded to automate the building process of heterogeneous lipid bilayers, with or without a protein and with support for up to 32 different lipid types.  The efficacy of these new features was tested by building and simulating lipid bilayers that resemble yeast membranes, composed of cholesterol, dipalmitoylphosphatidylcholine, dioleoylphosphatidylcholine, palmitoyloleoylphosphatidylethanolamine, palmitoyloleoylphosphatidylamine, and palmitoyloleoylphosphatidylserine.  Four membranes with varying concns. of cholesterol and phospholipids were simulated, for a total of 170 ns at 303.15 K.  Unsatd. phospholipid chain concn. had the largest influence on membrane properties, such as av. lipid surface area, d. profiles, deuterium order parameters, and cholesterol tilt angle.  Simulations with a high concn. of unsatd. chains (73%, membraneunsat) resulted in a significant increase in lipid surface area and a decrease in deuterium order parameters, compared with membranes with a high concn. of satd. chains (60-63%, membranesat).  The av. tilt angle of cholesterol with respect to bilayer normal was largest, and the distribution was significantly broader for membraneunsat.  Moreover, short-lived cholesterol orientations parallel to the membrane surface existed only for membraneunsat.  The membranesat simulations were in a liq.-ordered state, and agree with similar exptl. cholesterol-contg. membranes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxF3HFYL2Qa7Vg90H21EOLACvtfcHk0lgvnor0uak7CQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXpsFOmt7g%253D&md5=a6d57a8f42772e4c3674e324ea27ecaa</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1016%2Fj.bpj.2009.04.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bpj.2009.04.013%26sid%3Dliteratum%253Aachs%26aulast%3DJo%26aufirst%3DS.%26aulast%3DLim%26aufirst%3DJ.%2BB.%26aulast%3DKlauda%26aufirst%3DJ.%2BB.%26aulast%3DIm%26aufirst%3DW.%26atitle%3DCHARMM-GUI%2520membrane%2520builder%2520for%2520mixed%2520bilayers%2520and%2520its%2520application%2520to%2520yeast%2520membranes%26jtitle%3DBiophys.%2520J.%26date%3D2009%26volume%3D97%26spage%3D50%26epage%3D58%26doi%3D10.1016%2Fj.bpj.2009.04.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group">Case, D. A.; Berryman, J. T.; Betz, R. M.; Cerutti, D. S.; Cheatham,  I., T. E.; Darden, T. A.; Duke, R. E.; Giese, T. J.; Gohlke, H.; Goetz, A. W.; Homeyer, N.; Izadi, S.; Janowski, P.; Kaus, J.; Kovalenko, A.; Lee, T. S.; LeGrand, S.; Li, P.; Luchko, T.; Luo, R.; Madej, B.; Merz, K. M.; Monard, G.; Needham, P.; Nguyen, H.; Nguyen, H. T.; Omelyan, I.; Onufriev, A.; Roe, D. R.; Roitberg, A.; Salomon-Ferrer, R.; Simmerling, C. L.; Smith, W.; Swails, J.; Walker, R. C.; Wang, J.; Wolf, R. M.; Wu, X.; York, D. M.; Kollman, P. A.</span>AMBER 14; <span class="NLM_publisher-name">University of California</span>: <span class="NLM_publisher-loc">San Francisco</span>,<span class="NLM_x"> </span><span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=D.+A.+Case&author=J.+T.+Berryman&author=R.+M.+Betz&author=D.+S.+Cerutti&author=T.+E.+Cheatham&author=T.+A.+Darden&author=R.+E.+Duke&author=T.+J.+Giese&author=H.+Gohlke&author=A.+W.+Goetz&author=N.+Homeyer&author=S.+Izadi&author=P.+Janowski&author=J.+Kaus&author=A.+Kovalenko&author=T.+S.+Lee&author=S.+LeGrand&author=P.+Li&author=T.+Luchko&author=R.+Luo&author=B.+Madej&author=K.+M.+Merz&author=G.+Monard&author=P.+Needham&author=H.+Nguyen&author=H.+T.+Nguyen&author=I.+Omelyan&author=A.+Onufriev&author=D.+R.+Roe&author=A.+Roitberg&author=R.+Salomon-Ferrer&author=C.+L.+Simmerling&author=W.+Smith&author=J.+Swails&author=R.+C.+Walker&author=J.+Wang&author=R.+M.+Wolf&author=X.+Wu&author=D.+M.+York&author=P.+A.+Kollman&title=AMBER+14"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DCase%26aufirst%3DD.%2BA.%26jtitle%3DAMBER%252014%26pub%3DUniversity%2520of%2520California%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group">Ring, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Manglik, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kruse, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Enos, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weis, W. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garcia, K. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobilka, B. K.</span><span> </span><span class="NLM_article-title">Adrenaline-activated structure of β2-adrenoceptor stabilized by an engineered nanobody</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">502</span><span class="NLM_x">, </span> <span class="NLM_fpage">575</span><span class="NLM_x">–</span> <span class="NLM_lpage">579</span><span class="refDoi"> DOI: 10.1038/nature12572</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1038%2Fnature12572" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=24056936" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFSlsLzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=502&publication_year=2013&pages=575-579&author=A.+M.+Ringauthor=A.+Manglikauthor=A.+C.+Kruseauthor=M.+D.+Enosauthor=W.+I.+Weisauthor=K.+C.+Garciaauthor=B.+K.+Kobilka&title=Adrenaline-activated+structure+of+%CE%B22-adrenoceptor+stabilized+by+an+engineered+nanobody&doi=10.1038%2Fnature12572"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Adrenaline-activated structure of β2-adrenoceptor stabilized by an engineered nanobody</span></div><div class="casAuthors">Ring, Aaron M.; Manglik, Aashish; Kruse, Andrew C.; Enos, Michael D.; Weis, William I.; Garcia, K. Christopher; Kobilka, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">502</span>
        (<span class="NLM_cas:issue">7472</span>),
    <span class="NLM_cas:pages">575-579</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">G-protein-coupled receptors (GPCRs) are integral membrane proteins that play an essential role in human physiol., yet the mol. processes through which they bind to their endogenous agonists and activate effector proteins remain poorly understood.  So far, it has not been possible to capture an active-state GPCR bound to its native neurotransmitter.  Crystal structures of agonist-bound GPCRs have relied on the use of either exceptionally high-affinity agonists or receptor stabilization by mutagenesis.  Many natural agonists such as adrenaline, which activates the β2-adrenoceptor (β2AR), bind with relatively low affinity, and they are often chem. unstable.  Here, using directed evolution, the authors engineered a high-affinity conformationally selective single-domain camelid antibody fragment (nanobody; Nb6B9) that stabilized the active state of human β2AR, and used this to obtain crystal structures of the activated receptor bound to multiple ligands.  The authors present structures of the active-state human β2AR bound to 3 chem. distinct agonists: (1) the ultrahigh-affinity agonist BI167107; (2) the high-affinity catecholamine agonist, hydroxybenzylisoproterenol; and (3) the low-affinity endogenous agonist, adrenaline.  The crystal structures revealed a highly conserved overall ligand recognition and activation mode despite diverse ligand chem. structures and affinities that ranged from 100 nM to ∼80 pM.  Overall, the adrenaline-bound receptor structure was similar to the others, but it had substantial rearrangements in extracellular loop 3 and the extracellular tip of transmembrane helix 6.  These structures also revealed a water-mediated H-bond between 2 conserved Tyr residues, which appeared to stabilize the active state of the β2AR and related GPCRs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAp0VskflwWbVg90H21EOLACvtfcHk0lgvnor0uak7CQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFSlsLzP&md5=c5b844a0a86ee0a74843588a4a363592</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1038%2Fnature12572&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12572%26sid%3Dliteratum%253Aachs%26aulast%3DRing%26aufirst%3DA.%2BM.%26aulast%3DManglik%26aufirst%3DA.%26aulast%3DKruse%26aufirst%3DA.%2BC.%26aulast%3DEnos%26aufirst%3DM.%2BD.%26aulast%3DWeis%26aufirst%3DW.%2BI.%26aulast%3DGarcia%26aufirst%3DK.%2BC.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26atitle%3DAdrenaline-activated%2520structure%2520of%2520%25CE%25B22-adrenoceptor%2520stabilized%2520by%2520an%2520engineered%2520nanobody%26jtitle%3DNature%26date%3D2013%26volume%3D502%26spage%3D575%26epage%3D579%26doi%3D10.1038%2Fnature12572" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span>Small-Molecule Drug Discovery Suite 2015–1: Glide; <span class="NLM_publisher-name">Schrödinger</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Small-Molecule+Drug+Discovery+Suite+2015%E2%80%931%3A+Glide%3B+Schr%C3%B6dinger%3A+New+York%2C+2015."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DSmall-Molecule%2520Drug%2520Discovery%2520Suite%25202015%25E2%2580%25931%253A%2520Glide%26pub%3DSchr%25C3%25B6dinger%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group">Wacker, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fenalti, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Katritch, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abagyan, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cherezov, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stevens, R. C.</span><span> </span><span class="NLM_article-title">Conserved binding mode of human β2 adrenergic receptor inverse agonists and antagonist revealed by x-ray crystallography</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">132</span><span class="NLM_x">, </span> <span class="NLM_fpage">11443</span><span class="NLM_x">–</span> <span class="NLM_lpage">11445</span><span class="refDoi"> DOI: 10.1021/ja105108q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja105108q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BC3cXpsV2mtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2010&pages=11443-11445&author=D.+Wackerauthor=G.+Fenaltiauthor=M.+A.+Brownauthor=V.+Katritchauthor=R.+Abagyanauthor=V.+Cherezovauthor=R.+C.+Stevens&title=Conserved+binding+mode+of+human+%CE%B22+adrenergic+receptor+inverse+agonists+and+antagonist+revealed+by+x-ray+crystallography&doi=10.1021%2Fja105108q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Conserved Binding Mode of Human β2 Adrenergic Receptor Inverse Agonists and Antagonist Revealed by X-ray Crystallography</span></div><div class="casAuthors">Wacker, Daniel; Fenalti, Gustavo; Brown, Monica A.; Katritch, Vsevolod; Abagyan, Ruben; Cherezov, Vadim; Stevens, Raymond C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">132</span>
        (<span class="NLM_cas:issue">33</span>),
    <span class="NLM_cas:pages">11443-11445</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">G protein-coupled receptors (GPCRs) represent a large fraction of current pharmaceutical targets, and of the GPCRs, the β2 adrenergic receptor (β2AR) is one of the most extensively studied.  Previously, the x-ray crystal structure of β2AR has been detd. in complex with two partial inverse agonists, but the global impact of addnl. ligands on the structure or local impacts on the binding site are not well-understood.  To assess the extent of such ligand-induced conformational differences, we detd. the crystal structures of a previously described engineered β2AR construct in complex with two inverse agonists: ICI 118,551 (2.8 Å), a recently described compd. (2.8 Å) (Kolb et al, 2009), and the antagonist alprenolol (3.1 Å).  The structures show the same overall fold obsd. for the previous β2AR structures and demonstrate that the ligand binding site can accommodate compds. of different chem. and pharmacol. properties with only minor local structural rearrangements.  All three compds. contain a hydroxy-amine motif that establishes a conserved hydrogen bond network with the receptor and chem. diverse arom. moieties that form distinct interactions with β2AR.  Furthermore, receptor ligand cross-docking expts. revealed that a single β2AR complex can be suitable for docking of a range of antagonists and inverse agonists but also indicate that addnl. ligand-receptor structures may be useful to further improve performance for in-silico docking or lead-optimization in drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobJGADPHgkVrVg90H21EOLACvtfcHk0lhBCk-jucnzWg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXpsV2mtLo%253D&md5=0e231d630000e0155c2049c95067fb0e</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1021%2Fja105108q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja105108q%26sid%3Dliteratum%253Aachs%26aulast%3DWacker%26aufirst%3DD.%26aulast%3DFenalti%26aufirst%3DG.%26aulast%3DBrown%26aufirst%3DM.%2BA.%26aulast%3DKatritch%26aufirst%3DV.%26aulast%3DAbagyan%26aufirst%3DR.%26aulast%3DCherezov%26aufirst%3DV.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26atitle%3DConserved%2520binding%2520mode%2520of%2520human%2520%25CE%25B22%2520adrenergic%2520receptor%2520inverse%2520agonists%2520and%2520antagonist%2520revealed%2520by%2520x-ray%2520crystallography%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2010%26volume%3D132%26spage%3D11443%26epage%3D11445%26doi%3D10.1021%2Fja105108q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group">Götz, A. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williamson, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poole, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Le Grand, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walker, R. C.</span><span> </span><span class="NLM_article-title">Routine microsecond molecular dynamics simulations with AMBER on GPUs. 1. Generalized Born</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">1542</span><span class="NLM_x">–</span> <span class="NLM_lpage">1555</span><span class="refDoi"> DOI: 10.1021/ct200909j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct200909j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BC38XksFWns78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=1542-1555&author=A.+W.+G%C3%B6tzauthor=M.+J.+Williamsonauthor=D.+Xuauthor=D.+Pooleauthor=S.+Le+Grandauthor=R.+C.+Walker&title=Routine+microsecond+molecular+dynamics+simulations+with+AMBER+on+GPUs.+1.+Generalized+Born&doi=10.1021%2Fct200909j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Routine Microsecond Molecular Dynamics Simulations with AMBER on GPUs. 1. Generalized Born</span></div><div class="casAuthors">Gotz, Andreas W.; Williamson, Mark J.; Xu, Dong; Poole, Duncan; Le Grand, Scott; Walker, Ross C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1542-1555</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We present an implementation of generalized Born implicit solvent all-atom classical mol. dynamics (MD) within the AMBER program package that runs entirely on CUDA enabled NVIDIA graphics processing units (GPUs).  We discuss the algorithms that are used to exploit the processing power of the GPUs and show the performance that can be achieved in comparison to simulations on conventional CPU clusters.  The implementation supports three different precision models in which the contributions to the forces are calcd. in single precision floating point arithmetic but accumulated in double precision (SPDP), or everything is computed in single precision (SPSP) or double precision (DPDP).  In addn. to performance, we have focused on understanding the implications of the different precision models on the outcome of implicit solvent MD simulations.  We show results for a range of tests including the accuracy of single point force evaluations and energy conservation as well as structural properties pertaining to protein dynamics.  The numerical noise due to rounding errors within the SPSP precision model is sufficiently large to lead to an accumulation of errors which can result in unphys. trajectories for long time scale simulations.  We recommend the use of the mixed-precision SPDP model since the numerical results obtained are comparable with those of the full double precision DPDP model and the ref. double precision CPU implementation but at significantly reduced computational cost.  Our implementation provides performance for GB simulations on a single desktop that is on par with, and in some cases exceeds, that of traditional supercomputers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpshIijeXLcqLVg90H21EOLACvtfcHk0lhBCk-jucnzWg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XksFWns78%253D&md5=1e6f570db9cd504bb13706e7c56bc356</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1021%2Fct200909j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct200909j%26sid%3Dliteratum%253Aachs%26aulast%3DG%25C3%25B6tz%26aufirst%3DA.%2BW.%26aulast%3DWilliamson%26aufirst%3DM.%2BJ.%26aulast%3DXu%26aufirst%3DD.%26aulast%3DPoole%26aufirst%3DD.%26aulast%3DLe%2BGrand%26aufirst%3DS.%26aulast%3DWalker%26aufirst%3DR.%2BC.%26atitle%3DRoutine%2520microsecond%2520molecular%2520dynamics%2520simulations%2520with%2520AMBER%2520on%2520GPUs.%25201.%2520Generalized%2520Born%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2012%26volume%3D8%26spage%3D1542%26epage%3D1555%26doi%3D10.1021%2Fct200909j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group">Salomon-Ferrer, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Götz, A. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poole, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Le Grand, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walker, R. C.</span><span> </span><span class="NLM_article-title">Routine microsecond molecular dynamics simulations with AMBER on GPUs. 2. Explicit solvent particle mesh Ewald</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">3878</span><span class="NLM_x">–</span> <span class="NLM_lpage">3888</span><span class="refDoi"> DOI: 10.1021/ct400314y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct400314y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1arsrzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=3878-3888&author=R.+Salomon-Ferrerauthor=A.+W.+G%C3%B6tzauthor=D.+Pooleauthor=S.+Le+Grandauthor=R.+C.+Walker&title=Routine+microsecond+molecular+dynamics+simulations+with+AMBER+on+GPUs.+2.+Explicit+solvent+particle+mesh+Ewald&doi=10.1021%2Fct400314y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Routine Microsecond Molecular Dynamics Simulations with AMBER on GPUs. 2. Explicit Solvent Particle Mesh Ewald</span></div><div class="casAuthors">Salomon-Ferrer, Romelia; Gotz, Andreas W.; Poole, Duncan; Le Grand, Scott; Walker, Ross C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3878-3888</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We present an implementation of explicit solvent all atom classical mol. dynamics (MD) within the AMBER program package that runs entirely on CUDA-enabled GPUs.  First released publicly in Apr. 2010 as part of version 11 of the AMBER MD package and further improved and optimized over the last two years, this implementation supports the three most widely used statistical mech. ensembles (NVE, NVT, and NPT), uses particle mesh Ewald (PME) for the long-range electrostatics, and runs entirely on CUDA-enabled NVIDIA graphics processing units (GPUs), providing results that are statistically indistinguishable from the traditional CPU version of the software and with performance that exceeds that achievable by the CPU version of AMBER software running on all conventional CPU-based clusters and supercomputers.  We briefly discuss three different precision models developed specifically for this work (SPDP, SPFP, and DPDP) and highlight the tech. details of the approach as it extends beyond previously reported work [Goetz et al., J.  Chem.  Theory Comput.2012, DOI: 10.1021/ct200909j; Le Grand et al., Comp.  Phys.  Comm.2013, DOI: 10.1016/j.cpc.2012.09.022].We highlight the substantial improvements in performance that are seen over traditional CPU-only machines and provide validation of our implementation and precision models.  We also provide evidence supporting our decision to deprecate the previously described fully single precision (SPSP) model from the latest release of the AMBER software package.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoY-M2E_jgUmLVg90H21EOLACvtfcHk0lhBCk-jucnzWg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1arsrzP&md5=57e18d35e8b81e9a79788ef5af17d19f</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1021%2Fct400314y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct400314y%26sid%3Dliteratum%253Aachs%26aulast%3DSalomon-Ferrer%26aufirst%3DR.%26aulast%3DG%25C3%25B6tz%26aufirst%3DA.%2BW.%26aulast%3DPoole%26aufirst%3DD.%26aulast%3DLe%2BGrand%26aufirst%3DS.%26aulast%3DWalker%26aufirst%3DR.%2BC.%26atitle%3DRoutine%2520microsecond%2520molecular%2520dynamics%2520simulations%2520with%2520AMBER%2520on%2520GPUs.%25202.%2520Explicit%2520solvent%2520particle%2520mesh%2520Ewald%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2013%26volume%3D9%26spage%3D3878%26epage%3D3888%26doi%3D10.1021%2Fct400314y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group">Le Grand, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Götz, A. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walker, R. C.</span><span> </span><span class="NLM_article-title">SPFP: Speed without compromise—a mixed precision model for GPU accelerated molecular dynamics simulations</span> <span class="citation_source-journal">Comput. Phys. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">184</span><span class="NLM_x">, </span> <span class="NLM_fpage">374</span><span class="NLM_x">–</span> <span class="NLM_lpage">380</span><span class="refDoi"> DOI: 10.1016/j.cpc.2012.09.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1016%2Fj.cpc.2012.09.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsV2jsrzO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=184&publication_year=2013&pages=374-380&author=S.+Le+Grandauthor=A.+W.+G%C3%B6tzauthor=R.+C.+Walker&title=SPFP%3A+Speed+without+compromise%E2%80%94a+mixed+precision+model+for+GPU+accelerated+molecular+dynamics+simulations&doi=10.1016%2Fj.cpc.2012.09.022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">SPFP: Speed without compromise-A mixed precision model for GPU accelerated molecular dynamics simulations</span></div><div class="casAuthors">Le Grand, Scott; Gotz, Andreas W.; Walker, Ross C.</div><div class="citationInfo"><span class="NLM_cas:title">Computer Physics Communications</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">184</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">374-380</span>CODEN:
                <span class="NLM_cas:coden">CPHCBZ</span>;
        ISSN:<span class="NLM_cas:issn">0010-4655</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A new precision model is proposed for the acceleration of all-atom classical mol. dynamics (MD) simulations on graphics processing units (GPUs).  This precision model replaces double precision arithmetic with fixed point integer arithmetic for the accumulation of force components as compared to a previously introduced model that uses mixed single/double precision arithmetic.  This significantly boosts performance on modern GPU hardware without sacrificing numerical accuracy.  We present an implementation for NVIDIA GPUs of both generalized Born implicit solvent simulations as well as explicit solvent simulations using the particle mesh Ewald (PME) algorithm for long-range electrostatics using this precision model.  Tests demonstrate both the performance of this implementation as well as its numerical stability for const. energy and const. temp. biomol. MD as compared to a double precision CPU implementation and double and mixed single/double precision GPU implementations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJB_spY7SrSbVg90H21EOLACvtfcHk0li9cd9LV2KIoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsV2jsrzO&md5=2d9f102cb17f6e9bcbff3b5057d8c587</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1016%2Fj.cpc.2012.09.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cpc.2012.09.022%26sid%3Dliteratum%253Aachs%26aulast%3DLe%2BGrand%26aufirst%3DS.%26aulast%3DG%25C3%25B6tz%26aufirst%3DA.%2BW.%26aulast%3DWalker%26aufirst%3DR.%2BC.%26atitle%3DSPFP%253A%2520Speed%2520without%2520compromise%25E2%2580%2594a%2520mixed%2520precision%2520model%2520for%2520GPU%2520accelerated%2520molecular%2520dynamics%2520simulations%26jtitle%3DComput.%2520Phys.%2520Commun.%26date%3D2013%26volume%3D184%26spage%3D374%26epage%3D380%26doi%3D10.1016%2Fj.cpc.2012.09.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref69"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref69'); return false;" data-citation="" class="refNumLink">69</a></strong><div class="NLM_citation" id="cit69"><span><span class="NLM_contrib-group">Maier, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martinez, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kasavajhala, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wickstrom, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hauser, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simmerling, C.</span><span> </span><span class="NLM_article-title">ff14SB: improving the accuracy of protein side chain and backbone parameters from ff99SB</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">3696</span><span class="NLM_x">–</span> <span class="NLM_lpage">3713</span><span class="refDoi"> DOI: 10.1021/acs.jctc.5b00255</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jctc.5b00255" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFequ7rN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=3696-3713&author=J.+A.+Maierauthor=C.+Martinezauthor=K.+Kasavajhalaauthor=L.+Wickstromauthor=K.+E.+Hauserauthor=C.+Simmerling&title=ff14SB%3A+improving+the+accuracy+of+protein+side+chain+and+backbone+parameters+from+ff99SB&doi=10.1021%2Facs.jctc.5b00255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">ff14SB: Improving the Accuracy of Protein Side Chain and Backbone Parameters from ff99SB</span></div><div class="casAuthors">Maier, James A.; Martinez, Carmenza; Kasavajhala, Koushik; Wickstrom, Lauren; Hauser, Kevin E.; Simmerling, Carlos</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3696-3713</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. mechanics is powerful for its speed in atomistic simulations, but an accurate force field is required.  The Amber ff99SB force field improved protein secondary structure balance and dynamics from earlier force fields like ff99, but weaknesses in side chain rotamer and backbone secondary structure preferences have been identified.  Here, we performed a complete refit of all amino acid side chain dihedral parameters, which had been carried over from ff94.  The training set of conformations included multidimensional dihedral scans designed to improve transferability of the parameters.  Improvement in all amino acids was obtained as compared to ff99SB.  Parameters were also generated for alternate protonation states of ionizable side chains.  Av. errors in relative energies of pairs of conformations were under 1.0 kcal/mol as compared to QM, reduced 35% from ff99SB.  We also took the opportunity to make empirical adjustments to the protein backbone dihedral parameters as compared to ff99SB.  Multiple small adjustments of φ and ψ parameters were tested against NMR scalar coupling data and secondary structure content for short peptides.  The best results were obtained from a phys. motivated adjustment to the φ rotational profile that compensates for lack of ff99SB QM training data in the β-ppII transition region.  Together, these backbone and side chain modifications (hereafter called ff14SB) not only better reproduced their benchmarks, but also improved secondary structure content in small peptides and reprodn. of NMR χ1 scalar coupling measurements for proteins in soln.  We also discuss the Amber ff12SB parameter set, a preliminary version of ff14SB that includes most of its improvements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgntnN9ES97rVg90H21EOLACvtfcHk0li9cd9LV2KIoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFequ7rN&md5=7b803577b3b6912cc6750cfbd356596e</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1021%2Facs.jctc.5b00255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jctc.5b00255%26sid%3Dliteratum%253Aachs%26aulast%3DMaier%26aufirst%3DJ.%2BA.%26aulast%3DMartinez%26aufirst%3DC.%26aulast%3DKasavajhala%26aufirst%3DK.%26aulast%3DWickstrom%26aufirst%3DL.%26aulast%3DHauser%26aufirst%3DK.%2BE.%26aulast%3DSimmerling%26aufirst%3DC.%26atitle%3Dff14SB%253A%2520improving%2520the%2520accuracy%2520of%2520protein%2520side%2520chain%2520and%2520backbone%2520parameters%2520from%2520ff99SB%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2015%26volume%3D11%26spage%3D3696%26epage%3D3713%26doi%3D10.1021%2Facs.jctc.5b00255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref70"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref70'); return false;" data-citation="" class="refNumLink">70</a></strong><div class="NLM_citation" id="cit70"><span><span class="NLM_contrib-group">Bayly, C. I.; McKay, D.; Truchon, J.-F.</span> An informal AMBER small molecule force field: <a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="a7d7c6d5cae7e1d5c8d4d4d3">[email protected]</a>; <a href="http://www.ccl.net/cca/data/parm_at_Frosst/parm_Frosst_note.pdf" class="extLink">http://www.ccl.net/cca/data/parm_at_Frosst/parm_Frosst_note.pdf</a> (accessed Jan 15, 2015).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Bayly%2C+C.+I.%3B+McKay%2C+D.%3B+Truchon%2C+J.-F.+An+informal+AMBER+small+molecule+force+field%3A+parm%40Frosst%3B+http%3A%2F%2Fwww.ccl.net%2Fcca%2Fdata%2Fparm_at_Frosst%2Fparm_Frosst_note.pdf+%28accessed+Jan+15%2C+2015%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26aulast%3DBayly%26aufirst%3DC.%2BI." title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref71"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref71'); return false;" data-citation="" class="refNumLink">71</a></strong><div class="NLM_citation" id="cit71"><span><span class="NLM_contrib-group">Jakalian, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bush, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jack, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bayly, C. I.</span><span> </span><span class="NLM_article-title">Fast, efficient generation of high-quality atomic charges. AM1-BCC model: I. Method</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">132</span><span class="NLM_x">–</span> <span class="NLM_lpage">146</span><span class="refDoi"> DOI: 10.1002/(SICI)1096-987X(20000130)21:2<132::AID-JCC5>3.0.CO;2-P</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1002%2F%28SICI%291096-987X%2820000130%2921%3A2%3C132%3A%3AAID-JCC5%3E3.0.CO%3B2-P" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BD3cXkt1SgsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2000&pages=132-146&author=A.+Jakalianauthor=B.+L.+Bushauthor=D.+B.+Jackauthor=C.+I.+Bayly&title=Fast%2C+efficient+generation+of+high-quality+atomic+charges.+AM1-BCC+model%3A+I.+Method&doi=10.1002%2F%28SICI%291096-987X%2820000130%2921%3A2%3C132%3A%3AAID-JCC5%3E3.0.CO%3B2-P"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Fast, efficient generation of high-quality atomic charges. AM1-BCC model: I. Method</span></div><div class="casAuthors">Jakalian, Araz; Bush, Bruce L.; Jack, David B.; Bayly, Christopher I.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">132-146</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The AM1-BCC method quickly and efficiently generates high-quality at. charges for use in condensed-phase simulations.  The underlying features of the electron distribution including formal charge and delocalization are first captured by AM1 at. charges for the individual mol.  Bond charge corrections (BCCs), which have been parameterized against the HF/6-31G* electrostatic potential (ESP) of a training set of compds. contg. relevant functional groups, are then added using a formalism identical to the consensus BCI (bond charge increment) approach.  As a proof of the concept, we fit BCCs simultaneously to 45 compds. including O-, N-, and S-contg. functionalities, aroms., and heteroaroms., using only 41 BCC parameters.  AM1-BCC yields charge sets of comparable quality to HF/6-31G* ESP-derived charges in a fraction of the time while reducing instabilities in the at. charges compared to direct ESP-fit methods.  We then apply the BCC parameters to a small "test set" consisting of aspirin, D-glucose, and eryodictyol; the AM1-BCC model again provides at. charges of quality comparable with HF/6-31G* RESP charges, as judged by an increase of only 0.01 to 0.02 at. units in the root-mean-square (RMS) error in ESP.  Based on these encouraging results, we intend to parameterize the AM1-BCC model to provide a consistent charge model for any org. or biol. mol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruxZWGx7IgLbVg90H21EOLACvtfcHk0li9cd9LV2KIoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXkt1SgsQ%253D%253D&md5=aa4774739d3491dbc8dd95ed1948d856</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291096-987X%2820000130%2921%3A2%3C132%3A%3AAID-JCC5%3E3.0.CO%3B2-P&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291096-987X%252820000130%252921%253A2%253C132%253A%253AAID-JCC5%253E3.0.CO%253B2-P%26sid%3Dliteratum%253Aachs%26aulast%3DJakalian%26aufirst%3DA.%26aulast%3DBush%26aufirst%3DB.%2BL.%26aulast%3DJack%26aufirst%3DD.%2BB.%26aulast%3DBayly%26aufirst%3DC.%2BI.%26atitle%3DFast%252C%2520efficient%2520generation%2520of%2520high-quality%2520atomic%2520charges.%2520AM1-BCC%2520model%253A%2520I.%2520Method%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2000%26volume%3D21%26spage%3D132%26epage%3D146%26doi%3D10.1002%2F%28SICI%291096-987X%2820000130%2921%3A2%3C132%3A%3AAID-JCC5%3E3.0.CO%3B2-P" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref72"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref72'); return false;" data-citation="" class="refNumLink">72</a></strong><div class="NLM_citation" id="cit72"><span><span class="NLM_contrib-group">Jakalian, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jack, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bayly, C. I.</span><span> </span><span class="NLM_article-title">Fast, efficient generation of high-quality atomic charges. AM1-BCC model: II. Parameterization and validation</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">1623</span><span class="NLM_x">–</span> <span class="NLM_lpage">1641</span><span class="refDoi"> DOI: 10.1002/jcc.10128</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1002%2Fjcc.10128" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=12395429" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BD38XosF2rt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2002&pages=1623-1641&author=A.+Jakalianauthor=D.+B.+Jackauthor=C.+I.+Bayly&title=Fast%2C+efficient+generation+of+high-quality+atomic+charges.+AM1-BCC+model%3A+II.+Parameterization+and+validation&doi=10.1002%2Fjcc.10128"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Fast, efficient generation of high-quality atomic charges. AM1-BCC model: II. parameterization and validation</span></div><div class="casAuthors">Jakalian, Araz; Jack, David B.; Bayly, Christopher I.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1623-1641</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">We present the first global parameterization and validation of a novel charge model, called AM1-BCC, which quickly and efficiently generates high-quality at. charges for computer simulations of org. mols. in polar media.  The goal of the charge model is to produce at. charges that emulate the HF/6-31G* electrostatic potential (ESP) of a mol.  Underlying electronic structure features, including formal charge and electron delocalization, are first captured by AM1 population charges; simple additive bond charge corrections (BCCs) are then applied to these AM1 at. charges to produce the AM1-BCC charges.  The parameterization of BCCs was carried out by fitting to the HF/6-31G* ESP of a training set of >2700 mols.  Most org. functional groups and their combinations were sampled, as well as an extensive variety of cyclic and fused bicyclic heteroaryl systems.  The resulting BCC parameters allow the AM1-BCC charging scheme to handle virtually all types of org. compds. listed in The Merck Index and the NCI Database.  Validation of the model was done through comparisons of hydrogen-bonded dimer energies and relative free energies of solvation using AM1-BCC charges in conjunction with the 1994 Cornell et al. forcefield for AMBER.  Homo-dimer and hetero-dimer hydrogen-bond energies of a diverse set of org. mols. were reproduced to within 0.95 kcal/mol RMS deviation from the ab initio values, and for DNA dimers the energies were within 0.9 kcal/mol RMS deviation from ab initio values.  The calcd. relative free energies of solvation for a diverse set of monofunctional isosteres were reproduced to within 0.69 kcal/mol of expt.  In all these validation tests, AMBER with the AM1-BCC charge model maintained a correlation coeff. above 0.96.  Thus, the parameters presented here for use with the AM1-BCC method present a fast, accurate, and robust alternative to HF/6-31G* ESP-fit charges for general use with the AMBER force field in computer simulations involving org. small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpn9DkJ_gMzA7Vg90H21EOLACvtfcHk0lgqZpDUPT9aaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XosF2rt74%253D&md5=3f2b738617bb17b2898f7ac4d751d7ec</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1002%2Fjcc.10128&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.10128%26sid%3Dliteratum%253Aachs%26aulast%3DJakalian%26aufirst%3DA.%26aulast%3DJack%26aufirst%3DD.%2BB.%26aulast%3DBayly%26aufirst%3DC.%2BI.%26atitle%3DFast%252C%2520efficient%2520generation%2520of%2520high-quality%2520atomic%2520charges.%2520AM1-BCC%2520model%253A%2520II.%2520Parameterization%2520and%2520validation%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2002%26volume%3D23%26spage%3D1623%26epage%3D1641%26doi%3D10.1002%2Fjcc.10128" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref73"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref73'); return false;" data-citation="" class="refNumLink">73</a></strong><div class="NLM_citation" id="cit73"><span><span class="NLM_contrib-group">Beauchamp, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Behr, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rustenburg, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bayly, C. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kroenlein, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chodera, J. D.</span><span> </span><span class="NLM_article-title">Toward automated benchmarking of atomistic force fields: neat liquid densities and static dielectric constants from the ThermoML data archive</span> <span class="citation_source-journal">J. Phys. Chem. B</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">119</span><span class="NLM_x">, </span> <span class="NLM_fpage">12912</span><span class="NLM_x">–</span> <span class="NLM_lpage">12920</span><span class="refDoi"> DOI: 10.1021/acs.jpcb.5b06703</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jpcb.5b06703" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVGjsbfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2015&pages=12912-12920&author=K.+A.+Beauchampauthor=J.+M.+Behrauthor=A.+S.+Rustenburgauthor=C.+I.+Baylyauthor=K.+Kroenleinauthor=J.+D.+Chodera&title=Toward+automated+benchmarking+of+atomistic+force+fields%3A+neat+liquid+densities+and+static+dielectric+constants+from+the+ThermoML+data+archive&doi=10.1021%2Facs.jpcb.5b06703"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">Toward Automated Benchmarking of Atomistic Force Fields: Neat Liquid Densities and Static Dielectric Constants from the ThermoML Data Archive</span></div><div class="casAuthors">Beauchamp, Kyle A.; Behr, Julie M.; Rustenburg, Arien S.; Bayly, Christopher I.; Kroenlein, Kenneth; Chodera, John D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">40</span>),
    <span class="NLM_cas:pages">12912-12920</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1520-5207</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Atomistic mol. simulations are a powerful way to make quant. predictions, but the accuracy of these predictions depends entirely on the quality of the force field employed.  Although exptl. measurements of fundamental phys. properties offer a straightforward approach for evaluating force field quality, the bulk of this information has been tied up in formats that are not machine-readable.  Compiling benchmark data sets of phys. properties from non-machine-readable sources requires substantial human effort and is prone to the accumulation of human errors, hindering the development of reproducible benchmarks of force-field accuracy.  Here, we examine the feasibility of benchmarking atomistic force fields against the NIST ThermoML data archive of physicochem. measurements, which aggregates thousands of exptl. measurements in a portable, machine-readable, self-annotating IUPAC-std. format.  As a proof of concept, we present a detailed benchmark of the generalized Amber small-mol. force field (GAFF) using the AM1-BCC charge model against exptl. measurements (specifically, bulk liq. densities and static dielec. consts. at ambient pressure) automatically extd. from the archive and discuss the extent of data available for use in larger scale (or continuously performed) benchmarks.  The results of even this limited initial benchmark highlight a general problem with fixed-charge force fields in the representation low-dielec. environments, such as those seen in binding cavities or biol. membranes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3vmYoWhqCXrVg90H21EOLACvtfcHk0lgqZpDUPT9aaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVGjsbfJ&md5=8d4a5cdc9e975435ef3eec90fb222279</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1021%2Facs.jpcb.5b06703&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jpcb.5b06703%26sid%3Dliteratum%253Aachs%26aulast%3DBeauchamp%26aufirst%3DK.%2BA.%26aulast%3DBehr%26aufirst%3DJ.%2BM.%26aulast%3DRustenburg%26aufirst%3DA.%2BS.%26aulast%3DBayly%26aufirst%3DC.%2BI.%26aulast%3DKroenlein%26aufirst%3DK.%26aulast%3DChodera%26aufirst%3DJ.%2BD.%26atitle%3DToward%2520automated%2520benchmarking%2520of%2520atomistic%2520force%2520fields%253A%2520neat%2520liquid%2520densities%2520and%2520static%2520dielectric%2520constants%2520from%2520the%2520ThermoML%2520data%2520archive%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2015%26volume%3D119%26spage%3D12912%26epage%3D12920%26doi%3D10.1021%2Facs.jpcb.5b06703" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref74"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref74'); return false;" data-citation="" class="refNumLink">74</a></strong><div class="NLM_citation" id="cit74"><span><span class="NLM_contrib-group">Dickson, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosso, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Betz, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walker, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gould, I. R.</span><span> </span><span class="NLM_article-title">GAFFlipid: a General Amber Force Field for the accurate molecular dynamics simulation of phospholipid</span> <span class="citation_source-journal">Soft Matter</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">9617</span><span class="NLM_x">–</span> <span class="NLM_lpage">9627</span><span class="refDoi"> DOI: 10.1039/c2sm26007g</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1039%2Fc2sm26007g" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1ymsbzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=9617-9627&author=C.+J.+Dicksonauthor=L.+Rossoauthor=R.+M.+Betzauthor=R.+C.+Walkerauthor=I.+R.+Gould&title=GAFFlipid%3A+a+General+Amber+Force+Field+for+the+accurate+molecular+dynamics+simulation+of+phospholipid&doi=10.1039%2Fc2sm26007g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">GAFFlipid: a General Amber Force Field for the accurate molecular dynamics simulation of phospholipid</span></div><div class="casAuthors">Dickson, Callum J.; Rosso, Lula; Betz, Robin M.; Walker, Ross C.; Gould, Ian R.</div><div class="citationInfo"><span class="NLM_cas:title">Soft Matter</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">37</span>),
    <span class="NLM_cas:pages">9617-9627</span>CODEN:
                <span class="NLM_cas:coden">SMOABF</span>;
        ISSN:<span class="NLM_cas:issn">1744-683X</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Previous attempts to simulate phospholipid bilayers using the General Amber Force Field (GAFF) yielded many bilayer characteristics in agreement with expt., however when using a tensionless NPT ensemble the bilayer is seen to compress to an undesirable extent resulting in low areas per lipid and high order parameters in comparison to expt.  In this work, the GAFF Lennard-Jones parameters for the simulation of acyl chains are cor. to allow the accurate and stable simulation of pure lipid bilayers.  Lipid bilayers comprised of six phospholipid types were simulated for timescales approaching a quarter of a microsecond under tensionless const. pressure conditions using Graphics Processing Units.  Structural properties including area per lipid, vol. per lipid, bilayer thickness, order parameter and headgroup hydration show favorable agreement with available exptl. values.  Expanding the system size from 72 to 288 lipids and a more exptl. realistic 2 × 288 lipid bilayer stack induces little change in the obsd. properties.  This preliminary work is intended for combination with the new AMBER Lipid11 modular force field as part of on-going attempts to create a modular phospholipid AMBER force field allowing tensionless NPT simulations of complex lipid bilayers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplZfTlqhsP_bVg90H21EOLACvtfcHk0lgqZpDUPT9aaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1ymsbzL&md5=494ab83d6e79d084d2876c2169b9add1</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1039%2Fc2sm26007g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc2sm26007g%26sid%3Dliteratum%253Aachs%26aulast%3DDickson%26aufirst%3DC.%2BJ.%26aulast%3DRosso%26aufirst%3DL.%26aulast%3DBetz%26aufirst%3DR.%2BM.%26aulast%3DWalker%26aufirst%3DR.%2BC.%26aulast%3DGould%26aufirst%3DI.%2BR.%26atitle%3DGAFFlipid%253A%2520a%2520General%2520Amber%2520Force%2520Field%2520for%2520the%2520accurate%2520molecular%2520dynamics%2520simulation%2520of%2520phospholipid%26jtitle%3DSoft%2520Matter%26date%3D2012%26volume%3D8%26spage%3D9617%26epage%3D9627%26doi%3D10.1039%2Fc2sm26007g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref75"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref75'); return false;" data-citation="" class="refNumLink">75</a></strong><div class="NLM_citation" id="cit75"><span><span class="NLM_contrib-group">Skjevik, Å. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Madej, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walker, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teigen, K.</span><span> </span><span class="NLM_article-title">LIPID11: a modular framework for lipid simulations using amber</span> <span class="citation_source-journal">J. Phys. Chem. B</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">116</span><span class="NLM_x">, </span> <span class="NLM_fpage">11124</span><span class="NLM_x">–</span> <span class="NLM_lpage">11136</span><span class="refDoi"> DOI: 10.1021/jp3059992</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp3059992" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1Gmsr7K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2012&pages=11124-11136&author=%C3%85.+A.+Skjevikauthor=B.+D.+Madejauthor=R.+C.+Walkerauthor=K.+Teigen&title=LIPID11%3A+a+modular+framework+for+lipid+simulations+using+amber&doi=10.1021%2Fjp3059992"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">LIPID11: A Modular Framework for Lipid Simulations Using Amber</span></div><div class="casAuthors">Skjevik, Aage A.; Madej, Benjamin D.; Walker, Ross C.; Teigen, Knut</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">11124-11136</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1520-5207</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Accurate simulation of complex lipid bilayers has long been a goal in condensed phase mol. dynamics (MD).  Structure and function of membrane-bound proteins are highly dependent on the lipid bilayer environment and are challenging to study through exptl. methods.  Within Amber, there has been limited focus on lipid simulations, although some success has been seen with the use of the General Amber Force Field (GAFF).  However, to date there are no dedicated Amber lipid force fields.  In this paper we describe a new charge derivation strategy for lipids consistent with the Amber RESP approach and a new atom and residue naming and type convention.  In the first instance, we have combined this approach with GAFF parameters.  The result is LIPID11, a flexible, modular framework for the simulation of lipids that is fully compatible with the existing Amber force fields.  The charge derivation procedure, capping strategy, and nomenclature for LIPID11, along with preliminary simulation results and a discussion of the planned long-term parameter development are presented here.  Our findings suggest that LIPID11 is a modular framework feasible for phospholipids and a flexible starting point for the development of a comprehensive, Amber-compatible lipid force field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr22eGSs_c6cLVg90H21EOLACvtfcHk0lj0k1G3f1-f7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1Gmsr7K&md5=c5ec086a4f0e489bd6ec3141314d6592</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1021%2Fjp3059992&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp3059992%26sid%3Dliteratum%253Aachs%26aulast%3DSkjevik%26aufirst%3D%25C3%2585.%2BA.%26aulast%3DMadej%26aufirst%3DB.%2BD.%26aulast%3DWalker%26aufirst%3DR.%2BC.%26aulast%3DTeigen%26aufirst%3DK.%26atitle%3DLIPID11%253A%2520a%2520modular%2520framework%2520for%2520lipid%2520simulations%2520using%2520amber%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2012%26volume%3D116%26spage%3D11124%26epage%3D11136%26doi%3D10.1021%2Fjp3059992" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref76"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref76'); return false;" data-citation="" class="refNumLink">76</a></strong><div class="NLM_citation" id="cit76"><span><span class="NLM_contrib-group">Dickson, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Madej, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skjevik, Å. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Betz, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teigen, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gould, I. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walker, R. C.</span><span> </span><span class="NLM_article-title">Lipid14: the amber lipid force field</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">865</span><span class="NLM_x">–</span> <span class="NLM_lpage">879</span><span class="refDoi"> DOI: 10.1021/ct4010307</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct4010307" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1Wgt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=865-879&author=C.+J.+Dicksonauthor=B.+D.+Madejauthor=%C3%85.+A.+Skjevikauthor=R.+M.+Betzauthor=K.+Teigenauthor=I.+R.+Gouldauthor=R.+C.+Walker&title=Lipid14%3A+the+amber+lipid+force+field&doi=10.1021%2Fct4010307"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">Lipid14: The Amber Lipid Force Field</span></div><div class="casAuthors">Dickson, Callum J.; Madej, Benjamin D.; Skjevik, Age A.; Betz, Robin M.; Teigen, Knut; Gould, Ian R.; Walker, Ross C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">865-879</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The AMBER lipid force field has been updated to create Lipid14, allowing tensionless simulation of a no. of lipid types with the AMBER MD package.  The modular nature of this force field allows numerous combinations of head and tail groups to create different lipid types, enabling the easy insertion of new lipid species.  The Lennard-Jones and torsion parameters of both the head and tail groups have been revised and updated partial charges calcd.  The force field has been validated by simulating bilayers of six different lipid types for a total of 0.5 μs each without applying a surface tension; with favorable comparison to expt. for properties such as area per lipid, vol. per lipid, bilayer thickness, NMR order parameters, scattering data, and lipid lateral diffusion.  As the derivation of this force field is consistent with the AMBER development philosophy, Lipid14 is compatible with the AMBER protein, nucleic acid, carbohydrate, and small mol. force fields.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo60Ks38bQOe7Vg90H21EOLACvtfcHk0lj0k1G3f1-f7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1Wgt70%253D&md5=6efe3346bab426a93016d10418b6957a</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1021%2Fct4010307&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct4010307%26sid%3Dliteratum%253Aachs%26aulast%3DDickson%26aufirst%3DC.%2BJ.%26aulast%3DMadej%26aufirst%3DB.%2BD.%26aulast%3DSkjevik%26aufirst%3D%25C3%2585.%2BA.%26aulast%3DBetz%26aufirst%3DR.%2BM.%26aulast%3DTeigen%26aufirst%3DK.%26aulast%3DGould%26aufirst%3DI.%2BR.%26aulast%3DWalker%26aufirst%3DR.%2BC.%26atitle%3DLipid14%253A%2520the%2520amber%2520lipid%2520force%2520field%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2014%26volume%3D10%26spage%3D865%26epage%3D879%26doi%3D10.1021%2Fct4010307" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref77"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref77'); return false;" data-citation="" class="refNumLink">77</a></strong><div class="NLM_citation" id="cit77"><span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chandrasekhar, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Madura, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Impey, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klein, M. L.</span><span> </span><span class="NLM_article-title">Comparison of simple potential functions for simulating liquid water</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1983</span><span class="NLM_x">, </span> <span class="NLM_volume">79</span><span class="NLM_x">, </span> <span class="NLM_fpage">926</span><span class="NLM_x">–</span> <span class="NLM_lpage">935</span><span class="refDoi"> DOI: 10.1063/1.445869</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1063%2F1.445869" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADyaL3sXksF2htL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=1983&pages=926-935&author=W.+L.+Jorgensenauthor=J.+Chandrasekharauthor=J.+D.+Maduraauthor=R.+W.+Impeyauthor=M.+L.+Klein&title=Comparison+of+simple+potential+functions+for+simulating+liquid+water&doi=10.1063%2F1.445869"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of simple potential functions for simulating liquid water</span></div><div class="casAuthors">Jorgensen, William L.; Chandrasekhar, Jayaraman; Madura, Jeffry D.; Impey, Roger W.; Klein, Michael L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1983</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">926-35</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">Classical Monte Carlo simulations were carried out for liq. H2O in the NPT ensemble at 25° and 1 atm using 6 of the simpler intermol. potential functions for the dimer.  Comparisons were made with exptl. thermodn. and structural data including the neutron diffraction results of Thiessen and Narten (1982).  The computed densities and potential energies agree with expt. except for the original Bernal-Fowler model, which yields an 18% overest. of the d. and poor structural results.  The discrepancy may be due to the correction terms needed in processing the neutron data or to an effect uniformly neglected in the computations.  Comparisons were made for the self-diffusion coeffs. obtained from mol. dynamics simulations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpK4QUFMvwYFrVg90H21EOLACvtfcHk0lj0k1G3f1-f7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3sXksF2htL4%253D&md5=a1161334e381746be8c9b15a5e56f704</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1063%2F1.445869&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.445869%26sid%3Dliteratum%253Aachs%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DChandrasekhar%26aufirst%3DJ.%26aulast%3DMadura%26aufirst%3DJ.%2BD.%26aulast%3DImpey%26aufirst%3DR.%2BW.%26aulast%3DKlein%26aufirst%3DM.%2BL.%26atitle%3DComparison%2520of%2520simple%2520potential%2520functions%2520for%2520simulating%2520liquid%2520water%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1983%26volume%3D79%26spage%3D926%26epage%3D935%26doi%3D10.1063%2F1.445869" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref78"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref78'); return false;" data-citation="" class="refNumLink">78</a></strong><div class="NLM_citation" id="cit78"><span><span class="NLM_contrib-group">Press, W. H.; Teukolsky, S. A.; Vetterling, W. T.; Flannery, B. P.</span><span> </span><span class="NLM_article-title">Conjugate Gradient Methods in Multidimensions</span>. In  <span class="citation_source-book">Numerical Recipes: The Art of Scientific Computing</span>, <span class="NLM_edition">3rd</span> ed.; <span class="NLM_publisher-name">Cambridge University Press</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">2007</span><span class="NLM_x">; </span>Chapter 10.8, pp  <span class="NLM_fpage">515</span><span class="NLM_x">–</span> <span class="NLM_lpage">518</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2007&pages=515-518&author=W.+H.+Press&author=S.+A.+Teukolsky&author=W.+T.+Vetterling&author=B.+P.+Flannery&title=Numerical+Recipes%3A+The+Art+of+Scientific+Computing"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DPress%26aufirst%3DW.%2BH.%26atitle%3DConjugate%2520Gradient%2520Methods%2520in%2520Multidimensions%26btitle%3DNumerical%2520Recipes%253A%2520The%2520Art%2520of%2520Scientific%2520Computing%26pub%3DCambridge%2520University%2520Press%26date%3D2007%26spage%3D515%26epage%3D518" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref79"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref79'); return false;" data-citation="" class="refNumLink">79</a></strong><div class="NLM_citation" id="cit79"><span><span class="NLM_contrib-group">Pastor, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brooks, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Szabo, A.</span><span> </span><span class="NLM_article-title">An analysis of the accuracy of Langevin and molecular dynamics algorithms</span> <span class="citation_source-journal">Mol. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1988</span><span class="NLM_x">, </span> <span class="NLM_volume">65</span><span class="NLM_x">, </span> <span class="NLM_fpage">1409</span><span class="NLM_x">–</span> <span class="NLM_lpage">1419</span><span class="refDoi"> DOI: 10.1080/00268978800101881</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1080%2F00268978800101881" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=1988&pages=1409-1419&author=R.+Pastorauthor=B.+Brooksauthor=A.+Szabo&title=An+analysis+of+the+accuracy+of+Langevin+and+molecular+dynamics+algorithms&doi=10.1080%2F00268978800101881"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1080%2F00268978800101881&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F00268978800101881%26sid%3Dliteratum%253Aachs%26aulast%3DPastor%26aufirst%3DR.%26aulast%3DBrooks%26aufirst%3DB.%26aulast%3DSzabo%26aufirst%3DA.%26atitle%3DAn%2520analysis%2520of%2520the%2520accuracy%2520of%2520Langevin%2520and%2520molecular%2520dynamics%2520algorithms%26jtitle%3DMol.%2520Phys.%26date%3D1988%26volume%3D65%26spage%3D1409%26epage%3D1419%26doi%3D10.1080%2F00268978800101881" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref80"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref80'); return false;" data-citation="" class="refNumLink">80</a></strong><div class="NLM_citation" id="cit80"><span><span class="NLM_contrib-group">Berendsen, H. J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Postma, J. P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Gunsteren, W. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DiNola, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haak, J. R.</span><span> </span><span class="NLM_article-title">Molecular dynamics with coupling to an external bath</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1984</span><span class="NLM_x">, </span> <span class="NLM_volume">81</span><span class="NLM_x">, </span> <span class="NLM_fpage">3684</span><span class="NLM_x">–</span> <span class="NLM_lpage">3690</span><span class="refDoi"> DOI: 10.1063/1.448118</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1063%2F1.448118" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADyaL2cXmtlGksbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=1984&pages=3684-3690&author=H.+J.+C.+Berendsenauthor=J.+P.+M.+Postmaauthor=W.+F.+van+Gunsterenauthor=A.+DiNolaauthor=J.+R.+Haak&title=Molecular+dynamics+with+coupling+to+an+external+bath&doi=10.1063%2F1.448118"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit80R"><div class="casContent"><span class="casTitleNuber">80</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular dynamics with coupling to an external bath</span></div><div class="casAuthors">Berendsen, H. J. C.; Postma, J. P. M.; Van Gunsteren, W. F.; DiNola, A.; Haak, J. R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1984</span>),
    <span class="NLM_cas:volume">81</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3684-90</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">In mol. dynamics (MD) simulations, the need often arises to maintain such parameters as temp. or pressure rather than energy and vol., or to impose gradients for studying transport properties in nonequil. MD.  A method is described to realize coupling to an external bath with const. temp. or pressure with adjustable time consts. for the coupling.  The method is easily extendable to other variables and to gradients, and can be applied also to polyat. mols. involving internal constraints.  The influence of coupling time consts. on dynamical variables is evaluated.  A leap-frog algorithm is presented for the general case involving constraints with coupling to both a const. temp. and a const. pressure bath.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoecrpUZWkh37Vg90H21EOLACvtfcHk0ljjiN9Rq2q2jA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2cXmtlGksbY%253D&md5=5510dc00297d63b91ee3a7a4ae5aacb1</span></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1063%2F1.448118&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.448118%26sid%3Dliteratum%253Aachs%26aulast%3DBerendsen%26aufirst%3DH.%2BJ.%2BC.%26aulast%3DPostma%26aufirst%3DJ.%2BP.%2BM.%26aulast%3Dvan%2BGunsteren%26aufirst%3DW.%2BF.%26aulast%3DDiNola%26aufirst%3DA.%26aulast%3DHaak%26aufirst%3DJ.%2BR.%26atitle%3DMolecular%2520dynamics%2520with%2520coupling%2520to%2520an%2520external%2520bath%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1984%26volume%3D81%26spage%3D3684%26epage%3D3690%26doi%3D10.1063%2F1.448118" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref81"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref81'); return false;" data-citation="" class="refNumLink">81</a></strong><div class="NLM_citation" id="cit81"><span><span class="NLM_contrib-group">Ryckaert, J.-P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciccotti, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berendsen, H. J. C.</span><span> </span><span class="NLM_article-title">Numerical integration of the cartesian equations of motion of a system with constraints: molecular dynamics of n-alkanes</span> <span class="citation_source-journal">J. Comput. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1977</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">327</span><span class="NLM_x">–</span> <span class="NLM_lpage">341</span><span class="refDoi"> DOI: 10.1016/0021-9991(77)90098-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1016%2F0021-9991%2877%2990098-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADyaE2sXktVGhsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=1977&pages=327-341&author=J.-P.+Ryckaertauthor=G.+Ciccottiauthor=H.+J.+C.+Berendsen&title=Numerical+integration+of+the+cartesian+equations+of+motion+of+a+system+with+constraints%3A+molecular+dynamics+of+n-alkanes&doi=10.1016%2F0021-9991%2877%2990098-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit81R"><div class="casContent"><span class="casTitleNuber">81</span><div class="casTitle"><span class="NLM_cas:atitle">Numerical integration of the Cartesian equations of motion of a system with constraints:  molecular dynamics of n-alkanes</span></div><div class="casAuthors">Ryckaert, Jean Paul; Ciccotti, Giovanni; Berendsen, Herman J. C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Physics</span>
        (<span class="NLM_cas:date">1977</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">327-41</span>CODEN:
                <span class="NLM_cas:coden">JCTPAH</span>;
        ISSN:<span class="NLM_cas:issn">0021-9991</span>.
    </div><div class="casAbstract">A numerical algorithm integrating the 3N Cartesian equation of motion of a system of N points subject to holonomic constraints is applied to mol. dynamics simulation of a liq. of 64 butane mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYPlhG1U3Uw7Vg90H21EOLACvtfcHk0ljjiN9Rq2q2jA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE2sXktVGhsL4%253D&md5=b4aecddfde149117813a5ea4f5353ce2</span></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1016%2F0021-9991%2877%2990098-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0021-9991%252877%252990098-5%26sid%3Dliteratum%253Aachs%26aulast%3DRyckaert%26aufirst%3DJ.-P.%26aulast%3DCiccotti%26aufirst%3DG.%26aulast%3DBerendsen%26aufirst%3DH.%2BJ.%2BC.%26atitle%3DNumerical%2520integration%2520of%2520the%2520cartesian%2520equations%2520of%2520motion%2520of%2520a%2520system%2520with%2520constraints%253A%2520molecular%2520dynamics%2520of%2520n-alkanes%26jtitle%3DJ.%2520Comput.%2520Phys.%26date%3D1977%26volume%3D23%26spage%3D327%26epage%3D341%26doi%3D10.1016%2F0021-9991%2877%2990098-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref82"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref82'); return false;" data-citation="" class="refNumLink">82</a></strong><div class="NLM_citation" id="cit82"><span><span class="NLM_contrib-group">Darden, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">York, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pedersen, L.</span><span> </span><span class="NLM_article-title">Particle mesh Ewald: An N-log(N) method for Ewald sums in large systems</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">98</span><span class="NLM_x">, </span> <span class="NLM_fpage">10089</span><span class="NLM_x">–</span> <span class="NLM_lpage">10092</span><span class="refDoi"> DOI: 10.1063/1.464397</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1063%2F1.464397" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADyaK3sXks1Ohsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=1993&pages=10089-10092&author=T.+Dardenauthor=D.+Yorkauthor=L.+Pedersen&title=Particle+mesh+Ewald%3A+An+N-log%28N%29+method+for+Ewald+sums+in+large+systems&doi=10.1063%2F1.464397"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit82R"><div class="casContent"><span class="casTitleNuber">82</span><div class="casTitle"><span class="NLM_cas:atitle">Particle mesh Ewald: an N·log(N) method for Ewald sums in large systems</span></div><div class="casAuthors">Darden, Tom; York, Darrin; Pedersen, Lee</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">10089-92</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">An N·log(N) method for evaluating electrostatic energies and forces of large periodic systems is presented.  The method is based on interpolation of the reciprocal space Ewald sums and evaluation of the resulting convolution using fast Fourier transforms.  Timings and accuracies are presented for three large cryst. ionic systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDIJavf7AZPbVg90H21EOLACvtfcHk0ljjiN9Rq2q2jA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXks1Ohsr0%253D&md5=3c9f230bd01b7b714fd096d4d2e755f6</span></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.1063%2F1.464397&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.464397%26sid%3Dliteratum%253Aachs%26aulast%3DDarden%26aufirst%3DT.%26aulast%3DYork%26aufirst%3DD.%26aulast%3DPedersen%26aufirst%3DL.%26atitle%3DParticle%2520mesh%2520Ewald%253A%2520An%2520N-log%2528N%2529%2520method%2520for%2520Ewald%2520sums%2520in%2520large%2520systems%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1993%26volume%3D98%26spage%3D10089%26epage%3D10092%26doi%3D10.1063%2F1.464397" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref83"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref83'); return false;" data-citation="" class="refNumLink">83</a></strong><div class="NLM_citation" id="cit83"><span><span class="NLM_contrib-group">Roe, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheatham, T. E.</span><span> </span><span class="NLM_article-title">PTRAJ and CPPTRAJ: software for processing and analysis of molecular dynamics trajectory data</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">3084</span><span class="NLM_x">–</span> <span class="NLM_lpage">3095</span><span class="refDoi"> DOI: 10.1021/ct400341p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct400341p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BC3sXptFehtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=3084-3095&author=D.+R.+Roeauthor=T.+E.+Cheatham&title=PTRAJ+and+CPPTRAJ%3A+software+for+processing+and+analysis+of+molecular+dynamics+trajectory+data&doi=10.1021%2Fct400341p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit83R"><div class="casContent"><span class="casTitleNuber">83</span><div class="casTitle"><span class="NLM_cas:atitle">PTRAJ and CPPTRAJ: Software for Processing and Analysis of Molecular Dynamics Trajectory Data</span></div><div class="casAuthors">Roe, Daniel R.; Cheatham, Thomas E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3084-3095</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We describe PTRAJ and its successor CPPTRAJ, two complementary, portable, and freely available computer programs for the anal. and processing of time series of three-dimensional at. positions (i.e., coordinate trajectories) and the data therein derived.  Common tools include the ability to manipulate the data to convert among trajectory formats, process groups of trajectories generated with ensemble methods (e.g., replica exchange mol. dynamics), image with periodic boundary conditions, create av. structures, strip subsets of the system, and perform calcns. such as RMS fitting, measuring distances, B-factors, radii of gyration, radial distribution functions, and time correlations, among other actions and analyses.  Both the PTRAJ and CPPTRAJ programs and source code are freely available under the GNU General Public License version 3 and are currently distributed within the AmberTools 12 suite of support programs that make up part of the Amber package of computer programs (see http://ambermd.org).  This overview describes the general design, features, and history of these two programs, as well as algorithmic improvements and new features available in CPPTRAJ.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq97UsRfUy65bVg90H21EOLACvtfcHk0lhcDuDUVxSqMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXptFehtr8%253D&md5=6f1bee934f13f180bd7e1feb6b78036d</span></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.1021%2Fct400341p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct400341p%26sid%3Dliteratum%253Aachs%26aulast%3DRoe%26aufirst%3DD.%2BR.%26aulast%3DCheatham%26aufirst%3DT.%2BE.%26atitle%3DPTRAJ%2520and%2520CPPTRAJ%253A%2520software%2520for%2520processing%2520and%2520analysis%2520of%2520molecular%2520dynamics%2520trajectory%2520data%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2013%26volume%3D9%26spage%3D3084%26epage%3D3095%26doi%3D10.1021%2Fct400341p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref84"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref84'); return false;" data-citation="" class="refNumLink">84</a></strong><div class="NLM_citation" id="cit84"><span>The PyMOL Molecular Graphics System, <span class="NLM_edition">version 1.8</span>; <span class="NLM_publisher-name">Schrödinger</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=The+PyMOL+Molecular+Graphics+System%2C+version+1.8%3B+Schr%C3%B6dinger%3A+New+York%2C+2015."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit84&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520PyMOL%2520Molecular%2520Graphics%2520System%26pub%3DSchr%25C3%25B6dinger%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref85"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref85'); return false;" data-citation="" class="refNumLink">85</a></strong><div class="NLM_citation" id="cit85"><span><span class="NLM_contrib-group">Abel, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Farid, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berne, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friesner, R. A.</span><span> </span><span class="NLM_article-title">The role of the active site solvent in the thermodynamics of factor Xa-ligand binding</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">130</span><span class="NLM_x">, </span> <span class="NLM_fpage">2817</span><span class="NLM_x">–</span> <span class="NLM_lpage">2831</span><span class="refDoi"> DOI: 10.1021/ja0771033</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja0771033" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=130&publication_year=2008&pages=2817-2831&author=R.+Abelauthor=T.+Youngauthor=R.+Faridauthor=B.+J.+Berneauthor=R.+A.+Friesner&title=The+role+of+the+active+site+solvent+in+the+thermodynamics+of+factor+Xa-ligand+binding&doi=10.1021%2Fja0771033"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit85&amp;dbid=16384&amp;doi=10.1021%2Fja0771033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja0771033%26sid%3Dliteratum%253Aachs%26aulast%3DAbel%26aufirst%3DR.%26aulast%3DYoung%26aufirst%3DT.%26aulast%3DFarid%26aufirst%3DR.%26aulast%3DBerne%26aufirst%3DB.%2BJ.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26atitle%3DThe%2520role%2520of%2520the%2520active%2520site%2520solvent%2520in%2520the%2520thermodynamics%2520of%2520factor%2520Xa-ligand%2520binding%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2008%26volume%3D130%26spage%3D2817%26epage%3D2831%26doi%3D10.1021%2Fja0771033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref86"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref86'); return false;" data-citation="" class="refNumLink">86</a></strong><div class="NLM_citation" id="cit86"><span><span class="NLM_contrib-group">Nguyen, C. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kurtzman Young, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilson, M. K.</span><span> </span><span class="NLM_article-title">Grid inhomogeneous solvation theory: Hydration structure and thermodynamics of the miniature receptor cucurbit[7]uril</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">137</span><span class="NLM_x">, </span> <span class="NLM_fpage">044101</span><span class="refDoi"> DOI: 10.1063/1.4733951</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1063%2F1.4733951" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=22852591" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVOgurzJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=137&publication_year=2012&pages=044101&author=C.+N.+Nguyenauthor=T.+Kurtzman+Youngauthor=M.+K.+Gilson&title=Grid+inhomogeneous+solvation+theory%3A+Hydration+structure+and+thermodynamics+of+the+miniature+receptor+cucurbit%5B7%5Duril&doi=10.1063%2F1.4733951"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit86R"><div class="casContent"><span class="casTitleNuber">86</span><div class="casTitle"><span class="NLM_cas:atitle">Grid inhomogeneous solvation theory: Hydration structure and thermodynamics of the miniature receptor cucurbit[7]uril</span></div><div class="casAuthors">Nguyen, Crystal N.; Kurtzman Young, Tom; Gilson, Michael K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">137</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">044101/1-044101/17</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    
            (<span class="NLM_cas:orgname">American Institute of Physics</span>)
        </div><div class="casAbstract">The displacement of perturbed water upon binding is believed to play a crit. role in the thermodn. of biomol. recognition, but it is nontrivial to unambiguously define and answer questions about this process.  We address this issue by introducing grid inhomogeneous solvation theory (GIST), which discretizes the equations of inhomogeneous solvation theory (IST) onto a three-dimensional grid situated in the region of interest around a solute mol. or complex.  Snapshots from explicit solvent simulations are used to est. localized solvation entropies, energies, and free energies assocd. with the grid boxes, or voxels, and properly summing these thermodn. quantities over voxels yields information about hydration thermodn.  GIST thus provides a smoothly varying representation of water properties as a function of position, rather than focusing on hydration sites where solvent is present at high d.  It therefore accounts for full or partial displacement of water from sites that are highly occupied by water, as well as for partly occupied and water-depleted regions around the solute.  GIST can also provide a well-defined est. of the solvation free energy and therefore enables a rigorous end-states anal. of binding.  For example, one may not only use a first GIST calcn. to project the thermodn. consequences of displacing water from the surface of a receptor by a ligand, but also account, in a second GIST calcn., for the thermodn. of subsequent solvent reorganization around the bound complex.  In the present study, a first GIST anal. of the mol. host cucurbit7uril is found to yield a rich picture of hydration structure and thermodn. in and around this miniature receptor.  One of the most striking results is the observation of a toroidal region of high water d. at the center of the host's nonpolar cavity.  Despite its high d., the water in this toroidal region is disfavored energetically and entropically, and hence may contribute to the known ability of this small receptor to bind guest mols. with unusually high affinities.  Interestingly, the toroidal region of high water d. persists even when all partial charges of the receptor are set to zero.  Thus, localized regions of high solvent d. can be generated in a binding site without strong, attractive solute-solvent interactions.  (c) 2012 American Institute of Physics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrkKZ05RaZJbbVg90H21EOLACvtfcHk0lhcDuDUVxSqMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVOgurzJ&md5=d8eceb021e903950784786a656fdfb40</span></div><a href="/servlet/linkout?suffix=cit86&amp;dbid=16384&amp;doi=10.1063%2F1.4733951&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.4733951%26sid%3Dliteratum%253Aachs%26aulast%3DNguyen%26aufirst%3DC.%2BN.%26aulast%3DKurtzman%2BYoung%26aufirst%3DT.%26aulast%3DGilson%26aufirst%3DM.%2BK.%26atitle%3DGrid%2520inhomogeneous%2520solvation%2520theory%253A%2520Hydration%2520structure%2520and%2520thermodynamics%2520of%2520the%2520miniature%2520receptor%2520cucurbit%255B7%255Duril%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D2012%26volume%3D137%26spage%3D044101%26doi%3D10.1063%2F1.4733951" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref87"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref87'); return false;" data-citation="" class="refNumLink">87</a></strong><div class="NLM_citation" id="cit87"><span><span class="NLM_contrib-group">Hu, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lill, M. A.</span><span> </span><span class="NLM_article-title">WATsite: Hydration site prediction program with PyMOL interface</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">35</span><span class="NLM_x">, </span> <span class="NLM_fpage">1255</span><span class="NLM_x">–</span> <span class="NLM_lpage">1260</span><span class="refDoi"> DOI: 10.1002/jcc.23616</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1002%2Fjcc.23616" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=24752524" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmslant7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2014&pages=1255-1260&author=B.+Huauthor=M.+A.+Lill&title=WATsite%3A+Hydration+site+prediction+program+with+PyMOL+interface&doi=10.1002%2Fjcc.23616"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit87R"><div class="casContent"><span class="casTitleNuber">87</span><div class="casTitle"><span class="NLM_cas:atitle">WATsite: Hydration site prediction program with PyMOL interface</span></div><div class="casAuthors">Hu, Bingjie; Lill, Markus A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1255-1260</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Water mols. that mediate protein-ligand interactions or are released from the binding site on ligand binding can contribute both enthalpically and entropically to the free energy of ligand binding.  To elucidate the thermodn. profile of individual water mols. and their potential contribution to ligand binding, a hydration site anal. program WATsite was developed together with an easy-to-use graphical user interface based on PyMOL.  WATsite identifies hydration sites from a mol. dynamics simulation trajectory with explicit water mols.  The free energy profile of each hydration site is estd. by computing the enthalpy and entropy of the water mol. occupying a hydration site throughout the simulation.  The results of the hydration site anal. can be displayed in PyMOL.  A key feature of WATsite is that it is able to est. the protein desolvation free energy for any user specified ligand.  The WATsite program and its PyMOL plugin are available free of charge from http://people.pnhs.purdue.edu/∼mlill/software.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZ0FIFaoOGZ7Vg90H21EOLACvtfcHk0lhcDuDUVxSqMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmslant7Y%253D&md5=b7f10d1a30efd3bc5c69bbfc03ee947e</span></div><a href="/servlet/linkout?suffix=cit87&amp;dbid=16384&amp;doi=10.1002%2Fjcc.23616&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.23616%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DB.%26aulast%3DLill%26aufirst%3DM.%2BA.%26atitle%3DWATsite%253A%2520Hydration%2520site%2520prediction%2520program%2520with%2520PyMOL%2520interface%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2014%26volume%3D35%26spage%3D1255%26epage%3D1260%26doi%3D10.1002%2Fjcc.23616" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref88"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref88'); return false;" data-citation="" class="refNumLink">88</a></strong><div class="NLM_citation" id="cit88"><span><span class="NLM_contrib-group">Grant, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pickup, B. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nicholls, A.</span><span> </span><span class="NLM_article-title">A smooth permittivity function for Poisson–Boltzmann solvation methods</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">608</span><span class="NLM_x">–</span> <span class="NLM_lpage">640</span><span class="refDoi"> DOI: 10.1002/jcc.1032</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1002%2Fjcc.1032" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BD3MXisFaltro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2001&pages=608-640&author=J.+A.+Grantauthor=B.+T.+Pickupauthor=A.+Nicholls&title=A+smooth+permittivity+function+for+Poisson%E2%80%93Boltzmann+solvation+methods&doi=10.1002%2Fjcc.1032"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit88R"><div class="casContent"><span class="casTitleNuber">88</span><div class="casTitle"><span class="NLM_cas:atitle">A smooth permittivity function for poisson-boltzmann solvation methods</span></div><div class="casAuthors">Grant, J. Andrew; Pickup, Barry T.; Nicholls, Anthony</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">608-640</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">This work introduces a continuous smooth permittivity function into Poisson-Boltzmann techniques for continuum approaches to modeling the solvation of small mols. and proteins.  The permittivity function is derived using a Gaussian method to describe vol. exclusion.  The new method allows a rigorous detn. of solvent forces within a grid-based technol.  The generality of approach is demonstrated by considering a range of applications for small mols. and macromols.  We also present a very complete statistical anal. of grid errors, and show that the accuracy of our Gaussian-based method is improved over std. techniques.  The method has been implemented in a new code called ZAP, which is freely available to academic institutions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNBKMD0CGO77Vg90H21EOLACvtfcHk0liYwcb9MAH0sQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXisFaltro%253D&md5=5221cbe055fe89308d2abdc629e76f20</span></div><a href="/servlet/linkout?suffix=cit88&amp;dbid=16384&amp;doi=10.1002%2Fjcc.1032&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.1032%26sid%3Dliteratum%253Aachs%26aulast%3DGrant%26aufirst%3DJ.%2BA.%26aulast%3DPickup%26aufirst%3DB.%2BT.%26aulast%3DNicholls%26aufirst%3DA.%26atitle%3DA%2520smooth%2520permittivity%2520function%2520for%2520Poisson%25E2%2580%2593Boltzmann%2520solvation%2520methods%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2001%26volume%3D22%26spage%3D608%26epage%3D640%26doi%3D10.1002%2Fjcc.1032" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref89"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref89'); return false;" data-citation="" class="refNumLink">89</a></strong><div class="NLM_citation" id="cit89"><span><span class="NLM_contrib-group">Huggins, D. J.</span><span> </span><span class="NLM_article-title">Application of inhomogeneous fluid solvation theory to model the distribution and thermodynamics of water molecules around biomolecules</span> <span class="citation_source-journal">Phys. Chem. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">15106</span><span class="NLM_x">–</span> <span class="NLM_lpage">15117</span><span class="refDoi"> DOI: 10.1039/c2cp42631e</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1039%2Fc2cp42631e" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=23037989" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFersrbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2012&pages=15106-15117&author=D.+J.+Huggins&title=Application+of+inhomogeneous+fluid+solvation+theory+to+model+the+distribution+and+thermodynamics+of+water+molecules+around+biomolecules&doi=10.1039%2Fc2cp42631e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit89R"><div class="casContent"><span class="casTitleNuber">89</span><div class="casTitle"><span class="NLM_cas:atitle">Application of inhomogeneous fluid solvation theory to model the distribution and thermodynamics of water molecules around biomolecules</span></div><div class="casAuthors">Huggins, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Physical Chemistry Chemical Physics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">43</span>),
    <span class="NLM_cas:pages">15106-15117</span>CODEN:
                <span class="NLM_cas:coden">PPCPFQ</span>;
        ISSN:<span class="NLM_cas:issn">1463-9076</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The structures of biomols. and the strengths of assocn. between them depend critically on interactions with water mols.  Thus, understanding these interactions is a prerequisite for understanding the structure and function of all biomols.  Inhomogeneous fluid solvation theory provides a framework to derive thermodn. properties of individual water mols. from a statistical mech. anal.  In this work, two biomols. are analyzed to probe the distribution and thermodn. of surrounding water mols.  The great majority of hydration sites are predicted to contribute favorably to the total free energy with respect to bulk water, though hydration sites close to non-polar regions of the solute do not contribute significantly.  Anal. of a biomol. with a pos. and neg. charged functional group predicts that a charged species perturbs the free energy of water mols. to a distance of approx. 6.0 Å.  Interestingly, short simulations are found to provide converged predictions if samples are taken with sufficient frequency, a finding that has the potential to significantly reduce the required computational cost of such anal.  In addn., the predicted thermodn. properties of hydration sites with the potential for direct hydrogen bonding interactions are found to disagree significantly for two different water models.  This study provides important information on how inhomogeneous fluid solvation theory can be employed to understand the structures and intermol. interactions of biomols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqL-I9p9H8TSLVg90H21EOLACvtfcHk0liYwcb9MAH0sQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFersrbN&md5=828eb1eee170fba0526fff24d87b0fbd</span></div><a href="/servlet/linkout?suffix=cit89&amp;dbid=16384&amp;doi=10.1039%2Fc2cp42631e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc2cp42631e%26sid%3Dliteratum%253Aachs%26aulast%3DHuggins%26aufirst%3DD.%2BJ.%26atitle%3DApplication%2520of%2520inhomogeneous%2520fluid%2520solvation%2520theory%2520to%2520model%2520the%2520distribution%2520and%2520thermodynamics%2520of%2520water%2520molecules%2520around%2520biomolecules%26jtitle%3DPhys.%2520Chem.%2520Chem.%2520Phys.%26date%3D2012%26volume%3D14%26spage%3D15106%26epage%3D15117%26doi%3D10.1039%2Fc2cp42631e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref90"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref90'); return false;" data-citation="" class="refNumLink">90</a></strong><div class="NLM_citation" id="cit90"><span><span class="NLM_contrib-group">Gerogiokas, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Calabro, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Henchman, R. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Southey, M. W. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Law, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Michel, J.</span><span> </span><span class="NLM_article-title">Prediction of Small Molecule Hydration Thermodynamics with Grid Cell Theory</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">35</span><span class="NLM_x">–</span> <span class="NLM_lpage">48</span><span class="refDoi"> DOI: 10.1021/ct400783h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct400783h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvV2rsLzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=35-48&author=G.+Gerogiokasauthor=G.+Calabroauthor=R.+H.+Henchmanauthor=M.+W.+Y.+Southeyauthor=R.+J.+Lawauthor=J.+Michel&title=Prediction+of+Small+Molecule+Hydration+Thermodynamics+with+Grid+Cell+Theory&doi=10.1021%2Fct400783h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit90R"><div class="casContent"><span class="casTitleNuber">90</span><div class="casTitle"><span class="NLM_cas:atitle">Prediction of Small Molecule Hydration Thermodynamics with Grid Cell Theory</span></div><div class="casAuthors">Gerogiokas, Georgios; Calabro, Gaetano; Henchman, Richard H.; Southey, Michelle W. Y.; Law, Richard J.; Michel, Julien</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">35-48</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">An efficient methodol. has been developed to quantify water energetics by anal. of explicit solvent mol. simulations of org. and biomol. systems.  The approach, grid cell theory (GCT), relies on a discretization of the cell theory methodol. on a three-dimensional grid to spatially resolve the d., enthalpy, and entropy of water mols. in the vicinity of solute(s) of interest.  Entropies of hydration are found to converge more efficiently than enthalpies of hydration.  GCT predictions of free energies of hydration on a data set of small mols. are strongly correlated with thermodn. integration predictions.  Agreement with the expt. is comparable for both approaches.  A key advantage of GCT is its ability to provide from a single simulation insightful graphical analyses of spatially resolved components of the enthalpies and entropies of hydration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqOK8_kdK9hLLVg90H21EOLACvtfcHk0liYwcb9MAH0sQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvV2rsLzE&md5=2be9e31aa30232e9d60e73039177a634</span></div><a href="/servlet/linkout?suffix=cit90&amp;dbid=16384&amp;doi=10.1021%2Fct400783h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct400783h%26sid%3Dliteratum%253Aachs%26aulast%3DGerogiokas%26aufirst%3DG.%26aulast%3DCalabro%26aufirst%3DG.%26aulast%3DHenchman%26aufirst%3DR.%2BH.%26aulast%3DSouthey%26aufirst%3DM.%2BW.%2BY.%26aulast%3DLaw%26aufirst%3DR.%2BJ.%26aulast%3DMichel%26aufirst%3DJ.%26atitle%3DPrediction%2520of%2520Small%2520Molecule%2520Hydration%2520Thermodynamics%2520with%2520Grid%2520Cell%2520Theory%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2014%26volume%3D10%26spage%3D35%26epage%3D48%26doi%3D10.1021%2Fct400783h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref91"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref91'); return false;" data-citation="" class="refNumLink">91</a></strong><div class="NLM_citation" id="cit91"><span><span class="NLM_contrib-group">Baron, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Setny, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Andrew McCammon, J.</span><span> </span><span class="NLM_article-title">Water in Cavity–Ligand Recognition</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">132</span><span class="NLM_x">, </span> <span class="NLM_fpage">12091</span><span class="NLM_x">–</span> <span class="NLM_lpage">12097</span><span class="refDoi"> DOI: 10.1021/ja1050082</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja1050082" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BC3cXpvFKisb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2010&pages=12091-12097&author=R.+Baronauthor=P.+Setnyauthor=J.+Andrew+McCammon&title=Water+in+Cavity%E2%80%93Ligand+Recognition&doi=10.1021%2Fja1050082"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit91R"><div class="casContent"><span class="casTitleNuber">91</span><div class="casTitle"><span class="NLM_cas:atitle">Water in Cavity-Ligand Recognition</span></div><div class="casAuthors">Baron, Riccardo; Setny, Piotr; Andrew McCammon, J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">132</span>
        (<span class="NLM_cas:issue">34</span>),
    <span class="NLM_cas:pages">12091-12097</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We use explicit solvent mol. dynamics simulations to est. free energy, enthalpy, and entropy changes along the cavity-ligand assocn. coordinate for a set of seven model systems with varying physicochem. properties.  Owing to the simplicity of the considered systems we can directly investigate the role of water thermodn. in mol. recognition.  A broad range of thermodn. signatures is found in which water (rather than cavity or ligand) enthalpic or entropic contributions appear to drive cavity-ligand binding or rejection.  The unprecedented, nanoscale picture of hydration thermodn. can help the interpretation and design of protein-ligand binding expts.  Our study opens appealing perspectives to tackle the challenge of solvent entropy estn. in complex systems and for improving mol. simulation models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrh8IROwcNdbLVg90H21EOLACvtfcHk0lj1hw8cSlEJaQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXpvFKisb0%253D&md5=38100ba43b18ce488a768ce7da64b128</span></div><a href="/servlet/linkout?suffix=cit91&amp;dbid=16384&amp;doi=10.1021%2Fja1050082&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja1050082%26sid%3Dliteratum%253Aachs%26aulast%3DBaron%26aufirst%3DR.%26aulast%3DSetny%26aufirst%3DP.%26aulast%3DAndrew%2BMcCammon%26aufirst%3DJ.%26atitle%3DWater%2520in%2520Cavity%25E2%2580%2593Ligand%2520Recognition%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2010%26volume%3D132%26spage%3D12091%26epage%3D12097%26doi%3D10.1021%2Fja1050082" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68'],'ref69':['cit69'],'ref70':['cit70'],'ref71':['cit71'],'ref72':['cit72'],'ref73':['cit73'],'ref74':['cit74'],'ref75':['cit75'],'ref76':['cit76'],'ref77':['cit77'],'ref78':['cit78'],'ref79':['cit79'],'ref80':['cit80'],'ref81':['cit81'],'ref82':['cit82'],'ref83':['cit83'],'ref84':['cit84'],'ref85':['cit85'],'ref86':['cit86'],'ref87':['cit87'],'ref88':['cit88'],'ref89':['cit89'],'ref90':['cit90'],'ref91':['cit91']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 12 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Thilo Focken, Kristen Burford, Michael E. Grimwood, Alla Zenova, Jean-Christophe Andrez, Wei Gong, Michael Wilson, Matt Taron, Shannon Decker, Verner Lofstrand, Sultan Chowdhury, Noah Shuart, Sophia Lin, Samuel J. Goodchild, Clint Young, Maegan Soriano, Parisa K. Tari, Matthew Waldbrook, Karen Nelkenbrecher, Rainbow Kwan, Andrea Lindgren, Gina de Boer, Stephanie Lee, Luis Sojo, Robert J. DeVita, Charles J. Cohen, Steven S. Wesolowski, J. P. Johnson, Jr., Christoph M. Dehnhardt, <span class="NLM_string-name hlFld-ContribAuthor">James R. Empfield</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of CNS-Penetrant Aryl Sulfonamides as Isoform-Selective NaV1.6 Inhibitors with Efficacy in Mouse Models of Epilepsy. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (21)
                                     , 9618-9641. <a href="https://doi.org/10.1021/acs.jmedchem.9b01032" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01032</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01032&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b01032%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DIdentification%252Bof%252BCNS-Penetrant%252BAryl%252BSulfonamides%252Bas%252BIsoform-Selective%252BNaV1.6%252BInhibitors%252Bwith%252BEfficacy%252Bin%252BMouse%252BModels%252Bof%252BEpilepsy%26aulast%3DFocken%26aufirst%3DThilo%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D28062019%26date%3D03102019%26date%3D17092019%26volume%3D62%26issue%3D21%26spage%3D9618%26epage%3D9641" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Xiaojing Yuan, Stefano Raniolo, Vittorio Limongelli, <span class="NLM_string-name hlFld-ContribAuthor">Yechun Xu</span>. </span><span class="cited-content_cbyCitation_article-title">The Molecular Mechanism Underlying Ligand Binding to the Membrane-Embedded Site of a G-Protein-Coupled Receptor. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Theory and Computation</span><span> <strong>2018,</strong> <em>14 </em>
                                    (5)
                                     , 2761-2770. <a href="https://doi.org/10.1021/acs.jctc.8b00046" title="DOI URL">https://doi.org/10.1021/acs.jctc.8b00046</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jctc.8b00046&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jctc.8b00046%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Theory%2520and%2520Computation%26atitle%3DThe%252BMolecular%252BMechanism%252BUnderlying%252BLigand%252BBinding%252Bto%252Bthe%252BMembrane-Embedded%252BSite%252Bof%252Ba%252BG-Protein-Coupled%252BReceptor%26aulast%3DYuan%26aufirst%3DXiaojing%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D17012018%26date%3D27042018%26date%3D16042018%26volume%3D14%26issue%3D5%26spage%3D2761%26epage%3D2770" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Si  Yan</span>, <span class="hlFld-ContribAuthor ">Duncan E.  Shaw</span>, <span class="hlFld-ContribAuthor ">Linhong  Yang</span>, <span class="hlFld-ContribAuthor ">David A.  Sandham</span>, <span class="hlFld-ContribAuthor ">Mark P.  Healy</span>, <span class="hlFld-ContribAuthor ">John  Reilly</span>, and <span class="hlFld-ContribAuthor ">Bing  Wang</span>  . </span><span class="cited-content_cbyCitation_article-title">Interactions between β2-Adrenoceptor Ligands and Membrane: Atomic-Level Insights from Magic-Angle Spinning NMR. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2017,</strong> <em>60 </em>
                                    (16)
                                     , 6867-6879. <a href="https://doi.org/10.1021/acs.jmedchem.7b00205" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b00205</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b00205&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.7b00205%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DInteractions%252Bbetween%252B%2525CE%2525B22-Adrenoceptor%252BLigands%252Band%252BMembrane%25253A%252BAtomic-Level%252BInsights%252Bfrom%252BMagic-Angle%252BSpinning%252BNMR%26aulast%3DYan%26aufirst%3DSi%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D15022017%26date%3D04082017%26date%3D24082017%26date%3D13072017%26volume%3D60%26issue%3D16%26spage%3D6867%26epage%3D6879" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Agustín  Bruzzese</span>, <span class="hlFld-ContribAuthor ">James A. R.  Dalton</span>, <span class="hlFld-ContribAuthor ">Jesús  Giraldo</span>, . </span><span class="cited-content_cbyCitation_article-title">Insights into adenosine A2A receptor activation through cooperative modulation of agonist and allosteric lipid interactions. </span><span class="cited-content_cbyCitation_journal-name">PLOS Computational Biology</span><span> <strong>2020,</strong> <em>16 </em>
                                    (4)
                                     , e1007818. <a href="https://doi.org/10.1371/journal.pcbi.1007818" title="DOI URL">https://doi.org/10.1371/journal.pcbi.1007818</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1371/journal.pcbi.1007818&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1371%2Fjournal.pcbi.1007818%26sid%3Dliteratum%253Aachs%26jtitle%3DPLOS%2520Computational%2520Biology%26atitle%3DInsights%252Binto%252Badenosine%252BA2A%2525C2%2525A0receptor%252Bactivation%252Bthrough%252Bcooperative%252Bmodulation%252Bof%252Bagonist%252Band%252Ballosteric%252Blipid%252Binteractions%26aulast%3DBruzzese%26aufirst%3DAgust%25C3%25ADn%26date%3D2020%26date%3D2020%26volume%3D16%26issue%3D4%26spage%3De1007818" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Christopher T.  Szlenk</span>, <span class="hlFld-ContribAuthor ">Jeevan B.  GC</span>, <span class="hlFld-ContribAuthor ">Senthil  Natesan</span>. </span><span class="cited-content_cbyCitation_article-title">Does the Lipid Bilayer Orchestrate Access and Binding of Ligands to Transmembrane Orthosteric/Allosteric Sites of G Protein-Coupled Receptors?. </span><span class="cited-content_cbyCitation_journal-name">Molecular Pharmacology</span><span> <strong>2019,</strong> <em>96 </em>
                                    (5)
                                     , 527-541. <a href="https://doi.org/10.1124/mol.118.115113" title="DOI URL">https://doi.org/10.1124/mol.118.115113</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1124/mol.118.115113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1124%2Fmol.118.115113%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Pharmacology%26atitle%3DDoes%252Bthe%252BLipid%252BBilayer%252BOrchestrate%252BAccess%252Band%252BBinding%252Bof%252BLigands%252Bto%252BTransmembrane%252BOrthosteric%25252FAllosteric%252BSites%252Bof%252BG%252BProtein-Coupled%252BReceptors%25253F%26aulast%3DSzlenk%26aufirst%3DChristopher%2BT.%26date%3D2019%26date%3D2019%26volume%3D96%26issue%3D5%26spage%3D527%26epage%3D541" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Donglu  Zhang</span>, <span class="hlFld-ContribAuthor ">Cornelis E.C.A.  Hop</span>, <span class="hlFld-ContribAuthor ">Gabriela  Patilea-Vrana</span>, <span class="hlFld-ContribAuthor ">Gautham  Gampa</span>, <span class="hlFld-ContribAuthor ">Herana Kamal  Seneviratne</span>, <span class="hlFld-ContribAuthor ">Jashvant D.  Unadkat</span>, <span class="hlFld-ContribAuthor ">Jane R.  Kenny</span>, <span class="hlFld-ContribAuthor ">Karthik  Nagapudi</span>, <span class="hlFld-ContribAuthor ">Li  Di</span>, <span class="hlFld-ContribAuthor ">Lian  Zhou</span>, <span class="hlFld-ContribAuthor ">Mark  Zak</span>, <span class="hlFld-ContribAuthor ">Matthew R.  Wright</span>, <span class="hlFld-ContribAuthor ">Namandjé N.  Bumpus</span>, <span class="hlFld-ContribAuthor ">Richard  Zang</span>, <span class="hlFld-ContribAuthor ">Xingrong  Liu</span>, <span class="hlFld-ContribAuthor ">Yurong  Lai</span>, <span class="hlFld-ContribAuthor ">S. Cyrus  Khojasteh</span>. </span><span class="cited-content_cbyCitation_article-title">Drug Concentration Asymmetry in Tissues and Plasma for Small Molecule–Related Therapeutic Modalities. </span><span class="cited-content_cbyCitation_journal-name">Drug Metabolism and Disposition</span><span> <strong>2019,</strong> <em>47 </em>
                                    (10)
                                     , 1122-1135. <a href="https://doi.org/10.1124/dmd.119.086744" title="DOI URL">https://doi.org/10.1124/dmd.119.086744</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1124/dmd.119.086744&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1124%2Fdmd.119.086744%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Metabolism%2520and%2520Disposition%26atitle%3DDrug%252BConcentration%252BAsymmetry%252Bin%252BTissues%252Band%252BPlasma%252Bfor%252BSmall%252BMolecule%2525E2%252580%252593Related%252BTherapeutic%252BModalities%26aulast%3DZhang%26aufirst%3DDonglu%26date%3D2019%26date%3D2019%26volume%3D47%26issue%3D10%26spage%3D1122%26epage%3D1135" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Marvin A.  Soriano-Ursúa</span>, <span class="hlFld-ContribAuthor ">Martiniano  Bello</span>, <span class="hlFld-ContribAuthor ">Christian F.  Hernández-Martínez</span>, <span class="hlFld-ContribAuthor ">Iván  Santillán-Torres</span>, <span class="hlFld-ContribAuthor ">Ruth  Guerrero-Ramírez</span>, <span class="hlFld-ContribAuthor ">José  Correa-Basurto</span>, <span class="hlFld-ContribAuthor ">José-Antonio  Arias-Montaño</span>, <span class="hlFld-ContribAuthor ">José G.  Trujillo-Ferrara</span>. </span><span class="cited-content_cbyCitation_article-title">Cell-based assays and molecular dynamics analysis of a boron-containing agonist with different profiles of binding to human and guinea pig beta2 adrenoceptors. </span><span class="cited-content_cbyCitation_journal-name">European Biophysics Journal</span><span> <strong>2019,</strong> <em>48 </em>
                                    (1)
                                     , 83-97. <a href="https://doi.org/10.1007/s00249-018-1336-9" title="DOI URL">https://doi.org/10.1007/s00249-018-1336-9</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s00249-018-1336-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs00249-018-1336-9%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Biophysics%2520Journal%26atitle%3DCell-based%252Bassays%252Band%252Bmolecular%252Bdynamics%252Banalysis%252Bof%252Ba%252Bboron-containing%252Bagonist%252Bwith%252Bdifferent%252Bprofiles%252Bof%252Bbinding%252Bto%252Bhuman%252Band%252Bguinea%252Bpig%252Bbeta2%252Badrenoceptors%26aulast%3DSoriano-Urs%25C3%25BAa%26aufirst%3DMarvin%2BA.%26date%3D2019%26date%3D2018%26volume%3D48%26issue%3D1%26spage%3D83%26epage%3D97" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Agustín  Bruzzese</span>, <span class="hlFld-ContribAuthor ">Carles  Gil</span>, <span class="hlFld-ContribAuthor ">James A. R.  Dalton</span>, <span class="hlFld-ContribAuthor ">Jesús  Giraldo</span>. </span><span class="cited-content_cbyCitation_article-title">Structural insights into positive and negative allosteric regulation of a G protein-coupled receptor through protein-lipid interactions. </span><span class="cited-content_cbyCitation_journal-name">Scientific Reports</span><span> <strong>2018,</strong> <em>8 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41598-018-22735-6" title="DOI URL">https://doi.org/10.1038/s41598-018-22735-6</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41598-018-22735-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41598-018-22735-6%26sid%3Dliteratum%253Aachs%26jtitle%3DScientific%2520Reports%26atitle%3DStructural%252Binsights%252Binto%252Bpositive%252Band%252Bnegative%252Ballosteric%252Bregulation%252Bof%252Ba%252BG%252Bprotein-coupled%252Breceptor%252Bthrough%252Bprotein-lipid%252Binteractions%26aulast%3DBruzzese%26aufirst%3DAgust%25C3%25ADn%26date%3D2018%26date%3D2018%26volume%3D8%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiaojing  Yuan</span>, <span class="hlFld-ContribAuthor ">Yechun  Xu</span>. </span><span class="cited-content_cbyCitation_article-title">Recent Trends and Applications of Molecular Modeling in GPCR–Ligand Recognition and Structure-Based Drug Design. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2018,</strong> <em>19 </em>
                                    (7)
                                     , 2105. <a href="https://doi.org/10.3390/ijms19072105" title="DOI URL">https://doi.org/10.3390/ijms19072105</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms19072105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms19072105%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DRecent%252BTrends%252Band%252BApplications%252Bof%252BMolecular%252BModeling%252Bin%252BGPCR%2525E2%252580%252593Ligand%252BRecognition%252Band%252BStructure-Based%252BDrug%252BDesign%26aulast%3DYuan%26aufirst%3DXiaojing%26date%3D2018%26date%3D2018%26volume%3D19%26issue%3D7%26spage%3D2105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Naushad  Velgy</span>, <span class="hlFld-ContribAuthor ">George  Hedger</span>, <span class="hlFld-ContribAuthor ">Philip C.  Biggin</span>. </span><span class="cited-content_cbyCitation_article-title">GPCRs: What Can We Learn from Molecular Dynamics Simulations?. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2018,</strong>,, 133-158. <a href="https://doi.org/10.1007/978-1-4939-7465-8_6" title="DOI URL">https://doi.org/10.1007/978-1-4939-7465-8_6</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-1-4939-7465-8_6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-1-4939-7465-8_6%26sid%3Dliteratum%253Aachs%26atitle%3DGPCRs%25253A%252BWhat%252BCan%252BWe%252BLearn%252Bfrom%252BMolecular%252BDynamics%252BSimulations%25253F%26aulast%3DVelgy%26aufirst%3DNaushad%26date%3D2018%26date%3D2017%26spage%3D133%26epage%3D158%26pub%3DSpringer%2520New%2520York%26atitle%3DComputational%252BMethods%252Bfor%252BGPCR%252BDrug%252BDiscovery%26aulast%3DHeifetz%26aufirst%3DAlexander%26date%3D2018%26volume%3D1705" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Georges  Vauquelin</span>. </span><span class="cited-content_cbyCitation_article-title">Link between a high
              k
              on
              for drug binding and a fast clinical action: to be or not to be?. </span><span class="cited-content_cbyCitation_journal-name">MedChemComm</span><span> <strong>2018,</strong> <em>9 </em>
                                    (9)
                                     , 1426-1438. <a href="https://doi.org/10.1039/C8MD00296G" title="DOI URL">https://doi.org/10.1039/C8MD00296G</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C8MD00296G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC8MD00296G%26sid%3Dliteratum%253Aachs%26jtitle%3DMedChemComm%26atitle%3DLink%252Bbetween%252Ba%252Bhigh%252Bk%252Bon%252Bfor%252Bdrug%252Bbinding%252Band%252Ba%252Bfast%252Bclinical%252Baction%25253A%252Bto%252Bbe%252Bor%252Bnot%252Bto%252Bbe%25253F%26aulast%3DVauquelin%26aufirst%3DGeorges%26date%3D2018%26date%3D2018%26volume%3D9%26issue%3D9%26spage%3D1426%26epage%3D1438" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shawn  Johnstone</span>, <span class="hlFld-ContribAuthor ">Jeffrey S.  Albert</span>. </span><span class="cited-content_cbyCitation_article-title">Pharmacological property optimization for allosteric ligands: A medicinal chemistry perspective. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2017,</strong> <em>27 </em>
                                    (11)
                                     , 2239-2258. <a href="https://doi.org/10.1016/j.bmcl.2017.03.084" title="DOI URL">https://doi.org/10.1016/j.bmcl.2017.03.084</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2017.03.084&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2017.03.084%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DPharmacological%252Bproperty%252Boptimization%252Bfor%252Ballosteric%252Bligands%25253A%252BA%252Bmedicinal%252Bchemistry%252Bperspective%26aulast%3DJohnstone%26aufirst%3DShawn%26date%3D2017%26volume%3D27%26issue%3D11%26spage%3D2239%26epage%3D2258" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-12/acs.jmedchem.6b00358/20160617/images/medium/jm-2016-003586_0007.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-12/acs.jmedchem.6b00358/20160617/images/large/jm-2016-003586_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00358&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-12/acs.jmedchem.6b00358/20160617/images/medium/jm-2016-003586_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-12/acs.jmedchem.6b00358/20160617/images/large/jm-2016-003586_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Membrane receptor binding model. Observed binding (<i>K</i><sub>d</sub><sup>obs</sup>) results from both the affinity of the ligand to associate with the membrane (approximated by measured <i>K</i><sub>IAM</sub>) and “true”/intrinsic affinity for the protein (<i>K</i><sub>d</sub><sup>int</sup>).  On the basis of the microkinetic model, the ligand association with the membrane may create higher “local” concentration in the target compartment (dark-blue region), <i>L</i><sub>assoc</sub>, which can result in higher receptor occupancy than would have been expected from the bulk ligand concentration <i>L</i><sub>bulk</sub>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-12/acs.jmedchem.6b00358/20160617/images/large/jm-2016-003586_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00358&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-12/acs.jmedchem.6b00358/20160617/images/medium/jm-2016-003586_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-12/acs.jmedchem.6b00358/20160617/images/large/jm-2016-003586_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Structures, <i>K</i><sub>d</sub><sup>obs</sup>, corrected <i>K</i><sub>d</sub><sup>int</sup>, and <i>K</i><sub>IAM</sub> values for structurally related groups of β2-AR ligands: salmeterol series (agonists isoprenaline, salbutamol, salmeterol), atenolol series (antagonists atenolol, metoprolol, bisoprolol), and cyanopindolol series (antagonists cyanopindolol, CGP12177, nadolol, propranolol, ICI-118,551). Data (<i>K</i><sub>d</sub><sup>obs</sup>, <i>K</i><sub>IAM</sub>) from Sykes et al.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> *, racemic mixture; #, racemic mixture of the 1-diastereoisomer; ∧, mixture of four diastereoisomers with a cis-relationship in the tetralin ring. A full list of binding and kinetic data is available in the Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.6b00358/suppl_file/jm6b00358_si_001.pdf" class="ext-link">Table S1</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-12/acs.jmedchem.6b00358/20160617/images/large/jm-2016-003586_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00358&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-12/acs.jmedchem.6b00358/20160617/images/medium/jm-2016-003586_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-12/acs.jmedchem.6b00358/20160617/images/large/jm-2016-003586_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Salmeterol series. The modeled binding modes of agonist isoprenaline, salbutamol, and salmeterol. All three make similar hydrogen bonds to hydrophilic residues at the rear of the binding pocket, either directly or via a bridging water. Salmeterol is able to form an additional hydrogen bond to Phe193 (circled in red for clarity). High occupancy solvation sites are represented by spheres whose size correlates with the magnitude of their occupancy with respect to bulk water. Only those with occupancy higher than 2× bulk are shown. In turn, the coloring of each water site correlates with its relative hydrogen:oxygen occupancy. Namely, each sphere is colored in red–white–blue spectrum, with intense red corresponding to oxygen-only occupancy, deep-blue to hydrogen only occupancy, and white to a 2:1 hydrogen:oxygen (i.e., water-like) occupancy. Additional information concerning visualization of WATMD structures can be found in ref <a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">26</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-12/acs.jmedchem.6b00358/20160617/images/large/jm-2016-003586_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00358&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-12/acs.jmedchem.6b00358/20160617/images/medium/jm-2016-003586_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-12/acs.jmedchem.6b00358/20160617/images/large/jm-2016-003586_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Atenolol series. The modeled binding modes of atenolol, metoprolol, and bisoprolol. Atenolol becomes the most potent in the series after correction for membrane binding and is observed to form a greater number of hydrogen bonds with the protein than metoprolol or bisoprolol.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-12/acs.jmedchem.6b00358/20160617/images/large/jm-2016-003586_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00358&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-12/acs.jmedchem.6b00358/20160617/images/medium/jm-2016-003586_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-12/acs.jmedchem.6b00358/20160617/images/large/jm-2016-003586_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Cyanopindolol series. The modeled binding modes of cyanopindolol, CGP12177, nadolol, propranolol, and ICI-118,551 with WATMD results. The first panel shows the WATMD results from simulation of the apo β2-AR protein (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2RH1">2RH1</a> with antagonist carazolol removed), with cyanopindolol binding mode shown for reference. After correction for membrane binding, CGP12177, and nadolol become more potent than propranolol or ICI-118,551, which is consistent with the greater number of hydrogen bonds the former two compounds make with the protein. Cyanopindolol remains the most potent in the series, which correlates with efficient replacement of unfavorable water at the rear of the binding pocket.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-12/acs.jmedchem.6b00358/20160617/images/large/jm-2016-003586_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00358&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-12/acs.jmedchem.6b00358/20160617/images/medium/jm-2016-003586_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-12/acs.jmedchem.6b00358/20160617/images/large/jm-2016-003586_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. This model accounts for both increased free ligand concentration in the target compartment (dark-blue zone) relative to bulk solvent (light-blue zone) and ligand partitioned into the membrane (i.e., not available for binding from solution phase but possibly available for binding from the membrane phase) (gray zone). While this scheme is more realistic, it is also more complex, and experimental determination of <i>K</i><sub>assoc</sub>, <i>k</i><sub>1</sub>, k<sub>–1</sub>, <i>K</i><sub>part</sub>, <i>k</i><sub>2</sub>, and <i>k</i><sub>–2</sub> may prove challenging. Additional terms (<i>K</i><sub>d</sub><sup>mem,int</sup>, <i>k</i><sub>3</sub>, <i>k</i><sub>–3</sub>) are needed to account for binding at membrane-exposed protein binding sites via lateral diffusion of partitioned compounds and to account for compounds that simply permeate through the membrane exiting into the intracellular compartment (<i>k</i><sub>4</sub>, <i>k</i><sub>–4</sub>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-12/acs.jmedchem.6b00358/20160617/images/large/jm-2016-003586_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00358&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i10">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_55722" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_55722" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 91 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Pierce, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Premont, R. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lefkowitz, R. J.</span><span> </span><span class="NLM_article-title">Seven-transmembrane receptors</span> <span class="citation_source-journal">Nat. Rev. Mol. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">639</span><span class="NLM_x">–</span> <span class="NLM_lpage">650</span><span class="refDoi"> DOI: 10.1038/nrm908</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1038%2Fnrm908" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=12209124" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BD38Xmslansr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2002&pages=639-650&author=K.+L.+Pierceauthor=R.+T.+Premontauthor=R.+J.+Lefkowitz&title=Seven-transmembrane+receptors&doi=10.1038%2Fnrm908"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Seven-transmembrane receptors</span></div><div class="casAuthors">Pierce, Kristen L.; Premont, Richard T.; Lefkowitz, Robert J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">639-650</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Seven-transmembrane receptors, which constitute the largest, most ubiquitous, and most versatile family of membrane receptors, are also the most common target of therapeutic drugs.  Recent findings indicate that the classical models of G-protein coupling and activation of second-messenger-generating enzymes do not fully explain their remarkably diverse biol. actions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5tA1lRPo14bVg90H21EOLACvtfcHk0lgU_b66OTJnNg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xmslansr4%253D&md5=35c79407b9a30df8eb018e31529ada81</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrm908&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm908%26sid%3Dliteratum%253Aachs%26aulast%3DPierce%26aufirst%3DK.%2BL.%26aulast%3DPremont%26aufirst%3DR.%2BT.%26aulast%3DLefkowitz%26aufirst%3DR.%2BJ.%26atitle%3DSeven-transmembrane%2520receptors%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2002%26volume%3D3%26spage%3D639%26epage%3D650%26doi%3D10.1038%2Fnrm908" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Anderson, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Linden, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rabe, K.</span><span> </span><span class="NLM_article-title">Why are long-acting beta-adrenoceptor agonists long-acting?</span> <span class="citation_source-journal">Eur. Respir. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">569</span><span class="NLM_x">–</span> <span class="NLM_lpage">578</span><span class="refDoi"> DOI: 10.1183/09031936.94.07030569</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1183%2F09031936.94.07030569" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=7912202" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADyaK2cXkvFyrurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=1994&pages=569-578&author=G.+Andersonauthor=A.+Lindenauthor=K.+Rabe&title=Why+are+long-acting+beta-adrenoceptor+agonists+long-acting%3F&doi=10.1183%2F09031936.94.07030569"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Why are long-acting beta-adrenoceptor agonists long-acting?</span></div><div class="casAuthors">Anderson, G. P.; Linden, A.; Rabe, K. F.</div><div class="citationInfo"><span class="NLM_cas:title">European Respiratory Journal</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">569-78</span>CODEN:
                <span class="NLM_cas:coden">ERJOEI</span>;
        ISSN:<span class="NLM_cas:issn">0903-1936</span>.
    </div><div class="casAbstract">A review with 60 refs.  The extended duration of bronchodilation due to formoterol and salmeterol greatly exceeds that of short acting beta2-adrenoceptor agonists, such as salbutamol or terbutaline.  This extended duration and their capacity to "reassert" airway smooth muscle relaxation in vitro despite repeated washing has prompted considerable debate on the underlying mechanism(s).  The comparative pharmacol., and mol. modeling of these drugs and of the beta2-adrenoceptor and its ligand binding core have cast doubt on the exosite/exoceptor model previously proposed to explain the behavior of salmeterol.  The authors present evidence supporting a unifying hypothesis that the duration of action both of formoterol and salmeterol is detd. principally by their physicochem. interactions with membrane lipid bilayers (plasmalemma diffusion microkinetic model), rather than putative distinct exosite/exoceptor binding sites in or near the beta2-adrenoceptor.  This model provides a clearer understanding of the pharmacol. profile of these drugs (rate of onset, duration, "reassertion", interaction with hydrophilic and hydrophobic beta2-adrenoceptor antagonists), and explains why in human airway smooth muscle in vitro a true relaxation-concn. response may not exist for salmeterol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbiLdcPisP0LVg90H21EOLACvtfcHk0lgU_b66OTJnNg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXkvFyrurc%253D&md5=e3235f7cb3d120c886336564010db96e</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1183%2F09031936.94.07030569&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1183%252F09031936.94.07030569%26sid%3Dliteratum%253Aachs%26aulast%3DAnderson%26aufirst%3DG.%26aulast%3DLinden%26aufirst%3DA.%26aulast%3DRabe%26aufirst%3DK.%26atitle%3DWhy%2520are%2520long-acting%2520beta-adrenoceptor%2520agonists%2520long-acting%253F%26jtitle%3DEur.%2520Respir.%2520J.%26date%3D1994%26volume%3D7%26spage%3D569%26epage%3D578%26doi%3D10.1183%2F09031936.94.07030569" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Sargent, D. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwyzer, R.</span><span> </span><span class="NLM_article-title">Membrane lipid phase as catalyst for peptide-receptor interactions</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1986</span><span class="NLM_x">, </span> <span class="NLM_volume">83</span><span class="NLM_x">, </span> <span class="NLM_fpage">5774</span><span class="NLM_x">–</span> <span class="NLM_lpage">5778</span><span class="refDoi"> DOI: 10.1073/pnas.83.16.5774</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1073%2Fpnas.83.16.5774" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=83&publication_year=1986&pages=5774-5778&author=D.+F.+Sargentauthor=R.+Schwyzer&title=Membrane+lipid+phase+as+catalyst+for+peptide-receptor+interactions&doi=10.1073%2Fpnas.83.16.5774"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1073%2Fpnas.83.16.5774&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.83.16.5774%26sid%3Dliteratum%253Aachs%26aulast%3DSargent%26aufirst%3DD.%2BF.%26aulast%3DSchwyzer%26aufirst%3DR.%26atitle%3DMembrane%2520lipid%2520phase%2520as%2520catalyst%2520for%2520peptide-receptor%2520interactions%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1986%26volume%3D83%26spage%3D5774%26epage%3D5778%26doi%3D10.1073%2Fpnas.83.16.5774" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Vauquelin, G.</span><span> </span><span class="NLM_article-title">On the ‘micro’-pharmacodynamic and pharmacokinetic mechanisms that contribute to long-lasting drug action</span> <span class="citation_source-journal">Expert Opin. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">1085</span><span class="NLM_x">–</span> <span class="NLM_lpage">1098</span><span class="refDoi"> DOI: 10.1517/17460441.2015.1067196</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1517%2F17460441.2015.1067196" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=1085-1098&author=G.+Vauquelin&title=On+the+%E2%80%98micro%E2%80%99-pharmacodynamic+and+pharmacokinetic+mechanisms+that+contribute+to+long-lasting+drug+action&doi=10.1517%2F17460441.2015.1067196"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1517%2F17460441.2015.1067196&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F17460441.2015.1067196%26sid%3Dliteratum%253Aachs%26aulast%3DVauquelin%26aufirst%3DG.%26atitle%3DOn%2520the%2520%25E2%2580%2598micro%25E2%2580%2599-pharmacodynamic%2520and%2520pharmacokinetic%2520mechanisms%2520that%2520contribute%2520to%2520long-lasting%2520drug%2520action%26jtitle%3DExpert%2520Opin.%2520Drug%2520Discovery%26date%3D2015%26volume%3D10%26spage%3D1085%26epage%3D1098%26doi%3D10.1517%2F17460441.2015.1067196" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Sykes, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parry, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reilly, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wright, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fairhurst, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Charlton, S. J.</span><span> </span><span class="NLM_article-title">Observed drug-receptor association rates are governed by membrane affinity: the importance of establishing “micro-pharmacokinetic/pharmacodynamic relationships” at the β2-adrenoceptor</span> <span class="citation_source-journal">Mol. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">85</span><span class="NLM_x">, </span> <span class="NLM_fpage">608</span><span class="NLM_x">–</span> <span class="NLM_lpage">617</span><span class="refDoi"> DOI: 10.1124/mol.113.090209</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1124%2Fmol.113.090209" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=24476583" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmsVyntbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=85&publication_year=2014&pages=608-617&author=D.+A.+Sykesauthor=C.+Parryauthor=J.+Reillyauthor=P.+Wrightauthor=R.+A.+Fairhurstauthor=S.+J.+Charlton&title=Observed+drug-receptor+association+rates+are+governed+by+membrane+affinity%3A+the+importance+of+establishing+%E2%80%9Cmicro-pharmacokinetic%2Fpharmacodynamic+relationships%E2%80%9D+at+the+%CE%B22-adrenoceptor&doi=10.1124%2Fmol.113.090209"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Observed drug-receptor association rates are governed by membrane affinity: the importance of establishing "micro-pharmacokinetic/pharmacodynamic relationships" at the β2-adrenoceptor</span></div><div class="casAuthors">Sykes, David A.; Parry, Cheryl; Reilly, John; Wright, Penny; Fairhurst, Robin A.; Charlton, Steven J.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">85</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">608-617, 10 pp.</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">1521-0111</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Current pharmacol. models for detg. affinity and kinetics of drugs for membrane receptors assume the interacting mols. are homogeneously distributed in the bulk aq. phase.  The phospholipid membrane can, however, provide a second compartment into which drugs can partition, particularly lipophilic/basic compds.  In this study we measured the phospholipid affinity and receptor binding kinetics of several clin. relevant β2-adrenoceptor agonists and antagonists and demonstrated that the degree of phospholipid interaction directly affects the obsd. kinetic assocn. rate (kon) and dissocn. const. (kd), but not the dissocn. rate (koff) from the target, by concg. drug in the local environment around the receptor.  When the local drug concn. was accounted for, the kon was comparable across the cohort and the cor. kd was directly related to the koff.  In conclusion, we propose a new approach to detg. the pharmacol. of drugs for membrane targets that accounts for differences in local drug concn. brought about by direct affinity for phospholipids, establishing "micro-pharmacokinetic/pharmacodynamic relationships" for drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYPaYJeo0NAbVg90H21EOLACvtfcHk0li-p1QqLq4AXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmsVyntbg%253D&md5=b8038cbc7d9409ec8358f2836bd4df1f</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1124%2Fmol.113.090209&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.113.090209%26sid%3Dliteratum%253Aachs%26aulast%3DSykes%26aufirst%3DD.%2BA.%26aulast%3DParry%26aufirst%3DC.%26aulast%3DReilly%26aufirst%3DJ.%26aulast%3DWright%26aufirst%3DP.%26aulast%3DFairhurst%26aufirst%3DR.%2BA.%26aulast%3DCharlton%26aufirst%3DS.%2BJ.%26atitle%3DObserved%2520drug-receptor%2520association%2520rates%2520are%2520governed%2520by%2520membrane%2520affinity%253A%2520the%2520importance%2520of%2520establishing%2520%25E2%2580%259Cmicro-pharmacokinetic%252Fpharmacodynamic%2520relationships%25E2%2580%259D%2520at%2520the%2520%25CE%25B22-adrenoceptor%26jtitle%3DMol.%2520Pharmacol.%26date%3D2014%26volume%3D85%26spage%3D608%26epage%3D617%26doi%3D10.1124%2Fmol.113.090209" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Alikhani, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beer, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bentley, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bruce, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cuenoud, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fairhurst, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gedeck, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haberthuer, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hayden, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janus, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jordan, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lewis, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smithies, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wissler, E.</span><span> </span><span class="NLM_article-title">Long-chain formoterol analogues: an investigation into the effect of increasing amino-substituent chain length on the β2-adrenoceptor activity</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">4705</span><span class="NLM_x">–</span> <span class="NLM_lpage">4710</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2004.06.086</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1016%2Fj.bmcl.2004.06.086" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=15324892" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmvV2ht7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2004&pages=4705-4710&author=V.+Alikhaniauthor=D.+Beerauthor=D.+Bentleyauthor=I.+Bruceauthor=B.+M.+Cuenoudauthor=R.+A.+Fairhurstauthor=P.+Gedeckauthor=S.+Haberthuerauthor=C.+Haydenauthor=D.+Janusauthor=L.+Jordanauthor=C.+Lewisauthor=K.+Smithiesauthor=E.+Wissler&title=Long-chain+formoterol+analogues%3A+an+investigation+into+the+effect+of+increasing+amino-substituent+chain+length+on+the+%CE%B22-adrenoceptor+activity&doi=10.1016%2Fj.bmcl.2004.06.086"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Long-chain formoterol analogues: an investigation into the effect of increasing amino-substituent chain length on the β2-adrenoceptor activity</span></div><div class="casAuthors">Alikhani, Vahid; Beer, David; Bentley, David; Bruce, Ian; Cuenoud, Bernard M.; Fairhurst, Robin A.; Gedeck, Peter; Haberthuer, Sandra; Hayden, Claire; Janus, Diana; Jordan, Lynne; Lewis, Christine; Smithies, Kirsty; Wissler, Elke</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">4705-4710</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The synthesis of a series of long-chain formoterol analogs in which the terminal ether residue of the β-phenethylamino-substituent has been extended beyond the Me ether residue present in the parent compd. are described.  Evaluation of these analogs as β2-adrenoceptor agonists was used to provide an insight into the factors controlling the magnitude and duration of receptor activation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp69yYgHdjw2bVg90H21EOLACvtfcHk0li-p1QqLq4AXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmvV2ht7o%253D&md5=55b9db6af59617a1e5dfbf2c80cb53bd</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2004.06.086&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2004.06.086%26sid%3Dliteratum%253Aachs%26aulast%3DAlikhani%26aufirst%3DV.%26aulast%3DBeer%26aufirst%3DD.%26aulast%3DBentley%26aufirst%3DD.%26aulast%3DBruce%26aufirst%3DI.%26aulast%3DCuenoud%26aufirst%3DB.%2BM.%26aulast%3DFairhurst%26aufirst%3DR.%2BA.%26aulast%3DGedeck%26aufirst%3DP.%26aulast%3DHaberthuer%26aufirst%3DS.%26aulast%3DHayden%26aufirst%3DC.%26aulast%3DJanus%26aufirst%3DD.%26aulast%3DJordan%26aufirst%3DL.%26aulast%3DLewis%26aufirst%3DC.%26aulast%3DSmithies%26aufirst%3DK.%26aulast%3DWissler%26aufirst%3DE.%26atitle%3DLong-chain%2520formoterol%2520analogues%253A%2520an%2520investigation%2520into%2520the%2520effect%2520of%2520increasing%2520amino-substituent%2520chain%2520length%2520on%2520the%2520%25CE%25B22-adrenoceptor%2520activity%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2004%26volume%3D14%26spage%3D4705%26epage%3D4710%26doi%3D10.1016%2Fj.bmcl.2004.06.086" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Stocks, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alcaraz, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bailey, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonnert, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cadogan, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Christie, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dixon, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Connolly, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cook, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fisher, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flaherty, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Humphries, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ingall, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jordan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lawson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mullen, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nicholls, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paine, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pairaudeau, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, A.</span><span> </span><span class="NLM_article-title">Discovery of AZD3199, an inhaled ultralong acting β2 receptor agonist with rapid onset of action</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">416</span><span class="NLM_x">–</span> <span class="NLM_lpage">421</span><span class="refDoi"> DOI: 10.1021/ml4005232</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml4005232" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=416-421&author=M.+J.+Stocksauthor=L.+Alcarazauthor=A.+Baileyauthor=R.+Bonnertauthor=E.+Cadoganauthor=J.+Christieauthor=J.+Dixonauthor=S.+Connollyauthor=A.+Cookauthor=A.+Fisherauthor=A.+Flahertyauthor=A.+Humphriesauthor=A.+Ingallauthor=S.+Jordanauthor=M.+Lawsonauthor=A.+Mullenauthor=D.+Nichollsauthor=S.+Paineauthor=G.+Pairaudeauauthor=A.+Young&title=Discovery+of+AZD3199%2C+an+inhaled+ultralong+acting+%CE%B22+receptor+agonist+with+rapid+onset+of+action&doi=10.1021%2Fml4005232"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Fml4005232&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml4005232%26sid%3Dliteratum%253Aachs%26aulast%3DStocks%26aufirst%3DM.%2BJ.%26aulast%3DAlcaraz%26aufirst%3DL.%26aulast%3DBailey%26aufirst%3DA.%26aulast%3DBonnert%26aufirst%3DR.%26aulast%3DCadogan%26aufirst%3DE.%26aulast%3DChristie%26aufirst%3DJ.%26aulast%3DDixon%26aufirst%3DJ.%26aulast%3DConnolly%26aufirst%3DS.%26aulast%3DCook%26aufirst%3DA.%26aulast%3DFisher%26aufirst%3DA.%26aulast%3DFlaherty%26aufirst%3DA.%26aulast%3DHumphries%26aufirst%3DA.%26aulast%3DIngall%26aufirst%3DA.%26aulast%3DJordan%26aufirst%3DS.%26aulast%3DLawson%26aufirst%3DM.%26aulast%3DMullen%26aufirst%3DA.%26aulast%3DNicholls%26aufirst%3DD.%26aulast%3DPaine%26aufirst%3DS.%26aulast%3DPairaudeau%26aufirst%3DG.%26aulast%3DYoung%26aufirst%3DA.%26atitle%3DDiscovery%2520of%2520AZD3199%252C%2520an%2520inhaled%2520ultralong%2520acting%2520%25CE%25B22%2520receptor%2520agonist%2520with%2520rapid%2520onset%2520of%2520action%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D5%26spage%3D416%26epage%3D421%26doi%3D10.1021%2Fml4005232" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">McCloskey, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poo, M. M.</span><span> </span><span class="NLM_article-title">Rates of membrane-associated reactions: reduction of dimensionality revisited</span> <span class="citation_source-journal">J. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1986</span><span class="NLM_x">, </span> <span class="NLM_volume">102</span><span class="NLM_x">, </span> <span class="NLM_fpage">88</span><span class="NLM_x">–</span> <span class="NLM_lpage">96</span><span class="refDoi"> DOI: 10.1083/jcb.102.1.88</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1083%2Fjcb.102.1.88" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=1986&pages=88-96&author=M.+A.+McCloskeyauthor=M.+M.+Poo&title=Rates+of+membrane-associated+reactions%3A+reduction+of+dimensionality+revisited&doi=10.1083%2Fjcb.102.1.88"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1083%2Fjcb.102.1.88&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1083%252Fjcb.102.1.88%26sid%3Dliteratum%253Aachs%26aulast%3DMcCloskey%26aufirst%3DM.%2BA.%26aulast%3DPoo%26aufirst%3DM.%2BM.%26atitle%3DRates%2520of%2520membrane-associated%2520reactions%253A%2520reduction%2520of%2520dimensionality%2520revisited%26jtitle%3DJ.%2520Cell%2520Biol.%26date%3D1986%26volume%3D102%26spage%3D88%26epage%3D96%26doi%3D10.1083%2Fjcb.102.1.88" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Vauquelin, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Packeu, A.</span><span> </span><span class="NLM_article-title">Ligands, their receptors and··· plasma membranes</span> <span class="citation_source-journal">Mol. Cell. Endocrinol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">311</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">10</span><span class="refDoi"> DOI: 10.1016/j.mce.2009.07.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1016%2Fj.mce.2009.07.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=19647036" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtV2ksL%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=311&publication_year=2009&pages=1-10&author=G.+Vauquelinauthor=A.+Packeu&title=Ligands%2C+their+receptors+and%C2%B7%C2%B7%C2%B7+plasma+membranes&doi=10.1016%2Fj.mce.2009.07.022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Ligands, their receptors and ... plasma membranes</span></div><div class="casAuthors">Vauquelin, G.; Packeu, A.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Endocrinology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">311</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">1-10</span>CODEN:
                <span class="NLM_cas:coden">MCEND6</span>;
        ISSN:<span class="NLM_cas:issn">0303-7207</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ireland Ltd.</span>)
        </div><div class="casAbstract">A review.  Ligand-receptor interactions are customarily described by equations that apply to solutes.  Yet, most receptors are present in cell membranes so that sufficiently lipophilic ligands could reach the receptor by a 2-dimensional approach within the membrane.  As summarized in this review, this may affect the ligand-receptor interaction in many ways.  Biophysicists have calcd. that, compared to a 3-dimensional approach from the liq. phase, such an approach could alter the time the ligands need to find a receptor.  Biochemists have found that ligand incorporation into lipid bilayers modifies their conformation.  This, along with the depth at which the ligands reside in the bilayer, will affect the probability of successful receptor interaction.  Novel mechanisms have also been introduced, including "exosite" binding and ligand translocation between the receptor α-helical transmembrane domains.  Pharmacologists have focused attention at ligand concns. in the membrane, their adsorption and release rates, and the effects thereof on ligand potency and residence time at the receptor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGquzgYx7fkdlbVg90H21EOLACvtfcHk0lggOy2OWOocFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtV2ksL%252FO&md5=e2ffcdac2bb4cb2a905df184a20e31b5</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.mce.2009.07.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.mce.2009.07.022%26sid%3Dliteratum%253Aachs%26aulast%3DVauquelin%26aufirst%3DG.%26aulast%3DPackeu%26aufirst%3DA.%26atitle%3DLigands%252C%2520their%2520receptors%2520and%25C2%25B7%25C2%25B7%25C2%25B7%2520plasma%2520membranes%26jtitle%3DMol.%2520Cell.%2520Endocrinol.%26date%3D2009%26volume%3D311%26spage%3D1%26epage%3D10%26doi%3D10.1016%2Fj.mce.2009.07.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Johnson, M.</span><span> </span><span class="NLM_article-title">Salmeterol: a novel drug for the treatment of asthma</span> <span class="citation_source-journal">Agents Actions Suppl.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1991</span><span class="NLM_x">, </span> <span class="NLM_volume">34</span><span class="NLM_x">, </span> <span class="NLM_fpage">79</span><span class="NLM_x">–</span> <span class="NLM_lpage">95</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1686530" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADyaK38XltFamsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=1991&pages=79-95&author=M.+Johnson&title=Salmeterol%3A+a+novel+drug+for+the+treatment+of+asthma"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Salmeterol:  a novel drug for the treatment of asthma</span></div><div class="casAuthors">Johnson, M.</div><div class="citationInfo"><span class="NLM_cas:title">Agents and Actions Supplements</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">New Drugs Asthma Ther.</span>),
    <span class="NLM_cas:pages">79-95</span>CODEN:
                <span class="NLM_cas:coden">AASUDJ</span>;
        ISSN:<span class="NLM_cas:issn">0379-0363</span>.
    </div><div class="casAbstract">Salmeterol (I) is the first of a new generation of rationally-designed, potent and selective β2-adrenoceptor agonists.  It has a unique pharmacol. profile that arises from a novel mechanism of action, whereby the mol. is long-acting by persisting at exo-receptor binding sites in the cell membrane.  I induces sustained bronchodilatation and has significant anti-inflammatory activity.  This combination of properties suggests that salmeterol may represent an important new advance in the treatment of bronchial asthma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpT2ecouZFxS7Vg90H21EOLACvtfcHk0lggOy2OWOocFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38XltFamsg%253D%253D&md5=9d55110aa68898b4cc92a8e46da9b224</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DJohnson%26aufirst%3DM.%26atitle%3DSalmeterol%253A%2520a%2520novel%2520drug%2520for%2520the%2520treatment%2520of%2520asthma%26jtitle%3DAgents%2520Actions%2520Suppl.%26date%3D1991%26volume%3D34%26spage%3D79%26epage%3D95" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Green, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spasoff, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coleman, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liggett, S. B.</span><span> </span><span class="NLM_article-title">Sustained activation of a G protein-coupled receptor via “anchored” agonist binding: molecular localization of the salmeterol exosite within the 2-adrenergic receptor</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">271</span><span class="NLM_x">, </span> <span class="NLM_fpage">24029</span><span class="NLM_x">–</span> <span class="NLM_lpage">24035</span><span class="refDoi"> DOI: 10.1074/jbc.271.39.24029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1074%2Fjbc.271.39.24029" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=271&publication_year=1996&pages=24029-24035&author=S.+A.+Greenauthor=A.+P.+Spasoffauthor=R.+A.+Colemanauthor=M.+Johnsonauthor=S.+B.+Liggett&title=Sustained+activation+of+a+G+protein-coupled+receptor+via+%E2%80%9Canchored%E2%80%9D+agonist+binding%3A+molecular+localization+of+the+salmeterol+exosite+within+the+2-adrenergic+receptor&doi=10.1074%2Fjbc.271.39.24029"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1074%2Fjbc.271.39.24029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.271.39.24029%26sid%3Dliteratum%253Aachs%26aulast%3DGreen%26aufirst%3DS.%2BA.%26aulast%3DSpasoff%26aufirst%3DA.%2BP.%26aulast%3DColeman%26aufirst%3DR.%2BA.%26aulast%3DJohnson%26aufirst%3DM.%26aulast%3DLiggett%26aufirst%3DS.%2BB.%26atitle%3DSustained%2520activation%2520of%2520a%2520G%2520protein-coupled%2520receptor%2520via%2520%25E2%2580%259Canchored%25E2%2580%259D%2520agonist%2520binding%253A%2520molecular%2520localization%2520of%2520the%2520salmeterol%2520exosite%2520within%2520the%25202-adrenergic%2520receptor%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1996%26volume%3D271%26spage%3D24029%26epage%3D24035%26doi%3D10.1074%2Fjbc.271.39.24029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Vauquelin, G.</span><span> </span><span class="NLM_article-title">Rebinding: or why drugs may act longer in vivo than expected from their in vitro target residence time</span> <span class="citation_source-journal">Expert Opin. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">927</span><span class="NLM_x">–</span> <span class="NLM_lpage">941</span><span class="refDoi"> DOI: 10.1517/17460441.2010.512037</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1517%2F17460441.2010.512037" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2010&pages=927-941&author=G.+Vauquelin&title=Rebinding%3A+or+why+drugs+may+act+longer+in+vivo+than+expected+from+their+in+vitro+target+residence+time&doi=10.1517%2F17460441.2010.512037"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1517%2F17460441.2010.512037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F17460441.2010.512037%26sid%3Dliteratum%253Aachs%26aulast%3DVauquelin%26aufirst%3DG.%26atitle%3DRebinding%253A%2520or%2520why%2520drugs%2520may%2520act%2520longer%2520in%2520vivo%2520than%2520expected%2520from%2520their%2520in%2520vitro%2520target%2520residence%2520time%26jtitle%3DExpert%2520Opin.%2520Drug%2520Discovery%26date%3D2010%26volume%3D5%26spage%3D927%26epage%3D941%26doi%3D10.1517%2F17460441.2010.512037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Vauquelin, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Charlton, S. J.</span><span> </span><span class="NLM_article-title">Long-lasting target binding and rebinding as mechanisms to prolong in vivo drug action</span> <span class="citation_source-journal">Br. J. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">161</span><span class="NLM_x">, </span> <span class="NLM_fpage">488</span><span class="NLM_x">–</span> <span class="NLM_lpage">508</span><span class="refDoi"> DOI: 10.1111/j.1476-5381.2010.00936.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1111%2Fj.1476-5381.2010.00936.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=20880390" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1Kju7zE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=161&publication_year=2010&pages=488-508&author=G.+Vauquelinauthor=S.+J.+Charlton&title=Long-lasting+target+binding+and+rebinding+as+mechanisms+to+prolong+in+vivo+drug+action&doi=10.1111%2Fj.1476-5381.2010.00936.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Long-lasting target binding and rebinding as mechanisms to prolong in vivo drug action</span></div><div class="casAuthors">Vauquelin, Georges; Charlton, Steven J.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">161</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">488-508</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">1476-5381</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  An increasing no. of examples in the literature suggest that the in vivo duration of drug action not only depends on macroscopic pharmacokinetic properties like plasma half-life and the time needed to equilibrate between the plasma and the effect compartments, but is also influenced by long-lasting target binding and rebinding.  The present review combines information from different research areas and simulations to explore the nature of these mechanisms and the conditions in which they are most prevalent.  Simulations reveal that these latter phenomena become esp. influential when there is no longer sufficient free drug around to maintain high levels of receptor occupancy.  There is not always a direct link between slow dissocn. and long-lasting in vivo target protection, as the rate of free drug elimination from the effect compartment is also a key influencing factor.  Local phenomena that hinder the diffusion of free drug mols. away from their target may allow them to consecutively bind to the same target and/or targets nearby (denoted as 'rebinding') even when their concn. in the bulk phase has already dropped to insignificant levels.  The micro-anat. properties of many effect compartments are likely to intensify this phenomenon.  By mimicking the complexity of tissues, intact cells offer the opportunity to investigate both mechanisms under the same, physiol. relevant conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqzSGNmKeaTzrVg90H21EOLACvtfcHk0lgIQivNFiDLow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht1Kju7zE&md5=bd91e09a0fd79a2de6bf9ef0cfd3443e</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1111%2Fj.1476-5381.2010.00936.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1476-5381.2010.00936.x%26sid%3Dliteratum%253Aachs%26aulast%3DVauquelin%26aufirst%3DG.%26aulast%3DCharlton%26aufirst%3DS.%2BJ.%26atitle%3DLong-lasting%2520target%2520binding%2520and%2520rebinding%2520as%2520mechanisms%2520to%2520prolong%2520in%2520vivo%2520drug%2520action%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2010%26volume%3D161%26spage%3D488%26epage%3D508%26doi%3D10.1111%2Fj.1476-5381.2010.00936.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Coleman, R. A.</span><span> </span><span class="NLM_article-title">On the mechanism of the persistent action of salmeterol: what is the current position?</span> <span class="citation_source-journal">Br. J. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">158</span><span class="NLM_x">, </span> <span class="NLM_fpage">180</span><span class="NLM_x">–</span> <span class="NLM_lpage">182</span><span class="refDoi"> DOI: 10.1111/j.1476-5381.2009.00370.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1111%2Fj.1476-5381.2009.00370.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=19719780" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsVKmu7fL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=158&publication_year=2009&pages=180-182&author=R.+A.+Coleman&title=On+the+mechanism+of+the+persistent+action+of+salmeterol%3A+what+is+the+current+position%3F&doi=10.1111%2Fj.1476-5381.2009.00370.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">On the mechanism of the persistent action of salmeterol: what is the current position?</span></div><div class="casAuthors">Coleman, R. A.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">158</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">180-182</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">1476-5381</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  The mechanism of the long duration of action of salmeterol at β2-adrenoceptors has long been a matter of debate, and is still unresolved.  Szczuka and colleagues have both summarized the position to date and suggested a new mechanistic contender, receptor rebinding.  Despite this, they still do not come to any clear conclusion.  Much of the literature data that they have drawn upon appears contradictory, and math. models are inevitably flawed by the questionable validity of key values applied to them.  Although the issue will undoubtedly eventually be resolved, it will probably require investigators to apply carefully designed studies on simple exptl. systems such as isolated membranes or cultured cells.  Only then should studies be extended to more complex systems such as isolated prepns. of airways smooth muscle, where tissue bulk inevitably presents a complicating factor, particularly where relatively lipophilic compds. are concerned.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojZUYZ-ov-JrVg90H21EOLACvtfcHk0lgIQivNFiDLow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsVKmu7fL&md5=f7c10b7848968bc91725fd8939ffb6c2</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1111%2Fj.1476-5381.2009.00370.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1476-5381.2009.00370.x%26sid%3Dliteratum%253Aachs%26aulast%3DColeman%26aufirst%3DR.%2BA.%26atitle%3DOn%2520the%2520mechanism%2520of%2520the%2520persistent%2520action%2520of%2520salmeterol%253A%2520what%2520is%2520the%2520current%2520position%253F%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2009%26volume%3D158%26spage%3D180%26epage%3D182%26doi%3D10.1111%2Fj.1476-5381.2009.00370.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Szczuka, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wennerberg, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Packeu, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vauquelin, G.</span><span> </span><span class="NLM_article-title">Molecular mechanisms for the persistent bronchodilatory effect of the β2-adrenoceptor agonist salmeterol</span> <span class="citation_source-journal">Br. J. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">158</span><span class="NLM_x">, </span> <span class="NLM_fpage">183</span><span class="NLM_x">–</span> <span class="NLM_lpage">194</span><span class="refDoi"> DOI: 10.1111/j.1476-5381.2009.00296.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1111%2Fj.1476-5381.2009.00296.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=19594756" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsVKlsr7M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=158&publication_year=2009&pages=183-194&author=A.+Szczukaauthor=M.+Wennerbergauthor=A.+Packeuauthor=G.+Vauquelin&title=Molecular+mechanisms+for+the+persistent+bronchodilatory+effect+of+the+%CE%B22-adrenoceptor+agonist+salmeterol&doi=10.1111%2Fj.1476-5381.2009.00296.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular mechanisms for the persistent bronchodilatory effect of the β2-adrenoceptor agonist salmeterol</span></div><div class="casAuthors">Szczuka, A.; Wennerberg, M.; Packeu, A.; Vauquelin, G.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">158</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">183-194</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">1476-5381</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">β2-adrenoceptor agonists are effective bronchodilators.  In vitro studies demonstrated long-lasting airway smooth muscle relaxation by salmeterol after washout, the quick disappearance of this effect in presence of antagonists and its recovery after antagonist removal.  Current explanations invoke salmeterol accumulation in the membrane ('diffusion microkinetic' model) or the existence of salmeterol-binding 'exosites'.  An alternative model based on 'rebinding' of a dissocd. ligand to the receptor mols. also produces an apparent decrease in the ligand's dissocn. rate in the absence of competing ligands.  Computer-assisted simulations were performed to follow the receptor-occupation by a salmeterol-like ligand and a competing ligand as a function of time.  The aptness of the models to describe the above in vitro findings was evaluated.  The 'diffusion microkinetic' model is sufficient to explain a long-lasting β2-adrenoceptor stimulation and reassertion as long as the membrane harbors a high concn. of the agonist.  At lower concn., 'rebinding' and, in second place, 'exosite' binding are likely to become operational.  The 'rebinding' and 'exosite' binding mechanisms take place at a sub-cellular/mol. scale.  Pending their demonstration by expts. on appropriate, simple models such as intact cells or membranes thereof, these mechanisms remain hypothetical in the case of salmeterol.  Airway smooth muscle contraction could also be governed by addnl. mechanisms that are particular to this macroscopic approach.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEC9rfvp5YbbVg90H21EOLACvtfcHk0lgIQivNFiDLow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsVKlsr7M&md5=0515669d4f02ba8bd14f1bca84da4275</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1111%2Fj.1476-5381.2009.00296.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1476-5381.2009.00296.x%26sid%3Dliteratum%253Aachs%26aulast%3DSzczuka%26aufirst%3DA.%26aulast%3DWennerberg%26aufirst%3DM.%26aulast%3DPackeu%26aufirst%3DA.%26aulast%3DVauquelin%26aufirst%3DG.%26atitle%3DMolecular%2520mechanisms%2520for%2520the%2520persistent%2520bronchodilatory%2520effect%2520of%2520the%2520%25CE%25B22-adrenoceptor%2520agonist%2520salmeterol%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2009%26volume%3D158%26spage%3D183%26epage%3D194%26doi%3D10.1111%2Fj.1476-5381.2009.00296.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Slack, R. J.</span><span> </span><span class="NLM_article-title">Evidence for a non-β2-adrenoceptor binding site in human lung tissue for a subset of β2-adrenoceptor agonists</span> <span class="citation_source-journal">Pharmacol. Pharm.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">30</span><span class="NLM_x">–</span> <span class="NLM_lpage">36</span><span class="refDoi"> DOI: 10.4236/pp.2014.51006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.4236%2Fpp.2014.51006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlejt7vJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=30-36&author=R.+J.+Slack&title=Evidence+for+a+non-%CE%B22-adrenoceptor+binding+site+in+human+lung+tissue+for+a+subset+of+%CE%B22-adrenoceptor+agonists&doi=10.4236%2Fpp.2014.51006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Evidence for a non-β2-adrenoceptor binding site in human lung tissue for a subset of β2-adrenoceptor agonists</span></div><div class="casAuthors">Slack, Robert J.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacology & Pharmacy</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">30-36</span>CODEN:
                <span class="NLM_cas:coden">PPHHBN</span>;
        ISSN:<span class="NLM_cas:issn">2157-9431</span>.
    
            (<span class="NLM_cas:orgname">Scientific Research Publishing, Inc.</span>)
        </div><div class="casAbstract">The aim of this study was to compare the binding profile of a range of β2-adrenoceptor (β2-AR) agonists and antagonists in human lung tissue.  Radioligand satn. and competition binding expts. were performed by filtration with a β2-AR antagonist ([3H]propranolol) or agonist ([3H]vilanterol) radioligand and membrane fragments generated from lung parenchyma in the presence of 100 μM guanosine 5'-[β,γ-imido]triphosphate (Gpp(NH)p).  In membranes prepd. from human lung parenchyma, carmoterol, formoterol, ICI118551, propranolol and salbutamol resulted in inhibition of [3H]vilanterol binding to levels that were significantly different from indacaterol, salmeterol and vilanterol (ANOVA, Bonferroni post-test, P < 0.001 except formoterol vs indacaterol where P < 0.01).  Indacaterol and salmeterol resulted in inhibition of [3H]vilanterol binding to levels that were not significantly different from vilanterol (ANOVA, Bonferroni post-test, P > 0.05).  Indacaterol, salmeterol and vilanterol resulted in full inhibition of [3H]propranolol binding to levels not significantly different from ICI118551 (ANOVA, Bonferroni post-test, P > 0.05).  Indacaterol, salmeterol and vilanterol bind to an addnl. site in human lung parenchyma membranes that is distinct from the β2-AR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGryywv3XAzCzLVg90H21EOLACvtfcHk0ljSDtd6PVyZPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlejt7vJ&md5=0e090c02447be8d16ff0742fe1ad16f7</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.4236%2Fpp.2014.51006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4236%252Fpp.2014.51006%26sid%3Dliteratum%253Aachs%26aulast%3DSlack%26aufirst%3DR.%2BJ.%26atitle%3DEvidence%2520for%2520a%2520non-%25CE%25B22-adrenoceptor%2520binding%2520site%2520in%2520human%2520lung%2520tissue%2520for%2520a%2520subset%2520of%2520%25CE%25B22-adrenoceptor%2520agonists%26jtitle%3DPharmacol.%2520Pharm.%26date%3D2014%26volume%3D5%26spage%3D30%26epage%3D36%26doi%3D10.4236%2Fpp.2014.51006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Ochsner, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jaekel, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mutz, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anderson, G. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">John, E.</span><span> </span><span class="NLM_article-title">Comparative biophysical analysis of the interaction of bronchodilating β2-adrenoceptor agonists with lipid membranes</span> <span class="citation_source-journal">Eur. J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">34</span><span class="NLM_x">, </span> <span class="NLM_fpage">451</span><span class="NLM_x">–</span> <span class="NLM_lpage">462</span><span class="refDoi"> DOI: 10.1016/S0223-5234(99)80095-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1016%2FS0223-5234%2899%2980095-6" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=1999&pages=451-462&author=M.+Ochsnerauthor=K.+Jaekelauthor=M.+Mutzauthor=G.+P.+Andersonauthor=E.+John&title=Comparative+biophysical+analysis+of+the+interaction+of+bronchodilating+%CE%B22-adrenoceptor+agonists+with+lipid+membranes&doi=10.1016%2FS0223-5234%2899%2980095-6"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2FS0223-5234%2899%2980095-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0223-5234%252899%252980095-6%26sid%3Dliteratum%253Aachs%26aulast%3DOchsner%26aufirst%3DM.%26aulast%3DJaekel%26aufirst%3DK.%26aulast%3DMutz%26aufirst%3DM.%26aulast%3DAnderson%26aufirst%3DG.%2BP.%26aulast%3DJohn%26aufirst%3DE.%26atitle%3DComparative%2520biophysical%2520analysis%2520of%2520the%2520interaction%2520of%2520bronchodilating%2520%25CE%25B22-adrenoceptor%2520agonists%2520with%2520lipid%2520membranes%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D1999%26volume%3D34%26spage%3D451%26epage%3D462%26doi%3D10.1016%2FS0223-5234%2899%2980095-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Bergendal, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lindén, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skoogh, B. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerspacher, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anderson, G. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Löfdahl, C. G.</span><span> </span><span class="NLM_article-title">Extent of salmeterol-mediated reassertion of relaxation in guinea-pig trachea pretreated with aliphatic side chain structural analogues</span> <span class="citation_source-journal">Br. J. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">117</span><span class="NLM_x">, </span> <span class="NLM_fpage">1009</span><span class="NLM_x">–</span> <span class="NLM_lpage">1015</span><span class="refDoi"> DOI: 10.1111/j.1476-5381.1996.tb16690.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1111%2Fj.1476-5381.1996.tb16690.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=8882590" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADyaK28XitVentLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=117&publication_year=1996&pages=1009-1015&author=A.+Bergendalauthor=A.+Lind%C3%A9nauthor=B.+E.+Skooghauthor=M.+Gerspacherauthor=G.+P.+Andersonauthor=C.+G.+L%C3%B6fdahl&title=Extent+of+salmeterol-mediated+reassertion+of+relaxation+in+guinea-pig+trachea+pretreated+with+aliphatic+side+chain+structural+analogues&doi=10.1111%2Fj.1476-5381.1996.tb16690.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Extent of salmeterol-mediated reassertion of relaxation in guinea pig trachea pretreated with aliphatic side chain structural analogs</span></div><div class="casAuthors">Bergendal, A.; Linden, A.; Skoogh, B.-E.; Gerspacher, M.; Anderson, G. P.; Loefdahl, C.-G.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">117</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1009-15</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">0007-1188</span>.
    
            (<span class="NLM_cas:orgname">Stockton</span>)
        </div><div class="casAbstract">Salmeterol is a potent, selective and long acting β2-adrenoceptor agonist.  In vitro, salmeterol exerts 'reassertion' relaxation of airways smooth muscle.  Reassertion relaxation refers to the capacity of salmeterol to cause repeated functional antagonism of induced contraction when airway smooth muscle is intermittently exposed to, then washed free from, β-adrenoceptor antagonists such as sotalol.  The mechanism(s) underlying reassertion relaxation are unknown but may relate to high affinity binding of the long aliph. side chain of salmeterol to an accessory site, distinct from the agonist recognition site, in or near the β2-adrenoceptor (exosite binding hypothesis).  To test the exosite hypothesis, three pure analogs of salmeterol, each exactly preserving the mol. structure of the aliph. side chain but with zero or low efficacy at the β2-adrenoceptor were synthesized.  The effect of pre-incubating guinea-pig tracheal smooth muscle with these analogs on salmeterol-induced reassertion relaxation was detd.  Computer Assisted Mol. Modeling of these mols. revealed that each of them exactly preserved the low energy linear conformation of the aliph. side chain of salmeterol.  Measurement of lipophilicity (octanol:water partition coeff.; log P) and direct partition into synthetic membranes (membrane partition coeff.; Kpmem) showed that all compds. had high affinity for lipids and membranes.  In particular the biophys. properties of CGP 59162 (log P 1.89, Kpmem 16500) were very similar to salmeterol (log P 1.73, Kpmem 16800).  Two of the analogs, CGP 54103 and D 2543 (1 μM), which are structural mimics of the sie chain of salmeterol, differing slightly in their length, did not prevent either the initial relaxation induced by salmeterol (0.1 μM) or the reassertion relaxation; however, it was not possible to det. whether either of these mols. occupied the β2-adrenoceptor.  The third analog, CGP 59162, which has the substituents on the active saligenin head group of salmeterol in transposed positions, itself exerted a weak β2-adrenoceptor-mediated relaxation antagonized by ICI 118551 (β2-selective antagonist) but not CGP 20712 (β1-selective antagonist) and, at higher concns. CGP 59162 caused reassertion relaxation suggesting that it may occupy and activate the β2-adrenoceptor in a manner analogous to salmeterol.  CGP 59162, at concns. up to ten fold molar excess, did not prevent or reduce salmeterol-induced reassertion relaxation.  In conclusion these data are not consistent with the existence of a distinct 'exosite' recognizing the aliph. side chain of salmeterol mediating reassertion.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIwfxNXAL4k7Vg90H21EOLACvtfcHk0ljSDtd6PVyZPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XitVentLg%253D&md5=84fc71792ca22dcb18f6b0fda3b4c7c2</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1111%2Fj.1476-5381.1996.tb16690.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1476-5381.1996.tb16690.x%26sid%3Dliteratum%253Aachs%26aulast%3DBergendal%26aufirst%3DA.%26aulast%3DLind%25C3%25A9n%26aufirst%3DA.%26aulast%3DSkoogh%26aufirst%3DB.%2BE.%26aulast%3DGerspacher%26aufirst%3DM.%26aulast%3DAnderson%26aufirst%3DG.%2BP.%26aulast%3DL%25C3%25B6fdahl%26aufirst%3DC.%2BG.%26atitle%3DExtent%2520of%2520salmeterol-mediated%2520reassertion%2520of%2520relaxation%2520in%2520guinea-pig%2520trachea%2520pretreated%2520with%2520aliphatic%2520side%2520chain%2520structural%2520analogues%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D1996%26volume%3D117%26spage%3D1009%26epage%3D1015%26doi%3D10.1111%2Fj.1476-5381.1996.tb16690.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Jeppsson, A. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Löfdahl, C. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waldeck, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Widmark, E.</span><span> </span><span class="NLM_article-title">On the predictive value of experiments in vitro in the evaluation of the effect duration of bronchodilator drugs for local administration</span> <span class="citation_source-journal">Pulm. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1989</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">81</span><span class="NLM_x">–</span> <span class="NLM_lpage">85</span><span class="refDoi"> DOI: 10.1016/0952-0600(89)90028-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1016%2F0952-0600%2889%2990028-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=2577691" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A280%3ADyaK383hs1ClsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=1989&pages=81-85&author=A.+B.+Jeppssonauthor=C.+G.+L%C3%B6fdahlauthor=B.+Waldeckauthor=E.+Widmark&title=On+the+predictive+value+of+experiments+in+vitro+in+the+evaluation+of+the+effect+duration+of+bronchodilator+drugs+for+local+administration&doi=10.1016%2F0952-0600%2889%2990028-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">On the predictive value of experiments in vitro in the evaluation of the effect duration of bronchodilator drugs for local administration</span></div><div class="casAuthors">Jeppsson A B; Lofdahl C G; Waldeck B; Widmark E</div><div class="citationInfo"><span class="NLM_cas:title">Pulmonary pharmacology</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">81-5</span>
        ISSN:<span class="NLM_cas:issn">0952-0600</span>.
    </div><div class="casAbstract">Six bronchodilating beta-adrenoceptor agonists, clinically documented with respect to the duration of action after inhalation, were included in this study in vitro on the guinea-pig trachea.  Relaxation of carbachol contracted trachea strip preparations and inhibition of contraction of a vagus nerve-tracheal tube preparation were measured.  The relaxing effects of salbutamol and fenoterol (both with relatively short duration in man) were rapid in onset and easily reversed by washing in a drug-free medium.  The relaxation by salmeterol (long duration) and D2343 (intermediate duration) developed more slowly, resisted washing but was reversed by propranolol.  Formoterol (long duration) and salmefamol (intermediate duration) showed properties between these two extremes.  All test compounds inhibited the vagally-induced contractions of tracheal concentration dependently.  The EC50 values for the hydrophilic compounds salbutamol and fenoterol were higher with intra- as compared with extratracheal administration.  For the more lipophilic compounds formoterol, salmefamol, salmeterol and D2343, this difference was less pronounced.  A high lipophilicity and a retention by the tissue in vitro of a beta-adrenoceptor agonist may be factors contributing to a long effect duration after inhalation but a further selection has to be made in vivo since metabolic and circulatory effects may influence the effect kinetics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR6jE8Lji0hK_EeosoOnpOLfW6udTcc2eaPPDEBo2HCq7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK383hs1ClsQ%253D%253D&md5=8a4d18187e98c34daf7f2735b922c727</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2F0952-0600%2889%2990028-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0952-0600%252889%252990028-8%26sid%3Dliteratum%253Aachs%26aulast%3DJeppsson%26aufirst%3DA.%2BB.%26aulast%3DL%25C3%25B6fdahl%26aufirst%3DC.%2BG.%26aulast%3DWaldeck%26aufirst%3DB.%26aulast%3DWidmark%26aufirst%3DE.%26atitle%3DOn%2520the%2520predictive%2520value%2520of%2520experiments%2520in%2520vitro%2520in%2520the%2520evaluation%2520of%2520the%2520effect%2520duration%2520of%2520bronchodilator%2520drugs%2520for%2520local%2520administration%26jtitle%3DPulm.%2520Pharmacol.%26date%3D1989%26volume%3D2%26spage%3D81%26epage%3D85%26doi%3D10.1016%2F0952-0600%2889%2990028-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Löfdahl, C.-G.</span><span> </span><span class="NLM_article-title">Basic pharmacology of new long-acting sympathomimetics</span> <span class="citation_source-journal">Lung</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1990</span><span class="NLM_x">, </span> <span class="NLM_volume">168</span><span class="NLM_x">, </span> <span class="NLM_fpage">18</span><span class="NLM_x">–</span> <span class="NLM_lpage">21</span><span class="refDoi"> DOI: 10.1007/BF02718108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1007%2FBF02718108" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=168&publication_year=1990&pages=18-21&author=C.-G.+L%C3%B6fdahl&title=Basic+pharmacology+of+new+long-acting+sympathomimetics&doi=10.1007%2FBF02718108"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1007%2FBF02718108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF02718108%26sid%3Dliteratum%253Aachs%26aulast%3DL%25C3%25B6fdahl%26aufirst%3DC.-G.%26atitle%3DBasic%2520pharmacology%2520of%2520new%2520long-acting%2520sympathomimetics%26jtitle%3DLung%26date%3D1990%26volume%3D168%26spage%3D18%26epage%3D21%26doi%3D10.1007%2FBF02718108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Valko, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Du, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bevan, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reynolds, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abraham, M. H.</span><span> </span><span class="NLM_article-title">Rapid-gradient HPLC method for measuring drug interactions with immobilized artificial membrane: Comparison with other lipophilicity measures</span> <span class="citation_source-journal">J. Pharm. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">89</span><span class="NLM_x">, </span> <span class="NLM_fpage">1085</span><span class="NLM_x">–</span> <span class="NLM_lpage">1096</span><span class="refDoi"> DOI: 10.1002/1520-6017(200008)89:8<1085::AID-JPS13>3.0.CO;2-N</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1002%2F1520-6017%28200008%2989%3A8%3C1085%3A%3AAID-JPS13%3E3.0.CO%3B2-N" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=89&publication_year=2000&pages=1085-1096&author=K.+Valkoauthor=C.+M.+Duauthor=C.+D.+Bevanauthor=D.+P.+Reynoldsauthor=M.+H.+Abraham&title=Rapid-gradient+HPLC+method+for+measuring+drug+interactions+with+immobilized+artificial+membrane%3A+Comparison+with+other+lipophilicity+measures&doi=10.1002%2F1520-6017%28200008%2989%3A8%3C1085%3A%3AAID-JPS13%3E3.0.CO%3B2-N"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1002%2F1520-6017%28200008%2989%3A8%3C1085%3A%3AAID-JPS13%3E3.0.CO%3B2-N&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F1520-6017%2528200008%252989%253A8%253C1085%253A%253AAID-JPS13%253E3.0.CO%253B2-N%26sid%3Dliteratum%253Aachs%26aulast%3DValko%26aufirst%3DK.%26aulast%3DDu%26aufirst%3DC.%2BM.%26aulast%3DBevan%26aufirst%3DC.%2BD.%26aulast%3DReynolds%26aufirst%3DD.%2BP.%26aulast%3DAbraham%26aufirst%3DM.%2BH.%26atitle%3DRapid-gradient%2520HPLC%2520method%2520for%2520measuring%2520drug%2520interactions%2520with%2520immobilized%2520artificial%2520membrane%253A%2520Comparison%2520with%2520other%2520lipophilicity%2520measures%26jtitle%3DJ.%2520Pharm.%2520Sci.%26date%3D2000%26volume%3D89%26spage%3D1085%26epage%3D1096%26doi%3D10.1002%2F1520-6017%28200008%2989%3A8%3C1085%3A%3AAID-JPS13%3E3.0.CO%3B2-N" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Abdiche, Y. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Myszka, D. G.</span><span> </span><span class="NLM_article-title">Probing the mechanism of drug/lipid membrane interactions using Biacore</span> <span class="citation_source-journal">Anal. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">328</span><span class="NLM_x">, </span> <span class="NLM_fpage">233</span><span class="NLM_x">–</span> <span class="NLM_lpage">243</span><span class="refDoi"> DOI: 10.1016/j.ab.2004.01.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1016%2Fj.ab.2004.01.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=15113702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjsVWmurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=328&publication_year=2004&pages=233-243&author=Y.+N.+Abdicheauthor=D.+G.+Myszka&title=Probing+the+mechanism+of+drug%2Flipid+membrane+interactions+using+Biacore&doi=10.1016%2Fj.ab.2004.01.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Probing the mechanism of drug/lipid membrane interactions using Biacore</span></div><div class="casAuthors">Abdiche, Yasmina N.; Myszka, David G.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">328</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">233-243</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science</span>)
        </div><div class="casAbstract">Assay conditions were established to screen a panel of drugs for binding to liposome surfaces using a surface plasmon resonance (SPR) biosensor.  Drugs were found to bind negligibly or reversibly or were retained on the liposome surface.  Cationic amphiphilic drugs fell into the last class and correlated with drugs that induce phospholipidosis in vivo.  To a first approxn., a single-site model yielded apparent binding affinities that adequately described a drug's dose-dependent binding to liposome surfaces.  Affinities ranged at least 1000-fold within the drug panel.  A liposome's drug-binding capacity and affinity depended on both the lipid headgroup and the drug's structure.  Although a drug's charge state generally dominated whether or not it remained bound to the liposome, subtle structural differences between members of certain drug families led to them having widely differing binding affinities.  A comparison between the dissocn. of drugs from liposome surfaces by Biacore and the lipid retention measurements detd. by a parallel artificial membrane permeability assay was drawn.  The results from this study demonstrate the potential of using SPR-based assays to characterize drug/liposome-binding interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBtD87YZF42bVg90H21EOLACvtfcHk0liRbxdYaA3wpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjsVWmurc%253D&md5=e57a50d7112633499a19ea91b78b1ca1</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.ab.2004.01.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ab.2004.01.018%26sid%3Dliteratum%253Aachs%26aulast%3DAbdiche%26aufirst%3DY.%2BN.%26aulast%3DMyszka%26aufirst%3DD.%2BG.%26atitle%3DProbing%2520the%2520mechanism%2520of%2520drug%252Flipid%2520membrane%2520interactions%2520using%2520Biacore%26jtitle%3DAnal.%2520Biochem.%26date%3D2004%26volume%3D328%26spage%3D233%26epage%3D243%26doi%3D10.1016%2Fj.ab.2004.01.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Jiang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reilly, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Everatt, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Briard, E.</span><span> </span><span class="NLM_article-title">A rapid vesicle electrokinetic chromatography method for the in vitro prediction of non-specific binding for potential PET ligands</span> <span class="citation_source-journal">J. Pharm. Biomed. Anal.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">722</span><span class="NLM_x">–</span> <span class="NLM_lpage">729</span><span class="refDoi"> DOI: 10.1016/j.jpba.2010.11.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1016%2Fj.jpba.2010.11.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=21126842" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsF2mtrrO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=722-729&author=Z.+Jiangauthor=J.+Reillyauthor=B.+Everattauthor=E.+Briard&title=A+rapid+vesicle+electrokinetic+chromatography+method+for+the+in+vitro+prediction+of+non-specific+binding+for+potential+PET+ligands&doi=10.1016%2Fj.jpba.2010.11.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">A rapid vesicle electrokinetic chromatography method for the in vitro prediction of non-specific binding for potential PET ligands</span></div><div class="casAuthors">Jiang, Zhengjin; Reilly, John; Everatt, Brian; Briard, Emmanuelle</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmaceutical and Biomedical Analysis</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">722-729</span>CODEN:
                <span class="NLM_cas:coden">JPBADA</span>;
        ISSN:<span class="NLM_cas:issn">0731-7085</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">High non-specific binding (NSB) is one of the most common reasons for candidate failure in potential positron emission tomog. (PET) radiotracer development.  It is of interest to develop high throughput in vitro methods for predicting non-specific binding prior to radiolabeling, which would help guide radiotracer candidate selection and assist decision making in new radiotracer discovery.  We evaluated several electrokinetic chromatog. (EKC) systems to help identify PET ligands with low non-specific binding characteristics by mimicking the ligand-brain tissue interaction.  The measured retention factors of tracers in clin. use or terminated candidates within AOT vesicle EKC systems were compared with literature in vitro or in vivo NSB data.  We conclude that there is a statistical correlation between the chromatog. retention parameters of tested drugs and their NSB.  The AOT vesicle EKC method can provide NSB in vitro trend anal. for a large no. of drug candidates early in the novel radiotracer discovery process with minimal resources.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphoegUxlpyHrVg90H21EOLACvtfcHk0liRbxdYaA3wpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsF2mtrrO&md5=dd95312fc52995ed2233f2d9ccf7012a</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.jpba.2010.11.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jpba.2010.11.004%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DZ.%26aulast%3DReilly%26aufirst%3DJ.%26aulast%3DEveratt%26aufirst%3DB.%26aulast%3DBriard%26aufirst%3DE.%26atitle%3DA%2520rapid%2520vesicle%2520electrokinetic%2520chromatography%2520method%2520for%2520the%2520in%2520vitro%2520prediction%2520of%2520non-specific%2520binding%2520for%2520potential%2520PET%2520ligands%26jtitle%3DJ.%2520Pharm.%2520Biomed.%2520Anal.%26date%3D2011%26volume%3D54%26spage%3D722%26epage%3D729%26doi%3D10.1016%2Fj.jpba.2010.11.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Mason, R. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rhodes, D. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herbette, L. G.</span><span> </span><span class="NLM_article-title">Reevaluating equilibrium and kinetic binding parameters for lipophilic drugs based on a structural model for drug interactions with biological membranes</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1991</span><span class="NLM_x">, </span> <span class="NLM_volume">34</span><span class="NLM_x">, </span> <span class="NLM_fpage">869</span><span class="NLM_x">–</span> <span class="NLM_lpage">877</span><span class="refDoi"> DOI: 10.1021/jm00107a001</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00107a001" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=1991&pages=869-877&author=R.+P.+Masonauthor=D.+G.+Rhodesauthor=L.+G.+Herbette&title=Reevaluating+equilibrium+and+kinetic+binding+parameters+for+lipophilic+drugs+based+on+a+structural+model+for+drug+interactions+with+biological+membranes&doi=10.1021%2Fjm00107a001"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fjm00107a001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00107a001%26sid%3Dliteratum%253Aachs%26aulast%3DMason%26aufirst%3DR.%2BP.%26aulast%3DRhodes%26aufirst%3DD.%2BG.%26aulast%3DHerbette%26aufirst%3DL.%2BG.%26atitle%3DReevaluating%2520equilibrium%2520and%2520kinetic%2520binding%2520parameters%2520for%2520lipophilic%2520drugs%2520based%2520on%2520a%2520structural%2520model%2520for%2520drug%2520interactions%2520with%2520biological%2520membranes%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1991%26volume%3D34%26spage%3D869%26epage%3D877%26doi%3D10.1021%2Fjm00107a001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Castanho, M. A. R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fernandes, M. X.</span><span> </span><span class="NLM_article-title">Lipid membrane-induced optimization for ligand–receptor docking: recent tools and insights for the “membrane catalysis” model</span> <span class="citation_source-journal">Eur. Biophys. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">35</span><span class="NLM_x">, </span> <span class="NLM_fpage">92</span><span class="NLM_x">–</span> <span class="NLM_lpage">103</span><span class="refDoi"> DOI: 10.1007/s00249-005-0007-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1007%2Fs00249-005-0007-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=16217647" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BD28XpsFKh" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2006&pages=92-103&author=M.+A.+R.+B.+Castanhoauthor=M.+X.+Fernandes&title=Lipid+membrane-induced+optimization+for+ligand%E2%80%93receptor+docking%3A+recent+tools+and+insights+for+the+%E2%80%9Cmembrane+catalysis%E2%80%9D+model&doi=10.1007%2Fs00249-005-0007-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Lipid membrane-induced optimization for ligand-receptor docking: recent tools and insights for the "membrane catalysis" model</span></div><div class="casAuthors">Castanho, Miguel A. R. B.; Fernandes, Miguel X.</div><div class="citationInfo"><span class="NLM_cas:title">European Biophysics Journal</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">92-103</span>CODEN:
                <span class="NLM_cas:coden">EBJOE8</span>;
        ISSN:<span class="NLM_cas:issn">0175-7571</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  Cells in living organisms are regulated by chem. and phys. stimuli from their environment.  Often, ligands interact with membrane receptors to trigger responses and Sargent and Schwyzer conceived a model to describe this process, "membrane catalysis".  There is a notion that the phys. organization of membranes can control the response of cells by speeding up reactions.  The authors revisit the "membrane catalysis" model in the light of recent tech., methodol. and theor. advances and how they can be exploited to highlight the details of membrane mediated ligand-receptor interactions.  The authors examine the possible effects that ligand concn. causes in the membrane catalysis and focus the authors' attention in techniques used to det. the partition const.  The hypothetical diffusional advantage assocd. with membrane catalysis is discussed and the applicability of existing models is assessed.  The role of in-depth location and orientation of ligands is explored emphasizing the contribution of new anal. methods and spectroscopic techniques.  Results suggest that membranes can optimize the interaction between ligands and receptors through several different effects but the relative contribution of each must be carefully investigated.  The authors certainly hope that the conjugation of the methodol. and tech. advances here reported will revive the interest in the membrane catalysis model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-spWQhjU7D7Vg90H21EOLACvtfcHk0lgDEBlu0EtsHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XpsFKh&md5=bf2a385e80f2d655cae22a1aad370f7a</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1007%2Fs00249-005-0007-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00249-005-0007-9%26sid%3Dliteratum%253Aachs%26aulast%3DCastanho%26aufirst%3DM.%2BA.%2BR.%2BB.%26aulast%3DFernandes%26aufirst%3DM.%2BX.%26atitle%3DLipid%2520membrane-induced%2520optimization%2520for%2520ligand%25E2%2580%2593receptor%2520docking%253A%2520recent%2520tools%2520and%2520insights%2520for%2520the%2520%25E2%2580%259Cmembrane%2520catalysis%25E2%2580%259D%2520model%26jtitle%3DEur.%2520Biophys.%2520J.%26date%3D2006%26volume%3D35%26spage%3D92%26epage%3D103%26doi%3D10.1007%2Fs00249-005-0007-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Velez-Vega, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McKay, D. J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aravamuthan, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pearlstein, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duca, J. S.</span><span> </span><span class="NLM_article-title">Time-averaged distributions of solute and solvent motions: exploring proton wires of GFP and PfM2DH</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">3344</span><span class="NLM_x">–</span> <span class="NLM_lpage">3361</span><span class="refDoi"> DOI: 10.1021/ci500571h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci500571h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFGktbbO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=3344-3361&author=C.+Velez-Vegaauthor=D.+J.+J.+McKayauthor=V.+Aravamuthanauthor=R.+Pearlsteinauthor=J.+S.+Duca&title=Time-averaged+distributions+of+solute+and+solvent+motions%3A+exploring+proton+wires+of+GFP+and+PfM2DH&doi=10.1021%2Fci500571h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Time-Averaged Distributions of Solute and Solvent Motions: Exploring Proton Wires of GFP and PfM2DH</span></div><div class="casAuthors">Velez-Vega, Camilo; McKay, Daniel J. J.; Aravamuthan, Vibhas; Pearlstein, Robert; Duca, Jose S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3344-3361</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Proton translocation pathways of selected variants of the green fluorescent protein (GFP) and Pseudomonas fluorescens mannitol 2-dehydrogenase (PfM2DH) were studied via an explicit solvent mol. dynamics-based anal. protocol that allows for direct quant. relation between a crystal structure and its time-averaged solute-solvent structure obtained from simulation.  The authors' study of GFP is in good agreement with previous research suggesting that the proton released from the chromophore upon photoexcitation can diffuse through an extended internal hydrogen bonding network that allows for the proton to exit to bulk or be recaptured by the anionic chromophore.  Conversely for PfM2DH, the authors identified the most probable ionization states of key residues along the proton escape channel from the catalytic site to bulk solvent, wherein the solute and high-d. solvent crystal structures of binary and ternary complexes were properly reproduced.  Furthermore, the authors proposed a plausible mechanism for this proton translocation process that is consistent with the state-dependent structural shifts obsd. in the authors' anal.  The time-averaged structures generated from the authors' analyses facilitate validation of MD simulation results and provide a comprehensive profile of the dynamic all-occupancy solvation network within and around a flexible solute, from which detailed hydrogen-bonding networks can be inferred.  In this way, potential drawbacks arising from the elucidation of these networks by examn. of static crystal structures or via alternate rigid-protein solvation anal. procedures can be overcome.  Complementary studies aimed at the effective use of the authors' methodol. for alternate implementations (e.g., ligand design) are currently underway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpub7nL0UsqGrVg90H21EOLACvtfcHk0lgDEBlu0EtsHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFGktbbO&md5=d34a5696e577371ff4db5630923dedb0</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fci500571h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci500571h%26sid%3Dliteratum%253Aachs%26aulast%3DVelez-Vega%26aufirst%3DC.%26aulast%3DMcKay%26aufirst%3DD.%2BJ.%2BJ.%26aulast%3DAravamuthan%26aufirst%3DV.%26aulast%3DPearlstein%26aufirst%3DR.%26aulast%3DDuca%26aufirst%3DJ.%2BS.%26atitle%3DTime-averaged%2520distributions%2520of%2520solute%2520and%2520solvent%2520motions%253A%2520exploring%2520proton%2520wires%2520of%2520GFP%2520and%2520PfM2DH%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D3344%26epage%3D3361%26doi%3D10.1021%2Fci500571h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Velez-Vega, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McKay, D. J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kurtzman, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aravamuthan, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pearlstein, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duca, J. S.</span><span> </span><span class="NLM_article-title">Estimation of solvation entropy and enthalpy via analysis of water oxygen–hydrogen correlations</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">5090</span><span class="NLM_x">–</span> <span class="NLM_lpage">5102</span><span class="refDoi"> DOI: 10.1021/acs.jctc.5b00439</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jctc.5b00439" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1Wqs77F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=5090-5102&author=C.+Velez-Vegaauthor=D.+J.+J.+McKayauthor=T.+Kurtzmanauthor=V.+Aravamuthanauthor=R.+A.+Pearlsteinauthor=J.+S.+Duca&title=Estimation+of+solvation+entropy+and+enthalpy+via+analysis+of+water+oxygen%E2%80%93hydrogen+correlations&doi=10.1021%2Facs.jctc.5b00439"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Estimation of Solvation Entropy and Enthalpy via Analysis of Water Oxygen-Hydrogen Correlations</span></div><div class="casAuthors">Velez-Vega, Camilo; McKay, Daniel J. J.; Kurtzman, Tom; Aravamuthan, Vibhas; Pearlstein, Robert A.; Duca, Jose S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">5090-5102</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A statistical-mech. framework for estn. of solvation entropies and enthalpies is proposed, which is based on the anal. of water as a mixt. of correlated water oxygens and water hydrogens.  Entropic contributions of increasing order are cast in terms of a Mutual Information Expansion that is evaluated to pairwise interactions.  In turn, the enthalpy is computed directly from a distance-based hydrogen bonding energy algorithm.  The resulting expressions are employed for grid-based analyses of Mol. Dynamics simulations.  In this first assessment of the methodol., we obtained global ests. of the excess entropy and enthalpy of water that are in good agreement with expt. and examd. the method's ability to enable detailed elucidation of solvation thermodn. structures, which can provide valuable knowledge toward mol. design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcC8EPpRstJbVg90H21EOLACvtfcHk0lhAfwHSz8bOOQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1Wqs77F&md5=a53e1cb93683f72291d013f77c181d2a</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Facs.jctc.5b00439&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jctc.5b00439%26sid%3Dliteratum%253Aachs%26aulast%3DVelez-Vega%26aufirst%3DC.%26aulast%3DMcKay%26aufirst%3DD.%2BJ.%2BJ.%26aulast%3DKurtzman%26aufirst%3DT.%26aulast%3DAravamuthan%26aufirst%3DV.%26aulast%3DPearlstein%26aufirst%3DR.%2BA.%26aulast%3DDuca%26aufirst%3DJ.%2BS.%26atitle%3DEstimation%2520of%2520solvation%2520entropy%2520and%2520enthalpy%2520via%2520analysis%2520of%2520water%2520oxygen%25E2%2580%2593hydrogen%2520correlations%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2015%26volume%3D11%26spage%3D5090%26epage%3D5102%26doi%3D10.1021%2Facs.jctc.5b00439" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Leeson, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Springthorpe, B.</span><span> </span><span class="NLM_article-title">The influence of drug-like concepts on decision-making in medicinal chemistry</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">881</span><span class="NLM_x">–</span> <span class="NLM_lpage">890</span><span class="refDoi"> DOI: 10.1038/nrd2445</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1038%2Fnrd2445" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=17971784" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BD2sXht1els7rL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2007&pages=881-890&author=P.+D.+Leesonauthor=B.+Springthorpe&title=The+influence+of+drug-like+concepts+on+decision-making+in+medicinal+chemistry&doi=10.1038%2Fnrd2445"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">The influence of drug-like concepts on decision-making in medicinal chemistry</span></div><div class="casAuthors">Leeson, Paul D.; Springthorpe, Brian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">881-890</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Despite the wide acceptance of drug-like principles such as the 'rule of five', this anal. of mols. currently being synthesized in leading pharmaceutical companies reveals that their phys. properties differ significantly from those of recently discovered oral drugs.  The marked increase in lipophilicity in particular could increase the likelihood of attrition in drug development.  The application of guidelines linked to the concept of drug-likeness, such as the 'rule of five', has gained wide acceptance as an approach to reduce attrition in drug discovery and development.  However, despite this acceptance, anal. of recent trends reveals that the phys. properties of mols. that are currently being synthesized in leading drug discovery companies differ significantly from those of recently discovered oral drugs and compds. in clin. development.  The consequences of the marked increase in lipophilicity - the most important drug-like phys. property - include a greater likelihood of lack of selectivity and attrition in drug development.  Tackling the threat of compd.-related toxicol. attrition needs to move to the mainstream of medicinal chem. decision-making.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTUeonLD6iebVg90H21EOLACvtfcHk0lhAfwHSz8bOOQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXht1els7rL&md5=5665a9e2dc266a1ee096e20625757ef7</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1038%2Fnrd2445&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2445%26sid%3Dliteratum%253Aachs%26aulast%3DLeeson%26aufirst%3DP.%2BD.%26aulast%3DSpringthorpe%26aufirst%3DB.%26atitle%3DThe%2520influence%2520of%2520drug-like%2520concepts%2520on%2520decision-making%2520in%2520medicinal%2520chemistry%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2007%26volume%3D6%26spage%3D881%26epage%3D890%26doi%3D10.1038%2Fnrd2445" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Katritch, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reynolds, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cherezov, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hanson, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, C. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yeager, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abagyan, R.</span><span> </span><span class="NLM_article-title">Analysis of full and partial agonists binding to β(2)-adrenergic receptor suggests a role of transmembrane helix V in agonist-specific conformational changes</span> <span class="citation_source-journal">J. Mol. Recognit.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">307</span><span class="NLM_x">–</span> <span class="NLM_lpage">318</span><span class="refDoi"> DOI: 10.1002/jmr.949</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1002%2Fjmr.949" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=19353579" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BD1MXnslGis7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2009&pages=307-318&author=V.+Katritchauthor=K.+A.+Reynoldsauthor=V.+Cherezovauthor=M.+A.+Hansonauthor=C.+B.+Rothauthor=M.+Yeagerauthor=R.+Abagyan&title=Analysis+of+full+and+partial+agonists+binding+to+%CE%B2%282%29-adrenergic+receptor+suggests+a+role+of+transmembrane+helix+V+in+agonist-specific+conformational+changes&doi=10.1002%2Fjmr.949"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Analysis of full and partial agonists binding to β2-adrenergic receptor suggests a role of transmembrane helix V in agonist-specific conformational changes</span></div><div class="casAuthors">Katritch, Vsevolod; Reynolds, Kimberly A.; Cherezov, Vadim; Hanson, Michael A.; Roth, Christopher B.; Yeager, Mark; Abagyan, Ruben</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Recognition</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">307-318</span>CODEN:
                <span class="NLM_cas:coden">JMORE4</span>;
        ISSN:<span class="NLM_cas:issn">0952-3499</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons Ltd.</span>)
        </div><div class="casAbstract">The 2.4 Å crystal structure of the β2-adrenergic receptor (β2AR) in complex with the high-affinity inverse agonist (-)-carazolol provides a detailed structural framework for the anal. of ligand recognition by adrenergic receptors.  Insights into agonist binding and the corresponding conformational changes triggering G-protein coupled receptor (GPCR) activation mechanism are of special interest.  While the carazolol pocket captured in the β2AR crystal structure accommodates (-)-isoproterenol and other agonists without steric clashes, a finite movement of the flexible extracellular part of TM-V helix (TM-Ve) obtained by receptor optimization in the presence of docked ligand can further improve the calcd. binding affinities for agonist compds.  Tilting of TM-Ve towards the receptor axis provides a more complete description of polar receptor-ligand interactions for full and partial agonists, by enabling optimal engagement of agonists with two exptl. identified anchor sites, formed by Asp 113/Asn 312 and Ser 203/Ser 204/Ser 207 side chains.  Further, receptor models incorporating a flexible TM-V backbone allow reliable prediction of binding affinities for a set of diverse ligands, suggesting potential utility of this approach to design of effective and subtype-specific agonists for adrenergic receptors.  Systematic differences in capacity of partial, full and inverse agonists to induce TM-V helix tilt in the β2AR model suggest potential role of TM-V as a conformational "rheostat" involved in the whole spectrum of β2AR responses to small mol. signals.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjjBCjcVJWTrVg90H21EOLACvtfcHk0lhAfwHSz8bOOQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXnslGis7k%253D&md5=01e2c841b200697382e551b3cf104451</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1002%2Fjmr.949&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjmr.949%26sid%3Dliteratum%253Aachs%26aulast%3DKatritch%26aufirst%3DV.%26aulast%3DReynolds%26aufirst%3DK.%2BA.%26aulast%3DCherezov%26aufirst%3DV.%26aulast%3DHanson%26aufirst%3DM.%2BA.%26aulast%3DRoth%26aufirst%3DC.%2BB.%26aulast%3DYeager%26aufirst%3DM.%26aulast%3DAbagyan%26aufirst%3DR.%26atitle%3DAnalysis%2520of%2520full%2520and%2520partial%2520agonists%2520binding%2520to%2520%25CE%25B2%25282%2529-adrenergic%2520receptor%2520suggests%2520a%2520role%2520of%2520transmembrane%2520helix%2520V%2520in%2520agonist-specific%2520conformational%2520changes%26jtitle%3DJ.%2520Mol.%2520Recognit.%26date%3D2009%26volume%3D22%26spage%3D307%26epage%3D318%26doi%3D10.1002%2Fjmr.949" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Pearlstein, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sherman, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abel, R.</span><span> </span><span class="NLM_article-title">Contributions of water transfer energy to protein-ligand association and dissociation barriers: WaterMap analysis of a series of p38α MAP kinase inhibitors</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">81</span><span class="NLM_x">, </span> <span class="NLM_fpage">1509</span><span class="NLM_x">–</span> <span class="NLM_lpage">1526</span><span class="refDoi"> DOI: 10.1002/prot.24276</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1002%2Fprot.24276" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=23468227" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVaktbrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=2013&pages=1509-1526&author=R.+A.+Pearlsteinauthor=W.+Shermanauthor=R.+Abel&title=Contributions+of+water+transfer+energy+to+protein-ligand+association+and+dissociation+barriers%3A+WaterMap+analysis+of+a+series+of+p38%CE%B1+MAP+kinase+inhibitors&doi=10.1002%2Fprot.24276"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Contributions of water transfer energy to protein-ligand association and dissociation barriers: Watermap analysis of a series of p38α MAP kinase inhibitors</span></div><div class="casAuthors">Pearlstein, Robert A.; Sherman, Woody; Abel, Robert</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Bioinformatics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">81</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1509-1526</span>CODEN:
                <span class="NLM_cas:coden">PSFBAF</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">In our previous work, we proposed that desolvation and resolvation of the binding sites of proteins can serve as the slowest steps during ligand assocn. and dissocn., resp., and tested this hypothesis on two protein-ligand systems with known binding kinetics behavior.  In the present work, we test this hypothesis on another kinetically-detd. protein-ligand system-that of p38α and eight Type II BIRB 796 inhibitor analogs.  The kon values among the inhibitor analogs are narrowly distributed (104 ≤ kon ≤ 105 M-1s-1), suggesting a common rate-detg. step, whereas the koff values are widely distributed (10-1 ≤ koff ≤ 10-6 s-1), suggesting a spectrum of rate-detg. steps.  We calcd. the solvation properties of the DFG-out protein conformation using an explicit solvent mol. dynamics simulation and thermodn. anal. method implemented in WaterMap to predict the enthalpic and entropic costs of water transfer to and from bulk solvent incurred upon assocn. and dissocn. of each inhibitor.  The results suggest that the rate-detg. step for assocn. consists of the transfer of a common set of enthalpically favorable solvating water mols. from the binding site to bulk solvent.  The rate-detg. step for inhibitor dissocn. consists of the transfer of water from bulk solvent to specific binding site positions that are unfavorably solvated in the apo protein, and evacuated during ligand assocn.  Different sets of unfavorable solvation are evacuated by each ligand, and the obsd. dissocn. barriers are qual. consistent with the calcd. solvation free energies of those sets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4gR0ZeehncLVg90H21EOLACvtfcHk0lhAfwHSz8bOOQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVaktbrK&md5=45e9cac74fbe0e13b0469b5ce9df622c</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1002%2Fprot.24276&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.24276%26sid%3Dliteratum%253Aachs%26aulast%3DPearlstein%26aufirst%3DR.%2BA.%26aulast%3DSherman%26aufirst%3DW.%26aulast%3DAbel%26aufirst%3DR.%26atitle%3DContributions%2520of%2520water%2520transfer%2520energy%2520to%2520protein-ligand%2520association%2520and%2520dissociation%2520barriers%253A%2520WaterMap%2520analysis%2520of%2520a%2520series%2520of%2520p38%25CE%25B1%2520MAP%2520kinase%2520inhibitors%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2013%26volume%3D81%26spage%3D1509%26epage%3D1526%26doi%3D10.1002%2Fprot.24276" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Pearlstein, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, Q.-Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yowe, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levell, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dale, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaushik, V. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Daniels, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hanrahan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sherman, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abel, R.</span><span> </span><span class="NLM_article-title">New hypotheses about the structure–function of proprotein convertase subtilisin/kexin type 9: analysis of the epidermal growth factor-like repeat A docking site using WaterMap</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">78</span><span class="NLM_x">, </span> <span class="NLM_fpage">2571</span><span class="NLM_x">–</span> <span class="NLM_lpage">2586</span><span class="refDoi"> DOI: 10.1002/prot.22767</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1002%2Fprot.22767" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=2010&pages=2571-2586&author=R.+A.+Pearlsteinauthor=Q.-Y.+Huauthor=J.+Zhouauthor=D.+Yoweauthor=J.+Levellauthor=B.+Daleauthor=V.+K.+Kaushikauthor=D.+Danielsauthor=S.+Hanrahanauthor=W.+Shermanauthor=R.+Abel&title=New+hypotheses+about+the+structure%E2%80%93function+of+proprotein+convertase+subtilisin%2Fkexin+type+9%3A+analysis+of+the+epidermal+growth+factor-like+repeat+A+docking+site+using+WaterMap&doi=10.1002%2Fprot.22767"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1002%2Fprot.22767&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.22767%26sid%3Dliteratum%253Aachs%26aulast%3DPearlstein%26aufirst%3DR.%2BA.%26aulast%3DHu%26aufirst%3DQ.-Y.%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DYowe%26aufirst%3DD.%26aulast%3DLevell%26aufirst%3DJ.%26aulast%3DDale%26aufirst%3DB.%26aulast%3DKaushik%26aufirst%3DV.%2BK.%26aulast%3DDaniels%26aufirst%3DD.%26aulast%3DHanrahan%26aufirst%3DS.%26aulast%3DSherman%26aufirst%3DW.%26aulast%3DAbel%26aufirst%3DR.%26atitle%3DNew%2520hypotheses%2520about%2520the%2520structure%25E2%2580%2593function%2520of%2520proprotein%2520convertase%2520subtilisin%252Fkexin%2520type%25209%253A%2520analysis%2520of%2520the%2520epidermal%2520growth%2520factor-like%2520repeat%2520A%2520docking%2520site%2520using%2520WaterMap%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2010%26volume%3D78%26spage%3D2571%26epage%3D2586%26doi%3D10.1002%2Fprot.22767" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Tran, Q.-T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Farid, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Erdemli, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pearlstein, R.</span><span> </span><span class="NLM_article-title">The translocation kinetics of antibiotics through porin OmpC: insights from structure-based solvation mapping using WaterMap</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">81</span><span class="NLM_x">, </span> <span class="NLM_fpage">291</span><span class="NLM_x">–</span> <span class="NLM_lpage">299</span><span class="refDoi"> DOI: 10.1002/prot.24185</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1002%2Fprot.24185" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=2013&pages=291-299&author=Q.-T.+Tranauthor=S.+Williamsauthor=R.+Faridauthor=G.+Erdemliauthor=R.+Pearlstein&title=The+translocation+kinetics+of+antibiotics+through+porin+OmpC%3A+insights+from+structure-based+solvation+mapping+using+WaterMap&doi=10.1002%2Fprot.24185"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1002%2Fprot.24185&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.24185%26sid%3Dliteratum%253Aachs%26aulast%3DTran%26aufirst%3DQ.-T.%26aulast%3DWilliams%26aufirst%3DS.%26aulast%3DFarid%26aufirst%3DR.%26aulast%3DErdemli%26aufirst%3DG.%26aulast%3DPearlstein%26aufirst%3DR.%26atitle%3DThe%2520translocation%2520kinetics%2520of%2520antibiotics%2520through%2520porin%2520OmpC%253A%2520insights%2520from%2520structure-based%2520solvation%2520mapping%2520using%2520WaterMap%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2013%26volume%3D81%26spage%3D291%26epage%3D299%26doi%3D10.1002%2Fprot.24185" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Warren, G. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Andrews, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Capelli, A.-M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clarke, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">LaLonde, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lambert, M. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lindvall, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nevins, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Semus, S. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senger, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tedesco, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wall, I. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woolven, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peishoff, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Head, M. S.</span><span> </span><span class="NLM_article-title">A Critical Assessment of Docking Programs and Scoring Functions</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">5912</span><span class="NLM_x">–</span> <span class="NLM_lpage">5931</span><span class="refDoi"> DOI: 10.1021/jm050362n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm050362n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BD2MXnslOrtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=5912-5931&author=G.+L.+Warrenauthor=C.+W.+Andrewsauthor=A.-M.+Capelliauthor=B.+Clarkeauthor=J.+LaLondeauthor=M.+H.+Lambertauthor=M.+Lindvallauthor=N.+Nevinsauthor=S.+F.+Semusauthor=S.+Sengerauthor=G.+Tedescoauthor=I.+D.+Wallauthor=J.+M.+Woolvenauthor=C.+E.+Peishoffauthor=M.+S.+Head&title=A+Critical+Assessment+of+Docking+Programs+and+Scoring+Functions&doi=10.1021%2Fjm050362n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">A Critical Assessment of Docking Programs and Scoring Functions</span></div><div class="casAuthors">Warren, Gregory L.; Andrews, C. Webster; Capelli, Anna-Maria; Clarke, Brian; LaLonde, Judith; Lambert, Millard H.; Lindvall, Mika; Nevins, Neysa; Semus, Simon F.; Senger, Stefan; Tedesco, Giovanna; Wall, Ian D.; Woolven, James M.; Peishoff, Catherine E.; Head, Martha S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">5912-5931</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Docking is a computational technique that samples conformations of small mols. in protein binding sites; scoring functions are used to assess which of these conformations best complements the protein binding site.  An evaluation of 10 docking programs and 37 scoring functions was conducted against eight proteins of seven protein types for three tasks: binding mode prediction, virtual screening for lead identification, and rank-ordering by affinity for lead optimization.  All of the docking programs were able to generate ligand conformations similar to crystallog. detd. protein/ligand complex structures for at least one of the targets.  However, scoring functions were less successful at distinguishing the crystallog. conformation from the set of docked poses.  Docking programs identified active compds. from a pharmaceutically relevant pool of decoy compds.; however, no single program performed well for all of the targets.  For prediction of compd. affinity, none of the docking programs or scoring functions made a useful prediction of ligand binding affinity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfJn8_WHSjurVg90H21EOLACvtfcHk0ljOFj8GvreQOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXnslOrtrY%253D&md5=463ab40c3d23e67b62df408fc5e7f08d</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fjm050362n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm050362n%26sid%3Dliteratum%253Aachs%26aulast%3DWarren%26aufirst%3DG.%2BL.%26aulast%3DAndrews%26aufirst%3DC.%2BW.%26aulast%3DCapelli%26aufirst%3DA.-M.%26aulast%3DClarke%26aufirst%3DB.%26aulast%3DLaLonde%26aufirst%3DJ.%26aulast%3DLambert%26aufirst%3DM.%2BH.%26aulast%3DLindvall%26aufirst%3DM.%26aulast%3DNevins%26aufirst%3DN.%26aulast%3DSemus%26aufirst%3DS.%2BF.%26aulast%3DSenger%26aufirst%3DS.%26aulast%3DTedesco%26aufirst%3DG.%26aulast%3DWall%26aufirst%3DI.%2BD.%26aulast%3DWoolven%26aufirst%3DJ.%2BM.%26aulast%3DPeishoff%26aufirst%3DC.%2BE.%26aulast%3DHead%26aufirst%3DM.%2BS.%26atitle%3DA%2520Critical%2520Assessment%2520of%2520Docking%2520Programs%2520and%2520Scoring%2520Functions%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D5912%26epage%3D5931%26doi%3D10.1021%2Fjm050362n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Plewczynski, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Łaźniewski, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Augustyniak, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ginalski, K.</span><span> </span><span class="NLM_article-title">Can we trust docking results? Evaluation of seven commonly used programs on PDBbind database</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x">, </span> <span class="NLM_fpage">742</span><span class="NLM_x">–</span> <span class="NLM_lpage">755</span><span class="refDoi"> DOI: 10.1002/jcc.21643</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1002%2Fjcc.21643" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=20812323" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BC3MXosVemuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2011&pages=742-755&author=D.+Plewczynskiauthor=M.+%C5%81a%C5%BAniewskiauthor=R.+Augustyniakauthor=K.+Ginalski&title=Can+we+trust+docking+results%3F+Evaluation+of+seven+commonly+used+programs+on+PDBbind+database&doi=10.1002%2Fjcc.21643"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Can we trust docking results? Evaluation of seven commonly used programs on PDBbind database</span></div><div class="casAuthors">Plewczynski, Dariusz; Lazniewski, Michal; Augustyniak, Rafal; Ginalski, Krzysztof</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">742-755</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  Docking is one of the most commonly used techniques in drug design.  It is used for both identifying correct poses of a ligand in the binding site of a protein as well as for the estn. of the strength of protein-ligand interaction.  Because millions of compds. must be screened, before a suitable target for biol. testing can be identified, all calcns. should be done in a reasonable time frame.  Thus, all programs currently in use exploit empirically based algorithms, avoiding systematic search of the conformational space.  Similarly, the scoring is done using simple equations, which makes it possible to speed up the entire process.  Therefore, docking results have to be verified by subsequent in vitro studies.  The purpose of our work was to evaluate seven popular docking programs (Surflex, LigandFit, Glide, GOLD, FlexX, eHiTS, and AutoDock) on the extensive dataset composed of 1300 protein-ligands complexes from PDBbind 2007 database, where exptl. measured binding affinity values were also available.  We compared independently the ability of proper posing [according to Root mean square deviation (or Root mean square distance) of predicted conformations vs. the corresponding native one] and scoring (by calcg. the correlation between docking score and ligand binding strength).  To our knowledge, it is the first large-scale docking evaluation that covers both aspects of docking programs, i.e., predicting ligand conformation and calcg. the strength of its binding.  More than 1000 protein-ligand pairs cover a wide range of different protein families and inhibitor classes.  Our results clearly showed that the ligand binding conformation could be identified in most cases by using the existing software, yet we still obsd. the lack of universal scoring function for all types of mols. and protein families.  © 2010 Wiley Periodicals, Inc.  J Comput Chem, 2011.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrrK3tNTWoTdrVg90H21EOLACvtfcHk0lhQRl-a_7DwSA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXosVemuw%253D%253D&md5=61745292dd287a8139df98c6a2b72d40</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1002%2Fjcc.21643&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.21643%26sid%3Dliteratum%253Aachs%26aulast%3DPlewczynski%26aufirst%3DD.%26aulast%3D%25C5%2581a%25C5%25BAniewski%26aufirst%3DM.%26aulast%3DAugustyniak%26aufirst%3DR.%26aulast%3DGinalski%26aufirst%3DK.%26atitle%3DCan%2520we%2520trust%2520docking%2520results%253F%2520Evaluation%2520of%2520seven%2520commonly%2520used%2520programs%2520on%2520PDBbind%2520database%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2011%26volume%3D32%26spage%3D742%26epage%3D755%26doi%3D10.1002%2Fjcc.21643" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Kollman, P.</span><span> </span><span class="NLM_article-title">Free energy calculations: Applications to chemical and biochemical phenomena</span> <span class="citation_source-journal">Chem. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">93</span><span class="NLM_x">, </span> <span class="NLM_fpage">2395</span><span class="NLM_x">–</span> <span class="NLM_lpage">2417</span><span class="refDoi"> DOI: 10.1021/cr00023a004</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr00023a004" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADyaK3sXmt1Sktr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=1993&pages=2395-2417&author=P.+Kollman&title=Free+energy+calculations%3A+Applications+to+chemical+and+biochemical+phenomena&doi=10.1021%2Fcr00023a004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Free energy calculations: Applications to chemical and biochemical phenomena</span></div><div class="casAuthors">Kollman, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2395-417</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    </div><div class="casAbstract">A review with 252 refs. about applications of free energy calcns. employing mol. dynamics or Monte Carlo methods to a variety of chem. and biochem. phenomena.  The focus is on applications of such calcns. to mol. solvation, mol. assocn., macromol. stability, and enzyme catalysis.  The mols. discussed range from monovalent ions and small mols. to proteins and nucleic acids.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrN1w4SyFDqCLVg90H21EOLACvtfcHk0lhQRl-a_7DwSA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXmt1Sktr0%253D&md5=f12326b24734ea092996cf22efb6ebd6</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fcr00023a004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr00023a004%26sid%3Dliteratum%253Aachs%26aulast%3DKollman%26aufirst%3DP.%26atitle%3DFree%2520energy%2520calculations%253A%2520Applications%2520to%2520chemical%2520and%2520biochemical%2520phenomena%26jtitle%3DChem.%2520Rev.%26date%3D1993%26volume%3D93%26spage%3D2395%26epage%3D2417%26doi%3D10.1021%2Fcr00023a004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Gilson, M. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Given, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bush, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCammon, J. A.</span><span> </span><span class="NLM_article-title">The statistical-thermodynamic basis for computation of binding affinities: a critical review</span> <span class="citation_source-journal">Biophys. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">72</span><span class="NLM_x">, </span> <span class="NLM_fpage">1047</span><span class="NLM_x">–</span> <span class="NLM_lpage">1069</span><span class="refDoi"> DOI: 10.1016/S0006-3495(97)78756-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1016%2FS0006-3495%2897%2978756-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=9138555" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADyaK2sXhsVeisbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=1997&pages=1047-1069&author=M.+K.+Gilsonauthor=J.+A.+Givenauthor=B.+L.+Bushauthor=J.+A.+McCammon&title=The+statistical-thermodynamic+basis+for+computation+of+binding+affinities%3A+a+critical+review&doi=10.1016%2FS0006-3495%2897%2978756-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">The statistical-thermodynamic basis for computation of binding affinities: a critical review</span></div><div class="casAuthors">Gilson, Michael K.; Given, James A.; Bush, Bruce L.; Mccammon, J. Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Biophysical Journal</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1047-1069</span>CODEN:
                <span class="NLM_cas:coden">BIOJAU</span>;
        ISSN:<span class="NLM_cas:issn">0006-3495</span>.
    
            (<span class="NLM_cas:orgname">Biophysical Society</span>)
        </div><div class="casAbstract">A review with many refs.  Although the statistical thermodn. of noncovalent binding has been considered in a no. of theor. papers, few methods of computing binding affinities are derived explicitly from this underlying theory.  This has contributed to uncertainty and controversy in certain areas.  This article therefore reviews and extends the connections of some important computational methods with the underlying statistical thermodn.  A derivation of the std. free energy of binding forms the basis of this review.  This derivation should be useful in formulating novel computational methods for predicting binding affinities.  It also permits several important points to be established.  For example, it is found that the double annihilation method of computing binding energy does not yield the std. free energy of binding, but can be modified to yield this quantity.  The derivation also makes it possible to define clearly the changes in translational, rotational, configurational, and solvent entropy upon binding.  It is argued that mol. mass has a negligible effect upon the std. free energy of binding for biomol. systems, and that the cratic entropy defined by Gurney is not a useful concept.  In addn., the use of continuum models of the solvent in binding calcns. is reviewed, and a formalism is presented for incorporating a limited no. of solvent mols. explicitly.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtZwKHjQ8gW7Vg90H21EOLACvtfcHk0lhQRl-a_7DwSA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXhsVeisbo%253D&md5=2bf97b219b9d15fccab6cb50dc7686aa</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2FS0006-3495%2897%2978756-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0006-3495%252897%252978756-3%26sid%3Dliteratum%253Aachs%26aulast%3DGilson%26aufirst%3DM.%2BK.%26aulast%3DGiven%26aufirst%3DJ.%2BA.%26aulast%3DBush%26aufirst%3DB.%2BL.%26aulast%3DMcCammon%26aufirst%3DJ.%2BA.%26atitle%3DThe%2520statistical-thermodynamic%2520basis%2520for%2520computation%2520of%2520binding%2520affinities%253A%2520a%2520critical%2520review%26jtitle%3DBiophys.%2520J.%26date%3D1997%26volume%3D72%26spage%3D1047%26epage%3D1069%26doi%3D10.1016%2FS0006-3495%2897%2978756-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">Mobley, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Graves, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chodera, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McReynolds, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dill, K. A.</span><span> </span><span class="NLM_article-title">Predicting Absolute Ligand Binding Free Energies to a Simple Model Site</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">371</span><span class="NLM_x">, </span> <span class="NLM_fpage">1118</span><span class="NLM_x">–</span> <span class="NLM_lpage">1134</span><span class="refDoi"> DOI: 10.1016/j.jmb.2007.06.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1016%2Fj.jmb.2007.06.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=17599350" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BD2sXot12rt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=371&publication_year=2007&pages=1118-1134&author=D.+L.+Mobleyauthor=A.+P.+Gravesauthor=J.+D.+Choderaauthor=A.+C.+McReynoldsauthor=B.+K.+Shoichetauthor=K.+A.+Dill&title=Predicting+Absolute+Ligand+Binding+Free+Energies+to+a+Simple+Model+Site&doi=10.1016%2Fj.jmb.2007.06.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Predicting Absolute Ligand Binding Free Energies to a Simple Model Site</span></div><div class="casAuthors">Mobley, David L.; Graves, Alan P.; Chodera, John D.; McReynolds, Andrea C.; Shoichet, Brian K.; Dill, Ken A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">371</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1118-1134</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A central challenge in structure-based ligand design is the accurate prediction of binding free energies.  Here the authors apply alchem. free energy calcns. in explicit solvent to predict ligand binding in a model cavity in T4 lysozyme.  Even in this simple site, there are challenges.  The authors made systematic improvements, beginning with single poses from docking, then including multiple poses, addnl. protein conformational changes, and using an improved charge model.  Computed abs. binding free energies had an RMS error of 1.9 kcal/mol relative to previously detd. exptl. values.  In blind prospective tests, the methods correctly discriminated between several true ligands and decoys in a set of putative binders identified by docking.  In these prospective tests, the RMS error in predicted binding free energies relative to those subsequently detd. exptl. was only 0.6 kcal/mol.  X-ray crystal structures of the new ligands bound in the cavity corresponded closely to predictions from the free energy calcns., but sometimes differed from those predicted by docking.  Finally, the authors examd. the impact of holding the protein rigid, as in docking, with a view to learning how approxns. made in docking affect accuracy and how they may be improved.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqD1-EYEODQ47Vg90H21EOLACvtfcHk0liB1LwUqxdoOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXot12rt7o%253D&md5=2480139daa5a0857c59e821d2ecbce18</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2007.06.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2007.06.002%26sid%3Dliteratum%253Aachs%26aulast%3DMobley%26aufirst%3DD.%2BL.%26aulast%3DGraves%26aufirst%3DA.%2BP.%26aulast%3DChodera%26aufirst%3DJ.%2BD.%26aulast%3DMcReynolds%26aufirst%3DA.%2BC.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DDill%26aufirst%3DK.%2BA.%26atitle%3DPredicting%2520Absolute%2520Ligand%2520Binding%2520Free%2520Energies%2520to%2520a%2520Simple%2520Model%2520Site%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2007%26volume%3D371%26spage%3D1118%26epage%3D1134%26doi%3D10.1016%2Fj.jmb.2007.06.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Chodera, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mobley, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shirts, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dixon, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Branson, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pande, V. S.</span><span> </span><span class="NLM_article-title">Alchemical free energy methods for drug discovery: progress and challenges</span> <span class="citation_source-journal">Curr. Opin. Struct. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">150</span><span class="NLM_x">–</span> <span class="NLM_lpage">160</span><span class="refDoi"> DOI: 10.1016/j.sbi.2011.01.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1016%2Fj.sbi.2011.01.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=21349700" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BC3MXkt1Kisrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2011&pages=150-160&author=J.+D.+Choderaauthor=D.+L.+Mobleyauthor=M.+R.+Shirtsauthor=R.+W.+Dixonauthor=K.+Bransonauthor=V.+S.+Pande&title=Alchemical+free+energy+methods+for+drug+discovery%3A+progress+and+challenges&doi=10.1016%2Fj.sbi.2011.01.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Alchemical free energy methods for drug discovery: Progress and challenges</span></div><div class="casAuthors">Chodera, John D.; Mobley, David L.; Shirts, Michael R.; Dixon, Richard W.; Branson, Kim; Pande, Vijay S.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">150-160</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Improved rational drug design methods are needed to lower the cost and increase the success rate of drug discovery and development.  Alchem. binding free energy calcns., one potential tool for rational design, have progressed rapidly over the past decade, but still fall short of providing robust tools for pharmaceutical engineering.  Recent studies, esp. on model receptor systems, have clarified many of the challenges that must be overcome for robust predictions of binding affinity to be useful in rational design.  In this review, inspired by a recent joint academic/industry meeting organized by the authors, we discuss these challenges and suggest a no. of promising approaches for overcoming them.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobUBHsGUCIWLVg90H21EOLACvtfcHk0liB1LwUqxdoOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXkt1Kisrk%253D&md5=fa17c0982d921cd1c32cd095fd34420e</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.sbi.2011.01.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.sbi.2011.01.011%26sid%3Dliteratum%253Aachs%26aulast%3DChodera%26aufirst%3DJ.%2BD.%26aulast%3DMobley%26aufirst%3DD.%2BL.%26aulast%3DShirts%26aufirst%3DM.%2BR.%26aulast%3DDixon%26aufirst%3DR.%2BW.%26aulast%3DBranson%26aufirst%3DK.%26aulast%3DPande%26aufirst%3DV.%2BS.%26atitle%3DAlchemical%2520free%2520energy%2520methods%2520for%2520drug%2520discovery%253A%2520progress%2520and%2520challenges%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2011%26volume%3D21%26spage%3D150%26epage%3D160%26doi%3D10.1016%2Fj.sbi.2011.01.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pierce, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krilov, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lupyan, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robinson, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dahlgren, M. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greenwood, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Romero, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Masse, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knight, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steinbrecher, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beuming, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Damm, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harder, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sherman, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brewer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wester, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murcko, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Farid, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mobley, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berne, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friesner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abel, R.</span><span> </span><span class="NLM_article-title">Accurate and Reliable Prediction of Relative Ligand Binding Potency in Prospective Drug Discovery by Way of a Modern Free-Energy Calculation Protocol and Force Field</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">137</span><span class="NLM_x">, </span> <span class="NLM_fpage">2695</span><span class="NLM_x">–</span> <span class="NLM_lpage">2703</span><span class="refDoi"> DOI: 10.1021/ja512751q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja512751q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsF2iuro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=137&publication_year=2015&pages=2695-2703&author=L.+Wangauthor=Y.+Wuauthor=Y.+Dengauthor=B.+Kimauthor=L.+Pierceauthor=G.+Krilovauthor=D.+Lupyanauthor=S.+Robinsonauthor=M.+K.+Dahlgrenauthor=J.+Greenwoodauthor=D.+L.+Romeroauthor=C.+Masseauthor=J.+L.+Knightauthor=T.+Steinbrecherauthor=T.+Beumingauthor=W.+Dammauthor=E.+Harderauthor=W.+Shermanauthor=M.+Brewerauthor=R.+Westerauthor=M.+Murckoauthor=L.+Fryeauthor=R.+Faridauthor=T.+Linauthor=D.+L.+Mobleyauthor=W.+L.+Jorgensenauthor=B.+J.+Berneauthor=R.+A.+Friesnerauthor=R.+Abel&title=Accurate+and+Reliable+Prediction+of+Relative+Ligand+Binding+Potency+in+Prospective+Drug+Discovery+by+Way+of+a+Modern+Free-Energy+Calculation+Protocol+and+Force+Field&doi=10.1021%2Fja512751q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Accurate and Reliable Prediction of Relative Ligand Binding Potency in Prospective Drug Discovery by Way of a Modern Free-Energy Calculation Protocol and Force Field</span></div><div class="casAuthors">Wang, Lingle; Wu, Yujie; Deng, Yuqing; Kim, Byungchan; Pierce, Levi; Krilov, Goran; Lupyan, Dmitry; Robinson, Shaughnessy; Dahlgren, Markus K.; Greenwood, Jeremy; Romero, Donna L.; Masse, Craig; Knight, Jennifer L.; Steinbrecher, Thomas; Beuming, Thijs; Damm, Wolfgang; Harder, Ed; Sherman, Woody; Brewer, Mark; Wester, Ron; Murcko, Mark; Frye, Leah; Farid, Ramy; Lin, Teng; Mobley, David L.; Jorgensen, William L.; Berne, Bruce J.; Friesner, Richard A.; Abel, Robert</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">137</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2695-2703</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Designing tight-binding ligands is a primary objective of small-mol. drug discovery.  Over the past few decades, free-energy calcns. have benefited from improved force fields and sampling algorithms, as well as the advent of low-cost parallel computing.  However, it has proven to be challenging to reliably achieve the level of accuracy that would be needed to guide lead optimization (∼5× in binding affinity) for a wide range of ligands and protein targets.  Not surprisingly, widespread com. application of free-energy simulations has been limited due to the lack of large-scale validation coupled with the tech. challenges traditionally assocd. with running these types of calcns.  Here, we report an approach that achieves an unprecedented level of accuracy across a broad range of target classes and ligands, with retrospective results encompassing 200 ligands and a wide variety of chem. perturbations, many of which involve significant changes in ligand chem. structures.  In addn., we have applied the method in prospective drug discovery projects and found a significant improvement in the quality of the compds. synthesized that have been predicted to be potent.  Compds. predicted to be potent by this approach have a substantial redn. in false positives relative to compds. synthesized on the basis of other computational or medicinal chem. approaches.  Furthermore, the results are consistent with those obtained from our retrospective studies, demonstrating the robustness and broad range of applicability of this approach, which can be used to drive decisions in lead optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGor9UtXEyLPpLVg90H21EOLACvtfcHk0liB1LwUqxdoOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsF2iuro%253D&md5=37a4f4a6c085f47ed531342643b6c33b</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fja512751q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja512751q%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DDeng%26aufirst%3DY.%26aulast%3DKim%26aufirst%3DB.%26aulast%3DPierce%26aufirst%3DL.%26aulast%3DKrilov%26aufirst%3DG.%26aulast%3DLupyan%26aufirst%3DD.%26aulast%3DRobinson%26aufirst%3DS.%26aulast%3DDahlgren%26aufirst%3DM.%2BK.%26aulast%3DGreenwood%26aufirst%3DJ.%26aulast%3DRomero%26aufirst%3DD.%2BL.%26aulast%3DMasse%26aufirst%3DC.%26aulast%3DKnight%26aufirst%3DJ.%2BL.%26aulast%3DSteinbrecher%26aufirst%3DT.%26aulast%3DBeuming%26aufirst%3DT.%26aulast%3DDamm%26aufirst%3DW.%26aulast%3DHarder%26aufirst%3DE.%26aulast%3DSherman%26aufirst%3DW.%26aulast%3DBrewer%26aufirst%3DM.%26aulast%3DWester%26aufirst%3DR.%26aulast%3DMurcko%26aufirst%3DM.%26aulast%3DFrye%26aufirst%3DL.%26aulast%3DFarid%26aufirst%3DR.%26aulast%3DLin%26aufirst%3DT.%26aulast%3DMobley%26aufirst%3DD.%2BL.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DBerne%26aufirst%3DB.%2BJ.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DAbel%26aufirst%3DR.%26atitle%3DAccurate%2520and%2520Reliable%2520Prediction%2520of%2520Relative%2520Ligand%2520Binding%2520Potency%2520in%2520Prospective%2520Drug%2520Discovery%2520by%2520Way%2520of%2520a%2520Modern%2520Free-Energy%2520Calculation%2520Protocol%2520and%2520Force%2520Field%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2015%26volume%3D137%26spage%3D2695%26epage%3D2703%26doi%3D10.1021%2Fja512751q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">Guo, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heitman, L. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ijzerman, A. P.</span><span> </span><span class="NLM_article-title">The role of target binding kinetics in drug discovery</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">1793</span><span class="NLM_x">–</span> <span class="NLM_lpage">1796</span><span class="refDoi"> DOI: 10.1002/cmdc.201500310</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1002%2Fcmdc.201500310" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=26506404" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlyjurfF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=1793-1796&author=D.+Guoauthor=L.+H.+Heitmanauthor=A.+P.+Ijzerman&title=The+role+of+target+binding+kinetics+in+drug+discovery&doi=10.1002%2Fcmdc.201500310"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">The Role of Target Binding Kinetics in Drug Discovery</span></div><div class="casAuthors">Guo, Dong; Heitman, Laura H.; IJzerman, Adriaan P.</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1793-1796</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Traditionally structure-activity/affinity relationships (SAR) have dominated research in medicinal chem.  However, structure-kinetics relationships (SKR) can be very informative too.  In this viewpoint we explore the mol. determinants of binding kinetics and discuss challenges for future binding kinetics studies.  A scheme for future kinetics-directed drug design and discovery is also proposed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoiX7v30ryLZLVg90H21EOLACvtfcHk0lgeYJvkW8xCMg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlyjurfF&md5=8442ba4d4ec014762ed470f7f6131d09</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201500310&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201500310%26sid%3Dliteratum%253Aachs%26aulast%3DGuo%26aufirst%3DD.%26aulast%3DHeitman%26aufirst%3DL.%2BH.%26aulast%3DIjzerman%26aufirst%3DA.%2BP.%26atitle%3DThe%2520role%2520of%2520target%2520binding%2520kinetics%2520in%2520drug%2520discovery%26jtitle%3DChemMedChem%26date%3D2015%26volume%3D10%26spage%3D1793%26epage%3D1796%26doi%3D10.1002%2Fcmdc.201500310" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group">Guo, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hillger, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ijzerman, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heitman, L. H.</span><span> </span><span class="NLM_article-title">Drug-target residence time—a case for G protein-coupled receptors</span> <span class="citation_source-journal">Med. Res. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">34</span><span class="NLM_x">, </span> <span class="NLM_fpage">856</span><span class="NLM_x">–</span> <span class="NLM_lpage">892</span><span class="refDoi"> DOI: 10.1002/med.21307</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1002%2Fmed.21307" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=24549504" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BC2cXptVagtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2014&pages=856-892&author=D.+Guoauthor=J.+M.+Hillgerauthor=A.+P.+Ijzermanauthor=L.+H.+Heitman&title=Drug-target+residence+time%E2%80%94a+case+for+G+protein-coupled+receptors&doi=10.1002%2Fmed.21307"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Drug-Target Residence Time-A Case for G Protein-Coupled Receptors</span></div><div class="casAuthors">Guo, Dong; Hillger, Julia M.; Ijzerman, Adriaan P.; Heitman, Laura H.</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">856-892</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  A vast no. of marketed drugs act on G protein-coupled receptors (GPCRs), the most successful category of drug targets to date.  These drugs usually possess high target affinity and selectivity, and such combined features have been the driving force in the early phases of drug discovery.  However, attrition has also been high.  Many investigational new drugs eventually fail in clin. trials due to a demonstrated lack of efficacy.  A retrospective assessment of successfully launched drugs revealed that their beneficial effects in patients may be attributed to their long drug-target residence times (RTs).  Likewise, for some other GPCR drugs short RT could be beneficial to reduce the potential for on-target side effects.  Hence, the compds.' kinetics behavior might in fact be the guiding principle to obtain a desired and durable effect in vivo.  We therefore propose that drug-target RT should be taken into account as an addnl. parameter in the lead selection and optimization process.  This should ultimately lead to an increased no. of candidate drugs moving to the preclin. development phase and on to the market.  This review contains examples of the kinetics behavior of GPCR ligands with improved in vivo efficacy and summarizes methods for assessing drug-target RT.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpN5vhWI93LMbVg90H21EOLACvtfcHk0lgeYJvkW8xCMg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXptVagtbs%253D&md5=af48cdedf3c60ecb608d5c9ad03a0bbe</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1002%2Fmed.21307&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21307%26sid%3Dliteratum%253Aachs%26aulast%3DGuo%26aufirst%3DD.%26aulast%3DHillger%26aufirst%3DJ.%2BM.%26aulast%3DIjzerman%26aufirst%3DA.%2BP.%26aulast%3DHeitman%26aufirst%3DL.%2BH.%26atitle%3DDrug-target%2520residence%2520time%25E2%2580%2594a%2520case%2520for%2520G%2520protein-coupled%2520receptors%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2014%26volume%3D34%26spage%3D856%26epage%3D892%26doi%3D10.1002%2Fmed.21307" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pompliano, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meek, T. D.</span><span> </span><span class="NLM_article-title">Drug-target residence time and its implications for lead optimization</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">730</span><span class="NLM_x">–</span> <span class="NLM_lpage">739</span><span class="refDoi"> DOI: 10.1038/nrd2082</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1038%2Fnrd2082" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=16888652" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BD28XptVCltro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2006&pages=730-739&author=R.+A.+Copelandauthor=D.+L.+Pomplianoauthor=T.+D.+Meek&title=Drug-target+residence+time+and+its+implications+for+lead+optimization&doi=10.1038%2Fnrd2082"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Drug-target residence time and its implications for lead optimization</span></div><div class="casAuthors">Copeland, Robert A.; Pompliano, David L.; Meek, Thomas D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">730-739</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Much of drug discovery today is predicated on the concept of selective targeting of particular bioactive macromols. by low-mol.-mass drugs.  The binding of drugs to their macromol. targets is therefore seen as paramount for pharmacol. activity.  In vitro assessment of drug-target interactions is classically quantified in terms of binding parameters such as IC50 or Kd.  This article presents an alternative perspective on drug optimization in terms of drug-target binary complex residence time, as quantified by the dissociative half-life of the drug-target binary complex.  We describe the potential advantages of long residence time in terms of duration of pharmacol. effect and target selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGry6SJrMlrrTbVg90H21EOLACvtfcHk0lgeYJvkW8xCMg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XptVCltro%253D&md5=60ede2301584b10ac4e8fa18e1e6d107</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1038%2Fnrd2082&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2082%26sid%3Dliteratum%253Aachs%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DPompliano%26aufirst%3DD.%2BL.%26aulast%3DMeek%26aufirst%3DT.%2BD.%26atitle%3DDrug-target%2520residence%2520time%2520and%2520its%2520implications%2520for%2520lead%2520optimization%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2006%26volume%3D5%26spage%3D730%26epage%3D739%26doi%3D10.1038%2Fnrd2082" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group">Tian, H.; Sakmar, T. P.; Huber, T.</span><span> </span><span class="NLM_article-title">Site-specific labeling of genetically encoded azido groups for multicolor, single-molecule fluorescence imaging of GPCRs</span>. In  <span class="citation_source-book">Methods in Cell Biology</span>; <span class="NLM_contrib-group">Conn, P. M.</span>, Ed.; <span class="NLM_publisher-name">Academic Press</span>: <span class="NLM_publisher-loc">Cambridge, MA</span>,<span class="NLM_x"> </span><span class="NLM_year">2013</span><span class="NLM_x">; </span>Vol.  <span class="NLM_volume">117</span>, Chapter 15, pp  <span class="NLM_fpage">267</span><span class="NLM_x">–</span> <span class="NLM_lpage">303</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1016%2FB978-0-12-408143-7.00015-3" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&pages=267-303&author=H.+Tian&author=T.+P.+Sakmar&author=T.+Huberauthor=P.+M.+Conn&title=Methods+in+Cell+Biology"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2FB978-0-12-408143-7.00015-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1016%252FB978-0-12-408143-7.00015-3%26sid%3Dliteratum%253Aachs%26aulast%3DTian%26aufirst%3DH.%26atitle%3DSite-specific%2520labeling%2520of%2520genetically%2520encoded%2520azido%2520groups%2520for%2520multicolor%252C%2520single-molecule%2520fluorescence%2520imaging%2520of%2520GPCRs%26btitle%3DMethods%2520in%2520Cell%2520Biology%26aulast%3DConn%26aufirst%3DP.%2BM.%26pub%3DAcademic%2520Press%26date%3D2013%26volume%3D117%26spage%3D267%26epage%3D303" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group">Hurst, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmeisser, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reggio, P. H.</span><span> </span><span class="NLM_article-title">Endogenous lipid activated G protein-coupled receptors: emerging structural features from crystallography and molecular dynamics simulations</span> <span class="citation_source-journal">Chem. Phys. Lipids</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">169</span><span class="NLM_x">, </span> <span class="NLM_fpage">46</span><span class="NLM_x">–</span> <span class="NLM_lpage">56</span><span class="refDoi"> DOI: 10.1016/j.chemphyslip.2013.01.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1016%2Fj.chemphyslip.2013.01.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=23485612" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BC3sXltVGht7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=169&publication_year=2013&pages=46-56&author=D.+P.+Hurstauthor=M.+Schmeisserauthor=P.+H.+Reggio&title=Endogenous+lipid+activated+G+protein-coupled+receptors%3A+emerging+structural+features+from+crystallography+and+molecular+dynamics+simulations&doi=10.1016%2Fj.chemphyslip.2013.01.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Endogenous lipid activated G protein-coupled receptors: Emerging structural features from crystallography and molecular dynamics simulations</span></div><div class="casAuthors">Hurst, Dow P.; Schmeisser, Marianne; Reggio, Patricia H.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry and Physics of Lipids</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">169</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">46-56</span>CODEN:
                <span class="NLM_cas:coden">CPLIA4</span>;
        ISSN:<span class="NLM_cas:issn">0009-3084</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Class A G-protein coupled receptors (GPCRs) are thought to have a common topol. that includes seven transmembrane alpha helixes (TMHs) that are arranged to form a closed bundle.  This bundle forms the ligand binding pocket into which ligands are commonly thought to enter via the extracellular milieu.  This ligand approach direction makes sense for GPCRs that have small pos. charged ligands, such as the beta-2-adrenergic or the dopamine D2 receptor.  However, there is a growing sub-group of Class A GPCRs that bind lipid-derived endogenous ligands, such as the cannabinoid CB1 and CB2 receptors (with endogenous ligands, N-arachidonoylethanolamine (anandamide) and sn-2-arachidonylglycerol (2-AG)) and the S1P1-5 receptors (with endogenous ligand, sphingosine-1-phosphate).  Even the widely studied Class A GPCR, rhodopsin, binds a highly lipophillic chromophore (11-cis-retinal).  For these receptors, ligand approach from the extracellular milieu has seemed unlikely given that the ligands of these receptors readily partition into lipid or are actually synthesized in the lipid bilayer.  The recent X-ray-crystal structure of the sub-type 1 sphingosine-1-phosphate receptor (S1P1) provides important information on the key structural variations that may be the hallmarks for a Class A GPCR that binds lipid-derived ligands.  These include an extracellular domain that is closed off to the extracellular milieu and the existence of an opening between transmembrane helixes that may serve as a portal for ligand entry via the lipid bilayer.  This review examines structural aspects that the cannabinoid receptors may share with the S1P1 receptor based upon sequence homol.  This review also examines exptl. and simulation results that suggest ligand entry via a lipid portal is quite likely for this emerging sub-group.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRGcleyJ2OO7Vg90H21EOLACvtfcHk0ljWeeX0bBaMRg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXltVGht7c%253D&md5=65882b737337687be6527251a9cad9d1</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2Fj.chemphyslip.2013.01.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chemphyslip.2013.01.009%26sid%3Dliteratum%253Aachs%26aulast%3DHurst%26aufirst%3DD.%2BP.%26aulast%3DSchmeisser%26aufirst%3DM.%26aulast%3DReggio%26aufirst%3DP.%2BH.%26atitle%3DEndogenous%2520lipid%2520activated%2520G%2520protein-coupled%2520receptors%253A%2520emerging%2520structural%2520features%2520from%2520crystallography%2520and%2520molecular%2520dynamics%2520simulations%26jtitle%3DChem.%2520Phys.%2520Lipids%26date%3D2013%26volume%3D169%26spage%3D46%26epage%3D56%26doi%3D10.1016%2Fj.chemphyslip.2013.01.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group">Wang, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duan, Y.</span><span> </span><span class="NLM_article-title">Ligand entry and exit pathways in the β2-adrenergic receptor</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">392</span><span class="NLM_x">, </span> <span class="NLM_fpage">1102</span><span class="NLM_x">–</span> <span class="NLM_lpage">1115</span><span class="refDoi"> DOI: 10.1016/j.jmb.2009.07.093</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1016%2Fj.jmb.2009.07.093" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=392&publication_year=2009&pages=1102-1115&author=T.+Wangauthor=Y.+Duan&title=Ligand+entry+and+exit+pathways+in+the+%CE%B22-adrenergic+receptor&doi=10.1016%2Fj.jmb.2009.07.093"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2009.07.093&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2009.07.093%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DT.%26aulast%3DDuan%26aufirst%3DY.%26atitle%3DLigand%2520entry%2520and%2520exit%2520pathways%2520in%2520the%2520%25CE%25B22-adrenergic%2520receptor%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2009%26volume%3D392%26spage%3D1102%26epage%3D1115%26doi%3D10.1016%2Fj.jmb.2009.07.093" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group">Seddon, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casey, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Law, R. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gee, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Templer, R. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ces, O.</span><span> </span><span class="NLM_article-title">Drug interactions with lipid membranes</span> <span class="citation_source-journal">Chem. Soc. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">38</span><span class="NLM_x">, </span> <span class="NLM_fpage">2509</span><span class="NLM_x">–</span> <span class="NLM_lpage">2519</span><span class="refDoi"> DOI: 10.1039/b813853m</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1039%2Fb813853m" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=19690732" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVSlsLbL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2009&pages=2509-2519&author=A.+M.+Seddonauthor=D.+Caseyauthor=R.+V.+Lawauthor=A.+Geeauthor=R.+H.+Templerauthor=O.+Ces&title=Drug+interactions+with+lipid+membranes&doi=10.1039%2Fb813853m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Drug interactions with lipid membranes</span></div><div class="casAuthors">Seddon, Annela M.; Casey, Duncan; Law, Robert V.; Gee, Antony; Templer, Richard H.; Ces, Oscar</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Society Reviews</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2509-2519</span>CODEN:
                <span class="NLM_cas:coden">CSRVBR</span>;
        ISSN:<span class="NLM_cas:issn">0306-0012</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  The field of drug-membrane interactions is one that spans a wide range of scientific disciplines, from synthetic chem., through biophysics to pharmacol.  Cell membranes are complex dynamic systems whose structures can be affected by drug mols. and in turn can affect the pharmacol. properties of the drugs being administered.  In this tutorial review we aim to provide a guide for those new to the area of drug-membrane interactions and present an introduction to areas of this topic which need to be considered.  We address the lipid compn. and structure of the cell membrane and comment on the phys. forces present in the membrane which may impact on drug interactions.  We outline methods by which drugs may cross or bind to this membrane, including the well understood passive and active transport pathways.  We present a range of techniques which may be used to study the interactions of drugs with membranes both in vitro and in vivo and discuss the advantages and disadvantages of these techniques and highlight new methods being developed to further this field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyed_OqOLrjrVg90H21EOLACvtfcHk0ljWeeX0bBaMRg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVSlsLbL&md5=81e81a98d8de9c9a889ea378f927411b</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1039%2Fb813853m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fb813853m%26sid%3Dliteratum%253Aachs%26aulast%3DSeddon%26aufirst%3DA.%2BM.%26aulast%3DCasey%26aufirst%3DD.%26aulast%3DLaw%26aufirst%3DR.%2BV.%26aulast%3DGee%26aufirst%3DA.%26aulast%3DTempler%26aufirst%3DR.%2BH.%26aulast%3DCes%26aufirst%3DO.%26atitle%3DDrug%2520interactions%2520with%2520lipid%2520membranes%26jtitle%3DChem.%2520Soc.%2520Rev.%26date%3D2009%26volume%3D38%26spage%3D2509%26epage%3D2519%26doi%3D10.1039%2Fb813853m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group">Chen, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.-B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.-Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ren, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, C.-M.</span><span> </span><span class="NLM_article-title">Green tea polyphenol epigallocatechin-3-gallate (EGCG) induced intermolecular cross-linking of membrane proteins</span> <span class="citation_source-journal">Arch. Biochem. Biophys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">507</span><span class="NLM_x">, </span> <span class="NLM_fpage">343</span><span class="NLM_x">–</span> <span class="NLM_lpage">349</span><span class="refDoi"> DOI: 10.1016/j.abb.2010.12.033</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1016%2Fj.abb.2010.12.033" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=21211509" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BC3MXit12jt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=507&publication_year=2011&pages=343-349&author=R.+Chenauthor=J.-B.+Wangauthor=X.-Q.+Zhangauthor=J.+Renauthor=C.-M.+Zeng&title=Green+tea+polyphenol+epigallocatechin-3-gallate+%28EGCG%29+induced+intermolecular+cross-linking+of+membrane+proteins&doi=10.1016%2Fj.abb.2010.12.033"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Green tea polyphenol epigallocatechin-3-gallate (EGCG) induced intermolecular cross-linking of membrane proteins</span></div><div class="casAuthors">Chen, Rong; Wang, Jian-Bo; Zhang, Xian-Qing; Ren, Jing; Zeng, Cheng-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Archives of Biochemistry and Biophysics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">507</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">343-349</span>CODEN:
                <span class="NLM_cas:coden">ABBIA4</span>;
        ISSN:<span class="NLM_cas:issn">0003-9861</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Increasing evidence has demonstrated that EGCG possesses prooxidant potential in biol. systems, including modifying proteins, breaking DNA strands and inducing the generation of reactive oxygen species.  In the present study, the prooxidant effect of EGCG on erythrocyte membranes was investigated.  SDS-PAGE and NBT-staining assay were utilized to detect the catechol-protein adducts that generated upon treating the membranes with EGCG.  The results indicated that EGCG was able to bind covalently to sulfhydryl groups of membrane proteins, leading to the formation of protein aggregates with intermol. crosslinking.  We suggested that the catechol-quinone originated from the oxidn. of EGCG acted as a cross-linker on which peptide chains were combined through thiol-S-alkylation at the C2- and C6-sites of the gallyl ring.  EGC showed similar effects as EGCG on the ghost membranes, whereas ECG and EC did not, suggesting that a structure with a gallyl moiety is a prerequisite for a catechin to induce the aggregation of membrane proteins and to deplete membrane sulfhydryls.  EDTA and ascorbic acid inhibited the EGCG-induced aggregation of membrane proteins by blocking the formation of catechol-quinone.  The information of the present study may provide a fresh insight into the prooxidant effect and cytotoxicity of tea catechins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphG6UeJYYteLVg90H21EOLACvtfcHk0lj8hsF8cMzILA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXit12jt74%253D&md5=eeb20fbc69ad825c696971d8f93ed143</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.abb.2010.12.033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.abb.2010.12.033%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DR.%26aulast%3DWang%26aufirst%3DJ.-B.%26aulast%3DZhang%26aufirst%3DX.-Q.%26aulast%3DRen%26aufirst%3DJ.%26aulast%3DZeng%26aufirst%3DC.-M.%26atitle%3DGreen%2520tea%2520polyphenol%2520epigallocatechin-3-gallate%2520%2528EGCG%2529%2520induced%2520intermolecular%2520cross-linking%2520of%2520membrane%2520proteins%26jtitle%3DArch.%2520Biochem.%2520Biophys.%26date%3D2011%26volume%3D507%26spage%3D343%26epage%3D349%26doi%3D10.1016%2Fj.abb.2010.12.033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group">Ingólfsson, H. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thakur, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herold, K. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hobart, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramsey, N. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Periole, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Jong, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zwama, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yilmaz, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hall, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maretzky, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hemmings, H. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blobel, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marrink, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koçer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sack, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Andersen, O. S.</span><span> </span><span class="NLM_article-title">Phytochemicals perturb membranes and promiscuously alter protein function</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">1788</span><span class="NLM_x">–</span> <span class="NLM_lpage">1798</span><span class="refDoi"> DOI: 10.1021/cb500086e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb500086e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BC2cXptlyqt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=1788-1798&author=H.+I.+Ing%C3%B3lfssonauthor=P.+Thakurauthor=K.+F.+Heroldauthor=E.+A.+Hobartauthor=N.+B.+Ramseyauthor=X.+Perioleauthor=D.+H.+de+Jongauthor=M.+Zwamaauthor=D.+Yilmazauthor=K.+Hallauthor=T.+Maretzkyauthor=H.+C.+Hemmingsauthor=C.+Blobelauthor=S.+J.+Marrinkauthor=A.+Ko%C3%A7erauthor=J.+T.+Sackauthor=O.+S.+Andersen&title=Phytochemicals+perturb+membranes+and+promiscuously+alter+protein+function&doi=10.1021%2Fcb500086e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Phytochemicals Perturb Membranes and Promiscuously Alter Protein Function</span></div><div class="casAuthors">Ingolfsson, Helgi I.; Thakur, Pratima; Herold, Karl F.; Hobart, E. Ashley; Ramsey, Nicole B.; Periole, Xavier; de Jong, Djurre H.; Zwama, Martijn; Yilmaz, Duygu; Hall, Katherine; Maretzky, Thorsten; Hemmings, Hugh C.; Blobel, Carl; Marrink, Siewert J.; Kocer, Armagan; Sack, Jon T.; Andersen, Olaf S.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1788-1798</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A wide variety of phytochems. are consumed for their perceived health benefits.  Many of these phytochems. have been found to alter numerous cell functions, but the mechanisms underlying their biol. activity tend to be poorly understood.  Phenolic phytochems. are particularly promiscuous modifiers of membrane protein function, suggesting that some of their actions may be due to a common, membrane bilayer-mediated mechanism.  To test whether bilayer perturbation may underlie this diversity of actions, we examd. five bioactive phenols reported to have medicinal value: capsaicin from chili peppers, curcumin from turmeric, EGCG from green tea, genistein from soybeans, and resveratrol from grapes.  We find that each of these widely consumed phytochems. alters lipid bilayer properties and the function of diverse membrane proteins.  Mol. dynamics simulations show that these phytochems. modify bilayer properties by localizing to the bilayer/soln. interface.  Bilayer-modifying propensity was verified using a gramicidin-based assay, and indiscriminate modulation of membrane protein function was demonstrated using four proteins: membrane-anchored metalloproteases, mechanosensitive ion channels, and voltage-dependent potassium and sodium channels.  Each protein exhibited similar responses to multiple phytochems., consistent with a common, bilayer-mediated mechanism.  Our results suggest that many effects of amphiphilic phytochems. are due to cell membrane perturbations, rather than specific protein binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrL0NNb8_Q6qrVg90H21EOLACvtfcHk0lj8hsF8cMzILA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXptlyqt7s%253D&md5=b61876239e926b652adea2e130627bc3</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Fcb500086e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb500086e%26sid%3Dliteratum%253Aachs%26aulast%3DIng%25C3%25B3lfsson%26aufirst%3DH.%2BI.%26aulast%3DThakur%26aufirst%3DP.%26aulast%3DHerold%26aufirst%3DK.%2BF.%26aulast%3DHobart%26aufirst%3DE.%2BA.%26aulast%3DRamsey%26aufirst%3DN.%2BB.%26aulast%3DPeriole%26aufirst%3DX.%26aulast%3Dde%2BJong%26aufirst%3DD.%2BH.%26aulast%3DZwama%26aufirst%3DM.%26aulast%3DYilmaz%26aufirst%3DD.%26aulast%3DHall%26aufirst%3DK.%26aulast%3DMaretzky%26aufirst%3DT.%26aulast%3DHemmings%26aufirst%3DH.%2BC.%26aulast%3DBlobel%26aufirst%3DC.%26aulast%3DMarrink%26aufirst%3DS.%2BJ.%26aulast%3DKo%25C3%25A7er%26aufirst%3DA.%26aulast%3DSack%26aufirst%3DJ.%2BT.%26aulast%3DAndersen%26aufirst%3DO.%2BS.%26atitle%3DPhytochemicals%2520perturb%2520membranes%2520and%2520promiscuously%2520alter%2520protein%2520function%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2014%26volume%3D9%26spage%3D1788%26epage%3D1798%26doi%3D10.1021%2Fcb500086e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group">Jiang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reilly, J.</span><span> </span><span class="NLM_article-title">Chromatography approaches for early screening of the phospholipidosis-inducing potential of pharmaceuticals</span> <span class="citation_source-journal">J. Pharm. Biomed. Anal.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">61</span><span class="NLM_x">, </span> <span class="NLM_fpage">184</span><span class="NLM_x">–</span> <span class="NLM_lpage">190</span><span class="refDoi"> DOI: 10.1016/j.jpba.2011.11.033</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1016%2Fj.jpba.2011.11.033" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=22200505" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1aiu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2012&pages=184-190&author=Z.+Jiangauthor=J.+Reilly&title=Chromatography+approaches+for+early+screening+of+the+phospholipidosis-inducing+potential+of+pharmaceuticals&doi=10.1016%2Fj.jpba.2011.11.033"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Chromatography approaches for early screening of the phospholipidosis-inducing potential of pharmaceuticals</span></div><div class="casAuthors">Jiang, Zhengjin; Reilly, John</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmaceutical and Biomedical Analysis</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">184-190</span>CODEN:
                <span class="NLM_cas:coden">JPBADA</span>;
        ISSN:<span class="NLM_cas:issn">0731-7085</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Drug-induced phospholipidosis (PLD) is an excessive accumulation of polar phospholipids within cells as a result of medical treatment.  Even though a direct link between in vitro drug-induced PLD and toxicity in humans has not yet been firmly established, the development of PLD during preclin. testing in animals is a recognized problem in the pharmaceutical industry and can delay or abort the development process.  Therefore, it is of interest to investigate the potential PLD risk of candidates at an early stage of the drug discovery process.  In this work, a high throughput physicochem. approach, which is based on measuring the retention factors of the test compd. within several different sepn. systems, was developed for screening phospholipidosis-inducing potential (PLIP) of pharmaceuticals.  The measured retention factors of 36 drugs were compared with literature data on PLIP risk from three different sources.  It is clearly shown that there is a statistical correlation between the chromatog. retention parameters of tested drugs and their PLIP risk.  In conclusion, the retention factor (log k AOT) obsd. on a docusate sodium salt (AOT) surfactant vesicle electrokinetic chromatog. (EKC) system and the logarithm of the vol. of distribution (log V d) calcd. from immobilized artificial membrane chromatog. at pH 7.4 (CHI IAM7.4) and HSA binding value (% HSA) can provide primary profile prediction for a large no. of drug candidates early in the drug discovery process with minimal resources.  The observations are that the higher the value of both log k AOT and log V d, the higher the PLIP risk, and we would recommend this dual approach.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoutkXEtBQfHLVg90H21EOLACvtfcHk0lj8hsF8cMzILA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1aiu78%253D&md5=828cf8ec32952b5b01734816d69fafc4</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1016%2Fj.jpba.2011.11.033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jpba.2011.11.033%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DZ.%26aulast%3DReilly%26aufirst%3DJ.%26atitle%3DChromatography%2520approaches%2520for%2520early%2520screening%2520of%2520the%2520phospholipidosis-inducing%2520potential%2520of%2520pharmaceuticals%26jtitle%3DJ.%2520Pharm.%2520Biomed.%2520Anal.%26date%3D2012%26volume%3D61%26spage%3D184%26epage%3D190%26doi%3D10.1016%2Fj.jpba.2011.11.033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group">Reasor, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hastings, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ulrich, R. G.</span><span> </span><span class="NLM_article-title">Drug-induced phospholipidosis: issues and future directions</span> <span class="citation_source-journal">Expert Opin. Drug Saf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">567</span><span class="NLM_x">–</span> <span class="NLM_lpage">583</span><span class="refDoi"> DOI: 10.1517/14740338.5.4.567</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1517%2F14740338.5.4.567" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=16774494" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BD28XmslSksL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2006&pages=567-583&author=M.+J.+Reasorauthor=K.+L.+Hastingsauthor=R.+G.+Ulrich&title=Drug-induced+phospholipidosis%3A+issues+and+future+directions&doi=10.1517%2F14740338.5.4.567"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Drug-induced phospholipidosis: issues and future directions</span></div><div class="casAuthors">Reasor, Mark J.; Hastings, Kenneth L.; Ulrich, Roger G.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Safety</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">567-583</span>CODEN:
                <span class="NLM_cas:coden">EODSA9</span>;
        ISSN:<span class="NLM_cas:issn">1474-0338</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Numerous drugs contg. a cationic amphiphilic structure are capable of inducing phospholipidosis in cells under conditions of in vivo administration or ex vivo incubation.  The principal characteristics of this condition include the reversible accumulation of polar phospholipids in assocn. with the development of unicentric or multicentric lamellated bodies within cells.  There is an abundance of data providing an understanding of potential mechanisms for the induction of phospholipidosis; however, the process is likely to be complex and may differ from one drug to another.  The functional consequences of the presence of this condition on cellular or tissue function are not well understood.  The general consensus is that the condition is an adaptive response rather than a toxicol. manifestation; however, addnl. studies to examine this question are needed.  Until this issue is resolved, concerns about phospholipidosis will continue to exist at regulatory agencies.  Procedures for the screening of potential phospholipogenic candidate compds. are available.  In contrast, a clear need exists for the identification of valid biomarkers to assess the development of phospholipidosis in preclin. and clin. studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwYtmd7pqUebVg90H21EOLACvtfcHk0lj8hsF8cMzILA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XmslSksL8%253D&md5=a2801d85bc4f1730f69272eaea3ce17b</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1517%2F14740338.5.4.567&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F14740338.5.4.567%26sid%3Dliteratum%253Aachs%26aulast%3DReasor%26aufirst%3DM.%2BJ.%26aulast%3DHastings%26aufirst%3DK.%2BL.%26aulast%3DUlrich%26aufirst%3DR.%2BG.%26atitle%3DDrug-induced%2520phospholipidosis%253A%2520issues%2520and%2520future%2520directions%26jtitle%3DExpert%2520Opin.%2520Drug%2520Saf.%26date%3D2006%26volume%3D5%26spage%3D567%26epage%3D583%26doi%3D10.1517%2F14740338.5.4.567" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group">Reasor, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCloud, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beard, T. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ebert, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kacew, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gardner, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aldern, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hostetler, K. Y.</span><span> </span><span class="NLM_article-title">Comparative evaluation of amiodarone-induced phospholipidosis and drug accumulation in Fischer-344 and Sprague-Dawley rats</span> <span class="citation_source-journal">Toxicology</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">106</span><span class="NLM_x">, </span> <span class="NLM_fpage">139</span><span class="NLM_x">–</span> <span class="NLM_lpage">147</span><span class="refDoi"> DOI: 10.1016/0300-483X(95)03175-F</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1016%2F0300-483X%2895%2903175-F" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=8571385" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADyaK2MXpvV2nsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=1996&pages=139-147&author=M.+J.+Reasorauthor=C.+M.+McCloudauthor=T.+L.+Beardauthor=D.+C.+Ebertauthor=S.+Kacewauthor=M.+F.+Gardnerauthor=K.+A.+Aldernauthor=K.+Y.+Hostetler&title=Comparative+evaluation+of+amiodarone-induced+phospholipidosis+and+drug+accumulation+in+Fischer-344+and+Sprague-Dawley+rats&doi=10.1016%2F0300-483X%2895%2903175-F"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Comparative evaluation of amiodarone-induced phospholipidosis and drug accumulation in Fischer-344 and Sprague-Dawley rats</span></div><div class="casAuthors">Reasor, Mark J.; McCloud, Christy M.; Beard, Terry L.; Ebert, Debora C.; Kacew, Sam; Gardner, Michael F.; Aldern, Kathy A.; Hostetler, Karl Y.</div><div class="citationInfo"><span class="NLM_cas:title">Toxicology</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">139-47</span>CODEN:
                <span class="NLM_cas:coden">TXCYAC</span>;
        ISSN:<span class="NLM_cas:issn">0300-483X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Amiodarone (AD) and its major metabolite, desethylamiodarone (desethylAD), are both phospholipogenic.  The present study was undertaken to evaluate the comparative susceptibilities of male Fischer-344 and Sprague-Dawley rats to AD-induced phospholipidosis in alveolar macrophages (AMs), liver and kidney tissue and the concomitant accumulation of AD and desethylAD in these cells, tissues and plasma.  Rats were administered AD (100 mg/kg/day, p.o.) for 1 wk.  Plasma concns. of AD and desethylAD were approx. 4- and 12-fold higher, resp., in Fischer-344s compared to Sprague-Dawleys 24 h after the last dose.  AD and desethylAD levels in AMs were approx. 12- and 25-fold higher, resp., in Fischer-344s than Sprague-Dawleys.  In the liver and kidney, levels of both compds. were also significantly higher in Fischer-344s than Sprague-Dawleys.  Ultrastructural features indicative of phospholipidosis were not obsd. consistently in any tissue except AMs from treated Fischer-344s.  AM total phospholipid increased nearly 5-fold in Fischer-344s, while Sprague-Dawleys showed no increase over control.  AMs from both strains incubated with 10 μM AD or desethylAD in vitro were not significantly different in their accumulation of the compds.  When incubated with AD or desethylAD, the lysosomal phospholipases A1 partially purified from AMs of both strains were equally sensitive to inhibition as measured by the drug concn. giving 50% inhibition in activity (IC50).  The results of this study indicate that at the same administered dose, AD and desethylAD, accumulate to higher tissue levels and are more phospholipogenic in male Fischer-344 rats than in male Sprague-Dawley rats.  The basis for the high susceptibility of Fischer-344 rats to AM-induced phospholipidosis is unknown at present but appears not to be related to biochem. or cellular features of the AMs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSyPj0Leb8jrVg90H21EOLACvtfcHk0ljm_-cd8Ys0_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXpvV2nsb8%253D&md5=16395ca5e1a13917d6085b81ec7fec59</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1016%2F0300-483X%2895%2903175-F&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0300-483X%252895%252903175-F%26sid%3Dliteratum%253Aachs%26aulast%3DReasor%26aufirst%3DM.%2BJ.%26aulast%3DMcCloud%26aufirst%3DC.%2BM.%26aulast%3DBeard%26aufirst%3DT.%2BL.%26aulast%3DEbert%26aufirst%3DD.%2BC.%26aulast%3DKacew%26aufirst%3DS.%26aulast%3DGardner%26aufirst%3DM.%2BF.%26aulast%3DAldern%26aufirst%3DK.%2BA.%26aulast%3DHostetler%26aufirst%3DK.%2BY.%26atitle%3DComparative%2520evaluation%2520of%2520amiodarone-induced%2520phospholipidosis%2520and%2520drug%2520accumulation%2520in%2520Fischer-344%2520and%2520Sprague-Dawley%2520rats%26jtitle%3DToxicology%26date%3D1996%26volume%3D106%26spage%3D139%26epage%3D147%26doi%3D10.1016%2F0300-483X%2895%2903175-F" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group">Williard, J.</span><span> </span><span class="NLM_article-title">FDA phospholipidosis working group preliminary results and developing opinions</span>, Presented at the 27th Annual Meeting of the Society of Toxicologic Pathology; <span class="NLM_publisher-loc">San Francisco, CA, USA</span>,<span class="NLM_x"> </span><span class="NLM_year">2008</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2008&author=J.+Williard&title=FDA+phospholipidosis+working+group+preliminary+results+and+developing+opinions"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dconf-proc%26sid%3Dliteratum%253Aachs%26aulast%3DWilliard%26aufirst%3DJ.%26atitle%3DFDA%2520phospholipidosis%2520working%2520group%2520preliminary%2520results%2520and%2520developing%2520opinions%26jtitle%3D27th%2520Annual%2520Meeting%2520of%2520the%2520Society%2520of%2520Toxicologic%2520Pathology%26date%3D2008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group">Bernstein, P. R.; Ciaccio, P.; Morelli, J.</span><span> </span><span class="NLM_article-title">Drug-induced phospholipidosis</span>. In  <span class="citation_source-book">Annual Reports in Medicinal Chemistry</span>; <span class="NLM_contrib-group">John, E. M.</span>, Ed.; <span class="NLM_publisher-name">Academic Press</span>: <span class="NLM_publisher-loc">Cambridge, MA</span>,<span class="NLM_x"> </span><span class="NLM_year">2011</span><span class="NLM_x">; </span>Vol.  <span class="NLM_volume">46</span>, Chapter 25, pp  <span class="NLM_fpage">419</span><span class="NLM_x">–</span> <span class="NLM_lpage">430</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1016%2FB978-0-12-386009-5.00001-1" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2011&pages=419-430&author=P.+R.+Bernstein&author=P.+Ciaccio&author=J.+Morelliauthor=E.+M.+John&title=Annual+Reports+in+Medicinal+Chemistry"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1016%2FB978-0-12-386009-5.00001-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1016%252FB978-0-12-386009-5.00001-1%26sid%3Dliteratum%253Aachs%26aulast%3DBernstein%26aufirst%3DP.%2BR.%26atitle%3DDrug-induced%2520phospholipidosis%26btitle%3DAnnual%2520Reports%2520in%2520Medicinal%2520Chemistry%26aulast%3DJohn%26aufirst%3DE.%2BM.%26pub%3DAcademic%2520Press%26date%3D2011%26volume%3D46%26spage%3D419%26epage%3D430" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group">Barone, L. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boyer, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Damewood, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fikes, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciaccio, P. J.</span><span> </span><span class="NLM_article-title">Phospholipogenic pharmaceuticals are associated with a higher incidence of histological findings than nonphospholipogenic pharmaceuticals in preclinical toxicology studies</span> <span class="citation_source-journal">J. Toxicol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">2012</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">7</span><span class="refDoi"> DOI: 10.1155/2012/308594</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1155%2F2012%2F308594" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2012&publication_year=2012&pages=1-7&author=L.+R.+Baroneauthor=S.+Boyerauthor=J.+R.+Damewoodauthor=J.+Fikesauthor=P.+J.+Ciaccio&title=Phospholipogenic+pharmaceuticals+are+associated+with+a+higher+incidence+of+histological+findings+than+nonphospholipogenic+pharmaceuticals+in+preclinical+toxicology+studies&doi=10.1155%2F2012%2F308594"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1155%2F2012%2F308594&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1155%252F2012%252F308594%26sid%3Dliteratum%253Aachs%26aulast%3DBarone%26aufirst%3DL.%2BR.%26aulast%3DBoyer%26aufirst%3DS.%26aulast%3DDamewood%26aufirst%3DJ.%2BR.%26aulast%3DFikes%26aufirst%3DJ.%26aulast%3DCiaccio%26aufirst%3DP.%2BJ.%26atitle%3DPhospholipogenic%2520pharmaceuticals%2520are%2520associated%2520with%2520a%2520higher%2520incidence%2520of%2520histological%2520findings%2520than%2520nonphospholipogenic%2520pharmaceuticals%2520in%2520preclinical%2520toxicology%2520studies%26jtitle%3DJ.%2520Toxicol.%26date%3D2012%26volume%3D2012%26spage%3D1%26epage%3D7%26doi%3D10.1155%2F2012%2F308594" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group">Cherezov, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosenbaum, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hanson, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rasmussen, S. G. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thian, F. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobilka, T. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choi, H.-J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhn, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weis, W. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobilka, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stevens, R. C.</span><span> </span><span class="NLM_article-title">High-resolution crystal structure of an engineered human β2-adrenergic G protein–coupled receptor</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">318</span><span class="NLM_x">, </span> <span class="NLM_fpage">1258</span><span class="NLM_x">–</span> <span class="NLM_lpage">1265</span><span class="refDoi"> DOI: 10.1126/science.1150577</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1126%2Fscience.1150577" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=17962520" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlGmur7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=318&publication_year=2007&pages=1258-1265&author=V.+Cherezovauthor=D.+M.+Rosenbaumauthor=M.+A.+Hansonauthor=S.+G.+F.+Rasmussenauthor=F.+S.+Thianauthor=T.+S.+Kobilkaauthor=H.-J.+Choiauthor=P.+Kuhnauthor=W.+I.+Weisauthor=B.+K.+Kobilkaauthor=R.+C.+Stevens&title=High-resolution+crystal+structure+of+an+engineered+human+%CE%B22-adrenergic+G+protein%E2%80%93coupled+receptor&doi=10.1126%2Fscience.1150577"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">High-Resolution Crystal Structure of an Engineered Human β2-Adrenergic G Protein-Coupled Receptor</span></div><div class="casAuthors">Cherezov, Vadim; Rosenbaum, Daniel M.; Hanson, Michael A.; Rasmussen, Soren G. F.; Thian, Foon Sun; Kobilka, Tong Sun; Choi, Hee-Jung; Kuhn, Peter; Weis, William I.; Kobilka, Brian K.; Stevens, Raymond C.; Takeda, S.; Kadowaki, S.; Haga, T.; Takaesu, H.; Mitaku, S.; Fredriksson, R.; Lagerstrom, M. C.; Lundin, L. G.; Schioth, H. B.; Pierce, K. L.; Premont, R. T.; Lefkowitz, R. J.; Lefkowitz, R. J.; Shenoy, S. K.; Rosenbaum, D. M.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">318</span>
        (<span class="NLM_cas:issue">5854</span>),
    <span class="NLM_cas:pages">1258-1265</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Second extracellular loop, which in the β2-adrenergic receptor contains an unusual pair of disulfide bonds and an extra helix.  This loop and the absence Heterotrimeric guanine nucleotide-binding protein (G protein)-coupled receptors constitute the largest family of eukaryotic signal transduction proteins that communicate across the membrane.  We report the crystal structure of a human β2-adrenergic receptor-T4 lysozyme fusion protein bound to the partial inverse agonist carazolol at 2.4 angstrom resoln.  The structure provides a high-resoln. view of a human G protein-coupled receptor bound to a diffusible ligand.  Ligand-binding site accessibility is enabled by the second extracellular loop, which is held out of the binding cavity by a pair of closely spaced disulfide bridges and a short helical segment within the loop.  Cholesterol, a necessary component for crystn., mediates an intriguing parallel assocn. of receptor mols. in the crystal lattice.  Although the location of carazolol in the β2-adrenergic receptor is very similar to that of retinal in rhodopsin, structural differences in the ligand-binding site and other regions highlight the challenges in using rhodopsin as a template model for this large receptor family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpC-8wZ2k2GHLVg90H21EOLACvtfcHk0ljm_-cd8Ys0_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlGmur7I&md5=12c5bacb8464a4b243fe9341192b5b3b</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1126%2Fscience.1150577&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1150577%26sid%3Dliteratum%253Aachs%26aulast%3DCherezov%26aufirst%3DV.%26aulast%3DRosenbaum%26aufirst%3DD.%2BM.%26aulast%3DHanson%26aufirst%3DM.%2BA.%26aulast%3DRasmussen%26aufirst%3DS.%2BG.%2BF.%26aulast%3DThian%26aufirst%3DF.%2BS.%26aulast%3DKobilka%26aufirst%3DT.%2BS.%26aulast%3DChoi%26aufirst%3DH.-J.%26aulast%3DKuhn%26aufirst%3DP.%26aulast%3DWeis%26aufirst%3DW.%2BI.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26atitle%3DHigh-resolution%2520crystal%2520structure%2520of%2520an%2520engineered%2520human%2520%25CE%25B22-adrenergic%2520G%2520protein%25E2%2580%2593coupled%2520receptor%26jtitle%3DScience%26date%3D2007%26volume%3D318%26spage%3D1258%26epage%3D1265%26doi%3D10.1126%2Fscience.1150577" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group">Lomize, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lomize, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pogozheva, I. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mosberg, H. I.</span><span> </span><span class="NLM_article-title">OPM: orientations of proteins in membranes database</span> <span class="citation_source-journal">Bioinformatics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">623</span><span class="NLM_x">–</span> <span class="NLM_lpage">625</span><span class="refDoi"> DOI: 10.1093/bioinformatics/btk023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1093%2Fbioinformatics%2Fbtk023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=16397007" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BD28XhvVeksro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2006&pages=623-625&author=M.+A.+Lomizeauthor=A.+L.+Lomizeauthor=I.+D.+Pogozhevaauthor=H.+I.+Mosberg&title=OPM%3A+orientations+of+proteins+in+membranes+database&doi=10.1093%2Fbioinformatics%2Fbtk023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">OPM: Orientations of Proteins in Membranes database</span></div><div class="casAuthors">Lomize, Mikhail A.; Lomize, Andrei L.; Pogozheva, Irina D.; Mosberg, Henry I.</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">623-625</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The Orientations of Proteins in Membranes (OPM) database provides a collection of transmembrane, monotopic, and peripheral proteins from the Protein Data Bank whose spatial arrangements in the lipid bilayer have been calcd. theor. and compared with exptl. data.  The database allows anal., sorting, and searching of membrane proteins based on their structural classification, species, destination membrane, nos. of transmembrane segments and subunits, nos. of secondary structures, and the calcd. hydrophobic thickness or tilt angle with respect to the bilayer normal.  All coordinate files with the calcd. membrane boundaries are available for downloading.  The Internet site for the database is: http://opm.phar.umich.edu.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1WjpHiq7e8LVg90H21EOLACvtfcHk0lhLBA2TSCM8IA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhvVeksro%253D&md5=8a9590a32e912509a53ca0bfc7ede4ad</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtk023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtk023%26sid%3Dliteratum%253Aachs%26aulast%3DLomize%26aufirst%3DM.%2BA.%26aulast%3DLomize%26aufirst%3DA.%2BL.%26aulast%3DPogozheva%26aufirst%3DI.%2BD.%26aulast%3DMosberg%26aufirst%3DH.%2BI.%26atitle%3DOPM%253A%2520orientations%2520of%2520proteins%2520in%2520membranes%2520database%26jtitle%3DBioinformatics%26date%3D2006%26volume%3D22%26spage%3D623%26epage%3D625%26doi%3D10.1093%2Fbioinformatics%2Fbtk023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group">Zou, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weis, W. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobilka, B. K.</span><span> </span><span class="NLM_article-title">N-terminal T4 lysozyme fusion facilitates crystallization of a G protein coupled receptor</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">e46039</span><span class="refDoi"> DOI: 10.1371/journal.pone.0046039</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1371%2Fjournal.pone.0046039" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=23056231" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFaksr%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=e46039&author=Y.+Zouauthor=W.+I.+Weisauthor=B.+K.+Kobilka&title=N-terminal+T4+lysozyme+fusion+facilitates+crystallization+of+a+G+protein+coupled+receptor&doi=10.1371%2Fjournal.pone.0046039"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">N-Terminal T4 lysozyme fusion facilitates crystallization of a G protein coupled receptor</span></div><div class="casAuthors">Zou, Yaozhong; Weis, William I.; Kobilka, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">e46039</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">A highly crystallizable phage T4 lysozyme (T4L) was fused to the N-terminus of the β2 adrenergic receptor (β2AR), a G-protein coupled receptor (GPCR) for catecholamines.  It was demonstrated that the N-terminal fused T4L was sufficiently rigid relative to the receptor to facilitate crystallogenesis without thermostabilizing mutations or the use of a stabilizing antibody, G protein, or protein fused to the 3rd intracellular loop.  This approach adds to the protein engineering strategies that enable crystallog. studies of GPCRs alone or in complex with a signaling partner.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruYBtPrpjek7Vg90H21EOLACvtfcHk0lhLBA2TSCM8IA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFaksr%252FM&md5=058b6ea877cef9d7f86cf0894c965f23</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0046039&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0046039%26sid%3Dliteratum%253Aachs%26aulast%3DZou%26aufirst%3DY.%26aulast%3DWeis%26aufirst%3DW.%2BI.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26atitle%3DN-terminal%2520T4%2520lysozyme%2520fusion%2520facilitates%2520crystallization%2520of%2520a%2520G%2520protein%2520coupled%2520receptor%26jtitle%3DPLoS%2520One%26date%3D2012%26volume%3D7%26spage%3De46039%26doi%3D10.1371%2Fjournal.pone.0046039" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span>Molecular Operating Environment (MOE); <span class="NLM_publisher-name">Chemical Computing Group Inc.</span>, <span class="NLM_publisher-loc">1010 Sherbooke St. West, Suite #910, Montreal, QC, Canada, H3A 2R7</span>,<span class="NLM_x"> </span><span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Molecular+Operating+Environment+%28MOE%29%3B+Chemical+Computing+Group+Inc.%2C+1010+Sherbooke+St.+West%2C+Suite+%23910%2C+Montreal%2C+QC%2C+Canada%2C+H3A+2R7%2C+2015."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Operating%2520Environment%2520%2528MOE%2529%26pub%3DChemical%2520Computing%2520Group%2520Inc%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group">Sindhikara, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirata, F.</span><span> </span><span class="NLM_article-title">Analysis of biomolecular solvation sites by 3D-RISM theory</span> <span class="citation_source-journal">J. Phys. Chem. B</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">117</span><span class="NLM_x">, </span> <span class="NLM_fpage">6718</span><span class="NLM_x">–</span> <span class="NLM_lpage">6723</span><span class="refDoi"> DOI: 10.1021/jp4046116</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp4046116" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BC3sXns1Gms74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=117&publication_year=2013&pages=6718-6723&author=D.+J.+Sindhikaraauthor=F.+Hirata&title=Analysis+of+biomolecular+solvation+sites+by+3D-RISM+theory&doi=10.1021%2Fjp4046116"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Analysis of Biomolecular Solvation Sites by 3D-RISM Theory</span></div><div class="casAuthors">Sindhikara, Daniel J.; Hirata, Fumio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">117</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">6718-6723</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1520-5207</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We derive, implement, and apply equil. solvation site anal. for biomols.  Our method utilizes 3D-RISM calcns. to quickly obtain equil. solvent distributions without either necessity of simulation or limits of solvent sampling.  Our anal. of these distributions exts. highest likelihood poses of solvent as well as localized entropies, enthalpies, and solvation free energies.  We demonstrate our method on a structure of HIV-1 protease where excellent structural and thermodn. data are available for comparison.  Our results, obtained within minutes, show systematic agreement with available exptl. data.  Further, our results are in good agreement with established simulation-based solvent anal. methods.  This method can be used not only for visual anal. of active site solvation but also for virtual screening methods and exptl. refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6CAgZkVvDJLVg90H21EOLACvtfcHk0lhLBA2TSCM8IA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXns1Gms74%253D&md5=f709823750c912702cee0653b9170662</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1021%2Fjp4046116&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp4046116%26sid%3Dliteratum%253Aachs%26aulast%3DSindhikara%26aufirst%3DD.%2BJ.%26aulast%3DHirata%26aufirst%3DF.%26atitle%3DAnalysis%2520of%2520biomolecular%2520solvation%2520sites%2520by%25203D-RISM%2520theory%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2013%26volume%3D117%26spage%3D6718%26epage%3D6723%26doi%3D10.1021%2Fjp4046116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group">Sindhikara, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoshida, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirata, F.</span><span> </span><span class="NLM_article-title">Placevent: an algorithm for prediction of explicit solvent atom distribution—application to HIV-1 protease and F-ATP synthase</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x">, </span> <span class="NLM_fpage">1536</span><span class="NLM_x">–</span> <span class="NLM_lpage">1543</span><span class="refDoi"> DOI: 10.1002/jcc.22984</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1002%2Fjcc.22984" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=22522665" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BC38XlslyktLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2012&pages=1536-1543&author=D.+J.+Sindhikaraauthor=N.+Yoshidaauthor=F.+Hirata&title=Placevent%3A+an+algorithm+for+prediction+of+explicit+solvent+atom+distribution%E2%80%94application+to+HIV-1+protease+and+F-ATP+synthase&doi=10.1002%2Fjcc.22984"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Placevent: An algorithm for prediction of explicit solvent atom distribution - Application to HIV-1 protease and F-ATP synthase</span></div><div class="casAuthors">Sindhikara, Daniel J.; Yoshida, Norio; Hirata, Fumio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">1536-1543</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">We have created a simple algorithm for automatically predicting the explicit solvent atom distribution of biomols.  The explicit distribution is coerced from the three-dimensional (3D) continuous distribution resulting from a 3D ref. interaction site model (3D-RISM) calcn.  This procedure predicts optimal location of solvent mols. and ions given a rigid biomol. structure and the solvent compn.  We show examples of predicting water mols. near the KNI-272 bound form of HIV-1 protease and predicting both sodium ions and water mols. near the rotor ring of F-ATP synthase.  Our results give excellent agreement with exptl. structure with an av. prediction error of 0.39-0.65 Å.  Further, unlike exptl. methods, this method does not suffer from the partial occupancy limit.  Our method can be performed directly on 3D-RISM output within minutes.  It is extremely useful for examg. multiple specific solvent-solute interactions, as a convenient method for generating initial solvent structures for mol. dynamics calcns., and may assist in refinement of exptl. structures.  © 2012 Wiley Periodicals, Inc.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm61L3EIlF97Vg90H21EOLACvtfcHk0lhUvt3Kov1CSA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XlslyktLk%253D&md5=f80d4c1a0b2c5d87695481674da31814</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1002%2Fjcc.22984&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.22984%26sid%3Dliteratum%253Aachs%26aulast%3DSindhikara%26aufirst%3DD.%2BJ.%26aulast%3DYoshida%26aufirst%3DN.%26aulast%3DHirata%26aufirst%3DF.%26atitle%3DPlacevent%253A%2520an%2520algorithm%2520for%2520prediction%2520of%2520explicit%2520solvent%2520atom%2520distribution%25E2%2580%2594application%2520to%2520HIV-1%2520protease%2520and%2520F-ATP%2520synthase%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2012%26volume%3D33%26spage%3D1536%26epage%3D1543%26doi%3D10.1002%2Fjcc.22984" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group">Jo, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lim, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klauda, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Im, W.</span><span> </span><span class="NLM_article-title">CHARMM-GUI membrane builder for mixed bilayers and its application to yeast membranes</span> <span class="citation_source-journal">Biophys. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">97</span><span class="NLM_x">, </span> <span class="NLM_fpage">50</span><span class="NLM_x">–</span> <span class="NLM_lpage">58</span><span class="refDoi"> DOI: 10.1016/j.bpj.2009.04.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1016%2Fj.bpj.2009.04.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=19580743" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BD1MXpsFOmt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=97&publication_year=2009&pages=50-58&author=S.+Joauthor=J.+B.+Limauthor=J.+B.+Klaudaauthor=W.+Im&title=CHARMM-GUI+membrane+builder+for+mixed+bilayers+and+its+application+to+yeast+membranes&doi=10.1016%2Fj.bpj.2009.04.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">CHARMM-GUI Membrane Builder for mixed bilayers and its application to yeast membranes</span></div><div class="casAuthors">Jo, Sunhwan; Lim, Joseph B.; Klauda, Jeffery B.; Im, Wonpil</div><div class="citationInfo"><span class="NLM_cas:title">Biophysical Journal</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">97</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">50-58</span>CODEN:
                <span class="NLM_cas:coden">BIOJAU</span>;
        ISSN:<span class="NLM_cas:issn">0006-3495</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The CHARMM-GUI Membrane Builder (http://www.charmm-gui.org/input/membrane), an intuitive, straightforward, web-based graphical user interface, was expanded to automate the building process of heterogeneous lipid bilayers, with or without a protein and with support for up to 32 different lipid types.  The efficacy of these new features was tested by building and simulating lipid bilayers that resemble yeast membranes, composed of cholesterol, dipalmitoylphosphatidylcholine, dioleoylphosphatidylcholine, palmitoyloleoylphosphatidylethanolamine, palmitoyloleoylphosphatidylamine, and palmitoyloleoylphosphatidylserine.  Four membranes with varying concns. of cholesterol and phospholipids were simulated, for a total of 170 ns at 303.15 K.  Unsatd. phospholipid chain concn. had the largest influence on membrane properties, such as av. lipid surface area, d. profiles, deuterium order parameters, and cholesterol tilt angle.  Simulations with a high concn. of unsatd. chains (73%, membraneunsat) resulted in a significant increase in lipid surface area and a decrease in deuterium order parameters, compared with membranes with a high concn. of satd. chains (60-63%, membranesat).  The av. tilt angle of cholesterol with respect to bilayer normal was largest, and the distribution was significantly broader for membraneunsat.  Moreover, short-lived cholesterol orientations parallel to the membrane surface existed only for membraneunsat.  The membranesat simulations were in a liq.-ordered state, and agree with similar exptl. cholesterol-contg. membranes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxF3HFYL2Qa7Vg90H21EOLACvtfcHk0lhUvt3Kov1CSA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXpsFOmt7g%253D&md5=a6d57a8f42772e4c3674e324ea27ecaa</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1016%2Fj.bpj.2009.04.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bpj.2009.04.013%26sid%3Dliteratum%253Aachs%26aulast%3DJo%26aufirst%3DS.%26aulast%3DLim%26aufirst%3DJ.%2BB.%26aulast%3DKlauda%26aufirst%3DJ.%2BB.%26aulast%3DIm%26aufirst%3DW.%26atitle%3DCHARMM-GUI%2520membrane%2520builder%2520for%2520mixed%2520bilayers%2520and%2520its%2520application%2520to%2520yeast%2520membranes%26jtitle%3DBiophys.%2520J.%26date%3D2009%26volume%3D97%26spage%3D50%26epage%3D58%26doi%3D10.1016%2Fj.bpj.2009.04.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group">Case, D. A.; Berryman, J. T.; Betz, R. M.; Cerutti, D. S.; Cheatham,  I., T. E.; Darden, T. A.; Duke, R. E.; Giese, T. J.; Gohlke, H.; Goetz, A. W.; Homeyer, N.; Izadi, S.; Janowski, P.; Kaus, J.; Kovalenko, A.; Lee, T. S.; LeGrand, S.; Li, P.; Luchko, T.; Luo, R.; Madej, B.; Merz, K. M.; Monard, G.; Needham, P.; Nguyen, H.; Nguyen, H. T.; Omelyan, I.; Onufriev, A.; Roe, D. R.; Roitberg, A.; Salomon-Ferrer, R.; Simmerling, C. L.; Smith, W.; Swails, J.; Walker, R. C.; Wang, J.; Wolf, R. M.; Wu, X.; York, D. M.; Kollman, P. A.</span>AMBER 14; <span class="NLM_publisher-name">University of California</span>: <span class="NLM_publisher-loc">San Francisco</span>,<span class="NLM_x"> </span><span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=D.+A.+Case&author=J.+T.+Berryman&author=R.+M.+Betz&author=D.+S.+Cerutti&author=T.+E.+Cheatham&author=T.+A.+Darden&author=R.+E.+Duke&author=T.+J.+Giese&author=H.+Gohlke&author=A.+W.+Goetz&author=N.+Homeyer&author=S.+Izadi&author=P.+Janowski&author=J.+Kaus&author=A.+Kovalenko&author=T.+S.+Lee&author=S.+LeGrand&author=P.+Li&author=T.+Luchko&author=R.+Luo&author=B.+Madej&author=K.+M.+Merz&author=G.+Monard&author=P.+Needham&author=H.+Nguyen&author=H.+T.+Nguyen&author=I.+Omelyan&author=A.+Onufriev&author=D.+R.+Roe&author=A.+Roitberg&author=R.+Salomon-Ferrer&author=C.+L.+Simmerling&author=W.+Smith&author=J.+Swails&author=R.+C.+Walker&author=J.+Wang&author=R.+M.+Wolf&author=X.+Wu&author=D.+M.+York&author=P.+A.+Kollman&title=AMBER+14"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DCase%26aufirst%3DD.%2BA.%26jtitle%3DAMBER%252014%26pub%3DUniversity%2520of%2520California%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group">Ring, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Manglik, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kruse, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Enos, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weis, W. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garcia, K. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobilka, B. K.</span><span> </span><span class="NLM_article-title">Adrenaline-activated structure of β2-adrenoceptor stabilized by an engineered nanobody</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">502</span><span class="NLM_x">, </span> <span class="NLM_fpage">575</span><span class="NLM_x">–</span> <span class="NLM_lpage">579</span><span class="refDoi"> DOI: 10.1038/nature12572</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1038%2Fnature12572" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=24056936" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFSlsLzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=502&publication_year=2013&pages=575-579&author=A.+M.+Ringauthor=A.+Manglikauthor=A.+C.+Kruseauthor=M.+D.+Enosauthor=W.+I.+Weisauthor=K.+C.+Garciaauthor=B.+K.+Kobilka&title=Adrenaline-activated+structure+of+%CE%B22-adrenoceptor+stabilized+by+an+engineered+nanobody&doi=10.1038%2Fnature12572"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Adrenaline-activated structure of β2-adrenoceptor stabilized by an engineered nanobody</span></div><div class="casAuthors">Ring, Aaron M.; Manglik, Aashish; Kruse, Andrew C.; Enos, Michael D.; Weis, William I.; Garcia, K. Christopher; Kobilka, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">502</span>
        (<span class="NLM_cas:issue">7472</span>),
    <span class="NLM_cas:pages">575-579</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">G-protein-coupled receptors (GPCRs) are integral membrane proteins that play an essential role in human physiol., yet the mol. processes through which they bind to their endogenous agonists and activate effector proteins remain poorly understood.  So far, it has not been possible to capture an active-state GPCR bound to its native neurotransmitter.  Crystal structures of agonist-bound GPCRs have relied on the use of either exceptionally high-affinity agonists or receptor stabilization by mutagenesis.  Many natural agonists such as adrenaline, which activates the β2-adrenoceptor (β2AR), bind with relatively low affinity, and they are often chem. unstable.  Here, using directed evolution, the authors engineered a high-affinity conformationally selective single-domain camelid antibody fragment (nanobody; Nb6B9) that stabilized the active state of human β2AR, and used this to obtain crystal structures of the activated receptor bound to multiple ligands.  The authors present structures of the active-state human β2AR bound to 3 chem. distinct agonists: (1) the ultrahigh-affinity agonist BI167107; (2) the high-affinity catecholamine agonist, hydroxybenzylisoproterenol; and (3) the low-affinity endogenous agonist, adrenaline.  The crystal structures revealed a highly conserved overall ligand recognition and activation mode despite diverse ligand chem. structures and affinities that ranged from 100 nM to ∼80 pM.  Overall, the adrenaline-bound receptor structure was similar to the others, but it had substantial rearrangements in extracellular loop 3 and the extracellular tip of transmembrane helix 6.  These structures also revealed a water-mediated H-bond between 2 conserved Tyr residues, which appeared to stabilize the active state of the β2AR and related GPCRs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAp0VskflwWbVg90H21EOLACvtfcHk0lhUvt3Kov1CSA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFSlsLzP&md5=c5b844a0a86ee0a74843588a4a363592</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1038%2Fnature12572&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12572%26sid%3Dliteratum%253Aachs%26aulast%3DRing%26aufirst%3DA.%2BM.%26aulast%3DManglik%26aufirst%3DA.%26aulast%3DKruse%26aufirst%3DA.%2BC.%26aulast%3DEnos%26aufirst%3DM.%2BD.%26aulast%3DWeis%26aufirst%3DW.%2BI.%26aulast%3DGarcia%26aufirst%3DK.%2BC.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26atitle%3DAdrenaline-activated%2520structure%2520of%2520%25CE%25B22-adrenoceptor%2520stabilized%2520by%2520an%2520engineered%2520nanobody%26jtitle%3DNature%26date%3D2013%26volume%3D502%26spage%3D575%26epage%3D579%26doi%3D10.1038%2Fnature12572" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span>Small-Molecule Drug Discovery Suite 2015–1: Glide; <span class="NLM_publisher-name">Schrödinger</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Small-Molecule+Drug+Discovery+Suite+2015%E2%80%931%3A+Glide%3B+Schr%C3%B6dinger%3A+New+York%2C+2015."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DSmall-Molecule%2520Drug%2520Discovery%2520Suite%25202015%25E2%2580%25931%253A%2520Glide%26pub%3DSchr%25C3%25B6dinger%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group">Wacker, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fenalti, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Katritch, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abagyan, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cherezov, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stevens, R. C.</span><span> </span><span class="NLM_article-title">Conserved binding mode of human β2 adrenergic receptor inverse agonists and antagonist revealed by x-ray crystallography</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">132</span><span class="NLM_x">, </span> <span class="NLM_fpage">11443</span><span class="NLM_x">–</span> <span class="NLM_lpage">11445</span><span class="refDoi"> DOI: 10.1021/ja105108q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja105108q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BC3cXpsV2mtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2010&pages=11443-11445&author=D.+Wackerauthor=G.+Fenaltiauthor=M.+A.+Brownauthor=V.+Katritchauthor=R.+Abagyanauthor=V.+Cherezovauthor=R.+C.+Stevens&title=Conserved+binding+mode+of+human+%CE%B22+adrenergic+receptor+inverse+agonists+and+antagonist+revealed+by+x-ray+crystallography&doi=10.1021%2Fja105108q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Conserved Binding Mode of Human β2 Adrenergic Receptor Inverse Agonists and Antagonist Revealed by X-ray Crystallography</span></div><div class="casAuthors">Wacker, Daniel; Fenalti, Gustavo; Brown, Monica A.; Katritch, Vsevolod; Abagyan, Ruben; Cherezov, Vadim; Stevens, Raymond C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">132</span>
        (<span class="NLM_cas:issue">33</span>),
    <span class="NLM_cas:pages">11443-11445</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">G protein-coupled receptors (GPCRs) represent a large fraction of current pharmaceutical targets, and of the GPCRs, the β2 adrenergic receptor (β2AR) is one of the most extensively studied.  Previously, the x-ray crystal structure of β2AR has been detd. in complex with two partial inverse agonists, but the global impact of addnl. ligands on the structure or local impacts on the binding site are not well-understood.  To assess the extent of such ligand-induced conformational differences, we detd. the crystal structures of a previously described engineered β2AR construct in complex with two inverse agonists: ICI 118,551 (2.8 Å), a recently described compd. (2.8 Å) (Kolb et al, 2009), and the antagonist alprenolol (3.1 Å).  The structures show the same overall fold obsd. for the previous β2AR structures and demonstrate that the ligand binding site can accommodate compds. of different chem. and pharmacol. properties with only minor local structural rearrangements.  All three compds. contain a hydroxy-amine motif that establishes a conserved hydrogen bond network with the receptor and chem. diverse arom. moieties that form distinct interactions with β2AR.  Furthermore, receptor ligand cross-docking expts. revealed that a single β2AR complex can be suitable for docking of a range of antagonists and inverse agonists but also indicate that addnl. ligand-receptor structures may be useful to further improve performance for in-silico docking or lead-optimization in drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobJGADPHgkVrVg90H21EOLACvtfcHk0lhsTFny-BWuKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXpsV2mtLo%253D&md5=0e231d630000e0155c2049c95067fb0e</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1021%2Fja105108q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja105108q%26sid%3Dliteratum%253Aachs%26aulast%3DWacker%26aufirst%3DD.%26aulast%3DFenalti%26aufirst%3DG.%26aulast%3DBrown%26aufirst%3DM.%2BA.%26aulast%3DKatritch%26aufirst%3DV.%26aulast%3DAbagyan%26aufirst%3DR.%26aulast%3DCherezov%26aufirst%3DV.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26atitle%3DConserved%2520binding%2520mode%2520of%2520human%2520%25CE%25B22%2520adrenergic%2520receptor%2520inverse%2520agonists%2520and%2520antagonist%2520revealed%2520by%2520x-ray%2520crystallography%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2010%26volume%3D132%26spage%3D11443%26epage%3D11445%26doi%3D10.1021%2Fja105108q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group">Götz, A. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williamson, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poole, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Le Grand, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walker, R. C.</span><span> </span><span class="NLM_article-title">Routine microsecond molecular dynamics simulations with AMBER on GPUs. 1. Generalized Born</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">1542</span><span class="NLM_x">–</span> <span class="NLM_lpage">1555</span><span class="refDoi"> DOI: 10.1021/ct200909j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct200909j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BC38XksFWns78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=1542-1555&author=A.+W.+G%C3%B6tzauthor=M.+J.+Williamsonauthor=D.+Xuauthor=D.+Pooleauthor=S.+Le+Grandauthor=R.+C.+Walker&title=Routine+microsecond+molecular+dynamics+simulations+with+AMBER+on+GPUs.+1.+Generalized+Born&doi=10.1021%2Fct200909j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Routine Microsecond Molecular Dynamics Simulations with AMBER on GPUs. 1. Generalized Born</span></div><div class="casAuthors">Gotz, Andreas W.; Williamson, Mark J.; Xu, Dong; Poole, Duncan; Le Grand, Scott; Walker, Ross C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1542-1555</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We present an implementation of generalized Born implicit solvent all-atom classical mol. dynamics (MD) within the AMBER program package that runs entirely on CUDA enabled NVIDIA graphics processing units (GPUs).  We discuss the algorithms that are used to exploit the processing power of the GPUs and show the performance that can be achieved in comparison to simulations on conventional CPU clusters.  The implementation supports three different precision models in which the contributions to the forces are calcd. in single precision floating point arithmetic but accumulated in double precision (SPDP), or everything is computed in single precision (SPSP) or double precision (DPDP).  In addn. to performance, we have focused on understanding the implications of the different precision models on the outcome of implicit solvent MD simulations.  We show results for a range of tests including the accuracy of single point force evaluations and energy conservation as well as structural properties pertaining to protein dynamics.  The numerical noise due to rounding errors within the SPSP precision model is sufficiently large to lead to an accumulation of errors which can result in unphys. trajectories for long time scale simulations.  We recommend the use of the mixed-precision SPDP model since the numerical results obtained are comparable with those of the full double precision DPDP model and the ref. double precision CPU implementation but at significantly reduced computational cost.  Our implementation provides performance for GB simulations on a single desktop that is on par with, and in some cases exceeds, that of traditional supercomputers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpshIijeXLcqLVg90H21EOLACvtfcHk0lhsTFny-BWuKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XksFWns78%253D&md5=1e6f570db9cd504bb13706e7c56bc356</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1021%2Fct200909j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct200909j%26sid%3Dliteratum%253Aachs%26aulast%3DG%25C3%25B6tz%26aufirst%3DA.%2BW.%26aulast%3DWilliamson%26aufirst%3DM.%2BJ.%26aulast%3DXu%26aufirst%3DD.%26aulast%3DPoole%26aufirst%3DD.%26aulast%3DLe%2BGrand%26aufirst%3DS.%26aulast%3DWalker%26aufirst%3DR.%2BC.%26atitle%3DRoutine%2520microsecond%2520molecular%2520dynamics%2520simulations%2520with%2520AMBER%2520on%2520GPUs.%25201.%2520Generalized%2520Born%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2012%26volume%3D8%26spage%3D1542%26epage%3D1555%26doi%3D10.1021%2Fct200909j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group">Salomon-Ferrer, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Götz, A. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poole, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Le Grand, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walker, R. C.</span><span> </span><span class="NLM_article-title">Routine microsecond molecular dynamics simulations with AMBER on GPUs. 2. Explicit solvent particle mesh Ewald</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">3878</span><span class="NLM_x">–</span> <span class="NLM_lpage">3888</span><span class="refDoi"> DOI: 10.1021/ct400314y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct400314y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1arsrzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=3878-3888&author=R.+Salomon-Ferrerauthor=A.+W.+G%C3%B6tzauthor=D.+Pooleauthor=S.+Le+Grandauthor=R.+C.+Walker&title=Routine+microsecond+molecular+dynamics+simulations+with+AMBER+on+GPUs.+2.+Explicit+solvent+particle+mesh+Ewald&doi=10.1021%2Fct400314y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Routine Microsecond Molecular Dynamics Simulations with AMBER on GPUs. 2. Explicit Solvent Particle Mesh Ewald</span></div><div class="casAuthors">Salomon-Ferrer, Romelia; Gotz, Andreas W.; Poole, Duncan; Le Grand, Scott; Walker, Ross C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3878-3888</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We present an implementation of explicit solvent all atom classical mol. dynamics (MD) within the AMBER program package that runs entirely on CUDA-enabled GPUs.  First released publicly in Apr. 2010 as part of version 11 of the AMBER MD package and further improved and optimized over the last two years, this implementation supports the three most widely used statistical mech. ensembles (NVE, NVT, and NPT), uses particle mesh Ewald (PME) for the long-range electrostatics, and runs entirely on CUDA-enabled NVIDIA graphics processing units (GPUs), providing results that are statistically indistinguishable from the traditional CPU version of the software and with performance that exceeds that achievable by the CPU version of AMBER software running on all conventional CPU-based clusters and supercomputers.  We briefly discuss three different precision models developed specifically for this work (SPDP, SPFP, and DPDP) and highlight the tech. details of the approach as it extends beyond previously reported work [Goetz et al., J.  Chem.  Theory Comput.2012, DOI: 10.1021/ct200909j; Le Grand et al., Comp.  Phys.  Comm.2013, DOI: 10.1016/j.cpc.2012.09.022].We highlight the substantial improvements in performance that are seen over traditional CPU-only machines and provide validation of our implementation and precision models.  We also provide evidence supporting our decision to deprecate the previously described fully single precision (SPSP) model from the latest release of the AMBER software package.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoY-M2E_jgUmLVg90H21EOLACvtfcHk0lg0l4HLXhxn1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1arsrzP&md5=57e18d35e8b81e9a79788ef5af17d19f</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1021%2Fct400314y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct400314y%26sid%3Dliteratum%253Aachs%26aulast%3DSalomon-Ferrer%26aufirst%3DR.%26aulast%3DG%25C3%25B6tz%26aufirst%3DA.%2BW.%26aulast%3DPoole%26aufirst%3DD.%26aulast%3DLe%2BGrand%26aufirst%3DS.%26aulast%3DWalker%26aufirst%3DR.%2BC.%26atitle%3DRoutine%2520microsecond%2520molecular%2520dynamics%2520simulations%2520with%2520AMBER%2520on%2520GPUs.%25202.%2520Explicit%2520solvent%2520particle%2520mesh%2520Ewald%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2013%26volume%3D9%26spage%3D3878%26epage%3D3888%26doi%3D10.1021%2Fct400314y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group">Le Grand, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Götz, A. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walker, R. C.</span><span> </span><span class="NLM_article-title">SPFP: Speed without compromise—a mixed precision model for GPU accelerated molecular dynamics simulations</span> <span class="citation_source-journal">Comput. Phys. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">184</span><span class="NLM_x">, </span> <span class="NLM_fpage">374</span><span class="NLM_x">–</span> <span class="NLM_lpage">380</span><span class="refDoi"> DOI: 10.1016/j.cpc.2012.09.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1016%2Fj.cpc.2012.09.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsV2jsrzO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=184&publication_year=2013&pages=374-380&author=S.+Le+Grandauthor=A.+W.+G%C3%B6tzauthor=R.+C.+Walker&title=SPFP%3A+Speed+without+compromise%E2%80%94a+mixed+precision+model+for+GPU+accelerated+molecular+dynamics+simulations&doi=10.1016%2Fj.cpc.2012.09.022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">SPFP: Speed without compromise-A mixed precision model for GPU accelerated molecular dynamics simulations</span></div><div class="casAuthors">Le Grand, Scott; Gotz, Andreas W.; Walker, Ross C.</div><div class="citationInfo"><span class="NLM_cas:title">Computer Physics Communications</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">184</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">374-380</span>CODEN:
                <span class="NLM_cas:coden">CPHCBZ</span>;
        ISSN:<span class="NLM_cas:issn">0010-4655</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A new precision model is proposed for the acceleration of all-atom classical mol. dynamics (MD) simulations on graphics processing units (GPUs).  This precision model replaces double precision arithmetic with fixed point integer arithmetic for the accumulation of force components as compared to a previously introduced model that uses mixed single/double precision arithmetic.  This significantly boosts performance on modern GPU hardware without sacrificing numerical accuracy.  We present an implementation for NVIDIA GPUs of both generalized Born implicit solvent simulations as well as explicit solvent simulations using the particle mesh Ewald (PME) algorithm for long-range electrostatics using this precision model.  Tests demonstrate both the performance of this implementation as well as its numerical stability for const. energy and const. temp. biomol. MD as compared to a double precision CPU implementation and double and mixed single/double precision GPU implementations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJB_spY7SrSbVg90H21EOLACvtfcHk0lg0l4HLXhxn1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsV2jsrzO&md5=2d9f102cb17f6e9bcbff3b5057d8c587</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1016%2Fj.cpc.2012.09.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cpc.2012.09.022%26sid%3Dliteratum%253Aachs%26aulast%3DLe%2BGrand%26aufirst%3DS.%26aulast%3DG%25C3%25B6tz%26aufirst%3DA.%2BW.%26aulast%3DWalker%26aufirst%3DR.%2BC.%26atitle%3DSPFP%253A%2520Speed%2520without%2520compromise%25E2%2580%2594a%2520mixed%2520precision%2520model%2520for%2520GPU%2520accelerated%2520molecular%2520dynamics%2520simulations%26jtitle%3DComput.%2520Phys.%2520Commun.%26date%3D2013%26volume%3D184%26spage%3D374%26epage%3D380%26doi%3D10.1016%2Fj.cpc.2012.09.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref69"><div class="reference"><strong class="refLabel"><a href="#ref69" class="rightTabRefNumLink">69</a></strong><div class="NLM_citation" id="rightTab-cit69"><span><span class="NLM_contrib-group">Maier, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martinez, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kasavajhala, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wickstrom, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hauser, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simmerling, C.</span><span> </span><span class="NLM_article-title">ff14SB: improving the accuracy of protein side chain and backbone parameters from ff99SB</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">3696</span><span class="NLM_x">–</span> <span class="NLM_lpage">3713</span><span class="refDoi"> DOI: 10.1021/acs.jctc.5b00255</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jctc.5b00255" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFequ7rN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=3696-3713&author=J.+A.+Maierauthor=C.+Martinezauthor=K.+Kasavajhalaauthor=L.+Wickstromauthor=K.+E.+Hauserauthor=C.+Simmerling&title=ff14SB%3A+improving+the+accuracy+of+protein+side+chain+and+backbone+parameters+from+ff99SB&doi=10.1021%2Facs.jctc.5b00255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">ff14SB: Improving the Accuracy of Protein Side Chain and Backbone Parameters from ff99SB</span></div><div class="casAuthors">Maier, James A.; Martinez, Carmenza; Kasavajhala, Koushik; Wickstrom, Lauren; Hauser, Kevin E.; Simmerling, Carlos</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3696-3713</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. mechanics is powerful for its speed in atomistic simulations, but an accurate force field is required.  The Amber ff99SB force field improved protein secondary structure balance and dynamics from earlier force fields like ff99, but weaknesses in side chain rotamer and backbone secondary structure preferences have been identified.  Here, we performed a complete refit of all amino acid side chain dihedral parameters, which had been carried over from ff94.  The training set of conformations included multidimensional dihedral scans designed to improve transferability of the parameters.  Improvement in all amino acids was obtained as compared to ff99SB.  Parameters were also generated for alternate protonation states of ionizable side chains.  Av. errors in relative energies of pairs of conformations were under 1.0 kcal/mol as compared to QM, reduced 35% from ff99SB.  We also took the opportunity to make empirical adjustments to the protein backbone dihedral parameters as compared to ff99SB.  Multiple small adjustments of φ and ψ parameters were tested against NMR scalar coupling data and secondary structure content for short peptides.  The best results were obtained from a phys. motivated adjustment to the φ rotational profile that compensates for lack of ff99SB QM training data in the β-ppII transition region.  Together, these backbone and side chain modifications (hereafter called ff14SB) not only better reproduced their benchmarks, but also improved secondary structure content in small peptides and reprodn. of NMR χ1 scalar coupling measurements for proteins in soln.  We also discuss the Amber ff12SB parameter set, a preliminary version of ff14SB that includes most of its improvements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgntnN9ES97rVg90H21EOLACvtfcHk0lg0l4HLXhxn1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFequ7rN&md5=7b803577b3b6912cc6750cfbd356596e</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1021%2Facs.jctc.5b00255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jctc.5b00255%26sid%3Dliteratum%253Aachs%26aulast%3DMaier%26aufirst%3DJ.%2BA.%26aulast%3DMartinez%26aufirst%3DC.%26aulast%3DKasavajhala%26aufirst%3DK.%26aulast%3DWickstrom%26aufirst%3DL.%26aulast%3DHauser%26aufirst%3DK.%2BE.%26aulast%3DSimmerling%26aufirst%3DC.%26atitle%3Dff14SB%253A%2520improving%2520the%2520accuracy%2520of%2520protein%2520side%2520chain%2520and%2520backbone%2520parameters%2520from%2520ff99SB%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2015%26volume%3D11%26spage%3D3696%26epage%3D3713%26doi%3D10.1021%2Facs.jctc.5b00255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref70"><div class="reference"><strong class="refLabel"><a href="#ref70" class="rightTabRefNumLink">70</a></strong><div class="NLM_citation" id="rightTab-cit70"><span><span class="NLM_contrib-group">Bayly, C. I.; McKay, D.; Truchon, J.-F.</span> An informal AMBER small molecule force field: <a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="0979687b64494f7b667a7a7d">[email protected]</a>; <a href="http://www.ccl.net/cca/data/parm_at_Frosst/parm_Frosst_note.pdf" class="extLink">http://www.ccl.net/cca/data/parm_at_Frosst/parm_Frosst_note.pdf</a> (accessed Jan 15, 2015).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Bayly%2C+C.+I.%3B+McKay%2C+D.%3B+Truchon%2C+J.-F.+An+informal+AMBER+small+molecule+force+field%3A+parm%40Frosst%3B+http%3A%2F%2Fwww.ccl.net%2Fcca%2Fdata%2Fparm_at_Frosst%2Fparm_Frosst_note.pdf+%28accessed+Jan+15%2C+2015%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26aulast%3DBayly%26aufirst%3DC.%2BI." title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref71"><div class="reference"><strong class="refLabel"><a href="#ref71" class="rightTabRefNumLink">71</a></strong><div class="NLM_citation" id="rightTab-cit71"><span><span class="NLM_contrib-group">Jakalian, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bush, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jack, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bayly, C. I.</span><span> </span><span class="NLM_article-title">Fast, efficient generation of high-quality atomic charges. AM1-BCC model: I. Method</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">132</span><span class="NLM_x">–</span> <span class="NLM_lpage">146</span><span class="refDoi"> DOI: 10.1002/(SICI)1096-987X(20000130)21:2<132::AID-JCC5>3.0.CO;2-P</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1002%2F%28SICI%291096-987X%2820000130%2921%3A2%3C132%3A%3AAID-JCC5%3E3.0.CO%3B2-P" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BD3cXkt1SgsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2000&pages=132-146&author=A.+Jakalianauthor=B.+L.+Bushauthor=D.+B.+Jackauthor=C.+I.+Bayly&title=Fast%2C+efficient+generation+of+high-quality+atomic+charges.+AM1-BCC+model%3A+I.+Method&doi=10.1002%2F%28SICI%291096-987X%2820000130%2921%3A2%3C132%3A%3AAID-JCC5%3E3.0.CO%3B2-P"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Fast, efficient generation of high-quality atomic charges. AM1-BCC model: I. Method</span></div><div class="casAuthors">Jakalian, Araz; Bush, Bruce L.; Jack, David B.; Bayly, Christopher I.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">132-146</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The AM1-BCC method quickly and efficiently generates high-quality at. charges for use in condensed-phase simulations.  The underlying features of the electron distribution including formal charge and delocalization are first captured by AM1 at. charges for the individual mol.  Bond charge corrections (BCCs), which have been parameterized against the HF/6-31G* electrostatic potential (ESP) of a training set of compds. contg. relevant functional groups, are then added using a formalism identical to the consensus BCI (bond charge increment) approach.  As a proof of the concept, we fit BCCs simultaneously to 45 compds. including O-, N-, and S-contg. functionalities, aroms., and heteroaroms., using only 41 BCC parameters.  AM1-BCC yields charge sets of comparable quality to HF/6-31G* ESP-derived charges in a fraction of the time while reducing instabilities in the at. charges compared to direct ESP-fit methods.  We then apply the BCC parameters to a small "test set" consisting of aspirin, D-glucose, and eryodictyol; the AM1-BCC model again provides at. charges of quality comparable with HF/6-31G* RESP charges, as judged by an increase of only 0.01 to 0.02 at. units in the root-mean-square (RMS) error in ESP.  Based on these encouraging results, we intend to parameterize the AM1-BCC model to provide a consistent charge model for any org. or biol. mol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruxZWGx7IgLbVg90H21EOLACvtfcHk0lgFIl3-EHehag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXkt1SgsQ%253D%253D&md5=aa4774739d3491dbc8dd95ed1948d856</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291096-987X%2820000130%2921%3A2%3C132%3A%3AAID-JCC5%3E3.0.CO%3B2-P&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291096-987X%252820000130%252921%253A2%253C132%253A%253AAID-JCC5%253E3.0.CO%253B2-P%26sid%3Dliteratum%253Aachs%26aulast%3DJakalian%26aufirst%3DA.%26aulast%3DBush%26aufirst%3DB.%2BL.%26aulast%3DJack%26aufirst%3DD.%2BB.%26aulast%3DBayly%26aufirst%3DC.%2BI.%26atitle%3DFast%252C%2520efficient%2520generation%2520of%2520high-quality%2520atomic%2520charges.%2520AM1-BCC%2520model%253A%2520I.%2520Method%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2000%26volume%3D21%26spage%3D132%26epage%3D146%26doi%3D10.1002%2F%28SICI%291096-987X%2820000130%2921%3A2%3C132%3A%3AAID-JCC5%3E3.0.CO%3B2-P" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref72"><div class="reference"><strong class="refLabel"><a href="#ref72" class="rightTabRefNumLink">72</a></strong><div class="NLM_citation" id="rightTab-cit72"><span><span class="NLM_contrib-group">Jakalian, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jack, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bayly, C. I.</span><span> </span><span class="NLM_article-title">Fast, efficient generation of high-quality atomic charges. AM1-BCC model: II. Parameterization and validation</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">1623</span><span class="NLM_x">–</span> <span class="NLM_lpage">1641</span><span class="refDoi"> DOI: 10.1002/jcc.10128</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1002%2Fjcc.10128" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=12395429" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BD38XosF2rt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2002&pages=1623-1641&author=A.+Jakalianauthor=D.+B.+Jackauthor=C.+I.+Bayly&title=Fast%2C+efficient+generation+of+high-quality+atomic+charges.+AM1-BCC+model%3A+II.+Parameterization+and+validation&doi=10.1002%2Fjcc.10128"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Fast, efficient generation of high-quality atomic charges. AM1-BCC model: II. parameterization and validation</span></div><div class="casAuthors">Jakalian, Araz; Jack, David B.; Bayly, Christopher I.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1623-1641</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">We present the first global parameterization and validation of a novel charge model, called AM1-BCC, which quickly and efficiently generates high-quality at. charges for computer simulations of org. mols. in polar media.  The goal of the charge model is to produce at. charges that emulate the HF/6-31G* electrostatic potential (ESP) of a mol.  Underlying electronic structure features, including formal charge and electron delocalization, are first captured by AM1 population charges; simple additive bond charge corrections (BCCs) are then applied to these AM1 at. charges to produce the AM1-BCC charges.  The parameterization of BCCs was carried out by fitting to the HF/6-31G* ESP of a training set of >2700 mols.  Most org. functional groups and their combinations were sampled, as well as an extensive variety of cyclic and fused bicyclic heteroaryl systems.  The resulting BCC parameters allow the AM1-BCC charging scheme to handle virtually all types of org. compds. listed in The Merck Index and the NCI Database.  Validation of the model was done through comparisons of hydrogen-bonded dimer energies and relative free energies of solvation using AM1-BCC charges in conjunction with the 1994 Cornell et al. forcefield for AMBER.  Homo-dimer and hetero-dimer hydrogen-bond energies of a diverse set of org. mols. were reproduced to within 0.95 kcal/mol RMS deviation from the ab initio values, and for DNA dimers the energies were within 0.9 kcal/mol RMS deviation from ab initio values.  The calcd. relative free energies of solvation for a diverse set of monofunctional isosteres were reproduced to within 0.69 kcal/mol of expt.  In all these validation tests, AMBER with the AM1-BCC charge model maintained a correlation coeff. above 0.96.  Thus, the parameters presented here for use with the AM1-BCC method present a fast, accurate, and robust alternative to HF/6-31G* ESP-fit charges for general use with the AMBER force field in computer simulations involving org. small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpn9DkJ_gMzA7Vg90H21EOLACvtfcHk0lgFIl3-EHehag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XosF2rt74%253D&md5=3f2b738617bb17b2898f7ac4d751d7ec</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1002%2Fjcc.10128&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.10128%26sid%3Dliteratum%253Aachs%26aulast%3DJakalian%26aufirst%3DA.%26aulast%3DJack%26aufirst%3DD.%2BB.%26aulast%3DBayly%26aufirst%3DC.%2BI.%26atitle%3DFast%252C%2520efficient%2520generation%2520of%2520high-quality%2520atomic%2520charges.%2520AM1-BCC%2520model%253A%2520II.%2520Parameterization%2520and%2520validation%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2002%26volume%3D23%26spage%3D1623%26epage%3D1641%26doi%3D10.1002%2Fjcc.10128" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref73"><div class="reference"><strong class="refLabel"><a href="#ref73" class="rightTabRefNumLink">73</a></strong><div class="NLM_citation" id="rightTab-cit73"><span><span class="NLM_contrib-group">Beauchamp, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Behr, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rustenburg, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bayly, C. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kroenlein, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chodera, J. D.</span><span> </span><span class="NLM_article-title">Toward automated benchmarking of atomistic force fields: neat liquid densities and static dielectric constants from the ThermoML data archive</span> <span class="citation_source-journal">J. Phys. Chem. B</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">119</span><span class="NLM_x">, </span> <span class="NLM_fpage">12912</span><span class="NLM_x">–</span> <span class="NLM_lpage">12920</span><span class="refDoi"> DOI: 10.1021/acs.jpcb.5b06703</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jpcb.5b06703" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVGjsbfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2015&pages=12912-12920&author=K.+A.+Beauchampauthor=J.+M.+Behrauthor=A.+S.+Rustenburgauthor=C.+I.+Baylyauthor=K.+Kroenleinauthor=J.+D.+Chodera&title=Toward+automated+benchmarking+of+atomistic+force+fields%3A+neat+liquid+densities+and+static+dielectric+constants+from+the+ThermoML+data+archive&doi=10.1021%2Facs.jpcb.5b06703"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">Toward Automated Benchmarking of Atomistic Force Fields: Neat Liquid Densities and Static Dielectric Constants from the ThermoML Data Archive</span></div><div class="casAuthors">Beauchamp, Kyle A.; Behr, Julie M.; Rustenburg, Arien S.; Bayly, Christopher I.; Kroenlein, Kenneth; Chodera, John D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">40</span>),
    <span class="NLM_cas:pages">12912-12920</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1520-5207</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Atomistic mol. simulations are a powerful way to make quant. predictions, but the accuracy of these predictions depends entirely on the quality of the force field employed.  Although exptl. measurements of fundamental phys. properties offer a straightforward approach for evaluating force field quality, the bulk of this information has been tied up in formats that are not machine-readable.  Compiling benchmark data sets of phys. properties from non-machine-readable sources requires substantial human effort and is prone to the accumulation of human errors, hindering the development of reproducible benchmarks of force-field accuracy.  Here, we examine the feasibility of benchmarking atomistic force fields against the NIST ThermoML data archive of physicochem. measurements, which aggregates thousands of exptl. measurements in a portable, machine-readable, self-annotating IUPAC-std. format.  As a proof of concept, we present a detailed benchmark of the generalized Amber small-mol. force field (GAFF) using the AM1-BCC charge model against exptl. measurements (specifically, bulk liq. densities and static dielec. consts. at ambient pressure) automatically extd. from the archive and discuss the extent of data available for use in larger scale (or continuously performed) benchmarks.  The results of even this limited initial benchmark highlight a general problem with fixed-charge force fields in the representation low-dielec. environments, such as those seen in binding cavities or biol. membranes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3vmYoWhqCXrVg90H21EOLACvtfcHk0lgFIl3-EHehag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVGjsbfJ&md5=8d4a5cdc9e975435ef3eec90fb222279</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1021%2Facs.jpcb.5b06703&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jpcb.5b06703%26sid%3Dliteratum%253Aachs%26aulast%3DBeauchamp%26aufirst%3DK.%2BA.%26aulast%3DBehr%26aufirst%3DJ.%2BM.%26aulast%3DRustenburg%26aufirst%3DA.%2BS.%26aulast%3DBayly%26aufirst%3DC.%2BI.%26aulast%3DKroenlein%26aufirst%3DK.%26aulast%3DChodera%26aufirst%3DJ.%2BD.%26atitle%3DToward%2520automated%2520benchmarking%2520of%2520atomistic%2520force%2520fields%253A%2520neat%2520liquid%2520densities%2520and%2520static%2520dielectric%2520constants%2520from%2520the%2520ThermoML%2520data%2520archive%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2015%26volume%3D119%26spage%3D12912%26epage%3D12920%26doi%3D10.1021%2Facs.jpcb.5b06703" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref74"><div class="reference"><strong class="refLabel"><a href="#ref74" class="rightTabRefNumLink">74</a></strong><div class="NLM_citation" id="rightTab-cit74"><span><span class="NLM_contrib-group">Dickson, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosso, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Betz, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walker, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gould, I. R.</span><span> </span><span class="NLM_article-title">GAFFlipid: a General Amber Force Field for the accurate molecular dynamics simulation of phospholipid</span> <span class="citation_source-journal">Soft Matter</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">9617</span><span class="NLM_x">–</span> <span class="NLM_lpage">9627</span><span class="refDoi"> DOI: 10.1039/c2sm26007g</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1039%2Fc2sm26007g" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1ymsbzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=9617-9627&author=C.+J.+Dicksonauthor=L.+Rossoauthor=R.+M.+Betzauthor=R.+C.+Walkerauthor=I.+R.+Gould&title=GAFFlipid%3A+a+General+Amber+Force+Field+for+the+accurate+molecular+dynamics+simulation+of+phospholipid&doi=10.1039%2Fc2sm26007g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">GAFFlipid: a General Amber Force Field for the accurate molecular dynamics simulation of phospholipid</span></div><div class="casAuthors">Dickson, Callum J.; Rosso, Lula; Betz, Robin M.; Walker, Ross C.; Gould, Ian R.</div><div class="citationInfo"><span class="NLM_cas:title">Soft Matter</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">37</span>),
    <span class="NLM_cas:pages">9617-9627</span>CODEN:
                <span class="NLM_cas:coden">SMOABF</span>;
        ISSN:<span class="NLM_cas:issn">1744-683X</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Previous attempts to simulate phospholipid bilayers using the General Amber Force Field (GAFF) yielded many bilayer characteristics in agreement with expt., however when using a tensionless NPT ensemble the bilayer is seen to compress to an undesirable extent resulting in low areas per lipid and high order parameters in comparison to expt.  In this work, the GAFF Lennard-Jones parameters for the simulation of acyl chains are cor. to allow the accurate and stable simulation of pure lipid bilayers.  Lipid bilayers comprised of six phospholipid types were simulated for timescales approaching a quarter of a microsecond under tensionless const. pressure conditions using Graphics Processing Units.  Structural properties including area per lipid, vol. per lipid, bilayer thickness, order parameter and headgroup hydration show favorable agreement with available exptl. values.  Expanding the system size from 72 to 288 lipids and a more exptl. realistic 2 × 288 lipid bilayer stack induces little change in the obsd. properties.  This preliminary work is intended for combination with the new AMBER Lipid11 modular force field as part of on-going attempts to create a modular phospholipid AMBER force field allowing tensionless NPT simulations of complex lipid bilayers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplZfTlqhsP_bVg90H21EOLACvtfcHk0lhNCBeU7Vx33w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1ymsbzL&md5=494ab83d6e79d084d2876c2169b9add1</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1039%2Fc2sm26007g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc2sm26007g%26sid%3Dliteratum%253Aachs%26aulast%3DDickson%26aufirst%3DC.%2BJ.%26aulast%3DRosso%26aufirst%3DL.%26aulast%3DBetz%26aufirst%3DR.%2BM.%26aulast%3DWalker%26aufirst%3DR.%2BC.%26aulast%3DGould%26aufirst%3DI.%2BR.%26atitle%3DGAFFlipid%253A%2520a%2520General%2520Amber%2520Force%2520Field%2520for%2520the%2520accurate%2520molecular%2520dynamics%2520simulation%2520of%2520phospholipid%26jtitle%3DSoft%2520Matter%26date%3D2012%26volume%3D8%26spage%3D9617%26epage%3D9627%26doi%3D10.1039%2Fc2sm26007g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref75"><div class="reference"><strong class="refLabel"><a href="#ref75" class="rightTabRefNumLink">75</a></strong><div class="NLM_citation" id="rightTab-cit75"><span><span class="NLM_contrib-group">Skjevik, Å. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Madej, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walker, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teigen, K.</span><span> </span><span class="NLM_article-title">LIPID11: a modular framework for lipid simulations using amber</span> <span class="citation_source-journal">J. Phys. Chem. B</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">116</span><span class="NLM_x">, </span> <span class="NLM_fpage">11124</span><span class="NLM_x">–</span> <span class="NLM_lpage">11136</span><span class="refDoi"> DOI: 10.1021/jp3059992</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp3059992" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1Gmsr7K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2012&pages=11124-11136&author=%C3%85.+A.+Skjevikauthor=B.+D.+Madejauthor=R.+C.+Walkerauthor=K.+Teigen&title=LIPID11%3A+a+modular+framework+for+lipid+simulations+using+amber&doi=10.1021%2Fjp3059992"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">LIPID11: A Modular Framework for Lipid Simulations Using Amber</span></div><div class="casAuthors">Skjevik, Aage A.; Madej, Benjamin D.; Walker, Ross C.; Teigen, Knut</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">11124-11136</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1520-5207</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Accurate simulation of complex lipid bilayers has long been a goal in condensed phase mol. dynamics (MD).  Structure and function of membrane-bound proteins are highly dependent on the lipid bilayer environment and are challenging to study through exptl. methods.  Within Amber, there has been limited focus on lipid simulations, although some success has been seen with the use of the General Amber Force Field (GAFF).  However, to date there are no dedicated Amber lipid force fields.  In this paper we describe a new charge derivation strategy for lipids consistent with the Amber RESP approach and a new atom and residue naming and type convention.  In the first instance, we have combined this approach with GAFF parameters.  The result is LIPID11, a flexible, modular framework for the simulation of lipids that is fully compatible with the existing Amber force fields.  The charge derivation procedure, capping strategy, and nomenclature for LIPID11, along with preliminary simulation results and a discussion of the planned long-term parameter development are presented here.  Our findings suggest that LIPID11 is a modular framework feasible for phospholipids and a flexible starting point for the development of a comprehensive, Amber-compatible lipid force field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr22eGSs_c6cLVg90H21EOLACvtfcHk0lhNCBeU7Vx33w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1Gmsr7K&md5=c5ec086a4f0e489bd6ec3141314d6592</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1021%2Fjp3059992&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp3059992%26sid%3Dliteratum%253Aachs%26aulast%3DSkjevik%26aufirst%3D%25C3%2585.%2BA.%26aulast%3DMadej%26aufirst%3DB.%2BD.%26aulast%3DWalker%26aufirst%3DR.%2BC.%26aulast%3DTeigen%26aufirst%3DK.%26atitle%3DLIPID11%253A%2520a%2520modular%2520framework%2520for%2520lipid%2520simulations%2520using%2520amber%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2012%26volume%3D116%26spage%3D11124%26epage%3D11136%26doi%3D10.1021%2Fjp3059992" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref76"><div class="reference"><strong class="refLabel"><a href="#ref76" class="rightTabRefNumLink">76</a></strong><div class="NLM_citation" id="rightTab-cit76"><span><span class="NLM_contrib-group">Dickson, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Madej, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skjevik, Å. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Betz, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teigen, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gould, I. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walker, R. C.</span><span> </span><span class="NLM_article-title">Lipid14: the amber lipid force field</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">865</span><span class="NLM_x">–</span> <span class="NLM_lpage">879</span><span class="refDoi"> DOI: 10.1021/ct4010307</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct4010307" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1Wgt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=865-879&author=C.+J.+Dicksonauthor=B.+D.+Madejauthor=%C3%85.+A.+Skjevikauthor=R.+M.+Betzauthor=K.+Teigenauthor=I.+R.+Gouldauthor=R.+C.+Walker&title=Lipid14%3A+the+amber+lipid+force+field&doi=10.1021%2Fct4010307"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">Lipid14: The Amber Lipid Force Field</span></div><div class="casAuthors">Dickson, Callum J.; Madej, Benjamin D.; Skjevik, Age A.; Betz, Robin M.; Teigen, Knut; Gould, Ian R.; Walker, Ross C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">865-879</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The AMBER lipid force field has been updated to create Lipid14, allowing tensionless simulation of a no. of lipid types with the AMBER MD package.  The modular nature of this force field allows numerous combinations of head and tail groups to create different lipid types, enabling the easy insertion of new lipid species.  The Lennard-Jones and torsion parameters of both the head and tail groups have been revised and updated partial charges calcd.  The force field has been validated by simulating bilayers of six different lipid types for a total of 0.5 μs each without applying a surface tension; with favorable comparison to expt. for properties such as area per lipid, vol. per lipid, bilayer thickness, NMR order parameters, scattering data, and lipid lateral diffusion.  As the derivation of this force field is consistent with the AMBER development philosophy, Lipid14 is compatible with the AMBER protein, nucleic acid, carbohydrate, and small mol. force fields.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo60Ks38bQOe7Vg90H21EOLACvtfcHk0lhNCBeU7Vx33w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1Wgt70%253D&md5=6efe3346bab426a93016d10418b6957a</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1021%2Fct4010307&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct4010307%26sid%3Dliteratum%253Aachs%26aulast%3DDickson%26aufirst%3DC.%2BJ.%26aulast%3DMadej%26aufirst%3DB.%2BD.%26aulast%3DSkjevik%26aufirst%3D%25C3%2585.%2BA.%26aulast%3DBetz%26aufirst%3DR.%2BM.%26aulast%3DTeigen%26aufirst%3DK.%26aulast%3DGould%26aufirst%3DI.%2BR.%26aulast%3DWalker%26aufirst%3DR.%2BC.%26atitle%3DLipid14%253A%2520the%2520amber%2520lipid%2520force%2520field%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2014%26volume%3D10%26spage%3D865%26epage%3D879%26doi%3D10.1021%2Fct4010307" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref77"><div class="reference"><strong class="refLabel"><a href="#ref77" class="rightTabRefNumLink">77</a></strong><div class="NLM_citation" id="rightTab-cit77"><span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chandrasekhar, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Madura, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Impey, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klein, M. L.</span><span> </span><span class="NLM_article-title">Comparison of simple potential functions for simulating liquid water</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1983</span><span class="NLM_x">, </span> <span class="NLM_volume">79</span><span class="NLM_x">, </span> <span class="NLM_fpage">926</span><span class="NLM_x">–</span> <span class="NLM_lpage">935</span><span class="refDoi"> DOI: 10.1063/1.445869</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1063%2F1.445869" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADyaL3sXksF2htL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=1983&pages=926-935&author=W.+L.+Jorgensenauthor=J.+Chandrasekharauthor=J.+D.+Maduraauthor=R.+W.+Impeyauthor=M.+L.+Klein&title=Comparison+of+simple+potential+functions+for+simulating+liquid+water&doi=10.1063%2F1.445869"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of simple potential functions for simulating liquid water</span></div><div class="casAuthors">Jorgensen, William L.; Chandrasekhar, Jayaraman; Madura, Jeffry D.; Impey, Roger W.; Klein, Michael L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1983</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">926-35</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">Classical Monte Carlo simulations were carried out for liq. H2O in the NPT ensemble at 25° and 1 atm using 6 of the simpler intermol. potential functions for the dimer.  Comparisons were made with exptl. thermodn. and structural data including the neutron diffraction results of Thiessen and Narten (1982).  The computed densities and potential energies agree with expt. except for the original Bernal-Fowler model, which yields an 18% overest. of the d. and poor structural results.  The discrepancy may be due to the correction terms needed in processing the neutron data or to an effect uniformly neglected in the computations.  Comparisons were made for the self-diffusion coeffs. obtained from mol. dynamics simulations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpK4QUFMvwYFrVg90H21EOLACvtfcHk0lg2oHUmQdyaUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3sXksF2htL4%253D&md5=a1161334e381746be8c9b15a5e56f704</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1063%2F1.445869&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.445869%26sid%3Dliteratum%253Aachs%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DChandrasekhar%26aufirst%3DJ.%26aulast%3DMadura%26aufirst%3DJ.%2BD.%26aulast%3DImpey%26aufirst%3DR.%2BW.%26aulast%3DKlein%26aufirst%3DM.%2BL.%26atitle%3DComparison%2520of%2520simple%2520potential%2520functions%2520for%2520simulating%2520liquid%2520water%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1983%26volume%3D79%26spage%3D926%26epage%3D935%26doi%3D10.1063%2F1.445869" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref78"><div class="reference"><strong class="refLabel"><a href="#ref78" class="rightTabRefNumLink">78</a></strong><div class="NLM_citation" id="rightTab-cit78"><span><span class="NLM_contrib-group">Press, W. H.; Teukolsky, S. A.; Vetterling, W. T.; Flannery, B. P.</span><span> </span><span class="NLM_article-title">Conjugate Gradient Methods in Multidimensions</span>. In  <span class="citation_source-book">Numerical Recipes: The Art of Scientific Computing</span>, <span class="NLM_edition">3rd</span> ed.; <span class="NLM_publisher-name">Cambridge University Press</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">2007</span><span class="NLM_x">; </span>Chapter 10.8, pp  <span class="NLM_fpage">515</span><span class="NLM_x">–</span> <span class="NLM_lpage">518</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2007&pages=515-518&author=W.+H.+Press&author=S.+A.+Teukolsky&author=W.+T.+Vetterling&author=B.+P.+Flannery&title=Numerical+Recipes%3A+The+Art+of+Scientific+Computing"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DPress%26aufirst%3DW.%2BH.%26atitle%3DConjugate%2520Gradient%2520Methods%2520in%2520Multidimensions%26btitle%3DNumerical%2520Recipes%253A%2520The%2520Art%2520of%2520Scientific%2520Computing%26pub%3DCambridge%2520University%2520Press%26date%3D2007%26spage%3D515%26epage%3D518" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref79"><div class="reference"><strong class="refLabel"><a href="#ref79" class="rightTabRefNumLink">79</a></strong><div class="NLM_citation" id="rightTab-cit79"><span><span class="NLM_contrib-group">Pastor, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brooks, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Szabo, A.</span><span> </span><span class="NLM_article-title">An analysis of the accuracy of Langevin and molecular dynamics algorithms</span> <span class="citation_source-journal">Mol. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1988</span><span class="NLM_x">, </span> <span class="NLM_volume">65</span><span class="NLM_x">, </span> <span class="NLM_fpage">1409</span><span class="NLM_x">–</span> <span class="NLM_lpage">1419</span><span class="refDoi"> DOI: 10.1080/00268978800101881</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1080%2F00268978800101881" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=1988&pages=1409-1419&author=R.+Pastorauthor=B.+Brooksauthor=A.+Szabo&title=An+analysis+of+the+accuracy+of+Langevin+and+molecular+dynamics+algorithms&doi=10.1080%2F00268978800101881"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1080%2F00268978800101881&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F00268978800101881%26sid%3Dliteratum%253Aachs%26aulast%3DPastor%26aufirst%3DR.%26aulast%3DBrooks%26aufirst%3DB.%26aulast%3DSzabo%26aufirst%3DA.%26atitle%3DAn%2520analysis%2520of%2520the%2520accuracy%2520of%2520Langevin%2520and%2520molecular%2520dynamics%2520algorithms%26jtitle%3DMol.%2520Phys.%26date%3D1988%26volume%3D65%26spage%3D1409%26epage%3D1419%26doi%3D10.1080%2F00268978800101881" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref80"><div class="reference"><strong class="refLabel"><a href="#ref80" class="rightTabRefNumLink">80</a></strong><div class="NLM_citation" id="rightTab-cit80"><span><span class="NLM_contrib-group">Berendsen, H. J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Postma, J. P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Gunsteren, W. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DiNola, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haak, J. R.</span><span> </span><span class="NLM_article-title">Molecular dynamics with coupling to an external bath</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1984</span><span class="NLM_x">, </span> <span class="NLM_volume">81</span><span class="NLM_x">, </span> <span class="NLM_fpage">3684</span><span class="NLM_x">–</span> <span class="NLM_lpage">3690</span><span class="refDoi"> DOI: 10.1063/1.448118</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1063%2F1.448118" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADyaL2cXmtlGksbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=1984&pages=3684-3690&author=H.+J.+C.+Berendsenauthor=J.+P.+M.+Postmaauthor=W.+F.+van+Gunsterenauthor=A.+DiNolaauthor=J.+R.+Haak&title=Molecular+dynamics+with+coupling+to+an+external+bath&doi=10.1063%2F1.448118"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit80R"><div class="casContent"><span class="casTitleNuber">80</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular dynamics with coupling to an external bath</span></div><div class="casAuthors">Berendsen, H. J. C.; Postma, J. P. M.; Van Gunsteren, W. F.; DiNola, A.; Haak, J. R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1984</span>),
    <span class="NLM_cas:volume">81</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3684-90</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">In mol. dynamics (MD) simulations, the need often arises to maintain such parameters as temp. or pressure rather than energy and vol., or to impose gradients for studying transport properties in nonequil. MD.  A method is described to realize coupling to an external bath with const. temp. or pressure with adjustable time consts. for the coupling.  The method is easily extendable to other variables and to gradients, and can be applied also to polyat. mols. involving internal constraints.  The influence of coupling time consts. on dynamical variables is evaluated.  A leap-frog algorithm is presented for the general case involving constraints with coupling to both a const. temp. and a const. pressure bath.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoecrpUZWkh37Vg90H21EOLACvtfcHk0lg2oHUmQdyaUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2cXmtlGksbY%253D&md5=5510dc00297d63b91ee3a7a4ae5aacb1</span></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1063%2F1.448118&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.448118%26sid%3Dliteratum%253Aachs%26aulast%3DBerendsen%26aufirst%3DH.%2BJ.%2BC.%26aulast%3DPostma%26aufirst%3DJ.%2BP.%2BM.%26aulast%3Dvan%2BGunsteren%26aufirst%3DW.%2BF.%26aulast%3DDiNola%26aufirst%3DA.%26aulast%3DHaak%26aufirst%3DJ.%2BR.%26atitle%3DMolecular%2520dynamics%2520with%2520coupling%2520to%2520an%2520external%2520bath%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1984%26volume%3D81%26spage%3D3684%26epage%3D3690%26doi%3D10.1063%2F1.448118" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref81"><div class="reference"><strong class="refLabel"><a href="#ref81" class="rightTabRefNumLink">81</a></strong><div class="NLM_citation" id="rightTab-cit81"><span><span class="NLM_contrib-group">Ryckaert, J.-P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciccotti, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berendsen, H. J. C.</span><span> </span><span class="NLM_article-title">Numerical integration of the cartesian equations of motion of a system with constraints: molecular dynamics of n-alkanes</span> <span class="citation_source-journal">J. Comput. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1977</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">327</span><span class="NLM_x">–</span> <span class="NLM_lpage">341</span><span class="refDoi"> DOI: 10.1016/0021-9991(77)90098-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1016%2F0021-9991%2877%2990098-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADyaE2sXktVGhsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=1977&pages=327-341&author=J.-P.+Ryckaertauthor=G.+Ciccottiauthor=H.+J.+C.+Berendsen&title=Numerical+integration+of+the+cartesian+equations+of+motion+of+a+system+with+constraints%3A+molecular+dynamics+of+n-alkanes&doi=10.1016%2F0021-9991%2877%2990098-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit81R"><div class="casContent"><span class="casTitleNuber">81</span><div class="casTitle"><span class="NLM_cas:atitle">Numerical integration of the Cartesian equations of motion of a system with constraints:  molecular dynamics of n-alkanes</span></div><div class="casAuthors">Ryckaert, Jean Paul; Ciccotti, Giovanni; Berendsen, Herman J. C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Physics</span>
        (<span class="NLM_cas:date">1977</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">327-41</span>CODEN:
                <span class="NLM_cas:coden">JCTPAH</span>;
        ISSN:<span class="NLM_cas:issn">0021-9991</span>.
    </div><div class="casAbstract">A numerical algorithm integrating the 3N Cartesian equation of motion of a system of N points subject to holonomic constraints is applied to mol. dynamics simulation of a liq. of 64 butane mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYPlhG1U3Uw7Vg90H21EOLACvtfcHk0lg2oHUmQdyaUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE2sXktVGhsL4%253D&md5=b4aecddfde149117813a5ea4f5353ce2</span></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1016%2F0021-9991%2877%2990098-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0021-9991%252877%252990098-5%26sid%3Dliteratum%253Aachs%26aulast%3DRyckaert%26aufirst%3DJ.-P.%26aulast%3DCiccotti%26aufirst%3DG.%26aulast%3DBerendsen%26aufirst%3DH.%2BJ.%2BC.%26atitle%3DNumerical%2520integration%2520of%2520the%2520cartesian%2520equations%2520of%2520motion%2520of%2520a%2520system%2520with%2520constraints%253A%2520molecular%2520dynamics%2520of%2520n-alkanes%26jtitle%3DJ.%2520Comput.%2520Phys.%26date%3D1977%26volume%3D23%26spage%3D327%26epage%3D341%26doi%3D10.1016%2F0021-9991%2877%2990098-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref82"><div class="reference"><strong class="refLabel"><a href="#ref82" class="rightTabRefNumLink">82</a></strong><div class="NLM_citation" id="rightTab-cit82"><span><span class="NLM_contrib-group">Darden, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">York, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pedersen, L.</span><span> </span><span class="NLM_article-title">Particle mesh Ewald: An N-log(N) method for Ewald sums in large systems</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">98</span><span class="NLM_x">, </span> <span class="NLM_fpage">10089</span><span class="NLM_x">–</span> <span class="NLM_lpage">10092</span><span class="refDoi"> DOI: 10.1063/1.464397</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1063%2F1.464397" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADyaK3sXks1Ohsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=1993&pages=10089-10092&author=T.+Dardenauthor=D.+Yorkauthor=L.+Pedersen&title=Particle+mesh+Ewald%3A+An+N-log%28N%29+method+for+Ewald+sums+in+large+systems&doi=10.1063%2F1.464397"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit82R"><div class="casContent"><span class="casTitleNuber">82</span><div class="casTitle"><span class="NLM_cas:atitle">Particle mesh Ewald: an N·log(N) method for Ewald sums in large systems</span></div><div class="casAuthors">Darden, Tom; York, Darrin; Pedersen, Lee</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">10089-92</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">An N·log(N) method for evaluating electrostatic energies and forces of large periodic systems is presented.  The method is based on interpolation of the reciprocal space Ewald sums and evaluation of the resulting convolution using fast Fourier transforms.  Timings and accuracies are presented for three large cryst. ionic systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDIJavf7AZPbVg90H21EOLACvtfcHk0lijJ6rrxCpSxw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXks1Ohsr0%253D&md5=3c9f230bd01b7b714fd096d4d2e755f6</span></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.1063%2F1.464397&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.464397%26sid%3Dliteratum%253Aachs%26aulast%3DDarden%26aufirst%3DT.%26aulast%3DYork%26aufirst%3DD.%26aulast%3DPedersen%26aufirst%3DL.%26atitle%3DParticle%2520mesh%2520Ewald%253A%2520An%2520N-log%2528N%2529%2520method%2520for%2520Ewald%2520sums%2520in%2520large%2520systems%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1993%26volume%3D98%26spage%3D10089%26epage%3D10092%26doi%3D10.1063%2F1.464397" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref83"><div class="reference"><strong class="refLabel"><a href="#ref83" class="rightTabRefNumLink">83</a></strong><div class="NLM_citation" id="rightTab-cit83"><span><span class="NLM_contrib-group">Roe, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheatham, T. E.</span><span> </span><span class="NLM_article-title">PTRAJ and CPPTRAJ: software for processing and analysis of molecular dynamics trajectory data</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">3084</span><span class="NLM_x">–</span> <span class="NLM_lpage">3095</span><span class="refDoi"> DOI: 10.1021/ct400341p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct400341p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BC3sXptFehtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=3084-3095&author=D.+R.+Roeauthor=T.+E.+Cheatham&title=PTRAJ+and+CPPTRAJ%3A+software+for+processing+and+analysis+of+molecular+dynamics+trajectory+data&doi=10.1021%2Fct400341p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit83R"><div class="casContent"><span class="casTitleNuber">83</span><div class="casTitle"><span class="NLM_cas:atitle">PTRAJ and CPPTRAJ: Software for Processing and Analysis of Molecular Dynamics Trajectory Data</span></div><div class="casAuthors">Roe, Daniel R.; Cheatham, Thomas E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3084-3095</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We describe PTRAJ and its successor CPPTRAJ, two complementary, portable, and freely available computer programs for the anal. and processing of time series of three-dimensional at. positions (i.e., coordinate trajectories) and the data therein derived.  Common tools include the ability to manipulate the data to convert among trajectory formats, process groups of trajectories generated with ensemble methods (e.g., replica exchange mol. dynamics), image with periodic boundary conditions, create av. structures, strip subsets of the system, and perform calcns. such as RMS fitting, measuring distances, B-factors, radii of gyration, radial distribution functions, and time correlations, among other actions and analyses.  Both the PTRAJ and CPPTRAJ programs and source code are freely available under the GNU General Public License version 3 and are currently distributed within the AmberTools 12 suite of support programs that make up part of the Amber package of computer programs (see http://ambermd.org).  This overview describes the general design, features, and history of these two programs, as well as algorithmic improvements and new features available in CPPTRAJ.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq97UsRfUy65bVg90H21EOLACvtfcHk0lijJ6rrxCpSxw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXptFehtr8%253D&md5=6f1bee934f13f180bd7e1feb6b78036d</span></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.1021%2Fct400341p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct400341p%26sid%3Dliteratum%253Aachs%26aulast%3DRoe%26aufirst%3DD.%2BR.%26aulast%3DCheatham%26aufirst%3DT.%2BE.%26atitle%3DPTRAJ%2520and%2520CPPTRAJ%253A%2520software%2520for%2520processing%2520and%2520analysis%2520of%2520molecular%2520dynamics%2520trajectory%2520data%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2013%26volume%3D9%26spage%3D3084%26epage%3D3095%26doi%3D10.1021%2Fct400341p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref84"><div class="reference"><strong class="refLabel"><a href="#ref84" class="rightTabRefNumLink">84</a></strong><div class="NLM_citation" id="rightTab-cit84"><span>The PyMOL Molecular Graphics System, <span class="NLM_edition">version 1.8</span>; <span class="NLM_publisher-name">Schrödinger</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=The+PyMOL+Molecular+Graphics+System%2C+version+1.8%3B+Schr%C3%B6dinger%3A+New+York%2C+2015."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit84&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520PyMOL%2520Molecular%2520Graphics%2520System%26pub%3DSchr%25C3%25B6dinger%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref85"><div class="reference"><strong class="refLabel"><a href="#ref85" class="rightTabRefNumLink">85</a></strong><div class="NLM_citation" id="rightTab-cit85"><span><span class="NLM_contrib-group">Abel, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Farid, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berne, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friesner, R. A.</span><span> </span><span class="NLM_article-title">The role of the active site solvent in the thermodynamics of factor Xa-ligand binding</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">130</span><span class="NLM_x">, </span> <span class="NLM_fpage">2817</span><span class="NLM_x">–</span> <span class="NLM_lpage">2831</span><span class="refDoi"> DOI: 10.1021/ja0771033</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja0771033" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=130&publication_year=2008&pages=2817-2831&author=R.+Abelauthor=T.+Youngauthor=R.+Faridauthor=B.+J.+Berneauthor=R.+A.+Friesner&title=The+role+of+the+active+site+solvent+in+the+thermodynamics+of+factor+Xa-ligand+binding&doi=10.1021%2Fja0771033"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit85&amp;dbid=16384&amp;doi=10.1021%2Fja0771033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja0771033%26sid%3Dliteratum%253Aachs%26aulast%3DAbel%26aufirst%3DR.%26aulast%3DYoung%26aufirst%3DT.%26aulast%3DFarid%26aufirst%3DR.%26aulast%3DBerne%26aufirst%3DB.%2BJ.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26atitle%3DThe%2520role%2520of%2520the%2520active%2520site%2520solvent%2520in%2520the%2520thermodynamics%2520of%2520factor%2520Xa-ligand%2520binding%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2008%26volume%3D130%26spage%3D2817%26epage%3D2831%26doi%3D10.1021%2Fja0771033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref86"><div class="reference"><strong class="refLabel"><a href="#ref86" class="rightTabRefNumLink">86</a></strong><div class="NLM_citation" id="rightTab-cit86"><span><span class="NLM_contrib-group">Nguyen, C. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kurtzman Young, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilson, M. K.</span><span> </span><span class="NLM_article-title">Grid inhomogeneous solvation theory: Hydration structure and thermodynamics of the miniature receptor cucurbit[7]uril</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">137</span><span class="NLM_x">, </span> <span class="NLM_fpage">044101</span><span class="refDoi"> DOI: 10.1063/1.4733951</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1063%2F1.4733951" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=22852591" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVOgurzJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=137&publication_year=2012&pages=044101&author=C.+N.+Nguyenauthor=T.+Kurtzman+Youngauthor=M.+K.+Gilson&title=Grid+inhomogeneous+solvation+theory%3A+Hydration+structure+and+thermodynamics+of+the+miniature+receptor+cucurbit%5B7%5Duril&doi=10.1063%2F1.4733951"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit86R"><div class="casContent"><span class="casTitleNuber">86</span><div class="casTitle"><span class="NLM_cas:atitle">Grid inhomogeneous solvation theory: Hydration structure and thermodynamics of the miniature receptor cucurbit[7]uril</span></div><div class="casAuthors">Nguyen, Crystal N.; Kurtzman Young, Tom; Gilson, Michael K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">137</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">044101/1-044101/17</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    
            (<span class="NLM_cas:orgname">American Institute of Physics</span>)
        </div><div class="casAbstract">The displacement of perturbed water upon binding is believed to play a crit. role in the thermodn. of biomol. recognition, but it is nontrivial to unambiguously define and answer questions about this process.  We address this issue by introducing grid inhomogeneous solvation theory (GIST), which discretizes the equations of inhomogeneous solvation theory (IST) onto a three-dimensional grid situated in the region of interest around a solute mol. or complex.  Snapshots from explicit solvent simulations are used to est. localized solvation entropies, energies, and free energies assocd. with the grid boxes, or voxels, and properly summing these thermodn. quantities over voxels yields information about hydration thermodn.  GIST thus provides a smoothly varying representation of water properties as a function of position, rather than focusing on hydration sites where solvent is present at high d.  It therefore accounts for full or partial displacement of water from sites that are highly occupied by water, as well as for partly occupied and water-depleted regions around the solute.  GIST can also provide a well-defined est. of the solvation free energy and therefore enables a rigorous end-states anal. of binding.  For example, one may not only use a first GIST calcn. to project the thermodn. consequences of displacing water from the surface of a receptor by a ligand, but also account, in a second GIST calcn., for the thermodn. of subsequent solvent reorganization around the bound complex.  In the present study, a first GIST anal. of the mol. host cucurbit7uril is found to yield a rich picture of hydration structure and thermodn. in and around this miniature receptor.  One of the most striking results is the observation of a toroidal region of high water d. at the center of the host's nonpolar cavity.  Despite its high d., the water in this toroidal region is disfavored energetically and entropically, and hence may contribute to the known ability of this small receptor to bind guest mols. with unusually high affinities.  Interestingly, the toroidal region of high water d. persists even when all partial charges of the receptor are set to zero.  Thus, localized regions of high solvent d. can be generated in a binding site without strong, attractive solute-solvent interactions.  (c) 2012 American Institute of Physics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrkKZ05RaZJbbVg90H21EOLACvtfcHk0lijJ6rrxCpSxw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVOgurzJ&md5=d8eceb021e903950784786a656fdfb40</span></div><a href="/servlet/linkout?suffix=cit86&amp;dbid=16384&amp;doi=10.1063%2F1.4733951&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.4733951%26sid%3Dliteratum%253Aachs%26aulast%3DNguyen%26aufirst%3DC.%2BN.%26aulast%3DKurtzman%2BYoung%26aufirst%3DT.%26aulast%3DGilson%26aufirst%3DM.%2BK.%26atitle%3DGrid%2520inhomogeneous%2520solvation%2520theory%253A%2520Hydration%2520structure%2520and%2520thermodynamics%2520of%2520the%2520miniature%2520receptor%2520cucurbit%255B7%255Duril%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D2012%26volume%3D137%26spage%3D044101%26doi%3D10.1063%2F1.4733951" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref87"><div class="reference"><strong class="refLabel"><a href="#ref87" class="rightTabRefNumLink">87</a></strong><div class="NLM_citation" id="rightTab-cit87"><span><span class="NLM_contrib-group">Hu, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lill, M. A.</span><span> </span><span class="NLM_article-title">WATsite: Hydration site prediction program with PyMOL interface</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">35</span><span class="NLM_x">, </span> <span class="NLM_fpage">1255</span><span class="NLM_x">–</span> <span class="NLM_lpage">1260</span><span class="refDoi"> DOI: 10.1002/jcc.23616</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1002%2Fjcc.23616" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=24752524" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmslant7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2014&pages=1255-1260&author=B.+Huauthor=M.+A.+Lill&title=WATsite%3A+Hydration+site+prediction+program+with+PyMOL+interface&doi=10.1002%2Fjcc.23616"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit87R"><div class="casContent"><span class="casTitleNuber">87</span><div class="casTitle"><span class="NLM_cas:atitle">WATsite: Hydration site prediction program with PyMOL interface</span></div><div class="casAuthors">Hu, Bingjie; Lill, Markus A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1255-1260</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Water mols. that mediate protein-ligand interactions or are released from the binding site on ligand binding can contribute both enthalpically and entropically to the free energy of ligand binding.  To elucidate the thermodn. profile of individual water mols. and their potential contribution to ligand binding, a hydration site anal. program WATsite was developed together with an easy-to-use graphical user interface based on PyMOL.  WATsite identifies hydration sites from a mol. dynamics simulation trajectory with explicit water mols.  The free energy profile of each hydration site is estd. by computing the enthalpy and entropy of the water mol. occupying a hydration site throughout the simulation.  The results of the hydration site anal. can be displayed in PyMOL.  A key feature of WATsite is that it is able to est. the protein desolvation free energy for any user specified ligand.  The WATsite program and its PyMOL plugin are available free of charge from http://people.pnhs.purdue.edu/∼mlill/software.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZ0FIFaoOGZ7Vg90H21EOLACvtfcHk0lgmfYxf_b-0QA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmslant7Y%253D&md5=b7f10d1a30efd3bc5c69bbfc03ee947e</span></div><a href="/servlet/linkout?suffix=cit87&amp;dbid=16384&amp;doi=10.1002%2Fjcc.23616&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.23616%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DB.%26aulast%3DLill%26aufirst%3DM.%2BA.%26atitle%3DWATsite%253A%2520Hydration%2520site%2520prediction%2520program%2520with%2520PyMOL%2520interface%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2014%26volume%3D35%26spage%3D1255%26epage%3D1260%26doi%3D10.1002%2Fjcc.23616" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref88"><div class="reference"><strong class="refLabel"><a href="#ref88" class="rightTabRefNumLink">88</a></strong><div class="NLM_citation" id="rightTab-cit88"><span><span class="NLM_contrib-group">Grant, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pickup, B. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nicholls, A.</span><span> </span><span class="NLM_article-title">A smooth permittivity function for Poisson–Boltzmann solvation methods</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">608</span><span class="NLM_x">–</span> <span class="NLM_lpage">640</span><span class="refDoi"> DOI: 10.1002/jcc.1032</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1002%2Fjcc.1032" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BD3MXisFaltro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2001&pages=608-640&author=J.+A.+Grantauthor=B.+T.+Pickupauthor=A.+Nicholls&title=A+smooth+permittivity+function+for+Poisson%E2%80%93Boltzmann+solvation+methods&doi=10.1002%2Fjcc.1032"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit88R"><div class="casContent"><span class="casTitleNuber">88</span><div class="casTitle"><span class="NLM_cas:atitle">A smooth permittivity function for poisson-boltzmann solvation methods</span></div><div class="casAuthors">Grant, J. Andrew; Pickup, Barry T.; Nicholls, Anthony</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">608-640</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">This work introduces a continuous smooth permittivity function into Poisson-Boltzmann techniques for continuum approaches to modeling the solvation of small mols. and proteins.  The permittivity function is derived using a Gaussian method to describe vol. exclusion.  The new method allows a rigorous detn. of solvent forces within a grid-based technol.  The generality of approach is demonstrated by considering a range of applications for small mols. and macromols.  We also present a very complete statistical anal. of grid errors, and show that the accuracy of our Gaussian-based method is improved over std. techniques.  The method has been implemented in a new code called ZAP, which is freely available to academic institutions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNBKMD0CGO77Vg90H21EOLACvtfcHk0lgmfYxf_b-0QA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXisFaltro%253D&md5=5221cbe055fe89308d2abdc629e76f20</span></div><a href="/servlet/linkout?suffix=cit88&amp;dbid=16384&amp;doi=10.1002%2Fjcc.1032&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.1032%26sid%3Dliteratum%253Aachs%26aulast%3DGrant%26aufirst%3DJ.%2BA.%26aulast%3DPickup%26aufirst%3DB.%2BT.%26aulast%3DNicholls%26aufirst%3DA.%26atitle%3DA%2520smooth%2520permittivity%2520function%2520for%2520Poisson%25E2%2580%2593Boltzmann%2520solvation%2520methods%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2001%26volume%3D22%26spage%3D608%26epage%3D640%26doi%3D10.1002%2Fjcc.1032" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref89"><div class="reference"><strong class="refLabel"><a href="#ref89" class="rightTabRefNumLink">89</a></strong><div class="NLM_citation" id="rightTab-cit89"><span><span class="NLM_contrib-group">Huggins, D. J.</span><span> </span><span class="NLM_article-title">Application of inhomogeneous fluid solvation theory to model the distribution and thermodynamics of water molecules around biomolecules</span> <span class="citation_source-journal">Phys. Chem. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">15106</span><span class="NLM_x">–</span> <span class="NLM_lpage">15117</span><span class="refDoi"> DOI: 10.1039/c2cp42631e</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=10.1039%2Fc2cp42631e" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=23037989" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFersrbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2012&pages=15106-15117&author=D.+J.+Huggins&title=Application+of+inhomogeneous+fluid+solvation+theory+to+model+the+distribution+and+thermodynamics+of+water+molecules+around+biomolecules&doi=10.1039%2Fc2cp42631e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit89R"><div class="casContent"><span class="casTitleNuber">89</span><div class="casTitle"><span class="NLM_cas:atitle">Application of inhomogeneous fluid solvation theory to model the distribution and thermodynamics of water molecules around biomolecules</span></div><div class="casAuthors">Huggins, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Physical Chemistry Chemical Physics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">43</span>),
    <span class="NLM_cas:pages">15106-15117</span>CODEN:
                <span class="NLM_cas:coden">PPCPFQ</span>;
        ISSN:<span class="NLM_cas:issn">1463-9076</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The structures of biomols. and the strengths of assocn. between them depend critically on interactions with water mols.  Thus, understanding these interactions is a prerequisite for understanding the structure and function of all biomols.  Inhomogeneous fluid solvation theory provides a framework to derive thermodn. properties of individual water mols. from a statistical mech. anal.  In this work, two biomols. are analyzed to probe the distribution and thermodn. of surrounding water mols.  The great majority of hydration sites are predicted to contribute favorably to the total free energy with respect to bulk water, though hydration sites close to non-polar regions of the solute do not contribute significantly.  Anal. of a biomol. with a pos. and neg. charged functional group predicts that a charged species perturbs the free energy of water mols. to a distance of approx. 6.0 Å.  Interestingly, short simulations are found to provide converged predictions if samples are taken with sufficient frequency, a finding that has the potential to significantly reduce the required computational cost of such anal.  In addn., the predicted thermodn. properties of hydration sites with the potential for direct hydrogen bonding interactions are found to disagree significantly for two different water models.  This study provides important information on how inhomogeneous fluid solvation theory can be employed to understand the structures and intermol. interactions of biomols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqL-I9p9H8TSLVg90H21EOLACvtfcHk0lgmfYxf_b-0QA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFersrbN&md5=828eb1eee170fba0526fff24d87b0fbd</span></div><a href="/servlet/linkout?suffix=cit89&amp;dbid=16384&amp;doi=10.1039%2Fc2cp42631e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc2cp42631e%26sid%3Dliteratum%253Aachs%26aulast%3DHuggins%26aufirst%3DD.%2BJ.%26atitle%3DApplication%2520of%2520inhomogeneous%2520fluid%2520solvation%2520theory%2520to%2520model%2520the%2520distribution%2520and%2520thermodynamics%2520of%2520water%2520molecules%2520around%2520biomolecules%26jtitle%3DPhys.%2520Chem.%2520Chem.%2520Phys.%26date%3D2012%26volume%3D14%26spage%3D15106%26epage%3D15117%26doi%3D10.1039%2Fc2cp42631e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref90"><div class="reference"><strong class="refLabel"><a href="#ref90" class="rightTabRefNumLink">90</a></strong><div class="NLM_citation" id="rightTab-cit90"><span><span class="NLM_contrib-group">Gerogiokas, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Calabro, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Henchman, R. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Southey, M. W. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Law, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Michel, J.</span><span> </span><span class="NLM_article-title">Prediction of Small Molecule Hydration Thermodynamics with Grid Cell Theory</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">35</span><span class="NLM_x">–</span> <span class="NLM_lpage">48</span><span class="refDoi"> DOI: 10.1021/ct400783h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct400783h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvV2rsLzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=35-48&author=G.+Gerogiokasauthor=G.+Calabroauthor=R.+H.+Henchmanauthor=M.+W.+Y.+Southeyauthor=R.+J.+Lawauthor=J.+Michel&title=Prediction+of+Small+Molecule+Hydration+Thermodynamics+with+Grid+Cell+Theory&doi=10.1021%2Fct400783h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit90R"><div class="casContent"><span class="casTitleNuber">90</span><div class="casTitle"><span class="NLM_cas:atitle">Prediction of Small Molecule Hydration Thermodynamics with Grid Cell Theory</span></div><div class="casAuthors">Gerogiokas, Georgios; Calabro, Gaetano; Henchman, Richard H.; Southey, Michelle W. Y.; Law, Richard J.; Michel, Julien</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">35-48</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">An efficient methodol. has been developed to quantify water energetics by anal. of explicit solvent mol. simulations of org. and biomol. systems.  The approach, grid cell theory (GCT), relies on a discretization of the cell theory methodol. on a three-dimensional grid to spatially resolve the d., enthalpy, and entropy of water mols. in the vicinity of solute(s) of interest.  Entropies of hydration are found to converge more efficiently than enthalpies of hydration.  GCT predictions of free energies of hydration on a data set of small mols. are strongly correlated with thermodn. integration predictions.  Agreement with the expt. is comparable for both approaches.  A key advantage of GCT is its ability to provide from a single simulation insightful graphical analyses of spatially resolved components of the enthalpies and entropies of hydration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqOK8_kdK9hLLVg90H21EOLACvtfcHk0ljf2hTFXIhnRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvV2rsLzE&md5=2be9e31aa30232e9d60e73039177a634</span></div><a href="/servlet/linkout?suffix=cit90&amp;dbid=16384&amp;doi=10.1021%2Fct400783h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct400783h%26sid%3Dliteratum%253Aachs%26aulast%3DGerogiokas%26aufirst%3DG.%26aulast%3DCalabro%26aufirst%3DG.%26aulast%3DHenchman%26aufirst%3DR.%2BH.%26aulast%3DSouthey%26aufirst%3DM.%2BW.%2BY.%26aulast%3DLaw%26aufirst%3DR.%2BJ.%26aulast%3DMichel%26aufirst%3DJ.%26atitle%3DPrediction%2520of%2520Small%2520Molecule%2520Hydration%2520Thermodynamics%2520with%2520Grid%2520Cell%2520Theory%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2014%26volume%3D10%26spage%3D35%26epage%3D48%26doi%3D10.1021%2Fct400783h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref91"><div class="reference"><strong class="refLabel"><a href="#ref91" class="rightTabRefNumLink">91</a></strong><div class="NLM_citation" id="rightTab-cit91"><span><span class="NLM_contrib-group">Baron, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Setny, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Andrew McCammon, J.</span><span> </span><span class="NLM_article-title">Water in Cavity–Ligand Recognition</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">132</span><span class="NLM_x">, </span> <span class="NLM_fpage">12091</span><span class="NLM_x">–</span> <span class="NLM_lpage">12097</span><span class="refDoi"> DOI: 10.1021/ja1050082</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja1050082" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;key=1%3ACAS%3A528%3ADC%252BC3cXpvFKisb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2010&pages=12091-12097&author=R.+Baronauthor=P.+Setnyauthor=J.+Andrew+McCammon&title=Water+in+Cavity%E2%80%93Ligand+Recognition&doi=10.1021%2Fja1050082"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit91R"><div class="casContent"><span class="casTitleNuber">91</span><div class="casTitle"><span class="NLM_cas:atitle">Water in Cavity-Ligand Recognition</span></div><div class="casAuthors">Baron, Riccardo; Setny, Piotr; Andrew McCammon, J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">132</span>
        (<span class="NLM_cas:issue">34</span>),
    <span class="NLM_cas:pages">12091-12097</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We use explicit solvent mol. dynamics simulations to est. free energy, enthalpy, and entropy changes along the cavity-ligand assocn. coordinate for a set of seven model systems with varying physicochem. properties.  Owing to the simplicity of the considered systems we can directly investigate the role of water thermodn. in mol. recognition.  A broad range of thermodn. signatures is found in which water (rather than cavity or ligand) enthalpic or entropic contributions appear to drive cavity-ligand binding or rejection.  The unprecedented, nanoscale picture of hydration thermodn. can help the interpretation and design of protein-ligand binding expts.  Our study opens appealing perspectives to tackle the challenge of solvent entropy estn. in complex systems and for improving mol. simulation models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrh8IROwcNdbLVg90H21EOLACvtfcHk0ljf2hTFXIhnRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXpvFKisb0%253D&md5=38100ba43b18ce488a768ce7da64b128</span></div><a href="/servlet/linkout?suffix=cit91&amp;dbid=16384&amp;doi=10.1021%2Fja1050082&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja1050082%26sid%3Dliteratum%253Aachs%26aulast%3DBaron%26aufirst%3DR.%26aulast%3DSetny%26aufirst%3DP.%26aulast%3DAndrew%2BMcCammon%26aufirst%3DJ.%26atitle%3DWater%2520in%2520Cavity%25E2%2580%2593Ligand%2520Recognition%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2010%26volume%3D132%26spage%3D12091%26epage%3D12097%26doi%3D10.1021%2Fja1050082" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2RH1" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2RH1','PDB','2RH1'); return false;">PDB: 2RH1</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GBR" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GBR','PDB','4GBR'); return false;">PDB: 4GBR</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i6"><a href="/doi/suppl/10.1021/acs.jmedchem.6b00358">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_73211"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.6b00358">10.1021/acs.jmedchem.6b00358</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Full list of binding and kinetic data for all compounds used in this study, ligand RMSD plots from the molecular dynamics trajectories (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.6b00358/suppl_file/jm6b00358_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.6b00358/suppl_file/jm6b00358_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b00358/suppl_file/jm6b00358_si_001.pdf">jm6b00358_si_001.pdf (657.66 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b00358/suppl_file/jm6b00358_si_002.csv">jm6b00358_si_002.csv (0.53 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.6b00358&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.6b00358%3Bwgroup%3Astring%3AACHS+website+Group%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2016.59.issue-12%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.6b00358" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                4MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-b425991822a79ada59b3.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-fb88834b824b41b135ad.js"></script>

<script type="text/javascript" src="/wro/ke1q~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"66798a3fe961d1db","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.6.0","si":10}'></script>
</body>
</html>
