
<html lang="en"     class="pb-page"  data-request-id="1437dc80-2cad-4071-921c-e6eb9674326e"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;issue:issue:10.1021/jmcmar.2020.63.issue-4;pageGroup:string:Publication Pages;article:article:10.1021/acs.jmedchem.9b01502;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Structure-Based Design of a Potent and Selective Covalent Inhibitor for SRC Kinase That Targets a P-Loop Cysteine" /></meta><meta name="dc.Creator" content="Guangyan  Du" /></meta><meta name="dc.Creator" content="Suman  Rao" /></meta><meta name="dc.Creator" content="Deepak  Gurbani" /></meta><meta name="dc.Creator" content="Nathaniel J.  Henning" /></meta><meta name="dc.Creator" content="Jie  Jiang" /></meta><meta name="dc.Creator" content="Jianwei  Che" /></meta><meta name="dc.Creator" content="Annan  Yang" /></meta><meta name="dc.Creator" content="Scott B.  Ficarro" /></meta><meta name="dc.Creator" content="Jarrod A.  Marto" /></meta><meta name="dc.Creator" content="Andrew J.  Aguirre" /></meta><meta name="dc.Creator" content="Peter K.  Sorger" /></meta><meta name="dc.Creator" content="Kenneth D.  Westover" /></meta><meta name="dc.Creator" content="Tinghu  Zhang" /></meta><meta name="dc.Creator" content="Nathanael S.  Gray" /></meta><meta name="dc.Description" content="SRC is a major regulator of many signaling pathways and contributes to cancer development. However, development of a selective SRC inhibitor has been challenging, and FDA-approved SRC inhibitors, d..." /></meta><meta name="Description" content="SRC is a major regulator of many signaling pathways and contributes to cancer development. However, development of a selective SRC inhibitor has been challenging, and FDA-approved SRC inhibitors, d..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="January 14, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b01502" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2020 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b01502" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b01502" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b01502" /></link>
        
    
    

<title>Structure-Based Design of a Potent and Selective Covalent Inhibitor for SRC Kinase That Targets a P-Loop Cysteine | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b01502" /></meta><meta property="og:title" content="Structure-Based Design of a Potent and Selective Covalent Inhibitor for SRC Kinase That Targets a P-Loop Cysteine" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01502/20200221/images/large/jm9b01502_0017.jpeg" /></meta><meta property="og:description" content="SRC is a major regulator of many signaling pathways and contributes to cancer development. However, development of a selective SRC inhibitor has been challenging, and FDA-approved SRC inhibitors, dasatinib and bosutinib, are multitargeted kinase inhibitors. Here, we describe our efforts to develop a selective SRC covalent inhibitor by targeting cysteine 277 on the P-loop of SRC. Using a promiscuous covalent kinase inhibitor (CKI) SM1-71 as a starting point, we developed covalent inhibitor 15a, which discriminates SRC from other covalent targets of SM1-71 including TAK1 and FGFR1. As an irreversible covalent inhibitor, compound 15a exhibited sustained inhibition of SRC signaling both in vitro and in vivo. Moreover, 15a exhibited potent antiproliferative effects in nonsmall cell lung cancer cell lines harboring SRC activation, thus providing evidence that this approach may be promising for further drug development efforts." /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta><meta name="twitter:description" content="SRC is a major regulator of many signaling pathways and contributes to cancer development. However, development of a selective SRC inhibitor has been challenging, and FDA-approved SRC inhibitors, dasatinib and bosutinib, are multitargeted kinase inhibitors. Here, we describe our efforts to develop a selective SRC covalent inhibitor by targeting cysteine 277 on the P-loop of SRC. Using a promiscuous covalent kinase inhibitor (CKI) SM1-71 as a starting point, we developed covalent inhibitor 15a, which discriminates SRC from other covalent targets of SM1-71 including TAK1 and FGFR1. As an irreversible covalent inhibitor, compound 15a exhibited sustained inhibition of SRC signaling both in vitro and in vivo. Moreover, 15a exhibited potent antiproliferative effects in nonsmall cell lung cancer cell lines harboring SRC activation, thus providing evidence that this approach may be promising for further drug development efforts." /></meta><meta name="twitter:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01502/20200221/images/large/jm9b01502_0017.jpeg" /></meta><meta name="twitter:title" content="Structure-Based Design of a Potent and Selective Covalent Inhibitor for SRC Kinase That Targets a P-Loop Cysteine" /></meta>
        
            <meta property="og:type" content="Article" /></meta>
        

        
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b01502"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/ke1q~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-6ee29527b3d8ff00a106.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01502">
            
        





































    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-12/achre4.2021.54.issue-12/20210615/achre4.2021.54.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-6/amrcda.2021.2.issue-6/20210625/amrcda.2021.2.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-6/aabmcb.2021.4.issue-6/20210621/aabmcb.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-6/aaembp.2021.3.issue-6/20210622/aaembp.2021.3.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-6/aaemcq.2021.4.issue-6/20210628/aaemcq.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-25/aamick.2021.13.issue-25/20210630/aamick.2021.13.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-6/aanmf6.2021.4.issue-6/20210625/aanmf6.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-6/aapmcd.2021.3.issue-6/20210611/aapmcd.2021.3.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-6/abseba.2021.7.issue-6/20210614/abseba.2021.7.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-12/accacs.2021.11.issue-12/20210618/accacs.2021.11.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-6/acscii.2021.7.issue-6/20210623/acscii.2021.7.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-6/acbcct.2021.16.issue-6/20210618/acbcct.2021.16.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-3/achsc5.2021.28.issue-3/20210524/achsc5.2021.28.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-12/acncdm.2021.12.issue-12/20210616/acncdm.2021.12.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-6/aesccq.2021.5.issue-6/20210617/aesccq.2021.5.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-6/aelccp.2021.6.issue-6/20210611/aelccp.2021.6.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-6/aeecco.2021.1.issue-6/20210611/aeecco.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-6/aewcaa.2021.1.issue-6/20210611/aewcaa.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-5/afsthl.2021.1.issue-5/20210618/afsthl.2021.1.issue-5.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-6/aidcbc.2021.7.issue-6/20210611/aidcbc.2021.7.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-6/amlccd.2021.10.issue-6/20210615/amlccd.2021.10.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-6/amlcef.2021.3.issue-6/20210607/amlcef.2021.3.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-6/amclct.2021.12.issue-6/20210610/amclct.2021.12.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-6/ancac3.2021.15.issue-6/20210622/ancac3.2021.15.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-25/acsodf.2021.6.issue-25/20210629/acsodf.2021.6.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-6/apchd5.2021.8.issue-6/20210616/apchd5.2021.8.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-6/ascefj.2021.6.issue-6/20210625/ascefj.2021.6.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-25/ascecg.2021.9.issue-25/20210628/ascecg.2021.9.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-6/asbcd6.2021.10.issue-6/20210618/asbcd6.2021.10.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-25/ancham.2021.93.issue-25/20210629/ancham.2021.93.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-25/bichaw.2021.60.issue-25/20210629/bichaw.2021.60.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-6/bcches.2021.32.issue-6/20210616/bcches.2021.32.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-6/bomaf6.2021.22.issue-6/20210614/bomaf6.2021.22.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-24/cgeabj.2021.99.issue-24/20210628-01/cgeabj.2021.99.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-6/crtoec.2021.34.issue-6/20210621/crtoec.2021.34.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-12/chreay.2021.121.issue-12/20210623/chreay.2021.121.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-12/cmatex.2021.33.issue-12/20210622/cmatex.2021.33.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-6/cgdefu.2021.21.issue-6/20210602/cgdefu.2021.21.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-12/enfuem.2021.35.issue-12/20210617/enfuem.2021.35.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-12/esthag.2021.55.issue-12/20210615/esthag.2021.55.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-6/estlcu.2021.8.issue-6/20210608/estlcu.2021.8.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-25/iecred.2021.60.issue-25/20210630/iecred.2021.60.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-12/inocaj.2021.60.issue-12/20210621/inocaj.2021.60.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-6/jaaucr.2021.1.issue-6/20210628/jaaucr.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-25/jafcau.2021.69.issue-25/20210630/jafcau.2021.69.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-6/jceaax.2021.66.issue-6/20210610/jceaax.2021.66.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-6/jceda8.2021.98.issue-6/20210608/jceda8.2021.98.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-6/jcisd8.2021.61.issue-6/20210628/jcisd8.2021.61.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-6/jctcce.2021.17.issue-6/20210608/jctcce.2021.17.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/jmcmar.2021.64.issue-12/20210624/jmcmar.2021.64.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-6/jnprdf.2021.84.issue-6/20210625/jnprdf.2021.84.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-12/joceah.2021.86.issue-12/20210618/joceah.2021.86.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-24/jpcafh.2021.125.issue-24/20210624/jpcafh.2021.125.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-24/jpcbfk.2021.125.issue-24/20210624/jpcbfk.2021.125.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-24/jpccck.2021.125.issue-24/20210624/jpccck.2021.125.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-24/jpclcd.2021.12.issue-24/20210624/jpclcd.2021.12.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-6/jprobs.2021.20.issue-6/20210604/jprobs.2021.20.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-25/jacsat.2021.143.issue-25/20210630/jacsat.2021.143.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-6/jamsef.2021.32.issue-6/20210602/jamsef.2021.32.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-25/langd5.2021.37.issue-25/20210629/langd5.2021.37.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-12/mamobx.2021.54.issue-12/20210622/mamobx.2021.54.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-6/mpohbp.2021.18.issue-6/20210607/mpohbp.2021.18.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-12/nalefd.2021.21.issue-12/20210623/nalefd.2021.21.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-12/orlef7.2021.23.issue-12/20210618/orlef7.2021.23.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-6/oprdfk.2021.25.issue-6/20210618/oprdfk.2021.25.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-12/orgnd7.2021.40.issue-12/20210628/orgnd7.2021.40.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b01502&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b01502&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b01502&amp;href=/doi/10.1021/acs.jmedchem.9b01502" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 4</span><span class="cit-fg-pageRange">, 1624-1641</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/4" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.9b01423" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.9b01549" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Structure-Based Design of a Potent and Selective Covalent Inhibitor for SRC Kinase That Targets a P-Loop Cysteine</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Guangyan Du</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Guangyan Du</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts 02115, United States</div><div class="loa-info-affiliations-info">Department of Cancer Biology, Dana Farber Cancer Institute, 450 Brookline Avenue, Boston, Massachusetts 02215, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Guangyan++Du">Guangyan Du</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Suman Rao</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Suman Rao</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts 02115, United States</div><div class="loa-info-affiliations-info">Department of Cancer Biology, Dana Farber Cancer Institute, 450 Brookline Avenue, Boston, Massachusetts 02215, United States</div><div class="loa-info-affiliations-info">Laboratory of Systems Biology, Harvard Medical School, Boston, Massachusetts 02115, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Suman++Rao">Suman Rao</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Deepak Gurbani</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Deepak Gurbani</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Departments of Biochemistry and Radiation Oncology, The University of Texas Southwestern Medical Center at Dallas, Dallas, Texas 75390, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Deepak++Gurbani">Deepak Gurbani</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Nathaniel J. Henning</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Nathaniel J. Henning</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts 02115, United States</div><div class="loa-info-affiliations-info">Department of Cancer Biology, Dana Farber Cancer Institute, 450 Brookline Avenue, Boston, Massachusetts 02215, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Nathaniel+J.++Henning">Nathaniel J. Henning</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jie Jiang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jie Jiang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts 02115, United States</div><div class="loa-info-affiliations-info">Department of Cancer Biology, Dana Farber Cancer Institute, 450 Brookline Avenue, Boston, Massachusetts 02215, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jie++Jiang">Jie Jiang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jianwei Che</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jianwei Che</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts 02115, United States</div><div class="loa-info-affiliations-info">Department of Cancer Biology, Dana Farber Cancer Institute, 450 Brookline Avenue, Boston, Massachusetts 02215, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jianwei++Che">Jianwei Che</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Annan Yang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Annan Yang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medical Oncology, Dana Farber Cancer Institute, 450 Brookline Avenue, Boston, Massachusetts 02215, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Annan++Yang">Annan Yang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Scott B. Ficarro</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Scott B. Ficarro</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts 02115, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Scott+B.++Ficarro">Scott B. Ficarro</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jarrod A. Marto</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jarrod A. Marto</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts 02115, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jarrod+A.++Marto">Jarrod A. Marto</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Andrew J. Aguirre</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Andrew J. Aguirre</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medical Oncology, Dana Farber Cancer Institute, 450 Brookline Avenue, Boston, Massachusetts 02215, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Andrew+J.++Aguirre">Andrew J. Aguirre</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Peter K. Sorger</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Peter K. Sorger</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Laboratory of Systems Biology, Harvard Medical School, Boston, Massachusetts 02115, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Peter+K.++Sorger">Peter K. Sorger</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Kenneth D. Westover</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kenneth D. Westover</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Departments of Biochemistry and Radiation Oncology, The University of Texas Southwestern Medical Center at Dallas, Dallas, Texas 75390, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#ec87898282899884c29b899f98839a899eac99989f839998849b899f98899e82c2898899"><span class="__cf_email__" data-cfemail="5e353b30303b2a3670293b2d2a31283b2c1e2b2a2d312b2a36293b2d2a3b2c30703b3a2b">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kenneth+D.++Westover">Kenneth D. Westover</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-3653-5923" title="Orcid link">http://orcid.org/0000-0003-3653-5923</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Tinghu Zhang</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Tinghu Zhang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts 02115, United States</div><div class="loa-info-affiliations-info">Department of Cancer Biology, Dana Farber Cancer Institute, 450 Brookline Avenue, Boston, Massachusetts 02215, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#45112c2b222d301a1f2d242b22052123262c6b2d2437332437216b202130"><span class="__cf_email__" data-cfemail="82d6ebece5eaf7ddd8eae3ece5c2e6e4e1ebaceae3f0f4e3f0e6ace7e6f7">[email protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Tinghu++Zhang">Tinghu Zhang</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Nathanael S. Gray</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Nathanael S. Gray</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts 02115, United States</div><div class="loa-info-affiliations-info">Department of Cancer Biology, Dana Farber Cancer Institute, 450 Brookline Avenue, Boston, Massachusetts 02215, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#e3ad82978b828d82868fbca491829aa38785808acd8b829195829187cd868796"><span class="__cf_email__" data-cfemail="4e002f3a262f202f2b2211093c2f370e2a282d2760262f3c382f3c2a602b2a3b">[email protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Nathanael+S.++Gray">Nathanael S. Gray</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5354-7403" title="Orcid link">http://orcid.org/0000-0001-5354-7403</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b01502&amp;href=/doi/10.1021%2Facs.jmedchem.9b01502" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 4</span><span class="cit-pageRange">, 1624–1641</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">January 14, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>9 September 2019</li><li><span class="item_label"><b>Published</b> online</span>14 January 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 27 February 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b01502" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01502</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D1624%26pageCount%3D18%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DGuangyan%2BDu%252C%2BSuman%2BRao%252C%2BDeepak%2BGurbani%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D4%26contentID%3Dacs.jmedchem.9b01502%26title%3DStructure-Based%2BDesign%2Bof%2Ba%2BPotent%2Band%2BSelective%2BCovalent%2BInhibitor%2Bfor%2BSRC%2BKinase%2BThat%2BTargets%2Ba%2BP-Loop%2BCysteine%26numPages%3D18%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D1641%26publicationDate%3DFebruary%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b01502"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2934</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">7</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b01502" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Structure-Based Design of a Potent and Selective Covalent Inhibitor for SRC Kinase That Targets a P-Loop Cysteine&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Guangyan&quot;,&quot;last_name&quot;:&quot;Du&quot;},{&quot;first_name&quot;:&quot;Suman&quot;,&quot;last_name&quot;:&quot;Rao&quot;},{&quot;first_name&quot;:&quot;Deepak&quot;,&quot;last_name&quot;:&quot;Gurbani&quot;},{&quot;first_name&quot;:&quot;Nathaniel&quot;,&quot;last_name&quot;:&quot;J. Henning&quot;},{&quot;first_name&quot;:&quot;Jie&quot;,&quot;last_name&quot;:&quot;Jiang&quot;},{&quot;first_name&quot;:&quot;Jianwei&quot;,&quot;last_name&quot;:&quot;Che&quot;},{&quot;first_name&quot;:&quot;Annan&quot;,&quot;last_name&quot;:&quot;Yang&quot;},{&quot;first_name&quot;:&quot;Scott&quot;,&quot;last_name&quot;:&quot;B. Ficarro&quot;},{&quot;first_name&quot;:&quot;Jarrod&quot;,&quot;last_name&quot;:&quot;A. Marto&quot;},{&quot;first_name&quot;:&quot;Andrew&quot;,&quot;last_name&quot;:&quot;J. Aguirre&quot;},{&quot;first_name&quot;:&quot;Peter&quot;,&quot;last_name&quot;:&quot;K. Sorger&quot;},{&quot;first_name&quot;:&quot;Kenneth&quot;,&quot;last_name&quot;:&quot;D. Westover&quot;},{&quot;first_name&quot;:&quot;Tinghu&quot;,&quot;last_name&quot;:&quot;Zhang&quot;},{&quot;first_name&quot;:&quot;Nathanael&quot;,&quot;last_name&quot;:&quot;S. Gray&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;01&quot;,&quot;day&quot;:&quot;14&quot;,&quot;issue&quot;:&quot;4&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;1624-1641&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b01502&quot;},&quot;abstract&quot;:&quot;SRC is a major regulator of many signaling pathways and contributes to cancer development. However, development of a selective SRC inhibitor has been challenging, and FDA-approved SRC inhibitors, dasatinib and bosutinib, are multitargeted kinase inhibitors. Here, we describe our efforts to develop a selective SRC covalent inhibitor by targeting cysteine 277 on the P-loop of SRC. Using a promiscuous covalent kinase inhibitor (CKI) SM1-71 as a starting point, we developed covalent inhibitor 15a, which discriminates SRC from other covalent targets of SM1-71 including TAK1 and FGFR1. As an irreversible covalent inhibitor, compound 15a exhibited sustained inhibition of SRC signaling both in vitro and in vivo. Moreover, 15a exhibited potent antiproliferative effects in nonsmall cell lung cancer cell lines harboring SRC activation, thus providing evidence that this approach may be promising for further drug development efforts.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01502&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01502" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01502&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01502" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01502&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01502" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b01502&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01502&amp;href=/doi/10.1021/acs.jmedchem.9b01502" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b01502" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b01502" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (5 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01502&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b01502%26sid%3Dliteratum%253Aachs%26pmid%3D31935084%26genre%3Darticle%26aulast%3DDu%26date%3D2020%26atitle%3DStructure-Based%2BDesign%2Bof%2Ba%2BPotent%2Band%2BSelective%2BCovalent%2BInhibitor%2Bfor%2BSRC%2BKinase%2BThat%2BTargets%2Ba%2BP-Loop%2BCysteine%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D4%26spage%3D1624%26epage%3D1641%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (3)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291297" title="Cell signaling">Cell signaling</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=290831" title="Monomers">Monomers</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/4" title="Go to Volume 63, Issue 4"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/jmcmar.2020.63.issue-4/20200227/jmcmar.2020.63.issue-4.largecover.jpg" alt="Go to Volume 63, Issue 4"/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01502/20200221/images/medium/jm9b01502_0017.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01502/20200221/images/large/jm9b01502_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01502&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">SRC is a major regulator of many signaling pathways and contributes to cancer development. However, development of a selective SRC inhibitor has been challenging, and FDA-approved SRC inhibitors, dasatinib and bosutinib, are multitargeted kinase inhibitors. Here, we describe our efforts to develop a selective SRC covalent inhibitor by targeting cysteine 277 on the P-loop of SRC. Using a promiscuous covalent kinase inhibitor (CKI) <b>SM1-71</b> as a starting point, we developed covalent inhibitor <b>15a</b>, which discriminates SRC from other covalent targets of <b>SM1-71</b> including TAK1 and FGFR1. As an irreversible covalent inhibitor, compound <b>15a</b> exhibited sustained inhibition of SRC signaling both <i>in vitro</i> and <i>in vivo</i>. Moreover, <b>15a</b> exhibited potent antiproliferative effects in nonsmall cell lung cancer cell lines harboring SRC activation, thus providing evidence that this approach may be promising for further drug development efforts.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_13554" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_13554" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The product of the <i>c-SRC</i> gene, the first discovered proto-oncogene, SRC kinase belongs to a family of 11 nonreceptor tyrosine kinases known as SRC family kinases (SFKs).<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> SFKs have been suggested to play roles in various cancers, including breast, prostate, colon, head and neck, and lung.<a onclick="showRef(event, 'ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref2 ref3">(2,3)</a> SRC is a central signaling hub that can be activated by many factors, including G protein-coupled receptors (GPCRs), adhesion receptors, cytokine receptors, and receptor tyrosine kinases (RTKs) such as EGFR, VEGFR, HER2, and PDGFR.<a onclick="showRef(event, 'ref4 ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref4 ref5 ref6">(4−6)</a> As an effector of RTK signaling, SRC transduces survival signals to downstream effector pathways, such as PI3K/-AKT/mTOR and JAK/-STAT, to enhance diverse aspects of cancer development including proliferation, invasion, and differentiation.<a onclick="showRef(event, 'ref1 ref7 ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref1 ref7 ref8 ref9">(1,7−9)</a> For example, SRC transfection potentiates EGF-induced oncogenesis;<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> overexpression of SRC increases HER2–HER3 dimerization and enhances downstream signaling,<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> and under hypoxic conditions, activated SRC can lead to upregulation of VEGFR and thus spur angiogenesis.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> SRC overexpression/activation has been implicated in conferring resistance to several inhibitors across different tumors.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> For example, elevated SRC expression confers resistance to an anti-HER2 agent, Trastuzumab, in breast cancer and to ALK inhibitors in nonsmall cell lung cancer (NSCLC).<a onclick="showRef(event, 'ref14 ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref14 ref15 ref16">(14−16)</a> Due to its involvement in many cellular processes related to cancer development, SRC has long been considered a potential drug target in oncology.</div><div class="NLM_p">Extensive medicinal chemistry efforts have yielded a number of small molecule SRC inhibitors. Several compounds representing a diverse range of chemical scaffolds have entered clinical trials with two compounds ultimately approved by the FDA: dasatinib (approved in 2006) and bosutinib (approved in 2012).<a onclick="showRef(event, 'ref17 ref18 ref19 ref20 ref21 ref22 ref23 ref24 ref25'); return false;" href="javascript:void(0);" class="ref ref17 ref18 ref19 ref20 ref21 ref22 ref23 ref24 ref25">(17−25)</a> However, both of these drugs are also very potent inhibitors of BCR-ABL, a known oncogenic driver of chronic myelogenous leukemia (CML), which indicates these agents have demonstrated clinical efficacy. It is, therefore, unclear whether SRC inhibition contributes to the drug response in CML, especially given that drugs such as imatinib and nilotinib that do not inhibit SRC are also efficacious in CML. Furthermore, dasatinib is known to inhibit >30 kinases, while bosutinib inhibits >45 kinases, making it impossible to use these compounds as selective mechanistic probes for SRC-dependent pharmacology.<a onclick="showRef(event, 'ref26 ref27 ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref26 ref27 ref28 ref29">(26−29)</a> Similarly, most preclinical SRC inhibitors reported thus far lack selectivity and are not suitable for probing the biological functions of SRC. For example, although saracatinib (<b>AZD0530</b>) is a more selective SRC inhibitor than dasatinib, it also potently inhibits EGFR.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> To develop a SRC-selective small molecule inhibitor, we turned to covalent targeting whereby inhibitors containing a reactive warhead would be irreversibly bound to a unique cysteine on SRC.</div><div class="NLM_p">Targeted covalent inhibitors (TCIs) are an emerging pharmacological modality that offers several potential advantages such as improved potency, selectivity, pharmacodynamics, and efficacy over conventional reversible inhibitors.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> The main liabilities of TCIs are their potential for indiscriminate reactivity due to the presence of the reactive warhead that could result in covalent adduct formation with unexpected targets and the potential for idiosyncratic toxicities. Despite these potential issues, TCIs have proven to be effective and safe in the clinic, leading to the FDA approval of six covalent kinase inhibitors for different indications, including acalabrutinib (BTK), afatinib (EGFR), ibrutinib (BTK), neratinib (HER2), osimertinib (EGFR), and dacomitinib (EGFR).<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> Overall, TCIs represent a promising opportunity for the development of potent kinase inhibitors based on their ability to target specific cysteine residues while reducing reversible, off-target binding. A sequence alignment of the kinase family shows that SRC, along with eight other human protein kinases (Yes, Fgr, FGFR1–4, LIMK1, and TNK1), possess an equivalently positioned cysteine residue in the glycine-rich phosphate binding loop (P-loop) that can be targeted using TCIs. For example, we have previously described compound FIIN-2 that binds the P-loop cysteine of FGFRs (Cys477),<a onclick="showRef(event, 'ref33 ref34'); return false;" href="javascript:void(0);" class="ref ref33 ref34">(33,34)</a> while another study reported covalent targeting of SRC P-loop cysteine, Cys277, using the vinyl sulfonamide as the warhead.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a></div><div class="NLM_p last">We recently reported that <b>SM1-71</b>, a promiscuous covalent kinase inhibitor (CKI), reacts with the P-loop cysteine of SRC (Cys277), suggesting that development of selective CKIs for SRC may be feasible.<a onclick="showRef(event, 'ref36 ref37 ref38'); return false;" href="javascript:void(0);" class="ref ref36 ref37 ref38">(36−38)</a> However, SRC is just one of many <b>SM1-71</b> targets, as this compound reacts with cysteines located not only within the P-loop of a range of kinases but also at the DFG-1 position and within the activation loop as well. Here, we use <b>SM1-71</b> as a starting point for a structure-guided medicinal chemistry campaign to engineer out covalent binding to cysteines at the DFG-1 position, activation loop, and the P-loop cysteine of FGFR1–4, FGR, LIMK1, and TNK1, while maintaining covalent SRC activity. Our efforts resulted in compound <b>15a</b>, a SRC-directed CKI with an acrylamide warhead, that displayed efficient covalent target labeling and irreversible cellular inhibition of SRC signaling. Moreover, compound <b>15a</b> demonstrated preferential cell killing of NSCLC cell lines harboring SRC activation, motivating further therapeutic development of covalent SRC inhibitors.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Chemistry</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_87418" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_87418" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The syntheses of compounds <b>5a</b>–<b>5e</b>, <b>10a</b>–<b>10c</b>, and <b>15a</b>–<b>15i</b> are depicted in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. 5-Bromo-2,4-dichloropyrimidine (<b>1</b>) was sequentially coupled with benzene-1,2-diamine and 4-(4-methylpiperazin-1-yl)aniline at the 4 and 2 positions of the pyrimidine, respectively, to generate the intermediate <b>3</b>, which was subjected to a Suzuki coupling with an aryl or heteroaryl boronic acid or ester.</div><figure id="sch1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01502/20200221/images/medium/jm9b01502_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01502/20200221/images/large/jm9b01502_0006.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Syntheses of Compounds <b>5a</b>–<b>e</b>, <b>10a</b>–<b>c</b>, and <b>15a</b>–<b>i</b> from Commercially Available 5-Substituted-2,4-Dichloropyrimidine<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01502/20200221/images/large/jm9b01502_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01502&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) benzene-1,2-diamine, DIEA, <i>n</i>-BuOH, reflux; (b) 4-(4-methylpiperazin-1-yl)aniline, TFA, <i>s</i>-BuOH, reflux; (c) aryl boronic acid or ester, Pd(dppf)Cl<sub>2</sub>·DCM, Et<sub>3</sub>N, dioxane/H<sub>2</sub>O; (d) acryloyl chloride, NaHCO<sub>3</sub> aq., THF, ice bath; (e) different benzyl bromides, K<sub>2</sub>CO<sub>3</sub>, DMF, 50 °C; (f) different anilines, Et<sub>3</sub>N, DMF, ice bath; (g) benzene-1,2-diamine, DIEA, <i>n</i>-BuOH, rt.</p></p></figure><div class="NLM_p">Installation of the acrylamide on intermediate <b>4</b> with acryloyl chloride under basic aqueous conditions afforded final compounds <b>5a</b>–<b>e</b> in overall moderate yield. The commercially available 2,4-dichloropyrimidin-5-ol (<b>6</b>) and 2,4-dichloropyrimidine-5-carbonyl chloride (<b>11</b>) were reacted with benzyl bromides and anilines, respectively, and the resulting intermediates <b>7</b> and <b>12</b> were converted to final compounds <b>10</b> and <b>15</b> using the same reaction route as for compound <b>5</b>.</div><div class="NLM_p">For the synthesis of compounds <b>17</b>, we first established the amide group at the 5 position of the pyrimidine with the method used for intermediate <b>13a</b> in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. Then, the acrylamide was installed, leading to intermediate <b>16</b>, which is stable under the acidic conditions used for the amination (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>). <a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a> shows the synthesis for compounds that bear different covalent warheads. First, a S<sub>N</sub>2 substitution reaction with <i>meta</i>-, <i>ortho</i>-, and <i>para</i>-benzenediamine was performed, followed by a second substitution at the 2 position of intermediate <b>18</b>. Installation of an electrophile, such as chloroacetamide or vinyl sulfonamide, as the last step provided compounds <b>20a</b>–<b>20f</b>.</div><figure id="sch2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01502/20200221/images/medium/jm9b01502_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01502/20200221/images/large/jm9b01502_0007.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compounds <b>17a</b>–<b>f</b> from <b>13a</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01502/20200221/images/large/jm9b01502_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01502&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) acryloyl chloride, NaHCO<sub>3</sub> aq., THF, ice bath; (b) different anilines, TFA, <i>sec</i>-BuOH, reflux.</p></p></figure><figure id="sch3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01502/20200221/images/medium/jm9b01502_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01502/20200221/images/large/jm9b01502_0008.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Compounds <b>20a</b>–<b>f</b> from <b>12a</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01502/20200221/images/large/jm9b01502_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01502&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) different benzene diamines, DIEA, <i>n</i>-BuOH, rt; (b) 4-(4-methylpiperazin-1-yl)aniline, TFA, <i>s</i>-BuOH, reflux; (c) different electrophiles, NaHCO<sub>3</sub> aq., THF, ice bath or Et<sub>3</sub>N, DCM, ice bath.</p></p></figure></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i7">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_28846" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_28846" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Prior to this work, we established that <b>SM1-71</b> was a promiscuous CKI that labeled multiple cysteine sites, including at the DFG-1 position, at the activation loop, and on the P-loop.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> A chemoproteomics study of <b>SM1-71</b> revealed that SRC is one of the modified proteins, and a trypsin digestion MS/MS experiment demonstrated covalent labeling of Cys277 on the P-loop. Additionally, the crystal structure of <b>SM1-71</b> bound to SRC (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ATE">6ATE</a>) revealed that anilinopyrimidine of <b>SM1-71</b> engages the backbone of Met341 in the hinge region of SRC via two hydrogen bonds, thus bringing the acrylamide warhead in close proximity to Cys277 SRC. To better understand the binding mode and identify potential selectivity features, we performed covalent docking of <b>SM1-71</b> into FGFR1 using Glide.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> FGFR1 was chosen for this analysis because our previous study revealed <b>SM1-71</b>-SRC binding and also showed that <b>SM1-71</b> targeted FGFR1 via a P-loop cysteine.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> Our docking results suggested that <b>SM1-71</b> interacts with SRC and FGFR1 in a very similar manner (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). Both the docking result of <b>SM1-71</b> with FGFR1 and the previously solved crystal structure of <b>SM1-71</b> with SRC (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ATE">6ATE</a>) reveal that the hydrophobic pocket I is not occupied. Additionally, a cocrystal of <b>SM1-71</b> bound to TAK1 (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5J7S">5J7S</a>), another one of its covalent targets, showed a smaller hydrophobic pocket I relative to SRC. We were, therefore, interested in investigating whether selectivity for SRC could be achieved by exploiting this pocket. Furthermore, an inspection of the gatekeeper residues revealed another difference: the apolar and bulkier valine of FGFR and methionine of TAK1 (Met104), versus SRC’s threonine, supporting the idea that developing a SRC-selective CKI may be feasible.</div><figure id="fig1" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01502/20200221/images/medium/jm9b01502_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01502/20200221/images/large/jm9b01502_0001.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. (a) Superposition of the binding mode of <b>SM1-71</b> in SRC with FGFR1 and TAK1 kinase domains. Red dashed lines indicate hinge hydrogen bonds. The spheres show the size of the gatekeeper residues: red for SRC, gold for FGFR1, and cyan for TAK1. The red, gold, and cyan ribbons represent SRC, FGFR1, and TAK1 kinases, respectively. DFG-1 Cys of TAK1 is shown in sphere representation with a carbon backbone in cyan. (b) Crystal structure of <b>SM1-71</b> covalently bound to SRC kinase (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ATE">6ATE</a>). <b>SM1-71</b> covalently binds to P-loop Cys277. The structure alignment was done using Schrodinger Suite protein alignment tool with default settings, and the figures were generated with pymol.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01502/20200221/images/large/jm9b01502_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01502&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Starting from <b>SM1-71</b>, we tailored the modifications at the C5 position on the anilinopyrimidine ring to fine-tune the selectivity for SRC. We used commercial Z’LYTE and LanthaScreen kinase assays from Invitrogen to measure all biochemical IC<sub>50</sub>’s reported in this work. As shown in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, the first series of modifications involved the introduction of a monoaryl ring: thiophene (<b>5a</b>), furan (<b>5b</b>), and 6-methoxypyridine (<b>5c</b>) resulted in enhanced inhibitory activity against FGFR1, presumably as a consequence of improved hydrophobic interactions in the back-pocket. In contrast, the bulkiness of R1 groups on compounds <b>5a</b>–<b>5e</b> significantly abrogate TAK1 inhibition, likely due to a steric clash with Met104. Compounds <b>5d</b> and <b>5e</b> that bear 4-acetophenone or 4-oxidibenzene start to lose activity against FGFR, albeit to a limited extent. Next, we asked whether extending the phenyl ring further into the pocket by adding an ether linkage would further improve the selectivity. We therefore generated compounds <b>10a</b>, <b>10b</b>, and <b>10c</b>. Compounds <b>10a</b> and <b>10b</b> showed some improvement in selectivity, whereas <b>10c</b> was the most advanced compound, which resulted in an IC<sub>50</sub> > 1 μM on FGFR1, a 20-fold loss compared to <b>SM1-71</b>.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. IC<sub>50</sub> Values for Compounds <b>5a</b>–<b>e</b> and <b>10a</b>–<b>c</b> against SRC, FGFR1, and TAK1<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01502/20200221/images/medium/jm9b01502_0009.gif" alt="" id="gr5" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01502/20200221/images/medium/jm9b01502_0010.gif" alt="" id="gr6" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">“-” means not applicable.</p></div></div><div></div></div><div class="NLM_p">Next, we asked whether an amide bond, as a more polar functional group, at the C5 position influenced the activity on both SRC and FGFR1. Compared to compounds with a C–C (<b>5a</b>–<b>5e</b>) or C–O (<b>10a</b>–<b>10c</b>) bond, the <b>SM1-71</b> derivatives we synthesized with an amide bond (<b>15a</b>–<b>i</b>) generally demonstrated significantly decreased activity on FGFR1 (with compounds <b>15b</b>, <b>15d</b>, <b>15e</b>, <b>15h</b>, and <b>15i</b> showing activity above 10 μM) and a narrow structure–activity relationship (SAR) for SRC. For example, SRC tolerates a phenyl group with up to 3 substitutions such as methyl or chloride at the <i>ortho</i> or <i>para</i> position (<b>15a</b>, <b>15b</b>, <b>15c</b>, and <b>15d</b>) but not at the <i>meta</i> position (<b>15e</b>, <b>15f</b>, and <b>15g</b>). Replacement of the methyl group on compound <b>15a</b> with ethyl (<b>15h</b>) resulted in a 40-fold loss of activity on SRC, suggesting a steric clash. The trajectory of the amide-conjugated phenyl ring on compound <b>15i</b> likely differs from ether-linked compound <b>10c</b>, as a complete loss of activity on both SRC and FGFR1 was observed with <b>15i</b>. Overall, in this series, compound <b>15a</b> emerged as a lead for further optimization, given that it displayed an IC<sub>50</sub> of 3 nM in a fixed time point assay against SRC, while offering a broad selectivity window (more than 10<sup>3</sup>) versus FGFR1 (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>).</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. IC<sub>50</sub> Values for Compounds <b>15a</b>–<b>i</b> against SRC and FGFR1</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01502/20200221/images/medium/jm9b01502_0011.gif" alt="" id="gr7" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01502/20200221/images/medium/jm9b01502_0012.gif" alt="" id="gr8" /></img><div></div></div><div class="NLM_p">Given compound <b>15a</b>’s superior potency and selectivity, we made further changes to the <b>15a</b> scaffold, extending our optimization to parts of the molecule that mediate interactions with hydrophobic binding pocket II and the solvent-exposed region of SRC. The moiety that binds in the solvent-exposed region, such as piperazine, can be installed at the <i>meta</i> (<b>17a</b>) or <i>para</i> position (<b>17b</b>), whereas the <i>ortho</i> position (<b>17c</b>) results in loss of binding, likely due to interruption of binding to the kinase hinge interaction. A similar result was observed once a methoxy group was introduced at this position (<b>17d</b> and <b>17e</b>). In contrast, a methyl group at the <i>meta</i> position is tolerated as shown by compound <b>17f</b>, which displayed a single digit nanomolar IC<sub>50</sub> on SRC (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). These attempts to enhance binding by exploring interactions with hydrophobic binding pocket II and the solvent-exposed region on SRC resulted in no improvement over <b>15a</b> as assessed by biochemical assays.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. IC<sub>50</sub> Values for Compounds <b>17a</b>–<b>f</b> against SRC</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01502/20200221/images/medium/jm9b01502_0013.gif" alt="" id="gr9" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01502/20200221/images/medium/jm9b01502_0014.gif" alt="" id="gr10" /></img><div></div></div><div class="NLM_p">We next explored the position of the acrylamide warhead and demonstrated that the <i>ortho</i> position is optimal, as the <i>meta</i> or <i>para</i> position (<b>20a</b> and <b>20b</b>) resulted in 10–15-fold loss of activity on SRC. Furthermore, the <i>ortho</i> position tolerates other short warheads such as chloroacetamide (<b>20c</b>) and vinyl sulfonamide (<b>20d</b>), but adding a bulkier methylene dimethylamine to the acrylamide likely caused a steric clash with the P-loop, compromising the IC<sub>50</sub> of compound <b>20e</b> (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>).</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. IC<sub>50</sub> Values for Compounds <b>20a</b>–<b>e</b> against SRC</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01502/20200221/images/medium/jm9b01502_0015.gif" alt="" id="gr11" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01502/20200221/images/medium/jm9b01502_0016.gif" alt="" id="gr12" /></img><div></div></div><div class="NLM_p">Collectively, our exploration around the hydrophobic pocket I provided a wide selectivity window over TAK1 and FGFR1, likely due to the different gatekeeper residues among these three kinases. The investigation of the solvent-exposed region and variation of the covalent warhead proved less fruitful in developing more potent and selective molecules. Thus, compound <b>15a</b> remained our lead compound, and we focused on it for further detailed mechanistic validation and preliminary pharmacological characterization.</div><div id="sec3_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Compound <b>15a</b> Selectively Modifies SRC Cys277</h3><div class="NLM_p">To confirm the covalent bond formation of compound <b>15a</b> with Cys277 on SRC, we first analyzed the protein by LC-MS after incubation with a 10-fold molar excess of <b>15a</b> for 2 h at 37 °C and observed a mass shift consistent with <b>15a</b>, indicating that <b>15a</b> forms a single covalent adduct with SRC (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A). To confirm the site of modification, labeled protein was digested with trypsin, and peptides were analyzed by capillary electrophoresis-MS/MS. A database search revealed an exclusive modification of SRC by <b>15a</b> at Cys277 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B). Taken together, our mass spectrometry-based analysis showed that <b>15a</b> forms a covalent adduct with SRC by reacting with a single cysteine (out of 9), Cys277, thus mirroring the reactivity of the parental, pan-CKI <b>SM1-71</b>.</div><figure id="fig2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01502/20200221/images/medium/jm9b01502_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01502/20200221/images/large/jm9b01502_0002.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Src is exclusively labeled by <b>15a</b> at cysteine 277. (A) Mass spectra (left) and zero-charge mass spectra (right) of SRC incubated (top) with DMSO or (bottom) with a 10-fold molar excess of <b>15a</b> for 2 h at 37 °C. Unlabeled protein peaks are indicated with purple glyphs, and <b>15a</b> modified protein peaks are highlighted with blue glyphs. (B) MS/MS spectrum of the SRC tryptic peptide <sup>273</sup>LGQGC*FGEVWMGTWNGTTR<sub>287</sub> modified with <b>15a</b> at Cys 277 (indicated with “*” in the sequence). Ions of type y are indicated with red glyphs. Inhibitor modified ions are labeled with green glyphs.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01502/20200221/images/large/jm9b01502_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01502&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec3_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Kinase Profiling Elucidates SRC-Selective Inhibition by <b>15a</b></h3><div class="NLM_p">To map the selectivity of our SRC lead CKI <b>15a</b>, we performed KinomeScan profiling, which is an active site-directed binding assay for 468 kinases. The result represents the remaining kinase activity in the presence of the test compound compared to the DMSO control. As shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>, while parental CKI <b>SM1-71</b> hits more than 100 kinases at 1 μM, compound <b>15a</b>, at the same concentration, is far more selective, inhibiting around 20 kinases (data shown in <a href="/doi/suppl/10.1021/acs.jmedchem.9b01502/suppl_file/jm9b01502_si_001.pdf" class="ext-link">Tables S1 and S2</a>; scores of >35%, as an indication of weak binding, are not shown). BTK, ABL, and most SFK family members such as HCK, BLK, LCK, and YES1 were the main off-targets as confirmed with low nanomolar biochemical IC<sub>50</sub>’s (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01502/suppl_file/jm9b01502_si_001.pdf" class="ext-link">Table S3</a>). We tested HCK, one of the off-targets of <b>15a</b>, in a cellular antiproliferation assay. <b>15a</b> did not show antiproliferative activity on HCK-dependent Ba/F3 cells (IC<sub>50</sub> > 2927 nM; <a href="/doi/suppl/10.1021/acs.jmedchem.9b01502/suppl_file/jm9b01502_si_001.pdf" class="ext-link">Table S4</a>) despite its biochemical IC<sub>50</sub> of ∼3 nM. Unlike SRC, HCK lacks the P-loop cysteine, suggesting that <b>15a</b> binds this kinase in a reversible manner and thus explains the disparity between cellular and biochemical data. Strikingly, MEK family kinases are not inhibited by compound <b>15a</b>, although they are major targets of <b>SM1-71</b>.</div><figure id="fig3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01502/20200221/images/medium/jm9b01502_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01502/20200221/images/large/jm9b01502_0003.jpeg" id="gr14" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. KinomeScan profiling for <b>SM1-71</b> and <b>15a</b>. (KinomeScan profiling for compound <b>SM1-71</b> was previously published.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> Reprinted with permission from ref <a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a>. Copyright 2017 Elsevier.) Compounds <b>SM1-71</b> and <b>15a</b> were screened at 1 μM against a panel of 468 human kinases. The results for the binding interactions are reported as “% Ctrl”, where larger circles indicate stronger hits. The top hits of <b>15a</b> are SRC, YES1, ABL1, HCK, LCK, LYN, BTK, BLK, and GAK, shown as blue circles in the kinome tree. (See the biochemical IC<sub>50</sub>’s in <a href="/doi/suppl/10.1021/acs.jmedchem.9b01502/suppl_file/jm9b01502_si_001.pdf" class="ext-link">Table S3</a>.) The selectivity score was defined as the ratio of the number of kinases inhibited to a specified percentage versus the total number of kinases. For this experiment, specified percent inhibition was set at 35%, resulting in S(35) values of 0.43 for <b>SM1-71</b> and 0.07 for compound <b>15a</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01502/20200221/images/large/jm9b01502_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01502&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec3_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Growth Inhibitory Screen Reveals Potent Antiproliferative Effects in NSCLC Cells</h3><div class="NLM_p">Next, we sought to evaluate the antiproliferative potency of <b>15a</b> using a growth inhibition assay. We chose cell lines where SRC kinase activity is known to contribute to growth and proliferation.<a onclick="showRef(event, 'ref41 ref42 ref43'); return false;" href="javascript:void(0);" class="ref ref41 ref42 ref43">(41−43)</a> These included MDA-MB-231 (triple negative breast cancer cell line) and H1975 and HCC827 (two nonsmall cell lung cancer lines, NSCLC). We compared the growth inhibitory potency of <b>15a</b> with that of its reversible (noncovalent) analog, <b>20f</b>, as well as <b>AZD0530</b> (saracatinib), a reversible (noncovalent) SRC inhibitor developed by AstraZeneca.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> Briefly, cells were treated with varying concentrations of the compounds for a period of 72 h, and cell viability was measured using the CellTiter-Glo assay. In order to account for confounding effects arising from varying doubling times across different cell lines, we implemented the recently described growth rate (GR) corrected calculations.<a onclick="showRef(event, 'ref40 ref45'); return false;" href="javascript:void(0);" class="ref ref40 ref45">(40,45)</a> This analysis results in GR<sub>50</sub> (similar to IC<sub>50</sub>) and GR<sub>max</sub> (similar to <i>E</i><sub>max</sub>) values, where a GR<sub>max</sub> value that lies within 0 and 1 implies partial growth arrest, a value of 0 indicates complete growth arrest, and a negative value implies cytotoxic effects (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01502/suppl_file/jm9b01502_si_003.xlsx" class="ext-link">Data S1</a>). Our results showed that <b>15a</b> induced strong growth inhibitory effects across all three cell lines with GR<sub>50</sub> values of 0.3, 0.5, and 0.3 μM, respectively (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A, <a href="/doi/suppl/10.1021/acs.jmedchem.9b01502/suppl_file/jm9b01502_si_001.pdf" class="ext-link">Table S5</a>). Furthermore, the compound also led to cytotoxic effects in HCC827 (GR<sub>max</sub> = −0.6) and MDA-MB-231 (GR<sub>max</sub> = −0.4) cells (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01502/suppl_file/jm9b01502_si_001.pdf" class="ext-link">Table S5</a>). The reversible analog <b>20f</b> and <b>AZD0530</b> also induced growth inhibition in these three cell lines but showed a weaker potency compared to <b>15a</b> (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B). These results suggest that covalent and selective SRC targeting in these cell lines can achieve potent inhibition of growth and proliferation.</div><figure id="fig4" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01502/20200221/images/medium/jm9b01502_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01502/20200221/images/large/jm9b01502_0004.jpeg" id="gr15" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Evaluating biological potency of SRC inhibitors in lung and breast cancer cell lines. (A) Comparing growth inhibitory potency of <b>15a</b> in H1975 (nonsmall cell lung cancer, NSCLC), HCC827 (NSCLC), and MDA-MB-231 (triple negative breast cancer, TNBC) cell lines. (B) Comparing growth inhibitory potency of <b>15a</b>, <b>20f</b>, and <b>AZD0530</b> across cell lines. Growth inhibition was carried out by treating cells with different concentrations of <b>15a</b>, <b>20f</b>, or <b>AZD0530</b> for 72 h, followed by analysis using the CellTiter-Glo reagent. Dose–response curves and the corresponding GR values were calculated using the online tool GR calculator (<a href="http://www.grcalculator.org/grcalculator/" class="extLink">http://www.grcalculator.org/grcalculator/</a>). Both the curves and the GR<sub>50</sub> values are representative of two independent experiments, each carried out in technical triplicate.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01502/20200221/images/large/jm9b01502_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01502&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec3_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Cellular Target Engagement and Signaling Inhibition by <b>15a</b></h3><div class="NLM_p">Having demonstrated binding of <b>15a</b> to SRC in biochemical assays, we next investigated whether <b>15a</b> could covalently bind SRC in cells. To assess cellular binding potency, we conducted a cellular target engagement analysis in two NSCLC cell lines, H1975 and HCC827, that showed strong growth inhibition by <b>15a</b> (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A). We further chose these cell lines because SRC is known to promote tumor progression in NSCLC.<a onclick="showRef(event, 'ref43 ref46'); return false;" href="javascript:void(0);" class="ref ref43 ref46">(43,46)</a> H1975 and HCC827 cells were treated with 1 μM of <b>15a</b> or <b>20f</b> or DMSO for 2 h, followed by drug removal, cellular extraction, and lysis. The obtained lysates were then incubated with 1 μM of TL13-68 (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01502/suppl_file/jm9b01502_si_001.pdf" class="ext-link">Figure S2</a>),<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> a biotin-tagged version of <b>SM1-71</b> that competes with <b>15a</b> for covalent SRC binding. Western blotting was used to evaluate cellular target engagement. Our results showed that SRC was covalently modified by <b>15a</b> in both cell lines, as indicated by the lack of SRC binding to TL13-68 and pulldown by streptavidin beads (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A). The reversible analog <b>20f</b> resulted in weaker SRC inhibitory effects, presumably due to displacement by TL13-68 (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A). We further analyzed whether <b>15a</b> can bind other cysteine-containing kinases such as MEK1/2 and ERK1/2, which have a DFG-1 cysteine. Our results showed <b>15a</b> did not engage MEK1/2 or ERK1/2 in cells. In addition, we also inspected whether TL13-86 can compete with <b>15a</b> for binding to YES1, a kinase sharing a similar P-loop cysteine. Although the YES1 expression level is low in both cells, a rescue of YES1 pulldown from the TL13-68 probe was detected, indicating YES1 is a potential covalent target for <b>15a</b>.</div><figure id="fig5" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01502/20200221/images/medium/jm9b01502_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01502/20200221/images/large/jm9b01502_0005.jpeg" id="gr16" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Target engagement and inhibition of SRC <i>in vitro</i> and <i>in vivo</i>. (A) Target engagement in H1975 and HCC827 cells treated with 1 μM of <b>15a</b> or <b>20f</b> for 2 h followed by overnight incubation with 1 μM TL13-68 (biotin compound), pulldown by streptavidin beads, and analysis using Western blotting. (B) Signaling inhibition in H1975 and HCC827 cells treated with 1 μM of <b>15a</b> or <b>20f</b> for 2 h followed by analysis using Western blotting. (C) Cellular target engagement and (D) signaling inhibition in H1975 cells treated with 1 μM of <b>15a</b>, <b>20f</b>, <b>AZD0530</b>, or DMSO for 2 h, followed by drug removal and replacement with drug-free medium. Cells were collected and lysed 0, 2, and 4 h postdrug washout and analyzed using Western blotting. For evaluating SRC binding (C), lysates were incubated with 1 μM of TL13-68 (biotin compound), followed by pulldown by streptavidin beads and visualization using Western blotting. Blots shown in (A–D) are representative of one of two biological replicates carried out. (Both blots for each experiment are shown in <a href="/doi/suppl/10.1021/acs.jmedchem.9b01502/suppl_file/jm9b01502_si_001.pdf" class="ext-link">Figure S1</a>.) (E, F) Pharmacodynamic analysis of (E) inhibition of p-SRC<sup>Y416</sup> signaling and (F) inhibition of SRC binding in spleen samples. Briefly, C57B6 mice (<i>n</i> = 2/condition) were treated with 5 mg/kg of <b>15a</b> or vehicle three times and sacrificed 2 or 4 h following the last dose. Spleen samples collected were ground up, lysed, and analyzed using Western blotting.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01502/20200221/images/large/jm9b01502_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01502&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Next, we sought to evaluate whether <b>15a</b> could lead to inhibition of SRC signaling in cells. Briefly, H1975 and HCC827 cells were treated with 1 μM of <b>15a</b> or <b>20f</b> or DMSO for 2 h, followed by extraction and lysis. Western blotting was used to measure the inhibition of SRC phosphorylation. Our results showed that, consistent with SRC binding, <b>15a</b> inhibited p-SRC<sup>Y416</sup> signaling in both H1975 and HCC827 cells (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B). Furthermore, <b>20f</b> was also capable of inhibiting p-SRC<sup>Y416</sup> signaling in cells (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B). Together, our results indicated that <b>15a</b> is capable of inducing potent SRC binding and inhibition of SRC signaling in cells.</div></div><div id="sec3_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Washout-Assays Reveal Superior Target Engagement by Covalent Inhibitor</h3><div class="NLM_p last">To assess the duration of inhibition of SRC in cells and further distinguish between covalent and noncovalent modes of binding, we conducted target engagement and signaling analyses under drug-washout conditions. H1975 cells were treated with 1 μM of <b>15a</b>, <b>20f</b>, <b>AZD0530</b>, or DMSO for 2 h, followed by drug washout and replacement of drug-containing medium with fresh drug-free medium. Cells were subsequently collected and lysed at 0, 2, and 4 h postwashout. For the cellular target engagement assay, the resulting lysate was incubated with 1 μM of TL13-68 (biotin compound) prior to visualization using Western blotting. For signaling analysis, lysates were directly processed to visualize Western blotting, without incubation with the biotin compound. Our results showed that <b>15a</b> was able to inhibit SRC up to 4 h postwashout, unlike the reversible compounds, <b>20f</b> and <b>AZD0530</b> (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>C). Similarly, <b>15a</b> induced potent and sustained inhibition of p-SRC<sup>Y416</sup> signaling up to 4 h postwashout, unlike <b>AZD0530</b> (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>D). Despite showing weak target engagement, <b>20f</b> led to moderate inhibition of p-SRC<sup>Y416</sup> signaling (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>D). Together, these results indicate that covalent inhibition of SRC by <b>15a</b> leads to potent and sustained target engagement, compared to reversible SRC compounds.</div></div><div id="sec3_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Pharmacodynamic Analysis in Mice Spleen Tissue Demonstrates <i>in Vivo</i> Target Engagement</h3><div class="NLM_p last">Next, we sought to evaluate target engagement and signaling analysis <i>in vivo</i>. We conducted a PK study for our lead compound <b>15a</b>. We dosed <b>15a</b> at 5 mg/kg IP in 5% DMSO/95% D5W in B6 mice. The compound exhibited a short half-life and high exposure (<i>T</i><sub>1/2</sub> = 1.29 h, AUC = 12 746.25 min·ng/mL). Since compound <b>15a</b> has a short half-life, we decided to treat mice with three doses to ensure full target engagement. We conducted pharmacodynamic analysis in C57B6 mice (<i>n</i> = 2 per condition) using spleen tissue samples collected 2 and 4 h after administering the last dose of compound <b>15a</b>. A total of 3 doses (5 mg/kg) was administered to the mice prior to sacrificing and collecting tissue samples. Spleen tissue samples were ground up and lysed, and Western blotting was used to measure target engagement and signaling changes. Our results showed that <b>15a</b> led to inhibition of p-SRC<sup>Y416</sup> at 2 and 4 h postdosing, compared to the vehicle controls (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>E). Similarly, <b>15a</b> also demonstrated SRC binding and inhibition at both 2 and 4 h postdosing compared to the vehicle controls (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>F). These results suggest that <b>15a</b> is able to inhibit SRC for an extended duration, likely due to its ability to covalently bind the target.</div></div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i18">Discussion and Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_13581" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_13581" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Promiscuous kinase inhibitors, or drugs with polypharmacology, have proven beneficial in treating cancers (e.g., sunitinib, sorafenib, and dasatinib), and they have also been useful as pharmacological probes to assess dependence on different signaling pathways.<a onclick="showRef(event, 'ref47 ref48 ref49 ref50'); return false;" href="javascript:void(0);" class="ref ref47 ref48 ref49 ref50">(47−50)</a> In the present study, we used our recently characterized pan-kinase covalent inhibitor <b>SM1-71</b> as a chemical starting point for designing a selective, covalent SRC inhibitor. <b>SM1-71</b> bears a 2,4-diaminopyrimidine, a prevalent scaffold among kinase inhibitors, and the presence of a chlorine at the C5 position allows binding to many kinases regardless of the gatekeeper residue. <b>SM1-71</b> primarily reacts with cysteines that are positioned before the DFG motif (e.g., TAK1, MEK, ERK) or on the P-loop (e.g., SRC, FGFR1–4) through rotation of the acrylamide warhead.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> In designing <b>15a</b>, we explored tuning the trajectory of the covalent warhead and the differences between the hydrophobic pockets of various <b>SM1-71</b> targets. By inspecting the gatekeeper residues of TAK1, SRC, and FGFR1, we reasoned that both SRC and FGFR1 would tolerate a bulkier phenyl group at the C5 position of the pyrimidine but not TAK1, which has a larger methionine gatekeeper. Compound <b>15a</b>, with a phenyl group linked through an amide bond at the C5 position, preserves potent binding to SRC while abolishing activity against TAK1 and FGFR1. We propose that the selectivity between FGFR1 and SRC is due to a hydrogen bond interaction between the amide and the threonine gatekeeper on SRC, which is disrupted by the valine on FGFR, a more hydrophobic residue. Due to the high sequence, and likely structure, similarity, other SRC family kinases (HCK, BLK, LCK, and YES1) have been indicated as potential off-targets for <b>15a</b> on the basis of a low nanomolar biochemical IC<sub>50</sub> value. YES1 and FGR are the only proteins in this family that have a P-loop cysteine and may potentially be covalent targets, while other kinases in this group are likely biochemically inhibited through noncovalent, reversible binding. Further optimization to remove these off-targets is the subject of a future medicinal chemistry effort.</div><div class="NLM_p last">Irreversible SRC inhibition by compound <b>15a</b> led to sustained signaling inhibition, target engagement, and more potent antiproliferative effects across cancer cell lines. Although <b>15a</b> showed a half-life of 5.2 min in mouse liver microsomes (data not shown), the PD study demonstrated an extended duration of SRC signaling blockade. This suggests that the irreversible SRC inhibitor <b>15a</b> could have therapeutic potential and potential differentiation from noncovalent SRC inhibitors.</div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i19">Materials and Methods</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_86455" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_86455" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Chemistry</h3><div class="NLM_p">Unless otherwise noted, reagents and solvents were obtained from commercial suppliers and were used without further purification. <sup>1</sup>H NMR spectra were recorded on a 500 MHz Bruker A500, and chemical shifts are reported in parts per million (ppm, δ) downfield from tetramethylsilane (TMS). Coupling constants (<i>J</i>) are reported in Hz. Spin multiplicities are described as s (singlet), br (broad singlet), d (doublet), t (triplet), q (quartet), and m (multiplet). Mass spectra were obtained on a Waters Micromass ZQ instrument. Preparative HPLC was performed on a Waters Sunfire C18 column (19 × 50 mm, 5 μM) using a gradient of 15–95% methanol in water or acetonitrile in water containing 0.05% trifluoroacetic acid (TFA) over 22 min (28 min run time) or 35 min (45 min run time) at a flow rate of 20 mL/min. The purity of all the final compounds is ≥95%, and it is measured by analytical reverse-HPLC using a gradient of 15–85% acetonitrile in water containing TFA over 2.5 min at a flow rate of 2 mL/min.</div><div id="sec5_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> <i>N</i><sup><i>1</i></sup>-(5-Bromo-2-chloropyrimidin-4-yl)benzene-1,2-diamine (<b>2</b>)</h4><div class="NLM_p last">To the solution of 5-bromo-2,4-dichloropyrimidine (1.0 g, 4.39 mmol) and benzene-1,2-diamine (474 mg, 4.39 mmol) in <i>n</i>-BuOH (15 mL) was added DIEA (1.13 g, 8.78 mmol) at room temperature. The reaction mixture was heated up to 100 °C for 2 h. The solvent was evaporated under vacuum, and the residue was purified with flash chromatography to yield the product (1.10 g, 3.68 mmol, 84%). (M + H)<sup>+</sup> calculated 298.96; found 298.70, 300.70.</div></div><div id="sec5_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> <i>N</i><sup><i>4</i></sup>-(2-Aminophenyl)-5-bromo-<i>N</i><sup><i>2</i></sup>-(4-(4-methylpiperazin-1-yl)phenyl)pyrimidine-2,4-diamine (<b>3</b>)</h4><div class="NLM_p last">To the solution of intermediate <b>2</b> (550 mg, 1.84 mmol) and 4-(4-methylpiperazin-1-yl)aniline (352 mg, 1.84 mmol) in <i>s</i>-BuOH (5 mL) was added TFA (314 mg, 2.76 mmol) at room temperature. The reaction mixture was heated up to 100 °C for 2 h. The mixture was cooled to room temperature and concentrated to get the residue. The residue was purified with flash chromatography to obtain the product (760 mg, 1.67 mmol, 91%). (M + H)<sup>+</sup> calculated 454.13; found 454.19, 456.19.</div></div><div id="sec5_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> <i>N</i><sup><i>4</i></sup>-(2-Aminophenyl)-<i>N</i><sup><i>2</i></sup>-(4-(4-methylpiperazin-1-yl)phenyl)-5-(thiophen-2-yl)pyrimidine-2,4-diamine (<b>4a</b>)</h4><div class="NLM_p last">To the solution of intermediate <b>3</b> (100 mg, 0.22 mmol), thiophen-2-ylboronic acid (42 mg, 0.33 mmol), and K<sub>3</sub>PO<sub>4</sub> (279 mg, 1.32 mmol) in 1,4-dioxane/H<sub>2</sub>O (2 mL/0.5 mL) was added Pd(dppf)Cl<sub>2</sub>·DCM (18 mg, 0.022 mmol) at room temperature under a nitrogen atmosphere. The reaction mixture was heated up to 80 °C for 3 h. The mixture was extracted with ethyl acetate, and the organic layer was concentrated to get the residue. The residue was purified with chromatography to yield the product (40 mg, 0.087 mmol, 40%). (M + H)<sup>+</sup> calculated 458.20; found 457.88.</div></div><div id="sec5_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> <i>N</i>-(2-((2-((4-(4-Methylpiperazin-1-yl)phenyl)amino)-5-(thiophen-2-yl)pyrimidin-4-yl)amino)phenyl)acrylamide (<b>5a</b>)</h4><div class="NLM_p last">To the solution of intermediate <b>4a</b> (40 mg, 0.087 mmol) in NaHCO<sub>3</sub> (sat. aq., 1 mL) and THF (1 mL) was added acryloyl chloride (8.7 mg, 0.096 mmol) slowly under an ice bath. The reaction was extracted with ethyl acetate after all the intermediate <b>4a</b> was consumed. The organic layer was concentrated to get residue. The residue was purified by HPLC to afford the title compound (9.1 mg, 0.014 mmol, 36%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.18 (s, 1H), 9.80 (s, 2H), 8.53 (s, 1H), 8.03 (s, 1H), 7.84 (d, <i>J</i> = 7.9 Hz, 1H), 7.69 (dd, <i>J</i> = 4.9, 1.4 Hz, 1H), 7.40 (d, <i>J</i> = 8.5 Hz, 2H), 7.38–7.31 (m, 2H), 7.30–7.25 (m, 1H), 7.23–7.17 (m, 2H), 6.87 (d, <i>J</i> = 8.6 Hz, 2H), 6.41 (dd, <i>J</i> = 17.0, 10.2 Hz, 1H), 6.15 (dd, <i>J</i> = 17.0, 2.0 Hz, 1H), 5.78 (dd, <i>J</i> = 10.1, 2.0 Hz, 1H), 3.75 (d, <i>J</i> = 13.2 Hz, 2H), 3.53 (d, <i>J</i> = 12.1 Hz, 2H), 3.18 (s, 2H), 2.96–2.84 (m, 5H). (M + H)<sup>+</sup> calculated 512.22; found 511.88.</div></div><div id="sec5_1_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> <i>N</i>-(2-((5-(Furan-3-yl)-2-((4-(4-methylpiperazin-1-yl)phenyl)amino)pyrimidin-4-yl)amino)phenyl)acrylamide (<b>5b</b>)</h4><div class="NLM_p last">Compound <b>5b</b> was synthesized via the same route as compound <b>5a</b>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.16 (s, 1H), 9.94 (s, 1H), 9.81 (s, 1H), 8.57 (s, 1H), 8.02 (s, 1H), 7.94 (s, 1H), 7.85 (t, <i>J</i> = 1.7 Hz, 1H), 7.78 (d, <i>J</i> = 7.9 Hz, 1H), 7.44 (dd, <i>J</i> = 7.8, 1.7 Hz, 1H), 7.39–7.31 (m, 3H), 7.29 (td, <i>J</i> = 7.6, 1.6 Hz, 1H), 6.86 (d, <i>J</i> = 8.6 Hz, 2H), 6.72 (d, <i>J</i> = 1.8 Hz, 1H), 6.45 (dd, <i>J</i> = 16.9, 10.2 Hz, 1H), 6.20 (dd, <i>J</i> = 17.0, 2.0 Hz, 1H), 5.79 (dd, <i>J</i> = 10.1, 2.0 Hz, 1H), 3.74 (d, <i>J</i> = 13.1 Hz, 2H), 3.53 (d, <i>J</i> = 12.2 Hz, 2H), 3.17 (s, 2H), 2.91 (d, <i>J</i> = 11.5 Hz, 2H), 2.88 (s, 3H). (M + H)<sup>+</sup> calculated 496.24; found 496.38.</div></div><div id="sec5_1_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> <i>N</i>-(2-((5-(6-Methoxypyridin-3-yl)-2-((4-(4-methylpiperazin-1-yl)phenyl)amino)pyrimidin-4-yl)amino)phenyl)acrylamide (<b>5c</b>)</h4><div class="NLM_p last">Compound <b>5c</b> was synthesized via the same route as compound <b>5a</b>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.04 (s, 1H), 9.96 (s, 1H), 9.72 (s, 1H), 8.62 (s, 1H), 8.13 (d, <i>J</i> = 2.4 Hz, 1H), 7.86 (s, 1H), 7.73–7.66 (m, 2H), 7.35 (dd, <i>J</i> = 8.0, 1.7 Hz, 1H), 7.30 (d, <i>J</i> = 8.4 Hz, 2H), 7.25 (td, <i>J</i> = 7.7, 1.7 Hz, 1H), 7.20 (td, <i>J</i> = 7.6, 1.6 Hz, 1H), 6.85 (d, <i>J</i> = 8.6 Hz, 1H), 6.81 (d, <i>J</i> = 8.6 Hz, 2H), 6.33 (dd, <i>J</i> = 17.0, 10.2 Hz, 1H), 6.00 (dd, <i>J</i> = 17.0, 2.0 Hz, 1H), 5.70 (dd, <i>J</i> = 10.1, 2.0 Hz, 1H), 3.85 (s, 3H), 3.68 (d, <i>J</i> = 13.3 Hz, 2H), 3.46 (d, <i>J</i> = 12.2 Hz, 2H), 3.10 (d, <i>J</i> = 10.9 Hz, 2H), 2.84 (d, <i>J</i> = 12.2 Hz, 2H), 2.80 (s, 3H). (M + H)<sup>+</sup> calculated 537.26; found 537.39.</div></div><div id="sec5_1_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> <i>N</i>-(2-((5-(4-Acetylphenyl)-2-((4-(4-methylpiperazin-1-yl)phenyl)amino)pyrimidin-4-yl)amino)phenyl)acrylamide (<b>5d</b>)</h4><div class="NLM_p last">Compound <b>5d</b> was synthesized via the same route as compound <b>5a</b>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.13 (s, 1H), 9.85 (s, 1H) 9.82 (s, 1H), 8.50 (s, 1H), 8.05 (d, <i>J</i> = 8.2 Hz, 2H), 8.01 (s, 1H), 7.82 (d, <i>J</i> = 8.0 Hz, 1H), 7.63–7.58 (m, 2H), 7.44–7.36 (m, 3H), 7.35–7.30 (m, 1H), 7.29–7.21 (m, 1H), 6.87 (d, <i>J</i> = 8.6 Hz, 2H), 6.40 (dd, <i>J</i> = 16.9, 10.2 Hz, 1H), 6.01 (dd, <i>J</i> = 17.1, 2.0 Hz, 1H), 5.75 (dd, <i>J</i> = 10.1, 2.0 Hz, 1H), 3.75 (d, <i>J</i> = 13.2 Hz, 2H), 3.54 (d, <i>J</i> = 12.1 Hz, 2H), 3.17 (d, <i>J</i> = 10.7 Hz, 2H), 2.92 (d, <i>J</i> = 12.4 Hz, 2H), 2.88 (s, 3H), 2.64 (s, 3H). (M + H)<sup>+</sup> calculated 548.27; found 548.49.</div></div><div id="sec5_1_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> <i>N</i>-(2-((2-((4-(4-methylpiperazin-1-yl)phenyl)amino)-5-(4-phenoxyphenyl)pyrimidin-4-yl)amino)phenyl)acrylamide (<b>5e</b>)</h4><div class="NLM_p last">Compound <b>5e</b> was synthesized via the same route as compound <b>5a</b>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.10 (s, 1H), 9.77 (s, 2H), 8.41 (s, 1H), 7.86 (s, 1H), 7.75 (d, <i>J</i> = 7.8 Hz, 1H), 7.39–7.31 (m, 6H), 7.30–7.23 (m, 2H), 7.22–7.15 (m, 1H), 7.15–7.09 (m, 1H), 7.07–7.00 (m, 4H), 6.81 (d, <i>J</i> = 8.5 Hz, 2H), 6.32 (dd, <i>J</i> = 17.0, 10.2 Hz, 1H), 6.01 (dd, <i>J</i> = 17.0, 2.0 Hz, 1H), 5.65 (dd, <i>J</i> = 10.1, 2.0 Hz, 1H), 3.68 (d, <i>J</i> = 12.9 Hz, 2H), 3.46 (d, <i>J</i> = 12.2 Hz, 2H), 3.17–3.03 (m, 2H), 2.89–2.77 (m, 5H). (M + H)<sup>+</sup> calculated 598.29; found 598.59.</div></div><div id="sec5_1_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> 2,4-Dichloro-5-((4-methylbenzyl)oxy)pyrimidine (<b>7a</b>)</h4><div class="NLM_p last">To the solution of 2,4-dichloropyrimidin-5-ol (100 mg, 0.61 mmol), NaI (10 mg, 0.061 mmol), and K<sub>2</sub>CO<sub>3</sub> (210 mg, 1.52 mmol) in acetone (3 mL) was added 4-methyl benzylbromide (169 mg, 0.91 mmol) at room temperature. All the starting material was consumed overnight. The mixture was evaporated under vacuum, and the residue was purified with flash chromatography to yield the product (150 mg, 0.56 mmol, 91%). (M + H)<sup>+</sup> calculated 269.02; found 269.17.</div></div><div id="sec5_1_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> <i>N</i><sup><i>1</i></sup>-(2-Chloro-5-((4-methylbenzyl)oxy)pyrimidin-4-yl)benzene-1,2-diamine (<b>8a</b>)</h4><div class="NLM_p last">To the solution of intermediate <b>7a</b> (150 mg, 0.56 mmol) and benzene-1,2-diamine (60 mg, 0.56 mmol) in <i>n</i>-BuOH (3 mL) was added DIEA (145 mg, 1.12 mmol) at room temperature. The mixture was then heated up to 110 °C for 2 h. The mixture was cooled to room temperature and concentrated under vacuum. The residue was purified with flash chromatography to yield the product (170 mg, 0.50 mmol, 89%). (M + H)<sup>+</sup> calculated 341.11; found 341.27.</div></div><div id="sec5_1_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> <i>N</i><sup><i>4</i></sup>-(2-Aminophenyl)-5-((4-methylbenzyl)oxy)-<i>N</i><sup><i>2</i></sup>-(4-(4-methylpiperazin-1-yl)phenyl)pyrimidine-2,4-diamine (<b>9a</b>)</h4><div class="NLM_p last">To the solution of intermediate <b>8a</b> (170 mg, 0.50 mmol) and 4-(4-methylpiperazin-1-yl)aniline (96 mg, 0.50 mmol) in <i>s</i>-BuOH (3 mL) was added TFA (85 mg, 0.75 mmol) at room temperature. The mixture was then heated up to 90 °C for 2 h. The mixture was evaporated under vacuum, and the residue was purified with flash chromatography to yield the product (70 mg, 0.14 mmol, 28%). (M + H)<sup>+</sup> calculated 469.27; found 469.48.</div></div><div id="sec5_1_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> <i>N</i>-(2-((5-((4-Methylbenzyl)oxy)-2-((4-(4-methylpiperazin-1-yl)phenyl)amino)pyrimidin-4-yl)amino)phenyl)acrylamide (<b>10a</b>)</h4><div class="NLM_p last">To the solution of intermediate <b>9a</b> (70 mg, 0.14 mmol) in NaHCO<sub>3</sub> (sat. aq., 2 mL) and THF (2 mL) was added acryloyl chloride (14 mg, 0.15 mmol) under an ice bath. The reaction was extracted with ethyl acetate after all <b>9a</b> was consumed. The organic layer was concentrated under vacuum, and the residue was purified by HPLC to obtain the title compound (34 mg, 0.052 mmol, 37%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.24 (s, 1H), 9.81 (s, 2H), 9.24 (s, 1H), 7.88–7.66 (m, 1H), 7.39–7.32 (m, 2H), 7.31–7.24 (m, 2H), 7.24–7.19 (m, 1H), 7.18–7.12 (m, 3H), 7.06 (s, 1H), 6.96 (s, 1H), 6.85–6.77 (m, 2H), 6.42 (dd, <i>J</i> = 17.0, 10.2 Hz, 1H), 6.16 (dd, <i>J</i> = 17.0, 1.9 Hz, 1H), 5.74 (dd, <i>J</i> = 10.2, 1.9 Hz, 1H), 5.05 (s, 2H), 3.73–3.62 (m, 2H), 3.51–3.39 (m, 2H), 3.15–3.04 (m, 2H), 2.88–2.76 (m, 5H), 2.26 (s, 3H). (M + H)<sup>+</sup> calculated 550.29; found 550.29.</div></div><div id="sec5_1_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> <i>N</i>-(2-((5-((2-Chloro-6-methylbenzyl)oxy)-2-((4-(4-methylpiperazin-1-yl)phenyl)amino)pyrimidin-4-yl)amino)phenyl)acrylamide (<b>10b</b>)</h4><div class="NLM_p last">Compound <b>10b</b> was synthesized via the same route as compound <b>10a</b>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.14 (s, 1H), 9.96 (s, 1H), 9.77 (s, 1H), 9.17 (s, 1H), 7.82 (s, 1H), 7.78 (d, <i>J</i> = 8.0 Hz, 1H), 7.33–7.25 (m, 5H), 7.25 (d, <i>J</i> = 7.8 Hz, 1H), 7.23–7.14 (m, 2H), 6.84 (d, <i>J</i> = 8.7 Hz, 2H), 6.28 (dd, <i>J</i> = 17.0, 10.2 Hz, 1H), 5.81 (dd, <i>J</i> = 16.9, 1.9 Hz, 1H), 5.60 (dd, <i>J</i> = 10.1, 1.9 Hz, 1H), 5.17 (s, 2H), 3.68 (d, <i>J</i> = 13.1 Hz, 2H), 3.46 (d, <i>J</i> = 12.2 Hz, 2H), 3.10 (s, 2H), 2.84 (d, <i>J</i> = 12.8 Hz, 2H), 2.80 (s, 3H), 2.34 (s, 3H). (M + H)<sup>+</sup> calculated 584.25; found 584.89.</div></div><div id="sec5_1_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> <i>N</i>-(2-((2-((4-(4-Methylpiperazin-1-yl)phenyl)amino)-5-((4-phenoxybenzyl)oxy)pyrimidin-4-yl)amino)phenyl)acrylamide (<b>10c</b>)</h4><div class="NLM_p last">Compound <b>10c</b> was synthesized via the same route as compound <b>10a</b>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.24 (s, 1H), 9.88 (s, 2H), 9.33 (s, 1H), 7.79–7.73 (m, 2H), 7.51–7.46 (m, 2H), 7.40–7.31 (m, 3H), 7.28 (ddd, <i>J</i> = 11.2, 6.3, 2.4 Hz, 3H), 7.22 (td, <i>J</i> = 7.6, 1.6 Hz, 1H), 7.11 (t, <i>J</i> = 7.5 Hz, 1H), 7.02–6.93 (m, 4H), 6.86–6.79 (m, 2H), 6.41 (dd, <i>J</i> = 16.9, 10.2 Hz, 1H), 6.12 (dd, <i>J</i> = 17.0, 1.9 Hz, 1H), 5.68 (dd, <i>J</i> = 10.1, 2.0 Hz, 1H), 5.08 (s, 2H), 3.71–3.64 (m, 2H), 3.45 (d, <i>J</i> = 12.0 Hz, 2H), 3.09 (s, 2H), 2.85 (d, <i>J</i> = 12.5 Hz, 2H), 2.80 (s, 3H). (M + H)<sup>+</sup> calculated 628.30; found 628.50.</div></div><div id="sec5_1_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> 2,4-Dichloro-<i>N</i>-(2-chloro-6-methylphenyl)pyrimidine-5-carboxamide (<b>12a</b>)</h4><div class="NLM_p last">To the solution of 2-chloro-6-methylaniline (67 mg, 0.47 mmol) in THF (1 mL) was added 2,4-dichloropyrimidine-5-carbonyl chloride (100 mg, 0.47 mmol) at room temperature. The starting material was nearly consumed overnight. The mixture was evaporated under vacuum, and the residue was purified with flash chromatography to yield the product (120 mg, 0.38 mmol, 81%). (M + H)<sup>+</sup> calculated 315.97; found 316.07.</div></div><div id="sec5_1_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> 4-((2-Aminophenyl)amino)-2-chloro-<i>N</i>-(2-chloro-6-methylphenyl)pyrimidine-5-carboxamide (<b>13a</b>)</h4><div class="NLM_p last">To the solution of intermediate <b>12a</b> (120 mg, 0.38 mmol) and benzene-1,2-diamine (41 mg, 0.38 mmol) in <i>n</i>-BuOH (2 mL) was added DIEA at room temperature. The mixture was then heated up to 110 °C for 2 h. The mixture was cooled to room temperature and concentrated under vacuum. The residue was purified with flash chromatography to yield the product (147 mg, 0.38 mmol, quant.). (M + H)<sup>+</sup> calculated 388.07; found 388.27.</div></div><div id="sec5_1_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> 4-((2-Aminophenyl)amino)-<i>N</i>-(2-chloro-6-methylphenyl)-2-((4-(4-methylpiperazin-1-yl)phenyl)amino)pyrimidine-5-carboxamide (<b>14a</b>)</h4><div class="NLM_p last">To the solution of intermediate <b>13a</b> (147 mg, 0.38 mmol) and 4-(4-methylpiperazin-1-yl)aniline (73 mg, 0.38 mmol) in <i>s</i>-BuOH (2 mL) was added TFA (65 mg, 0.57 mmol) at room temperature. The mixture was then heated up to 90 °C for 2 h. The mixture was cooled down to room temperature and concentrated under vacuum. The residue was purified with flash chromatography to yield the product (120 mg, 0.22 mmol, 58%). (M + H)<sup>+</sup> calculated 543.23; found 543.29.</div></div><div id="sec5_1_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> 4-((2-Acrylamidophenyl)amino)-<i>N</i>-(2-chloro-6-methylphenyl)-2-((4-(4-methylpiperazin-1-yl)phenyl)amino)pyrimidine-5-carboxamide (<b>15a</b>)</h4><div class="NLM_p last">To the solution of intermediate <b>14a</b> (20 mg, 0.037 mmol) in NaHCO<sub>3</sub> (sat. aq., 1 mL) and THF (1 mL) was added acryloyl chloride (4 mg, 0.044 mmol) under an ice bath. The reaction was extracted with ethyl acetate after all intermediate <b>14a</b> was consumed. The organic layer was concentrated, and the residue was purified with HPLC to obtain the title compound (3.6 mg, 0.0050 mmol, 14%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.75 (s, 1H), 9.83 (s, 1H), 9.69 (s, 3H), 8.82 (s, 1H), 8.04 (s, 1H), 7.46 (d, <i>J</i> = 8.3 Hz, 2H), 7.36–7.26 (m, 2H), 7.26–7.16 (m, 3H), 7.09 (t, <i>J</i> = 7.5 Hz, 1H), 6.83 (d, <i>J</i> = 8.5 Hz, 2H), 6.27 (dd, <i>J</i> = 17.1, 10.3 Hz, 1H), 6.06 (dd, <i>J</i> = 17.1, 2.0 Hz, 1H), 5.56 (dd, <i>J</i> = 10.2, 2.0 Hz, 1H), 3.68 (d, <i>J</i> = 13.1 Hz, 2H), 3.46 (d, <i>J</i> = 12.2 Hz, 2H), 3.11 (d, <i>J</i> = 11.2 Hz, 2H), 2.82–2.77 (m, 5H), 2.16 (s, 3H). (M + H)<sup>+</sup> calculated 597.24; found 597.39.</div></div><div id="sec5_1_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> 4-((2-Acrylamidophenyl)amino)-<i>N</i>-(2,4-dichloro-6-methylphenyl)-2-((4-(4-methylpiperazin-1-yl)phenyl)amino)pyrimidine-5-carboxamide (<b>15b</b>)</h4><div class="NLM_p last">Compound <b>15b</b> was synthesized via the same route as compound 1<b>5a</b>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.69 (s, 1H), 9.85 (s, 1H), 9.70 (s, 1H), 9.65 (s, 2H), 8.82 (s, 1H), 8.00 (s, 1H), 7.51 (dd, <i>J</i> = 9.8, 2.4 Hz, 1H), 7.45 (s, 2H), 7.37 (dd, <i>J</i> = 11.0, 2.4 Hz, 1H), 7.29 (d, <i>J</i> = 7.8 Hz, 1H), 7.20 (d, <i>J</i> = 7.6 Hz, 1H), 7.09 (t, <i>J</i> = 7.6 Hz, 1H), 6.83 (d, <i>J</i> = 8.5 Hz, 2H), 6.28 (dd, <i>J</i> = 17.1, 10.3 Hz, 1H), 6.06 (dd, <i>J</i> = 17.1, 2.0 Hz, 1H), 5.57 (dd, <i>J</i> = 10.2, 2.0 Hz, 1H), 3.68 (d, <i>J</i> = 13.2 Hz, 2H), 3.46 (d, <i>J</i> = 12.0 Hz, 2H), 3.11 (d, <i>J</i> = 12.8 Hz, 2H), 2.85 (d, <i>J</i> = 12.6 Hz, 2H), 2.80 (s, 3H), 2.16 (s, 3H). (M + H)<sup>+</sup> calculated 611.26; found 610.79.</div></div><div id="sec5_1_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> 4-((2-Acrylamidophenyl)amino)-<i>N</i>-(2-chloro-4,6-dimethylphenyl)-2-((4-(4-methylpiperazin-1-yl)phenyl)amino)pyrimidine-5-carboxamide (<b>15c</b>)</h4><div class="NLM_p last">Compound <b>15c</b> was synthesized via the same route as compound <b>15a</b>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.78 (s, 1H), 9.75 (s, 1H), 9.69 (s, 3H), 8.82 (s, 1H), 8.00 (s, 1H), 7.48–7.41 (m, 2H), 7.29 (d, <i>J</i> = 7.9 Hz, 1H), 7.23–7.13 (m, 2H), 7.09 (t, <i>J</i> = 7.7 Hz, 1H), 7.04 (d, <i>J</i> = 11.2 Hz, 1H), 6.83 (d, <i>J</i> = 8.5 Hz, 2H), 6.28 (dd, <i>J</i> = 17.1, 10.2 Hz, 1H), 6.06 (dd, <i>J</i> = 17.1, 1.9 Hz, 1H), 5.57 (dd, <i>J</i> = 10.2, 2.0 Hz, 1H), 3.68 (d, <i>J</i> = 13.3 Hz, 2H), 3.46 (d, <i>J</i> = 12.0 Hz, 2H), 3.11 (d, <i>J</i> = 10.8 Hz, 2H), 2.88–2.78 (m, 5H), 2.23 (d, <i>J</i> = 3.3 Hz, 3H), 2.11 (s, 3H). (M + H)<sup>+</sup> calculated 631.20; found 630.70.</div></div><div id="sec5_1_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> 4-((2-Acrylamidophenyl)amino)-<i>N</i>-(2-chloro-4-cyano-6-methylphenyl)-2-((4-(4-methylpiperazin-1-yl)phenyl)amino)pyrimidine-5-carboxamide (<b>15d</b>)</h4><div class="NLM_p last">Compound <b>15d</b> was synthesized via the same route as compound <b>15a</b>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.61 (s, 1H), 10.05 (s, 1H), 9.70 (s, 1H), 9.68 (s, 2H), 8.82 (s, 1H), 7.96 (s, 2H), 7.76 (s, 1H), 7.48–7.43 (m, 2H), 7.29 (d, <i>J</i> = 7.9 Hz, 1H), 7.20 (t, <i>J</i> = 7.5 Hz, 1H), 7.10 (t, <i>J</i> = 7.6 Hz, 1H), 6.82 (d, <i>J</i> = 8.1 Hz, 2H), 6.27 (dd, <i>J</i> = 17.1, 10.3 Hz, 1H), 6.05 (dd, <i>J</i> = 17.1, 2.0 Hz, 1H), 5.57 (dd, <i>J</i> = 10.2, 2.0 Hz, 1H), 3.68 (d, <i>J</i> = 13.2 Hz, 2H), 3.46 (d, <i>J</i> = 12.0 Hz, 2H), 3.11 (q, <i>J</i> = 11.0 Hz, 2H), 2.88–2.78 (m, 5H), 2.21 (s, 3H). (M + H)<sup>+</sup> calculated 622.24; found 621.89.</div></div><div id="sec5_1_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> 4-((2-Acrylamidophenyl)amino)-<i>N</i>-(3-chloro-5-methylphenyl)-2-((4-(4-methylpiperazin-1-yl)phenyl)amino)pyrimidine-5-carboxamide (<b>15e</b>)</h4><div class="NLM_p last">Compound <b>15e</b> was synthesized via the same route as compound <b>15a</b>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.86 (s, 1H), 10.08 (s, 1H), 9.78 (s, 1H), 9.68 (s, 2H), 8.75 (s, 1H), 8.12 (s, 1H), 7.68–7.62 (m, 1H), 7.47–7.43 (m, 2H), 7.39 (s, 1H), 7.31 (d, <i>J</i> = 7.9 Hz, 1H), 7.20 (t, <i>J</i> = 7.8 Hz, 1H), 7.08 (t, <i>J</i> = 7.7 Hz, 1H), 6.94 (s, 1H), 6.86 (d, <i>J</i> = 8.5 Hz, 2H), 6.44 (dd, <i>J</i> = 17.1, 10.3 Hz, 1H), 6.18 (dd, <i>J</i> = 17.1, 2.0 Hz, 1H), 5.68 (dd, <i>J</i> = 10.2, 2.0 Hz, 1H), 3.70 (d, <i>J</i> = 13.0 Hz, 2H), 3.47 (d, <i>J</i> = 12.0 Hz, 2H), 3.10 (t, <i>J</i> = 10.9 Hz, 2H), 2.86 (d, <i>J</i> = 12.7 Hz, 2H), 2.81 (s, 3H), 2.24 (s, 3H). (M + H)<sup>+</sup> calculated 597.24; found 596.89.</div></div><div id="sec5_1_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> 4-((2-Acrylamidophenyl)amino)-<i>N</i>-(4,5-dimethoxy-2-methylphenyl)-2-((4-(4-methylpiperazin-1-yl)phenyl)amino)pyrimidine-5-carboxamide (<b>15f</b>)</h4><div class="NLM_p last">Compound <b>15f</b> was synthesized via the same route as compound <b>15a</b>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.92 (s, 1H), 9.73 (s, 1H), 9.69 (s, 1H), 9.60 (s, 1H), 9.56 (s, 1H), 8.76 (s, 1H), 8.02 (s, 1H), 7.47 (s, 2H), 7.32 (d, <i>J</i> = 7.9 Hz, 1H), 7.19 (t, <i>J</i> = 7.5 Hz, 1H), 7.07 (t, <i>J</i> = 7.5 Hz, 1H), 6.86–6.80 (m, 3H), 6.78 (s, 1H), 6.34 (dd, <i>J</i> = 17.1, 10.3 Hz, 1H), 6.10 (dd, <i>J</i> = 17.1, 2.0 Hz, 1H), 5.59 (dd, <i>J</i> = 10.3, 2.0 Hz, 1H), 3.69 (s, 3H), 3.67 (s, 2H), 3.65 (s, 3H), 3.46 (d, <i>J</i> = 12.1 Hz, 2H), 3.11 (d, <i>J</i> = 10.6 Hz, 2H), 2.85 (d, <i>J</i> = 12.6 Hz, 2H), 2.80 (d, <i>J</i> = 2.9 Hz, 3H), 2.08 (s, 3H). (M + H)<sup>+</sup> calculated 623.30; found 622.99.</div></div><div id="sec5_1_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> 4-((2-Acrylamidophenyl)amino)-<i>N</i>-(2,6-dichloro-3,5-dimethoxyphenyl)-2-((4-(4-methylpiperazin-1-yl)phenyl)amino)pyrimidine-5-carboxamide (<b>15g</b>)</h4><div class="NLM_p last">Compound <b>15g</b> was synthesized via the same route as compound <b>15a</b>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.59 (s, 1H), 10.01 (s, 1H), 9.67 (s, 3H), 8.81 (s, 1H), 7.93 (s, 1H), 7.44 (d, <i>J</i> = 8.5 Hz, 2H), 7.31 (d, <i>J</i> = 7.9 Hz, 1H), 7.20 (t, <i>J</i> = 7.8 Hz, 1H), 7.10 (t, <i>J</i> = 7.6 Hz, 1H), 6.88 (s, 1H), 6.83–6.78 (m, 2H), 6.27 (dd, <i>J</i> = 17.1, 10.2 Hz, 1H), 6.05 (dd, <i>J</i> = 17.1, 1.9 Hz, 1H), 5.55 (dd, <i>J</i> = 10.2, 2.0 Hz, 1H), 3.89 (s, 6H), 3.68 (d, <i>J</i> = 13.2 Hz, 2H), 3.46 (d, <i>J</i> = 12.1 Hz, 2H), 3.16–3.05 (m, 2H), 2.84 (d, <i>J</i> = 12.3 Hz, 2H), 2.80 (s, 3H). (M + H)<sup>+</sup> calculated 677.21; found 676.80.</div></div><div id="sec5_1_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> 4-((2-Acrylamidophenyl)amino)-<i>N</i>-(2-ethylphenyl)-2-((4-(4-methylpiperazin-1-yl)phenyl)amino)pyrimidine-5-carboxamide (<b>15h</b>)</h4><div class="NLM_p last">Compound <b>15h</b> was synthesized via the same route as compound <b>15a</b>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.87 (s, 1H), 9.73 (s, 1H), 9.72 (s, 1H), 9.70 (s, 1H), 9.61 (s, 1H), 8.76 (s, 1H), 8.05 (s, 1H), 7.46 (d, <i>J</i> = 8.8 Hz, 2H), 7.31 (d, <i>J</i> = 8.0 Hz, 1H), 7.27–7.21 (m, 1H), 7.23–7.12 (m, 4H), 7.08 (t, <i>J</i> = 7.7 Hz, 1H), 6.84 (d, <i>J</i> = 8.6 Hz, 2H), 6.31 (dd, <i>J</i> = 17.1, 10.3 Hz, 1H), 6.08 (dd, <i>J</i> = 17.1, 2.0 Hz, 1H), 5.59 (dd, <i>J</i> = 10.2, 2.0 Hz, 1H), 3.69 (d, <i>J</i> = 13.2 Hz, 2H), 3.46 (d, <i>J</i> = 12.0 Hz, 2H), 3.11 (d, <i>J</i> = 11.1 Hz, 2H), 2.85 (d, <i>J</i> = 12.6 Hz, 2H), 2.80 (d, <i>J</i> = 2.9 Hz, 3H), 2.55 (q, <i>J</i> = 7.5 Hz, 2H), 1.07 (t, <i>J</i> = 7.5 Hz, 3H). (M + H)<sup>+</sup> calculated 577.30; found 576.89.</div></div><div id="sec5_1_26" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> 4-((2-Acrylamidophenyl)amino)-2-((4-(4-methylpiperazin-1-yl)phenyl)amino)-<i>N</i>-(4-phenoxyphenyl)pyrimidine-5-carboxamide (<b>15i</b>)</h4><div class="NLM_p last">Compound <b>15i</b> was synthesized via the same route as compound <b>15a</b>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.89 (s, 1H), 10.12 (s, 1H), 9.81 (s, 1H), 9.61 (brs, 2H), 8.82 (s, 1H), 7.78–7.65 (m, 2H), 7.62–7.49 (m, 2H), 7.47–7.36 (m, 3H), 7.27 (t, <i>J</i> = 7.8 Hz, 1H), 7.21–7.08 (m, 2H), 7.08–6.96 (m, 4H), 6.91 (d, <i>J</i> = 8.4 Hz, 2H), 6.48 (dd, <i>J</i> = 17.1, 10.2 Hz, 1H), 6.22 (dd, <i>J</i> = 17.2, 2.0 Hz, 1H), 5.77–5.71 (m, 1H), 3.80–3.73 (m, 2H), 3.59–3.51 (m, 2H), 3.25–3.12 (m, 2H), 2.96–2.85 (m, 5H). (M + H)<sup>+</sup> calculated 641.29; found 641.50.</div></div><div id="sec5_1_27" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> 4-((2-Acrylamidophenyl)amino)-2-chloro-<i>N</i>-(2-chloro-6-methylphenyl)pyrimidine-5-carboxamide (<b>16</b>)</h4><div class="NLM_p last">To the solution of intermediate <b>13a</b> (54.4 mg, 0.15 mmol) in NaHCO<sub>3</sub> (aq. sat., 1 mL) and THF (1 mL) was added acryloyl chloride (12 μL, 0.15 mmol) under an ice bath. After 5 min, the intermediate was consumed, and the mixture was extracted with ethyl acetate. The organic layer was concentrated under vacuum, and the residue was purified with flash chromatography to yield the title product (66 mg, 0.15 mmol, quant.). (M + H)<sup>+</sup> calculated 442.08; found 441.78.</div></div><div id="sec5_1_28" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> 4-((2-Acrylamidophenyl)amino)-<i>N</i>-(2-chloro-6-methylphenyl)-2-((3-(4-methylpiperazin-1-yl)phenyl)amino)pyrimidine-5-carboxamide (<b>17a</b>)</h4><div class="NLM_p last">To the solution of intermediate <b>16</b> (20 mg, 0.045 mol) and 3-(4-methylpiperazin-1-yl)aniline (10 mg, 0.054 mmol) in <i>s</i>-BuOH (1 mL) was added TFA (10 μL, 0.068 mmol) at room temperature. The mixture was then heated up to 90 °C for 30 min. The mixture was cooled down to room temperature, and the crude residue was purified by HPLC to provide the title compound (23 mg, 0.032 mmol, 72%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.73 (s, 1H), 9.88 (s, 1H), 9.72 (s, 1H), 9.67 (s, 1H), 9.61 (s, 1H), 8.86 (s, 1H), 8.00 (s, 1H), 7.36–7.28 (m, 2H), 7.25–7.12 (m, 5H), 7.09 (td, <i>J</i> = 7.6, 1.6 Hz, 1H), 7.04 (t, <i>J</i> = 8.1 Hz, 1H), 6.60 (d, <i>J</i> = 8.9 Hz, 1H), 6.29 (dd, <i>J</i> = 17.1, 10.3 Hz, 1H), 6.07 (dd, <i>J</i> = 17.1, 2.0 Hz, 1H), 5.57 (dd, <i>J</i> = 10.2, 2.0 Hz, 1H), 3.55 (d, <i>J</i> = 13.3 Hz, 2H), 3.39 (d, <i>J</i> = 11.9 Hz, 2H), 3.06–3.00 (m, 2H), 2.84–2.76 (m, 5H), 2.16 (s, 3H). (M + H)<sup>+</sup> calculated 597.24; found 597.29.</div></div><div id="sec5_1_29" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> 4-((2-Acrylamidophenyl)amino)-<i>N</i>-(2-chloro-6-methylphenyl)-2-((2-(4-methylpiperazin-1-yl)phenyl)amino)pyrimidine-5-carboxamide (<b>17b</b>)</h4><div class="NLM_p last">Compound <b>17b</b> was synthesized via the same route as compound <b>17a</b>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.74 (s, 1H), 9.93 (s, 1H), 9.74 (s, 2H), 8.86 (s, 1H), 8.53 (s, 1H), 7.91 (dt, <i>J</i> = 7.9, 2.3 Hz, 2H), 7.34 (dd, <i>J</i> = 7.5, 2.0 Hz, 1H), 7.29 (dd, <i>J</i> = 7.8, 1.7 Hz, 1H), 7.26–7.18 (m, 2H), 7.17 (s, 1H), 7.15–7.09 (m, 1H), 7.07 (s, 1H), 7.01 (td, <i>J</i> = 7.6, 1.7 Hz, 1H), 6.99–6.92 (m, 1H), 6.36–6.21 (m, 1H), 6.06 (dd, <i>J</i> = 17.1, 1.9 Hz, 1H), 5.57 (dd, <i>J</i> = 10.2, 2.0 Hz, 1H), 3.44 (d, <i>J</i> = 12.6 Hz, 2H), 3.23 (s, 2H), 3.13–3.04 (m, 2H), 2.98–2.87 (m, 2H), 2.82 (s, 3H), 2.16 (s, 3H). (M + H)<sup>+</sup> calculated 597.24; found 596.89.</div></div><div id="sec5_1_30" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> 4-((2-Acrylamidophenyl)amino)-<i>N</i>-(2-chloro-6-methylphenyl)-2-((4-(4-(4-methylpiperazin-1-yl)piperidin-1-yl)phenyl)amino)pyrimidine-5-carboxamide (<b>17c</b>)</h4><div class="NLM_p last">Compound <b>17c</b> was synthesized via the same route as compound <b>17a</b>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.83 (s, 1H), 9.95 (s, 1H), 9.85 (s, 2H), 9.79 (s, 1H), 8.90 (s, 1H), 8.06 (s, 1H), 7.55 (s, 2H), 7.45–7.36 (m, 2H), 7.32–7.24 (m, 3H), 7.19 (t, <i>J</i> = 7.6 Hz, 1H), 7.02 (s, 2H), 6.36 (dd, <i>J</i> = 17.1, 10.3 Hz, 1H), 6.14 (dd, <i>J</i> = 17.1, 1.9 Hz, 1H), 5.64 (dd, <i>J</i> = 10.2, 2.0 Hz, 1H), 3.74 (d, <i>J</i> = 12.1 Hz, 2H), 3.55 (s, 4H), 3.15 (s, 4H), 2.83 (s, 5H), 2.24 (s, 3H), 2.13–2.06 (m, 2H), 1.78–1.72 (m, 2H). (M + H)<sup>+</sup> calculated 680.31; found 680.39.</div></div><div id="sec5_1_31" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> 4-((2-Acrylamidophenyl)amino)-<i>N</i>-(2-chloro-6-methylphenyl)-2-((2-methoxy-4-(4-methylpiperazin-1-yl)phenyl)amino)pyrimidine-5-carboxamide (<b>17d</b>)</h4><div class="NLM_p last">Compound <b>17d</b> was synthesized via the same route as compound <b>17a</b>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.88 (s, 1H), 9.91 (s, 1H), 9.75 (s, 2H), 9.69 (s, 1H), 8.77 (s, 2H), 7.95 (s, 1H), 7.40 (d, <i>J</i> = 8.6 Hz, 1H), 7.33 (dd, <i>J</i> = 7.4, 2.1 Hz, 1H), 7.25–7.16 (m, 3H), 7.08 (s, 1H), 7.03 (t, <i>J</i> = 7.3 Hz, 1H), 6.66 (d, <i>J</i> = 2.5 Hz, 1H), 6.47–6.41 (m, 1H), 6.25 (dd, <i>J</i> = 17.1, 10.3 Hz, 1H), 6.05 (dd, <i>J</i> = 17.1, 1.9 Hz, 1H), 5.55 (dd, <i>J</i> = 10.2, 2.0 Hz, 1H), 3.82 (d, <i>J</i> = 13.2 Hz, 2H), 3.74 (s, 3H), 3.49 (d, <i>J</i> = 12.0 Hz, 2H), 3.12 (d, <i>J</i> = 10.6 Hz, 2H), 2.90 (t, <i>J</i> = 12.4 Hz, 2H), 2.82 (d, <i>J</i> = 2.7 Hz, 3H), 2.15 (s, 3H). (M + H)<sup>+</sup> calculated 627.25; found 627.49.</div></div><div id="sec5_1_32" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> 4-((2-Acrylamidophenyl)amino)-<i>N</i>-(2-chloro-6-methylphenyl)-2-((2-methoxy-4-morpholinophenyl)amino)pyrimidine-5-carboxamide (<b>17e</b>)</h4><div class="NLM_p last">Compound <b>17b</b> was synthesized via the same route as compound <b>17a</b>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.04 (s, 1H), 10.10 (s, 1H), 9.83 (s, 1H), 9.05 (s, 1H), 8.81 (s, 1H), 7.99 (s, 1H), 7.41 (dd, <i>J</i> = 7.3, 2.2 Hz, 1H), 7.34–7.24 (m, 3H), 7.18 (s, 2H), 6.69 (d, <i>J</i> = 2.5 Hz, 1H), 6.47 (s, 1H), 6.33 (dd, <i>J</i> = 17.1, 10.2 Hz, 1H), 6.13 (dd, <i>J</i> = 17.0, 2.0 Hz, 1H), 5.64 (dd, <i>J</i> = 10.1, 2.0 Hz, 1H), 3.81 (s, 3H), 3.77 (t, <i>J</i> = 4.8 Hz, 4H), 3.17 (t, <i>J</i> = 4.8 Hz, 4H), 2.23 (s, 3H). (M + H)<sup>+</sup> calculated 614.22; found 614.39.</div></div><div id="sec5_1_33" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> 4-((2-Acrylamidophenyl)amino)-<i>N</i>-(2-chloro-6-methylphenyl)-2-((3-methyl-4-(4-methylpiperazin-1-yl)phenyl)amino)pyrimidine-5-carboxamide (<b>17f</b>)</h4><div class="NLM_p last">Compound <b>17f</b> was synthesized via the same route as compound <b>17a</b>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.77 (s, 1H), 9.85 (s, 1H), 9.69 (s, 1H), 9.66 (s, 1H), 9.61 (s, 1H), 8.84 (s, 1H), 8.01 (s, 1H), 7.46 (d, <i>J</i> = 2.6 Hz, 1H), 7.37–7.29 (m, 3H), 7.26–7.14 (m, 3H), 7.09 (t, <i>J</i> = 7.6 Hz, 1H), 6.87 (d, <i>J</i> = 8.6 Hz, 1H), 6.28 (dd, <i>J</i> = 17.1, 10.2 Hz, 1H), 6.06 (dd, <i>J</i> = 17.1, 2.0 Hz, 1H), 5.56 (dd, <i>J</i> = 10.3, 2.0 Hz, 1H), 3.43 (d, <i>J</i> = 11.7 Hz, 2H), 3.19–3.11 (m, 2H), 3.08 (d, <i>J</i> = 13.6 Hz, 2H), 2.88–2.80 (m, 5H), 2.16 (s, 3H), 2.11 (s, 3H). (M + H)<sup>+</sup> calculated 611.26; found 611.39.</div></div><div id="sec5_1_34" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> 4-((3-Aminophenyl)amino)-2-chloro-<i>N</i>-(2-chloro-6-methylphenyl)pyrimidine-5-carboxamide (<b>18a</b>)</h4><div class="NLM_p last">To the solution of intermediate <b>12a</b> (28 mg, 0.089 mmol) and bezene-1,3-diamine (9.6 mg, 0.089 mmol) in <i>n</i>-BuOH (2 mL) was added DIEA (31 μL, 0.18 mmol) at room temperature. The mixture was then heated up to 90 °C for 1 h. The crude residue was purified with flash chromatography to yield the product (34 mg, 0.089 mmol, quant.). (M + H)<sup>+</sup> calculated 388.07; found 388.20.</div></div><div id="sec5_1_35" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> 4-((3-Aminophenyl)amino)-<i>N</i>-(2-chloro-6-methylphenyl)-2-((4-(4-methylpiperazin-1-yl)phenyl)amino)pyrimidine-5-carboxamide (<b>19a</b>)</h4><div class="NLM_p last">To the solution of intermediate <b>18a</b> (34 mg, 0.089 mmol) and 4-(4-methylpiperazin-1-yl)aniline (19 mg, 0.098 mmol) in <i>s</i>-BuOH (1 mL) was added TFA (10 μL, 0.13 mmol) at room temperature. The mixture was heated up to 90 °C for 2 h. The mixture was cooled down to room temperature and was evaporated under vacuum to yield the crude product without further purification. (M + H)<sup>+</sup> calculated 543.23; found 543.42.</div></div><div id="sec5_1_36" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> 4-((3-Acrylamidophenyl)amino)-<i>N</i>-(2-chloro-6-methylphenyl)-2-((4-(4-methylpiperazin-1-yl)phenyl)amino)pyrimidine-5-carboxamide (<b>20a</b>)</h4><div class="NLM_p last">To the solution of intermediate <b>19a</b> (crude, 0.089 mmol) in NaHCO<sub>3</sub> (sat. aq., 1 mL) and THF (1 mL) was added acryloyl chloride (9 mg, 0.098 mmol) under an ice bath. The reaction was extracted with ethyl acetate after all <b>19a</b> was consumed. The crude residue was purified by HPLC to provide the title compound as a white solid (21 mg, 0.034 mmol, 38% over 2 steps). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.15 (s, 1H), 10.16 (s, 1H), 10.04 (s, 1H), 9.72 (s, 2H), 8.95 (s, 1H), 7.75 (s, 1H), 7.70–7.40 (s, 1H) 7.59 (d, <i>J</i> = 8.4 Hz, 2H), 7.43 (dd, <i>J</i> = 7.7, 1.9 Hz, 2H), 7.35–7.26 (m, 3H), 6.91 (s, 2H), 6.44 (dd, <i>J</i> = 16.9, 10.1 Hz, 1H), 6.26 (dd, <i>J</i> = 17.0, 2.0 Hz, 1H), 5.77 (dd, <i>J</i> = 10.1, 2.0 Hz, 1H), 3.75–3.70 (m, 2H), 3.53 (d, <i>J</i> = 12.1 Hz, 2H), 3.23–3.14 (m, 2H), 2.94–2.86 (m, 5H), 2.28 (s, 3H). (M + H)<sup>+</sup> calculated 597.24; found 597.39.</div></div><div id="sec5_1_37" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> 4-((4-Acrylamidophenyl)amino)-<i>N</i>-(2-chloro-6-methylphenyl)-2-((4-(4-methylpiperazin-1-yl)phenyl)amino)pyrimidine-5-carboxamide (<b>20b</b>)</h4><div class="NLM_p last">Compound <b>20b</b> was synthesized via the same route as compound <b>20a</b>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.09 (s, 1H), 10.17 (s, 1H), 10.02 (s, 1H), 9.74 (s, 2H), 8.93 (s, 1H), 7.73–7.49 (m, 6H), 7.42 (dd, <i>J</i> = 7.7, 1.8 Hz, 1H), 7.35–7.25 (m, 2H), 6.97 (d, <i>J</i> = 8.7 Hz, 2H), 6.45 (dd, <i>J</i> = 16.9, 10.1 Hz, 1H), 6.27 (dd, <i>J</i> = 16.9, 2.1 Hz, 1H), 5.77 (dd, <i>J</i> = 10.0, 2.1 Hz, 1H), 3.80 (d, <i>J</i> = 13.3 Hz, 2H), 3.57–3.51 (m, 2H), 3.20 (d, <i>J</i> = 10.4 Hz, 2H), 2.95 (t, <i>J</i> = 12.5 Hz, 2H), 2.89 (s, 3H), 2.27 (s, 3H). (M + H)<sup>+</sup> calculated 597.24; found 597.29.</div></div><div id="sec5_1_38" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> <i>N</i>-(2-Chloro-6-methylphenyl)-4-((2-(2-chloroacetamido)phenyl)amino)-2-((4-(4-methylpiperazin-1-yl)phenyl)amino)pyrimidine-5-carboxamide (<b>20c</b>)</h4><div class="NLM_p last">4-((2-Aminophenyl)amino)-<i>N</i>-(2-chloro-6-methylphenyl)-2-((4-(4-methylpiperazin-1-yl)phenyl)amino)pyrimidine-5-carboxamide (18 mg, 0.040 mmol) was dissolved in THF (1 mL) and sat. aqueous NaHCO<sub>3</sub> (1 mL). Chloroacetyl chloride dissolved in THF was slowly titrated at 0 °C until the starting aniline was consumed. Water was added followed by extraction with EtOAc. Combined extracts were washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, concentrated, and purified by HPLC to provide the title compound as a white solid (4.4 mg, 0.006 mmol, 15%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.02 (s, 1H), 9.99 (s, 1H), 9.90 (s, 1H), 9.77 (s, 2H), 8.93 (s, 1H), 8.15 (s, 1H), 7.55–7.51 (m, 2H), 7.42 (dd, <i>J</i> = 7.7, 1.8 Hz, 1H), 7.35–7.25 (m, 4H), 7.16 (t, <i>J</i> = 7.6 Hz, 1H), 6.93 (d, <i>J</i> = 8.5 Hz, 2H), 4.15 (s, 2H), 3.81–3.74 (m, 2H), 3.54 (d, <i>J</i> = 12.1 Hz, 2H), 3.19 (s, 2H), 2.97–2.86 (m, 5H), 2.27 (s, 3H). (M + H)<sup>+</sup> calculated 619.20; found 619.19.</div></div><div id="sec5_1_39" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> <i>N</i>-(2-Chloro-6-methylphenyl)-2-((4-(4-methylpiperazin-1-yl)phenyl)amino)-4-((2-(vinylsulfonamido)phenyl)amino)pyrimidine-5-carboxamide (<b>20d</b>)</h4><div class="NLM_p last">4-((2-Aminophenyl)amino)-<i>N</i>-(2-chloro-6-methylphenyl)-2-((4-(4-methylpiperazin-1-yl)phenyl)amino)pyrimidine-5-carboxamide (23 mg, 0.042 mmol) was dissolved in MeCN (1 mL) and DMF (1 mL) and treated with Et<sub>3</sub>N (23 μL, 0.17 mmol). Then, 2-chloroethanesulfonyl chloride (9.2 mg, 0.056 mmol) in a small amount of MeCN was added at 0 °C. After stirring for 30 min, water was added, followed by extraction with CHCl<sub>3</sub>/IPA (3:1). Combined extracts were washed with brine, concentrated, and purified by HPLC to provide the title compound as a white solid (3.5 mg, 0.0055 mmol, 13%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.15 (s, 1H), 9.98 (s, 1H), 9.84 (s, 1H), 9.71 (s, 1H), 9.42 (s, 1H), 8.93 (s, 1H), 8.24 (s, 1H), 7.52 (s, 2H), 7.43 (dd, <i>J</i> = 7.9, 1.7 Hz, 1H), 7.35–7.26 (m, 3H), 7.24 (dd, <i>J</i> = 8.0, 1.6 Hz, 1H), 7.11 (t, <i>J</i> = 7.4 Hz, 1H), 6.93 (d, <i>J</i> = 8.5 Hz, 2H), 6.77 (dd, <i>J</i> = 16.5, 9.9 Hz, 1H), 5.88 (d, <i>J</i> = 2.1 Hz, 1H), 5.86 (d, <i>J</i> = 4.8 Hz, 1H), 3.77 (d, <i>J</i> = 13.4 Hz, 2H), 3.53 (d, <i>J</i> = 12.2 Hz, 2H), 3.21–3.10 (m, 2H), 2.98–2.89 (m, 2H), 2.87 (s, 3H), 2.30 (s, 3H). (M + H)<sup>+</sup> calculated 633.20; found 633.20.</div></div><div id="sec5_1_40" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> (<i>E</i>)-<i>N</i>-(2-Chloro-6-methylphenyl)-4-((2-(4-(dimethylamino)but-2-enamido)phenyl)amino)-2-((4-(4-methylpiperazin-1-yl)phenyl)amino)pyrimidine-5-carboxamide (<b>20e</b>)</h4><div class="NLM_p last">4-((2-Aminophenyl)amino)-<i>N</i>-(2-chloro-6-methylphenyl)-2-((4-(4-methylpiperazin-1-yl)phenyl)amino)pyrimidine-5-carboxamide (23 mg, 0.042 mmol) was dissolved in THF (1 mL) and sat. aqueous NaHCO<sub>3</sub> (1 mL). Then, 4-bromocrotonyl chloride in THF was slowly titrated until the starting aniline was consumed. Water was added, followed by extraction with CHCl<sub>3</sub>/IPA (3:1). Combined extracts were washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated. The residue was then dissolved in 1 mL of MeCN and 1 mL of DMF. At 50 °C, dimethylamine solution (100 μL, 2.0 M in THF) was added. After stirring for 30 min, the mixture was diluted with water, followed by extraction with CHCl<sub>3</sub>/IPA (3:1). Combined extracts were washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, concentrated, and purified by HPLC to provide the title compound as a white solid (2.0 mg, 2.6 μmol, 6% over 2 steps). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.82 (s, 1H), 9.98 (s, 1H), 9.96 (s, 1H), 9.94 (s, 1H), 9.70 (s, 1H), 8.90 (s, 1H), 8.20 (s, 1H), 7.55 (s, 2H), 7.42 (dd, <i>J</i> = 7.6, 2.0 Hz, 1H), 7.34–7.25 (m, 4H), 7.15 (t, <i>J</i> = 7.7 Hz, 1H), 6.93 (d, <i>J</i> = 8.6 Hz, 2H), 6.67–6.57 (m, 1H), 6.40 (d, <i>J</i> = 15.3 Hz, 1H), 3.82 (d, <i>J</i> = 7.0 Hz, 2H), 3.76 (d, <i>J</i> = 12.9 Hz, 2H), 3.55–3.52 (m, 2H), 3.18 (s, 2H), 2.98–2.89 (m, 2H), 2.88 (s, 3H), 2.66 (s, 6H), 2.24 (s, 3H). (M + H)<sup>+</sup> calculated 654.30; found 654.40.</div></div><div id="sec5_1_41" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61"> <i>N</i>-(2-Chloro-6-methylphenyl)-2-((4-(4-methylpiperazin-1-yl)phenyl)amino)-4-((2-propionamidophenyl)amino)pyrimidine-5-carboxamide (<b>20f</b>)</h4><div class="NLM_p last">4-((2-Aminophenyl)amino)-<i>N</i>-(2-chloro-6-methylphenyl)-2-((4-(4-methylpiperazin-1-yl)phenyl)amino)pyrimidine-5-carboxamide was dissolved in THF (1 mL) and sat. aqueous NaHCO<sub>3</sub> (1 mL). Then, propionyl chloride in THF was slowly titrated until the starting aniline was consumed. Water was added, followed by extraction with EtOAc. Combined extracts were washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, concentrated, and purified by HPLC to provide the title compound as a white solid (13 mg, 0.021 mmol, 19%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.92 (s, 1H), 9.86 (s, 1H), 9.64 (s, 2H), 9.39 (s, 1H), 8.84 (s, 1H), 8.30–8.00 (s, 1H), 7.48 (d, <i>J</i> = 8.2 Hz, 2H), 7.34 (dd, <i>J</i> = 7.6, 1.8 Hz, 1H), 7.27–7.11 (m, 4H), 7.03 (t, <i>J</i> = 7.6 Hz, 1H), 6.86 (d, <i>J</i> = 8.4 Hz, 2H), 3.70 (d, <i>J</i> = 13.2 Hz, 2H), 3.47 (d, <i>J</i> = 12.1 Hz, 2H), 3.12 (d, <i>J</i> = 11.2 Hz, 2H), 2.89–2.79 (m, 5H), 2.18 (s, 3H), 2.14 (q, <i>J</i> = 7.6 Hz, 2H), 0.87 (t, <i>J</i> = 7.6 Hz, 3H). (M + H)<sup>+</sup> calculated 599.26; found 598.89.</div></div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i62" class="anchor-spacer"></div><h3 class="article-section__title" id="_i62"> Cell Culture</h3><div class="NLM_p last">In this study, three cell lines were used including H1975 and HCC827 (nonsmall cell lung cancer, NSCLC) and MDA-MB-231 (triple negative breast cancer, TNBC) cells. The cell lines were grown, maintained, and authenticated according to the methodology previously described.<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a></div></div><div id="sec5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i63" class="anchor-spacer"></div><h3 class="article-section__title" id="_i63"> Growth Inhibition Assay</h3><div class="NLM_p last">For the growth inhibition assay, cells were plated in 384-well plates (3764, Corning) (2000 cells/well) using the Multidrop Combi Reagent Dispenser (Thermo Fisher Scientific). The next day, cells were treated with varying doses of <b>15a</b>, <b>20f</b>, <b>AZD0530</b> (saracatinib), or DMSO (0.2%) using the D300 digital dispenser (HP-D300 Digital Dispenser). The treatment period lasted 72 h, followed by the addition of 25 μL/well of CellTiter-Glo reagent (G7572, Promega). Cell viability was measured using the Synergy H1 microplate reader (Biotek), and analysis was carried out as previously described.<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> The GR values were calculated using the online GR calculator (<a href="http://www.grcalculator.org/grtutorial/Home.html" class="extLink">http://www.grcalculator.org/grtutorial/Home.html</a>) according to the methodology recently described.<a onclick="showRef(event, 'ref40 ref45'); return false;" href="javascript:void(0);" class="ref ref40 ref45">(40,45)</a></div></div><div id="sec5_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i64" class="anchor-spacer"></div><h3 class="article-section__title" id="_i64"> Western Blotting</h3><div class="NLM_p last">For cellular target engagement and signaling analyses, H1975 and HCC827 NSCLC cells were treated with 1 μM of <b>15a</b>, <b>20f</b>, or DMSO for 2 h, followed by rinsing cells twice with cold PBS, extracting, and lysing with cold M-PER Mammalian Protein Extraction Reagent (78505, Thermo Fisher Scientific) substituted with 1× HALT protease and phosphatase inhibitor (78446, Thermo Fisher Scientific). In the case of the washout assay, drug-containing medium was replaced with drug-free medium and cells were lysed at 0, 2, or 4 h postwashout. To prepare samples for signaling analysis, a small volume of the lysate obtained was combined with loading buffer [5% v/v 2-mercaptoethanol (Sigma-Aldrich) + 4× Laemmli Sample Buffer (161-0747 Bio-Rad)]. To prepare samples for cellular target engagement, lysates were incubated with 1 μM TL13-68 with constant rocking at 4 °C overnight, followed by pulldown using streptavidin beads (20353, Thermo Fisher Scientific). Kinases pulled down on beads were then rigorously washed (5 times) using cold M-PER lysis buffer. Excess liquid was removed from the beads after washes and diluted using loading buffer. For both signaling analysis and target engagement, samples were assessed using Western blotting according to the methodology previously described.<a onclick="showRef(event, 'ref38 ref51'); return false;" href="javascript:void(0);" class="ref ref38 ref51">(38,51)</a> Kinases of interest were detected using the following antibodies from Cell Signaling Technology: phospho-SRCY416 (6943S, rabbit), SRC (2123S, rabbit), YES1 (65890S, rabbit), MEK1/2 (9126S, rabbit), and ERK1/2 (4696S, mouse). The β-actin antibody was purchased from Santa Cruz (sc-47778, mouse). Signaling and target engagement experiments were repeated twice, and blots from each independent experiment are presented in <a href="/doi/suppl/10.1021/acs.jmedchem.9b01502/suppl_file/jm9b01502_si_001.pdf" class="ext-link">Figure S2</a>.</div></div><div id="sec5_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i65" class="anchor-spacer"></div><h3 class="article-section__title" id="_i65"> Pharmacodynamic Analysis</h3><div class="NLM_p">All mouse procedures were reviewed and approved by the Dana-Farber Animal Care and Use Committee (Protocol number: 16-015) and carried out in the pathogen-free animal facilities in Dana-Farber Cancer Institute (Boston, MA). Adult C57B6 mice were purchased from Jackson Laboratory (#000058, Bar Harbor, ME) and dosed by vehicle (5% DMSO, 95% D5W) or <b>15a</b> (5 MPK) for 3 times every 12 h via intraperitoneal injection. Mouse spleens were harvested 2 or 4 h after the last dose.</div><div class="NLM_p last">Spleen tissue harvested from mice were lysed and homogenized using the M-PER Mammalian Protein Extraction Reagent (78505, Thermo Fisher Scientific) substituted with 1× HALT protease and phosphatase inhibitor (78446, Thermo Fisher Scientific). Sample preparation for both signaling and target engagement analyses was performed similar to the above cellular assays. Target inhibition was assessed using Western blotting according to the methodology described in Rao et al.<a onclick="showRef(event, 'ref38 ref51'); return false;" href="javascript:void(0);" class="ref ref38 ref51">(38,51)</a> Antibodies used in the study included phospho-SRCY416 (6943S, rabbit) and SRC (2123S, rabbit) from Cell Signaling Technologies and β-actin antibody from Santa Cruz (sc-47778, mouse).</div></div><div id="sec5_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i66" class="anchor-spacer"></div><h3 class="article-section__title" id="_i66"> Molecular Docking</h3><div class="NLM_p last">Docking of <b>SM1-71</b> was performed with the Glide covalent docking function (version 2019 release 1). It was docked into the FGFR1 cocrystal structure (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MZW">6MZW</a>), and the Cys488 at the P-loop was chosen as the reactive residue. Hinge hydrogen bonds shown in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a> were used as a constraint to ensure the proper binding mode. Default values were used for other parameters.</div></div><div id="sec5_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i67" class="anchor-spacer"></div><h3 class="article-section__title" id="_i67"> Mass Spectrometry</h3><div class="NLM_p last">Src protein (10 μg) was treated with DMSO or a 10-fold molar excess of DGY-06-116 for 2 h at 37 °C and analyzed by LC-MS using an HPLC (Shimadzu, Marlborough, MA) interfaced to an LTQ ion trap mass spectrometer (ThermoFisher Scientific, San Jose, CA). Protein (5 μg) was injected onto a reversed phase column (5 cm POROS 50R2, Applied Biosystems, Foster City, CA), desalted for 4 min (100% A), and gradient eluted (0–100% B in 1 min; A = 0.2 M acetic acid in water; B = 0.2 M acetic acid in acetonitrile) into the mass spectrometer (spray voltage = 4.8 kV). The mass spectrometer was programmed to acquire profile mass spectra (<i>m</i>/<i>z</i> 300–2000). Raw data was deconvoluted using MagTran version 1.03b2.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> To determine which amino acid was modified by the inhibitor, labeled protein was first reduced (10 mM dithiothreitol for 30 min at 56 °C), alkylated (22.5 mM iodoacetamide for 30 min at room temperature and protected from light), and digested with trypsin. Peptides were then desalted by C18, dried by vacuum centrifugation, reconstituted in 50% MeCN, 1% formic acid, and 100 mM ammonium acetate, and analyzed by CE-MS using a ZipChip autosampler and CE system (908 Devices, Boston, MA) interfaced to a QE-HF mass spectrometer (Thermofisher Scientific). The mass spectrometer was operated in data-dependent mode, and the 5 most abundant ions in each MS scan (<i>m</i>/<i>z</i> 300–2000, resolution = 60K, target = 3 × 10<sup>6</sup>, max fill time = 50 ms) were subjected to MS/MS (collision energy = 35%, target = 1 × 10<sup>5</sup>, max fill time = 100 ms). Dynamic exclusion was enabled with a repeat count of 1 and an exclusion duration of 6 s. Raw data were converted to .mgf using the multiplierz toolset<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> and searched using Mascot 2.6.1 against a forward reversed human refseq database (NCBI). Search parameters specified a precursor mass tolerance of 50 ppm, a product ion tolerance of 25 mmu, fixed carbamidomethylation of cysteine, and variable oxidation of methionine as well as variable DGY-06-116 modification of cysteine. Inhibitor related fragment ions were assigned as described.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i68"><a href="/doi/suppl/10.1021/acs.jmedchem.9b01502" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_00277" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_00277" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.9b01502?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01502</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Western blot analysis characterizing target engagement and signaling inhibition in cells for compounds <b>AZD0530</b>, <b>15a</b>, and <b>20f</b> (Figure S1); structure of <b>TL13-68</b> (Figure S2); KinomeScan profiling data for compound <b>SM1-71</b> (Table S1); KinomeScan profiling data for compound <b>15a</b> (Table S2); IC<sub>50</sub>’s of top off-targets for compound <b>15a</b> (Table S3); compound <b>15a</b> in a SRC-TEL and HCK-BCR Ba/F3 cell-based assay (Table S4); GR<sub>50</sub> and GR<sub>max</sub> values from a growth inhibitory screen across multiple cancer cell lines for compounds <b>AZD0530</b>, <b>15a</b>, and <b>20f</b> (Table S5) (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01502/suppl_file/jm9b01502_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01502/suppl_file/jm9b01502_si_002.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">Growth inhibtion assay for compounds <b>AZD0530</b>, <b>15a</b>, and <b>20f</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01502/suppl_file/jm9b01502_si_003.xlsx" class="ext-link">XLSX</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01502/suppl_file/jm9b01502_si_001.pdf">jm9b01502_si_001.pdf (1.25 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01502/suppl_file/jm9b01502_si_002.csv">jm9b01502_si_002.csv (6.49 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01502/suppl_file/jm9b01502_si_003.xlsx">jm9b01502_si_003.xlsx (10.41 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.9b01502" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_04328" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_04328" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kenneth D. Westover</span> - <span class="hlFld-Affiliation affiliation">Departments
of Biochemistry and Radiation Oncology, The University of Texas Southwestern Medical Center at Dallas, Dallas, Texas 75390, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-3653-5923" title="Orcid link">http://orcid.org/0000-0003-3653-5923</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#99f2fcf7f7fcedf1b7eefceaedf6effcebd9ecedeaf6ecedf1eefceaedfcebf7b7fcfdec"><span class="__cf_email__" data-cfemail="ddb6b8b3b3b8a9b5f3aab8aea9b2abb8af9da8a9aeb2a8a9b5aab8aea9b8afb3f3b8b9a8">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tinghu Zhang</span> - <span class="hlFld-Affiliation affiliation">Department
of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts 02115, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Cancer Biology, Dana Farber Cancer Institute, 450 Brookline Avenue, Boston, Massachusetts 02215, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#9bcff2f5fcf3eec4c1f3faf5fcdbfffdf8f2b5f3fae9edfae9ffb5feffee"><span class="__cf_email__" data-cfemail="e0b4898e878895bfba88818e87a084868389ce88819296819284ce858495">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nathanael S. Gray</span> - <span class="hlFld-Affiliation affiliation">Department
of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts 02115, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Cancer Biology, Dana Farber Cancer Institute, 450 Brookline Avenue, Boston, Massachusetts 02215, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5354-7403" title="Orcid link">http://orcid.org/0000-0001-5354-7403</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#3977584d515857585c55667e4b5840795d5f5a501751584b4f584b5d175c5d4c"><span class="__cf_email__" data-cfemail="97d9f6e3fff6f9f6f2fbc8d0e5f6eed7f3f1f4feb9fff6e5e1f6e5f3b9f2f3e2">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Guangyan Du</span> - <span class="hlFld-Affiliation affiliation">Department
of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts 02115, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Cancer Biology, Dana Farber Cancer Institute, 450 Brookline Avenue, Boston, Massachusetts 02215, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Suman Rao</span> - <span class="hlFld-Affiliation affiliation">Department
of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts 02115, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Cancer Biology, Dana Farber Cancer Institute, 450 Brookline Avenue, Boston, Massachusetts 02215, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Laboratory
of Systems Biology, Harvard Medical School, Boston, Massachusetts 02115, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Deepak Gurbani</span> - <span class="hlFld-Affiliation affiliation">Departments
of Biochemistry and Radiation Oncology, The University of Texas Southwestern Medical Center at Dallas, Dallas, Texas 75390, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nathaniel J. Henning</span> - <span class="hlFld-Affiliation affiliation">Department
of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts 02115, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Cancer Biology, Dana Farber Cancer Institute, 450 Brookline Avenue, Boston, Massachusetts 02215, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jie Jiang</span> - <span class="hlFld-Affiliation affiliation">Department
of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts 02115, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Cancer Biology, Dana Farber Cancer Institute, 450 Brookline Avenue, Boston, Massachusetts 02215, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jianwei Che</span> - <span class="hlFld-Affiliation affiliation">Department
of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts 02115, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Cancer Biology, Dana Farber Cancer Institute, 450 Brookline Avenue, Boston, Massachusetts 02215, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Annan Yang</span> - <span class="hlFld-Affiliation affiliation">Department
of Medical Oncology, Dana Farber Cancer
Institute, 450 Brookline Avenue, Boston, Massachusetts 02215, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Scott B. Ficarro</span> - <span class="hlFld-Affiliation affiliation">Department
of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts 02115, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jarrod A. Marto</span> - <span class="hlFld-Affiliation affiliation">Department
of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts 02115, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Andrew J. Aguirre</span> - <span class="hlFld-Affiliation affiliation">Department
of Medical Oncology, Dana Farber Cancer
Institute, 450 Brookline Avenue, Boston, Massachusetts 02215, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Peter K. Sorger</span> - <span class="hlFld-Affiliation affiliation">Laboratory
of Systems Biology, Harvard Medical School, Boston, Massachusetts 02115, United States</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>G.D., S.R., and D.G. contributed equally. Conceptualization: N.S.G., T.Z., K.D.W., P.K.S., G.D., S.R., and D.G.; Chemistry: G.D. and N.J.H.; Biology: S.R. and J.J.; Computational Docking: J.C.; Mass spectrum labeling: S.B.F., D.G., and J.A.M.; Animal study: A.Y. and A.J.A.; Writing: T.Z., G.D., S.R., D.G., N.J.H., K.D.W., and N.S.G. The manuscript was written through contributions of all authors. All authors have given approval to the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><p>The authors declare the following competing financial interest(s): N.S.G. is a founder, SAB, and equity holder in Gatekeeper, Syros, Petra, C4, B2S, and Soltego. The Gray lab receives/has received funding from Novartis, Takeda, Astellas, Taiho, Janssen, Kinogen, Voronoi, Her2llc, Deerfield, and Sanofi. A.J.A. has consulted for Oncorus, Inc. and Merck & Co., Inc. and has research funding from Mirati Therapeutics and Deerfield. G.D., N.J.H., S.R., D.G., T.Z., K.D.W., and N.S.G. are inventors on SRC covalent inhibitor patents.<br /></br></p></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i70">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_74017" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_74017" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank Jim Sun at the NMR facility of Dana-Farber cancer institute for his assistance on <sup>1</sup>H NMR and <sup>13</sup>C NMR data collection. Milka Kostic is greatly acknowledged for the editing and proof reading. This work was supported by Welch I-1829, ACS RSG-18-039-01-DMC, and a Career Enhancement Grant through NIH P50CA07090720 to K.D.W.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">TCIs</td><td class="NLM_def"><p class="first last">targeted covalent inhibitors</p></td></tr><tr><td class="NLM_term">NSCLC</td><td class="NLM_def"><p class="first last">nonsmall cell lung cancer</p></td></tr><tr><td class="NLM_term">SFKs</td><td class="NLM_def"><p class="first last">SRC family kinases</p></td></tr><tr><td class="NLM_term">GPCRs</td><td class="NLM_def"><p class="first last">G protein coupled receptors</p></td></tr><tr><td class="NLM_term">RTKs</td><td class="NLM_def"><p class="first last">receptor tyrosine kinases</p></td></tr><tr><td class="NLM_term">CKI</td><td class="NLM_def"><p class="first last">covalent kinase inhibitor</p></td></tr><tr><td class="NLM_term">GR</td><td class="NLM_def"><p class="first last">growth rate</p></td></tr><tr><td class="NLM_term">DIEA</td><td class="NLM_def"><p class="first last"><i>N</i>,<i>N</i>-diisopropylethylamine</p></td></tr><tr><td class="NLM_term">HATU</td><td class="NLM_def"><p class="first last">1-[bis(dimethylamino)methylene]-1<i>H</i>-1,2,3-triazolo[4,5-<i>b</i>]pyridinium 3-oxid hexafluorophosphate, hexafluorophosphate azabenzotriazole tetramethyl uronium</p></td></tr><tr><td class="NLM_term">Et<sub>3</sub>N</td><td class="NLM_def"><p class="first last">triethylamine</p></td></tr><tr><td class="NLM_term">DMF</td><td class="NLM_def"><p class="first last">dimethylformamide</p></td></tr><tr><td class="NLM_term"><i>n</i>-BuOH</td><td class="NLM_def"><p class="first last"><i>n</i>-butanol</p></td></tr><tr><td class="NLM_term"><i>s</i>-BuOH</td><td class="NLM_def"><p class="first last"><i>s</i>-butanol</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i72">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_01837" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_01837" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 51 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, F. M.</span></span> <span> </span><span class="NLM_article-title">Regulation of SRC family kinases in human cancers</span>. <i>J. Signal Transduction</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>2011</i></span>,  <span class="NLM_fpage">865819</span>, <span class="refDoi"> DOI: 10.1155/2011/865819</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.1155%2F2011%2F865819" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=21776389" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A280%3ADC%252BC3MnpvVKktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2011&publication_year=2011&pages=865819&author=B.+Senauthor=F.+M.+Johnson&title=Regulation+of+SRC+family+kinases+in+human+cancers&doi=10.1155%2F2011%2F865819"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of SRC family kinases in human cancers</span></div><div class="casAuthors">Sen Banibrata; Johnson Faye M</div><div class="citationInfo"><span class="NLM_cas:title">Journal of signal transduction</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">2011</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">865819</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The nonreceptor protein tyrosine kinase Src plays a crucial role in the signal transduction pathways involved in cell division, motility, adhesion, and survival in both normal and cancer cells.  Although the Src family kinases (SFKs) are activated in various types of cancers, the exact mechanisms through which they contribute to the progression of individual tumors remain to be defined.  The activation of Src in human cancers may occur through a variety of mechanisms that include domain interaction and structural remodeling in response to various activators or upstream kinases and phosphatastes.  Because of Src's prominent roles in invasion and tumor progression, epithelial-to-mesenchymal transition, angiogenesis, and the development of metastasis, Src is a promising target for cancer therapy.  Several small molecule inhibitors of Src are currently being investigated in clinical trials.  In this article, we will summarize the mechanisms regulating Src kinase activity in normal and cancer cells and discuss the status of Src inhibitor development against various types of cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSCfOL0q4nTvO-MdeMhXAHkfW6udTcc2eZFLCIg63hz7bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3MnpvVKktg%253D%253D&md5=f74b51d238ead275fd6d1179c3cc4346</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1155%2F2011%2F865819&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1155%252F2011%252F865819%26sid%3Dliteratum%253Aachs%26aulast%3DSen%26aufirst%3DB.%26aulast%3DJohnson%26aufirst%3DF.%2BM.%26atitle%3DRegulation%2520of%2520SRC%2520family%2520kinases%2520in%2520human%2520cancers%26jtitle%3DJ.%2520Signal%2520Transduction%26date%3D2011%26volume%3D2011%26spage%3D865819%26doi%3D10.1155%2F2011%2F865819" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wheeler, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iida, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dunn, E. F.</span></span> <span> </span><span class="NLM_article-title">The role of Src in solid tumors</span>. <i>Oncologist</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">667</span>– <span class="NLM_lpage">678</span>, <span class="refDoi"> DOI: 10.1634/theoncologist.2009-0009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.1634%2Ftheoncologist.2009-0009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=19581523" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVOmtrnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2009&pages=667-678&author=D.+L.+Wheelerauthor=M.+Iidaauthor=E.+F.+Dunn&title=The+role+of+Src+in+solid+tumors&doi=10.1634%2Ftheoncologist.2009-0009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">The role of Src in solid tumors</span></div><div class="casAuthors">Wheeler, Deric L.; Iida, Mari; Dunn, Emily F.</div><div class="citationInfo"><span class="NLM_cas:title">Oncologist</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">667-678</span>CODEN:
                <span class="NLM_cas:coden">OCOLF6</span>;
        ISSN:<span class="NLM_cas:issn">1083-7159</span>.
    
            (<span class="NLM_cas:orgname">AlphaMed Press</span>)
        </div><div class="casAbstract">A review.  The proto-oncogene c-Src (Src) encodes a nonreceptor tyrosine kinase whose expression and activity are correlated with advanced malignancy and poor prognosis in a variety of human cancers.  Nine addnl. enzymes with homol. to Src have been identified and collectively are referred to as Src family kinases (SFKs).  Together, SFKs represent the largest family of nonreceptor tyrosine kinases and interact directly with receptor tyrosine kinases, G-protein-coupled receptors, steroid receptors, signal transducers and activators of transcription, and mols. involved in cell adhesion and migration.  These interactions lead to a diverse array of biol. functions including proliferation, cell growth, differentiation, cell shape, motility, migration, angiogenesis, and survival.  Studies investigating mutational activation of Src in human cancers suggest that this may be a rare event and that wild-type Src is weakly oncogenic.  Thus, the role of Src in the development and progression of human cancer remains unclear.  Recently, it was suggested that increased SFK protein levels and, more importantly, SFK tyrosine kinase activity are linked to cancer progression and metastatic disease by facilitating the action of other signaling proteins.  This accumulating body of evidence indicates that SFKs may represent a promising therapeutic target for the treatment of solid tumors.  This review discusses the role of SFKs in solid tumors and the recent therapeutic advances aimed at targeting this family of tyrosine kinases in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpX_M5pcS60XLVg90H21EOLACvtfcHk0licVVZryJMvGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVOmtrnN&md5=9596bcc70d4a44cd7c7519fefd77cce0</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1634%2Ftheoncologist.2009-0009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1634%252Ftheoncologist.2009-0009%26sid%3Dliteratum%253Aachs%26aulast%3DWheeler%26aufirst%3DD.%2BL.%26aulast%3DIida%26aufirst%3DM.%26aulast%3DDunn%26aufirst%3DE.%2BF.%26atitle%3DThe%2520role%2520of%2520Src%2520in%2520solid%2520tumors%26jtitle%3DOncologist%26date%3D2009%26volume%3D14%26spage%3D667%26epage%3D678%26doi%3D10.1634%2Ftheoncologist.2009-0009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Irby, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeatman, T. J.</span></span> <span> </span><span class="NLM_article-title">Role of Src expression and activation in human cancer</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">5636</span>, <span class="refDoi"> DOI: 10.1038/sj.onc.1203912</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.1038%2Fsj.onc.1203912" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=11114744" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BD3cXosl2nurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2000&pages=5636&author=R.+B.+Irbyauthor=T.+J.+Yeatman&title=Role+of+Src+expression+and+activation+in+human+cancer&doi=10.1038%2Fsj.onc.1203912"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Role of Src expression and activation in human cancer</span></div><div class="casAuthors">Irby, Rosalyn B.; Yeatman, Timothy J.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">5636-5642</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review with ~ 78 refs.  Since the original identification of a transmissible agent responsible for the development of tumors in chickens, now known to be a retrovirus encoding the v-src gene, significant progress has been made in defining the potential functions of its human homolog, SRC.  The product of the human SRC gene, c-Src, is found to be over-expressed and highly activated in a wide variety of human cancers.  The relationship between Src activation and cancer progression appears to be significant.  Moreover, Src may have an influence on the development of the metastatic phenotype.  This review discusses the data supporting a role for c-Src as a crit. component of the signal transduction pathways that control cancer cell development and growth, and provides the rationale for targeting Src in drug discovery efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAGCSyZ76JqLVg90H21EOLACvtfcHk0ljZvhAL3Hj9Sg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXosl2nurs%253D&md5=1ec800320d1ae4976eb1c93af743cf6b</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1203912&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1203912%26sid%3Dliteratum%253Aachs%26aulast%3DIrby%26aufirst%3DR.%2BB.%26aulast%3DYeatman%26aufirst%3DT.%2BJ.%26atitle%3DRole%2520of%2520Src%2520expression%2520and%2520activation%2520in%2520human%2520cancer%26jtitle%3DOncogene%26date%3D2000%26volume%3D19%26spage%3D5636%26doi%3D10.1038%2Fsj.onc.1203912" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brugge, J. S.</span></span> <span> </span><span class="NLM_article-title">Cellular functions regulated by Src family kinases</span>. <i>Annu. Rev. Cell Dev. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">513</span>– <span class="NLM_lpage">609</span>, <span class="refDoi"> DOI: 10.1146/annurev.cellbio.13.1.513</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.1146%2Fannurev.cellbio.13.1.513" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=9442882" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADyaK1cXisFSrsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=1997&pages=513-609&author=S.+M.+Thomasauthor=J.+S.+Brugge&title=Cellular+functions+regulated+by+Src+family+kinases&doi=10.1146%2Fannurev.cellbio.13.1.513"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Cellular functions regulated by Src family kinases</span></div><div class="casAuthors">Thomas, Sheila M.; Brugge, Joan S.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Cell and Developmental Biology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">513-609</span>CODEN:
                <span class="NLM_cas:coden">ARDBF8</span>;
        ISSN:<span class="NLM_cas:issn">1081-0706</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review with many refs.  Src family protein tyrosine kinases are activated following engagement of many different classes of cellular receptors and participate in signaling pathways that control a diverse spectrum of receptor-induced biol. activities.  While several of these kinases have evolved to play distinct roles in specific receptor pathways, there is considerable redundancy in the functions of these kinases, both with respect to the receptor pathways that activate these kinases and the downstream effectors that mediate their biol. activities.  This chapter reviews the evidence implicating Src family kinases in specific receptor pathways and describes the mechanisms leading to their activation, the targets that interact with these kinases, and the biol. events that they regulate.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_psatEFVwjbVg90H21EOLACvtfcHk0ljZvhAL3Hj9Sg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXisFSrsQ%253D%253D&md5=4b8d18cc815109e3e0a593b846984aac</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1146%2Fannurev.cellbio.13.1.513&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.cellbio.13.1.513%26sid%3Dliteratum%253Aachs%26aulast%3DThomas%26aufirst%3DS.%2BM.%26aulast%3DBrugge%26aufirst%3DJ.%2BS.%26atitle%3DCellular%2520functions%2520regulated%2520by%2520Src%2520family%2520kinases%26jtitle%3DAnnu.%2520Rev.%2520Cell%2520Dev.%2520Biol.%26date%3D1997%26volume%3D13%26spage%3D513%26epage%3D609%26doi%3D10.1146%2Fannurev.cellbio.13.1.513" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Finn, R.</span></span> <span> </span><span class="NLM_article-title">Targeting Src in breast cancer</span>. <i>Annals of Oncology</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">1379</span>– <span class="NLM_lpage">1386</span>, <span class="refDoi"> DOI: 10.1093/annonc/mdn291</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.1093%2Fannonc%2Fmdn291" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=18487549" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A280%3ADC%252BD1cvotV2rsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2008&pages=1379-1386&author=R.+Finn&title=Targeting+Src+in+breast+cancer&doi=10.1093%2Fannonc%2Fmdn291"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Src in breast cancer</span></div><div class="casAuthors">Finn R S</div><div class="citationInfo"><span class="NLM_cas:title">Annals of oncology : official journal of the European Society for Medical Oncology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1379-86</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The clinical benefit of blocking oncogenic pathways in breast cancer and other malignancies has validated this approach and ushered in the era of molecularly targeted therapeutics.  Src and its family members make up the largest group of nonreceptor tyrosine kinases.  In laboratory models, these proteins have been shown to play a critical role in cellular growth and proliferation, angiogenesis, and invasion and metastasis.  In addition, Src plays an important role in osteoclast activation and bone resorption, which are often aberrantly activated in the setting of bone metastases.  Given its role in these functions, blocking Src kinase would be predicted to have a broad therapeutic benefit in patients with Src-dependent cancers.  In this review, we highlight the rationale for targeting Src in breast cancer, including laboratory and clinical data implicating it in these signaling pathways, and review small-molecule tyrosine kinase inhibitors currently in clinical development.  Identifying which patients should be selected for Src-directed therapies will be important to the clinical success of these agents.  Importantly, recent preclinical data support a role for this class of inhibitors in basal-type/triple-negative breast cancer, which represents a group of patients with limited effective treatment options.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSq3-Zi9cm6LUNHe_gFldTbfW6udTcc2ebm8fQLZYTEeLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1cvotV2rsw%253D%253D&md5=b4f29b2ada4dcb6c89c0301a8440c617</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1093%2Fannonc%2Fmdn291&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fannonc%252Fmdn291%26sid%3Dliteratum%253Aachs%26aulast%3DFinn%26aufirst%3DR.%26atitle%3DTargeting%2520Src%2520in%2520breast%2520cancer%26jtitle%3DAnnals%2520of%2520Oncology%26date%3D2008%26volume%3D19%26spage%3D1379%26epage%3D1386%26doi%3D10.1093%2Fannonc%2Fmdn291" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dehm, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonham, K.</span></span> <span> </span><span class="NLM_article-title">SRC gene expression in human cancer: the role of transcriptional activation</span>. <i>Biochem. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>82</i></span>,  <span class="NLM_fpage">263</span>– <span class="NLM_lpage">274</span>, <span class="refDoi"> DOI: 10.1139/o03-077</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.1139%2Fo03-077" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=15060621" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjvFChsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=82&publication_year=2004&pages=263-274&author=S.+M.+Dehmauthor=K.+Bonham&title=SRC+gene+expression+in+human+cancer%3A+the+role+of+transcriptional+activation&doi=10.1139%2Fo03-077"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">SRC gene expression in human cancer: The role of transcriptional activation</span></div><div class="casAuthors">Dehm, Scott M.; Bonham, Keith</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry and Cell Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">82</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">263-274</span>CODEN:
                <span class="NLM_cas:coden">BCBIEQ</span>;
        ISSN:<span class="NLM_cas:issn">0829-8211</span>.
    
            (<span class="NLM_cas:orgname">National Research Council of Canada</span>)
        </div><div class="casAbstract">A review.  Human pp60c-Src (or c-Src) is a 60 kDa non-receptor tyrosine kinase encoded by the SRC gene and is the cellular homolog to the potent transforming v-Src viral oncogene.  C-Src functions at the hub of a vast array of signal transduction cascades that influence cellular proliferation, differentiation, motility, and survival.  C-Src activation has been documented in upwards of 50% of tumors derived from the colon, liver, lung, breast, and pancreas.  Therefore, a major focus has been to understand the mechanisms of c-Src activation in human cancer.  Early studies concd. on post-translational mechanisms that lead to increased c-Src kinase activity, which often correlated with overexpression of c-Src protein.  More recently, the discovery of an activating SRC mutation in a small subset of advanced colon tumors has been reported.  In addn., elevated SRC transcription has been identified as yet another mechanism contributing significantly to c-Src activation in a subset of human colon cancer cell lines.  Interestingly, histone deacetylase (HDAC) inhibitors, agents with well-documented anti-cancer activity, repress SRC transcription in a wide variety of human cancer cell lines.  Anal. of the mechanisms behind HDAC inhibitor mediated repression could be utilized in the future to specifically inhibit SRC gene expression in human cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoi4CZWd0vRL7Vg90H21EOLACvtfcHk0ljZvhAL3Hj9Sg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjvFChsb8%253D&md5=3693d1b64959feb8218339c72e2c7b25</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1139%2Fo03-077&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1139%252Fo03-077%26sid%3Dliteratum%253Aachs%26aulast%3DDehm%26aufirst%3DS.%2BM.%26aulast%3DBonham%26aufirst%3DK.%26atitle%3DSRC%2520gene%2520expression%2520in%2520human%2520cancer%253A%2520the%2520role%2520of%2520transcriptional%2520activation%26jtitle%3DBiochem.%2520Cell%2520Biol.%26date%3D2004%26volume%3D82%26spage%3D263%26epage%3D274%26doi%3D10.1139%2Fo03-077" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koul, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMurray, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yung, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siminovitch, K. A.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Src family protein-tyrosine kinases alter the function of PTEN to regulate phosphatidylinositol 3-kinase/AKT cascades</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>278</i></span>,  <span class="NLM_fpage">40057</span>– <span class="NLM_lpage">40066</span>, <span class="refDoi"> DOI: 10.1074/jbc.M303621200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.1074%2Fjbc.M303621200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=12869565" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BD3sXnvFamsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=278&publication_year=2003&pages=40057-40066&author=Y.+Luauthor=Q.+Yuauthor=J.+H.+Liuauthor=J.+Zhangauthor=H.+Wangauthor=D.+Koulauthor=J.+S.+McMurrayauthor=X.+Fangauthor=W.+A.+Yungauthor=K.+A.+Siminovitch&title=Src+family+protein-tyrosine+kinases+alter+the+function+of+PTEN+to+regulate+phosphatidylinositol+3-kinase%2FAKT+cascades&doi=10.1074%2Fjbc.M303621200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Src Family Protein-tyrosine Kinases Alter the Function of PTEN to Regulate Phosphatidylinositol 3-Kinase/AKT Cascades</span></div><div class="casAuthors">Lu, Yiling; Yu, Qinghua; Liu, Jue Hui; Zhang, Jinyi; Wang, Hongwei; Koul, Dimpy; McMurray, John S.; Fang, Xianjun; Yung, W. K. Alfred; Siminovitch, Kathy A.; Mills, Gordon B.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">278</span>
        (<span class="NLM_cas:issue">41</span>),
    <span class="NLM_cas:pages">40057-40066</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Src family protein-tyrosine kinases, which play an important role in signal integration, have been implicated in tumorigenesis in multiple lineages, including breast cancer.  We demonstrate, herein, that Src kinases regulate the phosphatidylinositol 3-kinase (PI3K) signaling cascade via altering the function of the PTEN tumor suppressor.  Overexpression of activated Src protein-tyrosine kinases in PTEN-deficient breast cancer cells does not alter AKT phosphorylation, an indicator of signal transduction through the PI3K pathway.  However, in the presence of functional PTEN, Src reverses the activity of PTEN, resulting in an increase in AKT phosphorylation.  Activated Src reduces the ability of PTEN to dephosphorylate phosphatidylinositols in micelles and promotes AKT translocation to cellular plasma membranes but does not alter PTEN activity toward water-sol. phosphatidylinositols.  Thus, Src may alter the capacity of the PTEN C2 domain to bind cellular membranes rather than directly interfering with PTEN enzymic activity.  Tyrosine phosphorylation of PTEN is increased in breast cancer cells treated with pervanadate, suggesting that PTEN contains sites for tyrosine phosphorylation.  Src kinase inhibitors markedly decreased pervanadate-mediated tyrosine phosphorylation of PTEN.  Further, expression of activated Src results in marked tyrosine phosphorylation of PTEN.  SHP-1, a SH2 domain-contg. protein-tyrosine phosphatase, selectively binds and dephosphorylates PTEN in Src transfected cells.  Both Src inhibitors and SHP-1 overexpression reverse Src-induced loss of PTEN function.  Coexpression of PTEN with activated Src reduces the stability of PTEN.  Taken together, the data indicate that activated Src inhibits PTEN function leading to alterations in signaling through the PI3K/AKT pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWFNxCQ8gMqLVg90H21EOLACvtfcHk0ljbe9i8TF0zgw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXnvFamsLs%253D&md5=2696aba274ab4ee0f85736f0f7b8344d</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M303621200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M303621200%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DY.%26aulast%3DYu%26aufirst%3DQ.%26aulast%3DLiu%26aufirst%3DJ.%2BH.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DKoul%26aufirst%3DD.%26aulast%3DMcMurray%26aufirst%3DJ.%2BS.%26aulast%3DFang%26aufirst%3DX.%26aulast%3DYung%26aufirst%3DW.%2BA.%26aulast%3DSiminovitch%26aufirst%3DK.%2BA.%26atitle%3DSrc%2520family%2520protein-tyrosine%2520kinases%2520alter%2520the%2520function%2520of%2520PTEN%2520to%2520regulate%2520phosphatidylinositol%25203-kinase%252FAKT%2520cascades%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2003%26volume%3D278%26spage%3D40057%26epage%3D40066%26doi%3D10.1074%2Fjbc.M303621200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cantley, L. C.</span></span> <span> </span><span class="NLM_article-title">The phosphoinositide 3-kinase pathway</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>296</i></span>,  <span class="NLM_fpage">1655</span>– <span class="NLM_lpage">1657</span>, <span class="refDoi"> DOI: 10.1126/science.296.5573.1655</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.1126%2Fscience.296.5573.1655" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=12040186" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BD38XktlChu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=296&publication_year=2002&pages=1655-1657&author=L.+C.+Cantley&title=The+phosphoinositide+3-kinase+pathway&doi=10.1126%2Fscience.296.5573.1655"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">The phosphoinositide 3-kinase pathway</span></div><div class="casAuthors">Cantley, Lewis C.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">296</span>
        (<span class="NLM_cas:issue">5573</span>),
    <span class="NLM_cas:pages">1655-1657</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A review with 8 refs.  Phosphorylated lipids are produced at cellular membranes during signaling events and contribute to the recruitment and activation of various signaling components.  The role of phosphoinositide 3-kinase (PI3K), which catalyzes the prodn. of phosphatidylinositol 3,4,5-trisphosphate, in cell survival pathways; the regulation of gene expression and cell metab.; and cytoskeletal rearrangements are highlighted.  The PI3K pathway is implicated in human diseases including diabetes and cancer, and understanding the intricacies of this pathway may provide new avenues for therapeutic intervention.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9pxSplxmS1rVg90H21EOLACvtfcHk0ljbe9i8TF0zgw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XktlChu7s%253D&md5=110c8a351448aed6ab938b9ccf90a655</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1126%2Fscience.296.5573.1655&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.296.5573.1655%26sid%3Dliteratum%253Aachs%26aulast%3DCantley%26aufirst%3DL.%2BC.%26atitle%3DThe%2520phosphoinositide%25203-kinase%2520pathway%26jtitle%3DScience%26date%3D2002%26volume%3D296%26spage%3D1655%26epage%3D1657%26doi%3D10.1126%2Fscience.296.5573.1655" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lai, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, F. M.</span></span> <span> </span><span class="NLM_article-title">Defining the role of the JAK-STAT pathway in head and neck and thoracic malignancies: implications for future therapeutic approaches</span>. <i>Drug Resist. Updates</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">67</span>– <span class="NLM_lpage">78</span>, <span class="refDoi"> DOI: 10.1016/j.drup.2010.04.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.1016%2Fj.drup.2010.04.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=20471303" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BC3cXntVSmur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2010&pages=67-78&author=S.+Y.+Laiauthor=F.+M.+Johnson&title=Defining+the+role+of+the+JAK-STAT+pathway+in+head+and+neck+and+thoracic+malignancies%3A+implications+for+future+therapeutic+approaches&doi=10.1016%2Fj.drup.2010.04.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Defining the role of the JAK-STAT pathway in head and neck and thoracic malignancies: Implications for future therapeutic approaches</span></div><div class="casAuthors">Lai, Stephen Y.; Johnson, Faye M.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Resistance Updates</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">67-78</span>CODEN:
                <span class="NLM_cas:coden">DRUPFW</span>;
        ISSN:<span class="NLM_cas:issn">1368-7646</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Although the role of the Janus kinase (JAK)-signal transducer and activator of transcription (STAT) pathway has been most extensively studied in hematopoietic cells and hematol. malignancies, it is also activated in epithelial tumors, including those originating in the lungs and head and neck.  The canonical pathway involves the activation of JAK following ligand binding to cytokine receptors.  The activated JAKs then phosphorylate STAT proteins, leading to their dimerization and translocation into the nucleus.  In the nucleus, STATs act as transcription factors with pleiotropic downstream effects.  STATs can be activated independently of JAKs, most notably by c-Src kinases.  In cancer cells, STAT3 and STAT5 activation leads to the increased expression of downstream target genes, leading to increased cell proliferation, cell survival, angiogenesis, and immune system evasion.  STAT3 and STAT5 are expressed and activated in head and neck squamous cell carcinoma (HNSCC) where they contribute to cell survival and proliferation.  In HNSCC, STATs can be activated by a no. of signal transduction pathways, including the epidermal growth factor receptor (EGFR), α7 nicotinic receptor, interleukin (IL) receptor, and erythropoietin receptor pathways.  Activated STATs are also expressed in lung cancer, but the biol. effects of JAK/STAT inhibition in this cancer are variable.  In lung cancer, STAT3 can be activated by multiple pathways, including EGFR.  Several approaches have been used to inhibit STAT3 in the hopes of developing an antitumor agent.  Although several STAT3-specific agents are promising, none are in clin. development, mostly because of drug delivery and stability issues.  In contrast, several JAK inhibitors are in clin. development.  These orally available, ATP-competitive, small-mol. kinase inhibitors are being tested in myeloproliferative disorders.  Future studies will det. whether JAK inhibitors are useful in the treatment of HNSCC or lung cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZgZHT6MxwX7Vg90H21EOLACvtfcHk0ljbe9i8TF0zgw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXntVSmur8%253D&md5=76183fde53da20b0a15bb4f000feb8f5</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.drup.2010.04.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drup.2010.04.001%26sid%3Dliteratum%253Aachs%26aulast%3DLai%26aufirst%3DS.%2BY.%26aulast%3DJohnson%26aufirst%3DF.%2BM.%26atitle%3DDefining%2520the%2520role%2520of%2520the%2520JAK-STAT%2520pathway%2520in%2520head%2520and%2520neck%2520and%2520thoracic%2520malignancies%253A%2520implications%2520for%2520future%2520therapeutic%2520approaches%26jtitle%3DDrug%2520Resist.%2520Updates%26date%3D2010%26volume%3D13%26spage%3D67%26epage%3D78%26doi%3D10.1016%2Fj.drup.2010.04.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maa, M.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leu, T.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trandel, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, J.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parsons, S. J.</span></span> <span> </span><span class="NLM_article-title">A protein that is highly related to GTPase-activating protein-associated p62 complexes with phospholipase C gamma</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">5466</span>– <span class="NLM_lpage">5473</span>, <span class="refDoi"> DOI: 10.1128/MCB.14.8.5466</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.1128%2FMCB.14.8.5466" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=7518563" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADyaK2cXlvF2hsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=1994&pages=5466-5473&author=M.-C.+Maaauthor=T.-H.+Leuauthor=B.+J.+Trandelauthor=J.-H.+Changauthor=S.+J.+Parsons&title=A+protein+that+is+highly+related+to+GTPase-activating+protein-associated+p62+complexes+with+phospholipase+C+gamma&doi=10.1128%2FMCB.14.8.5466"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">A protein that is highly related to GTPase-activating protein-associated p62 complexes with phospholipase Cγ</span></div><div class="casAuthors">Maa, Ming-Chei; Leu, Tzeng-Horng; Trandel, Barbara J.; Chang, Jin-Hong; Parsons, Sarah J.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">5466-73</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    </div><div class="casAbstract">P62 is a highly tyrosyl phosphorylated protein that was first identified in immunoppts. of the GTPase-activating protein (GAP) of p21ras from cells transformed by oncogenic nonreceptor tyrosine kinases or stimulated through tyrosine kinase receptors (C. Ellis, M. Moran, F. McCormick, and T. Pawson, Nature 343:377-381, 1991).  In this article we describe a highly related 62-kDa protein that becomes tyrosyl phosphorylated and assocd. with phospholipase Cγ (PLCγ) in C3H10T1/2 cells stimulated with epidermal growth factor (EGF) or transformed by v-src.  GAP-assocd. and PLCγ-assocd. p62 comigrated in one-dimensional sodium dodecyl sulfate-polyacrylamide gel electrophoresis and exhibited nearly identical phosphotryptic peptide patterns.  That the assocn. of p62 with PLCγ was direct and not mediated through binding of GAP-p62 to PLCγ or to the EGF receptor (and copptn. of the receptor with PLCγ) was demonstrated by (i) the inability to detect GAP in PLCγ immunocomplexes or PLCγ in GAP immunocomplexes, (ii) the assocn. of p62 with PLCγ in v-src-transformed cells in the absence of EGF stimulation, and (iii) in vitro soln. binding and direct blotting of p62 with a glutathione S-transferase fusion protein contg. the Src homol. 2 (SH2) domains of PLCγ.  Unlike GAP, whose N-terminal SH2 mediates the interaction between GAP and p62, PLCγ was found to require both its N- and C-terminal SH2 regions for p62 binding.  These studies demonstrate that a protein identical to or highly related to GAP-assocd. p62 binds PLCγ and suggest a means by which "cross-talk" between PLCγ- and GAP-mediated signalling may occur.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_R-OwJ8Bl-LVg90H21EOLACvtfcHk0ljB3IY9p-0xRg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXlvF2hsbg%253D&md5=700f05d42b6812844eaed9c8a44c8855</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1128%2FMCB.14.8.5466&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.14.8.5466%26sid%3Dliteratum%253Aachs%26aulast%3DMaa%26aufirst%3DM.-C.%26aulast%3DLeu%26aufirst%3DT.-H.%26aulast%3DTrandel%26aufirst%3DB.%2BJ.%26aulast%3DChang%26aufirst%3DJ.-H.%26aulast%3DParsons%26aufirst%3DS.%2BJ.%26atitle%3DA%2520protein%2520that%2520is%2520highly%2520related%2520to%2520GTPase-activating%2520protein-associated%2520p62%2520complexes%2520with%2520phospholipase%2520C%2520gamma%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D1994%26volume%3D14%26spage%3D5466%26epage%3D5473%26doi%3D10.1128%2FMCB.14.8.5466" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ishizawar, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyake, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parsons, S.</span></span> <span> </span><span class="NLM_article-title">c-Src modulates ErbB2 and ErbB3 heterocomplex formation and function</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">3503</span>, <span class="refDoi"> DOI: 10.1038/sj.onc.1210138</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.1038%2Fsj.onc.1210138" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=17173075" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BD2sXls1Gqt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2007&pages=3503&author=R.+Ishizawarauthor=T.+Miyakeauthor=S.+Parsons&title=c-Src+modulates+ErbB2+and+ErbB3+heterocomplex+formation+and+function&doi=10.1038%2Fsj.onc.1210138"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">c-Src modulates ErbB2 and ErbB3 heterocomplex formation and function</span></div><div class="casAuthors">Ishizawar, R. C.; Miyake, T.; Parsons, S. J.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">3503-3510</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Overexpression and/or gene amplification of c-Src and members of the epidermal growth factor receptor (EGFR/ErbB) family have been implicated in the pathogenesis of breast cancer.  Although members of the EGFR family are known to form heterocomplexes with one another, c-Src has also been shown to phys. interact with members of this family in breast cancer cell lines and tumors.  This paper investigates the role of c-Src in modulating the phys. and functional interaction between ErbB2 and ErbB3, two family members that preferentially assoc. with one another and together exhibit high oncogenic potential.  We show that overexpressed wild-type c-Src enhances heterocomplex formation of ErbB2 and ErbB3 that results in increased basal and/or heregulin-induced activation of receptors, and their downstream intracellular effectors.  Expression of a kinase-inactive form of c-Src (K- c-Src) or pharmacol. inhibition of c-Src by PP2 neg. affects these events.  Furthermore, cellular motility and anchorage-independent growth promoted by the ErbB2/ErbB3 heterocomplex are dependent upon c-Src, as demonstrated by the effects of K- c-Src overexpression or treatment with PP2.  In contrast to previous studies that defined a role for c-Src downstream of ErbB2/ErbB3, the current work suggests an upstream mechanism, whereby c-Src enhances ErbB2/ErbB3 signaling and biol. functions by pos. modulating the assocn. between ErbB2 and ErbB3.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq78qm6P_s_hLVg90H21EOLACvtfcHk0ljB3IY9p-0xRg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXls1Gqt7g%253D&md5=9894b8815d113e7a7b5ec9d127a66bcf</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1210138&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1210138%26sid%3Dliteratum%253Aachs%26aulast%3DIshizawar%26aufirst%3DR.%26aulast%3DMiyake%26aufirst%3DT.%26aulast%3DParsons%26aufirst%3DS.%26atitle%3Dc-Src%2520modulates%2520ErbB2%2520and%2520ErbB3%2520heterocomplex%2520formation%2520and%2520function%26jtitle%3DOncogene%26date%3D2007%26volume%3D26%26spage%3D3503%26doi%3D10.1038%2Fsj.onc.1210138" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">López-Ocejo, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viloria-Petit, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bequet-Romero, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mukhopadhyay, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rak, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerbel, R. S.</span></span> <span> </span><span class="NLM_article-title">Oncogenes and tumor angiogenesis: the HPV-16 E6 oncoprotein activates the vascular endothelial growth factor (VEGF) gene promoter in a p53 independent manner</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">4611</span>, <span class="refDoi"> DOI: 10.1038/sj.onc.1203817</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.1038%2Fsj.onc.1203817" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=11030150" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BD3cXnsFensLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2000&pages=4611&author=O.+L%C3%B3pez-Ocejoauthor=A.+Viloria-Petitauthor=M.+Bequet-Romeroauthor=D.+Mukhopadhyayauthor=J.+Rakauthor=R.+S.+Kerbel&title=Oncogenes+and+tumor+angiogenesis%3A+the+HPV-16+E6+oncoprotein+activates+the+vascular+endothelial+growth+factor+%28VEGF%29+gene+promoter+in+a+p53+independent+manner&doi=10.1038%2Fsj.onc.1203817"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Oncogenes and tumor angiogenesis: the HPV-16 E6 oncoprotein activates the vascular endothelial growth factor (VEGF) gene promoter in a p53 independent manner</span></div><div class="casAuthors">Lopez-Ocejo, Omar; Viloria-Petit, Alicia; Bequet-Romero, Monica; Mukhopadhyay, Debabrata; Rak, Janusz; Kerbel, Robert S.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">40</span>),
    <span class="NLM_cas:pages">4611-4620</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Like other types of pre-malignant lesions and carcinoma, angiogenesis is assocd. with high-grade cervical dysplasia and with invasive squamous carcinoma of the cervix.  Vascular endothelial cell growth factor (VEGF) is known to be one of the most important inducers of angiogenesis and is upregulated in carcinoma of the cervix.  Human Papilloma Virus 16 (HPV-16) has been etiol. linked to human cervical cancer, and the major oncogenic proteins encoded by the viral genome, E6 and E7, are involved in the immortalization of target cells.  Because several oncogenes including mutant ras, EGF receptor, ErbB2/Her2, c-myc and v-src upregulate VEGF expression, the authors asked whether HVP-16 E6 oncoprotein could act in a similar fashion.  The authors found that HPV-16 E6-pos. cells generally express high levels of VEGF message.  Furthermore, co-expression of the VEGF promoter-Luc (luciferase) reporter gene with E6 in both human keratinocytes and mouse fibroblast showed that E6 oncoprotein upregulates VEGF promoter activity, and does so in a p53 independent manner.  An E6 responsive region which comprises four Sp-1 sites, between -194 and -50 bp of the VEGF promoter, is also necessary for constitutive VEGF transcription.  Taken together, the authors' results suggest the possibility that the HPV oncoprotein E6 may contribute to tumor angiogenesis by direct stimulation of the VEGF gene.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprVYPMZxLxp7Vg90H21EOLACvtfcHk0ljB3IY9p-0xRg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXnsFensLs%253D&md5=7e7b641648490b2e9aa6dcc7f65cfaa5</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1203817&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1203817%26sid%3Dliteratum%253Aachs%26aulast%3DL%25C3%25B3pez-Ocejo%26aufirst%3DO.%26aulast%3DViloria-Petit%26aufirst%3DA.%26aulast%3DBequet-Romero%26aufirst%3DM.%26aulast%3DMukhopadhyay%26aufirst%3DD.%26aulast%3DRak%26aufirst%3DJ.%26aulast%3DKerbel%26aufirst%3DR.%2BS.%26atitle%3DOncogenes%2520and%2520tumor%2520angiogenesis%253A%2520the%2520HPV-16%2520E6%2520oncoprotein%2520activates%2520the%2520vascular%2520endothelial%2520growth%2520factor%2520%2528VEGF%2529%2520gene%2520promoter%2520in%2520a%2520p53%2520independent%2520manner%26jtitle%3DOncogene%26date%3D2000%26volume%3D19%26spage%3D4611%26doi%3D10.1038%2Fsj.onc.1203817" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, D.</span></span> <span> </span><span class="NLM_article-title">Targeting Src family kinases in anti-cancer therapies: turning promise into triumph</span>. <i>Trends Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">122</span>– <span class="NLM_lpage">128</span>, <span class="refDoi"> DOI: 10.1016/j.tips.2011.11.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.1016%2Fj.tips.2011.11.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=22153719" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BC38XjtlOqtrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2012&pages=122-128&author=S.+Zhangauthor=D.+Yu&title=Targeting+Src+family+kinases+in+anti-cancer+therapies%3A+turning+promise+into+triumph&doi=10.1016%2Fj.tips.2011.11.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Src family kinases in anti-cancer therapies: turning promise into triumph</span></div><div class="casAuthors">Zhang, Siyuan; Yu, Dihua</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">122-128</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Src is a non-receptor tyrosine kinase that is deregulated in many types of cancer.  Decades of research have revealed the crucial role of Src in many aspects of tumor development, including proliferation, survival, adhesion, migration, invasion and, most importantly, metastasis, in multiple tumor types.  Despite extensive preclin. evidence that warrants targeting Src as a promising therapeutic approach for cancer, Src inhibitor(s) showed only minimal therapeutic activity in various types of solid tumors when used as a single agent in recent early-phase clin. trials.  In this review, we highlight the most recent advances from preclin. studies and clin. trials that shed light on potential clin. use of Src inhibitor-contg. combinatorial regimens in overcoming resistance to current anticancer therapies and in preventing metastatic recurrence.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZvQcvwj_23bVg90H21EOLACvtfcHk0lgE4mO2bVqjWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjtlOqtrk%253D&md5=22c9483e90ec6b390f42b74806d87d21</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2011.11.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2011.11.002%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DYu%26aufirst%3DD.%26atitle%3DTargeting%2520Src%2520family%2520kinases%2520in%2520anti-cancer%2520therapies%253A%2520turning%2520promise%2520into%2520triumph%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2012%26volume%3D33%26spage%3D122%26epage%3D128%26doi%3D10.1016%2Fj.tips.2011.11.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boland, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erdogan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasmatzis, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tillmans, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, M. R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halling, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliveira, A. M.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Anaplastic lymphoma kinase immunoreactivity correlates with ALK gene rearrangement and transcriptional up-regulation in non–small cell lung carcinomas</span>. <i>Hum. Pathol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">1152</span>– <span class="NLM_lpage">1158</span>, <span class="refDoi"> DOI: 10.1016/j.humpath.2009.01.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.1016%2Fj.humpath.2009.01.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=19386350" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BD1MXoslCrt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2009&pages=1152-1158&author=J.+M.+Bolandauthor=S.+Erdoganauthor=G.+Vasmatzisauthor=P.+Yangauthor=L.+S.+Tillmansauthor=M.+R.+E.+Johnsonauthor=X.+Wangauthor=L.+M.+Petersonauthor=K.+C.+Hallingauthor=A.+M.+Oliveira&title=Anaplastic+lymphoma+kinase+immunoreactivity+correlates+with+ALK+gene+rearrangement+and+transcriptional+up-regulation+in+non%E2%80%93small+cell+lung+carcinomas&doi=10.1016%2Fj.humpath.2009.01.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Anaplastic lymphoma kinase immunoreactivity correlates with ALK gene rearrangement and transcriptional up-regulation in non-small cell lung carcinomas</span></div><div class="casAuthors">Boland, Jennifer M.; Erdogan, Sibel; Vasmatzis, George; Yang, Ping; Tillmans, Lori S.; Johnson, Michele R. Erickson; Wang, Xiaoke; Peterson, Lisa M.; Halling, Kevin C.; Oliveira, Andre M.; Aubry, Marie Christine; Yi, Eunhee S.</div><div class="citationInfo"><span class="NLM_cas:title">Human Pathology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1152-1158</span>CODEN:
                <span class="NLM_cas:coden">HPCQA4</span>;
        ISSN:<span class="NLM_cas:issn">0046-8177</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Recently, the fusion gene EML4-ALK was identified in non-small cell lung carcinoma, which could be a potential therapeutic target.  We investigated the prevalence of anaplastic lymphoma kinase protein expression in these tumors by immunohistochem. and correlated the results with data from ALK mol. studies.  Gene expression profiling was performed on 35 adenocarcinomas to identify cases with ALK gene up-regulation, which was correlated with protein overexpression by immunohistochem.  Immunohistochem. was also performed on an independent cohort consisting of 150 adenocarcinomas and 150 squamous cell carcinomas to evaluate the utility of anaplastic lymphoma kinase immunostaining as a screening tool.  Florescence in situ hybridization for the ALK locus and reverse transcriptase-polymerase chain reaction for EML4-ALK were performed on tumors pos. for anaplastic lymphoma kinase by immunohistochem.  Transcriptional up-regulation of ALK was identified in 2 (6%) of 35 adenocarcinomas by gene expression profiling.  These 2 cases were pos. for anaplastic lymphoma kinase by immunohistochem., whereas the remaining 33 cases were completely neg.  In the independent cohort, anaplastic lymphoma kinase immunostaining was pos. in 1 of 150 squamous cell carcinomas and in 3 of 150 adenocarcinomas.  The 6 cases pos. for anaplastic lymphoma kinase by immunohistochem. showed evidence of ALK locus rearrangement by florescence in situ hybridization but were neg. for EGFR and KRAS mutation.  The presence of EML4-ALK fusion transcript was confirmed in 2 cases by reverse transcriptase-polymerase chain reaction.  In conclusion, anaplastic lymphoma kinase immunoreactivity in non-small cell lung carcinomas was assocd. with transcriptional up-regulation, ALK locus rearrangement, and the presence of EML4-ALK fusion transcript.  Anaplastic lymphoma kinase immunohistochem. may have utility as a screening tool or as a surrogate marker for the mol. techniques to detect the EML4-ALK fusion gene in these tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmuPu4P5jd7LVg90H21EOLACvtfcHk0lgE4mO2bVqjWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXoslCrt7w%253D&md5=787bd122ec1efff691f1095dfdead574</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.humpath.2009.01.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.humpath.2009.01.012%26sid%3Dliteratum%253Aachs%26aulast%3DBoland%26aufirst%3DJ.%2BM.%26aulast%3DErdogan%26aufirst%3DS.%26aulast%3DVasmatzis%26aufirst%3DG.%26aulast%3DYang%26aufirst%3DP.%26aulast%3DTillmans%26aufirst%3DL.%2BS.%26aulast%3DJohnson%26aufirst%3DM.%2BR.%2BE.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DPeterson%26aufirst%3DL.%2BM.%26aulast%3DHalling%26aufirst%3DK.%2BC.%26aulast%3DOliveira%26aufirst%3DA.%2BM.%26atitle%3DAnaplastic%2520lymphoma%2520kinase%2520immunoreactivity%2520correlates%2520with%2520ALK%2520gene%2520rearrangement%2520and%2520transcriptional%2520up-regulation%2520in%2520non%25E2%2580%2593small%2520cell%2520lung%2520carcinomas%26jtitle%3DHum.%2520Pathol.%26date%3D2009%26volume%3D40%26spage%3D1152%26epage%3D1158%26doi%3D10.1016%2Fj.humpath.2009.01.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Muthuswamy, S. K.</span></span> <span> </span><span class="NLM_article-title">Trastuzumab resistance: all roads lead to SRC</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">416</span>, <span class="refDoi"> DOI: 10.1038/nm0411-416</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.1038%2Fnm0411-416" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=21475230" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BC3MXksVCqsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2011&pages=416&author=S.+K.+Muthuswamy&title=Trastuzumab+resistance%3A+all+roads+lead+to+SRC&doi=10.1038%2Fnm0411-416"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Trastuzumab resistance: all roads lead to SRC</span></div><div class="casAuthors">Muthuswamy, Senthil K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">416-418</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  A new study shows how SRC, a nonmembrane tyrosine kinase, is a common signaling node in trastuzumab resistance caused by different mechanisms in HER2-pos. breast cancers (pages 461-469).  A SRC inhibitor restored trastuzumab sensitivity in vitro and in mouse tumor models, suggesting a new way to tackle drug resistance in breast tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsBPd2RG0rHbVg90H21EOLACvtfcHk0lgE4mO2bVqjWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXksVCqsrk%253D&md5=3617bbb8b395688050cff37927c71278</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fnm0411-416&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm0411-416%26sid%3Dliteratum%253Aachs%26aulast%3DMuthuswamy%26aufirst%3DS.%2BK.%26atitle%3DTrastuzumab%2520resistance%253A%2520all%2520roads%2520lead%2520to%2520SRC%26jtitle%3DNat.%2520Med.%26date%3D2011%26volume%3D17%26spage%3D416%26doi%3D10.1038%2Fnm0411-416" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yoshida, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sasaki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minami, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hibino, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okumura, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sado, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyokawa, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayashi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kitada, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohsaki, Y.</span></span> <span> </span><span class="NLM_article-title">Activation of Src signaling mediates acquired resistance to ALK inhibition in lung cancer</span>. <i>Int. J. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">1533</span>– <span class="NLM_lpage">1540</span>, <span class="refDoi"> DOI: 10.3892/ijo.2017.4140</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.3892%2Fijo.2017.4140" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=29048652" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVKjurjP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2017&pages=1533-1540&author=R.+Yoshidaauthor=T.+Sasakiauthor=Y.+Minamiauthor=Y.+Hibinoauthor=S.+Okumuraauthor=M.+Sadoauthor=N.+Miyokawaauthor=S.+Hayashiauthor=M.+Kitadaauthor=Y.+Ohsaki&title=Activation+of+Src+signaling+mediates+acquired+resistance+to+ALK+inhibition+in+lung+cancer&doi=10.3892%2Fijo.2017.4140"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Activation of Src signaling mediates acquired resistance to ALK inhibition in lung cancer</span></div><div class="casAuthors">Yoshida, Ryohei; Sasaki, Takaaki; Minami, Yoshinori; Hibino, Yukiko; Okumura, Shunsuke; Sado, Masatoshi; Miyokawa, Naoyuki; Hayashi, Satoshi; Kitada, Masahiro; Ohsaki, Yoshinobu</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Oncology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1533-1540</span>CODEN:
                <span class="NLM_cas:coden">IJONES</span>;
        ISSN:<span class="NLM_cas:issn">1791-2423</span>.
    
            (<span class="NLM_cas:orgname">Spandidos Publications Ltd.</span>)
        </div><div class="casAbstract">Anaplastic lymphoma kinase (ALK) fusion oncogenes occur in approx. 3-5% of non-small cell lung cancer (NSCLC) cases.  Various ALK inhibitors are in clin. use for the treatment of ALK-NSCLC, including the first generation ALK inhibitor, crizotinib, and recently the more highly potent alectinib and ceritinib.  However, most tumors eventually become resistant to ALK specific inhibitors.  To address the mechanisms underlying the development of ALK inhibitor resistance, we used iTRAQ quant. mass spectrometry and phosphor-receptor tyrosine kinase arrays to investigate intracellular signaling alterations in ALK inhibitor resistant NSCLC cell lines.  Src signaling was identified as an alectinib resistance mechanism, and combination treatment with ALK and Src inhibitors was highly effective for inhibiting the growth of ALK inhibitor resistant cells in vitro and in mouse xenograft models.  Furthermore, phospho-receptor tyrosine kinase activation and downstream PI3K/AKT signaling was effectively blocked by inhibiting Src in alectinib resistant cells.  Finally, we showed that the combined use of ALK and Src inhibitors inhibited the growth of other ALK-NSCLC cell lines, including those that were ceritinib or lorlatinib resistant.  Our data suggest that targeting Src signaling may be an effective approach to the treatment of ALK-NSCLC with acquired resistance to ALK inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRKtvR2ZB_R7Vg90H21EOLACvtfcHk0lgE4mO2bVqjWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVKjurjP&md5=3dda9a25c459c71bf8606db87a8076ad</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.3892%2Fijo.2017.4140&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3892%252Fijo.2017.4140%26sid%3Dliteratum%253Aachs%26aulast%3DYoshida%26aufirst%3DR.%26aulast%3DSasaki%26aufirst%3DT.%26aulast%3DMinami%26aufirst%3DY.%26aulast%3DHibino%26aufirst%3DY.%26aulast%3DOkumura%26aufirst%3DS.%26aulast%3DSado%26aufirst%3DM.%26aulast%3DMiyokawa%26aufirst%3DN.%26aulast%3DHayashi%26aufirst%3DS.%26aulast%3DKitada%26aufirst%3DM.%26aulast%3DOhsaki%26aufirst%3DY.%26atitle%3DActivation%2520of%2520Src%2520signaling%2520mediates%2520acquired%2520resistance%2520to%2520ALK%2520inhibition%2520in%2520lung%2520cancer%26jtitle%3DInt.%2520J.%2520Oncol.%26date%3D2017%26volume%3D51%26spage%3D1533%26epage%3D1540%26doi%3D10.3892%2Fijo.2017.4140" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Das, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norris, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Padmanabha, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moquin, R. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cook, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doweyko, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pitt, S.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">2-Aminothiazole as a novel kinase inhibitor template. Structure– activity relationship studies toward the discovery of N-(2-chloro-6-methylphenyl)-2-[[6-[4-(2-hydroxyethyl)-1-piperazinyl)]-2-methyl-4-pyrimidinyl] amino)]-1, 3-thiazole-5-carboxamide (dasatinib, BMS-354825) as a potent pan-Src kinase inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">6819</span>– <span class="NLM_lpage">6832</span>, <span class="refDoi"> DOI: 10.1021/jm060727j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm060727j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtFSlsLrI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=6819-6832&author=J.+Dasauthor=P.+Chenauthor=D.+Norrisauthor=R.+Padmanabhaauthor=J.+Linauthor=R.+V.+Moquinauthor=Z.+Shenauthor=L.+S.+Cookauthor=A.+M.+Doweykoauthor=S.+Pitt&title=2-Aminothiazole+as+a+novel+kinase+inhibitor+template.+Structure%E2%80%93+activity+relationship+studies+toward+the+discovery+of+N-%282-chloro-6-methylphenyl%29-2-%5B%5B6-%5B4-%282-hydroxyethyl%29-1-piperazinyl%29%5D-2-methyl-4-pyrimidinyl%5D+amino%29%5D-1%2C+3-thiazole-5-carboxamide+%28dasatinib%2C+BMS-354825%29+as+a+potent+pan-Src+kinase+inhibitor&doi=10.1021%2Fjm060727j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">2-Aminothiazole as a Novel Kinase Inhibitor Template. Structure-Activity Relationship Studies toward the Discovery of N-(2-Chloro-6-methylphenyl)-2-[[6-[4-(2-hydroxyethyl)-1-piperazinyl)]-2-methyl-4-pyrimidinyl]amino)]-1,3-thiazole-5-carboxamide (Dasatinib, BMS-354825) as a Potent pan-Src Kinase Inhibitor</span></div><div class="casAuthors">Das, Jagabandhu; Chen, Ping; Norris, Derek; Padmanabha, Ramesh; Lin, James; Moquin, Robert V.; Shen, Zhongqi; Cook, Lynda S.; Doweyko, Arthur M.; Pitt, Sidney; Pang, Suhong; Shen, Ding Ren; Fang, Qiong; de Fex, Henry F.; McIntyre, Kim W.; Shuster, David J.; Gillooly, Kathleen M.; Behnia, Kamelia; Schieven, Gary L.; Wityak, John; Barrish, Joel C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">6819-6832</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">2-Aminothiazole was discovered as a novel Src family kinase inhibitor template through screening of our internal compd. collection.  Optimization through successive structure-activity relationship iterations identified analogs I [R = 4-(2-hydroxyethyl)piperazino, Q] (Dasatinib, BMS-354825) and I [R = Me]as pan-Src inhibitors with nanomolar to subnanomolar potencies in biochem. and cellular assays.  Mol. modeling was used to construct a putative binding model for Lck inhibition by this class of compds.  The framework of key hydrogen-bond interactions proposed by this model was in agreement with the subsequent, published crystal structure of I [R = Q] bound to structurally similar Abl kinase.  The oral efficacy of this class of inhibitors was demonstrated with I [R = Me] in inhibiting the proinflammatory cytokine IL-2 ex vivo in mice (ED50 ∼ 5 mg/kg) and in reducing TNF levels in an acute murine model of inflammation (90% inhibition in LPS-induced TNFα prodn. when dosed orally at 60 mg/kg, 2 h prior to LPS administration).  The oral efficacy of I [R = Me] was further demonstrated in a chronic model of adjuvant arthritis in rats with established disease when administered orally at 0.3 and 3 mg/kg twice daily.  I [R = Q] is currently in clin. trials for the treatment of chronic myelogenous leukemia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrrbDd3Qks9cLVg90H21EOLACvtfcHk0ljKIASytpw7jw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtFSlsLrI&md5=9eed66048de7ab90b8a8015385d6f581</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fjm060727j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm060727j%26sid%3Dliteratum%253Aachs%26aulast%3DDas%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DP.%26aulast%3DNorris%26aufirst%3DD.%26aulast%3DPadmanabha%26aufirst%3DR.%26aulast%3DLin%26aufirst%3DJ.%26aulast%3DMoquin%26aufirst%3DR.%2BV.%26aulast%3DShen%26aufirst%3DZ.%26aulast%3DCook%26aufirst%3DL.%2BS.%26aulast%3DDoweyko%26aufirst%3DA.%2BM.%26aulast%3DPitt%26aufirst%3DS.%26atitle%3D2-Aminothiazole%2520as%2520a%2520novel%2520kinase%2520inhibitor%2520template.%2520Structure%25E2%2580%2593%2520activity%2520relationship%2520studies%2520toward%2520the%2520discovery%2520of%2520N-%25282-chloro-6-methylphenyl%2529-2-%255B%255B6-%255B4-%25282-hydroxyethyl%2529-1-piperazinyl%2529%255D-2-methyl-4-pyrimidinyl%255D%2520amino%2529%255D-1%252C%25203-thiazole-5-carboxamide%2520%2528dasatinib%252C%2520BMS-354825%2529%2520as%2520a%2520potent%2520pan-Src%2520kinase%2520inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D6819%26epage%3D6832%26doi%3D10.1021%2Fjm060727j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boschelli, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dutia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yaczko, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, M.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Optimization of 4-phenylamino-3-quinolinecarbonitriles as potent inhibitors of Src kinase activity</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">3965</span>– <span class="NLM_lpage">3977</span>, <span class="refDoi"> DOI: 10.1021/jm0102250</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0102250" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BD3MXnsFyhsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2001&pages=3965-3977&author=D.+H.+Boschelliauthor=F.+Yeauthor=Y.+D.+Wangauthor=M.+Dutiaauthor=S.+L.+Johnsonauthor=B.+Wuauthor=K.+Millerauthor=D.+W.+Powellauthor=D.+Yaczkoauthor=M.+Young&title=Optimization+of+4-phenylamino-3-quinolinecarbonitriles+as+potent+inhibitors+of+Src+kinase+activity&doi=10.1021%2Fjm0102250"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of 4-Phenylamino-3-quinolinecarbonitriles as Potent Inhibitors of Src Kinase Activity</span></div><div class="casAuthors">Boschelli, Diane H.; Ye, Fei; Wang, Yanong D.; Dutia, Minu; Johnson, Steve L.; Wu, Biqi; Miller, Karen; Powell, Dennis W.; Yaczko, Deanna; Young, Mairead; Tischler, Mark; Arndt, Kim; Discafani, Carolyn; Etienne, Carlo; Gibbons, Jay; Grod, Janet; Lucas, Judy; Weber, Jennifer M.; Boschelli, Frank</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">3965-3977</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Subsequent to the discovery of 4-[(2,4-dichlorophenyl)amino]-6,7-dimethoxy-3-quinolinecarbonitrile as an inhibitor of Src kinase activity (IC50 = 30 nM), several addnl. analogs were prepd.  Optimization of the C-4 anilino group led to I [R = Me].  Replacement of the methoxy group at C-7 with a 3-(morpholin-4-yl)propoxy group provided I [R = morpholinopropyl], resulting in increased inhibition of both Src kinase activity and Src-mediated cell proliferation.  Analogs of I [R = morpholinopropyl] with other trisubstituted anilines at C-4 were also potent Src inhibitors, and the propoxy group was preferred over ethoxy, butoxy, or pentoxy.  Replacement of the morpholine group with a 4-methylpiperazine group provided I [R = 4-methylpiperazinopropyl], which had an IC50 of 1.2 nM in the Src enzymic assay, an IC50 of 100 nM for the inhibition of Src-dependent cell proliferation and was selective for Src over non-Src family kinases.  I [R = 4-methylpiperazinopropyl], which had higher 1 and 4 h plasma levels than I [R = 4-morpholinopropyl], effectively inhibited tumor growth in xenograft models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIfrD66BPWqrVg90H21EOLACvtfcHk0ljKIASytpw7jw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXnsFyhsrk%253D&md5=88c2d8aa690b9aedbff034682aa3d73e</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fjm0102250&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0102250%26sid%3Dliteratum%253Aachs%26aulast%3DBoschelli%26aufirst%3DD.%2BH.%26aulast%3DYe%26aufirst%3DF.%26aulast%3DWang%26aufirst%3DY.%2BD.%26aulast%3DDutia%26aufirst%3DM.%26aulast%3DJohnson%26aufirst%3DS.%2BL.%26aulast%3DWu%26aufirst%3DB.%26aulast%3DMiller%26aufirst%3DK.%26aulast%3DPowell%26aufirst%3DD.%2BW.%26aulast%3DYaczko%26aufirst%3DD.%26aulast%3DYoung%26aufirst%3DM.%26atitle%3DOptimization%2520of%25204-phenylamino-3-quinolinecarbonitriles%2520as%2520potent%2520inhibitors%2520of%2520Src%2520kinase%2520activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2001%26volume%3D44%26spage%3D3965%26epage%3D3977%26doi%3D10.1021%2Fjm0102250" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Puls, L. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eadens, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Messersmith, W.</span></span> <span> </span><span class="NLM_article-title">Current status of SRC inhibitors in solid tumor malignancies</span>. <i>Oncologist</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">566</span>– <span class="NLM_lpage">578</span>, <span class="refDoi"> DOI: 10.1634/theoncologist.2010-0408</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.1634%2Ftheoncologist.2010-0408" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=21521831" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht12itr3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2011&pages=566-578&author=L.+N.+Pulsauthor=M.+Eadensauthor=W.+Messersmith&title=Current+status+of+SRC+inhibitors+in+solid+tumor+malignancies&doi=10.1634%2Ftheoncologist.2010-0408"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Current status of Src inhibitors in solid tumor malignancies</span></div><div class="casAuthors">Puls, Lauren N.; Eadens, Matthew; Messersmith, Wells</div><div class="citationInfo"><span class="NLM_cas:title">Oncologist</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">566-578</span>CODEN:
                <span class="NLM_cas:coden">OCOLF6</span>;
        ISSN:<span class="NLM_cas:issn">1083-7159</span>.
    
            (<span class="NLM_cas:orgname">AlphaMed Press</span>)
        </div><div class="casAbstract">A review.  Src is believed to play an important role in cancer, and several agents targeting Src are in clin. development.  Design. We reviewed Src structure and function and preclin. data supporting its role in the development of cancer via a PubMed search.  We conducted an extensive review of Src inhibitors by searching abstrs. from major oncol. meeting databases in the last 3 years and by comprehensively reviewing ongoing clin. trials on ClinicalTrials.gov.  Results. In this manuscript, we briefly review Src structure and function, mechanisms involving Src that lead to the development of cancer, and Src inhibitors and key preclin. data establishing a rationale for clin. application.  We then focus on clin. data supporting their use in solid tumor malignancies, a newer arena than their more well-established hematol. applications.  Particularly highlighted are clin. trials investigating new biomarkers as well as ongoing studies assessing Src inhibitor activity in biomarker-selected patient populations.  We also review newer investigational Src-targeting agents.  Conclusions. Src inhibitors have shown little activity in monotherapy trials in unselected solid tumor patient populations.  Combination studies and biomarker-driven clin. trials are under way.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYkp0FOYs6JLVg90H21EOLACvtfcHk0lguPMKY1FtYVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht12itr3F&md5=85afcbb274a84ec0de7e87b37a72eff4</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1634%2Ftheoncologist.2010-0408&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1634%252Ftheoncologist.2010-0408%26sid%3Dliteratum%253Aachs%26aulast%3DPuls%26aufirst%3DL.%2BN.%26aulast%3DEadens%26aufirst%3DM.%26aulast%3DMessersmith%26aufirst%3DW.%26atitle%3DCurrent%2520status%2520of%2520SRC%2520inhibitors%2520in%2520solid%2520tumor%2520malignancies%26jtitle%3DOncologist%26date%3D2011%26volume%3D16%26spage%3D566%26epage%3D578%26doi%3D10.1634%2Ftheoncologist.2010-0408" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brandvold, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steffey, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fox, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soellner, M. B.</span></span> <span> </span><span class="NLM_article-title">Development of a highly selective c-Src kinase inhibitor</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">1393</span>– <span class="NLM_lpage">1398</span>, <span class="refDoi"> DOI: 10.1021/cb300172e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb300172e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BC38XntFShu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=1393-1398&author=K.+R.+Brandvoldauthor=M.+E.+Steffeyauthor=C.+C.+Foxauthor=M.+B.+Soellner&title=Development+of+a+highly+selective+c-Src+kinase+inhibitor&doi=10.1021%2Fcb300172e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Development of a Highly Selective c-Src Kinase Inhibitor</span></div><div class="casAuthors">Brandvold, Kristoffer R.; Steffey, Michael E.; Fox, Christel C.; Soellner, Matthew B.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1393-1398</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Generating highly selective probes to interrogate protein kinase function in biol. studies remains a challenge, and new strategies are required.  Herein, we describe the development of the first highly selective and cell-permeable inhibitor of c-Src, a key signaling kinase in cancer.  Our strategy involves extension of traditional inhibitor design by appending functionality proposed to interact with the phosphate-binding loop of c-Src.  Using our selective inhibitor, we demonstrate that selective inhibition is significantly more efficacious than pan-kinase inhibition in slowing the growth of cancer cells.  We also show that inhibition of c-Abl kinase, an off-target of most c-Src inhibitors, promotes oncogenic cell growth.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0-UPmOeW6T7Vg90H21EOLACvtfcHk0lguPMKY1FtYVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XntFShu7c%253D&md5=4a6e7e2777ddc26687707c3bef8fdc02</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fcb300172e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb300172e%26sid%3Dliteratum%253Aachs%26aulast%3DBrandvold%26aufirst%3DK.%2BR.%26aulast%3DSteffey%26aufirst%3DM.%2BE.%26aulast%3DFox%26aufirst%3DC.%2BC.%26aulast%3DSoellner%26aufirst%3DM.%2BB.%26atitle%3DDevelopment%2520of%2520a%2520highly%2520selective%2520c-Src%2520kinase%2520inhibitor%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2012%26volume%3D7%26spage%3D1393%26epage%3D1398%26doi%3D10.1021%2Fcb300172e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, Z.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Discovery of 2-((3-Amino-4-methylphenyl) amino)-N-(2-methyl-5-(3-(trifluoromethyl) benzamido) phenyl)-4-(methylamino) pyrimidine-5-carboxamide (CHMFL-ABL-053) as a Potent, Selective, and Orally Available BCR-ABL/SRC/p38 Kinase Inhibitor for Chronic Myeloid Leukemia</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1984</span>– <span class="NLM_lpage">2004</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01618</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01618" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1aisLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1984-2004&author=X.+Liangauthor=X.+Liuauthor=B.+Wangauthor=F.+Zouauthor=A.+Wangauthor=S.+Qiauthor=C.+Chenauthor=Z.+Zhaoauthor=W.+Wangauthor=Z.+Qi&title=Discovery+of+2-%28%283-Amino-4-methylphenyl%29+amino%29-N-%282-methyl-5-%283-%28trifluoromethyl%29+benzamido%29+phenyl%29-4-%28methylamino%29+pyrimidine-5-carboxamide+%28CHMFL-ABL-053%29+as+a+Potent%2C+Selective%2C+and+Orally+Available+BCR-ABL%2FSRC%2Fp38+Kinase+Inhibitor+for+Chronic+Myeloid+Leukemia&doi=10.1021%2Facs.jmedchem.5b01618"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of 2-((3-Amino-4-methylphenyl)amino)-N-(2-methyl-5-(3-(trifluoromethyl)benzamido)phenyl)-4-(methylamino)pyrimidine-5-carboxamide (CHMFL-ABL-053) as a Potent, Selective, and Orally Available BCR-ABL/SRC/p38 Kinase Inhibitor for Chronic Myeloid Leukemia</span></div><div class="casAuthors">Liang, Xiaofei; Liu, Xiaochuan; Wang, Beilei; Zou, Fengming; Wang, Aoli; Qi, Shuang; Chen, Cheng; Zhao, Zheng; Wang, Wenchao; Qi, Ziping; Lv, Fengchao; Hu, Zhenquan; Wang, Li; Zhang, Shanchun; Liu, Qingsong; Liu, Jing</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1984-2004</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Starting from a dihydropyrimidopyrimidine core scaffold based compd. 27 (GNF-7), we discovered a highly potent (ABL1: IC50 of 70 nM) and selective (S score (1) = 0.02) BCR-ABL inhibitor 18a (CHMFL-ABL-053).  Compd. 18a did not exhibit apparent inhibitory activity against c-KIT kinase, which is the common target of currently clin. used BCR-ABL inhibitors.  Through significant suppression of the BCR-ABL autophosphorylation (EC50 about 100 nM) and downstream mediators such as STAT5, Crkl, and ERK's phosphorylation, 18a inhibited the proliferation of CML cell lines K562 (GI50 = 14 nM), KU812 (GI50 = 25 nM), and MEG-01 (GI50 = 16 nM).  A pharmacokinetic study revealed that 18a had over 4 h of half-life and 24% bioavailability in rats.  A 50 mg/kg/day dosage treatment could almost completely suppress tumor progression in the K562 cells inoculated xenograft mouse model.  As a potential useful drug candidate for CML, 18a is under extensive preclin. safety evaluation now.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3xXL7Oq_tALVg90H21EOLACvtfcHk0lguPMKY1FtYVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1aisLo%253D&md5=1a18d0735055b604b26951bc5794b7d4</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01618&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01618%26sid%3Dliteratum%253Aachs%26aulast%3DLiang%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DB.%26aulast%3DZou%26aufirst%3DF.%26aulast%3DWang%26aufirst%3DA.%26aulast%3DQi%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DC.%26aulast%3DZhao%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DQi%26aufirst%3DZ.%26atitle%3DDiscovery%2520of%25202-%2528%25283-Amino-4-methylphenyl%2529%2520amino%2529-N-%25282-methyl-5-%25283-%2528trifluoromethyl%2529%2520benzamido%2529%2520phenyl%2529-4-%2528methylamino%2529%2520pyrimidine-5-carboxamide%2520%2528CHMFL-ABL-053%2529%2520as%2520a%2520Potent%252C%2520Selective%252C%2520and%2520Orally%2520Available%2520BCR-ABL%252FSRC%252Fp38%2520Kinase%2520Inhibitor%2520for%2520Chronic%2520Myeloid%2520Leukemia%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1984%26epage%3D2004%26doi%3D10.1021%2Facs.jmedchem.5b01618" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, M.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, G.-B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, R.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, W.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiao, Y.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Design, synthesis, and structure–activity relationship studies of 3-(Phenylethynyl)-1 H-pyrazolo [3, 4-d] pyrimidin-4-amine derivatives as a new class of Src inhibitors with potent activities in models of triple negative breast cancer</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">3957</span>– <span class="NLM_lpage">3974</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00270</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00270" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlvVaksbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=3957-3974&author=C.-H.+Zhangauthor=M.-W.+Zhengauthor=Y.-P.+Liauthor=X.-D.+Linauthor=M.+Huangauthor=L.+Zhongauthor=G.-B.+Liauthor=R.-J.+Zhangauthor=W.-T.+Linauthor=Y.+Jiao&title=Design%2C+synthesis%2C+and+structure%E2%80%93activity+relationship+studies+of+3-%28Phenylethynyl%29-1+H-pyrazolo+%5B3%2C+4-d%5D+pyrimidin-4-amine+derivatives+as+a+new+class+of+Src+inhibitors+with+potent+activities+in+models+of+triple+negative+breast+cancer&doi=10.1021%2Facs.jmedchem.5b00270"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Design, Synthesis, and Structure-Activity Relationship Studies of 3-(Phenylethynyl)-1H-pyrazolo[3,4-d]pyrimidin-4-amine Derivatives as a New Class of Src Inhibitors with Potent Activities in Models of Triple Negative Breast Cancer</span></div><div class="casAuthors">Zhang, Chun-Hui; Zheng, Ming-Wu; Li, Ya-Ping; Lin, Xing-Dong; Huang, Mei; Zhong, Lei; Li, Guo-Bo; Zhang, Rong-Jie; Lin, Wan-Ting; Jiao, Yan; Wu, Xiao-Ai; Yang, Jiao; Xiang, Rong; Chen, Li-Juan; Zhao, Ying-Lan; Cheng, Wei; Wei, Yu-Quan; Yang, Sheng-Yong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3957-3974</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of 3-(phenylethynyl)-1H-pyrazolo[3,4-d]pyrimidin-4-amine derivs. were designed and synthesized.  Structure-activity relationship (SAR) anal. of these compds. led to the discovery of compd. 1j (I), which showed the highest inhibitory potency against the Src kinase and the most potent antiviability activity against the typical TNBC cell line MDA-MB-231 among all the synthesized compds.  Further kinase inhibition assays showed that compd. 1j was a multikinase inhibitor and potently inhibited Src (IC50 = 0.0009 μM) and MAPK signaling protein kinases B-RAF and C-RAF.  In an MDA-MB-231 xenograft mouse model, a once-daily dose of compd. 1j at 30 mg/kg for 18 days completely suppressed the tumor growth with a tumor inhibition rate larger than 100% without obvious toxicity.  It also displayed good pharmacokinetic properties in a preliminary pharmacokinetic assay.  Western blot and immunohistochem. assays revealed that compd. 1j significantly inhibited Src and MAPK signaling and markedly induced apoptosis in tumor tissues.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvXJluk6Zz57Vg90H21EOLACvtfcHk0li4k3VITHo_qg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlvVaksbg%253D&md5=a15a90d306d54efac683cc373940cde4</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00270&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00270%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DC.-H.%26aulast%3DZheng%26aufirst%3DM.-W.%26aulast%3DLi%26aufirst%3DY.-P.%26aulast%3DLin%26aufirst%3DX.-D.%26aulast%3DHuang%26aufirst%3DM.%26aulast%3DZhong%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DG.-B.%26aulast%3DZhang%26aufirst%3DR.-J.%26aulast%3DLin%26aufirst%3DW.-T.%26aulast%3DJiao%26aufirst%3DY.%26atitle%3DDesign%252C%2520synthesis%252C%2520and%2520structure%25E2%2580%2593activity%2520relationship%2520studies%2520of%25203-%2528Phenylethynyl%2529-1%2520H-pyrazolo%2520%255B3%252C%25204-d%255D%2520pyrimidin-4-amine%2520derivatives%2520as%2520a%2520new%2520class%2520of%2520Src%2520inhibitors%2520with%2520potent%2520activities%2520in%2520models%2520of%2520triple%2520negative%2520breast%2520cancer%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D3957%26epage%3D3974%26doi%3D10.1021%2Facs.jmedchem.5b00270" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fraser, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dowling, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Houston, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munro, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muir, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrington, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Webster, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frame, M. C.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Rapid discovery and structure–activity relationships of pyrazolopyrimidines that potently suppress breast cancer cell growth via SRC kinase inhibition with exceptional selectivity over ABL kinase</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">4697</span>– <span class="NLM_lpage">4710</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00065</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00065" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BC28Xmsl2rsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=4697-4710&author=C.+Fraserauthor=J.+C.+Dawsonauthor=R.+Dowlingauthor=D.+R.+Houstonauthor=J.+T.+Weissauthor=A.+F.+Munroauthor=M.+Muirauthor=L.+Harringtonauthor=S.+P.+Websterauthor=M.+C.+Frame&title=Rapid+discovery+and+structure%E2%80%93activity+relationships+of+pyrazolopyrimidines+that+potently+suppress+breast+cancer+cell+growth+via+SRC+kinase+inhibition+with+exceptional+selectivity+over+ABL+kinase&doi=10.1021%2Facs.jmedchem.6b00065"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid Discovery and Structure-Activity Relationships of Pyrazolopyrimidines That Potently Suppress Breast Cancer Cell Growth via SRC Kinase Inhibition with Exceptional Selectivity over ABL Kinase</span></div><div class="casAuthors">Fraser, Craig; Dawson, John C.; Dowling, Reece; Houston, Douglas R.; Weiss, Jason T.; Munro, Alison F.; Muir, Morwenna; Harrington, Lea; Webster, Scott P.; Frame, Margaret C.; Brunton, Valerie G.; Patton, E. Elizabeth; Carragher, Neil O.; Unciti-Broceta, Asier</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4697-4710</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Novel pyrazolopyrimidines displaying high potency and selectivity toward SRC family kinases have been developed by combining ligand-based design and phenotypic screening in an iterative manner.  Compds. were derived from the promiscuous kinase inhibitor PP1 to search for analogs that could potentially target a broad spectrum of kinases involved in cancer.  Phenotypic screening against MCF7 mammary adenocarcinoma cells generated target-agnostic structure-activity relationships that biased subsequent designs toward breast cancer treatment rather than to a particular target.  This strategy led to the discovery of two potent antiproliferative leads with phenotypically distinct anticancer mode of actions.  Kinase profiling and further optimization resulted in eCF506, the first small mol. with subnanomolar IC50 for SRC that requires 3 orders of magnitude greater concn. to inhibit ABL. eCF506 exhibits excellent water soly., an optimal DMPK profile and oral bioavailability, halts SRC-assocd. neuromast migration in zebrafish embryos without inducing life-threatening heart defects, and inhibits SRC phosphorylation in tumor xenografts in mice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq55yS4FWveyLVg90H21EOLACvtfcHk0li4k3VITHo_qg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xmsl2rsL0%253D&md5=e56206009ccca680432d12db23aa29fd</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00065&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00065%26sid%3Dliteratum%253Aachs%26aulast%3DFraser%26aufirst%3DC.%26aulast%3DDawson%26aufirst%3DJ.%2BC.%26aulast%3DDowling%26aufirst%3DR.%26aulast%3DHouston%26aufirst%3DD.%2BR.%26aulast%3DWeiss%26aufirst%3DJ.%2BT.%26aulast%3DMunro%26aufirst%3DA.%2BF.%26aulast%3DMuir%26aufirst%3DM.%26aulast%3DHarrington%26aufirst%3DL.%26aulast%3DWebster%26aufirst%3DS.%2BP.%26aulast%3DFrame%26aufirst%3DM.%2BC.%26atitle%3DRapid%2520discovery%2520and%2520structure%25E2%2580%2593activity%2520relationships%2520of%2520pyrazolopyrimidines%2520that%2520potently%2520suppress%2520breast%2520cancer%2520cell%2520growth%2520via%2520SRC%2520kinase%2520inhibition%2520with%2520exceptional%2520selectivity%2520over%2520ABL%2520kinase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D4697%26epage%3D4710%26doi%3D10.1021%2Facs.jmedchem.6b00065" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tintori, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fallacara, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Radi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zamperini, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dreassi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crespan, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maga, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schenone, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Musumeci, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brullo, C.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Combining X-ray crystallography and molecular modeling toward the optimization of pyrazolo [3, 4-d] pyrimidines as potent c-Src inhibitors active in vivo against neuroblastoma</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">347</span>– <span class="NLM_lpage">361</span>, <span class="refDoi"> DOI: 10.1021/jm5013159</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm5013159" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitVegur%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=347-361&author=C.+Tintoriauthor=A.+L.+Fallacaraauthor=M.+Radiauthor=C.+Zamperiniauthor=E.+Dreassiauthor=E.+Crespanauthor=G.+Magaauthor=S.+Schenoneauthor=F.+Musumeciauthor=C.+Brullo&title=Combining+X-ray+crystallography+and+molecular+modeling+toward+the+optimization+of+pyrazolo+%5B3%2C+4-d%5D+pyrimidines+as+potent+c-Src+inhibitors+active+in+vivo+against+neuroblastoma&doi=10.1021%2Fjm5013159"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Combining X-ray Crystallography and Molecular Modeling toward the Optimization of Pyrazolo[3,4-d]pyrimidines as Potent c-Src Inhibitors Active in Vivo against Neuroblastoma</span></div><div class="casAuthors">Tintori, Cristina; Fallacara, Anna Lucia; Radi, Marco; Zamperini, Claudio; Dreassi, Elena; Crespan, Emmanuele; Maga, Giovanni; Schenone, Silvia; Musumeci, Francesca; Brullo, Chiara; Richters, Andre; Gasparrini, Francesca; Angelucci, Adriano; Festuccia, Claudio; Delle Monache, Simona; Rauh, Daniel; Botta, Maurizio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">347-361</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">C-Src is a tyrosine kinase belonging to the Src-family kinases.  It is overexpressed and/or hyperactivated in a variety of cancer cells, thus its inhibition has been predicted to have therapeutic effects in solid tumors.  Recently, the pyrazolo[3,4-d]pyrimidine 3 was reported as a dual c-Src/Abl inhibitor.  Herein we describe a multidisciplinary drug discovery approach for the optimization of the lead 3 against c-Src.  Starting from the X-ray crystal structure of c-Src in complex with 3, Monte Carlo free energy perturbation calcns. were applied to guide the design of c-Src inhibitors with improved activities.  As a result, the introduction of a meta hydroxyl group on the C4 anilino ring was computed to be particularly favorable.  The potency of the synthesized inhibitors was increased with respect to the starting lead 3.  The best identified compds. were also found active in the inhibition of neuroblastoma cell proliferation.  Furthermore, compd. 29 also showed in vivo activity in xenograft model using SH-SY5Y cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpD79cr521X37Vg90H21EOLACvtfcHk0lihuZ-6zkbe0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitVegur%252FM&md5=c45f20a3cd5ce12b5afff5ce7b67f1f5</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fjm5013159&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm5013159%26sid%3Dliteratum%253Aachs%26aulast%3DTintori%26aufirst%3DC.%26aulast%3DFallacara%26aufirst%3DA.%2BL.%26aulast%3DRadi%26aufirst%3DM.%26aulast%3DZamperini%26aufirst%3DC.%26aulast%3DDreassi%26aufirst%3DE.%26aulast%3DCrespan%26aufirst%3DE.%26aulast%3DMaga%26aufirst%3DG.%26aulast%3DSchenone%26aufirst%3DS.%26aulast%3DMusumeci%26aufirst%3DF.%26aulast%3DBrullo%26aufirst%3DC.%26atitle%3DCombining%2520X-ray%2520crystallography%2520and%2520molecular%2520modeling%2520toward%2520the%2520optimization%2520of%2520pyrazolo%2520%255B3%252C%25204-d%255D%2520pyrimidines%2520as%2520potent%2520c-Src%2520inhibitors%2520active%2520in%2520vivo%2520against%2520neuroblastoma%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D347%26epage%3D361%26doi%3D10.1021%2Fjm5013159" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, W.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Azam, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sundaramoorthi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banda, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lentini, S. P.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">9-(Arenethenyl) purines as dual Src/Abl kinase inhibitors targeting the inactive conformation: design, synthesis, and biological evaluation</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">4743</span>– <span class="NLM_lpage">4756</span>, <span class="refDoi"> DOI: 10.1021/jm900166t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm900166t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BD1MXotVGktr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=4743-4756&author=W.-S.+Huangauthor=X.+Zhuauthor=Y.+Wangauthor=M.+Azamauthor=D.+Wenauthor=R.+Sundaramoorthiauthor=R.+M.+Thomasauthor=S.+Liuauthor=G.+Bandaauthor=S.+P.+Lentini&title=9-%28Arenethenyl%29+purines+as+dual+Src%2FAbl+kinase+inhibitors+targeting+the+inactive+conformation%3A+design%2C+synthesis%2C+and+biological+evaluation&doi=10.1021%2Fjm900166t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">9-(Arenethenyl)purines as Dual Src/Abl Kinase Inhibitors Targeting the Inactive Conformation: Design, Synthesis, and Biological Evaluation</span></div><div class="casAuthors">Huang, Wei-Sheng; Zhu, Xiaotian; Wang, Yihan; Azam, Mohammad; Wen, David; Sundaramoorthi, Raji; Thomas, R. Mathew; Liu, Shuangying; Banda, Geetha; Lentini, Scott P.; Das, Sasmita; Xu, Qihong; Keats, Jeff; Wang, Frank; Wardwell, Scott; Ning, Yaoyu; Snodgrass, Joseph T.; Broudy, Marc I.; Russian, Karin; Daley, George Q.; Iuliucci, John; Dalgarno, David C.; Clackson, Tim; Sawyer, Tomi K.; Shakespeare, William C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">4743-4756</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel series of potent dual Src/Abl kinase inhibitors based on a 9-(arenethenyl)purine core has been identified.  Unlike traditional dual Src/Abl inhibitors targeting the active enzyme conformation, these inhibitors bind to the inactive, DFG-out conformation of both kinases.  Extensive SAR studies led to the discovery of potent and orally bioavailable inhibitors, some of which demonstrated in vivo efficacy.  Once-daily oral administration of inhibitor I (AP24226) significantly prolonged the survival of mice injected i.v. with wild type Bcr-Abl expressing Ba/F3 cells at a dose of 10 mg/kg.  In a sep. model, oral administration of I to mice bearing s.c. xenografts of Src Y527F expressing NIH 3T3 cells elicited dose-dependent tumor shrinkage with complete tumor regression obsd. at the highest dose.  Notably, several inhibitors, e.g., II (AP24163), exhibited modest cellular potency (IC50 = 300-400 nM) against the Bcr-Abl mutant T315I, a variant resistant to all currently marketed therapies for chronic myeloid leukemia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpanmqXXD3_P7Vg90H21EOLACvtfcHk0lihuZ-6zkbe0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXotVGktr8%253D&md5=287f41409bf75e5384a2b6853bda3def</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fjm900166t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm900166t%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DW.-S.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DAzam%26aufirst%3DM.%26aulast%3DWen%26aufirst%3DD.%26aulast%3DSundaramoorthi%26aufirst%3DR.%26aulast%3DThomas%26aufirst%3DR.%2BM.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DBanda%26aufirst%3DG.%26aulast%3DLentini%26aufirst%3DS.%2BP.%26atitle%3D9-%2528Arenethenyl%2529%2520purines%2520as%2520dual%2520Src%252FAbl%2520kinase%2520inhibitors%2520targeting%2520the%2520inactive%2520conformation%253A%2520design%252C%2520synthesis%252C%2520and%2520biological%2520evaluation%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D4743%26epage%3D4756%26doi%3D10.1021%2Fjm900166t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kitagawa, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yokota, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gouda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narumi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohmoto, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishiwaki, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akita, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirii, Y.</span></span> <span> </span><span class="NLM_article-title">Activity-based kinase profiling of approved tyrosine kinase inhibitors</span>. <i>Genes to cells</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">110</span>– <span class="NLM_lpage">122</span>, <span class="refDoi"> DOI: 10.1111/gtc.12022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.1111%2Fgtc.12022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=23279183" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlajurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2013&pages=110-122&author=D.+Kitagawaauthor=K.+Yokotaauthor=M.+Goudaauthor=Y.+Narumiauthor=H.+Ohmotoauthor=E.+Nishiwakiauthor=K.+Akitaauthor=Y.+Kirii&title=Activity-based+kinase+profiling+of+approved+tyrosine+kinase+inhibitors&doi=10.1111%2Fgtc.12022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Activity-based kinase profiling of approved tyrosine kinase inhibitors</span></div><div class="casAuthors">Kitagawa, Daisuke; Yokota, Koichi; Gouda, Masaki; Narumi, Yugo; Ohmoto, Hiroshi; Nishiwaki, Eiji; Akita, Kensaku; Kirii, Yasuyuki</div><div class="citationInfo"><span class="NLM_cas:title">Genes to Cells</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">110-122</span>CODEN:
                <span class="NLM_cas:coden">GECEFL</span>;
        ISSN:<span class="NLM_cas:issn">1356-9597</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">The specificities of nine approved tyrosine kinase inhibitors (imatinib, dasatinib, nilotinib, gefitinib, erlotinib, lapatinib, sorafenib, sunitinib, and pazopanib) were detd. by activity-based kinase profiling using a large panel of human recombinant active kinases.  This panel consisted of 79 tyrosine kinases, 199 serine/threonine kinases, three lipid kinases, and 29 disease-relevant mutant kinases.  Many potential targets of each inhibitor were identified by kinase profiling at the Km for ATP.  In addn., profiling at a physiol. ATP concn. (1 mm) was carried out, and the IC50 values of the inhibitors against each kinase were compared with the estd. plasma-free concn. (calcd. from published pharmacokinetic parameters of plasma Ctrough and Cmax values).  This anal. revealed that the approved kinase inhibitors were well optimized for their target kinases.  This profiling also implicates activity at particular off-target kinases in drug side effects.  Thus, large-scale kinase profiling at both Km and physiol. ATP concns. could be useful in characterizing the targets and off-targets of kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUe0U00YKTHbVg90H21EOLACvtfcHk0lihuZ-6zkbe0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlajurs%253D&md5=a30fd15aa57d46e7d6d461a80907e0e2</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1111%2Fgtc.12022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fgtc.12022%26sid%3Dliteratum%253Aachs%26aulast%3DKitagawa%26aufirst%3DD.%26aulast%3DYokota%26aufirst%3DK.%26aulast%3DGouda%26aufirst%3DM.%26aulast%3DNarumi%26aufirst%3DY.%26aulast%3DOhmoto%26aufirst%3DH.%26aulast%3DNishiwaki%26aufirst%3DE.%26aulast%3DAkita%26aufirst%3DK.%26aulast%3DKirii%26aufirst%3DY.%26atitle%3DActivity-based%2520kinase%2520profiling%2520of%2520approved%2520tyrosine%2520kinase%2520inhibitors%26jtitle%3DGenes%2520to%2520cells%26date%3D2013%26volume%3D18%26spage%3D110%26epage%3D122%26doi%3D10.1111%2Fgtc.12022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rix, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hantschel, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dürnberger, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rix, L. L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Planyavsky, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernbach, N. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaupe, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valent, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colinge, J.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Chemical proteomic profiles of the BCR-ABL inhibitors imatinib, nilotinib, and dasatinib reveal novel kinase and nonkinase targets</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">4055</span>– <span class="NLM_lpage">4063</span>, <span class="refDoi"> DOI: 10.1182/blood-2007-07-102061</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.1182%2Fblood-2007-07-102061" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=17720881" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtl2qsb%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2007&pages=4055-4063&author=U.+Rixauthor=O.+Hantschelauthor=G.+D%C3%BCrnbergerauthor=L.+L.+R.+Rixauthor=M.+Planyavskyauthor=N.+V.+Fernbachauthor=I.+Kaupeauthor=K.+L.+Bennettauthor=P.+Valentauthor=J.+Colinge&title=Chemical+proteomic+profiles+of+the+BCR-ABL+inhibitors+imatinib%2C+nilotinib%2C+and+dasatinib+reveal+novel+kinase+and+nonkinase+targets&doi=10.1182%2Fblood-2007-07-102061"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical proteomic profiles of the BCR-ABL inhibitors imatinib, nilotinib, and dasatinib reveal novel kinase and nonkinase targets</span></div><div class="casAuthors">Rix, Uwe; Hantschel, Oliver; Duernberger, Gerhard; Rix, Lily L. Remsing; Planyavsky, Melanie; Fernbach, Nora V.; Kaupe, Ines; Bennett, Keiryn L.; Valent, Peter; Colinge, Jacques; Koecher, Thomas; Superti-Furga, Giulio</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4055-4063</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">The BCR-ABL tyrosine kinase inhibitor imatinib represents the current front-line therapy in chronic myeloid leukemia.  Because many patients develop imatinib resistance, 2 second-generation drugs, nilotinib and dasatinib, displaying increased potency against BCR-ABL were developed.  To predict potential side effects and novel medical uses, we generated comprehensive drug-protein interaction profiles by chem. proteomics for all 3 drugs.  Our studies yielded 4 major findings: (1) The interaction profiles of the 3 drugs displayed strong differences and only a small overlap covering the ABL kinases.  (2) Dasatinib bound in excess of 30 Tyr and Ser/Thr kinases, including major regulators of the immune system, suggesting that dasatinib might have a particular impact on immune function.  (3) Despite the high specificity of nilotinib, the receptor tyrosine kinase DDR1 was identified and validated as an addnl. major target.  (4) The oxidoreductase NQO2 was bound and inhibited by imatinib and nilotinib at physiol. relevant drug concns., representing the 1st nonkinase target of these drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqe8e6qqCnkrrVg90H21EOLACvtfcHk0ligffmZqP5tPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtl2qsb%252FF&md5=ff968982e9c20806c0c5fe9612f88407</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1182%2Fblood-2007-07-102061&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2007-07-102061%26sid%3Dliteratum%253Aachs%26aulast%3DRix%26aufirst%3DU.%26aulast%3DHantschel%26aufirst%3DO.%26aulast%3DD%25C3%25BCrnberger%26aufirst%3DG.%26aulast%3DRix%26aufirst%3DL.%2BL.%2BR.%26aulast%3DPlanyavsky%26aufirst%3DM.%26aulast%3DFernbach%26aufirst%3DN.%2BV.%26aulast%3DKaupe%26aufirst%3DI.%26aulast%3DBennett%26aufirst%3DK.%2BL.%26aulast%3DValent%26aufirst%3DP.%26aulast%3DColinge%26aufirst%3DJ.%26atitle%3DChemical%2520proteomic%2520profiles%2520of%2520the%2520BCR-ABL%2520inhibitors%2520imatinib%252C%2520nilotinib%252C%2520and%2520dasatinib%2520reveal%2520novel%2520kinase%2520and%2520nonkinase%2520targets%26jtitle%3DBlood%26date%3D2007%26volume%3D110%26spage%3D4055%26epage%3D4063%26doi%3D10.1182%2Fblood-2007-07-102061" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Remsing
Rix, L L</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rix, U</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colinge, J</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hantschel, O</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, K L</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stranzl, T</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, A</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baumgartner, C</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valent, P</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Augustin, M</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Till, J H</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Superti-Furga, G</span></span> <span> </span><span class="NLM_article-title">Global target profile of the kinase inhibitor bosutinib in primary chronic myeloid leukemia cells</span>. <i>Leukemia</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">477</span>, <span class="refDoi"> DOI: 10.1038/leu.2008.334</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.1038%2Fleu.2008.334" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=19039322" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BD1MXivVKhu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2009&pages=477&author=L+L+Remsing%0ARixauthor=U+Rixauthor=J+Colingeauthor=O+Hantschelauthor=K+L+Bennettauthor=T+Stranzlauthor=A+Mullerauthor=C+Baumgartnerauthor=P+Valentauthor=M+Augustinauthor=J+H+Tillauthor=G+Superti-Furga&title=Global+target+profile+of+the+kinase+inhibitor+bosutinib+in+primary+chronic+myeloid+leukemia+cells&doi=10.1038%2Fleu.2008.334"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Global target profile of the kinase inhibitor bosutinib in primary chronic myeloid leukemia cells</span></div><div class="casAuthors">Remsing Rix, L. L.; Rix, U.; Colinge, J.; Hantschel, O.; Bennett, K. L.; Stranzl, T.; Mueller, A.; Baumgartner, C.; Valent, P.; Augustin, M.; Till, J. H.; Superti-Furga, G.</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">477-485</span>CODEN:
                <span class="NLM_cas:coden">LEUKED</span>;
        ISSN:<span class="NLM_cas:issn">0887-6924</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The detailed mol. mechanism of action of second-generation BCR-ABL tyrosine kinase inhibitors, including perturbed targets and pathways, should contribute to rationalized therapy in chronic myeloid leukemia (CML) or in other affected diseases.  Here, we characterized the target profile of the dual SRC/ABL inhibitor bosutinib employing a two-tiered approach using chem. proteomics to identify natural binders in whole cell lysates of primary CML and K562 cells in parallel to in vitro kinase assays against a large recombinant kinase panel.  The combined strategy resulted in a global survey of bosutinib targets comprised of over 45 novel tyrosine and serine/threonine kinases.  We have found clear differences in the target patterns of bosutinib in primary CML cells vs. the K562 cell line.  A comparison of bosutinib with dasatinib across the whole kinase panel revealed overlapping, but distinct, inhibition profiles.  Common among those were the SRC, ABL and TEC family kinases.  Bosutinib did not inhibit KIT or platelet-derived growth factor receptor, but prominently targeted the apoptosis-linked STE20 kinases.  Although in vivo bosutinib is inactive against ABL T315I, we found this clin. important mutant to be enzymically inhibited in the mid-nanomolar range.  Finally, bosutinib is the first kinase inhibitor shown to target CAMK2G, recently implicated in myeloid leukemia cell proliferation.  Leukemia (2009) 23, 477-485; doi:10.1038/leu.2008.334; published online 27 Nov. 2008.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8u0ixFotIMLVg90H21EOLACvtfcHk0ligffmZqP5tPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXivVKhu7w%253D&md5=b0d56d468ef66ddc2a6f012d08f8c1a4</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1038%2Fleu.2008.334&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fleu.2008.334%26sid%3Dliteratum%253Aachs%26aulast%3DRemsing%2BRix%26aufirst%3DL%2BL%26aulast%3DRix%26aufirst%3DU%26aulast%3DColinge%26aufirst%3DJ%26aulast%3DHantschel%26aufirst%3DO%26aulast%3DBennett%26aufirst%3DK%2BL%26aulast%3DStranzl%26aufirst%3DT%26aulast%3DMuller%26aufirst%3DA%26aulast%3DBaumgartner%26aufirst%3DC%26aulast%3DValent%26aufirst%3DP%26aulast%3DAugustin%26aufirst%3DM%26aulast%3DTill%26aufirst%3DJ%2BH%26aulast%3DSuperti-Furga%26aufirst%3DG%26atitle%3DGlobal%2520target%2520profile%2520of%2520the%2520kinase%2520inhibitor%2520bosutinib%2520in%2520primary%2520chronic%2520myeloid%2520leukemia%2520cells%26jtitle%3DLeukemia%26date%3D2009%26volume%3D23%26spage%3D477%26doi%3D10.1038%2Fleu.2008.334" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Keller-von
Amsberg, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koschmieder, S.</span></span> <span> </span><span class="NLM_article-title">Profile of bosutinib and its clinical potential in the treatment of chronic myeloid leukemia</span>. <i>OncoTargets Ther.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">99</span>, <span class="refDoi"> DOI: 10.2147/OTT.S19901</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.2147%2FOTT.S19901" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=23493838" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BC3sXkt1Wltr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2013&pages=99&author=G.+Keller-von%0AAmsbergauthor=S.+Koschmieder&title=Profile+of+bosutinib+and+its+clinical+potential+in+the+treatment+of+chronic+myeloid+leukemia&doi=10.2147%2FOTT.S19901"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Profile of bosutinib and its clinical potential in the treatment of chronic myeloid leukemia</span></div><div class="casAuthors">Keller-von Amsberg, Gunhild; Koschmieder, Steffen</div><div class="citationInfo"><span class="NLM_cas:title">OncoTargets and Therapy</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">99-106</span>CODEN:
                <span class="NLM_cas:coden">OTNHAZ</span>;
        ISSN:<span class="NLM_cas:issn">1178-6930</span>.
    
            (<span class="NLM_cas:orgname">Dove Medical Press Ltd.</span>)
        </div><div class="casAbstract">A review.  Bosutinib (SKI-606) is an orally available, once-daily, dual Src and Abl kinase inhibitor with promising clin. potential in first-, second-, and third-line treatment of chronic myeloid leukemia (CML).  Bosutinib effectively inhibits wild-type BCR-ABL and most imatinib-resistant BCR-ABL mutations except for V299L and T315I.  Low hematol. toxicity is a remarkable characteristic of this novel second-generation tyrosine kinase inhibitor, and this has been ascribed to its minimal activity against the platelet-derived growth factor receptor and KIT.  Low-grade, typically self-limiting diarrhea, which usually appears within the first few weeks after treatment initiation, represents the predominant toxicity of bosutinib.  Other treatment-assocd. adverse events are mostly mild to moderate.  Bosutinib has been approved by the US Food and Drug Administration for the treatment of chronic, accelerated, or blast phase Philadelphia chromosome-pos. CML in adult patients with resistance or intolerance to prior therapy.  This review summarizes the main properties of bosutinib and the currently available data on its clin. potential in the treatment of CML.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGryIlbJ7tWle7Vg90H21EOLACvtfcHk0ligffmZqP5tPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXkt1Wltr8%253D&md5=3b42376e225c82e97a412daeabfc8aa9</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.2147%2FOTT.S19901&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2147%252FOTT.S19901%26sid%3Dliteratum%253Aachs%26aulast%3DKeller-von%2BAmsberg%26aufirst%3DG.%26aulast%3DKoschmieder%26aufirst%3DS.%26atitle%3DProfile%2520of%2520bosutinib%2520and%2520its%2520clinical%2520potential%2520in%2520the%2520treatment%2520of%2520chronic%2520myeloid%2520leukemia%26jtitle%3DOncoTargets%2520Ther.%26date%3D2013%26volume%3D6%26spage%3D99%26doi%3D10.2147%2FOTT.S19901" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hennequin, L. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breed, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curwen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fennell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambert-van der Brempt, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morgentin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norman, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olivier, A.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">N-(5-Chloro-1, 3-benzodioxol-4-yl)-7-[2-(4-methylpiperazin-1-yl) ethoxy]-5-(tetrahydro-2 H-pyran-4-yloxy) quinazolin-4-amine, a novel, highly selective, orally available, dual-specific c-Src/Abl kinase inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">6465</span>– <span class="NLM_lpage">6488</span>, <span class="refDoi"> DOI: 10.1021/jm060434q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm060434q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtVaku7%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=6465-6488&author=L.+F.+Hennequinauthor=J.+Allenauthor=J.+Breedauthor=J.+Curwenauthor=M.+Fennellauthor=T.+P.+Greenauthor=C.+Lambert-van+der+Bremptauthor=R.+Morgentinauthor=R.+A.+Normanauthor=A.+Olivier&title=N-%285-Chloro-1%2C+3-benzodioxol-4-yl%29-7-%5B2-%284-methylpiperazin-1-yl%29+ethoxy%5D-5-%28tetrahydro-2+H-pyran-4-yloxy%29+quinazolin-4-amine%2C+a+novel%2C+highly+selective%2C+orally+available%2C+dual-specific+c-Src%2FAbl+kinase+inhibitor&doi=10.1021%2Fjm060434q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">N-(5-Chloro-1,3-benzodioxol-4-yl)-7-[2-(4-methylpiperazin-1-yl)ethoxy]-5- (tetrahydro-2H-pyran-4-yloxy)quinazolin-4-amine, a Novel, Highly Selective, Orally Available, Dual-Specific c-Src/Abl Kinase Inhibitor</span></div><div class="casAuthors">Hennequin, Laurent F.; Allen, Jack; Breed, Jason; Curwen, Jon; Fennell, Michael; Green, Tim P.; Lambert van der Brempt, Christine; Morgentin, Remy; Norman, Richard A.; Olivier, Annie; Otterbein, Ludovic; Ple, Patrick A.; Warin, Nicolas; Costello, Gerard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">6465-6488</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Src family kinases (SFKs) are nonreceptor tyrosine kinases that are reported to be crit. for cancer progression.  We report here a novel subseries of C-5-substituted anilinoquinazolines that display high affinity and specificity for the tyrosine kinase domain of the c-Src and Abl enzymes.  These compds. exhibit high selectivity for SFKs over a panel of recombinant protein kinases, excellent pharmacokinetics, and in vivo activity following oral dosing. N-(5-Chloro-1,3-benzodioxol-4-yl)-7-[2-(4-methylpiperazin-1-yl)ethoxy]-5-(tetrahydro-2H-pyran-4-yloxy)quinazolin-4-amine(I) (AZD0530) inhibits c-Src and Abl enzymes at low nanomolar concns. and is highly selective over a range of kinases.  AZD0530 displays excellent pharmacokinetic parameters in animal preclinically and in man (t1/2 = 40 h).  AZD0530 is a potent inhibitor of tumor growth in a c-Src-transfected 3T3-fibroblast xenograft model in vivo and led to a significant increase in survival in a highly aggressive, orthotopic model of human pancreatic cancer when dosed orally once daily.  AZD0530 is currently undergoing clin. evaluation in man.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqexk9b8WOUPrVg90H21EOLACvtfcHk0ljnmVFBRQ1Ibg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtVaku7%252FP&md5=dd798dec4067acb6d21020c66544e8a0</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fjm060434q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm060434q%26sid%3Dliteratum%253Aachs%26aulast%3DHennequin%26aufirst%3DL.%2BF.%26aulast%3DAllen%26aufirst%3DJ.%26aulast%3DBreed%26aufirst%3DJ.%26aulast%3DCurwen%26aufirst%3DJ.%26aulast%3DFennell%26aufirst%3DM.%26aulast%3DGreen%26aufirst%3DT.%2BP.%26aulast%3DLambert-van%2Bder%2BBrempt%26aufirst%3DC.%26aulast%3DMorgentin%26aufirst%3DR.%26aulast%3DNorman%26aufirst%3DR.%2BA.%26aulast%3DOlivier%26aufirst%3DA.%26atitle%3DN-%25285-Chloro-1%252C%25203-benzodioxol-4-yl%2529-7-%255B2-%25284-methylpiperazin-1-yl%2529%2520ethoxy%255D-5-%2528tetrahydro-2%2520H-pyran-4-yloxy%2529%2520quinazolin-4-amine%252C%2520a%2520novel%252C%2520highly%2520selective%252C%2520orally%2520available%252C%2520dual-specific%2520c-Src%252FAbl%2520kinase%2520inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D6465%26epage%3D6488%26doi%3D10.1021%2Fjm060434q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Kinase inhibitors: the road ahead</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">353</span>, <span class="refDoi"> DOI: 10.1038/nrd.2018.21</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.1038%2Fnrd.2018.21" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=29545548" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BC1cXks12ktbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2018&pages=353&author=F.+M.+Fergusonauthor=N.+S.+Gray&title=Kinase+inhibitors%3A+the+road+ahead&doi=10.1038%2Fnrd.2018.21"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Kinase inhibitors: the road ahead</span></div><div class="casAuthors">Ferguson, Fleur M.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">353-377</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  Receptor tyrosine kinase signaling pathways have been successfully targeted to inhibit proliferation and angiogenesis for cancer therapy.  However, kinase deregulation has been firmly demonstrated to play an essential role in virtually all major disease areas.  Kinase inhibitor drug discovery programs have recently broadened their focus to include an expanded range of kinase targets and therapeutic areas.  In this Review, we provide an overview of the novel targets, biol. processes and disease areas that kinase-targeting small mols. are being developed against, highlight the assocd. challenges and assess the strategies and technologies that are enabling efficient generation of highly optimized kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEZPospb1TP7Vg90H21EOLACvtfcHk0ljnmVFBRQ1Ibg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXks12ktbg%253D&md5=34e42911d8a70301905460c3507e737a</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2018.21&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2018.21%26sid%3Dliteratum%253Aachs%26aulast%3DFerguson%26aufirst%3DF.%2BM.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DKinase%2520inhibitors%253A%2520the%2520road%2520ahead%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2018%26volume%3D17%26spage%3D353%26doi%3D10.1038%2Fnrd.2018.21" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roskoski, R.</span></span> <span> </span><span class="NLM_article-title">Properties of FDA-approved small molecule protein kinase inhibitors</span>. <i>Pharmacol. Res.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>144</i></span>,  <span class="NLM_fpage">19</span>, <span class="refDoi"> DOI: 10.1016/j.phrs.2019.03.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.1016%2Fj.phrs.2019.03.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=30877063" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BC1MXntVGnurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=144&publication_year=2019&pages=19&author=R.+Roskoski&title=Properties+of+FDA-approved+small+molecule+protein+kinase+inhibitors&doi=10.1016%2Fj.phrs.2019.03.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Properties of FDA-approved small molecule protein kinase inhibitors</span></div><div class="casAuthors">Roskoski, Robert Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Research</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">144</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">19-50</span>CODEN:
                <span class="NLM_cas:coden">PHMREP</span>;
        ISSN:<span class="NLM_cas:issn">1043-6618</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Because mutations, overexpression, and dysregulation of protein kinases play essential roles in the pathogenesis of many illnesses, this enzyme family has become one of the most important drug targets in the past 20 years.  The US FDA has approved 48 small mol. protein kinase inhibitors, nearly all of which are orally effective with the exceptions of netarsudil (which is given as an eye drop) and temsirolimus (which is given i.v.).  Of the 48 approved drugs, the majority (25) target receptor protein-tyrosine kinases, ten target non-receptor protein-tyrosine kinases, and 13 target protein-serine/threonine protein kinases.  The data indicate that 43 of these drugs are used in the treatment of malignancies (36 against solid tumors including lymphomas and seven against non-solid tumors, e.g., leukemias).  Seven drugs are used in the treatment of non-malignancies: baricitinib, rheumatoid arthritis; fostamatinib, chronic immune thrombocytopenia; ruxolitinib, myelofibrosis and polycythemia vera; nintedanib, idiopathic pulmonary fibrosis; sirolimus, renal graft vs. host disease; netarsudil, glaucoma; tofacitinib, rheumatoid arthritis, Crohn disease, and ulcerative colitis.  Moreover, ibrutinib and sirolimus are used for the treatment of both malignant and non-malignant diseases.  The most common drug targets include ALK, B-Raf, BCR-Abl, epidermal growth factor receptor (EGFR), and vascular endothelial growth factor receptor (VEGFR).  Most of the small mol. inhibitors (45) interact directly with the protein kinase domain.  In contrast, sirolimus, temsirolimus, and everolimus are larger mols. (MW ≈ 1000) that bind to FKBP-12 to generate a complex that inhibits mTOR (mammalian target of rapamycin).  This review presents the available drug-enzyme X-ray crystal structures for 27 of the approved drugs as well as the chem. structures and physicochem. properties of all of the FDA-approved small mol. protein kinase antagonists.  Six of the drugs bind covalently and irreversibly to their target.  Twenty of the 48 drugs have mol. wts. greater than 500, exceeding a Lipinski rule of five criterion.  Excluding the macrolides (everolimus, sirolimus, temsirolimus), the av. mol. wt. of drugs is 480 with a range of 306 (ruxolitinib) to 615 (trametinib).  Nearly half of the antagonists (23) have a lipophilic efficiency with values of less than five while the recommended optima range from 5-10.  One of the vexing problems is the near universal development of resistance that is assocd. with the use of small mol. protein kinase inhibitors for the treatment of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLum_hDK00hbVg90H21EOLACvtfcHk0ljnmVFBRQ1Ibg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXntVGnurg%253D&md5=a24dee87603bc4ea12bd34616e46f53d</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.phrs.2019.03.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.phrs.2019.03.006%26sid%3Dliteratum%253Aachs%26aulast%3DRoskoski%26aufirst%3DR.%26atitle%3DProperties%2520of%2520FDA-approved%2520small%2520molecule%2520protein%2520kinase%2520inhibitors%26jtitle%3DPharmacol.%2520Res.%26date%3D2019%26volume%3D144%26spage%3D19%26doi%3D10.1016%2Fj.phrs.2019.03.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sabnis, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buhrlage, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Developing irreversible inhibitors of the protein kinase cysteinome</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">146</span>– <span class="NLM_lpage">159</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2012.12.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.1016%2Fj.chembiol.2012.12.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=23438744" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtFWitbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2013&pages=146-159&author=Q.+Liuauthor=Y.+Sabnisauthor=Z.+Zhaoauthor=T.+Zhangauthor=S.+J.+Buhrlageauthor=L.+H.+Jonesauthor=N.+S.+Gray&title=Developing+irreversible+inhibitors+of+the+protein+kinase+cysteinome&doi=10.1016%2Fj.chembiol.2012.12.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Developing Irreversible Inhibitors of the Protein Kinase Cysteinome</span></div><div class="casAuthors">Liu, Qingsong; Sabnis, Yogesh; Zhao, Zheng; Zhang, Tinghu; Buhrlage, Sara J.; Jones, Lyn H.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">146-159</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Protein kinases are a large family of approx. 530 highly conserved enzymes that transfer a γ-phosphate group from ATP to a variety of amino acid residues, such as tyrosine, serine, and threonine, that serves as a ubiquitous mechanism for cellular signal transduction.  The clin. success of a no. of kinase-directed drugs and the frequent observation of disease causing mutations in protein kinases suggest that a large no. of kinases may represent therapeutically relevant targets.  To date, the majority of clin. and preclin. kinase inhibitors are ATP competitive, noncovalent inhibitors that achieve selectivity through recognition of unique features of particular protein kinases.  Recently, there has been renewed interest in the development of irreversible inhibitors that form covalent bonds with cysteine or other nucleophilic residues in the ATP-binding pocket.  Irreversible kinase inhibitors have a no. of potential advantages including prolonged pharmacodynamics, suitability for rational design, high potency, and ability to validate pharmacol. specificity through mutation of the reactive cysteine residue.  Here, we review recent efforts to develop cysteine-targeted irreversible protein kinase inhibitors and discuss their modes of recognizing the ATP-binding pocket and their biol. activity profiles.  In addn., we provided an informatics assessment of the potential "kinase cysteinome" and discuss strategies for the efficient development of new covalent inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVGD7TX3oIeLVg90H21EOLACvtfcHk0ljnmVFBRQ1Ibg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtFWitbs%253D&md5=cef80823123bae88362e816a0a638133</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2012.12.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2012.12.006%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DSabnis%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DBuhrlage%26aufirst%3DS.%2BJ.%26aulast%3DJones%26aufirst%3DL.%2BH.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DDeveloping%2520irreversible%2520inhibitors%2520of%2520the%2520protein%2520kinase%2520cysteinome%26jtitle%3DChem.%2520Biol.%26date%3D2013%26volume%3D20%26spage%3D146%26epage%3D159%26doi%3D10.1016%2Fj.chembiol.2012.12.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanizaki, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aref, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rusan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, S.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ercan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, R. G.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Development of covalent inhibitors that can overcome resistance to first-generation FGFR kinase inhibitors</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>111</i></span>,  <span class="NLM_fpage">E4869</span>– <span class="NLM_lpage">E4877</span>, <span class="refDoi"> DOI: 10.1073/pnas.1403438111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.1073%2Fpnas.1403438111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=25349422" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVWmsL%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2014&pages=E4869-E4877&author=L.+Tanauthor=J.+Wangauthor=J.+Tanizakiauthor=Z.+Huangauthor=A.+R.+Arefauthor=M.+Rusanauthor=S.-J.+Zhuauthor=Y.+Zhangauthor=D.+Ercanauthor=R.+G.+Liao&title=Development+of+covalent+inhibitors+that+can+overcome+resistance+to+first-generation+FGFR+kinase+inhibitors&doi=10.1073%2Fpnas.1403438111"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Development of covalent inhibitors that can overcome resistance to first-generation FGFR kinase inhibitors</span></div><div class="casAuthors">Tan, Li; Wang, Jun; Tanizaki, Junko; Huang, Zhifeng; Aref, Amir R.; Rusan, Maria; Zhu, Su-Jie; Zhang, Yiyun; Ercan, Dalia; Liao, Rachel G.; Capelletti, Marzia; Zhou, Wenjun; Hur, Wooyoung; Kim, NamDoo; Sim, Taebo; Gaudet, Suzanne; Barbie, David A.; Yeh, Jing-Ruey Joanna; Yun, Cai-Hong; Hammerman, Peter S.; Mohammadi, Moosa; Janne, Pasi A.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">45</span>),
    <span class="NLM_cas:pages">E4869-E4877</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The human FGF receptors (FGFRs) play crit. roles in various human cancers, and several FGFR inhibitors are currently under clin. investigation.  Resistance usually results from selection for mutant kinases that are impervious to the action of the drug or from up-regulation of compensatory signaling pathways.  Preclin. studies have demonstrated that resistance to FGFR inhibitors can be acquired through mutations in the FGFR gatekeeper residue, as clin. obsd. for FGFR4 in embryonal rhabdomyosarcoma and neuroendocrine breast carcinomas.  Here the authors report on the use of a structure-based drug design to develop two selective, next-generation covalent FGFR inhibitors, the FGFR irreversible inhibitors I and II.  To our knowledge, I and II are the first inhibitors that can potently inhibit the proliferation of cells dependent upon the gatekeeper mutants of FGFR1 or FGFR2, which confer resistance to first-generation clin. FGFR inhibitors such as NVP-BGJ398 and AZD4547.  Because of the conformational flexibility of the reactive acrylamide substituent, II has the unprecedented ability to inhibit both the EGF receptor (EGFR) and FGFR covalently by targeting two distinct cysteine residues.  The authors report the cocrystal structure of FGFR4 with I, which unexpectedly exhibits a "DFG-out" covalent binding mode.  The structural basis for dual FGFR and EGFR targeting by FIIN3 also is illustrated by crystal structures of II bound with FGFR4 V550L and EGFR L858R.  These results have important implications for the design of covalent FGFR inhibitors that can overcome clin. resistance and provide the first example, to the knowledge, of a kinase inhibitor that covalently targets cysteines located in different positions within the ATP-binding pocket.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmCbTDKPA32LVg90H21EOLACvtfcHk0lhnGufr7kWRLg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVWmsL%252FP&md5=fd4a7eac1e0c17e7c9f2651dd2d76ac7</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1403438111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1403438111%26sid%3Dliteratum%253Aachs%26aulast%3DTan%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DTanizaki%26aufirst%3DJ.%26aulast%3DHuang%26aufirst%3DZ.%26aulast%3DAref%26aufirst%3DA.%2BR.%26aulast%3DRusan%26aufirst%3DM.%26aulast%3DZhu%26aufirst%3DS.-J.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DErcan%26aufirst%3DD.%26aulast%3DLiao%26aufirst%3DR.%2BG.%26atitle%3DDevelopment%2520of%2520covalent%2520inhibitors%2520that%2520can%2520overcome%2520resistance%2520to%2520first-generation%2520FGFR%2520kinase%2520inhibitors%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2014%26volume%3D111%26spage%3DE4869%26epage%3DE4877%26doi%3D10.1073%2Fpnas.1403438111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kwarcinski, F. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fox, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steffey, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soellner, M. B.</span></span> <span> </span><span class="NLM_article-title">Irreversible inhibitors of c-Src kinase that target a nonconserved cysteine</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">1910</span>– <span class="NLM_lpage">1917</span>, <span class="refDoi"> DOI: 10.1021/cb300337u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb300337u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1Kms77J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=1910-1917&author=F.+E.+Kwarcinskiauthor=C.+C.+Foxauthor=M.+E.+Steffeyauthor=M.+B.+Soellner&title=Irreversible+inhibitors+of+c-Src+kinase+that+target+a+nonconserved+cysteine&doi=10.1021%2Fcb300337u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Irreversible Inhibitors of c-Src Kinase That Target a Nonconserved Cysteine</span></div><div class="casAuthors">Kwarcinski, Frank E.; Fox, Christel C.; Steffey, Michael E.; Soellner, Matthew B.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1910-1917</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We have developed the first irreversible inhibitors of wild-type c-Src kinase.  We demonstrate that our irreversible inhibitors display improved potency and selectivity relative to that of their reversible counterparts.  Our strategy involves modifying a promiscuous kinase inhibitor with an electrophile to generate covalent inhibitors of c-Src.  We applied this methodol. to two inhibitor scaffolds that exhibit increased cellular efficacy when rendered irreversible.  In addn., we have demonstrated the utility of irreversible inhibitors in studying the conformation of an important loop in kinases that can control inhibitor selectivity and cause drug resistance.  Together, we have developed a general and robust framework for generating selective irreversible inhibitors from reversible, promiscuous inhibitor scaffolds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcojrr4XAxmLVg90H21EOLACvtfcHk0lhnGufr7kWRLg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1Kms77J&md5=db928e772f1ed18be2ac37ec930d743c</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fcb300337u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb300337u%26sid%3Dliteratum%253Aachs%26aulast%3DKwarcinski%26aufirst%3DF.%2BE.%26aulast%3DFox%26aufirst%3DC.%2BC.%26aulast%3DSteffey%26aufirst%3DM.%2BE.%26aulast%3DSoellner%26aufirst%3DM.%2BB.%26atitle%3DIrreversible%2520inhibitors%2520of%2520c-Src%2520kinase%2520that%2520target%2520a%2520nonconserved%2520cysteine%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2012%26volume%3D7%26spage%3D1910%26epage%3D1917%26doi%3D10.1021%2Fcb300337u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gurbani, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weisberg, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunter, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ficarro, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mowafy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tam, C.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, S.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Structure-guided development of covalent TAK1 inhibitors</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">838</span>– <span class="NLM_lpage">846</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2016.11.035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.1016%2Fj.bmc.2016.11.035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=28011204" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFKlsbjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2017&pages=838-846&author=L.+Tanauthor=D.+Gurbaniauthor=E.+L.+Weisbergauthor=J.+C.+Hunterauthor=L.+Liauthor=D.+S.+Jonesauthor=S.+B.+Ficarroauthor=S.+Mowafyauthor=C.-P.+Tamauthor=S.+Rao&title=Structure-guided+development+of+covalent+TAK1+inhibitors&doi=10.1016%2Fj.bmc.2016.11.035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-guided development of covalent TAK1 inhibitors</span></div><div class="casAuthors">Tan, Li; Gurbani, Deepak; Weisberg, Ellen L.; Hunter, John C.; Li, Lianbo; Jones, Douglas S.; Ficarro, Scott B.; Mowafy, Samar; Tam, Chun-Pong; Rao, Suman; Du, Guangyan; Griffin, James D.; Sorger, Peter K.; Marto, Jarrod A.; Westover, Kenneth D.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">838-846</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">TAK1 (transforming growth factor-β-activated kinase 1) is an essential intracellular mediator of cytokine and growth factor signaling and a potential therapeutic target for the treatment of immune diseases and cancer.  Herein we report development of a series of 2,4-disubstituted pyrimidine covalent TAK1 inhibitors that target Cys174, a residue immediately adjacent to the 'DFG-motif' of the kinase activation loop.  Co-crystal structures of TAK1 with candidate compds. enabled iterative rounds of structure-based design and biol. testing to arrive at optimized compds.  Lead compds. such as 2 and 10 showed greater than 10-fold biochem. selectivity for TAK1 over the closely related kinases MEK1 and ERK1 which possess an equivalently positioned cysteine residue.  These compds. are smaller, more easily synthesized, and exhibit a different spectrum of kinase selectivity relative to previously reported macrocyclic natural product TAK1 inhibitors such as 5Z-7-oxozeanol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkSYJFH5tOKrVg90H21EOLACvtfcHk0lhnGufr7kWRLg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFKlsbjO&md5=8dda5fe018d344e1950e9641a01de54e</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2016.11.035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2016.11.035%26sid%3Dliteratum%253Aachs%26aulast%3DTan%26aufirst%3DL.%26aulast%3DGurbani%26aufirst%3DD.%26aulast%3DWeisberg%26aufirst%3DE.%2BL.%26aulast%3DHunter%26aufirst%3DJ.%2BC.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DJones%26aufirst%3DD.%2BS.%26aulast%3DFicarro%26aufirst%3DS.%2BB.%26aulast%3DMowafy%26aufirst%3DS.%26aulast%3DTam%26aufirst%3DC.-P.%26aulast%3DRao%26aufirst%3DS.%26atitle%3DStructure-guided%2520development%2520of%2520covalent%2520TAK1%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2017%26volume%3D25%26spage%3D838%26epage%3D846%26doi%3D10.1016%2Fj.bmc.2016.11.035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gurbani, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weisberg, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singer, W. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernard, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mowafy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenney, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, G.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Studies of TAK1-centered polypharmacology with novel covalent TAK1 inhibitors</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1320</span>– <span class="NLM_lpage">1328</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2016.11.034</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.1016%2Fj.bmc.2016.11.034" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=28038940" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BC2sXit1yktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2017&pages=1320-1328&author=L.+Tanauthor=D.+Gurbaniauthor=E.+L.+Weisbergauthor=D.+S.+Jonesauthor=S.+Raoauthor=W.+D.+Singerauthor=F.+M.+Bernardauthor=S.+Mowafyauthor=A.+Jenneyauthor=G.+Du&title=Studies+of+TAK1-centered+polypharmacology+with+novel+covalent+TAK1+inhibitors&doi=10.1016%2Fj.bmc.2016.11.034"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Studies of TAK1-centered polypharmacology with novel covalent TAK1 inhibitors</span></div><div class="casAuthors">Tan, Li; Gurbani, Deepak; Weisberg, Ellen L.; Jones, Douglas S.; Rao, Suman; Singer, William D.; Bernard, Faviola M.; Mowafy, Samar; Jenney, Annie; Du, Guangyan; Nonami, Atsushi; Griffin, James D.; Lauffenburger, Douglas A.; Westover, Kenneth D.; Sorger, Peter K.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1320-1328</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Targeted polypharmacol. provides an efficient method of treating diseases such as cancer with complex, multigenic causes provided that compds. with advantageous activity profiles can be discovered.  Novel covalent TAK1 inhibitors were validated in cellular contexts for their ability to inhibit the TAK1 kinase and for their polypharmacol.  Several inhibitors phenocopied reported TAK1 inhibitor 5Z-7-oxozaenol with comparable efficacy and complementary kinase selectivity profiles.  Compd. 5 exhibited the greatest potency in RAS-mutated and wild-type RAS cell lines from various cancer types.  A biotinylated deriv. of 5, 27, was used to verify TAK1 binding in cells.  The newly described inhibitors constitute useful tools for further development of multi-targeting TAK1-centered inhibitors for cancer and other diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolPDq91GVGo7Vg90H21EOLACvtfcHk0lgQBi1PNOjpUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXit1yktg%253D%253D&md5=ae5a9dffd65178d6430ce1c766735650</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2016.11.034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2016.11.034%26sid%3Dliteratum%253Aachs%26aulast%3DTan%26aufirst%3DL.%26aulast%3DGurbani%26aufirst%3DD.%26aulast%3DWeisberg%26aufirst%3DE.%2BL.%26aulast%3DJones%26aufirst%3DD.%2BS.%26aulast%3DRao%26aufirst%3DS.%26aulast%3DSinger%26aufirst%3DW.%2BD.%26aulast%3DBernard%26aufirst%3DF.%2BM.%26aulast%3DMowafy%26aufirst%3DS.%26aulast%3DJenney%26aufirst%3DA.%26aulast%3DDu%26aufirst%3DG.%26atitle%3DStudies%2520of%2520TAK1-centered%2520polypharmacology%2520with%2520novel%2520covalent%2520TAK1%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2017%26volume%3D25%26spage%3D1320%26epage%3D1328%26doi%3D10.1016%2Fj.bmc.2016.11.034" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gurbani, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Everley, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Browne, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaikuad, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schröder, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gondi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ficarro, S. B.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Leveraging compound promiscuity to identify targetable cysteines within the kinome</span>. <i>Cell chemical biology</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">818</span>– <span class="NLM_lpage">829.e9</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2019.02.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.1016%2Fj.chembiol.2019.02.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=30982749" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BC1MXnt1CksL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2019&pages=818-829.e9&author=S.+Raoauthor=D.+Gurbaniauthor=G.+Duauthor=R.+A.+Everleyauthor=C.+M.+Browneauthor=A.+Chaikuadauthor=L.+Tanauthor=M.+Schr%C3%B6derauthor=S.+Gondiauthor=S.+B.+Ficarro&title=Leveraging+compound+promiscuity+to+identify+targetable+cysteines+within+the+kinome&doi=10.1016%2Fj.chembiol.2019.02.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Leveraging Compound Promiscuity to Identify Targetable Cysteines within the Kinome</span></div><div class="casAuthors">Rao, Suman; Gurbani, Deepak; Du, Guangyan; Everley, Robert A.; Browne, Christopher M.; Chaikuad, Apirat; Li, Tan; Schroder, Martin; Gondi, Sudershan; Ficarro, Scott B.; Sim, Taebo; Kim, Nam Doo; Berberich, Matthew J.; Knapp, Stefan; Marto, Jarrod A.; Westover, Kenneth D.; Sorger, Peter K.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">818-829.e9</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Covalent kinase inhibitors, which typically target cysteine residues, represent an important class of clin. relevant compds.  Approx. 215 kinases are known to have potentially targetable cysteines distributed across 18 spatially distinct locations proximal to the ATP-binding pocket.  However, only 40 kinases have been covalently targeted, with certain cysteine sites being the primary focus.  To address this disparity, we have developed a strategy that combines the use of a multi-targeted acrylamide-modified inhibitor, SM1-71, with a suite of complementary chemoproteomic and cellular approaches to identify addnl. targetable cysteines.  Using this single multi-targeted compd., we successfully identified 23 kinases that are amenable to covalent inhibition including MKNK2, MAP2K1/2/3/4/6/7, GAK, AAK1, BMP2K, MAP3K7, MAPKAPK5, GSK3A/B, MAPK1/3, SRC, YES1, FGFR1, ZAK (MLTK), MAP3K1, LIMK1, and RSK2.  The identification of nine of these kinases previously not targeted by a covalent inhibitor increases the no. of targetable kinases and highlights opportunities for covalent kinase inhibitor development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUcHJFGWFTTrVg90H21EOLACvtfcHk0lgQBi1PNOjpUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXnt1CksL4%253D&md5=6a8656d083fc4d49fa8520b48e05d100</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2019.02.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2019.02.021%26sid%3Dliteratum%253Aachs%26aulast%3DRao%26aufirst%3DS.%26aulast%3DGurbani%26aufirst%3DD.%26aulast%3DDu%26aufirst%3DG.%26aulast%3DEverley%26aufirst%3DR.%2BA.%26aulast%3DBrowne%26aufirst%3DC.%2BM.%26aulast%3DChaikuad%26aufirst%3DA.%26aulast%3DTan%26aufirst%3DL.%26aulast%3DSchr%25C3%25B6der%26aufirst%3DM.%26aulast%3DGondi%26aufirst%3DS.%26aulast%3DFicarro%26aufirst%3DS.%2BB.%26atitle%3DLeveraging%2520compound%2520promiscuity%2520to%2520identify%2520targetable%2520cysteines%2520within%2520the%2520kinome%26jtitle%3DCell%2520chemical%2520biology%26date%3D2019%26volume%3D26%26spage%3D818%26epage%3D829.e9%26doi%3D10.1016%2Fj.chembiol.2019.02.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borrelli, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenwood, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Day, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farid, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harder, E.</span></span> <span> </span><span class="NLM_article-title">Docking covalent inhibitors: a parameter free approach to pose prediction and scoring</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">1932</span>– <span class="NLM_lpage">1940</span>, <span class="refDoi"> DOI: 10.1021/ci500118s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci500118s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BC2cXpsVKksLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=1932-1940&author=K.+Zhuauthor=K.+W.+Borrelliauthor=J.+R.+Greenwoodauthor=T.+Dayauthor=R.+Abelauthor=R.+S.+Faridauthor=E.+Harder&title=Docking+covalent+inhibitors%3A+a+parameter+free+approach+to+pose+prediction+and+scoring&doi=10.1021%2Fci500118s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Docking Covalent Inhibitors: A Parameter Free Approach To Pose Prediction and Scoring</span></div><div class="casAuthors">Zhu, Kai; Borrelli, Kenneth W.; Greenwood, Jeremy R.; Day, Tyler; Abel, Robert; Farid, Ramy S.; Harder, Edward</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1932-1940</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Although many popular docking programs include a facility to account for covalent ligands, large-scale systematic docking validation studies of covalent inhibitors have been sparse.  In this paper, we present the development and validation of a novel approach for docking and scoring covalent inhibitors, which consists of conventional noncovalent docking, heuristic formation of the covalent attachment point, and structural refinement of the protein-ligand complex.  This approach combines the strengths of the docking program Glide and the protein structure modeling program Prime and does not require any parameter fitting for the study of addnl. covalent reaction types.  We first test this method by predicting the native binding geometry of 38 covalently bound complexes.  The av. RMSD of the predicted poses is 1.52 Å, and 76% of test set inhibitors have an RMSD of less than 2.0 Å.  In addn., the apparent affinity score constructed herein is tested on a virtual screening study and the characterization of the SAR properties of two different series of congeneric compds. with satisfactory success.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbtjEb3EqzQ7Vg90H21EOLACvtfcHk0lgQBi1PNOjpUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXpsVKksLs%253D&md5=3633c29fa5dc29817b178fd10a4c7b15</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fci500118s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci500118s%26sid%3Dliteratum%253Aachs%26aulast%3DZhu%26aufirst%3DK.%26aulast%3DBorrelli%26aufirst%3DK.%2BW.%26aulast%3DGreenwood%26aufirst%3DJ.%2BR.%26aulast%3DDay%26aufirst%3DT.%26aulast%3DAbel%26aufirst%3DR.%26aulast%3DFarid%26aufirst%3DR.%2BS.%26aulast%3DHarder%26aufirst%3DE.%26atitle%3DDocking%2520covalent%2520inhibitors%253A%2520a%2520parameter%2520free%2520approach%2520to%2520pose%2520prediction%2520and%2520scoring%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D1932%26epage%3D1940%26doi%3D10.1021%2Fci500118s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clark, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hafner, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kouril, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muhlich, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pilarczyk, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niepel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sorger, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Medvedovic, M.</span></span> <span> </span><span class="NLM_article-title">GRcalculator: an online tool for calculating and mining dose–response data</span>. <i>BMC Cancer</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">698</span>, <span class="refDoi"> DOI: 10.1186/s12885-017-3689-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.1186%2Fs12885-017-3689-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=29065900" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVOgsr7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2017&pages=698&author=N.+A.+Clarkauthor=M.+Hafnerauthor=M.+Kourilauthor=E.+H.+Williamsauthor=J.+L.+Muhlichauthor=M.+Pilarczykauthor=M.+Niepelauthor=P.+K.+Sorgerauthor=M.+Medvedovic&title=GRcalculator%3A+an+online+tool+for+calculating+and+mining+dose%E2%80%93response+data&doi=10.1186%2Fs12885-017-3689-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">GRcalculator: an online tool for calculating and mining dose-response data</span></div><div class="casAuthors">Clark, Nicholas A.; Hafner, Marc; Kouril, Michal; Williams, Elizabeth H.; Muhlich, Jeremy L.; Pilarczyk, Marcin; Niepel, Mario; Sorger, Peter K.; Medvedovic, Mario</div><div class="citationInfo"><span class="NLM_cas:title">BMC Cancer</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">698/1-698/11</span>CODEN:
                <span class="NLM_cas:coden">BCMACL</span>;
        ISSN:<span class="NLM_cas:issn">1471-2407</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Background: Quantifying the response of cell lines to drugs or other perturbagens is the cornerstone of pre-clin. drug development and pharmacogenomics as well as a means to study factors that contribute to sensitivity and resistance.  In dividing cells, traditional metrics derived from dose-response curves such as IC50, AUC, and Emax, are confounded by the no. of cell divisions taking place during the assay, which varies widely for biol. and exptl. reasons.  Hafner et al. (Nat Meth 13:521.-+. 627, 2016) recently proposed an alternative way to quantify drug response, normalized growth rate (GR) inhibition, that is robust to such confounders.  Adoption of the GR method is expected to improve the reproducibility of dose - response assays and the reliability of pharmacogenomic assocns. (Hafner et al. 500-502, 2017).  Results: We describe here an interactive website for calcn., anal., and visualization of dose-response data using the GR approach and for comparison of GR and traditional metrics.  Data can be user-supplied or derived from published datasets.  The web tools are implemented in the form of three integrated Shiny applications (grcalculator, grbrowser, and grtutorial) deployed through a Shiny server.  Intuitive graphical user interfaces (GUIs) allow for interactive anal. and visualization of data.  The Shiny applications make use of two R packages (shinyLi and GRmetrics) specifically developed for this purpose.  The GRmetrics R package is also available via Bioconductor and can be used for offline data anal. and visualization.  Conclusions: GRcalculator is a powerful, user-friendly, and free tool to facilitate anal. of dose - response data.  It generates publication-ready figures and provides a unified platform for investigators to analyze dose - response data across diverse cell types and perturbagens (including drugs, biol. ligands, RNAi, etc.).  GRcalculator also provides access to data collected by the NIH LINCS Program and other public domain datasets.  The GRmetrics Bioconductor package provides computationally trained users with a platform for offline anal. of dose-response data and facilitates inclusion of GR metrics calcns. within existing R anal. pipelines.  These tools are therefore well suited to users in academia as well as industry.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpSN136VkelbrVg90H21EOLACvtfcHk0lhRM-YMkzID2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVOgsr7N&md5=35b31d3eecee58f552392a4611bf655d</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1186%2Fs12885-017-3689-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12885-017-3689-3%26sid%3Dliteratum%253Aachs%26aulast%3DClark%26aufirst%3DN.%2BA.%26aulast%3DHafner%26aufirst%3DM.%26aulast%3DKouril%26aufirst%3DM.%26aulast%3DWilliams%26aufirst%3DE.%2BH.%26aulast%3DMuhlich%26aufirst%3DJ.%2BL.%26aulast%3DPilarczyk%26aufirst%3DM.%26aulast%3DNiepel%26aufirst%3DM.%26aulast%3DSorger%26aufirst%3DP.%2BK.%26aulast%3DMedvedovic%26aufirst%3DM.%26atitle%3DGRcalculator%253A%2520an%2520online%2520tool%2520for%2520calculating%2520and%2520mining%2520dose%25E2%2580%2593response%2520data%26jtitle%3DBMC%2520Cancer%26date%3D2017%26volume%3D17%26spage%3D698%26doi%3D10.1186%2Fs12885-017-3689-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sánchez-Bailón, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calcabrini, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gómez-Domínguez, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morte, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martín-Forero, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gómez-López, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molinari, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, K.-U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martín-Pérez, J.</span></span> <span> </span><span class="NLM_article-title">Src kinases catalytic activity regulates proliferation, migration and invasiveness of MDA-MB-231 breast cancer cells</span>. <i>Cell. Signalling</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">1276</span>– <span class="NLM_lpage">1286</span>, <span class="refDoi"> DOI: 10.1016/j.cellsig.2012.02.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.1016%2Fj.cellsig.2012.02.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=22570868" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BC38XkvF2itrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2012&pages=1276-1286&author=M.+P.+S%C3%A1nchez-Bail%C3%B3nauthor=A.+Calcabriniauthor=D.+G%C3%B3mez-Dom%C3%ADnguezauthor=B.+Morteauthor=E.+Mart%C3%ADn-Foreroauthor=G.+G%C3%B3mez-L%C3%B3pezauthor=A.+Molinariauthor=K.-U.+Wagnerauthor=J.+Mart%C3%ADn-P%C3%A9rez&title=Src+kinases+catalytic+activity+regulates+proliferation%2C+migration+and+invasiveness+of+MDA-MB-231+breast+cancer+cells&doi=10.1016%2Fj.cellsig.2012.02.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Src kinases catalytic activity regulates proliferation, migration and invasiveness of MDA-MB-231 breast cancer cells</span></div><div class="casAuthors">Sanchez-Bailon, Maria Pilar; Calcabrini, Annarica; Gomez-Dominguez, Daniel; Morte, Beatriz; Martin-Forero, Esther; Gomez-Lopez, Gonzalo; Molinari, Agnese; Wagner, Kay-Uwe; Martin-Perez, Jorge</div><div class="citationInfo"><span class="NLM_cas:title">Cellular Signalling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1276-1286</span>CODEN:
                <span class="NLM_cas:coden">CESIEY</span>;
        ISSN:<span class="NLM_cas:issn">0898-6568</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">SFKs are frequently deregulated in cancer where they control cellular proliferation, migration, survival and metastasis.  Here we study the role of SFKs catalytic activity in triple-neg./basal-like and metastatic human breast cancer MDA-MB-231 cells employing three well-established inhibitors: Dasatinib, PP2 and SU6656.  These compds. inhibited migration and invasion.  Concomitantly, they reduced Fak, paxillin, p130CAS, caveolin-1 phosphorylation and altered cytoskeletal structures.  They also inhibited cell proliferation, but in different manners.  Dasatinib and PP2 increased p27Kip1 expression and reduced c-Myc levels, restraining G1-S transition.  In contrast, SU6656 did not modify p27Kip1 expression, slightly altered c-Myc levels and generated polyploid multinucleated cells, indicating inhibition of cytokinesis.  These later effects were also obsd. in SYF fibroblasts, suggesting a SFKs-independent action.  ZM447439, an Aurora B kinase inhibitor, produced similar cell cycle and morphol. alterations in MDA-MB-231 cells, indicating that SU6656 blocked Aurora B kinase.  This was confirmed by inhibition of histone H3 phosphorylation, the canonical Aurora B kinase substrate.  Furthermore, hierarchical clustering anal. of gene expression profiles showed that SU6656 defined a set of genes that differed from Dasatinib and PP2.  Addnl., Gene Set Enrichment Analyses revealed that SU6656 significantly reduces the Src pathway.  Together, these results show the importance of SFKs catalytic activity for MDA-MB-231 proliferation, migration and invasiveness.  They also illustrate that SU6656 acts as dual SFKs and Aurora B kinase inhibitor, suggesting its possible use as a therapeutic agent in breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxhZ62Ycaq7bVg90H21EOLACvtfcHk0lhRM-YMkzID2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkvF2itrw%253D&md5=527ed67289deb9acc1ab9a6fe15b527d</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2Fj.cellsig.2012.02.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cellsig.2012.02.011%26sid%3Dliteratum%253Aachs%26aulast%3DS%25C3%25A1nchez-Bail%25C3%25B3n%26aufirst%3DM.%2BP.%26aulast%3DCalcabrini%26aufirst%3DA.%26aulast%3DG%25C3%25B3mez-Dom%25C3%25ADnguez%26aufirst%3DD.%26aulast%3DMorte%26aufirst%3DB.%26aulast%3DMart%25C3%25ADn-Forero%26aufirst%3DE.%26aulast%3DG%25C3%25B3mez-L%25C3%25B3pez%26aufirst%3DG.%26aulast%3DMolinari%26aufirst%3DA.%26aulast%3DWagner%26aufirst%3DK.-U.%26aulast%3DMart%25C3%25ADn-P%25C3%25A9rez%26aufirst%3DJ.%26atitle%3DSrc%2520kinases%2520catalytic%2520activity%2520regulates%2520proliferation%252C%2520migration%2520and%2520invasiveness%2520of%2520MDA-MB-231%2520breast%2520cancer%2520cells%26jtitle%3DCell.%2520Signalling%26date%3D2012%26volume%3D24%26spage%3D1276%26epage%3D1286%26doi%3D10.1016%2Fj.cellsig.2012.02.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Formisano, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Amato, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Servetto, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brillante, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raimondo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Mauro, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marciano, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orsini, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cosconati, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Randazzo, A.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Src inhibitors act through different mechanisms in Non-Small Cell Lung Cancer models depending on EGFR and RAS mutational status</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">26090</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.4636</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.18632%2Foncotarget.4636" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=26325669" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A280%3ADC%252BC287otVejsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=26090&author=L.+Formisanoauthor=V.+D%E2%80%99Amatoauthor=A.+Servettoauthor=S.+Brillanteauthor=L.+Raimondoauthor=C.+Di+Mauroauthor=R.+Marcianoauthor=R.+C.+Orsiniauthor=S.+Cosconatiauthor=A.+Randazzo&title=Src+inhibitors+act+through+different+mechanisms+in+Non-Small+Cell+Lung+Cancer+models+depending+on+EGFR+and+RAS+mutational+status&doi=10.18632%2Foncotarget.4636"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Src inhibitors act through different mechanisms in Non-Small Cell Lung Cancer models depending on EGFR and RAS mutational status</span></div><div class="casAuthors">Formisano Luigi; D'Amato Valentina; Servetto Alberto; Brillante Simona; Raimondo Lucia; Di Mauro Concetta; Marciano Roberta; Orsini Roberta Clara; De Placido Sabino; Rosa Roberta; Bianco Roberto; Cosconati Sandro; Randazzo Antonio; Parsons Sarah J; Montuori Nunzia; Veneziani Bianca Maria</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">28</span>),
    <span class="NLM_cas:pages">26090-103</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Resistance to the EGFR tyrosine kinase inhibitors (TKIs) gefitinib and erlotinib, often related to Ras or secondary EGFR mutations, is a relevant clinical issue in Non-Small Cell Lung Cancer (NSCLC).  Although Src TK has been involved in such resistance, clinical development of its inhibitors has been so far limited.  To better define the molecular targets of the Src TKIs saracatinib, dasatinib and bosutinib, we used a variety of in vitro/in vivo studies.  Kinase assays supported by docking analysis demonstrated that all the compounds directly inhibit EGFR TK variants.  However, in live cells only saracatinib efficiently reduced EGFR activation, while dasatinib was the most effective agent in inhibiting Src TK.  Consistently, a pronounced anti-proliferative effect was achieved with saracatinib, in EGFR mutant cells, or with dasatinib, in wt EGFR/Ras mutant cells, poorly dependent on EGFR and erlotinib-resistant.  We then identified the most effective drug combinations to overcome resistance to EGFR inhibitors, both in vitro and in nude mice: in T790M EGFR erlotinib-resistant cells, saracatinib with the anti-EGFR mAb cetuximab; in Ras mutant erlotinib-resistant models, dasatinib with the MEK inhibitor selumetinib.  Src inhibitors may act with different mechanisms in NSCLCs, depending on EGFR/Ras mutational profile, and may be integrated with EGFR or MEK inhibitors for different cohorts of NSCLCs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTCDsEQcCnxNdFxYDaSSP8XfW6udTcc2eYANEZxWw99YLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC287otVejsg%253D%253D&md5=45a98a239abd2001198e5e6791efe32f</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.4636&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.4636%26sid%3Dliteratum%253Aachs%26aulast%3DFormisano%26aufirst%3DL.%26aulast%3DD%25E2%2580%2599Amato%26aufirst%3DV.%26aulast%3DServetto%26aufirst%3DA.%26aulast%3DBrillante%26aufirst%3DS.%26aulast%3DRaimondo%26aufirst%3DL.%26aulast%3DDi%2BMauro%26aufirst%3DC.%26aulast%3DMarciano%26aufirst%3DR.%26aulast%3DOrsini%26aufirst%3DR.%2BC.%26aulast%3DCosconati%26aufirst%3DS.%26aulast%3DRandazzo%26aufirst%3DA.%26atitle%3DSrc%2520inhibitors%2520act%2520through%2520different%2520mechanisms%2520in%2520Non-Small%2520Cell%2520Lung%2520Cancer%2520models%2520depending%2520on%2520EGFR%2520and%2520RAS%2520mutational%2520status%26jtitle%3DOncotarget%26date%3D2015%26volume%3D6%26spage%3D26090%26doi%3D10.18632%2Foncotarget.4636" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Giaccone, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zucali, P.</span></span> <span> </span><span class="NLM_article-title">Src as a potential therapeutic target in non-small-cell lung cancer</span>. <i>Annals of Oncology</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">1219</span>– <span class="NLM_lpage">1223</span>, <span class="refDoi"> DOI: 10.1093/annonc/mdn048</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.1093%2Fannonc%2Fmdn048" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=18388349" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A280%3ADC%252BD1cvgvVyrsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2008&pages=1219-1223&author=G.+Giacconeauthor=P.+Zucali&title=Src+as+a+potential+therapeutic+target+in+non-small-cell+lung+cancer&doi=10.1093%2Fannonc%2Fmdn048"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Src as a potential therapeutic target in non-small-cell lung cancer</span></div><div class="casAuthors">Giaccone G; Zucali P A</div><div class="citationInfo"><span class="NLM_cas:title">Annals of oncology : official journal of the European Society for Medical Oncology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1219-1223</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Lung cancer is the most common cause of cancer-related death, with non-small-cell lung cancer (NSCLC) accounting for 80%-85% of all cases.  Although survival rates are reasonably good for patients diagnosed with very early disease, the majority of patients present with advanced disease.  For these patients, palliation and improvements in quality of life are the primary goals of therapy.  Although chemotherapeutic agents remain the cornerstone of first-line therapy, these agents have limited use in patients who have relapsed and have metastatic disease.  Therefore, new strategies are required to improve survival and quality of life in this setting.  With the substantial advances in our understanding of tumour biology, it has been possible to identify signalling pathways involved in mediating tumour growth and progression.  These pathways offer targets for new biological agents such as small molecule inhibitors and monoclonal antibodies.  One such target is Src, a tyrosine kinase that is involved in multiple aspects of tumorigenesis including proliferation, migration and angiogenesis.  Increased levels of Src expression have been found in a range of cancers, especially breast, colorectal, prostate and lung.  Preliminary preclinical data and pharmacodynamic data suggest that Src inhibition is a viable therapeutic option in the treatment of advanced NSCLC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSDd4hYJocBQtj_Vg2vObihfW6udTcc2ebiQO1yVN51pLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1cvgvVyrsw%253D%253D&md5=9e95b141cb44a1999b0625e34f30861a</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1093%2Fannonc%2Fmdn048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fannonc%252Fmdn048%26sid%3Dliteratum%253Aachs%26aulast%3DGiaccone%26aufirst%3DG.%26aulast%3DZucali%26aufirst%3DP.%26atitle%3DSrc%2520as%2520a%2520potential%2520therapeutic%2520target%2520in%2520non-small-cell%2520lung%2520cancer%26jtitle%3DAnnals%2520of%2520Oncology%26date%3D2008%26volume%3D19%26spage%3D1219%26epage%3D1223%26doi%3D10.1093%2Fannonc%2Fmdn048" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Green, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fennell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whittaker, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curwen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobs, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Logie, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hargreaves, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hickinson, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilkinson, R. W.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Preclinical anticancer activity of the potent, oral Src inhibitor AZD0530</span>. <i>Mol. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">248</span>– <span class="NLM_lpage">261</span>, <span class="refDoi"> DOI: 10.1016/j.molonc.2009.01.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.1016%2Fj.molonc.2009.01.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=19393585" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjslKnsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2009&pages=248-261&author=T.+P.+Greenauthor=M.+Fennellauthor=R.+Whittakerauthor=J.+Curwenauthor=V.+Jacobsauthor=J.+Allenauthor=A.+Logieauthor=J.+Hargreavesauthor=D.+M.+Hickinsonauthor=R.+W.+Wilkinson&title=Preclinical+anticancer+activity+of+the+potent%2C+oral+Src+inhibitor+AZD0530&doi=10.1016%2Fj.molonc.2009.01.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Preclinical anticancer activity of the potent, oral Src inhibitor AZD0530</span></div><div class="casAuthors">Green, Tim P.; Fennell, Mike; Whittaker, Robin; Curwen, Jon; Jacobs, Vivien; Allen, Jack; Logie, Armelle; Hargreaves, Judith; Hickinson, D. Mark; Wilkinson, Robert W.; Elvin, Paul; Boyer, Brigitte; Carragher, Neil; Ple, Patrick A.; Bermingham, Alun; Holdgate, Geoffrey A.; Ward, Walter H. J.; Hennequin, Laurent F.; Davies, Barry R.; Costello, Gerard F.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Oncology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">248-261</span>CODEN:
                <span class="NLM_cas:coden">MOONC3</span>;
        ISSN:<span class="NLM_cas:issn">1574-7891</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">AZD0530, an orally available Src inhibitor, demonstrated potent antimigratory and anti-invasive effects in vitro, and inhibited metastasis in a murine model of bladder cancer.  Antiproliferative activity of AZD0530 in vitro varied between cell lines (IC50 0.2 -> 10 μM).  AZD0530 inhibited tumor growth in 4/10 xenograft models tested and dynamically inhibited in vivo phosphorylation of Src substrates paxillin and FAK in both growth-inhibition-resistant and -sensitive xenografts.  The activity of AZD0530 in NBT-II bladder cancer cells in vitro was consistent with inhibition of cell migration and stabilization of cell-cell adhesion.  These data suggest a dominant anti-invasive pharmacol. for AZD0530 that may limit tumor progression in a range of cancers.  AZD0530 is currently in Phase II clin. trials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAbzKlSDQyObVg90H21EOLACvtfcHk0li_-aALHrRoAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjslKnsL0%253D&md5=4c2ec94fde22bbec401009e106bb6caf</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2Fj.molonc.2009.01.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molonc.2009.01.002%26sid%3Dliteratum%253Aachs%26aulast%3DGreen%26aufirst%3DT.%2BP.%26aulast%3DFennell%26aufirst%3DM.%26aulast%3DWhittaker%26aufirst%3DR.%26aulast%3DCurwen%26aufirst%3DJ.%26aulast%3DJacobs%26aufirst%3DV.%26aulast%3DAllen%26aufirst%3DJ.%26aulast%3DLogie%26aufirst%3DA.%26aulast%3DHargreaves%26aufirst%3DJ.%26aulast%3DHickinson%26aufirst%3DD.%2BM.%26aulast%3DWilkinson%26aufirst%3DR.%2BW.%26atitle%3DPreclinical%2520anticancer%2520activity%2520of%2520the%2520potent%252C%2520oral%2520Src%2520inhibitor%2520AZD0530%26jtitle%3DMol.%2520Oncol.%26date%3D2009%26volume%3D3%26spage%3D248%26epage%3D261%26doi%3D10.1016%2Fj.molonc.2009.01.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hafner, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niepel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sorger, P. K.</span></span> <span> </span><span class="NLM_article-title">Growth rate inhibition metrics correct for confounders in measuring sensitivity to cancer drugs</span>. <i>Nat. Methods</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">521</span>, <span class="refDoi"> DOI: 10.1038/nmeth.3853</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.1038%2Fnmeth.3853" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=27135972" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BC28XntVCltr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2016&pages=521&author=M.+Hafnerauthor=M.+Niepelauthor=M.+Chungauthor=P.+K.+Sorger&title=Growth+rate+inhibition+metrics+correct+for+confounders+in+measuring+sensitivity+to+cancer+drugs&doi=10.1038%2Fnmeth.3853"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Growth rate inhibition metrics correct for confounders in measuring sensitivity to cancer drugs</span></div><div class="casAuthors">Hafner, Marc; Niepel, Mario; Chung, Mirra; Sorger, Peter K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Methods</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">521-527</span>CODEN:
                <span class="NLM_cas:coden">NMAEA3</span>;
        ISSN:<span class="NLM_cas:issn">1548-7091</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Drug sensitivity and resistance are conventionally quantified by IC50 or Emax values, but these metrics are highly sensitive to the no. of divisions taking place over the course of a response assay.  The dependency of IC50 and Emax on division rate creates artifactual correlations between genotype and drug sensitivity, while obscuring valuable biol. insights and interfering with biomarker discovery.  We derive alternative small mol. drug-response metrics that are insensitive to division no.  These are based on estn. of the magnitude of drug-induced growth rate inhibition (GR) using endpoint or time-course assays.  We show that GR50 and GRmax are superior to conventional metrics for assessing the effects of small mol. drugs in dividing cells.  Moreover, adopting GR metrics requires only modest changes in exptl. protocols.  We expect GR metrics to improve the study of cell signaling and growth using small mols. and biologics and to facilitate the discovery of drug-response biomarkers and the identification of drugs effective against specific patient-derived tumor cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5Byu3P4LAobVg90H21EOLACvtfcHk0li_-aALHrRoAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XntVCltr0%253D&md5=7383d0009d14697d6fb3e9a2de0f6da0</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1038%2Fnmeth.3853&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnmeth.3853%26sid%3Dliteratum%253Aachs%26aulast%3DHafner%26aufirst%3DM.%26aulast%3DNiepel%26aufirst%3DM.%26aulast%3DChung%26aufirst%3DM.%26aulast%3DSorger%26aufirst%3DP.%2BK.%26atitle%3DGrowth%2520rate%2520inhibition%2520metrics%2520correct%2520for%2520confounders%2520in%2520measuring%2520sensitivity%2520to%2520cancer%2520drugs%26jtitle%3DNat.%2520Methods%26date%3D2016%26volume%3D13%26spage%3D521%26doi%3D10.1038%2Fnmeth.3853" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalyankrishna, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wislez, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thilaganathan, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saigal, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wistuba, I. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurie, J. M.</span></span> <span> </span><span class="NLM_article-title">SRC-family kinases are activated in non-small cell lung cancer and promote the survival of epidermal growth factor receptor-dependent cell lines</span>. <i>Am. J. Pathol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>170</i></span>,  <span class="NLM_fpage">366</span>– <span class="NLM_lpage">376</span>, <span class="refDoi"> DOI: 10.2353/ajpath.2007.060706</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.2353%2Fajpath.2007.060706" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=17200208" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A280%3ADC%252BD2s%252FgtlOnsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=170&publication_year=2007&pages=366-376&author=J.+Zhangauthor=S.+Kalyankrishnaauthor=M.+Wislezauthor=N.+Thilaganathanauthor=B.+Saigalauthor=W.+Weiauthor=L.+Maauthor=I.+I.+Wistubaauthor=F.+M.+Johnsonauthor=J.+M.+Kurie&title=SRC-family+kinases+are+activated+in+non-small+cell+lung+cancer+and+promote+the+survival+of+epidermal+growth+factor+receptor-dependent+cell+lines&doi=10.2353%2Fajpath.2007.060706"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">SRC-family kinases are activated in non-small cell lung cancer and promote the survival of epidermal growth factor receptor-dependent cell lines</span></div><div class="casAuthors">Zhang Jie; Kalyankrishna Shailaja; Wislez Marie; Thilaganathan Nishan; Saigal Babita; Wei Wei; Ma Long; Wistuba Ignacio I; Johnson Faye M; Kurie Jonathan M</div><div class="citationInfo"><span class="NLM_cas:title">The American journal of pathology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">170</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">366-76</span>
        ISSN:<span class="NLM_cas:issn">0002-9440</span>.
    </div><div class="casAbstract">The role of Src-family kinases (SFKs) in non-small cell lung cancer (NSCLC) has not been fully defined.  Here we addressed this question by examining SFK phosphorylation in NSCLC biopsy samples and using genetic and pharmacological approaches to inhibit SFK expression and activity in cultured NSCLC cells.  Immunohistochemical analysis of NSCLC biopsy samples using a Tyr416 phosphorylation-specific, pan-SFK antibody revealed staining in 123 (33%) of 370 tumors.  Because c-Src is known to be both an upstream activator and downstream mediator of epidermal growth factor receptor (EGFR), we next investigated SFK phosphorylation in a panel of NSCLC cell lines, including ones that depend on EGFR for survival.  The EGFR-dependent NSCLC cell lines HCC827 and H3255 had increased phosphorylation of SFKs, and treatment of these cells with an SFK inhibitor (PP1 or SKI-606) induced apoptosis.  PP1 decreased phosphorylation of EGFR, ErbB2, and ErbB3 and strikingly enhanced apoptosis by gefitinib, an EGFR inhibitor.  HCC827 cells transfected with c-Src short hairpin RNA exhibited diminished phosphorylation of EGFR and ErbB2 and decreased sensitivity to apoptosis by PP1 or gefitinib.  We conclude that SFKs are activated in NSCLC biopsy samples, promote the survival of EGFR-dependent NSCLC cells, and should be investigated as therapeutic targets in NSCLC patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQZ2f3qQMjhE0kUaC_kzhvyfW6udTcc2eZ8CRLkiYouqrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2s%252FgtlOnsA%253D%253D&md5=391c770e7368377b4b9f5576a206d8d0</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.2353%2Fajpath.2007.060706&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2353%252Fajpath.2007.060706%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DKalyankrishna%26aufirst%3DS.%26aulast%3DWislez%26aufirst%3DM.%26aulast%3DThilaganathan%26aufirst%3DN.%26aulast%3DSaigal%26aufirst%3DB.%26aulast%3DWei%26aufirst%3DW.%26aulast%3DMa%26aufirst%3DL.%26aulast%3DWistuba%26aufirst%3DI.%2BI.%26aulast%3DJohnson%26aufirst%3DF.%2BM.%26aulast%3DKurie%26aufirst%3DJ.%2BM.%26atitle%3DSRC-family%2520kinases%2520are%2520activated%2520in%2520non-small%2520cell%2520lung%2520cancer%2520and%2520promote%2520the%2520survival%2520of%2520epidermal%2520growth%2520factor%2520receptor-dependent%2520cell%2520lines%26jtitle%3DAm.%2520J.%2520Pathol.%26date%3D2007%26volume%3D170%26spage%3D366%26epage%3D376%26doi%3D10.2353%2Fajpath.2007.060706" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ficarro, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Browne, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Card, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alexander, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNally, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhe-Paganon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seo, H.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamberto, I.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Leveraging gas-phase fragmentation pathways for improved identification and selective detection of targets modified by covalent probes</span>. <i>Anal. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>88</i></span>,  <span class="NLM_fpage">12248</span>– <span class="NLM_lpage">12254</span>, <span class="refDoi"> DOI: 10.1021/acs.analchem.6b03394</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.analchem.6b03394" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFGisrnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=88&publication_year=2016&pages=12248-12254&author=S.+B.+Ficarroauthor=C.+M.+Browneauthor=J.+D.+Cardauthor=W.+M.+Alexanderauthor=T.+Zhangauthor=E.+Parkauthor=R.+McNallyauthor=S.+Dhe-Paganonauthor=H.-S.+Seoauthor=I.+Lamberto&title=Leveraging+gas-phase+fragmentation+pathways+for+improved+identification+and+selective+detection+of+targets+modified+by+covalent+probes&doi=10.1021%2Facs.analchem.6b03394"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Leveraging Gas-Phase Fragmentation Pathways for Improved Identification and Selective Detection of Targets Modified by Covalent Probes</span></div><div class="casAuthors">Ficarro, Scott B.; Browne, Christopher M.; Card, Joseph D.; Alexander, William M.; Zhang, Tinghu; Park, Eunyoung; McNally, Randall; Dhe-Paganon, Sirano; Seo, Hyuk-Soo; Lamberto, Ilaria; Eck, Michael J.; Buhrlage, Sara J.; Gray, Nathanael S.; Marto, Jarrod A.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Chemistry (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">88</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">12248-12254</span>CODEN:
                <span class="NLM_cas:coden">ANCHAM</span>;
        ISSN:<span class="NLM_cas:issn">0003-2700</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The recent approval of covalent inhibitors for multiple clin. indications has reignited enthusiasm for this class of drugs.  As interest in covalent drugs has increased, so too has the need for anal. platforms that can leverage their mechanism-of-action to characterize modified protein targets.  Here the authors describe novel gas phase dissocn. pathways which yield predictable fragment ions during MS/MS of inhibitor-modified peptides.  The authors find that these dissocn. pathways are common to numerous cysteine-directed probes as well as the covalent drugs, Ibrutinib and Neratinib.  The authors leverage the predictable nature of these fragment ions to improve the confidence of peptide sequence assignment in proteomic analyses, and explore their potential use in selective mass spectrometry-based assays.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrNu24Jo7-IWLVg90H21EOLACvtfcHk0lgZG3HPErGbog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFGisrnO&md5=7932392d2d2ac88def4cacecbda41a30</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Facs.analchem.6b03394&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.analchem.6b03394%26sid%3Dliteratum%253Aachs%26aulast%3DFicarro%26aufirst%3DS.%2BB.%26aulast%3DBrowne%26aufirst%3DC.%2BM.%26aulast%3DCard%26aufirst%3DJ.%2BD.%26aulast%3DAlexander%26aufirst%3DW.%2BM.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DPark%26aufirst%3DE.%26aulast%3DMcNally%26aufirst%3DR.%26aulast%3DDhe-Paganon%26aufirst%3DS.%26aulast%3DSeo%26aufirst%3DH.-S.%26aulast%3DLamberto%26aufirst%3DI.%26atitle%3DLeveraging%2520gas-phase%2520fragmentation%2520pathways%2520for%2520improved%2520identification%2520and%2520selective%2520detection%2520of%2520targets%2520modified%2520by%2520covalent%2520probes%26jtitle%3DAnal.%2520Chem.%26date%3D2016%26volume%3D88%26spage%3D12248%26epage%3D12254%26doi%3D10.1021%2Facs.analchem.6b03394" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Papaetis, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Syrigos, K. N.</span></span> <span> </span><span class="NLM_article-title">Sunitinib</span>. <i>BioDrugs</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">377</span>– <span class="NLM_lpage">389</span>, <span class="refDoi"> DOI: 10.2165/11318860-000000000-00000</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.2165%2F11318860-000000000-00000" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=19894779" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtF2qs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2009&pages=377-389&author=G.+S.+Papaetisauthor=K.+N.+Syrigos&title=Sunitinib&doi=10.2165%2F11318860-000000000-00000"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Sunitinib: a multitargeted receptor tyrosine kinase inhibitor in the era of molecular cancer therapies</span></div><div class="casAuthors">Papaetis, Georgios S.; Syrigos, Kostas N.</div><div class="citationInfo"><span class="NLM_cas:title">BioDrugs</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">377-389</span>CODEN:
                <span class="NLM_cas:coden">BIDRF4</span>;
        ISSN:<span class="NLM_cas:issn">1173-8804</span>.
    
            (<span class="NLM_cas:orgname">Adis Data Information BV</span>)
        </div><div class="casAbstract">A review.  Sunitinib is an oral oxindole multitargeted kinase inhibitor that inhibits certain receptor tyrosine kinases (RTKs).  These include vascular endothelial growth factor receptors (VEGFR type 1 and 2), platelet-derived growth factor receptors (PDGFR-α and PDGFR-β), stem cell factor receptor (KIT), FMS-like tyrosine kinase-3 (FLT3), glial cell-line derived neurotrophic factor receptor (RET) and the receptor of macrophage-colony stimulating factor (CSF1R).  Examn. of the antitumor effect of sunitinib in a variety of cell lines in vitro suggested an antiproliferative activity that is dependent on the presence of constitutively active RTK targets.  The use of sunitinib as first-line therapy in advanced renal cell carcinoma (RCC) has improved the overall survival compared with that obsd. after cytokine therapy, while its administration in patients with gastrointestinal stromal tumors (GISTs) after progression or intolerance to imatinib achieved an objective response of 7%.  Sunitinib is currently approved for the treatment of GISTs in this setting, and as first-line therapy for the treatment of advanced RCC.  The relatively long half-life of sunitinib and its major metabolite allow for a once-daily dosing schedule.  An interesting antitumor activity of sunitinib was reported in phase II studies of patients with a variety of malignancies, such as hepatocellular cancer, pancreatic neuroendocrine tumors, and non-small cell lung cancer; results of phase III studies are urgently anticipated.  Fatigue is one of the most common adverse effects of sunitinib, as 50-70% of patients with advanced RCC and GIST complained of this adverse effect.  Other adverse effects are diarrhea, anorexia, nausea and vomiting, oral changes and bleeding events.  Most toxicities are reversible and should not result in discontinuation of sunitinib.  If necessary, dose adjustments or interruptions should be made.  Hypothyroidism has been described in the first 2 wk of sunitinib therapy and its incidence increases progressively with the duration of therapy.  Sunitinib may exert its hypertensive activity through a direct effect on the vasculature, while its most important cardiac adverse effect is left ventricular dysfunction.  A variety of skin adverse effects have been described with the use of sunitinib such as hand-foot syndrome, yellow discoloration of the skin, dry skin, subungual splinter hemorrhages, acral erythema, and generalized skin rashes.  Administration of sunitinib in the adjuvant and neoadjuvant setting of patients with RCC and of its combination with chemotherapy and other targeted therapies are currently under intense investigation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYS25sw5p707Vg90H21EOLACvtfcHk0lgZG3HPErGbog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtF2qs7g%253D&md5=dc5d34a3f2f6fc7cc2a013517c3195ab</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.2165%2F11318860-000000000-00000&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2165%252F11318860-000000000-00000%26sid%3Dliteratum%253Aachs%26aulast%3DPapaetis%26aufirst%3DG.%2BS.%26aulast%3DSyrigos%26aufirst%3DK.%2BN.%26atitle%3DSunitinib%26jtitle%3DBioDrugs%26date%3D2009%26volume%3D23%26spage%3D377%26epage%3D389%26doi%3D10.2165%2F11318860-000000000-00000" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wilhelm, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lynch, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lowinger, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dumas, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simantov, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelley, S.</span></span> <span> </span><span class="NLM_article-title">Discovery and development of sorafenib: a multikinase inhibitor for treating cancer</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">835</span>, <span class="refDoi"> DOI: 10.1038/nrd2130</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.1038%2Fnrd2130" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=17016424" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtVajsbbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2006&pages=835&author=S.+Wilhelmauthor=C.+Carterauthor=M.+Lynchauthor=T.+Lowingerauthor=J.+Dumasauthor=R.+A.+Smithauthor=B.+Schwartzauthor=R.+Simantovauthor=S.+Kelley&title=Discovery+and+development+of+sorafenib%3A+a+multikinase+inhibitor+for+treating+cancer&doi=10.1038%2Fnrd2130"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and development of sorafenib: a multikinase inhibitor for treating cancer</span></div><div class="casAuthors">Wilhelm, Scott; Carter, Christopher; Lynch, Mark; Lowinger, Timothy; Dumas, Jacques; Smith, Roger A.; Schwartz, Brian; Simantov, Ronit; Kelley, Susan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">835-844</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Since the mol. revolution of the 1980s, knowledge of the etiol. of cancer has increased considerably, which has led to the discovery and development of targeted therapies tailored to inhibit cancer-specific pathways.  The introduction and refinement of rapid, high-throughput screening technologies over the past decade has greatly facilitated this targeted discovery and development process.  Here, the authors describe the discovery and continuing development of sorafenib (previously known as BAY 43-9006), the first oral multikinase inhibitor that targets Raf and affects tumor signaling and the tumor vasculature.  The discovery cycle of sorafenib (Nexavar; Bayer Pharmaceuticals) - from initial screening for a lead compd. to FDA approval for the treatment of advanced renal cell carcinoma in Dec. 2005 - was completed in just 11 years, with approval being received ∼5 years after the initiation of the first Phase I trial.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6WjjfgNSCtbVg90H21EOLACvtfcHk0ligipzYi8Qxbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtVajsbbM&md5=490d4995170f3a7270ae9bbdd24b12e5</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1038%2Fnrd2130&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2130%26sid%3Dliteratum%253Aachs%26aulast%3DWilhelm%26aufirst%3DS.%26aulast%3DCarter%26aufirst%3DC.%26aulast%3DLynch%26aufirst%3DM.%26aulast%3DLowinger%26aufirst%3DT.%26aulast%3DDumas%26aufirst%3DJ.%26aulast%3DSmith%26aufirst%3DR.%2BA.%26aulast%3DSchwartz%26aufirst%3DB.%26aulast%3DSimantov%26aufirst%3DR.%26aulast%3DKelley%26aufirst%3DS.%26atitle%3DDiscovery%2520and%2520development%2520of%2520sorafenib%253A%2520a%2520multikinase%2520inhibitor%2520for%2520treating%2520cancer%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2006%26volume%3D5%26spage%3D835%26doi%3D10.1038%2Fnrd2130" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J. Y.</span></span> <span> </span><span class="NLM_article-title">Exploiting the promiscuity of imatinib</span>. <i>J. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">30</span>, <span class="refDoi"> DOI: 10.1186/jbiol134</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.1186%2Fjbiol134" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=19435483" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A280%3ADC%252BD1Mzos1eltA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2009&pages=30&author=S.+J.+Leeauthor=J.+Y.+Wang&title=Exploiting+the+promiscuity+of+imatinib&doi=10.1186%2Fjbiol134"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Exploiting the promiscuity of imatinib</span></div><div class="casAuthors">Lee Shun J; Wang Jean Y J</div><div class="citationInfo"><span class="NLM_cas:title">Journal of biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">30</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The protein kinase inhibitor imatinib, also known as Gleevec, has been a notable success in treating chronic myelogenous leukemia.  A recent paper in BMC Structural Biology reports a 1.75 A crystal structure of imatinib bound to the oxidoreductase NQO2 and reveals insights into the binding specificity and the off-target effects of the inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQvIsbZZF5A8n6CEoQgWY10fW6udTcc2eZuGcx7m4R87rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1Mzos1eltA%253D%253D&md5=13e34c684303d3976d50727e3f70c5d2</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1186%2Fjbiol134&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fjbiol134%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DS.%2BJ.%26aulast%3DWang%26aufirst%3DJ.%2BY.%26atitle%3DExploiting%2520the%2520promiscuity%2520of%2520imatinib%26jtitle%3DJ.%2520Biol.%26date%3D2009%26volume%3D8%26spage%3D30%26doi%3D10.1186%2Fjbiol134" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hafner, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Subramanian, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sorger, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">A multi-targeted probe-based strategy to identify signaling vulnerabilities in cancers</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>294</i></span>,  <span class="NLM_fpage">8664</span>, <span class="refDoi"> DOI: 10.1074/jbc.RA118.006805</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.1074%2Fjbc.RA118.006805" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=30858179" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtFOnurnJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=294&publication_year=2019&pages=8664&author=S.+Raoauthor=G.+Duauthor=M.+Hafnerauthor=K.+Subramanianauthor=P.+K.+Sorgerauthor=N.+S.+Gray&title=A+multi-targeted+probe-based+strategy+to+identify+signaling+vulnerabilities+in+cancers&doi=10.1074%2Fjbc.RA118.006805"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">A multitargeted probe-based strategy to identify signaling vulnerabilities in cancers</span></div><div class="casAuthors">Rao, Suman; Du, Guangyan; Hafner, Marc; Subramanian, Kartik; Sorger, Peter K.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">294</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">8664-8673</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Most cancer cells are dependent on a network of deregulated signaling pathways for survival and are insensitive, or rapidly evolve resistance, to selective inhibitors aimed at a single target.  For these reasons, drugs that target more than one protein (polypharmacol.) can be clin. advantageous.  The discovery of useful polypharmacol. remains serendipitous and is challenging to characterize and validate.  In this study, we developed a non-genetic strategy for the identification of pathways that drive cancer cell proliferation and represent exploitable signaling vulnerabilities.  Our approach is based on using a multitargeted kinase inhibitor, SM1-71, as a tool compd. to identify combinations of targets whose simultaneous inhibition elicits a potent cytotoxic effect.  As a proof of concept, we applied this approach to a KRAS-dependent non-small cell lung cancer (NSCLC) cell line, H23-KRASG12C.  Using a combination of phenotypic screens, signaling analyses, and kinase inhibitors, we found that dual inhibition of MEK1/2 and insulin-like growth factor 1 receptor (IGF1R)/insulin receptor (INSR) is crit. for blocking proliferation in cells.  Our work supports the value of multitargeted tool compds. with well-validated polypharmacol. and target space as tools to discover kinase dependences in cancer.  We propose that the strategy described here is complementary to existing genetics-based approaches, generalizable to other systems, and enabling for future mechanistic and translational studies of polypharmacol. in the context of signaling vulnerabilities in cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAvW-66Pk4brVg90H21EOLACvtfcHk0ligipzYi8Qxbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtFOnurnJ&md5=694fd4cb9adb5fb68d2600a5c608b27a</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1074%2Fjbc.RA118.006805&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.RA118.006805%26sid%3Dliteratum%253Aachs%26aulast%3DRao%26aufirst%3DS.%26aulast%3DDu%26aufirst%3DG.%26aulast%3DHafner%26aufirst%3DM.%26aulast%3DSubramanian%26aufirst%3DK.%26aulast%3DSorger%26aufirst%3DP.%2BK.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DA%2520multi-targeted%2520probe-based%2520strategy%2520to%2520identify%2520signaling%2520vulnerabilities%2520in%2520cancers%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2019%26volume%3D294%26spage%3D8664%26doi%3D10.1074%2Fjbc.RA118.006805" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 7 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Ryan A. Brawn, Andrew Cook, Kiyoyuki Omoto, Jiyuan Ke, Craig Karr, Federico Colombo, Milena Virrankoski, Sudeep Prajapati, Dominic Reynolds, David M. Bolduc, Tuong-Vi Nguyen, Patricia Gee, Deanna Borrelli, Benjamin Caleb, Shihua Yao, Sean Irwin, Nicholas A. Larsen, Anand Selvaraj, Xuesong Zhao, <span class="NLM_string-name hlFld-ContribAuthor">Stephanos Ioannidis</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of Aminopyrazole Derivatives as Potent Inhibitors of Wild-Type and Gatekeeper Mutant FGFR2 and 3. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2021,</strong> <em>12 </em>
                                    (1)
                                     , 93-98. <a href="https://doi.org/10.1021/acsmedchemlett.0c00517" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.0c00517</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.0c00517&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.0c00517%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DDiscovery%252Bof%252BAminopyrazole%252BDerivatives%252Bas%252BPotent%252BInhibitors%252Bof%252BWild-Type%252Band%252BGatekeeper%252BMutant%252BFGFR2%252Band%252B3%26aulast%3DBrawn%26aufirst%3DRyan%2BA.%26date%3D2021%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D25092020%26date%3D25112020%26date%3D02122020%26volume%3D12%26issue%3D1%26spage%3D93%26epage%3D98" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Christopher M.  Dustin</span>, <span class="hlFld-ContribAuthor ">Aida  Habibovic</span>, <span class="hlFld-ContribAuthor ">Milena  Hristova</span>, <span class="hlFld-ContribAuthor ">Caspar  Schiffers</span>, <span class="hlFld-ContribAuthor ">Carolyn R.  Morris</span>, <span class="hlFld-ContribAuthor ">Miao-Chong Joy  Lin</span>, <span class="hlFld-ContribAuthor ">Robert A.  Bauer</span>, <span class="hlFld-ContribAuthor ">David E.  Heppner</span>, <span class="hlFld-ContribAuthor ">Nirav  Daphtary</span>, <span class="hlFld-ContribAuthor ">Minara  Aliyeva</span>, <span class="hlFld-ContribAuthor ">Albert  van der Vliet</span>. </span><span class="cited-content_cbyCitation_article-title">Oxidation–Dependent Activation of Src Kinase Mediates Epithelial IL-33 Production and Signaling during Acute Airway Allergen Challenge. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Immunology</span><span> <strong>2021,</strong> <em>206 </em>
                                    (12)
                                     , 2989-2999. <a href="https://doi.org/10.4049/jimmunol.2000995" title="DOI URL">https://doi.org/10.4049/jimmunol.2000995</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4049/jimmunol.2000995&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4049%2Fjimmunol.2000995%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Immunology%26atitle%3DOxidation%2525E2%252580%252593Dependent%252BActivation%252Bof%252BSrc%252BKinase%252BMediates%252BEpithelial%252BIL-33%252BProduction%252Band%252BSignaling%252Bduring%252BAcute%252BAirway%252BAllergen%252BChallenge%26aulast%3DDustin%26aufirst%3DChristopher%2BM.%26date%3D2021%26date%3D2021%26volume%3D206%26issue%3D12%26spage%3D2989%26epage%3D2999" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Miljan  Kuljanin</span>, <span class="hlFld-ContribAuthor ">Dylan C.  Mitchell</span>, <span class="hlFld-ContribAuthor ">Devin K.  Schweppe</span>, <span class="hlFld-ContribAuthor ">Ajami S.  Gikandi</span>, <span class="hlFld-ContribAuthor ">David P.  Nusinow</span>, <span class="hlFld-ContribAuthor ">Nathan J.  Bulloch</span>, <span class="hlFld-ContribAuthor ">Ekaterina V.  Vinogradova</span>, <span class="hlFld-ContribAuthor ">David L.  Wilson</span>, <span class="hlFld-ContribAuthor ">Eric T.  Kool</span>, <span class="hlFld-ContribAuthor ">Joseph D.  Mancias</span>, <span class="hlFld-ContribAuthor ">Benjamin F.  Cravatt</span>, <span class="hlFld-ContribAuthor ">Steven P.  Gygi</span>. </span><span class="cited-content_cbyCitation_article-title">Reimagining high-throughput profiling of reactive cysteines for cell-based screening of large electrophile libraries. </span><span class="cited-content_cbyCitation_journal-name">Nature Biotechnology</span><span> <strong>2021,</strong> <em>39 </em>
                                    (5)
                                     , 630-641. <a href="https://doi.org/10.1038/s41587-020-00778-3" title="DOI URL">https://doi.org/10.1038/s41587-020-00778-3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41587-020-00778-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41587-020-00778-3%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Biotechnology%26atitle%3DReimagining%252Bhigh-throughput%252Bprofiling%252Bof%252Breactive%252Bcysteines%252Bfor%252Bcell-based%252Bscreening%252Bof%252Blarge%252Belectrophile%252Blibraries%26aulast%3DKuljanin%26aufirst%3DMiljan%26date%3D2021%26date%3D2021%26volume%3D39%26issue%3D5%26spage%3D630%26epage%3D641" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">David E.  Heppner</span>. </span><span class="cited-content_cbyCitation_article-title">Structural insights into redox-active cysteine residues of the Src family kinases. </span><span class="cited-content_cbyCitation_journal-name">Redox Biology</span><span> <strong>2021,</strong> <em>41 </em>, 101934. <a href="https://doi.org/10.1016/j.redox.2021.101934" title="DOI URL">https://doi.org/10.1016/j.redox.2021.101934</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.redox.2021.101934&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.redox.2021.101934%26sid%3Dliteratum%253Aachs%26jtitle%3DRedox%2520Biology%26atitle%3DStructural%252Binsights%252Binto%252Bredox-active%252Bcysteine%252Bresidues%252Bof%252Bthe%252BSrc%252Bfamily%252Bkinases%26aulast%3DHeppner%26aufirst%3DDavid%2BE.%26date%3D2021%26volume%3D41%26spage%3D101934" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yi  Zhang</span>, <span class="hlFld-ContribAuthor ">Ting-jian  Zhang</span>, <span class="hlFld-ContribAuthor ">Shun  Tu</span>, <span class="hlFld-ContribAuthor ">Zhen-hao  Zhang</span>, <span class="hlFld-ContribAuthor ">Fan-hao  Meng</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of Novel Src Inhibitors: Pharmacophore-Based Virtual Screening, Molecular Docking and Molecular Dynamics Simulations. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2020,</strong> <em>25 </em>
                                    (18)
                                     , 4094. <a href="https://doi.org/10.3390/molecules25184094" title="DOI URL">https://doi.org/10.3390/molecules25184094</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules25184094&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules25184094%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DIdentification%252Bof%252BNovel%252BSrc%252BInhibitors%25253A%252BPharmacophore-Based%252BVirtual%252BScreening%25252C%252BMolecular%252BDocking%252Band%252BMolecular%252BDynamics%252BSimulations%26aulast%3DZhang%26aufirst%3DYi%26date%3D2020%26date%3D2020%26volume%3D25%26issue%3D18%26spage%3D4094" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ranjeet P.  Dash</span>, <span class="hlFld-ContribAuthor ">Jennifer A.  Thomas</span>, <span class="hlFld-ContribAuthor ">Clint  Rosenfeld</span>, <span class="hlFld-ContribAuthor ">Nuggehally R.  Srinivas</span>. </span><span class="cited-content_cbyCitation_article-title">Protein Binding and Stability of Drug Candidates: The Achilles’ Heel in In Vitro Potency Assays. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Drug Metabolism and Pharmacokinetics</span><span> <strong>2020,</strong> <em>45 </em>
                                    (4)
                                     , 427-432. <a href="https://doi.org/10.1007/s13318-020-00619-3" title="DOI URL">https://doi.org/10.1007/s13318-020-00619-3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s13318-020-00619-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs13318-020-00619-3%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Drug%2520Metabolism%2520and%2520Pharmacokinetics%26atitle%3DProtein%252BBinding%252Band%252BStability%252Bof%252BDrug%252BCandidates%25253A%252BThe%252BAchilles%2525E2%252580%252599%252BHeel%252Bin%252BIn%252BVitro%252BPotency%252BAssays%26aulast%3DDash%26aufirst%3DRanjeet%2BP.%26date%3D2020%26date%3D2020%26volume%3D45%26issue%3D4%26spage%3D427%26epage%3D432" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Deepak  Gurbani</span>, <span class="hlFld-ContribAuthor ">Guangyan  Du</span>, <span class="hlFld-ContribAuthor ">Nathaniel J.  Henning</span>, <span class="hlFld-ContribAuthor ">Suman  Rao</span>, <span class="hlFld-ContribAuthor ">Asim K.  Bera</span>, <span class="hlFld-ContribAuthor ">Tinghu  Zhang</span>, <span class="hlFld-ContribAuthor ">Nathanael S.  Gray</span>, <span class="hlFld-ContribAuthor ">Kenneth D.  Westover</span>. </span><span class="cited-content_cbyCitation_article-title">Structure and Characterization of a Covalent Inhibitor of Src Kinase. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Molecular Biosciences</span><span> <strong>2020,</strong> <em>7 </em><a href="https://doi.org/10.3389/fmolb.2020.00081" title="DOI URL">https://doi.org/10.3389/fmolb.2020.00081</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fmolb.2020.00081&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffmolb.2020.00081%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Molecular%2520Biosciences%26atitle%3DStructure%252Band%252BCharacterization%252Bof%252Ba%252BCovalent%252BInhibitor%252Bof%252BSrc%252BKinase%26aulast%3DGurbani%26aufirst%3DDeepak%26date%3D2020%26date%3D2020%26volume%3D7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01502/20200221/images/medium/jm9b01502_0017.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01502/20200221/images/large/jm9b01502_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01502&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01502/20200221/images/medium/jm9b01502_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01502/20200221/images/large/jm9b01502_0006.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Syntheses of Compounds <b>5a</b>–<b>e</b>, <b>10a</b>–<b>c</b>, and <b>15a</b>–<b>i</b> from Commercially Available 5-Substituted-2,4-Dichloropyrimidine<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01502/20200221/images/large/jm9b01502_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01502&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) benzene-1,2-diamine, DIEA, <i>n</i>-BuOH, reflux; (b) 4-(4-methylpiperazin-1-yl)aniline, TFA, <i>s</i>-BuOH, reflux; (c) aryl boronic acid or ester, Pd(dppf)Cl<sub>2</sub>·DCM, Et<sub>3</sub>N, dioxane/H<sub>2</sub>O; (d) acryloyl chloride, NaHCO<sub>3</sub> aq., THF, ice bath; (e) different benzyl bromides, K<sub>2</sub>CO<sub>3</sub>, DMF, 50 °C; (f) different anilines, Et<sub>3</sub>N, DMF, ice bath; (g) benzene-1,2-diamine, DIEA, <i>n</i>-BuOH, rt.</p></p></figure><figure data-id="sch2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01502/20200221/images/medium/jm9b01502_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01502/20200221/images/large/jm9b01502_0007.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compounds <b>17a</b>–<b>f</b> from <b>13a</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01502/20200221/images/large/jm9b01502_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01502&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) acryloyl chloride, NaHCO<sub>3</sub> aq., THF, ice bath; (b) different anilines, TFA, <i>sec</i>-BuOH, reflux.</p></p></figure><figure data-id="sch3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01502/20200221/images/medium/jm9b01502_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01502/20200221/images/large/jm9b01502_0008.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Compounds <b>20a</b>–<b>f</b> from <b>12a</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01502/20200221/images/large/jm9b01502_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01502&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) different benzene diamines, DIEA, <i>n</i>-BuOH, rt; (b) 4-(4-methylpiperazin-1-yl)aniline, TFA, <i>s</i>-BuOH, reflux; (c) different electrophiles, NaHCO<sub>3</sub> aq., THF, ice bath or Et<sub>3</sub>N, DCM, ice bath.</p></p></figure><figure data-id="fig1" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01502/20200221/images/medium/jm9b01502_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01502/20200221/images/large/jm9b01502_0001.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. (a) Superposition of the binding mode of <b>SM1-71</b> in SRC with FGFR1 and TAK1 kinase domains. Red dashed lines indicate hinge hydrogen bonds. The spheres show the size of the gatekeeper residues: red for SRC, gold for FGFR1, and cyan for TAK1. The red, gold, and cyan ribbons represent SRC, FGFR1, and TAK1 kinases, respectively. DFG-1 Cys of TAK1 is shown in sphere representation with a carbon backbone in cyan. (b) Crystal structure of <b>SM1-71</b> covalently bound to SRC kinase (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ATE">6ATE</a>). <b>SM1-71</b> covalently binds to P-loop Cys277. The structure alignment was done using Schrodinger Suite protein alignment tool with default settings, and the figures were generated with pymol.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01502/20200221/images/large/jm9b01502_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01502&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01502/20200221/images/medium/jm9b01502_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01502/20200221/images/large/jm9b01502_0002.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Src is exclusively labeled by <b>15a</b> at cysteine 277. (A) Mass spectra (left) and zero-charge mass spectra (right) of SRC incubated (top) with DMSO or (bottom) with a 10-fold molar excess of <b>15a</b> for 2 h at 37 °C. Unlabeled protein peaks are indicated with purple glyphs, and <b>15a</b> modified protein peaks are highlighted with blue glyphs. (B) MS/MS spectrum of the SRC tryptic peptide <sup>273</sup>LGQGC*FGEVWMGTWNGTTR<sub>287</sub> modified with <b>15a</b> at Cys 277 (indicated with “*” in the sequence). Ions of type y are indicated with red glyphs. Inhibitor modified ions are labeled with green glyphs.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01502/20200221/images/large/jm9b01502_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01502&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01502/20200221/images/medium/jm9b01502_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01502/20200221/images/large/jm9b01502_0003.jpeg" id="rightTab-gr14" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. KinomeScan profiling for <b>SM1-71</b> and <b>15a</b>. (KinomeScan profiling for compound <b>SM1-71</b> was previously published.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> Reprinted with permission from ref <a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a>. Copyright 2017 Elsevier.) Compounds <b>SM1-71</b> and <b>15a</b> were screened at 1 μM against a panel of 468 human kinases. The results for the binding interactions are reported as “% Ctrl”, where larger circles indicate stronger hits. The top hits of <b>15a</b> are SRC, YES1, ABL1, HCK, LCK, LYN, BTK, BLK, and GAK, shown as blue circles in the kinome tree. (See the biochemical IC<sub>50</sub>’s in <a href="/doi/suppl/10.1021/acs.jmedchem.9b01502/suppl_file/jm9b01502_si_001.pdf" class="ext-link">Table S3</a>.) The selectivity score was defined as the ratio of the number of kinases inhibited to a specified percentage versus the total number of kinases. For this experiment, specified percent inhibition was set at 35%, resulting in S(35) values of 0.43 for <b>SM1-71</b> and 0.07 for compound <b>15a</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01502/20200221/images/large/jm9b01502_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01502&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01502/20200221/images/medium/jm9b01502_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01502/20200221/images/large/jm9b01502_0004.jpeg" id="rightTab-gr15" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Evaluating biological potency of SRC inhibitors in lung and breast cancer cell lines. (A) Comparing growth inhibitory potency of <b>15a</b> in H1975 (nonsmall cell lung cancer, NSCLC), HCC827 (NSCLC), and MDA-MB-231 (triple negative breast cancer, TNBC) cell lines. (B) Comparing growth inhibitory potency of <b>15a</b>, <b>20f</b>, and <b>AZD0530</b> across cell lines. Growth inhibition was carried out by treating cells with different concentrations of <b>15a</b>, <b>20f</b>, or <b>AZD0530</b> for 72 h, followed by analysis using the CellTiter-Glo reagent. Dose–response curves and the corresponding GR values were calculated using the online tool GR calculator (<a href="http://www.grcalculator.org/grcalculator/" class="extLink">http://www.grcalculator.org/grcalculator/</a>). Both the curves and the GR<sub>50</sub> values are representative of two independent experiments, each carried out in technical triplicate.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01502/20200221/images/large/jm9b01502_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01502&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01502/20200221/images/medium/jm9b01502_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01502/20200221/images/large/jm9b01502_0005.jpeg" id="rightTab-gr16" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Target engagement and inhibition of SRC <i>in vitro</i> and <i>in vivo</i>. (A) Target engagement in H1975 and HCC827 cells treated with 1 μM of <b>15a</b> or <b>20f</b> for 2 h followed by overnight incubation with 1 μM TL13-68 (biotin compound), pulldown by streptavidin beads, and analysis using Western blotting. (B) Signaling inhibition in H1975 and HCC827 cells treated with 1 μM of <b>15a</b> or <b>20f</b> for 2 h followed by analysis using Western blotting. (C) Cellular target engagement and (D) signaling inhibition in H1975 cells treated with 1 μM of <b>15a</b>, <b>20f</b>, <b>AZD0530</b>, or DMSO for 2 h, followed by drug removal and replacement with drug-free medium. Cells were collected and lysed 0, 2, and 4 h postdrug washout and analyzed using Western blotting. For evaluating SRC binding (C), lysates were incubated with 1 μM of TL13-68 (biotin compound), followed by pulldown by streptavidin beads and visualization using Western blotting. Blots shown in (A–D) are representative of one of two biological replicates carried out. (Both blots for each experiment are shown in <a href="/doi/suppl/10.1021/acs.jmedchem.9b01502/suppl_file/jm9b01502_si_001.pdf" class="ext-link">Figure S1</a>.) (E, F) Pharmacodynamic analysis of (E) inhibition of p-SRC<sup>Y416</sup> signaling and (F) inhibition of SRC binding in spleen samples. Briefly, C57B6 mice (<i>n</i> = 2/condition) were treated with 5 mg/kg of <b>15a</b> or vehicle three times and sacrificed 2 or 4 h following the last dose. Spleen samples collected were ground up, lysed, and analyzed using Western blotting.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01502/20200221/images/large/jm9b01502_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01502&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i72">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_47858" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_47858" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 51 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, F. M.</span></span> <span> </span><span class="NLM_article-title">Regulation of SRC family kinases in human cancers</span>. <i>J. Signal Transduction</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>2011</i></span>,  <span class="NLM_fpage">865819</span>, <span class="refDoi"> DOI: 10.1155/2011/865819</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.1155%2F2011%2F865819" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=21776389" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A280%3ADC%252BC3MnpvVKktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2011&publication_year=2011&pages=865819&author=B.+Senauthor=F.+M.+Johnson&title=Regulation+of+SRC+family+kinases+in+human+cancers&doi=10.1155%2F2011%2F865819"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of SRC family kinases in human cancers</span></div><div class="casAuthors">Sen Banibrata; Johnson Faye M</div><div class="citationInfo"><span class="NLM_cas:title">Journal of signal transduction</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">2011</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">865819</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The nonreceptor protein tyrosine kinase Src plays a crucial role in the signal transduction pathways involved in cell division, motility, adhesion, and survival in both normal and cancer cells.  Although the Src family kinases (SFKs) are activated in various types of cancers, the exact mechanisms through which they contribute to the progression of individual tumors remain to be defined.  The activation of Src in human cancers may occur through a variety of mechanisms that include domain interaction and structural remodeling in response to various activators or upstream kinases and phosphatastes.  Because of Src's prominent roles in invasion and tumor progression, epithelial-to-mesenchymal transition, angiogenesis, and the development of metastasis, Src is a promising target for cancer therapy.  Several small molecule inhibitors of Src are currently being investigated in clinical trials.  In this article, we will summarize the mechanisms regulating Src kinase activity in normal and cancer cells and discuss the status of Src inhibitor development against various types of cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSCfOL0q4nTvO-MdeMhXAHkfW6udTcc2eZcthb-hyYBELntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3MnpvVKktg%253D%253D&md5=f74b51d238ead275fd6d1179c3cc4346</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1155%2F2011%2F865819&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1155%252F2011%252F865819%26sid%3Dliteratum%253Aachs%26aulast%3DSen%26aufirst%3DB.%26aulast%3DJohnson%26aufirst%3DF.%2BM.%26atitle%3DRegulation%2520of%2520SRC%2520family%2520kinases%2520in%2520human%2520cancers%26jtitle%3DJ.%2520Signal%2520Transduction%26date%3D2011%26volume%3D2011%26spage%3D865819%26doi%3D10.1155%2F2011%2F865819" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wheeler, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iida, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dunn, E. F.</span></span> <span> </span><span class="NLM_article-title">The role of Src in solid tumors</span>. <i>Oncologist</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">667</span>– <span class="NLM_lpage">678</span>, <span class="refDoi"> DOI: 10.1634/theoncologist.2009-0009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.1634%2Ftheoncologist.2009-0009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=19581523" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVOmtrnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2009&pages=667-678&author=D.+L.+Wheelerauthor=M.+Iidaauthor=E.+F.+Dunn&title=The+role+of+Src+in+solid+tumors&doi=10.1634%2Ftheoncologist.2009-0009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">The role of Src in solid tumors</span></div><div class="casAuthors">Wheeler, Deric L.; Iida, Mari; Dunn, Emily F.</div><div class="citationInfo"><span class="NLM_cas:title">Oncologist</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">667-678</span>CODEN:
                <span class="NLM_cas:coden">OCOLF6</span>;
        ISSN:<span class="NLM_cas:issn">1083-7159</span>.
    
            (<span class="NLM_cas:orgname">AlphaMed Press</span>)
        </div><div class="casAbstract">A review.  The proto-oncogene c-Src (Src) encodes a nonreceptor tyrosine kinase whose expression and activity are correlated with advanced malignancy and poor prognosis in a variety of human cancers.  Nine addnl. enzymes with homol. to Src have been identified and collectively are referred to as Src family kinases (SFKs).  Together, SFKs represent the largest family of nonreceptor tyrosine kinases and interact directly with receptor tyrosine kinases, G-protein-coupled receptors, steroid receptors, signal transducers and activators of transcription, and mols. involved in cell adhesion and migration.  These interactions lead to a diverse array of biol. functions including proliferation, cell growth, differentiation, cell shape, motility, migration, angiogenesis, and survival.  Studies investigating mutational activation of Src in human cancers suggest that this may be a rare event and that wild-type Src is weakly oncogenic.  Thus, the role of Src in the development and progression of human cancer remains unclear.  Recently, it was suggested that increased SFK protein levels and, more importantly, SFK tyrosine kinase activity are linked to cancer progression and metastatic disease by facilitating the action of other signaling proteins.  This accumulating body of evidence indicates that SFKs may represent a promising therapeutic target for the treatment of solid tumors.  This review discusses the role of SFKs in solid tumors and the recent therapeutic advances aimed at targeting this family of tyrosine kinases in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpX_M5pcS60XLVg90H21EOLACvtfcHk0ljNuGfxRiiNdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVOmtrnN&md5=9596bcc70d4a44cd7c7519fefd77cce0</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1634%2Ftheoncologist.2009-0009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1634%252Ftheoncologist.2009-0009%26sid%3Dliteratum%253Aachs%26aulast%3DWheeler%26aufirst%3DD.%2BL.%26aulast%3DIida%26aufirst%3DM.%26aulast%3DDunn%26aufirst%3DE.%2BF.%26atitle%3DThe%2520role%2520of%2520Src%2520in%2520solid%2520tumors%26jtitle%3DOncologist%26date%3D2009%26volume%3D14%26spage%3D667%26epage%3D678%26doi%3D10.1634%2Ftheoncologist.2009-0009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Irby, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeatman, T. J.</span></span> <span> </span><span class="NLM_article-title">Role of Src expression and activation in human cancer</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">5636</span>, <span class="refDoi"> DOI: 10.1038/sj.onc.1203912</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.1038%2Fsj.onc.1203912" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=11114744" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BD3cXosl2nurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2000&pages=5636&author=R.+B.+Irbyauthor=T.+J.+Yeatman&title=Role+of+Src+expression+and+activation+in+human+cancer&doi=10.1038%2Fsj.onc.1203912"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Role of Src expression and activation in human cancer</span></div><div class="casAuthors">Irby, Rosalyn B.; Yeatman, Timothy J.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">5636-5642</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review with ~ 78 refs.  Since the original identification of a transmissible agent responsible for the development of tumors in chickens, now known to be a retrovirus encoding the v-src gene, significant progress has been made in defining the potential functions of its human homolog, SRC.  The product of the human SRC gene, c-Src, is found to be over-expressed and highly activated in a wide variety of human cancers.  The relationship between Src activation and cancer progression appears to be significant.  Moreover, Src may have an influence on the development of the metastatic phenotype.  This review discusses the data supporting a role for c-Src as a crit. component of the signal transduction pathways that control cancer cell development and growth, and provides the rationale for targeting Src in drug discovery efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAGCSyZ76JqLVg90H21EOLACvtfcHk0lgJSVk0ByHcHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXosl2nurs%253D&md5=1ec800320d1ae4976eb1c93af743cf6b</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1203912&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1203912%26sid%3Dliteratum%253Aachs%26aulast%3DIrby%26aufirst%3DR.%2BB.%26aulast%3DYeatman%26aufirst%3DT.%2BJ.%26atitle%3DRole%2520of%2520Src%2520expression%2520and%2520activation%2520in%2520human%2520cancer%26jtitle%3DOncogene%26date%3D2000%26volume%3D19%26spage%3D5636%26doi%3D10.1038%2Fsj.onc.1203912" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brugge, J. S.</span></span> <span> </span><span class="NLM_article-title">Cellular functions regulated by Src family kinases</span>. <i>Annu. Rev. Cell Dev. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">513</span>– <span class="NLM_lpage">609</span>, <span class="refDoi"> DOI: 10.1146/annurev.cellbio.13.1.513</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.1146%2Fannurev.cellbio.13.1.513" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=9442882" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADyaK1cXisFSrsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=1997&pages=513-609&author=S.+M.+Thomasauthor=J.+S.+Brugge&title=Cellular+functions+regulated+by+Src+family+kinases&doi=10.1146%2Fannurev.cellbio.13.1.513"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Cellular functions regulated by Src family kinases</span></div><div class="casAuthors">Thomas, Sheila M.; Brugge, Joan S.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Cell and Developmental Biology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">513-609</span>CODEN:
                <span class="NLM_cas:coden">ARDBF8</span>;
        ISSN:<span class="NLM_cas:issn">1081-0706</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review with many refs.  Src family protein tyrosine kinases are activated following engagement of many different classes of cellular receptors and participate in signaling pathways that control a diverse spectrum of receptor-induced biol. activities.  While several of these kinases have evolved to play distinct roles in specific receptor pathways, there is considerable redundancy in the functions of these kinases, both with respect to the receptor pathways that activate these kinases and the downstream effectors that mediate their biol. activities.  This chapter reviews the evidence implicating Src family kinases in specific receptor pathways and describes the mechanisms leading to their activation, the targets that interact with these kinases, and the biol. events that they regulate.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_psatEFVwjbVg90H21EOLACvtfcHk0lgJSVk0ByHcHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXisFSrsQ%253D%253D&md5=4b8d18cc815109e3e0a593b846984aac</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1146%2Fannurev.cellbio.13.1.513&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.cellbio.13.1.513%26sid%3Dliteratum%253Aachs%26aulast%3DThomas%26aufirst%3DS.%2BM.%26aulast%3DBrugge%26aufirst%3DJ.%2BS.%26atitle%3DCellular%2520functions%2520regulated%2520by%2520Src%2520family%2520kinases%26jtitle%3DAnnu.%2520Rev.%2520Cell%2520Dev.%2520Biol.%26date%3D1997%26volume%3D13%26spage%3D513%26epage%3D609%26doi%3D10.1146%2Fannurev.cellbio.13.1.513" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Finn, R.</span></span> <span> </span><span class="NLM_article-title">Targeting Src in breast cancer</span>. <i>Annals of Oncology</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">1379</span>– <span class="NLM_lpage">1386</span>, <span class="refDoi"> DOI: 10.1093/annonc/mdn291</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.1093%2Fannonc%2Fmdn291" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=18487549" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A280%3ADC%252BD1cvotV2rsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2008&pages=1379-1386&author=R.+Finn&title=Targeting+Src+in+breast+cancer&doi=10.1093%2Fannonc%2Fmdn291"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Src in breast cancer</span></div><div class="casAuthors">Finn R S</div><div class="citationInfo"><span class="NLM_cas:title">Annals of oncology : official journal of the European Society for Medical Oncology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1379-86</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The clinical benefit of blocking oncogenic pathways in breast cancer and other malignancies has validated this approach and ushered in the era of molecularly targeted therapeutics.  Src and its family members make up the largest group of nonreceptor tyrosine kinases.  In laboratory models, these proteins have been shown to play a critical role in cellular growth and proliferation, angiogenesis, and invasion and metastasis.  In addition, Src plays an important role in osteoclast activation and bone resorption, which are often aberrantly activated in the setting of bone metastases.  Given its role in these functions, blocking Src kinase would be predicted to have a broad therapeutic benefit in patients with Src-dependent cancers.  In this review, we highlight the rationale for targeting Src in breast cancer, including laboratory and clinical data implicating it in these signaling pathways, and review small-molecule tyrosine kinase inhibitors currently in clinical development.  Identifying which patients should be selected for Src-directed therapies will be important to the clinical success of these agents.  Importantly, recent preclinical data support a role for this class of inhibitors in basal-type/triple-negative breast cancer, which represents a group of patients with limited effective treatment options.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSq3-Zi9cm6LUNHe_gFldTbfW6udTcc2eZZh7oY-OfvKLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1cvotV2rsw%253D%253D&md5=b4f29b2ada4dcb6c89c0301a8440c617</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1093%2Fannonc%2Fmdn291&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fannonc%252Fmdn291%26sid%3Dliteratum%253Aachs%26aulast%3DFinn%26aufirst%3DR.%26atitle%3DTargeting%2520Src%2520in%2520breast%2520cancer%26jtitle%3DAnnals%2520of%2520Oncology%26date%3D2008%26volume%3D19%26spage%3D1379%26epage%3D1386%26doi%3D10.1093%2Fannonc%2Fmdn291" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dehm, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonham, K.</span></span> <span> </span><span class="NLM_article-title">SRC gene expression in human cancer: the role of transcriptional activation</span>. <i>Biochem. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>82</i></span>,  <span class="NLM_fpage">263</span>– <span class="NLM_lpage">274</span>, <span class="refDoi"> DOI: 10.1139/o03-077</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.1139%2Fo03-077" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=15060621" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjvFChsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=82&publication_year=2004&pages=263-274&author=S.+M.+Dehmauthor=K.+Bonham&title=SRC+gene+expression+in+human+cancer%3A+the+role+of+transcriptional+activation&doi=10.1139%2Fo03-077"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">SRC gene expression in human cancer: The role of transcriptional activation</span></div><div class="casAuthors">Dehm, Scott M.; Bonham, Keith</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry and Cell Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">82</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">263-274</span>CODEN:
                <span class="NLM_cas:coden">BCBIEQ</span>;
        ISSN:<span class="NLM_cas:issn">0829-8211</span>.
    
            (<span class="NLM_cas:orgname">National Research Council of Canada</span>)
        </div><div class="casAbstract">A review.  Human pp60c-Src (or c-Src) is a 60 kDa non-receptor tyrosine kinase encoded by the SRC gene and is the cellular homolog to the potent transforming v-Src viral oncogene.  C-Src functions at the hub of a vast array of signal transduction cascades that influence cellular proliferation, differentiation, motility, and survival.  C-Src activation has been documented in upwards of 50% of tumors derived from the colon, liver, lung, breast, and pancreas.  Therefore, a major focus has been to understand the mechanisms of c-Src activation in human cancer.  Early studies concd. on post-translational mechanisms that lead to increased c-Src kinase activity, which often correlated with overexpression of c-Src protein.  More recently, the discovery of an activating SRC mutation in a small subset of advanced colon tumors has been reported.  In addn., elevated SRC transcription has been identified as yet another mechanism contributing significantly to c-Src activation in a subset of human colon cancer cell lines.  Interestingly, histone deacetylase (HDAC) inhibitors, agents with well-documented anti-cancer activity, repress SRC transcription in a wide variety of human cancer cell lines.  Anal. of the mechanisms behind HDAC inhibitor mediated repression could be utilized in the future to specifically inhibit SRC gene expression in human cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoi4CZWd0vRL7Vg90H21EOLACvtfcHk0lgJSVk0ByHcHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjvFChsb8%253D&md5=3693d1b64959feb8218339c72e2c7b25</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1139%2Fo03-077&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1139%252Fo03-077%26sid%3Dliteratum%253Aachs%26aulast%3DDehm%26aufirst%3DS.%2BM.%26aulast%3DBonham%26aufirst%3DK.%26atitle%3DSRC%2520gene%2520expression%2520in%2520human%2520cancer%253A%2520the%2520role%2520of%2520transcriptional%2520activation%26jtitle%3DBiochem.%2520Cell%2520Biol.%26date%3D2004%26volume%3D82%26spage%3D263%26epage%3D274%26doi%3D10.1139%2Fo03-077" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koul, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMurray, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yung, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siminovitch, K. A.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Src family protein-tyrosine kinases alter the function of PTEN to regulate phosphatidylinositol 3-kinase/AKT cascades</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>278</i></span>,  <span class="NLM_fpage">40057</span>– <span class="NLM_lpage">40066</span>, <span class="refDoi"> DOI: 10.1074/jbc.M303621200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.1074%2Fjbc.M303621200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=12869565" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BD3sXnvFamsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=278&publication_year=2003&pages=40057-40066&author=Y.+Luauthor=Q.+Yuauthor=J.+H.+Liuauthor=J.+Zhangauthor=H.+Wangauthor=D.+Koulauthor=J.+S.+McMurrayauthor=X.+Fangauthor=W.+A.+Yungauthor=K.+A.+Siminovitch&title=Src+family+protein-tyrosine+kinases+alter+the+function+of+PTEN+to+regulate+phosphatidylinositol+3-kinase%2FAKT+cascades&doi=10.1074%2Fjbc.M303621200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Src Family Protein-tyrosine Kinases Alter the Function of PTEN to Regulate Phosphatidylinositol 3-Kinase/AKT Cascades</span></div><div class="casAuthors">Lu, Yiling; Yu, Qinghua; Liu, Jue Hui; Zhang, Jinyi; Wang, Hongwei; Koul, Dimpy; McMurray, John S.; Fang, Xianjun; Yung, W. K. Alfred; Siminovitch, Kathy A.; Mills, Gordon B.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">278</span>
        (<span class="NLM_cas:issue">41</span>),
    <span class="NLM_cas:pages">40057-40066</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Src family protein-tyrosine kinases, which play an important role in signal integration, have been implicated in tumorigenesis in multiple lineages, including breast cancer.  We demonstrate, herein, that Src kinases regulate the phosphatidylinositol 3-kinase (PI3K) signaling cascade via altering the function of the PTEN tumor suppressor.  Overexpression of activated Src protein-tyrosine kinases in PTEN-deficient breast cancer cells does not alter AKT phosphorylation, an indicator of signal transduction through the PI3K pathway.  However, in the presence of functional PTEN, Src reverses the activity of PTEN, resulting in an increase in AKT phosphorylation.  Activated Src reduces the ability of PTEN to dephosphorylate phosphatidylinositols in micelles and promotes AKT translocation to cellular plasma membranes but does not alter PTEN activity toward water-sol. phosphatidylinositols.  Thus, Src may alter the capacity of the PTEN C2 domain to bind cellular membranes rather than directly interfering with PTEN enzymic activity.  Tyrosine phosphorylation of PTEN is increased in breast cancer cells treated with pervanadate, suggesting that PTEN contains sites for tyrosine phosphorylation.  Src kinase inhibitors markedly decreased pervanadate-mediated tyrosine phosphorylation of PTEN.  Further, expression of activated Src results in marked tyrosine phosphorylation of PTEN.  SHP-1, a SH2 domain-contg. protein-tyrosine phosphatase, selectively binds and dephosphorylates PTEN in Src transfected cells.  Both Src inhibitors and SHP-1 overexpression reverse Src-induced loss of PTEN function.  Coexpression of PTEN with activated Src reduces the stability of PTEN.  Taken together, the data indicate that activated Src inhibits PTEN function leading to alterations in signaling through the PI3K/AKT pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWFNxCQ8gMqLVg90H21EOLACvtfcHk0liZm3B41bPFDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXnvFamsLs%253D&md5=2696aba274ab4ee0f85736f0f7b8344d</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M303621200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M303621200%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DY.%26aulast%3DYu%26aufirst%3DQ.%26aulast%3DLiu%26aufirst%3DJ.%2BH.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DKoul%26aufirst%3DD.%26aulast%3DMcMurray%26aufirst%3DJ.%2BS.%26aulast%3DFang%26aufirst%3DX.%26aulast%3DYung%26aufirst%3DW.%2BA.%26aulast%3DSiminovitch%26aufirst%3DK.%2BA.%26atitle%3DSrc%2520family%2520protein-tyrosine%2520kinases%2520alter%2520the%2520function%2520of%2520PTEN%2520to%2520regulate%2520phosphatidylinositol%25203-kinase%252FAKT%2520cascades%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2003%26volume%3D278%26spage%3D40057%26epage%3D40066%26doi%3D10.1074%2Fjbc.M303621200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cantley, L. C.</span></span> <span> </span><span class="NLM_article-title">The phosphoinositide 3-kinase pathway</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>296</i></span>,  <span class="NLM_fpage">1655</span>– <span class="NLM_lpage">1657</span>, <span class="refDoi"> DOI: 10.1126/science.296.5573.1655</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.1126%2Fscience.296.5573.1655" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=12040186" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BD38XktlChu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=296&publication_year=2002&pages=1655-1657&author=L.+C.+Cantley&title=The+phosphoinositide+3-kinase+pathway&doi=10.1126%2Fscience.296.5573.1655"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">The phosphoinositide 3-kinase pathway</span></div><div class="casAuthors">Cantley, Lewis C.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">296</span>
        (<span class="NLM_cas:issue">5573</span>),
    <span class="NLM_cas:pages">1655-1657</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A review with 8 refs.  Phosphorylated lipids are produced at cellular membranes during signaling events and contribute to the recruitment and activation of various signaling components.  The role of phosphoinositide 3-kinase (PI3K), which catalyzes the prodn. of phosphatidylinositol 3,4,5-trisphosphate, in cell survival pathways; the regulation of gene expression and cell metab.; and cytoskeletal rearrangements are highlighted.  The PI3K pathway is implicated in human diseases including diabetes and cancer, and understanding the intricacies of this pathway may provide new avenues for therapeutic intervention.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9pxSplxmS1rVg90H21EOLACvtfcHk0liZm3B41bPFDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XktlChu7s%253D&md5=110c8a351448aed6ab938b9ccf90a655</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1126%2Fscience.296.5573.1655&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.296.5573.1655%26sid%3Dliteratum%253Aachs%26aulast%3DCantley%26aufirst%3DL.%2BC.%26atitle%3DThe%2520phosphoinositide%25203-kinase%2520pathway%26jtitle%3DScience%26date%3D2002%26volume%3D296%26spage%3D1655%26epage%3D1657%26doi%3D10.1126%2Fscience.296.5573.1655" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lai, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, F. M.</span></span> <span> </span><span class="NLM_article-title">Defining the role of the JAK-STAT pathway in head and neck and thoracic malignancies: implications for future therapeutic approaches</span>. <i>Drug Resist. Updates</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">67</span>– <span class="NLM_lpage">78</span>, <span class="refDoi"> DOI: 10.1016/j.drup.2010.04.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.1016%2Fj.drup.2010.04.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=20471303" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BC3cXntVSmur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2010&pages=67-78&author=S.+Y.+Laiauthor=F.+M.+Johnson&title=Defining+the+role+of+the+JAK-STAT+pathway+in+head+and+neck+and+thoracic+malignancies%3A+implications+for+future+therapeutic+approaches&doi=10.1016%2Fj.drup.2010.04.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Defining the role of the JAK-STAT pathway in head and neck and thoracic malignancies: Implications for future therapeutic approaches</span></div><div class="casAuthors">Lai, Stephen Y.; Johnson, Faye M.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Resistance Updates</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">67-78</span>CODEN:
                <span class="NLM_cas:coden">DRUPFW</span>;
        ISSN:<span class="NLM_cas:issn">1368-7646</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Although the role of the Janus kinase (JAK)-signal transducer and activator of transcription (STAT) pathway has been most extensively studied in hematopoietic cells and hematol. malignancies, it is also activated in epithelial tumors, including those originating in the lungs and head and neck.  The canonical pathway involves the activation of JAK following ligand binding to cytokine receptors.  The activated JAKs then phosphorylate STAT proteins, leading to their dimerization and translocation into the nucleus.  In the nucleus, STATs act as transcription factors with pleiotropic downstream effects.  STATs can be activated independently of JAKs, most notably by c-Src kinases.  In cancer cells, STAT3 and STAT5 activation leads to the increased expression of downstream target genes, leading to increased cell proliferation, cell survival, angiogenesis, and immune system evasion.  STAT3 and STAT5 are expressed and activated in head and neck squamous cell carcinoma (HNSCC) where they contribute to cell survival and proliferation.  In HNSCC, STATs can be activated by a no. of signal transduction pathways, including the epidermal growth factor receptor (EGFR), α7 nicotinic receptor, interleukin (IL) receptor, and erythropoietin receptor pathways.  Activated STATs are also expressed in lung cancer, but the biol. effects of JAK/STAT inhibition in this cancer are variable.  In lung cancer, STAT3 can be activated by multiple pathways, including EGFR.  Several approaches have been used to inhibit STAT3 in the hopes of developing an antitumor agent.  Although several STAT3-specific agents are promising, none are in clin. development, mostly because of drug delivery and stability issues.  In contrast, several JAK inhibitors are in clin. development.  These orally available, ATP-competitive, small-mol. kinase inhibitors are being tested in myeloproliferative disorders.  Future studies will det. whether JAK inhibitors are useful in the treatment of HNSCC or lung cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZgZHT6MxwX7Vg90H21EOLACvtfcHk0liZm3B41bPFDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXntVSmur8%253D&md5=76183fde53da20b0a15bb4f000feb8f5</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.drup.2010.04.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drup.2010.04.001%26sid%3Dliteratum%253Aachs%26aulast%3DLai%26aufirst%3DS.%2BY.%26aulast%3DJohnson%26aufirst%3DF.%2BM.%26atitle%3DDefining%2520the%2520role%2520of%2520the%2520JAK-STAT%2520pathway%2520in%2520head%2520and%2520neck%2520and%2520thoracic%2520malignancies%253A%2520implications%2520for%2520future%2520therapeutic%2520approaches%26jtitle%3DDrug%2520Resist.%2520Updates%26date%3D2010%26volume%3D13%26spage%3D67%26epage%3D78%26doi%3D10.1016%2Fj.drup.2010.04.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maa, M.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leu, T.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trandel, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, J.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parsons, S. J.</span></span> <span> </span><span class="NLM_article-title">A protein that is highly related to GTPase-activating protein-associated p62 complexes with phospholipase C gamma</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">5466</span>– <span class="NLM_lpage">5473</span>, <span class="refDoi"> DOI: 10.1128/MCB.14.8.5466</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.1128%2FMCB.14.8.5466" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=7518563" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADyaK2cXlvF2hsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=1994&pages=5466-5473&author=M.-C.+Maaauthor=T.-H.+Leuauthor=B.+J.+Trandelauthor=J.-H.+Changauthor=S.+J.+Parsons&title=A+protein+that+is+highly+related+to+GTPase-activating+protein-associated+p62+complexes+with+phospholipase+C+gamma&doi=10.1128%2FMCB.14.8.5466"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">A protein that is highly related to GTPase-activating protein-associated p62 complexes with phospholipase Cγ</span></div><div class="casAuthors">Maa, Ming-Chei; Leu, Tzeng-Horng; Trandel, Barbara J.; Chang, Jin-Hong; Parsons, Sarah J.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">5466-73</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    </div><div class="casAbstract">P62 is a highly tyrosyl phosphorylated protein that was first identified in immunoppts. of the GTPase-activating protein (GAP) of p21ras from cells transformed by oncogenic nonreceptor tyrosine kinases or stimulated through tyrosine kinase receptors (C. Ellis, M. Moran, F. McCormick, and T. Pawson, Nature 343:377-381, 1991).  In this article we describe a highly related 62-kDa protein that becomes tyrosyl phosphorylated and assocd. with phospholipase Cγ (PLCγ) in C3H10T1/2 cells stimulated with epidermal growth factor (EGF) or transformed by v-src.  GAP-assocd. and PLCγ-assocd. p62 comigrated in one-dimensional sodium dodecyl sulfate-polyacrylamide gel electrophoresis and exhibited nearly identical phosphotryptic peptide patterns.  That the assocn. of p62 with PLCγ was direct and not mediated through binding of GAP-p62 to PLCγ or to the EGF receptor (and copptn. of the receptor with PLCγ) was demonstrated by (i) the inability to detect GAP in PLCγ immunocomplexes or PLCγ in GAP immunocomplexes, (ii) the assocn. of p62 with PLCγ in v-src-transformed cells in the absence of EGF stimulation, and (iii) in vitro soln. binding and direct blotting of p62 with a glutathione S-transferase fusion protein contg. the Src homol. 2 (SH2) domains of PLCγ.  Unlike GAP, whose N-terminal SH2 mediates the interaction between GAP and p62, PLCγ was found to require both its N- and C-terminal SH2 regions for p62 binding.  These studies demonstrate that a protein identical to or highly related to GAP-assocd. p62 binds PLCγ and suggest a means by which "cross-talk" between PLCγ- and GAP-mediated signalling may occur.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_R-OwJ8Bl-LVg90H21EOLACvtfcHk0lj7TjN37A9FNw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXlvF2hsbg%253D&md5=700f05d42b6812844eaed9c8a44c8855</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1128%2FMCB.14.8.5466&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.14.8.5466%26sid%3Dliteratum%253Aachs%26aulast%3DMaa%26aufirst%3DM.-C.%26aulast%3DLeu%26aufirst%3DT.-H.%26aulast%3DTrandel%26aufirst%3DB.%2BJ.%26aulast%3DChang%26aufirst%3DJ.-H.%26aulast%3DParsons%26aufirst%3DS.%2BJ.%26atitle%3DA%2520protein%2520that%2520is%2520highly%2520related%2520to%2520GTPase-activating%2520protein-associated%2520p62%2520complexes%2520with%2520phospholipase%2520C%2520gamma%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D1994%26volume%3D14%26spage%3D5466%26epage%3D5473%26doi%3D10.1128%2FMCB.14.8.5466" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ishizawar, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyake, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parsons, S.</span></span> <span> </span><span class="NLM_article-title">c-Src modulates ErbB2 and ErbB3 heterocomplex formation and function</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">3503</span>, <span class="refDoi"> DOI: 10.1038/sj.onc.1210138</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.1038%2Fsj.onc.1210138" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=17173075" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BD2sXls1Gqt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2007&pages=3503&author=R.+Ishizawarauthor=T.+Miyakeauthor=S.+Parsons&title=c-Src+modulates+ErbB2+and+ErbB3+heterocomplex+formation+and+function&doi=10.1038%2Fsj.onc.1210138"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">c-Src modulates ErbB2 and ErbB3 heterocomplex formation and function</span></div><div class="casAuthors">Ishizawar, R. C.; Miyake, T.; Parsons, S. J.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">3503-3510</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Overexpression and/or gene amplification of c-Src and members of the epidermal growth factor receptor (EGFR/ErbB) family have been implicated in the pathogenesis of breast cancer.  Although members of the EGFR family are known to form heterocomplexes with one another, c-Src has also been shown to phys. interact with members of this family in breast cancer cell lines and tumors.  This paper investigates the role of c-Src in modulating the phys. and functional interaction between ErbB2 and ErbB3, two family members that preferentially assoc. with one another and together exhibit high oncogenic potential.  We show that overexpressed wild-type c-Src enhances heterocomplex formation of ErbB2 and ErbB3 that results in increased basal and/or heregulin-induced activation of receptors, and their downstream intracellular effectors.  Expression of a kinase-inactive form of c-Src (K- c-Src) or pharmacol. inhibition of c-Src by PP2 neg. affects these events.  Furthermore, cellular motility and anchorage-independent growth promoted by the ErbB2/ErbB3 heterocomplex are dependent upon c-Src, as demonstrated by the effects of K- c-Src overexpression or treatment with PP2.  In contrast to previous studies that defined a role for c-Src downstream of ErbB2/ErbB3, the current work suggests an upstream mechanism, whereby c-Src enhances ErbB2/ErbB3 signaling and biol. functions by pos. modulating the assocn. between ErbB2 and ErbB3.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq78qm6P_s_hLVg90H21EOLACvtfcHk0lj7TjN37A9FNw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXls1Gqt7g%253D&md5=9894b8815d113e7a7b5ec9d127a66bcf</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1210138&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1210138%26sid%3Dliteratum%253Aachs%26aulast%3DIshizawar%26aufirst%3DR.%26aulast%3DMiyake%26aufirst%3DT.%26aulast%3DParsons%26aufirst%3DS.%26atitle%3Dc-Src%2520modulates%2520ErbB2%2520and%2520ErbB3%2520heterocomplex%2520formation%2520and%2520function%26jtitle%3DOncogene%26date%3D2007%26volume%3D26%26spage%3D3503%26doi%3D10.1038%2Fsj.onc.1210138" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">López-Ocejo, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viloria-Petit, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bequet-Romero, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mukhopadhyay, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rak, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerbel, R. S.</span></span> <span> </span><span class="NLM_article-title">Oncogenes and tumor angiogenesis: the HPV-16 E6 oncoprotein activates the vascular endothelial growth factor (VEGF) gene promoter in a p53 independent manner</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">4611</span>, <span class="refDoi"> DOI: 10.1038/sj.onc.1203817</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.1038%2Fsj.onc.1203817" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=11030150" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BD3cXnsFensLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2000&pages=4611&author=O.+L%C3%B3pez-Ocejoauthor=A.+Viloria-Petitauthor=M.+Bequet-Romeroauthor=D.+Mukhopadhyayauthor=J.+Rakauthor=R.+S.+Kerbel&title=Oncogenes+and+tumor+angiogenesis%3A+the+HPV-16+E6+oncoprotein+activates+the+vascular+endothelial+growth+factor+%28VEGF%29+gene+promoter+in+a+p53+independent+manner&doi=10.1038%2Fsj.onc.1203817"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Oncogenes and tumor angiogenesis: the HPV-16 E6 oncoprotein activates the vascular endothelial growth factor (VEGF) gene promoter in a p53 independent manner</span></div><div class="casAuthors">Lopez-Ocejo, Omar; Viloria-Petit, Alicia; Bequet-Romero, Monica; Mukhopadhyay, Debabrata; Rak, Janusz; Kerbel, Robert S.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">40</span>),
    <span class="NLM_cas:pages">4611-4620</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Like other types of pre-malignant lesions and carcinoma, angiogenesis is assocd. with high-grade cervical dysplasia and with invasive squamous carcinoma of the cervix.  Vascular endothelial cell growth factor (VEGF) is known to be one of the most important inducers of angiogenesis and is upregulated in carcinoma of the cervix.  Human Papilloma Virus 16 (HPV-16) has been etiol. linked to human cervical cancer, and the major oncogenic proteins encoded by the viral genome, E6 and E7, are involved in the immortalization of target cells.  Because several oncogenes including mutant ras, EGF receptor, ErbB2/Her2, c-myc and v-src upregulate VEGF expression, the authors asked whether HVP-16 E6 oncoprotein could act in a similar fashion.  The authors found that HPV-16 E6-pos. cells generally express high levels of VEGF message.  Furthermore, co-expression of the VEGF promoter-Luc (luciferase) reporter gene with E6 in both human keratinocytes and mouse fibroblast showed that E6 oncoprotein upregulates VEGF promoter activity, and does so in a p53 independent manner.  An E6 responsive region which comprises four Sp-1 sites, between -194 and -50 bp of the VEGF promoter, is also necessary for constitutive VEGF transcription.  Taken together, the authors' results suggest the possibility that the HPV oncoprotein E6 may contribute to tumor angiogenesis by direct stimulation of the VEGF gene.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprVYPMZxLxp7Vg90H21EOLACvtfcHk0lj7TjN37A9FNw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXnsFensLs%253D&md5=7e7b641648490b2e9aa6dcc7f65cfaa5</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1203817&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1203817%26sid%3Dliteratum%253Aachs%26aulast%3DL%25C3%25B3pez-Ocejo%26aufirst%3DO.%26aulast%3DViloria-Petit%26aufirst%3DA.%26aulast%3DBequet-Romero%26aufirst%3DM.%26aulast%3DMukhopadhyay%26aufirst%3DD.%26aulast%3DRak%26aufirst%3DJ.%26aulast%3DKerbel%26aufirst%3DR.%2BS.%26atitle%3DOncogenes%2520and%2520tumor%2520angiogenesis%253A%2520the%2520HPV-16%2520E6%2520oncoprotein%2520activates%2520the%2520vascular%2520endothelial%2520growth%2520factor%2520%2528VEGF%2529%2520gene%2520promoter%2520in%2520a%2520p53%2520independent%2520manner%26jtitle%3DOncogene%26date%3D2000%26volume%3D19%26spage%3D4611%26doi%3D10.1038%2Fsj.onc.1203817" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, D.</span></span> <span> </span><span class="NLM_article-title">Targeting Src family kinases in anti-cancer therapies: turning promise into triumph</span>. <i>Trends Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">122</span>– <span class="NLM_lpage">128</span>, <span class="refDoi"> DOI: 10.1016/j.tips.2011.11.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.1016%2Fj.tips.2011.11.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=22153719" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BC38XjtlOqtrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2012&pages=122-128&author=S.+Zhangauthor=D.+Yu&title=Targeting+Src+family+kinases+in+anti-cancer+therapies%3A+turning+promise+into+triumph&doi=10.1016%2Fj.tips.2011.11.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Src family kinases in anti-cancer therapies: turning promise into triumph</span></div><div class="casAuthors">Zhang, Siyuan; Yu, Dihua</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">122-128</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Src is a non-receptor tyrosine kinase that is deregulated in many types of cancer.  Decades of research have revealed the crucial role of Src in many aspects of tumor development, including proliferation, survival, adhesion, migration, invasion and, most importantly, metastasis, in multiple tumor types.  Despite extensive preclin. evidence that warrants targeting Src as a promising therapeutic approach for cancer, Src inhibitor(s) showed only minimal therapeutic activity in various types of solid tumors when used as a single agent in recent early-phase clin. trials.  In this review, we highlight the most recent advances from preclin. studies and clin. trials that shed light on potential clin. use of Src inhibitor-contg. combinatorial regimens in overcoming resistance to current anticancer therapies and in preventing metastatic recurrence.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZvQcvwj_23bVg90H21EOLACvtfcHk0li2mjl9KbRbdg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjtlOqtrk%253D&md5=22c9483e90ec6b390f42b74806d87d21</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2011.11.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2011.11.002%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DYu%26aufirst%3DD.%26atitle%3DTargeting%2520Src%2520family%2520kinases%2520in%2520anti-cancer%2520therapies%253A%2520turning%2520promise%2520into%2520triumph%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2012%26volume%3D33%26spage%3D122%26epage%3D128%26doi%3D10.1016%2Fj.tips.2011.11.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boland, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erdogan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasmatzis, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tillmans, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, M. R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halling, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliveira, A. M.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Anaplastic lymphoma kinase immunoreactivity correlates with ALK gene rearrangement and transcriptional up-regulation in non–small cell lung carcinomas</span>. <i>Hum. Pathol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">1152</span>– <span class="NLM_lpage">1158</span>, <span class="refDoi"> DOI: 10.1016/j.humpath.2009.01.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.1016%2Fj.humpath.2009.01.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=19386350" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BD1MXoslCrt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2009&pages=1152-1158&author=J.+M.+Bolandauthor=S.+Erdoganauthor=G.+Vasmatzisauthor=P.+Yangauthor=L.+S.+Tillmansauthor=M.+R.+E.+Johnsonauthor=X.+Wangauthor=L.+M.+Petersonauthor=K.+C.+Hallingauthor=A.+M.+Oliveira&title=Anaplastic+lymphoma+kinase+immunoreactivity+correlates+with+ALK+gene+rearrangement+and+transcriptional+up-regulation+in+non%E2%80%93small+cell+lung+carcinomas&doi=10.1016%2Fj.humpath.2009.01.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Anaplastic lymphoma kinase immunoreactivity correlates with ALK gene rearrangement and transcriptional up-regulation in non-small cell lung carcinomas</span></div><div class="casAuthors">Boland, Jennifer M.; Erdogan, Sibel; Vasmatzis, George; Yang, Ping; Tillmans, Lori S.; Johnson, Michele R. Erickson; Wang, Xiaoke; Peterson, Lisa M.; Halling, Kevin C.; Oliveira, Andre M.; Aubry, Marie Christine; Yi, Eunhee S.</div><div class="citationInfo"><span class="NLM_cas:title">Human Pathology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1152-1158</span>CODEN:
                <span class="NLM_cas:coden">HPCQA4</span>;
        ISSN:<span class="NLM_cas:issn">0046-8177</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Recently, the fusion gene EML4-ALK was identified in non-small cell lung carcinoma, which could be a potential therapeutic target.  We investigated the prevalence of anaplastic lymphoma kinase protein expression in these tumors by immunohistochem. and correlated the results with data from ALK mol. studies.  Gene expression profiling was performed on 35 adenocarcinomas to identify cases with ALK gene up-regulation, which was correlated with protein overexpression by immunohistochem.  Immunohistochem. was also performed on an independent cohort consisting of 150 adenocarcinomas and 150 squamous cell carcinomas to evaluate the utility of anaplastic lymphoma kinase immunostaining as a screening tool.  Florescence in situ hybridization for the ALK locus and reverse transcriptase-polymerase chain reaction for EML4-ALK were performed on tumors pos. for anaplastic lymphoma kinase by immunohistochem.  Transcriptional up-regulation of ALK was identified in 2 (6%) of 35 adenocarcinomas by gene expression profiling.  These 2 cases were pos. for anaplastic lymphoma kinase by immunohistochem., whereas the remaining 33 cases were completely neg.  In the independent cohort, anaplastic lymphoma kinase immunostaining was pos. in 1 of 150 squamous cell carcinomas and in 3 of 150 adenocarcinomas.  The 6 cases pos. for anaplastic lymphoma kinase by immunohistochem. showed evidence of ALK locus rearrangement by florescence in situ hybridization but were neg. for EGFR and KRAS mutation.  The presence of EML4-ALK fusion transcript was confirmed in 2 cases by reverse transcriptase-polymerase chain reaction.  In conclusion, anaplastic lymphoma kinase immunoreactivity in non-small cell lung carcinomas was assocd. with transcriptional up-regulation, ALK locus rearrangement, and the presence of EML4-ALK fusion transcript.  Anaplastic lymphoma kinase immunohistochem. may have utility as a screening tool or as a surrogate marker for the mol. techniques to detect the EML4-ALK fusion gene in these tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmuPu4P5jd7LVg90H21EOLACvtfcHk0li2mjl9KbRbdg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXoslCrt7w%253D&md5=787bd122ec1efff691f1095dfdead574</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.humpath.2009.01.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.humpath.2009.01.012%26sid%3Dliteratum%253Aachs%26aulast%3DBoland%26aufirst%3DJ.%2BM.%26aulast%3DErdogan%26aufirst%3DS.%26aulast%3DVasmatzis%26aufirst%3DG.%26aulast%3DYang%26aufirst%3DP.%26aulast%3DTillmans%26aufirst%3DL.%2BS.%26aulast%3DJohnson%26aufirst%3DM.%2BR.%2BE.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DPeterson%26aufirst%3DL.%2BM.%26aulast%3DHalling%26aufirst%3DK.%2BC.%26aulast%3DOliveira%26aufirst%3DA.%2BM.%26atitle%3DAnaplastic%2520lymphoma%2520kinase%2520immunoreactivity%2520correlates%2520with%2520ALK%2520gene%2520rearrangement%2520and%2520transcriptional%2520up-regulation%2520in%2520non%25E2%2580%2593small%2520cell%2520lung%2520carcinomas%26jtitle%3DHum.%2520Pathol.%26date%3D2009%26volume%3D40%26spage%3D1152%26epage%3D1158%26doi%3D10.1016%2Fj.humpath.2009.01.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Muthuswamy, S. K.</span></span> <span> </span><span class="NLM_article-title">Trastuzumab resistance: all roads lead to SRC</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">416</span>, <span class="refDoi"> DOI: 10.1038/nm0411-416</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.1038%2Fnm0411-416" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=21475230" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BC3MXksVCqsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2011&pages=416&author=S.+K.+Muthuswamy&title=Trastuzumab+resistance%3A+all+roads+lead+to+SRC&doi=10.1038%2Fnm0411-416"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Trastuzumab resistance: all roads lead to SRC</span></div><div class="casAuthors">Muthuswamy, Senthil K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">416-418</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  A new study shows how SRC, a nonmembrane tyrosine kinase, is a common signaling node in trastuzumab resistance caused by different mechanisms in HER2-pos. breast cancers (pages 461-469).  A SRC inhibitor restored trastuzumab sensitivity in vitro and in mouse tumor models, suggesting a new way to tackle drug resistance in breast tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsBPd2RG0rHbVg90H21EOLACvtfcHk0li6A5DybFSb6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXksVCqsrk%253D&md5=3617bbb8b395688050cff37927c71278</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fnm0411-416&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm0411-416%26sid%3Dliteratum%253Aachs%26aulast%3DMuthuswamy%26aufirst%3DS.%2BK.%26atitle%3DTrastuzumab%2520resistance%253A%2520all%2520roads%2520lead%2520to%2520SRC%26jtitle%3DNat.%2520Med.%26date%3D2011%26volume%3D17%26spage%3D416%26doi%3D10.1038%2Fnm0411-416" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yoshida, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sasaki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minami, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hibino, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okumura, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sado, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyokawa, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayashi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kitada, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohsaki, Y.</span></span> <span> </span><span class="NLM_article-title">Activation of Src signaling mediates acquired resistance to ALK inhibition in lung cancer</span>. <i>Int. J. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">1533</span>– <span class="NLM_lpage">1540</span>, <span class="refDoi"> DOI: 10.3892/ijo.2017.4140</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.3892%2Fijo.2017.4140" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=29048652" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVKjurjP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2017&pages=1533-1540&author=R.+Yoshidaauthor=T.+Sasakiauthor=Y.+Minamiauthor=Y.+Hibinoauthor=S.+Okumuraauthor=M.+Sadoauthor=N.+Miyokawaauthor=S.+Hayashiauthor=M.+Kitadaauthor=Y.+Ohsaki&title=Activation+of+Src+signaling+mediates+acquired+resistance+to+ALK+inhibition+in+lung+cancer&doi=10.3892%2Fijo.2017.4140"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Activation of Src signaling mediates acquired resistance to ALK inhibition in lung cancer</span></div><div class="casAuthors">Yoshida, Ryohei; Sasaki, Takaaki; Minami, Yoshinori; Hibino, Yukiko; Okumura, Shunsuke; Sado, Masatoshi; Miyokawa, Naoyuki; Hayashi, Satoshi; Kitada, Masahiro; Ohsaki, Yoshinobu</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Oncology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1533-1540</span>CODEN:
                <span class="NLM_cas:coden">IJONES</span>;
        ISSN:<span class="NLM_cas:issn">1791-2423</span>.
    
            (<span class="NLM_cas:orgname">Spandidos Publications Ltd.</span>)
        </div><div class="casAbstract">Anaplastic lymphoma kinase (ALK) fusion oncogenes occur in approx. 3-5% of non-small cell lung cancer (NSCLC) cases.  Various ALK inhibitors are in clin. use for the treatment of ALK-NSCLC, including the first generation ALK inhibitor, crizotinib, and recently the more highly potent alectinib and ceritinib.  However, most tumors eventually become resistant to ALK specific inhibitors.  To address the mechanisms underlying the development of ALK inhibitor resistance, we used iTRAQ quant. mass spectrometry and phosphor-receptor tyrosine kinase arrays to investigate intracellular signaling alterations in ALK inhibitor resistant NSCLC cell lines.  Src signaling was identified as an alectinib resistance mechanism, and combination treatment with ALK and Src inhibitors was highly effective for inhibiting the growth of ALK inhibitor resistant cells in vitro and in mouse xenograft models.  Furthermore, phospho-receptor tyrosine kinase activation and downstream PI3K/AKT signaling was effectively blocked by inhibiting Src in alectinib resistant cells.  Finally, we showed that the combined use of ALK and Src inhibitors inhibited the growth of other ALK-NSCLC cell lines, including those that were ceritinib or lorlatinib resistant.  Our data suggest that targeting Src signaling may be an effective approach to the treatment of ALK-NSCLC with acquired resistance to ALK inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRKtvR2ZB_R7Vg90H21EOLACvtfcHk0li6A5DybFSb6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVKjurjP&md5=3dda9a25c459c71bf8606db87a8076ad</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.3892%2Fijo.2017.4140&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3892%252Fijo.2017.4140%26sid%3Dliteratum%253Aachs%26aulast%3DYoshida%26aufirst%3DR.%26aulast%3DSasaki%26aufirst%3DT.%26aulast%3DMinami%26aufirst%3DY.%26aulast%3DHibino%26aufirst%3DY.%26aulast%3DOkumura%26aufirst%3DS.%26aulast%3DSado%26aufirst%3DM.%26aulast%3DMiyokawa%26aufirst%3DN.%26aulast%3DHayashi%26aufirst%3DS.%26aulast%3DKitada%26aufirst%3DM.%26aulast%3DOhsaki%26aufirst%3DY.%26atitle%3DActivation%2520of%2520Src%2520signaling%2520mediates%2520acquired%2520resistance%2520to%2520ALK%2520inhibition%2520in%2520lung%2520cancer%26jtitle%3DInt.%2520J.%2520Oncol.%26date%3D2017%26volume%3D51%26spage%3D1533%26epage%3D1540%26doi%3D10.3892%2Fijo.2017.4140" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Das, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norris, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Padmanabha, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moquin, R. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cook, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doweyko, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pitt, S.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">2-Aminothiazole as a novel kinase inhibitor template. Structure– activity relationship studies toward the discovery of N-(2-chloro-6-methylphenyl)-2-[[6-[4-(2-hydroxyethyl)-1-piperazinyl)]-2-methyl-4-pyrimidinyl] amino)]-1, 3-thiazole-5-carboxamide (dasatinib, BMS-354825) as a potent pan-Src kinase inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">6819</span>– <span class="NLM_lpage">6832</span>, <span class="refDoi"> DOI: 10.1021/jm060727j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm060727j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtFSlsLrI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=6819-6832&author=J.+Dasauthor=P.+Chenauthor=D.+Norrisauthor=R.+Padmanabhaauthor=J.+Linauthor=R.+V.+Moquinauthor=Z.+Shenauthor=L.+S.+Cookauthor=A.+M.+Doweykoauthor=S.+Pitt&title=2-Aminothiazole+as+a+novel+kinase+inhibitor+template.+Structure%E2%80%93+activity+relationship+studies+toward+the+discovery+of+N-%282-chloro-6-methylphenyl%29-2-%5B%5B6-%5B4-%282-hydroxyethyl%29-1-piperazinyl%29%5D-2-methyl-4-pyrimidinyl%5D+amino%29%5D-1%2C+3-thiazole-5-carboxamide+%28dasatinib%2C+BMS-354825%29+as+a+potent+pan-Src+kinase+inhibitor&doi=10.1021%2Fjm060727j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">2-Aminothiazole as a Novel Kinase Inhibitor Template. Structure-Activity Relationship Studies toward the Discovery of N-(2-Chloro-6-methylphenyl)-2-[[6-[4-(2-hydroxyethyl)-1-piperazinyl)]-2-methyl-4-pyrimidinyl]amino)]-1,3-thiazole-5-carboxamide (Dasatinib, BMS-354825) as a Potent pan-Src Kinase Inhibitor</span></div><div class="casAuthors">Das, Jagabandhu; Chen, Ping; Norris, Derek; Padmanabha, Ramesh; Lin, James; Moquin, Robert V.; Shen, Zhongqi; Cook, Lynda S.; Doweyko, Arthur M.; Pitt, Sidney; Pang, Suhong; Shen, Ding Ren; Fang, Qiong; de Fex, Henry F.; McIntyre, Kim W.; Shuster, David J.; Gillooly, Kathleen M.; Behnia, Kamelia; Schieven, Gary L.; Wityak, John; Barrish, Joel C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">6819-6832</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">2-Aminothiazole was discovered as a novel Src family kinase inhibitor template through screening of our internal compd. collection.  Optimization through successive structure-activity relationship iterations identified analogs I [R = 4-(2-hydroxyethyl)piperazino, Q] (Dasatinib, BMS-354825) and I [R = Me]as pan-Src inhibitors with nanomolar to subnanomolar potencies in biochem. and cellular assays.  Mol. modeling was used to construct a putative binding model for Lck inhibition by this class of compds.  The framework of key hydrogen-bond interactions proposed by this model was in agreement with the subsequent, published crystal structure of I [R = Q] bound to structurally similar Abl kinase.  The oral efficacy of this class of inhibitors was demonstrated with I [R = Me] in inhibiting the proinflammatory cytokine IL-2 ex vivo in mice (ED50 ∼ 5 mg/kg) and in reducing TNF levels in an acute murine model of inflammation (90% inhibition in LPS-induced TNFα prodn. when dosed orally at 60 mg/kg, 2 h prior to LPS administration).  The oral efficacy of I [R = Me] was further demonstrated in a chronic model of adjuvant arthritis in rats with established disease when administered orally at 0.3 and 3 mg/kg twice daily.  I [R = Q] is currently in clin. trials for the treatment of chronic myelogenous leukemia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrrbDd3Qks9cLVg90H21EOLACvtfcHk0lh8NpYNS1sADQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtFSlsLrI&md5=9eed66048de7ab90b8a8015385d6f581</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fjm060727j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm060727j%26sid%3Dliteratum%253Aachs%26aulast%3DDas%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DP.%26aulast%3DNorris%26aufirst%3DD.%26aulast%3DPadmanabha%26aufirst%3DR.%26aulast%3DLin%26aufirst%3DJ.%26aulast%3DMoquin%26aufirst%3DR.%2BV.%26aulast%3DShen%26aufirst%3DZ.%26aulast%3DCook%26aufirst%3DL.%2BS.%26aulast%3DDoweyko%26aufirst%3DA.%2BM.%26aulast%3DPitt%26aufirst%3DS.%26atitle%3D2-Aminothiazole%2520as%2520a%2520novel%2520kinase%2520inhibitor%2520template.%2520Structure%25E2%2580%2593%2520activity%2520relationship%2520studies%2520toward%2520the%2520discovery%2520of%2520N-%25282-chloro-6-methylphenyl%2529-2-%255B%255B6-%255B4-%25282-hydroxyethyl%2529-1-piperazinyl%2529%255D-2-methyl-4-pyrimidinyl%255D%2520amino%2529%255D-1%252C%25203-thiazole-5-carboxamide%2520%2528dasatinib%252C%2520BMS-354825%2529%2520as%2520a%2520potent%2520pan-Src%2520kinase%2520inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D6819%26epage%3D6832%26doi%3D10.1021%2Fjm060727j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boschelli, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dutia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yaczko, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, M.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Optimization of 4-phenylamino-3-quinolinecarbonitriles as potent inhibitors of Src kinase activity</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">3965</span>– <span class="NLM_lpage">3977</span>, <span class="refDoi"> DOI: 10.1021/jm0102250</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0102250" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BD3MXnsFyhsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2001&pages=3965-3977&author=D.+H.+Boschelliauthor=F.+Yeauthor=Y.+D.+Wangauthor=M.+Dutiaauthor=S.+L.+Johnsonauthor=B.+Wuauthor=K.+Millerauthor=D.+W.+Powellauthor=D.+Yaczkoauthor=M.+Young&title=Optimization+of+4-phenylamino-3-quinolinecarbonitriles+as+potent+inhibitors+of+Src+kinase+activity&doi=10.1021%2Fjm0102250"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of 4-Phenylamino-3-quinolinecarbonitriles as Potent Inhibitors of Src Kinase Activity</span></div><div class="casAuthors">Boschelli, Diane H.; Ye, Fei; Wang, Yanong D.; Dutia, Minu; Johnson, Steve L.; Wu, Biqi; Miller, Karen; Powell, Dennis W.; Yaczko, Deanna; Young, Mairead; Tischler, Mark; Arndt, Kim; Discafani, Carolyn; Etienne, Carlo; Gibbons, Jay; Grod, Janet; Lucas, Judy; Weber, Jennifer M.; Boschelli, Frank</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">3965-3977</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Subsequent to the discovery of 4-[(2,4-dichlorophenyl)amino]-6,7-dimethoxy-3-quinolinecarbonitrile as an inhibitor of Src kinase activity (IC50 = 30 nM), several addnl. analogs were prepd.  Optimization of the C-4 anilino group led to I [R = Me].  Replacement of the methoxy group at C-7 with a 3-(morpholin-4-yl)propoxy group provided I [R = morpholinopropyl], resulting in increased inhibition of both Src kinase activity and Src-mediated cell proliferation.  Analogs of I [R = morpholinopropyl] with other trisubstituted anilines at C-4 were also potent Src inhibitors, and the propoxy group was preferred over ethoxy, butoxy, or pentoxy.  Replacement of the morpholine group with a 4-methylpiperazine group provided I [R = 4-methylpiperazinopropyl], which had an IC50 of 1.2 nM in the Src enzymic assay, an IC50 of 100 nM for the inhibition of Src-dependent cell proliferation and was selective for Src over non-Src family kinases.  I [R = 4-methylpiperazinopropyl], which had higher 1 and 4 h plasma levels than I [R = 4-morpholinopropyl], effectively inhibited tumor growth in xenograft models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIfrD66BPWqrVg90H21EOLACvtfcHk0lh8NpYNS1sADQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXnsFyhsrk%253D&md5=88c2d8aa690b9aedbff034682aa3d73e</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fjm0102250&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0102250%26sid%3Dliteratum%253Aachs%26aulast%3DBoschelli%26aufirst%3DD.%2BH.%26aulast%3DYe%26aufirst%3DF.%26aulast%3DWang%26aufirst%3DY.%2BD.%26aulast%3DDutia%26aufirst%3DM.%26aulast%3DJohnson%26aufirst%3DS.%2BL.%26aulast%3DWu%26aufirst%3DB.%26aulast%3DMiller%26aufirst%3DK.%26aulast%3DPowell%26aufirst%3DD.%2BW.%26aulast%3DYaczko%26aufirst%3DD.%26aulast%3DYoung%26aufirst%3DM.%26atitle%3DOptimization%2520of%25204-phenylamino-3-quinolinecarbonitriles%2520as%2520potent%2520inhibitors%2520of%2520Src%2520kinase%2520activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2001%26volume%3D44%26spage%3D3965%26epage%3D3977%26doi%3D10.1021%2Fjm0102250" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Puls, L. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eadens, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Messersmith, W.</span></span> <span> </span><span class="NLM_article-title">Current status of SRC inhibitors in solid tumor malignancies</span>. <i>Oncologist</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">566</span>– <span class="NLM_lpage">578</span>, <span class="refDoi"> DOI: 10.1634/theoncologist.2010-0408</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.1634%2Ftheoncologist.2010-0408" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=21521831" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht12itr3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2011&pages=566-578&author=L.+N.+Pulsauthor=M.+Eadensauthor=W.+Messersmith&title=Current+status+of+SRC+inhibitors+in+solid+tumor+malignancies&doi=10.1634%2Ftheoncologist.2010-0408"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Current status of Src inhibitors in solid tumor malignancies</span></div><div class="casAuthors">Puls, Lauren N.; Eadens, Matthew; Messersmith, Wells</div><div class="citationInfo"><span class="NLM_cas:title">Oncologist</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">566-578</span>CODEN:
                <span class="NLM_cas:coden">OCOLF6</span>;
        ISSN:<span class="NLM_cas:issn">1083-7159</span>.
    
            (<span class="NLM_cas:orgname">AlphaMed Press</span>)
        </div><div class="casAbstract">A review.  Src is believed to play an important role in cancer, and several agents targeting Src are in clin. development.  Design. We reviewed Src structure and function and preclin. data supporting its role in the development of cancer via a PubMed search.  We conducted an extensive review of Src inhibitors by searching abstrs. from major oncol. meeting databases in the last 3 years and by comprehensively reviewing ongoing clin. trials on ClinicalTrials.gov.  Results. In this manuscript, we briefly review Src structure and function, mechanisms involving Src that lead to the development of cancer, and Src inhibitors and key preclin. data establishing a rationale for clin. application.  We then focus on clin. data supporting their use in solid tumor malignancies, a newer arena than their more well-established hematol. applications.  Particularly highlighted are clin. trials investigating new biomarkers as well as ongoing studies assessing Src inhibitor activity in biomarker-selected patient populations.  We also review newer investigational Src-targeting agents.  Conclusions. Src inhibitors have shown little activity in monotherapy trials in unselected solid tumor patient populations.  Combination studies and biomarker-driven clin. trials are under way.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYkp0FOYs6JLVg90H21EOLACvtfcHk0lh8NpYNS1sADQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht12itr3F&md5=85afcbb274a84ec0de7e87b37a72eff4</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1634%2Ftheoncologist.2010-0408&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1634%252Ftheoncologist.2010-0408%26sid%3Dliteratum%253Aachs%26aulast%3DPuls%26aufirst%3DL.%2BN.%26aulast%3DEadens%26aufirst%3DM.%26aulast%3DMessersmith%26aufirst%3DW.%26atitle%3DCurrent%2520status%2520of%2520SRC%2520inhibitors%2520in%2520solid%2520tumor%2520malignancies%26jtitle%3DOncologist%26date%3D2011%26volume%3D16%26spage%3D566%26epage%3D578%26doi%3D10.1634%2Ftheoncologist.2010-0408" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brandvold, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steffey, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fox, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soellner, M. B.</span></span> <span> </span><span class="NLM_article-title">Development of a highly selective c-Src kinase inhibitor</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">1393</span>– <span class="NLM_lpage">1398</span>, <span class="refDoi"> DOI: 10.1021/cb300172e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb300172e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BC38XntFShu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=1393-1398&author=K.+R.+Brandvoldauthor=M.+E.+Steffeyauthor=C.+C.+Foxauthor=M.+B.+Soellner&title=Development+of+a+highly+selective+c-Src+kinase+inhibitor&doi=10.1021%2Fcb300172e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Development of a Highly Selective c-Src Kinase Inhibitor</span></div><div class="casAuthors">Brandvold, Kristoffer R.; Steffey, Michael E.; Fox, Christel C.; Soellner, Matthew B.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1393-1398</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Generating highly selective probes to interrogate protein kinase function in biol. studies remains a challenge, and new strategies are required.  Herein, we describe the development of the first highly selective and cell-permeable inhibitor of c-Src, a key signaling kinase in cancer.  Our strategy involves extension of traditional inhibitor design by appending functionality proposed to interact with the phosphate-binding loop of c-Src.  Using our selective inhibitor, we demonstrate that selective inhibition is significantly more efficacious than pan-kinase inhibition in slowing the growth of cancer cells.  We also show that inhibition of c-Abl kinase, an off-target of most c-Src inhibitors, promotes oncogenic cell growth.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0-UPmOeW6T7Vg90H21EOLACvtfcHk0lg-JQG4CDggpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XntFShu7c%253D&md5=4a6e7e2777ddc26687707c3bef8fdc02</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fcb300172e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb300172e%26sid%3Dliteratum%253Aachs%26aulast%3DBrandvold%26aufirst%3DK.%2BR.%26aulast%3DSteffey%26aufirst%3DM.%2BE.%26aulast%3DFox%26aufirst%3DC.%2BC.%26aulast%3DSoellner%26aufirst%3DM.%2BB.%26atitle%3DDevelopment%2520of%2520a%2520highly%2520selective%2520c-Src%2520kinase%2520inhibitor%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2012%26volume%3D7%26spage%3D1393%26epage%3D1398%26doi%3D10.1021%2Fcb300172e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, Z.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Discovery of 2-((3-Amino-4-methylphenyl) amino)-N-(2-methyl-5-(3-(trifluoromethyl) benzamido) phenyl)-4-(methylamino) pyrimidine-5-carboxamide (CHMFL-ABL-053) as a Potent, Selective, and Orally Available BCR-ABL/SRC/p38 Kinase Inhibitor for Chronic Myeloid Leukemia</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1984</span>– <span class="NLM_lpage">2004</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01618</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01618" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1aisLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1984-2004&author=X.+Liangauthor=X.+Liuauthor=B.+Wangauthor=F.+Zouauthor=A.+Wangauthor=S.+Qiauthor=C.+Chenauthor=Z.+Zhaoauthor=W.+Wangauthor=Z.+Qi&title=Discovery+of+2-%28%283-Amino-4-methylphenyl%29+amino%29-N-%282-methyl-5-%283-%28trifluoromethyl%29+benzamido%29+phenyl%29-4-%28methylamino%29+pyrimidine-5-carboxamide+%28CHMFL-ABL-053%29+as+a+Potent%2C+Selective%2C+and+Orally+Available+BCR-ABL%2FSRC%2Fp38+Kinase+Inhibitor+for+Chronic+Myeloid+Leukemia&doi=10.1021%2Facs.jmedchem.5b01618"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of 2-((3-Amino-4-methylphenyl)amino)-N-(2-methyl-5-(3-(trifluoromethyl)benzamido)phenyl)-4-(methylamino)pyrimidine-5-carboxamide (CHMFL-ABL-053) as a Potent, Selective, and Orally Available BCR-ABL/SRC/p38 Kinase Inhibitor for Chronic Myeloid Leukemia</span></div><div class="casAuthors">Liang, Xiaofei; Liu, Xiaochuan; Wang, Beilei; Zou, Fengming; Wang, Aoli; Qi, Shuang; Chen, Cheng; Zhao, Zheng; Wang, Wenchao; Qi, Ziping; Lv, Fengchao; Hu, Zhenquan; Wang, Li; Zhang, Shanchun; Liu, Qingsong; Liu, Jing</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1984-2004</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Starting from a dihydropyrimidopyrimidine core scaffold based compd. 27 (GNF-7), we discovered a highly potent (ABL1: IC50 of 70 nM) and selective (S score (1) = 0.02) BCR-ABL inhibitor 18a (CHMFL-ABL-053).  Compd. 18a did not exhibit apparent inhibitory activity against c-KIT kinase, which is the common target of currently clin. used BCR-ABL inhibitors.  Through significant suppression of the BCR-ABL autophosphorylation (EC50 about 100 nM) and downstream mediators such as STAT5, Crkl, and ERK's phosphorylation, 18a inhibited the proliferation of CML cell lines K562 (GI50 = 14 nM), KU812 (GI50 = 25 nM), and MEG-01 (GI50 = 16 nM).  A pharmacokinetic study revealed that 18a had over 4 h of half-life and 24% bioavailability in rats.  A 50 mg/kg/day dosage treatment could almost completely suppress tumor progression in the K562 cells inoculated xenograft mouse model.  As a potential useful drug candidate for CML, 18a is under extensive preclin. safety evaluation now.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3xXL7Oq_tALVg90H21EOLACvtfcHk0lg-JQG4CDggpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1aisLo%253D&md5=1a18d0735055b604b26951bc5794b7d4</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01618&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01618%26sid%3Dliteratum%253Aachs%26aulast%3DLiang%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DB.%26aulast%3DZou%26aufirst%3DF.%26aulast%3DWang%26aufirst%3DA.%26aulast%3DQi%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DC.%26aulast%3DZhao%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DQi%26aufirst%3DZ.%26atitle%3DDiscovery%2520of%25202-%2528%25283-Amino-4-methylphenyl%2529%2520amino%2529-N-%25282-methyl-5-%25283-%2528trifluoromethyl%2529%2520benzamido%2529%2520phenyl%2529-4-%2528methylamino%2529%2520pyrimidine-5-carboxamide%2520%2528CHMFL-ABL-053%2529%2520as%2520a%2520Potent%252C%2520Selective%252C%2520and%2520Orally%2520Available%2520BCR-ABL%252FSRC%252Fp38%2520Kinase%2520Inhibitor%2520for%2520Chronic%2520Myeloid%2520Leukemia%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1984%26epage%3D2004%26doi%3D10.1021%2Facs.jmedchem.5b01618" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, M.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, G.-B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, R.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, W.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiao, Y.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Design, synthesis, and structure–activity relationship studies of 3-(Phenylethynyl)-1 H-pyrazolo [3, 4-d] pyrimidin-4-amine derivatives as a new class of Src inhibitors with potent activities in models of triple negative breast cancer</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">3957</span>– <span class="NLM_lpage">3974</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00270</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00270" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlvVaksbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=3957-3974&author=C.-H.+Zhangauthor=M.-W.+Zhengauthor=Y.-P.+Liauthor=X.-D.+Linauthor=M.+Huangauthor=L.+Zhongauthor=G.-B.+Liauthor=R.-J.+Zhangauthor=W.-T.+Linauthor=Y.+Jiao&title=Design%2C+synthesis%2C+and+structure%E2%80%93activity+relationship+studies+of+3-%28Phenylethynyl%29-1+H-pyrazolo+%5B3%2C+4-d%5D+pyrimidin-4-amine+derivatives+as+a+new+class+of+Src+inhibitors+with+potent+activities+in+models+of+triple+negative+breast+cancer&doi=10.1021%2Facs.jmedchem.5b00270"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Design, Synthesis, and Structure-Activity Relationship Studies of 3-(Phenylethynyl)-1H-pyrazolo[3,4-d]pyrimidin-4-amine Derivatives as a New Class of Src Inhibitors with Potent Activities in Models of Triple Negative Breast Cancer</span></div><div class="casAuthors">Zhang, Chun-Hui; Zheng, Ming-Wu; Li, Ya-Ping; Lin, Xing-Dong; Huang, Mei; Zhong, Lei; Li, Guo-Bo; Zhang, Rong-Jie; Lin, Wan-Ting; Jiao, Yan; Wu, Xiao-Ai; Yang, Jiao; Xiang, Rong; Chen, Li-Juan; Zhao, Ying-Lan; Cheng, Wei; Wei, Yu-Quan; Yang, Sheng-Yong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3957-3974</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of 3-(phenylethynyl)-1H-pyrazolo[3,4-d]pyrimidin-4-amine derivs. were designed and synthesized.  Structure-activity relationship (SAR) anal. of these compds. led to the discovery of compd. 1j (I), which showed the highest inhibitory potency against the Src kinase and the most potent antiviability activity against the typical TNBC cell line MDA-MB-231 among all the synthesized compds.  Further kinase inhibition assays showed that compd. 1j was a multikinase inhibitor and potently inhibited Src (IC50 = 0.0009 μM) and MAPK signaling protein kinases B-RAF and C-RAF.  In an MDA-MB-231 xenograft mouse model, a once-daily dose of compd. 1j at 30 mg/kg for 18 days completely suppressed the tumor growth with a tumor inhibition rate larger than 100% without obvious toxicity.  It also displayed good pharmacokinetic properties in a preliminary pharmacokinetic assay.  Western blot and immunohistochem. assays revealed that compd. 1j significantly inhibited Src and MAPK signaling and markedly induced apoptosis in tumor tissues.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvXJluk6Zz57Vg90H21EOLACvtfcHk0lg-JQG4CDggpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlvVaksbg%253D&md5=a15a90d306d54efac683cc373940cde4</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00270&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00270%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DC.-H.%26aulast%3DZheng%26aufirst%3DM.-W.%26aulast%3DLi%26aufirst%3DY.-P.%26aulast%3DLin%26aufirst%3DX.-D.%26aulast%3DHuang%26aufirst%3DM.%26aulast%3DZhong%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DG.-B.%26aulast%3DZhang%26aufirst%3DR.-J.%26aulast%3DLin%26aufirst%3DW.-T.%26aulast%3DJiao%26aufirst%3DY.%26atitle%3DDesign%252C%2520synthesis%252C%2520and%2520structure%25E2%2580%2593activity%2520relationship%2520studies%2520of%25203-%2528Phenylethynyl%2529-1%2520H-pyrazolo%2520%255B3%252C%25204-d%255D%2520pyrimidin-4-amine%2520derivatives%2520as%2520a%2520new%2520class%2520of%2520Src%2520inhibitors%2520with%2520potent%2520activities%2520in%2520models%2520of%2520triple%2520negative%2520breast%2520cancer%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D3957%26epage%3D3974%26doi%3D10.1021%2Facs.jmedchem.5b00270" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fraser, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dowling, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Houston, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munro, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muir, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrington, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Webster, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frame, M. C.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Rapid discovery and structure–activity relationships of pyrazolopyrimidines that potently suppress breast cancer cell growth via SRC kinase inhibition with exceptional selectivity over ABL kinase</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">4697</span>– <span class="NLM_lpage">4710</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00065</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00065" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BC28Xmsl2rsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=4697-4710&author=C.+Fraserauthor=J.+C.+Dawsonauthor=R.+Dowlingauthor=D.+R.+Houstonauthor=J.+T.+Weissauthor=A.+F.+Munroauthor=M.+Muirauthor=L.+Harringtonauthor=S.+P.+Websterauthor=M.+C.+Frame&title=Rapid+discovery+and+structure%E2%80%93activity+relationships+of+pyrazolopyrimidines+that+potently+suppress+breast+cancer+cell+growth+via+SRC+kinase+inhibition+with+exceptional+selectivity+over+ABL+kinase&doi=10.1021%2Facs.jmedchem.6b00065"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid Discovery and Structure-Activity Relationships of Pyrazolopyrimidines That Potently Suppress Breast Cancer Cell Growth via SRC Kinase Inhibition with Exceptional Selectivity over ABL Kinase</span></div><div class="casAuthors">Fraser, Craig; Dawson, John C.; Dowling, Reece; Houston, Douglas R.; Weiss, Jason T.; Munro, Alison F.; Muir, Morwenna; Harrington, Lea; Webster, Scott P.; Frame, Margaret C.; Brunton, Valerie G.; Patton, E. Elizabeth; Carragher, Neil O.; Unciti-Broceta, Asier</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4697-4710</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Novel pyrazolopyrimidines displaying high potency and selectivity toward SRC family kinases have been developed by combining ligand-based design and phenotypic screening in an iterative manner.  Compds. were derived from the promiscuous kinase inhibitor PP1 to search for analogs that could potentially target a broad spectrum of kinases involved in cancer.  Phenotypic screening against MCF7 mammary adenocarcinoma cells generated target-agnostic structure-activity relationships that biased subsequent designs toward breast cancer treatment rather than to a particular target.  This strategy led to the discovery of two potent antiproliferative leads with phenotypically distinct anticancer mode of actions.  Kinase profiling and further optimization resulted in eCF506, the first small mol. with subnanomolar IC50 for SRC that requires 3 orders of magnitude greater concn. to inhibit ABL. eCF506 exhibits excellent water soly., an optimal DMPK profile and oral bioavailability, halts SRC-assocd. neuromast migration in zebrafish embryos without inducing life-threatening heart defects, and inhibits SRC phosphorylation in tumor xenografts in mice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq55yS4FWveyLVg90H21EOLACvtfcHk0lgrpWLbs1y7uA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xmsl2rsL0%253D&md5=e56206009ccca680432d12db23aa29fd</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00065&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00065%26sid%3Dliteratum%253Aachs%26aulast%3DFraser%26aufirst%3DC.%26aulast%3DDawson%26aufirst%3DJ.%2BC.%26aulast%3DDowling%26aufirst%3DR.%26aulast%3DHouston%26aufirst%3DD.%2BR.%26aulast%3DWeiss%26aufirst%3DJ.%2BT.%26aulast%3DMunro%26aufirst%3DA.%2BF.%26aulast%3DMuir%26aufirst%3DM.%26aulast%3DHarrington%26aufirst%3DL.%26aulast%3DWebster%26aufirst%3DS.%2BP.%26aulast%3DFrame%26aufirst%3DM.%2BC.%26atitle%3DRapid%2520discovery%2520and%2520structure%25E2%2580%2593activity%2520relationships%2520of%2520pyrazolopyrimidines%2520that%2520potently%2520suppress%2520breast%2520cancer%2520cell%2520growth%2520via%2520SRC%2520kinase%2520inhibition%2520with%2520exceptional%2520selectivity%2520over%2520ABL%2520kinase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D4697%26epage%3D4710%26doi%3D10.1021%2Facs.jmedchem.6b00065" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tintori, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fallacara, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Radi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zamperini, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dreassi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crespan, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maga, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schenone, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Musumeci, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brullo, C.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Combining X-ray crystallography and molecular modeling toward the optimization of pyrazolo [3, 4-d] pyrimidines as potent c-Src inhibitors active in vivo against neuroblastoma</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">347</span>– <span class="NLM_lpage">361</span>, <span class="refDoi"> DOI: 10.1021/jm5013159</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm5013159" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitVegur%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=347-361&author=C.+Tintoriauthor=A.+L.+Fallacaraauthor=M.+Radiauthor=C.+Zamperiniauthor=E.+Dreassiauthor=E.+Crespanauthor=G.+Magaauthor=S.+Schenoneauthor=F.+Musumeciauthor=C.+Brullo&title=Combining+X-ray+crystallography+and+molecular+modeling+toward+the+optimization+of+pyrazolo+%5B3%2C+4-d%5D+pyrimidines+as+potent+c-Src+inhibitors+active+in+vivo+against+neuroblastoma&doi=10.1021%2Fjm5013159"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Combining X-ray Crystallography and Molecular Modeling toward the Optimization of Pyrazolo[3,4-d]pyrimidines as Potent c-Src Inhibitors Active in Vivo against Neuroblastoma</span></div><div class="casAuthors">Tintori, Cristina; Fallacara, Anna Lucia; Radi, Marco; Zamperini, Claudio; Dreassi, Elena; Crespan, Emmanuele; Maga, Giovanni; Schenone, Silvia; Musumeci, Francesca; Brullo, Chiara; Richters, Andre; Gasparrini, Francesca; Angelucci, Adriano; Festuccia, Claudio; Delle Monache, Simona; Rauh, Daniel; Botta, Maurizio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">347-361</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">C-Src is a tyrosine kinase belonging to the Src-family kinases.  It is overexpressed and/or hyperactivated in a variety of cancer cells, thus its inhibition has been predicted to have therapeutic effects in solid tumors.  Recently, the pyrazolo[3,4-d]pyrimidine 3 was reported as a dual c-Src/Abl inhibitor.  Herein we describe a multidisciplinary drug discovery approach for the optimization of the lead 3 against c-Src.  Starting from the X-ray crystal structure of c-Src in complex with 3, Monte Carlo free energy perturbation calcns. were applied to guide the design of c-Src inhibitors with improved activities.  As a result, the introduction of a meta hydroxyl group on the C4 anilino ring was computed to be particularly favorable.  The potency of the synthesized inhibitors was increased with respect to the starting lead 3.  The best identified compds. were also found active in the inhibition of neuroblastoma cell proliferation.  Furthermore, compd. 29 also showed in vivo activity in xenograft model using SH-SY5Y cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpD79cr521X37Vg90H21EOLACvtfcHk0lgrpWLbs1y7uA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitVegur%252FM&md5=c45f20a3cd5ce12b5afff5ce7b67f1f5</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fjm5013159&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm5013159%26sid%3Dliteratum%253Aachs%26aulast%3DTintori%26aufirst%3DC.%26aulast%3DFallacara%26aufirst%3DA.%2BL.%26aulast%3DRadi%26aufirst%3DM.%26aulast%3DZamperini%26aufirst%3DC.%26aulast%3DDreassi%26aufirst%3DE.%26aulast%3DCrespan%26aufirst%3DE.%26aulast%3DMaga%26aufirst%3DG.%26aulast%3DSchenone%26aufirst%3DS.%26aulast%3DMusumeci%26aufirst%3DF.%26aulast%3DBrullo%26aufirst%3DC.%26atitle%3DCombining%2520X-ray%2520crystallography%2520and%2520molecular%2520modeling%2520toward%2520the%2520optimization%2520of%2520pyrazolo%2520%255B3%252C%25204-d%255D%2520pyrimidines%2520as%2520potent%2520c-Src%2520inhibitors%2520active%2520in%2520vivo%2520against%2520neuroblastoma%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D347%26epage%3D361%26doi%3D10.1021%2Fjm5013159" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, W.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Azam, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sundaramoorthi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banda, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lentini, S. P.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">9-(Arenethenyl) purines as dual Src/Abl kinase inhibitors targeting the inactive conformation: design, synthesis, and biological evaluation</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">4743</span>– <span class="NLM_lpage">4756</span>, <span class="refDoi"> DOI: 10.1021/jm900166t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm900166t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BD1MXotVGktr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=4743-4756&author=W.-S.+Huangauthor=X.+Zhuauthor=Y.+Wangauthor=M.+Azamauthor=D.+Wenauthor=R.+Sundaramoorthiauthor=R.+M.+Thomasauthor=S.+Liuauthor=G.+Bandaauthor=S.+P.+Lentini&title=9-%28Arenethenyl%29+purines+as+dual+Src%2FAbl+kinase+inhibitors+targeting+the+inactive+conformation%3A+design%2C+synthesis%2C+and+biological+evaluation&doi=10.1021%2Fjm900166t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">9-(Arenethenyl)purines as Dual Src/Abl Kinase Inhibitors Targeting the Inactive Conformation: Design, Synthesis, and Biological Evaluation</span></div><div class="casAuthors">Huang, Wei-Sheng; Zhu, Xiaotian; Wang, Yihan; Azam, Mohammad; Wen, David; Sundaramoorthi, Raji; Thomas, R. Mathew; Liu, Shuangying; Banda, Geetha; Lentini, Scott P.; Das, Sasmita; Xu, Qihong; Keats, Jeff; Wang, Frank; Wardwell, Scott; Ning, Yaoyu; Snodgrass, Joseph T.; Broudy, Marc I.; Russian, Karin; Daley, George Q.; Iuliucci, John; Dalgarno, David C.; Clackson, Tim; Sawyer, Tomi K.; Shakespeare, William C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">4743-4756</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel series of potent dual Src/Abl kinase inhibitors based on a 9-(arenethenyl)purine core has been identified.  Unlike traditional dual Src/Abl inhibitors targeting the active enzyme conformation, these inhibitors bind to the inactive, DFG-out conformation of both kinases.  Extensive SAR studies led to the discovery of potent and orally bioavailable inhibitors, some of which demonstrated in vivo efficacy.  Once-daily oral administration of inhibitor I (AP24226) significantly prolonged the survival of mice injected i.v. with wild type Bcr-Abl expressing Ba/F3 cells at a dose of 10 mg/kg.  In a sep. model, oral administration of I to mice bearing s.c. xenografts of Src Y527F expressing NIH 3T3 cells elicited dose-dependent tumor shrinkage with complete tumor regression obsd. at the highest dose.  Notably, several inhibitors, e.g., II (AP24163), exhibited modest cellular potency (IC50 = 300-400 nM) against the Bcr-Abl mutant T315I, a variant resistant to all currently marketed therapies for chronic myeloid leukemia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpanmqXXD3_P7Vg90H21EOLACvtfcHk0ljZxRGcbR4xlg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXotVGktr8%253D&md5=287f41409bf75e5384a2b6853bda3def</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fjm900166t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm900166t%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DW.-S.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DAzam%26aufirst%3DM.%26aulast%3DWen%26aufirst%3DD.%26aulast%3DSundaramoorthi%26aufirst%3DR.%26aulast%3DThomas%26aufirst%3DR.%2BM.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DBanda%26aufirst%3DG.%26aulast%3DLentini%26aufirst%3DS.%2BP.%26atitle%3D9-%2528Arenethenyl%2529%2520purines%2520as%2520dual%2520Src%252FAbl%2520kinase%2520inhibitors%2520targeting%2520the%2520inactive%2520conformation%253A%2520design%252C%2520synthesis%252C%2520and%2520biological%2520evaluation%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D4743%26epage%3D4756%26doi%3D10.1021%2Fjm900166t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kitagawa, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yokota, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gouda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narumi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohmoto, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishiwaki, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akita, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirii, Y.</span></span> <span> </span><span class="NLM_article-title">Activity-based kinase profiling of approved tyrosine kinase inhibitors</span>. <i>Genes to cells</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">110</span>– <span class="NLM_lpage">122</span>, <span class="refDoi"> DOI: 10.1111/gtc.12022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.1111%2Fgtc.12022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=23279183" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlajurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2013&pages=110-122&author=D.+Kitagawaauthor=K.+Yokotaauthor=M.+Goudaauthor=Y.+Narumiauthor=H.+Ohmotoauthor=E.+Nishiwakiauthor=K.+Akitaauthor=Y.+Kirii&title=Activity-based+kinase+profiling+of+approved+tyrosine+kinase+inhibitors&doi=10.1111%2Fgtc.12022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Activity-based kinase profiling of approved tyrosine kinase inhibitors</span></div><div class="casAuthors">Kitagawa, Daisuke; Yokota, Koichi; Gouda, Masaki; Narumi, Yugo; Ohmoto, Hiroshi; Nishiwaki, Eiji; Akita, Kensaku; Kirii, Yasuyuki</div><div class="citationInfo"><span class="NLM_cas:title">Genes to Cells</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">110-122</span>CODEN:
                <span class="NLM_cas:coden">GECEFL</span>;
        ISSN:<span class="NLM_cas:issn">1356-9597</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">The specificities of nine approved tyrosine kinase inhibitors (imatinib, dasatinib, nilotinib, gefitinib, erlotinib, lapatinib, sorafenib, sunitinib, and pazopanib) were detd. by activity-based kinase profiling using a large panel of human recombinant active kinases.  This panel consisted of 79 tyrosine kinases, 199 serine/threonine kinases, three lipid kinases, and 29 disease-relevant mutant kinases.  Many potential targets of each inhibitor were identified by kinase profiling at the Km for ATP.  In addn., profiling at a physiol. ATP concn. (1 mm) was carried out, and the IC50 values of the inhibitors against each kinase were compared with the estd. plasma-free concn. (calcd. from published pharmacokinetic parameters of plasma Ctrough and Cmax values).  This anal. revealed that the approved kinase inhibitors were well optimized for their target kinases.  This profiling also implicates activity at particular off-target kinases in drug side effects.  Thus, large-scale kinase profiling at both Km and physiol. ATP concns. could be useful in characterizing the targets and off-targets of kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUe0U00YKTHbVg90H21EOLACvtfcHk0ljZxRGcbR4xlg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlajurs%253D&md5=a30fd15aa57d46e7d6d461a80907e0e2</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1111%2Fgtc.12022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fgtc.12022%26sid%3Dliteratum%253Aachs%26aulast%3DKitagawa%26aufirst%3DD.%26aulast%3DYokota%26aufirst%3DK.%26aulast%3DGouda%26aufirst%3DM.%26aulast%3DNarumi%26aufirst%3DY.%26aulast%3DOhmoto%26aufirst%3DH.%26aulast%3DNishiwaki%26aufirst%3DE.%26aulast%3DAkita%26aufirst%3DK.%26aulast%3DKirii%26aufirst%3DY.%26atitle%3DActivity-based%2520kinase%2520profiling%2520of%2520approved%2520tyrosine%2520kinase%2520inhibitors%26jtitle%3DGenes%2520to%2520cells%26date%3D2013%26volume%3D18%26spage%3D110%26epage%3D122%26doi%3D10.1111%2Fgtc.12022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rix, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hantschel, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dürnberger, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rix, L. L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Planyavsky, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernbach, N. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaupe, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valent, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colinge, J.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Chemical proteomic profiles of the BCR-ABL inhibitors imatinib, nilotinib, and dasatinib reveal novel kinase and nonkinase targets</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">4055</span>– <span class="NLM_lpage">4063</span>, <span class="refDoi"> DOI: 10.1182/blood-2007-07-102061</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.1182%2Fblood-2007-07-102061" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=17720881" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtl2qsb%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2007&pages=4055-4063&author=U.+Rixauthor=O.+Hantschelauthor=G.+D%C3%BCrnbergerauthor=L.+L.+R.+Rixauthor=M.+Planyavskyauthor=N.+V.+Fernbachauthor=I.+Kaupeauthor=K.+L.+Bennettauthor=P.+Valentauthor=J.+Colinge&title=Chemical+proteomic+profiles+of+the+BCR-ABL+inhibitors+imatinib%2C+nilotinib%2C+and+dasatinib+reveal+novel+kinase+and+nonkinase+targets&doi=10.1182%2Fblood-2007-07-102061"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical proteomic profiles of the BCR-ABL inhibitors imatinib, nilotinib, and dasatinib reveal novel kinase and nonkinase targets</span></div><div class="casAuthors">Rix, Uwe; Hantschel, Oliver; Duernberger, Gerhard; Rix, Lily L. Remsing; Planyavsky, Melanie; Fernbach, Nora V.; Kaupe, Ines; Bennett, Keiryn L.; Valent, Peter; Colinge, Jacques; Koecher, Thomas; Superti-Furga, Giulio</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4055-4063</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">The BCR-ABL tyrosine kinase inhibitor imatinib represents the current front-line therapy in chronic myeloid leukemia.  Because many patients develop imatinib resistance, 2 second-generation drugs, nilotinib and dasatinib, displaying increased potency against BCR-ABL were developed.  To predict potential side effects and novel medical uses, we generated comprehensive drug-protein interaction profiles by chem. proteomics for all 3 drugs.  Our studies yielded 4 major findings: (1) The interaction profiles of the 3 drugs displayed strong differences and only a small overlap covering the ABL kinases.  (2) Dasatinib bound in excess of 30 Tyr and Ser/Thr kinases, including major regulators of the immune system, suggesting that dasatinib might have a particular impact on immune function.  (3) Despite the high specificity of nilotinib, the receptor tyrosine kinase DDR1 was identified and validated as an addnl. major target.  (4) The oxidoreductase NQO2 was bound and inhibited by imatinib and nilotinib at physiol. relevant drug concns., representing the 1st nonkinase target of these drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqe8e6qqCnkrrVg90H21EOLACvtfcHk0lhq4LyIs_CpgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtl2qsb%252FF&md5=ff968982e9c20806c0c5fe9612f88407</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1182%2Fblood-2007-07-102061&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2007-07-102061%26sid%3Dliteratum%253Aachs%26aulast%3DRix%26aufirst%3DU.%26aulast%3DHantschel%26aufirst%3DO.%26aulast%3DD%25C3%25BCrnberger%26aufirst%3DG.%26aulast%3DRix%26aufirst%3DL.%2BL.%2BR.%26aulast%3DPlanyavsky%26aufirst%3DM.%26aulast%3DFernbach%26aufirst%3DN.%2BV.%26aulast%3DKaupe%26aufirst%3DI.%26aulast%3DBennett%26aufirst%3DK.%2BL.%26aulast%3DValent%26aufirst%3DP.%26aulast%3DColinge%26aufirst%3DJ.%26atitle%3DChemical%2520proteomic%2520profiles%2520of%2520the%2520BCR-ABL%2520inhibitors%2520imatinib%252C%2520nilotinib%252C%2520and%2520dasatinib%2520reveal%2520novel%2520kinase%2520and%2520nonkinase%2520targets%26jtitle%3DBlood%26date%3D2007%26volume%3D110%26spage%3D4055%26epage%3D4063%26doi%3D10.1182%2Fblood-2007-07-102061" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Remsing
Rix, L L</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rix, U</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colinge, J</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hantschel, O</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, K L</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stranzl, T</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, A</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baumgartner, C</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valent, P</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Augustin, M</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Till, J H</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Superti-Furga, G</span></span> <span> </span><span class="NLM_article-title">Global target profile of the kinase inhibitor bosutinib in primary chronic myeloid leukemia cells</span>. <i>Leukemia</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">477</span>, <span class="refDoi"> DOI: 10.1038/leu.2008.334</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.1038%2Fleu.2008.334" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=19039322" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BD1MXivVKhu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2009&pages=477&author=L+L+Remsing%0ARixauthor=U+Rixauthor=J+Colingeauthor=O+Hantschelauthor=K+L+Bennettauthor=T+Stranzlauthor=A+Mullerauthor=C+Baumgartnerauthor=P+Valentauthor=M+Augustinauthor=J+H+Tillauthor=G+Superti-Furga&title=Global+target+profile+of+the+kinase+inhibitor+bosutinib+in+primary+chronic+myeloid+leukemia+cells&doi=10.1038%2Fleu.2008.334"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Global target profile of the kinase inhibitor bosutinib in primary chronic myeloid leukemia cells</span></div><div class="casAuthors">Remsing Rix, L. L.; Rix, U.; Colinge, J.; Hantschel, O.; Bennett, K. L.; Stranzl, T.; Mueller, A.; Baumgartner, C.; Valent, P.; Augustin, M.; Till, J. H.; Superti-Furga, G.</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">477-485</span>CODEN:
                <span class="NLM_cas:coden">LEUKED</span>;
        ISSN:<span class="NLM_cas:issn">0887-6924</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The detailed mol. mechanism of action of second-generation BCR-ABL tyrosine kinase inhibitors, including perturbed targets and pathways, should contribute to rationalized therapy in chronic myeloid leukemia (CML) or in other affected diseases.  Here, we characterized the target profile of the dual SRC/ABL inhibitor bosutinib employing a two-tiered approach using chem. proteomics to identify natural binders in whole cell lysates of primary CML and K562 cells in parallel to in vitro kinase assays against a large recombinant kinase panel.  The combined strategy resulted in a global survey of bosutinib targets comprised of over 45 novel tyrosine and serine/threonine kinases.  We have found clear differences in the target patterns of bosutinib in primary CML cells vs. the K562 cell line.  A comparison of bosutinib with dasatinib across the whole kinase panel revealed overlapping, but distinct, inhibition profiles.  Common among those were the SRC, ABL and TEC family kinases.  Bosutinib did not inhibit KIT or platelet-derived growth factor receptor, but prominently targeted the apoptosis-linked STE20 kinases.  Although in vivo bosutinib is inactive against ABL T315I, we found this clin. important mutant to be enzymically inhibited in the mid-nanomolar range.  Finally, bosutinib is the first kinase inhibitor shown to target CAMK2G, recently implicated in myeloid leukemia cell proliferation.  Leukemia (2009) 23, 477-485; doi:10.1038/leu.2008.334; published online 27 Nov. 2008.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8u0ixFotIMLVg90H21EOLACvtfcHk0lhq4LyIs_CpgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXivVKhu7w%253D&md5=b0d56d468ef66ddc2a6f012d08f8c1a4</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1038%2Fleu.2008.334&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fleu.2008.334%26sid%3Dliteratum%253Aachs%26aulast%3DRemsing%2BRix%26aufirst%3DL%2BL%26aulast%3DRix%26aufirst%3DU%26aulast%3DColinge%26aufirst%3DJ%26aulast%3DHantschel%26aufirst%3DO%26aulast%3DBennett%26aufirst%3DK%2BL%26aulast%3DStranzl%26aufirst%3DT%26aulast%3DMuller%26aufirst%3DA%26aulast%3DBaumgartner%26aufirst%3DC%26aulast%3DValent%26aufirst%3DP%26aulast%3DAugustin%26aufirst%3DM%26aulast%3DTill%26aufirst%3DJ%2BH%26aulast%3DSuperti-Furga%26aufirst%3DG%26atitle%3DGlobal%2520target%2520profile%2520of%2520the%2520kinase%2520inhibitor%2520bosutinib%2520in%2520primary%2520chronic%2520myeloid%2520leukemia%2520cells%26jtitle%3DLeukemia%26date%3D2009%26volume%3D23%26spage%3D477%26doi%3D10.1038%2Fleu.2008.334" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Keller-von
Amsberg, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koschmieder, S.</span></span> <span> </span><span class="NLM_article-title">Profile of bosutinib and its clinical potential in the treatment of chronic myeloid leukemia</span>. <i>OncoTargets Ther.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">99</span>, <span class="refDoi"> DOI: 10.2147/OTT.S19901</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.2147%2FOTT.S19901" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=23493838" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BC3sXkt1Wltr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2013&pages=99&author=G.+Keller-von%0AAmsbergauthor=S.+Koschmieder&title=Profile+of+bosutinib+and+its+clinical+potential+in+the+treatment+of+chronic+myeloid+leukemia&doi=10.2147%2FOTT.S19901"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Profile of bosutinib and its clinical potential in the treatment of chronic myeloid leukemia</span></div><div class="casAuthors">Keller-von Amsberg, Gunhild; Koschmieder, Steffen</div><div class="citationInfo"><span class="NLM_cas:title">OncoTargets and Therapy</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">99-106</span>CODEN:
                <span class="NLM_cas:coden">OTNHAZ</span>;
        ISSN:<span class="NLM_cas:issn">1178-6930</span>.
    
            (<span class="NLM_cas:orgname">Dove Medical Press Ltd.</span>)
        </div><div class="casAbstract">A review.  Bosutinib (SKI-606) is an orally available, once-daily, dual Src and Abl kinase inhibitor with promising clin. potential in first-, second-, and third-line treatment of chronic myeloid leukemia (CML).  Bosutinib effectively inhibits wild-type BCR-ABL and most imatinib-resistant BCR-ABL mutations except for V299L and T315I.  Low hematol. toxicity is a remarkable characteristic of this novel second-generation tyrosine kinase inhibitor, and this has been ascribed to its minimal activity against the platelet-derived growth factor receptor and KIT.  Low-grade, typically self-limiting diarrhea, which usually appears within the first few weeks after treatment initiation, represents the predominant toxicity of bosutinib.  Other treatment-assocd. adverse events are mostly mild to moderate.  Bosutinib has been approved by the US Food and Drug Administration for the treatment of chronic, accelerated, or blast phase Philadelphia chromosome-pos. CML in adult patients with resistance or intolerance to prior therapy.  This review summarizes the main properties of bosutinib and the currently available data on its clin. potential in the treatment of CML.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGryIlbJ7tWle7Vg90H21EOLACvtfcHk0lgMSOm5Flecyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXkt1Wltr8%253D&md5=3b42376e225c82e97a412daeabfc8aa9</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.2147%2FOTT.S19901&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2147%252FOTT.S19901%26sid%3Dliteratum%253Aachs%26aulast%3DKeller-von%2BAmsberg%26aufirst%3DG.%26aulast%3DKoschmieder%26aufirst%3DS.%26atitle%3DProfile%2520of%2520bosutinib%2520and%2520its%2520clinical%2520potential%2520in%2520the%2520treatment%2520of%2520chronic%2520myeloid%2520leukemia%26jtitle%3DOncoTargets%2520Ther.%26date%3D2013%26volume%3D6%26spage%3D99%26doi%3D10.2147%2FOTT.S19901" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hennequin, L. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breed, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curwen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fennell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambert-van der Brempt, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morgentin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norman, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olivier, A.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">N-(5-Chloro-1, 3-benzodioxol-4-yl)-7-[2-(4-methylpiperazin-1-yl) ethoxy]-5-(tetrahydro-2 H-pyran-4-yloxy) quinazolin-4-amine, a novel, highly selective, orally available, dual-specific c-Src/Abl kinase inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">6465</span>– <span class="NLM_lpage">6488</span>, <span class="refDoi"> DOI: 10.1021/jm060434q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm060434q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtVaku7%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=6465-6488&author=L.+F.+Hennequinauthor=J.+Allenauthor=J.+Breedauthor=J.+Curwenauthor=M.+Fennellauthor=T.+P.+Greenauthor=C.+Lambert-van+der+Bremptauthor=R.+Morgentinauthor=R.+A.+Normanauthor=A.+Olivier&title=N-%285-Chloro-1%2C+3-benzodioxol-4-yl%29-7-%5B2-%284-methylpiperazin-1-yl%29+ethoxy%5D-5-%28tetrahydro-2+H-pyran-4-yloxy%29+quinazolin-4-amine%2C+a+novel%2C+highly+selective%2C+orally+available%2C+dual-specific+c-Src%2FAbl+kinase+inhibitor&doi=10.1021%2Fjm060434q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">N-(5-Chloro-1,3-benzodioxol-4-yl)-7-[2-(4-methylpiperazin-1-yl)ethoxy]-5- (tetrahydro-2H-pyran-4-yloxy)quinazolin-4-amine, a Novel, Highly Selective, Orally Available, Dual-Specific c-Src/Abl Kinase Inhibitor</span></div><div class="casAuthors">Hennequin, Laurent F.; Allen, Jack; Breed, Jason; Curwen, Jon; Fennell, Michael; Green, Tim P.; Lambert van der Brempt, Christine; Morgentin, Remy; Norman, Richard A.; Olivier, Annie; Otterbein, Ludovic; Ple, Patrick A.; Warin, Nicolas; Costello, Gerard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">6465-6488</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Src family kinases (SFKs) are nonreceptor tyrosine kinases that are reported to be crit. for cancer progression.  We report here a novel subseries of C-5-substituted anilinoquinazolines that display high affinity and specificity for the tyrosine kinase domain of the c-Src and Abl enzymes.  These compds. exhibit high selectivity for SFKs over a panel of recombinant protein kinases, excellent pharmacokinetics, and in vivo activity following oral dosing. N-(5-Chloro-1,3-benzodioxol-4-yl)-7-[2-(4-methylpiperazin-1-yl)ethoxy]-5-(tetrahydro-2H-pyran-4-yloxy)quinazolin-4-amine(I) (AZD0530) inhibits c-Src and Abl enzymes at low nanomolar concns. and is highly selective over a range of kinases.  AZD0530 displays excellent pharmacokinetic parameters in animal preclinically and in man (t1/2 = 40 h).  AZD0530 is a potent inhibitor of tumor growth in a c-Src-transfected 3T3-fibroblast xenograft model in vivo and led to a significant increase in survival in a highly aggressive, orthotopic model of human pancreatic cancer when dosed orally once daily.  AZD0530 is currently undergoing clin. evaluation in man.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqexk9b8WOUPrVg90H21EOLACvtfcHk0lgMSOm5Flecyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtVaku7%252FP&md5=dd798dec4067acb6d21020c66544e8a0</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fjm060434q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm060434q%26sid%3Dliteratum%253Aachs%26aulast%3DHennequin%26aufirst%3DL.%2BF.%26aulast%3DAllen%26aufirst%3DJ.%26aulast%3DBreed%26aufirst%3DJ.%26aulast%3DCurwen%26aufirst%3DJ.%26aulast%3DFennell%26aufirst%3DM.%26aulast%3DGreen%26aufirst%3DT.%2BP.%26aulast%3DLambert-van%2Bder%2BBrempt%26aufirst%3DC.%26aulast%3DMorgentin%26aufirst%3DR.%26aulast%3DNorman%26aufirst%3DR.%2BA.%26aulast%3DOlivier%26aufirst%3DA.%26atitle%3DN-%25285-Chloro-1%252C%25203-benzodioxol-4-yl%2529-7-%255B2-%25284-methylpiperazin-1-yl%2529%2520ethoxy%255D-5-%2528tetrahydro-2%2520H-pyran-4-yloxy%2529%2520quinazolin-4-amine%252C%2520a%2520novel%252C%2520highly%2520selective%252C%2520orally%2520available%252C%2520dual-specific%2520c-Src%252FAbl%2520kinase%2520inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D6465%26epage%3D6488%26doi%3D10.1021%2Fjm060434q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Kinase inhibitors: the road ahead</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">353</span>, <span class="refDoi"> DOI: 10.1038/nrd.2018.21</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.1038%2Fnrd.2018.21" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=29545548" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BC1cXks12ktbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2018&pages=353&author=F.+M.+Fergusonauthor=N.+S.+Gray&title=Kinase+inhibitors%3A+the+road+ahead&doi=10.1038%2Fnrd.2018.21"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Kinase inhibitors: the road ahead</span></div><div class="casAuthors">Ferguson, Fleur M.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">353-377</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  Receptor tyrosine kinase signaling pathways have been successfully targeted to inhibit proliferation and angiogenesis for cancer therapy.  However, kinase deregulation has been firmly demonstrated to play an essential role in virtually all major disease areas.  Kinase inhibitor drug discovery programs have recently broadened their focus to include an expanded range of kinase targets and therapeutic areas.  In this Review, we provide an overview of the novel targets, biol. processes and disease areas that kinase-targeting small mols. are being developed against, highlight the assocd. challenges and assess the strategies and technologies that are enabling efficient generation of highly optimized kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEZPospb1TP7Vg90H21EOLACvtfcHk0lilU7mltHNk9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXks12ktbg%253D&md5=34e42911d8a70301905460c3507e737a</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2018.21&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2018.21%26sid%3Dliteratum%253Aachs%26aulast%3DFerguson%26aufirst%3DF.%2BM.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DKinase%2520inhibitors%253A%2520the%2520road%2520ahead%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2018%26volume%3D17%26spage%3D353%26doi%3D10.1038%2Fnrd.2018.21" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roskoski, R.</span></span> <span> </span><span class="NLM_article-title">Properties of FDA-approved small molecule protein kinase inhibitors</span>. <i>Pharmacol. Res.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>144</i></span>,  <span class="NLM_fpage">19</span>, <span class="refDoi"> DOI: 10.1016/j.phrs.2019.03.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.1016%2Fj.phrs.2019.03.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=30877063" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BC1MXntVGnurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=144&publication_year=2019&pages=19&author=R.+Roskoski&title=Properties+of+FDA-approved+small+molecule+protein+kinase+inhibitors&doi=10.1016%2Fj.phrs.2019.03.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Properties of FDA-approved small molecule protein kinase inhibitors</span></div><div class="casAuthors">Roskoski, Robert Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Research</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">144</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">19-50</span>CODEN:
                <span class="NLM_cas:coden">PHMREP</span>;
        ISSN:<span class="NLM_cas:issn">1043-6618</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Because mutations, overexpression, and dysregulation of protein kinases play essential roles in the pathogenesis of many illnesses, this enzyme family has become one of the most important drug targets in the past 20 years.  The US FDA has approved 48 small mol. protein kinase inhibitors, nearly all of which are orally effective with the exceptions of netarsudil (which is given as an eye drop) and temsirolimus (which is given i.v.).  Of the 48 approved drugs, the majority (25) target receptor protein-tyrosine kinases, ten target non-receptor protein-tyrosine kinases, and 13 target protein-serine/threonine protein kinases.  The data indicate that 43 of these drugs are used in the treatment of malignancies (36 against solid tumors including lymphomas and seven against non-solid tumors, e.g., leukemias).  Seven drugs are used in the treatment of non-malignancies: baricitinib, rheumatoid arthritis; fostamatinib, chronic immune thrombocytopenia; ruxolitinib, myelofibrosis and polycythemia vera; nintedanib, idiopathic pulmonary fibrosis; sirolimus, renal graft vs. host disease; netarsudil, glaucoma; tofacitinib, rheumatoid arthritis, Crohn disease, and ulcerative colitis.  Moreover, ibrutinib and sirolimus are used for the treatment of both malignant and non-malignant diseases.  The most common drug targets include ALK, B-Raf, BCR-Abl, epidermal growth factor receptor (EGFR), and vascular endothelial growth factor receptor (VEGFR).  Most of the small mol. inhibitors (45) interact directly with the protein kinase domain.  In contrast, sirolimus, temsirolimus, and everolimus are larger mols. (MW ≈ 1000) that bind to FKBP-12 to generate a complex that inhibits mTOR (mammalian target of rapamycin).  This review presents the available drug-enzyme X-ray crystal structures for 27 of the approved drugs as well as the chem. structures and physicochem. properties of all of the FDA-approved small mol. protein kinase antagonists.  Six of the drugs bind covalently and irreversibly to their target.  Twenty of the 48 drugs have mol. wts. greater than 500, exceeding a Lipinski rule of five criterion.  Excluding the macrolides (everolimus, sirolimus, temsirolimus), the av. mol. wt. of drugs is 480 with a range of 306 (ruxolitinib) to 615 (trametinib).  Nearly half of the antagonists (23) have a lipophilic efficiency with values of less than five while the recommended optima range from 5-10.  One of the vexing problems is the near universal development of resistance that is assocd. with the use of small mol. protein kinase inhibitors for the treatment of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLum_hDK00hbVg90H21EOLACvtfcHk0lilU7mltHNk9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXntVGnurg%253D&md5=a24dee87603bc4ea12bd34616e46f53d</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.phrs.2019.03.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.phrs.2019.03.006%26sid%3Dliteratum%253Aachs%26aulast%3DRoskoski%26aufirst%3DR.%26atitle%3DProperties%2520of%2520FDA-approved%2520small%2520molecule%2520protein%2520kinase%2520inhibitors%26jtitle%3DPharmacol.%2520Res.%26date%3D2019%26volume%3D144%26spage%3D19%26doi%3D10.1016%2Fj.phrs.2019.03.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sabnis, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buhrlage, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Developing irreversible inhibitors of the protein kinase cysteinome</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">146</span>– <span class="NLM_lpage">159</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2012.12.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.1016%2Fj.chembiol.2012.12.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=23438744" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtFWitbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2013&pages=146-159&author=Q.+Liuauthor=Y.+Sabnisauthor=Z.+Zhaoauthor=T.+Zhangauthor=S.+J.+Buhrlageauthor=L.+H.+Jonesauthor=N.+S.+Gray&title=Developing+irreversible+inhibitors+of+the+protein+kinase+cysteinome&doi=10.1016%2Fj.chembiol.2012.12.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Developing Irreversible Inhibitors of the Protein Kinase Cysteinome</span></div><div class="casAuthors">Liu, Qingsong; Sabnis, Yogesh; Zhao, Zheng; Zhang, Tinghu; Buhrlage, Sara J.; Jones, Lyn H.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">146-159</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Protein kinases are a large family of approx. 530 highly conserved enzymes that transfer a γ-phosphate group from ATP to a variety of amino acid residues, such as tyrosine, serine, and threonine, that serves as a ubiquitous mechanism for cellular signal transduction.  The clin. success of a no. of kinase-directed drugs and the frequent observation of disease causing mutations in protein kinases suggest that a large no. of kinases may represent therapeutically relevant targets.  To date, the majority of clin. and preclin. kinase inhibitors are ATP competitive, noncovalent inhibitors that achieve selectivity through recognition of unique features of particular protein kinases.  Recently, there has been renewed interest in the development of irreversible inhibitors that form covalent bonds with cysteine or other nucleophilic residues in the ATP-binding pocket.  Irreversible kinase inhibitors have a no. of potential advantages including prolonged pharmacodynamics, suitability for rational design, high potency, and ability to validate pharmacol. specificity through mutation of the reactive cysteine residue.  Here, we review recent efforts to develop cysteine-targeted irreversible protein kinase inhibitors and discuss their modes of recognizing the ATP-binding pocket and their biol. activity profiles.  In addn., we provided an informatics assessment of the potential "kinase cysteinome" and discuss strategies for the efficient development of new covalent inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVGD7TX3oIeLVg90H21EOLACvtfcHk0lglKnAX9zrb9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtFWitbs%253D&md5=cef80823123bae88362e816a0a638133</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2012.12.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2012.12.006%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DSabnis%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DBuhrlage%26aufirst%3DS.%2BJ.%26aulast%3DJones%26aufirst%3DL.%2BH.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DDeveloping%2520irreversible%2520inhibitors%2520of%2520the%2520protein%2520kinase%2520cysteinome%26jtitle%3DChem.%2520Biol.%26date%3D2013%26volume%3D20%26spage%3D146%26epage%3D159%26doi%3D10.1016%2Fj.chembiol.2012.12.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanizaki, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aref, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rusan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, S.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ercan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, R. G.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Development of covalent inhibitors that can overcome resistance to first-generation FGFR kinase inhibitors</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>111</i></span>,  <span class="NLM_fpage">E4869</span>– <span class="NLM_lpage">E4877</span>, <span class="refDoi"> DOI: 10.1073/pnas.1403438111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.1073%2Fpnas.1403438111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=25349422" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVWmsL%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2014&pages=E4869-E4877&author=L.+Tanauthor=J.+Wangauthor=J.+Tanizakiauthor=Z.+Huangauthor=A.+R.+Arefauthor=M.+Rusanauthor=S.-J.+Zhuauthor=Y.+Zhangauthor=D.+Ercanauthor=R.+G.+Liao&title=Development+of+covalent+inhibitors+that+can+overcome+resistance+to+first-generation+FGFR+kinase+inhibitors&doi=10.1073%2Fpnas.1403438111"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Development of covalent inhibitors that can overcome resistance to first-generation FGFR kinase inhibitors</span></div><div class="casAuthors">Tan, Li; Wang, Jun; Tanizaki, Junko; Huang, Zhifeng; Aref, Amir R.; Rusan, Maria; Zhu, Su-Jie; Zhang, Yiyun; Ercan, Dalia; Liao, Rachel G.; Capelletti, Marzia; Zhou, Wenjun; Hur, Wooyoung; Kim, NamDoo; Sim, Taebo; Gaudet, Suzanne; Barbie, David A.; Yeh, Jing-Ruey Joanna; Yun, Cai-Hong; Hammerman, Peter S.; Mohammadi, Moosa; Janne, Pasi A.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">45</span>),
    <span class="NLM_cas:pages">E4869-E4877</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The human FGF receptors (FGFRs) play crit. roles in various human cancers, and several FGFR inhibitors are currently under clin. investigation.  Resistance usually results from selection for mutant kinases that are impervious to the action of the drug or from up-regulation of compensatory signaling pathways.  Preclin. studies have demonstrated that resistance to FGFR inhibitors can be acquired through mutations in the FGFR gatekeeper residue, as clin. obsd. for FGFR4 in embryonal rhabdomyosarcoma and neuroendocrine breast carcinomas.  Here the authors report on the use of a structure-based drug design to develop two selective, next-generation covalent FGFR inhibitors, the FGFR irreversible inhibitors I and II.  To our knowledge, I and II are the first inhibitors that can potently inhibit the proliferation of cells dependent upon the gatekeeper mutants of FGFR1 or FGFR2, which confer resistance to first-generation clin. FGFR inhibitors such as NVP-BGJ398 and AZD4547.  Because of the conformational flexibility of the reactive acrylamide substituent, II has the unprecedented ability to inhibit both the EGF receptor (EGFR) and FGFR covalently by targeting two distinct cysteine residues.  The authors report the cocrystal structure of FGFR4 with I, which unexpectedly exhibits a "DFG-out" covalent binding mode.  The structural basis for dual FGFR and EGFR targeting by FIIN3 also is illustrated by crystal structures of II bound with FGFR4 V550L and EGFR L858R.  These results have important implications for the design of covalent FGFR inhibitors that can overcome clin. resistance and provide the first example, to the knowledge, of a kinase inhibitor that covalently targets cysteines located in different positions within the ATP-binding pocket.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmCbTDKPA32LVg90H21EOLACvtfcHk0lglKnAX9zrb9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVWmsL%252FP&md5=fd4a7eac1e0c17e7c9f2651dd2d76ac7</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1403438111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1403438111%26sid%3Dliteratum%253Aachs%26aulast%3DTan%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DTanizaki%26aufirst%3DJ.%26aulast%3DHuang%26aufirst%3DZ.%26aulast%3DAref%26aufirst%3DA.%2BR.%26aulast%3DRusan%26aufirst%3DM.%26aulast%3DZhu%26aufirst%3DS.-J.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DErcan%26aufirst%3DD.%26aulast%3DLiao%26aufirst%3DR.%2BG.%26atitle%3DDevelopment%2520of%2520covalent%2520inhibitors%2520that%2520can%2520overcome%2520resistance%2520to%2520first-generation%2520FGFR%2520kinase%2520inhibitors%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2014%26volume%3D111%26spage%3DE4869%26epage%3DE4877%26doi%3D10.1073%2Fpnas.1403438111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kwarcinski, F. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fox, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steffey, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soellner, M. B.</span></span> <span> </span><span class="NLM_article-title">Irreversible inhibitors of c-Src kinase that target a nonconserved cysteine</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">1910</span>– <span class="NLM_lpage">1917</span>, <span class="refDoi"> DOI: 10.1021/cb300337u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb300337u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1Kms77J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=1910-1917&author=F.+E.+Kwarcinskiauthor=C.+C.+Foxauthor=M.+E.+Steffeyauthor=M.+B.+Soellner&title=Irreversible+inhibitors+of+c-Src+kinase+that+target+a+nonconserved+cysteine&doi=10.1021%2Fcb300337u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Irreversible Inhibitors of c-Src Kinase That Target a Nonconserved Cysteine</span></div><div class="casAuthors">Kwarcinski, Frank E.; Fox, Christel C.; Steffey, Michael E.; Soellner, Matthew B.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1910-1917</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We have developed the first irreversible inhibitors of wild-type c-Src kinase.  We demonstrate that our irreversible inhibitors display improved potency and selectivity relative to that of their reversible counterparts.  Our strategy involves modifying a promiscuous kinase inhibitor with an electrophile to generate covalent inhibitors of c-Src.  We applied this methodol. to two inhibitor scaffolds that exhibit increased cellular efficacy when rendered irreversible.  In addn., we have demonstrated the utility of irreversible inhibitors in studying the conformation of an important loop in kinases that can control inhibitor selectivity and cause drug resistance.  Together, we have developed a general and robust framework for generating selective irreversible inhibitors from reversible, promiscuous inhibitor scaffolds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcojrr4XAxmLVg90H21EOLACvtfcHk0ljHMx0v1_sKfg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1Kms77J&md5=db928e772f1ed18be2ac37ec930d743c</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fcb300337u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb300337u%26sid%3Dliteratum%253Aachs%26aulast%3DKwarcinski%26aufirst%3DF.%2BE.%26aulast%3DFox%26aufirst%3DC.%2BC.%26aulast%3DSteffey%26aufirst%3DM.%2BE.%26aulast%3DSoellner%26aufirst%3DM.%2BB.%26atitle%3DIrreversible%2520inhibitors%2520of%2520c-Src%2520kinase%2520that%2520target%2520a%2520nonconserved%2520cysteine%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2012%26volume%3D7%26spage%3D1910%26epage%3D1917%26doi%3D10.1021%2Fcb300337u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gurbani, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weisberg, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunter, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ficarro, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mowafy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tam, C.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, S.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Structure-guided development of covalent TAK1 inhibitors</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">838</span>– <span class="NLM_lpage">846</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2016.11.035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.1016%2Fj.bmc.2016.11.035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=28011204" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFKlsbjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2017&pages=838-846&author=L.+Tanauthor=D.+Gurbaniauthor=E.+L.+Weisbergauthor=J.+C.+Hunterauthor=L.+Liauthor=D.+S.+Jonesauthor=S.+B.+Ficarroauthor=S.+Mowafyauthor=C.-P.+Tamauthor=S.+Rao&title=Structure-guided+development+of+covalent+TAK1+inhibitors&doi=10.1016%2Fj.bmc.2016.11.035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-guided development of covalent TAK1 inhibitors</span></div><div class="casAuthors">Tan, Li; Gurbani, Deepak; Weisberg, Ellen L.; Hunter, John C.; Li, Lianbo; Jones, Douglas S.; Ficarro, Scott B.; Mowafy, Samar; Tam, Chun-Pong; Rao, Suman; Du, Guangyan; Griffin, James D.; Sorger, Peter K.; Marto, Jarrod A.; Westover, Kenneth D.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">838-846</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">TAK1 (transforming growth factor-β-activated kinase 1) is an essential intracellular mediator of cytokine and growth factor signaling and a potential therapeutic target for the treatment of immune diseases and cancer.  Herein we report development of a series of 2,4-disubstituted pyrimidine covalent TAK1 inhibitors that target Cys174, a residue immediately adjacent to the 'DFG-motif' of the kinase activation loop.  Co-crystal structures of TAK1 with candidate compds. enabled iterative rounds of structure-based design and biol. testing to arrive at optimized compds.  Lead compds. such as 2 and 10 showed greater than 10-fold biochem. selectivity for TAK1 over the closely related kinases MEK1 and ERK1 which possess an equivalently positioned cysteine residue.  These compds. are smaller, more easily synthesized, and exhibit a different spectrum of kinase selectivity relative to previously reported macrocyclic natural product TAK1 inhibitors such as 5Z-7-oxozeanol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkSYJFH5tOKrVg90H21EOLACvtfcHk0ljHMx0v1_sKfg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFKlsbjO&md5=8dda5fe018d344e1950e9641a01de54e</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2016.11.035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2016.11.035%26sid%3Dliteratum%253Aachs%26aulast%3DTan%26aufirst%3DL.%26aulast%3DGurbani%26aufirst%3DD.%26aulast%3DWeisberg%26aufirst%3DE.%2BL.%26aulast%3DHunter%26aufirst%3DJ.%2BC.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DJones%26aufirst%3DD.%2BS.%26aulast%3DFicarro%26aufirst%3DS.%2BB.%26aulast%3DMowafy%26aufirst%3DS.%26aulast%3DTam%26aufirst%3DC.-P.%26aulast%3DRao%26aufirst%3DS.%26atitle%3DStructure-guided%2520development%2520of%2520covalent%2520TAK1%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2017%26volume%3D25%26spage%3D838%26epage%3D846%26doi%3D10.1016%2Fj.bmc.2016.11.035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gurbani, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weisberg, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singer, W. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernard, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mowafy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenney, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, G.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Studies of TAK1-centered polypharmacology with novel covalent TAK1 inhibitors</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1320</span>– <span class="NLM_lpage">1328</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2016.11.034</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.1016%2Fj.bmc.2016.11.034" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=28038940" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BC2sXit1yktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2017&pages=1320-1328&author=L.+Tanauthor=D.+Gurbaniauthor=E.+L.+Weisbergauthor=D.+S.+Jonesauthor=S.+Raoauthor=W.+D.+Singerauthor=F.+M.+Bernardauthor=S.+Mowafyauthor=A.+Jenneyauthor=G.+Du&title=Studies+of+TAK1-centered+polypharmacology+with+novel+covalent+TAK1+inhibitors&doi=10.1016%2Fj.bmc.2016.11.034"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Studies of TAK1-centered polypharmacology with novel covalent TAK1 inhibitors</span></div><div class="casAuthors">Tan, Li; Gurbani, Deepak; Weisberg, Ellen L.; Jones, Douglas S.; Rao, Suman; Singer, William D.; Bernard, Faviola M.; Mowafy, Samar; Jenney, Annie; Du, Guangyan; Nonami, Atsushi; Griffin, James D.; Lauffenburger, Douglas A.; Westover, Kenneth D.; Sorger, Peter K.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1320-1328</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Targeted polypharmacol. provides an efficient method of treating diseases such as cancer with complex, multigenic causes provided that compds. with advantageous activity profiles can be discovered.  Novel covalent TAK1 inhibitors were validated in cellular contexts for their ability to inhibit the TAK1 kinase and for their polypharmacol.  Several inhibitors phenocopied reported TAK1 inhibitor 5Z-7-oxozaenol with comparable efficacy and complementary kinase selectivity profiles.  Compd. 5 exhibited the greatest potency in RAS-mutated and wild-type RAS cell lines from various cancer types.  A biotinylated deriv. of 5, 27, was used to verify TAK1 binding in cells.  The newly described inhibitors constitute useful tools for further development of multi-targeting TAK1-centered inhibitors for cancer and other diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolPDq91GVGo7Vg90H21EOLACvtfcHk0lgaSFw_QhYkyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXit1yktg%253D%253D&md5=ae5a9dffd65178d6430ce1c766735650</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2016.11.034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2016.11.034%26sid%3Dliteratum%253Aachs%26aulast%3DTan%26aufirst%3DL.%26aulast%3DGurbani%26aufirst%3DD.%26aulast%3DWeisberg%26aufirst%3DE.%2BL.%26aulast%3DJones%26aufirst%3DD.%2BS.%26aulast%3DRao%26aufirst%3DS.%26aulast%3DSinger%26aufirst%3DW.%2BD.%26aulast%3DBernard%26aufirst%3DF.%2BM.%26aulast%3DMowafy%26aufirst%3DS.%26aulast%3DJenney%26aufirst%3DA.%26aulast%3DDu%26aufirst%3DG.%26atitle%3DStudies%2520of%2520TAK1-centered%2520polypharmacology%2520with%2520novel%2520covalent%2520TAK1%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2017%26volume%3D25%26spage%3D1320%26epage%3D1328%26doi%3D10.1016%2Fj.bmc.2016.11.034" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gurbani, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Everley, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Browne, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaikuad, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schröder, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gondi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ficarro, S. B.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Leveraging compound promiscuity to identify targetable cysteines within the kinome</span>. <i>Cell chemical biology</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">818</span>– <span class="NLM_lpage">829.e9</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2019.02.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.1016%2Fj.chembiol.2019.02.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=30982749" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BC1MXnt1CksL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2019&pages=818-829.e9&author=S.+Raoauthor=D.+Gurbaniauthor=G.+Duauthor=R.+A.+Everleyauthor=C.+M.+Browneauthor=A.+Chaikuadauthor=L.+Tanauthor=M.+Schr%C3%B6derauthor=S.+Gondiauthor=S.+B.+Ficarro&title=Leveraging+compound+promiscuity+to+identify+targetable+cysteines+within+the+kinome&doi=10.1016%2Fj.chembiol.2019.02.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Leveraging Compound Promiscuity to Identify Targetable Cysteines within the Kinome</span></div><div class="casAuthors">Rao, Suman; Gurbani, Deepak; Du, Guangyan; Everley, Robert A.; Browne, Christopher M.; Chaikuad, Apirat; Li, Tan; Schroder, Martin; Gondi, Sudershan; Ficarro, Scott B.; Sim, Taebo; Kim, Nam Doo; Berberich, Matthew J.; Knapp, Stefan; Marto, Jarrod A.; Westover, Kenneth D.; Sorger, Peter K.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">818-829.e9</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Covalent kinase inhibitors, which typically target cysteine residues, represent an important class of clin. relevant compds.  Approx. 215 kinases are known to have potentially targetable cysteines distributed across 18 spatially distinct locations proximal to the ATP-binding pocket.  However, only 40 kinases have been covalently targeted, with certain cysteine sites being the primary focus.  To address this disparity, we have developed a strategy that combines the use of a multi-targeted acrylamide-modified inhibitor, SM1-71, with a suite of complementary chemoproteomic and cellular approaches to identify addnl. targetable cysteines.  Using this single multi-targeted compd., we successfully identified 23 kinases that are amenable to covalent inhibition including MKNK2, MAP2K1/2/3/4/6/7, GAK, AAK1, BMP2K, MAP3K7, MAPKAPK5, GSK3A/B, MAPK1/3, SRC, YES1, FGFR1, ZAK (MLTK), MAP3K1, LIMK1, and RSK2.  The identification of nine of these kinases previously not targeted by a covalent inhibitor increases the no. of targetable kinases and highlights opportunities for covalent kinase inhibitor development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUcHJFGWFTTrVg90H21EOLACvtfcHk0lgaSFw_QhYkyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXnt1CksL4%253D&md5=6a8656d083fc4d49fa8520b48e05d100</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2019.02.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2019.02.021%26sid%3Dliteratum%253Aachs%26aulast%3DRao%26aufirst%3DS.%26aulast%3DGurbani%26aufirst%3DD.%26aulast%3DDu%26aufirst%3DG.%26aulast%3DEverley%26aufirst%3DR.%2BA.%26aulast%3DBrowne%26aufirst%3DC.%2BM.%26aulast%3DChaikuad%26aufirst%3DA.%26aulast%3DTan%26aufirst%3DL.%26aulast%3DSchr%25C3%25B6der%26aufirst%3DM.%26aulast%3DGondi%26aufirst%3DS.%26aulast%3DFicarro%26aufirst%3DS.%2BB.%26atitle%3DLeveraging%2520compound%2520promiscuity%2520to%2520identify%2520targetable%2520cysteines%2520within%2520the%2520kinome%26jtitle%3DCell%2520chemical%2520biology%26date%3D2019%26volume%3D26%26spage%3D818%26epage%3D829.e9%26doi%3D10.1016%2Fj.chembiol.2019.02.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borrelli, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenwood, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Day, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farid, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harder, E.</span></span> <span> </span><span class="NLM_article-title">Docking covalent inhibitors: a parameter free approach to pose prediction and scoring</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">1932</span>– <span class="NLM_lpage">1940</span>, <span class="refDoi"> DOI: 10.1021/ci500118s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci500118s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BC2cXpsVKksLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=1932-1940&author=K.+Zhuauthor=K.+W.+Borrelliauthor=J.+R.+Greenwoodauthor=T.+Dayauthor=R.+Abelauthor=R.+S.+Faridauthor=E.+Harder&title=Docking+covalent+inhibitors%3A+a+parameter+free+approach+to+pose+prediction+and+scoring&doi=10.1021%2Fci500118s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Docking Covalent Inhibitors: A Parameter Free Approach To Pose Prediction and Scoring</span></div><div class="casAuthors">Zhu, Kai; Borrelli, Kenneth W.; Greenwood, Jeremy R.; Day, Tyler; Abel, Robert; Farid, Ramy S.; Harder, Edward</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1932-1940</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Although many popular docking programs include a facility to account for covalent ligands, large-scale systematic docking validation studies of covalent inhibitors have been sparse.  In this paper, we present the development and validation of a novel approach for docking and scoring covalent inhibitors, which consists of conventional noncovalent docking, heuristic formation of the covalent attachment point, and structural refinement of the protein-ligand complex.  This approach combines the strengths of the docking program Glide and the protein structure modeling program Prime and does not require any parameter fitting for the study of addnl. covalent reaction types.  We first test this method by predicting the native binding geometry of 38 covalently bound complexes.  The av. RMSD of the predicted poses is 1.52 Å, and 76% of test set inhibitors have an RMSD of less than 2.0 Å.  In addn., the apparent affinity score constructed herein is tested on a virtual screening study and the characterization of the SAR properties of two different series of congeneric compds. with satisfactory success.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbtjEb3EqzQ7Vg90H21EOLACvtfcHk0linI74jDaqjTA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXpsVKksLs%253D&md5=3633c29fa5dc29817b178fd10a4c7b15</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fci500118s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci500118s%26sid%3Dliteratum%253Aachs%26aulast%3DZhu%26aufirst%3DK.%26aulast%3DBorrelli%26aufirst%3DK.%2BW.%26aulast%3DGreenwood%26aufirst%3DJ.%2BR.%26aulast%3DDay%26aufirst%3DT.%26aulast%3DAbel%26aufirst%3DR.%26aulast%3DFarid%26aufirst%3DR.%2BS.%26aulast%3DHarder%26aufirst%3DE.%26atitle%3DDocking%2520covalent%2520inhibitors%253A%2520a%2520parameter%2520free%2520approach%2520to%2520pose%2520prediction%2520and%2520scoring%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D1932%26epage%3D1940%26doi%3D10.1021%2Fci500118s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clark, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hafner, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kouril, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muhlich, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pilarczyk, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niepel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sorger, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Medvedovic, M.</span></span> <span> </span><span class="NLM_article-title">GRcalculator: an online tool for calculating and mining dose–response data</span>. <i>BMC Cancer</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">698</span>, <span class="refDoi"> DOI: 10.1186/s12885-017-3689-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.1186%2Fs12885-017-3689-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=29065900" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVOgsr7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2017&pages=698&author=N.+A.+Clarkauthor=M.+Hafnerauthor=M.+Kourilauthor=E.+H.+Williamsauthor=J.+L.+Muhlichauthor=M.+Pilarczykauthor=M.+Niepelauthor=P.+K.+Sorgerauthor=M.+Medvedovic&title=GRcalculator%3A+an+online+tool+for+calculating+and+mining+dose%E2%80%93response+data&doi=10.1186%2Fs12885-017-3689-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">GRcalculator: an online tool for calculating and mining dose-response data</span></div><div class="casAuthors">Clark, Nicholas A.; Hafner, Marc; Kouril, Michal; Williams, Elizabeth H.; Muhlich, Jeremy L.; Pilarczyk, Marcin; Niepel, Mario; Sorger, Peter K.; Medvedovic, Mario</div><div class="citationInfo"><span class="NLM_cas:title">BMC Cancer</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">698/1-698/11</span>CODEN:
                <span class="NLM_cas:coden">BCMACL</span>;
        ISSN:<span class="NLM_cas:issn">1471-2407</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Background: Quantifying the response of cell lines to drugs or other perturbagens is the cornerstone of pre-clin. drug development and pharmacogenomics as well as a means to study factors that contribute to sensitivity and resistance.  In dividing cells, traditional metrics derived from dose-response curves such as IC50, AUC, and Emax, are confounded by the no. of cell divisions taking place during the assay, which varies widely for biol. and exptl. reasons.  Hafner et al. (Nat Meth 13:521.-+. 627, 2016) recently proposed an alternative way to quantify drug response, normalized growth rate (GR) inhibition, that is robust to such confounders.  Adoption of the GR method is expected to improve the reproducibility of dose - response assays and the reliability of pharmacogenomic assocns. (Hafner et al. 500-502, 2017).  Results: We describe here an interactive website for calcn., anal., and visualization of dose-response data using the GR approach and for comparison of GR and traditional metrics.  Data can be user-supplied or derived from published datasets.  The web tools are implemented in the form of three integrated Shiny applications (grcalculator, grbrowser, and grtutorial) deployed through a Shiny server.  Intuitive graphical user interfaces (GUIs) allow for interactive anal. and visualization of data.  The Shiny applications make use of two R packages (shinyLi and GRmetrics) specifically developed for this purpose.  The GRmetrics R package is also available via Bioconductor and can be used for offline data anal. and visualization.  Conclusions: GRcalculator is a powerful, user-friendly, and free tool to facilitate anal. of dose - response data.  It generates publication-ready figures and provides a unified platform for investigators to analyze dose - response data across diverse cell types and perturbagens (including drugs, biol. ligands, RNAi, etc.).  GRcalculator also provides access to data collected by the NIH LINCS Program and other public domain datasets.  The GRmetrics Bioconductor package provides computationally trained users with a platform for offline anal. of dose-response data and facilitates inclusion of GR metrics calcns. within existing R anal. pipelines.  These tools are therefore well suited to users in academia as well as industry.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpSN136VkelbrVg90H21EOLACvtfcHk0linI74jDaqjTA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVOgsr7N&md5=35b31d3eecee58f552392a4611bf655d</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1186%2Fs12885-017-3689-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12885-017-3689-3%26sid%3Dliteratum%253Aachs%26aulast%3DClark%26aufirst%3DN.%2BA.%26aulast%3DHafner%26aufirst%3DM.%26aulast%3DKouril%26aufirst%3DM.%26aulast%3DWilliams%26aufirst%3DE.%2BH.%26aulast%3DMuhlich%26aufirst%3DJ.%2BL.%26aulast%3DPilarczyk%26aufirst%3DM.%26aulast%3DNiepel%26aufirst%3DM.%26aulast%3DSorger%26aufirst%3DP.%2BK.%26aulast%3DMedvedovic%26aufirst%3DM.%26atitle%3DGRcalculator%253A%2520an%2520online%2520tool%2520for%2520calculating%2520and%2520mining%2520dose%25E2%2580%2593response%2520data%26jtitle%3DBMC%2520Cancer%26date%3D2017%26volume%3D17%26spage%3D698%26doi%3D10.1186%2Fs12885-017-3689-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sánchez-Bailón, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calcabrini, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gómez-Domínguez, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morte, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martín-Forero, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gómez-López, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molinari, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, K.-U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martín-Pérez, J.</span></span> <span> </span><span class="NLM_article-title">Src kinases catalytic activity regulates proliferation, migration and invasiveness of MDA-MB-231 breast cancer cells</span>. <i>Cell. Signalling</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">1276</span>– <span class="NLM_lpage">1286</span>, <span class="refDoi"> DOI: 10.1016/j.cellsig.2012.02.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.1016%2Fj.cellsig.2012.02.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=22570868" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BC38XkvF2itrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2012&pages=1276-1286&author=M.+P.+S%C3%A1nchez-Bail%C3%B3nauthor=A.+Calcabriniauthor=D.+G%C3%B3mez-Dom%C3%ADnguezauthor=B.+Morteauthor=E.+Mart%C3%ADn-Foreroauthor=G.+G%C3%B3mez-L%C3%B3pezauthor=A.+Molinariauthor=K.-U.+Wagnerauthor=J.+Mart%C3%ADn-P%C3%A9rez&title=Src+kinases+catalytic+activity+regulates+proliferation%2C+migration+and+invasiveness+of+MDA-MB-231+breast+cancer+cells&doi=10.1016%2Fj.cellsig.2012.02.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Src kinases catalytic activity regulates proliferation, migration and invasiveness of MDA-MB-231 breast cancer cells</span></div><div class="casAuthors">Sanchez-Bailon, Maria Pilar; Calcabrini, Annarica; Gomez-Dominguez, Daniel; Morte, Beatriz; Martin-Forero, Esther; Gomez-Lopez, Gonzalo; Molinari, Agnese; Wagner, Kay-Uwe; Martin-Perez, Jorge</div><div class="citationInfo"><span class="NLM_cas:title">Cellular Signalling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1276-1286</span>CODEN:
                <span class="NLM_cas:coden">CESIEY</span>;
        ISSN:<span class="NLM_cas:issn">0898-6568</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">SFKs are frequently deregulated in cancer where they control cellular proliferation, migration, survival and metastasis.  Here we study the role of SFKs catalytic activity in triple-neg./basal-like and metastatic human breast cancer MDA-MB-231 cells employing three well-established inhibitors: Dasatinib, PP2 and SU6656.  These compds. inhibited migration and invasion.  Concomitantly, they reduced Fak, paxillin, p130CAS, caveolin-1 phosphorylation and altered cytoskeletal structures.  They also inhibited cell proliferation, but in different manners.  Dasatinib and PP2 increased p27Kip1 expression and reduced c-Myc levels, restraining G1-S transition.  In contrast, SU6656 did not modify p27Kip1 expression, slightly altered c-Myc levels and generated polyploid multinucleated cells, indicating inhibition of cytokinesis.  These later effects were also obsd. in SYF fibroblasts, suggesting a SFKs-independent action.  ZM447439, an Aurora B kinase inhibitor, produced similar cell cycle and morphol. alterations in MDA-MB-231 cells, indicating that SU6656 blocked Aurora B kinase.  This was confirmed by inhibition of histone H3 phosphorylation, the canonical Aurora B kinase substrate.  Furthermore, hierarchical clustering anal. of gene expression profiles showed that SU6656 defined a set of genes that differed from Dasatinib and PP2.  Addnl., Gene Set Enrichment Analyses revealed that SU6656 significantly reduces the Src pathway.  Together, these results show the importance of SFKs catalytic activity for MDA-MB-231 proliferation, migration and invasiveness.  They also illustrate that SU6656 acts as dual SFKs and Aurora B kinase inhibitor, suggesting its possible use as a therapeutic agent in breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxhZ62Ycaq7bVg90H21EOLACvtfcHk0ljr-k7gRtthzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkvF2itrw%253D&md5=527ed67289deb9acc1ab9a6fe15b527d</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2Fj.cellsig.2012.02.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cellsig.2012.02.011%26sid%3Dliteratum%253Aachs%26aulast%3DS%25C3%25A1nchez-Bail%25C3%25B3n%26aufirst%3DM.%2BP.%26aulast%3DCalcabrini%26aufirst%3DA.%26aulast%3DG%25C3%25B3mez-Dom%25C3%25ADnguez%26aufirst%3DD.%26aulast%3DMorte%26aufirst%3DB.%26aulast%3DMart%25C3%25ADn-Forero%26aufirst%3DE.%26aulast%3DG%25C3%25B3mez-L%25C3%25B3pez%26aufirst%3DG.%26aulast%3DMolinari%26aufirst%3DA.%26aulast%3DWagner%26aufirst%3DK.-U.%26aulast%3DMart%25C3%25ADn-P%25C3%25A9rez%26aufirst%3DJ.%26atitle%3DSrc%2520kinases%2520catalytic%2520activity%2520regulates%2520proliferation%252C%2520migration%2520and%2520invasiveness%2520of%2520MDA-MB-231%2520breast%2520cancer%2520cells%26jtitle%3DCell.%2520Signalling%26date%3D2012%26volume%3D24%26spage%3D1276%26epage%3D1286%26doi%3D10.1016%2Fj.cellsig.2012.02.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Formisano, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Amato, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Servetto, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brillante, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raimondo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Mauro, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marciano, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orsini, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cosconati, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Randazzo, A.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Src inhibitors act through different mechanisms in Non-Small Cell Lung Cancer models depending on EGFR and RAS mutational status</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">26090</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.4636</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.18632%2Foncotarget.4636" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=26325669" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A280%3ADC%252BC287otVejsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=26090&author=L.+Formisanoauthor=V.+D%E2%80%99Amatoauthor=A.+Servettoauthor=S.+Brillanteauthor=L.+Raimondoauthor=C.+Di+Mauroauthor=R.+Marcianoauthor=R.+C.+Orsiniauthor=S.+Cosconatiauthor=A.+Randazzo&title=Src+inhibitors+act+through+different+mechanisms+in+Non-Small+Cell+Lung+Cancer+models+depending+on+EGFR+and+RAS+mutational+status&doi=10.18632%2Foncotarget.4636"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Src inhibitors act through different mechanisms in Non-Small Cell Lung Cancer models depending on EGFR and RAS mutational status</span></div><div class="casAuthors">Formisano Luigi; D'Amato Valentina; Servetto Alberto; Brillante Simona; Raimondo Lucia; Di Mauro Concetta; Marciano Roberta; Orsini Roberta Clara; De Placido Sabino; Rosa Roberta; Bianco Roberto; Cosconati Sandro; Randazzo Antonio; Parsons Sarah J; Montuori Nunzia; Veneziani Bianca Maria</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">28</span>),
    <span class="NLM_cas:pages">26090-103</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Resistance to the EGFR tyrosine kinase inhibitors (TKIs) gefitinib and erlotinib, often related to Ras or secondary EGFR mutations, is a relevant clinical issue in Non-Small Cell Lung Cancer (NSCLC).  Although Src TK has been involved in such resistance, clinical development of its inhibitors has been so far limited.  To better define the molecular targets of the Src TKIs saracatinib, dasatinib and bosutinib, we used a variety of in vitro/in vivo studies.  Kinase assays supported by docking analysis demonstrated that all the compounds directly inhibit EGFR TK variants.  However, in live cells only saracatinib efficiently reduced EGFR activation, while dasatinib was the most effective agent in inhibiting Src TK.  Consistently, a pronounced anti-proliferative effect was achieved with saracatinib, in EGFR mutant cells, or with dasatinib, in wt EGFR/Ras mutant cells, poorly dependent on EGFR and erlotinib-resistant.  We then identified the most effective drug combinations to overcome resistance to EGFR inhibitors, both in vitro and in nude mice: in T790M EGFR erlotinib-resistant cells, saracatinib with the anti-EGFR mAb cetuximab; in Ras mutant erlotinib-resistant models, dasatinib with the MEK inhibitor selumetinib.  Src inhibitors may act with different mechanisms in NSCLCs, depending on EGFR/Ras mutational profile, and may be integrated with EGFR or MEK inhibitors for different cohorts of NSCLCs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTCDsEQcCnxNdFxYDaSSP8XfW6udTcc2eZlp-4tYNXerbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC287otVejsg%253D%253D&md5=45a98a239abd2001198e5e6791efe32f</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.4636&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.4636%26sid%3Dliteratum%253Aachs%26aulast%3DFormisano%26aufirst%3DL.%26aulast%3DD%25E2%2580%2599Amato%26aufirst%3DV.%26aulast%3DServetto%26aufirst%3DA.%26aulast%3DBrillante%26aufirst%3DS.%26aulast%3DRaimondo%26aufirst%3DL.%26aulast%3DDi%2BMauro%26aufirst%3DC.%26aulast%3DMarciano%26aufirst%3DR.%26aulast%3DOrsini%26aufirst%3DR.%2BC.%26aulast%3DCosconati%26aufirst%3DS.%26aulast%3DRandazzo%26aufirst%3DA.%26atitle%3DSrc%2520inhibitors%2520act%2520through%2520different%2520mechanisms%2520in%2520Non-Small%2520Cell%2520Lung%2520Cancer%2520models%2520depending%2520on%2520EGFR%2520and%2520RAS%2520mutational%2520status%26jtitle%3DOncotarget%26date%3D2015%26volume%3D6%26spage%3D26090%26doi%3D10.18632%2Foncotarget.4636" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Giaccone, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zucali, P.</span></span> <span> </span><span class="NLM_article-title">Src as a potential therapeutic target in non-small-cell lung cancer</span>. <i>Annals of Oncology</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">1219</span>– <span class="NLM_lpage">1223</span>, <span class="refDoi"> DOI: 10.1093/annonc/mdn048</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.1093%2Fannonc%2Fmdn048" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=18388349" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A280%3ADC%252BD1cvgvVyrsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2008&pages=1219-1223&author=G.+Giacconeauthor=P.+Zucali&title=Src+as+a+potential+therapeutic+target+in+non-small-cell+lung+cancer&doi=10.1093%2Fannonc%2Fmdn048"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Src as a potential therapeutic target in non-small-cell lung cancer</span></div><div class="casAuthors">Giaccone G; Zucali P A</div><div class="citationInfo"><span class="NLM_cas:title">Annals of oncology : official journal of the European Society for Medical Oncology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1219-1223</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Lung cancer is the most common cause of cancer-related death, with non-small-cell lung cancer (NSCLC) accounting for 80%-85% of all cases.  Although survival rates are reasonably good for patients diagnosed with very early disease, the majority of patients present with advanced disease.  For these patients, palliation and improvements in quality of life are the primary goals of therapy.  Although chemotherapeutic agents remain the cornerstone of first-line therapy, these agents have limited use in patients who have relapsed and have metastatic disease.  Therefore, new strategies are required to improve survival and quality of life in this setting.  With the substantial advances in our understanding of tumour biology, it has been possible to identify signalling pathways involved in mediating tumour growth and progression.  These pathways offer targets for new biological agents such as small molecule inhibitors and monoclonal antibodies.  One such target is Src, a tyrosine kinase that is involved in multiple aspects of tumorigenesis including proliferation, migration and angiogenesis.  Increased levels of Src expression have been found in a range of cancers, especially breast, colorectal, prostate and lung.  Preliminary preclinical data and pharmacodynamic data suggest that Src inhibition is a viable therapeutic option in the treatment of advanced NSCLC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSDd4hYJocBQtj_Vg2vObihfW6udTcc2eZlp-4tYNXerbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1cvgvVyrsw%253D%253D&md5=9e95b141cb44a1999b0625e34f30861a</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1093%2Fannonc%2Fmdn048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fannonc%252Fmdn048%26sid%3Dliteratum%253Aachs%26aulast%3DGiaccone%26aufirst%3DG.%26aulast%3DZucali%26aufirst%3DP.%26atitle%3DSrc%2520as%2520a%2520potential%2520therapeutic%2520target%2520in%2520non-small-cell%2520lung%2520cancer%26jtitle%3DAnnals%2520of%2520Oncology%26date%3D2008%26volume%3D19%26spage%3D1219%26epage%3D1223%26doi%3D10.1093%2Fannonc%2Fmdn048" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Green, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fennell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whittaker, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curwen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobs, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Logie, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hargreaves, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hickinson, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilkinson, R. W.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Preclinical anticancer activity of the potent, oral Src inhibitor AZD0530</span>. <i>Mol. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">248</span>– <span class="NLM_lpage">261</span>, <span class="refDoi"> DOI: 10.1016/j.molonc.2009.01.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.1016%2Fj.molonc.2009.01.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=19393585" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjslKnsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2009&pages=248-261&author=T.+P.+Greenauthor=M.+Fennellauthor=R.+Whittakerauthor=J.+Curwenauthor=V.+Jacobsauthor=J.+Allenauthor=A.+Logieauthor=J.+Hargreavesauthor=D.+M.+Hickinsonauthor=R.+W.+Wilkinson&title=Preclinical+anticancer+activity+of+the+potent%2C+oral+Src+inhibitor+AZD0530&doi=10.1016%2Fj.molonc.2009.01.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Preclinical anticancer activity of the potent, oral Src inhibitor AZD0530</span></div><div class="casAuthors">Green, Tim P.; Fennell, Mike; Whittaker, Robin; Curwen, Jon; Jacobs, Vivien; Allen, Jack; Logie, Armelle; Hargreaves, Judith; Hickinson, D. Mark; Wilkinson, Robert W.; Elvin, Paul; Boyer, Brigitte; Carragher, Neil; Ple, Patrick A.; Bermingham, Alun; Holdgate, Geoffrey A.; Ward, Walter H. J.; Hennequin, Laurent F.; Davies, Barry R.; Costello, Gerard F.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Oncology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">248-261</span>CODEN:
                <span class="NLM_cas:coden">MOONC3</span>;
        ISSN:<span class="NLM_cas:issn">1574-7891</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">AZD0530, an orally available Src inhibitor, demonstrated potent antimigratory and anti-invasive effects in vitro, and inhibited metastasis in a murine model of bladder cancer.  Antiproliferative activity of AZD0530 in vitro varied between cell lines (IC50 0.2 -> 10 μM).  AZD0530 inhibited tumor growth in 4/10 xenograft models tested and dynamically inhibited in vivo phosphorylation of Src substrates paxillin and FAK in both growth-inhibition-resistant and -sensitive xenografts.  The activity of AZD0530 in NBT-II bladder cancer cells in vitro was consistent with inhibition of cell migration and stabilization of cell-cell adhesion.  These data suggest a dominant anti-invasive pharmacol. for AZD0530 that may limit tumor progression in a range of cancers.  AZD0530 is currently in Phase II clin. trials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAbzKlSDQyObVg90H21EOLACvtfcHk0lgbIQOhQrO5Rg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjslKnsL0%253D&md5=4c2ec94fde22bbec401009e106bb6caf</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2Fj.molonc.2009.01.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molonc.2009.01.002%26sid%3Dliteratum%253Aachs%26aulast%3DGreen%26aufirst%3DT.%2BP.%26aulast%3DFennell%26aufirst%3DM.%26aulast%3DWhittaker%26aufirst%3DR.%26aulast%3DCurwen%26aufirst%3DJ.%26aulast%3DJacobs%26aufirst%3DV.%26aulast%3DAllen%26aufirst%3DJ.%26aulast%3DLogie%26aufirst%3DA.%26aulast%3DHargreaves%26aufirst%3DJ.%26aulast%3DHickinson%26aufirst%3DD.%2BM.%26aulast%3DWilkinson%26aufirst%3DR.%2BW.%26atitle%3DPreclinical%2520anticancer%2520activity%2520of%2520the%2520potent%252C%2520oral%2520Src%2520inhibitor%2520AZD0530%26jtitle%3DMol.%2520Oncol.%26date%3D2009%26volume%3D3%26spage%3D248%26epage%3D261%26doi%3D10.1016%2Fj.molonc.2009.01.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hafner, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niepel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sorger, P. K.</span></span> <span> </span><span class="NLM_article-title">Growth rate inhibition metrics correct for confounders in measuring sensitivity to cancer drugs</span>. <i>Nat. Methods</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">521</span>, <span class="refDoi"> DOI: 10.1038/nmeth.3853</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.1038%2Fnmeth.3853" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=27135972" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BC28XntVCltr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2016&pages=521&author=M.+Hafnerauthor=M.+Niepelauthor=M.+Chungauthor=P.+K.+Sorger&title=Growth+rate+inhibition+metrics+correct+for+confounders+in+measuring+sensitivity+to+cancer+drugs&doi=10.1038%2Fnmeth.3853"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Growth rate inhibition metrics correct for confounders in measuring sensitivity to cancer drugs</span></div><div class="casAuthors">Hafner, Marc; Niepel, Mario; Chung, Mirra; Sorger, Peter K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Methods</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">521-527</span>CODEN:
                <span class="NLM_cas:coden">NMAEA3</span>;
        ISSN:<span class="NLM_cas:issn">1548-7091</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Drug sensitivity and resistance are conventionally quantified by IC50 or Emax values, but these metrics are highly sensitive to the no. of divisions taking place over the course of a response assay.  The dependency of IC50 and Emax on division rate creates artifactual correlations between genotype and drug sensitivity, while obscuring valuable biol. insights and interfering with biomarker discovery.  We derive alternative small mol. drug-response metrics that are insensitive to division no.  These are based on estn. of the magnitude of drug-induced growth rate inhibition (GR) using endpoint or time-course assays.  We show that GR50 and GRmax are superior to conventional metrics for assessing the effects of small mol. drugs in dividing cells.  Moreover, adopting GR metrics requires only modest changes in exptl. protocols.  We expect GR metrics to improve the study of cell signaling and growth using small mols. and biologics and to facilitate the discovery of drug-response biomarkers and the identification of drugs effective against specific patient-derived tumor cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5Byu3P4LAobVg90H21EOLACvtfcHk0lgbIQOhQrO5Rg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XntVCltr0%253D&md5=7383d0009d14697d6fb3e9a2de0f6da0</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1038%2Fnmeth.3853&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnmeth.3853%26sid%3Dliteratum%253Aachs%26aulast%3DHafner%26aufirst%3DM.%26aulast%3DNiepel%26aufirst%3DM.%26aulast%3DChung%26aufirst%3DM.%26aulast%3DSorger%26aufirst%3DP.%2BK.%26atitle%3DGrowth%2520rate%2520inhibition%2520metrics%2520correct%2520for%2520confounders%2520in%2520measuring%2520sensitivity%2520to%2520cancer%2520drugs%26jtitle%3DNat.%2520Methods%26date%3D2016%26volume%3D13%26spage%3D521%26doi%3D10.1038%2Fnmeth.3853" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalyankrishna, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wislez, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thilaganathan, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saigal, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wistuba, I. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurie, J. M.</span></span> <span> </span><span class="NLM_article-title">SRC-family kinases are activated in non-small cell lung cancer and promote the survival of epidermal growth factor receptor-dependent cell lines</span>. <i>Am. J. Pathol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>170</i></span>,  <span class="NLM_fpage">366</span>– <span class="NLM_lpage">376</span>, <span class="refDoi"> DOI: 10.2353/ajpath.2007.060706</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.2353%2Fajpath.2007.060706" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=17200208" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A280%3ADC%252BD2s%252FgtlOnsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=170&publication_year=2007&pages=366-376&author=J.+Zhangauthor=S.+Kalyankrishnaauthor=M.+Wislezauthor=N.+Thilaganathanauthor=B.+Saigalauthor=W.+Weiauthor=L.+Maauthor=I.+I.+Wistubaauthor=F.+M.+Johnsonauthor=J.+M.+Kurie&title=SRC-family+kinases+are+activated+in+non-small+cell+lung+cancer+and+promote+the+survival+of+epidermal+growth+factor+receptor-dependent+cell+lines&doi=10.2353%2Fajpath.2007.060706"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">SRC-family kinases are activated in non-small cell lung cancer and promote the survival of epidermal growth factor receptor-dependent cell lines</span></div><div class="casAuthors">Zhang Jie; Kalyankrishna Shailaja; Wislez Marie; Thilaganathan Nishan; Saigal Babita; Wei Wei; Ma Long; Wistuba Ignacio I; Johnson Faye M; Kurie Jonathan M</div><div class="citationInfo"><span class="NLM_cas:title">The American journal of pathology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">170</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">366-76</span>
        ISSN:<span class="NLM_cas:issn">0002-9440</span>.
    </div><div class="casAbstract">The role of Src-family kinases (SFKs) in non-small cell lung cancer (NSCLC) has not been fully defined.  Here we addressed this question by examining SFK phosphorylation in NSCLC biopsy samples and using genetic and pharmacological approaches to inhibit SFK expression and activity in cultured NSCLC cells.  Immunohistochemical analysis of NSCLC biopsy samples using a Tyr416 phosphorylation-specific, pan-SFK antibody revealed staining in 123 (33%) of 370 tumors.  Because c-Src is known to be both an upstream activator and downstream mediator of epidermal growth factor receptor (EGFR), we next investigated SFK phosphorylation in a panel of NSCLC cell lines, including ones that depend on EGFR for survival.  The EGFR-dependent NSCLC cell lines HCC827 and H3255 had increased phosphorylation of SFKs, and treatment of these cells with an SFK inhibitor (PP1 or SKI-606) induced apoptosis.  PP1 decreased phosphorylation of EGFR, ErbB2, and ErbB3 and strikingly enhanced apoptosis by gefitinib, an EGFR inhibitor.  HCC827 cells transfected with c-Src short hairpin RNA exhibited diminished phosphorylation of EGFR and ErbB2 and decreased sensitivity to apoptosis by PP1 or gefitinib.  We conclude that SFKs are activated in NSCLC biopsy samples, promote the survival of EGFR-dependent NSCLC cells, and should be investigated as therapeutic targets in NSCLC patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQZ2f3qQMjhE0kUaC_kzhvyfW6udTcc2eZ5gA-LMsS1erntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2s%252FgtlOnsA%253D%253D&md5=391c770e7368377b4b9f5576a206d8d0</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.2353%2Fajpath.2007.060706&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2353%252Fajpath.2007.060706%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DKalyankrishna%26aufirst%3DS.%26aulast%3DWislez%26aufirst%3DM.%26aulast%3DThilaganathan%26aufirst%3DN.%26aulast%3DSaigal%26aufirst%3DB.%26aulast%3DWei%26aufirst%3DW.%26aulast%3DMa%26aufirst%3DL.%26aulast%3DWistuba%26aufirst%3DI.%2BI.%26aulast%3DJohnson%26aufirst%3DF.%2BM.%26aulast%3DKurie%26aufirst%3DJ.%2BM.%26atitle%3DSRC-family%2520kinases%2520are%2520activated%2520in%2520non-small%2520cell%2520lung%2520cancer%2520and%2520promote%2520the%2520survival%2520of%2520epidermal%2520growth%2520factor%2520receptor-dependent%2520cell%2520lines%26jtitle%3DAm.%2520J.%2520Pathol.%26date%3D2007%26volume%3D170%26spage%3D366%26epage%3D376%26doi%3D10.2353%2Fajpath.2007.060706" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ficarro, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Browne, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Card, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alexander, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNally, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhe-Paganon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seo, H.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamberto, I.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Leveraging gas-phase fragmentation pathways for improved identification and selective detection of targets modified by covalent probes</span>. <i>Anal. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>88</i></span>,  <span class="NLM_fpage">12248</span>– <span class="NLM_lpage">12254</span>, <span class="refDoi"> DOI: 10.1021/acs.analchem.6b03394</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.analchem.6b03394" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFGisrnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=88&publication_year=2016&pages=12248-12254&author=S.+B.+Ficarroauthor=C.+M.+Browneauthor=J.+D.+Cardauthor=W.+M.+Alexanderauthor=T.+Zhangauthor=E.+Parkauthor=R.+McNallyauthor=S.+Dhe-Paganonauthor=H.-S.+Seoauthor=I.+Lamberto&title=Leveraging+gas-phase+fragmentation+pathways+for+improved+identification+and+selective+detection+of+targets+modified+by+covalent+probes&doi=10.1021%2Facs.analchem.6b03394"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Leveraging Gas-Phase Fragmentation Pathways for Improved Identification and Selective Detection of Targets Modified by Covalent Probes</span></div><div class="casAuthors">Ficarro, Scott B.; Browne, Christopher M.; Card, Joseph D.; Alexander, William M.; Zhang, Tinghu; Park, Eunyoung; McNally, Randall; Dhe-Paganon, Sirano; Seo, Hyuk-Soo; Lamberto, Ilaria; Eck, Michael J.; Buhrlage, Sara J.; Gray, Nathanael S.; Marto, Jarrod A.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Chemistry (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">88</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">12248-12254</span>CODEN:
                <span class="NLM_cas:coden">ANCHAM</span>;
        ISSN:<span class="NLM_cas:issn">0003-2700</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The recent approval of covalent inhibitors for multiple clin. indications has reignited enthusiasm for this class of drugs.  As interest in covalent drugs has increased, so too has the need for anal. platforms that can leverage their mechanism-of-action to characterize modified protein targets.  Here the authors describe novel gas phase dissocn. pathways which yield predictable fragment ions during MS/MS of inhibitor-modified peptides.  The authors find that these dissocn. pathways are common to numerous cysteine-directed probes as well as the covalent drugs, Ibrutinib and Neratinib.  The authors leverage the predictable nature of these fragment ions to improve the confidence of peptide sequence assignment in proteomic analyses, and explore their potential use in selective mass spectrometry-based assays.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrNu24Jo7-IWLVg90H21EOLACvtfcHk0lg4XlehVmlUpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFGisrnO&md5=7932392d2d2ac88def4cacecbda41a30</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Facs.analchem.6b03394&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.analchem.6b03394%26sid%3Dliteratum%253Aachs%26aulast%3DFicarro%26aufirst%3DS.%2BB.%26aulast%3DBrowne%26aufirst%3DC.%2BM.%26aulast%3DCard%26aufirst%3DJ.%2BD.%26aulast%3DAlexander%26aufirst%3DW.%2BM.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DPark%26aufirst%3DE.%26aulast%3DMcNally%26aufirst%3DR.%26aulast%3DDhe-Paganon%26aufirst%3DS.%26aulast%3DSeo%26aufirst%3DH.-S.%26aulast%3DLamberto%26aufirst%3DI.%26atitle%3DLeveraging%2520gas-phase%2520fragmentation%2520pathways%2520for%2520improved%2520identification%2520and%2520selective%2520detection%2520of%2520targets%2520modified%2520by%2520covalent%2520probes%26jtitle%3DAnal.%2520Chem.%26date%3D2016%26volume%3D88%26spage%3D12248%26epage%3D12254%26doi%3D10.1021%2Facs.analchem.6b03394" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Papaetis, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Syrigos, K. N.</span></span> <span> </span><span class="NLM_article-title">Sunitinib</span>. <i>BioDrugs</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">377</span>– <span class="NLM_lpage">389</span>, <span class="refDoi"> DOI: 10.2165/11318860-000000000-00000</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.2165%2F11318860-000000000-00000" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=19894779" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtF2qs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2009&pages=377-389&author=G.+S.+Papaetisauthor=K.+N.+Syrigos&title=Sunitinib&doi=10.2165%2F11318860-000000000-00000"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Sunitinib: a multitargeted receptor tyrosine kinase inhibitor in the era of molecular cancer therapies</span></div><div class="casAuthors">Papaetis, Georgios S.; Syrigos, Kostas N.</div><div class="citationInfo"><span class="NLM_cas:title">BioDrugs</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">377-389</span>CODEN:
                <span class="NLM_cas:coden">BIDRF4</span>;
        ISSN:<span class="NLM_cas:issn">1173-8804</span>.
    
            (<span class="NLM_cas:orgname">Adis Data Information BV</span>)
        </div><div class="casAbstract">A review.  Sunitinib is an oral oxindole multitargeted kinase inhibitor that inhibits certain receptor tyrosine kinases (RTKs).  These include vascular endothelial growth factor receptors (VEGFR type 1 and 2), platelet-derived growth factor receptors (PDGFR-α and PDGFR-β), stem cell factor receptor (KIT), FMS-like tyrosine kinase-3 (FLT3), glial cell-line derived neurotrophic factor receptor (RET) and the receptor of macrophage-colony stimulating factor (CSF1R).  Examn. of the antitumor effect of sunitinib in a variety of cell lines in vitro suggested an antiproliferative activity that is dependent on the presence of constitutively active RTK targets.  The use of sunitinib as first-line therapy in advanced renal cell carcinoma (RCC) has improved the overall survival compared with that obsd. after cytokine therapy, while its administration in patients with gastrointestinal stromal tumors (GISTs) after progression or intolerance to imatinib achieved an objective response of 7%.  Sunitinib is currently approved for the treatment of GISTs in this setting, and as first-line therapy for the treatment of advanced RCC.  The relatively long half-life of sunitinib and its major metabolite allow for a once-daily dosing schedule.  An interesting antitumor activity of sunitinib was reported in phase II studies of patients with a variety of malignancies, such as hepatocellular cancer, pancreatic neuroendocrine tumors, and non-small cell lung cancer; results of phase III studies are urgently anticipated.  Fatigue is one of the most common adverse effects of sunitinib, as 50-70% of patients with advanced RCC and GIST complained of this adverse effect.  Other adverse effects are diarrhea, anorexia, nausea and vomiting, oral changes and bleeding events.  Most toxicities are reversible and should not result in discontinuation of sunitinib.  If necessary, dose adjustments or interruptions should be made.  Hypothyroidism has been described in the first 2 wk of sunitinib therapy and its incidence increases progressively with the duration of therapy.  Sunitinib may exert its hypertensive activity through a direct effect on the vasculature, while its most important cardiac adverse effect is left ventricular dysfunction.  A variety of skin adverse effects have been described with the use of sunitinib such as hand-foot syndrome, yellow discoloration of the skin, dry skin, subungual splinter hemorrhages, acral erythema, and generalized skin rashes.  Administration of sunitinib in the adjuvant and neoadjuvant setting of patients with RCC and of its combination with chemotherapy and other targeted therapies are currently under intense investigation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYS25sw5p707Vg90H21EOLACvtfcHk0lg4XlehVmlUpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtF2qs7g%253D&md5=dc5d34a3f2f6fc7cc2a013517c3195ab</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.2165%2F11318860-000000000-00000&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2165%252F11318860-000000000-00000%26sid%3Dliteratum%253Aachs%26aulast%3DPapaetis%26aufirst%3DG.%2BS.%26aulast%3DSyrigos%26aufirst%3DK.%2BN.%26atitle%3DSunitinib%26jtitle%3DBioDrugs%26date%3D2009%26volume%3D23%26spage%3D377%26epage%3D389%26doi%3D10.2165%2F11318860-000000000-00000" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wilhelm, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lynch, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lowinger, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dumas, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simantov, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelley, S.</span></span> <span> </span><span class="NLM_article-title">Discovery and development of sorafenib: a multikinase inhibitor for treating cancer</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">835</span>, <span class="refDoi"> DOI: 10.1038/nrd2130</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.1038%2Fnrd2130" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=17016424" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtVajsbbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2006&pages=835&author=S.+Wilhelmauthor=C.+Carterauthor=M.+Lynchauthor=T.+Lowingerauthor=J.+Dumasauthor=R.+A.+Smithauthor=B.+Schwartzauthor=R.+Simantovauthor=S.+Kelley&title=Discovery+and+development+of+sorafenib%3A+a+multikinase+inhibitor+for+treating+cancer&doi=10.1038%2Fnrd2130"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and development of sorafenib: a multikinase inhibitor for treating cancer</span></div><div class="casAuthors">Wilhelm, Scott; Carter, Christopher; Lynch, Mark; Lowinger, Timothy; Dumas, Jacques; Smith, Roger A.; Schwartz, Brian; Simantov, Ronit; Kelley, Susan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">835-844</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Since the mol. revolution of the 1980s, knowledge of the etiol. of cancer has increased considerably, which has led to the discovery and development of targeted therapies tailored to inhibit cancer-specific pathways.  The introduction and refinement of rapid, high-throughput screening technologies over the past decade has greatly facilitated this targeted discovery and development process.  Here, the authors describe the discovery and continuing development of sorafenib (previously known as BAY 43-9006), the first oral multikinase inhibitor that targets Raf and affects tumor signaling and the tumor vasculature.  The discovery cycle of sorafenib (Nexavar; Bayer Pharmaceuticals) - from initial screening for a lead compd. to FDA approval for the treatment of advanced renal cell carcinoma in Dec. 2005 - was completed in just 11 years, with approval being received ∼5 years after the initiation of the first Phase I trial.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6WjjfgNSCtbVg90H21EOLACvtfcHk0liFx6xe5GzMcw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtVajsbbM&md5=490d4995170f3a7270ae9bbdd24b12e5</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1038%2Fnrd2130&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2130%26sid%3Dliteratum%253Aachs%26aulast%3DWilhelm%26aufirst%3DS.%26aulast%3DCarter%26aufirst%3DC.%26aulast%3DLynch%26aufirst%3DM.%26aulast%3DLowinger%26aufirst%3DT.%26aulast%3DDumas%26aufirst%3DJ.%26aulast%3DSmith%26aufirst%3DR.%2BA.%26aulast%3DSchwartz%26aufirst%3DB.%26aulast%3DSimantov%26aufirst%3DR.%26aulast%3DKelley%26aufirst%3DS.%26atitle%3DDiscovery%2520and%2520development%2520of%2520sorafenib%253A%2520a%2520multikinase%2520inhibitor%2520for%2520treating%2520cancer%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2006%26volume%3D5%26spage%3D835%26doi%3D10.1038%2Fnrd2130" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J. Y.</span></span> <span> </span><span class="NLM_article-title">Exploiting the promiscuity of imatinib</span>. <i>J. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">30</span>, <span class="refDoi"> DOI: 10.1186/jbiol134</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.1186%2Fjbiol134" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=19435483" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A280%3ADC%252BD1Mzos1eltA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2009&pages=30&author=S.+J.+Leeauthor=J.+Y.+Wang&title=Exploiting+the+promiscuity+of+imatinib&doi=10.1186%2Fjbiol134"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Exploiting the promiscuity of imatinib</span></div><div class="casAuthors">Lee Shun J; Wang Jean Y J</div><div class="citationInfo"><span class="NLM_cas:title">Journal of biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">30</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The protein kinase inhibitor imatinib, also known as Gleevec, has been a notable success in treating chronic myelogenous leukemia.  A recent paper in BMC Structural Biology reports a 1.75 A crystal structure of imatinib bound to the oxidoreductase NQO2 and reveals insights into the binding specificity and the off-target effects of the inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQvIsbZZF5A8n6CEoQgWY10fW6udTcc2ebZZig52snot7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1Mzos1eltA%253D%253D&md5=13e34c684303d3976d50727e3f70c5d2</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1186%2Fjbiol134&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fjbiol134%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DS.%2BJ.%26aulast%3DWang%26aufirst%3DJ.%2BY.%26atitle%3DExploiting%2520the%2520promiscuity%2520of%2520imatinib%26jtitle%3DJ.%2520Biol.%26date%3D2009%26volume%3D8%26spage%3D30%26doi%3D10.1186%2Fjbiol134" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hafner, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Subramanian, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sorger, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">A multi-targeted probe-based strategy to identify signaling vulnerabilities in cancers</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>294</i></span>,  <span class="NLM_fpage">8664</span>, <span class="refDoi"> DOI: 10.1074/jbc.RA118.006805</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=10.1074%2Fjbc.RA118.006805" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=30858179" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtFOnurnJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=294&publication_year=2019&pages=8664&author=S.+Raoauthor=G.+Duauthor=M.+Hafnerauthor=K.+Subramanianauthor=P.+K.+Sorgerauthor=N.+S.+Gray&title=A+multi-targeted+probe-based+strategy+to+identify+signaling+vulnerabilities+in+cancers&doi=10.1074%2Fjbc.RA118.006805"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">A multitargeted probe-based strategy to identify signaling vulnerabilities in cancers</span></div><div class="casAuthors">Rao, Suman; Du, Guangyan; Hafner, Marc; Subramanian, Kartik; Sorger, Peter K.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">294</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">8664-8673</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Most cancer cells are dependent on a network of deregulated signaling pathways for survival and are insensitive, or rapidly evolve resistance, to selective inhibitors aimed at a single target.  For these reasons, drugs that target more than one protein (polypharmacol.) can be clin. advantageous.  The discovery of useful polypharmacol. remains serendipitous and is challenging to characterize and validate.  In this study, we developed a non-genetic strategy for the identification of pathways that drive cancer cell proliferation and represent exploitable signaling vulnerabilities.  Our approach is based on using a multitargeted kinase inhibitor, SM1-71, as a tool compd. to identify combinations of targets whose simultaneous inhibition elicits a potent cytotoxic effect.  As a proof of concept, we applied this approach to a KRAS-dependent non-small cell lung cancer (NSCLC) cell line, H23-KRASG12C.  Using a combination of phenotypic screens, signaling analyses, and kinase inhibitors, we found that dual inhibition of MEK1/2 and insulin-like growth factor 1 receptor (IGF1R)/insulin receptor (INSR) is crit. for blocking proliferation in cells.  Our work supports the value of multitargeted tool compds. with well-validated polypharmacol. and target space as tools to discover kinase dependences in cancer.  We propose that the strategy described here is complementary to existing genetics-based approaches, generalizable to other systems, and enabling for future mechanistic and translational studies of polypharmacol. in the context of signaling vulnerabilities in cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAvW-66Pk4brVg90H21EOLACvtfcHk0liFx6xe5GzMcw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtFOnurnJ&md5=694fd4cb9adb5fb68d2600a5c608b27a</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1074%2Fjbc.RA118.006805&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.RA118.006805%26sid%3Dliteratum%253Aachs%26aulast%3DRao%26aufirst%3DS.%26aulast%3DDu%26aufirst%3DG.%26aulast%3DHafner%26aufirst%3DM.%26aulast%3DSubramanian%26aufirst%3DK.%26aulast%3DSorger%26aufirst%3DP.%2BK.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DA%2520multi-targeted%2520probe-based%2520strategy%2520to%2520identify%2520signaling%2520vulnerabilities%2520in%2520cancers%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2019%26volume%3D294%26spage%3D8664%26doi%3D10.1074%2Fjbc.RA118.006805" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ATE" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ATE','PDB','6ATE'); return false;">PDB: 6ATE</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5J7S" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5J7S','PDB','5J7S'); return false;">PDB: 5J7S</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MZW" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MZW','PDB','6MZW'); return false;">PDB: 6MZW</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i68"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01502">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_55080"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.9b01502?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01502</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Western blot analysis characterizing target engagement and signaling inhibition in cells for compounds <b>AZD0530</b>, <b>15a</b>, and <b>20f</b> (Figure S1); structure of <b>TL13-68</b> (Figure S2); KinomeScan profiling data for compound <b>SM1-71</b> (Table S1); KinomeScan profiling data for compound <b>15a</b> (Table S2); IC<sub>50</sub>’s of top off-targets for compound <b>15a</b> (Table S3); compound <b>15a</b> in a SRC-TEL and HCK-BCR Ba/F3 cell-based assay (Table S4); GR<sub>50</sub> and GR<sub>max</sub> values from a growth inhibitory screen across multiple cancer cell lines for compounds <b>AZD0530</b>, <b>15a</b>, and <b>20f</b> (Table S5) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01502/suppl_file/jm9b01502_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01502/suppl_file/jm9b01502_si_002.csv">CSV</a>)</p></li><li><p class="inline">Growth inhibtion assay for compounds <b>AZD0530</b>, <b>15a</b>, and <b>20f</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01502/suppl_file/jm9b01502_si_003.xlsx">XLSX</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01502/suppl_file/jm9b01502_si_001.pdf">jm9b01502_si_001.pdf (1.25 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01502/suppl_file/jm9b01502_si_002.csv">jm9b01502_si_002.csv (6.49 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01502/suppl_file/jm9b01502_si_003.xlsx">jm9b01502_si_003.xlsx (10.41 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b01502&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-4%3BpageGroup%3Astring%3APublication+Pages%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b01502%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b01502" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                5MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-b425991822a79ada59b3.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-fb88834b824b41b135ad.js"></script>

<script type="text/javascript" src="/wro/ke1q~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"66798842fb892374","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.6.0","si":10}'></script>
</body>
</html>
