
<html lang="en"     class="pb-page"  data-request-id="e46d8f97-5628-49ea-816f-f38ee8e8dc3e"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;article:article:10.1021/acs.jmedchem.1c00509"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Expedited Approach toward the Rational Design of Noncovalent SARS-CoV-2 Main Protease Inhibitors" /></meta><meta name="dc.Creator" content="Naoya  Kitamura" /></meta><meta name="dc.Creator" content="Michael Dominic  Sacco" /></meta><meta name="dc.Creator" content="Chunlong  Ma" /></meta><meta name="dc.Creator" content="Yanmei  Hu" /></meta><meta name="dc.Creator" content="Julia Alma  Townsend" /></meta><meta name="dc.Creator" content="Xiangzhi  Meng" /></meta><meta name="dc.Creator" content="Fushun  Zhang" /></meta><meta name="dc.Creator" content="Xiujun  Zhang" /></meta><meta name="dc.Creator" content="Mandy  Ba" /></meta><meta name="dc.Creator" content="Tommy  Szeto" /></meta><meta name="dc.Creator" content="Adis  Kukuljac" /></meta><meta name="dc.Creator" content="Michael Thomas  Marty" /></meta><meta name="dc.Creator" content="David  Schultz" /></meta><meta name="dc.Creator" content="Sara  Cherry" /></meta><meta name="dc.Creator" content="Yan  Xiang" /></meta><meta name="dc.Creator" content="Yu  Chen" /></meta><meta name="dc.Creator" content="Jun  Wang" /></meta><meta name="dc.Description" content="The main protease (Mpro) of SARS-CoV-2 is a validated antiviral drug target. Several Mpro inhibitors have been reported with potent enzymatic inhibition and cellular antiviral activity, including G..." /></meta><meta name="Description" content="The main protease (Mpro) of SARS-CoV-2 is a validated antiviral drug target. Several Mpro inhibitors have been reported with potent enzymatic inhibition and cellular antiviral activity, including G..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="April 23, 2021" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.1c00509" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2021 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.1c00509" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.1c00509" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.1c00509" /></link>
        
    
    

<title>Expedited Approach toward the Rational Design of Noncovalent SARS-CoV-2 Main Protease Inhibitors | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.1c00509" /></meta><meta property="og:title" content="Expedited Approach toward the Rational Design of Noncovalent SARS-CoV-2 Main Protease Inhibitors" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00509/20210512/images/large/jm1c00509_0015.jpeg" /></meta><meta property="og:description" content="The main protease (Mpro) of SARS-CoV-2 is a validated antiviral drug target. Several Mpro inhibitors have been reported with potent enzymatic inhibition and cellular antiviral activity, including GC376, boceprevir, calpain inhibitors II, and XII, with each containing a reactive warhead that covalently modifies the catalytic Cys145. Coupling structure-based drug design with the one-pot Ugi four-component reaction, we discovered one of the most potent noncovalent inhibitors, 23R (Jun8-76-3A) that is structurally distinct from the canonical Mpro inhibitor GC376. Significantly, 23R is highly selective compared with covalent inhibitors such as GC376, especially toward host proteases. The cocrystal structure of SARS-CoV-2 Mpro with 23R revealed a previously unexplored binding site located in between the S2 and S4 pockets. Overall, this study discovered 23R, one of the most potent and selective noncovalent SARS-CoV-2 Mpro inhibitors reported to date, and a novel binding pocket in Mpro that can be explored for inhibitor design." /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta><meta name="twitter:description" content="The main protease (Mpro) of SARS-CoV-2 is a validated antiviral drug target. Several Mpro inhibitors have been reported with potent enzymatic inhibition and cellular antiviral activity, including GC376, boceprevir, calpain inhibitors II, and XII, with each containing a reactive warhead that covalently modifies the catalytic Cys145. Coupling structure-based drug design with the one-pot Ugi four-component reaction, we discovered one of the most potent noncovalent inhibitors, 23R (Jun8-76-3A) that is structurally distinct from the canonical Mpro inhibitor GC376. Significantly, 23R is highly selective compared with covalent inhibitors such as GC376, especially toward host proteases. The cocrystal structure of SARS-CoV-2 Mpro with 23R revealed a previously unexplored binding site located in between the S2 and S4 pockets. Overall, this study discovered 23R, one of the most potent and selective noncovalent SARS-CoV-2 Mpro inhibitors reported to date, and a novel binding pocket in Mpro that can be explored for inhibitor design." /></meta><meta name="twitter:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00509/20210512/images/large/jm1c00509_0015.jpeg" /></meta><meta name="twitter:title" content="Expedited Approach toward the Rational Design of Noncovalent SARS-CoV-2 Main Protease Inhibitors" /></meta>
        
            <meta property="og:type" content="Article" /></meta>
        

        
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.1c00509"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/ke1q~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-6ee29527b3d8ff00a106.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00509">
            
        





































    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-12/achre4.2021.54.issue-12/20210615/achre4.2021.54.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-6/amrcda.2021.2.issue-6/20210625/amrcda.2021.2.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-6/aabmcb.2021.4.issue-6/20210621/aabmcb.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-6/aaembp.2021.3.issue-6/20210622/aaembp.2021.3.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-6/aaemcq.2021.4.issue-6/20210628/aaemcq.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-25/aamick.2021.13.issue-25/20210630/aamick.2021.13.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-6/aanmf6.2021.4.issue-6/20210625/aanmf6.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-6/aapmcd.2021.3.issue-6/20210611/aapmcd.2021.3.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-6/abseba.2021.7.issue-6/20210614/abseba.2021.7.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-12/accacs.2021.11.issue-12/20210618/accacs.2021.11.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-6/acscii.2021.7.issue-6/20210623/acscii.2021.7.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-6/acbcct.2021.16.issue-6/20210618/acbcct.2021.16.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-3/achsc5.2021.28.issue-3/20210524/achsc5.2021.28.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-12/acncdm.2021.12.issue-12/20210616/acncdm.2021.12.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-6/aesccq.2021.5.issue-6/20210617/aesccq.2021.5.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-6/aelccp.2021.6.issue-6/20210611/aelccp.2021.6.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-6/aeecco.2021.1.issue-6/20210611/aeecco.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-6/aewcaa.2021.1.issue-6/20210611/aewcaa.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-5/afsthl.2021.1.issue-5/20210618/afsthl.2021.1.issue-5.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-6/aidcbc.2021.7.issue-6/20210611/aidcbc.2021.7.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-6/amlccd.2021.10.issue-6/20210615/amlccd.2021.10.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-6/amlcef.2021.3.issue-6/20210607/amlcef.2021.3.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-6/amclct.2021.12.issue-6/20210610/amclct.2021.12.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-6/ancac3.2021.15.issue-6/20210622/ancac3.2021.15.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-25/acsodf.2021.6.issue-25/20210629/acsodf.2021.6.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-6/apchd5.2021.8.issue-6/20210616/apchd5.2021.8.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-6/ascefj.2021.6.issue-6/20210625/ascefj.2021.6.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-25/ascecg.2021.9.issue-25/20210628/ascecg.2021.9.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-6/asbcd6.2021.10.issue-6/20210618/asbcd6.2021.10.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-25/ancham.2021.93.issue-25/20210629/ancham.2021.93.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-25/bichaw.2021.60.issue-25/20210629/bichaw.2021.60.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-6/bcches.2021.32.issue-6/20210616/bcches.2021.32.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-6/bomaf6.2021.22.issue-6/20210614/bomaf6.2021.22.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-24/cgeabj.2021.99.issue-24/20210628-01/cgeabj.2021.99.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-6/crtoec.2021.34.issue-6/20210621/crtoec.2021.34.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-12/chreay.2021.121.issue-12/20210623/chreay.2021.121.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-12/cmatex.2021.33.issue-12/20210622/cmatex.2021.33.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-6/cgdefu.2021.21.issue-6/20210602/cgdefu.2021.21.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-12/enfuem.2021.35.issue-12/20210617/enfuem.2021.35.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-12/esthag.2021.55.issue-12/20210615/esthag.2021.55.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-6/estlcu.2021.8.issue-6/20210608/estlcu.2021.8.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-25/iecred.2021.60.issue-25/20210630/iecred.2021.60.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-12/inocaj.2021.60.issue-12/20210621/inocaj.2021.60.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-6/jaaucr.2021.1.issue-6/20210628/jaaucr.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-25/jafcau.2021.69.issue-25/20210630/jafcau.2021.69.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-6/jceaax.2021.66.issue-6/20210610/jceaax.2021.66.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-6/jceda8.2021.98.issue-6/20210608/jceda8.2021.98.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-6/jcisd8.2021.61.issue-6/20210628/jcisd8.2021.61.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-6/jctcce.2021.17.issue-6/20210608/jctcce.2021.17.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/jmcmar.2021.64.issue-12/20210624/jmcmar.2021.64.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-6/jnprdf.2021.84.issue-6/20210625/jnprdf.2021.84.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-12/joceah.2021.86.issue-12/20210618/joceah.2021.86.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-24/jpcafh.2021.125.issue-24/20210624/jpcafh.2021.125.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-24/jpcbfk.2021.125.issue-24/20210624/jpcbfk.2021.125.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-24/jpccck.2021.125.issue-24/20210624/jpccck.2021.125.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-24/jpclcd.2021.12.issue-24/20210624/jpclcd.2021.12.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-6/jprobs.2021.20.issue-6/20210604/jprobs.2021.20.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-25/jacsat.2021.143.issue-25/20210630/jacsat.2021.143.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-6/jamsef.2021.32.issue-6/20210602/jamsef.2021.32.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-25/langd5.2021.37.issue-25/20210629/langd5.2021.37.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-12/mamobx.2021.54.issue-12/20210622/mamobx.2021.54.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-6/mpohbp.2021.18.issue-6/20210607/mpohbp.2021.18.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-12/nalefd.2021.21.issue-12/20210623/nalefd.2021.21.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-12/orlef7.2021.23.issue-12/20210618/orlef7.2021.23.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-6/oprdfk.2021.25.issue-6/20210618/oprdfk.2021.25.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-12/orgnd7.2021.40.issue-12/20210628/orgnd7.2021.40.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.1c00509&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.1c00509&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.1c00509&amp;href=/doi/10.1021/acs.jmedchem.1c00509" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2021</span><span class="cit-fg-volume">, XXXX</span><span class="cit-fg-issue">, XXX</span><span class="cit-fg-pageRange">, XXX-XXX</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/0/0" title="Return to Articles ASAP" id="returnToASAP" class="content-navigation__btn--return"><span>RETURN TO ARTICLES ASAP</span></a><a href="/doi/10.1021/acs.jmedchem.1c00258" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c02258" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Expedited Approach toward the Rational Design of Noncovalent SARS-CoV-2 Main Protease Inhibitors</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Naoya Kitamura</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Naoya Kitamura</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmacology and Toxicology, College of Pharmacy, The University of Arizona, Tucson, Arizona 85721, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Naoya++Kitamura">Naoya Kitamura</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Michael Dominic Sacco</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Michael Dominic Sacco</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Molecular Medicine, Morsani College of Medicine, University of South Florida, Tampa, Florida 33612, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Michael+Dominic++Sacco">Michael Dominic Sacco</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Chunlong Ma</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Chunlong Ma</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmacology and Toxicology, College of Pharmacy, The University of Arizona, Tucson, Arizona 85721, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Chunlong++Ma">Chunlong Ma</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yanmei Hu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yanmei Hu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmacology and Toxicology, College of Pharmacy, The University of Arizona, Tucson, Arizona 85721, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yanmei++Hu">Yanmei Hu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Julia Alma Townsend</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Julia Alma Townsend</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry and Biochemistry, The University of Arizona, Tucson, Arizona 85721, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Julia+Alma++Townsend">Julia Alma Townsend</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Xiangzhi Meng</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xiangzhi Meng</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Microbiology, Immunology and Molecular Genetics, University of Texas Health Science Center at San Antonio, San Antonio, Texas 78229, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xiangzhi++Meng">Xiangzhi Meng</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Fushun Zhang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Fushun Zhang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Microbiology, Immunology and Molecular Genetics, University of Texas Health Science Center at San Antonio, San Antonio, Texas 78229, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Fushun++Zhang">Fushun Zhang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Xiujun Zhang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xiujun Zhang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Molecular Medicine, Morsani College of Medicine, University of South Florida, Tampa, Florida 33612, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xiujun++Zhang">Xiujun Zhang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Mandy Ba</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mandy Ba</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmacology and Toxicology, College of Pharmacy, The University of Arizona, Tucson, Arizona 85721, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mandy++Ba">Mandy Ba</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Tommy Szeto</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Tommy Szeto</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmacology and Toxicology, College of Pharmacy, The University of Arizona, Tucson, Arizona 85721, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Tommy++Szeto">Tommy Szeto</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Adis Kukuljac</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Adis Kukuljac</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Molecular Medicine, Morsani College of Medicine, University of South Florida, Tampa, Florida 33612, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Adis++Kukuljac">Adis Kukuljac</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Michael Thomas Marty</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Michael Thomas Marty</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry and Biochemistry, The University of Arizona, Tucson, Arizona 85721, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Michael+Thomas++Marty">Michael Thomas Marty</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">David Schultz</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">David Schultz</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry and Biophysics, University of Pennsylvania, Philadelphia, Pennsylvania 19104, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=David++Schultz">David Schultz</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Sara Cherry</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sara Cherry</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pathology and Laboratory Medicine, University of Pennsylvania, Philadelphia, Pennsylvania 19104, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Sara++Cherry">Sara Cherry</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yan Xiang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yan Xiang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Microbiology, Immunology and Molecular Genetics, University of Texas Health Science Center at San Antonio, San Antonio, Texas 78229, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yan++Xiang">Yan Xiang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yu Chen</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yu Chen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Molecular Medicine, Morsani College of Medicine, University of South Florida, Tampa, Florida 33612, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Yu Chen, Tel: 813-974-7809. Email: <a href="/cdn-cgi/l/email-protection#c9b0aaa1aca7f889bcbaafe7acadbc"><span class="__cf_email__" data-cfemail="41382229242f70013432276f242534">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yu++Chen">Yu Chen</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Jun Wang</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jun Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmacology and Toxicology, College of Pharmacy, The University of Arizona, Tucson, Arizona 85721, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Jun Wang, Tel: 520-626-1366. Fax: 520-626-0749. Email: <a href="/cdn-cgi/l/email-protection#e8829d869f89868fa89880899a85898b91c6899a8192878689c68d8c9d"><span class="__cf_email__" data-cfemail="dfb5aab1a8beb1b89fafb7beadb2bebca6f1beadb6a5b0b1bef1babbaa">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jun++Wang">Jun Wang</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-4845-4621" title="Orcid link">http://orcid.org/0000-0002-4845-4621</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.1c00509&amp;href=/doi/10.1021%2Facs.jmedchem.1c00509" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2021</span></span><span class="cit-volume">, XXXX</span><span class="cit-issue">, XXX</span><span class="cit-pageRange">, XXX-XXX</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">April 23, 2021</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>19 March 2021</li><li><span class="item_label"><b>Published</b> online</span>23 April 2021</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.1c00509" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00509</a></div><div class="article_header-article-copyright"><strong>© 2021 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D1%26pageCount%3D18%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DNaoya%2BKitamura%252C%2BMichael%2BDominic%2BSacco%252C%2BChunlong%2BMa%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D0%26issueNum%3D0%26contentID%3Dacs.jmedchem.1c00509%26title%3DExpedited%2BApproach%2Btoward%2Bthe%2BRational%2BDesign%2Bof%2BNoncovalent%2BSARS-CoV-2%2BMain%2BProtease%2BInhibitors%26numPages%3D18%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D18%26publicationDate%3DApril%2B2021">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.1c00509"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1644</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">1</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.1c00509" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Expedited Approach toward the Rational Design of Noncovalent SARS-CoV-2 Main Protease Inhibitors&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Naoya&quot;,&quot;last_name&quot;:&quot;Kitamura&quot;},{&quot;first_name&quot;:&quot;Michael&quot;,&quot;last_name&quot;:&quot;Dominic Sacco&quot;},{&quot;first_name&quot;:&quot;Chunlong&quot;,&quot;last_name&quot;:&quot;Ma&quot;},{&quot;first_name&quot;:&quot;Yanmei&quot;,&quot;last_name&quot;:&quot;Hu&quot;},{&quot;first_name&quot;:&quot;Julia&quot;,&quot;last_name&quot;:&quot;Alma Townsend&quot;},{&quot;first_name&quot;:&quot;Xiangzhi&quot;,&quot;last_name&quot;:&quot;Meng&quot;},{&quot;first_name&quot;:&quot;Fushun&quot;,&quot;last_name&quot;:&quot;Zhang&quot;},{&quot;first_name&quot;:&quot;Xiujun&quot;,&quot;last_name&quot;:&quot;Zhang&quot;},{&quot;first_name&quot;:&quot;Mandy&quot;,&quot;last_name&quot;:&quot;Ba&quot;},{&quot;first_name&quot;:&quot;Tommy&quot;,&quot;last_name&quot;:&quot;Szeto&quot;},{&quot;first_name&quot;:&quot;Adis&quot;,&quot;last_name&quot;:&quot;Kukuljac&quot;},{&quot;first_name&quot;:&quot;Michael&quot;,&quot;last_name&quot;:&quot;Thomas Marty&quot;},{&quot;first_name&quot;:&quot;David&quot;,&quot;last_name&quot;:&quot;Schultz&quot;},{&quot;first_name&quot;:&quot;Sara&quot;,&quot;last_name&quot;:&quot;Cherry&quot;},{&quot;first_name&quot;:&quot;Yan&quot;,&quot;last_name&quot;:&quot;Xiang&quot;},{&quot;first_name&quot;:&quot;Yu&quot;,&quot;last_name&quot;:&quot;Chen&quot;},{&quot;first_name&quot;:&quot;Jun&quot;,&quot;last_name&quot;:&quot;Wang&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2021&quot;,&quot;month&quot;:&quot;04&quot;,&quot;day&quot;:&quot;23&quot;,&quot;issue&quot;:&quot;0&quot;,&quot;volume&quot;:&quot;0&quot;,&quot;pages&quot;:&quot;&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.1c00509&quot;},&quot;abstract&quot;:&quot;The main protease (Mpro) of SARS-CoV-2 is a validated antiviral drug target. Several Mpro inhibitors have been reported with potent enzymatic inhibition and cellular antiviral activity, including GC376, boceprevir, calpain inhibitors II, and XII, with each containing a reactive warhead that covalently modifies the catalytic Cys145. Coupling structure-based drug design with the one-pot Ugi four-component reaction, we discovered one of the most potent noncovalent inhibitors, 23R (Jun8-76-3A) that is structurally distinct from the canonical Mpro inhibitor GC376. Significantly, 23R is highly selective compared with covalent inhibitors such as GC376, especially toward host proteases. The cocrystal structure of SARS-CoV-2 Mpro with 23R revealed a previously unexplored binding site located in between the S2 and S4 pockets. Overall, this study discovered 23R, one of the most potent and selective noncovalent SARS-CoV-2 Mpro inhibitors reported to date, and a novel binding pocket in Mpro that can be explored for inhibi&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00509&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00509" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00509&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00509" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00509&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00509" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.1c00509&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00509&amp;href=/doi/10.1021/acs.jmedchem.1c00509" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.1c00509" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.1c00509" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (8 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00509&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.1c00509%26sid%3Dliteratum%253Aachs%26pmid%3D33891389%26genre%3Darticle%26aulast%3DKitamura%26date%3D0%26atitle%3DExpedited%2BApproach%2Btoward%2Bthe%2BRational%2BDesign%2Bof%2BNoncovalent%2BSARS-CoV-2%2BMain%2BProtease%2BInhibitors%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D0%26issue%3D0%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292644" title="SARS-CoV-2">SARS-CoV-2</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=292641" title="SARS-CoV">SARS-CoV</a>,</li><li><a href="/action/doSearch?ConceptID=291362" title="Antimicrobial agents">Antimicrobial agents</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/0/0" title="Go to Volume 0, Issue 0"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/largecover.jpg" alt="Go to Volume 0, Issue 0"/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00509/20210512/images/medium/jm1c00509_0015.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00509/20210512/images/large/jm1c00509_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00509&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">The main protease (M<sup>pro</sup>) of SARS-CoV-2 is a validated antiviral drug target. Several M<sup>pro</sup> inhibitors have been reported with potent enzymatic inhibition and cellular antiviral activity, including <b>GC376</b>, <b>boceprevir</b>, <b>calpain inhibitors II</b>, and <b>XII</b>, with each containing a reactive warhead that covalently modifies the catalytic Cys145. Coupling structure-based drug design with the one-pot Ugi four-component reaction, we discovered one of the most potent noncovalent inhibitors, <b>23R</b> (<b>Jun8-76-3A</b>) that is structurally distinct from the canonical M<sup>pro</sup> inhibitor <b>GC376</b>. Significantly, <b>23R</b> is highly selective compared with covalent inhibitors such as <b>GC376</b>, especially toward host proteases. The cocrystal structure of SARS-CoV-2 M<sup>pro</sup> with <b>23R</b> revealed a previously unexplored binding site located in between the S2 and S4 pockets. Overall, this study discovered <b>23R</b>, one of the most potent and selective noncovalent SARS-CoV-2 M<sup>pro</sup> inhibitors reported to date, and a novel binding pocket in M<sup>pro</sup> that can be explored for inhibitor design.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="specialIssueNotice" class="extra-info-sec articleNote"><h4>SPECIAL ISSUE</h4><p class="last">This article is part of the <a href="/toc/jmcmar"><issue-title>COVID-19</issue-title></a> special issue.</p></div><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_20204" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_20204" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The COVID-19 pandemic had a significant impact on the global economy and public health, and there is an urgent need for therapeutic interventions. The viral polymerase inhibitor remdesivir gained FDA approval on October 22nd, 2020. The combination therapy of remdesivir with the Janus kinase (JAK) inhibitor baricitinib also received FDA emergency use authorization.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Among the other drug targets being pursued at preclinical and clinical stages,<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> the viral main protease (M<sup>pro</sup>), also called 3-chymotrypsin-like protease (3CL<sup>pro</sup>), is one of the most extensively explored high profile antiviral drug targets.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> M<sup>pro</sup> is a cysteine protease encoded in the viral polyprotein as nonstructural protein 5 (Nsp5) that cleaves the viral polyproteins pp1a and pp1ab at more than 11 sites. Despite its multiple proteolytic sites, M<sup>pro</sup> was shown to have a high substrate specificity of glutamine at the P1 position.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> As such, the majority of the reported M<sup>pro</sup> inhibitors were designed to contain a 2-pyrrolidone at the P1 substitution as a mimetic of the glutamine in the substrate.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> Most advanced M<sup>pro</sup> inhibitors including <b>PF-07304814</b>,<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a><b>GC376</b>,<a onclick="showRef(event, 'ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref7 ref8">(7,8)</a><b>6j</b>,<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a><b>MI-09</b>, and <b>MI-30</b><a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> all belong to this category (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). <b>PF-07304814</b>, an α-hydroxyl ketone prodrug, is being developed by Pfizer and has optimal pharmacokinetic properties and recently entered human clinical trials.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a><b>GC376</b> has <i>in vivo</i> antiviral efficacy in treating cats infected with lethal feline infectious peritonitis virus.<a onclick="showRef(event, 'ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref11 ref12">(11,12)</a> Recently, the <b>GC376</b> analog <b>6j</b> was shown to protect mice from MERS-CoV infection.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a><b>MI-09</b> and <b>MI-30</b> were shown to protect mice from lethal SARS-CoV-2 infection.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> These promising results highlight the translational potential of M<sup>pro</sup> inhibitors as potent SARS-CoV-2 antivirals and validate M<sup>pro</sup> as an antiviral drug target for coronaviruses.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00509/20210512/images/medium/jm1c00509_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00509/20210512/images/large/jm1c00509_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Promising SARS-CoV-2 M<sup>pro</sup> inhibitors reported in the literature with translational potential.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00509/20210512/images/large/jm1c00509_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00509&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Drug discovery is a lengthy process involving iterative cycles of design, synthesis, and pharmacological characterization. In the event of the COVID-19 pandemic, an expedited approach with a fast turnover of this development cycle is highly desired. Using SARS-CoV-2 M<sup>pro</sup> as a drug target, we report herein a fast-track drug discovery approach by coupling structure-based drug design and the Ugi four-component reaction (Ugi-4CR) methodology. The design was based on the superimposed structures of SARS-CoV or SARS-CoV-2 M<sup>pro</sup> in complex with inhibitors including <b>GC376</b>, <b>calpain inhibitor XII</b>, and <b>ML188 (R)</b>.<a onclick="showRef(event, 'ref7 ref8 ref13'); return false;" href="javascript:void(0);" class="ref ref7 ref8 ref13">(7,8,13)</a> The lead compound <b>23R</b> from this study is the most potent noncovalent SARS-CoV-2 M<sup>pro</sup> inhibitor reported to date in terms of enzymatic inhibition and cellular antiviral activity. The target selectivity of the designed inhibitors was profiled against a panel of viral proteases and host proteases, and the noncovalent inhibitor <b>23R</b> was found to be highly selective compared to the covalent inhibitor <b>GC376</b>. An X-ray crystal structure of SARS-CoV-2 M<sup>pro</sup> in complex with <b>23R</b> was solved, revealing a drug-induced conformational change and a previously unexplored binding site in between the S2 and S4 pockets. Overall, this study led to the discovery of the noncovalent M<sup>pro</sup> inhibitor <b>23R</b> with potent enzymatic inhibition and <i>in vitro</i> cellular antiviral activity with a novel mechanism of action.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i4">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_84172" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_84172" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Rational Design of Noncovalent SARS-CoV-2 M<sup>pro</sup> Inhibitors</h3><div class="NLM_p">Among the noncanonical SARS-CoV-2 M<sup>pro</sup> inhibitors we recently discovered, <b>calpain inhibitor XII</b> has an unexpected binding mode showing an inverted conformation in the active site (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A).<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> Instead of projecting the norvaline and leucine side chains into the S1 and S2 pockets as one would expect from its chemical structure, the pyridinyl substitution snuggly fits in the S1 pocket and forms a hydrogen bond with the H163 imidazole (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A). This hydrogen bond is essential, as replacing the pyridine with benzene led to an analog <b>UAWJ257</b> with a significant loss of enzymatic inhibition.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> Examining the X-ray crystal structures of SARS-CoV and SARS-CoV-2 M<sup>pro</sup> in the PDB database revealed another compound <b>ML188 (R)</b>,<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> which shares a similar binding mode with <b>calpain inhibitor XII</b>. <b>ML188 (R)</b> is a noncovalent SARS-CoV M<sup>pro</sup> inhibitor derived from a high-throughput screening hit.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> The pyridinyl from <b>ML188 (R)</b> similarly fits in the S1 pocket and forms a hydrogen bond with the H163 side chain imidazole (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B). In addition, the furyl oxygen and its amide oxygen both form a hydrogen bond with the G143 main chain amide amine. <b>ML188 (R)</b> was reported to inhibit the SARS-CoV M<sup>pro</sup> with an IC<sub>50</sub> value of 1.5 ± 0.3 μM and the SARS-CoV viral replication in Vero E6 cells with an EC<sub>50</sub> value of 12.9 μM.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> Several follow up studies have been conducted to optimize the enzymatic inhibition and cellular antiviral activity of this series of compounds; however, no significant improvement has been made.<a onclick="showRef(event, 'ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref14 ref15">(14,15)</a></div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00509/20210512/images/medium/jm1c00509_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00509/20210512/images/large/jm1c00509_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Structure of M<sup>pro</sup> with inhibitors. (A) X-ray crystal structure of SARS-CoV-2 M<sup>pro</sup> in complex with <b>calpain inhibitor XII</b> (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6XFN">6XFN</a>). (B) X-ray crystal structure of SARS-Co-V M<sup>pro</sup> in complex with <b>ML188 (R)</b> (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3V3M">3V3M</a>). Hydrogen bonds are shown in dashed lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00509/20210512/images/large/jm1c00509_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00509&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The similar binding mode of <b>ML188 (R)</b> with <b>calpain inhibitor XII</b>, coupled with the convenient synthesis through the one pot Ugi-4CR, inspired us to design noncovalent SARS-CoV-2 M<sup>pro</sup> inhibitors based on the <b>ML188 (R)</b> scaffold. Specifically, we leverage our understanding of the M<sup>pro</sup> inhibition mechanism based on the X-ray crystal structures of SARS-CoV-2 M<sup>pro</sup> with multiple inhibitors to guide the lead optimization (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A–D).<a onclick="showRef(event, 'ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref7 ref8">(7,8)</a> Overlaying the X-ray crystal structures of SARS-CoV M<sup>pro</sup> + <b>ML188 (R)</b> (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3V3M">3V3M</a>) and the SARS-CoV M<sup>pro</sup> H41A mutant + the peptide substrate (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2Q6G">2Q6G</a>) revealed that the furyl, 4-<i>tert</i>-butylphenyl, pyridinyl, and <i>tert</i>-butyl of <b>ML188 (R)</b> fit in the S1′, S2, S1, and S3 pockets, respectively (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A, D). Therefore, the furyl, 4-<i>tert</i>-butylphenyl, pyridinyl, and <i>tert</i>-butyl substitutions in <b>ML188 (R)</b> were defined as P1′, P2, P1, and P3, respectively. Next, overlaying the structure of SARS-CoV M<sup>pro</sup> + <b>ML188 (R)</b> (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3V3M">3V3M</a>) and SARS-CoV-2 M<sup>pro</sup> + <b>GC376</b> (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6WTT">6WTT</a>) suggested that the <i>tert</i>-butyl at the P3 substitution of <b>ML188 (R)</b> can be extended to fit in the S4 pocket (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B, D). Previous structure–activity relationship studies of <b>GC376</b> indicate that P4 substitution is important, while P3 substitution does not contribute significantly to the binding affinity, as it is solvent exposed.<a onclick="showRef(event, 'ref3 ref8 ref9 ref16'); return false;" href="javascript:void(0);" class="ref ref3 ref8 ref9 ref16">(3,8,9,16)</a> Similarly, the overlaying structures of SARS-CoV M<sup>pro</sup> + <b>ML188 (R)</b> (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3V3M">3V3M</a>) and SARS-CoV-2 M<sup>pro</sup> + <b>UAWJ247</b> (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6XBH">6XBH</a>) suggested that the 4-<i>tert</i>-butyl at the P2 substitution of <b>ML188 (R)</b> can be replaced by phenyl to occupy the extra space in the S2 pocket (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C, D). Overall, binding site analysis suggests that extending the P2 and P3 substitutions of <b>ML188 (R)</b> might lead to better shape complementarity with the SARS-CoV-2 M<sup>pro</sup> (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>E). In practice, we adopted a stepwise optimization procedure in which the P3 and P2 substitutions were optimized individually in step 1 and then the optimal P2/P3 substitutions were combined in step 2 (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>E).</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00509/20210512/images/medium/jm1c00509_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00509/20210512/images/large/jm1c00509_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Design rationale for the noncovalent SARS-CoV-2 M<sup>pro</sup> inhibitors. (A) Superimposed X-ray crystal structures of SARS-CoV M<sup>pro</sup> + ML188 (R) (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3V3M">3V3M</a>, green) and SARS-CoV M<sup>pro</sup> H41A mutant + peptide substrate (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2Q6G">2Q6G</a>, yellow with backbone shown as ribbon representation). (B) Superimposed X-ray crystal structures of SARS-CoV M<sup>pro</sup> + <b>ML188 (R)</b> (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3V3M">3V3M</a>) and SARS-CoV-2 M<sup>pro</sup> + <b>GC376</b> (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6WTT">6WTT</a>). (C) Superimposed X-ray crystal structures of SARS-CoV M<sup>pro</sup> + <b>ML188 (R)</b> (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3V3M">3V3M</a>) and SARS-CoV-2 M<sup>pro</sup> + <b>UAWJ247</b> (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6XBH">6XBH</a>). (D) Chemical structures of <b>ML188 (R)</b>, peptide substrate VLQS, <b>GC376</b>, and <b>UAWJ247</b>. (E) Stepwise optimization of <b>ML188 (R)</b> toward potent noncovalent SARS-CoV-2 M<sup>pro</sup> inhibitor.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00509/20210512/images/large/jm1c00509_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00509&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Guided by the design rationale elucidated above, a focused library of <b>ML188</b> analogs was designed and synthesized (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). As the P1′ furyl and P1 pyridinyl both form a critical hydrogen bond with the M<sup>pro</sup> (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A, B), the P1′ and P1 substitutions were kept with minimal variations for the design of noncovalent inhibitors (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>C). All designed compounds were synthesized using the one pot Ugi four-component reaction and tested as enantiomer/diastereomer mixtures (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>C). To circumvent the need of relying on an expensive chiral HPLC column for the separation of enantiomers, we strategically introduced the chiral isocyanide so that the diastereomer product mixture can be separated by convenient silica gel column or reverse phase HPLC column purification.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a></div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00509/20210512/images/medium/jm1c00509_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00509/20210512/images/large/jm1c00509_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Design and synthesis of a focused library of noncovalent SARS-CoV-2 M<sup>pro</sup> inhibitors. (A) X-ray crystal structure of SARS-CoV M<sup>pro</sup> + <b>ML188 (R)</b> (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3V3M">3V3M</a>). (B) Binding interactions of <b>ML188 (R)</b> with SARS-CoV M<sup>pro</sup>. (C) Synthesis of <b>ML188</b> analogs using the Ugi four-component reaction.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00509/20210512/images/large/jm1c00509_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00509&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Structure–Activity Relationship Studies of Noncovalent SARS-CoV-2 M<sup>pro</sup> Inhibitors</h3><div class="NLM_p">In total, 39 compounds were synthesized (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A–E) and all compounds were initially tested as a mixture of enantiomers or diastereomers in the FRET-based enzymatic assay against SARS-CoV-2 M<sup>pro</sup> at 20 μM (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>F). Compounds showing more than 50% inhibition at 20 μM were further titrated to determine the IC<sub>50</sub> values. Next, compounds with IC<sub>50</sub> values lower than 5 μM were selected for cellular cytotoxicity profiling in Vero E6 cells, the cell line which was used for the SARS-CoV-2 antiviral assay. The purpose was to prioritize lead candidates for the <i>in vitro</i> cellular antiviral assay with infectious SARS-CoV-2. As shown in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>, the majority of the designed compounds showed more than 50% inhibition when tested at 20 μM. Specifically, <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A lists compounds with P4 variations. As a reference, <b>ML188</b> (<b>1</b>) (racemic mixture) inhibits SARS-CoV-2 M<sup>pro</sup> with an IC<sub>50</sub> value of 10.96 ± 1.58 μM. It was found that compounds <b>2</b>, <b>3</b>, <b>5</b>, <b>6</b>, <b>7</b>, <b>8</b>, <b>10</b>, and <b>13</b> had improved enzymatic inhibition compared to <b>ML188</b> (<b>1</b>). These results suggest that (a) isopropyl (<b>2</b>), cyclopropyl (<b>3</b>), cyclopentyl (<b>5</b>), cyclohexyl (<b>6</b>), and phenyl (<b>7</b> and <b>8</b>) are the more favorable substitutions at the P3 position than <i>tert</i>-butyl and (b) compound <b>13</b> with the (<i>S</i>)-α-methylbenzyl substitution at the P3 position had improved potency, which suggests that extending the substitutions to the S4 pocket might improve the enzymatic inhibition (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B). Given the advantage of convenient separation of diastereomers over enantiomers, we therefore decided to fix the P3/P4 substitution as α-methylbenzyl substitution during the P2 optimization (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B). All compounds in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B were designed to have extended substitutions at the 4-position of benzyl to occupy the extra space in the S2 pocket (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C). Consistent with the design hypothesis, several compounds including <b>14</b>, <b>17</b>, <b>18</b>, <b>19</b>, <b>20</b>, <b>21</b>, and <b>23</b> had significantly improved enzymatic inhibition (IC<sub>50</sub> < 3 μM) compared to compound <b>13</b>. Replacing the <i>tert</i>-butyl in compound <b>13</b> with the bulkier trimethylsilyl led to compound <b>14</b> with a 2.9-fold increase in M<sup>pro</sup> inhibition. Cyclohexyl (<b>17</b>), thienyl (<b>19</b>), pyrrolyl (<b>20</b>), pyridinyl (<b>21</b>), and phenyl (<b>23</b>) were found to be the most favorable substitutions at the S2 pocket. Compound <b>16</b> with piperidyl substitution had similar potency as compound <b>13</b>, while compound <b>15</b> with O-<i>tert</i> butyl was less active. Further extending the substitution to benzyl led to compound <b>22</b> that was inactive, suggesting biphenyl might be the longest substitution that can be accommodated at the S2 pocket.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00509/20210512/images/medium/jm1c00509_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00509/20210512/images/large/jm1c00509_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Structures of noncovalent and covalent SARS-CoV-2 M<sup>pro</sup> inhibitors and the enzymatic inhibition against SARS-CoV-2 M<sup>pro</sup>. (A) Noncovalent analogs with P3/P4 modifications. (B) Noncovalent analogs with P2 modifications. (C) Noncovalent analogs with P1′ modifications. (D) Noncovalent analogs with P1 modifications. (E) Noncovalent analogs with combined P1′, P1, P2, and P3/P4 modifications. Compounds with potent enzymatic inhibition (IC<sub>50</sub> < 5 μM) but moderate to high cellular cytotoxicity (CC<sub>50</sub> < 100 μM) are labeled in red. Compounds with both potent enzymatic inhibition (IC<sub>50</sub> < 5 μM) and low cellular cytotoxicity (CC<sub>50</sub> > 100 μM) are labeled in blue. (F) Percentage enzymatic activity of SARS-CoV-2 M<sup>pro</sup> in the presence of the designed compounds at 20 μM concentration.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00509/20210512/images/large/jm1c00509_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00509&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The P1′ and P1 substitutions (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>C, D) were chosen to retain the critical hydrogen bonds in <b>ML188</b> (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A). It was found that imidazole (<b>24</b>) was tolerated at the P1′ position (IC<sub>50</sub> = 0.96 ± 0.09 μM), followed by isoxazole (<b>25</b>) (IC<sub>50</sub> = 2.47 ± 0.27 μM) and oxazole (<b>26</b>) (IC<sub>50</sub> = 4.97 ± 0.78 μM). Pyrazine (<b>27</b>) was tolerated at the P1 position (IC<sub>50</sub> = 4.93 ± 0.79 μM); however, imidazole (<b>28</b>) and pyrimidine (<b>29</b>) were not preferred (IC<sub>50</sub> > 20 μM).</div><div class="NLM_p">Next, the above identified favorable P1′, P2, P1, and P3/P4 substitutions were combined and the designed compounds were shown in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>E. Compounds <b>36</b>, <b>37</b>, and <b>38</b> were the most potent leads with IC<sub>50</sub> values of 0.81 ± 0.24, 0.67 ± 0.15, and 0.28 ± 0.07 μM, respectively. Compounds <b>39</b> and <b>40</b> were also highly active with IC<sub>50</sub> values of 1.48 ± 0.56 and 1.05 ± 0.09 μM, respectively.</div><div class="NLM_p last">Among the active compounds with IC<sub>50</sub> values lower than 5 μM, compounds <b>3</b>, <b>6</b>, <b>14</b>, <b>17</b>, <b>18</b>, <b>19</b>, <b>26</b>, <b>34</b>, and <b>36</b> had moderate to high cellular cytotoxicity in Vero E6 cells (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A–E red), while compounds <b>5</b>, <b>20</b>, <b>21</b>, <b>23</b>, <b>24</b>, <b>25</b>, <b>27</b>, <b>31</b>, <b>32</b>, <b>37</b>, <b>38</b>, <b>39</b>, and <b>40</b> were well tolerated and the CC<sub>50</sub> values were greater than 100 μM (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A–E blue).</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Cellular Antiviral Activity of Noncovalent SARS-CoV-2 M<sup>pro</sup> Inhibitors</h3><div class="NLM_p">Next, compounds with potent enzymatic inhibition (IC<sub>50</sub> ≤ 1 μM) and low cellular cytotoxicity (CC<sub>50</sub> > 100 μM) were prioritized for the cellular antiviral assay with infectious SARS-CoV-2 in Vero E6 cells using the immunofluorescence assay as the primary assay (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A). <b>ML188</b> (<b>1</b>) was included as a control. It was found that <b>ML188</b> (<b>1</b>) was inactive in the antiviral assay (EC<sub>50</sub> > 20 μM), probably due to its incomplete inhibition of the M<sup>pro</sup> in the cellular content. Gratifyingly, compounds <b>20</b>, <b>23</b>, <b>37</b>, <b>38</b>, and <b>40</b> all had potent cellular antiviral activity with EC<sub>50</sub> values ranging from 0.82 to 4.54 μM. Compound <b>24</b> was less active (EC<sub>50</sub> = 13.06 ± 2.30 μM), possibly due to the poor cellular membrane permeability.</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00509/20210512/images/medium/jm1c00509_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00509/20210512/images/large/jm1c00509_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Enzymatic inhibition, cellular antiviral activity, and selectivity index of noncovalent M<sup>pro</sup> inhibitors. (A) Antiviral activity and selectivity index of noncovalent SARS-CoV-2 M<sup>pro</sup> inhibitors. (Selection criteria IC<sub>50</sub> < 1 μM, CC<sub>50</sub> > 100 μM). Antiviral assay was performed using the SARS-CoV-2 USA-WA1/2020 isolate in Vero E6 cells with an MOI of 0.05. (B) Chemical structures of the two diastereomers <b>23R</b> and <b>23S</b>. The absolute stereochemistry of compound <b>23R</b> was determined in the cocrystal structure of this diastereomer with SARS-CoV-2 M<sup>pro</sup> (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7KX5">7KX5</a>). (C and D) Antiviral activity of <b>23R</b> against SARS-CoV-2 in Calu-3 cells. (C) Raw data of the percentage of immunofluorescence positive cells with different concentrations of <b>23R</b>. (D) Antiviral potency and cytotoxicity plots. N.T. = not tested. N.A. = not applicable.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00509/20210512/images/large/jm1c00509_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00509&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Given the potent antiviral activity and high selectivity index of these potent lead compounds, we then selected the noncovalent inhibitor <b>23</b> for further characterization. The two diastereomers of <b>23</b> were separated by reverse phase HPLC (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B). Both diastereomers were tested in the FRET-based enzymatic assay. <b>GC376</b> was included as a positive control. It was found that <b>23R</b> is the active diastereomer with an IC<sub>50</sub> value of 0.20 ± 0.01 μM, while the <b>23S</b> diastereomer was not active (IC<sub>50</sub> > 20 μM) (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A). The stereochemistry of <b>23R</b> was determined by the cocrystal structure with SARS-CoV-2 M<sup>pro</sup> as described in the following section. Compared with the parent compound <b>ML188</b> (<b>1</b>), the optimized lead <b>23R</b> had more than a 54-fold increase in enzymatic inhibition against SARS-CoV-2 M<sup>pro</sup>. Compound <b>23R</b> also showed comparable potency against SARS-CoV M<sup>pro</sup> with an IC<sub>50</sub> value of 0.27 ± 0.03 μM. Neither <b>ML188</b> (<b>1</b>) nor <b>23R</b> inhibited the SARS-CoV-2 papain-like protease (PL<sup>pro</sup>) (IC<sub>50</sub> > 20 μM) (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A), suggesting the inhibition of SARS-CoV-2 M<sup>pro</sup> by <b>23R</b> is specific.</div><div class="NLM_p last">Next, the antiviral activity of <b>23R</b> was tested against SARS-CoV-2 (USA-WA1/2020 isolate) in Vero E6 cells using the immunofluorescence assay. It was found that compound <b>23R</b> had an EC<sub>50</sub> value of 1.27 μM (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A), which was similar to the antiviral potency of the covalent inhibitor <b>GC376</b> (EC<sub>50</sub> = 1.51 μM). Compound <b>23R</b> was also not cytotoxic to Vero E6 cells at up to 100 μM. In contrast, the parent compound <b>ML188</b> (<b>1</b>) had no detectable antiviral activity when tested at up to 20 μM. To further confirm the antiviral activity of compound <b>23R</b>, we performed a secondary antiviral assay in the human lung epithelial Calu-3 cell line, which endogenously expresses TMPRSS2 and is widely used as a physiological relevant cell line for SARS-CoV-2 infection. It was found that compound <b>23R</b> inhibited SARS-CoV-2 (USA-WA1/2020 isolate) replication in Calu-3 cells with an EC<sub>50</sub> value of 3.03 μM, and it was not cytotoxic at up to 100 μM (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>C, D).</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Profiling the Target Selectivity of <b>23R</b> against Other Viral Cysteine Proteases and Host Proteases</h3><div class="NLM_p">One of the major challenges facing cysteine protease inhibitors is the target selectivity.<a onclick="showRef(event, 'ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref18 ref19">(18,19)</a> It was recently reported that <b>GC376</b> inhibits host cathepsins B and L in addition to SARS-CoV-2 M<sup>pro</sup>,<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> and <b>GC376</b> analogs <b>compound_1</b> and <b>zyy16</b> that contained the aldehyde and cyanohydrin warheads, respectively, inhibit both calpain 1 and cathepsin K.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> These results raised the concern of the selectivity of this class of covalent inhibitors. To profile the target selectivity of the lead compounds developed in this study, we selected compound <b>23R</b> as a representative example of the noncovalent inhibitor. <b>ML188</b> (<b>1</b>) and <b>GC376</b> were included as controls. The compounds were tested against a panel of viral cysteine proteases including SARS-CoV M<sup>pro</sup>, MERS-CoV M<sup>pro</sup>, HCoV-OC43 M<sup>pro</sup>, EV-A71 3C<sup>pro</sup>, EV-D68 3C<sup>pro</sup>, EV-A71 2A<sup>pro</sup>, EV-D68 2A<sup>pro</sup>, and SARS-CoV-2 PL<sup>pro</sup>, as well as host cysteine proteases calpain 1, cathepsin K, cathepsin L, and serine protease trypsin. It was found that <b>GC376</b> inhibited all the M<sup>pro</sup> and 3C<sup>pro</sup> tested including SARS-CoV-2 M<sup>pro</sup>, SARS-CoV M<sup>pro</sup>, MERS-CoV M<sup>pro</sup>, HCoV-OC43 M<sup>pro</sup>, EV-A71 3C<sup>pro</sup>, and EV-D68 3C<sup>pro</sup> (IC<sub>50</sub> ≤ 0.16 μM), but not the unrelated EV-A71 2A<sup>pro</sup>, EV-D68 2A<sup>pro</sup>, and SARS-CoV-2 PL<sup>pro</sup> (IC<sub>50</sub> > 50 μM) (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>A and <a class="ref internalNav" href="#notes-3" aria-label="Table S1">Table S1</a>). In addition, <b>GC376</b> also inhibited host cysteine proteases calpain 1, cathepsin K, and cathepsin L (IC<sub>50</sub> ≤ 0.074 μM) but not the serine protease trypsin (IC<sub>50</sub> > 50 μM) (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>B and <a class="ref internalNav" href="#notes-3" aria-label="Table S1">Table S1</a>). In contrast, the noncovalent inhibitor <b>23R</b> developed in this study only selectively inhibited SARS-CoV-2 M<sup>pro</sup> and SARS-CoV M<sup>pro</sup> with high potency (IC<sub>50</sub> ≤ 0.27 μM) but not other viral proteases tested, which was similar to <b>ML188</b> (<b>1</b>) (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>A and <a class="ref internalNav" href="#notes-3" aria-label="Table S1">Table S1</a>). Compound <b>23R</b> showed weak inhibition against calpain 1 and cathepsin L with IC<sub>50</sub> values of 6.00 μM and 10.5 μM, respectively, corresponding to selectivity indexes of 40 and 70 with regard to SARS-CoV-2 M<sup>pro</sup> inhibition (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>B and <a class="ref internalNav" href="#notes-3" aria-label="Table S1">Table S1</a>).</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00509/20210512/images/medium/jm1c00509_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00509/20210512/images/large/jm1c00509_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Selectivity of noncovalent and covalent SARS-CoV-2 M<sup>pro</sup> inhibitors shown in a color map. (A) Selectivity against viral cysteine proteases. (B) Selectivity against host proteases. The values plotted were the IC<sub>50</sub> values from the FRET-based enzymatic assay.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00509/20210512/images/large/jm1c00509_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00509&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Mechanism of Action of Noncovalent Inhibitor <b>23R</b> in Inhibiting SARS-CoV-2 M<sup>pro</sup></h3><div class="NLM_p">The mechanism of action was characterized using native mass spectrometry, the thermal shift-binding assay, and enzymatic kinetic studies (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>). In the native mass spectrometry binding assay, compound <b>23R</b> showed dose-dependent binding to SARS-CoV-2 M<sup>pro</sup>, similar to the positive control <b>GC376</b>, with a binding stoichiometry of one drug per monomer (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>A). Similarly, compound <b>23R</b> showed dose-dependent stabilization of the SARS-CoV-2 M<sup>pro</sup> in the thermal shift binding assay with an apparent K<sub>d</sub> value of 9.43 μM, a 9.3-fold decrease compared to <b>ML188</b> (<b>1</b>) (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>B). In the enzymatic kinetic studies, <b>23R</b> was shown to be a noncovalent inhibitor with a K<sub>i</sub> value of 0.07 μM (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>C, D top and middle panels). In comparison, the K<sub>i</sub> for the parent compound <b>ML188</b> (<b>1</b>) is 2.29 μM. The Lineweaver–Burk or double-reciprocal plot with different compound concentrations yielded an intercept at the <i>Y</i>-axis, suggesting that <b>23R</b> is a competitive inhibitor similar to <b>ML188</b> (<b>1</b>) (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>C, D bottom panel). Overall, the enzymatic kinetic studies confirmed that compound <b>23R</b> is a noncovalent inhibitor of SARS-CoV-2 M<sup>pro</sup>.</div><figure id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00509/20210512/images/medium/jm1c00509_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00509/20210512/images/large/jm1c00509_0008.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Characterization of the mechanism of action of <b>23R</b> to SARS-CoV-2 M<sup>pro</sup> using native mass spectrometry, thermal shift assay, and enzyme kinetic studies. (A) Binding of <b>23R</b> to SARS-CoV-2 M<sup>pro</sup> using native mass spectrometry. Native mass spectra with the inset deconvolved spectra revealing ligand binding with 10 μM or 30 μM <b>GC376</b> added (middle panel) and 10 μM and 30 μM <b>23R</b> (bottom panel) with 4 mM DTT added. The peaks are annotated with the blue circle as the dimer, the green down triangle as the dimer with one ligand bound, and the purple up triangle as the dimer with two ligands bound. (B) Dose-dependent melting temperature (<i>T</i><sub>m</sub>) shift in the thermal shift assay. 3 μM SARS-CoV-2 M<sup>pro</sup> protein was incubated with various concentrations of ML188 or <b>23R</b> in the presence of 4 mM DTT. Measured <i>T</i><sub>m</sub> was plotted against compound concentration with one-site binding function in Prism 8. (C and D) Enzymatic kinetic assay with <b>ML188</b> and compound <b>23R</b>. Kinetic parameters in the presence of various concentrations of <b>ML188</b> or <b>23R</b> were globally fitted with a Michaelis–Menten function in prism 8 (top panels); double-reciprocal plots are shown in the right panels. The middle panels show the Morrison plots of compounds <b>ML188</b> and <b>23R</b> with 20 μM FRET substrate used.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00509/20210512/images/large/jm1c00509_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00509&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> X-ray Crystal Structure of SARS-CoV-2 M<sup>pro</sup> with <b>23R</b></h3><div class="NLM_p">Using X-ray crystallography, we successfully determined the binding pose of <b>23R</b> with SARS-CoV-2 M<sup>pro</sup> at 2.6 Å resolution (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>A, <a class="ref internalNav" href="#notes-3" aria-label="Table S2">Table S2</a>). Electron density reveals the body of <b>23R</b> extends throughout the substrate binding channel, with side chains occupying S1′, S1, S2, and a previously unexplored subpocket in between S2 and S4. The binding pose is similar to the previously solved structure of SARS-CoV M<sup>pro</sup> with <b>ML188 (R)</b> (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3V3M">3V3M</a>),<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> consistent with the similarities between the two compounds and between the two proteins (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>B). The furyl moiety of <b>23R</b> binds to a portion of the P1′ site, which normally accommodates small hydrophobic residues. While the furylamide carbonyl group of <b>23R</b> does not insert into the oxyanion hole, it does form a bifurcated hydrogen bond with the apical residue of this oxyanion hole, Gly143. However, the furan ring oxygen is likely a weaker hydrogen bond acceptor than the amide oxygen, and it lies outside of the plane of Gly143’s amide NH. Directly attached to the furylamide moiety is a P2 biphenyl group and a P1 pyridinyl ring. The P2 biphenyl group projects directly into the S2 pocket, which prefers hydrophobic residues such as leucine and phenylalanine. As expected, the P1 pyridinyl ring occupies the S1 pocket, which is known for its strict preference for glutamine. While most M<sup>pro</sup> inhibitors bear a pyrrolidinone glutamine mimetic at the P1 position, we determined that more hydrophobic residues can also bind to the S1 site and that hydrogen bond formation with His163 is critical for inhibition.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> In this instance, the pyridinyl ring of <b>23R</b> is nearly superimposable with the same moiety from <b>calpain inhibitor XII</b> (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>C) forming a close (2.9 Å) hydrogen bond with His163. An amide bond connecting the pyridinyl ring to the α-methylbenzyl moiety forms a hydrogen bond with the main chain of Glu166. The benzyl ring of the α-methylbenzyl moiety is partially positioned in both the S2 and S4 pockets, a novel binding pose that has not been observed with existing M<sup>pro</sup> inhibitors. Normally, a substituent at this position would be expected to flip away from the enzyme core toward the solvent-exposed S3 pocket, which explains why P3 substitutions have little to no influence on the enzymatic inhibition.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> However, the chirality and hydrophobic nature of the benzyl ring in <b>23R</b> cause it to project toward the core near the S2 pocket, forcing Gln189 to rotate outward (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>D). This conformation is reinforced by π-stacking interactions with the first phenyl of the biphenyl substituent. Notably, the binding pose of <b>23R</b> features continuous intramolecular π-stacking, where the phenyl is sandwiched by furan and benzyl groups, potentially contributing to its potent inhibition of M<sup>pro</sup>. Meanwhile, the S4 pocket remains largely unoccupied by <b>23R</b>, leaving room for further improvement. In summary, the X-ray crystal structure of SARS-CoV-2 M<sup>pro</sup> in complex with <b>23R</b> revealed two interesting structural features: (1) The P2 biphenyl is probably the longest substitution that can be accommodated in the S2 pocket, which is consistent with our design hypothesis. (2) The benzyl group from the terminal α-methylbenzyl fits in a pocket in between the S2 and S4 pockets, and this is a ligand-induced binding site that has not been previously explored. Although this is unexpected from the design perspective, this novel binding mode suggests that the new binding pocket in between S2 and S4 can be explored for inhibitor design.</div><figure id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00509/20210512/images/medium/jm1c00509_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00509/20210512/images/large/jm1c00509_0009.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. X-ray crystal structure of SARS-CoV-2 M<sup>pro</sup> in complex with <b>23R</b>. (A) X-ray crystal structure of SARS-CoV-2 M<sup>pro</sup> in complex with <b>23R</b> (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7KX5">7KX5</a>). (B) Superimposed structures of SARS-CoV-2 M<sup>pro</sup> + <b>23R</b> (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7KX5">7KX5</a>) and SARS-CoV M<sup>pro</sup> + <b>ML188 (R)</b> (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3V3M">3V3M</a>). (C) Superimposed structures of SARS-CoV-2 M<sup>pro</sup> + <b>23R</b> (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7KX5">7KX5</a>) and SARS-CoV-2 M<sup>pro</sup> + <b>calpain inhibitor XII</b> (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6XFN">6XFN</a>). (D) Superimposed structures of SARS-CoV-2 M<sup>pro</sup> + <b>23R</b> (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7KX5">7KX5</a>) and SARS-CoV-2 M<sup>pro</sup> + <b>GC376</b> (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6WTT">6WTT</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00509/20210512/images/large/jm1c00509_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00509&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i11">Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_27713" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_27713" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">The viral M<sup>pro</sup> is a high profile antiviral drug target, and several M<sup>pro</sup> inhibitors are now in animal model studies and human clinical trials.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> Among the known M<sup>pro</sup> inhibitors, the majority are covalent inhibitors such as <b>GC376</b> analogs that contain a pyrrolidone in the P1 position as a glutamine mimetic. Several structurally distinct compounds including ebselen, disulfiram, carmofur, PX-12, tideglusib, and shiknonin were claimed as M<sup>pro</sup> inhibitors<a onclick="showRef(event, 'ref22 ref23'); return false;" href="javascript:void(0);" class="ref ref22 ref23">(22,23)</a> but were later proved to be promiscuous nonspecific cysteine protease inhibitors.<a onclick="showRef(event, 'ref24 ref25'); return false;" href="javascript:void(0);" class="ref ref24 ref25">(24,25)</a> In addition, noncovalent inhibitors such as <b>ML188 (R)</b> were developed and validated as SARS-CoV M<sup>pro</sup> inhibitors.<a onclick="showRef(event, 'ref5 ref13'); return false;" href="javascript:void(0);" class="ref ref5 ref13">(5,13)</a> Several follow up studies have been conducted to optimize the enzymatic potency of this series of compounds against SARS-CoV M<sup>pro</sup> and SARS-CoV-2 M<sup>pro</sup>. However, no significant improvement has been made, and <b>ML188 (R)</b> remains the only noncovalent inhibitor with moderate antiviral activity against SARS-CoV (EC<sub>50</sub> = 12.9 ± 0.7 μM).<a onclick="showRef(event, 'ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref14 ref15">(14,15)</a> Nevertheless, given the sequence and structural similarities between SARS-CoV and SARS-CoV-2 M<sup>pro</sup>, and the similar binding mode of <b>ML188 (R)</b> and <b>calpain inhibitor XII</b>, we hypothesize that <b>ML188 (R)</b> is a promising scaffold for structure-based drug design. In this study, we developed an expedited drug discovery approach for the design of noncovalent SARS-CoV-2 M<sup>pro</sup> inhibitors. The design approach couples the superimposed X-ray crystal structures with the one-pot Ugi 4CR synthetic methodology. We were able to improve the enzymatic inhibition potency of <b>ML188</b> (<b>1</b>) by 54-fold from a focused library of 39 noncovalent inhibitors. This is a significant advantage compared to covalent inhibitors such as <b>GC376</b>, which involves at least a five-step synthesis. In addition, by introducing the chiral isocyanide, the diastereomer product can be conveniently separated by either silica gel column or reverse phase HPLC column, bypassing the need for an expensive chiral HPLC column. Target selectivity profiling showed that the noncovalent inhibitor <b>23R</b> only selectively inhibits SARS-CoV-2 and SARS-CoV M<sup>pro</sup>, but not other viral proteases and host proteases including calpain 1, cathepsins L and K, and trypsin. In contrast, the covalent M<sup>pro</sup> inhibitor <b>GC376</b> is not selective and inhibits host cysteine proteases, which might result in potential side effects. Furthermore, the X-ray crystal structure of SARS-CoV-2 M<sup>pro</sup> in complex with <b>23R</b> reveals a ligand-induced binding pocket in between S2 and S4 sites that can be explored for drug design. Overall, using the expedited drug discovery approach, this study revealed a promising noncovalent M<sup>pro</sup> inhibitor <b>23R</b> with a well characterized mechanism of action and potent cellular antiviral activity for further development.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i12">Materials and Methods</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_07513" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_07513" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Cell Lines and Viruses</h3><div class="NLM_p last">VERO E6 cells (ATCC, CRL-1586) were cultured in Dulbecco’s modified Eagle’s medium (DMEM), supplemented with 5% heat inactivated FBS in a 37 °C incubator with 5% CO<sub>2</sub>. SARS-CoV-2, isolate USA-WA1/2020 (NR-52281), was obtained through BEI Resources and propagated once on VERO E6 cells before it was used for this study. Studies involving SARS-CoV-2 were performed at the UTHSCSA biosafety level-3 laboratory by personnel wearing powered air purifying respirators.</div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Protein Expression and Purification</h3><div class="NLM_p">SARS CoV-2 main protease (M<sup>pro</sup> or 3CL) gene from strain BetaCoV/Wuhan/WIV04/2019 and SARS-CoV main protease from strain CDC#200301157 in the pET29a(+) vector with <i>E. coli</i> codon optimization were ordered from GenScript (Piscataway, NJ). The M<sup>pro</sup> gene was then subcloned into pE-SUMO vector as described previously.<a onclick="showRef(event, 'ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref7 ref8">(7,8)</a> The expression and purification of SARS-CoV and SARS-CoV-2 M<sup>pro</sup> with unmodified N- and C-termini were detailed in our previous publication.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a></div><div class="NLM_p">The expression and purification of SARS CoV-2 papain-like protease (PL<sup>pro</sup>) were also described in our previous publications.<a onclick="showRef(event, 'ref7 ref8 ref24'); return false;" href="javascript:void(0);" class="ref ref7 ref8 ref24">(7,8,24)</a></div><div class="NLM_p">The expression and purification of EV-A71 2A<sup>pro</sup> and 3C<sup>pro</sup>, EV-D68 2A<sup>pro</sup> and 3C<sup>pro</sup> were described in our previous publications.<a onclick="showRef(event, 'ref24 ref26'); return false;" href="javascript:void(0);" class="ref ref24 ref26">(24,26)</a></div><div class="NLM_p">Cathepsin K and Cathepsin L were purchased from EMD Millipore (catalog #. 219461 and 219402, respectively).</div><div class="NLM_p last">Calpain 1 and trypsin were purchased from Sigma-Aldrich (catalog #. C6108 and T6763, respectively).</div></div><div id="sec4_1_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Peptide Synthesis</h3><div class="NLM_p">The SARS-CoV-2 M<sup>pro</sup> FRET substrate Dabcyl-KTSAVLQ/SGFRKME(Edans) was synthesized as described before.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> The SARS-CoV-2 PL<sup>pro</sup>, EV-A71 2A<sup>pro</sup> and 3C<sup>pro</sup>, and EV-D68 2A<sup>pro</sup> and 3C<sup>pro</sup> FRET substrates were listed in our previous publication<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> and were synthesized by solid-phase synthesis through iterative cycles of coupling and deprotection using the previously optimized procedure.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a></div><div id="sec4_1_3_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i16" class="anchor-spacer"></div><h4 class="article-section__title" id="_i16"> General Chemical Methods</h4><div class="NLM_p">All chemicals were purchased from commercial vendors and used without further purification unless otherwise noted. <sup>1</sup>H and <sup>13</sup>C NMR spectra were recorded on a Bruker-400 or -500 NMR spectrometer. Chemical shifts are reported in parts per million referenced with respect to residual solvent (CD<sub>3</sub>OD) 3.31 ppm, (DMSO-<i>d</i><sub>6</sub>) 2.50 ppm, and (CDCl<sub>3</sub>) 7.26 ppm or from internal standard tetramethylsilane (TMS) 0.00 ppm. The following abbreviations were used in reporting spectra: s, singlet; d, doublet; t, triplet; q, quartet; m, multiplet; dd, doublet of doublets; ddd, doublet of doublet of doublets. All reactions were carried out under N<sub>2</sub> atmosphere, unless otherwise stated. HPLC-grade solvents were used for all reactions. Flash column chromatography was performed using silica gel (230–400 mesh, Merck). High resolution mass spectra were obtained using an OrbitrapTM for all the compounds, obtained in an Ion Cyclotron Resonance (ICR) spectrometer. The purity was assessed by using Shimadzu UPLC with Shimdazu C18-AQ column (4.6 × 150 mm P/N #227-30767-05) at a flow rate of 1 mL/min; λ = 254 and 220 nm; mobile phase A, 0.1% trifluoroacetic acid in H<sub>2</sub>O, and mobile phase B, 0.1% trifluoroacetic acid in 90% CH<sub>3</sub>CN and 10% H<sub>2</sub>O. The gradients are 0–2 min 10% B, 2–15 min 10%–100% B, 15–18 min, 100% B, 18.1–20 min 10% B. All compounds submitted for testing were confirmed to be >95.0% purity by HPLC traces. All final products were characterized by proton and carbon NMR, HPLC, and HRMS.</div><div id="sec4_1_3_1_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i17" class="anchor-spacer"></div><h5 class="article-section__title" id="_i17"> General Procedure for Ugi-4CR Reaction</h5><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00509/20210512/images/medium/jm1c00509_0010.gif" alt="" id="GRAPHIC-d7e2398-autogenerated" /></img><div class="NLM_p last">Ug-4CR reaction was performed according to the reported procedure with modifications.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> Amine (1.0 equiv) and aldehyde (1.0 equiv) were mixed in methanol (10 mL) and stirred at room temperature for 30 min. Then carboxylic acid (1.0 equiv) and isocyanide (1.0 equiv) were added sequentially and the resulting mixture was stirred at room temperature overnight. After that, the solvent was removed under reduced pressure and the crude product was purified with flash silica gel chromatography (methanol in dichloromethane 1–5% or acetone in hexane 30–80%).</div></div><div id="sec4_1_3_1_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i18" class="anchor-spacer"></div><h5 class="article-section__title" id="_i18"> General Procedures for the Synthesis of Compounds <b>24, 28, 33, 35</b>, and <b>37</b> by TFA Deprotection</h5><div class="NLM_p last">To a solution of N-trityl-protected Ugi-4CR compound in dichloromethane (5 mL) was added TFA (1 mL). The mixture was stirred at room temperature for 2 h, and the solvent was removed under reduced pressure. The crude mixture was diluted in dichloromethane and purified by silica gel flash column chromatography (ammonia 10% methanol in dichloromethane 10–15%) to give the final product.</div></div><div id="sec4_1_3_1_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i19" class="anchor-spacer"></div><h5 class="article-section__title" id="_i19"> Procedure for the Synthesis of Compound <b>18</b> by Suzuki–Miyamura Cross-coupling</h5><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00509/20210512/images/medium/jm1c00509_0011.gif" alt="" id="GRAPHIC-d7e2420-autogenerated" /></img><div class="NLM_p last">To solution of 2-[N-(4-bromophenyl)-1-(furan-2-yl)formamido]-N-[(1S)-1-phenylethyl]-2-(pyridin-3-yl)acetamide (1 mmol) and furan-2-boronic acid (1 mmol) in 1,4-dioxane in a microwave reaction vial was added an aqueous solution of K<sub>2</sub>CO<sub>3</sub> (4 mmol). The resulting solution was purged with N<sub>2</sub> for 5 min. The catalyst, Pd(PPh<sub>3</sub>)<sub>4</sub> (0.1 mmol), was added in one portion. The vial was capped and heated to 140 °C for 30 min with microwave irradiation. After cooling down to room temperature, the reaction solution was diluted with dichloromethane and extracted with water, followed by brine. The organic layer was dried over MgSO<sub>4</sub>, filtrated, and concentrated under reduced pressure. The crude product was purified by silica gel flash column chromatography (methanol in dichloromethane 1–5%) to give the final product.</div></div><div id="sec4_1_3_1_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i20" class="anchor-spacer"></div><h5 class="article-section__title" id="_i20"> Procedure for the Synthesis of 4-(Thiophen-2-yl)aniline by Suzuki–Miyamura Cross-coupling</h5><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00509/20210512/images/medium/jm1c00509_0012.gif" alt="" id="GRAPHIC-d7e2445-autogenerated" /></img><div class="NLM_p">The starting material 4-(thiophen-2-yl)aniline used for the synthesis of compound <b>19</b> was prepared using the following procedure. To a suspension of thiophene-2-boronic acid (1.0 mmol) and sodium carbonate (1.0 mmol) in toluene/methanol (4:1, 40 mL) was added 4-bromoaniline (1.0 mmol). The resulting solution was purged with N<sub>2</sub> for 10 min. The catalyst, Pd(PPh<sub>3</sub>)<sub>4</sub> (0.1 mmol), was added in one portion. The reaction mixture was stirred for 16 h at 100 °C. After cooling to room temperature, the reaction mixture was diluted with ethyl acetate (100 mL) and washed with water (100 mL). The organic layer was dried over MgSO<sub>4</sub>, filtrated, and concentrated under reduced pressure. The crude product was purified by silica gel flash column chromatography (ethyl acetate in hexane 20–40%) to give the final product.</div><div class="NLM_p">4-(Thiophen-2-yl)aniline. White solid. 31% yield. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 7.43–7.38 (m, 2H), 7.17–7.12 (m, 2H), 7.01 (dd, <i>J</i> = 5.0, 3.6 Hz, 1H), 6.69–6.63 (m, 2H), 3.68 (s, 2H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) δ 146.10, 145.11, 127.93, 127.25, 125.24, 123.17, 121.36, 115.39.</div><div class="NLM_p last">The isocyanide used for the synthesis of compound <b>40</b> was prepared and used in the method below:</div></div><div id="sec4_1_3_1_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i21" class="anchor-spacer"></div><h5 class="article-section__title" id="_i21"> Procedure for Synthesis of N-(3-Phenylpropyl)formamide</h5><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00509/20210512/images/medium/jm1c00509_0013.gif" alt="" id="GRAPHIC-d7e2488-autogenerated" /></img><div class="NLM_p last">3-Phenyl propylamine (2 mmol) in a microwave reaction vial was mixed with ethyl formate (6 mL). The vial was capped and heated to 70 °C overnight. After reaction residual ethyl formate was evaporated under reduced pressure. The crude product was purified by silica gel flash column chromatography (methanol in dichloromethane 1–3%) to give the product N-(3-phenylpropyl)formamide. Oil, 98% yield. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 8.15–7.96 (m, 1H), 7.33–7.24 (m, 2H), 7.24–7.12 (m, 3H), 6.18–5.73 (m, 1H), 3.35–3.17 (m, 2H), 2.69–2.60 (m, 2H), 1.89–1.78 (m, 2H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) δ 170.33, 164.88, 161.44, 141.51, 141.27, 140.65, 128.67, 128.55, 128.52, 128.42, 128.41, 126.32, 126.13, 126.07, 41.15, 39.39, 37.84, 33.36, 33.22, 32.60, 32.54, 31.17, 31.16, 23.31.</div></div><div id="sec4_1_3_1_5_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i22" class="anchor-spacer"></div><h5 class="article-section__title" id="_i22">1.  Procedure for the Synthesis of Compound <b>40</b></h5><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00509/20210512/images/medium/jm1c00509_0014.gif" alt="" id="GRAPHIC-d7e2510-autogenerated" /></img><div class="NLM_p last">To solution of N-(3-phenylpropyl)formamide (2 mmol) in dichloromethane (50 mL) was added triethylamine (4.8 mmol). The solution was cooled to approximately −10 °C using an ethanol ice bath. Triphosgene (0.4 mmol) was added to the stirring mixture. The resulting mixture was stirred at −10 °C for 10 min. In a separate 250 mL round-bottomed flask, 4-aminobiphneyl (2 mmol) and 3-pyridinecarboxaldehyde (2 mmol) were mixed in methanol (100 mL) and stirred at room temperature for 30 min. Then carboxylic acid (2 mmol) and the resulting isocyanide solution (1.0 equiv) were added sequentially, and the resulting mixture was stirred at room temperature overnight. After that, the solvent was removed under reduced pressure and the crude product was purified with flash silica gel chromatography (methanol in dichloromethane 1–5%) to give the final product.</div></div><div id="sec4_1_3_1_5_1_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i23" class="anchor-spacer"></div><h5 class="article-section__title" id="_i23">a.  N-<i>tert</i>-Butyl-2-[N-(4-<i>tert</i>-butylphenyl)-1-(furan-2-yl)formamido]-2-(pyridin-3-yl)acetamide (<b>1</b>)</h5><div class="NLM_p last">Light yellow solid, 81% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.38–8.24 (m, 2H), 7.86 (s, 1H), 7.69–7.66 (m, 1H), 7.37 (dt, <i>J</i> = 7.9, 2.0 Hz, 1H), 7.26–7.17 (m, 2H), 7.26–7.02 (br s, 1H), 7.14–7.04 (m, 2H), 6.31 (dd, <i>J</i> = 3.6, 1.7 Hz, 1H), 6.15 (s, 1H), 5.24 (d, <i>J</i> = 3.5 Hz, 1H), 1.22 (s, 9H), 1.19 (s, 9H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 168.1, 158.1, 151.1, 150.9, 148.5, 146.4, 145.2, 137.3, 136.5, 131.2, 130.8, 125.2, 122.7, 115.8, 111.2, 62.3, 50.4, 34.3, 31.0, 28.3. C<sub>26</sub>H<sub>31</sub>N<sub>3</sub>O<sub>3</sub>, HRMS calculated for m/z [M + H]<sup>+</sup>: 434.24437, found 434.2438. HPLC purity: 97%.</div></div><div id="sec4_1_3_1_5_1_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i24" class="anchor-spacer"></div><h5 class="article-section__title" id="_i24">b.  2-[N-(4-<i>tert</i>-Butylphenyl)-1-(furan-2-yl)formamido]-N-(propan-2-yl)-2-(pyridin-3-yl)acetamide (<b>2</b>)</h5><div class="NLM_p last">White solid. 82% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.37–8.28 (m, 2H), 8.10 (d, <i>J</i> = 7.5 Hz, 1H), 7.76–7.61 (m, 1H), 7.37 (dt, <i>J</i> = 7.9, 2.0 Hz, 1H), 7.30–7.17 (m, 2H), 7.24–7.00 (br s, 1H), 7.17–7.07 (m, 2H), 6.30 (dd, <i>J</i> = 3.6, 1.7 Hz, 1H), 6.15 (s, 1H), 5.26 (d, <i>J</i> = 3.6 Hz, 1H), 3.99–3.76 (m, 1H), 1.20 (s, 9H), 1.07 (d, <i>J</i> = 6.6 Hz, 3H), 0.95 (d, <i>J</i> = 6.5 Hz, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.6, 158.2, 151.1, 151.0, 148.6, 146.3, 145.2, 137.4, 136.5, 131.0, 130.7, 125.2, 122.8, 115.8, 111.2, 62.1, 40.8, 34.3, 31.0, 22.0, 22.0. C<sub>25</sub>H<sub>29</sub>N<sub>3</sub>O<sub>3</sub>, HRMS calculated for m/z [M + H]<sup>+</sup>: 420.2287 (calculated), 420.2282 (found). HPLC purity: 98%.</div></div><div id="sec4_1_3_1_5_1_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i25" class="anchor-spacer"></div><h5 class="article-section__title" id="_i25">c.  2-[N-(4-<i>tert</i>-Butylphenyl)-1-(furan-2-yl)formamido]-N-cyclopropyl-2-(pyridin-3-yl)acetamide (<b>3</b>)</h5><div class="NLM_p last">White solid. 80% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.34–8.28 (m, 3H), 7.67 (dd, <i>J</i> = 1.7, 0.7 Hz, 1H), 7.37 (dt, <i>J</i> = 8.0, 1.9 Hz, 1H), 7.26–7.19 (m, 2H), 7.25–6.94 (br s, 1H), 7.16–7.09 (m, 2H), 6.31 (dd, <i>J</i> = 3.6, 1.7 Hz, 1H), 6.09 (s, 1H), 5.28 (dd, <i>J</i> = 3.6, 0.8 Hz, 1H), 2.71–2.60 (m, 1H), 1.20 (s, 9H), 0.67–0.53 (m, 2H), 0.40–0.25 (m, 2H).<sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 169.8, 158.2, 151.1, 148.7, 146.3, 145.2, 137.4, 136.4, 130.7, 125.3, 122.8, 115.4, 111.2, 62.0, 34.3, 31.0, 22.0, 5.1. C<sub>25</sub>H<sub>27</sub>N<sub>3</sub>O<sub>3</sub>, HRMS calculated for m/z [M + H]<sup>+</sup>: 418.2131 (calculated), 418.2125 (found). HPLC purity: 96%.</div></div><div id="sec4_1_3_1_5_1_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i26" class="anchor-spacer"></div><h5 class="article-section__title" id="_i26">d.  2-[N-(4-<i>tert</i>-Butylphenyl)-1-(furan-2-yl)formamido]-2-(pyridin-3-yl)-N-(2,4,4-trimethylpentan-2-yl)acetamide (<b>4</b>)</h5><div class="NLM_p last">Light yellow solid. 60% yield. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.36–8.28 (m, 2H), 7.79–7.59 (m, 2H), 7.43–7.35 (m, 1H), 7.32-6.97 (br s, 1H), 7.30–7.18 (m, 2H), 7.12 (dd, <i>J</i> = 8.0, 4.8 Hz, 1H), 6.36–6.26 (m, 1H), 6.18 (s, 1H), 5.25 (d, <i>J</i> = 3.6 Hz, 1H), 1.67 (dd, <i>J</i> = 75.5, 14.6 Hz, 2H), 1.35–1.23 (m, 7H), 1.21 (s, 9H), 0.86 (s, 9H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.6, 158.0, 151.1, 150.9, 148.5, 146.3, 145.1, 137.4, 136.6, 131.0, 130.8, 125.2, 122.5, 115.7, 111.2, 62.4, 54.3, 50.6, 34.2, 31.1, 31.1, 31.1, 31.0, 28.9, 28.3. C<sub>30</sub>H<sub>39</sub>N<sub>3</sub>O<sub>3</sub>, HRMS calculated for m/z [M + H]<sup>+</sup>: 490.3069 (calculated), 490.3064 (found). HPLC purity: 97%.</div></div><div id="sec4_1_3_1_5_1_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i27" class="anchor-spacer"></div><h5 class="article-section__title" id="_i27">e.  2-[N-(4-<i>tert</i>-Butylphenyl)-1-(furan-2-yl)formamido]-N-cyclopentyl-2-(pyridin-3-yl)acetamide (<b>5</b>)</h5><div class="NLM_p last">White solid. 85% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.36–8.28 (m, 2H), 8.18 (d, <i>J</i> = 7.0 Hz, 1H), 7.70–7.65 (m, 1H), 7.37 (dt, <i>J</i> = 7.9, 2.0 Hz, 1H), 7.28–7.18 (m, 2H), 7.25–6.94 (br s, 1H) 7.18–7.06 (m, 2H), 6.30 (dd, <i>J</i> = 3.6, 1.7 Hz, 1H), 6.16 (s, 1H), 5.26 (d, <i>J</i> = 3.6 Hz, 1H), 4.02 (h, <i>J</i> = 6.9 Hz, 1H), 1.86–1.67 (m, 2H), 1.65–1.31 (m, 5H), 1.30–1.21 (m, 1H), 1.20 (s, 9H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 168.1, 158.2, 151.1, 151.0, 148.6, 146.3, 145.2, 137.3, 136.5, 131.0, 130.7, 125.2, 122.8, 115.8, 111.2, 62.0, 50.6, 34.3, 32.0, 31.8, 31.0, 23.5, 23.4. C<sub>27</sub>H<sub>31</sub>N<sub>3</sub>O<sub>3</sub>, HRMS calculated for m/z [M + H]<sup>+</sup>: 446.2443 (calculated), 446.2438 (found). HPLC purity: 100%.</div></div><div id="sec4_1_3_1_5_1_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i28" class="anchor-spacer"></div><h5 class="article-section__title" id="_i28">f.  2-[N-(4-<i>tert</i>-Butylphenyl)-1-(furan-2-yl)formamido]-N-cyclohexyl-2-(pyridin-3-yl)acetamide (<b>6</b>)</h5><div class="NLM_p last">White solid. 81% yield. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.36–8.30 (m, 2H), 8.09 (d, <i>J</i> = 7.7 Hz, 1H), 7.67 (dd, <i>J</i> = 1.7, 0.7 Hz, 1H), 7.38 (dt, <i>J</i> = 7.9, 2.0 Hz, 1H), 7.28–7.02 (br s, 1H), 7.25–7.19 (m, 2H), 7.17–7.10 (m, 2H), 6.31 (dd, <i>J</i> = 3.6, 1.7 Hz, 1H), 6.18 (s, 1H), 5.28 (d, <i>J</i> = 3.6 Hz, 1H), 3.63–3.52 (m, 1H), 1.86–1.40 (m, 4H), 1.32–1.14 (m, 13H), 1.14–0.92 (m, 2H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.6, 157.9, 151.1, 151.0, 148.6, 146.3, 145.2, 137.3, 136.5, 131.0, 130.7, 125.2, 122.7, 116.2, 111.2, 62.0, 47.9, 34.2, 32.0, 31.0, 25.1, 24.5, 24.3. C<sub>28</sub>H<sub>33</sub>N<sub>3</sub>O<sub>3</sub>, HRMS calculated for m/z [M + H]<sup>+</sup>: 460.2600 (calculated), 460.2595 (found). HPLC purity: 96%.</div></div><div id="sec4_1_3_1_5_1_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i29" class="anchor-spacer"></div><h5 class="article-section__title" id="_i29">g.  2-[N-(4-<i>tert</i>-Butylphenyl)-1-(furan-2-yl)formamido]-N-(4-fluorophenyl)-2-(pyridin-3-yl)acetamide (<b>7</b>)</h5><div class="NLM_p last">Brown solid. 86% yield. <sup>1</sup>H NMR (400 MHz, MeOD-<i>d</i><sub>4</sub>) δ 8.43 (d, <i>J</i> = 2.3 Hz, 1H), 8.34 (dd, <i>J</i> = 4.9, 1.6 Hz, 1H), 7.63 (dt, <i>J</i> = 8.0, 2.0 Hz, 1H), 7.59–7.53 (m, 2H), 7.52 (d, <i>J</i> = 1.8 Hz, 1H), 7.45–6.60 (br s, 2H), 7.30 (s, 2H), 7.25–7.17 (m, 1H), 7.08–6.98 (m, 2H), 6.37 (s, 1H), 6.25 (dd, <i>J</i> = 3.6, 1.7 Hz, 1H), 5.46 (d, <i>J</i> = 3.6 Hz, 1H), 1.26 (s, 9H). <sup>13</sup>C NMR (101 MHz, MeOD-<i>d</i><sub>4</sub>) δ 169.5, 161.4, 153.8, 152.2, 149.9, 147.4, 146.7, 140.3, 139.4, 137.4, 134.5, 132.2, 131.9, 127.1, 125.5, 123.1, 123.0, 118.4, 116.4, 116.2, 112.3, 65.5, 35.5, 31.6. C<sub>28</sub>H<sub>26</sub>FN<sub>3</sub>O<sub>3</sub>, HRMS calculated for m/z [M + H]<sup>+</sup>: 472.2036 (calculated), 472.2031 (found). HPLC purity: 98%.</div></div><div id="sec4_1_3_1_5_1_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i30" class="anchor-spacer"></div><h5 class="article-section__title" id="_i30">h.  2-[N-(4-<i>tert</i>-Butylphenyl)-1-(furan-2-yl)formamido]-N-(4-methoxyphenyl)-2-(pyridin-3-yl)acetamide (<b>8</b>)</h5><div class="NLM_p last">Brown solid. 79% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.22 (s, 1H), 8.42 (d, <i>J</i> = 2.3 Hz, 1H), 8.34 (dd, <i>J</i> = 4.8, 1.6 Hz, 1H), 7.72–7.67 (m, 1H), 7.54–7.46 (m, 2H), 7.43 (dt, <i>J</i> = 8.0, 2.0 Hz, 1H), 7.35–7.05 (br s, 2H), 7.27–7.21 (m, 2H), 7.17–7.10 (m, 1H), 6.92–6.83 (m, 2H), 6.32 (dd, <i>J</i> = 3.6, 1.7 Hz, 1H), 6.30 (s, 1H), 5.29 (d, <i>J</i> = 3.6 Hz, 1H), 3.71 (s, 3H), 1.20 (s, 9H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.3, 158.3, 155.3, 151.3, 151.2, 149.0, 146.2, 145.4, 137.5, 136.3, 131.9, 130.8, 130.2, 125.4, 122.9, 120.7, 116.0, 113.9, 111.8, 62.7, 55.1, 34.3, 31.0. C<sub>29</sub>H<sub>29</sub>N<sub>3</sub>O<sub>4</sub>, HRMS calculated for m/z [M + H]<sup>+</sup>: 484.2236 (calculated), 484.2231 (found). HPLC purity: 96%.</div></div><div id="sec4_1_3_1_5_1_9" class="NLM_sec NLM_sec_level_4"><div id="ac_i31" class="anchor-spacer"></div><h5 class="article-section__title" id="_i31">i.  2-[N-(4-<i>tert</i>-Butylphenyl)-1-(furan-2-yl)formamido]-N-(2,6-dimethylphenyl)-2-(pyridin-3-yl)acetamide (<b>9</b>)</h5><div class="NLM_p last">Light yellow solid. 75% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.66 (s, 1H), 8.48 (s, 1H), 8.37 (d, <i>J</i> = 4.9 Hz, 1H), 7.70 (dd, <i>J</i> = 1.7, 0.7 Hz, 1H), 7.50 (dt, <i>J</i> = 8.0, 2.0 Hz, 1H), 7.28–7.08 (br s, 2H), 7.28–7.21 (m, 2H), 7.21–7.13 (m, 1H), 7.10–6.99 (m, 3H), 6.35 (s, 1H), 6.32 (dd, <i>J</i> = 3.6, 1.7 Hz, 1H), 5.28 (dd, <i>J</i> = 3.6, 0.8 Hz, 1H), 2.07 (s, 6H), 1.21 (s, 9H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.7, 158.3, 151.4, 151.2, 149.0, 146.2, 145.3, 137.7, 136.4, 135.4, 134.6, 130.8, 130.2, 127.6, 126.5, 125.4, 122.7, 115.9, 111.2, 62.6, 34.3, 31.0, 17.9. C<sub>30</sub>H<sub>31</sub>N<sub>3</sub>O<sub>3</sub>, HRMS calculated for m/z [M + H]<sup>+</sup>: 482.2443 (calculated), 482.2438 (found). HPLC purity: 98%.</div></div><div id="sec4_1_3_1_5_1_10" class="NLM_sec NLM_sec_level_4"><div id="ac_i32" class="anchor-spacer"></div><h5 class="article-section__title" id="_i32">j.  N-[(1H-1,2,3-Benzotriazol-1-yl)methyl]-2-[N-(4-<i>tert</i>-butylphenyl)-1-(furan-2-yl)formamido]-2-(pyridin-3-yl)acetamide (<b>10</b>)</h5><div class="NLM_p last">Yellow solid. 22% yield. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 8.80 (t, <i>J</i> = 6.7 Hz, 1H), 8.50–8.46 (m, 1H), 8.40–8.36 (m, 1H), 8.10 (d, <i>J</i> = 8.3 Hz, 1H), 8.02 (d, <i>J</i> = 8.6 Hz, 1H), 7.63–7.55 (m, 1H), 7.52–7.38 (m, 3H), 7.24 (s, 1H), 7.12–6.83 (m, 3H), 6.38 (s, 1H), 6.33–6.20 (m, 3H), 5.47–5.40 (m, 2H), 1.37 (s, 9H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) δ 169.8, 159.8, 152.7, 151.6, 149.7, 146.0, 145.9, 145.3, 138.3, 136.1, 132.5, 130.1, 129.6, 127.9, 126.2, 124.4, 122.9, 119.6, 117.5, 111.3, 111.0, 63.0, 51.4, 34.7, 31.3. C<sub>29</sub>H<sub>28</sub>N<sub>6</sub>O<sub>3</sub>, HRMS calculated for m/z [M + H]<sup>+</sup>: 509.2301 (calculated), 509.2296 (found). HPLC purity: 96%.</div></div><div id="sec4_1_3_1_5_1_11" class="NLM_sec NLM_sec_level_4"><div id="ac_i33" class="anchor-spacer"></div><h5 class="article-section__title" id="_i33">k.  2-[N-(4-<i>tert</i>-Butylphenyl)-1-(furan-2-yl)formamido]-N-[(4-methylbenzenesulfonyl)methyl]-2-(pyridin-3-yl)acetamide (<b>11</b>)</h5><div class="NLM_p last">Yellow solid. 49% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.28–9.20 (m, 1H), 8.37 (dd, <i>J</i> = 4.3, 2.2 Hz, 1H), 8.23 (s, 1H), 7.69 (dd, <i>J</i> = 1.8, 0.7 Hz, 1H), 7.56–7.49 (m, 2H), 7.30–7.26 (m, 2H), 7.24–7.17 (m, 2H), 7.15–7.05 (m, 2H), 6.96 (s, 2H), 6.31 (dd, <i>J</i> = 3.6, 1.7 Hz, 1H), 6.23 (s, 1H), 5.26 (d, <i>J</i> = 3.6 Hz, 1H), 4.93 (dd, <i>J</i> = 14.1, 7.5 Hz, 1H), 4.61 (dd, <i>J</i> = 14.1, 5.6 Hz, 1H), 2.34 (s, 3H), 1.20 (s, 9H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 169.3, 158.1, 151.4, 151.2, 148.9, 146.1, 145.4, 144.4, 137.7, 136.0, 134.4, 130.7, 129.5, 128.4, 125.3, 122.6, 116.0, 111.8, 61.6, 60.1, 34.3, 30.9, 21.0. C<sub>30</sub>H<sub>31</sub>N<sub>3</sub>O<sub>5</sub>S, HRMS calculated for m/z [M + H]<sup>+</sup>: 546.2062 (calculated), 546.2057 (found). HPLC purity: 96%.</div></div><div id="sec4_1_3_1_5_1_12" class="NLM_sec NLM_sec_level_4"><div id="ac_i34" class="anchor-spacer"></div><h5 class="article-section__title" id="_i34">l.  N-Benzyl-2-[N-(4-<i>tert</i>-butylphenyl)-1-(furan-2-yl)formamido]-2-(pyridin-3-yl)acetamide (<b>12</b>)</h5><div class="NLM_p last">Light yellow solid. 49% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.74 (t, <i>J</i> = 5.9 Hz, 1H), 8.37–8.31 (m, 2H), 7.72–7.66 (m, 1H), 7.39 (dt, <i>J</i> = 8.0, 1.9 Hz, 1H), 7.33–7.18 (m, 7H), 7.21–7.05 (br s, 1H), 7.17–7.08 (m, 2H), 6.31 (dd, <i>J</i> = 3.6, 1.7 Hz, 1H), 6.22 (s, 1H), 5.28 (d, J = 4.21 Hz, 1H), 4.42–4.27 (m, 2H), 1.21 (s, 9H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) <sup>13</sup>C NMR (101 MHz, DMSO) δ 168.9, 158.2, 151.3, 151.1, 148.8, 146.3, 145.3, 139.1, 137.6, 136.5, 130.7, 128.1, 127.1, 126.7, 125.3, 122.7, 115.9, 111.2, 62.4, 42.3, 34.3, 31.0. C<sub>29</sub>H<sub>29</sub>N<sub>3</sub>O<sub>3</sub>, HRMS calculated for m/z [M + H]<sup>+</sup>: 468.2287 (calculated), 468.2282 (found). HPLC purity: 98%.</div></div><div id="sec4_1_3_1_5_1_13" class="NLM_sec NLM_sec_level_4"><div id="ac_i35" class="anchor-spacer"></div><h5 class="article-section__title" id="_i35">m.  2-[N-(4-<i>tert</i>-Butylphenyl)-1-(furan-2-yl)formamido]-N-[(1S)-1-phenylethyl]-2-(pyridin-3-yl)acetamide (<b>13</b>)</h5><div class="NLM_p last">Light yellow solid. 84% yield. <sup>1</sup>H NMR (400 MHz, DMSO–D<sub>6</sub>) δ 8.68 (dd, <i>J</i> = 7.8, 2.7 Hz, 1H), 8.41–8.23 (m, 2H), 7.67 (ddd, <i>J</i> = 5.7, 1.7, 0.7 Hz, 1H), 7.46–7.24 (m, 3H), 7.28–6.94 (br s, 1H), 7.23–7.11 (m, 5H), 7.11–7.01 (m, 2H), 6.34–6.24 (m, 2H), 5.28 (dd, <i>J</i> = 3.6, 0.8 Hz, 0.5H), 5.22 (dd, <i>J</i> = 3.6, 0.8 Hz, 0.5H), 5.04–4.91 (m, 1H), 1.37 (d, <i>J</i> = 7.0 Hz, 1.5H), 1.25 (d, <i>J</i> = 7.0 Hz, 1.5H), 1.19 (d, <i>J</i> = 6.0 Hz, 9H). <sup>13</sup>C NMR (101 MHz, DMSO) δ 168.1, 167.9, 158.2, 158.1, 151.3, 151.2, 151.0, 148.7, 146.3, 146.2, 145.2, 145.2, 144.3, 144.0, 137.5, 137.4, 136.4, 136.3, 130.8, 130.4, 128.1, 128.0, 126.6, 126.5, 126.1, 125.7, 125.2, 122.8, 122.5, 115.8, 115.8, 111.2, 111.2, 62.0, 48.3, 48.2, 34.2, 31.0, 30.9, 22.2, 22.1. C<sub>30</sub>H<sub>31</sub>N<sub>3</sub>O<sub>3</sub>, HRMS calculated for m/z [M + H]<sup>+</sup>: 482.2443 (calculated), 482.2438 (found). HPLC purity: 97%.</div></div><div id="sec4_1_3_1_5_1_14" class="NLM_sec NLM_sec_level_4"><div id="ac_i36" class="anchor-spacer"></div><h5 class="article-section__title" id="_i36">n.  2-[1-(Furan-2-yl)-N-[4-(trimethylsilyl)phenyl]formamido]-N-[(1S)-1-phenylethyl]-2-(pyridinyl)acetamide (<b>14</b>)</h5><div class="NLM_p last">Light yellow solid. 50% yield. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 8.50–8.41 (m, 2H), 7.47 (dd, <i>J</i> = 49.6, 8.2 Hz, 1H), 7.39–7.16 (m, 8H), 7.10–6.81 (m, 4H), 6.25 (d, <i>J</i> = 13.4 Hz, 1H), 6.18–6.13 (m, 1H), 5.50–5.45 (m, 0.5H), 5.44–5.39 (m, 0.5H), 5.22–5.12 (m, 1H), 1.54 (d, <i>J</i> = 6.9 Hz, 1.5H), 1.48 (d, <i>J</i> = 7.0 Hz, 1.5H), 0.23 (s, 9H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) δ 168.0, 167.7, 159.8, 159.7, 151.5, 151.4, 149.8, 149.7, 146.2, 146.2, 145.1, 145.1, 143.1, 142.1, 142.0, 139.7, 139.5, 138.4, 138.1, 134.1, 134.1, 130.2, 130.1, 130.0, 129.8, 128.7, 128.6, 127.4, 127.3, 126.3, 126.1, 122.9, 122.9, 117.4, 117.4, 111.3, 111.3, 63.4, 62.9, 49.5, 49.5, 22.0, 21.9, 0.1. C<sub>29</sub>H<sub>31</sub>N<sub>3</sub>O<sub>3</sub>Si, HRMS calculated for m/z [M + H]<sup>+</sup>: 498.2212 (calculated), 498.2220 (found). HPLC purity: 96%.</div></div><div id="sec4_1_3_1_5_1_15" class="NLM_sec NLM_sec_level_4"><div id="ac_i37" class="anchor-spacer"></div><h5 class="article-section__title" id="_i37">o.  2-{N-[4-(<i>tert</i>-Butoxy)phenyl]-1-(furan-2-yl)formamido}-N-[(1S)-1-phenylethyl]-2-(pyridin-3-yl)acetamide (<b>15</b>)</h5><div class="NLM_p last">Yellow solid. 75% yield. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.72 (t, <i>J</i> = 8.17 Hz, 1H), 8.31 (dd, J = 55, 2.4 Hz, 1H), 8.33 (ddd, <i>J</i> = 13.5, 4.8, 1.6 Hz, 1H), 7.69 (ddd, <i>J</i> = 7.0, 1.7, 0.8 Hz, 1H), 7.50-7.00 (br s, 1H), 7.42-7.06 (m, 3H), 7.43–7.32 (m, 1H), 7.31–7.12 (m, 3H), 6.76 (s, 3H), 6.34 (ddd, <i>J</i> = 8.2, 3.6, 1.7 Hz, 1H), 6.27 (d, <i>J</i> = 13.2 Hz, 1H), 5.46 (d, <i>J</i> = 3.6 Hz, 0.5H), 5.41 (d, <i>J</i> = 3.6 Hz, 0.5H), 5.05–4.94 (m, 1H), 1.38 (d, <i>J</i> = 7.1 Hz, 1.5H), 1.25 (d, <i>J</i> = 7.0 Hz, 1.5H), 1.19 (d, <i>J</i> = 9.5 Hz, 9H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 168.3, 168.1, 166.3, 158.2, 158.1, 154.7, 154.7, 151.4, 151.4, 148.7, 146.4, 146.3, 145.3, 145.3, 144.3, 144.0, 137.6, 137.6, 134.1, 134.1, 132.0, 130.7, 130.3, 128.2, 128.1, 126.7, 126.6, 126.2, 125.7, 123.7, 122.9, 122.6, 116.1, 116.0, 111.2, 111.2, 78.3, 62.0, 48.4, 48.2, 28.4, 22.3, 22.1. C<sub>30</sub>H<sub>31</sub>N<sub>3</sub>O<sub>4</sub>, HRMS calculated for m/z [M + H]<sup>+</sup>: 498.2392 (calculated), 498.2387 (found). HPLC purity: 97%.</div></div><div id="sec4_1_3_1_5_1_16" class="NLM_sec NLM_sec_level_4"><div id="ac_i38" class="anchor-spacer"></div><h5 class="article-section__title" id="_i38">p.  2-[1-(Furan-2-yl)-N-[4-(piperidin-1-yl)phenyl]formamido]-N-[(1S)-1-phenylethyl]-2-(pyridin-3-yl)acetamide (<b>16</b>)</h5><div class="NLM_p last">Light brown solid. 78% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.65 (dd, <i>J</i> = 7.8, 3.6 Hz, 1H), 8.40–8.19 (m, 2H), 7.70 (ddd, <i>J</i> = 6.3, 1.7, 0.7 Hz, 1H), 7.64-7.01 (br s, 1H), 7.48–7.29 (m, 3H), 7.28–7.03 (m, 4H), 6.68 (s, 3H), 6.35–6.28 (m, 1H), 6.25 (d, <i>J</i> = 7.8 Hz, 1H), 5.30–5.23 (m, 0.5H), 5.22–5.13 (m, 0.5H), 4.98 (h, <i>J</i> = 7.1 Hz, 1H), 3.11–3.02 (m, 4H), 1.53 (s, 6H), 1.37 (d, <i>J</i> = 7.0 Hz, 1.5H), 1.24 (d, <i>J</i> = 7.0 Hz, 1.5H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 168.3, 168.1, 158.5, 158.5, 151.3, 151.2, 150.8, 148.7, 146.4, 146.3, 145.1, 145.1, 144.3, 144.0, 137.6, 137.6, 131.7, 128.8, 128.7, 128.1, 128.0, 126.6, 126.5, 126.1, 125.7, 115.7, 115.7, 114.5, 111.3, 62.0, 54.9, 48.6, 48.3, 48.1, 30.6, 25.7, 24.9, 24.9, 24.1, 23.7, 22.3. C<sub>31</sub>H<sub>32</sub>N<sub>4</sub>O<sub>3</sub>, HRMS calculated for m/z [M + H]<sup>+</sup>: 509.2552 (calculated), 509.2547 (found). HPLC purity: 97%.</div></div><div id="sec4_1_3_1_5_1_17" class="NLM_sec NLM_sec_level_4"><div id="ac_i39" class="anchor-spacer"></div><h5 class="article-section__title" id="_i39">q.  2-[N-(4-Cyclohexylphenyl)-1-(furan-2-yl)formamido]-N-[(1S)-1-phenylethyl]-2-(pyridin-3-yl)acetamide (<b>17</b>)</h5><div class="NLM_p last">White solid. 83% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.68 (dd, <i>J</i> = 7.7, 3.4 Hz, 1H), 8.32 (dd, <i>J</i> = 10.5, 4.7 Hz, 1H), 8.31 (d, J = 48 Hz, 1H), 7.70–7.64 (m, 1H), 7.44–7.28 (m, 3H), 7.28–7.11 (m, 4H), 7.10–6.98 (m, 4H),6.33–6.24 (m, 2H), 5.25 (d, <i>J</i> = 3.6 Hz, 0.5H), 5.20 (d, <i>J</i> = 3.6 Hz, 0.5H), 5.05–4.91 (m, 1H), 2.48–2.31 (m, 1H), 1.84–1.57 (m, 5H), 1.39-1.10 (m, 5H), 1.31 (dd, J = 41.17, 7.02 Hz, 3H) <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 168.1, 167.9, 158.2, 158.1, 151.3, 151.2, 148.7, 147.8, 146.3, 146.2, 145.2, 145.2, 144.3, 144.0, 137.5, 136.6, 136.6, 131.1, 128.1, 128.0, 126.7, 126.6, 126.5, 126.1, 125.7, 115.8, 115.8, 111.2, 62.0, 48.3, 48.2, 43.1, 33.8, 33.8, 33.7, 26.1, 25.4, 22.2, 22.1. C<sub>32</sub>H<sub>33</sub>N<sub>3</sub>O<sub>3</sub>, HRMS calculated for m/z [M + H]<sup>+</sup>: 508.2600 (calculated), 508.2595 (found). HPLC purity: 98%.</div></div><div id="sec4_1_3_1_5_1_18" class="NLM_sec NLM_sec_level_4"><div id="ac_i40" class="anchor-spacer"></div><h5 class="article-section__title" id="_i40">r.  2-[1-(Furan-2-yl)-N-[4-(furan-2-yl)phenyl]formamido]-N-[(1S)-1-phenylethyl]-2-(pyridin-3-yl)acetamide (<b>18</b>)</h5><div class="NLM_p last">Light brown solid. 35% yield. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.80 (dd, <i>J</i> = 13.9, 7.8 Hz, 1H), 8.66–8.45 (m, 2H), 7.82 (dt, <i>J</i> = 8.1, 1.9 Hz, 1H), 7.71 (dddd, <i>J</i> = 24.9, 6.9, 1.8, 0.8 Hz, 2H), 7.64–7.46 (m, 3H), 7.45–7.07 (m, 7H), 6.96 (ddd, <i>J</i> = 5.9, 3.4, 0.8 Hz, 1H), 6.62–6.55 (m, 1H), 6.38 (d, <i>J</i> = 11.7 Hz, 1H), 6.35 (ddd, <i>J</i> = 8.2, 3.6, 1.7 Hz, 1H), 5.66 (dd, <i>J</i> = 2.8 Hz, 0.8 Hz, 0.5H), 5.62 (d, <i>J</i> = 3.6, 0.8 Hz, 0.5H), 5.06–4.94 (m, 1H), 1.37 (d, <i>J</i> = 7.0 Hz, 1.5H), 1.29 (d, <i>J</i> = 7.0 Hz, 1.5H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.4, 167.2, 158.2, 158.2, 151.9, 148.3, 148.2, 146.2, 146.1, 145.5, 145.5, 144.1, 143.8, 143.4, 141.3, 141.1, 138.1, 138.0, 132.4, 132.0, 131.6, 131.6, 130.0, 128.2, 128.1, 126.7, 126.6, 126.1, 125.8, 124.4, 124.1, 123.5, 116.6, 116.6, 112.2, 111.4, 111.4, 107.0, 61.9, 48.5, 48.3, 22.2, 21.9. C<sub>30</sub>H<sub>25</sub>N<sub>3</sub>O<sub>4</sub>, HRMS calculated for m/z [M + H]<sup>+</sup>: 492.1923 (calculated), 492.1918 (found). HPLC purity: 98%.</div></div><div id="sec4_1_3_1_5_1_19" class="NLM_sec NLM_sec_level_4"><div id="ac_i41" class="anchor-spacer"></div><h5 class="article-section__title" id="_i41">s.  2-[1-(Furan-2-yl)-N-[4-(thiophen-2-yl)phenyl]formamido]-N-[(1S)-1-phenylethyl]-2-(pyridin-3-yl)acetamide (<b>19</b>)</h5><div class="NLM_p last">Light yellow solid. 81% yield. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.79–8.71 (m, 1H), 8.48–8.23 (m, 2H), 7.74–7.64 (m, 1H), 7.58–7.45 (m, 4H), 7.44–7.31 (m, 3H), 7.30–7.14 (m, 4H), 7.14–7.05 (m, 3H), 6.40–6.28 (m, 2H), 5.63–5.69 (m, 0.5H), 5.58–5.54 (m, 0.5H), 5.07–4.96 (m, 1H), 1.39 (d, <i>J</i> = 7.1 Hz, 1.5H), 1.26 (d, <i>J</i> = 7.0 Hz, 1.5H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 168.1, 167.9, 158.2, 158.1, 151.4, 151.3, 148.9, 146.3, 146.2, 145.4, 145.3, 144.2, 143.9, 142.0, 138.2, 138.2, 137.6, 137.6, 133.4, 133.3, 133.3, 131.9, 130.6, 130.2, 128.6, 128.5, 128.2, 128.1, 126.6, 126.6, 126.2, 126.1, 126.1, 125.7, 125.1, 124.4, 124.4, 123.0, 122.8, 116.3, 116.3, 111.3, 111.3, 62.02, 62.00, 48.3, 48.2, 22.3, 22.1. C<sub>30</sub>H<sub>25</sub>N<sub>3</sub>O<sub>3</sub>S, HRMS calculated for m/z [M + H]<sup>+</sup>: 508.1694 (calculated), 508.1689 (found). HPLC purity: 96%.</div></div><div id="sec4_1_3_1_5_1_20" class="NLM_sec NLM_sec_level_4"><div id="ac_i42" class="anchor-spacer"></div><h5 class="article-section__title" id="_i42">t.  2-[1-(Furan-2-yl)-N-[4-(1H-pyrrol-1-yl)phenyl]formamido]-N-[(1S)-1-phenylethyl]-2-(pyridin-3-yl)acetamide (<b>20</b>)</h5><div class="NLM_p last">Yellow solid. 80% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.76 (dd, <i>J</i> = 7.8, 4.0 Hz, 1H), 8.37 (d, J = 51.9 Hz 1H), 8.34 (dd, <i>J</i> = 12.0, 4.4 Hz, 1H), 7.72–7.66 (m, 1H), 7.57–7.30 (m, 7H), 7.30–7.12 (m, 4H), 7.11–7.05 (m, 2 H), 6.38–6.30 (m, 2H), 6.23 (dt, <i>J</i> = 5.2, 2.2 Hz, 2H), 5.58 (d, <i>J</i> = 3.6 Hz, 0.5H), 5.53 (d, <i>J</i> = 3.6 Hz, 0.5H), 5.01 (h, <i>J</i> = 7.3 Hz, 1H), 1.39 (d, <i>J</i> = 7.0 Hz, 1.5H), 1.26 (d, <i>J</i> = 7.0 Hz, 1.5H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 168.3, 168.1, 158.3, 158.2, 151.5, 151.4, 148.9, 146.3, 146.3, 145.5, 145.4, 144.3, 144.0, 139.1, 139.1, 137.7, 137.6, 135.8, 135.7, 132.6, 128.2, 128.1, 126.7, 126.7, 126.2, 125.8, 118.7, 118.3, 116.4, 116.4, 111.4, 110.9, 110.9, 61.9, 54.9, 48.4, 48.3, 22.3, 22.2. C<sub>30</sub>H<sub>26</sub>N<sub>4</sub>O<sub>3</sub>, HRMS calculated for m/z [M + H]<sup>+</sup>: 491.2083 (calculated), 491.2078 (found). HPLC purity: 100%.</div></div><div id="sec4_1_3_1_5_1_21" class="NLM_sec NLM_sec_level_4"><div id="ac_i43" class="anchor-spacer"></div><h5 class="article-section__title" id="_i43">u.  2-[1-(Furan-2-yl)-N-[4-(pyridin-2-yl)phenyl]formamido]-N-[(1S)-1-phenylethyl]-2-(pyridin-3-yl)acetamide (<b>21</b>)</h5><div class="NLM_p last">Yellow solid. 88% yield. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.77 (dd, <i>J</i> = 7.8, 5.8 Hz, 1H), 8.65–8.59 (m, 1H), 8.46-8.27 (m, 2H), 7.97–7.88 (m, 3H), 7.88–7.78 (m, 1H), 7.68 (ddd, <i>J</i> = 7.6, 1.7, 0.8 Hz, 1H), 7.57–7.00 (m, 10H), 6.36 (d, <i>J</i> = 14.4 Hz, 1H), 6.32 (ddd, <i>J</i> = 9.0, 3.6, 1.7 Hz, 1H), 5.59 (d, <i>J</i> = 3.6, 0.8 Hz, 0.5H), 5.54 (d, <i>J</i> = 3.6, 0.8 Hz, 0.5H), 5.09–4.94 (m, 1H), 1.39 (d, <i>J</i> = 7.0 Hz, 1.5H), 1.27 (d, <i>J</i> = 7.0 Hz, 1.5H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 168.2, 168.0, 158.2, 158.1, 154.6, 151.4, 151.3, 149.5, 149.5, 149.0, 148.9, 146.3, 146.3, 145.4, 145.4, 144.3, 144.0, 139.9, 139.9, 138.0, 137.7, 137.7, 137.3, 137.3, 131.6, 130.6, 130.3, 128.2, 128.1, 126.7, 126.6, 126.5, 126.2, 125.8, 123.1, 122.9, 122.8, 120.3, 120.3, 116.4, 116.4, 111.43, 111.41, 62.1, 48.4, 48.3, 22.3, 22.1. C<sub>31</sub>H<sub>26</sub>N<sub>4</sub>O<sub>3</sub>, HRMS calculated for m/z [M + H]<sup>+</sup>: 503.2083 (calculated), 503.2078 (found). HPLC purity: 98%.</div></div><div id="sec4_1_3_1_5_1_22" class="NLM_sec NLM_sec_level_4"><div id="ac_i44" class="anchor-spacer"></div><h5 class="article-section__title" id="_i44">v.  2-[N-(4-Benzylphenyl)-1-(furan-2-yl)formamido]-N-[(1S)-1-phenylethyl]-2-(pyridin-3-yl)acetamide (<b>22</b>)</h5><div class="NLM_p last">White solid. 80% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.71 (dd, <i>J</i> = 7.8, 4.2 Hz, 1H), 8.44–8.23 (m, 2H), 7.66 (ddd, <i>J</i> = 5.5, 1.7, 0.7 Hz, 1H), 7.45–7.31 (m, 3H), 7.31–7.11 (m, 7H), 7.10–6.96 (m, 6H), 6.39–6.18 (m, 2H), 5.43–5.38 (m, 0.5H), 5.36–5.30 (m, 0.5H), 5.06–4.89 (m, 1H), 3.86 (d, <i>J</i> = 6.6 Hz, 2H), 1.37 (d, <i>J</i> = 7.0 Hz, 1.5H), 1.25 (d, <i>J</i> = 7.0 Hz, 1.5H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 168.1, 167.9, 158.2, 158.1, 151.4, 151.3, 148.8, 146.3, 146.3, 145.2, 145.2, 144.2, 143.9, 141.1, 140.9, 137.5, 137.1, 137.0, 131.3, 130.7, 130.3, 129.2, 128.4, 128.4, 128.3, 128.3, 128.2, 128.0, 126.6, 126.6, 126.1, 125.9, 125.9, 125.7, 122.9, 122.6, 116.0, 116.0, 111.1, 62.0, 48.3, 48.2, 40.2, 22.3, 22.1. C<sub>33</sub>H<sub>29</sub>N<sub>3</sub>O<sub>3</sub>, HRMS calculated for m/z [M + H]<sup>+</sup>: 516.2287 (calculated), 516.2282 (found). HPLC purity: 98%.</div></div><div id="sec4_1_3_1_5_1_23" class="NLM_sec NLM_sec_level_4"><div id="ac_i45" class="anchor-spacer"></div><h5 class="article-section__title" id="_i45">w.  2-(N-{[1,1′-Biphenyl]-4-yl}-1-(furan-2-yl)formamido)-N-[(1S)-1-phenylethyl]-2-(pyridin-3-yl)acetamide (<b>23</b>)</h5><div class="NLM_p last">White solid. 89% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.75 (dd, <i>J</i> = 7.8, 5.0 Hz, 1H), 8.53–8.21 (m, 2H), 7.72–7.66 (m, 1H), 7.66–7.58 (m, 2H), 7.58–7.24 (m, 10H), 7.24–7.05 (m, 4H), 6.47–6.23 (m, 2H), 5.63–5.53 (m, 0.5H), 5.52–5.43 (m, 0.5H), 5.10–4.92 (m, 1H), 1.39 (d, <i>J</i> = 7.0 Hz, 1.5H), 1.27 (d, <i>J</i> = 7.0 Hz, 1.5H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 168.1, 167.9, 158.2, 158.1, 151.3, 151.3, 148.9, 146.3, 146.2, 145.3, 145.3, 144.2, 143.9, 139.4, 138.55, 138.50, 138.4, 137.6, 131.8, 128.95, 128.93, 128.2, 128.0, 127.8, 126.6, 126.6, 126.5, 126.4, 126.1, 125.7, 116.2, 116.2, 111.3, 62.1, 48.4, 48.3, 22.2, 22.1. C<sub>32</sub>H<sub>27</sub>N<sub>3</sub>O<sub>3</sub>, HRMS calculated for m/z [M + H]<sup>+</sup>: 502.2130 (calculated), 502.2132 (found). HPLC purity: 97%.</div></div><div id="sec4_1_3_1_5_1_24" class="NLM_sec NLM_sec_level_4"><div id="ac_i46" class="anchor-spacer"></div><h5 class="article-section__title" id="_i46">x.  (2R)-2-(N-{[1,1′-Biphenyl]-4-yl}-1-(furan-2-yl)formamido)-N-[(1S)-1-phenylethyl]-2-(pyridin- 3-yl)acetamide (<b>23R</b>, <b>Jun8-76-3A</b>)</h5><div class="NLM_p last">White solid, 45% yield. <sup>1</sup>H NMR (500 MHz, DMSO-d6) δ 8.74 (d, J = 7.9 Hz, 1H), 8.43 (d, J = 2.3 Hz, 1H), 8.35 (dd, J = 4.8, 1.6 Hz, 1H), 7.71–7.66 (m, 1H), 7.66–7.61 (m, 2H), 7.60–7.52 (m, 2H), 7.50 (dt, J = 8.0, 2.0 Hz, 1H), 7.46–7.32 (m, 8H), 7.30–7.15 (m, 3H), 6.36 (s, 1H), 6.32 (dd, J = 3.6, 1.7 Hz, 1H), 5.50 (d, J = 3.6 Hz, 1H), 5.02 (p, J = 7.1 Hz, 1H), 1.27 (d, J = 7.0 Hz, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-d6) δ 168.1, 158.1, 151.3, 148.9, 146.2, 145.3, 143.9, 139.4, 138.5, 138.5, 137.6, 131.8, 130.7, 128.9, 128.2, 127.8, 126.6, 126.5, 126.4, 126.1, 123.0, 116.2, 111.3, 62.1, 48.2, 22.1. C<sub>32</sub>H<sub>27</sub>N<sub>3</sub>O<sub>3</sub>, HRMS calculated for m/z [M + H]<sup>+</sup>: 502.2130 (calculated), 502.2132 (found). HPLC purity: 100%.</div></div><div id="sec4_1_3_1_5_1_25" class="NLM_sec NLM_sec_level_4"><div id="ac_i47" class="anchor-spacer"></div><h5 class="article-section__title" id="_i47">y.  (2S)-2-(N-{[1,1′-<b>b</b>iphenyl]-4-yl}-1-(furan-2-yl)formamido)-N-[(1S)-1-phenylethyl]-2-(pyridin-3- yl)acetamide (<b>23S</b>)</h5><div class="NLM_p last">White solid, 45% yield. <sup>1</sup>H NMR (500 MHz, DMSO-d6) δ 8.75 (d, J = 7.6 Hz, 1H), 8.38–8.25 (m, 2H), 7.73–7.66 (m, 1H), 7.66–7.59 (m, 2H), 7.53 (d, J = 8.0 Hz, 2H), 7.47–7.31 (m, 4H), 7.30–7.13 (m, 5H), 7.12–7.06 (m, 3H), 6.36–6.32 (m, 2H), 5.54 (d, J = 3.6 Hz, 1H), 5.00 (p, J = 7.1 Hz, 1H), 1.38 (d, J = 7.0 Hz, 3H). <sup>13</sup>C NMR (126 MHz, DMSO) δ 168.0, 158.2, 151.4, 148.9, 146.3, 145.4, 144.3, 139.4, 138.5, 138.5, 137.6, 131.8, 130.3, 128.9, 128.1, 127.8, 126.6, 126.5, 126.5, 125.8, 122.7, 116.3, 111.4, 62.1, 48.4, 22.3. C<sub>32</sub>H<sub>27</sub>N<sub>3</sub>O<sub>3</sub>, HRMS calculated for m/z [M + H]<sup>+</sup>: 502.2130 (calculated), 502.2132 (found). HPLC purity: 97%.</div></div><div id="sec4_1_3_1_5_1_26" class="NLM_sec NLM_sec_level_4"><div id="ac_i48" class="anchor-spacer"></div><h5 class="article-section__title" id="_i48">z.  2-(N-{[1,1′-Biphenyl]-4-yl}-1-(1H-imidazol-4-yl)formamido)-N-[(1S)-1-phenylethyl]-2-(pyridin-3-yl)acetamide (<b>24</b>)</h5><div class="NLM_p last">White solid. 38% yield. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.76 (s, 1H), 8.74 (d, <i>J</i> = 7.9 Hz, 1H), 8.44 (d, <i>J</i> = 2.4 Hz, 1H), 8.36 (dd, <i>J</i> = 4.8, 1.7 Hz, 1H), 7.73–7.32 (br s, 1H), 7.68–7.60 (m, 3H), 7.60–7.49 (m, 3H), 7.48–7.23 (m, 9H), 7.19 (ddd, <i>J</i> = 7.8, 4.8, 0.8 Hz, 1H), 6.40 (s, 1H), 5.53 (s, 1H), 5.03 (p, <i>J</i> = 7.2 Hz, 1H), 1.26 (d, <i>J</i> = 7.0 Hz, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 168.3, 151.3, 148.8, 144.0, 139.6, 138.6, 137.6, 137.0, 132.1, 130.9, 128.9, 128.2, 127.8, 126.6, 126.5, 126.1, 123.0, 62.0, 48.2, 22.2. C<sub>31</sub>H<sub>27</sub>N<sub>5</sub>O<sub>2</sub>, HRMS calculated for m/z [M + H]<sup>+</sup>: 502.2242 (calculated), 502.2238 (found). HPLC purity: 96%.</div></div><div id="sec4_1_3_1_5_1_27" class="NLM_sec NLM_sec_level_4"><div id="ac_i49" class="anchor-spacer"></div><h5 class="article-section__title" id="_i49">aa.  2-(N-{[1,1′-Biphenyl]-4-yl}-1-(1,2-oxazol-5-yl)formamido)-N-[(1S)-1-phenylethyl]-2-(pyridin-3-yl)acetamide (<b>25</b>)</h5><div class="NLM_p last">Light yellow solid. 82% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.83 (dd, <i>J</i> = 7.7, 5.7 Hz, 1H), 8.56–8.26 (m, 3H), 7.65–7.45 (m, 5H), 7.44–7.06 (m, 11H), 6.35 (d, <i>J</i> = 11.2 Hz, 1H), 5.88–5.77 (m, 1H), 5.13–4.94 (m, 1H), 1.40 (d, <i>J</i> = 7.0 Hz, 1.5H), 1.28 (d, <i>J</i> = 7.0 Hz, 1.5H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.5, 167.3, 161.7, 161.6, 157.0, 157.0, 151.4, 151.3, 150.7, 149.1, 144.1, 143.8, 139.7, 138.4, 137.6, 137.4, 137.3, 131.4, 130.1, 129.7, 128.9, 128.9, 128.2, 128.1, 127.9, 126.7, 126.7, 126.56, 126.54, 126.1, 125.8, 123.1, 122.8, 107.11, 107.10, 62.3, 48.5, 48.4, 22.3, 22.1. C<sub>31</sub>H<sub>26</sub>N<sub>4</sub>O<sub>3</sub>, HRMS calculated for m/z [M + H]<sup>+</sup>: 503.2083 (calculated), 503.2078 (found). HPLC purity: 98%.</div></div><div id="sec4_1_3_1_5_1_28" class="NLM_sec NLM_sec_level_4"><div id="ac_i50" class="anchor-spacer"></div><h5 class="article-section__title" id="_i50">ab.  2-(N-{[1,1′-Biphenyl]-4-yl}-1-(1,3-oxazol-5-yl)formamido)-N-[(1S)-1-phenylethyl]-2-(pyridin-3-yl)acetamide (<b>26</b>)</h5><div class="NLM_p last">Yellow solid. 85% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.80 (dd, <i>J</i> = 7.8, 4.7 Hz, 1H), 8.57–8.23 (m, 3H), 7.69–7.49 (m, 5H), 7.48–7.32 (m, 6H), 7.31–7.07 (m, 5H), 6.34 (d, <i>J</i> = 9.3 Hz, 1H), 5.87 (d, <i>J</i> = 16.3 Hz, 1H), 5.02 (h, <i>J</i> = 7.6 Hz, 1H), 1.40 (d, <i>J</i> = 7.0 Hz, 1.5H), 1.27 (d, <i>J</i> = 7.0 Hz, 1.5H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.8, 167.6, 156.9, 156.8, 153.69, 153.65, 151.4, 151.3, 149.1, 144.2, 144.24, 144.21, 143.9, 140.1, 138.4, 137.6, 137.6, 137.5, 137.4, 131.8, 130.4, 130.3, 129.9, 128.9, 128.9, 128.2, 128.1, 127.9, 126.8, 126.7, 126.65, 126.64, 126.1, 125.7, 123.1, 122.8, 62.1, 48.4, 48.3, 22.2, 22.1. C<sub>31</sub>H<sub>26</sub>N<sub>4</sub>O<sub>3</sub>, HRMS calculated for m/z [M + H]<sup>+</sup>: 503.2083 (calculated), 503.2078 (found). HPLC purity: 97%.</div></div><div id="sec4_1_3_1_5_1_29" class="NLM_sec NLM_sec_level_4"><div id="ac_i51" class="anchor-spacer"></div><h5 class="article-section__title" id="_i51">ac.  N-<i>tert</i>-Butyl-2-[N-(4-<i>tert</i>-butylphenyl)-1-(furan-2-yl)formamido]-2-(pyrazin-2-yl)acetamide (<b>27</b>)</h5><div class="NLM_p last">Light yellow solid. 86% yield. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.53 (dd, <i>J</i> = 2.5, 1.5 Hz, 1H), 8.47 (d, <i>J</i> = 1.5 Hz, 1H), 8.43 (d, <i>J</i> = 2.6 Hz, 1H), 8.00 (s, 1H), 7.68 (dd, <i>J</i> = 1.7, 0.7 Hz, 1H), 7.35–7.16 (m, 4H), 6.33 (dd, <i>J</i> = 3.6, 1.7 Hz, 1H), 6.28 (s, 1H), 5.39 (d, <i>J</i> = 3.6 Hz, 1H), 1.22 (s, 9H), 1.14 (s, 9H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.0, 158.3, 152.0, 151.0, 146.2, 145.3, 145.2, 143.6, 143.0, 136.9, 130.2, 125.2, 116.0, 111.2, 65.1, 50.5, 34.3, 31.0, 28.1. C<sub>25</sub>H<sub>30</sub>N<sub>4</sub>O<sub>3</sub>, HRMS calculated for m/z [M + H]<sup>+</sup>: 435.2396 (calculated), 435.2391 (found). HPLC purity: 96%.</div></div><div id="sec4_1_3_1_5_1_30" class="NLM_sec NLM_sec_level_4"><div id="ac_i52" class="anchor-spacer"></div><h5 class="article-section__title" id="_i52">ad.  N-<i>tert</i>-Butyl-2-[N-(4-<i>tert</i>-butylphenyl)-1-(furan-2-yl)formamido]-2-(1H-imidazol-4-yl)acetamide (<b>28</b>)</h5><div class="NLM_p last">Light yellow solid. 60% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.91 (d, <i>J</i> = 1.3 Hz, 1H), 8.02 (s, 1H), 7.73–7.66 (m, 1H), 7.36–7.29 (m, 2H), 7.23 (dd, <i>J</i> = 1.3, 0.7 Hz, 3H), 6.34 (dd, <i>J</i> = 3.6, 1.7 Hz, 1H), 6.27 (s, 1H), 5.34 (d, <i>J</i> = 3.6 Hz, 1H), 1.24 (s, 9H), 1.23 (s, 9H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.4, 158.2, 151.4, 146.0, 145.5, 136.4, 134.5, 129.9, 128.3, 125.5, 119.6, 116.2, 111.4, 56.2, 50.7, 34.4, 31.0, 28.2. C<sub>24</sub>H<sub>30</sub>N<sub>4</sub>O<sub>3</sub>, HRMS calculated for m/z [M + H]<sup>+</sup>: 423.2396 (calculated), 423.2391 (found). HPLC purity: 97%.</div></div><div id="sec4_1_3_1_5_1_31" class="NLM_sec NLM_sec_level_4"><div id="ac_i53" class="anchor-spacer"></div><h5 class="article-section__title" id="_i53">ae.  N-<i>tert</i>-Butyl-2-[N-(4-<i>tert</i>-butylphenyl)-1-(furan-2-yl)formamido]-2-(pyrimidin-5-yl)acetamide (<b>29</b>)</h5><div class="NLM_p last">Yellow solid. 78% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.93 (s, 1H), 8.46 (s, 2H), 7.95 (s, 1H), 7.68 (dd, <i>J</i> = 1.7, 0.8 Hz, 1H), 7.33-7.12 (br s, 2H)7.27 (d, <i>J</i> = 8.0 Hz, 2H), 6.32 (dd, <i>J</i> = 3.6, 1.7 Hz, 1H), 6.16 (s, 1H), 5.32 (d, <i>J</i> = 3.6 Hz, 1H), 1.22 (s, 9H), 1.21 (s, 9H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.1, 158.1, 157.8, 157.2, 151.3, 146.2, 145.3, 136.4, 130.7, 129.6, 125.5, 116.0, 111.3, 60.5, 50.5, 34.3, 30.9, 28.2. C<sub>25</sub>H<sub>30</sub>N<sub>4</sub>O<sub>3</sub>, HRMS calculated for m/z [M + H]<sup>+</sup>: 435.2396 (calculated), 435.2391 (found). HPLC purity: 98%.</div></div><div id="sec4_1_3_1_5_1_32" class="NLM_sec NLM_sec_level_4"><div id="ac_i54" class="anchor-spacer"></div><h5 class="article-section__title" id="_i54">af.  2-(N-{[1,1′-Biphenyl]-4-yl}-1-(furan-2-yl)formamido)-N-cyclopropyl-2-(pyridin-3-yl)acetamide (<b>30</b>)</h5><div class="NLM_p last">Light yellow solid. 89% yield. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.46–8.26 (m, 3H), 7.69 (dd, <i>J</i> = 1.7, 0.8 Hz, 1H), 7.66–7.62 (m, 2H), 7.62–7.50 (m, 2H), 7.49-7.14 (br s, 1H), 7.48–7.31 (m, 5H), 7.17 (dd, <i>J</i> = 7.9, 4.8 Hz, 1H), 6.34 (dd, <i>J</i> = 3.6, 1.7 Hz, 1H), 6.18 (s, 1H), 5.55 (dd, <i>J</i> = 3.6, 0.8 Hz, 1H), 2.74–2.65 (m, 1H), 0.73–0.55 (m, 2H), 0.46–0.26 (m, 2H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 169.9, 158.2, 151.1, 148.9, 146.3, 145.4, 139.5, 138.5, 138.5, 137.5, 131.7, 130.6, 128.9, 127.8, 126.5, 126.5, 123.0, 116.3, 111.3, 62.0, 22.5, 5.6, 5.5. C<sub>27</sub>H<sub>23</sub>N<sub>3</sub>O<sub>3</sub>, HRMS calculated for m/z [M + H]<sup>+</sup>: 438.1817 (calculated), 438.1812 (found). HPLC purity: 98%.</div></div><div id="sec4_1_3_1_5_1_33" class="NLM_sec NLM_sec_level_4"><div id="ac_i55" class="anchor-spacer"></div><h5 class="article-section__title" id="_i55">ag.  2-(N-{[1,1′-Biphenyl]-4-yl}-1-(furan-2-yl)formamido)-2-(pyridin-3-yl)-N-(2,4,4-trimethylpentan-2-yl)acetamide (<b>31</b>)</h5><div class="NLM_p last">Light yellow solid. 62% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.48–8.22 (m, 2H), 7.75–7.66 (m, 2H), 7.66–7.60 (m, 2H), 7.60–7.49 (m, 2H), 7.51-7.09 (br s, 1H), 7.49–7.30 (m, 5H), 7.15 (ddd, <i>J</i> = 7.9, 4.8, 0.8 Hz, 1H), 6.32 (dd, <i>J</i> = 3.6, 1.7 Hz, 1H), 6.25 (s, 1H), 5.49 (dd, <i>J</i> = 3.6, 0.8 Hz, 1H), 1.68 (dd, <i>J</i> = 75.4, 14.5 Hz, 2H), 1.29 (d, <i>J</i> = 16.4 Hz, 6H), 0.87 (s, 9H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.7, 158.0, 151.3, 148.6, 146.3, 145.2, 139.4, 138.6, 138.6, 137.6, 131.9, 130.9, 128.9, 127.8, 126.5, 126.4, 122.8, 116.1, 111.3, 62.4, 50.7, 31.2, 31.1, 29.0, 28.4. C<sub>32</sub>H<sub>35</sub>N<sub>3</sub>O<sub>3</sub>, HRMS calculated for m/z [M + H]<sup>+</sup>: 510.2756 (calculated), 510.2751 (found). HPLC purity: 100%.</div></div><div id="sec4_1_3_1_5_1_34" class="NLM_sec NLM_sec_level_4"><div id="ac_i56" class="anchor-spacer"></div><h5 class="article-section__title" id="_i56">ah.  2-(N-{[1,1′-Biphenyl]-4-yl}-1-(furan-2-yl)formamido)-N-cyclopentyl-2-(pyridin-3-yl)acetamide (<b>32</b>)</h5><div class="NLM_p last">White solid. 78% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.35 (d, <i>J</i> = 20.6 Hz, 2H), 8.24 (d, <i>J</i> = 7.0 Hz, 1H), 7.68 (dd, <i>J</i> = 1.7, 0.7 Hz, 1H), 7.66–7.60 (m, 2H), 7.55 (d, <i>J</i> = 8.1 Hz, 2H), 7.51-7.12 (br s, 1H), 7.50–7.29 (m, 5H), 7.16 (dd, <i>J</i> = 7.9, 4.7 Hz, 1H), 6.33 (dd, <i>J</i> = 3.6, 1.7 Hz, 1H), 6.24 (s, 1H), 5.53 (d, <i>J</i> = 3.6 Hz, 1H), 4.06 (h, <i>J</i> = 6.6 Hz, 1H), 1.90–1.70 (m, 2H), 1.68–1.37 (m, 5H), 1.33–1.19 (m, 1H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 168.2, 158.2, 151.2, 148.8, 146.3, 145.3, 139.4, 138.5, 137.5, 131.7, 130.9, 128.9, 127.8, 126.5, 126.4, 123.0, 116.2, 111.3, 62.0, 50.7, 32.1, 31.8, 23.5, 23.4. C<sub>29</sub>H<sub>27</sub>N<sub>3</sub>O<sub>3</sub>, HRMS calculated for m/z [M + H]<sup>+</sup>: 466.2130 (calculated), 466.2125 (found). HPLC purity: 96%.</div></div><div id="sec4_1_3_1_5_1_35" class="NLM_sec NLM_sec_level_4"><div id="ac_i57" class="anchor-spacer"></div><h5 class="article-section__title" id="_i57">ai.  2-(N-{[1,1′-Biphenyl]-4-yl}-1-(1H-imidazol-4-yl)formamido)-N-cyclopentyl-2-(pyridin-3-yl)acetamide (<b>33</b>)</h5><div class="NLM_p last">White solid. 44% yield. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.38 (d, <i>J</i> = 2.3 Hz, 1H), 8.33 (dd, <i>J</i> = 4.8, 1.7 Hz, 1H), 8.24 (d, <i>J</i> = 7.1 Hz, 1H), 7.82-7.32 (br s, 1H), 7.73–7.55 (m, 5H), 7.53–7.29 (m, 5H), 7.23–7.12 (m, 1H), 6.24 (s, 1H), 5.54 (s, 1H), 4.12–4.00 (m, 1H) 1.90–1.70 (m, 2H), 1.67–1.40 (m, 5H), 1.32–1.19 (m, 1H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 168.4, 151.2, 148.8, 139.6, 138.7, 138.6, 137.5, 132.1, 131.1, 129.0, 127.9, 126.6, 123.0, 62.0, 50.7, 32.1, 31.9, 23.5, 23.4. C<sub>28</sub>H<sub>27</sub>N<sub>5</sub>O<sub>2</sub>, HRMS calculated for m/z [M + H]<sup>+</sup>: 466.2242 (calculated), 466.2238 (found). HPLC purity: 98%.</div></div><div id="sec4_1_3_1_5_1_36" class="NLM_sec NLM_sec_level_4"><div id="ac_i58" class="anchor-spacer"></div><h5 class="article-section__title" id="_i58">aj.  2-(N-{[1,1′-Biphenyl]-4-yl}-1-(furan-2-yl)formamido)-N-cyclohexyl-2-(pyridin-3-yl)acetamide (<b>34</b>)</h5><div class="NLM_p last">White solid. 83% yield. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3/</sub> MeOD-<i>d</i><sub>4</sub>) δ 8.43 (s, 1H), 8.38–8.31 (m, 1H), 7.55–7.48 (m, 3H), 7.48–7.27 (m, 6H), 7.26–6.95 (br s, 1H), 7.16–7.05 (m, 2H), 6.22 (s, 1H), 6.16–6.08 (m, 1H), 5.55 (d, <i>J</i> = 3.6 Hz, 1H), 3.79 (s, 1H), 3.77–3.69 (m, 1H), 2.04–1.46 (m, 5H), 1.43–0.9 (m, 5H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3/</sub> MeOD-<i>d</i><sub>4</sub>) δ 168.1, 168.0, 159.8, 150.9, 148.9, 145.9, 145.1, 141.5, 139.3, 138.5, 137.9, 131.3, 130.8, 128.8, 127.8, 127.4, 126.8, 123.3, 117.5, 111.3, 62.7, 62.6, 32.5, 32.4, 25.3, 24.8, 24.7. C<sub>30</sub>H<sub>29</sub>N<sub>3</sub>O<sub>3</sub>, HRMS calculated for m/z [M + H]<sup>+</sup>: 480.2287 (calculated), 480.2282 (found). HPLC purity: 97%.</div></div><div id="sec4_1_3_1_5_1_37" class="NLM_sec NLM_sec_level_4"><div id="ac_i59" class="anchor-spacer"></div><h5 class="article-section__title" id="_i59">ak.  2-(N-{[1,1′-Biphenyl]-4-yl}-1-(1H-imidazol-4-yl)formamido)-N-cyclohexyl-2-(pyridin-3-yl)acetamide (<b>35</b>)</h5><div class="NLM_p last">White solid. 42% yield. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.40 (d, <i>J</i> = 2.3 Hz, 1H), 8.33 (dd, <i>J</i> = 4.8, 1.7 Hz, 1H), 8.17 (d, <i>J</i> = 7.7 Hz, 1H), 7.89 (s, 1H), 7.69–7.63 (m, 2H), 7.59 (d, <i>J</i> = 7.2 Hz, 2H), 7.54-6.99 (br s, 1 H), 7.51–7.32 (m, 5H), 7.17 (ddd, <i>J</i> = 7.9, 4.8, 0.8 Hz, 1H), 6.26 (s, 1H), 5.56 (s, 1H), 3.67-3.57 (m, 1H), 1.86–1.48 (m, 5H), 1.35–1.16 (m, 3H), 1.15–0.96 (m, 2H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.8, 159.8, 151.2, 148.8, 139.8, 138.5, 138.4, 137.6, 136.8, 132.0, 130.9, 129.0, 127.9, 126.7, 126.6, 123.0, 62.1, 48.0, 32.1, 25.1, 24.5, 24.4. C<sub>29</sub>H<sub>29</sub>N<sub>5</sub>O<sub>2</sub>, HRMS calculated for m/z [M + H]<sup>+</sup>: 480.2399 (calculated), 480.2394 (found). HPLC purity: 96%.</div></div><div id="sec4_1_3_1_5_1_38" class="NLM_sec NLM_sec_level_4"><div id="ac_i60" class="anchor-spacer"></div><h5 class="article-section__title" id="_i60">al.  2-[N-(4-Cyclohexylphenyl)-1-(1,2-oxazol-5-yl)formamido]-N-[(1S)-1-phenylethyl]-2-(pyridin-3-yl)acetamide (<b>36</b>)</h5><div class="NLM_p last">Yellow solid. 76% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.69 (dd, <i>J</i> = 7.8, 3.1 Hz, 1H), 8.32 (ddd, <i>J</i> = 10.3, 4.8, 1.6 Hz, 1H), 8.31 (dd, J = 46.3, 2.2 Hz, 1H), 7.74–7.60 (m, 1H), 7.46–7.29 (m, 3H), 7.28–6.96 (m, 7H), 6.35–6.19 (m, 2H), 5.26–5.22 (m, 0.5H), 5.21–5.15 (m, 0.5H), 5.08–4.87 (m, 1H), 2.48–2.34 (m, 1H), 1.87–1.59 (m, 5H), 1.47–1.07 (m, 5H), 1.31 (dd, J = 47.0 7.0 Hz, 3 H) <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 168.2, 167.9, 158.2, 158.1, 151.3, 151.3, 148.8, 147.8, 146.3, 146.2, 145.3, 145.2, 144.3, 144.0, 137.5, 136.6, 136.6, 131.1, 130.7, 130.4, 128.2, 128.0, 126.7, 126.6, 126.6, 126.1, 125.7, 122.8, 122.5, 115.8, 115.8, 111.2, 62.0, 54.9, 48.3, 48.2, 43.1, 33.8, 33.8, 33.7, 26.1, 25.4, 22.3, 22.1. C<sub>31</sub>H<sub>32</sub>N<sub>4</sub>O<sub>3</sub>, HRMS calculated for m/z [M + H]<sup>+</sup>: 509.2552 (calculated), 509.2547 (found). HPLC purity: 97%.</div></div><div id="sec4_1_3_1_5_1_39" class="NLM_sec NLM_sec_level_4"><div id="ac_i61" class="anchor-spacer"></div><h5 class="article-section__title" id="_i61">am.  2-(N-{[1,1′-Biphenyl]-4-yl}-1-(1H-imidazol-4-yl)formamido)-N-[(1S)-1-phenylethyl]-2-(pyrazin-2-yl)acetamide (<b>37</b>)</h5><div class="NLM_p last">Yellow solid. 42% yield. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.89–8.37 (m, 5H), 7.70–7.61 (m, 3H), 7.57 (dd, <i>J</i> = 16.7, 8.1 Hz, 2H), 7.52–7.41 (m, 3H), 7.40–7.29 (m, 4H), 7.29–7.16 (m, 3H), 6.61–6.47 (m, 1H), 5.07–4.93 (m, 1H), 1.33 (d, <i>J</i> = 7.0 Hz, 1.5H), 1.30 (d, <i>J</i> = 7.0 Hz, 1.5H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.7, 166.5, 151.4, 151.3, 146.1, 146.0, 144.2, 144.1, 143.8, 143.7, 143.4, 138.7, 131.6, 131.5, 129.0, 128.2, 128.1, 127.9, 126.6, 126.6, 126.1, 126.0, 64.5, 48.4, 48.3, 22.2, 22.2. C<sub>30</sub>H<sub>26</sub>N<sub>6</sub>O<sub>2</sub>, HRMS calculated for m/z [M + H]<sup>+</sup>: 503.2195 (calculated), 503.2190 (found). HPLC purity: 99%.</div></div><div id="sec4_1_3_1_5_1_40" class="NLM_sec NLM_sec_level_4"><div id="ac_i62" class="anchor-spacer"></div><h5 class="article-section__title" id="_i62">an.  2-(N-{[1,1′-Biphenyl]-4-yl}-1-(furan-2-yl)formamido)-N-[(1S)-1-phenylethyl]-2-(pyrazin-2-yl)acetamide (<b>38</b>)</h5><div class="NLM_p last">Yellow solid. 83% yield. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.79 (dd, <i>J</i> = 24.5, 7.8 Hz, 1H), 8.68–8.34 (m, 3H), 7.72–7.67 (m, 1H), 7.66–7.61 (m, 2H), 7.60–7.51 (m, 2H), 7.49-7.26 (br s, 1H), 7.48–7.41 (m, 2H), 7.39–7.30 (m, 4H), 7.29–7.16 (m, 3H), 6.56–6.46 (m, 1H), 6.40–6.31 (m, 1H), 5.66–5.64 (m, 0.5H), 5.62–5.59 (m, 0.5H), 5.10–4.90 (m, 1H), 1.39–1.24 (m, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.5, 166.3, 158.3, 158.3, 151.2, 151.1, 146.2, 146.1, 146.0, 145.5, 145.4, 144.1, 144.0, 143.8, 143.7, 143.4, 139.6, 139.5, 138.8, 138.8, 138.6, 138.6, 131.2, 131.1, 129.0, 128.2, 128.1, 127.9, 126.7, 126.6, 126.1, 126.0, 116.6, 116.5, 111.4, 111.4, 64.5, 64.4, 48.5, 48.4, 22.2. C<sub>31</sub>H<sub>26</sub>N<sub>4</sub>O<sub>3</sub>, HRMS calculated for m/z [M + H]<sup>+</sup>: 503.2083 (calculated), 503.2078 (found). HPLC purity: 100%.</div></div><div id="sec4_1_3_1_5_1_41" class="NLM_sec NLM_sec_level_4"><div id="ac_i63" class="anchor-spacer"></div><h5 class="article-section__title" id="_i63">ao.  2-(N-{[1,1′-Biphenyl]-4-yl}-1-(furan-2-yl)formamido)-N-(2,2-diphenylethyl)-2-(pyrazin-2-yl)acetamide (<b>39</b>)</h5><div class="NLM_p last">Yellow solid. 65% yield. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.39–8.19 (m, 3H), 7.69 (dd, <i>J</i> = 1.7, 0.7 Hz, 1H), 7.67–7.60 (m, 2H), 7.58–7.50 (m, 2H), 7.46–7.42 (m, 2H), 7.38–7.33 (m, 1H), 7.32–7.10 (m, 12H), 7.05 (dt, <i>J</i> = 7.9, 2.0 Hz, 1H), 6.96 (dd, <i>J</i> = 8.0, 4.8 Hz, 1H), 6.33 (dd, <i>J</i> = 3.6, 1.7 Hz, 1H), 6.21 (s, 1H), 5.52 (dd, <i>J</i> = 3.6, 0.8 Hz, 1H), 4.23 (t, <i>J</i> = 7.9 Hz, 1H), 4.06–3.90 (m, 1H), 3.73–3.57 (m, 1H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 168.7, 158.2, 151.4, 148.5, 146.3, 145.3, 142.7, 142.6, 139.4, 138.5, 138.5, 137.4, 131.6, 130.5, 129.0, 128.4, 128.3, 127.9, 127.8, 127.8, 126.5, 126.5, 126.3, 126.3, 122.7, 116.3, 111.3, 62.1, 50.0, 43.6. C<sub>38</sub>H<sub>31</sub>N<sub>3</sub>O<sub>3</sub>, HRMS calculated for m/z [M + H]<sup>+</sup>: 578.2443 (calculated), 578.2438 (found). HPLC purity: 96%.</div></div><div id="sec4_1_3_1_5_1_42" class="NLM_sec NLM_sec_level_4"><div id="ac_i64" class="anchor-spacer"></div><h5 class="article-section__title" id="_i64">ap.  2-(N-{[1,1′-Biphenyl]-4-yl}-1-(furan-2-yl)formamido)-N-(3-phenylpropyl)-2-(pyridin-3-yl)acetamide (<b>40</b>)</h5><div class="NLM_p last">Light yellow solid. 76% yield. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.55 (d, <i>J</i> = 2.2 Hz, 1H), 8.49 (dd, <i>J</i> = 5.2, 1.6 Hz, 1H), 8.32 (t, <i>J</i> = 5.5 Hz, 1H), 7.89–7.79 (m, 1H), 7.64 (d, <i>J</i> = 1.5 Hz, 1H), 7.63–7.51 (m, 4H), 7.49–7.42 (m, 1H), 7.42–7.21 (m, 5H), 7.21–7.13 (m, 2H), 7.12–7.06 (m, 3H), 6.30 (dd, <i>J</i> = 3.6, 1.7 Hz, 1H), 6.19 (s, 1H), 5.58 (d, <i>J</i> = 3.5 Hz, 1H), 3.08 (q, <i>J</i> = 6.6 Hz, 2H), 2.48–2.39 (m, 2H), 1.69–1.55 (m, 2H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.7, 158.3, 146.1, 145.5, 141.6, 139.7, 138.8, 138.5, 132.8, 131.2, 128.9, 128.2, 128.2, 127.9, 126.8, 126.6, 125.7, 124.5, 116.6, 111.4, 62.6, 38.6, 32.4, 30.7. C<sub>33</sub>H<sub>29</sub>N<sub>3</sub>O<sub>3</sub>, HRMS calculated for m/z [M + H]<sup>+</sup>: 516.2287 (calculated), 516.2282 (found). HPLC purity: 98%.</div></div></div></div><div id="sec4_1_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i65" class="anchor-spacer"></div><h3 class="article-section__title" id="_i65"> Native Mass Spectrometry</h3><div class="NLM_p">Prior to analysis, the protein was buffer exchanged into 0.2 M ammonium acetate (pH 6.8) and diluted to 10 μM. DTT was dissolved in water and prepared at a 400 mM stock. Each ligand was dissolved in ethanol and diluted to 10× stock concentrations. The final mixture was prepared by adding 4 μL of protein, 0.5 μL of DTT stock, and 0.5 μL of ligand stock for a final concentration of 4 mM DTT and 70 μM protein. The final ligand concentrations were 10 μM and 30 μM. The mixtures were then incubated for 10 min at room temperature prior to analysis. Each sample was mixed and analyzed in triplicate.</div><div class="NLM_p last">Native mass spectrometry (MS) was performed using a Q-Exactive HF quadrupole-Orbitrap mass spectrometer with the Ultra-High Mass Range research modifications (Thermo Fisher Scientific). Samples were ionized using nanoelectrospray ionization in positive ion mode using 1.0 kV capillary voltage at a 150 °C capillary temperature. The samples were all analyzed with a 1,000–25,000 <i>m</i>/<i>z</i> range, the resolution set to 30,000, and a trapping gas pressure set to 3. Between 10 and 50 V of source fragmentation was applied to all samples to aid in desolvation. Data were deconvolved and analyzed with UniDec.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a></div></div><div id="sec4_1_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i66" class="anchor-spacer"></div><h3 class="article-section__title" id="_i66"> Enzymatic Assays</h3><div class="NLM_p">The main protease (M<sup>pro</sup>) enzymatic assays were carried out in M<sup>pro</sup> reaction buffer containing 20 mM HEPES pH 6.5, 120 mM NaCl, 0.4 mM EDTA, 20% glycerol, and 4 mM DTT, and the SARS-CoV-2 papain-like protease (PL<sup>pro</sup>) enzymatic assays were carried out in PL<sup>Pro</sup> reaction buffer containing 50 mM HEPES, pH7.5, 0.01% triton-100, and 5 mM DTT. The percentage of inhibition and enzymatic IC<sub>50</sub> values were calculated as previously described.<a onclick="showRef(event, 'ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref7 ref8">(7,8)</a> Briefly, the assay was performed in 96-well plates with 100 μL of 100 nM M<sup>pro</sup> protein or 200 nM PL<sup>Pro</sup> protein in their respective reaction buffers. Then 1 μL testing compound at various concentrations was added to each well and incubated at 30 °C for 30 min. The enzymatic reaction was initiated by adding 1 μL of 1 mM corresponding FRET substrate (the final substrate concentration is 10 μM). The reaction was monitored in a Cytation 5 image reader with filters for excitation at 360/40 nm and emission at 460/40 nm at 30 °C for 1 h. The initial velocity of the enzymatic reaction with and without testing compounds was calculated by linear regression for the first 15 min of the kinetic progress curve.</div><div class="NLM_p">For the Morrison plot, 10 μL of 100 nM SARS-CoV-2 M<sup>pro</sup> protein was added to 190 μL of M<sup>pro</sup> reaction buffer containing testing compound and the FRET substrate, and the reaction was monitored for 2 h. The final FRET substrate concentration in this assay is 20 μM. Detailed curve fitting and K<sub>I</sub> determination were described previously.<a onclick="showRef(event, 'ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref7 ref8">(7,8)</a></div><div class="NLM_p">For Michaelis–Menten and Lineweaver–Burk plots, assay was carried out as follows: 50 μL of 50 μM M<sup>pro</sup> protein was added to 50 μL of reaction buffer containing testing compound and various concentrations of FRET substrate to initiate the enzyme reaction. The initial velocity of the enzymatic reaction with and without testing compounds was calculated by linear regression for the first 15 min of the kinetic progress curve, plotted against substrate concentrations in Prism 8 with the Michaelis–Menten equation.</div><div class="NLM_p">Cathepsin K and cathepsin L enzymatic assay was carried out as follows: cathepsin K or cathepsin L was activated by incubating in reaction buffer [20 mM sodium acetate, 1 mM EDTA, and 5 mM DTT (pH 5.5)] for 30 min at 30 °C. Upon activation, the assay was assembled in 96-well plates with 100 μL of cathepsin K at 200 pM or cathepsin L at 300 pM in reaction buffer. Then, 1 μL of testing compound at various concentrations was added to each well and incubated at 30 °C for 30 min. The enzymatic reaction was initiated by adding 1 μL of FRET substrate Z-Phe-Arg-AMC (BACHEM, catalog #. 4003379.0050) (the final substrate concentration is about 5 μM for cathepsin K and 1 μM for cathepsin L). The reaction was monitored in a Cytation 5 image reader with filters for excitation at 360/40 nm and emission at 460/40 nm at 30 °C for 1 h. The IC<sub>50</sub> values were calculated as described in the previous section.</div><div class="NLM_p">Calpain 1 enzymatic assay was carried out as follows: 1 μL of 1 μg/μL calpain 1 protein was added to 100 μL of calpain I reaction buffer [50 mM HEPES, 50 mM NaCl, and 10 mM DTT and 5 mM CaCl<sub>2</sub> (pH 7.5)]; the enzymatic reaction was iniated by adding 1 μL of 4 mM N-succinyl-Leu-Leu-Val-Tyr-7-amido-4-methylcoumarin substrate (Sigma-Aldrich catalog #. S6510). The reaction was monitored with a Cytation 5 image reader with filters for excitation at 360/40 nm and emission at 460/40 nm at 30 °C for 1 h. The IC<sub>50</sub> values were calculated as described in the previous section.</div><div class="NLM_p last">Trypsin enzymatic assay was carried out in a 50 μL volume containing 100 nM trypsin in 50 mM HEPES (pH7.6) reaction buffer, 100 μM Bz-Arg-AMC·HCl (BACHEM, catalog no. 4002540.0050), and serial concentrations of test compounds in duplicate. The reaction was monitored with a Cytation 5 image reader with filters for excitation at 360/40 nm and emission at 460/40 nm at 30 °C for 1 h. The IC<sub>50</sub> values were calculated as described in the previous section.</div></div><div id="sec4_1_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i67" class="anchor-spacer"></div><h3 class="article-section__title" id="_i67"> Differential Scanning Fluorimetry (DSF)</h3><div class="NLM_p last">The thermal shift binding assay (TSA) was carried out using a Thermal Fisher QuantStudio 5 Real-Time PCR System as described previously.<a onclick="showRef(event, 'ref7 ref8 ref29'); return false;" href="javascript:void(0);" class="ref ref7 ref8 ref29">(7,8,29)</a> Briefly, 3 μM SARS-CoV-2 M<sup>pro</sup> protein in M<sup>pro</sup> reaction buffer was incubated with various concentrations of compound <b>ML188</b> or <b>23R</b> at 30 °C for 30 min. 1X SYPRO orange dye was added, and the fluorescence of the well was monitored under a temperature gradient range from 20 to 90 °C with a 0.05 °C/s incremental step. Measured <i>T</i><sub>m</sub> was plotted against compound concentration with one-site binding function in Prism 8.</div></div><div id="sec4_1_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i68" class="anchor-spacer"></div><h3 class="article-section__title" id="_i68"> Cytotoxicity Measurement</h3><div class="NLM_p last">Evaluation of the cytotoxicity of compounds was carried out using the neutral red uptake assay.<a onclick="showRef(event, 'ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref30 ref31">(30,31)</a> Briefly, 80,000 cells/mL of the tested cell lines was dispensed into 96-well cell culture plates at 100 μL/well. Twenty-four hours later, the growth medium was removed and washed with 150 μL PBS buffer. 200 μL of fresh serum-free medium containing serial diluted compounds was added to each well. After incubating for 5 days at 37 °C, the medium was removed and replaced with 100 μL DMEM medium containing 40 μg/mL neutral red and incubated for 2–4 h at 37 °C. The amount of neutral red taken up was determined by measuring the absorbance at 540 nm using a Multiskan FC Microplate Photometer (Fisher Scientific). The CC<sub>50</sub> values were calculated from best-fit dose response curves with variable slope in Prism 8.</div></div><div id="sec4_1_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i69" class="anchor-spacer"></div><h3 class="article-section__title" id="_i69"> Immunofluorescence Assay</h3><div class="NLM_p last">Antiviral immunofluorescence assay was carried out as previously described.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> Briefly, Vero E6 cells in 96-well plates (Corning) were infected with SARS-CoV-2 (USA-WA1/2020 isolate) at MOI of 0.05 in DMEM supplemented with 1% FBS. Immediately before the viral inoculation, the tested compounds in a 3-fold dilution concentration series were also added to the wells in triplicate. The infection proceeded for 48 h without the removal of the viruses or the compounds. The cells were then fixed with 4% paraformaldehyde, permeabilized with 0.1% Triton-100, blocked with DMEM containing 10% FBS, and stained with a rabbit monoclonal antibody against SARS-CoV-2 NP (GeneTex, GTX635679) and an Alexa Fluor 488-conjugated goat antimouse secondary antibody (ThermoFisher Scientific). Hoechst 33342 was added in the final step to counterstain the nuclei. Fluorescence images of approximately ten thousand cells were acquired per well with a 10× objective in a Cytation 5 (BioTek). The total number of cells, as indicated by the nuclei staining, and the fraction of the infected cells, as indicated by the NP staining, was quantified with the cellular analysis module of the Gen5 software (BioTek).</div></div><div id="sec4_1_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i70" class="anchor-spacer"></div><h3 class="article-section__title" id="_i70"> Antiviral Assay in Calu-3 Cells</h3><div class="NLM_p last">Calu-3 cells (ATCC, HTB-55) grown in Minimal Eagles Medium supplemented with 1% nonessential amino acids, 1% penicillin/streptomycin, and 10% FBS are plated in 384 well plates. The next day, 50 nL of drug suspended in DMSO is added as an 8-pt dose response with 3-fold dilutions between test concentrations in triplicate, starting at 40 μM final concentration. The negative control (DMSO, n = 32) and positive control (10 μM Remdesivir, n = 32) are included on each assay plate. Calu3 cells are pretreated with controls and test drugs (in triplicate) for 2 h prior to infection. In BSL3 containment, SARS-CoV-2 (isolate USA-WA1/2020) diluted in serum free growth medium is added to plates to achieve an MOI = 0.5. Cells are incubated continuously with drugs and SARS-CoV-2 for 48 h. Cells are fixed and then immunstained with anti-dsRNA (J2), and nuclei are counterstained with Hoechst 33342 for automated microscopy. Automated image analysis quantifies the number of cells per well (toxicity) and the percentage of infected cells (dsRNA+ cells/cell number) per well. SARS-CoV-2 infection at each drug concentration was normalized to aggregated DMSO plate control wells and expressed as percentage-of-control (POC = % Infection <sub>sample</sub>/Avg % Infection <sub>DMSO cont</sub>). A nonlinear regression curve fit analysis (GraphPad Prism 8) of POC infection and cell viability versus the log<sub>10</sub> transformed concentration values to calculate EC<sub>50</sub> values for infection and CC<sub>50</sub> values for cell viability. Selectivity index (SI) was calculated as a ratio of drug’s CC<sub>50</sub> and EC<sub>50</sub> values (SI = CC<sub>50</sub>/IC<sub>50</sub>).</div></div><div id="sec4_1_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i71" class="anchor-spacer"></div><h3 class="article-section__title" id="_i71"> M<sup>pro</sup> Crystallization and Structure Determination</h3><div class="NLM_p"><b>23R</b> was added to 20 mg/mL SARS-CoV-2 M<sup>pro</sup> to a final concentration of 1.75 mM and incubated overnight at 4°C. This mixture was then diluted 4-fold with protein stock buffer (20 mM Tris pH 7.5, 200 mM NaCl, 1 mM DTT) and then spun down at 13,000<i>g</i> for 1 min to remove precipitate. Crystals were grown by mixing the protein-inhibitor sample with an equal volume of crystallization buffer (20% PEG 3350, 0.2 M NaF) in a vapor diffusion, hanging drop apparatus. Crystals were then transferred to a drop with crystallization buffer containing 5 mM <b>23R</b> for 1 h, followed by a brief soaking in a cryoprotectant solution of 30% PEG 3350 and 15% glycerol with 2 mM <b>23R</b>. Crystals were then flash frozen in liquid nitrogen for X-ray diffraction.</div><div class="NLM_p last">X-ray diffraction data for the SARS-CoV-2 M<sup>pro</sup> structures were collected on the SBC 19-ID beamline at the Advanced Photon Source (APS) in Argonne, IL, and processed with the HKL3000 software suite. The CCP4 versions of MOLREP<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> was used for molecular replacement using a previously solved SARS-CoV-2 M<sup>pro</sup> structure, 6YB7. Structural refinement was performed using REFMAC5<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> and COOT.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> The crystallographic statistics is shown in <a class="ref internalNav" href="#notes-3" aria-label="Supporting Information">Supporting Information</a> Table S2. The complex structure for SARS-CoV-2 M<sup>pro</sup> with <b>23R</b> has been deposited in the Protein Data Bank with the accession ID of <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7KX5">7KX5</a> (SARS-CoV-2 M<sup>Pro</sup> + Jun8-76-3A).</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-3"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i72"><a href="/doi/suppl/10.1021/acs.jmedchem.1c00509" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_88228" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_88228" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.1c00509?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00509</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Target selectivity of noncovalent inhibitor <b>23R</b>; crystallographic statistics; H NMR and C NMR spectra for final products; HPLC traces for representative compounds (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00509/suppl_file/jm1c00509_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00509/suppl_file/jm1c00509_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00509/suppl_file/jm1c00509_si_001.pdf">jm1c00509_si_001.pdf (4.14 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00509/suppl_file/jm1c00509_si_002.csv">jm1c00509_si_002.csv (4.46 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.1c00509" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_58221" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_58221" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yu Chen</span> - <span class="hlFld-Affiliation affiliation">Department
of Molecular Medicine, Morsani College of Medicine, University of South Florida, Tampa, Florida 33612, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#245d474c414a15645157420a414051"><span class="__cf_email__" data-cfemail="c0b9a3a8a5aef180b5b3a6eea5a4b5">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jun Wang</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmacology and Toxicology, College of Pharmacy, The University of Arizona, Tucson, Arizona 85721, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-4845-4621" title="Orcid link">http://orcid.org/0000-0002-4845-4621</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#2d4758435a4c434a6d5d454c5f404c4e54034c5f445742434c03484958"><span class="__cf_email__" data-cfemail="dcb6a9b2abbdb2bb9cacb4bdaeb1bdbfa5f2bdaeb5a6b3b2bdf2b9b8a9">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Naoya Kitamura</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmacology and Toxicology, College of Pharmacy, The University of Arizona, Tucson, Arizona 85721, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Michael Dominic Sacco</span> - <span class="hlFld-Affiliation affiliation">Department
of Molecular Medicine, Morsani College of Medicine, University of South Florida, Tampa, Florida 33612, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chunlong Ma</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmacology and Toxicology, College of Pharmacy, The University of Arizona, Tucson, Arizona 85721, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yanmei Hu</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmacology and Toxicology, College of Pharmacy, The University of Arizona, Tucson, Arizona 85721, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Julia Alma Townsend</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry and Biochemistry, The University
of Arizona, Tucson, Arizona 85721, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xiangzhi Meng</span> - <span class="hlFld-Affiliation affiliation">Department
of Microbiology, Immunology and Molecular Genetics, University of Texas Health Science Center at San Antonio, San Antonio, Texas 78229, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Fushun Zhang</span> - <span class="hlFld-Affiliation affiliation">Department
of Microbiology, Immunology and Molecular Genetics, University of Texas Health Science Center at San Antonio, San Antonio, Texas 78229, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xiujun Zhang</span> - <span class="hlFld-Affiliation affiliation">Department
of Molecular Medicine, Morsani College of Medicine, University of South Florida, Tampa, Florida 33612, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mandy Ba</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmacology and Toxicology, College of Pharmacy, The University of Arizona, Tucson, Arizona 85721, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tommy Szeto</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmacology and Toxicology, College of Pharmacy, The University of Arizona, Tucson, Arizona 85721, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Adis Kukuljac</span> - <span class="hlFld-Affiliation affiliation">Department
of Molecular Medicine, Morsani College of Medicine, University of South Florida, Tampa, Florida 33612, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Michael Thomas Marty</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry and Biochemistry, The University
of Arizona, Tucson, Arizona 85721, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">David Schultz</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Biophysics, University
of Pennsylvania, Philadelphia, Pennsylvania 19104, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sara Cherry</span> - <span class="hlFld-Affiliation affiliation">Department
of Pathology and Laboratory Medicine, University
of Pennsylvania, Philadelphia, Pennsylvania 19104, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yan Xiang</span> - <span class="hlFld-Affiliation affiliation">Department
of Microbiology, Immunology and Molecular Genetics, University of Texas Health Science Center at San Antonio, San Antonio, Texas 78229, United States</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>J.W. conceived and designed the study; N.K. synthesized and characterized the compounds; M.S. carried out Mpro crystallization and structure determination with the assistance of X.Z. and analyzed the data with Y.C.; C.M. expressed the Mpro and PLpro and performed the IC50 determination, thermal shift-binding assay, and enzymatic kinetic studies; Y.H. performed the cytotoxicity assay; X.M. and F.Z. performed the SARS-CoV-2 immunofluorescence assay in Vero E6 cells under the guidance of Y.X. D.S. performed the SARS-CoV-2 immunofluorescence assay in Calu-3 cells under the guidance of S.C. J.T. performed the native mass spectrometry experiments with the guidance from M.T.M.; J.W. and Y.C. secured funding and supervised the study; J.W., Y.C., and M.S. wrote the manuscript with input from the others.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><p>The authors declare the following competing financial interest(s): Naoya Kitamura, Chunlong Ma, Yanmei Hu, and Jun Wang are inventors of a filed patent claiming the use of 23R and related compounds as potential COVID-19 antiviral drugs.<br /></br></p></li></ul></div><div class="ack" id="ACK-d7e5194-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i74">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_36584" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_36584" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This research was partially supported by the National Institutes of Health (NIH) (Grants AI147325 and AI157046) and the Arizona Biomedical Research Centre Young Investigator grant (ADHS18-198859) to J.W. The antiviral assay in Calu-3 cells was conducted through the NIAID preclinical service under a nonclinical evaluation agreement. J.A.T. and M.T.M. were funded by the National Institute of General Medical Sciences and National Institutes of Health (Grant R35 GM128624 to M.T.M.). We thank Michael Kemp for assistance with crystallization and X-ray diffraction data collection. We also thank the staff members of the Advanced Photon Source of Argonne National Laboratory, particularly those at the Structural Biology Center (SBC), with X-ray diffraction data collection. SBC-CAT is operated by UChicago Argonne, LLC, for the U.S. Department of Energy, Office of Biological and Environmental Research under contract DE-AC02-06CH11357. The SARS-CoV-2 experiments were supported by a COVID-19 pilot grant from UTHSCSA and NIH grant AI151638 to Y.X. SARS-Related Coronavirus 2, Isolate USA-WA1/2020 (NR-52281) was deposited by the Centers for Disease Control and Prevention and obtained through BEI Resources, NIAID, NIH.</p></div><div class="postAsapNotesSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="postAsapNotesSection">Note Added After ASAP Publication</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_61684" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_61684" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This paper was published ASAP on April 23, 2021, with errors in Figure 6. These were corrected in the version published ASAP on May 12, 2021.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">SARS-CoV-2</td><td class="NLM_def"><p class="first last">severe acute respiratory syndrome coronavirus 2</p></td></tr><tr><td class="NLM_term">JAK</td><td class="NLM_def"><p class="first last">janus kinase</p></td></tr><tr><td class="NLM_term">Ugi-4CR</td><td class="NLM_def"><p class="first last">Ugi four-component reaction</p></td></tr><tr><td class="NLM_term">M<sup>pro</sup></td><td class="NLM_def"><p class="first last">main protease</p></td></tr><tr><td class="NLM_term">3CL<sup>pro</sup></td><td class="NLM_def"><p class="first last">3-chymotrypsin-like protease</p></td></tr><tr><td class="NLM_term">Nsp5</td><td class="NLM_def"><p class="first last">nonstructural protein 5</p></td></tr><tr><td class="NLM_term">2A<sup>pro</sup></td><td class="NLM_def"><p class="first last">2A protease</p></td></tr><tr><td class="NLM_term">3C<sup>pro</sup></td><td class="NLM_def"><p class="first last">3C protease</p></td></tr><tr><td class="NLM_term">PL<sup>pro</sup></td><td class="NLM_def"><p class="first last">papain-like protease</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i76">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_61133" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_61133" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 34 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kalil, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patterson, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mehta, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomashek, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolfe, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghazaryan, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marconi, V. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruiz-Palacios, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsieh, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kline, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tapson, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iovine, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sweeney, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El Sahly, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Branche, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Regalado Pineda, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lye, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sandkovsky, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luetkemeyer, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finberg, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, P. E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taiwo, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paules, C. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arguinchona, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goepfert, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahuja, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frank, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oh, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mularski, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nielsen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ponce, P. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larson, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rouphael, N. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saklawi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantos, V. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ko, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engemann, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amin, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watanabe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Billings, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elie, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davey, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burgess, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferreira, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Makowski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cardoso, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Bono, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonnett, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Proschan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deye, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dempsey, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nayak, S. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodd, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beigel, J. H.</span></span>; <span class="NLM_contrib-group">Members, A.-S. G.</span> <span> </span><span class="NLM_article-title">Baricitinib plus remdesivir for hospitalized adults with Covid-19</span>. <i>N. Engl. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>384</i></span>,  <span class="NLM_fpage">795</span>– <span class="NLM_lpage">807</span>, <span class="refDoi"> DOI: 10.1056/NEJMoa2031994</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=10.1056%2FNEJMoa2031994" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=33306283" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=1%3ACAS%3A528%3ADC%252BB3MXlvFCksrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=384&publication_year=2021&pages=795-807&author=A.+C.+Kalilauthor=T.+F.+Pattersonauthor=A.+K.+Mehtaauthor=K.+M.+Tomashekauthor=C.+R.+Wolfeauthor=V.+Ghazaryanauthor=V.+C.+Marconiauthor=G.+M.+Ruiz-Palaciosauthor=L.+Hsiehauthor=S.+Klineauthor=V.+Tapsonauthor=N.+M.+Iovineauthor=M.+K.+Jainauthor=D.+A.+Sweeneyauthor=H.+M.+El+Sahlyauthor=A.+R.+Brancheauthor=J.+Regalado+Pinedaauthor=D.+C.+Lyeauthor=U.+Sandkovskyauthor=A.+F.+Luetkemeyerauthor=S.+H.+Cohenauthor=R.+W.+Finbergauthor=P.+E.+H.+Jacksonauthor=B.+Taiwoauthor=C.+I.+Paulesauthor=H.+Arguinchonaauthor=P.+Goepfertauthor=N.+Ahujaauthor=M.+Frankauthor=M.+D.+Ohauthor=E.+S.+Kimauthor=S.+Y.+Tanauthor=R.+A.+Mularskiauthor=H.+Nielsenauthor=P.+O.+Ponceauthor=B.+S.+Taylorauthor=L.+Larsonauthor=N.+G.+Rouphaelauthor=Y.+Saklawiauthor=V.+D.+Cantosauthor=E.+R.+Koauthor=J.+J.+Engemannauthor=A.+N.+Aminauthor=M.+Watanabeauthor=J.+Billingsauthor=M.+C.+Elieauthor=R.+T.+Daveyauthor=T.+H.+Burgessauthor=J.+Ferreiraauthor=M.+Greenauthor=M.+Makowskiauthor=A.+Cardosoauthor=S.+de+Bonoauthor=T.+Bonnettauthor=M.+Proschanauthor=G.+A.+Deyeauthor=W.+Dempseyauthor=S.+U.+Nayakauthor=L.+E.+Doddauthor=J.+H.+Beigelauthor=Members%2C+A.-S.+G.&title=Baricitinib+plus+remdesivir+for+hospitalized+adults+with+Covid-19&doi=10.1056%2FNEJMoa2031994"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Baricitinib plus remdesivir for hospitalized adults with Covid-19</span></div><div class="casAuthors">Kalil, A. C.; Patterson, T. F.; Mehta, A. K.; Tomashek, K. M.; Wolfe, C. R.; Ghazaryan, V.; Marconi, V. C.; Palacios, G. M. Ruiz; Hsieh, L.; Kline, S.; Tapson, V.; Iovine, N. M.; Jain, M. K.; Sweeney, D. A.; El Sahly, H. M.; Branche, A. R.; Regalado Pineda, J.; Lye, D. C.; Sandkovsky, U.; Luetkemeyer, A. F.; Cohen, S. H.; Finberg, R. W.; Jackson, P. E. H.; Taiwo, B.; Paules, C. I.; Arguinchona, H.; Erdmann, N.; Ahuja, N.; Frank, M.; Oh, M.; Kim, E.-S.; Tan, S. Y.; Mularski, R. A.; Nielsen, H.; Ponce, P. O.; Taylor, B. S.; Larson, L. A.; Rouphael, N. G.; Saklawi, Y.; Cantos, V. D.; Ko, E. R.; Engemann, J. J.; Amin, A. N.; Watanabe, M.; Billings, J.; Elie, M.-C.; Davey, R. T.; Burgess, T. H.; Ferreira, J.; Green, M.; Makowski, M.; Cardoso, A.; de Bono, S.; Bonnett, T.; Proschan, M.; Deye, G. A.; Dempsey, W.; Nayak, S. U.; Dodd, L. E.; Beigel, J. H.; Kalil, Andre C.; Patterson, Thomas F.; Mehta, Aneesh K.; Tomashek, Kay M.; Wolfe, Cameron R.; Ghazaryan, Varduhi; Marconi, Vincent C.; Ruiz-Palacios, Guillermo M.; Hsieh, Lanny; Kline, Susan; Tapson, Victor; Iovine, Nicole M.; Jain, Mamta K.; Sweeney, Daniel A.; El Sahly, Hana M.; Branche, Angela R.; Pineda, Justino Regalado; Lye, David C.; Sandkovsky, Uriel; Luetkemeyer, Anne F.; Cohen, Stuart H.; Finberg, Robert W.; Jackson, Patrick E. H.; Taiwo, Babafemi; Paules, Catharine I.; Arguinchona, Henry; Erdmann, Nathaniel; Ahuja, Neera; Frank, Maria; Oh, Myoung-don; Kim, Eu-Suk; Tan, Seow Y.; Mularski, Richard A.; Nielsen, Henrik; Ponce, Philip O.; Taylor, Barbara S.; Larson, LuAnn; Rouphael, Nadine G.; Saklawi, Youssef; Cantos, Valeria D.; Ko, Emily R.; Engemann, John J.; Amin, Alpesh N.; Watanabe, Miki; Billings, Joanne; Elie, Marie-Carmelle; Davey, Richard T.; Burgess, Timothy H.; Ferreira, Jennifer; Green, Michelle; Makowski, Mat; Cardoso, Anabela; de Bono, Stephanie; Bonnett, Tyler; Proschan, Michael; Deye, Gregory A.; Dempsey, Walla; Nayak, Seema U.; Dodd, Lori E.; Beigel, John H.</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">384</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">795-807</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">1533-4406</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">Background Severe coronavirus disease 2019 (Covid-19) is assocd. with dysregulated inflammation.  The effects of combination treatment with baricitinib, a Janus kinase inhibitor, plus remdesivir are not known.  We conducted a double-blind, randomized, placebo-controlled trial evaluating baricitinib plus remdesivir in hospitalized adults with Covid-19.  All the patients received remdesivir (≤10 days) and either baricitinib (≤14 days) or placebo (control).  The primary outcome was the time to recovery.  The key secondary outcome was clin. status at day 15. results A total of 1033 patients underwent randomization (with 515 assigned to combination treatment and 518 to control).  Patients receiving baricitinib had a median time to recovery of 7 days (95% confidence interval [CI], 6 to 8), as compared with 8 days (95% CI, 7 to 9) with control (rate ratio for recovery, 1.16; 95% CI, 1.01 to 1.32; P = 0.03), and a 30% higher odds of improvement in clin. status at day 15 (odds ratio, 1.3; 95% CI, 1.0 to 1.6).  Patients receiving high-flow oxygen or noninvasive ventilation at enrollment had a time to recovery of 10 days with combination treatment and 18 days with control (rate ratio for recovery, 1.51; 95% CI, 1.10 to 2.08).  The 28-day mortality was 5.1% in the combination group and 7.8% in the control group (hazard ratio for death, 0.65; 95% CI, 0.39 to 1.09).  Serious adverse events were less frequent in the combination group than in the control group (16.0% vs. 21.0%; difference, -5.0 percentage points; 95% CI, -9.8 to -0.3; P = 0.03), as were new infections (5.9% vs. 11.2%; difference, -5.3 percentage points; 95% CI, -8.7 to -1.9; P = 0.003).  Baricitinib plus remdesivir was superior to remdesivir alone in reducing recovery time and accelerating improvement in clin. status among patients with Covid-19, notably among those receiving high-flow oxygen or noninvasive ventilation.  The combination was assocd. with fewer serious adverse events.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoszT1RNk0q0bVg90H21EOLACvtfcHk0lhPPlxOX_WLtQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXlvFCksrw%253D&md5=0fe106348eabfd2e844f046340ee833e</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1056%2FNEJMoa2031994&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMoa2031994%26sid%3Dliteratum%253Aachs%26aulast%3DKalil%26aufirst%3DA.%2BC.%26aulast%3DPatterson%26aufirst%3DT.%2BF.%26aulast%3DMehta%26aufirst%3DA.%2BK.%26aulast%3DTomashek%26aufirst%3DK.%2BM.%26aulast%3DWolfe%26aufirst%3DC.%2BR.%26aulast%3DGhazaryan%26aufirst%3DV.%26aulast%3DMarconi%26aufirst%3DV.%2BC.%26aulast%3DRuiz-Palacios%26aufirst%3DG.%2BM.%26aulast%3DHsieh%26aufirst%3DL.%26aulast%3DKline%26aufirst%3DS.%26aulast%3DTapson%26aufirst%3DV.%26aulast%3DIovine%26aufirst%3DN.%2BM.%26aulast%3DJain%26aufirst%3DM.%2BK.%26aulast%3DSweeney%26aufirst%3DD.%2BA.%26aulast%3DEl%2BSahly%26aufirst%3DH.%2BM.%26aulast%3DBranche%26aufirst%3DA.%2BR.%26aulast%3DRegalado%2BPineda%26aufirst%3DJ.%26aulast%3DLye%26aufirst%3DD.%2BC.%26aulast%3DSandkovsky%26aufirst%3DU.%26aulast%3DLuetkemeyer%26aufirst%3DA.%2BF.%26aulast%3DCohen%26aufirst%3DS.%2BH.%26aulast%3DFinberg%26aufirst%3DR.%2BW.%26aulast%3DJackson%26aufirst%3DP.%2BE.%2BH.%26aulast%3DTaiwo%26aufirst%3DB.%26aulast%3DPaules%26aufirst%3DC.%2BI.%26aulast%3DArguinchona%26aufirst%3DH.%26aulast%3DGoepfert%26aufirst%3DP.%26aulast%3DAhuja%26aufirst%3DN.%26aulast%3DFrank%26aufirst%3DM.%26aulast%3DOh%26aufirst%3DM.%2BD.%26aulast%3DKim%26aufirst%3DE.%2BS.%26aulast%3DTan%26aufirst%3DS.%2BY.%26aulast%3DMularski%26aufirst%3DR.%2BA.%26aulast%3DNielsen%26aufirst%3DH.%26aulast%3DPonce%26aufirst%3DP.%2BO.%26aulast%3DTaylor%26aufirst%3DB.%2BS.%26aulast%3DLarson%26aufirst%3DL.%26aulast%3DRouphael%26aufirst%3DN.%2BG.%26aulast%3DSaklawi%26aufirst%3DY.%26aulast%3DCantos%26aufirst%3DV.%2BD.%26aulast%3DKo%26aufirst%3DE.%2BR.%26aulast%3DEngemann%26aufirst%3DJ.%2BJ.%26aulast%3DAmin%26aufirst%3DA.%2BN.%26aulast%3DWatanabe%26aufirst%3DM.%26aulast%3DBillings%26aufirst%3DJ.%26aulast%3DElie%26aufirst%3DM.%2BC.%26aulast%3DDavey%26aufirst%3DR.%2BT.%26aulast%3DBurgess%26aufirst%3DT.%2BH.%26aulast%3DFerreira%26aufirst%3DJ.%26aulast%3DGreen%26aufirst%3DM.%26aulast%3DMakowski%26aufirst%3DM.%26aulast%3DCardoso%26aufirst%3DA.%26aulast%3Dde%2BBono%26aufirst%3DS.%26aulast%3DBonnett%26aufirst%3DT.%26aulast%3DProschan%26aufirst%3DM.%26aulast%3DDeye%26aufirst%3DG.%2BA.%26aulast%3DDempsey%26aufirst%3DW.%26aulast%3DNayak%26aufirst%3DS.%2BU.%26aulast%3DDodd%26aufirst%3DL.%2BE.%26aulast%3DBeigel%26aufirst%3DJ.%2BH.%26aulast%3D%26atitle%3DBaricitinib%2520plus%2520remdesivir%2520for%2520hospitalized%2520adults%2520with%2520Covid-19%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2021%26volume%3D384%26spage%3D795%26epage%3D807%26doi%3D10.1056%2FNEJMoa2031994" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gil, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ginex, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maestro, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nozal, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrado-Gil, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuesta-Geijo, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urquiza, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramirez, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alonso, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campillo, N. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, A.</span></span> <span> </span><span class="NLM_article-title">COVID-19: drug targets and potential treatments</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">12359</span>– <span class="NLM_lpage">12386</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00606</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00606" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtFWmsL3I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=12359-12386&author=C.+Gilauthor=T.+Ginexauthor=I.+Maestroauthor=V.+Nozalauthor=L.+Barrado-Gilauthor=M.+A.+Cuesta-Geijoauthor=J.+Urquizaauthor=D.+Ramirezauthor=C.+Alonsoauthor=N.+E.+Campilloauthor=A.+Martinez&title=COVID-19%3A+drug+targets+and+potential+treatments&doi=10.1021%2Facs.jmedchem.0c00606"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">COVID-19: Drug Targets and Potential Treatments</span></div><div class="casAuthors">Gil, Carmen; Ginex, Tiziana; Maestro, Ines; Nozal, Vanesa; Barrado-Gil, Lucia; Cuesta-Geijo, Miguel Angel; Urquiza, Jesus; Ramirez, David; Alonso, Covadonga; Campillo, Nuria E.; Martinez, Ana</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">12359-12386</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Currently, the authors are immersed in a pandemic caused by the emerging severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), which severely threatens public health worldwide.  Until now, no drug or vaccine has been approved to treat the severe disease caused by this coronavirus, COVID-19.  The authors will focus on the main virus-based and host-based targets that can guide medicinal chem. efforts to discover new drugs for this devastating disease.  In principle, all CoVs enzymes and proteins involved in viral replication and the control of host cellular machineries are potentially druggable targets in the search for therapeutic options for SARS-CoV-2.  This perspective provides an overview of the main targets from a structural point of view, together with reported therapeutic compds. with activity against SARS-CoV-2 and/or other CoVs.  Also, the role of innate immune response to coronavirus infection and the related therapeutic options will be presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsg-YRktOknrVg90H21EOLACvtfcHk0lha57MB3MxY7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtFWmsL3I&md5=6a0dc17c5f601f09a92da933f81bbbea</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00606&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00606%26sid%3Dliteratum%253Aachs%26aulast%3DGil%26aufirst%3DC.%26aulast%3DGinex%26aufirst%3DT.%26aulast%3DMaestro%26aufirst%3DI.%26aulast%3DNozal%26aufirst%3DV.%26aulast%3DBarrado-Gil%26aufirst%3DL.%26aulast%3DCuesta-Geijo%26aufirst%3DM.%2BA.%26aulast%3DUrquiza%26aufirst%3DJ.%26aulast%3DRamirez%26aufirst%3DD.%26aulast%3DAlonso%26aufirst%3DC.%26aulast%3DCampillo%26aufirst%3DN.%2BE.%26aulast%3DMartinez%26aufirst%3DA.%26atitle%3DCOVID-19%253A%2520drug%2520targets%2520and%2520potential%2520treatments%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D12359%26epage%3D12386%26doi%3D10.1021%2Facs.jmedchem.0c00606" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ullrich, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nitsche, C.</span></span> <span> </span><span class="NLM_article-title">The SARS-CoV-2 main protease as drug target</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">127377</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2020.127377</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=10.1016%2Fj.bmcl.2020.127377" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=32738988" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtlaisrbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2020&pages=127377&author=S.+Ullrichauthor=C.+Nitsche&title=The+SARS-CoV-2+main+protease+as+drug+target&doi=10.1016%2Fj.bmcl.2020.127377"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The SARS-CoV-2 main protease as drug target</span></div><div class="casAuthors">Ullrich, Sven; Nitsche, Christoph</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">127377</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The unprecedented pandemic of the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is threatening global health.  The virus emerged in late 2019 and can cause a severe disease assocd. with significant mortality.  Several vaccine development and drug discovery campaigns are underway.  The SARS-CoV-2 main protease is considered a promising drug target, as it is dissimilar to human proteases.  Sequence and structure of the main protease are closely related to those from other betacoronaviruses, facilitating drug discovery attempts based on previous lead compds.  Covalently binding peptidomimetics and small mols. are investigated.  Various compds. show antiviral activity in infected human cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVeovBguTwDbVg90H21EOLACvtfcHk0lha57MB3MxY7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtlaisrbJ&md5=b2fd8ea99ddf6d3e553b137904d406a9</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2020.127377&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2020.127377%26sid%3Dliteratum%253Aachs%26aulast%3DUllrich%26aufirst%3DS.%26aulast%3DNitsche%26aufirst%3DC.%26atitle%3DThe%2520SARS-CoV-2%2520main%2520protease%2520as%2520drug%2520target%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2020%26volume%3D30%26spage%3D127377%26doi%3D10.1016%2Fj.bmcl.2020.127377" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rut, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groborz, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zmudzinski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pawlik, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jochmans, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neyts, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Młynarski, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilgenfeld, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drag, M.</span></span> <span> </span><span class="NLM_article-title">SARS-CoV-2 Mpro inhibitors and activity-based probes for patient-sample imaging</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">222</span>– <span class="NLM_lpage">228</span>, <span class="refDoi"> DOI: 10.1038/s41589-020-00689-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=10.1038%2Fs41589-020-00689-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=33093684" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=1%3ACAS%3A528%3ADC%252BB3cXitFKjsbbK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2021&pages=222-228&author=W.+Rutauthor=K.+Groborzauthor=L.+Zhangauthor=X.+Sunauthor=M.+Zmudzinskiauthor=B.+Pawlikauthor=X.+Wangauthor=D.+Jochmansauthor=J.+Neytsauthor=W.+M%C5%82ynarskiauthor=R.+Hilgenfeldauthor=M.+Drag&title=SARS-CoV-2+Mpro+inhibitors+and+activity-based+probes+for+patient-sample+imaging&doi=10.1038%2Fs41589-020-00689-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">SARS-CoV-2 Mpro inhibitors and activity-based probes for patient-sample imaging</span></div><div class="casAuthors">Rut, Wioletta; Groborz, Katarzyna; Zhang, Linlin; Sun, Xinyuanyuan; Zmudzinski, Mikolaj; Pawlik, Bartlomiej; Wang, Xinyu; Jochmans, Dirk; Neyts, Johan; Mlynarski, Wojciech; Hilgenfeld, Rolf; Drag, Marcin</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">222-228</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">In Dec. 2019, the first cases of infection with a novel coronavirus, SARS-CoV-2, were diagnosed.  Currently, there is no effective antiviral treatment for COVID-19.  To address this emerging problem, we focused on the SARS-CoV-2 main protease that constitutes one of the most attractive antiviral drug targets.  We have synthesized a combinatorial library of fluorogenic substrates with glutamine in the P1 position.  We used it to det. the substrate preferences of the SARS-CoV and SARS-CoV-2 main proteases.  On the basis of these findings, we designed and synthesized a potent SARS-CoV-2 inhibitor (Ac-Abu-DTyr-Leu-Gln-VS, half-maximal effective concn. of 3.7μM) and two activity-based probes, for one of which we detd. the crystal structure of its complex with the SARS-CoV-2 Mpro.  We visualized active SARS-CoV-2 Mpro in nasopharyngeal epithelial cells of patients suffering from COVID-19 infection.  The results of our work provide a structural framework for the design of inhibitors as antiviral agents and/or diagnostic tests.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowwaL39Q6HFrVg90H21EOLACvtfcHk0lha57MB3MxY7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXitFKjsbbK&md5=11ae0092711da91cf820234e8dcb6e27</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fs41589-020-00689-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-020-00689-z%26sid%3Dliteratum%253Aachs%26aulast%3DRut%26aufirst%3DW.%26aulast%3DGroborz%26aufirst%3DK.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DSun%26aufirst%3DX.%26aulast%3DZmudzinski%26aufirst%3DM.%26aulast%3DPawlik%26aufirst%3DB.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DJochmans%26aufirst%3DD.%26aulast%3DNeyts%26aufirst%3DJ.%26aulast%3DM%25C5%2582ynarski%26aufirst%3DW.%26aulast%3DHilgenfeld%26aufirst%3DR.%26aulast%3DDrag%26aufirst%3DM.%26atitle%3DSARS-CoV-2%2520Mpro%2520inhibitors%2520and%2520activity-based%2520probes%2520for%2520patient-sample%2520imaging%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2021%26volume%3D17%26spage%3D222%26epage%3D228%26doi%3D10.1038%2Fs41589-020-00689-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brindisi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shahabi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chapman, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mesecar, A. D.</span></span> <span> </span><span class="NLM_article-title">Drug development and medicinal chemistry efforts toward SARS-coronavirus and Covid-19 therapeutics</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">907</span>– <span class="NLM_lpage">932</span>, <span class="refDoi"> DOI: 10.1002/cmdc.202000223</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=10.1002%2Fcmdc.202000223" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=32324951" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=1%3ACAS%3A528%3ADC%252BB3cXos1GmsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2020&pages=907-932&author=A.+K.+Ghoshauthor=M.+Brindisiauthor=D.+Shahabiauthor=M.+E.+Chapmanauthor=A.+D.+Mesecar&title=Drug+development+and+medicinal+chemistry+efforts+toward+SARS-coronavirus+and+Covid-19+therapeutics&doi=10.1002%2Fcmdc.202000223"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Drug Development and Medicinal Chemistry Efforts toward SARS-Coronavirus and Covid-19 Therapeutics</span></div><div class="casAuthors">Ghosh, Arun K.; Brindisi, Margherita; Shahabi, Dana; Chapman, Mackenzie E.; Mesecar, Andrew D.</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">907-932</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  The COVID-19 pandemic caused by SARS-CoV-2 infection is spreading at an alarming rate and has created an unprecedented health emergency around the globe.  There is no effective vaccine or approved drug treatment against COVID-19 and other pathogenic coronaviruses.  The development of antiviral agents is an urgent priority.  Biochem. events crit. to the coronavirus replication cycle provided a no. of attractive targets for drug development.  These include, spike protein for binding to host cell-surface receptors, proteolytic enzymes that are essential for processing polyproteins into mature viruses, and RNA-dependent RNA polymerase for RNA replication.  There has been a lot of ground work for drug discovery and development against these targets.  Also, high-throughput screening efforts have led to the identification of diverse lead structures, including natural product-derived mols.  This review highlights past and present drug discovery and medicinal-chem. approaches against SARS-CoV, MERS-CoV and COVID-19 targets.  The review hopes to stimulate further research and will be a useful guide to the development of effective therapies against COVID-19 and other pathogenic coronaviruses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqcgVQF9N3zf7Vg90H21EOLACvtfcHk0lhx6lLKsBMSOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXos1GmsL4%253D&md5=8b4c3c83c05a7b64f39ba7d803ea1709</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.202000223&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.202000223%26sid%3Dliteratum%253Aachs%26aulast%3DGhosh%26aufirst%3DA.%2BK.%26aulast%3DBrindisi%26aufirst%3DM.%26aulast%3DShahabi%26aufirst%3DD.%26aulast%3DChapman%26aufirst%3DM.%2BE.%26aulast%3DMesecar%26aufirst%3DA.%2BD.%26atitle%3DDrug%2520development%2520and%2520medicinal%2520chemistry%2520efforts%2520toward%2520SARS-coronavirus%2520and%2520Covid-19%2520therapeutics%26jtitle%3DChemMedChem%26date%3D2020%26volume%3D15%26spage%3D907%26epage%3D932%26doi%3D10.1002%2Fcmdc.202000223" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boras, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anson, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arenson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aschenbrenner, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bakowski, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beutler, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Binder, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammond, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kadar, E. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kania, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kimoto, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirkpatrick, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lanyon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lendy, E. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lillis, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luthra, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Obach, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Brien, M. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Connor, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogilvie, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pettersson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reese, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossulek, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sathish, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steppan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ticehurst, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Updyke, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chatterjee, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mesecar, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allerton, C.</span></span> <span> </span><span class="NLM_article-title">Discovery of a novel inhibitor of coronavirus 3CL protease as a clinical candidate for the potential treatment of COVID-19</span>. <i>bioRxiv</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_fpage">2020.09.12.293498</span>, <span class="refDoi"> DOI: 10.1101/2020.09.12.293498</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=10.1101%2F2020.09.12.293498" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=32935104" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&pages=2020.09.12.293498&author=B.+Borasauthor=R.+M.+Jonesauthor=B.+J.+Ansonauthor=D.+Arensonauthor=L.+Aschenbrennerauthor=M.+A.+Bakowskiauthor=N.+Beutlerauthor=J.+Binderauthor=E.+Chenauthor=H.+Engauthor=J.+Hammondauthor=R.+Hoffmanauthor=E.+P.+Kadarauthor=R.+Kaniaauthor=E.+Kimotoauthor=M.+G.+Kirkpatrickauthor=L.+Lanyonauthor=E.+K.+Lendyauthor=J.+R.+Lillisauthor=S.+A.+Luthraauthor=C.+Maauthor=S.+Noellauthor=R.+S.+Obachauthor=M.+N.+O%E2%80%99Brienauthor=R.+O%E2%80%99Connorauthor=K.+Ogilvieauthor=D.+Owenauthor=M.+Petterssonauthor=M.+R.+Reeseauthor=T.+Rogersauthor=M.+I.+Rossulekauthor=J.+G.+Sathishauthor=C.+Steppanauthor=M.+Ticehurstauthor=L.+W.+Updykeauthor=Y.+Zhuauthor=J.+Wangauthor=A.+K.+Chatterjeeauthor=A.+D.+Mesecarauthor=A.+S.+Andersonauthor=C.+Allerton&title=Discovery+of+a+novel+inhibitor+of+coronavirus+3CL+protease+as+a+clinical+candidate+for+the+potential+treatment+of+COVID-19&doi=10.1101%2F2020.09.12.293498"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1101%2F2020.09.12.293498&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252F2020.09.12.293498%26sid%3Dliteratum%253Aachs%26aulast%3DBoras%26aufirst%3DB.%26aulast%3DJones%26aufirst%3DR.%2BM.%26aulast%3DAnson%26aufirst%3DB.%2BJ.%26aulast%3DArenson%26aufirst%3DD.%26aulast%3DAschenbrenner%26aufirst%3DL.%26aulast%3DBakowski%26aufirst%3DM.%2BA.%26aulast%3DBeutler%26aufirst%3DN.%26aulast%3DBinder%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DE.%26aulast%3DEng%26aufirst%3DH.%26aulast%3DHammond%26aufirst%3DJ.%26aulast%3DHoffman%26aufirst%3DR.%26aulast%3DKadar%26aufirst%3DE.%2BP.%26aulast%3DKania%26aufirst%3DR.%26aulast%3DKimoto%26aufirst%3DE.%26aulast%3DKirkpatrick%26aufirst%3DM.%2BG.%26aulast%3DLanyon%26aufirst%3DL.%26aulast%3DLendy%26aufirst%3DE.%2BK.%26aulast%3DLillis%26aufirst%3DJ.%2BR.%26aulast%3DLuthra%26aufirst%3DS.%2BA.%26aulast%3DMa%26aufirst%3DC.%26aulast%3DNoell%26aufirst%3DS.%26aulast%3DObach%26aufirst%3DR.%2BS.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DM.%2BN.%26aulast%3DO%25E2%2580%2599Connor%26aufirst%3DR.%26aulast%3DOgilvie%26aufirst%3DK.%26aulast%3DOwen%26aufirst%3DD.%26aulast%3DPettersson%26aufirst%3DM.%26aulast%3DReese%26aufirst%3DM.%2BR.%26aulast%3DRogers%26aufirst%3DT.%26aulast%3DRossulek%26aufirst%3DM.%2BI.%26aulast%3DSathish%26aufirst%3DJ.%2BG.%26aulast%3DSteppan%26aufirst%3DC.%26aulast%3DTicehurst%26aufirst%3DM.%26aulast%3DUpdyke%26aufirst%3DL.%2BW.%26aulast%3DZhu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DChatterjee%26aufirst%3DA.%2BK.%26aulast%3DMesecar%26aufirst%3DA.%2BD.%26aulast%3DAnderson%26aufirst%3DA.%2BS.%26aulast%3DAllerton%26aufirst%3DC.%26atitle%3DDiscovery%2520of%2520a%2520novel%2520inhibitor%2520of%2520coronavirus%25203CL%2520protease%2520as%2520a%2520clinical%2520candidate%2520for%2520the%2520potential%2520treatment%2520of%2520COVID-19%26jtitle%3DbioRxiv%26date%3D2020%26spage%3D2020.09.12.293498%26doi%3D10.1101%2F2020.09.12.293498" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ma, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sacco, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurst, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Townsend, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szeto, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tarbet, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marty, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span> <span> </span><span class="NLM_article-title">Boceprevir, GC-376, and calpain inhibitors II, XII inhibit SARS-CoV-2 viral replication by targeting the viral main protease</span>. <i>Cell Res.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">678</span>– <span class="NLM_lpage">692</span>, <span class="refDoi"> DOI: 10.1038/s41422-020-0356-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=10.1038%2Fs41422-020-0356-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=32541865" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=1%3ACAS%3A528%3ADC%252BB3cXht1WmsL%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2020&pages=678-692&author=C.+Maauthor=M.+D.+Saccoauthor=B.+Hurstauthor=J.+A.+Townsendauthor=Y.+Huauthor=T.+Szetoauthor=X.+Zhangauthor=B.+Tarbetauthor=M.+T.+Martyauthor=Y.+Chenauthor=J.+Wang&title=Boceprevir%2C+GC-376%2C+and+calpain+inhibitors+II%2C+XII+inhibit+SARS-CoV-2+viral+replication+by+targeting+the+viral+main+protease&doi=10.1038%2Fs41422-020-0356-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Boceprevir, GC-376, and calpain inhibitors II, XII inhibit SARS-CoV-2 viral replication by targeting the viral main protease</span></div><div class="casAuthors">Ma, Chunlong; Sacco, Michael Dominic; Hurst, Brett; Townsend, Julia Alma; Hu, Yanmei; Szeto, Tommy; Zhang, Xiujun; Tarbet, Bart; Marty, Michael Thomas; Chen, Yu; Wang, Jun</div><div class="citationInfo"><span class="NLM_cas:title">Cell Research</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">678-692</span>CODEN:
                <span class="NLM_cas:coden">CREEB6</span>;
        ISSN:<span class="NLM_cas:issn">1001-0602</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A new coronavirus SARS-CoV-2, also called novel coronavirus 2019 (2019-nCoV), started to circulate among humans around Dec. 2019, and it is now widespread as a global pandemic.  The disease caused by SARS-CoV-2 virus is called COVID-19, which is highly contagious and has an overall mortality rate of 6.35% as of May 26, 2020.  There is no vaccine or antiviral available for SARS-CoV-2.  We report our discovery of inhibitors targeting the SARS-CoV-2 main protease (Mpro).  Using the FRET-based enzymic assay, several inhibitors including boceprevir, GC-376, and calpain inhibitors II and XII were identified to have potent activity with single-digit to submicromolar IC50 values in the enzymic assay.  The mechanism of action of the hits was further characterized using enzyme kinetic studies, thermal shift binding assays, and native mass spectrometry.  Significantly, 4 compds. (boceprevir, GC-376, and calpain inhibitors II and XII) inhibit SARS-CoV-2 viral replication in cell culture with EC50 values ranging 0.49-3.37μM.  Notably, boceprevir and calpain inhibitors II and XII represent novel chemotypes that are distinct from known substrate-based peptidomimetic Mpro inhibitors.  A complex crystal structure of SARS-CoV-2 Mpro with GC-376, detd. at 2.15 Å resoln. with 3 protomers per asym. unit, revealed 2 unique binding configurations, shedding light on the mol. interactions and protein conformational flexibility underlying substrate and inhibitor binding by Mpro.  Overall, the compds. identified provide promising starting points for the further development of SARS-CoV-2 therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVnFC-t0uFW7Vg90H21EOLACvtfcHk0lhx6lLKsBMSOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXht1WmsL%252FL&md5=225d518c313474cdfbcfe678d35e26a4</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fs41422-020-0356-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41422-020-0356-z%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DC.%26aulast%3DSacco%26aufirst%3DM.%2BD.%26aulast%3DHurst%26aufirst%3DB.%26aulast%3DTownsend%26aufirst%3DJ.%2BA.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DSzeto%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DTarbet%26aufirst%3DB.%26aulast%3DMarty%26aufirst%3DM.%2BT.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DJ.%26atitle%3DBoceprevir%252C%2520GC-376%252C%2520and%2520calpain%2520inhibitors%2520II%252C%2520XII%2520inhibit%2520SARS-CoV-2%2520viral%2520replication%2520by%2520targeting%2520the%2520viral%2520main%2520protease%26jtitle%3DCell%2520Res.%26date%3D2020%26volume%3D30%26spage%3D678%26epage%3D692%26doi%3D10.1038%2Fs41422-020-0356-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sacco, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lagarias, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Townsend, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dube, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kitamura, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurst, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tarbet, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marty, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolocouris, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span> <span> </span><span class="NLM_article-title">Structure and inhibition of the SARS-CoV-2 main protease reveal strategy for developing dual inhibitors against M(pro) and cathepsin L</span>. <i>Sci. Adv.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">eabe0751</span>, <span class="refDoi"> DOI: 10.1126/sciadv.abe0751</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=10.1126%2Fsciadv.abe0751" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=33158912" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2020&pages=eabe0751&author=M.+D.+Saccoauthor=C.+Maauthor=P.+Lagariasauthor=A.+Gaoauthor=J.+A.+Townsendauthor=X.+Mengauthor=P.+Dubeauthor=X.+Zhangauthor=Y.+Huauthor=N.+Kitamuraauthor=B.+Hurstauthor=B.+Tarbetauthor=M.+T.+Martyauthor=A.+Kolocourisauthor=Y.+Xiangauthor=Y.+Chenauthor=J.+Wang&title=Structure+and+inhibition+of+the+SARS-CoV-2+main+protease+reveal+strategy+for+developing+dual+inhibitors+against+M%28pro%29+and+cathepsin+L&doi=10.1126%2Fsciadv.abe0751"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1126%2Fsciadv.abe0751&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fsciadv.abe0751%26sid%3Dliteratum%253Aachs%26aulast%3DSacco%26aufirst%3DM.%2BD.%26aulast%3DMa%26aufirst%3DC.%26aulast%3DLagarias%26aufirst%3DP.%26aulast%3DGao%26aufirst%3DA.%26aulast%3DTownsend%26aufirst%3DJ.%2BA.%26aulast%3DMeng%26aufirst%3DX.%26aulast%3DDube%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DKitamura%26aufirst%3DN.%26aulast%3DHurst%26aufirst%3DB.%26aulast%3DTarbet%26aufirst%3DB.%26aulast%3DMarty%26aufirst%3DM.%2BT.%26aulast%3DKolocouris%26aufirst%3DA.%26aulast%3DXiang%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DJ.%26atitle%3DStructure%2520and%2520inhibition%2520of%2520the%2520SARS-CoV-2%2520main%2520protease%2520reveal%2520strategy%2520for%2520developing%2520dual%2520inhibitors%2520against%2520M%2528pro%2529%2520and%2520cathepsin%2520L%26jtitle%3DSci.%2520Adv.%26date%3D2020%26volume%3D6%26spage%3Deabe0751%26doi%3D10.1126%2Fsciadv.abe0751" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rathnayake, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perera, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mackin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyerholz, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kashipathy, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Battaile, K. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lovell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perlman, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groutas, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, K. O.</span></span> <span> </span><span class="NLM_article-title">3C-like protease inhibitors block coronavirus replication in vitro and improve survival in MERS-CoV-infected mice</span>. <i>Sci. Transl. Med.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">eabc5332</span>, <span class="refDoi"> DOI: 10.1126/scitranslmed.abc5332</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=10.1126%2Fscitranslmed.abc5332" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=32747425" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2020&pages=eabc5332&author=A.+D.+Rathnayakeauthor=J.+Zhengauthor=Y.+Kimauthor=K.+D.+Pereraauthor=S.+Mackinauthor=D.+K.+Meyerholzauthor=M.+M.+Kashipathyauthor=K.+P.+Battaileauthor=S.+Lovellauthor=S.+Perlmanauthor=W.+C.+Groutasauthor=K.+O.+Chang&title=3C-like+protease+inhibitors+block+coronavirus+replication+in+vitro+and+improve+survival+in+MERS-CoV-infected+mice&doi=10.1126%2Fscitranslmed.abc5332"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1126%2Fscitranslmed.abc5332&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscitranslmed.abc5332%26sid%3Dliteratum%253Aachs%26aulast%3DRathnayake%26aufirst%3DA.%2BD.%26aulast%3DZheng%26aufirst%3DJ.%26aulast%3DKim%26aufirst%3DY.%26aulast%3DPerera%26aufirst%3DK.%2BD.%26aulast%3DMackin%26aufirst%3DS.%26aulast%3DMeyerholz%26aufirst%3DD.%2BK.%26aulast%3DKashipathy%26aufirst%3DM.%2BM.%26aulast%3DBattaile%26aufirst%3DK.%2BP.%26aulast%3DLovell%26aufirst%3DS.%26aulast%3DPerlman%26aufirst%3DS.%26aulast%3DGroutas%26aufirst%3DW.%2BC.%26aulast%3DChang%26aufirst%3DK.%2BO.%26atitle%3D3C-like%2520protease%2520inhibitors%2520block%2520coronavirus%2520replication%2520in%2520vitro%2520and%2520improve%2520survival%2520in%2520MERS-CoV-infected%2520mice%26jtitle%3DSci.%2520Transl.%2520Med.%26date%3D2020%26volume%3D12%26spage%3Deabc5332%26doi%3D10.1126%2Fscitranslmed.abc5332" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Qiao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ni, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, Z. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, Q. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, Y. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, G. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, Y. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S.</span></span> <span> </span><span class="NLM_article-title">SARS-CoV-2 M(pro) inhibitors with antiviral activity in a transgenic mouse model</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>371</i></span>,  <span class="NLM_fpage">1374</span>– <span class="NLM_lpage">1378</span>, <span class="refDoi"> DOI: 10.1126/science.abf1611</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=10.1126%2Fscience.abf1611" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=33602867" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=1%3ACAS%3A528%3ADC%252BB3MXnsVyru7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=371&publication_year=2021&pages=1374-1378&author=J.+Qiaoauthor=Y.+S.+Liauthor=R.+Zengauthor=F.+L.+Liuauthor=R.+H.+Luoauthor=C.+Huangauthor=Y.+F.+Wangauthor=J.+Zhangauthor=B.+Quanauthor=C.+Shenauthor=X.+Maoauthor=X.+Liuauthor=W.+Sunauthor=W.+Yangauthor=X.+Niauthor=K.+Wangauthor=L.+Xuauthor=Z.+L.+Duanauthor=Q.+C.+Zouauthor=H.+L.+Zhangauthor=W.+Quauthor=Y.+H.+Longauthor=M.+H.+Liauthor=R.+C.+Yangauthor=X.+Liuauthor=J.+Youauthor=Y.+Zhouauthor=R.+Yaoauthor=W.+P.+Liauthor=J.+M.+Liuauthor=P.+Chenauthor=Y.+Liuauthor=G.+F.+Linauthor=X.+Yangauthor=J.+Zouauthor=L.+Liauthor=Y.+Huauthor=G.+W.+Luauthor=W.+M.+Liauthor=Y.+Q.+Weiauthor=Y.+T.+Zhengauthor=J.+Leiauthor=S.+Yang&title=SARS-CoV-2+M%28pro%29+inhibitors+with+antiviral+activity+in+a+transgenic+mouse+model&doi=10.1126%2Fscience.abf1611"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">SARS-CoV-2 Mpro inhibitors with antiviral activity in a transgenic mouse model</span></div><div class="casAuthors">Qiao, Jingxin; Li, Yue-Shan; Zeng, Rui; Liu, Feng-Liang; Luo, Rong-Hua; Huang, Chong; Wang, Yi-Fei; Zhang, Jie; Quan, Baoxue; Shen, Chenjian; Mao, Xin; Liu, Xinlei; Sun, Weining; Yang, Wei; Ni, Xincheng; Wang, Kai; Xu, Ling; Duan, Zi-Lei; Zou, Qing-Cui; Zhang, Hai-Lin; Qu, Wang; Long, Yang-Hao-Peng; Li, Ming-Hua; Yang, Rui-Cheng; Liu, Xiaolong; You, Jing; Zhou, Yangli; Yao, Rui; Li, Wen-Pei; Liu, Jing-Ming; Chen, Pei; Liu, Yang; Lin, Gui-Feng; Yang, Xin; Zou, Jun; Li, Linli; Hu, Yiguo; Lu, Guang-Wen; Li, Wei-Min; Wei, Yu-Quan; Zheng, Yong-Tang; Lei, Jian; Yang, Shengyong</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">371</span>
        (<span class="NLM_cas:issue">6536</span>),
    <span class="NLM_cas:pages">1374-1378</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">1095-9203</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The COVID-19 pandemic caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) continually poses serious threats to global public health.  The main protease (Mpro) of SARS-CoV-2 plays a central role in viral replication.  We designed and synthesized 32 new bicycloproline-contg. Mpro inhibitors derived from either boceprevir or telaprevir, both of which are approved antivirals.  All compds. inhibited SARS-CoV-2 Mpro activity in vitro, with 50% inhibitory concn. values ranging from 7.6 to 748.5 nM.  The cocrystal structure of Mpro in complex with MI-23, one of the most potent compds., revealed its interaction mode.  Two compds. (MI-09 and MI-30) showed excellent antiviral activity in cell-based assays.  In a transgenic mouse model of SARS-CoV-2 infection, oral or i.p. treatment with MI-09 or MI-30 significantly reduced lung viral loads and lung lesions.  Both also displayed good pharmacokinetic properties and safety in rats.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVMv8s8s-oWLVg90H21EOLACvtfcHk0lgihF_DMF9OWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXnsVyru7g%253D&md5=555937b1776c8353e59601d567e2ebb9</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1126%2Fscience.abf1611&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.abf1611%26sid%3Dliteratum%253Aachs%26aulast%3DQiao%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DY.%2BS.%26aulast%3DZeng%26aufirst%3DR.%26aulast%3DLiu%26aufirst%3DF.%2BL.%26aulast%3DLuo%26aufirst%3DR.%2BH.%26aulast%3DHuang%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DY.%2BF.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DQuan%26aufirst%3DB.%26aulast%3DShen%26aufirst%3DC.%26aulast%3DMao%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DSun%26aufirst%3DW.%26aulast%3DYang%26aufirst%3DW.%26aulast%3DNi%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DK.%26aulast%3DXu%26aufirst%3DL.%26aulast%3DDuan%26aufirst%3DZ.%2BL.%26aulast%3DZou%26aufirst%3DQ.%2BC.%26aulast%3DZhang%26aufirst%3DH.%2BL.%26aulast%3DQu%26aufirst%3DW.%26aulast%3DLong%26aufirst%3DY.%2BH.%26aulast%3DLi%26aufirst%3DM.%2BH.%26aulast%3DYang%26aufirst%3DR.%2BC.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DYou%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DYao%26aufirst%3DR.%26aulast%3DLi%26aufirst%3DW.%2BP.%26aulast%3DLiu%26aufirst%3DJ.%2BM.%26aulast%3DChen%26aufirst%3DP.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DLin%26aufirst%3DG.%2BF.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DZou%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DLu%26aufirst%3DG.%2BW.%26aulast%3DLi%26aufirst%3DW.%2BM.%26aulast%3DWei%26aufirst%3DY.%2BQ.%26aulast%3DZheng%26aufirst%3DY.%2BT.%26aulast%3DLei%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DS.%26atitle%3DSARS-CoV-2%2520M%2528pro%2529%2520inhibitors%2520with%2520antiviral%2520activity%2520in%2520a%2520transgenic%2520mouse%2520model%26jtitle%3DScience%26date%3D2021%26volume%3D371%26spage%3D1374%26epage%3D1378%26doi%3D10.1126%2Fscience.abf1611" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pedersen, N. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galasiti Kankanamalage, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eckstrand, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groutas, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bannasch, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meadows, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, K. O.</span></span> <span> </span><span class="NLM_article-title">Efficacy of a 3C-like protease inhibitor in treating various forms of acquired feline infectious peritonitis</span>. <i>J. Feline Med. Surg.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">378</span>– <span class="NLM_lpage">392</span>, <span class="refDoi"> DOI: 10.1177/1098612X17729626</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=10.1177%2F1098612X17729626" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=28901812" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=1%3ACAS%3A280%3ADC%252BC1cbns12gsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2018&pages=378-392&author=N.+C.+Pedersenauthor=Y.+Kimauthor=H.+Liuauthor=A.+C.+Galasiti+Kankanamalageauthor=C.+Eckstrandauthor=W.+C.+Groutasauthor=M.+Bannaschauthor=J.+M.+Meadowsauthor=K.+O.+Chang&title=Efficacy+of+a+3C-like+protease+inhibitor+in+treating+various+forms+of+acquired+feline+infectious+peritonitis&doi=10.1177%2F1098612X17729626"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Efficacy of a 3C-like protease inhibitor in treating various forms of acquired feline infectious peritonitis</span></div><div class="casAuthors">Pedersen Niels C; Liu Hongwei; Bannasch Michael; Kim Yunjeong; Chang Kyeong-Ok; Galasiti Kankanamalage Anushka C; Groutas William C; Eckstrand Chrissy; Meadows Juliana M</div><div class="citationInfo"><span class="NLM_cas:title">Journal of feline medicine and surgery</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">378-392</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Objectives The safety and efficacy of the 3C-like protease inhibitor GC376 was tested on a cohort of client-owned cats with various forms of feline infectious peritonitis (FIP).  Methods Twenty cats from 3.3-82 months of age (mean 10.4 months) with various forms of FIP were accepted into a field trial.  Fourteen cats presented with wet or dry-to-wet FIP and six cats presented with dry FIP.  GC376 was administered subcutaneously every 12 h at a dose of 15 mg/kg.  Cats with neurologic signs were excluded from the study.  Results Nineteen of 20 cats treated with GC376 regained outward health within 2 weeks of initial treatment.  However, disease signs recurred 1-7 weeks after primary treatment and relapses and new cases were ultimately treated for a minimum of 12 weeks.  Relapses no longer responsive to treatment occurred in 13 of these 19 cats within 1-7 weeks of initial or repeat treatment(s).  Severe neurologic disease occurred in 8/13 cats that failed treatment and five cats had recurrences of abdominal lesions.  At the time of writing, seven cats were in disease remission.  Five kittens aged 3.3-4.4 months with wet FIP were treated for 12 weeks and have been in disease remission after stopping treatment and at the time of writing for 5-14 months (mean 11.2 months).  A sixth kitten was in remission for 10 weeks after 12 weeks of treatment, relapsed and is responding to a second round of GC376.  The seventh was a 6.8-year-old cat with only mesenteric lymph node involvement that went into remission after three relapses that required progressively longer repeat treatments over a 10 month period.  Side effects of treatment included transient stinging upon injection and occasional foci of subcutaneous fibrosis and hair loss.  There was retarded development and abnormal eruption of permanent teeth in cats treated before 16-18 weeks of age.  Conclusions and relevance GC376 showed promise in treating cats with certain presentations of FIP and has opened the door to targeted antiviral drug therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTFFwhHX-Ibonqj_yg7twvpfW6udTcc2ebTz9nFhCGg57ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cbns12gsg%253D%253D&md5=69b4ebebb24b351bfb0f409e275b3153</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1177%2F1098612X17729626&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1098612X17729626%26sid%3Dliteratum%253Aachs%26aulast%3DPedersen%26aufirst%3DN.%2BC.%26aulast%3DKim%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DGalasiti%2BKankanamalage%26aufirst%3DA.%2BC.%26aulast%3DEckstrand%26aufirst%3DC.%26aulast%3DGroutas%26aufirst%3DW.%2BC.%26aulast%3DBannasch%26aufirst%3DM.%26aulast%3DMeadows%26aufirst%3DJ.%2BM.%26aulast%3DChang%26aufirst%3DK.%2BO.%26atitle%3DEfficacy%2520of%2520a%25203C-like%2520protease%2520inhibitor%2520in%2520treating%2520various%2520forms%2520of%2520acquired%2520feline%2520infectious%2520peritonitis%26jtitle%3DJ.%2520Feline%2520Med.%2520Surg.%26date%3D2018%26volume%3D20%26spage%3D378%26epage%3D392%26doi%3D10.1177%2F1098612X17729626" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galasiti Kankanamalage, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weerasekara, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hua, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groutas, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, K. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pedersen, N. C.</span></span> <span> </span><span class="NLM_article-title">Reversal of the progression of fatal coronavirus infection in cats by a broad-spectrum coronavirus protease inhibitor</span>. <i>PLoS Pathog.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">e1005531</span>, <span class="refDoi"> DOI: 10.1371/journal.ppat.1005531</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=10.1371%2Fjournal.ppat.1005531" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=27027316" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlGru7fN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=e1005531&author=Y.+Kimauthor=H.+Liuauthor=A.+C.+Galasiti+Kankanamalageauthor=S.+Weerasekaraauthor=D.+H.+Huaauthor=W.+C.+Groutasauthor=K.+O.+Changauthor=N.+C.+Pedersen&title=Reversal+of+the+progression+of+fatal+coronavirus+infection+in+cats+by+a+broad-spectrum+coronavirus+protease+inhibitor&doi=10.1371%2Fjournal.ppat.1005531"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Reversal of the progression of fatal coronavirus infection in cats by a broad-spectrum coronavirus protease inhibitor</span></div><div class="casAuthors">Kim, Yunjeong; Liu, Hongwei; Kankanamalage, Anushka C. Galasiti; Weerasekara, Sahani; Hua, Duy H.; Groutas, William C.; Chang, Kyeong-Ok; Pedersen, Niels C.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Pathogens</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">e1005531/1-e1005531/18</span>CODEN:
                <span class="NLM_cas:coden">PPLACN</span>;
        ISSN:<span class="NLM_cas:issn">1553-7374</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Coronaviruses infect animals and humans causing a wide range of diseases.  The diversity of coronaviruses in many mammalian species is contributed by relatively high mutation and recombination rates during replication.  This dynamic nature of coronaviruses may facilitate cross-species transmission and shifts in tissue or cell tropism in a host, resulting in substantial change in virulence.  Feline enteric coronavirus (FECV) causes inapparent or mild enteritis in cats, but a highly fatal disease, called feline infectious peritonitis (FIP), can arise through mutation of FECV to FIP virus (FIPV).  The pathogenesis of FIP is intimately assocd. with immune responses and involves depletion of T cells, features shared by some other coronaviruses like Severe Acute Respiratory Syndrome Coronavirus.  The increasing risks of highly virulent coronavirus infections in humans or animals call for effective antiviral drugs, but no such measures are yet available.  Previously, we have reported the inhibitors that target 3C-like protease (3CLpro) with broad-spectrum activity against important human and animal coronaviruses.  Here, we evaluated the therapeutic efficacy of our 3CLpro inhibitor in lab. cats with FIP.  Exptl. FIP is 100% fatal once certain clin. and lab. signs become apparent.We found that antiviral treatment led to full recovery of cats when treatment was started at a stage of disease that would be otherwise fatal if left untreated.  Antiviral treatment was assocd. with a rapid improvement in fever, ascites, lymphopenia and gross signs of illness and cats returned to normal health within 20 days or less of treatment.  Significant redn. in viral titers was also obsd. in cats.  These results indicate that continuous virus replication is required for progression of immune-mediated inflammatory disease of FIP.  These findings may provide important insights into devising therapeutic strategies and selection of antiviral compds. for further development for important coronaviruses in animals and humans.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprSH3BnhFwZLVg90H21EOLACvtfcHk0lgHr-FpTNPDIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlGru7fN&md5=7900b2718caeaf528f10f553e0e8fa16</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1371%2Fjournal.ppat.1005531&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.ppat.1005531%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DGalasiti%2BKankanamalage%26aufirst%3DA.%2BC.%26aulast%3DWeerasekara%26aufirst%3DS.%26aulast%3DHua%26aufirst%3DD.%2BH.%26aulast%3DGroutas%26aufirst%3DW.%2BC.%26aulast%3DChang%26aufirst%3DK.%2BO.%26aulast%3DPedersen%26aufirst%3DN.%2BC.%26atitle%3DReversal%2520of%2520the%2520progression%2520of%2520fatal%2520coronavirus%2520infection%2520in%2520cats%2520by%2520a%2520broad-spectrum%2520coronavirus%2520protease%2520inhibitor%26jtitle%3DPLoS%2520Pathog.%26date%3D2016%26volume%3D12%26spage%3De1005531%26doi%3D10.1371%2Fjournal.ppat.1005531" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jacobs, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grum-Tokars, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turlington, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saldanha, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chase, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eggler, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baez-Santos, Y. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mielech, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindsley, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hodder, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mesecar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stauffer, S. R.</span></span> <span> </span><span class="NLM_article-title">Discovery, synthesis, and structure-based optimization of a series of N-(tert-butyl)-2-(N-arylamido)-2-(pyridin-3-yl) acetamides (ML188) as potent noncovalent small molecule inhibitors of the severe acute respiratory syndrome coronavirus (SARS-CoV) 3CL protease</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">534</span>– <span class="NLM_lpage">46</span>, <span class="refDoi"> DOI: 10.1021/jm301580n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm301580n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVWqtL%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=534-46&author=J.+Jacobsauthor=V.+Grum-Tokarsauthor=Y.+Zhouauthor=M.+Turlingtonauthor=S.+A.+Saldanhaauthor=P.+Chaseauthor=A.+Egglerauthor=E.+S.+Dawsonauthor=Y.+M.+Baez-Santosauthor=S.+Tomarauthor=A.+M.+Mielechauthor=S.+C.+Bakerauthor=C.+W.+Lindsleyauthor=P.+Hodderauthor=A.+Mesecarauthor=S.+R.+Stauffer&title=Discovery%2C+synthesis%2C+and+structure-based+optimization+of+a+series+of+N-%28tert-butyl%29-2-%28N-arylamido%29-2-%28pyridin-3-yl%29+acetamides+%28ML188%29+as+potent+noncovalent+small+molecule+inhibitors+of+the+severe+acute+respiratory+syndrome+coronavirus+%28SARS-CoV%29+3CL+protease&doi=10.1021%2Fjm301580n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery, Synthesis, And Structure-Based Optimization of a Series of N-(tert-Butyl)-2-(N-arylamido)-2-(pyridin-3-yl) Acetamides (ML188) as Potent Noncovalent Small Molecule Inhibitors of the Severe Acute Respiratory Syndrome Coronavirus (SARS-CoV) 3CL Protease</span></div><div class="casAuthors">Jacobs, Jon; Grum-Tokars, Valerie; Zhou, Ya; Turlington, Mark; Saldanha, S. Adrian; Chase, Peter; Eggler, Aimee; Dawson, Eric S.; Baez-Santos, Yahira M.; Tomar, Sakshi; Mielech, Anna M.; Baker, Susan C.; Lindsley, Craig W.; Hodder, Peter; Mesecar, Andrew; Stauffer, Shaun R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">534-546</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A high-throughput screen of the NIH mol. libraries sample collection and subsequent optimization of a lead dipeptide-like series of severe acute respiratory syndrome (SARS) main protease (3CLpro) inhibitors led to the identification of probe compd. ML188 (I), [(R)-N-(4-(tert-butyl)phenyl)-N-(2-(tert-butylamino)-2-oxo-1-(pyridin-3-yl)ethyl)furan-2-carboxamide, Pubchem CID: 46897844].  Unlike the majority of reported coronavirus 3CLpro inhibitors that act via covalent modification of the enzyme, I is a noncovalent SARS-CoV 3CLpro inhibitor with moderate MW and good enzyme and antiviral inhibitory activity.  A multicomponent Ugi reaction was utilized to rapidly explore structure-activity relationships within S1', S1, and S2 enzyme binding pockets.  The X-ray structure of SARS-CoV 3CLpro bound with I was instrumental in guiding subsequent rounds of chem. optimization.  I provides an excellent starting point for the further design and refinement of 3CLpro inhibitors that act by a noncovalent mechanism of action.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJ_0gMuIcLErVg90H21EOLACvtfcHk0lgHr-FpTNPDIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVWqtL%252FE&md5=6f619d5642a8d29bdb9eee0b28a80247</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Fjm301580n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm301580n%26sid%3Dliteratum%253Aachs%26aulast%3DJacobs%26aufirst%3DJ.%26aulast%3DGrum-Tokars%26aufirst%3DV.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DTurlington%26aufirst%3DM.%26aulast%3DSaldanha%26aufirst%3DS.%2BA.%26aulast%3DChase%26aufirst%3DP.%26aulast%3DEggler%26aufirst%3DA.%26aulast%3DDawson%26aufirst%3DE.%2BS.%26aulast%3DBaez-Santos%26aufirst%3DY.%2BM.%26aulast%3DTomar%26aufirst%3DS.%26aulast%3DMielech%26aufirst%3DA.%2BM.%26aulast%3DBaker%26aufirst%3DS.%2BC.%26aulast%3DLindsley%26aufirst%3DC.%2BW.%26aulast%3DHodder%26aufirst%3DP.%26aulast%3DMesecar%26aufirst%3DA.%26aulast%3DStauffer%26aufirst%3DS.%2BR.%26atitle%3DDiscovery%252C%2520synthesis%252C%2520and%2520structure-based%2520optimization%2520of%2520a%2520series%2520of%2520N-%2528tert-butyl%2529-2-%2528N-arylamido%2529-2-%2528pyridin-3-yl%2529%2520acetamides%2520%2528ML188%2529%2520as%2520potent%2520noncovalent%2520small%2520molecule%2520inhibitors%2520of%2520the%2520severe%2520acute%2520respiratory%2520syndrome%2520coronavirus%2520%2528SARS-CoV%2529%25203CL%2520protease%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D534%26epage%3D46%26doi%3D10.1021%2Fjm301580n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Turlington, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chun, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eggler, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grum-Tokars, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobs, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniels, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saldanha, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chase, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baez-Santos, Y. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindsley, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hodder, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mesecar, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stauffer, S. R.</span></span> <span> </span><span class="NLM_article-title">Discovery of N-(benzo[1,2,3]triazol-1-yl)-N-(benzyl)acetamido)phenyl) carboxamides as severe acute respiratory syndrome coronavirus (SARS-CoV) 3CLpro inhibitors: Identification of ML300 and noncovalent nanomolar inhibitors with an induced-fit binding</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">6172</span>– <span class="NLM_lpage">6177</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2013.08.112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=10.1016%2Fj.bmcl.2013.08.112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=24080461" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFGqtr3L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2013&pages=6172-6177&author=M.+Turlingtonauthor=A.+Chunauthor=S.+Tomarauthor=A.+Egglerauthor=V.+Grum-Tokarsauthor=J.+Jacobsauthor=J.+S.+Danielsauthor=E.+Dawsonauthor=A.+Saldanhaauthor=P.+Chaseauthor=Y.+M.+Baez-Santosauthor=C.+W.+Lindsleyauthor=P.+Hodderauthor=A.+D.+Mesecarauthor=S.+R.+Stauffer&title=Discovery+of+N-%28benzo%5B1%2C2%2C3%5Dtriazol-1-yl%29-N-%28benzyl%29acetamido%29phenyl%29+carboxamides+as+severe+acute+respiratory+syndrome+coronavirus+%28SARS-CoV%29+3CLpro+inhibitors%3A+Identification+of+ML300+and+noncovalent+nanomolar+inhibitors+with+an+induced-fit+binding&doi=10.1016%2Fj.bmcl.2013.08.112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of N-(benzo[1,2,3]triazol-1-yl)-N-((benzyl)acetamido)phenyl carboxamides as severe acute respiratory syndrome coronavirus (SARS-CoV) 3CLpro inhibitors: Identification of ML300 and noncovalent nanomolar inhibitors with an induced-fit binding</span></div><div class="casAuthors">Turlington, Mark; Chun, Aspen; Tomar, Sakshi; Eggler, Aimee; Grum-Tokars, Valerie; Jacobs, Jon; Daniels, J. Scott; Dawson, Eric; Saldanha, Adrian; Chase, Peter; Baez-Santos, Yahira M.; Lindsley, Craig W.; Hodder, Peter; Mesecar, Andrew D.; Stauffer, Shaun R.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">6172-6177</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Herein we report the discovery and SAR of a novel series of SARS-CoV 3CLpro inhibitors identified through the NIH Mol. Libraries Probe Prodn. Centers Network (MLPCN).  In addn. to ML188, ML300 represents the second probe declared for 3CLpro from this collaborative effort.  The x-ray structure of SARS-CoV 3CLpro bound with a ML300 analog highlights a unique induced-fit reorganization of the S2-S4 binding pockets leading to the first sub-micromolar noncovalent 3CLpro inhibitors retaining a single amide bond.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSJzh-nO55O7Vg90H21EOLACvtfcHk0lhdNJDNlmT2qw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFGqtr3L&md5=f4ee0a6a9222900ad75822851f22b2be</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2013.08.112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2013.08.112%26sid%3Dliteratum%253Aachs%26aulast%3DTurlington%26aufirst%3DM.%26aulast%3DChun%26aufirst%3DA.%26aulast%3DTomar%26aufirst%3DS.%26aulast%3DEggler%26aufirst%3DA.%26aulast%3DGrum-Tokars%26aufirst%3DV.%26aulast%3DJacobs%26aufirst%3DJ.%26aulast%3DDaniels%26aufirst%3DJ.%2BS.%26aulast%3DDawson%26aufirst%3DE.%26aulast%3DSaldanha%26aufirst%3DA.%26aulast%3DChase%26aufirst%3DP.%26aulast%3DBaez-Santos%26aufirst%3DY.%2BM.%26aulast%3DLindsley%26aufirst%3DC.%2BW.%26aulast%3DHodder%26aufirst%3DP.%26aulast%3DMesecar%26aufirst%3DA.%2BD.%26aulast%3DStauffer%26aufirst%3DS.%2BR.%26atitle%3DDiscovery%2520of%2520N-%2528benzo%255B1%252C2%252C3%255Dtriazol-1-yl%2529-N-%2528benzyl%2529acetamido%2529phenyl%2529%2520carboxamides%2520as%2520severe%2520acute%2520respiratory%2520syndrome%2520coronavirus%2520%2528SARS-CoV%2529%25203CLpro%2520inhibitors%253A%2520Identification%2520of%2520ML300%2520and%2520noncovalent%2520nanomolar%2520inhibitors%2520with%2520an%2520induced-fit%2520binding%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2013%26volume%3D23%26spage%3D6172%26epage%3D6177%26doi%3D10.1016%2Fj.bmcl.2013.08.112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zaidman, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gehrtz, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filep, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fearon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prilusky, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duberstein, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Resnick, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strain-Damerell, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lukacik, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barr, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walsh, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Delft, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">London, N.</span></span> <span> </span><span class="NLM_article-title">An automatic pipeline for the design of irreversible derivatives identifies a potent SARS-CoV-2 Mpro inhibitor</span>. <i>bioRxiv</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_fpage">2020.09.21.299776</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&pages=2020.09.21.299776&author=D.+Zaidmanauthor=P.+Gehrtzauthor=M.+Filepauthor=D.+Fearonauthor=J.+Priluskyauthor=S.+Dubersteinauthor=G.+Cohenauthor=D.+Owenauthor=E.+Resnickauthor=C.+Strain-Damerellauthor=P.+Lukacikauthor=H.+Barrauthor=M.+A.+Walshauthor=F.+von+Delftauthor=N.+London&title=An+automatic+pipeline+for+the+design+of+irreversible+derivatives+identifies+a+potent+SARS-CoV-2+Mpro+inhibitor"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DZaidman%26aufirst%3DD.%26aulast%3DGehrtz%26aufirst%3DP.%26aulast%3DFilep%26aufirst%3DM.%26aulast%3DFearon%26aufirst%3DD.%26aulast%3DPrilusky%26aufirst%3DJ.%26aulast%3DDuberstein%26aufirst%3DS.%26aulast%3DCohen%26aufirst%3DG.%26aulast%3DOwen%26aufirst%3DD.%26aulast%3DResnick%26aufirst%3DE.%26aulast%3DStrain-Damerell%26aufirst%3DC.%26aulast%3DLukacik%26aufirst%3DP.%26aulast%3DBarr%26aufirst%3DH.%26aulast%3DWalsh%26aufirst%3DM.%2BA.%26aulast%3Dvon%2BDelft%26aufirst%3DF.%26aulast%3DLondon%26aufirst%3DN.%26atitle%3DAn%2520automatic%2520pipeline%2520for%2520the%2520design%2520of%2520irreversible%2520derivatives%2520identifies%2520a%2520potent%2520SARS-CoV-2%2520Mpro%2520inhibitor%26jtitle%3DbioRxiv%26date%3D2020%26spage%3D2020.09.21.299776" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, D. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X. Y. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curth, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drosten, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sauerhering, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rox, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilgenfeld, R.</span></span> <span> </span><span class="NLM_article-title">Crystal structure of SARS-CoV-2 main protease provides a basis for design of improved alpha-ketoamide inhibitors</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>368</i></span>,  <span class="NLM_fpage">409</span>– <span class="NLM_lpage">412</span>, <span class="refDoi"> DOI: 10.1126/science.abb3405</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=10.1126%2Fscience.abb3405" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=32198291" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=1%3ACAS%3A528%3ADC%252BB3cXnslKrtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=368&publication_year=2020&pages=409-412&author=L.+L.+Zhangauthor=D.+Z.+Linauthor=X.+Y.+Y.+Sunauthor=U.+Curthauthor=C.+Drostenauthor=L.+Sauerheringauthor=S.+Beckerauthor=K.+Roxauthor=R.+Hilgenfeld&title=Crystal+structure+of+SARS-CoV-2+main+protease+provides+a+basis+for+design+of+improved+alpha-ketoamide+inhibitors&doi=10.1126%2Fscience.abb3405"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of SARS-CoV-2 main protease provides a basis for design of improved α-ketoamide inhibitors</span></div><div class="casAuthors">Zhang, Linlin; Lin, Daizong; Sun, Xinyuanyuan; Curth, Ute; Drosten, Christian; Sauerhering, Lucie; Becker, Stephan; Rox, Katharina; Hilgenfeld, Rolf</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">368</span>
        (<span class="NLM_cas:issue">6489</span>),
    <span class="NLM_cas:pages">409-412</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">1095-9203</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The coronavirus disease 2019 (COVID-19) pandemic caused by severe acute respiratory syndrome-coronavirus 2 (SARS-CoV-2) is a global health emergency.  An attractive drug target among coronaviruses is the main protease (Mpro, also called 3CLpro) because of its essential role in processing the polyproteins that are translated from the viral RNA.  We report the x-ray structures of the unliganded SARS-CoV-2 Mpro and its complex with an α-ketoamide inhibitor.  This was derived from a previously designed inhibitor but with the P3-P2 amide bond incorporated into a pyridone ring to enhance the half-life of the compd. in plasma.  On the basis of the unliganded structure, we developed the lead compd. into a potent inhibitor of the SARS-CoV-2 Mpro.  The pharmacokinetic characterization of the optimized inhibitor reveals a pronounced lung tropism and suitability for administration by the inhalative route.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrT531N47m4BbVg90H21EOLACvtfcHk0liOalupV9Skaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXnslKrtL8%253D&md5=9ac417c20f54c3327f9de9088b512d52</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1126%2Fscience.abb3405&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.abb3405%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DL.%2BL.%26aulast%3DLin%26aufirst%3DD.%2BZ.%26aulast%3DSun%26aufirst%3DX.%2BY.%2BY.%26aulast%3DCurth%26aufirst%3DU.%26aulast%3DDrosten%26aufirst%3DC.%26aulast%3DSauerhering%26aufirst%3DL.%26aulast%3DBecker%26aufirst%3DS.%26aulast%3DRox%26aufirst%3DK.%26aulast%3DHilgenfeld%26aufirst%3DR.%26atitle%3DCrystal%2520structure%2520of%2520SARS-CoV-2%2520main%2520protease%2520provides%2520a%2520basis%2520for%2520design%2520of%2520improved%2520alpha-ketoamide%2520inhibitors%26jtitle%3DScience%26date%3D2020%26volume%3D368%26spage%3D409%26epage%3D412%26doi%3D10.1126%2Fscience.abb3405" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foley, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Musharrafieh, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hulme, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span> <span> </span><span class="NLM_article-title">Exploring ugi-azide four-component reaction products for broad-spectrum influenza antivirals with a high genetic barrier to drug resistance</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">4653</span>, <span class="refDoi"> DOI: 10.1038/s41598-018-22875-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=10.1038%2Fs41598-018-22875-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=29545578" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=1%3ACAS%3A280%3ADC%252BC1MnhslKhsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&pages=4653&author=J.+Zhangauthor=Y.+Huauthor=C.+Foleyauthor=Y.+Wangauthor=R.+Musharrafiehauthor=S.+Xuauthor=Y.+Zhangauthor=C.+Maauthor=C.+Hulmeauthor=J.+Wang&title=Exploring+ugi-azide+four-component+reaction+products+for+broad-spectrum+influenza+antivirals+with+a+high+genetic+barrier+to+drug+resistance&doi=10.1038%2Fs41598-018-22875-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Exploring Ugi-Azide Four-Component Reaction Products for Broad-Spectrum Influenza Antivirals with a High Genetic Barrier to Drug Resistance</span></div><div class="casAuthors">Zhang Jiantao; Hu Yanmei; Wang Yuanxiang; Wang Jun; Foley Christopher; Musharrafieh Rami; Xu Shuting; Zhang Yongtao; Hulme Christopher; Ma Chunlong; Wang Jun</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">4653</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Influenza viruses are respiratory pathogens that are responsible for seasonal influenza and sporadic influenza pandemic.  The therapeutic efficacy of current influenza vaccines and small molecule antiviral drugs is limited due to the emergence of multidrug-resistant influenza viruses.  In response to the urgent need for the next generation of influenza antivirals, we utilized a fast-track drug discovery platform by exploring multi-component reaction products for antiviral drug candidates.  Specifically, molecular docking was applied to screen a small molecule library derived from the Ugi-azide four-component reaction methodology for inhibitors that target the influenza polymerase PAC-PB1N interactions.  One hit compound 5 was confirmed to inhibit PAC-PB1N interactions in an ELISA assay and had potent antiviral activity in an antiviral plaque assay.  Subsequent structure-activity relationship studies led to the discovery of compound 12a, which had broad-spectrum antiviral activity and a higher in vitro genetic barrier to drug resistance than oseltamivir.  Overall, the discovery of compound 12a as a broad-spectrum influenza antiviral with a high in vitro genetic barrier to drug resistance is significant, as it offers a second line of defense to combat the next influenza epidemics and pandemics if vaccines and oseltamivir fail to confine the disease outbreak.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRR96PkqgseZ6_dlEzl_fBafW6udTcc2eZEevu01r_fI7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MnhslKhsA%253D%253D&md5=efc7cdeea4a5099a341aa8d177f0f8c8</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fs41598-018-22875-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-018-22875-9%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DFoley%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DMusharrafieh%26aufirst%3DR.%26aulast%3DXu%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DC.%26aulast%3DHulme%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DJ.%26atitle%3DExploring%2520ugi-azide%2520four-component%2520reaction%2520products%2520for%2520broad-spectrum%2520influenza%2520antivirals%2520with%2520a%2520high%2520genetic%2520barrier%2520to%2520drug%2520resistance%26jtitle%3DSci.%2520Rep.%26date%3D2018%26volume%3D8%26spage%3D4653%26doi%3D10.1038%2Fs41598-018-22875-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Siklos, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">BenAissa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thatcher, G. R. J.</span></span> <span> </span><span class="NLM_article-title">Cysteine proteases as therapeutic targets: does selectivity matter? A systematic review of calpain and cathepsin inhibitors</span>. <i>Acta Pharm. Sin. B</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">506</span>– <span class="NLM_lpage">519</span>, <span class="refDoi"> DOI: 10.1016/j.apsb.2015.08.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=10.1016%2Fj.apsb.2015.08.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=26713267" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=1%3ACAS%3A280%3ADC%252BC28rlslWjsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=506-519&author=M.+Siklosauthor=M.+BenAissaauthor=G.+R.+J.+Thatcher&title=Cysteine+proteases+as+therapeutic+targets%3A+does+selectivity+matter%3F+A+systematic+review+of+calpain+and+cathepsin+inhibitors&doi=10.1016%2Fj.apsb.2015.08.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Cysteine proteases as therapeutic targets: does selectivity matter? A systematic review of calpain and cathepsin inhibitors</span></div><div class="casAuthors">Siklos Marton; BenAissa Manel; Thatcher Gregory R J</div><div class="citationInfo"><span class="NLM_cas:title">Acta pharmaceutica Sinica. B</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">506-19</span>
        ISSN:<span class="NLM_cas:issn">2211-3835</span>.
    </div><div class="casAbstract">Cysteine proteases continue to provide validated targets for treatment of human diseases.  In neurodegenerative disorders, multiple cysteine proteases provide targets for enzyme inhibitors, notably caspases, calpains, and cathepsins.  The reactive, active-site cysteine provides specificity for many inhibitor designs over other families of proteases, such as aspartate and serine; however, a) inhibitor strategies often use covalent enzyme modification, and b) obtaining selectivity within families of cysteine proteases and their isozymes is problematic.  This review provides a general update on strategies for cysteine protease inhibitor design and a focus on cathepsin B and calpain 1 as drug targets for neurodegenerative disorders; the latter focus providing an interesting query for the contemporary assumptions that irreversible, covalent protein modification and low selectivity are anathema to therapeutic safety and efficacy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQE6JaFL9P83mBAiYWy6n3SfW6udTcc2eZEevu01r_fI7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28rlslWjsg%253D%253D&md5=cbd246a39984c0d9e3a32fd26db8902d</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.apsb.2015.08.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.apsb.2015.08.001%26sid%3Dliteratum%253Aachs%26aulast%3DSiklos%26aufirst%3DM.%26aulast%3DBenAissa%26aufirst%3DM.%26aulast%3DThatcher%26aufirst%3DG.%2BR.%2BJ.%26atitle%3DCysteine%2520proteases%2520as%2520therapeutic%2520targets%253A%2520does%2520selectivity%2520matter%253F%2520A%2520systematic%2520review%2520of%2520calpain%2520and%2520cathepsin%2520inhibitors%26jtitle%3DActa%2520Pharm.%2520Sin.%2520B%26date%3D2015%26volume%3D5%26spage%3D506%26epage%3D519%26doi%3D10.1016%2Fj.apsb.2015.08.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cianni, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feldmann, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilberg, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gutschow, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Juliano, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leitao, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montanari, C. A.</span></span> <span> </span><span class="NLM_article-title">Can cysteine protease cross-class inhibitors achieve selectivity?</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">10497</span>– <span class="NLM_lpage">10525</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00683</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00683" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsVGkur7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=10497-10525&author=L.+Cianniauthor=C.+W.+Feldmannauthor=E.+Gilbergauthor=M.+Gutschowauthor=L.+Julianoauthor=A.+Leitaoauthor=J.+Bajorathauthor=C.+A.+Montanari&title=Can+cysteine+protease+cross-class+inhibitors+achieve+selectivity%3F&doi=10.1021%2Facs.jmedchem.9b00683"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Can Cysteine Protease Cross-Class Inhibitors Achieve Selectivity?</span></div><div class="casAuthors">Cianni, Lorenzo; Feldmann, Christian Wolfgang; Gilberg, Erik; Gutschow, Michael; Juliano, Luiz; Leitao, Andrei; Bajorath, Jurgen; Montanari, Carlos A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">10497-10525</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Cysteine proteases are important targets for the discovery of novel therapeutics for many human diseases.  From parasitic diseases to cancer, cysteine proteases follow a common mechanism, the formation of an encounter complex with subsequent nucleophilic reactivity of the catalytic cysteine thiol group toward the carbonyl carbon of a peptide bond or an electrophilic group of an inhibitor.  Modulation of target enzymes occurs preferably by covalent modification, which imposes challenges in balancing cross-reactivity and selectivity.  Given the resurgence of irreversible covalent inhibitors, can they impair off-target effects or are reversible covalent inhibitors a better route to selectivity.  This Perspective addresses how small mol. inhibitors may achieve selectivity for different cathepsins, cruzain, rhodesain, and falcipain-2.  We discuss target- and ligand-based designs emphasizing repurposing inhibitors from one cysteine protease to others.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr75s_t-OU9gbVg90H21EOLACvtfcHk0ligbU07_a-P7g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsVGkur7I&md5=49fc0900174dfad403d33e906498ae28</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00683&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00683%26sid%3Dliteratum%253Aachs%26aulast%3DCianni%26aufirst%3DL.%26aulast%3DFeldmann%26aufirst%3DC.%2BW.%26aulast%3DGilberg%26aufirst%3DE.%26aulast%3DGutschow%26aufirst%3DM.%26aulast%3DJuliano%26aufirst%3DL.%26aulast%3DLeitao%26aufirst%3DA.%26aulast%3DBajorath%26aufirst%3DJ.%26aulast%3DMontanari%26aufirst%3DC.%2BA.%26atitle%3DCan%2520cysteine%2520protease%2520cross-class%2520inhibitors%2520achieve%2520selectivity%253F%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D10497%26epage%3D10525%26doi%3D10.1021%2Facs.jmedchem.9b00683" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Steuten, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Widen, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Babin, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Onguka, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lovell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolgi, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cerikan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cortese, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muir, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geiss-Friedlander, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peters, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartenschlager, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bogyo, M.</span></span> <span> </span><span class="NLM_article-title">Challenges for targeting SARS-CoV-2 proteases as a therapeutic strategy for COVID-19</span>. <i>ACS Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>, <span class="refDoi"> DOI: 10.1021/acsinfecdis.0c00815</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsinfecdis.0c00815" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2021&author=K.+Steutenauthor=H.+Kimauthor=J.+C.+Widenauthor=B.+M.+Babinauthor=O.+Ongukaauthor=S.+Lovellauthor=O.+Bolgiauthor=B.+Cerikanauthor=M.+Corteseauthor=R.+K.+Muirauthor=J.+M.+Bennettauthor=R.+Geiss-Friedlanderauthor=C.+Petersauthor=R.+Bartenschlagerauthor=M.+Bogyo&title=Challenges+for+targeting+SARS-CoV-2+proteases+as+a+therapeutic+strategy+for+COVID-19&doi=10.1021%2Facsinfecdis.0c00815"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facsinfecdis.0c00815&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsinfecdis.0c00815%26sid%3Dliteratum%253Aachs%26aulast%3DSteuten%26aufirst%3DK.%26aulast%3DKim%26aufirst%3DH.%26aulast%3DWiden%26aufirst%3DJ.%2BC.%26aulast%3DBabin%26aufirst%3DB.%2BM.%26aulast%3DOnguka%26aufirst%3DO.%26aulast%3DLovell%26aufirst%3DS.%26aulast%3DBolgi%26aufirst%3DO.%26aulast%3DCerikan%26aufirst%3DB.%26aulast%3DCortese%26aufirst%3DM.%26aulast%3DMuir%26aufirst%3DR.%2BK.%26aulast%3DBennett%26aufirst%3DJ.%2BM.%26aulast%3DGeiss-Friedlander%26aufirst%3DR.%26aulast%3DPeters%26aufirst%3DC.%26aulast%3DBartenschlager%26aufirst%3DR.%26aulast%3DBogyo%26aufirst%3DM.%26atitle%3DChallenges%2520for%2520targeting%2520SARS-CoV-2%2520proteases%2520as%2520a%2520therapeutic%2520strategy%2520for%2520COVID-19%26jtitle%3DACS%2520Infect.%2520Dis.%26date%3D2021%26doi%3D10.1021%2Facsinfecdis.0c00815" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meek, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shang, L.</span></span> <span> </span><span class="NLM_article-title">Application of dually activated michael acceptor to the rational design of reversible covalent inhibitor for enterovirus 71 3C protease</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">6146</span>– <span class="NLM_lpage">6162</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00387</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00387" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtFGqsLjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=6146-6162&author=Y.+Maauthor=L.+Liauthor=S.+Heauthor=C.+Shangauthor=Y.+Sunauthor=N.+Liuauthor=T.+D.+Meekauthor=Y.+Wangauthor=L.+Shang&title=Application+of+dually+activated+michael+acceptor+to+the+rational+design+of+reversible+covalent+inhibitor+for+enterovirus+71+3C+protease&doi=10.1021%2Facs.jmedchem.9b00387"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Application of Dually Activated Michael Acceptor to the Rational Design of Reversible Covalent Inhibitor for Enterovirus 71 3C Protease</span></div><div class="casAuthors">Ma, Yuying; Li, Linfeng; He, Shuai; Shang, Chengyou; Sun, Yang; Liu, Ning; Meek, Thomas D.; Wang, Yaxin; Shang, Luqing</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">6146-6162</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Targeted covalent inhibitors (TCIs) have attracted growing attention from the pharmaceutical industry in recent decades because they have potential advantages in terms of efficacy, selectivity, and safety.  TCIs have recently evolved into a new version with reversibility that can be systematically modulated.  This feature may diminish the risk of haptenization and help optimize the drug-target residence time as needed.  The enteroviral 3C protease (3Cpro) is a valuable therapeutic target, but the development of 3Cpro inhibitors is far from satisfactory.  Therefore, we aimed to apply a reversible TCI approach to the design of novel 3Cpro inhibitors.  The introduction of various substituents onto the α-carbon of classical Michael acceptors yielded inhibitors bearing several classes of warheads.  Using steady-state kinetics and biomol. mass spectrometry, we confirmed the mode of reversible covalent inhibition and elucidated the mechanism by which the potency and reversibility were affected by electronic and steric factors.  This research produced several potent inhibitors with good selectivity and suitable reversibility; moreover, it validated the reversible TCI approach in the field of viral infection, suggesting broader applications in the design of reversible covalent inhibitors for other proteases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkpF34VsXG8bVg90H21EOLACvtfcHk0ligbU07_a-P7g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtFGqsLjI&md5=2de757026f2542bff8092060a72107a5</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00387&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00387%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DHe%26aufirst%3DS.%26aulast%3DShang%26aufirst%3DC.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DN.%26aulast%3DMeek%26aufirst%3DT.%2BD.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DShang%26aufirst%3DL.%26atitle%3DApplication%2520of%2520dually%2520activated%2520michael%2520acceptor%2520to%2520the%2520rational%2520design%2520of%2520reversible%2520covalent%2520inhibitor%2520for%2520enterovirus%252071%25203C%2520protease%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D6146%26epage%3D6162%26doi%3D10.1021%2Facs.jmedchem.9b00387" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jin, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guddat, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span> <span> </span><span class="NLM_article-title">Structure of M(pro) from SARS-CoV-2 and discovery of its inhibitors</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>582</i></span>,  <span class="NLM_fpage">289</span>– <span class="NLM_lpage">293</span>, <span class="refDoi"> DOI: 10.1038/s41586-020-2223-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=10.1038%2Fs41586-020-2223-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=32272481" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtVyhsrrO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=582&publication_year=2020&pages=289-293&author=Z.+Jinauthor=X.+Duauthor=Y.+Xuauthor=Y.+Dengauthor=M.+Liuauthor=Y.+Zhaoauthor=B.+Zhangauthor=X.+Liauthor=L.+Zhangauthor=C.+Pengauthor=Y.+Duanauthor=J.+Yuauthor=L.+Wangauthor=K.+Yangauthor=F.+Liuauthor=R.+Jiangauthor=X.+Yangauthor=T.+Youauthor=X.+Liuauthor=X.+Yangauthor=F.+Baiauthor=H.+Liuauthor=X.+Liuauthor=L.+W.+Guddatauthor=W.+Xuauthor=G.+Xiaoauthor=C.+Qinauthor=Z.+Shiauthor=H.+Jiangauthor=Z.+Raoauthor=H.+Yang&title=Structure+of+M%28pro%29+from+SARS-CoV-2+and+discovery+of+its+inhibitors&doi=10.1038%2Fs41586-020-2223-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of Mpro from SARS-CoV-2 and discovery of its inhibitors</span></div><div class="casAuthors">Jin, Zhenming; Du, Xiaoyu; Xu, Yechun; Deng, Yongqiang; Liu, Meiqin; Zhao, Yao; Zhang, Bing; Li, Xiaofeng; Zhang, Leike; Peng, Chao; Duan, Yinkai; Yu, Jing; Wang, Lin; Yang, Kailin; Liu, Fengjiang; Jiang, Rendi; Yang, Xinglou; You, Tian; Liu, Xiaoce; Yang, Xiuna; Bai, Fang; Liu, Hong; Liu, Xiang; Guddat, Luke W.; Xu, Wenqing; Xiao, Gengfu; Qin, Chengfeng; Shi, Zhengli; Jiang, Hualiang; Rao, Zihe; Yang, Haitao</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">582</span>
        (<span class="NLM_cas:issue">7811</span>),
    <span class="NLM_cas:pages">289-293</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: A new coronavirus, known as severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), is the etiol. agent responsible for the 2019-2020 viral pneumonia outbreak of coronavirus disease 2019 (COVID-19).  Currently, there are no targeted therapeutic agents for the treatment of this disease, and effective treatment options remain very limited.  Here, we describe the results of a program that aimed to rapidly discover lead compds. for clin. use, by combining structure-assisted drug design, virtual drug screening and high-throughput screening.  This program focused on identifying drug leads that target main protease (Mpro) of SARS-CoV-2: Mpro is a key enzyme of coronaviruses and has a pivotal role in mediating viral replication and transcription, making it an attractive drug target for SARS-CoV-2.  We identified a mechanism-based inhibitor (N3) by computer-aided drug design, and then detd. the crystal structure of Mpro of SARS-CoV-2 in complex with this compd.  Through a combination of structure-based virtual and high-throughput screening, we assayed more than 10,000 compds.-including approved drugs, drug candidates in clin. trials and other pharmacol. active compds.-as inhibitors of Mpro.  Six of these compds. inhibited Mpro, showing half-maximal inhibitory concn. values that ranged from 0.67 to 21.4μM.  One of these compds. (ebselen) also exhibited promising antiviral activity in cell-based assays.  Our results demonstrate the efficacy of our screening strategy, which can lead to the rapid discovery of drug leads with clin. potential in response to new infectious diseases for which no specific drugs or vaccines are available.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFupuh6WIyW7Vg90H21EOLACvtfcHk0ljngCn2Z9YH1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtVyhsrrO&md5=b84f350fe9ce1109485df6caf814ba82</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fs41586-020-2223-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-020-2223-y%26sid%3Dliteratum%253Aachs%26aulast%3DJin%26aufirst%3DZ.%26aulast%3DDu%26aufirst%3DX.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DDeng%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DM.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DB.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DPeng%26aufirst%3DC.%26aulast%3DDuan%26aufirst%3DY.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DJiang%26aufirst%3DR.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DYou%26aufirst%3DT.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DBai%26aufirst%3DF.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DGuddat%26aufirst%3DL.%2BW.%26aulast%3DXu%26aufirst%3DW.%26aulast%3DXiao%26aufirst%3DG.%26aulast%3DQin%26aufirst%3DC.%26aulast%3DShi%26aufirst%3DZ.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DRao%26aufirst%3DZ.%26aulast%3DYang%26aufirst%3DH.%26atitle%3DStructure%2520of%2520M%2528pro%2529%2520from%2520SARS-CoV-2%2520and%2520discovery%2520of%2520its%2520inhibitors%26jtitle%3DNature%26date%3D2020%26volume%3D582%26spage%3D289%26epage%3D293%26doi%3D10.1038%2Fs41586-020-2223-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCormick, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">Crystal structure of SARS-CoV-2 main protease in complex with the natural product inhibitor shikonin illuminates a unique binding mode</span>. <i>Sci. Bull. (Beijing)</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">661</span>– <span class="NLM_lpage">663</span>, <span class="refDoi"> DOI: 10.1016/j.scib.2020.10.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=10.1016%2Fj.scib.2020.10.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=33163253" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=1%3ACAS%3A280%3ADC%252BB3s3itFSisQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2021&pages=661-663&author=J.+Liauthor=X.+Zhouauthor=Y.+Zhangauthor=F.+Zhongauthor=C.+Linauthor=P.+J.+McCormickauthor=F.+Jiangauthor=J.+Luoauthor=H.+Zhouauthor=Q.+Wangauthor=Y.+Fuauthor=J.+Duanauthor=J.+Zhang&title=Crystal+structure+of+SARS-CoV-2+main+protease+in+complex+with+the+natural+product+inhibitor+shikonin+illuminates+a+unique+binding+mode&doi=10.1016%2Fj.scib.2020.10.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of SARS-CoV-2 main protease in complex with the natural product inhibitor shikonin illuminates a unique binding mode</span></div><div class="casAuthors">Li Jian; Jiang Feng; Zhou Xuelan; Zhang Jin; Zhang Yan; Luo Jun; Zhong Fanglin; Zhong Fanglin; Lin Cheng; McCormick Peter J; Zhou Huan; Wang Qisheng; Fu Yang; Duan Jingjing</div><div class="citationInfo"><span class="NLM_cas:title">Science bulletin</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">661-663</span>
        ISSN:<span class="NLM_cas:issn">2095-9273</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSWIZtDY9yqhtTiMd2aNQVUfW6udTcc2eZrvaEcdU1G6Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3s3itFSisQ%253D%253D&md5=48cdb7f3617a95c92e6cb6a9381f24aa</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.scib.2020.10.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.scib.2020.10.018%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DZhong%26aufirst%3DF.%26aulast%3DLin%26aufirst%3DC.%26aulast%3DMcCormick%26aufirst%3DP.%2BJ.%26aulast%3DJiang%26aufirst%3DF.%26aulast%3DLuo%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DFu%26aufirst%3DY.%26aulast%3DDuan%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DCrystal%2520structure%2520of%2520SARS-CoV-2%2520main%2520protease%2520in%2520complex%2520with%2520the%2520natural%2520product%2520inhibitor%2520shikonin%2520illuminates%2520a%2520unique%2520binding%2520mode%26jtitle%3DSci.%2520Bull.%2520%2528Beijing%2529%26date%3D2021%26volume%3D66%26spage%3D661%26epage%3D663%26doi%3D10.1016%2Fj.scib.2020.10.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ma, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Townsend, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lagarias, P. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marty, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolocouris, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span> <span> </span><span class="NLM_article-title">Ebselen, disulfiram, carmofur, PX-12, tideglusib, and shikonin are nonspecific promiscuous SARS-CoV-2 main protease inhibitors</span>. <i>ACS Pharmacol. Transl. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">1265</span>– <span class="NLM_lpage">1277</span>, <span class="refDoi"> DOI: 10.1021/acsptsci.0c00130</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsptsci.0c00130" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=1%3ACAS%3A528%3ADC%252BB3cXitVajsrjK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2020&pages=1265-1277&author=C.+Maauthor=Y.+Huauthor=J.+A.+Townsendauthor=P.+I.+Lagariasauthor=M.+T.+Martyauthor=A.+Kolocourisauthor=J.+Wang&title=Ebselen%2C+disulfiram%2C+carmofur%2C+PX-12%2C+tideglusib%2C+and+shikonin+are+nonspecific+promiscuous+SARS-CoV-2+main+protease+inhibitors&doi=10.1021%2Facsptsci.0c00130"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Ebselen, Disulfiram, Carmofur, PX-12, Tideglusib, and Shikonin Are Nonspecific Promiscuous SARS-CoV-2 Main Protease Inhibitors</span></div><div class="casAuthors">Ma, Chunlong; Hu, Yanmei; Townsend, Julia Alma; Lagarias, Panagiotis I.; Marty, Michael Thomas; Kolocouris, Antonios; Wang, Jun</div><div class="citationInfo"><span class="NLM_cas:title">ACS Pharmacology & Translational Science</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1265-1277</span>CODEN:
                <span class="NLM_cas:coden">APTSFN</span>;
        ISSN:<span class="NLM_cas:issn">2575-9108</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Among the drug targets being investigated for SARS-CoV-2, the viral main protease (Mpro) is one of the most extensively studied.  Mpro is a cysteine protease that hydrolyzes the viral polyprotein at more than 11 sites.  It is highly conserved and has a unique substrate preference for glutamine in the P1 position.  Therefore, Mpro inhibitors are expected to have broad-spectrum antiviral activity and a high selectivity index.  Structurally diverse compds. have been reported as Mpro inhibitors.  In this study, we investigated the mechanism of action of six previously reported Mpro inhibitors, ebselen, disulfiram, tideglusib, carmofur, shikonin, and PX-12, using a consortium of techniques including FRET-based enzymic assay, thermal shift assay, native mass spectrometry, cellular antiviral assays, and mol. dynamics simulations.  Collectively, the results showed that the inhibition of Mpro by these six compds. is nonspecific and that the inhibition is abolished or greatly reduced with the addn. of reducing reagent 1,4-dithiothreitol (DTT).  Without DTT, these six compds. inhibit not only Mpro but also a panel of viral cysteine proteases including SARS-CoV-2 papain-like protease and 2Apro and 3Cpro from enterovirus A71 (EV-A71) and EV-D68.  However, none of the compds. inhibits the viral replication of EV-A71 or EV-D68, suggesting that the enzymic inhibition potency IC50 values obtained in the absence of DTT cannot be used to faithfully predict their cellular antiviral activity.  Overall, we provide compelling evidence suggesting that these six compds. are nonspecific SARS-CoV-2 Mpro inhibitors and urge the scientific community to be stringent with hit validation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFcJU-O-vjurVg90H21EOLACvtfcHk0ljngCn2Z9YH1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXitVajsrjK&md5=29d0e69db20f681ed4ef7cddd153cc6d</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Facsptsci.0c00130&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsptsci.0c00130%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DC.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DTownsend%26aufirst%3DJ.%2BA.%26aulast%3DLagarias%26aufirst%3DP.%2BI.%26aulast%3DMarty%26aufirst%3DM.%2BT.%26aulast%3DKolocouris%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DJ.%26atitle%3DEbselen%252C%2520disulfiram%252C%2520carmofur%252C%2520PX-12%252C%2520tideglusib%252C%2520and%2520shikonin%2520are%2520nonspecific%2520promiscuous%2520SARS-CoV-2%2520main%2520protease%2520inhibitors%26jtitle%3DACS%2520Pharmacol.%2520Transl.%2520Sci.%26date%3D2020%26volume%3D3%26spage%3D1265%26epage%3D1277%26doi%3D10.1021%2Facsptsci.0c00130" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gurard-Levin, Z. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jekle, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaisinghani, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vandyck, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jochmans, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leyssen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neyts, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blatt, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beigelman, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Symons, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raboisson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scholle, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deval, J.</span></span> <span> </span><span class="NLM_article-title">Evaluation of SARS-CoV-2 3C-like protease inhibitors using self-assembled monolayer desorption ionization mass spectrometry</span>. <i>Antiviral Res.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>182</i></span>,  <span class="NLM_fpage">104924</span>, <span class="refDoi"> DOI: 10.1016/j.antiviral.2020.104924</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=10.1016%2Fj.antiviral.2020.104924" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=32896566" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhvVSksrvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=182&publication_year=2020&pages=104924&author=Z.+A.+Gurard-Levinauthor=C.+Liuauthor=A.+Jekleauthor=R.+Jaisinghaniauthor=S.+Renauthor=K.+Vandyckauthor=D.+Jochmansauthor=P.+Leyssenauthor=J.+Neytsauthor=L.+M.+Blattauthor=L.+Beigelmanauthor=J.+A.+Symonsauthor=P.+Raboissonauthor=M.+D.+Scholleauthor=J.+Deval&title=Evaluation+of+SARS-CoV-2+3C-like+protease+inhibitors+using+self-assembled+monolayer+desorption+ionization+mass+spectrometry&doi=10.1016%2Fj.antiviral.2020.104924"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Evaluation of SARS-CoV-2 3C-like protease inhibitors using self-assembled monolayer desorption ionization mass spectrometry</span></div><div class="casAuthors">Gurard-Levin, Zachary A.; Liu, Cheng; Jekle, Andreas; Jaisinghani, Ruchika; Ren, Suping; Vandyck, Koen; Jochmans, Dirk; Leyssen, Pieter; Neyts, Johan; Blatt, Lawrence M.; Beigelman, Leonid; Symons, Julian A.; Raboisson, Pierre; Scholle, Michael D.; Deval, Jerome</div><div class="citationInfo"><span class="NLM_cas:title">Antiviral Research</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">182</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">104924</span>CODEN:
                <span class="NLM_cas:coden">ARSRDR</span>;
        ISSN:<span class="NLM_cas:issn">0166-3542</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is the cause of the COVID-19 pandemic that began in 2019.  The coronavirus 3-chymotrypsin-like cysteine protease (3CLpro) controls replication and is therefore considered a major target for antiviral discovery.  This study describes the evaluation of SARS-CoV-2 3CLpro inhibitors in a novel self-assembled monolayer desorption ionization mass spectrometry (SAMDI-MS) enzymic assay.  Compared with a traditional FRET readout, the label-free SAMDI-MS assay offers greater sensitivity and eliminates false pos. inhibition from compd. interference with the optical signal.  The SAMDI-MS assay was optimized and validated with known inhibitors of coronavirus 3CLpro such as GC376 (IC50 = 0.060μM), calpain inhibitors II and XII (IC50 ∼20-25μM).  The FDA-approved drugs shikonin, disulfiram, and ebselen did not inhibit SARS-CoV-2 3CLpro activity in the SAMDI-MS assay under physiol. relevant reducing conditions.  The three drugs did not directly inhibit human β-coronavirus OC-43 or SARS-CoV-2 in vitro, but instead induced cell death.  In conclusion, the SAMDI-MS 3CLpro assay, combined with antiviral and cytotoxic assessment, provides a robust platform to evaluate antiviral agents directed against SARS-CoV-2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-LzIy-MQKmbVg90H21EOLACvtfcHk0lgYMs7o5anU7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhvVSksrvP&md5=94693043624b8fdd562165834d746267</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.antiviral.2020.104924&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.antiviral.2020.104924%26sid%3Dliteratum%253Aachs%26aulast%3DGurard-Levin%26aufirst%3DZ.%2BA.%26aulast%3DLiu%26aufirst%3DC.%26aulast%3DJekle%26aufirst%3DA.%26aulast%3DJaisinghani%26aufirst%3DR.%26aulast%3DRen%26aufirst%3DS.%26aulast%3DVandyck%26aufirst%3DK.%26aulast%3DJochmans%26aufirst%3DD.%26aulast%3DLeyssen%26aufirst%3DP.%26aulast%3DNeyts%26aufirst%3DJ.%26aulast%3DBlatt%26aufirst%3DL.%2BM.%26aulast%3DBeigelman%26aufirst%3DL.%26aulast%3DSymons%26aufirst%3DJ.%2BA.%26aulast%3DRaboisson%26aufirst%3DP.%26aulast%3DScholle%26aufirst%3DM.%2BD.%26aulast%3DDeval%26aufirst%3DJ.%26atitle%3DEvaluation%2520of%2520SARS-CoV-2%25203C-like%2520protease%2520inhibitors%2520using%2520self-assembled%2520monolayer%2520desorption%2520ionization%2520mass%2520spectrometry%26jtitle%3DAntiviral%2520Res.%26date%3D2020%26volume%3D182%26spage%3D104924%26doi%3D10.1016%2Fj.antiviral.2020.104924" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Musharrafieh, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diesing, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marty, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span> <span> </span><span class="NLM_article-title">Validating enterovirus D68-2A(pro) as an antiviral drug target and the discovery of telaprevir as a potent D68–2A(pro) inhibitor</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>93</i></span>,  <span class="NLM_fpage">e02221</span>– <span class="NLM_lpage">18</span>, <span class="refDoi"> DOI: 10.1128/JVI.02221-18</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=10.1128%2FJVI.02221-18" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=30674624" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=1%3ACAS%3A528%3ADC%252BC1MXpvFSkur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=2019&pages=e02221-18&author=R.+Musharrafiehauthor=C.+Maauthor=J.+Zhangauthor=Y.+Huauthor=J.+M.+Diesingauthor=M.+T.+Martyauthor=J.+Wang&title=Validating+enterovirus+D68-2A%28pro%29+as+an+antiviral+drug+target+and+the+discovery+of+telaprevir+as+a+potent+D68%E2%80%932A%28pro%29+inhibitor&doi=10.1128%2FJVI.02221-18"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Validating enterovirus D68-2Apro as an antiviral drug target and the discovery of telaprevir as a potent D68-2Apro inhibitor</span></div><div class="casAuthors">Musharrafieh, Rami; Ma, Chunlong; Zhang, Jiantao; Hu, Yanmei; Diesing, Jessica M.; Marty, Michael T.; Wang, Jun</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">e02221-18/1-e02221-18/16</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">1098-5514</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Enterovirus D68 (EV-D68) is a viral pathogen that leads to severe respiratory illness and has been linked with the development of acute flaccid myelitis (AFM) in children.  No vaccines or antivirals are currently available for EV-D68 infection, and treatment options for hospitalized patients are limited to supportive care.  Here, we report the expression of the EV-D68 2A protease (2Apro) and characterization of its enzymic activity.  Furthermore, we discovered that telaprevir, an FDA-approved drug used for the treatment of hepatitis C virus (HCV) infections, is a potent antiviral against EV-D68 by targeting the 2Apro enzyme.  Using a fluorescence resonance energy transfer-based substrate cleavage assay, we showed that the purified EV-D68 2Apro has proteolytic activity selective against a peptide sequence corresponding to the viral VP1-2A polyprotein junction.  Telaprevir inhibits EV-D68 2Apro through a nearly irreversible, biphasic binding mechanism.  In cell culture, telaprevir showed submicromolar-to-low-micromolar potency against several recently circulating neurotropic strains of EV-D68 in different human cell lines.  To further confirm the antiviral drug target, serial viral passage expts. were performed to select for resistance against telaprevir.  An N84T mutation near the active site of 2Apro was identified in resistant viruses, and this mutation reduced the potency of telaprevir in both the enzymic and cellular antiviral assays.  Collectively, we report for the first time the in vitro enzymic activity of EV-D68 2Apro and the identification of telaprevir as a potent EV-D68 2Apro inhibitor.  These findings implicate EV-D68 2Apro as an antiviral drug target and highlight the repurposing potential of telaprevir to treat EV-D68 infection.  IMPORTANCE A 2014 EV-D68 outbreak in the United States has been linked to the development of acute flaccid myelitis in children.  Unfortunately, no treatment options against EV-D68 are currently available, and the development of effective therapeutics is urgently needed.  Here, we characterize and validate a new EV-D68 drug target, the 2Apro, and identify telaprevir-an FDA-approved drug used to treat hepatitis C virus (HCV) infections-as a potent antiviral with a novel mechanism of action toward 2Apro. 2Apro functions as a viral protease that cleaves a peptide sequence corresponding to the VP1-2A polyprotein junction.  The binding of telaprevir potently inhibits its enzymic activity, and using drug resistance selection, we show that the potent antiviral activity of telaprevir was due to 2Apro inhibition.  This is the first inhibitor to selectively target the 2Apro from EV-D68 and can be used as a starting point for the development of therapeutics with selective activity against EV-D68.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdA_RBOmKB6bVg90H21EOLACvtfcHk0lgYMs7o5anU7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXpvFSkur8%253D&md5=a016c137e0637bcbdc533d21ff69aa2d</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1128%2FJVI.02221-18&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.02221-18%26sid%3Dliteratum%253Aachs%26aulast%3DMusharrafieh%26aufirst%3DR.%26aulast%3DMa%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DDiesing%26aufirst%3DJ.%2BM.%26aulast%3DMarty%26aufirst%3DM.%2BT.%26aulast%3DWang%26aufirst%3DJ.%26atitle%3DValidating%2520enterovirus%2520D68-2A%2528pro%2529%2520as%2520an%2520antiviral%2520drug%2520target%2520and%2520the%2520discovery%2520of%2520telaprevir%2520as%2520a%2520potent%2520D68%25E2%2580%25932A%2528pro%2529%2520inhibitor%26jtitle%3DJ.%2520Virol.%26date%3D2019%26volume%3D93%26spage%3De02221%26epage%3D18%26doi%3D10.1128%2FJVI.02221-18" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cady, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeGrado, W. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, M.</span></span> <span> </span><span class="NLM_article-title">Specific binding of adamantane drugs and direction of their polar amines in the pore of the influenza M2 transmembrane domain in lipid bilayers and dodecylphosphocholine micelles determined by NMR spectroscopy</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>133</i></span>,  <span class="NLM_fpage">4274</span>– <span class="NLM_lpage">4284</span>, <span class="refDoi"> DOI: 10.1021/ja102581n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja102581n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=1%3ACAS%3A528%3ADC%252BC3MXisl2qtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2011&pages=4274-4284&author=S.+D.+Cadyauthor=J.+Wangauthor=Y.+Wuauthor=W.+F.+DeGradoauthor=M.+Hong&title=Specific+binding+of+adamantane+drugs+and+direction+of+their+polar+amines+in+the+pore+of+the+influenza+M2+transmembrane+domain+in+lipid+bilayers+and+dodecylphosphocholine+micelles+determined+by+NMR+spectroscopy&doi=10.1021%2Fja102581n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Specific Binding of Adamantane Drugs and Direction of Their Polar Amines in the Pore of the Influenza M2 Transmembrane Domain in Lipid Bilayers and Dodecylphosphocholine Micelles Determined by NMR Spectroscopy</span></div><div class="casAuthors">Cady, Sarah D.; Wang, Jun; Wu, Yibing; DeGrado, William F.; Hong, Mei</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">133</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4274-4284</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The transmembrane domain of the influenza M2 protein (M2TM) forms a tetrameric proton channel important for the virus life-cycle.  The proton-channel activity is inhibited by amine-contg. adamantyl drugs amantadine and rimantadine, which have been shown to bind specifically to the pore of M2TM near Ser31.  However, whether the polar amine points to the N- or C-terminus of the channel has not yet been detd.  Elucidating the polar group direction will shed light on the mechanism by which drug binding inhibits this proton channel and will facilitate rational design of new inhibitors.  In this study, we det. the polar amine direction using M2TM reconstituted in lipid bilayers as well as dodecylphosphocholine (DPC) micelles. 13C-2H rotational-echo double-resonance NMR expts. of 13C-labeled M2TM and methyl-deuterated rimantadine in lipid bilayers showed that the polar amine pointed to the C-terminus of the channel, with the Me group close to Gly34.  Soln. NMR expts. of M2TM in DPC micelles indicate that drug binding causes significant chem. shift perturbations of the protein that are very similar to those seen for M2TM and M2(18-60) bound to lipid bilayers.  Specific 2H-labeling of the drugs permitted the assignment of drug-protein cross peaks, which indicate that amantadine and rimantadine bind to the pore in the same fashion as for bilayer-bound M2TM.  These results strongly suggest that adamantyl inhibition of M2TM is achieved not only by direct phys. occlusion of the channel, but also by perturbing the equil. const. of the proton-sensing residue His37.  The reprodn. of the pharmacol. relevant specific pore-binding site in DPC micelles, which was not obsd. with a different detergent, DHPC, underscores the significant influence of the detergent environment on the functional structure of this membrane protein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHrFx8uRujFLVg90H21EOLACvtfcHk0lgYMs7o5anU7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXisl2qtbs%253D&md5=d2e9d577835a89045092bc2c0b87a24b</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fja102581n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja102581n%26sid%3Dliteratum%253Aachs%26aulast%3DCady%26aufirst%3DS.%2BD.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DDeGrado%26aufirst%3DW.%2BF.%26aulast%3DHong%26aufirst%3DM.%26atitle%3DSpecific%2520binding%2520of%2520adamantane%2520drugs%2520and%2520direction%2520of%2520their%2520polar%2520amines%2520in%2520the%2520pore%2520of%2520the%2520influenza%2520M2%2520transmembrane%2520domain%2520in%2520lipid%2520bilayers%2520and%2520dodecylphosphocholine%2520micelles%2520determined%2520by%2520NMR%2520spectroscopy%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2011%26volume%3D133%26spage%3D4274%26epage%3D4284%26doi%3D10.1021%2Fja102581n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marty, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baldwin, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marklund, E. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hochberg, G. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benesch, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, C. V.</span></span> <span> </span><span class="NLM_article-title">Bayesian deconvolution of mass and ion mobility spectra: from binary interactions to polydisperse ensembles</span>. <i>Anal. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>87</i></span>,  <span class="NLM_fpage">4370</span>– <span class="NLM_lpage">4376</span>, <span class="refDoi"> DOI: 10.1021/acs.analchem.5b00140</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.analchem.5b00140" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=1%3ACAS%3A528%3ADC%252BC2MXkvF2jtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=87&publication_year=2015&pages=4370-4376&author=M.+T.+Martyauthor=A.+J.+Baldwinauthor=E.+G.+Marklundauthor=G.+K.+Hochbergauthor=J.+L.+Beneschauthor=C.+V.+Robinson&title=Bayesian+deconvolution+of+mass+and+ion+mobility+spectra%3A+from+binary+interactions+to+polydisperse+ensembles&doi=10.1021%2Facs.analchem.5b00140"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Bayesian Deconvolution of Mass and Ion Mobility Spectra: From Binary Interactions to Polydisperse Ensembles</span></div><div class="casAuthors">Marty, Michael T.; Baldwin, Andrew J.; Marklund, Erik G.; Hochberg, Georg K. A.; Benesch, Justin L. P.; Robinson, Carol V.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Chemistry (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">87</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">4370-4376</span>CODEN:
                <span class="NLM_cas:coden">ANCHAM</span>;
        ISSN:<span class="NLM_cas:issn">0003-2700</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Interpretation of mass spectra is challenging because they report a ratio of 2 phys. quantities, mass and charge, which may each have multiple components that overlap in m/z.  Previous approaches to disentangling the 2 have focused on peak assignment or fitting.  However, the former struggle with complex spectra, and the latter are generally computationally intensive and may require substantial manual intervention.  The authors propose a new data anal. approach that employs a Bayesian framework to sep. the mass and charge dimensions.  From this approach, the authors developed UniDec (Universal Deconvolution), software that provides a rapid, robust, and flexible deconvolution of mass spectra and ion mobility-mass spectra with minimal user intervention.  Incorporation of the charge-state distribution in the Bayesian prior probabilities provides sepn. of the m/z spectrum into its phys. mass and charge components.  The authors have evaluated approach using systems of increasing complexity, enabling one to deduce lipid binding to membrane proteins, to probe the dynamics of subunit exchange reactions, and to characterize polydispersity in both protein assemblies and lipoprotein Nanodiscs.  The general utility of approach will greatly facilitate anal. of ion mobility and mass spectra.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEW75ktwO86rVg90H21EOLACvtfcHk0lguRoi2phwryw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXkvF2jtLc%253D&md5=d2d1272237752ba52cdf2a976608f6bf</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Facs.analchem.5b00140&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.analchem.5b00140%26sid%3Dliteratum%253Aachs%26aulast%3DMarty%26aufirst%3DM.%2BT.%26aulast%3DBaldwin%26aufirst%3DA.%2BJ.%26aulast%3DMarklund%26aufirst%3DE.%2BG.%26aulast%3DHochberg%26aufirst%3DG.%2BK.%26aulast%3DBenesch%26aufirst%3DJ.%2BL.%26aulast%3DRobinson%26aufirst%3DC.%2BV.%26atitle%3DBayesian%2520deconvolution%2520of%2520mass%2520and%2520ion%2520mobility%2520spectra%253A%2520from%2520binary%2520interactions%2520to%2520polydisperse%2520ensembles%26jtitle%3DAnal.%2520Chem.%26date%3D2015%26volume%3D87%26spage%3D4370%26epage%3D4376%26doi%3D10.1021%2Facs.analchem.5b00140" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span> <span> </span><span class="NLM_article-title">The in vitro antiviral activity of lactoferrin against common human coronaviruses and SARS-CoV-2 is mediated by targeting the heparan sulfate co-receptor</span>. <i>Emerging Microbes Infect.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">317</span>– <span class="NLM_lpage">330</span>, <span class="refDoi"> DOI: 10.1080/22221751.2021.1888660</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=10.1080%2F22221751.2021.1888660" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=33560940" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=1%3ACAS%3A528%3ADC%252BB3MXltlegtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2021&pages=317-330&author=Y.+Huauthor=X.+Mengauthor=F.+Zhangauthor=Y.+Xiangauthor=J.+Wang&title=The+in+vitro+antiviral+activity+of+lactoferrin+against+common+human+coronaviruses+and+SARS-CoV-2+is+mediated+by+targeting+the+heparan+sulfate+co-receptor&doi=10.1080%2F22221751.2021.1888660"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">The in vitro antiviral activity of lactoferrin against common human coronaviruses and SARS-CoV-2 is mediated by targeting the heparan sulfate co-receptor</span></div><div class="casAuthors">Hu, Yanmei; Meng, Xiangzhi; Zhang, Fushun; Xiang, Yan; Wang, Jun</div><div class="citationInfo"><span class="NLM_cas:title">Emerging Microbes & Infections</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">317-330</span>CODEN:
                <span class="NLM_cas:coden">EMIMC4</span>;
        ISSN:<span class="NLM_cas:issn">2222-1751</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">Coronavirus disease 2019 (COVID-19) is an ongoing pandemic that lacks effective therapeutic interventions.  SARS-CoV-2 infects ACE2-expressing cells and gains cell entry through either direct plasma membrane fusion or endocytosis.  Recent studies have shown that in addn. to ACE2, heparan sulfate proteoglycans (HSPGs) also play an important role in SARS-CoV-2 cell attachment by serving as an attachment factor.  Binding of viral spike protein to HSPGs leads to the enrichment of local concn. for the subsequent specific binding with ACE2.  We therefore hypothesize that blocking the interactions between viral spike protein and the HSPGs will lead to inhibition of viral replication.  In this study, we report our findings of the broad-spectrum antiviral activity and the mechanism of action of lactoferrin (LF) against multiple common human coronaviruses as well as SARS-CoV-2.  Our study has shown that LF has broad-spectrum antiviral activity against SARS-CoV-2, HCoV-OC43, HCoV-NL63, and HCoV-229E in cell culture, and bovine lactoferrin (BLF) is more potent than human lactoferrin.  Mechanistic studies revealed that BLF binds to HSPGs, thereby blocking viral attachment to the host cell.  The antiviral activity of BLF can be antagonized by the HSPG mimetic heparin.  Combination therapy expt. showed that the antiviral activity of LF is synergistic with remdesivir in cell culture.  Mol. modeling suggests that the N-terminal pos. charged region in BLF (residues 17-41) confers the binding to HSPGs.  Overall, LF appears to be a promising drug candidate for COVID-19 that warrants further investigation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-KJMYvpJE-LVg90H21EOLACvtfcHk0lguRoi2phwryw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXltlegtrg%253D&md5=a5c5c402b46462fe9a0cc836f5b90d9d</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1080%2F22221751.2021.1888660&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F22221751.2021.1888660%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DY.%26aulast%3DMeng%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DF.%26aulast%3DXiang%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DJ.%26atitle%3DThe%2520in%2520vitro%2520antiviral%2520activity%2520of%2520lactoferrin%2520against%2520common%2520human%2520coronaviruses%2520and%2520SARS-CoV-2%2520is%2520mediated%2520by%2520targeting%2520the%2520heparan%2520sulfate%2520co-receptor%26jtitle%3DEmerging%2520Microbes%2520Infect.%26date%3D2021%26volume%3D10%26spage%3D317%26epage%3D330%26doi%3D10.1080%2F22221751.2021.1888660" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Repetto, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">del Peso, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zurita, J. L.</span></span> <span> </span><span class="NLM_article-title">Neutral red uptake assay for the estimation of cell viability/cytotoxicity</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">1125</span>– <span class="NLM_lpage">1131</span>, <span class="refDoi"> DOI: 10.1038/nprot.2008.75</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=10.1038%2Fnprot.2008.75" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=18600217" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=1%3ACAS%3A528%3ADC%252BD1cXotFakurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2008&pages=1125-1131&author=G.+Repettoauthor=A.+del+Pesoauthor=J.+L.+Zurita&title=Neutral+red+uptake+assay+for+the+estimation+of+cell+viability%2Fcytotoxicity&doi=10.1038%2Fnprot.2008.75"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Neutral red uptake assay for the estimation of cell viability/cytotoxicity</span></div><div class="casAuthors">Repetto, Guillermo; del Peso, Ana; Zurita, Jorge L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1125-1131</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The neutral red uptake assay provides a quant. estn. of the no. of viable cells in a culture.  It is one of the most used cytotoxicity tests with many biomedical and environmental applications.  It is based on the ability of viable cells to incorporate and bind the supravital dye neutral red in the lysosomes.  Most primary cells and cell lines from diverse origin may be successfully used.  Cells are seeded in 96-well tissue culture plates and are treated for the appropriate period.  The plates are then incubated for 2 h with a medium contg. neutral red.  The cells are subsequently washed, the dye is extd. in each well and the absorbance is read using a spectrophotometer.  The procedure is cheaper and more sensitive than other cytotoxicity tests (tetrazolium salts, enzyme leakage or protein content).  Once the cells were treated, the assay can be completed in <3 h.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqCcpd7QjH04rVg90H21EOLACvtfcHk0lguRoi2phwryw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXotFakurk%253D&md5=30461f51c2ad668a3cadb4843c93b43e</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2008.75&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2008.75%26sid%3Dliteratum%253Aachs%26aulast%3DRepetto%26aufirst%3DG.%26aulast%3Ddel%2BPeso%26aufirst%3DA.%26aulast%3DZurita%26aufirst%3DJ.%2BL.%26atitle%3DNeutral%2520red%2520uptake%2520assay%2520for%2520the%2520estimation%2520of%2520cell%2520viability%252Fcytotoxicity%26jtitle%3DNat.%2520Protoc.%26date%3D2008%26volume%3D3%26spage%3D1125%26epage%3D1131%26doi%3D10.1038%2Fnprot.2008.75" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ma, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span> <span> </span><span class="NLM_article-title">Pharmacological characterization of the spectrum of antiviral activity and genetic barrier to drug resistance of M2-S31N channel blockers</span>. <i>Mol. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>90</i></span>,  <span class="NLM_fpage">188</span>– <span class="NLM_lpage">98</span>, <span class="refDoi"> DOI: 10.1124/mol.116.105346</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=10.1124%2Fmol.116.105346" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=27385729" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVektr%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=90&publication_year=2016&pages=188-98&author=C.+Maauthor=J.+Zhangauthor=J.+Wang&title=Pharmacological+characterization+of+the+spectrum+of+antiviral+activity+and+genetic+barrier+to+drug+resistance+of+M2-S31N+channel+blockers&doi=10.1124%2Fmol.116.105346"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacological characterization of the spectrum of antiviral activity and genetic barrier to drug resistance of M2-S31N channel blockers</span></div><div class="casAuthors">Ma, Chunlong; Zhang, Jiantao; Wang, Jun</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">90</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">188-198</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">1521-0111</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Adamantanes (amantadine and rimantadine) are one of the two classes of Food and Drug Administration-approved antiviral drugs used for the prevention and treatment of influenza A virus infections.  They inhibit viral replication by blocking the wild-type (WT) M2 proton channel, thus preventing viral uncoating.  However, their use was discontinued due to widespread drug resistance.  Among a handful of drug-resistant mutants, M2-S31N is the predominant mutation and persists in more than 95% of currently circulating influenza A strains.  We recently designed two classes of M2-S31N inhibitors, S31N-specific inhibitors and S31N/WT dual inhibitors, which are represented by N-[(5- cyclopropyl-1,2-oxazol-3-yl)methyl]adamantan-1-amine (WJ379) and N-[(5-bromothiophen-2-yl)methyl]adamantan-1-amine (BC035), resp.  However, their antiviral activities against currently circulating influenza A viruses and their genetic barrier to drug resistance are unknown.  In this report, we evaluated the therapeutic potential of these two classes of M2-S31N inhibitors (WJ379 and BC035) by profiling their antiviral efficacy against multidrug-resistant influenza A viruses, in vitro drug resistance barrier, and synergistic effect with oseltamivir.  We found that M2- S31N inhibitors were active against several influenza A viruses that are resistant to one or both classes of Food and Drug Administration-approved anti-influenza drugs.  In addn., M2- S31N inhibitors display a higher in vitro genetic barrier to drug resistance than amantadine.  The antiviral effect of WJ379 was also synergistic with oseltamivir carboxylate.  Overall, these results reaffirm that M2-S31N inhibitors are promising antiviral drug candidates that warrant further development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4eDLHN9ABdLVg90H21EOLACvtfcHk0liXjqatYQ8c6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVektr%252FF&md5=bef9d1512339524ce625cda89aa6feb7</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1124%2Fmol.116.105346&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.116.105346%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DJ.%26atitle%3DPharmacological%2520characterization%2520of%2520the%2520spectrum%2520of%2520antiviral%2520activity%2520and%2520genetic%2520barrier%2520to%2520drug%2520resistance%2520of%2520M2-S31N%2520channel%2520blockers%26jtitle%3DMol.%2520Pharmacol.%26date%3D2016%26volume%3D90%26spage%3D188%26epage%3D98%26doi%3D10.1124%2Fmol.116.105346" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teplyakov, A.</span></span> <span> </span><span class="NLM_article-title">MOLREP: an automated program for molecular replacement</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">1022</span>– <span class="NLM_lpage">1025</span>, <span class="refDoi"> DOI: 10.1107/S0021889897006766</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=10.1107%2FS0021889897006766" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=1%3ACAS%3A528%3ADyaK1cXhslehu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=1997&pages=1022-1025&author=A.+Vaginauthor=A.+Teplyakov&title=MOLREP%3A+an+automated+program+for+molecular+replacement&doi=10.1107%2FS0021889897006766"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">MOLREP: an automated program for molecular replacement</span></div><div class="casAuthors">Vagin, Alexei; Teplyakov, Alexei</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1022-1025</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard International Publishers Ltd.</span>)
        </div><div class="casAbstract">MOLREP is an automated program for mol. replacement which uses effective new approaches in data processing and rotational and translational searching.  These include an automatic choice of all parameters, scaling by Patterson origin peaks and soft resoln. cutoff.  One of the cornerstones of the program is an original full-symmetry translation function combined with a packing function.  Information from the model already placed in the cell is incorporated in both translation and packing functions.  A no. of tests using exptl. data proved the ability of the program to find the correct soln. in difficult cases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkGXdZ-nTKxLVg90H21EOLACvtfcHk0liXjqatYQ8c6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXhslehu7Y%253D&md5=96af64aedd8a8f018c3c18b57a6b4b21</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1107%2FS0021889897006766&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889897006766%26sid%3Dliteratum%253Aachs%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DTeplyakov%26aufirst%3DA.%26atitle%3DMOLREP%253A%2520an%2520automated%2520program%2520for%2520molecular%2520replacement%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D1997%26volume%3D30%26spage%3D1022%26epage%3D1025%26doi%3D10.1107%2FS0021889897006766" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skubak, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lebedev, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannu, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steiner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicholls, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A. A.</span></span> <span> </span><span class="NLM_article-title">REFMAC5 for the refinement of macromolecular crystal structures</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">355</span>– <span class="NLM_lpage">367</span>, <span class="refDoi"> DOI: 10.1107/S0907444911001314</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=10.1107%2FS0907444911001314" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=21460454" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=355-367&author=G.+N.+Murshudovauthor=P.+Skubakauthor=A.+A.+Lebedevauthor=N.+S.+Pannuauthor=R.+A.+Steinerauthor=R.+A.+Nichollsauthor=M.+D.+Winnauthor=F.+Longauthor=A.+A.+Vagin&title=REFMAC5+for+the+refinement+of+macromolecular+crystal+structures&doi=10.1107%2FS0907444911001314"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">REFMAC5 for the refinement of macromolecular crystal structures</span></div><div class="casAuthors">Murshudov, Garib N.; Skubak, Pavol; Lebedev, Andrey A.; Pannu, Navraj S.; Steiner, Roberto A.; Nicholls, Robert A.; Winn, Martyn D.; Long, Fei; Vagin, Alexei A.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">355-367</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">This paper describes various components of the macromol. crystallog. refinement program REFMAC5, which is distributed as part of the CCP4 suite.  REFMAC5 utilizes different likelihood functions depending on the diffraction data employed (amplitudes or intensities), the presence of twinning and the availability of SAD/SIRAS exptl. diffraction data.  To ensure chem. and structural integrity of the refined model, REFMAC5 offers several classes of restraints and choices of model parameterization.  Reliable models at resolns. at least as low as 4 Å can be achieved thanks to low-resoln. refinement tools such as secondary-structure restraints, restraints to known homologous structures, automatic global and local NCS restraints, 'jelly-body' restraints and the use of novel long-range restraints on at. displacement parameters (ADPs) based on the Kullback-Leibler divergence.  REFMAC5 addnl. offers TLS parameterization and, when high-resoln. data are available, fast refinement of anisotropic ADPs.  Refinement in the presence of twinning is performed in a fully automated fashion.  REFMAC5 is a flexible and highly optimized refinement package that is ideally suited for refinement across the entire resoln. spectrum encountered in macromol. crystallog.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5Jq-i2qIq5rVg90H21EOLACvtfcHk0liXjqatYQ8c6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqtbk%253D&md5=f8f3202d246908500057ad7c71015b7b</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1107%2FS0907444911001314&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444911001314%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DSkubak%26aufirst%3DP.%26aulast%3DLebedev%26aufirst%3DA.%2BA.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DSteiner%26aufirst%3DR.%2BA.%26aulast%3DNicholls%26aufirst%3DR.%2BA.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DLong%26aufirst%3DF.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26atitle%3DREFMAC5%2520for%2520the%2520refinement%2520of%2520macromolecular%2520crystal%2520structures%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D355%26epage%3D367%26doi%3D10.1107%2FS0907444911001314" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Coot: model-building tools for molecular graphics</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2126</span>– <span class="NLM_lpage">2132</span>, <span class="refDoi"> DOI: 10.1107/S0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+model-building+tools+for+molecular+graphics&doi=10.1107%2FS0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0liXjqatYQ8c6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2FS0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 1 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Zilei Xia, Michael Sacco, Yanmei Hu, Chunlong Ma, Xiangzhi Meng, Fushun Zhang, Tommy Szeto, Yan Xiang, Yu Chen, <span class="NLM_string-name hlFld-ContribAuthor">Jun Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Rational Design of Hybrid SARS-CoV-2 Main Protease Inhibitors Guided by the Superimposed Cocrystal Structures with the Peptidomimetic Inhibitors GC-376, Telaprevir, and Boceprevir. </span><span class="cited-content_cbyCitation_journal-name">ACS Pharmacology & Translational Science</span><span> <strong>2021</strong>, Article ASAP.</span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsptsci.1c00099&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsptsci.1c00099%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Pharmacology%2520%2526%2520Translational%2520Science%26atitle%3DRational%252BDesign%252Bof%252BHybrid%252BSARS-CoV-2%252BMain%252BProtease%252BInhibitors%252BGuided%252Bby%252Bthe%252BSuperimposed%252BCocrystal%252BStructures%252Bwith%252Bthe%252BPeptidomimetic%252BInhibitors%252BGC-376%25252C%252BTelaprevir%25252C%252Band%252BBoceprevir%26aulast%3DXia%26aufirst%3DZilei%26date%3D2021%26date%3D2021%26date%3D06042021%26volume%3D0%26issue%3D0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00509/20210512/images/medium/jm1c00509_0015.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00509/20210512/images/large/jm1c00509_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00509&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00509/20210512/images/medium/jm1c00509_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00509/20210512/images/large/jm1c00509_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Promising SARS-CoV-2 M<sup>pro</sup> inhibitors reported in the literature with translational potential.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00509/20210512/images/large/jm1c00509_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00509&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00509/20210512/images/medium/jm1c00509_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00509/20210512/images/large/jm1c00509_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Structure of M<sup>pro</sup> with inhibitors. (A) X-ray crystal structure of SARS-CoV-2 M<sup>pro</sup> in complex with <b>calpain inhibitor XII</b> (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6XFN">6XFN</a>). (B) X-ray crystal structure of SARS-Co-V M<sup>pro</sup> in complex with <b>ML188 (R)</b> (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3V3M">3V3M</a>). Hydrogen bonds are shown in dashed lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00509/20210512/images/large/jm1c00509_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00509&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00509/20210512/images/medium/jm1c00509_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00509/20210512/images/large/jm1c00509_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Design rationale for the noncovalent SARS-CoV-2 M<sup>pro</sup> inhibitors. (A) Superimposed X-ray crystal structures of SARS-CoV M<sup>pro</sup> + ML188 (R) (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3V3M">3V3M</a>, green) and SARS-CoV M<sup>pro</sup> H41A mutant + peptide substrate (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2Q6G">2Q6G</a>, yellow with backbone shown as ribbon representation). (B) Superimposed X-ray crystal structures of SARS-CoV M<sup>pro</sup> + <b>ML188 (R)</b> (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3V3M">3V3M</a>) and SARS-CoV-2 M<sup>pro</sup> + <b>GC376</b> (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6WTT">6WTT</a>). (C) Superimposed X-ray crystal structures of SARS-CoV M<sup>pro</sup> + <b>ML188 (R)</b> (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3V3M">3V3M</a>) and SARS-CoV-2 M<sup>pro</sup> + <b>UAWJ247</b> (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6XBH">6XBH</a>). (D) Chemical structures of <b>ML188 (R)</b>, peptide substrate VLQS, <b>GC376</b>, and <b>UAWJ247</b>. (E) Stepwise optimization of <b>ML188 (R)</b> toward potent noncovalent SARS-CoV-2 M<sup>pro</sup> inhibitor.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00509/20210512/images/large/jm1c00509_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00509&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00509/20210512/images/medium/jm1c00509_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00509/20210512/images/large/jm1c00509_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Design and synthesis of a focused library of noncovalent SARS-CoV-2 M<sup>pro</sup> inhibitors. (A) X-ray crystal structure of SARS-CoV M<sup>pro</sup> + <b>ML188 (R)</b> (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3V3M">3V3M</a>). (B) Binding interactions of <b>ML188 (R)</b> with SARS-CoV M<sup>pro</sup>. (C) Synthesis of <b>ML188</b> analogs using the Ugi four-component reaction.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00509/20210512/images/large/jm1c00509_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00509&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00509/20210512/images/medium/jm1c00509_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00509/20210512/images/large/jm1c00509_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Structures of noncovalent and covalent SARS-CoV-2 M<sup>pro</sup> inhibitors and the enzymatic inhibition against SARS-CoV-2 M<sup>pro</sup>. (A) Noncovalent analogs with P3/P4 modifications. (B) Noncovalent analogs with P2 modifications. (C) Noncovalent analogs with P1′ modifications. (D) Noncovalent analogs with P1 modifications. (E) Noncovalent analogs with combined P1′, P1, P2, and P3/P4 modifications. Compounds with potent enzymatic inhibition (IC<sub>50</sub> < 5 μM) but moderate to high cellular cytotoxicity (CC<sub>50</sub> < 100 μM) are labeled in red. Compounds with both potent enzymatic inhibition (IC<sub>50</sub> < 5 μM) and low cellular cytotoxicity (CC<sub>50</sub> > 100 μM) are labeled in blue. (F) Percentage enzymatic activity of SARS-CoV-2 M<sup>pro</sup> in the presence of the designed compounds at 20 μM concentration.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00509/20210512/images/large/jm1c00509_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00509&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00509/20210512/images/medium/jm1c00509_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00509/20210512/images/large/jm1c00509_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Enzymatic inhibition, cellular antiviral activity, and selectivity index of noncovalent M<sup>pro</sup> inhibitors. (A) Antiviral activity and selectivity index of noncovalent SARS-CoV-2 M<sup>pro</sup> inhibitors. (Selection criteria IC<sub>50</sub> < 1 μM, CC<sub>50</sub> > 100 μM). Antiviral assay was performed using the SARS-CoV-2 USA-WA1/2020 isolate in Vero E6 cells with an MOI of 0.05. (B) Chemical structures of the two diastereomers <b>23R</b> and <b>23S</b>. The absolute stereochemistry of compound <b>23R</b> was determined in the cocrystal structure of this diastereomer with SARS-CoV-2 M<sup>pro</sup> (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7KX5">7KX5</a>). (C and D) Antiviral activity of <b>23R</b> against SARS-CoV-2 in Calu-3 cells. (C) Raw data of the percentage of immunofluorescence positive cells with different concentrations of <b>23R</b>. (D) Antiviral potency and cytotoxicity plots. N.T. = not tested. N.A. = not applicable.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00509/20210512/images/large/jm1c00509_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00509&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00509/20210512/images/medium/jm1c00509_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00509/20210512/images/large/jm1c00509_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Selectivity of noncovalent and covalent SARS-CoV-2 M<sup>pro</sup> inhibitors shown in a color map. (A) Selectivity against viral cysteine proteases. (B) Selectivity against host proteases. The values plotted were the IC<sub>50</sub> values from the FRET-based enzymatic assay.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00509/20210512/images/large/jm1c00509_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00509&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00509/20210512/images/medium/jm1c00509_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00509/20210512/images/large/jm1c00509_0008.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Characterization of the mechanism of action of <b>23R</b> to SARS-CoV-2 M<sup>pro</sup> using native mass spectrometry, thermal shift assay, and enzyme kinetic studies. (A) Binding of <b>23R</b> to SARS-CoV-2 M<sup>pro</sup> using native mass spectrometry. Native mass spectra with the inset deconvolved spectra revealing ligand binding with 10 μM or 30 μM <b>GC376</b> added (middle panel) and 10 μM and 30 μM <b>23R</b> (bottom panel) with 4 mM DTT added. The peaks are annotated with the blue circle as the dimer, the green down triangle as the dimer with one ligand bound, and the purple up triangle as the dimer with two ligands bound. (B) Dose-dependent melting temperature (<i>T</i><sub>m</sub>) shift in the thermal shift assay. 3 μM SARS-CoV-2 M<sup>pro</sup> protein was incubated with various concentrations of ML188 or <b>23R</b> in the presence of 4 mM DTT. Measured <i>T</i><sub>m</sub> was plotted against compound concentration with one-site binding function in Prism 8. (C and D) Enzymatic kinetic assay with <b>ML188</b> and compound <b>23R</b>. Kinetic parameters in the presence of various concentrations of <b>ML188</b> or <b>23R</b> were globally fitted with a Michaelis–Menten function in prism 8 (top panels); double-reciprocal plots are shown in the right panels. The middle panels show the Morrison plots of compounds <b>ML188</b> and <b>23R</b> with 20 μM FRET substrate used.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00509/20210512/images/large/jm1c00509_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00509&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00509/20210512/images/medium/jm1c00509_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00509/20210512/images/large/jm1c00509_0009.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. X-ray crystal structure of SARS-CoV-2 M<sup>pro</sup> in complex with <b>23R</b>. (A) X-ray crystal structure of SARS-CoV-2 M<sup>pro</sup> in complex with <b>23R</b> (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7KX5">7KX5</a>). (B) Superimposed structures of SARS-CoV-2 M<sup>pro</sup> + <b>23R</b> (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7KX5">7KX5</a>) and SARS-CoV M<sup>pro</sup> + <b>ML188 (R)</b> (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3V3M">3V3M</a>). (C) Superimposed structures of SARS-CoV-2 M<sup>pro</sup> + <b>23R</b> (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7KX5">7KX5</a>) and SARS-CoV-2 M<sup>pro</sup> + <b>calpain inhibitor XII</b> (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6XFN">6XFN</a>). (D) Superimposed structures of SARS-CoV-2 M<sup>pro</sup> + <b>23R</b> (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7KX5">7KX5</a>) and SARS-CoV-2 M<sup>pro</sup> + <b>GC376</b> (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6WTT">6WTT</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00509/20210512/images/large/jm1c00509_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00509&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i76">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_42062" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_42062" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 34 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kalil, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patterson, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mehta, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomashek, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolfe, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghazaryan, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marconi, V. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruiz-Palacios, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsieh, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kline, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tapson, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iovine, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sweeney, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El Sahly, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Branche, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Regalado Pineda, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lye, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sandkovsky, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luetkemeyer, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finberg, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, P. E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taiwo, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paules, C. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arguinchona, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goepfert, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahuja, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frank, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oh, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mularski, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nielsen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ponce, P. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larson, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rouphael, N. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saklawi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantos, V. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ko, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engemann, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amin, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watanabe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Billings, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elie, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davey, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burgess, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferreira, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Makowski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cardoso, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Bono, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonnett, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Proschan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deye, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dempsey, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nayak, S. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodd, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beigel, J. H.</span></span>; <span class="NLM_contrib-group">Members, A.-S. G.</span> <span> </span><span class="NLM_article-title">Baricitinib plus remdesivir for hospitalized adults with Covid-19</span>. <i>N. Engl. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>384</i></span>,  <span class="NLM_fpage">795</span>– <span class="NLM_lpage">807</span>, <span class="refDoi"> DOI: 10.1056/NEJMoa2031994</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=10.1056%2FNEJMoa2031994" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=33306283" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=1%3ACAS%3A528%3ADC%252BB3MXlvFCksrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=384&publication_year=2021&pages=795-807&author=A.+C.+Kalilauthor=T.+F.+Pattersonauthor=A.+K.+Mehtaauthor=K.+M.+Tomashekauthor=C.+R.+Wolfeauthor=V.+Ghazaryanauthor=V.+C.+Marconiauthor=G.+M.+Ruiz-Palaciosauthor=L.+Hsiehauthor=S.+Klineauthor=V.+Tapsonauthor=N.+M.+Iovineauthor=M.+K.+Jainauthor=D.+A.+Sweeneyauthor=H.+M.+El+Sahlyauthor=A.+R.+Brancheauthor=J.+Regalado+Pinedaauthor=D.+C.+Lyeauthor=U.+Sandkovskyauthor=A.+F.+Luetkemeyerauthor=S.+H.+Cohenauthor=R.+W.+Finbergauthor=P.+E.+H.+Jacksonauthor=B.+Taiwoauthor=C.+I.+Paulesauthor=H.+Arguinchonaauthor=P.+Goepfertauthor=N.+Ahujaauthor=M.+Frankauthor=M.+D.+Ohauthor=E.+S.+Kimauthor=S.+Y.+Tanauthor=R.+A.+Mularskiauthor=H.+Nielsenauthor=P.+O.+Ponceauthor=B.+S.+Taylorauthor=L.+Larsonauthor=N.+G.+Rouphaelauthor=Y.+Saklawiauthor=V.+D.+Cantosauthor=E.+R.+Koauthor=J.+J.+Engemannauthor=A.+N.+Aminauthor=M.+Watanabeauthor=J.+Billingsauthor=M.+C.+Elieauthor=R.+T.+Daveyauthor=T.+H.+Burgessauthor=J.+Ferreiraauthor=M.+Greenauthor=M.+Makowskiauthor=A.+Cardosoauthor=S.+de+Bonoauthor=T.+Bonnettauthor=M.+Proschanauthor=G.+A.+Deyeauthor=W.+Dempseyauthor=S.+U.+Nayakauthor=L.+E.+Doddauthor=J.+H.+Beigelauthor=Members%2C+A.-S.+G.&title=Baricitinib+plus+remdesivir+for+hospitalized+adults+with+Covid-19&doi=10.1056%2FNEJMoa2031994"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Baricitinib plus remdesivir for hospitalized adults with Covid-19</span></div><div class="casAuthors">Kalil, A. C.; Patterson, T. F.; Mehta, A. K.; Tomashek, K. M.; Wolfe, C. R.; Ghazaryan, V.; Marconi, V. C.; Palacios, G. M. Ruiz; Hsieh, L.; Kline, S.; Tapson, V.; Iovine, N. M.; Jain, M. K.; Sweeney, D. A.; El Sahly, H. M.; Branche, A. R.; Regalado Pineda, J.; Lye, D. C.; Sandkovsky, U.; Luetkemeyer, A. F.; Cohen, S. H.; Finberg, R. W.; Jackson, P. E. H.; Taiwo, B.; Paules, C. I.; Arguinchona, H.; Erdmann, N.; Ahuja, N.; Frank, M.; Oh, M.; Kim, E.-S.; Tan, S. Y.; Mularski, R. A.; Nielsen, H.; Ponce, P. O.; Taylor, B. S.; Larson, L. A.; Rouphael, N. G.; Saklawi, Y.; Cantos, V. D.; Ko, E. R.; Engemann, J. J.; Amin, A. N.; Watanabe, M.; Billings, J.; Elie, M.-C.; Davey, R. T.; Burgess, T. H.; Ferreira, J.; Green, M.; Makowski, M.; Cardoso, A.; de Bono, S.; Bonnett, T.; Proschan, M.; Deye, G. A.; Dempsey, W.; Nayak, S. U.; Dodd, L. E.; Beigel, J. H.; Kalil, Andre C.; Patterson, Thomas F.; Mehta, Aneesh K.; Tomashek, Kay M.; Wolfe, Cameron R.; Ghazaryan, Varduhi; Marconi, Vincent C.; Ruiz-Palacios, Guillermo M.; Hsieh, Lanny; Kline, Susan; Tapson, Victor; Iovine, Nicole M.; Jain, Mamta K.; Sweeney, Daniel A.; El Sahly, Hana M.; Branche, Angela R.; Pineda, Justino Regalado; Lye, David C.; Sandkovsky, Uriel; Luetkemeyer, Anne F.; Cohen, Stuart H.; Finberg, Robert W.; Jackson, Patrick E. H.; Taiwo, Babafemi; Paules, Catharine I.; Arguinchona, Henry; Erdmann, Nathaniel; Ahuja, Neera; Frank, Maria; Oh, Myoung-don; Kim, Eu-Suk; Tan, Seow Y.; Mularski, Richard A.; Nielsen, Henrik; Ponce, Philip O.; Taylor, Barbara S.; Larson, LuAnn; Rouphael, Nadine G.; Saklawi, Youssef; Cantos, Valeria D.; Ko, Emily R.; Engemann, John J.; Amin, Alpesh N.; Watanabe, Miki; Billings, Joanne; Elie, Marie-Carmelle; Davey, Richard T.; Burgess, Timothy H.; Ferreira, Jennifer; Green, Michelle; Makowski, Mat; Cardoso, Anabela; de Bono, Stephanie; Bonnett, Tyler; Proschan, Michael; Deye, Gregory A.; Dempsey, Walla; Nayak, Seema U.; Dodd, Lori E.; Beigel, John H.</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">384</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">795-807</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">1533-4406</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">Background Severe coronavirus disease 2019 (Covid-19) is assocd. with dysregulated inflammation.  The effects of combination treatment with baricitinib, a Janus kinase inhibitor, plus remdesivir are not known.  We conducted a double-blind, randomized, placebo-controlled trial evaluating baricitinib plus remdesivir in hospitalized adults with Covid-19.  All the patients received remdesivir (≤10 days) and either baricitinib (≤14 days) or placebo (control).  The primary outcome was the time to recovery.  The key secondary outcome was clin. status at day 15. results A total of 1033 patients underwent randomization (with 515 assigned to combination treatment and 518 to control).  Patients receiving baricitinib had a median time to recovery of 7 days (95% confidence interval [CI], 6 to 8), as compared with 8 days (95% CI, 7 to 9) with control (rate ratio for recovery, 1.16; 95% CI, 1.01 to 1.32; P = 0.03), and a 30% higher odds of improvement in clin. status at day 15 (odds ratio, 1.3; 95% CI, 1.0 to 1.6).  Patients receiving high-flow oxygen or noninvasive ventilation at enrollment had a time to recovery of 10 days with combination treatment and 18 days with control (rate ratio for recovery, 1.51; 95% CI, 1.10 to 2.08).  The 28-day mortality was 5.1% in the combination group and 7.8% in the control group (hazard ratio for death, 0.65; 95% CI, 0.39 to 1.09).  Serious adverse events were less frequent in the combination group than in the control group (16.0% vs. 21.0%; difference, -5.0 percentage points; 95% CI, -9.8 to -0.3; P = 0.03), as were new infections (5.9% vs. 11.2%; difference, -5.3 percentage points; 95% CI, -8.7 to -1.9; P = 0.003).  Baricitinib plus remdesivir was superior to remdesivir alone in reducing recovery time and accelerating improvement in clin. status among patients with Covid-19, notably among those receiving high-flow oxygen or noninvasive ventilation.  The combination was assocd. with fewer serious adverse events.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoszT1RNk0q0bVg90H21EOLACvtfcHk0lje73Qtt64RvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXlvFCksrw%253D&md5=0fe106348eabfd2e844f046340ee833e</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1056%2FNEJMoa2031994&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMoa2031994%26sid%3Dliteratum%253Aachs%26aulast%3DKalil%26aufirst%3DA.%2BC.%26aulast%3DPatterson%26aufirst%3DT.%2BF.%26aulast%3DMehta%26aufirst%3DA.%2BK.%26aulast%3DTomashek%26aufirst%3DK.%2BM.%26aulast%3DWolfe%26aufirst%3DC.%2BR.%26aulast%3DGhazaryan%26aufirst%3DV.%26aulast%3DMarconi%26aufirst%3DV.%2BC.%26aulast%3DRuiz-Palacios%26aufirst%3DG.%2BM.%26aulast%3DHsieh%26aufirst%3DL.%26aulast%3DKline%26aufirst%3DS.%26aulast%3DTapson%26aufirst%3DV.%26aulast%3DIovine%26aufirst%3DN.%2BM.%26aulast%3DJain%26aufirst%3DM.%2BK.%26aulast%3DSweeney%26aufirst%3DD.%2BA.%26aulast%3DEl%2BSahly%26aufirst%3DH.%2BM.%26aulast%3DBranche%26aufirst%3DA.%2BR.%26aulast%3DRegalado%2BPineda%26aufirst%3DJ.%26aulast%3DLye%26aufirst%3DD.%2BC.%26aulast%3DSandkovsky%26aufirst%3DU.%26aulast%3DLuetkemeyer%26aufirst%3DA.%2BF.%26aulast%3DCohen%26aufirst%3DS.%2BH.%26aulast%3DFinberg%26aufirst%3DR.%2BW.%26aulast%3DJackson%26aufirst%3DP.%2BE.%2BH.%26aulast%3DTaiwo%26aufirst%3DB.%26aulast%3DPaules%26aufirst%3DC.%2BI.%26aulast%3DArguinchona%26aufirst%3DH.%26aulast%3DGoepfert%26aufirst%3DP.%26aulast%3DAhuja%26aufirst%3DN.%26aulast%3DFrank%26aufirst%3DM.%26aulast%3DOh%26aufirst%3DM.%2BD.%26aulast%3DKim%26aufirst%3DE.%2BS.%26aulast%3DTan%26aufirst%3DS.%2BY.%26aulast%3DMularski%26aufirst%3DR.%2BA.%26aulast%3DNielsen%26aufirst%3DH.%26aulast%3DPonce%26aufirst%3DP.%2BO.%26aulast%3DTaylor%26aufirst%3DB.%2BS.%26aulast%3DLarson%26aufirst%3DL.%26aulast%3DRouphael%26aufirst%3DN.%2BG.%26aulast%3DSaklawi%26aufirst%3DY.%26aulast%3DCantos%26aufirst%3DV.%2BD.%26aulast%3DKo%26aufirst%3DE.%2BR.%26aulast%3DEngemann%26aufirst%3DJ.%2BJ.%26aulast%3DAmin%26aufirst%3DA.%2BN.%26aulast%3DWatanabe%26aufirst%3DM.%26aulast%3DBillings%26aufirst%3DJ.%26aulast%3DElie%26aufirst%3DM.%2BC.%26aulast%3DDavey%26aufirst%3DR.%2BT.%26aulast%3DBurgess%26aufirst%3DT.%2BH.%26aulast%3DFerreira%26aufirst%3DJ.%26aulast%3DGreen%26aufirst%3DM.%26aulast%3DMakowski%26aufirst%3DM.%26aulast%3DCardoso%26aufirst%3DA.%26aulast%3Dde%2BBono%26aufirst%3DS.%26aulast%3DBonnett%26aufirst%3DT.%26aulast%3DProschan%26aufirst%3DM.%26aulast%3DDeye%26aufirst%3DG.%2BA.%26aulast%3DDempsey%26aufirst%3DW.%26aulast%3DNayak%26aufirst%3DS.%2BU.%26aulast%3DDodd%26aufirst%3DL.%2BE.%26aulast%3DBeigel%26aufirst%3DJ.%2BH.%26aulast%3D%26atitle%3DBaricitinib%2520plus%2520remdesivir%2520for%2520hospitalized%2520adults%2520with%2520Covid-19%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2021%26volume%3D384%26spage%3D795%26epage%3D807%26doi%3D10.1056%2FNEJMoa2031994" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gil, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ginex, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maestro, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nozal, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrado-Gil, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuesta-Geijo, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urquiza, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramirez, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alonso, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campillo, N. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, A.</span></span> <span> </span><span class="NLM_article-title">COVID-19: drug targets and potential treatments</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">12359</span>– <span class="NLM_lpage">12386</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00606</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00606" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtFWmsL3I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=12359-12386&author=C.+Gilauthor=T.+Ginexauthor=I.+Maestroauthor=V.+Nozalauthor=L.+Barrado-Gilauthor=M.+A.+Cuesta-Geijoauthor=J.+Urquizaauthor=D.+Ramirezauthor=C.+Alonsoauthor=N.+E.+Campilloauthor=A.+Martinez&title=COVID-19%3A+drug+targets+and+potential+treatments&doi=10.1021%2Facs.jmedchem.0c00606"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">COVID-19: Drug Targets and Potential Treatments</span></div><div class="casAuthors">Gil, Carmen; Ginex, Tiziana; Maestro, Ines; Nozal, Vanesa; Barrado-Gil, Lucia; Cuesta-Geijo, Miguel Angel; Urquiza, Jesus; Ramirez, David; Alonso, Covadonga; Campillo, Nuria E.; Martinez, Ana</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">12359-12386</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Currently, the authors are immersed in a pandemic caused by the emerging severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), which severely threatens public health worldwide.  Until now, no drug or vaccine has been approved to treat the severe disease caused by this coronavirus, COVID-19.  The authors will focus on the main virus-based and host-based targets that can guide medicinal chem. efforts to discover new drugs for this devastating disease.  In principle, all CoVs enzymes and proteins involved in viral replication and the control of host cellular machineries are potentially druggable targets in the search for therapeutic options for SARS-CoV-2.  This perspective provides an overview of the main targets from a structural point of view, together with reported therapeutic compds. with activity against SARS-CoV-2 and/or other CoVs.  Also, the role of innate immune response to coronavirus infection and the related therapeutic options will be presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsg-YRktOknrVg90H21EOLACvtfcHk0lje73Qtt64RvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtFWmsL3I&md5=6a0dc17c5f601f09a92da933f81bbbea</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00606&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00606%26sid%3Dliteratum%253Aachs%26aulast%3DGil%26aufirst%3DC.%26aulast%3DGinex%26aufirst%3DT.%26aulast%3DMaestro%26aufirst%3DI.%26aulast%3DNozal%26aufirst%3DV.%26aulast%3DBarrado-Gil%26aufirst%3DL.%26aulast%3DCuesta-Geijo%26aufirst%3DM.%2BA.%26aulast%3DUrquiza%26aufirst%3DJ.%26aulast%3DRamirez%26aufirst%3DD.%26aulast%3DAlonso%26aufirst%3DC.%26aulast%3DCampillo%26aufirst%3DN.%2BE.%26aulast%3DMartinez%26aufirst%3DA.%26atitle%3DCOVID-19%253A%2520drug%2520targets%2520and%2520potential%2520treatments%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D12359%26epage%3D12386%26doi%3D10.1021%2Facs.jmedchem.0c00606" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ullrich, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nitsche, C.</span></span> <span> </span><span class="NLM_article-title">The SARS-CoV-2 main protease as drug target</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">127377</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2020.127377</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=10.1016%2Fj.bmcl.2020.127377" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=32738988" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtlaisrbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2020&pages=127377&author=S.+Ullrichauthor=C.+Nitsche&title=The+SARS-CoV-2+main+protease+as+drug+target&doi=10.1016%2Fj.bmcl.2020.127377"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The SARS-CoV-2 main protease as drug target</span></div><div class="casAuthors">Ullrich, Sven; Nitsche, Christoph</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">127377</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The unprecedented pandemic of the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is threatening global health.  The virus emerged in late 2019 and can cause a severe disease assocd. with significant mortality.  Several vaccine development and drug discovery campaigns are underway.  The SARS-CoV-2 main protease is considered a promising drug target, as it is dissimilar to human proteases.  Sequence and structure of the main protease are closely related to those from other betacoronaviruses, facilitating drug discovery attempts based on previous lead compds.  Covalently binding peptidomimetics and small mols. are investigated.  Various compds. show antiviral activity in infected human cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVeovBguTwDbVg90H21EOLACvtfcHk0lhvXnTvcct3iA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtlaisrbJ&md5=b2fd8ea99ddf6d3e553b137904d406a9</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2020.127377&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2020.127377%26sid%3Dliteratum%253Aachs%26aulast%3DUllrich%26aufirst%3DS.%26aulast%3DNitsche%26aufirst%3DC.%26atitle%3DThe%2520SARS-CoV-2%2520main%2520protease%2520as%2520drug%2520target%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2020%26volume%3D30%26spage%3D127377%26doi%3D10.1016%2Fj.bmcl.2020.127377" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rut, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groborz, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zmudzinski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pawlik, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jochmans, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neyts, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Młynarski, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilgenfeld, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drag, M.</span></span> <span> </span><span class="NLM_article-title">SARS-CoV-2 Mpro inhibitors and activity-based probes for patient-sample imaging</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">222</span>– <span class="NLM_lpage">228</span>, <span class="refDoi"> DOI: 10.1038/s41589-020-00689-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=10.1038%2Fs41589-020-00689-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=33093684" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=1%3ACAS%3A528%3ADC%252BB3cXitFKjsbbK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2021&pages=222-228&author=W.+Rutauthor=K.+Groborzauthor=L.+Zhangauthor=X.+Sunauthor=M.+Zmudzinskiauthor=B.+Pawlikauthor=X.+Wangauthor=D.+Jochmansauthor=J.+Neytsauthor=W.+M%C5%82ynarskiauthor=R.+Hilgenfeldauthor=M.+Drag&title=SARS-CoV-2+Mpro+inhibitors+and+activity-based+probes+for+patient-sample+imaging&doi=10.1038%2Fs41589-020-00689-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">SARS-CoV-2 Mpro inhibitors and activity-based probes for patient-sample imaging</span></div><div class="casAuthors">Rut, Wioletta; Groborz, Katarzyna; Zhang, Linlin; Sun, Xinyuanyuan; Zmudzinski, Mikolaj; Pawlik, Bartlomiej; Wang, Xinyu; Jochmans, Dirk; Neyts, Johan; Mlynarski, Wojciech; Hilgenfeld, Rolf; Drag, Marcin</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">222-228</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">In Dec. 2019, the first cases of infection with a novel coronavirus, SARS-CoV-2, were diagnosed.  Currently, there is no effective antiviral treatment for COVID-19.  To address this emerging problem, we focused on the SARS-CoV-2 main protease that constitutes one of the most attractive antiviral drug targets.  We have synthesized a combinatorial library of fluorogenic substrates with glutamine in the P1 position.  We used it to det. the substrate preferences of the SARS-CoV and SARS-CoV-2 main proteases.  On the basis of these findings, we designed and synthesized a potent SARS-CoV-2 inhibitor (Ac-Abu-DTyr-Leu-Gln-VS, half-maximal effective concn. of 3.7μM) and two activity-based probes, for one of which we detd. the crystal structure of its complex with the SARS-CoV-2 Mpro.  We visualized active SARS-CoV-2 Mpro in nasopharyngeal epithelial cells of patients suffering from COVID-19 infection.  The results of our work provide a structural framework for the design of inhibitors as antiviral agents and/or diagnostic tests.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowwaL39Q6HFrVg90H21EOLACvtfcHk0lhvXnTvcct3iA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXitFKjsbbK&md5=11ae0092711da91cf820234e8dcb6e27</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fs41589-020-00689-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-020-00689-z%26sid%3Dliteratum%253Aachs%26aulast%3DRut%26aufirst%3DW.%26aulast%3DGroborz%26aufirst%3DK.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DSun%26aufirst%3DX.%26aulast%3DZmudzinski%26aufirst%3DM.%26aulast%3DPawlik%26aufirst%3DB.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DJochmans%26aufirst%3DD.%26aulast%3DNeyts%26aufirst%3DJ.%26aulast%3DM%25C5%2582ynarski%26aufirst%3DW.%26aulast%3DHilgenfeld%26aufirst%3DR.%26aulast%3DDrag%26aufirst%3DM.%26atitle%3DSARS-CoV-2%2520Mpro%2520inhibitors%2520and%2520activity-based%2520probes%2520for%2520patient-sample%2520imaging%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2021%26volume%3D17%26spage%3D222%26epage%3D228%26doi%3D10.1038%2Fs41589-020-00689-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brindisi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shahabi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chapman, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mesecar, A. D.</span></span> <span> </span><span class="NLM_article-title">Drug development and medicinal chemistry efforts toward SARS-coronavirus and Covid-19 therapeutics</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">907</span>– <span class="NLM_lpage">932</span>, <span class="refDoi"> DOI: 10.1002/cmdc.202000223</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=10.1002%2Fcmdc.202000223" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=32324951" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=1%3ACAS%3A528%3ADC%252BB3cXos1GmsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2020&pages=907-932&author=A.+K.+Ghoshauthor=M.+Brindisiauthor=D.+Shahabiauthor=M.+E.+Chapmanauthor=A.+D.+Mesecar&title=Drug+development+and+medicinal+chemistry+efforts+toward+SARS-coronavirus+and+Covid-19+therapeutics&doi=10.1002%2Fcmdc.202000223"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Drug Development and Medicinal Chemistry Efforts toward SARS-Coronavirus and Covid-19 Therapeutics</span></div><div class="casAuthors">Ghosh, Arun K.; Brindisi, Margherita; Shahabi, Dana; Chapman, Mackenzie E.; Mesecar, Andrew D.</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">907-932</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  The COVID-19 pandemic caused by SARS-CoV-2 infection is spreading at an alarming rate and has created an unprecedented health emergency around the globe.  There is no effective vaccine or approved drug treatment against COVID-19 and other pathogenic coronaviruses.  The development of antiviral agents is an urgent priority.  Biochem. events crit. to the coronavirus replication cycle provided a no. of attractive targets for drug development.  These include, spike protein for binding to host cell-surface receptors, proteolytic enzymes that are essential for processing polyproteins into mature viruses, and RNA-dependent RNA polymerase for RNA replication.  There has been a lot of ground work for drug discovery and development against these targets.  Also, high-throughput screening efforts have led to the identification of diverse lead structures, including natural product-derived mols.  This review highlights past and present drug discovery and medicinal-chem. approaches against SARS-CoV, MERS-CoV and COVID-19 targets.  The review hopes to stimulate further research and will be a useful guide to the development of effective therapies against COVID-19 and other pathogenic coronaviruses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqcgVQF9N3zf7Vg90H21EOLACvtfcHk0lhvXnTvcct3iA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXos1GmsL4%253D&md5=8b4c3c83c05a7b64f39ba7d803ea1709</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.202000223&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.202000223%26sid%3Dliteratum%253Aachs%26aulast%3DGhosh%26aufirst%3DA.%2BK.%26aulast%3DBrindisi%26aufirst%3DM.%26aulast%3DShahabi%26aufirst%3DD.%26aulast%3DChapman%26aufirst%3DM.%2BE.%26aulast%3DMesecar%26aufirst%3DA.%2BD.%26atitle%3DDrug%2520development%2520and%2520medicinal%2520chemistry%2520efforts%2520toward%2520SARS-coronavirus%2520and%2520Covid-19%2520therapeutics%26jtitle%3DChemMedChem%26date%3D2020%26volume%3D15%26spage%3D907%26epage%3D932%26doi%3D10.1002%2Fcmdc.202000223" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boras, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anson, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arenson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aschenbrenner, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bakowski, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beutler, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Binder, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammond, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kadar, E. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kania, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kimoto, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirkpatrick, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lanyon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lendy, E. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lillis, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luthra, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Obach, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Brien, M. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Connor, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogilvie, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pettersson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reese, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossulek, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sathish, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steppan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ticehurst, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Updyke, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chatterjee, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mesecar, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allerton, C.</span></span> <span> </span><span class="NLM_article-title">Discovery of a novel inhibitor of coronavirus 3CL protease as a clinical candidate for the potential treatment of COVID-19</span>. <i>bioRxiv</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_fpage">2020.09.12.293498</span>, <span class="refDoi"> DOI: 10.1101/2020.09.12.293498</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=10.1101%2F2020.09.12.293498" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=32935104" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&pages=2020.09.12.293498&author=B.+Borasauthor=R.+M.+Jonesauthor=B.+J.+Ansonauthor=D.+Arensonauthor=L.+Aschenbrennerauthor=M.+A.+Bakowskiauthor=N.+Beutlerauthor=J.+Binderauthor=E.+Chenauthor=H.+Engauthor=J.+Hammondauthor=R.+Hoffmanauthor=E.+P.+Kadarauthor=R.+Kaniaauthor=E.+Kimotoauthor=M.+G.+Kirkpatrickauthor=L.+Lanyonauthor=E.+K.+Lendyauthor=J.+R.+Lillisauthor=S.+A.+Luthraauthor=C.+Maauthor=S.+Noellauthor=R.+S.+Obachauthor=M.+N.+O%E2%80%99Brienauthor=R.+O%E2%80%99Connorauthor=K.+Ogilvieauthor=D.+Owenauthor=M.+Petterssonauthor=M.+R.+Reeseauthor=T.+Rogersauthor=M.+I.+Rossulekauthor=J.+G.+Sathishauthor=C.+Steppanauthor=M.+Ticehurstauthor=L.+W.+Updykeauthor=Y.+Zhuauthor=J.+Wangauthor=A.+K.+Chatterjeeauthor=A.+D.+Mesecarauthor=A.+S.+Andersonauthor=C.+Allerton&title=Discovery+of+a+novel+inhibitor+of+coronavirus+3CL+protease+as+a+clinical+candidate+for+the+potential+treatment+of+COVID-19&doi=10.1101%2F2020.09.12.293498"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1101%2F2020.09.12.293498&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252F2020.09.12.293498%26sid%3Dliteratum%253Aachs%26aulast%3DBoras%26aufirst%3DB.%26aulast%3DJones%26aufirst%3DR.%2BM.%26aulast%3DAnson%26aufirst%3DB.%2BJ.%26aulast%3DArenson%26aufirst%3DD.%26aulast%3DAschenbrenner%26aufirst%3DL.%26aulast%3DBakowski%26aufirst%3DM.%2BA.%26aulast%3DBeutler%26aufirst%3DN.%26aulast%3DBinder%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DE.%26aulast%3DEng%26aufirst%3DH.%26aulast%3DHammond%26aufirst%3DJ.%26aulast%3DHoffman%26aufirst%3DR.%26aulast%3DKadar%26aufirst%3DE.%2BP.%26aulast%3DKania%26aufirst%3DR.%26aulast%3DKimoto%26aufirst%3DE.%26aulast%3DKirkpatrick%26aufirst%3DM.%2BG.%26aulast%3DLanyon%26aufirst%3DL.%26aulast%3DLendy%26aufirst%3DE.%2BK.%26aulast%3DLillis%26aufirst%3DJ.%2BR.%26aulast%3DLuthra%26aufirst%3DS.%2BA.%26aulast%3DMa%26aufirst%3DC.%26aulast%3DNoell%26aufirst%3DS.%26aulast%3DObach%26aufirst%3DR.%2BS.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DM.%2BN.%26aulast%3DO%25E2%2580%2599Connor%26aufirst%3DR.%26aulast%3DOgilvie%26aufirst%3DK.%26aulast%3DOwen%26aufirst%3DD.%26aulast%3DPettersson%26aufirst%3DM.%26aulast%3DReese%26aufirst%3DM.%2BR.%26aulast%3DRogers%26aufirst%3DT.%26aulast%3DRossulek%26aufirst%3DM.%2BI.%26aulast%3DSathish%26aufirst%3DJ.%2BG.%26aulast%3DSteppan%26aufirst%3DC.%26aulast%3DTicehurst%26aufirst%3DM.%26aulast%3DUpdyke%26aufirst%3DL.%2BW.%26aulast%3DZhu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DChatterjee%26aufirst%3DA.%2BK.%26aulast%3DMesecar%26aufirst%3DA.%2BD.%26aulast%3DAnderson%26aufirst%3DA.%2BS.%26aulast%3DAllerton%26aufirst%3DC.%26atitle%3DDiscovery%2520of%2520a%2520novel%2520inhibitor%2520of%2520coronavirus%25203CL%2520protease%2520as%2520a%2520clinical%2520candidate%2520for%2520the%2520potential%2520treatment%2520of%2520COVID-19%26jtitle%3DbioRxiv%26date%3D2020%26spage%3D2020.09.12.293498%26doi%3D10.1101%2F2020.09.12.293498" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ma, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sacco, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurst, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Townsend, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szeto, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tarbet, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marty, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span> <span> </span><span class="NLM_article-title">Boceprevir, GC-376, and calpain inhibitors II, XII inhibit SARS-CoV-2 viral replication by targeting the viral main protease</span>. <i>Cell Res.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">678</span>– <span class="NLM_lpage">692</span>, <span class="refDoi"> DOI: 10.1038/s41422-020-0356-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=10.1038%2Fs41422-020-0356-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=32541865" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=1%3ACAS%3A528%3ADC%252BB3cXht1WmsL%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2020&pages=678-692&author=C.+Maauthor=M.+D.+Saccoauthor=B.+Hurstauthor=J.+A.+Townsendauthor=Y.+Huauthor=T.+Szetoauthor=X.+Zhangauthor=B.+Tarbetauthor=M.+T.+Martyauthor=Y.+Chenauthor=J.+Wang&title=Boceprevir%2C+GC-376%2C+and+calpain+inhibitors+II%2C+XII+inhibit+SARS-CoV-2+viral+replication+by+targeting+the+viral+main+protease&doi=10.1038%2Fs41422-020-0356-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Boceprevir, GC-376, and calpain inhibitors II, XII inhibit SARS-CoV-2 viral replication by targeting the viral main protease</span></div><div class="casAuthors">Ma, Chunlong; Sacco, Michael Dominic; Hurst, Brett; Townsend, Julia Alma; Hu, Yanmei; Szeto, Tommy; Zhang, Xiujun; Tarbet, Bart; Marty, Michael Thomas; Chen, Yu; Wang, Jun</div><div class="citationInfo"><span class="NLM_cas:title">Cell Research</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">678-692</span>CODEN:
                <span class="NLM_cas:coden">CREEB6</span>;
        ISSN:<span class="NLM_cas:issn">1001-0602</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A new coronavirus SARS-CoV-2, also called novel coronavirus 2019 (2019-nCoV), started to circulate among humans around Dec. 2019, and it is now widespread as a global pandemic.  The disease caused by SARS-CoV-2 virus is called COVID-19, which is highly contagious and has an overall mortality rate of 6.35% as of May 26, 2020.  There is no vaccine or antiviral available for SARS-CoV-2.  We report our discovery of inhibitors targeting the SARS-CoV-2 main protease (Mpro).  Using the FRET-based enzymic assay, several inhibitors including boceprevir, GC-376, and calpain inhibitors II and XII were identified to have potent activity with single-digit to submicromolar IC50 values in the enzymic assay.  The mechanism of action of the hits was further characterized using enzyme kinetic studies, thermal shift binding assays, and native mass spectrometry.  Significantly, 4 compds. (boceprevir, GC-376, and calpain inhibitors II and XII) inhibit SARS-CoV-2 viral replication in cell culture with EC50 values ranging 0.49-3.37μM.  Notably, boceprevir and calpain inhibitors II and XII represent novel chemotypes that are distinct from known substrate-based peptidomimetic Mpro inhibitors.  A complex crystal structure of SARS-CoV-2 Mpro with GC-376, detd. at 2.15 Å resoln. with 3 protomers per asym. unit, revealed 2 unique binding configurations, shedding light on the mol. interactions and protein conformational flexibility underlying substrate and inhibitor binding by Mpro.  Overall, the compds. identified provide promising starting points for the further development of SARS-CoV-2 therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVnFC-t0uFW7Vg90H21EOLACvtfcHk0lgbdiXyJG7H3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXht1WmsL%252FL&md5=225d518c313474cdfbcfe678d35e26a4</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fs41422-020-0356-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41422-020-0356-z%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DC.%26aulast%3DSacco%26aufirst%3DM.%2BD.%26aulast%3DHurst%26aufirst%3DB.%26aulast%3DTownsend%26aufirst%3DJ.%2BA.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DSzeto%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DTarbet%26aufirst%3DB.%26aulast%3DMarty%26aufirst%3DM.%2BT.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DJ.%26atitle%3DBoceprevir%252C%2520GC-376%252C%2520and%2520calpain%2520inhibitors%2520II%252C%2520XII%2520inhibit%2520SARS-CoV-2%2520viral%2520replication%2520by%2520targeting%2520the%2520viral%2520main%2520protease%26jtitle%3DCell%2520Res.%26date%3D2020%26volume%3D30%26spage%3D678%26epage%3D692%26doi%3D10.1038%2Fs41422-020-0356-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sacco, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lagarias, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Townsend, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dube, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kitamura, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurst, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tarbet, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marty, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolocouris, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span> <span> </span><span class="NLM_article-title">Structure and inhibition of the SARS-CoV-2 main protease reveal strategy for developing dual inhibitors against M(pro) and cathepsin L</span>. <i>Sci. Adv.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">eabe0751</span>, <span class="refDoi"> DOI: 10.1126/sciadv.abe0751</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=10.1126%2Fsciadv.abe0751" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=33158912" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2020&pages=eabe0751&author=M.+D.+Saccoauthor=C.+Maauthor=P.+Lagariasauthor=A.+Gaoauthor=J.+A.+Townsendauthor=X.+Mengauthor=P.+Dubeauthor=X.+Zhangauthor=Y.+Huauthor=N.+Kitamuraauthor=B.+Hurstauthor=B.+Tarbetauthor=M.+T.+Martyauthor=A.+Kolocourisauthor=Y.+Xiangauthor=Y.+Chenauthor=J.+Wang&title=Structure+and+inhibition+of+the+SARS-CoV-2+main+protease+reveal+strategy+for+developing+dual+inhibitors+against+M%28pro%29+and+cathepsin+L&doi=10.1126%2Fsciadv.abe0751"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1126%2Fsciadv.abe0751&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fsciadv.abe0751%26sid%3Dliteratum%253Aachs%26aulast%3DSacco%26aufirst%3DM.%2BD.%26aulast%3DMa%26aufirst%3DC.%26aulast%3DLagarias%26aufirst%3DP.%26aulast%3DGao%26aufirst%3DA.%26aulast%3DTownsend%26aufirst%3DJ.%2BA.%26aulast%3DMeng%26aufirst%3DX.%26aulast%3DDube%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DKitamura%26aufirst%3DN.%26aulast%3DHurst%26aufirst%3DB.%26aulast%3DTarbet%26aufirst%3DB.%26aulast%3DMarty%26aufirst%3DM.%2BT.%26aulast%3DKolocouris%26aufirst%3DA.%26aulast%3DXiang%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DJ.%26atitle%3DStructure%2520and%2520inhibition%2520of%2520the%2520SARS-CoV-2%2520main%2520protease%2520reveal%2520strategy%2520for%2520developing%2520dual%2520inhibitors%2520against%2520M%2528pro%2529%2520and%2520cathepsin%2520L%26jtitle%3DSci.%2520Adv.%26date%3D2020%26volume%3D6%26spage%3Deabe0751%26doi%3D10.1126%2Fsciadv.abe0751" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rathnayake, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perera, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mackin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyerholz, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kashipathy, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Battaile, K. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lovell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perlman, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groutas, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, K. O.</span></span> <span> </span><span class="NLM_article-title">3C-like protease inhibitors block coronavirus replication in vitro and improve survival in MERS-CoV-infected mice</span>. <i>Sci. Transl. Med.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">eabc5332</span>, <span class="refDoi"> DOI: 10.1126/scitranslmed.abc5332</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=10.1126%2Fscitranslmed.abc5332" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=32747425" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2020&pages=eabc5332&author=A.+D.+Rathnayakeauthor=J.+Zhengauthor=Y.+Kimauthor=K.+D.+Pereraauthor=S.+Mackinauthor=D.+K.+Meyerholzauthor=M.+M.+Kashipathyauthor=K.+P.+Battaileauthor=S.+Lovellauthor=S.+Perlmanauthor=W.+C.+Groutasauthor=K.+O.+Chang&title=3C-like+protease+inhibitors+block+coronavirus+replication+in+vitro+and+improve+survival+in+MERS-CoV-infected+mice&doi=10.1126%2Fscitranslmed.abc5332"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1126%2Fscitranslmed.abc5332&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscitranslmed.abc5332%26sid%3Dliteratum%253Aachs%26aulast%3DRathnayake%26aufirst%3DA.%2BD.%26aulast%3DZheng%26aufirst%3DJ.%26aulast%3DKim%26aufirst%3DY.%26aulast%3DPerera%26aufirst%3DK.%2BD.%26aulast%3DMackin%26aufirst%3DS.%26aulast%3DMeyerholz%26aufirst%3DD.%2BK.%26aulast%3DKashipathy%26aufirst%3DM.%2BM.%26aulast%3DBattaile%26aufirst%3DK.%2BP.%26aulast%3DLovell%26aufirst%3DS.%26aulast%3DPerlman%26aufirst%3DS.%26aulast%3DGroutas%26aufirst%3DW.%2BC.%26aulast%3DChang%26aufirst%3DK.%2BO.%26atitle%3D3C-like%2520protease%2520inhibitors%2520block%2520coronavirus%2520replication%2520in%2520vitro%2520and%2520improve%2520survival%2520in%2520MERS-CoV-infected%2520mice%26jtitle%3DSci.%2520Transl.%2520Med.%26date%3D2020%26volume%3D12%26spage%3Deabc5332%26doi%3D10.1126%2Fscitranslmed.abc5332" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Qiao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ni, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, Z. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, Q. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, Y. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, G. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, Y. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S.</span></span> <span> </span><span class="NLM_article-title">SARS-CoV-2 M(pro) inhibitors with antiviral activity in a transgenic mouse model</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>371</i></span>,  <span class="NLM_fpage">1374</span>– <span class="NLM_lpage">1378</span>, <span class="refDoi"> DOI: 10.1126/science.abf1611</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=10.1126%2Fscience.abf1611" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=33602867" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=1%3ACAS%3A528%3ADC%252BB3MXnsVyru7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=371&publication_year=2021&pages=1374-1378&author=J.+Qiaoauthor=Y.+S.+Liauthor=R.+Zengauthor=F.+L.+Liuauthor=R.+H.+Luoauthor=C.+Huangauthor=Y.+F.+Wangauthor=J.+Zhangauthor=B.+Quanauthor=C.+Shenauthor=X.+Maoauthor=X.+Liuauthor=W.+Sunauthor=W.+Yangauthor=X.+Niauthor=K.+Wangauthor=L.+Xuauthor=Z.+L.+Duanauthor=Q.+C.+Zouauthor=H.+L.+Zhangauthor=W.+Quauthor=Y.+H.+Longauthor=M.+H.+Liauthor=R.+C.+Yangauthor=X.+Liuauthor=J.+Youauthor=Y.+Zhouauthor=R.+Yaoauthor=W.+P.+Liauthor=J.+M.+Liuauthor=P.+Chenauthor=Y.+Liuauthor=G.+F.+Linauthor=X.+Yangauthor=J.+Zouauthor=L.+Liauthor=Y.+Huauthor=G.+W.+Luauthor=W.+M.+Liauthor=Y.+Q.+Weiauthor=Y.+T.+Zhengauthor=J.+Leiauthor=S.+Yang&title=SARS-CoV-2+M%28pro%29+inhibitors+with+antiviral+activity+in+a+transgenic+mouse+model&doi=10.1126%2Fscience.abf1611"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">SARS-CoV-2 Mpro inhibitors with antiviral activity in a transgenic mouse model</span></div><div class="casAuthors">Qiao, Jingxin; Li, Yue-Shan; Zeng, Rui; Liu, Feng-Liang; Luo, Rong-Hua; Huang, Chong; Wang, Yi-Fei; Zhang, Jie; Quan, Baoxue; Shen, Chenjian; Mao, Xin; Liu, Xinlei; Sun, Weining; Yang, Wei; Ni, Xincheng; Wang, Kai; Xu, Ling; Duan, Zi-Lei; Zou, Qing-Cui; Zhang, Hai-Lin; Qu, Wang; Long, Yang-Hao-Peng; Li, Ming-Hua; Yang, Rui-Cheng; Liu, Xiaolong; You, Jing; Zhou, Yangli; Yao, Rui; Li, Wen-Pei; Liu, Jing-Ming; Chen, Pei; Liu, Yang; Lin, Gui-Feng; Yang, Xin; Zou, Jun; Li, Linli; Hu, Yiguo; Lu, Guang-Wen; Li, Wei-Min; Wei, Yu-Quan; Zheng, Yong-Tang; Lei, Jian; Yang, Shengyong</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">371</span>
        (<span class="NLM_cas:issue">6536</span>),
    <span class="NLM_cas:pages">1374-1378</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">1095-9203</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The COVID-19 pandemic caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) continually poses serious threats to global public health.  The main protease (Mpro) of SARS-CoV-2 plays a central role in viral replication.  We designed and synthesized 32 new bicycloproline-contg. Mpro inhibitors derived from either boceprevir or telaprevir, both of which are approved antivirals.  All compds. inhibited SARS-CoV-2 Mpro activity in vitro, with 50% inhibitory concn. values ranging from 7.6 to 748.5 nM.  The cocrystal structure of Mpro in complex with MI-23, one of the most potent compds., revealed its interaction mode.  Two compds. (MI-09 and MI-30) showed excellent antiviral activity in cell-based assays.  In a transgenic mouse model of SARS-CoV-2 infection, oral or i.p. treatment with MI-09 or MI-30 significantly reduced lung viral loads and lung lesions.  Both also displayed good pharmacokinetic properties and safety in rats.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVMv8s8s-oWLVg90H21EOLACvtfcHk0lh6ahnQgYOekA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXnsVyru7g%253D&md5=555937b1776c8353e59601d567e2ebb9</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1126%2Fscience.abf1611&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.abf1611%26sid%3Dliteratum%253Aachs%26aulast%3DQiao%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DY.%2BS.%26aulast%3DZeng%26aufirst%3DR.%26aulast%3DLiu%26aufirst%3DF.%2BL.%26aulast%3DLuo%26aufirst%3DR.%2BH.%26aulast%3DHuang%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DY.%2BF.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DQuan%26aufirst%3DB.%26aulast%3DShen%26aufirst%3DC.%26aulast%3DMao%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DSun%26aufirst%3DW.%26aulast%3DYang%26aufirst%3DW.%26aulast%3DNi%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DK.%26aulast%3DXu%26aufirst%3DL.%26aulast%3DDuan%26aufirst%3DZ.%2BL.%26aulast%3DZou%26aufirst%3DQ.%2BC.%26aulast%3DZhang%26aufirst%3DH.%2BL.%26aulast%3DQu%26aufirst%3DW.%26aulast%3DLong%26aufirst%3DY.%2BH.%26aulast%3DLi%26aufirst%3DM.%2BH.%26aulast%3DYang%26aufirst%3DR.%2BC.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DYou%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DYao%26aufirst%3DR.%26aulast%3DLi%26aufirst%3DW.%2BP.%26aulast%3DLiu%26aufirst%3DJ.%2BM.%26aulast%3DChen%26aufirst%3DP.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DLin%26aufirst%3DG.%2BF.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DZou%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DLu%26aufirst%3DG.%2BW.%26aulast%3DLi%26aufirst%3DW.%2BM.%26aulast%3DWei%26aufirst%3DY.%2BQ.%26aulast%3DZheng%26aufirst%3DY.%2BT.%26aulast%3DLei%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DS.%26atitle%3DSARS-CoV-2%2520M%2528pro%2529%2520inhibitors%2520with%2520antiviral%2520activity%2520in%2520a%2520transgenic%2520mouse%2520model%26jtitle%3DScience%26date%3D2021%26volume%3D371%26spage%3D1374%26epage%3D1378%26doi%3D10.1126%2Fscience.abf1611" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pedersen, N. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galasiti Kankanamalage, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eckstrand, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groutas, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bannasch, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meadows, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, K. O.</span></span> <span> </span><span class="NLM_article-title">Efficacy of a 3C-like protease inhibitor in treating various forms of acquired feline infectious peritonitis</span>. <i>J. Feline Med. Surg.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">378</span>– <span class="NLM_lpage">392</span>, <span class="refDoi"> DOI: 10.1177/1098612X17729626</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=10.1177%2F1098612X17729626" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=28901812" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=1%3ACAS%3A280%3ADC%252BC1cbns12gsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2018&pages=378-392&author=N.+C.+Pedersenauthor=Y.+Kimauthor=H.+Liuauthor=A.+C.+Galasiti+Kankanamalageauthor=C.+Eckstrandauthor=W.+C.+Groutasauthor=M.+Bannaschauthor=J.+M.+Meadowsauthor=K.+O.+Chang&title=Efficacy+of+a+3C-like+protease+inhibitor+in+treating+various+forms+of+acquired+feline+infectious+peritonitis&doi=10.1177%2F1098612X17729626"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Efficacy of a 3C-like protease inhibitor in treating various forms of acquired feline infectious peritonitis</span></div><div class="casAuthors">Pedersen Niels C; Liu Hongwei; Bannasch Michael; Kim Yunjeong; Chang Kyeong-Ok; Galasiti Kankanamalage Anushka C; Groutas William C; Eckstrand Chrissy; Meadows Juliana M</div><div class="citationInfo"><span class="NLM_cas:title">Journal of feline medicine and surgery</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">378-392</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Objectives The safety and efficacy of the 3C-like protease inhibitor GC376 was tested on a cohort of client-owned cats with various forms of feline infectious peritonitis (FIP).  Methods Twenty cats from 3.3-82 months of age (mean 10.4 months) with various forms of FIP were accepted into a field trial.  Fourteen cats presented with wet or dry-to-wet FIP and six cats presented with dry FIP.  GC376 was administered subcutaneously every 12 h at a dose of 15 mg/kg.  Cats with neurologic signs were excluded from the study.  Results Nineteen of 20 cats treated with GC376 regained outward health within 2 weeks of initial treatment.  However, disease signs recurred 1-7 weeks after primary treatment and relapses and new cases were ultimately treated for a minimum of 12 weeks.  Relapses no longer responsive to treatment occurred in 13 of these 19 cats within 1-7 weeks of initial or repeat treatment(s).  Severe neurologic disease occurred in 8/13 cats that failed treatment and five cats had recurrences of abdominal lesions.  At the time of writing, seven cats were in disease remission.  Five kittens aged 3.3-4.4 months with wet FIP were treated for 12 weeks and have been in disease remission after stopping treatment and at the time of writing for 5-14 months (mean 11.2 months).  A sixth kitten was in remission for 10 weeks after 12 weeks of treatment, relapsed and is responding to a second round of GC376.  The seventh was a 6.8-year-old cat with only mesenteric lymph node involvement that went into remission after three relapses that required progressively longer repeat treatments over a 10 month period.  Side effects of treatment included transient stinging upon injection and occasional foci of subcutaneous fibrosis and hair loss.  There was retarded development and abnormal eruption of permanent teeth in cats treated before 16-18 weeks of age.  Conclusions and relevance GC376 showed promise in treating cats with certain presentations of FIP and has opened the door to targeted antiviral drug therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTFFwhHX-Ibonqj_yg7twvpfW6udTcc2eYCxIIqrmwyjbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cbns12gsg%253D%253D&md5=69b4ebebb24b351bfb0f409e275b3153</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1177%2F1098612X17729626&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1098612X17729626%26sid%3Dliteratum%253Aachs%26aulast%3DPedersen%26aufirst%3DN.%2BC.%26aulast%3DKim%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DGalasiti%2BKankanamalage%26aufirst%3DA.%2BC.%26aulast%3DEckstrand%26aufirst%3DC.%26aulast%3DGroutas%26aufirst%3DW.%2BC.%26aulast%3DBannasch%26aufirst%3DM.%26aulast%3DMeadows%26aufirst%3DJ.%2BM.%26aulast%3DChang%26aufirst%3DK.%2BO.%26atitle%3DEfficacy%2520of%2520a%25203C-like%2520protease%2520inhibitor%2520in%2520treating%2520various%2520forms%2520of%2520acquired%2520feline%2520infectious%2520peritonitis%26jtitle%3DJ.%2520Feline%2520Med.%2520Surg.%26date%3D2018%26volume%3D20%26spage%3D378%26epage%3D392%26doi%3D10.1177%2F1098612X17729626" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galasiti Kankanamalage, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weerasekara, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hua, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groutas, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, K. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pedersen, N. C.</span></span> <span> </span><span class="NLM_article-title">Reversal of the progression of fatal coronavirus infection in cats by a broad-spectrum coronavirus protease inhibitor</span>. <i>PLoS Pathog.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">e1005531</span>, <span class="refDoi"> DOI: 10.1371/journal.ppat.1005531</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=10.1371%2Fjournal.ppat.1005531" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=27027316" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlGru7fN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=e1005531&author=Y.+Kimauthor=H.+Liuauthor=A.+C.+Galasiti+Kankanamalageauthor=S.+Weerasekaraauthor=D.+H.+Huaauthor=W.+C.+Groutasauthor=K.+O.+Changauthor=N.+C.+Pedersen&title=Reversal+of+the+progression+of+fatal+coronavirus+infection+in+cats+by+a+broad-spectrum+coronavirus+protease+inhibitor&doi=10.1371%2Fjournal.ppat.1005531"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Reversal of the progression of fatal coronavirus infection in cats by a broad-spectrum coronavirus protease inhibitor</span></div><div class="casAuthors">Kim, Yunjeong; Liu, Hongwei; Kankanamalage, Anushka C. Galasiti; Weerasekara, Sahani; Hua, Duy H.; Groutas, William C.; Chang, Kyeong-Ok; Pedersen, Niels C.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Pathogens</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">e1005531/1-e1005531/18</span>CODEN:
                <span class="NLM_cas:coden">PPLACN</span>;
        ISSN:<span class="NLM_cas:issn">1553-7374</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Coronaviruses infect animals and humans causing a wide range of diseases.  The diversity of coronaviruses in many mammalian species is contributed by relatively high mutation and recombination rates during replication.  This dynamic nature of coronaviruses may facilitate cross-species transmission and shifts in tissue or cell tropism in a host, resulting in substantial change in virulence.  Feline enteric coronavirus (FECV) causes inapparent or mild enteritis in cats, but a highly fatal disease, called feline infectious peritonitis (FIP), can arise through mutation of FECV to FIP virus (FIPV).  The pathogenesis of FIP is intimately assocd. with immune responses and involves depletion of T cells, features shared by some other coronaviruses like Severe Acute Respiratory Syndrome Coronavirus.  The increasing risks of highly virulent coronavirus infections in humans or animals call for effective antiviral drugs, but no such measures are yet available.  Previously, we have reported the inhibitors that target 3C-like protease (3CLpro) with broad-spectrum activity against important human and animal coronaviruses.  Here, we evaluated the therapeutic efficacy of our 3CLpro inhibitor in lab. cats with FIP.  Exptl. FIP is 100% fatal once certain clin. and lab. signs become apparent.We found that antiviral treatment led to full recovery of cats when treatment was started at a stage of disease that would be otherwise fatal if left untreated.  Antiviral treatment was assocd. with a rapid improvement in fever, ascites, lymphopenia and gross signs of illness and cats returned to normal health within 20 days or less of treatment.  Significant redn. in viral titers was also obsd. in cats.  These results indicate that continuous virus replication is required for progression of immune-mediated inflammatory disease of FIP.  These findings may provide important insights into devising therapeutic strategies and selection of antiviral compds. for further development for important coronaviruses in animals and humans.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprSH3BnhFwZLVg90H21EOLACvtfcHk0lgf1no_DOWUhg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlGru7fN&md5=7900b2718caeaf528f10f553e0e8fa16</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1371%2Fjournal.ppat.1005531&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.ppat.1005531%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DGalasiti%2BKankanamalage%26aufirst%3DA.%2BC.%26aulast%3DWeerasekara%26aufirst%3DS.%26aulast%3DHua%26aufirst%3DD.%2BH.%26aulast%3DGroutas%26aufirst%3DW.%2BC.%26aulast%3DChang%26aufirst%3DK.%2BO.%26aulast%3DPedersen%26aufirst%3DN.%2BC.%26atitle%3DReversal%2520of%2520the%2520progression%2520of%2520fatal%2520coronavirus%2520infection%2520in%2520cats%2520by%2520a%2520broad-spectrum%2520coronavirus%2520protease%2520inhibitor%26jtitle%3DPLoS%2520Pathog.%26date%3D2016%26volume%3D12%26spage%3De1005531%26doi%3D10.1371%2Fjournal.ppat.1005531" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jacobs, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grum-Tokars, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turlington, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saldanha, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chase, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eggler, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baez-Santos, Y. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mielech, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindsley, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hodder, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mesecar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stauffer, S. R.</span></span> <span> </span><span class="NLM_article-title">Discovery, synthesis, and structure-based optimization of a series of N-(tert-butyl)-2-(N-arylamido)-2-(pyridin-3-yl) acetamides (ML188) as potent noncovalent small molecule inhibitors of the severe acute respiratory syndrome coronavirus (SARS-CoV) 3CL protease</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">534</span>– <span class="NLM_lpage">46</span>, <span class="refDoi"> DOI: 10.1021/jm301580n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm301580n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVWqtL%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=534-46&author=J.+Jacobsauthor=V.+Grum-Tokarsauthor=Y.+Zhouauthor=M.+Turlingtonauthor=S.+A.+Saldanhaauthor=P.+Chaseauthor=A.+Egglerauthor=E.+S.+Dawsonauthor=Y.+M.+Baez-Santosauthor=S.+Tomarauthor=A.+M.+Mielechauthor=S.+C.+Bakerauthor=C.+W.+Lindsleyauthor=P.+Hodderauthor=A.+Mesecarauthor=S.+R.+Stauffer&title=Discovery%2C+synthesis%2C+and+structure-based+optimization+of+a+series+of+N-%28tert-butyl%29-2-%28N-arylamido%29-2-%28pyridin-3-yl%29+acetamides+%28ML188%29+as+potent+noncovalent+small+molecule+inhibitors+of+the+severe+acute+respiratory+syndrome+coronavirus+%28SARS-CoV%29+3CL+protease&doi=10.1021%2Fjm301580n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery, Synthesis, And Structure-Based Optimization of a Series of N-(tert-Butyl)-2-(N-arylamido)-2-(pyridin-3-yl) Acetamides (ML188) as Potent Noncovalent Small Molecule Inhibitors of the Severe Acute Respiratory Syndrome Coronavirus (SARS-CoV) 3CL Protease</span></div><div class="casAuthors">Jacobs, Jon; Grum-Tokars, Valerie; Zhou, Ya; Turlington, Mark; Saldanha, S. Adrian; Chase, Peter; Eggler, Aimee; Dawson, Eric S.; Baez-Santos, Yahira M.; Tomar, Sakshi; Mielech, Anna M.; Baker, Susan C.; Lindsley, Craig W.; Hodder, Peter; Mesecar, Andrew; Stauffer, Shaun R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">534-546</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A high-throughput screen of the NIH mol. libraries sample collection and subsequent optimization of a lead dipeptide-like series of severe acute respiratory syndrome (SARS) main protease (3CLpro) inhibitors led to the identification of probe compd. ML188 (I), [(R)-N-(4-(tert-butyl)phenyl)-N-(2-(tert-butylamino)-2-oxo-1-(pyridin-3-yl)ethyl)furan-2-carboxamide, Pubchem CID: 46897844].  Unlike the majority of reported coronavirus 3CLpro inhibitors that act via covalent modification of the enzyme, I is a noncovalent SARS-CoV 3CLpro inhibitor with moderate MW and good enzyme and antiviral inhibitory activity.  A multicomponent Ugi reaction was utilized to rapidly explore structure-activity relationships within S1', S1, and S2 enzyme binding pockets.  The X-ray structure of SARS-CoV 3CLpro bound with I was instrumental in guiding subsequent rounds of chem. optimization.  I provides an excellent starting point for the further design and refinement of 3CLpro inhibitors that act by a noncovalent mechanism of action.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJ_0gMuIcLErVg90H21EOLACvtfcHk0lgf1no_DOWUhg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVWqtL%252FE&md5=6f619d5642a8d29bdb9eee0b28a80247</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Fjm301580n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm301580n%26sid%3Dliteratum%253Aachs%26aulast%3DJacobs%26aufirst%3DJ.%26aulast%3DGrum-Tokars%26aufirst%3DV.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DTurlington%26aufirst%3DM.%26aulast%3DSaldanha%26aufirst%3DS.%2BA.%26aulast%3DChase%26aufirst%3DP.%26aulast%3DEggler%26aufirst%3DA.%26aulast%3DDawson%26aufirst%3DE.%2BS.%26aulast%3DBaez-Santos%26aufirst%3DY.%2BM.%26aulast%3DTomar%26aufirst%3DS.%26aulast%3DMielech%26aufirst%3DA.%2BM.%26aulast%3DBaker%26aufirst%3DS.%2BC.%26aulast%3DLindsley%26aufirst%3DC.%2BW.%26aulast%3DHodder%26aufirst%3DP.%26aulast%3DMesecar%26aufirst%3DA.%26aulast%3DStauffer%26aufirst%3DS.%2BR.%26atitle%3DDiscovery%252C%2520synthesis%252C%2520and%2520structure-based%2520optimization%2520of%2520a%2520series%2520of%2520N-%2528tert-butyl%2529-2-%2528N-arylamido%2529-2-%2528pyridin-3-yl%2529%2520acetamides%2520%2528ML188%2529%2520as%2520potent%2520noncovalent%2520small%2520molecule%2520inhibitors%2520of%2520the%2520severe%2520acute%2520respiratory%2520syndrome%2520coronavirus%2520%2528SARS-CoV%2529%25203CL%2520protease%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D534%26epage%3D46%26doi%3D10.1021%2Fjm301580n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Turlington, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chun, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eggler, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grum-Tokars, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobs, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniels, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saldanha, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chase, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baez-Santos, Y. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindsley, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hodder, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mesecar, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stauffer, S. R.</span></span> <span> </span><span class="NLM_article-title">Discovery of N-(benzo[1,2,3]triazol-1-yl)-N-(benzyl)acetamido)phenyl) carboxamides as severe acute respiratory syndrome coronavirus (SARS-CoV) 3CLpro inhibitors: Identification of ML300 and noncovalent nanomolar inhibitors with an induced-fit binding</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">6172</span>– <span class="NLM_lpage">6177</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2013.08.112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=10.1016%2Fj.bmcl.2013.08.112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=24080461" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFGqtr3L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2013&pages=6172-6177&author=M.+Turlingtonauthor=A.+Chunauthor=S.+Tomarauthor=A.+Egglerauthor=V.+Grum-Tokarsauthor=J.+Jacobsauthor=J.+S.+Danielsauthor=E.+Dawsonauthor=A.+Saldanhaauthor=P.+Chaseauthor=Y.+M.+Baez-Santosauthor=C.+W.+Lindsleyauthor=P.+Hodderauthor=A.+D.+Mesecarauthor=S.+R.+Stauffer&title=Discovery+of+N-%28benzo%5B1%2C2%2C3%5Dtriazol-1-yl%29-N-%28benzyl%29acetamido%29phenyl%29+carboxamides+as+severe+acute+respiratory+syndrome+coronavirus+%28SARS-CoV%29+3CLpro+inhibitors%3A+Identification+of+ML300+and+noncovalent+nanomolar+inhibitors+with+an+induced-fit+binding&doi=10.1016%2Fj.bmcl.2013.08.112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of N-(benzo[1,2,3]triazol-1-yl)-N-((benzyl)acetamido)phenyl carboxamides as severe acute respiratory syndrome coronavirus (SARS-CoV) 3CLpro inhibitors: Identification of ML300 and noncovalent nanomolar inhibitors with an induced-fit binding</span></div><div class="casAuthors">Turlington, Mark; Chun, Aspen; Tomar, Sakshi; Eggler, Aimee; Grum-Tokars, Valerie; Jacobs, Jon; Daniels, J. Scott; Dawson, Eric; Saldanha, Adrian; Chase, Peter; Baez-Santos, Yahira M.; Lindsley, Craig W.; Hodder, Peter; Mesecar, Andrew D.; Stauffer, Shaun R.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">6172-6177</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Herein we report the discovery and SAR of a novel series of SARS-CoV 3CLpro inhibitors identified through the NIH Mol. Libraries Probe Prodn. Centers Network (MLPCN).  In addn. to ML188, ML300 represents the second probe declared for 3CLpro from this collaborative effort.  The x-ray structure of SARS-CoV 3CLpro bound with a ML300 analog highlights a unique induced-fit reorganization of the S2-S4 binding pockets leading to the first sub-micromolar noncovalent 3CLpro inhibitors retaining a single amide bond.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSJzh-nO55O7Vg90H21EOLACvtfcHk0lgf1no_DOWUhg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFGqtr3L&md5=f4ee0a6a9222900ad75822851f22b2be</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2013.08.112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2013.08.112%26sid%3Dliteratum%253Aachs%26aulast%3DTurlington%26aufirst%3DM.%26aulast%3DChun%26aufirst%3DA.%26aulast%3DTomar%26aufirst%3DS.%26aulast%3DEggler%26aufirst%3DA.%26aulast%3DGrum-Tokars%26aufirst%3DV.%26aulast%3DJacobs%26aufirst%3DJ.%26aulast%3DDaniels%26aufirst%3DJ.%2BS.%26aulast%3DDawson%26aufirst%3DE.%26aulast%3DSaldanha%26aufirst%3DA.%26aulast%3DChase%26aufirst%3DP.%26aulast%3DBaez-Santos%26aufirst%3DY.%2BM.%26aulast%3DLindsley%26aufirst%3DC.%2BW.%26aulast%3DHodder%26aufirst%3DP.%26aulast%3DMesecar%26aufirst%3DA.%2BD.%26aulast%3DStauffer%26aufirst%3DS.%2BR.%26atitle%3DDiscovery%2520of%2520N-%2528benzo%255B1%252C2%252C3%255Dtriazol-1-yl%2529-N-%2528benzyl%2529acetamido%2529phenyl%2529%2520carboxamides%2520as%2520severe%2520acute%2520respiratory%2520syndrome%2520coronavirus%2520%2528SARS-CoV%2529%25203CLpro%2520inhibitors%253A%2520Identification%2520of%2520ML300%2520and%2520noncovalent%2520nanomolar%2520inhibitors%2520with%2520an%2520induced-fit%2520binding%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2013%26volume%3D23%26spage%3D6172%26epage%3D6177%26doi%3D10.1016%2Fj.bmcl.2013.08.112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zaidman, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gehrtz, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filep, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fearon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prilusky, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duberstein, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Resnick, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strain-Damerell, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lukacik, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barr, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walsh, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Delft, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">London, N.</span></span> <span> </span><span class="NLM_article-title">An automatic pipeline for the design of irreversible derivatives identifies a potent SARS-CoV-2 Mpro inhibitor</span>. <i>bioRxiv</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_fpage">2020.09.21.299776</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&pages=2020.09.21.299776&author=D.+Zaidmanauthor=P.+Gehrtzauthor=M.+Filepauthor=D.+Fearonauthor=J.+Priluskyauthor=S.+Dubersteinauthor=G.+Cohenauthor=D.+Owenauthor=E.+Resnickauthor=C.+Strain-Damerellauthor=P.+Lukacikauthor=H.+Barrauthor=M.+A.+Walshauthor=F.+von+Delftauthor=N.+London&title=An+automatic+pipeline+for+the+design+of+irreversible+derivatives+identifies+a+potent+SARS-CoV-2+Mpro+inhibitor"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DZaidman%26aufirst%3DD.%26aulast%3DGehrtz%26aufirst%3DP.%26aulast%3DFilep%26aufirst%3DM.%26aulast%3DFearon%26aufirst%3DD.%26aulast%3DPrilusky%26aufirst%3DJ.%26aulast%3DDuberstein%26aufirst%3DS.%26aulast%3DCohen%26aufirst%3DG.%26aulast%3DOwen%26aufirst%3DD.%26aulast%3DResnick%26aufirst%3DE.%26aulast%3DStrain-Damerell%26aufirst%3DC.%26aulast%3DLukacik%26aufirst%3DP.%26aulast%3DBarr%26aufirst%3DH.%26aulast%3DWalsh%26aufirst%3DM.%2BA.%26aulast%3Dvon%2BDelft%26aufirst%3DF.%26aulast%3DLondon%26aufirst%3DN.%26atitle%3DAn%2520automatic%2520pipeline%2520for%2520the%2520design%2520of%2520irreversible%2520derivatives%2520identifies%2520a%2520potent%2520SARS-CoV-2%2520Mpro%2520inhibitor%26jtitle%3DbioRxiv%26date%3D2020%26spage%3D2020.09.21.299776" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, D. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X. Y. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curth, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drosten, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sauerhering, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rox, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilgenfeld, R.</span></span> <span> </span><span class="NLM_article-title">Crystal structure of SARS-CoV-2 main protease provides a basis for design of improved alpha-ketoamide inhibitors</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>368</i></span>,  <span class="NLM_fpage">409</span>– <span class="NLM_lpage">412</span>, <span class="refDoi"> DOI: 10.1126/science.abb3405</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=10.1126%2Fscience.abb3405" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=32198291" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=1%3ACAS%3A528%3ADC%252BB3cXnslKrtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=368&publication_year=2020&pages=409-412&author=L.+L.+Zhangauthor=D.+Z.+Linauthor=X.+Y.+Y.+Sunauthor=U.+Curthauthor=C.+Drostenauthor=L.+Sauerheringauthor=S.+Beckerauthor=K.+Roxauthor=R.+Hilgenfeld&title=Crystal+structure+of+SARS-CoV-2+main+protease+provides+a+basis+for+design+of+improved+alpha-ketoamide+inhibitors&doi=10.1126%2Fscience.abb3405"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of SARS-CoV-2 main protease provides a basis for design of improved α-ketoamide inhibitors</span></div><div class="casAuthors">Zhang, Linlin; Lin, Daizong; Sun, Xinyuanyuan; Curth, Ute; Drosten, Christian; Sauerhering, Lucie; Becker, Stephan; Rox, Katharina; Hilgenfeld, Rolf</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">368</span>
        (<span class="NLM_cas:issue">6489</span>),
    <span class="NLM_cas:pages">409-412</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">1095-9203</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The coronavirus disease 2019 (COVID-19) pandemic caused by severe acute respiratory syndrome-coronavirus 2 (SARS-CoV-2) is a global health emergency.  An attractive drug target among coronaviruses is the main protease (Mpro, also called 3CLpro) because of its essential role in processing the polyproteins that are translated from the viral RNA.  We report the x-ray structures of the unliganded SARS-CoV-2 Mpro and its complex with an α-ketoamide inhibitor.  This was derived from a previously designed inhibitor but with the P3-P2 amide bond incorporated into a pyridone ring to enhance the half-life of the compd. in plasma.  On the basis of the unliganded structure, we developed the lead compd. into a potent inhibitor of the SARS-CoV-2 Mpro.  The pharmacokinetic characterization of the optimized inhibitor reveals a pronounced lung tropism and suitability for administration by the inhalative route.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrT531N47m4BbVg90H21EOLACvtfcHk0lgl8QrJIjSOfg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXnslKrtL8%253D&md5=9ac417c20f54c3327f9de9088b512d52</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1126%2Fscience.abb3405&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.abb3405%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DL.%2BL.%26aulast%3DLin%26aufirst%3DD.%2BZ.%26aulast%3DSun%26aufirst%3DX.%2BY.%2BY.%26aulast%3DCurth%26aufirst%3DU.%26aulast%3DDrosten%26aufirst%3DC.%26aulast%3DSauerhering%26aufirst%3DL.%26aulast%3DBecker%26aufirst%3DS.%26aulast%3DRox%26aufirst%3DK.%26aulast%3DHilgenfeld%26aufirst%3DR.%26atitle%3DCrystal%2520structure%2520of%2520SARS-CoV-2%2520main%2520protease%2520provides%2520a%2520basis%2520for%2520design%2520of%2520improved%2520alpha-ketoamide%2520inhibitors%26jtitle%3DScience%26date%3D2020%26volume%3D368%26spage%3D409%26epage%3D412%26doi%3D10.1126%2Fscience.abb3405" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foley, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Musharrafieh, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hulme, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span> <span> </span><span class="NLM_article-title">Exploring ugi-azide four-component reaction products for broad-spectrum influenza antivirals with a high genetic barrier to drug resistance</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">4653</span>, <span class="refDoi"> DOI: 10.1038/s41598-018-22875-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=10.1038%2Fs41598-018-22875-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=29545578" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=1%3ACAS%3A280%3ADC%252BC1MnhslKhsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&pages=4653&author=J.+Zhangauthor=Y.+Huauthor=C.+Foleyauthor=Y.+Wangauthor=R.+Musharrafiehauthor=S.+Xuauthor=Y.+Zhangauthor=C.+Maauthor=C.+Hulmeauthor=J.+Wang&title=Exploring+ugi-azide+four-component+reaction+products+for+broad-spectrum+influenza+antivirals+with+a+high+genetic+barrier+to+drug+resistance&doi=10.1038%2Fs41598-018-22875-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Exploring Ugi-Azide Four-Component Reaction Products for Broad-Spectrum Influenza Antivirals with a High Genetic Barrier to Drug Resistance</span></div><div class="casAuthors">Zhang Jiantao; Hu Yanmei; Wang Yuanxiang; Wang Jun; Foley Christopher; Musharrafieh Rami; Xu Shuting; Zhang Yongtao; Hulme Christopher; Ma Chunlong; Wang Jun</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">4653</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Influenza viruses are respiratory pathogens that are responsible for seasonal influenza and sporadic influenza pandemic.  The therapeutic efficacy of current influenza vaccines and small molecule antiviral drugs is limited due to the emergence of multidrug-resistant influenza viruses.  In response to the urgent need for the next generation of influenza antivirals, we utilized a fast-track drug discovery platform by exploring multi-component reaction products for antiviral drug candidates.  Specifically, molecular docking was applied to screen a small molecule library derived from the Ugi-azide four-component reaction methodology for inhibitors that target the influenza polymerase PAC-PB1N interactions.  One hit compound 5 was confirmed to inhibit PAC-PB1N interactions in an ELISA assay and had potent antiviral activity in an antiviral plaque assay.  Subsequent structure-activity relationship studies led to the discovery of compound 12a, which had broad-spectrum antiviral activity and a higher in vitro genetic barrier to drug resistance than oseltamivir.  Overall, the discovery of compound 12a as a broad-spectrum influenza antiviral with a high in vitro genetic barrier to drug resistance is significant, as it offers a second line of defense to combat the next influenza epidemics and pandemics if vaccines and oseltamivir fail to confine the disease outbreak.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRR96PkqgseZ6_dlEzl_fBafW6udTcc2eZCb0l4peLHz7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MnhslKhsA%253D%253D&md5=efc7cdeea4a5099a341aa8d177f0f8c8</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fs41598-018-22875-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-018-22875-9%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DFoley%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DMusharrafieh%26aufirst%3DR.%26aulast%3DXu%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DC.%26aulast%3DHulme%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DJ.%26atitle%3DExploring%2520ugi-azide%2520four-component%2520reaction%2520products%2520for%2520broad-spectrum%2520influenza%2520antivirals%2520with%2520a%2520high%2520genetic%2520barrier%2520to%2520drug%2520resistance%26jtitle%3DSci.%2520Rep.%26date%3D2018%26volume%3D8%26spage%3D4653%26doi%3D10.1038%2Fs41598-018-22875-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Siklos, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">BenAissa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thatcher, G. R. J.</span></span> <span> </span><span class="NLM_article-title">Cysteine proteases as therapeutic targets: does selectivity matter? A systematic review of calpain and cathepsin inhibitors</span>. <i>Acta Pharm. Sin. B</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">506</span>– <span class="NLM_lpage">519</span>, <span class="refDoi"> DOI: 10.1016/j.apsb.2015.08.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=10.1016%2Fj.apsb.2015.08.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=26713267" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=1%3ACAS%3A280%3ADC%252BC28rlslWjsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=506-519&author=M.+Siklosauthor=M.+BenAissaauthor=G.+R.+J.+Thatcher&title=Cysteine+proteases+as+therapeutic+targets%3A+does+selectivity+matter%3F+A+systematic+review+of+calpain+and+cathepsin+inhibitors&doi=10.1016%2Fj.apsb.2015.08.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Cysteine proteases as therapeutic targets: does selectivity matter? A systematic review of calpain and cathepsin inhibitors</span></div><div class="casAuthors">Siklos Marton; BenAissa Manel; Thatcher Gregory R J</div><div class="citationInfo"><span class="NLM_cas:title">Acta pharmaceutica Sinica. B</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">506-19</span>
        ISSN:<span class="NLM_cas:issn">2211-3835</span>.
    </div><div class="casAbstract">Cysteine proteases continue to provide validated targets for treatment of human diseases.  In neurodegenerative disorders, multiple cysteine proteases provide targets for enzyme inhibitors, notably caspases, calpains, and cathepsins.  The reactive, active-site cysteine provides specificity for many inhibitor designs over other families of proteases, such as aspartate and serine; however, a) inhibitor strategies often use covalent enzyme modification, and b) obtaining selectivity within families of cysteine proteases and their isozymes is problematic.  This review provides a general update on strategies for cysteine protease inhibitor design and a focus on cathepsin B and calpain 1 as drug targets for neurodegenerative disorders; the latter focus providing an interesting query for the contemporary assumptions that irreversible, covalent protein modification and low selectivity are anathema to therapeutic safety and efficacy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQE6JaFL9P83mBAiYWy6n3SfW6udTcc2eYRoO0BswHyz7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28rlslWjsg%253D%253D&md5=cbd246a39984c0d9e3a32fd26db8902d</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.apsb.2015.08.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.apsb.2015.08.001%26sid%3Dliteratum%253Aachs%26aulast%3DSiklos%26aufirst%3DM.%26aulast%3DBenAissa%26aufirst%3DM.%26aulast%3DThatcher%26aufirst%3DG.%2BR.%2BJ.%26atitle%3DCysteine%2520proteases%2520as%2520therapeutic%2520targets%253A%2520does%2520selectivity%2520matter%253F%2520A%2520systematic%2520review%2520of%2520calpain%2520and%2520cathepsin%2520inhibitors%26jtitle%3DActa%2520Pharm.%2520Sin.%2520B%26date%3D2015%26volume%3D5%26spage%3D506%26epage%3D519%26doi%3D10.1016%2Fj.apsb.2015.08.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cianni, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feldmann, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilberg, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gutschow, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Juliano, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leitao, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montanari, C. A.</span></span> <span> </span><span class="NLM_article-title">Can cysteine protease cross-class inhibitors achieve selectivity?</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">10497</span>– <span class="NLM_lpage">10525</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00683</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00683" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsVGkur7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=10497-10525&author=L.+Cianniauthor=C.+W.+Feldmannauthor=E.+Gilbergauthor=M.+Gutschowauthor=L.+Julianoauthor=A.+Leitaoauthor=J.+Bajorathauthor=C.+A.+Montanari&title=Can+cysteine+protease+cross-class+inhibitors+achieve+selectivity%3F&doi=10.1021%2Facs.jmedchem.9b00683"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Can Cysteine Protease Cross-Class Inhibitors Achieve Selectivity?</span></div><div class="casAuthors">Cianni, Lorenzo; Feldmann, Christian Wolfgang; Gilberg, Erik; Gutschow, Michael; Juliano, Luiz; Leitao, Andrei; Bajorath, Jurgen; Montanari, Carlos A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">10497-10525</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Cysteine proteases are important targets for the discovery of novel therapeutics for many human diseases.  From parasitic diseases to cancer, cysteine proteases follow a common mechanism, the formation of an encounter complex with subsequent nucleophilic reactivity of the catalytic cysteine thiol group toward the carbonyl carbon of a peptide bond or an electrophilic group of an inhibitor.  Modulation of target enzymes occurs preferably by covalent modification, which imposes challenges in balancing cross-reactivity and selectivity.  Given the resurgence of irreversible covalent inhibitors, can they impair off-target effects or are reversible covalent inhibitors a better route to selectivity.  This Perspective addresses how small mol. inhibitors may achieve selectivity for different cathepsins, cruzain, rhodesain, and falcipain-2.  We discuss target- and ligand-based designs emphasizing repurposing inhibitors from one cysteine protease to others.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr75s_t-OU9gbVg90H21EOLACvtfcHk0lg4B-TKu8ZBGw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsVGkur7I&md5=49fc0900174dfad403d33e906498ae28</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00683&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00683%26sid%3Dliteratum%253Aachs%26aulast%3DCianni%26aufirst%3DL.%26aulast%3DFeldmann%26aufirst%3DC.%2BW.%26aulast%3DGilberg%26aufirst%3DE.%26aulast%3DGutschow%26aufirst%3DM.%26aulast%3DJuliano%26aufirst%3DL.%26aulast%3DLeitao%26aufirst%3DA.%26aulast%3DBajorath%26aufirst%3DJ.%26aulast%3DMontanari%26aufirst%3DC.%2BA.%26atitle%3DCan%2520cysteine%2520protease%2520cross-class%2520inhibitors%2520achieve%2520selectivity%253F%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D10497%26epage%3D10525%26doi%3D10.1021%2Facs.jmedchem.9b00683" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Steuten, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Widen, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Babin, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Onguka, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lovell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolgi, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cerikan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cortese, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muir, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geiss-Friedlander, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peters, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartenschlager, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bogyo, M.</span></span> <span> </span><span class="NLM_article-title">Challenges for targeting SARS-CoV-2 proteases as a therapeutic strategy for COVID-19</span>. <i>ACS Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>, <span class="refDoi"> DOI: 10.1021/acsinfecdis.0c00815</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsinfecdis.0c00815" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2021&author=K.+Steutenauthor=H.+Kimauthor=J.+C.+Widenauthor=B.+M.+Babinauthor=O.+Ongukaauthor=S.+Lovellauthor=O.+Bolgiauthor=B.+Cerikanauthor=M.+Corteseauthor=R.+K.+Muirauthor=J.+M.+Bennettauthor=R.+Geiss-Friedlanderauthor=C.+Petersauthor=R.+Bartenschlagerauthor=M.+Bogyo&title=Challenges+for+targeting+SARS-CoV-2+proteases+as+a+therapeutic+strategy+for+COVID-19&doi=10.1021%2Facsinfecdis.0c00815"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facsinfecdis.0c00815&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsinfecdis.0c00815%26sid%3Dliteratum%253Aachs%26aulast%3DSteuten%26aufirst%3DK.%26aulast%3DKim%26aufirst%3DH.%26aulast%3DWiden%26aufirst%3DJ.%2BC.%26aulast%3DBabin%26aufirst%3DB.%2BM.%26aulast%3DOnguka%26aufirst%3DO.%26aulast%3DLovell%26aufirst%3DS.%26aulast%3DBolgi%26aufirst%3DO.%26aulast%3DCerikan%26aufirst%3DB.%26aulast%3DCortese%26aufirst%3DM.%26aulast%3DMuir%26aufirst%3DR.%2BK.%26aulast%3DBennett%26aufirst%3DJ.%2BM.%26aulast%3DGeiss-Friedlander%26aufirst%3DR.%26aulast%3DPeters%26aufirst%3DC.%26aulast%3DBartenschlager%26aufirst%3DR.%26aulast%3DBogyo%26aufirst%3DM.%26atitle%3DChallenges%2520for%2520targeting%2520SARS-CoV-2%2520proteases%2520as%2520a%2520therapeutic%2520strategy%2520for%2520COVID-19%26jtitle%3DACS%2520Infect.%2520Dis.%26date%3D2021%26doi%3D10.1021%2Facsinfecdis.0c00815" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meek, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shang, L.</span></span> <span> </span><span class="NLM_article-title">Application of dually activated michael acceptor to the rational design of reversible covalent inhibitor for enterovirus 71 3C protease</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">6146</span>– <span class="NLM_lpage">6162</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00387</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00387" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtFGqsLjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=6146-6162&author=Y.+Maauthor=L.+Liauthor=S.+Heauthor=C.+Shangauthor=Y.+Sunauthor=N.+Liuauthor=T.+D.+Meekauthor=Y.+Wangauthor=L.+Shang&title=Application+of+dually+activated+michael+acceptor+to+the+rational+design+of+reversible+covalent+inhibitor+for+enterovirus+71+3C+protease&doi=10.1021%2Facs.jmedchem.9b00387"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Application of Dually Activated Michael Acceptor to the Rational Design of Reversible Covalent Inhibitor for Enterovirus 71 3C Protease</span></div><div class="casAuthors">Ma, Yuying; Li, Linfeng; He, Shuai; Shang, Chengyou; Sun, Yang; Liu, Ning; Meek, Thomas D.; Wang, Yaxin; Shang, Luqing</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">6146-6162</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Targeted covalent inhibitors (TCIs) have attracted growing attention from the pharmaceutical industry in recent decades because they have potential advantages in terms of efficacy, selectivity, and safety.  TCIs have recently evolved into a new version with reversibility that can be systematically modulated.  This feature may diminish the risk of haptenization and help optimize the drug-target residence time as needed.  The enteroviral 3C protease (3Cpro) is a valuable therapeutic target, but the development of 3Cpro inhibitors is far from satisfactory.  Therefore, we aimed to apply a reversible TCI approach to the design of novel 3Cpro inhibitors.  The introduction of various substituents onto the α-carbon of classical Michael acceptors yielded inhibitors bearing several classes of warheads.  Using steady-state kinetics and biomol. mass spectrometry, we confirmed the mode of reversible covalent inhibition and elucidated the mechanism by which the potency and reversibility were affected by electronic and steric factors.  This research produced several potent inhibitors with good selectivity and suitable reversibility; moreover, it validated the reversible TCI approach in the field of viral infection, suggesting broader applications in the design of reversible covalent inhibitors for other proteases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkpF34VsXG8bVg90H21EOLACvtfcHk0lh6OHoPQAk8yA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtFGqsLjI&md5=2de757026f2542bff8092060a72107a5</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00387&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00387%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DHe%26aufirst%3DS.%26aulast%3DShang%26aufirst%3DC.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DN.%26aulast%3DMeek%26aufirst%3DT.%2BD.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DShang%26aufirst%3DL.%26atitle%3DApplication%2520of%2520dually%2520activated%2520michael%2520acceptor%2520to%2520the%2520rational%2520design%2520of%2520reversible%2520covalent%2520inhibitor%2520for%2520enterovirus%252071%25203C%2520protease%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D6146%26epage%3D6162%26doi%3D10.1021%2Facs.jmedchem.9b00387" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jin, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guddat, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span> <span> </span><span class="NLM_article-title">Structure of M(pro) from SARS-CoV-2 and discovery of its inhibitors</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>582</i></span>,  <span class="NLM_fpage">289</span>– <span class="NLM_lpage">293</span>, <span class="refDoi"> DOI: 10.1038/s41586-020-2223-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=10.1038%2Fs41586-020-2223-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=32272481" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtVyhsrrO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=582&publication_year=2020&pages=289-293&author=Z.+Jinauthor=X.+Duauthor=Y.+Xuauthor=Y.+Dengauthor=M.+Liuauthor=Y.+Zhaoauthor=B.+Zhangauthor=X.+Liauthor=L.+Zhangauthor=C.+Pengauthor=Y.+Duanauthor=J.+Yuauthor=L.+Wangauthor=K.+Yangauthor=F.+Liuauthor=R.+Jiangauthor=X.+Yangauthor=T.+Youauthor=X.+Liuauthor=X.+Yangauthor=F.+Baiauthor=H.+Liuauthor=X.+Liuauthor=L.+W.+Guddatauthor=W.+Xuauthor=G.+Xiaoauthor=C.+Qinauthor=Z.+Shiauthor=H.+Jiangauthor=Z.+Raoauthor=H.+Yang&title=Structure+of+M%28pro%29+from+SARS-CoV-2+and+discovery+of+its+inhibitors&doi=10.1038%2Fs41586-020-2223-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of Mpro from SARS-CoV-2 and discovery of its inhibitors</span></div><div class="casAuthors">Jin, Zhenming; Du, Xiaoyu; Xu, Yechun; Deng, Yongqiang; Liu, Meiqin; Zhao, Yao; Zhang, Bing; Li, Xiaofeng; Zhang, Leike; Peng, Chao; Duan, Yinkai; Yu, Jing; Wang, Lin; Yang, Kailin; Liu, Fengjiang; Jiang, Rendi; Yang, Xinglou; You, Tian; Liu, Xiaoce; Yang, Xiuna; Bai, Fang; Liu, Hong; Liu, Xiang; Guddat, Luke W.; Xu, Wenqing; Xiao, Gengfu; Qin, Chengfeng; Shi, Zhengli; Jiang, Hualiang; Rao, Zihe; Yang, Haitao</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">582</span>
        (<span class="NLM_cas:issue">7811</span>),
    <span class="NLM_cas:pages">289-293</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: A new coronavirus, known as severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), is the etiol. agent responsible for the 2019-2020 viral pneumonia outbreak of coronavirus disease 2019 (COVID-19).  Currently, there are no targeted therapeutic agents for the treatment of this disease, and effective treatment options remain very limited.  Here, we describe the results of a program that aimed to rapidly discover lead compds. for clin. use, by combining structure-assisted drug design, virtual drug screening and high-throughput screening.  This program focused on identifying drug leads that target main protease (Mpro) of SARS-CoV-2: Mpro is a key enzyme of coronaviruses and has a pivotal role in mediating viral replication and transcription, making it an attractive drug target for SARS-CoV-2.  We identified a mechanism-based inhibitor (N3) by computer-aided drug design, and then detd. the crystal structure of Mpro of SARS-CoV-2 in complex with this compd.  Through a combination of structure-based virtual and high-throughput screening, we assayed more than 10,000 compds.-including approved drugs, drug candidates in clin. trials and other pharmacol. active compds.-as inhibitors of Mpro.  Six of these compds. inhibited Mpro, showing half-maximal inhibitory concn. values that ranged from 0.67 to 21.4μM.  One of these compds. (ebselen) also exhibited promising antiviral activity in cell-based assays.  Our results demonstrate the efficacy of our screening strategy, which can lead to the rapid discovery of drug leads with clin. potential in response to new infectious diseases for which no specific drugs or vaccines are available.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFupuh6WIyW7Vg90H21EOLACvtfcHk0lh6OHoPQAk8yA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtVyhsrrO&md5=b84f350fe9ce1109485df6caf814ba82</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fs41586-020-2223-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-020-2223-y%26sid%3Dliteratum%253Aachs%26aulast%3DJin%26aufirst%3DZ.%26aulast%3DDu%26aufirst%3DX.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DDeng%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DM.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DB.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DPeng%26aufirst%3DC.%26aulast%3DDuan%26aufirst%3DY.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DJiang%26aufirst%3DR.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DYou%26aufirst%3DT.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DBai%26aufirst%3DF.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DGuddat%26aufirst%3DL.%2BW.%26aulast%3DXu%26aufirst%3DW.%26aulast%3DXiao%26aufirst%3DG.%26aulast%3DQin%26aufirst%3DC.%26aulast%3DShi%26aufirst%3DZ.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DRao%26aufirst%3DZ.%26aulast%3DYang%26aufirst%3DH.%26atitle%3DStructure%2520of%2520M%2528pro%2529%2520from%2520SARS-CoV-2%2520and%2520discovery%2520of%2520its%2520inhibitors%26jtitle%3DNature%26date%3D2020%26volume%3D582%26spage%3D289%26epage%3D293%26doi%3D10.1038%2Fs41586-020-2223-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCormick, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">Crystal structure of SARS-CoV-2 main protease in complex with the natural product inhibitor shikonin illuminates a unique binding mode</span>. <i>Sci. Bull. (Beijing)</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">661</span>– <span class="NLM_lpage">663</span>, <span class="refDoi"> DOI: 10.1016/j.scib.2020.10.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=10.1016%2Fj.scib.2020.10.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=33163253" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=1%3ACAS%3A280%3ADC%252BB3s3itFSisQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2021&pages=661-663&author=J.+Liauthor=X.+Zhouauthor=Y.+Zhangauthor=F.+Zhongauthor=C.+Linauthor=P.+J.+McCormickauthor=F.+Jiangauthor=J.+Luoauthor=H.+Zhouauthor=Q.+Wangauthor=Y.+Fuauthor=J.+Duanauthor=J.+Zhang&title=Crystal+structure+of+SARS-CoV-2+main+protease+in+complex+with+the+natural+product+inhibitor+shikonin+illuminates+a+unique+binding+mode&doi=10.1016%2Fj.scib.2020.10.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of SARS-CoV-2 main protease in complex with the natural product inhibitor shikonin illuminates a unique binding mode</span></div><div class="casAuthors">Li Jian; Jiang Feng; Zhou Xuelan; Zhang Jin; Zhang Yan; Luo Jun; Zhong Fanglin; Zhong Fanglin; Lin Cheng; McCormick Peter J; Zhou Huan; Wang Qisheng; Fu Yang; Duan Jingjing</div><div class="citationInfo"><span class="NLM_cas:title">Science bulletin</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">661-663</span>
        ISSN:<span class="NLM_cas:issn">2095-9273</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSWIZtDY9yqhtTiMd2aNQVUfW6udTcc2eZO1i0h4DMfrrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3s3itFSisQ%253D%253D&md5=48cdb7f3617a95c92e6cb6a9381f24aa</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.scib.2020.10.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.scib.2020.10.018%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DZhong%26aufirst%3DF.%26aulast%3DLin%26aufirst%3DC.%26aulast%3DMcCormick%26aufirst%3DP.%2BJ.%26aulast%3DJiang%26aufirst%3DF.%26aulast%3DLuo%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DFu%26aufirst%3DY.%26aulast%3DDuan%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DCrystal%2520structure%2520of%2520SARS-CoV-2%2520main%2520protease%2520in%2520complex%2520with%2520the%2520natural%2520product%2520inhibitor%2520shikonin%2520illuminates%2520a%2520unique%2520binding%2520mode%26jtitle%3DSci.%2520Bull.%2520%2528Beijing%2529%26date%3D2021%26volume%3D66%26spage%3D661%26epage%3D663%26doi%3D10.1016%2Fj.scib.2020.10.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ma, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Townsend, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lagarias, P. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marty, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolocouris, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span> <span> </span><span class="NLM_article-title">Ebselen, disulfiram, carmofur, PX-12, tideglusib, and shikonin are nonspecific promiscuous SARS-CoV-2 main protease inhibitors</span>. <i>ACS Pharmacol. Transl. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">1265</span>– <span class="NLM_lpage">1277</span>, <span class="refDoi"> DOI: 10.1021/acsptsci.0c00130</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsptsci.0c00130" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=1%3ACAS%3A528%3ADC%252BB3cXitVajsrjK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2020&pages=1265-1277&author=C.+Maauthor=Y.+Huauthor=J.+A.+Townsendauthor=P.+I.+Lagariasauthor=M.+T.+Martyauthor=A.+Kolocourisauthor=J.+Wang&title=Ebselen%2C+disulfiram%2C+carmofur%2C+PX-12%2C+tideglusib%2C+and+shikonin+are+nonspecific+promiscuous+SARS-CoV-2+main+protease+inhibitors&doi=10.1021%2Facsptsci.0c00130"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Ebselen, Disulfiram, Carmofur, PX-12, Tideglusib, and Shikonin Are Nonspecific Promiscuous SARS-CoV-2 Main Protease Inhibitors</span></div><div class="casAuthors">Ma, Chunlong; Hu, Yanmei; Townsend, Julia Alma; Lagarias, Panagiotis I.; Marty, Michael Thomas; Kolocouris, Antonios; Wang, Jun</div><div class="citationInfo"><span class="NLM_cas:title">ACS Pharmacology & Translational Science</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1265-1277</span>CODEN:
                <span class="NLM_cas:coden">APTSFN</span>;
        ISSN:<span class="NLM_cas:issn">2575-9108</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Among the drug targets being investigated for SARS-CoV-2, the viral main protease (Mpro) is one of the most extensively studied.  Mpro is a cysteine protease that hydrolyzes the viral polyprotein at more than 11 sites.  It is highly conserved and has a unique substrate preference for glutamine in the P1 position.  Therefore, Mpro inhibitors are expected to have broad-spectrum antiviral activity and a high selectivity index.  Structurally diverse compds. have been reported as Mpro inhibitors.  In this study, we investigated the mechanism of action of six previously reported Mpro inhibitors, ebselen, disulfiram, tideglusib, carmofur, shikonin, and PX-12, using a consortium of techniques including FRET-based enzymic assay, thermal shift assay, native mass spectrometry, cellular antiviral assays, and mol. dynamics simulations.  Collectively, the results showed that the inhibition of Mpro by these six compds. is nonspecific and that the inhibition is abolished or greatly reduced with the addn. of reducing reagent 1,4-dithiothreitol (DTT).  Without DTT, these six compds. inhibit not only Mpro but also a panel of viral cysteine proteases including SARS-CoV-2 papain-like protease and 2Apro and 3Cpro from enterovirus A71 (EV-A71) and EV-D68.  However, none of the compds. inhibits the viral replication of EV-A71 or EV-D68, suggesting that the enzymic inhibition potency IC50 values obtained in the absence of DTT cannot be used to faithfully predict their cellular antiviral activity.  Overall, we provide compelling evidence suggesting that these six compds. are nonspecific SARS-CoV-2 Mpro inhibitors and urge the scientific community to be stringent with hit validation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFcJU-O-vjurVg90H21EOLACvtfcHk0lhvFAH-JgG7eQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXitVajsrjK&md5=29d0e69db20f681ed4ef7cddd153cc6d</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Facsptsci.0c00130&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsptsci.0c00130%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DC.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DTownsend%26aufirst%3DJ.%2BA.%26aulast%3DLagarias%26aufirst%3DP.%2BI.%26aulast%3DMarty%26aufirst%3DM.%2BT.%26aulast%3DKolocouris%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DJ.%26atitle%3DEbselen%252C%2520disulfiram%252C%2520carmofur%252C%2520PX-12%252C%2520tideglusib%252C%2520and%2520shikonin%2520are%2520nonspecific%2520promiscuous%2520SARS-CoV-2%2520main%2520protease%2520inhibitors%26jtitle%3DACS%2520Pharmacol.%2520Transl.%2520Sci.%26date%3D2020%26volume%3D3%26spage%3D1265%26epage%3D1277%26doi%3D10.1021%2Facsptsci.0c00130" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gurard-Levin, Z. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jekle, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaisinghani, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vandyck, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jochmans, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leyssen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neyts, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blatt, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beigelman, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Symons, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raboisson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scholle, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deval, J.</span></span> <span> </span><span class="NLM_article-title">Evaluation of SARS-CoV-2 3C-like protease inhibitors using self-assembled monolayer desorption ionization mass spectrometry</span>. <i>Antiviral Res.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>182</i></span>,  <span class="NLM_fpage">104924</span>, <span class="refDoi"> DOI: 10.1016/j.antiviral.2020.104924</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=10.1016%2Fj.antiviral.2020.104924" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=32896566" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhvVSksrvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=182&publication_year=2020&pages=104924&author=Z.+A.+Gurard-Levinauthor=C.+Liuauthor=A.+Jekleauthor=R.+Jaisinghaniauthor=S.+Renauthor=K.+Vandyckauthor=D.+Jochmansauthor=P.+Leyssenauthor=J.+Neytsauthor=L.+M.+Blattauthor=L.+Beigelmanauthor=J.+A.+Symonsauthor=P.+Raboissonauthor=M.+D.+Scholleauthor=J.+Deval&title=Evaluation+of+SARS-CoV-2+3C-like+protease+inhibitors+using+self-assembled+monolayer+desorption+ionization+mass+spectrometry&doi=10.1016%2Fj.antiviral.2020.104924"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Evaluation of SARS-CoV-2 3C-like protease inhibitors using self-assembled monolayer desorption ionization mass spectrometry</span></div><div class="casAuthors">Gurard-Levin, Zachary A.; Liu, Cheng; Jekle, Andreas; Jaisinghani, Ruchika; Ren, Suping; Vandyck, Koen; Jochmans, Dirk; Leyssen, Pieter; Neyts, Johan; Blatt, Lawrence M.; Beigelman, Leonid; Symons, Julian A.; Raboisson, Pierre; Scholle, Michael D.; Deval, Jerome</div><div class="citationInfo"><span class="NLM_cas:title">Antiviral Research</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">182</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">104924</span>CODEN:
                <span class="NLM_cas:coden">ARSRDR</span>;
        ISSN:<span class="NLM_cas:issn">0166-3542</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is the cause of the COVID-19 pandemic that began in 2019.  The coronavirus 3-chymotrypsin-like cysteine protease (3CLpro) controls replication and is therefore considered a major target for antiviral discovery.  This study describes the evaluation of SARS-CoV-2 3CLpro inhibitors in a novel self-assembled monolayer desorption ionization mass spectrometry (SAMDI-MS) enzymic assay.  Compared with a traditional FRET readout, the label-free SAMDI-MS assay offers greater sensitivity and eliminates false pos. inhibition from compd. interference with the optical signal.  The SAMDI-MS assay was optimized and validated with known inhibitors of coronavirus 3CLpro such as GC376 (IC50 = 0.060μM), calpain inhibitors II and XII (IC50 ∼20-25μM).  The FDA-approved drugs shikonin, disulfiram, and ebselen did not inhibit SARS-CoV-2 3CLpro activity in the SAMDI-MS assay under physiol. relevant reducing conditions.  The three drugs did not directly inhibit human β-coronavirus OC-43 or SARS-CoV-2 in vitro, but instead induced cell death.  In conclusion, the SAMDI-MS 3CLpro assay, combined with antiviral and cytotoxic assessment, provides a robust platform to evaluate antiviral agents directed against SARS-CoV-2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-LzIy-MQKmbVg90H21EOLACvtfcHk0lgkT-Qj2LKFew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhvVSksrvP&md5=94693043624b8fdd562165834d746267</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.antiviral.2020.104924&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.antiviral.2020.104924%26sid%3Dliteratum%253Aachs%26aulast%3DGurard-Levin%26aufirst%3DZ.%2BA.%26aulast%3DLiu%26aufirst%3DC.%26aulast%3DJekle%26aufirst%3DA.%26aulast%3DJaisinghani%26aufirst%3DR.%26aulast%3DRen%26aufirst%3DS.%26aulast%3DVandyck%26aufirst%3DK.%26aulast%3DJochmans%26aufirst%3DD.%26aulast%3DLeyssen%26aufirst%3DP.%26aulast%3DNeyts%26aufirst%3DJ.%26aulast%3DBlatt%26aufirst%3DL.%2BM.%26aulast%3DBeigelman%26aufirst%3DL.%26aulast%3DSymons%26aufirst%3DJ.%2BA.%26aulast%3DRaboisson%26aufirst%3DP.%26aulast%3DScholle%26aufirst%3DM.%2BD.%26aulast%3DDeval%26aufirst%3DJ.%26atitle%3DEvaluation%2520of%2520SARS-CoV-2%25203C-like%2520protease%2520inhibitors%2520using%2520self-assembled%2520monolayer%2520desorption%2520ionization%2520mass%2520spectrometry%26jtitle%3DAntiviral%2520Res.%26date%3D2020%26volume%3D182%26spage%3D104924%26doi%3D10.1016%2Fj.antiviral.2020.104924" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Musharrafieh, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diesing, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marty, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span> <span> </span><span class="NLM_article-title">Validating enterovirus D68-2A(pro) as an antiviral drug target and the discovery of telaprevir as a potent D68–2A(pro) inhibitor</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>93</i></span>,  <span class="NLM_fpage">e02221</span>– <span class="NLM_lpage">18</span>, <span class="refDoi"> DOI: 10.1128/JVI.02221-18</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=10.1128%2FJVI.02221-18" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=30674624" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=1%3ACAS%3A528%3ADC%252BC1MXpvFSkur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=2019&pages=e02221-18&author=R.+Musharrafiehauthor=C.+Maauthor=J.+Zhangauthor=Y.+Huauthor=J.+M.+Diesingauthor=M.+T.+Martyauthor=J.+Wang&title=Validating+enterovirus+D68-2A%28pro%29+as+an+antiviral+drug+target+and+the+discovery+of+telaprevir+as+a+potent+D68%E2%80%932A%28pro%29+inhibitor&doi=10.1128%2FJVI.02221-18"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Validating enterovirus D68-2Apro as an antiviral drug target and the discovery of telaprevir as a potent D68-2Apro inhibitor</span></div><div class="casAuthors">Musharrafieh, Rami; Ma, Chunlong; Zhang, Jiantao; Hu, Yanmei; Diesing, Jessica M.; Marty, Michael T.; Wang, Jun</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">e02221-18/1-e02221-18/16</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">1098-5514</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Enterovirus D68 (EV-D68) is a viral pathogen that leads to severe respiratory illness and has been linked with the development of acute flaccid myelitis (AFM) in children.  No vaccines or antivirals are currently available for EV-D68 infection, and treatment options for hospitalized patients are limited to supportive care.  Here, we report the expression of the EV-D68 2A protease (2Apro) and characterization of its enzymic activity.  Furthermore, we discovered that telaprevir, an FDA-approved drug used for the treatment of hepatitis C virus (HCV) infections, is a potent antiviral against EV-D68 by targeting the 2Apro enzyme.  Using a fluorescence resonance energy transfer-based substrate cleavage assay, we showed that the purified EV-D68 2Apro has proteolytic activity selective against a peptide sequence corresponding to the viral VP1-2A polyprotein junction.  Telaprevir inhibits EV-D68 2Apro through a nearly irreversible, biphasic binding mechanism.  In cell culture, telaprevir showed submicromolar-to-low-micromolar potency against several recently circulating neurotropic strains of EV-D68 in different human cell lines.  To further confirm the antiviral drug target, serial viral passage expts. were performed to select for resistance against telaprevir.  An N84T mutation near the active site of 2Apro was identified in resistant viruses, and this mutation reduced the potency of telaprevir in both the enzymic and cellular antiviral assays.  Collectively, we report for the first time the in vitro enzymic activity of EV-D68 2Apro and the identification of telaprevir as a potent EV-D68 2Apro inhibitor.  These findings implicate EV-D68 2Apro as an antiviral drug target and highlight the repurposing potential of telaprevir to treat EV-D68 infection.  IMPORTANCE A 2014 EV-D68 outbreak in the United States has been linked to the development of acute flaccid myelitis in children.  Unfortunately, no treatment options against EV-D68 are currently available, and the development of effective therapeutics is urgently needed.  Here, we characterize and validate a new EV-D68 drug target, the 2Apro, and identify telaprevir-an FDA-approved drug used to treat hepatitis C virus (HCV) infections-as a potent antiviral with a novel mechanism of action toward 2Apro. 2Apro functions as a viral protease that cleaves a peptide sequence corresponding to the VP1-2A polyprotein junction.  The binding of telaprevir potently inhibits its enzymic activity, and using drug resistance selection, we show that the potent antiviral activity of telaprevir was due to 2Apro inhibition.  This is the first inhibitor to selectively target the 2Apro from EV-D68 and can be used as a starting point for the development of therapeutics with selective activity against EV-D68.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdA_RBOmKB6bVg90H21EOLACvtfcHk0lgkT-Qj2LKFew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXpvFSkur8%253D&md5=a016c137e0637bcbdc533d21ff69aa2d</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1128%2FJVI.02221-18&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.02221-18%26sid%3Dliteratum%253Aachs%26aulast%3DMusharrafieh%26aufirst%3DR.%26aulast%3DMa%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DDiesing%26aufirst%3DJ.%2BM.%26aulast%3DMarty%26aufirst%3DM.%2BT.%26aulast%3DWang%26aufirst%3DJ.%26atitle%3DValidating%2520enterovirus%2520D68-2A%2528pro%2529%2520as%2520an%2520antiviral%2520drug%2520target%2520and%2520the%2520discovery%2520of%2520telaprevir%2520as%2520a%2520potent%2520D68%25E2%2580%25932A%2528pro%2529%2520inhibitor%26jtitle%3DJ.%2520Virol.%26date%3D2019%26volume%3D93%26spage%3De02221%26epage%3D18%26doi%3D10.1128%2FJVI.02221-18" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cady, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeGrado, W. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, M.</span></span> <span> </span><span class="NLM_article-title">Specific binding of adamantane drugs and direction of their polar amines in the pore of the influenza M2 transmembrane domain in lipid bilayers and dodecylphosphocholine micelles determined by NMR spectroscopy</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>133</i></span>,  <span class="NLM_fpage">4274</span>– <span class="NLM_lpage">4284</span>, <span class="refDoi"> DOI: 10.1021/ja102581n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja102581n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=1%3ACAS%3A528%3ADC%252BC3MXisl2qtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2011&pages=4274-4284&author=S.+D.+Cadyauthor=J.+Wangauthor=Y.+Wuauthor=W.+F.+DeGradoauthor=M.+Hong&title=Specific+binding+of+adamantane+drugs+and+direction+of+their+polar+amines+in+the+pore+of+the+influenza+M2+transmembrane+domain+in+lipid+bilayers+and+dodecylphosphocholine+micelles+determined+by+NMR+spectroscopy&doi=10.1021%2Fja102581n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Specific Binding of Adamantane Drugs and Direction of Their Polar Amines in the Pore of the Influenza M2 Transmembrane Domain in Lipid Bilayers and Dodecylphosphocholine Micelles Determined by NMR Spectroscopy</span></div><div class="casAuthors">Cady, Sarah D.; Wang, Jun; Wu, Yibing; DeGrado, William F.; Hong, Mei</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">133</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4274-4284</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The transmembrane domain of the influenza M2 protein (M2TM) forms a tetrameric proton channel important for the virus life-cycle.  The proton-channel activity is inhibited by amine-contg. adamantyl drugs amantadine and rimantadine, which have been shown to bind specifically to the pore of M2TM near Ser31.  However, whether the polar amine points to the N- or C-terminus of the channel has not yet been detd.  Elucidating the polar group direction will shed light on the mechanism by which drug binding inhibits this proton channel and will facilitate rational design of new inhibitors.  In this study, we det. the polar amine direction using M2TM reconstituted in lipid bilayers as well as dodecylphosphocholine (DPC) micelles. 13C-2H rotational-echo double-resonance NMR expts. of 13C-labeled M2TM and methyl-deuterated rimantadine in lipid bilayers showed that the polar amine pointed to the C-terminus of the channel, with the Me group close to Gly34.  Soln. NMR expts. of M2TM in DPC micelles indicate that drug binding causes significant chem. shift perturbations of the protein that are very similar to those seen for M2TM and M2(18-60) bound to lipid bilayers.  Specific 2H-labeling of the drugs permitted the assignment of drug-protein cross peaks, which indicate that amantadine and rimantadine bind to the pore in the same fashion as for bilayer-bound M2TM.  These results strongly suggest that adamantyl inhibition of M2TM is achieved not only by direct phys. occlusion of the channel, but also by perturbing the equil. const. of the proton-sensing residue His37.  The reprodn. of the pharmacol. relevant specific pore-binding site in DPC micelles, which was not obsd. with a different detergent, DHPC, underscores the significant influence of the detergent environment on the functional structure of this membrane protein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHrFx8uRujFLVg90H21EOLACvtfcHk0lgkT-Qj2LKFew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXisl2qtbs%253D&md5=d2e9d577835a89045092bc2c0b87a24b</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fja102581n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja102581n%26sid%3Dliteratum%253Aachs%26aulast%3DCady%26aufirst%3DS.%2BD.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DDeGrado%26aufirst%3DW.%2BF.%26aulast%3DHong%26aufirst%3DM.%26atitle%3DSpecific%2520binding%2520of%2520adamantane%2520drugs%2520and%2520direction%2520of%2520their%2520polar%2520amines%2520in%2520the%2520pore%2520of%2520the%2520influenza%2520M2%2520transmembrane%2520domain%2520in%2520lipid%2520bilayers%2520and%2520dodecylphosphocholine%2520micelles%2520determined%2520by%2520NMR%2520spectroscopy%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2011%26volume%3D133%26spage%3D4274%26epage%3D4284%26doi%3D10.1021%2Fja102581n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marty, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baldwin, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marklund, E. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hochberg, G. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benesch, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, C. V.</span></span> <span> </span><span class="NLM_article-title">Bayesian deconvolution of mass and ion mobility spectra: from binary interactions to polydisperse ensembles</span>. <i>Anal. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>87</i></span>,  <span class="NLM_fpage">4370</span>– <span class="NLM_lpage">4376</span>, <span class="refDoi"> DOI: 10.1021/acs.analchem.5b00140</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.analchem.5b00140" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=1%3ACAS%3A528%3ADC%252BC2MXkvF2jtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=87&publication_year=2015&pages=4370-4376&author=M.+T.+Martyauthor=A.+J.+Baldwinauthor=E.+G.+Marklundauthor=G.+K.+Hochbergauthor=J.+L.+Beneschauthor=C.+V.+Robinson&title=Bayesian+deconvolution+of+mass+and+ion+mobility+spectra%3A+from+binary+interactions+to+polydisperse+ensembles&doi=10.1021%2Facs.analchem.5b00140"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Bayesian Deconvolution of Mass and Ion Mobility Spectra: From Binary Interactions to Polydisperse Ensembles</span></div><div class="casAuthors">Marty, Michael T.; Baldwin, Andrew J.; Marklund, Erik G.; Hochberg, Georg K. A.; Benesch, Justin L. P.; Robinson, Carol V.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Chemistry (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">87</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">4370-4376</span>CODEN:
                <span class="NLM_cas:coden">ANCHAM</span>;
        ISSN:<span class="NLM_cas:issn">0003-2700</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Interpretation of mass spectra is challenging because they report a ratio of 2 phys. quantities, mass and charge, which may each have multiple components that overlap in m/z.  Previous approaches to disentangling the 2 have focused on peak assignment or fitting.  However, the former struggle with complex spectra, and the latter are generally computationally intensive and may require substantial manual intervention.  The authors propose a new data anal. approach that employs a Bayesian framework to sep. the mass and charge dimensions.  From this approach, the authors developed UniDec (Universal Deconvolution), software that provides a rapid, robust, and flexible deconvolution of mass spectra and ion mobility-mass spectra with minimal user intervention.  Incorporation of the charge-state distribution in the Bayesian prior probabilities provides sepn. of the m/z spectrum into its phys. mass and charge components.  The authors have evaluated approach using systems of increasing complexity, enabling one to deduce lipid binding to membrane proteins, to probe the dynamics of subunit exchange reactions, and to characterize polydispersity in both protein assemblies and lipoprotein Nanodiscs.  The general utility of approach will greatly facilitate anal. of ion mobility and mass spectra.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEW75ktwO86rVg90H21EOLACvtfcHk0lg4DkWzJ2AmUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXkvF2jtLc%253D&md5=d2d1272237752ba52cdf2a976608f6bf</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Facs.analchem.5b00140&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.analchem.5b00140%26sid%3Dliteratum%253Aachs%26aulast%3DMarty%26aufirst%3DM.%2BT.%26aulast%3DBaldwin%26aufirst%3DA.%2BJ.%26aulast%3DMarklund%26aufirst%3DE.%2BG.%26aulast%3DHochberg%26aufirst%3DG.%2BK.%26aulast%3DBenesch%26aufirst%3DJ.%2BL.%26aulast%3DRobinson%26aufirst%3DC.%2BV.%26atitle%3DBayesian%2520deconvolution%2520of%2520mass%2520and%2520ion%2520mobility%2520spectra%253A%2520from%2520binary%2520interactions%2520to%2520polydisperse%2520ensembles%26jtitle%3DAnal.%2520Chem.%26date%3D2015%26volume%3D87%26spage%3D4370%26epage%3D4376%26doi%3D10.1021%2Facs.analchem.5b00140" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span> <span> </span><span class="NLM_article-title">The in vitro antiviral activity of lactoferrin against common human coronaviruses and SARS-CoV-2 is mediated by targeting the heparan sulfate co-receptor</span>. <i>Emerging Microbes Infect.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">317</span>– <span class="NLM_lpage">330</span>, <span class="refDoi"> DOI: 10.1080/22221751.2021.1888660</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=10.1080%2F22221751.2021.1888660" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=33560940" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=1%3ACAS%3A528%3ADC%252BB3MXltlegtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2021&pages=317-330&author=Y.+Huauthor=X.+Mengauthor=F.+Zhangauthor=Y.+Xiangauthor=J.+Wang&title=The+in+vitro+antiviral+activity+of+lactoferrin+against+common+human+coronaviruses+and+SARS-CoV-2+is+mediated+by+targeting+the+heparan+sulfate+co-receptor&doi=10.1080%2F22221751.2021.1888660"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">The in vitro antiviral activity of lactoferrin against common human coronaviruses and SARS-CoV-2 is mediated by targeting the heparan sulfate co-receptor</span></div><div class="casAuthors">Hu, Yanmei; Meng, Xiangzhi; Zhang, Fushun; Xiang, Yan; Wang, Jun</div><div class="citationInfo"><span class="NLM_cas:title">Emerging Microbes & Infections</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">317-330</span>CODEN:
                <span class="NLM_cas:coden">EMIMC4</span>;
        ISSN:<span class="NLM_cas:issn">2222-1751</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">Coronavirus disease 2019 (COVID-19) is an ongoing pandemic that lacks effective therapeutic interventions.  SARS-CoV-2 infects ACE2-expressing cells and gains cell entry through either direct plasma membrane fusion or endocytosis.  Recent studies have shown that in addn. to ACE2, heparan sulfate proteoglycans (HSPGs) also play an important role in SARS-CoV-2 cell attachment by serving as an attachment factor.  Binding of viral spike protein to HSPGs leads to the enrichment of local concn. for the subsequent specific binding with ACE2.  We therefore hypothesize that blocking the interactions between viral spike protein and the HSPGs will lead to inhibition of viral replication.  In this study, we report our findings of the broad-spectrum antiviral activity and the mechanism of action of lactoferrin (LF) against multiple common human coronaviruses as well as SARS-CoV-2.  Our study has shown that LF has broad-spectrum antiviral activity against SARS-CoV-2, HCoV-OC43, HCoV-NL63, and HCoV-229E in cell culture, and bovine lactoferrin (BLF) is more potent than human lactoferrin.  Mechanistic studies revealed that BLF binds to HSPGs, thereby blocking viral attachment to the host cell.  The antiviral activity of BLF can be antagonized by the HSPG mimetic heparin.  Combination therapy expt. showed that the antiviral activity of LF is synergistic with remdesivir in cell culture.  Mol. modeling suggests that the N-terminal pos. charged region in BLF (residues 17-41) confers the binding to HSPGs.  Overall, LF appears to be a promising drug candidate for COVID-19 that warrants further investigation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-KJMYvpJE-LVg90H21EOLACvtfcHk0lg4DkWzJ2AmUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXltlegtrg%253D&md5=a5c5c402b46462fe9a0cc836f5b90d9d</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1080%2F22221751.2021.1888660&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F22221751.2021.1888660%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DY.%26aulast%3DMeng%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DF.%26aulast%3DXiang%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DJ.%26atitle%3DThe%2520in%2520vitro%2520antiviral%2520activity%2520of%2520lactoferrin%2520against%2520common%2520human%2520coronaviruses%2520and%2520SARS-CoV-2%2520is%2520mediated%2520by%2520targeting%2520the%2520heparan%2520sulfate%2520co-receptor%26jtitle%3DEmerging%2520Microbes%2520Infect.%26date%3D2021%26volume%3D10%26spage%3D317%26epage%3D330%26doi%3D10.1080%2F22221751.2021.1888660" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Repetto, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">del Peso, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zurita, J. L.</span></span> <span> </span><span class="NLM_article-title">Neutral red uptake assay for the estimation of cell viability/cytotoxicity</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">1125</span>– <span class="NLM_lpage">1131</span>, <span class="refDoi"> DOI: 10.1038/nprot.2008.75</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=10.1038%2Fnprot.2008.75" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=18600217" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=1%3ACAS%3A528%3ADC%252BD1cXotFakurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2008&pages=1125-1131&author=G.+Repettoauthor=A.+del+Pesoauthor=J.+L.+Zurita&title=Neutral+red+uptake+assay+for+the+estimation+of+cell+viability%2Fcytotoxicity&doi=10.1038%2Fnprot.2008.75"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Neutral red uptake assay for the estimation of cell viability/cytotoxicity</span></div><div class="casAuthors">Repetto, Guillermo; del Peso, Ana; Zurita, Jorge L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1125-1131</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The neutral red uptake assay provides a quant. estn. of the no. of viable cells in a culture.  It is one of the most used cytotoxicity tests with many biomedical and environmental applications.  It is based on the ability of viable cells to incorporate and bind the supravital dye neutral red in the lysosomes.  Most primary cells and cell lines from diverse origin may be successfully used.  Cells are seeded in 96-well tissue culture plates and are treated for the appropriate period.  The plates are then incubated for 2 h with a medium contg. neutral red.  The cells are subsequently washed, the dye is extd. in each well and the absorbance is read using a spectrophotometer.  The procedure is cheaper and more sensitive than other cytotoxicity tests (tetrazolium salts, enzyme leakage or protein content).  Once the cells were treated, the assay can be completed in <3 h.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqCcpd7QjH04rVg90H21EOLACvtfcHk0lg4DkWzJ2AmUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXotFakurk%253D&md5=30461f51c2ad668a3cadb4843c93b43e</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2008.75&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2008.75%26sid%3Dliteratum%253Aachs%26aulast%3DRepetto%26aufirst%3DG.%26aulast%3Ddel%2BPeso%26aufirst%3DA.%26aulast%3DZurita%26aufirst%3DJ.%2BL.%26atitle%3DNeutral%2520red%2520uptake%2520assay%2520for%2520the%2520estimation%2520of%2520cell%2520viability%252Fcytotoxicity%26jtitle%3DNat.%2520Protoc.%26date%3D2008%26volume%3D3%26spage%3D1125%26epage%3D1131%26doi%3D10.1038%2Fnprot.2008.75" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ma, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span> <span> </span><span class="NLM_article-title">Pharmacological characterization of the spectrum of antiviral activity and genetic barrier to drug resistance of M2-S31N channel blockers</span>. <i>Mol. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>90</i></span>,  <span class="NLM_fpage">188</span>– <span class="NLM_lpage">98</span>, <span class="refDoi"> DOI: 10.1124/mol.116.105346</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=10.1124%2Fmol.116.105346" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=27385729" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVektr%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=90&publication_year=2016&pages=188-98&author=C.+Maauthor=J.+Zhangauthor=J.+Wang&title=Pharmacological+characterization+of+the+spectrum+of+antiviral+activity+and+genetic+barrier+to+drug+resistance+of+M2-S31N+channel+blockers&doi=10.1124%2Fmol.116.105346"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacological characterization of the spectrum of antiviral activity and genetic barrier to drug resistance of M2-S31N channel blockers</span></div><div class="casAuthors">Ma, Chunlong; Zhang, Jiantao; Wang, Jun</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">90</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">188-198</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">1521-0111</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Adamantanes (amantadine and rimantadine) are one of the two classes of Food and Drug Administration-approved antiviral drugs used for the prevention and treatment of influenza A virus infections.  They inhibit viral replication by blocking the wild-type (WT) M2 proton channel, thus preventing viral uncoating.  However, their use was discontinued due to widespread drug resistance.  Among a handful of drug-resistant mutants, M2-S31N is the predominant mutation and persists in more than 95% of currently circulating influenza A strains.  We recently designed two classes of M2-S31N inhibitors, S31N-specific inhibitors and S31N/WT dual inhibitors, which are represented by N-[(5- cyclopropyl-1,2-oxazol-3-yl)methyl]adamantan-1-amine (WJ379) and N-[(5-bromothiophen-2-yl)methyl]adamantan-1-amine (BC035), resp.  However, their antiviral activities against currently circulating influenza A viruses and their genetic barrier to drug resistance are unknown.  In this report, we evaluated the therapeutic potential of these two classes of M2-S31N inhibitors (WJ379 and BC035) by profiling their antiviral efficacy against multidrug-resistant influenza A viruses, in vitro drug resistance barrier, and synergistic effect with oseltamivir.  We found that M2- S31N inhibitors were active against several influenza A viruses that are resistant to one or both classes of Food and Drug Administration-approved anti-influenza drugs.  In addn., M2- S31N inhibitors display a higher in vitro genetic barrier to drug resistance than amantadine.  The antiviral effect of WJ379 was also synergistic with oseltamivir carboxylate.  Overall, these results reaffirm that M2-S31N inhibitors are promising antiviral drug candidates that warrant further development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4eDLHN9ABdLVg90H21EOLACvtfcHk0lg4DkWzJ2AmUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVektr%252FF&md5=bef9d1512339524ce625cda89aa6feb7</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1124%2Fmol.116.105346&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.116.105346%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DJ.%26atitle%3DPharmacological%2520characterization%2520of%2520the%2520spectrum%2520of%2520antiviral%2520activity%2520and%2520genetic%2520barrier%2520to%2520drug%2520resistance%2520of%2520M2-S31N%2520channel%2520blockers%26jtitle%3DMol.%2520Pharmacol.%26date%3D2016%26volume%3D90%26spage%3D188%26epage%3D98%26doi%3D10.1124%2Fmol.116.105346" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teplyakov, A.</span></span> <span> </span><span class="NLM_article-title">MOLREP: an automated program for molecular replacement</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">1022</span>– <span class="NLM_lpage">1025</span>, <span class="refDoi"> DOI: 10.1107/S0021889897006766</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=10.1107%2FS0021889897006766" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=1%3ACAS%3A528%3ADyaK1cXhslehu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=1997&pages=1022-1025&author=A.+Vaginauthor=A.+Teplyakov&title=MOLREP%3A+an+automated+program+for+molecular+replacement&doi=10.1107%2FS0021889897006766"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">MOLREP: an automated program for molecular replacement</span></div><div class="casAuthors">Vagin, Alexei; Teplyakov, Alexei</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1022-1025</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard International Publishers Ltd.</span>)
        </div><div class="casAbstract">MOLREP is an automated program for mol. replacement which uses effective new approaches in data processing and rotational and translational searching.  These include an automatic choice of all parameters, scaling by Patterson origin peaks and soft resoln. cutoff.  One of the cornerstones of the program is an original full-symmetry translation function combined with a packing function.  Information from the model already placed in the cell is incorporated in both translation and packing functions.  A no. of tests using exptl. data proved the ability of the program to find the correct soln. in difficult cases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkGXdZ-nTKxLVg90H21EOLACvtfcHk0lg4UYMAXmHLPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXhslehu7Y%253D&md5=96af64aedd8a8f018c3c18b57a6b4b21</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1107%2FS0021889897006766&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889897006766%26sid%3Dliteratum%253Aachs%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DTeplyakov%26aufirst%3DA.%26atitle%3DMOLREP%253A%2520an%2520automated%2520program%2520for%2520molecular%2520replacement%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D1997%26volume%3D30%26spage%3D1022%26epage%3D1025%26doi%3D10.1107%2FS0021889897006766" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skubak, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lebedev, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannu, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steiner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicholls, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A. A.</span></span> <span> </span><span class="NLM_article-title">REFMAC5 for the refinement of macromolecular crystal structures</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">355</span>– <span class="NLM_lpage">367</span>, <span class="refDoi"> DOI: 10.1107/S0907444911001314</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=10.1107%2FS0907444911001314" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=21460454" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=355-367&author=G.+N.+Murshudovauthor=P.+Skubakauthor=A.+A.+Lebedevauthor=N.+S.+Pannuauthor=R.+A.+Steinerauthor=R.+A.+Nichollsauthor=M.+D.+Winnauthor=F.+Longauthor=A.+A.+Vagin&title=REFMAC5+for+the+refinement+of+macromolecular+crystal+structures&doi=10.1107%2FS0907444911001314"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">REFMAC5 for the refinement of macromolecular crystal structures</span></div><div class="casAuthors">Murshudov, Garib N.; Skubak, Pavol; Lebedev, Andrey A.; Pannu, Navraj S.; Steiner, Roberto A.; Nicholls, Robert A.; Winn, Martyn D.; Long, Fei; Vagin, Alexei A.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">355-367</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">This paper describes various components of the macromol. crystallog. refinement program REFMAC5, which is distributed as part of the CCP4 suite.  REFMAC5 utilizes different likelihood functions depending on the diffraction data employed (amplitudes or intensities), the presence of twinning and the availability of SAD/SIRAS exptl. diffraction data.  To ensure chem. and structural integrity of the refined model, REFMAC5 offers several classes of restraints and choices of model parameterization.  Reliable models at resolns. at least as low as 4 Å can be achieved thanks to low-resoln. refinement tools such as secondary-structure restraints, restraints to known homologous structures, automatic global and local NCS restraints, 'jelly-body' restraints and the use of novel long-range restraints on at. displacement parameters (ADPs) based on the Kullback-Leibler divergence.  REFMAC5 addnl. offers TLS parameterization and, when high-resoln. data are available, fast refinement of anisotropic ADPs.  Refinement in the presence of twinning is performed in a fully automated fashion.  REFMAC5 is a flexible and highly optimized refinement package that is ideally suited for refinement across the entire resoln. spectrum encountered in macromol. crystallog.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5Jq-i2qIq5rVg90H21EOLACvtfcHk0lg4UYMAXmHLPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqtbk%253D&md5=f8f3202d246908500057ad7c71015b7b</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1107%2FS0907444911001314&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444911001314%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DSkubak%26aufirst%3DP.%26aulast%3DLebedev%26aufirst%3DA.%2BA.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DSteiner%26aufirst%3DR.%2BA.%26aulast%3DNicholls%26aufirst%3DR.%2BA.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DLong%26aufirst%3DF.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26atitle%3DREFMAC5%2520for%2520the%2520refinement%2520of%2520macromolecular%2520crystal%2520structures%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D355%26epage%3D367%26doi%3D10.1107%2FS0907444911001314" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Coot: model-building tools for molecular graphics</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2126</span>– <span class="NLM_lpage">2132</span>, <span class="refDoi"> DOI: 10.1107/S0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+model-building+tools+for+molecular+graphics&doi=10.1107%2FS0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0lg4UYMAXmHLPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2FS0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6XFN" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6XFN','PDB','6XFN'); return false;">PDB: 6XFN</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3V3M" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3V3M','PDB','3V3M'); return false;">PDB: 3V3M</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2Q6G" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2Q6G','PDB','2Q6G'); return false;">PDB: 2Q6G</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6WTT" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6WTT','PDB','6WTT'); return false;">PDB: 6WTT</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6XBH" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6XBH','PDB','6XBH'); return false;">PDB: 6XBH</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7KX5" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7KX5','PDB','7KX5'); return false;">PDB: 7KX5</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-3"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i72"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00509">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_62477"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.1c00509?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00509</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Target selectivity of noncovalent inhibitor <b>23R</b>; crystallographic statistics; H NMR and C NMR spectra for final products; HPLC traces for representative compounds (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00509/suppl_file/jm1c00509_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00509/suppl_file/jm1c00509_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00509/suppl_file/jm1c00509_si_001.pdf">jm1c00509_si_001.pdf (4.14 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00509/suppl_file/jm1c00509_si_002.csv">jm1c00509_si_002.csv (4.46 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.1c00509&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.1c00509" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.1c00509" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                8MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-b425991822a79ada59b3.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-fb88834b824b41b135ad.js"></script>

<script type="text/javascript" src="/wro/ke1q~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"667980f34cfb24ad","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.6.0","si":10}'></script>
</body>
</html>
