
<html lang="en"     class="pb-page"  data-request-id="50ea3c90-b2ec-49c0-81fa-33b628ab8785"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;article:article:10.1021/jm500118j;issue:issue:10.1021/jmcmar.2014.57.issue-8;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Discovery of AMG 925, a FLT3 and CDK4 Dual Kinase Inhibitor with Preferential Affinity for the Activated State of FLT3" /></meta><meta name="dc.Creator" content="Zhihong  Li" /></meta><meta name="dc.Creator" content="Xianghong  Wang" /></meta><meta name="dc.Creator" content="John  Eksterowicz" /></meta><meta name="dc.Creator" content="Michael W.  Gribble, Jr." /></meta><meta name="dc.Creator" content="Grace Q.  Alba" /></meta><meta name="dc.Creator" content="Merrill  Ayres" /></meta><meta name="dc.Creator" content="Timothy J.  Carlson" /></meta><meta name="dc.Creator" content="Ada  Chen" /></meta><meta name="dc.Creator" content="Xiaoqi  Chen" /></meta><meta name="dc.Creator" content="Robert  Cho" /></meta><meta name="dc.Creator" content="Richard V.  Connors" /></meta><meta name="dc.Creator" content="Michael  DeGraffenreid" /></meta><meta name="dc.Creator" content="Jeffrey T.  Deignan" /></meta><meta name="dc.Creator" content="Jason  Duquette" /></meta><meta name="dc.Creator" content="Pingchen  Fan" /></meta><meta name="dc.Creator" content="Benjamin  Fisher" /></meta><meta name="dc.Creator" content="Jiasheng  Fu" /></meta><meta name="dc.Creator" content="Justin N.  Huard" /></meta><meta name="dc.Creator" content="Jacob  Kaizerman" /></meta><meta name="dc.Creator" content="Kathleen S.  Keegan" /></meta><meta name="dc.Creator" content="Cong  Li" /></meta><meta name="dc.Creator" content="Kexue  Li" /></meta><meta name="dc.Creator" content="Yunxiao  Li" /></meta><meta name="dc.Creator" content="Lingming  Liang" /></meta><meta name="dc.Creator" content="Wen  Liu" /></meta><meta name="dc.Creator" content="Sarah E.  Lively" /></meta><meta name="dc.Creator" content="Mei-Chu  Lo" /></meta><meta name="dc.Creator" content="Ji  Ma" /></meta><meta name="dc.Creator" content="Dustin L.  McMinn" /></meta><meta name="dc.Creator" content="Jeffrey T.  Mihalic" /></meta><meta name="dc.Creator" content="Kriti  Modi" /></meta><meta name="dc.Creator" content="Rachel  Ngo" /></meta><meta name="dc.Creator" content="Kanaka  Pattabiraman" /></meta><meta name="dc.Creator" content="Derek E.  Piper" /></meta><meta name="dc.Creator" content="Christophe  Queva" /></meta><meta name="dc.Creator" content="Mark L.  Ragains" /></meta><meta name="dc.Creator" content="Julia  Suchomel" /></meta><meta name="dc.Creator" content="Steve  Thibault" /></meta><meta name="dc.Creator" content="Nigel  Walker" /></meta><meta name="dc.Creator" content="Xiaodong  Wang" /></meta><meta name="dc.Creator" content="Zhulun  Wang" /></meta><meta name="dc.Creator" content="Malgorzata  Wanska" /></meta><meta name="dc.Creator" content="Paul M.  Wehn" /></meta><meta name="dc.Creator" content="Margaret F.  Weidner" /></meta><meta name="dc.Creator" content="Alex J.  Zhang" /></meta><meta name="dc.Creator" content="Xiaoning  Zhao" /></meta><meta name="dc.Creator" content="Alexander  Kamb" /></meta><meta name="dc.Creator" content="Dineli  Wickramasinghe" /></meta><meta name="dc.Creator" content="Kang  Dai" /></meta><meta name="dc.Creator" content="Lawrence R.  McGee" /></meta><meta name="dc.Creator" content="Julio C.  Medina" /></meta><meta name="dc.Description" content="We describe the structural optimization of a lead compound 1 that exhibits dual inhibitory activities against FLT3 and CDK4. A series of pyrido[4′,3′:4,5]pyrrolo[2,3-d]pyrimidine derivatives was sy..." /></meta><meta name="Description" content="We describe the structural optimization of a lead compound 1 that exhibits dual inhibitory activities against FLT3 and CDK4. A series of pyrido[4′,3′:4,5]pyrrolo[2,3-d]pyrimidine derivatives was sy..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="April 2, 2014" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/jm500118j" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2014 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Fjm500118j" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Fjm500118j" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Fjm500118j" /></link>
        
    
    

<title>Discovery of AMG 925, a FLT3 and CDK4 Dual Kinase Inhibitor with Preferential Affinity for the Activated State of FLT3 | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/jm500118j" /></meta><meta property="og:title" content="Discovery of AMG 925, a FLT3 and CDK4 Dual Kinase Inhibitor with Preferential Affinity for the Activated State of FLT3" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0028.jpeg" /></meta><meta property="og:description" content="We describe the structural optimization of a lead compound 1 that exhibits dual inhibitory activities against FLT3 and CDK4. A series of pyrido[4′,3′:4,5]pyrrolo[2,3-d]pyrimidine derivatives was synthesized, and SAR analysis, using cell-based assays, led to the discovery of 28 (AMG 925), a potent and orally bioavailable dual inhibitor of CDK4 and FLT3, including many FLT3 mutants reported to date. Compound 28 inhibits the proliferation of a panel of human tumor cell lines including Colo205 (Rb+) and U937 (FLT3WT) and induced cell death in MOLM13 (FLT3ITD) and even in MOLM13 (FLT3ITD, D835Y), which exhibits resistance to a number of FLT3 inhibitors currently under clinical development. At well-tolerated doses, compound 28 leads to significant growth inhibition of MOLM13 xenografts in nude mice, and the activity correlates with inhibition of STAT5 and Rb phosphorylation." /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta><meta name="twitter:description" content="We describe the structural optimization of a lead compound 1 that exhibits dual inhibitory activities against FLT3 and CDK4. A series of pyrido[4′,3′:4,5]pyrrolo[2,3-d]pyrimidine derivatives was synthesized, and SAR analysis, using cell-based assays, led to the discovery of 28 (AMG 925), a potent and orally bioavailable dual inhibitor of CDK4 and FLT3, including many FLT3 mutants reported to date. Compound 28 inhibits the proliferation of a panel of human tumor cell lines including Colo205 (Rb+) and U937 (FLT3WT) and induced cell death in MOLM13 (FLT3ITD) and even in MOLM13 (FLT3ITD, D835Y), which exhibits resistance to a number of FLT3 inhibitors currently under clinical development. At well-tolerated doses, compound 28 leads to significant growth inhibition of MOLM13 xenografts in nude mice, and the activity correlates with inhibition of STAT5 and Rb phosphorylation." /></meta><meta name="twitter:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0028.jpeg" /></meta><meta name="twitter:title" content="Discovery of AMG 925, a FLT3 and CDK4 Dual Kinase Inhibitor with Preferential Affinity for the Activated State of FLT3" /></meta>
        
            <meta property="og:type" content="Article" /></meta>
        

        
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/jm500118j"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/ke1q~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-6ee29527b3d8ff00a106.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/jm500118j">
            
        





































    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-12/achre4.2021.54.issue-12/20210615/achre4.2021.54.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-6/amrcda.2021.2.issue-6/20210625/amrcda.2021.2.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-6/aabmcb.2021.4.issue-6/20210621/aabmcb.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-6/aaembp.2021.3.issue-6/20210622/aaembp.2021.3.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-6/aaemcq.2021.4.issue-6/20210628/aaemcq.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-25/aamick.2021.13.issue-25/20210630/aamick.2021.13.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-6/aanmf6.2021.4.issue-6/20210625/aanmf6.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-6/aapmcd.2021.3.issue-6/20210611/aapmcd.2021.3.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-6/abseba.2021.7.issue-6/20210614/abseba.2021.7.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-12/accacs.2021.11.issue-12/20210618/accacs.2021.11.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-6/acscii.2021.7.issue-6/20210623/acscii.2021.7.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-6/acbcct.2021.16.issue-6/20210618/acbcct.2021.16.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-3/achsc5.2021.28.issue-3/20210524/achsc5.2021.28.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-12/acncdm.2021.12.issue-12/20210616/acncdm.2021.12.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-6/aesccq.2021.5.issue-6/20210617/aesccq.2021.5.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-6/aelccp.2021.6.issue-6/20210611/aelccp.2021.6.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-6/aeecco.2021.1.issue-6/20210611/aeecco.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-6/aewcaa.2021.1.issue-6/20210611/aewcaa.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-5/afsthl.2021.1.issue-5/20210618/afsthl.2021.1.issue-5.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-6/aidcbc.2021.7.issue-6/20210611/aidcbc.2021.7.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-6/amlccd.2021.10.issue-6/20210615/amlccd.2021.10.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-6/amlcef.2021.3.issue-6/20210607/amlcef.2021.3.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-6/amclct.2021.12.issue-6/20210610/amclct.2021.12.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-6/ancac3.2021.15.issue-6/20210622/ancac3.2021.15.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-25/acsodf.2021.6.issue-25/20210629/acsodf.2021.6.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-6/apchd5.2021.8.issue-6/20210616/apchd5.2021.8.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-6/ascefj.2021.6.issue-6/20210625/ascefj.2021.6.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-25/ascecg.2021.9.issue-25/20210628/ascecg.2021.9.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-6/asbcd6.2021.10.issue-6/20210618/asbcd6.2021.10.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-25/ancham.2021.93.issue-25/20210629/ancham.2021.93.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-25/bichaw.2021.60.issue-25/20210629/bichaw.2021.60.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-6/bcches.2021.32.issue-6/20210616/bcches.2021.32.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-6/bomaf6.2021.22.issue-6/20210614/bomaf6.2021.22.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-24/cgeabj.2021.99.issue-24/20210628-01/cgeabj.2021.99.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-6/crtoec.2021.34.issue-6/20210621/crtoec.2021.34.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-12/chreay.2021.121.issue-12/20210623/chreay.2021.121.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-12/cmatex.2021.33.issue-12/20210622/cmatex.2021.33.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-6/cgdefu.2021.21.issue-6/20210602/cgdefu.2021.21.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-12/enfuem.2021.35.issue-12/20210617/enfuem.2021.35.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-12/esthag.2021.55.issue-12/20210615/esthag.2021.55.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-6/estlcu.2021.8.issue-6/20210608/estlcu.2021.8.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-25/iecred.2021.60.issue-25/20210630/iecred.2021.60.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-12/inocaj.2021.60.issue-12/20210621/inocaj.2021.60.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-6/jaaucr.2021.1.issue-6/20210628/jaaucr.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-25/jafcau.2021.69.issue-25/20210630/jafcau.2021.69.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-6/jceaax.2021.66.issue-6/20210610/jceaax.2021.66.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-6/jceda8.2021.98.issue-6/20210608/jceda8.2021.98.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-6/jcisd8.2021.61.issue-6/20210628/jcisd8.2021.61.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-6/jctcce.2021.17.issue-6/20210608/jctcce.2021.17.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/jmcmar.2021.64.issue-12/20210624/jmcmar.2021.64.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-6/jnprdf.2021.84.issue-6/20210625/jnprdf.2021.84.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-12/joceah.2021.86.issue-12/20210618/joceah.2021.86.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-24/jpcafh.2021.125.issue-24/20210624/jpcafh.2021.125.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-24/jpcbfk.2021.125.issue-24/20210624/jpcbfk.2021.125.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-24/jpccck.2021.125.issue-24/20210624/jpccck.2021.125.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-24/jpclcd.2021.12.issue-24/20210624/jpclcd.2021.12.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-6/jprobs.2021.20.issue-6/20210604/jprobs.2021.20.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-25/jacsat.2021.143.issue-25/20210630/jacsat.2021.143.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-6/jamsef.2021.32.issue-6/20210602/jamsef.2021.32.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-25/langd5.2021.37.issue-25/20210629/langd5.2021.37.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-12/mamobx.2021.54.issue-12/20210622/mamobx.2021.54.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-6/mpohbp.2021.18.issue-6/20210607/mpohbp.2021.18.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-12/nalefd.2021.21.issue-12/20210623/nalefd.2021.21.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-12/orlef7.2021.23.issue-12/20210618/orlef7.2021.23.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-6/oprdfk.2021.25.issue-6/20210618/oprdfk.2021.25.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-12/orgnd7.2021.40.issue-12/20210628/orgnd7.2021.40.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Fjm500118j&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Fjm500118j&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/jm500118j&amp;href=/doi/10.1021/jm500118j" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2014</span><span class="cit-fg-volume">, 57</span><span class="cit-fg-issue">, 8</span><span class="cit-fg-pageRange">, 3430-3449</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/57/8" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/jm500117r" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/jm500126s" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery of AMG 925, a FLT3 and CDK4 Dual Kinase Inhibitor with Preferential Affinity for the Activated State of FLT3</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Zhihong++Li">Zhihong Li</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Xianghong++Wang">Xianghong Wang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=John++Eksterowicz">John Eksterowicz</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Michael+W.++Gribble%2C++Jr.">Michael W. Gribble, Jr.</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Grace+Q.++Alba">Grace Q. Alba</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Merrill++Ayres">Merrill Ayres</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Timothy+J.++Carlson">Timothy J. Carlson</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Ada++Chen">Ada Chen</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Xiaoqi++Chen">Xiaoqi Chen</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Robert++Cho">Robert Cho</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Richard+V.++Connors">Richard V. Connors</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Michael++DeGraffenreid">Michael DeGraffenreid</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jeffrey+T.++Deignan">Jeffrey T. Deignan</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jason++Duquette">Jason Duquette</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Pingchen++Fan">Pingchen Fan</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Benjamin++Fisher">Benjamin Fisher</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jiasheng++Fu">Jiasheng Fu</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Justin+N.++Huard">Justin N. Huard</a></span><span class="author-xref-symbol author-aff-symbol"><sup>⊥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jacob++Kaizerman">Jacob Kaizerman</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Kathleen+S.++Keegan">Kathleen S. Keegan</a></span><span class="author-xref-symbol author-aff-symbol"><sup>⊥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Cong++Li">Cong Li</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Kexue++Li">Kexue Li</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Yunxiao++Li">Yunxiao Li</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Lingming++Liang">Lingming Liang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Wen++Liu">Wen Liu</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Sarah+E.++Lively">Sarah E. Lively</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Mei-Chu++Lo">Mei-Chu Lo</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Ji++Ma">Ji Ma</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Dustin+L.++McMinn">Dustin L. McMinn</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jeffrey+T.++Mihalic">Jeffrey T. Mihalic</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Kriti++Modi">Kriti Modi</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Rachel++Ngo">Rachel Ngo</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Kanaka++Pattabiraman">Kanaka Pattabiraman</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Derek+E.++Piper">Derek E. Piper</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Christophe++Queva">Christophe Queva</a></span><span class="author-xref-symbol author-aff-symbol"><sup>⊥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Mark+L.++Ragains">Mark L. Ragains</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Julia++Suchomel">Julia Suchomel</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Steve++Thibault">Steve Thibault</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Nigel++Walker">Nigel Walker</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Xiaodong++Wang">Xiaodong Wang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Zhulun++Wang">Zhulun Wang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Malgorzata++Wanska">Malgorzata Wanska</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Paul+M.++Wehn">Paul M. Wehn</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Margaret+F.++Weidner">Margaret F. Weidner</a></span><span class="author-xref-symbol author-aff-symbol"><sup>⊥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Alex+J.++Zhang">Alex J. Zhang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Xiaoning++Zhao">Xiaoning Zhao</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Alexander++Kamb">Alexander Kamb</a></span><span class="author-xref-symbol author-aff-symbol"><sup>#</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Dineli++Wickramasinghe">Dineli Wickramasinghe</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Kang++Dai">Kang Dai</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Lawrence+R.++McGee">Lawrence R. McGee</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Julio+C.++Medina">Julio C. Medina</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-text">Departments of <sup>†</sup>Therapeutic Discovery, <sup>‡</sup>Oncology Research, <sup>§</sup>Pharmaceutics, and <sup>∥</sup>Pharmacokinetics and Drug Metabolism, Amgen Inc., 1120 Veterans Boulevard, South San Francisco, California 94080, United States</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">⊥</span> <span class="aff-text">Therapeutic Innovation Unit, Amgen Inc., 1201 Amgen Court West, Seattle, Washington 98119, United States</span></div><div class="aff-info" id="aff3"><span class="aff-symbol">#</span> <span class="aff-text">Discovery Research, Amgen Inc., One Amgen Center Drive, Thousand Oaks, Callifornia 91320, United States</span></div><div class="corresp-info"><strong>*</strong>Phone: 650-244-2501. E-mail: <a href="/cdn-cgi/l/email-protection#52283a3b3a3d3c353e12333f35373c7c313d3f"><span class="__cf_email__" data-cfemail="9de7f5f4f5f2f3faf1ddfcf0faf8f3b3fef2f0">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Fjm500118j&amp;href=/doi/10.1021%2Fjm500118j" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2014</span></span><span class="cit-volume">, 57</span><span class="cit-issue">, 8</span><span class="cit-pageRange">, 3430–3449</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">March 18, 2014</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>22 January 2014</li><li><span class="item_label"><b>Published</b> online</span>2 April 2014</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 24 April 2014</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/jm500118j" title="DOI URL">https://doi.org/10.1021/jm500118j</a></div><div class="article_header-article-copyright"><strong>Copyright © 2014 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D3430%26pageCount%3D20%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DZhihong%2BLi%252C%2BXianghong%2BWang%252C%2BJohn%2BEksterowicz%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D57%26issueNum%3D8%26contentID%3Djm500118j%26title%3DDiscovery%2Bof%2BAMG%2B925%252C%2Ba%2BFLT3%2Band%2BCDK4%2BDual%2BKinase%2BInhibitor%2Bwith%2BPreferential%2BAffinity%2Bfor%2Bthe%2BActivated%2BState%2Bof%2BFLT3%26numPages%3D20%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D3449%26publicationDate%3DApril%2B2014">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/jm500118j"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">4732</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">42</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/jm500118j" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery of AMG 925, a FLT3 and CDK4 Dual Kinase Inhibitor with Preferential Affinity for the Activated State of FLT3&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Zhihong&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Xianghong&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;John&quot;,&quot;last_name&quot;:&quot;Eksterowicz&quot;},{&quot;first_name&quot;:&quot;Michael&quot;,&quot;last_name&quot;:&quot;W. Gribble&quot;},{&quot;first_name&quot;:&quot;Grace&quot;,&quot;last_name&quot;:&quot;Q. Alba&quot;},{&quot;first_name&quot;:&quot;Merrill&quot;,&quot;last_name&quot;:&quot;Ayres&quot;},{&quot;first_name&quot;:&quot;Timothy&quot;,&quot;last_name&quot;:&quot;J. Carlson&quot;},{&quot;first_name&quot;:&quot;Ada&quot;,&quot;last_name&quot;:&quot;Chen&quot;},{&quot;first_name&quot;:&quot;Xiaoqi&quot;,&quot;last_name&quot;:&quot;Chen&quot;},{&quot;first_name&quot;:&quot;Robert&quot;,&quot;last_name&quot;:&quot;Cho&quot;},{&quot;first_name&quot;:&quot;Richard&quot;,&quot;last_name&quot;:&quot;V. Connors&quot;},{&quot;first_name&quot;:&quot;Michael&quot;,&quot;last_name&quot;:&quot;DeGraffenreid&quot;},{&quot;first_name&quot;:&quot;Jeffrey&quot;,&quot;last_name&quot;:&quot;T. Deignan&quot;},{&quot;first_name&quot;:&quot;Jason&quot;,&quot;last_name&quot;:&quot;Duquette&quot;},{&quot;first_name&quot;:&quot;Pingchen&quot;,&quot;last_name&quot;:&quot;Fan&quot;},{&quot;first_name&quot;:&quot;Benjamin&quot;,&quot;last_name&quot;:&quot;Fisher&quot;},{&quot;first_name&quot;:&quot;Jiasheng&quot;,&quot;last_name&quot;:&quot;Fu&quot;},{&quot;first_name&quot;:&quot;Justin&quot;,&quot;last_name&quot;:&quot;N. Huard&quot;},{&quot;first_name&quot;:&quot;Jacob&quot;,&quot;last_name&quot;:&quot;Kaizerman&quot;},{&quot;first_name&quot;:&quot;Kathleen&quot;,&quot;last_name&quot;:&quot;S. Keegan&quot;},{&quot;first_name&quot;:&quot;Cong&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Kexue&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Yunxiao&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Lingming&quot;,&quot;last_name&quot;:&quot;Liang&quot;},{&quot;first_name&quot;:&quot;Wen&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Sarah&quot;,&quot;last_name&quot;:&quot;E. Lively&quot;},{&quot;first_name&quot;:&quot;Mei-Chu&quot;,&quot;last_name&quot;:&quot;Lo&quot;},{&quot;first_name&quot;:&quot;Ji&quot;,&quot;last_name&quot;:&quot;Ma&quot;},{&quot;first_name&quot;:&quot;Dustin&quot;,&quot;last_name&quot;:&quot;L. McMinn&quot;},{&quot;first_name&quot;:&quot;Jeffrey&quot;,&quot;last_name&quot;:&quot;T. Mihalic&quot;},{&quot;first_name&quot;:&quot;Kriti&quot;,&quot;last_name&quot;:&quot;Modi&quot;},{&quot;first_name&quot;:&quot;Rachel&quot;,&quot;last_name&quot;:&quot;Ngo&quot;},{&quot;first_name&quot;:&quot;Kanaka&quot;,&quot;last_name&quot;:&quot;Pattabiraman&quot;},{&quot;first_name&quot;:&quot;Derek&quot;,&quot;last_name&quot;:&quot;E. Piper&quot;},{&quot;first_name&quot;:&quot;Christophe&quot;,&quot;last_name&quot;:&quot;Queva&quot;},{&quot;first_name&quot;:&quot;Mark&quot;,&quot;last_name&quot;:&quot;L. Ragains&quot;},{&quot;first_name&quot;:&quot;Julia&quot;,&quot;last_name&quot;:&quot;Suchomel&quot;},{&quot;first_name&quot;:&quot;Steve&quot;,&quot;last_name&quot;:&quot;Thibault&quot;},{&quot;first_name&quot;:&quot;Nigel&quot;,&quot;last_name&quot;:&quot;Walker&quot;},{&quot;first_name&quot;:&quot;Xiaodong&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Zhulun&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Malgorzata&quot;,&quot;last_name&quot;:&quot;Wanska&quot;},{&quot;first_name&quot;:&quot;Paul&quot;,&quot;last_name&quot;:&quot;M. Wehn&quot;},{&quot;first_name&quot;:&quot;Margaret&quot;,&quot;last_name&quot;:&quot;F. Weidner&quot;},{&quot;first_name&quot;:&quot;Alex&quot;,&quot;last_name&quot;:&quot;J. Zhang&quot;},{&quot;first_name&quot;:&quot;Xiaoning&quot;,&quot;last_name&quot;:&quot;Zhao&quot;},{&quot;first_name&quot;:&quot;Alexander&quot;,&quot;last_name&quot;:&quot;Kamb&quot;},{&quot;first_name&quot;:&quot;Dineli&quot;,&quot;last_name&quot;:&quot;Wickramasinghe&quot;},{&quot;first_name&quot;:&quot;Kang&quot;,&quot;last_name&quot;:&quot;Dai&quot;},{&quot;first_name&quot;:&quot;Lawrence&quot;,&quot;last_name&quot;:&quot;R. McGee&quot;},{&quot;first_name&quot;:&quot;Julio&quot;,&quot;last_name&quot;:&quot;C. Medina&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2014&quot;,&quot;month&quot;:&quot;04&quot;,&quot;day&quot;:&quot;02&quot;,&quot;issue&quot;:&quot;8&quot;,&quot;volume&quot;:&quot;57&quot;,&quot;pages&quot;:&quot;3430-3449&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/jm500118j&quot;},&quot;abstract&quot;:&quot;We describe the structural optimization of a lead compound 1 that exhibits dual inhibitory activities against FLT3 and CDK4. A series of pyrido[4′,3′:4,5]pyrrolo[2,3-d]pyrimidine derivatives was synthesized, and SAR analysis, using cell-based assays, led to the discovery of 28 (AMG 925), a potent and orally bioavailable dual inhibitor of CDK4 and FLT3, including many FLT3 mutants reported to date. Compound 28 inhibits the proliferation of a panel of human tumor cell lines including Colo205 (Rb+) and U937 (FLT3WT) and induced cell death in MOLM13 (FLT3ITD) and even in MOLM13 (FLT3ITD, D835Y), which exhibits resistance to a number of FLT3 inhibitors currently under clinical development. At well-tolerated doses, compound 28 leads to significant growth inhibition of MOLM13 xenografts in nude mice, and the activity correlates with inhibition of STAT5 and Rb phosphorylation.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Fjm500118j&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm500118j" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm500118j&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=jm500118j" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm500118j&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=jm500118j" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Fjm500118j&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm500118j&amp;href=/doi/10.1021/jm500118j" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/jm500118j" /></input><a href="/doi/pdf/10.1021/jm500118j" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (3 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/jm500118j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Fjm500118j%26sid%3Dliteratum%253Aachs%26pmid%3D24641103%26genre%3Darticle%26aulast%3DLi%26date%3D2014%26atitle%3DDiscovery%2Bof%2BAMG%2B925%252C%2Ba%2BFLT3%2Band%2BCDK4%2BDual%2BKinase%2BInhibitor%2Bwith%2BPreferential%2BAffinity%2Bfor%2Bthe%2BActivated%2BState%2Bof%2BFLT3%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D57%26issue%3D8%26spage%3D3430%26epage%3D3449%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (1)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291300" title="Palladium">Palladium</a>,</li><li><a href="/action/doSearch?ConceptID=292464" title="Rodent models">Rodent models</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/57/8" title="Go to Volume 57, Issue 8"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jmcmar.2014.57.issue-8/production/jmcmar.2014.57.issue-8.largecover.jpg" alt="Go to Volume 57, Issue 8"/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/medium/jm-2014-00118j_0028.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0028.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500118j&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">We describe the structural optimization of a lead compound <b>1</b> that exhibits dual inhibitory activities against FLT3 and CDK4. A series of pyrido[4′,3′:4,5]pyrrolo[2,3-<i>d</i>]pyrimidine derivatives was synthesized, and SAR analysis, using cell-based assays, led to the discovery of <b>28</b> (<b>AMG 925</b>), a potent and orally bioavailable dual inhibitor of CDK4 and FLT3, including many FLT3 mutants reported to date. Compound <b>28</b> inhibits the proliferation of a panel of human tumor cell lines including Colo205 (Rb<sup>+</sup>) and U937 (FLT3<sup>WT</sup>) and induced cell death in MOLM13 (FLT3<sup>ITD</sup>) and even in MOLM13 (FLT3<sup>ITD, D835Y</sup>), which exhibits resistance to a number of FLT3 inhibitors currently under clinical development. At well-tolerated doses, compound <b>28</b> leads to significant growth inhibition of MOLM13 xenografts in nude mice, and the activity correlates with inhibition of STAT5 and Rb phosphorylation.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_28351" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_28351" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Acute myeloid leukemia (AML) is a grievous disease for which there is significant unmet medical need. There are 14600 cases diagnosed each year, with an estimated 10400 deaths in the U.S. alone.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> The standard of care for AML is poorly tolerated among elderly patients. Durable responses are also lacking, and rapid relapse is common. FMS-like tyrosine kinase 3 (FLT3) belongs to the receptor tyrosine kinase class III family and is expressed at high levels in most clinical samples from AML and B-precursor acute lymphoblastic leukemia (ALL) patients.<a onclick="showRef(event, 'ref2 ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref2 ref3 ref4">(2-4)</a> Activating mutations in FLT3, mostly internal tandem duplication (ITD), are detected in approximately 30% of AML patients and are associated with poor prognosis.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> The important role that FLT3 plays in the survival and proliferation of AML blast through the STAT5, RAS-ERK, and phosphatidylinositide 3 kinase (PI3K) signaling pathways, as well as its mutation and overexpression in large numbers of AML patients, make FLT3 a particularly attractive target for AML therapy.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a></div><div class="NLM_p">Despite extensive efforts toward developing targeted agents for treatment of AML,<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> a critical unmet need remains. FLT3 tyrosine kinase inhibitors used as monotherapies for AML have effected reductions in peripheral blood and bone marrow blasts in only a minority of AML patients, and these effects have usually been transient. This may be due to selection of drug-resistant mutants which are the primary resistance mechanism observed in the clinic,<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> increases in FLT3 expression during treatment, or activation of other signaling pathways.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a></div><div class="NLM_p">Cyclin-dependent kinase 4 (CDK4), which is downstream of FLT3<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> and other growth signaling pathways, is also up-regulated in AML by overexpression of cyclin D through activation of tyrosine kinase growth signaling pathways and loss of p15 inhibitory function.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> Misregulated CDKs cause uncontrolled proliferation as well as genomic and chromosomal instability.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> In addition, synergistic activity has been observed in preclinical studies when FLT3 and CDK4 were inhibited in AML cell lines.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a></div><div class="NLM_p last">These findings together suggest the potential for a FLT3/CDK4 dual inhibitor to provide significant benefit in the treatment of AML, including improving durability of clinical response by simultaneously modulating two synergistic and commonly dysregulated targets.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_65548" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_65548" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Previously, we reported the discovery of substituted pyrido[4′,3′:4,5]pyrrolo[2,3-<i>d</i>]pyrimidine <b>1</b> (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>) as a novel CDK4/6 inhibitor.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> During kinase profiling it was found that compound <b>1</b> also inhibits FLT3, yet possesses promising selectivity against other kinases. This finding can be rationalized on the basis of the high degree of similarity between the binding domains of CDK4/6 (co-crystal structure of CDK6 with <b>1</b> in Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>a) and APO FLT3 (modeled with <b>1</b> in Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>b). The active sites of the two enzymes differ by only a few residues, and the binding mode of <b>1</b> obtained by modeling with FLT3 very closely resembles that observed in the co-crystal structure with CDK6.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/medium/jm-2014-00118j_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Lead compound <b>1</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500118j&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/medium/jm-2014-00118j_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Binding of compound <b>1</b> in CDK6 and FLT3. (a) Co-crystal structure of compound <b>1</b> in CDK6 at 2.9 Å resolution (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4P41">4P41</a>). (b) Compound <b>1</b> with homology model of FLT3.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> The key active site residues are highlighted in stick representation.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500118j&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In both structures, the fused pyridine moiety of the inhibitor engages the phenylalanine gatekeeper residue in an edge-to-face interaction, while the aminopyrimidine ring system participates in two hydrogen bonding interactions with residues in the hinge loop. In both structures, the cyclopentyl group sits in a small hydrophobic cleft between a leucine floor residue and small side chain alanine (CDK4) or cysteine (FLT3). Taking advantage of these shared structural features and binding interactions, we sought to develop dual kinase inhibitors of FLT3/CDK4 for the treatment of AML.</div><div class="NLM_p">Lead optimization was conducted to address three main issues associated with <b>1</b>: CYP3A4 inhibition, suboptimal selectivity, and poor oral bioavailability. For structure–activity relationship (SAR) analysis, FLT3 and CDK family kinases assays were used as a primary screen to monitor on/off target activities, with the hope that the selectivity within the CDK family may be reflective of broader kinase selectivity. The selectivity over CDK1 is necessary for reducing toxicity and confounding in vivo analysis.<a onclick="showRef(event, 'ref11 ref15'); return false;" href="javascript:void(0);" class="ref ref11 ref15">(11, 15)</a> Using biochemical assays to optimize against 500 kinases was deemed impractical, therefore cell-based FLT3 and CDK4 specific assays were used for screening inhibitors. Three tumor cell lines, Colo205, MOLM13, and MDA-MB436, were chosen. Colo205 is a human colorectal cancer cell line whose viability depends on the activation of the cyclin D-CDK4-Rb pathway. MOLM13 is a human AML cell line whose viability depends on a constitutively activated FLT3 kinase. MDA-MB436 is a human breast cancer cell line whose viability is independent of FLT3 and CDK4 and was thus included to monitor for off-target effects. Compounds were anticipated to be selective if they had high potency against MOLM13 and Colo205 cells but low activity against MDA-MB436 cell line.</div><div class="NLM_p">While the CYP3A4 inhibitory activity of the lead <b>1</b> stems from the fused pyridine ring of the tricyclic core, our earlier SAR studies<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> had established that this feature was critical for maintaining good cellular potency. Our strategy for attenuating the CYP3A4 inhibition, therefore sought not to eliminate this feature but rather to modulate its ability to coordinate heme iron through introduction of either sterically demanding or electron-withdrawing substituents <i>ortho</i> to the pyridine nitrogen (Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>). The applicability of this strategy quickly proved limited, however, as only small substituents, such as chloro or methyl, were tolerated by the kinases. While these changes effectively eliminated the CYP3A4 inhibition, they diminished either cellular or enzymatic potency and in some cases incurred potential cardiovascular liabilities by markedly increasing affinity for the hERG ion channel.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Modification of the Fused Pyridine Ring<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/medium/jm-2014-00118j_0021.gif" alt="" id="fx1" /></img><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">Y</th><th class="colsep0 rowsep0" align="center"><b>1</b> CH</th><th class="colsep0 rowsep0" align="center"><b>2</b> CF</th><th class="colsep0 rowsep0" align="center"><b>3</b> CCl</th><th class="colsep0 rowsep0" align="center"><b>4</b> CCH<sub>3</sub></th><th class="colsep0 rowsep0" align="center"><b>5</b> COH</th><th class="colsep0 rowsep0" align="center"><b>6</b> CCN</th><th class="colsep0 rowsep0" align="center"><b>7</b> N</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CDK4/D1 IC<sub>50</sub> (μM)</td><td class="colsep0 rowsep0" align="left">0.003</td><td class="colsep0 rowsep0" align="left">0.003</td><td class="colsep0 rowsep0" align="left">0.004</td><td class="colsep0 rowsep0" align="left">0.020</td><td class="colsep0 rowsep0" align="left">0.005</td><td class="colsep0 rowsep0" align="left">0.027</td><td class="colsep0 rowsep0" align="left">0.051</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">FLT3 IC<sub>50</sub> (μM)</td><td class="colsep0 rowsep0" align="left">0.014</td><td class="colsep0 rowsep0" align="left">0.011</td><td class="colsep0 rowsep0" align="left">0.005</td><td class="colsep0 rowsep0" align="left">0.022</td><td class="colsep0 rowsep0" align="left">0.017</td><td class="colsep0 rowsep0" align="left">0.046</td><td class="colsep0 rowsep0" align="left">0.095</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CDK1/B IC<sub>50</sub> (μM)</td><td class="colsep0 rowsep0" align="left">3.00</td><td class="colsep0 rowsep0" align="left">7.56</td><td class="colsep0 rowsep0" align="left">3.30</td><td class="colsep0 rowsep0" align="left">2.26</td><td class="colsep0 rowsep0" align="left">10.0</td><td class="colsep0 rowsep0" align="left">30.0</td><td class="colsep0 rowsep0" align="left">4.78</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Colo205 IC<sub>50</sub> (μM)</td><td class="colsep0 rowsep0" align="left">0.025</td><td class="colsep0 rowsep0" align="left">0.052</td><td class="colsep0 rowsep0" align="left">0.063</td><td class="colsep0 rowsep0" align="left">0.28</td><td class="colsep0 rowsep0" align="left">0.12</td><td class="colsep0 rowsep0" align="left">0.35</td><td class="colsep0 rowsep0" align="left">0.051</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MOLM13 IC<sub>50</sub> (μM)</td><td class="colsep0 rowsep0" align="left">0.005</td><td class="colsep0 rowsep0" align="left">0.021</td><td class="colsep0 rowsep0" align="left">0.009</td><td class="colsep0 rowsep0" align="left">0.005</td><td class="colsep0 rowsep0" align="left">0.005</td><td class="colsep0 rowsep0" align="left">0.062</td><td class="colsep0 rowsep0" align="left">ND</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MDA-MB436 IC<sub>50</sub> (μM)</td><td class="colsep0 rowsep0" align="left">1.76</td><td class="colsep0 rowsep0" align="left">2.61</td><td class="colsep0 rowsep0" align="left">2.10</td><td class="colsep0 rowsep0" align="left">3.42</td><td class="colsep0 rowsep0" align="left">2.10</td><td class="colsep0 rowsep0" align="left">1.38</td><td class="colsep0 rowsep0" align="left">7.05</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CYP3A4 INH%@3 μM</td><td class="colsep0 rowsep0" align="left">90</td><td class="colsep0 rowsep0" align="left"><10</td><td class="colsep0 rowsep0" align="left"><10</td><td class="colsep0 rowsep0" align="left"><10</td><td class="colsep0 rowsep0" align="left">18</td><td class="colsep0 rowsep0" align="left">ND</td><td class="colsep0 rowsep0" align="left"><10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CYP3A4 IC<sub>50</sub> (μM)</td><td class="colsep0 rowsep0" align="left">0.55</td><td class="colsep0 rowsep0" align="left">>10</td><td class="colsep0 rowsep0" align="left">>10</td><td class="colsep0 rowsep0" align="left">>10</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Rat CL (L/h/kg)</td><td class="colsep0 rowsep0" align="left">0.99</td><td class="colsep0 rowsep0" align="left">2.1</td><td class="colsep0 rowsep0" align="left">1.7</td><td class="colsep0 rowsep0" align="left">ND</td><td class="colsep0 rowsep0" align="left">ND</td><td class="colsep0 rowsep0" align="left">ND</td><td class="colsep0 rowsep0" align="left">5.6</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup>a</sup><p class="last">Values were the means of at least three determinations; standard deviation was ±30%. ND: not determined.</p></div></div></div><div class="NLM_p">Thus, substitution with fluoro or chloro maintained enzymatic potency and selectivity but resulted in reduced cellular potencies and unacceptably high affinities for hERG (<b>2</b> and <b>3</b>, hERG IC<sub>50</sub> = 0.72, 2.0 μM, respectively). Additionally, these halopyridine analogues suffered from increased clearances in rat (from 0.98 L/h/kg for <b>1</b> to 2.1 L/h/kg for <b>2</b> and 1.7 L/h/kg for <b>3</b>). Substitution with methyl and hydroxyl groups (<b>4</b> and <b>5</b>) maintained cellular potency against MOLM13 but significantly decreased the CDK4-related activity against Colo205 cells. Substitution with a cyano group (<b>6</b>) resulted in decreased potency for both CDK4 and FLT3. Replacement of the pyridine ring with a fused pyridazine (<b>7</b>) likewise eliminated CYP3A4 inhibition at the expense of potency and pharmacokinetic properties (rat CL = 5.6 L/h/kg). The pyridine-modification strategy for attenuating CYP3A4 inhibition was deemed at this point to be very challenging. One after another, new analogues reinforced the view of the unsubstituted pyridine of <b>1</b> as critical for potency and physicochemical and pharmacokinetic properties.</div><div class="NLM_p">As we explored additional SAR of the cycloalkane moiety, we noticed an opportunity for attenuating CYP3A4 inhibition through remote influences on the heme iron coordination sphere. The cyclopentane ring of <b>1</b> occupies a region of the kinases known as the sugar pocket, where the ribose moiety of ATP normally binds. The sugar pockets of CDK4 and FLT3 are formed by both polar and nonpolar residues, with the more deeply buried region of the pocket being hydrophobic in character and surrounded by a solvent exposed, polar rim (Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>). These amphipathic pockets can accommodate a variety of cyclic moieties, including rings substituted with either polar or nonpolar groups. We also considered the complementary possibility that replacing the cyclopentyl ring with a polar surrogate could impact CYP3A4 inhibition by changing the physicochemical properties of the molecule. Extensive exploration of this region was performed, and only selected examples are discussed here (Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>).</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/medium/jm-2014-00118j_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. CDK6/compound <b>1</b> co-crystal structure highlighting ATP binding site with P-loop surface removed for clarity (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4P41">4P41</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500118j&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Optimization of the Sugar Pocket Binding Moiety<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/medium/jm-2014-00118j_0022.gif" alt="" id="fx2" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/medium/jm-2014-00118j_0023.gif" alt="" id="fx3" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup>Table a</sup><p class="last">Values were the means of at least three determinations; standard deviation was ±30%. ND: not determined.</p></div></div><div></div></div><div class="NLM_p">As evidenced by compound <b>8</b>, an acyclic moiety containing a polar group can significantly reduce CYP3A4 inhibition; however, in most cases this modification resulted in a more than 10-fold loss of the binding affinity to the kinase target. However, the hydroxyl substituted cyclohexane (<b>9</b>) did not significantly reduce CYP3A4 inhibition and additionally suffered losses of cellular potency and selectivity relative to <b>1</b>. Likewise, bulkier aliphatic rings, like cyclohexyl (<b>10</b>) and cycloheptyl (data not shown), did not modulate CYP3A4 inhibitory activity either. However, addition of a methyl group at the 4-position of the cyclohexyl moiety led to a significant reduction in CYP3A4 inhibition (<b>11</b>). Entry <b>11</b> corresponds to a mixture of <i>cis</i> and <i>trans</i> diastereomers (1/1 ratio), and based on data collected from previous analogues, we predicted that the <i>trans</i>-isomer was responsible for the decreased affinity for CYP3A4. Subsequently, the <i>trans</i>-isomer (<b>12</b>) was prepared in stereochemically pure form and found to be free of this liability (CYP3A4 IC<sub>50</sub> >10 μM). Docking studies<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> of a series of inhibitors in the active site of CYP3A4 supported the hypothesis that ligation of the heme iron could be disfavored by a remote steric interaction with a sterically bulky cycloalkane. While the studies revealed favorable binding modes associated with coordination of the heme iron by the pyridine moiety of <b>1</b>, steric interactions developed between the cycloalkane and key residues of the CYP active site when the cycloalkane was appropriately substituted, and these destabilizing interactions grew more pronounced as the substituent was made longer and bulkier. Compound <b>14</b> illustrates that this effect does not derive from nonspecific lipophilicity but requires the specific steric demand available in compounds <b>12</b> and <b>13</b>. As the <i>trans</i>-4-methycyclohexyl group abolished CYP3A4 inhibition without diminishing FLT3 or CDK4 potency or inducing any other known liabilities, this group was chosen for further analogues.</div><div class="NLM_p">Having overcome one of the three major challenges associated with this series by optimizing the tricyclic core and its cycloalkyl substituent, we next sought to improve bioavailability and FLT3 potency through appropriate modifications of the distal amine function (i.e., the piperazine ring of <b>1</b>) and the heteroarene ring linking it to the tricyclic core. In the co-crystal structure, the piperazine ring sits in a channel of the kinase that opens toward solvent and can accommodate a wide array of polar groups. We therefore anticipated that optimization of the physicochemical properties of the dual inhibitors by variation of the pyridinylpiperazine could be especially fruitful. Our initial work focused on developing a series that retained a basic amine because this feature was known to contribute favorably to selectivity within the CDK family and to solubility. The piperazine basic nitrogen of <b>1</b>, which is protonated under physiological conditions, rests above a threonine residue in the X-ray co-crystal structure with CDK6 (Thr-102 in CDK4) and engages this residue in a hydrogen-bond interaction (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>), whereas the corresponding lysine-89 of CDK1 should interact unfavorably with the basic amine function.</div><div class="NLM_p">Extending the basic nitrogen farther into the solvent channel by making it exocyclic (<b>17</b>) (Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>) substantially increased cellular and FLT3 potency but diminished selectivity for CDK4 over CDK1 compared to <b>12</b>. Selectivity for CDK4 was restored when the primary amine of <b>17</b> was bis-methylated (<b>18</b>) while maintaining similar potency across the biochemical and cellular assays.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Optimization of the Heteroaryl Linker<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/medium/jm-2014-00118j_0024.gif" alt="" id="fx4" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/medium/jm-2014-00118j_0025.gif" alt="" id="fx5" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup>Table a</sup><p class="last">Values were the means of at least three determinations; standard deviation was ±30%.</p></div><div class="footnote" id="t3fn2"><sup>Table b</sup><p class="last">Formulation: 1.5% sodium acetate/pH4. ND: not determined.</p></div></div><div></div></div><div class="NLM_p">In parallel with these studies, we also explored the effects of atom substitution within the heteroarene linker (<b>15</b>, <b>16</b>, <b>19</b>, <b>22</b>–<b>25</b>) (Tables <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a> and <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a>) as well as installation of an additional substituent <i>ortho</i> to the pyridine nitrogen (<b>20</b>, <b>21</b>, <b>27</b>, <b>28</b>). Transposing the pyridine nitrogen markedly improved potency against FLT3 but severely eroded selectivity within the CDK family (<b>12</b> versus <b>15</b>). Replacing the pyridine with a more polar pyridazine (<b>16</b>, <b>19</b>, <b>22</b>–<b>25</b>) did not affect potency or selectivity but did favorably impact a number of important pharmacokinetic parameters such as solubility and bioavailability. Unexpectedly, heteroatom substitution <i>ortho</i> to the pyridine nitrogen gave remarkable increases in FLT3 potencies (picomolar IC<sub>50</sub>s for <b>20</b>, <b>21</b>) and dramatic improvements in oral bioavailability while maintaining acceptable selectivity among the CDKs. Unfortunately, these compounds were not tolerated in mice at doses that led to 100% tumor growth inhibition (TGI) in xenograft studies.</div><div class="NLM_p">Convergence of the heteroarene and distal amine-modification strategies resulted in compound <b>19</b> (CDK4 IC<sub>50</sub> = 2 nM; FLT3 IC<sub>50</sub> = 4 nM; MOLM13 IC<sub>50</sub> = 9 nM; Colo205 IC<sub>50</sub> = 11 nM; >1000-fold selectivity for CDK4 over CDK1; >200-fold cellular off/on targets ratio; pRb IC<sub>50</sub> = 23 nM and pSTAT5 IC<sub>50</sub> = 52 nM in MOLM13 cell line, respectively), and we were pleased to find that this molecule exhibited good-to-moderate pharmacokinetics across the three species studied (rat, mouse, and dog). A dose-dependent decrease in Rb phosphorylation (pRb) was observed in a MOLM13 mouse xenograft PD study with <b>19</b> starting at the 37.5 mg/kg dose (Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>b). In addition, a dose-dependent increase in drug concentration in plasma was observed. However, drug exposures in mouse neither led to a dose-dependent reduction in STAT5 phosphorylation (pSTAT5) nor a sufficient decrease in pSTAT5 even at the 150 mg/kg dose (Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>a). This may have been due to limited drug exposure levels in the mouse plasma (unbound <i>C</i><sub>max</sub> = 0.024 μM at 150 mg/kg dose, much lower than cellular IC<sub>50</sub> of pSTAT5 in MOLM13 of 52 nM).</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/medium/jm-2014-00118j_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. STAT5 and Rb phosphorylation in MOLM13 tumors treated with <b>19</b>. MOLM13 tumor bearing mice were dosed by oral administration of <b>19</b> at 37.5, 75, or 150 mg/kg, and tumors were harvested at 3 and 24 h after treatment. Tumors lysates were prepared and the level of pSTAT5 (at 3 h post dose, (a)) or pRb (at 24 h post dose, (b)) was determined. Unbound plasma levels of <b>19</b> (μM) for the same time points are shown. Data are presented as mean ± SEM, <i>n</i> = 3 except for vehicle group (<i>n</i> = 6).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500118j&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Reasoning that further optimization of the physicochemical properties would be required for achieving efficacious drug exposures, and knowing that the charged anime group of our inhibitors was associated with reduced phospholipid membrane permeability and intestinal absorption and with increased efflux rates, we next turned our attention toward less basic polar surrogates for the amine moiety of <b>19</b>. Representative nonbasic analogues are shown in Figures <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a> and <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a> and discussed in detail below.</div><div class="NLM_p">First, we considered that ethanolamine derivatives, exemplified by <b>22</b>, might benefit from several advantages, including reductions in basicity compared to <b>19</b> (p<i>K</i><sub>a</sub> = 9.2) through either inductive or hydrogen-bonding effects as well as possible stabilization of the ion pairs required for efficient passive diffusion of the protonated structure (Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>).<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> Propanolamine <b>22</b> indeed gave much higher drug exposure in the rat (total <i>C</i><sub>max</sub> = 0.28 μM and AUC = 3.24 μM·h at the 2 mg/kg oral dose) than <b>19</b> (total <i>C</i><sub>max</sub> = 0.13 μM and AUC = 1.57 μM·h at the 2 mg/kg oral dose). Unfortunately, further development of this series was precluded by high binding affinities for hERG (IC<sub>50</sub> = 0.92 μM) and moderate-to-high clearance in the dog.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/medium/jm-2014-00118j_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (a) Experimentally measured p<i>K</i><sub>a</sub> of compound <b>19</b>; (b) R<sub>4</sub>N<sup>+</sup> and Cl<sup>–</sup> ion pair may be stabilized by hydrogen bonds.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500118j&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Ultimately, we sought to replace the basic amine with an effectively nonbasic polar surrogate (e.g., <b>23</b>–<b>28</b>). Glycolamides, sulfones, and various spiro-lactams all resulted in dramatically improved pharmacokinetic properties, particularly increased drug exposures in high-dose mouse experiments (Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>) at the expense of modest reductions in potency (Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a>).</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/medium/jm-2014-00118j_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Drug exposure levels vs molecular basicity (C<sub>3h</sub>: total drug concentration at 3 h post 150 mg/kg oral dose in mouse PK studies).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500118j&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Optimization of the Heteroaryl Linker<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/medium/jm-2014-00118j_0026.gif" alt="" id="fx6" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/medium/jm-2014-00118j_0027.gif" alt="" id="fx7" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup>Table a</sup><p class="last">Values were the means of at least three determinations; standard deviation was ±30%.</p></div><div class="footnote" id="t4fn2"><sup>Table b</sup><p class="last">Formulation: 1.5% sodium acetate/pH4.</p></div></div><div></div></div><div class="NLM_p">Among these analogues, compound <b>28</b> (<b>AMG 925</b>) emerged as the most suitable candidate for clinical development by virtue of its excellent pharmacokinetic properties (in rat, CL = 0.39 L/h/kg, <i>F</i>% = 75), well-balanced inhibitory activities on FLT3 and CDK4 with good selectivity against the Kinome Scan panel<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> (<a class="ref internalNav" href="#notes-1" aria-label="Supporting Information Tables 1 and 2">Supporting Information Tables 1 and 2</a>), potent antiproliferation efficacy against MOLM13 cells, and superior safety profile and in vitro ADME properties (data not shown).</div><div class="NLM_p">With a suitable candidate now in hand, we evaluated <b>28</b> against a panel of activated variants of FLT3. Activating mutations of kinases are commonly observed in cancer and can be selected by treatment with type II inhibitors, promoting the development of cancer that is more difficult to treat and leading to an acute need for agents that efficaciously target these mutations.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> Gratifyingly, <b>28</b> exhibited excellent binding affinities (1–4 nM) for all of the FLT3 mutants available in the KINOMEscan (data in Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a>). This high affinity for mutated forms of FLT3, particularly those mutants known to confer pronounced resistance to FLT3-inhibition monotherapy, make compound <b>28</b> very attractive for clinical development.</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Binding Affinity of <b>28</b> to Wild-Type and Mutant FLT3</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center" char=".">FLT3<sup>WT</sup></th><th class="colsep0 rowsep0" align="center" char=".">FLT3<sup>ITD</sup></th><th class="colsep0 rowsep0" align="center" char=".">FLT3<sup>D835Y</sup></th><th class="colsep0 rowsep0" align="center" char=".">FLT3<sup>D835H</sup></th><th class="colsep0 rowsep0" align="center" char=".">FLT3<sup>K663Q</sup></th><th class="colsep0 rowsep0" align="center" char=".">FLT3 <sup>N841I</sup></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>K</i><sub>d</sub> (μM)</td><td class="colsep0 rowsep0" align="char" char=".">0.004</td><td class="colsep0 rowsep0" align="char" char=".">0.004</td><td class="colsep0 rowsep0" align="char" char=".">0.001</td><td class="colsep0 rowsep0" align="char" char=".">0.001</td><td class="colsep0 rowsep0" align="char" char=".">0.004</td><td class="colsep0 rowsep0" align="char" char=".">0.004</td></tr></tbody></table></div></div><div class="NLM_p">The activity of <b>28</b> in various tumor cell lines demonstrated its ability to block FLT3 and CDK4. Compound <b>28</b> showed potent and broad antiproliferation activities against AML cell lines independent of the FLT3 mutation status while maintaining simultaneous inhibition of CDK4. Compound <b>28</b> inhibited pSTAT5 in MOLM13 and pRb in Colo205 with IC<sub>50</sub>s of 0.005 and 0.023 μM, respectively, indicating that the observed efficacy in vitro was consistent with FLT3 and CDK4 inhibition (Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a>). In contrast, sorafenib,<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> a multikinase inhibitor with FLT3 activity, showed potent inhibitory activity toward the FLT3<sup>ITD</sup> mutant MOLM13 cell line but almost no activity against U937, a FLT3 wild-type AML cell line. In contrast to palbociclib,<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> a CDK4/6 inhibitor, which arrests Rb positive tumor cells including MOLM13 in G1 stage, <b>28</b> induced apoptosis in FLT3<sup>ITD</sup> AML cell lines and the FLT3<sup>ITD,D835Y</sup> mutant AML cell line, the latter being a mutant strain cultivated to exhibit marked resistance to sorafenib by continuously treating MOLM13(FLT3<sup>ITD</sup>) cells with gradually increased concentration of sorafenib (from 1 to 1000 nM) over the course of several weeks (Table <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">7</a>). Analogous experiments conducted with the concentration of <b>28</b> ranging as high as 10 nM over treatment durations as long as 4 months failed to produce appreciable resistance to <b>28</b> (IC<sub>50</sub> = 0.028 μM), and sequencing analysis showed that no additional FLT3 mutations had been produced under these conditions.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> These results are consistent with the hypothesis that dual inhibition of FLT3 and CDK4 make it more challenging for AML cells to develop resistance to FLT3 inhibition than in the case of FLT3-inhibition monotherapy. In aggregate, these data indicate that the FLT3/CDK4 dual inhibitor <b>28</b> has broader activity, and is potentially more effective in treating AML, than the current therapeutic agents.</div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. The Antiproliferation Activity of <b>28</b> in Comparison with a CDK4/6 and FLT3 Inhibitor<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center" char="."><b>28</b></th><th class="colsep0 rowsep0" align="center" char=".">palbociclib</th><th class="colsep0 rowsep0" align="center" char=".">sorafenib</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">FLT3 IC<sub>50</sub> (μM)</td><td class="colsep0 rowsep0" align="char" char=".">0.001</td><td class="colsep0 rowsep0" align="char" char=".">3.48</td><td class="colsep0 rowsep0" align="char" char=".">0.004</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CDK4 IC<sub>50</sub> (μM)</td><td class="colsep0 rowsep0" align="char" char=".">0.003</td><td class="colsep0 rowsep0" align="char" char=".">0.002</td><td class="colsep0 rowsep0" align="char" char=".">>3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">pSTAT5 IC<sub>50</sub> (μM)</td><td class="colsep0 rowsep0" align="char" char=".">0.005</td><td class="colsep0 rowsep0" align="char" char=".">>3</td><td class="colsep0 rowsep0" align="char" char=".">0.002</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">pRb IC<sub>50</sub> (μM)</td><td class="colsep0 rowsep0" align="char" char=".">0.023</td><td class="colsep0 rowsep0" align="char" char=".">0.011</td><td class="colsep0 rowsep0" align="char" char=".">>3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MOLM13 IC<sub>50</sub> (μM)</td><td class="colsep0 rowsep0" align="char" char=".">0.019</td><td class="colsep0 rowsep0" align="char" char=".">0.096</td><td class="colsep0 rowsep0" align="char" char=".">0.005</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">U937 IC<sub>50</sub> (μM)</td><td class="colsep0 rowsep0" align="char" char=".">0.052</td><td class="colsep0 rowsep0" align="char" char=".">0.14</td><td class="colsep0 rowsep0" align="char" char=".">>3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MOLM13<sup>SR</sup> IC<sub>50</sub> (μM)</td><td class="colsep0 rowsep0" align="char" char=".">0.023</td><td class="colsep0 rowsep0" align="char" char=".">0.096</td><td class="colsep0 rowsep0" align="char" char=".">>3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Colo205 IC<sub>50</sub> (μM)</td><td class="colsep0 rowsep0" align="char" char=".">0.055</td><td class="colsep0 rowsep0" align="char" char=".">0.036</td><td class="colsep0 rowsep0" align="char" char=".">>3</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t6fn1"><div class="footnote" id="t6fn1"><sup>a</sup><p class="last">The antiproliferation activity was determined by <sup>14</sup>C-thymindine incorporation DNA synthesis assay; MOLM13<sup>SR</sup>: (FLT3<sup>ITD+D835Y</sup>) clone was isolated by culturing MOLM13 in the presence of 1 nM Sorafenib. Resistance cells were isolated in about 6 weeks, and the cells can rapidly adapt to >1 μM of sorafenib in culture (about 2 weeks from 1 nM to 1 μM).</p></div></div></div><div class="NLM_table-wrap" id="tbl7"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 7. Compound <b>28</b> Induced Apoptosis in FLT3 Mutant AML Cells<a class="ref internalNav" href="#t7fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center" char="."><b>28</b></th><th class="colsep0 rowsep0" align="center" char=".">palbociclib</th><th class="colsep0 rowsep0" align="center" char=".">sorafenib</th><th class="colsep0 rowsep0" align="center" char=".">control</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">U937 (FLT3<sup>WT</sup>)</td><td class="colsep0 rowsep0" align="char" char=".">23.68</td><td class="colsep0 rowsep0" align="char" char=".">9.09</td><td class="colsep0 rowsep0" align="char" char=".">8.76</td><td class="colsep0 rowsep0" align="char" char=".">7.14</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MOLM13 (FLT3<sup>ITD</sup>)</td><td class="colsep0 rowsep0" align="char" char=".">82.30</td><td class="colsep0 rowsep0" align="char" char=".">16.95</td><td class="colsep0 rowsep0" align="char" char=".">84.60</td><td class="colsep0 rowsep0" align="char" char=".">5.44</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MOLM13<sup>SR</sup> (FLT3<sup>ITD+D835Y</sup>)</td><td class="colsep0 rowsep0" align="char" char=".">70.86</td><td class="colsep0 rowsep0" align="char" char=".">8.35</td><td class="colsep0 rowsep0" align="char" char=".">5.16</td><td class="colsep0 rowsep0" align="char" char=".">7.57</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t7fn1"><div class="footnote" id="t7fn1"><sup>a</sup><p class="last">MOLM13, MOLM13SR, and U937 cells were treated with <b>28</b>, palbocilib, and sorafenib at 1 μM for 48 h, individually. The treated cells were stained with Annexin V/Sytox Green and analyzed by flow cytometry for apoptosis. Data represent percentage of apoptotic cells in total cells analyzed.</p></div></div></div><div class="NLM_p">To understand the structural properties that result in the differential activities against FLT3 mutants of sorafenib and <b>28</b>, respectively, we modeled the binding modes of the two compounds to FLT3, the conformation of the activation loop (DFG-in, DFG-out),<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> and key FLT3 activation mutations and how they affect the activation loop conformation. The activation loop in a kinase is inherently flexible and capable of multiple conformations. For this analysis, we focused on two representative conformations, DFG-in as the active conformation and DFG-out as the inactive conformation. Previously published binding data suggest that sorafenib is a type II kinase inhibitor that preferentially binds to the inactive DFG-out kinase conformation.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> In this sorafenib–FLT3 model, the phenolic ring of sorafenib would form a perpendicular aromatic–aromatic interaction with F830 in the DFG motif (Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a>c), which is one of the key interactions with the protein. This interaction would not be possible in the active DFG-in kinase conformation. In this inactive, DFG-out conformation, FLT3 native residues D835 and Y842 stabilize this conformation of the activation loop by forming hydrogen bonds with the main chain amide group S838 and side chain D811, respectively (Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a>a). Thus, replacement of either residue might destabilize the inactive conformation of the activation loop, which would then be expected to hinder the binding of sorafenib.</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/medium/jm-2014-00118j_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Computational models of binding modes and activation loop conformations of inactive DFG-out and active DFG-in activation loop conformations.<a onclick="showRef(event, 'ref14 ref23'); return false;" href="javascript:void(0);" class="ref ref14 ref23">(14, 23)</a> (a) DFG-out kinase with the activiation loop (green) in the inactive conformation stabilized by intramolecular hydrogen bonds. (b) DFG-in kinase with the activation loop (green) in the active DFG-in conformation. (c) DFG-out kinase with sorafenib model showing type 2 binding mode. (d) DFG-in kinase with <b>28</b> model showing type I binding mode.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500118j&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In contrast, <b>28</b> is a type I kinase inhibitor and preferentially binds to the active DFG-in conformations (Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a>d) and maintains activity against those activation loop single point mutations.</div><div class="NLM_p">The pharmacokinetic profiles of <b>28</b> were investigated in rat, dog, and cynomolgus monkey (Table <a class="ref showTableEvent internalNav" data-ID="tbl8" href="#tbl8">8</a>). Plasma elimination half-lives following intravenous dosing range from 1.7 h in dog to 5.6 h in monkey. Steady state volumes of distribution values (<i>V</i><sub>ss</sub>) were fairly constant across species, with values ranging from 1.8 L/kg in rat to 3.8 L/kg in cynomolgus monkey. The plasma clearance in rat was low relative to hepatic blood flow, while it was moderate to high in dog and moderate in cynomolgus monkey. In all species, the compound showed moderate to high oral bioavailability following gavage administration in solution formulation. This preclinical pharmacokinetic profile was the basis for predicting, using in vitro–in vivo extrapolation methods, human oral clearance of 0.21 L/h/kg and a terminal phase half-life of 3.3 h, a profile consistent with twice-daily dosing.</div><div class="NLM_table-wrap" id="tbl8"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 8. Pharmacokinetic Parameters of <b>28</b> in Rat, Dog, and Cynomolgus Monkey</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="center" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">species</th><th class="colsep0 rowsep0" align="center" char=".">CL (L/h/kg)</th><th class="colsep0 rowsep0" align="center" char="."><i>V</i><sub>ss</sub> (L/kg)</th><th class="colsep0 rowsep0" align="center" char="."><i>T</i><sub>1/2,<i>z</i></sub> (h)</th><th class="colsep0 rowsep0" align="center"><i>F</i> (%)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">rat<a class="ref internalNav" href="#t8fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="char" char=".">0.39</td><td class="colsep0 rowsep0" align="char" char=".">1.8</td><td class="colsep0 rowsep0" align="char" char=".">5.5</td><td class="colsep0 rowsep0" align="center">75</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">dog<a class="ref internalNav" href="#t8fn2" aria-label="b">b</a>,<a class="ref internalNav" href="#t8fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="char" char=".">1.41</td><td class="colsep0 rowsep0" align="char" char=".">2.6</td><td class="colsep0 rowsep0" align="char" char=".">4.1</td><td class="colsep0 rowsep0" align="center">ca. 100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">monkey<a class="ref internalNav" href="#t8fn2" aria-label="b">b</a>,<a class="ref internalNav" href="#t8fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="char" char=".">1.42</td><td class="colsep0 rowsep0" align="char" char=".">3.2</td><td class="colsep0 rowsep0" align="char" char=".">6.6</td><td class="colsep0 rowsep0" align="center">44</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t8fn1"><div class="footnote" id="t8fn1"><sup>a</sup><p class="last">Vehicle used in iv and po studies in rat: water/acetate/tween at pH4, 96/3/1 (v/v/v).</p></div><div class="footnote" id="t8fn2"><sup>b</sup><p class="last">Vehicle used in iv studies in dog and monkey: 40% PG aqueous solution at pH2.</p></div><div class="footnote" id="t8fn3"><sup>c</sup><p class="last">Vehicle used in po studies in dog and monkey: 20% captisol aqueous solution.</p></div></div></div><div class="NLM_p">The combination of potent tumor cell growth inhibition in vitro and high exposure led to significant antitumor effects in vivo. NCR nude mice bearing MOLM13 tumors were treated with <b>28</b> orally at doses of 25, 50, 75, and 150 mg/kg once daily for 8 days. Dose-dependent inhibition of tumor growth was observed with 100% tumor growth inhibition (TGI) at 150 mg/kg dose (Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a>). The calculated ED<sub>50</sub> was 22 mg/kg, and the corresponding AUC was determined to be 28 μM·h from terminal PK analyzed from plasma harvested after the final dose (<a class="ref internalNav" href="#notes-1" aria-label="Supporting Information Figure 1">Supporting Information Figure 1</a>). Body weight losses were within an acceptable range (10%) throughout the study (<a class="ref internalNav" href="#notes-1" aria-label="Supporting Information Figure 2">Supporting Information Figure 2</a>), and <b>28</b> was well tolerated. In addition, the phosphorylation of STAT5, which is an important downstream signaling protein of FLT3 signaling, and the phosphorylation of Rb, a downstream protein of the CDK4 pathway, were also significantly reduced in MOLM13 tumor tissues obtained from the <b>28</b> treated group compared with the vehicle group (Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a>), indicating that the observed efficacy in vivo was consistent with FLT3 and CDK4 signaling inhibition.</div><figure id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/medium/jm-2014-00118j_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0008.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. In vivo tumor growth inhibition for <b>28</b>. NCR nude mice bearing established human MOLM13 AML xenografts were dosed orally with either vehicle or <b>28</b> at 25, 50, 75, and 150 mg/kg (day 14 TGI = 100%, <i>p</i> < 0.0001) once daily from day 5 to day 13.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500118j&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/medium/jm-2014-00118j_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0009.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Compound <b>28</b> inhibited STAT5 (a) and Rb (b) phosphorylation in MOLM13 subcutaneous tumors. MOLM13 tumor bearing mice were dosed by oral administration of <b>28</b> at 50, 75, or 150 mg/kg, and tumors were harvested at 3, 6, and 24 h after treatment. Tumors lysates were prepared, and the level of pSTAT5 (upper panel) or pRb (lower panel) was determined.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> Unbound plasma levels of <b>28</b> (μM) same time points are shown. Data are presented as mean ± SEM, <i>n</i> = 3 except for vehicle group (<i>n</i> = 6).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500118j&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i11">Synthetic Chemistry</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_46242" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_46242" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The general approach to the pyrido[4′,3′:4,5]pyrrolo[2,3-<i>d</i>]pyrimidine series of dual inhibitors (Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a>) fashioned the key C–N bonds using nucleophilic aromatic substitution and Hartwig–Buchwald amination sequences and established the C4a–C4b heterobiaryl bond through either Negishi or Suzuki cross-coupling (Scheme <a class="ref internalNav" href="#sch1" aria-label="1">1</a>). Early variants of this strategy generated bromopyrimidine Suzuki substrates <b>31a</b>–<b>e</b> in a two-step substitution sequence beginning with primary aliphatic amines of interest and commercially available 5-bromo-2,4-dichloropyrimdine <b>29</b>. Variation of the amine substituent at this juncture enabled broad characterization of SAR trends associated with the cycloalkane sugar-pocket group displayed at N9 of the tricyclic core (Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>, <b>8</b>–<b>14</b>).<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> The N9–C8a bond establishing the fused pyrrole ring of the core was formed by intramolecular Hartwig–Buchwald amination of heterobiaryl cross-coupling products <b>32a</b>–<b>e</b>, with subsequent installation of the heteroarene substituent at N1′ by intermolecular amination of haloheteroarenes <b>34</b>. Piperazine derivatives (e.g., lead compound <b>1</b>) were liberated upon acidic cleavage of the <i>t</i>-butylcarbamoyl group, and alkylation or acylation of the resulting basic nitrogen function enabled the preparation of an array of higher analogues (e.g., <b>23</b>).</div><figure id="fig10" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/medium/jm-2014-00118j_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0010.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Numbering convention used for the pyrido[4′,3′:4,5]pyrrolo[2,3-<i>d</i>]pyrimidine core.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500118j&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="sch1" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/medium/jm-2014-00118j_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0011.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. First-Generation Synthetic Approach to the Pyrido[4′,3′:4,5]pyrrolo[2,3-<i>d</i>]pyrimidine Dual Inhibitors<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500118j&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) RNH<sub>2</sub>, DIPEA, reflux; (b) NH<sub>4</sub>OH, <i>i</i>-PrOH, 120 °C; (c) PdCl<sub>2</sub>(PPh<sub>3</sub>)<sub>2</sub>, Na<sub>2</sub>CO<sub>3</sub>, 1,4-dioxane, 120–150 °C; (d) Pd<sub>2</sub>dba<sub>3</sub>, XantPhos, <i>t</i>-BuONa, 1,4-dioxane, 150 °C; (e) Pd<sub>2</sub>dba<sub>3</sub>, Xantphos, <i>t</i>-BuONa, 1,4-dioxane, 120 °C; (f) TFA/CH<sub>2</sub>Cl<sub>2</sub>.</p></p></figure><div class="NLM_p">Our discovery efforts ultimately led us toward extensive characterization of SAR trends associated with the fused pyridine ring of the tricyclic core, with particular concentration in studying the effects of substitution at C8. We found it most expedient to prepare many of the substituted pyridine analogues via interconversions between pyridone derivatives (Scheme <a class="ref internalNav" href="#sch2" aria-label="2">2</a>). The parent pyridone core <b>38</b> could be readily obtained from methoxypyridine <b>37</b> and thus from commercially available methoxypyridylboronic acid <b>35</b>. Certain analogues more easily accessed when the synthesis design was modified to employ a milder construction of the heterobiaryl C4a–C4b bond by Negishi coupling of iodopyrimidines <b>44a</b>,<b>b</b> (Scheme <a class="ref internalNav" href="#sch3" aria-label="3">3</a>) with suitable arylzinc precursors. <i>ortho</i>-Fluoropyridine <b>2</b> could thus be prepared by directed lithiation/transmetalation of iodobromofluoropyridine <b>45</b> (Scheme <a class="ref internalNav" href="#sch3" aria-label="3">3</a>). <i>ortho</i>-Cyano derivatives (e.g., <b>6</b>) were in turn generated from a common fluoropyridine intermediate <b>47</b> by the action of tetraethylammonium cyanide in hot DMF (Scheme <a class="ref internalNav" href="#sch3" aria-label="3">3</a>). Synthesis of pyridazine derivative <b>7</b> also proceeded by way of Negishi chemistry (Scheme <a class="ref internalNav" href="#sch4" aria-label="4">4</a>), with formation of the central ring via nucleophilic displacement.</div><figure id="sch2" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/medium/jm-2014-00118j_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0012.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of <i>ortho</i>-Substituted Pyridine Analogues via Pyridone <b>38</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500118j&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) <b>31a</b>, PdCl<sub>2</sub>(PPh<sub>3</sub>)<sub>2</sub>, Na<sub>2</sub>CO<sub>3</sub>, 1,4-dioxane, 120 °C; (b) Pd<sub>2</sub>dba<sub>3</sub>, XantPhos, <i>t</i>-BuONa, 1,4-dioxane, 150 °C; (c) HCl, 1,4-dioxane/MeOH, 80 °C; (d) POCl<sub>3</sub>, 100 °C; (e) Pd<sub>2</sub>dba<sub>3</sub>, XantPhos, <i>t</i>-BuONa, 1,4-dioxane, 80 °C; (f) TFA/CH<sub>2</sub>Cl<sub>2</sub>; (g) Tf<sub>2</sub>NPh, NaH, DMF; (h) Fe(acac)<sub>3</sub>, MeMgBr, THF/NMP; (i) <b>34a</b>, Pd<sub>2</sub>dba<sub>3</sub>, XantPhos, <i>t</i>-BuONa, 1,4-dioxane, 120 °C; (j) TFA/CH<sub>2</sub>Cl<sub>2</sub>; (k) <b>34a</b>, Pd<sub>2</sub>dba<sub>3</sub>, XantPhos, <i>t</i>-BuONa, 1,4-dioxane, 120 °C; (l) TFA/CH<sub>2</sub>Cl<sub>2</sub>.</p></p></figure><figure id="sch3" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/medium/jm-2014-00118j_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0013.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Negishi Coupling Approach to Substituted Fused-Pyridine Derivatives <b>2</b> and <b>6</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500118j&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagent and conditions: (a) cyclopentylamine or <i>trans</i>-Me-cyclohexylamine, 1,4-dioxane or BuOH, 100 °C; (b) NIS, DMF or AcOH, 80 °C; (c) <i>i</i>-PrMgCl, ZnCl<sub>2</sub>; (d) <b>44a</b>, Pd(PPh<sub>3</sub>)<sub>4</sub>, THF, reflux; (e) X-phos, Cs<sub>2</sub>CO<sub>3</sub>, 1,4-dioxane, 120 °C; (f) <b>34a</b>, Pd<sub>2</sub>dba<sub>3</sub>, XantPhos, <i>t</i>-BuONa, 1,4-dioxane, 120 °C; (g) TFA/CH<sub>2</sub>Cl<sub>2</sub>; (h) NEt<sub>4</sub>CN, DMF, 150 °C; (i) <b>34a</b>, Pd<sub>2</sub>dba<sub>3</sub>, XantPhos, <i>t</i>-BuONa, 1,4-dioxane, 120 °C; (j) TFA/CH<sub>2</sub>Cl<sub>2</sub>.</p></p></figure><figure id="sch4" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/medium/jm-2014-00118j_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0014.jpeg" id="gr14" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Negishi Route to Fused Pyridazine Derivative <b>7</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500118j&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) LiTMP, ZnCl<sub>2</sub>; (b) <b>44a</b>, Pd(PPh<sub>3</sub>)<sub>4</sub>, THF, reflux; (c) NaH, THF, 150 °C; (d) PPTS, 180 °C; (e) PhNTf<sub>2</sub>, TEA, DMAP; (f) Pd(OAc)<sub>2</sub>, dppf, TEA, HCO<sub>2</sub>H, DMF; (g) <b>34a</b>, Pd<sub>2</sub>dba<sub>3</sub>, XantPhos, <i>t</i>-BuONa, 1,4-dioxane, 120 °C; (h) TFA/CH<sub>2</sub>Cl<sub>2</sub>.</p></p></figure><div class="NLM_p">Surveying the effects of varying the cycloalkane and fused-pyridine binding motifs led to identification of <b>33d</b> as the optimal scaffold, and our escalating demand for this material necessitated the development of a highly efficient and scaleable route from cost-effective precursors. Applying the Negishi methodology developed above, we were gratified to discover that the readily available feedstock 3-fluoropyridine <b>54</b> underwent high-yielding and facile cross-coupling with <b>44b</b> upon directed lithiation by LiTMP at cryogenic temperature followed by transmetalation to zinc bromide (Scheme <a class="ref internalNav" href="#sch5" aria-label="5">5</a>).<a onclick="showRef(event, 'ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref27 ref28">(27, 28)</a> Intramolecular displacement of fluoride by the pendant secondary amine function in the presence of LiHMDS completed the synthesis of <b>33d</b>. The optimized sequence enabled rapid preparation of kilogram quantities of this critical intermediate.</div><figure id="sch5" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/medium/jm-2014-00118j_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0015.jpeg" id="gr15" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Process Synthesis of Optimized Tricyclic Core <b>33d</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500118j&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) LiTMP, ZnBr<sub>2</sub>, −78 °C; (b) <b>44b</b>, Pd(PPh<sub>3</sub>)<sub>4</sub>, THF, reflux; (c) LiHMDS, NMP, 90 °C.</p></p></figure><div class="NLM_p">Exploring SAR trends associated with the N1′ pyridyl substituent and the piperazine ring (more generally, the distal solubilizing group) situated at its 5 position required us to prepare a wide array of haloheteroarene Hartwig–Buchwald substrates (<b>58a</b>–<b>d</b>, Scheme <a class="ref internalNav" href="#sch6" aria-label="6">6</a>); this was generally accomplished by arylating the corresponding secondary amines (e.g., <b>57</b>). Studies directed at optimizing substituents on the basic nitrogen in the 4-aminopiperidine series effected late-stage diversification by reductive amination of advanced ketones <b>60a</b>,<b>b</b> (illustrated for <b>17</b> and <b>22</b>, Scheme <a class="ref internalNav" href="#sch7" aria-label="7">7</a>).</div><figure id="sch6" data-index="16" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/medium/jm-2014-00118j_0016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0016.jpeg" id="gr16" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 6. Synthesis and Coupling of Haloheteroarene Amination Substrates<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500118j&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) Pd<sub>2</sub>dba<sub>3</sub>, XantPhos, <i>t</i>-BuONa or Cs<sub>2</sub>CO<sub>3</sub>, 1,4-dioxane or PhMe, 90–120 °C (for <b>58a</b>, <b>58c</b>–<b>d</b>), DIPEA, 1,4-dioxane, 80 °C (for <b>58b</b>,<b>e</b>); (b) <b>33d</b>, Pd<sub>2</sub>dba<sub>3</sub>, XantPhos, <i>t</i>-BuONa or Cs<sub>2</sub>CO<sub>3</sub>, 1,4-dioxane or PhMe, 100–120 °C; (c) TFA/CH<sub>2</sub>Cl<sub>2</sub>.</p></p></figure><figure id="sch7" data-index="17" class="article__inlineFigure"><h2 class="fig-label">Scheme 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/medium/jm-2014-00118j_0017.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0017.jpeg" id="gr17" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 7. Modular Approach to 4-Aminopiperidine Derivatives <b>17</b> and <b>22</b> via Late-Stage Reductive Amination<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500118j&amp;id=sch7"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) <b>33d</b>, Pd<sub>2</sub>dba<sub>3</sub>, XantPhos, <i>t</i>-BuONa, 1,4-dioxane; (b) 6N HCl (aq); (c) NaBH<sub>3</sub>(CN), NH<sub>4</sub>OAc, MeOH, 60 °C; (d) NaBH(OAc)<sub>3</sub>, H<sub>2</sub>CO (aq), CH<sub>2</sub>Cl<sub>2</sub>.</p></p></figure><div class="NLM_p">Derivatives featuring nonbasic distal solubilizing moieties often required the development of more involved, target-specific methodologies. Syntheses of representative compounds from the most important classes are outlined in Schemes <a class="ref internalNav" href="#sch8" aria-label="8">8</a>–<a class="ref internalNav" href="#sch10" aria-label="10">10</a>. The halopyridazine precursor <b>64</b> to aliphatic sulfone <b>24</b> was prepared from primary alcohol <b>61</b> by a straightforward amination/mesylation/SN2 methylsulfanation/oxidation sequence (Scheme <a class="ref internalNav" href="#sch8" aria-label="8">8</a>).</div><figure id="sch8" data-index="18" class="article__inlineFigure"><h2 class="fig-label">Scheme 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/medium/jm-2014-00118j_0018.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0018.jpeg" id="gr18" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 8. Synthesis of the Precursor to Aliphatic Sulfone <b>24</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500118j&amp;id=sch8"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) <b>56b</b>, 90 °C; (b) MsCl, TEA; (c) NaSMe, DMF; (d) oxone, MeOH; (e) <b>33d</b>, Pd<sub>2</sub>dba<sub>3</sub>, XantPhos, <i>t</i>-BuONa, 1,4-dioxane, 120 °C.</p></p></figure><div class="NLM_p">Arylation of the spirocyclic piperidines <b>65</b> and <b>68</b> gave the corresponding amidation substrates <b>66</b>, <b>69a</b>, and <b>69b</b> used to prepare compounds <b>25</b>–<b>27</b> (Scheme <a class="ref internalNav" href="#sch9" aria-label="9">9</a>).</div><figure id="sch9" data-index="19" class="article__inlineFigure"><h2 class="fig-label">Scheme 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/medium/jm-2014-00118j_0019.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0019.jpeg" id="gr19" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 9. Synthesis of the Precursor to Spirocyclic Sulfone and Spirocyclic Lactams<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0019.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500118j&amp;id=sch9"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) <b>56b</b>, DIPEA, 1,4-dioxane, 90 °C; (b) <b>33d</b>, Pd<sub>2</sub>dba<sub>3</sub>, XantPhos, <i>t</i>-BuONa, 1,4-dioxane, 150 °C; (c) HCl, 1,4-dioxane; (d) <b>56a</b> or <b>56d</b>, XantPhos, Pd<sub>2</sub>dba<sub>3</sub>, <i>t</i>-BuONa, PhMe; (e) <b>33d</b>, Pd<sub>2</sub>dba<sub>3</sub>, Xantphos, <i>t</i>-BuONa, 1,4-dioxane, 150 °C.</p></p></figure><div class="NLM_p">Finally, the requisite amination substrate for the synthesis of <b>28</b> could be prepared by Boc protection of commercially available chlorotetrahydronaphthyridine <b>70</b> (Scheme <a class="ref internalNav" href="#sch10" aria-label="10">10</a>). Hartwig–Buchwald coupling of <b>71</b> with <b>33d</b> and subsequent Boc deprotection both occurred in virtually quantitative yields under optimized conditions. Acetoxyacetylation and acetate saponification completed the synthesis, giving <b>28</b> in high overall yield (97% from <b>70</b>).</div><figure id="sch10" data-index="20" class="article__inlineFigure"><h2 class="fig-label">Scheme 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/medium/jm-2014-00118j_0020.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0020.jpeg" id="gr20" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 10. Synthesis of <b>28</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0020.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500118j&amp;id=sch10"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) (Boc)<sub>2</sub>O, DIPEA, DCM; (b) <b>33d</b>, Pd<sub>2</sub>dba<sub>3</sub>, XantPhos, <i>t</i>-BuONa, 1,4-dioxane, 100 °C; (c) HCl, MeOH/H<sub>2</sub>O; NaOH; (d) DIPEA, CHCl<sub>3</sub>; (e) MeONa, DCM/MeOH.</p></p></figure></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i22">Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_00808" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_00808" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">From lead compound <b>1</b>, a combination of structure-based drug design, traditional, and novel medicinal chemistry approaches led to the discovery of <b>28</b>, a FLT3/CDK4 dual inhibitor with preferential affinity for the activated state of FLT3. Compound <b>28</b> shows potent in vitro activities against CDK4 and FLT3, including several key FLT3 mutants, and displays potent antitumor efficacy mediated by inhibition of FLT3 and Rb phosphorylation in vivo. Compound <b>28</b> possesses excellent cross-species pharmacokinetic properties in multiple preclinical models. Compound <b>28</b> may improve the durability of clinical response in the treatment of AML by simultaneously modulating two key targets.</div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i23">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_64424" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_64424" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> General Chemistry</h3><div class="NLM_p">All reactions were conducted under an inert gas atmosphere (nitrogen or argon) using a Teflon-coated magnetic stirbar at the temperature indicated. Commercial reagents and anhydrous solvents were used without further purification. Removal of solvents was conducted by using a rotary evaporator, and residual solvent was removed from nonvolatile compounds using a vacuum manifold maintained at approximately 1 Torr. All yields reported are isolated yields. Preparative reversed-phase high pressure liquid chromatography (RP-HPLC) was performed using an Agilent 1100 series HPLC and Phenomenex Gemini C18 column (5 μm, 100 mm × 30 mm i.d.), eluting with a binary solvent system A and B using a gradient elusion [A, H<sub>2</sub>O with 0.1% trifluoroacetic acid (TFA); B, CH<sub>3</sub>CN with 0.1% TFA] with UV detection at 220 nm. All bioassayed compounds were purified to ≥95% purity as determined by a Agilent 1100 series HPLC with UV detection at 220 nm using the following method: Zorbax SB-C8 column (3.5 μm, 150 mm × 4.6 mm i.d.); mobile phase, A = H<sub>2</sub>O with 0.1% TFA, B = CH<sub>3</sub>CN with 0.1% TFA; gradient, 5–95% B (0.0–15.0 min); flow rate, 1.5 mL/min. Low-resolution mass spectral (MS) data were determined on an Agilent 1100 series LCMS with UV detection at 254 nm and a low-resolution electrospray mode (ESI). High-resolution mass spectra (HRMS) were obtained on an Agilent 6510 Q-TOF MS with an Agilent 1200 LC on the front end. <sup>1</sup>H NMR spectra were obtained on a Bruker Avance III 500 (500 MHz) or Bruker Avance II 400 (400 MHz) spectrometer. Chemical shifts (δ) are reported in parts per million (ppm) relative to residual undeuterated solvent as an internal reference. The following abbreviations were used to explain the multiplicities: s = single, d = doublet, t = triplet, q = quartet, dd = doublet of doublets, dt = doublet of triplets, m = multiplet, br = broad.</div><div id="sec5_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> 5-Bromo-2-chloro-<i>N</i>-cyclopentylpyrimidin-4-amine (<b>30a</b>)</h4><div class="NLM_p last">To a solution of 5-bromo-2,4-dichloropyrimidine (45.6 g, 200 mmol) in dioxane (400 mL) was added <i>N</i>-cyclopentylamine (20.4 g, 240 mmol) at room temperature. The mixture thus obtained was stirred at room temperature for 6 h. The reaction mixture was then diluted with ethyl acetate, washed with brine, and dried over MgSO<sub>4</sub>. The solvent was evaporated to give compound <b>30a</b> as light-yellow solid (56 g, 100% yield), which was used in next step without further purification. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 8.23 (1H, s), 7.37 (1H, d, <i>J</i> = 7.3 Hz), 4.31 (1H, m), 1.92 (2H, m), 1.71 (2H, m), 1.53–1.59 (4H, m). MS (ESI) <i>m</i>/<i>z</i>: 276.0 [M + H]<sup>+</sup>.</div></div><div id="sec5_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> 5-Bromo-<i>N</i><sup>4</sup>-cyclopentylpyrimidine-2,4-diamine (<b>31a</b>)</h4><div class="NLM_p last">A solution of <b>30a</b> (45 g, 200 mmol) in 28% NH<sub>4</sub>OH/2-propanol (1/1, 400 mL) was heated at 120 °C in sealed tube for 22 h. The product was extracted with dichloromethane, and the organic layers were washed with brine and dried. The solvent was evaporated, and the residue was purified by flash chromatography on silica gel eluting with 25% ethyl acetate in hexanes to give compound <b>31a</b> as white solid (34 g, 66% yield). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 7.77 (1 H, s), 6.19 (2 H, br s), 6.12 (1 H, d, <i>J</i> = 7.3 Hz), 4.33 (1 H, m), 1.90 (2 H, m), 1.69 (2 H, m), 1.49–1.55 (4 H, m). MS (ESI) <i>m</i>/<i>z</i>: 257.0 [M + H]<sup>+</sup>.</div></div><div id="sec5_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> 5-(3-Chloropyridin-4-yl)-<i>N</i><sup>4</sup>-cyclopentylpyrimidine-2,4-diamine (<b>32a</b>)</h4><div class="NLM_p last">To a solution of <b>31a</b> (2.57 g, 10.0 mmol) in dioxane (75 mL) were added 3-chrolopyridine-4-boronic acid (4.72 g, 10.0 mmol), PdCl<sub>2</sub>(PPh<sub>3</sub>)<sub>2</sub> (702 mg, 1.0 mmol), and sodium carbonate (3.82 g, 36 mmol, in 36 mL of water). The mixture thus obtained was purged with N<sub>2</sub> for 10 min and heated at 120 °C in a sealed tube for 22 h. The reaction mixture was diluted with water, and the product was extracted with chloroform. The organic layers were dried over MgSO<sub>4</sub> and concentrated. The residue was purified by flash chromatography on silica gel, eluting with 2.5% methanol in dichloromethane to give compound <b>32a</b> as white solid (2.31g, 80% yield). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 8.64 (1 H, s), 8.50 (1 H, d, <i>J</i> = 4.9 Hz), 7.51 (1 H, s), 7.34 (1 H, d, <i>J</i> = 4.9 Hz), 6.21 (2 H, br s), 5.97 (1 H, d, <i>J</i> = 7.4 Hz), 4.43 (1 H, p, <i>J</i> = 7.4 Hz), 1.84 (2 H, m), 1.62 (2 H, m), 1.40–1.49 (4 H, m). MS (ESI) <i>m</i>/<i>z</i>: 290.0 [M + H]<sup>+</sup>.</div></div><div id="sec5_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> 9-Cyclopentyl-9<i>H</i>-pyrido[4′,3′:4,5]pyrrolo[2,3-<i>d</i>]pyrimidin-2-amine (<b>33a</b>)</h4><div class="NLM_p last">To a solution of <b>32a</b> (2.3 g, 7.9 mmol) in 1,4-dioxane (60 mL) were added Pd<sub>2</sub>dba<sub>3</sub> (368 mg, 0.4 mmol), XantPhos (696 mg, 1.2 mmol), and sodium <i>t</i>-butoxide (1.15 g, 12 mmol). The mixture thus obtained was heated at 150 °C under microwave irradiation for 3 h. The reaction mixture was passed through a short pack silica gel column and concentrated. The residue was purified by flash chromatography on silica gel eluting with 2% methanol in dichloromethane to give compound <b>33a</b> as white solid (1.71 g, 85% yield). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 9.05 (1 H, s), 8.87 (1 H, br s), 8.37 (1 H, d, <i>J</i> = 4.9 Hz), 7.93 (1 H, d, <i>J</i> = 5.1 Hz), 6.97 (2 H, br s), 5.33 (1 H, quin, <i>J</i> = 8.6 Hz), 2.27 (2 H, m), 2.04 (4 H, m), 1.76 (2 H, m). HRMS (ESI) <i>m</i>/<i>z</i>: calculated for [M + H]<sup>+</sup> 254.1400, found 254.1406.</div></div><div id="sec5_1_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> 4-(4-Chloro-pyrid-2-yl)-piperidine-1-carboxylic Acid <i>t</i>-Butyl Ester (<b>34a</b>)</h4><div class="NLM_p last">To a solution of 5-bromo-2-chloropyridine (11.54 g, 60 mmol) in toluene (300 mL) were added <i>t</i>-butyl 1-piperazinecarboxylate (11.18 g, 60 mmol), sodium <i>t</i>-butoxide (8.64 g, 90 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (1.10 g, 1.20 mmol), and XantPhos (2.08 g, 3.6 mmol). The mixture was evacuated and purged with argon (3 cycles), then heated at 100 °C for 5 h. After the reaction went to completion, the mixture was cooled to room temperature, diluted with ethyl acetate (1200 mL), and washed with water (300 mL). The organic solution was concentrated under pressure, and the residue was purified by flash chromatography on silica gel eluting with 17% ethyl acetate in hexane to give compound <b>34a</b> as a light-yellow solid (14.6 g, 82% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm 8.02 (1 H, s), 7.19–7.18 (2 H, m), 3.59 (4 H, dd, <i>J</i> = 12 Hz, <i>J</i> = 4 Hz), 3.14 (4 H, dd, <i>J</i> = 12 Hz, <i>J</i> = 4 Hz), 1.49 (9 H, s). MS (ESI) <i>m</i>/<i>z</i>: 298.1 [M + H]<sup>+</sup>.</div></div><div id="sec5_1_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> 9-Cyclopentyl-<i>N</i>-(5-(1-piperazinyl)-2-pyridinyl)-9<i>H</i>-pyrido[4′,3′:4,5]pyrrolo[2,3-<i>d</i>]pyrimidin-2-amine (<b>1</b>)</h4><div class="NLM_p last">To a solution of <b>33a</b> (1.6 g, 6.3 mmol) in dioxane (60 mL) were added <b>34a</b> (2.3 g, 7.6 mmol), Pd<sub>2</sub>dba<sub>3</sub> (293 mg, 0.32 mmol), XantPhos (370 mg, 0.64 mmol), and sodium <i>t</i>-butoxide (908 mg, 9.45 mmol). The mixture thus obtained was heated at 150 °C under microwave irradiation for 1 h. The reaction mixture was passed through a short pack silica gel column and concentrated to give <i>t</i>-butyl 4-(6-((9-cyclopentyl-9<i>H</i>-pyrido[4′,3′:4,5]pyrrolo[2,3-<i>d</i>]pyrimidin-2-yl)amino)-3-pyridinyl)-1-piperazinecarboxylate as a yellow solid (3.28 g), which was then treated with TFA/DCM (60 mL, 1:1) at room temperature for 30 min. The reaction mixture was concentrated, and the residue was purified by chromatography on silica gel eluting with CH<sub>2</sub>Cl<sub>2</sub>/MeOH/NH<sub>4</sub>OH (200:10:1) to give compound (<b>1</b>) as an off-white solid (2.60 g, 98% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 9.86 (1 H, s), 9.28 (1 H, s), 9.01 (1 H, s), 8.68 (1 H, br s), 8.45 (1 H, d, <i>J</i> = 4 Hz), 8.23 (1 H, d, <i>J</i> = 8 Hz), 8.10 (1 H, d, <i>J</i> = 4 Hz), 8.04 (1 H, d, <i>J</i> = 8 Hz), 7.54 (1 H, dd, <i>J</i> = 8 Hz, <i>J</i> = 4 Hz), 5.43 (1 H, quin, <i>J</i> = 8 Hz), 3.35 (4 H, m), 3.26 (4 H, m), 2.40 (2 H, m), 2.03–2.10 (4 H, m), 1.78 (2 H, m). HRMS (ESI) <i>m</i>/<i>z</i>: calculated for [M + H]<sup>+</sup> 415.2354, found 415.2350.</div></div><div id="sec5_1_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> <i>N</i><sup>4</sup>-Cyclopentyl-5-iodo-2,4-pyrimidinediamine (<b>44a</b>)</h4><div class="NLM_p last">To a stirred solution of compound <b>43a</b> (25.5 g, 143 mmol) in DMF (120 mL) was added <i>N</i>-iodosuccinimide (32 g, 143 mmol) in two portions at rt. The resulting mixture was stirred at rt for 2 h. Upon workup, the mixture was poured into a mixture of ice and saturated aqueous sodium carbonate with some sodium sulfite, then extracted with ethyl acetate (2×). The combined organics were washed with saturated aqueous sodium carbonate (3×), dried over sodium sulfate, and concentrated in vacuo. The residue was subjected to flash column chromatography on silica gel eluting with 1% methanol in DCM to give compound <b>44a</b> as a light-yellow solid (39 g, 73% yield). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 7.89 (1 H, s), 6.18 (2 H, s), 5.63 (1 H, d, <i>J</i> = 7.6 Hz), 4.30 (1 H, p, <i>J</i> = 7.4 Hz), 1.84–1.96 (2 H, m), 1.61–1.74 (2 H, m), 1.42–1.58 (4 H, m). MS (ESI) <i>m</i>/<i>z</i>: 305.0 [M + H]<sup>+</sup>.</div></div><div id="sec5_1_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> 5-Iodo-<i>N</i><sup>4</sup>-((1<i>r</i>,4<i>r</i>)-4-methylcyclohexyl)pyridine-2,4-diamine (<b>44b</b>)</h4><div class="NLM_p last">4-Chloropyrimidine-2-amine (1000 g, 7.72 mol, 1.0 equiv), <i>trans</i>-4-methylcyclohexylamine hydrochloride (1500 g, 10.03 mol, 1.3 equiv) and TEA (3.23 L, 23.2 mol, 3.0 equiv) were mixed together in <i>n</i>-butanol (8 L). The reaction mixture was heated at reflux for 36 h and monitored using LCMS. Upon completion, the reaction mixture was cooled to room temperature, diluted with water (8 L), and extracted with ethyl acetate (2 × 10 L). The organic layers were combined, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated under reduced pressure to give <i>N</i><sup>4</sup>-((1<i>r</i>,4<i>r</i>)-4-methylcyclohexyl)pyridine-2,4-diamine (<b>43b</b>) (1770 g), which was used in the next step without further purification. Compound <b>43b</b> (1770 g, 8.58 mol, 1.0 equiv) was dissolved in anhydrous DMF (8 L). To this solution under N<sub>2</sub> atmosphere at 10 °C was added iodosuccinmide (1.93 g, 8.58 mol, 1.0 equiv) in portions over 10 min. Upon completion of the addition, the reaction mixture was stirred at room temperature for 2 h. The reaction was monitored using LCMS. Upon completion, the reaction mixture was cooled using an ice bath, quenched with saturated aqueous sodium carbonate (5 L), and extracted with ethyl acetate (2 × 15 L). The combined organic extracts were washed with saturated aqueous sodium carbonate (2 × 5 L), water (3 × 2 L), dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated under reduced pressure. The residue was purified using column chromatography on silica gel eluting with 25–40% ethyl acetate in hexanes to provide compound <b>44b</b> (1470 g, 57% over two steps). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 7.86 (1H, s), 6.16 (2H, br s), 5.48 (1H, d, <i>J</i> = 8.1 Hz), 3.85 (1H, m), 1.78 (2H, d, <i>J</i> = 12.3 Hz), 1.66 (2H, d, <i>J</i> = 12.3 Hz), 1.41–1.27 (3H, m), 0.98 (1H, dd, <i>J</i> = 12.9, 2.7 Hz), 0.85 (3H, d, <i>J</i> = 7.2 Hz). MS <i>m</i>/<i>z</i>: 333.0 [M + H]<sup>+</sup>.</div></div><div id="sec5_1_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> 5-(3-Bromo-2-fluoro-4-pyridinyl)-<i>N</i><sup>4</sup>-cyclopentyl-2,4-pyrimidinediamine (<b>46</b>)</h4><div class="NLM_p last">In a dry 3 neck 2 L flask equipped with a dry addition funnel, thermometer, and stir bar was added 3-bromo-2-fluoro-4-iodopyridine (83.4 g, 276 mmol) followed by 300 mL of anhydrous THF under an atmosphere of nitrogen. The solution was cooled to −70 °C in a 2-propanol/dry ice bath. A solution of isopropylmagnesium chloride 2.0 M in diethyl ether (145 mL, 290 mmol) was added dropwise over a period of 30 min. The solution was then stirred for 30 min before zinc(II) chloride (0.5 M in THF, 276 mL, 138 mmol) was cannulated in. The solution was warmed to room temperature and stirred for 1 h. The addition funnel was replaced with a reflux condenser and <b>44a</b> (28.00 g, 92.1 mmol) was added, followed by Pd(PPh<sub>3</sub>)<sub>4</sub> (5.32 g, 4.60 mmol). The solution was heated overnight at a gentle reflux. After concentrating the solution to 1/10th of the volume under vacuum, it was cooled in an ice bath. To this was added 100 mL of cold saturated NH<sub>4</sub>Cl, followed by 500 mL of water. Then 500 mL of 12% 2-propanol/DCM was then added and the solution was stirred at room temperature for 1 h before being filtered through filter paper. The filter cake was washed in succession with water, DCM, and 12% 2-propanol/DCM. The filtrate was partitioned in a separation funnel, and the aqueous layer was washed with 12% 2-propanol/DCM. The organics were dried over MgSO<sub>4</sub> and then concentrated under vacuum. The residue obtained was partially dissolved in DCM then sonicated and filtered to give compound <b>46</b> (26.5 g, 82% yield) as a light-yellow solid. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 8.18 (1 H, d, <i>J</i> = 4.9 Hz), 7.52 (1 H, s), 7.26 (1 H, dd, <i>J</i> = 4.9, 0.7 Hz), 6.34 (2 H, s), 6.12 (1 H, d, <i>J</i> = 7.6 Hz), 4.42 (1 H, p, <i>J</i> = 7.4 Hz), 1.76–1.96 (2 H, m), 1.55–1.68 (2 H, m), 1.31–1.53 (4 H, m)). MS (ESI) <i>m</i>/<i>z</i>: 352.0 [M + H]<sup>+</sup>.</div></div><div id="sec5_1_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> 9-Cyclopentyl-8-fluoro-9<i>H</i>-pyrido[4′,3′:4,5]pyrrolo[2,3-<i>d</i>]pyrimidin-2-amine (<b>47</b>)</h4><div class="NLM_p last">Compound <b>46</b> (0.143 g, 0.41 mmol), and cesium carbonate (0.40 g, 1.2 mmol) were combined in anhydrous 1,4-dioxane (5 mL). Nitrogen was briefly bubbled through the solution before adding Pd<sub>2</sub>(dba)<sub>3</sub> (0.037 g, 0.041 mmol) and XantPhos (0.047 g, 0.081 mmol). The solution was heated at 120 °C under microwave irradiation for 1 h. The solution was purified with flash column chromatography on silica gel eluting with DCM/MeOH/NH<sub>4</sub>OH (200:10:1) to give compound <b>47</b> (94 mg, 85% yield) as a light-brownish solid. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 9.10 (1 H, s), 7.94 (1 H, dd, <i>J</i> = 5.2, 1.5 Hz), 7.91 (1 H, dd, <i>J</i> = 5.1, 2.9 Hz), 7.10 (2 H, br s), 5.38–5.49 (1 H, m), 2.10–2.23 (2 H, m), 1.93–2.08 (4 H, m), 1.64–1.79 (2 H, m). MS (ESI) <i>m</i>/<i>z</i>: 272.0 [M + H]<sup>+</sup>.</div></div><div id="sec5_1_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> 9-Cyclopentyl-8-fluoro-<i>N</i>-(5-(1-piperazinyl)-2-pyridinyl)-9<i>H</i>-pyrido[4′,3′:4,5]pyrrolo[2,3-<i>d</i>]pyrimidin-2-amine (<b>2</b>)</h4><div class="NLM_p last">Compound <b>2</b> was prepared from <b>47</b> using chemistry similar to that described in the synthesis of <b>1</b>. <sup>1</sup>H NMR (500 MHz, CD<sub>3</sub>OD) δ ppm 9.46 (1 H, s), 8.10–8.15 (3 H, m), 8.05 (1 H, d, <i>J</i> = 2.9 Hz), 7.64 (1 H, d, <i>J</i> = 9.5 Hz), 5.20 (1 H, m), 3.51–3.57 (4 H, m), 3.42–3.51 (4 H, m), 2.40–2.50 (2 H, m), 2.10–2.25 (4 H, m), 1.80–1.90 (2H, m). HRMS (ESI) <i>m</i>/<i>z</i>: calculated for [M + H]<sup>+</sup> 433.2260, found 433.2258.</div></div><div id="sec5_1_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> 8-Chloro-9-cyclopentyl-9<i>H</i>-pyrido[4′,3′:4,5]pyrrolo[2,3-<i>d</i>]pyrimidin-2-amine (<b>39</b>)</h4><div class="NLM_p last">Compound <b>39</b> was prepared from 2-chloro-3-fluoropyridine using chemistry similar to that described in the synthesis of <b>47.</b><sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 9.11 (1 H, s), 8.14 (1 H, d, <i>J</i> = 4.9 Hz), 7.98 (1 H, d, <i>J</i> = 4.9 Hz), 7.07 (2 H, br s), 5.90 (1 H, quin, <i>J</i> = 8.7 Hz), 2.44–2.49 (2 H, m), 2.04–2.13 (2 H, m), 1.95–2.03 (2 H, m), 1.63–1.73 (2 H, m). MS (ESI) <i>m</i>/<i>z</i>: 2.88.1 [M + H]<sup>+</sup>.</div></div><div id="sec5_1_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> 8-Chloro-9-cyclopentyl-<i>N</i>-(5-(1-piperazinyl)-2-pyridinyl)-9<i>H</i>-pyrido[4′,3′:4,5]-pyrrolo[2,3-<i>d</i>]pyrimidin-2-amine (<b>3</b>)</h4><div class="NLM_p last">Compound <b>3</b> was prepared using chemistry similar to that described in the synthesis of <b>1</b>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 10.56 (1 H, br s), 9.42 (1 H, s), 8.87 (2 H, br s), 8.28 (1 H, d, <i>J</i> = 4.9 Hz), 8.19 (1 H, d, <i>J</i> = 5.1 Hz), 8.09 (1 H, d, <i>J</i> = 2.9 Hz), 7.97 (1 H, d, <i>J</i> = 9.0 Hz), 7.75 (1 H, dd, <i>J</i> = 9.3, 2.0 Hz), 5.96–6.07 (2 H, m, <i>J</i> = 9.2, 8.9, 8.8, 8.8 Hz), 3.37–3.43 (4 H, m), 3.26–3.33 (4 H, m), 2.54–2.65 (2 H, m), 2.01–2.12 (4 H, m), 1.66–1.77 (2 H, m). HRMS (ESI) <i>m</i>/<i>z</i>: calculated for [M + H]<sup>+</sup> 449.1963, found 449.1963.</div></div><div id="sec5_1_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> 9-Cyclopentyl-8-methoxy-9<i>H</i>-pyrido[4′,3′:4,5]pyrrolo[2,3-<i>d</i>]pyrimidin-2-amine (<b>37</b>)</h4><div class="NLM_p last">Compound <b>37</b> was prepared from (3-chloro-2-methoxypyridin-4-yl)boronic acid using chemistry similar to that described in the synthesis of <b>33a</b>. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm 8.87 (1 H, s) 7.98 (1 H, d, <i>J</i> = 5.3 Hz), 7.42 (1 H, d, <i>J</i> = 5.3 Hz), 5.70 (1 H, t, <i>J</i> = 9.0 Hz), 5.21 (2 H, br s), 4.13 (3 H, s), 2.33–2.47 (2 H, m), 1.98–2.15 (4 H, m), 1.67–1.83 (2 H, m). MS (ESI) <i>m</i>/<i>z</i>: 284.0 [M + H]<sup>+</sup>.</div></div><div id="sec5_1_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> 2-Amino-9-cyclopentyl-7,9-dihydro-8<i>H</i>-pyrido[4′,3′:4,5]pyrrolo[2,3-<i>d</i>]pyrimidin-8-one (<b>38</b>)</h4><div class="NLM_p last">A solution of compound <b>37</b> (1.42 g, 5.0 mmol) was dissoved in conc HCl (10.4 mL, 125 mmol), and the solution thus obtained was heated at 140 °C under microwave irradiation for 1 h. The reaction mixture was neutrallized to pH 7–8 by addition of sodium carbonate, and the precepitate was collected by filtration, washed with water, and dried under high vacuum to give <b>38</b> as a yellow solid (1.35 g, 100% yield) that was used in the next step without further purification. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 11.38 (1 H, br s), 8.95 (1 H, s), 7.14 (1 H, d, <i>J</i> = 8.0 Hz), 6.87 (1 H, d, <i>J</i> = 8.0 Hz), 6.75 (2 H, br s), 5.92 (1 H, quin, <i>J</i> = 9.0 Hz), 2.32–2.42 (2 H, m), 1.80–2.07 (4 H, m), 1.54–1.70 (2 H, m). MS (ESI) <i>m</i>/<i>z</i>: 270.1 [M + H]<sup>+</sup>.</div></div><div id="sec5_1_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> 9-Cyclopentyl-2-((5-(1-piperazinyl)-2-pyridinyl)amino)-7,9-dihydro-8<i>H</i>-pyrido[4′,3′:4,5]-pyrrolo[2,3-<i>d</i>]pyrimidin-8-one (<b>5</b>)</h4><div class="NLM_p last">Compound <b>5</b> was prepared from <b>38</b> using the methods described in the synthesis of <b>1</b>. <sup>1</sup>H NMR (500 MHz, CD<sub>3</sub>OD) δ ppm 9.31 (1 H, s), 8.03 (1 H, s), 8.01 (1 H, d, <i>J</i> = 4.o Hz), 7.75 (1 H, d, <i>J</i> = 8.0 Hz), 7.35 (1 H, d, <i>J</i> = 4.0 Hz), 7.16 (1 H, d, <i>J</i> = 8.0 Hz), 6.05 (1 H, m), 3.52 (4 H, m), 3.47 (4 H, m), 2.50–2.60 (2 H, m), 2.09–2.25 (4 H, m), 1.55–1.75 (2H, m). HRMS (ESI) <i>m</i>/<i>z</i>: calculated for [M + H]<sup>+</sup> 431.2303, found 431.2314.</div></div><div id="sec5_1_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> 2-Amino-9-cyclopentyl-9<i>H</i>-pyrido[4′,3′:4,5]pyrrolo[2,3-<i>d</i>]pyrimidin-8-yl Trifluoromethanesulfonate (<b>40</b>)</h4><div class="NLM_p last">Sodium hydride (60% in oil, 16 mg) was added to a solution of compound <b>38</b> (0.10 g, 0.371 mmol) in DMF (2 mL) and stirred at 0 °C for 30 min, then <i>N</i>-phenyl-bis-(trifluoromethanesulfonimide) (0.146 g, 0.408 mmol) in DMF (1 mL) was added dropwise at 0 °C, and the resulting mixture was stirred at 0 °C for 2 h. The reaction mixture was concentrated and dissolved in DCM. The resulting organic solution was washed with brine, dried, and concentrated . The residue was purified by flash chromatography on silica gel, eluting with DCM/MeOH/NH<sub>4</sub>OH (100:10:1) to give compound <b>40</b> as a yellow solid (0.102 g, 67% yield). MS (ESI) <i>m</i>/<i>z</i>: 402.1 [M + H]<sup>+</sup>.</div></div><div id="sec5_1_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> 9-Cyclopentyl-8-methyl-9<i>H</i>-pyrido[4′,3′:4,5]pyrrolo[2,3-<i>d</i>]pyrimidin-2-amine (<b>41</b>)</h4><div class="NLM_p last">MeMgBr (3 M, in ether, 5.5 mmol, 1.8 mL) was added dropwise to a stirred solution of <b>40</b> (1.0 g, 1.8 mmol) and ferric acetylacetonate (1. 93 g, 5.5 mmol) in THF (20 mL) and NMP (1 mL) under argon. The resulting mixture was stirred at room temperature for 30 min. The reaction mixture was concentrated, and the residue was purified by flash chromatography on silica gel eluting with DCM/MeOH/NH<sub>4</sub>OH (200:10:1) to give compound <b>41</b> as a light-yellow solid (260 mg, 54% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm 8.90 (1 H, s), 8.33 (1 H, d, <i>J</i> = 8.0 Hz), 7.64 (1 H, d, <i>J</i> = 8.0 Hz), 5.38 (1 H, quin, <i>J</i> = 8.6 Hz), 5.15 (2 H, br s), 3.01 (3 H, s), 2.56–2.86 (2 H, m), 1.96–2.23 (4 H, m), 1.67–1.89 (2 H, m). MS (ESI) <i>m</i>/<i>z</i>: 268.1 [M + H]<sup>+</sup>.</div></div><div id="sec5_1_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> 9-Cyclopentyl-8-methyl-<i>N</i>-(5-(1-piperazinyl)-2-pyridinyl)-9<i>H</i>-pyrido[4′,3′:4,5]pyrrolo[2,3-<i>d</i>]pyrimidin-2-amine (<b>4</b>)</h4><div class="NLM_p last">Compound <b>4</b> was prepared according to the methods described in the synthesis of <b>1</b>. <sup>1</sup>H NMR (500 MHz, CD<sub>3</sub>OD) δ ppm 9.01 (1 H, s), 8.36 (1 H,s), 8.29 (1 H, d, <i>J</i> = 8.0 Hz), 8.01 (1 H, d, <i>J</i> = 4.0 Hz), 7.62 (1 H, d, <i>J</i> = 8.0 Hz), 7.28 (1 H, d, <i>J</i> = 4.0 Hz), 5.36 (1 H, m), 3.09 (4 H, m), 3.02 (4 H, m), 2.95 (3 H, s), 2.73–2.70 (2 H, m), 2.09–1.98 (4 H, m), 1.80 (2 H, m). HRMS (ESI) <i>m</i>/<i>z</i>: calculated for [M + H]<sup>+</sup> 429.2510, found 429.2527.</div></div><div id="sec5_1_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> 8-Cyano-9-cyclopentyl-9<i>H</i>-pyrido[4′,3′:4,5]pyrrolo[2,3-<i>d</i>]pyrimidin-2-amine (<b>48</b>)</h4><div class="NLM_p last">A mixture of <b>47</b> (1.50 g, 5.53 mmol) and tetraethylammonium cyanide (1.30 g, 8.29 mmol) in DMF (10 mL) was heated at 150 °C under microwave irradiation for 2 h. The reaction mixture was concentrated under reduced pressure and the residue was purified by flash chromatography on silica gel, eluting with DCM/MeOH/NH<sub>4</sub>OH (100:10:1) to give <b>48</b> as a tan solid (1.17 g, 76% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 9,17 (1 H, s), 8.49 (1 H, d, <i>J</i> = 5.1 Hz), 8.25 (1 H, d, <i>J</i> = 5.1 Hz), 7.24 (2 H, br s), 5.59 (1 H, m), 2.49–2.56 (2 H, m), 2.05–2.14 (4 H, m), 1.70–1.75 (2 H, m). MS (ESI) <i>m</i>/<i>z</i>: 279.2 [M + H]<sup>+</sup>.</div></div><div id="sec5_1_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> 9-Cyclopentyl-2-((5-(1-piperazinyl)-2-pyridinyl)amino)-9<i>H</i>-pyrido[4′,3′:4,5]-pyrrolo[2,3-<i>d</i>]pyrimidine-8-carbonitrile (<b>6</b>)</h4><div class="NLM_p last">Compound <b>6</b> was prepared according to the methods described in the synthesis of <b>1</b>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 10.42 (1 H, br s), 9.40–9.49 (1 H, m), 8.67–8.80 (2 H, m), 8.58–8.65 (1 H, m), 8.39–8.48 (1 H, m), 8.09–8.16 (1 H, m), 7.98–8.07 (1 H, m), 5.62–5.73 (1 H, m), 3.34–3.44 (5 H, m), 3.25–3.34 (4 H, m), 2.64 (2 H, m), 2.01–2.20 (4 H, m), 1.68–1.80 (2 H, m). HRMS (ESI) <i>m</i>/<i>z</i>: calculated for [M + H]<sup>+</sup> 440.2307, found 440.2302.</div></div><div id="sec5_1_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> <i>N</i><sup>4</sup>-Cyclopentyl-5-(3,6-dimethoxy-4-pyridazinyl)-2,4-pyrimidinediamine (<b>50</b>)</h4><div class="NLM_p last">A solution of nBuLi (2.5 M solution in hexanes) (7.5 mL, 19 mmol) was added to a cold (0 °C) solution of 2,2,6,6-tetramethylpiperidine (3.4 mL, 20 mmol) in THF (24 mL). The resulting mixture was allowed to stand at 0 °C for 30 min. The ice–water bath was then replaced with a dry ice–2-propanol bath. A precooled solution of 3,6-dimethoxypyridazine (2.40 g, 17 mmol) in THF (40 mL) was added through a cannula over a period of 30 min. The resulting mixture was stirred at −70 °C for 1.5 h. A solution of zinc(II) chloride (0.5 M in THF) (34 mL, 17 mmol) was added through a syringe over a period of 15 min. The cold bath was removed, and the resulting mixture was allowed to warm up to rt. A solution of compound <b>44a</b> (1.7 g, 5.7 mmol) and Pd<sub>2</sub>(PPh<sub>3</sub>)<sub>4</sub> (0.33 g, 0.29 mmol) in THF (20 mL) was added, and the resulting mixture was heated at reflux for 18 h. Upon workup, the crude mixture was cooled in an ice–water bath before being poured onto ice and saturated NH<sub>4</sub>Cl aqueous solution and extracted with ethyl acetate (2×). The combined organics were dried over sodium sulfate and concentrated in vacuo. The residue was purified by flash chromatography on silica gel, eluting with DCM/MeOH/NH<sub>4</sub>OH (200:10:1) to give compound <b>50</b> as an off-white solid (1.2 g, 66% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 7.53 (1 H, s), 7.01 (1 H, s), 6.18–6.38 (3 H, m), 4.35–4.51 (1 H, m, <i>J</i> = 7.3, 7.3, 7.3, 7.3, 7.0 Hz), 3.95 (3 H, s), 3.91 (3 H, s), 1.77–1.90 (2 H, m), 1.55–1.69 (2 H, m), 1.35–1.55 (4 H, m). MS (ESI) <i>m</i>/<i>z</i>: 317.2 [M + H]<sup>+</sup>.</div></div><div id="sec5_1_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> 9-Cyclopentyl-3-methoxy-9<i>H</i>-pyrimido[5′,4′:4,5]pyrrolo[2,3-<i>c</i>]pyridazin-7-amine (<b>51</b>)</h4><div class="NLM_p last">To a 35 mL microwave reaction vessel was added compound <b>50</b> (200 mg, 0.63 mmol) followed by THF (15 mL) and sodium hydride (60% in mineral oil, 0.063 g, 1.6 mmol). The resulting mixture was stirred at room temperature for 10 min to allow gas release. The vessel was then subjected to microwave irradiation at 150 °C for an hour. The reaction mixture was concentrated, and the residue was purified by flash chromatography on silica gel eluting with 2% methanol in DCM to give compound <b>51</b> as a light-yellow solid (170 mg, 95% yield). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 8.97 (1 H, s), 7.57 (1 H, s), 7.28 (2 H, s), 5.28 (1 H, quin, <i>J</i> = 8.7 Hz), 4.02 (3 H, s), 2.38–2.46 (2 H, m), 1.89–2.10 (4 H, m), 1.62–1.78 (2 H, m). MS (ESI) <i>m</i>/<i>z</i>: 285.1 [M + H]<sup>+</sup>.</div></div><div id="sec5_1_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> 7-Amino-9-cyclopentyl-9<i>H</i>-pyrimido[5′,4′:4,5]pyrrolo[2,3-<i>c</i>]pyridazin-3-yl Trifluoromethanesulfonate (<b>52</b>)</h4><div class="NLM_p last">A mixture of compound <b>51</b> (0.17 g, 0.60 mmol) and pyridine hydrochloride (0.69 g, 6.0 mmol) was heated at 200 °C overnight. After cooling to room temperature, the mixture was dissolved in THF (6 mL). Triethylamine (1.3 mL, 9.0 mmol), <i>N</i>,<i>N</i>-dimethylpyridin-4-amine (7.3 mg, 0.060 mmol), and <i>N</i>-phenyltrifluoromethanesulfonimide (0.54 g, 1.5 mmol) were added at room temperature. The resulting mixture was sonicated for 10 min and stirred for 2 h. The reaction mixture was concentrated, and the residue was purified by flash chromatography on silica gel eluting with 15–65% ethyl actate in hexane to give compound <b>52</b> as an off-white solid (150 mg, 62% yield). MS (ESI) <i>m</i>/<i>z</i>: 403.0 [M + H]<sup>+</sup>.</div></div><div id="sec5_1_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> 9-Cyclopentyl-9<i>H</i>-pyrimido[5′,4′:4,5]pyrrolo[2,3-<i>c</i>]pyridazin-7-amine (<b>53</b>)</h4><div class="NLM_p last">To a 25 mL single-necked round-bottom flask were added compound <b>52</b> (94 mg, 234 μmol), palladium diacetate (21 mg, 93 μmol), and 1,1′-bis(diphenylphosphino)ferrocene (130 mg, 234 μmol). The flask was subjected to three cycles of evacuation and backfilling with N<sub>2</sub>. DMF (3 mL) was added under N<sub>2</sub>, followed by triethylamine (715 μL, 5140 μmol) and formic acid (176 μL, 4.67 mmol). The resulting mixture was stirred at 65 °C for 1.5 h. The mixture was poured into ice and saturated NaHCO<sub>3</sub> aqueous solution, then extracted with 10% <i>i</i>-PrOH/DCM (2×). The combined organics were dried over sodium sulfate and concentrated in vacuo. The residue was purified by flash chromatography on silica gel eluting with DCM/MeOH/NH<sub>4</sub>OH (200:10:1) to give compound <b>53</b> as a reddish solid (48 mg, 80% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 9.08 (1 H, s), 9.01 (1 H, d, <i>J</i> = 5.1 Hz), 8.07 (1 H, d, <i>J</i> = 5.1 Hz), 7.28 (2 H, s), 5.42 (1 H, quin, <i>J</i> = 8.6 Hz), 2.40–2.49 (2 H, m), 1.92–2.14 (4 H, m), 1.63–1.80 (2 H, m). MS (ESI) <i>m</i>/<i>z</i>: 255.1 [M + H]<sup>+</sup>.</div></div><div id="sec5_1_26" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> 9-Cyclopentyl-<i>N</i>-(5-(1-piperazinyl)-2-pyridinyl)-9<i>H</i>-pyrimido[5′,4′:4,5]pyrrolo[2,3-<i>c</i>]pyridazin-7-amine (<b>7</b>)</h4><div class="NLM_p last">Compound <b>7</b> was prepared using chemistry similar to that described in the synthesis of <b>1</b>. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ ppm 9.47 (1 H, s), 9.23 (1 H, d, <i>J</i> = 5.5 Hz), 8.49 (1 H, d, <i>J</i> = 5.5 Hz), 7.99–8.11 (2 H, m), 7.75 (1 H, d, <i>J</i> = 9.4 Hz), 5.68 (1 H, quin, <i>J</i> = 8.5 Hz), 3.49–3.58 (4 H, m), 3.41–3.49 (4 H, m), 2.50–2.71 (2 H, m), 2.12–2.29 (4 H, m), 1.76–1.94 (2 H, m). HRMS (ESI) <i>m</i>/<i>z</i>: calculated for [M + H]+ 416.2307, found 416.2310.</div></div><div id="sec5_1_27" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> (<i>S</i>)-2-(2-((5-(Piperazin-1-yl)pyridin-2-yl)amino)-9<i>H</i>-pyrido[4′,3′:4,5]pyrrolo[2,3-<i>d</i>]pyrimidin-9-yl)propan-1-ol (<b>8</b>)</h4><div class="NLM_p last">Compound <b>8</b> was prepared using chemistry similar to that described in the synthesis of <b>1</b>. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ ppm 9.66 (1 H, s), 9.40 (1 H, s), 8.68 (1 H, d, <i>J</i> = 5.3 Hz), 8.64 (1 H, d, <i>J</i> = 5.3 Hz), 8.05–8.11 (3 H, m), 7.77 (1 H, d, <i>J</i> = 9.8 Hz), 5.29 (1 H, m), 4.38 (1 H, dd, <i>J</i> = 12.0, 8.0 Hz), 4.03 (1 H, dd, <i>J</i> = 8.0, 4.0 Hz), 3.50–3.57 (4 H, m), 3.40–3.47 (4 H, m), 1.79 (3 H, d, <i>J</i> = 8.0 Hz). HRMS (ESI) <i>m</i>/<i>z</i>: calculated for [M + H]<sup>+</sup> 405.2147, found 405.2127.</div></div><div id="sec5_1_28" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> 4-[2-(5-Piperazin-1-yl-pyridin-2-ylamino)-pyrido[4′,3′:4,5]pyrrolo[2,3-<i>d</i>]pyrimidin-9-yl]-<i>trans</i>-cyclohexanol (<b>9</b>)</h4><div class="NLM_p last">Compound <b>9</b> was prepared using chemistry similar to that described in the synthesis of <b>1</b>. <sup>1</sup>H NMR (500 MHz, CD<sub>3</sub>OD) δ ppm 9.62 (1 H, s), 9.42 (1 H, s), 8.64 (2 H, s), 8.11 (1 H, d, <i>J</i> = 2.5 Hz), 8.02 (1 H, d, <i>J</i> = 10.0 Hz), 7.87 (1 H, d, <i>J</i> = 10.0 Hz), 4.98 (1 H, m), 3.89 (1 H, m), 3.55 (4 H, m), 3.47 (4 H, m), 2.77 (2 H, m), 2.21 (1 H, d, <i>J</i> = 15 Hz), 2.05 (2 H, m), 1.86 (1 H, m), 1.65 (2 H, m). HRMS (ESI) <i>m</i>/<i>z</i>: calculated for [M + H]<sup>+</sup> 445.2459, found 445.2458.</div></div><div id="sec5_1_29" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> (9-Cyclohexyl-9<i>H</i>-pyrido[4′,3′,4,5]pyrrolo[2,3-<i>d</i>]pyrimidin-2-yl)-(5-piperazin-1-yl-pyridin-2-yl)-amine (<b>10</b>)</h4><div class="NLM_p last">Compound <b>10</b> was prepared using chemistry similar to that described in the synthesis of <b>1</b>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 9.56 (1 H, s), 9.47 (1 H, s), 8.77 (1 H, d, <i>J</i> = 6.1 Hz), 8.65 (1 H, d, <i>J</i> = 6.1 Hz), 8.10 (1 H, dd, <i>J</i> = 9.5, 2.7 Hz), 8.00 (1 H, d, <i>J</i> = 2.7 Hz), 7.54 (1 H, s), 7.42 (1 H, d, <i>J</i> = 9.5 Hz), 4.77–4.84 (1 H, m), 3.50–3.54 (4 H, m), 3.43–3.48 (4 H, m), 2.31–2.44 (2 H, m), 2.04–2.16 (4 H, m), 1.86–1.95 (1 H, m), 1.65–1.77 (2 H, m), 1.46–1.59 (1 H, m). HRMS (ESI) <i>m</i>/<i>z</i>: calculated for [M + H]<sup>+</sup> 429.2510, found 429.2507.</div></div><div id="sec5_1_30" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> 9-(4-Methylcyclohexyl)-<i>N</i>-(5-(1-piperazinyl)-2-pyridinyl)-9<i>H</i>-pyrido[4′,3′:4,5]pyrrolo[2,3-<i>d</i>]pyrimidin-2-amine (<b>11</b>)</h4><div class="NLM_p last">Compound <b>11</b> was prepared according to the methods described in the synthesis of <b>1</b>. <sup>1</sup>H NMR (500 MHz, CD<sub>3</sub>OD) δ ppm 9.19 (1 H, s), 8.97 (1H, s), 8.39 (1 H, d, <i>J</i> = 6.1 Hz), 8.27 (1 H,, d, <i>J</i> = 6.1 Hz), 8.05 (1 H, m), 7.53 (1 H, m), 3.20 (4 H, m), 3.07 (4 H, m), 2.80 (1 H, m), 2.60 (1 H, m), 2.10 (1 H br s), 1.97 (3 H, m), 1.75 (3 H, m), 1.40 (1 H, m), 1.08–1.25 (3 H, m). HRMS (ESI) <i>m</i>/<i>z</i>: calculated for [M + H]+ 443.2666, found 443.2666.</div></div><div id="sec5_1_31" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> 5-(3-Fluoropyridin-4-yl)-<i>N</i><sup>4</sup>-((1<i>r</i>,4<i>r</i>)-4-methylcyclohexyl)pyrimidine-2,4-diamine (<b>55</b>)</h4><div class="NLM_p last">To a solution of 2,2′,6,6′-tetramethylpiperidine (997 mL, 5.87 mol, 3 equiv) in anhydrous THF (6 L) under N<sub>2</sub> atmosphere at 0 °C, was added <i>n</i>-BuLi (2.5 M in hexanes, 2350 mL, 5.87 mol, 3 equiv) via an addition funnel over 30 min. Upon completion of the addition, the reaction mixture was stirred at 0 °C for 1 h. The reaction mixture was cooled to −74 °C, and a solution of 3-fluoropyridine (561 g, 5.773 mol, 2.95 equiv) in anhydrous THF (500 mL) was added over 15 min, keeping the temperature below −63 °C. Upon completion of the addition, the reaction mixture was stirred at −74 °C for an additional 2 h. A solution of ZnBr<sub>2</sub> (1422 g, 6.32 mol, 3.22 equiv) in anhydrous THF (3 L) was then added dropwise over 35 min, keeping the temperature below −60 °C. Upon completion of the addition, the cold bath was removed and the reaction mixture was allowed to warm to room temperature. Then <b>44b</b> (650 g, 1.95 mol, 1.0 equiv) was added in one portion, followed by Pd(PPh<sub>3</sub>)<sub>4</sub> (113 g, 97.8 mmol, 0.05 equiv). The reaction mixture was heated at reflux overnight and monitored using LCMS. Upon completion, the reaction mixture was cooled to room temperature, quenched with saturated aqueous NaHCO<sub>3</sub> (6 L), and extracted with ethyl acetate (2 × 10 L). The organic extracts were washed with saturated NaHCO<sub>3</sub> (2 × 2.5 L) and brine (2.5 L) and were then concentrated under vacuum. The residue was dissolved in 2N HCl (2.5 L) and washed with DCM (3 × 1.25 L). The aqueous phase was adjusted to pH 10–12 by addition of aqueous 4N NaOH and extracted with DCM (3 × 1.5 L). The organic extracts were washed with water (2 × 1.25 L), dried, and concentrated to give compound <b>55</b> (540 g, 92% yield). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 8.50 (1 H, d, <i>J</i> = 6.6 Hz), 8.37 (1 H, d, <i>J</i> = 4.8 Hz), 7.58 (1 H, s), 7.35 (1 H, dd, <i>J</i> = 6.6, 4.4 Hz), 6.24 (2 H, br s), 5.00 (1 H, d, <i>J</i> = 8.4 Hz), 3.96 (1 H, m), 1.74 (2 H, d, <i>J</i> = 11.7 Hz), 1.64 (2H, d, <i>J</i> = 12.3 Hz), 1.30–1.18 (3H, m), 0.98 (1H, dd, <i>J</i> = 12.9, 2.7 Hz), 0.85 (3H, d, <i>J</i> = 7.2 Hz). MS <i>m</i>/<i>z</i>: 302.2 [M + H]<sup>+</sup>.</div></div><div id="sec5_1_32" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> 9-((1<i>r</i>,4<i>r</i>)-4-Methylcyclohexyl)-9<i>H</i>-pyrido[4′,3′:4,5]pyrrolo[2,3-<i>d</i>]pyrimidin-2-amine (<b>33d</b>)</h4><div class="NLM_p last">To a solution of <b>55</b> (854 g, 2.84 mol, 1.0 equiv) in anhydrous 1-methyl-2-pyrrolidinone (8 L) under N<sub>2</sub> atmosphere at room temperature was added LiHMDS (1.0 M in toluene, 8.5 L, 8.5 mol, 3.0 equiv) over 30 min. Upon completion of the addition, the reaction mixture was heated at 90 °C overnight and monitored using LCMS. Upon completion, the reaction mixture was cooled to room temperature, quenched with ice-cold water (10 L), and extracted with ethyl acetate (12 L). The organic phase was washed with saturated aqueous NaHCO<sub>3</sub> (2 × 4 L) and water (3 × 2 L). The aqueous layers were combined and back-extracted with ethyl acetate (2 × 15 L). The organic layers were combined, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated under reduced pressure. The solid thus obtained was suspended in DCM (2.5 L) and agitated using a rotary evaporator for 30 minutes. The solid was collected by filtration, washed with DCM, and dried to afford compound <b>33d</b> (400 g). The mother liquor was purified by column chromatography (eluting with DCM/MeOH = 50:1) to afford, after triturating with DCM (750 mL), additional compound <b>33d</b> (277 g, total: 677 g, 84% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 8.95 (1 H, s), 8.92 (1 H, s), 8.46 (1 H, d, <i>J</i> = 5.3 Hz), 7.78 (1 H, d, <i>J</i> = 5.3 Hz), 5.22 (2 H, br s), 4.72–4.88 (1 H, m), 2.45 (2 H, qd, <i>J</i> = 12.9, 4.0 Hz), 1.82–2.12 (4 H, m), 1.56–1.71 (1 H, m), 1.21–1.42 (2 H, m), 1.02 (3 H, d, <i>J</i> = 6.5 Hz). HRMS (ESI) <i>m</i>/<i>z</i>: calculated for [M + H]<sup>+</sup> 282.1713, found 282.1709.</div></div><div id="sec5_1_33" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> 9-((1<i>r</i>,4<i>r</i>)-4-Methylcyclohexyl)-<i>N</i>-(5-(piperazin-1-yl)pyridin-2-yl)-9<i>H</i>-pyrido[4′,3′:4,5]pyrrolo[2,3-<i>d</i>]pyrimidin-2-amine (<b>12</b>)</h4><div class="NLM_p last">Compound <b>12</b> was prepared according to the methods described in the synthesis of <b>1</b>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 9.76 (1 H, s), 9.26 (1 H, s), 9.10 (1 H, s), 8.42 (1 H, d, <i>J</i> = 5.1 Hz), 8.20 (1 H, d, <i>J</i> = 9.0 Hz), 7.98–8.07 (2 H, m), 7.44 (1 H, dd, <i>J</i> = 9.0, 5.1 Hz), 4.80 (1 H, m), 3.04–3.06 (4 H, m), 2.06–2.08 (4 H, m), 2.55–2.61 (2 H, m), 1.80–1.93 (4 H, m), 1.66 (1 H, m), 1.21–1.29 (2 H, m), 1.01 (3 H, d, <i>J</i> = 6.4 Hz). HRMS (ESI) <i>m</i>/<i>z</i>: calculated for [M + H]<sup>+</sup> 443.2666, found 443.2681.</div></div><div id="sec5_1_34" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> 9-(4,4-Dimethylcyclohexyl)-<i>N</i>-(5-(1-piperazinyl)-2-pyridinyl)-9<i>H</i>-pyrido[4′,3′:4,5]-pyrrolo[2,3-<i>d</i>]pyrimidin-2-amine (<b>13</b>)</h4><div class="NLM_p last">Compound <b>13</b> was prepared according to the methods described in the synthesis of <b>1</b>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 10.35 (1 H, s), 9.52 (1 H, s), 9.35 (1 H, s), 8.62 (1 H, d, <i>J</i> = 5.9 Hz), 8.49 (1 H, d, <i>J</i> = 5.9 Hz), 8.15 (1 H, d, <i>J</i> = 2.9 Hz), 8.04 (1 H, d, <i>J</i> = 9.0 Hz), 7.51 (1 H, dd, <i>J</i> = 9.0, 2.9 Hz), 4.66–4.79 (1 H, m), 3.34–3.43 (4 H, m), 3.30 (4 H, br s), 2.62–2.77 (2 H, m), 1.73 (2 H, d, <i>J</i> = 11.3 Hz), 1.46–1.63 (4 H, m), 1.11 (3 H, s), 1.02 (3 H, s). HRMS (ESI) <i>m</i>/<i>z</i>: calculated for [M + H]<sup>+</sup> 457.2822, found 457.2806.</div></div><div id="sec5_1_35" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> 9-(3,3-Dimethylcyclohexyl)-<i>N</i>-(5-(piperazin-1-yl)pyridin-2-yl)-9<i>H</i>-pyrido[4′,3′:4,5]pyrrolo[2,3-<i>d</i>]pyrimidin-2-amine (<b>14</b>)</h4><div class="NLM_p last">Compound <b>14</b> was prepared according to the methods described in the synthesis of <b>1</b>. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ ppm 9.70 (1 H, s), 9.51 (1 H, s), 8.75 (1 H, d, <i>J</i> = 5.3 Hz), 8.68 (1 H, d, <i>J</i> = 5.3 Hz), 8.17 (1 H, dd, <i>J</i> = 9.8, 3.2 Hz), 8.06 (1 H, d, <i>J</i> = 3.2 Hz), 7.74 (1 H, d, <i>J</i> = 9.8 Hz), 5.21 (1 H, m), 3.52–3.57 (4 H, m), 3.44–3.47 (4 H, m), 2.45–2.55 (2 H, m), 2.03 (1 H, m), 1.82–1.91 (2 H, m), 1.71 (1 H, m), 1.47–1.57 (2 H, m), 1.21 (3 H, s), 1.10 (3 H, s). HRMS (ESI) <i>m</i>/<i>z</i>: calculated for [M + H]<sup>+</sup> 457.2822, found 457.2829.</div></div><div id="sec5_1_36" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> 9-((1<i>r</i>,4<i>r</i>)-4-Methylcyclohexyl)-<i>N</i>-(6-(piperazin-1-yl)pyridin-3-yl)-9<i>H</i>-pyrido[4′,3′:4,5]pyrrolo[2,3-<i>d</i>]pyrimidin-2-amine (<b>15</b>)</h4><div class="NLM_p last">Compound <b>15</b> was prepared according to the methods described in the synthesis of <b>1</b>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 9.61 (1 H, s), 9.19 (1 H, s), 9.05 (1 H, s), 8.53 (1 H, s), 8.39 (1 H, d, <i>J</i> = 5.1 Hz), 7.87–8.01 (2 H, m), 6.81 (1 H, d, <i>J</i> = 9.0 Hz), 4.75 (1 H, m), 3.28–3.41 (4 H, m), 2.73–2.88 (4 H, m), 2.53–2.62 (2 H, m), 1.80–1.87 (4 H, m), 1.67 (1 H, m), 1.15–1.25 (2 H, m), 0.99 (3 H, d, <i>J</i> = 6.5 Hz). HRMS (ESI) <i>m</i>/<i>z</i>: calculated for [M + H]<sup>+</sup> 443.2666, found 443.2680.</div></div><div id="sec5_1_37" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61"> 9-((1<i>r</i>,4<i>r</i>)-4-Methylcyclohexyl)-<i>N</i>-(6-(piperazin-1-yl)pyridazin-3-yl)-9<i>H</i>-pyrido[4′,3′:4,5]pyrrolo[2,3-<i>d</i>]pyrimidin-2-amine (<b>16</b>)</h4><div class="NLM_p last">Compound <b>16</b> was prepared from <b>33d</b> and <b>58e</b> using chemistry similar to that described in the synthesis of <b>1</b>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 10.86 (1 H, br s), 9.51 (1 H, s), 9.40 (1 H, s), 8.98 (1 H, br s), 8.62 (1 H, d, <i>J</i> = 5.9 Hz), 8.47 (1 H, d, <i>J</i> = 5.9 Hz), 8.25 (1 H, d, <i>J</i> = 9.8 Hz), 7.58 (1 H, d, <i>J</i> = 9.8 Hz), 4.82 (1 H, t, <i>J</i> = 3.8 Hz), 3.75–3.84 (4 H, m), 3.27–3.33 (4 H, m), 2.49–2.55 (2 H, m), 1.87 (4 H, ddd, <i>J</i> = 10.1, 1.4, 1.2 Hz), 1.61 (1 H, m), 1.17–1.25 (2 H, m), 0.99 (3 H, d, <i>J</i> = 6.4 Hz). HRMS (ESI) <i>m</i>/<i>z</i>: calculated for [M + H]<sup>+</sup> 444.2619, found 444.2617.</div></div><div id="sec5_1_38" class="NLM_sec NLM_sec_level_3"><div id="ac_i62" class="anchor-spacer"></div><h4 class="article-section__title" id="_i62"> 8-(6-Chloropyridin-3-yl)-1,4-dioxa-8-azaspiro[4.5]decane (<b>59a</b>)</h4><div class="NLM_p last">To a solution of 5-bromo-2-chloropyridine (7.70 g, 40 mmol) in toluene (200 mL) were added 1,4-dioxa-8-azaspiro-[4,5]decane (6.30 g, 44 mmol), sodium <i>t</i>-butoxide (5.76 g, 60 mmol), Pd<sub>2</sub>dba<sub>3</sub> (548 mg, 0.60 mmol), and XantPhos (1388 mg, 2.40 mmol). The mixture was purged with argon then heated at 100 °C for 5 h. The reaction mixture was then cooled to room temperature, diluted with ethyl acetate, and washed with water. The organic layer was concentrated, and the residue was purified by flash chromatography on silica gel, eluting with 15–50% ethyl acetate in hexanes to give <b>59a</b> as a light-yellow solid (7.77 g, 76% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm 8.03 (1 H, d, <i>J</i> = 2.2 Hz), 7.19 (1 H, dd, <i>J</i> = 8.0, 2.2 Hz), 7.15 (1 H, d, <i>J</i> = 8.0 Hz), 4.0 (4 H, s), 3.32–3.34 (4 H, m), 1.82–1.85 (4 H, m). MS (ESI) <i>m</i>/<i>z</i>: 255.2 [M + H]<sup>+</sup>.</div></div><div id="sec5_1_39" class="NLM_sec NLM_sec_level_3"><div id="ac_i63" class="anchor-spacer"></div><h4 class="article-section__title" id="_i63"> 1-(6-((9-((1<i>r</i>,4<i>r</i>)-4-Methylcyclohexyl)-9<i>H</i>-pyrido[4′,3′:4,5]pyrrolo[2,3-<i>d</i>]pyrimidin-2-yl)amino)pyridin-3-yl)piperidin-4-one (<b>60a</b>)</h4><div class="NLM_p last">To a solution of <b>33d</b> (563 mg, 2.0 mmol) in 1,4-dioxane (20 mL) were added <b>59a</b> (611 mg, 2.40 mmol), Pd<sub>2</sub>dba<sub>3</sub> (92 mg, 0.10 mmol), XantPhos (175 mg, 0.30 mmol), and sodium <i>t</i>-butoxide (290 mg, 3.0 mmol). The reaction mixture was purged with argon, then heated at 150 °C under microwave irradiation for 2 h. The reaction mixture was subjected to flash chromatography on silica gel to give <i>N</i>-(5-(1,4-dioxa-8-azaspiro[4.5]decan-8-yl)pyridin-2-yl)-9-((1<i>r</i>,4<i>r</i>)-4-methylcyclohexyl)-9<i>H</i>-pyrido[4′,3′:4,5]pyrrolo[2,3-<i>d</i>]pyrimidin-2-amine as a yellow solid (1 g, 90% purity). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ ppm 9.07 (1 H, s), 8.96 (1 H, s), 8.49 (1 H, d, <i>J</i> = 5.1 Hz), 8.42 (1 H, d, <i>J</i> = 9.0 Hz), 8.14 (1 H, br s), 8.09 (1 H, d, <i>J</i> = 3.2 Hz), 7.32 (1 H, d, <i>J</i> = 5.1 Hz), 7.40 (1 H, dd, <i>J</i> = 9.0, 3.2 Hz), 4.75 (1 H, m), 4.02 (4 H, s), 3.31–3.33 (4 H, m), 2.51–2.68 (2 H, m), 1.97–2.01 (4 H, m), 1.90–1.92 (4 H, m), 1.87 (1 H, m), 1.18–1.41 (2 H, m), 1.07 (3 H, d, <i>J</i> = 6.6 Hz). MS (ESI) <i>m</i>/<i>z</i>: 500.2 [M + H]<sup>+</sup>]. The yellow solid was dissoved in THF (10 mL) and 3 N HCl (10 mL), then stirred at 60 °C overnight. To the reaction mixture was added 1 N NaOH (30 mL), and the resulting mixture was extracted with DCM. The organic layer was dried over MgSO<sub>4</sub> and concentrated under reduced pressure to give crude <b>60a</b> as a yellow sold (1.1 g, 80% purity) that was used in the next step without further purification. MS (ESI) <i>m</i>/<i>z</i>: 456.2 [M + H]<sup>+</sup>.</div></div><div id="sec5_1_40" class="NLM_sec NLM_sec_level_3"><div id="ac_i64" class="anchor-spacer"></div><h4 class="article-section__title" id="_i64"> <i>N</i>-(5-(4-Aminopiperidin-1-yl)pyridin-2-yl)-9-((1<i>r</i>,4<i>r</i>)-4-methylcyclohexyl)-9<i>H</i>-pyrido[4′,3′:4,5]pyrrolo[2,3-<i>d</i>]pyrimidin-2-amine (<b>17</b>)</h4><div class="NLM_p last">To a solution of <b>60a</b> (285 mg, 80% purity, 0.5 mmol) in methanol (10 mL) were added molecular sieves (3 Å) (1.5 g) and amonium acetate (385 mg, 5 mmol), and the mixture thus obtained was stirred at room temperature for 20 min. To the reaction mixture was added sodium cyanoborohydride (126 mg, 4 mmol), and the mixture was stirred at 60 °C for 2 h. The reaction mixture was filtered, and the filtrate was concentrated under reduced pressure. The residue was purified by flash chromatography on silica gel, eluting with DCM/MeOH/NH<sub>4</sub>OH to give <b>17</b> as a light-yellow solid (78 mg, 34% yield). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 9.73 (1 H, s), 9.25 (1 H, s), 9.10 (1 H, s), 8.42 (1 H, d, <i>J</i> = 5.1 Hz), 8.17 (1 H, d, <i>J</i> = 9.0 Hz), 8.01–8.04 (2 H, m), 7.42 (1 H, dd, <i>J</i> = 9.0, 3.2 Hz), 4.79 (1 H, m), 3.60 (2 H, ddd, <i>J</i> = 12.8, 4.0, 3.9 Hz), 3.20–3.44 (2 H, m), 2.65–2.89 (3 H, m), 2.51–2.65 (2 H, m), 1.80–1.89 (6 H, m), 1.68 (1 H, m), 1.31–1.44 (2 H, m), 1.12–1.31 (2 H, m), 1.00 (3 H, d, <i>J</i> = 6.6 Hz). HRMS (ESI) <i>m</i>/<i>z</i>: calculated for [M + H]<sup>+</sup> 457.2822, found 457.2811.</div></div><div id="sec5_1_41" class="NLM_sec NLM_sec_level_3"><div id="ac_i65" class="anchor-spacer"></div><h4 class="article-section__title" id="_i65"> 1-(6-Chloropyridin-3-yl)-<i>N</i>,<i>N</i>-dimethylpiperidin-4-amine (<b>58a</b>)</h4><div class="NLM_p last">Compound <b>58a</b> was prepared according to the methods described in the synthesis of <b>59a</b> with a 93% yield. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ ppm 8.02 (1 H, d, <i>J</i> = 2.9 Hz), 7.19 (1 H, dd, <i>J</i> = 9.8, 2.9 Hz), 7.15 (1 H, d, <i>J</i> = 9.8 Hz), 3.64–3.74 (2 H, m), 2.75–2.81 (2 H, m), 2.31 (6 H, s), 2.16–2.30 (1 H, m), 1.91–1.95 (2 H, m), 1.55–1.78 (2 H, m). MS (ESI) <i>m</i>/<i>z</i>: 240.1 [M + H]<sup>+</sup>.</div></div><div id="sec5_1_42" class="NLM_sec NLM_sec_level_3"><div id="ac_i66" class="anchor-spacer"></div><h4 class="article-section__title" id="_i66"> <i>N</i>-(5-(4-(Dimethylamino)piperidin-1-yl)pyridin-2-yl)-9-((1<i>r</i>,4<i>r</i>)-4-methylcyclohexyl)-9<i>H</i>-pyrido[4′,3′:4,5]pyrrolo[2,3-<i>d</i>]pyrimidin-2-amine (<b>18</b>)</h4><div class="NLM_p last">To a solution of <b>33d</b> (110 mg, 0.39 mmol) in 1,4-dioxane (4 mL) were added <b>58a</b> (112 mg, 0.47 mmol), Pd<sub>2</sub>dba<sub>3</sub> (18 mg), XantPhos (35 mg), and sodium <i>t</i>-butoxide (43 mg, 0.59 mmol). The reaction mixture was purged with argon then heated at 150 °C under microwave irradiation for 1 h. The reaction mixture was purified by flash chromatography on silica gel, eluting with DCM/MeOH/NH<sub>4</sub>OH (100:10:1) to give <b>18</b> as a yellow solid (140 mg, 74% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 9.81 (1 H, s), 9.28 (1 H, s), 9.13 (1 H, s), 8.44 (1 H, d, <i>J</i> = 5.3 Hz), 8.21 (1 H, d, <i>J</i> = 9.2 Hz), 8.08 (1 H, d, <i>J</i> = 3.1 Hz), 8.06 (1 H, d, <i>J</i> = 5.3 Hz), 7.51 (1 H, dd, <i>J</i> = 9.2, 3.1 Hz), 4.78 (1 H, m), 3.82–3.85 (2 H, m), 2.77 (6 H, s), 2.72–2.77 (3 H, m), 2.51–2.65 (2 H, m), 2.10–2.14 (2 H, m), 1.72–1.91 (6 H, m), 1.64 (1 H, m), 1.20–1.29 (2 H, m), 1,01 (3 H, d, <i>J</i> = 6.5 Hz). HRMS (ESI) <i>m</i>/<i>z</i>: calculated for [M + H]<sup>+</sup> 485.3134, found 485.3128.</div></div><div id="sec5_1_43" class="NLM_sec NLM_sec_level_3"><div id="ac_i67" class="anchor-spacer"></div><h4 class="article-section__title" id="_i67"> <i>N</i>-(6-(4-(Dimethylamino)piperidin-1-yl)pyridazin-3-yl)-9-((1r,4r)-4-methylcyclohexyl)-9<i>H</i>-pyrido[4′,3′:4,5]pyrrolo[2,3-<i>d</i>]pyrimidin-2-amine (<b>19</b>)</h4><div class="NLM_p last">Compound <b>19</b> was prepared according to the methods described in the synthesis of <b>18</b>. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ ppm 9.07 (1 H, s), 8.98 (1 H, s), 8.55 (1 H, d, <i>J</i> = 9.8 Hz), 8.51 (1 H, d, <i>J</i> = 5.4 Hz), 8,25 (1 H, s), 7.84 (1 H, d, <i>J</i> = 5.4 Hz), 7.10 (1 H, d, <i>J</i> = 9.8 Hz), 4.77 (1 H, m), 4.34–4.38 (2 H, m), 2.92–3.04 (2 H, m), 2.46–2.62 (2 H, m), 2.41 (1 H, m), 2.33 (6 H, s), 1.82–2.03 (6 H, m), 1.55–1.69 (3 H, m), 1.24–1.38 (2 H, m), 1.07 (3 H, d, <i>J</i> = 6.4 Hz). HRMS (ESI) <i>m</i>/<i>z</i>: calculated for [M + H]<sup>+</sup> 486.3087, found 486.3094.</div></div><div id="sec5_1_44" class="NLM_sec NLM_sec_level_3"><div id="ac_i68" class="anchor-spacer"></div><h4 class="article-section__title" id="_i68"> <i>N</i>-(5-(4-(Dimethylamino)piperidin-1-yl)-6-fluoropyridin-2-yl)-9-((1<i>r</i>,4<i>r</i>)-4-methylcyclohexyl)-9<i>H</i>-pyrido[4′,3′:4,5]pyrrolo[2,3-<i>d</i>]pyrimidin-2-amine (<b>20</b>)</h4><div class="NLM_p last">Compound <b>20</b> was prepared according to the methods described in the synthesis of <b>18</b>. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ ppm 9.44 (1 H, s), 9.27 (1 H, s), 8.39–8.58 (2 H, m), 8.26 (1 H, d, <i>J</i> = 8.4 Hz), 7.63 (1 H, m), 4.98 (1 H, m), 3.59–3.66 (2 H, m), 3.39 (1 H, m), 2.96 (6 H, s), 2.85–3.00 (2 H, m), 2.62–2.80 (2 H, m), 2.20–2.27 (2 H, m), 1.90–2.10 (6 H, m), 1.73 (1 H, m), 1.28–1.47 (2 H, m), 1.10 (3 H, d, <i>J</i> = 5.6 Hz). HRMS (ESI) <i>m</i>/<i>z</i>: calculated for [M + H]<sup>+</sup> 503.3040, found 503.3044.</div></div><div id="sec5_1_45" class="NLM_sec NLM_sec_level_3"><div id="ac_i69" class="anchor-spacer"></div><h4 class="article-section__title" id="_i69"> <i>N</i>-(5-(4-(Dimethylamino)piperidin-1-yl)-6-methoxypyridin-2-yl)-9-((1<i>r</i>,4<i>r</i>)-4-methylcyclohexyl)-9<i>H</i>-pyrido[4′,3′:4,5]pyrrolo[2,3-<i>d</i>]pyrimidin-2-amine (<b>21</b>)</h4><div class="NLM_p last">Compound <b>21</b> was prepared according to the methods described in the synthesis of <b>18</b>. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ ppm 9.06 (1 H, s), 8.97 (1 H, s), 8,49 (1 H, d, <i>J</i> = 5.1 Hz), 7.98 (1 H, d, <i>J</i> = 8.3 Hz), 7.90 (1 H, s), 7.82 (1 H, d, <i>J</i> = 5.1 Hz), 7.26 (1 H, d, <i>J</i> = 8.3 Hz), 4.78 (1 H, m), 3.99 (3 H, s), 3.40–3.62 (2 H, m), 2.58–2.77 (4 H, m), 2.35 (6 H, s), 2.34 (1 H, m), 1.91–2.10 (6 H, m), 1.76–1.82 (2 H, m), 1.66 (1 H, m), 1.18–1.39 (2 H, m), 1.07 (3 H, d, <i>J</i> = 6.6 Hz). HRMS (ESI) <i>m</i>/<i>z</i>: calculated for [M + H]<sup>+</sup> 515.3239, found 515.3239.</div></div><div id="sec5_1_46" class="NLM_sec NLM_sec_level_3"><div id="ac_i70" class="anchor-spacer"></div><h4 class="article-section__title" id="_i70"> (<i>R</i>)-1-(Methyl(1-(6-((9-((1<i>r</i>,4<i>r</i>)-4-methylcyclohexyl)-9<i>H</i>-pyrido[4′,3′:4,5]pyrrolo[2,3-<i>d</i>]pyrimidin-2-yl)amino)pyridazin-3-yl)piperidin-4-yl)amino)propan-2-ol (<b>22</b>)</h4><div class="NLM_p last">Compound <b>22</b> was prepared according to the methods described in the synthesis of <b>17</b>. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ ppm 9.09 (1 H, s), 8.98 (1 H, s), 8.56 (1 H, d, <i>J</i> = 9.8 Hz), 8.51 (1 H, d, <i>J</i> = 5.1 Hz), 8.41 (1 H, s), 7.84 (1 H, d, <i>J</i> = 5.1 Hz), 7.10 (1 H, d, <i>J</i> = 9.8 Hz), 5.30 (1 H, s), 4.77 (1 H, m), 4.32–4.56 (2 H, m), 3.78 (1 H, ddd, <i>J</i> = 10.4, 6.1, 3.1 Hz), 2.80–3.03 (2 H, m), 2.69 (1 H, m), 2.46–2.62 (2 H, m), 2.41 (1 H, m), 2.31 (3 H, s), 2.30 (1 H, m), 1.95–2.00 (4 H, m), 1.81 (1 H, m), 1.56–1.77 (3 H, m), 1.20 −1.38 (2 H, m), 1.14 (3 H, d, <i>J</i> = 6.1 Hz), 1.06 (3 H, d, <i>J</i> = 6.4 Hz). HRMS (ESI) <i>m</i>/<i>z</i>: calculated for [M + H]<sup>+</sup> 530.3348, found 530.3349.</div></div><div id="sec5_1_47" class="NLM_sec NLM_sec_level_3"><div id="ac_i71" class="anchor-spacer"></div><h4 class="article-section__title" id="_i71"> 2-Hydroxy-1-(4-(6-((9-((1<i>r</i>,4<i>r</i>)-4-methylcyclohexyl)-9<i>H</i>-pyrido[4′,3′:4,5]pyrrolo[2,3-<i>d</i>]pyrimidin-2-yl)amino)pyridazin-3-yl)piperazin-1-yl)ethanone (<b>23</b>)</h4><div class="NLM_p last">To a solution of <b>16</b> (108 mg, 0.243 mmol) in DMF (10 mL) were added glycolic acid (0.023 mL, 0.365 mmol), PyBroP (170 mg, 0.365 mmol), and DIPEA (0.127 mL, 0.730 mmol), and the resulting mixture was stirred at room temperature overnight. The reaction mixture was concentrated under reduced pressure, and the residue was purified by flash chromatography on silica gel, eluting with DCM/MeOH/NH<sub>4</sub>OH (100:10:1) to give compound <b>23</b> as a yellow solid (85 mg, 70% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 10.25 (1 H, s), 9.27 (1 H, s), 9.11 (1 H, s), 8.43 (1 H, d, <i>J</i> = 5.1 Hz), 8.27 (1 H, d, <i>J</i> = 9.8 Hz), 8.04 (1 H, d, <i>J</i> = 5.1 Hz), 7.46 (1 H, d, <i>J</i> = 9.8 Hz), 4.80 (1 H, m), 4.64 (1 H, t, <i>J</i> = 8.2 Hz), 4.16 (2 H, d, <i>J</i> = 8.2 Hz), 3.50–3.65 (8 H, m), 2.50–2.55 (2 H, m), 1.80–1.90 (4 H, m), 1.66 (1 H, m), 1.20–1.27 (2 H, m), 0.99 (3 H, d, <i>J</i> = 6.5 Hz). HRMS (ESI) <i>m</i>/<i>z</i>: calculated for [M + H]<sup>+</sup> 502.2673, found 502.2676.</div></div><div id="sec5_1_48" class="NLM_sec NLM_sec_level_3"><div id="ac_i72" class="anchor-spacer"></div><h4 class="article-section__title" id="_i72"> (1-(6-Chloropyridazin-3-yl)piperidin-4-yl)methanol (<b>62</b>)</h4><div class="NLM_p last">A mixture of 3,6-dichloropyridazine (13.58 g, 91 mmol), DIPEA (11.78 g, 15.92 mL, 91 mmol), and 4-piperidinemethanol (10.50 g, 91 mmol) was heated at 90 °C for 2 h. The mixture was purified by flash chromatography on silica gel, eluting with 40–100% ethyl acetate in hexanes to give <b>62</b> as a white solid (20.28 g, 98% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm 7.19 (1 H, d, <i>J</i> = 9.6 Hz), 6.93 (1 H, d, <i>J</i> = 9.6 Hz), 4.27–4.45 (2 H, m), 3.56 (2 H, d, <i>J</i> = 6.1 Hz), 2.84–3.12 (2 H, m), 1.65–1.98 (4 H, m), 1.10–1.45 (2 H, m). MS (ESI) <i>m</i>/<i>z</i>: 228.1 [M + H]<sup>+</sup>.</div></div><div id="sec5_1_49" class="NLM_sec NLM_sec_level_3"><div id="ac_i73" class="anchor-spacer"></div><h4 class="article-section__title" id="_i73"> 3-Chloro-6-(4-((methylthio)methyl)piperidin-1-yl)pyridazine (<b>63</b>)</h4><div class="NLM_p last">To a solution of <b>62</b> (1.55 g, 6.81 mmol) in DCM (10 mL) were added triethylamine (1.14 mL, 8.17 mmol) and methane sulfonyl chloride (0.579 mL, 7.49 mmol). The mixture was stirred for 3 h at room temperature then purified by flash chromatography on silica gel to give (1-(6-chloropyridazin-3-yl)piperidin-4-yl)methyl methanesulfonate (2.05 g, 98% yield, MS (ESI) <i>m</i>/<i>z</i>: 306.0 [M + H]<sup>+</sup>). The mesylate (500 mg, 1.64 mmol) was dissolved in DMF (2 mL), and sodium methanethiolate (115 mg, 1.64 mmol) was added. The mixture thus obtained was stirred overnight at room temperature. The reaction mixture was diluted with ethyl acetate and washed with water and brine, then dried over MgSO<sub>4</sub>. The solvent was evaporated under reduced pressure to give crude <b>63</b> (500 mg, 85% purity) which was used in next step without further purification. MS (ESI) <i>m</i>/<i>z</i>: 258.0 [M + H]<sup>+</sup>.</div></div><div id="sec5_1_50" class="NLM_sec NLM_sec_level_3"><div id="ac_i74" class="anchor-spacer"></div><h4 class="article-section__title" id="_i74"> 3-Chloro-6-(4-((methylsulfonyl)methyl)piperidin-1-yl)pyridazine (<b>64</b>)</h4><div class="NLM_p last">To a solution of <b>63</b> (500 mg, 85% purity, 1.64 mmol) in 90% methanol in water was added oxone (2.22 g, 3.61 mmol). The reaction mixture was stirred overnight at room temperature, then diluted with ethyl acetate, washed with water and brine, and then dried. The solvent was evaporated and the residue was purified by flash chromatography on silica gel eluting with DCM/MeOH/NH<sub>4</sub>OH(200:10:1) to give compound <b>64</b> (436 mg, 92% yield). MS (ESI) <i>m</i>/<i>z</i>: 290.0 [M + H]<sup>+</sup>.</div></div><div id="sec5_1_51" class="NLM_sec NLM_sec_level_3"><div id="ac_i75" class="anchor-spacer"></div><h4 class="article-section__title" id="_i75"> 9-((1r,4r)-4-Methylcyclohexyl)-<i>N</i>-(6-(4-((methylsulfonyl)methyl)piperidin-1-yl)pyridazin-3-yl)-9<i>H</i>-pyrido[4′,3′:4,5]pyrrolo[2,3-<i>d</i>]pyrimidin-2-amine (<b>24</b>)</h4><div class="NLM_p last">Compound <b>24</b> was prepared from <b>33d</b> and <b>64</b> using similar chemistry described in the synthesis of <b>1</b>. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ ppm 9.07 (1 H, s), 8.99 (1 H, s), 8.58 (1 H, d, <i>J</i> = 10.0 Hz), 8.51 (1 H, d, <i>J</i> = 5.3 Hz), 7.85 (1 H, d, <i>J</i> = 5.3 Hz), 7.10 (1 H, d, <i>J</i> = 10.0 Hz), 4.78 (1 H, m), 4.29–4.37 (2 H, m), 3.00–3.09 (5 H, m), 2.98 (3 H, s), 2.40–2.60 (3 H. M), 2.09–2.16 (2 H, m), 1.94–2.06 (4 H, m), 1.58–1.65 (2 H, m), 1.24–1.36 (2 H, m), 1.07 (3 H, d, <i>J</i> = 6.5 Hz). HRMS (ESI) <i>m</i>/<i>z</i>: calculated for [M + H]<sup>+</sup> 535.2597, found 535.2601.</div></div><div id="sec5_1_52" class="NLM_sec NLM_sec_level_3"><div id="ac_i76" class="anchor-spacer"></div><h4 class="article-section__title" id="_i76"> 8-(6-Chloropyridazin-3-yl)-2-thia-8-azaspiro[4.5]decane 2,2-Dioxide (<b>66</b>)</h4><div class="NLM_p last">To a solution of 3,6-dichloropyridazine (5.57 g, 37.4 mmol) and 2-thia-8-azaspiro[4.5]decane 2,2-dioxide hydrochloride (<b>65</b>)(2.11 g, 9.35 mmol) in 1.4-dioxane (5 mL) was added DIPEA (5.41 mL, 32.7 mmol). The reaction mixture was heated at 80 °C overnight and then concentrated. The residue was purified by flash chromatography on silica gel, eluting with 60–100% ethyl acetate in hexanes to give <b>66</b> as a white solid (2.58 g, 91% yield). NMR (400 MHz, CDCl<sub>3</sub>) δ ppm 7.24 (1 H, d, <i>J</i> = 9.6 Hz), 6.94 (1 H, d, <i>J</i> = 9.6 Hz), 3.95–4.04 (2 H, m), 3.35 (2 H, ddd, <i>J</i> = 13.7, 10.0, 3.2 Hz), 3.23 (2 H, s), 2.18 (2 H, t, <i>J</i> = 7.6 Hz), 1.88–1.96 (2 H, m), 1.74–1.83 (2 H, m). MS (ESI) <i>m</i>/<i>z</i>: 302.0 [M + H]<sup>+</sup>.</div></div><div id="sec5_1_53" class="NLM_sec NLM_sec_level_3"><div id="ac_i77" class="anchor-spacer"></div><h4 class="article-section__title" id="_i77"> 8-(6-((9-((1<i>r</i>,4<i>r</i>)-4-Methylcyclohexyl)-9<i>H</i>-pyrido[4′,3′:4,5]pyrrolo[2,3-<i>d</i>]pyrimidin-2-yl)amino)pyridazin-3-yl)-2-thia-8-azaspiro[4.5]decane 2,2-Dioxide (<b>25</b>)</h4><div class="NLM_p last">Compound <b>25</b> was prepared from <b>33d</b> and <b>66</b> using similar chemistry described in the synthesis of <b>1</b>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 10.18 (1 H, s), 9.26 (1 H, s), 9.10 (1 H, s), 8.42 (1 H, d, <i>J</i> = 5.3 Hz), 8.19 (1 H, d, <i>J</i> = 9.6 Hz), 8.40 (1 H, d, <i>J</i> = 5.3 Hz), 7.44 (1 H, d, <i>J</i> = 9.6 Hz), 4.77 (1 H, m), 3.81–3.90 (2 H, m), 3.26–3.37 (2 H, m), 3.24 (2 H, t, <i>J</i> = 7.6 Hz), 3.17 (2 H, s), 2.50–2.55 (2 H, m), 2.09 (2 H, t, <i>J</i> = 7.6 Hz), 1.79–1.88 (6 H, m), 1.63–1.69 (3 H, m), 1.19–1.26 (2 H, m), 0.99 (3 H, d, <i>J</i> = 6.6 Hz). HRMS (ESI) <i>m</i>/<i>z</i>: calculated for [M + H]<sup>+</sup> 547.2597, found 547.2607.</div></div><div id="sec5_1_54" class="NLM_sec NLM_sec_level_3"><div id="ac_i78" class="anchor-spacer"></div><h4 class="article-section__title" id="_i78"> 1,8-Diazaspiro[4.5]decan-2-one hydrochloride (<b>68</b>)</h4><div class="NLM_p last">To a solution of <b>67</b> (268 mg, 1.06 mmol) in 1,4-dioxane (6 mL) was added 4 M HCl in dioxane (6 mL), and the solution thus obtained was stirred 2 h at 50 °C. The reaction mixture was concentrated, and the residue was dried under high vacuum to give <b>68</b> as a white solid (201 mg, 100% yield). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 8.85 (1 H, br s), 3.05–3.15 (4 H, m), 2.22 (2 H, t, <i>J</i> = 8.1 Hz), 1.88 (2 H, t, <i>J</i> = 8.1 Hz), 1.69–1.82 (4 H, m). MS (ESI) <i>m</i>/<i>z</i>: 155.1 [M + H]<sup>+</sup>.</div></div><div id="sec5_1_55" class="NLM_sec NLM_sec_level_3"><div id="ac_i79" class="anchor-spacer"></div><h4 class="article-section__title" id="_i79"> 8-(6-Chloropyridin-3-yl)-1,8-diazaspiro[4.5]decan-2-one (<b>69a</b>)</h4><div class="NLM_p last">Compound <b>69a</b> was prepared from <b>56a</b> and <b>68</b> using similar chemistry described in the synthesis of <b>34a</b>. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ ppm 8.04 (1 H, d, <i>J</i> = 2.7 Hz), 2.7 (1 H, s), 7.16–7.23 (2 H, m), 6.23 (1 H, br s), 3.19–3.31 (4 H, m), 2.45 (2 H, t, <i>J</i> = 8.1 Hz), 2.04 (2 H, t, <i>J</i> = 8.1 Hz), 1.83–1.89 (4 H, m). MS (ESI) <i>m</i>/<i>z</i>: 266.0 [M + H]<sup>+</sup>.</div></div><div id="sec5_1_56" class="NLM_sec NLM_sec_level_3"><div id="ac_i80" class="anchor-spacer"></div><h4 class="article-section__title" id="_i80"> 8-(6-Chloro-2-methoxypyridin-3-yl)-1,8-diazaspiro[4.5]decan-2-one (<b>69b</b>)</h4><div class="NLM_p last">Compound <b>69b</b> was prepared from <b>56d</b> and <b>68</b> using similar chemistry described in the synthesis of <b>34a</b>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 7.95 (1 H, br s), 7.23 (1 H, d, <i>J</i> = 8.1 Hz), 6.98 (1 H, d, <i>J</i> = 8.1 Hz), 3.87 (3 H, s), 3.08–3.15 (2 H, m), 2.80–2.90 (2 H, m), 2.21 (2 H, t, <i>J</i> = 8.0 Hz), 1.86 (2 H, t, <i>J</i> = 8.0 Hz), 1.60–1.75 (4 H, m). MS (ESI) <i>m</i>/<i>z</i>: 296.1 [M + H]<sup>+</sup>.</div></div><div id="sec5_1_57" class="NLM_sec NLM_sec_level_3"><div id="ac_i81" class="anchor-spacer"></div><h4 class="article-section__title" id="_i81"> 8-(6-((9-((1<i>r</i>,4<i>r</i>)-4-Methylcyclohexyl)-9<i>H</i>-pyrido[4′,3′:4,5]pyrrolo[2,3-<i>d</i>]pyrimidin-2-yl)amino)pyridin-3-yl)-1,8-diazaspiro[4.5]decan-2-one (<b>26</b>)</h4><div class="NLM_p last">Compound <b>26</b> was prepared from <b>33d</b> and <b>69a</b> using similar chemistry described in the synthesis of <b>1</b>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 9.73 (1 H, s), 9.26 (1 H, s), 9.10 (1 H, s), 8.42 (1 H, d, <i>J</i> = 8.1 Hz), 8.19 (1 H, d, <i>J</i> = 9.8 Hz), 8.04 (1 H, d, <i>J</i> = 5.0 Hz), 8.03 (1 H, d, <i>J</i> = 8.1 Hz), 8.00 (1 H, s), 7.48 (1 H, dd, <i>J</i> = 9.8, 5.0 Hz), 4.80 (1 H, m), 3.30–3.40 (2 H, m), 3.10–3.14 (2 H, m), 2.54–2.64 (2 H, m), 2.23 (2 H, t, <i>J</i> = 8.1 Hz), 1.90 (2 H, t, <i>J</i> = 8.1 Hz), 1.82–1.91 (4 H, m), 1.62–1.77 (5 H, m), 1.22–1.29 (2 H, m), 1.01 (3 H, d, <i>J</i> = 6.5 Hz). HRMS (ESI) <i>m</i>/<i>z</i>: calculated for [M + H]<sup>+</sup> 511.2927, found 511.2931.</div></div><div id="sec5_1_58" class="NLM_sec NLM_sec_level_3"><div id="ac_i82" class="anchor-spacer"></div><h4 class="article-section__title" id="_i82"> 8-(2-Methoxy-6-((9-((1<i>r</i>,4<i>r</i>)-4-methylcyclohexyl)-9<i>H</i>-pyrido[4′,3′:4,5]pyrrolo[2,3-<i>d</i>]pyrimidin-2-yl)amino)pyridin-3-yl)-1,8-diazaspiro[4.5]decan-2-one (<b>27</b>)</h4><div class="NLM_p last">Compound <b>27</b> was prepared from <b>33d</b> and <b>69b</b> using similar chemistry described in the synthesis of <b>1</b>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 9.55 (1 H, s), 9.29 (1 H, s), 9.12 (1 H, s), 8.43 (1 H, d, <i>J</i> = 5.3 Hz), 8.05 (1 H, d, <i>J</i> = 5.3 Hz), 7.97 (1 H, br s), 7.84 (1 H, d, <i>J</i> = 9.8 Hz), 7.28 (1 H, d, <i>J</i> = 9.8 Hz), 4.83 (1 H, m), 3.92 (3 H, s), 3.30–3.38 (2 H, m), 3.10–3.14 (2 H, m), 2.56–2.64 (2 H, m), 2.23 (2 H, d, <i>J</i> = 8.1 Hz), 1.90 (2 H, t, <i>J</i> = 8.1 Hz), 1.82–1.90 (4 H, m), 1.66–1.80 (5 H, m), 1.23–1.30 (2 H, m), 1.02 (3 H, d, <i>J</i> = 6.5 Hz). HRMS (ESI) <i>m</i>/<i>z</i>: calculated for [M + H]+ 541.3039, found 541.3040.</div></div><div id="sec5_1_59" class="NLM_sec NLM_sec_level_3"><div id="ac_i83" class="anchor-spacer"></div><h4 class="article-section__title" id="_i83"> <i>t</i>-Butyl 2-Chloro-7,8-dihydro-1,6-naphthyridine-6(5<i>H</i>)-carboxylate (<b>71</b>)</h4><div class="NLM_p last">To a slurry of 2-chloro-5,6,7,8-tetrahydro-1,6-naphthyridine hydrochloride (106.1 g, 517 mmol) and <i>N</i>,<i>N</i>-diisopropylethylamine (80 g, 108 mL, 621 mmol, 1.2 equiv) in DCM (1 L) was added a solution of di-<i>tert</i>-butyl dicarbonate (119 g, 543 mmol, 1.05 equiv) in DCM (100 mL) via an addition funnel within 1 h. The reaction mixture became a clear solution, and the solution thus obtained was stirred at room temperature for an additional hour and monitored using LCMS. Upon completion, the reaction mixture was concentrated. The residue was dissolved in ethyl acetate (1 L) and washed with water (3 × 300 mL), brine (300 mL), and dried over MgSO<sub>4</sub>. The solvent was evaporated under vacuum to give compound <b>71</b> as an off-white solid (139 g, 100% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm 7.38 (1 H, d, <i>J</i> = 8.0 Hz), 7.17 (1 H, d, <i>J</i> = 8.0 Hz), 4.57 (2 H, s), 3.73 (2 H, t, <i>J</i> = 5.9 Hz), 2.97 (2 H, t, <i>J</i> = 5.9 Hz), 1.49 (9H, s). LCMS (ESI) <i>m</i>/<i>z</i>: 269 [M + H]<sup>+</sup>.</div></div><div id="sec5_1_60" class="NLM_sec NLM_sec_level_3"><div id="ac_i84" class="anchor-spacer"></div><h4 class="article-section__title" id="_i84"> <i>t</i>-Butyl 2-((9-((1<i>r</i>,4<i>r</i>)-4-Methylcyclohexyl)-9<i>H</i>-pyrido[4′,3′:4,5]pyrrolo[2,3-<i>d</i>]pyrimidin-2-yl)amino)-7,8-dihydro-1,6-naphthyridine-6(5<i>H</i>)-carboxylate (<b>72</b>)</h4><div class="NLM_p last">To a solution of <b>33d</b> (2.81 g, 10 mmol) in 1,4-dioxane (45 mL) were added <b>71</b> (2.57 g, 9.55 mmol), XantPhos (231 mg, 0.40 mmol), and sodium <i>t</i>-butoxide (1.44 g, 15 mmol). Argon was bubbled through the mixture for 10 min. Pd2dba3 (183 mg, 0.20 mmol) was added, and argon was again bubbled through the mixture for 5 min. The reaction mixture thus obtained was stirred at 100 °C for 3 h, whereupon HPLC-MS analysis indicated that the reaction was complete. The reaction mixture was cooled to 40 °C, diluted with DCM (90 mL), and treated with Si-triamine (functionalized silica gel) (2.8 g) overnight at room temperature. Celite brand filter aid 545 (6 g) was added, the mixture was filtered with a sintered glass funnel, and the solid phase was rinsed with DCM (100 mL). The filtrate was concentrated to 25 mL on a rotary evaporator and diluted with a mixture of ethyl acetate and hexane (20 mL, 4:1). The resulting slurry was stirred at room temperature for 5 h. The solid was collected by filtration, washed with a mixture of ethyl acetate and hexane (20 mL, 1:1), and air-dried for a few hours to provide compound <b>72</b> as an off-white solid (4.90 g, 100% yield). <sup>1</sup>H NMR (500 MHz, CD<sub>2</sub>Cl<sub>2</sub>) δ ppm 9.10 (1 H, s), 8.97 (1 H, s), 8.46 (1 H, d, <i>J</i> = 5.1 Hz), 8.42 (1 H, d, <i>J</i> = 8.6 Hz), 8.10 (1 H, br s), 7.85 (1 H, d, <i>J</i> = 5.1 Hz), 7.51 (1 H, d, <i>J</i> = 8.6 Hz), 4.83 (1 H, m), 4.57 (2 H, s), 3.74 (2 H, t, <i>J</i> = 6.0 Hz), 2,88 (2 H, t, <i>J</i> = 6.0 Hz), 2.60–2.66 (2 H, m), 1.93–2.02 (4 H, m), 1.67 (1 H, m), 1.48 (9H, s), 1.22–1.34 (2 H, m), 1.06 (3H, d, <i>J</i> = 6.4 Hz). HRMS (ESI) <i>m</i>/<i>z</i>: calculated for [M + H]<sup>+</sup> 514.2925, found 514.2928.</div></div><div id="sec5_1_61" class="NLM_sec NLM_sec_level_3"><div id="ac_i85" class="anchor-spacer"></div><h4 class="article-section__title" id="_i85"> 9-((1<i>r</i>,4<i>r</i>)-4-Methylcyclohexyl)-<i>N</i>-(5,6,7,8-tetrahydro-1,6-naphthyridin-2-yl)-9<i>H</i>-pyrido[4′,3′:4,5]pyrrolo[2,3-<i>d</i>]pyrimidin-2-amine (<b>73</b>)</h4><div class="NLM_p last">To a suspension of <b>72</b> (4.65 g, 9.05 mmol) in MeOH (30 mL) were added concentrated HCl (6.74 mL) and water (14 mL). The mixture thus obtained was stirred at room temperature overnight. Analysis by HPLC-MS indicated that the reaction was complete. Then 50% NaOH in water (4.8 mL) was added at 0 °C to the reaction mixture to adjust the pH value to 9. The precipitated yellow solid was collected by filtration, rinsed with water (25 mL), and air-dried for 3 days to give compound <b>73</b> (3.75 g, 100% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm 9.08 (1 H, s), 8.96 (1 H, s), 8.50 (1 H, d, <i>J</i> = 5.3 Hz), 8.34 (1 H, d, <i>J</i> = 8.4 Hz), 8.03 (1 H, br s), 7.84 (1 H, d, <i>J</i> = 5.3 Hz), 7.41 (1 H, d, <i>J</i> = 8.4 Hz), 4.71 (1 H, m), 4.04 (2 H, s), 3.26 (2 H, t, <i>J</i> = 6.0 Hz), 2,89 (2 H, t, <i>J</i> = 6.0 Hz), 2.53–2.69 (2 H, m), 2.02 (2 H, s), 1.95–2.00 (3 H, m), 1.25–1.29 (3 H, m), 1.07 (3 H, d, <i>J</i> = 6.5 Hz). HRMS (ESI) <i>m</i>/<i>z</i>: calculated for [M + H]<sup>+</sup> 414.2401, found 414.2404.</div></div><div id="sec5_1_62" class="NLM_sec NLM_sec_level_3"><div id="ac_i86" class="anchor-spacer"></div><h4 class="article-section__title" id="_i86"> 2,5-Dioxopyrrolidin-1-yl 2-acetoxyacetate (<b>74</b>)</h4><div class="NLM_p last">A 3-neck round-bottom flask equipped with a mechanical stirrer, thermocouple, and addition funnel with nitrogen inlet was charged with <i>N</i>-hydroxysuccinimide (211 g, 1.83 mol) and DCM (2.25 L) at room temperature, resulting in a suspension. Pyridine (178 mL, 2.2 mol) was added in one portion with no change in the internal temperature. A solution of acetoxyacetyl chloride (197 mL, 1.83 mol) in DCM (225 mL) was added dropwise over 60 min, and the temperature rose to 35 °C. Stirring was continued at room temperature for 2.5 h. The stirring was stopped, and the reaction mixture was washed with water (1 L), 1 N HCl (2 L), and brine (1 L). The organic layer was concentrated under vacuum and azeotroped with toluene (1 L) to obtain compound <b>74</b> as a white solid (367 g, 93% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm 4.96 (2 H, s), 2.86 (4 H, s), 2.19 (3 H, s) ppm. LCMS <i>m</i>/<i>z</i>: 238.0 [M + Na]<sup>+</sup>.</div></div><div id="sec5_1_63" class="NLM_sec NLM_sec_level_3"><div id="ac_i87" class="anchor-spacer"></div><h4 class="article-section__title" id="_i87"> 2-(2-((9-((1<i>r</i>,4<i>r</i>)-4-Methylcyclohexyl)-9<i>H</i>-pyrido[4′,3′:4,5]pyrrolo[2,3-<i>d</i>]pyrimidin-2-yl)amino)-7,8-dihydro-1,6-naphthyridin-6(5<i>H</i>)-yl)-2-oxoethyl acetate (<b>75</b>)</h4><div class="NLM_p last">To a suspension of <b>73</b> (827 mg, 2.0 mmol) in chloroform (10 mL) were added diisopropylethylamine (258 mg, 348 uL, 2.0 mmol) and <b>74</b> (560 mg, 2.6 mmol). The reaction mixture thus obtained was stirred at room temperature for 30 min, whereupon the mixture became a yellow solution. HPLC-MS analysis indicated that the reaction was complete. The reaction mixture was concentrated. MeOH (5 mL) and water (6 mL) were added to form a slurry, which was stirred at room temperature for 1 h. The solid was collected by filtration to give compound <b>75</b> as a light-yellow solid (1.04 g, 98% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>, rotamers) δ ppm 9.11 (1 H, s), 8.98 (1 H, s), 8.52 (1 H, d, <i>J</i> = 5.3 Hz), 8.40–8.49 (1 H, m), 8.10–8.21 (1 H, m), 7.86 (1 h, d, <i>J</i> = 5.3 Hz), 7.45–7.57 (1 H, m), 4.82–4.90 (2 H, m), 4.65–4.80 (2 H, m), 4.59 (1 H, m), 3.97 (1 H, t, <i>J</i> = 5.9 Hz), 3.75 (1 H, t, <i>J</i> = 5.9 Hz), 2.93–3.08 (2 H, m), 2.52–2.69 (2 H, m), 2.22 (3 H, s), 1.97–2.03 (4 H, m), 1.20–1.37 (2 H, m), 1.08 (3 H, d, <i>J</i> = 6.5 Hz). HRMS (ESI) <i>m</i>/<i>z</i>: calculated for [M + H]<sup>+</sup> 514.2561, found 514.2563.</div></div><div id="sec5_1_64" class="NLM_sec NLM_sec_level_3"><div id="ac_i88" class="anchor-spacer"></div><h4 class="article-section__title" id="_i88"> 2-Hydroxy-1-(2-((9-((1<i>r</i>,4<i>r</i>)-4-methylcyclohexyl)-9<i>H</i>-pyrido[4′,3′:4,5]pyrrolo[2,3-<i>d</i>]pyrimidin-2-yl)amino)-7,8-dihydro-1,6-naphthyridin-6(5<i>H</i>)-yl)ethanone (<b>28</b>)</h4><div class="NLM_p last">To a solution of <b>75</b> (514 mg, 1.0 mmol) in DCM (7.5 mL) and MeOH (2.5 mL) was added 0.5 M sodium methoxide solution in MeOH (0.30 mL, 0.15 mmol), and the reaction mixture was stirred at room temperature for 1 h, monitored using LCMS. Upon completion, the reaction mixture was concentrated. The residue was treated with EtOH (5 mL) and water (10 mL) to provide a solid which was collected by filtration, washed with water, and dried in a vacuum oven at 55 °C overnight to give compound <b>28</b> as a white solid (468 mg, 99% yield). <sup>1</sup>H NMR (500 MHz, acetic acid-<i>d</i><sub>4</sub>, 373 K) δ ppm 9.43 (1 H, s), 9.35 (1 H, s), 8.63 (1H, d, <i>J</i> = 6.0 Hz), 8.40 (1 H, d, <i>J</i> = 6.0 Hz), 8.33 (1 H, d, <i>J</i> = 8.5 Hz), 7.76 (1 H, d, <i>J</i> = 8.5 Hz), 4.90 (1 H, m), 4.77 (2 H, br s), 4.46 (2 H, br s), 3.88 (2 H, br s), 3.10 (2 H, t, <i>J</i> = 5.4 Hz), 2.68 (2 H, dq, <i>J</i> = 12.7, 3.3 Hz), 2.06–2.13 (2 H, m), 1.99–2.03 (2 H, m), 1.70–1.80 (1 H, m), 1.31–1.43 (2 H, m), 1.09 (3H, d, <i>J</i> = 6.5 Hz). HRMS (ESI) <i>m</i>/<i>z</i>: calculated for [M + H]<sup>+</sup> 472.2455, found 472.2461.</div></div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i89" class="anchor-spacer"></div><h3 class="article-section__title" id="_i89"> FLT3 Kinase Assay</h3><div class="NLM_p last">The inhibitory activity of FLT3 was determined using a time-resolved fluorescence resonance energy transfer (TR-FRET) assay. The FLT3 enzyme (GST-fused human FLT3 cytoplasmic domain, amino acids 564–993) was purchased from Carna Biosciences (Natick, MA). An ULight-labeled synthetic peptide (ULight-JAK1, PerkinElmer, Waltham, MA) derived from human Janus kinase 1 (amino acids 1015–1027) was used as the phosphoacceptor substrate. The FLT3 assay was conducted in a 384-well white OptiPlate (PerkinElmer) in a total volume of 20 μL. The reaction mixture contained 50 nM ULight-JAK1, 116 μM ATP (equal to <i>K</i><sub>m</sub>), 0.5 nM FLT3, and serially diluted test compounds in a reaction buffer of 50 mM Hepes, pH 7.6, 1 mM EGTA, 10 mM MgCl<sub>2</sub>, 2 mM DTT, and 0.005% Tween 20. The reaction was allowed to proceed for 1 h at room temperature and stopped by adding 20 μL of 20 mM EDTA and 4 nM LANCE Eu-W1024 antiphospho-tyrosine antibody (PerkinElmer) in LANCE detection buffer (PerkinElmer). The plates were incubated at room temperature for 2 h after addition of detection reagents and were then read on an Envision multimode reader (PerkinElmer). Fluorescence signals measured at 615 nm (8.5 nm bandwidth) and 665 nm (7.5 nm bandwidth) with a 60 μs delay after excitation at 320 nm (75 nm bandwidth). The signal ratio at 665/615 nm (APC/Eu) was used in all data analyses. IC<sub>50</sub> values were obtained by analyzing competition curves using a four-parameter sigmoidal model in GraphPad Prism v5.01 (GraphPad).</div></div><div id="sec5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i90" class="anchor-spacer"></div><h3 class="article-section__title" id="_i90"> CDK Kinases Assay</h3><div class="NLM_p last">CDK4/Cyclin D<sub>1</sub>, CDK6/Cyclin D<sub>1</sub>, CDK1/Cyclin B, and CDK2/Cyclin A were purchased from Cell Signaling Technology (Danvers, MA). For CDK4 and CDK6 kinase assays, Rb (amino acids 773–928), and histone H1 were from Millipore (Bedford, MA) and used as substrate for CDK4/6 and CDK1/2, respectively. [r-<sup>33</sup>P]-ATP was purchased from PerkinElmer (Shelton, CT). Assays were performed in 96-well filter plates (MSDVN6B50, Millipore, Bedford, MA) with a final volume of 100 μL, containing 1 μg Rb, 25 ng CDK4/cyclin D1, 25 μM ATP, 1 μCi [r-<sup>33</sup>P]-ATP, and compound in kinase reaction buffer (20 mM Tris-HCl, pH 7.4, 10 mM magnesium chloride, 5 mM β-glycerophosphate, 1 mM dithiothreitol, and 0.1% bovine serum albumin). Plates with reaction mix were incubated at room temperature for 60 min, and reactions were terminated by addition of 200 μL of 20% trichloroacetic acid. Wells were washed with 200 μL of 10% trichloroacetic acid and left to dry at room temperature. The bottom of the plates were sealed with tape, and 100 μL of MicroScint-20 scintillation cocktail (PerkinElmer, Shelton, CT) were added to each well. Plates were read on a TopCount (PerkinElmer, Shelton, CT) to determine radioactive incorporation. IC<sub>50</sub> values were calculated by nonlinear regression curve fitting using GraphPad Prism v5.01 (GraphPad).</div></div><div id="sec5_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i91" class="anchor-spacer"></div><h3 class="article-section__title" id="_i91"> Cell Growth Inhibition Assay</h3><div class="NLM_p last">Cell growth was measured by a DNA synthesis assay. Cells were seeded in a 96-well Cytostar T plate (GE Healthcare Biosciences, Piscataway, NJ) at a density of 5 × 10<sup>3</sup> cells/well in a total volume of 160 μL. The plates were incubated overnight to allow cells to attach. Test compounds were serially diluted into the plate (20 μL/well), and 20 μL/0.1 μCi of [<sup>14</sup>C]-thymidine (GE Healthcare Biosciences, Piscataway, NJ) were added to each well. Isotope incorporation was determined using a β plate counter (Wallac, Gaithersburg, MD) after a further 72 h. IC<sub>50</sub> values were determined as described above.</div></div><div id="sec5_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i92" class="anchor-spacer"></div><h3 class="article-section__title" id="_i92"> Analysis of pSTAT5 and pRb in Vitro</h3><div class="NLM_p last">To determine levels of pSTAT5 and pRb, cells were seeded in a 96-well cell culture plate at a density of 5000–10000 cells per well in a total volume of 180 μL and incubated overnight in medium containing 10% fetal bovine serum (Life Technologies, Rockville, MD). Then 20 μL of serially diluted compounds were added to each well the next day. Cell lysates were harvested after 24 h and pSTAT5 or pRb was quantified using assay kits for pSTAT5 (Tyr694)/total STAT5 and pRb (Ser780)/total Rb (Meso Scale Discovery) following the manufacturer’s protocols. Finally, 10 μg of total protein from each sample was analyzed.</div></div><div id="sec5_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i93" class="anchor-spacer"></div><h3 class="article-section__title" id="_i93"> Apoptosis Assay</h3><div class="NLM_p last">Apoptosis of the AML cells treated with compounds was assayed by using a Vybrant Apoptosis Assay Kit no. 9 following the manufacturer’s protocol (Invitrogen, catalogue no. V35113). Briefly, the MOLM13, MOLM13<sup>SR</sup>, or U937 cells in exponential growth phase were seeded into a 6-well plate at 5 × 10<sup>5</sup> cells per well and treated with compounds at indicated concentrations for 24 h. Cells were then stained with the reagents in the kit and analyzed by flow cytometry. The Sytox Green fluorescence versus APC fluorescence dot plot shows resolution of live, apoptotic and dead cells, which were quantified using Flowjo software.</div></div><div id="sec5_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i94" class="anchor-spacer"></div><h3 class="article-section__title" id="_i94"> Sorafenib-Resistant MOLM13 Cells</h3><div class="NLM_p last">Sorafenib resistant MOLM13 (MOLM13<sup>SR</sup>) was isolated by passaging the cells in growth medium containing increasing concentrations of sorafenib (1–1000 nM) over 3–4 months. The resistant cells were cloned by limiting dilution. RNA was isolated from each clone, and cDNA was sequenced to identify FLT3 kinase domain mutations.</div></div><div id="sec5_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i95" class="anchor-spacer"></div><h3 class="article-section__title" id="_i95"> Xenograft Tumor Models</h3><div class="NLM_p last">CrTac:NCR-<i>Foxn1</i><sup><i>nu</i></sup> (NCR) nude mice were treated with ip injection of 100 μL of antiasialo GM (WAKO Chemicals) antibody to abolish NK activity and allow for enhanced growth of subsequently inoculated tumor cells. The following day, 7.5 million MOLM13 or U937 tumor cells in PBS were formulated as a 1:1 mixture with matrigel (BD Biosciences) and injected into the subcutaneous space on the right flank of the mice. Tumors were measured with PRO-MAX electronic digital caliper (Japan Micrometer Mfg. Co. LTD), and the mice were weighed every other day prior to each tumor measurement. Tumor volumes were calculated as follows: tumor volume (mm<sup>3</sup>) = [(<i>W</i><sup>2</sup> × <i>L</i>)/2], where width (<i>W</i>) is defined as the smaller of the two measurements and length (<i>L</i>) is defined as the larger of the two measurements. Tumor growth inhibition (TGI) is calculated as 100 – [(measured – initial volume)/(control – initial volume) × 100].</div></div><div id="sec5_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i96" class="anchor-spacer"></div><h3 class="article-section__title" id="_i96"> Statistical Analysis of Tumor Growth in Vivo</h3><div class="NLM_p last">Tumor volumes are expressed as means plus or minus standard errors and plotted as a function of time. Statistical significance of observed differences between growth curves was evaluated by repeated measures analysis of covariance of the log transformed tumor volume data with Dunnett adjusted multiple comparisons. The analysis was done using SAS proc mixed with model effects of baseline log tumor volume, day, treatment, and day-by-treatment interaction; a repeated statement where day was a repeated value, animal the subject, and a Toeplitz covariance structure; and an lsmeans statement to do a Dunnett analysis comparing the control group to the other treatment groups. The data was log transformed because larger volumes tended to have larger variances, and baseline log tumor volume was included as a covariate in the model to account for possible pretreatment tumor volume differences. All statistical calculations were made through the use of JMP software v7.0 interfaced with SAS v9.1 (SAS Institute Inc., Cary, NC).</div></div><div id="sec5_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i97" class="anchor-spacer"></div><h3 class="article-section__title" id="_i97"> Analysis of pRb and pSTAT5 in Tumor Samples</h3><div class="NLM_p last">Tumors were harvested by dissection and snap-frozen. Frozen tumors were weighed and lysed in 150 mM NaCl, 20 mM Tris pH 7.5, 1 mM EDTA, 1 mM EGTA, 1% Triton-X-100, and 2× protease and phosphatase inhibitors (Meso Scale Discovery) at twice the volume of tumor mass. Samples were processed on a Genogrinder (Spec SamplePrep) and shaken twice for 30 s at 1200 rpm with cooling on ice between shaking. Samples were then centrifuged at 1200 rpm, and cleared lysates were analyzed using assay kits for phosphorylated STAT5 (Tyr694), total STAT5a and b, phosphorylated Rb (Ser780)/total Rb, and total p38<sup>MAPK</sup>.</div></div><div id="sec5_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i98" class="anchor-spacer"></div><h3 class="article-section__title" id="_i98"> X-ray Crystallography</h3><div class="NLM_p last">The CDK6-Vcyclin complex was expressed and crystallized, using minor modifications on the methods described by Lu and Schulze-Gahmen.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> In short, the CDK6 and Vcyclin proteins were expressed separately in baculovirus infected Hi5 cells. The proteins were mixed, and the complex was purified. The purified complex was concentrated to 10–15 mg/mL, and unliganded CDK6-Vcyclin was crystallized from 10 mM Tris pH 7.9, 100 mM calcium acetate, 10 mM DTT, 8–12% PEG 3350, and 100 mM NDSB-201. Crystals were soaked with 0.25 mM compound <b>1</b> and cryoprotected with glycerol prior to flash cooling. A 2.9 Å X-ray diffraction data set was collected at beamline 5.0.2 at the Berkeley Advanced Light Source and processed with iMOSFLM.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> The structure was solved by molecular replacement with MOLREP<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> using 2EUF as the search model. Final refinement and model building were carried out using PHENIX<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> and Coot,<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> respectively. Atomic coordinates have been deposited in the RCSB Protein Data Bank under accession code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4P41">4P41</a>.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i99"><a href="/doi/suppl/10.1021/jm500118j">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_48844" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_48844" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">POC kinase selectivity data of <b>28</b>, <i>K</i><sub>d</sub> data for <b>28</b> for the set of kinases with POC less than 20 from the initial POC screen, terminal exposure levels of <b>28</b> in the MOLM13 xenograft model, and average body weights for the <b>28</b> treated mouse group in the MOLM13 xenograft model.  This material is available free of charge via the Internet at <a href="http://pubs.acs.org" class="extLink">http://pubs.acs.org</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm500118j/suppl_file/jm500118j_si_001.pdf">jm500118j_si_001.pdf (220.09 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The coordinates of <b>1</b> with CDK6 have been deposited in the PDB with accession code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4P41">4P41</a>.</p><div class="testing" data-doi="10.1021/jm500118j" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_35846" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_35846" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Zhihong Li</span> - <span class="hlFld-Affiliation affiliation">Departments of †Therapeutic
Discovery, ‡Oncology Research, §Pharmaceutics, and ∥Pharmacokinetics and Drug Metabolism, Amgen
Inc., 1120 Veterans Boulevard, South San Francisco, California 94080, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Therapeutic
Innovation Unit, Amgen Inc., 1201 Amgen Court West, Seattle, Washington 98119, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Discovery
Research, Amgen Inc., One Amgen Center Drive, Thousand Oaks, Callifornia 91320, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#99e3f1f0f1f6f7fef5d9f8f4fefcf7b7faf6f4"><span class="__cf_email__" data-cfemail="c8b2a0a1a0a7a6afa488a9a5afada6e6aba7a5">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xianghong Wang</span> - <span class="hlFld-Affiliation affiliation">Departments of †Therapeutic
Discovery, ‡Oncology Research, §Pharmaceutics, and ∥Pharmacokinetics and Drug Metabolism, Amgen
Inc., 1120 Veterans Boulevard, South San Francisco, California 94080, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Therapeutic
Innovation Unit, Amgen Inc., 1201 Amgen Court West, Seattle, Washington 98119, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Discovery
Research, Amgen Inc., One Amgen Center Drive, Thousand Oaks, Callifornia 91320, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">John Eksterowicz</span> - <span class="hlFld-Affiliation affiliation">Departments of †Therapeutic
Discovery, ‡Oncology Research, §Pharmaceutics, and ∥Pharmacokinetics and Drug Metabolism, Amgen
Inc., 1120 Veterans Boulevard, South San Francisco, California 94080, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Therapeutic
Innovation Unit, Amgen Inc., 1201 Amgen Court West, Seattle, Washington 98119, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Discovery
Research, Amgen Inc., One Amgen Center Drive, Thousand Oaks, Callifornia 91320, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Michael W. Gribble</span> - <span class="hlFld-Affiliation affiliation">Departments of †Therapeutic
Discovery, ‡Oncology Research, §Pharmaceutics, and ∥Pharmacokinetics and Drug Metabolism, Amgen
Inc., 1120 Veterans Boulevard, South San Francisco, California 94080, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Therapeutic
Innovation Unit, Amgen Inc., 1201 Amgen Court West, Seattle, Washington 98119, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Discovery
Research, Amgen Inc., One Amgen Center Drive, Thousand Oaks, Callifornia 91320, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Grace Q. Alba</span> - <span class="hlFld-Affiliation affiliation">Departments of †Therapeutic
Discovery, ‡Oncology Research, §Pharmaceutics, and ∥Pharmacokinetics and Drug Metabolism, Amgen
Inc., 1120 Veterans Boulevard, South San Francisco, California 94080, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Therapeutic
Innovation Unit, Amgen Inc., 1201 Amgen Court West, Seattle, Washington 98119, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Discovery
Research, Amgen Inc., One Amgen Center Drive, Thousand Oaks, Callifornia 91320, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Merrill Ayres</span> - <span class="hlFld-Affiliation affiliation">Departments of †Therapeutic
Discovery, ‡Oncology Research, §Pharmaceutics, and ∥Pharmacokinetics and Drug Metabolism, Amgen
Inc., 1120 Veterans Boulevard, South San Francisco, California 94080, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Therapeutic
Innovation Unit, Amgen Inc., 1201 Amgen Court West, Seattle, Washington 98119, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Discovery
Research, Amgen Inc., One Amgen Center Drive, Thousand Oaks, Callifornia 91320, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Timothy J. Carlson</span> - <span class="hlFld-Affiliation affiliation">Departments of †Therapeutic
Discovery, ‡Oncology Research, §Pharmaceutics, and ∥Pharmacokinetics and Drug Metabolism, Amgen
Inc., 1120 Veterans Boulevard, South San Francisco, California 94080, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Therapeutic
Innovation Unit, Amgen Inc., 1201 Amgen Court West, Seattle, Washington 98119, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Discovery
Research, Amgen Inc., One Amgen Center Drive, Thousand Oaks, Callifornia 91320, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ada Chen</span> - <span class="hlFld-Affiliation affiliation">Departments of †Therapeutic
Discovery, ‡Oncology Research, §Pharmaceutics, and ∥Pharmacokinetics and Drug Metabolism, Amgen
Inc., 1120 Veterans Boulevard, South San Francisco, California 94080, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Therapeutic
Innovation Unit, Amgen Inc., 1201 Amgen Court West, Seattle, Washington 98119, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Discovery
Research, Amgen Inc., One Amgen Center Drive, Thousand Oaks, Callifornia 91320, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xiaoqi Chen</span> - <span class="hlFld-Affiliation affiliation">Departments of †Therapeutic
Discovery, ‡Oncology Research, §Pharmaceutics, and ∥Pharmacokinetics and Drug Metabolism, Amgen
Inc., 1120 Veterans Boulevard, South San Francisco, California 94080, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Therapeutic
Innovation Unit, Amgen Inc., 1201 Amgen Court West, Seattle, Washington 98119, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Discovery
Research, Amgen Inc., One Amgen Center Drive, Thousand Oaks, Callifornia 91320, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Robert Cho</span> - <span class="hlFld-Affiliation affiliation">Departments of †Therapeutic
Discovery, ‡Oncology Research, §Pharmaceutics, and ∥Pharmacokinetics and Drug Metabolism, Amgen
Inc., 1120 Veterans Boulevard, South San Francisco, California 94080, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Therapeutic
Innovation Unit, Amgen Inc., 1201 Amgen Court West, Seattle, Washington 98119, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Discovery
Research, Amgen Inc., One Amgen Center Drive, Thousand Oaks, Callifornia 91320, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Richard V. Connors</span> - <span class="hlFld-Affiliation affiliation">Departments of †Therapeutic
Discovery, ‡Oncology Research, §Pharmaceutics, and ∥Pharmacokinetics and Drug Metabolism, Amgen
Inc., 1120 Veterans Boulevard, South San Francisco, California 94080, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Therapeutic
Innovation Unit, Amgen Inc., 1201 Amgen Court West, Seattle, Washington 98119, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Discovery
Research, Amgen Inc., One Amgen Center Drive, Thousand Oaks, Callifornia 91320, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Michael DeGraffenreid</span> - <span class="hlFld-Affiliation affiliation">Departments of †Therapeutic
Discovery, ‡Oncology Research, §Pharmaceutics, and ∥Pharmacokinetics and Drug Metabolism, Amgen
Inc., 1120 Veterans Boulevard, South San Francisco, California 94080, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Therapeutic
Innovation Unit, Amgen Inc., 1201 Amgen Court West, Seattle, Washington 98119, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Discovery
Research, Amgen Inc., One Amgen Center Drive, Thousand Oaks, Callifornia 91320, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jeffrey T. Deignan</span> - <span class="hlFld-Affiliation affiliation">Departments of †Therapeutic
Discovery, ‡Oncology Research, §Pharmaceutics, and ∥Pharmacokinetics and Drug Metabolism, Amgen
Inc., 1120 Veterans Boulevard, South San Francisco, California 94080, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Therapeutic
Innovation Unit, Amgen Inc., 1201 Amgen Court West, Seattle, Washington 98119, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Discovery
Research, Amgen Inc., One Amgen Center Drive, Thousand Oaks, Callifornia 91320, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jason Duquette</span> - <span class="hlFld-Affiliation affiliation">Departments of †Therapeutic
Discovery, ‡Oncology Research, §Pharmaceutics, and ∥Pharmacokinetics and Drug Metabolism, Amgen
Inc., 1120 Veterans Boulevard, South San Francisco, California 94080, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Therapeutic
Innovation Unit, Amgen Inc., 1201 Amgen Court West, Seattle, Washington 98119, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Discovery
Research, Amgen Inc., One Amgen Center Drive, Thousand Oaks, Callifornia 91320, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Pingchen Fan</span> - <span class="hlFld-Affiliation affiliation">Departments of †Therapeutic
Discovery, ‡Oncology Research, §Pharmaceutics, and ∥Pharmacokinetics and Drug Metabolism, Amgen
Inc., 1120 Veterans Boulevard, South San Francisco, California 94080, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Therapeutic
Innovation Unit, Amgen Inc., 1201 Amgen Court West, Seattle, Washington 98119, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Discovery
Research, Amgen Inc., One Amgen Center Drive, Thousand Oaks, Callifornia 91320, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Benjamin Fisher</span> - <span class="hlFld-Affiliation affiliation">Departments of †Therapeutic
Discovery, ‡Oncology Research, §Pharmaceutics, and ∥Pharmacokinetics and Drug Metabolism, Amgen
Inc., 1120 Veterans Boulevard, South San Francisco, California 94080, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Therapeutic
Innovation Unit, Amgen Inc., 1201 Amgen Court West, Seattle, Washington 98119, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Discovery
Research, Amgen Inc., One Amgen Center Drive, Thousand Oaks, Callifornia 91320, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jiasheng Fu</span> - <span class="hlFld-Affiliation affiliation">Departments of †Therapeutic
Discovery, ‡Oncology Research, §Pharmaceutics, and ∥Pharmacokinetics and Drug Metabolism, Amgen
Inc., 1120 Veterans Boulevard, South San Francisco, California 94080, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Therapeutic
Innovation Unit, Amgen Inc., 1201 Amgen Court West, Seattle, Washington 98119, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Discovery
Research, Amgen Inc., One Amgen Center Drive, Thousand Oaks, Callifornia 91320, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Justin N. Huard</span> - <span class="hlFld-Affiliation affiliation">Therapeutic
Innovation Unit, Amgen Inc., 1201 Amgen Court West, Seattle, Washington 98119, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jacob Kaizerman</span> - <span class="hlFld-Affiliation affiliation">Departments of †Therapeutic
Discovery, ‡Oncology Research, §Pharmaceutics, and ∥Pharmacokinetics and Drug Metabolism, Amgen
Inc., 1120 Veterans Boulevard, South San Francisco, California 94080, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Therapeutic
Innovation Unit, Amgen Inc., 1201 Amgen Court West, Seattle, Washington 98119, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Discovery
Research, Amgen Inc., One Amgen Center Drive, Thousand Oaks, Callifornia 91320, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kathleen S. Keegan</span> - <span class="hlFld-Affiliation affiliation">Therapeutic
Innovation Unit, Amgen Inc., 1201 Amgen Court West, Seattle, Washington 98119, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Cong Li</span> - <span class="hlFld-Affiliation affiliation">Departments of †Therapeutic
Discovery, ‡Oncology Research, §Pharmaceutics, and ∥Pharmacokinetics and Drug Metabolism, Amgen
Inc., 1120 Veterans Boulevard, South San Francisco, California 94080, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Therapeutic
Innovation Unit, Amgen Inc., 1201 Amgen Court West, Seattle, Washington 98119, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Discovery
Research, Amgen Inc., One Amgen Center Drive, Thousand Oaks, Callifornia 91320, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kexue Li</span> - <span class="hlFld-Affiliation affiliation">Departments of †Therapeutic
Discovery, ‡Oncology Research, §Pharmaceutics, and ∥Pharmacokinetics and Drug Metabolism, Amgen
Inc., 1120 Veterans Boulevard, South San Francisco, California 94080, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Therapeutic
Innovation Unit, Amgen Inc., 1201 Amgen Court West, Seattle, Washington 98119, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Discovery
Research, Amgen Inc., One Amgen Center Drive, Thousand Oaks, Callifornia 91320, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yunxiao Li</span> - <span class="hlFld-Affiliation affiliation">Departments of †Therapeutic
Discovery, ‡Oncology Research, §Pharmaceutics, and ∥Pharmacokinetics and Drug Metabolism, Amgen
Inc., 1120 Veterans Boulevard, South San Francisco, California 94080, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Therapeutic
Innovation Unit, Amgen Inc., 1201 Amgen Court West, Seattle, Washington 98119, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Discovery
Research, Amgen Inc., One Amgen Center Drive, Thousand Oaks, Callifornia 91320, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lingming Liang</span> - <span class="hlFld-Affiliation affiliation">Departments of †Therapeutic
Discovery, ‡Oncology Research, §Pharmaceutics, and ∥Pharmacokinetics and Drug Metabolism, Amgen
Inc., 1120 Veterans Boulevard, South San Francisco, California 94080, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Therapeutic
Innovation Unit, Amgen Inc., 1201 Amgen Court West, Seattle, Washington 98119, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Discovery
Research, Amgen Inc., One Amgen Center Drive, Thousand Oaks, Callifornia 91320, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Wen Liu</span> - <span class="hlFld-Affiliation affiliation">Departments of †Therapeutic
Discovery, ‡Oncology Research, §Pharmaceutics, and ∥Pharmacokinetics and Drug Metabolism, Amgen
Inc., 1120 Veterans Boulevard, South San Francisco, California 94080, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Therapeutic
Innovation Unit, Amgen Inc., 1201 Amgen Court West, Seattle, Washington 98119, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Discovery
Research, Amgen Inc., One Amgen Center Drive, Thousand Oaks, Callifornia 91320, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sarah E. Lively</span> - <span class="hlFld-Affiliation affiliation">Departments of †Therapeutic
Discovery, ‡Oncology Research, §Pharmaceutics, and ∥Pharmacokinetics and Drug Metabolism, Amgen
Inc., 1120 Veterans Boulevard, South San Francisco, California 94080, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Therapeutic
Innovation Unit, Amgen Inc., 1201 Amgen Court West, Seattle, Washington 98119, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Discovery
Research, Amgen Inc., One Amgen Center Drive, Thousand Oaks, Callifornia 91320, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mei-Chu Lo</span> - <span class="hlFld-Affiliation affiliation">Departments of †Therapeutic
Discovery, ‡Oncology Research, §Pharmaceutics, and ∥Pharmacokinetics and Drug Metabolism, Amgen
Inc., 1120 Veterans Boulevard, South San Francisco, California 94080, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Therapeutic
Innovation Unit, Amgen Inc., 1201 Amgen Court West, Seattle, Washington 98119, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Discovery
Research, Amgen Inc., One Amgen Center Drive, Thousand Oaks, Callifornia 91320, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ji Ma</span> - <span class="hlFld-Affiliation affiliation">Departments of †Therapeutic
Discovery, ‡Oncology Research, §Pharmaceutics, and ∥Pharmacokinetics and Drug Metabolism, Amgen
Inc., 1120 Veterans Boulevard, South San Francisco, California 94080, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Therapeutic
Innovation Unit, Amgen Inc., 1201 Amgen Court West, Seattle, Washington 98119, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Discovery
Research, Amgen Inc., One Amgen Center Drive, Thousand Oaks, Callifornia 91320, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dustin L. McMinn</span> - <span class="hlFld-Affiliation affiliation">Departments of †Therapeutic
Discovery, ‡Oncology Research, §Pharmaceutics, and ∥Pharmacokinetics and Drug Metabolism, Amgen
Inc., 1120 Veterans Boulevard, South San Francisco, California 94080, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Therapeutic
Innovation Unit, Amgen Inc., 1201 Amgen Court West, Seattle, Washington 98119, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Discovery
Research, Amgen Inc., One Amgen Center Drive, Thousand Oaks, Callifornia 91320, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jeffrey T. Mihalic</span> - <span class="hlFld-Affiliation affiliation">Departments of †Therapeutic
Discovery, ‡Oncology Research, §Pharmaceutics, and ∥Pharmacokinetics and Drug Metabolism, Amgen
Inc., 1120 Veterans Boulevard, South San Francisco, California 94080, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Therapeutic
Innovation Unit, Amgen Inc., 1201 Amgen Court West, Seattle, Washington 98119, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Discovery
Research, Amgen Inc., One Amgen Center Drive, Thousand Oaks, Callifornia 91320, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kriti Modi</span> - <span class="hlFld-Affiliation affiliation">Departments of †Therapeutic
Discovery, ‡Oncology Research, §Pharmaceutics, and ∥Pharmacokinetics and Drug Metabolism, Amgen
Inc., 1120 Veterans Boulevard, South San Francisco, California 94080, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Therapeutic
Innovation Unit, Amgen Inc., 1201 Amgen Court West, Seattle, Washington 98119, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Discovery
Research, Amgen Inc., One Amgen Center Drive, Thousand Oaks, Callifornia 91320, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rachel Ngo</span> - <span class="hlFld-Affiliation affiliation">Departments of †Therapeutic
Discovery, ‡Oncology Research, §Pharmaceutics, and ∥Pharmacokinetics and Drug Metabolism, Amgen
Inc., 1120 Veterans Boulevard, South San Francisco, California 94080, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Therapeutic
Innovation Unit, Amgen Inc., 1201 Amgen Court West, Seattle, Washington 98119, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Discovery
Research, Amgen Inc., One Amgen Center Drive, Thousand Oaks, Callifornia 91320, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kanaka Pattabiraman</span> - <span class="hlFld-Affiliation affiliation">Departments of †Therapeutic
Discovery, ‡Oncology Research, §Pharmaceutics, and ∥Pharmacokinetics and Drug Metabolism, Amgen
Inc., 1120 Veterans Boulevard, South San Francisco, California 94080, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Therapeutic
Innovation Unit, Amgen Inc., 1201 Amgen Court West, Seattle, Washington 98119, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Discovery
Research, Amgen Inc., One Amgen Center Drive, Thousand Oaks, Callifornia 91320, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Derek E. Piper</span> - <span class="hlFld-Affiliation affiliation">Departments of †Therapeutic
Discovery, ‡Oncology Research, §Pharmaceutics, and ∥Pharmacokinetics and Drug Metabolism, Amgen
Inc., 1120 Veterans Boulevard, South San Francisco, California 94080, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Therapeutic
Innovation Unit, Amgen Inc., 1201 Amgen Court West, Seattle, Washington 98119, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Discovery
Research, Amgen Inc., One Amgen Center Drive, Thousand Oaks, Callifornia 91320, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Christophe Queva</span> - <span class="hlFld-Affiliation affiliation">Therapeutic
Innovation Unit, Amgen Inc., 1201 Amgen Court West, Seattle, Washington 98119, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mark L. Ragains</span> - <span class="hlFld-Affiliation affiliation">Departments of †Therapeutic
Discovery, ‡Oncology Research, §Pharmaceutics, and ∥Pharmacokinetics and Drug Metabolism, Amgen
Inc., 1120 Veterans Boulevard, South San Francisco, California 94080, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Therapeutic
Innovation Unit, Amgen Inc., 1201 Amgen Court West, Seattle, Washington 98119, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Discovery
Research, Amgen Inc., One Amgen Center Drive, Thousand Oaks, Callifornia 91320, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Julia Suchomel</span> - <span class="hlFld-Affiliation affiliation">Departments of †Therapeutic
Discovery, ‡Oncology Research, §Pharmaceutics, and ∥Pharmacokinetics and Drug Metabolism, Amgen
Inc., 1120 Veterans Boulevard, South San Francisco, California 94080, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Therapeutic
Innovation Unit, Amgen Inc., 1201 Amgen Court West, Seattle, Washington 98119, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Discovery
Research, Amgen Inc., One Amgen Center Drive, Thousand Oaks, Callifornia 91320, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Steve Thibault</span> - <span class="hlFld-Affiliation affiliation">Departments of †Therapeutic
Discovery, ‡Oncology Research, §Pharmaceutics, and ∥Pharmacokinetics and Drug Metabolism, Amgen
Inc., 1120 Veterans Boulevard, South San Francisco, California 94080, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Therapeutic
Innovation Unit, Amgen Inc., 1201 Amgen Court West, Seattle, Washington 98119, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Discovery
Research, Amgen Inc., One Amgen Center Drive, Thousand Oaks, Callifornia 91320, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nigel Walker</span> - <span class="hlFld-Affiliation affiliation">Departments of †Therapeutic
Discovery, ‡Oncology Research, §Pharmaceutics, and ∥Pharmacokinetics and Drug Metabolism, Amgen
Inc., 1120 Veterans Boulevard, South San Francisco, California 94080, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Therapeutic
Innovation Unit, Amgen Inc., 1201 Amgen Court West, Seattle, Washington 98119, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Discovery
Research, Amgen Inc., One Amgen Center Drive, Thousand Oaks, Callifornia 91320, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xiaodong Wang</span> - <span class="hlFld-Affiliation affiliation">Departments of †Therapeutic
Discovery, ‡Oncology Research, §Pharmaceutics, and ∥Pharmacokinetics and Drug Metabolism, Amgen
Inc., 1120 Veterans Boulevard, South San Francisco, California 94080, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Therapeutic
Innovation Unit, Amgen Inc., 1201 Amgen Court West, Seattle, Washington 98119, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Discovery
Research, Amgen Inc., One Amgen Center Drive, Thousand Oaks, Callifornia 91320, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Zhulun Wang</span> - <span class="hlFld-Affiliation affiliation">Departments of †Therapeutic
Discovery, ‡Oncology Research, §Pharmaceutics, and ∥Pharmacokinetics and Drug Metabolism, Amgen
Inc., 1120 Veterans Boulevard, South San Francisco, California 94080, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Therapeutic
Innovation Unit, Amgen Inc., 1201 Amgen Court West, Seattle, Washington 98119, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Discovery
Research, Amgen Inc., One Amgen Center Drive, Thousand Oaks, Callifornia 91320, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Malgorzata Wanska</span> - <span class="hlFld-Affiliation affiliation">Departments of †Therapeutic
Discovery, ‡Oncology Research, §Pharmaceutics, and ∥Pharmacokinetics and Drug Metabolism, Amgen
Inc., 1120 Veterans Boulevard, South San Francisco, California 94080, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Therapeutic
Innovation Unit, Amgen Inc., 1201 Amgen Court West, Seattle, Washington 98119, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Discovery
Research, Amgen Inc., One Amgen Center Drive, Thousand Oaks, Callifornia 91320, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Paul M. Wehn</span> - <span class="hlFld-Affiliation affiliation">Departments of †Therapeutic
Discovery, ‡Oncology Research, §Pharmaceutics, and ∥Pharmacokinetics and Drug Metabolism, Amgen
Inc., 1120 Veterans Boulevard, South San Francisco, California 94080, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Therapeutic
Innovation Unit, Amgen Inc., 1201 Amgen Court West, Seattle, Washington 98119, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Discovery
Research, Amgen Inc., One Amgen Center Drive, Thousand Oaks, Callifornia 91320, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Margaret F. Weidner</span> - <span class="hlFld-Affiliation affiliation">Therapeutic
Innovation Unit, Amgen Inc., 1201 Amgen Court West, Seattle, Washington 98119, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Alex J. Zhang</span> - <span class="hlFld-Affiliation affiliation">Departments of †Therapeutic
Discovery, ‡Oncology Research, §Pharmaceutics, and ∥Pharmacokinetics and Drug Metabolism, Amgen
Inc., 1120 Veterans Boulevard, South San Francisco, California 94080, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Therapeutic
Innovation Unit, Amgen Inc., 1201 Amgen Court West, Seattle, Washington 98119, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Discovery
Research, Amgen Inc., One Amgen Center Drive, Thousand Oaks, Callifornia 91320, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xiaoning Zhao</span> - <span class="hlFld-Affiliation affiliation">Departments of †Therapeutic
Discovery, ‡Oncology Research, §Pharmaceutics, and ∥Pharmacokinetics and Drug Metabolism, Amgen
Inc., 1120 Veterans Boulevard, South San Francisco, California 94080, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Therapeutic
Innovation Unit, Amgen Inc., 1201 Amgen Court West, Seattle, Washington 98119, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Discovery
Research, Amgen Inc., One Amgen Center Drive, Thousand Oaks, Callifornia 91320, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Alexander Kamb</span> - <span class="hlFld-Affiliation affiliation">Discovery
Research, Amgen Inc., One Amgen Center Drive, Thousand Oaks, Callifornia 91320, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dineli Wickramasinghe</span> - <span class="hlFld-Affiliation affiliation">Departments of †Therapeutic
Discovery, ‡Oncology Research, §Pharmaceutics, and ∥Pharmacokinetics and Drug Metabolism, Amgen
Inc., 1120 Veterans Boulevard, South San Francisco, California 94080, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Therapeutic
Innovation Unit, Amgen Inc., 1201 Amgen Court West, Seattle, Washington 98119, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Discovery
Research, Amgen Inc., One Amgen Center Drive, Thousand Oaks, Callifornia 91320, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kang Dai</span> - <span class="hlFld-Affiliation affiliation">Departments of †Therapeutic
Discovery, ‡Oncology Research, §Pharmaceutics, and ∥Pharmacokinetics and Drug Metabolism, Amgen
Inc., 1120 Veterans Boulevard, South San Francisco, California 94080, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Therapeutic
Innovation Unit, Amgen Inc., 1201 Amgen Court West, Seattle, Washington 98119, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Discovery
Research, Amgen Inc., One Amgen Center Drive, Thousand Oaks, Callifornia 91320, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lawrence R. McGee</span> - <span class="hlFld-Affiliation affiliation">Departments of †Therapeutic
Discovery, ‡Oncology Research, §Pharmaceutics, and ∥Pharmacokinetics and Drug Metabolism, Amgen
Inc., 1120 Veterans Boulevard, South San Francisco, California 94080, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Therapeutic
Innovation Unit, Amgen Inc., 1201 Amgen Court West, Seattle, Washington 98119, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Discovery
Research, Amgen Inc., One Amgen Center Drive, Thousand Oaks, Callifornia 91320, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Julio C. Medina</span> - <span class="hlFld-Affiliation affiliation">Departments of †Therapeutic
Discovery, ‡Oncology Research, §Pharmaceutics, and ∥Pharmacokinetics and Drug Metabolism, Amgen
Inc., 1120 Veterans Boulevard, South San Francisco, California 94080, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Therapeutic
Innovation Unit, Amgen Inc., 1201 Amgen Court West, Seattle, Washington 98119, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Discovery
Research, Amgen Inc., One Amgen Center Drive, Thousand Oaks, Callifornia 91320, United States</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><p>The authors declare no competing financial interest.<br /></br></p></li></ul></div><div class="ack" id="ACK-d1189e6971-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i101">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_36383" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_36383" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank David Chaw for assistance of NMR spectra, Guifen Xu for bioanalytical support, Simon Wong for determination of CYP IC<sub>50</sub> values, and Dhanashri Bagal and Iain Campuzano for determination of high-resolution mass spectra. We also thank Xiaolin Hao for useful discussion.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i102" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i102"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i103" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i103"> Abbreviations Used</h2><tr><td class="NLM_term">AML</td><td class="NLM_def"><p class="first last">acute myeloid leukemia</p></td></tr><tr><td class="NLM_term">ATP</td><td class="NLM_def"><p class="first last">adensoine triphosphate</p></td></tr><tr><td class="NLM_term">AUC</td><td class="NLM_def"><p class="first last">area under curve</p></td></tr><tr><td class="NLM_term">CDK4</td><td class="NLM_def"><p class="first last">cyclin-dependent kinase 4</p></td></tr><tr><td class="NLM_term">CL</td><td class="NLM_def"><p class="first last">clearance</p></td></tr><tr><td class="NLM_term"><i>C</i><sub>max</sub></td><td class="NLM_def"><p class="first last">maxium concentration</p></td></tr><tr><td class="NLM_term">CYP3A4</td><td class="NLM_def"><p class="first last">cytochrome p450 3A4</p></td></tr><tr><td class="NLM_term">DCM</td><td class="NLM_def"><p class="first last">dichloromethane</p></td></tr><tr><td class="NLM_term">DIPEA</td><td class="NLM_def"><p class="first last"><i>N</i>,<i>N</i>-diisopropylethylamine</p></td></tr><tr><td class="NLM_term">DMAP</td><td class="NLM_def"><p class="first last">dimethylaminopyridine</p></td></tr><tr><td class="NLM_term">DMF</td><td class="NLM_def"><p class="first last">dimethylformamide</p></td></tr><tr><td class="NLM_term">dppf</td><td class="NLM_def"><p class="first last">ferrocene,1,1′-bis(diphenylphosphino)–</p></td></tr><tr><td class="NLM_term">ED</td><td class="NLM_def"><p class="first last">effective dose</p></td></tr><tr><td class="NLM_term">EDTA</td><td class="NLM_def"><p class="first last">ethylenediamine-<i>N</i>,<i>N</i>,<i>N</i>′,<i>N</i>′- tetraacetic acid</p></td></tr><tr><td class="NLM_term">EGTA</td><td class="NLM_def"><p class="first last">ethylene glycol bis(2-aminoethyl ether)-<i>N</i>,<i>N</i>,<i>N</i>′,<i>N</i>′-tetraacetic acid</p></td></tr><tr><td class="NLM_term"><i>F</i></td><td class="NLM_def"><p class="first last">bioavailability</p></td></tr><tr><td class="NLM_term">Fe(acac)<sub>3</sub></td><td class="NLM_def"><p class="first last">tris(acetylacetonato) iron(III)</p></td></tr><tr><td class="NLM_term">FLT3</td><td class="NLM_def"><p class="first last">FMS-like tyrosine kinase 3</p></td></tr><tr><td class="NLM_term">hERG</td><td class="NLM_def"><p class="first last">human ether-à-go-go-related gene</p></td></tr><tr><td class="NLM_term">IC<sub>50</sub></td><td class="NLM_def"><p class="first last">half-maximal inhibitory concentration</p></td></tr><tr><td class="NLM_term">KHMDS</td><td class="NLM_def"><p class="first last">potassium bis(trimethylsilyl)amide</p></td></tr><tr><td class="NLM_term">LiHMDS</td><td class="NLM_def"><p class="first last">lithium hexamethyldisilazide</p></td></tr><tr><td class="NLM_term">LiTMP</td><td class="NLM_def"><p class="first last">1-lithio-2,2,6,6-tetramethylpiperidine</p></td></tr><tr><td class="NLM_term">NIS</td><td class="NLM_def"><p class="first last"><i>n</i>-iodosuccinimide</p></td></tr><tr><td class="NLM_term">NMP</td><td class="NLM_def"><p class="first last"><i>N</i>-methyl-2-pyrrolidone</p></td></tr><tr><td class="NLM_term">PBS</td><td class="NLM_def"><p class="first last">phosphate buffered saline</p></td></tr><tr><td class="NLM_term">PI3K</td><td class="NLM_def"><p class="first last">phosphatidylinositide 3 kinase</p></td></tr><tr><td class="NLM_term">PK</td><td class="NLM_def"><p class="first last">pharmacokinetic</p></td></tr><tr><td class="NLM_term">PPTS</td><td class="NLM_def"><p class="first last">pyridinium <i>p</i>-toluenesulfonate</p></td></tr><tr><td class="NLM_term">Rb</td><td class="NLM_def"><p class="first last">retinoblastoma protein</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr><tr><td class="NLM_term">TEA</td><td class="NLM_def"><p class="first last">triethylamine</p></td></tr><tr><td class="NLM_term">TFA</td><td class="NLM_def"><p class="first last">trifluoroacetic acid</p></td></tr><tr><td class="NLM_term">THF</td><td class="NLM_def"><p class="first last">tetrahydofuran</p></td></tr><tr><td class="NLM_term">TLC</td><td class="NLM_def"><p class="first last">thin layer chromatography</p></td></tr><tr><td class="NLM_term"><i>V</i><sub>ss</sub></td><td class="NLM_def"><p class="first last">volume of distribution</p></td></tr><tr><td class="NLM_term">XantPhos</td><td class="NLM_def"><p class="first last">4,5-bis(diphenylphosphino)-9,9-dimethylxanthene</p></td></tr><tr><td class="NLM_term">X-Phos</td><td class="NLM_def"><p class="first last">2-dicyclohexylphosphophino-2′4′6′-triisopropylbiphenyl</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i104">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_60182" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_60182" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 33 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span>Cancer Facts & Figures 2013; <span class="NLM_publisher-name">American Cancer Society</span>: <span class="NLM_publisher-loc">Atlanta</span>,<span class="NLM_x"> </span><span class="NLM_year">2013</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Cancer+Facts+%26+Figures+2013%3B+American+Cancer+Society%3A+Atlanta%2C+2013."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dreport%26sid%3Dliteratum%253Aachs%26jtitle%3DCancer%2520Facts%2520%2526%2520Figures%25202013%26pub%3DAmerican%2520Cancer%2520Society%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Birg, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Courcoul, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosnet, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bardin, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pebusque, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marchetto, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tabilio, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mannoni, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Birnbaum, D.</span><span> </span><span class="NLM_article-title">Expression of the FMS/KIT-like gene FLT3 in human acute leukemia of the myeloid and lymphoid lineages</span> <span class="citation_source-journal">Blood</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1992</span><span class="NLM_x">, </span> <span class="NLM_volume">80</span><span class="NLM_x">, </span> <span class="NLM_fpage">2584</span><span class="NLM_x">–</span> <span class="NLM_lpage">2593</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=80&publication_year=1992&pages=2584-2593&author=F.+Birgauthor=M.+Courcoulauthor=O.+Rosnetauthor=F.+Bardinauthor=M.+J.+Pebusqueauthor=S.+Marchettoauthor=A.+Tabilioauthor=P.+Mannoniauthor=D.+Birnbaum&title=Expression+of+the+FMS%2FKIT-like+gene+FLT3+in+human+acute+leukemia+of+the+myeloid+and+lymphoid+lineages"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBirg%26aufirst%3DF.%26aulast%3DCourcoul%26aufirst%3DM.%26aulast%3DRosnet%26aufirst%3DO.%26aulast%3DBardin%26aufirst%3DF.%26aulast%3DPebusque%26aufirst%3DM.%2BJ.%26aulast%3DMarchetto%26aufirst%3DS.%26aulast%3DTabilio%26aufirst%3DA.%26aulast%3DMannoni%26aufirst%3DP.%26aulast%3DBirnbaum%26aufirst%3DD.%26atitle%3DExpression%2520of%2520the%2520FMS%252FKIT-like%2520gene%2520FLT3%2520in%2520human%2520acute%2520leukemia%2520of%2520the%2520myeloid%2520and%2520lymphoid%2520lineages%26jtitle%3DBlood%26date%3D1992%26volume%3D80%26spage%3D2584%26epage%3D2593" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Carow, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levenstein, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaufmann, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Amin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rockwell, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Witte, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Borowitz, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Civin, C. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Small, D.</span><span> </span><span class="NLM_article-title">Expression of the hematopoietic growth factor receptor FLT3 (STK-1/Flk2) in human leukemia</span> <span class="citation_source-journal">Blood</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">87</span><span class="NLM_x">, </span> <span class="NLM_fpage">1089</span><span class="NLM_x">–</span> <span class="NLM_lpage">1096</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=87&publication_year=1996&pages=1089-1096&author=C.+E.+Carowauthor=M.+Levensteinauthor=S.+H.+Kaufmannauthor=J.+Chenauthor=S.+Aminauthor=P.+Rockwellauthor=L.+Witteauthor=M.+J.+Borowitzauthor=C.+I.+Civinauthor=D.+Small&title=Expression+of+the+hematopoietic+growth+factor+receptor+FLT3+%28STK-1%2FFlk2%29+in+human+leukemia"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCarow%26aufirst%3DC.%2BE.%26aulast%3DLevenstein%26aufirst%3DM.%26aulast%3DKaufmann%26aufirst%3DS.%2BH.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DAmin%26aufirst%3DS.%26aulast%3DRockwell%26aufirst%3DP.%26aulast%3DWitte%26aufirst%3DL.%26aulast%3DBorowitz%26aufirst%3DM.%2BJ.%26aulast%3DCivin%26aufirst%3DC.%2BI.%26aulast%3DSmall%26aufirst%3DD.%26atitle%3DExpression%2520of%2520the%2520hematopoietic%2520growth%2520factor%2520receptor%2520FLT3%2520%2528STK-1%252FFlk2%2529%2520in%2520human%2520leukemia%26jtitle%3DBlood%26date%3D1996%26volume%3D87%26spage%3D1089%26epage%3D1096" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Drexler, H. G.</span><span> </span><span class="NLM_article-title">Expression of FLT3 receptor and response to FLT3 ligand by leukemic cells</span> <span class="citation_source-journal">Leukemia</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">588</span><span class="NLM_x">–</span> <span class="NLM_lpage">599</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Fjm500118j&amp;key=8618433" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Fjm500118j&amp;key=1%3ACAS%3A280%3ADyaK283gtV2qsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=1996&pages=588-599&author=H.+G.+Drexler&title=Expression+of+FLT3+receptor+and+response+to+FLT3+ligand+by+leukemic+cells"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Expression of FLT3 receptor and response to FLT3 ligand by leukemic cells</span></div><div class="casAuthors">Drexler H G</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">588-99</span>
        ISSN:<span class="NLM_cas:issn">0887-6924</span>.
    </div><div class="casAbstract">The novel hematopoietic growth factor FLT3 ligand (FL) is the cognate ligand for the FLT3, tyrosine kinase receptor (R), also referred to as FLK-2 and STK-1.  The FLT3R belongs to a family of receptor tyrosine kinases involved in hematopoiesis that also includes KIT, the receptor for SCF (stem cell factor), and FMS. the receptor for M-CSF (macrophage colony- stimulating factor).  Restricted FLT3R expression was seen on human and murine hematopoietic progenitor cells.  In functional assays recombinant FL stimulated the proliferation and colony formation of human hematopoietic progenitor cells, i.e.  CD34+ cord and peripheral blood, bone marrow and fetal liver cells.  Synergy was reported for co-stimulation with G-CSF (granulocyte-CSF).  GM-CSF (granulocyte-macrophage CSF), M-CSF, interleukin-3 (IL-3), PIXY-321 (an IL-3/GM-CSF fusion protein) and SCF.  In the mouse, FL potently enhanced growth of various types of progenitor/precursor cells in synergy with G-CSF, GM-CSF, M-CSF, IL-3, IL-6, IL-7, IL-11, IL-12 and SCF.  The well-documented involvement of this ligand-receptor pair in physiological hematopoiesis brought forth the question whether FLT3R and FL might also have a role in the pathobiology of leukemia.  At the mRNA level FLT3R was expressed by most (80-100%) cases of AML (acute myeloid leukemia) throughout the different morphological subtypes (MO-M7), of ALL(acute lymphoblastic leukemia) of the immunological subtypes T-ALL and BCP-ALL (B cell precursor ALL including pre-pre B-ALL, cALL and pre B-ALL), of AMLL (acute mixed-lineage leukemia), and of CML (chronic myeloid leukemia) in lymphoid or mixed blast crisis.  Analysis of cell surface expression of FLT3R by flow cytometry confirmed these observations for AML (66% positivity when the data from all studies are combined), BCP-ALL (64%) and CML lymphoid blast crisis (86%) whereas less than 30% of T-ALL were FLT3R+.  The myeloid, monocytic and pre B cell type categories also contained the highest proportions of FLT3R+ leukemia cell lines .  In contrast to the selective expression of the receptor, FL expression was detected in 90-100% of the various cell types of leukemia cell lines from all hematopoietic cell lineages.  The potential of FL to induce proliferation of leukemia cells in vitro was also examined in primary and continuously cultured leukemia cells.  The data on FL-stimulated leukemia cell growth underline the extensive heterogeneity of primary AML and ALL samples in terms of cytokine-inducible DNA synthesis that has been seen with other effective cytokines.  While the majority of T-ALL (0-33% of the cases responded proliferatively; mean 11%) and BCP-ALL (0-30%; mean 20%) failed to proliferate in the presence of FL despite strong expression of surface FLT3R, FL caused a proliferative response in a significantly higher percentage of AML cases (22-90%; mean 53%).  In the panel of leukemia cell lines examined only myeloid and monocytic growth factor- dependent cell lines increased their proliferation upon incubation with FL, whereas all growth factor-independent cell lines were refractory to stimulation.  Combinations of FL with G-CSF, GM-CSF, M-CSF, IL-3, PIXY- 321 or SCF and FL with IL-3 or IL-7 had synergistic or additive mitogenic effects on primary AML and ALL cells, respectively.  The potent stimulation of the myelomonocytic cell lines was further augmented by addition of bFGF (basic fibroblast growth factor), GM-CSF, IL-3 or SCF.  The inhibitory effects of TGF-beta 1 (transforming growth factor-beta 1) on FL- supported proliferation were abrogated by bFGF.  Taken together, these results demonstrate the expression of functional FLT3R capable of mediating FL- dependent mitogenic signaling in a subset of AML and ALL cases further underline the heterogeneity of AML and ALL samples in their proliferative response to cytokine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT_MDUZMlgnZlm8bhe6roZYfW6udTcc2eYivdLuIB04yrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK283gtV2qsA%253D%253D&md5=d9d427a65fa9b8b1b279c442d353f18f</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DDrexler%26aufirst%3DH.%2BG.%26atitle%3DExpression%2520of%2520FLT3%2520receptor%2520and%2520response%2520to%2520FLT3%2520ligand%2520by%2520leukemic%2520cells%26jtitle%3DLeukemia%26date%3D1996%26volume%3D10%26spage%3D588%26epage%3D599" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Thiede, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steudel, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mohr, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schaich, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schakel, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Platzbecker, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wermke, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bornhauser, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ritter, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neubauer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ehninger, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Illmer, T.</span><span> </span><span class="NLM_article-title">Analysis of FLT3-activating mutations in 979 patients with acute myelogenous leukemia: association with FAB subtypes and identification of subgroups with poor prognosis</span> <span class="citation_source-journal">Blood</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">99</span><span class="NLM_x">, </span> <span class="NLM_fpage">4326</span><span class="NLM_x">–</span> <span class="NLM_lpage">4335</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=2002&pages=4326-4335&author=C.+Thiedeauthor=C.+Steudelauthor=B.+Mohrauthor=M.+Schaichauthor=U.+Schakelauthor=U.+Platzbeckerauthor=M.+Wermkeauthor=M.+Bornhauserauthor=M.+Ritterauthor=A.+Neubauerauthor=G.+Ehningerauthor=T.+Illmer&title=Analysis+of+FLT3-activating+mutations+in+979+patients+with+acute+myelogenous+leukemia%3A+association+with+FAB+subtypes+and+identification+of+subgroups+with+poor+prognosis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DThiede%26aufirst%3DC.%26aulast%3DSteudel%26aufirst%3DC.%26aulast%3DMohr%26aufirst%3DB.%26aulast%3DSchaich%26aufirst%3DM.%26aulast%3DSchakel%26aufirst%3DU.%26aulast%3DPlatzbecker%26aufirst%3DU.%26aulast%3DWermke%26aufirst%3DM.%26aulast%3DBornhauser%26aufirst%3DM.%26aulast%3DRitter%26aufirst%3DM.%26aulast%3DNeubauer%26aufirst%3DA.%26aulast%3DEhninger%26aufirst%3DG.%26aulast%3DIllmer%26aufirst%3DT.%26atitle%3DAnalysis%2520of%2520FLT3-activating%2520mutations%2520in%2520979%2520patients%2520with%2520acute%2520myelogenous%2520leukemia%253A%2520association%2520with%2520FAB%2520subtypes%2520and%2520identification%2520of%2520subgroups%2520with%2520poor%2520prognosis%26jtitle%3DBlood%26date%3D2002%26volume%3D99%26spage%3D4326%26epage%3D4335" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Pratz, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levis, M. J.</span><span> </span><span class="NLM_article-title">Bench to bedside targeting of FLT3 in acute leukemia</span> <span class="citation_source-journal">Curr. Drug Targets</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">781</span><span class="NLM_x">–</span> <span class="NLM_lpage">789</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Fjm500118j&amp;key=10.2174%2F138945010791320782" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Fjm500118j&amp;key=20370649" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Fjm500118j&amp;key=1%3ACAS%3A528%3ADC%252BC3cXpsFeisLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2010&pages=781-789&author=K.+W.+Pratzauthor=M.+J.+Levis&title=Bench+to+bedside+targeting+of+FLT3+in+acute+leukemia"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Bench to bedside targeting of FLT3 in acute leukemia</span></div><div class="casAuthors">Pratz, Keith W.; Levis, Mark J.</div><div class="citationInfo"><span class="NLM_cas:title">Current Drug Targets</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">781-789</span>CODEN:
                <span class="NLM_cas:coden">CDTUAU</span>;
        ISSN:<span class="NLM_cas:issn">1389-4501</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  FMS-Like-Tyrosine kinase-3 (FLT3) mutations are found in about 30% of cases of acute myeloid leukemia and confer an increased relapse rate and reduced overall survival.  Targeting this tyrosine kinase by direction inhibition is the focus of both preclin. and clin. research in AML.  Several mols. in clin. development inhibit FLT3, but thus far clin. responses have been limited.  Correlative studies from monotherapy trials have established that responses require sustained, effective FLT3 inhibition in vivo.  Studies combining FLT3 inhibitors with chemotherapy have demonstrated increased remission rates to date but have yet to produce a survival advantage.  Currently the only approved FLT3 inhibitor available for off-label use is sorafenib, which clearly has clin. activity but does not commonly lead to a complete response.  Several FLT3 inhibitors are currently being tested as single agents and in combination with chemotherapy, and it seems likely that a clin. useful drug will eventually emerge.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTnebgO9Va-rVg90H21EOLACvtfcHk0lj1vmXHTj_ftg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXpsFeisLw%253D&md5=dde8a09c2b96f400ad4646f046a45744</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.2174%2F138945010791320782&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138945010791320782%26sid%3Dliteratum%253Aachs%26aulast%3DPratz%26aufirst%3DK.%2BW.%26aulast%3DLevis%26aufirst%3DM.%2BJ.%26atitle%3DBench%2520to%2520bedside%2520targeting%2520of%2520FLT3%2520in%2520acute%2520leukemia%26jtitle%3DCurr.%2520Drug%2520Targets%26date%3D2010%26volume%3D11%26spage%3D781%26epage%3D789" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Kindler, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lipka, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fischer, T.</span><span> </span><span class="NLM_article-title">FLT3 as a therapeutic target in AML; still challenging after all these years</span> <span class="citation_source-journal">Blood</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">5089</span><span class="NLM_x">–</span> <span class="NLM_lpage">5102</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2010&pages=5089-5102&author=T.+Kindlerauthor=D.+B.+Lipkaauthor=T.+Fischer&title=FLT3+as+a+therapeutic+target+in+AML%3B+still+challenging+after+all+these+years"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKindler%26aufirst%3DT.%26aulast%3DLipka%26aufirst%3DD.%2BB.%26aulast%3DFischer%26aufirst%3DT.%26atitle%3DFLT3%2520as%2520a%2520therapeutic%2520target%2520in%2520AML%253B%2520still%2520challenging%2520after%2520all%2520these%2520years%26jtitle%3DBlood%26date%3D2010%26volume%3D9%26spage%3D5089%26epage%3D5102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Smith, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Salerno, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Damon, L. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levis, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perl, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Travers, K. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunt, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zarrinkar, P. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schadt, E. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kasarskis, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuriyan, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shah, N. P.</span><span> </span><span class="NLM_article-title">Validation of ITD mutations in FLT3 as a therapeutic target in human acute myeloid leukaemia</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">485</span><span class="NLM_x">, </span> <span class="NLM_fpage">260</span><span class="NLM_x">–</span> <span class="NLM_lpage">263</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Fjm500118j&amp;key=10.1038%2Fnature11016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Fjm500118j&amp;key=22504184" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Fjm500118j&amp;key=1%3ACAS%3A528%3ADC%252BC38XmtVeqtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=485&publication_year=2012&pages=260-263&author=C.+C.+Smithauthor=Q.+Wangauthor=C.+Chinauthor=S.+Salernoauthor=L.+E.+Damonauthor=M.+J.+Levisauthor=A.+E.+Perlauthor=K.+J.+Traversauthor=S.+Wangauthor=J.+P.+Huntauthor=P.+P.+Zarrinkarauthor=E.+E.+Schadtauthor=A.+Kasarskisauthor=J.+Kuriyanauthor=N.+P.+Shah&title=Validation+of+ITD+mutations+in+FLT3+as+a+therapeutic+target+in+human+acute+myeloid+leukaemia"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Validation of ITD mutations in FLT3 as a therapeutic target in human acute myeloid leukemia</span></div><div class="casAuthors">Smith, Catherine C.; Wang, Qi; Chin, Chen-Shan; Salerno, Sara; Damon, Lauren E.; Levis, Mark J.; Perl, Alexander E.; Travers, Kevin J.; Wang, Susana; Hunt, Jeremy P.; Zarrinkar, Patrick P.; Schadt, Eric E.; Kasarskis, Andrew; Kuriyan, John; Shah, Neil P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">485</span>
        (<span class="NLM_cas:issue">7397</span>),
    <span class="NLM_cas:pages">260-263</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Effective targeted cancer therapeutic development depends upon distinguishing disease-assocd. driver' mutations, which have causative roles in malignancy pathogenesis, from passenger' mutations, which are dispensable for cancer initiation and maintenance.  Translational studies of clin. active targeted therapeutics can definitively discriminate driver from passenger lesions and provide valuable insights into human cancer biol.  Activating internal tandem duplication (ITD) mutations in FLT3 (FLT3-ITD) are detected in approx. 20% of acute myeloid leukemia (AML) patients and are assocd. with a poor prognosis.  Abundant scientific and clin. evidence, including the lack of convincing clin. activity of early FLT3 inhibitors, suggests that FLT3-ITD probably represents a passenger lesion.  Here we report point mutations at three residues within the kinase domain of FLT3-ITD that confer substantial in vitro resistance to AC220 (quizartinib), an active investigational inhibitor of FLT3, KIT, PDGFRA, PDGFRB and RET; evolution of AC220-resistant substitutions at two of these amino acid positions was obsd. in eight of eight FLT3-ITD-pos. AML patients with acquired resistance to AC220.  Our findings demonstrate that FLT3-ITD can represent a driver lesion and valid therapeutic target in human AML.  AC220-resistant FLT3 kinase domain mutants represent high-value targets for future FLT3 inhibitor development efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqq0PeqhlWLObVg90H21EOLACvtfcHk0lj1vmXHTj_ftg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmtVeqtr8%253D&md5=228dae85b08e65918f74ee8c5d20ddb0</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnature11016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature11016%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DC.%2BC.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DChin%26aufirst%3DC.%26aulast%3DSalerno%26aufirst%3DS.%26aulast%3DDamon%26aufirst%3DL.%2BE.%26aulast%3DLevis%26aufirst%3DM.%2BJ.%26aulast%3DPerl%26aufirst%3DA.%2BE.%26aulast%3DTravers%26aufirst%3DK.%2BJ.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DHunt%26aufirst%3DJ.%2BP.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26aulast%3DSchadt%26aufirst%3DE.%2BE.%26aulast%3DKasarskis%26aufirst%3DA.%26aulast%3DKuriyan%26aufirst%3DJ.%26aulast%3DShah%26aufirst%3DN.%2BP.%26atitle%3DValidation%2520of%2520ITD%2520mutations%2520in%2520FLT3%2520as%2520a%2520therapeutic%2520target%2520in%2520human%2520acute%2520myeloid%2520leukaemia%26jtitle%3DNature%26date%3D2012%26volume%3D485%26spage%3D260%26epage%3D263" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Knapper, S.</span><span> </span><span class="NLM_article-title">FLT3 inhibition in acute myeloid leukaemia</span> <span class="citation_source-journal">Br. J. Haematol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">138</span><span class="NLM_x">, </span> <span class="NLM_fpage">687</span><span class="NLM_x">–</span> <span class="NLM_lpage">699</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Fjm500118j&amp;key=10.1111%2Fj.1365-2141.2007.06700.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Fjm500118j&amp;key=17655729" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Fjm500118j&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFCksrfL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=138&publication_year=2007&pages=687-699&author=S.+Knapper&title=FLT3+inhibition+in+acute+myeloid+leukaemia"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">FLT3 inhibition in acute myeloid leukaemia</span></div><div class="casAuthors">Knapper, Steven</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Haematology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">138</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">687-699</span>CODEN:
                <span class="NLM_cas:coden">BJHEAL</span>;
        ISSN:<span class="NLM_cas:issn">0007-1048</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">A review.  FMS-like tyrosine kinase 3 (FLT3) is a receptor tyrosine kinase that appears to play a significant role in leukemogenesis.  Activating mutations of FLT3 are present in approx. one-third of acute myeloid leukemia patients and are assocd. with adverse clin. outcome, while many nonmutated cases also show evidence of FLT3 activation.  FLT3 thus represents a potentially exciting mol. therapeutic target.  A no. of small-mol. tyrosine kinase inhibitors with anti-FLT3 activity have been developed and several of these compds. have entered early phase clin. trials where clin. anti-leukemic activity has been demonstrated.  The depth and duration of clin. responses to FLT3 inhibitor monotherapy have been modest, however, and a no. of mechanisms by which blasts may acquire resistance have been proposed.  Based on preclin. evidence of synergy with conventional chemotherapy, several combination trials are now underway.  FLT3 inhibition may also be effective used in combination with other molecularly targeted agents, in postchemotherapy stem-cell-directed maintenance therapy and in MLL-rearranged infant acute lymphoblastic leukemia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq35LdA5Gz4drVg90H21EOLACvtfcHk0ljrF9tL6g0Srw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFCksrfL&md5=f10153f7e194e393e133e402a90c739f</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1111%2Fj.1365-2141.2007.06700.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1365-2141.2007.06700.x%26sid%3Dliteratum%253Aachs%26aulast%3DKnapper%26aufirst%3DS.%26atitle%3DFLT3%2520inhibition%2520in%2520acute%2520myeloid%2520leukaemia%26jtitle%3DBr.%2520J.%2520Haematol.%26date%3D2007%26volume%3D138%26spage%3D687%26epage%3D699" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blaser, B. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duchemin, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kusewitt, D. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caligiuri, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Briesewitz, R.</span><span> </span><span class="NLM_article-title">Pharmacologic inhibition of CDK4/6: mechanistic evidence for selective activity or acquired resistance in acute myeloid leukemia</span> <span class="citation_source-journal">Blood</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">110</span><span class="NLM_x">, </span> <span class="NLM_fpage">2075</span><span class="NLM_x">–</span> <span class="NLM_lpage">2083</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Fjm500118j&amp;key=10.1182%2Fblood-2007-02-071266" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Fjm500118j&amp;key=17537993" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Fjm500118j&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtVGjtbbK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2007&pages=2075-2083&author=L.+Wangauthor=J.+Wangauthor=B.+W.+Blaserauthor=A.+M.+Ducheminauthor=D.+F.+Kusewittauthor=T.+Liuauthor=M.+A.+Caligiuriauthor=R.+Briesewitz&title=Pharmacologic+inhibition+of+CDK4%2F6%3A+mechanistic+evidence+for+selective+activity+or+acquired+resistance+in+acute+myeloid+leukemia"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacologic inhibition of CDK4/6: mechanistic evidence for selective activity or acquired resistance in acute myeloid leukemia</span></div><div class="casAuthors">Wang, Lisheng; Wang, Jie; Blaser, Bradley W.; Duchemin, Anne-Marie; Kusewitt, Donna F.; Liu, Tom; Caligiuri, Michael A.; Briesewitz, Roger</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2075-2083</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">Entry into the cell cycle is mediated by cyclin-dependent kinase 4/6 (CDK4/6) activation, followed by CDK2 activation.  We found that pharmacol. inhibition of the Flt3 internal tandem duplication (ITD), a mutated receptor tyrosine kinase commonly found in patients with acute myelogenous leukemia (AML), led to the down-regulation of cyclin D2 and D3 followed by retinoblastoma protein (pRb) dephosphorylation and G1 cell-cycle arrest.  This implicated the D-cyclin-CDK4/6 complex as a downstream effector of Flt3 ITD signaling.  Indeed, single-agent PD0332991, a selective CDK4/6 inhibitor, caused sustained cell-cycle arrest in Flt3 ITD AML cell lines and prolonged survival in an in vivo model of Flt3 ITD AML.  PD0332991 caused an initial cell-cycle arrest in well-established Flt3 wild-type (wt) AML cell lines, but this was overcome by down-regulation of p27Kip and reactivation of CDK2.  This acquired resistance was not obsd. in a Flt3 ITD and a Flt3 wt sample from a patient with primary AML.  In summary, the mechanism of cell-cycle arrest after treatment of Flt3 ITD AML with a Flt3 inhibitor involves down-regulation of cyclin D2 and D3.  As such, CDK4/6 can be a therapeutic target in Flt3 ITD AML but also in primary Flt3 wt AML.  Finally, acquired resistance to CDK4/6 inhibition can arise through activation CDK2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoiH_wOIP8jJLVg90H21EOLACvtfcHk0ljrF9tL6g0Srw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtVGjtbbK&md5=5f68ef6deacf3789e428731722f8c5ff</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1182%2Fblood-2007-02-071266&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2007-02-071266%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DBlaser%26aufirst%3DB.%2BW.%26aulast%3DDuchemin%26aufirst%3DA.%2BM.%26aulast%3DKusewitt%26aufirst%3DD.%2BF.%26aulast%3DLiu%26aufirst%3DT.%26aulast%3DCaligiuri%26aufirst%3DM.%2BA.%26aulast%3DBriesewitz%26aufirst%3DR.%26atitle%3DPharmacologic%2520inhibition%2520of%2520CDK4%252F6%253A%2520mechanistic%2520evidence%2520for%2520selective%2520activity%2520or%2520acquired%2520resistance%2520in%2520acute%2520myeloid%2520leukemia%26jtitle%3DBlood%26date%3D2007%26volume%3D110%26spage%3D2075%26epage%3D2083" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Drexler, H. G.</span><span> </span><span class="NLM_article-title">Review of alterations of the cyclin-dependent kinase inhibitor INK4 family genes p15, p16, p18 and p19 in human leukemia-lymphoma cells</span> <span class="citation_source-journal">Leukemia</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">845</span><span class="NLM_x">–</span> <span class="NLM_lpage">859</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=1998&pages=845-859&author=H.+G.+Drexler&title=Review+of+alterations+of+the+cyclin-dependent+kinase+inhibitor+INK4+family+genes+p15%2C+p16%2C+p18+and+p19+in+human+leukemia-lymphoma+cells"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DDrexler%26aufirst%3DH.%2BG.%26atitle%3DReview%2520of%2520alterations%2520of%2520the%2520cyclin-dependent%2520kinase%2520inhibitor%2520INK4%2520family%2520genes%2520p15%252C%2520p16%252C%2520p18%2520and%2520p19%2520in%2520human%2520leukemia-lymphoma%2520cells%26jtitle%3DLeukemia%26date%3D1998%26volume%3D12%26spage%3D845%26epage%3D859" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Malumbres, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barbacid, M.</span><span> </span><span class="NLM_article-title">Cell cycle, CDKs and cancer: a changing paradigm</span> <span class="citation_source-journal">Nature Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">253</span><span class="NLM_x">–</span> <span class="NLM_lpage">266</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Fjm500118j&amp;key=10.1038%2Fnrc2602" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Fjm500118j&amp;key=19242414" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2009&pages=253-266&author=M.+Malumbresauthor=M.+Barbacid&title=Cell+cycle%2C+CDKs+and+cancer%3A+a+changing+paradigm"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fnrc2602&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2602%26sid%3Dliteratum%253Aachs%26aulast%3DMalumbres%26aufirst%3DM.%26aulast%3DBarbacid%26aufirst%3DM.%26atitle%3DCell%2520cycle%252C%2520CDKs%2520and%2520cancer%253A%2520a%2520changing%2520paradigm%26jtitle%3DNature%2520Rev.%2520Cancer%26date%3D2009%26volume%3D9%26spage%3D253%26epage%3D266" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Li, Z.; Dai, K.; Keegan, K.; Ma, J.; Ragains, M.; Kaizerman, J.; McMinn, D.; Fu, J.; Fisher, B.; Gribble, M.</span><span class="NLM_etal"></span><span> </span><span class="NLM_article-title">CDK4/FLT3 dual inhibitors as potential therapeutics for acute myeloid leukemia</span>. In Drug Discovery and Lead Optimization; <span class="NLM_conf-name">Proceedings of the American Association for Cancer Research, AACR Annual Meeting 2013</span>, <span class="NLM_conf-loc">Washington, DC</span>, <span class="NLM_conf-date">April 6–10, 2013</span>;  <span class="NLM_fpage">Abstract 2351, p 575</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Li%2C+Z.%3B+Dai%2C+K.%3B+Keegan%2C+K.%3B+Ma%2C+J.%3B+Ragains%2C+M.%3B+Kaizerman%2C+J.%3B+McMinn%2C+D.%3B+Fu%2C+J.%3B+Fisher%2C+B.%3B+Gribble%2C+M.+CDK4%2FFLT3+dual+inhibitors+as+potential+therapeutics+for+acute+myeloid+leukemia.+In+Drug+Discovery+and+Lead+Optimization%3B+Proceedings+of+the+American+Association+for+Cancer+Research%2C+AACR+Annual+Meeting+2013%2C+Washington%2C+DC%2C+April+6%E2%80%9310%2C+2013%3B+Abstract+2351%2C+p+575."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dconf-proc%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DZ.%26atitle%3DCDK4%252FFLT3%2520dual%2520inhibitors%2520as%2520potential%2520therapeutics%2520for%2520acute%2520myeloid%2520leukemia%26jtitle%3DDrug%2520Discovery%2520and%2520Lead%2520Optimization%26spage%3DAbstract%25202351%252C%2520p%2520575" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="note"><p class="first last">All software programs used for molecular modeling are from Schrödinger (New York, NY). The homology model for FLT3 with the activation loop in the DFG-in conformation was built with cKit as the scaffold (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1PKG">1PKG</a>) using the Prime module. Compound <b>1</b> was modeled into the ATP binding site of the FLT3 model using Glide. Protein representations were prepared via PyMOL 1.5.0.5.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Malumbres, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pevarello, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barbacid, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bischoff, J. R.</span><span> </span><span class="NLM_article-title">CDK inhibitors in cancer therapy: What is next?</span> <span class="citation_source-journal">Trends Pharmacol. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">16</span><span class="NLM_x">–</span> <span class="NLM_lpage">21</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2008&pages=16-21&author=M.+Malumbresauthor=P.+Pevarelloauthor=M.+Barbacidauthor=J.+R.+Bischoff&title=CDK+inhibitors+in+cancer+therapy%3A+What+is+next%3F"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMalumbres%26aufirst%3DM.%26aulast%3DPevarello%26aufirst%3DP.%26aulast%3DBarbacid%26aufirst%3DM.%26aulast%3DBischoff%26aufirst%3DJ.%2BR.%26atitle%3DCDK%2520inhibitors%2520in%2520cancer%2520therapy%253A%2520What%2520is%2520next%253F%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2008%26volume%3D29%26spage%3D16%26epage%3D21" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="note"><p class="first last">Detailed docking studies of our compound to the CYP3A4 active site will be published elsewhere in due course.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Takács-Novák, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Szasz, G.</span><span> </span><span class="NLM_article-title">ion pair partition of quaternary ammonium drugs: the influence of counter ions of different lipophilicity, size, and flexibility</span> <span class="citation_source-journal">Pharm. Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">1633</span><span class="NLM_x">–</span> <span class="NLM_lpage">1638</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=1999&pages=1633-1638&author=K.+Tak%C3%A1cs-Nov%C3%A1kauthor=G.+Szasz&title=ion+pair+partition+of+quaternary+ammonium+drugs%3A+the+influence+of+counter+ions+of+different+lipophilicity%2C+size%2C+and+flexibility"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DTak%25C3%25A1cs-Nov%25C3%25A1k%26aufirst%3DK.%26aulast%3DSzasz%26aufirst%3DG.%26atitle%3Dion%2520pair%2520partition%2520of%2520quaternary%2520ammonium%2520drugs%253A%2520the%2520influence%2520of%2520counter%2520ions%2520of%2520different%2520lipophilicity%252C%2520size%252C%2520and%2520flexibility%26jtitle%3DPharm.%2520Res.%26date%3D1999%26volume%3D16%26spage%3D1633%26epage%3D1638" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span>KINOMEscan Assay Platform; <span class="NLM_publisher-name">DiscoveRx Corporation</span>: <span class="NLM_publisher-loc">Fremont, CA</span>; <a href="http://www.discoverx.com/technologies-platforms/competitive-binding-technology/kinomescan-technology-platform" class="extLink">http://www.discoverx.com/technologies-platforms/competitive-binding-technology/kinomescan-technology-platform</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=KINOMEscan+Assay+Platform%3B+DiscoveRx+Corporation%3A+Fremont%2C+CA%3B+http%3A%2F%2Fwww.discoverx.com%2Ftechnologies-platforms%2Fcompetitive-binding-technology%2Fkinomescan-technology-platform."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26jtitle%3DKINOMEscan%2520Assay%2520Platform%26pub%3DDiscoveRx%2520Corporation" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="note"><p class="first last">Sorafenib and palbociclib were purchased from AdooQ BioScience (Irvine, CA).</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="note"><p class="first last">More detailed in vitro and in vivo studies of <b>28</b> and its abilities to simultaneously inhibit CDK4 and FLT3 and suppress mutations in FLT3 will be published elsewhere in due course.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Zuccotto, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ardini, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casale, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Angiolini, M.</span><span> </span><span class="NLM_article-title">Through the “gatekeeper door”: Exploiting active kinase conformation</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">2681</span><span class="NLM_x">–</span> <span class="NLM_lpage">2694</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm901443h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2681-2694&author=F.+Zuccottoauthor=E.+Ardiniauthor=E.+Casaleauthor=M.+Angiolini&title=Through+the+%E2%80%9Cgatekeeper+door%E2%80%9D%3A+Exploiting+active+kinase+conformation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fjm901443h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901443h%26sid%3Dliteratum%253Aachs%26aulast%3DZuccotto%26aufirst%3DF.%26aulast%3DArdini%26aufirst%3DE.%26aulast%3DCasale%26aufirst%3DE.%26aulast%3DAngiolini%26aufirst%3DM.%26atitle%3DThrough%2520the%2520%25E2%2580%259Cgatekeeper%2520door%25E2%2580%259D%253A%2520Exploiting%2520active%2520kinase%2520conformation%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D2681%26epage%3D2694" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N. S.</span><span> </span><span class="NLM_article-title">Rational design of inhibitors that bind to inactive kinase conformations</span> <span class="citation_source-journal">Nature Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">358</span><span class="NLM_x">–</span> <span class="NLM_lpage">364</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Fjm500118j&amp;key=10.1038%2Fnchembio799" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Fjm500118j&amp;key=16783341" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Fjm500118j&amp;key=1%3ACAS%3A528%3ADC%252BD28XlvFGjs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2006&pages=358-364&author=Y.+Liuauthor=N.+S.+Gray&title=Rational+design+of+inhibitors+that+bind+to+inactive+kinase+conformations"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Rational design of inhibitors that bind to inactive kinase conformations</span></div><div class="casAuthors">Liu, Yi; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">358-364</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The majority of kinase inhibitors, known as type I inhibitors that have been developed so far target the ATP binding site of the kinase in its active conformation, in which the activation loop is phosphorylated.  Recently, crystal structures of inhibitors such as imatinib, BIRB796, and sorafenib (BAY43-9006), known as type II inhibitors, have revealed a new binding mode that exploits an addnl. binding site immediately adjacent to the region occupied by ATP.  This pocket is made accessible by an activation-loop rearrangement that is characteristic of kinases in an inactive conformation.  The examn. of binding features of known type II inhibitors is discussed and a pharmacophore model currently used in the design of a new generation of structurally diverse type II inhibitors is presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxahsMPlhgDbVg90H21EOLACvtfcHk0ljESiiKamKqBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XlvFGjs7g%253D&md5=4a9347a44a2cf4a412c9741fdcc5696b</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fnchembio799&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio799%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DRational%2520design%2520of%2520inhibitors%2520that%2520bind%2520to%2520inactive%2520kinase%2520conformations%26jtitle%3DNature%2520Chem.%2520Biol.%26date%3D2006%26volume%3D2%26spage%3D358%26epage%3D364" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="note"><p class="first last">FLT3 apo structure (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1RJB">1RJB</a>) used for DFG-out kinase with Sorafenib modeled with Glide. FLT3 DFG-in homology model (cKIT, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1PKG">1PKG</a>) used to model <b>28</b>.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="note"><p class="first last">The deference in inhibition between pRb and pSTAT5 at 24 h time point in 75 and 150 mg/kg dosing groups is due to the fact that total STAT5 was stable with the compound treatment, while total Rb decreased with the compound treatment. At 24 h at doses of 150 and 75 mg/kg, compound concentrations in plasma went down, so STAT5 phosphorylation was not inhibited, and back to normal. In contrast, although Rb phosphorylation was not inhibited at 24 h, but because total Rb level was low, pRb appeared low compared to vehicle.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="note"><p class="first last">Syntheses of certain substituted cycloalkane derivatives (e.g., <b>8</b>, <b>9</b>, and <b>14</b>) employed minor modifications of the general route depicted in Scheme <a class="ref internalNav" href="#sch1" aria-label="1">1</a>, such as permutation of steps (<b>14</b>) or inclusion of additional protecting group manipulations (<b>8</b>, <b>9</b>). These deviations from the mainstream approach are detailed in the experimental procedures for the relevant compounds.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="note"><p class="first last">The exocyclic amine nitrogen was arbitrarily numbered as 1′ in this scheme to facilitate unambiguous reference to this atom in the discussion. IUPAC names for the fully elaborated derivatives number this position differently.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Gribble, G. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saulnier, M. G.</span><span> </span><span class="NLM_article-title">Regioselective ortho lithiation of halopyridines</span> <span class="citation_source-journal">Tetrahedron Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1980</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">4137</span><span class="NLM_x">–</span> <span class="NLM_lpage">4140</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=1980&pages=4137-4140&author=G.+W.+Gribbleauthor=M.+G.+Saulnier&title=Regioselective+ortho+lithiation+of+halopyridines"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGribble%26aufirst%3DG.%2BW.%26aulast%3DSaulnier%26aufirst%3DM.%2BG.%26atitle%3DRegioselective%2520ortho%2520lithiation%2520of%2520halopyridines%26jtitle%3DTetrahedron%2520Lett.%26date%3D1980%26volume%3D21%26spage%3D4137%26epage%3D4140" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Marsais, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Queguiner, G.</span><span> </span><span class="NLM_article-title">Review on the metallation of p-deficient heteroaromatic compounds. Regioselective ortho-lithiation of 3-fluoropyridine: directing effects and application to synthesis of 2,3- or 3,4-disubstituted pyridines</span> <span class="citation_source-journal">Tetrahedron</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1983</span><span class="NLM_x">, </span> <span class="NLM_volume">39</span><span class="NLM_x">, </span> <span class="NLM_fpage">2009</span><span class="NLM_x">–</span> <span class="NLM_lpage">2021</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1983&pages=2009-2021&author=F.+Marsaisauthor=G.+Queguiner&title=Review+on+the+metallation+of+p-deficient+heteroaromatic+compounds.+Regioselective+ortho-lithiation+of+3-fluoropyridine%3A+directing+effects+and+application+to+synthesis+of+2%2C3-+or+3%2C4-disubstituted+pyridines"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMarsais%26aufirst%3DF.%26aulast%3DQueguiner%26aufirst%3DG.%26atitle%3DReview%2520on%2520the%2520metallation%2520of%2520p-deficient%2520heteroaromatic%2520compounds.%2520Regioselective%2520ortho-lithiation%2520of%25203-fluoropyridine%253A%2520directing%2520effects%2520and%2520application%2520to%2520synthesis%2520of%25202%252C3-%2520or%25203%252C4-disubstituted%2520pyridines%26jtitle%3DTetrahedron%26date%3D1983%26volume%3D39%26spage%3D2009%26epage%3D2021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Liu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schulze-Gahmen, U.</span><span> </span><span class="NLM_article-title">Toward understanding the structural basis of cyclin-dependent kinase 6 specific inhibition</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">3826</span><span class="NLM_x">–</span> <span class="NLM_lpage">3831</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=3826-3831&author=H.+Liuauthor=U.+Schulze-Gahmen&title=Toward+understanding+the+structural+basis+of+cyclin-dependent+kinase+6+specific+inhibition"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DSchulze-Gahmen%26aufirst%3DU.%26atitle%3DToward%2520understanding%2520the%2520structural%2520basis%2520of%2520cyclin-dependent%2520kinase%25206%2520specific%2520inhibition%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D3826%26epage%3D3831" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Battye, T. G. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kontogiannis, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Powell, H. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leslie, A. G. W.</span><span> </span><span class="NLM_article-title">iMOSFLM: a new graphical interface for diffraction-image processing with MOSFLM</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">67</span><span class="NLM_x">, </span> <span class="NLM_fpage">271</span><span class="NLM_x">–</span> <span class="NLM_lpage">281</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=271-281&author=T.+G.+G.+Battyeauthor=L.+Kontogiannisauthor=O.+Johnsonauthor=H.+R.+Powellauthor=A.+G.+W.+Leslie&title=iMOSFLM%3A+a+new+graphical+interface+for+diffraction-image+processing+with+MOSFLM"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBattye%26aufirst%3DT.%2BG.%2BG.%26aulast%3DKontogiannis%26aufirst%3DL.%26aulast%3DJohnson%26aufirst%3DO.%26aulast%3DPowell%26aufirst%3DH.%2BR.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%2BW.%26atitle%3DiMOSFLM%253A%2520a%2520new%2520graphical%2520interface%2520for%2520diffraction-image%2520processing%2520with%2520MOSFLM%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D271%26epage%3D281" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Winn, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ballard, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dodson, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Evans, P. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keegan, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krissinel, E. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leslie, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCoy, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McNicholas, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murshudov, G. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pannu, N. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Potterton, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Powell, H. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vagin, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, K. S.</span><span> </span><span class="NLM_article-title">Overview of the CCP4 suite and current developments</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">67</span><span class="NLM_x">, </span> <span class="NLM_fpage">235</span><span class="NLM_x">–</span> <span class="NLM_lpage">242</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Fjm500118j&amp;key=10.1107%2FS0907444910045749" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Fjm500118j&amp;key=21460441" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Fjm500118j&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=235-242&author=M.+D.+Winnauthor=C.+C.+Ballardauthor=K.+D.+Cowtanauthor=E.+J.+Dodsonauthor=P.+Emsleyauthor=P.+R.+Evansauthor=R.+M.+Keeganauthor=E.+B.+Krissinelauthor=A.+G.+Leslieauthor=A.+McCoyauthor=S.+J.+McNicholasauthor=G.+N.+Murshudovauthor=N.+S.+Pannuauthor=E.+A.+Pottertonauthor=H.+R.+Powellauthor=R.+J.+Readauthor=A.+Vaginauthor=K.+S.+Wilson&title=Overview+of+the+CCP4+suite+and+current+developments"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Overview of the CCP4 suite and current developments</span></div><div class="casAuthors">Winn, Martyn D.; Ballard, Charles C.; Cowtan, Kevin D.; Dodson, Eleanor J.; Emsley, Paul; Evans, Phil R.; Keegan, Ronan M.; Krissinel, Eugene B.; Leslie, Andrew G. W.; McCoy, Airlie; McNicholas, Stuart J.; Murshudov, Garib N.; Pannu, Navraj S.; Potterton, Elizabeth A.; Powell, Harold R.; Read, Randy J.; Vagin, Alexei; Wilson, Keith S.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  The CCP4 (Collaborative Computational Project, No. 4) software suite is a collection of programs and assocd. data and software libraries which can be used for macromol. structure detn. by X-ray crystallog.  The suite is designed to be flexible, allowing users a no. of methods of achieving their aims.  The programs are from a wide variety of sources but are connected by a common infrastructure provided by std. file formats, data objects and graphical interfaces.  Structure soln. by macromol. crystallog. is becoming increasingly automated and the CCP4 suite includes several automation pipelines.  After giving a brief description of the evolution of CCP4 over the last 30 years, an overview of the current suite is given.  While detailed descriptions are given in the accompanying articles, here it is shown how the individual programs contribute to a complete software package.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_wWMMIol6UrVg90H21EOLACvtfcHk0lhrEJDXVXTFfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D&md5=c407e4d47bef46864be336d60147c17d</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1107%2FS0907444910045749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910045749%26sid%3Dliteratum%253Aachs%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DBallard%26aufirst%3DC.%2BC.%26aulast%3DCowtan%26aufirst%3DK.%2BD.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DKeegan%26aufirst%3DR.%2BM.%26aulast%3DKrissinel%26aufirst%3DE.%2BB.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%26aulast%3DMcCoy%26aufirst%3DA.%26aulast%3DMcNicholas%26aufirst%3DS.%2BJ.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DPotterton%26aufirst%3DE.%2BA.%26aulast%3DPowell%26aufirst%3DH.%2BR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DWilson%26aufirst%3DK.%2BS.%26atitle%3DOverview%2520of%2520the%2520CCP4%2520suite%2520and%2520current%2520developments%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D235%26epage%3D242" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Adams, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Afonine, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunkóczi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, V. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davis, I. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Echols, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Headd, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hung, L.-W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kapral, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCoy, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moriarty, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oeffner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Terwilliger, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zwart, P. H.</span><span> </span><span class="NLM_article-title">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">213</span><span class="NLM_x">–</span> <span class="NLM_lpage">221</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Fjm500118j&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Fjm500118j&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Fjm500118j&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunk%C3%B3cziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.-W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+a+comprehensive+Python-based+system+for+macromolecular+structure+solution"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0lgqSnOcJkihgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunk%25C3%25B3czi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.-W.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520a%2520comprehensive%2520Python-based%2520system%2520for%2520macromolecular%2520structure%2520solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lohkamp, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, W. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K.</span><span> </span><span class="NLM_article-title">Features and development of Coot</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">486</span><span class="NLM_x">–</span> <span class="NLM_lpage">501</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Fjm500118j&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Fjm500118j&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Fjm500118j&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+of+Coot"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0lgqSnOcJkihgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':[],'ref15':['cit15'],'ref16':[],'ref17':['cit17'],'ref18':['cit18'],'ref19':[],'ref20':[],'ref21':['cit21'],'ref22':['cit22'],'ref23':[],'ref24':[],'ref25':[],'ref26':[],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 42 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Junwei Wang, Xiang Pan, Yi Song, Jian Liu, Fei Ma, Ping Wang, Yan Liu, Lin Zhao, Di Kang, <span class="NLM_string-name hlFld-ContribAuthor">Lihong Hu</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of a Potent and Selective FLT3 Inhibitor (Z)-N-(5-((5-Fluoro-2-oxoindolin-3-ylidene)methyl)-4-methyl-1H-pyrrol-3-yl)-3-(pyrrolidin-1-yl)propanamide with Improved Drug-like Properties and Superior Efficacy in FLT3-ITD-Positive Acute Myeloid Leukemia. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (8)
                                     , 4870-4890. <a href="https://doi.org/10.1021/acs.jmedchem.0c02247" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02247</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02247&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c02247%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Ba%252BPotent%252Band%252BSelective%252BFLT3%252BInhibitor%252B%252528Z%252529-N-%2525285-%252528%2525285-Fluoro-2-oxoindolin-3-ylidene%252529methyl%252529-4-methyl-1H-pyrrol-3-yl%252529-3-%252528pyrrolidin-1-yl%252529propanamide%252Bwith%252BImproved%252BDrug-like%252BProperties%252Band%252BSuperior%252BEfficacy%252Bin%252BFLT3-ITD-Positive%252BAcute%252BMyeloid%252BLeukemia%26aulast%3DWang%26aufirst%3DJunwei%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D28122020%26date%3D02042021%26volume%3D64%26issue%3D8%26spage%3D4870%26epage%3D4890" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Zhijie Wang, Jiongheng Cai, Jie Cheng, Wenqianzi Yang, Yifan Zhu, Hongmei Li, Tao Lu, Yadong Chen, <span class="NLM_string-name hlFld-ContribAuthor">Shuai Lu</span>. </span><span class="cited-content_cbyCitation_article-title">FLT3 Inhibitors in Acute Myeloid Leukemia: Challenges and Recent Developments in Overcoming Resistance. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (6)
                                     , 2878-2900. <a href="https://doi.org/10.1021/acs.jmedchem.0c01851" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01851</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01851&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01851%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DFLT3%252BInhibitors%252Bin%252BAcute%252BMyeloid%252BLeukemia%25253A%252BChallenges%252Band%252BRecent%252BDevelopments%252Bin%252BOvercoming%252BResistance%26aulast%3DWang%26aufirst%3DZhijie%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D23102020%26date%3D10032021%26volume%3D64%26issue%3D6%26spage%3D2878%26epage%3D2900" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Yue Zhong, Run-Ze Qiu, Shan-Liang Sun, Chao Zhao, Tian-Yuan Fan, Min Chen, Nian-Guang Li, <span class="NLM_string-name hlFld-ContribAuthor">Zhi-Hao Shi</span>. </span><span class="cited-content_cbyCitation_article-title">Small-Molecule Fms-like Tyrosine Kinase 3 Inhibitors: An Attractive and Efficient Method for the Treatment of Acute Myeloid Leukemia. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (21)
                                     , 12403-12428. <a href="https://doi.org/10.1021/acs.jmedchem.0c00696" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00696</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00696&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00696%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DSmall-Molecule%252BFms-like%252BTyrosine%252BKinase%252B3%252BInhibitors%25253A%252BAn%252BAttractive%252Band%252BEfficient%252BMethod%252Bfor%252Bthe%252BTreatment%252Bof%252BAcute%252BMyeloid%252BLeukemia%26aulast%3DZhong%26aufirst%3DYue%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D26042020%26date%3D21072020%26date%3D13072020%26volume%3D63%26issue%3D21%26spage%3D12403%26epage%3D12428" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Lucia Veselovská, Radek Pohl, Eva Tloušt′ová, Soňa Gurská, Petr Džubák, Marián Hajdúch, <span class="NLM_string-name hlFld-ContribAuthor">Michal Hocek</span>. </span><span class="cited-content_cbyCitation_article-title">Pyrido-Fused Deazapurine Bases: Synthesis and Glycosylation of 4-Substituted 9H-Pyrido[2′,3′:4,5]- and Pyrido[4′,3′:4,5]pyrrolo[2,3-d]pyrimidines. </span><span class="cited-content_cbyCitation_journal-name">ACS Omega</span><span> <strong>2020,</strong> <em>5 </em>
                                    (40)
                                     , 26278-26286. <a href="https://doi.org/10.1021/acsomega.0c04302" title="DOI URL">https://doi.org/10.1021/acsomega.0c04302</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsomega.0c04302&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsomega.0c04302%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Omega%26atitle%3DPyrido-Fused%252BDeazapurine%252BBases%25253A%252BSynthesis%252Band%252BGlycosylation%252Bof%252B4-Substituted%252B9H-Pyrido%25255B2%2525E2%252580%2525B2%25252C3%2525E2%252580%2525B2%25253A4%25252C5%25255D-%252Band%252BPyrido%25255B4%2525E2%252580%2525B2%25252C3%2525E2%252580%2525B2%25253A4%25252C5%25255Dpyrrolo%25255B2%25252C3-d%25255Dpyrimidines%26aulast%3DVeselovsk%25C3%25A1%26aufirst%3DLucia%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D03092020%26date%3D24092020%26date%3D02102020%26volume%3D5%26issue%3D40%26spage%3D26278%26epage%3D26286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Tao Yang, Mengshi Hu, Wenyan Qi, Zhuang Yang, Minghai Tang, Jun He, Yong Chen, Peng Bai, Xue Yuan, Chufeng Zhang, Kongjun Liu, Yulin Lu, Mingli Xiang, <span class="NLM_string-name hlFld-ContribAuthor">Lijuan Chen</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of Potent and Orally Effective Dual Janus Kinase 2/FLT3 Inhibitors for the Treatment of Acute Myelogenous Leukemia and Myeloproliferative Neoplasms. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (22)
                                     , 10305-10320. <a href="https://doi.org/10.1021/acs.jmedchem.9b01348" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01348</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01348&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b01348%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BPotent%252Band%252BOrally%252BEffective%252BDual%252BJanus%252BKinase%252B2%25252FFLT3%252BInhibitors%252Bfor%252Bthe%252BTreatment%252Bof%252BAcute%252BMyelogenous%252BLeukemia%252Band%252BMyeloproliferative%252BNeoplasms%26aulast%3DYang%26aufirst%3DTao%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D15082019%26date%3D08112019%26date%3D31102019%26volume%3D62%26issue%3D22%26spage%3D10305%26epage%3D10320" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Tomáš Gucký, Eva Řezníčková, Tereza Radošová Muchová, Radek Jorda, Zuzana Klejová, Veronika Malínková, Karel Berka, Václav Bazgier, Haresh Ajani, Martin Lepšík, Vladimír Divoký, <span class="NLM_string-name hlFld-ContribAuthor">Vladimír Kryštof</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of N2-(4-Amino-cyclohexyl)-9-cyclopentyl-N6-(4-morpholin-4-ylmethyl-phenyl)-9H-purine-2,6-diamine as a Potent FLT3 Kinase Inhibitor for Acute Myeloid Leukemia with FLT3 Mutations. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (9)
                                     , 3855-3869. <a href="https://doi.org/10.1021/acs.jmedchem.7b01529" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b01529</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b01529&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.7b01529%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BN2-%2525284-Amino-cyclohexyl%252529-9-cyclopentyl-N6-%2525284-morpholin-4-ylmethyl-phenyl%252529-9H-purine-2%25252C6-diamine%252Bas%252Ba%252BPotent%252BFLT3%252BKinase%252BInhibitor%252Bfor%252BAcute%252BMyeloid%252BLeukemia%252Bwith%252BFLT3%252BMutations%26aulast%3DGuck%25C3%25BD%26aufirst%3DTom%25C3%25A1%25C5%25A1%26date%3D2018%26date%3D2018%26date%3D2017%26date%3D13102017%26date%3D30042018%26date%3D19042018%26volume%3D61%26issue%3D9%26spage%3D3855%26epage%3D3869" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yue  Wang</span>, <span class="hlFld-ContribAuthor ">Yanle  Zhi</span>, <span class="hlFld-ContribAuthor ">Qiaomei  Jin</span>, <span class="hlFld-ContribAuthor ">Shuai  Lu</span>, <span class="hlFld-ContribAuthor ">Guowu  Lin</span>, <span class="hlFld-ContribAuthor ">Haoliang  Yuan</span>, <span class="hlFld-ContribAuthor ">Taotao  Yang</span>, <span class="hlFld-ContribAuthor ">Zhanwei  Wang</span>, <span class="hlFld-ContribAuthor ">Chao  Yao</span>, <span class="hlFld-ContribAuthor ">Jun  Ling</span>, <span class="hlFld-ContribAuthor ">Hao  Guo</span>, <span class="hlFld-ContribAuthor ">Tonghui  Li</span>, <span class="hlFld-ContribAuthor ">Jianlin  Jin</span>, <span class="hlFld-ContribAuthor ">Baoquan  Li</span>, <span class="hlFld-ContribAuthor ">Li  Zhang</span>, <span class="hlFld-ContribAuthor ">Yadong  Chen</span>, and <span class="hlFld-ContribAuthor ">Tao  Lu</span>  . </span><span class="cited-content_cbyCitation_article-title">Discovery of 4-((7H-Pyrrolo[2,3-d]pyrimidin-4-yl)amino)-N-(4-((4-methylpiperazin-1-yl)methyl)phenyl)-1H-pyrazole-3-carboxamide (FN-1501), an FLT3- and CDK-Kinase Inhibitor with Potentially High Efficiency against Acute Myelocytic Leukemia. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (4)
                                     , 1499-1518. <a href="https://doi.org/10.1021/acs.jmedchem.7b01261" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b01261</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b01261&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.7b01261%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252B4-%252528%2525287H-Pyrrolo%25255B2%25252C3-d%25255Dpyrimidin-4-yl%252529amino%252529-N-%2525284-%252528%2525284-methylpiperazin-1-yl%252529methyl%252529phenyl%252529-1H-pyrazole-3-carboxamide%252B%252528FN-1501%252529%25252C%252Ban%252BFLT3-%252Band%252BCDK-Kinase%252BInhibitor%252Bwith%252BPotentially%252BHigh%252BEfficiency%252Bagainst%252BAcute%252BMyelocytic%252BLeukemia%26aulast%3DWang%26aufirst%3DYue%26date%3D2018%26date%3D2018%26date%3D2017%26date%3D24082017%26date%3D12022018%26date%3D22022018%26date%3D22012018%26volume%3D61%26issue%3D4%26spage%3D1499%26epage%3D1518" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lewis D.  Pennington</span> and <span class="hlFld-ContribAuthor ">Demetri T.  Moustakas</span>  . </span><span class="cited-content_cbyCitation_article-title">The Necessary Nitrogen Atom: A Versatile High-Impact Design Element for Multiparameter Optimization. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2017,</strong> <em>60 </em>
                                    (9)
                                     , 3552-3579. <a href="https://doi.org/10.1021/acs.jmedchem.6b01807" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.6b01807</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.6b01807&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.6b01807%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DThe%252BNecessary%252BNitrogen%252BAtom%25253A%252BA%252BVersatile%252BHigh-Impact%252BDesign%252BElement%252Bfor%252BMultiparameter%252BOptimization%26aulast%3DPennington%26aufirst%3DLewis%2BD.%26date%3D2017%26date%3D2017%26date%3D2016%26date%3D09122016%26date%3D08022017%26date%3D11052017%26volume%3D60%26issue%3D9%26spage%3D3552%26epage%3D3579" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Deheng  Sun</span>, <span class="hlFld-ContribAuthor ">Yu  Yang</span>, <span class="hlFld-ContribAuthor ">Jiankun  Lyu</span>, <span class="hlFld-ContribAuthor ">Wei  Zhou</span>, <span class="hlFld-ContribAuthor ">Wenlin  Song</span>, <span class="hlFld-ContribAuthor ">Zhenjiang  Zhao</span>, <span class="hlFld-ContribAuthor ">Zhuo  Chen</span>, <span class="hlFld-ContribAuthor ">Yufang  Xu</span>, and <span class="hlFld-ContribAuthor ">Honglin  Li</span>  . </span><span class="cited-content_cbyCitation_article-title">Discovery and Rational Design of Pteridin-7(8H)-one-Based Inhibitors Targeting FMS-like Tyrosine Kinase 3 (FLT3) and Its Mutants. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2016,</strong> <em>59 </em>
                                    (13)
                                     , 6187-6200. <a href="https://doi.org/10.1021/acs.jmedchem.6b00374" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.6b00374</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.6b00374&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.6b00374%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Band%252BRational%252BDesign%252Bof%252BPteridin-7%2525288H%252529-one-Based%252BInhibitors%252BTargeting%252BFMS-like%252BTyrosine%252BKinase%252B3%252B%252528FLT3%252529%252Band%252BIts%252BMutants%26aulast%3DSun%26aufirst%3DDeheng%26date%3D2016%26date%3D2016%26date%3D2016%26date%3D11032016%26date%3D22062016%26date%3D14072016%26date%3D07062016%26volume%3D59%26issue%3D13%26spage%3D6187%26epage%3D6200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xixiang  Li</span>, <span class="hlFld-ContribAuthor ">Aoli  Wang</span>, <span class="hlFld-ContribAuthor ">Kailin  Yu</span>, <span class="hlFld-ContribAuthor ">Ziping  Qi</span>, <span class="hlFld-ContribAuthor ">Cheng  Chen</span>, <span class="hlFld-ContribAuthor ">Wenchao  Wang</span>, <span class="hlFld-ContribAuthor ">Chen  Hu</span>, <span class="hlFld-ContribAuthor ">Hong  Wu</span>, <span class="hlFld-ContribAuthor ">Jiaxin  Wu</span>, <span class="hlFld-ContribAuthor ">Zheng  Zhao</span>, <span class="hlFld-ContribAuthor ">Juan  Liu</span>, <span class="hlFld-ContribAuthor ">Fengming  Zou</span>, <span class="hlFld-ContribAuthor ">Li  Wang</span>, <span class="hlFld-ContribAuthor ">Beilei  Wang</span>, <span class="hlFld-ContribAuthor ">Wei  Wang</span>, <span class="hlFld-ContribAuthor ">Shanchun  Zhang</span>, <span class="hlFld-ContribAuthor ">Jing  Liu</span>, and <span class="hlFld-ContribAuthor ">Qingsong  Liu</span>  . </span><span class="cited-content_cbyCitation_article-title">Discovery of (R)-1-(3-(4-Amino-3-(4-phenoxyphenyl)-1H-pyrazolo[3,4-d]pyrimidin-1-yl)piperidin-1-yl)-2-(dimethylamino)ethanone (CHMFL-FLT3-122) as a Potent and Orally Available FLT3 Kinase Inhibitor for FLT3-ITD Positive Acute Myeloid Leukemia. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2015,</strong> <em>58 </em>
                                    (24)
                                     , 9625-9638. <a href="https://doi.org/10.1021/acs.jmedchem.5b01611" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.5b01611</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.5b01611&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.5b01611%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252B%252528R%252529-1-%2525283-%2525284-Amino-3-%2525284-phenoxyphenyl%252529-1H-pyrazolo%25255B3%25252C4-d%25255Dpyrimidin-1-yl%252529piperidin-1-yl%252529-2-%252528dimethylamino%252529ethanone%252B%252528CHMFL-FLT3-122%252529%252Bas%252Ba%252BPotent%252Band%252BOrally%252BAvailable%252BFLT3%252BKinase%252BInhibitor%252Bfor%252BFLT3-ITD%252BPositive%252BAcute%252BMyeloid%252BLeukemia%26aulast%3DLi%26aufirst%3DXixiang%26date%3D2015%26date%3D2015%26date%3D2015%26date%3D05082015%26date%3D14122015%26date%3D24122015%26date%3D02122015%26volume%3D58%26issue%3D24%26spage%3D9625%26epage%3D9638" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Caroline  Affouard</span>, <span class="hlFld-ContribAuthor ">Richard D.  Crockett</span>, <span class="hlFld-ContribAuthor ">Khalid  Diker</span>, <span class="hlFld-ContribAuthor ">Robert P.  Farrell</span>, <span class="hlFld-ContribAuthor ">Gilles  Gorins</span>, <span class="hlFld-ContribAuthor ">John R.  Huckins</span>, and <span class="hlFld-ContribAuthor ">Seb  Caille</span>  . </span><span class="cited-content_cbyCitation_article-title">Multi-Kilo Delivery of AMG 925 Featuring a Buchwald–Hartwig Amination and Processing with Insoluble Synthetic Intermediates. </span><span class="cited-content_cbyCitation_journal-name">Organic Process Research & Development</span><span> <strong>2015,</strong> <em>19 </em>
                                    (3)
                                     , 476-485. <a href="https://doi.org/10.1021/op500367p" title="DOI URL">https://doi.org/10.1021/op500367p</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/op500367p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fop500367p%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520Process%2520Research%2520%2526%2520Development%26atitle%3DMulti-Kilo%252BDelivery%252Bof%252BAMG%252B925%252BFeaturing%252Ba%252BBuchwald%2525E2%252580%252593Hartwig%252BAmination%252Band%252BProcessing%252Bwith%252BInsoluble%252BSynthetic%252BIntermediates%26aulast%3DAffouard%26aufirst%3DCaroline%26date%3D2015%26date%3D2015%26date%3D2014%26date%3D25112014%26date%3D27022015%26date%3D20032015%26date%3D23022015%26volume%3D19%26issue%3D3%26spage%3D476%26epage%3D485" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Haochao  Zhang</span>, <span class="hlFld-ContribAuthor ">Liandong  Jing</span>, <span class="hlFld-ContribAuthor ">Man  Liu</span>, <span class="hlFld-ContribAuthor ">Masuo  Goto</span>, <span class="hlFld-ContribAuthor ">Fangfang  Lai</span>, <span class="hlFld-ContribAuthor ">Xiao  Liu</span>, <span class="hlFld-ContribAuthor ">Li  Sheng</span>, <span class="hlFld-ContribAuthor ">Yajun  Yang</span>, <span class="hlFld-ContribAuthor ">Ying  Yang</span>, <span class="hlFld-ContribAuthor ">Yan  Li</span>, <span class="hlFld-ContribAuthor ">Xiaoguang  Chen</span>, <span class="hlFld-ContribAuthor ">Kuo-Hsiung  Lee</span>, <span class="hlFld-ContribAuthor ">Zhiyan  Xiao</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of 3, 4-disubstituted pyridine derivatives as novel CDK8 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>223 </em>, 113634. <a href="https://doi.org/10.1016/j.ejmech.2021.113634" title="DOI URL">https://doi.org/10.1016/j.ejmech.2021.113634</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2021.113634&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2021.113634%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DIdentification%252Bof%252B3%25252C%252B4-disubstituted%252Bpyridine%252Bderivatives%252Bas%252Bnovel%252BCDK8%252Binhibitors%26aulast%3DZhang%26aufirst%3DHaochao%26date%3D2021%26volume%3D223%26spage%3D113634" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhihong  Zeng</span>, <span class="hlFld-ContribAuthor ">Charlie  Ly</span>, <span class="hlFld-ContribAuthor ">Naval  Daver</span>, <span class="hlFld-ContribAuthor ">Jorge  Cortes</span>, <span class="hlFld-ContribAuthor ">Hagop M.  Kantarjian</span>, <span class="hlFld-ContribAuthor ">Michael  Andreeff</span>, <span class="hlFld-ContribAuthor ">Marina  Konopleva</span>. </span><span class="cited-content_cbyCitation_article-title">High-throughput proteomic profiling reveals mechanisms of action of AMG925, a dual FLT3-CDK4/6 kinase inhibitor targeting AML and AML stem/progenitor cells. </span><span class="cited-content_cbyCitation_journal-name">Annals of Hematology</span><span> <strong>2021,</strong> <em>100 </em>
                                    (6)
                                     , 1485-1496. <a href="https://doi.org/10.1007/s00277-021-04493-0" title="DOI URL">https://doi.org/10.1007/s00277-021-04493-0</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s00277-021-04493-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs00277-021-04493-0%26sid%3Dliteratum%253Aachs%26jtitle%3DAnnals%2520of%2520Hematology%26atitle%3DHigh-throughput%252Bproteomic%252Bprofiling%252Breveals%252Bmechanisms%252Bof%252Baction%252Bof%252BAMG925%25252C%252Ba%252Bdual%252BFLT3-CDK4%25252F6%252Bkinase%252Binhibitor%252Btargeting%252BAML%252Band%252BAML%252Bstem%25252Fprogenitor%252Bcells%26aulast%3DZeng%26aufirst%3DZhihong%26date%3D2021%26date%3D2021%26volume%3D100%26issue%3D6%26spage%3D1485%26epage%3D1496" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiaojiao  Luo</span>, <span class="hlFld-ContribAuthor ">Yu  Zhao</span>, <span class="hlFld-ContribAuthor ">Pan  Tang</span>, <span class="hlFld-ContribAuthor ">Xingkai  Du</span>, <span class="hlFld-ContribAuthor ">Feng  Li</span>, <span class="hlFld-ContribAuthor ">Qingying  Wang</span>, <span class="hlFld-ContribAuthor ">Rong  Li</span>, <span class="hlFld-ContribAuthor ">Jun  He</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of new small-molecule cyclin-dependent kinase 6 inhibitors through computational approaches. </span><span class="cited-content_cbyCitation_journal-name">Molecular Diversity</span><span> <strong>2021,</strong> <em>25 </em>
                                    (1)
                                     , 367-382. <a href="https://doi.org/10.1007/s11030-020-10120-3" title="DOI URL">https://doi.org/10.1007/s11030-020-10120-3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s11030-020-10120-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs11030-020-10120-3%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Diversity%26atitle%3DDiscovery%252Bof%252Bnew%252Bsmall-molecule%252Bcyclin-dependent%252Bkinase%252B6%252Binhibitors%252Bthrough%252Bcomputational%252Bapproaches%26aulast%3DLuo%26aufirst%3DXiaojiao%26date%3D2021%26date%3D2020%26volume%3D25%26issue%3D1%26spage%3D367%26epage%3D382" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kai  Yuan</span>, <span class="hlFld-ContribAuthor ">Xiao  Wang</span>, <span class="hlFld-ContribAuthor ">Haojie  Dong</span>, <span class="hlFld-ContribAuthor ">Wenjian  Min</span>, <span class="hlFld-ContribAuthor ">Haiping  Hao</span>, <span class="hlFld-ContribAuthor ">Peng  Yang</span>. </span><span class="cited-content_cbyCitation_article-title">Selective inhibition of CDK4/6: A safe and effective strategy for developing anticancer drugs. </span><span class="cited-content_cbyCitation_journal-name">Acta Pharmaceutica Sinica B</span><span> <strong>2021,</strong> <em>11 </em>
                                    (1)
                                     , 30-54. <a href="https://doi.org/10.1016/j.apsb.2020.05.001" title="DOI URL">https://doi.org/10.1016/j.apsb.2020.05.001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.apsb.2020.05.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.apsb.2020.05.001%26sid%3Dliteratum%253Aachs%26jtitle%3DActa%2520Pharmaceutica%2520Sinica%2520B%26atitle%3DSelective%252Binhibition%252Bof%252BCDK4%25252F6%25253A%252BA%252Bsafe%252Band%252Beffective%252Bstrategy%252Bfor%252Bdeveloping%252Banticancer%252Bdrugs%26aulast%3DYuan%26aufirst%3DKai%26date%3D2021%26volume%3D11%26issue%3D1%26spage%3D30%26epage%3D54" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lisa  Scheiblecker</span>, <span class="hlFld-ContribAuthor ">Karoline  Kollmann</span>, <span class="hlFld-ContribAuthor ">Veronika  Sexl</span>. </span><span class="cited-content_cbyCitation_article-title">CDK4/6 and MAPK—Crosstalk as Opportunity for Cancer Treatment. </span><span class="cited-content_cbyCitation_journal-name">Pharmaceuticals</span><span> <strong>2020,</strong> <em>13 </em>
                                    (12)
                                     , 418. <a href="https://doi.org/10.3390/ph13120418" title="DOI URL">https://doi.org/10.3390/ph13120418</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ph13120418&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fph13120418%26sid%3Dliteratum%253Aachs%26jtitle%3DPharmaceuticals%26atitle%3DCDK4%25252F6%252Band%252BMAPK%2525E2%252580%252594Crosstalk%252Bas%252BOpportunity%252Bfor%252BCancer%252BTreatment%26aulast%3DScheiblecker%26aufirst%3DLisa%26date%3D2020%26date%3D2020%26volume%3D13%26issue%3D12%26spage%3D418" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yingxiu  Li</span>, <span class="hlFld-ContribAuthor ">Peng  Wang</span>, <span class="hlFld-ContribAuthor ">Cong  Chen</span>, <span class="hlFld-ContribAuthor ">Tianyu  Ye</span>, <span class="hlFld-ContribAuthor ">Yufei  Han</span>, <span class="hlFld-ContribAuthor ">Yunlei  Hou</span>, <span class="hlFld-ContribAuthor ">Yajing  Liu</span>, <span class="hlFld-ContribAuthor ">Ping  Gong</span>, <span class="hlFld-ContribAuthor ">Mingze  Qin</span>, <span class="hlFld-ContribAuthor ">Yanfang  Zhao</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery and rational design of 2-aminopyrimidine-based derivatives targeting Janus kinase 2 (JAK2) and FMS-like tyrosine kinase 3 (FLT3). </span><span class="cited-content_cbyCitation_journal-name">Bioorganic Chemistry</span><span> <strong>2020,</strong> <em>104 </em>, 104361. <a href="https://doi.org/10.1016/j.bioorg.2020.104361" title="DOI URL">https://doi.org/10.1016/j.bioorg.2020.104361</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bioorg.2020.104361&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bioorg.2020.104361%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520Chemistry%26atitle%3DDiscovery%252Band%252Brational%252Bdesign%252Bof%252B2-aminopyrimidine-based%252Bderivatives%252Btargeting%252BJanus%252Bkinase%252B2%252B%252528JAK2%252529%252Band%252BFMS-like%252Btyrosine%252Bkinase%252B3%252B%252528FLT3%252529%26aulast%3DLi%26aufirst%3DYingxiu%26date%3D2020%26volume%3D104%26spage%3D104361" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lexian  Tong</span>, <span class="hlFld-ContribAuthor ">Xuemei  Li</span>, <span class="hlFld-ContribAuthor ">Yongzhou  Hu</span>, <span class="hlFld-ContribAuthor ">Tao  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Recent advances in FLT3 inhibitors for acute myeloid leukemia. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2020,</strong> <em>12 </em>
                                    (10)
                                     , 961-981. <a href="https://doi.org/10.4155/fmc-2019-0365" title="DOI URL">https://doi.org/10.4155/fmc-2019-0365</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc-2019-0365&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc-2019-0365%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DRecent%252Badvances%252Bin%252BFLT3%252Binhibitors%252Bfor%252Bacute%252Bmyeloid%252Bleukemia%26aulast%3DTong%26aufirst%3DLexian%26date%3D2020%26volume%3D12%26issue%3D10%26spage%3D961%26epage%3D981" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Iris Z.  Uras</span>, <span class="hlFld-ContribAuthor ">Veronika  Sexl</span>, <span class="hlFld-ContribAuthor ">Karoline  Kollmann</span>. </span><span class="cited-content_cbyCitation_article-title">CDK6 Inhibition: A Novel Approach in AML Management. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2020,</strong> <em>21 </em>
                                    (7)
                                     , 2528. <a href="https://doi.org/10.3390/ijms21072528" title="DOI URL">https://doi.org/10.3390/ijms21072528</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms21072528&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms21072528%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DCDK6%252BInhibition%25253A%252BA%252BNovel%252BApproach%252Bin%252BAML%252BManagement%26aulast%3DUras%26aufirst%3DIris%2BZ.%26date%3D2020%26date%3D2020%26volume%3D21%26issue%3D7%26spage%3D2528" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chunqing  Li</span>, <span class="hlFld-ContribAuthor ">Qi  Sun</span>, <span class="hlFld-ContribAuthor ">Qiang  Zhao</span>, <span class="hlFld-ContribAuthor ">Xinjian  Cheng</span>. </span><span class="cited-content_cbyCitation_article-title">Highly selective ratiometric fluorescent probes for the detection of Fe3+ and its application in living cells. </span><span class="cited-content_cbyCitation_journal-name">Spectrochimica Acta Part A: Molecular and Biomolecular Spectroscopy</span><span> <strong>2020,</strong> <em>228 </em>, 117720. <a href="https://doi.org/10.1016/j.saa.2019.117720" title="DOI URL">https://doi.org/10.1016/j.saa.2019.117720</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.saa.2019.117720&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.saa.2019.117720%26sid%3Dliteratum%253Aachs%26jtitle%3DSpectrochimica%2520Acta%2520Part%2520A%253A%2520Molecular%2520and%2520Biomolecular%2520Spectroscopy%26atitle%3DHighly%252Bselective%252Bratiometric%252Bfluorescent%252Bprobes%252Bfor%252Bthe%252Bdetection%252Bof%252BFe3%25252B%252Band%252Bits%252Bapplication%252Bin%252Bliving%252Bcells%26aulast%3DLi%26aufirst%3DChunqing%26date%3D2020%26volume%3D228%26spage%3D117720" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Pingping  Chen</span>, <span class="hlFld-ContribAuthor ">Yinqiu  Xu</span>, <span class="hlFld-ContribAuthor ">Xuanyi  Li</span>, <span class="hlFld-ContribAuthor ">Hequan  Yao</span>, <span class="hlFld-ContribAuthor ">Kejiang  Lin</span>. </span><span class="cited-content_cbyCitation_article-title">Development and strategies of CDK4/6 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2020,</strong> <em>12 </em>
                                    (2)
                                     , 127-145. <a href="https://doi.org/10.4155/fmc-2019-0062" title="DOI URL">https://doi.org/10.4155/fmc-2019-0062</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc-2019-0062&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc-2019-0062%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DDevelopment%252Band%252Bstrategies%252Bof%252BCDK4%25252F6%252Binhibitors%26aulast%3DChen%26aufirst%3DPingping%26date%3D2020%26volume%3D12%26issue%3D2%26spage%3D127%26epage%3D145" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Concepción  Sánchez-Martínez</span>, <span class="hlFld-ContribAuthor ">María José  Lallena</span>, <span class="hlFld-ContribAuthor ">Sonia Gutiérrez  Sanfeliciano</span>, <span class="hlFld-ContribAuthor ">Alfonso  de Dios</span>. </span><span class="cited-content_cbyCitation_article-title">Cyclin dependent kinase (CDK) inhibitors as anticancer drugs: Recent advances (2015–2019). </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2019,</strong> <em>29 </em>
                                    (20)
                                     , 126637. <a href="https://doi.org/10.1016/j.bmcl.2019.126637" title="DOI URL">https://doi.org/10.1016/j.bmcl.2019.126637</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2019.126637&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2019.126637%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DCyclin%252Bdependent%252Bkinase%252B%252528CDK%252529%252Binhibitors%252Bas%252Banticancer%252Bdrugs%25253A%252BRecent%252Badvances%252B%2525282015%2525E2%252580%2525932019%252529%26aulast%3DS%25C3%25A1nchez-Mart%25C3%25ADnez%26aufirst%3DConcepci%25C3%25B3n%26date%3D2019%26volume%3D29%26issue%3D20%26spage%3D126637" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ting  Yuan</span>, <span class="hlFld-ContribAuthor ">Baowen  Qi</span>, <span class="hlFld-ContribAuthor ">Zhongliang  Jiang</span>, <span class="hlFld-ContribAuthor ">Wenjuan  Dong</span>, <span class="hlFld-ContribAuthor ">Lei  Zhong</span>, <span class="hlFld-ContribAuthor ">Lan  Bai</span>, <span class="hlFld-ContribAuthor ">Rongsheng  Tong</span>, <span class="hlFld-ContribAuthor ">Jiying  Yu</span>, <span class="hlFld-ContribAuthor ">Jianyou  Shi</span>. </span><span class="cited-content_cbyCitation_article-title">Dual FLT3 inhibitors: Against the drug resistance of acute myeloid leukemia in recent decade. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>178 </em>, 468-483. <a href="https://doi.org/10.1016/j.ejmech.2019.06.002" title="DOI URL">https://doi.org/10.1016/j.ejmech.2019.06.002</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2019.06.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2019.06.002%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDual%252BFLT3%252Binhibitors%25253A%252BAgainst%252Bthe%252Bdrug%252Bresistance%252Bof%252Bacute%252Bmyeloid%252Bleukemia%252Bin%252Brecent%252Bdecade%26aulast%3DYuan%26aufirst%3DTing%26date%3D2019%26volume%3D178%26spage%3D468%26epage%3D483" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lexian  Tong</span>, <span class="hlFld-ContribAuthor ">Pinrao  Song</span>, <span class="hlFld-ContribAuthor ">Kailong  Jiang</span>, <span class="hlFld-ContribAuthor ">Lei  Xu</span>, <span class="hlFld-ContribAuthor ">Tingting  Jin</span>, <span class="hlFld-ContribAuthor ">Peipei  Wang</span>, <span class="hlFld-ContribAuthor ">Xiaobei  Hu</span>, <span class="hlFld-ContribAuthor ">Sui  Fang</span>, <span class="hlFld-ContribAuthor ">Anhui  Gao</span>, <span class="hlFld-ContribAuthor ">Yubo  Zhou</span>, <span class="hlFld-ContribAuthor ">Tao  Liu</span>, <span class="hlFld-ContribAuthor ">Jia  Li</span>, <span class="hlFld-ContribAuthor ">Yongzhou  Hu</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of (R)-5-((5-(1-methyl-1H-pyrazol-4-yl)-4-(methylamino)pyrimidin-2-yl)amino)-3-(piperidin-3-yloxy)picolinonitrile, a novel CHK1 inhibitor for hematologic malignancies. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>173 </em>, 44-62. <a href="https://doi.org/10.1016/j.ejmech.2019.03.062" title="DOI URL">https://doi.org/10.1016/j.ejmech.2019.03.062</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2019.03.062&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2019.03.062%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252B%252528R%252529-5-%252528%2525285-%2525281-methyl-1H-pyrazol-4-yl%252529-4-%252528methylamino%252529pyrimidin-2-yl%252529amino%252529-3-%252528piperidin-3-yloxy%252529picolinonitrile%25252C%252Ba%252Bnovel%252BCHK1%252Binhibitor%252Bfor%252Bhematologic%252Bmalignancies%26aulast%3DTong%26aufirst%3DLexian%26date%3D2019%26volume%3D173%26spage%3D44%26epage%3D62" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Wenxiu  Qi</span>, <span class="hlFld-ContribAuthor ">Xiaohao  Xu</span>, <span class="hlFld-ContribAuthor ">Manying  Wang</span>, <span class="hlFld-ContribAuthor ">Xiangyan  Li</span>, <span class="hlFld-ContribAuthor ">Chaonan  Wang</span>, <span class="hlFld-ContribAuthor ">Liping  Sun</span>, <span class="hlFld-ContribAuthor ">Daqing  Zhao</span>, <span class="hlFld-ContribAuthor ">Liwei  Sun</span>. </span><span class="cited-content_cbyCitation_article-title">Inhibition of Wee1 sensitizes AML cells to ATR inhibitor VE-822-induced DNA damage and apoptosis. </span><span class="cited-content_cbyCitation_journal-name">Biochemical Pharmacology</span><span> <strong>2019,</strong> <em>164 </em>, 273-282. <a href="https://doi.org/10.1016/j.bcp.2019.04.022" title="DOI URL">https://doi.org/10.1016/j.bcp.2019.04.022</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bcp.2019.04.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bcp.2019.04.022%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemical%2520Pharmacology%26atitle%3DInhibition%252Bof%252BWee1%252Bsensitizes%252BAML%252Bcells%252Bto%252BATR%252Binhibitor%252BVE-822-induced%252BDNA%252Bdamage%252Band%252Bapoptosis%26aulast%3DQi%26aufirst%3DWenxiu%26date%3D2019%26volume%3D164%26spage%3D273%26epage%3D282" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Reem Fawaz  Abutayeh</span>, <span class="hlFld-ContribAuthor ">Mutasem O.  Taha</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of novel Flt3 inhibitory chemotypes through extensive ligand-based and new structure-based pharmacophore modelling methods. </span><span class="cited-content_cbyCitation_journal-name">Journal of Molecular Graphics and Modelling</span><span> <strong>2019,</strong> <em>88 </em>, 128-151. <a href="https://doi.org/10.1016/j.jmgm.2019.01.011" title="DOI URL">https://doi.org/10.1016/j.jmgm.2019.01.011</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jmgm.2019.01.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jmgm.2019.01.011%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Molecular%2520Graphics%2520and%2520Modelling%26atitle%3DDiscovery%252Bof%252Bnovel%252BFlt3%252Binhibitory%252Bchemotypes%252Bthrough%252Bextensive%252Bligand-based%252Band%252Bnew%252Bstructure-based%252Bpharmacophore%252Bmodelling%252Bmethods%26aulast%3DAbutayeh%26aufirst%3DReem%2BFawaz%26date%3D2019%26volume%3D88%26spage%3D128%26epage%3D151" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Weiyan  Cheng</span>, <span class="hlFld-ContribAuthor ">Zhiheng  Yang</span>, <span class="hlFld-ContribAuthor ">Suhua  Wang</span>, <span class="hlFld-ContribAuthor ">Ying  Li</span>, <span class="hlFld-ContribAuthor ">Han  Wei</span>, <span class="hlFld-ContribAuthor ">Xin  Tian</span>, <span class="hlFld-ContribAuthor ">Quancheng  Kan</span>. </span><span class="cited-content_cbyCitation_article-title">Recent development of CDK inhibitors: An overview of CDK/inhibitor co-crystal structures. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>164 </em>, 615-639. <a href="https://doi.org/10.1016/j.ejmech.2019.01.003" title="DOI URL">https://doi.org/10.1016/j.ejmech.2019.01.003</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2019.01.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2019.01.003%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DRecent%252Bdevelopment%252Bof%252BCDK%252Binhibitors%25253A%252BAn%252Boverview%252Bof%252BCDK%25252Finhibitor%252Bco-crystal%252Bstructures%26aulast%3DCheng%26aufirst%3DWeiyan%26date%3D2019%26volume%3D164%26spage%3D615%26epage%3D639" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shuting  Li</span>, <span class="hlFld-ContribAuthor ">Wanfeng  Yang</span>, <span class="hlFld-ContribAuthor ">Min  Ji</span>, <span class="hlFld-ContribAuthor ">Jin  Cai</span>, <span class="hlFld-ContribAuthor ">Junqing  Chen</span>. </span><span class="cited-content_cbyCitation_article-title">A new and efficient protocol for the synthesis of the key intermediate of Palbociclib. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Research</span><span> <strong>2019,</strong> <em>43 </em>
                                    (1-2)
                                     , 14-19. <a href="https://doi.org/10.1177/1747519819831858" title="DOI URL">https://doi.org/10.1177/1747519819831858</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1177/1747519819831858&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1177%2F1747519819831858%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Research%26atitle%3DA%252Bnew%252Band%252Befficient%252Bprotocol%252Bfor%252Bthe%252Bsynthesis%252Bof%252Bthe%252Bkey%252Bintermediate%252Bof%252BPalbociclib%26aulast%3DLi%26aufirst%3DShuting%26date%3D2019%26date%3D2019%26volume%3D43%26issue%3D1-2%26spage%3D14%26epage%3D19" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mei  Wu</span>, <span class="hlFld-ContribAuthor ">Chuntuan  Li</span>, <span class="hlFld-ContribAuthor ">Xiongpeng  Zhu</span>. </span><span class="cited-content_cbyCitation_article-title">FLT3 inhibitors in acute myeloid leukemia. </span><span class="cited-content_cbyCitation_journal-name">Journal of Hematology & Oncology</span><span> <strong>2018,</strong> <em>11 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13045-018-0675-4" title="DOI URL">https://doi.org/10.1186/s13045-018-0675-4</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13045-018-0675-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13045-018-0675-4%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Hematology%2520%2526%2520Oncology%26atitle%3DFLT3%252Binhibitors%252Bin%252Bacute%252Bmyeloid%252Bleukemia%26aulast%3DWu%26aufirst%3DMei%26date%3D2018%26date%3D2018%26volume%3D11%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">V.  Divya</span>, <span class="hlFld-ContribAuthor ">V.L.  Pushpa</span>, <span class="hlFld-ContribAuthor ">S.  Sarithamol</span>, <span class="hlFld-ContribAuthor ">K.B.  Manoj</span>. </span><span class="cited-content_cbyCitation_article-title">Computational approach for generating robust models for discovering novel molecules as Cyclin Dependent Kinase 4 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Molecular Graphics and Modelling</span><span> <strong>2018,</strong> <em>82 </em>, 48-58. <a href="https://doi.org/10.1016/j.jmgm.2018.04.001" title="DOI URL">https://doi.org/10.1016/j.jmgm.2018.04.001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jmgm.2018.04.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jmgm.2018.04.001%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Molecular%2520Graphics%2520and%2520Modelling%26atitle%3DComputational%252Bapproach%252Bfor%252Bgenerating%252Brobust%252Bmodels%252Bfor%252Bdiscovering%252Bnovel%252Bmolecules%252Bas%252BCyclin%252BDependent%252BKinase%252B4%252Binhibitors%26aulast%3DDivya%26aufirst%3DV.%26date%3D2018%26volume%3D82%26spage%3D48%26epage%3D58" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jingxiao  Zhang</span>, <span class="hlFld-ContribAuthor ">Lilei  Zhang</span>, <span class="hlFld-ContribAuthor ">Yangcheng  Xu</span>, <span class="hlFld-ContribAuthor ">Shanshan  Jiang</span>, <span class="hlFld-ContribAuthor ">Yueyue  Shao</span>, . </span><span class="cited-content_cbyCitation_article-title">Deciphering the binding behavior of flavonoids to the cyclin dependent kinase 6/cyclin D complex. </span><span class="cited-content_cbyCitation_journal-name">PLOS ONE</span><span> <strong>2018,</strong> <em>13 </em>
                                    (5)
                                     , e0196651. <a href="https://doi.org/10.1371/journal.pone.0196651" title="DOI URL">https://doi.org/10.1371/journal.pone.0196651</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1371/journal.pone.0196651&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1371%2Fjournal.pone.0196651%26sid%3Dliteratum%253Aachs%26jtitle%3DPLOS%2520ONE%26atitle%3DDeciphering%252Bthe%252Bbinding%252Bbehavior%252Bof%252Bflavonoids%252Bto%252Bthe%252Bcyclin%252Bdependent%252Bkinase%252B6%25252Fcyclin%252BD%252Bcomplex%26aulast%3DZhang%26aufirst%3DJingxiao%26date%3D2018%26date%3D2018%26volume%3D13%26issue%3D5%26spage%3De0196651" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chuantao  Zha</span>, <span class="hlFld-ContribAuthor ">Wenjia  Deng</span>, <span class="hlFld-ContribAuthor ">Yan  Fu</span>, <span class="hlFld-ContribAuthor ">Shuai  Tang</span>, <span class="hlFld-ContribAuthor ">Xiaojing  Lan</span>, <span class="hlFld-ContribAuthor ">Yan  Ye</span>, <span class="hlFld-ContribAuthor ">Yi  Su</span>, <span class="hlFld-ContribAuthor ">Lei  Jiang</span>, <span class="hlFld-ContribAuthor ">Yi  Chen</span>, <span class="hlFld-ContribAuthor ">Ying  Huang</span>, <span class="hlFld-ContribAuthor ">Jian  Ding</span>, <span class="hlFld-ContribAuthor ">Meiyu  Geng</span>, <span class="hlFld-ContribAuthor ">Min  Huang</span>, <span class="hlFld-ContribAuthor ">Huixin  Wan</span>. </span><span class="cited-content_cbyCitation_article-title">Design, synthesis and biological evaluation of tetrahydronaphthyridine derivatives as bioavailable CDK4/6 inhibitors for cancer therapy. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>148 </em>, 140-153. <a href="https://doi.org/10.1016/j.ejmech.2018.02.022" title="DOI URL">https://doi.org/10.1016/j.ejmech.2018.02.022</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2018.02.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2018.02.022%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDesign%25252C%252Bsynthesis%252Band%252Bbiological%252Bevaluation%252Bof%252Btetrahydronaphthyridine%252Bderivatives%252Bas%252Bbioavailable%252BCDK4%25252F6%252Binhibitors%252Bfor%252Bcancer%252Btherapy%26aulast%3DZha%26aufirst%3DChuantao%26date%3D2018%26volume%3D148%26spage%3D140%26epage%3D153" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">, , <span class="hlFld-ContribAuthor ">Mathew P.  Martin</span>, <span class="hlFld-ContribAuthor ">Jane A.  Endicott</span>, <span class="hlFld-ContribAuthor ">Martin E.M.  Noble</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-based discovery of cyclin-dependent protein kinase inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Essays in Biochemistry</span><span> <strong>2017,</strong> <em>61 </em>
                                    (5)
                                     , 439-452. <a href="https://doi.org/10.1042/EBC20170040" title="DOI URL">https://doi.org/10.1042/EBC20170040</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1042/EBC20170040&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1042%2FEBC20170040%26sid%3Dliteratum%253Aachs%26jtitle%3DEssays%2520in%2520Biochemistry%26atitle%3DStructure-based%252Bdiscovery%252Bof%252Bcyclin-dependent%252Bprotein%252Bkinase%252Binhibitors%26aulast%3Dvan%2BMontfort%26aufirst%3DRob%2BL.M.%26date%3D2017%26date%3D2017%26volume%3D61%26issue%3D5%26spage%3D439%26epage%3D452" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Elizabeth  Larocque</span>, <span class="hlFld-ContribAuthor ">N  Naganna</span>, <span class="hlFld-ContribAuthor ">Xiaochu  Ma</span>, <span class="hlFld-ContribAuthor ">Clement  Opoku-Temeng</span>, <span class="hlFld-ContribAuthor ">Brandon  Carter-Cooper</span>, <span class="hlFld-ContribAuthor ">Gaurav  Chopra</span>, <span class="hlFld-ContribAuthor ">Rena G  Lapidus</span>, <span class="hlFld-ContribAuthor ">Herman O  Sintim</span>. </span><span class="cited-content_cbyCitation_article-title">Aminoisoquinoline benzamides, FLT3 and Src-family kinase inhibitors, potently inhibit proliferation of acute myeloid leukemia cell lines. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2017,</strong> <em>9 </em>
                                    (11)
                                     , 1213-1225. <a href="https://doi.org/10.4155/fmc-2017-0067" title="DOI URL">https://doi.org/10.4155/fmc-2017-0067</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc-2017-0067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc-2017-0067%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DAminoisoquinoline%252Bbenzamides%25252C%252BFLT3%252Band%252BSrc-family%252Bkinase%252Binhibitors%25252C%252Bpotently%252Binhibit%252Bproliferation%252Bof%252Bacute%252Bmyeloid%252Bleukemia%252Bcell%252Blines%26aulast%3DLarocque%26aufirst%3DElizabeth%26date%3D2017%26volume%3D9%26issue%3D11%26spage%3D1213%26epage%3D1225" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Stella  Hernandez Maganhi</span>, <span class="hlFld-ContribAuthor ">Patrizia  Jensen</span>, <span class="hlFld-ContribAuthor ">Ignez  Caracelli</span>, <span class="hlFld-ContribAuthor ">Julio  Zukerman Schpector</span>, <span class="hlFld-ContribAuthor ">Stefan  Fröhling</span>, <span class="hlFld-ContribAuthor ">Ran  Friedman</span>. </span><span class="cited-content_cbyCitation_article-title">Palbociclib can overcome mutations in cyclin dependent kinase 6 that break hydrogen bonds between the drug and the protein. </span><span class="cited-content_cbyCitation_journal-name">Protein Science</span><span> <strong>2017,</strong> <em>26 </em>
                                    (4)
                                     , 870-879. <a href="https://doi.org/10.1002/pro.3135" title="DOI URL">https://doi.org/10.1002/pro.3135</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/pro.3135&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fpro.3135%26sid%3Dliteratum%253Aachs%26jtitle%3DProtein%2520Science%26atitle%3DPalbociclib%252Bcan%252Bovercome%252Bmutations%252Bin%252Bcyclin%252Bdependent%252Bkinase%252B6%252Bthat%252Bbreak%252Bhydrogen%252Bbonds%252Bbetween%252Bthe%252Bdrug%252Band%252Bthe%252Bprotein%26aulast%3DHernandez%2BMaganhi%26aufirst%3DStella%26date%3D2017%26date%3D2017%26volume%3D26%26issue%3D4%26spage%3D870%26epage%3D879" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tinghan  Li</span>, <span class="hlFld-ContribAuthor ">Tianwei  Weng</span>, <span class="hlFld-ContribAuthor ">Minzan  Zuo</span>, <span class="hlFld-ContribAuthor ">Zhihui  Wei</span>, <span class="hlFld-ContribAuthor ">Ming  Chen</span>, <span class="hlFld-ContribAuthor ">Zhiyu  Li</span>. </span><span class="cited-content_cbyCitation_article-title">Recent progress of cyclin-dependent kinase inhibitors as potential anticancer agents. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2016,</strong> <em>8 </em>
                                    (17)
                                     , 2047-2076. <a href="https://doi.org/10.4155/fmc-2016-0129" title="DOI URL">https://doi.org/10.4155/fmc-2016-0129</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc-2016-0129&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc-2016-0129%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DRecent%252Bprogress%252Bof%252Bcyclin-dependent%252Bkinase%252Binhibitors%252Bas%252Bpotential%252Banticancer%252Bagents%26aulast%3DLi%26aufirst%3DTinghan%26date%3D2016%26volume%3D8%26issue%3D17%26spage%3D2047%26epage%3D2076" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Carmen  Di Giovanni</span>, <span class="hlFld-ContribAuthor ">Ettore  Novellino</span>, <span class="hlFld-ContribAuthor ">Adriana  Chilin</span>, <span class="hlFld-ContribAuthor ">Antonio  Lavecchia</span>, <span class="hlFld-ContribAuthor ">Giovanni  Marzaro</span>. </span><span class="cited-content_cbyCitation_article-title">Investigational drugs targeting cyclin-dependent kinases for the treatment of cancer: an update on recent findings (2013-2016). </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Investigational Drugs</span><span> <strong>2016,</strong> <em>25 </em>
                                    (10)
                                     , 1215-1230. <a href="https://doi.org/10.1080/13543784.2016.1234603" title="DOI URL">https://doi.org/10.1080/13543784.2016.1234603</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/13543784.2016.1234603&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F13543784.2016.1234603%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Investigational%2520Drugs%26atitle%3DInvestigational%252Bdrugs%252Btargeting%252Bcyclin-dependent%252Bkinases%252Bfor%252Bthe%252Btreatment%252Bof%252Bcancer%25253A%252Ban%252Bupdate%252Bon%252Brecent%252Bfindings%252B%2525282013-2016%252529%26aulast%3DDi%2BGiovanni%26aufirst%3DCarmen%26date%3D2016%26date%3D2016%26volume%3D25%26issue%3D10%26spage%3D1215%26epage%3D1230" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Y  Chen</span>, <span class="hlFld-ContribAuthor ">Y  Guo</span>, <span class="hlFld-ContribAuthor ">W  Zhao</span>, <span class="hlFld-ContribAuthor ">W-T  Tina Ho</span>, <span class="hlFld-ContribAuthor ">X  Fu</span>, <span class="hlFld-ContribAuthor ">Z J  Zhao</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of an orally available compound with potent and broad FLT3 inhibition activity. </span><span class="cited-content_cbyCitation_journal-name">Oncogene</span><span> <strong>2016,</strong> <em>35 </em>
                                    (23)
                                     , 2971-2978. <a href="https://doi.org/10.1038/onc.2015.362" title="DOI URL">https://doi.org/10.1038/onc.2015.362</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/onc.2015.362&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fonc.2015.362%26sid%3Dliteratum%253Aachs%26jtitle%3DOncogene%26atitle%3DIdentification%252Bof%252Ban%252Borally%252Bavailable%252Bcompound%252Bwith%252Bpotent%252Band%252Bbroad%252BFLT3%252Binhibition%252Bactivity%26aulast%3DChen%26aufirst%3DY%26date%3D2016%26date%3D2015%26volume%3D35%26issue%3D23%26spage%3D2971%26epage%3D2978" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ling  Wang</span>, <span class="hlFld-ContribAuthor ">Yecheng  Li</span>, <span class="hlFld-ContribAuthor ">Mengyan  Xu</span>, <span class="hlFld-ContribAuthor ">Xiaoqian  Pang</span>, <span class="hlFld-ContribAuthor ">Zhihong  Liu</span>, <span class="hlFld-ContribAuthor ">Wen  Tan</span>, <span class="hlFld-ContribAuthor ">Jun  Xu</span>. </span><span class="cited-content_cbyCitation_article-title">Chemical fragment-based CDK4/6 inhibitors prediction and web server. </span><span class="cited-content_cbyCitation_journal-name">RSC Advances</span><span> <strong>2016,</strong> <em>6 </em>
                                    (21)
                                     , 16972-16981. <a href="https://doi.org/10.1039/C5RA23289A" title="DOI URL">https://doi.org/10.1039/C5RA23289A</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C5RA23289A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC5RA23289A%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Advances%26atitle%3DChemical%252Bfragment-based%252BCDK4%25252F6%252Binhibitors%252Bprediction%252Band%252Bweb%252Bserver%26aulast%3DWang%26aufirst%3DLing%26date%3D2016%26date%3D2016%26volume%3D6%26issue%3D21%26spage%3D16972%26epage%3D16981" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Solomon  Tadesse</span>, <span class="hlFld-ContribAuthor ">Mingfeng  Yu</span>, <span class="hlFld-ContribAuthor ">Malika  Kumarasiri</span>, <span class="hlFld-ContribAuthor ">Bich Thuy  Le</span>, <span class="hlFld-ContribAuthor ">Shudong  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting CDK6 in cancer: State of the art and new insights. </span><span class="cited-content_cbyCitation_journal-name">Cell Cycle</span><span> <strong>2015,</strong> <em>14 </em>
                                    (20)
                                     , 3220-3230. <a href="https://doi.org/10.1080/15384101.2015.1084445" title="DOI URL">https://doi.org/10.1080/15384101.2015.1084445</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/15384101.2015.1084445&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F15384101.2015.1084445%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Cycle%26atitle%3DTargeting%252BCDK6%252Bin%252Bcancer%25253A%252BState%252Bof%252Bthe%252Bart%252Band%252Bnew%252Binsights%26aulast%3DTadesse%26aufirst%3DSolomon%26date%3D2015%26date%3D2015%26volume%3D14%26issue%3D20%26spage%3D3220%26epage%3D3230" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Concepción  Sánchez-Martínez</span>, <span class="hlFld-ContribAuthor ">Lawrence M.  Gelbert</span>, <span class="hlFld-ContribAuthor ">María José  Lallena</span>, <span class="hlFld-ContribAuthor ">Alfonso  de Dios</span>. </span><span class="cited-content_cbyCitation_article-title">Cyclin dependent kinase (CDK) inhibitors as anticancer drugs. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2015,</strong> <em>25 </em>
                                    (17)
                                     , 3420-3435. <a href="https://doi.org/10.1016/j.bmcl.2015.05.100" title="DOI URL">https://doi.org/10.1016/j.bmcl.2015.05.100</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2015.05.100&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2015.05.100%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DCyclin%252Bdependent%252Bkinase%252B%252528CDK%252529%252Binhibitors%252Bas%252Banticancer%252Bdrugs%26aulast%3DS%25C3%25A1nchez-Mart%25C3%25ADnez%26aufirst%3DConcepci%25C3%25B3n%26date%3D2015%26volume%3D25%26issue%3D17%26spage%3D3420%26epage%3D3435" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Cong  Li</span>, <span class="hlFld-ContribAuthor ">Liqin  Liu</span>, <span class="hlFld-ContribAuthor ">Lingming  Liang</span>, <span class="hlFld-ContribAuthor ">Zhen  Xia</span>, <span class="hlFld-ContribAuthor ">Zhihong  Li</span>, <span class="hlFld-ContribAuthor ">Xianghong  Wang</span>, <span class="hlFld-ContribAuthor ">Lawrence R.  McGee</span>, <span class="hlFld-ContribAuthor ">Katie  Newhall</span>, <span class="hlFld-ContribAuthor ">Angus  Sinclair</span>, <span class="hlFld-ContribAuthor ">Alexander  Kamb</span>, <span class="hlFld-ContribAuthor ">Dineli  Wickramasinghe</span>, <span class="hlFld-ContribAuthor ">Kang  Dai</span>. </span><span class="cited-content_cbyCitation_article-title">AMG 925 Is a Dual FLT3/CDK4 Inhibitor with the Potential to Overcome FLT3 Inhibitor Resistance in Acute Myeloid Leukemia. </span><span class="cited-content_cbyCitation_journal-name">Molecular Cancer Therapeutics</span><span> <strong>2015,</strong> <em>14 </em>
                                    (2)
                                     , 375-383. <a href="https://doi.org/10.1158/1535-7163.MCT-14-0388" title="DOI URL">https://doi.org/10.1158/1535-7163.MCT-14-0388</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1158/1535-7163.MCT-14-0388&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1158%2F1535-7163.MCT-14-0388%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Cancer%2520Therapeutics%26atitle%3DAMG%252B925%252BIs%252Ba%252BDual%252BFLT3%25252FCDK4%252BInhibitor%252Bwith%252Bthe%252BPotential%252Bto%252BOvercome%252BFLT3%252BInhibitor%252BResistance%252Bin%252BAcute%252BMyeloid%252BLeukemia%26aulast%3DLi%26aufirst%3DCong%26date%3D2015%26date%3D2014%26volume%3D14%26issue%3D2%26spage%3D375%26epage%3D383" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/medium/jm-2014-00118j_0028.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0028.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500118j&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/medium/jm-2014-00118j_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Lead compound <b>1</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500118j&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/medium/jm-2014-00118j_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Binding of compound <b>1</b> in CDK6 and FLT3. (a) Co-crystal structure of compound <b>1</b> in CDK6 at 2.9 Å resolution (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4P41">4P41</a>). (b) Compound <b>1</b> with homology model of FLT3.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> The key active site residues are highlighted in stick representation.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500118j&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/medium/jm-2014-00118j_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. CDK6/compound <b>1</b> co-crystal structure highlighting ATP binding site with P-loop surface removed for clarity (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4P41">4P41</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500118j&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/medium/jm-2014-00118j_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. STAT5 and Rb phosphorylation in MOLM13 tumors treated with <b>19</b>. MOLM13 tumor bearing mice were dosed by oral administration of <b>19</b> at 37.5, 75, or 150 mg/kg, and tumors were harvested at 3 and 24 h after treatment. Tumors lysates were prepared and the level of pSTAT5 (at 3 h post dose, (a)) or pRb (at 24 h post dose, (b)) was determined. Unbound plasma levels of <b>19</b> (μM) for the same time points are shown. Data are presented as mean ± SEM, <i>n</i> = 3 except for vehicle group (<i>n</i> = 6).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500118j&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/medium/jm-2014-00118j_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (a) Experimentally measured p<i>K</i><sub>a</sub> of compound <b>19</b>; (b) R<sub>4</sub>N<sup>+</sup> and Cl<sup>–</sup> ion pair may be stabilized by hydrogen bonds.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500118j&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/medium/jm-2014-00118j_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Drug exposure levels vs molecular basicity (C<sub>3h</sub>: total drug concentration at 3 h post 150 mg/kg oral dose in mouse PK studies).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500118j&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/medium/jm-2014-00118j_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Computational models of binding modes and activation loop conformations of inactive DFG-out and active DFG-in activation loop conformations.<a onclick="showRef(event, 'ref14 ref23'); return false;" href="javascript:void(0);" class="ref ref14 ref23">(14, 23)</a> (a) DFG-out kinase with the activiation loop (green) in the inactive conformation stabilized by intramolecular hydrogen bonds. (b) DFG-in kinase with the activation loop (green) in the active DFG-in conformation. (c) DFG-out kinase with sorafenib model showing type 2 binding mode. (d) DFG-in kinase with <b>28</b> model showing type I binding mode.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500118j&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/medium/jm-2014-00118j_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0008.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. In vivo tumor growth inhibition for <b>28</b>. NCR nude mice bearing established human MOLM13 AML xenografts were dosed orally with either vehicle or <b>28</b> at 25, 50, 75, and 150 mg/kg (day 14 TGI = 100%, <i>p</i> < 0.0001) once daily from day 5 to day 13.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500118j&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/medium/jm-2014-00118j_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0009.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Compound <b>28</b> inhibited STAT5 (a) and Rb (b) phosphorylation in MOLM13 subcutaneous tumors. MOLM13 tumor bearing mice were dosed by oral administration of <b>28</b> at 50, 75, or 150 mg/kg, and tumors were harvested at 3, 6, and 24 h after treatment. Tumors lysates were prepared, and the level of pSTAT5 (upper panel) or pRb (lower panel) was determined.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> Unbound plasma levels of <b>28</b> (μM) same time points are shown. Data are presented as mean ± SEM, <i>n</i> = 3 except for vehicle group (<i>n</i> = 6).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500118j&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig10" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/medium/jm-2014-00118j_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0010.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Numbering convention used for the pyrido[4′,3′:4,5]pyrrolo[2,3-<i>d</i>]pyrimidine core.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500118j&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/medium/jm-2014-00118j_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0011.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. First-Generation Synthetic Approach to the Pyrido[4′,3′:4,5]pyrrolo[2,3-<i>d</i>]pyrimidine Dual Inhibitors<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500118j&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) RNH<sub>2</sub>, DIPEA, reflux; (b) NH<sub>4</sub>OH, <i>i</i>-PrOH, 120 °C; (c) PdCl<sub>2</sub>(PPh<sub>3</sub>)<sub>2</sub>, Na<sub>2</sub>CO<sub>3</sub>, 1,4-dioxane, 120–150 °C; (d) Pd<sub>2</sub>dba<sub>3</sub>, XantPhos, <i>t</i>-BuONa, 1,4-dioxane, 150 °C; (e) Pd<sub>2</sub>dba<sub>3</sub>, Xantphos, <i>t</i>-BuONa, 1,4-dioxane, 120 °C; (f) TFA/CH<sub>2</sub>Cl<sub>2</sub>.</p></p></figure><figure data-id="sch2" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/medium/jm-2014-00118j_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0012.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of <i>ortho</i>-Substituted Pyridine Analogues via Pyridone <b>38</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500118j&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) <b>31a</b>, PdCl<sub>2</sub>(PPh<sub>3</sub>)<sub>2</sub>, Na<sub>2</sub>CO<sub>3</sub>, 1,4-dioxane, 120 °C; (b) Pd<sub>2</sub>dba<sub>3</sub>, XantPhos, <i>t</i>-BuONa, 1,4-dioxane, 150 °C; (c) HCl, 1,4-dioxane/MeOH, 80 °C; (d) POCl<sub>3</sub>, 100 °C; (e) Pd<sub>2</sub>dba<sub>3</sub>, XantPhos, <i>t</i>-BuONa, 1,4-dioxane, 80 °C; (f) TFA/CH<sub>2</sub>Cl<sub>2</sub>; (g) Tf<sub>2</sub>NPh, NaH, DMF; (h) Fe(acac)<sub>3</sub>, MeMgBr, THF/NMP; (i) <b>34a</b>, Pd<sub>2</sub>dba<sub>3</sub>, XantPhos, <i>t</i>-BuONa, 1,4-dioxane, 120 °C; (j) TFA/CH<sub>2</sub>Cl<sub>2</sub>; (k) <b>34a</b>, Pd<sub>2</sub>dba<sub>3</sub>, XantPhos, <i>t</i>-BuONa, 1,4-dioxane, 120 °C; (l) TFA/CH<sub>2</sub>Cl<sub>2</sub>.</p></p></figure><figure data-id="sch3" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/medium/jm-2014-00118j_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0013.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Negishi Coupling Approach to Substituted Fused-Pyridine Derivatives <b>2</b> and <b>6</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500118j&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagent and conditions: (a) cyclopentylamine or <i>trans</i>-Me-cyclohexylamine, 1,4-dioxane or BuOH, 100 °C; (b) NIS, DMF or AcOH, 80 °C; (c) <i>i</i>-PrMgCl, ZnCl<sub>2</sub>; (d) <b>44a</b>, Pd(PPh<sub>3</sub>)<sub>4</sub>, THF, reflux; (e) X-phos, Cs<sub>2</sub>CO<sub>3</sub>, 1,4-dioxane, 120 °C; (f) <b>34a</b>, Pd<sub>2</sub>dba<sub>3</sub>, XantPhos, <i>t</i>-BuONa, 1,4-dioxane, 120 °C; (g) TFA/CH<sub>2</sub>Cl<sub>2</sub>; (h) NEt<sub>4</sub>CN, DMF, 150 °C; (i) <b>34a</b>, Pd<sub>2</sub>dba<sub>3</sub>, XantPhos, <i>t</i>-BuONa, 1,4-dioxane, 120 °C; (j) TFA/CH<sub>2</sub>Cl<sub>2</sub>.</p></p></figure><figure data-id="sch4" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/medium/jm-2014-00118j_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0014.jpeg" id="rightTab-gr14" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Negishi Route to Fused Pyridazine Derivative <b>7</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500118j&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) LiTMP, ZnCl<sub>2</sub>; (b) <b>44a</b>, Pd(PPh<sub>3</sub>)<sub>4</sub>, THF, reflux; (c) NaH, THF, 150 °C; (d) PPTS, 180 °C; (e) PhNTf<sub>2</sub>, TEA, DMAP; (f) Pd(OAc)<sub>2</sub>, dppf, TEA, HCO<sub>2</sub>H, DMF; (g) <b>34a</b>, Pd<sub>2</sub>dba<sub>3</sub>, XantPhos, <i>t</i>-BuONa, 1,4-dioxane, 120 °C; (h) TFA/CH<sub>2</sub>Cl<sub>2</sub>.</p></p></figure><figure data-id="sch5" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/medium/jm-2014-00118j_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0015.jpeg" id="rightTab-gr15" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Process Synthesis of Optimized Tricyclic Core <b>33d</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500118j&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) LiTMP, ZnBr<sub>2</sub>, −78 °C; (b) <b>44b</b>, Pd(PPh<sub>3</sub>)<sub>4</sub>, THF, reflux; (c) LiHMDS, NMP, 90 °C.</p></p></figure><figure data-id="sch6" data-index="16" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/medium/jm-2014-00118j_0016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0016.jpeg" id="rightTab-gr16" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 6. Synthesis and Coupling of Haloheteroarene Amination Substrates<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500118j&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) Pd<sub>2</sub>dba<sub>3</sub>, XantPhos, <i>t</i>-BuONa or Cs<sub>2</sub>CO<sub>3</sub>, 1,4-dioxane or PhMe, 90–120 °C (for <b>58a</b>, <b>58c</b>–<b>d</b>), DIPEA, 1,4-dioxane, 80 °C (for <b>58b</b>,<b>e</b>); (b) <b>33d</b>, Pd<sub>2</sub>dba<sub>3</sub>, XantPhos, <i>t</i>-BuONa or Cs<sub>2</sub>CO<sub>3</sub>, 1,4-dioxane or PhMe, 100–120 °C; (c) TFA/CH<sub>2</sub>Cl<sub>2</sub>.</p></p></figure><figure data-id="sch7" data-index="17" class="article__inlineFigure"><h2 class="fig-label">Scheme 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/medium/jm-2014-00118j_0017.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0017.jpeg" id="rightTab-gr17" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 7. Modular Approach to 4-Aminopiperidine Derivatives <b>17</b> and <b>22</b> via Late-Stage Reductive Amination<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500118j&amp;id=sch7"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) <b>33d</b>, Pd<sub>2</sub>dba<sub>3</sub>, XantPhos, <i>t</i>-BuONa, 1,4-dioxane; (b) 6N HCl (aq); (c) NaBH<sub>3</sub>(CN), NH<sub>4</sub>OAc, MeOH, 60 °C; (d) NaBH(OAc)<sub>3</sub>, H<sub>2</sub>CO (aq), CH<sub>2</sub>Cl<sub>2</sub>.</p></p></figure><figure data-id="sch8" data-index="18" class="article__inlineFigure"><h2 class="fig-label">Scheme 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/medium/jm-2014-00118j_0018.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0018.jpeg" id="rightTab-gr18" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 8. Synthesis of the Precursor to Aliphatic Sulfone <b>24</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500118j&amp;id=sch8"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) <b>56b</b>, 90 °C; (b) MsCl, TEA; (c) NaSMe, DMF; (d) oxone, MeOH; (e) <b>33d</b>, Pd<sub>2</sub>dba<sub>3</sub>, XantPhos, <i>t</i>-BuONa, 1,4-dioxane, 120 °C.</p></p></figure><figure data-id="sch9" data-index="19" class="article__inlineFigure"><h2 class="fig-label">Scheme 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/medium/jm-2014-00118j_0019.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0019.jpeg" id="rightTab-gr19" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 9. Synthesis of the Precursor to Spirocyclic Sulfone and Spirocyclic Lactams<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0019.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500118j&amp;id=sch9"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) <b>56b</b>, DIPEA, 1,4-dioxane, 90 °C; (b) <b>33d</b>, Pd<sub>2</sub>dba<sub>3</sub>, XantPhos, <i>t</i>-BuONa, 1,4-dioxane, 150 °C; (c) HCl, 1,4-dioxane; (d) <b>56a</b> or <b>56d</b>, XantPhos, Pd<sub>2</sub>dba<sub>3</sub>, <i>t</i>-BuONa, PhMe; (e) <b>33d</b>, Pd<sub>2</sub>dba<sub>3</sub>, Xantphos, <i>t</i>-BuONa, 1,4-dioxane, 150 °C.</p></p></figure><figure data-id="sch10" data-index="20" class="article__inlineFigure"><h2 class="fig-label">Scheme 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/medium/jm-2014-00118j_0020.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0020.jpeg" id="rightTab-gr20" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 10. Synthesis of <b>28</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-8/jm500118j/production/images/large/jm-2014-00118j_0020.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500118j&amp;id=sch10"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) (Boc)<sub>2</sub>O, DIPEA, DCM; (b) <b>33d</b>, Pd<sub>2</sub>dba<sub>3</sub>, XantPhos, <i>t</i>-BuONa, 1,4-dioxane, 100 °C; (c) HCl, MeOH/H<sub>2</sub>O; NaOH; (d) DIPEA, CHCl<sub>3</sub>; (e) MeONa, DCM/MeOH.</p></p></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i104">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_53214" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_53214" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 33 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span>Cancer Facts & Figures 2013; <span class="NLM_publisher-name">American Cancer Society</span>: <span class="NLM_publisher-loc">Atlanta</span>,<span class="NLM_x"> </span><span class="NLM_year">2013</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Cancer+Facts+%26+Figures+2013%3B+American+Cancer+Society%3A+Atlanta%2C+2013."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dreport%26sid%3Dliteratum%253Aachs%26jtitle%3DCancer%2520Facts%2520%2526%2520Figures%25202013%26pub%3DAmerican%2520Cancer%2520Society%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Birg, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Courcoul, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosnet, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bardin, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pebusque, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marchetto, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tabilio, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mannoni, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Birnbaum, D.</span><span> </span><span class="NLM_article-title">Expression of the FMS/KIT-like gene FLT3 in human acute leukemia of the myeloid and lymphoid lineages</span> <span class="citation_source-journal">Blood</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1992</span><span class="NLM_x">, </span> <span class="NLM_volume">80</span><span class="NLM_x">, </span> <span class="NLM_fpage">2584</span><span class="NLM_x">–</span> <span class="NLM_lpage">2593</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=80&publication_year=1992&pages=2584-2593&author=F.+Birgauthor=M.+Courcoulauthor=O.+Rosnetauthor=F.+Bardinauthor=M.+J.+Pebusqueauthor=S.+Marchettoauthor=A.+Tabilioauthor=P.+Mannoniauthor=D.+Birnbaum&title=Expression+of+the+FMS%2FKIT-like+gene+FLT3+in+human+acute+leukemia+of+the+myeloid+and+lymphoid+lineages"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBirg%26aufirst%3DF.%26aulast%3DCourcoul%26aufirst%3DM.%26aulast%3DRosnet%26aufirst%3DO.%26aulast%3DBardin%26aufirst%3DF.%26aulast%3DPebusque%26aufirst%3DM.%2BJ.%26aulast%3DMarchetto%26aufirst%3DS.%26aulast%3DTabilio%26aufirst%3DA.%26aulast%3DMannoni%26aufirst%3DP.%26aulast%3DBirnbaum%26aufirst%3DD.%26atitle%3DExpression%2520of%2520the%2520FMS%252FKIT-like%2520gene%2520FLT3%2520in%2520human%2520acute%2520leukemia%2520of%2520the%2520myeloid%2520and%2520lymphoid%2520lineages%26jtitle%3DBlood%26date%3D1992%26volume%3D80%26spage%3D2584%26epage%3D2593" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Carow, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levenstein, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaufmann, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Amin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rockwell, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Witte, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Borowitz, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Civin, C. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Small, D.</span><span> </span><span class="NLM_article-title">Expression of the hematopoietic growth factor receptor FLT3 (STK-1/Flk2) in human leukemia</span> <span class="citation_source-journal">Blood</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">87</span><span class="NLM_x">, </span> <span class="NLM_fpage">1089</span><span class="NLM_x">–</span> <span class="NLM_lpage">1096</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=87&publication_year=1996&pages=1089-1096&author=C.+E.+Carowauthor=M.+Levensteinauthor=S.+H.+Kaufmannauthor=J.+Chenauthor=S.+Aminauthor=P.+Rockwellauthor=L.+Witteauthor=M.+J.+Borowitzauthor=C.+I.+Civinauthor=D.+Small&title=Expression+of+the+hematopoietic+growth+factor+receptor+FLT3+%28STK-1%2FFlk2%29+in+human+leukemia"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCarow%26aufirst%3DC.%2BE.%26aulast%3DLevenstein%26aufirst%3DM.%26aulast%3DKaufmann%26aufirst%3DS.%2BH.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DAmin%26aufirst%3DS.%26aulast%3DRockwell%26aufirst%3DP.%26aulast%3DWitte%26aufirst%3DL.%26aulast%3DBorowitz%26aufirst%3DM.%2BJ.%26aulast%3DCivin%26aufirst%3DC.%2BI.%26aulast%3DSmall%26aufirst%3DD.%26atitle%3DExpression%2520of%2520the%2520hematopoietic%2520growth%2520factor%2520receptor%2520FLT3%2520%2528STK-1%252FFlk2%2529%2520in%2520human%2520leukemia%26jtitle%3DBlood%26date%3D1996%26volume%3D87%26spage%3D1089%26epage%3D1096" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Drexler, H. G.</span><span> </span><span class="NLM_article-title">Expression of FLT3 receptor and response to FLT3 ligand by leukemic cells</span> <span class="citation_source-journal">Leukemia</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">588</span><span class="NLM_x">–</span> <span class="NLM_lpage">599</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Fjm500118j&amp;key=8618433" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Fjm500118j&amp;key=1%3ACAS%3A280%3ADyaK283gtV2qsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=1996&pages=588-599&author=H.+G.+Drexler&title=Expression+of+FLT3+receptor+and+response+to+FLT3+ligand+by+leukemic+cells"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Expression of FLT3 receptor and response to FLT3 ligand by leukemic cells</span></div><div class="casAuthors">Drexler H G</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">588-99</span>
        ISSN:<span class="NLM_cas:issn">0887-6924</span>.
    </div><div class="casAbstract">The novel hematopoietic growth factor FLT3 ligand (FL) is the cognate ligand for the FLT3, tyrosine kinase receptor (R), also referred to as FLK-2 and STK-1.  The FLT3R belongs to a family of receptor tyrosine kinases involved in hematopoiesis that also includes KIT, the receptor for SCF (stem cell factor), and FMS. the receptor for M-CSF (macrophage colony- stimulating factor).  Restricted FLT3R expression was seen on human and murine hematopoietic progenitor cells.  In functional assays recombinant FL stimulated the proliferation and colony formation of human hematopoietic progenitor cells, i.e.  CD34+ cord and peripheral blood, bone marrow and fetal liver cells.  Synergy was reported for co-stimulation with G-CSF (granulocyte-CSF).  GM-CSF (granulocyte-macrophage CSF), M-CSF, interleukin-3 (IL-3), PIXY-321 (an IL-3/GM-CSF fusion protein) and SCF.  In the mouse, FL potently enhanced growth of various types of progenitor/precursor cells in synergy with G-CSF, GM-CSF, M-CSF, IL-3, IL-6, IL-7, IL-11, IL-12 and SCF.  The well-documented involvement of this ligand-receptor pair in physiological hematopoiesis brought forth the question whether FLT3R and FL might also have a role in the pathobiology of leukemia.  At the mRNA level FLT3R was expressed by most (80-100%) cases of AML (acute myeloid leukemia) throughout the different morphological subtypes (MO-M7), of ALL(acute lymphoblastic leukemia) of the immunological subtypes T-ALL and BCP-ALL (B cell precursor ALL including pre-pre B-ALL, cALL and pre B-ALL), of AMLL (acute mixed-lineage leukemia), and of CML (chronic myeloid leukemia) in lymphoid or mixed blast crisis.  Analysis of cell surface expression of FLT3R by flow cytometry confirmed these observations for AML (66% positivity when the data from all studies are combined), BCP-ALL (64%) and CML lymphoid blast crisis (86%) whereas less than 30% of T-ALL were FLT3R+.  The myeloid, monocytic and pre B cell type categories also contained the highest proportions of FLT3R+ leukemia cell lines .  In contrast to the selective expression of the receptor, FL expression was detected in 90-100% of the various cell types of leukemia cell lines from all hematopoietic cell lineages.  The potential of FL to induce proliferation of leukemia cells in vitro was also examined in primary and continuously cultured leukemia cells.  The data on FL-stimulated leukemia cell growth underline the extensive heterogeneity of primary AML and ALL samples in terms of cytokine-inducible DNA synthesis that has been seen with other effective cytokines.  While the majority of T-ALL (0-33% of the cases responded proliferatively; mean 11%) and BCP-ALL (0-30%; mean 20%) failed to proliferate in the presence of FL despite strong expression of surface FLT3R, FL caused a proliferative response in a significantly higher percentage of AML cases (22-90%; mean 53%).  In the panel of leukemia cell lines examined only myeloid and monocytic growth factor- dependent cell lines increased their proliferation upon incubation with FL, whereas all growth factor-independent cell lines were refractory to stimulation.  Combinations of FL with G-CSF, GM-CSF, M-CSF, IL-3, PIXY- 321 or SCF and FL with IL-3 or IL-7 had synergistic or additive mitogenic effects on primary AML and ALL cells, respectively.  The potent stimulation of the myelomonocytic cell lines was further augmented by addition of bFGF (basic fibroblast growth factor), GM-CSF, IL-3 or SCF.  The inhibitory effects of TGF-beta 1 (transforming growth factor-beta 1) on FL- supported proliferation were abrogated by bFGF.  Taken together, these results demonstrate the expression of functional FLT3R capable of mediating FL- dependent mitogenic signaling in a subset of AML and ALL cases further underline the heterogeneity of AML and ALL samples in their proliferative response to cytokine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT_MDUZMlgnZlm8bhe6roZYfW6udTcc2eaT8sOeob0SE7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK283gtV2qsA%253D%253D&md5=d9d427a65fa9b8b1b279c442d353f18f</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DDrexler%26aufirst%3DH.%2BG.%26atitle%3DExpression%2520of%2520FLT3%2520receptor%2520and%2520response%2520to%2520FLT3%2520ligand%2520by%2520leukemic%2520cells%26jtitle%3DLeukemia%26date%3D1996%26volume%3D10%26spage%3D588%26epage%3D599" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Thiede, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steudel, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mohr, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schaich, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schakel, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Platzbecker, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wermke, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bornhauser, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ritter, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neubauer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ehninger, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Illmer, T.</span><span> </span><span class="NLM_article-title">Analysis of FLT3-activating mutations in 979 patients with acute myelogenous leukemia: association with FAB subtypes and identification of subgroups with poor prognosis</span> <span class="citation_source-journal">Blood</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">99</span><span class="NLM_x">, </span> <span class="NLM_fpage">4326</span><span class="NLM_x">–</span> <span class="NLM_lpage">4335</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=2002&pages=4326-4335&author=C.+Thiedeauthor=C.+Steudelauthor=B.+Mohrauthor=M.+Schaichauthor=U.+Schakelauthor=U.+Platzbeckerauthor=M.+Wermkeauthor=M.+Bornhauserauthor=M.+Ritterauthor=A.+Neubauerauthor=G.+Ehningerauthor=T.+Illmer&title=Analysis+of+FLT3-activating+mutations+in+979+patients+with+acute+myelogenous+leukemia%3A+association+with+FAB+subtypes+and+identification+of+subgroups+with+poor+prognosis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DThiede%26aufirst%3DC.%26aulast%3DSteudel%26aufirst%3DC.%26aulast%3DMohr%26aufirst%3DB.%26aulast%3DSchaich%26aufirst%3DM.%26aulast%3DSchakel%26aufirst%3DU.%26aulast%3DPlatzbecker%26aufirst%3DU.%26aulast%3DWermke%26aufirst%3DM.%26aulast%3DBornhauser%26aufirst%3DM.%26aulast%3DRitter%26aufirst%3DM.%26aulast%3DNeubauer%26aufirst%3DA.%26aulast%3DEhninger%26aufirst%3DG.%26aulast%3DIllmer%26aufirst%3DT.%26atitle%3DAnalysis%2520of%2520FLT3-activating%2520mutations%2520in%2520979%2520patients%2520with%2520acute%2520myelogenous%2520leukemia%253A%2520association%2520with%2520FAB%2520subtypes%2520and%2520identification%2520of%2520subgroups%2520with%2520poor%2520prognosis%26jtitle%3DBlood%26date%3D2002%26volume%3D99%26spage%3D4326%26epage%3D4335" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Pratz, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levis, M. J.</span><span> </span><span class="NLM_article-title">Bench to bedside targeting of FLT3 in acute leukemia</span> <span class="citation_source-journal">Curr. Drug Targets</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">781</span><span class="NLM_x">–</span> <span class="NLM_lpage">789</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Fjm500118j&amp;key=10.2174%2F138945010791320782" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Fjm500118j&amp;key=20370649" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Fjm500118j&amp;key=1%3ACAS%3A528%3ADC%252BC3cXpsFeisLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2010&pages=781-789&author=K.+W.+Pratzauthor=M.+J.+Levis&title=Bench+to+bedside+targeting+of+FLT3+in+acute+leukemia"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Bench to bedside targeting of FLT3 in acute leukemia</span></div><div class="casAuthors">Pratz, Keith W.; Levis, Mark J.</div><div class="citationInfo"><span class="NLM_cas:title">Current Drug Targets</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">781-789</span>CODEN:
                <span class="NLM_cas:coden">CDTUAU</span>;
        ISSN:<span class="NLM_cas:issn">1389-4501</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  FMS-Like-Tyrosine kinase-3 (FLT3) mutations are found in about 30% of cases of acute myeloid leukemia and confer an increased relapse rate and reduced overall survival.  Targeting this tyrosine kinase by direction inhibition is the focus of both preclin. and clin. research in AML.  Several mols. in clin. development inhibit FLT3, but thus far clin. responses have been limited.  Correlative studies from monotherapy trials have established that responses require sustained, effective FLT3 inhibition in vivo.  Studies combining FLT3 inhibitors with chemotherapy have demonstrated increased remission rates to date but have yet to produce a survival advantage.  Currently the only approved FLT3 inhibitor available for off-label use is sorafenib, which clearly has clin. activity but does not commonly lead to a complete response.  Several FLT3 inhibitors are currently being tested as single agents and in combination with chemotherapy, and it seems likely that a clin. useful drug will eventually emerge.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTnebgO9Va-rVg90H21EOLACvtfcHk0lh19jV8DKzg5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXpsFeisLw%253D&md5=dde8a09c2b96f400ad4646f046a45744</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.2174%2F138945010791320782&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138945010791320782%26sid%3Dliteratum%253Aachs%26aulast%3DPratz%26aufirst%3DK.%2BW.%26aulast%3DLevis%26aufirst%3DM.%2BJ.%26atitle%3DBench%2520to%2520bedside%2520targeting%2520of%2520FLT3%2520in%2520acute%2520leukemia%26jtitle%3DCurr.%2520Drug%2520Targets%26date%3D2010%26volume%3D11%26spage%3D781%26epage%3D789" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Kindler, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lipka, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fischer, T.</span><span> </span><span class="NLM_article-title">FLT3 as a therapeutic target in AML; still challenging after all these years</span> <span class="citation_source-journal">Blood</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">5089</span><span class="NLM_x">–</span> <span class="NLM_lpage">5102</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2010&pages=5089-5102&author=T.+Kindlerauthor=D.+B.+Lipkaauthor=T.+Fischer&title=FLT3+as+a+therapeutic+target+in+AML%3B+still+challenging+after+all+these+years"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKindler%26aufirst%3DT.%26aulast%3DLipka%26aufirst%3DD.%2BB.%26aulast%3DFischer%26aufirst%3DT.%26atitle%3DFLT3%2520as%2520a%2520therapeutic%2520target%2520in%2520AML%253B%2520still%2520challenging%2520after%2520all%2520these%2520years%26jtitle%3DBlood%26date%3D2010%26volume%3D9%26spage%3D5089%26epage%3D5102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Smith, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Salerno, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Damon, L. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levis, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perl, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Travers, K. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunt, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zarrinkar, P. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schadt, E. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kasarskis, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuriyan, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shah, N. P.</span><span> </span><span class="NLM_article-title">Validation of ITD mutations in FLT3 as a therapeutic target in human acute myeloid leukaemia</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">485</span><span class="NLM_x">, </span> <span class="NLM_fpage">260</span><span class="NLM_x">–</span> <span class="NLM_lpage">263</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Fjm500118j&amp;key=10.1038%2Fnature11016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Fjm500118j&amp;key=22504184" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Fjm500118j&amp;key=1%3ACAS%3A528%3ADC%252BC38XmtVeqtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=485&publication_year=2012&pages=260-263&author=C.+C.+Smithauthor=Q.+Wangauthor=C.+Chinauthor=S.+Salernoauthor=L.+E.+Damonauthor=M.+J.+Levisauthor=A.+E.+Perlauthor=K.+J.+Traversauthor=S.+Wangauthor=J.+P.+Huntauthor=P.+P.+Zarrinkarauthor=E.+E.+Schadtauthor=A.+Kasarskisauthor=J.+Kuriyanauthor=N.+P.+Shah&title=Validation+of+ITD+mutations+in+FLT3+as+a+therapeutic+target+in+human+acute+myeloid+leukaemia"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Validation of ITD mutations in FLT3 as a therapeutic target in human acute myeloid leukemia</span></div><div class="casAuthors">Smith, Catherine C.; Wang, Qi; Chin, Chen-Shan; Salerno, Sara; Damon, Lauren E.; Levis, Mark J.; Perl, Alexander E.; Travers, Kevin J.; Wang, Susana; Hunt, Jeremy P.; Zarrinkar, Patrick P.; Schadt, Eric E.; Kasarskis, Andrew; Kuriyan, John; Shah, Neil P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">485</span>
        (<span class="NLM_cas:issue">7397</span>),
    <span class="NLM_cas:pages">260-263</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Effective targeted cancer therapeutic development depends upon distinguishing disease-assocd. driver' mutations, which have causative roles in malignancy pathogenesis, from passenger' mutations, which are dispensable for cancer initiation and maintenance.  Translational studies of clin. active targeted therapeutics can definitively discriminate driver from passenger lesions and provide valuable insights into human cancer biol.  Activating internal tandem duplication (ITD) mutations in FLT3 (FLT3-ITD) are detected in approx. 20% of acute myeloid leukemia (AML) patients and are assocd. with a poor prognosis.  Abundant scientific and clin. evidence, including the lack of convincing clin. activity of early FLT3 inhibitors, suggests that FLT3-ITD probably represents a passenger lesion.  Here we report point mutations at three residues within the kinase domain of FLT3-ITD that confer substantial in vitro resistance to AC220 (quizartinib), an active investigational inhibitor of FLT3, KIT, PDGFRA, PDGFRB and RET; evolution of AC220-resistant substitutions at two of these amino acid positions was obsd. in eight of eight FLT3-ITD-pos. AML patients with acquired resistance to AC220.  Our findings demonstrate that FLT3-ITD can represent a driver lesion and valid therapeutic target in human AML.  AC220-resistant FLT3 kinase domain mutants represent high-value targets for future FLT3 inhibitor development efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqq0PeqhlWLObVg90H21EOLACvtfcHk0lh19jV8DKzg5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmtVeqtr8%253D&md5=228dae85b08e65918f74ee8c5d20ddb0</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnature11016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature11016%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DC.%2BC.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DChin%26aufirst%3DC.%26aulast%3DSalerno%26aufirst%3DS.%26aulast%3DDamon%26aufirst%3DL.%2BE.%26aulast%3DLevis%26aufirst%3DM.%2BJ.%26aulast%3DPerl%26aufirst%3DA.%2BE.%26aulast%3DTravers%26aufirst%3DK.%2BJ.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DHunt%26aufirst%3DJ.%2BP.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26aulast%3DSchadt%26aufirst%3DE.%2BE.%26aulast%3DKasarskis%26aufirst%3DA.%26aulast%3DKuriyan%26aufirst%3DJ.%26aulast%3DShah%26aufirst%3DN.%2BP.%26atitle%3DValidation%2520of%2520ITD%2520mutations%2520in%2520FLT3%2520as%2520a%2520therapeutic%2520target%2520in%2520human%2520acute%2520myeloid%2520leukaemia%26jtitle%3DNature%26date%3D2012%26volume%3D485%26spage%3D260%26epage%3D263" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Knapper, S.</span><span> </span><span class="NLM_article-title">FLT3 inhibition in acute myeloid leukaemia</span> <span class="citation_source-journal">Br. J. Haematol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">138</span><span class="NLM_x">, </span> <span class="NLM_fpage">687</span><span class="NLM_x">–</span> <span class="NLM_lpage">699</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Fjm500118j&amp;key=10.1111%2Fj.1365-2141.2007.06700.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Fjm500118j&amp;key=17655729" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Fjm500118j&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFCksrfL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=138&publication_year=2007&pages=687-699&author=S.+Knapper&title=FLT3+inhibition+in+acute+myeloid+leukaemia"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">FLT3 inhibition in acute myeloid leukaemia</span></div><div class="casAuthors">Knapper, Steven</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Haematology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">138</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">687-699</span>CODEN:
                <span class="NLM_cas:coden">BJHEAL</span>;
        ISSN:<span class="NLM_cas:issn">0007-1048</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">A review.  FMS-like tyrosine kinase 3 (FLT3) is a receptor tyrosine kinase that appears to play a significant role in leukemogenesis.  Activating mutations of FLT3 are present in approx. one-third of acute myeloid leukemia patients and are assocd. with adverse clin. outcome, while many nonmutated cases also show evidence of FLT3 activation.  FLT3 thus represents a potentially exciting mol. therapeutic target.  A no. of small-mol. tyrosine kinase inhibitors with anti-FLT3 activity have been developed and several of these compds. have entered early phase clin. trials where clin. anti-leukemic activity has been demonstrated.  The depth and duration of clin. responses to FLT3 inhibitor monotherapy have been modest, however, and a no. of mechanisms by which blasts may acquire resistance have been proposed.  Based on preclin. evidence of synergy with conventional chemotherapy, several combination trials are now underway.  FLT3 inhibition may also be effective used in combination with other molecularly targeted agents, in postchemotherapy stem-cell-directed maintenance therapy and in MLL-rearranged infant acute lymphoblastic leukemia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq35LdA5Gz4drVg90H21EOLACvtfcHk0lh19jV8DKzg5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFCksrfL&md5=f10153f7e194e393e133e402a90c739f</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1111%2Fj.1365-2141.2007.06700.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1365-2141.2007.06700.x%26sid%3Dliteratum%253Aachs%26aulast%3DKnapper%26aufirst%3DS.%26atitle%3DFLT3%2520inhibition%2520in%2520acute%2520myeloid%2520leukaemia%26jtitle%3DBr.%2520J.%2520Haematol.%26date%3D2007%26volume%3D138%26spage%3D687%26epage%3D699" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blaser, B. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duchemin, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kusewitt, D. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caligiuri, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Briesewitz, R.</span><span> </span><span class="NLM_article-title">Pharmacologic inhibition of CDK4/6: mechanistic evidence for selective activity or acquired resistance in acute myeloid leukemia</span> <span class="citation_source-journal">Blood</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">110</span><span class="NLM_x">, </span> <span class="NLM_fpage">2075</span><span class="NLM_x">–</span> <span class="NLM_lpage">2083</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Fjm500118j&amp;key=10.1182%2Fblood-2007-02-071266" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Fjm500118j&amp;key=17537993" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Fjm500118j&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtVGjtbbK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2007&pages=2075-2083&author=L.+Wangauthor=J.+Wangauthor=B.+W.+Blaserauthor=A.+M.+Ducheminauthor=D.+F.+Kusewittauthor=T.+Liuauthor=M.+A.+Caligiuriauthor=R.+Briesewitz&title=Pharmacologic+inhibition+of+CDK4%2F6%3A+mechanistic+evidence+for+selective+activity+or+acquired+resistance+in+acute+myeloid+leukemia"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacologic inhibition of CDK4/6: mechanistic evidence for selective activity or acquired resistance in acute myeloid leukemia</span></div><div class="casAuthors">Wang, Lisheng; Wang, Jie; Blaser, Bradley W.; Duchemin, Anne-Marie; Kusewitt, Donna F.; Liu, Tom; Caligiuri, Michael A.; Briesewitz, Roger</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2075-2083</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">Entry into the cell cycle is mediated by cyclin-dependent kinase 4/6 (CDK4/6) activation, followed by CDK2 activation.  We found that pharmacol. inhibition of the Flt3 internal tandem duplication (ITD), a mutated receptor tyrosine kinase commonly found in patients with acute myelogenous leukemia (AML), led to the down-regulation of cyclin D2 and D3 followed by retinoblastoma protein (pRb) dephosphorylation and G1 cell-cycle arrest.  This implicated the D-cyclin-CDK4/6 complex as a downstream effector of Flt3 ITD signaling.  Indeed, single-agent PD0332991, a selective CDK4/6 inhibitor, caused sustained cell-cycle arrest in Flt3 ITD AML cell lines and prolonged survival in an in vivo model of Flt3 ITD AML.  PD0332991 caused an initial cell-cycle arrest in well-established Flt3 wild-type (wt) AML cell lines, but this was overcome by down-regulation of p27Kip and reactivation of CDK2.  This acquired resistance was not obsd. in a Flt3 ITD and a Flt3 wt sample from a patient with primary AML.  In summary, the mechanism of cell-cycle arrest after treatment of Flt3 ITD AML with a Flt3 inhibitor involves down-regulation of cyclin D2 and D3.  As such, CDK4/6 can be a therapeutic target in Flt3 ITD AML but also in primary Flt3 wt AML.  Finally, acquired resistance to CDK4/6 inhibition can arise through activation CDK2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoiH_wOIP8jJLVg90H21EOLACvtfcHk0lj6cSWzFpdvcw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtVGjtbbK&md5=5f68ef6deacf3789e428731722f8c5ff</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1182%2Fblood-2007-02-071266&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2007-02-071266%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DBlaser%26aufirst%3DB.%2BW.%26aulast%3DDuchemin%26aufirst%3DA.%2BM.%26aulast%3DKusewitt%26aufirst%3DD.%2BF.%26aulast%3DLiu%26aufirst%3DT.%26aulast%3DCaligiuri%26aufirst%3DM.%2BA.%26aulast%3DBriesewitz%26aufirst%3DR.%26atitle%3DPharmacologic%2520inhibition%2520of%2520CDK4%252F6%253A%2520mechanistic%2520evidence%2520for%2520selective%2520activity%2520or%2520acquired%2520resistance%2520in%2520acute%2520myeloid%2520leukemia%26jtitle%3DBlood%26date%3D2007%26volume%3D110%26spage%3D2075%26epage%3D2083" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Drexler, H. G.</span><span> </span><span class="NLM_article-title">Review of alterations of the cyclin-dependent kinase inhibitor INK4 family genes p15, p16, p18 and p19 in human leukemia-lymphoma cells</span> <span class="citation_source-journal">Leukemia</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">845</span><span class="NLM_x">–</span> <span class="NLM_lpage">859</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=1998&pages=845-859&author=H.+G.+Drexler&title=Review+of+alterations+of+the+cyclin-dependent+kinase+inhibitor+INK4+family+genes+p15%2C+p16%2C+p18+and+p19+in+human+leukemia-lymphoma+cells"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DDrexler%26aufirst%3DH.%2BG.%26atitle%3DReview%2520of%2520alterations%2520of%2520the%2520cyclin-dependent%2520kinase%2520inhibitor%2520INK4%2520family%2520genes%2520p15%252C%2520p16%252C%2520p18%2520and%2520p19%2520in%2520human%2520leukemia-lymphoma%2520cells%26jtitle%3DLeukemia%26date%3D1998%26volume%3D12%26spage%3D845%26epage%3D859" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Malumbres, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barbacid, M.</span><span> </span><span class="NLM_article-title">Cell cycle, CDKs and cancer: a changing paradigm</span> <span class="citation_source-journal">Nature Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">253</span><span class="NLM_x">–</span> <span class="NLM_lpage">266</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Fjm500118j&amp;key=10.1038%2Fnrc2602" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Fjm500118j&amp;key=19242414" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2009&pages=253-266&author=M.+Malumbresauthor=M.+Barbacid&title=Cell+cycle%2C+CDKs+and+cancer%3A+a+changing+paradigm"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fnrc2602&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2602%26sid%3Dliteratum%253Aachs%26aulast%3DMalumbres%26aufirst%3DM.%26aulast%3DBarbacid%26aufirst%3DM.%26atitle%3DCell%2520cycle%252C%2520CDKs%2520and%2520cancer%253A%2520a%2520changing%2520paradigm%26jtitle%3DNature%2520Rev.%2520Cancer%26date%3D2009%26volume%3D9%26spage%3D253%26epage%3D266" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Li, Z.; Dai, K.; Keegan, K.; Ma, J.; Ragains, M.; Kaizerman, J.; McMinn, D.; Fu, J.; Fisher, B.; Gribble, M.</span><span class="NLM_etal"></span><span> </span><span class="NLM_article-title">CDK4/FLT3 dual inhibitors as potential therapeutics for acute myeloid leukemia</span>. In Drug Discovery and Lead Optimization; <span class="NLM_conf-name">Proceedings of the American Association for Cancer Research, AACR Annual Meeting 2013</span>, <span class="NLM_conf-loc">Washington, DC</span>, <span class="NLM_conf-date">April 6–10, 2013</span>;  <span class="NLM_fpage">Abstract 2351, p 575</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Li%2C+Z.%3B+Dai%2C+K.%3B+Keegan%2C+K.%3B+Ma%2C+J.%3B+Ragains%2C+M.%3B+Kaizerman%2C+J.%3B+McMinn%2C+D.%3B+Fu%2C+J.%3B+Fisher%2C+B.%3B+Gribble%2C+M.+CDK4%2FFLT3+dual+inhibitors+as+potential+therapeutics+for+acute+myeloid+leukemia.+In+Drug+Discovery+and+Lead+Optimization%3B+Proceedings+of+the+American+Association+for+Cancer+Research%2C+AACR+Annual+Meeting+2013%2C+Washington%2C+DC%2C+April+6%E2%80%9310%2C+2013%3B+Abstract+2351%2C+p+575."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dconf-proc%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DZ.%26atitle%3DCDK4%252FFLT3%2520dual%2520inhibitors%2520as%2520potential%2520therapeutics%2520for%2520acute%2520myeloid%2520leukemia%26jtitle%3DDrug%2520Discovery%2520and%2520Lead%2520Optimization%26spage%3DAbstract%25202351%252C%2520p%2520575" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="note"><p class="first last">All software programs used for molecular modeling are from Schrödinger (New York, NY). The homology model for FLT3 with the activation loop in the DFG-in conformation was built with cKit as the scaffold (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1PKG">1PKG</a>) using the Prime module. Compound <b>1</b> was modeled into the ATP binding site of the FLT3 model using Glide. Protein representations were prepared via PyMOL 1.5.0.5.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Malumbres, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pevarello, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barbacid, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bischoff, J. R.</span><span> </span><span class="NLM_article-title">CDK inhibitors in cancer therapy: What is next?</span> <span class="citation_source-journal">Trends Pharmacol. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">16</span><span class="NLM_x">–</span> <span class="NLM_lpage">21</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2008&pages=16-21&author=M.+Malumbresauthor=P.+Pevarelloauthor=M.+Barbacidauthor=J.+R.+Bischoff&title=CDK+inhibitors+in+cancer+therapy%3A+What+is+next%3F"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMalumbres%26aufirst%3DM.%26aulast%3DPevarello%26aufirst%3DP.%26aulast%3DBarbacid%26aufirst%3DM.%26aulast%3DBischoff%26aufirst%3DJ.%2BR.%26atitle%3DCDK%2520inhibitors%2520in%2520cancer%2520therapy%253A%2520What%2520is%2520next%253F%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2008%26volume%3D29%26spage%3D16%26epage%3D21" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="note"><p class="first last">Detailed docking studies of our compound to the CYP3A4 active site will be published elsewhere in due course.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Takács-Novák, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Szasz, G.</span><span> </span><span class="NLM_article-title">ion pair partition of quaternary ammonium drugs: the influence of counter ions of different lipophilicity, size, and flexibility</span> <span class="citation_source-journal">Pharm. Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">1633</span><span class="NLM_x">–</span> <span class="NLM_lpage">1638</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=1999&pages=1633-1638&author=K.+Tak%C3%A1cs-Nov%C3%A1kauthor=G.+Szasz&title=ion+pair+partition+of+quaternary+ammonium+drugs%3A+the+influence+of+counter+ions+of+different+lipophilicity%2C+size%2C+and+flexibility"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DTak%25C3%25A1cs-Nov%25C3%25A1k%26aufirst%3DK.%26aulast%3DSzasz%26aufirst%3DG.%26atitle%3Dion%2520pair%2520partition%2520of%2520quaternary%2520ammonium%2520drugs%253A%2520the%2520influence%2520of%2520counter%2520ions%2520of%2520different%2520lipophilicity%252C%2520size%252C%2520and%2520flexibility%26jtitle%3DPharm.%2520Res.%26date%3D1999%26volume%3D16%26spage%3D1633%26epage%3D1638" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span>KINOMEscan Assay Platform; <span class="NLM_publisher-name">DiscoveRx Corporation</span>: <span class="NLM_publisher-loc">Fremont, CA</span>; <a href="http://www.discoverx.com/technologies-platforms/competitive-binding-technology/kinomescan-technology-platform" class="extLink">http://www.discoverx.com/technologies-platforms/competitive-binding-technology/kinomescan-technology-platform</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=KINOMEscan+Assay+Platform%3B+DiscoveRx+Corporation%3A+Fremont%2C+CA%3B+http%3A%2F%2Fwww.discoverx.com%2Ftechnologies-platforms%2Fcompetitive-binding-technology%2Fkinomescan-technology-platform."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26jtitle%3DKINOMEscan%2520Assay%2520Platform%26pub%3DDiscoveRx%2520Corporation" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="note"><p class="first last">Sorafenib and palbociclib were purchased from AdooQ BioScience (Irvine, CA).</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="note"><p class="first last">More detailed in vitro and in vivo studies of <b>28</b> and its abilities to simultaneously inhibit CDK4 and FLT3 and suppress mutations in FLT3 will be published elsewhere in due course.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Zuccotto, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ardini, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casale, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Angiolini, M.</span><span> </span><span class="NLM_article-title">Through the “gatekeeper door”: Exploiting active kinase conformation</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">2681</span><span class="NLM_x">–</span> <span class="NLM_lpage">2694</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm901443h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2681-2694&author=F.+Zuccottoauthor=E.+Ardiniauthor=E.+Casaleauthor=M.+Angiolini&title=Through+the+%E2%80%9Cgatekeeper+door%E2%80%9D%3A+Exploiting+active+kinase+conformation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fjm901443h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901443h%26sid%3Dliteratum%253Aachs%26aulast%3DZuccotto%26aufirst%3DF.%26aulast%3DArdini%26aufirst%3DE.%26aulast%3DCasale%26aufirst%3DE.%26aulast%3DAngiolini%26aufirst%3DM.%26atitle%3DThrough%2520the%2520%25E2%2580%259Cgatekeeper%2520door%25E2%2580%259D%253A%2520Exploiting%2520active%2520kinase%2520conformation%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D2681%26epage%3D2694" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N. S.</span><span> </span><span class="NLM_article-title">Rational design of inhibitors that bind to inactive kinase conformations</span> <span class="citation_source-journal">Nature Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">358</span><span class="NLM_x">–</span> <span class="NLM_lpage">364</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Fjm500118j&amp;key=10.1038%2Fnchembio799" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Fjm500118j&amp;key=16783341" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Fjm500118j&amp;key=1%3ACAS%3A528%3ADC%252BD28XlvFGjs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2006&pages=358-364&author=Y.+Liuauthor=N.+S.+Gray&title=Rational+design+of+inhibitors+that+bind+to+inactive+kinase+conformations"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Rational design of inhibitors that bind to inactive kinase conformations</span></div><div class="casAuthors">Liu, Yi; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">358-364</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The majority of kinase inhibitors, known as type I inhibitors that have been developed so far target the ATP binding site of the kinase in its active conformation, in which the activation loop is phosphorylated.  Recently, crystal structures of inhibitors such as imatinib, BIRB796, and sorafenib (BAY43-9006), known as type II inhibitors, have revealed a new binding mode that exploits an addnl. binding site immediately adjacent to the region occupied by ATP.  This pocket is made accessible by an activation-loop rearrangement that is characteristic of kinases in an inactive conformation.  The examn. of binding features of known type II inhibitors is discussed and a pharmacophore model currently used in the design of a new generation of structurally diverse type II inhibitors is presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxahsMPlhgDbVg90H21EOLACvtfcHk0lg1s3lETGOaXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XlvFGjs7g%253D&md5=4a9347a44a2cf4a412c9741fdcc5696b</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fnchembio799&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio799%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DRational%2520design%2520of%2520inhibitors%2520that%2520bind%2520to%2520inactive%2520kinase%2520conformations%26jtitle%3DNature%2520Chem.%2520Biol.%26date%3D2006%26volume%3D2%26spage%3D358%26epage%3D364" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="note"><p class="first last">FLT3 apo structure (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1RJB">1RJB</a>) used for DFG-out kinase with Sorafenib modeled with Glide. FLT3 DFG-in homology model (cKIT, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1PKG">1PKG</a>) used to model <b>28</b>.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="note"><p class="first last">The deference in inhibition between pRb and pSTAT5 at 24 h time point in 75 and 150 mg/kg dosing groups is due to the fact that total STAT5 was stable with the compound treatment, while total Rb decreased with the compound treatment. At 24 h at doses of 150 and 75 mg/kg, compound concentrations in plasma went down, so STAT5 phosphorylation was not inhibited, and back to normal. In contrast, although Rb phosphorylation was not inhibited at 24 h, but because total Rb level was low, pRb appeared low compared to vehicle.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="note"><p class="first last">Syntheses of certain substituted cycloalkane derivatives (e.g., <b>8</b>, <b>9</b>, and <b>14</b>) employed minor modifications of the general route depicted in Scheme <a class="ref internalNav" href="#sch1" aria-label="1">1</a>, such as permutation of steps (<b>14</b>) or inclusion of additional protecting group manipulations (<b>8</b>, <b>9</b>). These deviations from the mainstream approach are detailed in the experimental procedures for the relevant compounds.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="note"><p class="first last">The exocyclic amine nitrogen was arbitrarily numbered as 1′ in this scheme to facilitate unambiguous reference to this atom in the discussion. IUPAC names for the fully elaborated derivatives number this position differently.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Gribble, G. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saulnier, M. G.</span><span> </span><span class="NLM_article-title">Regioselective ortho lithiation of halopyridines</span> <span class="citation_source-journal">Tetrahedron Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1980</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">4137</span><span class="NLM_x">–</span> <span class="NLM_lpage">4140</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=1980&pages=4137-4140&author=G.+W.+Gribbleauthor=M.+G.+Saulnier&title=Regioselective+ortho+lithiation+of+halopyridines"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGribble%26aufirst%3DG.%2BW.%26aulast%3DSaulnier%26aufirst%3DM.%2BG.%26atitle%3DRegioselective%2520ortho%2520lithiation%2520of%2520halopyridines%26jtitle%3DTetrahedron%2520Lett.%26date%3D1980%26volume%3D21%26spage%3D4137%26epage%3D4140" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Marsais, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Queguiner, G.</span><span> </span><span class="NLM_article-title">Review on the metallation of p-deficient heteroaromatic compounds. Regioselective ortho-lithiation of 3-fluoropyridine: directing effects and application to synthesis of 2,3- or 3,4-disubstituted pyridines</span> <span class="citation_source-journal">Tetrahedron</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1983</span><span class="NLM_x">, </span> <span class="NLM_volume">39</span><span class="NLM_x">, </span> <span class="NLM_fpage">2009</span><span class="NLM_x">–</span> <span class="NLM_lpage">2021</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1983&pages=2009-2021&author=F.+Marsaisauthor=G.+Queguiner&title=Review+on+the+metallation+of+p-deficient+heteroaromatic+compounds.+Regioselective+ortho-lithiation+of+3-fluoropyridine%3A+directing+effects+and+application+to+synthesis+of+2%2C3-+or+3%2C4-disubstituted+pyridines"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMarsais%26aufirst%3DF.%26aulast%3DQueguiner%26aufirst%3DG.%26atitle%3DReview%2520on%2520the%2520metallation%2520of%2520p-deficient%2520heteroaromatic%2520compounds.%2520Regioselective%2520ortho-lithiation%2520of%25203-fluoropyridine%253A%2520directing%2520effects%2520and%2520application%2520to%2520synthesis%2520of%25202%252C3-%2520or%25203%252C4-disubstituted%2520pyridines%26jtitle%3DTetrahedron%26date%3D1983%26volume%3D39%26spage%3D2009%26epage%3D2021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Liu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schulze-Gahmen, U.</span><span> </span><span class="NLM_article-title">Toward understanding the structural basis of cyclin-dependent kinase 6 specific inhibition</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">3826</span><span class="NLM_x">–</span> <span class="NLM_lpage">3831</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=3826-3831&author=H.+Liuauthor=U.+Schulze-Gahmen&title=Toward+understanding+the+structural+basis+of+cyclin-dependent+kinase+6+specific+inhibition"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DSchulze-Gahmen%26aufirst%3DU.%26atitle%3DToward%2520understanding%2520the%2520structural%2520basis%2520of%2520cyclin-dependent%2520kinase%25206%2520specific%2520inhibition%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D3826%26epage%3D3831" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Battye, T. G. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kontogiannis, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Powell, H. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leslie, A. G. W.</span><span> </span><span class="NLM_article-title">iMOSFLM: a new graphical interface for diffraction-image processing with MOSFLM</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">67</span><span class="NLM_x">, </span> <span class="NLM_fpage">271</span><span class="NLM_x">–</span> <span class="NLM_lpage">281</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=271-281&author=T.+G.+G.+Battyeauthor=L.+Kontogiannisauthor=O.+Johnsonauthor=H.+R.+Powellauthor=A.+G.+W.+Leslie&title=iMOSFLM%3A+a+new+graphical+interface+for+diffraction-image+processing+with+MOSFLM"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBattye%26aufirst%3DT.%2BG.%2BG.%26aulast%3DKontogiannis%26aufirst%3DL.%26aulast%3DJohnson%26aufirst%3DO.%26aulast%3DPowell%26aufirst%3DH.%2BR.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%2BW.%26atitle%3DiMOSFLM%253A%2520a%2520new%2520graphical%2520interface%2520for%2520diffraction-image%2520processing%2520with%2520MOSFLM%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D271%26epage%3D281" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Winn, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ballard, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dodson, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Evans, P. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keegan, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krissinel, E. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leslie, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCoy, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McNicholas, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murshudov, G. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pannu, N. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Potterton, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Powell, H. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vagin, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, K. S.</span><span> </span><span class="NLM_article-title">Overview of the CCP4 suite and current developments</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">67</span><span class="NLM_x">, </span> <span class="NLM_fpage">235</span><span class="NLM_x">–</span> <span class="NLM_lpage">242</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Fjm500118j&amp;key=10.1107%2FS0907444910045749" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Fjm500118j&amp;key=21460441" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Fjm500118j&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=235-242&author=M.+D.+Winnauthor=C.+C.+Ballardauthor=K.+D.+Cowtanauthor=E.+J.+Dodsonauthor=P.+Emsleyauthor=P.+R.+Evansauthor=R.+M.+Keeganauthor=E.+B.+Krissinelauthor=A.+G.+Leslieauthor=A.+McCoyauthor=S.+J.+McNicholasauthor=G.+N.+Murshudovauthor=N.+S.+Pannuauthor=E.+A.+Pottertonauthor=H.+R.+Powellauthor=R.+J.+Readauthor=A.+Vaginauthor=K.+S.+Wilson&title=Overview+of+the+CCP4+suite+and+current+developments"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Overview of the CCP4 suite and current developments</span></div><div class="casAuthors">Winn, Martyn D.; Ballard, Charles C.; Cowtan, Kevin D.; Dodson, Eleanor J.; Emsley, Paul; Evans, Phil R.; Keegan, Ronan M.; Krissinel, Eugene B.; Leslie, Andrew G. W.; McCoy, Airlie; McNicholas, Stuart J.; Murshudov, Garib N.; Pannu, Navraj S.; Potterton, Elizabeth A.; Powell, Harold R.; Read, Randy J.; Vagin, Alexei; Wilson, Keith S.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  The CCP4 (Collaborative Computational Project, No. 4) software suite is a collection of programs and assocd. data and software libraries which can be used for macromol. structure detn. by X-ray crystallog.  The suite is designed to be flexible, allowing users a no. of methods of achieving their aims.  The programs are from a wide variety of sources but are connected by a common infrastructure provided by std. file formats, data objects and graphical interfaces.  Structure soln. by macromol. crystallog. is becoming increasingly automated and the CCP4 suite includes several automation pipelines.  After giving a brief description of the evolution of CCP4 over the last 30 years, an overview of the current suite is given.  While detailed descriptions are given in the accompanying articles, here it is shown how the individual programs contribute to a complete software package.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_wWMMIol6UrVg90H21EOLACvtfcHk0lhzmm_L4LSAVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D&md5=c407e4d47bef46864be336d60147c17d</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1107%2FS0907444910045749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910045749%26sid%3Dliteratum%253Aachs%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DBallard%26aufirst%3DC.%2BC.%26aulast%3DCowtan%26aufirst%3DK.%2BD.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DKeegan%26aufirst%3DR.%2BM.%26aulast%3DKrissinel%26aufirst%3DE.%2BB.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%26aulast%3DMcCoy%26aufirst%3DA.%26aulast%3DMcNicholas%26aufirst%3DS.%2BJ.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DPotterton%26aufirst%3DE.%2BA.%26aulast%3DPowell%26aufirst%3DH.%2BR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DWilson%26aufirst%3DK.%2BS.%26atitle%3DOverview%2520of%2520the%2520CCP4%2520suite%2520and%2520current%2520developments%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D235%26epage%3D242" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Adams, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Afonine, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunkóczi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, V. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davis, I. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Echols, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Headd, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hung, L.-W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kapral, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCoy, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moriarty, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oeffner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Terwilliger, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zwart, P. H.</span><span> </span><span class="NLM_article-title">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">213</span><span class="NLM_x">–</span> <span class="NLM_lpage">221</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Fjm500118j&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Fjm500118j&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Fjm500118j&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunk%C3%B3cziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.-W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+a+comprehensive+Python-based+system+for+macromolecular+structure+solution"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0lhzmm_L4LSAVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunk%25C3%25B3czi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.-W.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520a%2520comprehensive%2520Python-based%2520system%2520for%2520macromolecular%2520structure%2520solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lohkamp, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, W. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K.</span><span> </span><span class="NLM_article-title">Features and development of Coot</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">486</span><span class="NLM_x">–</span> <span class="NLM_lpage">501</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Fjm500118j&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Fjm500118j&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Fjm500118j&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+of+Coot"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0lhzmm_L4LSAVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4P41" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4P41','PDB','4P41'); return false;">PDB: 4P41</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1PKG" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1PKG','PDB','1PKG'); return false;">PDB: 1PKG</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1RJB" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1RJB','PDB','1RJB'); return false;">PDB: 1RJB</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i99"><a href="/doi/suppl/10.1021/jm500118j">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_66423"></div></div></div></div></div><hr /></hr><p class="last">POC kinase selectivity data of <b>28</b>, <i>K</i><sub>d</sub> data for <b>28</b> for the set of kinases with POC less than 20 from the initial POC screen, terminal exposure levels of <b>28</b> in the MOLM13 xenograft model, and average body weights for the <b>28</b> treated mouse group in the MOLM13 xenograft model.  This material is available free of charge via the Internet at <a class="extLink" href="http://pubs.acs.org">http://pubs.acs.org</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm500118j/suppl_file/jm500118j_si_001.pdf">jm500118j_si_001.pdf (220.09 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The coordinates of <b>1</b> with CDK6 have been deposited in the PDB with accession code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4P41">4P41</a>.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Fjm500118j&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Barticle%3Aarticle%3A10.1021%2Fjm500118j%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2014.57.issue-8%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/jm500118j" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                3MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-b425991822a79ada59b3.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-fb88834b824b41b135ad.js"></script>

<script type="text/javascript" src="/wro/ke1q~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"66797e5d9e41227f","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.6.0","si":10}'></script>
</body>
</html>
