
<html lang="en"     class="pb-page"  data-request-id="31928412-e8f1-46a4-9cf1-30df7c79117c"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.9b00004;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;issue:issue:10.1021/jmcmar.2019.62.issue-22"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Are We Opening the Door to a New Era of Medicinal Chemistry or Being Collapsed to a Chemical Singularity?" /></meta><meta name="dc.Creator" content="Yan A.  Ivanenkov" /></meta><meta name="dc.Creator" content="Bogdan A.  Zagribelnyy" /></meta><meta name="dc.Creator" content="Vladimir A.  Aladinskiy" /></meta><meta name="dc.Description" content="The paradigm of “drug-like-ness” dramatically altered the behavior of the medicinal chemistry community for a long time. In recent years, scientists have empirically found a significant increase in..." /></meta><meta name="Description" content="The paradigm of “drug-like-ness” dramatically altered the behavior of the medicinal chemistry community for a long time. In recent years, scientists have empirically found a significant increase in..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="June 12, 2019" /></meta><meta name="dc.Type" content="review-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b00004" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2019 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b00004" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b00004" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b00004" /></link>
        
    
    

<title>Are We Opening the Door to a New Era of Medicinal Chemistry or Being Collapsed to a Chemical Singularity? | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b00004" /></meta><meta property="og:title" content="Are We Opening the Door to a New Era of Medicinal Chemistry or Being Collapsed to a Chemical Singularity?" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/large/jm-2019-00004q_0015.jpeg" /></meta><meta property="og:description" content="The paradigm of “drug-like-ness” dramatically altered the behavior of the medicinal chemistry community for a long time. In recent years, scientists have empirically found a significant increase in key properties of drugs that have moved structures closer to the periphery or the outside of the rule-of-five “cage”. Herein, we show that for the past decade, the number of molecules claimed in patent records by major pharmaceutical companies has dramatically decreased, which may lead to a “chemical singularity”. New compounds containing fragments with increased 3D complexity are generally larger, slightly more lipophilic, and more polar. A core difference between this study and recently published papers is that we consider the nature and quality of sp3-rich frameworks rather than sp3 count. We introduce the original descriptor MCE-18, which stands for medicinal chemistry evolution, 2018, and this measure can effectively score molecules by novelty in terms of their cumulative sp3 complexity." /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta><meta name="twitter:description" content="The paradigm of “drug-like-ness” dramatically altered the behavior of the medicinal chemistry community for a long time. In recent years, scientists have empirically found a significant increase in key properties of drugs that have moved structures closer to the periphery or the outside of the rule-of-five “cage”. Herein, we show that for the past decade, the number of molecules claimed in patent records by major pharmaceutical companies has dramatically decreased, which may lead to a “chemical singularity”. New compounds containing fragments with increased 3D complexity are generally larger, slightly more lipophilic, and more polar. A core difference between this study and recently published papers is that we consider the nature and quality of sp3-rich frameworks rather than sp3 count. We introduce the original descriptor MCE-18, which stands for medicinal chemistry evolution, 2018, and this measure can effectively score molecules by novelty in terms of their cumulative sp3 complexity." /></meta><meta name="twitter:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/large/jm-2019-00004q_0015.jpeg" /></meta><meta name="twitter:title" content="Are We Opening the Door to a New Era of Medicinal Chemistry or Being Collapsed to a Chemical Singularity?" /></meta>
        
            <meta property="og:type" content="Article" /></meta>
        

        
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b00004"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/ke1q~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-6ee29527b3d8ff00a106.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b00004">
            
        





































    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-12/achre4.2021.54.issue-12/20210615/achre4.2021.54.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-6/amrcda.2021.2.issue-6/20210625/amrcda.2021.2.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-6/aabmcb.2021.4.issue-6/20210621/aabmcb.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-6/aaembp.2021.3.issue-6/20210622/aaembp.2021.3.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-6/aaemcq.2021.4.issue-6/20210628/aaemcq.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-25/aamick.2021.13.issue-25/20210630/aamick.2021.13.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-6/aanmf6.2021.4.issue-6/20210625/aanmf6.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-6/aapmcd.2021.3.issue-6/20210611/aapmcd.2021.3.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-6/abseba.2021.7.issue-6/20210614/abseba.2021.7.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-12/accacs.2021.11.issue-12/20210618/accacs.2021.11.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-6/acscii.2021.7.issue-6/20210623/acscii.2021.7.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-6/acbcct.2021.16.issue-6/20210618/acbcct.2021.16.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-3/achsc5.2021.28.issue-3/20210524/achsc5.2021.28.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-12/acncdm.2021.12.issue-12/20210616/acncdm.2021.12.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-6/aesccq.2021.5.issue-6/20210617/aesccq.2021.5.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-6/aelccp.2021.6.issue-6/20210611/aelccp.2021.6.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-6/aeecco.2021.1.issue-6/20210611/aeecco.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-6/aewcaa.2021.1.issue-6/20210611/aewcaa.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-5/afsthl.2021.1.issue-5/20210618/afsthl.2021.1.issue-5.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-6/aidcbc.2021.7.issue-6/20210611/aidcbc.2021.7.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-6/amlccd.2021.10.issue-6/20210615/amlccd.2021.10.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-6/amlcef.2021.3.issue-6/20210607/amlcef.2021.3.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-6/amclct.2021.12.issue-6/20210610/amclct.2021.12.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-6/ancac3.2021.15.issue-6/20210622/ancac3.2021.15.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-25/acsodf.2021.6.issue-25/20210629/acsodf.2021.6.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-6/apchd5.2021.8.issue-6/20210616/apchd5.2021.8.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-6/ascefj.2021.6.issue-6/20210625/ascefj.2021.6.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-25/ascecg.2021.9.issue-25/20210628/ascecg.2021.9.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-6/asbcd6.2021.10.issue-6/20210618/asbcd6.2021.10.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-25/ancham.2021.93.issue-25/20210629/ancham.2021.93.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-25/bichaw.2021.60.issue-25/20210629/bichaw.2021.60.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-6/bcches.2021.32.issue-6/20210616/bcches.2021.32.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-6/bomaf6.2021.22.issue-6/20210614/bomaf6.2021.22.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-24/cgeabj.2021.99.issue-24/20210628-01/cgeabj.2021.99.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-6/crtoec.2021.34.issue-6/20210621/crtoec.2021.34.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-12/chreay.2021.121.issue-12/20210623/chreay.2021.121.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-12/cmatex.2021.33.issue-12/20210622/cmatex.2021.33.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-6/cgdefu.2021.21.issue-6/20210602/cgdefu.2021.21.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-12/enfuem.2021.35.issue-12/20210617/enfuem.2021.35.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-12/esthag.2021.55.issue-12/20210615/esthag.2021.55.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-6/estlcu.2021.8.issue-6/20210608/estlcu.2021.8.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-25/iecred.2021.60.issue-25/20210630/iecred.2021.60.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-12/inocaj.2021.60.issue-12/20210621/inocaj.2021.60.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-6/jaaucr.2021.1.issue-6/20210628/jaaucr.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-25/jafcau.2021.69.issue-25/20210630/jafcau.2021.69.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-6/jceaax.2021.66.issue-6/20210610/jceaax.2021.66.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-6/jceda8.2021.98.issue-6/20210608/jceda8.2021.98.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-6/jcisd8.2021.61.issue-6/20210628/jcisd8.2021.61.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-6/jctcce.2021.17.issue-6/20210608/jctcce.2021.17.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/jmcmar.2021.64.issue-12/20210624/jmcmar.2021.64.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-6/jnprdf.2021.84.issue-6/20210625/jnprdf.2021.84.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-12/joceah.2021.86.issue-12/20210618/joceah.2021.86.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-24/jpcafh.2021.125.issue-24/20210624/jpcafh.2021.125.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-24/jpcbfk.2021.125.issue-24/20210624/jpcbfk.2021.125.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-24/jpccck.2021.125.issue-24/20210624/jpccck.2021.125.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-24/jpclcd.2021.12.issue-24/20210624/jpclcd.2021.12.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-6/jprobs.2021.20.issue-6/20210604/jprobs.2021.20.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-25/jacsat.2021.143.issue-25/20210630/jacsat.2021.143.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-6/jamsef.2021.32.issue-6/20210602/jamsef.2021.32.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-25/langd5.2021.37.issue-25/20210629/langd5.2021.37.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-12/mamobx.2021.54.issue-12/20210622/mamobx.2021.54.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-6/mpohbp.2021.18.issue-6/20210607/mpohbp.2021.18.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-12/nalefd.2021.21.issue-12/20210623/nalefd.2021.21.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-12/orlef7.2021.23.issue-12/20210618/orlef7.2021.23.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-6/oprdfk.2021.25.issue-6/20210618/oprdfk.2021.25.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-12/orgnd7.2021.40.issue-12/20210628/orgnd7.2021.40.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b00004&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b00004&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b00004&amp;href=/doi/10.1021/acs.jmedchem.9b00004" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2019</span><span class="cit-fg-volume">, 62</span><span class="cit-fg-issue">, 22</span><span class="cit-fg-pageRange">, 10026-10043</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/62/22" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.8b01732" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Perspective</span><a href="/doi/10.1021/acs.jmedchem.9b00248" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Are We Opening the Door to a New Era of Medicinal Chemistry or Being Collapsed to a Chemical Singularity?</span></h1><h2 class="article_header-subtitle"><span class="hlFld-SubTitle">Perspective</span></h2><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Yan A. Ivanenkov</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yan A. Ivanenkov</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Insilico Medicine Hong Kong Limited (previously Insilico Medicine, Inc.), Unit 307A, Core Building 1, 1 Science Park East Avenue, Hong Kong Science Park, Pak Shek Kok, Hong Kong</div><div class="loa-info-affiliations-info">Institute of Biochemistry and Genetics Russian Academy of Science (IBG RAS) Ufa Scientific Centre, Oktyabrya Prospekt 71, Ufa 450054, Russian Federation</div><div class="loa-info-affiliations-info">Moscow Institute of Physics and Technology (State University), 9 Institutskiy Lane, Dolgoprudny, Moscow 141700, Russian Federation</div><div class="loa-info-affiliations-info">Chemistry Department, Lomonosov Moscow State University, Leninskie Gory, Building 1/3, GSP-1, Moscow 119991, Russian Federation</div></div><span class="conrtib-corresp"><strong>*</strong>Phone: +852 96685251. E-mail: <a href="/cdn-cgi/l/email-protection#136a727d537a7d607a7f7a707c7e76777a707a7d763d707c7e"><span class="__cf_email__" data-cfemail="fe879f90be97908d9792979d91939b9a979d97909bd09d9193">[email protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yan+A.++Ivanenkov">Yan A. Ivanenkov</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath1">View Biography</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8968-0879" title="Orcid link">http://orcid.org/0000-0002-8968-0879</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Bogdan A. Zagribelnyy</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Bogdan A. Zagribelnyy</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Insilico Medicine Hong Kong Limited (previously Insilico Medicine, Inc.), Unit 307A, Core Building 1, 1 Science Park East Avenue, Hong Kong Science Park, Pak Shek Kok, Hong Kong</div><div class="loa-info-affiliations-info">Chemistry Department, Lomonosov Moscow State University, Leninskie Gory, Building 1/3, GSP-1, Moscow 119991, Russian Federation</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Bogdan+A.++Zagribelnyy">Bogdan A. Zagribelnyy</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath2">View Biography</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Vladimir A. Aladinskiy</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Vladimir A. Aladinskiy</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Insilico Medicine Hong Kong Limited (previously Insilico Medicine, Inc.), Unit 307A, Core Building 1, 1 Science Park East Avenue, Hong Kong Science Park, Pak Shek Kok, Hong Kong</div><div class="loa-info-affiliations-info">Moscow Institute of Physics and Technology (State University), 9 Institutskiy Lane, Dolgoprudny, Moscow 141700, Russian Federation</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Vladimir+A.++Aladinskiy">Vladimir A. Aladinskiy</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath3">View Biography</a></span></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b00004&amp;href=/doi/10.1021%2Facs.jmedchem.9b00004" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2019</span></span><span class="cit-volume">, 62</span><span class="cit-issue">, 22</span><span class="cit-pageRange">, 10026–10043</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">June 12, 2019</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>10 January 2019</li><li><span class="item_label"><b>Published</b> online</span>12 June 2019</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 27 November 2019</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b00004" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00004</a></div><div class="article_header-article-copyright"><strong>Copyright © 2019 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D10026%26pageCount%3D18%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DYan%2BA.%2BIvanenkov%252C%2BBogdan%2BA.%2BZagribelnyy%252C%2BVladimir%2BA.%2BAladinskiy%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D62%26issueNum%3D22%26contentID%3Dacs.jmedchem.9b00004%26title%3DAre%2BWe%2BOpening%2Bthe%2BDoor%2Bto%2Ba%2BNew%2BEra%2Bof%2BMedicinal%2BChemistry%2Bor%2BBeing%2BCollapsed%2Bto%2Ba%2BChemical%2BSingularity%253F%26numPages%3D18%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D10043%26publicationDate%3DNovember%2B2019">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b00004"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">6767</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">13</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b00004" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Are We Opening the Door to a New Era of Medicinal Chemistry or Being Collapsed to a Chemical Singularity?&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Yan&quot;,&quot;last_name&quot;:&quot;A. Ivanenkov&quot;},{&quot;first_name&quot;:&quot;Bogdan&quot;,&quot;last_name&quot;:&quot;A. Zagribelnyy&quot;},{&quot;first_name&quot;:&quot;Vladimir&quot;,&quot;last_name&quot;:&quot;A. Aladinskiy&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2019&quot;,&quot;month&quot;:&quot;06&quot;,&quot;day&quot;:&quot;12&quot;,&quot;issue&quot;:&quot;22&quot;,&quot;volume&quot;:&quot;62&quot;,&quot;pages&quot;:&quot;10026-10043&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b00004&quot;},&quot;abstract&quot;:&quot;The paradigm of “drug-like-ness” dramatically altered the behavior of the medicinal chemistry community for a long time. In recent years, scientists have empirically found a significant increase in key properties of drugs that have moved structures closer to the periphery or the outside of the rule-of-five “cage”. Herein, we show that for the past decade, the number of molecules claimed in patent records by major pharmaceutical companies has dramatically decreased, which may lead to a “chemical singularity”. New compounds containing fragments with increased 3D complexity are generally larger, slightly more lipophilic, and more polar. A core difference between this study and recently published papers is that we consider the nature and quality of sp3-rich frameworks rather than sp3 count. We introduce the original descriptor MCE-18, which stands for medicinal chemistry evolution, 2018, and this measure can effectively score molecules by novelty in terms of their cumulative sp3 complexity.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00004&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00004" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00004&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00004" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00004&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00004" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b00004&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00004&amp;href=/doi/10.1021/acs.jmedchem.9b00004" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b00004" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b00004" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (10 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b00004%26sid%3Dliteratum%253Aachs%26pmid%3D31188596%26genre%3Darticle%26aulast%3DIvanenkov%26date%3D2019%26atitle%3DAre%2BWe%2BOpening%2Bthe%2BDoor%2Bto%2Ba%2BNew%2BEra%2Bof%2BMedicinal%2BChemistry%2Bor%2BBeing%2BCollapsed%2Bto%2Ba%2BChemical%2BSingularity%253F%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D62%26issue%3D22%26spage%3D10026%26epage%3D10043%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (1)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291445" title="Chemical structure">Chemical structure</a>,</li><li><a href="/action/doSearch?ConceptID=291494" title="Medicinal chemistry">Medicinal chemistry</a>,</li><li><a href="/action/doSearch?ConceptID=291070" title="Pharmaceuticals">Pharmaceuticals</a>,</li><li><a href="/action/doSearch?ConceptID=291754" title="Molecules">Molecules</a>,</li><li><a href="/action/doSearch?ConceptID=290731" title="Biological databases">Biological databases</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/62/22" title="Go to Volume 62, Issue 22"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/jmcmar.2019.62.issue-22/20191127/jmcmar.2019.62.issue-22.largecover.jpg" alt="Go to Volume 62, Issue 22"/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/medium/jm-2019-00004q_0015.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/large/jm-2019-00004q_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00004&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">The paradigm of “drug-like-ness” dramatically altered the behavior of the medicinal chemistry community for a long time. In recent years, scientists have empirically found a significant increase in key properties of drugs that have moved structures closer to the periphery or the outside of the rule-of-five “cage”. Herein, we show that for the past decade, the number of molecules claimed in patent records by major pharmaceutical companies has dramatically decreased, which may lead to a “chemical singularity”. New compounds containing fragments with increased 3D complexity are generally larger, slightly more lipophilic, and more polar. A core difference between this study and recently published papers is that we consider the nature and quality of sp<sup>3</sup>-rich frameworks rather than sp<sup>3</sup> count. We introduce the original descriptor MCE-18, which stands for medicinal chemistry evolution, 2018, and this measure can effectively score molecules by novelty in terms of their cumulative sp<sup>3</sup> complexity.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_34028" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_34028" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">More than one-and-a-half centuries ago, Charles Darwin and Alfred Wallace enunciated the guiding principles of the ground-breaking theory of evolution as a self-organizing, generally entropy-driven, but not chaotic, process of natural selection and adaptation of living organisms to their environment. Since that time, enormous progress has been made in almost every realm of life, science, and technology. The spectacular growth in many biological disciplines has allowed scientists to formulate unambiguous mechanistic and philosophical explanations for “how and why” evolution has transpired and how it is fueled by a range of vital impulses. In contrast to technological and industrial progress, an opposing hypothesis on biological nature has been presented that postulates the basic principles of a devolution scenario in parallel with or instead of evolution. Ontologically, we can apply the fundamental outlook of evolution and devolution to medicinal chemistry and drug design via the prism of the crucial drivers that dominate these areas. Chemically, the “big bang” was a valuable and versatile point source of a practically inexhaustible universe abundantly populated with an astronomical number of hypothetical drug-like structures (∼10<sup>60</sup>).<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1,2)</a> Although more than 135 million molecules have been reported,<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> in-house collections held by various corporations cover only ∼8–10 million unique molecules (excluding building blocks), and most of them are assemblies of “vecchio” clusters and are generally not attractive. These compounds are frequently beyond the current criteria of novelty and are not aligned with the major trends observed in modern drug discovery. The HTS boom<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> significantly exhausted available stocks of chemical libraries, resulting in an immense number of molecules languishing in vaults that have excellent chances of becoming tombs or “trashure boxes”. Within this niche, the vast majority of scaffolds and chemotypes are now considered unattractive and have limited potential, and like dinosaurs, these scaffolds are slipping away one by one. The diversity and phylogeny of biological targets are closely related to medicinal chemistry. These disciplines have merged into one symbiotic continuum and have been evolving together for a long time. As a rule, spectacular explosions in medicinal chemistry echo biological breakthroughs. However, if an elegant and novel molecule bearing a harmonious ensemble of potential binding points that meet key criteria is synthesized, an appropriate target can likely be identified. The evolution of biological targets shifts the focus of medicinal chemistry primarily toward complex natural product-like, three-dimensional structures mainly due to the considerable progress that has been made in the field of protein–protein interactions (PPIs), which are involved in disease signaling routes. Currently, scientists estimate that they have identified over 400 000 PPIs<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> involving a wide variety of binding grooves with volumes of 800–2000 Å<sup>3</sup> and binding sites spread along 250–1000 Å<sup>3</sup>.<a onclick="showRef(event, 'ref3 ref5'); return false;" href="javascript:void(0);" class="ref ref3 ref5">(3,5)</a> It is not surprising that companies have deployed extensive programs for designing PPI-focused libraries containing a large number of novel aliphatic and heteroaliphatic systems, <i>spiro</i> and chiral centers, bridged and fused rings to achieve better coverage of the 3D chemical space.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> However, annually, only approximately 15 new molecular entities (NMEs) are brought to the market, which highlights the stagnation of the pharmaceutical industry, mainly due to the high project attrition rate and growing IP barriers.<a onclick="showRef(event, 'ref7 ref8 ref9 ref10 ref11 ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref7 ref8 ref9 ref10 ref11 ref12 ref13">(7−13)</a> Thus, a tremendous burden looms over medicinal chemists who have been implicated in this nontrivial process. A substantial time investment is now needed to design novel molecules that match the prevailing criteria, which are gradually moving farther from the optimal values set by traditional Lipinski’s rules<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> and related indices.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> Therefore, we consider this as a manifestation of “chemical singularity”. We draw the analogy (allegory) between a black hole’s gravitational singularity and HTS. At the beginning of the century, HTS technology was maturing, becoming more powerful and commercially accessible for many research organizations. That period, there were a vast number of molecules available within vendor’s collections as a matter condensed around a black hole. For the following 2 decades many of these molecules have been analyzed during different HTS campaigns. In addition, medicinal chemists and specific filters moved a considerable number of compounds beyond drug-like-ness space a priori. As a result, the amount of this real “chemical matter” has reduced significantly and is collapsing toward a “chemical singularity” as a one-dimensional point that contains a huge chemical mass. De facto, for this moment, these molecules are actually beyond the event horizon and a relatively few number of compounds can escape it. Novel compounds have obviously more complex structure and are strikingly different from old chemical entities. Many current drug design programs start with de novo synthesis and it is not surprising that the “chemical matter” cannot be refilled rapidly. Considering this, medicinal chemistry evolution is becoming more apparent. We clearly illustrate it particularly using the number of unique patent records on novel leading compounds ordered by priority date (vide infra) and have made a conscious effort to explore this hypothesis via a broad statistical analysis using machine-learning techniques. We revealed that sp<sup>3</sup> index, routinely applied for different goals in drug design, in many cases can lead to incorrect results and cannot be used for describing medicinal chemistry evolution due to many reasons listed below. Moreover, several other metrics for assessing molecular complexity have obvious limitations. The main drawback of sp<sup>3</sup> index (Fsp<sup>3</sup>) introduced by Lovering is that the spatial geometry, composition, and complexity of structures especially for state-of-the-art molecules are not considered at all because these nontrivial features cannot be reflected a priori using only the portion of sp<sup>3</sup>-hybridized carbon atoms.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> Moreover, this count does not reflect true 3D complexity of molecules and their synthetic feasibility. For instance, testosterone, nonadecenoic acid, and a relatively novel cyclobutyl-[(3a<i>R</i>,6a<i>R</i>)-3a-(3-methyl[1,2,4]oxadiazol-5-yl)-5-(tetrahydropyran-4-yl)hexahydropyrrolo[3,4-<i>c</i>]pyrrol-2-yl]methanone (available in the ChemDiv collection since 2018) have the same Fsp<sup>3</sup> value of 84.21, but they are undoubtedly different from the medicinal chemistry point of view. Moreover, as for any statistical analysis where these features should be taken into account properly, e.g., for the investigation of how nonplanarity of molecules influences clinical success or lipophilicity as well as medicinal chemistry evolution, simple sp<sup>3</sup> count can lead to faulty results because there are a large number of compounds of natural origin, including their derivatives (see discussion section), and the majority of these molecules were launched into the market many years ago, for instance, cephalotaxines, nucleosides, oligopeptides, penicillin, or macrolide antibiotics. In this case, a thorough preprocessing procedure of a reference database is of high importance for reducing the number of very similar nature-based compounds to overcome an invalid statistical inference and incorrect interpretation. Lovering and colleagues used the GV KBIO database as a reference data set. This base was not preprocessed, and all compounds reported prior to 1980 were simply removed from the study (more than 1000 molecules, 46% of the whole database). Authors speculated that highly saturated molecules were more likely to succeed in drug discovery pipeline. However, if duplicates, prodrugs and molecules, especially of natural origin, with high similarity in structure, e.g., “me-too drugs”, were not excluded, the obtained results seem rather cursory. Several papers addressing the complexity of organic structures, including nature-like molecules, have been published earlier.<a onclick="showRef(event, 'ref17 ref18 ref19 ref20 ref21 ref22 ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref17 ref18 ref19 ref20 ref21 ref22 ref23 ref24">(17−24)</a> However, all these approaches suffer from multiple limitations and have not been applied to estimate medicinal chemistry evolution. For example, Allu and Oprea proposed synthetic and molecular complexity index (SMCM) calculated based on atomic electronegativities and bond parameters for the assessment of molecular complexity and synthetic feasibility.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> A medium-sized subset (261 thousand small-molecule compounds with MW < 700) selected from the ChemDiv in-house collection was used as a reference database.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> The authors also provided a comparative analysis between SMCM, TWC (total walk count) by Rucker, and Barone’s complexity (BC) metrics and demonstrated that, in general, SMCM outperformed TWC and BC. However, after a thorough inspection, we have concluded that the claims by the authors are rather hasty and superficial. For example, the authors emphasized that high SMCM values negatively correlate with synthetic feasibility and positively correlate with molecular complexity, but this conclusion is highly questionable. They particularly noted that 1,2-bis(2-dimethylaminoethoxy)ethane (SMCM = 17.03) can be easily synthesized due to its symmetry,<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> in contrast to thiophen-3-ylmethylamine (SMCM = 11.41)<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> or also symmetric tris-pyridyl-1,2,4-triazole (SMCM = 47.45)<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> or <i>N</i>-{2-[4-(6-acetylamino-4-phenylquinazolin-2-yl)phenyl]-4-phenylquinazolin-6-yl}acetamide (SMCM = 84.722)<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> as well as {2-[3,5-bis-(2-dimethylaminoethyl)[1,3,5]triazinan-1-yl]ethyl}dimethylamine (SMCM = 26.75).<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> However, all these compounds can be readily synthesized via relatively simple reactions under mild conditions although they have very different SMCM scores. With regard to molecular complexity, SMCM is also unable to translate this feature properly. For example, 2-oxo-1,2-dihydrobenzo[<i>cd</i>]indole-6-sulfonic acid biphenyl-4-ylamide and (<i>S</i>)-squalene-2,3-epoxide both have similar SMCM values and molecular weight, 65.6 (MW = 400.4) and 57.8 (MW = 426.7), respectively, but these compounds are radically different in their chemical nature, atomic composition, and complexity.</div><div class="NLM_p">Therefore, one of the main goals of this paper is to define a robust function that can discriminate between old and novel drug-like compounds from major pharmaceutical enterprises, thereby reflecting the evolution of medicinal chemistry. Prior to the study, we closely inspected the structures of compounds by large pharmaceutical companies from patent literature mainly focusing on novel fragments that were appearing in structures through the years. On the basis of the complexity of these moieties, we selected and prioritized simple descriptors that play a key role in evolutionary driven transformation of molecular architecture and topology. After statistical analysis, we validated several equations to achieve the best coverage and prediction ability. Considering a range of issues and evident limitations of different metrics proposed previously for assessing the complexity of a structure, we paid special attention to the possibility of translation of two-dimensional features, e.g., the number of <i>spiro</i> and chiral atoms, cyclic and linear carbon atoms that are sp<sup>3</sup> hybridized as well as branching score and aromatic moieties, into 3D complexity and molecular framework. After multiple attempts, a superior function was found, and its prediction power was critically estimated using different and well-preprocessed data sets.</div><div class="NLM_p">On the basis of the issues raised above, we have investigated the major trends in medicinal chemistry and drug design (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) with a focus on the evolution of the chemical structures of drugs that have been investigated in clinics, launched in the pharmaceutical market, or claimed in patent records. For the first time, we suggest “MCE-18” as a new molecular descriptor that can effectively score structures by their novelty and current lead potential in contrast to simple and in many cases false-positive sp<sup>3</sup> index, and given by the following equation:<span class="NLM_disp-formula" id="ueq1"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/medium/jm-2019-00004q_m001.gif" alt="" id="_i3" /></img></span>where AR is the presence of an aromatic or heteroaromatic ring (0 or 1), NAR is the presence of an aliphatic or a heteroaliphatic ring (0 or 1), CHIRAL is the presence of a chiral center (0 or 1), SPIRO is the presence of a <i>spiro</i> point (0 or 1), sp<sup>3</sup> is the portion of sp<sup>3</sup>-hybridized carbon atoms (from 0 to 1), Cyc is the portion of cyclic carbons that are sp<sup>3</sup> hybridized (from 0 to 1), Acyc is a portion of acyclic carbon atoms that are sp<sup>3</sup> hybridized (from 0 to 1), and Q<sup>1</sup> is the normalized quadratic index.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/medium/jm-2019-00004q_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/large/jm-2019-00004q_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Simplified illustration of the evolution of medicinal chemistry.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/large/jm-2019-00004q_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00004&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">MCE-18 can be readily applied for the assessment of the novelty of pharmacologically relevant molecules and may help researchers in designing new chemical entities that have great potential in modern drug design. This index can also be effectively used for profiling HTS focused libraries and for the prioritization of molecules. Moreover, compounds with optimal MCE-18 values presumably have more reliable IP positions.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i4">Materials and Methods</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_68033" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_68033" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Database Preprocessing and Normalization</h3><div id="sec2_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i6" class="anchor-spacer"></div><h4 class="article-section__title" id="_i6"> Patent Database</h4><div class="NLM_p last">To avoid reagents and intermediates, we extracted only unique structures from pharmaceutical patent records by major pharmaceutical companies that are available in the Clarivate Analytics Integrity database under “Patents”.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> All these molecules were claimed to have the best activity (lead compounds) and were not found among the reactants or reagents. Since we mostly focus on analyzing only new entities, each structure was assigned to the earliest year (priority date) in which it appeared. The top 23 pharmaceutical companies were selected for this analysis<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> (see the <a href="/doi/suppl/10.1021/acs.jmedchem.9b00004/suppl_file/jm9b00004_si_001.pdf" class="ext-link">Supporting Information Table S1</a>). Patent records on new drug substances by these players with priority dates from 1950 to 2018 were downloaded. The filtration procedure to isolate only the unique reports resulted in 30 153 records. Then, the reports were redownloaded as structure-data files (sdf) from the Clarivate Analytics Integrity database “Drugs & Biologics” and were preprocessed using ChemoSoft software. Thus, the core “Patent year” field was assigned for each structure, and then all the items were merged into one sdf, which was then subjected to the primary preparation process: charged structures were redrawn and presented in their neutral form, counterions were deleted, and errors in structures were manually corrected. Soft MCFs were applied to the database, resulting in the elimination of metal-, silicon-, and phospho-organic compounds; isotopes; etc. The first preprocessing step excluded 957 structures. Then, 281 samples were deleted because of disconnections between the “Patent year” and the launched date. In other words, the drug was launched prior to the patent record by a company not in the group evaluated herein. Structures that met the following criteria were also excluded: MW > 1200, more than 20 oxygen atoms, or more than 10 fluorine atoms, as well as those with no carbon atoms. Duplicates were removed, and the database was thus reduced to 28 161 samples. To increase the compound diversity and decrease the number of overrepresented chemotypes, a clustering procedure was carried out in ChemoSoft. A routine Tanimoto metric was used as a measure of 2D similarity. Compounds with similarity scores over 0.5 were assigned to the same cluster (at least 10 records per cluster). Then, the structures in each cluster were ranked by their diversity coefficients, and the samples with the highest 10% (for clusters containing fewer than 200 records) or 5% (for clusters containing more than 200 records) of diversity scores as well as all the compounds not assigned to a cluster were retained. The resulting database contained 24 232 structures. As a result, preprocessing and subsequent normalization led to the elimination of a total of ∼4000 structures (14%). The excluded structures might be meaningful; however, it was very important to execute the listed operations to achieve an adequate statistical outcome and to allow reliable in silico modeling. For instance, a great bit of “chemical turbulence” was introduced by the abundance of old chemotypes, e.g., morphines, penicillins, cephalosporins, steroids, macrolides, tetracyclines, and fluoroquinolones.</div></div><div id="sec2_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i7" class="anchor-spacer"></div><h4 class="article-section__title" id="_i7"> Launched Drugs by Year</h4><div class="NLM_p last">The raw collection of 4385 launched drugs was obtained from the Clarivate Analytics Integrity “Drugs & Biologics” database. We found that there was no information about the issue date associated with 1312 drugs; therefore, these records were excluded. The remaining 3073 records were redownloaded from “Drugs & Biologics” as an sdf. Of the remaining records, 1383 did not contain any information on the chemical structure (biologics); therefore, they were excluded as well. The earliest date of approval/registration/launch was assigned to each sample. This information was accessible within the milestone table. As a result, 1690 records with their primary launch date were included in the final data set with no duplicates. Then, the database was preprocessed as described above, resulting in the elimination of another 14 records. Tanimoto-based clusterization was then performed by analogy to the procedure described above. Molecules with high structural similarity were deleted from each cluster (the minimum number of samples per cluster was 10). All compounds not assigned to a cluster were retained. Preprocessing and normalization yielded 1370 unique structures.</div></div><div id="sec2_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i8" class="anchor-spacer"></div><h4 class="article-section__title" id="_i8"> Preclinical Evaluation, Clinical Trials, and Launched Drugs</h4><div class="NLM_p last">The same data source was used to compare this data set. All the molecules that served as lead compounds (preclinical trials), that were clinically evaluated as drug candidates (phase I–III), and that were launched as drugs were downloaded as an sdf. Only the highest milestone point achieved by a sample was considered. For each category, preprocessing and normalization were performed following the procedure described above. As a result, the final database contained ∼30 000 preclinical, 1678 phase I, 1837 phase II, 464 phase III, and 1370 launched compounds.</div></div><div id="sec2_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i9" class="anchor-spacer"></div><h4 class="article-section__title" id="_i9"> Specific Subsets</h4><div class="NLM_p last">The classification ability of MCE-18 function toward different target families/classes was also evaluated using the following subsets. We have selected 13 265 records from the entire preprocessed patent database with the activity claimed against GPCRs (8249 molecules), protein kinases (3397 compounds), and proteases (1619 molecules), a total of 13 265 molecules (∼55% of all the compounds included in the parent data set). The key mechanism of action (MoA) was then assigned for each record using Integrity Database. Then, we have isolated molecules with MCE-18 values of ≥50 to investigate their populations among different target classes. As a result, the data sets of 1273 protease targeting molecules, 4616 GPCRs ligands, and 2103 protein kinase inhibitors were prepared (a total of 7992 compounds). For the remaining molecules in the database other targets were reported, including nuclear factors, hydrolases, and transferases. To estimate the classification power of MCE-18 toward drugs launched into the market or entered in clinical trials, we have collected the database containing molecules with the highest phase tag (phase I–III, preregistered, recommended approval, registered or launched). The related milestone date was also assigned for each sample. The basic cellular mechanism of action was added to each molecule: drugs targeting proteases, drugs acting on GPCRs, or drugs targeting protein kinases. All the records saved were then sorted by milestone date, and duplicates were eliminated using drug name field. After the follow-up preprocessing procedure, 810 GPCR ligands, 459 kinase inhibitors, and 171 protease targeting molecules were selected and used for the analysis.</div></div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Molecular Descriptors</h3><div class="NLM_p last">For all the compounds in the database, key molecular descriptors were calculated using ChemoSoft<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> and SmartMining software.<a onclick="showRef(event, 'ref34 ref35'); return false;" href="javascript:void(0);" class="ref ref34 ref35">(34,35)</a> These descriptors include MCE-18, MW (molecular weight), LogP (lipophilicity, the calculated partition coefficient in the 1-octanol/water system), LogS<sub>w</sub> (predicted solubility in water), PSA (polar surface area, Å<sup>2</sup>), HBD (number of potential H-bond donors), HBA (number of potential H-bond acceptors), and SS (common electrotopological index) as well as AR, NAR, CHIRAL, SPIRO, NCSPTR, Q<sup>1</sup>, and sp<sup>3</sup>, which are defined below. For the selected descriptors, bivariate Student’s <i>t</i>-values were calculated (see the <a href="/doi/suppl/10.1021/acs.jmedchem.9b00004/suppl_file/jm9b00004_si_001.pdf" class="ext-link">Supporting Information</a>). We used the MCE-18, MW, PSA, HBD, HBA, LogS<sub>w</sub>, and SS values for in silico modeling, and this set of parameters were selected in accordance with their theoretical impact on the studied phenomenon.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Kohonen Self-Organizing Mapping</h3><div class="NLM_p last">Depending on the training set, map size 20 × 20 or 15 × 15 2D representations; learning epochs, 2000; initial learning rate, 0.3 (linear decay); initial learning radius, 15 or 10 (linear decay); initial weight coefficients, random distribution (not normalized); activation function, Gaussian; and the winning neuron was determined using Euclidean metrics. After the training process was completed, the areas populated by different classes of compounds were highlighted. We generally observed that samples from different categories were mainly located in distinct regions within the same map. Neurons were then prioritized based on the following privileged factor (PF): <i>N</i><sup><i>i</i></sup><sub>C1</sub>(%)/<i>N</i><sup><i>i</i></sup><sub>C2</sub>(%), where <i>N</i><sup><i>i</i></sup><sub>C1</sub> is the percent of compounds from class 1 located in the <i>i-</i>th neuron, while <i>N</i><sup><i>i</i></sup><sub>C2</sub> is the percent of other molecules from class 2 located in the same neuron and vice versa. A PF value greater than 1 was used as a threshold to assign neurons to one of the two classes. There were no or few “death” neurons within the maps.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Nonlinear Sammon Mapping</h3><div class="NLM_p last">To investigate how new molecules cover RO5 space, we performed Sammon-based mapping.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> The main goal of this algorithm is the approximation of local geometric and topological relationships hidden in the input chemical space and to visually present these relationships in a 2D- or 3D-dimensional plot. The fundamental idea of this approach is to perform the dimensionality reduction of the initial data set into fewer dimensions, and in this respect, this technique resembles self-organizing map (SOM) and multidimensional scaling. However, unlike other algorithms, classical Sammon nonlinear mapping allows scientists to construct a projection that reflects global topographic relationships as pairwise distances between all objects within the whole range of input vector samples. For modeling, we used the same set of molecular descriptors that were used for Kohonen SOM. Structure similarity was not considered in the study. Euclidean distance was selected as the similarity metric, the stress threshold was 0.01, the iteration number was 1000, and the optimization step was 0.1.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i13">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_66051" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_66051" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec3_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> A Brief Insight into the History and Conception</h3><div class="NLM_p last">Prior to the main discussion, we should provide a compact overview of the seminal research papers that address molecular properties and complexity based on descriptors and structural patterns. In a pioneering study, Bemis and Murcko<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> showed that more than half of drug substances (a total of 5K structures) contained slightly more than 30 unique 2D molecular frameworks. These moieties included mostly planar aromatic or fused aromatic systems, natural-product-derived fragments (e.g., steroids and penicillins), and “naive” cycloaliphatic fragments. Since then, many papers have focused on drug features and the proximal chemical space. In 2010, Wang and Hou revealed only minor changes in the structures of approved and clinically evaluated drugs in terms of their molecular frameworks.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> They found that 50 fragments covered, on average, 50% of the accumulated drug database (more than 8K entities). Recently, Taylor and colleagues analyzed all linear combinations of ring systems among FDA-approved drugs.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> Their results showed that only 1% of the theoretically possible combinations appeared in commercially available libraries. Other authors reached similar conclusions<a onclick="showRef(event, 'ref3 ref40 ref41'); return false;" href="javascript:void(0);" class="ref ref3 ref40 ref41">(3,40,41)</a> and highlighted that the field of medicinal chemistry has primarily focused on a limited area within the available drug-like chemical space and uses a relatively small number of organic reactions. Brown and Boström<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> have recently investigated this issue and have demonstrated that only a small number of organic reactions dominate contemporary practice and account for more than 80% of the synthetic routes used in drug discovery. Clemons and co-workers showed that the structural complexity of small molecules of different origins was well correlated with their protein-binding profile.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> In contrast to the sp<sup>3</sup>-oriented approach, Kombo et al. revealed that shape-based 3D descriptors, e.g., shadow indices and radius of gyration, strongly influence the off-target activity profile.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> The common theme of the above-mentioned papers is the increasing role of spatial complexity in modern drug discovery. This complexity can be achieved by introducing new nontrivial cores during development. However, these moieties are not common, and it can be quite difficult to conjugate them with existing fragments. This issue was recently directly addressed in a comprehensive review by Schneider and colleagues.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> The authors critically assessed the toolbox of medicinal chemists based on a large number of reactions (>3 million) taken from U.S. patents published from 1976 to 2015. All the mined reactions were categorized into widely used types, e.g., Wittig olefination, Buchwald–Hartwig amination, Mitsunobu reaction, Suzuki–Miyaura coupling, or Sonogashira coupling. As a result, they concluded that the medicinal chemistry toolkit generally operates with a tight set of standard reactions. They estimated that little more than 120 different types of reactions accounted for 95% of the total reactions, and this percentage has increased over the past 3 decades. In addition, the authors performed a thorough analysis of more than 385K molecules claimed in pharmaceutical patent records as final reaction products (sorted by priority date) in terms of their key molecular properties. They revealed that the number of unique products over time followed a trend similar to that seen in the reactions: rapid growth was observed from 2005 to 2010 followed by a plateau up to 2015. Substructure searching showed that the overall number of cyclic moieties, e.g., heteroaromatic rings, aliphatic rings, and aliphatic heterocycles, gradually increased from a mean of 2.6 in 1976 to 3.5 in 2015 in contrast to aromatic carbocycles, such as phenyl, biphenyl, or naphthalene. The authors directly attributed this finding to Suzuki-type coupling reactions (1979) and their popularity. The more important finding was that most of the increase was due to a growth in the number of aliphatic heterocycles, e.g., morpholine, piperazine, and pyrrolidine rings, which can substantially improve water solubility. The authors also noticed an increase in the total number of saturated sp<sup>3</sup>-rich rings; however, they did not provide a detailed analysis of the structural and spatial complexity of these fragments, probably because of a relatively simple array of fingerprints implemented in RDKit. The authors also investigated the evolution of key molecular properties (MW, LogP, TPSA, sp<sup>3</sup> count, HBA, HBD, and FRB) over time. They showed that the reaction products disclosed in pharmaceutical records were becoming larger, more lipophilic, more rigid, and more soluble over the studied period. For instance, the average molecular weight and lipophilicity of the target molecules claimed in 1976 were 331 Da and 3.1, respectively, while in 2015, these values increased overall by 24% (MW = 409 Da) and 16% (LogP = 3.6). In 2011, Walters and co-workers<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> uncovered similar but less striking trends, which might be a result of pronounced “evolution slope” or of the nonpreprocessed data set used by the Schneider team. The polar surface area of the products increased by more than 15% over the studied period despite a simultaneous increase in lipophilicity, whereas the sp<sup>3</sup> carbon index, heteroatomic index, HBA, and HBD did not show consistent trends. The authors attributed the drastic decrease in the fraction of free-rotatable bonds and stagnation (or slight decrease) in the sp<sup>3</sup> term to the increase in rigidity. However, the growing number of more rigid molecular frameworks can be associated with the increasing number of sp<sup>3</sup>-rich rings and chiral centers. Moreover, the above-described results were obtained using a nonpreprocessed data set; in particular, the authors did not reduce the number of compounds with high structural similarity within overloaded clusters. Notably, the preprocessing procedure is of high importance for valid statistical analysis and computational modeling because there are many clusters containing a large number of natural products, such as triterpenoid and steroidal saponins, alkaloids, glycosides, glucans, oligosaccharides, fatty acids, benzophenanthridine alkaloids, macrolides, catechins, ibogamine, podophyllotoxins, anthracyclines, aporphine polypeptides, detergents, and penicillin-based molecules. All these compounds and those with similar structures contribute substantially to the sp<sup>3</sup> index, leading to misleading conclusions and false-positive results. Therefore, we speculate that, unlike the hypothesis formulated by the Schneider group, the increase in rigidity is primarily associated with the evolution of synthetic approaches that have allowed scientists to access novel <i>spiro</i>-compounds, annulated derivatives, and novel building blocks with higher degrees of 3D complexity with an increasing number of cyclic sp<sup>3</sup> carbons vs acyclic carbons. The same authors indicated that although the “average reaction product” generally met the drug-like criteria introduced by Lipinski for oral drugs, the number of products violating the RO5 has grown over the years. For example, within 1985–2005 a significant increase in the number of unique products violating lipophilicity rule (RO5) and possessing LogP > 5 was observed (+15%, slope value of 0.78) followed by decrease until 2015 (−5%, slope value of −0.56). During 1982–2010, the number of unique products with molecular weight beyond RO5 (MW > 500) grew as well (+16.5%, slope value of 0.6), then a slight decay was observed. However, this decrease in MW can be attributed to an insufficient statistical data to that moment (2015 year) because the paper by Schneider was published at the beginning of 2016 year. This issue has recently been described in more detail by Doak and co-workers.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> Using different sources, the authors collected a database of 485 drugs (phase I–III or launched) that did not follow the RO5 criteria. Notably, they found that de novo designed molecules accounted for a substantial portion (43%) of the entities included in the data set, followed by natural products (26%) and peptides and peptide mimetics (26%). Almost 65% of de novo designed compounds were targeted for oral administration. This clearly indicates that successful oral drugs can be discovered within the chemical space outside of the RO5 criteria or beyond the Lipinski landscape completely. Through the years, the number of drugs brought to the market with MW > 500 Da steadily increased up to 2010. A high number of oral de novo designed drugs were approved during 1995–2010, whereas natural products were the most common type of approved drugs prior to 1995. Recently, Michael Shultz at Novartis coined the term “Lipinski’s anchor” and critically argued that the standard drug-like-ness rules do not stand the test of time, especially the parameters of MW and HBA, which have risen considerably over the past 20 years.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> He concludes that the observed trend and reported data call into question the hypothesis that a “drug-like El Dorado” even exists. We have shown that this tendency has become more evident and exciting. Michael Shultz also discusses the likelihood of reaching a new golden age of drug discovery. In particular, he notices that the sharp increase in drug approvals in 1996 was associated with the Prescription Drug User Fee Act (PDUFA) of 1992 in response to the global AIDS epidemic. Hypothetically, many of these drugs could have been approved prior to this time point, and as the author states, “this ‘surge’ actually distorts what was a steady increase in productivity since the 1980s.”<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> The past decade has been regarded as the most prolific era in drug discovery considering the large number of registered NCEs. Indeed, during this period, there were more than 200 oral NCEs, which amounts to nearly a third of all approved oral drugs. In contrast to the results by Schneider (vide supra), Michael Shultz suggests that the parameters associated with MW, e.g., FRBs and HBA, have also significantly increased, whereas no considerable changes have been observed in HBD. With respect to TPSA, both authors highlight meaningful growth. Notably, none of the authors correctly addressed the sp<sup>3</sup> characteristics of new molecules using simple sp<sup>3</sup> index or poorly preprocessed data sets. Depending on the method used, Michael Shultz also found a slight positive correlation with the calculated LogP values.</div></div><div id="sec3_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Main Discussion</h3><div id="sec3_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i16" class="anchor-spacer"></div><h4 class="article-section__title" id="_i16"> MCE-18</h4><div class="NLM_p">For the past decade, PPI research has progressed substantially and has resulted in many druggable interactions, including p53/MDM2 and Bcl-2, GLP1R, Hsp70, NS5A, FAK, β-catenin, and BRDs (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>).<a onclick="showRef(event, 'ref49 ref50'); return false;" href="javascript:void(0);" class="ref ref49 ref50">(49,50)</a> Javier Luque recently published a comprehensive review describing the evolution of CADD techniques within the field of PPI inhibitors.<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> Many authors note that PPIs lie far beyond the prevailing drug design paradigm because molecules that may disturb such interaction are significantly different from classic drug templates and chemotypes. A large number of innovative synthetic scaffolds with notable 3D complexity have been developed in recent years, thereby allowing scientists to design nontrivial molecules with a high degree of spatial diversity not only as PPI inhibitors but also for other applications, for example, as kinase inhibitors.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/medium/jm-2019-00004q_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/large/jm-2019-00004q_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Representative examples of PPI inhibitors that reached the market or are being evaluated in clinics.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/large/jm-2019-00004q_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00004&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">During the past decade, PPI inhibitors have attracted a great deal of attention in the field of drug design and development<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> since a large number of PPIs have already been discovered and many of them are now considered as attractive biological targets. From the classical medicinal chemistry point of view, novel compounds should meet several trivial criteria, e.g., Lipinski’s RO5 or RO3 to reach success in clinical trials as peroral drug candidates. However, PPI inhibitors are frequently much larger and more bulky than traditional drugs. As a rule, they are equipped with a balanced number of HBA, HBD, and hydrophobic anchors well-adjusted to crucial hot-spots observed along a PPI interface. On the other hand, teachable examples of successfully launched or developing PPI inhibitors (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>) that are not consistent with RO5 parameters have shown high MCE-18 scores. In these cases, medicinal chemists are faced with awkward challenges and need to improve instruments and techniques for discovering new drugs beyond Lipinski’s rules. The overarching aim of these scaffolds is to arrange the attached substituents to achieve the best site interactions and hot-spot matching. Frequently, these building blocks or skeletons contain an intricate arrangement of sp<sup>3</sup> atoms that forms a spatially predefined and rigid core equipped with appropriate diversification points. Considering that a simple sp<sup>3</sup> index that corresponds to the number of sp<sup>3</sup>-hybridized carbon atoms provides a high rate of false-positive results, we addressed this issue with the MCE-18 descriptor.</div><div class="NLM_p">This new equation reflects the quality and complexity of the whole molecular framework and content of sp<sup>3</sup> carbons. As shown below, the quadratic index increases with the priority date assigned for a patent record (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>) as well as with the launch date (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). This Zagreb-based index encodes the degree of branching of a structural framework in terms of vertex degree and implicitly accounts for 2D complexity. The AR term was included in the equation because Shinji Soga and co-workers<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> demonstrated that Phe, Trp, and Tyr are more common in the amino acid composition at ligand-binding sites of a protein, and they have been implicated in many PPIs but not in surface interactions. Moreover, this term allows us to isolate branched aliphatic and cycloaliphatic substances, e.g., fatty acids, steroid-based structures, macrolides, and polycarbohydrates, from the pool of new molecules as their structures are qualitatively different from those of “old style” compounds. Cumulatively, all these terms contribute significantly to the evolution of medicinal chemistry and to the sp<sup>3</sup> quality from both theoretical and statistical perspectives.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Mean Values of the Molecular Descriptors Calculated for the Structures Disclosed in Patent Records by Major Pharmaceutical Companies over the Years</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">no.</th><th class="colsep0 rowsep0" align="center">MD</th><th class="colsep0 rowsep0" align="center">1950–1983</th><th class="colsep0 rowsep0" align="center">1984–1990</th><th class="colsep0 rowsep0" align="center">1991–1997</th><th class="colsep0 rowsep0" align="center">1998–2004</th><th class="colsep0 rowsep0" align="center">2005–2011</th><th class="colsep0 rowsep0" align="center">2012–2018</th><th class="colsep0 rowsep0" align="center">increase factor</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">MCE-18</td><td class="colsep0 rowsep0" align="left">43.2</td><td class="colsep0 rowsep0" align="left">50.1</td><td class="colsep0 rowsep0" align="left">56.3</td><td class="colsep0 rowsep0" align="left">57.1</td><td class="colsep0 rowsep0" align="left">64.9</td><td class="colsep0 rowsep0" align="left">75.9</td><td class="colsep0 rowsep0" align="left">1.12</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">MW</td><td class="colsep0 rowsep0" align="left">345.4</td><td class="colsep0 rowsep0" align="left">382.1</td><td class="colsep0 rowsep0" align="left">424.7</td><td class="colsep0 rowsep0" align="left">430.6</td><td class="colsep0 rowsep0" align="left">453.0</td><td class="colsep0 rowsep0" align="left">471.8</td><td class="colsep0 rowsep0" align="left">1.07</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">AR</td><td class="colsep0 rowsep0" align="left">0.86</td><td class="colsep0 rowsep0" align="left">0.90</td><td class="colsep0 rowsep0" align="left">0.95</td><td class="colsep0 rowsep0" align="left">0.98</td><td class="colsep0 rowsep0" align="left">0.99</td><td class="colsep0 rowsep0" align="left">0.99</td><td class="colsep0 rowsep0" align="left">1.03</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">NAR</td><td class="colsep0 rowsep0" align="left">0.72</td><td class="colsep0 rowsep0" align="left">0.71</td><td class="colsep0 rowsep0" align="left">0.75</td><td class="colsep0 rowsep0" align="left">0.72</td><td class="colsep0 rowsep0" align="left">0.77</td><td class="colsep0 rowsep0" align="left">0.83</td><td class="colsep0 rowsep0" align="left">1.03</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">5</td><td class="colsep0 rowsep0" align="left">CHIRAL</td><td class="colsep0 rowsep0" align="left">0.51</td><td class="colsep0 rowsep0" align="left">0.55</td><td class="colsep0 rowsep0" align="left">0.53</td><td class="colsep0 rowsep0" align="left">0.48</td><td class="colsep0 rowsep0" align="left">0.49</td><td class="colsep0 rowsep0" align="left">0.61</td><td class="colsep0 rowsep0" align="left">1.04</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">6</td><td class="colsep0 rowsep0" align="left">SPIRO</td><td class="colsep0 rowsep0" align="left">0.017</td><td class="colsep0 rowsep0" align="left">0.031</td><td class="colsep0 rowsep0" align="left">0.027</td><td class="colsep0 rowsep0" align="left">0.025</td><td class="colsep0 rowsep0" align="left">0.048</td><td class="colsep0 rowsep0" align="left">0.061</td><td class="colsep0 rowsep0" align="left">1.36</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">7</td><td class="colsep0 rowsep0" align="left">NCSPTR</td><td class="colsep0 rowsep0" align="left">0.22</td><td class="colsep0 rowsep0" align="left">0.23</td><td class="colsep0 rowsep0" align="left">0.21</td><td class="colsep0 rowsep0" align="left">0.22</td><td class="colsep0 rowsep0" align="left">0.24</td><td class="colsep0 rowsep0" align="left">0.26</td><td class="colsep0 rowsep0" align="left">1.04</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left">Q<sup>1</sup></td><td class="colsep0 rowsep0" align="left">17.5</td><td class="colsep0 rowsep0" align="left">19.6</td><td class="colsep0 rowsep0" align="left">21.8</td><td class="colsep0 rowsep0" align="left">22.7</td><td class="colsep0 rowsep0" align="left">24.9</td><td class="colsep0 rowsep0" align="left">26.8</td><td class="colsep0 rowsep0" align="left">1.09</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">9</td><td class="colsep0 rowsep0" align="left">sp<sup>3</sup></td><td class="colsep0 rowsep0" align="left">38.3</td><td class="colsep0 rowsep0" align="left">39.4</td><td class="colsep0 rowsep0" align="left">35.7</td><td class="colsep0 rowsep0" align="left">33.0</td><td class="colsep0 rowsep0" align="left">33.1</td><td class="colsep0 rowsep0" align="left">34.5</td><td class="colsep0 rowsep0" align="left">0.98</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">10</td><td class="colsep0 rowsep0" align="left">LogP</td><td class="colsep0 rowsep0" align="left">2.6</td><td class="colsep0 rowsep0" align="left">3.1</td><td class="colsep0 rowsep0" align="left">3.5</td><td class="colsep0 rowsep0" align="left">3.7</td><td class="colsep0 rowsep0" align="left">3.6</td><td class="colsep0 rowsep0" align="left">3.3</td><td class="colsep0 rowsep0" align="left">1.05</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">11</td><td class="colsep0 rowsep0" align="left">LogS<sub>w</sub></td><td class="colsep0 rowsep0" align="left">–4.1</td><td class="colsep0 rowsep0" align="left">–4.4</td><td class="colsep0 rowsep0" align="left">–4.7</td><td class="colsep0 rowsep0" align="left">–4.8</td><td class="colsep0 rowsep0" align="left">–4.8</td><td class="colsep0 rowsep0" align="left">–4.8</td><td class="colsep0 rowsep0" align="left">1.03</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">12</td><td class="colsep0 rowsep0" align="left">PSA</td><td class="colsep0 rowsep0" align="left">79.0</td><td class="colsep0 rowsep0" align="left">84.2</td><td class="colsep0 rowsep0" align="left">92.5</td><td class="colsep0 rowsep0" align="left">88.0</td><td class="colsep0 rowsep0" align="left">94.0</td><td class="colsep0 rowsep0" align="left">101.9</td><td class="colsep0 rowsep0" align="left">1.05</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">13</td><td class="colsep0 rowsep0" align="left">HBD</td><td class="colsep0 rowsep0" align="left">1.7</td><td class="colsep0 rowsep0" align="left">1.7</td><td class="colsep0 rowsep0" align="left">2.1</td><td class="colsep0 rowsep0" align="left">1.8</td><td class="colsep0 rowsep0" align="left">1.8</td><td class="colsep0 rowsep0" align="left">2.0</td><td class="colsep0 rowsep0" align="left">1.04</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">14</td><td class="colsep0 rowsep0" align="left">HBA</td><td class="colsep0 rowsep0" align="left">3.0</td><td class="colsep0 rowsep0" align="left">3.3</td><td class="colsep0 rowsep0" align="left">3.4</td><td class="colsep0 rowsep0" align="left">2.8</td><td class="colsep0 rowsep0" align="left">2.8</td><td class="colsep0 rowsep0" align="left">3.0</td><td class="colsep0 rowsep0" align="left">1.01</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">15</td><td class="colsep0 rowsep0" align="left">SS</td><td class="colsep0 rowsep0" align="left">56.7</td><td class="colsep0 rowsep0" align="left">62.2</td><td class="colsep0 rowsep0" align="left">71.7</td><td class="colsep0 rowsep0" align="left">73.5</td><td class="colsep0 rowsep0" align="left">77.6</td><td class="colsep0 rowsep0" align="left">82.6</td><td class="colsep0 rowsep0" align="left">1.08</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="2" align="left">entities</td><td class="colsep0 rowsep0" align="left">765</td><td class="colsep0 rowsep0" align="left">2179</td><td class="colsep0 rowsep0" align="left">2741</td><td class="colsep0 rowsep0" align="left">6521</td><td class="colsep0 rowsep0" align="left">8932</td><td class="colsep0 rowsep0" align="left">3094</td><td class="colsep0 rowsep0" align="left">1.64</td></tr></tbody></table></div></div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Mean Values of the Molecular Descriptors Calculated for the Launched Drugs</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">no.</th><th class="colsep0 rowsep0" align="center">MD</th><th class="colsep0 rowsep0" align="center">1910–1959</th><th class="colsep0 rowsep0" align="center">1960–1969</th><th class="colsep0 rowsep0" align="center">1970–1979</th><th class="colsep0 rowsep0" align="center">1980–1989</th><th class="colsep0 rowsep0" align="center">1990–1999</th><th class="colsep0 rowsep0" align="center">2000–2005</th><th class="colsep0 rowsep0" align="center">2006–2011</th><th class="colsep0 rowsep0" align="center">2012–2018</th><th class="colsep0 rowsep0" align="center">increase factor</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">MCE-18</td><td class="colsep0 rowsep0" align="left">37.2</td><td class="colsep0 rowsep0" align="left">41.2</td><td class="colsep0 rowsep0" align="left">40.2</td><td class="colsep0 rowsep0" align="left">44.1</td><td class="colsep0 rowsep0" align="left">46.4</td><td class="colsep0 rowsep0" align="left">50.5</td><td class="colsep0 rowsep0" align="left">52.8</td><td class="colsep0 rowsep0" align="left">55.4</td><td class="colsep0 rowsep0" align="left">1.06</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">MW</td><td class="colsep0 rowsep0" align="left">302.3</td><td class="colsep0 rowsep0" align="left">332.2</td><td class="colsep0 rowsep0" align="left">338.4</td><td class="colsep0 rowsep0" align="left">363.6</td><td class="colsep0 rowsep0" align="left">376.5</td><td class="colsep0 rowsep0" align="left">387.3</td><td class="colsep0 rowsep0" align="left">402.8</td><td class="colsep0 rowsep0" align="left">430.5</td><td class="colsep0 rowsep0" align="left">1.05</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">AR</td><td class="colsep0 rowsep0" align="left">0.74</td><td class="colsep0 rowsep0" align="left">0.82</td><td class="colsep0 rowsep0" align="left">0.83</td><td class="colsep0 rowsep0" align="left">0.78</td><td class="colsep0 rowsep0" align="left">0.79</td><td class="colsep0 rowsep0" align="left">0.76</td><td class="colsep0 rowsep0" align="left">0.80</td><td class="colsep0 rowsep0" align="left">0.90</td><td class="colsep0 rowsep0" align="left">1.03</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">NAR</td><td class="colsep0 rowsep0" align="left">0.62</td><td class="colsep0 rowsep0" align="left">0.67</td><td class="colsep0 rowsep0" align="left">0.63</td><td class="colsep0 rowsep0" align="left">0.65</td><td class="colsep0 rowsep0" align="left">0.70</td><td class="colsep0 rowsep0" align="left">0.67</td><td class="colsep0 rowsep0" align="left">0.72</td><td class="colsep0 rowsep0" align="left">0.69</td><td class="colsep0 rowsep0" align="left">1.02</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">5</td><td class="colsep0 rowsep0" align="left">CHIRAL</td><td class="colsep0 rowsep0" align="left">0.57</td><td class="colsep0 rowsep0" align="left">0.45</td><td class="colsep0 rowsep0" align="left">0.52</td><td class="colsep0 rowsep0" align="left">0.60</td><td class="colsep0 rowsep0" align="left">0.62</td><td class="colsep0 rowsep0" align="left">0.64</td><td class="colsep0 rowsep0" align="left">0.59</td><td class="colsep0 rowsep0" align="left">0.48</td><td class="colsep0 rowsep0" align="left">0.99</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">6</td><td class="colsep0 rowsep0" align="left">SPIRO</td><td class="colsep0 rowsep0" align="left">0.008</td><td class="colsep0 rowsep0" align="left">0.010</td><td class="colsep0 rowsep0" align="left">0</td><td class="colsep0 rowsep0" align="left">0.014</td><td class="colsep0 rowsep0" align="left">0.015</td><td class="colsep0 rowsep0" align="left">0.015</td><td class="colsep0 rowsep0" align="left">0.015</td><td class="colsep0 rowsep0" align="left">0.010</td><td class="colsep0 rowsep0" align="left">0.00</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">7</td><td class="colsep0 rowsep0" align="left">NCSPTR</td><td class="colsep0 rowsep0" align="left">0.21</td><td class="colsep0 rowsep0" align="left">0.23</td><td class="colsep0 rowsep0" align="left">0.22</td><td class="colsep0 rowsep0" align="left">0.24</td><td class="colsep0 rowsep0" align="left">0.24</td><td class="colsep0 rowsep0" align="left">0.24</td><td class="colsep0 rowsep0" align="left">0.27</td><td class="colsep0 rowsep0" align="left">0.22</td><td class="colsep0 rowsep0" align="left">1.01</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left">Q<sup>1</sup></td><td class="colsep0 rowsep0" align="left">14.6</td><td class="colsep0 rowsep0" align="left">16.3</td><td class="colsep0 rowsep0" align="left">16.6</td><td class="colsep0 rowsep0" align="left">17.1</td><td class="colsep0 rowsep0" align="left">17.9</td><td class="colsep0 rowsep0" align="left">18.9</td><td class="colsep0 rowsep0" align="left">20.2</td><td class="colsep0 rowsep0" align="left">22.2</td><td class="colsep0 rowsep0" align="left">1.06</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">9</td><td class="colsep0 rowsep0" align="left">sp<sup>3</sup></td><td class="colsep0 rowsep0" align="left">46.1</td><td class="colsep0 rowsep0" align="left">42.4</td><td class="colsep0 rowsep0" align="left">43.2</td><td class="colsep0 rowsep0" align="left">47.6</td><td class="colsep0 rowsep0" align="left">45.6</td><td class="colsep0 rowsep0" align="left">47.8</td><td class="colsep0 rowsep0" align="left">46.9</td><td class="colsep0 rowsep0" align="left">37.6</td><td class="colsep0 rowsep0" align="left">0.98</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">10</td><td class="colsep0 rowsep0" align="left">LogP</td><td class="colsep0 rowsep0" align="left">1.5</td><td class="colsep0 rowsep0" align="left">2.3</td><td class="colsep0 rowsep0" align="left">2.4</td><td class="colsep0 rowsep0" align="left">1.8</td><td class="colsep0 rowsep0" align="left">1.9</td><td class="colsep0 rowsep0" align="left">1.5</td><td class="colsep0 rowsep0" align="left">2.9</td><td class="colsep0 rowsep0" align="left">2.8</td><td class="colsep0 rowsep0" align="left">1.15</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">11</td><td class="colsep0 rowsep0" align="left">LogS<sub>w</sub></td><td class="colsep0 rowsep0" align="left">–3.6</td><td class="colsep0 rowsep0" align="left">–3.9</td><td class="colsep0 rowsep0" align="left">–4.0</td><td class="colsep0 rowsep0" align="left">–3.8</td><td class="colsep0 rowsep0" align="left">–4.0</td><td class="colsep0 rowsep0" align="left">–3.9</td><td class="colsep0 rowsep0" align="left">–4.4</td><td class="colsep0 rowsep0" align="left">–4.5</td><td class="colsep0 rowsep0" align="left">1.03</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">12</td><td class="colsep0 rowsep0" align="left">PSA</td><td class="colsep0 rowsep0" align="left">75.3</td><td class="colsep0 rowsep0" align="left">75.8</td><td class="colsep0 rowsep0" align="left">78.4</td><td class="colsep0 rowsep0" align="left">91.5</td><td class="colsep0 rowsep0" align="left">94.8</td><td class="colsep0 rowsep0" align="left">98.7</td><td class="colsep0 rowsep0" align="left">92.7</td><td class="colsep0 rowsep0" align="left">97.4</td><td class="colsep0 rowsep0" align="left">1.04</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">13</td><td class="colsep0 rowsep0" align="left">HBD</td><td class="colsep0 rowsep0" align="left">1.7</td><td class="colsep0 rowsep0" align="left">2.1</td><td class="colsep0 rowsep0" align="left">2.1</td><td class="colsep0 rowsep0" align="left">2.2</td><td class="colsep0 rowsep0" align="left">2.2</td><td class="colsep0 rowsep0" align="left">2.2</td><td class="colsep0 rowsep0" align="left">2.2</td><td class="colsep0 rowsep0" align="left">2.1</td><td class="colsep0 rowsep0" align="left">1.03</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">14</td><td class="colsep0 rowsep0" align="left">HBA</td><td class="colsep0 rowsep0" align="left">3.0</td><td class="colsep0 rowsep0" align="left">2.9</td><td class="colsep0 rowsep0" align="left">3.1</td><td class="colsep0 rowsep0" align="left">3.8</td><td class="colsep0 rowsep0" align="left">3.7</td><td class="colsep0 rowsep0" align="left">3.9</td><td class="colsep0 rowsep0" align="left">3.5</td><td class="colsep0 rowsep0" align="left">3.6</td><td class="colsep0 rowsep0" align="left">1.03</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">15</td><td class="colsep0 rowsep0" align="left">SS</td><td class="colsep0 rowsep0" align="left">49.6</td><td class="colsep0 rowsep0" align="left">51.6</td><td class="colsep0 rowsep0" align="left">56.0</td><td class="colsep0 rowsep0" align="left">59.5</td><td class="colsep0 rowsep0" align="left">61.5</td><td class="colsep0 rowsep0" align="left">63.8</td><td class="colsep0 rowsep0" align="left">67.0</td><td class="colsep0 rowsep0" align="left">73.4</td><td class="colsep0 rowsep0" align="left">1.06</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="2" align="left">entities</td><td class="colsep0 rowsep0" align="left">129</td><td class="colsep0 rowsep0" align="left">102</td><td class="colsep0 rowsep0" align="left">120</td><td class="colsep0 rowsep0" align="left">284</td><td class="colsep0 rowsep0" align="left">271</td><td class="colsep0 rowsep0" align="left">135</td><td class="colsep0 rowsep0" align="left">133</td><td class="colsep0 rowsep0" align="left">196</td><td class="colsep0 rowsep0" align="left">1.18</td></tr></tbody></table></div></div></div><div id="sec3_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> Industry Productivity in Terms of Patent Records: “Chemical Singularity” or a New Age of Drug Discovery?</h4><div class="NLM_p">Considering the above-mentioned trends, we suspect that the number of unique chemical entities being claimed in patent literature is dramatically decreasing. To evaluate this hypothesis, we analyzed patent records from the top 23 pharmaceutical companies<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> involving new small-molecule compounds disclosed from 1950 to 2018 (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>). As clearly shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>, a spectacular growth in the number of new structures claimed from 1998 to 2004 was observed, but this was followed by a dramatic decline. Schneider’s team demonstrated that the number of U.S. pharmacological applications peaked in 2008, while in the subsequent 7 years, the number of applications decreased to the level seen in 2003. In contrast, the authors revealed a continuous growth in the number of U.S. pharmacological patent grants between 1975 and 2015. Interestingly, at the final time point, these lines intersected each other. Although the authors noted that a vast majority of the patents and unique reactions in the database were issued after the year 2000, they did not include any discussion of the observed behavior. All the patent records published during the selected time period were used in the study without regard to patent holder and were mined in an automatic fashion without proper expert inspection. Furthermore, the final product of the reactions listed in the patent record may show no relevant activity in the biological trial described therein. Therefore, “pharmacological” character cannot be assigned to such products. In this case, a rigorous database preprocessing and filtration procedure is of great importance to avoid misclassification. We hypothesize that the intellectual property assigned to the major pharmaceutical companies is much more exact and meaningful for reflecting the evolution of the field and industry’s overall health. Considering that the number of patent records issued in 2015–2016 is comparable to the number granted at the end of the previous century, “chemical singularity” is not a mythical point, and this collapse is creeping closer and closer. Under these circumstances, drug-repurposing<a onclick="showRef(event, 'ref54 ref55'); return false;" href="javascript:void(0);" class="ref ref54 ref55">(54,55)</a> is unable to fill this vacuum. Therefore, next-generation molecules are vital for targeting newly identified druggable proteins. Although “old” molecules still significantly influence the statistical outcome, novel chemistry gradually forces out this class of compounds, and we are beginning to see the changes in this field more clearly. Notably, this hypothetical outcome and its aftermath correlate well with previously published ideas. Although the obtained statistical results directly show signs of decreasing activity and continuous regression, MCE-18 scores have been steadily increasing over the years. These tendencies are easily visible and arguably associated with qualitative changes in medicinal chemistry. Therefore, instead of a “chemical singularity”, this era can be reasonably considered an intriguing turning point in drug design and development, and the traditional concept of drug development needs to be radically rethought. With respect to patent holders, <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a> illustrates the dynamic changes in MCE-18 among the leaders. As shown in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>, during the past 10 years, Merck and AstraZeneca have primarily focused on molecules with MCE-18 values over 60, while Pfizer and Sanofi have demonstrated less productivity but have reached similar scores.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/medium/jm-2019-00004q_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/large/jm-2019-00004q_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Devolution of medicinal chemistry expressed in the number of patent records by major pharmaceutical companies (bar chart, prior to normalization) sorted by the priority date vs continual growth of MCE-18 scores (black solid line, after normalization). ∗ indicates the year that the Lipinski seminal paper<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> was published.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/large/jm-2019-00004q_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00004&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/medium/jm-2019-00004q_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/large/jm-2019-00004q_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Top four pharmaceutical companies ranked by MCE-18 (after normalization).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/large/jm-2019-00004q_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00004&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The data set of small-molecule compounds derived from the selected patent records with the claimed activity against one of the most paramount target families (GPCRs, protein kinases, and proteases) was used to investigate the classification ability of MCE-18. <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a> shows the distribution of molecules from the unique patent records through the years and corresponding MCE-18 values. As shown in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>, in contrast to the number of unique patent records, MCE-18 has been gradually increasing over the whole estimated period reaching the highest mean values in the past 10 years: MCE-18 = 90, 75, and 74 for protease targeting molecules, GPCR ligands, and kinase inhibitors, respectively. The revealed behavior correlates well with the common tendency described above for all the compounds claimed in patent records (see <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>).</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/medium/jm-2019-00004q_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/large/jm-2019-00004q_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Number of patent records by major pharmaceutical companies (bar chart) sorted by the priority date vs continual growth of MCE-18 scores (solid lines): GPCR ligands (dark gray), protein kinase inhibitors (black), and molecules acting on proteases (gray).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/large/jm-2019-00004q_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00004&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Subsequently, within these core families we performed the detailed statistical analysis and uncovered the most populated areas (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>).</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/medium/jm-2019-00004q_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/large/jm-2019-00004q_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Distribution (%) of molecules with MCE-18 ≥ 50 among three key target families: (A) protein kinases; (B) proteases; (C) GPCRs.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/large/jm-2019-00004q_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00004&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">As shown in <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A, molecules by major pharmaceutical companies with the inhibition potency against CDKs (7.4%), JAKs (6%), IRAKs (5%), Aurora kinases (4.2%), and BTK (4.1%) account for more than 25% of all the kinase inhibitors claimed in the patent literature. It should be especially noted that during the past 5 years small-molecule inhibitors of JAKs and IRAKs are among the most attractive biological targets particularly for the treatment of rheumatoid arthritis, alopecia areata, and psoriasis. For instance, PF-06700841 (MCE-18 = 102.0), developed by Pfizer, is currently evaluated in phase II clinical trial against alopecia areata, psoriasis, vitiligo, and Crohn’s disease; ABBV-712 (AbbVie, structure is uncovered) is being investigated in phase I study for the treatment of psoriasis; TD-1473, a pan-Janus kinase inhibitor by Janssen Biotech (structure is closed), against inflammatory bowel disease (phase II), itacitinib (MCE-19 = 87.9); a selective JAK1 inhibitor, by Incyte, for the treatment of Graft-versus-host disease (phase III) and ulcerative colitis (phase II); as well as PF-06651600 (MCE-19 = 55.6), a suicidal JAK3 inhibitor (Pfizer), developed against alopecia areata (phase II/III) and rheumatoid arthritis (phase II). Within the protease family (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B), compounds targeted on secretases, DPP-IV, thrombin, and MMP-13 were found to feature in the top positions (55% of all the ligands disclosed). The leaders in the GPCR group (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>C) include molecules acting on serotonin (8.4%), tachykinin (5.9%), chemokine (5.7%), cannabinoid (4.9), and histamine (4.4%) receptors, a total 30% of all the GPCRs ligands described in the patent records. Because only major therapeutically significant and abundantly populated target classes were analyzed, minor targets were included into the “other” category. The structures of several molecules with MCE-18 ≥ 50 from different target classes are shown in <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>.</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/medium/jm-2019-00004q_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/large/jm-2019-00004q_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Representative examples of structures with high MCE-18 values claimed in patent records by major pharmaceutical companies during the past 5 years: (A) protein kinase inhibitors, (B) GPCR ligands, and (C) molecules acting on proteases.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/large/jm-2019-00004q_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00004&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">During the next step, we performed a statistical analysis of molecular properties using a Kohonen-based machine-learning technique to investigate evolutionary tendencies. As described above, prior to the statistical analysis and in silico modeling, the patent database was preprocessed and normalized to exclude molecules with high structural similarity in each cluster to overcome a statistical bias. Molecular descriptors were selected considering their theoretical impact on the studied phenomenon; the calculated <i>t</i>-values are presented in the <a href="/doi/suppl/10.1021/acs.jmedchem.9b00004/suppl_file/jm9b00004_si_001.pdf" class="ext-link">Supporting Information (Table S2)</a>. Among the evaluated parameters, there were no descriptors with Pearson’s correlation coefficient greater than 0.83 (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00004/suppl_file/jm9b00004_si_001.pdf" class="ext-link">Table S3</a>). The obtained results are presented in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>, and <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>. As shown in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, MCE-18 scores gradually increased over the estimated period. The average growth coefficient is 1.12 (per period), while the difference between the scores in the first (1950–1983) and the final (2012–2018) time intervals is close to 1.76. The SPIRO, Q<sup>1</sup>, SS, and MW values increased by factors of 1.36, 1.09, 1.08, and 1.07, respectively. Although SPIRO showed the greatest slope, it showed no clear trend over the studied period. Among “old” (1950–1990) structures bearing <i>spiro</i> centers, a reasonable portion of these molecules were of natural origin, and this function was present a priori. Therefore, the synthetic efforts were primarily targeted toward other diversification points instead of this ring juncture. During the past decade, the proportion of compounds with <i>spiro</i> centers has been dramatically increasing due to new scaffolds, thereby providing high growth coefficients. Considering the trend, unlike the <i>SPIRO</i> term, MCE-18 is a much more informative metric and adequately reflects the progress of synthetic medicinal chemistry. Molecular weight and PSA have constantly increased from the beginning of the studied period, and this tendency correlates well with the results reported previously (vide supra). Unlike PSA, the calculated LogS<sub>w</sub> values, which relate to aqueous solubility, decreased from the beginning of the studied period the end of the 1990s. Since this point, the LogS<sub>w</sub> values have plateaued at an average of approximately −4.8. This average value is approximately equal to the minimum aqueous solubility that is statistically allowed for drugs. However, as in the case of the predicted lipophilicity, LogS<sub>w</sub> varies dramatically with the method used for calculation. We also observed a progressive increase in the degree of 2D branching (Q<sup>1</sup>) over the years (increased by a factor of 1.09). This trend can be directly attributed to the simultaneous growths in MW and molecular “gravity”. Notably, molecular weight is a less valuable descriptor for the assessment of structural complexity because it strongly depends on the types of atoms present in the molecule. The lipophilicity of compounds constantly increased from 1950 to 2004 followed by a slight decline observed from 2005 (mean LogP<sup>calc</sup> value is 3.3 within the last period). However, it would be wrong to speculate around this behavior because there are many of factors that influence significantly this parameter. A similar tendency was noted by Michael Shultz. The AR term also demonstrates the expected growth over the studied time interval for the reasons described above.</div><figure id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/medium/jm-2019-00004q_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/large/jm-2019-00004q_0008.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Changes in MCE-18 components over the years.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/large/jm-2019-00004q_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00004&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/medium/jm-2019-00004q_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/large/jm-2019-00004q_0009.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. 20 × 20 2D Kohonen SOM reflecting the trends in key molecular properties of structures disclosed during the following periods: (a) 2015–2018; (b) 2001–2003; (c) 1980–1985. The color gradient indicates the number of molecules. Areas encircled by the white dotted lines correspond to high MCE-18 values. Basic contours of the map were smoothed for convenient visual inspection.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/large/jm-2019-00004q_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00004&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">The resulting Kohonen map is depicted in <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>. As shown in <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>a, new compounds are located predominantly within two small regions of the same map and are distinct from the nodes abundantly populated by old molecules (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>b,c). In the final cycle, the learning vector quantization error (LVQE) was relatively low, reaching a maximum value of 0.012. More than 90% of the samples showed LVQE values less than 0.002. Therefore, we can conclude that the constructed model has very good generalization ability and learning outcomes. The stability of the model was verified using three independent randomizations. Moreover, the addition of a fuzzy input with stochastic variables did not substantially impact the quality of classification. Upon examination, there were few “dead” neurons within the constructed map. The average classification accuracies for the polar categories were close to 86% and 77% without and with a random threshold, respectively.</div></div><div id="sec3_2_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> Launched Drugs: Difference between Old and New Molecules</h4><div class="NLM_p">The main conclusion we can draw from the previous section is that new molecules are larger, more polar and show greater 3D complexity relative to “vecchio” chemistry. However, the synthesis of such compounds requires more time and innovative building blocks. Currently, we are seeing a dramatic decline in the industry’s productivity in terms of new patent records on pharmacological substances of synthetic origin. MCE-18 scores allow us to trace the evolution of medicinal chemistry and to assess the “distance” between old and new chemistry. In this section, we explore the dominant trends emerging within the common pool of drugs launched to the pharmaceutical market using MCE-18. As shown in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>, MCE-18, MW, Q<sup>1</sup>, PSA, and SS all gradually increased over time. MCE-18 and Q<sup>1</sup> showed high growth coefficients and continuous progress. Similar to the results described above, we did not find any significant correlations between lipophilicity and launch date. The average LogP is 2.13, while for patented compounds, this value is up to 1.5 times higher. We speculate that the drugs that will be brought to market within the foreseeable future will have somewhat higher lipophilicities with the same or even greater PSA. Therefore, we are convinced that the sp<sup>3</sup> index is much less informative and rather ambiguous compared with MCE-18, which is quite well balanced.</div><div class="NLM_p">Using these data, we constructed a Kohonen-based SOM (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>) to elucidate the inherent trends hidden in the marketed drugs while keeping the focus on evolutionary changes. The mapping quality was very similar to that achieved above and provided good LVQE values and stability. As shown in <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>a, since the beginning of the studied period, drugs have gradually evolved, and we are now observing considerable changes in their aggregate properties. Positive dynamics were revealed for MCE-18 (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>b). Thus, recent drugs obviously tend to be more complex in terms of MCE-18. Unlike drugs brought to marked in the past 7 years (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>d), molecules launched from 1910 to 1979 are predominantly located at the top of the map (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>c). This model clearly shows that the properties of the drugs have dramatically evolved over the years, and many key players in the pharmaceutical industry have followed this trend. Minor companies and academia, if they do not want to miss out, need to consider these trends in their research programs.</div><figure id="fig10" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/medium/jm-2019-00004q_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/large/jm-2019-00004q_0010.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Kohonen SOM constructed for launched drugs: (a) white arrows indicate the route by which drugs have evolved over the years; (b) distribution of MCE-18 values within the same map; (c) areas occupied by the drugs brought to market during 1910–1979; (d) locations of new drugs (2011–2018).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/large/jm-2019-00004q_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00004&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Similar results were obtained using the database of drugs acting on GPCRs, protein kinases, or proteases already launched into the market or evaluated in clinical studies (CS). As shown in <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">11</a></a>, the number of kinase inhibitors has been constantly increasing over the past 30 years in contrast to protease targeting compounds where a slight decrease has been observed during the past 13 years and GPCR ligands: more than 2-fold decrease for the last period. However, we revealed a permanent increase in MCE-18 values over the whole time scale. The obtained results clearly indicate the MCE-18 conception reflects not only the common tendency in medicinal chemistry and drug discovery but also the evolution proceeded in key therapeutic groups.</div><figure id="fig11" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/medium/jm-2019-00004q_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/large/jm-2019-00004q_0011.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. Number of drugs (bar chart) sorted by the related milestone date vs continual growth of MCE-18 values (solid lines): GPCR ligands (dark gray), protein kinase inhibitors (black), and molecules acting on proteases (gray).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/large/jm-2019-00004q_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00004&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec3_2_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> Voyage to Market: MCE-18 Probe and Nonlinear Mapping</h4><div class="NLM_p">Although we have demonstrated that MCE-18 is in excellent agreement with the issue date assigned in a patent claim and with the launched date, we have also investigated the relationship between this descriptor and the routine drug development pipeline, from preclinical evaluation to the market via clinical trials. In this case, we anticipated that we would find somewhat different results. As shown in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>, MCE-18 falls gradually over the whole route from 57.3 (preclinical studies) to 45.98 (launched drugs), showing an overall decrease by a factor of 1.25. A similar tendency was observed for MW, AR, NAR, Q<sup>1</sup>, LogS<sub>w</sub>, and SS. The average MCE-18 values calculated for all the molecules ever launched are 1.2 and 1.65 times lower than those of new drugs (2012–2018) and structures that have been disclosed in patent literature for the past 6 years. Likewise, comparable values were obtained for MW, LogP, Q<sup>1</sup>, SS, and SPIRO. Interestingly, the sp<sup>3</sup> index constantly increases from 36.4 to 44.65 (1.22-fold gain); however, this is not surprising because there is a large population of old structures among the launched drugs that skew the results and cause molecular “turbulence”. In contrast, new molecules that have recently entered preclinical studies or early clinical trials demonstrate higher MCE-18 values. Considering these trends, we anticipate that if approximately 10% of these molecules reach the market, the statistical results will change.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Mean Values of the Molecular Descriptors Calculated for Lead Molecules and Drugs</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">no.</th><th class="colsep0 rowsep0" align="center">MD</th><th class="colsep0 rowsep0" align="center">preclinical studies</th><th class="colsep0 rowsep0" align="center">phase I</th><th class="colsep0 rowsep0" align="center">phase II</th><th class="colsep0 rowsep0" align="center">phase III</th><th class="colsep0 rowsep0" align="center">launched</th><th class="colsep0 rowsep0" align="center">increase factor</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">MCE-18</td><td class="colsep0 rowsep0" align="left">57.3</td><td class="colsep0 rowsep0" align="left">56.2</td><td class="colsep0 rowsep0" align="left">55.9</td><td class="colsep0 rowsep0" align="left">54.0</td><td class="colsep0 rowsep0" align="left">45.98</td><td class="colsep0 rowsep0" align="left">0.95</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">MW</td><td class="colsep0 rowsep0" align="left">418.0</td><td class="colsep0 rowsep0" align="left">412.9</td><td class="colsep0 rowsep0" align="left">402.4</td><td class="colsep0 rowsep0" align="left">397.0</td><td class="colsep0 rowsep0" align="left">366.70</td><td class="colsep0 rowsep0" align="left">0.97</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">AR</td><td class="colsep0 rowsep0" align="left">0.91</td><td class="colsep0 rowsep0" align="left">0.87</td><td class="colsep0 rowsep0" align="left">0.87</td><td class="colsep0 rowsep0" align="left">0.85</td><td class="colsep0 rowsep0" align="left">0.80</td><td class="colsep0 rowsep0" align="left">0.97</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">NAR</td><td class="colsep0 rowsep0" align="left">0.72</td><td class="colsep0 rowsep0" align="left">0.72</td><td class="colsep0 rowsep0" align="left">0.71</td><td class="colsep0 rowsep0" align="left">0.71</td><td class="colsep0 rowsep0" align="left">0.67</td><td class="colsep0 rowsep0" align="left">0.98</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">5</td><td class="colsep0 rowsep0" align="left">CHIRAL</td><td class="colsep0 rowsep0" align="left">0.50</td><td class="colsep0 rowsep0" align="left">0.55</td><td class="colsep0 rowsep0" align="left">0.59</td><td class="colsep0 rowsep0" align="left">0.61</td><td class="colsep0 rowsep0" align="left">0.56</td><td class="colsep0 rowsep0" align="left">1.03</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">6</td><td class="colsep0 rowsep0" align="left">SPIRO</td><td class="colsep0 rowsep0" align="left">0.027</td><td class="colsep0 rowsep0" align="left">0.020</td><td class="colsep0 rowsep0" align="left">0.020</td><td class="colsep0 rowsep0" align="left">0.030</td><td class="colsep0 rowsep0" align="left">0.01</td><td class="colsep0 rowsep0" align="left">0.89</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">7</td><td class="colsep0 rowsep0" align="left">NCSPTR</td><td class="colsep0 rowsep0" align="left">0.22</td><td class="colsep0 rowsep0" align="left">0.24</td><td class="colsep0 rowsep0" align="left">0.24</td><td class="colsep0 rowsep0" align="left">0.24</td><td class="colsep0 rowsep0" align="left">0.23</td><td class="colsep0 rowsep0" align="left">1.01</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left">Q<sup>1</sup></td><td class="colsep0 rowsep0" align="left">22.5</td><td class="colsep0 rowsep0" align="left">21.8</td><td class="colsep0 rowsep0" align="left">21.3</td><td class="colsep0 rowsep0" align="left">21.0</td><td class="colsep0 rowsep0" align="left">17.98</td><td class="colsep0 rowsep0" align="left">0.95</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">9</td><td class="colsep0 rowsep0" align="left">sp<sup>3</sup></td><td class="colsep0 rowsep0" align="left">36.4</td><td class="colsep0 rowsep0" align="left">40.5</td><td class="colsep0 rowsep0" align="left">41.5</td><td class="colsep0 rowsep0" align="left">41.8</td><td class="colsep0 rowsep0" align="left">44.65</td><td class="colsep0 rowsep0" align="left">1.05</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">10</td><td class="colsep0 rowsep0" align="left">LogP</td><td class="colsep0 rowsep0" align="left">3.4</td><td class="colsep0 rowsep0" align="left">2.8</td><td class="colsep0 rowsep0" align="left">2.9</td><td class="colsep0 rowsep0" align="left">2.6</td><td class="colsep0 rowsep0" align="left">2.14</td><td class="colsep0 rowsep0" align="left">0.89</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">11</td><td class="colsep0 rowsep0" align="left">LogS<sub>w</sub></td><td class="colsep0 rowsep0" align="left">–4.6</td><td class="colsep0 rowsep0" align="left">–4.4</td><td class="colsep0 rowsep0" align="left">–4.4</td><td class="colsep0 rowsep0" align="left">–4.3</td><td class="colsep0 rowsep0" align="left">–4.01</td><td class="colsep0 rowsep0" align="left">0.97</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">12</td><td class="colsep0 rowsep0" align="left">PSA</td><td class="colsep0 rowsep0" align="left">91.6</td><td class="colsep0 rowsep0" align="left">95.2</td><td class="colsep0 rowsep0" align="left">94.0</td><td class="colsep0 rowsep0" align="left">94.7</td><td class="colsep0 rowsep0" align="left">88.08</td><td class="colsep0 rowsep0" align="left">0.99</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">13</td><td class="colsep0 rowsep0" align="left">HBD</td><td class="colsep0 rowsep0" align="left">2.0</td><td class="colsep0 rowsep0" align="left">2.2</td><td class="colsep0 rowsep0" align="left">2.2</td><td class="colsep0 rowsep0" align="left">2.2</td><td class="colsep0 rowsep0" align="left">2.10</td><td class="colsep0 rowsep0" align="left">1.01</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">14</td><td class="colsep0 rowsep0" align="left">HBA</td><td class="colsep0 rowsep0" align="left">3.2</td><td class="colsep0 rowsep0" align="left">3.4</td><td class="colsep0 rowsep0" align="left">3.4</td><td class="colsep0 rowsep0" align="left">3.5</td><td class="colsep0 rowsep0" align="left">3.44</td><td class="colsep0 rowsep0" align="left">1.02</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">15</td><td class="colsep0 rowsep0" align="left">SS</td><td class="colsep0 rowsep0" align="left">71.3</td><td class="colsep0 rowsep0" align="left">71.4</td><td class="colsep0 rowsep0" align="left">69.4</td><td class="colsep0 rowsep0" align="left">69.0</td><td class="colsep0 rowsep0" align="left">60.30</td><td class="colsep0 rowsep0" align="left">0.96</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="2" align="left">entities</td><td class="colsep0 rowsep0" align="left">38338</td><td class="colsep0 rowsep0" align="left">1678</td><td class="colsep0 rowsep0" align="left">1837</td><td class="colsep0 rowsep0" align="left">464</td><td class="colsep0 rowsep0" align="left">1370</td><td class="colsep0 rowsep0" align="left"> </td></tr></tbody></table></div></div><div class="NLM_p">The Kohonen map showing the distribution of compounds by the development stage is depicted in <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">12</a></a>. As clearly demonstrated in <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">12</a></a>a, lead compounds are homogeneously distributed almost through the whole map with several overpopulated clusters, while drug candidates are mainly located inside these densely populated regions (<a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">12</a></a>b). Three distinct areas correspond to moderate-to-high MCE-18 values (<a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">12</a></a>c). Interestingly, the location of old drugs that entered the market in the 1910–1987 period (<a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">12</a></a>d) is radically different from that of drugs launched in the past decade (<a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">12</a></a>e). Almost all the older drugs are characterized by relatively low MCE-18 values and are mainly grouped within two areas, while new drugs tend to be incorporated in the regions associated with moderate MCE-18 values. This result is in full accordance with that described above for launch periods (see <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>a). The average classification accuracy between these two categories is approximately 70%, which is close to that achieved for clinically evaluated vs launched drugs.</div><figure id="fig12" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 12</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/medium/jm-2019-00004q_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/large/jm-2019-00004q_0012.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 12. Distribution of lead compounds and drugs within the Kohonen map: (a) molecules in preclinical investigation, (b) compounds examined (or are currently being examined) in clinical trials, (c) the distribution of MCE-18 values, (d) areas occupied by drugs launched from 1910 to 1987, and (e) new drugs brought to market in recent years (2011–2018).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/large/jm-2019-00004q_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00004&amp;id=fig12"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec3_2_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> Sammon Nonlinear Mapping</h4><div class="NLM_p">As discussed above, statistical analysis and Kohonen-based nonlinear mapping show that new drugs that have been brought to market during the past decade differ remarkably from those launched in the previous century. However, we did not assess their aggregate properties in terms of RO5+. To address this issue, we carried out Sammon nonlinear mapping using the same set of molecular descriptors. The resulting plot is presented in <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">13</a></a> and shows that molecules that successfully meet the criteria are tightly grouped within the bottom left area of the map (triangles), whereas new drugs are mainly spread outside this region or are tightly distributed within the periphery of the RO5 region. The obtained result visually presents the expected tendency for next-generation molecules to deviate from Lipinski’s rules and particularly reflects one of many aspects of the evolution of medicinal chemistry.</div><figure id="fig13" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 13</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/medium/jm-2019-00004q_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/large/jm-2019-00004q_0013.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 13. Sammon map constructed for the launched drugs. “+” means that PSA < 140 Å was added to the classic Lipinski’s rules.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/large/jm-2019-00004q_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00004&amp;id=fig13"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In addition, we analyzed the small-molecule compounds in terms of their parent company and their MCE-18 score. The in-house collections of compounds were obtained by ChemDiv (CD regular collection),<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> Enamine (EA hits and advanced collections),<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> and InterBioscreen (IBS synthetic collection)<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> and preprocessed using soft MCFs to exclude reagents, building blocks, and “marginal” compounds. Molecules containing reactive groups and undesired fragments were eliminated as well. Entities with MWs in the range of 200–700 were used for the study. In this case, we did not perform any normalization procedures and used all the molecules available in the stocks that meet the criteria above. The resulting statistical outcome is depicted in <a data-tab="pane-pcw-Figures" href="#fig14" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig14" id="" class=" internalNav">14</a></a>. Thus, molecules with MCE-18 less than 45 are generally uninteresting, trivial, contain old scaffolds, and have low degrees of 3D complexity and novelty. These compounds should be discarded and excluded from further investigations. Molecules with MCE-18 values in the range of 45–63 offer sufficient novelty and basically follow the trends currently observed in medicinal chemistry, while compounds with MCE-18 values of 63–78 generally show high structural similarity to compounds disclosed in patent records by major pharmaceutical enterprises during the past decade. The structures of entities with MCE-18 > 78 need to be inspected visually to assess their target profile and drug-like-ness. Overall, the collections can be sorted by 45 < MCE-18 < 78 as follows: CD (47%), EA (AC, 44%), IBS (36%), and EA (HC, 34%). Notably, within the past decade, the synthetic teams of Enamine and ChemDiv have focused on molecules having MCE-18 > 45 and on building blocks with a high degree of 3D complexity.</div><figure id="fig14" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Figure 14</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/medium/jm-2019-00004q_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/large/jm-2019-00004q_0014.jpeg" id="gr14" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 14. Vendor collections scored by MCE-18.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/large/jm-2019-00004q_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00004&amp;id=fig14"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Below, we have listed the key factors responsible for the dramatic decrease in major pharmaceutical companies productivity observed over the past 15 years.</div><div class="NLM_p">(1) The “golden era” of major pharmaceutical companies has long passed, and during the rapid advance of the 20th century, the most-workable and accessible compounds (blockbuster drugs) were fully exploited. Many of the patents on blockbuster drugs have expired or will expire soon; therefore, major pharmaceutical enterprises are continuing to lose its market position, as was predicted by Kessel.<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> National Pharmaceutical Services (NPS)<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a> has recently published a list of drugs, including best-selling drugs, coming off patent by 2022; their generic versions have not been considered. In total, approximately $100 billion in sales have been lost during the past years. Indeed, key pharmaceutical players are facing daunting challenges and have lost their main source of income. These losses are mitigated by structural reforms, especially within the R&D sector. As a result, the medicinal chemistry community has shrunk considerably over the past decade as a consequence of mergers and downsizing.<a onclick="showRef(event, 'ref60 ref61'); return false;" href="javascript:void(0);" class="ref ref60 ref61">(60,61)</a> Cumulatively, this has led to a dramatic decrease in productivity in terms of the number of patent records.</div><div class="NLM_p">(2) A drop in profits has also been associated with U.S. healthcare reform and European government “austerity” measures. GSK reported that these events had a profound impact on the drug industry.<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> Tsukamoto has noted that increasing regulatory burdens significantly influences the costs and time spent on launching a drug to market.<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a> The improvements in quality control and the tightening of rules for the assessment of clinical efficacy have had a meaningful impact on productivity, and in turn, governments and payers will continue to bear the burdens of increased prices and changes in access, utilization, and prescribing patterns. These obstacles in the legal field will likely indirectly negatively affect patent activity through a reduction in the reinvestment of profits in the R&D sector.</div><div class="NLM_p">(3) The reasons above have forced major pharmaceutical companies to critically rethink the role of medical chemistry. Tsukamoto noted that “medicinal chemists had to make some adjustments to satisfy the appetite of the fast-paced screening paradigm”<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a> and harshly criticized the classic approach in which medicinal chemistry was solely focused on HTS and did not sufficiently value creativity in routine drug discovery. Mainly, industry is guided by pragmatic and economic principles that in many cases have resulted in the most effective and simple decision. As a consequence, “drug discovery research operations are turning into a game of numbers”.<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a> The author noted that “complex reactions that require considerable knowledge and technical skills were avoided as much as possible”,<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a> and the main focus was placed on more trivial routes. Ultimately, the simplicity of the final molecules led to an overall degradation of the organic chemistry toolbox, and this was also highlighted by Schneider. Tsukamoto said the following: “When entry-level chemists fresh from academic laboratories propose complex target molecules that can only be made by a series of unconventional reactions, how often do we dismiss them by saying, “That’s very interesting, but let’s be realistic. Why don’t you make this series of compounds instead? Three easy steps and you can make several analogs?’”<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a> The author agreed with Kessel, who referred to the analysis by Bain & Company under which a “‘broken innovation culture’ lies at the core of big pharma’s problems”,<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> and “big pharma executives need to recognize that truly creative steps in product generation are not scalable like other manufacturing processes”.<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a></div><div class="NLM_p">(4) Considering the industry’s needs, Rafferty recommends paying more attention to the significant role of high-quality education in drug discovery.<a onclick="showRef(event, 'ref63'); return false;" href="javascript:void(0);" class="ref ref63">(63)</a> Medicinal chemists who graduate from universities are less attractive employees for large pharmaceutical companies. Therefore, academic researchers should shift their focus from grants and publications to actual business needs. The industry itself should collaborate with the academic community more effectively and encourage industry to develop innovative solutions that may become alternative sources of research funding. Hoffman notes that the role of medicinal chemistry in industry has changed substantially over the past few decades; however, it remains too narrow.<a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(64)</a> In the future, there will likely be increased demand for truly expert medicinal chemists with a deep understanding of drug discovery disciplines and related subjects to manage innovative projects in this area.<a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(65)</a> In contrast, the number of ordinary scientists in major pharmaceutical companies will continually decrease, and emphasis will be placed on the quality of their work.</div><div class="NLM_p last">In the designing of the MCE-18 metric, the main goal was to reflect the opinions summarized above as well as our own experience regarding the evolution of medicinal chemistry in a unified equation. MCE-18 contains objective terms that can be readily calculated using the structure of a molecule in a convenient and insightful way. Despite the decline in the total number of patents for new drugs, the quality and complexity of novel molecules by major pharmaceutical companies have drastically increased over the past decade. Painstaking synthetic efforts have yielded a series of new building blocks bearing nontrivial 3D architectures, providing a robust IP position. As mentioned above, a considerable number of novel molecules are PPI inhibitors, which requires a shift in the attention of medicinal chemists toward structures with a relatively higher degree of 3D complexity. As we demonstrate herein, MCE-18 can be used to monitor key features of “next-generation” molecules and trace the evolution of medicinal chemistry through the years. Skilled medicinal chemists continue to make good and original molecules that give us confidence that the “chemical singularity” will swallow up vecchio compounds, opening the door to a new era of medicinal chemistry.</div></div></div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i24">Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_62641" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_62641" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Evolutionary theory is one of the most important paradigms in science. The fundamental principles of evolution run through almost all fields where incremental progress is observed. One of the crucial drivers of evolution is adaptation to permanently changing environments. From this point of view, progress in the pharmacological industry significantly depends on achievements in disease-associated biology as well as advances in the toolbox of medicinal chemists. Major pharmaceutical companies set the tone of this industry and have many resources to dominate the minor players. They can be regarded as one of the core drivers of medicinal chemistry evolution. In this context, understanding the strategic trends in drug design and development and in the industry breathing space is of great importance. Herein, we shed light on this issue using MCE-18 as a novel metric as well as machine-learning techniques.</div><div class="NLM_p last">Historically, the routine and epidemic-like application of drug-likeness paradigm has resulted in a range of common principles and efficiency indices that play a pivotal role in decision making by drug-hunting teams in industry and scientists searching for novel drugs. Indeed, many core players in the field of drug discovery understand these flaws, but they rather prefer to follow the rules than to seek success abroad. However, during the past decade, a dramatic decrease in the number of patent records involving new chemical entities by major pharmaceutical enterprises has been observed. Considering that the activities of a vast majority of small-molecule compounds have been fully elucidated in different biological trials, the escalation of HTS has resulted in a significant depletion of available collections. Many molecules shared in stocks are now considered to have a relatively low attraction rate and little promise. On the other hand, great advances in the field of PPIs have forced scientists to explore new areas of purportedly synthetically accessible chemical space. This impulse is very important; methods and principles of organic chemistry have evolved over the years and have resulted in original building blocks and templates that are primarily characterized by increased 3D complexity. Formally, these valuable insights offer the opportunity to empirically formulate a basis for the “chemical singularity” that is probably our impending eschaton compared with the dogmatic position. In contrast, we can reasonably regard this as a novel turning point in chemical evolution and state that medicinal chemistry has ushered in a new era of drug design and development. To investigate the current trends observed in drug design and development, we used well-preprocessed pharmacological and patent databases as well as the collections of various companies. Equipped with the newly developed MCE-18 descriptor and in silico tools, we have clearly shown that scaffolds are becoming increasingly sophisticated with higher degrees of 3D complexity not only in the PPI realm but also in the field of kinase and protease inhibitors, GPCR-targeted compounds, and others. On the basis of these challenges, we strived to provide readers with statistically adequate and relevant information regarding the dominated trends.</div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i25"><a href="/doi/suppl/10.1021/acs.jmedchem.9b00004" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_78448" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_78448" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.9b00004" class="ext-link">10.1021/acs.jmedchem.9b00004</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Tables of statistical data for the calculated descriptors and a list of top 23 pharmaceutical companies (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00004/suppl_file/jm9b00004_si_001.pdf" class="ext-link">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00004/suppl_file/jm9b00004_si_001.pdf">jm9b00004_si_001.pdf (240.13 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.9b00004" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_33438" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_33438" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yan A. Ivanenkov</span> - <span class="hlFld-Affiliation affiliation">Insilico
Medicine Hong Kong Limited (previously Insilico Medicine, Inc.), Unit 307A, Core Building 1, 1 Science
Park East Avenue, Hong Kong Science Park, Pak
Shek Kok, Hong Kong</span>; 
    <span class="hlFld-Affiliation affiliation">Institute of Biochemistry
and Genetics Russian Academy of Science (IBG RAS) Ufa Scientific Centre, Oktyabrya Prospekt 71, Ufa 450054, Russian Federation</span>; 
    <span class="hlFld-Affiliation affiliation">Moscow
Institute of Physics and Technology (State University), 9 Institutskiy Lane, Dolgoprudny, Moscow 141700, Russian Federation</span>; 
    <span class="hlFld-Affiliation affiliation">Chemistry
Department, Lomonosov Moscow State University, Leninskie Gory, Building 1/3, GSP-1, Moscow 119991, Russian Federation</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8968-0879" title="Orcid link">http://orcid.org/0000-0002-8968-0879</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#265f4748664f48554f4a4f45494b43424f454f48430845494b"><span class="__cf_email__" data-cfemail="087169664861667b6164616b67656d6c616b61666d266b6765">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Bogdan A. Zagribelnyy</span> - <span class="hlFld-Affiliation affiliation">Insilico
Medicine Hong Kong Limited (previously Insilico Medicine, Inc.), Unit 307A, Core Building 1, 1 Science
Park East Avenue, Hong Kong Science Park, Pak
Shek Kok, Hong Kong</span>; 
    <span class="hlFld-Affiliation affiliation">Chemistry
Department, Lomonosov Moscow State University, Leninskie Gory, Building 1/3, GSP-1, Moscow 119991, Russian Federation</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Vladimir A. Aladinskiy</span> - <span class="hlFld-Affiliation affiliation">Insilico
Medicine Hong Kong Limited (previously Insilico Medicine, Inc.), Unit 307A, Core Building 1, 1 Science
Park East Avenue, Hong Kong Science Park, Pak
Shek Kok, Hong Kong</span>; 
    <span class="hlFld-Affiliation affiliation">Moscow
Institute of Physics and Technology (State University), 9 Institutskiy Lane, Dolgoprudny, Moscow 141700, Russian Federation</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><p>The authors declare no competing financial interest.<br /></br></p></li></ul></div><div class="bio" rid="ath1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="biographies">Biographies</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_21153" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_21153" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><figure id="ath1" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Yan A. Ivanenkov</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00004&amp;id=BIO-d7e2686-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Yan A. Ivanenkov</b> graduated from the Lomonosov Moscow State University, Faculty of Chemistry (organic and medicinal chemistry) in 2002. Then, in 2010, he was awarded a Ph.D. degree in Biochemistry, and then he became an Associate Professor of Medicinal Chemistry. For more than 20 years he has been working in the field of drug design, medicinal chemistry and biochemistry, QSAR analysis, organic and combinatorial synthesis, and in silico modeling. More than 100 scientific papers have been published in various authoritative and high IF journals. Several small molecule drugs have been developed and are currently evaluated in clinics, including neuraminidase and NS5A inhibitors and androgen receptor and 5HT<sub>6</sub>R antagonists. Since 2017, he has been the chief leader and the Head of Medicinal Chemistry Department at Insilico Medicine, Inc.</p></figure></div><div class="bio" rid="ath2"><figure id="ath2" data-index="16" class="article__inlineFigure"><h2 class="fig-label">Bogdan A. Zagribelnyy</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00004&amp;id=BIO-d7e2694-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Bogdan A. Zagribelnyy</b> is pursuing a specialist’s degree (to date, 6th year) in organic, bioorganic, and medicinal chemistry at Lomonosov Moscow State University (Moscow, Russia), currently under the supervision of Dr. Yan Ivanenkov and Prof. Alexander Majouga. He started consulting at Insilico Medicine Inc. in 2017. He is involved with the development of new androgen receptor ligands as therapeutic tools in prostate cancer, medicinal chemistry data collection, and training set preparation for AI-driven drug discovery platforms.</p></figure></div><div class="bio" rid="ath3"><figure id="ath3" data-index="17" class="article__inlineFigure"><h2 class="fig-label">Vladimir A. Aladinskiy</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00004&amp;id=BIO-d7e2699-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Vladimir A. Aladinskiy</b> received his B.S. (2012) and M.S (2014) in Applied Mathematics and Physics from Moscow Institute of Physics and Technology. Then he joined Dr. Yan Ivanenkov’s research group at the MIPT and obtained his Ph.D. in Computational Chemistry with research focusing on computer-aided drug design of new NS5A HCV inhibitors. Since 2017, he is a Computational Chemistry Team Lead at Insilico Medicine, Inc. where he supervises projects on the application of AI-driven methods to the generation of new drug-like molecules. His main interests include CADD as well as machine learning and big data in drug design.</p></figure></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i29">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_41321" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_41321" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The authors gratefully acknowledge the financial support of the Ministry of Education and Science of the Russian Federation [Grant 20.9907.2017/VU] (expert opinion, discussion, and manuscript preparation) and Russian Science Foundation [Grant 17-74-30012], IBG RAS Ufa (in silico study and statistical analysis). The authors gratefully acknowledge the valuable assistance of Prof. Nicholas Meanwell in discussions.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">Bcl-2</td><td class="NLM_def"><p class="first last">B-cell lymphoma 2 protein</p></td></tr><tr><td class="NLM_term">BRD</td><td class="NLM_def"><p class="first last">bromodomain</p></td></tr><tr><td class="NLM_term">ChemoSoft</td><td class="NLM_def"><p class="first last">ChemDiv Inc. chemical database software</p></td></tr><tr><td class="NLM_term">FAK</td><td class="NLM_def"><p class="first last">focal adhesion kinase</p></td></tr><tr><td class="NLM_term">FRB</td><td class="NLM_def"><p class="first last">freely rotatable bonds</p></td></tr><tr><td class="NLM_term">GLP1R</td><td class="NLM_def"><p class="first last">glucagon-like peptide-1 receptor</p></td></tr><tr><td class="NLM_term">GSK</td><td class="NLM_def"><p class="first last">GlaxoSmithKline Inc.</p></td></tr><tr><td class="NLM_term">Hsp70</td><td class="NLM_def"><p class="first last">70 kDa heat shock proteins</p></td></tr><tr><td class="NLM_term">IP</td><td class="NLM_def"><p class="first last">intellectual property</p></td></tr><tr><td class="NLM_term">LogS<sub>w</sub></td><td class="NLM_def"><p class="first last">predicted solubility in water</p></td></tr><tr><td class="NLM_term">RDKit</td><td class="NLM_def"><p class="first last">open-source cheminformatics software</p></td></tr><tr><td class="NLM_term">LVQE</td><td class="NLM_def"><p class="first last">learning vector quantization error</p></td></tr><tr><td class="NLM_term">MCF</td><td class="NLM_def"><p class="first last">medicinal chemistry filter</p></td></tr><tr><td class="NLM_term">NS5A</td><td class="NLM_def"><p class="first last">nonstructural protein 5A</p></td></tr><tr><td class="NLM_term">p53/MDM2</td><td class="NLM_def"><p class="first last">p53 protein and mouse double minute 2 homolog protein interaction</p></td></tr><tr><td class="NLM_term">RO3</td><td class="NLM_def"><p class="first last">rule of 3</p></td></tr><tr><td class="NLM_term">RO5+</td><td class="NLM_def"><p class="first last">beyond “rule of 5” (Lipinski)</p></td></tr><tr><td class="NLM_term">TPSA</td><td class="NLM_def"><p class="first last">topological polar surface area</p></td></tr><tr><td class="NLM_term">SOM</td><td class="NLM_def"><p class="first last">self-organizing map</p></td></tr><tr><td class="NLM_term">MoA</td><td class="NLM_def"><p class="first last">mechanism of action</p></td></tr><tr><td class="NLM_term">SMCM</td><td class="NLM_def"><p class="first last">synthetic and molecular complexity index</p></td></tr><tr><td class="NLM_term">JAK</td><td class="NLM_def"><p class="first last">Janus protein kinase</p></td></tr><tr><td class="NLM_term">BTK</td><td class="NLM_def"><p class="first last">Bruton’s tyrosine kinase</p></td></tr><tr><td class="NLM_term">TWC</td><td class="NLM_def"><p class="first last">total walk count</p></td></tr><tr><td class="NLM_term">BC</td><td class="NLM_def"><p class="first last">Barone’s complexity</p></td></tr><tr><td class="NLM_term">GPCR</td><td class="NLM_def"><p class="first last">G-protein-coupled receptor</p></td></tr><tr><td class="NLM_term">CDK</td><td class="NLM_def"><p class="first last">cyclin-dependent kinase</p></td></tr><tr><td class="NLM_term">IRAK</td><td class="NLM_def"><p class="first last">interleukin-1 receptor (IL-1R) associated kinase</p></td></tr><tr><td class="NLM_term">CS</td><td class="NLM_def"><p class="first last">clinical studies</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i31">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_43306" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_43306" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 65 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mullard, A.</span></span> <span> </span><span class="NLM_article-title">The drug maker’s guide to the galaxy</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>549</i></span>,  <span class="NLM_fpage">445</span>– <span class="NLM_lpage">447</span>, <span class="refDoi"> DOI: 10.1038/549445a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=10.1038%2F549445a" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=28959982" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A280%3ADC%252BC1M%252FjsFSmtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=549&publication_year=2017&pages=445-447&author=A.+Mullard&title=The+drug+maker%E2%80%99s+guide+to+the+galaxy&doi=10.1038%2F549445a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The drug-maker's guide to the galaxy</span></div><div class="casAuthors">Mullard Asher</div><div class="citationInfo"><span class="NLM_cas:title">Nature</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">549</span>
        (<span class="NLM_cas:issue">7673</span>),
    <span class="NLM_cas:pages">445-447</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSItGWG-TCuJEg6wdEZmT2rfW6udTcc2eYxhazmENVld7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M%252FjsFSmtA%253D%253D&md5=4ff8347906c8f3b459d05a5fe7ad0b56</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2F549445a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F549445a%26sid%3Dliteratum%253Aachs%26aulast%3DMullard%26aufirst%3DA.%26atitle%3DThe%2520drug%2520maker%25E2%2580%2599s%2520guide%2520to%2520the%2520galaxy%26jtitle%3DNature%26date%3D2017%26volume%3D549%26spage%3D445%26epage%3D447%26doi%3D10.1038%2F549445a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bohacek, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMartin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guida, W. C.</span></span> <span> </span><span class="NLM_article-title">The art and practice of structure-based drug design: a molecular modeling perspective</span>. <i>Med. Res. Rev.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">3</span>– <span class="NLM_lpage">50</span>, <span class="refDoi"> DOI: 10.1002/(SICI)1098-1128(199601)16:1<3::AID-MED1>3.0.CO;2-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=10.1002%2F%28SICI%291098-1128%28199601%2916%3A1%3C3%3A%3AAID-MED1%3E3.0.CO%3B2-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=8788213" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADyaK28XhtFyls78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=1996&pages=3-50&author=R.+S.+Bohacekauthor=C.+McMartinauthor=W.+C.+Guida&title=The+art+and+practice+of+structure-based+drug+design%3A+a+molecular+modeling+perspective&doi=10.1002%2F%28SICI%291098-1128%28199601%2916%3A1%3C3%3A%3AAID-MED1%3E3.0.CO%3B2-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">The art and practice of structure-based drug design: a molecular modeling perspective</span></div><div class="casAuthors">Bohacek, Regine S.; McMartin, Colin; Guida, Wayne C.</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">3-50</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">Wiley</span>)
        </div><div class="casAbstract">A review with 120 refs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmfrWdtPosdrVg90H21EOLACvtfcHk0lgOWadTnRyZsQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XhtFyls78%253D&md5=edfa592791c111479117feed600dead6</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291098-1128%28199601%2916%3A1%3C3%3A%3AAID-MED1%3E3.0.CO%3B2-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291098-1128%2528199601%252916%253A1%253C3%253A%253AAID-MED1%253E3.0.CO%253B2-6%26sid%3Dliteratum%253Aachs%26aulast%3DBohacek%26aufirst%3DR.%2BS.%26aulast%3DMcMartin%26aufirst%3DC.%26aulast%3DGuida%26aufirst%3DW.%2BC.%26atitle%3DThe%2520art%2520and%2520practice%2520of%2520structure-based%2520drug%2520design%253A%2520a%2520molecular%2520modeling%2520perspective%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D1996%26volume%3D16%26spage%3D3%26epage%3D50%26doi%3D10.1002%2F%28SICI%291098-1128%28199601%2916%3A1%3C3%3A%3AAID-MED1%3E3.0.CO%3B2-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boström, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keserü, G. M.</span></span> <span> </span><span class="NLM_article-title">Expanding the medicinal chemistry synthetic toolbox</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">709</span>– <span class="NLM_lpage">727</span>, <span class="refDoi"> DOI: 10.1038/nrd.2018.116</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=10.1038%2Fnrd.2018.116" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=30140018" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A280%3ADC%252BB3c3htlWisw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2018&pages=709-727&author=J.+G.+Bostr%C3%B6mauthor=D.+G.+Brownauthor=R.+J.+Youngauthor=G.+M.+Keser%C3%BC&title=Expanding+the+medicinal+chemistry+synthetic+toolbox&doi=10.1038%2Fnrd.2018.116"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Expanding the medicinal chemistry synthetic toolbox</span></div><div class="casAuthors">Bostrom Jonas; Brown Dean G; Young Robert J; Keseru Gyorgy M</div><div class="citationInfo"><span class="NLM_cas:title">Nature reviews. Drug discovery</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">709-727</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The key objectives of medicinal chemistry are to efficiently design and synthesize bioactive compounds that have the potential to become safe and efficacious drugs.  Most medicinal chemistry programmes rely on screening compound collections populated by a range of molecules derived from a set of known and robust chemistry reactions.  Analysis of the role of synthetic organic chemistry in subsequent hit and lead optimization efforts suggests that only a few reactions dominate.  Thus, the uptake of new synthetic methodologies in drug discovery is limited.  Starting from the known limitations of reaction parameters, synthesis design tools, synthetic strategies and innovative chemistries, here we highlight opportunities for the expansion of the medicinal chemists' synthetic toolbox.  More intense crosstalk between synthetic and medicinal chemists in industry and academia should enable enhanced impact of new methodologies in future drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTy8c_ueHSwcYMcDoIzZtNkfW6udTcc2eYxhazmENVld7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c3htlWisw%253D%253D&md5=d87e988602808139597576ced56faa88</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2018.116&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2018.116%26sid%3Dliteratum%253Aachs%26aulast%3DBostr%25C3%25B6m%26aufirst%3DJ.%2BG.%26aulast%3DBrown%26aufirst%3DD.%2BG.%26aulast%3DYoung%26aufirst%3DR.%2BJ.%26aulast%3DKeser%25C3%25BC%26aufirst%3DG.%2BM.%26atitle%3DExpanding%2520the%2520medicinal%2520chemistry%2520synthetic%2520toolbox%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2018%26volume%3D17%26spage%3D709%26epage%3D727%26doi%3D10.1038%2Fnrd.2018.116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span>BioGRID
Database Statistics. <a href="https://wiki.thebiogrid.org/doku.php/statistics" class="extLink">https://wiki.thebiogrid.org/doku.php/statistics</a> (accessed Feb 27, <span class="NLM_year">2019</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=BioGRID%0ADatabase+Statistics.+https%3A%2F%2Fwiki.thebiogrid.org%2Fdoku.php%2Fstatistics+%28accessed+Feb+27%2C+2019%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ran, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gestwicki, J. E.</span></span> <span> </span><span class="NLM_article-title">Inhibitors of protein-protein interactions (PPIs): an analysis of scaffold choices and buried surface area</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">75</span>– <span class="NLM_lpage">86</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2018.06.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=10.1016%2Fj.cbpa.2018.06.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=29908451" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFCmsbnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2018&pages=75-86&author=X.+Ranauthor=J.+E.+Gestwicki&title=Inhibitors+of+protein-protein+interactions+%28PPIs%29%3A+an+analysis+of+scaffold+choices+and+buried+surface+area&doi=10.1016%2Fj.cbpa.2018.06.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitors of protein-protein interactions (PPIs): an analysis of scaffold choices and buried surface area</span></div><div class="casAuthors">Ran, Xu; Gestwicki, Jason E.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">75-86</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Protein-protein interactions (PPI) were once considered 'undruggable', but clin. successes, driven by advanced methods in drug discovery, have challenged that notion.  Here, we review the last three years of literature on PPI inhibitors to understand what is working and why.  From the 66 recently reported PPI inhibitors, we found that the av. mol. wt. was significantly greater than 500 Da, but that this trend was driven, in large part, by the contribution of peptide-based compds.  Despite differences in av. mol. wt., we found that compds. based on small mols. or peptides were almost equally likely to be potent inhibitors (KD < 1 μM).  Finally, we found PPIs with buried surface area (BSA) less than 2000 Å2 were more likely to be inhibited by small mols., while PPIs with larger BSA values were typically inhibited by peptides.  PPIs with BSA values over 4000 Å2 seemed to create a particular challenge, esp. for orthosteric small mols.  Thus, it seems important to choose the inhibitor scaffold based on the properties of the target interaction.  Moreover, this survey suggests a (more nuanced) conclusion to the question of whether PPIs are good drug targets; namely, that some PPIs are readily 'druggable' given the right choice of scaffold, while others still seem to deserve the 'undruggable' moniker.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpoS9hriXFuS7Vg90H21EOLACvtfcHk0liGYtM43s3UtQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFCmsbnE&md5=037a06e18fc78112243cd68ce6752599</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2018.06.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2018.06.004%26sid%3Dliteratum%253Aachs%26aulast%3DRan%26aufirst%3DX.%26aulast%3DGestwicki%26aufirst%3DJ.%2BE.%26atitle%3DInhibitors%2520of%2520protein-protein%2520interactions%2520%2528PPIs%2529%253A%2520an%2520analysis%2520of%2520scaffold%2520choices%2520and%2520buried%2520surface%2520area%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2018%26volume%3D44%26spage%3D75%26epage%3D86%26doi%3D10.1016%2Fj.cbpa.2018.06.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tice, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, S. B.</span></span> <span> </span><span class="NLM_article-title">The use of spirocyclic scaffolds in drug discovery</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">3673</span>– <span class="NLM_lpage">3682</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2014.06.081</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=10.1016%2Fj.bmcl.2014.06.081" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=25052427" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFykt7bM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2014&pages=3673-3682&author=Y.+Zhengauthor=C.+M.+Ticeauthor=S.+B.+Singh&title=The+use+of+spirocyclic+scaffolds+in+drug+discovery&doi=10.1016%2Fj.bmcl.2014.06.081"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">The use of spirocyclic scaffolds in drug discovery</span></div><div class="casAuthors">Zheng, Yajun; Tice, Colin M.; Singh, Suresh B.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">3673-3682</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Owing to their inherent three-dimensionality and structural novelty, spiro scaffolds have been increasingly utilized in drug discovery.  In this brief review, we highlight selected examples from the primary medicinal chem. literature during the last three years to demonstrate the versatility of spiro scaffolds.  With recent progress in synthetic methods providing access to spiro building blocks, spiro scaffolds are likely to be used more frequently in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqzTjJrsRkyLVg90H21EOLACvtfcHk0liGYtM43s3UtQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFykt7bM&md5=9a2a644cf2b97edf7876b43b6457f6d5</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2014.06.081&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2014.06.081%26sid%3Dliteratum%253Aachs%26aulast%3DZheng%26aufirst%3DY.%26aulast%3DTice%26aufirst%3DC.%2BM.%26aulast%3DSingh%26aufirst%3DS.%2BB.%26atitle%3DThe%2520use%2520of%2520spirocyclic%2520scaffolds%2520in%2520drug%2520discovery%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D24%26spage%3D3673%26epage%3D3682%26doi%3D10.1016%2Fj.bmcl.2014.06.081" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pammolli, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magazzini, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riccaboni, M.</span></span> <span> </span><span class="NLM_article-title">The productivity crisis in pharmaceutical R&D</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">428</span>– <span class="NLM_lpage">438</span>, <span class="refDoi"> DOI: 10.1038/nrd3405</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=10.1038%2Fnrd3405" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=21629293" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmvVegt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=428-438&author=F.+Pammolliauthor=L.+Magazziniauthor=M.+Riccaboni&title=The+productivity+crisis+in+pharmaceutical+R%26D&doi=10.1038%2Fnrd3405"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">The productivity crisis in pharmaceutical R&D</span></div><div class="casAuthors">Pammolli, Fabio; Magazzini, Laura; Riccaboni, Massimo</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">428-438</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Advances in the understanding of the mol. basis of diseases have expanded the no. of plausible therapeutic targets for the development of innovative agents in recent decades.  However, although investment in pharmaceutical research and development (R&D) has increased substantially in this time, the lack of a corresponding increase in the output in terms of new drugs being approved indicates that therapeutic innovation has become more challenging.  Here, using a large database that contains information on R&D projects for more than 28,000 compds. investigated since 1990, we examine the decline of R&D productivity in pharmaceuticals in the past two decades and its determinants.  We show that this decline is assocd. with an increasing concn. of R&D investments in areas in which the risk of failure is high, which correspond to unmet therapeutic needs and unexploited biol. mechanisms.  We also investigate the potential variations in productivity with regard to the regional location of companies and find that although companies based in the United States and Europe differ in the compn. of their R&D portfolios, there is no evidence of any productivity gap.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4lNnATFMYqLVg90H21EOLACvtfcHk0liGYtM43s3UtQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmvVegt7o%253D&md5=8adb683913f6b27f212f05ee0220ff05</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnrd3405&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3405%26sid%3Dliteratum%253Aachs%26aulast%3DPammolli%26aufirst%3DF.%26aulast%3DMagazzini%26aufirst%3DL.%26aulast%3DRiccaboni%26aufirst%3DM.%26atitle%3DThe%2520productivity%2520crisis%2520in%2520pharmaceutical%2520R%2526D%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2011%26volume%3D10%26spage%3D428%26epage%3D438%26doi%3D10.1038%2Fnrd3405" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scannell, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blanckley, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boldon, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warrington, B.</span></span> <span> </span><span class="NLM_article-title">Diagnosing the decline in pharmaceutical R&D efficiency</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">191</span>– <span class="NLM_lpage">200</span>, <span class="refDoi"> DOI: 10.1038/nrd3681</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=10.1038%2Fnrd3681" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=22378269" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADC%252BC38XivFyhtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=191-200&author=J.+W.+Scannellauthor=A.+Blanckleyauthor=H.+Boldonauthor=B.+Warrington&title=Diagnosing+the+decline+in+pharmaceutical+R%26D+efficiency&doi=10.1038%2Fnrd3681"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Diagnosing the decline in pharmaceutical R&D efficiency</span></div><div class="casAuthors">Scannell, Jack W.; Blanckley, Alex; Boldon, Helen; Warrington, Brian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">191-200</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The past 60 years have seen huge advances in many of the scientific, technol. and managerial factors that should tend to raise the efficiency of com. drug research and development (RD).  Yet the no. of new drugs approved per billion US dollars spent on RD has halved roughly every 9 years since 1950, falling around 80-fold in inflation-adjusted terms.  There have been many proposed solns. to the problem of declining RD efficiency.  However, their apparent lack of impact so far and the contrast between improving inputs and declining output in terms of the no. of new drugs make it sensible to ask whether the underlying problems have been correctly diagnosed.  Here, we discuss four factors that we consider to be primary causes, which we call the 'better than the Beatles' problem; the 'cautious regulator' problem; the 'throw money at it' tendency; and the 'basic research-brute force' bias.  Our aim is to provoke a more systematic anal. of the causes of the decline in RD efficiency.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5e_81DC6r8LVg90H21EOLACvtfcHk0liGYtM43s3UtQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XivFyhtrY%253D&md5=db3e4548260e55f47a9964e8d1b216fd</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnrd3681&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3681%26sid%3Dliteratum%253Aachs%26aulast%3DScannell%26aufirst%3DJ.%2BW.%26aulast%3DBlanckley%26aufirst%3DA.%26aulast%3DBoldon%26aufirst%3DH.%26aulast%3DWarrington%26aufirst%3DB.%26atitle%3DDiagnosing%2520the%2520decline%2520in%2520pharmaceutical%2520R%2526D%2520efficiency%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2012%26volume%3D11%26spage%3D191%26epage%3D200%26doi%3D10.1038%2Fnrd3681" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shih, H. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aronov, A. M.</span></span> <span> </span><span class="NLM_article-title">Drug discovery effectiveness from the standpoint of therapeutic mechanisms and indications</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">19</span>– <span class="NLM_lpage">33</span>, <span class="refDoi"> DOI: 10.1038/nrd.2017.194</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=10.1038%2Fnrd.2017.194" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=29075002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslansbzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2018&pages=19-33&author=H.+P.+Shihauthor=X.+Zhangauthor=A.+M.+Aronov&title=Drug+discovery+effectiveness+from+the+standpoint+of+therapeutic+mechanisms+and+indications&doi=10.1038%2Fnrd.2017.194"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Drug discovery effectiveness from the standpoint of therapeutic mechanisms and indications</span></div><div class="casAuthors">Shih, Hsin-Pei; Zhang, Xiaodan; Aronov, Alex M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">19-33</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The productivity of the pharmaceutical industry has been widely discussed in recent years, particularly with regard to concerns that substantial expenditures on research and development have failed to translate into approved drugs.  Various analyses of this productivity challenge have focused on aspects such as attrition rates at particular clin. phases or the physicochem. properties of drug candidates, but relatively little attention has been paid to how the industry has performed from the standpoint of the choice of therapeutic mechanisms and their intended indications.  This article examines what the pharmaceutical industry has achieved in this respect by analyzing comprehensive industry-wide data on the mechanism-indication pairs that have been investigated during the past 20 years.  Our findings indicate several points and trends that we hope will be useful in understanding and improving the productivity of the industry, including areas in which the industry has had substantial success or failure and the relative extent of novelty in completed and ongoing projects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmeZIs-mlGdLVg90H21EOLACvtfcHk0lh2htBc14N0rA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslansbzL&md5=aa47b6bbc0c2ee0fbb51927ce9b9b369</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2017.194&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2017.194%26sid%3Dliteratum%253Aachs%26aulast%3DShih%26aufirst%3DH.%2BP.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DAronov%26aufirst%3DA.%2BM.%26atitle%3DDrug%2520discovery%2520effectiveness%2520from%2520the%2520standpoint%2520of%2520therapeutic%2520mechanisms%2520and%2520indications%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2018%26volume%3D17%26spage%3D19%26epage%3D33%26doi%3D10.1038%2Fnrd.2017.194" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pammolli, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magazzini, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riccaboni, M.</span></span> <span> </span><span class="NLM_article-title">The productivity crisis in pharmaceutical R&D</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">428</span>– <span class="NLM_lpage">438</span>, <span class="refDoi"> DOI: 10.1038/nrd3405</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=10.1038%2Fnrd3405" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=21629293" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmvVegt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=428-438&author=F.+Pammolliauthor=L.+Magazziniauthor=M.+Riccaboni&title=The+productivity+crisis+in+pharmaceutical+R%26D&doi=10.1038%2Fnrd3405"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">The productivity crisis in pharmaceutical R&D</span></div><div class="casAuthors">Pammolli, Fabio; Magazzini, Laura; Riccaboni, Massimo</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">428-438</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Advances in the understanding of the mol. basis of diseases have expanded the no. of plausible therapeutic targets for the development of innovative agents in recent decades.  However, although investment in pharmaceutical research and development (R&D) has increased substantially in this time, the lack of a corresponding increase in the output in terms of new drugs being approved indicates that therapeutic innovation has become more challenging.  Here, using a large database that contains information on R&D projects for more than 28,000 compds. investigated since 1990, we examine the decline of R&D productivity in pharmaceuticals in the past two decades and its determinants.  We show that this decline is assocd. with an increasing concn. of R&D investments in areas in which the risk of failure is high, which correspond to unmet therapeutic needs and unexploited biol. mechanisms.  We also investigate the potential variations in productivity with regard to the regional location of companies and find that although companies based in the United States and Europe differ in the compn. of their R&D portfolios, there is no evidence of any productivity gap.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4lNnATFMYqLVg90H21EOLACvtfcHk0lh2htBc14N0rA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmvVegt7o%253D&md5=8adb683913f6b27f212f05ee0220ff05</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnrd3405&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3405%26sid%3Dliteratum%253Aachs%26aulast%3DPammolli%26aufirst%3DF.%26aulast%3DMagazzini%26aufirst%3DL.%26aulast%3DRiccaboni%26aufirst%3DM.%26atitle%3DThe%2520productivity%2520crisis%2520in%2520pharmaceutical%2520R%2526D%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2011%26volume%3D10%26spage%3D428%26epage%3D438%26doi%3D10.1038%2Fnrd3405" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Waring, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leach, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leeson, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mandrell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owen, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pairaudeau, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pennie, W. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pickett, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wallace, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weir, A.</span></span> <span> </span><span class="NLM_article-title">An analysis of the attrition of drug candidates from four major pharmaceutical companies</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">475</span>– <span class="NLM_lpage">486</span>, <span class="refDoi"> DOI: 10.1038/nrd4609</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=10.1038%2Fnrd4609" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=26091267" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7jM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2015&pages=475-486&author=M.+J.+Waringauthor=J.+Arrowsmithauthor=A.+R.+Leachauthor=P.+D.+Leesonauthor=S.+Mandrellauthor=R.+M.+Owenauthor=G.+Pairaudeauauthor=W.+D.+Pennieauthor=S.+D.+Pickettauthor=J.+Wangauthor=O.+Wallaceauthor=A.+Weir&title=An+analysis+of+the+attrition+of+drug+candidates+from+four+major+pharmaceutical+companies&doi=10.1038%2Fnrd4609"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">An analysis of the attrition of drug candidates from four major pharmaceutical companies</span></div><div class="casAuthors">Waring, Michael J.; Arrowsmith, John; Leach, Andrew R.; Leeson, Paul D.; Mandrell, Sam; Owen, Robert M.; Pairaudeau, Garry; Pennie, William D.; Pickett, Stephen D.; Wang, Jibo; Wallace, Owen; Weir, Alex</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">475-486</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The pharmaceutical industry remains under huge pressure to address the high attrition rates in drug development.  Attempts to reduce the no. of efficacy- and safety-related failures by analyzing possible links to the physicochem. properties of small-mol. drug candidates have been inconclusive because of the limited size of data sets from individual companies.  Here, we describe the compilation and anal. of combined data on the attrition of drug candidates from AstraZeneca, Eli Lilly and Company, GlaxoSmithKline and Pfizer.  The anal. reaffirms that control of physicochem. properties during compd. optimization is beneficial in identifying compds. of candidate drug quality and indicates for the first time a link between the physicochem. properties of compds. and clin. failure due to safety issues.  The results also suggest that further control of physicochem. properties is unlikely to have a significant effect on attrition rates and that addnl. work is required to address safety-related failures.  Further cross-company collaborations will be crucial to future progress in this area.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsLfBWrLC3ZLVg90H21EOLACvtfcHk0lh2htBc14N0rA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7jM&md5=1fdc374d32816b1e91438152299dd1b1</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fnrd4609&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4609%26sid%3Dliteratum%253Aachs%26aulast%3DWaring%26aufirst%3DM.%2BJ.%26aulast%3DArrowsmith%26aufirst%3DJ.%26aulast%3DLeach%26aufirst%3DA.%2BR.%26aulast%3DLeeson%26aufirst%3DP.%2BD.%26aulast%3DMandrell%26aufirst%3DS.%26aulast%3DOwen%26aufirst%3DR.%2BM.%26aulast%3DPairaudeau%26aufirst%3DG.%26aulast%3DPennie%26aufirst%3DW.%2BD.%26aulast%3DPickett%26aufirst%3DS.%2BD.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWallace%26aufirst%3DO.%26aulast%3DWeir%26aufirst%3DA.%26atitle%3DAn%2520analysis%2520of%2520the%2520attrition%2520of%2520drug%2520candidates%2520from%2520four%2520major%2520pharmaceutical%2520companies%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2015%26volume%3D14%26spage%3D475%26epage%3D486%26doi%3D10.1038%2Fnrd4609" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mirasol, F.</span></span> 2016 NME Drug Approvals Drop After Recent Highs. <a href="https://www.dcatvci.org/286-2016-nme-drug-approvals-drop-after-recent-highs" class="extLink">https://www.dcatvci.org/286-2016-nme-drug-approvals-drop-after-recent-highs</a> (accessed Feb 27, 2019).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Mirasol%2C+F.+2016+NME+Drug+Approvals+Drop+After+Recent+Highs.+https%3A%2F%2Fwww.dcatvci.org%2F286-2016-nme-drug-approvals-drop-after-recent-highs+%28accessed+Feb+27%2C+2019%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26aulast%3DMirasol%26aufirst%3DF." title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kola, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landis, J.</span></span> <span> </span><span class="NLM_article-title">Can the pharmaceutical industry reduce attrition rates?</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">711</span>– <span class="NLM_lpage">715</span>, <span class="refDoi"> DOI: 10.1038/nrd1470</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=10.1038%2Fnrd1470" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=15286737" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmtVOhtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2004&pages=711-715&author=I.+Kolaauthor=J.+Landis&title=Can+the+pharmaceutical+industry+reduce+attrition+rates%3F&doi=10.1038%2Fnrd1470"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Opinion: Can the pharmaceutical industry reduce attrition rates?</span></div><div class="casAuthors">Kola, Ismail; Landis, John</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">711-716</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The pharmaceutical industry faces considerable challenges, both politically and fiscally.  Politically, governments around the world are trying to contain costs and, as health care budgets constitute a very significant part of governmental spending, these costs are the subject of intense scrutiny.  In the United States, drug costs are also the subject of intense political discourse.  This article deals with the fiscal pressures that face the industry from the perspective of R&D.  What impinges on productivity How can we improve current reduced R&D productivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruAlrpFPFmiLVg90H21EOLACvtfcHk0lhbw7AqkkM1lg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmtVOhtLs%253D&md5=f9025c13a1506f607aaf68415570ed01</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fnrd1470&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd1470%26sid%3Dliteratum%253Aachs%26aulast%3DKola%26aufirst%3DI.%26aulast%3DLandis%26aufirst%3DJ.%26atitle%3DCan%2520the%2520pharmaceutical%2520industry%2520reduce%2520attrition%2520rates%253F%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2004%26volume%3D3%26spage%3D711%26epage%3D715%26doi%3D10.1038%2Fnrd1470" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lipinski, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lombardo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dominy, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feeney, P. J.</span></span> <span> </span><span class="NLM_article-title">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span>. <i>Adv. Drug Delivery Rev.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">3</span>– <span class="NLM_lpage">25</span>, <span class="refDoi"> DOI: 10.1016/S0169-409X(96)00423-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=10.1016%2FS0169-409X%2896%2900423-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADyaK2sXktlKlsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=1997&pages=3-25&author=C.+A.+Lipinskiauthor=F.+Lombardoauthor=B.+W.+Dominyauthor=P.+J.+Feeney&title=Experimental+and+computational+approaches+to+estimate+solubility+and+permeability+in+drug+discovery+and+development+settings&doi=10.1016%2FS0169-409X%2896%2900423-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span></div><div class="casAuthors">Lipinski, Christopher A.; Lombardo, Franco; Dominy, Beryl W.; Feeney, Paul J.</div><div class="citationInfo"><span class="NLM_cas:title">Advanced Drug Delivery Reviews</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">3-25</span>CODEN:
                <span class="NLM_cas:coden">ADDREP</span>;
        ISSN:<span class="NLM_cas:issn">0169-409X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review with 50 refs.  Exptl. and computational approaches to est. soly. and permeability in discovery and development settings are described.  In the discovery setting 'the rule of 5' predicts that poor absorption or permeation is more likely when there are more than 5 H-bond donors, 10 H-bond acceptors, the mol. wt. (MWT) is >500 and the calcd. Log P (CLogP) is >5.  Computational methodol. for the rule-based Moriguchi Log P (MLogP) calcn. is described.  Turbidimetric soly. measurement is described and applied to known drugs.  High throughput screening (HTS) leads tend to have higher MWT and Log P and lower turbidimetric soly. than leads in the pre-HTS era.  In the development setting, soly. calcns. focus on exact value prediction and are difficult because of polymorphism.  Recent work on linear free energy relationships and Log P approaches are critically reviewed.  Useful predictions are possible in closely related analog series when coupled with exptl. thermodn. soly. measurements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGon9gEeZ7cboLVg90H21EOLACvtfcHk0lhbw7AqkkM1lg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXktlKlsQ%253D%253D&md5=405f70b0594d428f1275e1d56642cd3a</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2FS0169-409X%2896%2900423-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0169-409X%252896%252900423-1%26sid%3Dliteratum%253Aachs%26aulast%3DLipinski%26aufirst%3DC.%2BA.%26aulast%3DLombardo%26aufirst%3DF.%26aulast%3DDominy%26aufirst%3DB.%2BW.%26aulast%3DFeeney%26aufirst%3DP.%2BJ.%26atitle%3DExperimental%2520and%2520computational%2520approaches%2520to%2520estimate%2520solubility%2520and%2520permeability%2520in%2520drug%2520discovery%2520and%2520development%2520settings%26jtitle%3DAdv.%2520Drug%2520Delivery%2520Rev.%26date%3D1997%26volume%3D23%26spage%3D3%26epage%3D25%26doi%3D10.1016%2FS0169-409X%2896%2900423-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Degoey, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wendt, M. D.</span></span> <span> </span><span class="NLM_article-title">Beyond the Rule of 5: lessons learned from AbbVie’s drugs and compound collection</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">2636</span>– <span class="NLM_lpage">2651</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00717</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00717" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFeitbjJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=2636-2651&author=D.+A.+Degoeyauthor=H.+Chenauthor=P.+B.+Coxauthor=M.+D.+Wendt&title=Beyond+the+Rule+of+5%3A+lessons+learned+from+AbbVie%E2%80%99s+drugs+and+compound+collection&doi=10.1021%2Facs.jmedchem.7b00717"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Beyond the Rule of 5: Lessons Learned from AbbVie's Drugs and Compound Collection</span></div><div class="casAuthors">DeGoey, David A.; Chen, Hui-Ju; Cox, Philip B.; Wendt, Michael D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2636-2651</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Recently, there has been an increasing focus on the pursuit of targets considered to be less druggable that offer potential for development of promising new therapeutic agents for the treatment of diseases with large unmet medical need, particularly in the areas of oncol. and virol.  However, conducting drug discovery campaigns in "beyond rule of 5" (bRo5) chem. space presents a significant drug design and development challenge to medicinal chemists to achieve acceptable oral pharmacokinetics.  Retrospective anal. of past successes and failures in drug discovery bRo5 may shed light on the key principles that contribute to the oral bioavailability of successful bRo5 compds. and improve the efficiency of drug design for future projects.  We present here highlights and case studies of lessons learned from discovery of bRo5 compds.  A simple multiparametric scoring function (AB-MPS) was devised that correlated preclin. PK results with cLogD, no. of rotatable bonds, and no. of arom. rings.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTHuaXbtjlALVg90H21EOLACvtfcHk0lhbw7AqkkM1lg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFeitbjJ&md5=48984dc65410d83d4ac8ddaca030b024</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00717&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00717%26sid%3Dliteratum%253Aachs%26aulast%3DDegoey%26aufirst%3DD.%2BA.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DCox%26aufirst%3DP.%2BB.%26aulast%3DWendt%26aufirst%3DM.%2BD.%26atitle%3DBeyond%2520the%2520Rule%2520of%25205%253A%2520lessons%2520learned%2520from%2520AbbVie%25E2%2580%2599s%2520drugs%2520and%2520compound%2520collection%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D2636%26epage%3D2651%26doi%3D10.1021%2Facs.jmedchem.7b00717" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lovering, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bikker, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humblet, C.</span></span> <span> </span><span class="NLM_article-title">Escape from flatland: increasing saturation as an approach to improving clinical success</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">6752</span>– <span class="NLM_lpage">6756</span>, <span class="refDoi"> DOI: 10.1021/jm901241e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901241e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1KjtLvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=6752-6756&author=F.+Loveringauthor=J.+Bikkerauthor=C.+Humblet&title=Escape+from+flatland%3A+increasing+saturation+as+an+approach+to+improving+clinical+success&doi=10.1021%2Fjm901241e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Escape from Flatland: Increasing Saturation as an Approach to Improving Clinical Success</span></div><div class="casAuthors">Lovering, Frank; Bikker, Jack; Humblet, Christine</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">6752-6756</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The medicinal chem. community has become increasingly aware of the value of tracking calcd. phys. properties such as mol. wt., topol. polar surface area, rotatable bonds, and hydrogen bond donors and acceptors.  The authors hypothesized that the shift to high-throughput synthetic practices over the past decade may be another factor that may predispose mols. to fail by steering discovery efforts toward achiral, arom. compds.  The authors have proposed two simple and interpretable measures of the complexity of mols. prepd. as potential drug candidates.  The first is carbon bond satn. as defined by fraction Sp3 (Fsp3) where Fsp3 = (no. of Sp3 hybridized carbons/total carbon count).  The second is simply whether a chiral carbon exists in the mol.  The authors demonstrate that both complexity (as measured by Fsp3) and the presence of chiral centers correlate with success as compds. transition from discovery, through clin. testing, to drugs.  To explain these observations, the authors further demonstrate that satn. correlates with soly., an exptl. phys. property important to success in the drug discovery setting.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoC1f3Ed7nUBbVg90H21EOLACvtfcHk0lhWYuD8dSXuQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1KjtLvN&md5=4ca92c30c17c53d77ad376719bad951e</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fjm901241e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901241e%26sid%3Dliteratum%253Aachs%26aulast%3DLovering%26aufirst%3DF.%26aulast%3DBikker%26aufirst%3DJ.%26aulast%3DHumblet%26aufirst%3DC.%26atitle%3DEscape%2520from%2520flatland%253A%2520increasing%2520saturation%2520as%2520an%2520approach%2520to%2520improving%2520clinical%2520success%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D6752%26epage%3D6756%26doi%3D10.1021%2Fjm901241e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Allu, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oprea, T. I.</span></span> <span> </span><span class="NLM_article-title">Rapid evaluation of synthetic and molecular complexity for in silico chemistry</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">1237</span>– <span class="NLM_lpage">1243</span>, <span class="refDoi"> DOI: 10.1021/ci0501387</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci0501387" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADC%252BD2MXmsFCqtbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2005&pages=1237-1243&author=T.+K.+Alluauthor=T.+I.+Oprea&title=Rapid+evaluation+of+synthetic+and+molecular+complexity+for+in+silico+chemistry&doi=10.1021%2Fci0501387"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid Evaluation of Synthetic and Molecular Complexity for in Silico Chemistry</span></div><div class="casAuthors">Allu, Tharun Kumar; Oprea, Tudor I.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1237-1243</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Methods that rapidly evaluate mol. complexity and synthetic feasibility are becoming increasingly important for in silico chem.  We propose a new metric based on relative at. electronegativities and bond parameters that evaluate both synthetic and mol. complexity (SMCM) starting from chem. structures.  Against mol. wt., SMCM has the lowest fraction of adjusted variance (R2=0.535) on a series of 261 048 diverse compds., when compared to the complexity metric of Baron and Chanon (R2=0.777; J. Chem. Inf. Comput. Sci. 2001, 41, 269-272) and Ruecker (R2=0.895 for log complexity values; J. Chem. Inf. Comput. Sci. 2004, 44, 378-386), resp.  These metrics are in general agreement when the metabolic synthesis of cholesterol from S-3-hydroxy-3-methyl-glutaryl CoA is monitored, indicating that SMCM can be useful in discerning increases in complexity.  Because the presence of substructure patterns can be directly incorporated into this scheme, SMCM is relatively straightforward and can be easily tailored to rapidly evaluate virtual (combinatorial) libraries and high throughput screening hits.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpsYN3I-eGvKbVg90H21EOLACvtfcHk0lhWYuD8dSXuQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXmsFCqtbc%253D&md5=67e9ee49ad8be800d3a57d98adafab97</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fci0501387&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci0501387%26sid%3Dliteratum%253Aachs%26aulast%3DAllu%26aufirst%3DT.%2BK.%26aulast%3DOprea%26aufirst%3DT.%2BI.%26atitle%3DRapid%2520evaluation%2520of%2520synthetic%2520and%2520molecular%2520complexity%2520for%2520in%2520silico%2520chemistry%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2005%26volume%3D45%26spage%3D1237%26epage%3D1243%26doi%3D10.1021%2Fci0501387" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bertz, S. H.</span></span> <span> </span><span class="NLM_article-title">The first general index of molecular complexity</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1981</span>,  <span class="NLM_volume"><i>103</i></span>,  <span class="NLM_fpage">3599</span>– <span class="NLM_lpage">3603</span>, <span class="refDoi"> DOI: 10.1021/ja00402a071</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00402a071" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADyaL3MXkt1Cjtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=1981&pages=3599-3603&author=S.+H.+Bertz&title=The+first+general+index+of+molecular+complexity&doi=10.1021%2Fja00402a071"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">The first general index of molecular complexity</span></div><div class="casAuthors">Bertz, Steven H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1981</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3599-601</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    </div><div class="casAbstract">An index of mol. complexity is introduced which is based upon the concepts of graph theory and information theory.  It is the only one extant that is completely general, being applicable to compds. contg. such features as branching, rings, multiple bonds, and heteroatoms.  It also reflects the sym. present in a mol.  All these features are important in synthetic anal., as illustrated by an example including sample calcns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXuz8eEH2Cl7Vg90H21EOLACvtfcHk0lhWYuD8dSXuQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3MXkt1Cjtbk%253D&md5=9a7602510035ab906839fd67a47148e7</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fja00402a071&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00402a071%26sid%3Dliteratum%253Aachs%26aulast%3DBertz%26aufirst%3DS.%2BH.%26atitle%3DThe%2520first%2520general%2520index%2520of%2520molecular%2520complexity%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1981%26volume%3D103%26spage%3D3599%26epage%3D3603%26doi%3D10.1021%2Fja00402a071" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barone, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chanon, M.</span></span> <span> </span><span class="NLM_article-title">A new and simple approach to chemical complexity. application to the synthesis of natural products</span>. <i>J. Chem. Inf. Comput. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">269</span>– <span class="NLM_lpage">272</span>, <span class="refDoi"> DOI: 10.1021/ci000145p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci000145p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADC%252BD3MXosVGjuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2001&pages=269-272&author=R.+Baroneauthor=M.+Chanon&title=A+new+and+simple+approach+to+chemical+complexity.+application+to+the+synthesis+of+natural+products&doi=10.1021%2Fci000145p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">A New and Simple Approach to Chemical Complexity. Application to the Synthesis of Natural Products</span></div><div class="casAuthors">Barone, Rene; Chanon, Michel</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">269-272</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A new approach to describe the complexity of chem. structures is proposed.  This index is simple and can be easily programmed.  It derives from Whitlock's index and, despite its empirical character, provides results paralleling those obtained with the math. Bertz index.  It has been applied to follow the strategic progress of some natural product syntheses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqE3_LE6fy8jbVg90H21EOLACvtfcHk0lhWYuD8dSXuQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXosVGjuw%253D%253D&md5=92ae068a5f0102499a566e0d1c62e39c</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fci000145p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci000145p%26sid%3Dliteratum%253Aachs%26aulast%3DBarone%26aufirst%3DR.%26aulast%3DChanon%26aufirst%3DM.%26atitle%3DA%2520new%2520and%2520simple%2520approach%2520to%2520chemical%2520complexity.%2520application%2520to%2520the%2520synthesis%2520of%2520natural%2520products%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D2001%26volume%3D41%26spage%3D269%26epage%3D272%26doi%3D10.1021%2Fci000145p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schuffenhauer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Selzer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ertl, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacoby, E.</span></span> <span> </span><span class="NLM_article-title">Relationships between molecular complexity, biological activity, and structural diversity</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">525</span>– <span class="NLM_lpage">535</span>, <span class="refDoi"> DOI: 10.1021/ci0503558</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci0503558" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlShtLvF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2006&pages=525-535&author=A.+Schuffenhauerauthor=N.+Brownauthor=P.+Selzerauthor=P.+Ertlauthor=E.+Jacoby&title=Relationships+between+molecular+complexity%2C+biological+activity%2C+and+structural+diversity&doi=10.1021%2Fci0503558"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Relationships between Molecular Complexity, Biological Activity, and Structural Diversity</span></div><div class="casAuthors">Schuffenhauer, Ansgar; Brown, Nathan; Selzer, Paul; Ertl, Peter; Jacoby, Edgar</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">525-535</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Following the theor. model by Hann et al. moderately complex structures are preferable lead compds. since they lead to specific binding events involving the complete ligand mol.  To make this concept usable in practice for library design, the authors studied several complexity measures on the biol. activity of ligand mols.  The authors applied the historical IC50/EC50 summary data of 160 assays run at Novartis covering a diverse range of targets, among them kinases, proteases, GPCRs, and protein-protein interactions, and compared this to the background of "inactive" compds. which have been screened for 2 years but have never shown any activity in any primary screen.  As complexity measures the authors used the no. of structural features present in various mol. fingerprints and descriptors.  The authors found generally that with increasing activity of the ligands, their av. complexity also increased, and the authors could therefore establish a min. no. of structural features in each descriptor needed for biol. activity.  Esp. well suited in this context were the Similog keys and circular substructure fingerprints.  These are those descriptors, which also perform esp. well in the identification of bioactive compds. by similarity search, suggesting that structural features encoded in these descriptors have a high relevance for bioactivity.  Since the no. of features correlates with the no. of atoms present in the mol., also the no. of atoms serves as a reasonable complexity measure and larger mols. have, in general, higher activities.  Due to the relationship between feature counts and densities on one hand and biol. activity on the other, the size bias present in almost all similarity coeffs. becomes esp. important.  Diversity selections using these coeffs. can influence the overall complexity of the resulting set of mols., which has an impact on the biol. activity that they exhibit.  Using sphere-exclusion based diversity selection methods, such as OptiSim together with the Tanimoto dissimilarity, the av. feature count distribution of the resulting selections is shifted toward lower complexity than that of the original set, particularly when applying tight diversity constraints.  This size bias reduces the fraction of mols. in the subsets having the complexity required for a high, submicromolar activity.  None of the diversity selection methods studied, namely OptiSim, divisive K-means clustering, and self-organizing maps, yielded subsets covering the activity space of the IC50 summary data set better than subsets selected randomly.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrniA9sx4FiTbVg90H21EOLACvtfcHk0lhQYO3dL_jrrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtlShtLvF&md5=9647fda24ecda3ec961621c775b8ebcc</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fci0503558&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci0503558%26sid%3Dliteratum%253Aachs%26aulast%3DSchuffenhauer%26aufirst%3DA.%26aulast%3DBrown%26aufirst%3DN.%26aulast%3DSelzer%26aufirst%3DP.%26aulast%3DErtl%26aufirst%3DP.%26aulast%3DJacoby%26aufirst%3DE.%26atitle%3DRelationships%2520between%2520molecular%2520complexity%252C%2520biological%2520activity%252C%2520and%2520structural%2520diversity%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2006%26volume%3D46%26spage%3D525%26epage%3D535%26doi%3D10.1021%2Fci0503558" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Feher, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, J. M.</span></span> <span> </span><span class="NLM_article-title">Property distributions: differences between drugs, natural products, and molecules from combinatorial chemistry</span>. <i>J. Chem. Inf. Comput. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">218</span>– <span class="NLM_lpage">227</span>, <span class="refDoi"> DOI: 10.1021/ci0200467</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci0200467" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADC%252BD38XptlOksLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2003&pages=218-227&author=M.+Feherauthor=J.+M.+Schmidt&title=Property+distributions%3A+differences+between+drugs%2C+natural+products%2C+and+molecules+from+combinatorial+chemistry&doi=10.1021%2Fci0200467"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Property Distributions: Differences between Drugs, Natural Products, and Molecules from Combinatorial Chemistry</span></div><div class="casAuthors">Feher, Miklos; Schmidt, Jonathan M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">218-227</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The differences between three different compd. classes, natural products, mols. from combinatorial synthesis, and drug mols., were investigated.  The major structural differences between natural and combinatorial compds. originate mainly from properties introduced to make combinatorial synthesis more efficient.  These include the no. of chiral centers, the prevalence of arom. rings, the introduction of complex ring systems, and the degree of the satn. of the mol. as well as the no. and ratios of different heteroatoms.  As drug mols. derive from both natural and synthetic sources, they cover a joint area in property space of natural and combinatorial compds.  A PCA-based scheme is presented that differentiates the three classes of compds.  It is suggested that by mimicking certain distribution properties of natural compds., combinatorial products might be made that are substantially more diverse and have greater biol. relevance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpma3fkS14voLVg90H21EOLACvtfcHk0lhQYO3dL_jrrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XptlOksLg%253D&md5=eb86ed83699fc487dfb7f2cb55296e69</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fci0200467&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci0200467%26sid%3Dliteratum%253Aachs%26aulast%3DFeher%26aufirst%3DM.%26aulast%3DSchmidt%26aufirst%3DJ.%2BM.%26atitle%3DProperty%2520distributions%253A%2520differences%2520between%2520drugs%252C%2520natural%2520products%252C%2520and%2520molecules%2520from%2520combinatorial%2520chemistry%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D2003%26volume%3D43%26spage%3D218%26epage%3D227%26doi%3D10.1021%2Fci0200467" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Selzer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ertl, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuffenhauer, A.</span></span> <span> </span><span class="NLM_article-title">Complex molecules: do they add value?</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">310</span>– <span class="NLM_lpage">316</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2005.04.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=10.1016%2Fj.cbpa.2005.04.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=15939334" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADC%252BD2MXltVaqt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2005&pages=310-316&author=P.+Selzerauthor=H.+J.+Rothauthor=P.+Ertlauthor=A.+Schuffenhauer&title=Complex+molecules%3A+do+they+add+value%3F&doi=10.1016%2Fj.cbpa.2005.04.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Complex molecules: do they add value?</span></div><div class="casAuthors">Selzer, Paul; Roth, Hans-Joerg; Ertl, Peter; Schuffenhauer, Ansgar</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">310-316</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The concept of complexity in chem. has received much interest in the research community.  Various measures to assess mol. complexity have been published, ranging from abstr. complexity definitions to very specific application-oriented definitions.  In this article we focus on mol. complexity in relation to biol. activity.  Connectivity and feature-based structural descriptors have been evaluated with ref. to their potential as complexity measures.  Our goal was to discuss the potential of the complexity concept to support the drug discovery process, helping to design suitable lead candidates.  The studies have shown that highly active compds., on av., are more complex than inactive compds.  However, complexity must be balanced with other mol. properties because more complex mols. have a higher probability to exhibit pharmacokinetic problems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqk3nmT5dNhrbVg90H21EOLACvtfcHk0lhQYO3dL_jrrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXltVaqt7k%253D&md5=3dfb7de5351efa8b0e9b8e3dfd98f422</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2005.04.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2005.04.001%26sid%3Dliteratum%253Aachs%26aulast%3DSelzer%26aufirst%3DP.%26aulast%3DRoth%26aufirst%3DH.%2BJ.%26aulast%3DErtl%26aufirst%3DP.%26aulast%3DSchuffenhauer%26aufirst%3DA.%26atitle%3DComplex%2520molecules%253A%2520do%2520they%2520add%2520value%253F%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2005%26volume%3D9%26spage%3D310%26epage%3D316%26doi%3D10.1016%2Fj.cbpa.2005.04.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Whitlock, H. W.</span></span> <span> </span><span class="NLM_article-title">On the structure of total synthesis of complex natural products</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">7982</span>– <span class="NLM_lpage">7989</span>, <span class="refDoi"> DOI: 10.1021/jo9814546</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo9814546" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADyaK1cXmtlanu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=1998&pages=7982-7989&author=H.+W.+Whitlock&title=On+the+structure+of+total+synthesis+of+complex+natural+products&doi=10.1021%2Fjo9814546"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">On the Structure of Total Synthesis of Complex Natural Products</span></div><div class="casAuthors">Whitlock, H. W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">7982-7989</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A chem. intuitive measure (metric) for mol. complexity is described.  It is then applied to several syntheses of complex natural products.  The authors conclude that the metric corresponds well with chem. ideas of complexity, and that different syntheses do in fact have appreciably different logical structures as measured by this.  Possible implications of this interesting observation are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxo_c0K7vuF7Vg90H21EOLACvtfcHk0lgAc8m8QDeB1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXmtlanu7Y%253D&md5=5d0bc62e676d8951e30460e7182c4557</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Fjo9814546&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo9814546%26sid%3Dliteratum%253Aachs%26aulast%3DWhitlock%26aufirst%3DH.%2BW.%26atitle%3DOn%2520the%2520structure%2520of%2520total%2520synthesis%2520of%2520complex%2520natural%2520products%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D1998%26volume%3D63%26spage%3D7982%26epage%3D7989%26doi%3D10.1021%2Fjo9814546" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rucker, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rucker, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertz, S. H.</span></span> <span> </span><span class="NLM_article-title">Organic synthesis - art or science?</span>. <i>J. Chem. Inf. Comput. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">378</span>– <span class="NLM_lpage">386</span>, <span class="refDoi"> DOI: 10.1021/ci030415e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci030415e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A280%3ADC%252BD2c7ktFajtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2004&pages=378-386&author=C.+Ruckerauthor=G.+Ruckerauthor=S.+H.+Bertz&title=Organic+synthesis+-+art+or+science%3F&doi=10.1021%2Fci030415e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Organic synthesis--art or science?</span></div><div class="casAuthors">Rucker Christoph; Rucker Gerta; Bertz Steven H</div><div class="citationInfo"><span class="NLM_cas:title">Journal of chemical information and computer sciences</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">378-86</span>
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    </div><div class="casAbstract">The LHASA rules for finding strategic bonds in polycyclic target structures are analyzed with respect to the following question: Do the strategic bonds tend to give the greatest simplification upon disconnection, as measured by recently introduced indices of molecular complexity? The answer is yes, at least for the more general rules.  This result implies that the bonds most useful for retrosynthetic disconnection can now be identified by a simple calculation rather than by application of a body of rules.  It is concluded that organic synthesis, as far as described by these rules, has a mathematical basis and consequently can be considered a science as well as an art.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTG-WUYX89vCNNTrvPZP1IpfW6udTcc2eYM26hv8hhguLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2c7ktFajtQ%253D%253D&md5=7b5c58ae6b96398d2263fdee0d700651</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fci030415e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci030415e%26sid%3Dliteratum%253Aachs%26aulast%3DRucker%26aufirst%3DC.%26aulast%3DRucker%26aufirst%3DG.%26aulast%3DBertz%26aufirst%3DS.%2BH.%26atitle%3DOrganic%2520synthesis%2520-%2520art%2520or%2520science%253F%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D2004%26volume%3D44%26spage%3D378%26epage%3D386%26doi%3D10.1021%2Fci030415e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span>ChemDiv
Inc. Home Page. <a href="https://chemdiv.com" class="extLink">https://chemdiv.com</a> (accessed Feb 27, <span class="NLM_year">2019</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=ChemDiv%0AInc.+Home+Page.+https%3A%2F%2Fchemdiv.com+%28accessed+Feb+27%2C+2019%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Delfort, B.</span>; <span class="NLM_string-name">Le Pennec, D.</span>; <span class="NLM_string-name">Grandjean, J.</span></span> <span> </span><span class="NLM_article-title">A Method of Synthesizing a Mixture of N,N,N′,N′-Tetramethyl-1,6-hexanediamine and N,N,N′,N′-Tetramethyldiaminoethers</span>. France Patent <span class="NLM_patent">FR3014100A1</span>, <span class="NLM_year">2013</span>. 1,2-Bis(2-dimethylaminoethoxy)ethane can be obtained in two reaction steps: the reaction of commercially available triethylene glycol with ammonia to yield 1,2-bis(2-aminoethoxy)ethane, a well-known reaction, is generally carried out in the presence of hydrogen and a suitable catalyst, followed by an aminoalkylation reaction, for example, by reductive amination using formaldehyde in the presence of hydrogen and a suitable catalyst.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&author=B.+Delfort&author=D.+Le+Pennec&author=J.+Grandjean&title=A+Method+of+Synthesizing+a+Mixture+of+N%2CN%2CN%E2%80%B2%2CN%E2%80%B2-Tetramethyl-1%2C6-hexanediamine+and+N%2CN%2CN%E2%80%B2%2CN%E2%80%B2-Tetramethyldiaminoethers"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DDelfort%26aufirst%3DB.%26atitle%3DA%2520Method%2520of%2520Synthesizing%2520a%2520Mixture%2520of%2520N%252CN%252CN%25E2%2580%25B2%252CN%25E2%2580%25B2-Tetramethyl-1%252C6-hexanediamine%2520and%2520N%252CN%252CN%25E2%2580%25B2%252CN%25E2%2580%25B2-Tetramethyldiaminoethers%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, F.</span>; <span class="NLM_string-name">Deng, Z.</span>; <span class="NLM_string-name">Li, S.</span></span> <span> </span><span class="NLM_article-title">Preparation Method of <i>N</i>-<i>tert</i>-Butyloxycarbonyl-3-methylaminothiophene.</span> China Patent <span class="NLM_patent">104193722A</span>, <span class="NLM_year">2014</span>. The compound can be synthesized in three steps using 3-thienylformaldehyde as an initial raw material: (1) NaBH<sub>4</sub>, MeOH, 25 °C, 5 h; (2) HBr, H<sub>2</sub>O/D, 12 h; (3) NH<sub>3</sub>, DMF, 150 °C, 18 h.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=F.+Chen&author=Z.+Deng&author=S.+Li&title=Preparation+Method+of+N-tert-Butyloxycarbonyl-3-methylaminothiophene."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DF.%26atitle%3DPreparation%2520Method%2520of%2520N-tert-Butyloxycarbonyl-3-methylaminothiophene.%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span>This
compound was prepared in ChemDiv by analogy with the synthetic procedure
described in <span class="NLM_contrib-group"><span class="NLM_string-name">Klingsberg, E.</span></span> <span> </span><span class="NLM_article-title">Preparation of triaryl-<i>s</i>-triazoles from diaroylhydrazines</span>.  <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1958</span>,  <span class="NLM_volume">23</span>,  <span class="NLM_fpage">1086</span>– <span class="NLM_lpage">1087</span>, <span class="refDoi"> DOI: 10.1021/jo01101a629</span> , starting from commercially available <i>N</i>,<i>N</i>′-di(pyridine-2-carbonyl)hydrazide in two steps.</span><div class="citationLinks">[<a href="/doi/10.1021/jo01101a629" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADyaG1MXovFCisA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=1958&pages=1086-1087&author=E.+Klingsberg&title=Preparation+of+triaryl-s-triazoles+from+diaroylhydrazines&doi=10.1021%2Fjo01101a629"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Preparation of triaryl-s-triazoles from diaroylhydrazines</span></div><div class="casAuthors">Klingsberg, Erwin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of  Organic Chemistry</span>
        (<span class="NLM_cas:date">1958</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1086-7</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    </div><div class="casAbstract">Arylphosphazoanilides, from the amine and PCl3, reacted with diaroylhydrazones to give triaryl-s-triazoles in good yields.  Thus, to 0.06 mole Ar1NH2 in 20-30 ml. ο-C6H4Cl2 was added 1.51 g. PCl3, and warmed on the steam bath, 0.01 mole Ar2CONHNHCOPh added, the whole refluxed 3 hrs. at 190-200°, cooled and digested in H2O or dil. HCl, and recrystd.  (AcOH) to give Ar1N.CAr2:N.N:CPh.  Ar1, Ar2, m.p., % yield given: Ph, Ph, 299-300°, 95; p-MeC6H4, Ph, 301-2°, 88; ο-MeC6H4, Ph, 191-2°, 72; m-MeC6H4, Ph, 256-7°, 83; p-ClC6H4, Ph, 261.5-2.0, 88; Ph, 4-pyridyl, 268-9°, 95; 2-napthyl, Ph, 275.5-6.5°, 52.  Cyclohexylamine and (PhCONH)2 gave diphenyloxadiazole.  AcNHNHCOPh and PhNH2 gave no product.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMtzQdPt4SW7Vg90H21EOLACvtfcHk0lh7wRl0msfiww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaG1MXovFCisA%253D%253D&md5=fe4d2f142fbc36266afe85ac6a05a1de</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fjo01101a629&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo01101a629%26sid%3Dliteratum%253Aachs%26aulast%3DKlingsberg%26aufirst%3DE.%26atitle%3DPreparation%2520of%2520triaryl-s-triazoles%2520from%2520diaroylhydrazines%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D1958%26volume%3D23%26spage%3D1086%26epage%3D1087%26doi%3D10.1021%2Fjo01101a629" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="note"><p class="first last">This compound was synthesized following a proprietary ChemDiv protocol: Commercially available 2-bromo-6-nitro-4-phenylquinazoline was subjected to Suzuki–Miyaura coupling reaction with benzene-1,4-diboronic acid followed by nitro group reduction and acylation using acetic anhydride.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kauffman, W. J.</span></span> <span> </span><span class="NLM_article-title">Observations on the synthesis and characterization of <i>N</i>,<i>N</i>′,<i>N</i>″-tris-(dimethylaminopropyl)hexahydro-s-triazine and isolable intermediates</span>. <i>J. Heterocycl. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1975</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">409</span>– <span class="NLM_lpage">411</span>, <span class="refDoi"> DOI: 10.1002/jhet.5570120244</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=10.1002%2Fjhet.5570120244" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADyaE2MXkvFOhsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=1975&pages=409-411&author=W.+J.+Kauffman&title=Observations+on+the+synthesis+and+characterization+of+N%2CN%E2%80%B2%2CN%E2%80%B3-tris-%28dimethylaminopropyl%29hexahydro-s-triazine+and+isolable+intermediates&doi=10.1002%2Fjhet.5570120244"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and characterization of N,N',N''-tris(dimethylaminopropyl)hexahydro-s-triazine and isolable intermediates</span></div><div class="casAuthors">Kauffman, W. J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Heterocyclic Chemistry</span>
        (<span class="NLM_cas:date">1975</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">409-11</span>CODEN:
                <span class="NLM_cas:coden">JHTCAD</span>;
        ISSN:<span class="NLM_cas:issn">0022-152X</span>.
    </div><div class="casAbstract">The cyclization of Me2N(CH2)3NH2 with H2CO gave I-III.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohHDTgLHZbVrVg90H21EOLACvtfcHk0lh7wRl0msfiww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE2MXkvFOhsL4%253D&md5=672d8c018c96a16f5a025c3e1f8c35c4</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1002%2Fjhet.5570120244&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjhet.5570120244%26sid%3Dliteratum%253Aachs%26aulast%3DKauffman%26aufirst%3DW.%2BJ.%26atitle%3DObservations%2520on%2520the%2520synthesis%2520and%2520characterization%2520of%2520N%252CN%25E2%2580%25B2%252CN%25E2%2580%25B3-tris-%2528dimethylaminopropyl%2529hexahydro-s-triazine%2520and%2520isolable%2520intermediates%26jtitle%3DJ.%2520Heterocycl.%2520Chem.%26date%3D1975%26volume%3D12%26spage%3D409%26epage%3D411%26doi%3D10.1002%2Fjhet.5570120244" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit30.b"><span>HHT can be prepared by the reaction
of <i>N</i>,<i>N</i>-dimethylaminopropylamine
with aqueous formaldehyde via aldimine intermediate formation in situ
followed by its rapid trimerization (one-pot synthesis, up to 90%
yield).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=HHT+can+be+prepared+by+the+reaction%0Aof+N%2CN-dimethylaminopropylamine%0Awith+aqueous+formaldehyde+via+aldimine+intermediate+formation+in+situ%0Afollowed+by+its+rapid+trimerization+%28one-pot+synthesis%2C+up+to+90%25%0Ayield%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30.b&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Balaban, A. T.</span></span> <span> </span><span class="NLM_article-title">Chemical graphs</span>. <i>Theoret. Chim. Acta.</i> <span class="NLM_year" style="font-weight: bold;">1979</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">355</span>– <span class="NLM_lpage">375</span>, <span class="refDoi"> DOI: 10.1007/BF00555695</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=10.1007%2FBF00555695" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADyaL3cXhvFOksLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1979&pages=355-375&author=A.+T.+Balaban&title=Chemical+graphs&doi=10.1007%2FBF00555695"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical graphs.  XXXIV.  Five new topological indexes for the branching of tree-like graphs</span></div><div class="casAuthors">Balaban, Alexandru T.</div><div class="citationInfo"><span class="NLM_cas:title">Theoretica Chimica Acta</span>
        (<span class="NLM_cas:date">1979</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">355-75</span>CODEN:
                <span class="NLM_cas:coden">TCHAAM</span>;
        ISSN:<span class="NLM_cas:issn">0040-5744</span>.
    </div><div class="casAbstract">In order to find the center of an acyclic connected graph (of a tree), vertexes of degree one (endpoints) are removed stepwise.  The nos. δi of vertexes thus removed at each step form a digit sequence S (pruning sequence) which reflects the branching of the tree.  The sum of squares of digits in the sequence S affords a new topol. centric index B = Σiδ2i for the branching of trees.  Comparisons with other topol. indexes are presented evidencing that B. induces an ordering of isomeric trees distinct from those induced by all other indexes devised so far, because B emphasizes equally branches of similar length.  It is shown that Rouvray's index I is equiv. to Wiener's index w, and that the Gordon-Scantlebury index N2 and Gutman et al's index M1 belong to the same family, called quadratic indexes, and induce the same ordering.  Since all topol. indexes vary both with the branching and the no. of vertexes in the free, four new indexes are devised from B. and M1 to account only (or mainly) for the branching, by normalization (imposing a lower bound equal to zero for chain-graphs, i.e. n-alkanes) or binormalization (same lower bound, and upper bound equal to one for star-graphs).  Normalized and binormalized centric (C,C') and quadratic indexes (Q,Q') are presented for the lower alkanes.  From the 5 new topol. indexes, the centric indexes (B, C, C') are limited to trees, but the quadratic indexes (Q,Q') apply to any graph.  Binormalized indexes (C',Q') express the topol. shape of the graph.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJcbJr_BNy1rVg90H21EOLACvtfcHk0lh7wRl0msfiww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3cXhvFOksLw%253D&md5=c3bbddeaa4ac25b71b2c72de89334743</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1007%2FBF00555695&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF00555695%26sid%3Dliteratum%253Aachs%26aulast%3DBalaban%26aufirst%3DA.%2BT.%26atitle%3DChemical%2520graphs%26jtitle%3DTheoret.%2520Chim.%2520Acta.%26date%3D1979%26volume%3D53%26spage%3D355%26epage%3D375%26doi%3D10.1007%2FBF00555695" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span>Clarivate
Analytics Integrity Access Page. <a href="https://integrity.thomson-pharma.com" class="extLink">https://integrity.thomson-pharma.com</a> (accessed Feb 27, <span class="NLM_year">2019</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Clarivate%0AAnalytics+Integrity+Access+Page.+https%3A%2F%2Fintegrity.thomson-pharma.com+%28accessed+Feb+27%2C+2019%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Christel, M.</span></span> <span> </span><span class="NLM_article-title">Pharm Exec’s Top 50 Companies 2017</span>. <i>Pharm. Exec.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">4</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2017&pages=4&author=M.+Christel&title=Pharm+Exec%E2%80%99s+Top+50+Companies+2017"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DChristel%26aufirst%3DM.%26atitle%3DPharm%2520Exec%25E2%2580%2599s%2520Top%252050%2520Companies%25202017%26jtitle%3DPharm.%2520Exec.%26date%3D2017%26volume%3D37%26spage%3D4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit33b"><span><a href="http://www.pharmexec.com/pharm-execs-top-50-companies-2017" class="extLink">http://www.pharmexec.com/pharm-execs-top-50-companies-2017</a> (accessed
Feb 27, 2019).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=http%3A%2F%2Fwww.pharmexec.com%2Fpharm-execs-top-50-companies-2017+%28accessed%0AFeb+27%2C+2019%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33b&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ivanenkov, Y. A.</span>; <span class="NLM_string-name">Khandarova, L. M.</span></span> <span> </span><span class="NLM_article-title">Advanced Artificial Intelligence Methods Used in the Design of Pharmaceutical Agents</span>. In  <i>Pharmaceutical Data Mining: Approaches and Applications for Drug Discovery</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balakin, K. V.</span></span>, Ed.; <span class="NLM_publisher-name">Wiley</span>: <span class="NLM_publisher-loc">Hoboken, NJ</span>, <span class="NLM_year">2010</span>; pp  <span class="NLM_fpage">457</span>– <span class="NLM_lpage">489</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2010&pages=457-489&author=Y.+A.+Ivanenkov&author=L.+M.+Khandarovaauthor=K.+V.+Balakin&title=Pharmaceutical+Data+Mining%3A+Approaches+and+Applications+for+Drug+Discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DIvanenkov%26aufirst%3DY.%2BA.%26atitle%3DAdvanced%2520Artificial%2520Intelligence%2520Methods%2520Used%2520in%2520the%2520Design%2520of%2520Pharmaceutical%2520Agents%26btitle%3DPharmaceutical%2520Data%2520Mining%253A%2520Approaches%2520and%2520Applications%2520for%2520Drug%2520Discovery%26aulast%3DBalakin%26aufirst%3DK.%2BV.%26pub%3DWiley%26date%3D2010%26spage%3D457%26epage%3D489" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pletnev, I. V.</span>; <span class="NLM_string-name">Ivanenkov, Y. A.</span>; <span class="NLM_string-name">Tarasov, A. V.</span></span> <span> </span><span class="NLM_article-title">Dimensionality Reduction Techniques for Pharmaceutical Data Mining</span>. In  <i>Pharmaceutical Data Mining: Approaches and Applications for Drug Discovery</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balakin, K. V.</span></span>, Ed.; <span class="NLM_publisher-name">Wiley</span>: <span class="NLM_publisher-loc">Hoboken, NJ</span>, <span class="NLM_year">2010</span>; pp  <span class="NLM_fpage">423</span>– <span class="NLM_lpage">455</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2010&pages=423-455&author=I.+V.+Pletnev&author=Y.+A.+Ivanenkov&author=A.+V.+Tarasovauthor=K.+V.+Balakin&title=Pharmaceutical+Data+Mining%3A+Approaches+and+Applications+for+Drug+Discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DPletnev%26aufirst%3DI.%2BV.%26atitle%3DDimensionality%2520Reduction%2520Techniques%2520for%2520Pharmaceutical%2520Data%2520Mining%26btitle%3DPharmaceutical%2520Data%2520Mining%253A%2520Approaches%2520and%2520Applications%2520for%2520Drug%2520Discovery%26aulast%3DBalakin%26aufirst%3DK.%2BV.%26pub%3DWiley%26date%3D2010%26spage%3D423%26epage%3D455" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sammon, J. W.</span></span> <span> </span><span class="NLM_article-title">A nonlinear mapping for data structure analysis</span>. <i>IEEE Trans. Comput.</i> <span class="NLM_year" style="font-weight: bold;">1969</span>,  <span class="NLM_volume"><i>C-18</i></span>,  <span class="NLM_fpage">401</span>– <span class="NLM_lpage">409</span>, <span class="refDoi"> DOI: 10.1109/T-C.1969.222678</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=10.1109%2FT-C.1969.222678" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=C-18&publication_year=1969&pages=401-409&author=J.+W.+Sammon&title=A+nonlinear+mapping+for+data+structure+analysis&doi=10.1109%2FT-C.1969.222678"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1109%2FT-C.1969.222678&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1109%252FT-C.1969.222678%26sid%3Dliteratum%253Aachs%26aulast%3DSammon%26aufirst%3DJ.%2BW.%26atitle%3DA%2520nonlinear%2520mapping%2520for%2520data%2520structure%2520analysis%26jtitle%3DIEEE%2520Trans.%2520Comput.%26date%3D1969%26volume%3DC-18%26spage%3D401%26epage%3D409%26doi%3D10.1109%2FT-C.1969.222678" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bemis, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murcko, M. A.</span></span> <span> </span><span class="NLM_article-title">The properties of known drugs. 1. Molecular frameworks</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">2887</span>– <span class="NLM_lpage">2893</span>, <span class="refDoi"> DOI: 10.1021/jm9602928</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm9602928" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADyaK28XjvVejtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1996&pages=2887-2893&author=G.+W.+Bemisauthor=M.+A.+Murcko&title=The+properties+of+known+drugs.+1.+Molecular+frameworks&doi=10.1021%2Fjm9602928"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">The Properties of Known Drugs. 1. Molecular Frameworks</span></div><div class="casAuthors">Bemis, Guy W.; Murcko, Mark A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">2887-2893</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">To better understand the common features present in drug mols., we use shape description methods to analyze a database of com. available drugs and prep. a list of common drug shapes.  A useful way of organizing this structural data is to group the atoms of each drug mol. into ring, linker, framework, and side chain atoms.  On the basis of the two-dimensional mol. structures (without regard to atom type, hybridization, and bond order), there are 1179 different frameworks among the 5120 compds. analyzed.  However, the shapes of half of the drugs in the database are described by the 32 most frequently occurring frameworks.  This suggests that the diversity of shapes in the set of known drugs is extremely low.  In our second method of anal., in which atom type, hybridization, and bond order are considered, more diversity is seen; there are 2506 different frameworks among the 5120 compds. in the database, and the most frequently occurring 42 frameworks account for only one-fourth of the drugs.  We discuss the possible interpretations of these findings and the way they may be used to guide future drug discovery research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoodEb3sfDSdrVg90H21EOLACvtfcHk0lhtGBcqhKEEkg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XjvVejtro%253D&md5=5e2c4fdfea9434456a0cca83de4185b3</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fjm9602928&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm9602928%26sid%3Dliteratum%253Aachs%26aulast%3DBemis%26aufirst%3DG.%2BW.%26aulast%3DMurcko%26aufirst%3DM.%2BA.%26atitle%3DThe%2520properties%2520of%2520known%2520drugs.%25201.%2520Molecular%2520frameworks%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1996%26volume%3D39%26spage%3D2887%26epage%3D2893%26doi%3D10.1021%2Fjm9602928" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, T.</span></span> <span> </span><span class="NLM_article-title">Drug and drug candidate building block analysis</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">55</span>– <span class="NLM_lpage">67</span>, <span class="refDoi"> DOI: 10.1021/ci900398f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci900398f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFOisrbI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=55-67&author=J.+Wangauthor=T.+Hou&title=Drug+and+drug+candidate+building+block+analysis&doi=10.1021%2Fci900398f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Drug and Drug Candidate Building Block Analysis</span></div><div class="casAuthors">Wang, Junmei; Hou, Tingjun</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">55-67</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Drug likeness anal. is widely used in modern drug design.  However, most drug likeness filters, represented by Lipinski's "Rule of 5", are based on drugs' simple structural features and some physiochem. properties.  In this study, we conducted thorough structural analyses for two drug datasets.  The first dataset, ADDS, is composed of 1240 FDA-approved drugs, and the second drug dataset, EDDS, is a nonredundant collection of FDA-approved drugs and exptl. drugs in different phases of clin. trials from several drug databases (6932 entries).  For each mol., all possible fragments were enumerated using a brutal force approach.  Three kinds of building blocks, namely, the drug scaffold, ring system, and the small fragment, were identified and ranked according to the frequencies of their occurrence in drug mols.  The major finding is that most top fragments are essentially common for both drug datasets; the top 50 fragments cover 52.6% and 48.6% drugs for ADDS and EDDS, resp.  The identified building blocks were further ranked according to their relative hit rates in the drug datasets and in a screening dataset, which is a nonredundant collection of screening compds. from many resources.  In comparison with the previous reports in the field, we have identified many more high-quality building blocks.  The results obtained in this study could provide useful hints to medicinal chemists in designing drug-like compds. as well as prioritizing screening libraries to filter out those mols. lack of functional building blocks.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUmeynz19y1LVg90H21EOLACvtfcHk0lhtGBcqhKEEkg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFOisrbI&md5=9981fa62c5476e8063370bb7a659a752</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fci900398f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci900398f%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DHou%26aufirst%3DT.%26atitle%3DDrug%2520and%2520drug%2520candidate%2520building%2520block%2520analysis%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D55%26epage%3D67%26doi%3D10.1021%2Fci900398f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacCoss, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawson, A. D. G.</span></span> <span> </span><span class="NLM_article-title">Combining molecular scaffolds from FDA approved drugs: application to drug discovery</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">1638</span>– <span class="NLM_lpage">1647</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01367</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01367" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVGrurrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=1638-1647&author=R.+D.+Taylorauthor=M.+MacCossauthor=A.+D.+G.+Lawson&title=Combining+molecular+scaffolds+from+FDA+approved+drugs%3A+application+to+drug+discovery&doi=10.1021%2Facs.jmedchem.6b01367"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Combining Molecular Scaffolds from FDA Approved Drugs: Application to Drug Discovery</span></div><div class="casAuthors">Taylor, Richard D.; MacCoss, Malcolm; Lawson, Alastair D. G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1638-1647</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The authors have enumerated all linear combinations of ring systems from FDA approved drugs, up to three rings in length and up to four bonds linkers to give an in-silico database of approx. 14 million mols.  This virtual library was compared with mol. databases of published and com. available compds. to assess the prevalence of drug ring combinations in modern medicinal chem. and to identify areas of under-represented, but clin. validated, chem. space.  From the 10 trillion mol. comparisons, the authors found that less than 1% of the possible combinations of drug ring systems appear in com. available libraries.  This key observation highlights significant opportunities to design new fragment-like and lead-like libraries aimed at improving success rates and reducing risk in small mol. drug discovery, as based on the previous anal. (Taylor et al. J. Med. Chem. 2014; 57: 5845-9) approx. 70% of all new drugs are made up of only ring systems that have been used in existing drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpy9yeKHBkbbrVg90H21EOLACvtfcHk0lhtGBcqhKEEkg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVGrurrJ&md5=b4fea6326e337e85d39aa827a180b634</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01367&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01367%26sid%3Dliteratum%253Aachs%26aulast%3DTaylor%26aufirst%3DR.%2BD.%26aulast%3DMacCoss%26aufirst%3DM.%26aulast%3DLawson%26aufirst%3DA.%2BD.%2BG.%26atitle%3DCombining%2520molecular%2520scaffolds%2520from%2520FDA%2520approved%2520drugs%253A%2520application%2520to%2520drug%2520discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D1638%26epage%3D1647%26doi%3D10.1021%2Facs.jmedchem.6b01367" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pitt, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parry, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perry, B. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groom, C. R.</span></span> <span> </span><span class="NLM_article-title">Heteroaromatic rings of the future</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">2952</span>– <span class="NLM_lpage">2963</span>, <span class="refDoi"> DOI: 10.1021/jm801513z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm801513z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADC%252BD1MXktFemur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=2952-2963&author=W.+R.+Pittauthor=D.+M.+Parryauthor=B.+G.+Perryauthor=C.+R.+Groom&title=Heteroaromatic+rings+of+the+future&doi=10.1021%2Fjm801513z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Heteroaromatic Rings of the Future</span></div><div class="casAuthors">Pitt, William R.; Parry, David M.; Perry, Benjamin G.; Groom, Colin R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2952-2963</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Small arom. ring systems are of central importance in the development of novel synthetic protein ligands.  Here we generate a complete list of 24847 such ring systems.  We call this list and assocd. annotations VEHICLe, which stands for virtual exploratory heterocyclic library.  Searches of literature and compd. databases, using this list as substructure queries, identified only 1701 as synthesized.  Using a carefully validated machine learning approach, we were able to est. that the no. of unpublished, but synthetically tractable, VEHICLe rings could be over 3000.  However, anal. also shows that the rate of publication of novel examples to be as low as 5-10 per yr.  With this work, we aim to provide fresh stimulus to creative org. chemists by highlighting a small set of apparently simple ring systems that are predicted to be tractable but are, to the best of our knowledge, unconquered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoV3aLlF9Lb5bVg90H21EOLACvtfcHk0lhWpZs8W0LplA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXktFemur8%253D&md5=12a9d335658be309339d57db02b037e6</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fjm801513z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm801513z%26sid%3Dliteratum%253Aachs%26aulast%3DPitt%26aufirst%3DW.%2BR.%26aulast%3DParry%26aufirst%3DD.%2BM.%26aulast%3DPerry%26aufirst%3DB.%2BG.%26aulast%3DGroom%26aufirst%3DC.%2BR.%26atitle%3DHeteroaromatic%2520rings%2520of%2520the%2520future%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D2952%26epage%3D2963%26doi%3D10.1021%2Fjm801513z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Visini, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arús-Pous, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Awale, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reymond, J. L.</span></span> <span> </span><span class="NLM_article-title">Virtual exploration of the ring systems chemical universe</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">2707</span>– <span class="NLM_lpage">2718</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.7b00457</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.7b00457" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1ehtbnL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2017&pages=2707-2718&author=R.+Visiniauthor=J.+Ar%C3%BAs-Pousauthor=M.+Awaleauthor=J.+L.+Reymond&title=Virtual+exploration+of+the+ring+systems+chemical+universe&doi=10.1021%2Facs.jcim.7b00457"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Virtual Exploration of the Ring Systems Chemical Universe</span></div><div class="casAuthors">Visini, Ricardo; Arus-Pous, Josep; Awale, Mahendra; Reymond, Jean-Louis</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2707-2718</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Here, we explore the chem. space of all virtually possible org. mols., focusing on ring systems, which represent the cyclic cores of org. mols. obtained by removing all acyclic bonds and converting all remaining atoms to carbon.  This approach circumvents the combinatorial explosion encountered when enumerating the mols. themselves.  We report the chem. universe database GDB4c contg. 916,130 ring systems up to four satd. or arom. rings and max. ring size of 14 atoms, and GDB4c3D contg. the corresponding 6,555,929 stereoisomers.  Almost all (98.6 %) of these ring systems are unknown and represent chiral 3D-shaped macrocycles contg. small rings and quaternary centers reminiscent of polycyclic natural products.  We envision that GDB4c can serve to select new ring systems from which to design analogs of such natural products.  The database is available for download at www.gdb.unibe.ch together with interactive visualization and search tools as a resource for mol. design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEMq0Jsdw0drVg90H21EOLACvtfcHk0lhWpZs8W0LplA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1ehtbnL&md5=8a6584de6dea49661707265a519da95c</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.7b00457&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.7b00457%26sid%3Dliteratum%253Aachs%26aulast%3DVisini%26aufirst%3DR.%26aulast%3DAr%25C3%25BAs-Pous%26aufirst%3DJ.%26aulast%3DAwale%26aufirst%3DM.%26aulast%3DReymond%26aufirst%3DJ.%2BL.%26atitle%3DVirtual%2520exploration%2520of%2520the%2520ring%2520systems%2520chemical%2520universe%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2017%26volume%3D57%26spage%3D2707%26epage%3D2718%26doi%3D10.1021%2Facs.jcim.7b00457" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brown, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boström, J.</span></span> <span> </span><span class="NLM_article-title">Analysis of past and present synthetic methodologies on medicinal chemistry: where have all the new reactions gone?</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">4443</span>– <span class="NLM_lpage">4458</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01409</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01409" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVeqsrzM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=4443-4458&author=D.+G.+Brownauthor=J.+Bostr%C3%B6m&title=Analysis+of+past+and+present+synthetic+methodologies+on+medicinal+chemistry%3A+where+have+all+the+new+reactions+gone%3F&doi=10.1021%2Facs.jmedchem.5b01409"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Analysis of Past and Present Synthetic Methodologies on Medicinal Chemistry: Where Have All the New Reactions Gone?</span></div><div class="casAuthors">Brown, Dean G.; Bostrom, Jonas</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4443-4458</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  An anal. of chem. reactions used in current medicinal chem. (2014), three decades ago (1984), and in natural product total synthesis has been conducted.  The anal. revealed that of the current most frequently used synthetic reactions, none were discovered within the past 20 years and only two in the 1980s and 1990s (Suzuki-Miyaura and Buchwald-Hartwig).  This suggests an inherent high bar of impact for new synthetic reactions in drug discovery.  The most frequently used reactions were amide bond formation, Suzuki-Miyaura coupling, and SNAr reactions, most likely due to com. availability of reagents, high chemoselectivity, and a pressure on delivery.  The authors show that these practices result in overpopulation of certain types of mol. shapes to the exclusion of others using simple PMI plots.  The authors hope that these results will help catalyze improvements in integration of new synthetic methodologies as well as new library design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLWvt5DxktAbVg90H21EOLACvtfcHk0lhWpZs8W0LplA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVeqsrzM&md5=fd56ba8418f6d4e8c271f4e977ee2a93</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01409&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01409%26sid%3Dliteratum%253Aachs%26aulast%3DBrown%26aufirst%3DD.%2BG.%26aulast%3DBostr%25C3%25B6m%26aufirst%3DJ.%26atitle%3DAnalysis%2520of%2520past%2520and%2520present%2520synthetic%2520methodologies%2520on%2520medicinal%2520chemistry%253A%2520where%2520have%2520all%2520the%2520new%2520reactions%2520gone%253F%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D4443%26epage%3D4458%26doi%3D10.1021%2Facs.jmedchem.5b01409" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clemons, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bodycombe, N. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carrinski, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shamji, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koehler, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, S. L.</span></span> <span> </span><span class="NLM_article-title">Small molecules of different origins have distinct distributions of structural complexity that correlate with protein-binding profiles</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>107</i></span>,  <span class="NLM_fpage">18787</span>– <span class="NLM_lpage">18792</span>, <span class="refDoi"> DOI: 10.1073/pnas.1012741107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=10.1073%2Fpnas.1012741107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=20956335" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVCqsb%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2010&pages=18787-18792&author=P.+A.+Clemonsauthor=N.+E.+Bodycombeauthor=H.+A.+Carrinskiauthor=J.+A.+Wilsonauthor=A.+F.+Shamjiauthor=B.+K.+Wagnerauthor=A.+N.+Koehlerauthor=S.+L.+Schreiber&title=Small+molecules+of+different+origins+have+distinct+distributions+of+structural+complexity+that+correlate+with+protein-binding+profiles&doi=10.1073%2Fpnas.1012741107"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecules of different origins have distinct distributions of structural complexity that correlate with protein-binding profiles</span></div><div class="casAuthors">Clemons, Paul A.; Bodycombe, Nicole E.; Carrinski, Hyman A.; Wilson, J. Anthony; Shamji, Alykhan F.; Wagner, Bridget K.; Koehler, Angela N.; Schreiber, Stuart L.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">107</span>
        (<span class="NLM_cas:issue">44</span>),
    <span class="NLM_cas:pages">18787-18792, S18787/1-S18787/5</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Using a diverse collection of small mols. generated from a variety of sources, we measured protein-binding activities of each individual compd. against each of 100 diverse (sequence-unrelated) proteins using small-mol. microarrays.  We also analyzed structural features, including complexity, of the small mols.  We found that compds. from different sources (com., academic, natural) have different protein-binding behaviors and that these behaviors correlate with general trends in stereochem. and shape descriptors for these compd. collections.  Increasing the content of sp3-hybridized and stereogenic atoms relative to compds. from com. sources, which comprise the majority of current screening collections, improved binding selectivity and frequency.  The results suggest structural features that synthetic chemists can target when synthesizing screening collections for biol. discovery.  Because binding proteins selectively can be a key feature of high-value probes and drugs, synthesizing com- pounds having features identified in this study may result in improved performance of screening collections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBrE7Js72yZrVg90H21EOLACvtfcHk0ljTuMH8WDhB1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVCqsb%252FN&md5=198e298ca5391d7dfd91184680e3c2ec</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1012741107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1012741107%26sid%3Dliteratum%253Aachs%26aulast%3DClemons%26aufirst%3DP.%2BA.%26aulast%3DBodycombe%26aufirst%3DN.%2BE.%26aulast%3DCarrinski%26aufirst%3DH.%2BA.%26aulast%3DWilson%26aufirst%3DJ.%2BA.%26aulast%3DShamji%26aufirst%3DA.%2BF.%26aulast%3DWagner%26aufirst%3DB.%2BK.%26aulast%3DKoehler%26aufirst%3DA.%2BN.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26atitle%3DSmall%2520molecules%2520of%2520different%2520origins%2520have%2520distinct%2520distributions%2520of%2520structural%2520complexity%2520that%2520correlate%2520with%2520protein-binding%2520profiles%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2010%26volume%3D107%26spage%3D18787%26epage%3D18792%26doi%3D10.1073%2Fpnas.1012741107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kombo, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallapragada, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chewning, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mazurov, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Speake, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hauser, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toler, S.</span></span> <span> </span><span class="NLM_article-title">3D Molecular descriptors important for clinical success</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">327</span>– <span class="NLM_lpage">342</span>, <span class="refDoi"> DOI: 10.1021/ci300445e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci300445e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVCjurnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2013&pages=327-342&author=D.+C.+Komboauthor=K.+Tallapragadaauthor=R.+Jainauthor=J.+Chewningauthor=A.+A.+Mazurovauthor=J.+D.+Speakeauthor=T.+A.+Hauserauthor=S.+Toler&title=3D+Molecular+descriptors+important+for+clinical+success&doi=10.1021%2Fci300445e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">3D Molecular Descriptors Important for Clinical Success</span></div><div class="casAuthors">Kombo, David C.; Tallapragada, Kartik; Jain, Rachit; Chewning, Joseph; Mazurov, Anatoly A.; Speake, Jason D.; Hauser, Terry A.; Toler, Steve</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">327-342</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The pharmacokinetic and safety profiles of clin. drug candidates are greatly influenced by their requisite physicochem. properties.  In particular, it has been shown that 2D mol. descriptors such as fraction of Sp3 carbon atoms (Fsp3) and no. of stereo centers correlate with clin. success.  Using the proteomic off-target hit rate of nicotinic ligands, we found that shape-based 3D descriptors such as the radius of gyration and shadow indexes discriminate off-target promiscuity better than do Fsp3 and the no. of stereo centers.  We have deduced the relevant descriptor values required for a ligand to be nonpromiscuous.  Investigating the MDL Drug Data Report (MDDR) database as compds. move from the preclin. stage toward the market, we have found that these shape-based 3D descriptors predict clin. success of compds. at preclin. and phase1 stages vs compds. withdrawn from the market better than do Fsp3 and LogD.  Further, these computed 3D mol. descriptors correlate well with exptl. obsd. soly., which is among well-known physicochem. properties that drive clin. success.  We also found that about 84% of launched drugs satisfy either Shadow index or Fsp3 criteria, whereas withdrawn and discontinued compds. fail to meet the same criteria.  Our studies suggest that spherical compds. (rather than their elongated counterparts) with a minimal no. of arom. rings may exhibit a high propensity to advance from clin. trials to market.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNRhBgMppmbbVg90H21EOLACvtfcHk0ljTuMH8WDhB1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVCjurnE&md5=c750cdc150c2ba3626d98e2eea8c3144</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Fci300445e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci300445e%26sid%3Dliteratum%253Aachs%26aulast%3DKombo%26aufirst%3DD.%2BC.%26aulast%3DTallapragada%26aufirst%3DK.%26aulast%3DJain%26aufirst%3DR.%26aulast%3DChewning%26aufirst%3DJ.%26aulast%3DMazurov%26aufirst%3DA.%2BA.%26aulast%3DSpeake%26aufirst%3DJ.%2BD.%26aulast%3DHauser%26aufirst%3DT.%2BA.%26aulast%3DToler%26aufirst%3DS.%26atitle%3D3D%2520Molecular%2520descriptors%2520important%2520for%2520clinical%2520success%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2013%26volume%3D53%26spage%3D327%26epage%3D342%26doi%3D10.1021%2Fci300445e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lowe, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sayle, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tarselli, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landrum, G. A.</span></span> <span> </span><span class="NLM_article-title">Big Data from pharmaceutical patents: a computational analysis of medicinal chemists’ bread and butter</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">4385</span>– <span class="NLM_lpage">4402</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00153</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00153" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADC%252BC28XltFGgurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=4385-4402&author=N.+Schneiderauthor=D.+M.+Loweauthor=R.+A.+Sayleauthor=M.+A.+Tarselliauthor=G.+A.+Landrum&title=Big+Data+from+pharmaceutical+patents%3A+a+computational+analysis+of+medicinal+chemists%E2%80%99+bread+and+butter&doi=10.1021%2Facs.jmedchem.6b00153"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Big Data from Pharmaceutical Patents: A Computational Analysis of Medicinal Chemists' Bread and Butter</span></div><div class="casAuthors">Schneider, Nadine; Lowe, Daniel M.; Sayle, Roger A.; Tarselli, Michael A.; Landrum, Gregory A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4385-4402</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Multiple recent studies have focused on unraveling the content of the medicinal chemist's toolbox.  Here, we present an investigation of chem. reactions and mols. retrieved from U.S. patents over the past 40 years (1976-2015).  We used a sophisticated text-mining pipeline to ext. 1.15 million unique whole reaction schemes, including reaction roles and yields, from pharmaceutical patents.  The reactions were assigned to well-known reaction types such as Wittig olefination or Buchwald-Hartwig amination using an expert system.  Analyzing the evolution of reaction types over time, we observe the previously reported bias toward reaction classes like amide bond formations or Suzuki couplings.  Our study also shows a steady increase in the no. of different reaction types used in pharmaceutical patents but a trend toward lower median yield for some of the reaction classes.  Finally, we found that today's typical product mol. is larger, more hydrophobic, and more rigid than 40 years ago.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSnvOtnWAKX7Vg90H21EOLACvtfcHk0ljTuMH8WDhB1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XltFGgurs%253D&md5=16d3f6bd45b4ca1bf024fee2ef9f5744</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00153&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00153%26sid%3Dliteratum%253Aachs%26aulast%3DSchneider%26aufirst%3DN.%26aulast%3DLowe%26aufirst%3DD.%2BM.%26aulast%3DSayle%26aufirst%3DR.%2BA.%26aulast%3DTarselli%26aufirst%3DM.%2BA.%26aulast%3DLandrum%26aufirst%3DG.%2BA.%26atitle%3DBig%2520Data%2520from%2520pharmaceutical%2520patents%253A%2520a%2520computational%2520analysis%2520of%2520medicinal%2520chemists%25E2%2580%2599%2520bread%2520and%2520butter%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D4385%26epage%3D4402%26doi%3D10.1021%2Facs.jmedchem.6b00153" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Walters, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murcko, M. A.</span></span> <span> </span><span class="NLM_article-title">What do medicinal chemists actually make? A 50-Year Retrospective</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">6405</span>– <span class="NLM_lpage">6416</span>, <span class="refDoi"> DOI: 10.1021/jm200504p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm200504p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFWjsr7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=6405-6416&author=W.+P.+Waltersauthor=J.+Greenauthor=J.+R.+Weissauthor=M.+A.+Murcko&title=What+do+medicinal+chemists+actually+make%3F+A+50-Year+Retrospective&doi=10.1021%2Fjm200504p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">What Do Medicinal Chemists Actually Make? A 50-Year Retrospective</span></div><div class="casAuthors">Walters, W. Patrick; Green, Jeremy; Weiss, Jonathan R.; Murcko, Mark A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">6405-6416</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Despite the dramatically increasing costs of pharmaceutical R&D productivity, as measured by the no. of new drugs approved each year, has languished during the past decade.  Many different explanations have been proposed to account for this decline, such as increasing regulatory scrutiny, the inherent inefficiency of very large organizations, the pursuit of highly speculative targets identified from genomics, and more objective decision-making about advancing marginal compds. into latest age clin. trials.  Another factor often cited to help explain low pharmaceutical productivity is that, paradoxically, many compds. synthesized by medicinal chemists are not "druglike", at least as measured by various criteria.  The authors discuss the past, present and perspectives in the development of medicinal chem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqg5xuF4wHqKrVg90H21EOLACvtfcHk0lg-0w2LaYNx-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFWjsr7O&md5=33109f9d55967be0d34f377f433621e4</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Fjm200504p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200504p%26sid%3Dliteratum%253Aachs%26aulast%3DWalters%26aufirst%3DW.%2BP.%26aulast%3DGreen%26aufirst%3DJ.%26aulast%3DWeiss%26aufirst%3DJ.%2BR.%26aulast%3DMurcko%26aufirst%3DM.%2BA.%26atitle%3DWhat%2520do%2520medicinal%2520chemists%2520actually%2520make%253F%2520A%252050-Year%2520Retrospective%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D6405%26epage%3D6416%26doi%3D10.1021%2Fjm200504p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Doak, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Over, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giordanetto, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kihlberg, J.</span></span> <span> </span><span class="NLM_article-title">Oral druggable space beyond the Rule of 5: insights from drugs and clinical candidates</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">1115</span>– <span class="NLM_lpage">1142</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2014.08.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=10.1016%2Fj.chembiol.2014.08.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=25237858" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFyrur7K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2014&pages=1115-1142&author=B.+C.+Doakauthor=B.+Overauthor=F.+Giordanettoauthor=J.+Kihlberg&title=Oral+druggable+space+beyond+the+Rule+of+5%3A+insights+from+drugs+and+clinical+candidates&doi=10.1016%2Fj.chembiol.2014.08.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Oral Druggable Space beyond the Rule of 5: Insights from Drugs and Clinical Candidates</span></div><div class="casAuthors">Doak, Bradley Croy; Over, Bjorn; Giordanetto, Fabrizio; Kihlberg, Jan</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1115-1142</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The rule of 5 (Ro5) is a set of in silico guidelines applied to drug discovery to prioritize compds. with an increased likelihood of high oral absorption.  It has been influential in reducing attrition due to poor pharmacokinetics over the last 15 years.  However, strict reliance on the Ro5 may have resulted in lost opportunities, particularly for difficult targets.  To identify opportunities for oral drug discovery beyond the Ro5 (bRo5), we have comprehensively analyzed drugs and clin. candidates with mol. wt. (MW) > 500 Da.  We conclude that oral drugs are found far bRo5 and properties such as intramol. hydrogen bonding, macrocyclization, dosage, and formulations can be used to improve bRo5 bioavailability.  Natural products and structure-based design, often from peptidic leads, are key sources for oral bRo5 drugs.  These insights should help guide the design of oral drugs in bRo5 space, which is of particular interest for difficult targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvYqpeA0dh6rVg90H21EOLACvtfcHk0lg-0w2LaYNx-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFyrur7K&md5=b8a88c793572ad48de3809807a4699d8</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2014.08.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2014.08.013%26sid%3Dliteratum%253Aachs%26aulast%3DDoak%26aufirst%3DB.%2BC.%26aulast%3DOver%26aufirst%3DB.%26aulast%3DGiordanetto%26aufirst%3DF.%26aulast%3DKihlberg%26aufirst%3DJ.%26atitle%3DOral%2520druggable%2520space%2520beyond%2520the%2520Rule%2520of%25205%253A%2520insights%2520from%2520drugs%2520and%2520clinical%2520candidates%26jtitle%3DChem.%2520Biol.%26date%3D2014%26volume%3D21%26spage%3D1115%26epage%3D1142%26doi%3D10.1016%2Fj.chembiol.2014.08.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shultz, M. D.</span></span> <span> </span><span class="NLM_article-title">Two decades under the influence of the Rule of five and the changing properties of approved oral drugs</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">1701</span>– <span class="NLM_lpage">1714</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00686</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00686" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslait73K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=1701-1714&issue=4&author=M.+D.+Shultz&title=Two+decades+under+the+influence+of+the+Rule+of+five+and+the+changing+properties+of+approved+oral+drugs&doi=10.1021%2Facs.jmedchem.8b00686"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Two Decades under the Influence of the Rule of Five and the Changing Properties of Approved Oral Drugs</span></div><div class="casAuthors">Shultz, Michael D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1701-1714</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Two decades have passed since the rule of five ushered in the concept of "drug-like" properties.  Attempts to quantify, correlate, and categorize mols. based on Ro5 parameters evolved into the introduction of efficiency metrics with far reaching consequences in decision making by industry leaders and scientists seeking to discover new medicines.  Examn. of oral drug parameters approved before and after the original Ro5 anal. demonstrates that some parameters such as clogP and HBD remained const. while the cutoffs for parameters such as mol. wt. and HBA have increased substantially over the past 20 years.  The time dependent increase in the mol. wt. of oral drugs during the past 20 years provides compelling evidence to disprove the hypothesis that mol. wt. is a "drug-like" property.  This anal. does not validate parameters that have not changed as being "drug-like" but instead calls into question the entire hypothesis that "drug-like" properties exist.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAnoxpQsv2kbVg90H21EOLACvtfcHk0lg-0w2LaYNx-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslait73K&md5=d3c8b1107133dc0b7b277d22fa029597</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00686&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00686%26sid%3Dliteratum%253Aachs%26aulast%3DShultz%26aufirst%3DM.%2BD.%26atitle%3DTwo%2520decades%2520under%2520the%2520influence%2520of%2520the%2520Rule%2520of%2520five%2520and%2520the%2520changing%2520properties%2520of%2520approved%2520oral%2520drugs%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26issue%3D4%26spage%3D1701%26epage%3D1714%26doi%3D10.1021%2Facs.jmedchem.8b00686" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ran, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gestwicki, J. E.</span></span> <span> </span><span class="NLM_article-title">Inhibitors of protein-protein interactions (PPIs): an analysis of scaffold choices and buried surface area</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">75</span>– <span class="NLM_lpage">86</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2018.06.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=10.1016%2Fj.cbpa.2018.06.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=29908451" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFCmsbnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2018&pages=75-86&author=X.+Ranauthor=J.+E.+Gestwicki&title=Inhibitors+of+protein-protein+interactions+%28PPIs%29%3A+an+analysis+of+scaffold+choices+and+buried+surface+area&doi=10.1016%2Fj.cbpa.2018.06.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitors of protein-protein interactions (PPIs): an analysis of scaffold choices and buried surface area</span></div><div class="casAuthors">Ran, Xu; Gestwicki, Jason E.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">75-86</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Protein-protein interactions (PPI) were once considered 'undruggable', but clin. successes, driven by advanced methods in drug discovery, have challenged that notion.  Here, we review the last three years of literature on PPI inhibitors to understand what is working and why.  From the 66 recently reported PPI inhibitors, we found that the av. mol. wt. was significantly greater than 500 Da, but that this trend was driven, in large part, by the contribution of peptide-based compds.  Despite differences in av. mol. wt., we found that compds. based on small mols. or peptides were almost equally likely to be potent inhibitors (KD < 1 μM).  Finally, we found PPIs with buried surface area (BSA) less than 2000 Å2 were more likely to be inhibited by small mols., while PPIs with larger BSA values were typically inhibited by peptides.  PPIs with BSA values over 4000 Å2 seemed to create a particular challenge, esp. for orthosteric small mols.  Thus, it seems important to choose the inhibitor scaffold based on the properties of the target interaction.  Moreover, this survey suggests a (more nuanced) conclusion to the question of whether PPIs are good drug targets; namely, that some PPIs are readily 'druggable' given the right choice of scaffold, while others still seem to deserve the 'undruggable' moniker.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpoS9hriXFuS7Vg90H21EOLACvtfcHk0lg-0w2LaYNx-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFCmsbnE&md5=037a06e18fc78112243cd68ce6752599</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2018.06.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2018.06.004%26sid%3Dliteratum%253Aachs%26aulast%3DRan%26aufirst%3DX.%26aulast%3DGestwicki%26aufirst%3DJ.%2BE.%26atitle%3DInhibitors%2520of%2520protein-protein%2520interactions%2520%2528PPIs%2529%253A%2520an%2520analysis%2520of%2520scaffold%2520choices%2520and%2520buried%2520surface%2520area%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2018%26volume%3D44%26spage%3D75%26epage%3D86%26doi%3D10.1016%2Fj.cbpa.2018.06.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cossar, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCluskey, A.</span></span> <span> </span><span class="NLM_article-title">Protein protein interactions as antibiotic targets: a medicinal chemistry perspective</span>. <i>Med. Res. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">26</span>, <span class="refDoi"> DOI: 10.1002/med.21519</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=10.1002%2Fmed.21519" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&pages=1-26&author=P.+J.+Cossarauthor=P.+J.+Lewisauthor=A.+McCluskey&title=Protein+protein+interactions+as+antibiotic+targets%3A+a+medicinal+chemistry+perspective&doi=10.1002%2Fmed.21519"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1002%2Fmed.21519&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21519%26sid%3Dliteratum%253Aachs%26aulast%3DCossar%26aufirst%3DP.%2BJ.%26aulast%3DLewis%26aufirst%3DP.%2BJ.%26aulast%3DMcCluskey%26aufirst%3DA.%26atitle%3DProtein%2520protein%2520interactions%2520as%2520antibiotic%2520targets%253A%2520a%2520medicinal%2520chemistry%2520perspective%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2018%26spage%3D1%26epage%3D26%26doi%3D10.1002%2Fmed.21519" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Macalino, S. J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Basith, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clavio, N. A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, S.</span></span> <span> </span><span class="NLM_article-title">Evolution of in silico strategies for protein-protein interaction drug discovery</span>. <i>Molecules</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1963</span>, <span class="refDoi"> DOI: 10.3390/molecules23081963</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=10.3390%2Fmolecules23081963" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvFGgtbzJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2018&pages=1963&author=S.+J.+Y.+Macalinoauthor=S.+Basithauthor=N.+A.+B.+Clavioauthor=H.+Changauthor=S.+Kangauthor=S.+Choi&title=Evolution+of+in+silico+strategies+for+protein-protein+interaction+drug+discovery&doi=10.3390%2Fmolecules23081963"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Evolution of in silico strategies for protein-protein interaction drug discovery</span></div><div class="casAuthors">Macalino, Stephani Joy Y.; Basith, Shaherin; Clavio, Nina Abigail B.; Chang, Hyerim; Kang, Soosung; Choi, Sun</div><div class="citationInfo"><span class="NLM_cas:title">Molecules</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1963/1-1963/44</span>CODEN:
                <span class="NLM_cas:coden">MOLEFW</span>;
        ISSN:<span class="NLM_cas:issn">1420-3049</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">The advent of advanced mol. modeling software, big data analytics, and high-speed processing units has led to the exponential evolution of modern drug discovery and better insights into complex biol. processes and disease networks.  This has progressively steered current research interests to understanding protein-protein interaction (PPI) systems that are related to a no. of relevant diseases, such as cancer, neurol. illnesses, metabolic disorders, etc.  However, targeting PPIs are challenging due to their "undruggable" binding interfaces.  In this review, we focus on the current obstacles that impede PPI drug discovery, and how recent discoveries and advances in in silico approaches can alleviate these barriers to expedite the search for potential leads, as shown in several exemplary studies.  We will also discuss about currently available information on PPI compds. and systems, along with their usefulness in mol. modeling.  Finally, we conclude by presenting the limits of in silico application in drug discovery and offer a perspective in the field of computer-aided PPI drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqu3s0Z07xfAbVg90H21EOLACvtfcHk0lgqAdtBzDcBHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvFGgtbzJ&md5=f700f8a40a79c10f1e796cab26295849</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.3390%2Fmolecules23081963&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules23081963%26sid%3Dliteratum%253Aachs%26aulast%3DMacalino%26aufirst%3DS.%2BJ.%2BY.%26aulast%3DBasith%26aufirst%3DS.%26aulast%3DClavio%26aufirst%3DN.%2BA.%2BB.%26aulast%3DChang%26aufirst%3DH.%26aulast%3DKang%26aufirst%3DS.%26aulast%3DChoi%26aufirst%3DS.%26atitle%3DEvolution%2520of%2520in%2520silico%2520strategies%2520for%2520protein-protein%2520interaction%2520drug%2520discovery%26jtitle%3DMolecules%26date%3D2018%26volume%3D23%26spage%3D1963%26doi%3D10.3390%2Fmolecules23081963" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scott, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayly, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abell, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skidmore, J.</span></span> <span> </span><span class="NLM_article-title">Small molecules, big targets: drug discovery faces the protein- protein interaction challenge</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">533</span>– <span class="NLM_lpage">550</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.29</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=10.1038%2Fnrd.2016.29" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=27050677" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADC%252BC28XlsFOgur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=533-550&author=D.+E.+Scottauthor=A.+R.+Baylyauthor=C.+Abellauthor=J.+Skidmore&title=Small+molecules%2C+big+targets%3A+drug+discovery+faces+the+protein-+protein+interaction+challenge&doi=10.1038%2Fnrd.2016.29"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecules, big targets: drug discovery faces the protein-protein interaction challenge</span></div><div class="casAuthors">Scott, Duncan E.; Bayly, Andrew R.; Abell, Chris; Skidmore, John</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">533-550</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein-protein interactions (PPIs) are of pivotal importance in the regulation of biol. systems and are consequently implicated in the development of disease states.  Recent work has begun to show that, with the right tools, certain classes of PPI can yield to the efforts of medicinal chemists to develop inhibitors, and the first PPI inhibitors have reached clin. development.  In this Review, we describe the research leading to these breakthroughs and highlight the existence of groups of structurally related PPIs within the PPI target class.  For each of these groups, we use examples of successful discovery efforts to illustrate the research strategies that have proved most useful.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoK-aGzHa9mP7Vg90H21EOLACvtfcHk0lgqAdtBzDcBHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XlsFOgur4%253D&md5=9f0953567ae9338b8b36aa78b8160655</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.29&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.29%26sid%3Dliteratum%253Aachs%26aulast%3DScott%26aufirst%3DD.%2BE.%26aulast%3DBayly%26aufirst%3DA.%2BR.%26aulast%3DAbell%26aufirst%3DC.%26aulast%3DSkidmore%26aufirst%3DJ.%26atitle%3DSmall%2520molecules%252C%2520big%2520targets%253A%2520drug%2520discovery%2520faces%2520the%2520protein-%2520protein%2520interaction%2520challenge%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2016%26volume%3D15%26spage%3D533%26epage%3D550%26doi%3D10.1038%2Fnrd.2016.29" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Soga, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shirai, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobori, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirayama, N.</span></span> <span> </span><span class="NLM_article-title">Use of amino acid composition to predict ligand-binding sites</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">400</span>– <span class="NLM_lpage">406</span>, <span class="refDoi"> DOI: 10.1021/ci6002202</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci6002202" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADC%252BD2sXns1OmsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2007&pages=400-406&author=S.+Sogaauthor=H.+Shiraiauthor=M.+Koboriauthor=N.+Hirayama&title=Use+of+amino+acid+composition+to+predict+ligand-binding+sites&doi=10.1021%2Fci6002202"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Use of Amino Acid Composition to Predict Ligand-Binding Sites</span></div><div class="casAuthors">Soga, Shinji; Shirai, Hiroki; Kobori, Masato; Hirayama, Noriaki</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">400-406</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel method for predicting the binding sites for druglike compds. on the surface of proteins was developed on the basis of the specific amino acid compn. obsd. at the ligand-binding sites of ligand-protein complexes detd. by x-ray anal.  A profile representing the preference of each of the 20 std. amino acids at the binding sites of druglike mols. was obtained for a small set of high-quality complex structures.  An index termed propensity for ligand binding (PLB) was created from these profiles.  The PLB index was used to predict the propensity of binding for 804 ligands at all potential binding sites on the proteins whose structures were detd. by x-ray anal.  If the sites with the first 2 highest PLB indexes are taken into consideration, the successfully predicted sites reached a high percentage of 86.  The PLB prediction is relatively simple, but the validation study showed that it is both fast and accurate to detect ligand-binding sites, esp. the binding sites of druglike mols.  Therefore, the PLB index can be used to predict the ligand-binding sites of uncharacterized protein structures and also to identify novel drug-binding sites of known drug targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptxKvbII_oZbVg90H21EOLACvtfcHk0lgqAdtBzDcBHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXns1OmsQ%253D%253D&md5=53bddb67710c84f45c99173b0d08d9ce</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Fci6002202&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci6002202%26sid%3Dliteratum%253Aachs%26aulast%3DSoga%26aufirst%3DS.%26aulast%3DShirai%26aufirst%3DH.%26aulast%3DKobori%26aufirst%3DM.%26aulast%3DHirayama%26aufirst%3DN.%26atitle%3DUse%2520of%2520amino%2520acid%2520composition%2520to%2520predict%2520ligand-binding%2520sites%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2007%26volume%3D47%26spage%3D400%26epage%3D406%26doi%3D10.1021%2Fci6002202" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pushpakom, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iorio, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eyers, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Escott, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hopper, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doig, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guilliams, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Latimer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNamee, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norris, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanseau, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavalla, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pirmohamed, M.</span></span> <span> </span><span class="NLM_article-title">Drug repurposing: progress, challenges and recommendations</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">41</span>– <span class="NLM_lpage">58</span>, <span class="refDoi"> DOI: 10.1038/nrd.2018.168</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=10.1038%2Fnrd.2018.168" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=30310233" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvV2gtr7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=41-58&author=S.+Pushpakomauthor=F.+Iorioauthor=P.+A.+Eyersauthor=K.+J.+Escottauthor=S.+Hopperauthor=A.+Wellsauthor=A.+Doigauthor=T.+Guilliamsauthor=J.+Latimerauthor=C.+McNameeauthor=A.+Norrisauthor=P.+Sanseauauthor=D.+Cavallaauthor=M.+Pirmohamed&title=Drug+repurposing%3A+progress%2C+challenges+and+recommendations&doi=10.1038%2Fnrd.2018.168"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Drug repurposing: progress, challenges and recommendations</span></div><div class="casAuthors">Pushpakom, Sudeep; Iorio, Francesco; Eyers, Patrick A.; Escott, K. Jane; Hopper, Shirley; Wells, Andrew; Doig, Andrew; Guilliams, Tim; Latimer, Joanna; McNamee, Christine; Norris, Alan; Sanseau, Philippe; Cavalla, David; Pirmohamed, Munir</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">41-58</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  Given the high attrition rates, substantial costs and slow pace of new drug discovery and development, repurposing of 'old' drugs to treat both common and rare diseases is increasingly becoming an attractive proposition because it involves the use of de-risked compds., with potentially lower overall development costs and shorter development timelines.  Various data-driven and exptl. approaches have been suggested for the identification of repurposable drug candidates; however, there are also major technol. and regulatory challenges that need to be addressed.  In this Review, we present approaches used for drug repurposing (also known as drug repositioning), discuss the challenges faced by the repurposing community and recommend innovative ways by which these challenges could be addressed to help realize the full potential of drug repurposing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUl_N5c2iDerVg90H21EOLACvtfcHk0liJrv9xz9lw1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvV2gtr7I&md5=6bcc8f26828414de02564494fb4d3296</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2018.168&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2018.168%26sid%3Dliteratum%253Aachs%26aulast%3DPushpakom%26aufirst%3DS.%26aulast%3DIorio%26aufirst%3DF.%26aulast%3DEyers%26aufirst%3DP.%2BA.%26aulast%3DEscott%26aufirst%3DK.%2BJ.%26aulast%3DHopper%26aufirst%3DS.%26aulast%3DWells%26aufirst%3DA.%26aulast%3DDoig%26aufirst%3DA.%26aulast%3DGuilliams%26aufirst%3DT.%26aulast%3DLatimer%26aufirst%3DJ.%26aulast%3DMcNamee%26aufirst%3DC.%26aulast%3DNorris%26aufirst%3DA.%26aulast%3DSanseau%26aufirst%3DP.%26aulast%3DCavalla%26aufirst%3DD.%26aulast%3DPirmohamed%26aufirst%3DM.%26atitle%3DDrug%2520repurposing%253A%2520progress%252C%2520challenges%2520and%2520recommendations%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2019%26volume%3D18%26spage%3D41%26epage%3D58%26doi%3D10.1038%2Fnrd.2018.168" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ashburn, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thor, K. B.</span></span> <span> </span><span class="NLM_article-title">Drug repositioning: identifying and developing new uses for existing drugs</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">673</span>– <span class="NLM_lpage">683</span>, <span class="refDoi"> DOI: 10.1038/nrd1468</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=10.1038%2Fnrd1468" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=15286734" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmtVOhtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2004&pages=673-683&author=T.+T.+Ashburnauthor=K.+B.+Thor&title=Drug+repositioning%3A+identifying+and+developing+new+uses+for+existing+drugs&doi=10.1038%2Fnrd1468"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Drug repositioning: identifying and developing new uses for existing drugs</span></div><div class="casAuthors">Ashburn, Ted T.; Thor, Karl B.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">673-683</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Biopharmaceutical companies attempting to increase productivity through novel discovery technologies have fallen short of achieving the desired results.  Repositioning existing drugs for new indications could deliver the productivity increases that the industry needs while shifting the locus of prodn. to biotechnol. companies.  More and more companies are scanning the existing pharmacopeia for repositioning candidates, and the no. of repositioning success stories is increasing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLkWl3DnePhrVg90H21EOLACvtfcHk0liJrv9xz9lw1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmtVOhtL8%253D&md5=0145c02423ab81ace6e3b3d92585cccc</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1038%2Fnrd1468&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd1468%26sid%3Dliteratum%253Aachs%26aulast%3DAshburn%26aufirst%3DT.%2BT.%26aulast%3DThor%26aufirst%3DK.%2BB.%26atitle%3DDrug%2520repositioning%253A%2520identifying%2520and%2520developing%2520new%2520uses%2520for%2520existing%2520drugs%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2004%26volume%3D3%26spage%3D673%26epage%3D683%26doi%3D10.1038%2Fnrd1468" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span>Enamine
Ltd. Home Page. <a href="https://enamine.net" class="extLink">https://enamine.net</a> (accessed Feb 27, <span class="NLM_year">2019</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Enamine%0ALtd.+Home+Page.+https%3A%2F%2Fenamine.net+%28accessed+Feb+27%2C+2019%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span>InterBioscreen
Ltd. Home Page. <a href="https://www.ibscreen.com" class="extLink">https://www.ibscreen.com</a> (accessed Feb 27, <span class="NLM_year">2019</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=InterBioscreen%0ALtd.+Home+Page.+https%3A%2F%2Fwww.ibscreen.com+%28accessed+Feb+27%2C+2019%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kessel, M.</span></span> <span> </span><span class="NLM_article-title">The problems with today’s pharmaceutical business—an outsider’s view</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">27</span>– <span class="NLM_lpage">33</span>, <span class="refDoi"> DOI: 10.1038/nbt.1748</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=10.1038%2Fnbt.1748" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=21221096" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjsFWhuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2011&pages=27-33&author=M.+Kessel&title=The+problems+with+today%E2%80%99s+pharmaceutical+business%E2%80%94an+outsider%E2%80%99s+view&doi=10.1038%2Fnbt.1748"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">The problems with today's pharmaceutical business - an outsider's view</span></div><div class="casAuthors">Kessel, Mark</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">27-33</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The pharmaceutical industry must devote greater resources, investment and effort to address its anemic drug pipeline in the long term, rather than focusing on its bottom line in the near term.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-UuhFn-1nFbVg90H21EOLACvtfcHk0liJrv9xz9lw1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXjsFWhuw%253D%253D&md5=cbb7728a68cfffea1ea189164239fefd</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1038%2Fnbt.1748&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.1748%26sid%3Dliteratum%253Aachs%26aulast%3DKessel%26aufirst%3DM.%26atitle%3DThe%2520problems%2520with%2520today%25E2%2580%2599s%2520pharmaceutical%2520business%25E2%2580%2594an%2520outsider%25E2%2580%2599s%2520view%26jtitle%3DNat.%2520Biotechnol.%26date%3D2011%26volume%3D29%26spage%3D27%26epage%3D33%26doi%3D10.1038%2Fnbt.1748" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span>NPS,
National Pharmaceutical Services Home Page. <a href="https://www.pti-nps.com/nps/" class="extLink">https://www.pti-nps.com/nps/</a> (accessed Feb 27, <span class="NLM_year">2019</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=NPS%2C%0ANational+Pharmaceutical+Services+Home+Page.+https%3A%2F%2Fwww.pti-nps.com%2Fnps%2F+%28accessed+Feb+27%2C+2019%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mullin, R.</span></span> <span> </span><span class="NLM_article-title">More job cuts at Merck</span>. <i>Chem. Eng. News</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>89</i></span>,  <span class="NLM_fpage">10</span>, <span class="refDoi"> DOI: 10.1021/cen-v089n032.p010</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cen-v089n034.p010a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=89&publication_year=2011&pages=10&author=R.+Mullin&title=More+job+cuts+at+Merck&doi=10.1021%2Fcen-v089n032.p010"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1021%2Fcen-v089n034.p010a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcen-v089n034.p010a%26sid%3Dliteratum%253Aachs%26aulast%3DMullin%26aufirst%3DR.%26atitle%3DMore%2520job%2520cuts%2520at%2520Merck%26jtitle%3DChem.%2520Eng.%2520News%26date%3D2011%26volume%3D89%26spage%3D10%26doi%3D10.1021%2Fcen-v089n032.p010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jarvis, L. M.</span></span> <span> </span><span class="NLM_article-title">Pfizer reveals more R&D cuts</span>. <i>Chem. Eng. News</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>89</i></span>,  <span class="NLM_fpage">5</span>, <span class="refDoi"> DOI: 10.1021/cen-v089n006.p005</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cen-v089n006.p005" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=89&publication_year=2011&pages=5&author=L.+M.+Jarvis&title=Pfizer+reveals+more+R%26D+cuts&doi=10.1021%2Fcen-v089n006.p005"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1021%2Fcen-v089n006.p005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcen-v089n006.p005%26sid%3Dliteratum%253Aachs%26aulast%3DJarvis%26aufirst%3DL.%2BM.%26atitle%3DPfizer%2520reveals%2520more%2520R%2526D%2520cuts%26jtitle%3DChem.%2520Eng.%2520News%26date%3D2011%26volume%3D89%26spage%3D5%26doi%3D10.1021%2Fcen-v089n006.p005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tsukamoto, T.</span></span> <span> </span><span class="NLM_article-title">Tough times for medicinal chemists: are we to blame?</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">369</span>– <span class="NLM_lpage">370</span>, <span class="refDoi"> DOI: 10.1021/ml400074k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml400074k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtlWgtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=369-370&author=T.+Tsukamoto&title=Tough+times+for+medicinal+chemists%3A+are+we+to+blame%3F&doi=10.1021%2Fml400074k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Tough Times for Medicinal Chemists: Are We to Blame?</span></div><div class="casAuthors">Tsukamoto, Takashi</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">369-370</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  In the United States, medicinal chemists continue to face job insecurity and high rates of unemployment.  The situation is unlikely to improve in the near future.  Is there a light at the end of the tunnel.  Is there anything we can do to revitalize our community.  The answer may be right in front of us.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-HjhOmBhZK7Vg90H21EOLACvtfcHk0lgG5sTuN_8D_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtlWgtL4%253D&md5=383e3292fc5110fcec0bd1b104f692ca</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1021%2Fml400074k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml400074k%26sid%3Dliteratum%253Aachs%26aulast%3DTsukamoto%26aufirst%3DT.%26atitle%3DTough%2520times%2520for%2520medicinal%2520chemists%253A%2520are%2520we%2520to%2520blame%253F%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2013%26volume%3D4%26spage%3D369%26epage%3D370%26doi%3D10.1021%2Fml400074k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rafferty, M. F.</span></span> <span> </span><span class="NLM_article-title">No denying it: medicinal chemistry training is in big trouble</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">10859</span>– <span class="NLM_lpage">10864</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00741</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00741" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFKmsLjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=10859-10864&author=M.+F.+Rafferty&title=No+denying+it%3A+medicinal+chemistry+training+is+in+big+trouble&doi=10.1021%2Facs.jmedchem.6b00741"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">No Denying It: Medicinal Chemistry Training Is in Big Trouble</span></div><div class="casAuthors">Rafferty, Michael F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">10859-10864</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">There has been little consensus between the pharmaceutical industry and academic communities concerning the best approach to train medicinal chemists for drug discovery.  For decades the pharmaceutical industry has shown preference for synthetic org. graduates over candidates with degrees from medicinal chem. programs on the assumption that medicinal chem. expertise will be acquired on the job.  However, ongoing changes to pharmaceutical drug discovery organizations and practices threaten to undermine this training model.  There is a compelling argument to be made for establishment of a strong industry-academic partnership to train new candidates with sophisticated knowledge of contemporary drug design concepts and techniques to ensure that the future needs of both industry and academic drug discovery research can be served.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgSt0oxSWvabVg90H21EOLACvtfcHk0lgG5sTuN_8D_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFKmsLjO&md5=7c5583586b8d6939bc2a52cb4a033455</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00741&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00741%26sid%3Dliteratum%253Aachs%26aulast%3DRafferty%26aufirst%3DM.%2BF.%26atitle%3DNo%2520denying%2520it%253A%2520medicinal%2520chemistry%2520training%2520is%2520in%2520big%2520trouble%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D10859%26epage%3D10864%26doi%3D10.1021%2Facs.jmedchem.6b00741" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hoffmann, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bishop, C.</span></span> <span> </span><span class="NLM_article-title">The future of discovery chemistry: quo vadis? Academic to industrial - the maturation of medicinal chemistry to chemical biology</span>. <i>Drug Discovery Today.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">260</span>– <span class="NLM_lpage">264</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2010.02.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=10.1016%2Fj.drudis.2010.02.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=20149893" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A280%3ADC%252BC3c3jt1Ggsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2010&pages=260-264&author=T.+Hoffmannauthor=C.+Bishop&title=The+future+of+discovery+chemistry%3A+quo+vadis%3F+Academic+to+industrial+-+the+maturation+of+medicinal+chemistry+to+chemical+biology&doi=10.1016%2Fj.drudis.2010.02.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">The future of discovery chemistry: quo vadis? Academic to industrial--the maturation of medicinal chemistry to chemical biology</span></div><div class="casAuthors">Hoffmann Torsten; Bishop Cheryl</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">7-8</span>),
    <span class="NLM_cas:pages">260-4</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">At Roche, we set out to think about the future role of medicinal chemistry in drug discovery in a project involving both Roche internal stakeholders and external experts in drug discovery chemistry.  To derive a coherent strategy, selected scientists were asked to take extreme positions and to derive two orthogonal strategic options: chemistry as the traditional mainstream science and chemistry as the central entrepreneurial science.  We believe today's role of medicinal chemistry in industry has remained too narrow.  To provide the innovation that industry requires, medicinal chemistry must play its part and diversify at pace with our increasing understanding of chemical biology and network pharmacology.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSk2WZdrxq9mEH3Rlp_k91zfW6udTcc2eZKLodRipP4qrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3c3jt1Ggsg%253D%253D&md5=528caf07919ff2f5c6544e166fa5cec0</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2010.02.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2010.02.002%26sid%3Dliteratum%253Aachs%26aulast%3DHoffmann%26aufirst%3DT.%26aulast%3DBishop%26aufirst%3DC.%26atitle%3DThe%2520future%2520of%2520discovery%2520chemistry%253A%2520quo%2520vadis%253F%2520Academic%2520to%2520industrial%2520-%2520the%2520maturation%2520of%2520medicinal%2520chemistry%2520to%2520chemical%2520biology%26jtitle%3DDrug%2520Discovery%2520Today.%26date%3D2010%26volume%3D15%26spage%3D260%26epage%3D264%26doi%3D10.1016%2Fj.drudis.2010.02.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, I. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macdonald, S. J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Procopiou, P. A.</span></span> <span> </span><span class="NLM_article-title">Medicinal chemistry in drug discovery in big pharma: past, present and future</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">219</span>– <span class="NLM_lpage">234</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2017.10.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=10.1016%2Fj.drudis.2017.10.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=29031621" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A280%3ADC%252BC1M7gs1GltQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2018&pages=219-234&author=I.+B.+Campbellauthor=S.+J.+F.+Macdonaldauthor=P.+A.+Procopiou&title=Medicinal+chemistry+in+drug+discovery+in+big+pharma%3A+past%2C+present+and+future&doi=10.1016%2Fj.drudis.2017.10.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Medicinal chemistry in drug discovery in big pharma: past, present and future</span></div><div class="casAuthors">Campbell Ian B; Procopiou Panayiotis A; Macdonald Simon J F</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">219-234</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The changes in synthetic and medicinal chemistry and related drug discovery science as practiced in big pharma over the past few decades are described.  These have been predominantly driven by wider changes in society namely the computer, internet and globalisation.  Thoughts about the future of medicinal chemistry are also discussed including sharing the risks and costs of drug discovery and the future of outsourcing.  The continuing impact of access to substantial computing power and big data, the use of algorithms in data analysis and drug design are also presented.  The next generation of medicinal chemists will communicate in ways that reflect social media and the results of constantly being connected to each other and data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS2kUsYSYxxc_IW7hznt0zpfW6udTcc2eaheflYR-jRhrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M7gs1GltQ%253D%253D&md5=6e711ae6f4fcb7fd848c39c876bf3bfa</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2017.10.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2017.10.007%26sid%3Dliteratum%253Aachs%26aulast%3DCampbell%26aufirst%3DI.%2BB.%26aulast%3DMacdonald%26aufirst%3DS.%2BJ.%2BF.%26aulast%3DProcopiou%26aufirst%3DP.%2BA.%26atitle%3DMedicinal%2520chemistry%2520in%2520drug%2520discovery%2520in%2520big%2520pharma%253A%2520past%252C%2520present%2520and%2520future%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2018%26volume%3D23%26spage%3D219%26epage%3D234%26doi%3D10.1016%2Fj.drudis.2017.10.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':[],'ref30':['cit30','cit30.b'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33','cit33b'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 13 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Paul D. Leeson, A. Patricia Bento, Anna Gaulton, Anne Hersey, Emma J. Manners, Chris J. Radoux, <span class="NLM_string-name hlFld-ContribAuthor">Andrew R. Leach</span>. </span><span class="cited-content_cbyCitation_article-title">Target-Based Evaluation of “Drug-Like” Properties and Ligand Efficiencies. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (11)
                                     , 7210-7230. <a href="https://doi.org/10.1021/acs.jmedchem.1c00416" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00416</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00416&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.1c00416%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DTarget-Based%252BEvaluation%252Bof%252B%2525E2%252580%25259CDrug-Like%2525E2%252580%25259D%252BProperties%252Band%252BLigand%252BEfficiencies%26aulast%3DLeeson%26aufirst%3DPaul%2BD.%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D08032021%26date%3D13052021%26volume%3D64%26issue%3D11%26spage%3D7210%26epage%3D7230" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Marta Serafini, Sarah Cargnin, Alberto Massarotti, Gian Cesare Tron, Tracey Pirali, <span class="NLM_string-name hlFld-ContribAuthor">Armando A. Genazzani</span>. </span><span class="cited-content_cbyCitation_article-title">What’s in a Name? Drug Nomenclature and Medicinal Chemistry Trends using INN Publications. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (8)
                                     , 4410-4429. <a href="https://doi.org/10.1021/acs.jmedchem.1c00181" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00181</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00181&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.1c00181%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DWhat%2525E2%252580%252599s%252Bin%252Ba%252BName%25253F%252BDrug%252BNomenclature%252Band%252BMedicinal%252BChemistry%252BTrends%252Busing%252BINN%252BPublications%26aulast%3DSerafini%26aufirst%3DMarta%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D01022021%26date%3D13042021%26volume%3D64%26issue%3D8%26spage%3D4410%26epage%3D4429" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Huanan Wen, Rui Ge, Yi Qu, Jialin Sun, Xiaodong Shi, Weiren Cui, Hao Yan, Qi Zhang, Yulong An, Wenji Su, Hongfang Yang, Letian Kuai, Alexander L. Satz, <span class="NLM_string-name hlFld-ContribAuthor">Xuanjia Peng</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis of 1,2-Amino Alcohols by Photoredox-Mediated Decarboxylative Coupling of α-Amino Acids and DNA-Conjugated Carbonyls. </span><span class="cited-content_cbyCitation_journal-name">Organic Letters</span><span> <strong>2020,</strong> <em>22 </em>
                                    (24)
                                     , 9484-9489. <a href="https://doi.org/10.1021/acs.orglett.0c03461" title="DOI URL">https://doi.org/10.1021/acs.orglett.0c03461</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.orglett.0c03461&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.orglett.0c03461%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520Letters%26atitle%3DSynthesis%252Bof%252B1%25252C2-Amino%252BAlcohols%252Bby%252BPhotoredox-Mediated%252BDecarboxylative%252BCoupling%252Bof%252B%2525CE%2525B1-Amino%252BAcids%252Band%252BDNA-Conjugated%252BCarbonyls%26aulast%3DWen%26aufirst%3DHuanan%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D16102020%26date%3D10112020%26volume%3D22%26issue%3D24%26spage%3D9484%26epage%3D9489" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Yi-Wen Liu, Zhuo-Ya Mao, Xiao-Di Nie, Chang-Mei Si, Bang-Guo Wei, <span class="NLM_string-name hlFld-ContribAuthor">Guo-Qiang Lin</span>. </span><span class="cited-content_cbyCitation_article-title">Approach to Tertiary-Type β-Hydroxyl Carboxamides Through Sc(OTf)3-Catalyzed Addition of Ynamides and Ketones. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Organic Chemistry</span><span> <strong>2019,</strong> <em>84 </em>
                                    (24)
                                     , 16254-16261. <a href="https://doi.org/10.1021/acs.joc.9b02854" title="DOI URL">https://doi.org/10.1021/acs.joc.9b02854</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.joc.9b02854&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.joc.9b02854%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Organic%2520Chemistry%26atitle%3DApproach%252Bto%252BTertiary-Type%252B%2525CE%2525B2-Hydroxyl%252BCarboxamides%252BThrough%252BSc%252528OTf%2525293-Catalyzed%252BAddition%252Bof%252BYnamides%252Band%252BKetones%26aulast%3DLiu%26aufirst%3DYi-Wen%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D20102019%26date%3D12122019%26date%3D28112019%26volume%3D84%26issue%3D24%26spage%3D16254%26epage%3D16261" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Guoli  Xiong</span>, <span class="hlFld-ContribAuthor ">Zhenxing  Wu</span>, <span class="hlFld-ContribAuthor ">Jiacai  Yi</span>, <span class="hlFld-ContribAuthor ">Li  Fu</span>, <span class="hlFld-ContribAuthor ">Zhijiang  Yang</span>, <span class="hlFld-ContribAuthor ">Changyu  Hsieh</span>, <span class="hlFld-ContribAuthor ">Mingzhu  Yin</span>, <span class="hlFld-ContribAuthor ">Xiangxiang  Zeng</span>, <span class="hlFld-ContribAuthor ">Chengkun  Wu</span>, <span class="hlFld-ContribAuthor ">Aiping  Lu</span>, <span class="hlFld-ContribAuthor ">Xiang  Chen</span>, <span class="hlFld-ContribAuthor ">Tingjun  Hou</span>, <span class="hlFld-ContribAuthor ">Dongsheng  Cao</span>. </span><span class="cited-content_cbyCitation_article-title">ADMETlab 2.0: an integrated online platform for accurate and comprehensive predictions of ADMET properties. </span><span class="cited-content_cbyCitation_journal-name">Nucleic Acids Research</span><span> <strong>2021,</strong> <em>12 </em><a href="https://doi.org/10.1093/nar/gkab255" title="DOI URL">https://doi.org/10.1093/nar/gkab255</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/nar/gkab255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fnar%2Fgkab255%26sid%3Dliteratum%253Aachs%26jtitle%3DNucleic%2520Acids%2520Research%26atitle%3DADMETlab%252B2.0%25253A%252Ban%252Bintegrated%252Bonline%252Bplatform%252Bfor%252Baccurate%252Band%252Bcomprehensive%252Bpredictions%252Bof%252BADMET%252Bproperties%26aulast%3DXiong%26aufirst%3DGuoli%26date%3D2021%26date%3D2021%26volume%3D12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lewis D.  Pennington</span>, <span class="hlFld-ContribAuthor ">Ingo  Muegge</span>. </span><span class="cited-content_cbyCitation_article-title">Holistic Drug Design for Multiparameter Optimization in Modern Small Molecule Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2021,</strong> <em>10 </em>, 128003. <a href="https://doi.org/10.1016/j.bmcl.2021.128003" title="DOI URL">https://doi.org/10.1016/j.bmcl.2021.128003</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2021.128003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2021.128003%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DHolistic%252BDrug%252BDesign%252Bfor%252BMultiparameter%252BOptimization%252Bin%252BModern%252BSmall%252BMolecule%252BDrug%252BDiscovery%26aulast%3DPennington%26aufirst%3DLewis%2BD.%26date%3D2021%26volume%3D10%26spage%3D128003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Pedro  Brandão</span>, <span class="hlFld-ContribAuthor ">Carolina  Marques</span>, <span class="hlFld-ContribAuthor ">Anthony J.  Burke</span>, <span class="hlFld-ContribAuthor ">Marta  Pineiro</span>. </span><span class="cited-content_cbyCitation_article-title">The application of isatin-based multicomponent-reactions in the quest for new bioactive and druglike molecules. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>211 </em>, 113102. <a href="https://doi.org/10.1016/j.ejmech.2020.113102" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.113102</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.113102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.113102%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DThe%252Bapplication%252Bof%252Bisatin-based%252Bmulticomponent-reactions%252Bin%252Bthe%252Bquest%252Bfor%252Bnew%252Bbioactive%252Band%252Bdruglike%252Bmolecules%26aulast%3DBrand%25C3%25A3o%26aufirst%3DPedro%26date%3D2021%26volume%3D211%26spage%3D113102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tianbiao  Yang</span>, <span class="hlFld-ContribAuthor ">Zhaojun  Li</span>, <span class="hlFld-ContribAuthor ">Yingjia  Chen</span>, <span class="hlFld-ContribAuthor ">Dan  Feng</span>, <span class="hlFld-ContribAuthor ">Guangchao  Wang</span>, <span class="hlFld-ContribAuthor ">Zunyun  Fu</span>, <span class="hlFld-ContribAuthor ">Xiaoyu  Ding</span>, <span class="hlFld-ContribAuthor ">Xiaoqin  Tan</span>, <span class="hlFld-ContribAuthor ">Jihui  Zhao</span>, <span class="hlFld-ContribAuthor ">Xiaomin  Luo</span>, <span class="hlFld-ContribAuthor ">Kaixian  Chen</span>, <span class="hlFld-ContribAuthor ">Hualiang  Jiang</span>, <span class="hlFld-ContribAuthor ">Mingyue  Zheng</span>. </span><span class="cited-content_cbyCitation_article-title">DrugSpaceX: a large screenable and synthetically tractable database extending drug space. </span><span class="cited-content_cbyCitation_journal-name">Nucleic Acids Research</span><span> <strong>2021,</strong> <em>49 </em>
                                    (D1)
                                     , D1170-D1178. <a href="https://doi.org/10.1093/nar/gkaa920" title="DOI URL">https://doi.org/10.1093/nar/gkaa920</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/nar/gkaa920&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fnar%2Fgkaa920%26sid%3Dliteratum%253Aachs%26jtitle%3DNucleic%2520Acids%2520Research%26atitle%3DDrugSpaceX%25253A%252Ba%252Blarge%252Bscreenable%252Band%252Bsynthetically%252Btractable%252Bdatabase%252Bextending%252Bdrug%252Bspace%26aulast%3DYang%26aufirst%3DTianbiao%26date%3D2021%26date%3D2020%26volume%3D49%26issue%3DD1%26spage%3DD1170%26epage%3DD1178" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Wenxiu  Wei</span>, <span class="hlFld-ContribAuthor ">Srinivasulu  Cherukupalli</span>, <span class="hlFld-ContribAuthor ">Lanlan  Jing</span>, <span class="hlFld-ContribAuthor ">Xinyong  Liu</span>, <span class="hlFld-ContribAuthor ">Peng  Zhan</span>. </span><span class="cited-content_cbyCitation_article-title">Fsp3: A new parameter for drug-likeness. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today</span><span> <strong>2020,</strong> <em>25 </em>
                                    (10)
                                     , 1839-1845. <a href="https://doi.org/10.1016/j.drudis.2020.07.017" title="DOI URL">https://doi.org/10.1016/j.drudis.2020.07.017</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.drudis.2020.07.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.drudis.2020.07.017%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%26atitle%3DFsp3%25253A%252BA%252Bnew%252Bparameter%252Bfor%252Bdrug-likeness%26aulast%3DWei%26aufirst%3DWenxiu%26date%3D2020%26volume%3D25%26issue%3D10%26spage%3D1839%26epage%3D1845" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shu  Fan</span>, <span class="hlFld-ContribAuthor ">Yong-Xiao  Cao</span>, <span class="hlFld-ContribAuthor ">Guang-Yan  Li</span>, <span class="hlFld-ContribAuthor ">Hao  Lei</span>, <span class="hlFld-ContribAuthor ">Mawusse K.I.  Attiogbe</span>, <span class="hlFld-ContribAuthor ">Jing-Chun  Yao</span>, <span class="hlFld-ContribAuthor ">Xue-Yan  Yang</span>, <span class="hlFld-ContribAuthor ">Yan-Jie  Liu</span>, <span class="hlFld-ContribAuthor ">Yuan-Yuan  Hei</span>, <span class="hlFld-ContribAuthor ">Hao  Zhang</span>, <span class="hlFld-ContribAuthor ">Lei  Cao</span>, <span class="hlFld-ContribAuthor ">Xiao-Yan  Zhang</span>, <span class="hlFld-ContribAuthor ">Shuai-Shuai  Du</span>, <span class="hlFld-ContribAuthor ">Gui-Min  Zhang</span>, <span class="hlFld-ContribAuthor ">San-Qi  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">F10, a new camptothecin derivative, was identified as a new orally–bioavailable, potent antitumor agent. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>202 </em>, 112528. <a href="https://doi.org/10.1016/j.ejmech.2020.112528" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112528</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112528&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112528%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DF10%25252C%252Ba%252Bnew%252Bcamptothecin%252Bderivative%25252C%252Bwas%252Bidentified%252Bas%252Ba%252Bnew%252Borally%2525E2%252580%252593bioavailable%25252C%252Bpotent%252Bantitumor%252Bagent%26aulast%3DFan%26aufirst%3DShu%26date%3D2020%26volume%3D202%26spage%3D112528" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Badvel  Pallavi</span>, <span class="hlFld-ContribAuthor ">Prachi  Sharma</span>, <span class="hlFld-ContribAuthor ">Noorullah  Baig</span>, <span class="hlFld-ContribAuthor ">Vimal  Kumar Madduluri</span>, <span class="hlFld-ContribAuthor ">Ajay K.  Sah</span>, <span class="hlFld-ContribAuthor ">Udit  Saumya</span>, <span class="hlFld-ContribAuthor ">Uma S.  Dubey</span>, <span class="hlFld-ContribAuthor ">Paritosh  Shukla</span>. </span><span class="cited-content_cbyCitation_article-title">Quinoline Glycoconjugates as Potentially Anticancer and Anti‐Inflammatory Agents: An Investigation Involving Synthesis, Biological Screening, and Docking. </span><span class="cited-content_cbyCitation_journal-name">ChemistrySelect</span><span> <strong>2020,</strong> <em>5 </em>
                                    (31)
                                     , 9878-9882. <a href="https://doi.org/10.1002/slct.202002345" title="DOI URL">https://doi.org/10.1002/slct.202002345</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/slct.202002345&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fslct.202002345%26sid%3Dliteratum%253Aachs%26jtitle%3DChemistrySelect%26atitle%3DQuinoline%252BGlycoconjugates%252Bas%252BPotentially%252BAnticancer%252Band%252BAnti%2525E2%252580%252590Inflammatory%252BAgents%25253A%252BAn%252BInvestigation%252BInvolving%252BSynthesis%25252C%252BBiological%252BScreening%25252C%252Band%252BDocking%26aulast%3DPallavi%26aufirst%3DBadvel%26date%3D2020%26date%3D2020%26volume%3D5%26issue%3D31%26spage%3D9878%26epage%3D9882" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Oleksandr D.  Tereshchenko</span>, <span class="hlFld-ContribAuthor ">Maryana Y.  Perebiynis</span>, <span class="hlFld-ContribAuthor ">Irina V.  Knysh</span>, <span class="hlFld-ContribAuthor ">Olesia V.  Vasylets</span>, <span class="hlFld-ContribAuthor ">Anna A.  Sorochenko</span>, <span class="hlFld-ContribAuthor ">Eugeniy Y.  Slobodyanyuk</span>, <span class="hlFld-ContribAuthor ">Eduard B.  Rusanov</span>, <span class="hlFld-ContribAuthor ">Oleksandr V.  Borysov</span>, <span class="hlFld-ContribAuthor ">Sergey V.  Kolotilov</span>, <span class="hlFld-ContribAuthor ">Sergey V.  Ryabukhin</span>, <span class="hlFld-ContribAuthor ">Dmitriy M.  Volochnyuk</span>. </span><span class="cited-content_cbyCitation_article-title">Electrochemical Scaled‐up Synthesis of Cyclic Enecarbamates as Starting Materials for Medicinal Chemistry Relevant Building Bocks. </span><span class="cited-content_cbyCitation_journal-name">Advanced Synthesis & Catalysis</span><span> <strong>2020,</strong> <em>362 </em>
                                    (15)
                                     , 3229-3242. <a href="https://doi.org/10.1002/adsc.202000450" title="DOI URL">https://doi.org/10.1002/adsc.202000450</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/adsc.202000450&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fadsc.202000450%26sid%3Dliteratum%253Aachs%26jtitle%3DAdvanced%2520Synthesis%2520%2526%2520Catalysis%26atitle%3DElectrochemical%252BScaled%2525E2%252580%252590up%252BSynthesis%252Bof%252BCyclic%252BEnecarbamates%252Bas%252BStarting%252BMaterials%252Bfor%252BMedicinal%252BChemistry%252BRelevant%252BBuilding%252BBocks%26aulast%3DTereshchenko%26aufirst%3DOleksandr%2BD.%26date%3D2020%26date%3D2020%26volume%3D362%26issue%3D15%26spage%3D3229%26epage%3D3242" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Eiichi  Watanabe</span>, <span class="hlFld-ContribAuthor ">Yiding  Chen</span>, <span class="hlFld-ContribAuthor ">Oliver  May</span>, <span class="hlFld-ContribAuthor ">Steven V.  Ley</span>. </span><span class="cited-content_cbyCitation_article-title">A Practical Method for Continuous Production of sp3‐Rich Compounds from (Hetero)Aryl Halides and Redox‐Active Esters. </span><span class="cited-content_cbyCitation_journal-name">Chemistry – A European Journal</span><span> <strong>2020,</strong> <em>26 </em>
                                    (1)
                                     , 186-191. <a href="https://doi.org/10.1002/chem.201905048" title="DOI URL">https://doi.org/10.1002/chem.201905048</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/chem.201905048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fchem.201905048%26sid%3Dliteratum%253Aachs%26jtitle%3DChemistry%2520%25E2%2580%2593%2520A%2520European%2520Journal%26atitle%3DA%252BPractical%252BMethod%252Bfor%252BContinuous%252BProduction%252Bof%252Bsp3%2525E2%252580%252590Rich%252BCompounds%252Bfrom%252B%252528Hetero%252529Aryl%252BHalides%252Band%252BRedox%2525E2%252580%252590Active%252BEsters%26aulast%3DWatanabe%26aufirst%3DEiichi%26date%3D2020%26date%3D2019%26volume%3D26%26issue%3D1%26spage%3D186%26epage%3D191" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/medium/jm-2019-00004q_0015.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/large/jm-2019-00004q_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00004&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/medium/jm-2019-00004q_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/large/jm-2019-00004q_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Simplified illustration of the evolution of medicinal chemistry.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/large/jm-2019-00004q_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00004&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/medium/jm-2019-00004q_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/large/jm-2019-00004q_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Representative examples of PPI inhibitors that reached the market or are being evaluated in clinics.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/large/jm-2019-00004q_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00004&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/medium/jm-2019-00004q_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/large/jm-2019-00004q_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Devolution of medicinal chemistry expressed in the number of patent records by major pharmaceutical companies (bar chart, prior to normalization) sorted by the priority date vs continual growth of MCE-18 scores (black solid line, after normalization). ∗ indicates the year that the Lipinski seminal paper<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> was published.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/large/jm-2019-00004q_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00004&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/medium/jm-2019-00004q_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/large/jm-2019-00004q_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Top four pharmaceutical companies ranked by MCE-18 (after normalization).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/large/jm-2019-00004q_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00004&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/medium/jm-2019-00004q_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/large/jm-2019-00004q_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Number of patent records by major pharmaceutical companies (bar chart) sorted by the priority date vs continual growth of MCE-18 scores (solid lines): GPCR ligands (dark gray), protein kinase inhibitors (black), and molecules acting on proteases (gray).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/large/jm-2019-00004q_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00004&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/medium/jm-2019-00004q_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/large/jm-2019-00004q_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Distribution (%) of molecules with MCE-18 ≥ 50 among three key target families: (A) protein kinases; (B) proteases; (C) GPCRs.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/large/jm-2019-00004q_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00004&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/medium/jm-2019-00004q_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/large/jm-2019-00004q_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Representative examples of structures with high MCE-18 values claimed in patent records by major pharmaceutical companies during the past 5 years: (A) protein kinase inhibitors, (B) GPCR ligands, and (C) molecules acting on proteases.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/large/jm-2019-00004q_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00004&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/medium/jm-2019-00004q_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/large/jm-2019-00004q_0008.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Changes in MCE-18 components over the years.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/large/jm-2019-00004q_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00004&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/medium/jm-2019-00004q_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/large/jm-2019-00004q_0009.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. 20 × 20 2D Kohonen SOM reflecting the trends in key molecular properties of structures disclosed during the following periods: (a) 2015–2018; (b) 2001–2003; (c) 1980–1985. The color gradient indicates the number of molecules. Areas encircled by the white dotted lines correspond to high MCE-18 values. Basic contours of the map were smoothed for convenient visual inspection.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/large/jm-2019-00004q_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00004&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig10" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/medium/jm-2019-00004q_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/large/jm-2019-00004q_0010.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Kohonen SOM constructed for launched drugs: (a) white arrows indicate the route by which drugs have evolved over the years; (b) distribution of MCE-18 values within the same map; (c) areas occupied by the drugs brought to market during 1910–1979; (d) locations of new drugs (2011–2018).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/large/jm-2019-00004q_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00004&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig11" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/medium/jm-2019-00004q_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/large/jm-2019-00004q_0011.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. Number of drugs (bar chart) sorted by the related milestone date vs continual growth of MCE-18 values (solid lines): GPCR ligands (dark gray), protein kinase inhibitors (black), and molecules acting on proteases (gray).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/large/jm-2019-00004q_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00004&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig12" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 12</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/medium/jm-2019-00004q_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/large/jm-2019-00004q_0012.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 12. Distribution of lead compounds and drugs within the Kohonen map: (a) molecules in preclinical investigation, (b) compounds examined (or are currently being examined) in clinical trials, (c) the distribution of MCE-18 values, (d) areas occupied by drugs launched from 1910 to 1987, and (e) new drugs brought to market in recent years (2011–2018).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/large/jm-2019-00004q_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00004&amp;id=fig12"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig13" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 13</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/medium/jm-2019-00004q_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/large/jm-2019-00004q_0013.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 13. Sammon map constructed for the launched drugs. “+” means that PSA < 140 Å was added to the classic Lipinski’s rules.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/large/jm-2019-00004q_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00004&amp;id=fig13"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig14" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Figure 14</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/medium/jm-2019-00004q_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/large/jm-2019-00004q_0014.jpeg" id="rightTab-gr14" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 14. Vendor collections scored by MCE-18.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00004/20191122/images/large/jm-2019-00004q_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00004&amp;id=fig14"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i31">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_28722" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_28722" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 65 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mullard, A.</span></span> <span> </span><span class="NLM_article-title">The drug maker’s guide to the galaxy</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>549</i></span>,  <span class="NLM_fpage">445</span>– <span class="NLM_lpage">447</span>, <span class="refDoi"> DOI: 10.1038/549445a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=10.1038%2F549445a" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=28959982" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A280%3ADC%252BC1M%252FjsFSmtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=549&publication_year=2017&pages=445-447&author=A.+Mullard&title=The+drug+maker%E2%80%99s+guide+to+the+galaxy&doi=10.1038%2F549445a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The drug-maker's guide to the galaxy</span></div><div class="casAuthors">Mullard Asher</div><div class="citationInfo"><span class="NLM_cas:title">Nature</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">549</span>
        (<span class="NLM_cas:issue">7673</span>),
    <span class="NLM_cas:pages">445-447</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSItGWG-TCuJEg6wdEZmT2rfW6udTcc2eYNlOe2sx2qh7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M%252FjsFSmtA%253D%253D&md5=4ff8347906c8f3b459d05a5fe7ad0b56</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2F549445a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F549445a%26sid%3Dliteratum%253Aachs%26aulast%3DMullard%26aufirst%3DA.%26atitle%3DThe%2520drug%2520maker%25E2%2580%2599s%2520guide%2520to%2520the%2520galaxy%26jtitle%3DNature%26date%3D2017%26volume%3D549%26spage%3D445%26epage%3D447%26doi%3D10.1038%2F549445a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bohacek, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMartin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guida, W. C.</span></span> <span> </span><span class="NLM_article-title">The art and practice of structure-based drug design: a molecular modeling perspective</span>. <i>Med. Res. Rev.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">3</span>– <span class="NLM_lpage">50</span>, <span class="refDoi"> DOI: 10.1002/(SICI)1098-1128(199601)16:1<3::AID-MED1>3.0.CO;2-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=10.1002%2F%28SICI%291098-1128%28199601%2916%3A1%3C3%3A%3AAID-MED1%3E3.0.CO%3B2-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=8788213" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADyaK28XhtFyls78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=1996&pages=3-50&author=R.+S.+Bohacekauthor=C.+McMartinauthor=W.+C.+Guida&title=The+art+and+practice+of+structure-based+drug+design%3A+a+molecular+modeling+perspective&doi=10.1002%2F%28SICI%291098-1128%28199601%2916%3A1%3C3%3A%3AAID-MED1%3E3.0.CO%3B2-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">The art and practice of structure-based drug design: a molecular modeling perspective</span></div><div class="casAuthors">Bohacek, Regine S.; McMartin, Colin; Guida, Wayne C.</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">3-50</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">Wiley</span>)
        </div><div class="casAbstract">A review with 120 refs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmfrWdtPosdrVg90H21EOLACvtfcHk0liKENZYljILJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XhtFyls78%253D&md5=edfa592791c111479117feed600dead6</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291098-1128%28199601%2916%3A1%3C3%3A%3AAID-MED1%3E3.0.CO%3B2-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291098-1128%2528199601%252916%253A1%253C3%253A%253AAID-MED1%253E3.0.CO%253B2-6%26sid%3Dliteratum%253Aachs%26aulast%3DBohacek%26aufirst%3DR.%2BS.%26aulast%3DMcMartin%26aufirst%3DC.%26aulast%3DGuida%26aufirst%3DW.%2BC.%26atitle%3DThe%2520art%2520and%2520practice%2520of%2520structure-based%2520drug%2520design%253A%2520a%2520molecular%2520modeling%2520perspective%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D1996%26volume%3D16%26spage%3D3%26epage%3D50%26doi%3D10.1002%2F%28SICI%291098-1128%28199601%2916%3A1%3C3%3A%3AAID-MED1%3E3.0.CO%3B2-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boström, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keserü, G. M.</span></span> <span> </span><span class="NLM_article-title">Expanding the medicinal chemistry synthetic toolbox</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">709</span>– <span class="NLM_lpage">727</span>, <span class="refDoi"> DOI: 10.1038/nrd.2018.116</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=10.1038%2Fnrd.2018.116" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=30140018" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A280%3ADC%252BB3c3htlWisw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2018&pages=709-727&author=J.+G.+Bostr%C3%B6mauthor=D.+G.+Brownauthor=R.+J.+Youngauthor=G.+M.+Keser%C3%BC&title=Expanding+the+medicinal+chemistry+synthetic+toolbox&doi=10.1038%2Fnrd.2018.116"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Expanding the medicinal chemistry synthetic toolbox</span></div><div class="casAuthors">Bostrom Jonas; Brown Dean G; Young Robert J; Keseru Gyorgy M</div><div class="citationInfo"><span class="NLM_cas:title">Nature reviews. Drug discovery</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">709-727</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The key objectives of medicinal chemistry are to efficiently design and synthesize bioactive compounds that have the potential to become safe and efficacious drugs.  Most medicinal chemistry programmes rely on screening compound collections populated by a range of molecules derived from a set of known and robust chemistry reactions.  Analysis of the role of synthetic organic chemistry in subsequent hit and lead optimization efforts suggests that only a few reactions dominate.  Thus, the uptake of new synthetic methodologies in drug discovery is limited.  Starting from the known limitations of reaction parameters, synthesis design tools, synthetic strategies and innovative chemistries, here we highlight opportunities for the expansion of the medicinal chemists' synthetic toolbox.  More intense crosstalk between synthetic and medicinal chemists in industry and academia should enable enhanced impact of new methodologies in future drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTy8c_ueHSwcYMcDoIzZtNkfW6udTcc2eafAWeuZCZt37ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c3htlWisw%253D%253D&md5=d87e988602808139597576ced56faa88</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2018.116&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2018.116%26sid%3Dliteratum%253Aachs%26aulast%3DBostr%25C3%25B6m%26aufirst%3DJ.%2BG.%26aulast%3DBrown%26aufirst%3DD.%2BG.%26aulast%3DYoung%26aufirst%3DR.%2BJ.%26aulast%3DKeser%25C3%25BC%26aufirst%3DG.%2BM.%26atitle%3DExpanding%2520the%2520medicinal%2520chemistry%2520synthetic%2520toolbox%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2018%26volume%3D17%26spage%3D709%26epage%3D727%26doi%3D10.1038%2Fnrd.2018.116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span>BioGRID
Database Statistics. <a href="https://wiki.thebiogrid.org/doku.php/statistics" class="extLink">https://wiki.thebiogrid.org/doku.php/statistics</a> (accessed Feb 27, <span class="NLM_year">2019</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=BioGRID%0ADatabase+Statistics.+https%3A%2F%2Fwiki.thebiogrid.org%2Fdoku.php%2Fstatistics+%28accessed+Feb+27%2C+2019%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ran, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gestwicki, J. E.</span></span> <span> </span><span class="NLM_article-title">Inhibitors of protein-protein interactions (PPIs): an analysis of scaffold choices and buried surface area</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">75</span>– <span class="NLM_lpage">86</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2018.06.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=10.1016%2Fj.cbpa.2018.06.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=29908451" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFCmsbnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2018&pages=75-86&author=X.+Ranauthor=J.+E.+Gestwicki&title=Inhibitors+of+protein-protein+interactions+%28PPIs%29%3A+an+analysis+of+scaffold+choices+and+buried+surface+area&doi=10.1016%2Fj.cbpa.2018.06.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitors of protein-protein interactions (PPIs): an analysis of scaffold choices and buried surface area</span></div><div class="casAuthors">Ran, Xu; Gestwicki, Jason E.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">75-86</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Protein-protein interactions (PPI) were once considered 'undruggable', but clin. successes, driven by advanced methods in drug discovery, have challenged that notion.  Here, we review the last three years of literature on PPI inhibitors to understand what is working and why.  From the 66 recently reported PPI inhibitors, we found that the av. mol. wt. was significantly greater than 500 Da, but that this trend was driven, in large part, by the contribution of peptide-based compds.  Despite differences in av. mol. wt., we found that compds. based on small mols. or peptides were almost equally likely to be potent inhibitors (KD < 1 μM).  Finally, we found PPIs with buried surface area (BSA) less than 2000 Å2 were more likely to be inhibited by small mols., while PPIs with larger BSA values were typically inhibited by peptides.  PPIs with BSA values over 4000 Å2 seemed to create a particular challenge, esp. for orthosteric small mols.  Thus, it seems important to choose the inhibitor scaffold based on the properties of the target interaction.  Moreover, this survey suggests a (more nuanced) conclusion to the question of whether PPIs are good drug targets; namely, that some PPIs are readily 'druggable' given the right choice of scaffold, while others still seem to deserve the 'undruggable' moniker.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpoS9hriXFuS7Vg90H21EOLACvtfcHk0liKENZYljILJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFCmsbnE&md5=037a06e18fc78112243cd68ce6752599</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2018.06.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2018.06.004%26sid%3Dliteratum%253Aachs%26aulast%3DRan%26aufirst%3DX.%26aulast%3DGestwicki%26aufirst%3DJ.%2BE.%26atitle%3DInhibitors%2520of%2520protein-protein%2520interactions%2520%2528PPIs%2529%253A%2520an%2520analysis%2520of%2520scaffold%2520choices%2520and%2520buried%2520surface%2520area%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2018%26volume%3D44%26spage%3D75%26epage%3D86%26doi%3D10.1016%2Fj.cbpa.2018.06.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tice, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, S. B.</span></span> <span> </span><span class="NLM_article-title">The use of spirocyclic scaffolds in drug discovery</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">3673</span>– <span class="NLM_lpage">3682</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2014.06.081</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=10.1016%2Fj.bmcl.2014.06.081" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=25052427" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFykt7bM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2014&pages=3673-3682&author=Y.+Zhengauthor=C.+M.+Ticeauthor=S.+B.+Singh&title=The+use+of+spirocyclic+scaffolds+in+drug+discovery&doi=10.1016%2Fj.bmcl.2014.06.081"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">The use of spirocyclic scaffolds in drug discovery</span></div><div class="casAuthors">Zheng, Yajun; Tice, Colin M.; Singh, Suresh B.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">3673-3682</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Owing to their inherent three-dimensionality and structural novelty, spiro scaffolds have been increasingly utilized in drug discovery.  In this brief review, we highlight selected examples from the primary medicinal chem. literature during the last three years to demonstrate the versatility of spiro scaffolds.  With recent progress in synthetic methods providing access to spiro building blocks, spiro scaffolds are likely to be used more frequently in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqzTjJrsRkyLVg90H21EOLACvtfcHk0lgTpX2vV5hRBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFykt7bM&md5=9a2a644cf2b97edf7876b43b6457f6d5</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2014.06.081&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2014.06.081%26sid%3Dliteratum%253Aachs%26aulast%3DZheng%26aufirst%3DY.%26aulast%3DTice%26aufirst%3DC.%2BM.%26aulast%3DSingh%26aufirst%3DS.%2BB.%26atitle%3DThe%2520use%2520of%2520spirocyclic%2520scaffolds%2520in%2520drug%2520discovery%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D24%26spage%3D3673%26epage%3D3682%26doi%3D10.1016%2Fj.bmcl.2014.06.081" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pammolli, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magazzini, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riccaboni, M.</span></span> <span> </span><span class="NLM_article-title">The productivity crisis in pharmaceutical R&D</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">428</span>– <span class="NLM_lpage">438</span>, <span class="refDoi"> DOI: 10.1038/nrd3405</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=10.1038%2Fnrd3405" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=21629293" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmvVegt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=428-438&author=F.+Pammolliauthor=L.+Magazziniauthor=M.+Riccaboni&title=The+productivity+crisis+in+pharmaceutical+R%26D&doi=10.1038%2Fnrd3405"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">The productivity crisis in pharmaceutical R&D</span></div><div class="casAuthors">Pammolli, Fabio; Magazzini, Laura; Riccaboni, Massimo</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">428-438</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Advances in the understanding of the mol. basis of diseases have expanded the no. of plausible therapeutic targets for the development of innovative agents in recent decades.  However, although investment in pharmaceutical research and development (R&D) has increased substantially in this time, the lack of a corresponding increase in the output in terms of new drugs being approved indicates that therapeutic innovation has become more challenging.  Here, using a large database that contains information on R&D projects for more than 28,000 compds. investigated since 1990, we examine the decline of R&D productivity in pharmaceuticals in the past two decades and its determinants.  We show that this decline is assocd. with an increasing concn. of R&D investments in areas in which the risk of failure is high, which correspond to unmet therapeutic needs and unexploited biol. mechanisms.  We also investigate the potential variations in productivity with regard to the regional location of companies and find that although companies based in the United States and Europe differ in the compn. of their R&D portfolios, there is no evidence of any productivity gap.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4lNnATFMYqLVg90H21EOLACvtfcHk0lgTpX2vV5hRBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmvVegt7o%253D&md5=8adb683913f6b27f212f05ee0220ff05</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnrd3405&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3405%26sid%3Dliteratum%253Aachs%26aulast%3DPammolli%26aufirst%3DF.%26aulast%3DMagazzini%26aufirst%3DL.%26aulast%3DRiccaboni%26aufirst%3DM.%26atitle%3DThe%2520productivity%2520crisis%2520in%2520pharmaceutical%2520R%2526D%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2011%26volume%3D10%26spage%3D428%26epage%3D438%26doi%3D10.1038%2Fnrd3405" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scannell, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blanckley, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boldon, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warrington, B.</span></span> <span> </span><span class="NLM_article-title">Diagnosing the decline in pharmaceutical R&D efficiency</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">191</span>– <span class="NLM_lpage">200</span>, <span class="refDoi"> DOI: 10.1038/nrd3681</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=10.1038%2Fnrd3681" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=22378269" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADC%252BC38XivFyhtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=191-200&author=J.+W.+Scannellauthor=A.+Blanckleyauthor=H.+Boldonauthor=B.+Warrington&title=Diagnosing+the+decline+in+pharmaceutical+R%26D+efficiency&doi=10.1038%2Fnrd3681"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Diagnosing the decline in pharmaceutical R&D efficiency</span></div><div class="casAuthors">Scannell, Jack W.; Blanckley, Alex; Boldon, Helen; Warrington, Brian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">191-200</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The past 60 years have seen huge advances in many of the scientific, technol. and managerial factors that should tend to raise the efficiency of com. drug research and development (RD).  Yet the no. of new drugs approved per billion US dollars spent on RD has halved roughly every 9 years since 1950, falling around 80-fold in inflation-adjusted terms.  There have been many proposed solns. to the problem of declining RD efficiency.  However, their apparent lack of impact so far and the contrast between improving inputs and declining output in terms of the no. of new drugs make it sensible to ask whether the underlying problems have been correctly diagnosed.  Here, we discuss four factors that we consider to be primary causes, which we call the 'better than the Beatles' problem; the 'cautious regulator' problem; the 'throw money at it' tendency; and the 'basic research-brute force' bias.  Our aim is to provoke a more systematic anal. of the causes of the decline in RD efficiency.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5e_81DC6r8LVg90H21EOLACvtfcHk0lgTpX2vV5hRBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XivFyhtrY%253D&md5=db3e4548260e55f47a9964e8d1b216fd</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnrd3681&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3681%26sid%3Dliteratum%253Aachs%26aulast%3DScannell%26aufirst%3DJ.%2BW.%26aulast%3DBlanckley%26aufirst%3DA.%26aulast%3DBoldon%26aufirst%3DH.%26aulast%3DWarrington%26aufirst%3DB.%26atitle%3DDiagnosing%2520the%2520decline%2520in%2520pharmaceutical%2520R%2526D%2520efficiency%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2012%26volume%3D11%26spage%3D191%26epage%3D200%26doi%3D10.1038%2Fnrd3681" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shih, H. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aronov, A. M.</span></span> <span> </span><span class="NLM_article-title">Drug discovery effectiveness from the standpoint of therapeutic mechanisms and indications</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">19</span>– <span class="NLM_lpage">33</span>, <span class="refDoi"> DOI: 10.1038/nrd.2017.194</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=10.1038%2Fnrd.2017.194" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=29075002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslansbzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2018&pages=19-33&author=H.+P.+Shihauthor=X.+Zhangauthor=A.+M.+Aronov&title=Drug+discovery+effectiveness+from+the+standpoint+of+therapeutic+mechanisms+and+indications&doi=10.1038%2Fnrd.2017.194"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Drug discovery effectiveness from the standpoint of therapeutic mechanisms and indications</span></div><div class="casAuthors">Shih, Hsin-Pei; Zhang, Xiaodan; Aronov, Alex M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">19-33</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The productivity of the pharmaceutical industry has been widely discussed in recent years, particularly with regard to concerns that substantial expenditures on research and development have failed to translate into approved drugs.  Various analyses of this productivity challenge have focused on aspects such as attrition rates at particular clin. phases or the physicochem. properties of drug candidates, but relatively little attention has been paid to how the industry has performed from the standpoint of the choice of therapeutic mechanisms and their intended indications.  This article examines what the pharmaceutical industry has achieved in this respect by analyzing comprehensive industry-wide data on the mechanism-indication pairs that have been investigated during the past 20 years.  Our findings indicate several points and trends that we hope will be useful in understanding and improving the productivity of the industry, including areas in which the industry has had substantial success or failure and the relative extent of novelty in completed and ongoing projects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmeZIs-mlGdLVg90H21EOLACvtfcHk0lgTpX2vV5hRBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslansbzL&md5=aa47b6bbc0c2ee0fbb51927ce9b9b369</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2017.194&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2017.194%26sid%3Dliteratum%253Aachs%26aulast%3DShih%26aufirst%3DH.%2BP.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DAronov%26aufirst%3DA.%2BM.%26atitle%3DDrug%2520discovery%2520effectiveness%2520from%2520the%2520standpoint%2520of%2520therapeutic%2520mechanisms%2520and%2520indications%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2018%26volume%3D17%26spage%3D19%26epage%3D33%26doi%3D10.1038%2Fnrd.2017.194" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pammolli, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magazzini, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riccaboni, M.</span></span> <span> </span><span class="NLM_article-title">The productivity crisis in pharmaceutical R&D</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">428</span>– <span class="NLM_lpage">438</span>, <span class="refDoi"> DOI: 10.1038/nrd3405</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=10.1038%2Fnrd3405" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=21629293" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmvVegt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=428-438&author=F.+Pammolliauthor=L.+Magazziniauthor=M.+Riccaboni&title=The+productivity+crisis+in+pharmaceutical+R%26D&doi=10.1038%2Fnrd3405"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">The productivity crisis in pharmaceutical R&D</span></div><div class="casAuthors">Pammolli, Fabio; Magazzini, Laura; Riccaboni, Massimo</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">428-438</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Advances in the understanding of the mol. basis of diseases have expanded the no. of plausible therapeutic targets for the development of innovative agents in recent decades.  However, although investment in pharmaceutical research and development (R&D) has increased substantially in this time, the lack of a corresponding increase in the output in terms of new drugs being approved indicates that therapeutic innovation has become more challenging.  Here, using a large database that contains information on R&D projects for more than 28,000 compds. investigated since 1990, we examine the decline of R&D productivity in pharmaceuticals in the past two decades and its determinants.  We show that this decline is assocd. with an increasing concn. of R&D investments in areas in which the risk of failure is high, which correspond to unmet therapeutic needs and unexploited biol. mechanisms.  We also investigate the potential variations in productivity with regard to the regional location of companies and find that although companies based in the United States and Europe differ in the compn. of their R&D portfolios, there is no evidence of any productivity gap.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4lNnATFMYqLVg90H21EOLACvtfcHk0liYaB6naEtgTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmvVegt7o%253D&md5=8adb683913f6b27f212f05ee0220ff05</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnrd3405&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3405%26sid%3Dliteratum%253Aachs%26aulast%3DPammolli%26aufirst%3DF.%26aulast%3DMagazzini%26aufirst%3DL.%26aulast%3DRiccaboni%26aufirst%3DM.%26atitle%3DThe%2520productivity%2520crisis%2520in%2520pharmaceutical%2520R%2526D%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2011%26volume%3D10%26spage%3D428%26epage%3D438%26doi%3D10.1038%2Fnrd3405" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Waring, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leach, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leeson, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mandrell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owen, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pairaudeau, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pennie, W. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pickett, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wallace, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weir, A.</span></span> <span> </span><span class="NLM_article-title">An analysis of the attrition of drug candidates from four major pharmaceutical companies</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">475</span>– <span class="NLM_lpage">486</span>, <span class="refDoi"> DOI: 10.1038/nrd4609</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=10.1038%2Fnrd4609" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=26091267" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7jM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2015&pages=475-486&author=M.+J.+Waringauthor=J.+Arrowsmithauthor=A.+R.+Leachauthor=P.+D.+Leesonauthor=S.+Mandrellauthor=R.+M.+Owenauthor=G.+Pairaudeauauthor=W.+D.+Pennieauthor=S.+D.+Pickettauthor=J.+Wangauthor=O.+Wallaceauthor=A.+Weir&title=An+analysis+of+the+attrition+of+drug+candidates+from+four+major+pharmaceutical+companies&doi=10.1038%2Fnrd4609"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">An analysis of the attrition of drug candidates from four major pharmaceutical companies</span></div><div class="casAuthors">Waring, Michael J.; Arrowsmith, John; Leach, Andrew R.; Leeson, Paul D.; Mandrell, Sam; Owen, Robert M.; Pairaudeau, Garry; Pennie, William D.; Pickett, Stephen D.; Wang, Jibo; Wallace, Owen; Weir, Alex</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">475-486</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The pharmaceutical industry remains under huge pressure to address the high attrition rates in drug development.  Attempts to reduce the no. of efficacy- and safety-related failures by analyzing possible links to the physicochem. properties of small-mol. drug candidates have been inconclusive because of the limited size of data sets from individual companies.  Here, we describe the compilation and anal. of combined data on the attrition of drug candidates from AstraZeneca, Eli Lilly and Company, GlaxoSmithKline and Pfizer.  The anal. reaffirms that control of physicochem. properties during compd. optimization is beneficial in identifying compds. of candidate drug quality and indicates for the first time a link between the physicochem. properties of compds. and clin. failure due to safety issues.  The results also suggest that further control of physicochem. properties is unlikely to have a significant effect on attrition rates and that addnl. work is required to address safety-related failures.  Further cross-company collaborations will be crucial to future progress in this area.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsLfBWrLC3ZLVg90H21EOLACvtfcHk0liYaB6naEtgTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7jM&md5=1fdc374d32816b1e91438152299dd1b1</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fnrd4609&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4609%26sid%3Dliteratum%253Aachs%26aulast%3DWaring%26aufirst%3DM.%2BJ.%26aulast%3DArrowsmith%26aufirst%3DJ.%26aulast%3DLeach%26aufirst%3DA.%2BR.%26aulast%3DLeeson%26aufirst%3DP.%2BD.%26aulast%3DMandrell%26aufirst%3DS.%26aulast%3DOwen%26aufirst%3DR.%2BM.%26aulast%3DPairaudeau%26aufirst%3DG.%26aulast%3DPennie%26aufirst%3DW.%2BD.%26aulast%3DPickett%26aufirst%3DS.%2BD.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWallace%26aufirst%3DO.%26aulast%3DWeir%26aufirst%3DA.%26atitle%3DAn%2520analysis%2520of%2520the%2520attrition%2520of%2520drug%2520candidates%2520from%2520four%2520major%2520pharmaceutical%2520companies%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2015%26volume%3D14%26spage%3D475%26epage%3D486%26doi%3D10.1038%2Fnrd4609" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mirasol, F.</span></span> 2016 NME Drug Approvals Drop After Recent Highs. <a href="https://www.dcatvci.org/286-2016-nme-drug-approvals-drop-after-recent-highs" class="extLink">https://www.dcatvci.org/286-2016-nme-drug-approvals-drop-after-recent-highs</a> (accessed Feb 27, 2019).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Mirasol%2C+F.+2016+NME+Drug+Approvals+Drop+After+Recent+Highs.+https%3A%2F%2Fwww.dcatvci.org%2F286-2016-nme-drug-approvals-drop-after-recent-highs+%28accessed+Feb+27%2C+2019%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26aulast%3DMirasol%26aufirst%3DF." title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kola, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landis, J.</span></span> <span> </span><span class="NLM_article-title">Can the pharmaceutical industry reduce attrition rates?</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">711</span>– <span class="NLM_lpage">715</span>, <span class="refDoi"> DOI: 10.1038/nrd1470</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=10.1038%2Fnrd1470" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=15286737" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmtVOhtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2004&pages=711-715&author=I.+Kolaauthor=J.+Landis&title=Can+the+pharmaceutical+industry+reduce+attrition+rates%3F&doi=10.1038%2Fnrd1470"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Opinion: Can the pharmaceutical industry reduce attrition rates?</span></div><div class="casAuthors">Kola, Ismail; Landis, John</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">711-716</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The pharmaceutical industry faces considerable challenges, both politically and fiscally.  Politically, governments around the world are trying to contain costs and, as health care budgets constitute a very significant part of governmental spending, these costs are the subject of intense scrutiny.  In the United States, drug costs are also the subject of intense political discourse.  This article deals with the fiscal pressures that face the industry from the perspective of R&D.  What impinges on productivity How can we improve current reduced R&D productivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruAlrpFPFmiLVg90H21EOLACvtfcHk0liYaB6naEtgTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmtVOhtLs%253D&md5=f9025c13a1506f607aaf68415570ed01</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fnrd1470&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd1470%26sid%3Dliteratum%253Aachs%26aulast%3DKola%26aufirst%3DI.%26aulast%3DLandis%26aufirst%3DJ.%26atitle%3DCan%2520the%2520pharmaceutical%2520industry%2520reduce%2520attrition%2520rates%253F%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2004%26volume%3D3%26spage%3D711%26epage%3D715%26doi%3D10.1038%2Fnrd1470" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lipinski, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lombardo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dominy, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feeney, P. J.</span></span> <span> </span><span class="NLM_article-title">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span>. <i>Adv. Drug Delivery Rev.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">3</span>– <span class="NLM_lpage">25</span>, <span class="refDoi"> DOI: 10.1016/S0169-409X(96)00423-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=10.1016%2FS0169-409X%2896%2900423-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADyaK2sXktlKlsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=1997&pages=3-25&author=C.+A.+Lipinskiauthor=F.+Lombardoauthor=B.+W.+Dominyauthor=P.+J.+Feeney&title=Experimental+and+computational+approaches+to+estimate+solubility+and+permeability+in+drug+discovery+and+development+settings&doi=10.1016%2FS0169-409X%2896%2900423-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span></div><div class="casAuthors">Lipinski, Christopher A.; Lombardo, Franco; Dominy, Beryl W.; Feeney, Paul J.</div><div class="citationInfo"><span class="NLM_cas:title">Advanced Drug Delivery Reviews</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">3-25</span>CODEN:
                <span class="NLM_cas:coden">ADDREP</span>;
        ISSN:<span class="NLM_cas:issn">0169-409X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review with 50 refs.  Exptl. and computational approaches to est. soly. and permeability in discovery and development settings are described.  In the discovery setting 'the rule of 5' predicts that poor absorption or permeation is more likely when there are more than 5 H-bond donors, 10 H-bond acceptors, the mol. wt. (MWT) is >500 and the calcd. Log P (CLogP) is >5.  Computational methodol. for the rule-based Moriguchi Log P (MLogP) calcn. is described.  Turbidimetric soly. measurement is described and applied to known drugs.  High throughput screening (HTS) leads tend to have higher MWT and Log P and lower turbidimetric soly. than leads in the pre-HTS era.  In the development setting, soly. calcns. focus on exact value prediction and are difficult because of polymorphism.  Recent work on linear free energy relationships and Log P approaches are critically reviewed.  Useful predictions are possible in closely related analog series when coupled with exptl. thermodn. soly. measurements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGon9gEeZ7cboLVg90H21EOLACvtfcHk0lh9KdzQbJUxtQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXktlKlsQ%253D%253D&md5=405f70b0594d428f1275e1d56642cd3a</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2FS0169-409X%2896%2900423-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0169-409X%252896%252900423-1%26sid%3Dliteratum%253Aachs%26aulast%3DLipinski%26aufirst%3DC.%2BA.%26aulast%3DLombardo%26aufirst%3DF.%26aulast%3DDominy%26aufirst%3DB.%2BW.%26aulast%3DFeeney%26aufirst%3DP.%2BJ.%26atitle%3DExperimental%2520and%2520computational%2520approaches%2520to%2520estimate%2520solubility%2520and%2520permeability%2520in%2520drug%2520discovery%2520and%2520development%2520settings%26jtitle%3DAdv.%2520Drug%2520Delivery%2520Rev.%26date%3D1997%26volume%3D23%26spage%3D3%26epage%3D25%26doi%3D10.1016%2FS0169-409X%2896%2900423-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Degoey, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wendt, M. D.</span></span> <span> </span><span class="NLM_article-title">Beyond the Rule of 5: lessons learned from AbbVie’s drugs and compound collection</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">2636</span>– <span class="NLM_lpage">2651</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00717</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00717" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFeitbjJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=2636-2651&author=D.+A.+Degoeyauthor=H.+Chenauthor=P.+B.+Coxauthor=M.+D.+Wendt&title=Beyond+the+Rule+of+5%3A+lessons+learned+from+AbbVie%E2%80%99s+drugs+and+compound+collection&doi=10.1021%2Facs.jmedchem.7b00717"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Beyond the Rule of 5: Lessons Learned from AbbVie's Drugs and Compound Collection</span></div><div class="casAuthors">DeGoey, David A.; Chen, Hui-Ju; Cox, Philip B.; Wendt, Michael D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2636-2651</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Recently, there has been an increasing focus on the pursuit of targets considered to be less druggable that offer potential for development of promising new therapeutic agents for the treatment of diseases with large unmet medical need, particularly in the areas of oncol. and virol.  However, conducting drug discovery campaigns in "beyond rule of 5" (bRo5) chem. space presents a significant drug design and development challenge to medicinal chemists to achieve acceptable oral pharmacokinetics.  Retrospective anal. of past successes and failures in drug discovery bRo5 may shed light on the key principles that contribute to the oral bioavailability of successful bRo5 compds. and improve the efficiency of drug design for future projects.  We present here highlights and case studies of lessons learned from discovery of bRo5 compds.  A simple multiparametric scoring function (AB-MPS) was devised that correlated preclin. PK results with cLogD, no. of rotatable bonds, and no. of arom. rings.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTHuaXbtjlALVg90H21EOLACvtfcHk0lh9KdzQbJUxtQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFeitbjJ&md5=48984dc65410d83d4ac8ddaca030b024</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00717&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00717%26sid%3Dliteratum%253Aachs%26aulast%3DDegoey%26aufirst%3DD.%2BA.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DCox%26aufirst%3DP.%2BB.%26aulast%3DWendt%26aufirst%3DM.%2BD.%26atitle%3DBeyond%2520the%2520Rule%2520of%25205%253A%2520lessons%2520learned%2520from%2520AbbVie%25E2%2580%2599s%2520drugs%2520and%2520compound%2520collection%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D2636%26epage%3D2651%26doi%3D10.1021%2Facs.jmedchem.7b00717" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lovering, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bikker, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humblet, C.</span></span> <span> </span><span class="NLM_article-title">Escape from flatland: increasing saturation as an approach to improving clinical success</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">6752</span>– <span class="NLM_lpage">6756</span>, <span class="refDoi"> DOI: 10.1021/jm901241e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901241e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1KjtLvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=6752-6756&author=F.+Loveringauthor=J.+Bikkerauthor=C.+Humblet&title=Escape+from+flatland%3A+increasing+saturation+as+an+approach+to+improving+clinical+success&doi=10.1021%2Fjm901241e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Escape from Flatland: Increasing Saturation as an Approach to Improving Clinical Success</span></div><div class="casAuthors">Lovering, Frank; Bikker, Jack; Humblet, Christine</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">6752-6756</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The medicinal chem. community has become increasingly aware of the value of tracking calcd. phys. properties such as mol. wt., topol. polar surface area, rotatable bonds, and hydrogen bond donors and acceptors.  The authors hypothesized that the shift to high-throughput synthetic practices over the past decade may be another factor that may predispose mols. to fail by steering discovery efforts toward achiral, arom. compds.  The authors have proposed two simple and interpretable measures of the complexity of mols. prepd. as potential drug candidates.  The first is carbon bond satn. as defined by fraction Sp3 (Fsp3) where Fsp3 = (no. of Sp3 hybridized carbons/total carbon count).  The second is simply whether a chiral carbon exists in the mol.  The authors demonstrate that both complexity (as measured by Fsp3) and the presence of chiral centers correlate with success as compds. transition from discovery, through clin. testing, to drugs.  To explain these observations, the authors further demonstrate that satn. correlates with soly., an exptl. phys. property important to success in the drug discovery setting.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoC1f3Ed7nUBbVg90H21EOLACvtfcHk0lh9KdzQbJUxtQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1KjtLvN&md5=4ca92c30c17c53d77ad376719bad951e</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fjm901241e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901241e%26sid%3Dliteratum%253Aachs%26aulast%3DLovering%26aufirst%3DF.%26aulast%3DBikker%26aufirst%3DJ.%26aulast%3DHumblet%26aufirst%3DC.%26atitle%3DEscape%2520from%2520flatland%253A%2520increasing%2520saturation%2520as%2520an%2520approach%2520to%2520improving%2520clinical%2520success%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D6752%26epage%3D6756%26doi%3D10.1021%2Fjm901241e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Allu, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oprea, T. I.</span></span> <span> </span><span class="NLM_article-title">Rapid evaluation of synthetic and molecular complexity for in silico chemistry</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">1237</span>– <span class="NLM_lpage">1243</span>, <span class="refDoi"> DOI: 10.1021/ci0501387</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci0501387" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADC%252BD2MXmsFCqtbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2005&pages=1237-1243&author=T.+K.+Alluauthor=T.+I.+Oprea&title=Rapid+evaluation+of+synthetic+and+molecular+complexity+for+in+silico+chemistry&doi=10.1021%2Fci0501387"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid Evaluation of Synthetic and Molecular Complexity for in Silico Chemistry</span></div><div class="casAuthors">Allu, Tharun Kumar; Oprea, Tudor I.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1237-1243</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Methods that rapidly evaluate mol. complexity and synthetic feasibility are becoming increasingly important for in silico chem.  We propose a new metric based on relative at. electronegativities and bond parameters that evaluate both synthetic and mol. complexity (SMCM) starting from chem. structures.  Against mol. wt., SMCM has the lowest fraction of adjusted variance (R2=0.535) on a series of 261 048 diverse compds., when compared to the complexity metric of Baron and Chanon (R2=0.777; J. Chem. Inf. Comput. Sci. 2001, 41, 269-272) and Ruecker (R2=0.895 for log complexity values; J. Chem. Inf. Comput. Sci. 2004, 44, 378-386), resp.  These metrics are in general agreement when the metabolic synthesis of cholesterol from S-3-hydroxy-3-methyl-glutaryl CoA is monitored, indicating that SMCM can be useful in discerning increases in complexity.  Because the presence of substructure patterns can be directly incorporated into this scheme, SMCM is relatively straightforward and can be easily tailored to rapidly evaluate virtual (combinatorial) libraries and high throughput screening hits.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpsYN3I-eGvKbVg90H21EOLACvtfcHk0lj1qULbWeBZ5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXmsFCqtbc%253D&md5=67e9ee49ad8be800d3a57d98adafab97</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fci0501387&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci0501387%26sid%3Dliteratum%253Aachs%26aulast%3DAllu%26aufirst%3DT.%2BK.%26aulast%3DOprea%26aufirst%3DT.%2BI.%26atitle%3DRapid%2520evaluation%2520of%2520synthetic%2520and%2520molecular%2520complexity%2520for%2520in%2520silico%2520chemistry%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2005%26volume%3D45%26spage%3D1237%26epage%3D1243%26doi%3D10.1021%2Fci0501387" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bertz, S. H.</span></span> <span> </span><span class="NLM_article-title">The first general index of molecular complexity</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1981</span>,  <span class="NLM_volume"><i>103</i></span>,  <span class="NLM_fpage">3599</span>– <span class="NLM_lpage">3603</span>, <span class="refDoi"> DOI: 10.1021/ja00402a071</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00402a071" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADyaL3MXkt1Cjtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=1981&pages=3599-3603&author=S.+H.+Bertz&title=The+first+general+index+of+molecular+complexity&doi=10.1021%2Fja00402a071"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">The first general index of molecular complexity</span></div><div class="casAuthors">Bertz, Steven H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1981</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3599-601</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    </div><div class="casAbstract">An index of mol. complexity is introduced which is based upon the concepts of graph theory and information theory.  It is the only one extant that is completely general, being applicable to compds. contg. such features as branching, rings, multiple bonds, and heteroatoms.  It also reflects the sym. present in a mol.  All these features are important in synthetic anal., as illustrated by an example including sample calcns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXuz8eEH2Cl7Vg90H21EOLACvtfcHk0lj1qULbWeBZ5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3MXkt1Cjtbk%253D&md5=9a7602510035ab906839fd67a47148e7</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fja00402a071&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00402a071%26sid%3Dliteratum%253Aachs%26aulast%3DBertz%26aufirst%3DS.%2BH.%26atitle%3DThe%2520first%2520general%2520index%2520of%2520molecular%2520complexity%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1981%26volume%3D103%26spage%3D3599%26epage%3D3603%26doi%3D10.1021%2Fja00402a071" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barone, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chanon, M.</span></span> <span> </span><span class="NLM_article-title">A new and simple approach to chemical complexity. application to the synthesis of natural products</span>. <i>J. Chem. Inf. Comput. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">269</span>– <span class="NLM_lpage">272</span>, <span class="refDoi"> DOI: 10.1021/ci000145p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci000145p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADC%252BD3MXosVGjuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2001&pages=269-272&author=R.+Baroneauthor=M.+Chanon&title=A+new+and+simple+approach+to+chemical+complexity.+application+to+the+synthesis+of+natural+products&doi=10.1021%2Fci000145p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">A New and Simple Approach to Chemical Complexity. Application to the Synthesis of Natural Products</span></div><div class="casAuthors">Barone, Rene; Chanon, Michel</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">269-272</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A new approach to describe the complexity of chem. structures is proposed.  This index is simple and can be easily programmed.  It derives from Whitlock's index and, despite its empirical character, provides results paralleling those obtained with the math. Bertz index.  It has been applied to follow the strategic progress of some natural product syntheses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqE3_LE6fy8jbVg90H21EOLACvtfcHk0lhvBZyJbAy7aw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXosVGjuw%253D%253D&md5=92ae068a5f0102499a566e0d1c62e39c</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fci000145p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci000145p%26sid%3Dliteratum%253Aachs%26aulast%3DBarone%26aufirst%3DR.%26aulast%3DChanon%26aufirst%3DM.%26atitle%3DA%2520new%2520and%2520simple%2520approach%2520to%2520chemical%2520complexity.%2520application%2520to%2520the%2520synthesis%2520of%2520natural%2520products%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D2001%26volume%3D41%26spage%3D269%26epage%3D272%26doi%3D10.1021%2Fci000145p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schuffenhauer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Selzer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ertl, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacoby, E.</span></span> <span> </span><span class="NLM_article-title">Relationships between molecular complexity, biological activity, and structural diversity</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">525</span>– <span class="NLM_lpage">535</span>, <span class="refDoi"> DOI: 10.1021/ci0503558</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci0503558" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlShtLvF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2006&pages=525-535&author=A.+Schuffenhauerauthor=N.+Brownauthor=P.+Selzerauthor=P.+Ertlauthor=E.+Jacoby&title=Relationships+between+molecular+complexity%2C+biological+activity%2C+and+structural+diversity&doi=10.1021%2Fci0503558"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Relationships between Molecular Complexity, Biological Activity, and Structural Diversity</span></div><div class="casAuthors">Schuffenhauer, Ansgar; Brown, Nathan; Selzer, Paul; Ertl, Peter; Jacoby, Edgar</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">525-535</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Following the theor. model by Hann et al. moderately complex structures are preferable lead compds. since they lead to specific binding events involving the complete ligand mol.  To make this concept usable in practice for library design, the authors studied several complexity measures on the biol. activity of ligand mols.  The authors applied the historical IC50/EC50 summary data of 160 assays run at Novartis covering a diverse range of targets, among them kinases, proteases, GPCRs, and protein-protein interactions, and compared this to the background of "inactive" compds. which have been screened for 2 years but have never shown any activity in any primary screen.  As complexity measures the authors used the no. of structural features present in various mol. fingerprints and descriptors.  The authors found generally that with increasing activity of the ligands, their av. complexity also increased, and the authors could therefore establish a min. no. of structural features in each descriptor needed for biol. activity.  Esp. well suited in this context were the Similog keys and circular substructure fingerprints.  These are those descriptors, which also perform esp. well in the identification of bioactive compds. by similarity search, suggesting that structural features encoded in these descriptors have a high relevance for bioactivity.  Since the no. of features correlates with the no. of atoms present in the mol., also the no. of atoms serves as a reasonable complexity measure and larger mols. have, in general, higher activities.  Due to the relationship between feature counts and densities on one hand and biol. activity on the other, the size bias present in almost all similarity coeffs. becomes esp. important.  Diversity selections using these coeffs. can influence the overall complexity of the resulting set of mols., which has an impact on the biol. activity that they exhibit.  Using sphere-exclusion based diversity selection methods, such as OptiSim together with the Tanimoto dissimilarity, the av. feature count distribution of the resulting selections is shifted toward lower complexity than that of the original set, particularly when applying tight diversity constraints.  This size bias reduces the fraction of mols. in the subsets having the complexity required for a high, submicromolar activity.  None of the diversity selection methods studied, namely OptiSim, divisive K-means clustering, and self-organizing maps, yielded subsets covering the activity space of the IC50 summary data set better than subsets selected randomly.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrniA9sx4FiTbVg90H21EOLACvtfcHk0lhvBZyJbAy7aw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtlShtLvF&md5=9647fda24ecda3ec961621c775b8ebcc</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fci0503558&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci0503558%26sid%3Dliteratum%253Aachs%26aulast%3DSchuffenhauer%26aufirst%3DA.%26aulast%3DBrown%26aufirst%3DN.%26aulast%3DSelzer%26aufirst%3DP.%26aulast%3DErtl%26aufirst%3DP.%26aulast%3DJacoby%26aufirst%3DE.%26atitle%3DRelationships%2520between%2520molecular%2520complexity%252C%2520biological%2520activity%252C%2520and%2520structural%2520diversity%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2006%26volume%3D46%26spage%3D525%26epage%3D535%26doi%3D10.1021%2Fci0503558" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Feher, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, J. M.</span></span> <span> </span><span class="NLM_article-title">Property distributions: differences between drugs, natural products, and molecules from combinatorial chemistry</span>. <i>J. Chem. Inf. Comput. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">218</span>– <span class="NLM_lpage">227</span>, <span class="refDoi"> DOI: 10.1021/ci0200467</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci0200467" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADC%252BD38XptlOksLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2003&pages=218-227&author=M.+Feherauthor=J.+M.+Schmidt&title=Property+distributions%3A+differences+between+drugs%2C+natural+products%2C+and+molecules+from+combinatorial+chemistry&doi=10.1021%2Fci0200467"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Property Distributions: Differences between Drugs, Natural Products, and Molecules from Combinatorial Chemistry</span></div><div class="casAuthors">Feher, Miklos; Schmidt, Jonathan M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">218-227</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The differences between three different compd. classes, natural products, mols. from combinatorial synthesis, and drug mols., were investigated.  The major structural differences between natural and combinatorial compds. originate mainly from properties introduced to make combinatorial synthesis more efficient.  These include the no. of chiral centers, the prevalence of arom. rings, the introduction of complex ring systems, and the degree of the satn. of the mol. as well as the no. and ratios of different heteroatoms.  As drug mols. derive from both natural and synthetic sources, they cover a joint area in property space of natural and combinatorial compds.  A PCA-based scheme is presented that differentiates the three classes of compds.  It is suggested that by mimicking certain distribution properties of natural compds., combinatorial products might be made that are substantially more diverse and have greater biol. relevance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpma3fkS14voLVg90H21EOLACvtfcHk0lhvBZyJbAy7aw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XptlOksLg%253D&md5=eb86ed83699fc487dfb7f2cb55296e69</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fci0200467&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci0200467%26sid%3Dliteratum%253Aachs%26aulast%3DFeher%26aufirst%3DM.%26aulast%3DSchmidt%26aufirst%3DJ.%2BM.%26atitle%3DProperty%2520distributions%253A%2520differences%2520between%2520drugs%252C%2520natural%2520products%252C%2520and%2520molecules%2520from%2520combinatorial%2520chemistry%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D2003%26volume%3D43%26spage%3D218%26epage%3D227%26doi%3D10.1021%2Fci0200467" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Selzer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ertl, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuffenhauer, A.</span></span> <span> </span><span class="NLM_article-title">Complex molecules: do they add value?</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">310</span>– <span class="NLM_lpage">316</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2005.04.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=10.1016%2Fj.cbpa.2005.04.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=15939334" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADC%252BD2MXltVaqt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2005&pages=310-316&author=P.+Selzerauthor=H.+J.+Rothauthor=P.+Ertlauthor=A.+Schuffenhauer&title=Complex+molecules%3A+do+they+add+value%3F&doi=10.1016%2Fj.cbpa.2005.04.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Complex molecules: do they add value?</span></div><div class="casAuthors">Selzer, Paul; Roth, Hans-Joerg; Ertl, Peter; Schuffenhauer, Ansgar</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">310-316</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The concept of complexity in chem. has received much interest in the research community.  Various measures to assess mol. complexity have been published, ranging from abstr. complexity definitions to very specific application-oriented definitions.  In this article we focus on mol. complexity in relation to biol. activity.  Connectivity and feature-based structural descriptors have been evaluated with ref. to their potential as complexity measures.  Our goal was to discuss the potential of the complexity concept to support the drug discovery process, helping to design suitable lead candidates.  The studies have shown that highly active compds., on av., are more complex than inactive compds.  However, complexity must be balanced with other mol. properties because more complex mols. have a higher probability to exhibit pharmacokinetic problems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqk3nmT5dNhrbVg90H21EOLACvtfcHk0lhvBZyJbAy7aw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXltVaqt7k%253D&md5=3dfb7de5351efa8b0e9b8e3dfd98f422</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2005.04.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2005.04.001%26sid%3Dliteratum%253Aachs%26aulast%3DSelzer%26aufirst%3DP.%26aulast%3DRoth%26aufirst%3DH.%2BJ.%26aulast%3DErtl%26aufirst%3DP.%26aulast%3DSchuffenhauer%26aufirst%3DA.%26atitle%3DComplex%2520molecules%253A%2520do%2520they%2520add%2520value%253F%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2005%26volume%3D9%26spage%3D310%26epage%3D316%26doi%3D10.1016%2Fj.cbpa.2005.04.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Whitlock, H. W.</span></span> <span> </span><span class="NLM_article-title">On the structure of total synthesis of complex natural products</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">7982</span>– <span class="NLM_lpage">7989</span>, <span class="refDoi"> DOI: 10.1021/jo9814546</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo9814546" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADyaK1cXmtlanu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=1998&pages=7982-7989&author=H.+W.+Whitlock&title=On+the+structure+of+total+synthesis+of+complex+natural+products&doi=10.1021%2Fjo9814546"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">On the Structure of Total Synthesis of Complex Natural Products</span></div><div class="casAuthors">Whitlock, H. W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">7982-7989</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A chem. intuitive measure (metric) for mol. complexity is described.  It is then applied to several syntheses of complex natural products.  The authors conclude that the metric corresponds well with chem. ideas of complexity, and that different syntheses do in fact have appreciably different logical structures as measured by this.  Possible implications of this interesting observation are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxo_c0K7vuF7Vg90H21EOLACvtfcHk0lgXl_JI-CmFpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXmtlanu7Y%253D&md5=5d0bc62e676d8951e30460e7182c4557</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Fjo9814546&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo9814546%26sid%3Dliteratum%253Aachs%26aulast%3DWhitlock%26aufirst%3DH.%2BW.%26atitle%3DOn%2520the%2520structure%2520of%2520total%2520synthesis%2520of%2520complex%2520natural%2520products%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D1998%26volume%3D63%26spage%3D7982%26epage%3D7989%26doi%3D10.1021%2Fjo9814546" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rucker, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rucker, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertz, S. H.</span></span> <span> </span><span class="NLM_article-title">Organic synthesis - art or science?</span>. <i>J. Chem. Inf. Comput. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">378</span>– <span class="NLM_lpage">386</span>, <span class="refDoi"> DOI: 10.1021/ci030415e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci030415e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A280%3ADC%252BD2c7ktFajtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2004&pages=378-386&author=C.+Ruckerauthor=G.+Ruckerauthor=S.+H.+Bertz&title=Organic+synthesis+-+art+or+science%3F&doi=10.1021%2Fci030415e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Organic synthesis--art or science?</span></div><div class="casAuthors">Rucker Christoph; Rucker Gerta; Bertz Steven H</div><div class="citationInfo"><span class="NLM_cas:title">Journal of chemical information and computer sciences</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">378-86</span>
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    </div><div class="casAbstract">The LHASA rules for finding strategic bonds in polycyclic target structures are analyzed with respect to the following question: Do the strategic bonds tend to give the greatest simplification upon disconnection, as measured by recently introduced indices of molecular complexity? The answer is yes, at least for the more general rules.  This result implies that the bonds most useful for retrosynthetic disconnection can now be identified by a simple calculation rather than by application of a body of rules.  It is concluded that organic synthesis, as far as described by these rules, has a mathematical basis and consequently can be considered a science as well as an art.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTG-WUYX89vCNNTrvPZP1IpfW6udTcc2eaITnCaJGS8F7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2c7ktFajtQ%253D%253D&md5=7b5c58ae6b96398d2263fdee0d700651</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fci030415e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci030415e%26sid%3Dliteratum%253Aachs%26aulast%3DRucker%26aufirst%3DC.%26aulast%3DRucker%26aufirst%3DG.%26aulast%3DBertz%26aufirst%3DS.%2BH.%26atitle%3DOrganic%2520synthesis%2520-%2520art%2520or%2520science%253F%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D2004%26volume%3D44%26spage%3D378%26epage%3D386%26doi%3D10.1021%2Fci030415e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span>ChemDiv
Inc. Home Page. <a href="https://chemdiv.com" class="extLink">https://chemdiv.com</a> (accessed Feb 27, <span class="NLM_year">2019</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=ChemDiv%0AInc.+Home+Page.+https%3A%2F%2Fchemdiv.com+%28accessed+Feb+27%2C+2019%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Delfort, B.</span>; <span class="NLM_string-name">Le Pennec, D.</span>; <span class="NLM_string-name">Grandjean, J.</span></span> <span> </span><span class="NLM_article-title">A Method of Synthesizing a Mixture of N,N,N′,N′-Tetramethyl-1,6-hexanediamine and N,N,N′,N′-Tetramethyldiaminoethers</span>. France Patent <span class="NLM_patent">FR3014100A1</span>, <span class="NLM_year">2013</span>. 1,2-Bis(2-dimethylaminoethoxy)ethane can be obtained in two reaction steps: the reaction of commercially available triethylene glycol with ammonia to yield 1,2-bis(2-aminoethoxy)ethane, a well-known reaction, is generally carried out in the presence of hydrogen and a suitable catalyst, followed by an aminoalkylation reaction, for example, by reductive amination using formaldehyde in the presence of hydrogen and a suitable catalyst.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&author=B.+Delfort&author=D.+Le+Pennec&author=J.+Grandjean&title=A+Method+of+Synthesizing+a+Mixture+of+N%2CN%2CN%E2%80%B2%2CN%E2%80%B2-Tetramethyl-1%2C6-hexanediamine+and+N%2CN%2CN%E2%80%B2%2CN%E2%80%B2-Tetramethyldiaminoethers"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DDelfort%26aufirst%3DB.%26atitle%3DA%2520Method%2520of%2520Synthesizing%2520a%2520Mixture%2520of%2520N%252CN%252CN%25E2%2580%25B2%252CN%25E2%2580%25B2-Tetramethyl-1%252C6-hexanediamine%2520and%2520N%252CN%252CN%25E2%2580%25B2%252CN%25E2%2580%25B2-Tetramethyldiaminoethers%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, F.</span>; <span class="NLM_string-name">Deng, Z.</span>; <span class="NLM_string-name">Li, S.</span></span> <span> </span><span class="NLM_article-title">Preparation Method of <i>N</i>-<i>tert</i>-Butyloxycarbonyl-3-methylaminothiophene.</span> China Patent <span class="NLM_patent">104193722A</span>, <span class="NLM_year">2014</span>. The compound can be synthesized in three steps using 3-thienylformaldehyde as an initial raw material: (1) NaBH<sub>4</sub>, MeOH, 25 °C, 5 h; (2) HBr, H<sub>2</sub>O/D, 12 h; (3) NH<sub>3</sub>, DMF, 150 °C, 18 h.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=F.+Chen&author=Z.+Deng&author=S.+Li&title=Preparation+Method+of+N-tert-Butyloxycarbonyl-3-methylaminothiophene."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DF.%26atitle%3DPreparation%2520Method%2520of%2520N-tert-Butyloxycarbonyl-3-methylaminothiophene.%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span>This
compound was prepared in ChemDiv by analogy with the synthetic procedure
described in <span class="NLM_contrib-group"><span class="NLM_string-name">Klingsberg, E.</span></span> <span> </span><span class="NLM_article-title">Preparation of triaryl-<i>s</i>-triazoles from diaroylhydrazines</span>.  <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1958</span>,  <span class="NLM_volume">23</span>,  <span class="NLM_fpage">1086</span>– <span class="NLM_lpage">1087</span>, <span class="refDoi"> DOI: 10.1021/jo01101a629</span> , starting from commercially available <i>N</i>,<i>N</i>′-di(pyridine-2-carbonyl)hydrazide in two steps.</span><div class="citationLinks">[<a href="/doi/10.1021/jo01101a629" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADyaG1MXovFCisA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=1958&pages=1086-1087&author=E.+Klingsberg&title=Preparation+of+triaryl-s-triazoles+from+diaroylhydrazines&doi=10.1021%2Fjo01101a629"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Preparation of triaryl-s-triazoles from diaroylhydrazines</span></div><div class="casAuthors">Klingsberg, Erwin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of  Organic Chemistry</span>
        (<span class="NLM_cas:date">1958</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1086-7</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    </div><div class="casAbstract">Arylphosphazoanilides, from the amine and PCl3, reacted with diaroylhydrazones to give triaryl-s-triazoles in good yields.  Thus, to 0.06 mole Ar1NH2 in 20-30 ml. ο-C6H4Cl2 was added 1.51 g. PCl3, and warmed on the steam bath, 0.01 mole Ar2CONHNHCOPh added, the whole refluxed 3 hrs. at 190-200°, cooled and digested in H2O or dil. HCl, and recrystd.  (AcOH) to give Ar1N.CAr2:N.N:CPh.  Ar1, Ar2, m.p., % yield given: Ph, Ph, 299-300°, 95; p-MeC6H4, Ph, 301-2°, 88; ο-MeC6H4, Ph, 191-2°, 72; m-MeC6H4, Ph, 256-7°, 83; p-ClC6H4, Ph, 261.5-2.0, 88; Ph, 4-pyridyl, 268-9°, 95; 2-napthyl, Ph, 275.5-6.5°, 52.  Cyclohexylamine and (PhCONH)2 gave diphenyloxadiazole.  AcNHNHCOPh and PhNH2 gave no product.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMtzQdPt4SW7Vg90H21EOLACvtfcHk0lgXl_JI-CmFpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaG1MXovFCisA%253D%253D&md5=fe4d2f142fbc36266afe85ac6a05a1de</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fjo01101a629&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo01101a629%26sid%3Dliteratum%253Aachs%26aulast%3DKlingsberg%26aufirst%3DE.%26atitle%3DPreparation%2520of%2520triaryl-s-triazoles%2520from%2520diaroylhydrazines%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D1958%26volume%3D23%26spage%3D1086%26epage%3D1087%26doi%3D10.1021%2Fjo01101a629" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="note"><p class="first last">This compound was synthesized following a proprietary ChemDiv protocol: Commercially available 2-bromo-6-nitro-4-phenylquinazoline was subjected to Suzuki–Miyaura coupling reaction with benzene-1,4-diboronic acid followed by nitro group reduction and acylation using acetic anhydride.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kauffman, W. J.</span></span> <span> </span><span class="NLM_article-title">Observations on the synthesis and characterization of <i>N</i>,<i>N</i>′,<i>N</i>″-tris-(dimethylaminopropyl)hexahydro-s-triazine and isolable intermediates</span>. <i>J. Heterocycl. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1975</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">409</span>– <span class="NLM_lpage">411</span>, <span class="refDoi"> DOI: 10.1002/jhet.5570120244</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=10.1002%2Fjhet.5570120244" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADyaE2MXkvFOhsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=1975&pages=409-411&author=W.+J.+Kauffman&title=Observations+on+the+synthesis+and+characterization+of+N%2CN%E2%80%B2%2CN%E2%80%B3-tris-%28dimethylaminopropyl%29hexahydro-s-triazine+and+isolable+intermediates&doi=10.1002%2Fjhet.5570120244"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and characterization of N,N',N''-tris(dimethylaminopropyl)hexahydro-s-triazine and isolable intermediates</span></div><div class="casAuthors">Kauffman, W. J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Heterocyclic Chemistry</span>
        (<span class="NLM_cas:date">1975</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">409-11</span>CODEN:
                <span class="NLM_cas:coden">JHTCAD</span>;
        ISSN:<span class="NLM_cas:issn">0022-152X</span>.
    </div><div class="casAbstract">The cyclization of Me2N(CH2)3NH2 with H2CO gave I-III.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohHDTgLHZbVrVg90H21EOLACvtfcHk0ljE-34jt_NbrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE2MXkvFOhsL4%253D&md5=672d8c018c96a16f5a025c3e1f8c35c4</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1002%2Fjhet.5570120244&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjhet.5570120244%26sid%3Dliteratum%253Aachs%26aulast%3DKauffman%26aufirst%3DW.%2BJ.%26atitle%3DObservations%2520on%2520the%2520synthesis%2520and%2520characterization%2520of%2520N%252CN%25E2%2580%25B2%252CN%25E2%2580%25B3-tris-%2528dimethylaminopropyl%2529hexahydro-s-triazine%2520and%2520isolable%2520intermediates%26jtitle%3DJ.%2520Heterocycl.%2520Chem.%26date%3D1975%26volume%3D12%26spage%3D409%26epage%3D411%26doi%3D10.1002%2Fjhet.5570120244" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit30.b"><span>HHT can be prepared by the reaction
of <i>N</i>,<i>N</i>-dimethylaminopropylamine
with aqueous formaldehyde via aldimine intermediate formation in situ
followed by its rapid trimerization (one-pot synthesis, up to 90%
yield).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=HHT+can+be+prepared+by+the+reaction%0Aof+N%2CN-dimethylaminopropylamine%0Awith+aqueous+formaldehyde+via+aldimine+intermediate+formation+in+situ%0Afollowed+by+its+rapid+trimerization+%28one-pot+synthesis%2C+up+to+90%25%0Ayield%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30.b&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Balaban, A. T.</span></span> <span> </span><span class="NLM_article-title">Chemical graphs</span>. <i>Theoret. Chim. Acta.</i> <span class="NLM_year" style="font-weight: bold;">1979</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">355</span>– <span class="NLM_lpage">375</span>, <span class="refDoi"> DOI: 10.1007/BF00555695</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=10.1007%2FBF00555695" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADyaL3cXhvFOksLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1979&pages=355-375&author=A.+T.+Balaban&title=Chemical+graphs&doi=10.1007%2FBF00555695"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical graphs.  XXXIV.  Five new topological indexes for the branching of tree-like graphs</span></div><div class="casAuthors">Balaban, Alexandru T.</div><div class="citationInfo"><span class="NLM_cas:title">Theoretica Chimica Acta</span>
        (<span class="NLM_cas:date">1979</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">355-75</span>CODEN:
                <span class="NLM_cas:coden">TCHAAM</span>;
        ISSN:<span class="NLM_cas:issn">0040-5744</span>.
    </div><div class="casAbstract">In order to find the center of an acyclic connected graph (of a tree), vertexes of degree one (endpoints) are removed stepwise.  The nos. δi of vertexes thus removed at each step form a digit sequence S (pruning sequence) which reflects the branching of the tree.  The sum of squares of digits in the sequence S affords a new topol. centric index B = Σiδ2i for the branching of trees.  Comparisons with other topol. indexes are presented evidencing that B. induces an ordering of isomeric trees distinct from those induced by all other indexes devised so far, because B emphasizes equally branches of similar length.  It is shown that Rouvray's index I is equiv. to Wiener's index w, and that the Gordon-Scantlebury index N2 and Gutman et al's index M1 belong to the same family, called quadratic indexes, and induce the same ordering.  Since all topol. indexes vary both with the branching and the no. of vertexes in the free, four new indexes are devised from B. and M1 to account only (or mainly) for the branching, by normalization (imposing a lower bound equal to zero for chain-graphs, i.e. n-alkanes) or binormalization (same lower bound, and upper bound equal to one for star-graphs).  Normalized and binormalized centric (C,C') and quadratic indexes (Q,Q') are presented for the lower alkanes.  From the 5 new topol. indexes, the centric indexes (B, C, C') are limited to trees, but the quadratic indexes (Q,Q') apply to any graph.  Binormalized indexes (C',Q') express the topol. shape of the graph.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJcbJr_BNy1rVg90H21EOLACvtfcHk0ljE-34jt_NbrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3cXhvFOksLw%253D&md5=c3bbddeaa4ac25b71b2c72de89334743</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1007%2FBF00555695&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF00555695%26sid%3Dliteratum%253Aachs%26aulast%3DBalaban%26aufirst%3DA.%2BT.%26atitle%3DChemical%2520graphs%26jtitle%3DTheoret.%2520Chim.%2520Acta.%26date%3D1979%26volume%3D53%26spage%3D355%26epage%3D375%26doi%3D10.1007%2FBF00555695" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span>Clarivate
Analytics Integrity Access Page. <a href="https://integrity.thomson-pharma.com" class="extLink">https://integrity.thomson-pharma.com</a> (accessed Feb 27, <span class="NLM_year">2019</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Clarivate%0AAnalytics+Integrity+Access+Page.+https%3A%2F%2Fintegrity.thomson-pharma.com+%28accessed+Feb+27%2C+2019%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Christel, M.</span></span> <span> </span><span class="NLM_article-title">Pharm Exec’s Top 50 Companies 2017</span>. <i>Pharm. Exec.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">4</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2017&pages=4&author=M.+Christel&title=Pharm+Exec%E2%80%99s+Top+50+Companies+2017"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DChristel%26aufirst%3DM.%26atitle%3DPharm%2520Exec%25E2%2580%2599s%2520Top%252050%2520Companies%25202017%26jtitle%3DPharm.%2520Exec.%26date%3D2017%26volume%3D37%26spage%3D4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit33b"><span><a href="http://www.pharmexec.com/pharm-execs-top-50-companies-2017" class="extLink">http://www.pharmexec.com/pharm-execs-top-50-companies-2017</a> (accessed
Feb 27, 2019).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=http%3A%2F%2Fwww.pharmexec.com%2Fpharm-execs-top-50-companies-2017+%28accessed%0AFeb+27%2C+2019%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33b&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ivanenkov, Y. A.</span>; <span class="NLM_string-name">Khandarova, L. M.</span></span> <span> </span><span class="NLM_article-title">Advanced Artificial Intelligence Methods Used in the Design of Pharmaceutical Agents</span>. In  <i>Pharmaceutical Data Mining: Approaches and Applications for Drug Discovery</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balakin, K. V.</span></span>, Ed.; <span class="NLM_publisher-name">Wiley</span>: <span class="NLM_publisher-loc">Hoboken, NJ</span>, <span class="NLM_year">2010</span>; pp  <span class="NLM_fpage">457</span>– <span class="NLM_lpage">489</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2010&pages=457-489&author=Y.+A.+Ivanenkov&author=L.+M.+Khandarovaauthor=K.+V.+Balakin&title=Pharmaceutical+Data+Mining%3A+Approaches+and+Applications+for+Drug+Discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DIvanenkov%26aufirst%3DY.%2BA.%26atitle%3DAdvanced%2520Artificial%2520Intelligence%2520Methods%2520Used%2520in%2520the%2520Design%2520of%2520Pharmaceutical%2520Agents%26btitle%3DPharmaceutical%2520Data%2520Mining%253A%2520Approaches%2520and%2520Applications%2520for%2520Drug%2520Discovery%26aulast%3DBalakin%26aufirst%3DK.%2BV.%26pub%3DWiley%26date%3D2010%26spage%3D457%26epage%3D489" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pletnev, I. V.</span>; <span class="NLM_string-name">Ivanenkov, Y. A.</span>; <span class="NLM_string-name">Tarasov, A. V.</span></span> <span> </span><span class="NLM_article-title">Dimensionality Reduction Techniques for Pharmaceutical Data Mining</span>. In  <i>Pharmaceutical Data Mining: Approaches and Applications for Drug Discovery</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balakin, K. V.</span></span>, Ed.; <span class="NLM_publisher-name">Wiley</span>: <span class="NLM_publisher-loc">Hoboken, NJ</span>, <span class="NLM_year">2010</span>; pp  <span class="NLM_fpage">423</span>– <span class="NLM_lpage">455</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2010&pages=423-455&author=I.+V.+Pletnev&author=Y.+A.+Ivanenkov&author=A.+V.+Tarasovauthor=K.+V.+Balakin&title=Pharmaceutical+Data+Mining%3A+Approaches+and+Applications+for+Drug+Discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DPletnev%26aufirst%3DI.%2BV.%26atitle%3DDimensionality%2520Reduction%2520Techniques%2520for%2520Pharmaceutical%2520Data%2520Mining%26btitle%3DPharmaceutical%2520Data%2520Mining%253A%2520Approaches%2520and%2520Applications%2520for%2520Drug%2520Discovery%26aulast%3DBalakin%26aufirst%3DK.%2BV.%26pub%3DWiley%26date%3D2010%26spage%3D423%26epage%3D455" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sammon, J. W.</span></span> <span> </span><span class="NLM_article-title">A nonlinear mapping for data structure analysis</span>. <i>IEEE Trans. Comput.</i> <span class="NLM_year" style="font-weight: bold;">1969</span>,  <span class="NLM_volume"><i>C-18</i></span>,  <span class="NLM_fpage">401</span>– <span class="NLM_lpage">409</span>, <span class="refDoi"> DOI: 10.1109/T-C.1969.222678</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=10.1109%2FT-C.1969.222678" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=C-18&publication_year=1969&pages=401-409&author=J.+W.+Sammon&title=A+nonlinear+mapping+for+data+structure+analysis&doi=10.1109%2FT-C.1969.222678"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1109%2FT-C.1969.222678&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1109%252FT-C.1969.222678%26sid%3Dliteratum%253Aachs%26aulast%3DSammon%26aufirst%3DJ.%2BW.%26atitle%3DA%2520nonlinear%2520mapping%2520for%2520data%2520structure%2520analysis%26jtitle%3DIEEE%2520Trans.%2520Comput.%26date%3D1969%26volume%3DC-18%26spage%3D401%26epage%3D409%26doi%3D10.1109%2FT-C.1969.222678" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bemis, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murcko, M. A.</span></span> <span> </span><span class="NLM_article-title">The properties of known drugs. 1. Molecular frameworks</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">2887</span>– <span class="NLM_lpage">2893</span>, <span class="refDoi"> DOI: 10.1021/jm9602928</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm9602928" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADyaK28XjvVejtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1996&pages=2887-2893&author=G.+W.+Bemisauthor=M.+A.+Murcko&title=The+properties+of+known+drugs.+1.+Molecular+frameworks&doi=10.1021%2Fjm9602928"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">The Properties of Known Drugs. 1. Molecular Frameworks</span></div><div class="casAuthors">Bemis, Guy W.; Murcko, Mark A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">2887-2893</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">To better understand the common features present in drug mols., we use shape description methods to analyze a database of com. available drugs and prep. a list of common drug shapes.  A useful way of organizing this structural data is to group the atoms of each drug mol. into ring, linker, framework, and side chain atoms.  On the basis of the two-dimensional mol. structures (without regard to atom type, hybridization, and bond order), there are 1179 different frameworks among the 5120 compds. analyzed.  However, the shapes of half of the drugs in the database are described by the 32 most frequently occurring frameworks.  This suggests that the diversity of shapes in the set of known drugs is extremely low.  In our second method of anal., in which atom type, hybridization, and bond order are considered, more diversity is seen; there are 2506 different frameworks among the 5120 compds. in the database, and the most frequently occurring 42 frameworks account for only one-fourth of the drugs.  We discuss the possible interpretations of these findings and the way they may be used to guide future drug discovery research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoodEb3sfDSdrVg90H21EOLACvtfcHk0ljPIaG4VoYsZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XjvVejtro%253D&md5=5e2c4fdfea9434456a0cca83de4185b3</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fjm9602928&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm9602928%26sid%3Dliteratum%253Aachs%26aulast%3DBemis%26aufirst%3DG.%2BW.%26aulast%3DMurcko%26aufirst%3DM.%2BA.%26atitle%3DThe%2520properties%2520of%2520known%2520drugs.%25201.%2520Molecular%2520frameworks%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1996%26volume%3D39%26spage%3D2887%26epage%3D2893%26doi%3D10.1021%2Fjm9602928" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, T.</span></span> <span> </span><span class="NLM_article-title">Drug and drug candidate building block analysis</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">55</span>– <span class="NLM_lpage">67</span>, <span class="refDoi"> DOI: 10.1021/ci900398f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci900398f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFOisrbI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=55-67&author=J.+Wangauthor=T.+Hou&title=Drug+and+drug+candidate+building+block+analysis&doi=10.1021%2Fci900398f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Drug and Drug Candidate Building Block Analysis</span></div><div class="casAuthors">Wang, Junmei; Hou, Tingjun</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">55-67</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Drug likeness anal. is widely used in modern drug design.  However, most drug likeness filters, represented by Lipinski's "Rule of 5", are based on drugs' simple structural features and some physiochem. properties.  In this study, we conducted thorough structural analyses for two drug datasets.  The first dataset, ADDS, is composed of 1240 FDA-approved drugs, and the second drug dataset, EDDS, is a nonredundant collection of FDA-approved drugs and exptl. drugs in different phases of clin. trials from several drug databases (6932 entries).  For each mol., all possible fragments were enumerated using a brutal force approach.  Three kinds of building blocks, namely, the drug scaffold, ring system, and the small fragment, were identified and ranked according to the frequencies of their occurrence in drug mols.  The major finding is that most top fragments are essentially common for both drug datasets; the top 50 fragments cover 52.6% and 48.6% drugs for ADDS and EDDS, resp.  The identified building blocks were further ranked according to their relative hit rates in the drug datasets and in a screening dataset, which is a nonredundant collection of screening compds. from many resources.  In comparison with the previous reports in the field, we have identified many more high-quality building blocks.  The results obtained in this study could provide useful hints to medicinal chemists in designing drug-like compds. as well as prioritizing screening libraries to filter out those mols. lack of functional building blocks.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUmeynz19y1LVg90H21EOLACvtfcHk0ljPIaG4VoYsZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFOisrbI&md5=9981fa62c5476e8063370bb7a659a752</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fci900398f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci900398f%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DHou%26aufirst%3DT.%26atitle%3DDrug%2520and%2520drug%2520candidate%2520building%2520block%2520analysis%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D55%26epage%3D67%26doi%3D10.1021%2Fci900398f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacCoss, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawson, A. D. G.</span></span> <span> </span><span class="NLM_article-title">Combining molecular scaffolds from FDA approved drugs: application to drug discovery</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">1638</span>– <span class="NLM_lpage">1647</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01367</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01367" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVGrurrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=1638-1647&author=R.+D.+Taylorauthor=M.+MacCossauthor=A.+D.+G.+Lawson&title=Combining+molecular+scaffolds+from+FDA+approved+drugs%3A+application+to+drug+discovery&doi=10.1021%2Facs.jmedchem.6b01367"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Combining Molecular Scaffolds from FDA Approved Drugs: Application to Drug Discovery</span></div><div class="casAuthors">Taylor, Richard D.; MacCoss, Malcolm; Lawson, Alastair D. G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1638-1647</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The authors have enumerated all linear combinations of ring systems from FDA approved drugs, up to three rings in length and up to four bonds linkers to give an in-silico database of approx. 14 million mols.  This virtual library was compared with mol. databases of published and com. available compds. to assess the prevalence of drug ring combinations in modern medicinal chem. and to identify areas of under-represented, but clin. validated, chem. space.  From the 10 trillion mol. comparisons, the authors found that less than 1% of the possible combinations of drug ring systems appear in com. available libraries.  This key observation highlights significant opportunities to design new fragment-like and lead-like libraries aimed at improving success rates and reducing risk in small mol. drug discovery, as based on the previous anal. (Taylor et al. J. Med. Chem. 2014; 57: 5845-9) approx. 70% of all new drugs are made up of only ring systems that have been used in existing drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpy9yeKHBkbbrVg90H21EOLACvtfcHk0ljPIaG4VoYsZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVGrurrJ&md5=b4fea6326e337e85d39aa827a180b634</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01367&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01367%26sid%3Dliteratum%253Aachs%26aulast%3DTaylor%26aufirst%3DR.%2BD.%26aulast%3DMacCoss%26aufirst%3DM.%26aulast%3DLawson%26aufirst%3DA.%2BD.%2BG.%26atitle%3DCombining%2520molecular%2520scaffolds%2520from%2520FDA%2520approved%2520drugs%253A%2520application%2520to%2520drug%2520discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D1638%26epage%3D1647%26doi%3D10.1021%2Facs.jmedchem.6b01367" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pitt, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parry, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perry, B. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groom, C. R.</span></span> <span> </span><span class="NLM_article-title">Heteroaromatic rings of the future</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">2952</span>– <span class="NLM_lpage">2963</span>, <span class="refDoi"> DOI: 10.1021/jm801513z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm801513z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADC%252BD1MXktFemur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=2952-2963&author=W.+R.+Pittauthor=D.+M.+Parryauthor=B.+G.+Perryauthor=C.+R.+Groom&title=Heteroaromatic+rings+of+the+future&doi=10.1021%2Fjm801513z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Heteroaromatic Rings of the Future</span></div><div class="casAuthors">Pitt, William R.; Parry, David M.; Perry, Benjamin G.; Groom, Colin R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2952-2963</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Small arom. ring systems are of central importance in the development of novel synthetic protein ligands.  Here we generate a complete list of 24847 such ring systems.  We call this list and assocd. annotations VEHICLe, which stands for virtual exploratory heterocyclic library.  Searches of literature and compd. databases, using this list as substructure queries, identified only 1701 as synthesized.  Using a carefully validated machine learning approach, we were able to est. that the no. of unpublished, but synthetically tractable, VEHICLe rings could be over 3000.  However, anal. also shows that the rate of publication of novel examples to be as low as 5-10 per yr.  With this work, we aim to provide fresh stimulus to creative org. chemists by highlighting a small set of apparently simple ring systems that are predicted to be tractable but are, to the best of our knowledge, unconquered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoV3aLlF9Lb5bVg90H21EOLACvtfcHk0ljoHIM0BovmOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXktFemur8%253D&md5=12a9d335658be309339d57db02b037e6</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fjm801513z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm801513z%26sid%3Dliteratum%253Aachs%26aulast%3DPitt%26aufirst%3DW.%2BR.%26aulast%3DParry%26aufirst%3DD.%2BM.%26aulast%3DPerry%26aufirst%3DB.%2BG.%26aulast%3DGroom%26aufirst%3DC.%2BR.%26atitle%3DHeteroaromatic%2520rings%2520of%2520the%2520future%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D2952%26epage%3D2963%26doi%3D10.1021%2Fjm801513z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Visini, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arús-Pous, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Awale, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reymond, J. L.</span></span> <span> </span><span class="NLM_article-title">Virtual exploration of the ring systems chemical universe</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">2707</span>– <span class="NLM_lpage">2718</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.7b00457</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.7b00457" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1ehtbnL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2017&pages=2707-2718&author=R.+Visiniauthor=J.+Ar%C3%BAs-Pousauthor=M.+Awaleauthor=J.+L.+Reymond&title=Virtual+exploration+of+the+ring+systems+chemical+universe&doi=10.1021%2Facs.jcim.7b00457"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Virtual Exploration of the Ring Systems Chemical Universe</span></div><div class="casAuthors">Visini, Ricardo; Arus-Pous, Josep; Awale, Mahendra; Reymond, Jean-Louis</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2707-2718</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Here, we explore the chem. space of all virtually possible org. mols., focusing on ring systems, which represent the cyclic cores of org. mols. obtained by removing all acyclic bonds and converting all remaining atoms to carbon.  This approach circumvents the combinatorial explosion encountered when enumerating the mols. themselves.  We report the chem. universe database GDB4c contg. 916,130 ring systems up to four satd. or arom. rings and max. ring size of 14 atoms, and GDB4c3D contg. the corresponding 6,555,929 stereoisomers.  Almost all (98.6 %) of these ring systems are unknown and represent chiral 3D-shaped macrocycles contg. small rings and quaternary centers reminiscent of polycyclic natural products.  We envision that GDB4c can serve to select new ring systems from which to design analogs of such natural products.  The database is available for download at www.gdb.unibe.ch together with interactive visualization and search tools as a resource for mol. design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEMq0Jsdw0drVg90H21EOLACvtfcHk0ljoHIM0BovmOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1ehtbnL&md5=8a6584de6dea49661707265a519da95c</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.7b00457&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.7b00457%26sid%3Dliteratum%253Aachs%26aulast%3DVisini%26aufirst%3DR.%26aulast%3DAr%25C3%25BAs-Pous%26aufirst%3DJ.%26aulast%3DAwale%26aufirst%3DM.%26aulast%3DReymond%26aufirst%3DJ.%2BL.%26atitle%3DVirtual%2520exploration%2520of%2520the%2520ring%2520systems%2520chemical%2520universe%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2017%26volume%3D57%26spage%3D2707%26epage%3D2718%26doi%3D10.1021%2Facs.jcim.7b00457" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brown, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boström, J.</span></span> <span> </span><span class="NLM_article-title">Analysis of past and present synthetic methodologies on medicinal chemistry: where have all the new reactions gone?</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">4443</span>– <span class="NLM_lpage">4458</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01409</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01409" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVeqsrzM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=4443-4458&author=D.+G.+Brownauthor=J.+Bostr%C3%B6m&title=Analysis+of+past+and+present+synthetic+methodologies+on+medicinal+chemistry%3A+where+have+all+the+new+reactions+gone%3F&doi=10.1021%2Facs.jmedchem.5b01409"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Analysis of Past and Present Synthetic Methodologies on Medicinal Chemistry: Where Have All the New Reactions Gone?</span></div><div class="casAuthors">Brown, Dean G.; Bostrom, Jonas</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4443-4458</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  An anal. of chem. reactions used in current medicinal chem. (2014), three decades ago (1984), and in natural product total synthesis has been conducted.  The anal. revealed that of the current most frequently used synthetic reactions, none were discovered within the past 20 years and only two in the 1980s and 1990s (Suzuki-Miyaura and Buchwald-Hartwig).  This suggests an inherent high bar of impact for new synthetic reactions in drug discovery.  The most frequently used reactions were amide bond formation, Suzuki-Miyaura coupling, and SNAr reactions, most likely due to com. availability of reagents, high chemoselectivity, and a pressure on delivery.  The authors show that these practices result in overpopulation of certain types of mol. shapes to the exclusion of others using simple PMI plots.  The authors hope that these results will help catalyze improvements in integration of new synthetic methodologies as well as new library design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLWvt5DxktAbVg90H21EOLACvtfcHk0lhX4sRRWiZDNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVeqsrzM&md5=fd56ba8418f6d4e8c271f4e977ee2a93</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01409&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01409%26sid%3Dliteratum%253Aachs%26aulast%3DBrown%26aufirst%3DD.%2BG.%26aulast%3DBostr%25C3%25B6m%26aufirst%3DJ.%26atitle%3DAnalysis%2520of%2520past%2520and%2520present%2520synthetic%2520methodologies%2520on%2520medicinal%2520chemistry%253A%2520where%2520have%2520all%2520the%2520new%2520reactions%2520gone%253F%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D4443%26epage%3D4458%26doi%3D10.1021%2Facs.jmedchem.5b01409" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clemons, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bodycombe, N. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carrinski, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shamji, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koehler, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, S. L.</span></span> <span> </span><span class="NLM_article-title">Small molecules of different origins have distinct distributions of structural complexity that correlate with protein-binding profiles</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>107</i></span>,  <span class="NLM_fpage">18787</span>– <span class="NLM_lpage">18792</span>, <span class="refDoi"> DOI: 10.1073/pnas.1012741107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=10.1073%2Fpnas.1012741107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=20956335" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVCqsb%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2010&pages=18787-18792&author=P.+A.+Clemonsauthor=N.+E.+Bodycombeauthor=H.+A.+Carrinskiauthor=J.+A.+Wilsonauthor=A.+F.+Shamjiauthor=B.+K.+Wagnerauthor=A.+N.+Koehlerauthor=S.+L.+Schreiber&title=Small+molecules+of+different+origins+have+distinct+distributions+of+structural+complexity+that+correlate+with+protein-binding+profiles&doi=10.1073%2Fpnas.1012741107"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecules of different origins have distinct distributions of structural complexity that correlate with protein-binding profiles</span></div><div class="casAuthors">Clemons, Paul A.; Bodycombe, Nicole E.; Carrinski, Hyman A.; Wilson, J. Anthony; Shamji, Alykhan F.; Wagner, Bridget K.; Koehler, Angela N.; Schreiber, Stuart L.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">107</span>
        (<span class="NLM_cas:issue">44</span>),
    <span class="NLM_cas:pages">18787-18792, S18787/1-S18787/5</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Using a diverse collection of small mols. generated from a variety of sources, we measured protein-binding activities of each individual compd. against each of 100 diverse (sequence-unrelated) proteins using small-mol. microarrays.  We also analyzed structural features, including complexity, of the small mols.  We found that compds. from different sources (com., academic, natural) have different protein-binding behaviors and that these behaviors correlate with general trends in stereochem. and shape descriptors for these compd. collections.  Increasing the content of sp3-hybridized and stereogenic atoms relative to compds. from com. sources, which comprise the majority of current screening collections, improved binding selectivity and frequency.  The results suggest structural features that synthetic chemists can target when synthesizing screening collections for biol. discovery.  Because binding proteins selectively can be a key feature of high-value probes and drugs, synthesizing com- pounds having features identified in this study may result in improved performance of screening collections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBrE7Js72yZrVg90H21EOLACvtfcHk0lhX4sRRWiZDNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVCqsb%252FN&md5=198e298ca5391d7dfd91184680e3c2ec</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1012741107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1012741107%26sid%3Dliteratum%253Aachs%26aulast%3DClemons%26aufirst%3DP.%2BA.%26aulast%3DBodycombe%26aufirst%3DN.%2BE.%26aulast%3DCarrinski%26aufirst%3DH.%2BA.%26aulast%3DWilson%26aufirst%3DJ.%2BA.%26aulast%3DShamji%26aufirst%3DA.%2BF.%26aulast%3DWagner%26aufirst%3DB.%2BK.%26aulast%3DKoehler%26aufirst%3DA.%2BN.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26atitle%3DSmall%2520molecules%2520of%2520different%2520origins%2520have%2520distinct%2520distributions%2520of%2520structural%2520complexity%2520that%2520correlate%2520with%2520protein-binding%2520profiles%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2010%26volume%3D107%26spage%3D18787%26epage%3D18792%26doi%3D10.1073%2Fpnas.1012741107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kombo, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallapragada, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chewning, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mazurov, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Speake, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hauser, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toler, S.</span></span> <span> </span><span class="NLM_article-title">3D Molecular descriptors important for clinical success</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">327</span>– <span class="NLM_lpage">342</span>, <span class="refDoi"> DOI: 10.1021/ci300445e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci300445e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVCjurnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2013&pages=327-342&author=D.+C.+Komboauthor=K.+Tallapragadaauthor=R.+Jainauthor=J.+Chewningauthor=A.+A.+Mazurovauthor=J.+D.+Speakeauthor=T.+A.+Hauserauthor=S.+Toler&title=3D+Molecular+descriptors+important+for+clinical+success&doi=10.1021%2Fci300445e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">3D Molecular Descriptors Important for Clinical Success</span></div><div class="casAuthors">Kombo, David C.; Tallapragada, Kartik; Jain, Rachit; Chewning, Joseph; Mazurov, Anatoly A.; Speake, Jason D.; Hauser, Terry A.; Toler, Steve</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">327-342</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The pharmacokinetic and safety profiles of clin. drug candidates are greatly influenced by their requisite physicochem. properties.  In particular, it has been shown that 2D mol. descriptors such as fraction of Sp3 carbon atoms (Fsp3) and no. of stereo centers correlate with clin. success.  Using the proteomic off-target hit rate of nicotinic ligands, we found that shape-based 3D descriptors such as the radius of gyration and shadow indexes discriminate off-target promiscuity better than do Fsp3 and the no. of stereo centers.  We have deduced the relevant descriptor values required for a ligand to be nonpromiscuous.  Investigating the MDL Drug Data Report (MDDR) database as compds. move from the preclin. stage toward the market, we have found that these shape-based 3D descriptors predict clin. success of compds. at preclin. and phase1 stages vs compds. withdrawn from the market better than do Fsp3 and LogD.  Further, these computed 3D mol. descriptors correlate well with exptl. obsd. soly., which is among well-known physicochem. properties that drive clin. success.  We also found that about 84% of launched drugs satisfy either Shadow index or Fsp3 criteria, whereas withdrawn and discontinued compds. fail to meet the same criteria.  Our studies suggest that spherical compds. (rather than their elongated counterparts) with a minimal no. of arom. rings may exhibit a high propensity to advance from clin. trials to market.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNRhBgMppmbbVg90H21EOLACvtfcHk0liSv1JxqF3o3g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVCjurnE&md5=c750cdc150c2ba3626d98e2eea8c3144</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Fci300445e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci300445e%26sid%3Dliteratum%253Aachs%26aulast%3DKombo%26aufirst%3DD.%2BC.%26aulast%3DTallapragada%26aufirst%3DK.%26aulast%3DJain%26aufirst%3DR.%26aulast%3DChewning%26aufirst%3DJ.%26aulast%3DMazurov%26aufirst%3DA.%2BA.%26aulast%3DSpeake%26aufirst%3DJ.%2BD.%26aulast%3DHauser%26aufirst%3DT.%2BA.%26aulast%3DToler%26aufirst%3DS.%26atitle%3D3D%2520Molecular%2520descriptors%2520important%2520for%2520clinical%2520success%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2013%26volume%3D53%26spage%3D327%26epage%3D342%26doi%3D10.1021%2Fci300445e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lowe, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sayle, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tarselli, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landrum, G. A.</span></span> <span> </span><span class="NLM_article-title">Big Data from pharmaceutical patents: a computational analysis of medicinal chemists’ bread and butter</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">4385</span>– <span class="NLM_lpage">4402</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00153</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00153" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADC%252BC28XltFGgurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=4385-4402&author=N.+Schneiderauthor=D.+M.+Loweauthor=R.+A.+Sayleauthor=M.+A.+Tarselliauthor=G.+A.+Landrum&title=Big+Data+from+pharmaceutical+patents%3A+a+computational+analysis+of+medicinal+chemists%E2%80%99+bread+and+butter&doi=10.1021%2Facs.jmedchem.6b00153"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Big Data from Pharmaceutical Patents: A Computational Analysis of Medicinal Chemists' Bread and Butter</span></div><div class="casAuthors">Schneider, Nadine; Lowe, Daniel M.; Sayle, Roger A.; Tarselli, Michael A.; Landrum, Gregory A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4385-4402</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Multiple recent studies have focused on unraveling the content of the medicinal chemist's toolbox.  Here, we present an investigation of chem. reactions and mols. retrieved from U.S. patents over the past 40 years (1976-2015).  We used a sophisticated text-mining pipeline to ext. 1.15 million unique whole reaction schemes, including reaction roles and yields, from pharmaceutical patents.  The reactions were assigned to well-known reaction types such as Wittig olefination or Buchwald-Hartwig amination using an expert system.  Analyzing the evolution of reaction types over time, we observe the previously reported bias toward reaction classes like amide bond formations or Suzuki couplings.  Our study also shows a steady increase in the no. of different reaction types used in pharmaceutical patents but a trend toward lower median yield for some of the reaction classes.  Finally, we found that today's typical product mol. is larger, more hydrophobic, and more rigid than 40 years ago.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSnvOtnWAKX7Vg90H21EOLACvtfcHk0liSv1JxqF3o3g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XltFGgurs%253D&md5=16d3f6bd45b4ca1bf024fee2ef9f5744</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00153&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00153%26sid%3Dliteratum%253Aachs%26aulast%3DSchneider%26aufirst%3DN.%26aulast%3DLowe%26aufirst%3DD.%2BM.%26aulast%3DSayle%26aufirst%3DR.%2BA.%26aulast%3DTarselli%26aufirst%3DM.%2BA.%26aulast%3DLandrum%26aufirst%3DG.%2BA.%26atitle%3DBig%2520Data%2520from%2520pharmaceutical%2520patents%253A%2520a%2520computational%2520analysis%2520of%2520medicinal%2520chemists%25E2%2580%2599%2520bread%2520and%2520butter%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D4385%26epage%3D4402%26doi%3D10.1021%2Facs.jmedchem.6b00153" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Walters, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murcko, M. A.</span></span> <span> </span><span class="NLM_article-title">What do medicinal chemists actually make? A 50-Year Retrospective</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">6405</span>– <span class="NLM_lpage">6416</span>, <span class="refDoi"> DOI: 10.1021/jm200504p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm200504p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFWjsr7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=6405-6416&author=W.+P.+Waltersauthor=J.+Greenauthor=J.+R.+Weissauthor=M.+A.+Murcko&title=What+do+medicinal+chemists+actually+make%3F+A+50-Year+Retrospective&doi=10.1021%2Fjm200504p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">What Do Medicinal Chemists Actually Make? A 50-Year Retrospective</span></div><div class="casAuthors">Walters, W. Patrick; Green, Jeremy; Weiss, Jonathan R.; Murcko, Mark A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">6405-6416</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Despite the dramatically increasing costs of pharmaceutical R&D productivity, as measured by the no. of new drugs approved each year, has languished during the past decade.  Many different explanations have been proposed to account for this decline, such as increasing regulatory scrutiny, the inherent inefficiency of very large organizations, the pursuit of highly speculative targets identified from genomics, and more objective decision-making about advancing marginal compds. into latest age clin. trials.  Another factor often cited to help explain low pharmaceutical productivity is that, paradoxically, many compds. synthesized by medicinal chemists are not "druglike", at least as measured by various criteria.  The authors discuss the past, present and perspectives in the development of medicinal chem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqg5xuF4wHqKrVg90H21EOLACvtfcHk0liSv1JxqF3o3g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFWjsr7O&md5=33109f9d55967be0d34f377f433621e4</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Fjm200504p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200504p%26sid%3Dliteratum%253Aachs%26aulast%3DWalters%26aufirst%3DW.%2BP.%26aulast%3DGreen%26aufirst%3DJ.%26aulast%3DWeiss%26aufirst%3DJ.%2BR.%26aulast%3DMurcko%26aufirst%3DM.%2BA.%26atitle%3DWhat%2520do%2520medicinal%2520chemists%2520actually%2520make%253F%2520A%252050-Year%2520Retrospective%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D6405%26epage%3D6416%26doi%3D10.1021%2Fjm200504p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Doak, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Over, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giordanetto, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kihlberg, J.</span></span> <span> </span><span class="NLM_article-title">Oral druggable space beyond the Rule of 5: insights from drugs and clinical candidates</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">1115</span>– <span class="NLM_lpage">1142</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2014.08.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=10.1016%2Fj.chembiol.2014.08.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=25237858" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFyrur7K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2014&pages=1115-1142&author=B.+C.+Doakauthor=B.+Overauthor=F.+Giordanettoauthor=J.+Kihlberg&title=Oral+druggable+space+beyond+the+Rule+of+5%3A+insights+from+drugs+and+clinical+candidates&doi=10.1016%2Fj.chembiol.2014.08.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Oral Druggable Space beyond the Rule of 5: Insights from Drugs and Clinical Candidates</span></div><div class="casAuthors">Doak, Bradley Croy; Over, Bjorn; Giordanetto, Fabrizio; Kihlberg, Jan</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1115-1142</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The rule of 5 (Ro5) is a set of in silico guidelines applied to drug discovery to prioritize compds. with an increased likelihood of high oral absorption.  It has been influential in reducing attrition due to poor pharmacokinetics over the last 15 years.  However, strict reliance on the Ro5 may have resulted in lost opportunities, particularly for difficult targets.  To identify opportunities for oral drug discovery beyond the Ro5 (bRo5), we have comprehensively analyzed drugs and clin. candidates with mol. wt. (MW) > 500 Da.  We conclude that oral drugs are found far bRo5 and properties such as intramol. hydrogen bonding, macrocyclization, dosage, and formulations can be used to improve bRo5 bioavailability.  Natural products and structure-based design, often from peptidic leads, are key sources for oral bRo5 drugs.  These insights should help guide the design of oral drugs in bRo5 space, which is of particular interest for difficult targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvYqpeA0dh6rVg90H21EOLACvtfcHk0liOKLQoHgs24g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFyrur7K&md5=b8a88c793572ad48de3809807a4699d8</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2014.08.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2014.08.013%26sid%3Dliteratum%253Aachs%26aulast%3DDoak%26aufirst%3DB.%2BC.%26aulast%3DOver%26aufirst%3DB.%26aulast%3DGiordanetto%26aufirst%3DF.%26aulast%3DKihlberg%26aufirst%3DJ.%26atitle%3DOral%2520druggable%2520space%2520beyond%2520the%2520Rule%2520of%25205%253A%2520insights%2520from%2520drugs%2520and%2520clinical%2520candidates%26jtitle%3DChem.%2520Biol.%26date%3D2014%26volume%3D21%26spage%3D1115%26epage%3D1142%26doi%3D10.1016%2Fj.chembiol.2014.08.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shultz, M. D.</span></span> <span> </span><span class="NLM_article-title">Two decades under the influence of the Rule of five and the changing properties of approved oral drugs</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">1701</span>– <span class="NLM_lpage">1714</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00686</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00686" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslait73K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=1701-1714&issue=4&author=M.+D.+Shultz&title=Two+decades+under+the+influence+of+the+Rule+of+five+and+the+changing+properties+of+approved+oral+drugs&doi=10.1021%2Facs.jmedchem.8b00686"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Two Decades under the Influence of the Rule of Five and the Changing Properties of Approved Oral Drugs</span></div><div class="casAuthors">Shultz, Michael D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1701-1714</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Two decades have passed since the rule of five ushered in the concept of "drug-like" properties.  Attempts to quantify, correlate, and categorize mols. based on Ro5 parameters evolved into the introduction of efficiency metrics with far reaching consequences in decision making by industry leaders and scientists seeking to discover new medicines.  Examn. of oral drug parameters approved before and after the original Ro5 anal. demonstrates that some parameters such as clogP and HBD remained const. while the cutoffs for parameters such as mol. wt. and HBA have increased substantially over the past 20 years.  The time dependent increase in the mol. wt. of oral drugs during the past 20 years provides compelling evidence to disprove the hypothesis that mol. wt. is a "drug-like" property.  This anal. does not validate parameters that have not changed as being "drug-like" but instead calls into question the entire hypothesis that "drug-like" properties exist.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAnoxpQsv2kbVg90H21EOLACvtfcHk0liOKLQoHgs24g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslait73K&md5=d3c8b1107133dc0b7b277d22fa029597</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00686&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00686%26sid%3Dliteratum%253Aachs%26aulast%3DShultz%26aufirst%3DM.%2BD.%26atitle%3DTwo%2520decades%2520under%2520the%2520influence%2520of%2520the%2520Rule%2520of%2520five%2520and%2520the%2520changing%2520properties%2520of%2520approved%2520oral%2520drugs%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26issue%3D4%26spage%3D1701%26epage%3D1714%26doi%3D10.1021%2Facs.jmedchem.8b00686" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ran, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gestwicki, J. E.</span></span> <span> </span><span class="NLM_article-title">Inhibitors of protein-protein interactions (PPIs): an analysis of scaffold choices and buried surface area</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">75</span>– <span class="NLM_lpage">86</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2018.06.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=10.1016%2Fj.cbpa.2018.06.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=29908451" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFCmsbnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2018&pages=75-86&author=X.+Ranauthor=J.+E.+Gestwicki&title=Inhibitors+of+protein-protein+interactions+%28PPIs%29%3A+an+analysis+of+scaffold+choices+and+buried+surface+area&doi=10.1016%2Fj.cbpa.2018.06.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitors of protein-protein interactions (PPIs): an analysis of scaffold choices and buried surface area</span></div><div class="casAuthors">Ran, Xu; Gestwicki, Jason E.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">75-86</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Protein-protein interactions (PPI) were once considered 'undruggable', but clin. successes, driven by advanced methods in drug discovery, have challenged that notion.  Here, we review the last three years of literature on PPI inhibitors to understand what is working and why.  From the 66 recently reported PPI inhibitors, we found that the av. mol. wt. was significantly greater than 500 Da, but that this trend was driven, in large part, by the contribution of peptide-based compds.  Despite differences in av. mol. wt., we found that compds. based on small mols. or peptides were almost equally likely to be potent inhibitors (KD < 1 μM).  Finally, we found PPIs with buried surface area (BSA) less than 2000 Å2 were more likely to be inhibited by small mols., while PPIs with larger BSA values were typically inhibited by peptides.  PPIs with BSA values over 4000 Å2 seemed to create a particular challenge, esp. for orthosteric small mols.  Thus, it seems important to choose the inhibitor scaffold based on the properties of the target interaction.  Moreover, this survey suggests a (more nuanced) conclusion to the question of whether PPIs are good drug targets; namely, that some PPIs are readily 'druggable' given the right choice of scaffold, while others still seem to deserve the 'undruggable' moniker.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpoS9hriXFuS7Vg90H21EOLACvtfcHk0liFlfqN8Veq3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFCmsbnE&md5=037a06e18fc78112243cd68ce6752599</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2018.06.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2018.06.004%26sid%3Dliteratum%253Aachs%26aulast%3DRan%26aufirst%3DX.%26aulast%3DGestwicki%26aufirst%3DJ.%2BE.%26atitle%3DInhibitors%2520of%2520protein-protein%2520interactions%2520%2528PPIs%2529%253A%2520an%2520analysis%2520of%2520scaffold%2520choices%2520and%2520buried%2520surface%2520area%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2018%26volume%3D44%26spage%3D75%26epage%3D86%26doi%3D10.1016%2Fj.cbpa.2018.06.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cossar, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCluskey, A.</span></span> <span> </span><span class="NLM_article-title">Protein protein interactions as antibiotic targets: a medicinal chemistry perspective</span>. <i>Med. Res. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">26</span>, <span class="refDoi"> DOI: 10.1002/med.21519</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=10.1002%2Fmed.21519" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&pages=1-26&author=P.+J.+Cossarauthor=P.+J.+Lewisauthor=A.+McCluskey&title=Protein+protein+interactions+as+antibiotic+targets%3A+a+medicinal+chemistry+perspective&doi=10.1002%2Fmed.21519"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1002%2Fmed.21519&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21519%26sid%3Dliteratum%253Aachs%26aulast%3DCossar%26aufirst%3DP.%2BJ.%26aulast%3DLewis%26aufirst%3DP.%2BJ.%26aulast%3DMcCluskey%26aufirst%3DA.%26atitle%3DProtein%2520protein%2520interactions%2520as%2520antibiotic%2520targets%253A%2520a%2520medicinal%2520chemistry%2520perspective%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2018%26spage%3D1%26epage%3D26%26doi%3D10.1002%2Fmed.21519" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Macalino, S. J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Basith, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clavio, N. A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, S.</span></span> <span> </span><span class="NLM_article-title">Evolution of in silico strategies for protein-protein interaction drug discovery</span>. <i>Molecules</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1963</span>, <span class="refDoi"> DOI: 10.3390/molecules23081963</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=10.3390%2Fmolecules23081963" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvFGgtbzJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2018&pages=1963&author=S.+J.+Y.+Macalinoauthor=S.+Basithauthor=N.+A.+B.+Clavioauthor=H.+Changauthor=S.+Kangauthor=S.+Choi&title=Evolution+of+in+silico+strategies+for+protein-protein+interaction+drug+discovery&doi=10.3390%2Fmolecules23081963"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Evolution of in silico strategies for protein-protein interaction drug discovery</span></div><div class="casAuthors">Macalino, Stephani Joy Y.; Basith, Shaherin; Clavio, Nina Abigail B.; Chang, Hyerim; Kang, Soosung; Choi, Sun</div><div class="citationInfo"><span class="NLM_cas:title">Molecules</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1963/1-1963/44</span>CODEN:
                <span class="NLM_cas:coden">MOLEFW</span>;
        ISSN:<span class="NLM_cas:issn">1420-3049</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">The advent of advanced mol. modeling software, big data analytics, and high-speed processing units has led to the exponential evolution of modern drug discovery and better insights into complex biol. processes and disease networks.  This has progressively steered current research interests to understanding protein-protein interaction (PPI) systems that are related to a no. of relevant diseases, such as cancer, neurol. illnesses, metabolic disorders, etc.  However, targeting PPIs are challenging due to their "undruggable" binding interfaces.  In this review, we focus on the current obstacles that impede PPI drug discovery, and how recent discoveries and advances in in silico approaches can alleviate these barriers to expedite the search for potential leads, as shown in several exemplary studies.  We will also discuss about currently available information on PPI compds. and systems, along with their usefulness in mol. modeling.  Finally, we conclude by presenting the limits of in silico application in drug discovery and offer a perspective in the field of computer-aided PPI drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqu3s0Z07xfAbVg90H21EOLACvtfcHk0liFlfqN8Veq3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvFGgtbzJ&md5=f700f8a40a79c10f1e796cab26295849</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.3390%2Fmolecules23081963&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules23081963%26sid%3Dliteratum%253Aachs%26aulast%3DMacalino%26aufirst%3DS.%2BJ.%2BY.%26aulast%3DBasith%26aufirst%3DS.%26aulast%3DClavio%26aufirst%3DN.%2BA.%2BB.%26aulast%3DChang%26aufirst%3DH.%26aulast%3DKang%26aufirst%3DS.%26aulast%3DChoi%26aufirst%3DS.%26atitle%3DEvolution%2520of%2520in%2520silico%2520strategies%2520for%2520protein-protein%2520interaction%2520drug%2520discovery%26jtitle%3DMolecules%26date%3D2018%26volume%3D23%26spage%3D1963%26doi%3D10.3390%2Fmolecules23081963" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scott, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayly, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abell, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skidmore, J.</span></span> <span> </span><span class="NLM_article-title">Small molecules, big targets: drug discovery faces the protein- protein interaction challenge</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">533</span>– <span class="NLM_lpage">550</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.29</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=10.1038%2Fnrd.2016.29" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=27050677" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADC%252BC28XlsFOgur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=533-550&author=D.+E.+Scottauthor=A.+R.+Baylyauthor=C.+Abellauthor=J.+Skidmore&title=Small+molecules%2C+big+targets%3A+drug+discovery+faces+the+protein-+protein+interaction+challenge&doi=10.1038%2Fnrd.2016.29"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecules, big targets: drug discovery faces the protein-protein interaction challenge</span></div><div class="casAuthors">Scott, Duncan E.; Bayly, Andrew R.; Abell, Chris; Skidmore, John</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">533-550</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein-protein interactions (PPIs) are of pivotal importance in the regulation of biol. systems and are consequently implicated in the development of disease states.  Recent work has begun to show that, with the right tools, certain classes of PPI can yield to the efforts of medicinal chemists to develop inhibitors, and the first PPI inhibitors have reached clin. development.  In this Review, we describe the research leading to these breakthroughs and highlight the existence of groups of structurally related PPIs within the PPI target class.  For each of these groups, we use examples of successful discovery efforts to illustrate the research strategies that have proved most useful.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoK-aGzHa9mP7Vg90H21EOLACvtfcHk0ljOROD6WJsRKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XlsFOgur4%253D&md5=9f0953567ae9338b8b36aa78b8160655</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.29&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.29%26sid%3Dliteratum%253Aachs%26aulast%3DScott%26aufirst%3DD.%2BE.%26aulast%3DBayly%26aufirst%3DA.%2BR.%26aulast%3DAbell%26aufirst%3DC.%26aulast%3DSkidmore%26aufirst%3DJ.%26atitle%3DSmall%2520molecules%252C%2520big%2520targets%253A%2520drug%2520discovery%2520faces%2520the%2520protein-%2520protein%2520interaction%2520challenge%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2016%26volume%3D15%26spage%3D533%26epage%3D550%26doi%3D10.1038%2Fnrd.2016.29" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Soga, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shirai, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobori, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirayama, N.</span></span> <span> </span><span class="NLM_article-title">Use of amino acid composition to predict ligand-binding sites</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">400</span>– <span class="NLM_lpage">406</span>, <span class="refDoi"> DOI: 10.1021/ci6002202</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci6002202" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADC%252BD2sXns1OmsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2007&pages=400-406&author=S.+Sogaauthor=H.+Shiraiauthor=M.+Koboriauthor=N.+Hirayama&title=Use+of+amino+acid+composition+to+predict+ligand-binding+sites&doi=10.1021%2Fci6002202"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Use of Amino Acid Composition to Predict Ligand-Binding Sites</span></div><div class="casAuthors">Soga, Shinji; Shirai, Hiroki; Kobori, Masato; Hirayama, Noriaki</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">400-406</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel method for predicting the binding sites for druglike compds. on the surface of proteins was developed on the basis of the specific amino acid compn. obsd. at the ligand-binding sites of ligand-protein complexes detd. by x-ray anal.  A profile representing the preference of each of the 20 std. amino acids at the binding sites of druglike mols. was obtained for a small set of high-quality complex structures.  An index termed propensity for ligand binding (PLB) was created from these profiles.  The PLB index was used to predict the propensity of binding for 804 ligands at all potential binding sites on the proteins whose structures were detd. by x-ray anal.  If the sites with the first 2 highest PLB indexes are taken into consideration, the successfully predicted sites reached a high percentage of 86.  The PLB prediction is relatively simple, but the validation study showed that it is both fast and accurate to detect ligand-binding sites, esp. the binding sites of druglike mols.  Therefore, the PLB index can be used to predict the ligand-binding sites of uncharacterized protein structures and also to identify novel drug-binding sites of known drug targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptxKvbII_oZbVg90H21EOLACvtfcHk0ljOROD6WJsRKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXns1OmsQ%253D%253D&md5=53bddb67710c84f45c99173b0d08d9ce</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Fci6002202&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci6002202%26sid%3Dliteratum%253Aachs%26aulast%3DSoga%26aufirst%3DS.%26aulast%3DShirai%26aufirst%3DH.%26aulast%3DKobori%26aufirst%3DM.%26aulast%3DHirayama%26aufirst%3DN.%26atitle%3DUse%2520of%2520amino%2520acid%2520composition%2520to%2520predict%2520ligand-binding%2520sites%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2007%26volume%3D47%26spage%3D400%26epage%3D406%26doi%3D10.1021%2Fci6002202" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pushpakom, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iorio, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eyers, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Escott, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hopper, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doig, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guilliams, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Latimer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNamee, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norris, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanseau, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavalla, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pirmohamed, M.</span></span> <span> </span><span class="NLM_article-title">Drug repurposing: progress, challenges and recommendations</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">41</span>– <span class="NLM_lpage">58</span>, <span class="refDoi"> DOI: 10.1038/nrd.2018.168</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=10.1038%2Fnrd.2018.168" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=30310233" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvV2gtr7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=41-58&author=S.+Pushpakomauthor=F.+Iorioauthor=P.+A.+Eyersauthor=K.+J.+Escottauthor=S.+Hopperauthor=A.+Wellsauthor=A.+Doigauthor=T.+Guilliamsauthor=J.+Latimerauthor=C.+McNameeauthor=A.+Norrisauthor=P.+Sanseauauthor=D.+Cavallaauthor=M.+Pirmohamed&title=Drug+repurposing%3A+progress%2C+challenges+and+recommendations&doi=10.1038%2Fnrd.2018.168"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Drug repurposing: progress, challenges and recommendations</span></div><div class="casAuthors">Pushpakom, Sudeep; Iorio, Francesco; Eyers, Patrick A.; Escott, K. Jane; Hopper, Shirley; Wells, Andrew; Doig, Andrew; Guilliams, Tim; Latimer, Joanna; McNamee, Christine; Norris, Alan; Sanseau, Philippe; Cavalla, David; Pirmohamed, Munir</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">41-58</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  Given the high attrition rates, substantial costs and slow pace of new drug discovery and development, repurposing of 'old' drugs to treat both common and rare diseases is increasingly becoming an attractive proposition because it involves the use of de-risked compds., with potentially lower overall development costs and shorter development timelines.  Various data-driven and exptl. approaches have been suggested for the identification of repurposable drug candidates; however, there are also major technol. and regulatory challenges that need to be addressed.  In this Review, we present approaches used for drug repurposing (also known as drug repositioning), discuss the challenges faced by the repurposing community and recommend innovative ways by which these challenges could be addressed to help realize the full potential of drug repurposing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUl_N5c2iDerVg90H21EOLACvtfcHk0lgzjOV-K3F64g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvV2gtr7I&md5=6bcc8f26828414de02564494fb4d3296</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2018.168&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2018.168%26sid%3Dliteratum%253Aachs%26aulast%3DPushpakom%26aufirst%3DS.%26aulast%3DIorio%26aufirst%3DF.%26aulast%3DEyers%26aufirst%3DP.%2BA.%26aulast%3DEscott%26aufirst%3DK.%2BJ.%26aulast%3DHopper%26aufirst%3DS.%26aulast%3DWells%26aufirst%3DA.%26aulast%3DDoig%26aufirst%3DA.%26aulast%3DGuilliams%26aufirst%3DT.%26aulast%3DLatimer%26aufirst%3DJ.%26aulast%3DMcNamee%26aufirst%3DC.%26aulast%3DNorris%26aufirst%3DA.%26aulast%3DSanseau%26aufirst%3DP.%26aulast%3DCavalla%26aufirst%3DD.%26aulast%3DPirmohamed%26aufirst%3DM.%26atitle%3DDrug%2520repurposing%253A%2520progress%252C%2520challenges%2520and%2520recommendations%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2019%26volume%3D18%26spage%3D41%26epage%3D58%26doi%3D10.1038%2Fnrd.2018.168" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ashburn, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thor, K. B.</span></span> <span> </span><span class="NLM_article-title">Drug repositioning: identifying and developing new uses for existing drugs</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">673</span>– <span class="NLM_lpage">683</span>, <span class="refDoi"> DOI: 10.1038/nrd1468</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=10.1038%2Fnrd1468" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=15286734" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmtVOhtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2004&pages=673-683&author=T.+T.+Ashburnauthor=K.+B.+Thor&title=Drug+repositioning%3A+identifying+and+developing+new+uses+for+existing+drugs&doi=10.1038%2Fnrd1468"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Drug repositioning: identifying and developing new uses for existing drugs</span></div><div class="casAuthors">Ashburn, Ted T.; Thor, Karl B.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">673-683</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Biopharmaceutical companies attempting to increase productivity through novel discovery technologies have fallen short of achieving the desired results.  Repositioning existing drugs for new indications could deliver the productivity increases that the industry needs while shifting the locus of prodn. to biotechnol. companies.  More and more companies are scanning the existing pharmacopeia for repositioning candidates, and the no. of repositioning success stories is increasing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLkWl3DnePhrVg90H21EOLACvtfcHk0lgzjOV-K3F64g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmtVOhtL8%253D&md5=0145c02423ab81ace6e3b3d92585cccc</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1038%2Fnrd1468&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd1468%26sid%3Dliteratum%253Aachs%26aulast%3DAshburn%26aufirst%3DT.%2BT.%26aulast%3DThor%26aufirst%3DK.%2BB.%26atitle%3DDrug%2520repositioning%253A%2520identifying%2520and%2520developing%2520new%2520uses%2520for%2520existing%2520drugs%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2004%26volume%3D3%26spage%3D673%26epage%3D683%26doi%3D10.1038%2Fnrd1468" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span>Enamine
Ltd. Home Page. <a href="https://enamine.net" class="extLink">https://enamine.net</a> (accessed Feb 27, <span class="NLM_year">2019</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Enamine%0ALtd.+Home+Page.+https%3A%2F%2Fenamine.net+%28accessed+Feb+27%2C+2019%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span>InterBioscreen
Ltd. Home Page. <a href="https://www.ibscreen.com" class="extLink">https://www.ibscreen.com</a> (accessed Feb 27, <span class="NLM_year">2019</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=InterBioscreen%0ALtd.+Home+Page.+https%3A%2F%2Fwww.ibscreen.com+%28accessed+Feb+27%2C+2019%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kessel, M.</span></span> <span> </span><span class="NLM_article-title">The problems with today’s pharmaceutical business—an outsider’s view</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">27</span>– <span class="NLM_lpage">33</span>, <span class="refDoi"> DOI: 10.1038/nbt.1748</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=10.1038%2Fnbt.1748" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=21221096" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjsFWhuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2011&pages=27-33&author=M.+Kessel&title=The+problems+with+today%E2%80%99s+pharmaceutical+business%E2%80%94an+outsider%E2%80%99s+view&doi=10.1038%2Fnbt.1748"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">The problems with today's pharmaceutical business - an outsider's view</span></div><div class="casAuthors">Kessel, Mark</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">27-33</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The pharmaceutical industry must devote greater resources, investment and effort to address its anemic drug pipeline in the long term, rather than focusing on its bottom line in the near term.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-UuhFn-1nFbVg90H21EOLACvtfcHk0lgy1jDa-DGp-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXjsFWhuw%253D%253D&md5=cbb7728a68cfffea1ea189164239fefd</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1038%2Fnbt.1748&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.1748%26sid%3Dliteratum%253Aachs%26aulast%3DKessel%26aufirst%3DM.%26atitle%3DThe%2520problems%2520with%2520today%25E2%2580%2599s%2520pharmaceutical%2520business%25E2%2580%2594an%2520outsider%25E2%2580%2599s%2520view%26jtitle%3DNat.%2520Biotechnol.%26date%3D2011%26volume%3D29%26spage%3D27%26epage%3D33%26doi%3D10.1038%2Fnbt.1748" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span>NPS,
National Pharmaceutical Services Home Page. <a href="https://www.pti-nps.com/nps/" class="extLink">https://www.pti-nps.com/nps/</a> (accessed Feb 27, <span class="NLM_year">2019</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=NPS%2C%0ANational+Pharmaceutical+Services+Home+Page.+https%3A%2F%2Fwww.pti-nps.com%2Fnps%2F+%28accessed+Feb+27%2C+2019%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mullin, R.</span></span> <span> </span><span class="NLM_article-title">More job cuts at Merck</span>. <i>Chem. Eng. News</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>89</i></span>,  <span class="NLM_fpage">10</span>, <span class="refDoi"> DOI: 10.1021/cen-v089n032.p010</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cen-v089n034.p010a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=89&publication_year=2011&pages=10&author=R.+Mullin&title=More+job+cuts+at+Merck&doi=10.1021%2Fcen-v089n032.p010"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1021%2Fcen-v089n034.p010a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcen-v089n034.p010a%26sid%3Dliteratum%253Aachs%26aulast%3DMullin%26aufirst%3DR.%26atitle%3DMore%2520job%2520cuts%2520at%2520Merck%26jtitle%3DChem.%2520Eng.%2520News%26date%3D2011%26volume%3D89%26spage%3D10%26doi%3D10.1021%2Fcen-v089n032.p010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jarvis, L. M.</span></span> <span> </span><span class="NLM_article-title">Pfizer reveals more R&D cuts</span>. <i>Chem. Eng. News</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>89</i></span>,  <span class="NLM_fpage">5</span>, <span class="refDoi"> DOI: 10.1021/cen-v089n006.p005</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cen-v089n006.p005" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=89&publication_year=2011&pages=5&author=L.+M.+Jarvis&title=Pfizer+reveals+more+R%26D+cuts&doi=10.1021%2Fcen-v089n006.p005"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1021%2Fcen-v089n006.p005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcen-v089n006.p005%26sid%3Dliteratum%253Aachs%26aulast%3DJarvis%26aufirst%3DL.%2BM.%26atitle%3DPfizer%2520reveals%2520more%2520R%2526D%2520cuts%26jtitle%3DChem.%2520Eng.%2520News%26date%3D2011%26volume%3D89%26spage%3D5%26doi%3D10.1021%2Fcen-v089n006.p005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tsukamoto, T.</span></span> <span> </span><span class="NLM_article-title">Tough times for medicinal chemists: are we to blame?</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">369</span>– <span class="NLM_lpage">370</span>, <span class="refDoi"> DOI: 10.1021/ml400074k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml400074k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtlWgtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=369-370&author=T.+Tsukamoto&title=Tough+times+for+medicinal+chemists%3A+are+we+to+blame%3F&doi=10.1021%2Fml400074k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Tough Times for Medicinal Chemists: Are We to Blame?</span></div><div class="casAuthors">Tsukamoto, Takashi</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">369-370</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  In the United States, medicinal chemists continue to face job insecurity and high rates of unemployment.  The situation is unlikely to improve in the near future.  Is there a light at the end of the tunnel.  Is there anything we can do to revitalize our community.  The answer may be right in front of us.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-HjhOmBhZK7Vg90H21EOLACvtfcHk0lgy1jDa-DGp-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtlWgtL4%253D&md5=383e3292fc5110fcec0bd1b104f692ca</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1021%2Fml400074k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml400074k%26sid%3Dliteratum%253Aachs%26aulast%3DTsukamoto%26aufirst%3DT.%26atitle%3DTough%2520times%2520for%2520medicinal%2520chemists%253A%2520are%2520we%2520to%2520blame%253F%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2013%26volume%3D4%26spage%3D369%26epage%3D370%26doi%3D10.1021%2Fml400074k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rafferty, M. F.</span></span> <span> </span><span class="NLM_article-title">No denying it: medicinal chemistry training is in big trouble</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">10859</span>– <span class="NLM_lpage">10864</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00741</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00741" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFKmsLjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=10859-10864&author=M.+F.+Rafferty&title=No+denying+it%3A+medicinal+chemistry+training+is+in+big+trouble&doi=10.1021%2Facs.jmedchem.6b00741"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">No Denying It: Medicinal Chemistry Training Is in Big Trouble</span></div><div class="casAuthors">Rafferty, Michael F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">10859-10864</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">There has been little consensus between the pharmaceutical industry and academic communities concerning the best approach to train medicinal chemists for drug discovery.  For decades the pharmaceutical industry has shown preference for synthetic org. graduates over candidates with degrees from medicinal chem. programs on the assumption that medicinal chem. expertise will be acquired on the job.  However, ongoing changes to pharmaceutical drug discovery organizations and practices threaten to undermine this training model.  There is a compelling argument to be made for establishment of a strong industry-academic partnership to train new candidates with sophisticated knowledge of contemporary drug design concepts and techniques to ensure that the future needs of both industry and academic drug discovery research can be served.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgSt0oxSWvabVg90H21EOLACvtfcHk0lgs3sg5YJdzrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFKmsLjO&md5=7c5583586b8d6939bc2a52cb4a033455</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00741&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00741%26sid%3Dliteratum%253Aachs%26aulast%3DRafferty%26aufirst%3DM.%2BF.%26atitle%3DNo%2520denying%2520it%253A%2520medicinal%2520chemistry%2520training%2520is%2520in%2520big%2520trouble%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D10859%26epage%3D10864%26doi%3D10.1021%2Facs.jmedchem.6b00741" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hoffmann, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bishop, C.</span></span> <span> </span><span class="NLM_article-title">The future of discovery chemistry: quo vadis? Academic to industrial - the maturation of medicinal chemistry to chemical biology</span>. <i>Drug Discovery Today.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">260</span>– <span class="NLM_lpage">264</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2010.02.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=10.1016%2Fj.drudis.2010.02.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=20149893" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A280%3ADC%252BC3c3jt1Ggsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2010&pages=260-264&author=T.+Hoffmannauthor=C.+Bishop&title=The+future+of+discovery+chemistry%3A+quo+vadis%3F+Academic+to+industrial+-+the+maturation+of+medicinal+chemistry+to+chemical+biology&doi=10.1016%2Fj.drudis.2010.02.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">The future of discovery chemistry: quo vadis? Academic to industrial--the maturation of medicinal chemistry to chemical biology</span></div><div class="casAuthors">Hoffmann Torsten; Bishop Cheryl</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">7-8</span>),
    <span class="NLM_cas:pages">260-4</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">At Roche, we set out to think about the future role of medicinal chemistry in drug discovery in a project involving both Roche internal stakeholders and external experts in drug discovery chemistry.  To derive a coherent strategy, selected scientists were asked to take extreme positions and to derive two orthogonal strategic options: chemistry as the traditional mainstream science and chemistry as the central entrepreneurial science.  We believe today's role of medicinal chemistry in industry has remained too narrow.  To provide the innovation that industry requires, medicinal chemistry must play its part and diversify at pace with our increasing understanding of chemical biology and network pharmacology.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSk2WZdrxq9mEH3Rlp_k91zfW6udTcc2ebrQA4rdrH5pLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3c3jt1Ggsg%253D%253D&md5=528caf07919ff2f5c6544e166fa5cec0</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2010.02.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2010.02.002%26sid%3Dliteratum%253Aachs%26aulast%3DHoffmann%26aufirst%3DT.%26aulast%3DBishop%26aufirst%3DC.%26atitle%3DThe%2520future%2520of%2520discovery%2520chemistry%253A%2520quo%2520vadis%253F%2520Academic%2520to%2520industrial%2520-%2520the%2520maturation%2520of%2520medicinal%2520chemistry%2520to%2520chemical%2520biology%26jtitle%3DDrug%2520Discovery%2520Today.%26date%3D2010%26volume%3D15%26spage%3D260%26epage%3D264%26doi%3D10.1016%2Fj.drudis.2010.02.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, I. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macdonald, S. J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Procopiou, P. A.</span></span> <span> </span><span class="NLM_article-title">Medicinal chemistry in drug discovery in big pharma: past, present and future</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">219</span>– <span class="NLM_lpage">234</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2017.10.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=10.1016%2Fj.drudis.2017.10.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=29031621" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;key=1%3ACAS%3A280%3ADC%252BC1M7gs1GltQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2018&pages=219-234&author=I.+B.+Campbellauthor=S.+J.+F.+Macdonaldauthor=P.+A.+Procopiou&title=Medicinal+chemistry+in+drug+discovery+in+big+pharma%3A+past%2C+present+and+future&doi=10.1016%2Fj.drudis.2017.10.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Medicinal chemistry in drug discovery in big pharma: past, present and future</span></div><div class="casAuthors">Campbell Ian B; Procopiou Panayiotis A; Macdonald Simon J F</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">219-234</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The changes in synthetic and medicinal chemistry and related drug discovery science as practiced in big pharma over the past few decades are described.  These have been predominantly driven by wider changes in society namely the computer, internet and globalisation.  Thoughts about the future of medicinal chemistry are also discussed including sharing the risks and costs of drug discovery and the future of outsourcing.  The continuing impact of access to substantial computing power and big data, the use of algorithms in data analysis and drug design are also presented.  The next generation of medicinal chemists will communicate in ways that reflect social media and the results of constantly being connected to each other and data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS2kUsYSYxxc_IW7hznt0zpfW6udTcc2ebrQA4rdrH5pLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M7gs1GltQ%253D%253D&md5=6e711ae6f4fcb7fd848c39c876bf3bfa</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2017.10.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2017.10.007%26sid%3Dliteratum%253Aachs%26aulast%3DCampbell%26aufirst%3DI.%2BB.%26aulast%3DMacdonald%26aufirst%3DS.%2BJ.%2BF.%26aulast%3DProcopiou%26aufirst%3DP.%2BA.%26atitle%3DMedicinal%2520chemistry%2520in%2520drug%2520discovery%2520in%2520big%2520pharma%253A%2520past%252C%2520present%2520and%2520future%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2018%26volume%3D23%26spage%3D219%26epage%3D234%26doi%3D10.1016%2Fj.drudis.2017.10.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i25"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00004">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_86030"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.9b00004">10.1021/acs.jmedchem.9b00004</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Tables of statistical data for the calculated descriptors and a list of top 23 pharmaceutical companies (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00004/suppl_file/jm9b00004_si_001.pdf">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00004/suppl_file/jm9b00004_si_001.pdf">jm9b00004_si_001.pdf (240.13 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b00004&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b00004%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2019.62.issue-22" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b00004" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                10MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-b425991822a79ada59b3.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-fb88834b824b41b135ad.js"></script>

<script type="text/javascript" src="/wro/ke1q~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"66797f258e332374","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.6.0","si":10}'></script>
</body>
</html>
