
<html lang="en"     class="pb-page"  data-request-id="10dd6a5c-92b4-487f-b235-cc6740c4a236"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;issue:issue:10.1021/jmcmar.2018.61.issue-23;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.8b01626;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Computational Method to Evaluate Progress in Lead Optimization" /></meta><meta name="dc.Creator" content="Martin  Vogt" /></meta><meta name="dc.Creator" content="Dimitar  Yonchev" /></meta><meta name="dc.Creator" content="Jürgen  Bajorath" /></meta><meta name="dc.Description" content="In medicinal chemistry, lead optimization is a critically important task and a highly empirical process, largely driven by chemical knowledge and intuition. Only very few approaches are available t..." /></meta><meta name="Description" content="In medicinal chemistry, lead optimization is a critically important task and a highly empirical process, largely driven by chemical knowledge and intuition. Only very few approaches are available t..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="November 30, 2018" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.8b01626" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2018 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.8b01626" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.8b01626" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.8b01626" /></link>
        
    
    

<title>Computational Method to Evaluate Progress in Lead Optimization | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.8b01626" /></meta><meta property="og:title" content="Computational Method to Evaluate Progress in Lead Optimization" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01626/20181207/images/large/jm-2018-01626v_0006.jpeg" /></meta><meta property="og:description" content="In medicinal chemistry, lead optimization is a critically important task and a highly empirical process, largely driven by chemical knowledge and intuition. Only very few approaches are available to guide and evaluate optimization efforts. It is often very difficult to understand when a compound series is exhausted and the generation of additional analogs unlikely to yield further progress toward potent and efficacious candidates. Rationalizing lead optimization remains an essentially unsolved problem. Herein, we introduce a new computational method to aid in evaluating whether sufficient numbers of analogs have been made and further progress is unlikely. The approach integrates the assessment of chemical saturation and structure–activity relationship progression of compound series. Easy-to-calculate scores characterize evolving analog series and identify candidates with high or low priority for further chemical exploration." /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta><meta name="twitter:description" content="In medicinal chemistry, lead optimization is a critically important task and a highly empirical process, largely driven by chemical knowledge and intuition. Only very few approaches are available to guide and evaluate optimization efforts. It is often very difficult to understand when a compound series is exhausted and the generation of additional analogs unlikely to yield further progress toward potent and efficacious candidates. Rationalizing lead optimization remains an essentially unsolved problem. Herein, we introduce a new computational method to aid in evaluating whether sufficient numbers of analogs have been made and further progress is unlikely. The approach integrates the assessment of chemical saturation and structure–activity relationship progression of compound series. Easy-to-calculate scores characterize evolving analog series and identify candidates with high or low priority for further chemical exploration." /></meta><meta name="twitter:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01626/20181207/images/large/jm-2018-01626v_0006.jpeg" /></meta><meta name="twitter:title" content="Computational Method to Evaluate Progress in Lead Optimization" /></meta>
        
            <meta property="og:type" content="Article" /></meta>
        

        
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.8b01626"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/ke1q~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-6ee29527b3d8ff00a106.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.8b01626">
            
        





































    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-12/achre4.2021.54.issue-12/20210615/achre4.2021.54.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-6/amrcda.2021.2.issue-6/20210625/amrcda.2021.2.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-6/aabmcb.2021.4.issue-6/20210621/aabmcb.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-6/aaembp.2021.3.issue-6/20210622/aaembp.2021.3.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-6/aaemcq.2021.4.issue-6/20210628/aaemcq.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-25/aamick.2021.13.issue-25/20210630/aamick.2021.13.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-6/aanmf6.2021.4.issue-6/20210625/aanmf6.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-6/aapmcd.2021.3.issue-6/20210611/aapmcd.2021.3.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-6/abseba.2021.7.issue-6/20210614/abseba.2021.7.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-12/accacs.2021.11.issue-12/20210618/accacs.2021.11.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-6/acscii.2021.7.issue-6/20210623/acscii.2021.7.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-6/acbcct.2021.16.issue-6/20210618/acbcct.2021.16.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-3/achsc5.2021.28.issue-3/20210524/achsc5.2021.28.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-12/acncdm.2021.12.issue-12/20210616/acncdm.2021.12.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-6/aesccq.2021.5.issue-6/20210617/aesccq.2021.5.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-6/aelccp.2021.6.issue-6/20210611/aelccp.2021.6.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-6/aeecco.2021.1.issue-6/20210611/aeecco.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-6/aewcaa.2021.1.issue-6/20210611/aewcaa.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-5/afsthl.2021.1.issue-5/20210618/afsthl.2021.1.issue-5.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-6/aidcbc.2021.7.issue-6/20210611/aidcbc.2021.7.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-6/amlccd.2021.10.issue-6/20210615/amlccd.2021.10.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-6/amlcef.2021.3.issue-6/20210607/amlcef.2021.3.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-6/amclct.2021.12.issue-6/20210610/amclct.2021.12.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-6/ancac3.2021.15.issue-6/20210622/ancac3.2021.15.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-25/acsodf.2021.6.issue-25/20210629/acsodf.2021.6.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-6/apchd5.2021.8.issue-6/20210616/apchd5.2021.8.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-6/ascefj.2021.6.issue-6/20210625/ascefj.2021.6.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-25/ascecg.2021.9.issue-25/20210628/ascecg.2021.9.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-6/asbcd6.2021.10.issue-6/20210618/asbcd6.2021.10.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-25/ancham.2021.93.issue-25/20210629/ancham.2021.93.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-25/bichaw.2021.60.issue-25/20210629/bichaw.2021.60.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-6/bcches.2021.32.issue-6/20210616/bcches.2021.32.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-6/bomaf6.2021.22.issue-6/20210614/bomaf6.2021.22.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-24/cgeabj.2021.99.issue-24/20210628-01/cgeabj.2021.99.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-6/crtoec.2021.34.issue-6/20210621/crtoec.2021.34.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-12/chreay.2021.121.issue-12/20210623/chreay.2021.121.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-12/cmatex.2021.33.issue-12/20210622/cmatex.2021.33.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-6/cgdefu.2021.21.issue-6/20210602/cgdefu.2021.21.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-12/enfuem.2021.35.issue-12/20210617/enfuem.2021.35.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-12/esthag.2021.55.issue-12/20210615/esthag.2021.55.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-6/estlcu.2021.8.issue-6/20210608/estlcu.2021.8.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-25/iecred.2021.60.issue-25/20210630/iecred.2021.60.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-12/inocaj.2021.60.issue-12/20210621/inocaj.2021.60.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-6/jaaucr.2021.1.issue-6/20210628/jaaucr.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-25/jafcau.2021.69.issue-25/20210630/jafcau.2021.69.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-6/jceaax.2021.66.issue-6/20210610/jceaax.2021.66.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-6/jceda8.2021.98.issue-6/20210608/jceda8.2021.98.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-6/jcisd8.2021.61.issue-6/20210628/jcisd8.2021.61.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-6/jctcce.2021.17.issue-6/20210608/jctcce.2021.17.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/jmcmar.2021.64.issue-12/20210624/jmcmar.2021.64.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-6/jnprdf.2021.84.issue-6/20210625/jnprdf.2021.84.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-12/joceah.2021.86.issue-12/20210618/joceah.2021.86.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-24/jpcafh.2021.125.issue-24/20210624/jpcafh.2021.125.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-24/jpcbfk.2021.125.issue-24/20210624/jpcbfk.2021.125.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-24/jpccck.2021.125.issue-24/20210624/jpccck.2021.125.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-24/jpclcd.2021.12.issue-24/20210624/jpclcd.2021.12.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-6/jprobs.2021.20.issue-6/20210604/jprobs.2021.20.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-25/jacsat.2021.143.issue-25/20210630/jacsat.2021.143.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-6/jamsef.2021.32.issue-6/20210602/jamsef.2021.32.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-25/langd5.2021.37.issue-25/20210629/langd5.2021.37.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-12/mamobx.2021.54.issue-12/20210622/mamobx.2021.54.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-6/mpohbp.2021.18.issue-6/20210607/mpohbp.2021.18.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-12/nalefd.2021.21.issue-12/20210623/nalefd.2021.21.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-12/orlef7.2021.23.issue-12/20210618/orlef7.2021.23.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-6/oprdfk.2021.25.issue-6/20210618/oprdfk.2021.25.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-12/orgnd7.2021.40.issue-12/20210628/orgnd7.2021.40.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.8b01626&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.8b01626&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.8b01626&amp;href=/doi/10.1021/acs.jmedchem.8b01626" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2018</span><span class="cit-fg-volume">, 61</span><span class="cit-fg-issue">, 23</span><span class="cit-fg-pageRange">, 10895-10900</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/61/23" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.8b01544" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.8b01633" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Computational Method to Evaluate Progress in Lead Optimization</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Martin Vogt</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Martin Vogt</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Life Science Informatics, B-IT, LIMES Program Unit Chemical Biology and Medicinal Chemistry, Rheinische Friedrich-Wilhelms-Universität, Endenicher Allee 19c, D-53115 Bonn, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Martin++Vogt">Martin Vogt</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Dimitar Yonchev</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Dimitar Yonchev</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Life Science Informatics, B-IT, LIMES Program Unit Chemical Biology and Medicinal Chemistry, Rheinische Friedrich-Wilhelms-Universität, Endenicher Allee 19c, D-53115 Bonn, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Dimitar++Yonchev">Dimitar Yonchev</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Jürgen Bajorath</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jürgen Bajorath</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Life Science Informatics, B-IT, LIMES Program Unit Chemical Biology and Medicinal Chemistry, Rheinische Friedrich-Wilhelms-Universität, Endenicher Allee 19c, D-53115 Bonn, Germany</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#0b696a6164796a7f634b69627f257e65622669646565256f6e"><span class="__cf_email__" data-cfemail="c7a5a6ada8b5a6b3af87a5aeb3e9b2a9aeeaa5a8a9a9e9a3a2">[email protected]</span></a>. Phone: +49-228-7369-100. Fax: +49-228-7369-101.</span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=J%C3%BCrgen++Bajorath">Jürgen Bajorath</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-0557-5714" title="Orcid link">http://orcid.org/0000-0002-0557-5714</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.8b01626&amp;href=/doi/10.1021%2Facs.jmedchem.8b01626" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2018</span></span><span class="cit-volume">, 61</span><span class="cit-issue">, 23</span><span class="cit-pageRange">, 10895–10900</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">November 30, 2018</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>19 October 2018</li><li><span class="item_label"><b>Published</b> online</span>30 November 2018</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 13 December 2018</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.8b01626" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b01626</a></div><div class="article_header-article-copyright"><strong>Copyright © 2018 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D10895%26pageCount%3D6%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DMartin%2BVogt%252C%2BDimitar%2BYonchev%252C%2BJ%25C3%25BCrgen%2BBajorath%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D61%26issueNum%3D23%26contentID%3Dacs.jmedchem.8b01626%26title%3DComputational%2BMethod%2Bto%2BEvaluate%2BProgress%2Bin%2BLead%2BOptimization%26numPages%3D6%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D10900%26publicationDate%3DDecember%2B2018">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.8b01626"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1899</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">8</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.8b01626" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Computational Method to Evaluate Progress in Lead Optimization&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Martin&quot;,&quot;last_name&quot;:&quot;Vogt&quot;},{&quot;first_name&quot;:&quot;Dimitar&quot;,&quot;last_name&quot;:&quot;Yonchev&quot;},{&quot;first_name&quot;:&quot;Jürgen&quot;,&quot;last_name&quot;:&quot;Bajorath&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2018&quot;,&quot;month&quot;:&quot;11&quot;,&quot;day&quot;:&quot;30&quot;,&quot;issue&quot;:&quot;23&quot;,&quot;volume&quot;:&quot;61&quot;,&quot;pages&quot;:&quot;10895-10900&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.8b01626&quot;},&quot;abstract&quot;:&quot;In medicinal chemistry, lead optimization is a critically important task and a highly empirical process, largely driven by chemical knowledge and intuition. Only very few approaches are available to guide and evaluate optimization efforts. It is often very difficult to understand when a compound series is exhausted and the generation of additional analogs unlikely to yield further progress toward potent and efficacious candidates. Rationalizing lead optimization remains an essentially unsolved problem. Herein, we introduce a new computational method to aid in evaluating whether sufficient numbers of analogs have been made and further progress is unlikely. The approach integrates the assessment of chemical saturation and structure–activity relationship progression of compound series. Easy-to-calculate scores characterize evolving analog series and identify candidates with high or low priority for further chemical exploration.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b01626&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01626" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b01626&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01626" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b01626&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01626" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.8b01626&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01626&amp;href=/doi/10.1021/acs.jmedchem.8b01626" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.8b01626" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.8b01626" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (760 KB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b01626&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.8b01626%26sid%3Dliteratum%253Aachs%26genre%3Darticle%26aulast%3DVogt%26date%3D2018%26atitle%3DComputational%2BMethod%2Bto%2BEvaluate%2BProgress%2Bin%2BLead%2BOptimization%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D61%26issue%3D23%26spage%3D10895%26epage%3D10900%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (1)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292549" title="Saturation">Saturation</a>,</li><li><a href="/action/doSearch?ConceptID=291494" title="Medicinal chemistry">Medicinal chemistry</a>,</li><li><a href="/action/doSearch?ConceptID=292304" title="Optimization">Optimization</a>,</li><li><a href="/action/doSearch?ConceptID=291901" title="Chemical calculations">Chemical calculations</a>,</li><li><a href="/action/doSearch?ConceptID=291465" title="Structure activity relationship">Structure activity relationship</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/61/23" title="Go to Volume 61, Issue 23"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/jmcmar.2018.61.issue-23/20181213/jmcmar.2018.61.issue-23.largecover.jpg" alt="Go to Volume 61, Issue 23"/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01626/20181207/images/medium/jm-2018-01626v_0006.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01626/20181207/images/large/jm-2018-01626v_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01626&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">In medicinal chemistry, lead optimization is a critically important task and a highly empirical process, largely driven by chemical knowledge and intuition. Only very few approaches are available to guide and evaluate optimization efforts. It is often very difficult to understand when a compound series is exhausted and the generation of additional analogs unlikely to yield further progress toward potent and efficacious candidates. Rationalizing lead optimization remains an essentially unsolved problem. Herein, we introduce a new computational method to aid in evaluating whether sufficient numbers of analogs have been made and further progress is unlikely. The approach integrates the assessment of chemical saturation and structure–activity relationship progression of compound series. Easy-to-calculate scores characterize evolving analog series and identify candidates with high or low priority for further chemical exploration.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_37563" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_37563" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">One of the great challenges in medicinal chemistry is rationalizing lead optimization (LO).<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> At each stage of this time-consuming and costly process, a medicinal chemist must decide which compounds to make next in order to reach optimization milestones. Moreover, as an analog series (AS) evolves, it is difficult to judge the odds that further progress will be made in exploring and exploiting structure–activity relationships (SARs). During LO, multiple properties must be balanced to reach the stage when an active compound can be considered for in vivo studies. However, if a compound is not sufficiently potent, it will not become a candidate, irrespective of other features. This requirement makes SAR exploration a central task during early to mid-stages of LO. Given the complexity of LO, this process is largely driven by chemical knowledge, experience, and intuition. There are neither generally applicable optimization routines available nor methods to rationalize LO campaigns and estimate their outcomes. More often than not, roadblocks are encountered during LO and progress made is less than anticipated. However, once substantial efforts have been expanded to advance ASs, terminating a project is a difficult decision to make, even if it is questionable or unlikely that final goals are within reach. Given the essentially subjective nature of LO and the absence of unbiased evaluation criteria, it is often easier to make more analogs and hope to hit a home run than to discontinue a high-profile project. As a consequence, many LO campaigns are carried on for a long time and large numbers of analogs are generated, despite the lack of ultimate success.</div><div class="NLM_p">In light of this situation, methodologies that help to characterize ASs and evaluate their potential are sought after in medicinal chemistry. Of course, it can hardly be expected that any single methodology will be capable of solving the LO problem. However, any method that can provide decision support during LO is desirable, especially when assessing project progression. Only very few approaches are currently available that can be considered in this context. Among computational methods, quantitative SAR (QSAR) analysis<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> has long been a standard for predicting active analogs for an evolving series. However, QSAR is not capable of assessing the progression of an AS. Going beyond compound potency prediction, statistical multi-objective optimization and analysis methods are applicable to suggest candidate compounds with desirable properties<a onclick="showRef(event, 'ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref3 ref4">(3,4)</a> or monitor compounds during LO that meet predefined optimization criteria.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> Furthermore, coverage of chemical space around ASs has been estimated by defining neighborhoods (NBHs) of experimental analogs and screening these NBHs with virtual compounds.<a onclick="showRef(event, 'ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref5 ref6">(5,6)</a> Other approaches have been introduced to visualize SAR information<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> and monitor SAR characteristics of evolving data sets.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> Furthermore, numerical SAR analysis functions can be applied to assess SAR progression<a onclick="showRef(event, 'ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref9 ref10">(9,10)</a> and statistical techniques to identify compounds that make significant contributions to positive progression and reduce the risk of failure.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> However, although SAR visualization methods and numerical functions enable retrospective analysis, they cannot be used to predict whether further progress in LO might be made.</div><div class="NLM_p">Herein, we introduce a computational method to estimate whether an AS can be further advanced by generating more analogs or whether this would be unlikely. The approach was designed to integrate the assessment of chemical saturation and SAR progression of ASs. Evaluation of our methodology was based on the largest ASs from medicinal chemistry sources that we were able to identify in the public domain. <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a> shows representative examples.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01626/20181207/images/medium/jm-2018-01626v_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01626/20181207/images/large/jm-2018-01626v_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Exemplary ASs. Shown are AS-based scaffolds representing individual ASs. For each AS, the ChEMBL ID, target name, number of analogs, and their potency range are reported.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01626/20181207/images/large/jm-2018-01626v_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01626&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_82532" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_82532" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Methodological Concept</h3><div class="NLM_p">The new method is based on the following principle. To evaluate LO progress, it must be determined (i) how extensively chemical space around a given AS is covered and (ii) whether analogs display significant potency variations. The tasks in (i) and (ii) can be rationalized as determining <i>chemical saturation</i> and <i>SAR progression</i>, respectively. Varying potency of structural analogs indicates SAR discontinuity,<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> the apex of which are activity cliffs.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> In the absence of SAR discontinuity, that is, when all analogs made have comparable potency values, there is no SAR progression. Combining (i) and (ii) enables the characterization of LO progress of ASs. In the presence of extensive chemical space coverage around an AS and the absence of SAR discontinuity, it is unlikely that additional analogs with further increased potency will be available. To assess (i) <i>chemical saturation</i>, populations of virtual analogs are generated for a given AS whose distribution serves as an indicator of chemical space coverage around the series. To evaluate (ii) <i>SAR progression</i>, potency changes among analogs that have overlapping NBHs with shared virtual analogs are quantified and weighted, as further rationalized below.</div><div class="NLM_p last">Central to the methodology is the definition of analog NBHs in chemical space. The vastness of chemical space makes the enumeration of all conceivable compounds infeasible, even if chemical space is confined to a single AS. For a given series, the space may be mapped using representative virtual analogs. This concept was adapted from a recent chemical saturation analysis,<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> modified and further extended. For quantification of parameters, it must be ensured that the sample of virtual analogs is large enough to yield consistent results. To evaluate compound distributions in AS-centered chemical space and across NBHs of analogs, distances are calculated. Specifically, the distance <i>d</i>(<i>a</i>,<i>b</i>) between two compounds <i>a</i> and <i>b</i> is given by the Euclidean distance between two multidimensional vectors encoding molecular properties after unit-variance scaling on the basis of the virtual analog sample of an AS. A threshold distance, <i>t</i>, must be set to determine the NBH radius of an active analog. The derivation of the corresponding scoring scheme is presented in the following.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Scoring Scheme</h3><div class="NLM_p">On the basis of our methodological concept, a scoring scheme was developed for profiling ASs that addresses three important questions concerning the level of chemical saturation and SAR progression:<table class="listgroup" border="0" width="95%" id="list1" list-type="label"><tr class="li1"><td valign="top">1.</td><td colspan="5" valign="top" class="paragraph"><p class="last">How extensively does an AS cover chemical space restricted by its core structure?</p></td></tr><tr class="li1"><td valign="top">2.</td><td colspan="5" valign="top" class="paragraph"><p class="last">How densely does an AS sample cover chemical space?</p></td></tr><tr class="li1"><td valign="top">3.</td><td colspan="5" valign="top" class="paragraph"><p class="last">How strongly do potencies of close analogs vary?</p></td></tr></table></div><div class="NLM_p">The first two questions relate to chemical saturation, whereas the third one relates to SAR progression. Corresponding parameters can be estimated from the structures and potency values of analogs comprising a given AS. The resulting scores are simple and intuitive. Their formal derivation is presented in the following.</div><div id="sec2_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i6" class="anchor-spacer"></div><h4 class="article-section__title" id="_i6"> Chemical Saturation</h4><div class="NLM_p">Sampled chemical space and NBH definition form the basis for quantifying a set of key parameters. Given an AS <i>A</i> = {<i>a</i><sub>1</sub>, <i>a</i><sub>2</sub>, ..., <i>a</i><sub><i>n</i><sub>A</sub></sub>} consisting of <i>n</i><sub>A</sub> active analogs and a sampled chemical space <i>V</i> = {<i>v</i><sub>1</sub>, <i>v</i><sub>2</sub>, ..., <i>v</i><sub><i>n</i><sub>V</sub></sub>} of <i>n</i><sub>V</sub> virtual analogs, the sets <i>V</i><sub>i</sub> = {<i>v</i> ∈ <i>V</i> | <i>d</i>(<i>v</i>,<i>a</i><sub><i>i</i></sub>) ≤ <i>t</i>} are defined as virtual analogs falling into the NBHs of <i>a</i><sub><i>i</i></sub>. The union <span class="NLM_inline-formula"><img class="imgWithoutFigViewer inline-formula-styles" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01626/20181207/images/medium/jm-2018-01626v_m001.gif" alt="" id="_i7" /></img></span> then gives the set of all virtual analogs that are contained in an NBH of at least one active compound <i>a</i><sub><i>i</i></sub>. Let <i>n</i><sub>N</sub> = |<i>V</i><sub>N</sub>| be the number of virtual analogs in NBHs. The coverage<span class="NLM_disp-formula" id="ueq1"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01626/20181207/images/medium/jm-2018-01626v_m002.gif" alt="" id="_i8" /></img></span>with range [0, 1] is the fraction of virtual analogs in the combined NBHs of all active compounds <i>A</i> and used to estimate coverage of relevant chemical space for a given AS.</div><div class="NLM_p">Virtual analogs <i>v</i> of <i>V</i><sub>N</sub> can be contained in one or more NBHs. The larger the number of NBHs becomes for a given <i>v</i>, the more densely the virtual analog is surrounded by actives. The average number <i>d</i><sub>mean</sub> of overlapping NBHs containing a virtual analog in <i>V</i><sub>N</sub> can be determined by summing all NBHs and dividing by the number of virtual analogs in <i>V</i><sub>N</sub><span class="NLM_disp-formula" id="ueq2"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01626/20181207/images/medium/jm-2018-01626v_m003.gif" alt="" id="_i9" /></img></span></div><div class="NLM_p">This parameter is normalized to the range [0, 1], yielding a density score<span class="NLM_disp-formula" id="ueq3"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01626/20181207/images/medium/jm-2018-01626v_m004.gif" alt="" id="_i10" /></img></span></div><div class="NLM_p last">The extent and density of chemical space coverage around a given AS indicate its chemical saturation. Accordingly, the coverage and density scores can be combined to yield a single saturation score <i>S</i>, which is defined as the harmonic mean of <i>C</i> and <i>D.</i><span class="NLM_disp-formula" id="ueq4"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01626/20181207/images/medium/jm-2018-01626v_m005.gif" alt="" id="_i11" /></img></span></div></div><div id="sec2_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i12" class="anchor-spacer"></div><h4 class="article-section__title" id="_i12"> SAR Progression</h4><div class="NLM_p">If a virtual analog is present in overlapping NBHs of multiple actives, the magnitude of potency variations among active analogs indicates the degree of SAR discontinuity in this chemical sub-space. The parameter Δ̅<sub><i>i</i></sub> quantifies the potency range for multiple actives, with NBHs containing the same virtual analog. For <i>v</i><sub><i>i</i></sub> ∈ <i>V</i><sub>N</sub>, present in NBHs of <i>m</i><sub><i>i</i></sub> actives {<i>a</i><sub>1</sub>, ..., <i>a</i><sub><i>m</i><sub><i>i</i></sub></sub>}, Δ̅<sub><i>i</i></sub> is calculated as the average potency difference over all pairs of active analogs.<span class="NLM_disp-formula" id="ueq5"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01626/20181207/images/medium/jm-2018-01626v_m006.gif" alt="" id="_i13" /></img></span>Here, pot(<i>a</i>) denotes the potency of compound <i>a</i> on a logarithmic scale.</div><div class="NLM_p">Parameter <i>P</i> is then calculated as the mean potency range of NBHs over all virtual analogs using a weighting scheme <i>w</i><sub><i>i</i></sub> = 1/<i>m</i><sub><i>i</i></sub> if <i>m</i><sub><i>i</i></sub> > 1 and <i>w</i><sub><i>i</i></sub> = 0 if <i>m</i><sub><i>i</i></sub> = 1.<span class="NLM_disp-formula" id="ueq6"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01626/20181207/images/medium/jm-2018-01626v_m007.gif" alt="" id="_i14" /></img></span></div><div class="NLM_p last">This score accounts for SAR discontinuity in NBHs of active analogs. The weighting scheme emphasizes SAR discontinuity in smaller numbers of overlapping NBHs associated with a given virtual compound, that is, less-explored regions of chemical space around a given AS where highly potent analogs might preferentially be identified.</div></div><div id="sec2_2_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i15" class="anchor-spacer"></div><h4 class="article-section__title" id="_i15"> Score Combinations</h4><div class="NLM_p last">Considering <i>S</i> and <i>P</i> scores in combination makes it possible to estimate the potential of an AS for LO. An AS displaying a high degree of chemical saturation and little SAR progression is thought to have low LO potential, corresponding to a large <i>S</i> and small <i>P</i> score. By contrast, an AS with small <i>S</i> and large <i>P</i> score has high LO potential. A continuum of score combinations can be considered to distinguish between different levels of LO potential.</div></div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> NBH Radius</h3><div class="NLM_p">A threshold distance <i>t</i> determines the NBH radius of an active analog. It is derived from distance distributions between virtual analogs of a given AS. Virtual analogs outnumber active analogs and serve as an indicator of chemical space coverage. With increasing NBH radii, the likelihood increases that NBHs of active analogs are overlapping and populated with virtual analogs. Using 10 000 virtual analogs per AS, we calculated scores for increasing NBH radii. The dependence of coverage <i>C</i> on NBH radii is monitored in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>, which shows the distribution of <i>C</i> values for all ASs at a given radius. As expected, <i>C</i> increased with increasing NBH radii and the score distributions notably widened. Very similar observations were made for the <i>D</i> and <i>P</i> scores. Initially, we profiled ASs by comparing two NBH radii with percentile values of 1.0 and 5.0, respectively, and obtained similar results. Taking into consideration that publicly available ASs were confined in size, with five of 34 series consisting of more than 100 analogs, we consistently set the threshold of the NBH radius for subsequent calculations to the 1st percentile of the distribution of pairwise distances between virtual analogs.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01626/20181207/images/medium/jm-2018-01626v_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01626/20181207/images/large/jm-2018-01626v_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Score dependence on NBH radii. Boxplots report the distribution of coverage <i>C</i> for NBHs with increasing percentile threshold (radius). For each AS, the mean of 10 independent calculations was used. Boxplots report the smallest value (bottom line), first quartile (lower boundary of the box), median value (thick line), third quartile (upper boundary of the box), and largest value (top line).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01626/20181207/images/large/jm-2018-01626v_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01626&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Virtual Analog Samples</h3><div class="NLM_p last">Next, <i>C</i>, <i>D</i>, and <i>P</i> scores were calculated by systematically increasing the number of virtual analogs per series from 500 to 13 000 (in 10 increments of gradually increasing size). Each calculation was repeated 10 times by randomly selecting the respective number of virtual analogs from the source set of each AS. <i>C</i>, <i>D</i>, and <i>P</i> scores remained essentially constant for all ASs when virtual analogs at least on the order of 5000 were used. For consistency with calculations varying NBH radii, the results discussed in the following were obtained for a constant number of 10 000 virtual analogs per AS.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Comparison of Coverage and Density</h3><div class="NLM_p"><a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a> compares the distribution of <i>C</i> and <i>D</i> scores for all ASs. Given the confined NBH radius used for moderately sized ASs, <i>C</i> scores were distributed over the range [0, 0.25] (see also <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). <i>D</i> scores were widely distributed, mostly over the range [0.4, 0.8]. Two important observations were made. First, <i>C</i> and <i>D</i> scores were uncorrelated and ASs with small <i>C</i> and large <i>D</i> scores and vice versa were detected. Second, <i>C</i> and <i>D</i> scores did not correlate with the size of ASs; both smaller and larger series were found to yield a variety of <i>C</i> or <i>D</i> scores within the observed ranges. Taken together, these observations indicated that score distributions were influenced by chemical characteristics of ASs and that <i>C</i> and <i>D</i> scores captured different aspects of chemical saturation, consistent with their design. Accordingly, the complementary nature of <i>C</i> and <i>D</i> supported the generation of a composite chemical saturation score <i>S</i>.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01626/20181207/images/medium/jm-2018-01626v_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01626/20181207/images/large/jm-2018-01626v_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Density vs coverage. The scatter plot compares <i>D</i> and <i>C</i> scores for all ASs. Each AS is represented by a dot that is scaled in size according to the number of analogs it contains (smallest dot, 51 analogs; largest, 166). Exemplary ASs are numbered according to <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01626/20181207/images/large/jm-2018-01626v_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01626&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Profiling of Analog Series</h3><div class="NLM_p">As a central part of our analysis, chemical saturation and SAR progression were explored in context. <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a> shows the distribution of progression score <i>P</i> and saturation score <i>S</i> for all ASs. Both scores were broadly distributed. For ASs with <i>S</i> scores greater than 0.2, the distribution of <i>P</i> scores widened, covered a large range [0.2, 1.2], and separated ASs from each other. For example, AS 2 in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a> (phosphodiesterase 10A inhibitors, 146 analogs; see also <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) was characterized by one of the highest degrees of chemical saturation in our set of ASs and the overall highest level of SAR progression. <i>S</i> scores of available ASs were generally smaller than 0.4. Therefore, AS 2 was considered to have further LO potential, given its intermediate <i>S</i> and high <i>P</i> score. Accordingly, the generation of additional analogs would be expected to yield potent compounds. In addition, AS 4 (kinase JAK-1 inhibitors, 81 analogs), which was smaller in size than AS 2, had an <i>S</i> score comparable to AS 2 and the overall second highest <i>P</i> score. Therefore, similar conclusions would be drawn for AS 2 and AS 4 and generating additional analogs for AS 4 would be expected to yield further LO progress. However, other ASs with more than 100 analogs yielded some of the smallest <i>S</i> and intermediate <i>P</i> scores. Hence, score differences were not resulting from differences in size, but chemical composition. For example, this was the case for AS 1 (acetyl-CoA carboxylase 2 inhibitors, 126 analogs), which represented one of the series with lowest degree of chemical saturation, but detectable SAR progression. In this case, the generation of additional analogs should be carefully monitored for potential increases in SAR discontinuity. The observed continuum of <i>S</i>/<i>P</i> score combinations made it possible to assign different levels of LO potential to ASs. Notably, ASs with largest <i>S</i> scores within our set and smallest <i>P</i> were expected to have lowest LO potential among the series we compared. An example for such ASs was provided by AS 3 (dopamine D2 receptor antagonists, 69 analogs), which had low priority for further exploration. By contrast, other ASs with <i>S</i> scores around 0.2 and <i>P</i> scores of 0.6 or greater had higher LO potential. Overall, a variety of score combinations were observed for ASs of varying size, which clearly differentiated ASs and made it possible to prioritize series for further optimization efforts. For example, <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a> shows exemplary analogs of a series of ATPase inhibitors and virtual analogs falling into their NBHs. This AS was among the smallest we profiled (53 analogs) and had one of the lowest <i>S</i> scores (0.11). However, despite low chemical saturation, the AS reached an intermediate <i>P</i> score (0.46) within our set and should thus be considered to have further optimization potential.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01626/20181207/images/medium/jm-2018-01626v_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01626/20181207/images/large/jm-2018-01626v_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. SAR progression vs chemical saturation. The scatter plot compares <i>P</i> and <i>S</i> scores for all ASs. Each AS is represented by a dot that is scaled in size according to the number of analogs. Exemplary ASs are numbered according to <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01626/20181207/images/large/jm-2018-01626v_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01626&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01626/20181207/images/medium/jm-2018-01626v_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01626/20181207/images/large/jm-2018-01626v_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Progressing AS. Shown are exemplary analogs belonging to a series of endoplasmic reticulum ATPase inhibitors (black) and their NBHs (light blue) and virtual analogs (red) falling into the NBHs. Substitution sites are encircled.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01626/20181207/images/large/jm-2018-01626v_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01626&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i20">Discussion and Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_37261" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_37261" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In medicinal chemistry, LO is a largely subjective process that is difficult to rationalize and formalize. Generally applicable LO criteria or protocols are not available, and it is very difficult to understand when optimization efforts become unlikely to further advance an AS. Consequently, approaches that provide guidance during LO and decision support are highly desirable. We have introduced a computational method to evaluate LO progress. The approach combines the assessment of chemical saturation and SAR progression and makes it possible to characterize and differentiate ASs. Sets of virtual analogs generated for individual ASs aid in defining NBHs of active analogs and serve as an indicator of chemical space coverage. Furthermore, detectable SAR discontinuity within overlapping NBHs is regarded as a prerequisite for obtaining increasingly potent compounds. The combined analysis of chemical saturation and SAR progression is translated into a scoring scheme for profiling and prioritizing ASs. High <i>S</i> scores approaching 1 and low <i>P</i> score approaching 0 indicate that ASs are exhausted and provide a criterion for discontinuation beyond subjective assessment. On the other hand, opposite scores are indicative of optimization potential.</div><div class="NLM_p">The approach is expected to be influenced by the choice of descriptors and chemical reference spaces as well as virtual compound populations. A previous study investigated metrics for the assessment of chemical saturation of ASs under varying conditions.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> This included investigating the influence of chemical space representations based on different descriptor sets and alternative approaches to virtual compound generation for sampling chemical space. The saturation assessment for different sets of ASs remained essentially stable under these varying conditions. For example, when a 14-dimensional reference space was used instead of the seven chemically intuitive descriptors used herein, comparable results were obtained.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> Similarly, only little changes in chemical saturation were observed when virtual analog populations were generated using different R-group sets and design strategies. In this study, we used R-group sets that were systematically extracted from bioactive compounds in ChEMBL to enumerate virtual analogs on the basis of ASB scaffolds representing series, without additional filtering of analogs. Importantly, as introduced herein, the methodology can be tested with many different descriptors and candidate compounds, depending on preferences and project specifics. There are no intrinsic limitations. This also applies to distance measures for defining NBHs in chemical reference space. After initially exploring various alternatives, including Tanimoto distances in fingerprint spaces, our preference is the straightforward calculation of Euclidian distances on the basis of numerical descriptors, but more complex measures can certainly be explored. Essentially, calculation parameters can be adjusted at will and virtual compounds from different sources be used. We also note that the approach is applicable to ASs of any composition and not intrinsically limited by the number of substitution sites. For our study, ASs have been systematically extracted from public domain compounds originating from the medicinal chemistry literature. The ASs studied herein represented the largest we have been able to identify via automated AS extraction, requiring the availability of reliable activity measurements, a prerequisite for meaningful profiling analysis. These ASs contained from one to three substitution sites. In the practice of medicinal chemistry, ASs with more than four or five site substitution sites are rare. However, ASs with three or more sites can be readily profiled.</div><div class="NLM_p">Given the complexity of LO, it is anticipated that the new methodology will be of interest to practicing chemists. For computational medicinal chemists, the scoring scheme is straightforward to implement and scores are easy to calculate.</div><div class="NLM_p last">Although we are currently limited to studying the relatively small number of sizable ASs that are available in the public domain, it will be exciting to see applications of the methodology to mid- and late-stage series from drug discovery projects.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i21">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_25014" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_25014" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Chemical Reference Space</h3><div class="NLM_p last">For profiling of ASs, a 7-dimensional chemical reference space was generated using chemically intuitive descriptors accounting for molecular properties known to be relevant for ligand–target interactions, as previously used for assessing chemical saturation.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> These descriptors included the number of hydrogen bond donors, acceptors, and rotatable bonds; logarithmic octanol/water partition coefficient and aqueous solubility, topological polar surface area, and molecular weight were calculated using RDKit<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> and, in the case of aqueous solubility, using a freely available custom implementation based on the ESOL method.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Analog Series</h3><div class="NLM_p">ASs with single and multiple substitution sites, more than 50 compounds, and available high-confidence activity data were systematically extracted from ChEMBL (release 23)<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> using a computational analog selection method<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> based on the matched molecular pair (MMP) formalism.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> An MMP is defined as a pair of compounds that are distinguished by a chemical change at a single site.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a></div><div class="NLM_p last">A total of 34 ASs were obtained, including 15 ASs with single and 19 with multiple (two to three) substitution sites. The 34 ASs consisted of 51–166 compounds (five series contained more than 100 analogs). Each AS was active against a unique target. These compound data sets represented the largest and most diverse collection of ASs from medicinal chemistry sources that we were able to identify in the public domain. <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a> shows exemplary ASs. Each AS is represented by its AS-based (ASB) scaffold<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> covering all substitution sites (reminiscent of a Markush structure). For all ASs and virtual analogs of AS 4 in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>, molecular formula strings are provided as <a href="/doi/suppl/10.1021/acs.jmedchem.8b01626/suppl_file/jm8b01626_si_001.csv" class="ext-link">Supporting Information</a>.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> Virtual Analogs</h3><div class="NLM_p">From each AS, the ASB scaffold<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> was isolated. From ChEMBL (release 23), a total of 14 026 unique R-groups were extracted by systematically calculating MMPs with size-restricted chemical changes<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> from bioactive compounds. On the basis of each ASB scaffold representing an AS, virtual analogs were enumerated by systematically adding R-groups to substitution sites. The calculations were carried out with the aid of the OpenEye toolkit.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a></div><div class="NLM_p last">For ASs with single substitution sites, 13 965 to 14 012 virtual analogs were generated (i.e., one per unique R-group). The number of virtual analogs per AS was in each case smaller than 14 026 because existing analogs were frequently reproduced using the pool of R-groups extracted from ChEMBL. For each AS with multiple substitution sites, 140 260 distinct virtual analogs were enumerated by randomly selecting R-groups for each substitution site from the ChEMBL pool. Virtual analogs generated for each AS provided source sets for score calculations.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i25"><a href="/doi/suppl/10.1021/acs.jmedchem.8b01626" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_85040" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_85040" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.8b01626" class="ext-link">10.1021/acs.jmedchem.8b01626</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Molecular formula strings of ASs and virtual analogs for an exemplary AS are provided (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01626/suppl_file/jm8b01626_si_001.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b01626/suppl_file/jm8b01626_si_001.csv">jm8b01626_si_001.csv (1.42 MB)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.8b01626" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_37775" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_37775" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jürgen Bajorath</span> - <span class="hlFld-Affiliation affiliation">Department of Life Science Informatics,
B-IT, LIMES Program Unit Chemical Biology and Medicinal Chemistry, Rheinische Friedrich-Wilhelms-Universität, Endenicher Allee 19c, D-53115 Bonn, Germany</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-0557-5714" title="Orcid link">http://orcid.org/0000-0002-0557-5714</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#2745464d485546534f67454e530952494e0a45484949094342"><span class="__cf_email__" data-cfemail="3f5d5e55504d5e4b577f5d564b114a5156125d505151115b5a">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Martin Vogt</span> - <span class="hlFld-Affiliation affiliation">Department of Life Science Informatics,
B-IT, LIMES Program Unit Chemical Biology and Medicinal Chemistry, Rheinische Friedrich-Wilhelms-Universität, Endenicher Allee 19c, D-53115 Bonn, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dimitar Yonchev</span> - <span class="hlFld-Affiliation affiliation">Department of Life Science Informatics,
B-IT, LIMES Program Unit Chemical Biology and Medicinal Chemistry, Rheinische Friedrich-Wilhelms-Universität, Endenicher Allee 19c, D-53115 Bonn, Germany</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><p>The authors declare no competing financial interest.<br /></br></p></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i26">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_37787" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_37787" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The authors thank OpenEye Scientific Software for an academic license and Pat Walters of Relay Therapeutics for the implementation of the ESOL method made available on GitHub (<a href="https://github.com/PatWalters/solubility" class="extLink">https://github.com/PatWalters/solubility</a>). D.Y. is a fellow of the Jürgen Manchot Stiftung, Düsseldorf, Germany.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">AS</td><td class="NLM_def"><p class="first last">analog series</p></td></tr><tr><td class="NLM_term">ASB</td><td class="NLM_def"><p class="first last">analog series based</p></td></tr><tr><td class="NLM_term">LO</td><td class="NLM_def"><p class="first last">lead optimization</p></td></tr><tr><td class="NLM_term">MMP</td><td class="NLM_def"><p class="first last">matched molecular pair</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship.</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i28">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_78308" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_78308" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 20 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span> <i>The
Practice of Medicinal Chemistry</i>, <span class="NLM_edition">3</span>rd ed,; <span class="NLM_contrib-group"><span class="NLM_string-name">Wermuth, C. G.</span></span>, Ed.; <span class="NLM_publisher-name">Academic Press-Elsevier</span>: <span class="NLM_publisher-loc">Burlington, San Diego, USA, London, U.K.</span>, <span class="NLM_year">2008</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2008&author=C.+G.+Wermuth&title=The%0APractice+of+Medicinal+Chemistry"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DThe%250APractice%2520of%2520Medicinal%2520Chemistry%26aulast%3DWermuth%26aufirst%3DC.%2BG.%26pub%3DAcademic%2520Press-Elsevier%26date%3D2008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cherkasov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muratov, E. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fourches, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varnek, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baskin, I. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cronin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dearden, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gramatica, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Todeschini, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Consonni, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuz’min, V. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cramer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benigni, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rathman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terfloth, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gasteiger, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richard, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tropsha, A.</span></span> <span> </span><span class="NLM_article-title">QSAR Modeling: Where Have You Been? Where Are You Going to?</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">4977</span>– <span class="NLM_lpage">5010</span>, <span class="refDoi"> DOI: 10.1021/jm4004285</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm4004285" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01626&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFCrsLnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=4977-5010&author=A.+Cherkasovauthor=E.+N.+Muratovauthor=D.+Fourchesauthor=A.+Varnekauthor=I.+I.+Baskinauthor=M.+Croninauthor=J.+Deardenauthor=P.+Gramaticaauthor=Y.+C.+Martinauthor=R.+Todeschiniauthor=V.+Consonniauthor=V.+E.+Kuz%E2%80%99minauthor=R.+Cramerauthor=R.+Benigniauthor=C.+Yangauthor=J.+Rathmanauthor=L.+Terflothauthor=J.+Gasteigerauthor=A.+Richardauthor=A.+Tropsha&title=QSAR+Modeling%3A+Where+Have+You+Been%3F+Where+Are+You+Going+to%3F&doi=10.1021%2Fjm4004285"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">QSAR Modeling: Where Have You Been? Where Are You Going To?</span></div><div class="casAuthors">Cherkasov, Artem; Muratov, Eugene N.; Fourches, Denis; Varnek, Alexandre; Baskin, Igor I.; Cronin, Mark; Dearden, John; Gramatica, Paola; Martin, Yvonne C.; Todeschini, Roberto; Consonni, Viviana; Kuzmin, Victor E.; Cramer, Richard; Benigni, Romualdo; Yang, Chihae; Rathman, James; Terfloth, Lothar; Gasteiger, Johann; Richard, Ann; Tropsha, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4977-5010</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Quant. structure-activity relationship modeling is one of the major computational tools employed in medicinal chem.  However, throughout its entire history it has drawn both praise and criticism concerning its reliability, limitations, successes, and failures.  In this paper, we discuss (i) the development and evolution of QSAR; (ii) the current trends, unsolved problems, and pressing challenges; and (iii) several novel and emerging applications of QSAR modeling.  Throughout this discussion, we provide guidelines for QSAR development, validation, and application, which are summarized in best practices for building rigorously validated and externally predictive QSAR models.  We hope that this Perspective will help communications between computational and exptl. chemists toward collaborative development and use of QSAR models.  We also believe that the guidelines presented here will help journal editors and reviewers apply more stringent scientific stds. to manuscripts reporting new QSAR studies, as well as encourage the use of high quality, validated QSARs for regulatory decision making.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbLWydJbspbLVg90H21EOLACvtfcHk0ljpI6KuZyxs6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFCrsLnE&md5=a2d371ee33fbecf48ef59ecf914e4aa8</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1021%2Fjm4004285&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm4004285%26sid%3Dliteratum%253Aachs%26aulast%3DCherkasov%26aufirst%3DA.%26aulast%3DMuratov%26aufirst%3DE.%2BN.%26aulast%3DFourches%26aufirst%3DD.%26aulast%3DVarnek%26aufirst%3DA.%26aulast%3DBaskin%26aufirst%3DI.%2BI.%26aulast%3DCronin%26aufirst%3DM.%26aulast%3DDearden%26aufirst%3DJ.%26aulast%3DGramatica%26aufirst%3DP.%26aulast%3DMartin%26aufirst%3DY.%2BC.%26aulast%3DTodeschini%26aufirst%3DR.%26aulast%3DConsonni%26aufirst%3DV.%26aulast%3DKuz%25E2%2580%2599min%26aufirst%3DV.%2BE.%26aulast%3DCramer%26aufirst%3DR.%26aulast%3DBenigni%26aufirst%3DR.%26aulast%3DYang%26aufirst%3DC.%26aulast%3DRathman%26aufirst%3DJ.%26aulast%3DTerfloth%26aufirst%3DL.%26aulast%3DGasteiger%26aufirst%3DJ.%26aulast%3DRichard%26aufirst%3DA.%26aulast%3DTropsha%26aufirst%3DA.%26atitle%3DQSAR%2520Modeling%253A%2520Where%2520Have%2520You%2520Been%253F%2520Where%2520Are%2520You%2520Going%2520to%253F%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D4977%26epage%3D5010%26doi%3D10.1021%2Fjm4004285" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Segall, M.</span></span> <span> </span><span class="NLM_article-title">Advances in Multi-Parameter Optimization Methods for <i>De Novo</i> Drug Design</span>. <i>Expert Opin. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">803</span>– <span class="NLM_lpage">817</span>, <span class="refDoi"> DOI: 10.1517/17460441.2014.913565</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01626&amp;key=10.1517%2F17460441.2014.913565" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=803-817&author=M.+Segall&title=Advances+in+Multi-Parameter+Optimization+Methods+for+De+Novo+Drug+Design&doi=10.1517%2F17460441.2014.913565"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1517%2F17460441.2014.913565&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F17460441.2014.913565%26sid%3Dliteratum%253Aachs%26aulast%3DSegall%26aufirst%3DM.%26atitle%3DAdvances%2520in%2520Multi-Parameter%2520Optimization%2520Methods%2520for%2520De%2520Novo%2520Drug%2520Design%26jtitle%3DExpert%2520Opin.%2520Drug%2520Discovery%26date%3D2014%26volume%3D9%26spage%3D803%26epage%3D817%26doi%3D10.1517%2F17460441.2014.913565" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Munson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lieberman, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tserlin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rocnik, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ge, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fitzgerald, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia-Echeverria, C.</span></span> <span> </span><span class="NLM_article-title">Lead Optimization Attrition Analysis (LOAA): A Novel and General Methodology for Medicinal Chemistry</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">978</span>– <span class="NLM_lpage">987</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2015.03.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01626&amp;key=10.1016%2Fj.drudis.2015.03.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01626&amp;key=25814036" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01626&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlsV2gs7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2015&pages=978-987&author=M.+Munsonauthor=H.+Liebermanauthor=E.+Tserlinauthor=J.+Rocnikauthor=J.+Geauthor=M.+Fitzgeraldauthor=V.+Patelauthor=C.+Garcia-Echeverria&title=Lead+Optimization+Attrition+Analysis+%28LOAA%29%3A+A+Novel+and+General+Methodology+for+Medicinal+Chemistry&doi=10.1016%2Fj.drudis.2015.03.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Lead optimization attrition analysis (LOAA): a novel and general methodology for medicinal chemistry</span></div><div class="casAuthors">Munson, Mark; Lieberman, Harvey; Tserlin, Elina; Rocnik, Jennifer; Ge, Jie; Fitzgerald, Maria; Patel, Vinod; Garcia-Echeverria, Carlos</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">978-987</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Herein, we report a novel and general method, lead optimization attrition anal. (LOAA), to benchmark two distinct small-mol. lead series using a relatively unbiased, simple technique and com. available software.  We illustrate this approach with data collected during lead optimization of two independent oncol. programs as a case study.  Easily generated graphics and attrition curves enabled us to calibrate progress and support go/no go decisions on each program.  We believe that this data-driven technique could be used broadly by medicinal chemists and management to guide strategic decisions during drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8Es9dpgoC2bVg90H21EOLACvtfcHk0lh_cSExKMziwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlsV2gs7Y%253D&md5=52831d6178321347e32161d61aea978c</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2015.03.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2015.03.010%26sid%3Dliteratum%253Aachs%26aulast%3DMunson%26aufirst%3DM.%26aulast%3DLieberman%26aufirst%3DH.%26aulast%3DTserlin%26aufirst%3DE.%26aulast%3DRocnik%26aufirst%3DJ.%26aulast%3DGe%26aufirst%3DJ.%26aulast%3DFitzgerald%26aufirst%3DM.%26aulast%3DPatel%26aufirst%3DV.%26aulast%3DGarcia-Echeverria%26aufirst%3DC.%26atitle%3DLead%2520Optimization%2520Attrition%2520Analysis%2520%2528LOAA%2529%253A%2520A%2520Novel%2520and%2520General%2520Methodology%2520for%2520Medicinal%2520Chemistry%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2015%26volume%3D20%26spage%3D978%26epage%3D987%26doi%3D10.1016%2Fj.drudis.2015.03.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kunimoto, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyao, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span> <span> </span><span class="NLM_article-title">Computational Method for Estimating Progression Saturation of Analog Series</span>. <i>RSC Adv.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">5484</span>– <span class="NLM_lpage">5492</span>, <span class="refDoi"> DOI: 10.1039/c7ra13748f</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01626&amp;key=10.1039%2Fc7ra13748f" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01626&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvFantLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&pages=5484-5492&author=R.+Kunimotoauthor=T.+Miyaoauthor=J.+Bajorath&title=Computational+Method+for+Estimating+Progression+Saturation+of+Analog+Series&doi=10.1039%2Fc7ra13748f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Computational method for estimating progression saturation of analog series</span></div><div class="casAuthors">Kunimoto, Ryo; Miyao, Tomoyuki; Bajorath, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">RSC Advances</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">5484-5492</span>CODEN:
                <span class="NLM_cas:coden">RSCACL</span>;
        ISSN:<span class="NLM_cas:issn">2046-2069</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">In lead optimization, it is difficult to est. when an analog series might be satd. and synthesis of addnl. compds. would be unlikely to yield further progress.  Rather than terminating a series, one often continues to generate analogs hoping to reach the final optimization goal, even if obstacles that are faced ultimately prove to be unsurmountable.  Clearly, methodologies to better understand series progression and satn. are highly desirable.  However, only a few approaches are currently available to monitor series progression and aid in decision making.  Herein, we introduce a new computational method to assess progression satn. of an analog series by relating the properties of existing compds. to those of synthetic candidates and comparing their distributions in chem. space.  The neighborhoods of analogs are analyzed and the distance relationships between existing and candidate compds. quantified.  An intuitive dual scoring scheme makes it possible to characterize analog series and their degree of progression satn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGonQeSyaIHLVg90H21EOLACvtfcHk0lh_cSExKMziwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvFantLY%253D&md5=3d944bfe016a757fc1e105d4cc45c1bd</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1039%2Fc7ra13748f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc7ra13748f%26sid%3Dliteratum%253Aachs%26aulast%3DKunimoto%26aufirst%3DR.%26aulast%3DMiyao%26aufirst%3DT.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DComputational%2520Method%2520for%2520Estimating%2520Progression%2520Saturation%2520of%2520Analog%2520Series%26jtitle%3DRSC%2520Adv.%26date%3D2018%26volume%3D8%26spage%3D5484%26epage%3D5492%26doi%3D10.1039%2Fc7ra13748f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yonchev, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stumpfe, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kunimoto, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyao, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span> <span> </span><span class="NLM_article-title">Computational Assessment of Chemical Saturation of Analog Series under Varying Conditions</span>. <i>ACS Omega</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">15799</span>– <span class="NLM_lpage">15808</span>, <span class="refDoi"> DOI: 10.1021/acsomega.8b02087</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsomega.8b02087" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2018&pages=15799-15808&author=D.+Yonchevauthor=M.+Vogtauthor=D.+Stumpfeauthor=R.+Kunimotoauthor=T.+Miyaoauthor=J.+Bajorath&title=Computational+Assessment+of+Chemical+Saturation+of+Analog+Series+under+Varying+Conditions&doi=10.1021%2Facsomega.8b02087"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Facsomega.8b02087&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsomega.8b02087%26sid%3Dliteratum%253Aachs%26aulast%3DYonchev%26aufirst%3DD.%26aulast%3DVogt%26aufirst%3DM.%26aulast%3DStumpfe%26aufirst%3DD.%26aulast%3DKunimoto%26aufirst%3DR.%26aulast%3DMiyao%26aufirst%3DT.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DComputational%2520Assessment%2520of%2520Chemical%2520Saturation%2520of%2520Analog%2520Series%2520under%2520Varying%2520Conditions%26jtitle%3DACS%2520Omega%26date%3D2018%26volume%3D3%26spage%3D15799%26epage%3D15808%26doi%3D10.1021%2Facsomega.8b02087" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wassermann, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wawer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span> <span> </span><span class="NLM_article-title">Activity Landscape Representations for Structure-Activity Relationship Analysis</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">8209</span>– <span class="NLM_lpage">8223</span>, <span class="refDoi"> DOI: 10.1021/jm100933w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm100933w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01626&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFKmt7bF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=8209-8223&author=A.+M.+Wassermannauthor=M.+Wawerauthor=J.+Bajorath&title=Activity+Landscape+Representations+for+Structure-Activity+Relationship+Analysis&doi=10.1021%2Fjm100933w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Activity Landscape Representations for Structure-Activity Relationship Analysis</span></div><div class="casAuthors">Wassermann, Anne Mai; Wawer, Mathias; Bajorath, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">8209-8223</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The study of compd. structure-activity relationships (SARs) is one of the central themes in medicinal chem.  SAR information is analyzed in different contexts, from screening and hit-to-lead to lead optimization projects.  For the exploration of SARs, the concept of an activity landscape, which integrates mol. similarity and potency information, is of high relevance.  The computational study of activity landscapes is still an evolving field.  Activity landscape models are designed to rationalize SAR features of compd. data sets and select key compds. for chem. exploration.  The choice of mol. representations and the way mol. similarity is assessed are critically important factors for landscape generation and anal.  Graphical representation of SAR features is a major focal point of landscape modeling.  Although complex activity landscapes are generally difficult to analyze, much progress has recently been made in extg. SAR information from various landscape views.  This article aims to provide an overview of the state-of-the-art in activity landscape anal. and a discussion of its potential for medicinal chem. applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBe8xXwkO-_bVg90H21EOLACvtfcHk0lidxGoL-jW_PA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFKmt7bF&md5=4867772191beabe154c8d221640c956a</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Fjm100933w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100933w%26sid%3Dliteratum%253Aachs%26aulast%3DWassermann%26aufirst%3DA.%2BM.%26aulast%3DWawer%26aufirst%3DM.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DActivity%2520Landscape%2520Representations%2520for%2520Structure-Activity%2520Relationship%2520Analysis%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D8209%26epage%3D8223%26doi%3D10.1021%2Fjm100933w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Iyer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span> <span> </span><span class="NLM_article-title">SAR Monitoring of Evolving Compound Data Sets Using Activity Landscapes</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">532</span>– <span class="NLM_lpage">540</span>, <span class="refDoi"> DOI: 10.1021/ci100505m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci100505m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01626&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhvVylsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2011&pages=532-540&author=P.+Iyerauthor=Y.+Huauthor=J.+Bajorath&title=SAR+Monitoring+of+Evolving+Compound+Data+Sets+Using+Activity+Landscapes&doi=10.1021%2Fci100505m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">SAR Monitoring of Evolving Compound Data Sets Using Activity Landscapes</span></div><div class="casAuthors">Iyer, Preeti; Hu, Ye; Bajorath, Jurgen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">532-540</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In pharmaceutical research, collections of active compds. directed against specific therapeutic targets usually evolve over time.  Small mol. discovery is an iterative process.  New compds. are discovered, alternative compd. series explored, some series discontinued, and others prioritized.  The design of new compds. usually takes into consideration prior chem. and structure-activity relationship (SAR) knowledge.  Hence, historically grown compd. collections represent a viable source of chem. and SAR information that might be utilized to retrospectively analyze roadblocks in compd. optimization and further guide discovery projects.  However, SAR anal. of large and heterogeneous sets of active compds. is also principally complicated.  We have subjected evolving compd. data sets to SAR monitoring using activity landscape models in order to evaluate how compn. and SAR characteristics might change over time.  Chemotype and potency distributions in evolving data sets directed against different therapeutic targets were analyzed and alternative activity landscape representations generated at different points in time to monitor the progression of global and local SAR features.  Our results show that the evolving data sets studied here have predominantly grown around seed clusters of active compds. that often emerged early on, while other SAR islands remained largely unexplored.  Moreover, increasing scaffold diversity in evolving data sets did not necessarily yield new SAR patterns, indicating a rather significant influence of "me-too-ism" (i.e., introducing new chemotypes that are similar to already known ones) on the compn. and SAR information content of the data sets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKpgeL5xQDQLVg90H21EOLACvtfcHk0lidxGoL-jW_PA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhvVylsb4%253D&md5=85d5290d13c5c9aede37f5360385419c</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Fci100505m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100505m%26sid%3Dliteratum%253Aachs%26aulast%3DIyer%26aufirst%3DP.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DSAR%2520Monitoring%2520of%2520Evolving%2520Compound%2520Data%2520Sets%2520Using%2520Activity%2520Landscapes%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2011%26volume%3D51%26spage%3D532%26epage%3D540%26doi%3D10.1021%2Fci100505m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Peltason, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span> <span> </span><span class="NLM_article-title">SAR Index: Quantifying the Nature of Structure-Activity Relationships</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">5571</span>– <span class="NLM_lpage">5578</span>, <span class="refDoi"> DOI: 10.1021/jm0705713</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0705713" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01626&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtV2msLfP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2007&pages=5571-5578&author=L.+Peltasonauthor=J.+Bajorath&title=SAR+Index%3A+Quantifying+the+Nature+of+Structure-Activity+Relationships&doi=10.1021%2Fjm0705713"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">SAR index: quantifying the nature of structure-activity relationships</span></div><div class="casAuthors">Peltason, Lisa; Bajorath, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">5571-5578</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Structure-activity relationships (SARs) can display very different features.  Small chem. modifications of active mols. often dramatically alter biol. responses.  By contrast, structurally diverse mols. can have similar activity.  SARs can also be heterogeneous in nature.  For example, for structurally diverse mols. with similar activity, closely related analogs might have significant differences in potency.  Given the inherent complexity of SARs, it has been very difficult to est. SAR characteristics from mol. structure.  On the basis of systematic correlation of 2D structural similarity and compd. potency, we have developed a function termed "SAR Index" that quant. describes the nature of SARs and establishes different SAR categories: continuous, discontinuous, heterogeneous-relaxed, and heterogeneous-constrained.  These heterogeneous SAR categories are described for the first time.  Given a set of active compds. and their potency values, SAR Index calcns. can est. how likely it is to identify structurally distinct mols. having similar activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhD6NtNDFPBbVg90H21EOLACvtfcHk0lidxGoL-jW_PA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtV2msLfP&md5=74c7fadf125c6d660cb24a54cbf44dc1</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Fjm0705713&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0705713%26sid%3Dliteratum%253Aachs%26aulast%3DPeltason%26aufirst%3DL.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DSAR%2520Index%253A%2520Quantifying%2520the%2520Nature%2520of%2520Structure-Activity%2520Relationships%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2007%26volume%3D50%26spage%3D5571%26epage%3D5578%26doi%3D10.1021%2Fjm0705713" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shanmugasundaram, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kayastha, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de la Vega de León, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dimova, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span> <span> </span><span class="NLM_article-title">Monitoring the Progression of Structure–Activity Relationship Information during Lead Optimization</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">4235</span>– <span class="NLM_lpage">4244</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01428</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01428" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2015&pages=4235-4244&author=V.+Shanmugasundaramauthor=L.+Zhangauthor=S.+Kayasthaauthor=A.+de+la+Vega+de+Le%C3%B3nauthor=D.+Dimovaauthor=J.+Bajorath&title=Monitoring+the+Progression+of+Structure%E2%80%93Activity+Relationship+Information+during+Lead+Optimization&doi=10.1021%2Facs.jmedchem.5b01428"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01428&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01428%26sid%3Dliteratum%253Aachs%26aulast%3DShanmugasundaram%26aufirst%3DV.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DKayastha%26aufirst%3DS.%26aulast%3Dde%2Bla%2BVega%2Bde%2BLe%25C3%25B3n%26aufirst%3DA.%26aulast%3DDimova%26aufirst%3DD.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DMonitoring%2520the%2520Progression%2520of%2520Structure%25E2%2580%2593Activity%2520Relationship%2520Information%2520during%2520Lead%2520Optimization%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D59%26spage%3D4235%26epage%3D4244%26doi%3D10.1021%2Facs.jmedchem.5b01428" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maynard, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, C. D.</span></span> <span> </span><span class="NLM_article-title">Quantifying, Visualizing, and Monitoring Lead Optimization</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">4189</span>– <span class="NLM_lpage">4201</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00948</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00948" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2015&pages=4189-4201&author=A.+T.+Maynardauthor=C.+D.+Roberts&title=Quantifying%2C+Visualizing%2C+and+Monitoring+Lead+Optimization&doi=10.1021%2Facs.jmedchem.5b00948"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00948&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00948%26sid%3Dliteratum%253Aachs%26aulast%3DMaynard%26aufirst%3DA.%2BT.%26aulast%3DRoberts%26aufirst%3DC.%2BD.%26atitle%3DQuantifying%252C%2520Visualizing%252C%2520and%2520Monitoring%2520Lead%2520Optimization%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D59%26spage%3D4189%26epage%3D4201%26doi%3D10.1021%2Facs.jmedchem.5b00948" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stumpfe, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span> <span> </span><span class="NLM_article-title">Exploring Activity Cliffs in Medicinal Chemistry</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">2932</span>– <span class="NLM_lpage">2942</span>, <span class="refDoi"> DOI: 10.1021/jm201706b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm201706b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01626&amp;key=1%3ACAS%3A528%3ADC%252BC38XmtFajsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=2932-2942&author=D.+Stumpfeauthor=J.+Bajorath&title=Exploring+Activity+Cliffs+in+Medicinal+Chemistry&doi=10.1021%2Fjm201706b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Exploring Activity Cliffs in Medicinal Chemistry</span></div><div class="casAuthors">Stumpfe, Dagmar; Bajorath, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2932-2942</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In recent years, structure-activity relationships (SARs) have increasingly been studied through mining of large compd. data sets, leading to a renaissance of the activity landscape concept.  In general terms, an activity landscape is defined as any graphical representation that integrates similarity and potency relationships between compds. sharing a specific activity.  From activity landscape representations of different design, both global and local SAR features present in compd. data sets can be extd.  Modeling and rationalizing activity landscapes require the application of computational methods.  As such, the activity landscape concept is of comparable relevance for chemoinformatics (having a strong focus on data mining and representation) and medicinal chem. (given its immediate relevance for SAR anal.).  Herein, we describe in detail the multifaceted nature of activity cliffs, discuss underlying scientific concepts, and explain how their individual or systematic anal. might (or might not) provide useful information for medicinal chem. programs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGql1wydp65qQLVg90H21EOLACvtfcHk0lgIh8AijyI_bA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmtFajsQ%253D%253D&md5=ceddd1de5798b585d6381ac18ba67fc9</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Fjm201706b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm201706b%26sid%3Dliteratum%253Aachs%26aulast%3DStumpfe%26aufirst%3DD.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DExploring%2520Activity%2520Cliffs%2520in%2520Medicinal%2520Chemistry%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D2932%26epage%3D2942%26doi%3D10.1021%2Fjm201706b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span> </span><span class="NLM_article-title">RDKit: Cheminformatics
and Machine Learning Software, 2013</span>. <a href="http://www.rdkit.org" class="extLink">http://www.rdkit.org</a> (accessed
July 1, <span class="NLM_year">2018</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=RDKit%3A+Cheminformatics%0Aand+Machine+Learning+Software%2C+2013.+http%3A%2F%2Fwww.rdkit.org+%28accessed%0AJuly+1%2C+2018%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26atitle%3DRDKit%253A%2520Cheminformatics%250Aand%2520Machine%2520Learning%2520Software%252C%25202013%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Delaney, J.
S.</span></span> <span> </span><span class="NLM_article-title">ESOL: Estimating Aqueous Solubility Directly from Molecular Structure</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">1000</span>– <span class="NLM_lpage">1005</span>, <span class="refDoi"> DOI: 10.1021/ci034243x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci034243x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2004&pages=1000-1005&author=J.%0AS.+Delaney&title=ESOL%3A+Estimating+Aqueous+Solubility+Directly+from+Molecular+Structure&doi=10.1021%2Fci034243x"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fci034243x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci034243x%26sid%3Dliteratum%253Aachs%26aulast%3DDelaney%26aufirst%3DJ.%2BS.%26atitle%3DESOL%253A%2520Estimating%2520Aqueous%2520Solubility%2520Directly%2520from%2520Molecular%2520Structure%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2004%26volume%3D44%26spage%3D1000%26epage%3D1005%26doi%3D10.1021%2Fci034243x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bento, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaulton, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hersey, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellis, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chambers, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krüger, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Light, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mak, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGlinchey, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowotka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papadatos, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santos, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overington, J. P.</span></span> <span> </span><span class="NLM_article-title">The ChEMBL Bioactivity Database: An Update</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">D1083</span>– <span class="NLM_lpage">D1090</span>, <span class="refDoi"> DOI: 10.1093/nar/gkt1031</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01626&amp;key=10.1093%2Fnar%2Fgkt1031" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01626&amp;key=24214965" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01626&amp;key=1%3ACAS%3A528%3ADC%252BC2cXoslWl" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2014&pages=D1083-D1090&author=A.+P.+Bentoauthor=A.+Gaultonauthor=A.+Herseyauthor=L.+J.+Bellisauthor=J.+Chambersauthor=M.+Daviesauthor=F.+A.+Kr%C3%BCgerauthor=Y.+Lightauthor=L.+Makauthor=S.+McGlincheyauthor=M.+Nowotkaauthor=G.+Papadatosauthor=R.+Santosauthor=J.+P.+Overington&title=The+ChEMBL+Bioactivity+Database%3A+An+Update&doi=10.1093%2Fnar%2Fgkt1031"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">The ChEMBL bioactivity database: an update</span></div><div class="casAuthors">Bento, A. Patricia; Gaulton, Anna; Hersey, Anne; Bellis, Louisa J.; Chambers, Jon; Davies, Mark; Krueger, Felix A.; Light, Yvonne; Mak, Lora; McGlinchey, Shaun; Nowotka, Michal; Papadatos, George; Santos, Rita; Overington, John P.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D1083-D1090</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">ChEMBL is an open large-scale bioactivity database (https://www.ebi.ac.uk/chembl), previously described in the 2012 Nucleic Acids Research Database Issue.  Since then, a variety of new data sources and improvements in functionality have contributed to the growth and utility of the resource.  In particular, more comprehensive tracking of compds. from research stages through clin. development to market is provided through the inclusion of data from United States Adopted Name applications; a new richer data model for representing drug targets has been developed; and a no. of methods have been put in place to allow users to more easily identify reliable data.  Finally, access to ChEMBL is now available via a new Resource Description Framework format, in addn. to the web-based interface, data downloads and web services.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXb5k1IvJNaLVg90H21EOLACvtfcHk0lgIh8AijyI_bA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXoslWl&md5=31b832d03d56ea3065d7aa29618362bc</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkt1031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkt1031%26sid%3Dliteratum%253Aachs%26aulast%3DBento%26aufirst%3DA.%2BP.%26aulast%3DGaulton%26aufirst%3DA.%26aulast%3DHersey%26aufirst%3DA.%26aulast%3DBellis%26aufirst%3DL.%2BJ.%26aulast%3DChambers%26aufirst%3DJ.%26aulast%3DDavies%26aufirst%3DM.%26aulast%3DKr%25C3%25BCger%26aufirst%3DF.%2BA.%26aulast%3DLight%26aufirst%3DY.%26aulast%3DMak%26aufirst%3DL.%26aulast%3DMcGlinchey%26aufirst%3DS.%26aulast%3DNowotka%26aufirst%3DM.%26aulast%3DPapadatos%26aufirst%3DG.%26aulast%3DSantos%26aufirst%3DR.%26aulast%3DOverington%26aufirst%3DJ.%2BP.%26atitle%3DThe%2520ChEMBL%2520Bioactivity%2520Database%253A%2520An%2520Update%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2014%26volume%3D42%26spage%3DD1083%26epage%3DD1090%26doi%3D10.1093%2Fnar%2Fgkt1031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stumpfe, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dimova, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span> <span> </span><span class="NLM_article-title">Computational Method for Systematic Identification of Analog Series and Key Compounds Representing Series and Their Biological Activity Profiles</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">7667</span>– <span class="NLM_lpage">7676</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00906</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00906" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=7667-7676&author=D.+Stumpfeauthor=D.+Dimovaauthor=J.+Bajorath&title=Computational+Method+for+Systematic+Identification+of+Analog+Series+and+Key+Compounds+Representing+Series+and+Their+Biological+Activity+Profiles&doi=10.1021%2Facs.jmedchem.6b00906"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00906%26sid%3Dliteratum%253Aachs%26aulast%3DStumpfe%26aufirst%3DD.%26aulast%3DDimova%26aufirst%3DD.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DComputational%2520Method%2520for%2520Systematic%2520Identification%2520of%2520Analog%2520Series%2520and%2520Key%2520Compounds%2520Representing%2520Series%2520and%2520Their%2520Biological%2520Activity%2520Profiles%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D7667%26epage%3D7676%26doi%3D10.1021%2Facs.jmedchem.6b00906" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kenny, P. W.</span>; <span class="NLM_string-name">Sadowski, J.</span></span> <span> </span><span class="NLM_article-title">Structure Modification in Chemical Databases</span>. In  <i>Chemoinformatics in Drug Discovery</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oprea, T. I.</span></span>, Ed.; <span class="NLM_publisher-name">Wiley-VCH</span>: <span class="NLM_publisher-loc">Weinheim, Germany</span>, <span class="NLM_year">2004</span>; pp  <span class="NLM_fpage">271</span>– <span class="NLM_lpage">285</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2004&pages=271-285&author=P.+W.+Kenny&author=J.+Sadowskiauthor=T.+I.+Oprea&title=Chemoinformatics+in+Drug+Discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DKenny%26aufirst%3DP.%2BW.%26atitle%3DStructure%2520Modification%2520in%2520Chemical%2520Databases%26btitle%3DChemoinformatics%2520in%2520Drug%2520Discovery%26aulast%3DOprea%26aufirst%3DT.%2BI.%26pub%3DWiley-VCH%26date%3D2004%26spage%3D271%26epage%3D285" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dimova, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span> <span> </span><span class="NLM_article-title">Computational Design of New Molecular Scaffolds for Medicinal Chemistry, Part II: Generalization of Analog Series-Based Scaffolds</span>. <i>Future Sci. OA</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">FSO287</span>, <span class="refDoi"> DOI: 10.4155/fsoa-2017-0135</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01626&amp;key=10.4155%2Ffsoa-2017-0135" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01626&amp;key=29682322" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2018&pages=FSO287&author=D.+Dimovaauthor=J.+Bajorath&title=Computational+Design+of+New+Molecular+Scaffolds+for+Medicinal+Chemistry%2C+Part+II%3A+Generalization+of+Analog+Series-Based+Scaffolds&doi=10.4155%2Ffsoa-2017-0135"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.4155%2Ffsoa-2017-0135&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffsoa-2017-0135%26sid%3Dliteratum%253Aachs%26aulast%3DDimova%26aufirst%3DD.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DComputational%2520Design%2520of%2520New%2520Molecular%2520Scaffolds%2520for%2520Medicinal%2520Chemistry%252C%2520Part%2520II%253A%2520Generalization%2520of%2520Analog%2520Series-Based%2520Scaffolds%26jtitle%3DFuture%2520Sci.%2520OA%26date%3D2018%26volume%3D4%26spage%3DFSO287%26doi%3D10.4155%2Ffsoa-2017-0135" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stumpfe, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span> <span> </span><span class="NLM_article-title">MMP-Cliffs: Systematic Identification of Activity Cliffs on the Basis of Matched Molecular Pairs</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">1138</span>– <span class="NLM_lpage">1145</span>, <span class="refDoi"> DOI: 10.1021/ci3001138</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci3001138" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01626&amp;key=1%3ACAS%3A528%3ADC%252BC38Xlt1GgtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=1138-1145&author=X.+Huauthor=Y.+Huauthor=M.+Vogtauthor=D.+Stumpfeauthor=J.+Bajorath&title=MMP-Cliffs%3A+Systematic+Identification+of+Activity+Cliffs+on+the+Basis+of+Matched+Molecular+Pairs&doi=10.1021%2Fci3001138"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">MMP-Cliffs: Systematic Identification of Activity Cliffs on the Basis of Matched Molecular Pairs</span></div><div class="casAuthors">Hu, Xiaoying; Hu, Ye; Vogt, Martin; Stumpfe, Dagmar; Bajorath, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1138-1145</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Activity cliffs are generally defined as pairs of structurally similar compds. having large differences in potency.  The anal. of activity cliffs is of general interest because structure-activity relationship (SAR) determinants can often be deduced from them.  Crit. questions for the study of activity cliffs include how similar compds. should be to qualify as cliff partners, how similarity should be assessed, and how large potency differences between participating compds. should be.  Thus far, activity cliffs have mostly been defined on the basis of calcd. Tanimoto similarity values using structural descriptors, esp. 2D fingerprints.  As any theor. assessment of mol. similarity, this approach has its limitations.  For example, calcd. Tanimoto similarities might often be difficult to reconcile and interpret from a chem. perspective, a point of critique frequently raised in medicinal chem.  Herein, we have explored activity cliffs by considering well-defined substructure replacements instead of calcd. similarity values.  For this purpose, the matched mol. pair (MMP) formalism has been applied.  MMPs were systematically derived from public domain compds., and activity cliffs were extd. from them, termed MMP-cliffs.  The frequency of cliff formation was detd. for compds. active against different targets, MMP-cliffs were analyzed in detail, and re-evaluated on the basis of Tanimoto similarity.  In many instances, chem. intuitive activity cliffs were only detected on the basis of MMPs, but not Tanimoto similarity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCYSOxNqPjGbVg90H21EOLACvtfcHk0lgYKdXlTVJh8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xlt1GgtLg%253D&md5=958a76d85d0a06424b43eb1b694cf3ca</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fci3001138&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci3001138%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DX.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DVogt%26aufirst%3DM.%26aulast%3DStumpfe%26aufirst%3DD.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DMMP-Cliffs%253A%2520Systematic%2520Identification%2520of%2520Activity%2520Cliffs%2520on%2520the%2520Basis%2520of%2520Matched%2520Molecular%2520Pairs%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D1138%26epage%3D1145%26doi%3D10.1021%2Fci3001138" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span> <i>OEChem TK</i>, version 1.7.7; <span class="NLM_publisher-name">OpenEye
Scientific Software, Inc.</span>: <span class="NLM_publisher-loc">Santa Fe,
NM, USA</span>, <span class="NLM_year">2012</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+OEChem+TK%2C+version+1.7.7%3B+OpenEye%0AScientific+Software%2C+Inc.%3A+Santa+Fe%2C%0ANM%2C+USA%2C+2012."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DOEChem%2520TK%26pub%3DOpenEye%250AScientific%2520Software%252C%2520Inc%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 8 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Atsushi Yoshimori, Toru Tanoue, <span class="NLM_string-name hlFld-ContribAuthor">Jürgen Bajorath</span>. </span><span class="cited-content_cbyCitation_article-title">Integrating the Structure–Activity Relationship Matrix Method with Molecular Grid Maps and Activity Landscape Models for Medicinal Chemistry Applications. </span><span class="cited-content_cbyCitation_journal-name">ACS Omega</span><span> <strong>2019,</strong> <em>4 </em>
                                    (4)
                                     , 7061-7069. <a href="https://doi.org/10.1021/acsomega.9b00595" title="DOI URL">https://doi.org/10.1021/acsomega.9b00595</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsomega.9b00595&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsomega.9b00595%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Omega%26atitle%3DIntegrating%252Bthe%252BStructure%2525E2%252580%252593Activity%252BRelationship%252BMatrix%252BMethod%252Bwith%252BMolecular%252BGrid%252BMaps%252Band%252BActivity%252BLandscape%252BModels%252Bfor%252BMedicinal%252BChemistry%252BApplications%26aulast%3DYoshimori%26aufirst%3DAtsushi%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D03032019%26date%3D08042019%26volume%3D4%26issue%3D4%26spage%3D7061%26epage%3D7069" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Atsushi  Yoshimori</span>, <span class="hlFld-ContribAuthor ">Huabin  Hu</span>, <span class="hlFld-ContribAuthor ">Jürgen  Bajorath</span>. </span><span class="cited-content_cbyCitation_article-title">Adapting the DeepSARM approach for dual-target ligand design. </span><span class="cited-content_cbyCitation_journal-name">Journal of Computer-Aided Molecular Design</span><span> <strong>2021,</strong> <em>35 </em>
                                    (5)
                                     , 587-600. <a href="https://doi.org/10.1007/s10822-021-00379-5" title="DOI URL">https://doi.org/10.1007/s10822-021-00379-5</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10822-021-00379-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10822-021-00379-5%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Computer-Aided%2520Molecular%2520Design%26atitle%3DAdapting%252Bthe%252BDeepSARM%252Bapproach%252Bfor%252Bdual-target%252Bligand%252Bdesign%26aulast%3DYoshimori%26aufirst%3DAtsushi%26date%3D2021%26date%3D2021%26volume%3D35%26issue%3D5%26spage%3D587%26epage%3D600" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Bonggun  Shin</span>, <span class="hlFld-ContribAuthor ">Sungsoo  Park</span>, <span class="hlFld-ContribAuthor ">JinYeong  Bak</span>, <span class="hlFld-ContribAuthor ">Joyce C.  Ho</span>. </span><span class="cited-content_cbyCitation_article-title">Controlled molecule generator for optimizing multiple chemical properties. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 146-153. <a href="https://doi.org/10.1145/3450439.3451879" title="DOI URL">https://doi.org/10.1145/3450439.3451879</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1145/3450439.3451879&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1145%2F3450439.3451879%26sid%3Dliteratum%253Aachs%26atitle%3DControlled%252Bmolecule%252Bgenerator%252Bfor%252Boptimizing%252Bmultiple%252Bchemical%252Bproperties%26aulast%3DShin%26aufirst%3DBonggun%26date%3D2021%26date%3D2021%26spage%3D146%26epage%3D153%26pub%3DACM%26atitle%3DProceedings%252Bof%252Bthe%252BConference%252Bon%252BHealth%25252C%252BInference%25252C%252Band%252BLearning%26aulast%3DGhassemi%26aufirst%3DMarzyeh%26date%3D2021%26volume%3D58" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dimitar  Yonchev</span>, <span class="hlFld-ContribAuthor ">Martin  Vogt</span>, <span class="hlFld-ContribAuthor ">Jürgen  Bajorath</span>. </span><span class="cited-content_cbyCitation_article-title">From SAR Diagnostics to Compound Design: Development Chronology of the Compound Optimization Monitor (COMO) Method. </span><span class="cited-content_cbyCitation_journal-name">Molecular Informatics</span><span> <strong>2020,</strong> <em>39 </em>
                                    (12)
                                     , 2000046. <a href="https://doi.org/10.1002/minf.202000046" title="DOI URL">https://doi.org/10.1002/minf.202000046</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/minf.202000046&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fminf.202000046%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Informatics%26atitle%3DFrom%252BSAR%252BDiagnostics%252Bto%252BCompound%252BDesign%25253A%252BDevelopment%252BChronology%252Bof%252Bthe%252BCompound%252BOptimization%252BMonitor%252B%252528COMO%252529%252BMethod%26aulast%3DYonchev%26aufirst%3DDimitar%26date%3D2020%26date%3D2020%26volume%3D39%26issue%3D12%26spage%3D2000046" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dimitar  Yonchev</span>, <span class="hlFld-ContribAuthor ">Jürgen  Bajorath</span>. </span><span class="cited-content_cbyCitation_article-title">DeepCOMO: from structure-activity relationship diagnostics to generative molecular design using the compound optimization monitor methodology. </span><span class="cited-content_cbyCitation_journal-name">Journal of Computer-Aided Molecular Design</span><span> <strong>2020,</strong> <em>34 </em>
                                    (12)
                                     , 1207-1218. <a href="https://doi.org/10.1007/s10822-020-00349-3" title="DOI URL">https://doi.org/10.1007/s10822-020-00349-3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10822-020-00349-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10822-020-00349-3%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Computer-Aided%2520Molecular%2520Design%26atitle%3DDeepCOMO%25253A%252Bfrom%252Bstructure-activity%252Brelationship%252Bdiagnostics%252Bto%252Bgenerative%252Bmolecular%252Bdesign%252Busing%252Bthe%252Bcompound%252Boptimization%252Bmonitor%252Bmethodology%26aulast%3DYonchev%26aufirst%3DDimitar%26date%3D2020%26date%3D2020%26volume%3D34%26issue%3D12%26spage%3D1207%26epage%3D1218" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dimitar  Yonchev</span>, <span class="hlFld-ContribAuthor ">Jürgen  Bajorath</span>. </span><span class="cited-content_cbyCitation_article-title">Integrating computational lead optimization diagnostics with analog design and candidate selection. </span><span class="cited-content_cbyCitation_journal-name">Future Science OA</span><span> <strong>2020,</strong> <em>6 </em>
                                    (3)
                                     , FSO451. <a href="https://doi.org/10.2144/fsoa-2019-0131" title="DOI URL">https://doi.org/10.2144/fsoa-2019-0131</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2144/fsoa-2019-0131&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2144%2Ffsoa-2019-0131%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Science%2520OA%26atitle%3DIntegrating%252Bcomputational%252Blead%252Boptimization%252Bdiagnostics%252Bwith%252Banalog%252Bdesign%252Band%252Bcandidate%252Bselection%26aulast%3DYonchev%26aufirst%3DDimitar%26date%3D2020%26volume%3D6%26issue%3D3%26spage%3DFSO451" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dimitar  Yonchev</span>, <span class="hlFld-ContribAuthor ">Martin  Vogt</span>, <span class="hlFld-ContribAuthor ">Jürgen  Bajorath</span>. </span><span class="cited-content_cbyCitation_article-title">Compound optimization monitor (COMO) method for computational evaluation of progress in medicinal chemistry projects. </span><span class="cited-content_cbyCitation_journal-name">Future Drug Discovery</span><span> <strong>2019,</strong> <em>1 </em>
                                    (2)
                                     , FDD15. <a href="https://doi.org/10.4155/fdd-2019-0016" title="DOI URL">https://doi.org/10.4155/fdd-2019-0016</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fdd-2019-0016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffdd-2019-0016%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Drug%2520Discovery%26atitle%3DCompound%252Boptimization%252Bmonitor%252B%252528COMO%252529%252Bmethod%252Bfor%252Bcomputational%252Bevaluation%252Bof%252Bprogress%252Bin%252Bmedicinal%252Bchemistry%252Bprojects%26aulast%3DYonchev%26aufirst%3DDimitar%26date%3D2019%26volume%3D1%26issue%3D2%26spage%3DFDD15" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">J. Jesús  Naveja</span>, <span class="hlFld-ContribAuthor ">José L.  Medina-Franco</span>. </span><span class="cited-content_cbyCitation_article-title">Finding Constellations in Chemical Space Through Core Analysis. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Chemistry</span><span> <strong>2019,</strong> <em>7 </em><a href="https://doi.org/10.3389/fchem.2019.00510" title="DOI URL">https://doi.org/10.3389/fchem.2019.00510</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fchem.2019.00510&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffchem.2019.00510%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Chemistry%26atitle%3DFinding%252BConstellations%252Bin%252BChemical%252BSpace%252BThrough%252BCore%252BAnalysis%26aulast%3DNaveja%26aufirst%3DJ.%2BJes%25C3%25BAs%26date%3D2019%26date%3D2019%26volume%3D7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01626/20181207/images/medium/jm-2018-01626v_0006.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01626/20181207/images/large/jm-2018-01626v_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01626&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01626/20181207/images/medium/jm-2018-01626v_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01626/20181207/images/large/jm-2018-01626v_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Exemplary ASs. Shown are AS-based scaffolds representing individual ASs. For each AS, the ChEMBL ID, target name, number of analogs, and their potency range are reported.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01626/20181207/images/large/jm-2018-01626v_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01626&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01626/20181207/images/medium/jm-2018-01626v_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01626/20181207/images/large/jm-2018-01626v_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Score dependence on NBH radii. Boxplots report the distribution of coverage <i>C</i> for NBHs with increasing percentile threshold (radius). For each AS, the mean of 10 independent calculations was used. Boxplots report the smallest value (bottom line), first quartile (lower boundary of the box), median value (thick line), third quartile (upper boundary of the box), and largest value (top line).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01626/20181207/images/large/jm-2018-01626v_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01626&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01626/20181207/images/medium/jm-2018-01626v_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01626/20181207/images/large/jm-2018-01626v_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Density vs coverage. The scatter plot compares <i>D</i> and <i>C</i> scores for all ASs. Each AS is represented by a dot that is scaled in size according to the number of analogs it contains (smallest dot, 51 analogs; largest, 166). Exemplary ASs are numbered according to <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01626/20181207/images/large/jm-2018-01626v_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01626&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01626/20181207/images/medium/jm-2018-01626v_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01626/20181207/images/large/jm-2018-01626v_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. SAR progression vs chemical saturation. The scatter plot compares <i>P</i> and <i>S</i> scores for all ASs. Each AS is represented by a dot that is scaled in size according to the number of analogs. Exemplary ASs are numbered according to <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01626/20181207/images/large/jm-2018-01626v_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01626&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01626/20181207/images/medium/jm-2018-01626v_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01626/20181207/images/large/jm-2018-01626v_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Progressing AS. Shown are exemplary analogs belonging to a series of endoplasmic reticulum ATPase inhibitors (black) and their NBHs (light blue) and virtual analogs (red) falling into the NBHs. Substitution sites are encircled.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01626/20181207/images/large/jm-2018-01626v_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01626&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i28">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_26032" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_26032" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 20 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span> <i>The
Practice of Medicinal Chemistry</i>, <span class="NLM_edition">3</span>rd ed,; <span class="NLM_contrib-group"><span class="NLM_string-name">Wermuth, C. G.</span></span>, Ed.; <span class="NLM_publisher-name">Academic Press-Elsevier</span>: <span class="NLM_publisher-loc">Burlington, San Diego, USA, London, U.K.</span>, <span class="NLM_year">2008</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2008&author=C.+G.+Wermuth&title=The%0APractice+of+Medicinal+Chemistry"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DThe%250APractice%2520of%2520Medicinal%2520Chemistry%26aulast%3DWermuth%26aufirst%3DC.%2BG.%26pub%3DAcademic%2520Press-Elsevier%26date%3D2008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cherkasov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muratov, E. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fourches, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varnek, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baskin, I. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cronin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dearden, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gramatica, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Todeschini, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Consonni, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuz’min, V. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cramer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benigni, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rathman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terfloth, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gasteiger, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richard, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tropsha, A.</span></span> <span> </span><span class="NLM_article-title">QSAR Modeling: Where Have You Been? Where Are You Going to?</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">4977</span>– <span class="NLM_lpage">5010</span>, <span class="refDoi"> DOI: 10.1021/jm4004285</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm4004285" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01626&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFCrsLnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=4977-5010&author=A.+Cherkasovauthor=E.+N.+Muratovauthor=D.+Fourchesauthor=A.+Varnekauthor=I.+I.+Baskinauthor=M.+Croninauthor=J.+Deardenauthor=P.+Gramaticaauthor=Y.+C.+Martinauthor=R.+Todeschiniauthor=V.+Consonniauthor=V.+E.+Kuz%E2%80%99minauthor=R.+Cramerauthor=R.+Benigniauthor=C.+Yangauthor=J.+Rathmanauthor=L.+Terflothauthor=J.+Gasteigerauthor=A.+Richardauthor=A.+Tropsha&title=QSAR+Modeling%3A+Where+Have+You+Been%3F+Where+Are+You+Going+to%3F&doi=10.1021%2Fjm4004285"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">QSAR Modeling: Where Have You Been? Where Are You Going To?</span></div><div class="casAuthors">Cherkasov, Artem; Muratov, Eugene N.; Fourches, Denis; Varnek, Alexandre; Baskin, Igor I.; Cronin, Mark; Dearden, John; Gramatica, Paola; Martin, Yvonne C.; Todeschini, Roberto; Consonni, Viviana; Kuzmin, Victor E.; Cramer, Richard; Benigni, Romualdo; Yang, Chihae; Rathman, James; Terfloth, Lothar; Gasteiger, Johann; Richard, Ann; Tropsha, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4977-5010</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Quant. structure-activity relationship modeling is one of the major computational tools employed in medicinal chem.  However, throughout its entire history it has drawn both praise and criticism concerning its reliability, limitations, successes, and failures.  In this paper, we discuss (i) the development and evolution of QSAR; (ii) the current trends, unsolved problems, and pressing challenges; and (iii) several novel and emerging applications of QSAR modeling.  Throughout this discussion, we provide guidelines for QSAR development, validation, and application, which are summarized in best practices for building rigorously validated and externally predictive QSAR models.  We hope that this Perspective will help communications between computational and exptl. chemists toward collaborative development and use of QSAR models.  We also believe that the guidelines presented here will help journal editors and reviewers apply more stringent scientific stds. to manuscripts reporting new QSAR studies, as well as encourage the use of high quality, validated QSARs for regulatory decision making.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbLWydJbspbLVg90H21EOLACvtfcHk0ljjYEhzF49XfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFCrsLnE&md5=a2d371ee33fbecf48ef59ecf914e4aa8</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1021%2Fjm4004285&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm4004285%26sid%3Dliteratum%253Aachs%26aulast%3DCherkasov%26aufirst%3DA.%26aulast%3DMuratov%26aufirst%3DE.%2BN.%26aulast%3DFourches%26aufirst%3DD.%26aulast%3DVarnek%26aufirst%3DA.%26aulast%3DBaskin%26aufirst%3DI.%2BI.%26aulast%3DCronin%26aufirst%3DM.%26aulast%3DDearden%26aufirst%3DJ.%26aulast%3DGramatica%26aufirst%3DP.%26aulast%3DMartin%26aufirst%3DY.%2BC.%26aulast%3DTodeschini%26aufirst%3DR.%26aulast%3DConsonni%26aufirst%3DV.%26aulast%3DKuz%25E2%2580%2599min%26aufirst%3DV.%2BE.%26aulast%3DCramer%26aufirst%3DR.%26aulast%3DBenigni%26aufirst%3DR.%26aulast%3DYang%26aufirst%3DC.%26aulast%3DRathman%26aufirst%3DJ.%26aulast%3DTerfloth%26aufirst%3DL.%26aulast%3DGasteiger%26aufirst%3DJ.%26aulast%3DRichard%26aufirst%3DA.%26aulast%3DTropsha%26aufirst%3DA.%26atitle%3DQSAR%2520Modeling%253A%2520Where%2520Have%2520You%2520Been%253F%2520Where%2520Are%2520You%2520Going%2520to%253F%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D4977%26epage%3D5010%26doi%3D10.1021%2Fjm4004285" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Segall, M.</span></span> <span> </span><span class="NLM_article-title">Advances in Multi-Parameter Optimization Methods for <i>De Novo</i> Drug Design</span>. <i>Expert Opin. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">803</span>– <span class="NLM_lpage">817</span>, <span class="refDoi"> DOI: 10.1517/17460441.2014.913565</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01626&amp;key=10.1517%2F17460441.2014.913565" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=803-817&author=M.+Segall&title=Advances+in+Multi-Parameter+Optimization+Methods+for+De+Novo+Drug+Design&doi=10.1517%2F17460441.2014.913565"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1517%2F17460441.2014.913565&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F17460441.2014.913565%26sid%3Dliteratum%253Aachs%26aulast%3DSegall%26aufirst%3DM.%26atitle%3DAdvances%2520in%2520Multi-Parameter%2520Optimization%2520Methods%2520for%2520De%2520Novo%2520Drug%2520Design%26jtitle%3DExpert%2520Opin.%2520Drug%2520Discovery%26date%3D2014%26volume%3D9%26spage%3D803%26epage%3D817%26doi%3D10.1517%2F17460441.2014.913565" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Munson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lieberman, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tserlin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rocnik, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ge, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fitzgerald, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia-Echeverria, C.</span></span> <span> </span><span class="NLM_article-title">Lead Optimization Attrition Analysis (LOAA): A Novel and General Methodology for Medicinal Chemistry</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">978</span>– <span class="NLM_lpage">987</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2015.03.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01626&amp;key=10.1016%2Fj.drudis.2015.03.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01626&amp;key=25814036" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01626&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlsV2gs7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2015&pages=978-987&author=M.+Munsonauthor=H.+Liebermanauthor=E.+Tserlinauthor=J.+Rocnikauthor=J.+Geauthor=M.+Fitzgeraldauthor=V.+Patelauthor=C.+Garcia-Echeverria&title=Lead+Optimization+Attrition+Analysis+%28LOAA%29%3A+A+Novel+and+General+Methodology+for+Medicinal+Chemistry&doi=10.1016%2Fj.drudis.2015.03.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Lead optimization attrition analysis (LOAA): a novel and general methodology for medicinal chemistry</span></div><div class="casAuthors">Munson, Mark; Lieberman, Harvey; Tserlin, Elina; Rocnik, Jennifer; Ge, Jie; Fitzgerald, Maria; Patel, Vinod; Garcia-Echeverria, Carlos</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">978-987</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Herein, we report a novel and general method, lead optimization attrition anal. (LOAA), to benchmark two distinct small-mol. lead series using a relatively unbiased, simple technique and com. available software.  We illustrate this approach with data collected during lead optimization of two independent oncol. programs as a case study.  Easily generated graphics and attrition curves enabled us to calibrate progress and support go/no go decisions on each program.  We believe that this data-driven technique could be used broadly by medicinal chemists and management to guide strategic decisions during drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8Es9dpgoC2bVg90H21EOLACvtfcHk0ljjYEhzF49XfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlsV2gs7Y%253D&md5=52831d6178321347e32161d61aea978c</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2015.03.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2015.03.010%26sid%3Dliteratum%253Aachs%26aulast%3DMunson%26aufirst%3DM.%26aulast%3DLieberman%26aufirst%3DH.%26aulast%3DTserlin%26aufirst%3DE.%26aulast%3DRocnik%26aufirst%3DJ.%26aulast%3DGe%26aufirst%3DJ.%26aulast%3DFitzgerald%26aufirst%3DM.%26aulast%3DPatel%26aufirst%3DV.%26aulast%3DGarcia-Echeverria%26aufirst%3DC.%26atitle%3DLead%2520Optimization%2520Attrition%2520Analysis%2520%2528LOAA%2529%253A%2520A%2520Novel%2520and%2520General%2520Methodology%2520for%2520Medicinal%2520Chemistry%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2015%26volume%3D20%26spage%3D978%26epage%3D987%26doi%3D10.1016%2Fj.drudis.2015.03.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kunimoto, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyao, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span> <span> </span><span class="NLM_article-title">Computational Method for Estimating Progression Saturation of Analog Series</span>. <i>RSC Adv.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">5484</span>– <span class="NLM_lpage">5492</span>, <span class="refDoi"> DOI: 10.1039/c7ra13748f</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01626&amp;key=10.1039%2Fc7ra13748f" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01626&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvFantLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&pages=5484-5492&author=R.+Kunimotoauthor=T.+Miyaoauthor=J.+Bajorath&title=Computational+Method+for+Estimating+Progression+Saturation+of+Analog+Series&doi=10.1039%2Fc7ra13748f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Computational method for estimating progression saturation of analog series</span></div><div class="casAuthors">Kunimoto, Ryo; Miyao, Tomoyuki; Bajorath, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">RSC Advances</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">5484-5492</span>CODEN:
                <span class="NLM_cas:coden">RSCACL</span>;
        ISSN:<span class="NLM_cas:issn">2046-2069</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">In lead optimization, it is difficult to est. when an analog series might be satd. and synthesis of addnl. compds. would be unlikely to yield further progress.  Rather than terminating a series, one often continues to generate analogs hoping to reach the final optimization goal, even if obstacles that are faced ultimately prove to be unsurmountable.  Clearly, methodologies to better understand series progression and satn. are highly desirable.  However, only a few approaches are currently available to monitor series progression and aid in decision making.  Herein, we introduce a new computational method to assess progression satn. of an analog series by relating the properties of existing compds. to those of synthetic candidates and comparing their distributions in chem. space.  The neighborhoods of analogs are analyzed and the distance relationships between existing and candidate compds. quantified.  An intuitive dual scoring scheme makes it possible to characterize analog series and their degree of progression satn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGonQeSyaIHLVg90H21EOLACvtfcHk0lgu9NvwJAOO9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvFantLY%253D&md5=3d944bfe016a757fc1e105d4cc45c1bd</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1039%2Fc7ra13748f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc7ra13748f%26sid%3Dliteratum%253Aachs%26aulast%3DKunimoto%26aufirst%3DR.%26aulast%3DMiyao%26aufirst%3DT.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DComputational%2520Method%2520for%2520Estimating%2520Progression%2520Saturation%2520of%2520Analog%2520Series%26jtitle%3DRSC%2520Adv.%26date%3D2018%26volume%3D8%26spage%3D5484%26epage%3D5492%26doi%3D10.1039%2Fc7ra13748f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yonchev, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stumpfe, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kunimoto, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyao, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span> <span> </span><span class="NLM_article-title">Computational Assessment of Chemical Saturation of Analog Series under Varying Conditions</span>. <i>ACS Omega</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">15799</span>– <span class="NLM_lpage">15808</span>, <span class="refDoi"> DOI: 10.1021/acsomega.8b02087</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsomega.8b02087" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2018&pages=15799-15808&author=D.+Yonchevauthor=M.+Vogtauthor=D.+Stumpfeauthor=R.+Kunimotoauthor=T.+Miyaoauthor=J.+Bajorath&title=Computational+Assessment+of+Chemical+Saturation+of+Analog+Series+under+Varying+Conditions&doi=10.1021%2Facsomega.8b02087"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Facsomega.8b02087&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsomega.8b02087%26sid%3Dliteratum%253Aachs%26aulast%3DYonchev%26aufirst%3DD.%26aulast%3DVogt%26aufirst%3DM.%26aulast%3DStumpfe%26aufirst%3DD.%26aulast%3DKunimoto%26aufirst%3DR.%26aulast%3DMiyao%26aufirst%3DT.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DComputational%2520Assessment%2520of%2520Chemical%2520Saturation%2520of%2520Analog%2520Series%2520under%2520Varying%2520Conditions%26jtitle%3DACS%2520Omega%26date%3D2018%26volume%3D3%26spage%3D15799%26epage%3D15808%26doi%3D10.1021%2Facsomega.8b02087" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wassermann, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wawer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span> <span> </span><span class="NLM_article-title">Activity Landscape Representations for Structure-Activity Relationship Analysis</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">8209</span>– <span class="NLM_lpage">8223</span>, <span class="refDoi"> DOI: 10.1021/jm100933w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm100933w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01626&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFKmt7bF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=8209-8223&author=A.+M.+Wassermannauthor=M.+Wawerauthor=J.+Bajorath&title=Activity+Landscape+Representations+for+Structure-Activity+Relationship+Analysis&doi=10.1021%2Fjm100933w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Activity Landscape Representations for Structure-Activity Relationship Analysis</span></div><div class="casAuthors">Wassermann, Anne Mai; Wawer, Mathias; Bajorath, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">8209-8223</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The study of compd. structure-activity relationships (SARs) is one of the central themes in medicinal chem.  SAR information is analyzed in different contexts, from screening and hit-to-lead to lead optimization projects.  For the exploration of SARs, the concept of an activity landscape, which integrates mol. similarity and potency information, is of high relevance.  The computational study of activity landscapes is still an evolving field.  Activity landscape models are designed to rationalize SAR features of compd. data sets and select key compds. for chem. exploration.  The choice of mol. representations and the way mol. similarity is assessed are critically important factors for landscape generation and anal.  Graphical representation of SAR features is a major focal point of landscape modeling.  Although complex activity landscapes are generally difficult to analyze, much progress has recently been made in extg. SAR information from various landscape views.  This article aims to provide an overview of the state-of-the-art in activity landscape anal. and a discussion of its potential for medicinal chem. applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBe8xXwkO-_bVg90H21EOLACvtfcHk0lgu9NvwJAOO9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFKmt7bF&md5=4867772191beabe154c8d221640c956a</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Fjm100933w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100933w%26sid%3Dliteratum%253Aachs%26aulast%3DWassermann%26aufirst%3DA.%2BM.%26aulast%3DWawer%26aufirst%3DM.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DActivity%2520Landscape%2520Representations%2520for%2520Structure-Activity%2520Relationship%2520Analysis%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D8209%26epage%3D8223%26doi%3D10.1021%2Fjm100933w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Iyer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span> <span> </span><span class="NLM_article-title">SAR Monitoring of Evolving Compound Data Sets Using Activity Landscapes</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">532</span>– <span class="NLM_lpage">540</span>, <span class="refDoi"> DOI: 10.1021/ci100505m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci100505m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01626&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhvVylsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2011&pages=532-540&author=P.+Iyerauthor=Y.+Huauthor=J.+Bajorath&title=SAR+Monitoring+of+Evolving+Compound+Data+Sets+Using+Activity+Landscapes&doi=10.1021%2Fci100505m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">SAR Monitoring of Evolving Compound Data Sets Using Activity Landscapes</span></div><div class="casAuthors">Iyer, Preeti; Hu, Ye; Bajorath, Jurgen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">532-540</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In pharmaceutical research, collections of active compds. directed against specific therapeutic targets usually evolve over time.  Small mol. discovery is an iterative process.  New compds. are discovered, alternative compd. series explored, some series discontinued, and others prioritized.  The design of new compds. usually takes into consideration prior chem. and structure-activity relationship (SAR) knowledge.  Hence, historically grown compd. collections represent a viable source of chem. and SAR information that might be utilized to retrospectively analyze roadblocks in compd. optimization and further guide discovery projects.  However, SAR anal. of large and heterogeneous sets of active compds. is also principally complicated.  We have subjected evolving compd. data sets to SAR monitoring using activity landscape models in order to evaluate how compn. and SAR characteristics might change over time.  Chemotype and potency distributions in evolving data sets directed against different therapeutic targets were analyzed and alternative activity landscape representations generated at different points in time to monitor the progression of global and local SAR features.  Our results show that the evolving data sets studied here have predominantly grown around seed clusters of active compds. that often emerged early on, while other SAR islands remained largely unexplored.  Moreover, increasing scaffold diversity in evolving data sets did not necessarily yield new SAR patterns, indicating a rather significant influence of "me-too-ism" (i.e., introducing new chemotypes that are similar to already known ones) on the compn. and SAR information content of the data sets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKpgeL5xQDQLVg90H21EOLACvtfcHk0lhBwHgQZpJJcg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhvVylsb4%253D&md5=85d5290d13c5c9aede37f5360385419c</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Fci100505m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100505m%26sid%3Dliteratum%253Aachs%26aulast%3DIyer%26aufirst%3DP.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DSAR%2520Monitoring%2520of%2520Evolving%2520Compound%2520Data%2520Sets%2520Using%2520Activity%2520Landscapes%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2011%26volume%3D51%26spage%3D532%26epage%3D540%26doi%3D10.1021%2Fci100505m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Peltason, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span> <span> </span><span class="NLM_article-title">SAR Index: Quantifying the Nature of Structure-Activity Relationships</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">5571</span>– <span class="NLM_lpage">5578</span>, <span class="refDoi"> DOI: 10.1021/jm0705713</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0705713" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01626&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtV2msLfP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2007&pages=5571-5578&author=L.+Peltasonauthor=J.+Bajorath&title=SAR+Index%3A+Quantifying+the+Nature+of+Structure-Activity+Relationships&doi=10.1021%2Fjm0705713"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">SAR index: quantifying the nature of structure-activity relationships</span></div><div class="casAuthors">Peltason, Lisa; Bajorath, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">5571-5578</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Structure-activity relationships (SARs) can display very different features.  Small chem. modifications of active mols. often dramatically alter biol. responses.  By contrast, structurally diverse mols. can have similar activity.  SARs can also be heterogeneous in nature.  For example, for structurally diverse mols. with similar activity, closely related analogs might have significant differences in potency.  Given the inherent complexity of SARs, it has been very difficult to est. SAR characteristics from mol. structure.  On the basis of systematic correlation of 2D structural similarity and compd. potency, we have developed a function termed "SAR Index" that quant. describes the nature of SARs and establishes different SAR categories: continuous, discontinuous, heterogeneous-relaxed, and heterogeneous-constrained.  These heterogeneous SAR categories are described for the first time.  Given a set of active compds. and their potency values, SAR Index calcns. can est. how likely it is to identify structurally distinct mols. having similar activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhD6NtNDFPBbVg90H21EOLACvtfcHk0lhBwHgQZpJJcg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtV2msLfP&md5=74c7fadf125c6d660cb24a54cbf44dc1</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Fjm0705713&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0705713%26sid%3Dliteratum%253Aachs%26aulast%3DPeltason%26aufirst%3DL.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DSAR%2520Index%253A%2520Quantifying%2520the%2520Nature%2520of%2520Structure-Activity%2520Relationships%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2007%26volume%3D50%26spage%3D5571%26epage%3D5578%26doi%3D10.1021%2Fjm0705713" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shanmugasundaram, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kayastha, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de la Vega de León, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dimova, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span> <span> </span><span class="NLM_article-title">Monitoring the Progression of Structure–Activity Relationship Information during Lead Optimization</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">4235</span>– <span class="NLM_lpage">4244</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01428</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01428" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2015&pages=4235-4244&author=V.+Shanmugasundaramauthor=L.+Zhangauthor=S.+Kayasthaauthor=A.+de+la+Vega+de+Le%C3%B3nauthor=D.+Dimovaauthor=J.+Bajorath&title=Monitoring+the+Progression+of+Structure%E2%80%93Activity+Relationship+Information+during+Lead+Optimization&doi=10.1021%2Facs.jmedchem.5b01428"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01428&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01428%26sid%3Dliteratum%253Aachs%26aulast%3DShanmugasundaram%26aufirst%3DV.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DKayastha%26aufirst%3DS.%26aulast%3Dde%2Bla%2BVega%2Bde%2BLe%25C3%25B3n%26aufirst%3DA.%26aulast%3DDimova%26aufirst%3DD.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DMonitoring%2520the%2520Progression%2520of%2520Structure%25E2%2580%2593Activity%2520Relationship%2520Information%2520during%2520Lead%2520Optimization%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D59%26spage%3D4235%26epage%3D4244%26doi%3D10.1021%2Facs.jmedchem.5b01428" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maynard, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, C. D.</span></span> <span> </span><span class="NLM_article-title">Quantifying, Visualizing, and Monitoring Lead Optimization</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">4189</span>– <span class="NLM_lpage">4201</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00948</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00948" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2015&pages=4189-4201&author=A.+T.+Maynardauthor=C.+D.+Roberts&title=Quantifying%2C+Visualizing%2C+and+Monitoring+Lead+Optimization&doi=10.1021%2Facs.jmedchem.5b00948"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00948&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00948%26sid%3Dliteratum%253Aachs%26aulast%3DMaynard%26aufirst%3DA.%2BT.%26aulast%3DRoberts%26aufirst%3DC.%2BD.%26atitle%3DQuantifying%252C%2520Visualizing%252C%2520and%2520Monitoring%2520Lead%2520Optimization%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D59%26spage%3D4189%26epage%3D4201%26doi%3D10.1021%2Facs.jmedchem.5b00948" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stumpfe, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span> <span> </span><span class="NLM_article-title">Exploring Activity Cliffs in Medicinal Chemistry</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">2932</span>– <span class="NLM_lpage">2942</span>, <span class="refDoi"> DOI: 10.1021/jm201706b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm201706b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01626&amp;key=1%3ACAS%3A528%3ADC%252BC38XmtFajsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=2932-2942&author=D.+Stumpfeauthor=J.+Bajorath&title=Exploring+Activity+Cliffs+in+Medicinal+Chemistry&doi=10.1021%2Fjm201706b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Exploring Activity Cliffs in Medicinal Chemistry</span></div><div class="casAuthors">Stumpfe, Dagmar; Bajorath, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2932-2942</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In recent years, structure-activity relationships (SARs) have increasingly been studied through mining of large compd. data sets, leading to a renaissance of the activity landscape concept.  In general terms, an activity landscape is defined as any graphical representation that integrates similarity and potency relationships between compds. sharing a specific activity.  From activity landscape representations of different design, both global and local SAR features present in compd. data sets can be extd.  Modeling and rationalizing activity landscapes require the application of computational methods.  As such, the activity landscape concept is of comparable relevance for chemoinformatics (having a strong focus on data mining and representation) and medicinal chem. (given its immediate relevance for SAR anal.).  Herein, we describe in detail the multifaceted nature of activity cliffs, discuss underlying scientific concepts, and explain how their individual or systematic anal. might (or might not) provide useful information for medicinal chem. programs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGql1wydp65qQLVg90H21EOLACvtfcHk0liP--BBCrTR7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmtFajsQ%253D%253D&md5=ceddd1de5798b585d6381ac18ba67fc9</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Fjm201706b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm201706b%26sid%3Dliteratum%253Aachs%26aulast%3DStumpfe%26aufirst%3DD.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DExploring%2520Activity%2520Cliffs%2520in%2520Medicinal%2520Chemistry%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D2932%26epage%3D2942%26doi%3D10.1021%2Fjm201706b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span> </span><span class="NLM_article-title">RDKit: Cheminformatics
and Machine Learning Software, 2013</span>. <a href="http://www.rdkit.org" class="extLink">http://www.rdkit.org</a> (accessed
July 1, <span class="NLM_year">2018</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=RDKit%3A+Cheminformatics%0Aand+Machine+Learning+Software%2C+2013.+http%3A%2F%2Fwww.rdkit.org+%28accessed%0AJuly+1%2C+2018%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26atitle%3DRDKit%253A%2520Cheminformatics%250Aand%2520Machine%2520Learning%2520Software%252C%25202013%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Delaney, J.
S.</span></span> <span> </span><span class="NLM_article-title">ESOL: Estimating Aqueous Solubility Directly from Molecular Structure</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">1000</span>– <span class="NLM_lpage">1005</span>, <span class="refDoi"> DOI: 10.1021/ci034243x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci034243x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2004&pages=1000-1005&author=J.%0AS.+Delaney&title=ESOL%3A+Estimating+Aqueous+Solubility+Directly+from+Molecular+Structure&doi=10.1021%2Fci034243x"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fci034243x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci034243x%26sid%3Dliteratum%253Aachs%26aulast%3DDelaney%26aufirst%3DJ.%2BS.%26atitle%3DESOL%253A%2520Estimating%2520Aqueous%2520Solubility%2520Directly%2520from%2520Molecular%2520Structure%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2004%26volume%3D44%26spage%3D1000%26epage%3D1005%26doi%3D10.1021%2Fci034243x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bento, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaulton, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hersey, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellis, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chambers, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krüger, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Light, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mak, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGlinchey, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowotka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papadatos, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santos, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overington, J. P.</span></span> <span> </span><span class="NLM_article-title">The ChEMBL Bioactivity Database: An Update</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">D1083</span>– <span class="NLM_lpage">D1090</span>, <span class="refDoi"> DOI: 10.1093/nar/gkt1031</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01626&amp;key=10.1093%2Fnar%2Fgkt1031" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01626&amp;key=24214965" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01626&amp;key=1%3ACAS%3A528%3ADC%252BC2cXoslWl" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2014&pages=D1083-D1090&author=A.+P.+Bentoauthor=A.+Gaultonauthor=A.+Herseyauthor=L.+J.+Bellisauthor=J.+Chambersauthor=M.+Daviesauthor=F.+A.+Kr%C3%BCgerauthor=Y.+Lightauthor=L.+Makauthor=S.+McGlincheyauthor=M.+Nowotkaauthor=G.+Papadatosauthor=R.+Santosauthor=J.+P.+Overington&title=The+ChEMBL+Bioactivity+Database%3A+An+Update&doi=10.1093%2Fnar%2Fgkt1031"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">The ChEMBL bioactivity database: an update</span></div><div class="casAuthors">Bento, A. Patricia; Gaulton, Anna; Hersey, Anne; Bellis, Louisa J.; Chambers, Jon; Davies, Mark; Krueger, Felix A.; Light, Yvonne; Mak, Lora; McGlinchey, Shaun; Nowotka, Michal; Papadatos, George; Santos, Rita; Overington, John P.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D1083-D1090</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">ChEMBL is an open large-scale bioactivity database (https://www.ebi.ac.uk/chembl), previously described in the 2012 Nucleic Acids Research Database Issue.  Since then, a variety of new data sources and improvements in functionality have contributed to the growth and utility of the resource.  In particular, more comprehensive tracking of compds. from research stages through clin. development to market is provided through the inclusion of data from United States Adopted Name applications; a new richer data model for representing drug targets has been developed; and a no. of methods have been put in place to allow users to more easily identify reliable data.  Finally, access to ChEMBL is now available via a new Resource Description Framework format, in addn. to the web-based interface, data downloads and web services.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXb5k1IvJNaLVg90H21EOLACvtfcHk0liP--BBCrTR7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXoslWl&md5=31b832d03d56ea3065d7aa29618362bc</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkt1031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkt1031%26sid%3Dliteratum%253Aachs%26aulast%3DBento%26aufirst%3DA.%2BP.%26aulast%3DGaulton%26aufirst%3DA.%26aulast%3DHersey%26aufirst%3DA.%26aulast%3DBellis%26aufirst%3DL.%2BJ.%26aulast%3DChambers%26aufirst%3DJ.%26aulast%3DDavies%26aufirst%3DM.%26aulast%3DKr%25C3%25BCger%26aufirst%3DF.%2BA.%26aulast%3DLight%26aufirst%3DY.%26aulast%3DMak%26aufirst%3DL.%26aulast%3DMcGlinchey%26aufirst%3DS.%26aulast%3DNowotka%26aufirst%3DM.%26aulast%3DPapadatos%26aufirst%3DG.%26aulast%3DSantos%26aufirst%3DR.%26aulast%3DOverington%26aufirst%3DJ.%2BP.%26atitle%3DThe%2520ChEMBL%2520Bioactivity%2520Database%253A%2520An%2520Update%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2014%26volume%3D42%26spage%3DD1083%26epage%3DD1090%26doi%3D10.1093%2Fnar%2Fgkt1031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stumpfe, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dimova, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span> <span> </span><span class="NLM_article-title">Computational Method for Systematic Identification of Analog Series and Key Compounds Representing Series and Their Biological Activity Profiles</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">7667</span>– <span class="NLM_lpage">7676</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00906</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00906" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=7667-7676&author=D.+Stumpfeauthor=D.+Dimovaauthor=J.+Bajorath&title=Computational+Method+for+Systematic+Identification+of+Analog+Series+and+Key+Compounds+Representing+Series+and+Their+Biological+Activity+Profiles&doi=10.1021%2Facs.jmedchem.6b00906"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00906&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00906%26sid%3Dliteratum%253Aachs%26aulast%3DStumpfe%26aufirst%3DD.%26aulast%3DDimova%26aufirst%3DD.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DComputational%2520Method%2520for%2520Systematic%2520Identification%2520of%2520Analog%2520Series%2520and%2520Key%2520Compounds%2520Representing%2520Series%2520and%2520Their%2520Biological%2520Activity%2520Profiles%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D7667%26epage%3D7676%26doi%3D10.1021%2Facs.jmedchem.6b00906" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kenny, P. W.</span>; <span class="NLM_string-name">Sadowski, J.</span></span> <span> </span><span class="NLM_article-title">Structure Modification in Chemical Databases</span>. In  <i>Chemoinformatics in Drug Discovery</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oprea, T. I.</span></span>, Ed.; <span class="NLM_publisher-name">Wiley-VCH</span>: <span class="NLM_publisher-loc">Weinheim, Germany</span>, <span class="NLM_year">2004</span>; pp  <span class="NLM_fpage">271</span>– <span class="NLM_lpage">285</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2004&pages=271-285&author=P.+W.+Kenny&author=J.+Sadowskiauthor=T.+I.+Oprea&title=Chemoinformatics+in+Drug+Discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DKenny%26aufirst%3DP.%2BW.%26atitle%3DStructure%2520Modification%2520in%2520Chemical%2520Databases%26btitle%3DChemoinformatics%2520in%2520Drug%2520Discovery%26aulast%3DOprea%26aufirst%3DT.%2BI.%26pub%3DWiley-VCH%26date%3D2004%26spage%3D271%26epage%3D285" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dimova, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span> <span> </span><span class="NLM_article-title">Computational Design of New Molecular Scaffolds for Medicinal Chemistry, Part II: Generalization of Analog Series-Based Scaffolds</span>. <i>Future Sci. OA</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">FSO287</span>, <span class="refDoi"> DOI: 10.4155/fsoa-2017-0135</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01626&amp;key=10.4155%2Ffsoa-2017-0135" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01626&amp;key=29682322" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2018&pages=FSO287&author=D.+Dimovaauthor=J.+Bajorath&title=Computational+Design+of+New+Molecular+Scaffolds+for+Medicinal+Chemistry%2C+Part+II%3A+Generalization+of+Analog+Series-Based+Scaffolds&doi=10.4155%2Ffsoa-2017-0135"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.4155%2Ffsoa-2017-0135&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffsoa-2017-0135%26sid%3Dliteratum%253Aachs%26aulast%3DDimova%26aufirst%3DD.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DComputational%2520Design%2520of%2520New%2520Molecular%2520Scaffolds%2520for%2520Medicinal%2520Chemistry%252C%2520Part%2520II%253A%2520Generalization%2520of%2520Analog%2520Series-Based%2520Scaffolds%26jtitle%3DFuture%2520Sci.%2520OA%26date%3D2018%26volume%3D4%26spage%3DFSO287%26doi%3D10.4155%2Ffsoa-2017-0135" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stumpfe, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span> <span> </span><span class="NLM_article-title">MMP-Cliffs: Systematic Identification of Activity Cliffs on the Basis of Matched Molecular Pairs</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">1138</span>– <span class="NLM_lpage">1145</span>, <span class="refDoi"> DOI: 10.1021/ci3001138</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci3001138" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01626&amp;key=1%3ACAS%3A528%3ADC%252BC38Xlt1GgtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=1138-1145&author=X.+Huauthor=Y.+Huauthor=M.+Vogtauthor=D.+Stumpfeauthor=J.+Bajorath&title=MMP-Cliffs%3A+Systematic+Identification+of+Activity+Cliffs+on+the+Basis+of+Matched+Molecular+Pairs&doi=10.1021%2Fci3001138"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">MMP-Cliffs: Systematic Identification of Activity Cliffs on the Basis of Matched Molecular Pairs</span></div><div class="casAuthors">Hu, Xiaoying; Hu, Ye; Vogt, Martin; Stumpfe, Dagmar; Bajorath, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1138-1145</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Activity cliffs are generally defined as pairs of structurally similar compds. having large differences in potency.  The anal. of activity cliffs is of general interest because structure-activity relationship (SAR) determinants can often be deduced from them.  Crit. questions for the study of activity cliffs include how similar compds. should be to qualify as cliff partners, how similarity should be assessed, and how large potency differences between participating compds. should be.  Thus far, activity cliffs have mostly been defined on the basis of calcd. Tanimoto similarity values using structural descriptors, esp. 2D fingerprints.  As any theor. assessment of mol. similarity, this approach has its limitations.  For example, calcd. Tanimoto similarities might often be difficult to reconcile and interpret from a chem. perspective, a point of critique frequently raised in medicinal chem.  Herein, we have explored activity cliffs by considering well-defined substructure replacements instead of calcd. similarity values.  For this purpose, the matched mol. pair (MMP) formalism has been applied.  MMPs were systematically derived from public domain compds., and activity cliffs were extd. from them, termed MMP-cliffs.  The frequency of cliff formation was detd. for compds. active against different targets, MMP-cliffs were analyzed in detail, and re-evaluated on the basis of Tanimoto similarity.  In many instances, chem. intuitive activity cliffs were only detected on the basis of MMPs, but not Tanimoto similarity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCYSOxNqPjGbVg90H21EOLACvtfcHk0lgF_29ijoNoNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xlt1GgtLg%253D&md5=958a76d85d0a06424b43eb1b694cf3ca</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fci3001138&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci3001138%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DX.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DVogt%26aufirst%3DM.%26aulast%3DStumpfe%26aufirst%3DD.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DMMP-Cliffs%253A%2520Systematic%2520Identification%2520of%2520Activity%2520Cliffs%2520on%2520the%2520Basis%2520of%2520Matched%2520Molecular%2520Pairs%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D1138%26epage%3D1145%26doi%3D10.1021%2Fci3001138" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span> <i>OEChem TK</i>, version 1.7.7; <span class="NLM_publisher-name">OpenEye
Scientific Software, Inc.</span>: <span class="NLM_publisher-loc">Santa Fe,
NM, USA</span>, <span class="NLM_year">2012</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+OEChem+TK%2C+version+1.7.7%3B+OpenEye%0AScientific+Software%2C+Inc.%3A+Santa+Fe%2C%0ANM%2C+USA%2C+2012."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DOEChem%2520TK%26pub%3DOpenEye%250AScientific%2520Software%252C%2520Inc%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i25"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b01626">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_84826"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.8b01626">10.1021/acs.jmedchem.8b01626</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Molecular formula strings of ASs and virtual analogs for an exemplary AS are provided (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b01626/suppl_file/jm8b01626_si_001.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b01626/suppl_file/jm8b01626_si_001.csv">jm8b01626_si_001.csv (1.42 MB)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.8b01626&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2018.61.issue-23%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.8b01626%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.8b01626" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                760KB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-b425991822a79ada59b3.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-fb88834b824b41b135ad.js"></script>

<script type="text/javascript" src="/wro/ke1q~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"667986f21af232b5","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.6.0","si":10}'></script>
</body>
</html>
