
<html lang="en"     class="pb-page"  data-request-id="58cdb0f3-7e20-453c-96ce-5c4b1e50a58a"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";article:article:10.1021/jm501893k;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;issue:issue:10.1021/jmcmar.2015.58.issue-6;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="9H-Purine Scaffold Reveals Induced-Fit Pocket Plasticity of the BRD9 Bromodomain" /></meta><meta name="dc.Creator" content="Sarah  Picaud" /></meta><meta name="dc.Creator" content="Maria  Strocchia" /></meta><meta name="dc.Creator" content="Stefania  Terracciano" /></meta><meta name="dc.Creator" content="Gianluigi  Lauro" /></meta><meta name="dc.Creator" content="Jacqui  Mendez" /></meta><meta name="dc.Creator" content="Danette L.  Daniels" /></meta><meta name="dc.Creator" content="Raffaele  Riccio" /></meta><meta name="dc.Creator" content="Giuseppe  Bifulco" /></meta><meta name="dc.Creator" content="Ines  Bruno" /></meta><meta name="dc.Creator" content="Panagis  Filippakopoulos" /></meta><meta name="dc.Description" content="The 2-amine-9H-purine scaffold was identified as a weak bromodomain template and was developed via iterative structure based design into a potent nanomolar ligand for the bromodomain of human BRD9 ..." /></meta><meta name="Description" content="The 2-amine-9H-purine scaffold was identified as a weak bromodomain template and was developed via iterative structure based design into a potent nanomolar ligand for the bromodomain of human BRD9 ..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="March 5, 2015" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/jm501893k" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2015 American Chemical Society" /></meta><meta name="citation_fulltext_world_readable" content=""/></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Fjm501893k" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Fjm501893k" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Fjm501893k" /></link>
        
    
    

<title>9H-Purine Scaffold Reveals Induced-Fit Pocket Plasticity of the BRD9 Bromodomain | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/jm501893k" /></meta><meta property="og:title" content="9H-Purine Scaffold Reveals Induced-Fit Pocket Plasticity of the BRD9 Bromodomain" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-6/jm501893k/20150418/images/large/jm-2014-01893k_0007.jpeg" /></meta><meta property="og:description" content="The 2-amine-9H-purine scaffold was identified as a weak bromodomain template and was developed via iterative structure based design into a potent nanomolar ligand for the bromodomain of human BRD9 with small residual micromolar affinity toward the bromodomain of BRD4. Binding of the lead compound 11 to the bromodomain of BRD9 results in an unprecedented rearrangement of residues forming the acetyllysine recognition site, affecting plasticity of the protein in an induced-fit pocket. The compound does not exhibit any cytotoxic effect in HEK293 cells and displaces the BRD9 bromodomain from chromatin in bioluminescence proximity assays without affecting the BRD4/histone complex. The 2-amine-9H-purine scaffold represents a novel template that can be further modified to yield highly potent and selective tool compounds to interrogate the biological role of BRD9 in diverse cellular systems." /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta><meta name="twitter:description" content="The 2-amine-9H-purine scaffold was identified as a weak bromodomain template and was developed via iterative structure based design into a potent nanomolar ligand for the bromodomain of human BRD9 with small residual micromolar affinity toward the bromodomain of BRD4. Binding of the lead compound 11 to the bromodomain of BRD9 results in an unprecedented rearrangement of residues forming the acetyllysine recognition site, affecting plasticity of the protein in an induced-fit pocket. The compound does not exhibit any cytotoxic effect in HEK293 cells and displaces the BRD9 bromodomain from chromatin in bioluminescence proximity assays without affecting the BRD4/histone complex. The 2-amine-9H-purine scaffold represents a novel template that can be further modified to yield highly potent and selective tool compounds to interrogate the biological role of BRD9 in diverse cellular systems." /></meta><meta name="twitter:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-6/jm501893k/20150418/images/large/jm-2014-01893k_0007.jpeg" /></meta><meta name="twitter:title" content="9H-Purine Scaffold Reveals Induced-Fit Pocket Plasticity of the BRD9 Bromodomain" /></meta>
        
            <meta property="og:type" content="Article" /></meta>
        

        
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/jm501893k"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/ke1q~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-6ee29527b3d8ff00a106.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/jm501893k">
            
        





































    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-12/achre4.2021.54.issue-12/20210615/achre4.2021.54.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-6/amrcda.2021.2.issue-6/20210625/amrcda.2021.2.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-6/aabmcb.2021.4.issue-6/20210621/aabmcb.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-6/aaembp.2021.3.issue-6/20210622/aaembp.2021.3.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-6/aaemcq.2021.4.issue-6/20210628/aaemcq.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-25/aamick.2021.13.issue-25/20210630/aamick.2021.13.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-6/aanmf6.2021.4.issue-6/20210625/aanmf6.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-6/aapmcd.2021.3.issue-6/20210611/aapmcd.2021.3.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-6/abseba.2021.7.issue-6/20210614/abseba.2021.7.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-12/accacs.2021.11.issue-12/20210618/accacs.2021.11.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-6/acscii.2021.7.issue-6/20210623/acscii.2021.7.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-6/acbcct.2021.16.issue-6/20210618/acbcct.2021.16.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-3/achsc5.2021.28.issue-3/20210524/achsc5.2021.28.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-12/acncdm.2021.12.issue-12/20210616/acncdm.2021.12.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-6/aesccq.2021.5.issue-6/20210617/aesccq.2021.5.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-6/aelccp.2021.6.issue-6/20210611/aelccp.2021.6.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-6/aeecco.2021.1.issue-6/20210611/aeecco.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-6/aewcaa.2021.1.issue-6/20210611/aewcaa.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-5/afsthl.2021.1.issue-5/20210618/afsthl.2021.1.issue-5.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-6/aidcbc.2021.7.issue-6/20210611/aidcbc.2021.7.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-6/amlccd.2021.10.issue-6/20210615/amlccd.2021.10.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-6/amlcef.2021.3.issue-6/20210607/amlcef.2021.3.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-6/amclct.2021.12.issue-6/20210610/amclct.2021.12.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-6/ancac3.2021.15.issue-6/20210622/ancac3.2021.15.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-25/acsodf.2021.6.issue-25/20210629/acsodf.2021.6.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-6/apchd5.2021.8.issue-6/20210616/apchd5.2021.8.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-6/ascefj.2021.6.issue-6/20210625/ascefj.2021.6.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-25/ascecg.2021.9.issue-25/20210628/ascecg.2021.9.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-6/asbcd6.2021.10.issue-6/20210618/asbcd6.2021.10.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-25/ancham.2021.93.issue-25/20210629/ancham.2021.93.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-25/bichaw.2021.60.issue-25/20210629/bichaw.2021.60.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-6/bcches.2021.32.issue-6/20210616/bcches.2021.32.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-6/bomaf6.2021.22.issue-6/20210614/bomaf6.2021.22.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-24/cgeabj.2021.99.issue-24/20210628-01/cgeabj.2021.99.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-6/crtoec.2021.34.issue-6/20210621/crtoec.2021.34.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-12/chreay.2021.121.issue-12/20210623/chreay.2021.121.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-12/cmatex.2021.33.issue-12/20210622/cmatex.2021.33.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-6/cgdefu.2021.21.issue-6/20210602/cgdefu.2021.21.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-12/enfuem.2021.35.issue-12/20210617/enfuem.2021.35.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-12/esthag.2021.55.issue-12/20210615/esthag.2021.55.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-6/estlcu.2021.8.issue-6/20210608/estlcu.2021.8.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-25/iecred.2021.60.issue-25/20210630/iecred.2021.60.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-12/inocaj.2021.60.issue-12/20210621/inocaj.2021.60.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-6/jaaucr.2021.1.issue-6/20210628/jaaucr.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-25/jafcau.2021.69.issue-25/20210630/jafcau.2021.69.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-6/jceaax.2021.66.issue-6/20210610/jceaax.2021.66.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-6/jceda8.2021.98.issue-6/20210608/jceda8.2021.98.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-6/jcisd8.2021.61.issue-6/20210628/jcisd8.2021.61.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-6/jctcce.2021.17.issue-6/20210608/jctcce.2021.17.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/jmcmar.2021.64.issue-12/20210624/jmcmar.2021.64.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-6/jnprdf.2021.84.issue-6/20210625/jnprdf.2021.84.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-12/joceah.2021.86.issue-12/20210618/joceah.2021.86.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-24/jpcafh.2021.125.issue-24/20210624/jpcafh.2021.125.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-24/jpcbfk.2021.125.issue-24/20210624/jpcbfk.2021.125.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-24/jpccck.2021.125.issue-24/20210624/jpccck.2021.125.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-24/jpclcd.2021.12.issue-24/20210624/jpclcd.2021.12.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-6/jprobs.2021.20.issue-6/20210604/jprobs.2021.20.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-25/jacsat.2021.143.issue-25/20210630/jacsat.2021.143.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-6/jamsef.2021.32.issue-6/20210602/jamsef.2021.32.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-25/langd5.2021.37.issue-25/20210629/langd5.2021.37.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-12/mamobx.2021.54.issue-12/20210622/mamobx.2021.54.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-6/mpohbp.2021.18.issue-6/20210607/mpohbp.2021.18.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-12/nalefd.2021.21.issue-12/20210623/nalefd.2021.21.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-12/orlef7.2021.23.issue-12/20210618/orlef7.2021.23.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-6/oprdfk.2021.25.issue-6/20210618/oprdfk.2021.25.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-12/orgnd7.2021.40.issue-12/20210628/orgnd7.2021.40.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Fjm501893k&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Fjm501893k&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/jm501893k&amp;href=/doi/10.1021/jm501893k" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2015</span><span class="cit-fg-volume">, 58</span><span class="cit-fg-issue">, 6</span><span class="cit-fg-pageRange">, 2718-2736</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/58/6" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/jm501889t" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/jm501908a" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">9<i>H</i>-Purine Scaffold Reveals Induced-Fit Pocket Plasticity of the BRD9 Bromodomain</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Sarah++Picaud">Sarah Picaud</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Maria++Strocchia">Maria Strocchia</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Stefania++Terracciano">Stefania Terracciano</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Gianluigi++Lauro">Gianluigi Lauro</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jacqui++Mendez">Jacqui Mendez</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Danette+L.++Daniels">Danette L. Daniels</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Raffaele++Riccio">Raffaele Riccio</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Giuseppe++Bifulco">Giuseppe Bifulco</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Ines++Bruno">Ines Bruno</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Panagis++Filippakopoulos">Panagis Filippakopoulos</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">†</span> <span class="aff-text">Nuffield Department of Clinical Medicine, Structural Genomics Consortium, University of Oxford, Old Road Campus Research Building, Roosevelt Drive, Oxford OX3 7DQ, U.K.</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">‡</span> <span class="aff-text">Department of Pharmacy, University of Salerno, Via Giovanni Paolo II, 132, 84084 Fisciano, Italy</span></div><div class="aff-info" id="aff3"><span class="aff-symbol">§</span> <span class="aff-text">Promega Corporation, 2800 Woods Hollow Road, Madison, Wisconsin 53711, United States</span></div><div class="aff-info" id="aff4"><span class="aff-symbol">∥</span> <span class="aff-text">Nuffield Department of Clinical Medicine, Ludwig Cancer Research, University of Oxford, Old Road Campus Research Building, Roosevelt Drive, Oxford OX3 7DQ, U.K.</span></div><div class="corresp-info"><strong>*</strong>G.B.: e-mail, <a href="/cdn-cgi/l/email-protection#385a515e4d545b57784d56514b5916514c"><span class="__cf_email__" data-cfemail="ff9d96998a939c90bf8a91968c9ed1968b">[email protected]</span></a>; phone, +39 (0) 89 969741; fax, +39 (0) 89 962804.</div><div class="corresp-info"><strong>*</strong>I.B.: e-mail, <a href="/cdn-cgi/l/email-protection#bad8c8cfd4d5d3d4facfd4d3c9db94d3ce"><span class="__cf_email__" data-cfemail="b3d1c1c6dddcdaddf3c6dddac0d29ddac7">[email protected]</span></a>; phone, +39 (0) 89 969743; fax, +39 (0) 89 962804.</div><div class="corresp-info"><strong>*</strong>P.F.: e-mail, <a href="/cdn-cgi/l/email-protection#5525343b34323c267b333c393c2525343e3a253a20393a26152632367b3a2d7b34367b203e"><span class="__cf_email__" data-cfemail="5c2c3d323d3b352f723a3530352c2c3d37332c332930332f1c2f3b3f723324723d3f722937">[email protected]</span></a>; phone, +44 (0) 1865 617759; fax, +44 (0) 1865 617575.</div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Fjm501893k&amp;href=/doi/10.1021%2Fjm501893k" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2015</span></span><span class="cit-volume">, 58</span><span class="cit-issue">, 6</span><span class="cit-pageRange">, 2718–2736</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">February 22, 2015</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>5 December 2014</li><li><span class="item_label"><b>Published</b> online</span>5 March 2015</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 26 March 2015</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/jm501893k" title="DOI URL">https://doi.org/10.1021/jm501893k</a></div><div class="article_header-article-copyright"><strong>Copyright © 2015 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/page/rightslinkno.jsp">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"><a href="/page/policy/authorchoice/index.html"><img src="/pb-assets/images/programsLogos/OA_Icons_Ac-1612971054130.svg" alt="ACS AuthorChoice" /></img><span>ACS AuthorChoice</span></a><span class="article_header-cc-by text">with CC-BY<a href="/page/policy/authorchoice_ccby_termsofuse.html" title="License">license</a></span></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/jm501893k"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">3340</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">49</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/jm501893k" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;9H-Purine Scaffold Reveals Induced-Fit Pocket Plasticity of the BRD9 Bromodomain&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Sarah&quot;,&quot;last_name&quot;:&quot;Picaud&quot;},{&quot;first_name&quot;:&quot;Maria&quot;,&quot;last_name&quot;:&quot;Strocchia&quot;},{&quot;first_name&quot;:&quot;Stefania&quot;,&quot;last_name&quot;:&quot;Terracciano&quot;},{&quot;first_name&quot;:&quot;Gianluigi&quot;,&quot;last_name&quot;:&quot;Lauro&quot;},{&quot;first_name&quot;:&quot;Jacqui&quot;,&quot;last_name&quot;:&quot;Mendez&quot;},{&quot;first_name&quot;:&quot;Danette&quot;,&quot;last_name&quot;:&quot;L. Daniels&quot;},{&quot;first_name&quot;:&quot;Raffaele&quot;,&quot;last_name&quot;:&quot;Riccio&quot;},{&quot;first_name&quot;:&quot;Giuseppe&quot;,&quot;last_name&quot;:&quot;Bifulco&quot;},{&quot;first_name&quot;:&quot;Ines&quot;,&quot;last_name&quot;:&quot;Bruno&quot;},{&quot;first_name&quot;:&quot;Panagis&quot;,&quot;last_name&quot;:&quot;Filippakopoulos&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2015&quot;,&quot;month&quot;:&quot;03&quot;,&quot;day&quot;:&quot;05&quot;,&quot;issue&quot;:&quot;6&quot;,&quot;volume&quot;:&quot;58&quot;,&quot;pages&quot;:&quot;2718-2736&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/jm501893k&quot;},&quot;abstract&quot;:&quot;The 2-amine-9H-purine scaffold was identified as a weak bromodomain template and was developed via iterative structure based design into a potent nanomolar ligand for the bromodomain of human BRD9 with small residual micromolar affinity toward the bromodomain of BRD4. Binding of the lead compound 11 to the bromodomain of BRD9 results in an unprecedented rearrangement of residues forming the acetyllysine recognition site, affecting plasticity of the protein in an induced-fit pocket. The compound does not exhibit any cytotoxic effect in HEK293 cells and displaces the BRD9 bromodomain from chromatin in bioluminescence proximity assays without affecting the BRD4/histone complex. The 2-amine-9H-purine scaffold represents a novel template that can be further modified to yield highly potent and selective tool compounds to interrogate the biological role of BRD9 in diverse cellular systems.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Fjm501893k&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm501893k" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm501893k&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=jm501893k" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm501893k&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=jm501893k" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Fjm501893k&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm501893k&amp;href=/doi/10.1021/jm501893k" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/jm501893k" /></input><a href="/doi/pdf/10.1021/jm501893k" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (5 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/jm501893k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Fjm501893k%26sid%3Dliteratum%253Aachs%26pmid%3D25703523%26genre%3Darticle%26aulast%3DPicaud%26date%3D2015%26atitle%3D9H-Purine%2BScaffold%2BReveals%2BInduced-Fit%2BPocket%2BPlasticity%2Bof%2Bthe%2BBRD9%2BBromodomain%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D58%26issue%3D6%26spage%3D2718%26epage%3D2736%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=292201" title="Scaffolds">Scaffolds</a>,</li><li><a href="/action/doSearch?ConceptID=292529" title="Aromatic compounds">Aromatic compounds</a>,</li><li><a href="/action/doSearch?ConceptID=292541" title="Cavities">Cavities</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/58/6" title="Go to Volume 58, Issue 6"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-6/jmcmar.2015.58.issue-6/20150326/jmcmar.2015.58.issue-6.largecover.jpg" alt="Go to Volume 58, Issue 6"/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-6/jm501893k/20150418/images/medium/jm-2014-01893k_0007.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-6/jm501893k/20150418/images/large/jm-2014-01893k_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm501893k&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">The 2-amine-9<i>H</i>-purine scaffold was identified as a weak bromodomain template and was developed via iterative structure based design into a potent nanomolar ligand for the bromodomain of human BRD9 with small residual micromolar affinity toward the bromodomain of BRD4. Binding of the lead compound <b>11</b> to the bromodomain of BRD9 results in an unprecedented rearrangement of residues forming the acetyllysine recognition site, affecting plasticity of the protein in an induced-fit pocket. The compound does not exhibit any cytotoxic effect in HEK293 cells and displaces the BRD9 bromodomain from chromatin in bioluminescence proximity assays without affecting the BRD4/histone complex. The 2-amine-9<i>H</i>-purine scaffold represents a novel template that can be further modified to yield highly potent and selective tool compounds to interrogate the biological role of BRD9 in diverse cellular systems.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_86674" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_86674" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The term “epigenome” describes the array of chemical modifications on DNA and histone proteins that dynamically regulate gene expression.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> The complex set of reversible post-translational modifications (PTMs) that decorate histones ultimately determines the overall state of chromatin, giving rise to the so-called “histone code”, a cellular language involving specific proteins that introduce (writers), remove (erasers), or recognize (readers) PTMs.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> ε-N-acetylation of lysine residues is a widespread post-translational mark,<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> deposited on proteins throughout the entire proteome by lysine acetyl transferases (KATs), removed by lysine deacetylases (KDACs), and recognized by evolutionary conserved protein modules such as bromodomains<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> (named after the <i>Drosophila melanogaster</i><i>brahma</i> gene<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a>) as well as the more recently discovered YEATS domains.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> The human proteome encodes 61 bromodomains (BRDs) found on 42 diverse proteins, including histone acetyl transferases (HATs) and HAT-associated proteins such as GCN5, PCAF, and bromodomain 9 (BRD9),<a onclick="showRef(event, 'ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref7 ref8">(7, 8)</a> transcriptional coactivators such as the TAF and TRIM/TIF proteins,<a onclick="showRef(event, 'ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref9 ref10">(9, 10)</a> ATP-dependent chromatin remodeling complexes such as BAZ1B,<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> helicases such as SMARCA,<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> nuclear scaffolding proteins such as the polybromo PB1<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> as well as transcriptional regulators, such as the bromo and extra terminal (BET) proteins.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> The family of human BRD modules has almost been completely structurally characterized resulting in a recent classification of eight distinct structural subfamilies.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a></div><div class="NLM_p">The BET subfamily of BRDs has attracted a lot of attention, as its members (BRD2, BRD3, BRD4, and the testis specific BRDT) play a central role in cell cycle progression, cellular proliferation, and apoptosis.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> BETs contain two N-terminal BRD modules that interact with acetylated histones,<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> transcription factors<a onclick="showRef(event, 'ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref17 ref18">(17, 18)</a> or other acetylated transcriptional regulators,<a onclick="showRef(event, 'ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref19 ref20">(19, 20)</a> an extra terminal (ET) recruitment domain,<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> and a C-terminal motif (in the case of BRD4 and BRDT) responsible for the recruitment of the positive transcription elongation factor B (P-TEFb),<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> ultimately controlling transcriptional elongation. Importantly, BET BRDs have been successfully targeted by small molecule inhibitors, such as the triazolothienodiazepine (+)-JQ1<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> and the triazolobenzodiazepine IBET762,<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> which were identified employing phenotypic screening and have in the past few years consolidated the emerging role of BRDs as viable therapeutic targets. Indeed, BET inhibition suppresses tumor growth in diverse mouse models of cancer, such as NUT midline carcinoma, acute myeloid and mixed lineage leukemia, multiple myeloma, glioblastoma, melanoma, Burkitt’s lymphoma, neuroblastoma and prostate cancer, leading to a number of clinical trials seeking to modulate BET function in diverse tumor settings.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a></div><div class="NLM_p">The initial success targeting BET proteins and the availability of robust recombinant systems of expression as well as biophysical assays to probe BET–ligand interactions have spawned a number of medicinal chemistry efforts seeking to identify novel scaffolds that can block binding of acetylated lysines to these protein interaction modules. Phenotypic screening, molecular docking, and fragment-based approaches have emerged as successful tools for discovering other Kac mimetics, leading to the identification of a number of new chemotypes, including 3,4-dimethylisoxazoles,<a onclick="showRef(event, 'ref26 ref27'); return false;" href="javascript:void(0);" class="ref ref26 ref27">(26, 27)</a> 3-methyl-3,4-dihydroquinazolinones,<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> indolizinethanones, <i>N</i>-phenylacetamides, and <i>N</i>-acety-2-methyltetrahydroquinolines<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> triazolopyrimidines, methylquinoline, and chloropyridones,<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> thiazolidinones,<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> 4-acylpyrroles,<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> and triazolophtalazines<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> (summerized in Chart <a class="ref internalNav" href="#cht1" aria-label="1">1</a>). Starting from fragment hits, highly potent and selective BET inhibitors have also emerged,<a onclick="showRef(event, 'ref34 ref35'); return false;" href="javascript:void(0);" class="ref ref34 ref35">(34, 35)</a> suggesting that it is possible to access new chemical space for inhibitor development via initial fragment screening. Druggability analysis based on the available structural information<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> suggests that it is possible to target most BRD structural classes.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> In fact, BRDs annotated as “hard” to target, such as BAZ2B, have recently been successfully targeted by highly selective and potent inhibitors<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> and fragment based approaches have now yielded chemotypes targeting BRDs outside the BET family, including CREBBP/p300,<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> ATAD2,<a onclick="showRef(event, 'ref39 ref40'); return false;" href="javascript:void(0);" class="ref ref39 ref40">(39, 40)</a> BAZ2B,<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> and BRPF1.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a></div><figure id="cht1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Chart 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-6/jm501893k/20150418/images/medium/jm-2014-01893k_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-6/jm501893k/20150418/images/large/jm-2014-01893k_0009.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Chart 1. Acetyllysine Mimetic Templates Reported To Bind to Bromodomain Proteins<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-6/jm501893k/20150418/images/large/jm-2014-01893k_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm501893k&amp;id=cht1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Chart a</span>The Kac mimetic portion of each substructure is highlighted in red. (a) 3,4-Dimethylisoxazole; (b) 3-methyl-3,4-dihydroquinazolinone; (c) indolizinethanone; (d) <i>N</i>-phenylacetamide; (e) <i>N</i>-acetyl-2-methyltetrahydroquinoline; (f) triazolopyrimidine; (g) methylquinoline; (h) chloropyridone; (i) thiazolidinone; (j) 4-acylpyrrole; (k) triazolophtalazine; (l) methyltriazoles.</p></p></figure><div class="NLM_p">An unexpected but interesting finding that recently attracted attention was the identification of clinically approved kinase inhibitors and tool compounds that exhibited binding with high affinity, and selectivity across the human BRD family, to BET BRDs.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> Crystal structures with the first bromodomain of BRD4 revealed acetyllysine mimetic binding of the PLK inhibitor BI-2536 and the JAK inhibitor fedratinib without any significant distortion of the inhibitors when compared to kinase complexes, suggesting the potential to develop polypharmacology targeting both BRD and kinases at the same time.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> Interestingly, the cyclin-dependent kinase inhibitor dinaciclib was also identified as a binder of BRD4,<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> suggesting that other inhibitor classes might be good starting points for developing inhibitors for BRDs.</div><div class="NLM_p">In light of the successful fragment-based programs and the reliability for discovering BRDs inhibitors, we started to investigate the purine scaffold as a putative Kac mimetic. Purine is a privileged chemical template, as it is one of the most abundant N-based heterocycles in nature,<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> and a number of purine-based drugs are currently approved and used for the treatment of cancer (6-mercaptopurine, 6-thioguanine), viral infections such as AIDS and herpes (carbovir, abacavir, acyclovir, ganciclovir), hairy cell leukemia (cladribine), and organ rejection (azathioprine).<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> Moreover, purine based compounds have emerged as reliable chemical–biology tools, since they can interact with a variety of biological targets involved in a number of diseases. Some such examples include their activity as microtubule (myoseverin), 90-heat shock protein (PU3), sulfotransferase (NG38), adenosine receptor (KW-6002), and cyclin-dependent kinase (olomoucine, Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>A; roscovitine) inhibitors.<a onclick="showRef(event, 'ref47 ref48'); return false;" href="javascript:void(0);" class="ref ref47 ref48">(47, 48)</a></div><figure id="fig1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-6/jm501893k/20150418/images/medium/jm-2014-01893k_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-6/jm501893k/20150418/images/large/jm-2014-01893k_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Purine fragments bind to human bromodomains. (A) Structure of olomoucine, a potent cyclin-dependent kinase inhibitor and numbering of the 2-amine-9<i>H</i>-purine core scaffold. (B) Purine fragments tested as bromodomain ligands. (C) Docking pose of <b>1</b> (yellow stick representation) onto the bromodomain of BRD4(1) positions the bulky halogen group on the top of the binding pocket. The protein is shown as a white ribbon (starting model, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4MEN">4MEN</a>) or in magenta (docked model) with characteristic residues shown as sticks in the same color scheme. (D) Alternative binding of compound <b>1</b> into the bromodomain of BRD4(1) with the 6-chloro substituent adopting an acetyllysine mimetic pose. The compound and protein are colored as in (C). PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4MEN">4MEN</a> was used as the starting model for docking. (E) Docking of compound <b>2a</b> positions the 2-amine-9<i>H</i>-purine ring system of the ligand within the Kac cavity, sterically packing between V87 and I146, suggesting that this scaffold topology can be further utilized to target bromodomains. The same color scheme as in (C) and (D) is used with the ligand shown as orange sticks. PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4MEN">4MEN</a> was used as the starting model for docking. (F) Fragments were tested in a thermal shift assay against bromodomains of the BET subfamily as well as representative members from other families. Thermal shifts are color-coded as indicated in the inset. 9<i>H</i>-Purines showed weak binding across the panel.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-6/jm501893k/20150418/images/large/jm-2014-01893k_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm501893k&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Computational analyses followed by in vitro evaluation of purine-based fragments identified this scaffold as a novel effective Kac mimetic. Interestingly, initial purine fragment hits also demonstrated affinity outside the BET subfamily of BRDs, toward the less explored BRDs of PB1, CREBBP, and BRD9. To our knowledge, the only known BRD9 inhibitors today show cross-reactivity toward BET BRDs and CREBBP employing a triazolophthalazine template.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> The precise biological function of BRD9 remains elusive, although it has been identified as a component of the SWI/SNF complex<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> and has been associated with a number of different cancer types, including non-small-cell lung cancer,<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> cervical,<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> and hepatocellular carcinoma.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> Notably, its BRD reader module has been frequently found mutated in lung squamous cell carcinoma, prostate adenocarcinoma, and uterine corpus endometrial carcinoma.<a onclick="showRef(event, 'ref53 ref54 ref55'); return false;" href="javascript:void(0);" class="ref ref53 ref54 ref55">(53-55)</a></div><div class="NLM_p">We chose a small purine fragment, compound <b>2a</b> (Chart <a class="ref internalNav" href="#cht2" aria-label="2">2</a>), from our fragment hits for structural optimization and found that some of its 6-aryl derivatives exhibited nanomolar affinity toward BRD9, with lower activity toward BRD4. Importantly, the developed inhibitors induced an appreciable change in the three-dimensional shape of the receptor binding cavity, indicating that their binding occurs through an “induced-fit” mechanism.<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> Previously an induced fit binding was observed when a dihydroquinoxalinone was shown to insert under an arginine residue of the CREBBP BRD, resulting in restructuring of the Kac binding site of this bromodomain.<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> In our case the rearrangement of the binding site of BRD9 was more extensive, with several side chains rotating and shifting to accommodate the small purine ligand. The optimized compound <b>11</b> (Chart <a class="ref internalNav" href="#cht2" aria-label="2">2</a>) exhibited nanomolar affinity for BRD9 with weaker micromolar affinity for BRD4 in vitro, displaced the BRD9 BRD from chromatin without affecting BRD4 binding to histones, and did not show any cytotoxicity toward human embryonic kidney cells. Our work establishes the proof-of-concept of using 2-amino-9<i>H</i>-purines as a starting point to develop Kac mimetic compounds targeting BRDs outside the BET family, with compound <b>11</b> representing a promising low nanomolar starting point toward the discovery of selective BRD9 ligands to be used as chemical probes for deep biological and pharmacological investigation of BRD9.</div><figure id="cht2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Chart 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-6/jm501893k/20150418/images/medium/jm-2014-01893k_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-6/jm501893k/20150418/images/large/jm-2014-01893k_0010.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Chart 2. Chemical Structures of Compounds <b>1</b>–<b>11</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-6/jm501893k/20150418/images/large/jm-2014-01893k_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm501893k&amp;id=cht2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i5">Experimental Details</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_30351" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_30351" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Input Files Preparation for Docking</h3><div class="NLM_p last">Protein 3D models of BRD4 (first bromodomain, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4MEN">4MEN</a><a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a>) and BRD9 (apo form, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3HME">3HME</a><a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a>) were prepared using the Schrödinger Protein Preparation Wizard workflow (Schrödinger, LLC, New York, NY, 2013). Briefly, water molecules that were found 5 Å or more away from heteroatom groups were removed and cap termini were included. Additionally, all hydrogen atoms were added, and bond orders were assigned. The resulting PDB files were converted to the MAE format. Chemical structures of investigated compounds were built with Maestro’s Build Panel (version 9.6; Schrödinger, LLC, New York, NY, 2013) and subsequently processed with LigPrep (version 2.8; Schrödinger, LLC, New York, NY, 2013) in order to generate all the possible stereoisomers, tautomers, and protonation states at a pH of 7.4 ± 1.0; the resulting ligands were finally minimized employing the OPLS 2005 force field.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Induced Fit Docking</h3><div class="NLM_p last">Binding sites for the initial Glide<a onclick="showRef(event, 'ref58 ref59 ref60'); return false;" href="javascript:void(0);" class="ref ref58 ref59 ref60">(58-60)</a> (version 6.1, Schrödinger, LLC, New York, NY, 2013) docking phases of the Induced Fit Workflow (Induced Fit Docking protocol 2013-3, Glide version 6.1, Prime version 3.4, Schrödinger, LLC, New York, NY, 2013)<a onclick="showRef(event, 'ref61 ref62'); return false;" href="javascript:void(0);" class="ref ref61 ref62">(61, 62)</a> were calculated on the BRD4(1) and BRD9 structures, considering the centroid of the cocrystallized ligand (<i>N</i>,5-dimethyl-<i>N</i>-(4-methylbenzyl)[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-amine for BRD4(1), from PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4MEN">4MEN</a>), or cocrystallized ligand <b>7d</b> (from the BRD9/<b>7d</b> complex), or Tyr106 (for BRD9, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3HME">3HME</a>) for grid generation. In all cases, cubic inner boxes with dimensions of 10 Å were applied to the proteins, and outer boxes were automatically detected. Ring conformations of the investigated compounds were sampled using an energy window of 2.5 kcal/mol; conformations featuring nonplanar conformations of amide bonds were penalized. Side chains of residues close to the docking outputs (within 8.0 Å of ligand poses) were reoriented using Prime (version 3.4, Schrödinger, LLC, New York, NY, 2013), and ligands were redocked into their corresponding low energy protein structures (Glide Standard Precision Mode), considering inner boxes dimensions of 5.0 Å (outer boxes automatically detected), with resulting complexes ranked according to GlideScore.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Rigid Docking</h3><div class="NLM_p last">Calculations were performed using the Glide software package (SP mode, version 6.1, Schrödinger package)<a onclick="showRef(event, 'ref58 ref59 ref60'); return false;" href="javascript:void(0);" class="ref ref58 ref59 ref60">(58-60)</a> in order to determine the binding mode of compound <b>11</b> into the acetyllysine cavity of BRD9. First the receptor grid was generated focused on the BRD9 binding site taking as a reference structure the experimentally determined complex structure of compound <b>7d</b> with BRD9, with inner- and outer-box dimensions of 10 × 10 × 10 and 21.71 × 21.71 × 21.71, respectively. Standard Precision (SP) Glide mode was employed accounting for compound flexibility in the sampling of compound <b>11</b>. The sampling step was set to expanded sampling mode (4 times), keeping 10 000 ligand poses for the initial phase of docking, followed by 800 ligand poses selection for energy minimization. A maximum number of 50 output structures were saved for each ligand, with a scaling factor of 0.8 related to van der Waals radii with a partial charge cutoff of 0.15. A postdocking optimization of the obtained docking outputs was performed, accounting for a maximum of 50 poses based on a 0.5 kcal/mol rejection cutoff for the obtained minimized poses.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> General Synthetic Information</h3><div class="NLM_p last">All commercially available starting materials were purchased from Sigma-Aldrich and were used as received. Solvents used for the synthesis were of HPLC grade and were purchased from Sigma-Aldrich or Carlo Erba Reagenti. NMR spectra were recorded on Bruker Avance 600 or 300 MHz instruments. Compounds were dissolved in 0.5 mL of MeOD, CDCl<sub>3</sub>, or DMSO-<i>d</i><sub>6</sub>. Coupling constants (<i>J</i>) are reported in hertz, and chemical shifts are expressed in parts per million (ppm) on the δ scale relative to the solvent peak as internal reference. <sup>13</sup>C spectra are reported for representative compounds. Electrospray mass spectrometry (ESI-MS) was performed on a LCQ DECA TermoQuest (San Jose, CA, USA) mass spectrometer. Chemical reactions were monitored on silica gel 60 F<sub>254</sub> plates (Merck), and spots were visualized under UV light. Analytical and semipreparative reversed-phase HPLC were performed on an Agilent Technologies 1200 series high performance liquid chromatography system using Jupiter Proteo C<sub>18</sub> reversed-phase columns ((a) 250 mm × 4.60 mm, 4 μm, 90 Å, flow rate = 1 mL/min; (b) 250 mm × 10.00 mm, 10 μm, 90 Å, flow rate = 4 mL/min respectively, Phenomenex). The binary solvent system (A/B) was as follows: 0.1% TFA in water (A) and 0.1% TFA in CH<sub>3</sub>CN (B). Absorbance was detected at 240 nm. The purity of all tested compound (>98%) was determined by HPLC analysis. Microwave irradiation reactions were carried out in a dedicated CEM-Discover focused microwave synthesis apparatus, operating with continuous irradiation power from 0 to 300 W utilizing the standard absorbance level of 300 W maximum power. Reactions were carried out in 10 mL sealed microwave glass vials. The Discover system also included controllable ramp time, hold time (reaction time), and uniform stirring. After the irradiation period, reaction vessels were cooled rapidly (60–120 s) to ambient temperature by air jet cooling. Compounds synthesized are shown in Chart <a class="ref internalNav" href="#cht2" aria-label="2">2</a>.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> General Procedure <i>a</i> for the Suzuki–Miyaura Cross-Coupling of Free Halopurines (<b>3a</b>,<b>b</b>, <b>3d</b>, <b>3f</b>–<b>h</b>, <b>4a</b>–<b>d</b>, <b>5a</b>, <b>6a</b>–<b>c</b>, <b>7a</b>–<b>e</b>, <b>8a</b>, <b>9a</b>,<b>b</b>, <b>10</b>, <b>11</b>)</h3><div class="NLM_p last">2-Amino-6-bromopurine (50.0 mg, 0.23 mmol), commercially available boronic acids (0.29 mmol), Pd(OAc)<sub>2</sub> (2.70 mg, 0.012 mmol), P(C<sub>6</sub>H<sub>4</sub>SO<sub>3</sub>Na)<sub>3</sub> (34.0 mg, 0.06 mmol), and Cs<sub>2</sub>CO<sub>3</sub> (228.0 mg, 0.70 mmol) were added to a 10 mL microwave vial equipped with a magnetic stirrer. The vial was evacuated and backfilled with nitrogen three times. Degassed acetonitrile (0.5 mL) and degassed water (1.0 mL) were added by means of an airtight syringe. The mixture was heated under microwave irradiation at 150 °C for 5–15 min. After irradiation, the vial was cooled to ambient temperature by air jet cooling and a mixture of cold water and HCl (1.5 M) was added (5.0 and 2.0 mL, respectively). The mixture was subsequently poured into crushed ice and then left at 4 °C overnight. The resulting precipitate was filtered and purified by HPLC to give the desired product in good yields (53–90%). HPLC purification was performed by semipreparative reversed-phase HPLC using the gradient conditions reported below for each compound. The final products were obtained with high purity (>95%) detected by HPLC analysis and were fully characterized by ESI-MS and NMR spectroscopy.</div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> General Procedure <i>b</i> for TBAF-Assisted N9-Alkylation of Purine Rings (<b>2b</b>, <b>3c</b>, <b>3e</b>, <b>4e</b>, <b>5b</b>, <b>8b</b>, <b>8c</b>)</h3><div class="NLM_p last">2-Amino-6-arylpurine (0.1 mmol) was dissolved in 0.4 mL of THF at room temperature. To this mixture 0.2 mL (0.2 mmol) of TBAF (1.0 M solution in THF, Aldrich) and iodomethane (12.5 μL, 0.2 mmol) or chloroacetone (16.0 μL, 0.2 mmol) were added. The reaction was stirred at room temperature for 10 min. Water was added, and the aqueous layer was extracted three times with dichloromethane. The combined organic layers were washed with water, dried with anhydrous Na<sub>2</sub>SO<sub>4</sub> and concentrated under vacuum. The crude mixture was purified by semipreparative reversed-phase HPLC using the gradient conditions reported below for each compound. Compounds were obtained in good yields (50–88%) and high purity (>98%) and were fully characterized by ESI-MS and NMR spectroscopy.</div></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> General Procedure <i>c</i> for the Synthesis of 2-Hydroxyl-6-arylpurines (<b>8d</b>–<b>f</b>)</h3><div class="NLM_p last">A three-necked flask was charged with the 2-amino-6-arylpurine derivative (<b>7b</b>–<b>d</b>) (0.5 mmol) and 50% H<sub>2</sub>SO<sub>4</sub> (2.0 mL). The mixture was stirred at room temperature for 30 min and then cooled to −5 °C. A solution of NaNO<sub>2</sub> (48.3 mg, 0.7 mmol) in H<sub>2</sub>O (200 μL) was added dropwise, and the release of nitrogen gas was immediately observed. The reaction mixture was then stirred at −10 °C for 2 h, and urea (24.0 mg, 0.4 mmol) was added to decompose the excess of NaNO<sub>2</sub>. The mixture was then stirred at 50 °C for 1 h and neutralized with 50% NaOH solution, diluted with water, and extracted three times with EtOAc. The combined organic layers were dried with anhydrous Na<sub>2</sub>SO<sub>4</sub> and concentrated under vacuum. The crude mixture was purified by semipreparative reversed-phase HPLC to get the pure products in good yields (47–63%). Compounds were fully characterized by ESI-MS and NMR spectroscopy.</div></div><div id="sec2_7_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> 2-Amino-6-bromo-9-methylpurine (<b>2b</b>)</h3><div class="NLM_p last"><b>2b</b> was obtained from commercially available <b>2a</b> following general procedure <i>b</i> as a yellow powder in 85% yield. RP-HPLC: <i>t</i><sub>R</sub> = 12.4 min, gradient condition from 5% B to 100% B in 95 min, flow rate of 4 mL/min, λ = 240 nm. <sup>1</sup>H NMR (600 MHz, MeOD): δ = 3.74 (s, 3H), 8.25 (s, 1H). <sup>13</sup>C NMR (150 MHz, MeOD): δ = 30.6, 126.3, 142.5, 149.4, 155.62, 160.8. ESI-MS, calculated for C<sub>6</sub>H<sub>6</sub>BrN<sub>5</sub> 227.0; found <i>m</i>/<i>z</i> = 228.1 [M + H]<sup>+</sup>.</div></div><div id="sec2_7_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> 2-Amino-6-phenyl-9<i>H</i>-purine (<b>3a</b>)</h3><div class="NLM_p last"><b>3a</b> was obtained following general procedure <i>a</i> as a white powder in 90% yield from <b>2a</b> and phenylboronic acid <b>13</b>. RP-HPLC <i>t</i><sub>R</sub> = 12.1 min, gradient condition from 5% B to 100% B in 60 min, flow rate of 4 mL/min, λ = 240 nm. Spectral data were in accord with previously published data.<a onclick="showRef(event, 'ref63'); return false;" href="javascript:void(0);" class="ref ref63">(63)</a></div></div><div id="sec2_7_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> 2-Amino-6-(4-methoxyphenyl)-9<i>H</i>-purine (<b>3b</b>)</h3><div class="NLM_p last"><b>3b</b> was obtained following general procedure <i>a</i> as a pale yellow powder in 86% yield from <b>2a</b> and 4-methoxyphenylboronic acid <b>14</b>. RP-HPLC <i>t</i><sub>R</sub> = 14.6 min, gradient condition from 5% B to 100% B in 65 min, flow rate of 4 mL/min, λ = 240 nm. Spectral data were in accord with previously published data.<a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(64)</a></div></div><div id="sec2_7_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> 2-Amino-6-(4-methoxyphenyl)-9-methylpurine (<b>3c</b>)</h3><div class="NLM_p last"><b>3c</b> was obtained following general procedure <i>b</i> as a yellow powder in 88% yield from <b>3b</b>. RP-HPLC <i>t</i><sub>R</sub> = 15.5 min, gradient condition from 5% B to 100% B in 65 min, flow rate of 4 mL/min, λ = 240 nm. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ = 3.79 (s, 3H), 3.94 (s, 3H), 7.12 (d, <i>J</i> = 8.6 Hz, 2H), 7.95 (s, 1H), 8.55 (d, <i>J</i> = 8.5 Hz, 2H). ESI-MS, calculated for C<sub>13</sub>H<sub>13</sub>N<sub>5</sub>O 255.1; found <i>m</i>/<i>z</i> = 256.3 [M + H]<sup>+</sup>.</div></div><div id="sec2_7_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> 2-Amino-6-(4-phenoxyphenyl)-9<i>H</i>-purine (<b>3d</b>)</h3><div class="NLM_p last"><b>3d</b> was obtained following general procedure <i>a</i> as a pale yellow powder in 90% yield from <b>2a</b> and 4-phenoxyphenylboronic acid <b>27</b>. RP-HPLC <i>t</i><sub>R</sub> = 24.1 min, gradient condition from 5% B to 100% B in 65 min, flow rate of 4 mL/min, λ = 240 nm. <sup>1</sup>H NMR (300 MHz, MeOD): δ = 7.10–7.20 (m, 4H), 7.25 (t, <i>J</i> = 7.1 Hz, 1H), 7.46 (t, <i>J</i> = 7.5 Hz, 2H), 8.36 (br s, 3H). ESI-MS, calculated for C<sub>17</sub>H<sub>13</sub>N<sub>5</sub>O 303.1; found <i>m</i>/<i>z</i> = 304.3 [M + H]<sup>+</sup>.</div></div><div id="sec2_7_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> 2-Amino-6-(4-phenoxyphenyl)-9-methylpurine (<b>3e</b>)</h3><div class="NLM_p last"><b>3e</b> was obtained from <b>3d</b> following general procedure <i>b</i> as a yellow powder in 82% yield. RP-HPLC <i>t</i><sub>R</sub> = 27.5 min, gradient condition from 5% B to 100% B in 75 min, flow rate of 4 mL/min, λ = 240 nm. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ = 3.77 (s, 3H), 7.05–7.18 (m, 5H), 7.37 (t, <i>J</i> = 7.8 Hz, 2H), 7.79 (s, 1H), 8.70 (br s, 2H). <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ = 30.6, 118.3, 121.6, 124.8, 125.7, 127.1, 131.2, 133.9, 142.8, 147.6, 150.7, 156.3, 157.8, 163.3. ESI-MS, calculated for C<sub>18</sub>H<sub>15</sub>N<sub>5</sub>O 317.1; found <i>m</i>/<i>z</i> = 318.2 [M + H]<sup>+</sup>.</div></div><div id="sec2_7_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> 2-Amino-6-(4-(benzyloxy)phenyl)-9<i>H</i>-purine (<b>3f</b>)</h3><div class="NLM_p last"><b>3f</b> was obtained following general procedure <i>a</i> as a yellow powder in 77% yield from <b>2a</b> and 4-(benzyloxy)phenylboronic acid <b>30</b>. RP-HPLC <i>t</i><sub>R</sub> = 17.4 min, gradient condition from 5% B to 100% B in 40 min, flow rate of 4 mL/min, λ = 240 nm. <sup>1</sup>H NMR (300 MHz, MeOD): δ = 5.28 (s, 2H), 7.30 (d, <i>J</i> = 8.8 Hz, 2H), 7.36–7.46 (m, 3H), 7.50 (br s, 2H), 8.37–8.45 (m, 3H). ESI-MS, calculated for C<sub>18</sub>H<sub>15</sub>N<sub>5</sub>O 317.1; found <i>m</i>/<i>z</i> = 318.1 [M + H]<sup>+</sup>.</div></div><div id="sec2_7_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> 2-Amino-6-(4-(3′-(trifluoromethyl)phenoxymethyl)phenyl)-9<i>H</i>-purine (<b>3g</b>)</h3><div class="NLM_p last"><b>3g</b> was obtained following general procedure <i>a</i> as a yellow powder in 79% yield from <b>2a</b> and 4-(3′-(trifluoromethyl)phenoxymethyl)phenylboronic acid <b>31</b>. RP-HPLC <i>t</i><sub>R</sub> = 25.0 min, gradient condition from 5% B to 100% B in 50 min, flow rate of 4 mL/min, λ = 240 nm. <sup>1</sup>H NMR (300 MHz, MeOD): δ = 5.31 (s, 2H), 7.22–7.33 (m, 3H), 7.48 (t, <i>J</i> = 7.9 Hz, 1H), 7.73 (d, <i>J</i> = 8.1 Hz, 2H), 8.37 (br s, 3H). ESI-MS, calculated for C<sub>19</sub>H<sub>14</sub>F<sub>3</sub>N<sub>5</sub>O 385.1; found <i>m</i>/<i>z</i> = 386.1 [M + H]<sup>+</sup>.</div></div><div id="sec2_7_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> 2-Amino-6-(4-((4′-(2-methoxyethyl)phenoxy)methyl)phenyl)-9<i>H</i>-purine (<b>3h</b>)</h3><div class="NLM_p last"><b>3h</b> was obtained following general procedure <i>a</i> as a yellow powder in 77% yield from <b>2a</b> and 4-((4′-(2-methoxyethyl)phenoxy)methyl)phenylboronic acid <b>32</b>. RP-HPLC <i>t</i><sub>R</sub> = 17.9 min, gradient condition from 5% B to 100% B in 40 min, flow rate of 4 mL/min, λ = 240 nm. <sup>1</sup>H NMR (300 MHz, MeOD): δ = 2.80 (t, <i>J</i> = 6.9 Hz, 2H), 3.33 (s, 3H), 3.57 (t, <i>J</i> = 6.9 Hz, 2H), 5.22 (s, 2H), 6.96 (d, <i>J</i> = 8.7 Hz, 2H), 7.16 (d, <i>J</i> = 8.6 Hz, 2H), 7.70 (d, <i>J</i> = 8.1 Hz, 2H), 8.33 (s, 1H), 8.40 (d, <i>J</i> = 8.3 Hz, 2H). ESI-MS, calculated for C<sub>21</sub>H<sub>21</sub>N<sub>5</sub>O<sub>2</sub> 375.2; found <i>m</i>/<i>z</i> = 376.1 [M + H]<sup>+</sup>.</div></div><div id="sec2_7_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> 2-Amino-6-(4-benzyloxy-3-chlorophenyl)-9<i>H</i>-purine (<b>4a</b>)</h3><div class="NLM_p last"><b>4a</b> was obtained following general procedure <i>a</i> as a yellow powder in 84% yield from <b>2a</b> and 4-benzyloxy-3-chlorophenylboronic acid <b>28</b>. RP-HPLC <i>t</i><sub>R</sub> = 20.4 min, gradient condition from 5% B to 100% B in 40 min, flow rate of 4 mL/min, λ = 240 nm. <sup>1</sup>H NMR (300 MHz, MeOD): δ = 5.32 (s, 2H), 7.33–7.44 (m, 4H), 7.50 (br s, 2H), 8.34 (br s, 2H), 8.50 (s, 1H). ESI-MS, calculated for C<sub>18</sub>H<sub>14</sub>ClN<sub>5</sub>O 351.1; found <i>m</i>/<i>z</i> = 352.1 [M + H]<sup>+</sup>.</div></div><div id="sec2_7_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> 2-Amino-6-(3-chloro-4-(3′,5′-dimethoxybenzyloxy)phenyl)-9<i>H</i>-purine (<b>4b</b>)</h3><div class="NLM_p last"><b>4b</b> was obtained following general procedure <i>a</i> as a yellow powder in 86% yield from <b>2a</b> and 3-chloro-4-(3′,5′-dimethoxybenzyloxy)phenylboronic acid <b>33</b>. RP-HPLC <i>t</i><sub>R</sub> = 20.5 min, gradient condition from 5% B to 100% B in 40 min, flow rate of 4 mL/min, λ = 240 nm. <sup>1</sup>H NMR (600 MHz, MeOD): δ = 3.76 (s, 6H), 5.28 (s, 2H), 6.48 (s, 1H), 6.68 (s, 2H), 7.32 (d, <i>J</i> = 8.8 Hz, 1H), 8.18 (s, 1H), 8.47 (br s, 1H), 8.61 (s, 1H). <sup>13</sup>C NMR (150 MHz, MeOD): δ = 56.2, 71.6, 100.7, 106.2, 115.6, 123.2, 127.0, 128.7, 132.1, 139.7, 143.1, 149.5, 150.3, 155.7, 157.2, 160.4, 161.8. ESI-MS, calculated for C<sub>20</sub>H<sub>18</sub>ClN<sub>5</sub>O<sub>3</sub> 411.1; found <i>m</i>/<i>z</i> = 412.1 [M + H]<sup>+</sup>.</div></div><div id="sec2_7_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> 2-Amino-6-(2,3-dihydrobenzo[<i>b</i>][1,4]dioxin-6-yl)-9<i>H</i>-purine (<b>4c</b>)</h3><div class="NLM_p last"><b>4c</b> was obtained following general procedure <i>a</i> as a pale yellow powder in 84% yield from <b>2a</b> and 1,4-benzodioxane-6-boronic acid <b>26</b>. RP-HPLC <i>t</i><sub>R</sub> = 11.6. min, gradient condition from 5% B to 100% B in 50 min, flow rate of 4 mL/min, λ = 240 nm. <sup>1</sup>H NMR (300 MHz, MeOD): δ = 4.34–4.40 (m, 4H), 7.09 (d, <i>J</i> = 8.5 Hz, 1H), 7.89–7.98 (m, 2H), 8.34 (s, 1H). ESI-MS, calculated for C<sub>13</sub>H<sub>11</sub>N<sub>5</sub>O<sub>2</sub> 269.1; found <i>m</i>/<i>z</i> = 270.2 [M + H]<sup>+</sup>.</div></div><div id="sec2_7_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> 2-Amino-6-(3-(4′-chlorobenzyloxy)phenyl)-9<i>H</i>-purine (<b>4d</b>)</h3><div class="NLM_p last"><b>4d</b> was obtained following general procedure <i>a</i> as a yellow powder in 78% yield from <b>2a</b> and 3-(4′-chlorobenzyloxy)phenylboronic acid <b>29</b>. RP-HPLC <i>t</i><sub>R</sub> = 29.8 min, gradient condition from 5% B to 100% B in 70 min, flow rate of 4 mL/min, λ = 240 nm. <sup>1</sup>H NMR (300 MHz, MeOD): δ = 5.19 (s, 2H), 7.29 (d, <i>J</i> = 7.6 Hz, 1H), 7.40 (d, <i>J</i> = 8.4 Hz, 2H), 7.45–7.58 (m, 3H), 7.90 (d, <i>J</i> = 6.9 Hz, 1H), 8.01 (s, 1H), 8.37 (s, 1H). ESI-MS, calculated for C<sub>18</sub>H<sub>14</sub>ClN<sub>5</sub>O 351.1; found <i>m</i>/<i>z</i> = 352.2 [M + H]<sup>+</sup>.</div></div><div id="sec2_7_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> 2-Amino-6-(3-(4′-chlorobenzyloxy)phenyl)-9-methylpurine (<b>4e</b>)</h3><div class="NLM_p last"><b>4e</b> was obtained following general procedure <i>b</i> as a yellow powder in 63% yield from <b>4d</b>. RP-HPLC <i>t</i><sub>R</sub> = 28.3 min, gradient condition from 5% B to 100% B in 60 min, flow rate of 4 mL/min, λ = 240 nm. <sup>1</sup>H NMR (300 MHz, CDCl3): δ = 3.77 (s, 3H), 5.20 (s, 2H), 7.13 (d, <i>J</i> = 7.8 Hz, 1H), 7.36 (d, <i>J</i> = 8.2 Hz, 2H), 7.41–7.51 (m, 3H), 7.83 (s, 1H), 8.31–8.42 (m, 2H). ESI-MS, calculated for C<sub>19</sub>H<sub>16</sub>ClN<sub>5</sub>O 365.1; found <i>m</i>/<i>z</i> = 366.2 [M + H]<sup>+</sup>.</div></div><div id="sec2_7_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> 2-Amino-6-(3-bromo-5-butoxyphenyl)-9<i>H</i>-purine (<b>5a</b>)</h3><div class="NLM_p last"><b>5a</b> was obtained following general procedure <i>a</i> as a white powder in 83% yield from <b>2a</b> and 3-bromo-5-butoxyphenylboronic acid <b>21</b>. RP-HPLC <i>t</i><sub>R</sub> = 26.7 min, gradient condition from 5% B to 100% B in 50 min, flow rate of 4 mL/min, λ = 240 nm. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ = 0.93 (t, <i>J</i> = 7.3 Hz, 3H), 1.39–1.51 (m, 2H), 1.68–1.76 (m, 2H), 4.06 (t, <i>J</i> = 6.3 Hz, 2H), 6.46 (s, 2H), 7.28 (s, 1H), 8.16 (s, 1H), 8.37 (s, 1H), 8.50 (s, 1H), 12.72 (s, 1H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>): δ = 14.9, 20.1, 32.1, 69.5, 115.8, 120.9, 124.4, 125.8, 138.7, 142.2, 148.1, 151.3, 157.2, 161.3. ESI-MS, calculated for C<sub>15</sub>H<sub>16</sub>BrN<sub>5</sub>O 361.1; found <i>m</i>/<i>z</i> = 362.3 [M + H]<sup>+</sup>.</div></div><div id="sec2_7_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> 2-Amino-6-(3-bromo-5-butoxyphenyl)-9-methylpurine (<b>5b</b>)</h3><div class="NLM_p last"><b>5b</b> was obtained following general procedure <i>b</i> as a yellow powder in 85% yield from <b>5a</b>. RP-HPLC <i>t</i><sub>R</sub> = 27.9 min, gradient condition from 5% B to 100% B in 45 min, flow rate of 4 mL/min, λ = 240 nm. <sup>1</sup>H NMR (300 MHz, CDCl3): δ = 0.97 (t, <i>J</i> = 7.3 Hz, 3H), 1.46–1.55 (m, 2H), 1.73–1.83 (m, 2H), 3.79 (s, 3H), 4.11 (t, <i>J</i> = 6.2 Hz, 2H), 7.29 (s, 1H), 7.92 (s, 1H), 7.97 (s, 1H), 8.33 (s, 1H). ESI-MS, calculated for C<sub>16</sub>H<sub>18</sub>BrN<sub>5</sub>O 375.1; found <i>m</i>/<i>z</i> = 376.2 [M + H]<sup>+</sup>.</div></div><div id="sec2_7_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> 2-Amino-6-(2,6-dimethoxyphenyl)-9<i>H</i>-purine (<b>6a</b>)</h3><div class="NLM_p last"><b>6a</b> was obtained following general procedure <i>a</i> as a white powder in 53% yield from <b>2a</b> and 2,6-dimethoxyphenylboronic acid <b>22</b>. RP-HPLC <i>t</i><sub>R</sub> = 13.0 min, gradient condition from 5% B to 100% B in 80 min, flow rate of 4 mL/min, λ = 240 nm. <sup>1</sup>H NMR (300 MHz, MeOD): δ = 3.82 (s, 6H), 6.87 (d, <i>J</i> = 8.5 Hz, 2H), 7.60 (t, <i>J</i> = 8.5 Hz, 1H), 8.44 (s, 1H). ESI-MS, calculated for C<sub>13</sub>H<sub>13</sub>N<sub>5</sub>O<sub>2</sub> 271.1; found <i>m</i>/<i>z</i> = 272.2 [M + H]<sup>+</sup>.</div></div><div id="sec2_7_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> 2-Amino-6-(2-isopropoxy-6-methoxyphenyl)-9<i>H</i>-purine (<b>6b</b>)</h3><div class="NLM_p last"><b>6b</b> was obtained following general procedure <i>a</i> as a white powder in 62% yield from <b>2a</b> and 2-isopropoxy-6-methoxyphenylboronic acid <b>23</b>. RP-HPLC <i>t</i><sub>R</sub> = 18.1 min, gradient condition from 5% B to 100% B in 40 min, flow rate of 4 mL/min, λ = 240 nm. <sup>1</sup>H NMR (300 MHz, MeOD): δ = 1.18 (s, 6H), 3.80 (s, 3H), 4.60–4.71 (m, 1H), 6.80–6.88 (m, 2H), 7.56 (t, <i>J</i> = 8.5 Hz, 1H), 8.43 (s, 1H). ESI-MS, calculated for C<sub>15</sub>H<sub>17</sub>N<sub>5</sub>O<sub>2</sub> 299.1; found <i>m</i>/<i>z</i> = 300.1 [M + H]<sup>+</sup>.</div></div><div id="sec2_7_19" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> 2-Amino-6-(2-isobutoxy-6-methoxyphenyl)-9<i>H</i>-purine (<b>6c</b>)</h3><div class="NLM_p last"><b>6c</b> was obtained following general procedure <i>a</i> as a white powder in 76% yield from <b>2a</b> and 2-isobutoxy-6-methoxy-phenylboronic acid <b>24</b>. RP-HPLC <i>t</i><sub>R</sub> = 14.2 min, gradient condition from 5% B to 100% B in 40 min, flow rate of 4 mL/min, λ = 240 nm. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ = 0.69 (s, 6H), 1.72–1.79 (m, 1H), 3.71 (br s, 5H), 6.81–6.86 (m, 2H), 7.52 (t, <i>J</i> = 8.1 Hz, 1H), 8.47 (s, 1H). <sup>13</sup>C NMR (150 MHz, MeOD): δ = 20.3, 29.0, 57.6, 76.1, 106.2, 118.6, 125.8, 134.7, 137.4, 142.7, 154.9, 159.3, 160.8, 173.0. ESI-MS, calculated for C<sub>16</sub>H<sub>19</sub>N<sub>5</sub>O<sub>2</sub> 313.2; found <i>m</i>/<i>z</i> = 314.1 [M + H]<sup>+</sup>.</div></div><div id="sec2_7_20" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> 2-Amino-6-(2-methoxyphenyl)-9<i>H</i>-purine (<b>7a</b>)</h3><div class="NLM_p last"><b>7a</b> was obtained following general procedure <i>a</i> as a pale yellow powder in 78% yield from <b>2a</b> and 2-methoxyphenylboronic acid <b>15</b>. RP-HPLC <i>t</i><sub>R</sub> = 14.9 min, gradient condition from 5% B to 100% B in 80 min, flow rate of 4 mL/min, λ = 240 nm. <sup>1</sup>H NMR (300 MHz, MeOD): δ = 3.97 (s, 3H), 7.21 (t, <i>J</i> = 7.5 Hz, 1H), 7.30 (d, <i>J</i> = 8.4 Hz, 1H), 7.67 (t, <i>J</i> = 7.8 Hz, 1H), 8.10 (d, <i>J</i> = 7.4 Hz, 1H), 8.45 (s, 1H). ESI-MS, calculated for C<sub>12</sub>H<sub>11</sub>N<sub>5</sub>O 241.1; found <i>m</i>/<i>z</i> = 242.2 [M + H]<sup>+</sup>.</div></div><div id="sec2_7_21" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> 2-Amino-6-(5-fluoro-2-methoxyphenyl)-9<i>H</i>-purine (<b>7b</b>)</h3><div class="NLM_p last"><b>7b</b> was obtained following general procedure <i>a</i> as a pale yellow powder in 70% yield from <b>2a</b> and 5-fluoro-2-methoxyphenylboronic acid <b>16</b>. RP-HPLC <i>t</i><sub>R</sub> = 14.0 min, gradient condition from 5% B to 100% B in 60 min, flow rate of 4 mL/min, λ = 240 nm. <sup>1</sup>H NMR (300 MHz, MeOD): δ = 3.95 (s, 3H), 7.29 (d, <i>J</i> = 8.9 Hz, 1H), 7.41 (dd, <i>J</i> = 8.9, 2.4 Hz, 1H), 7.92 (br s, 1H), 8.54 (s, 1H). ESI-MS, calculated for C<sub>12</sub>H<sub>10</sub>FN<sub>5</sub>O 259.1; found <i>m</i>/<i>z</i> = 260.1 [M + H]<sup>+</sup>.</div></div><div id="sec2_7_22" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> 2-Amino-6-(5-chloro-2-methoxyphenyl)-9<i>H</i>-purine (<b>7c</b>)</h3><div class="NLM_p last"><b>7c</b> was obtained following general procedure <i>a</i> as a pale yellow powder in 78% yield from <b>2a</b> and 5-chloro-2-methoxyphenylboronic acid <b>17</b>. RP-HPLC <i>t</i><sub>R</sub> = 18.1 min, gradient condition from 5% B to 100% B in 80 min, flow rate of 4 mL/min, λ = 240 nm. <sup>1</sup>H NMR (300 MHz, MeOD): δ = 3.95 (s, 3H), 7.28 (d, <i>J</i> = 8.9 Hz, 1H), 7.63 (dd, <i>J</i> = 8.9, 2.5 Hz, 1H), 8.03 (br s, 1H), 8.53 (s, 1H). ESI-MS, calculated for C<sub>12</sub>H<sub>10</sub>ClN<sub>5</sub>O 275.1; found <i>m</i>/<i>z</i> = 276.1 [M + H]<sup>+</sup>.</div></div><div id="sec2_7_23" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> 2-Amino-6-(5-bromo-2-methoxyphenyl)-9<i>H</i>-purine (<b>7d</b>)</h3><div class="NLM_p last"><b>7d</b> was obtained following general procedure <i>a</i> as a yellow powder in 77% yield from <b>2a</b> and 5-bromo-2-methoxyphenylboronic acid <b>18</b>. RP-HPLC <i>t</i><sub>R</sub> = 17.3 min, gradient condition from 5% B to 100% B in 60 min, flow rate of 4 mL/min, λ = 240 nm. <sup>1</sup>H NMR (600 MHz, MeOD): δ = 3.95 (s, 3H), 7.23 (d, <i>J</i> = 9.0 Hz, 1H), 7.76 (dd, <i>J</i> = 8.9, 2.4 Hz, 1H), 8.14 (br s, 1H), 8.53 (s, 1H). <sup>13</sup>C NMR (150 MHz, MeOD): δ = 56.3, 113.3, 114.4, 123.7, 126.4, 135.4, 135.9, 142.6, 149.5, 155.1, 157.8, 160.9. ESI-MS, calculated for C<sub>12</sub>H<sub>10</sub>BrN<sub>5</sub>O 319.0; found <i>m</i>/<i>z</i> = 320.3 [M + H]<sup>+</sup>.</div></div><div id="sec2_7_24" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> 2-Amino-6-(5-bromo-2-ethoxyphenyl)-9<i>H</i>-purine (<b>7e</b>)</h3><div class="NLM_p last"><b>7e</b> was obtained following general procedure <i>a</i> as a yellow powder in 82% yield from <b>2a</b> and 5-bromo-2-ethoxyphenylboronic acid <b>19</b>. RP-HPLC <i>t</i><sub>R</sub> = 21.6 min, gradient condition from 5% B to 100% B in 80 min, flow rate of 4 mL/min, λ = 240 nm. <sup>1</sup>H NMR (300 MHz, MeOD): δ = 1.31 (t, <i>J</i> = 6.9 Hz, 3H), 4.13–4.24 (m, 2H), 7.19 (d, <i>J</i> = 8.9 Hz, 1H), 7.72 (dd, <i>J</i> = 8.9, 2.3 Hz, 1H), 7.98 (brs, 1H), 8.53 (s, 1H). ESI-MS, calculated for C<sub>13</sub>H<sub>12</sub>BrN<sub>5</sub>O 333.0; found <i>m</i>/<i>z</i> = 334.1 [M + H]<sup>+</sup>.</div></div><div id="sec2_7_25" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> 6-(5-Bromo-2-methoxyphenyl)-2-chloro-9<i>H</i>-purine (<b>8a</b>)</h3><div class="NLM_p last"><b>8a</b> was obtained following general procedure <i>a</i> as a yellow powder in 68% yield from <b>1</b> and 5-bromo-2-methoxyphenylboronic acid <b>18</b>. RP-HPLC <i>t</i><sub>R</sub> = 24.3 min, gradient condition from 5% B to 100% B in 45 min, flow rate of 4 mL/min, λ = 240 nm. <sup>1</sup>H NMR (300 MHz, CDCl3): δ = 3.91 (s, 3H), 7.00 (d, <i>J</i> = 8.8 Hz, 1H), 7.23 (br s, 1H) 7.63 (d, <i>J</i> = 8.1 Hz, 1H), 8.04 (s, 1H). ESI-MS, calculated for C<sub>12</sub>H<sub>8</sub>BrClN<sub>4</sub>O 338.0; found <i>m</i>/<i>z</i> = 339.2 [M + H]<sup>+</sup>.</div></div><div id="sec2_7_26" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> 2-Amino-6-(5-bromo-2-methoxyphenyl)-9-methylpurine (<b>8b</b>)</h3><div class="NLM_p last"><b>8b</b> was obtained following general procedure <i>b</i> as a yellow powder in 87% yield from <b>7d</b>. RP-HPLC <i>t</i><sub>R</sub> = 18.9 min, gradient condition from 5% B to 100% B in 70 min, flow rate of 4 mL/min, λ = 240 nm. <sup>1</sup>H NMR (300 MHz, CDCl3): δ = 3.79 (s, 3H), 3.87 (s, 3H), 6.94–7.01 (m, 1H), 7.24 (br s, 1H), 8.02 (s, 1H). ESI-MS, calculated for C<sub>13</sub>H<sub>12</sub>BrN<sub>5</sub>O 333.0; found <i>m</i>/<i>z</i> = 334.2 [M + H]<sup>+</sup>.</div></div><div id="sec2_7_27" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> 2-Amino-6-(5-bromo-2-methoxyphenyl)-9-(2-oxopropyl)-purine (<b>8c</b>)</h3><div class="NLM_p last"><b>8c</b> was obtained following general procedure <i>b</i> as a yellow powder in 50% yield from <b>7d</b>. RP-HPLC <i>t</i><sub>R</sub> = 22.9 min, gradient condition from 5% B to 100% B in 80 min, flow rate of 4 mL/min, λ = 240 nm. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ = 2.40 (s, 3H), 4.01 (s, 3H), 5.00 (s, 2H), 7.03 (d, <i>J</i> = 8.9 Hz, 1H), 7.70 (dd, <i>J</i> = 8.9, 2.4 Hz, 1H), 8.02 (s, 1H), 8.19 (s, 1H). <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ = 27.3, 52.5, 57.1, 114.3, 118.8, 125.1, 126.9, 136.3, 138.4, 143.3, 148.2, 154.3, 156.7, 158.4, 199.2. ESI-MS, calculated for C<sub>15</sub>H<sub>14</sub>BrN<sub>5</sub>O<sub>2</sub> 375.0; found <i>m</i>/<i>z</i> = 376.1 [M + H]<sup>+</sup>.</div></div><div id="sec2_7_28" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> 6-(5-Fluoro-2-methoxyphenyl)-2-hydroxy-9<i>H</i>-purine (<b>8d</b>)</h3><div class="NLM_p last"><b>8d</b> was obtained following general procedure <i>c</i> as a white powder in 47% yield from <b>7b</b>. RP-HPLC <i>t</i><sub>R</sub> = 13.9 min, gradient condition from 5% B to 100% B in 50 min, flow rate of 1 mL/min, λ = 240 nm. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ = 3.93 (s, 3H), 7.31 (d, <i>J</i> = 8.9 Hz, 1H), 7.40 (dd, <i>J</i> = 8.9, 2.4 Hz, 1H), 7.89 (br s, 1H), 8.54 (s, 1H). ESI-MS, calculated for C<sub>12</sub>H<sub>9</sub>FN<sub>4</sub>O<sub>2</sub> 260.1; found <i>m</i>/<i>z</i> = 261.1 [M + H]<sup>+</sup>.</div></div><div id="sec2_7_29" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41"> 6-(5-Chloro-2-methoxyphenyl)-2-hydroxy-9<i>H</i>-purine (<b>8e</b>)</h3><div class="NLM_p last"><b>8e</b> was obtained following general procedure <i>c</i> as a white powder in 63% yield from <b>7c</b>. RP-HPLC <i>t</i><sub>R</sub> = 14.8 min, gradient condition from 5% B to 100% B in 50 min, flow rate of 4 mL/min, λ = 240 nm. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ = 3.94 (s, 3H), 7.26 (d, <i>J</i> = 8.9 Hz, 1H), 7.63 (dd, <i>J</i> = 8.9, 2.5 Hz, 1H), 8.03 (br s, 1H), 8.53 (s, 1H). ESI-MS, calculated for C<sub>12</sub>H<sub>9</sub>ClN<sub>4</sub>O<sub>2</sub> 276.0; found <i>m</i>/<i>z</i> = 277.1 [M + H]<sup>+</sup>.</div></div><div id="sec2_7_30" class="NLM_sec NLM_sec_level_2"><div id="ac_i42" class="anchor-spacer"></div><h3 class="article-section__title" id="_i42"> 6-(5-Bromo-2-methoxyphenyl)-2-hydroxy-9<i>H</i>-purine (<b>8f</b>)</h3><div class="NLM_p last"><b>8f</b> was obtained following general procedure <i>c</i> as a white powder in 55% yield from <b>7d</b>. RP-HPLC <i>t</i><sub>R</sub> = 15.8 min, gradient condition from 5% B to 100% B in 50 min, flow rate of 1 mL/min, λ = 240 nm. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ = 3.97 (s, 3H), 7.25 (d, <i>J</i> = 9.0 Hz, 1H), 7.76 (dd, <i>J</i> = 8.9, 2.4 Hz, 1H), 8.14 (br s, 1H), 8.53 (s, 1H). ESI-MS, calculated for C<sub>12</sub>H<sub>9</sub>BrN<sub>4</sub>O<sub>2</sub> 320.0; found <i>m</i>/<i>z</i> = 321.1[M + H]<sup>+</sup>.</div></div><div id="sec2_7_31" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> 6-(5-Bromo-2-methoxyphenyl)-8-methyl-9<i>H</i>-purine (<b>9a</b>)</h3><div class="NLM_p last"><b>9a</b> was obtained following general procedure <i>a</i> as a pale yellow powder in 70% yield from 6-chloro-8-methyl-9<i>H</i>-purine <b>12</b> and 5-bromo-2-methoxyphenylboronic acid <b>18</b>. RP-HPLC <i>t</i><sub>R</sub> = 21.7 min, gradient condition from 5% B to 100% B in 70 min, flow rate of 4 mL/min, λ = 240 nm. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ = 2.69 (s, 3H), 3.87 (s, 3H), 7.20 (d, <i>J</i> = 9.0 Hz, 1H), 7.71 (dd, <i>J</i> = 8.9, 2.4 Hz, 1H), 7.82 (s, 1H), 8.95 (s, 1H). ESI-MS, calculated for C<sub>13</sub>H<sub>11</sub>BrN<sub>4</sub>O 318.0; found <i>m</i>/<i>z</i> = 319.2 [M + H]<sup>+</sup>.</div></div><div id="sec2_7_32" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> 6-(5-Fluoro-2-methoxyphenyl)-8-methyl-9<i>H</i>-purine (<b>9b</b>)</h3><div class="NLM_p last"><b>9b</b> was obtained following general procedure <i>a</i> as a pale yellow powder in 81% yield from 6-chloro-8-methyl-9<i>H</i>-purine <b>12</b> and 5-fluoro-2-methoxyphenylboronic acid <b>16</b>. RP-HPLC <i>t</i><sub>R</sub> = 15.6 min, gradient condition from 5% B to 100% B in 60 min, flow rate of 4 mL/min, λ = 240 nm. <sup>1</sup>H NMR (300 MHz, MeOD): δ = 2.76 (s, 3H), 3.90 (s, 3H), 7.27 (d, <i>J</i> = 9.0 Hz, 1H), 7.39 (br s, 1H), 7.67 (dd, <i>J</i> = 8.9, 2.4 Hz, 1H), 9.04 (s, 1H). <sup>13</sup>C NMR (75 MHz, MeOD): δ = 14.1, 56.2, 113.5, 117.9, 118.3, 119.2, 126.9, 147.7, 151.3, 154.5, 155.9, 158.2, 159.2. ESI-MS, calculated for C<sub>13</sub>H<sub>11</sub>FN<sub>4</sub>O 258.1; found <i>m</i>/<i>z</i> = 259.1 [M + H]<sup>+</sup>.</div></div><div id="sec2_7_33" class="NLM_sec NLM_sec_level_2"><div id="ac_i45" class="anchor-spacer"></div><h3 class="article-section__title" id="_i45"> 2-Amino-6-thianthrenyl-9<i>H</i>-purine (<b>10</b>)</h3><div class="NLM_p last"><b>10</b> was obtained following general procedure <i>a</i> as a yellow powder in 82% yield from <b>2a</b> and 1-thianthrenylboronic acid <b>25</b>. RP-HPLC <i>t</i><sub>R</sub> = 18.5 min, gradient condition from 5% B to 100% B in 40 min, flow rate of 4 mL/min, λ = 240 nm. <sup>1</sup>H NMR (300 MHz, MeOD): δ = 7.22–7.37 (m, 3H), 7.51 (t, <i>J</i> = 7.5 Hz, 2H), 7.65 (d, <i>J</i> = 7.4 Hz, 1H), 7.75 (d, <i>J</i> = 7.7 Hz, 1H), 8.37 (s, 1H). ESI-MS, calculated for C<sub>17</sub>H<sub>11</sub>N<sub>5</sub>S<sub>2</sub> 349.0; found <i>m</i>/<i>z</i> = 350.1 [M + H]<sup>+</sup>.</div></div><div id="sec2_7_34" class="NLM_sec NLM_sec_level_2"><div id="ac_i46" class="anchor-spacer"></div><h3 class="article-section__title" id="_i46"> 2-Amino-6-(5-bromo-2,3-dihydrobenzo[<i>b</i>]furan-7-yl)-9<i>H</i>-purine (<b>11</b>)</h3><div class="NLM_p last"><b>11</b> was obtained following general procedure <i>a</i> as a pale yellow powder in 79% yield from <b>2a</b> and 5-bromo-2,3-dihydrobenzo[<i>b</i>]furan-7-boronic acid <b>20</b>. RP-HPLC <i>t</i><sub>R</sub> = 15.2 min, gradient condition from 5% B to 100% B in 40 min, flow rate of 4 mL/min, λ = 240 nm. <sup>1</sup>H NMR (600 MHz, MeOD): δ = 3.35–3.42 (m, 2H), 4.87–4.93 (m, 2H), 7.65 (s, 1H), 8.50 (s, 1H), 8.57 (s, 1H). <sup>13</sup>C NMR (150 MHz, MeOD): δ = 29.1, 74.3, 113.6, 121.2, 125.8, 131.5, 132.1, 134.0, 146.1, 154.9, 158.8, 159.9, 160.7. ESI-MS, calculated for C<sub>13</sub>H<sub>10</sub>BrN<sub>5</sub>O 331.0; found <i>m</i>/<i>z</i> = 332.2 [M + H]<sup>+</sup>.</div></div><div id="sec2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i47" class="anchor-spacer"></div><h3 class="article-section__title" id="_i47"> Protein Expression and Purification</h3><div class="NLM_p last">Proteins were cloned, expressed, and purified as previously described.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a></div></div><div id="sec2_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i48" class="anchor-spacer"></div><h3 class="article-section__title" id="_i48"> Protein Stability Shift Assay</h3><div class="NLM_p last">Thermal melting experiments were carried out using an Mx3005p real time PCR machine (Stratagene). Proteins were buffered in 10 mM HEPES, pH 7.5, 500 mM NaCl and assayed in a 96-well plate at a final concentration of 2 μM in 20 μL volume. Compounds were added at a final concentration of 10 or 100 μM. SYPRO Orange (Molecular Probes) was added as a fluorescence probe at a dilution of 1 in 1000. Excitation and emission filters for the SYPRO-Orange dye were set to 465 and 590 nm, respectively. The temperature was raised with a step of 3 °C per minute from 25 to 96 °C, and fluorescence readings were taken at each interval. The temperature dependence of the fluorescence during the protein denaturation process was approximated by the equation<span class="NLM_disp-formula" id="ueq1"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-6/jm501893k/20150418/images/jm-2014-01893k_m001.gif" alt="" /></img></span>where ΔuG is the difference in unfolding free energy between the folded and unfolded state, <i>R</i> is the gas constant, and <i>y</i><sub>F</sub> and <i>y</i><sub>U</sub> are the fluorescence intensity of the probe in the presence of completely folded and unfolded protein, respectively.<a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(65)</a> The baselines of the denatured and native states were approximated by a linear fit. The observed temperature shifts, Δ<i>T</i><sub>m</sub><sup>obs</sup>, were recorded as the difference between the transition midpoints of sample and reference wells containing protein without ligand in the same plate and determined by nonlinear least-squares fit. Temperature shifts (Δ<i>T</i><sub>m</sub><sup>obs</sup>) for three independent measurements per proteins/compound are summarized in <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information Table S1">Supporting Information Table S1</a>.</div></div><div id="sec2_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i49" class="anchor-spacer"></div><h3 class="article-section__title" id="_i49"> Isothermal Titration Calorimetry</h3><div class="NLM_p">Experiments were carried out on an ITC200 titration microcalorimeter from MicroCal, LLC (GE Healthcare) equipped with a Washing module, with a cell volume of 0.2003 mL, and a 40 μL microsyringe. Experiments were carried out at 15 °C while stirring at 1000 rpm, in ITC buffer (50 mM HEPES, pH 7.5 (at 25 °C), 150 mM NaCl). The microsyringe was loaded with a solution of the protein sample (300−740 μM protein for BRD9 and BRD4(1), respectively, in ITC buffer) and was carefully inserted into the calorimetric cell which was filled with an amount of the ligand (0.2 mL, 20–25 μM in ITC buffer). The system was first allowed to equilibrate until the cell temperature reached 15 °C, and an additional delay of 60 s was applied. All titrations were conducted using an initial control injection of 0.3 μL followed by 38 identical injections of 1 μL with a duration of 2 s (per injection) and a spacing of 120 s between injections. The titration experiments were designed in such a fashion as to ensure complete saturation of the proteins before the final injection. The heat of dilution for the proteins was independent of their concentration and corresponded to the heat observed from the last injection, following saturation of ligand binding, thus facilitating the estimation of the baseline of each titration from the last injection. The collected data were corrected for protein heats of dilution (measured on separate experiments by titrating the proteins into ITC buffer) and deconvoluted using the MicroCal Origin software supplied with the instrument to yield enthalpies of binding (Δ<i>H</i>) and binding constants (<i>K</i><sub>B</sub>) in the same fashion as that previously described in detail by Wiseman and co-workers.<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> Thermodynamic parameters were calculated using the basic equation of thermodynamics (Δ<i>G</i> = Δ<i>H</i> – <i>T</i>Δ<i>S</i> = −<i>RT</i> ln <i>K</i><sub>B</sub>, where Δ<i>G</i>, Δ<i>H</i>, and Δ<i>S</i> are the changes in free energy, enthalpy, and entropy of binding, respectively). In all cases a single binding site model was employed, supplied with the MicroCal Origin software package. Dissociation constants and thermodynamic parameters are listed in Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>, <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>, and <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Isothermal Titration Calorimetry of Human BRD4(1) with 9<i>H</i>-Purine Compounds<a class="ref internalNav" href="#tbl1-fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">ligand</th><th class="colsep0 rowsep0" align="center">[P] (μM)</th><th class="colsep0 rowsep0" align="center">[L] (μM)</th><th class="colsep0 rowsep0" align="center"><i>K</i><sub>D</sub> (nM)</th><th class="colsep0 rowsep0" align="center">Δ<i>H</i><sup>obs</sup> (kcal/mol)</th><th class="colsep0 rowsep0" align="center"><i>N</i></th><th class="colsep0 rowsep0" align="center"><i>T</i>Δ<i>S</i> (kcal/mol)</th><th class="colsep0 rowsep0" align="center">Δ<i>G</i> (kcal/mol)</th><th class="colsep0 rowsep0" align="center">LE</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>2a</b></td><td class="colsep0 rowsep0" align="left">680</td><td class="colsep0 rowsep0" align="left">15</td><td class="colsep0 rowsep0" colspan="6" align="center">no binding/weak</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3a</b></td><td class="colsep0 rowsep0" align="left">402</td><td class="colsep0 rowsep0" align="left">16</td><td class="colsep0 rowsep0" align="left">11990 ± 743</td><td class="colsep0 rowsep0" align="left">–9.45 ± 0.55</td><td class="colsep0 rowsep0" align="left">1.03 ± 0.049</td><td class="colsep0 rowsep0" align="left">–2.97</td><td class="colsep0 rowsep0" align="left">–6.48</td><td class="colsep0 rowsep0" align="left">0.41</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7c</b></td><td class="colsep0 rowsep0" align="left">485</td><td class="colsep0 rowsep0" align="left">14</td><td class="colsep0 rowsep0" align="left">2037 ± 118</td><td class="colsep0 rowsep0" align="left">–6.21 ± 0.09</td><td class="colsep0 rowsep0" align="left">1.05 ± 0.012</td><td class="colsep0 rowsep0" align="left">1.29</td><td class="colsep0 rowsep0" align="left">–7.50</td><td class="colsep0 rowsep0" align="left">0.39</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7d</b></td><td class="colsep0 rowsep0" align="left">307</td><td class="colsep0 rowsep0" align="left">12</td><td class="colsep0 rowsep0" align="left">4651 ± 197</td><td class="colsep0 rowsep0" align="left">–6.09 ± 0.14</td><td class="colsep0 rowsep0" align="left">0.99 ± 0.018</td><td class="colsep0 rowsep0" align="left">0.94</td><td class="colsep0 rowsep0" align="left">–7.03</td><td class="colsep0 rowsep0" align="left">0.37</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11</b></td><td class="colsep0 rowsep0" align="left">382</td><td class="colsep0 rowsep0" align="left">30</td><td class="colsep0 rowsep0" align="left">1370 ± 29</td><td class="colsep0 rowsep0" align="left">–6.39 ± 0.02</td><td class="colsep0 rowsep0" align="left">1.09 ± 0.002</td><td class="colsep0 rowsep0" align="left">1.34</td><td class="colsep0 rowsep0" align="left">–7.73</td><td class="colsep0 rowsep0" align="left">0.39</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl1-fn1"><div class="footnote" id="tbl1-fn1"><sup>a</sup><p class="last">Titrations were carried out in 50 mM HEPES, pH 7.5 (at 25 °C), 150 mM NaCl, and 15 °C while stirring at 1000 rpm. In both cases the protein was titrated into the ligand solution (reverse titration). Titrations were performed in triplicate. Ligand efficiencies (LE) have also been calculated where Δ<i>G</i> values were available (LE = Δ<i>G</i>/<i>N</i> where <i>N</i> is the number of non-hydrogen atoms).</p></div></div></div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Isothermal Titration Calorimetry of Human BRD9 with 9<i>H</i>-Purine Compounds<a class="ref internalNav" href="#tbl2-fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">ligand</th><th class="colsep0 rowsep0" align="center">[P] (μM)</th><th class="colsep0 rowsep0" align="center">[L] (μM)</th><th class="colsep0 rowsep0" align="center"><i>K</i><sub>D</sub> (nM)</th><th class="colsep0 rowsep0" align="center">Δ<i>H</i><sup>obs</sup> (kcal/mol)</th><th class="colsep0 rowsep0" align="center"><i>N</i></th><th class="colsep0 rowsep0" align="center"><i>T</i>Δ<i>S</i> (kcal/mol)</th><th class="colsep0 rowsep0" align="center">Δ<i>G</i> (kcal/mol)</th><th class="colsep0 rowsep0" align="center">LE</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>2a</b></td><td class="colsep0 rowsep0" align="left">740</td><td class="colsep0 rowsep0" align="left">30</td><td class="colsep0 rowsep0" colspan="6" align="center">no binding/weak</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>2b</b></td><td class="colsep0 rowsep0" align="left">385</td><td class="colsep0 rowsep0" align="left">30</td><td class="colsep0 rowsep0" colspan="6" align="center">no binding/weak</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3a</b></td><td class="colsep0 rowsep0" align="left">477</td><td class="colsep0 rowsep0" align="left">26</td><td class="colsep0 rowsep0" align="left">8475 ± 237</td><td class="colsep0 rowsep0" align="left">–9.11 ± 0.12</td><td class="colsep0 rowsep0" align="left">0.99 ± 0.010</td><td class="colsep0 rowsep0" align="left">–2.42</td><td class="colsep0 rowsep0" align="left">–6.69</td><td class="colsep0 rowsep0" align="left">0.42</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5b</b></td><td class="colsep0 rowsep0" align="left">392</td><td class="colsep0 rowsep0" align="left">30</td><td class="colsep0 rowsep0" colspan="6" align="center">no binding/weak</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7a</b></td><td class="colsep0 rowsep0" align="left">385</td><td class="colsep0 rowsep0" align="left">34</td><td class="colsep0 rowsep0" align="left">641 ± 33</td><td class="colsep0 rowsep0" align="left">–12.71 ± 0.07</td><td class="colsep0 rowsep0" align="left">1.06 ± 0.004</td><td class="colsep0 rowsep0" align="left">–4.55</td><td class="colsep0 rowsep0" align="left">–8.16</td><td class="colsep0 rowsep0" align="left">0.45</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7b</b></td><td class="colsep0 rowsep0" align="left">385</td><td class="colsep0 rowsep0" align="left">13.5</td><td class="colsep0 rowsep0" align="left">351 ± 18</td><td class="colsep0 rowsep0" align="left">–13.04 ± 0.07</td><td class="colsep0 rowsep0" align="left">0.97 ± 0.004</td><td class="colsep0 rowsep0" align="left">–4.52</td><td class="colsep0 rowsep0" align="left">–8.52</td><td class="colsep0 rowsep0" align="left">0.45</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7c</b></td><td class="colsep0 rowsep0" align="left">378</td><td class="colsep0 rowsep0" align="left">14</td><td class="colsep0 rowsep0" align="left">297 ± 10</td><td class="colsep0 rowsep0" align="left">–12.05 ± 0.04</td><td class="colsep0 rowsep0" align="left">0.98 ± 0.003</td><td class="colsep0 rowsep0" align="left">–3.46</td><td class="colsep0 rowsep0" align="left">–8.59</td><td class="colsep0 rowsep0" align="left">0.45</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7d</b></td><td class="colsep0 rowsep0" align="left">235</td><td class="colsep0 rowsep0" align="left">10</td><td class="colsep0 rowsep0" align="left">397 ± 19</td><td class="colsep0 rowsep0" align="left">–9.63 ± 0.06</td><td class="colsep0 rowsep0" align="left">0.97 ± 0.005</td><td class="colsep0 rowsep0" align="left">–1.18</td><td class="colsep0 rowsep0" align="left">–8.45</td><td class="colsep0 rowsep0" align="left">0.44</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8a</b></td><td class="colsep0 rowsep0" align="left">381</td><td class="colsep0 rowsep0" align="left">18</td><td class="colsep0 rowsep0" align="left">7874 ± 258</td><td class="colsep0 rowsep0" align="left">–8.30 ± 0.15</td><td class="colsep0 rowsep0" align="left">1.06 ± 0.014</td><td class="colsep0 rowsep0" align="left">–1.57</td><td class="colsep0 rowsep0" align="left">–6.73</td><td class="colsep0 rowsep0" align="left">0.35</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8b</b></td><td class="colsep0 rowsep0" align="left">392</td><td class="colsep0 rowsep0" align="left">30</td><td class="colsep0 rowsep0" colspan="6" align="center">no binding/weak</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8e</b></td><td class="colsep0 rowsep0" align="left">451</td><td class="colsep0 rowsep0" align="left">32</td><td class="colsep0 rowsep0" align="left">7576 ± 365</td><td class="colsep0 rowsep0" align="left">–5.35 ± 0.09</td><td class="colsep0 rowsep0" align="left">1.05 ± 0.013</td><td class="colsep0 rowsep0" align="left">1.40</td><td class="colsep0 rowsep0" align="left">–6.75</td><td class="colsep0 rowsep0" align="left">0.36</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9a</b></td><td class="colsep0 rowsep0" align="left">378</td><td class="colsep0 rowsep0" align="left">20</td><td class="colsep0 rowsep0" colspan="6" align="center">no binding/weak</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11</b></td><td class="colsep0 rowsep0" align="left">381</td><td class="colsep0 rowsep0" align="left">30</td><td class="colsep0 rowsep0" align="left">278 ± 15</td><td class="colsep0 rowsep0" align="left">–10.28 ± 0.04</td><td class="colsep0 rowsep0" align="left">1.03 ± 0.003</td><td class="colsep0 rowsep0" align="left">–1.63</td><td class="colsep0 rowsep0" align="left">–8.65</td><td class="colsep0 rowsep0" align="left">0.43</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl2-fn1"><div class="footnote" id="tbl2-fn1"><sup>a</sup><p class="last">Titrations were carried out in 50 mM HEPES, pH 7.5 (at 25 °C), 150 mM NaCl, and 15 °C while stirring at 1000 rpm. In both cases the protein was titrated into the ligand solution (reverse titration). Titrations were performed in triplicate. Ligand efficiencies (LE) have also been calculated where Δ<i>G</i> values were available (LE = Δ<i>G</i>/<i>N</i> where <i>N</i> is the number of non-hydrogen atoms).</p></div></div></div></div><div id="sec2_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i52" class="anchor-spacer"></div><h3 class="article-section__title" id="_i52"> Bioluminescence Resonance Energy Transfer Assay (BRET)</h3><div class="NLM_p last">HEK293 cells (8 × 10<sup>5</sup>) were plated in each well of a 6-well plate and co-transfected with histone H3.3-HaloTag (NM_002107) and NanoLuc-BRD9 (Q9H8M2-BRD amino acids 120–240) or NanoLuc-BRD4 full-length (060885). Twenty hours post-transfection 2 × 10<sup>4</sup> cells were trypsinized, washed with PBS, and exchanged into media containing phenol red-free DMEM, 10% FBS in the absence (control sample), or the presence (experimental sample) of 100 nM NanoBRET 618 fluorescent ligand (Promega). Cell density was adjusted to 2 × 10<sup>5</sup> cells/mL and then replated in a 96-well assay white plate (Corning Costar no. 3917). At the time of replating compound <b>7d</b> or <b>11</b> were added at a final concentration spanning from 0.005 to 33 μM and the plates were incubated for 18 h at 37 °C in the presence of 5% CO<sub>2</sub>. NanoBRET substrate (Promega) was added to both control and experimental samples at a final concentration of 10 μM. Readings were performed within 5 min using the CLARIOstar reader (BMG Lifetechnologies) equipped with 450/80 nm bandpass and 610 nm long-pass filters. A corrected BRET ratio was calculated and is defined as the ratio of the emission at 610 nm/450 nm for experimental samples (i.e., those treated with NanoBRET fluorescent ligand) subtracted by and the emission at 610 nm/450 nm for control samples (not treated with NanoBRET fluorescent). BRET ratios are expressed as milliBRET units (mBU), where 1 mBU corresponds to the corrected BRET ratio multiplied by 1000.</div></div><div id="sec2_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i53" class="anchor-spacer"></div><h3 class="article-section__title" id="_i53"> Confocal Imaging</h3><div class="NLM_p last">HEK293 cells were transfected with histone H3.3-HaloTag using FuGENE HD (Promega). Twenty-four hours post-transfection cells were labeled with 5 μM HaloTag TMR ligand (Promega) in complete medium (DMEM and 10% FBS) for 15 min at 37 °C and 5% CO<sub>2</sub>. Medium containing HaloTag-TMR ligand was then washed twice with fresh complete medium. Cells were placed back at 37 °C and 5% CO<sub>2</sub> for 30 min and then imaged. Images were acquired on an Olympus Fluoview FV500 confocal microscope (Olympus, Center Valley, PA, USA) containing a 37 °C and CO<sub>2</sub> environmental chamber (Solent Scientific Ltd., Segensworth, U.K.) using appropriate filter sets.</div></div><div id="sec2_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i54" class="anchor-spacer"></div><h3 class="article-section__title" id="_i54"> In Vitro Cytotoxicity</h3><div class="NLM_p last">Following NanoBRET substrate addition and NanoBRET measurements, to the same wells an equal volume of CellTiter-Glo reagent (Promega) was added and the plates were incubated for 30 min at room temperature. Total luminescence was measured, and relative compound toxicity was determined by comparing the RLUs (relative light units) of a sample containing DMSO vehicle (in the absence of compound <b>11</b>) to the RLUs of the samples containing 0.005–33 μM compound <b>11</b>.</div></div><div id="sec2_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i55" class="anchor-spacer"></div><h3 class="article-section__title" id="_i55"> Crystallization</h3><div class="NLM_p last">Aliquots of the purified proteins were set up for crystallization using a mosquito crystallization robot (TTP Labtech, Royston, U.K.). Coarse screens were typically set up onto Greiner 3-well plates using three different drop ratios of precipitant to protein per condition (100 + 50 nL, 75 + 75 nL, and 50 + 100 nL). Initial hits were optimized further scaling up the drop sizes. All crystallizations were carried out using the sitting drop vapor diffusion method at 4 °C. BRD9 crystals with <b>7d</b> were grown by mixing 150 nL of the protein (17.9 mg/mL and 2 mM final ligand concentration) with 150 nL of reservoir solution containing 0.20 M sodium bromate, 0.1 M BT-propane, pH 7.5, 20% PEG3350, and 10% ethylene glycol. BRD4(1) crystals with <b>7d</b> were grown by mixing 200 nL of protein (12 mg/mL and 2 mM final ligand concentration) with 100 nL of reservoir solution containing 0.2 M sodium nitrate, 20% PEG3350, and 10% ethylene glycol. BRD4(1) crystals with <b>11</b> were grown by mixing 150 nL of protein (8.25 mg/mL and 2 mM final ligand concentration) with 150 nL of reservoir solution containing 0.1 M bis-tris-propane, pH 8.5, 20% PEG3350, and 10% ethylene glycol. In all cases diffraction quality crystals grew within a few days.</div></div><div id="sec2_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i56" class="anchor-spacer"></div><h3 class="article-section__title" id="_i56"> Data Collection and Structure Solution</h3><div class="NLM_p">All crystals were cryoprotected using the well solution supplemented with additional ethylene glycol and were flash frozen in liquid nitrogen. Data were collected in-house on a Rigaku FRE rotating anode system equipped with a RAXIS-IV detector at 1.52 Å (BRD9/<b>7d</b>) or on a Bruker MicroStar equipped with an APEX II detector at 1.54 Å (BRD4(1)/<b>7d</b> and BRD4(1)/<b>11</b>). Indexing and integration were carried out using SAINT (version 8.3, Bruker AXS Inc., 2013) or MOSFLM<a onclick="showRef(event, 'ref67'); return false;" href="javascript:void(0);" class="ref ref67">(67)</a> or XDS,<a onclick="showRef(event, 'ref68 ref69'); return false;" href="javascript:void(0);" class="ref ref68 ref69">(68, 69)</a> and scaling was performed with SCALA<a onclick="showRef(event, 'ref70'); return false;" href="javascript:void(0);" class="ref ref70">(70)</a> or XPREP (version 2008/2, Bruker AXS Inc.). Initial phases were calculated by molecular replacement with PHASER<a onclick="showRef(event, 'ref71'); return false;" href="javascript:void(0);" class="ref ref71">(71)</a> using the known models of BRD9 and BRD4(1) (PDB codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3HME">3HME</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2OSS">2OSS</a>, respectively). Initial models were built by ARP/wARP<a onclick="showRef(event, 'ref72'); return false;" href="javascript:void(0);" class="ref ref72">(72)</a> followed by manual building in COOT.<a onclick="showRef(event, 'ref73'); return false;" href="javascript:void(0);" class="ref ref73">(73)</a> Refinement was carried out in REFMAC5.<a onclick="showRef(event, 'ref74'); return false;" href="javascript:void(0);" class="ref ref74">(74)</a> In all cases thermal motions were analyzed using TLSMD<a onclick="showRef(event, 'ref75'); return false;" href="javascript:void(0);" class="ref ref75">(75)</a> and hydrogen atoms were included in late refinement cycles. Data collection and refinement statistics can be found in Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a>. The models and structure factors have been deposited in the PDB with accession codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4XY8">4XY8</a> (BRD9/<b>7d</b>), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4XY9">4XY9</a> (BRD4(1)/<b>7d</b>), and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4XYA">4XYA</a> (BRD4(1)/<b>11</b>).</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Isothermal Titration Calorimetry of Human BRD2 and BRD3 with Selected 9<i>H</i>-Purine Compounds<a class="ref internalNav" href="#tbl3-fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">protein/ligand</th><th class="colsep0 rowsep0" align="center">[P] (μM)</th><th class="colsep0 rowsep0" align="center">[L] (μM)</th><th class="colsep0 rowsep0" align="center"><i>K</i><sub>D</sub> (nM)</th><th class="colsep0 rowsep0" align="center">Δ<i>H</i><sup>obs</sup> (kcal/mol)</th><th class="colsep0 rowsep0" align="center"><i>N</i></th><th class="colsep0 rowsep0" align="center"><i>T</i>Δ<i>S</i> (kcal/mol)</th><th class="colsep0 rowsep0" align="center">Δ<i>G</i> (kcal/mol)</th><th class="colsep0 rowsep0" align="center">LE</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BRD2(1)/<b>7d</b></td><td class="colsep0 rowsep0" align="left">518</td><td class="colsep0 rowsep0" align="left">20</td><td class="colsep0 rowsep0" align="left">4444 ± 162</td><td class="colsep0 rowsep0" align="left">–6.62 ± 0.09</td><td class="colsep0 rowsep0" align="left">0.99 ± 0.011</td><td class="colsep0 rowsep0" align="left">0.44</td><td class="colsep0 rowsep0" align="left">–7.06</td><td class="colsep0 rowsep0" align="left">0.37</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BRD3(1)/<b>7d</b></td><td class="colsep0 rowsep0" align="left">532</td><td class="colsep0 rowsep0" align="left">20</td><td class="colsep0 rowsep0" align="left">2037 ± 105</td><td class="colsep0 rowsep0" align="left">–7.17 ± 0.09</td><td class="colsep0 rowsep0" align="left">1.01 ± 0.009</td><td class="colsep0 rowsep0" align="left">0.33</td><td class="colsep0 rowsep0" align="left">–7.50</td><td class="colsep0 rowsep0" align="left">0.39</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BRD2(1)/<b>11</b></td><td class="colsep0 rowsep0" align="left">488</td><td class="colsep0 rowsep0" align="left">20</td><td class="colsep0 rowsep0" align="left">1421 ± 60</td><td class="colsep0 rowsep0" align="left">–5.65 ± 0.04</td><td class="colsep0 rowsep0" align="left">1.02 ± 0.006</td><td class="colsep0 rowsep0" align="left">2.06</td><td class="colsep0 rowsep0" align="left">–7.71</td><td class="colsep0 rowsep0" align="left">0.39</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BRD3(1)/<b>11</b></td><td class="colsep0 rowsep0" align="left">510</td><td class="colsep0 rowsep0" align="left">20</td><td class="colsep0 rowsep0" align="left">2037 ± 105</td><td class="colsep0 rowsep0" align="left">–7.17 ± 0.09</td><td class="colsep0 rowsep0" align="left">1.01 ± 0.009</td><td class="colsep0 rowsep0" align="left">0.33</td><td class="colsep0 rowsep0" align="left">–7.50</td><td class="colsep0 rowsep0" align="left">0.38</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl3-fn1"><div class="footnote" id="tbl3-fn1"><sup>a</sup><p class="last">Titrations were carried out in 50 mM HEPES, pH 7.5 (at 25 °C), 150 mM NaCl, and 15 °C while stirring at 1000 rpm. In both cases the protein was titrated into the ligand solution (reverse titration). Titrations were performed in triplicate. Ligand efficiencies (LE) have also been calculated where Δ<i>G</i> values were available (LE = Δ<i>G</i>/<i>N</i> where <i>N</i> is the number of non-hydrogen atoms).</p></div></div></div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Data Collection and Refinement Statistics for Compound <b>7d</b> Complexes with BRD9 and BRD4(1) and Compound <b>11</b> with BRD4(1)</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" colspan="4" align="center">Data Collection</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="3" align="center">PDB code</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4XY8">4XY8</a></th><th class="colsep0 rowsep0" align="center"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4XY9">4XY9</a></th><th class="colsep0 rowsep0" align="center"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4XYA">4XYA</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">protein/ligand</td><td class="colsep0 rowsep0" align="left">BRD9/<b>7d</b></td><td class="colsep0 rowsep0" align="left">BRD4(1)/<b>7d</b></td><td class="colsep0 rowsep0" align="left">BRD4(1)/<b>11</b></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">space group</td><td class="colsep0 rowsep0" align="left">C2</td><td class="colsep0 rowsep0" align="left"><i>P</i>2<sub>1</sub>2<sub>1</sub>2<sub>1</sub></td><td class="colsep0 rowsep0" align="left"><i>P</i>2<sub>1</sub>2<sub>1</sub>2<sub>1</sub></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">cell dimensions</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>a</i>, <i>b</i>, <i>c</i> (Å)</td><td class="colsep0 rowsep0" align="left">80.63, 43.58, 40.81</td><td class="colsep0 rowsep0" align="left">37.23, 44.12, 78.58</td><td class="colsep0 rowsep0" align="left">37.24, 44.37, 79.01</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">α, β, γ (deg)</td><td class="colsep0 rowsep0" align="left">90.00, 104.25, 90.00</td><td class="colsep0 rowsep0" align="left">90.00, 90.00, 90.00</td><td class="colsep0 rowsep0" align="left">90.00, 90.00, 90.00</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">resolution<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a> (Å)</td><td class="colsep0 rowsep0" align="left">1.70 (1.79–1.70)</td><td class="colsep0 rowsep0" align="left">1.68 (1.77–1.68)</td><td class="colsep0 rowsep0" align="left">2.05 (2.15–2.05)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">unique observations<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">14841 (2064)</td><td class="colsep0 rowsep0" align="left">15354 (2213)</td><td class="colsep0 rowsep0" align="left">8708 (1126)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">completeness<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a> (%)</td><td class="colsep0 rowsep0" align="left">97.7 (95.2)</td><td class="colsep0 rowsep0" align="left">99.8 (99.4)</td><td class="colsep0 rowsep0" align="left">99.9 (99.9)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">redundancy<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">9.6 (9.4)</td><td class="colsep0 rowsep0" align="left">5.3 (5.0)</td><td class="colsep0 rowsep0" align="left">4.7 (3.2)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>R</i><sub>merge</sub><a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">0.086 (0.337)</td><td class="colsep0 rowsep0" align="left">0.154 (0.802)</td><td class="colsep0 rowsep0" align="left">0.079 (0.205)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>I</i>/σ<i>I</i><a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">20.3 (7.4)</td><td class="colsep0 rowsep0" align="left">8.2 (2.0)</td><td class="colsep0 rowsep0" align="left">12.4 (4.9)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="4" align="center">Refinement</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">resolution (Å)</td><td class="colsep0 rowsep0" align="left">1.60</td><td class="colsep0 rowsep0" align="left">1.68</td><td class="colsep0 rowsep0" align="left">2.05</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>R</i><sub>work</sub>/<i>R</i><sub>free</sub> (%)</td><td class="colsep0 rowsep0" align="left">17.4/27.5</td><td class="colsep0 rowsep0" align="left">25.9/28.1</td><td class="colsep0 rowsep0" align="left">14.9/20.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">no. of atoms (protein/other/water)</td><td class="colsep0 rowsep0" align="left">933/20/191</td><td class="colsep0 rowsep0" align="left">1059/27/143</td><td class="colsep0 rowsep0" align="left">1056/24/148</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>B</i>-factors (Å<sup>2</sup>) (protein/other/water)</td><td class="colsep0 rowsep0" align="left">14.23/6.78/27.23</td><td class="colsep0 rowsep0" align="left">14.21/22.85/24.34</td><td class="colsep0 rowsep0" align="left">13.99/10.49/22.04</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">rmsd bonds (Å)</td><td class="colsep0 rowsep0" align="left">0.016</td><td class="colsep0 rowsep0" align="left">0.016</td><td class="colsep0 rowsep0" align="left">0.015</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">rmsd angles (deg)</td><td class="colsep0 rowsep0" align="left">1.747</td><td class="colsep0 rowsep0" align="left">1.680</td><td class="colsep0 rowsep0" align="left">1.652</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Ramachadran</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">favored (%) </td><td class="colsep0 rowsep0" align="left">98.17</td><td class="colsep0 rowsep0" align="left">97.60</td><td class="colsep0 rowsep0" align="left">97.60</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">allowed (%) </td><td class="colsep0 rowsep0" align="left">1.83 </td><td class="colsep0 rowsep0" align="left">2.40 </td><td class="colsep0 rowsep0" align="left">2.40</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">disallowed (%)</td><td class="colsep0 rowsep0" align="left">0.00</td><td class="colsep0 rowsep0" align="left">0.00</td><td class="colsep0 rowsep0" align="left">0.00</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup>a</sup><p class="last">Values in parentheses correspond to the highest resolution shell.</p></div></div></div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i59">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_73671" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_73671" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In order to assess the binding of purine fragments on human BRDs, we first performed molecular docking experiments employing the previously determined crystal structure of the complex of BRD4(1) with a 5-methyltriazolopyrimidine ligand (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4MEN">4MEN</a><a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a>). To this end we investigated binding of purine fragments <b>1</b>, <b>2a</b>, and <b>2b</b> (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>B), seeking to determine acetyllysine competitive binding modes within the BRD cavity with promising predicted binding affinities, ideally establishing favorable interactions with residues implicated in acetyllysine peptide recognition. In order to account for putative conformational changes of the receptor’s binding site cavity upon ligand binding, we employed the Induced Fit docking protocol<a onclick="showRef(event, 'ref61 ref62'); return false;" href="javascript:void(0);" class="ref ref61 ref62">(61, 62)</a> (as implemented in the Schrödinger software package). Molecular modeling resulted in good accommodation of the investigated purine fragments within the Kac binding site of BRD4(1), mainly packing between the ZA-loop hydrophobic residues (Val87, Leu92, Leu94) and Ile146 from helix C, in a groove that is capped on one end by Tyr97 and Tyr139 and Trp81 on the other end (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>C). We observed different conformations of compound <b>1</b> within the BRD4(1) cavity, with the two chloro functions pointing to the top of the pocket (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>C) or adopting a Kac mimetic pose with one chlorine inserting deep into the pocket (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>D). Compound <b>2a</b> was also found in two different states in our calculations, either orienting its primary amine function away from the conserved asparagine (Asn140, Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>E) or directly engaging this residue while orienting its 6-Br substituent toward the ZA-loop (<a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a> Figure 1A,B). In all cases the ligand poses resulted in promising predicted binding energy values (−9.13 kcal/mol for <b>1</b>, −9.95 kcal/mol for <b>2a</b>, and −9.13 kcal/mol for <b>2b</b>). We identified in our calculations, in the case of compound <b>2a</b>, poses that exposed the halogen in position 6 so that the purine core scaffold may be further optimized based on its topology within the BRD binding site. The opposite was true in the case of the 2-Cl substituent or the methyl substituent at N9 (<b>1</b> or <b>2b</b>, respectively), which resulted in steric clashes and topologies that would not allow for subsequent optimizations. To better understand the binding mode of the purine scaffold to BRDs, given the multiple docking conformations observed, we decided to systematically probe the topology of these fragments employing synthetic chemistry and structure–activity relationships.</div><div class="NLM_p">We purchased 2,6-dichloro-9<i>H</i>-purine <b>1</b> and 2-amino-6-bromo-9<i>H</i>-purine <b>2a</b> and synthesized compound <b>2b</b> employing a TBAF-assisted N-9 methylation on the purine ring of <b>2a</b>. To confirm binding of these fragments to human bromodomains, we employed a thermal shift assay (Δ<i>T</i><sub>m</sub> assay) which we have previously used successfully with fragments and various human bromodomains.<a onclick="showRef(event, 'ref23 ref26 ref28 ref76 ref77'); return false;" href="javascript:void(0);" class="ref ref23 ref26 ref28 ref76 ref77">(23, 26, 28, 76, 77)</a> Typically we perform the assay using 100 μM compounds in the case of fragments and very weak ligands, but in the case of the purine analogues tested we were surprised to see binding at 10 μM compound to BET BRDs and in particular to BRD4(1); however, the optimized CDK inhibitor olomoucine did not show any significant stabilization toward any proteins in the panel (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>F, <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information Table S1">Supporting Information Table S1</a>). Encouraged by this result, we tested these compounds at the same concentration against five other diverse BRDs in an effort to cover most of the human BRD phylogenetic tree (<a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a> Figure 1C) and found that despite their structural diversity, the BRDs of CREBBP, PB1(5), and BRD9 exhibited weak binding. We were particularly intrigued by the interaction of <b>2a</b> with BRD9, since to our knowledge only a small subset of compounds has been previously shown to bind to this module.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> To further evaluate binding of <b>2a</b> onto BRD9, we performed docking experiments, using the recently crystallized apo structure of the protein (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3HME">3HME</a><a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a>) accounting for the flexibility of key residues after ligand binding.<a onclick="showRef(event, 'ref61 ref62'); return false;" href="javascript:void(0);" class="ref ref61 ref62">(61, 62)</a> Similar to our BRD4(1) docking experiment, we obtained two main binding poses for this compound, with the most energetically favored one (predicted binding affinity of −9.06 kcal/mol) exhibiting an extended hydrogen bond network with the conserved Asn100 and π–π interactions with the ZA-loop Tyr57 and Tyr106 from helix C, while the 6-Br substituent was found oriented toward the external part of the binding site toward the ZA-loop Phe47, suggesting that modifications on this position for subsequent optimization would not be tolerated without affecting the ligand orientation in the acetyllysine cavity (<a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a> Figure 1D). Conversely, we observed subtle rearrangement of the side chains of Phe44 and Tyr106 accompanied by almost a 90° rotation of the side chain of Phe47, suggesting an induced-fit binding mode. We also observed an alternative binding pose (predicted binding energy of −8.62 kcal/mol) in which the ligand maintained the hydrogen bond to Asn100, albeit from its primary amine function which inserted toward the conserved asparagine, as well as the π–π interaction with Tyr106 from helix C, while orienting the modifiable 6-Br substituent toward the top of the BRD cavity (<a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a> Figure 1E), offering a promising vector for subsequent modifications. Using both possible orientations as starting points, we decided to further interrogate how the purine core scaffold binds to this BRD.</div><div class="NLM_p">Encouraged by our initial findings, we synthesized a number of 2-amino-9<i>H</i>-purine analogues and tested their ability to interact with human BRDs, primarily of the BET subfamily (subfamily II), while systematically testing binding to representative BRDs from other structural subfamilies (family I, PCAF; family III, CREBBP; family IV, BRD9; family V, BAZ2B; family VIII, PB1(5); see <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a> Figure 1C) in order to probe structurally diverse proteins against this core purine scaffold. We utilized an aqueous-phase Suzuki–Miyaura cross-coupling reaction to synthesize 2-amino-6-aryl-9<i>H</i>-purine derivatives, yielding highly C-6 decorated 9<i>H</i>-purines in a one-step procedure and performed a subsequent TBAF-assisted N-9 alkylation to access N-9 substituted analogues (Scheme <a class="ref internalNav" href="#sch1" aria-label="1">1</a> and Chart <a class="ref internalNav" href="#cht2" aria-label="2">2</a>). We accomplished the coupling step under microwave irradiation with Pd(OAc)<sub>2</sub> and triphenylphosphine-3,3′,3″-trisulfonic acid trisodium salt as the catalytic system, with Cs<sub>2</sub>CO<sub>3</sub> as base, in a water–acetonitrile reaction solvent. This approach allowed synthesizing 2-amino-6-aryl-9<i>H</i>-purines with very short reaction times (5–15 min) at high yields and purity.</div><figure id="sch1" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-6/jm501893k/20150418/images/medium/jm-2014-01893k_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-6/jm501893k/20150418/images/large/jm-2014-01893k_0006.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. General Procedures for the Synthesis of Purine Derivatives: (A) Suzuki–Miyaura Cross-Coupling of Free Halopurines and (B) TBAF-Assisted N-9 Alkylation on the Purine Ring<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-6/jm501893k/20150418/images/large/jm-2014-01893k_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm501893k&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) Pd(OAc)<sub>2</sub>/P(C<sub>6</sub>H<sub>4</sub>SO<sub>3</sub>Na)<sub>3</sub>, Cs<sub>2</sub>CO<sub>3</sub>, MeCN/H<sub>2</sub>O (1:2), microwaves, 150 °C, 5–15 min; (b) CH<sub>3</sub>I or CH<sub>3</sub>COCH<sub>2</sub>Cl, TBAF, THF, rt, 10 min; (c) 50% H<sub>2</sub>SO<sub>4</sub>, NaNO<sub>2</sub>, −10 °C, 2 h, then 50 °C, 1 h.</p></p></figure><div class="NLM_p">First we introduced a phenyl substituent at position 6 of the core purine scaffold (compound <b>3a</b>) only to find that BRD2(1), BRD4(1), and PCAF were stabilized in thermal melt assays by this compound (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>A,B). We confirmed binding to BRD4(1) by isothermal titration calorimetry and measured a dissociation constant of 11.99 μM (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>C and Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>). The interaction of <b>3a</b> with BRD4(1) was mainly driven by enthalpic contributions (Δ<i>H</i> = −9.45 kcal/mol), opposed by negative entropy (<i>T</i>Δ<i>S</i> = −2.97 kcal/mol). We were also intrigued by the lack of affinity toward BRD9 in the Δ<i>T</i><sub>m</sub> assay, since our initial fragment hit, compound <b>2a</b>, had exhibited a thermal shift of 1.6 °C toward that domain. We therefore tested this scaffold by isothermal titration calorimetry against BRD9 and measured a weak dissociation constant of 8.5 μM (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>D, Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>) suggesting that our primary assay (Δ<i>T</i><sub>m</sub>) may not be very robust in the case of BRD9 when applied to weak ligands. This is important, since the first bromodomain of BRD4 has been repeatedly shown to bind to weak compounds employing this assay,<a onclick="showRef(event, 'ref76'); return false;" href="javascript:void(0);" class="ref ref76">(76)</a> while it has been noted that other BRDs do not always exhibit high temperature shifts although they bind to several compounds very potently.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> Additionally, in the case of BET proteins it has been demonstrated that thermal melt data correlate well with in vitro dissociation constants.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> Importantly, similar to the BRD4(1)/<b>3a</b> interaction, the compound interacted with BRD9 mainly driven by enthalpic contributions (Δ<i>H</i> = −9.11 kcal/mol) opposed by negative entropy (<i>T</i>Δ<i>S</i> = −2.42 kcal/mol).</div><figure id="fig2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-6/jm501893k/20150418/images/medium/jm-2014-01893k_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-6/jm501893k/20150418/images/large/jm-2014-01893k_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. SAR of 9<i>H</i>-purines. The 9<i>H</i>-purine core scaffold was iteratively decorated and tested for binding to human bromodomains. (A) Substitution patterns explored. (B) Thermal shift assay against human bromodomains. Thermal shifts are color-coded as indicated in the inset. Compounds highlighted with a colored star were further validated by isothermal titration calorimetry as shown in (C)/(D). (C) Isothermal titration calorimetry validation of key compounds binding to BRD4(1) showing raw injection heats for titrations of protein into compound. The inset shows the normalized binding enthalpies corrected for the heat of protein dilution as a function of binding site saturation (symbols as indicated in the figure). Solid lines represent a nonlinear least-squares fit using a single-site binding model. (D) Compounds bearing ortho,meta′ substitutions gain potency toward BRD9 as demonstrated by ITC experiments. Data have been corrected and displayed as described in (C). All ITC titrations were carried out in 50 mM HEPES, pH 7.5 (at 25 °C), 150 mM NaCl, and 15 °C while stirring at 1000 rpm.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-6/jm501893k/20150418/images/large/jm-2014-01893k_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm501893k&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In order to better explore the structure–activity relationship of 6-phenyl substituted 9<i>H</i>-purines, we synthesized a range of compounds carrying different patterns of functions, including para-substitutions (compounds <b>3b</b>–<b>h</b>), meta,para substitutions (compounds <b>4a</b>–<b>e</b>), meta,meta′ substitutions (compounds <b>5a</b>,<b>b</b>), ortho,ortho′ substitutions (compounds <b>6a</b>–<b>c</b>), and ortho,meta′ substitutions (compounds <b>7a</b>–<b>e</b>) (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>A and Chart <a class="ref internalNav" href="#cht2" aria-label="2">2</a>). We tested binding of these analogues to the eight BET BRDs, as well as the five more diverse BRDs previously mentioned, employing the same thermal shift assay as above. Interestingly, compounds synthesized that carried an additional methyl modification at N9 (compounds <b>3c</b>, <b>3e</b>, <b>4e</b>, <b>5b</b>) exhibited very weak or no binding toward most BRDs while showing small thermal shits (1.0–1.3 °C) for the bromodomain of CREBBP. Para substitution of the 6-phenyl-9<i>H</i>-purines (compounds <b>3b</b>–<b>h</b>) resulted in lower stabilization of all BRDs; however, compound <b>3f</b> showed binding toward all BRDs without any hints of selectivity toward BRD9. Meta substitutions (compounds <b>4a</b>–<b>e</b>) were also very weak across BRDs with no affinity for BRD9, suggesting that modifications on that vector were not tolerated. Interestingly meta,meta′ substitution (compound <b>5a</b>) resulted in binding to most BRDs albeit weak, with Δ<i>T</i><sub>m</sub> values between 1.1 and 1.8 °C. As expected, this binding event was abolished when the compound was methylated at N9 (compound <b>5b</b>). Binding was not improved with ortho,ortho′ substitutions of the 6-phenyl 9<i>H</i>-purine scaffold (compounds <b>6a</b>–<b>c</b>) (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>B and <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information Table S1">Supporting Information Table S1</a>). Since methyl substitution at N9 could not be tolerated in BRD4(1) or BRD9 binding, we concluded at this stage that the five-membered ring is probably not oriented toward the top of the BRD cavity but points toward the bottom of the acetyllysine binding cavity, as predicted in our docking model (<a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a> Figure 1A,B), with the 6-substituted position toward the front of the pocket in order to accommodate the larger phenyl-substituted functions.</div><div class="NLM_p">We decided to further test combinations in ortho,meta′ substituted compounds by first maintaining a methoxy functionality at the ortho position while changing the steric bulk at the meta′ position (compounds <b>7a</b>–<b>d</b>). 2-Methoxyphenyl substitution (compound <b>7a</b>) yielded thermal shifts between 1.4 and 2.5 °C for BET BRDs as well as 1.5 °C for the BRD of CREBBP while significantly stabilizing BRD9 compared to all previous compounds tested (2.9 °C). We confirmed this binding by ITC and measured a dissociation constant of 641 nM against BRD9 (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>D, Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>). Notably the change in affinity was accompanied by negative entropic contributions (<i>T</i>Δ<i>S</i> = −4.55 kcal/mol). Intrigued by this step change in affinity, we tested halide analogues at the meta′ position while retaining the ortho-methoxy functionality (compounds <b>7b</b>–<b>d</b>) and observed improved thermal shifts for all compounds tested against BRD9, while BET affinity seemed to be variable. Notably, we observed Δ<i>T</i><sub>m</sub> values following the order H < F < Cl > Br, suggesting that steric bulk and charge at the meta′ position is important, with compound <b>7c</b> showing a Δ<i>T</i><sub>m</sub> of 3.8 °C against BRD9. We validated binding using ITC as an orthogonal method and measured dissociation constants of 351, 297, and 397 nM against BRD9 following the same ranking as the thermal shift assay for compounds <b>7b</b>, <b>7c</b>, and <b>7d</b>, respectively (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>D, Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>). Importantly, we observed a gradual increase in entropic contributions with increasing bulk of the 5-halide-2-methoxyphenyl substitution, with compound <b>7a</b> exhibiting the lowest entropic term (<i>T</i>Δ<i>S</i> = −4.55 kcal/mol) and compound <b>7d</b> the highest (<i>T</i>Δ<i>S</i> = −1.57 kcal/mol). Interestingly, BRD4(1), which exhibited Δ<i>T</i><sub>m</sub> values of 1.1 and 3.2 °C for compounds <b>7c</b> and <b>7d</b>, was found to bind weakly to these scaffolds by ITC, and the dissociation constants that we measured were 2.04 and 4.7 μM, respectively, following again the same trend seen in the thermal melt assay (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>C, Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>). Affinity for BRD9 was lost when we synthesized compound <b>7e</b> which carried a bromine function at the meta′ position and an ethoxy substituent at the ortho position, suggesting that the longer and bulkier group probably affects rotation of the 6-aryl ring with respect to the core 9<i>H</i>-purine-2-amine fold.</div><div class="NLM_p">In order to verify the mode of interaction of the 9<i>H</i>-purine core within the BRD acetyllysine binding cavity, we crystallized and determined the complexes of compound <b>7d</b> with the bromodomain of BRD9 and the first bromodomain of BRD4. The ligand was found in both cases to occupy the acetyllysine recognition pocket (Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>A and <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a> Figure 2A) and was clearly defined in the electron density map (Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>B and <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a> Figure 2B). Compound <b>7d</b> directly engaged, via the primary amine function as well as the nitrogen at position 3, the conserved asparagine in both structures (Asn100 in BRD9; Asn140 in BRD4(1)) and was additionally held in place via a number of hydrogen bonds to the protein backbone and the network of conserved water molecules previously described<a onclick="showRef(event, 'ref76'); return false;" href="javascript:void(0);" class="ref ref76">(76)</a> (Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>C and <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a> Figure 2C). Notably, compound <b>7d</b> initiated hydrogen bonds to a water molecule located on top of the BRD cavity, which in turn linked the ligand to the top of the ZA-loop, either to the carbonyl of Ile53 (in the case of BRD9) or Asn93 (in the case of BRD4(1)). This tight network of interactions explains the potency of this compound for BRDs. The mode of interaction is similar to that observed before for triazolothienodiazepine complexes such as (+)-JQ1, with the five-membered ring of the purine system acting as the acetyllysine mimetic moiety, superimposing well with the methyltriazolo ring of (+)-JQ1 (<a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a> Figure 2D) while the bromomethoxyphenyl substituent of <b>7d</b> stacks well between the ZA-loop Leu92 and the ZA-channel’s Phe81/Pro82 of BRD4(1) (<a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a> Figure 2E). Consistent with our induced fit computational models of binding of <b>2a</b> to BRD9, superimposition of the BRD9/<b>7d</b> complex to the apo structure of BRD9 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3HME">3HME</a>)<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> revealed rotations of the side chains of Phe47 and Phe44 while the top of the ZA loop collapsed toward the ligand (Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>D,E), resulting in an unprecedented induced fit pocket of BRD9. Particularly the side chain of Phe47 rotated 120°, thus blocking the ZA-channel of the protein resulting in a steric bulk around compound <b>7d</b>. On the basis of our thermodynamic measurements showing high entropic contributions to binding (Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>), we concluded that all compounds in this subseries (<b>7a</b>–<b>d</b>) should be inducing a rearrangement of the binding cavity residues of BRD9 upon binding to the acetyllysine site. Intriguingly as the substituent size increases, the affinity also increases as exemplified by the determined dissociation constants, with the chloro analogue (<b>7c</b>) exhibiting a dissociation constant of 297 nM against BRD9 while the bromo analogue (<b>7d</b>) is slightly weaker (397 nM against BRD9; 4.65 μM against BRD4(1)). However, the structural rearrangement that we observed was unique to BRD9; the structure of compound <b>7d</b> in complex with the first bromodomain of BRD4 did not reveal any rearrangements of the acetyllysine binding cavity as the inhibitor packed between Trp81 and Leu92 of the ZA-loop (<a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a> Figure 2D,E). It is tempting to speculate that the observed increase in entropic contributions within the series of compounds <b>7a</b>–<b>d</b> measured by ITC, following the increase in the halide substituent size, is associated with the side chain rearrangement observed by docking as well as the crystal structure of compound <b>7d</b> with the bromodomain of BRD9. However, entropy changes may be due to a number of factors, such as release of water molecules, therefore making further interpretation of the small data set obtained very difficult.</div><figure id="fig3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-6/jm501893k/20150418/images/medium/jm-2014-01893k_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-6/jm501893k/20150418/images/large/jm-2014-01893k_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Induced fit binding of 9<i>H</i>-purines to BRD9. (A) Overall fold of BRD9/<b>7d</b> complex. (B) 2<i>F</i><sub>c</sub><i>F</i><sub>o</sub> map of <b>7d</b> in complex with BRD9 contoured at 2σ. (C) <b>7d</b> occupies the acetyllysine binding cavity of the bromodomain module initiating direct interactions with the conserved asparagine (N100) and packing onto the ZA-loop hydrophobic backbone (F44, F47, I53, A54, P56, Y57). The conserved water network is preserved in the structure, and the ligand initiates hydrogen bonds that further stabilize binding. Notably the ligand engages a water molecule located on the top of the BRD cavity in a network of hydrogen bonds to the backbone of I53. (D) Binding of <b>7d</b> to BRD9 results in a distinct rearrangement of the BRD fold. Residues on both sides of the ligand shift toward it, forming an induced-fit pocket with F47 rotating 120° capping the channel found in the apo BRD9 structure at the front of the ZA-loop. (E) Surface view of the side chain rearrangement described in (C) highlighting the induced pocket upon binding of <b>7d</b> to BRD9. The model and structure factors of the BRD9/<b>7d</b> complex shown have been deposited to the PDB with ascension code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4XY8">4XY8</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-6/jm501893k/20150418/images/large/jm-2014-01893k_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm501893k&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We next questioned whether the primary amine function at position 2 of the 9<i>H</i>-purine core scaffold is necessary for binding to bromodomains. First we substituted the amine with a chlorine group (compound <b>8a</b>, Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>A) while retaining the 6-(5-bromo-2-methoxyphenyl) substitution, resulting in loss of affinity toward all BRDs in our panel (Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>B). In the case of BRD9 we confirmed this observation by performing an isothermal titration calorimetry measurement which yielded a <i>K</i><sub>d</sub> of 7.9 μM (Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>C). As with compounds from previous series, methyl substitution at position 9 completely abolished bromodomain binding (compound <b>8b</b>, Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>A), measured by both thermal melt (Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>B) and ITC assays in the case of BRD9 (Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>C); larger substituents (compound <b>8c</b>) could not be tolerated suggesting that the core scaffold retained its pose within the bromodomain binding cavity. Hydroxy substitution at position 2 while retaining a 6-(5-halide-2-methoxyphenyl) substituent (compounds <b>8d</b>–<b>f</b>) had variable effects on the 9<i>H</i>-purine affinity toward BRDs. Moreover, fluoro (<b>8d</b>) and bromo (<b>8f</b>) substituted compounds lost affinity across the panel, while the chloro-substituted compound (<b>8e</b>) promiscuously bound to most bromodomains in the Δ<i>T</i><sub>m</sub> assay, albeit weaker than its primary amine analogue <b>7c</b> (Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>A,B), thus suggesting that the interactions initiated by the hydroxyl group and the conserved asparagine (Asn100 in BRD9; Asn140 in BRD4(1)) are not favored over the primary amine.</div><figure id="fig4" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-6/jm501893k/20150418/images/medium/jm-2014-01893k_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-6/jm501893k/20150418/images/large/jm-2014-01893k_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. BRD pocket SAR. (A) Compounds designed to probe the acetyllysine mimetic character of the purine scaffold, disrupting interactions (N9-methyl analogues) or reaching deeper into the acetyllysine cavity (8-methyl analogues). (B) Thermal shift assay against human bromodomains. Compound <b>10</b> was heavily colored and interfered with the assay. Thermal shifts are color-coded as indicated in the inset. Compounds highlighted with a colored star were further validated as shown in (C)/(D). (C) Substitution of the primary amine group to a hydroxyl (compound <b>8a</b>) impairs binding toward BRD9 as demonstrated by ITC experiments, while cyclization of the aromatic substituent results in enhanced potency (compound <b>11</b>). The inset shows the normalized binding enthalpies corrected for the heat of protein dilution as a function of binding site saturation (symbols as indicated in the figure). Solid lines represent a nonlinear least-squares fit using a single-site binding model. (D) Isothermal titration calorimetry validation of compound <b>11</b> binding to BRD4(1) showing raw injection heats for titrations of protein into compound. Data have been corrected and displayed as described in (C). All ITC titrations were carried out in 50 mM HEPES, pH 7.5 (at 25 °C), 150 mM NaCl, and 15 °C while stirring at 1000 rpm.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-6/jm501893k/20150418/images/large/jm-2014-01893k_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm501893k&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The 9<i>H</i>-purine crystal structure complexes that we obtained with the bromodomains of BRD9 and BRD4(1) highlighted the failure of the ligand to insert deep inside the bromodomain cavity, thus not replacing the conserved network of water molecules. In an attempt to reach deeper within the cavity, we synthesized compounds <b>9a</b> and <b>9b</b>, introducing a methyl group at position 8 of the 9<i>H</i>-purine core. The low solubility of compound <b>9a</b> did not allow for any measurements, but compound <b>9b</b> exhibited very low affinity for all bromodomains in our panel with the exception of BRDT(2) (Δ<i>T</i><sub>m</sub> of 2.9 °C), suggesting that substitution at this position of the 9<i>H</i>-purine core in the absence of a 2-amine function cannot be tolerated and these compounds could not displace the conserved water molecule network deep inside the bromodomain pocket. Our synthetic efforts to insert a fluorine atom at position 8 through a C-8 electrophilic fluorination on the bis(tetrahydropyran-2-yl)-protected derivative of <b>2a</b>, following a reported metalation–fluorination reaction with <i>N</i>-fluorobenzenesulfonimide,<a onclick="showRef(event, 'ref78'); return false;" href="javascript:void(0);" class="ref ref78">(78)</a> were also unsuccessful; we observed formation of the corresponding 8-phenylsulfonyl product instead of the 8-fluoro derivative, similar to reported work done by Roy and co-workers,<a onclick="showRef(event, 'ref79'); return false;" href="javascript:void(0);" class="ref ref79">(79)</a> even under heterogeneous conditions.</div><div class="NLM_p">Our structural insight suggested that augmentation of the 6-(5-halide-2-methoxyphenyl) substituent would take a toll on BRD9, as it would force the structure toward an apo-like conformation, sterically pushing Phe47 toward the apo-conformation of BRD9, while it should not affect binding to BET bromodomains which contained an open channel between Trp81 and Leu92 (Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>E and <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a> Figure 2E) . To test this hypothesis, we synthesized compound <b>10</b> (Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>A) and tested its ability to bind to human BRDs. Unfortunately its bright yellow color and low solubility did not allow for further validation. We therefore decided to increase the size of the substituent found in <b>7d</b> by cyclizing the 2-methoxyphenyl ring into a 2,3-dihydrobenzofuran-7-yl while retaining the bromo function at position 5. We tested the resulting compound <b>11</b> in the Δ<i>T</i><sub>m</sub> assay against the panel of bromodomains and observed increased temperature shifts among BET BRDs (between 1.7 and 5.4 °C) together with a remarkable increase in the case of BRD9 (6.5 °C). To verify this observation, we performed isothermal titration calorimetry measurements and obtained a dissociation constant of 278 nM for BRD9 (Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>C) while BRD4(1) binding resulted in a much weaker affinity (1.4 μM) (Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>D). Notably the entropic contribution of compound <b>11</b> binding to BRD9 was comparable to compound <b>7d</b> for both BRD9 and BRD4(1), and thus we speculated that a similar rearrangement should be taking place upon binding to the bromodomain of BRD9.</div><div class="NLM_p">In order to test our hypothesis that compound <b>11</b> binding would also result in structural rearrangement of the bromodomain binding cavity of BRD9, in a similar way to compound <b>7d</b>, while not affecting the cavity of BRD4(1), we attempted to determine the crystal structures of this compound in complex with these two bromodomains. Compound <b>11</b> readily crystallized with BRD4(1) and was found to occupy the acetyllysine binding cavity of the bromodomain (Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>A) and was very well-defined in the density (Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>B) despite its weaker binding affinity of 1.37 μM toward BRD4(1) (Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>). The ligand was found to directly interact with the conserved asparagine (Asn140) as well as with the conserved network of water molecules while packing between the ZA-channel tryptophan (Trp81) and the ZA-loop leucine (Leu92) (Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>C). Our efforts however to obtain a BRD9/<b>11</b> complex did not yield diffracting quality crystals suitable for structure determination, and as such, we employed computational methods to account for its binding to BRD9. Rigid docking into the BRD9/<b>7d</b> complex structure resulted in a conformation similar to that observed with compound <b>7d</b>, with the ligand engaging the conserved asparagine via its primary amine function and the 6-aryl-substituted ring packing betweent the ZA-loop Ile53 and Phe44 (<a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a> Figure 3A). We then performed induced-fit docking employing the algorithms previously described for purine fragments using the complex of BRD9/<b>7d</b> as starting point and obtained a pose whereby the 2-amine function inverted and inserted in the BRD pocket, without any changes in the surrounding side chains of Phe44, Phe47, Ile53, and Tyr106 (<a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a> Figure 3B). Intrigued by this finding, we performed another induced fit docking experiment, starting with the BRD9 apo structure and allowing residues to freely move in the presence of the ligand. We observed a similar set of side chain rearrangement within the BRD9 acetyllysine cavity, including a rotation of Phe47 resulting in capping of the binding groove, accompanied by repositioning of Phe44 from helix C and Ile53 from the ZA-loop (<a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a> Figure 3C). We therefore concluded that the 2-amine-9<i>H</i>-purine scaffolds that we developed can induce a closed pocket within the bromodomain of BRD9 resulting in tight binding, without at the same time exhibiting high affinity for BRD4(1) or other BET family members. Indeed, we determined the dissociation constants for binding to BET domains employing ITC and found that both compounds <b>7d</b> and <b>11</b> exhibited low micromolar affinities (1.4–4.6 μM) correlating well with the higher thermal shifts observed (<a class="ref internalNav" href="#notes-1" aria-label="Supporting Information Table S1">Supporting Information Table S1</a>), while retaining higher affinity against the bromodomain of BRD9 (<a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a> Figure 4 and Tables <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a> and <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>).</div><figure id="fig5" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-6/jm501893k/20150418/images/medium/jm-2014-01893k_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-6/jm501893k/20150418/images/large/jm-2014-01893k_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Complex of compound <b>11</b> and 9<i>H</i>-purine activity in cells. (A) Overview of the complex of compound <b>11</b> with the first bromodomain of BRD4. The ligand retained the acetyllysine mimetic pose that was observed in the case of <b>7d</b>. (B) Compound <b>11</b><i>F</i><sub>c</sub><i>F</i><sub>o</sub> omit map from the BRD4(1)/<b>11</b> complex contoured at 2σ. (C) Detail of compound <b>11</b> biding to BRD4(1) demonstrating the acetyllysine mimetic binding mode, initiating interactions with the conserved asparagine (N140), and packing between the ZA-loop L92 and the ZA-channel W81 while retaining the network of conserved water interactions. The model and structure factors of the BRD4(1)/<b>11</b> complex shown in panels A, B, and C have been deposited to the PDB with ascension code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4XYA">4XYA</a>. (D) Titration of compounds <b>7d</b> and <b>11</b> into HEK293 cells transfected with NanoLuc-fused bromodomain of BRD9 and Halo-tagged histone H3.3. The ligands disrupt the histone/BRD9 interaction with an apparent IC<sub>50</sub> of 3.5 μM (<b>7d</b>) and 480 nM (<b>11</b>), resulting in loss of signal due to separation of the BRD9-histone complex. (E) Titration of compounds <b>7d</b> and <b>11</b> into HEK293 cells transfected with NanoLuc-fused full length BRD4 (UniProt code <a href="https://O60885" class="ext-link">O60885</a>) and Halo-tagged histone H3.3. Although the ligands gradually disrupt the BRD4-FL/H3.3 interaction, they fail to elicit the same effect as in the case of BRD9 in the concentration range tested.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-6/jm501893k/20150418/images/large/jm-2014-01893k_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm501893k&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Having obtained a small window of selectivity over BRD4, we wanted to verify that the 2-amine-9<i>H</i>-purine scaffolds that we developed are in fact active in a cellular environment and can perturb the interaction of BRD9 with acetylated histones. Previous studies have established that potent compounds that competitively bind to the acetyllysine binding cavity of BET proteins can displace the entire protein from chromatin in fluorescence recovery after photobleaching (FRAP) assays,<a onclick="showRef(event, 'ref23 ref35 ref77'); return false;" href="javascript:void(0);" class="ref ref23 ref35 ref77">(23, 35, 77)</a> while larger BRD-containing proteins that contain in addition multiple reader domains are harder to fully displace; in the case of CREBBP this was solved by creating an artificial GFP-fusion construct that contained three BRD modules of CREBBP and a nuclear localization signal.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> BRD9 is part of the large SWI/SNF complex,<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> and its bromodomain has been shown to bind to acetylated histone H3 peptides.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> We speculated that 9<i>H</i>-purines should be able to competitively displace the bromodomain of BRD9 from chromatin and constructed a bioluminescence resonance energy transfer (BRET) system that combined NanoLuc luciferase fusions of the BRD9 bromodomain (<a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a> Figure 5A) or full length BRD4 and Halo-tagged histone H3.3 as BRET pairs. This assay is an excellent tool to quantify protein–ligand interactions in a cellular system<a onclick="showRef(event, 'ref80'); return false;" href="javascript:void(0);" class="ref ref80">(80)</a> and has recently been used to determine cellular IC<sub>50</sub> values for the inhibition of the histone–bromodomain interaction in the case of BRPF1 using a 1,3-dimethylbenzimidazolone scaffold.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> First we established by fluorescence microscopy that Halo-tagged histone H3.3 is readily incorporated into chromatin (<a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a> Figure 5B). We then performed dose–response experiments and demonstrated the NanoLuc-BRD9 bromodomain was readily displaced from chromatin by compounds <b>7d</b> and <b>11</b> with cellular IC<sub>50</sub> values of 3.5 ± 0.11 μM and 477 ± 194 nM, respectively (Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>D), in agreement with the in vitro affinities determined by ITC (Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>). In contrast, full-length BRD4 was not completely displaced in this assay up to concentrations of 33 μM of both compounds (Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>E), suggesting that the compounds retained the in vitro selectivity toward BRD9 in this cellular system. We further examined the toxicity of compound <b>11</b> toward HEK293 cells, using cell viability in the presence of the compound in the concentration regime of our BRET experiments as a readout and did not observe any cytotoxicty (<a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a> Figure 5C), suggesting that this compound can be used in cellular systems to target BRD9/Kac interactions without affecting BRD4/Kac interactions or causing any cytotoxic responses.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i61">Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_38156" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_38156" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">We have described here a structure-guided approach to identify inhibitors for diverse BRDs starting from small fragment-like 9<i>H</i>-purine scaffolds. Through structure–activity relationship we established compounds <b>7d</b> and <b>11</b> and characterized them as nanomolar binders for the BRD of BRD9 while retaining weak in vitro activity against the first bromodomain of BRD4. These compounds are not cytotoxic at concentrations up to 33 μM in a HEK293 system and can competitively displace the BRD9 bromodomain from chromatin while failing to displace full length human BRD4 at the same concentration range in bioluminescence proximity assays, despite the fact that the in vitro affinity difference for these two proteins is not large. Importantly, compound <b>7d</b> induces structural rearrangement on the acetyllysine binding cavity of BRD9 resulting in an unprecedented cavity shape which accommodates this scaffold, explaining the higher affinity toward this protein, while docking studies suggest that compound <b>11</b> elicits the same type of structural rearrangement. The 9<i>H</i>-purine scaffold therefore offers a simple template that can be used to generate initial tools that will no doubt prove useful in interrogating the biology of bromodomains beyond the BET subfamily, such as BRD9, which have not attracted attention until now, avoiding cross-reactivity with BET bromodomains.</div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i62"><a href="/doi/suppl/10.1021/jm501893k">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_53316" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_53316" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">Figures showing alternative docking of <b>2a</b>, binding of <b>7d</b> to BRD4(1), docking of <b>11</b> to BRD9, and binding of <b>7d</b> and <b>11</b> to BET bromodomains and a file in xlsx format listing Δ<i>T</i><sub>m</sub> shifts of compounds. This material is available free of charge via the Internet at <a href="http://pubs.acs.org" class="extLink">http://pubs.acs.org</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm501893k/suppl_file/jm501893k_si_001.xlsx">jm501893k_si_001.xlsx (28.79 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm501893k/suppl_file/jm501893k_si_002.pdf">jm501893k_si_002.pdf (3.15 MB)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The models and structure factors have been deposited in the PDB with accession codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4XY8">4XY8</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4XY9">4XY9</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4XYA">4XYA</a>.</p><div class="testing" data-doi="10.1021/jm501893k" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_78101" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_78101" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Giuseppe Bifulco</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmacy, University of Salerno, Via Giovanni Paolo II, 132, 84084 Fisciano, Italy</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#b5d7dcd3c0d9d6daf5c0dbdcc6d49bdcc1"><span class="__cf_email__" data-cfemail="eb89828d9e878884ab9e8582988ac5829f">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ines Bruno</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmacy, University of Salerno, Via Giovanni Paolo II, 132, 84084 Fisciano, Italy</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#ceacbcbba0a1a7a08ebba0a7bdafe0a7ba"><span class="__cf_email__" data-cfemail="2f4d5d5a414046416f5a41465c4e01465b">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Panagis Filippakopoulos</span> - <span class="hlFld-Affiliation affiliation">Nuffield
Department of Clinical Medicine, Structural
Genomics Consortium, University of Oxford, Old Road Campus Research Building, Roosevelt Drive, Oxford OX3 7DQ, U.K.</span>; 
    <span class="hlFld-Affiliation affiliation">Nuffield
Department of Clinical Medicine, Ludwig
Cancer Research, University of Oxford, Old Road Campus Research Building, Roosevelt Drive, Oxford OX3 7DQ, U.K.</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#a7d7c6c9c6c0ced489c1cecbced7d7c6ccc8d7c8d2cbc8d4e7d4c0c489c8df89c6c489d2cc"><span class="__cf_email__" data-cfemail="f8889996999f918bd69e919491888899939788978d94978bb88b9f9bd69780d6999bd68d93">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sarah Picaud</span> - <span class="hlFld-Affiliation affiliation">Nuffield
Department of Clinical Medicine, Structural
Genomics Consortium, University of Oxford, Old Road Campus Research Building, Roosevelt Drive, Oxford OX3 7DQ, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Maria Strocchia</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmacy, University of Salerno, Via Giovanni Paolo II, 132, 84084 Fisciano, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Stefania Terracciano</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmacy, University of Salerno, Via Giovanni Paolo II, 132, 84084 Fisciano, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Gianluigi Lauro</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmacy, University of Salerno, Via Giovanni Paolo II, 132, 84084 Fisciano, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jacqui Mendez</span> - <span class="hlFld-Affiliation affiliation">Promega
Corporation, 2800 Woods
Hollow Road, Madison, Wisconsin 53711, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Danette L. Daniels</span> - <span class="hlFld-Affiliation affiliation">Promega
Corporation, 2800 Woods
Hollow Road, Madison, Wisconsin 53711, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Raffaele Riccio</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmacy, University of Salerno, Via Giovanni Paolo II, 132, 84084 Fisciano, Italy</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>S.P. and M.S. contributed equally. The manuscript was written through contributions of all authors. All authors have given approval to the final version of the manuscript. </p></li><li></li><li><span class="author-information-subsection-header">Notes</span><p>The authors declare no competing financial interest.<br /></br></p></li></ul></div><div class="ack" id="ACK-d2482e5407-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i65">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_61563" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_61563" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">Financial support by the University of Salerno is gratefully acknowledged. The authors also acknowledge the use of the instrument facilities of the Centre of Competence in Diagnostics and Molecular Pharmaceutics supported by Regione Campania (Italy) through POR funds, an Associazione Italiana Ricerca sul Cancro (AIRC) fellowship (G.L.), and Grant IG 2012-IG_12777 (G.B.). The SGC is a registered charity (No. 1097737) that receives funds from AbbVie, Bayer, Boehringer Ingelheim, and Genome Canada through Ontario Genomics Institute Grant OGI-055, GlaxoSmithKline, Janssen, Lilly Canada, the Novartis Research Foundation, the Ontario Ministry of Economic Development and Innovation, Pfizer, Takeda, and the Wellcome Trust (Grant 092809/Z/10/Z). P.F. and S.P. are supported by a Wellcome Trust Career Development Fellowship (Grant 095751/Z/11/Z).</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i66" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i66"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i67" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i67"> Abbreviations Used</h2><tr><td class="NLM_term">ATAD2</td><td class="NLM_def"><p class="first last">ATPase family, AAA domain containing 2</p></td></tr><tr><td class="NLM_term">BAZ2B</td><td class="NLM_def"><p class="first last">bromodomain adjacent to zinc finger domain 2B</p></td></tr><tr><td class="NLM_term">BET</td><td class="NLM_def"><p class="first last">bromo and extra C-terminal domain protein</p></td></tr><tr><td class="NLM_term">BRD</td><td class="NLM_def"><p class="first last">bromodomain</p></td></tr><tr><td class="NLM_term">BRD2</td><td class="NLM_def"><p class="first last">bromodomain containing protein 2</p></td></tr><tr><td class="NLM_term">BRD3</td><td class="NLM_def"><p class="first last">bromodomain containing protein 3</p></td></tr><tr><td class="NLM_term">BRD4</td><td class="NLM_def"><p class="first last">bromodomain containing protein 4</p></td></tr><tr><td class="NLM_term">BRD7</td><td class="NLM_def"><p class="first last">bromodomain containing protein 7</p></td></tr><tr><td class="NLM_term">BRD9</td><td class="NLM_def"><p class="first last">bromodomain containing protein 9</p></td></tr><tr><td class="NLM_term">BRDT</td><td class="NLM_def"><p class="first last">bromodomain testis-specific protein</p></td></tr><tr><td class="NLM_term">BRET</td><td class="NLM_def"><p class="first last">bioluminescence resonance energy transfer</p></td></tr><tr><td class="NLM_term">BRPF1</td><td class="NLM_def"><p class="first last">bromodomain and PHD finger containing 1</p></td></tr><tr><td class="NLM_term">CREBBP</td><td class="NLM_def"><p class="first last">cAMP response element binding protein (CREB) binding protein</p></td></tr><tr><td class="NLM_term">Kac</td><td class="NLM_def"><p class="first last"><i>N</i>-ε-acetyllysine</p></td></tr><tr><td class="NLM_term">PB1</td><td class="NLM_def"><p class="first last">polybromo-containing protein</p></td></tr><tr><td class="NLM_term">PCAF</td><td class="NLM_def"><p class="first last">P300/CBP-associated factor</p></td></tr><tr><td class="NLM_term">TBAF</td><td class="NLM_def"><p class="first last">tetrabutylammonium fluoride</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i68">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_87885" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_87885" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 80 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Bernstein, B. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meissner, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lander, E. S.</span><span> </span><span class="NLM_article-title">The mammalian epigenome</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">128</span><span class="NLM_x">, </span> <span class="NLM_fpage">669</span><span class="NLM_x">–</span> <span class="NLM_lpage">681</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Fjm501893k&amp;key=10.1016%2Fj.cell.2007.01.033" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Fjm501893k&amp;key=17320505" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BD2sXis12ju7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2007&pages=669-681&author=B.+E.+Bernsteinauthor=A.+Meissnerauthor=E.+S.+Lander&title=The+mammalian+epigenome"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The mammalian epigenome</span></div><div class="casAuthors">Bernstein, Bradley E.; Meissner, Alexander; Lander, Eric S.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">669-681</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Chem. modifications to DNA and histone proteins form a complex regulatory network that modulates chromatin structure and genome function.  The epigenome refers to the complete description of these potentially heritable changes across the genome.  The compn. of the epigenome within a given cell is a function of genetic determinants, lineage, and environment.  With the sequencing of the human genome completed, investigators now seek a comprehensive view of the epigenetic changes that det. how genetic information is made manifest across an incredibly varied background of developmental stages, tissue types, and disease states.  Here we review current research efforts, with an emphasis on large-scale studies, emerging technologies, and challenges ahead.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKJfRRDsxAHbVg90H21EOLACvtfcHk0ljAf03Zbn5Hyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXis12ju7g%253D&md5=963fb487bbf473ac801e09a558ac0b0e</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2007.01.033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2007.01.033%26sid%3Dliteratum%253Aachs%26aulast%3DBernstein%26aufirst%3DB.%2BE.%26aulast%3DMeissner%26aufirst%3DA.%26aulast%3DLander%26aufirst%3DE.%2BS.%26atitle%3DThe%2520mammalian%2520epigenome%26jtitle%3DCell%26date%3D2007%26volume%3D128%26spage%3D669%26epage%3D681" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Strahl, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allis, C. D.</span><span> </span><span class="NLM_article-title">The language of covalent histone modifications</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">403</span><span class="NLM_x">, </span> <span class="NLM_fpage">41</span><span class="NLM_x">–</span> <span class="NLM_lpage">45</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Fjm501893k&amp;key=10.1038%2F47412" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Fjm501893k&amp;key=10638745" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A280%3ADC%252BD3c7gt1arsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=403&publication_year=2000&pages=41-45&author=B.+D.+Strahlauthor=C.+D.+Allis&title=The+language+of+covalent+histone+modifications"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">The language of covalent histone modifications</span></div><div class="casAuthors">Strahl B D; Allis C D</div><div class="citationInfo"><span class="NLM_cas:title">Nature</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">403</span>
        (<span class="NLM_cas:issue">6765</span>),
    <span class="NLM_cas:pages">41-5</span>
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">Histone proteins and the nucleosomes they form with DNA are the fundamental building blocks of eukaryotic chromatin.  A diverse array of post-translational modifications that often occur on tail domains of these proteins has been well documented.  Although the function of these highly conserved modifications has remained elusive, converging biochemical and genetic evidence suggests functions in several chromatin-based processes.  We propose that distinct histone modifications, on one or more tails, act sequentially or in combination to form a 'histone code' that is, read by other proteins to bring about distinct downstream events.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSyzhdlYh9sX7t7lRqEFUJVfW6udTcc2eb_-QCKH8smNLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD3c7gt1arsQ%253D%253D&md5=2eb7d093151c6122ccd70fa2adcfd94b</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2F47412&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F47412%26sid%3Dliteratum%253Aachs%26aulast%3DStrahl%26aufirst%3DB.%2BD.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26atitle%3DThe%2520language%2520of%2520covalent%2520histone%2520modifications%26jtitle%3DNature%26date%3D2000%26volume%3D403%26spage%3D41%26epage%3D45" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Choudhary, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kumar, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gnad, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nielsen, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rehman, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walther, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olsen, J. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mann, M.</span><span> </span><span class="NLM_article-title">Lysine acetylation targets protein complexes and co-regulates major cellular functions</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">325</span><span class="NLM_x">, </span> <span class="NLM_fpage">834</span><span class="NLM_x">–</span> <span class="NLM_lpage">840</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Fjm501893k&amp;key=10.1126%2Fscience.1175371" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Fjm501893k&amp;key=19608861" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BD1MXps1Ogt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=325&publication_year=2009&pages=834-840&author=C.+Choudharyauthor=C.+Kumarauthor=F.+Gnadauthor=M.+L.+Nielsenauthor=M.+Rehmanauthor=T.+C.+Waltherauthor=J.+V.+Olsenauthor=M.+Mann&title=Lysine+acetylation+targets+protein+complexes+and+co-regulates+major+cellular+functions"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Lysine Acetylation Targets Protein Complexes and Co-Regulates Major Cellular Functions</span></div><div class="casAuthors">Choudhary, Chunaram; Kumar, Chanchal; Gnad, Florian; Nielsen, Michael L.; Rehman, Michael; Walther, Tobias C.; Olsen, Jesper V.; Mann, Matthias</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">325</span>
        (<span class="NLM_cas:issue">5942</span>),
    <span class="NLM_cas:pages">834-840</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Lysine acetylation is a reversible posttranslational modification of proteins and plays a key role in regulating gene expression.  Technol. limitations have so far prevented a global anal. of lysine acetylation's cellular roles.  We used high-resoln. mass spectrometry to identify 3600 lysine acetylation sites on 1750 proteins and quantified acetylation changes in response to the deacetylase inhibitors suberoylanilide hydroxamic acid and MS-275.  Lysine acetylation preferentially targets large macromol. complexes involved in diverse cellular processes, such as chromatin remodeling, cell cycle, splicing, nuclear transport, and actin nucleation.  Acetylation impaired phosphorylation-dependent interactions of 14-3-3 and regulated the yeast cyclin-dependent kinase Cdc28.  Our data demonstrate that the regulatory scope of lysine acetylation is broad and comparable with that of other major posttranslational modifications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5eICW58ngkrVg90H21EOLACvtfcHk0lhc0hR4f-LIew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXps1Ogt70%253D&md5=8f643db5ba91c50f0b61763ff3095d2c</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1126%2Fscience.1175371&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1175371%26sid%3Dliteratum%253Aachs%26aulast%3DChoudhary%26aufirst%3DC.%26aulast%3DKumar%26aufirst%3DC.%26aulast%3DGnad%26aufirst%3DF.%26aulast%3DNielsen%26aufirst%3DM.%2BL.%26aulast%3DRehman%26aufirst%3DM.%26aulast%3DWalther%26aufirst%3DT.%2BC.%26aulast%3DOlsen%26aufirst%3DJ.%2BV.%26aulast%3DMann%26aufirst%3DM.%26atitle%3DLysine%2520acetylation%2520targets%2520protein%2520complexes%2520and%2520co-regulates%2520major%2520cellular%2520functions%26jtitle%3DScience%26date%3D2009%26volume%3D325%26spage%3D834%26epage%3D840" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span> </span><span class="NLM_article-title">The bromodomain interaction module</span> <span class="citation_source-journal">FEBS Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">586</span><span class="NLM_x">, </span> <span class="NLM_fpage">2692</span><span class="NLM_x">–</span> <span class="NLM_lpage">2704</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Fjm501893k&amp;key=10.1016%2Fj.febslet.2012.04.045" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Fjm501893k&amp;key=22710155" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BC38XntFahsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=586&publication_year=2012&pages=2692-2704&author=P.+Filippakopoulosauthor=S.+Knapp&title=The+bromodomain+interaction+module"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">The bromodomain interaction module</span></div><div class="casAuthors">Filippakopoulos, Panagis; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Letters</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">586</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">2692-2704</span>CODEN:
                <span class="NLM_cas:coden">FEBLAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-5793</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The ε-N-acetylation of protein Lys residues (Kac) is one of the most abundant post-translation modifications (PTMs) in the human proteome.  In the nucleus, the acetylation of histones has been linked to transcriptional activation of genes but the functional consequences of most acetylation events and proteins recruited to these sites remains largely unknown.  Bromodomains (BRDs) are small helical interaction modules that specifically recognize acetylation sites in proteins.  BRDs have recently emerged as interesting targets for the development of specific protein interaction inhibitors, enabling a novel exiting strategy for the development of new therapies.  Here, the author provides an overview over sequence requirements of BRDs, known substrates, and the structural mechanisms of specific Kac recognition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7LaThygz_HrVg90H21EOLACvtfcHk0lhc0hR4f-LIew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XntFahsbg%253D&md5=c02dee22c330b1d5c1ba39579360ebc5</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.febslet.2012.04.045&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.febslet.2012.04.045%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DThe%2520bromodomain%2520interaction%2520module%26jtitle%3DFEBS%2520Lett.%26date%3D2012%26volume%3D586%26spage%3D2692%26epage%3D2704" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Tamkun, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deuring, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kissinger, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pattatucci, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaufman, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kennison, J. A.</span><span> </span><span class="NLM_article-title">brahma: a regulator of <i>Drosophila</i> homeotic genes structurally related to the yeast transcriptional activator SNF2/SWI2</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1992</span><span class="NLM_x">, </span> <span class="NLM_volume">68</span><span class="NLM_x">, </span> <span class="NLM_fpage">561</span><span class="NLM_x">–</span> <span class="NLM_lpage">572</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=1992&pages=561-572&author=J.+W.+Tamkunauthor=R.+Deuringauthor=M.+P.+Scottauthor=M.+Kissingerauthor=A.+M.+Pattatucciauthor=T.+C.+Kaufmanauthor=J.+A.+Kennison&title=brahma%3A+a+regulator+of+Drosophila+homeotic+genes+structurally+related+to+the+yeast+transcriptional+activator+SNF2%2FSWI2"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DTamkun%26aufirst%3DJ.%2BW.%26aulast%3DDeuring%26aufirst%3DR.%26aulast%3DScott%26aufirst%3DM.%2BP.%26aulast%3DKissinger%26aufirst%3DM.%26aulast%3DPattatucci%26aufirst%3DA.%2BM.%26aulast%3DKaufman%26aufirst%3DT.%2BC.%26aulast%3DKennison%26aufirst%3DJ.%2BA.%26atitle%3Dbrahma%253A%2520a%2520regulator%2520of%2520Drosophila%2520homeotic%2520genes%2520structurally%2520related%2520to%2520the%2520yeast%2520transcriptional%2520activator%2520SNF2%252FSWI2%26jtitle%3DCell%26date%3D1992%26volume%3D68%26spage%3D561%26epage%3D572" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tanaka, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peng, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ren, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dent, S. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, X.</span><span> </span><span class="NLM_article-title">AF9 YEATS domain links histone acetylation to DOT1L-mediated H3K79 methylation</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">159</span><span class="NLM_x">, </span> <span class="NLM_fpage">558</span><span class="NLM_x">–</span> <span class="NLM_lpage">571</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=159&publication_year=2014&pages=558-571&author=Y.+Liauthor=H.+Wenauthor=Y.+Xiauthor=K.+Tanakaauthor=H.+Wangauthor=D.+Pengauthor=Y.+Renauthor=Q.+Jinauthor=S.+Y.+Dentauthor=W.+Liauthor=H.+Liauthor=X.+Shi&title=AF9+YEATS+domain+links+histone+acetylation+to+DOT1L-mediated+H3K79+methylation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DY.%26aulast%3DWen%26aufirst%3DH.%26aulast%3DXi%26aufirst%3DY.%26aulast%3DTanaka%26aufirst%3DK.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DPeng%26aufirst%3DD.%26aulast%3DRen%26aufirst%3DY.%26aulast%3DJin%26aufirst%3DQ.%26aulast%3DDent%26aufirst%3DS.%2BY.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DShi%26aufirst%3DX.%26atitle%3DAF9%2520YEATS%2520domain%2520links%2520histone%2520acetylation%2520to%2520DOT1L-mediated%2520H3K79%2520methylation%26jtitle%3DCell%26date%3D2014%26volume%3D159%26spage%3D558%26epage%3D571" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Yang, X. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ogryzko, V. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nishikawa, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Howard, B. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakatani, Y.</span><span> </span><span class="NLM_article-title">A p300/CBP-associated factor that competes with the adenoviral oncoprotein E1A</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">382</span><span class="NLM_x">, </span> <span class="NLM_fpage">319</span><span class="NLM_x">–</span> <span class="NLM_lpage">324</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=382&publication_year=1996&pages=319-324&author=X.+J.+Yangauthor=V.+V.+Ogryzkoauthor=J.+Nishikawaauthor=B.+H.+Howardauthor=Y.+Nakatani&title=A+p300%2FCBP-associated+factor+that+competes+with+the+adenoviral+oncoprotein+E1A"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DX.%2BJ.%26aulast%3DOgryzko%26aufirst%3DV.%2BV.%26aulast%3DNishikawa%26aufirst%3DJ.%26aulast%3DHoward%26aufirst%3DB.%2BH.%26aulast%3DNakatani%26aufirst%3DY.%26atitle%3DA%2520p300%252FCBP-associated%2520factor%2520that%2520competes%2520with%2520the%2520adenoviral%2520oncoprotein%2520E1A%26jtitle%3DNature%26date%3D1996%26volume%3D382%26spage%3D319%26epage%3D324" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Dhalluin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carlson, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aggarwal, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, M. M.</span><span> </span><span class="NLM_article-title">Structure and ligand of a histone acetyltransferase bromodomain</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">399</span><span class="NLM_x">, </span> <span class="NLM_fpage">491</span><span class="NLM_x">–</span> <span class="NLM_lpage">496</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Fjm501893k&amp;key=10.1038%2F20974" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Fjm501893k&amp;key=10365964" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADyaK1MXjvV2isLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=399&publication_year=1999&pages=491-496&author=C.+Dhalluinauthor=J.+E.+Carlsonauthor=L.+Zengauthor=C.+Heauthor=A.+K.+Aggarwalauthor=M.+M.+Zhou&title=Structure+and+ligand+of+a+histone+acetyltransferase+bromodomain"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and ligand of a histone acetyltransferase bromodomain</span></div><div class="casAuthors">Dhalluin, Christophe; Carlson, Justin E.; Zeng, Lei; He, Cheng; Aggarwal, Aneel K.; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">399</span>
        (<span class="NLM_cas:issue">6735</span>),
    <span class="NLM_cas:pages">491-496</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines</span>)
        </div><div class="casAbstract">Histone acetylation is important in chromatin remodelling and gene activation.  Nearly all known histone-acetyltransferase (HAT)-assocd. transcriptional co-activators contain bromodomains, which are ∼110-amino-acid modules found in many chromatin-assocd. proteins.  Despite the wide occurrence of these bromodomains, their three-dimensional structure and binding partners remain unknown.  Here we report the soln. structure of the bromodomain of the HAT co-activator P/CAF (p300/CBP-assocd. factor).  The structure reveals an unusual left-handed up-and-down four-helix bundle.  In addn., we show by a combination of structural and site-directed mutagenesis studies that bromodomains can interact specifically with acetylated lysine, making them the first known protein modules to do so.  The nature of the recognition of acetyl-lysine by the P/CAF bromodomain is similar to that of acetyl-CoA by histone acetyltransferase.  Thus, the bromodomain is functionally linked to the HAT activity of co-activators in the regulation of gene transcription.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsTbwcjeBG_rVg90H21EOLACvtfcHk0lhc0hR4f-LIew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXjvV2isLo%253D&md5=ff1063ba81d81b431065dd6e3751ba79</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2F20974&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F20974%26sid%3Dliteratum%253Aachs%26aulast%3DDhalluin%26aufirst%3DC.%26aulast%3DCarlson%26aufirst%3DJ.%2BE.%26aulast%3DZeng%26aufirst%3DL.%26aulast%3DHe%26aufirst%3DC.%26aulast%3DAggarwal%26aufirst%3DA.%2BK.%26aulast%3DZhou%26aufirst%3DM.%2BM.%26atitle%3DStructure%2520and%2520ligand%2520of%2520a%2520histone%2520acetyltransferase%2520bromodomain%26jtitle%3DNature%26date%3D1999%26volume%3D399%26spage%3D491%26epage%3D496" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Jacobson, R. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ladurner, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">King, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tjian, R.</span><span> </span><span class="NLM_article-title">Structure and function of a human TAFII250 double bromodomain module</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">288</span><span class="NLM_x">, </span> <span class="NLM_fpage">1422</span><span class="NLM_x">–</span> <span class="NLM_lpage">1425</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Fjm501893k&amp;key=10.1126%2Fscience.288.5470.1422" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Fjm501893k&amp;key=10827952" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BD3cXjslGrs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=288&publication_year=2000&pages=1422-1425&author=R.+H.+Jacobsonauthor=A.+G.+Ladurnerauthor=D.+S.+Kingauthor=R.+Tjian&title=Structure+and+function+of+a+human+TAFII250+double+bromodomain+module"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and function of a human TAFII250 double bromodomain module</span></div><div class="casAuthors">Jacobson, Raymond H.; Ladurner, Andreas G.; King, David S.; Tjian, Robert</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">288</span>
        (<span class="NLM_cas:issue">5470</span>),
    <span class="NLM_cas:pages">1422-1425</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">TFIID is a large multiprotein complex that initiates assembly of the transcription machinery.  It is unclear how TFIID recognizes promoters in vivo when templates are nucleosome-bound.  Here, it is shown that TAFII250, the largest subunit of TFIID, contains two tandem bromodomain modules that bind selectively to multiply acetylated histone H4 peptides.  The 2.1 angstrom crystal structure of the double bromodomain reveals two side-by-side, four-helix bundles with a highly polarized surface charge distribution.  Each bundle contains an Nε-acetyllysine binding pocket at its center, which results in a structure ideally suited for recognition of diacetylated histone H4 tails.  Thus, TFIID may be targeted to specific chromatin-bound promoters and may play a role in chromatin recognition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnYe1raEZOPLVg90H21EOLACvtfcHk0ljE3Kyrwyl1yg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXjslGrs7g%253D&md5=d0a43e3283c3a9f0eb25252e1980e580</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1126%2Fscience.288.5470.1422&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.288.5470.1422%26sid%3Dliteratum%253Aachs%26aulast%3DJacobson%26aufirst%3DR.%2BH.%26aulast%3DLadurner%26aufirst%3DA.%2BG.%26aulast%3DKing%26aufirst%3DD.%2BS.%26aulast%3DTjian%26aufirst%3DR.%26atitle%3DStructure%2520and%2520function%2520of%2520a%2520human%2520TAFII250%2520double%2520bromodomain%2520module%26jtitle%3DScience%26date%3D2000%26volume%3D288%26spage%3D1422%26epage%3D1425" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Venturini, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">You, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stadler, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Galien, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lallemand, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koken, M. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattei, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ganser, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chambon, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Losson, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de The, H.</span><span> </span><span class="NLM_article-title">TIF1gamma, a novel member of the transcriptional intermediary factor 1 family</span> <span class="citation_source-journal">Oncogene</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">1209</span><span class="NLM_x">–</span> <span class="NLM_lpage">1217</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=1999&pages=1209-1217&author=L.+Venturiniauthor=J.+Youauthor=M.+Stadlerauthor=R.+Galienauthor=V.+Lallemandauthor=M.+H.+Kokenauthor=M.+G.+Matteiauthor=A.+Ganserauthor=P.+Chambonauthor=R.+Lossonauthor=H.+de+The&title=TIF1gamma%2C+a+novel+member+of+the+transcriptional+intermediary+factor+1+family"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DVenturini%26aufirst%3DL.%26aulast%3DYou%26aufirst%3DJ.%26aulast%3DStadler%26aufirst%3DM.%26aulast%3DGalien%26aufirst%3DR.%26aulast%3DLallemand%26aufirst%3DV.%26aulast%3DKoken%26aufirst%3DM.%2BH.%26aulast%3DMattei%26aufirst%3DM.%2BG.%26aulast%3DGanser%26aufirst%3DA.%26aulast%3DChambon%26aufirst%3DP.%26aulast%3DLosson%26aufirst%3DR.%26aulast%3Dde%2BThe%26aufirst%3DH.%26atitle%3DTIF1gamma%252C%2520a%2520novel%2520member%2520of%2520the%2520transcriptional%2520intermediary%2520factor%25201%2520family%26jtitle%3DOncogene%26date%3D1999%26volume%3D18%26spage%3D1209%26epage%3D1217" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Cavellan, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Asp, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Percipalle, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Farrants, A. K.</span><span> </span><span class="NLM_article-title">The WSTF-SNF2h chromatin remodeling complex interacts with several nuclear proteins in transcription</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">281</span><span class="NLM_x">, </span> <span class="NLM_fpage">16264</span><span class="NLM_x">–</span> <span class="NLM_lpage">16271</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=281&publication_year=2006&pages=16264-16271&author=E.+Cavellanauthor=P.+Aspauthor=P.+Percipalleauthor=A.+K.+Farrants&title=The+WSTF-SNF2h+chromatin+remodeling+complex+interacts+with+several+nuclear+proteins+in+transcription"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCavellan%26aufirst%3DE.%26aulast%3DAsp%26aufirst%3DP.%26aulast%3DPercipalle%26aufirst%3DP.%26aulast%3DFarrants%26aufirst%3DA.%2BK.%26atitle%3DThe%2520WSTF-SNF2h%2520chromatin%2520remodeling%2520complex%2520interacts%2520with%2520several%2520nuclear%2520proteins%2520in%2520transcription%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2006%26volume%3D281%26spage%3D16264%26epage%3D16271" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Trotter, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Archer, T. K.</span><span> </span><span class="NLM_article-title">The BRG1 transcriptional coregulator</span> <span class="citation_source-journal">Nucl. Recept Signaling</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">e004</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2008&pages=e004&author=K.+W.+Trotterauthor=T.+K.+Archer&title=The+BRG1+transcriptional+coregulator"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DTrotter%26aufirst%3DK.%2BW.%26aulast%3DArcher%26aufirst%3DT.%2BK.%26atitle%3DThe%2520BRG1%2520transcriptional%2520coregulator%26jtitle%3DNucl.%2520Recept%2520Signaling%26date%3D2008%26volume%3D6%26spage%3De004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Xue, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Canman, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nie, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moreno, G. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, M. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Salmon, E. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, W.</span><span> </span><span class="NLM_article-title">The human SWI/SNF-B chromatin-remodeling complex is related to yeast rsc and localizes at kinetochores of mitotic chromosomes</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">97</span><span class="NLM_x">, </span> <span class="NLM_fpage">13015</span><span class="NLM_x">–</span> <span class="NLM_lpage">13020</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=97&publication_year=2000&pages=13015-13020&author=Y.+Xueauthor=J.+C.+Canmanauthor=C.+S.+Leeauthor=Z.+Nieauthor=D.+Yangauthor=G.+T.+Morenoauthor=M.+K.+Youngauthor=E.+D.+Salmonauthor=W.+Wang&title=The+human+SWI%2FSNF-B+chromatin-remodeling+complex+is+related+to+yeast+rsc+and+localizes+at+kinetochores+of+mitotic+chromosomes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DXue%26aufirst%3DY.%26aulast%3DCanman%26aufirst%3DJ.%2BC.%26aulast%3DLee%26aufirst%3DC.%2BS.%26aulast%3DNie%26aufirst%3DZ.%26aulast%3DYang%26aufirst%3DD.%26aulast%3DMoreno%26aufirst%3DG.%2BT.%26aulast%3DYoung%26aufirst%3DM.%2BK.%26aulast%3DSalmon%26aufirst%3DE.%2BD.%26aulast%3DWang%26aufirst%3DW.%26atitle%3DThe%2520human%2520SWI%252FSNF-B%2520chromatin-remodeling%2520complex%2520is%2520related%2520to%2520yeast%2520rsc%2520and%2520localizes%2520at%2520kinetochores%2520of%2520mitotic%2520chromosomes%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2000%26volume%3D97%26spage%3D13015%26epage%3D13020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Wu, S. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chiang, C. M.</span><span> </span><span class="NLM_article-title">The double bromodomain-containing chromatin adaptor Brd4 and transcriptional regulation</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">282</span><span class="NLM_x">, </span> <span class="NLM_fpage">13141</span><span class="NLM_x">–</span> <span class="NLM_lpage">13145</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Fjm501893k&amp;key=10.1074%2Fjbc.R700001200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Fjm501893k&amp;key=17329240" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BD2sXks1Ghtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=282&publication_year=2007&pages=13141-13145&author=S.+Y.+Wuauthor=C.+M.+Chiang&title=The+double+bromodomain-containing+chromatin+adaptor+Brd4+and+transcriptional+regulation"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">The Double Bromodomain-containing Chromatin Adaptor Brd4 and Transcriptional Regulation</span></div><div class="casAuthors">Wu, Shwu-Yuan; Chiang, Cheng-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">282</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">13141-13145</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">A review.  Brd4 is a double bromodomain-contg. protein that binds preferentially to acetylated chromatin.  It belongs to the BET (bromodomains and extraterminal) family that includes mammalian Brd2, Brd3, Brd4, Brdt, Drosophila Fsh, yeast Bdf1, Bdf2, and corresponding homologues in other species.  Brd4 is essential for cellular growth and has been implicated in cell cycle control, DNA replication, and gene rearrangement found in t(15;19)-assocd. carcinomas.  Recently, Brd4 has been found in several transcription complexes, including the general cofactor Mediator and the P-TEFb elongation factor, and is capable of stimulating HIV-1 transcription in a Tat-independent manner.  In addn., Brd4 is used as a cellular adaptor by some animal and human papillomaviruses (HPV) for anchoring viral genomes to mitotic chromosomes.  This tethering, mediated by Brd4 interaction with virus-encoded E2 protein, facilitates viral genome segregation during mitosis.  Interestingly, Brd4 is also identified in a transcriptional silencing complex assembled by HPV E2 and turns out to be the long sought cellular corepressor that inhibits the expression of HPV-encoded E6 and E7 oncoproteins that antagonize p53 and pRB tumor suppressor activity, resp.  The dual role of Brd4 in gene activation and repression illustrates how a dynamic chromatin-binding adaptor is able to recruit distinct transcriptional regulators to modulate promoter activity through cell cycle progression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohccDdZC9iKLVg90H21EOLACvtfcHk0ljE3Kyrwyl1yg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXks1Ghtbo%253D&md5=b025a5222b510a7009eca4289370b1d4</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1074%2Fjbc.R700001200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.R700001200%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DS.%2BY.%26aulast%3DChiang%26aufirst%3DC.%2BM.%26atitle%3DThe%2520double%2520bromodomain-containing%2520chromatin%2520adaptor%2520Brd4%2520and%2520transcriptional%2520regulation%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2007%26volume%3D282%26spage%3D13141%26epage%3D13145" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Picaud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mangos, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keates, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lambert, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Felletar, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Volkmer, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muller, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pawson, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gingras, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span> </span><span class="NLM_article-title">Histone recognition and large-scale structural analysis of the human bromodomain family</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">149</span><span class="NLM_x">, </span> <span class="NLM_fpage">214</span><span class="NLM_x">–</span> <span class="NLM_lpage">231</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Fjm501893k&amp;key=10.1016%2Fj.cell.2012.02.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Fjm501893k&amp;key=22464331" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=149&publication_year=2012&pages=214-231&author=P.+Filippakopoulosauthor=S.+Picaudauthor=M.+Mangosauthor=T.+Keatesauthor=J.+P.+Lambertauthor=D.+Barsyte-Lovejoyauthor=I.+Felletarauthor=R.+Volkmerauthor=S.+Mullerauthor=T.+Pawsonauthor=A.+C.+Gingrasauthor=C.+H.+Arrowsmithauthor=S.+Knapp&title=Histone+recognition+and+large-scale+structural+analysis+of+the+human+bromodomain+family"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Histone Recognition and Large-Scale Structural Analysis of the Human Bromodomain Family</span></div><div class="casAuthors">Filippakopoulos, Panagis; Picaud, Sarah; Mangos, Maria; Keates, Tracy; Lambert, Jean-Philippe; Barsyte-Lovejoy, Dalia; Felletar, Ildiko; Volkmer, Rudolf; Muller, Susanne; Pawson, Tony; Gingras, Anne-Claude; Arrowsmith, Cheryl H.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">149</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">214-231</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Bromodomains (BRDs) are protein interaction modules that specifically recognize ε-N-lysine acetylation motifs, a key event in the reading process of epigenetic marks.  The 61 BRDs in the human genome cluster into eight families based on structure/sequence similarity.  Here, we present 29 high-resoln. crystal structures, covering all BRD families.  These proteins are: ASH1L, ATAD2, BAZ2B, BPTF, BRD1, BRD3(1), BRD3(2), BRD4(1), BRD4(2), BRD9, BRDT(1), CECR2, EP300, CREBBP, GCN5L2, KIAA1240, PB1(1), PB1(2), PB1(3), PB1(4), PB1(5), PB1(6), PCAF, PHIP(2), TAF1(2), WDR9(2), BRD4(1).  Comprehensive crossfamily structural anal. identifies conserved and family-specific structural features that are necessary for specific acetylation-dependent substrate recognition.  Screening of more than 30 representative BRDs against systematic histone-peptide arrays identifies new BRD substrates and reveals a strong influence of flanking posttranslational modifications, such as acetylation and phosphorylation, suggesting that BRDs recognize combinations of marks rather than singly acetylated sequences.  We further uncovered a structural mechanism for the simultaneous binding and recognition of diverse diacetyl-contg. peptides by BRD4.  These data provide a foundation for structure-based drug design of specific inhibitors for this emerging target family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoK2FSIcPoXZLVg90H21EOLACvtfcHk0licgzobQj723Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D&md5=05583a01a1cd4c3b59fc59c00e4c4777</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2012.02.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2012.02.013%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DMangos%26aufirst%3DM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DLambert%26aufirst%3DJ.%2BP.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DVolkmer%26aufirst%3DR.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DPawson%26aufirst%3DT.%26aulast%3DGingras%26aufirst%3DA.%2BC.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DHistone%2520recognition%2520and%2520large-scale%2520structural%2520analysis%2520of%2520the%2520human%2520bromodomain%2520family%26jtitle%3DCell%26date%3D2012%26volume%3D149%26spage%3D214%26epage%3D231" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Belkina, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Denis, G. V.</span><span> </span><span class="NLM_article-title">BET domain co-regulators in obesity, inflammation and cancer</span> <span class="citation_source-journal">Nat. Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">465</span><span class="NLM_x">–</span> <span class="NLM_lpage">477</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Fjm501893k&amp;key=10.1038%2Fnrc3256" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Fjm501893k&amp;key=22722403" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BC38XovFyks7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2012&pages=465-477&author=A.+C.+Belkinaauthor=G.+V.+Denis&title=BET+domain+co-regulators+in+obesity%2C+inflammation+and+cancer"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">BET domain co-regulators in obesity, inflammation and cancer</span></div><div class="casAuthors">Belkina, Anna C.; Denis, Gerald V.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">465-477</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The bromodomain is a highly conserved motif of 110 amino acids that is bundled into four anti-parallel α-helixes and found in proteins that interact with chromatin, such as transcription factors, histone acetylases and nucleosome remodelling complexes.  Bromodomain proteins are chromatin 'readers'; they recruit chromatin-regulating enzymes, including 'writers' and 'erasers' of histone modification, to target promoters and to regulate gene expression.  Conventional wisdom held that complexes involved in chromatin dynamics are not 'druggable' targets.  However, small mols. that inhibit bromodomain and extraterminal (BET) proteins have been described.  We examine these developments and discuss the implications for small mol. epigenetic targeting of chromatin networks in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1x-bD67oukrVg90H21EOLACvtfcHk0licgzobQj723Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XovFyks7s%253D&md5=d06d88ef0d5d4d71115f40a2f901f53d</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fnrc3256&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc3256%26sid%3Dliteratum%253Aachs%26aulast%3DBelkina%26aufirst%3DA.%2BC.%26aulast%3DDenis%26aufirst%3DG.%2BV.%26atitle%3DBET%2520domain%2520co-regulators%2520in%2520obesity%252C%2520inflammation%2520and%2520cancer%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2012%26volume%3D12%26spage%3D465%26epage%3D477" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Lamonica, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kadauke, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campbell, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gamsjaeger, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Billin, A. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hardison, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mackay, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blobel, G. A.</span><span> </span><span class="NLM_article-title">Bromodomain protein Brd3 associates with acetylated GATA1 to promote its chromatin occupancy at erythroid target genes</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">108</span><span class="NLM_x">, </span> <span class="NLM_fpage">E159</span><span class="NLM_x">–</span> <span class="NLM_lpage">168</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Fjm501893k&amp;key=10.1073%2Fpnas.1102140108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Fjm501893k&amp;key=21536911" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A280%3ADC%252BC3MrmtlGgtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2011&pages=E159-168&author=J.+M.+Lamonicaauthor=W.+Dengauthor=S.+Kadaukeauthor=A.+E.+Campbellauthor=R.+Gamsjaegerauthor=H.+Wangauthor=Y.+Chengauthor=A.+N.+Billinauthor=R.+C.+Hardisonauthor=J.+P.+Mackayauthor=G.+A.+Blobel&title=Bromodomain+protein+Brd3+associates+with+acetylated+GATA1+to+promote+its+chromatin+occupancy+at+erythroid+target+genes"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomain protein Brd3 associates with acetylated GATA1 to promote its chromatin occupancy at erythroid target genes</span></div><div class="casAuthors">Lamonica Janine M; Deng Wulan; Kadauke Stephan; Campbell Amy E; Gamsjaeger Roland; Wang Hongxin; Cheng Yong; Billin Andrew N; Hardison Ross C; Mackay Joel P; Blobel Gerd A</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">108</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">E159-68</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Acetylation of histones triggers association with bromodomain-containing proteins that regulate diverse chromatin-related processes.  Although acetylation of transcription factors has been appreciated for some time, the mechanistic consequences are less well understood.  The hematopoietic transcription factor GATA1 is acetylated at conserved lysines that are required for its stable association with chromatin.  We show that the BET family protein Brd3 binds via its first bromodomain (BD1) to GATA1 in an acetylation-dependent manner in vitro and in vivo.  Mutation of a single residue in BD1 that is involved in acetyl-lysine binding abrogated recruitment of Brd3 by GATA1, demonstrating that acetylation of GATA1 is essential for Brd3 association with chromatin.  Notably, Brd3 is recruited by GATA1 to both active and repressed target genes in a fashion seemingly independent of histone acetylation.  Anti-Brd3 ChIP followed by massively parallel sequencing in GATA1-deficient erythroid precursor cells and those that are GATA1 replete revealed that GATA1 is a major determinant of Brd3 recruitment to genomic targets within chromatin.  A pharmacologic compound that occupies the acetyl-lysine binding pockets of Brd3 bromodomains disrupts the Brd3-GATA1 interaction, diminishes the chromatin occupancy of both proteins, and inhibits erythroid maturation.  Together these findings provide a mechanism for GATA1 acetylation and suggest that Brd3 "reads" acetyl marks on nuclear factors to promote their stable association with chromatin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQeZ8VSdvvG7p6KlWarZTAzfW6udTcc2eYUqJgnlyk_pLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3MrmtlGgtQ%253D%253D&md5=b81056e4aee1351dce3899fb8d4e86e8</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1102140108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1102140108%26sid%3Dliteratum%253Aachs%26aulast%3DLamonica%26aufirst%3DJ.%2BM.%26aulast%3DDeng%26aufirst%3DW.%26aulast%3DKadauke%26aufirst%3DS.%26aulast%3DCampbell%26aufirst%3DA.%2BE.%26aulast%3DGamsjaeger%26aufirst%3DR.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DCheng%26aufirst%3DY.%26aulast%3DBillin%26aufirst%3DA.%2BN.%26aulast%3DHardison%26aufirst%3DR.%2BC.%26aulast%3DMackay%26aufirst%3DJ.%2BP.%26aulast%3DBlobel%26aufirst%3DG.%2BA.%26atitle%3DBromodomain%2520protein%2520Brd3%2520associates%2520with%2520acetylated%2520GATA1%2520to%2520promote%2520its%2520chromatin%2520occupancy%2520at%2520erythroid%2520target%2520genes%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2011%26volume%3D108%26spage%3DE159%26epage%3D168" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Shi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kang, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tao, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rusinova, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, Y. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Evers, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, B. P.</span><span> </span><span class="NLM_article-title">Disrupting the interaction of BRD4 with diacetylated Twist suppresses tumorigenesis in basal-like breast cancer</span> <span class="citation_source-journal">Cancer Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">210</span><span class="NLM_x">–</span> <span class="NLM_lpage">225</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Fjm501893k&amp;key=10.1016%2Fj.ccr.2014.01.028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Fjm501893k&amp;key=24525235" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitlCisrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2014&pages=210-225&author=J.+Shiauthor=Y.+Wangauthor=L.+Zengauthor=Y.+Wuauthor=J.+Dengauthor=Q.+Zhangauthor=Y.+Linauthor=J.+Liauthor=T.+Kangauthor=M.+Taoauthor=E.+Rusinovaauthor=G.+Zhangauthor=C.+Wangauthor=H.+Zhuauthor=J.+Yaoauthor=Y.+X.+Zengauthor=B.+M.+Eversauthor=M.+M.+Zhouauthor=B.+P.+Zhou&title=Disrupting+the+interaction+of+BRD4+with+diacetylated+Twist+suppresses+tumorigenesis+in+basal-like+breast+cancer"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Disrupting the Interaction of BRD4 with Diacetylated Twist Suppresses Tumorigenesis in Basal-like Breast Cancer</span></div><div class="casAuthors">Shi, Jian; Wang, Yifan; Zeng, Lei; Wu, Yadi; Deng, Jiong; Zhang, Qiang; Lin, Yiwei; Li, Junlin; Kang, Tiebang; Tao, Min; Rusinova, Elena; Zhang, Guangtao; Wang, Chi; Zhu, Haining; Yao, Jun; Zeng, Yi-Xin; Evers, B. Mark; Zhou, Ming-Ming; Zhou, Binhua P.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">210-225</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Twist is a key transcription activator of epithelial-mesenchymal transition (EMT).  It remains unclear how Twist induces gene expression.  Here we report a mechanism by which Twist recruits BRD4 to direct WNT5A expression in basal-like breast cancer (BLBC).  Twist contains a "histone H4-mimic" GK-X-GK motif that is diacetylated by Tip60.  The diacetylated Twist binds the second bromodomain of BRD4, whose first bromodomain interacts with acetylated H4, thereby constructing an activated Twist/BRD4/P-TEFb/RNA-Pol II complex at the WNT5A promoter and enhancer.  Pharmacol. inhibition of the Twist-BRD4 assocn. reduced WNT5A expression and suppressed invasion, cancer stem cell (CSC)-like properties, and tumorigenicity of BLBC cells.  Our study indicates that the interaction with BRD4 is crit. for the oncogenic function of Twist in BLBC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovIzriI8A6ZrVg90H21EOLACvtfcHk0lj_ppeLs2d8Gg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitlCisrs%253D&md5=d207cbc1abb187a53d807452b4f10d8f</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2014.01.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2014.01.028%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DZeng%26aufirst%3DL.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DDeng%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DKang%26aufirst%3DT.%26aulast%3DTao%26aufirst%3DM.%26aulast%3DRusinova%26aufirst%3DE.%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DZhu%26aufirst%3DH.%26aulast%3DYao%26aufirst%3DJ.%26aulast%3DZeng%26aufirst%3DY.%2BX.%26aulast%3DEvers%26aufirst%3DB.%2BM.%26aulast%3DZhou%26aufirst%3DM.%2BM.%26aulast%3DZhou%26aufirst%3DB.%2BP.%26atitle%3DDisrupting%2520the%2520interaction%2520of%2520BRD4%2520with%2520diacetylated%2520Twist%2520suppresses%2520tumorigenesis%2520in%2520basal-like%2520breast%2520cancer%26jtitle%3DCancer%2520Cell%26date%3D2014%26volume%3D25%26spage%3D210%26epage%3D225" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Huang, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, X. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ozato, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L. F.</span><span> </span><span class="NLM_article-title">Brd4 coactivates transcriptional activation of NF-kappaB via specific binding to acetylated RelA</span> <span class="citation_source-journal">Mol. Cell. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">1375</span><span class="NLM_x">–</span> <span class="NLM_lpage">1387</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Fjm501893k&amp;key=10.1128%2FMCB.01365-08" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Fjm501893k&amp;key=19103749" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BD1MXitlylsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2009&pages=1375-1387&author=B.+Huangauthor=X.+D.+Yangauthor=M.+M.+Zhouauthor=K.+Ozatoauthor=L.+F.+Chen&title=Brd4+coactivates+transcriptional+activation+of+NF-kappaB+via+specific+binding+to+acetylated+RelA"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Brd4 coactivates transcriptional activation of NF-κB via specific binding to acetylated RelA</span></div><div class="casAuthors">Huang, Bo; Yang, Xiao-Dong; Zhou, Ming-Ming; Ozato, Keiko; Chen, Lin-Feng</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1375-1387</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Acetylation of the RelA subunit of NF-κB, esp. at lysine-310, is crit. for the transcriptional activation of NF-κB and the expression of inflammatory genes.  In this study, we demonstrate that bromodomains of Brd4 bind to acetylated lysine-310.  Brd4 enhances transcriptional activation of NF-κB and the expression of a subset of NF-κB-responsive inflammatory genes in an acetylated lysine-310-dependent manner.  Bromodomains of Brd4 and acetylated lysine-310 of RelA are both required for the mutual interaction and coactivation function of Brd4.  Finally, we demonstrate that Brd4 further recruits CDK9 to phosphorylate C-terminal domain of RNA polymerase II and facilitate the transcription of NF-κB-dependent inflammatory genes.  Our results identify Brd4 as a novel coactivator of NF-κB through specifically binding to acetylated lysine-310 of RelA.  In addn., these studies reveal a mechanism by which acetylated RelA stimulates the transcriptional activity of NF-κB and the NF-κB-dependent inflammatory response.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBnS4p5wzT2LVg90H21EOLACvtfcHk0lj_ppeLs2d8Gg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXitlylsrc%253D&md5=ae4e16c84d17c5e0e268ff80703a4608</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1128%2FMCB.01365-08&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.01365-08%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DB.%26aulast%3DYang%26aufirst%3DX.%2BD.%26aulast%3DZhou%26aufirst%3DM.%2BM.%26aulast%3DOzato%26aufirst%3DK.%26aulast%3DChen%26aufirst%3DL.%2BF.%26atitle%3DBrd4%2520coactivates%2520transcriptional%2520activation%2520of%2520NF-kappaB%2520via%2520specific%2520binding%2520to%2520acetylated%2520RelA%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2009%26volume%3D29%26spage%3D1375%26epage%3D1387" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Schroder, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cho, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaehlcke, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mak, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lau, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bisgrove, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schnolzer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verdin, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ott, M.</span><span> </span><span class="NLM_article-title">Two-pronged binding with bromodomain-containing protein 4 liberates positive transcription elongation factor b from inactive ribonucleoprotein complexes</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">287</span><span class="NLM_x">, </span> <span class="NLM_fpage">1090</span><span class="NLM_x">–</span> <span class="NLM_lpage">1099</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=287&publication_year=2012&pages=1090-1099&author=S.+Schroderauthor=S.+Choauthor=L.+Zengauthor=Q.+Zhangauthor=K.+Kaehlckeauthor=L.+Makauthor=J.+Lauauthor=D.+Bisgroveauthor=M.+Schnolzerauthor=E.+Verdinauthor=M.+M.+Zhouauthor=M.+Ott&title=Two-pronged+binding+with+bromodomain-containing+protein+4+liberates+positive+transcription+elongation+factor+b+from+inactive+ribonucleoprotein+complexes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSchroder%26aufirst%3DS.%26aulast%3DCho%26aufirst%3DS.%26aulast%3DZeng%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DKaehlcke%26aufirst%3DK.%26aulast%3DMak%26aufirst%3DL.%26aulast%3DLau%26aufirst%3DJ.%26aulast%3DBisgrove%26aufirst%3DD.%26aulast%3DSchnolzer%26aufirst%3DM.%26aulast%3DVerdin%26aufirst%3DE.%26aulast%3DZhou%26aufirst%3DM.%2BM.%26aulast%3DOtt%26aufirst%3DM.%26atitle%3DTwo-pronged%2520binding%2520with%2520bromodomain-containing%2520protein%25204%2520liberates%2520positive%2520transcription%2520elongation%2520factor%2520b%2520from%2520inactive%2520ribonucleoprotein%2520complexes%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2012%26volume%3D287%26spage%3D1090%26epage%3D1099" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Rahman, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sowa, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ottinger, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harper, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Howley, P. M.</span><span> </span><span class="NLM_article-title">The Brd4 extraterminal domain confers transcription activation independent of pTEFb by recruiting multiple proteins, including NSD3</span> <span class="citation_source-journal">Mol. Cell. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">2641</span><span class="NLM_x">–</span> <span class="NLM_lpage">2652</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Fjm501893k&amp;key=10.1128%2FMCB.01341-10" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Fjm501893k&amp;key=21555454" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BC3MXosFKjs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2011&pages=2641-2652&author=S.+Rahmanauthor=M.+E.+Sowaauthor=M.+Ottingerauthor=J.+A.+Smithauthor=Y.+Shiauthor=J.+W.+Harperauthor=P.+M.+Howley&title=The+Brd4+extraterminal+domain+confers+transcription+activation+independent+of+pTEFb+by+recruiting+multiple+proteins%2C+including+NSD3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">The Brd4 extraterminal domain confers transcription activation independent of pTEFb by recruiting multiple proteins, including NSD3</span></div><div class="casAuthors">Rahman, Shaila; Sowa, Mathew E.; Ottinger, Matthias; Smith, Jennifer A.; Shi, Yang; Harper, J. Wade; Howley, Peter M.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">2641-2652</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Bromodomain protein 4 (Brd4) plays crit. roles in development, cancer progression, and virus-host pathogenesis.  To gain mechanistic insight into the various biol. functions of Brd4, we performed a proteomic anal. to identify and characterize Brd4-assocd. cellular proteins.  We found that the extraterminal (ET) domain, whose function has to date not been detd., interacts with NSD3, JMJD6, CHD4, GLTSCR1, and ATAD5.  These ET-domain interactions were also conserved for Brd2 and Brd3, the other human BET proteins tested.  We demonstrated that GLTSCR1, NSD3, and JMJD6 impart a pTEFb-independent transcriptional activation function on Brd4.  NSD3 as well as JMJD6 is recruited to regulated genes in a Brd4-dependent manner.  Moreover, we found that depletion of Brd4 or NSD3 reduces H3K36 methylation, demonstrating that the Brd4/NSD3 complex regulates this specific histone modification.  Our results indicate that the Brd4 ET domain through the recruitment of the specific effectors regulates transcriptional activity.  In particular, we show that one of these effectors, NSD3, regulates transcription by modifying the chromatin microenvironment at Brd4 target genes.  Our study thus identifies the ET domain as a second important transcriptional regulatory domain for Brd4 in addn. to the carboxyl-terminal domain (CTD) that interacts with pTEFb.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrlypbenIW4s7Vg90H21EOLACvtfcHk0liIhce0sHtXvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXosFKjs7s%253D&md5=921d55b32a568783ffffdb20cef2f27c</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1128%2FMCB.01341-10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.01341-10%26sid%3Dliteratum%253Aachs%26aulast%3DRahman%26aufirst%3DS.%26aulast%3DSowa%26aufirst%3DM.%2BE.%26aulast%3DOttinger%26aufirst%3DM.%26aulast%3DSmith%26aufirst%3DJ.%2BA.%26aulast%3DShi%26aufirst%3DY.%26aulast%3DHarper%26aufirst%3DJ.%2BW.%26aulast%3DHowley%26aufirst%3DP.%2BM.%26atitle%3DThe%2520Brd4%2520extraterminal%2520domain%2520confers%2520transcription%2520activation%2520independent%2520of%2520pTEFb%2520by%2520recruiting%2520multiple%2520proteins%252C%2520including%2520NSD3%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2011%26volume%3D31%26spage%3D2641%26epage%3D2652" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Bres, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoh, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, K. A.</span><span> </span><span class="NLM_article-title">The multi-tasking P-TEFb complex</span> <span class="citation_source-journal">Curr. Opin. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">334</span><span class="NLM_x">–</span> <span class="NLM_lpage">340</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Fjm501893k&amp;key=10.1016%2Fj.ceb.2008.04.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Fjm501893k&amp;key=18513937" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BD1cXmvV2lsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2008&pages=334-340&author=V.+Bresauthor=S.+M.+Yohauthor=K.+A.+Jones&title=The+multi-tasking+P-TEFb+complex"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">The multi-tasking P-TEFb complex</span></div><div class="casAuthors">Bres, Vanessa; Yoh, Sunnie M.; Jones, Katherine A.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Cell Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">334-340</span>CODEN:
                <span class="NLM_cas:coden">COCBE3</span>;
        ISSN:<span class="NLM_cas:issn">0955-0674</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  P-TEFb (CycT1:Cdk9), the metazoan RNA polymerase II Ser2 C-terminal domain (CTD) kinase, regulates transcription elongation at many genes and integrates mRNA synthesis with histone modification, pre-mRNA processing, and mRNA export.  Recruitment of P-TEFb to target genes requires deubiquitination of H2Bub, phosphorylation of H3S10, and the bromodomain protein, Brd4.  Brd4 activates growth-related genes in the G1 phase of the cell cycle and can also tether P-TEFb to mitotic chromosomes, possibly to mark sites of active transcription throughout cell division.  P-TEFb cooperates with c-Myc during transactivation and cell transformation, and also requires SKIP (c-Ski-interacting protein), an mRNA elongation and splicing factor.  Some functions of the P-TEFb/Ser2P CTD are executed by the Spt6 transcription elongation factor, which binds directly to the phosphorylated CTD and recruits the Iws1 (interacts with Spt6') protein.  Iws1, in turn, interacts with the REF1/Aly nuclear export adaptor and stimulates the kinetics of mRNA export.  Given the prominent role of Spt6 in regulating chromatin structure, the CTD-bound Spt6:Iws1 complex may also control histone modifications during elongation.  Following transcription, P-TEFb accompanies the mature mRNA to the cytoplasm to promote translation elongation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAd7sU5Jw85rVg90H21EOLACvtfcHk0liIhce0sHtXvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXmvV2lsL8%253D&md5=9e6ecbe1323fd5abc0f13e593c1d9110</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.ceb.2008.04.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ceb.2008.04.008%26sid%3Dliteratum%253Aachs%26aulast%3DBres%26aufirst%3DV.%26aulast%3DYoh%26aufirst%3DS.%2BM.%26aulast%3DJones%26aufirst%3DK.%2BA.%26atitle%3DThe%2520multi-tasking%2520P-TEFb%2520complex%26jtitle%3DCurr.%2520Opin.%2520Cell%2520Biol.%26date%3D2008%26volume%3D20%26spage%3D334%26epage%3D340" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Picaud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, W. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morse, E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keates, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hickman, T. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Felletar, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Philpott, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Munro, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McKeown, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Christie, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">West, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cameron, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwartz, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heightman, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">La Thangue, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">French, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wiest, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kung, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bradner, J. E.</span><span> </span><span class="NLM_article-title">Selective inhibition of BET bromodomains</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">468</span><span class="NLM_x">, </span> <span class="NLM_fpage">1067</span><span class="NLM_x">–</span> <span class="NLM_lpage">1073</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Fjm501893k&amp;key=10.1038%2Fnature09504" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Fjm501893k&amp;key=20871596" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=1067-1073&author=P.+Filippakopoulosauthor=J.+Qiauthor=S.+Picaudauthor=Y.+Shenauthor=W.+B.+Smithauthor=O.+Fedorovauthor=E.+M.+Morseauthor=T.+Keatesauthor=T.+T.+Hickmanauthor=I.+Felletarauthor=M.+Philpottauthor=S.+Munroauthor=M.+R.+McKeownauthor=Y.+Wangauthor=A.+L.+Christieauthor=N.+Westauthor=M.+J.+Cameronauthor=B.+Schwartzauthor=T.+D.+Heightmanauthor=N.+La+Thangueauthor=C.+A.+Frenchauthor=O.+Wiestauthor=A.+L.+Kungauthor=S.+Knappauthor=J.+E.+Bradner&title=Selective+inhibition+of+BET+bromodomains"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibition of BET bromodomains</span></div><div class="casAuthors">Filippakopoulos, Panagis; Qi, Jun; Picaud, Sarah; Shen, Yao; Smith, William B.; Fedorov, Oleg; Morse, Elizabeth M.; Keates, Tracey; Hickman, Tyler T.; Felletar, Ildiko; Philpott, Martin; Munro, Shongah; McKeown, Michael R.; Wang, Yuchuan; Christie, Amanda L.; West, Nathan; Cameron, Michael J.; Schwartz, Brian; Heightman, Tom D.; La Thangue, Nicholas; French, Christopher; Wiest, Olaf; Kung, Andrew L.; Knapp, Stefan; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7327</span>),
    <span class="NLM_cas:pages">1067-1073</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Epigenetic proteins are intently pursued targets in ligand discovery.  So far, successful efforts have been limited to chromatin modifying enzymes, or so-called epigenetic 'writers' and 'erasers'.  Potent inhibitors of histone binding modules have not yet been described.  Here the authors report a cell-permeable small mol. (I,JQ1) that binds competitively to acetyl-lysine recognition motifs, or bromodomains.  High potency and specificity towards a subset of human bromodomains is explained by co-crystal structures with bromodomain and extra-terminal (BET) family member BRD4, revealing excellent shape complementarity with the acetyl-lysine binding cavity.  Recurrent translocation of BRD4 is obsd. in a genetically-defined, incurable subtype of human squamous carcinoma.  Competitive binding by JQ1 displaces the BRD4 fusion oncoprotein from chromatin, prompting squamous differentiation and specific antiproliferative effects in BRD4-dependent cell lines and patient-derived xenograft models.  These data establish proof-of-concept for targeting protein-protein interactions of epigenetic 'readers', and provide a versatile chem. scaffold for the development of chem. probes more broadly throughout the bromodomain family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2ZbtmZM8m_LVg90H21EOLACvtfcHk0liIhce0sHtXvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D&md5=bd9cadac86124d2c6c3cd5b2a92d68c2</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fnature09504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09504%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DSmith%26aufirst%3DW.%2BB.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMorse%26aufirst%3DE.%2BM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DHickman%26aufirst%3DT.%2BT.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DMunro%26aufirst%3DS.%26aulast%3DMcKeown%26aufirst%3DM.%2BR.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DChristie%26aufirst%3DA.%2BL.%26aulast%3DWest%26aufirst%3DN.%26aulast%3DCameron%26aufirst%3DM.%2BJ.%26aulast%3DSchwartz%26aufirst%3DB.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DLa%2BThangue%26aufirst%3DN.%26aulast%3DFrench%26aufirst%3DC.%2BA.%26aulast%3DWiest%26aufirst%3DO.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DSelective%2520inhibition%2520of%2520BET%2520bromodomains%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D1067%26epage%3D1073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Nicodeme, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeffrey, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schaefer, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beinke, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dewell, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chandwani, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marazzi, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coste, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">White, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kirilovsky, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rice, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lora, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prinjha, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tarakhovsky, A.</span><span> </span><span class="NLM_article-title">Suppression of inflammation by a synthetic histone mimic</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">468</span><span class="NLM_x">, </span> <span class="NLM_fpage">1119</span><span class="NLM_x">–</span> <span class="NLM_lpage">1123</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Fjm501893k&amp;key=10.1038%2Fnature09589" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Fjm501893k&amp;key=21068722" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtl2rtb%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=1119-1123&author=E.+Nicodemeauthor=K.+L.+Jeffreyauthor=U.+Schaeferauthor=S.+Beinkeauthor=S.+Dewellauthor=C.+W.+Chungauthor=R.+Chandwaniauthor=I.+Marazziauthor=P.+Wilsonauthor=H.+Costeauthor=J.+Whiteauthor=J.+Kirilovskyauthor=C.+M.+Riceauthor=J.+M.+Loraauthor=R.+K.+Prinjhaauthor=K.+Leeauthor=A.+Tarakhovsky&title=Suppression+of+inflammation+by+a+synthetic+histone+mimic"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Suppression of inflammation by a synthetic histone mimic</span></div><div class="casAuthors">Nicodeme, Edwige; Jeffrey, Kate L.; Schaefer, Uwe; Beinke, Soren; Dewell, Scott; Chung, Chun-wa; Chandwani, Rohit; Marazzi, Ivan; Wilson, Paul; Coste, Herve; White, Julia; Kirilovsky, Jorge; Rice, Charles M.; Lora, Jose M.; Prinjha, Rab K.; Lee, Kevin; Tarakhovsky, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7327</span>),
    <span class="NLM_cas:pages">1119-1123</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Interaction of pathogens with cells of the immune system results in activation of inflammatory gene expression.  This response, although vital for immune defense, is frequently deleterious to the host due to the exaggerated prodn. of inflammatory proteins.  The scope of inflammatory responses reflects the activation state of signaling proteins upstream of inflammatory genes as well as signal-induced assembly of nuclear chromatin complexes that support mRNA expression.  Recognition of post-translationally modified histones by nuclear proteins that initiate mRNA transcription and support mRNA elongation is a crit. step in the regulation of gene expression.  Here we present a novel pharmacol. approach that targets inflammatory gene expression by interfering with the recognition of acetylated histones by the bromodomain and extra terminal domain (BET) family of proteins.  We describe a synthetic compd. (I-BET) that by mimicking' acetylated histones disrupts chromatin complexes responsible for the expression of key inflammatory genes in activated macrophages, and confers protection against lipopolysaccharide-induced endotoxic shock and bacteria-induced sepsis.  Our findings suggest that synthetic compds. specifically targeting proteins that recognize post-translationally modified histones can serve as a new generation of immunomodulatory drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxc3_PNaiuKLVg90H21EOLACvtfcHk0livcN66xzls0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtl2rtb%252FP&md5=0dfccd9d01b87859512bf591e1c5a93c</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fnature09589&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09589%26sid%3Dliteratum%253Aachs%26aulast%3DNicodeme%26aufirst%3DE.%26aulast%3DJeffrey%26aufirst%3DK.%2BL.%26aulast%3DSchaefer%26aufirst%3DU.%26aulast%3DBeinke%26aufirst%3DS.%26aulast%3DDewell%26aufirst%3DS.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DChandwani%26aufirst%3DR.%26aulast%3DMarazzi%26aufirst%3DI.%26aulast%3DWilson%26aufirst%3DP.%26aulast%3DCoste%26aufirst%3DH.%26aulast%3DWhite%26aufirst%3DJ.%26aulast%3DKirilovsky%26aufirst%3DJ.%26aulast%3DRice%26aufirst%3DC.%2BM.%26aulast%3DLora%26aufirst%3DJ.%2BM.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DTarakhovsky%26aufirst%3DA.%26atitle%3DSuppression%2520of%2520inflammation%2520by%2520a%2520synthetic%2520histone%2520mimic%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D1119%26epage%3D1123" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span> </span><span class="NLM_article-title">Targeting bromodomains: epigenetic readers of lysine acetylation</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">337</span><span class="NLM_x">–</span> <span class="NLM_lpage">356</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Fjm501893k&amp;key=10.1038%2Fnrd4286" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Fjm501893k&amp;key=24751816" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=337-356&author=P.+Filippakopoulosauthor=S.+Knapp&title=Targeting+bromodomains%3A+epigenetic+readers+of+lysine+acetylation"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting bromodomains: epigenetic readers of lysine acetylation</span></div><div class="casAuthors">Filippakopoulos, Panagis; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">337-356</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Lysine acetylation is a key mechanism that regulates chromatin structure; aberrant acetylation levels have been linked to the development of several diseases.  Acetyl-lysine modifications create docking sites for bromodomains, which are small interaction modules found on diverse proteins, some of which have a key role in the acetylation-dependent assembly of transcriptional regulator complexes.  These complexes can then initiate transcriptional programs that result in phenotypic changes.  The recent discovery of potent and highly specific inhibitors for the BET (bromodomain and extra-terminal) family of bromodomains has stimulated intensive research activity in diverse therapeutic areas, particularly in oncol., where BET proteins regulate the expression of key oncogenes and anti-apoptotic proteins.  In addn., targeting BET bromodomains could hold potential for the treatment of inflammation and viral infection.  Here, we highlight recent progress in the development of bromodomain inhibitors, and their potential applications in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOSIcwBJKj4LVg90H21EOLACvtfcHk0livcN66xzls0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D&md5=77049c82ce6a19b4685e3b7950399a07</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fnrd4286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4286%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DTargeting%2520bromodomains%253A%2520epigenetic%2520readers%2520of%2520lysine%2520acetylation%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D337%26epage%3D356" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Hewings, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Philpott, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Uttarkar, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Picaud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vuppusetty, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marsden, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Conway, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heightman, T. D.</span><span> </span><span class="NLM_article-title">3,5-Dimethylisoxazoles act as acetyl-lysine-mimetic bromodomain ligands</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">6761</span><span class="NLM_x">–</span> <span class="NLM_lpage">6770</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm200640v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=6761-6770&author=D.+S.+Hewingsauthor=M.+Wangauthor=M.+Philpottauthor=O.+Fedorovauthor=S.+Uttarkarauthor=P.+Filippakopoulosauthor=S.+Picaudauthor=C.+Vuppusettyauthor=B.+Marsdenauthor=S.+Knappauthor=S.+J.+Conwayauthor=T.+D.+Heightman&title=3%2C5-Dimethylisoxazoles+act+as+acetyl-lysine-mimetic+bromodomain+ligands"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fjm200640v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200640v%26sid%3Dliteratum%253Aachs%26aulast%3DHewings%26aufirst%3DD.%2BS.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DUttarkar%26aufirst%3DS.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DVuppusetty%26aufirst%3DC.%26aulast%3DMarsden%26aufirst%3DB.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DConway%26aufirst%3DS.%2BJ.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26atitle%3D3%252C5-Dimethylisoxazoles%2520act%2520as%2520acetyl-lysine-mimetic%2520bromodomain%2520ligands%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D6761%26epage%3D6770" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">McKeown, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marineau, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buckley, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kadam, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blacklow, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bradner, J. E.</span><span> </span><span class="NLM_article-title">Biased multicomponent reactions to develop novel bromodomain inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">9019</span><span class="NLM_x">–</span> <span class="NLM_lpage">9027</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm501120z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=9019-9027&author=M.+R.+McKeownauthor=D.+L.+Shawauthor=H.+Fuauthor=S.+Liuauthor=X.+Xuauthor=J.+J.+Marineauauthor=Y.+Huangauthor=X.+Zhangauthor=D.+L.+Buckleyauthor=A.+Kadamauthor=Z.+Zhangauthor=S.+C.+Blacklowauthor=J.+Qiauthor=W.+Zhangauthor=J.+E.+Bradner&title=Biased+multicomponent+reactions+to+develop+novel+bromodomain+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fjm501120z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501120z%26sid%3Dliteratum%253Aachs%26aulast%3DMcKeown%26aufirst%3DM.%2BR.%26aulast%3DShaw%26aufirst%3DD.%2BL.%26aulast%3DFu%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DXu%26aufirst%3DX.%26aulast%3DMarineau%26aufirst%3DJ.%2BJ.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DKadam%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DBlacklow%26aufirst%3DS.%2BC.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DBiased%2520multicomponent%2520reactions%2520to%2520develop%2520novel%2520bromodomain%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D9019%26epage%3D9027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Fish, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bish, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brennan, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunnage, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cook, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Federov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerstenberger, B. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marsden, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nocka, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Owen, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Philpott, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Picaud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Primiano, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ralph, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sciammetta, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trzupek, J. D.</span><span> </span><span class="NLM_article-title">Identification of a chemical probe for bromo and extra C-terminal bromodomain inhibition through optimization of a fragment-derived hit</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">9831</span><span class="NLM_x">–</span> <span class="NLM_lpage">9837</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm3010515" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFGmtLnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=9831-9837&author=P.+V.+Fishauthor=P.+Filippakopoulosauthor=G.+Bishauthor=P.+E.+Brennanauthor=M.+E.+Bunnageauthor=A.+S.+Cookauthor=O.+Federovauthor=B.+S.+Gerstenbergerauthor=H.+Jonesauthor=S.+Knappauthor=B.+Marsdenauthor=K.+Nockaauthor=D.+R.+Owenauthor=M.+Philpottauthor=S.+Picaudauthor=M.+J.+Primianoauthor=M.+J.+Ralphauthor=N.+Sciammettaauthor=J.+D.+Trzupek&title=Identification+of+a+chemical+probe+for+bromo+and+extra+C-terminal+bromodomain+inhibition+through+optimization+of+a+fragment-derived+hit"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a Chemical Probe for Bromo and Extra C-Terminal Bromodomain Inhibition through Optimization of a Fragment-Derived Hit</span></div><div class="casAuthors">Fish, Paul V.; Filippakopoulos, Panagis; Bish, Gerwyn; Brennan, Paul E.; Bunnage, Mark E.; Cook, Andrew S.; Federov, Oleg; Gerstenberger, Brian S.; Jones, Hannah; Knapp, Stefan; Marsden, Brian; Nocka, Karl; Owen, Dafydd R.; Philpott, Martin; Picaud, Sarah; Primiano, Michael J.; Ralph, Michael J.; Sciammetta, Nunzio; Trzupek, John D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">9831-9837</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The posttranslational modification of chromatin through acetylation at selected histone lysine residues is governed by histone acetyltransferases (HATs) and histone deacetylases (HDACs).  The significance of this subset of the epigenetic code is interrogated and interpreted by an acetyllysine-specific protein-protein interaction with bromodomain reader modules.  Selective inhibition of the bromo and extra C-terminal domain (BET) family of bromodomains with a small mol. is feasible, and this may represent an opportunity for disease intervention through the recently disclosed antiproliferative and anti-inflammatory properties of such inhibitors.  Herein, we describe the discovery and structure-activity relationship (SAR) of a novel, small-mol. chem. probe for BET family inhibition that was identified through the application of structure-based fragment assessment and optimization techniques.  This has yielded a potent, selective compd. with cell-based activity (PFI-1) that may further add to the understanding of BET family function within the bromodomains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphzpWnuXoLdrVg90H21EOLACvtfcHk0ljOwIb1qKQ8Cg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFGmtLnE&md5=ba864f3916bea5f793dd0b26732784f0</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fjm3010515&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm3010515%26sid%3Dliteratum%253Aachs%26aulast%3DFish%26aufirst%3DP.%2BV.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DBish%26aufirst%3DG.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DCook%26aufirst%3DA.%2BS.%26aulast%3DFederov%26aufirst%3DO.%26aulast%3DGerstenberger%26aufirst%3DB.%2BS.%26aulast%3DJones%26aufirst%3DH.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DMarsden%26aufirst%3DB.%26aulast%3DNocka%26aufirst%3DK.%26aulast%3DOwen%26aufirst%3DD.%2BR.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DPrimiano%26aufirst%3DM.%2BJ.%26aulast%3DRalph%26aufirst%3DM.%2BJ.%26aulast%3DSciammetta%26aufirst%3DN.%26aulast%3DTrzupek%26aufirst%3DJ.%2BD.%26atitle%3DIdentification%2520of%2520a%2520chemical%2520probe%2520for%2520bromo%2520and%2520extra%2520C-terminal%2520bromodomain%2520inhibition%2520through%2520optimization%2520of%2520a%2520fragment-derived%2520hit%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D9831%26epage%3D9837" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Chung, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dean, A. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woolven, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bamborough, P.</span><span> </span><span class="NLM_article-title">Fragment-based discovery of bromodomain inhibitors part 1: inhibitor binding modes and implications for lead discovery</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">576</span><span class="NLM_x">–</span> <span class="NLM_lpage">586</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm201320w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFKktrnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=576-586&author=C.+W.+Chungauthor=A.+W.+Deanauthor=J.+M.+Woolvenauthor=P.+Bamborough&title=Fragment-based+discovery+of+bromodomain+inhibitors+part+1%3A+inhibitor+binding+modes+and+implications+for+lead+discovery"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Discovery of Bromodomain Inhibitors Part 1: Inhibitor Binding Modes and Implications for Lead Discovery</span></div><div class="casAuthors">Chung, Chun-wa; Dean, Anthony W.; Woolven, James M.; Bamborough, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">576-586</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomain-contg. proteins are key epigenetic regulators of gene transcription and readers of the histone code.  However, the therapeutic benefits of modulating this target class are largely unexplored due to the lack of suitable chem. probes.  This article describes the generation of lead mols. for the BET bromodomains through screening a fragment set chosen using structural insights and computational approaches.  Anal. of 40 BRD2/fragment x-ray complexes highlights both shared and disparate interaction features that may be exploited for affinity and selectivity.  Six representative crystal structures are then exemplified in detail.  Two of the fragments are completely new bromodomain chemotypes, and three have never before been crystd. in a bromodomain, so our results significantly extend the limited public knowledge-base of crystallog. small mol./bromodomain interactions.  Certain fragments (including paracetamol) bind in a consistent mode to different bromodomains such as CREBBP, suggesting their potential to act as generic bromodomain templates.  An important implication is that the bromodomains are not only a phylogenetic family but also a system in which chem. and structural knowledge of one bromodomain gives insights transferable to others.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonNb7GnZnS_rVg90H21EOLACvtfcHk0ljOwIb1qKQ8Cg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFKktrnN&md5=5b7d7239d0de57266d94beef1ea080ae</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fjm201320w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm201320w%26sid%3Dliteratum%253Aachs%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DDean%26aufirst%3DA.%2BW.%26aulast%3DWoolven%26aufirst%3DJ.%2BM.%26aulast%3DBamborough%26aufirst%3DP.%26atitle%3DFragment-based%2520discovery%2520of%2520bromodomain%2520inhibitors%2520part%25201%253A%2520inhibitor%2520binding%2520modes%2520and%2520implications%2520for%2520lead%2520discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D576%26epage%3D586" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Vidler, L. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Picaud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tomsett, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woodward, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoelder, S.</span><span> </span><span class="NLM_article-title">Discovery of novel small-molecule inhibitors of BRD4 using structure-based virtual screening</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">8073</span><span class="NLM_x">–</span> <span class="NLM_lpage">8088</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm4011302" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFOjurrI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=8073-8088&author=L.+R.+Vidlerauthor=P.+Filippakopoulosauthor=O.+Fedorovauthor=S.+Picaudauthor=S.+Martinauthor=M.+Tomsettauthor=H.+Woodwardauthor=N.+Brownauthor=S.+Knappauthor=S.+Hoelder&title=Discovery+of+novel+small-molecule+inhibitors+of+BRD4+using+structure-based+virtual+screening"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Novel Small-Molecule Inhibitors of BRD4 Using Structure-Based Virtual Screening</span></div><div class="casAuthors">Vidler, Lewis R.; Filippakopoulos, Panagis; Fedorov, Oleg; Picaud, Sarah; Martin, Sarah; Tomsett, Michael; Woodward, Hannah; Brown, Nathan; Knapp, Stefan; Hoelder, Swen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">8073-8088</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomains (BRDs) are epigenetic readers that recognize acetylated-lysine (KAc) on proteins and are implicated in a no. of diseases.  We describe a virtual screening approach to identify BRD inhibitors.  Key elements of this approach are the extensive design and use of substructure queries to compile a set of com. available compds. featuring novel putative KAc mimetics and docking this set for final compd. selection.  We describe the validation of this approach by applying it to the first BRD of BRD4.  The selection and testing of 143 compds. lead to the discovery of six novel hits, including four unprecedented KAc mimetics.  We solved the crystal structure of four hits, detd. their binding mode, and improved their potency through synthesis and the purchase of derivs.  This work provides a validated virtual screening approach that is applicable to other BRDs and describes novel KAc mimetics that can be further explored to design more potent inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGow5yj8bnohUbVg90H21EOLACvtfcHk0li2blTqDAtACg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFOjurrI&md5=c1f76e4838713335be7a93ebe4c68701</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fjm4011302&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm4011302%26sid%3Dliteratum%253Aachs%26aulast%3DVidler%26aufirst%3DL.%2BR.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DTomsett%26aufirst%3DM.%26aulast%3DWoodward%26aufirst%3DH.%26aulast%3DBrown%26aufirst%3DN.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DHoelder%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520novel%2520small-molecule%2520inhibitors%2520of%2520BRD4%2520using%2520structure-based%2520virtual%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D8073%26epage%3D8088" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Zhao, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miao, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Du, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiong, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, J.</span><span> </span><span class="NLM_article-title">Fragment-based drug discovery of 2-thiazolidinones as inhibitors of the histone reader BRD4 bromodomain</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">3833</span><span class="NLM_x">–</span> <span class="NLM_lpage">3851</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm301793a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BC3sXksF2htL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=3833-3851&author=L.+Zhaoauthor=D.+Caoauthor=T.+Chenauthor=Y.+Wangauthor=Z.+Miaoauthor=Y.+Xuauthor=W.+Chenauthor=X.+Wangauthor=Y.+Liauthor=Z.+Duauthor=B.+Xiongauthor=J.+Liauthor=C.+Xuauthor=N.+Zhangauthor=J.+Heauthor=J.+Shen&title=Fragment-based+drug+discovery+of+2-thiazolidinones+as+inhibitors+of+the+histone+reader+BRD4+bromodomain"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Drug Discovery of 2-Thiazolidinones as Inhibitors of the Histone Reader BRD4 Bromodomain</span></div><div class="casAuthors">Zhao, Lele; Cao, Danyan; Chen, Tiantian; Wang, Yingqing; Miao, Zehong; Xu, Yechun; Chen, Wuyan; Wang, Xin; Li, Yanlian; Du, Zhiyan; Xiong, Bing; Li, Jian; Xu, Chunyan; Zhang, Naixia; He, Jianhua; Shen, Jingkang</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3833-3851</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Recognizing acetyllysine of histone is a vital process of epigenetic regulation that is mediated by a protein module called bromodomain.  To contribute novel scaffolds for developing into bromodomain inhibitors, the authors utilize a fragment-based drug discovery approach.  By successively applying docking and x-ray crystallog., they were able to identify 9 fragment hits from diffracting more than 60 crystals.  In the present work, they described four of them and carried out the integrated lead optimization for fragment I, which bears a 2-thiazolidinone core.  After several rounds of structure guided modifications, they assessed the druggability of 2-thiazolidinone by modulating in vitro pharmacokinetic studies and cellular activity assay.  The results showed that two potent of 2-thiazolidinone compds. II and III have good metabolic stability.  Also, the cellular assay confirmed the activities of 2-thiazolidinones.  Together, they hope the identified 2-thiazolidinone chemotype and other fragment hits described herein can stimulate researchers to develop more diversified bromodomain inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfi2pyYq6c4LVg90H21EOLACvtfcHk0li2blTqDAtACg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXksF2htL4%253D&md5=8308db5d83cb72b29f261cc134514ece</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fjm301793a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm301793a%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DL.%26aulast%3DCao%26aufirst%3DD.%26aulast%3DChen%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DMiao%26aufirst%3DZ.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DDu%26aufirst%3DZ.%26aulast%3DXiong%26aufirst%3DB.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DN.%26aulast%3DHe%26aufirst%3DJ.%26aulast%3DShen%26aufirst%3DJ.%26atitle%3DFragment-based%2520drug%2520discovery%2520of%25202-thiazolidinones%2520as%2520inhibitors%2520of%2520the%2520histone%2520reader%2520BRD4%2520bromodomain%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D3833%26epage%3D3851" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Lucas, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wohlwend, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hugle, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmidtkunz, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerhardt, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schule, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jung, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Einsle, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gunther, S.</span><span> </span><span class="NLM_article-title">4-Acyl pyrroles: mimicking acetylated lysines in histone code reading</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">14055</span><span class="NLM_x">–</span> <span class="NLM_lpage">14059</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2013&pages=14055-14059&author=X.+Lucasauthor=D.+Wohlwendauthor=M.+Hugleauthor=K.+Schmidtkunzauthor=S.+Gerhardtauthor=R.+Schuleauthor=M.+Jungauthor=O.+Einsleauthor=S.+Gunther&title=4-Acyl+pyrroles%3A+mimicking+acetylated+lysines+in+histone+code+reading"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLucas%26aufirst%3DX.%26aulast%3DWohlwend%26aufirst%3DD.%26aulast%3DHugle%26aufirst%3DM.%26aulast%3DSchmidtkunz%26aufirst%3DK.%26aulast%3DGerhardt%26aufirst%3DS.%26aulast%3DSchule%26aufirst%3DR.%26aulast%3DJung%26aufirst%3DM.%26aulast%3DEinsle%26aufirst%3DO.%26aulast%3DGunther%26aufirst%3DS.%26atitle%3D4-Acyl%2520pyrroles%253A%2520mimicking%2520acetylated%2520lysines%2520in%2520histone%2520code%2520reading%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2013%26volume%3D52%26spage%3D14055%26epage%3D14059" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Fedorov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lingard, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wells, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Monteiro, O. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Picaud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keates, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yapp, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Philpott, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Felletar, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marsden, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muller, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brennan, P. E.</span><span> </span><span class="NLM_article-title">[1,2,4]Triazolo[4,3-a]phthalazines: inhibitors of diverse bromodomains</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">462</span><span class="NLM_x">–</span> <span class="NLM_lpage">476</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm401568s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=462-476&author=O.+Fedorovauthor=H.+Lingardauthor=C.+Wellsauthor=O.+P.+Monteiroauthor=S.+Picaudauthor=T.+Keatesauthor=C.+Yappauthor=M.+Philpottauthor=S.+J.+Martinauthor=I.+Felletarauthor=B.+D.+Marsdenauthor=P.+Filippakopoulosauthor=S.+Mullerauthor=S.+Knappauthor=P.+E.+Brennan&title=%5B1%2C2%2C4%5DTriazolo%5B4%2C3-a%5Dphthalazines%3A+inhibitors+of+diverse+bromodomains"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fjm401568s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401568s%26sid%3Dliteratum%253Aachs%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DLingard%26aufirst%3DH.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DMonteiro%26aufirst%3DO.%2BP.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DYapp%26aufirst%3DC.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DMartin%26aufirst%3DS.%2BJ.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DMarsden%26aufirst%3DB.%2BD.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26atitle%3D%255B1%252C2%252C4%255DTriazolo%255B4%252C3-a%255Dphthalazines%253A%2520inhibitors%2520of%2520diverse%2520bromodomains%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D462%26epage%3D476" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Bamborough, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diallo, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goodacre, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gordon, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lewis, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seal, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woodrow, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, C. W.</span><span> </span><span class="NLM_article-title">Fragment-based discovery of bromodomain inhibitors part 2: optimization of phenylisoxazole sulfonamides</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">587</span><span class="NLM_x">–</span> <span class="NLM_lpage">596</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm201283q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFKktrfF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=587-596&author=P.+Bamboroughauthor=H.+Dialloauthor=J.+D.+Goodacreauthor=L.+Gordonauthor=A.+Lewisauthor=J.+T.+Sealauthor=D.+M.+Wilsonauthor=M.+D.+Woodrowauthor=C.+W.+Chung&title=Fragment-based+discovery+of+bromodomain+inhibitors+part+2%3A+optimization+of+phenylisoxazole+sulfonamides"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Discovery of Bromodomain Inhibitors Part 2: Optimization of Phenylisoxazole Sulfonamides</span></div><div class="casAuthors">Bamborough, Paul; Diallo, Hawa; Goodacre, Jonathan D.; Gordon, Laurie; Lewis, Antonia; Seal, Jonathan T.; Wilson, David M.; Woodrow, Michael D.; Chung, Chun-wa</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">587-596</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomains are epigenetic reader modules that regulate gene transcription through their recognition of acetyl-lysine modified histone tails.  Inhibitors of this protein-protein interaction have the potential to modulate multiple diseases as demonstrated by the profound anti-inflammatory and antiproliferative effects of a recently disclosed class of BET compds.  While these compds. were discovered using phenotypic assays, here we present a highly efficient alternative approach to find new chem. templates, exploiting the abundant structural knowledge that exists for this target class.  A Ph di-Me isoxazole chemotype resulting from a focused fragment screen has been rapidly optimized through structure-based design, leading to a sulfonamide series showing anti-inflammatory activity in cellular assays.  This proof-of-principle expt. demonstrates the tractability of the BET family and bromodomain target class to fragment-based hit discovery and structure-based lead optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqr6lsic9qjJrVg90H21EOLACvtfcHk0lis5zjvBkjbEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFKktrfF&md5=e73b16b3fddf88b43b037264b000b276</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fjm201283q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm201283q%26sid%3Dliteratum%253Aachs%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DDiallo%26aufirst%3DH.%26aulast%3DGoodacre%26aufirst%3DJ.%2BD.%26aulast%3DGordon%26aufirst%3DL.%26aulast%3DLewis%26aufirst%3DA.%26aulast%3DSeal%26aufirst%3DJ.%2BT.%26aulast%3DWilson%26aufirst%3DD.%2BM.%26aulast%3DWoodrow%26aufirst%3DM.%2BD.%26aulast%3DChung%26aufirst%3DC.%2BW.%26atitle%3DFragment-based%2520discovery%2520of%2520bromodomain%2520inhibitors%2520part%25202%253A%2520optimization%2520of%2520phenylisoxazole%2520sulfonamides%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D587%26epage%3D596" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Picaud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Da Costa, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thanasopoulou, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fish, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Philpott, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brennan, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunnage, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Owen, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bradner, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taniere, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Sullivan, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muller, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwaller, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stankovic, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span> </span><span class="NLM_article-title">PFI-1, a highly selective protein interaction inhibitor, targeting BET Bromodomains</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">73</span><span class="NLM_x">, </span> <span class="NLM_fpage">3336</span><span class="NLM_x">–</span> <span class="NLM_lpage">3346</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2013&pages=3336-3346&author=S.+Picaudauthor=D.+Da+Costaauthor=A.+Thanasopoulouauthor=P.+Filippakopoulosauthor=P.+V.+Fishauthor=M.+Philpottauthor=O.+Fedorovauthor=P.+Brennanauthor=M.+E.+Bunnageauthor=D.+R.+Owenauthor=J.+E.+Bradnerauthor=P.+Taniereauthor=B.+O%E2%80%99Sullivanauthor=S.+Mullerauthor=J.+Schwallerauthor=T.+Stankovicauthor=S.+Knapp&title=PFI-1%2C+a+highly+selective+protein+interaction+inhibitor%2C+targeting+BET+Bromodomains"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DDa%2BCosta%26aufirst%3DD.%26aulast%3DThanasopoulou%26aufirst%3DA.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DFish%26aufirst%3DP.%2BV.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DBrennan%26aufirst%3DP.%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DOwen%26aufirst%3DD.%2BR.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DTaniere%26aufirst%3DP.%26aulast%3DO%25E2%2580%2599Sullivan%26aufirst%3DB.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DSchwaller%26aufirst%3DJ.%26aulast%3DStankovic%26aufirst%3DT.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DPFI-1%252C%2520a%2520highly%2520selective%2520protein%2520interaction%2520inhibitor%252C%2520targeting%2520BET%2520Bromodomains%26jtitle%3DCancer%2520Res.%26date%3D2013%26volume%3D73%26spage%3D3336%26epage%3D3346" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Vidler, L. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoelder, S.</span><span> </span><span class="NLM_article-title">Druggability analysis and structural classification of bromodomain acetyl-lysine binding sites</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">7346</span><span class="NLM_x">–</span> <span class="NLM_lpage">7359</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm300346w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVWht7fN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=7346-7359&author=L.+R.+Vidlerauthor=N.+Brownauthor=S.+Knappauthor=S.+Hoelder&title=Druggability+analysis+and+structural+classification+of+bromodomain+acetyl-lysine+binding+sites"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Druggability Analysis and Structural Classification of Bromodomain Acetyl-lysine Binding Sites</span></div><div class="casAuthors">Vidler, Lewis R.; Brown, Nathan; Knapp, Stefan; Hoelder, Swen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">7346-7359</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomains are readers of the epigenetic code that specifically bind acetyl-lysine contg. recognition sites on proteins.  Recently the BET family of bromodomains has been demonstrated to be druggable through the discovery of potent inhibitors, sparking an interest in protein-protein interaction inhibitors that directly target gene transcription.  Here, we assess the druggability of diverse members of the bromodomain family using SiteMap and show that there are significant differences in predicted druggability.  Furthermore, we trace these differences in druggability back to unique amino acid signatures in the bromodomain acetyl-lysine binding sites.  These signatures were then used to generate a new classification of the bromodomain family, visualized as a classification tree.  This represents the first anal. of this type for the bromodomain family and can prove useful in the discovery of inhibitors, particularly for anticipating screening hit rates, identifying inhibitors that can be explored for lead hopping approaches, and selecting proteins for selectivity screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxYqQyqqj7JrVg90H21EOLACvtfcHk0lis5zjvBkjbEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVWht7fN&md5=bfb08154c1964aab5df51cb3513f2c1d</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fjm300346w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm300346w%26sid%3Dliteratum%253Aachs%26aulast%3DVidler%26aufirst%3DL.%2BR.%26aulast%3DBrown%26aufirst%3DN.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DHoelder%26aufirst%3DS.%26atitle%3DDruggability%2520analysis%2520and%2520structural%2520classification%2520of%2520bromodomain%2520acetyl-lysine%2520binding%2520sites%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D7346%26epage%3D7359" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span>SGC. GSK2801: A Selective Chemical Probe for BAZ2B/A Bromodomains. <a href="http://www.thesgc.org/chemical-probes/GSK2801" class="extLink">http://www.thesgc.org/chemical-probes/GSK2801</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=SGC.+GSK2801%3A+A+Selective+Chemical+Probe+for+BAZ2B%2FA+Bromodomains.+http%3A%2F%2Fwww.thesgc.org%2Fchemical-probes%2FGSK2801."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Hay, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Singleton, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tallant, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wells, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Picaud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Philpott, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Monteiro, O. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rogers, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Conway, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rooney, T. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tumber, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yapp, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunnage, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muller, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brennan, P. E.</span><span> </span><span class="NLM_article-title">Discovery and optimization of small-molecule ligands for the CBP/p300 bromodomains</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">136</span><span class="NLM_x">, </span> <span class="NLM_fpage">9308</span><span class="NLM_x">–</span> <span class="NLM_lpage">9319</span></span><div class="citationLinks">[<a href="/doi/10.1021/ja412434f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVShsLvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=136&publication_year=2014&pages=9308-9319&author=D.+A.+Hayauthor=O.+Fedorovauthor=S.+Martinauthor=D.+C.+Singletonauthor=C.+Tallantauthor=C.+Wellsauthor=S.+Picaudauthor=M.+Philpottauthor=O.+P.+Monteiroauthor=C.+M.+Rogersauthor=S.+J.+Conwayauthor=T.+P.+Rooneyauthor=A.+Tumberauthor=C.+Yappauthor=P.+Filippakopoulosauthor=M.+E.+Bunnageauthor=S.+Mullerauthor=S.+Knappauthor=C.+J.+Schofieldauthor=P.+E.+Brennan&title=Discovery+and+optimization+of+small-molecule+ligands+for+the+CBP%2Fp300+bromodomains"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Optimization of Small-Molecule Ligands for the CBP/p300 Bromodomains</span></div><div class="casAuthors">Hay, Duncan A.; Fedorov, Oleg; Martin, Sarah; Singleton, Dean C.; Tallant, Cynthia; Wells, Christopher; Picaud, Sarah; Philpott, Martin; Monteiro, Octovia P.; Rogers, Catherine M.; Conway, Stuart J.; Rooney, Timothy P. C.; Tumber, Anthony; Yapp, Clarence; Filippakopoulos, Panagis; Bunnage, Mark E.; Muller, Susanne; Knapp, Stefan; Schofield, Christopher J.; Brennan, Paul E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">136</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">9308-9319</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Small-mol. inhibitors that target bromodomains outside of the bromodomain and extra-terminal (BET) sub-family are lacking.  Here, the authors describe highly potent and selective ligands for the bromodomain module of the human lysine acetyl transferase CBP/p300, developed from a series of 5-isoxazolyl-benzimidazoles.  The starting point was a fragment hit, which was optimized into a more potent and selective lead using parallel synthesis employing Suzuki couplings, benzimidazole-forming reactions, and reductive aminations.  The selectivity of the lead compd. against other bromodomain family members was investigated using a thermal stability assay, which revealed some inhibition of the structurally related BET family members.  To address the BET selectivity issue, x-ray crystal structures of the lead compd. bound to the CREB binding protein (CBP) and the first bromodomain of BRD4 (BRD4(1)) were used to guide the design of more selective compds.  The crystal structures obtained revealed two distinct binding modes.  By varying the aryl substitution pattern and developing conformationally constrained analogs, selectivity for CBP over BRD4(1) was increased.  The optimized compd. is highly potent (Kd = 21 nM) and selective, displaying 40-fold selectivity over BRD4(1).  Cellular activity was demonstrated using fluorescence recovery after photo-bleaching (FRAP) and a p53 reporter assay.  The optimized compds. are cell-active and have nanomolar affinity for CBP/p300; therefore, they should be useful in studies investigating the biol. roles of CBP and p300 and to validate the CBP and p300 bromodomains as therapeutic targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqFzgs1DD4QZLVg90H21EOLACvtfcHk0lis5zjvBkjbEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVShsLvP&md5=e5da3e96a4d234453a481c7ee5e788f2</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fja412434f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja412434f%26sid%3Dliteratum%253Aachs%26aulast%3DHay%26aufirst%3DD.%2BA.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DSingleton%26aufirst%3DD.%2BC.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DMonteiro%26aufirst%3DO.%2BP.%26aulast%3DRogers%26aufirst%3DC.%2BM.%26aulast%3DConway%26aufirst%3DS.%2BJ.%26aulast%3DRooney%26aufirst%3DT.%2BP.%26aulast%3DTumber%26aufirst%3DA.%26aulast%3DYapp%26aufirst%3DC.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26atitle%3DDiscovery%2520and%2520optimization%2520of%2520small-molecule%2520ligands%2520for%2520the%2520CBP%252Fp300%2520bromodomains%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2014%26volume%3D136%26spage%3D9308%26epage%3D9319" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Chaikuad, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petros, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span> </span><span class="NLM_article-title">Structure-based approaches towards identification of fragments for the low-druggability ATAD2 bromodomain</span> <span class="citation_source-journal">Med. Chem. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">1843</span><span class="NLM_x">–</span> <span class="NLM_lpage">1848</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Fjm501893k&amp;key=10.1039%2FC4MD00237G" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=1843-1848&author=A.+Chaikuadauthor=A.+M.+Petrosauthor=O.+Fedorovauthor=J.+Xuauthor=S.+Knapp&title=Structure-based+approaches+towards+identification+of+fragments+for+the+low-druggability+ATAD2+bromodomain"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1039%2FC4MD00237G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC4MD00237G%26sid%3Dliteratum%253Aachs%26aulast%3DChaikuad%26aufirst%3DA.%26aulast%3DPetros%26aufirst%3DA.%2BM.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DStructure-based%2520approaches%2520towards%2520identification%2520of%2520fragments%2520for%2520the%2520low-druggability%2520ATAD2%2520bromodomain%26jtitle%3DMed.%2520Chem.%2520Commun.%26date%3D2014%26volume%3D5%26spage%3D1843%26epage%3D1848" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">Harner, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chauder, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Phan, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fesik, S. W.</span><span> </span><span class="NLM_article-title">Fragment-Based Screening of the Bromodomain of ATAD2</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">9687</span><span class="NLM_x">–</span> <span class="NLM_lpage">9692</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm501035j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=9687-9692&author=M.+J.+Harnerauthor=B.+A.+Chauderauthor=J.+Phanauthor=S.+W.+Fesik&title=Fragment-Based+Screening+of+the+Bromodomain+of+ATAD2"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fjm501035j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501035j%26sid%3Dliteratum%253Aachs%26aulast%3DHarner%26aufirst%3DM.%2BJ.%26aulast%3DChauder%26aufirst%3DB.%2BA.%26aulast%3DPhan%26aufirst%3DJ.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26atitle%3DFragment-Based%2520Screening%2520of%2520the%2520Bromodomain%2520of%2520ATAD2%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D9687%26epage%3D9692" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group">Ferguson, F. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chaikuad, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Philpott, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muniz, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Felletar, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">von Delft, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heightman, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abell, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciulli, A.</span><span> </span><span class="NLM_article-title">Targeting low-druggability bromodomains: fragment based screening and inhibitor design against the BAZ2B bromodomain</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">10183</span><span class="NLM_x">–</span> <span class="NLM_lpage">10187</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm401582c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=10183-10187&author=F.+M.+Fergusonauthor=O.+Fedorovauthor=A.+Chaikuadauthor=M.+Philpottauthor=J.+R.+Munizauthor=I.+Felletarauthor=F.+von+Delftauthor=T.+Heightmanauthor=S.+Knappauthor=C.+Abellauthor=A.+Ciulli&title=Targeting+low-druggability+bromodomains%3A+fragment+based+screening+and+inhibitor+design+against+the+BAZ2B+bromodomain"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Fjm401582c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401582c%26sid%3Dliteratum%253Aachs%26aulast%3DFerguson%26aufirst%3DF.%2BM.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DChaikuad%26aufirst%3DA.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DMuniz%26aufirst%3DJ.%2BR.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3Dvon%2BDelft%26aufirst%3DF.%26aulast%3DHeightman%26aufirst%3DT.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DAbell%26aufirst%3DC.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DTargeting%2520low-druggability%2520bromodomains%253A%2520fragment%2520based%2520screening%2520and%2520inhibitor%2520design%2520against%2520the%2520BAZ2B%2520bromodomain%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D10183%26epage%3D10187" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group">Demont, E. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bamborough, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Craggs, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fallon, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gordon, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grandi, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hobbs, C. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hussain, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Le Gall, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Michon, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mitchell, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prinjha, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roberts, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sheppard, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watson, R. J.</span><span> </span><span class="NLM_article-title">1,3-Dimethyl benzimidazolones are potent, selective inhibitors of the BRPF1 bromodomain</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">1190</span><span class="NLM_x">–</span> <span class="NLM_lpage">1195</span></span><div class="citationLinks">[<a href="/doi/10.1021/ml5002932" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=1190-1195&author=E.+H.+Demontauthor=P.+Bamboroughauthor=C.+W.+Chungauthor=P.+D.+Craggsauthor=D.+Fallonauthor=L.+J.+Gordonauthor=P.+Grandiauthor=C.+I.+Hobbsauthor=J.+Hussainauthor=E.+J.+Jonesauthor=A.+Le+Gallauthor=A.+M.+Michonauthor=D.+J.+Mitchellauthor=R.+K.+Prinjhaauthor=A.+D.+Robertsauthor=R.+J.+Sheppardauthor=R.+J.+Watson&title=1%2C3-Dimethyl+benzimidazolones+are+potent%2C+selective+inhibitors+of+the+BRPF1+bromodomain"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fml5002932&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml5002932%26sid%3Dliteratum%253Aachs%26aulast%3DDemont%26aufirst%3DE.%2BH.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DCraggs%26aufirst%3DP.%2BD.%26aulast%3DFallon%26aufirst%3DD.%26aulast%3DGordon%26aufirst%3DL.%2BJ.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DHobbs%26aufirst%3DC.%2BI.%26aulast%3DHussain%26aufirst%3DJ.%26aulast%3DJones%26aufirst%3DE.%2BJ.%26aulast%3DLe%2BGall%26aufirst%3DA.%26aulast%3DMichon%26aufirst%3DA.%2BM.%26aulast%3DMitchell%26aufirst%3DD.%2BJ.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DRoberts%26aufirst%3DA.%2BD.%26aulast%3DSheppard%26aufirst%3DR.%2BJ.%26aulast%3DWatson%26aufirst%3DR.%2BJ.%26atitle%3D1%252C3-Dimethyl%2520benzimidazolones%2520are%2520potent%252C%2520selective%2520inhibitors%2520of%2520the%2520BRPF1%2520bromodomain%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D5%26spage%3D1190%26epage%3D1195" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group">Ciceri, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muller, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Mahony, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunt, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lasater, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pallares, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Picaud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wells, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wodicka, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shah, N. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Treiber, D. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span> </span><span class="NLM_article-title">Dual kinase-bromodomain inhibitors for rationally designed polypharmacology</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">305</span><span class="NLM_x">–</span> <span class="NLM_lpage">312</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Fjm501893k&amp;key=10.1038%2Fnchembio.1471" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Fjm501893k&amp;key=24584101" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjtl2itro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=305-312&author=P.+Ciceriauthor=S.+Mullerauthor=A.+O%E2%80%99Mahonyauthor=O.+Fedorovauthor=P.+Filippakopoulosauthor=J.+P.+Huntauthor=E.+A.+Lasaterauthor=G.+Pallaresauthor=S.+Picaudauthor=C.+Wellsauthor=S.+Martinauthor=L.+M.+Wodickaauthor=N.+P.+Shahauthor=D.+K.+Treiberauthor=S.+Knapp&title=Dual+kinase-bromodomain+inhibitors+for+rationally+designed+polypharmacology"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Dual kinase-bromodomain inhibitors for rationally designed polypharmacology</span></div><div class="casAuthors">Ciceri, Pietro; Mueller, Susanne; O'Mahony, Alison; Fedorov, Oleg; Filippakopoulos, Panagis; Hunt, Jeremy P.; Lasater, Elisabeth A.; Pallares, Gabriel; Picaud, Sarah; Wells, Christopher; Martin, Sarah; Wodicka, Lisa M.; Shah, Neil P.; Treiber, Daniel K.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">305-312</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Concomitant inhibition of multiple cancer-driving kinases is an established strategy to improve the durability of clin. responses to targeted therapies.  The difficulty of discovering kinase inhibitors with an appropriate multitarget profile has, however, necessitated the application of combination therapies, which can pose major clin. development challenges.  Epigenetic reader domains of the bromodomain family have recently emerged as new targets for cancer therapy.  Here the authors report that several clin. kinase inhibitors also inhibit bromodomains with therapeutically relevant potencies and are best classified as dual kinase-bromodomain inhibitors.  Nanomolar activity on BRD4 by BI-2536 and TG-101348, which are clin. PLK1 and JAK2-FLT3 kinase inhibitors, resp., is particularly noteworthy as these combinations of activities on independent oncogenic pathways exemplify a new strategy for rational single-agent polypharmacol. targeting.  Furthermore, structure-activity relationships and co-crystal structures identify design features that enable a general platform for the rational design of dual kinase-bromodomain inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotPedDED4FJLVg90H21EOLACvtfcHk0liGWooLol0y_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjtl2itro%253D&md5=a00117db2648ea38dd03750faf3b3cb9</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1471&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1471%26sid%3Dliteratum%253Aachs%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DO%25E2%2580%2599Mahony%26aufirst%3DA.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DHunt%26aufirst%3DJ.%2BP.%26aulast%3DLasater%26aufirst%3DE.%2BA.%26aulast%3DPallares%26aufirst%3DG.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DShah%26aufirst%3DN.%2BP.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DDual%2520kinase-bromodomain%2520inhibitors%2520for%2520rationally%2520designed%2520polypharmacology%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3D305%26epage%3D312" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group">Martin, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olesen, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Georg, G. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schonbrunn, E.</span><span> </span><span class="NLM_article-title">Cyclin-dependent kinase inhibitor dinaciclib interacts with the acetyl-lysine recognition site of bromodomains</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">2360</span><span class="NLM_x">–</span> <span class="NLM_lpage">2365</span></span><div class="citationLinks">[<a href="/doi/10.1021/cb4003283" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtl2rtbvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=2360-2365&author=M.+P.+Martinauthor=S.+H.+Olesenauthor=G.+I.+Georgauthor=E.+Schonbrunn&title=Cyclin-dependent+kinase+inhibitor+dinaciclib+interacts+with+the+acetyl-lysine+recognition+site+of+bromodomains"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclin-Dependent Kinase Inhibitor Dinaciclib Interacts with the Acetyl-Lysine Recognition Site of Bromodomains</span></div><div class="casAuthors">Martin, Mathew P.; Olesen, Sanne H.; Georg, Gunda I.; Schonbrunn, Ernst</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2360-2365</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomain-contg. proteins are considered atypical kinases, but their potential to interact with kinase inhibitors is unknown.  Dinaciclib is a potent inhibitor of cyclin-dependent kinases (CDKs), which recently advanced to Phase III clin. trials for the treatment of leukemia.  The authors detd. the crystal structure of dinaciclib in complex with CDK2 at 1.7 Å resoln., revealing an elaborate network of binding interactions in the ATP site, which explains the extraordinary potency and selectivity of this inhibitor.  Remarkably, dinaciclib also interacted with the acetyl-lysine recognition site of the bromodomain testis-specific protein BRDT, a member of the BET family of bromodomains.  The binding mode of dinaciclib to BRDT at 2.0 Å resoln. suggests that general kinase inhibitors (hinge binders) possess a previously unrecognized potential to act as protein-protein inhibitors of bromodomains.  The findings may provide a new structural framework for the design of next-generation bromodomain inhibitors using the vast chem. space of kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonrxl9QUxkE7Vg90H21EOLACvtfcHk0lho4q-5BnNDlg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtl2rtbvN&md5=1ae089e8e571ad11aadcdc4a1a5769b5</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Fcb4003283&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb4003283%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DM.%2BP.%26aulast%3DOlesen%26aufirst%3DS.%2BH.%26aulast%3DGeorg%26aufirst%3DG.%2BI.%26aulast%3DSchonbrunn%26aufirst%3DE.%26atitle%3DCyclin-dependent%2520kinase%2520inhibitor%2520dinaciclib%2520interacts%2520with%2520the%2520acetyl-lysine%2520recognition%2520site%2520of%2520bromodomains%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2013%26volume%3D8%26spage%3D2360%26epage%3D2365" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group">Rosemeyer, H.</span><span> </span><span class="NLM_article-title">The chemodiversity of purine as a constituent of natural products</span> <span class="citation_source-journal">Chem. Biodiversity</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">361</span><span class="NLM_x">–</span> <span class="NLM_lpage">401</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Fjm501893k&amp;key=10.1002%2Fcbdv.200490033" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Fjm501893k&amp;key=17191854" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjtlOht7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2004&pages=361-401&author=H.+Rosemeyer&title=The+chemodiversity+of+purine+as+a+constituent+of+natural+products"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">The chemodiversity of purine as a constituent of natural products</span></div><div class="casAuthors">Rosemeyer, Helmut</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biodiversity</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">361-401</span>CODEN:
                <span class="NLM_cas:coden">CBHIAM</span>;
        ISSN:<span class="NLM_cas:issn">1612-1872</span>.
    
            (<span class="NLM_cas:orgname">Verlag Helvetica Chimica Acta AG</span>)
        </div><div class="casAbstract">In this review, various natural manifestations of purine systems, i.e., methylated, higher-alkylated, and glycosylated forms, are described.  These comprise the purine alkaloids, cytokinines, as well as the purine nucleoside antibiotics.  In part, the compds. described herein were isolated from natural sources already long ago.  However, some have been reported only during the last few years.  The biol. activities of most of the purine derivs. are briefly described, and, in some cases, syntheses are formulated.  In particular, this article introduces the main synthetic principles for the generation of the purine ring system.  The last chapter describes modern preparative routes for C-alkylation and C-arylation of purines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppvY0HCBfJirVg90H21EOLACvtfcHk0lho4q-5BnNDlg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjtlOht7k%253D&md5=5653bce55e74efb0a8b797514d33b862</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1002%2Fcbdv.200490033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbdv.200490033%26sid%3Dliteratum%253Aachs%26aulast%3DRosemeyer%26aufirst%3DH.%26atitle%3DThe%2520chemodiversity%2520of%2520purine%2520as%2520a%2520constituent%2520of%2520natural%2520products%26jtitle%3DChem.%2520Biodiversity%26date%3D2004%26volume%3D1%26spage%3D361%26epage%3D401" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span>U.S. Food and Drug Administration. <a href="http://www.fda.gov/" class="extLink">http://www.fda.gov/</a> (accessed November 24,<span class="NLM_x"> </span><span class="NLM_year">2014</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=U.S.+Food+and+Drug+Administration.+http%3A%2F%2Fwww.fda.gov%2F+%28accessed+November+24%2C+2014%29+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group">Gray, N. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wodicka, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thunnissen, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Norman, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kwon, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Espinoza, F. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morgan, D. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barnes, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">LeClerc, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meijer, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lockhart, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schultz, P. G.</span><span> </span><span class="NLM_article-title">Exploiting chemical libraries, structure, and genomics in the search for kinase inhibitors</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">281</span><span class="NLM_x">, </span> <span class="NLM_fpage">533</span><span class="NLM_x">–</span> <span class="NLM_lpage">538</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Fjm501893k&amp;key=10.1126%2Fscience.281.5376.533" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Fjm501893k&amp;key=9677190" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADyaK1cXkvFCltr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=281&publication_year=1998&pages=533-538&author=N.+S.+Grayauthor=L.+Wodickaauthor=A.+M.+Thunnissenauthor=T.+C.+Normanauthor=S.+Kwonauthor=F.+H.+Espinozaauthor=D.+O.+Morganauthor=G.+Barnesauthor=S.+LeClercauthor=L.+Meijerauthor=S.+H.+Kimauthor=D.+J.+Lockhartauthor=P.+G.+Schultz&title=Exploiting+chemical+libraries%2C+structure%2C+and+genomics+in+the+search+for+kinase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Exploiting chemical libraries, structure, and genomics in the search for kinase inhibitors</span></div><div class="casAuthors">Gray, Nathanael S.; Wodicka, Lisa; Thunnissen, Andy-Mark W. H.; Norman, Thea C.; Kwon, Soojin; Espinoza, F. Hernan; Morgan, David O.; Barnes, Georjana; LeClerc, Sophie; Meijer, Laurent; Kim, Sung-Hou; Lockhart, David J.; Schultz, Peter G.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">281</span>
        (<span class="NLM_cas:issue">5376</span>),
    <span class="NLM_cas:pages">533-538</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Selective protein kinase inhibitors were developed on the basis of the unexpected binding mode of 2,6,9-trisubstituted purines to the ATP-binding site of the human cyclin-dependent kinase 2 (CDK2).  By iterating chem. library synthesis and biol. screening, potent inhibitors of the human CDK2-cyclin A kinase complex and of Saccharomyces cerevisiae Cdc28p were identified.  The structural basis for the binding affinity and selectivity was detd. by anal. of a three-dimensional crystal structure of a CDK2-inhibitor complex.  The cellular effects of these compds. were characterized in mammalian cells and yeast.  In the latter case the effects were characterized on a genome-wide scale by monitoring changes in mRNA levels in treated cells with high-d. oligonucleotide probe arrays.  Purine libraries could provide useful tools for analyzing a variety of signaling and regulatory pathways and may led to the development of new therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRTOOPjk16tbVg90H21EOLACvtfcHk0lho4q-5BnNDlg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXkvFCltr8%253D&md5=08b8debe4d05c1169c152ccd6676def8</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1126%2Fscience.281.5376.533&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.281.5376.533%26sid%3Dliteratum%253Aachs%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DWodicka%26aufirst%3DL.%26aulast%3DThunnissen%26aufirst%3DA.%2BM.%26aulast%3DNorman%26aufirst%3DT.%2BC.%26aulast%3DKwon%26aufirst%3DS.%26aulast%3DEspinoza%26aufirst%3DF.%2BH.%26aulast%3DMorgan%26aufirst%3DD.%2BO.%26aulast%3DBarnes%26aufirst%3DG.%26aulast%3DLeClerc%26aufirst%3DS.%26aulast%3DMeijer%26aufirst%3DL.%26aulast%3DKim%26aufirst%3DS.%2BH.%26aulast%3DLockhart%26aufirst%3DD.%2BJ.%26aulast%3DSchultz%26aufirst%3DP.%2BG.%26atitle%3DExploiting%2520chemical%2520libraries%252C%2520structure%252C%2520and%2520genomics%2520in%2520the%2520search%2520for%2520kinase%2520inhibitors%26jtitle%3DScience%26date%3D1998%26volume%3D281%26spage%3D533%26epage%3D538" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group">Legraverend, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grierson, D. S.</span><span> </span><span class="NLM_article-title">The purines: potent and versatile small molecule inhibitors and modulators of key biological targets</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">3987</span><span class="NLM_x">–</span> <span class="NLM_lpage">4006</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2006&pages=3987-4006&author=M.+Legraverendauthor=D.+S.+Grierson&title=The+purines%3A+potent+and+versatile+small+molecule+inhibitors+and+modulators+of+key+biological+targets"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLegraverend%26aufirst%3DM.%26aulast%3DGrierson%26aufirst%3DD.%2BS.%26atitle%3DThe%2520purines%253A%2520potent%2520and%2520versatile%2520small%2520molecule%2520inhibitors%2520and%2520modulators%2520of%2520key%2520biological%2520targets%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2006%26volume%3D14%26spage%3D3987%26epage%3D4006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group">Kadoch, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hargreaves, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hodges, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elias, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ho, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ranish, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crabtree, G. R.</span><span> </span><span class="NLM_article-title">Proteomic and bioinformatic analysis of mammalian SWI/SNF complexes identifies extensive roles in human malignancy</span> <span class="citation_source-journal">Nat. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">592</span><span class="NLM_x">–</span> <span class="NLM_lpage">601</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Fjm501893k&amp;key=10.1038%2Fng.2628" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Fjm501893k&amp;key=23644491" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BC3sXntVart7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2013&pages=592-601&author=C.+Kadochauthor=D.+C.+Hargreavesauthor=C.+Hodgesauthor=L.+Eliasauthor=L.+Hoauthor=J.+Ranishauthor=G.+R.+Crabtree&title=Proteomic+and+bioinformatic+analysis+of+mammalian+SWI%2FSNF+complexes+identifies+extensive+roles+in+human+malignancy"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Proteomic and bioinformatic analysis of mammalian SWI/SNF complexes identifies extensive roles in human malignancy</span></div><div class="casAuthors">Kadoch, Cigall; Hargreaves, Diana C.; Hodges, Courtney; Elias, Laura; Ho, Lena; Ranish, Jeff; Crabtree, Gerald R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">592-601</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Subunits of mammalian SWI/SNF (mSWI/SNF or BAF) complexes have recently been implicated as tumor suppressors in human malignancies.  To understand the full extent of their involvement, we conducted a proteomic anal. of endogenous mSWI/SNF complexes, which identified several new dedicated, stable subunits not found in yeast SWI/SNF complexes, including BCL7A, BCL7B and BCL7C, BCL11A and BCL11B, BRD9 and SS18.  Incorporating these new members, we detd. mSWI/SNF subunit mutation frequency in exome and whole-genome sequencing studies of primary human tumors.  Notably, mSWI/SNF subunits are mutated in 19.6% of all human tumors reported in 44 studies.  Our anal. suggests that specific subunits protect against cancer in specific tissues.  In addn., mutations affecting more than one subunit, defined here as compd. heterozygosity, are prevalent in certain cancers.  Our studies demonstrate that mSWI/SNF is the most frequently mutated chromatin-regulatory complex (CRC) in human cancer, exhibiting a broad mutation pattern, similar to that of TP53.  Thus, proper functioning of polymorphic BAF complexes may constitute a major mechanism of tumor suppression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqP873DCpn7JrVg90H21EOLACvtfcHk0lic-prX7HBkhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXntVart7s%253D&md5=0f49cf3c59e7bc04b74970fe656b1e17</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1038%2Fng.2628&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fng.2628%26sid%3Dliteratum%253Aachs%26aulast%3DKadoch%26aufirst%3DC.%26aulast%3DHargreaves%26aufirst%3DD.%2BC.%26aulast%3DHodges%26aufirst%3DC.%26aulast%3DElias%26aufirst%3DL.%26aulast%3DHo%26aufirst%3DL.%26aulast%3DRanish%26aufirst%3DJ.%26aulast%3DCrabtree%26aufirst%3DG.%2BR.%26atitle%3DProteomic%2520and%2520bioinformatic%2520analysis%2520of%2520mammalian%2520SWI%252FSNF%2520complexes%2520identifies%2520extensive%2520roles%2520in%2520human%2520malignancy%26jtitle%3DNat.%2520Genet.%26date%3D2013%26volume%3D45%26spage%3D592%26epage%3D601" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group">Kang, J. U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koo, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kwon, K. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, J. M.</span><span> </span><span class="NLM_article-title">Gain at chromosomal region 5p15.33, containing TERT, is the most frequent genetic event in early stages of non-small cell lung cancer</span> <span class="citation_source-journal">Cancer Genet. Cytogenet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">182</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">11</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Fjm501893k&amp;key=10.1016%2Fj.cancergencyto.2007.12.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Fjm501893k&amp;key=18328944" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjtVWmt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=182&publication_year=2008&pages=1-11&author=J.+U.+Kangauthor=S.+H.+Kooauthor=K.+C.+Kwonauthor=J.+W.+Parkauthor=J.+M.+Kim&title=Gain+at+chromosomal+region+5p15.33%2C+containing+TERT%2C+is+the+most+frequent+genetic+event+in+early+stages+of+non-small+cell+lung+cancer"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Gain at chromosomal region 5p15.33, containing TERT, is the most frequent genetic event in early stages of non-small cell lung cancer</span></div><div class="casAuthors">Kang, Ji Un; Koo, Sun Hoe; Kwon, Kye Chul; Park, Jong Woo; Kim, Jin Man</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Genetics and Cytogenetics</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">182</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-11</span>CODEN:
                <span class="NLM_cas:coden">CGCYDF</span>;
        ISSN:<span class="NLM_cas:issn">0165-4608</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Chromosomal imbalances resulting in altered gene dosage play a role in the mol. pathogenesis of non-small cell lung cancer (NSCLC), but the target genes remain to be identified.  To identify early-stage genetic events that drive progression of NSCLC, we conducted a high-resoln. array comparative genomic hybridization (CGH) study, using an array of 4046 bacterial artificial chromosome clones to screen for DNA copy no. changes assocd. with individual genes in 36 tumors obtained from patients in early stages of NSCLC.  Multiple early genetic events occurring on chromosome 5p were identified, with a minimal detection region at 5p15.33∼12.  The most frequent finding involved gain of 5p15.33, obsd. in 15 of 19 stage I (A+B) cancers (79%) and in 28 of the total 36 NSCLC cases (78%).  This locus harbors the genes TERT, SLC6A19, and SLC6A18 and is a telomeric boundary at bacterial artificial chromosome (BAC) clone 91_J20.  Other potential candidate genes evidencing high nos. of genomic copy no. changes (≥40% of patients) included the following genes, encountered in >50% of 19 stage I (A+B) cancers: CEP72 and TPPP (14 of 19; 74%); AHRR, EXOC3 (previously SEC6L1), SLC9A3, LOC442126, ZDHHC11, BRD9, and TRIP13 (13/19; 68%); and CLPTM1L (alias CRR9), SLC6A3 (previously DAT1), and LOC401169 (10/19; 53%).  Fluorescence in situ hybridization validated the array CGH findings.  The gain of 5p15.33 is thus one of the most consistent alterations in the early stages of lung cancer, and a series of genes in the crit. 5p15.33 region may be used as novel biomarkers for the early detection and classification of lung cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvKwLSWMQRnbVg90H21EOLACvtfcHk0lic-prX7HBkhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjtVWmt78%253D&md5=514aec6952253ed612003471df40a4ba</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1016%2Fj.cancergencyto.2007.12.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cancergencyto.2007.12.004%26sid%3Dliteratum%253Aachs%26aulast%3DKang%26aufirst%3DJ.%2BU.%26aulast%3DKoo%26aufirst%3DS.%2BH.%26aulast%3DKwon%26aufirst%3DK.%2BC.%26aulast%3DPark%26aufirst%3DJ.%2BW.%26aulast%3DKim%26aufirst%3DJ.%2BM.%26atitle%3DGain%2520at%2520chromosomal%2520region%25205p15.33%252C%2520containing%2520TERT%252C%2520is%2520the%2520most%2520frequent%2520genetic%2520event%2520in%2520early%2520stages%2520of%2520non-small%2520cell%2520lung%2520cancer%26jtitle%3DCancer%2520Genet.%2520Cytogenet.%26date%3D2008%26volume%3D182%26spage%3D1%26epage%3D11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group">Scotto, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Narayan, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nandula, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Subramaniyam, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaufmann, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wright, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pothuri, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mansukhani, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schneider, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arias-Pulido, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murty, V. V.</span><span> </span><span class="NLM_article-title">Integrative genomics analysis of chromosome 5p gain in cervical cancer reveals target over-expressed genes, including Drosha</span> <span class="citation_source-journal">Mol. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">58</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Fjm501893k&amp;key=10.1186%2F1476-4598-7-58" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Fjm501893k&amp;key=18559093" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A280%3ADC%252BD1cvgvFGktw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2008&pages=58&author=L.+Scottoauthor=G.+Narayanauthor=S.+V.+Nandulaauthor=S.+Subramaniyamauthor=A.+M.+Kaufmannauthor=J.+D.+Wrightauthor=B.+Pothuriauthor=M.+Mansukhaniauthor=A.+Schneiderauthor=H.+Arias-Pulidoauthor=V.+V.+Murty&title=Integrative+genomics+analysis+of+chromosome+5p+gain+in+cervical+cancer+reveals+target+over-expressed+genes%2C+including+Drosha"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Integrative genomics analysis of chromosome 5p gain in cervical cancer reveals target over-expressed genes, including Drosha</span></div><div class="casAuthors">Scotto Luigi; Narayan Gopeshwar; Nandula Subhadra V; Subramaniyam Shivakumar; Kaufmann Andreas M; Wright Jason D; Pothuri Bhavana; Mansukhani Mahesh; Schneider Achim; Arias-Pulido Hugo; Murty Vundavalli V</div><div class="citationInfo"><span class="NLM_cas:title">Molecular cancer</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">58</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Copy number gains and amplifications are characteristic feature of cervical cancer (CC) genomes for which the underlying mechanisms are unclear.  These changes may possess oncogenic properties by deregulating tumor-related genes.  Gain of short arm of chromosome 5 (5p) is the most frequent karyotypic change in CC.  METHODS:  To examine the role of 5p gain, we performed a combination of single nucleotide polymorphism (SNP) array, fluorescence in situ hybridization (FISH), and gene expression analyses on invasive cancer and in various stages of CC progression.  RESULTS:  The SNP and FISH analyses revealed copy number increase (CNI) of 5p in 63% of invasive CC, which arises at later stages of precancerous lesions in CC development.  We integrated chromosome 5 genomic copy number and gene expression data to identify key target over expressed genes as a consequence of 5p gain.  One of the candidates identified was Drosha (RNASEN), a gene that is required in the first step of microRNA (miRNA) processing in the nucleus.  Other 5p genes identified as targets of CNI play a role in DNA repair and cell cycle regulation (BASP1, TARS, PAIP1, BRD9, RAD1, SKP2, and POLS), signal transduction (OSMR), and mitochondrial oxidative phosphorylation (NNT, SDHA, and NDUFS6), suggesting that disruption of pathways involving these genes may contribute to CC progression.  CONCLUSION:  Taken together, we demonstrate the power of integrating genomics data with expression data in deciphering tumor-related targets of CNI.  Identification of 5p gene targets in CC denotes an important step towards biomarker development and forms a framework for testing as molecular therapeutic targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTJ-og-xr-IWuxsqfvU0ziafW6udTcc2eYhMUJF1voqb7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1cvgvFGktw%253D%253D&md5=9bb25d18ffeaebf79760b3b760c91e15</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1186%2F1476-4598-7-58&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1476-4598-7-58%26sid%3Dliteratum%253Aachs%26aulast%3DScotto%26aufirst%3DL.%26aulast%3DNarayan%26aufirst%3DG.%26aulast%3DNandula%26aufirst%3DS.%2BV.%26aulast%3DSubramaniyam%26aufirst%3DS.%26aulast%3DKaufmann%26aufirst%3DA.%2BM.%26aulast%3DWright%26aufirst%3DJ.%2BD.%26aulast%3DPothuri%26aufirst%3DB.%26aulast%3DMansukhani%26aufirst%3DM.%26aulast%3DSchneider%26aufirst%3DA.%26aulast%3DArias-Pulido%26aufirst%3DH.%26aulast%3DMurty%26aufirst%3DV.%2BV.%26atitle%3DIntegrative%2520genomics%2520analysis%2520of%2520chromosome%25205p%2520gain%2520in%2520cervical%2520cancer%2520reveals%2520target%2520over-expressed%2520genes%252C%2520including%2520Drosha%26jtitle%3DMol.%2520Cancer%26date%3D2008%26volume%3D7%26spage%3D58" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group">Cleary, S. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeck, W. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Selitsky, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Savich, G. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tan, T. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Getz, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lawrence, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parker, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Powers, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fischer, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guindi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ghanekar, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chiang, D. Y.</span><span> </span><span class="NLM_article-title">Identification of driver genes in hepatocellular carcinoma by exome sequencing</span> <span class="citation_source-journal">Hepatology</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">1693</span><span class="NLM_x">–</span> <span class="NLM_lpage">1702</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2013&pages=1693-1702&author=S.+P.+Clearyauthor=W.+R.+Jeckauthor=X.+Zhaoauthor=K.+Chenauthor=S.+R.+Selitskyauthor=G.+L.+Savichauthor=T.+X.+Tanauthor=M.+C.+Wuauthor=G.+Getzauthor=M.+S.+Lawrenceauthor=J.+S.+Parkerauthor=J.+Liauthor=S.+Powersauthor=H.+Kimauthor=S.+Fischerauthor=M.+Guindiauthor=A.+Ghanekarauthor=D.+Y.+Chiang&title=Identification+of+driver+genes+in+hepatocellular+carcinoma+by+exome+sequencing"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCleary%26aufirst%3DS.%2BP.%26aulast%3DJeck%26aufirst%3DW.%2BR.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DSelitsky%26aufirst%3DS.%2BR.%26aulast%3DSavich%26aufirst%3DG.%2BL.%26aulast%3DTan%26aufirst%3DT.%2BX.%26aulast%3DWu%26aufirst%3DM.%2BC.%26aulast%3DGetz%26aufirst%3DG.%26aulast%3DLawrence%26aufirst%3DM.%2BS.%26aulast%3DParker%26aufirst%3DJ.%2BS.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DPowers%26aufirst%3DS.%26aulast%3DKim%26aufirst%3DH.%26aulast%3DFischer%26aufirst%3DS.%26aulast%3DGuindi%26aufirst%3DM.%26aulast%3DGhanekar%26aufirst%3DA.%26aulast%3DChiang%26aufirst%3DD.%2BY.%26atitle%3DIdentification%2520of%2520driver%2520genes%2520in%2520hepatocellular%2520carcinoma%2520by%2520exome%2520sequencing%26jtitle%3DHepatology%26date%3D2013%26volume%3D58%26spage%3D1693%26epage%3D1702" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group">The Cancer Genome Atlas Research Network</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kandoth, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schultz, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cherniack, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Akbani, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robertson, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pashtan, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benz, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yau, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Laird, P. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ding, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mills, G. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kucherlapati, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mardis, E. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levine, D. A.</span><span> </span><span class="NLM_article-title">Integrated genomic characterization of endometrial carcinoma</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">497</span><span class="NLM_x">, </span> <span class="NLM_fpage">67</span><span class="NLM_x">–</span> <span class="NLM_lpage">73</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=497&publication_year=2013&pages=67-73&author=The+Cancer+Genome+Atlas+Research+Networkauthor=C.+Kandothauthor=N.+Schultzauthor=A.+D.+Cherniackauthor=R.+Akbaniauthor=Y.+Liuauthor=H.+Shenauthor=A.+G.+Robertsonauthor=I.+Pashtanauthor=R.+Shenauthor=C.+C.+Benzauthor=C.+Yauauthor=P.+W.+Lairdauthor=L.+Dingauthor=W.+Zhangauthor=G.+B.+Millsauthor=R.+Kucherlapatiauthor=E.+R.+Mardisauthor=D.+A.+Levine&title=Integrated+genomic+characterization+of+endometrial+carcinoma"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3D%26aulast%3DKandoth%26aufirst%3DC.%26aulast%3DSchultz%26aufirst%3DN.%26aulast%3DCherniack%26aufirst%3DA.%2BD.%26aulast%3DAkbani%26aufirst%3DR.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DShen%26aufirst%3DH.%26aulast%3DRobertson%26aufirst%3DA.%2BG.%26aulast%3DPashtan%26aufirst%3DI.%26aulast%3DShen%26aufirst%3DR.%26aulast%3DBenz%26aufirst%3DC.%2BC.%26aulast%3DYau%26aufirst%3DC.%26aulast%3DLaird%26aufirst%3DP.%2BW.%26aulast%3DDing%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DMills%26aufirst%3DG.%2BB.%26aulast%3DKucherlapati%26aufirst%3DR.%26aulast%3DMardis%26aufirst%3DE.%2BR.%26aulast%3DLevine%26aufirst%3DD.%2BA.%26atitle%3DIntegrated%2520genomic%2520characterization%2520of%2520endometrial%2520carcinoma%26jtitle%3DNature%26date%3D2013%26volume%3D497%26spage%3D67%26epage%3D73" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group">The Cancer Genome Atlas Research Network.</span><span> </span><span class="NLM_article-title">Comprehensive genomic characterization of squamous cell lung cancers</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">489</span><span class="NLM_x">, </span> <span class="NLM_fpage">519</span><span class="NLM_x">–</span> <span class="NLM_lpage">525</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=489&publication_year=2012&pages=519-525&author=The+Cancer+Genome+Atlas+Research+Network.&title=Comprehensive+genomic+characterization+of+squamous+cell+lung+cancers"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DComprehensive%2520genomic%2520characterization%2520of%2520squamous%2520cell%2520lung%2520cancers%26jtitle%3DNature%26date%3D2012%26volume%3D489%26spage%3D519%26epage%3D525" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group">Barbieri, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baca, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lawrence, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Demichelis, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blattner, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Theurillat, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">White, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stojanov, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Allen, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stransky, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nickerson, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chae, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boysen, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Auclair, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Onofrio, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kitabayashi, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacDonald, T. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sheikh, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vuong, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guiducci, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cibulskis, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sivachenko, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carter, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saksena, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Voet, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hussain, W. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramos, A. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winckler, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Redman, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ardlie, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tewari, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mosquera, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rupp, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wild, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moch, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morrissey, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nelson, P. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kantoff, P. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gabriel, S. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Golub, T. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meyerson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lander, E. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Getz, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rubin, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garraway, L. A.</span><span> </span><span class="NLM_article-title">Exome sequencing identifies recurrent SPOP, FOXA1 and MED12 mutations in prostate cancer</span> <span class="citation_source-journal">Nat. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">685</span><span class="NLM_x">–</span> <span class="NLM_lpage">689</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Fjm501893k&amp;key=10.1038%2Fng.2279" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Fjm501893k&amp;key=22610119" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BC38XntF2nt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2012&pages=685-689&author=C.+E.+Barbieriauthor=S.+C.+Bacaauthor=M.+S.+Lawrenceauthor=F.+Demichelisauthor=M.+Blattnerauthor=J.+P.+Theurillatauthor=T.+A.+Whiteauthor=P.+Stojanovauthor=E.+Van+Allenauthor=N.+Stranskyauthor=E.+Nickersonauthor=S.+S.+Chaeauthor=G.+Boysenauthor=D.+Auclairauthor=R.+C.+Onofrioauthor=K.+Parkauthor=N.+Kitabayashiauthor=T.+Y.+MacDonaldauthor=K.+Sheikhauthor=T.+Vuongauthor=C.+Guiducciauthor=K.+Cibulskisauthor=A.+Sivachenkoauthor=S.+L.+Carterauthor=G.+Saksenaauthor=D.+Voetauthor=W.+M.+Hussainauthor=A.+H.+Ramosauthor=W.+Wincklerauthor=M.+C.+Redmanauthor=K.+Ardlieauthor=A.+K.+Tewariauthor=J.+M.+Mosqueraauthor=N.+Ruppauthor=P.+J.+Wildauthor=H.+Mochauthor=C.+Morrisseyauthor=P.+S.+Nelsonauthor=P.+W.+Kantoffauthor=S.+B.+Gabrielauthor=T.+R.+Golubauthor=M.+Meyersonauthor=E.+S.+Landerauthor=G.+Getzauthor=M.+A.+Rubinauthor=L.+A.+Garraway&title=Exome+sequencing+identifies+recurrent+SPOP%2C+FOXA1+and+MED12+mutations+in+prostate+cancer"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Exome sequencing identifies recurrent SPOP, FOXA1 and MED12 mutations in prostate cancer</span></div><div class="casAuthors">Barbieri, Christopher E.; Baca, Sylvan C.; Lawrence, Michael S.; Demichelis, Francesca; Blattner, Mirjam; Theurillat, Jean-Philippe; White, Thomas A.; Stojanov, Petar; Van Allen, Eliezer; Stransky, Nicolas; Nickerson, Elizabeth; Chae, Sung-Suk; Boysen, Gunther; Auclair, Daniel; Onofrio, Robert C.; Park, Kyung; Kitabayashi, Naoki; MacDonald, Theresa Y.; Sheikh, Karen; Vuong, Terry; Guiducci, Candace; Cibulskis, Kristian; Sivachenko, Andrey; Carter, Scott L.; Saksena, Gordon; Voet, Douglas; Hussain, Wasay M.; Ramos, Alex H.; Winckler, Wendy; Redman, Michelle C.; Ardlie, Kristin; Tewari, Ashutosh K.; Mosquera, Juan Miguel; Rupp, Niels; Wild, Peter J.; Moch, Holger; Morrissey, Colm; Nelson, Peter S.; Kantoff, Philip W.; Gabriel, Stacey B.; Golub, Todd R.; Meyerson, Matthew; Lander, Eric S.; Getz, Gad; Rubin, Mark A.; Garraway, Levi A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">685-689</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Prostate cancer is the second most common cancer in men worldwide and causes over 250,000 deaths each year.  Overtreatment of indolent disease also results in significant morbidity.  Common genetic alterations in prostate cancer include losses of NKX3.1 (8p21) and PTEN (10q23), gains of AR (the androgen receptor gene) and fusion of ETS family transcription factor genes with androgen-responsive promoters.  Recurrent somatic base-pair substitutions are believed to be less contributory in prostate tumorigenesis but have not been systematically analyzed in large cohorts.  Here, we sequenced the exomes of 112 prostate tumor and normal tissue pairs.  New recurrent mutations were identified in multiple genes, including MED12 and FOXA1.  SPOP was the most frequently mutated gene, with mutation.s involving the SPOP substrate-binding cleft in 6-15% of tumors across multiple independent cohorts.  Prostate cancers with mutant SPOP lacked ETS family gene rearrangements and showed a distinct pattern of genomic alterations.  Thus, SPOP mutations may define a new mol. subtype of prostate cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6_0sm909vLbVg90H21EOLACvtfcHk0ljXkKGwE9K-KA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XntF2nt7w%253D&md5=b27e8ce7bdfa0cd142546dab6b521f6b</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1038%2Fng.2279&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fng.2279%26sid%3Dliteratum%253Aachs%26aulast%3DBarbieri%26aufirst%3DC.%2BE.%26aulast%3DBaca%26aufirst%3DS.%2BC.%26aulast%3DLawrence%26aufirst%3DM.%2BS.%26aulast%3DDemichelis%26aufirst%3DF.%26aulast%3DBlattner%26aufirst%3DM.%26aulast%3DTheurillat%26aufirst%3DJ.%2BP.%26aulast%3DWhite%26aufirst%3DT.%2BA.%26aulast%3DStojanov%26aufirst%3DP.%26aulast%3DVan%2BAllen%26aufirst%3DE.%26aulast%3DStransky%26aufirst%3DN.%26aulast%3DNickerson%26aufirst%3DE.%26aulast%3DChae%26aufirst%3DS.%2BS.%26aulast%3DBoysen%26aufirst%3DG.%26aulast%3DAuclair%26aufirst%3DD.%26aulast%3DOnofrio%26aufirst%3DR.%2BC.%26aulast%3DPark%26aufirst%3DK.%26aulast%3DKitabayashi%26aufirst%3DN.%26aulast%3DMacDonald%26aufirst%3DT.%2BY.%26aulast%3DSheikh%26aufirst%3DK.%26aulast%3DVuong%26aufirst%3DT.%26aulast%3DGuiducci%26aufirst%3DC.%26aulast%3DCibulskis%26aufirst%3DK.%26aulast%3DSivachenko%26aufirst%3DA.%26aulast%3DCarter%26aufirst%3DS.%2BL.%26aulast%3DSaksena%26aufirst%3DG.%26aulast%3DVoet%26aufirst%3DD.%26aulast%3DHussain%26aufirst%3DW.%2BM.%26aulast%3DRamos%26aufirst%3DA.%2BH.%26aulast%3DWinckler%26aufirst%3DW.%26aulast%3DRedman%26aufirst%3DM.%2BC.%26aulast%3DArdlie%26aufirst%3DK.%26aulast%3DTewari%26aufirst%3DA.%2BK.%26aulast%3DMosquera%26aufirst%3DJ.%2BM.%26aulast%3DRupp%26aufirst%3DN.%26aulast%3DWild%26aufirst%3DP.%2BJ.%26aulast%3DMoch%26aufirst%3DH.%26aulast%3DMorrissey%26aufirst%3DC.%26aulast%3DNelson%26aufirst%3DP.%2BS.%26aulast%3DKantoff%26aufirst%3DP.%2BW.%26aulast%3DGabriel%26aufirst%3DS.%2BB.%26aulast%3DGolub%26aufirst%3DT.%2BR.%26aulast%3DMeyerson%26aufirst%3DM.%26aulast%3DLander%26aufirst%3DE.%2BS.%26aulast%3DGetz%26aufirst%3DG.%26aulast%3DRubin%26aufirst%3DM.%2BA.%26aulast%3DGarraway%26aufirst%3DL.%2BA.%26atitle%3DExome%2520sequencing%2520identifies%2520recurrent%2520SPOP%252C%2520FOXA1%2520and%2520MED12%2520mutations%2520in%2520prostate%2520cancer%26jtitle%3DNat.%2520Genet.%26date%3D2012%26volume%3D44%26spage%3D685%26epage%3D689" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group">Bosshard, H. R.</span><span> </span><span class="NLM_article-title">Molecular recognition by induced fit: How fit is the concept?</span> <span class="citation_source-journal">News Physiol. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">171</span><span class="NLM_x">–</span> <span class="NLM_lpage">173</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Fjm501893k&amp;key=11479367" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BD3MXntVahs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2001&pages=171-173&author=H.+R.+Bosshard&title=Molecular+recognition+by+induced+fit%3A+How+fit+is+the+concept%3F"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular recognition by induced fit: how fit is the concept?</span></div><div class="casAuthors">Bosshard, Hans Rudolf</div><div class="citationInfo"><span class="NLM_cas:title">News in Physiological Sciences</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">Aug.</span>),
    <span class="NLM_cas:pages">171-174</span>CODEN:
                <span class="NLM_cas:coden">NEPSEY</span>;
        ISSN:<span class="NLM_cas:issn">0886-1714</span>.
    
            (<span class="NLM_cas:orgname">International Union of Physiological Sciences</span>)
        </div><div class="casAbstract">A review.  Induced fit explains why biomols. can bind together even if they are not optimized for binding.  However, induced fit can lead to a kinetic bottleneck and does not describe every interaction in the absence of prior complementarity.  Preselection of a fitting conformer is an alternative to induced fit.  Topics discussed include the expt. demonstration of binding by conformational selection and the energy landscape model of protein conformation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwY5pcaD1H1rVg90H21EOLACvtfcHk0ljVcdtJgNtCrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXntVahs7k%253D&md5=033a7cd9621a8cec3b481014455e9cef</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBosshard%26aufirst%3DH.%2BR.%26atitle%3DMolecular%2520recognition%2520by%2520induced%2520fit%253A%2520How%2520fit%2520is%2520the%2520concept%253F%26jtitle%3DNews%2520Physiol.%2520Sci.%26date%3D2001%26volume%3D16%26spage%3D171%26epage%3D173" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group">Rooney, T. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Picaud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cortopassi, W. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hay, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tumber, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rogers, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Philpott, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thompson, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heightman, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pryde, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cook, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paton, R. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muller, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brennan, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Conway, S. J.</span><span> </span><span class="NLM_article-title">A series of potent CREBBP bromodomain ligands reveals an induced-fit pocket stabilized by a cation-pi interaction</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">6126</span><span class="NLM_x">–</span> <span class="NLM_lpage">6130</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2014&pages=6126-6130&author=T.+P.+Rooneyauthor=P.+Filippakopoulosauthor=O.+Fedorovauthor=S.+Picaudauthor=W.+A.+Cortopassiauthor=D.+A.+Hayauthor=S.+Martinauthor=A.+Tumberauthor=C.+M.+Rogersauthor=M.+Philpottauthor=M.+Wangauthor=A.+L.+Thompsonauthor=T.+D.+Heightmanauthor=D.+C.+Prydeauthor=A.+Cookauthor=R.+S.+Patonauthor=S.+Mullerauthor=S.+Knappauthor=P.+E.+Brennanauthor=S.+J.+Conway&title=A+series+of+potent+CREBBP+bromodomain+ligands+reveals+an+induced-fit+pocket+stabilized+by+a+cation-pi+interaction"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRooney%26aufirst%3DT.%2BP.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DCortopassi%26aufirst%3DW.%2BA.%26aulast%3DHay%26aufirst%3DD.%2BA.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DTumber%26aufirst%3DA.%26aulast%3DRogers%26aufirst%3DC.%2BM.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DThompson%26aufirst%3DA.%2BL.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DPryde%26aufirst%3DD.%2BC.%26aulast%3DCook%26aufirst%3DA.%26aulast%3DPaton%26aufirst%3DR.%2BS.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DConway%26aufirst%3DS.%2BJ.%26atitle%3DA%2520series%2520of%2520potent%2520CREBBP%2520bromodomain%2520ligands%2520reveals%2520an%2520induced-fit%2520pocket%2520stabilized%2520by%2520a%2520cation-pi%2520interaction%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2014%26volume%3D53%26spage%3D6126%26epage%3D6130" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group">Friesner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banks, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murphy, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Halgren, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klicic, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mainz, D. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Repasky, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knoll, E. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shelley, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perry, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Francis, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shenkin, P. S.</span><span> </span><span class="NLM_article-title">Glide: a new approach for rapid, accurate docking and scoring. 1. Method and assessment of docking accuracy</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">1739</span><span class="NLM_x">–</span> <span class="NLM_lpage">1749</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm0306430" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1739-1749&author=R.+A.+Friesnerauthor=J.+L.+Banksauthor=R.+B.+Murphyauthor=T.+A.+Halgrenauthor=J.+J.+Klicicauthor=D.+T.+Mainzauthor=M.+P.+Repaskyauthor=E.+H.+Knollauthor=M.+Shelleyauthor=J.+K.+Perryauthor=D.+E.+Shawauthor=P.+Francisauthor=P.+S.+Shenkin&title=Glide%3A+a+new+approach+for+rapid%2C+accurate+docking+and+scoring.+1.+Method+and+assessment+of+docking+accuracy"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 1. method and assessment of docking accuracy</span></div><div class="casAuthors">Friesner, Richard A.; Banks, Jay L.; Murphy, Robert B.; Halgren, Thomas A.; Klicic, Jasna J.; Mainz, Daniel T.; Repasky, Matthew P.; Knoll, Eric H.; Shelley, Mee; Perry, Jason K.; Shaw, David E.; Francis, Perry; Shenkin, Peter S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1739-1749</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Unlike other methods for docking ligands to the rigid 3D structure of a known protein receptor, Glide approximates a complete systematic search of the conformational, orientational, and positional space of the docked ligand.  In this search, an initial rough positioning and scoring phase that dramatically narrows the search space is followed by torsionally flexible energy optimization on an OPLS-AA nonbonded potential grid for a few hundred surviving candidate poses.  The very best candidates are further refined via a Monte Carlo sampling of pose conformation; in some cases, this is crucial to obtaining an accurate docked pose.  Selection of the best docked pose uses a model energy function that combines empirical and force-field-based terms.  Docking accuracy is assessed by redocking ligands from 282 cocrystd. PDB complexes starting from conformationally optimized ligand geometries that bear no memory of the correctly docked pose.  Errors in geometry for the top-ranked pose are less than 1 Å in nearly half of the cases and are greater than 2 Å in only about one-third of them.  Comparisons to published data on rms deviations show that Glide is nearly twice as accurate as GOLD and more than twice as accurate as FlexX for ligands having up to 20 rotatable bonds.  Glide is also found to be more accurate than the recently described Surflex method.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPuigu31qMMLVg90H21EOLACvtfcHk0ljVcdtJgNtCrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D&md5=8cc2f0022318b12dd972e9c493375bf9</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1021%2Fjm0306430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0306430%26sid%3Dliteratum%253Aachs%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DKlicic%26aufirst%3DJ.%2BJ.%26aulast%3DMainz%26aufirst%3DD.%2BT.%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DKnoll%26aufirst%3DE.%2BH.%26aulast%3DShelley%26aufirst%3DM.%26aulast%3DPerry%26aufirst%3DJ.%2BK.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26aulast%3DFrancis%26aufirst%3DP.%26aulast%3DShenkin%26aufirst%3DP.%2BS.%26atitle%3DGlide%253A%2520a%2520new%2520approach%2520for%2520rapid%252C%2520accurate%2520docking%2520and%2520scoring.%25201.%2520Method%2520and%2520assessment%2520of%2520docking%2520accuracy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1739%26epage%3D1749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group">Halgren, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murphy, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friesner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beard, H. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, L. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollard, W. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banks, J. L.</span><span> </span><span class="NLM_article-title">Glide: a new approach for rapid, accurate docking and scoring. 2. Enrichment factors in database screening</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">1750</span><span class="NLM_x">–</span> <span class="NLM_lpage">1759</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm030644s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1750-1759&author=T.+A.+Halgrenauthor=R.+B.+Murphyauthor=R.+A.+Friesnerauthor=H.+S.+Beardauthor=L.+L.+Fryeauthor=W.+T.+Pollardauthor=J.+L.+Banks&title=Glide%3A+a+new+approach+for+rapid%2C+accurate+docking+and+scoring.+2.+Enrichment+factors+in+database+screening"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 2. Enrichment factors in database screening</span></div><div class="casAuthors">Halgren, Thomas A.; Murphy, Robert B.; Friesner, Richard A.; Beard, Hege S.; Frye, Leah L.; Pollard, W. Thomas; Banks, Jay L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1750-1759</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Glide's ability to identify active compds. in a database screen is characterized by applying Glide to a diverse set of nine protein receptors.  In many cases, two, or even three, protein sites are employed to probe the sensitivity of the results to the site geometry.  To make the database screens as realistic as possible, the screens use sets of "druglike" decoy ligands that have been selected to be representative of what we believe is likely to be found in the compd. collection of a pharmaceutical or biotechnol. company.  Results are presented for releases 1.8, 2.0, and 2.5 of Glide.  The comparisons show that av. measures for both "early" and "global" enrichment for Glide 2.5 are 3 times higher than for Glide 1.8 and more than 2 times higher than for Glide 2.0 because of better results for the least well-handled screens.  This improvement in enrichment stems largely from the better balance of the more widely parametrized GlideScore 2.5 function and the inclusion of terms that penalize ligand-protein interactions that violate established principles of phys. chem., particularly as it concerns the exposure to solvent of charged protein and ligand groups.  Comparisons to results for the thymidine kinase and estrogen receptors published by Rognan and co-workers (J. Med. Chem. 2000, 43, 4759-4767) show that Glide 2.5 performs better than GOLD 1.1, FlexX 1.8, or DOCK 4.01.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoT2ptubbKbvLVg90H21EOLACvtfcHk0ljp-6rfTjez1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D&md5=33d68dd968e65626b449df61e44e37be</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1021%2Fjm030644s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030644s%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBeard%26aufirst%3DH.%2BS.%26aulast%3DFrye%26aufirst%3DL.%2BL.%26aulast%3DPollard%26aufirst%3DW.%2BT.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26atitle%3DGlide%253A%2520a%2520new%2520approach%2520for%2520rapid%252C%2520accurate%2520docking%2520and%2520scoring.%25202.%2520Enrichment%2520factors%2520in%2520database%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1750%26epage%3D1759" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group">Friesner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murphy, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Repasky, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, L. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greenwood, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Halgren, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanschagrin, P. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mainz, D. T.</span><span> </span><span class="NLM_article-title">Extra precision glide: docking and scoring incorporating a model of hydrophobic enclosure for protein–ligand complexes</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">6177</span><span class="NLM_x">–</span> <span class="NLM_lpage">6196</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm051256o" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=6177-6196&author=R.+A.+Friesnerauthor=R.+B.+Murphyauthor=M.+P.+Repaskyauthor=L.+L.+Fryeauthor=J.+R.+Greenwoodauthor=T.+A.+Halgrenauthor=P.+C.+Sanschagrinauthor=D.+T.+Mainz&title=Extra+precision+glide%3A+docking+and+scoring+incorporating+a+model+of+hydrophobic+enclosure+for+protein%E2%80%93ligand+complexes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1021%2Fjm051256o&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm051256o%26sid%3Dliteratum%253Aachs%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DFrye%26aufirst%3DL.%2BL.%26aulast%3DGreenwood%26aufirst%3DJ.%2BR.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DSanschagrin%26aufirst%3DP.%2BC.%26aulast%3DMainz%26aufirst%3DD.%2BT.%26atitle%3DExtra%2520precision%2520glide%253A%2520docking%2520and%2520scoring%2520incorporating%2520a%2520model%2520of%2520hydrophobic%2520enclosure%2520for%2520protein%25E2%2580%2593ligand%2520complexes%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D6177%26epage%3D6196" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group">Sherman, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Day, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobson, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friesner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Farid, R.</span><span> </span><span class="NLM_article-title">Novel procedure for modeling ligand/receptor induced fit effects</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">534</span><span class="NLM_x">–</span> <span class="NLM_lpage">553</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm050540c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlCgsr7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=534-553&author=W.+Shermanauthor=T.+Dayauthor=M.+P.+Jacobsonauthor=R.+A.+Friesnerauthor=R.+Farid&title=Novel+procedure+for+modeling+ligand%2Freceptor+induced+fit+effects"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Procedure for Modeling Ligand/Receptor Induced Fit Effects</span></div><div class="casAuthors">Sherman, Woody; Day, Tyler; Jacobson, Matthew P.; Friesner, Richard A.; Farid, Ramy</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">534-553</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We present a novel protein-ligand docking method that accurately accounts for both ligand and receptor flexibility by iteratively combining rigid receptor docking (Glide) with protein structure prediction (Prime) techniques.  While traditional rigid-receptor docking methods are useful when the receptor structure does not change substantially upon ligand binding, success is limited when the protein must be "induced" into the correct binding conformation for a given ligand.  We provide an in-depth description of our novel methodol. and present results for 21 pharmaceutically relevant examples.  Traditional rigid-receptor docking for these 21 cases yields an av. RMSD of 5.5 Å.  The av. ligand RMSD for docking to a flexible receptor for the 21 pairs is 1.4 Å; the RMSD is ≤1.8 Å for 18 of the cases.  For the three cases with RMSDs greater than 1.8 Å, the core of the ligand is properly docked and all key protein/ligand interactions are captured.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqm3E6ctnPGSrVg90H21EOLACvtfcHk0ljp-6rfTjez1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtlCgsr7I&md5=388811ead5cee1fd460951263de486cb</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1021%2Fjm050540c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm050540c%26sid%3Dliteratum%253Aachs%26aulast%3DSherman%26aufirst%3DW.%26aulast%3DDay%26aufirst%3DT.%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DFarid%26aufirst%3DR.%26atitle%3DNovel%2520procedure%2520for%2520modeling%2520ligand%252Freceptor%2520induced%2520fit%2520effects%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D534%26epage%3D553" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group">Sherman, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beard, H. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Farid, R.</span><span> </span><span class="NLM_article-title">Use of an induced fit receptor structure in virtual screening</span> <span class="citation_source-journal">Chem. Biol. Drug Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">67</span><span class="NLM_x">, </span> <span class="NLM_fpage">83</span><span class="NLM_x">–</span> <span class="NLM_lpage">84</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Fjm501893k&amp;key=10.1111%2Fj.1747-0285.2005.00327.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Fjm501893k&amp;key=16492153" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BD28XhsVSjtL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2006&pages=83-84&author=W.+Shermanauthor=H.+S.+Beardauthor=R.+Farid&title=Use+of+an+induced+fit+receptor+structure+in+virtual+screening"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Use of an induced fit receptor structure in virtual screening</span></div><div class="casAuthors">Sherman, Woody; Beard, Hege S.; Farid, Ramy</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">83-84</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">A review.  The automated induced fit docking protocol was used to generate the DFG-out conformation from a p38 MAP kinase activation loop starting from a DFG-in structure (1a9u) and the ligand from 1kv1 (BMU).  In a virtual screening study of 25K decoy ligands and 46 known actives, using an ensemble consisting of the induced fit docking structure (DFG-out) and the 1a9u crystal structure (DFG-in), 14 actives were identified in the top 1% of the database, including BMU and BIRB 796.  3 Actives were identified when 1a9u was used alone.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7PRgtFzy8u7Vg90H21EOLACvtfcHk0ljp-6rfTjez1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhsVSjtL0%253D&md5=efe9900aacaaea6af805c1cbdb350c73</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1111%2Fj.1747-0285.2005.00327.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1747-0285.2005.00327.x%26sid%3Dliteratum%253Aachs%26aulast%3DSherman%26aufirst%3DW.%26aulast%3DBeard%26aufirst%3DH.%2BS.%26aulast%3DFarid%26aufirst%3DR.%26atitle%3DUse%2520of%2520an%2520induced%2520fit%2520receptor%2520structure%2520in%2520virtual%2520screening%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2006%26volume%3D67%26spage%3D83%26epage%3D84" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group">Hocek, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holy, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Votruba, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dvorakova, H.</span><span> </span><span class="NLM_article-title">Synthesis and cytostatic activity of substituted 6-phenylpurine bases and nucleosides: application of the Suzuki–Miyaura cross-coupling reactions of 6-chloropurine derivatives with phenylboronic acids</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">43</span><span class="NLM_x">, </span> <span class="NLM_fpage">1817</span><span class="NLM_x">–</span> <span class="NLM_lpage">1825</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm991167%2B" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2000&pages=1817-1825&author=M.+Hocekauthor=A.+Holyauthor=I.+Votrubaauthor=H.+Dvorakova&title=Synthesis+and+cytostatic+activity+of+substituted+6-phenylpurine+bases+and+nucleosides%3A+application+of+the+Suzuki%E2%80%93Miyaura+cross-coupling+reactions+of+6-chloropurine+derivatives+with+phenylboronic+acids"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1021%2Fjm991167%2B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm991167%252B%26sid%3Dliteratum%253Aachs%26aulast%3DHocek%26aufirst%3DM.%26aulast%3DHoly%26aufirst%3DA.%26aulast%3DVotruba%26aufirst%3DI.%26aulast%3DDvorakova%26aufirst%3DH.%26atitle%3DSynthesis%2520and%2520cytostatic%2520activity%2520of%2520substituted%25206-phenylpurine%2520bases%2520and%2520nucleosides%253A%2520application%2520of%2520the%2520Suzuki%25E2%2580%2593Miyaura%2520cross-coupling%2520reactions%2520of%25206-chloropurine%2520derivatives%2520with%2520phenylboronic%2520acids%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2000%26volume%3D43%26spage%3D1817%26epage%3D1825" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group">Capek, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vrabel, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hasnik, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pohl, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hocek, M.</span><span> </span><span class="NLM_article-title">Aqueous-phase Suzuki–Miyaura cross-coupling reactions of free halopurine bases</span> <span class="citation_source-journal">Synthesis</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_fpage">3515</span><span class="NLM_x">–</span> <span class="NLM_lpage">3526</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtF2rsrrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2006&pages=3515-3526&author=P.+Capekauthor=M.+Vrabelauthor=Z.+Hasnikauthor=R.+Pohlauthor=M.+Hocek&title=Aqueous-phase+Suzuki%E2%80%93Miyaura+cross-coupling+reactions+of+free+halopurine+bases"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Aqueous-phase Suzuki-Miyaura cross-coupling reactions of free halopurine bases</span></div><div class="casAuthors">Capek, Petr; Vrabel, Milan; Hasnik, Zbynek; Pohl, Radek; Hocek, Michal</div><div class="citationInfo"><span class="NLM_cas:title">Synthesis</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume"></span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">3515-3526</span>CODEN:
                <span class="NLM_cas:coden">SYNTBF</span>;
        ISSN:<span class="NLM_cas:issn">0039-7881</span>.
    
            (<span class="NLM_cas:orgname">Georg Thieme Verlag</span>)
        </div><div class="casAbstract">The Suzuki-Miyaura reaction of 9-unsubstituted 2-, 6-, and 8-halopurine bases, e.g., I, II and III, resp., with diverse aryl- and alkenylboronic acids in water-acetonitrile mixts. under microwave irradn. was used for the single-step synthesis of arylpurines.  In most cases the product crystd. directly from the reaction mixt. in high yields and good purity.  The scope and limitations of these reactions were studied.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-m8oRscawvLVg90H21EOLACvtfcHk0lj7nIuXF567ow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtF2rsrrK&md5=b86910cdcddbaee4b6102dce66f5843a</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCapek%26aufirst%3DP.%26aulast%3DVrabel%26aufirst%3DM.%26aulast%3DHasnik%26aufirst%3DZ.%26aulast%3DPohl%26aufirst%3DR.%26aulast%3DHocek%26aufirst%3DM.%26atitle%3DAqueous-phase%2520Suzuki%25E2%2580%2593Miyaura%2520cross-coupling%2520reactions%2520of%2520free%2520halopurine%2520bases%26jtitle%3DSynthesis%26date%3D2006%26spage%3D3515%26epage%3D3526" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group">Matulis, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kranz, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Salemme, F. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Todd, M. J.</span><span> </span><span class="NLM_article-title">Thermodynamic stability of carbonic anhydrase: measurements of binding affinity and stoichiometry using ThermoFluor</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">5258</span><span class="NLM_x">–</span> <span class="NLM_lpage">5266</span></span><div class="citationLinks">[<a href="/doi/10.1021/bi048135v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BD2MXitVCksr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2005&pages=5258-5266&author=D.+Matulisauthor=J.+K.+Kranzauthor=F.+R.+Salemmeauthor=M.+J.+Todd&title=Thermodynamic+stability+of+carbonic+anhydrase%3A+measurements+of+binding+affinity+and+stoichiometry+using+ThermoFluor"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Thermodynamic Stability of Carbonic Anhydrase: Measurements of Binding Affinity and Stoichiometry Using ThermoFluor</span></div><div class="casAuthors">Matulis, Daumantas; Kranz, James K.; Salemme, F. Raymond; Todd, Matthew J.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">5258-5266</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">ThermoFluor (a miniaturized high-throughput protein stability assay) was used to analyze the linkage between protein thermal stability and ligand binding.  Equil. binding ligands increase protein thermal stability by an amt. proportional to the concn. and affinity of the ligand.  Binding consts. (Kb) were measured by examg. the systematic effect of ligand concn. on protein stability.  The precise ligand effects depend on the thermodn. of protein stability: in particular, the unfolding enthalpy.  An extension of current theor. treatments was developed for tight binding inhibitors, where ligand effect on Tm can also reveal binding stoichiometry.  A thermodn. anal. of carbonic anhydrase by differential scanning calorimetry (DSC) enabled a dissection of the Gibbs free energy of stability into enthalpic and entropic components.  Under certain conditions, thermal stability increased by over 30°; the heat capacity of protein unfolding was estd. from the dependence of calorimetric enthalpy on Tm.  The binding affinity of six sulfonamide inhibitors to two isoenzymes (human type 1 and bovine type 2) was analyzed by both ThermoFluor and isothermal titrn. calorimetry (ITC), resulting in a good correlation in the rank ordering of ligand affinity.  This combined investigation by ThermoFluor, ITC, and DSC provides a detailed picture of the linkage between ligand binding and protein stability.  The systematic effect of ligands on stability is shown to be a general tool to measure affinity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrc--PfKRLn6rVg90H21EOLACvtfcHk0lj7nIuXF567ow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXitVCksr4%253D&md5=79dd266c74abdeeab1bdf197dfa2affb</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1021%2Fbi048135v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi048135v%26sid%3Dliteratum%253Aachs%26aulast%3DMatulis%26aufirst%3DD.%26aulast%3DKranz%26aufirst%3DJ.%2BK.%26aulast%3DSalemme%26aufirst%3DF.%2BR.%26aulast%3DTodd%26aufirst%3DM.%2BJ.%26atitle%3DThermodynamic%2520stability%2520of%2520carbonic%2520anhydrase%253A%2520measurements%2520of%2520binding%2520affinity%2520and%2520stoichiometry%2520using%2520ThermoFluor%26jtitle%3DBiochemistry%26date%3D2005%26volume%3D44%26spage%3D5258%26epage%3D5266" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group">Wiseman, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williston, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brandts, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, L. N.</span><span> </span><span class="NLM_article-title">Rapid measurement of binding constants and heats of binding using a new titration calorimeter</span> <span class="citation_source-journal">Anal. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1989</span><span class="NLM_x">, </span> <span class="NLM_volume">179</span><span class="NLM_x">, </span> <span class="NLM_fpage">131</span><span class="NLM_x">–</span> <span class="NLM_lpage">137</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Fjm501893k&amp;key=10.1016%2F0003-2697%2889%2990213-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Fjm501893k&amp;key=2757186" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADyaL1MXktlKjsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=179&publication_year=1989&pages=131-137&author=T.+Wisemanauthor=S.+Willistonauthor=J.+F.+Brandtsauthor=L.+N.+Lin&title=Rapid+measurement+of+binding+constants+and+heats+of+binding+using+a+new+titration+calorimeter"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid measurement of binding constants and heats of binding using a new titration calorimeter</span></div><div class="casAuthors">Wiseman, Thomas; Williston, Samuel; Brandts, John F.; Lin, Lung Nan</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">179</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">131-7</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    </div><div class="casAbstract">A new titrn. calorimeter is described and results are presented for the binding of 2'-CMP to the active site of RNase A.  The instrument characteristics include very high sensitivity, rapid calorimetric response, and fast thermal equilibration.  Convenient software is available for instrument operation, data collection, data redn., and deconvolution to obtain least-squares ests. of binding parameters n, ΔH°, ΔS°, and the binding const. K.  Sample through-put for the instrument is high, and under favorable conditions binding consts. as large as 108 M-1 can be measured.  The bovine RNase A (RNase)/2'-CMP system was studied over a 50-fold range of RNase concn. and at 2 different temps.  The binding consts. were in the 105 to 106 M-1 range, depending on conditions, and heats of binding ∼-15,000 cal/mol.  Repeat detns. suggested errors of only a few percent in n, ΔH°, and K values over the most favorable concn. range.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJRWydqrfWArVg90H21EOLACvtfcHk0lj7nIuXF567ow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1MXktlKjsrw%253D&md5=53fcdb7144fd85e57963add0bcdb6041</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1016%2F0003-2697%2889%2990213-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0003-2697%252889%252990213-3%26sid%3Dliteratum%253Aachs%26aulast%3DWiseman%26aufirst%3DT.%26aulast%3DWilliston%26aufirst%3DS.%26aulast%3DBrandts%26aufirst%3DJ.%2BF.%26aulast%3DLin%26aufirst%3DL.%2BN.%26atitle%3DRapid%2520measurement%2520of%2520binding%2520constants%2520and%2520heats%2520of%2520binding%2520using%2520a%2520new%2520titration%2520calorimeter%26jtitle%3DAnal.%2520Biochem.%26date%3D1989%26volume%3D179%26spage%3D131%26epage%3D137" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group">Leslie, A. G. W.; Powell, H.</span>MOSFLM, version <span class="NLM_edition">7.01</span>; <span class="NLM_publisher-name">MRC Laboratory of Molecular Biology</span>: <span class="NLM_publisher-loc">Cambridge, U.K.</span>,<span class="NLM_x"> </span><span class="NLM_year">2007</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2007&author=A.+G.+W.+Leslie&author=H.+Powell&title=MOSFLM"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DLeslie%26aufirst%3DA.%2BG.%2BW.%26jtitle%3DMOSFLM%26pub%3DMRC%2520Laboratory%2520of%2520Molecular%2520Biology%26date%3D2007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group">Kabsch, W.</span><span> </span><span class="NLM_article-title">Evaluation of single-crystal X-ray-diffraction data from a position-sensitive detector</span> <span class="citation_source-journal">J. Appl. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1988</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">916</span><span class="NLM_x">–</span> <span class="NLM_lpage">924</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Fjm501893k&amp;key=10.1107%2FS0021889888007903" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADyaL1MXhtVGmu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=1988&pages=916-924&author=W.+Kabsch&title=Evaluation+of+single-crystal+X-ray-diffraction+data+from+a+position-sensitive+detector"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Evaluation of single-crystal x-ray diffraction data from a position-sensitive detector</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">1988</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">916-24</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    </div><div class="casAbstract">A Fortran program was developed for the redn. of single-crystal diffraction data from a sequence of adjacent rotation pictures recorded at a fixed x-ray wavelength by an electronic area detector.  The electronic pictures (data frames) covering the first 5° of crystal rotation are used to locate strong diffraction spots and to est. the background.  The orientation of the crystal is derived automatically from the list of obsd. spots and all parameters describing the diffraction pattern are refined.  The only input required from the user is the specification of the space group, approx. cell dimensions, and detector setting.  When the initialization step is finished the program goes back to the first picture and evaluates all data frames in the order they arrive from the measurement.  Each element of an electronic picture (pixel) is labeled by the indexes of the nearest reflection using the current refined parameters describing the diffraction geometry.  If the pixel is close to its nearest reflection the counts contribute to the 3-dimensional profile; otherwise the counts are used to update the background.  Each profile is represented as if the reflection had followed the shortest path through the Ewald sphere and had been recorded on the surface of the sphere.  Reflections close to the Ewald sphere are kept in a hash table to allow rapid access for updating the profiles.  Reflections which have completely passed through the Ewald sphere are removed from the table and saved for further processing.  Intensities are estd. by fitting their profiles to an av. shape learned from strong neighboring reflections.  Smoothly varying scaling factors are applied to the Lp-cor. intensities which minimize discrepancies between symmetry-related reflections and fit to a ref. data set if available.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtr9BhTPWbOLVg90H21EOLACvtfcHk0liG2XsJUDes1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1MXhtVGmu7c%253D&md5=f8d8c37891ec0d60758000879e7be9d6</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1107%2FS0021889888007903&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889888007903%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DEvaluation%2520of%2520single-crystal%2520X-ray-diffraction%2520data%2520from%2520a%2520position-sensitive%2520detector%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D1988%26volume%3D21%26spage%3D916%26epage%3D924" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref69"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref69'); return false;" data-citation="" class="refNumLink">69</a></strong><div class="NLM_citation" id="cit69"><span><span class="NLM_contrib-group">Kabsch, W.</span><span> </span><span class="NLM_article-title">Automatic-indexing of rotation diffraction patterns</span> <span class="citation_source-journal">J. Appl. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1988</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">67</span><span class="NLM_x">–</span> <span class="NLM_lpage">71</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Fjm501893k&amp;key=10.1107%2FS0021889887009737" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADyaL1cXhtVCgsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=1988&pages=67-71&author=W.+Kabsch&title=Automatic-indexing+of+rotation+diffraction+patterns"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Automatic indexing of rotation diffraction patterns</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">1988</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">67-71</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    </div><div class="casAbstract">A method is described which assigns indexes to a set of single-crystal reflections recorded by the rotation-oscillation technique using a fixed x-ray wavelength.  The space group and approx. unit-cell parameters are known.  The unknown crystal orientation is detd. directly from the obsd. diffraction pattern of one or several oscillation data records.  A local indexing procedure is described which allows for large initial errors in the parameters controlling the diffraction pattern.  These parameters are refined subsequently, thereby satisfying the constraints imposed by the space-group symmetry.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9WshttyE38LVg90H21EOLACvtfcHk0liG2XsJUDes1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1cXhtVCgsb8%253D&md5=268552b3f6069d2465b3916a985b840f</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1107%2FS0021889887009737&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889887009737%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DAutomatic-indexing%2520of%2520rotation%2520diffraction%2520patterns%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D1988%26volume%3D21%26spage%3D67%26epage%3D71" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref70"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref70'); return false;" data-citation="" class="refNumLink">70</a></strong><div class="NLM_citation" id="cit70"><span><span class="NLM_contrib-group">Evans, P.</span>SCALA—Scale Together Multiple Observations of Reflections, version <span class="NLM_edition">3.3.0</span>; <span class="NLM_publisher-name">MRC Laboratory of Molecular Biology</span>: <span class="NLM_publisher-loc">Cambridge, U.K.</span>,<span class="NLM_x"> </span><span class="NLM_year">2007</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2007&author=P.+Evans&title=SCALA%E2%80%94Scale+Together+Multiple+Observations+of+Reflections"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DP.%26jtitle%3DSCALA%25E2%2580%2594Scale%2520Together%2520Multiple%2520Observations%2520of%2520Reflections%26pub%3DMRC%2520Laboratory%2520of%2520Molecular%2520Biology%26date%3D2007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref71"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref71'); return false;" data-citation="" class="refNumLink">71</a></strong><div class="NLM_citation" id="cit71"><span><span class="NLM_contrib-group">McCoy, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Storoni, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span> </span><span class="NLM_article-title">Likelihood-enhanced fast translation functions</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">61</span><span class="NLM_x">, </span> <span class="NLM_fpage">458</span><span class="NLM_x">–</span> <span class="NLM_lpage">464</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Fjm501893k&amp;key=10.1107%2FS0907444905001617" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Fjm501893k&amp;key=15805601" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BD2MXivVeisrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2005&pages=458-464&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Likelihood-enhanced+fast+translation+functions"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Likelihood-enhanced fast translation functions</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">D61</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">458-464</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">This paper is a companion to a recent paper on fast rotation functions [Storoni et al. (2004), Acta Cryst. D60, 432-438], which showed how a Taylor-series expansion of the max.-likelihood rotation function leads to improved likelihood-enhanced fast rotation functions.  In a similar manner, it is shown here how linear and quadratic Taylor-series expansions and least-squares approxns. of the max.-likelihood translation function lead to likelihood-enhanced translation functions, which can be calcd. by FFT and which are more sensitive to the correct translation than the traditional correlation-coeff. fast translation function.  These likelihood-enhanced translation targets for mol.-replacement searches have been implemented in the program Phaser using the Computational Crystallog. Toolbox (cctbx).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDd3wovyUbWLVg90H21EOLACvtfcHk0liG2XsJUDes1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXivVeisrY%253D&md5=bac9cc121bcc0d5cd314955a0e65e746</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1107%2FS0907444905001617&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444905001617%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DLikelihood-enhanced%2520fast%2520translation%2520functions%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2005%26volume%3D61%26spage%3D458%26epage%3D464" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref72"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref72'); return false;" data-citation="" class="refNumLink">72</a></strong><div class="NLM_citation" id="cit72"><span><span class="NLM_contrib-group">Perrakis, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morris, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lamzin, V. S.</span><span> </span><span class="NLM_article-title">Automated protein model building combined with iterative structure refinement</span> <span class="citation_source-journal">Nat. Struct. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">458</span><span class="NLM_x">–</span> <span class="NLM_lpage">463</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Fjm501893k&amp;key=10.1038%2F8263" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Fjm501893k&amp;key=10331874" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADyaK1MXjtlajurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1999&pages=458-463&author=A.+Perrakisauthor=R.+Morrisauthor=V.+S.+Lamzin&title=Automated+protein+model+building+combined+with+iterative+structure+refinement"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Automated protein model building combined with iterative structure refinement</span></div><div class="casAuthors">Perrakis, Anastassis; Morris, Richard; Lamzin, Victor S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural Biology</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">458-463</span>CODEN:
                <span class="NLM_cas:coden">NSBIEW</span>;
        ISSN:<span class="NLM_cas:issn">1072-8368</span>.
    
            (<span class="NLM_cas:orgname">Nature America</span>)
        </div><div class="casAbstract">In protein crystallog., much time and effort are often required to trace an initial model from an interpretable electron d. map and to refine it until it best agrees with the crystallog. data.  Here, we present a method to build and refine a protein model automatically and without user intervention, starting from diffraction data extending to resoln. higher than 2.3 Å and reasonable ests. of crystallog. phases.  The method is based on an iterative procedure that describes the electron d. map as a set of unconnected atoms and then searches for protein-like patterns.  Automatic pattern recognition (model building) combined with refinement, allows a structural model to be obtained reliably within a few CPU hours.  We demonstrate the power of the method with examples of a few recently solved structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbcrkUoPTzsLVg90H21EOLACvtfcHk0liG2XsJUDes1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXjtlajurs%253D&md5=d12fc80eb73e1faebee29e2f6635c86d</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1038%2F8263&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F8263%26sid%3Dliteratum%253Aachs%26aulast%3DPerrakis%26aufirst%3DA.%26aulast%3DMorris%26aufirst%3DR.%26aulast%3DLamzin%26aufirst%3DV.%2BS.%26atitle%3DAutomated%2520protein%2520model%2520building%2520combined%2520with%2520iterative%2520structure%2520refinement%26jtitle%3DNat.%2520Struct.%2520Biol.%26date%3D1999%26volume%3D6%26spage%3D458%26epage%3D463" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref73"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref73'); return false;" data-citation="" class="refNumLink">73</a></strong><div class="NLM_citation" id="cit73"><span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K.</span><span> </span><span class="NLM_article-title">Coot: model-building tools for molecular graphics</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">60</span><span class="NLM_x">, </span> <span class="NLM_fpage">2126</span><span class="NLM_x">–</span> <span class="NLM_lpage">2132</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Fjm501893k&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Fjm501893k&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+model-building+tools+for+molecular+graphics"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0ljqsLrzHx0cog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref74"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref74'); return false;" data-citation="" class="refNumLink">74</a></strong><div class="NLM_citation" id="cit74"><span><span class="NLM_contrib-group">Murshudov, G. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vagin, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dodson, E. J.</span><span> </span><span class="NLM_article-title">Refinement of macromolecular structures by the maximum-likelihood method</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">240</span><span class="NLM_x">–</span> <span class="NLM_lpage">255</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Fjm501893k&amp;key=10.1107%2FS0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=8&amp;doi=10.1021%2Fjm501893k&amp;key=15299926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=240-255&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+macromolecular+structures+by+the+maximum-likelihood+method"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. σA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial α-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0ljqsLrzHx0cog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1107%2FS0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520macromolecular%2520structures%2520by%2520the%2520maximum-likelihood%2520method%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1997%26volume%3D53%26spage%3D240%26epage%3D255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref75"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref75'); return false;" data-citation="" class="refNumLink">75</a></strong><div class="NLM_citation" id="cit75"><span><span class="NLM_contrib-group">Painter, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Merritt, E. A.</span><span> </span><span class="NLM_article-title">Optimal description of a protein structure in terms of multiple groups undergoing TLS motion</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">62</span><span class="NLM_x">, </span> <span class="NLM_fpage">439</span><span class="NLM_x">–</span> <span class="NLM_lpage">450</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=16&amp;doi=10.1021%2Fjm501893k&amp;key=10.1107%2FS0907444906005270" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=8&amp;doi=10.1021%2Fjm501893k&amp;key=16552146" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BD28Xis1Gkt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2006&pages=439-450&author=J.+Painterauthor=E.+A.+Merritt&title=Optimal+description+of+a+protein+structure+in+terms+of+multiple+groups+undergoing+TLS+motion"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">Optimal description of a protein structure in terms of multiple groups undergoing TLS motion</span></div><div class="casAuthors">Painter, Jay; Merritt, Ethan A.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">D62</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">439-450</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">A single protein crystal structure contains information about dynamic properties of the protein as well as providing a static view of one three-dimensional conformation.  This addnl. information is to be found in the distribution of obsd. electron d. about the mean position of each atom.  It is general practice to account for this by refining a sep. at. displacement parameter (ADP) for each at. center.  However, these same displacements are often described well by simpler models based on TLS (translation/libration/screw) rigid-body motion of large groups of atoms, for example interdomain hinge motion.  A procedure, TLSMD, has been developed that analyzes the distribution of ADPs in a previously refined protein crystal structure in order to generate optimal multi-group TLS descriptions of the constituent protein chains.  TLSMD is applicable to crystal structures at any resoln.  The models generated by TLSMD anal. can significantly improve the std. crystallog. residuals R and Rfree and can reveal intrinsic dynamic properties of the protein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrIR_NcneJBNbVg90H21EOLACvtfcHk0ljqsLrzHx0cog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xis1Gkt74%253D&md5=6289483d19fb6dd64fe9b032e9c45005</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1107%2FS0907444906005270&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444906005270%26sid%3Dliteratum%253Aachs%26aulast%3DPainter%26aufirst%3DJ.%26aulast%3DMerritt%26aufirst%3DE.%2BA.%26atitle%3DOptimal%2520description%2520of%2520a%2520protein%2520structure%2520in%2520terms%2520of%2520multiple%2520groups%2520undergoing%2520TLS%2520motion%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2006%26volume%3D62%26spage%3D439%26epage%3D450" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref76"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref76'); return false;" data-citation="" class="refNumLink">76</a></strong><div class="NLM_citation" id="cit76"><span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Picaud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keller, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wrobel, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morgenstern, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bracher, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span> </span><span class="NLM_article-title">Benzodiazepines and benzotriazepines as protein interaction inhibitors targeting bromodomains of the BET family</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">1878</span><span class="NLM_x">–</span> <span class="NLM_lpage">1886</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=16&amp;doi=10.1021%2Fjm501893k&amp;key=10.1016%2Fj.bmc.2011.10.080" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=8&amp;doi=10.1021%2Fjm501893k&amp;key=22137933" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BC38XjtlOksrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2012&pages=1878-1886&author=P.+Filippakopoulosauthor=S.+Picaudauthor=O.+Fedorovauthor=M.+Kellerauthor=M.+Wrobelauthor=O.+Morgensternauthor=F.+Bracherauthor=S.+Knapp&title=Benzodiazepines+and+benzotriazepines+as+protein+interaction+inhibitors+targeting+bromodomains+of+the+BET+family"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">Benzodiazepines and benzotriazepines as protein interaction inhibitors targeting bromodomains of the BET family</span></div><div class="casAuthors">Filippakopoulos, Panagis; Picaud, Sarah; Fedorov, Oleg; Keller, Marco; Wrobel, Matthias; Morgenstern, Olaf; Bracher, Franz; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1878-1886</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Benzodiazepines are psychoactive drugs with anxiolytic, sedative, skeletal muscle relaxant and amnestic properties.  Recently triazolo-benzodiazepines have been also described as potent and highly selective protein interaction inhibitors of bromodomain and extra-terminal (BET) proteins, a family of transcriptional co-regulators that play a key role in cancer cell survival and proliferation, but the requirements for high affinity interaction of this compd. class with bromodomains has not been described.  Here we provide insight into the structure-activity relationship (SAR) and selectivity of this versatile scaffold.  In addn., using high resoln. crystal structures we compared the binding mode of a series of benzodiazepine (BzD) and related triazolo-benzotriazepines (BzT) derivs. including clin. approved drugs such as alprazolam and midazolam.  Our anal. revealed the importance of the 1-Me triazolo ring system for BET binding and suggests modifications for the development of further high affinity bromodomain inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9BT8VUvs8_LVg90H21EOLACvtfcHk0ljqsLrzHx0cog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjtlOksrw%253D&md5=aed7023bdb44ee0b83af1605f0be0ddd</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2011.10.080&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2011.10.080%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DKeller%26aufirst%3DM.%26aulast%3DWrobel%26aufirst%3DM.%26aulast%3DMorgenstern%26aufirst%3DO.%26aulast%3DBracher%26aufirst%3DF.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DBenzodiazepines%2520and%2520benzotriazepines%2520as%2520protein%2520interaction%2520inhibitors%2520targeting%2520bromodomains%2520of%2520the%2520BET%2520family%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2012%26volume%3D20%26spage%3D1878%26epage%3D1886" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref77"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref77'); return false;" data-citation="" class="refNumLink">77</a></strong><div class="NLM_citation" id="cit77"><span><span class="NLM_contrib-group">Picaud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wells, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Felletar, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brotherton, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Savitsky, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diez-Dacal, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Philpott, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bountra, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lingard, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muller, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brennan, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span> </span><span class="NLM_article-title">RVX-208, an inhibitor of BET transcriptional regulators with selectivity for the second bromodomain</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">110</span><span class="NLM_x">, </span> <span class="NLM_fpage">19754</span><span class="NLM_x">–</span> <span class="NLM_lpage">19759</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Fjm501893k&amp;key=10.1073%2Fpnas.1310658110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=8&amp;doi=10.1021%2Fjm501893k&amp;key=24248379" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFKms7zN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=19754-19759&author=S.+Picaudauthor=C.+Wellsauthor=I.+Felletarauthor=D.+Brothertonauthor=S.+Martinauthor=P.+Savitskyauthor=B.+Diez-Dacalauthor=M.+Philpottauthor=C.+Bountraauthor=H.+Lingardauthor=O.+Fedorovauthor=S.+Mullerauthor=P.+E.+Brennanauthor=S.+Knappauthor=P.+Filippakopoulos&title=RVX-208%2C+an+inhibitor+of+BET+transcriptional+regulators+with+selectivity+for+the+second+bromodomain"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">RVX-208, an inhibitor of BET transcriptional regulators with selectivity for the second bromodomain</span></div><div class="casAuthors">Picaud, Sarah; Wells, Christopher; Felletar, Ildiko; Brotherton, Deborah; Martin, Sarah; Savitsky, Pavel; Diez-Dacal, Beatriz; Philpott, Martin; Bountra, Chas; Lingard, Hannah; Fedorov, Oleg; Muller, Susanne; Brennan, Paul E.; Knapp, Stefan; Filippakopoulos, Panagis</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">19754-19759,S19754/1-S19754/10</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Bromodomains have emerged as attractive candidates for the development of inhibitors targeting gene transcription.  Inhibitors of the bromo and extraterminal (BET) family recently showed promising activity in diverse disease models.  However, the pleiotropic nature of BET proteins regulating tissue-specific transcription has raised safety concerns and suggested that attempts should be made for domain-specific targeting.  Here, we report that RVX-208, a compd. currently in phase II clin. trials, is a BET bromodomain inhibitor specific for second bromodomains (BD2s).  Cocrystal structures revealed binding modes of RVX-208 and its synthetic precursor, and fluorescent recovery after photobleaching demonstrated that RVX-208 displaces BET proteins from chromatin.  However, gene-expression data showed that BD2 inhibition only modestly affects BET-dependent gene transcription.  Our data demonstrate the feasibility of specific targeting within the BET family resulting in different transcriptional outcomes and highlight the importance of BD1 in transcriptional regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqamW8vnIopKrVg90H21EOLACvtfcHk0ljrlMf750pMpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFKms7zN&md5=c918702c9fdb91241bc837732730f7f6</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1310658110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1310658110%26sid%3Dliteratum%253Aachs%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DBrotherton%26aufirst%3DD.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DSavitsky%26aufirst%3DP.%26aulast%3DDiez-Dacal%26aufirst%3DB.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DLingard%26aufirst%3DH.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26atitle%3DRVX-208%252C%2520an%2520inhibitor%2520of%2520BET%2520transcriptional%2520regulators%2520with%2520selectivity%2520for%2520the%2520second%2520bromodomain%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2013%26volume%3D110%26spage%3D19754%26epage%3D19759" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref78"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref78'); return false;" data-citation="" class="refNumLink">78</a></strong><div class="NLM_citation" id="cit78"><span><span class="NLM_contrib-group">Ghosh, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lagisetty, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zajc, B.</span><span> </span><span class="NLM_article-title">Direct synthesis of 8-fluoro purine nucleosides via metalation–fluorination</span> <span class="citation_source-journal">J. Org. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">72</span><span class="NLM_x">, </span> <span class="NLM_fpage">8222</span><span class="NLM_x">–</span> <span class="NLM_lpage">8226</span></span><div class="citationLinks">[<a href="/doi/10.1021/jo071121l" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2007&pages=8222-8226&author=A.+K.+Ghoshauthor=P.+Lagisettyauthor=B.+Zajc&title=Direct+synthesis+of+8-fluoro+purine+nucleosides+via+metalation%E2%80%93fluorination"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1021%2Fjo071121l&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo071121l%26sid%3Dliteratum%253Aachs%26aulast%3DGhosh%26aufirst%3DA.%2BK.%26aulast%3DLagisetty%26aufirst%3DP.%26aulast%3DZajc%26aufirst%3DB.%26atitle%3DDirect%2520synthesis%2520of%25208-fluoro%2520purine%2520nucleosides%2520via%2520metalation%25E2%2580%2593fluorination%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2007%26volume%3D72%26spage%3D8222%26epage%3D8226" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref79"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref79'); return false;" data-citation="" class="refNumLink">79</a></strong><div class="NLM_citation" id="cit79"><span><span class="NLM_contrib-group">Roy, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schneller, S. W.</span><span> </span><span class="NLM_article-title">An unusual occurrence on attempted purine C-8 electrophilic fluorination of 5′-noraristeromycin</span> <span class="citation_source-journal">Org. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">3889</span><span class="NLM_x">–</span> <span class="NLM_lpage">3891</span></span><div class="citationLinks">[<a href="/doi/10.1021/ol051297e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BD2MXntVersrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2005&pages=3889-3891&author=A.+Royauthor=S.+W.+Schneller&title=An+unusual+occurrence+on+attempted+purine+C-8+electrophilic+fluorination+of+5%E2%80%B2-noraristeromycin"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">An Unusual Occurrence on Attempted Purine C-8 Electrophilic Fluorination of 5'-Noraristeromycin</span></div><div class="casAuthors">Roy, Atanu; Schneller, Stewart W.</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">3889-3891</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7060</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Reaction of the in situ generated purine C-8 carbanion of a protected 5'-noraristeromycin deriv. with N-fluorobenzenesulfonimide gave 8-phenylsulfonyl-5'-noraristeromycin rather than the expected 8-fluoro deriv.  A single electron transfer (SET) mechanism is proposed for this occurrence.  The phenylsulfonyl product offers a structural feature common to some anti-HIV agents (no biol. data).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxmIetT-7UJ7Vg90H21EOLACvtfcHk0ljrlMf750pMpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXntVersrc%253D&md5=9049560d481807b4714aecb94e4a04ae</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1021%2Fol051297e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol051297e%26sid%3Dliteratum%253Aachs%26aulast%3DRoy%26aufirst%3DA.%26aulast%3DSchneller%26aufirst%3DS.%2BW.%26atitle%3DAn%2520unusual%2520occurrence%2520on%2520attempted%2520purine%2520C-8%2520electrophilic%2520fluorination%2520of%25205%25E2%2580%25B2-noraristeromycin%26jtitle%3DOrg.%2520Lett.%26date%3D2005%26volume%3D7%26spage%3D3889%26epage%3D3891" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref80"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref80'); return false;" data-citation="" class="refNumLink">80</a></strong><div class="NLM_citation" id="cit80"><span><span class="NLM_contrib-group">De, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jasani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arora, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gambhir, S. S.</span><span> </span><span class="NLM_article-title">Evolution of BRET biosensors from live cell to tissue-scale in vivo imaging</span> <span class="citation_source-journal">Front. Endocrinol. (Lausanne, Switz.)</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">131</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=131&author=A.+Deauthor=A.+Jasaniauthor=R.+Aroraauthor=S.+S.+Gambhir&title=Evolution+of+BRET+biosensors+from+live+cell+to+tissue-scale+in+vivo+imaging"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DDe%26aufirst%3DA.%26aulast%3DJasani%26aufirst%3DA.%26aulast%3DArora%26aufirst%3DR.%26aulast%3DGambhir%26aufirst%3DS.%2BS.%26atitle%3DEvolution%2520of%2520BRET%2520biosensors%2520from%2520live%2520cell%2520to%2520tissue-scale%2520in%2520vivo%2520imaging%26jtitle%3DFront.%2520Endocrinol.%2520%2528Lausanne%252C%2520Switz.%2529%26date%3D2013%26volume%3D4%26spage%3D131" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68'],'ref69':['cit69'],'ref70':['cit70'],'ref71':['cit71'],'ref72':['cit72'],'ref73':['cit73'],'ref74':['cit74'],'ref75':['cit75'],'ref76':['cit76'],'ref77':['cit77'],'ref78':['cit78'],'ref79':['cit79'],'ref80':['cit80']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 49 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Simona De Vita, Gianluigi Lauro, Dafne Ruggiero, Stefania Terracciano, Raffaele Riccio, <span class="NLM_string-name hlFld-ContribAuthor">Giuseppe Bifulco</span>. </span><span class="cited-content_cbyCitation_article-title">Protein Preparation Automatic Protocol for High-Throughput Inverse Virtual Screening: Accelerating the Target Identification by Computational Methods. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2019,</strong> <em>59 </em>
                                    (11)
                                     , 4678-4690. <a href="https://doi.org/10.1021/acs.jcim.9b00428" title="DOI URL">https://doi.org/10.1021/acs.jcim.9b00428</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.9b00428&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.9b00428%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DProtein%252BPreparation%252BAutomatic%252BProtocol%252Bfor%252BHigh-Throughput%252BInverse%252BVirtual%252BScreening%25253A%252BAccelerating%252Bthe%252BTarget%252BIdentification%252Bby%252BComputational%252BMethods%26aulast%3DDe%2BVita%26aufirst%3DSimona%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D23052019%26date%3D01112019%26date%3D08102019%26volume%3D59%26issue%3D11%26spage%3D4678%26epage%3D4690" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Jian Zhu, Jing Dong, Laurent Batiste, Andrea Unzue, Aymeric Dolbois, Vlad Pascanu, Paweł Śledź, Cristina Nevado, <span class="NLM_string-name hlFld-ContribAuthor">Amedeo Caflisch</span>. </span><span class="cited-content_cbyCitation_article-title">Binding Motifs in the CBP Bromodomain: An Analysis of 20 Crystal Structures of Complexes with Small Molecules. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2018,</strong> <em>9 </em>
                                    (9)
                                     , 929-934. <a href="https://doi.org/10.1021/acsmedchemlett.8b00286" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.8b00286</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.8b00286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.8b00286%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DBinding%252BMotifs%252Bin%252Bthe%252BCBP%252BBromodomain%25253A%252BAn%252BAnalysis%252Bof%252B20%252BCrystal%252BStructures%252Bof%252BComplexes%252Bwith%252BSmall%252BMolecules%26aulast%3DZhu%26aufirst%3DJian%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D22062018%26date%3D08082018%26date%3D13082018%26date%3D08082018%26volume%3D9%26issue%3D9%26spage%3D929%26epage%3D934" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Zhoulong Fan, Heng Lu, <span class="NLM_string-name hlFld-ContribAuthor">Ao Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">PMes3-Promoted Ruthenium-Catalyzed Meta C–H Nitration of 6-Arylpurines. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Organic Chemistry</span><span> <strong>2018,</strong> <em>83 </em>
                                    (6)
                                     , 3245-3251. <a href="https://doi.org/10.1021/acs.joc.8b00149" title="DOI URL">https://doi.org/10.1021/acs.joc.8b00149</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.joc.8b00149&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.joc.8b00149%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Organic%2520Chemistry%26atitle%3DPMes3-Promoted%252BRuthenium-Catalyzed%252BMeta%252BC%2525E2%252580%252593H%252BNitration%252Bof%252B6-Arylpurines%26aulast%3DFan%26aufirst%3DZhoulong%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D17012018%26date%3D02032018%26date%3D27022018%26volume%3D83%26issue%3D6%26spage%3D3245%26epage%3D3251" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jian  Zhu</span> and <span class="hlFld-ContribAuthor ">Amedeo  Caflisch</span>  . </span><span class="cited-content_cbyCitation_article-title">Twenty Crystal Structures of Bromodomain and PHD Finger Containing Protein 1 (BRPF1)/Ligand Complexes Reveal Conserved Binding Motifs and Rare Interactions. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2016,</strong> <em>59 </em>
                                    (11)
                                     , 5555-5561. <a href="https://doi.org/10.1021/acs.jmedchem.6b00215" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.6b00215</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.6b00215&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.6b00215%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DTwenty%252BCrystal%252BStructures%252Bof%252BBromodomain%252Band%252BPHD%252BFinger%252BContaining%252BProtein%252B1%252B%252528BRPF1%252529%25252FLigand%252BComplexes%252BReveal%252BConserved%252BBinding%252BMotifs%252Band%252BRare%252BInteractions%26aulast%3DZhu%26aufirst%3DJian%26date%3D2016%26date%3D2016%26date%3D2016%26date%3D11022016%26date%3D24052016%26date%3D09062016%26date%3D11052016%26volume%3D59%26issue%3D11%26spage%3D5555%26epage%3D5561" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Laetitia J.  Martin</span>, <span class="hlFld-ContribAuthor ">Manfred  Koegl</span>, <span class="hlFld-ContribAuthor ">Gerd  Bader</span>, <span class="hlFld-ContribAuthor ">Xiao-Ling  Cockcroft</span>, <span class="hlFld-ContribAuthor ">Oleg  Fedorov</span>, <span class="hlFld-ContribAuthor ">Dennis  Fiegen</span>, <span class="hlFld-ContribAuthor ">Thomas  Gerstberger</span>, <span class="hlFld-ContribAuthor ">Marco H.  Hofmann</span>, <span class="hlFld-ContribAuthor ">Anja F.  Hohmann</span>, <span class="hlFld-ContribAuthor ">Dirk  Kessler</span>, <span class="hlFld-ContribAuthor ">Stefan  Knapp</span>, <span class="hlFld-ContribAuthor ">Petr  Knesl</span>, <span class="hlFld-ContribAuthor ">Stefan  Kornigg</span>, <span class="hlFld-ContribAuthor ">Susanne  Müller</span>, <span class="hlFld-ContribAuthor ">Herbert  Nar</span>, <span class="hlFld-ContribAuthor ">Catherine  Rogers</span>, <span class="hlFld-ContribAuthor ">Klaus  Rumpel</span>, <span class="hlFld-ContribAuthor ">Otmar  Schaaf</span>, <span class="hlFld-ContribAuthor ">Steffen  Steurer</span>, <span class="hlFld-ContribAuthor ">Cynthia  Tallant</span>, <span class="hlFld-ContribAuthor ">Christopher R.  Vakoc</span>, <span class="hlFld-ContribAuthor ">Markus  Zeeb</span>, <span class="hlFld-ContribAuthor ">Andreas  Zoephel</span>, <span class="hlFld-ContribAuthor ">Mark  Pearson</span>, <span class="hlFld-ContribAuthor ">Guido  Boehmelt</span>, and <span class="hlFld-ContribAuthor ">Darryl  McConnell</span>  . </span><span class="cited-content_cbyCitation_article-title">Structure-Based Design of an in Vivo Active Selective BRD9 Inhibitor. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2016,</strong> <em>59 </em>
                                    (10)
                                     , 4462-4475. <a href="https://doi.org/10.1021/acs.jmedchem.5b01865" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.5b01865</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.5b01865&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.5b01865%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DStructure-Based%252BDesign%252Bof%252Ban%252Bin%252BVivo%252BActive%252BSelective%252BBRD9%252BInhibitor%26aulast%3DMartin%26aufirst%3DLaetitia%2BJ.%26date%3D2016%26date%3D2016%26date%3D2015%26date%3D02122015%26date%3D10032016%26date%3D26052016%26date%3D25022016%26volume%3D59%26issue%3D10%26spage%3D4462%26epage%3D4475" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rezaul M.  Karim</span> and <span class="hlFld-ContribAuthor ">Ernst  Schönbrunn</span>  . </span><span class="cited-content_cbyCitation_article-title">An Advanced Tool To Interrogate BRD9. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2016,</strong> <em>59 </em>
                                    (10)
                                     , 4459-4461. <a href="https://doi.org/10.1021/acs.jmedchem.6b00550" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.6b00550</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.6b00550&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.6b00550%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DAn%252BAdvanced%252BTool%252BTo%252BInterrogate%252BBRD9%26aulast%3DKarim%26aufirst%3DRezaul%2BM.%26date%3D2016%26date%3D2016%26date%3D2016%26date%3D11042016%26date%3D27042016%26date%3D26052016%26volume%3D59%26issue%3D10%26spage%3D4459%26epage%3D4461" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Steven G.  Smith</span> and <span class="hlFld-ContribAuthor ">Ming-Ming  Zhou</span>  . </span><span class="cited-content_cbyCitation_article-title">The Bromodomain: A New Target in Emerging Epigenetic Medicine. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2016,</strong> <em>11 </em>
                                    (3)
                                     , 598-608. <a href="https://doi.org/10.1021/acschembio.5b00831" title="DOI URL">https://doi.org/10.1021/acschembio.5b00831</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschembio.5b00831&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschembio.5b00831%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DThe%252BBromodomain%25253A%252BA%252BNew%252BTarget%252Bin%252BEmerging%252BEpigenetic%252BMedicine%26aulast%3DSmith%26aufirst%3DSteven%2BG.%26date%3D2016%26date%3D2015%26date%3D2015%26date%3D2015%26date%3D09102015%26date%3D23112015%26date%3D03122015%26date%3D18032016%26date%3D23112015%26volume%3D11%26issue%3D3%26spage%3D598%26epage%3D608" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Graziano  Lolli</span> and <span class="hlFld-ContribAuthor ">Amedeo  Caflisch</span>  . </span><span class="cited-content_cbyCitation_article-title">High-Throughput Fragment Docking into the BAZ2B Bromodomain: Efficient in Silico Screening for X-Ray Crystallography. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2016,</strong> <em>11 </em>
                                    (3)
                                     , 800-807. <a href="https://doi.org/10.1021/acschembio.5b00914" title="DOI URL">https://doi.org/10.1021/acschembio.5b00914</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschembio.5b00914&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschembio.5b00914%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DHigh-Throughput%252BFragment%252BDocking%252Binto%252Bthe%252BBAZ2B%252BBromodomain%25253A%252BEfficient%252Bin%252BSilico%252BScreening%252Bfor%252BX-Ray%252BCrystallography%26aulast%3DLolli%26aufirst%3DGraziano%26date%3D2016%26date%3D2016%26date%3D2016%26date%3D2015%26date%3D06112015%26date%3D04032016%26date%3D18032016%26date%3D04032016%26volume%3D11%26issue%3D3%26spage%3D800%26epage%3D807" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Wylie S.  Palmer</span>, <span class="hlFld-ContribAuthor ">Guillaume  Poncet-Montange</span>, <span class="hlFld-ContribAuthor ">Gang  Liu</span>, <span class="hlFld-ContribAuthor ">Alessia  Petrocchi</span>, <span class="hlFld-ContribAuthor ">Naphtali  Reyna</span>, <span class="hlFld-ContribAuthor ">Govindan  Subramanian</span>, <span class="hlFld-ContribAuthor ">Jay  Theroff</span>, <span class="hlFld-ContribAuthor ">Anne  Yau</span>, <span class="hlFld-ContribAuthor ">Maria  Kost-Alimova</span>, <span class="hlFld-ContribAuthor ">Jennifer P.  Bardenhagen</span>, <span class="hlFld-ContribAuthor ">Elisabetta  Leo</span>, <span class="hlFld-ContribAuthor ">Hannah E.  Shepard</span>, <span class="hlFld-ContribAuthor ">Trang N.  Tieu</span>, <span class="hlFld-ContribAuthor ">Xi  Shi</span>, <span class="hlFld-ContribAuthor ">Yanai  Zhan</span>, <span class="hlFld-ContribAuthor ">Shuping  Zhao</span>, <span class="hlFld-ContribAuthor ">Michelle C.  Barton</span>, <span class="hlFld-ContribAuthor ">Giulio  Draetta</span>, <span class="hlFld-ContribAuthor ">Carlo  Toniatti</span>, <span class="hlFld-ContribAuthor ">Philip  Jones</span>, <span class="hlFld-ContribAuthor ">Mary  Geck Do</span>, and <span class="hlFld-ContribAuthor ">Jannik N.  Andersen</span>  . </span><span class="cited-content_cbyCitation_article-title">Structure-Guided Design of IACS-9571, a Selective High-Affinity Dual TRIM24-BRPF1 Bromodomain Inhibitor. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2016,</strong> <em>59 </em>
                                    (4)
                                     , 1440-1454. <a href="https://doi.org/10.1021/acs.jmedchem.5b00405" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.5b00405</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.5b00405&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.5b00405%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DStructure-Guided%252BDesign%252Bof%252BIACS-9571%25252C%252Ba%252BSelective%252BHigh-Affinity%252BDual%252BTRIM24-BRPF1%252BBromodomain%252BInhibitor%26aulast%3DPalmer%26aufirst%3DWylie%2BS.%26date%3D2016%26date%3D2015%26date%3D2015%26date%3D11032015%26date%3D06072015%26date%3D25022016%26date%3D10062015%26volume%3D59%26issue%3D4%26spage%3D1440%26epage%3D1454" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">F. Anthony  Romero</span>, <span class="hlFld-ContribAuthor ">Alexander M.  Taylor</span>, <span class="hlFld-ContribAuthor ">Terry D.  Crawford</span>, <span class="hlFld-ContribAuthor ">Vickie  Tsui</span>, <span class="hlFld-ContribAuthor ">Alexandre  Côté</span>, and <span class="hlFld-ContribAuthor ">Steven  Magnuson</span>  . </span><span class="cited-content_cbyCitation_article-title">Disrupting Acetyl-Lysine Recognition: Progress in the Development of Bromodomain Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2016,</strong> <em>59 </em>
                                    (4)
                                     , 1271-1298. <a href="https://doi.org/10.1021/acs.jmedchem.5b01514" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.5b01514</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.5b01514&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.5b01514%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDisrupting%252BAcetyl-Lysine%252BRecognition%25253A%252BProgress%252Bin%252Bthe%252BDevelopment%252Bof%252BBromodomain%252BInhibitors%26aulast%3DRomero%26aufirst%3DF.%2BAnthony%26date%3D2016%26date%3D2015%26date%3D2015%26date%3D29092015%26date%3D01122015%26date%3D25022016%26date%3D16112015%26volume%3D59%26issue%3D4%26spage%3D1271%26epage%3D1298" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Guangtao  Zhang</span>, <span class="hlFld-ContribAuthor ">Steven G.  Smith</span>, and <span class="hlFld-ContribAuthor ">Ming-Ming  Zhou</span>  . </span><span class="cited-content_cbyCitation_article-title">Discovery of Chemical Inhibitors of Human Bromodomains. </span><span class="cited-content_cbyCitation_journal-name">Chemical Reviews</span><span> <strong>2015,</strong> <em>115 </em>
                                    (21)
                                     , 11625-11668. <a href="https://doi.org/10.1021/acs.chemrev.5b00205" title="DOI URL">https://doi.org/10.1021/acs.chemrev.5b00205</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.chemrev.5b00205&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.chemrev.5b00205%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Reviews%26atitle%3DDiscovery%252Bof%252BChemical%252BInhibitors%252Bof%252BHuman%252BBromodomains%26aulast%3DZhang%26aufirst%3DGuangtao%26date%3D2015%26date%3D2015%26date%3D2015%26date%3D06042015%26date%3D23102015%26date%3D11112015%26volume%3D115%26issue%3D21%26spage%3D11625%26epage%3D11668" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kai-Chun  Li</span>, <span class="hlFld-ContribAuthor ">Enrico  Girardi</span>, <span class="hlFld-ContribAuthor ">Felix  Kartnig</span>, <span class="hlFld-ContribAuthor ">Sarah  Grosche</span>, <span class="hlFld-ContribAuthor ">Tea  Pemovska</span>, <span class="hlFld-ContribAuthor ">Johannes W.  Bigenzahn</span>, <span class="hlFld-ContribAuthor ">Ulrich  Goldmann</span>, <span class="hlFld-ContribAuthor ">Vitaly  Sedlyarov</span>, <span class="hlFld-ContribAuthor ">Ariel  Bensimon</span>, <span class="hlFld-ContribAuthor ">Sandra  Schick</span>, <span class="hlFld-ContribAuthor ">Jung-Ming G.  Lin</span>, <span class="hlFld-ContribAuthor ">Bettina  Gürtl</span>, <span class="hlFld-ContribAuthor ">Daniela  Reil</span>, <span class="hlFld-ContribAuthor ">Kristaps  Klavins</span>, <span class="hlFld-ContribAuthor ">Stefan  Kubicek</span>, <span class="hlFld-ContribAuthor ">Sara  Sdelci</span>, <span class="hlFld-ContribAuthor ">Giulio  Superti-Furga</span>. </span><span class="cited-content_cbyCitation_article-title">Cell-surface SLC nucleoside transporters and purine levels modulate BRD4-dependent chromatin states. </span><span class="cited-content_cbyCitation_journal-name">Nature Metabolism</span><span> <strong>2021,</strong> <em>3 </em>
                                    (5)
                                     , 651-664. <a href="https://doi.org/10.1038/s42255-021-00386-8" title="DOI URL">https://doi.org/10.1038/s42255-021-00386-8</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s42255-021-00386-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs42255-021-00386-8%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Metabolism%26atitle%3DCell-surface%252BSLC%252Bnucleoside%252Btransporters%252Band%252Bpurine%252Blevels%252Bmodulate%252BBRD4-dependent%252Bchromatin%252Bstates%26aulast%3DLi%26aufirst%3DKai-Chun%26date%3D2021%26date%3D2021%26volume%3D3%26issue%3D5%26spage%3D651%26epage%3D664" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Pata-Eting  Kougnassoukou Tchara</span>, <span class="hlFld-ContribAuthor ">Panagis  Filippakopoulos</span>, <span class="hlFld-ContribAuthor ">Jean-Philippe  Lambert</span>. </span><span class="cited-content_cbyCitation_article-title">Emerging tools to investigate bromodomain functions. </span><span class="cited-content_cbyCitation_journal-name">Methods</span><span> <strong>2020,</strong> <em>184 </em>, 40-52. <a href="https://doi.org/10.1016/j.ymeth.2019.11.003" title="DOI URL">https://doi.org/10.1016/j.ymeth.2019.11.003</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ymeth.2019.11.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ymeth.2019.11.003%26sid%3Dliteratum%253Aachs%26jtitle%3DMethods%26atitle%3DEmerging%252Btools%252Bto%252Binvestigate%252Bbromodomain%252Bfunctions%26aulast%3DKougnassoukou%2BTchara%26aufirst%3DPata-Eting%26date%3D2020%26volume%3D184%26spage%3D40%26epage%3D52" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Srihari  Konduri</span>, <span class="hlFld-ContribAuthor ">Jyothi  Prashanth</span>, <span class="hlFld-ContribAuthor ">Vagolu Siva  Krishna</span>, <span class="hlFld-ContribAuthor ">Dharmarajan  Sriram</span>, <span class="hlFld-ContribAuthor ">J.N.  Behera</span>, <span class="hlFld-ContribAuthor ">Dionicio  Siegel</span>, <span class="hlFld-ContribAuthor ">Koya Prabhakara  Rao</span>. </span><span class="cited-content_cbyCitation_article-title">Design and synthesis of purine connected piperazine derivatives as novel inhibitors of Mycobacterium tuberculosis. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2020,</strong> <em>30 </em>
                                    (22)
                                     , 127512. <a href="https://doi.org/10.1016/j.bmcl.2020.127512" title="DOI URL">https://doi.org/10.1016/j.bmcl.2020.127512</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2020.127512&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2020.127512%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DDesign%252Band%252Bsynthesis%252Bof%252Bpurine%252Bconnected%252Bpiperazine%252Bderivatives%252Bas%252Bnovel%252Binhibitors%252Bof%252BMycobacterium%252Btuberculosis%26aulast%3DKonduri%26aufirst%3DSrihari%26date%3D2020%26volume%3D30%26issue%3D22%26spage%3D127512" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Daniela  Tomaselli</span>, <span class="hlFld-ContribAuthor ">Alessia  Lucidi</span>, <span class="hlFld-ContribAuthor ">Dante  Rotili</span>, <span class="hlFld-ContribAuthor ">Antonello  Mai</span>. </span><span class="cited-content_cbyCitation_article-title">Epigenetic polypharmacology: A new frontier for epi‐drug discovery. </span><span class="cited-content_cbyCitation_journal-name">Medicinal Research Reviews</span><span> <strong>2020,</strong> <em>40 </em>
                                    (1)
                                     , 190-244. <a href="https://doi.org/10.1002/med.21600" title="DOI URL">https://doi.org/10.1002/med.21600</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/med.21600&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fmed.21600%26sid%3Dliteratum%253Aachs%26jtitle%3DMedicinal%2520Research%2520Reviews%26atitle%3DEpigenetic%252Bpolypharmacology%25253A%252BA%252Bnew%252Bfrontier%252Bfor%252Bepi%2525E2%252580%252590drug%252Bdiscovery%26aulast%3DTomaselli%26aufirst%3DDaniela%26date%3D2020%26date%3D2019%26volume%3D40%26issue%3D1%26spage%3D190%26epage%3D244" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Abdellatif  Tikad</span>, <span class="hlFld-ContribAuthor ">Saïd  El Kazzouli</span>, <span class="hlFld-ContribAuthor ">Gérald  Guillaumet</span>. </span><span class="cited-content_cbyCitation_article-title">1,4-Dioxins, Oxathiins, Dithiins and Their Benzo Derivatives. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,<a href="https://doi.org/10.1016/B978-0-12-818655-8.00015-9" title="DOI URL">https://doi.org/10.1016/B978-0-12-818655-8.00015-9</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-818655-8.00015-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-818655-8.00015-9%26sid%3Dliteratum%253Aachs%26atitle%3D1%25252C4-Dioxins%25252C%252BOxathiins%25252C%252BDithiins%252Band%252BTheir%252BBenzo%252BDerivatives%26aulast%3DTikad%26aufirst%3DAbdellatif%26date%3D2020%26pub%3DElsevier%26atitle%3DReference%252BModule%252Bin%252BChemistry%25252C%252BMolecular%252BSciences%252Band%252BChemical%252BEngineering%26date%3D2020%26volume%3D32" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhongyi  Hu</span>, <span class="hlFld-ContribAuthor ">Junzhi  Zhou</span>, <span class="hlFld-ContribAuthor ">Junjie  Jiang</span>, <span class="hlFld-ContribAuthor ">Jiao  Yuan</span>, <span class="hlFld-ContribAuthor ">Youyou  Zhang</span>, <span class="hlFld-ContribAuthor ">Xuepeng  Wei</span>, <span class="hlFld-ContribAuthor ">Nicki  Loo</span>, <span class="hlFld-ContribAuthor ">Yueying  Wang</span>, <span class="hlFld-ContribAuthor ">Yutian  Pan</span>, <span class="hlFld-ContribAuthor ">Tianli  Zhang</span>, <span class="hlFld-ContribAuthor ">Xiaomin  Zhong</span>, <span class="hlFld-ContribAuthor ">Meixiao  Long</span>, <span class="hlFld-ContribAuthor ">Kathleen T.  Montone</span>, <span class="hlFld-ContribAuthor ">Janos L.  Tanyi</span>, <span class="hlFld-ContribAuthor ">Yi  Fan</span>, <span class="hlFld-ContribAuthor ">Tian-Li  Wang</span>, <span class="hlFld-ContribAuthor ">Ie-Ming  Shih</span>, <span class="hlFld-ContribAuthor ">Xiaowen  Hu</span>, <span class="hlFld-ContribAuthor ">Lin  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Genomic characterization of genes encoding histone acetylation modulator proteins identifies therapeutic targets for cancer treatment. </span><span class="cited-content_cbyCitation_journal-name">Nature Communications</span><span> <strong>2019,</strong> <em>10 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41467-019-08554-x" title="DOI URL">https://doi.org/10.1038/s41467-019-08554-x</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41467-019-08554-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41467-019-08554-x%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Communications%26atitle%3DGenomic%252Bcharacterization%252Bof%252Bgenes%252Bencoding%252Bhistone%252Bacetylation%252Bmodulator%252Bproteins%252Bidentifies%252Btherapeutic%252Btargets%252Bfor%252Bcancer%252Btreatment%26aulast%3DHu%26aufirst%3DZhongyi%26date%3D2019%26date%3D2019%26volume%3D10%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Nilesh  Zaware</span>, <span class="hlFld-ContribAuthor ">Ming-Ming  Zhou</span>. </span><span class="cited-content_cbyCitation_article-title">Bromodomain biology and drug discovery. </span><span class="cited-content_cbyCitation_journal-name">Nature Structural & Molecular Biology</span><span> <strong>2019,</strong> <em>26 </em>
                                    (10)
                                     , 870-879. <a href="https://doi.org/10.1038/s41594-019-0309-8" title="DOI URL">https://doi.org/10.1038/s41594-019-0309-8</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41594-019-0309-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41594-019-0309-8%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Structural%2520%2526%2520Molecular%2520Biology%26atitle%3DBromodomain%252Bbiology%252Band%252Bdrug%252Bdiscovery%26aulast%3DZaware%26aufirst%3DNilesh%26date%3D2019%26date%3D2019%26volume%3D26%26issue%3D10%26spage%3D870%26epage%3D879" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Andrea G.  Cochran</span>, <span class="hlFld-ContribAuthor ">Andrew R.  Conery</span>, <span class="hlFld-ContribAuthor ">Robert J.  Sims</span>. </span><span class="cited-content_cbyCitation_article-title">Bromodomains: a new target class for drug development. </span><span class="cited-content_cbyCitation_journal-name">Nature Reviews Drug Discovery</span><span> <strong>2019,</strong> <em>18 </em>
                                    (8)
                                     , 609-628. <a href="https://doi.org/10.1038/s41573-019-0030-7" title="DOI URL">https://doi.org/10.1038/s41573-019-0030-7</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41573-019-0030-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41573-019-0030-7%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Reviews%2520Drug%2520Discovery%26atitle%3DBromodomains%25253A%252Ba%252Bnew%252Btarget%252Bclass%252Bfor%2525C2%2525A0drug%252Bdevelopment%26aulast%3DCochran%26aufirst%3DAndrea%2BG.%26date%3D2019%26date%3D2019%26volume%3D18%26issue%3D8%26spage%3D609%26epage%3D628" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Helai P.  Mohammad</span>, <span class="hlFld-ContribAuthor ">Olena  Barbash</span>, <span class="hlFld-ContribAuthor ">Caretha L.  Creasy</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting epigenetic modifications in cancer therapy: erasing the roadmap to cancer. </span><span class="cited-content_cbyCitation_journal-name">Nature Medicine</span><span> <strong>2019,</strong> <em>25 </em>
                                    (3)
                                     , 403-418. <a href="https://doi.org/10.1038/s41591-019-0376-8" title="DOI URL">https://doi.org/10.1038/s41591-019-0376-8</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41591-019-0376-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41591-019-0376-8%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Medicine%26atitle%3DTargeting%252Bepigenetic%252Bmodifications%252Bin%252Bcancer%252Btherapy%25253A%252Berasing%252Bthe%252Broadmap%252Bto%252Bcancer%26aulast%3DMohammad%26aufirst%3DHelai%2BP.%26date%3D2019%26date%3D2019%26volume%3D25%26issue%3D3%26spage%3D403%26epage%3D418" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tomomi  Noguchi-Yachide</span>. </span><span class="cited-content_cbyCitation_article-title">Regulators of Histone Acetylation: Bromodomain Inhibitors. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 463-473. <a href="https://doi.org/10.1016/B978-0-12-813939-4.00010-3" title="DOI URL">https://doi.org/10.1016/B978-0-12-813939-4.00010-3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-813939-4.00010-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-813939-4.00010-3%26sid%3Dliteratum%253Aachs%26atitle%3DRegulators%252Bof%252BHistone%252BAcetylation%25253A%252BBromodomain%252BInhibitors%26aulast%3DNoguchi-Yachide%26aufirst%3DTomomi%26date%3D2019%26spage%3D463%26epage%3D473%26pub%3DElsevier%26atitle%3DPharmacoepigenetics%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ma  Hui</span>, <span class="hlFld-ContribAuthor ">Zhang  Jian</span>, <span class="hlFld-ContribAuthor ">Zheng  Peiyuan</span>, <span class="hlFld-ContribAuthor ">Wu  Zhenwei</span>, <span class="hlFld-ContribAuthor ">Zhang  Huibin</span>. </span><span class="cited-content_cbyCitation_article-title">Research progress of selective small molecule bromodomain-containing protein 9 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2018,</strong> <em>10 </em>
                                    (8)
                                     , 895-906. <a href="https://doi.org/10.4155/fmc-2017-0243" title="DOI URL">https://doi.org/10.4155/fmc-2017-0243</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc-2017-0243&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc-2017-0243%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DResearch%252Bprogress%252Bof%252Bselective%252Bsmall%252Bmolecule%252Bbromodomain-containing%252Bprotein%252B9%252Binhibitors%26aulast%3DHui%26aufirst%3DMa%26date%3D2018%26volume%3D10%26issue%3D8%26spage%3D895%26epage%3D906" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jonathan T.  Lloyd</span>, <span class="hlFld-ContribAuthor ">Karen C.  Glass</span>. </span><span class="cited-content_cbyCitation_article-title">Biological function and histone recognition of family IV bromodomain-containing proteins. </span><span class="cited-content_cbyCitation_journal-name">Journal of Cellular Physiology</span><span> <strong>2018,</strong> <em>233 </em>
                                    (3)
                                     , 1877-1886. <a href="https://doi.org/10.1002/jcp.26010" title="DOI URL">https://doi.org/10.1002/jcp.26010</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/jcp.26010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fjcp.26010%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Cellular%2520Physiology%26atitle%3DBiological%252Bfunction%252Band%252Bhistone%252Brecognition%252Bof%252Bfamily%252BIV%252Bbromodomain-containing%252Bproteins%26aulast%3DLloyd%26aufirst%3DJonathan%2BT.%26date%3D2018%26date%3D2017%26volume%3D233%26issue%3D3%26spage%3D1877%26epage%3D1886" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Guoyi  Yan</span>, <span class="hlFld-ContribAuthor ">Manzhou  Hou</span>, <span class="hlFld-ContribAuthor ">Jiang  Luo</span>, <span class="hlFld-ContribAuthor ">Chunlan  Pu</span>, <span class="hlFld-ContribAuthor ">Xueyan  Hou</span>, <span class="hlFld-ContribAuthor ">Suke  Lan</span>, <span class="hlFld-ContribAuthor ">Rui  Li</span>. </span><span class="cited-content_cbyCitation_article-title">Pharmacophore-based virtual screening, molecular docking, molecular dynamics simulation, and biological evaluation for the discovery of novel BRD4 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Chemical Biology & Drug Design</span><span> <strong>2018,</strong> <em>91 </em>
                                    (2)
                                     , 478-490. <a href="https://doi.org/10.1111/cbdd.13109" title="DOI URL">https://doi.org/10.1111/cbdd.13109</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1111/cbdd.13109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1111%2Fcbdd.13109%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Biology%2520%2526%2520Drug%2520Design%26atitle%3DPharmacophore-based%252Bvirtual%252Bscreening%25252C%252Bmolecular%252Bdocking%25252C%252Bmolecular%252Bdynamics%252Bsimulation%25252C%252Band%252Bbiological%252Bevaluation%252Bfor%252Bthe%252Bdiscovery%252Bof%252Bnovel%252BBRD4%252Binhibitors%26aulast%3DYan%26aufirst%3DGuoyi%26date%3D2018%26date%3D2017%26volume%3D91%26issue%3D2%26spage%3D478%26epage%3D490" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ran  Zhao</span>, <span class="hlFld-ContribAuthor ">Yukun  Liu</span>, <span class="hlFld-ContribAuthor ">Heran  Wang</span>, <span class="hlFld-ContribAuthor ">Jing  Yang</span>, <span class="hlFld-ContribAuthor ">Weihong  Niu</span>, <span class="hlFld-ContribAuthor ">Songqing  Fan</span>, <span class="hlFld-ContribAuthor ">Wei  Xiong</span>, <span class="hlFld-ContribAuthor ">Jian  Ma</span>, <span class="hlFld-ContribAuthor ">Xiaoling  Li</span>, <span class="hlFld-ContribAuthor ">Joshua B  Phillips</span>, <span class="hlFld-ContribAuthor ">Ming  Tan</span>, <span class="hlFld-ContribAuthor ">Yuanzheng  Qiu</span>, <span class="hlFld-ContribAuthor ">Guiyuan  Li</span>, <span class="hlFld-ContribAuthor ">Ming  Zhou</span>. </span><span class="cited-content_cbyCitation_article-title">BRD7 plays an anti-inflammatory role during early acute inflammation by inhibiting activation of the NF-кB signaling pathway. </span><span class="cited-content_cbyCitation_journal-name">Cellular & Molecular Immunology</span><span> <strong>2017,</strong> <em>14 </em>
                                    (10)
                                     , 830-841. <a href="https://doi.org/10.1038/cmi.2016.31" title="DOI URL">https://doi.org/10.1038/cmi.2016.31</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/cmi.2016.31&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fcmi.2016.31%26sid%3Dliteratum%253Aachs%26jtitle%3DCellular%2520%2526%2520Molecular%2520Immunology%26atitle%3DBRD7%252Bplays%252Ban%252Banti-inflammatory%252Brole%252Bduring%252Bearly%252Bacute%252Binflammation%252Bby%252Binhibiting%252Bactivation%252Bof%252Bthe%252BNF-%2525D0%2525BAB%252Bsignaling%252Bpathway%26aulast%3DZhao%26aufirst%3DRan%26date%3D2017%26date%3D2016%26volume%3D14%26issue%3D10%26spage%3D830%26epage%3D841" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Maria  Petrosino</span>, <span class="hlFld-ContribAuthor ">Daniela  Bonetti</span>, <span class="hlFld-ContribAuthor ">Alessandra  Pasquo</span>, <span class="hlFld-ContribAuthor ">Laura  Lori</span>, <span class="hlFld-ContribAuthor ">Roberta  Chiaraluce</span>, <span class="hlFld-ContribAuthor ">Valerio  Consalvi</span>, <span class="hlFld-ContribAuthor ">Carlo  Travaglini-Allocatelli</span>. </span><span class="cited-content_cbyCitation_article-title">Unveiling the folding mechanism of the Bromodomains. </span><span class="cited-content_cbyCitation_journal-name">Biochemistry and Biophysics Reports</span><span> <strong>2017,</strong> <em>11 </em>, 99-104. <a href="https://doi.org/10.1016/j.bbrep.2017.06.009" title="DOI URL">https://doi.org/10.1016/j.bbrep.2017.06.009</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bbrep.2017.06.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bbrep.2017.06.009%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemistry%2520and%2520Biophysics%2520Reports%26atitle%3DUnveiling%252Bthe%252Bfolding%252Bmechanism%252Bof%252Bthe%252BBromodomains%26aulast%3DPetrosino%26aufirst%3DMaria%26date%3D2017%26volume%3D11%26spage%3D99%26epage%3D104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Seika  Amemiya</span>, <span class="hlFld-ContribAuthor ">Takao  Yamaguchi</span>, <span class="hlFld-ContribAuthor ">Yuichi  Hashimoto</span>, <span class="hlFld-ContribAuthor ">Tomomi  Noguchi-Yachide</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis and evaluation of novel dual BRD4/HDAC inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2017,</strong> <em>25 </em>
                                    (14)
                                     , 3677-3684. <a href="https://doi.org/10.1016/j.bmc.2017.04.043" title="DOI URL">https://doi.org/10.1016/j.bmc.2017.04.043</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2017.04.043&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2017.04.043%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DSynthesis%252Band%252Bevaluation%252Bof%252Bnovel%252Bdual%252BBRD4%25252FHDAC%252Binhibitors%26aulast%3DAmemiya%26aufirst%3DSeika%26date%3D2017%26volume%3D25%26issue%3D14%26spage%3D3677%26epage%3D3684" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Miao  Tian</span>, <span class="hlFld-ContribAuthor ">Mingwu  Yu</span>, <span class="hlFld-ContribAuthor ">Tingting  Shi</span>, <span class="hlFld-ContribAuthor ">Junbin  Hu</span>, <span class="hlFld-ContribAuthor ">Shunlai  Li</span>, <span class="hlFld-ContribAuthor ">Jiaxi  Xu</span>, <span class="hlFld-ContribAuthor ">Ning  Chen</span>, <span class="hlFld-ContribAuthor ">Hongguang  Du</span>. </span><span class="cited-content_cbyCitation_article-title">Silver-Catalyzed Direct C6-H Arylation of Purines and Purine Nucleosides with Arylboronic Acids. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Organic Chemistry</span><span> <strong>2017,</strong> <em>2017 </em>
                                    (24)
                                     , 3415-3420. <a href="https://doi.org/10.1002/ejoc.201700406" title="DOI URL">https://doi.org/10.1002/ejoc.201700406</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ejoc.201700406&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fejoc.201700406%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Organic%2520Chemistry%26atitle%3DSilver-Catalyzed%252BDirect%252BC6-H%252BArylation%252Bof%252BPurines%252Band%252BPurine%252BNucleosides%252Bwith%252BArylboronic%252BAcids%26aulast%3DTian%26aufirst%3DMiao%26date%3D2017%26date%3D2017%26volume%3D2017%26issue%3D24%26spage%3D3415%26epage%3D3420" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">David  Remillard</span>, <span class="hlFld-ContribAuthor ">Dennis L.  Buckley</span>, <span class="hlFld-ContribAuthor ">Joshiawa  Paulk</span>, <span class="hlFld-ContribAuthor ">Gerard L.  Brien</span>, <span class="hlFld-ContribAuthor ">Matthew  Sonnett</span>, <span class="hlFld-ContribAuthor ">Hyuk-Soo  Seo</span>, <span class="hlFld-ContribAuthor ">Shiva  Dastjerdi</span>, <span class="hlFld-ContribAuthor ">Martin  Wühr</span>, <span class="hlFld-ContribAuthor ">Sirano  Dhe-Paganon</span>, <span class="hlFld-ContribAuthor ">Scott A.  Armstrong</span>, <span class="hlFld-ContribAuthor ">James E.  Bradner</span>. </span><span class="cited-content_cbyCitation_article-title">Degradation of the BAF Complex Factor BRD9 by Heterobifunctional Ligands. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie</span><span> <strong>2017,</strong> <em>129 </em>
                                    (21)
                                     , 5832-5837. <a href="https://doi.org/10.1002/ange.201611281" title="DOI URL">https://doi.org/10.1002/ange.201611281</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ange.201611281&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fange.201611281%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%26atitle%3DDegradation%252Bof%252Bthe%252BBAF%252BComplex%252BFactor%252BBRD9%252Bby%252BHeterobifunctional%252BLigands%26aulast%3DRemillard%26aufirst%3DDavid%26date%3D2017%26date%3D2017%26volume%3D129%26issue%3D21%26spage%3D5832%26epage%3D5837" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">David  Remillard</span>, <span class="hlFld-ContribAuthor ">Dennis L.  Buckley</span>, <span class="hlFld-ContribAuthor ">Joshiawa  Paulk</span>, <span class="hlFld-ContribAuthor ">Gerard L.  Brien</span>, <span class="hlFld-ContribAuthor ">Matthew  Sonnett</span>, <span class="hlFld-ContribAuthor ">Hyuk-Soo  Seo</span>, <span class="hlFld-ContribAuthor ">Shiva  Dastjerdi</span>, <span class="hlFld-ContribAuthor ">Martin  Wühr</span>, <span class="hlFld-ContribAuthor ">Sirano  Dhe-Paganon</span>, <span class="hlFld-ContribAuthor ">Scott A.  Armstrong</span>, <span class="hlFld-ContribAuthor ">James E.  Bradner</span>. </span><span class="cited-content_cbyCitation_article-title">Degradation of the BAF Complex Factor BRD9 by Heterobifunctional Ligands. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie International Edition</span><span> <strong>2017,</strong> <em>56 </em>
                                    (21)
                                     , 5738-5743. <a href="https://doi.org/10.1002/anie.201611281" title="DOI URL">https://doi.org/10.1002/anie.201611281</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/anie.201611281&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fanie.201611281%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%2520International%2520Edition%26atitle%3DDegradation%252Bof%252Bthe%252BBAF%252BComplex%252BFactor%252BBRD9%252Bby%252BHeterobifunctional%252BLigands%26aulast%3DRemillard%26aufirst%3DDavid%26date%3D2017%26date%3D2017%26volume%3D56%26issue%3D21%26spage%3D5738%26epage%3D5743" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Montserrat  Pérez-Salvia</span>, <span class="hlFld-ContribAuthor ">Manel  Esteller</span>. </span><span class="cited-content_cbyCitation_article-title">Bromodomain inhibitors and cancer therapy: From structures to applications. </span><span class="cited-content_cbyCitation_journal-name">Epigenetics</span><span> <strong>2017,</strong> <em>12 </em>
                                    (5)
                                     , 323-339. <a href="https://doi.org/10.1080/15592294.2016.1265710" title="DOI URL">https://doi.org/10.1080/15592294.2016.1265710</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/15592294.2016.1265710&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F15592294.2016.1265710%26sid%3Dliteratum%253Aachs%26jtitle%3DEpigenetics%26atitle%3DBromodomain%252Binhibitors%252Band%252Bcancer%252Btherapy%25253A%252BFrom%252Bstructures%252Bto%252Bapplications%26aulast%3DP%25C3%25A9rez-Salvia%26aufirst%3DMontserrat%26date%3D2017%26date%3D2016%26volume%3D12%26issue%3D5%26spage%3D323%26epage%3D339" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Maria G.  Chini</span>, <span class="hlFld-ContribAuthor ">Nicola  Malafronte</span>, <span class="hlFld-ContribAuthor ">Maria C.  Vaccaro</span>, <span class="hlFld-ContribAuthor ">Maria J.  Gualtieri</span>, <span class="hlFld-ContribAuthor ">Antonio  Vassallo</span>, <span class="hlFld-ContribAuthor ">Michele  Vasaturo</span>, <span class="hlFld-ContribAuthor ">Sabrina  Castellano</span>, <span class="hlFld-ContribAuthor ">Ciro  Milite</span>, <span class="hlFld-ContribAuthor ">Antonietta  Leone</span>, <span class="hlFld-ContribAuthor ">Giuseppe  Bifulco</span>, <span class="hlFld-ContribAuthor ">Nunziatina  De Tommasi</span>, <span class="hlFld-ContribAuthor ">Fabrizio  Dal Piaz</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of Limonol Derivatives as Heat Shock Protein 90 (Hsp90) Inhibitors through a Multidisciplinary Approach. </span><span class="cited-content_cbyCitation_journal-name">Chemistry - A European Journal</span><span> <strong>2016,</strong> <em>22 </em>
                                    (37)
                                     , 13236-13250. <a href="https://doi.org/10.1002/chem.201602242" title="DOI URL">https://doi.org/10.1002/chem.201602242</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/chem.201602242&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fchem.201602242%26sid%3Dliteratum%253Aachs%26jtitle%3DChemistry%2520-%2520A%2520European%2520Journal%26atitle%3DIdentification%252Bof%252BLimonol%252BDerivatives%252Bas%252BHeat%252BShock%252BProtein%2525E2%252580%25258590%252B%252528Hsp90%252529%252BInhibitors%252Bthrough%252Ba%252BMultidisciplinary%252BApproach%26aulast%3DChini%26aufirst%3DMaria%2BG.%26date%3D2016%26date%3D2016%26volume%3D22%26issue%3D37%26spage%3D13236%26epage%3D13250" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Carles  Galdeano</span>, <span class="hlFld-ContribAuthor ">Alessio  Ciulli</span>. </span><span class="cited-content_cbyCitation_article-title">Selectivity on-target of bromodomain chemical probes by structure-guided medicinal chemistry and chemical biology. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2016,</strong> <em>8 </em>
                                    (13)
                                     , 1655-1680. <a href="https://doi.org/10.4155/fmc-2016-0059" title="DOI URL">https://doi.org/10.4155/fmc-2016-0059</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc-2016-0059&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc-2016-0059%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DSelectivity%252Bon-target%252Bof%252Bbromodomain%252Bchemical%252Bprobes%252Bby%252Bstructure-guided%252Bmedicinal%252Bchemistry%252Band%252Bchemical%252Bbiology%26aulast%3DGaldeano%26aufirst%3DCarles%26date%3D2016%26volume%3D8%26issue%3D13%26spage%3D1655%26epage%3D1680" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Na  Wang</span>, <span class="hlFld-ContribAuthor ">Fudong  Li</span>, <span class="hlFld-ContribAuthor ">Hongyu  Bao</span>, <span class="hlFld-ContribAuthor ">Jie  Li</span>, <span class="hlFld-ContribAuthor ">Jihui  Wu</span>, <span class="hlFld-ContribAuthor ">Ke  Ruan</span>. </span><span class="cited-content_cbyCitation_article-title">NMR Fragment Screening Hit Induces Plasticity of BRD7/9 Bromodomains. </span><span class="cited-content_cbyCitation_journal-name">ChemBioChem</span><span> <strong>2016,</strong> <em>17 </em>
                                    (15)
                                     , 1456-1463. <a href="https://doi.org/10.1002/cbic.201600184" title="DOI URL">https://doi.org/10.1002/cbic.201600184</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cbic.201600184&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcbic.201600184%26sid%3Dliteratum%253Aachs%26jtitle%3DChemBioChem%26atitle%3DNMR%252BFragment%252BScreening%252BHit%252BInduces%252BPlasticity%252Bof%252BBRD7%25252F9%252BBromodomains%26aulast%3DWang%26aufirst%3DNa%26date%3D2016%26date%3D2016%26volume%3D17%26issue%3D15%26spage%3D1456%26epage%3D1463" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Natalie H  Theodoulou</span>, <span class="hlFld-ContribAuthor ">Nicholas CO  Tomkinson</span>, <span class="hlFld-ContribAuthor ">Rab K  Prinjha</span>, <span class="hlFld-ContribAuthor ">Philip G  Humphreys</span>. </span><span class="cited-content_cbyCitation_article-title">Clinical progress and pharmacology of small molecule bromodomain inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Chemical Biology</span><span> <strong>2016,</strong> <em>33 </em>, 58-66. <a href="https://doi.org/10.1016/j.cbpa.2016.05.028" title="DOI URL">https://doi.org/10.1016/j.cbpa.2016.05.028</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cbpa.2016.05.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cbpa.2016.05.028%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Chemical%2520Biology%26atitle%3DClinical%252Bprogress%252Band%252Bpharmacology%252Bof%252Bsmall%252Bmolecule%252Bbromodomain%252Binhibitors%26aulast%3DTheodoulou%26aufirst%3DNatalie%2BH%26date%3D2016%26volume%3D33%26spage%3D58%26epage%3D66" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Paolo  Filipponi</span>, <span class="hlFld-ContribAuthor ">Ian R.  Baxendale</span>. </span><span class="cited-content_cbyCitation_article-title">The Generation of a Library of ­Bromodomain-Containing Protein Modulators Expedited by Continuous Flow Synthesis. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Organic Chemistry</span><span> <strong>2016,</strong> <em>2016 </em>
                                    (11)
                                     , 2000-2012. <a href="https://doi.org/10.1002/ejoc.201600222" title="DOI URL">https://doi.org/10.1002/ejoc.201600222</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ejoc.201600222&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fejoc.201600222%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Organic%2520Chemistry%26atitle%3DThe%252BGeneration%252Bof%252Ba%252BLibrary%252Bof%252B%2525C2%2525ADBromodomain-Containing%252BProtein%252BModulators%252BExpedited%252Bby%252BContinuous%252BFlow%252BSynthesis%26aulast%3DFilipponi%26aufirst%3DPaolo%26date%3D2016%26date%3D2016%26volume%3D2016%26issue%3D11%26spage%3D2000%26epage%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Elena  Ferri</span>, <span class="hlFld-ContribAuthor ">Carlo  Petosa</span>, <span class="hlFld-ContribAuthor ">Charles E.  McKenna</span>. </span><span class="cited-content_cbyCitation_article-title">Bromodomains: Structure, function and pharmacology of inhibition. </span><span class="cited-content_cbyCitation_journal-name">Biochemical Pharmacology</span><span> <strong>2016,</strong> <em>106 </em>, 1-18. <a href="https://doi.org/10.1016/j.bcp.2015.12.005" title="DOI URL">https://doi.org/10.1016/j.bcp.2015.12.005</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bcp.2015.12.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bcp.2015.12.005%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemical%2520Pharmacology%26atitle%3DBromodomains%25253A%252BStructure%25252C%252Bfunction%252Band%252Bpharmacology%252Bof%252Binhibition%26aulast%3DFerri%26aufirst%3DElena%26date%3D2016%26volume%3D106%26spage%3D1%26epage%3D18" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Natalie H.  Theodoulou</span>, <span class="hlFld-ContribAuthor ">Nicholas C. O.  Tomkinson</span>, <span class="hlFld-ContribAuthor ">Rab K.  Prinjha</span>, <span class="hlFld-ContribAuthor ">Philip G.  Humphreys</span>. </span><span class="cited-content_cbyCitation_article-title">Progress in the Development of non-BET Bromodomain Chemical Probes. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2016,</strong> <em>11 </em>
                                    (5)
                                     , 477-487. <a href="https://doi.org/10.1002/cmdc.201500540" title="DOI URL">https://doi.org/10.1002/cmdc.201500540</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.201500540&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.201500540%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DProgress%252Bin%252Bthe%252BDevelopment%252Bof%252Bnon-BET%252BBromodomain%252BChemical%252BProbes%26aulast%3DTheodoulou%26aufirst%3DNatalie%2BH.%26date%3D2016%26date%3D2016%26volume%3D11%26issue%3D5%26spage%3D477%26epage%3D487" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Peter G.K.  Clark</span>, <span class="hlFld-ContribAuthor ">Darren J.  Dixon</span>, <span class="hlFld-ContribAuthor ">Paul E.  Brennan</span>. </span><span class="cited-content_cbyCitation_article-title">Development of chemical probes for the bromodomains of BRD7 and BRD9. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today: Technologies</span><span> <strong>2016,</strong> <em>19 </em>, 73-80. <a href="https://doi.org/10.1016/j.ddtec.2016.05.002" title="DOI URL">https://doi.org/10.1016/j.ddtec.2016.05.002</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ddtec.2016.05.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ddtec.2016.05.002%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technologies%26atitle%3DDevelopment%252Bof%252Bchemical%252Bprobes%252Bfor%252Bthe%252Bbromodomains%252Bof%252BBRD7%252Band%252BBRD9%26aulast%3DClark%26aufirst%3DPeter%2BG.K.%26date%3D2016%26volume%3D19%26spage%3D73%26epage%3D80" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dimitrios  Spiliotopoulos</span>, <span class="hlFld-ContribAuthor ">Amedeo  Caflisch</span>. </span><span class="cited-content_cbyCitation_article-title">Fragment-based in silico screening of bromodomain ligands. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today: Technologies</span><span> <strong>2016,</strong> <em>19 </em>, 81-90. <a href="https://doi.org/10.1016/j.ddtec.2016.06.003" title="DOI URL">https://doi.org/10.1016/j.ddtec.2016.06.003</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ddtec.2016.06.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ddtec.2016.06.003%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technologies%26atitle%3DFragment-based%252Bin%252Bsilico%252Bscreening%252Bof%252Bbromodomain%252Bligands%26aulast%3DSpiliotopoulos%26aufirst%3DDimitrios%26date%3D2016%26volume%3D19%26spage%3D81%26epage%3D90" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jamel  Meslamani</span>, <span class="hlFld-ContribAuthor ">Steven G.  Smith</span>, <span class="hlFld-ContribAuthor ">Roberto  Sanchez</span>, <span class="hlFld-ContribAuthor ">Ming-Ming  Zhou</span>. </span><span class="cited-content_cbyCitation_article-title">Structural features and inhibitors of bromodomains. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today: Technologies</span><span> <strong>2016,</strong> <em>19 </em>, 3-15. <a href="https://doi.org/10.1016/j.ddtec.2016.09.001" title="DOI URL">https://doi.org/10.1016/j.ddtec.2016.09.001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ddtec.2016.09.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ddtec.2016.09.001%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technologies%26atitle%3DStructural%252Bfeatures%252Band%252Binhibitors%252Bof%252Bbromodomains%26aulast%3DMeslamani%26aufirst%3DJamel%26date%3D2016%26volume%3D19%26spage%3D3%26epage%3D15" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">S.  Teracciano</span>, <span class="hlFld-ContribAuthor ">M. G.  Chini</span>, <span class="hlFld-ContribAuthor ">M. C.  Vaccaro</span>, <span class="hlFld-ContribAuthor ">M.  Strocchia</span>, <span class="hlFld-ContribAuthor ">A.  Foglia</span>, <span class="hlFld-ContribAuthor ">A.  Vassallo</span>, <span class="hlFld-ContribAuthor ">C.  Saturnino</span>, <span class="hlFld-ContribAuthor ">R.  Riccio</span>, <span class="hlFld-ContribAuthor ">G.  Bifulco</span>, <span class="hlFld-ContribAuthor ">I.  Bruno</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of the key structural elements of a dihydropyrimidinone core driving toward more potent Hsp90 C-terminal inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Chemical Communications</span><span> <strong>2016,</strong> <em>52 </em>
                                    (87)
                                     , 12857-12860. <a href="https://doi.org/10.1039/C6CC06379A" title="DOI URL">https://doi.org/10.1039/C6CC06379A</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C6CC06379A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC6CC06379A%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Communications%26atitle%3DIdentification%252Bof%252Bthe%252Bkey%252Bstructural%252Belements%252Bof%252Ba%252Bdihydropyrimidinone%252Bcore%252Bdriving%252Btoward%252Bmore%252Bpotent%252BHsp90%252BC-terminal%252Binhibitors%26aulast%3DTeracciano%26aufirst%3DS.%26date%3D2016%26date%3D2016%26volume%3D52%26issue%3D87%26spage%3D12857%26epage%3D12860" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zinaida M.  Kaskova</span>, <span class="hlFld-ContribAuthor ">Aleksandra S.  Tsarkova</span>, <span class="hlFld-ContribAuthor ">Ilia V.  Yampolsky</span>. </span><span class="cited-content_cbyCitation_article-title">1001 lights: luciferins, luciferases, their mechanisms of action and applications in chemical analysis, biology and medicine. </span><span class="cited-content_cbyCitation_journal-name">Chemical Society Reviews</span><span> <strong>2016,</strong> <em>45 </em>
                                    (21)
                                     , 6048-6077. <a href="https://doi.org/10.1039/C6CS00296J" title="DOI URL">https://doi.org/10.1039/C6CS00296J</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C6CS00296J&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC6CS00296J%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Society%2520Reviews%26atitle%3D1001%252Blights%25253A%252Bluciferins%25252C%252Bluciferases%25252C%252Btheir%252Bmechanisms%252Bof%252Baction%252Band%252Bapplications%252Bin%252Bchemical%252Banalysis%25252C%252Bbiology%252Band%252Bmedicine%26aulast%3DKaskova%26aufirst%3DZinaida%2BM.%26date%3D2016%26date%3D2016%26volume%3D45%26issue%3D21%26spage%3D6048%26epage%3D6077" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Moses  Moustakim</span>, <span class="hlFld-ContribAuthor ">Peter G. K.  Clark</span>, <span class="hlFld-ContribAuthor ">Duncan A.  Hay</span>, <span class="hlFld-ContribAuthor ">Darren J.  Dixon</span>, <span class="hlFld-ContribAuthor ">Paul E.  Brennan</span>. </span><span class="cited-content_cbyCitation_article-title">Chemical probes and inhibitors of bromodomains outside the BET family. </span><span class="cited-content_cbyCitation_journal-name">MedChemComm</span><span> <strong>2016,</strong> <em>7 </em>
                                    (12)
                                     , 2246-2264. <a href="https://doi.org/10.1039/C6MD00373G" title="DOI URL">https://doi.org/10.1039/C6MD00373G</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C6MD00373G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC6MD00373G%26sid%3Dliteratum%253Aachs%26jtitle%3DMedChemComm%26atitle%3DChemical%252Bprobes%252Band%252Binhibitors%252Bof%252Bbromodomains%252Boutside%252Bthe%252BBET%252Bfamily%26aulast%3DMoustakim%26aufirst%3DMoses%26date%3D2016%26date%3D2016%26volume%3D7%26issue%3D12%26spage%3D2246%26epage%3D2264" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">S.  Terracciano</span>, <span class="hlFld-ContribAuthor ">A.  Foglia</span>, <span class="hlFld-ContribAuthor ">M. G.  Chini</span>, <span class="hlFld-ContribAuthor ">M. C.  Vaccaro</span>, <span class="hlFld-ContribAuthor ">A.  Russo</span>, <span class="hlFld-ContribAuthor ">F. Dal  Piaz</span>, <span class="hlFld-ContribAuthor ">C.  Saturnino</span>, <span class="hlFld-ContribAuthor ">R.  Riccio</span>, <span class="hlFld-ContribAuthor ">G.  Bifulco</span>, <span class="hlFld-ContribAuthor ">I.  Bruno</span>. </span><span class="cited-content_cbyCitation_article-title">New dihydropyrimidin-2(1H)-one based Hsp90 C-terminal inhibitors. </span><span class="cited-content_cbyCitation_journal-name">RSC Advances</span><span> <strong>2016,</strong> <em>6 </em>
                                    (85)
                                     , 82330-82340. <a href="https://doi.org/10.1039/C6RA17235K" title="DOI URL">https://doi.org/10.1039/C6RA17235K</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C6RA17235K&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC6RA17235K%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Advances%26atitle%3DNew%252Bdihydropyrimidin-2%2525281H%252529-one%252Bbased%252BHsp90%252BC-terminal%252Binhibitors%26aulast%3DTerracciano%26aufirst%3DS.%26date%3D2016%26date%3D2016%26volume%3D6%26issue%3D85%26spage%3D82330%26epage%3D82340" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tomomi  Noguchi-Yachide</span>. </span><span class="cited-content_cbyCitation_article-title">BET Bromodomain as a Target of Epigenetic Therapy. </span><span class="cited-content_cbyCitation_journal-name">CHEMICAL & PHARMACEUTICAL BULLETIN</span><span> <strong>2016,</strong> <em>64 </em>
                                    (6)
                                     , 540-547. <a href="https://doi.org/10.1248/cpb.c16-00225" title="DOI URL">https://doi.org/10.1248/cpb.c16-00225</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1248/cpb.c16-00225&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1248%2Fcpb.c16-00225%26sid%3Dliteratum%253Aachs%26jtitle%3DCHEMICAL%2520%2526%2520PHARMACEUTICAL%2520BULLETIN%26atitle%3DBET%252BBromodomain%252Bas%252Ba%252BTarget%252Bof%252BEpigenetic%252BTherapy%26aulast%3DNoguchi-Yachide%26aufirst%3DTomomi%26date%3D2016%26volume%3D64%26issue%3D6%26spage%3D540%26epage%3D547" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Seika  Amemiya</span>, <span class="hlFld-ContribAuthor ">Takao  Yamaguchi</span>, <span class="hlFld-ContribAuthor ">Taki  Sakai</span>, <span class="hlFld-ContribAuthor ">Yuichi  Hashimoto</span>, <span class="hlFld-ContribAuthor ">Tomomi  Noguchi-Yachide</span>. </span><span class="cited-content_cbyCitation_article-title">Structure–Activity Relationship Study of N6-Benzoyladenine-Type BRD4 Inhibitors and Their Effects on Cell Differentiation and TNF-α Production. </span><span class="cited-content_cbyCitation_journal-name">CHEMICAL & PHARMACEUTICAL BULLETIN</span><span> <strong>2016,</strong> <em>64 </em>
                                    (9)
                                     , 1378-1383. <a href="https://doi.org/10.1248/cpb.c16-00410" title="DOI URL">https://doi.org/10.1248/cpb.c16-00410</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1248/cpb.c16-00410&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1248%2Fcpb.c16-00410%26sid%3Dliteratum%253Aachs%26jtitle%3DCHEMICAL%2520%2526%2520PHARMACEUTICAL%2520BULLETIN%26atitle%3DStructure%2525E2%252580%252593Activity%252BRelationship%252BStudy%252Bof%252BN6-Benzoyladenine-Type%252BBRD4%252BInhibitors%252Band%252BTheir%252BEffects%252Bon%252BCell%252BDifferentiation%252Band%252BTNF-%2525CE%2525B1%252BProduction%26aulast%3DAmemiya%26aufirst%3DSeika%26date%3D2016%26volume%3D64%26issue%3D9%26spage%3D1378%26epage%3D1383" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Duncan A.  Hay</span>, <span class="hlFld-ContribAuthor ">Catherine M.  Rogers</span>, <span class="hlFld-ContribAuthor ">Oleg  Fedorov</span>, <span class="hlFld-ContribAuthor ">Cynthia  Tallant</span>, <span class="hlFld-ContribAuthor ">Sarah  Martin</span>, <span class="hlFld-ContribAuthor ">Octovia P.  Monteiro</span>, <span class="hlFld-ContribAuthor ">Susanne  Müller</span>, <span class="hlFld-ContribAuthor ">Stefan  Knapp</span>, <span class="hlFld-ContribAuthor ">Christopher J.  Schofield</span>, <span class="hlFld-ContribAuthor ">Paul E.  Brennan</span>. </span><span class="cited-content_cbyCitation_article-title">Design and synthesis of potent and selective inhibitors of BRD7 and BRD9 bromodomains. </span><span class="cited-content_cbyCitation_journal-name">MedChemComm</span><span> <strong>2015,</strong> <em>6 </em>
                                    (7)
                                     , 1381-1386. <a href="https://doi.org/10.1039/C5MD00152H" title="DOI URL">https://doi.org/10.1039/C5MD00152H</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C5MD00152H&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC5MD00152H%26sid%3Dliteratum%253Aachs%26jtitle%3DMedChemComm%26atitle%3DDesign%252Band%252Bsynthesis%252Bof%252Bpotent%252Band%252Bselective%252Binhibitors%252Bof%252BBRD7%252Band%252BBRD9%252Bbromodomains%26aulast%3DHay%26aufirst%3DDuncan%2BA.%26date%3D2015%26date%3D2015%26volume%3D6%26issue%3D7%26spage%3D1381%26epage%3D1386" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Paul  Bamborough</span>, <span class="hlFld-ContribAuthor ">Chun-wa  Chung</span>. </span><span class="cited-content_cbyCitation_article-title">Fragments in bromodomain drug discovery. </span><span class="cited-content_cbyCitation_journal-name">MedChemComm</span><span> <strong>2015,</strong> <em>6 </em>
                                    (9)
                                     , 1587-1604. <a href="https://doi.org/10.1039/C5MD00209E" title="DOI URL">https://doi.org/10.1039/C5MD00209E</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C5MD00209E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC5MD00209E%26sid%3Dliteratum%253Aachs%26jtitle%3DMedChemComm%26atitle%3DFragments%252Bin%252Bbromodomain%252Bdrug%252Bdiscovery%26aulast%3DBamborough%26aufirst%3DPaul%26date%3D2015%26date%3D2015%26volume%3D6%26issue%3D9%26spage%3D1587%26epage%3D1604" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-6/jm501893k/20150418/images/medium/jm-2014-01893k_0007.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-6/jm501893k/20150418/images/large/jm-2014-01893k_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm501893k&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="cht1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Chart 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-6/jm501893k/20150418/images/medium/jm-2014-01893k_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-6/jm501893k/20150418/images/large/jm-2014-01893k_0009.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Chart 1. Acetyllysine Mimetic Templates Reported To Bind to Bromodomain Proteins<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-6/jm501893k/20150418/images/large/jm-2014-01893k_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm501893k&amp;id=cht1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Chart a</span>The Kac mimetic portion of each substructure is highlighted in red. (a) 3,4-Dimethylisoxazole; (b) 3-methyl-3,4-dihydroquinazolinone; (c) indolizinethanone; (d) <i>N</i>-phenylacetamide; (e) <i>N</i>-acetyl-2-methyltetrahydroquinoline; (f) triazolopyrimidine; (g) methylquinoline; (h) chloropyridone; (i) thiazolidinone; (j) 4-acylpyrrole; (k) triazolophtalazine; (l) methyltriazoles.</p></p></figure><figure data-id="fig1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-6/jm501893k/20150418/images/medium/jm-2014-01893k_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-6/jm501893k/20150418/images/large/jm-2014-01893k_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Purine fragments bind to human bromodomains. (A) Structure of olomoucine, a potent cyclin-dependent kinase inhibitor and numbering of the 2-amine-9<i>H</i>-purine core scaffold. (B) Purine fragments tested as bromodomain ligands. (C) Docking pose of <b>1</b> (yellow stick representation) onto the bromodomain of BRD4(1) positions the bulky halogen group on the top of the binding pocket. The protein is shown as a white ribbon (starting model, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4MEN">4MEN</a>) or in magenta (docked model) with characteristic residues shown as sticks in the same color scheme. (D) Alternative binding of compound <b>1</b> into the bromodomain of BRD4(1) with the 6-chloro substituent adopting an acetyllysine mimetic pose. The compound and protein are colored as in (C). PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4MEN">4MEN</a> was used as the starting model for docking. (E) Docking of compound <b>2a</b> positions the 2-amine-9<i>H</i>-purine ring system of the ligand within the Kac cavity, sterically packing between V87 and I146, suggesting that this scaffold topology can be further utilized to target bromodomains. The same color scheme as in (C) and (D) is used with the ligand shown as orange sticks. PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4MEN">4MEN</a> was used as the starting model for docking. (F) Fragments were tested in a thermal shift assay against bromodomains of the BET subfamily as well as representative members from other families. Thermal shifts are color-coded as indicated in the inset. 9<i>H</i>-Purines showed weak binding across the panel.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-6/jm501893k/20150418/images/large/jm-2014-01893k_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm501893k&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="cht2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Chart 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-6/jm501893k/20150418/images/medium/jm-2014-01893k_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-6/jm501893k/20150418/images/large/jm-2014-01893k_0010.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Chart 2. Chemical Structures of Compounds <b>1</b>–<b>11</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-6/jm501893k/20150418/images/large/jm-2014-01893k_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm501893k&amp;id=cht2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-6/jm501893k/20150418/images/medium/jm-2014-01893k_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-6/jm501893k/20150418/images/large/jm-2014-01893k_0006.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. General Procedures for the Synthesis of Purine Derivatives: (A) Suzuki–Miyaura Cross-Coupling of Free Halopurines and (B) TBAF-Assisted N-9 Alkylation on the Purine Ring<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-6/jm501893k/20150418/images/large/jm-2014-01893k_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm501893k&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) Pd(OAc)<sub>2</sub>/P(C<sub>6</sub>H<sub>4</sub>SO<sub>3</sub>Na)<sub>3</sub>, Cs<sub>2</sub>CO<sub>3</sub>, MeCN/H<sub>2</sub>O (1:2), microwaves, 150 °C, 5–15 min; (b) CH<sub>3</sub>I or CH<sub>3</sub>COCH<sub>2</sub>Cl, TBAF, THF, rt, 10 min; (c) 50% H<sub>2</sub>SO<sub>4</sub>, NaNO<sub>2</sub>, −10 °C, 2 h, then 50 °C, 1 h.</p></p></figure><figure data-id="fig2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-6/jm501893k/20150418/images/medium/jm-2014-01893k_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-6/jm501893k/20150418/images/large/jm-2014-01893k_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. SAR of 9<i>H</i>-purines. The 9<i>H</i>-purine core scaffold was iteratively decorated and tested for binding to human bromodomains. (A) Substitution patterns explored. (B) Thermal shift assay against human bromodomains. Thermal shifts are color-coded as indicated in the inset. Compounds highlighted with a colored star were further validated by isothermal titration calorimetry as shown in (C)/(D). (C) Isothermal titration calorimetry validation of key compounds binding to BRD4(1) showing raw injection heats for titrations of protein into compound. The inset shows the normalized binding enthalpies corrected for the heat of protein dilution as a function of binding site saturation (symbols as indicated in the figure). Solid lines represent a nonlinear least-squares fit using a single-site binding model. (D) Compounds bearing ortho,meta′ substitutions gain potency toward BRD9 as demonstrated by ITC experiments. Data have been corrected and displayed as described in (C). All ITC titrations were carried out in 50 mM HEPES, pH 7.5 (at 25 °C), 150 mM NaCl, and 15 °C while stirring at 1000 rpm.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-6/jm501893k/20150418/images/large/jm-2014-01893k_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm501893k&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-6/jm501893k/20150418/images/medium/jm-2014-01893k_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-6/jm501893k/20150418/images/large/jm-2014-01893k_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Induced fit binding of 9<i>H</i>-purines to BRD9. (A) Overall fold of BRD9/<b>7d</b> complex. (B) 2<i>F</i><sub>c</sub><i>F</i><sub>o</sub> map of <b>7d</b> in complex with BRD9 contoured at 2σ. (C) <b>7d</b> occupies the acetyllysine binding cavity of the bromodomain module initiating direct interactions with the conserved asparagine (N100) and packing onto the ZA-loop hydrophobic backbone (F44, F47, I53, A54, P56, Y57). The conserved water network is preserved in the structure, and the ligand initiates hydrogen bonds that further stabilize binding. Notably the ligand engages a water molecule located on the top of the BRD cavity in a network of hydrogen bonds to the backbone of I53. (D) Binding of <b>7d</b> to BRD9 results in a distinct rearrangement of the BRD fold. Residues on both sides of the ligand shift toward it, forming an induced-fit pocket with F47 rotating 120° capping the channel found in the apo BRD9 structure at the front of the ZA-loop. (E) Surface view of the side chain rearrangement described in (C) highlighting the induced pocket upon binding of <b>7d</b> to BRD9. The model and structure factors of the BRD9/<b>7d</b> complex shown have been deposited to the PDB with ascension code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4XY8">4XY8</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-6/jm501893k/20150418/images/large/jm-2014-01893k_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm501893k&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-6/jm501893k/20150418/images/medium/jm-2014-01893k_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-6/jm501893k/20150418/images/large/jm-2014-01893k_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. BRD pocket SAR. (A) Compounds designed to probe the acetyllysine mimetic character of the purine scaffold, disrupting interactions (N9-methyl analogues) or reaching deeper into the acetyllysine cavity (8-methyl analogues). (B) Thermal shift assay against human bromodomains. Compound <b>10</b> was heavily colored and interfered with the assay. Thermal shifts are color-coded as indicated in the inset. Compounds highlighted with a colored star were further validated as shown in (C)/(D). (C) Substitution of the primary amine group to a hydroxyl (compound <b>8a</b>) impairs binding toward BRD9 as demonstrated by ITC experiments, while cyclization of the aromatic substituent results in enhanced potency (compound <b>11</b>). The inset shows the normalized binding enthalpies corrected for the heat of protein dilution as a function of binding site saturation (symbols as indicated in the figure). Solid lines represent a nonlinear least-squares fit using a single-site binding model. (D) Isothermal titration calorimetry validation of compound <b>11</b> binding to BRD4(1) showing raw injection heats for titrations of protein into compound. Data have been corrected and displayed as described in (C). All ITC titrations were carried out in 50 mM HEPES, pH 7.5 (at 25 °C), 150 mM NaCl, and 15 °C while stirring at 1000 rpm.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-6/jm501893k/20150418/images/large/jm-2014-01893k_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm501893k&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-6/jm501893k/20150418/images/medium/jm-2014-01893k_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-6/jm501893k/20150418/images/large/jm-2014-01893k_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Complex of compound <b>11</b> and 9<i>H</i>-purine activity in cells. (A) Overview of the complex of compound <b>11</b> with the first bromodomain of BRD4. The ligand retained the acetyllysine mimetic pose that was observed in the case of <b>7d</b>. (B) Compound <b>11</b><i>F</i><sub>c</sub><i>F</i><sub>o</sub> omit map from the BRD4(1)/<b>11</b> complex contoured at 2σ. (C) Detail of compound <b>11</b> biding to BRD4(1) demonstrating the acetyllysine mimetic binding mode, initiating interactions with the conserved asparagine (N140), and packing between the ZA-loop L92 and the ZA-channel W81 while retaining the network of conserved water interactions. The model and structure factors of the BRD4(1)/<b>11</b> complex shown in panels A, B, and C have been deposited to the PDB with ascension code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4XYA">4XYA</a>. (D) Titration of compounds <b>7d</b> and <b>11</b> into HEK293 cells transfected with NanoLuc-fused bromodomain of BRD9 and Halo-tagged histone H3.3. The ligands disrupt the histone/BRD9 interaction with an apparent IC<sub>50</sub> of 3.5 μM (<b>7d</b>) and 480 nM (<b>11</b>), resulting in loss of signal due to separation of the BRD9-histone complex. (E) Titration of compounds <b>7d</b> and <b>11</b> into HEK293 cells transfected with NanoLuc-fused full length BRD4 (UniProt code <a href="https://O60885" class="ext-link">O60885</a>) and Halo-tagged histone H3.3. Although the ligands gradually disrupt the BRD4-FL/H3.3 interaction, they fail to elicit the same effect as in the case of BRD9 in the concentration range tested.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-6/jm501893k/20150418/images/large/jm-2014-01893k_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm501893k&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i68">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_10418" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_10418" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 80 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Bernstein, B. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meissner, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lander, E. S.</span><span> </span><span class="NLM_article-title">The mammalian epigenome</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">128</span><span class="NLM_x">, </span> <span class="NLM_fpage">669</span><span class="NLM_x">–</span> <span class="NLM_lpage">681</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Fjm501893k&amp;key=10.1016%2Fj.cell.2007.01.033" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Fjm501893k&amp;key=17320505" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BD2sXis12ju7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2007&pages=669-681&author=B.+E.+Bernsteinauthor=A.+Meissnerauthor=E.+S.+Lander&title=The+mammalian+epigenome"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The mammalian epigenome</span></div><div class="casAuthors">Bernstein, Bradley E.; Meissner, Alexander; Lander, Eric S.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">669-681</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Chem. modifications to DNA and histone proteins form a complex regulatory network that modulates chromatin structure and genome function.  The epigenome refers to the complete description of these potentially heritable changes across the genome.  The compn. of the epigenome within a given cell is a function of genetic determinants, lineage, and environment.  With the sequencing of the human genome completed, investigators now seek a comprehensive view of the epigenetic changes that det. how genetic information is made manifest across an incredibly varied background of developmental stages, tissue types, and disease states.  Here we review current research efforts, with an emphasis on large-scale studies, emerging technologies, and challenges ahead.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKJfRRDsxAHbVg90H21EOLACvtfcHk0ljb6NlTiK0w7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXis12ju7g%253D&md5=963fb487bbf473ac801e09a558ac0b0e</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2007.01.033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2007.01.033%26sid%3Dliteratum%253Aachs%26aulast%3DBernstein%26aufirst%3DB.%2BE.%26aulast%3DMeissner%26aufirst%3DA.%26aulast%3DLander%26aufirst%3DE.%2BS.%26atitle%3DThe%2520mammalian%2520epigenome%26jtitle%3DCell%26date%3D2007%26volume%3D128%26spage%3D669%26epage%3D681" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Strahl, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allis, C. D.</span><span> </span><span class="NLM_article-title">The language of covalent histone modifications</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">403</span><span class="NLM_x">, </span> <span class="NLM_fpage">41</span><span class="NLM_x">–</span> <span class="NLM_lpage">45</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Fjm501893k&amp;key=10.1038%2F47412" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Fjm501893k&amp;key=10638745" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A280%3ADC%252BD3c7gt1arsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=403&publication_year=2000&pages=41-45&author=B.+D.+Strahlauthor=C.+D.+Allis&title=The+language+of+covalent+histone+modifications"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">The language of covalent histone modifications</span></div><div class="casAuthors">Strahl B D; Allis C D</div><div class="citationInfo"><span class="NLM_cas:title">Nature</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">403</span>
        (<span class="NLM_cas:issue">6765</span>),
    <span class="NLM_cas:pages">41-5</span>
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">Histone proteins and the nucleosomes they form with DNA are the fundamental building blocks of eukaryotic chromatin.  A diverse array of post-translational modifications that often occur on tail domains of these proteins has been well documented.  Although the function of these highly conserved modifications has remained elusive, converging biochemical and genetic evidence suggests functions in several chromatin-based processes.  We propose that distinct histone modifications, on one or more tails, act sequentially or in combination to form a 'histone code' that is, read by other proteins to bring about distinct downstream events.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSyzhdlYh9sX7t7lRqEFUJVfW6udTcc2eaZE7RHfZoqDbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD3c7gt1arsQ%253D%253D&md5=2eb7d093151c6122ccd70fa2adcfd94b</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2F47412&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F47412%26sid%3Dliteratum%253Aachs%26aulast%3DStrahl%26aufirst%3DB.%2BD.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26atitle%3DThe%2520language%2520of%2520covalent%2520histone%2520modifications%26jtitle%3DNature%26date%3D2000%26volume%3D403%26spage%3D41%26epage%3D45" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Choudhary, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kumar, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gnad, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nielsen, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rehman, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walther, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olsen, J. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mann, M.</span><span> </span><span class="NLM_article-title">Lysine acetylation targets protein complexes and co-regulates major cellular functions</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">325</span><span class="NLM_x">, </span> <span class="NLM_fpage">834</span><span class="NLM_x">–</span> <span class="NLM_lpage">840</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Fjm501893k&amp;key=10.1126%2Fscience.1175371" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Fjm501893k&amp;key=19608861" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BD1MXps1Ogt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=325&publication_year=2009&pages=834-840&author=C.+Choudharyauthor=C.+Kumarauthor=F.+Gnadauthor=M.+L.+Nielsenauthor=M.+Rehmanauthor=T.+C.+Waltherauthor=J.+V.+Olsenauthor=M.+Mann&title=Lysine+acetylation+targets+protein+complexes+and+co-regulates+major+cellular+functions"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Lysine Acetylation Targets Protein Complexes and Co-Regulates Major Cellular Functions</span></div><div class="casAuthors">Choudhary, Chunaram; Kumar, Chanchal; Gnad, Florian; Nielsen, Michael L.; Rehman, Michael; Walther, Tobias C.; Olsen, Jesper V.; Mann, Matthias</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">325</span>
        (<span class="NLM_cas:issue">5942</span>),
    <span class="NLM_cas:pages">834-840</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Lysine acetylation is a reversible posttranslational modification of proteins and plays a key role in regulating gene expression.  Technol. limitations have so far prevented a global anal. of lysine acetylation's cellular roles.  We used high-resoln. mass spectrometry to identify 3600 lysine acetylation sites on 1750 proteins and quantified acetylation changes in response to the deacetylase inhibitors suberoylanilide hydroxamic acid and MS-275.  Lysine acetylation preferentially targets large macromol. complexes involved in diverse cellular processes, such as chromatin remodeling, cell cycle, splicing, nuclear transport, and actin nucleation.  Acetylation impaired phosphorylation-dependent interactions of 14-3-3 and regulated the yeast cyclin-dependent kinase Cdc28.  Our data demonstrate that the regulatory scope of lysine acetylation is broad and comparable with that of other major posttranslational modifications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5eICW58ngkrVg90H21EOLACvtfcHk0ljb6NlTiK0w7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXps1Ogt70%253D&md5=8f643db5ba91c50f0b61763ff3095d2c</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1126%2Fscience.1175371&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1175371%26sid%3Dliteratum%253Aachs%26aulast%3DChoudhary%26aufirst%3DC.%26aulast%3DKumar%26aufirst%3DC.%26aulast%3DGnad%26aufirst%3DF.%26aulast%3DNielsen%26aufirst%3DM.%2BL.%26aulast%3DRehman%26aufirst%3DM.%26aulast%3DWalther%26aufirst%3DT.%2BC.%26aulast%3DOlsen%26aufirst%3DJ.%2BV.%26aulast%3DMann%26aufirst%3DM.%26atitle%3DLysine%2520acetylation%2520targets%2520protein%2520complexes%2520and%2520co-regulates%2520major%2520cellular%2520functions%26jtitle%3DScience%26date%3D2009%26volume%3D325%26spage%3D834%26epage%3D840" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span> </span><span class="NLM_article-title">The bromodomain interaction module</span> <span class="citation_source-journal">FEBS Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">586</span><span class="NLM_x">, </span> <span class="NLM_fpage">2692</span><span class="NLM_x">–</span> <span class="NLM_lpage">2704</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Fjm501893k&amp;key=10.1016%2Fj.febslet.2012.04.045" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Fjm501893k&amp;key=22710155" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BC38XntFahsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=586&publication_year=2012&pages=2692-2704&author=P.+Filippakopoulosauthor=S.+Knapp&title=The+bromodomain+interaction+module"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">The bromodomain interaction module</span></div><div class="casAuthors">Filippakopoulos, Panagis; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Letters</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">586</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">2692-2704</span>CODEN:
                <span class="NLM_cas:coden">FEBLAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-5793</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The ε-N-acetylation of protein Lys residues (Kac) is one of the most abundant post-translation modifications (PTMs) in the human proteome.  In the nucleus, the acetylation of histones has been linked to transcriptional activation of genes but the functional consequences of most acetylation events and proteins recruited to these sites remains largely unknown.  Bromodomains (BRDs) are small helical interaction modules that specifically recognize acetylation sites in proteins.  BRDs have recently emerged as interesting targets for the development of specific protein interaction inhibitors, enabling a novel exiting strategy for the development of new therapies.  Here, the author provides an overview over sequence requirements of BRDs, known substrates, and the structural mechanisms of specific Kac recognition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7LaThygz_HrVg90H21EOLACvtfcHk0li-jL39rQG_QQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XntFahsbg%253D&md5=c02dee22c330b1d5c1ba39579360ebc5</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.febslet.2012.04.045&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.febslet.2012.04.045%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DThe%2520bromodomain%2520interaction%2520module%26jtitle%3DFEBS%2520Lett.%26date%3D2012%26volume%3D586%26spage%3D2692%26epage%3D2704" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Tamkun, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deuring, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kissinger, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pattatucci, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaufman, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kennison, J. A.</span><span> </span><span class="NLM_article-title">brahma: a regulator of <i>Drosophila</i> homeotic genes structurally related to the yeast transcriptional activator SNF2/SWI2</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1992</span><span class="NLM_x">, </span> <span class="NLM_volume">68</span><span class="NLM_x">, </span> <span class="NLM_fpage">561</span><span class="NLM_x">–</span> <span class="NLM_lpage">572</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=1992&pages=561-572&author=J.+W.+Tamkunauthor=R.+Deuringauthor=M.+P.+Scottauthor=M.+Kissingerauthor=A.+M.+Pattatucciauthor=T.+C.+Kaufmanauthor=J.+A.+Kennison&title=brahma%3A+a+regulator+of+Drosophila+homeotic+genes+structurally+related+to+the+yeast+transcriptional+activator+SNF2%2FSWI2"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DTamkun%26aufirst%3DJ.%2BW.%26aulast%3DDeuring%26aufirst%3DR.%26aulast%3DScott%26aufirst%3DM.%2BP.%26aulast%3DKissinger%26aufirst%3DM.%26aulast%3DPattatucci%26aufirst%3DA.%2BM.%26aulast%3DKaufman%26aufirst%3DT.%2BC.%26aulast%3DKennison%26aufirst%3DJ.%2BA.%26atitle%3Dbrahma%253A%2520a%2520regulator%2520of%2520Drosophila%2520homeotic%2520genes%2520structurally%2520related%2520to%2520the%2520yeast%2520transcriptional%2520activator%2520SNF2%252FSWI2%26jtitle%3DCell%26date%3D1992%26volume%3D68%26spage%3D561%26epage%3D572" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tanaka, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peng, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ren, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dent, S. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, X.</span><span> </span><span class="NLM_article-title">AF9 YEATS domain links histone acetylation to DOT1L-mediated H3K79 methylation</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">159</span><span class="NLM_x">, </span> <span class="NLM_fpage">558</span><span class="NLM_x">–</span> <span class="NLM_lpage">571</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=159&publication_year=2014&pages=558-571&author=Y.+Liauthor=H.+Wenauthor=Y.+Xiauthor=K.+Tanakaauthor=H.+Wangauthor=D.+Pengauthor=Y.+Renauthor=Q.+Jinauthor=S.+Y.+Dentauthor=W.+Liauthor=H.+Liauthor=X.+Shi&title=AF9+YEATS+domain+links+histone+acetylation+to+DOT1L-mediated+H3K79+methylation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DY.%26aulast%3DWen%26aufirst%3DH.%26aulast%3DXi%26aufirst%3DY.%26aulast%3DTanaka%26aufirst%3DK.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DPeng%26aufirst%3DD.%26aulast%3DRen%26aufirst%3DY.%26aulast%3DJin%26aufirst%3DQ.%26aulast%3DDent%26aufirst%3DS.%2BY.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DShi%26aufirst%3DX.%26atitle%3DAF9%2520YEATS%2520domain%2520links%2520histone%2520acetylation%2520to%2520DOT1L-mediated%2520H3K79%2520methylation%26jtitle%3DCell%26date%3D2014%26volume%3D159%26spage%3D558%26epage%3D571" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Yang, X. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ogryzko, V. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nishikawa, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Howard, B. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakatani, Y.</span><span> </span><span class="NLM_article-title">A p300/CBP-associated factor that competes with the adenoviral oncoprotein E1A</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">382</span><span class="NLM_x">, </span> <span class="NLM_fpage">319</span><span class="NLM_x">–</span> <span class="NLM_lpage">324</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=382&publication_year=1996&pages=319-324&author=X.+J.+Yangauthor=V.+V.+Ogryzkoauthor=J.+Nishikawaauthor=B.+H.+Howardauthor=Y.+Nakatani&title=A+p300%2FCBP-associated+factor+that+competes+with+the+adenoviral+oncoprotein+E1A"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DX.%2BJ.%26aulast%3DOgryzko%26aufirst%3DV.%2BV.%26aulast%3DNishikawa%26aufirst%3DJ.%26aulast%3DHoward%26aufirst%3DB.%2BH.%26aulast%3DNakatani%26aufirst%3DY.%26atitle%3DA%2520p300%252FCBP-associated%2520factor%2520that%2520competes%2520with%2520the%2520adenoviral%2520oncoprotein%2520E1A%26jtitle%3DNature%26date%3D1996%26volume%3D382%26spage%3D319%26epage%3D324" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Dhalluin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carlson, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aggarwal, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, M. M.</span><span> </span><span class="NLM_article-title">Structure and ligand of a histone acetyltransferase bromodomain</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">399</span><span class="NLM_x">, </span> <span class="NLM_fpage">491</span><span class="NLM_x">–</span> <span class="NLM_lpage">496</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Fjm501893k&amp;key=10.1038%2F20974" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Fjm501893k&amp;key=10365964" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADyaK1MXjvV2isLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=399&publication_year=1999&pages=491-496&author=C.+Dhalluinauthor=J.+E.+Carlsonauthor=L.+Zengauthor=C.+Heauthor=A.+K.+Aggarwalauthor=M.+M.+Zhou&title=Structure+and+ligand+of+a+histone+acetyltransferase+bromodomain"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and ligand of a histone acetyltransferase bromodomain</span></div><div class="casAuthors">Dhalluin, Christophe; Carlson, Justin E.; Zeng, Lei; He, Cheng; Aggarwal, Aneel K.; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">399</span>
        (<span class="NLM_cas:issue">6735</span>),
    <span class="NLM_cas:pages">491-496</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines</span>)
        </div><div class="casAbstract">Histone acetylation is important in chromatin remodelling and gene activation.  Nearly all known histone-acetyltransferase (HAT)-assocd. transcriptional co-activators contain bromodomains, which are ∼110-amino-acid modules found in many chromatin-assocd. proteins.  Despite the wide occurrence of these bromodomains, their three-dimensional structure and binding partners remain unknown.  Here we report the soln. structure of the bromodomain of the HAT co-activator P/CAF (p300/CBP-assocd. factor).  The structure reveals an unusual left-handed up-and-down four-helix bundle.  In addn., we show by a combination of structural and site-directed mutagenesis studies that bromodomains can interact specifically with acetylated lysine, making them the first known protein modules to do so.  The nature of the recognition of acetyl-lysine by the P/CAF bromodomain is similar to that of acetyl-CoA by histone acetyltransferase.  Thus, the bromodomain is functionally linked to the HAT activity of co-activators in the regulation of gene transcription.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsTbwcjeBG_rVg90H21EOLACvtfcHk0li-jL39rQG_QQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXjvV2isLo%253D&md5=ff1063ba81d81b431065dd6e3751ba79</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2F20974&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F20974%26sid%3Dliteratum%253Aachs%26aulast%3DDhalluin%26aufirst%3DC.%26aulast%3DCarlson%26aufirst%3DJ.%2BE.%26aulast%3DZeng%26aufirst%3DL.%26aulast%3DHe%26aufirst%3DC.%26aulast%3DAggarwal%26aufirst%3DA.%2BK.%26aulast%3DZhou%26aufirst%3DM.%2BM.%26atitle%3DStructure%2520and%2520ligand%2520of%2520a%2520histone%2520acetyltransferase%2520bromodomain%26jtitle%3DNature%26date%3D1999%26volume%3D399%26spage%3D491%26epage%3D496" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Jacobson, R. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ladurner, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">King, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tjian, R.</span><span> </span><span class="NLM_article-title">Structure and function of a human TAFII250 double bromodomain module</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">288</span><span class="NLM_x">, </span> <span class="NLM_fpage">1422</span><span class="NLM_x">–</span> <span class="NLM_lpage">1425</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Fjm501893k&amp;key=10.1126%2Fscience.288.5470.1422" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Fjm501893k&amp;key=10827952" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BD3cXjslGrs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=288&publication_year=2000&pages=1422-1425&author=R.+H.+Jacobsonauthor=A.+G.+Ladurnerauthor=D.+S.+Kingauthor=R.+Tjian&title=Structure+and+function+of+a+human+TAFII250+double+bromodomain+module"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and function of a human TAFII250 double bromodomain module</span></div><div class="casAuthors">Jacobson, Raymond H.; Ladurner, Andreas G.; King, David S.; Tjian, Robert</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">288</span>
        (<span class="NLM_cas:issue">5470</span>),
    <span class="NLM_cas:pages">1422-1425</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">TFIID is a large multiprotein complex that initiates assembly of the transcription machinery.  It is unclear how TFIID recognizes promoters in vivo when templates are nucleosome-bound.  Here, it is shown that TAFII250, the largest subunit of TFIID, contains two tandem bromodomain modules that bind selectively to multiply acetylated histone H4 peptides.  The 2.1 angstrom crystal structure of the double bromodomain reveals two side-by-side, four-helix bundles with a highly polarized surface charge distribution.  Each bundle contains an Nε-acetyllysine binding pocket at its center, which results in a structure ideally suited for recognition of diacetylated histone H4 tails.  Thus, TFIID may be targeted to specific chromatin-bound promoters and may play a role in chromatin recognition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnYe1raEZOPLVg90H21EOLACvtfcHk0li-jL39rQG_QQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXjslGrs7g%253D&md5=d0a43e3283c3a9f0eb25252e1980e580</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1126%2Fscience.288.5470.1422&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.288.5470.1422%26sid%3Dliteratum%253Aachs%26aulast%3DJacobson%26aufirst%3DR.%2BH.%26aulast%3DLadurner%26aufirst%3DA.%2BG.%26aulast%3DKing%26aufirst%3DD.%2BS.%26aulast%3DTjian%26aufirst%3DR.%26atitle%3DStructure%2520and%2520function%2520of%2520a%2520human%2520TAFII250%2520double%2520bromodomain%2520module%26jtitle%3DScience%26date%3D2000%26volume%3D288%26spage%3D1422%26epage%3D1425" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Venturini, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">You, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stadler, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Galien, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lallemand, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koken, M. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattei, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ganser, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chambon, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Losson, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de The, H.</span><span> </span><span class="NLM_article-title">TIF1gamma, a novel member of the transcriptional intermediary factor 1 family</span> <span class="citation_source-journal">Oncogene</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">1209</span><span class="NLM_x">–</span> <span class="NLM_lpage">1217</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=1999&pages=1209-1217&author=L.+Venturiniauthor=J.+Youauthor=M.+Stadlerauthor=R.+Galienauthor=V.+Lallemandauthor=M.+H.+Kokenauthor=M.+G.+Matteiauthor=A.+Ganserauthor=P.+Chambonauthor=R.+Lossonauthor=H.+de+The&title=TIF1gamma%2C+a+novel+member+of+the+transcriptional+intermediary+factor+1+family"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DVenturini%26aufirst%3DL.%26aulast%3DYou%26aufirst%3DJ.%26aulast%3DStadler%26aufirst%3DM.%26aulast%3DGalien%26aufirst%3DR.%26aulast%3DLallemand%26aufirst%3DV.%26aulast%3DKoken%26aufirst%3DM.%2BH.%26aulast%3DMattei%26aufirst%3DM.%2BG.%26aulast%3DGanser%26aufirst%3DA.%26aulast%3DChambon%26aufirst%3DP.%26aulast%3DLosson%26aufirst%3DR.%26aulast%3Dde%2BThe%26aufirst%3DH.%26atitle%3DTIF1gamma%252C%2520a%2520novel%2520member%2520of%2520the%2520transcriptional%2520intermediary%2520factor%25201%2520family%26jtitle%3DOncogene%26date%3D1999%26volume%3D18%26spage%3D1209%26epage%3D1217" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Cavellan, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Asp, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Percipalle, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Farrants, A. K.</span><span> </span><span class="NLM_article-title">The WSTF-SNF2h chromatin remodeling complex interacts with several nuclear proteins in transcription</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">281</span><span class="NLM_x">, </span> <span class="NLM_fpage">16264</span><span class="NLM_x">–</span> <span class="NLM_lpage">16271</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=281&publication_year=2006&pages=16264-16271&author=E.+Cavellanauthor=P.+Aspauthor=P.+Percipalleauthor=A.+K.+Farrants&title=The+WSTF-SNF2h+chromatin+remodeling+complex+interacts+with+several+nuclear+proteins+in+transcription"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCavellan%26aufirst%3DE.%26aulast%3DAsp%26aufirst%3DP.%26aulast%3DPercipalle%26aufirst%3DP.%26aulast%3DFarrants%26aufirst%3DA.%2BK.%26atitle%3DThe%2520WSTF-SNF2h%2520chromatin%2520remodeling%2520complex%2520interacts%2520with%2520several%2520nuclear%2520proteins%2520in%2520transcription%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2006%26volume%3D281%26spage%3D16264%26epage%3D16271" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Trotter, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Archer, T. K.</span><span> </span><span class="NLM_article-title">The BRG1 transcriptional coregulator</span> <span class="citation_source-journal">Nucl. Recept Signaling</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">e004</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2008&pages=e004&author=K.+W.+Trotterauthor=T.+K.+Archer&title=The+BRG1+transcriptional+coregulator"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DTrotter%26aufirst%3DK.%2BW.%26aulast%3DArcher%26aufirst%3DT.%2BK.%26atitle%3DThe%2520BRG1%2520transcriptional%2520coregulator%26jtitle%3DNucl.%2520Recept%2520Signaling%26date%3D2008%26volume%3D6%26spage%3De004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Xue, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Canman, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nie, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moreno, G. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, M. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Salmon, E. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, W.</span><span> </span><span class="NLM_article-title">The human SWI/SNF-B chromatin-remodeling complex is related to yeast rsc and localizes at kinetochores of mitotic chromosomes</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">97</span><span class="NLM_x">, </span> <span class="NLM_fpage">13015</span><span class="NLM_x">–</span> <span class="NLM_lpage">13020</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=97&publication_year=2000&pages=13015-13020&author=Y.+Xueauthor=J.+C.+Canmanauthor=C.+S.+Leeauthor=Z.+Nieauthor=D.+Yangauthor=G.+T.+Morenoauthor=M.+K.+Youngauthor=E.+D.+Salmonauthor=W.+Wang&title=The+human+SWI%2FSNF-B+chromatin-remodeling+complex+is+related+to+yeast+rsc+and+localizes+at+kinetochores+of+mitotic+chromosomes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DXue%26aufirst%3DY.%26aulast%3DCanman%26aufirst%3DJ.%2BC.%26aulast%3DLee%26aufirst%3DC.%2BS.%26aulast%3DNie%26aufirst%3DZ.%26aulast%3DYang%26aufirst%3DD.%26aulast%3DMoreno%26aufirst%3DG.%2BT.%26aulast%3DYoung%26aufirst%3DM.%2BK.%26aulast%3DSalmon%26aufirst%3DE.%2BD.%26aulast%3DWang%26aufirst%3DW.%26atitle%3DThe%2520human%2520SWI%252FSNF-B%2520chromatin-remodeling%2520complex%2520is%2520related%2520to%2520yeast%2520rsc%2520and%2520localizes%2520at%2520kinetochores%2520of%2520mitotic%2520chromosomes%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2000%26volume%3D97%26spage%3D13015%26epage%3D13020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Wu, S. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chiang, C. M.</span><span> </span><span class="NLM_article-title">The double bromodomain-containing chromatin adaptor Brd4 and transcriptional regulation</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">282</span><span class="NLM_x">, </span> <span class="NLM_fpage">13141</span><span class="NLM_x">–</span> <span class="NLM_lpage">13145</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Fjm501893k&amp;key=10.1074%2Fjbc.R700001200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Fjm501893k&amp;key=17329240" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BD2sXks1Ghtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=282&publication_year=2007&pages=13141-13145&author=S.+Y.+Wuauthor=C.+M.+Chiang&title=The+double+bromodomain-containing+chromatin+adaptor+Brd4+and+transcriptional+regulation"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">The Double Bromodomain-containing Chromatin Adaptor Brd4 and Transcriptional Regulation</span></div><div class="casAuthors">Wu, Shwu-Yuan; Chiang, Cheng-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">282</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">13141-13145</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">A review.  Brd4 is a double bromodomain-contg. protein that binds preferentially to acetylated chromatin.  It belongs to the BET (bromodomains and extraterminal) family that includes mammalian Brd2, Brd3, Brd4, Brdt, Drosophila Fsh, yeast Bdf1, Bdf2, and corresponding homologues in other species.  Brd4 is essential for cellular growth and has been implicated in cell cycle control, DNA replication, and gene rearrangement found in t(15;19)-assocd. carcinomas.  Recently, Brd4 has been found in several transcription complexes, including the general cofactor Mediator and the P-TEFb elongation factor, and is capable of stimulating HIV-1 transcription in a Tat-independent manner.  In addn., Brd4 is used as a cellular adaptor by some animal and human papillomaviruses (HPV) for anchoring viral genomes to mitotic chromosomes.  This tethering, mediated by Brd4 interaction with virus-encoded E2 protein, facilitates viral genome segregation during mitosis.  Interestingly, Brd4 is also identified in a transcriptional silencing complex assembled by HPV E2 and turns out to be the long sought cellular corepressor that inhibits the expression of HPV-encoded E6 and E7 oncoproteins that antagonize p53 and pRB tumor suppressor activity, resp.  The dual role of Brd4 in gene activation and repression illustrates how a dynamic chromatin-binding adaptor is able to recruit distinct transcriptional regulators to modulate promoter activity through cell cycle progression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohccDdZC9iKLVg90H21EOLACvtfcHk0lhO7nSTkDV_xQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXks1Ghtbo%253D&md5=b025a5222b510a7009eca4289370b1d4</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1074%2Fjbc.R700001200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.R700001200%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DS.%2BY.%26aulast%3DChiang%26aufirst%3DC.%2BM.%26atitle%3DThe%2520double%2520bromodomain-containing%2520chromatin%2520adaptor%2520Brd4%2520and%2520transcriptional%2520regulation%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2007%26volume%3D282%26spage%3D13141%26epage%3D13145" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Picaud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mangos, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keates, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lambert, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Felletar, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Volkmer, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muller, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pawson, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gingras, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span> </span><span class="NLM_article-title">Histone recognition and large-scale structural analysis of the human bromodomain family</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">149</span><span class="NLM_x">, </span> <span class="NLM_fpage">214</span><span class="NLM_x">–</span> <span class="NLM_lpage">231</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Fjm501893k&amp;key=10.1016%2Fj.cell.2012.02.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Fjm501893k&amp;key=22464331" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=149&publication_year=2012&pages=214-231&author=P.+Filippakopoulosauthor=S.+Picaudauthor=M.+Mangosauthor=T.+Keatesauthor=J.+P.+Lambertauthor=D.+Barsyte-Lovejoyauthor=I.+Felletarauthor=R.+Volkmerauthor=S.+Mullerauthor=T.+Pawsonauthor=A.+C.+Gingrasauthor=C.+H.+Arrowsmithauthor=S.+Knapp&title=Histone+recognition+and+large-scale+structural+analysis+of+the+human+bromodomain+family"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Histone Recognition and Large-Scale Structural Analysis of the Human Bromodomain Family</span></div><div class="casAuthors">Filippakopoulos, Panagis; Picaud, Sarah; Mangos, Maria; Keates, Tracy; Lambert, Jean-Philippe; Barsyte-Lovejoy, Dalia; Felletar, Ildiko; Volkmer, Rudolf; Muller, Susanne; Pawson, Tony; Gingras, Anne-Claude; Arrowsmith, Cheryl H.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">149</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">214-231</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Bromodomains (BRDs) are protein interaction modules that specifically recognize ε-N-lysine acetylation motifs, a key event in the reading process of epigenetic marks.  The 61 BRDs in the human genome cluster into eight families based on structure/sequence similarity.  Here, we present 29 high-resoln. crystal structures, covering all BRD families.  These proteins are: ASH1L, ATAD2, BAZ2B, BPTF, BRD1, BRD3(1), BRD3(2), BRD4(1), BRD4(2), BRD9, BRDT(1), CECR2, EP300, CREBBP, GCN5L2, KIAA1240, PB1(1), PB1(2), PB1(3), PB1(4), PB1(5), PB1(6), PCAF, PHIP(2), TAF1(2), WDR9(2), BRD4(1).  Comprehensive crossfamily structural anal. identifies conserved and family-specific structural features that are necessary for specific acetylation-dependent substrate recognition.  Screening of more than 30 representative BRDs against systematic histone-peptide arrays identifies new BRD substrates and reveals a strong influence of flanking posttranslational modifications, such as acetylation and phosphorylation, suggesting that BRDs recognize combinations of marks rather than singly acetylated sequences.  We further uncovered a structural mechanism for the simultaneous binding and recognition of diverse diacetyl-contg. peptides by BRD4.  These data provide a foundation for structure-based drug design of specific inhibitors for this emerging target family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoK2FSIcPoXZLVg90H21EOLACvtfcHk0lhO7nSTkDV_xQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D&md5=05583a01a1cd4c3b59fc59c00e4c4777</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2012.02.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2012.02.013%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DMangos%26aufirst%3DM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DLambert%26aufirst%3DJ.%2BP.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DVolkmer%26aufirst%3DR.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DPawson%26aufirst%3DT.%26aulast%3DGingras%26aufirst%3DA.%2BC.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DHistone%2520recognition%2520and%2520large-scale%2520structural%2520analysis%2520of%2520the%2520human%2520bromodomain%2520family%26jtitle%3DCell%26date%3D2012%26volume%3D149%26spage%3D214%26epage%3D231" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Belkina, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Denis, G. V.</span><span> </span><span class="NLM_article-title">BET domain co-regulators in obesity, inflammation and cancer</span> <span class="citation_source-journal">Nat. Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">465</span><span class="NLM_x">–</span> <span class="NLM_lpage">477</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Fjm501893k&amp;key=10.1038%2Fnrc3256" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Fjm501893k&amp;key=22722403" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BC38XovFyks7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2012&pages=465-477&author=A.+C.+Belkinaauthor=G.+V.+Denis&title=BET+domain+co-regulators+in+obesity%2C+inflammation+and+cancer"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">BET domain co-regulators in obesity, inflammation and cancer</span></div><div class="casAuthors">Belkina, Anna C.; Denis, Gerald V.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">465-477</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The bromodomain is a highly conserved motif of 110 amino acids that is bundled into four anti-parallel α-helixes and found in proteins that interact with chromatin, such as transcription factors, histone acetylases and nucleosome remodelling complexes.  Bromodomain proteins are chromatin 'readers'; they recruit chromatin-regulating enzymes, including 'writers' and 'erasers' of histone modification, to target promoters and to regulate gene expression.  Conventional wisdom held that complexes involved in chromatin dynamics are not 'druggable' targets.  However, small mols. that inhibit bromodomain and extraterminal (BET) proteins have been described.  We examine these developments and discuss the implications for small mol. epigenetic targeting of chromatin networks in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1x-bD67oukrVg90H21EOLACvtfcHk0lhwfmoFmvZ9mQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XovFyks7s%253D&md5=d06d88ef0d5d4d71115f40a2f901f53d</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fnrc3256&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc3256%26sid%3Dliteratum%253Aachs%26aulast%3DBelkina%26aufirst%3DA.%2BC.%26aulast%3DDenis%26aufirst%3DG.%2BV.%26atitle%3DBET%2520domain%2520co-regulators%2520in%2520obesity%252C%2520inflammation%2520and%2520cancer%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2012%26volume%3D12%26spage%3D465%26epage%3D477" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Lamonica, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kadauke, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campbell, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gamsjaeger, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Billin, A. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hardison, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mackay, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blobel, G. A.</span><span> </span><span class="NLM_article-title">Bromodomain protein Brd3 associates with acetylated GATA1 to promote its chromatin occupancy at erythroid target genes</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">108</span><span class="NLM_x">, </span> <span class="NLM_fpage">E159</span><span class="NLM_x">–</span> <span class="NLM_lpage">168</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Fjm501893k&amp;key=10.1073%2Fpnas.1102140108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Fjm501893k&amp;key=21536911" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A280%3ADC%252BC3MrmtlGgtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2011&pages=E159-168&author=J.+M.+Lamonicaauthor=W.+Dengauthor=S.+Kadaukeauthor=A.+E.+Campbellauthor=R.+Gamsjaegerauthor=H.+Wangauthor=Y.+Chengauthor=A.+N.+Billinauthor=R.+C.+Hardisonauthor=J.+P.+Mackayauthor=G.+A.+Blobel&title=Bromodomain+protein+Brd3+associates+with+acetylated+GATA1+to+promote+its+chromatin+occupancy+at+erythroid+target+genes"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomain protein Brd3 associates with acetylated GATA1 to promote its chromatin occupancy at erythroid target genes</span></div><div class="casAuthors">Lamonica Janine M; Deng Wulan; Kadauke Stephan; Campbell Amy E; Gamsjaeger Roland; Wang Hongxin; Cheng Yong; Billin Andrew N; Hardison Ross C; Mackay Joel P; Blobel Gerd A</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">108</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">E159-68</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Acetylation of histones triggers association with bromodomain-containing proteins that regulate diverse chromatin-related processes.  Although acetylation of transcription factors has been appreciated for some time, the mechanistic consequences are less well understood.  The hematopoietic transcription factor GATA1 is acetylated at conserved lysines that are required for its stable association with chromatin.  We show that the BET family protein Brd3 binds via its first bromodomain (BD1) to GATA1 in an acetylation-dependent manner in vitro and in vivo.  Mutation of a single residue in BD1 that is involved in acetyl-lysine binding abrogated recruitment of Brd3 by GATA1, demonstrating that acetylation of GATA1 is essential for Brd3 association with chromatin.  Notably, Brd3 is recruited by GATA1 to both active and repressed target genes in a fashion seemingly independent of histone acetylation.  Anti-Brd3 ChIP followed by massively parallel sequencing in GATA1-deficient erythroid precursor cells and those that are GATA1 replete revealed that GATA1 is a major determinant of Brd3 recruitment to genomic targets within chromatin.  A pharmacologic compound that occupies the acetyl-lysine binding pockets of Brd3 bromodomains disrupts the Brd3-GATA1 interaction, diminishes the chromatin occupancy of both proteins, and inhibits erythroid maturation.  Together these findings provide a mechanism for GATA1 acetylation and suggest that Brd3 "reads" acetyl marks on nuclear factors to promote their stable association with chromatin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQeZ8VSdvvG7p6KlWarZTAzfW6udTcc2ebTiPrdOKDhO7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3MrmtlGgtQ%253D%253D&md5=b81056e4aee1351dce3899fb8d4e86e8</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1102140108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1102140108%26sid%3Dliteratum%253Aachs%26aulast%3DLamonica%26aufirst%3DJ.%2BM.%26aulast%3DDeng%26aufirst%3DW.%26aulast%3DKadauke%26aufirst%3DS.%26aulast%3DCampbell%26aufirst%3DA.%2BE.%26aulast%3DGamsjaeger%26aufirst%3DR.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DCheng%26aufirst%3DY.%26aulast%3DBillin%26aufirst%3DA.%2BN.%26aulast%3DHardison%26aufirst%3DR.%2BC.%26aulast%3DMackay%26aufirst%3DJ.%2BP.%26aulast%3DBlobel%26aufirst%3DG.%2BA.%26atitle%3DBromodomain%2520protein%2520Brd3%2520associates%2520with%2520acetylated%2520GATA1%2520to%2520promote%2520its%2520chromatin%2520occupancy%2520at%2520erythroid%2520target%2520genes%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2011%26volume%3D108%26spage%3DE159%26epage%3D168" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Shi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kang, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tao, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rusinova, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, Y. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Evers, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, B. P.</span><span> </span><span class="NLM_article-title">Disrupting the interaction of BRD4 with diacetylated Twist suppresses tumorigenesis in basal-like breast cancer</span> <span class="citation_source-journal">Cancer Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">210</span><span class="NLM_x">–</span> <span class="NLM_lpage">225</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Fjm501893k&amp;key=10.1016%2Fj.ccr.2014.01.028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Fjm501893k&amp;key=24525235" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitlCisrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2014&pages=210-225&author=J.+Shiauthor=Y.+Wangauthor=L.+Zengauthor=Y.+Wuauthor=J.+Dengauthor=Q.+Zhangauthor=Y.+Linauthor=J.+Liauthor=T.+Kangauthor=M.+Taoauthor=E.+Rusinovaauthor=G.+Zhangauthor=C.+Wangauthor=H.+Zhuauthor=J.+Yaoauthor=Y.+X.+Zengauthor=B.+M.+Eversauthor=M.+M.+Zhouauthor=B.+P.+Zhou&title=Disrupting+the+interaction+of+BRD4+with+diacetylated+Twist+suppresses+tumorigenesis+in+basal-like+breast+cancer"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Disrupting the Interaction of BRD4 with Diacetylated Twist Suppresses Tumorigenesis in Basal-like Breast Cancer</span></div><div class="casAuthors">Shi, Jian; Wang, Yifan; Zeng, Lei; Wu, Yadi; Deng, Jiong; Zhang, Qiang; Lin, Yiwei; Li, Junlin; Kang, Tiebang; Tao, Min; Rusinova, Elena; Zhang, Guangtao; Wang, Chi; Zhu, Haining; Yao, Jun; Zeng, Yi-Xin; Evers, B. Mark; Zhou, Ming-Ming; Zhou, Binhua P.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">210-225</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Twist is a key transcription activator of epithelial-mesenchymal transition (EMT).  It remains unclear how Twist induces gene expression.  Here we report a mechanism by which Twist recruits BRD4 to direct WNT5A expression in basal-like breast cancer (BLBC).  Twist contains a "histone H4-mimic" GK-X-GK motif that is diacetylated by Tip60.  The diacetylated Twist binds the second bromodomain of BRD4, whose first bromodomain interacts with acetylated H4, thereby constructing an activated Twist/BRD4/P-TEFb/RNA-Pol II complex at the WNT5A promoter and enhancer.  Pharmacol. inhibition of the Twist-BRD4 assocn. reduced WNT5A expression and suppressed invasion, cancer stem cell (CSC)-like properties, and tumorigenicity of BLBC cells.  Our study indicates that the interaction with BRD4 is crit. for the oncogenic function of Twist in BLBC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovIzriI8A6ZrVg90H21EOLACvtfcHk0lhwfmoFmvZ9mQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitlCisrs%253D&md5=d207cbc1abb187a53d807452b4f10d8f</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2014.01.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2014.01.028%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DZeng%26aufirst%3DL.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DDeng%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DKang%26aufirst%3DT.%26aulast%3DTao%26aufirst%3DM.%26aulast%3DRusinova%26aufirst%3DE.%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DZhu%26aufirst%3DH.%26aulast%3DYao%26aufirst%3DJ.%26aulast%3DZeng%26aufirst%3DY.%2BX.%26aulast%3DEvers%26aufirst%3DB.%2BM.%26aulast%3DZhou%26aufirst%3DM.%2BM.%26aulast%3DZhou%26aufirst%3DB.%2BP.%26atitle%3DDisrupting%2520the%2520interaction%2520of%2520BRD4%2520with%2520diacetylated%2520Twist%2520suppresses%2520tumorigenesis%2520in%2520basal-like%2520breast%2520cancer%26jtitle%3DCancer%2520Cell%26date%3D2014%26volume%3D25%26spage%3D210%26epage%3D225" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Huang, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, X. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ozato, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L. F.</span><span> </span><span class="NLM_article-title">Brd4 coactivates transcriptional activation of NF-kappaB via specific binding to acetylated RelA</span> <span class="citation_source-journal">Mol. Cell. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">1375</span><span class="NLM_x">–</span> <span class="NLM_lpage">1387</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Fjm501893k&amp;key=10.1128%2FMCB.01365-08" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Fjm501893k&amp;key=19103749" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BD1MXitlylsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2009&pages=1375-1387&author=B.+Huangauthor=X.+D.+Yangauthor=M.+M.+Zhouauthor=K.+Ozatoauthor=L.+F.+Chen&title=Brd4+coactivates+transcriptional+activation+of+NF-kappaB+via+specific+binding+to+acetylated+RelA"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Brd4 coactivates transcriptional activation of NF-κB via specific binding to acetylated RelA</span></div><div class="casAuthors">Huang, Bo; Yang, Xiao-Dong; Zhou, Ming-Ming; Ozato, Keiko; Chen, Lin-Feng</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1375-1387</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Acetylation of the RelA subunit of NF-κB, esp. at lysine-310, is crit. for the transcriptional activation of NF-κB and the expression of inflammatory genes.  In this study, we demonstrate that bromodomains of Brd4 bind to acetylated lysine-310.  Brd4 enhances transcriptional activation of NF-κB and the expression of a subset of NF-κB-responsive inflammatory genes in an acetylated lysine-310-dependent manner.  Bromodomains of Brd4 and acetylated lysine-310 of RelA are both required for the mutual interaction and coactivation function of Brd4.  Finally, we demonstrate that Brd4 further recruits CDK9 to phosphorylate C-terminal domain of RNA polymerase II and facilitate the transcription of NF-κB-dependent inflammatory genes.  Our results identify Brd4 as a novel coactivator of NF-κB through specifically binding to acetylated lysine-310 of RelA.  In addn., these studies reveal a mechanism by which acetylated RelA stimulates the transcriptional activity of NF-κB and the NF-κB-dependent inflammatory response.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBnS4p5wzT2LVg90H21EOLACvtfcHk0ljdBtjj3QUwLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXitlylsrc%253D&md5=ae4e16c84d17c5e0e268ff80703a4608</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1128%2FMCB.01365-08&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.01365-08%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DB.%26aulast%3DYang%26aufirst%3DX.%2BD.%26aulast%3DZhou%26aufirst%3DM.%2BM.%26aulast%3DOzato%26aufirst%3DK.%26aulast%3DChen%26aufirst%3DL.%2BF.%26atitle%3DBrd4%2520coactivates%2520transcriptional%2520activation%2520of%2520NF-kappaB%2520via%2520specific%2520binding%2520to%2520acetylated%2520RelA%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2009%26volume%3D29%26spage%3D1375%26epage%3D1387" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Schroder, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cho, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaehlcke, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mak, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lau, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bisgrove, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schnolzer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verdin, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ott, M.</span><span> </span><span class="NLM_article-title">Two-pronged binding with bromodomain-containing protein 4 liberates positive transcription elongation factor b from inactive ribonucleoprotein complexes</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">287</span><span class="NLM_x">, </span> <span class="NLM_fpage">1090</span><span class="NLM_x">–</span> <span class="NLM_lpage">1099</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=287&publication_year=2012&pages=1090-1099&author=S.+Schroderauthor=S.+Choauthor=L.+Zengauthor=Q.+Zhangauthor=K.+Kaehlckeauthor=L.+Makauthor=J.+Lauauthor=D.+Bisgroveauthor=M.+Schnolzerauthor=E.+Verdinauthor=M.+M.+Zhouauthor=M.+Ott&title=Two-pronged+binding+with+bromodomain-containing+protein+4+liberates+positive+transcription+elongation+factor+b+from+inactive+ribonucleoprotein+complexes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSchroder%26aufirst%3DS.%26aulast%3DCho%26aufirst%3DS.%26aulast%3DZeng%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DKaehlcke%26aufirst%3DK.%26aulast%3DMak%26aufirst%3DL.%26aulast%3DLau%26aufirst%3DJ.%26aulast%3DBisgrove%26aufirst%3DD.%26aulast%3DSchnolzer%26aufirst%3DM.%26aulast%3DVerdin%26aufirst%3DE.%26aulast%3DZhou%26aufirst%3DM.%2BM.%26aulast%3DOtt%26aufirst%3DM.%26atitle%3DTwo-pronged%2520binding%2520with%2520bromodomain-containing%2520protein%25204%2520liberates%2520positive%2520transcription%2520elongation%2520factor%2520b%2520from%2520inactive%2520ribonucleoprotein%2520complexes%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2012%26volume%3D287%26spage%3D1090%26epage%3D1099" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Rahman, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sowa, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ottinger, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harper, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Howley, P. M.</span><span> </span><span class="NLM_article-title">The Brd4 extraterminal domain confers transcription activation independent of pTEFb by recruiting multiple proteins, including NSD3</span> <span class="citation_source-journal">Mol. Cell. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">2641</span><span class="NLM_x">–</span> <span class="NLM_lpage">2652</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Fjm501893k&amp;key=10.1128%2FMCB.01341-10" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Fjm501893k&amp;key=21555454" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BC3MXosFKjs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2011&pages=2641-2652&author=S.+Rahmanauthor=M.+E.+Sowaauthor=M.+Ottingerauthor=J.+A.+Smithauthor=Y.+Shiauthor=J.+W.+Harperauthor=P.+M.+Howley&title=The+Brd4+extraterminal+domain+confers+transcription+activation+independent+of+pTEFb+by+recruiting+multiple+proteins%2C+including+NSD3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">The Brd4 extraterminal domain confers transcription activation independent of pTEFb by recruiting multiple proteins, including NSD3</span></div><div class="casAuthors">Rahman, Shaila; Sowa, Mathew E.; Ottinger, Matthias; Smith, Jennifer A.; Shi, Yang; Harper, J. Wade; Howley, Peter M.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">2641-2652</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Bromodomain protein 4 (Brd4) plays crit. roles in development, cancer progression, and virus-host pathogenesis.  To gain mechanistic insight into the various biol. functions of Brd4, we performed a proteomic anal. to identify and characterize Brd4-assocd. cellular proteins.  We found that the extraterminal (ET) domain, whose function has to date not been detd., interacts with NSD3, JMJD6, CHD4, GLTSCR1, and ATAD5.  These ET-domain interactions were also conserved for Brd2 and Brd3, the other human BET proteins tested.  We demonstrated that GLTSCR1, NSD3, and JMJD6 impart a pTEFb-independent transcriptional activation function on Brd4.  NSD3 as well as JMJD6 is recruited to regulated genes in a Brd4-dependent manner.  Moreover, we found that depletion of Brd4 or NSD3 reduces H3K36 methylation, demonstrating that the Brd4/NSD3 complex regulates this specific histone modification.  Our results indicate that the Brd4 ET domain through the recruitment of the specific effectors regulates transcriptional activity.  In particular, we show that one of these effectors, NSD3, regulates transcription by modifying the chromatin microenvironment at Brd4 target genes.  Our study thus identifies the ET domain as a second important transcriptional regulatory domain for Brd4 in addn. to the carboxyl-terminal domain (CTD) that interacts with pTEFb.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrlypbenIW4s7Vg90H21EOLACvtfcHk0ljdBtjj3QUwLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXosFKjs7s%253D&md5=921d55b32a568783ffffdb20cef2f27c</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1128%2FMCB.01341-10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.01341-10%26sid%3Dliteratum%253Aachs%26aulast%3DRahman%26aufirst%3DS.%26aulast%3DSowa%26aufirst%3DM.%2BE.%26aulast%3DOttinger%26aufirst%3DM.%26aulast%3DSmith%26aufirst%3DJ.%2BA.%26aulast%3DShi%26aufirst%3DY.%26aulast%3DHarper%26aufirst%3DJ.%2BW.%26aulast%3DHowley%26aufirst%3DP.%2BM.%26atitle%3DThe%2520Brd4%2520extraterminal%2520domain%2520confers%2520transcription%2520activation%2520independent%2520of%2520pTEFb%2520by%2520recruiting%2520multiple%2520proteins%252C%2520including%2520NSD3%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2011%26volume%3D31%26spage%3D2641%26epage%3D2652" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Bres, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoh, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, K. A.</span><span> </span><span class="NLM_article-title">The multi-tasking P-TEFb complex</span> <span class="citation_source-journal">Curr. Opin. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">334</span><span class="NLM_x">–</span> <span class="NLM_lpage">340</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Fjm501893k&amp;key=10.1016%2Fj.ceb.2008.04.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Fjm501893k&amp;key=18513937" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BD1cXmvV2lsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2008&pages=334-340&author=V.+Bresauthor=S.+M.+Yohauthor=K.+A.+Jones&title=The+multi-tasking+P-TEFb+complex"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">The multi-tasking P-TEFb complex</span></div><div class="casAuthors">Bres, Vanessa; Yoh, Sunnie M.; Jones, Katherine A.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Cell Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">334-340</span>CODEN:
                <span class="NLM_cas:coden">COCBE3</span>;
        ISSN:<span class="NLM_cas:issn">0955-0674</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  P-TEFb (CycT1:Cdk9), the metazoan RNA polymerase II Ser2 C-terminal domain (CTD) kinase, regulates transcription elongation at many genes and integrates mRNA synthesis with histone modification, pre-mRNA processing, and mRNA export.  Recruitment of P-TEFb to target genes requires deubiquitination of H2Bub, phosphorylation of H3S10, and the bromodomain protein, Brd4.  Brd4 activates growth-related genes in the G1 phase of the cell cycle and can also tether P-TEFb to mitotic chromosomes, possibly to mark sites of active transcription throughout cell division.  P-TEFb cooperates with c-Myc during transactivation and cell transformation, and also requires SKIP (c-Ski-interacting protein), an mRNA elongation and splicing factor.  Some functions of the P-TEFb/Ser2P CTD are executed by the Spt6 transcription elongation factor, which binds directly to the phosphorylated CTD and recruits the Iws1 (interacts with Spt6') protein.  Iws1, in turn, interacts with the REF1/Aly nuclear export adaptor and stimulates the kinetics of mRNA export.  Given the prominent role of Spt6 in regulating chromatin structure, the CTD-bound Spt6:Iws1 complex may also control histone modifications during elongation.  Following transcription, P-TEFb accompanies the mature mRNA to the cytoplasm to promote translation elongation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAd7sU5Jw85rVg90H21EOLACvtfcHk0liX8f4kO9OYyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXmvV2lsL8%253D&md5=9e6ecbe1323fd5abc0f13e593c1d9110</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.ceb.2008.04.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ceb.2008.04.008%26sid%3Dliteratum%253Aachs%26aulast%3DBres%26aufirst%3DV.%26aulast%3DYoh%26aufirst%3DS.%2BM.%26aulast%3DJones%26aufirst%3DK.%2BA.%26atitle%3DThe%2520multi-tasking%2520P-TEFb%2520complex%26jtitle%3DCurr.%2520Opin.%2520Cell%2520Biol.%26date%3D2008%26volume%3D20%26spage%3D334%26epage%3D340" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Picaud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, W. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morse, E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keates, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hickman, T. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Felletar, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Philpott, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Munro, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McKeown, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Christie, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">West, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cameron, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwartz, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heightman, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">La Thangue, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">French, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wiest, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kung, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bradner, J. E.</span><span> </span><span class="NLM_article-title">Selective inhibition of BET bromodomains</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">468</span><span class="NLM_x">, </span> <span class="NLM_fpage">1067</span><span class="NLM_x">–</span> <span class="NLM_lpage">1073</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Fjm501893k&amp;key=10.1038%2Fnature09504" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Fjm501893k&amp;key=20871596" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=1067-1073&author=P.+Filippakopoulosauthor=J.+Qiauthor=S.+Picaudauthor=Y.+Shenauthor=W.+B.+Smithauthor=O.+Fedorovauthor=E.+M.+Morseauthor=T.+Keatesauthor=T.+T.+Hickmanauthor=I.+Felletarauthor=M.+Philpottauthor=S.+Munroauthor=M.+R.+McKeownauthor=Y.+Wangauthor=A.+L.+Christieauthor=N.+Westauthor=M.+J.+Cameronauthor=B.+Schwartzauthor=T.+D.+Heightmanauthor=N.+La+Thangueauthor=C.+A.+Frenchauthor=O.+Wiestauthor=A.+L.+Kungauthor=S.+Knappauthor=J.+E.+Bradner&title=Selective+inhibition+of+BET+bromodomains"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibition of BET bromodomains</span></div><div class="casAuthors">Filippakopoulos, Panagis; Qi, Jun; Picaud, Sarah; Shen, Yao; Smith, William B.; Fedorov, Oleg; Morse, Elizabeth M.; Keates, Tracey; Hickman, Tyler T.; Felletar, Ildiko; Philpott, Martin; Munro, Shongah; McKeown, Michael R.; Wang, Yuchuan; Christie, Amanda L.; West, Nathan; Cameron, Michael J.; Schwartz, Brian; Heightman, Tom D.; La Thangue, Nicholas; French, Christopher; Wiest, Olaf; Kung, Andrew L.; Knapp, Stefan; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7327</span>),
    <span class="NLM_cas:pages">1067-1073</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Epigenetic proteins are intently pursued targets in ligand discovery.  So far, successful efforts have been limited to chromatin modifying enzymes, or so-called epigenetic 'writers' and 'erasers'.  Potent inhibitors of histone binding modules have not yet been described.  Here the authors report a cell-permeable small mol. (I,JQ1) that binds competitively to acetyl-lysine recognition motifs, or bromodomains.  High potency and specificity towards a subset of human bromodomains is explained by co-crystal structures with bromodomain and extra-terminal (BET) family member BRD4, revealing excellent shape complementarity with the acetyl-lysine binding cavity.  Recurrent translocation of BRD4 is obsd. in a genetically-defined, incurable subtype of human squamous carcinoma.  Competitive binding by JQ1 displaces the BRD4 fusion oncoprotein from chromatin, prompting squamous differentiation and specific antiproliferative effects in BRD4-dependent cell lines and patient-derived xenograft models.  These data establish proof-of-concept for targeting protein-protein interactions of epigenetic 'readers', and provide a versatile chem. scaffold for the development of chem. probes more broadly throughout the bromodomain family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2ZbtmZM8m_LVg90H21EOLACvtfcHk0liX8f4kO9OYyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D&md5=bd9cadac86124d2c6c3cd5b2a92d68c2</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fnature09504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09504%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DSmith%26aufirst%3DW.%2BB.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMorse%26aufirst%3DE.%2BM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DHickman%26aufirst%3DT.%2BT.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DMunro%26aufirst%3DS.%26aulast%3DMcKeown%26aufirst%3DM.%2BR.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DChristie%26aufirst%3DA.%2BL.%26aulast%3DWest%26aufirst%3DN.%26aulast%3DCameron%26aufirst%3DM.%2BJ.%26aulast%3DSchwartz%26aufirst%3DB.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DLa%2BThangue%26aufirst%3DN.%26aulast%3DFrench%26aufirst%3DC.%2BA.%26aulast%3DWiest%26aufirst%3DO.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DSelective%2520inhibition%2520of%2520BET%2520bromodomains%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D1067%26epage%3D1073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Nicodeme, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeffrey, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schaefer, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beinke, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dewell, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chandwani, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marazzi, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coste, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">White, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kirilovsky, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rice, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lora, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prinjha, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tarakhovsky, A.</span><span> </span><span class="NLM_article-title">Suppression of inflammation by a synthetic histone mimic</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">468</span><span class="NLM_x">, </span> <span class="NLM_fpage">1119</span><span class="NLM_x">–</span> <span class="NLM_lpage">1123</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Fjm501893k&amp;key=10.1038%2Fnature09589" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Fjm501893k&amp;key=21068722" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtl2rtb%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=1119-1123&author=E.+Nicodemeauthor=K.+L.+Jeffreyauthor=U.+Schaeferauthor=S.+Beinkeauthor=S.+Dewellauthor=C.+W.+Chungauthor=R.+Chandwaniauthor=I.+Marazziauthor=P.+Wilsonauthor=H.+Costeauthor=J.+Whiteauthor=J.+Kirilovskyauthor=C.+M.+Riceauthor=J.+M.+Loraauthor=R.+K.+Prinjhaauthor=K.+Leeauthor=A.+Tarakhovsky&title=Suppression+of+inflammation+by+a+synthetic+histone+mimic"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Suppression of inflammation by a synthetic histone mimic</span></div><div class="casAuthors">Nicodeme, Edwige; Jeffrey, Kate L.; Schaefer, Uwe; Beinke, Soren; Dewell, Scott; Chung, Chun-wa; Chandwani, Rohit; Marazzi, Ivan; Wilson, Paul; Coste, Herve; White, Julia; Kirilovsky, Jorge; Rice, Charles M.; Lora, Jose M.; Prinjha, Rab K.; Lee, Kevin; Tarakhovsky, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7327</span>),
    <span class="NLM_cas:pages">1119-1123</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Interaction of pathogens with cells of the immune system results in activation of inflammatory gene expression.  This response, although vital for immune defense, is frequently deleterious to the host due to the exaggerated prodn. of inflammatory proteins.  The scope of inflammatory responses reflects the activation state of signaling proteins upstream of inflammatory genes as well as signal-induced assembly of nuclear chromatin complexes that support mRNA expression.  Recognition of post-translationally modified histones by nuclear proteins that initiate mRNA transcription and support mRNA elongation is a crit. step in the regulation of gene expression.  Here we present a novel pharmacol. approach that targets inflammatory gene expression by interfering with the recognition of acetylated histones by the bromodomain and extra terminal domain (BET) family of proteins.  We describe a synthetic compd. (I-BET) that by mimicking' acetylated histones disrupts chromatin complexes responsible for the expression of key inflammatory genes in activated macrophages, and confers protection against lipopolysaccharide-induced endotoxic shock and bacteria-induced sepsis.  Our findings suggest that synthetic compds. specifically targeting proteins that recognize post-translationally modified histones can serve as a new generation of immunomodulatory drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxc3_PNaiuKLVg90H21EOLACvtfcHk0liX8f4kO9OYyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtl2rtb%252FP&md5=0dfccd9d01b87859512bf591e1c5a93c</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fnature09589&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09589%26sid%3Dliteratum%253Aachs%26aulast%3DNicodeme%26aufirst%3DE.%26aulast%3DJeffrey%26aufirst%3DK.%2BL.%26aulast%3DSchaefer%26aufirst%3DU.%26aulast%3DBeinke%26aufirst%3DS.%26aulast%3DDewell%26aufirst%3DS.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DChandwani%26aufirst%3DR.%26aulast%3DMarazzi%26aufirst%3DI.%26aulast%3DWilson%26aufirst%3DP.%26aulast%3DCoste%26aufirst%3DH.%26aulast%3DWhite%26aufirst%3DJ.%26aulast%3DKirilovsky%26aufirst%3DJ.%26aulast%3DRice%26aufirst%3DC.%2BM.%26aulast%3DLora%26aufirst%3DJ.%2BM.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DTarakhovsky%26aufirst%3DA.%26atitle%3DSuppression%2520of%2520inflammation%2520by%2520a%2520synthetic%2520histone%2520mimic%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D1119%26epage%3D1123" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span> </span><span class="NLM_article-title">Targeting bromodomains: epigenetic readers of lysine acetylation</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">337</span><span class="NLM_x">–</span> <span class="NLM_lpage">356</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Fjm501893k&amp;key=10.1038%2Fnrd4286" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Fjm501893k&amp;key=24751816" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=337-356&author=P.+Filippakopoulosauthor=S.+Knapp&title=Targeting+bromodomains%3A+epigenetic+readers+of+lysine+acetylation"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting bromodomains: epigenetic readers of lysine acetylation</span></div><div class="casAuthors">Filippakopoulos, Panagis; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">337-356</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Lysine acetylation is a key mechanism that regulates chromatin structure; aberrant acetylation levels have been linked to the development of several diseases.  Acetyl-lysine modifications create docking sites for bromodomains, which are small interaction modules found on diverse proteins, some of which have a key role in the acetylation-dependent assembly of transcriptional regulator complexes.  These complexes can then initiate transcriptional programs that result in phenotypic changes.  The recent discovery of potent and highly specific inhibitors for the BET (bromodomain and extra-terminal) family of bromodomains has stimulated intensive research activity in diverse therapeutic areas, particularly in oncol., where BET proteins regulate the expression of key oncogenes and anti-apoptotic proteins.  In addn., targeting BET bromodomains could hold potential for the treatment of inflammation and viral infection.  Here, we highlight recent progress in the development of bromodomain inhibitors, and their potential applications in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOSIcwBJKj4LVg90H21EOLACvtfcHk0lhNiw-X_KCEaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D&md5=77049c82ce6a19b4685e3b7950399a07</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fnrd4286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4286%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DTargeting%2520bromodomains%253A%2520epigenetic%2520readers%2520of%2520lysine%2520acetylation%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D337%26epage%3D356" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Hewings, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Philpott, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Uttarkar, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Picaud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vuppusetty, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marsden, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Conway, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heightman, T. D.</span><span> </span><span class="NLM_article-title">3,5-Dimethylisoxazoles act as acetyl-lysine-mimetic bromodomain ligands</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">6761</span><span class="NLM_x">–</span> <span class="NLM_lpage">6770</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm200640v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=6761-6770&author=D.+S.+Hewingsauthor=M.+Wangauthor=M.+Philpottauthor=O.+Fedorovauthor=S.+Uttarkarauthor=P.+Filippakopoulosauthor=S.+Picaudauthor=C.+Vuppusettyauthor=B.+Marsdenauthor=S.+Knappauthor=S.+J.+Conwayauthor=T.+D.+Heightman&title=3%2C5-Dimethylisoxazoles+act+as+acetyl-lysine-mimetic+bromodomain+ligands"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fjm200640v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200640v%26sid%3Dliteratum%253Aachs%26aulast%3DHewings%26aufirst%3DD.%2BS.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DUttarkar%26aufirst%3DS.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DVuppusetty%26aufirst%3DC.%26aulast%3DMarsden%26aufirst%3DB.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DConway%26aufirst%3DS.%2BJ.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26atitle%3D3%252C5-Dimethylisoxazoles%2520act%2520as%2520acetyl-lysine-mimetic%2520bromodomain%2520ligands%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D6761%26epage%3D6770" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">McKeown, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marineau, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buckley, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kadam, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blacklow, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bradner, J. E.</span><span> </span><span class="NLM_article-title">Biased multicomponent reactions to develop novel bromodomain inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">9019</span><span class="NLM_x">–</span> <span class="NLM_lpage">9027</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm501120z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=9019-9027&author=M.+R.+McKeownauthor=D.+L.+Shawauthor=H.+Fuauthor=S.+Liuauthor=X.+Xuauthor=J.+J.+Marineauauthor=Y.+Huangauthor=X.+Zhangauthor=D.+L.+Buckleyauthor=A.+Kadamauthor=Z.+Zhangauthor=S.+C.+Blacklowauthor=J.+Qiauthor=W.+Zhangauthor=J.+E.+Bradner&title=Biased+multicomponent+reactions+to+develop+novel+bromodomain+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fjm501120z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501120z%26sid%3Dliteratum%253Aachs%26aulast%3DMcKeown%26aufirst%3DM.%2BR.%26aulast%3DShaw%26aufirst%3DD.%2BL.%26aulast%3DFu%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DXu%26aufirst%3DX.%26aulast%3DMarineau%26aufirst%3DJ.%2BJ.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DKadam%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DBlacklow%26aufirst%3DS.%2BC.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DBiased%2520multicomponent%2520reactions%2520to%2520develop%2520novel%2520bromodomain%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D9019%26epage%3D9027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Fish, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bish, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brennan, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunnage, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cook, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Federov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerstenberger, B. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marsden, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nocka, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Owen, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Philpott, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Picaud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Primiano, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ralph, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sciammetta, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trzupek, J. D.</span><span> </span><span class="NLM_article-title">Identification of a chemical probe for bromo and extra C-terminal bromodomain inhibition through optimization of a fragment-derived hit</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">9831</span><span class="NLM_x">–</span> <span class="NLM_lpage">9837</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm3010515" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFGmtLnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=9831-9837&author=P.+V.+Fishauthor=P.+Filippakopoulosauthor=G.+Bishauthor=P.+E.+Brennanauthor=M.+E.+Bunnageauthor=A.+S.+Cookauthor=O.+Federovauthor=B.+S.+Gerstenbergerauthor=H.+Jonesauthor=S.+Knappauthor=B.+Marsdenauthor=K.+Nockaauthor=D.+R.+Owenauthor=M.+Philpottauthor=S.+Picaudauthor=M.+J.+Primianoauthor=M.+J.+Ralphauthor=N.+Sciammettaauthor=J.+D.+Trzupek&title=Identification+of+a+chemical+probe+for+bromo+and+extra+C-terminal+bromodomain+inhibition+through+optimization+of+a+fragment-derived+hit"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a Chemical Probe for Bromo and Extra C-Terminal Bromodomain Inhibition through Optimization of a Fragment-Derived Hit</span></div><div class="casAuthors">Fish, Paul V.; Filippakopoulos, Panagis; Bish, Gerwyn; Brennan, Paul E.; Bunnage, Mark E.; Cook, Andrew S.; Federov, Oleg; Gerstenberger, Brian S.; Jones, Hannah; Knapp, Stefan; Marsden, Brian; Nocka, Karl; Owen, Dafydd R.; Philpott, Martin; Picaud, Sarah; Primiano, Michael J.; Ralph, Michael J.; Sciammetta, Nunzio; Trzupek, John D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">9831-9837</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The posttranslational modification of chromatin through acetylation at selected histone lysine residues is governed by histone acetyltransferases (HATs) and histone deacetylases (HDACs).  The significance of this subset of the epigenetic code is interrogated and interpreted by an acetyllysine-specific protein-protein interaction with bromodomain reader modules.  Selective inhibition of the bromo and extra C-terminal domain (BET) family of bromodomains with a small mol. is feasible, and this may represent an opportunity for disease intervention through the recently disclosed antiproliferative and anti-inflammatory properties of such inhibitors.  Herein, we describe the discovery and structure-activity relationship (SAR) of a novel, small-mol. chem. probe for BET family inhibition that was identified through the application of structure-based fragment assessment and optimization techniques.  This has yielded a potent, selective compd. with cell-based activity (PFI-1) that may further add to the understanding of BET family function within the bromodomains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphzpWnuXoLdrVg90H21EOLACvtfcHk0lhNiw-X_KCEaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFGmtLnE&md5=ba864f3916bea5f793dd0b26732784f0</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fjm3010515&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm3010515%26sid%3Dliteratum%253Aachs%26aulast%3DFish%26aufirst%3DP.%2BV.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DBish%26aufirst%3DG.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DCook%26aufirst%3DA.%2BS.%26aulast%3DFederov%26aufirst%3DO.%26aulast%3DGerstenberger%26aufirst%3DB.%2BS.%26aulast%3DJones%26aufirst%3DH.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DMarsden%26aufirst%3DB.%26aulast%3DNocka%26aufirst%3DK.%26aulast%3DOwen%26aufirst%3DD.%2BR.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DPrimiano%26aufirst%3DM.%2BJ.%26aulast%3DRalph%26aufirst%3DM.%2BJ.%26aulast%3DSciammetta%26aufirst%3DN.%26aulast%3DTrzupek%26aufirst%3DJ.%2BD.%26atitle%3DIdentification%2520of%2520a%2520chemical%2520probe%2520for%2520bromo%2520and%2520extra%2520C-terminal%2520bromodomain%2520inhibition%2520through%2520optimization%2520of%2520a%2520fragment-derived%2520hit%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D9831%26epage%3D9837" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Chung, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dean, A. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woolven, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bamborough, P.</span><span> </span><span class="NLM_article-title">Fragment-based discovery of bromodomain inhibitors part 1: inhibitor binding modes and implications for lead discovery</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">576</span><span class="NLM_x">–</span> <span class="NLM_lpage">586</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm201320w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFKktrnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=576-586&author=C.+W.+Chungauthor=A.+W.+Deanauthor=J.+M.+Woolvenauthor=P.+Bamborough&title=Fragment-based+discovery+of+bromodomain+inhibitors+part+1%3A+inhibitor+binding+modes+and+implications+for+lead+discovery"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Discovery of Bromodomain Inhibitors Part 1: Inhibitor Binding Modes and Implications for Lead Discovery</span></div><div class="casAuthors">Chung, Chun-wa; Dean, Anthony W.; Woolven, James M.; Bamborough, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">576-586</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomain-contg. proteins are key epigenetic regulators of gene transcription and readers of the histone code.  However, the therapeutic benefits of modulating this target class are largely unexplored due to the lack of suitable chem. probes.  This article describes the generation of lead mols. for the BET bromodomains through screening a fragment set chosen using structural insights and computational approaches.  Anal. of 40 BRD2/fragment x-ray complexes highlights both shared and disparate interaction features that may be exploited for affinity and selectivity.  Six representative crystal structures are then exemplified in detail.  Two of the fragments are completely new bromodomain chemotypes, and three have never before been crystd. in a bromodomain, so our results significantly extend the limited public knowledge-base of crystallog. small mol./bromodomain interactions.  Certain fragments (including paracetamol) bind in a consistent mode to different bromodomains such as CREBBP, suggesting their potential to act as generic bromodomain templates.  An important implication is that the bromodomains are not only a phylogenetic family but also a system in which chem. and structural knowledge of one bromodomain gives insights transferable to others.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonNb7GnZnS_rVg90H21EOLACvtfcHk0lim1MtJSkiEHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFKktrnN&md5=5b7d7239d0de57266d94beef1ea080ae</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fjm201320w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm201320w%26sid%3Dliteratum%253Aachs%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DDean%26aufirst%3DA.%2BW.%26aulast%3DWoolven%26aufirst%3DJ.%2BM.%26aulast%3DBamborough%26aufirst%3DP.%26atitle%3DFragment-based%2520discovery%2520of%2520bromodomain%2520inhibitors%2520part%25201%253A%2520inhibitor%2520binding%2520modes%2520and%2520implications%2520for%2520lead%2520discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D576%26epage%3D586" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Vidler, L. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Picaud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tomsett, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woodward, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoelder, S.</span><span> </span><span class="NLM_article-title">Discovery of novel small-molecule inhibitors of BRD4 using structure-based virtual screening</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">8073</span><span class="NLM_x">–</span> <span class="NLM_lpage">8088</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm4011302" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFOjurrI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=8073-8088&author=L.+R.+Vidlerauthor=P.+Filippakopoulosauthor=O.+Fedorovauthor=S.+Picaudauthor=S.+Martinauthor=M.+Tomsettauthor=H.+Woodwardauthor=N.+Brownauthor=S.+Knappauthor=S.+Hoelder&title=Discovery+of+novel+small-molecule+inhibitors+of+BRD4+using+structure-based+virtual+screening"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Novel Small-Molecule Inhibitors of BRD4 Using Structure-Based Virtual Screening</span></div><div class="casAuthors">Vidler, Lewis R.; Filippakopoulos, Panagis; Fedorov, Oleg; Picaud, Sarah; Martin, Sarah; Tomsett, Michael; Woodward, Hannah; Brown, Nathan; Knapp, Stefan; Hoelder, Swen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">8073-8088</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomains (BRDs) are epigenetic readers that recognize acetylated-lysine (KAc) on proteins and are implicated in a no. of diseases.  We describe a virtual screening approach to identify BRD inhibitors.  Key elements of this approach are the extensive design and use of substructure queries to compile a set of com. available compds. featuring novel putative KAc mimetics and docking this set for final compd. selection.  We describe the validation of this approach by applying it to the first BRD of BRD4.  The selection and testing of 143 compds. lead to the discovery of six novel hits, including four unprecedented KAc mimetics.  We solved the crystal structure of four hits, detd. their binding mode, and improved their potency through synthesis and the purchase of derivs.  This work provides a validated virtual screening approach that is applicable to other BRDs and describes novel KAc mimetics that can be further explored to design more potent inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGow5yj8bnohUbVg90H21EOLACvtfcHk0lim1MtJSkiEHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFOjurrI&md5=c1f76e4838713335be7a93ebe4c68701</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fjm4011302&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm4011302%26sid%3Dliteratum%253Aachs%26aulast%3DVidler%26aufirst%3DL.%2BR.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DTomsett%26aufirst%3DM.%26aulast%3DWoodward%26aufirst%3DH.%26aulast%3DBrown%26aufirst%3DN.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DHoelder%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520novel%2520small-molecule%2520inhibitors%2520of%2520BRD4%2520using%2520structure-based%2520virtual%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D8073%26epage%3D8088" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Zhao, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miao, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Du, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiong, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, J.</span><span> </span><span class="NLM_article-title">Fragment-based drug discovery of 2-thiazolidinones as inhibitors of the histone reader BRD4 bromodomain</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">3833</span><span class="NLM_x">–</span> <span class="NLM_lpage">3851</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm301793a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BC3sXksF2htL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=3833-3851&author=L.+Zhaoauthor=D.+Caoauthor=T.+Chenauthor=Y.+Wangauthor=Z.+Miaoauthor=Y.+Xuauthor=W.+Chenauthor=X.+Wangauthor=Y.+Liauthor=Z.+Duauthor=B.+Xiongauthor=J.+Liauthor=C.+Xuauthor=N.+Zhangauthor=J.+Heauthor=J.+Shen&title=Fragment-based+drug+discovery+of+2-thiazolidinones+as+inhibitors+of+the+histone+reader+BRD4+bromodomain"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Drug Discovery of 2-Thiazolidinones as Inhibitors of the Histone Reader BRD4 Bromodomain</span></div><div class="casAuthors">Zhao, Lele; Cao, Danyan; Chen, Tiantian; Wang, Yingqing; Miao, Zehong; Xu, Yechun; Chen, Wuyan; Wang, Xin; Li, Yanlian; Du, Zhiyan; Xiong, Bing; Li, Jian; Xu, Chunyan; Zhang, Naixia; He, Jianhua; Shen, Jingkang</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3833-3851</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Recognizing acetyllysine of histone is a vital process of epigenetic regulation that is mediated by a protein module called bromodomain.  To contribute novel scaffolds for developing into bromodomain inhibitors, the authors utilize a fragment-based drug discovery approach.  By successively applying docking and x-ray crystallog., they were able to identify 9 fragment hits from diffracting more than 60 crystals.  In the present work, they described four of them and carried out the integrated lead optimization for fragment I, which bears a 2-thiazolidinone core.  After several rounds of structure guided modifications, they assessed the druggability of 2-thiazolidinone by modulating in vitro pharmacokinetic studies and cellular activity assay.  The results showed that two potent of 2-thiazolidinone compds. II and III have good metabolic stability.  Also, the cellular assay confirmed the activities of 2-thiazolidinones.  Together, they hope the identified 2-thiazolidinone chemotype and other fragment hits described herein can stimulate researchers to develop more diversified bromodomain inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfi2pyYq6c4LVg90H21EOLACvtfcHk0lim1MtJSkiEHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXksF2htL4%253D&md5=8308db5d83cb72b29f261cc134514ece</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fjm301793a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm301793a%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DL.%26aulast%3DCao%26aufirst%3DD.%26aulast%3DChen%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DMiao%26aufirst%3DZ.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DDu%26aufirst%3DZ.%26aulast%3DXiong%26aufirst%3DB.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DN.%26aulast%3DHe%26aufirst%3DJ.%26aulast%3DShen%26aufirst%3DJ.%26atitle%3DFragment-based%2520drug%2520discovery%2520of%25202-thiazolidinones%2520as%2520inhibitors%2520of%2520the%2520histone%2520reader%2520BRD4%2520bromodomain%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D3833%26epage%3D3851" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Lucas, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wohlwend, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hugle, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmidtkunz, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerhardt, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schule, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jung, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Einsle, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gunther, S.</span><span> </span><span class="NLM_article-title">4-Acyl pyrroles: mimicking acetylated lysines in histone code reading</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">14055</span><span class="NLM_x">–</span> <span class="NLM_lpage">14059</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2013&pages=14055-14059&author=X.+Lucasauthor=D.+Wohlwendauthor=M.+Hugleauthor=K.+Schmidtkunzauthor=S.+Gerhardtauthor=R.+Schuleauthor=M.+Jungauthor=O.+Einsleauthor=S.+Gunther&title=4-Acyl+pyrroles%3A+mimicking+acetylated+lysines+in+histone+code+reading"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLucas%26aufirst%3DX.%26aulast%3DWohlwend%26aufirst%3DD.%26aulast%3DHugle%26aufirst%3DM.%26aulast%3DSchmidtkunz%26aufirst%3DK.%26aulast%3DGerhardt%26aufirst%3DS.%26aulast%3DSchule%26aufirst%3DR.%26aulast%3DJung%26aufirst%3DM.%26aulast%3DEinsle%26aufirst%3DO.%26aulast%3DGunther%26aufirst%3DS.%26atitle%3D4-Acyl%2520pyrroles%253A%2520mimicking%2520acetylated%2520lysines%2520in%2520histone%2520code%2520reading%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2013%26volume%3D52%26spage%3D14055%26epage%3D14059" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Fedorov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lingard, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wells, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Monteiro, O. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Picaud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keates, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yapp, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Philpott, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Felletar, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marsden, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muller, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brennan, P. E.</span><span> </span><span class="NLM_article-title">[1,2,4]Triazolo[4,3-a]phthalazines: inhibitors of diverse bromodomains</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">462</span><span class="NLM_x">–</span> <span class="NLM_lpage">476</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm401568s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=462-476&author=O.+Fedorovauthor=H.+Lingardauthor=C.+Wellsauthor=O.+P.+Monteiroauthor=S.+Picaudauthor=T.+Keatesauthor=C.+Yappauthor=M.+Philpottauthor=S.+J.+Martinauthor=I.+Felletarauthor=B.+D.+Marsdenauthor=P.+Filippakopoulosauthor=S.+Mullerauthor=S.+Knappauthor=P.+E.+Brennan&title=%5B1%2C2%2C4%5DTriazolo%5B4%2C3-a%5Dphthalazines%3A+inhibitors+of+diverse+bromodomains"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fjm401568s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401568s%26sid%3Dliteratum%253Aachs%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DLingard%26aufirst%3DH.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DMonteiro%26aufirst%3DO.%2BP.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DYapp%26aufirst%3DC.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DMartin%26aufirst%3DS.%2BJ.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DMarsden%26aufirst%3DB.%2BD.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26atitle%3D%255B1%252C2%252C4%255DTriazolo%255B4%252C3-a%255Dphthalazines%253A%2520inhibitors%2520of%2520diverse%2520bromodomains%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D462%26epage%3D476" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Bamborough, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diallo, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goodacre, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gordon, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lewis, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seal, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woodrow, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, C. W.</span><span> </span><span class="NLM_article-title">Fragment-based discovery of bromodomain inhibitors part 2: optimization of phenylisoxazole sulfonamides</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">587</span><span class="NLM_x">–</span> <span class="NLM_lpage">596</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm201283q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFKktrfF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=587-596&author=P.+Bamboroughauthor=H.+Dialloauthor=J.+D.+Goodacreauthor=L.+Gordonauthor=A.+Lewisauthor=J.+T.+Sealauthor=D.+M.+Wilsonauthor=M.+D.+Woodrowauthor=C.+W.+Chung&title=Fragment-based+discovery+of+bromodomain+inhibitors+part+2%3A+optimization+of+phenylisoxazole+sulfonamides"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Discovery of Bromodomain Inhibitors Part 2: Optimization of Phenylisoxazole Sulfonamides</span></div><div class="casAuthors">Bamborough, Paul; Diallo, Hawa; Goodacre, Jonathan D.; Gordon, Laurie; Lewis, Antonia; Seal, Jonathan T.; Wilson, David M.; Woodrow, Michael D.; Chung, Chun-wa</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">587-596</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomains are epigenetic reader modules that regulate gene transcription through their recognition of acetyl-lysine modified histone tails.  Inhibitors of this protein-protein interaction have the potential to modulate multiple diseases as demonstrated by the profound anti-inflammatory and antiproliferative effects of a recently disclosed class of BET compds.  While these compds. were discovered using phenotypic assays, here we present a highly efficient alternative approach to find new chem. templates, exploiting the abundant structural knowledge that exists for this target class.  A Ph di-Me isoxazole chemotype resulting from a focused fragment screen has been rapidly optimized through structure-based design, leading to a sulfonamide series showing anti-inflammatory activity in cellular assays.  This proof-of-principle expt. demonstrates the tractability of the BET family and bromodomain target class to fragment-based hit discovery and structure-based lead optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqr6lsic9qjJrVg90H21EOLACvtfcHk0lgy34XdxIemLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFKktrfF&md5=e73b16b3fddf88b43b037264b000b276</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fjm201283q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm201283q%26sid%3Dliteratum%253Aachs%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DDiallo%26aufirst%3DH.%26aulast%3DGoodacre%26aufirst%3DJ.%2BD.%26aulast%3DGordon%26aufirst%3DL.%26aulast%3DLewis%26aufirst%3DA.%26aulast%3DSeal%26aufirst%3DJ.%2BT.%26aulast%3DWilson%26aufirst%3DD.%2BM.%26aulast%3DWoodrow%26aufirst%3DM.%2BD.%26aulast%3DChung%26aufirst%3DC.%2BW.%26atitle%3DFragment-based%2520discovery%2520of%2520bromodomain%2520inhibitors%2520part%25202%253A%2520optimization%2520of%2520phenylisoxazole%2520sulfonamides%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D587%26epage%3D596" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Picaud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Da Costa, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thanasopoulou, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fish, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Philpott, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brennan, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunnage, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Owen, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bradner, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taniere, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Sullivan, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muller, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwaller, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stankovic, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span> </span><span class="NLM_article-title">PFI-1, a highly selective protein interaction inhibitor, targeting BET Bromodomains</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">73</span><span class="NLM_x">, </span> <span class="NLM_fpage">3336</span><span class="NLM_x">–</span> <span class="NLM_lpage">3346</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2013&pages=3336-3346&author=S.+Picaudauthor=D.+Da+Costaauthor=A.+Thanasopoulouauthor=P.+Filippakopoulosauthor=P.+V.+Fishauthor=M.+Philpottauthor=O.+Fedorovauthor=P.+Brennanauthor=M.+E.+Bunnageauthor=D.+R.+Owenauthor=J.+E.+Bradnerauthor=P.+Taniereauthor=B.+O%E2%80%99Sullivanauthor=S.+Mullerauthor=J.+Schwallerauthor=T.+Stankovicauthor=S.+Knapp&title=PFI-1%2C+a+highly+selective+protein+interaction+inhibitor%2C+targeting+BET+Bromodomains"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DDa%2BCosta%26aufirst%3DD.%26aulast%3DThanasopoulou%26aufirst%3DA.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DFish%26aufirst%3DP.%2BV.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DBrennan%26aufirst%3DP.%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DOwen%26aufirst%3DD.%2BR.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DTaniere%26aufirst%3DP.%26aulast%3DO%25E2%2580%2599Sullivan%26aufirst%3DB.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DSchwaller%26aufirst%3DJ.%26aulast%3DStankovic%26aufirst%3DT.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DPFI-1%252C%2520a%2520highly%2520selective%2520protein%2520interaction%2520inhibitor%252C%2520targeting%2520BET%2520Bromodomains%26jtitle%3DCancer%2520Res.%26date%3D2013%26volume%3D73%26spage%3D3336%26epage%3D3346" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Vidler, L. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoelder, S.</span><span> </span><span class="NLM_article-title">Druggability analysis and structural classification of bromodomain acetyl-lysine binding sites</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">7346</span><span class="NLM_x">–</span> <span class="NLM_lpage">7359</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm300346w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVWht7fN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=7346-7359&author=L.+R.+Vidlerauthor=N.+Brownauthor=S.+Knappauthor=S.+Hoelder&title=Druggability+analysis+and+structural+classification+of+bromodomain+acetyl-lysine+binding+sites"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Druggability Analysis and Structural Classification of Bromodomain Acetyl-lysine Binding Sites</span></div><div class="casAuthors">Vidler, Lewis R.; Brown, Nathan; Knapp, Stefan; Hoelder, Swen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">7346-7359</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomains are readers of the epigenetic code that specifically bind acetyl-lysine contg. recognition sites on proteins.  Recently the BET family of bromodomains has been demonstrated to be druggable through the discovery of potent inhibitors, sparking an interest in protein-protein interaction inhibitors that directly target gene transcription.  Here, we assess the druggability of diverse members of the bromodomain family using SiteMap and show that there are significant differences in predicted druggability.  Furthermore, we trace these differences in druggability back to unique amino acid signatures in the bromodomain acetyl-lysine binding sites.  These signatures were then used to generate a new classification of the bromodomain family, visualized as a classification tree.  This represents the first anal. of this type for the bromodomain family and can prove useful in the discovery of inhibitors, particularly for anticipating screening hit rates, identifying inhibitors that can be explored for lead hopping approaches, and selecting proteins for selectivity screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxYqQyqqj7JrVg90H21EOLACvtfcHk0lgy34XdxIemLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVWht7fN&md5=bfb08154c1964aab5df51cb3513f2c1d</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fjm300346w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm300346w%26sid%3Dliteratum%253Aachs%26aulast%3DVidler%26aufirst%3DL.%2BR.%26aulast%3DBrown%26aufirst%3DN.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DHoelder%26aufirst%3DS.%26atitle%3DDruggability%2520analysis%2520and%2520structural%2520classification%2520of%2520bromodomain%2520acetyl-lysine%2520binding%2520sites%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D7346%26epage%3D7359" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span>SGC. GSK2801: A Selective Chemical Probe for BAZ2B/A Bromodomains. <a href="http://www.thesgc.org/chemical-probes/GSK2801" class="extLink">http://www.thesgc.org/chemical-probes/GSK2801</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=SGC.+GSK2801%3A+A+Selective+Chemical+Probe+for+BAZ2B%2FA+Bromodomains.+http%3A%2F%2Fwww.thesgc.org%2Fchemical-probes%2FGSK2801."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Hay, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Singleton, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tallant, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wells, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Picaud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Philpott, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Monteiro, O. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rogers, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Conway, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rooney, T. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tumber, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yapp, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunnage, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muller, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brennan, P. E.</span><span> </span><span class="NLM_article-title">Discovery and optimization of small-molecule ligands for the CBP/p300 bromodomains</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">136</span><span class="NLM_x">, </span> <span class="NLM_fpage">9308</span><span class="NLM_x">–</span> <span class="NLM_lpage">9319</span></span><div class="citationLinks">[<a href="/doi/10.1021/ja412434f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVShsLvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=136&publication_year=2014&pages=9308-9319&author=D.+A.+Hayauthor=O.+Fedorovauthor=S.+Martinauthor=D.+C.+Singletonauthor=C.+Tallantauthor=C.+Wellsauthor=S.+Picaudauthor=M.+Philpottauthor=O.+P.+Monteiroauthor=C.+M.+Rogersauthor=S.+J.+Conwayauthor=T.+P.+Rooneyauthor=A.+Tumberauthor=C.+Yappauthor=P.+Filippakopoulosauthor=M.+E.+Bunnageauthor=S.+Mullerauthor=S.+Knappauthor=C.+J.+Schofieldauthor=P.+E.+Brennan&title=Discovery+and+optimization+of+small-molecule+ligands+for+the+CBP%2Fp300+bromodomains"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Optimization of Small-Molecule Ligands for the CBP/p300 Bromodomains</span></div><div class="casAuthors">Hay, Duncan A.; Fedorov, Oleg; Martin, Sarah; Singleton, Dean C.; Tallant, Cynthia; Wells, Christopher; Picaud, Sarah; Philpott, Martin; Monteiro, Octovia P.; Rogers, Catherine M.; Conway, Stuart J.; Rooney, Timothy P. C.; Tumber, Anthony; Yapp, Clarence; Filippakopoulos, Panagis; Bunnage, Mark E.; Muller, Susanne; Knapp, Stefan; Schofield, Christopher J.; Brennan, Paul E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">136</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">9308-9319</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Small-mol. inhibitors that target bromodomains outside of the bromodomain and extra-terminal (BET) sub-family are lacking.  Here, the authors describe highly potent and selective ligands for the bromodomain module of the human lysine acetyl transferase CBP/p300, developed from a series of 5-isoxazolyl-benzimidazoles.  The starting point was a fragment hit, which was optimized into a more potent and selective lead using parallel synthesis employing Suzuki couplings, benzimidazole-forming reactions, and reductive aminations.  The selectivity of the lead compd. against other bromodomain family members was investigated using a thermal stability assay, which revealed some inhibition of the structurally related BET family members.  To address the BET selectivity issue, x-ray crystal structures of the lead compd. bound to the CREB binding protein (CBP) and the first bromodomain of BRD4 (BRD4(1)) were used to guide the design of more selective compds.  The crystal structures obtained revealed two distinct binding modes.  By varying the aryl substitution pattern and developing conformationally constrained analogs, selectivity for CBP over BRD4(1) was increased.  The optimized compd. is highly potent (Kd = 21 nM) and selective, displaying 40-fold selectivity over BRD4(1).  Cellular activity was demonstrated using fluorescence recovery after photo-bleaching (FRAP) and a p53 reporter assay.  The optimized compds. are cell-active and have nanomolar affinity for CBP/p300; therefore, they should be useful in studies investigating the biol. roles of CBP and p300 and to validate the CBP and p300 bromodomains as therapeutic targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqFzgs1DD4QZLVg90H21EOLACvtfcHk0ljdFhUpvK2-xg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVShsLvP&md5=e5da3e96a4d234453a481c7ee5e788f2</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fja412434f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja412434f%26sid%3Dliteratum%253Aachs%26aulast%3DHay%26aufirst%3DD.%2BA.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DSingleton%26aufirst%3DD.%2BC.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DMonteiro%26aufirst%3DO.%2BP.%26aulast%3DRogers%26aufirst%3DC.%2BM.%26aulast%3DConway%26aufirst%3DS.%2BJ.%26aulast%3DRooney%26aufirst%3DT.%2BP.%26aulast%3DTumber%26aufirst%3DA.%26aulast%3DYapp%26aufirst%3DC.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26atitle%3DDiscovery%2520and%2520optimization%2520of%2520small-molecule%2520ligands%2520for%2520the%2520CBP%252Fp300%2520bromodomains%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2014%26volume%3D136%26spage%3D9308%26epage%3D9319" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Chaikuad, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petros, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span> </span><span class="NLM_article-title">Structure-based approaches towards identification of fragments for the low-druggability ATAD2 bromodomain</span> <span class="citation_source-journal">Med. Chem. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">1843</span><span class="NLM_x">–</span> <span class="NLM_lpage">1848</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Fjm501893k&amp;key=10.1039%2FC4MD00237G" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=1843-1848&author=A.+Chaikuadauthor=A.+M.+Petrosauthor=O.+Fedorovauthor=J.+Xuauthor=S.+Knapp&title=Structure-based+approaches+towards+identification+of+fragments+for+the+low-druggability+ATAD2+bromodomain"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1039%2FC4MD00237G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC4MD00237G%26sid%3Dliteratum%253Aachs%26aulast%3DChaikuad%26aufirst%3DA.%26aulast%3DPetros%26aufirst%3DA.%2BM.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DStructure-based%2520approaches%2520towards%2520identification%2520of%2520fragments%2520for%2520the%2520low-druggability%2520ATAD2%2520bromodomain%26jtitle%3DMed.%2520Chem.%2520Commun.%26date%3D2014%26volume%3D5%26spage%3D1843%26epage%3D1848" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">Harner, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chauder, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Phan, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fesik, S. W.</span><span> </span><span class="NLM_article-title">Fragment-Based Screening of the Bromodomain of ATAD2</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">9687</span><span class="NLM_x">–</span> <span class="NLM_lpage">9692</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm501035j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=9687-9692&author=M.+J.+Harnerauthor=B.+A.+Chauderauthor=J.+Phanauthor=S.+W.+Fesik&title=Fragment-Based+Screening+of+the+Bromodomain+of+ATAD2"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fjm501035j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501035j%26sid%3Dliteratum%253Aachs%26aulast%3DHarner%26aufirst%3DM.%2BJ.%26aulast%3DChauder%26aufirst%3DB.%2BA.%26aulast%3DPhan%26aufirst%3DJ.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26atitle%3DFragment-Based%2520Screening%2520of%2520the%2520Bromodomain%2520of%2520ATAD2%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D9687%26epage%3D9692" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group">Ferguson, F. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chaikuad, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Philpott, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muniz, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Felletar, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">von Delft, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heightman, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abell, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciulli, A.</span><span> </span><span class="NLM_article-title">Targeting low-druggability bromodomains: fragment based screening and inhibitor design against the BAZ2B bromodomain</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">10183</span><span class="NLM_x">–</span> <span class="NLM_lpage">10187</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm401582c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=10183-10187&author=F.+M.+Fergusonauthor=O.+Fedorovauthor=A.+Chaikuadauthor=M.+Philpottauthor=J.+R.+Munizauthor=I.+Felletarauthor=F.+von+Delftauthor=T.+Heightmanauthor=S.+Knappauthor=C.+Abellauthor=A.+Ciulli&title=Targeting+low-druggability+bromodomains%3A+fragment+based+screening+and+inhibitor+design+against+the+BAZ2B+bromodomain"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Fjm401582c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401582c%26sid%3Dliteratum%253Aachs%26aulast%3DFerguson%26aufirst%3DF.%2BM.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DChaikuad%26aufirst%3DA.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DMuniz%26aufirst%3DJ.%2BR.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3Dvon%2BDelft%26aufirst%3DF.%26aulast%3DHeightman%26aufirst%3DT.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DAbell%26aufirst%3DC.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DTargeting%2520low-druggability%2520bromodomains%253A%2520fragment%2520based%2520screening%2520and%2520inhibitor%2520design%2520against%2520the%2520BAZ2B%2520bromodomain%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D10183%26epage%3D10187" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group">Demont, E. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bamborough, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Craggs, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fallon, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gordon, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grandi, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hobbs, C. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hussain, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Le Gall, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Michon, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mitchell, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prinjha, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roberts, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sheppard, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watson, R. J.</span><span> </span><span class="NLM_article-title">1,3-Dimethyl benzimidazolones are potent, selective inhibitors of the BRPF1 bromodomain</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">1190</span><span class="NLM_x">–</span> <span class="NLM_lpage">1195</span></span><div class="citationLinks">[<a href="/doi/10.1021/ml5002932" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=1190-1195&author=E.+H.+Demontauthor=P.+Bamboroughauthor=C.+W.+Chungauthor=P.+D.+Craggsauthor=D.+Fallonauthor=L.+J.+Gordonauthor=P.+Grandiauthor=C.+I.+Hobbsauthor=J.+Hussainauthor=E.+J.+Jonesauthor=A.+Le+Gallauthor=A.+M.+Michonauthor=D.+J.+Mitchellauthor=R.+K.+Prinjhaauthor=A.+D.+Robertsauthor=R.+J.+Sheppardauthor=R.+J.+Watson&title=1%2C3-Dimethyl+benzimidazolones+are+potent%2C+selective+inhibitors+of+the+BRPF1+bromodomain"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fml5002932&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml5002932%26sid%3Dliteratum%253Aachs%26aulast%3DDemont%26aufirst%3DE.%2BH.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DCraggs%26aufirst%3DP.%2BD.%26aulast%3DFallon%26aufirst%3DD.%26aulast%3DGordon%26aufirst%3DL.%2BJ.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DHobbs%26aufirst%3DC.%2BI.%26aulast%3DHussain%26aufirst%3DJ.%26aulast%3DJones%26aufirst%3DE.%2BJ.%26aulast%3DLe%2BGall%26aufirst%3DA.%26aulast%3DMichon%26aufirst%3DA.%2BM.%26aulast%3DMitchell%26aufirst%3DD.%2BJ.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DRoberts%26aufirst%3DA.%2BD.%26aulast%3DSheppard%26aufirst%3DR.%2BJ.%26aulast%3DWatson%26aufirst%3DR.%2BJ.%26atitle%3D1%252C3-Dimethyl%2520benzimidazolones%2520are%2520potent%252C%2520selective%2520inhibitors%2520of%2520the%2520BRPF1%2520bromodomain%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D5%26spage%3D1190%26epage%3D1195" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group">Ciceri, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muller, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Mahony, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunt, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lasater, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pallares, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Picaud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wells, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wodicka, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shah, N. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Treiber, D. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span> </span><span class="NLM_article-title">Dual kinase-bromodomain inhibitors for rationally designed polypharmacology</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">305</span><span class="NLM_x">–</span> <span class="NLM_lpage">312</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Fjm501893k&amp;key=10.1038%2Fnchembio.1471" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Fjm501893k&amp;key=24584101" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjtl2itro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=305-312&author=P.+Ciceriauthor=S.+Mullerauthor=A.+O%E2%80%99Mahonyauthor=O.+Fedorovauthor=P.+Filippakopoulosauthor=J.+P.+Huntauthor=E.+A.+Lasaterauthor=G.+Pallaresauthor=S.+Picaudauthor=C.+Wellsauthor=S.+Martinauthor=L.+M.+Wodickaauthor=N.+P.+Shahauthor=D.+K.+Treiberauthor=S.+Knapp&title=Dual+kinase-bromodomain+inhibitors+for+rationally+designed+polypharmacology"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Dual kinase-bromodomain inhibitors for rationally designed polypharmacology</span></div><div class="casAuthors">Ciceri, Pietro; Mueller, Susanne; O'Mahony, Alison; Fedorov, Oleg; Filippakopoulos, Panagis; Hunt, Jeremy P.; Lasater, Elisabeth A.; Pallares, Gabriel; Picaud, Sarah; Wells, Christopher; Martin, Sarah; Wodicka, Lisa M.; Shah, Neil P.; Treiber, Daniel K.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">305-312</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Concomitant inhibition of multiple cancer-driving kinases is an established strategy to improve the durability of clin. responses to targeted therapies.  The difficulty of discovering kinase inhibitors with an appropriate multitarget profile has, however, necessitated the application of combination therapies, which can pose major clin. development challenges.  Epigenetic reader domains of the bromodomain family have recently emerged as new targets for cancer therapy.  Here the authors report that several clin. kinase inhibitors also inhibit bromodomains with therapeutically relevant potencies and are best classified as dual kinase-bromodomain inhibitors.  Nanomolar activity on BRD4 by BI-2536 and TG-101348, which are clin. PLK1 and JAK2-FLT3 kinase inhibitors, resp., is particularly noteworthy as these combinations of activities on independent oncogenic pathways exemplify a new strategy for rational single-agent polypharmacol. targeting.  Furthermore, structure-activity relationships and co-crystal structures identify design features that enable a general platform for the rational design of dual kinase-bromodomain inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotPedDED4FJLVg90H21EOLACvtfcHk0lhA9TxEjnrWyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjtl2itro%253D&md5=a00117db2648ea38dd03750faf3b3cb9</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1471&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1471%26sid%3Dliteratum%253Aachs%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DO%25E2%2580%2599Mahony%26aufirst%3DA.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DHunt%26aufirst%3DJ.%2BP.%26aulast%3DLasater%26aufirst%3DE.%2BA.%26aulast%3DPallares%26aufirst%3DG.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DShah%26aufirst%3DN.%2BP.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DDual%2520kinase-bromodomain%2520inhibitors%2520for%2520rationally%2520designed%2520polypharmacology%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3D305%26epage%3D312" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group">Martin, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olesen, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Georg, G. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schonbrunn, E.</span><span> </span><span class="NLM_article-title">Cyclin-dependent kinase inhibitor dinaciclib interacts with the acetyl-lysine recognition site of bromodomains</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">2360</span><span class="NLM_x">–</span> <span class="NLM_lpage">2365</span></span><div class="citationLinks">[<a href="/doi/10.1021/cb4003283" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtl2rtbvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=2360-2365&author=M.+P.+Martinauthor=S.+H.+Olesenauthor=G.+I.+Georgauthor=E.+Schonbrunn&title=Cyclin-dependent+kinase+inhibitor+dinaciclib+interacts+with+the+acetyl-lysine+recognition+site+of+bromodomains"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclin-Dependent Kinase Inhibitor Dinaciclib Interacts with the Acetyl-Lysine Recognition Site of Bromodomains</span></div><div class="casAuthors">Martin, Mathew P.; Olesen, Sanne H.; Georg, Gunda I.; Schonbrunn, Ernst</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2360-2365</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomain-contg. proteins are considered atypical kinases, but their potential to interact with kinase inhibitors is unknown.  Dinaciclib is a potent inhibitor of cyclin-dependent kinases (CDKs), which recently advanced to Phase III clin. trials for the treatment of leukemia.  The authors detd. the crystal structure of dinaciclib in complex with CDK2 at 1.7 Å resoln., revealing an elaborate network of binding interactions in the ATP site, which explains the extraordinary potency and selectivity of this inhibitor.  Remarkably, dinaciclib also interacted with the acetyl-lysine recognition site of the bromodomain testis-specific protein BRDT, a member of the BET family of bromodomains.  The binding mode of dinaciclib to BRDT at 2.0 Å resoln. suggests that general kinase inhibitors (hinge binders) possess a previously unrecognized potential to act as protein-protein inhibitors of bromodomains.  The findings may provide a new structural framework for the design of next-generation bromodomain inhibitors using the vast chem. space of kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonrxl9QUxkE7Vg90H21EOLACvtfcHk0lhA9TxEjnrWyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtl2rtbvN&md5=1ae089e8e571ad11aadcdc4a1a5769b5</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Fcb4003283&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb4003283%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DM.%2BP.%26aulast%3DOlesen%26aufirst%3DS.%2BH.%26aulast%3DGeorg%26aufirst%3DG.%2BI.%26aulast%3DSchonbrunn%26aufirst%3DE.%26atitle%3DCyclin-dependent%2520kinase%2520inhibitor%2520dinaciclib%2520interacts%2520with%2520the%2520acetyl-lysine%2520recognition%2520site%2520of%2520bromodomains%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2013%26volume%3D8%26spage%3D2360%26epage%3D2365" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group">Rosemeyer, H.</span><span> </span><span class="NLM_article-title">The chemodiversity of purine as a constituent of natural products</span> <span class="citation_source-journal">Chem. Biodiversity</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">361</span><span class="NLM_x">–</span> <span class="NLM_lpage">401</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Fjm501893k&amp;key=10.1002%2Fcbdv.200490033" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Fjm501893k&amp;key=17191854" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjtlOht7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2004&pages=361-401&author=H.+Rosemeyer&title=The+chemodiversity+of+purine+as+a+constituent+of+natural+products"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">The chemodiversity of purine as a constituent of natural products</span></div><div class="casAuthors">Rosemeyer, Helmut</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biodiversity</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">361-401</span>CODEN:
                <span class="NLM_cas:coden">CBHIAM</span>;
        ISSN:<span class="NLM_cas:issn">1612-1872</span>.
    
            (<span class="NLM_cas:orgname">Verlag Helvetica Chimica Acta AG</span>)
        </div><div class="casAbstract">In this review, various natural manifestations of purine systems, i.e., methylated, higher-alkylated, and glycosylated forms, are described.  These comprise the purine alkaloids, cytokinines, as well as the purine nucleoside antibiotics.  In part, the compds. described herein were isolated from natural sources already long ago.  However, some have been reported only during the last few years.  The biol. activities of most of the purine derivs. are briefly described, and, in some cases, syntheses are formulated.  In particular, this article introduces the main synthetic principles for the generation of the purine ring system.  The last chapter describes modern preparative routes for C-alkylation and C-arylation of purines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppvY0HCBfJirVg90H21EOLACvtfcHk0lhA9TxEjnrWyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjtlOht7k%253D&md5=5653bce55e74efb0a8b797514d33b862</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1002%2Fcbdv.200490033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbdv.200490033%26sid%3Dliteratum%253Aachs%26aulast%3DRosemeyer%26aufirst%3DH.%26atitle%3DThe%2520chemodiversity%2520of%2520purine%2520as%2520a%2520constituent%2520of%2520natural%2520products%26jtitle%3DChem.%2520Biodiversity%26date%3D2004%26volume%3D1%26spage%3D361%26epage%3D401" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span>U.S. Food and Drug Administration. <a href="http://www.fda.gov/" class="extLink">http://www.fda.gov/</a> (accessed November 24,<span class="NLM_x"> </span><span class="NLM_year">2014</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=U.S.+Food+and+Drug+Administration.+http%3A%2F%2Fwww.fda.gov%2F+%28accessed+November+24%2C+2014%29+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group">Gray, N. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wodicka, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thunnissen, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Norman, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kwon, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Espinoza, F. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morgan, D. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barnes, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">LeClerc, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meijer, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lockhart, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schultz, P. G.</span><span> </span><span class="NLM_article-title">Exploiting chemical libraries, structure, and genomics in the search for kinase inhibitors</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">281</span><span class="NLM_x">, </span> <span class="NLM_fpage">533</span><span class="NLM_x">–</span> <span class="NLM_lpage">538</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Fjm501893k&amp;key=10.1126%2Fscience.281.5376.533" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Fjm501893k&amp;key=9677190" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADyaK1cXkvFCltr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=281&publication_year=1998&pages=533-538&author=N.+S.+Grayauthor=L.+Wodickaauthor=A.+M.+Thunnissenauthor=T.+C.+Normanauthor=S.+Kwonauthor=F.+H.+Espinozaauthor=D.+O.+Morganauthor=G.+Barnesauthor=S.+LeClercauthor=L.+Meijerauthor=S.+H.+Kimauthor=D.+J.+Lockhartauthor=P.+G.+Schultz&title=Exploiting+chemical+libraries%2C+structure%2C+and+genomics+in+the+search+for+kinase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Exploiting chemical libraries, structure, and genomics in the search for kinase inhibitors</span></div><div class="casAuthors">Gray, Nathanael S.; Wodicka, Lisa; Thunnissen, Andy-Mark W. H.; Norman, Thea C.; Kwon, Soojin; Espinoza, F. Hernan; Morgan, David O.; Barnes, Georjana; LeClerc, Sophie; Meijer, Laurent; Kim, Sung-Hou; Lockhart, David J.; Schultz, Peter G.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">281</span>
        (<span class="NLM_cas:issue">5376</span>),
    <span class="NLM_cas:pages">533-538</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Selective protein kinase inhibitors were developed on the basis of the unexpected binding mode of 2,6,9-trisubstituted purines to the ATP-binding site of the human cyclin-dependent kinase 2 (CDK2).  By iterating chem. library synthesis and biol. screening, potent inhibitors of the human CDK2-cyclin A kinase complex and of Saccharomyces cerevisiae Cdc28p were identified.  The structural basis for the binding affinity and selectivity was detd. by anal. of a three-dimensional crystal structure of a CDK2-inhibitor complex.  The cellular effects of these compds. were characterized in mammalian cells and yeast.  In the latter case the effects were characterized on a genome-wide scale by monitoring changes in mRNA levels in treated cells with high-d. oligonucleotide probe arrays.  Purine libraries could provide useful tools for analyzing a variety of signaling and regulatory pathways and may led to the development of new therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRTOOPjk16tbVg90H21EOLACvtfcHk0lhNxxFzvmCEMg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXkvFCltr8%253D&md5=08b8debe4d05c1169c152ccd6676def8</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1126%2Fscience.281.5376.533&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.281.5376.533%26sid%3Dliteratum%253Aachs%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DWodicka%26aufirst%3DL.%26aulast%3DThunnissen%26aufirst%3DA.%2BM.%26aulast%3DNorman%26aufirst%3DT.%2BC.%26aulast%3DKwon%26aufirst%3DS.%26aulast%3DEspinoza%26aufirst%3DF.%2BH.%26aulast%3DMorgan%26aufirst%3DD.%2BO.%26aulast%3DBarnes%26aufirst%3DG.%26aulast%3DLeClerc%26aufirst%3DS.%26aulast%3DMeijer%26aufirst%3DL.%26aulast%3DKim%26aufirst%3DS.%2BH.%26aulast%3DLockhart%26aufirst%3DD.%2BJ.%26aulast%3DSchultz%26aufirst%3DP.%2BG.%26atitle%3DExploiting%2520chemical%2520libraries%252C%2520structure%252C%2520and%2520genomics%2520in%2520the%2520search%2520for%2520kinase%2520inhibitors%26jtitle%3DScience%26date%3D1998%26volume%3D281%26spage%3D533%26epage%3D538" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group">Legraverend, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grierson, D. S.</span><span> </span><span class="NLM_article-title">The purines: potent and versatile small molecule inhibitors and modulators of key biological targets</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">3987</span><span class="NLM_x">–</span> <span class="NLM_lpage">4006</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2006&pages=3987-4006&author=M.+Legraverendauthor=D.+S.+Grierson&title=The+purines%3A+potent+and+versatile+small+molecule+inhibitors+and+modulators+of+key+biological+targets"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLegraverend%26aufirst%3DM.%26aulast%3DGrierson%26aufirst%3DD.%2BS.%26atitle%3DThe%2520purines%253A%2520potent%2520and%2520versatile%2520small%2520molecule%2520inhibitors%2520and%2520modulators%2520of%2520key%2520biological%2520targets%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2006%26volume%3D14%26spage%3D3987%26epage%3D4006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group">Kadoch, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hargreaves, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hodges, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elias, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ho, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ranish, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crabtree, G. R.</span><span> </span><span class="NLM_article-title">Proteomic and bioinformatic analysis of mammalian SWI/SNF complexes identifies extensive roles in human malignancy</span> <span class="citation_source-journal">Nat. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">592</span><span class="NLM_x">–</span> <span class="NLM_lpage">601</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Fjm501893k&amp;key=10.1038%2Fng.2628" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Fjm501893k&amp;key=23644491" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BC3sXntVart7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2013&pages=592-601&author=C.+Kadochauthor=D.+C.+Hargreavesauthor=C.+Hodgesauthor=L.+Eliasauthor=L.+Hoauthor=J.+Ranishauthor=G.+R.+Crabtree&title=Proteomic+and+bioinformatic+analysis+of+mammalian+SWI%2FSNF+complexes+identifies+extensive+roles+in+human+malignancy"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Proteomic and bioinformatic analysis of mammalian SWI/SNF complexes identifies extensive roles in human malignancy</span></div><div class="casAuthors">Kadoch, Cigall; Hargreaves, Diana C.; Hodges, Courtney; Elias, Laura; Ho, Lena; Ranish, Jeff; Crabtree, Gerald R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">592-601</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Subunits of mammalian SWI/SNF (mSWI/SNF or BAF) complexes have recently been implicated as tumor suppressors in human malignancies.  To understand the full extent of their involvement, we conducted a proteomic anal. of endogenous mSWI/SNF complexes, which identified several new dedicated, stable subunits not found in yeast SWI/SNF complexes, including BCL7A, BCL7B and BCL7C, BCL11A and BCL11B, BRD9 and SS18.  Incorporating these new members, we detd. mSWI/SNF subunit mutation frequency in exome and whole-genome sequencing studies of primary human tumors.  Notably, mSWI/SNF subunits are mutated in 19.6% of all human tumors reported in 44 studies.  Our anal. suggests that specific subunits protect against cancer in specific tissues.  In addn., mutations affecting more than one subunit, defined here as compd. heterozygosity, are prevalent in certain cancers.  Our studies demonstrate that mSWI/SNF is the most frequently mutated chromatin-regulatory complex (CRC) in human cancer, exhibiting a broad mutation pattern, similar to that of TP53.  Thus, proper functioning of polymorphic BAF complexes may constitute a major mechanism of tumor suppression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqP873DCpn7JrVg90H21EOLACvtfcHk0lhNxxFzvmCEMg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXntVart7s%253D&md5=0f49cf3c59e7bc04b74970fe656b1e17</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1038%2Fng.2628&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fng.2628%26sid%3Dliteratum%253Aachs%26aulast%3DKadoch%26aufirst%3DC.%26aulast%3DHargreaves%26aufirst%3DD.%2BC.%26aulast%3DHodges%26aufirst%3DC.%26aulast%3DElias%26aufirst%3DL.%26aulast%3DHo%26aufirst%3DL.%26aulast%3DRanish%26aufirst%3DJ.%26aulast%3DCrabtree%26aufirst%3DG.%2BR.%26atitle%3DProteomic%2520and%2520bioinformatic%2520analysis%2520of%2520mammalian%2520SWI%252FSNF%2520complexes%2520identifies%2520extensive%2520roles%2520in%2520human%2520malignancy%26jtitle%3DNat.%2520Genet.%26date%3D2013%26volume%3D45%26spage%3D592%26epage%3D601" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group">Kang, J. U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koo, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kwon, K. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, J. M.</span><span> </span><span class="NLM_article-title">Gain at chromosomal region 5p15.33, containing TERT, is the most frequent genetic event in early stages of non-small cell lung cancer</span> <span class="citation_source-journal">Cancer Genet. Cytogenet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">182</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">11</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Fjm501893k&amp;key=10.1016%2Fj.cancergencyto.2007.12.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Fjm501893k&amp;key=18328944" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjtVWmt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=182&publication_year=2008&pages=1-11&author=J.+U.+Kangauthor=S.+H.+Kooauthor=K.+C.+Kwonauthor=J.+W.+Parkauthor=J.+M.+Kim&title=Gain+at+chromosomal+region+5p15.33%2C+containing+TERT%2C+is+the+most+frequent+genetic+event+in+early+stages+of+non-small+cell+lung+cancer"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Gain at chromosomal region 5p15.33, containing TERT, is the most frequent genetic event in early stages of non-small cell lung cancer</span></div><div class="casAuthors">Kang, Ji Un; Koo, Sun Hoe; Kwon, Kye Chul; Park, Jong Woo; Kim, Jin Man</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Genetics and Cytogenetics</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">182</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-11</span>CODEN:
                <span class="NLM_cas:coden">CGCYDF</span>;
        ISSN:<span class="NLM_cas:issn">0165-4608</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Chromosomal imbalances resulting in altered gene dosage play a role in the mol. pathogenesis of non-small cell lung cancer (NSCLC), but the target genes remain to be identified.  To identify early-stage genetic events that drive progression of NSCLC, we conducted a high-resoln. array comparative genomic hybridization (CGH) study, using an array of 4046 bacterial artificial chromosome clones to screen for DNA copy no. changes assocd. with individual genes in 36 tumors obtained from patients in early stages of NSCLC.  Multiple early genetic events occurring on chromosome 5p were identified, with a minimal detection region at 5p15.33∼12.  The most frequent finding involved gain of 5p15.33, obsd. in 15 of 19 stage I (A+B) cancers (79%) and in 28 of the total 36 NSCLC cases (78%).  This locus harbors the genes TERT, SLC6A19, and SLC6A18 and is a telomeric boundary at bacterial artificial chromosome (BAC) clone 91_J20.  Other potential candidate genes evidencing high nos. of genomic copy no. changes (≥40% of patients) included the following genes, encountered in >50% of 19 stage I (A+B) cancers: CEP72 and TPPP (14 of 19; 74%); AHRR, EXOC3 (previously SEC6L1), SLC9A3, LOC442126, ZDHHC11, BRD9, and TRIP13 (13/19; 68%); and CLPTM1L (alias CRR9), SLC6A3 (previously DAT1), and LOC401169 (10/19; 53%).  Fluorescence in situ hybridization validated the array CGH findings.  The gain of 5p15.33 is thus one of the most consistent alterations in the early stages of lung cancer, and a series of genes in the crit. 5p15.33 region may be used as novel biomarkers for the early detection and classification of lung cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvKwLSWMQRnbVg90H21EOLACvtfcHk0lhNxxFzvmCEMg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjtVWmt78%253D&md5=514aec6952253ed612003471df40a4ba</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1016%2Fj.cancergencyto.2007.12.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cancergencyto.2007.12.004%26sid%3Dliteratum%253Aachs%26aulast%3DKang%26aufirst%3DJ.%2BU.%26aulast%3DKoo%26aufirst%3DS.%2BH.%26aulast%3DKwon%26aufirst%3DK.%2BC.%26aulast%3DPark%26aufirst%3DJ.%2BW.%26aulast%3DKim%26aufirst%3DJ.%2BM.%26atitle%3DGain%2520at%2520chromosomal%2520region%25205p15.33%252C%2520containing%2520TERT%252C%2520is%2520the%2520most%2520frequent%2520genetic%2520event%2520in%2520early%2520stages%2520of%2520non-small%2520cell%2520lung%2520cancer%26jtitle%3DCancer%2520Genet.%2520Cytogenet.%26date%3D2008%26volume%3D182%26spage%3D1%26epage%3D11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group">Scotto, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Narayan, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nandula, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Subramaniyam, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaufmann, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wright, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pothuri, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mansukhani, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schneider, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arias-Pulido, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murty, V. V.</span><span> </span><span class="NLM_article-title">Integrative genomics analysis of chromosome 5p gain in cervical cancer reveals target over-expressed genes, including Drosha</span> <span class="citation_source-journal">Mol. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">58</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Fjm501893k&amp;key=10.1186%2F1476-4598-7-58" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Fjm501893k&amp;key=18559093" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A280%3ADC%252BD1cvgvFGktw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2008&pages=58&author=L.+Scottoauthor=G.+Narayanauthor=S.+V.+Nandulaauthor=S.+Subramaniyamauthor=A.+M.+Kaufmannauthor=J.+D.+Wrightauthor=B.+Pothuriauthor=M.+Mansukhaniauthor=A.+Schneiderauthor=H.+Arias-Pulidoauthor=V.+V.+Murty&title=Integrative+genomics+analysis+of+chromosome+5p+gain+in+cervical+cancer+reveals+target+over-expressed+genes%2C+including+Drosha"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Integrative genomics analysis of chromosome 5p gain in cervical cancer reveals target over-expressed genes, including Drosha</span></div><div class="casAuthors">Scotto Luigi; Narayan Gopeshwar; Nandula Subhadra V; Subramaniyam Shivakumar; Kaufmann Andreas M; Wright Jason D; Pothuri Bhavana; Mansukhani Mahesh; Schneider Achim; Arias-Pulido Hugo; Murty Vundavalli V</div><div class="citationInfo"><span class="NLM_cas:title">Molecular cancer</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">58</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Copy number gains and amplifications are characteristic feature of cervical cancer (CC) genomes for which the underlying mechanisms are unclear.  These changes may possess oncogenic properties by deregulating tumor-related genes.  Gain of short arm of chromosome 5 (5p) is the most frequent karyotypic change in CC.  METHODS:  To examine the role of 5p gain, we performed a combination of single nucleotide polymorphism (SNP) array, fluorescence in situ hybridization (FISH), and gene expression analyses on invasive cancer and in various stages of CC progression.  RESULTS:  The SNP and FISH analyses revealed copy number increase (CNI) of 5p in 63% of invasive CC, which arises at later stages of precancerous lesions in CC development.  We integrated chromosome 5 genomic copy number and gene expression data to identify key target over expressed genes as a consequence of 5p gain.  One of the candidates identified was Drosha (RNASEN), a gene that is required in the first step of microRNA (miRNA) processing in the nucleus.  Other 5p genes identified as targets of CNI play a role in DNA repair and cell cycle regulation (BASP1, TARS, PAIP1, BRD9, RAD1, SKP2, and POLS), signal transduction (OSMR), and mitochondrial oxidative phosphorylation (NNT, SDHA, and NDUFS6), suggesting that disruption of pathways involving these genes may contribute to CC progression.  CONCLUSION:  Taken together, we demonstrate the power of integrating genomics data with expression data in deciphering tumor-related targets of CNI.  Identification of 5p gene targets in CC denotes an important step towards biomarker development and forms a framework for testing as molecular therapeutic targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTJ-og-xr-IWuxsqfvU0ziafW6udTcc2eZKr9RL-ehFe7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1cvgvFGktw%253D%253D&md5=9bb25d18ffeaebf79760b3b760c91e15</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1186%2F1476-4598-7-58&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1476-4598-7-58%26sid%3Dliteratum%253Aachs%26aulast%3DScotto%26aufirst%3DL.%26aulast%3DNarayan%26aufirst%3DG.%26aulast%3DNandula%26aufirst%3DS.%2BV.%26aulast%3DSubramaniyam%26aufirst%3DS.%26aulast%3DKaufmann%26aufirst%3DA.%2BM.%26aulast%3DWright%26aufirst%3DJ.%2BD.%26aulast%3DPothuri%26aufirst%3DB.%26aulast%3DMansukhani%26aufirst%3DM.%26aulast%3DSchneider%26aufirst%3DA.%26aulast%3DArias-Pulido%26aufirst%3DH.%26aulast%3DMurty%26aufirst%3DV.%2BV.%26atitle%3DIntegrative%2520genomics%2520analysis%2520of%2520chromosome%25205p%2520gain%2520in%2520cervical%2520cancer%2520reveals%2520target%2520over-expressed%2520genes%252C%2520including%2520Drosha%26jtitle%3DMol.%2520Cancer%26date%3D2008%26volume%3D7%26spage%3D58" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group">Cleary, S. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeck, W. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Selitsky, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Savich, G. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tan, T. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Getz, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lawrence, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parker, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Powers, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fischer, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guindi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ghanekar, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chiang, D. Y.</span><span> </span><span class="NLM_article-title">Identification of driver genes in hepatocellular carcinoma by exome sequencing</span> <span class="citation_source-journal">Hepatology</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">1693</span><span class="NLM_x">–</span> <span class="NLM_lpage">1702</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2013&pages=1693-1702&author=S.+P.+Clearyauthor=W.+R.+Jeckauthor=X.+Zhaoauthor=K.+Chenauthor=S.+R.+Selitskyauthor=G.+L.+Savichauthor=T.+X.+Tanauthor=M.+C.+Wuauthor=G.+Getzauthor=M.+S.+Lawrenceauthor=J.+S.+Parkerauthor=J.+Liauthor=S.+Powersauthor=H.+Kimauthor=S.+Fischerauthor=M.+Guindiauthor=A.+Ghanekarauthor=D.+Y.+Chiang&title=Identification+of+driver+genes+in+hepatocellular+carcinoma+by+exome+sequencing"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCleary%26aufirst%3DS.%2BP.%26aulast%3DJeck%26aufirst%3DW.%2BR.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DSelitsky%26aufirst%3DS.%2BR.%26aulast%3DSavich%26aufirst%3DG.%2BL.%26aulast%3DTan%26aufirst%3DT.%2BX.%26aulast%3DWu%26aufirst%3DM.%2BC.%26aulast%3DGetz%26aufirst%3DG.%26aulast%3DLawrence%26aufirst%3DM.%2BS.%26aulast%3DParker%26aufirst%3DJ.%2BS.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DPowers%26aufirst%3DS.%26aulast%3DKim%26aufirst%3DH.%26aulast%3DFischer%26aufirst%3DS.%26aulast%3DGuindi%26aufirst%3DM.%26aulast%3DGhanekar%26aufirst%3DA.%26aulast%3DChiang%26aufirst%3DD.%2BY.%26atitle%3DIdentification%2520of%2520driver%2520genes%2520in%2520hepatocellular%2520carcinoma%2520by%2520exome%2520sequencing%26jtitle%3DHepatology%26date%3D2013%26volume%3D58%26spage%3D1693%26epage%3D1702" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group">The Cancer Genome Atlas Research Network</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kandoth, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schultz, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cherniack, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Akbani, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robertson, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pashtan, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benz, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yau, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Laird, P. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ding, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mills, G. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kucherlapati, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mardis, E. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levine, D. A.</span><span> </span><span class="NLM_article-title">Integrated genomic characterization of endometrial carcinoma</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">497</span><span class="NLM_x">, </span> <span class="NLM_fpage">67</span><span class="NLM_x">–</span> <span class="NLM_lpage">73</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=497&publication_year=2013&pages=67-73&author=The+Cancer+Genome+Atlas+Research+Networkauthor=C.+Kandothauthor=N.+Schultzauthor=A.+D.+Cherniackauthor=R.+Akbaniauthor=Y.+Liuauthor=H.+Shenauthor=A.+G.+Robertsonauthor=I.+Pashtanauthor=R.+Shenauthor=C.+C.+Benzauthor=C.+Yauauthor=P.+W.+Lairdauthor=L.+Dingauthor=W.+Zhangauthor=G.+B.+Millsauthor=R.+Kucherlapatiauthor=E.+R.+Mardisauthor=D.+A.+Levine&title=Integrated+genomic+characterization+of+endometrial+carcinoma"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3D%26aulast%3DKandoth%26aufirst%3DC.%26aulast%3DSchultz%26aufirst%3DN.%26aulast%3DCherniack%26aufirst%3DA.%2BD.%26aulast%3DAkbani%26aufirst%3DR.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DShen%26aufirst%3DH.%26aulast%3DRobertson%26aufirst%3DA.%2BG.%26aulast%3DPashtan%26aufirst%3DI.%26aulast%3DShen%26aufirst%3DR.%26aulast%3DBenz%26aufirst%3DC.%2BC.%26aulast%3DYau%26aufirst%3DC.%26aulast%3DLaird%26aufirst%3DP.%2BW.%26aulast%3DDing%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DMills%26aufirst%3DG.%2BB.%26aulast%3DKucherlapati%26aufirst%3DR.%26aulast%3DMardis%26aufirst%3DE.%2BR.%26aulast%3DLevine%26aufirst%3DD.%2BA.%26atitle%3DIntegrated%2520genomic%2520characterization%2520of%2520endometrial%2520carcinoma%26jtitle%3DNature%26date%3D2013%26volume%3D497%26spage%3D67%26epage%3D73" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group">The Cancer Genome Atlas Research Network.</span><span> </span><span class="NLM_article-title">Comprehensive genomic characterization of squamous cell lung cancers</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">489</span><span class="NLM_x">, </span> <span class="NLM_fpage">519</span><span class="NLM_x">–</span> <span class="NLM_lpage">525</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=489&publication_year=2012&pages=519-525&author=The+Cancer+Genome+Atlas+Research+Network.&title=Comprehensive+genomic+characterization+of+squamous+cell+lung+cancers"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DComprehensive%2520genomic%2520characterization%2520of%2520squamous%2520cell%2520lung%2520cancers%26jtitle%3DNature%26date%3D2012%26volume%3D489%26spage%3D519%26epage%3D525" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group">Barbieri, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baca, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lawrence, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Demichelis, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blattner, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Theurillat, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">White, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stojanov, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Allen, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stransky, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nickerson, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chae, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boysen, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Auclair, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Onofrio, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kitabayashi, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacDonald, T. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sheikh, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vuong, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guiducci, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cibulskis, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sivachenko, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carter, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saksena, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Voet, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hussain, W. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramos, A. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winckler, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Redman, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ardlie, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tewari, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mosquera, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rupp, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wild, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moch, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morrissey, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nelson, P. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kantoff, P. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gabriel, S. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Golub, T. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meyerson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lander, E. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Getz, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rubin, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garraway, L. A.</span><span> </span><span class="NLM_article-title">Exome sequencing identifies recurrent SPOP, FOXA1 and MED12 mutations in prostate cancer</span> <span class="citation_source-journal">Nat. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">685</span><span class="NLM_x">–</span> <span class="NLM_lpage">689</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Fjm501893k&amp;key=10.1038%2Fng.2279" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Fjm501893k&amp;key=22610119" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BC38XntF2nt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2012&pages=685-689&author=C.+E.+Barbieriauthor=S.+C.+Bacaauthor=M.+S.+Lawrenceauthor=F.+Demichelisauthor=M.+Blattnerauthor=J.+P.+Theurillatauthor=T.+A.+Whiteauthor=P.+Stojanovauthor=E.+Van+Allenauthor=N.+Stranskyauthor=E.+Nickersonauthor=S.+S.+Chaeauthor=G.+Boysenauthor=D.+Auclairauthor=R.+C.+Onofrioauthor=K.+Parkauthor=N.+Kitabayashiauthor=T.+Y.+MacDonaldauthor=K.+Sheikhauthor=T.+Vuongauthor=C.+Guiducciauthor=K.+Cibulskisauthor=A.+Sivachenkoauthor=S.+L.+Carterauthor=G.+Saksenaauthor=D.+Voetauthor=W.+M.+Hussainauthor=A.+H.+Ramosauthor=W.+Wincklerauthor=M.+C.+Redmanauthor=K.+Ardlieauthor=A.+K.+Tewariauthor=J.+M.+Mosqueraauthor=N.+Ruppauthor=P.+J.+Wildauthor=H.+Mochauthor=C.+Morrisseyauthor=P.+S.+Nelsonauthor=P.+W.+Kantoffauthor=S.+B.+Gabrielauthor=T.+R.+Golubauthor=M.+Meyersonauthor=E.+S.+Landerauthor=G.+Getzauthor=M.+A.+Rubinauthor=L.+A.+Garraway&title=Exome+sequencing+identifies+recurrent+SPOP%2C+FOXA1+and+MED12+mutations+in+prostate+cancer"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Exome sequencing identifies recurrent SPOP, FOXA1 and MED12 mutations in prostate cancer</span></div><div class="casAuthors">Barbieri, Christopher E.; Baca, Sylvan C.; Lawrence, Michael S.; Demichelis, Francesca; Blattner, Mirjam; Theurillat, Jean-Philippe; White, Thomas A.; Stojanov, Petar; Van Allen, Eliezer; Stransky, Nicolas; Nickerson, Elizabeth; Chae, Sung-Suk; Boysen, Gunther; Auclair, Daniel; Onofrio, Robert C.; Park, Kyung; Kitabayashi, Naoki; MacDonald, Theresa Y.; Sheikh, Karen; Vuong, Terry; Guiducci, Candace; Cibulskis, Kristian; Sivachenko, Andrey; Carter, Scott L.; Saksena, Gordon; Voet, Douglas; Hussain, Wasay M.; Ramos, Alex H.; Winckler, Wendy; Redman, Michelle C.; Ardlie, Kristin; Tewari, Ashutosh K.; Mosquera, Juan Miguel; Rupp, Niels; Wild, Peter J.; Moch, Holger; Morrissey, Colm; Nelson, Peter S.; Kantoff, Philip W.; Gabriel, Stacey B.; Golub, Todd R.; Meyerson, Matthew; Lander, Eric S.; Getz, Gad; Rubin, Mark A.; Garraway, Levi A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">685-689</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Prostate cancer is the second most common cancer in men worldwide and causes over 250,000 deaths each year.  Overtreatment of indolent disease also results in significant morbidity.  Common genetic alterations in prostate cancer include losses of NKX3.1 (8p21) and PTEN (10q23), gains of AR (the androgen receptor gene) and fusion of ETS family transcription factor genes with androgen-responsive promoters.  Recurrent somatic base-pair substitutions are believed to be less contributory in prostate tumorigenesis but have not been systematically analyzed in large cohorts.  Here, we sequenced the exomes of 112 prostate tumor and normal tissue pairs.  New recurrent mutations were identified in multiple genes, including MED12 and FOXA1.  SPOP was the most frequently mutated gene, with mutation.s involving the SPOP substrate-binding cleft in 6-15% of tumors across multiple independent cohorts.  Prostate cancers with mutant SPOP lacked ETS family gene rearrangements and showed a distinct pattern of genomic alterations.  Thus, SPOP mutations may define a new mol. subtype of prostate cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6_0sm909vLbVg90H21EOLACvtfcHk0ljt2YAfQ4vmag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XntF2nt7w%253D&md5=b27e8ce7bdfa0cd142546dab6b521f6b</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1038%2Fng.2279&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fng.2279%26sid%3Dliteratum%253Aachs%26aulast%3DBarbieri%26aufirst%3DC.%2BE.%26aulast%3DBaca%26aufirst%3DS.%2BC.%26aulast%3DLawrence%26aufirst%3DM.%2BS.%26aulast%3DDemichelis%26aufirst%3DF.%26aulast%3DBlattner%26aufirst%3DM.%26aulast%3DTheurillat%26aufirst%3DJ.%2BP.%26aulast%3DWhite%26aufirst%3DT.%2BA.%26aulast%3DStojanov%26aufirst%3DP.%26aulast%3DVan%2BAllen%26aufirst%3DE.%26aulast%3DStransky%26aufirst%3DN.%26aulast%3DNickerson%26aufirst%3DE.%26aulast%3DChae%26aufirst%3DS.%2BS.%26aulast%3DBoysen%26aufirst%3DG.%26aulast%3DAuclair%26aufirst%3DD.%26aulast%3DOnofrio%26aufirst%3DR.%2BC.%26aulast%3DPark%26aufirst%3DK.%26aulast%3DKitabayashi%26aufirst%3DN.%26aulast%3DMacDonald%26aufirst%3DT.%2BY.%26aulast%3DSheikh%26aufirst%3DK.%26aulast%3DVuong%26aufirst%3DT.%26aulast%3DGuiducci%26aufirst%3DC.%26aulast%3DCibulskis%26aufirst%3DK.%26aulast%3DSivachenko%26aufirst%3DA.%26aulast%3DCarter%26aufirst%3DS.%2BL.%26aulast%3DSaksena%26aufirst%3DG.%26aulast%3DVoet%26aufirst%3DD.%26aulast%3DHussain%26aufirst%3DW.%2BM.%26aulast%3DRamos%26aufirst%3DA.%2BH.%26aulast%3DWinckler%26aufirst%3DW.%26aulast%3DRedman%26aufirst%3DM.%2BC.%26aulast%3DArdlie%26aufirst%3DK.%26aulast%3DTewari%26aufirst%3DA.%2BK.%26aulast%3DMosquera%26aufirst%3DJ.%2BM.%26aulast%3DRupp%26aufirst%3DN.%26aulast%3DWild%26aufirst%3DP.%2BJ.%26aulast%3DMoch%26aufirst%3DH.%26aulast%3DMorrissey%26aufirst%3DC.%26aulast%3DNelson%26aufirst%3DP.%2BS.%26aulast%3DKantoff%26aufirst%3DP.%2BW.%26aulast%3DGabriel%26aufirst%3DS.%2BB.%26aulast%3DGolub%26aufirst%3DT.%2BR.%26aulast%3DMeyerson%26aufirst%3DM.%26aulast%3DLander%26aufirst%3DE.%2BS.%26aulast%3DGetz%26aufirst%3DG.%26aulast%3DRubin%26aufirst%3DM.%2BA.%26aulast%3DGarraway%26aufirst%3DL.%2BA.%26atitle%3DExome%2520sequencing%2520identifies%2520recurrent%2520SPOP%252C%2520FOXA1%2520and%2520MED12%2520mutations%2520in%2520prostate%2520cancer%26jtitle%3DNat.%2520Genet.%26date%3D2012%26volume%3D44%26spage%3D685%26epage%3D689" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group">Bosshard, H. R.</span><span> </span><span class="NLM_article-title">Molecular recognition by induced fit: How fit is the concept?</span> <span class="citation_source-journal">News Physiol. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">171</span><span class="NLM_x">–</span> <span class="NLM_lpage">173</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Fjm501893k&amp;key=11479367" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BD3MXntVahs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2001&pages=171-173&author=H.+R.+Bosshard&title=Molecular+recognition+by+induced+fit%3A+How+fit+is+the+concept%3F"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular recognition by induced fit: how fit is the concept?</span></div><div class="casAuthors">Bosshard, Hans Rudolf</div><div class="citationInfo"><span class="NLM_cas:title">News in Physiological Sciences</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">Aug.</span>),
    <span class="NLM_cas:pages">171-174</span>CODEN:
                <span class="NLM_cas:coden">NEPSEY</span>;
        ISSN:<span class="NLM_cas:issn">0886-1714</span>.
    
            (<span class="NLM_cas:orgname">International Union of Physiological Sciences</span>)
        </div><div class="casAbstract">A review.  Induced fit explains why biomols. can bind together even if they are not optimized for binding.  However, induced fit can lead to a kinetic bottleneck and does not describe every interaction in the absence of prior complementarity.  Preselection of a fitting conformer is an alternative to induced fit.  Topics discussed include the expt. demonstration of binding by conformational selection and the energy landscape model of protein conformation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwY5pcaD1H1rVg90H21EOLACvtfcHk0ljt2YAfQ4vmag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXntVahs7k%253D&md5=033a7cd9621a8cec3b481014455e9cef</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBosshard%26aufirst%3DH.%2BR.%26atitle%3DMolecular%2520recognition%2520by%2520induced%2520fit%253A%2520How%2520fit%2520is%2520the%2520concept%253F%26jtitle%3DNews%2520Physiol.%2520Sci.%26date%3D2001%26volume%3D16%26spage%3D171%26epage%3D173" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group">Rooney, T. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Picaud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cortopassi, W. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hay, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tumber, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rogers, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Philpott, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thompson, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heightman, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pryde, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cook, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paton, R. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muller, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brennan, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Conway, S. J.</span><span> </span><span class="NLM_article-title">A series of potent CREBBP bromodomain ligands reveals an induced-fit pocket stabilized by a cation-pi interaction</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">6126</span><span class="NLM_x">–</span> <span class="NLM_lpage">6130</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2014&pages=6126-6130&author=T.+P.+Rooneyauthor=P.+Filippakopoulosauthor=O.+Fedorovauthor=S.+Picaudauthor=W.+A.+Cortopassiauthor=D.+A.+Hayauthor=S.+Martinauthor=A.+Tumberauthor=C.+M.+Rogersauthor=M.+Philpottauthor=M.+Wangauthor=A.+L.+Thompsonauthor=T.+D.+Heightmanauthor=D.+C.+Prydeauthor=A.+Cookauthor=R.+S.+Patonauthor=S.+Mullerauthor=S.+Knappauthor=P.+E.+Brennanauthor=S.+J.+Conway&title=A+series+of+potent+CREBBP+bromodomain+ligands+reveals+an+induced-fit+pocket+stabilized+by+a+cation-pi+interaction"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRooney%26aufirst%3DT.%2BP.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DCortopassi%26aufirst%3DW.%2BA.%26aulast%3DHay%26aufirst%3DD.%2BA.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DTumber%26aufirst%3DA.%26aulast%3DRogers%26aufirst%3DC.%2BM.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DThompson%26aufirst%3DA.%2BL.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DPryde%26aufirst%3DD.%2BC.%26aulast%3DCook%26aufirst%3DA.%26aulast%3DPaton%26aufirst%3DR.%2BS.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DConway%26aufirst%3DS.%2BJ.%26atitle%3DA%2520series%2520of%2520potent%2520CREBBP%2520bromodomain%2520ligands%2520reveals%2520an%2520induced-fit%2520pocket%2520stabilized%2520by%2520a%2520cation-pi%2520interaction%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2014%26volume%3D53%26spage%3D6126%26epage%3D6130" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group">Friesner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banks, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murphy, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Halgren, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klicic, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mainz, D. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Repasky, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knoll, E. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shelley, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perry, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Francis, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shenkin, P. S.</span><span> </span><span class="NLM_article-title">Glide: a new approach for rapid, accurate docking and scoring. 1. Method and assessment of docking accuracy</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">1739</span><span class="NLM_x">–</span> <span class="NLM_lpage">1749</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm0306430" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1739-1749&author=R.+A.+Friesnerauthor=J.+L.+Banksauthor=R.+B.+Murphyauthor=T.+A.+Halgrenauthor=J.+J.+Klicicauthor=D.+T.+Mainzauthor=M.+P.+Repaskyauthor=E.+H.+Knollauthor=M.+Shelleyauthor=J.+K.+Perryauthor=D.+E.+Shawauthor=P.+Francisauthor=P.+S.+Shenkin&title=Glide%3A+a+new+approach+for+rapid%2C+accurate+docking+and+scoring.+1.+Method+and+assessment+of+docking+accuracy"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 1. method and assessment of docking accuracy</span></div><div class="casAuthors">Friesner, Richard A.; Banks, Jay L.; Murphy, Robert B.; Halgren, Thomas A.; Klicic, Jasna J.; Mainz, Daniel T.; Repasky, Matthew P.; Knoll, Eric H.; Shelley, Mee; Perry, Jason K.; Shaw, David E.; Francis, Perry; Shenkin, Peter S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1739-1749</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Unlike other methods for docking ligands to the rigid 3D structure of a known protein receptor, Glide approximates a complete systematic search of the conformational, orientational, and positional space of the docked ligand.  In this search, an initial rough positioning and scoring phase that dramatically narrows the search space is followed by torsionally flexible energy optimization on an OPLS-AA nonbonded potential grid for a few hundred surviving candidate poses.  The very best candidates are further refined via a Monte Carlo sampling of pose conformation; in some cases, this is crucial to obtaining an accurate docked pose.  Selection of the best docked pose uses a model energy function that combines empirical and force-field-based terms.  Docking accuracy is assessed by redocking ligands from 282 cocrystd. PDB complexes starting from conformationally optimized ligand geometries that bear no memory of the correctly docked pose.  Errors in geometry for the top-ranked pose are less than 1 Å in nearly half of the cases and are greater than 2 Å in only about one-third of them.  Comparisons to published data on rms deviations show that Glide is nearly twice as accurate as GOLD and more than twice as accurate as FlexX for ligands having up to 20 rotatable bonds.  Glide is also found to be more accurate than the recently described Surflex method.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPuigu31qMMLVg90H21EOLACvtfcHk0ljGbJ_NRPnAeg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D&md5=8cc2f0022318b12dd972e9c493375bf9</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1021%2Fjm0306430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0306430%26sid%3Dliteratum%253Aachs%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DKlicic%26aufirst%3DJ.%2BJ.%26aulast%3DMainz%26aufirst%3DD.%2BT.%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DKnoll%26aufirst%3DE.%2BH.%26aulast%3DShelley%26aufirst%3DM.%26aulast%3DPerry%26aufirst%3DJ.%2BK.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26aulast%3DFrancis%26aufirst%3DP.%26aulast%3DShenkin%26aufirst%3DP.%2BS.%26atitle%3DGlide%253A%2520a%2520new%2520approach%2520for%2520rapid%252C%2520accurate%2520docking%2520and%2520scoring.%25201.%2520Method%2520and%2520assessment%2520of%2520docking%2520accuracy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1739%26epage%3D1749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group">Halgren, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murphy, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friesner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beard, H. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, L. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollard, W. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banks, J. L.</span><span> </span><span class="NLM_article-title">Glide: a new approach for rapid, accurate docking and scoring. 2. Enrichment factors in database screening</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">1750</span><span class="NLM_x">–</span> <span class="NLM_lpage">1759</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm030644s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1750-1759&author=T.+A.+Halgrenauthor=R.+B.+Murphyauthor=R.+A.+Friesnerauthor=H.+S.+Beardauthor=L.+L.+Fryeauthor=W.+T.+Pollardauthor=J.+L.+Banks&title=Glide%3A+a+new+approach+for+rapid%2C+accurate+docking+and+scoring.+2.+Enrichment+factors+in+database+screening"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 2. Enrichment factors in database screening</span></div><div class="casAuthors">Halgren, Thomas A.; Murphy, Robert B.; Friesner, Richard A.; Beard, Hege S.; Frye, Leah L.; Pollard, W. Thomas; Banks, Jay L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1750-1759</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Glide's ability to identify active compds. in a database screen is characterized by applying Glide to a diverse set of nine protein receptors.  In many cases, two, or even three, protein sites are employed to probe the sensitivity of the results to the site geometry.  To make the database screens as realistic as possible, the screens use sets of "druglike" decoy ligands that have been selected to be representative of what we believe is likely to be found in the compd. collection of a pharmaceutical or biotechnol. company.  Results are presented for releases 1.8, 2.0, and 2.5 of Glide.  The comparisons show that av. measures for both "early" and "global" enrichment for Glide 2.5 are 3 times higher than for Glide 1.8 and more than 2 times higher than for Glide 2.0 because of better results for the least well-handled screens.  This improvement in enrichment stems largely from the better balance of the more widely parametrized GlideScore 2.5 function and the inclusion of terms that penalize ligand-protein interactions that violate established principles of phys. chem., particularly as it concerns the exposure to solvent of charged protein and ligand groups.  Comparisons to results for the thymidine kinase and estrogen receptors published by Rognan and co-workers (J. Med. Chem. 2000, 43, 4759-4767) show that Glide 2.5 performs better than GOLD 1.1, FlexX 1.8, or DOCK 4.01.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoT2ptubbKbvLVg90H21EOLACvtfcHk0ljGbJ_NRPnAeg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D&md5=33d68dd968e65626b449df61e44e37be</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1021%2Fjm030644s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030644s%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBeard%26aufirst%3DH.%2BS.%26aulast%3DFrye%26aufirst%3DL.%2BL.%26aulast%3DPollard%26aufirst%3DW.%2BT.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26atitle%3DGlide%253A%2520a%2520new%2520approach%2520for%2520rapid%252C%2520accurate%2520docking%2520and%2520scoring.%25202.%2520Enrichment%2520factors%2520in%2520database%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1750%26epage%3D1759" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group">Friesner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murphy, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Repasky, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, L. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greenwood, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Halgren, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanschagrin, P. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mainz, D. T.</span><span> </span><span class="NLM_article-title">Extra precision glide: docking and scoring incorporating a model of hydrophobic enclosure for protein–ligand complexes</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">6177</span><span class="NLM_x">–</span> <span class="NLM_lpage">6196</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm051256o" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=6177-6196&author=R.+A.+Friesnerauthor=R.+B.+Murphyauthor=M.+P.+Repaskyauthor=L.+L.+Fryeauthor=J.+R.+Greenwoodauthor=T.+A.+Halgrenauthor=P.+C.+Sanschagrinauthor=D.+T.+Mainz&title=Extra+precision+glide%3A+docking+and+scoring+incorporating+a+model+of+hydrophobic+enclosure+for+protein%E2%80%93ligand+complexes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1021%2Fjm051256o&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm051256o%26sid%3Dliteratum%253Aachs%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DFrye%26aufirst%3DL.%2BL.%26aulast%3DGreenwood%26aufirst%3DJ.%2BR.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DSanschagrin%26aufirst%3DP.%2BC.%26aulast%3DMainz%26aufirst%3DD.%2BT.%26atitle%3DExtra%2520precision%2520glide%253A%2520docking%2520and%2520scoring%2520incorporating%2520a%2520model%2520of%2520hydrophobic%2520enclosure%2520for%2520protein%25E2%2580%2593ligand%2520complexes%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D6177%26epage%3D6196" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group">Sherman, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Day, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobson, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friesner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Farid, R.</span><span> </span><span class="NLM_article-title">Novel procedure for modeling ligand/receptor induced fit effects</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">534</span><span class="NLM_x">–</span> <span class="NLM_lpage">553</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm050540c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlCgsr7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=534-553&author=W.+Shermanauthor=T.+Dayauthor=M.+P.+Jacobsonauthor=R.+A.+Friesnerauthor=R.+Farid&title=Novel+procedure+for+modeling+ligand%2Freceptor+induced+fit+effects"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Procedure for Modeling Ligand/Receptor Induced Fit Effects</span></div><div class="casAuthors">Sherman, Woody; Day, Tyler; Jacobson, Matthew P.; Friesner, Richard A.; Farid, Ramy</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">534-553</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We present a novel protein-ligand docking method that accurately accounts for both ligand and receptor flexibility by iteratively combining rigid receptor docking (Glide) with protein structure prediction (Prime) techniques.  While traditional rigid-receptor docking methods are useful when the receptor structure does not change substantially upon ligand binding, success is limited when the protein must be "induced" into the correct binding conformation for a given ligand.  We provide an in-depth description of our novel methodol. and present results for 21 pharmaceutically relevant examples.  Traditional rigid-receptor docking for these 21 cases yields an av. RMSD of 5.5 Å.  The av. ligand RMSD for docking to a flexible receptor for the 21 pairs is 1.4 Å; the RMSD is ≤1.8 Å for 18 of the cases.  For the three cases with RMSDs greater than 1.8 Å, the core of the ligand is properly docked and all key protein/ligand interactions are captured.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqm3E6ctnPGSrVg90H21EOLACvtfcHk0ljGbJ_NRPnAeg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtlCgsr7I&md5=388811ead5cee1fd460951263de486cb</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1021%2Fjm050540c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm050540c%26sid%3Dliteratum%253Aachs%26aulast%3DSherman%26aufirst%3DW.%26aulast%3DDay%26aufirst%3DT.%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DFarid%26aufirst%3DR.%26atitle%3DNovel%2520procedure%2520for%2520modeling%2520ligand%252Freceptor%2520induced%2520fit%2520effects%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D534%26epage%3D553" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group">Sherman, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beard, H. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Farid, R.</span><span> </span><span class="NLM_article-title">Use of an induced fit receptor structure in virtual screening</span> <span class="citation_source-journal">Chem. Biol. Drug Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">67</span><span class="NLM_x">, </span> <span class="NLM_fpage">83</span><span class="NLM_x">–</span> <span class="NLM_lpage">84</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Fjm501893k&amp;key=10.1111%2Fj.1747-0285.2005.00327.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Fjm501893k&amp;key=16492153" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BD28XhsVSjtL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2006&pages=83-84&author=W.+Shermanauthor=H.+S.+Beardauthor=R.+Farid&title=Use+of+an+induced+fit+receptor+structure+in+virtual+screening"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Use of an induced fit receptor structure in virtual screening</span></div><div class="casAuthors">Sherman, Woody; Beard, Hege S.; Farid, Ramy</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">83-84</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">A review.  The automated induced fit docking protocol was used to generate the DFG-out conformation from a p38 MAP kinase activation loop starting from a DFG-in structure (1a9u) and the ligand from 1kv1 (BMU).  In a virtual screening study of 25K decoy ligands and 46 known actives, using an ensemble consisting of the induced fit docking structure (DFG-out) and the 1a9u crystal structure (DFG-in), 14 actives were identified in the top 1% of the database, including BMU and BIRB 796.  3 Actives were identified when 1a9u was used alone.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7PRgtFzy8u7Vg90H21EOLACvtfcHk0lgc8ygQuX_SDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhsVSjtL0%253D&md5=efe9900aacaaea6af805c1cbdb350c73</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1111%2Fj.1747-0285.2005.00327.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1747-0285.2005.00327.x%26sid%3Dliteratum%253Aachs%26aulast%3DSherman%26aufirst%3DW.%26aulast%3DBeard%26aufirst%3DH.%2BS.%26aulast%3DFarid%26aufirst%3DR.%26atitle%3DUse%2520of%2520an%2520induced%2520fit%2520receptor%2520structure%2520in%2520virtual%2520screening%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2006%26volume%3D67%26spage%3D83%26epage%3D84" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group">Hocek, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holy, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Votruba, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dvorakova, H.</span><span> </span><span class="NLM_article-title">Synthesis and cytostatic activity of substituted 6-phenylpurine bases and nucleosides: application of the Suzuki–Miyaura cross-coupling reactions of 6-chloropurine derivatives with phenylboronic acids</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">43</span><span class="NLM_x">, </span> <span class="NLM_fpage">1817</span><span class="NLM_x">–</span> <span class="NLM_lpage">1825</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm991167%2B" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2000&pages=1817-1825&author=M.+Hocekauthor=A.+Holyauthor=I.+Votrubaauthor=H.+Dvorakova&title=Synthesis+and+cytostatic+activity+of+substituted+6-phenylpurine+bases+and+nucleosides%3A+application+of+the+Suzuki%E2%80%93Miyaura+cross-coupling+reactions+of+6-chloropurine+derivatives+with+phenylboronic+acids"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1021%2Fjm991167%2B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm991167%252B%26sid%3Dliteratum%253Aachs%26aulast%3DHocek%26aufirst%3DM.%26aulast%3DHoly%26aufirst%3DA.%26aulast%3DVotruba%26aufirst%3DI.%26aulast%3DDvorakova%26aufirst%3DH.%26atitle%3DSynthesis%2520and%2520cytostatic%2520activity%2520of%2520substituted%25206-phenylpurine%2520bases%2520and%2520nucleosides%253A%2520application%2520of%2520the%2520Suzuki%25E2%2580%2593Miyaura%2520cross-coupling%2520reactions%2520of%25206-chloropurine%2520derivatives%2520with%2520phenylboronic%2520acids%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2000%26volume%3D43%26spage%3D1817%26epage%3D1825" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group">Capek, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vrabel, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hasnik, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pohl, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hocek, M.</span><span> </span><span class="NLM_article-title">Aqueous-phase Suzuki–Miyaura cross-coupling reactions of free halopurine bases</span> <span class="citation_source-journal">Synthesis</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_fpage">3515</span><span class="NLM_x">–</span> <span class="NLM_lpage">3526</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtF2rsrrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2006&pages=3515-3526&author=P.+Capekauthor=M.+Vrabelauthor=Z.+Hasnikauthor=R.+Pohlauthor=M.+Hocek&title=Aqueous-phase+Suzuki%E2%80%93Miyaura+cross-coupling+reactions+of+free+halopurine+bases"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Aqueous-phase Suzuki-Miyaura cross-coupling reactions of free halopurine bases</span></div><div class="casAuthors">Capek, Petr; Vrabel, Milan; Hasnik, Zbynek; Pohl, Radek; Hocek, Michal</div><div class="citationInfo"><span class="NLM_cas:title">Synthesis</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume"></span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">3515-3526</span>CODEN:
                <span class="NLM_cas:coden">SYNTBF</span>;
        ISSN:<span class="NLM_cas:issn">0039-7881</span>.
    
            (<span class="NLM_cas:orgname">Georg Thieme Verlag</span>)
        </div><div class="casAbstract">The Suzuki-Miyaura reaction of 9-unsubstituted 2-, 6-, and 8-halopurine bases, e.g., I, II and III, resp., with diverse aryl- and alkenylboronic acids in water-acetonitrile mixts. under microwave irradn. was used for the single-step synthesis of arylpurines.  In most cases the product crystd. directly from the reaction mixt. in high yields and good purity.  The scope and limitations of these reactions were studied.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-m8oRscawvLVg90H21EOLACvtfcHk0lgc8ygQuX_SDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtF2rsrrK&md5=b86910cdcddbaee4b6102dce66f5843a</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCapek%26aufirst%3DP.%26aulast%3DVrabel%26aufirst%3DM.%26aulast%3DHasnik%26aufirst%3DZ.%26aulast%3DPohl%26aufirst%3DR.%26aulast%3DHocek%26aufirst%3DM.%26atitle%3DAqueous-phase%2520Suzuki%25E2%2580%2593Miyaura%2520cross-coupling%2520reactions%2520of%2520free%2520halopurine%2520bases%26jtitle%3DSynthesis%26date%3D2006%26spage%3D3515%26epage%3D3526" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group">Matulis, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kranz, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Salemme, F. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Todd, M. J.</span><span> </span><span class="NLM_article-title">Thermodynamic stability of carbonic anhydrase: measurements of binding affinity and stoichiometry using ThermoFluor</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">5258</span><span class="NLM_x">–</span> <span class="NLM_lpage">5266</span></span><div class="citationLinks">[<a href="/doi/10.1021/bi048135v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BD2MXitVCksr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2005&pages=5258-5266&author=D.+Matulisauthor=J.+K.+Kranzauthor=F.+R.+Salemmeauthor=M.+J.+Todd&title=Thermodynamic+stability+of+carbonic+anhydrase%3A+measurements+of+binding+affinity+and+stoichiometry+using+ThermoFluor"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Thermodynamic Stability of Carbonic Anhydrase: Measurements of Binding Affinity and Stoichiometry Using ThermoFluor</span></div><div class="casAuthors">Matulis, Daumantas; Kranz, James K.; Salemme, F. Raymond; Todd, Matthew J.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">5258-5266</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">ThermoFluor (a miniaturized high-throughput protein stability assay) was used to analyze the linkage between protein thermal stability and ligand binding.  Equil. binding ligands increase protein thermal stability by an amt. proportional to the concn. and affinity of the ligand.  Binding consts. (Kb) were measured by examg. the systematic effect of ligand concn. on protein stability.  The precise ligand effects depend on the thermodn. of protein stability: in particular, the unfolding enthalpy.  An extension of current theor. treatments was developed for tight binding inhibitors, where ligand effect on Tm can also reveal binding stoichiometry.  A thermodn. anal. of carbonic anhydrase by differential scanning calorimetry (DSC) enabled a dissection of the Gibbs free energy of stability into enthalpic and entropic components.  Under certain conditions, thermal stability increased by over 30°; the heat capacity of protein unfolding was estd. from the dependence of calorimetric enthalpy on Tm.  The binding affinity of six sulfonamide inhibitors to two isoenzymes (human type 1 and bovine type 2) was analyzed by both ThermoFluor and isothermal titrn. calorimetry (ITC), resulting in a good correlation in the rank ordering of ligand affinity.  This combined investigation by ThermoFluor, ITC, and DSC provides a detailed picture of the linkage between ligand binding and protein stability.  The systematic effect of ligands on stability is shown to be a general tool to measure affinity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrc--PfKRLn6rVg90H21EOLACvtfcHk0lgc8ygQuX_SDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXitVCksr4%253D&md5=79dd266c74abdeeab1bdf197dfa2affb</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1021%2Fbi048135v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi048135v%26sid%3Dliteratum%253Aachs%26aulast%3DMatulis%26aufirst%3DD.%26aulast%3DKranz%26aufirst%3DJ.%2BK.%26aulast%3DSalemme%26aufirst%3DF.%2BR.%26aulast%3DTodd%26aufirst%3DM.%2BJ.%26atitle%3DThermodynamic%2520stability%2520of%2520carbonic%2520anhydrase%253A%2520measurements%2520of%2520binding%2520affinity%2520and%2520stoichiometry%2520using%2520ThermoFluor%26jtitle%3DBiochemistry%26date%3D2005%26volume%3D44%26spage%3D5258%26epage%3D5266" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group">Wiseman, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williston, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brandts, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, L. N.</span><span> </span><span class="NLM_article-title">Rapid measurement of binding constants and heats of binding using a new titration calorimeter</span> <span class="citation_source-journal">Anal. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1989</span><span class="NLM_x">, </span> <span class="NLM_volume">179</span><span class="NLM_x">, </span> <span class="NLM_fpage">131</span><span class="NLM_x">–</span> <span class="NLM_lpage">137</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Fjm501893k&amp;key=10.1016%2F0003-2697%2889%2990213-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Fjm501893k&amp;key=2757186" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADyaL1MXktlKjsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=179&publication_year=1989&pages=131-137&author=T.+Wisemanauthor=S.+Willistonauthor=J.+F.+Brandtsauthor=L.+N.+Lin&title=Rapid+measurement+of+binding+constants+and+heats+of+binding+using+a+new+titration+calorimeter"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid measurement of binding constants and heats of binding using a new titration calorimeter</span></div><div class="casAuthors">Wiseman, Thomas; Williston, Samuel; Brandts, John F.; Lin, Lung Nan</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">179</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">131-7</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    </div><div class="casAbstract">A new titrn. calorimeter is described and results are presented for the binding of 2'-CMP to the active site of RNase A.  The instrument characteristics include very high sensitivity, rapid calorimetric response, and fast thermal equilibration.  Convenient software is available for instrument operation, data collection, data redn., and deconvolution to obtain least-squares ests. of binding parameters n, ΔH°, ΔS°, and the binding const. K.  Sample through-put for the instrument is high, and under favorable conditions binding consts. as large as 108 M-1 can be measured.  The bovine RNase A (RNase)/2'-CMP system was studied over a 50-fold range of RNase concn. and at 2 different temps.  The binding consts. were in the 105 to 106 M-1 range, depending on conditions, and heats of binding ∼-15,000 cal/mol.  Repeat detns. suggested errors of only a few percent in n, ΔH°, and K values over the most favorable concn. range.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJRWydqrfWArVg90H21EOLACvtfcHk0lhxjYXNbBzt2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1MXktlKjsrw%253D&md5=53fcdb7144fd85e57963add0bcdb6041</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1016%2F0003-2697%2889%2990213-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0003-2697%252889%252990213-3%26sid%3Dliteratum%253Aachs%26aulast%3DWiseman%26aufirst%3DT.%26aulast%3DWilliston%26aufirst%3DS.%26aulast%3DBrandts%26aufirst%3DJ.%2BF.%26aulast%3DLin%26aufirst%3DL.%2BN.%26atitle%3DRapid%2520measurement%2520of%2520binding%2520constants%2520and%2520heats%2520of%2520binding%2520using%2520a%2520new%2520titration%2520calorimeter%26jtitle%3DAnal.%2520Biochem.%26date%3D1989%26volume%3D179%26spage%3D131%26epage%3D137" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group">Leslie, A. G. W.; Powell, H.</span>MOSFLM, version <span class="NLM_edition">7.01</span>; <span class="NLM_publisher-name">MRC Laboratory of Molecular Biology</span>: <span class="NLM_publisher-loc">Cambridge, U.K.</span>,<span class="NLM_x"> </span><span class="NLM_year">2007</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2007&author=A.+G.+W.+Leslie&author=H.+Powell&title=MOSFLM"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DLeslie%26aufirst%3DA.%2BG.%2BW.%26jtitle%3DMOSFLM%26pub%3DMRC%2520Laboratory%2520of%2520Molecular%2520Biology%26date%3D2007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group">Kabsch, W.</span><span> </span><span class="NLM_article-title">Evaluation of single-crystal X-ray-diffraction data from a position-sensitive detector</span> <span class="citation_source-journal">J. Appl. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1988</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">916</span><span class="NLM_x">–</span> <span class="NLM_lpage">924</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Fjm501893k&amp;key=10.1107%2FS0021889888007903" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADyaL1MXhtVGmu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=1988&pages=916-924&author=W.+Kabsch&title=Evaluation+of+single-crystal+X-ray-diffraction+data+from+a+position-sensitive+detector"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Evaluation of single-crystal x-ray diffraction data from a position-sensitive detector</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">1988</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">916-24</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    </div><div class="casAbstract">A Fortran program was developed for the redn. of single-crystal diffraction data from a sequence of adjacent rotation pictures recorded at a fixed x-ray wavelength by an electronic area detector.  The electronic pictures (data frames) covering the first 5° of crystal rotation are used to locate strong diffraction spots and to est. the background.  The orientation of the crystal is derived automatically from the list of obsd. spots and all parameters describing the diffraction pattern are refined.  The only input required from the user is the specification of the space group, approx. cell dimensions, and detector setting.  When the initialization step is finished the program goes back to the first picture and evaluates all data frames in the order they arrive from the measurement.  Each element of an electronic picture (pixel) is labeled by the indexes of the nearest reflection using the current refined parameters describing the diffraction geometry.  If the pixel is close to its nearest reflection the counts contribute to the 3-dimensional profile; otherwise the counts are used to update the background.  Each profile is represented as if the reflection had followed the shortest path through the Ewald sphere and had been recorded on the surface of the sphere.  Reflections close to the Ewald sphere are kept in a hash table to allow rapid access for updating the profiles.  Reflections which have completely passed through the Ewald sphere are removed from the table and saved for further processing.  Intensities are estd. by fitting their profiles to an av. shape learned from strong neighboring reflections.  Smoothly varying scaling factors are applied to the Lp-cor. intensities which minimize discrepancies between symmetry-related reflections and fit to a ref. data set if available.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtr9BhTPWbOLVg90H21EOLACvtfcHk0lhxjYXNbBzt2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1MXhtVGmu7c%253D&md5=f8d8c37891ec0d60758000879e7be9d6</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1107%2FS0021889888007903&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889888007903%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DEvaluation%2520of%2520single-crystal%2520X-ray-diffraction%2520data%2520from%2520a%2520position-sensitive%2520detector%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D1988%26volume%3D21%26spage%3D916%26epage%3D924" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref69"><div class="reference"><strong class="refLabel"><a href="#ref69" class="rightTabRefNumLink">69</a></strong><div class="NLM_citation" id="rightTab-cit69"><span><span class="NLM_contrib-group">Kabsch, W.</span><span> </span><span class="NLM_article-title">Automatic-indexing of rotation diffraction patterns</span> <span class="citation_source-journal">J. Appl. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1988</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">67</span><span class="NLM_x">–</span> <span class="NLM_lpage">71</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Fjm501893k&amp;key=10.1107%2FS0021889887009737" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADyaL1cXhtVCgsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=1988&pages=67-71&author=W.+Kabsch&title=Automatic-indexing+of+rotation+diffraction+patterns"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Automatic indexing of rotation diffraction patterns</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">1988</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">67-71</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    </div><div class="casAbstract">A method is described which assigns indexes to a set of single-crystal reflections recorded by the rotation-oscillation technique using a fixed x-ray wavelength.  The space group and approx. unit-cell parameters are known.  The unknown crystal orientation is detd. directly from the obsd. diffraction pattern of one or several oscillation data records.  A local indexing procedure is described which allows for large initial errors in the parameters controlling the diffraction pattern.  These parameters are refined subsequently, thereby satisfying the constraints imposed by the space-group symmetry.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9WshttyE38LVg90H21EOLACvtfcHk0lhxjYXNbBzt2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1cXhtVCgsb8%253D&md5=268552b3f6069d2465b3916a985b840f</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1107%2FS0021889887009737&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889887009737%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DAutomatic-indexing%2520of%2520rotation%2520diffraction%2520patterns%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D1988%26volume%3D21%26spage%3D67%26epage%3D71" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref70"><div class="reference"><strong class="refLabel"><a href="#ref70" class="rightTabRefNumLink">70</a></strong><div class="NLM_citation" id="rightTab-cit70"><span><span class="NLM_contrib-group">Evans, P.</span>SCALA—Scale Together Multiple Observations of Reflections, version <span class="NLM_edition">3.3.0</span>; <span class="NLM_publisher-name">MRC Laboratory of Molecular Biology</span>: <span class="NLM_publisher-loc">Cambridge, U.K.</span>,<span class="NLM_x"> </span><span class="NLM_year">2007</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2007&author=P.+Evans&title=SCALA%E2%80%94Scale+Together+Multiple+Observations+of+Reflections"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DP.%26jtitle%3DSCALA%25E2%2580%2594Scale%2520Together%2520Multiple%2520Observations%2520of%2520Reflections%26pub%3DMRC%2520Laboratory%2520of%2520Molecular%2520Biology%26date%3D2007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref71"><div class="reference"><strong class="refLabel"><a href="#ref71" class="rightTabRefNumLink">71</a></strong><div class="NLM_citation" id="rightTab-cit71"><span><span class="NLM_contrib-group">McCoy, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Storoni, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span> </span><span class="NLM_article-title">Likelihood-enhanced fast translation functions</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">61</span><span class="NLM_x">, </span> <span class="NLM_fpage">458</span><span class="NLM_x">–</span> <span class="NLM_lpage">464</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Fjm501893k&amp;key=10.1107%2FS0907444905001617" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Fjm501893k&amp;key=15805601" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BD2MXivVeisrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2005&pages=458-464&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Likelihood-enhanced+fast+translation+functions"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Likelihood-enhanced fast translation functions</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">D61</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">458-464</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">This paper is a companion to a recent paper on fast rotation functions [Storoni et al. (2004), Acta Cryst. D60, 432-438], which showed how a Taylor-series expansion of the max.-likelihood rotation function leads to improved likelihood-enhanced fast rotation functions.  In a similar manner, it is shown here how linear and quadratic Taylor-series expansions and least-squares approxns. of the max.-likelihood translation function lead to likelihood-enhanced translation functions, which can be calcd. by FFT and which are more sensitive to the correct translation than the traditional correlation-coeff. fast translation function.  These likelihood-enhanced translation targets for mol.-replacement searches have been implemented in the program Phaser using the Computational Crystallog. Toolbox (cctbx).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDd3wovyUbWLVg90H21EOLACvtfcHk0lhxjYXNbBzt2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXivVeisrY%253D&md5=bac9cc121bcc0d5cd314955a0e65e746</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1107%2FS0907444905001617&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444905001617%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DLikelihood-enhanced%2520fast%2520translation%2520functions%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2005%26volume%3D61%26spage%3D458%26epage%3D464" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref72"><div class="reference"><strong class="refLabel"><a href="#ref72" class="rightTabRefNumLink">72</a></strong><div class="NLM_citation" id="rightTab-cit72"><span><span class="NLM_contrib-group">Perrakis, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morris, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lamzin, V. S.</span><span> </span><span class="NLM_article-title">Automated protein model building combined with iterative structure refinement</span> <span class="citation_source-journal">Nat. Struct. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">458</span><span class="NLM_x">–</span> <span class="NLM_lpage">463</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Fjm501893k&amp;key=10.1038%2F8263" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Fjm501893k&amp;key=10331874" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADyaK1MXjtlajurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1999&pages=458-463&author=A.+Perrakisauthor=R.+Morrisauthor=V.+S.+Lamzin&title=Automated+protein+model+building+combined+with+iterative+structure+refinement"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Automated protein model building combined with iterative structure refinement</span></div><div class="casAuthors">Perrakis, Anastassis; Morris, Richard; Lamzin, Victor S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural Biology</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">458-463</span>CODEN:
                <span class="NLM_cas:coden">NSBIEW</span>;
        ISSN:<span class="NLM_cas:issn">1072-8368</span>.
    
            (<span class="NLM_cas:orgname">Nature America</span>)
        </div><div class="casAbstract">In protein crystallog., much time and effort are often required to trace an initial model from an interpretable electron d. map and to refine it until it best agrees with the crystallog. data.  Here, we present a method to build and refine a protein model automatically and without user intervention, starting from diffraction data extending to resoln. higher than 2.3 Å and reasonable ests. of crystallog. phases.  The method is based on an iterative procedure that describes the electron d. map as a set of unconnected atoms and then searches for protein-like patterns.  Automatic pattern recognition (model building) combined with refinement, allows a structural model to be obtained reliably within a few CPU hours.  We demonstrate the power of the method with examples of a few recently solved structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbcrkUoPTzsLVg90H21EOLACvtfcHk0ljugPNFc4drtw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXjtlajurs%253D&md5=d12fc80eb73e1faebee29e2f6635c86d</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1038%2F8263&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F8263%26sid%3Dliteratum%253Aachs%26aulast%3DPerrakis%26aufirst%3DA.%26aulast%3DMorris%26aufirst%3DR.%26aulast%3DLamzin%26aufirst%3DV.%2BS.%26atitle%3DAutomated%2520protein%2520model%2520building%2520combined%2520with%2520iterative%2520structure%2520refinement%26jtitle%3DNat.%2520Struct.%2520Biol.%26date%3D1999%26volume%3D6%26spage%3D458%26epage%3D463" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref73"><div class="reference"><strong class="refLabel"><a href="#ref73" class="rightTabRefNumLink">73</a></strong><div class="NLM_citation" id="rightTab-cit73"><span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K.</span><span> </span><span class="NLM_article-title">Coot: model-building tools for molecular graphics</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">60</span><span class="NLM_x">, </span> <span class="NLM_fpage">2126</span><span class="NLM_x">–</span> <span class="NLM_lpage">2132</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Fjm501893k&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Fjm501893k&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+model-building+tools+for+molecular+graphics"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0ljugPNFc4drtw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref74"><div class="reference"><strong class="refLabel"><a href="#ref74" class="rightTabRefNumLink">74</a></strong><div class="NLM_citation" id="rightTab-cit74"><span><span class="NLM_contrib-group">Murshudov, G. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vagin, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dodson, E. J.</span><span> </span><span class="NLM_article-title">Refinement of macromolecular structures by the maximum-likelihood method</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">240</span><span class="NLM_x">–</span> <span class="NLM_lpage">255</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Fjm501893k&amp;key=10.1107%2FS0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=8&amp;doi=10.1021%2Fjm501893k&amp;key=15299926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=240-255&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+macromolecular+structures+by+the+maximum-likelihood+method"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. σA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial α-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0ljugPNFc4drtw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1107%2FS0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520macromolecular%2520structures%2520by%2520the%2520maximum-likelihood%2520method%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1997%26volume%3D53%26spage%3D240%26epage%3D255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref75"><div class="reference"><strong class="refLabel"><a href="#ref75" class="rightTabRefNumLink">75</a></strong><div class="NLM_citation" id="rightTab-cit75"><span><span class="NLM_contrib-group">Painter, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Merritt, E. A.</span><span> </span><span class="NLM_article-title">Optimal description of a protein structure in terms of multiple groups undergoing TLS motion</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">62</span><span class="NLM_x">, </span> <span class="NLM_fpage">439</span><span class="NLM_x">–</span> <span class="NLM_lpage">450</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=16&amp;doi=10.1021%2Fjm501893k&amp;key=10.1107%2FS0907444906005270" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=8&amp;doi=10.1021%2Fjm501893k&amp;key=16552146" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BD28Xis1Gkt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2006&pages=439-450&author=J.+Painterauthor=E.+A.+Merritt&title=Optimal+description+of+a+protein+structure+in+terms+of+multiple+groups+undergoing+TLS+motion"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">Optimal description of a protein structure in terms of multiple groups undergoing TLS motion</span></div><div class="casAuthors">Painter, Jay; Merritt, Ethan A.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">D62</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">439-450</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">A single protein crystal structure contains information about dynamic properties of the protein as well as providing a static view of one three-dimensional conformation.  This addnl. information is to be found in the distribution of obsd. electron d. about the mean position of each atom.  It is general practice to account for this by refining a sep. at. displacement parameter (ADP) for each at. center.  However, these same displacements are often described well by simpler models based on TLS (translation/libration/screw) rigid-body motion of large groups of atoms, for example interdomain hinge motion.  A procedure, TLSMD, has been developed that analyzes the distribution of ADPs in a previously refined protein crystal structure in order to generate optimal multi-group TLS descriptions of the constituent protein chains.  TLSMD is applicable to crystal structures at any resoln.  The models generated by TLSMD anal. can significantly improve the std. crystallog. residuals R and Rfree and can reveal intrinsic dynamic properties of the protein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrIR_NcneJBNbVg90H21EOLACvtfcHk0ljugPNFc4drtw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xis1Gkt74%253D&md5=6289483d19fb6dd64fe9b032e9c45005</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1107%2FS0907444906005270&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444906005270%26sid%3Dliteratum%253Aachs%26aulast%3DPainter%26aufirst%3DJ.%26aulast%3DMerritt%26aufirst%3DE.%2BA.%26atitle%3DOptimal%2520description%2520of%2520a%2520protein%2520structure%2520in%2520terms%2520of%2520multiple%2520groups%2520undergoing%2520TLS%2520motion%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2006%26volume%3D62%26spage%3D439%26epage%3D450" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref76"><div class="reference"><strong class="refLabel"><a href="#ref76" class="rightTabRefNumLink">76</a></strong><div class="NLM_citation" id="rightTab-cit76"><span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Picaud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keller, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wrobel, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morgenstern, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bracher, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span> </span><span class="NLM_article-title">Benzodiazepines and benzotriazepines as protein interaction inhibitors targeting bromodomains of the BET family</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">1878</span><span class="NLM_x">–</span> <span class="NLM_lpage">1886</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=16&amp;doi=10.1021%2Fjm501893k&amp;key=10.1016%2Fj.bmc.2011.10.080" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=8&amp;doi=10.1021%2Fjm501893k&amp;key=22137933" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BC38XjtlOksrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2012&pages=1878-1886&author=P.+Filippakopoulosauthor=S.+Picaudauthor=O.+Fedorovauthor=M.+Kellerauthor=M.+Wrobelauthor=O.+Morgensternauthor=F.+Bracherauthor=S.+Knapp&title=Benzodiazepines+and+benzotriazepines+as+protein+interaction+inhibitors+targeting+bromodomains+of+the+BET+family"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">Benzodiazepines and benzotriazepines as protein interaction inhibitors targeting bromodomains of the BET family</span></div><div class="casAuthors">Filippakopoulos, Panagis; Picaud, Sarah; Fedorov, Oleg; Keller, Marco; Wrobel, Matthias; Morgenstern, Olaf; Bracher, Franz; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1878-1886</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Benzodiazepines are psychoactive drugs with anxiolytic, sedative, skeletal muscle relaxant and amnestic properties.  Recently triazolo-benzodiazepines have been also described as potent and highly selective protein interaction inhibitors of bromodomain and extra-terminal (BET) proteins, a family of transcriptional co-regulators that play a key role in cancer cell survival and proliferation, but the requirements for high affinity interaction of this compd. class with bromodomains has not been described.  Here we provide insight into the structure-activity relationship (SAR) and selectivity of this versatile scaffold.  In addn., using high resoln. crystal structures we compared the binding mode of a series of benzodiazepine (BzD) and related triazolo-benzotriazepines (BzT) derivs. including clin. approved drugs such as alprazolam and midazolam.  Our anal. revealed the importance of the 1-Me triazolo ring system for BET binding and suggests modifications for the development of further high affinity bromodomain inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9BT8VUvs8_LVg90H21EOLACvtfcHk0liNPk45Yx6p2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjtlOksrw%253D&md5=aed7023bdb44ee0b83af1605f0be0ddd</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2011.10.080&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2011.10.080%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DKeller%26aufirst%3DM.%26aulast%3DWrobel%26aufirst%3DM.%26aulast%3DMorgenstern%26aufirst%3DO.%26aulast%3DBracher%26aufirst%3DF.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DBenzodiazepines%2520and%2520benzotriazepines%2520as%2520protein%2520interaction%2520inhibitors%2520targeting%2520bromodomains%2520of%2520the%2520BET%2520family%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2012%26volume%3D20%26spage%3D1878%26epage%3D1886" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref77"><div class="reference"><strong class="refLabel"><a href="#ref77" class="rightTabRefNumLink">77</a></strong><div class="NLM_citation" id="rightTab-cit77"><span><span class="NLM_contrib-group">Picaud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wells, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Felletar, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brotherton, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Savitsky, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diez-Dacal, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Philpott, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bountra, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lingard, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muller, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brennan, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span> </span><span class="NLM_article-title">RVX-208, an inhibitor of BET transcriptional regulators with selectivity for the second bromodomain</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">110</span><span class="NLM_x">, </span> <span class="NLM_fpage">19754</span><span class="NLM_x">–</span> <span class="NLM_lpage">19759</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Fjm501893k&amp;key=10.1073%2Fpnas.1310658110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=8&amp;doi=10.1021%2Fjm501893k&amp;key=24248379" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFKms7zN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=19754-19759&author=S.+Picaudauthor=C.+Wellsauthor=I.+Felletarauthor=D.+Brothertonauthor=S.+Martinauthor=P.+Savitskyauthor=B.+Diez-Dacalauthor=M.+Philpottauthor=C.+Bountraauthor=H.+Lingardauthor=O.+Fedorovauthor=S.+Mullerauthor=P.+E.+Brennanauthor=S.+Knappauthor=P.+Filippakopoulos&title=RVX-208%2C+an+inhibitor+of+BET+transcriptional+regulators+with+selectivity+for+the+second+bromodomain"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">RVX-208, an inhibitor of BET transcriptional regulators with selectivity for the second bromodomain</span></div><div class="casAuthors">Picaud, Sarah; Wells, Christopher; Felletar, Ildiko; Brotherton, Deborah; Martin, Sarah; Savitsky, Pavel; Diez-Dacal, Beatriz; Philpott, Martin; Bountra, Chas; Lingard, Hannah; Fedorov, Oleg; Muller, Susanne; Brennan, Paul E.; Knapp, Stefan; Filippakopoulos, Panagis</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">19754-19759,S19754/1-S19754/10</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Bromodomains have emerged as attractive candidates for the development of inhibitors targeting gene transcription.  Inhibitors of the bromo and extraterminal (BET) family recently showed promising activity in diverse disease models.  However, the pleiotropic nature of BET proteins regulating tissue-specific transcription has raised safety concerns and suggested that attempts should be made for domain-specific targeting.  Here, we report that RVX-208, a compd. currently in phase II clin. trials, is a BET bromodomain inhibitor specific for second bromodomains (BD2s).  Cocrystal structures revealed binding modes of RVX-208 and its synthetic precursor, and fluorescent recovery after photobleaching demonstrated that RVX-208 displaces BET proteins from chromatin.  However, gene-expression data showed that BD2 inhibition only modestly affects BET-dependent gene transcription.  Our data demonstrate the feasibility of specific targeting within the BET family resulting in different transcriptional outcomes and highlight the importance of BD1 in transcriptional regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqamW8vnIopKrVg90H21EOLACvtfcHk0liNPk45Yx6p2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFKms7zN&md5=c918702c9fdb91241bc837732730f7f6</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1310658110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1310658110%26sid%3Dliteratum%253Aachs%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DBrotherton%26aufirst%3DD.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DSavitsky%26aufirst%3DP.%26aulast%3DDiez-Dacal%26aufirst%3DB.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DLingard%26aufirst%3DH.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26atitle%3DRVX-208%252C%2520an%2520inhibitor%2520of%2520BET%2520transcriptional%2520regulators%2520with%2520selectivity%2520for%2520the%2520second%2520bromodomain%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2013%26volume%3D110%26spage%3D19754%26epage%3D19759" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref78"><div class="reference"><strong class="refLabel"><a href="#ref78" class="rightTabRefNumLink">78</a></strong><div class="NLM_citation" id="rightTab-cit78"><span><span class="NLM_contrib-group">Ghosh, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lagisetty, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zajc, B.</span><span> </span><span class="NLM_article-title">Direct synthesis of 8-fluoro purine nucleosides via metalation–fluorination</span> <span class="citation_source-journal">J. Org. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">72</span><span class="NLM_x">, </span> <span class="NLM_fpage">8222</span><span class="NLM_x">–</span> <span class="NLM_lpage">8226</span></span><div class="citationLinks">[<a href="/doi/10.1021/jo071121l" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2007&pages=8222-8226&author=A.+K.+Ghoshauthor=P.+Lagisettyauthor=B.+Zajc&title=Direct+synthesis+of+8-fluoro+purine+nucleosides+via+metalation%E2%80%93fluorination"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1021%2Fjo071121l&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo071121l%26sid%3Dliteratum%253Aachs%26aulast%3DGhosh%26aufirst%3DA.%2BK.%26aulast%3DLagisetty%26aufirst%3DP.%26aulast%3DZajc%26aufirst%3DB.%26atitle%3DDirect%2520synthesis%2520of%25208-fluoro%2520purine%2520nucleosides%2520via%2520metalation%25E2%2580%2593fluorination%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2007%26volume%3D72%26spage%3D8222%26epage%3D8226" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref79"><div class="reference"><strong class="refLabel"><a href="#ref79" class="rightTabRefNumLink">79</a></strong><div class="NLM_citation" id="rightTab-cit79"><span><span class="NLM_contrib-group">Roy, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schneller, S. W.</span><span> </span><span class="NLM_article-title">An unusual occurrence on attempted purine C-8 electrophilic fluorination of 5′-noraristeromycin</span> <span class="citation_source-journal">Org. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">3889</span><span class="NLM_x">–</span> <span class="NLM_lpage">3891</span></span><div class="citationLinks">[<a href="/doi/10.1021/ol051297e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Fjm501893k&amp;key=1%3ACAS%3A528%3ADC%252BD2MXntVersrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2005&pages=3889-3891&author=A.+Royauthor=S.+W.+Schneller&title=An+unusual+occurrence+on+attempted+purine+C-8+electrophilic+fluorination+of+5%E2%80%B2-noraristeromycin"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">An Unusual Occurrence on Attempted Purine C-8 Electrophilic Fluorination of 5'-Noraristeromycin</span></div><div class="casAuthors">Roy, Atanu; Schneller, Stewart W.</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">3889-3891</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7060</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Reaction of the in situ generated purine C-8 carbanion of a protected 5'-noraristeromycin deriv. with N-fluorobenzenesulfonimide gave 8-phenylsulfonyl-5'-noraristeromycin rather than the expected 8-fluoro deriv.  A single electron transfer (SET) mechanism is proposed for this occurrence.  The phenylsulfonyl product offers a structural feature common to some anti-HIV agents (no biol. data).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxmIetT-7UJ7Vg90H21EOLACvtfcHk0liNPk45Yx6p2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXntVersrc%253D&md5=9049560d481807b4714aecb94e4a04ae</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1021%2Fol051297e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol051297e%26sid%3Dliteratum%253Aachs%26aulast%3DRoy%26aufirst%3DA.%26aulast%3DSchneller%26aufirst%3DS.%2BW.%26atitle%3DAn%2520unusual%2520occurrence%2520on%2520attempted%2520purine%2520C-8%2520electrophilic%2520fluorination%2520of%25205%25E2%2580%25B2-noraristeromycin%26jtitle%3DOrg.%2520Lett.%26date%3D2005%26volume%3D7%26spage%3D3889%26epage%3D3891" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref80"><div class="reference"><strong class="refLabel"><a href="#ref80" class="rightTabRefNumLink">80</a></strong><div class="NLM_citation" id="rightTab-cit80"><span><span class="NLM_contrib-group">De, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jasani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arora, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gambhir, S. S.</span><span> </span><span class="NLM_article-title">Evolution of BRET biosensors from live cell to tissue-scale in vivo imaging</span> <span class="citation_source-journal">Front. Endocrinol. (Lausanne, Switz.)</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">131</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=131&author=A.+Deauthor=A.+Jasaniauthor=R.+Aroraauthor=S.+S.+Gambhir&title=Evolution+of+BRET+biosensors+from+live+cell+to+tissue-scale+in+vivo+imaging"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DDe%26aufirst%3DA.%26aulast%3DJasani%26aufirst%3DA.%26aulast%3DArora%26aufirst%3DR.%26aulast%3DGambhir%26aufirst%3DS.%2BS.%26atitle%3DEvolution%2520of%2520BRET%2520biosensors%2520from%2520live%2520cell%2520to%2520tissue-scale%2520in%2520vivo%2520imaging%26jtitle%3DFront.%2520Endocrinol.%2520%2528Lausanne%252C%2520Switz.%2529%26date%3D2013%26volume%3D4%26spage%3D131" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4MEN" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4MEN','PDB','4MEN'); return false;">PDB: 4MEN</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3HME" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3HME','PDB','3HME'); return false;">PDB: 3HME</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2OSS" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2OSS','PDB','2OSS'); return false;">PDB: 2OSS</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4XY8" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4XY8','PDB','4XY8'); return false;">PDB: 4XY8</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4XY9" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4XY9','PDB','4XY9'); return false;">PDB: 4XY9</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4XYA" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4XYA','PDB','4XYA'); return false;">PDB: 4XYA</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i62"><a href="/doi/suppl/10.1021/jm501893k">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_65521"></div></div></div></div></div><hr /></hr><p class="last">Figures showing alternative docking of <b>2a</b>, binding of <b>7d</b> to BRD4(1), docking of <b>11</b> to BRD9, and binding of <b>7d</b> and <b>11</b> to BET bromodomains and a file in xlsx format listing Δ<i>T</i><sub>m</sub> shifts of compounds. This material is available free of charge via the Internet at <a class="extLink" href="http://pubs.acs.org">http://pubs.acs.org</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm501893k/suppl_file/jm501893k_si_001.xlsx">jm501893k_si_001.xlsx (28.79 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm501893k/suppl_file/jm501893k_si_002.pdf">jm501893k_si_002.pdf (3.15 MB)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The models and structure factors have been deposited in the PDB with accession codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4XY8">4XY8</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4XY9">4XY9</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4XYA">4XYA</a>.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Fjm501893k&amp;pbContext=%3Barticle%3Aarticle%3A10.1021%2Fjm501893k%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2015.58.issue-6%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/jm501893k" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                5MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-b425991822a79ada59b3.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-fb88834b824b41b135ad.js"></script>

<script type="text/javascript" src="/wro/ke1q~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"667988f1cda71248","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.6.0","si":10}'></script>
</body>
</html>
