
<html lang="en"     class="pb-page"  data-request-id="0f441ce6-0e03-44b9-b070-be49e3d17603"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;article:article:10.1021/acs.jmedchem.0c00192;pageGroup:string:Publication Pages;subPage:string:Full Text;issue:issue:10.1021/jmcmar.2020.63.issue-10"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Selective Janus Kinase 2 (JAK2) Pseudokinase Ligands with a Diaminotriazole Core" /></meta><meta name="dc.Creator" content="Maria-Elena  Liosi" /></meta><meta name="dc.Creator" content="Stefan G.  Krimmer" /></meta><meta name="dc.Creator" content="Ana S.  Newton" /></meta><meta name="dc.Creator" content="Thomas K.  Dawson" /></meta><meta name="dc.Creator" content="David E.  Puleo" /></meta><meta name="dc.Creator" content="Kara J.  Cutrona" /></meta><meta name="dc.Creator" content="Yoshihisa  Suzuki" /></meta><meta name="dc.Creator" content="Joseph  Schlessinger" /></meta><meta name="dc.Creator" content="William L.  Jorgensen" /></meta><meta name="dc.Description" content="Janus kinases (JAKs) are non-receptor tyrosine kinases that are essential components of the JAK-STAT signaling pathway. Associated aberrant signaling is responsible for many forms of cancer and dis..." /></meta><meta name="Description" content="Janus kinases (JAKs) are non-receptor tyrosine kinases that are essential components of the JAK-STAT signaling pathway. Associated aberrant signaling is responsible for many forms of cancer and dis..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="April 24, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c00192" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright Â© 2020 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00192" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c00192" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00192" /></link>
        
    
    

<title>Selective Janus Kinase 2 (JAK2) Pseudokinase Ligands with a Diaminotriazole Core | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c00192" /></meta><meta property="og:title" content="Selective Janus Kinase 2 (JAK2) Pseudokinase Ligands with a Diaminotriazole Core" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0020.jpeg" /></meta><meta property="og:description" content="Janus kinases (JAKs) are non-receptor tyrosine kinases that are essential components of the JAK-STAT signaling pathway. Associated aberrant signaling is responsible for many forms of cancer and disorders of the immune system. The present focus is on the discovery of molecules that may regulate the activity of JAK2 by selective binding to the JAK2 pseudokinase domain, JH2. Specifically, the Val617Phe mutation in JH2 stimulates the activity of the adjacent kinase domain (JH1) resulting in myeloproliferative disorders. Starting from a non-selective screening hit, we have achieved the goal of discovering molecules that preferentially bind to the ATP binding site in JH2 instead of JH1. We report the design and synthesis of the compounds and binding results for the JH1, JH2, and JH2 V617F domains, as well as five crystal structures for JH2 complexes. Testing with a selective and non-selective JH2 binder on the autophosphorylation of wild-type and V617F JAK2 is also contrasted." /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta><meta name="twitter:description" content="Janus kinases (JAKs) are non-receptor tyrosine kinases that are essential components of the JAK-STAT signaling pathway. Associated aberrant signaling is responsible for many forms of cancer and disorders of the immune system. The present focus is on the discovery of molecules that may regulate the activity of JAK2 by selective binding to the JAK2 pseudokinase domain, JH2. Specifically, the Val617Phe mutation in JH2 stimulates the activity of the adjacent kinase domain (JH1) resulting in myeloproliferative disorders. Starting from a non-selective screening hit, we have achieved the goal of discovering molecules that preferentially bind to the ATP binding site in JH2 instead of JH1. We report the design and synthesis of the compounds and binding results for the JH1, JH2, and JH2 V617F domains, as well as five crystal structures for JH2 complexes. Testing with a selective and non-selective JH2 binder on the autophosphorylation of wild-type and V617F JAK2 is also contrasted." /></meta><meta name="twitter:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0020.jpeg" /></meta><meta name="twitter:title" content="Selective Janus Kinase 2 (JAK2) Pseudokinase Ligands with a Diaminotriazole Core" /></meta>
        
            <meta property="og:type" content="Article" /></meta>
        

        
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c00192"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/ke1q~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-6ee29527b3d8ff00a106.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00192">
            
        





































    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=âtrueâ></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=âtrueâ></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=âtrueâ></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=âtrueâ></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-12/achre4.2021.54.issue-12/20210615/achre4.2021.54.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-6/amrcda.2021.2.issue-6/20210625/amrcda.2021.2.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-6/aabmcb.2021.4.issue-6/20210621/aabmcb.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-6/aaembp.2021.3.issue-6/20210622/aaembp.2021.3.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-6/aaemcq.2021.4.issue-6/20210628/aaemcq.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-25/aamick.2021.13.issue-25/20210630/aamick.2021.13.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-6/aanmf6.2021.4.issue-6/20210625/aanmf6.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-6/aapmcd.2021.3.issue-6/20210611/aapmcd.2021.3.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-6/abseba.2021.7.issue-6/20210614/abseba.2021.7.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-12/accacs.2021.11.issue-12/20210618/accacs.2021.11.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-6/acscii.2021.7.issue-6/20210623/acscii.2021.7.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-6/acbcct.2021.16.issue-6/20210618/acbcct.2021.16.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-3/achsc5.2021.28.issue-3/20210524/achsc5.2021.28.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-12/acncdm.2021.12.issue-12/20210616/acncdm.2021.12.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-6/aesccq.2021.5.issue-6/20210617/aesccq.2021.5.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-6/aelccp.2021.6.issue-6/20210611/aelccp.2021.6.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-6/aeecco.2021.1.issue-6/20210611/aeecco.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-6/aewcaa.2021.1.issue-6/20210611/aewcaa.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-5/afsthl.2021.1.issue-5/20210618/afsthl.2021.1.issue-5.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-6/aidcbc.2021.7.issue-6/20210611/aidcbc.2021.7.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-6/amlccd.2021.10.issue-6/20210615/amlccd.2021.10.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-6/amlcef.2021.3.issue-6/20210607/amlcef.2021.3.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-6/amclct.2021.12.issue-6/20210610/amclct.2021.12.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-6/ancac3.2021.15.issue-6/20210622/ancac3.2021.15.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-25/acsodf.2021.6.issue-25/20210629/acsodf.2021.6.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-6/apchd5.2021.8.issue-6/20210616/apchd5.2021.8.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-6/ascefj.2021.6.issue-6/20210625/ascefj.2021.6.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-25/ascecg.2021.9.issue-25/20210628/ascecg.2021.9.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-6/asbcd6.2021.10.issue-6/20210618/asbcd6.2021.10.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-25/ancham.2021.93.issue-25/20210629/ancham.2021.93.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-25/bichaw.2021.60.issue-25/20210629/bichaw.2021.60.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-6/bcches.2021.32.issue-6/20210616/bcches.2021.32.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-6/bomaf6.2021.22.issue-6/20210614/bomaf6.2021.22.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-24/cgeabj.2021.99.issue-24/20210628-01/cgeabj.2021.99.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-6/crtoec.2021.34.issue-6/20210621/crtoec.2021.34.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-12/chreay.2021.121.issue-12/20210623/chreay.2021.121.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-12/cmatex.2021.33.issue-12/20210622/cmatex.2021.33.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-6/cgdefu.2021.21.issue-6/20210602/cgdefu.2021.21.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-12/enfuem.2021.35.issue-12/20210617/enfuem.2021.35.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-12/esthag.2021.55.issue-12/20210615/esthag.2021.55.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-6/estlcu.2021.8.issue-6/20210608/estlcu.2021.8.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-25/iecred.2021.60.issue-25/20210630/iecred.2021.60.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-12/inocaj.2021.60.issue-12/20210621/inocaj.2021.60.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-6/jaaucr.2021.1.issue-6/20210628/jaaucr.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-25/jafcau.2021.69.issue-25/20210630/jafcau.2021.69.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-6/jceaax.2021.66.issue-6/20210610/jceaax.2021.66.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-6/jceda8.2021.98.issue-6/20210608/jceda8.2021.98.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-6/jcisd8.2021.61.issue-6/20210628/jcisd8.2021.61.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-6/jctcce.2021.17.issue-6/20210608/jctcce.2021.17.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/jmcmar.2021.64.issue-12/20210624/jmcmar.2021.64.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-6/jnprdf.2021.84.issue-6/20210625/jnprdf.2021.84.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-12/joceah.2021.86.issue-12/20210618/joceah.2021.86.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-24/jpcafh.2021.125.issue-24/20210624/jpcafh.2021.125.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-24/jpcbfk.2021.125.issue-24/20210624/jpcbfk.2021.125.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-24/jpccck.2021.125.issue-24/20210624/jpccck.2021.125.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-24/jpclcd.2021.12.issue-24/20210624/jpclcd.2021.12.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-6/jprobs.2021.20.issue-6/20210604/jprobs.2021.20.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-25/jacsat.2021.143.issue-25/20210630/jacsat.2021.143.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-6/jamsef.2021.32.issue-6/20210602/jamsef.2021.32.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-25/langd5.2021.37.issue-25/20210629/langd5.2021.37.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-12/mamobx.2021.54.issue-12/20210622/mamobx.2021.54.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-6/mpohbp.2021.18.issue-6/20210607/mpohbp.2021.18.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-12/nalefd.2021.21.issue-12/20210623/nalefd.2021.21.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-12/orlef7.2021.23.issue-12/20210618/orlef7.2021.23.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-6/oprdfk.2021.25.issue-6/20210618/oprdfk.2021.25.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-12/orgnd7.2021.40.issue-12/20210628/orgnd7.2021.40.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c00192&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c00192&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c00192&amp;href=/doi/10.1021/acs.jmedchem.0c00192" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 10</span><span class="cit-fg-pageRange">, 5324-5340</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/10" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c00082" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c00102" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Selective Janus Kinase 2 (JAK2) Pseudokinase Ligands with a Diaminotriazole Core</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Maria-Elena Liosi</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Maria-Elena Liosi</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, Yale University, New Haven, Connecticut 06520-8107, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Maria-Elena++Liosi">Maria-Elena Liosi</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Stefan G. Krimmer</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Stefan G. Krimmer</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, Yale University, New Haven, Connecticut 06520-8107, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Stefan+G.++Krimmer">Stefan G. Krimmer</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Ana S. Newton</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ana S. Newton</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, Yale University, New Haven, Connecticut 06520-8107, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ana+S.++Newton">Ana S. Newton</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Thomas K. Dawson</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Thomas K. Dawson</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, Yale University, New Haven, Connecticut 06520-8107, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Thomas+K.++Dawson">Thomas K. Dawson</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">David E. Puleo</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">David E. Puleo</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmacology, Yale University School of Medicine, New Haven, Connecticut 06520-8066, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=David+E.++Puleo">David E. Puleo</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Kara J. Cutrona</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kara J. Cutrona</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, Yale University, New Haven, Connecticut 06520-8107, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kara+J.++Cutrona">Kara J. Cutrona</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Yoshihisa Suzuki</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yoshihisa Suzuki</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmacology, Yale University School of Medicine, New Haven, Connecticut 06520-8066, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yoshihisa++Suzuki">Yoshihisa Suzuki</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Joseph Schlessinger</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Joseph Schlessinger</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmacology, Yale University School of Medicine, New Haven, Connecticut 06520-8066, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Joseph++Schlessinger">Joseph Schlessinger</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-5085-5969" title="Orcid link">http://orcid.org/0000-0002-5085-5969</a></div></div></span></li><span class="comma-separator">,Â andÂ </span><li><span><span class="hlFld-ContribAuthor">William L. Jorgensen</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">William L. Jorgensen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, Yale University, New Haven, Connecticut 06520-8107, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#384f5154545159551652574a5f5d564b5d56784159545d165d5c4d"><span class="__cf_email__" data-cfemail="7007191c1c19111d5e1a1f0217151e03151e3009111c155e151405">[emailÂ protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=William+L.++Jorgensen">William L. Jorgensen</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-3993-9520" title="Orcid link">http://orcid.org/0000-0002-3993-9520</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00192&amp;href=/doi/10.1021%2Facs.jmedchem.0c00192" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 10</span><span class="cit-pageRange">, 5324â5340</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">April 24, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>3 February 2020</li><li><span class="item_label"><b>Published</b> online</span>24 April 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 28 May 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c00192" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00192</a></div><div class="article_header-article-copyright"><strong>Copyright Â© 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D5324%26pageCount%3D17%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DMaria-Elena%2BLiosi%252C%2BStefan%2BG.%2BKrimmer%252C%2BAna%2BS.%2BNewton%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D10%26contentID%3Dacs.jmedchem.0c00192%26title%3DSelective%2BJanus%2BKinase%2B2%2B%2528JAK2%2529%2BPseudokinase%2BLigands%2Bwith%2Ba%2BDiaminotriazole%2BCore%26numPages%3D17%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D5340%26publicationDate%3DMay%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c00192"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1824</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">6</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c00192" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Selective Janus Kinase 2 (JAK2) Pseudokinase Ligands with a Diaminotriazole Core&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Maria-Elena&quot;,&quot;last_name&quot;:&quot;Liosi&quot;},{&quot;first_name&quot;:&quot;Stefan&quot;,&quot;last_name&quot;:&quot;G. Krimmer&quot;},{&quot;first_name&quot;:&quot;Ana&quot;,&quot;last_name&quot;:&quot;S. Newton&quot;},{&quot;first_name&quot;:&quot;Thomas&quot;,&quot;last_name&quot;:&quot;K. Dawson&quot;},{&quot;first_name&quot;:&quot;David&quot;,&quot;last_name&quot;:&quot;E. Puleo&quot;},{&quot;first_name&quot;:&quot;Kara&quot;,&quot;last_name&quot;:&quot;J. Cutrona&quot;},{&quot;first_name&quot;:&quot;Yoshihisa&quot;,&quot;last_name&quot;:&quot;Suzuki&quot;},{&quot;first_name&quot;:&quot;Joseph&quot;,&quot;last_name&quot;:&quot;Schlessinger&quot;},{&quot;first_name&quot;:&quot;William&quot;,&quot;last_name&quot;:&quot;L. Jorgensen&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;04&quot;,&quot;day&quot;:&quot;24&quot;,&quot;issue&quot;:&quot;10&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;5324-5340&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c00192&quot;},&quot;abstract&quot;:&quot;Janus kinases (JAKs) are non-receptor tyrosine kinases that are essential components of the JAK-STAT signaling pathway. Associated aberrant signaling is responsible for many forms of cancer and disorders of the immune system. The present focus is on the discovery of molecules that may regulate the activity of JAK2 by selective binding to the JAK2 pseudokinase domain, JH2. Specifically, the Val617Phe mutation in JH2 stimulates the activity of the adjacent kinase domain (JH1) resulting in myeloproliferative disorders. Starting from a non-selective screening hit, we have achieved the goal of discovering molecules that preferentially bind to the ATP binding site in JH2 instead of JH1. We report the design and synthesis of the compounds and binding results for the JH1, JH2, and JH2 V617F domains, as well as five crystal structures for JH2 complexes. Testing with a selective and non-selective JH2 binder on the autophosphorylation of wild-type and V617F JAK2 is also contrasted.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00192&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00192" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00192&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00192" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00192&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00192" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00192&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00192&amp;href=/doi/10.1021/acs.jmedchem.0c00192" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c00192" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c00192" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (6 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00192&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c00192%26sid%3Dliteratum%253Aachs%26pmid%3D32329617%26genre%3Darticle%26aulast%3DLiosi%26date%3D2020%26atitle%3DSelective%2BJanus%2BKinase%2B2%2B%2528JAK2%2529%2BPseudokinase%2BLigands%2Bwith%2Ba%2BDiaminotriazole%2BCore%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D10%26spage%3D5324%26epage%3D5340%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">Â»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=291763" title="Noncovalent interactions">Noncovalent interactions</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291439" title="Genetics">Genetics</a>,</li><li><a href="/action/doSearch?ConceptID=291754" title="Molecules">Molecules</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/10" title="Go to Volume 63, Issue 10"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/jmcmar.2020.63.issue-10/20200528/jmcmar.2020.63.issue-10.largecover.jpg" alt="Go to Volume 63, Issue 10"/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/medium/jm0c00192_0020.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0020.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00192&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Janus kinases (JAKs) are non-receptor tyrosine kinases that are essential components of the JAK-STAT signaling pathway. Associated aberrant signaling is responsible for many forms of cancer and disorders of the immune system. The present focus is on the discovery of molecules that may regulate the activity of JAK2 by selective binding to the JAK2 pseudokinase domain, JH2. Specifically, the Val617Phe mutation in JH2 stimulates the activity of the adjacent kinase domain (JH1) resulting in myeloproliferative disorders. Starting from a non-selective screening hit, we have achieved the goal of discovering molecules that preferentially bind to the ATP binding site in JH2 instead of JH1. We report the design and synthesis of the compounds and binding results for the JH1, JH2, and JH2 V617F domains, as well as five crystal structures for JH2 complexes. Testing with a selective and non-selective JH2 binder on the autophosphorylation of wild-type and V617F JAK2 is also contrasted.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_10726" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_10726" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">JAK1, JAK2, JAK3, and TYK2 are members of the Janus family of non-receptor tyrosine kinases, which are activated by and mediate the cellular responses induced by binding of a variety of cytokines to specific cytokine receptors.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Cytokine-induced activation of the JAK-STAT signaling pathway and other intracellular pathways plays important roles in the control of cell proliferation, hematopoiesis, and immune functions. In addition to a canonical tyrosine kinase domain (JH1) located in the C-terminal region, JAK proteins contain a pseudokinase domain designated JH2. Although JH2 domains have an ATP binding site, they show little or no catalytic activity. However, JH2 domains do have a regulatory function for the JH1 kinase activity such that mutations in JH2 can cause hyperactivation leading to numerous diseases and cancer.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> In particular, the single point mutation Val617Phe (V617F) in JAK2 JH2 is responsible for the majority of myeloproliferative disorders including polycythemia vera, myelofibrosis, and essential thrombocythemia.<a onclick="showRef(event, 'ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref3 ref4">(3,4)</a> Undesirable side effects such as anemia and thrombocytopenia are associated with inhibition of the wild-type kinase activity of the JH1 domain of JAK2.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> Thus, selective inhibition of V617F JAK2 could be therapeutically valuable. Interestingly, mutations of residues that eliminate ATP binding by JAK2 JH2 were found to enhance the autophosphorylation of wild-type JAK2 by 2â5 fold, while they generally reduced the activity of V617F JAK2 by 5â15 fold.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> An exception was the L579F JAK2 mutation, which does not remove ATP binding, and showed a reduced effect on V617F JAK2 activation. This work has raised the possibility of selective reversal of the activating effect of the V617F mutation by blocking ATP binding to JAK2 JH2.<a onclick="showRef(event, 'ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref5 ref6">(5,6)</a> To test this hypothesis, our first goal was to discover small molecules that can bind strongly to the ATP binding site of JAK2 JH2 but bind at most weakly to the ATP binding site of JAK2 JH1. The next step is testing the effects on the autophosphorylation activity of full-length wild-type and V617F JAK2.</div><div class="NLM_p">Toward these ends, in seeking molecules that bind to the JH2 ATP site, we reported results of a high-throughput screen that yielded the known pan-CDK and pan-JAK inhibitor JNJ7706621 (<b>1</b>, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>)<a onclick="showRef(event, 'ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref7 ref8">(7,8)</a> as the strongest JH2 binder in a fluorescence polarization (FP) assay.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> Subsequent analysis by isothermal titration calorimetry (ITC) yielded binding constants <i>K</i><sub>d</sub> of 106 and 31 nM with JAK2 JH2 and JH1, respectively, showing an undesirable greater than three-fold preference for binding JH1.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> It may be noted that these values are lower than an earlier <i>K</i><sub>d</sub> report of 220 nM for <b>1</b> with a JH1-JH2 construct of JAK2 in a competition binding assay.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> Crystal structures for <b>1</b> with JAK2 JH2 and JH1 were also determined, as illustrated in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> The binding sites are similar in the hinge regions, Glu627-Phe628-Val629 for JH2 and Glu930-Tyr931-Leu932 for JH1; the diaminotriazole fragment of <b>1</b> engages in three hydrogen bonds with the backbone in both cases. In addition, for JAK2 JH2, there are hydrogen bonds between the amino group of <b>1</b> and the side chain oxygen of the gatekeeper, Gln626, and between the carbonyl group of <b>1</b> and Lys581. The strong binding of <b>1</b> to both JAK2 JH1 and JH2 is consistent with the similar binding modes in the hinge regions and the limited contact of the difluorophenyl substituent. The molecule is U-shaped with the aryl group directed out of the binding site toward the solvent.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/medium/jm0c00192_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Compound <b>1</b> is the known pan-CDK and pan-JAK kinase inhibitor JNJ7706621. Compounds <b>2</b>â<b>15</b> are JAK2 JH2-binding molecules reported in this work.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00192&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/medium/jm0c00192_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (left) <b>1</b> bound to JAK2 JH2 (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5USZ">5USZ</a>) and (right) <b>1</b> bound to JAK2 JH1 (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5USY">5USY</a>). Hydrogen bonds with <i>r</i><sub>ON</sub> < 3.6 Ã are indicated with dashed lines. Selective binding to JAK2 JH2 is sought by growing the ligands toward the vacant region near T555 and R715.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00192&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In order to seek molecules that may selectively bind to the ATP site in JAK2 JH2 and avoid inhibition of the kinase activity of wild-type (WT) JAK2, the immediate goal was to discover small molecules whose affinities are at least as strong as <b>1</b> for JAK2 JH2, while showing much diminished affinity for JAK2 JH1. For the present series, <b>1</b> was chosen as the starting point with retention of the 3,5-diamino-1,2,4-triazole substructure. In viewing <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>, it is apparent that the desired selectivity needs to exploit differences in the vacant eastern part of the binding clefts heading toward Thr555 and Arg715 in JAK2 JH2. However, this is the region where the triphosphate portion of ATP is bound with the terminal phosphate group in a hydrogen bond cluster with Thr555, Asn678, and Arg715, as shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> Thus, it is a very polar region with the concomitant targeting difficulties owing to the competition between an envisioned polar substituent being well-solvated in water and in the polar binding site. Nevertheless, the challenge was taken on with the notion that precise fit of a relatively rigid, polar substituent ending in an anionic mimic of the terminal phosphate might be successful. Competition with ATP is not expected to be problematic since its binding is relatively weak with binding constants of 1.3 Î¼M for both WT JAK2 JH2 and the V617F mutant.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> It should also be noted that the principal structural changes for JAK2 V617F occur in this vicinity.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> Val617 is located above Phe595 in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>, and upon mutation, Phe617-Phe595-Phe594 forms an edge-face-edge stack with downward displacement of Phe594 toward the terminal phosphate in the ATP-bound structures.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> Phe595 has been shown to be indispensable for the constitutive activity of JAK2 V617F; mutation to non-aromatic residues significantly decreases the activity.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> Thus, exploration of alternative ligands that occupy the ATP site could reasonably be expected to affect the activity of JAK2 V617F, e.g., by reversing the displacement of F594.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/medium/jm0c00192_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Rendering from the crystal structure for ATP bound to JAK2 JH2 (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4FVQ">4FVQ</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00192&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_37187" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_37187" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Design and Assays</h3><div class="NLM_p">For thoroughness, the molecular design emphasized inspection of hundreds of trial structures for JAK2 JH2 complexes built with the BOMB (Biochemical and Organic Model Builder) program<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> and energy-minimized with MCPRO<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> using the OPLS-AA/M force field for proteins<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> and OPLS/CM1A for ligands.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> Free-energy perturbation (FEP) calculations were also carried out in some cases to better predict potential differences in free energies of binding for analogs;<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> these calculations included ca. 2000 explicit water molecules and extensive sampling of configurations for the JAK2 JH2 protein, ligands, and water. The principal assays measured binding constants with JAK2 JH2, V617F JH2, and JH1 via fluorescence polarization (FP)<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> using a fluorescein conjugate of <b>1</b> as the tracer.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> The assay followed the previous description<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> with minor changes to the buffer composition, as fully described in the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>. For several compounds, microscale thermophoresis was carried out as an additional approach for obtaining <i>K</i><sub>d</sub> values. The present measurements like the prior ITC and FP assays were carried out at pH 8.0, which provided more consistent results than current tests at pH 7.0 and 8.5.</div><div class="NLM_p">For the FP assays with JAK2 JH2, <b>1</b> was used as a control compound, so multiple <i>K</i><sub>d</sub> results were obtained yielding an average value of 0.456 Â± 0.124 Î¼M (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). The result from the MST measurements, 0.489 Â± 0.084 Î¼M, was notably consistent given the differences in the utilized buffers, as detailed in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c00192/suppl_file/jm0c00192_si_001.pdf" class="ext-link">Supporting Information</a>. These values are somewhat lower than the previous FP result of 0.80 Â± 0.05 Î¼M,<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> and they are significantly higher than the ITC result of 0.106 Î¼M<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> and of a recent report of 0.094 Î¼M from intrinsic tryptophan fluorescence.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> We have also assayed a second compound, BI-D1870, that was tested in ref <a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a>; our <i>K</i><sub>d</sub> of 2.0 Â± 0.2 Î¼M is again several-fold higher than 0.62 Â± 0.04 Î¼M in the prior report.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> Clearly, alternative binding assays are yielding somewhat different results, which likely reflects differences in the assay conditions and preparation of the protein. Thus, it is advisable to make comparisons relative to a common control, e.g., compound <b>1</b>.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Binding Affinity for JAK2 JH2 (<i>K</i><sub>d</sub>, Î¼M) from the FP Assay</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="char" char="." /></col><col align="char" char="Â±" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center" char=".">compound</th><th class="colsep0 rowsep0" align="center" char="Â±"><i>K</i><sub>d</sub> (Î¼M)<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>1</b></td><td class="colsep0 rowsep0" align="char" char="Â±">0.456 Â± 0.124</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>2</b></td><td class="colsep0 rowsep0" align="char" char="Â±">16.7 Â± 5.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>4</b></td><td class="colsep0 rowsep0" align="char" char="Â±">4.7 Â± 0.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>5</b></td><td class="colsep0 rowsep0" align="char" char="Â±">47.8 Â± 9.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>6</b></td><td class="colsep0 rowsep0" align="char" char="Â±">30.5 Â± 3.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>7</b></td><td class="colsep0 rowsep0" align="char" char="Â±">12.3 Â± 0.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>8</b></td><td class="colsep0 rowsep0" align="char" char="Â±">1.9 Â± 0.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>9</b></td><td class="colsep0 rowsep0" align="char" char="Â±">0.643 Â± 0.019</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>10</b></td><td class="colsep0 rowsep0" align="char" char="Â±">0.571 Â± 0.034</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>11</b></td><td class="colsep0 rowsep0" align="char" char="Â±">4.0 Â± 0.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>12</b></td><td class="colsep0 rowsep0" align="char" char="Â±">0.346 Â± 0.034</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>13</b></td><td class="colsep0 rowsep0" align="char" char="Â±">0.439 Â± 0.064</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>14</b></td><td class="colsep0 rowsep0" align="char" char="Â±">7.0 Â± 0.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>15</b></td><td class="colsep0 rowsep0" align="char" char="Â±">0.374 Â± 0.013</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last"><i>K</i><sub>d</sub> data from quadruplicate measurements in three independent assays. Mean Â± SEM.</p></div></div></div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Synthesis of New Compounds</h3><div class="NLM_p">The new compounds reported here are numbered <b>2</b>â<b>15</b> in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>; complete synthetic details and characterization data are reported in the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>. However, the general approach can be illustrated with the synthetic route for <b>12</b> in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. The key coupling step is the regioselective acylation of a 1<i>H</i>-[1,2,4]triazole-3,5-diamine<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> with a phenyl carbamate (step e in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). Of course, obstacles arose. Indeed, for the coupling, the original intention was to employ a common procedure for urea formation, the reaction of two amines with triphosgene in the presence of triethylamine. This strategy afforded compound <b>3</b> in â¼10% yield but failed for more complex ureas, even when attempted in several different solvents (DCM, THF, dioxane, DMSO) and at various temperatures (r.t., 80 Â°C, 130 Â°C). The problems appeared to be associated with the poor solubility in organic solvents for the polar diaminotriazole precursor and the presence of its tautomeric 2-<i>H</i> form, which leads to the undesirable and difficult-to-separate, regioisomeric 2-<i>H</i> byproduct. In search of a more efficient route, phenyl carbamates were investigated as alternative carbonyl sources.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> This approach proved to be effective in generating the desired products when dioxane was used as the solvent, but the yields were not optimal since the problems with the diaminotriazole precursor remained. Better results were obtained by increasing the reaction temperature from 80 to 110 Â°C, extending the reaction time, and diluting the mixture from 1.0 to 0.5 M to improve dissolution of the diaminotriazole. Finally, the use of DCM/MeCN as the solvent system for chromatography allowed the successful purification of the ester precursor of <b>12</b>.</div><figure id="sch1" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/medium/jm0c00192_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0010.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Compound <b>12</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00192&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) Boc<sub>2</sub>O, Et<sub>3</sub>N, <i>i</i>PrOH, r.t., and 14 h; (b) 10 mol % Pd(OAc)<sub>2</sub>, 20 mol % JohnPhos, Cs<sub>2</sub>CO<sub>3</sub>, dioxane, 110 Â°C, and 19 h; (c) TFA, DCM, 0 Â°C to r.t., and 30 min; (d) NaHCO<sub>3</sub>, H<sub>2</sub>O/THF, 0 Â°C, and 1 h; (e) Et<sub>3</sub>N, dioxane, 110 Â°C, and 50 min; (f) DBN, LiBr, MeCN, 2 vol % H<sub>2</sub>O, r.t., and 68 h.</p></p></figure><div class="NLM_p last">In several cases, preparation of the carbamates also proved challenging. For <b>12</b>, the original plan to prepare the 2-aryloxazole (step c in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>) was to utilize regioselective, palladium-catalyzed direct arylation of the commercially available ethyl oxazole-4-carboxylate with unprotected 4-iodo aniline, under the conditions described by Verrier et al.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> However, the suggested conditions (5 mol % Pd(OAc)<sub>2</sub>, 10 mol % JohnPhos) proved inefficient, affording the desired product in single-digit yields. Consequently, it was decided to protect the aniline and repeat the arylation with increased catalyst and ligand loading (10 mol % Pd(OAc)<sub>2</sub>, 20 mol % JohnPhos). This approach allowed access to the 2-anilinyloxazole in a viable yield (30%). Also, finally, hydrolysis of the ester precursor of <b>12</b> in the last step required care. Standard conditions using strong, nucleophilic bases led to decomposition of the urea. Fortunately, heteroatoms were present in the Î± or Î²-position of most of the ester precursors, which allowed use of the mild ester hydrolysis method introduced by Mattsson et al.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> In this transformation, lithium coordination to the carbonyl group and the neighboring heteroatom increases the electrophilicity and thus selectivity of the ester toward nucleophilic attack, allowing hydrolysis by water at room temperature. However, long reaction times were required due to the low solubility of the compounds in the reaction media.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Lead Optimization</h3><div class="NLM_p">An initial question that was addressed was the importance of the carbonyl group in <b>1</b> for binding to JAK2 JH2. Although the carbonyl oxygen atom participates in a hydrogen bond with the ammonium group of Lys581 (<i>r</i><sub>NO</sub> = 3.16 Ã), it is in repulsive contact, 3.06 Ã, with the side chain carbonyl oxygen atom of Gln626 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). Thus, <b>2</b>, the desoxy analog of <b>1</b>, was prepared; it was found to yield a much-reduced affinity for JAK2 JH2 with a <i>K</i><sub>d</sub> of 16.7 Î¼M in the FP assay (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Apparently, the hydrogen bond with the charged ammonium group more than offsets the repulsion with Gln626, and the reduced torsional flexibility of <b>1</b> compared to <b>2</b> is beneficial.</div><div class="NLM_p">Having established the desirability of the carbonyl group, the next step was to append a substituent that should project eastwardly past Lys581 toward Thr555 rather than outward from the binding site as for <b>1</b>. Although several alternatives were considered, a secondary amide linkage modeled well, especially with an <i>N</i>-phenyl substituent that might form a cationâÏ interaction with Lys581. This notion led to the synthesis of <b>3</b> and <b>4</b>, which only differ by the <i>p</i>-anilinyl substituent being cyano or sulfamyl. In general, we have observed little difference in binding affinity for these alternatives, but the lesser solubility of the cyano analogs sometimes led to assay difficulties as for <b>3</b>. The FP result for the <i>N</i>-phenylamide <b>4</b> with a 3-fold improvement over <b>2</b> to 4.7 Î¼M was encouraging in view of the fact that the phenyl ring is being placed in a polar environment. It was also possible to obtain crystal structures for the complexes of both <b>3</b> and <b>4</b> with JAK2 JH2 at 1.94 and 1.90 Ã resolution, respectively. The structures clearly reveal the desired eastwardly projection and the cationâÏ interaction with Lys581, as illustrated in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a> for <b>4</b>. The hydrogen bond length between the amide carbonyl oxygen of <b>4</b> and the ammonium nitrogen atom of Lys581 is 2.85 Ã, and the distances between the ammonium nitrogen and the carbon atoms of the phenyl ring are as short as 3.92 Ã for the ipso carbon atom and 3.77 Ã for the nearer ortho carbon atom, while the hydrogen bonding in the hinge region is essentially the same for <b>1</b>, <b>3</b>, and <b>4</b>.</div><figure id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/medium/jm0c00192_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Rendering from the 1.90 Ã crystal structure for <b>4</b> bound to JAK2 JH2 (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6OBB">6OBB</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00192&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To expand the structureâactivity data, <b>5</b>, the tertiary methylamide corresponding to <b>4</b>, was checked. Since there is a water molecule hydrogen-bonded to the amide NH in <b>4</b> (<i>r</i><sub>NO</sub> = 3.11 Ã, <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>), it might seem likely that <b>4</b> and <b>5</b> would have similar <i>K</i><sub>d</sub> values. However, a conformational issue should be noted for acetanilides. From small-molecule crystallography and quantum chemical calculations, it is known that acetanilide prefers the <i>Z</i>-conformer, while <i>N</i>-methylacetanilide prefers the <i>E</i>-conformer.<a onclick="showRef(event, 'ref22 ref23'); return false;" href="javascript:void(0);" class="ref ref22 ref23">(22,23)</a> We confirmed this with DFT B3LYP/6-31G(d) geometry optimizations and subsequent single-point energy evaluations at the MP2/6-311+G(d,p) level.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> The B3LYP and MP2 calculations favor the <i>Z</i>-conformer for acetanilide by 3.19 and 1.29 kcal/mol, respectively, while the <i>E</i>-conformer is preferred by 3.87 and 3.39 kcal/mol for <i>N</i>-methylacetanilide (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>). Assuming this preference carries over to <b>4</b> and <b>5</b>, there would be a significant conformational penalty for <b>5</b> to achieve the <i>Z</i>-conformation that is needed for the cationâÏ interaction with Lys581. As listed in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>, FEP calculations were also performed for the conversion of <i>Z</i>-<b>5</b> to <i>Z</i>-<b>4</b>; when adjusted for the conformational penalty, weaker binding is expected for <b>5</b>. This was borne out by the measured <i>K</i><sub>d</sub> of 47.8 Î¼M for <b>5</b> in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>.</div><figure id="sch2" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/medium/jm0c00192_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0012.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Computed Conformational Preferences (kcal/mol) for Acetanilides</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00192&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Differences in Free Energies of Binding with JAK2 JH2 from FEP Calculations</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="Â±" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">A â B</th><th class="colsep0 rowsep0" align="center">transformation</th><th class="colsep0 rowsep0" align="center" char="Â±">Î<i>G</i><sub>b</sub> (kcal/mol)<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5</b> â <b>4</b></td><td class="colsep0 rowsep0" align="left">(<i>Z</i>)-OâCNMePh â OâCNHPh</td><td class="colsep0 rowsep0" align="char" char="Â±">1.47 Â± 0.54</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6</b> â <b>4</b></td><td class="colsep0 rowsep0" align="left">2-pyridinyl â phenyl</td><td class="colsep0 rowsep0" align="char" char="Â±">â0.43 Â± 0.59</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7</b> â <b>4</b></td><td class="colsep0 rowsep0" align="left">3-pyridinyl â phenyl</td><td class="colsep0 rowsep0" align="char" char="Â±">â1.44 Â± 0.45</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4-</b>Pyr<sup>b</sup> â <b>4</b></td><td class="colsep0 rowsep0" align="left">4-pyridinyl â phenyl</td><td class="colsep0 rowsep0" align="char" char="Â±">â0.64 Â± 0.16</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">4-pyridinyl analog of <b>6</b> or <b>7</b>.</p></div></div></div><div class="NLM_p">Pyridinyl replacements <b>6</b> and <b>7</b> for the phenyl group were also considered. Initial structure building with BOMB indicated that constructive interaction of the pyridinyl nitrogen atoms with Lys581 might be possible with NâN contacts of 3.32 Ã for <b>6</b> (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>) and 3.72 Ã for <b>7</b>. Energy minimizations with the force field find a more favorable interaction with the protein by 3.0 kcal/mol for <b>6</b> than <b>7</b>; however, this is offset by a 3.4 kcal/mol conformational penalty for placement of the pyridine nitrogen atom syn rather than anti to the carbonyl oxygen in <b>6</b>, which is needed for the hydrogen bond with Lys581. B3LYP/6-31G(d) optimizations for the corresponding syn and anti conformers of <i>N</i>-(pyridine-2-yl)acetamide indicate an even higher conformational penalty to achieve the syn conformer of <b>6</b>, 8.98 kcal/mol.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> For both <b>6</b> and <b>7</b>, the azine ring is now basically in-plane with the amide group, which replaces the cationâÏ interaction found for <b>4</b> with Lys581 (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>) with a hydrogen bond (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>). Results of FEP calculations were also obtained for conversion of <b>6</b> and <b>7</b> to <b>4</b> with the predictions that the pyridinyl analogs would be weaker binders for JAK2 JH2 (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). The FEP calculations for <b>6</b> considered multiple starting conformers; both conformers with the pyridine nitrogen syn or anti to the carbonyl group were stable, and the results indicated that the syn conformer is favored by 0.7 Â± 0.2 kcal/mol. The compounds were synthesized and the <i>K</i><sub>d</sub> results of 30.5 (<b>6</b>) and 12.3 (<b>7</b>) Î¼M confirmed the weaker binding than for <b>4</b> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). These relatively simple changes illustrate well the challenges in making even qualitatively correct predictions for differences in proteinâligand binding owing to the complexities of the inter- and intramolecular energetics and solvation.</div><figure id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/medium/jm0c00192_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Modeled structure illustrating the potential hydrogen bonding between Lys581 and the amide carbonyl oxygen atom and the pyridinyl nitrogen atom for 2-pyridinyl analogs of <b>3</b> and <b>4</b> such as <b>6</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00192&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The next step in the design was to consider substituents in the para position of the phenyl ring of <b>4</b> that would extend further toward Thr555 and Arg715 (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). In view of the placement of the terminal phosphate of ATP in this region (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>), substituents terminating in an anionic group seemed desirable. Modeling was performed for various carboxyalkyl and carboxyalkoxy alternatives with the conclusion that carboxymethoxy should be particularly promising; the OCH<sub>2</sub> fragment should be in-plane with the phenyl ring, as in anisole, and the carboxylate group should be extended to form hydrogen bonds with Thr555. For comparison, both carboxyethyl and carboxyethoxy were expected to incorporate a gauche dihedral angle and be less extended. Thus, the carboxymethoxy analogs <b>9</b> and <b>10</b> were synthesized and assayed along with <b>8</b>, the methyl-ester precursor of <b>10</b>. The results were gratifying with a ca. 8-fold boost in binding strength with JAK2 JH2 for <b>9</b> (0.64 Î¼M) and <b>10</b> (0.57 Î¼M) compared to <b>4</b>. The corresponding result for the ester <b>8</b> (1.9 Î¼M) confirmed the importance of an anionic terminus for the substituent although it is not obvious intrinsically that a carboxylate group would yield any improvement in binding since it would be well hydrated unbound in water. <b>11</b>, the meta-substituted isomer of <b>9</b>, was also synthesized and showed reduced binding with a <i>K</i><sub>d</sub> of 4.0 Î¼M (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). The modeling in this case predicted that the substituent would likely be directed into the solvent from the upper meta position in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>.</div><div class="NLM_p">It was also possible to obtain crystal structures for the complexes of <b>9</b> and <b>10</b> with JAK2 JH2 at resolutions of 2.06 and 1.71 Ã, respectively. As shown in <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>, the carboxylate group of <b>10</b> is extended, as expected, with a COCC dihedral angle of 170Â°. It is positioned similarly to the terminal phosphate group of ATP in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>, and it forms hydrogen bonds with the backbone NH and side chain OH of Thr555 with lengths of 3.06 and 2.83 Ã. As expanded upon in <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>, the carboxylate group also participates in an extensive network of hydrogen bonds with localized water molecules in the crystal structure including three hydrogen bonds between the carboxylate oxygen atoms and water molecules.</div><figure id="fig6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/medium/jm0c00192_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Rendering from the 1.71 Ã crystal structure for <b>10</b> bound to JAK2 JH2 (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6OBF">6OBF</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00192&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig7" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/medium/jm0c00192_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Illustration of the hydrogen bonding in the vicinity of the carboxylate group of <b>10</b> from the 1.71 Ã crystal structure with JAK2 JH2 (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6OBF">6OBF</a>). Red spheres represent oxygen atoms of localized water molecules.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00192&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In viewing the crystal structure for <b>10</b>, it was believed that a 1 Ã additional extension of the carboxylate group toward Arg715 to form a salt bridge would likely be beneficial for both binding and selectivity. Addition of a methylene group was not expected to be optimal based on the modeling noted above. Instead, replacement of the methoxy linking group with a five-membered heterocycle yielded computed structures that delivered the desired contact. The added rigidity of the heterocycle was also viewed as a desirable feature although there were concerns about unfavorable disruption of the hydrogen-bonding networks (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>). More than 30 alternative structures were built with BOMB and visually inspected; our preference was for heterocycles with multiple hydrogen-bonding sites. The syntheses brought new challenges; however, several illustrative compounds were prepared starting with the oxazole-4-carboxylic acid <b>12</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). The FP assay result of 0.346 Â± 0.034 Î¼M showed the validity of the design and yielded the first compound that is a clearly stronger binder for JAK2 JH2 than JNJ7706621 (<b>1</b>). Again, we were fortunate to obtain a crystal structure for this complex at a resolution of 2.03 Ã (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>). The shortened contact with Arg715 was achieved with an OÂ·Â·Â·N distance of 3.56 Ã, while maintaining the hydrogen bonds with the NH (2.98 Ã) and side chain OH (2.70 and 3.58 Ã) of Thr555. There are again three water molecules hydrogen-bonded to the carboxylate group with OÂ·Â·Â·O separations of 2.28, 2.82, and 3.09 Ã. The water molecule with the 2.28 Ã contact is part of a striking triangle of hydrogen-bonded water molecules that bridges between the carboxylate groups of the ligand and Asp699. The central water molecule in the cluster is also hydrogen-bonded to the nitrogen atom of the oxazole at a distance of 2.79 Ã. Furthermore, the cluster is in van der Waals contact with a meta carbon atom of Phe594 with separations of 3.37, 3.91, and 4.27 Ã. Thus, this interesting water trimer is providing a carefully arranged molecular cushion between the ligand and JAK2 JH2 residues Asp699 and Phe594.</div><figure id="fig8" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/medium/jm0c00192_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0008.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Rendering from the 2.03 Ã crystal structure for <b>12</b> bound to JAK2 JH2 (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6OCC">6OCC</a>). Red spheres represent oxygen atoms of localized water molecules.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00192&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Among additional five-membered heterocycles as linking groups, the furan (<b>13</b>) and 1,3,4-oxadiazole (<b>14</b>) analogs of <b>12</b> were synthesized. These were expected to be less robust binders for JAK2 JH2 than the oxazole <b>12</b>. First, a furan-type oxygen atom is a weak hydrogen-bond acceptor, and its replacement of the oxazole nitrogen atom in <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a> would weaken the link to the water cluster. Furthermore, furan and thiophene are the most lipophilic five-membered heteroaromatic molecules with octanol/water log <i>P</i> values of 1.34 and 1.81 versus, for example, 0.12 for oxazole, 0.08 for isoxazole, and 0.44 for thiazole.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> Finally, the oxazole oxygen atom in the crystal structure for <b>12</b> (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>) does not participate in a hydrogen bond, and in fact, there is no room for a neighboring water molecule. Thus, if the oxadiazole ring in <b>14</b> is oriented as for <b>12</b> in <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>, there would be a significant penalty for desolvation of one of the nitrogen atoms. The qualitative expectations were borne out by the measured <i>K</i><sub>d</sub> values of 0.44 and 7.0 Î¼M for <b>13</b> and <b>14</b> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>).</div><div class="NLM_p last">From viewing the crystal structure for <b>10</b> (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a> and <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a>) and modeling with BOMB, the (<i>E</i>)-cinnamic acid analog <b>15</b> was also suggested with the vinyl group replacing the OCH<sub>2</sub> linker. This compound was synthesized and turned out to be a somewhat better binder than <b>10</b> with a <i>K</i><sub>d</sub> of 0.37 Î¼M. The improvement can be attributed to the greater rigidity of <b>15</b> stemming from removal of one torsional degree of freedom. It is clear from the modeling of <b>15</b> and the crystal structures of ATP, <b>10</b>, and <b>12</b> bound to the JH2 domain in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>, <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>, andÂ <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a> that the terminal carboxylate group in the designed compounds overlaps closely with the terminal phosphate of ATP; in all cases, hydrogen bonds are formed between these groups and both the hydroxyl group and backbone NH of Thr555.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> JH2/JH1 Selectivity</h3><div class="NLM_p">The next step was to study the selectivity of the reference compound <b>1</b> and the potent new compounds <b>10</b>, <b>12</b>, and <b>15</b> toward wild-type JAK2 JH1 and JH2 and the V617F JH2 variant. <i>K</i><sub>d</sub> results were obtained in all cases via fluorescence polarization using the same tracer.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> As listed in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>, the pan-JAK binder JNJ7706621 (<b>1</b>) shows no significant selectivity in the FP assays toward the three proteins, which is in accord with prior reports and expectations from crystallography (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>).<a onclick="showRef(event, 'ref8 ref9 ref18'); return false;" href="javascript:void(0);" class="ref ref8 ref9 ref18">(8,9,18)</a> However, as designed, compound <b>10</b> exhibits strong (ca. 75-fold) selectivity for binding the wild-type or V617F JH2 pseudokinase domain over the JH1 kinase domain. For the oxazole <b>12</b> and cinnamic acid <b>15</b>, there are also ca. 20-fold preferences for binding wild-type JH2 over the kinase domain, and there are again modest differences in binding for wild-type JH2 and the V617F variant.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Binding Affinities (<i>K</i><sub>d</sub>, Î¼M) from the FP and MST Assays for JAK2 Domains</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="char" char="." /></col><col align="char" char="Â±" /></col><col align="char" char="Â±" /></col><col align="char" char="Â±" /></col><col align="char" char="Â±" /></col><col align="char" char="Â±" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center" char=".">Â </th><th class="colsep0 rowsep0" align="center" char="Â±">JH1</th><th class="colsep0 rowsep0" align="center" char="Â±">JH2</th><th class="colsep0 rowsep0" align="center" char="Â±">JH2</th><th class="colsep0 rowsep0" align="center" char="Â±">V617F JH2</th><th class="colsep0 rowsep0" align="center" char="Â±">V617F JH2</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center" char=".">compound</th><th class="colsep0 rowsep0" align="center" char="Â±">FP</th><th class="colsep0 rowsep0" align="center" char="Â±">FP</th><th class="colsep0 rowsep0" align="center" char="Â±">MST</th><th class="colsep0 rowsep0" align="center" char="Â±">FP</th><th class="colsep0 rowsep0" align="center" char="Â±">MST</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>1</b></td><td class="colsep0 rowsep0" align="char" char="Â±">0.67 Â± 0.18</td><td class="colsep0 rowsep0" align="char" char="Â±">0.46 Â± 0.12</td><td class="colsep0 rowsep0" align="char" char="Â±">0.49 Â± 0.08</td><td class="colsep0 rowsep0" align="char" char="Â±">0.60 Â± 0.09</td><td class="colsep0 rowsep0" align="char" char="Â±">0.82 Â± 0.12</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>10</b></td><td class="colsep0 rowsep0" align="char" char="Â±">42.3 Â± 2.3</td><td class="colsep0 rowsep0" align="char" char="Â±">0.57 Â± 0.03</td><td class="colsep0 rowsep0" align="char" char="Â±">0.31 Â± 0.03</td><td class="colsep0 rowsep0" align="char" char="Â±">0.54 Â± 0.13</td><td class="colsep0 rowsep0" align="char" char="Â±">0.40 Â± 0.05</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>12</b></td><td class="colsep0 rowsep0" align="char" char="Â±">6.6 Â± 0.9</td><td class="colsep0 rowsep0" align="char" char="Â±">0.35 Â± 0.03</td><td class="colsep0 rowsep0" align="char" char="Â±">0.29 Â± 0.07</td><td class="colsep0 rowsep0" align="char" char="Â±">0.48 Â± 0.14</td><td class="colsep0 rowsep0" align="char" char="Â±">0.43 Â± 0.03</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>15</b></td><td class="colsep0 rowsep0" align="char" char="Â±">6.0 Â± 1.0</td><td class="colsep0 rowsep0" align="char" char="Â±">0.37 Â± 0.01</td><td class="colsep0 rowsep0" align="char" char="Â±">0.20 Â± 0.05</td><td class="colsep0 rowsep0" align="char" char="Â±">0.18 Â± 0.08</td><td class="colsep0 rowsep0" align="char" char="Â±">0.23 Â± 0.03</td></tr></tbody></table></div></div><div class="NLM_p">In addition, eight <i>K</i><sub>d</sub> values were determined using microscale thermophoresis.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> Optimization of this assay was possible for the wild-type JH2 and JH2 V617F domains, but an adequate signal-to-noise ratio could not be obtained for JH1. MST was found to give similar <i>K</i><sub>d</sub> values as FP (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>) although some differences could be expected in view of the variations in optimized buffer compositions. With MST, compounds <b>10</b> and <b>12</b> are found to have identical <i>K</i><sub>d</sub> values within the error limits for JAK2 JH2 near 0.30 Î¼M, which improves on the 0.49 Î¼M result for <b>1</b>. The binding constants for JAK2 JH2 and the V617F variant are again close. Finally, compound <b>15</b> is found with MST to be the most tenacious binder with <i>K</i><sub>d</sub> values at the 200 nM level for both the JH2 and V617F JH2 domains.</div><div class="NLM_p last">At this point, the primary goals were achieved. Compounds were discovered that show good selectivity for binding to the JAK2 JH2 domain in preference to the JH1 kinase domain and that were at least as potent as the non-selective JNJ7706621 (<b>1</b>), which has previously been reported from alternative binding measurements to give <i>K</i><sub>d</sub> values of 0.106 and 0.094 Î¼M.<a onclick="showRef(event, 'ref9 ref18'); return false;" href="javascript:void(0);" class="ref ref9 ref18">(9,18)</a></div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Results with Full-Length Wild-Type and V617F JAK2</h3><div class="NLM_p">The next step was to test compounds <b>1</b> and <b>10</b> for their influence on the autophosphorylation of wild-type and the V617F JAK2 mutant. These compounds were chosen to contrast the non-selective JH1/JH2 binder <b>1</b> with the JH2-selective <b>10</b>. The expectation was that <b>1</b> would be an inhibitor of the JH1-based kinase activity for both the WT and variant proteins, while the outcome with <b>10</b> was unclear. Although the mutational studies suggested that displacement of ATP from the JH2 domain by selective binders could deactivate V617F JAK2,<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> the JH2 binders may also be viewed as potential ATP surrogates leading to no effect. Since activation of kinase activity with some mutations was also observed, this outcome is also possible with the present compounds.</div><div class="NLM_p">As detailed in the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>, full-length cDNAs encoding human WT and V617F JAK2 (residues 1â1132) were amplified by PCR and cloned into an expression plasmid, and the JAK2 constructs were expressed in HEK293T cells grown at 37 Â°C. The transfected cells were lysed, and the JAK2 proteins were immunoprecipitated from the supernatant and used to measure kinase activity by treatment with [Î³-<sup>32</sup>P]ATP in the presence or absence of increasing concentrations of <b>1</b> and <b>10</b>. The autophosphorylation reaction was stopped after 15 min at 30 Â°C, and the extent of phosphorylation was gauged by autoradiography. Details of the protocols were worked out with compound <b>1</b> to establish a good signal-to-noise ratio and reproducibility in triplicate experiments.</div><div class="NLM_p">Although <b>1</b> has previously been established to bind to the JH1 and JH2 domains of JAK2,<a onclick="showRef(event, 'ref8 ref9 ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref8 ref9 ref17 ref18">(8,9,17,18)</a> its status as an inhibitor has not been determined until now. As shown in <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>A, we do find that <b>1</b> is an inhibitor of the autophosphorylation of both WT and V617F JAK2. There are steady, dose-dependent decreases in phosphorylation in the presence of <b>1</b>. It was possible to quantify the gel bands from the phosphor autoradiography to yield IC<sub>50</sub> values of 2.96 Â± 0.41 Î¼M for <b>1</b> with WT JAK2 and 11.05 Â± 3.21 Î¼M with V617F JAK2. These results seem consistent with the <i>K</i><sub>d</sub> of 0.67 Î¼M for <b>1</b> with the JH1 kinase domain of JAK2 (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>).</div><figure id="fig9" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/medium/jm0c00192_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0009.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. In vitro [Î³-<sup>32</sup>P]ATP kinase activity assay of immunoprecipitated full-length JAK2. Autoradiography of autophosphorylated JAK2 wild-type and V617F mutant exposed to different concentrations of (A) <b>1</b> and (B) <b>10</b>. Shown is representative data from experiments performed in triplicate.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00192&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">In contrast, with the JH2-selective <b>10</b>, higher concentrations are needed to see clear effects (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>B). Inhibition of WT JAK2 is apparent above ca. 10 Î¼M, and an IC<sub>50</sub> of 46.53 Â± 4.74 Î¼M could be assigned. The weak inhibition of the kinase activity is consistent with the <i>K</i><sub>d</sub> of 42 Î¼M that was found for <b>10</b> with JAK2 JH1 (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). Thus, it is expected that <b>10</b> is binding to the JH2 domain at low concentrations, but it has negligible effects on the kinase kinetics until significant binding to the ATP site in the JH1 domain sets in at concentrations above 10 Î¼M. For V617F JAK2, the effects of <b>10</b> are further diminished with little inhibition apparent below 25 Î¼M, with an IC<sub>50</sub> in the 100â200 Î¼M range. Although <b>10</b> is expected to be binding to the ATP site in the pseudokinase JH2 domain at low concentrations (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>), it is not affecting the kinase kinetics. This result contrasts the expectations from the mutational studies<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> although it is consistent with an alternative view of JH2-selective <b>10</b> as an ATP surrogate. The present finding for the one compound, <b>10</b>, does not imply that all JH2-selective binders will yield qualitatively similar results. If sufficient structural variety in JH2 binders can be explored, then there will be a range of structural effects that can be expected to provide alternative outcomes in the present or in cell-based assays.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i14">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_84341" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_84341" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">Janus kinases are intracellular, non-receptor tyrosine kinases that are activated by cytokine binding to cytokine receptors resulting in stimulation of the JAK-STAT signaling pathway. Aberrant activation of this signaling pathway is associated with numerous immune disorders and cancer. The present focus is on selective reduction of the hyperactivation that occurs with the V617F mutation in the JH2 domain of JAK2 kinase, which is the primary source of myeloproliferative neoplasms. The initial goal has been to discover small molecules that selectively bind to the ATP binding site in the JH2 pseudokinase domain of JAK2 in preference to the ATP binding site in the JH1 kinase domain. Starting from the non-selective, multikinase inhibitor JNJ7706621 (<b>1</b>), new compounds were systematically designed, synthesized, and assayed that ultimately yielded compounds such as <b>10</b>, <b>12</b>, and <b>15</b>, which bind more strongly to JAK2 JH2 than <b>1</b> and that show a substantial preference for binding JAK2 JH2 over JH1. The design was particularly challenging owing to the polarity of the ATP binding sites in the area that is amenable to differentiation of binding to JAK2 JH1 and JH2, which is proximal to the region affected structurally by the V617F mutation. The ability to obtain multiple crystal structures for complexes with JAK2 JH2 was most valuable in monitoring the progress of the designs, and the ability to perform fluorescence polarization assays for binding to all three component proteins, JAK2 JH1, JH2, and V617F, was essential. Initial further exploration of <b>1</b> and <b>10</b> was pursued by examining the effects of the compounds on autophosphorylation of full-length JAK2 and V617F JAK2. <b>1</b> was established to be a low-Î¼M inhibitor of both enzymes, while <b>10</b> showed little effects on the kinase activity below concentrations of ca. 25 Î¼M. The interpretation is that in contrast to expectations from mutational studies,<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a><b>10</b> binds well to the ATP site of the pseudokinase JH2 domain and performs much of the same function as ATP. However, it may also be expected that alternative JH2 ligands may perturb the protein structure in different manners and lead to possible deactivation or activation of the oncogenic V617F JAK2.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_42773" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_42773" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Generation of Recombinant Baculoviruses</h3><div class="NLM_p last">The isolated JH1 and JH2 domains of human JAK2 were expressed in baculovirus-infected Sf9 insect cells and purified similarly to the previously reported procedure.<a onclick="showRef(event, 'ref9 ref17'); return false;" href="javascript:void(0);" class="ref ref9 ref17">(9,17)</a> The two JH2 domain constructs contain residues 536â812 (with either mutations W659A, W777A, F794H or mutations W777A, F794H, V617F) followed by a C-terminal thrombin cleavage site and 6ÃHis-tag. The reported JH1 domain construct included an N-terminal 6ÃHis-tag followed by a TEV cleavage site and residues 840â1132. Recombinant bacmid and baculoviruses were generated using the Bac-to-Bac baculovirus expression system (Invitrogene). DH10Bac competent cells were transformed with recombinant pFastBac plasmid containing the gene of interest to generate the recombinant expression bacmid. P1, P2, and P3 baculovirus stocks were produced according to the manufacturerâs instructions. Sf9 cells were grown in HyClone SFX-Insect cell culture media (GE Healthcare) at 27 Â°C.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> JAK2 Protein Expression and Purification</h3><div class="NLM_p last">Sf9 cells were grown in HyClone SFX-Insect cell culture media to a density of 2.5â4.0 Ã 10<sup>6</sup> cells/mL followed by transfection with P3 baculovirus stock. After incubation for 48 h at 27 Â°C, cells were harvested and separated from the supernatant by centrifugation (4000 rpm, 30 min). Purification of the JH1 and JH2 domains of JAK2 was performed in an identical manner. Cell pellets were resuspended in lysis buffer composed of 20 mM Tris pH 8.0, 500 mM NaCl, 20% glycerol, 0.25 mM TCEP, and cOmplete EDTA-free protease inhibitor (Roche). Cells were lysed by sonication followed by pressure homogenization using an Emulsiflex cell disruptor (Avestin). Lysate was separated from cell debris by centrifugation (45 min, 16,500 rpm). Ni-NTA agarose beads (Qiagen) were added in batch mode and incubated for 2 h at 4 Â°C. Beads were washed with lysis buffer containing 10 mM imidazole, and JAK2 protein was eluted with lysis buffer containing 200 mM imidazole. The eluate was dialyzed overnight at 4 Â°C using a MWCO 3.5 kDa Slide-A-Lyzer dialysis cassette (Thermo Fisher Scientific) against a low salt dialysis buffer composed of 20 mM Tris pH 8.0, 25 mM NaCl, 20% glycerol, and 0.25 mM TCEP. The dialysis product was filtered through a 0.45 Î¼m membrane and loaded onto a pre-equilibrated Mono Q HR 16/19 column (GE Healthcare) linked to an ÃKTA pure protein purification system (GE Healthcare). Protein was eluted by applying a linear gradient starting with dialysis buffer and ending with dialysis buffer containing 500 mM NaCl. JAK2 fractions were pooled and applied to a Superdex 75 10/300 (GE Healthcare) pre-equilibrated with a buffer composed of 20 mM Tris pH 8.0, 100 mM NaCl, 10% glycerol, and 1.0 mM TCEP. Purified protein was aliquoted, flash-frozen in liquid nitrogen, and stored at â80 Â°C.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Competitive FP assay<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a></h3><div class="NLM_p last">In a flat black bottom 96 well plate (Corning), 200 Î¼L of FP buffer (20 mM Tris-HCl pH 8.0, 150 mM NaCl, 20% glycerol, 0.5 mM TCEP, 0.01% Tween 20) was added to column 1 (blank), 150 Î¼L to column 2, and 140 Î¼L to columns 3â12. Ten microliters of 2.96 Î¼M JAK2-JH2 WT (3.52 Î¼M for JAK2-JH2-VF and 6.93 Î¼M for JAK2-JH1) was added to columns 3â12 followed by the addition of 2 Î¼L of DMSO to columns 1â3. Two microliters of inhibitor in DMSO at different concentrations was added from column 4 to 12. Fifty microliters of 24 nM tracer were added to columns 2â12. Fluorescence polarization was measured at Î»<sub>exc</sub> = 485 Â± 20 nm and Î»<sub>em</sub> = 535 Â± 25 nm for 1 h. Experiments were carried out in quadruplicate with three independent experiments. Data were analyzed by a least-squares non-linear fit, generated using Prism 7 in order to determine the compoundâs IC<sub>50</sub>. <i>K</i><sub>d</sub> values for each inhibitor were calculated using the following equation based on the IC<sub>50</sub>, <i>K</i><sub>d</sub> of the tracer (<i>K</i><sub class="stack">d</sub><sup class="stack">t</sup>), and total (<i>L</i><sub>t</sub>) and bound (<i>L</i><sub>b</sub>) tracer, as well as total protein concentration (<i>P</i><sub>t</sub>).<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a><span class="NLM_disp-formula" id="ueq1"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/medium/jm0c00192_m001.gif" alt="" id="_i19" /></img></span></div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Microscale Thermophoresis Assay</h3><div class="NLM_p last">MST measurements were performed with a Monolith NT.115Pico device (NanoTemper Technologies). JH2 domain protein (triple mutant W659A, W777A, F794H or triple mutant W777A, F794H, V617F) was fluorescently labeled with the His-tag labeling kit RED-tris-NTA 2nd generation. All dilutions were prepared with a buffer composed of 20 mM HEPES pH 8.0, 150 mM NaCl, 5% glycerol, and 0.05% Tween. Protein was labeled by incubating a mixture of 150 nM protein and 50 nM dye for 30 min at ambient temperature. MST measurements were performed with protein and dye concentrations adjusted to 30 and 10 nM, respectively. The serial dilution of <b>1</b> and <b>10</b> ranged from 160 to 0.00488 Î¼M, for <b>12</b> from 20 to 0.00488 Î¼M, and for <b>15</b> from 20 to 0.00061 Î¼M. All measurement samples contained a constant DMSO concentration of 3%. Measurements were performed with standard capillaries (mutant W659A, W777A, F794H) or premium capillaries (mutant W777A, F794H, V617F), medium MST power, and 5% excitation power at ambient temperature. All measurements were performed in triplicate and were analyzed with the MO.Affinity Analysis software (NanoTemper). All doseâresponse curves are displayed in <a href="/doi/suppl/10.1021/acs.jmedchem.0c00192/suppl_file/jm0c00192_si_001.pdf" class="ext-link">Figure S2</a>.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Protein Crystallization</h3><div class="NLM_p last">Crystals of JAK2 JH2 were grown by hanging drop vapor diffusion at 4 Â°C. Crystals were prepared by adding 1 Î¼L of protein (6 mg/mL) in a solution composed of 20 mM Tris pH 8.0, 100 mM NaCl, 10% glycerol, and 1 mM TCEP to 1 Î¼L of reservoir solution composed of 0.1 M Tris pH 8.0, 0.2 M sodium acetate, 12â24% PEG4000, and 1 mM TCEP. Crystallization was induced by streak seeding, and crystals grew to full size within a week. Complexes of JH2 with small-molecule ligands were prepared by ligand soaking. Therefore, crystals were transferred into a solution of 0.1 M Tris pH 8.0, 0.2 M sodium acetate, 22% PEG4000, 1 mM TCEP, 8% DMSO, and 4 mM of the respective small-molecule ligand. After 24 h of incubation, crystals were briefly exposed to a cryobuffer (0.1 M Tris pH 8.0, 0.2 M sodium acetate, 22% PEG4000, 1 mM TCEP, 8% DMSO, 4 mM of the respective small-molecule ligand, and 20% glycerol) and flash-frozen in liquid nitrogen.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Collection of X-ray Diffraction Data and Data Processing</h3><div class="NLM_p last">Crystal structures for complexes of five of the new compounds (<b>3</b>, <b>4</b>, <b>9</b>, <b>10</b>, <b>12</b>) with JAK2 JH2 were obtained at 1.7â2.1 Ã resolution. All data sets were collected at 100 K. Data sets of JH2 in complex with <b>4</b>, <b>10</b>, and <b>12</b> were collected in-house on a Rigaku MicroMax-007HF X-ray generator (Cu rotating anode; Î» = 1.54 Ã) with a Dectris Pilatus 200K detector. The data set of JH2 in complex with <b>3</b> was collected at the Advanced Photon Source (APS; Argonne, Illinois, USA) on beamline 24-ID-E with a Dectris Eiger 16M detector (Î» = 0.97918 Ã). The data set of JH2 in complex with <b>9</b> was collected at the APS beamline 24-ID-C with a Dectris Pilatus 6M detector (Î» = 0.97910 Ã). Data sets of <b>4</b>, <b>10</b>, and <b>12</b> were indexed, integrated, and scaled with HKL2000.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> Data sets of <b>3</b> and <b>9</b> were indexed, integrated, and scaled with XDS.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> Diffraction data and refinement statistics are listed in <a href="/doi/suppl/10.1021/acs.jmedchem.0c00192/suppl_file/jm0c00192_si_001.pdf" class="ext-link">Table S1</a>. Refinement procedures are also provided along with omit maps in <a href="/doi/suppl/10.1021/acs.jmedchem.0c00192/suppl_file/jm0c00192_si_001.pdf" class="ext-link">Figure S3</a>.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Cloning and Expression of Full-Length JAK2</h3><div class="NLM_p last">Full-length cDNA encoding human JAK2 (residues 1â1132) wild-type (NP_001309123) and V617F mutant with C-terminal FLAG-Tag (DYKDDDDK) were amplified by PCR and subcloned into a modified pOptiVec expression vector (Invitrogen). JAK2 constructs were expressed in HEK293T cells grown at 37 Â°C at 5% CO<sub>2</sub> in DMEM (Gibco) supplemented with 10% (v/v) FBS (Gibco) and 1% (v/v) penicillinâstreptomycin (Gibco). HEK293T cells were transiently transfected using Lipofectamin 2000 (Invitrogen) according to manufacturerâs instructions. At 36 h post-transfection, cells were washed twice with ice-cold PBS and lysed with ice-cold lysis buffer (50 mM HEPES pH 7.4, 150 mM NaCl, 10% (v/v) glycerol, 1% (v/v) Triton-X 100, 1 mM EDTA, 1 mM EGTA, 25 mM NaF, 1.5 mM MgCl<sub>2</sub>, 1.0 mM Na<sub>3</sub>VO<sub>4</sub>, Roche cOmplete mini EDTA-free protease inhibitor cocktail mixture). The lysate was centrifuged (20 min, 13,000Ã<i>g</i>), and the lysate supernatant was flash-frozen in liquid nitrogen and stored at â80 Â°C.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> Immunoprecipitation and In Vitro Kinase Assay</h3><div class="NLM_p last">JAK2 protein was immunoprecipitated from the lysate supernatant by adding an anti-FLAG M2 antibody (Sigma-Aldrich, no. F1804) and protein G-PLUS agarose (Santa Cruz Biotechnology, no. sc-2002) followed by incubation overnight while rocking at 4 Â°C. Immunoprecipitates were washed four times with wash buffer (50 mM HEPES pH 7.4, 150 mM NaCl, 10% (v/v) glycerol, 0.1% (v/v) Triton-X 100, 1 mM EDTA, 1 mM EGTA, 25 mM NaF, 1.5 mM MgCl<sub>2</sub>, 1.0 mM Na<sub>3</sub>VO<sub>4</sub>, Roche cOmplete mini EDTA-free protease inhibitor cocktail mixture) and once with kinase reaction buffer (50 mM HEPES pH 7.4, 150 mM NaCl, 0.5 mM DTT, 5 mM MnCl<sub>2</sub>, 5% DMSO). The [Î³-<sup>32</sup>P]ATP in vitro kinase activity assay was performed based on previously published protocols.<a onclick="showRef(event, 'ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref30 ref31">(30,31)</a> Washed immunoprecipitates were divided into equal parts and centrifuged, their residual solvent was removed, and the resulting pellets were resuspended in 25 Î¼L kinase reaction buffer containing different concentrations of <b>1</b> or <b>10</b> followed by incubation for 1 h at 4 Â°C. The autophosphorylation reaction of JAK2 was initiated by adding 25 Î¼L of phosphorylation mixture consisting of kinase reaction buffer supplemented with 10 Î¼M cold ATP and 2 Î¼Ci (for JAK2 V617F) or 5 Î¼Ci (for JAK2 wild type) of [Î³-<sup>32</sup>P]ATP (EasyTides, PerkinElmer) per reaction. The mixture was allowed to react for 15 min at 30 Â°C (within the linear range of kinase activity) and stopped by putting on ice and adding 18 Î¼L of reducing Laemmli sample buffer (4Ã). Samples were heated at 95 Â°C for 5 min and run by 7.5% SDS-PAGE. Gels were rocked in a solution of 10% glycerol, 20% ethanol for 30 min, dried with a vacuum drier, and autoradiographed using a phosphor imager. To calculate IC<sub>50</sub> values, phosphor autoradiography was quantified using ImageJ,<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> and curves were plotted with Prism 8.0 (GraphPad Software Inc., La Jolla, CA).</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> General Synthetic Methods</h3><div class="NLM_p last">All purchased compounds were used as received from vendors without further purification. Reactions were conducted under a nitrogen atmosphere and monitored by thin-layer chromatography on Merck silica gel plates pre-coated with fluorescent indicator F254. Visualization of plates was achieved with UV light or potassium permanganate stain. Mass analysis of intermediates was done with an Agilent 6120 quadrupole LC/MS instrument via electrospray ionization. Chromatographic purification was performed with a Teledyne ISCO CombiFlash automated system employing RediSep Normal Phase Silica (particle size, 35â70 Î¼m; pore size, 60 Ã) or RediSep Gold Normal Phase Silica (particle size, 20â40 Î¼m; pore size, 60 Ã) disposable cartridge columns. RediSep Gold C18 reusable columns (particle size, 20â40 Î¼m (spherical); pore size, 100 Ã) were employed for reversed-phase chromatography. Preparative reverse phase HPLC was performed using a Shimadzu Prominence system equipped with LC-20AP pumps, CBM-20A Communications BUS module, an SPD-20A UV/vis detector, SIL-10AP autosampler, FRC-10A fraction collector, and a Waters SymmetryPrepTM C8, 19 Ã 300 mm column (particle size, 7 Î¼m; pore size, 100 Ã) with a gradient of 0.1% formic acid in water and 0.1% formic acid in acetonitrile as the mobile phase. <sup>1</sup>H and <sup>13</sup>C NMR spectra were recorded on Agilent DD<sub>2</sub> 400 MHz, DD<sub>2</sub> 500 MHz, or DD<sub>2</sub> 600 MHz instruments. The IR spectrum was recorded on a Thermo Scientific Nicolet 6700 FT-IR spectrometer using a thin film of a solid sample. HRMS analysis of final products was done on a Waters Xevo QTOF with a Z-spray electrospray ionization source. The purity of all assayed compounds was determined on a Shimadzu Prominence HPLC equipped with an Agilent Poroshell 120 SB-C18 2.7 Î¼m column, using 0.1% TFA in water and 0.1% TFA in acetonitrile as the mobile phase. The compounds (<b>2</b>â<b>4</b>, <b>6</b>â<b>10</b>, <b>12</b>, <b>14</b>, <b>15</b>) were â¥95% pure by analytical HPLC, while the purities of three non-key compounds, <b>5</b>, <b>11</b>, and <b>13</b>, were 94, 91, and 90%, respectively. The HPLC traces of all compounds showed no minor peaks above 3%. It is possible that the HPLC results may be affected by formation of dimers of the carboxylic acids or alternative protonation states. The systematic SAR data and multiple crystal structures reported here demonstrate that the reported binding data for this series is reflective of the compounds themselves. The spectra of all new compounds are provided in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c00192/suppl_file/jm0c00192_si_001.pdf" class="ext-link">Supporting Information</a>.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> Synthesis of Intermediates and Final Compounds</h3><div id="sec4_10_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> Phenyl-<i>N</i>â²-cyano-<i>N</i>-(4-sulfamoylphenyl)carbamimidate (<b>16a</b>)</h4><div class="NLM_p last">Sulfanilamide (4.0 g, 23.2 mmol) was combined with diphenyl cyanocarbonimidate (5.5 g, 23.2 mmol) in anhydrous tetrahydrofuran (26 mL, 0.9 M). The reaction mixture was stirred at reflux for 24 h and afterward was concentrated under reduced pressure. The residue was suspended in dichloromethane, filtered, washed with dichloromethane, and dried overnight to provide the title product as a pale white solid (6.2 g, 84% yield). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 11.12 (s, 1H), 7.83 (d, <i>J</i> = 8.5 Hz, 2H), 7.66 (d, <i>J</i> = 8.5 Hz, 2H), 7.46 (t, <i>J</i> = 7.8 Hz, 2H), 7.36 (s, 2H), 7.35â7.29 (m, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 151.43, 140.97, 139.12, 129.85, 127.39, 126.51, 126.39, 123.32, 120.79, 112.39. ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup>, 317.1.</div></div><div id="sec4_10_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> Phenyl-<i>N</i>â²-cyano-<i>N</i>-(4-cyanophenyl)carbamimidate (<b>16b</b>)</h4><div class="NLM_p last">Sodium hydride (760 mg (60% dispersion in mineral oil), 19.0 mmol) was added at 0 Â°C to a mixture of 4-aminobenzonitrile (2.2 g, 19.0 mmol) in anhydrous tetrahydrofuran (65 mL, 0.3 M). The mixture was stirred at 60 Â°C for 45 min followed by portion-wise addition of diphenyl cyanocarbonimidate (4.0 g, 17.0 mmol). The reaction was brought to reflux and allowed to run for 24 h. The crude mixture was quenched with methanol, and the solvent was evaporated under reduced pressure. Hot acetone was added until complete dissolution of the residue, at which point hexanes were added until the solution became cloudy. The precipitate was filtered to provide the title product as a white, fluffy solid (3.11 g, 70% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 11.21 (s, 1H), 7.86 (d, <i>J</i> = 8.1 Hz, 2H), 7.66 (d, <i>J</i> = 8.1 Hz, 2H), 7.46 (t, <i>J</i> = 7.6 Hz, 2H), 7.37â7.28 (m, 3H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 157.71, 155.53, 154.16, 132.12, 128.84, 124.11, 123.67, 121.98, 121.78, 120.32, 100.73. ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup>, 263.1.</div></div><div id="sec4_10_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> 4-((5-Amino-1<i>H</i>-1,2,4-triazol-3-yl)amino)benzenesulfonamide (<b>17a</b>)<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a></h4><div class="NLM_p"><b>16a</b> (5.0 g, 15.9 mmol) was suspended in anhydrous tetrahydrofuran (18 mL, 0.9 M), and hydrazine (17.5 mL (1 M in THF), 17.5 mmol) was added at 0 Â°C. The reaction mixture was warmed to room temperature and stirred at reflux for 7.5 h. The mixture was then filtered, washed three times with tetrahydrofuran, and dried overnight to afford the title product as a white solid (3.4 g, 84.5% yield; <a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 11.30 (s, 1H), 9.19 (s, 1H), 7.60 (q, <i>J</i> = 9.0 Hz, 4H), 7.03 (s, 2H), 5.95 (s, 2H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 157.22, 155.50, 145.46, 133.07, 126.72, 114.55. ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup>, 255.1.</div><figure id="sch3" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/medium/jm0c00192_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0013.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Syntheses of <b>17a</b>,<b>b</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00192&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) THF, reflux, and 24 h; (b) NaH, THF, reflux, and 24 h; (c) hydrazine, reflux, and 8â19 h.</p></p></figure></div><div id="sec4_10_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> 4-((5-Amino-1<i>H</i>-1,2,4-triazol-3-yl)amino)benzonitrile (<b>17b</b>)</h4><div class="NLM_p last"><b>16b</b> (1.0 g, 3.8 mmol) was suspended in dry tetrahydrofuran (11 mL, 0.3 M) and combined with hydrazine (7.6 mL (1 M in THF), 7.6 mmol) at 0 Â°C. The reaction mixture was brought to reflux and stirred for 19 h and was then concentrated and purified by column chromatography (DCM/MeOH) to afford a yellow solid (461 mg, 61% yield; <a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 11.38 (s, 1H), 9.40 (s, 1H), 7.60 (q, <i>J</i> = 8.9 Hz, 4H), 5.97 (s, 2H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 155.23, 147.97, 146.51, 133.04, 120.16, 115.49, 99.10. ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup>, 201.1.</div></div><div id="sec4_10_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> Methyl 2-(4-Nitrophenoxy)acetate (<b>18aâ²</b>)<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a></h4><div class="NLM_p last"><i>p-</i>Nitrophenol (1.5 g, 10.8 mmol) was suspended in dry acetonitrile (50 mL, 0.2 M) followed by addition of potassium carbonate (2.98 g, 21.6 mmol). The mixture was allowed to stir until homogenization, methyl-2-bromoacetate (1.0 mL, 10.8 mmol) was added, and stirring continued for 16.5 h at room temperature. The crude mixture was concentrated and diluted with ethyl acetate (50 mL), washed with water (2 Ã 20 mL) and brine (2 x 15 mL), and was dried over anhydrous sodium sulfate. The solvent was evaporated under vacuum to afford the title compound as a white solid (2.0 g, 89% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 8.21 (d, <i>J</i> = 9.3 Hz, 2H), 7.17 (d, <i>J</i> = 9.3 Hz, 2H), 5.01 (s, 2H), 3.71 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 168.47, 162.78, 141.35, 125.78, 115.23, 65.02, 52.01. ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup>, 212.1.</div></div><div id="sec4_10_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> Methyl 2-(4-Aminophenoxy)acetate (<b>18a</b>)<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a></h4><div class="NLM_p">Pd/C (10% Pd basis, 565 mg) was added to a solution of <b>18aâ²</b> (1.85 g, 8.8 mmol) in 1:1 tetrahydrofuran/methanol (50 mL). The reaction mixture was stirred under a hydrogen atmosphere for 20 h. The mixture was then filtered through celite and purified by column chromatography (hexanes/EtOAc) to provide the title compound as an orange solid (1.3 g, 82% yield; <a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 6.64 (d, <i>J</i> = 8.8 Hz, 2H), 6.49 (d, <i>J</i> = 8.8 Hz, 2H), 4.65 (s, 2H), 4.59 (s, 2H), 3.67 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 169.71, 148.93, 143.08, 115.49, 114.75, 65.52, 51.60. ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup>, 182.1.</div><figure id="sch4" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/medium/jm0c00192_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0014.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Syntheses of <b>18a</b>,<b>b</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00192&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) K<sub>2</sub>CO<sub>3</sub>, MeCN, r.t., and 16.5 h; (b) K<sub>2</sub>CO<sub>3</sub>, acetone, reflux, and 5 h; (c) H<sub>2</sub>, Pd/C, THF/MeOH, and 20 h.</p></p></figure></div><div id="sec4_10_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> Methyl 2-(3-Nitrophenoxy)acetate (<b>18bâ²</b>)<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a></h4><div class="NLM_p last">Potassium carbonate (2.0 g, 14.5 mmol) was added to a mixture of <i>m-</i>nitrophenol (1.5 g, 10.8 mmol) in dry acetone (5 mL, 2 M). The mixture was allowed to stir until homogenization, and then, methyl-2-bromoacetate (1.3 mL, 14.1 mmol) was added. The reaction was refluxed for 5 h and was then cooled to room temperature and poured into water (25 mL). The precipitate was filtered and dried in a BÃ¼chner funnel under house vacuum overnight to provide the title compound as a pale white solid (2.2 g, 96% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 7.85 (dd, <i>J</i> = 8.1, 2.0 Hz, 1H), 7.73 (t, <i>J</i> = 2.3 Hz, 1H), 7.59 (t, <i>J</i> = 8.2 Hz, 1H), 7.45 (dd, <i>J</i> = 8.3, 2.5 Hz, 1H), 5.00 (s, 2H), 3.71 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 168.79, 158.15, 148.70, 130.79, 121.96, 116.22, 109.10, 65.01, 52.00. ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup>, 212.1.</div></div><div id="sec4_10_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> Methyl 2-(3-Aminophenoxy)acetate (<b>18b</b>)<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a></h4><div class="NLM_p last">Pd/C (10% Pd basis, 565 mg) was added to a solution of <b>18bâ²</b> (1.85 g, 8.8 mmol) in 1:1 THF/MeOH (50 mL). The reaction mixture was stirred under a hydrogen atmosphere for 20 h. The mixture was then filtered through celite and purified by column chromatography (hexanes/EtOAc) to provide the title compound as a pale white solid (1.5 g, 92% yield; <a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 6.90 (t, <i>J</i> = 8.0 Hz, 1H), 6.19 (d, <i>J</i> = 7.9 Hz, 1H), 6.11 (s, 1H), 6.05 (d, <i>J</i> = 8.1 Hz, 1H), 5.07 (s, 2H), 4.65 (s, 2H), 3.69 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 169.45, 158.62, 150.03, 129.58, 107.53, 101.65, 100.01, 64.31, 51.71. ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup>,182.1.</div></div><div id="sec4_10_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> <i>N</i>-Boc-4-Iodoaniline (<b>18câ³</b>)<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a></h4><div class="NLM_p last">Triethylamine (1.27 mL, 9.1 mmol) was added to a mixture of 4-iodoaniline (1.0 g, 4.6 mmol) and Boc anhydride (997 mg, 4.6 mmol) in isopropanol (23 mL, 0.2 M). The reaction mixture was stirred at room temperature for 14 h and then concentrated under reduced pressure. The crude mixture was purified by column chromatography (hexanes/EtOAc) to provide the title product as a pale yellow solid (908 mg, 62% yield). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) Î´ 7.57 (d, <i>J</i> = 8.8 Hz, 2H), 7.14 (d, <i>J</i> = 8.7 Hz, 2H), 6.43 (s, 1H), 1.51 (s, 9H). <sup>13</sup>C NMR (101 MHz, chloroform-<i>d</i>) Î´ 152.55, 138.31, 137.96, 120.49, 85.86, 81.07, 28.45. HRMS (ESI): calc for [IC<sub>6</sub>H<sub>4</sub>NHCO<sub>2</sub>H + H]<sup>+</sup>, 263.9521; found, 263.9529.</div></div><div id="sec4_10_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> Ethyl 2-(4-((<i>tert</i>-Butoxycarbonyl)amino)phenyl)oxazole-4-carboxylate (<b>18câ²</b>)<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a></h4><div class="NLM_p last">Ethyl 4-oxazole carboxylate (200 mg, 1.4 mmol) was combined with <b>18câ³</b> (452 mg, 1.4 mmol), palladium acetate (32 mg, 0.14 mmol), JohnPhos (85 mg, 0.28 mmol), and cesium carbonate (923 mg, 2.8 mmol) in dry dioxane (4 mL, 0.35 M). The reaction was sealed under a N<sub>2</sub> atmosphere and heated at 110 Â°C for 19 h. The crude mixture was filtered through celite and purified by column chromatography (hexanes/EtOAc) to provide the title product as an off-white solid (141 mg, 30% yield). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 9.76 (s, 1H), 8.86 (s, 1H), 7.91 (d, <i>J</i> = 8.7 Hz, 2H), 7.64 (d, <i>J</i> = 8.7 Hz, 2H), 4.31 (q, <i>J</i> = 7.1 Hz, 2H), 1.49 (s, 9H), 1.30 (t, <i>J</i> = 7.1 Hz, 3H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 161.43, 160.79, 152.57, 145.16, 142.45, 133.55, 127.24, 119.52, 118.07, 79.69, 60.63, 28.07, 14.21. ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup>, 333.1.</div></div><div id="sec4_10_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> Ethyl 2-(4-Aminophenyl)oxazole-4-carboxylate (<b>18c</b>)</h4><div class="NLM_p"><b>18câ²</b> (101 mg, 0.3 mmol) was dissolved in anhydrous dichloromethane (0.8 mL, 0.38 M). The reaction mixture was cooled to 0 Â°C, and trifluoroacetic acid (0.79 mL, 10.3 mmol) was added dropwise under vigorous stirring. The reaction mixture was warmed to room temperature and stirred for 30 min. The pH was then adjusted to â¼8 by adding sodium bicarbonate at 0 Â°C, and the mixture was extracted with ethyl acetate. The combined organic phases were dried over anhydrous sodium sulfate, filtered, and concentrated to provide the product as a gray, shiny solid (71 mg, 100% yield; <a class="ref internalNav" href="#sch5" aria-label="Scheme 5">Scheme 5</a>). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) Î´ 8.19 (s, 1H), 7.91 (d, <i>J</i> = 8.5 Hz, 2H), 6.71 (d, <i>J</i> = 8.4 Hz, 2H), 4.42 (q, <i>J</i> = 7.1 Hz, 2H), 3.98 (s, 2H), 1.40 (t, <i>J</i> = 7.1 Hz, 3H). <sup>13</sup>C NMR (101 MHz, chloroform-<i>d</i>) Î´ 163.26, 161.83, 149.31, 142.92, 134.48, 128.76, 116.76, 114.70, 61.32, 14.51. ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup>, 233.1.</div><figure id="sch5" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/medium/jm0c00192_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0015.jpeg" id="gr14" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of <b>18c</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00192&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) Boc<sub>2</sub>O, Et<sub>3</sub>N, <i>i</i>PrOH, r.t., and 14 h; (b) 10 mol % Pd(OAc)<sub>2</sub>, 20 mol % JohnPhos, Cs<sub>2</sub>CO<sub>3</sub>, dioxane, 110 Â°C, and 19 h; (c) TFA, DCM, 0 Â°C to r.t., and 30 min.</p></p></figure></div><div id="sec4_10_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> Ethyl 5-(4-Aminophenyl)-1,3,4-oxadiazole-2-carboxylate (<b>18d</b>)<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a></h4><div class="NLM_p">Ethyl glyoxylate (1.98 mL (50% solution in THF), 10 mmol) in ethanol (100 mL, 0.1 M) was heated to reflux for 5 min. 4-Aminobenzoic hydrazide (1.5 g, 10 mmol) was then added portion-wise. After refluxing for 10â15 min, condensation was completed, as indicated by MS. The reaction mixture was concentrated under reduced pressure, and the residue was dissolved in dimethylsulfoxide (50 mL). Potassium carbonate (4.2 g, 30 mmol) and iodine (3.0 g, 12 mmol) were added, and then, the reaction mixture was heated at 100 Â°C. At 40 min, MS indicated completion of the cyclization, and the reaction mixture was cooled to room temperature, diluted with 5% Na<sub>2</sub>S<sub>2</sub>O<sub>3</sub> (200 mL), and stirred for 30 min. The resulting suspension was extracted with ethyl acetate (4 Ã 100 mL), washed with brine, and dried over sodium sulfate. Purification by column chromatography (hexanes/EtOAc) followed by recrystallization from hot methanol afforded the title product as a yellow solid (277 mg, 12% yield over two steps; <a class="ref internalNav" href="#sch6" aria-label="Scheme 6">Scheme 6</a>). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 7.71 (d, <i>J</i> = 8.7 Hz, 2H), 6.69 (d, <i>J</i> = 8.7 Hz, 2H), 6.13 (s, 2H), 4.42 (q, <i>J</i> = 7.1 Hz, 2H), 1.35 (t, <i>J</i> = 7.1 Hz, 3H). <sup>13</sup>C NMR (151 MHz DMSO-<i>d</i><sub>6</sub>) Î´ 166.34, 155.26, 154.21, 153.26, 128.84, 113.62, 108.32, 62.65, 13.92. ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup>, 234.1.</div><figure id="sch6" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/medium/jm0c00192_0016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0016.jpeg" id="gr15" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 6. Synthesis of 1<b>8d</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00192&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) EtOH, reflux, and 10 min; (b) K<sub>2</sub>CO<sub>3</sub>, I<sub>2</sub>, DMSO, 100 Â°C, and 40 min.</p></p></figure></div><div id="sec4_10_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> General Procedure A: Synthesis of Phenyl Carbamates <b>19a</b>, <b>e</b>â<b>j</b><a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a></h4><div class="NLM_p last">The appropriate aromatic amine (1.0 equiv) was suspended in a mixture of tetrahydrofuran (2 M) and water (1 M) containing sodium bicarbonate (1.2 equiv for all except <b>19j</b>; 2.2 equiv for <b>19j</b>). The mixture was cooled to 0 Â°C, and a solution of phenyl chloroformate (1.05 equiv) in tetrahydrofuran (1 M) was slowly added to the reaction mixture. The reaction mixture was stirred at 0 Â°C unless otherwise stated until completion as determined by TLC and MS. Then, the reaction mixture was diluted with ethyl acetate (20 mL), washed with water (3 Ã 5 mL) and brine (2 x 5 mL), and dried over sodium sulfate. The solvent was evaporated under reduced pressure to afford the corresponding phenyl carbamate without further purification, unless otherwise stated.</div></div><div id="sec4_10_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> Phenyl Phenyl Carbamate (<b>19a</b>)</h4><div class="NLM_p">Aniline (0.3 mL, 3.3 mmol) was stirred for 40 min with phenyl chloroformate (0.44 mL, 3.5 mmol) as described in <a class="ref internalNav" href="#sec4_10_13" aria-label="General Procedure A">General Procedure A</a>, to afford the title compound as a white solid (700 mg, 99% yield; <a class="ref internalNav" href="#sch7" aria-label="Scheme 7">Scheme 7</a>). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 10.21 (s, 1H), 7.51 (d, <i>J</i> = 7.7 Hz, 2H), 7.43 (t, <i>J</i> = 7.8 Hz, 2H), 7.32 (t, <i>J</i> = 7.8 Hz, 2H), 7.27 (d, <i>J</i> = 7.5 Hz, 1H), 7.23 (d, <i>J</i> = 8.4 Hz, 2H), 7.05 (t, <i>J</i> = 7.3 Hz, 1H).<sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 151.68, 150.50, 138.61, 129.41, 128.87, 125.41, 122.96, 121.95, 118.44. ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup>, 214.1.</div><figure id="sch7" data-index="16" class="article__inlineFigure"><h2 class="fig-label">Scheme 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/medium/jm0c00192_0017.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0017.jpeg" id="gr16" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 7. Syntheses of Phenyl Carbamates <b>19a</b>, <b>c</b>â<b>j</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00192&amp;id=sch7"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) NaHCO<sub>3</sub>, H<sub>2</sub>O/THF, 0 Â°C, and 40 min to 1 h; (b) NaHCO<sub>3</sub>, H<sub>2</sub>O/THF, 0 Â°C to r.t., and 45 h; (c) pyridine, MeCN, r.t., and 2.5 h.</p></p></figure></div><div id="sec4_10_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> Phenyl Pyridin-2-ylcarbamate (<b>19c</b>)</h4><div class="NLM_p last">2-Aminopyridine (311 mg, 3.3 mmol) was combined with phenyl chloroformate (3.1 mL, 24.4 mmol) in the presence of sodium bicarbonate (2.3 g, 27.8 mmol), using the solvent composition mentioned in <a class="ref internalNav" href="#sec4_10_13" aria-label="General Procedure A">General Procedure A</a>. The reaction mixture was stirred at 0 Â°C for 2â3 h and was then warmed to r.t. and stirred for 45 h. The reaction mixture was extracted as in Procedure A and subsequently purified by column chromatography (hexanes/EtOAc) to afford the title product as a white solid (94 mg, 13% yield; <a class="ref internalNav" href="#sch7" aria-label="Scheme 7">Scheme 7</a>). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 10.71 (s, 1H), 8.33â8.31 (m, 1H), 7.83â7.77 (m, 2H), 7.46â7.41 (m, 2H), 7.29â7.25 (m, 1H), 7.24â7.20 (m, 2H), 7.12â7.08 (m, 1H). ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup>, 215.1.</div></div><div id="sec4_10_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> Phenyl Pyridin-3-ylcarbamate (<b>19d</b>)<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a></h4><div class="NLM_p last">3-Aminopyridine (941 mg, 10 mmol) was dissolved in a mixture of acetonitrile (8 mL) and pyridine (0.9 mL, 11.0 mmol). The reaction mixture was cooled to 0 Â°C, phenyl chloroformate (1.25 mL, 10.0 mmol) was added, and the reaction mixture was warmed to room temperature and stirred for 2.5 h. Upon quenching with water (40 mL), the title product precipitated out of the reaction mixture and was washed with water and dried under vacuum to give a reddish-brown solid (1.7 g, 79% yield; <a class="ref internalNav" href="#sch7" aria-label="Scheme 7">Scheme 7</a>). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 10.47 (s, 1H), 8.69 (d, <i>J</i> = 2.4 Hz, 1H), 8.27 (dd, <i>J</i> = 4.7, 1.3 Hz, 1H), 7.93 (d, <i>J</i> = 8.7 Hz, 1H), 7.44 (t, <i>J</i> = 7.9 Hz, 2H), 7.37 (dd, <i>J</i> = 8.3, 4.7 Hz, 1H), 7.31â7.21 (m, 3H).<sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 151.87, 150.37, 144.04, 140.28, 135.42, 129.48, 125.64, 125.34, 123.74, 121.94. ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup>, 215.1.</div></div><div id="sec4_10_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> Methyl 2-(4-((Phenoxycarbonyl)amino)phenoxy)acetate (<b>19e</b>)</h4><div class="NLM_p last">Amine <b>18a</b> (600 mg, 3.3 mmol) reacted with phenyl chloroformate (0.44 mL, 3.5 mmol) for 40 min to afford a white, crystalline solid (919 mg, 92% yield; <a class="ref internalNav" href="#sch7" aria-label="Scheme 7">Scheme 7</a>). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 10.08 (s, 1H), 7.46â7.36 (m, 4H), 7.25 (t, <i>J</i> = 7.4 Hz, 1H), 7.21 (d, <i>J</i> = 7.6 Hz, 2H), 6.91 (d, <i>J</i> = 9.1 Hz, 2H), 4.75 (s, 2H), 3.69 (s, 3H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 169.30, 153.41, 151.81, 150.59, 132.26, 129.40, 125.34, 121.95, 119.95, 114.82, 64.80, 51.78. ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup>, 302.1.</div></div><div id="sec4_10_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> Methyl 2-(3-((Phenoxycarbonyl)amino)phenoxy)acetate (<b>19f</b>)</h4><div class="NLM_p last">Amine <b>18b</b> (600 mg, 3.3 mmol) reacted with phenyl chloroformate (0.44 mL, 3.5 mmol) for 40 min to afford a white solid (895 mg, 90% yield; <a class="ref internalNav" href="#sch7" aria-label="Scheme 7">Scheme 7</a>).<sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 10.26 (s, 1H), 7.43 (t, <i>J</i> = 7.7 Hz, 2H), 7.29â7.19 (m, 3H), 7.18â7.10 (m, 2H), 6.78â6.72 (m, 1H), 6.61 (dd, <i>J</i> = 8.2, 2.3 Hz, 1H), 4.75 (s, 2H), 3.69 (s, 3H). ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup>, 302.1.</div></div><div id="sec4_10_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> Ethyl 2-(4-((Phenoxycarbonyl)amino)phenyl)oxazole-4-carboxylate (<b>19g</b>)</h4><div class="NLM_p last">Amine <b>18c</b> (71 mg, 0.31 mmol) reacted with phenyl chloroformate (42 mL, 0.33 mmol) for 1 h to provide the title product as a light brown solid (99 mg, 91% yield; <a class="ref internalNav" href="#sch7" aria-label="Scheme 7">Scheme 7</a>). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 10.62 (s, 1H), 8.90 (s, 1H), 7.99 (d, <i>J</i> = 8.8 Hz, 2H), 7.70 (d, <i>J</i> = 8.7 Hz, 2H), 7.45 (t, <i>J</i> = 7.9 Hz, 2H), 7.30â7.24 (m, 3H), 4.31 (q, <i>J</i> = 7.1 Hz, 2H), 1.31 (t, <i>J</i> = 7.1 Hz, 3H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 161.25, 160.75, 151.58, 150.33, 145.32, 141.51, 133.60, 129.49, 127.44, 125.64, 121.94, 120.44, 118.51, 60.64, 14.20. ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup>, 353.1.</div></div><div id="sec4_10_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> Methyl 5-(4-((Phenoxycarbonyl)amino)phenyl)furan-2-carboxylate (<b>19h</b>)</h4><div class="NLM_p last">Methyl 5-(4-aminophenyl)furan-2-carboxylate (719 mg, 3.3 mmol) reacted with phenyl chloroformate (0.44 mL, 3.5 mmol) for 1 h to afford the title product as a cream white solid (1.09 g, 97% yield; <a class="ref internalNav" href="#sch7" aria-label="Scheme 7">Scheme 7</a>). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 10.48 (s, 1H), 7.79 (d, <i>J</i> = 8.7 Hz, 2H), 7.63 (d, <i>J</i> = 8.6 Hz, 2H), 7.44 (t, <i>J</i> = 7.9 Hz, 2H), 7.40 (d, <i>J</i> = 3.6 Hz, 1H), 7.27 (t, <i>J</i> = 7.4 Hz, 1H), 7.25 (d, <i>J</i> = 7.7 Hz, 2H), 7.07 (d, <i>J</i> = 3.6 Hz, 1H), 3.83 (s, 3H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 158.31, 156.82, 151.62, 150.40, 142.51, 139.53, 129.46, 125.56, 125.42, 123.66, 121.96, 120.71, 118.66, 107.06, 51.76. ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup>, 338.1.</div></div><div id="sec4_10_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> Ethyl 5-(4-((Phenoxycarbonyl)amino)phenyl)-1,3,4-oxadiazole-2-carboxylate (<b>19i</b>)</h4><div class="NLM_p last">Amine <b>18d</b> (200 mg, 0.86 mmol) was reacted with phenyl chloroformate (115 mL, 0.91 mmol) for 50 min. The crude mixture was processed according to Procedure A. The product was further purified by column chromatography (hexanes/EtOAc) to afford the title product as a yellow solid (117 mg, 39% yield; <a class="ref internalNav" href="#sch7" aria-label="Scheme 7">Scheme 7</a>). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 10.74 (s, 1H), 8.05 (d, <i>J</i> = 8.7 Hz, 2H), 7.77 (d, <i>J</i> = 8.7 Hz, 2H), 7.45 (t, <i>J</i> = 7.9 Hz, 2H), 7.28 (dd, <i>J</i> = 16.7, 7.9 Hz, 3H), 4.45 (q, <i>J</i> = 7.1 Hz, 2H), 1.37 (t, <i>J</i> = 7.1 Hz, 3H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 165.66, 156.61, 154.51, 152.01, 150.70, 143.31, 129.94, 128.82, 126.14, 122.36, 119.12, 117.08, 63.30, 14.33. ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup>, 354.4.</div></div><div id="sec4_10_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> (<i>E</i>)-3-(4-((Phenoxycarbonyl)amino)phenyl)acrylic Acid (<b>19j</b>)</h4><div class="NLM_p last">(<i>E</i>)-4-Aminocinnamic acid (540 mg, 3.3 mmol) was reacted with phenyl chloroformate (0.44 mL, 3.5 mmol) in the presence of 2.2 equiv of sodium bicarbonate (612 mg, 7.3 mmol), for 1 h. The reaction mixture was acidified to pH 4 followed by an extraction workup, as described in Procedure A. The product was further purified by column chromatography (DCM/MeCN) to afford the title compound as a white solid (500 mg, 53% yield; <a class="ref internalNav" href="#sch7" aria-label="Scheme 7">Scheme 7</a>). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 12.29 (s, 1H), 10.47 (s, 1H), 7.66 (d, <i>J</i> = 8.7 Hz, 2H), 7.58â7.51 (m, 3H), 7.44 (t, <i>J</i> = 7.9 Hz, 2H), 7.27 (t, <i>J</i> = 7.6 Hz, 1H), 7.24 (d, <i>J</i> = 7.6 Hz, 2H), 6.43 (d, <i>J</i> = 16.0 Hz, 1H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 167.73, 151.56, 150.37, 143.53, 140.51, 129.47, 129.21, 128.85, 125.58, 121.96, 118.32, 117.40. ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup>, 284.0.</div></div><div id="sec4_10_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> 4-((5-Amino-1-(2,6-difluorobenzyl)-1<i>H</i>-1,2,4-triazol-3-yl)amino)benzenesulfonamide (<b>2</b>)</h4><div class="NLM_p"><b>17a</b> (150 mg, 0.6 mmol), 2-(bromomethyl)-1,3-difluorobenzene (75 Î¼L, 0.6 mmol), and Cs<sub>2</sub>CO<sub>3</sub> (192 mg, 0.6 mmol) were dissolved in dimethylformamide (2.4 mL, 0.25 M). The reaction mixture was stirred for 16 h at room temperature. The solvent was then evaporated, and the residue was purified by flash chromatography (DCM/MeOH) to afford <b>2</b> (54 mg, 24% yield; <a class="ref internalNav" href="#sch8" aria-label="Scheme 8">Scheme 8</a>). The purity was 97%. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 9.24 (s, 1H), 7.56 (d, <i>J</i> = 8.9 Hz, 2H), 7.49 (d, <i>J</i> = 8.9 Hz, 2H), 7.46â7.40 (m, 1H), 7.12 (t, <i>J</i> = 7.9 Hz, 2H), 7.02 (s, 2H), 6.46 (s, 2H), 5.08 (s, 2H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 160.99 (dd, <i>J</i> = 248.7, 7.9 Hz), 156.55, 154.20, 145.26, 133.28, 130.64 (t, <i>J</i> = 10.3 Hz), 126.64, 114.56, 112.38 (t, <i>J</i> = 19.2 Hz), 111.62 (dd, <i>J</i> = 20.6, 4.7 Hz), 37.21 (t, <i>J</i> = 3.4 Hz). HRMS (ESI): calc for [M + H]<sup>+</sup> C<sub>15</sub>H<sub>15</sub>F<sub>2</sub>N<sub>6</sub>O<sub>2</sub>S, 381.0945; found, 381.0956.</div><figure id="sch8" data-index="17" class="article__inlineFigure"><h2 class="fig-label">Scheme 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/medium/jm0c00192_0018.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0018.jpeg" id="gr17" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 8. Synthesis of <b>2</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00192&amp;id=sch8"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) Cs<sub>2</sub>CO<sub>3</sub>, DMF, r.t., and 16 h.</p></p></figure></div><div id="sec4_10_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> General Procedure B: Syntheses of Ureas <b>3</b>, <b>4</b>, <b>6</b>â<b>8</b>, <b>9â²</b>, <b>11â²-14â²</b>, and <b>15</b></h4><div class="NLM_p">The appropriate phenyl carbamate (1.0 equiv) was dissolved in dry dioxane (0.4 M for <b>15</b>; 0.5 M for <b>3</b>, <b>8</b>, <b>12â²</b>, <b>13â²</b>, and <b>14â²</b>; 1 M for <b>4</b>, <b>6</b>, <b>7</b>, <b>9â²</b>, and <b>11â²</b>), triethylamine (1.0 equiv for all except <b>15</b>; 2.0 equiv for <b>15</b>) was added, and the mixture was heated at 90â100 Â°C for 5 min. In a separate vial, a mixture of <b>17a</b> or <b>17b</b> (1.0 equiv) and triethylamine (1.0 equiv) in dry dioxane (0.2 M for <b>15</b>; 0.5 M for <b>3</b>, <b>8</b>, <b>12â²</b>, <b>13â²</b>, and <b>14â²</b>; 1 M for <b>4</b>, <b>6</b>, <b>7</b>, <b>9â²</b>, and <b>11â²</b>) was sonicated at 80â100 Â°C for 5â10 min and was added to the reaction mixture dropwise. The reaction vial was sealed under a N<sub>2</sub> atmosphere, and the mixture was stirred for 1â3 h at 90â110 Â°C (90 Â°C for <b>11â²</b>; 100 Â°C for <b>3</b>, <b>4</b>, <b>6</b>â<b>7</b>, <b>9â²</b>, and <b>15</b>; 110 Â°C for <b>8</b>, <b>12â²</b>, <b>13â²</b>, and <b>14â²</b>; <a class="ref internalNav" href="#sch9" aria-label="Scheme 9">Scheme 9</a>).</div><figure id="sch9" data-index="18" class="article__inlineFigure"><h2 class="fig-label">Scheme 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/medium/jm0c00192_0019.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0019.jpeg" id="gr18" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 9. Syntheses of Compounds <b>3</b>, <b>4</b>, and <b>6</b>â<b>15</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0019.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00192&amp;id=sch9"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) Et<sub>3</sub>N, dioxane, 90â110 Â°C, and 35 min to 2.5 h; (b) base, LiBr, MeCN, 2 vol % H<sub>2</sub>O, r.t., and 11â288 h.</p></p></figure></div><div id="sec4_10_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> Workup A (<b>3</b>, <b>8</b>, <b>9â²</b>, <b>11â²</b>, and <b>14â²</b>)</h4><div class="NLM_p last">The reaction mixture was diluted with ethyl acetate (50 mL) and was washed with water (2 Ã 15 mL) and brine (1 Ã 15 mL). The organic layer was collected, dried over sodium sulfate, and concentrated under reduced pressure. The crude mixture was purified by column chromatography (DCM/MeOH).</div></div><div id="sec4_10_26" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> Workup B (<b>7</b>, <b>12â²</b>, and <b>13â²</b>)</h4><div class="NLM_p last">The solvent was evaporated under reduced pressure, and the reaction mixture was purified directly by chromatography (DCM/MeOH).</div></div><div id="sec4_10_27" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> Workup C (<b>4</b> and <b>6</b>)</h4><div class="NLM_p last">The solvent was evaporated under reduced pressure, and the crude mixture was triturated with ethyl acetate (3 Ã 10 mL). The combined organic phases were purified by column chromatography (DCM/MeOH).</div></div><div id="sec4_10_28" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61"> General Procedure C: Hydrolysis of Esters <b>8</b>, <b>9â²</b>, and <b>11â²</b>â<b>14â²</b><a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a></h4><div class="NLM_p last">Esters <b>8</b>, <b>9â²</b>, and <b>11â²</b>â<b>14â²</b> were suspended in a mixture of acetonitrile with 2 vol % water. Base (DBN, 3.0 equiv for <b>8</b>, <b>12â²</b>, and <b>13â²</b>; Et<sub>3</sub>N, 3.0 equiv for <b>9â²</b>, <b>11â²</b>, and <b>14â²</b>) and lithium bromide (10 equiv) were then added, and the reaction was allowed to run at r.t. for the indicated time.</div></div><div id="sec4_10_29" class="NLM_sec NLM_sec_level_3"><div id="ac_i62" class="anchor-spacer"></div><h4 class="article-section__title" id="_i62"> Workup D (<b>9</b> and <b>11</b>)</h4><div class="NLM_p last">The solvent was evaporated, and sat. NaHCO<sub>3</sub> was added to the reaction residue. The mixture was washed with ethyl acetate, and the aqueous phase was acidified to pH â¼4. The precipitate was collected, washed with small amounts of ethyl acetate and water, and dried under vacuum to provide the desired product.</div></div><div id="sec4_10_30" class="NLM_sec NLM_sec_level_3"><div id="ac_i63" class="anchor-spacer"></div><h4 class="article-section__title" id="_i63"> Workup E (<b>10</b> and <b>12</b>â<b>14</b>)</h4><div class="NLM_p last">The solvent was evaporated, and a small amount of water was added to the residue. The pH was adjusted to â¼4, and the mixture was kept at low temperature for 6â12 h. The mixture was centrifuged, and the precipitate was dried and purified by normal phase column chromatography (DCM/MeOH) for <b>10</b>, <b>12</b>, and <b>14</b> and by reverse phase column chromatography (H<sub>2</sub>O/MeCN) for <b>13</b>. Compound <b>10</b> was further purified by triturating with methanol.</div></div><div id="sec4_10_31" class="NLM_sec NLM_sec_level_3"><div id="ac_i64" class="anchor-spacer"></div><h4 class="article-section__title" id="_i64"> 5-Amino-3-((4-cyanophenyl)amino)-<i>N</i>-phenyl-1<i>H</i>-1,2,4-triazole-1-carboxamide (<b>3</b>)</h4><div class="NLM_p last"><b>19a</b> (105 mg, 0.5 mmol) reacted with <b>17b</b> (99 mg, 0.5 mmol) for 1.5 h to afford the title compound as a white, fluffy solid (45 mg, 29% yield). The purity was 99%. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 9.87 (s, 1H), 9.64 (s, 1H), 7.85 (d, <i>J</i> = 8.8 Hz, 2H), 7.67 (d, <i>J</i> = 8.8 Hz, 2H), 7.66â7.63 (m, 2H), 7.45 (s, 2H), 7.42â7.37 (m, 2H), 7.18 (tt, <i>J</i> = 7.3, 1.2 Hz, 1H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 156.38, 155.91, 149.08, 145.05, 137.06, 133.12, 128.65, 124.51, 121.84, 119.83, 116.99, 101.05. HRMS (ESI): calc for [M + H]<sup>+</sup> C<sub>16</sub>H<sub>14</sub>N<sub>7</sub>O, 320.1260; found, 320.1245. [<b>17b</b> + H]<sup>+</sup> of 201.0882 was also observed due to in-source fragmentation.</div></div><div id="sec4_10_32" class="NLM_sec NLM_sec_level_3"><div id="ac_i65" class="anchor-spacer"></div><h4 class="article-section__title" id="_i65"> 5-Amino-<i>N</i>-phenyl-3-((4-sulfamoylphenyl)amino)-1<i>H</i>-1,2,4-triazole-1-carboxamide (<b>4</b>)</h4><div class="NLM_p last"><b>19a</b> (59 mg, 0.3 mmol) reacted with <b>17a</b> (70 mg, 0.3 mmol) for 1 h to afford the title compound as a white, fluffy solid (39 mg, 38% yield). The purity was 99%. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 9.68 (s, 1H), 9.59 (s, 1H), 7.82 (d, <i>J</i> = 8.9 Hz, 2H), 7.71 (d, <i>J</i> = 9.0 Hz, 2H), 7.65 (d, <i>J</i> = 8.7 Hz, 2H), 7.46â7.37 (m, 4H), 7.20â7.16 (m, 1H), 7.14 (s, 2H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 156.69, 155.90, 149.14, 143.90, 137.10, 134.88, 128.70, 126.84, 124.54, 121.87, 116.24. HRMS (ESI): calc for [M + H]<sup>+</sup> C<sub>15</sub>H<sub>16</sub>N<sub>7</sub>O<sub>3</sub>S, 374.1035; found, 374.1009. [<b>17a</b> + H]<sup>+</sup> of 255.0645 was also observed due to in-source fragmentation.</div></div><div id="sec4_10_33" class="NLM_sec NLM_sec_level_3"><div id="ac_i66" class="anchor-spacer"></div><h4 class="article-section__title" id="_i66"> Phenyl 5-Amino-3-((4-sulfamoylphenyl)amino)-1<i>H</i>-1,2,4-triazole-1-carboxylate (<b>19b</b>)</h4><div class="NLM_p last">To a suspension of <b>17a</b> (509 mg, 2.0 mmol) in tetrahydrofuran/water was added sodium bicarbonate (168 mg, 2.0 mmol). The mixture was cooled to 0 Â°C. Phenyl chloroformate (0.25 mL, 2.0 mmol) was added followed by additional sodium bicarbonate (168 mg, 2.0 mmol). The reaction mixture was stirred at 0 Â°C for 1 h, and the crude mixture was processed according to Procedure A. The product was further purified by column chromatography (DCM/MeOH) to afford the title product as a pale white solid (581 mg, 78% yield). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 9.78 (s, 1H), 7.69 (s, 4H), 7.57 (s, 2H), 7.52â7.48 (m, 2H), 7.41â7.34 (m, 3H), 7.14 (s, 2H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 158.11, 157.01, 149.91, 148.58, 143.93, 135.04, 129.70, 126.80, 126.61, 121.87, 115.89. ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup>, 375.0.</div></div><div id="sec4_10_34" class="NLM_sec NLM_sec_level_3"><div id="ac_i67" class="anchor-spacer"></div><h4 class="article-section__title" id="_i67"> 5-Amino-<i>N</i>-methyl-<i>N</i>-phenyl-3-((4-sulfamoylphenyl)amino)-1<i>H</i>-1,2,4-triazole-1-carboxamide (<b>5</b>)</h4><div class="NLM_p"><i>N</i>-Methyl aniline (0.23 mL, 2.1 mmol) was dissolved in tetrahydrofuran (4.0 mL). Sodium hydride (51 mg, 2.1 mmol) was added at 0 Â°C, and the mixture was allowed to stir for 1 h. Subsequently, this mixture was transferred dropwise to a separate vial containing <b>19b</b> (400 mg, 1.1 mmol) in dioxane (2 mL), and the reaction was heated to reflux. After 20 h, the solvent was evaporated, and the crude mixture was purified by column chromatography (DCM/MeCN) to provide the product as a light brown solid (2 mg, yield of <1%; <a class="ref internalNav" href="#sch10" aria-label="Scheme 10">Scheme 10</a>). The purity was 94%. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 9.33 (s, 1H), 7.43 (dd, <i>J</i> = 13.3, 7.9 Hz, 4H), 7.27 (dd, <i>J</i> = 13.5, 7.5 Hz, 5H), 7.09 (s, 2H), 6.87 (d, <i>J</i> = 8.5 Hz, 2H), 3.36 (s, 3H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 156.93, 155.37, 151.42, 145.19, 143.66, 134.31, 129.00, 126.30, 126.08, 125.79, 115.34, 29.62. HRMS (ESI): calc for [M + H]<sup>+</sup> C<sub>16</sub>H<sub>18</sub>N<sub>7</sub>O<sub>3</sub>S, 388.1192; found, 388.1156.</div><figure id="sch10" data-index="19" class="article__inlineFigure"><h2 class="fig-label">Scheme 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/medium/jm0c00192_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0011.jpeg" id="gr19" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 10. Synthesis of Compound <b>5</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00192&amp;id=sch10"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) NaHCO<sub>3</sub>, H<sub>2</sub>O/THF, 0 Â°C, and 1 h; (b) NaH, THF/dioxane, reflux, and 20 h.</p></p></figure></div><div id="sec4_10_35" class="NLM_sec NLM_sec_level_3"><div id="ac_i69" class="anchor-spacer"></div><h4 class="article-section__title" id="_i69"> 5-Amino-<i>N</i>-(pyridin-2-yl)-3-((4-sulfamoylphenyl)amino)-1<i>H</i>-1,2,4-triazole-1-carboxamide (<b>6</b>)</h4><div class="NLM_p last"><b>19c</b> (59 mg, 0.3 mmol) reacted with <b>17a</b> (70 mg, 0.3 mmol) for 1 h to afford the title compound as a white, fluffy solid (15 mg, 14% yield). The purity was 99%. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 9.76 (s, 1H), 9.42 (s, 1H), 8.41 (d, <i>J</i> = 4.1 Hz, 1H), 7.95 (d, <i>J</i> = 8.3 Hz, 1H), 7.92â7.88 (m, 1H), 7.75â7.70 (m, 4H), 7.52 (s, 2H), 7.25â7.21 (m, 1H), 7.15 (s, 2H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 156.89, 155.90, 150.13, 148.34, 143.79, 138.61, 135.13, 126.81, 120.34, 116.16, 113.66, 109.56. HRMS (ESI): calc for [M + H]<sup>+</sup> C<sub>14</sub>H<sub>15</sub>N<sub>8</sub>O<sub>3</sub>S, 375.0988; found, 375.0410. [<b>17a</b> + H]<sup>+</sup> of 255.0290 was also observed due to in-source fragmentation.</div></div><div id="sec4_10_36" class="NLM_sec NLM_sec_level_3"><div id="ac_i70" class="anchor-spacer"></div><h4 class="article-section__title" id="_i70"> 5-Amino-<i>N</i>-(pyridin-3-yl)-3-((4-sulfamoylphenyl)amino)-1<i>H</i>-1,2,4-triazole-1-carboxamide (<b>7</b>)</h4><div class="NLM_p last"><b>19d</b> (59 mg, 0.3 mmol) was reacted with <b>17a</b> (70 mg, 0.3 mmol) for 1 h to afford the title compound as a white, fluffy solid (20 mg, 20% yield). The purity was 100%. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 9.80 (s, 1H), 9.70 (s, 1H), 8.85 (d, <i>J</i> = 2.5 Hz, 1H), 8.38 (dd, <i>J</i> = 4.7, 1.4 Hz, 1H), 8.06 (ddd, <i>J</i> = 8.3, 2.6, 1.5 Hz, 1H), 7.84â7.80 (m, 2H), 7.72â7.69 (m, 2H), 7.47â7.42 (m, 3H), 7.15 (s, 2H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 156.81, 155.91, 149.28, 145.35, 143.80, 143.55, 134.91, 133.94, 129.16, 126.77, 123.45, 116.21. HRMS (ESI): calc for [M + H]<sup>+</sup> C<sub>14</sub>H<sub>15</sub>N<sub>8</sub>O<sub>3</sub>S, 375.0988; found, 375.0970. [<b>17a</b> + H]<sup>+</sup> of 255.0662 was also observed due to in-source fragmentation.</div></div><div id="sec4_10_37" class="NLM_sec NLM_sec_level_3"><div id="ac_i71" class="anchor-spacer"></div><h4 class="article-section__title" id="_i71"> Methyl 2-(4-(5-Amino-3-((4-sulfamoylphenyl)amino)-1<i>H</i>-1,2,4-triazole-1-carboxamido)phenoxy)acetate (<b>8</b>)</h4><div class="NLM_p last"><b>19e</b> (331 mg, 1.1 mmol) reacted with <b>17a</b> (280 mg, 1.1 mmol) for 1.5 h. The reaction was processed using Workup A with modified column chromatography (DCM/MeCN) to afford the title compound as a white, fluffy solid (209 mg, 41% yield). The purity was 98%. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 9.66 (s, 1H), 9.53 (s, 1H), 7.81 (d, <i>J</i> = 8.8 Hz, 2H), 7.69 (d, <i>J</i> = 8.7 Hz, 2H), 7.52 (d, <i>J</i> = 9.0 Hz, 2H), 7.38 (s, 2H), 7.14 (s, 2H), 6.97 (d, <i>J</i> = 9.0 Hz, 2H), 4.81 (s, 2H), 3.71 (s, 3H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 169.26, 156.61, 155.80, 154.60, 149.31, 143.88, 134.79, 130.47, 126.77, 123.89, 116.16, 114.49, 64.77, 51.81. HRMS (ESI): calc for [M + H]<sup>+</sup> C<sub>18</sub>H<sub>20</sub>N<sub>7</sub>O<sub>6</sub>S, 462.1196; found, 462.1172. [<b>17a</b> + H]<sup>+</sup> of 255.0659 was also observed due to in-source fragmentation.</div></div><div id="sec4_10_38" class="NLM_sec NLM_sec_level_3"><div id="ac_i72" class="anchor-spacer"></div><h4 class="article-section__title" id="_i72"> Methyl 2-(4-(5-Amino-3-((4-cyanophenyl)amino)-1<i>H</i>-1,2,4-triazole-1-carboxamido)phenoxy)acetate (<b>9â²</b>)</h4><div class="NLM_p last"><b>19e</b> (151 mg, 0.5 mmol) reacted with <b>17b</b> (100 mg, 0.5 mmol) for 35 min to afford the title compound as a white fluffy solid. (65 mg, 32% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 9.84 (s, 1H), 9.57 (s, 1H), 7.84 (d, <i>J</i> = 8.8 Hz, 2H), 7.65 (d, <i>J</i> = 8.9 Hz, 2H), 7.51 (d, <i>J</i> = 9.1 Hz, 2H), 7.40 (s, 2H), 6.97 (d, <i>J</i> = 9.1 Hz, 2H), 4.80 (s, 2H), 3.71 (s, 3H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 169.26, 156.34, 155.84, 154.62, 149.28, 145.07, 133.10, 130.45, 123.89, 119.85, 116.98, 114.49, 101.00, 64.76, 51.82. ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup>, 408.1.</div></div><div id="sec4_10_39" class="NLM_sec NLM_sec_level_3"><div id="ac_i73" class="anchor-spacer"></div><h4 class="article-section__title" id="_i73"> 2-(4-(5-Amino-3-((4-cyanophenyl)amino)-1<i>H</i>-1,2,4-triazole-1-carboxamido)phenoxy)acetic Acid (<b>9</b>)</h4><div class="NLM_p last"><b>9â²</b> (14 mg, 35 Î¼mol) was hydrolyzed according to <a class="ref internalNav" href="#sec4_10_28" aria-label="General Procedure C">General Procedure C</a> (MeCN, 0.2 M, Et<sub>3</sub>N, 11 h), and the reaction was processed using Workup D to afford the title compound as a white solid (10 mg, 70% yield). The purity was 97%. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 12.94 (bs, 1H), 9.86 (s, 1H), 9.57 (s, 1H), 7.85 (d, <i>J</i> = 8.5 Hz, 2H), 7.65 (d, <i>J</i> = 8.5 Hz, 2H), 7.51 (d, <i>J</i> = 8.7 Hz, 2H), 7.42 (s, 2H), 6.94 (d, <i>J</i> = 8.7 Hz, 2H), 4.68 (s, 2H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 170.25, 156.38, 155.88, 154.85, 149.33, 145.11, 133.14, 130.25, 123.91, 119.90, 117.03, 114.45, 101.04, 64.73. HRMS (ESI): calc for [M + H]<sup>+</sup> C<sub>18</sub>H<sub>16</sub>N<sub>7</sub>O<sub>4</sub>, 394.1264; found, 394.1285.</div></div><div id="sec4_10_40" class="NLM_sec NLM_sec_level_3"><div id="ac_i74" class="anchor-spacer"></div><h4 class="article-section__title" id="_i74"> 2-(4-(5-Amino-3-((4-sulfamoylphenyl)amino)-1<i>H</i>-1,2,4-triazole-1-carboxamido)phenoxy)acetic Acid (<b>10</b>)</h4><div class="NLM_p last"><b>8</b> (187 mg, 405 Î¼mol) was hydrolyzed according to <a class="ref internalNav" href="#sec4_10_28" aria-label="General Procedure C">General Procedure C</a> (MeCN, 0.03 M, DBN, 38 h), and the reaction was processed using Workup Î to afford the title compound as a white solid (15.1 mg, 8% yield). The purity was 95%.<sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 12.95 (bs, 1H), 9.66 (s, 1H), 9.52 (s, 1H), 7.81 (d, <i>J</i> = 8.8 Hz, 2H), 7.69 (d, <i>J</i> = 8.8 Hz, 2H), 7.50 (d, <i>J</i> = 8.9 Hz, 2H), 7.38 (s, 2H), 7.14 (s, 2H), 6.93 (d, <i>J</i> = 8.9 Hz, 2H), 4.65 (s, 2H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 170.25, 156.60, 155.80, 154.89, 149.32, 143.89, 134.78, 130.17, 126.78, 123.85, 116.16, 114.40, 64.90. HRMS (ESI): calc for [M + H]<sup>+</sup> C<sub>17</sub>H<sub>18</sub>N<sub>7</sub>O<sub>6</sub>S, 448.1039; found, 448.1033. [<b>17a</b> + H]<sup>+</sup> of 255.0668 was also observed due to in-source fragmentation.</div></div><div id="sec4_10_41" class="NLM_sec NLM_sec_level_3"><div id="ac_i75" class="anchor-spacer"></div><h4 class="article-section__title" id="_i75"> Methyl 2-(3-(5-Amino-3-((4-cyanophenyl)amino)-1<i>H</i>-1,2,4-triazole-1-carboxamido)phenoxy)acetate (<b>11â²</b>)</h4><div class="NLM_p last"><b>19f</b> (151 mg, 0.5 mmol) was reacted with <b>17b</b> (100 mg, 0.5 mmol) for 50 min to afford the title compound as a white fluffy solid. (59 mg, 29% yield). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 9.87 (s, 1H), 9.61 (s, 1H), 7.83 (d, <i>J</i> = 8.9 Hz, 2H), 7.67 (d, <i>J</i> = 9.0 Hz, 2H), 7.46 (s, 2H), 7.33â7.27 (m, 3H), 6.74 (dt, <i>J</i> = 7.0, 2.3 Hz, 1H), 4.80 (s, 2H), 3.72 (s, 3H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 169.14, 157.78, 156.40, 155.92, 148.93, 145.03, 138.35, 133.14, 129.50, 119.83, 117.00, 114.41, 110.05, 108.17, 101.08, 64.59, 51.87. ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup>, 408.1.</div></div><div id="sec4_10_42" class="NLM_sec NLM_sec_level_3"><div id="ac_i76" class="anchor-spacer"></div><h4 class="article-section__title" id="_i76"> 2-(3-(5-Amino-3-((4-cyanophenyl)amino)-1<i>H</i>-1,2,4-triazole-1-carboxamido)phenoxy)acetic Acid (<b>11</b>)</h4><div class="NLM_p last"><b>11â²</b> (16 mg, 39 Î¼mol) was hydrolyzed according to <a class="ref internalNav" href="#sec4_10_28" aria-label="General Procedure C">General Procedure C</a> (MeCN, 0.2 M, Et<sub>3</sub>N, 21 h), and the reaction was processed using Workup D to afford the title compound as a white solid (5 mg, 35% yield). The purity was 91%. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 12.99 (bs, 1H), 9.87 (s, 1H), 9.60 (s, 1H), 7.83 (d, <i>J</i> = 8.5 Hz, 2H), 7.67 (d, <i>J</i> = 8.7 Hz, 2H), 7.45 (s, 2H), 7.32â7.27 (m, 3H), 6.74â6.69 (m, 1H), 4.67 (s, 2H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 170.08, 157.99, 156.41, 155.92, 148.93, 145.05, 138.32, 133.16, 129.45, 119.84, 117.00, 114.17, 110.12, 108.01, 101.08, 64.53. HRMS (ESI): calc for [M + H]<sup>+</sup> C<sub>18</sub>H<sub>16</sub>N<sub>7</sub>O<sub>4</sub>, 394.1264; found, 394.1337.</div></div><div id="sec4_10_43" class="NLM_sec NLM_sec_level_3"><div id="ac_i77" class="anchor-spacer"></div><h4 class="article-section__title" id="_i77"> Ethyl 2-(4-(5-Amino-3-((4-sulfamoylphenyl)amino)-1<i>H</i>-1,2,4-triazole-1-carboxamido)phenyl)oxazole-4-carboxylate (<b>12â²</b>)</h4><div class="NLM_p last"><b>19g</b> (60 mg, 0.2 mmol) reacted with <b>17a</b> (43 mg, 0.2 mmol) for 50 min. The reaction was processed using Workup B with modified column chromatography (DCM/MeCN) to provide the title compound as a white solid. (21 mg, 25% yield). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 9.86 (s, 1H), 9.72 (s, 1H), 8.94 (s, 1H), 8.04 (d, <i>J</i> = 8.7 Hz, 2H), 7.93 (d, <i>J</i> = 8.7 Hz, 2H), 7.84 (d, <i>J</i> = 8.8 Hz, 2H), 7.72 (d, <i>J</i> = 8.9 Hz, 2H), 7.49 (s, 2H), 7.16 (s, 2H), 4.33 (q, <i>J</i> = 7.1 Hz, 2H), 1.32 (t, <i>J</i> = 7.1 Hz, 3H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 161.16, 160.74, 156.77, 155.95, 148.87, 145.51, 143.79, 140.15, 134.96, 133.68, 127.03, 126.79, 121.57, 121.39, 116.26, 60.68, 14.21. ESI -MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup>, 513.1.</div></div><div id="sec4_10_44" class="NLM_sec NLM_sec_level_3"><div id="ac_i78" class="anchor-spacer"></div><h4 class="article-section__title" id="_i78"> 2-(4-(5-Amino-3-((4-sulfamoylphenyl)amino)-1<i>H</i>-1,2,4-triazole-1-carboxamido)phenyl)oxazole-4-carboxylic Acid (<b>12</b>)</h4><div class="NLM_p last"><b>12â²</b> (14 mg, 27 Î¼mol) was hydrolyzed according to <a class="ref internalNav" href="#sec4_10_28" aria-label="General Procedure C">General Procedure C</a> (MeCN, 0.14 M, DBN, 68 h). The reaction was processed using Workup Î and was further purified with HPLC to afford the title compound as a white solid (1.5 mg, 11% yield). The purity was 98%. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 9.82 (s, 1H), 9.72 (s, 1H), 8.33 (bs, 2H), 8.04 (bs, 2H), 7.88 (d, <i>J</i> = 7.6 Hz, 2H), 7.83 (d, <i>J</i> = 8.4 Hz, 2H), 7.72 (d, <i>J</i> = 8.4 Hz, 2H), 7.48 (s, 2H), 7.16 (s, 2H).<sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 163.86, 163.23, 159.80, 156.77, 155.95, 148.91, 143.82, 139.44, 134.95, 126.81, 126.75, 122.65, 121.39, 116.26, 114.54. HRMS (ESI): calc for [M + H]<sup>+</sup> C<sub>19</sub>H<sub>17</sub>N<sub>8</sub>O<sub>6</sub>S, 485.0992; found, 485.0979. [<b>17a</b> + H]<sup>+</sup> of 255.0667 was also observed due to in-source fragmentation.</div></div><div id="sec4_10_45" class="NLM_sec NLM_sec_level_3"><div id="ac_i79" class="anchor-spacer"></div><h4 class="article-section__title" id="_i79"> Methyl 5-(4-(5-Amino-3-((4-sulfamoylphenyl)amino)-1<i>H</i>-1,2,4-triazole-1-carboxamido)phenyl)furan-2-carboxylate (<b>13â²</b>)</h4><div class="NLM_p last"><b>19h</b> (86 mg, 0.26 mmol) was reacted with <b>17a</b> (65 mg, 0.26 mmol) for 2.5 h according to <a class="ref internalNav" href="#sec4_10_24" aria-label="General Procedure B">General Procedure B</a>. The reaction was processed using Workup B with modified column chromatography (DCM/MeCN) to provide the title compound as a white solid (20 mg, 16% yield). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 9.76 (s, 1H), 9.70 (s, 1H), 7.87â7.81 (m, 6H), 7.72 (d, <i>J</i> = 8.7 Hz, 2H), 7.46 (s, 2H), 7.43 (d, <i>J</i> = 3.7 Hz, 1H), 7.17â7.13 (m, 3H), 3.85 (s, 3H). ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup>, 498.1.</div></div><div id="sec4_10_46" class="NLM_sec NLM_sec_level_3"><div id="ac_i80" class="anchor-spacer"></div><h4 class="article-section__title" id="_i80"> 5-(4-(5-Amino-3-((4-sulfamoylphenyl)amino)-1<i>H</i>-1,2,4-triazole-1-carboxamido)phenyl)furan-2-carboxylic Acid (<b>13</b>)</h4><div class="NLM_p last"><b>13â²</b> (20 mg, 40 Î¼mol), in MeCN (0.035 M) with DBN, was hydrolyzed for 288 h (12 days) to afford the title product as a pale yellow solid (2 mg, 10% yield). The purity was 90%. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 9.70 (s, 2H), 7.83 (d, <i>J</i> = 8.6 Hz, 2H), 7.76 (s, 4H), 7.72 (d, <i>J</i> = 8.8 Hz, 2H), 7.46 (s, 2H), 7.15 (s, 2H), 6.91 (s, 1H), 6.86 (s, 1H). Carboxylic acid hydrogen is not observed. <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 161.02, 156.71, 155.90, 148.96, 143.84, 137.01, 134.90, 126.81, 126.14, 124.70, 124.33, 121.76, 117.56, 116.24, 114.54, 106.81. HRMS (ESI): calc for [M + H]<sup>+</sup> C<sub>20</sub>H<sub>18</sub>N<sub>7</sub>O<sub>6</sub>S, 484.1039; found, 484.1027.</div></div><div id="sec4_10_47" class="NLM_sec NLM_sec_level_3"><div id="ac_i81" class="anchor-spacer"></div><h4 class="article-section__title" id="_i81"> Ethyl 5-(4-(5-Amino-3-((4-sulfamoylphenyl)amino)-1<i>H</i>-1,2,4-triazole-1-carboxamido)phenyl)-1,3,4-oxadiazole-2-carboxylate (<b>14â²</b>)</h4><div class="NLM_p last"><b>19i</b> (90 mg, 0.26 mmol) was reacted with <b>17a</b> (65 mg, 0.26 mmol) for 2.5 h. The reaction was processed using Workup A with modified column chromatography (DCM/MeCN) to provide the title product as a white solid (59 mg, 45% yield). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 9.96 (s, 1H), 9.72 (s, 1H), 8.11 (d, <i>J</i> = 8.7 Hz, 2H), 8.01 (d, <i>J</i> = 8.7 Hz, 2H), 7.84 (d, <i>J</i> = 8.8 Hz, 2H), 7.72 (d, <i>J</i> = 8.8 Hz, 2H), 7.50 (s, 2H), 7.16 (s, 2H), 4.46 (q, <i>J</i> = 7.1 Hz, 2H), 1.38 (t, <i>J</i> = 7.1 Hz, 3H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 165.21, 156.83, 156.31, 156.00, 154.11, 148.87, 143.79, 141.58, 135.00, 127.98, 126.82, 121.49, 117.82, 116.30, 62.94, 13.94. ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup>, 514.2.</div></div><div id="sec4_10_48" class="NLM_sec NLM_sec_level_3"><div id="ac_i82" class="anchor-spacer"></div><h4 class="article-section__title" id="_i82"> 5-(4-(5-Amino-3-((4-sulfamoylphenyl)amino)-1<i>H</i>-1,2,4-triazole-1-carboxamido)phenyl)-1,3,4-oxadiazole-2-carboxylic Acid (<b>14</b>)</h4><div class="NLM_p last"><b>14â²</b> (58 mg, 112 Î¼mol) in MeCN (0.035 M) with Et<sub>3</sub>N was hydrolyzed for 96 h (4 days). Additional purification with HPLC was needed to afford the product as a white solid (7 mg, 13% yield). The purity was 99%. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 9.90 (s, 1H), 9.72 (s, 1H), 9.33 (s, 1H), 8.06 (d, <i>J</i> = 8.7 Hz, 2H), 7.96 (d, <i>J</i> = 8.8 Hz, 2H), 7.83 (d, <i>J</i> = 8.8 Hz, 2H), 7.72 (d, <i>J</i> = 8.9 Hz, 2H), 7.49 (s, 2H), 7.16 (s, 2H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 163.43, 156.80, 155.97, 154.31, 148.89, 143.79, 140.76, 134.97, 127.42, 127.12, 126.79, 121.50, 118.65, 116.27. IR (solid, cm<sup>â1</sup>): 3291.7 (broad) [OâH stretch], 1724.8 [CâO stretch, âCOOH]. HRMS (ESI): calc for [MâCO<sub>2</sub> + H]<sup>+</sup> C<sub>17</sub>H<sub>16</sub>N<sub>9</sub>O<sub>4</sub>S, 442.1046; found, 442.1042. [<b>17a</b>+H]<sup>+</sup> of 255.0678 was also observed due to in-source fragmentation.</div></div><div id="sec4_10_49" class="NLM_sec NLM_sec_level_3"><div id="ac_i83" class="anchor-spacer"></div><h4 class="article-section__title" id="_i83"> (<i>E</i>)-3-(4-(5-Amino-3-((4-sulfamoylphenyl)amino)-1<i>H</i>-1,2,4-triazole-1-carboxamido)phenyl)acrylic Acid (<b>15</b>)</h4><div class="NLM_p last"><b>19j</b> (100 mg, 0.35 mmol) was reacted with <b>17a</b> (90 mg, 0.35 mmol) and 3.0 equiv triethylamine according to <a class="ref internalNav" href="#sec4_10_24" aria-label="General Procedure B">General Procedure B</a>. After 1 h, the reaction was stopped, the solvent was evaporated, and the residue was triturated with methanol. Subsequently, water was added, and the mixture was acidified to pH â¼4 and was kept at low temperature for 14 h. Afterward, the suspension was filtered, and the solid was allowed to dry under vacuum and was purified first by normal phase column chromatography (DCM/MeCN/MeOH) and last by HPLC to afford the title compound as a white solid (8 mg, 5% yield). The purity was 99%. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 9.73 (s, 1H), 9.70 (s, 1H), 7.82 (d, <i>J</i> = 8.8 Hz, 2H), 7.75 (d, <i>J</i> = 8.6 Hz, 2H), 7.71 (d, <i>J</i> = 8.6 Hz, 4H), 7.55 (d, <i>J</i> = 16.0 Hz, 1H), 7.46 (s, 2H), 7.15 (s, 2H), 6.49 (d, <i>J</i> = 16.0 Hz, 1H). Carboxylic acid hydrogen is not observed. <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 167.93, 156.75, 155.93, 148.89, 143.83, 142.86, 138.94, 134.93, 130.28, 128.73, 126.82, 121.29, 118.82, 116.24. HRMS (ESI): calc for [M + H]<sup>+</sup> C<sub>18</sub>H<sub>18</sub>N<sub>7</sub>O<sub>5</sub>S, 444.1090; found, 444.1081.</div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes2"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i84"><a href="/doi/suppl/10.1021/acs.jmedchem.0c00192" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_51440" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_51440" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c00192?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00192</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline"><sup>1</sup>H and <sup>13</sup>C NMR spectra for final compounds <b>2</b>â<b>15</b>; IR spectrum for compound <b>14</b>; details for the FP and MST assays; crystallographic data for complexes of <b>3</b>, <b>4</b>, <b>9</b>, <b>10</b>, and <b>12</b> with JAK2 JH2, which have been deposited in the RCSB Protein Data Bank with the PDB ID codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6OAV">6OAV</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6OBB">6OBB</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6OBL">6OBL</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6OBF">6OBF</a>, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6OCC">6OCC</a> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00192/suppl_file/jm0c00192_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings for compounds <b>1</b>â<b>15</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00192/suppl_file/jm0c00192_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00192/suppl_file/jm0c00192_si_001.pdf">jm0c00192_si_001.pdf (2.45 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00192/suppl_file/jm0c00192_si_002.csv">jm0c00192_si_002.csv (1.91 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.0c00192" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_40618" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_40618" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">William L. Jorgensen</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, Yale University, New Haven, Connecticut 06520-8107, United States</span>;Â 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-3993-9520" title="Orcid link">http://orcid.org/0000-0002-3993-9520</a>;Â 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#85f2ece9e9ece4e8abefeaf7e2e0ebf6e0ebc5fce4e9e0abe0e1f0"><span class="__cf_email__" data-cfemail="1c6b757070757d713276736e7b79726f79725c657d707932797869">[emailÂ protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Maria-Elena Liosi</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, Yale University, New Haven, Connecticut 06520-8107, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Stefan G. Krimmer</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, Yale University, New Haven, Connecticut 06520-8107, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ana S. Newton</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, Yale University, New Haven, Connecticut 06520-8107, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Thomas K. Dawson</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, Yale University, New Haven, Connecticut 06520-8107, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">David E. Puleo</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmacology, Yale University School
of Medicine, New Haven, Connecticut 06520-8066, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kara J. Cutrona</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, Yale University, New Haven, Connecticut 06520-8107, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yoshihisa Suzuki</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmacology, Yale University School
of Medicine, New Haven, Connecticut 06520-8066, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Joseph Schlessinger</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmacology, Yale University School
of Medicine, New Haven, Connecticut 06520-8066, United States</span>;Â 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-5085-5969" title="Orcid link">http://orcid.org/0000-0002-5085-5969</a></span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>M.-E.L. and S.G.K. contributed equally to this work.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><p>The authors declare no competing financial interest.<br /></br></p></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i86">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_18585" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_18585" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">Gratitude is expressed to the U.S. National Institutes of Health (GM32136) for research support, to the Deutsche Forschungsgemeinschaft (DFG) for a postdoctoral fellowship to S.G.K. (KR 5228/1-1), and to the NIH for training support to D.E.P. (GM007324). This research used resources of the Advanced Photon Source, a U.S. Department of Energy (DOE) Office of Science User Facility operated for the DOE Office of Science by Argonne National Laboratory under contract no. DE-AC02-06CH11357 and also funded by the National Institutes of Health (P30 GM124165) and an NIH-ORIP HEI grant (S10OD021527). This research also used resources of the Yale Macromolecular X-ray Core Facility (NIH grant 1S10OD018007-01). The authors thank Dr. Irit Lax for help with establishing the in vitro kinase assay.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">ATP</td><td class="NLM_def"><p class="first last">adenosine triphosphate</p></td></tr><tr><td class="NLM_term">JAKs</td><td class="NLM_def"><p class="first last">Janus kinases</p></td></tr><tr><td class="NLM_term">JH1</td><td class="NLM_def"><p class="first last">kinase domain</p></td></tr><tr><td class="NLM_term">JH2</td><td class="NLM_def"><p class="first last">pseudokinase domain</p></td></tr><tr><td class="NLM_term">FEP</td><td class="NLM_def"><p class="first last">free-energy perturbation</p></td></tr><tr><td class="NLM_term">FP</td><td class="NLM_def"><p class="first last">fluorescence polarization</p></td></tr><tr><td class="NLM_term">MST</td><td class="NLM_def"><p class="first last">microscale thermophoresis</p></td></tr><tr><td class="NLM_term">STAT</td><td class="NLM_def"><p class="first last">signal transducer and activator of transcription</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i88">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_68373" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_68373" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 37 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">OâShea, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holland, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Staudt, L. M.</span></span> <span> </span><span class="NLM_article-title">JAKs and STATs in immunity, immunodeficiency, and cancer</span>. <i>N. Engl. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>368</i></span>,  <span class="NLM_fpage">161</span>â <span class="NLM_lpage">170</span>, <span class="refDoi">Â DOI: 10.1056/NEJMra1202117</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=10.1056%2FNEJMra1202117" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=23301733" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=1%3ACAS%3A528%3ADC%252BC3sXovVyhuw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=368&publication_year=2013&pages=161-170&author=J.+J.+O%E2%80%99Sheaauthor=S.+M.+Hollandauthor=L.+M.+Staudt&title=JAKs+and+STATs+in+immunity%2C+immunodeficiency%2C+and+cancer&doi=10.1056%2FNEJMra1202117"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">JAKs and STATs in immunity, immunodeficiency, and cancer</span></div><div class="casAuthors">O'Shea, John J.; Holland, Steven M.; Staudt, Louis M.</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">368</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">161-170</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">A review highlights interferon-mediated gene regulation has resulted in a new framework for the study of cell signaling and JAKs STATs in immunity, immunodeficiency, and cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqk1n-UgWrlWrVg90H21EOLACvtfcHk0lhrNZullU4VTQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXovVyhuw%253D%253D&md5=5d87b67f72b721eb0ca9413adf92a7b9</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1056%2FNEJMra1202117&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMra1202117%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Shea%26aufirst%3DJ.%2BJ.%26aulast%3DHolland%26aufirst%3DS.%2BM.%26aulast%3DStaudt%26aufirst%3DL.%2BM.%26atitle%3DJAKs%2520and%2520STATs%2520in%2520immunity%252C%2520immunodeficiency%252C%2520and%2520cancer%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2013%26volume%3D368%26spage%3D161%26epage%3D170%26doi%3D10.1056%2FNEJMra1202117" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Leroy, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Constantinescu, S. N.</span></span> <span> </span><span class="NLM_article-title">Rethinking JAK2 inhibition: towards novel strategies of more specific and versatile Janus kinase inhibition</span>. <i>Leukemia</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">1023</span>â <span class="NLM_lpage">1038</span>, <span class="refDoi">Â DOI: 10.1038/leu.2017.43</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=10.1038%2Fleu.2017.43" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=28119526" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=1%3ACAS%3A528%3ADC%252BC2sXislyms7w%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2017&pages=1023-1038&author=E.+Leroyauthor=S.+N.+Constantinescu&title=Rethinking+JAK2+inhibition%3A+towards+novel+strategies+of+more+specific+and+versatile+Janus+kinase+inhibition&doi=10.1038%2Fleu.2017.43"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Rethinking JAK2 inhibition: towards novel strategies of more specific and versatile janus kinase inhibition</span></div><div class="casAuthors">Leroy, E.; Constantinescu, S. N.</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1023-1038</span>CODEN:
                <span class="NLM_cas:coden">LEUKED</span>;
        ISSN:<span class="NLM_cas:issn">0887-6924</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Janus kinases (JAKs) are required for cytokine receptor signaling.  Since the discovery of the highly prevalent JAK2 V617F mutation in myeloproliferative neoplasms (MPNs), JAK2 became a prime target for inhibition.  Only one approved JAK2 inhibitor exists, with pos., but not curative effects in MPNs, and promising effects in autoimmune diseases and cancer.  On the basis of recent advances in the structural features regulating both normal and mutant JAKs, as well as in small-mol. targeting, we review the current state of JAK2 inhibitor development and present novel avenues of selecting JAK2 inhibitors, with broad and narrow specificities and extend these approaches to other JAKs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLVJS8PwHOlbVg90H21EOLACvtfcHk0lhych49XBYvNg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXislyms7w%253D&md5=0dc333d94e27daad1d32110dee26e637</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fleu.2017.43&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fleu.2017.43%26sid%3Dliteratum%253Aachs%26aulast%3DLeroy%26aufirst%3DE.%26aulast%3DConstantinescu%26aufirst%3DS.%2BN.%26atitle%3DRethinking%2520JAK2%2520inhibition%253A%2520towards%2520novel%2520strategies%2520of%2520more%2520specific%2520and%2520versatile%2520Janus%2520kinase%2520inhibition%26jtitle%3DLeukemia%26date%3D2017%26volume%3D31%26spage%3D1023%26epage%3D1038%26doi%3D10.1038%2Fleu.2017.43" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baxter, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">East, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fourouclas, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swanton, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vassiliou, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bench, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boyd, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curtin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erber, W. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, A. R.</span></span> <span> </span><span class="NLM_article-title">Acquired mutation of the tyrosine kinase JAK2 in human myeloproliferative disorders</span>. <i>Lancet</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>365</i></span>,  <span class="NLM_fpage">1054</span>â <span class="NLM_lpage">1061</span>, <span class="refDoi">Â DOI: 10.1016/S0140-6736(05)71142-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=10.1016%2FS0140-6736%2805%2971142-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=15781101" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=1%3ACAS%3A528%3ADC%252BD2MXisFGrs7g%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=365&publication_year=2005&pages=1054-1061&author=E.+J.+Baxterauthor=L.+M.+Scottauthor=P.+J.+Campbellauthor=C.+Eastauthor=N.+Fourouclasauthor=S.+Swantonauthor=G.+S.+Vassiliouauthor=A.+J.+Benchauthor=E.+M.+Boydauthor=N.+Curtinauthor=M.+A.+Scottauthor=W.+N.+Erberauthor=A.+R.+Green&title=Acquired+mutation+of+the+tyrosine+kinase+JAK2+in+human+myeloproliferative+disorders&doi=10.1016%2FS0140-6736%2805%2971142-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Acquired mutation of the tyrosine kinase JAK2 in human myeloproliferative disorders</span></div><div class="casAuthors">Baxter, E. Joanna; Scott, Linda M.; Campbell, Peter J.; East, Clare; Fourouclas, Nasios; Swanton, Soheila; Vassiliou, George S.; Bench, Anthony J.; Boyd, Elaine M.; Curtin, Natasha; Scott, Mike A.; Erber, Wendy N.; Green, Anthony R.</div><div class="citationInfo"><span class="NLM_cas:title">Lancet</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">365</span>
        (<span class="NLM_cas:issue">9464</span>),
    <span class="NLM_cas:pages">1054-1061</span>CODEN:
                <span class="NLM_cas:coden">LANCAO</span>;
        ISSN:<span class="NLM_cas:issn">0140-6736</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Background: Human myeloproliferative disorders form a range of clonal haematol. malignant diseases, the main members of which are polycythemia vera, essential thrombocythemia, and idiopathic myelofibrosis.  The mol. pathogenesis of these disorders is unknown, but tyrosine kinases have been implicated in several related disorders.  We investigated the role of the cytoplasmic tyrosine kinase JAK2 in patients with a myeloproliferative disorder.  Methods: We obtained DNA samples from patients with polycythemia vera, essential thrombocythemia, or idiopathic myelofibrosis.  The coding exons of JAK2 were bidirectionally sequenced from peripheral-blood granulocytes, T cells, or both.  Allele-specific PCR, mol. cytogenetic studies, microsatellite PCR, Affymetrix single nucleotide polymorphism array analyses, and colony assays were undertaken on subgroups of patients.  Findings: A single point mutation (Val617Phe) was identified in JAK2 in 71 (97%) of 73 patients with polycythemia vera, 29 (57%) of 51 with essential thrombocythemia, and eight (50%) of 16 with idiopathic myelofibrosis.  The mutation is acquired, is present in a variable proportion of granulocytes, alters a highly conserved valine present in the neg. regulatory JH2 domain, and is predicted to dysregulate kinase activity.  It was heterozygous in most patients, homozygous in a subset as a result of mitotic recombination, and arose in a multipotent progenitor capable of giving rise to erythroid and myeloid cells.  The mutation was present in all erythropoietin-independent erythroid colonies.  Interpretation: A single acquired mutation of JAK2 was noted in more than half of patients with a myeloproliferative disorder.  Its presence in all erythropoietin-independent erythroid colonies demonstrates a link with growth factor hypersensitivity, a key biol. feature of these disorders.  Relevance to practice: Identification of the Val617Phe JAK2 mutation lays the foundation for new approaches to the diagnosis, classification, and treatment of myeloproliferative disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8cc-kg2leZrVg90H21EOLACvtfcHk0lhych49XBYvNg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXisFGrs7g%253D&md5=3c9dc565a1986a47d12e4bccd8c8cc1a</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2FS0140-6736%2805%2971142-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0140-6736%252805%252971142-9%26sid%3Dliteratum%253Aachs%26aulast%3DBaxter%26aufirst%3DE.%2BJ.%26aulast%3DScott%26aufirst%3DL.%2BM.%26aulast%3DCampbell%26aufirst%3DP.%2BJ.%26aulast%3DEast%26aufirst%3DC.%26aulast%3DFourouclas%26aufirst%3DN.%26aulast%3DSwanton%26aufirst%3DS.%26aulast%3DVassiliou%26aufirst%3DG.%2BS.%26aulast%3DBench%26aufirst%3DA.%2BJ.%26aulast%3DBoyd%26aufirst%3DE.%2BM.%26aulast%3DCurtin%26aufirst%3DN.%26aulast%3DScott%26aufirst%3DM.%2BA.%26aulast%3DErber%26aufirst%3DW.%2BN.%26aulast%3DGreen%26aufirst%3DA.%2BR.%26atitle%3DAcquired%2520mutation%2520of%2520the%2520tyrosine%2520kinase%2520JAK2%2520in%2520human%2520myeloproliferative%2520disorders%26jtitle%3DLancet%26date%3D2005%26volume%3D365%26spage%3D1054%26epage%3D1061%26doi%3D10.1016%2FS0140-6736%2805%2971142-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Levine, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wadleigh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cools, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ebert, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wernig, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huntly, B. J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boggon, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wlodarska, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moore, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adelsperger, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabriel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mercher, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DâAndrea, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">FrÃ¶hling, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DÃ¶hner, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marynen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vandenberghe, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mesa, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tefferi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffein, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eck, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sellers, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyerson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golub, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilliland, D. G.</span></span> <span> </span><span class="NLM_article-title">Activating mutation in the tyrosine kinase JAK2 in polycythemia vera, essential thrombocythemia, and myeloid metaplasia with myelofibrosis</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">387</span>â <span class="NLM_lpage">397</span>, <span class="refDoi">Â DOI: 10.1016/j.ccr.2005.03.023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=10.1016%2Fj.ccr.2005.03.023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=15837627" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=1%3ACAS%3A528%3ADC%252BD2MXjvFSku7o%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2005&pages=387-397&author=R.+L.+Levineauthor=M.+Wadleighauthor=J.+Coolsauthor=B.+L.+Ebertauthor=G.+Wernigauthor=B.+J.+P.+Huntlyauthor=T.+J.+Boggonauthor=L.+Wlodarskaauthor=J.+J.+Clarkauthor=S.+Mooreauthor=J.+Adelspergerauthor=S.+Kooauthor=J.+C.+Leeauthor=S.+Gabrielauthor=T.+Mercherauthor=A.+D%E2%80%99Andreaauthor=S.+Fr%C3%B6hlingauthor=K.+D%C3%B6hnerauthor=P.+Marynenauthor=P.+Vandenbergheauthor=R.+A.+Mesaauthor=A.+Tefferiauthor=J.+D.+Griffeinauthor=M.+J.+Eckauthor=W.+R.+Sellersauthor=M.+Meyersonauthor=T.+R.+Golubauthor=S.+J.+Leeauthor=D.+G.+Gilliland&title=Activating+mutation+in+the+tyrosine+kinase+JAK2+in+polycythemia+vera%2C+essential+thrombocythemia%2C+and+myeloid+metaplasia+with+myelofibrosis&doi=10.1016%2Fj.ccr.2005.03.023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Activating mutation in the tyrosine kinase JAK2 in polycythemia vera, essential thrombocythemia, and myeloid metaplasia with myelofibrosis</span></div><div class="casAuthors">Levine, Ross L.; Wadleigh, Martha; Cools, Jan; Ebert, Benjamin L.; Wernig, Gerlinde; Huntly, Brian J. P.; Boggon, Titus J.; Wlodarska, Iwona; Clark, Jennifer J.; Moore, Sandra; Adelsperger, Jennifer; Koo, Sumin; Lee, Jeffrey C.; Gabriel, Stacey; Mercher, Thomas; D'Andrea, Alan; Froehling, Stefan; Doehner, Konstanze; Marynen, Peter; Vandenberghe, Peter; Mesa, Ruben A.; Tefferi, Ayalew; Griffin, James D.; Eck, Michael J.; Sellers, William R.; Meyerson, Matthew; Golub, Todd R.; Lee, Stephanie J.; Gilliland, D. Gary</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">387-397</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Polycythemia vera (PV), essential thrombocythemia (ET), and myeloid metaplasia with myelofibrosis (MMM) are clonal disorders arising from hematopoietic progenitors.  An internet-based protocol was used to collect clin. information and biol. specimens from patients with these diseases.  High-throughput DNA re-sequencing identified a recurrent somatic missense mutation JAK2V617F in granulocyte DNA samples of 121 of 164 PV patients, of which 41 had homozygous and 80 had heterozygous mutations.  Mol. and cytogenetic analyses demonstrated that homozygous mutations were due to duplication of the mutant allele.  JAK2V617F was also identified in granulocyte DNA samples from 37 of 115 ET and 16 of 46 MMM patients, but was not obsd. in 269 normal individuals.  In vitro anal. demonstrated that JAK2V617F is a constitutively active tyrosine kinase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXJNzUFm3Q3bVg90H21EOLACvtfcHk0lhych49XBYvNg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXjvFSku7o%253D&md5=2a3e893867b114e8749cd6f5e55990b7</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2005.03.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2005.03.023%26sid%3Dliteratum%253Aachs%26aulast%3DLevine%26aufirst%3DR.%2BL.%26aulast%3DWadleigh%26aufirst%3DM.%26aulast%3DCools%26aufirst%3DJ.%26aulast%3DEbert%26aufirst%3DB.%2BL.%26aulast%3DWernig%26aufirst%3DG.%26aulast%3DHuntly%26aufirst%3DB.%2BJ.%2BP.%26aulast%3DBoggon%26aufirst%3DT.%2BJ.%26aulast%3DWlodarska%26aufirst%3DL.%26aulast%3DClark%26aufirst%3DJ.%2BJ.%26aulast%3DMoore%26aufirst%3DS.%26aulast%3DAdelsperger%26aufirst%3DJ.%26aulast%3DKoo%26aufirst%3DS.%26aulast%3DLee%26aufirst%3DJ.%2BC.%26aulast%3DGabriel%26aufirst%3DS.%26aulast%3DMercher%26aufirst%3DT.%26aulast%3DD%25E2%2580%2599Andrea%26aufirst%3DA.%26aulast%3DFr%25C3%25B6hling%26aufirst%3DS.%26aulast%3DD%25C3%25B6hner%26aufirst%3DK.%26aulast%3DMarynen%26aufirst%3DP.%26aulast%3DVandenberghe%26aufirst%3DP.%26aulast%3DMesa%26aufirst%3DR.%2BA.%26aulast%3DTefferi%26aufirst%3DA.%26aulast%3DGriffein%26aufirst%3DJ.%2BD.%26aulast%3DEck%26aufirst%3DM.%2BJ.%26aulast%3DSellers%26aufirst%3DW.%2BR.%26aulast%3DMeyerson%26aufirst%3DM.%26aulast%3DGolub%26aufirst%3DT.%2BR.%26aulast%3DLee%26aufirst%3DS.%2BJ.%26aulast%3DGilliland%26aufirst%3DD.%2BG.%26atitle%3DActivating%2520mutation%2520in%2520the%2520tyrosine%2520kinase%2520JAK2%2520in%2520polycythemia%2520vera%252C%2520essential%2520thrombocythemia%252C%2520and%2520myeloid%2520metaplasia%2520with%2520myelofibrosis%26jtitle%3DCancer%2520Cell%26date%3D2005%26volume%3D7%26spage%3D387%26epage%3D397%26doi%3D10.1016%2Fj.ccr.2005.03.023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">HammarÃ©n, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ungureanu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grisouard, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skoda, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silvennoinen, O.</span></span> <span> </span><span class="NLM_article-title">ATP binding to the pseudokinase domain of JAK2 is critical for pathogenic activation</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>112</i></span>,  <span class="NLM_fpage">4642</span>â <span class="NLM_lpage">4647</span>, <span class="refDoi">Â DOI: 10.1073/pnas.1423201112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=10.1073%2Fpnas.1423201112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=25825724" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlsVOntL8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2015&pages=4642-4647&author=H.+M.+Hammar%C3%A9nauthor=D.+Ungureanuauthor=J.+Grisouardauthor=R.+C.+Skodaauthor=S.+R.+Hubbardauthor=O.+Silvennoinen&title=ATP+binding+to+the+pseudokinase+domain+of+JAK2+is+critical+for+pathogenic+activation&doi=10.1073%2Fpnas.1423201112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">ATP binding to the pseudokinase domain of JAK2 is critical for pathogenic activation</span></div><div class="casAuthors">Hammaren, Henrik M.; Ungureanu, Daniela; Grisouard, Jean; Skoda, Radek C.; Hubbard, Stevan R.; Silvennoinen, Olli</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">4642-4647</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Pseudokinases lack conserved motifs typically required for kinase activity.  Nearly half of pseudokinases bind ATP, but only few retain phosphotransfer activity, leaving the functional role of nucleotide binding in most cases unknown.  Janus kinases (JAKs) are nonreceptor tyrosine kinases with a tandem pseudokinase-kinase domain configuration, where the pseudokinase domain (JAK homol. 2, JH2) has important regulatory functions and harbors mutations underlying hematol. and immunol. diseases.  JH2 of JAK1, JAK2, and TYK2 all bind ATP, but the significance of this is unclear.  We characterize the role of nucleotide binding in normal and pathogenic JAK signaling using comprehensive structure-based mutagenesis.  Disruption of JH2 ATP binding in wild-type JAK2 has only minor effects, and in the presence of type I cytokine receptors, the mutations do not affect JAK2 activation.  However, JH2 mutants devoid of ATP binding ameliorate the hyperactivation of JAK2 V617F.  Disrupting ATP binding in JH2 also inhibits the hyperactivity of other pathogenic JAK2 mutants, as well as of JAK1 V658F, and prevents induction of erythrocytosis in a JAK2 V617F myeloproliferative neoplasm mouse model.  Mol. dynamic simulations and thermal-shift anal. indicate that ATP binding stabilizes JH2, with a pronounced effect on the C helix region, which plays a crit. role in pathogenic activation of JAK2.  Taken together, our results suggest that ATP binding to JH2 serves a structural role in JAKs, which is required for aberrant activity of pathogenic JAK mutants.  The inhibitory effect of abrogating JH2 ATP binding in pathogenic JAK mutants may warrant novel therapeutic approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowpDHSf5yGQ7Vg90H21EOLACvtfcHk0lhFL6-6DGqu2w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlsVOntL8%253D&md5=7cb7b90a1978ef02628f9ce9e85622ad</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1423201112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1423201112%26sid%3Dliteratum%253Aachs%26aulast%3DHammar%25C3%25A9n%26aufirst%3DH.%2BM.%26aulast%3DUngureanu%26aufirst%3DD.%26aulast%3DGrisouard%26aufirst%3DJ.%26aulast%3DSkoda%26aufirst%3DR.%2BC.%26aulast%3DHubbard%26aufirst%3DS.%2BR.%26aulast%3DSilvennoinen%26aufirst%3DO.%26atitle%3DATP%2520binding%2520to%2520the%2520pseudokinase%2520domain%2520of%2520JAK2%2520is%2520critical%2520for%2520pathogenic%2520activation%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2015%26volume%3D112%26spage%3D4642%26epage%3D4647%26doi%3D10.1073%2Fpnas.1423201112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kung, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jura, N.</span></span> <span> </span><span class="NLM_article-title">Prospects for pharmacological targeting of pseudokinases</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">501</span>â <span class="NLM_lpage">526</span>, <span class="refDoi">Â DOI: 10.1038/s41573-019-0018-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=10.1038%2Fs41573-019-0018-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=30850748" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=1%3ACAS%3A528%3ADC%252BC1MXms1GjtLw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=501-526&author=J.+E.+Kungauthor=N.+Jura&title=Prospects+for+pharmacological+targeting+of+pseudokinases&doi=10.1038%2Fs41573-019-0018-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Prospects for pharmacological targeting of pseudokinases</span></div><div class="casAuthors">Kung, Jennifer E.; Jura, Natalia</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">501-526</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  Pseudokinases are members of the protein kinase superfamily but signal primarily through noncatalytic mechanisms.  Many pseudokinases contribute to the pathologies of human diseases, yet they remain largely unexplored as drug targets owing to challenges assocd. with modulation of their biol. functions.  Our understanding of the structure and physiol. roles of pseudokinases has improved substantially over the past decade, revealing intriguing similarities between pseudokinases and their catalytically active counterparts.  Pseudokinases often adopt conformations that are analogous to those seen in catalytically active kinases and, in some cases, can also bind metal cations and/or nucleotides.  Several clin. approved kinase inhibitors have been shown to influence the noncatalytic functions of active kinases, providing hope that similar properties in pseudokinases could be pharmacol. regulated.  In this Review, we discuss known roles of pseudokinases in disease, their unique structural features and the progress that has been made towards developing pseudokinase-directed therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUFxQnlHTnn7Vg90H21EOLACvtfcHk0lhFL6-6DGqu2w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXms1GjtLw%253D&md5=f3414770154e8d6449b454f9bc3f19bd</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fs41573-019-0018-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41573-019-0018-3%26sid%3Dliteratum%253Aachs%26aulast%3DKung%26aufirst%3DJ.%2BE.%26aulast%3DJura%26aufirst%3DN.%26atitle%3DProspects%2520for%2520pharmacological%2520targeting%2520of%2520pseudokinases%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2019%26volume%3D18%26spage%3D501%26epage%3D526%26doi%3D10.1038%2Fs41573-019-0018-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Connolly, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wetter, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, W. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emanuel, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gruninger, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuentes-Pesquera, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rugg, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Middleton, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jolliffe, L. K.</span></span> <span> </span><span class="NLM_article-title">1-acyl-1<i>H</i>-[1,2,4]triazole-3,5-diamine analogues as novel and potent anticancer cyclin-dependent kinase inhibitors: synthesis and evaluation of biological activities</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">4208</span>â <span class="NLM_lpage">4211</span>, <span class="refDoi">Â DOI: 10.1021/jm050267e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm050267e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=1%3ACAS%3A528%3ADC%252BD2MXkslSgsbs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=4208-4211&author=R.+Linauthor=P.+J.+Connollyauthor=S.+Huangauthor=S.+K.+Wetterauthor=Y.+Luauthor=W.+V.+Murrayauthor=S.+L.+Emanuelauthor=R.+H.+Gruningerauthor=A.+R.+Fuentes-Pesqueraauthor=C.+A.+Ruggauthor=S.+A.+Middletonauthor=L.+K.+Jolliffe&title=1-acyl-1H-%5B1%2C2%2C4%5Dtriazole-3%2C5-diamine+analogues+as+novel+and+potent+anticancer+cyclin-dependent+kinase+inhibitors%3A+synthesis+and+evaluation+of+biological+activities&doi=10.1021%2Fjm050267e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">1-Acyl-1H-[1,2,4]triazole-3,5-diamine Analogues as Novel and Potent Anticancer Cyclin-Dependent Kinase Inhibitors: Synthesis and Evaluation of Biological Activities</span></div><div class="casAuthors">Lin, Ronghui; Connolly, Peter J.; Huang, Shenlin; Wetter, Steven K.; Lu, Yanhua; Murray, William V.; Emanuel, Stuart L.; Gruninger, Robert H.; Fuentes-Pesquera, Angel R.; Rugg, Catherine A.; Middleton, Steven A.; Jolliffe, Linda K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">4208-4211</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of 1-acyl-1H-[1,2,4]triazole-3,5-diamine analogs, e.g., I, were synthesized as cyclin-dependent kinase (CDK) inhibitors.  These compds. showed potent and selective CDK1 and CDK2 inhibitory activities and inhibited in vitro cellular proliferation in various human tumor cells.  I demonstrated in vivo efficacy in a human melanoma A375 xenograft model in nude mice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrN9p2ArpQa4bVg90H21EOLACvtfcHk0lhCNcV2r4CKPQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXkslSgsbs%253D&md5=6ccbcbc6136bfd61d20c5e369d8fb31a</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Fjm050267e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm050267e%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DR.%26aulast%3DConnolly%26aufirst%3DP.%2BJ.%26aulast%3DHuang%26aufirst%3DS.%26aulast%3DWetter%26aufirst%3DS.%2BK.%26aulast%3DLu%26aufirst%3DY.%26aulast%3DMurray%26aufirst%3DW.%2BV.%26aulast%3DEmanuel%26aufirst%3DS.%2BL.%26aulast%3DGruninger%26aufirst%3DR.%2BH.%26aulast%3DFuentes-Pesquera%26aufirst%3DA.%2BR.%26aulast%3DRugg%26aufirst%3DC.%2BA.%26aulast%3DMiddleton%26aufirst%3DS.%2BA.%26aulast%3DJolliffe%26aufirst%3DL.%2BK.%26atitle%3D1-acyl-1H-%255B1%252C2%252C4%255Dtriazole-3%252C5-diamine%2520analogues%2520as%2520novel%2520and%2520potent%2520anticancer%2520cyclin-dependent%2520kinase%2520inhibitors%253A%2520synthesis%2520and%2520evaluation%2520of%2520biological%2520activities%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D4208%26epage%3D4211%26doi%3D10.1021%2Fjm050267e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Karaman, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrgard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treiber, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallant, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atteridge, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciceri, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edeen, P. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faraoni, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Floyd, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lockhart, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milanov, Z. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morrison, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pallares, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, H. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pritchard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wodicka, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zarrinkar, P. P.</span></span> <span> </span><span class="NLM_article-title">A quantitative analysis of kinase inhibitor selectivity</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">127</span>â <span class="NLM_lpage">132</span>, <span class="refDoi">Â DOI: 10.1038/nbt1358</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=10.1038%2Fnbt1358" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=18183025" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=1%3ACAS%3A528%3ADC%252BD1cXisFGlsQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2008&pages=127-132&author=M.+W.+Karamanauthor=S.+Herrgardauthor=D.+K.+Treiberauthor=P.+Gallantauthor=C.+E.+Atteridgeauthor=B.+T.+Campbellauthor=K.+W.+Chanauthor=P.+Ciceriauthor=M.+I.+Davisauthor=P.+T.+Edeenauthor=R.+Faraoniauthor=M.+Floydauthor=J.+P.+Huntauthor=D.+J.+Lockhartauthor=Z.+V.+Milanovauthor=M.+J.+Morrisonauthor=G.+Pallaresauthor=H.+K.+Patelauthor=S.+Pritchardauthor=L.+M.+Wodickaauthor=P.+P.+Zarrinkar&title=A+quantitative+analysis+of+kinase+inhibitor+selectivity&doi=10.1038%2Fnbt1358"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">A quantitative analysis of kinase inhibitor selectivity</span></div><div class="casAuthors">Karaman, Mazen W.; Herrgard, Sanna; Treiber, Daniel K.; Gallant, Paul; Atteridge, Corey E.; Campbell, Brian T.; Chan, Katrina W.; Ciceri, Pietro; Davis, Mindy I.; Edeen, Philip T.; Faraoni, Raffaella; Floyd, Mark; Hunt, Jeremy P.; Lockhart, Daniel J.; Milanov, Zdravko V.; Morrison, Michael J.; Pallares, Gabriel; Patel, Hitesh K.; Pritchard, Stephanie; Wodicka, Lisa M.; Zarrinkar, Patrick P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">127-132</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Kinase inhibitors are a new class of therapeutics with a propensity to inhibit multiple targets.  The biol. consequences of multikinase activity are poorly defined, and an important step toward understanding the relationship between selectivity, efficacy and safety is the exploration of how inhibitors interact with the human kinome.  The authors present interaction maps for 38 kinase inhibitors across a panel of 317 kinases representing >50% of the predicted human protein kinome.  The data constitute the most comprehensive study of kinase inhibitor selectivity to date and reveal a wide diversity of interaction patterns.  To enable a global anal. of the results, the authors introduce the concept of a selectivity score as a general tool to quantify and differentiate the obsd. interaction patterns.  The authors further investigate the impact of panel size and find that small assay panels do not provide a robust measure of selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiArJSq2_lPrVg90H21EOLACvtfcHk0lhCNcV2r4CKPQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXisFGlsQ%253D%253D&md5=346265d412853ced636ad4128ed8a76f</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnbt1358&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1358%26sid%3Dliteratum%253Aachs%26aulast%3DKaraman%26aufirst%3DM.%2BW.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DGallant%26aufirst%3DP.%26aulast%3DAtteridge%26aufirst%3DC.%2BE.%26aulast%3DCampbell%26aufirst%3DB.%2BT.%26aulast%3DChan%26aufirst%3DK.%2BW.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DDavis%26aufirst%3DM.%2BI.%26aulast%3DEdeen%26aufirst%3DP.%2BT.%26aulast%3DFaraoni%26aufirst%3DR.%26aulast%3DFloyd%26aufirst%3DM.%26aulast%3DHunt%26aufirst%3DJ.%2BP.%26aulast%3DLockhart%26aufirst%3DD.%2BJ.%26aulast%3DMilanov%26aufirst%3DZ.%2BV.%26aulast%3DMorrison%26aufirst%3DM.%2BJ.%26aulast%3DPallares%26aufirst%3DG.%26aulast%3DPatel%26aufirst%3DH.%2BK.%26aulast%3DPritchard%26aufirst%3DS.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26atitle%3DA%2520quantitative%2520analysis%2520of%2520kinase%2520inhibitor%2520selectivity%26jtitle%3DNat.%2520Biotechnol.%26date%3D2008%26volume%3D26%26spage%3D127%26epage%3D132%26doi%3D10.1038%2Fnbt1358" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Puleo, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kucera, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">HammarÃ©n, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ungureanu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Newton, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silvennoinen, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jorgensen, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlessinger, J.</span></span> <span> </span><span class="NLM_article-title">Identification and characterization of JAK2 pseudokinase domain small molecule binders</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">618</span>â <span class="NLM_lpage">621</span>, <span class="refDoi">Â DOI: 10.1021/acsmedchemlett.7b00153</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.7b00153" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=1%3ACAS%3A528%3ADC%252BC2sXnvFClsr4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=618-621&author=D.+E.+Puleoauthor=K.+Kuceraauthor=H.+M.+Hammar%C3%A9nauthor=D.+Ungureanuauthor=A.+S.+Newtonauthor=O.+Silvennoinenauthor=W.+L.+Jorgensenauthor=J.+Schlessinger&title=Identification+and+characterization+of+JAK2+pseudokinase+domain+small+molecule+binders&doi=10.1021%2Facsmedchemlett.7b00153"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and Characterization of JAK2 Pseudokinase Domain Small Molecule Binders</span></div><div class="casAuthors">Puleo, David E.; Kucera, Kaury; Hammaren, Henrik M.; Ungureanu, Daniela; Newton, Ana S.; Silvennoinen, Olli; Jorgensen, William L.; Schlessinger, Joseph</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">618-621</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Janus kinases (JAKs) regulate hematopoiesis via the cytokine-mediated JAK-STAT signaling pathway.  JAKs contain tandem C-terminal pseudokinase (JH2) and tyrosine kinase (JH1) domains.  The JAK2 pseudokinase domain adopts a protein kinase fold and, despite its pseudokinase designation, binds ATP with micromolar affinity.  Recent evidence shows that displacing ATP from the JAK2 JH2 domain alters the hyperactivation state of the oncogenic JAK2 V617F protein while sparing the wild type JAK2 protein.  In this study, small mol. binders of JAK2 JH2 were identified via an in vitro screen.  Top hits were characterized using biophys. and structural approaches.  Development of pseudokinase-selective compds. may offer novel pharmacol. opportunities for treating cancers driven by JAK2 V617F and other oncogenic JAK mutants.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmtcoH5qmUWrVg90H21EOLACvtfcHk0lhCNcV2r4CKPQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXnvFClsr4%253D&md5=a3f1f6e10a8a1d3b64bb5691d246f5d0</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.7b00153&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.7b00153%26sid%3Dliteratum%253Aachs%26aulast%3DPuleo%26aufirst%3DD.%2BE.%26aulast%3DKucera%26aufirst%3DK.%26aulast%3DHammar%25C3%25A9n%26aufirst%3DH.%2BM.%26aulast%3DUngureanu%26aufirst%3DD.%26aulast%3DNewton%26aufirst%3DA.%2BS.%26aulast%3DSilvennoinen%26aufirst%3DO.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DSchlessinger%26aufirst%3DJ.%26atitle%3DIdentification%2520and%2520characterization%2520of%2520JAK2%2520pseudokinase%2520domain%2520small%2520molecule%2520binders%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D8%26spage%3D618%26epage%3D621%26doi%3D10.1021%2Facsmedchemlett.7b00153" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bandaranayake, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ungureanu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silvennoinen, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, S. R.</span></span> <span> </span><span class="NLM_article-title">Crystal structures of the JAK2 pseudokinase domain and the pathogenic mutant V617F</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">754</span>â <span class="NLM_lpage">759</span>, <span class="refDoi">Â DOI: 10.1038/nsmb.2348</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=10.1038%2Fnsmb.2348" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=22820988" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVKntLbM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2012&pages=754-759&author=R.+M.+Bandaranayakeauthor=D.+Ungureanuauthor=Y.+Shanauthor=D.+E.+Shawauthor=O.+Silvennoinenauthor=S.+R.+Hubbard&title=Crystal+structures+of+the+JAK2+pseudokinase+domain+and+the+pathogenic+mutant+V617F&doi=10.1038%2Fnsmb.2348"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structures of the JAK2 pseudokinase domain and the pathogenic mutant V617F</span></div><div class="casAuthors">Bandaranayake, Rajintha M.; Ungureanu, Daniela; Shan, Yibing; Shaw, David E.; Silvennoinen, Olli; Hubbard, Stevan R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">754-759</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The protein tyrosine kinase JAK2 mediates signaling through numerous cytokine receptors.  JAK2 possesses a pseudokinase domain (JH2) and a tyrosine kinase domain (JH1).  Through unknown mechanisms, JH2 regulates the catalytic activity of JH1, and hyperactivating mutations in the JH2 region of human JAK2 cause myeloproliferative neoplasms (MPNs).  We showed previously that JAK2 JH2 is, in fact, catalytically active.  Here we present crystal structures of human JAK2 JH2, including both wild type and the most prevalent MPN mutant, V617F.  The structures reveal that JH2 adopts the fold of a prototypical protein kinase but binds Mg-ATP noncanonically.  The structural and biochem. data indicate that the V617F mutation rigidifies Î±-helix C in the N lobe of JH2, facilitating trans-phosphorylation of JH1.  The crystal structures of JH2 afford new opportunities for the design of novel JAK2 therapeutics targeting MPNs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGj9DoN3fj37Vg90H21EOLACvtfcHk0lgAUq5Ik-toVA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVKntLbM&md5=da64398430cd3279e131b2569cdc7391</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.2348&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.2348%26sid%3Dliteratum%253Aachs%26aulast%3DBandaranayake%26aufirst%3DR.%2BM.%26aulast%3DUngureanu%26aufirst%3DD.%26aulast%3DShan%26aufirst%3DY.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26aulast%3DSilvennoinen%26aufirst%3DO.%26aulast%3DHubbard%26aufirst%3DS.%2BR.%26atitle%3DCrystal%2520structures%2520of%2520the%2520JAK2%2520pseudokinase%2520domain%2520and%2520the%2520pathogenic%2520mutant%2520V617F%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2012%26volume%3D19%26spage%3D754%26epage%3D759%26doi%3D10.1038%2Fnsmb.2348" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dusa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mouton, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pecquet, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herman, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Constantinescu, S. N.</span></span> <span> </span><span class="NLM_article-title">JAK2 V617F constitutive activation requires JH2 residue F595: a pseudokinase domain target for specific inhibitors</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>5</i></span>, <span class="NLM_elocation-id">e11157</span> <span class="refDoi">Â DOI: 10.1371/journal.pone.0011157</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=10.1371%2Fjournal.pone.0011157" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=20585391" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=1%3ACAS%3A280%3ADC%252BC3cnivVShsw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2010&author=A.+Dusaauthor=C.+Moutonauthor=C.+Pecquetauthor=M.+Hermanauthor=S.+N.+Constantinescu&title=JAK2+V617F+constitutive+activation+requires+JH2+residue+F595%3A+a+pseudokinase+domain+target+for+specific+inhibitors&doi=10.1371%2Fjournal.pone.0011157"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">JAK2 V617F constitutive activation requires JH2 residue F595: a pseudokinase domain target for specific inhibitors</span></div><div class="casAuthors">Dusa Alexandra; Mouton Celine; Pecquet Christian; Herman Murielle; Constantinescu Stefan N</div><div class="citationInfo"><span class="NLM_cas:title">PloS one</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">e11157</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The JAK2 V617F mutation present in over 95% of Polycythemia Vera patients and in 50% of Essential Thrombocythemia and Primary Myelofibrosis patients renders the kinase constitutively active.  In the absence of a three-dimensional structure for the full-length protein, the mechanism of activation of JAK2 V617F has remained elusive.  In this study, we used functional mutagenesis to investigate the involvement of the JH2 alphaC helix in the constitutive activation of JAK2 V617F.  We show that residue F595, located in the middle of the alphaC helix of JH2, is indispensable for the constitutive activity of JAK2 V617F.  Mutation of F595 to Ala, Lys, Val or Ile significantly decreases the constitutive activity of JAK2 V617F, but F595W and F595Y are able to restore it, implying an aromaticity requirement at position 595.  Substitution of F595 to Ala was also able to decrease the constitutive activity of two other JAK2 mutants, T875N and R683G, as well as JAK2 K539L, albeit to a lower extent.  In contrast, the F595 mutants are activated by erythropoietin-bound EpoR.  We also explored the relationship between the dimeric conformation of EpoR and several JAK2 mutants.  Since residue F595 is crucial to the constitutive activation of JAK2 V617F but not to initiation of JAK2 activation by cytokines, we suggest that small molecules that target the region around this residue might specifically block oncogenic JAK2 and spare JAK2 wild-type.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ-dhwX-6R6Z8KL0FlyGHDofW6udTcc2ebL1BO2mpHfc7ntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3cnivVShsw%253D%253D&md5=0b95c5aa9f0644c1f1d94d2a09c858e5</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0011157&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0011157%26sid%3Dliteratum%253Aachs%26aulast%3DDusa%26aufirst%3DA.%26aulast%3DMouton%26aufirst%3DC.%26aulast%3DPecquet%26aufirst%3DC.%26aulast%3DHerman%26aufirst%3DM.%26aulast%3DConstantinescu%26aufirst%3DS.%2BN.%26atitle%3DJAK2%2520V617F%2520constitutive%2520activation%2520requires%2520JH2%2520residue%2520F595%253A%2520a%2520pseudokinase%2520domain%2520target%2520for%2520specific%2520inhibitors%26jtitle%3DPLoS%2520One%26date%3D2010%26volume%3D5%26doi%3D10.1371%2Fjournal.pone.0011157" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jorgensen, W. L.</span></span> <span> </span><span class="NLM_article-title">Efficient drug lead discovery and optimization</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">724</span>â <span class="NLM_lpage">733</span>, <span class="refDoi">Â DOI: 10.1021/ar800236t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ar800236t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjsFentr4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2009&pages=724-733&author=W.+L.+Jorgensen&title=Efficient+drug+lead+discovery+and+optimization&doi=10.1021%2Far800236t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Efficient Drug Lead Discovery and Optimization</span></div><div class="casAuthors">Jorgensen, William L.</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">724-733</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  During the 1980s, advances in the abilities to perform computer simulations of chem. and biomol. systems and to calc. free energy changes led to the expectation that such methodol. would soon show great utility for guiding mol. design.  Important potential applications included design of selective receptors, catalysts, and regulators of biol. function including enzyme inhibitors.  This time also saw the rise of high-throughput screening and combinatorial chem. along with complementary computational methods for de novo design and virtual screening including docking.  These technologies appeared poised to deliver diverse lead compds. for any biol. target.  As with many technol. advances, realization of the expectations required significant addnl. effort and time.  However, as summarized here, striking success has now been achieved for computer-aided drug lead generation and optimization.  De novo design using both mol. growing and docking are illustrated for lead generation, and lead optimization features free energy perturbation calcns. in conjunction with Monte Carlo statistical mechanics simulations for protein-inhibitor complexes in aq. soln.  The specific applications are to the discovery of non-nucleoside inhibitors of HIV reverse transcriptase (HIV-RT) and inhibitors of the binding of the proinflammatory cytokine MIF to its receptor CD74.  A std. protocol is presented that includes scans for possible addns. of small substituents to a mol. core, interchange of heterocycles, and focused optimization of substituents at one site.  Initial leads with activities at low-micromolar concns. have been advanced rapidly to low-nanomolar inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrn0YpWuo1jCLVg90H21EOLACvtfcHk0lgAUq5Ik-toVA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjsFentr4%253D&md5=9641587769701c6541b38e18fa05538a</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Far800236t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Far800236t%26sid%3Dliteratum%253Aachs%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26atitle%3DEfficient%2520drug%2520lead%2520discovery%2520and%2520optimization%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2009%26volume%3D42%26spage%3D724%26epage%3D733%26doi%3D10.1021%2Far800236t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jorgensen, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tirado-Rives, J.</span></span> <span> </span><span class="NLM_article-title">Molecular modeling of organic and biomolecular systems using BOSS and MCPRO</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">1689</span>â <span class="NLM_lpage">1700</span>, <span class="refDoi">Â DOI: 10.1002/jcc.20297</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=10.1002%2Fjcc.20297" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=16200637" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1SlsbbN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2005&pages=1689-1700&author=W.+L.+Jorgensenauthor=J.+Tirado-Rives&title=Molecular+modeling+of+organic+and+biomolecular+systems+using+BOSS+and+MCPRO&doi=10.1002%2Fjcc.20297"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular modeling of organic and biomolecular systems using BOSS and MCPRO</span></div><div class="casAuthors">Jorgensen, William L.; Tirado-Rives, Julian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1689-1700</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">An overview is provided of the capabilities for the current versions of the BOSS and MCPRO programs for mol. modeling of org. and biomol. systems.  Recent applications are noted, particularly for QM/MM studies of org. and enzymic reactions and for protein-ligand binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdrU3aGV4BQLVg90H21EOLACvtfcHk0lgAUq5Ik-toVA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1SlsbbN&md5=e487e9838d39e4b39a5d06a240bfc1e0</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20297&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20297%26sid%3Dliteratum%253Aachs%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DTirado-Rives%26aufirst%3DJ.%26atitle%3DMolecular%2520modeling%2520of%2520organic%2520and%2520biomolecular%2520systems%2520using%2520BOSS%2520and%2520MCPRO%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2005%26volume%3D26%26spage%3D1689%26epage%3D1700%26doi%3D10.1002%2Fjcc.20297" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Robertson, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tirado-Rives, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jorgensen, W. L.</span></span> <span> </span><span class="NLM_article-title">Improved peptide and protein torsional energetics with the OPLS-AA force field</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">3499</span>â <span class="NLM_lpage">3509</span>, <span class="refDoi">Â DOI: 10.1021/acs.jctc.5b00356</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jctc.5b00356" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=1%3ACAS%3A528%3ADC%252BC2MXpsFOlsbc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=3499-3509&author=M.+J.+Robertsonauthor=J.+Tirado-Rivesauthor=W.+L.+Jorgensen&title=Improved+peptide+and+protein+torsional+energetics+with+the+OPLS-AA+force+field&doi=10.1021%2Facs.jctc.5b00356"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Improved Peptide and Protein Torsional Energetics with the OPLS-AA Force Field</span></div><div class="casAuthors">Robertson, Michael J.; Tirado-Rives, Julian; Jorgensen, William L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3499-3509</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The development and validation of new peptide dihedral parameters are reported for the OPLS-AA force field.  High accuracy quantum chem. methods were used to scan Ï, Ï, Ï1, and Ï2 potential energy surfaces for blocked dipeptides.  New Fourier coeffs. for the dihedral angle terms of the OPLS-AA force field were fit to these surfaces, utilizing a Boltzmann-weighted error function and systematically examg. the effects of weighting temp.  To prevent overfitting to the available data, a minimal no. of new residue-specific and peptide-specific torsion terms were developed.  Extensive exptl. soln.-phase and quantum chem. gas-phase benchmarks were used to assess the quality of the new parameters, named OPLS-AA/M, demonstrating significant improvement over previous OPLS-AA force fields.  A Boltzmann weighting temp. of 2000 K was detd. to be optimal for fitting the new Fourier coeffs. for dihedral angle parameters.  Conclusions are drawn from the results for best practices for developing new torsion parameters for protein force fields.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxE-hNg6xKqLVg90H21EOLACvtfcHk0lhc179ZHUgZ7w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXpsFOlsbc%253D&md5=b9836b19acadc1a86369bca744ca9b13</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Facs.jctc.5b00356&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jctc.5b00356%26sid%3Dliteratum%253Aachs%26aulast%3DRobertson%26aufirst%3DM.%2BJ.%26aulast%3DTirado-Rives%26aufirst%3DJ.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26atitle%3DImproved%2520peptide%2520and%2520protein%2520torsional%2520energetics%2520with%2520the%2520OPLS-AA%2520force%2520field%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2015%26volume%3D11%26spage%3D3499%26epage%3D3509%26doi%3D10.1021%2Facs.jctc.5b00356" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jorgensen, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tirado-Rives, J.</span></span> <span> </span><span class="NLM_article-title">Potential energy functions for atomic-level simulations of water and organic and biomolecular systems</span>. <i>Proc. Nat. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>102</i></span>,  <span class="NLM_fpage">6665</span>â <span class="NLM_lpage">6670</span>, <span class="refDoi">Â DOI: 10.1073/pnas.0408037102</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=10.1073%2Fpnas.0408037102" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=15870211" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=1%3ACAS%3A528%3ADC%252BD2MXksVKgt7s%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2005&pages=6665-6670&author=W.+L.+Jorgensenauthor=J.+Tirado-Rives&title=Potential+energy+functions+for+atomic-level+simulations+of+water+and+organic+and+biomolecular+systems&doi=10.1073%2Fpnas.0408037102"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Potential energy functions for atomic-level simulations of water and organic and biomolecular systems</span></div><div class="casAuthors">Jorgensen, William L.; Tirado-Rives, Julian</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">6665-6670</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">A review.  An overview is provided on the development and status of potential energy functions that are used in at.-level statistical mechanics and mol. dynamics simulations of water and of org. and biomol. systems.  Some topics that are considered are the form of force fields, their parameterization and performance, simulations of org. liqs., computation of free energies of hydration, universal extension for org. mols., and choice of at. charges.  The discussion of water models covers some history, performance issues, and special topics such as nuclear quantum effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkn9AFAagzKbVg90H21EOLACvtfcHk0lhc179ZHUgZ7w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXksVKgt7s%253D&md5=633dfc421f1a18cf99ca8cb11d347b3d</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0408037102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0408037102%26sid%3Dliteratum%253Aachs%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DTirado-Rives%26aufirst%3DJ.%26atitle%3DPotential%2520energy%2520functions%2520for%2520atomic-level%2520simulations%2520of%2520water%2520and%2520organic%2520and%2520biomolecular%2520systems%26jtitle%3DProc.%2520Nat.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2005%26volume%3D102%26spage%3D6665%26epage%3D6670%26doi%3D10.1073%2Fpnas.0408037102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lea, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simeonov, A.</span></span> <span> </span><span class="NLM_article-title">Fluorescence polarization assays in small molecule screening</span>. <i>Expert Opin. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">17</span>â <span class="NLM_lpage">32</span>, <span class="refDoi">Â DOI: 10.1517/17460441.2011.537322</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=10.1517%2F17460441.2011.537322" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=22328899" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SrtbvP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=17-32&author=W.+A.+Leaauthor=A.+Simeonov&title=Fluorescence+polarization+assays+in+small+molecule+screening&doi=10.1517%2F17460441.2011.537322"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Fluorescence polarization assays in small molecule screening</span></div><div class="casAuthors">Lea, Wendy A.; Simeonov, Anton</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Discovery</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">17-32</span>CODEN:
                <span class="NLM_cas:coden">EODDBX</span>;
        ISSN:<span class="NLM_cas:issn">1746-0441</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Importance of the field: Fluorescence polarization (FP) is a homogeneous method that allows rapid and quant. anal. of diverse mol. interactions and enzyme activities.  This technique has been widely utilized in clin. and biomedical settings, including the diagnosis of certain diseases and monitoring therapeutic drug levels in body fluids.  Recent developments in the field have been symbolized by the facile adoption of FP in high-throughput screening and small mol. drug discovery of an increasing range of target classes.  Areas covered in this review: The article provides a brief overview of the theor. foundation of FP, followed by updates on recent advancements in its application for various drug target classes, including GPCRs, enzymes and protein-protein interactions.  The strengths and weaknesses of this method, practical considerations in assay design, novel applications and future directions are also discussed.  What the reader will gain: The reader is informed of the most recent advancements and future directions of FP application to small mol. screening.  Take home message: In addn. to its continued utilization in high-throughput screening, FP has expanded into new disease and target areas and has been marked by increased use of labeled small mol. ligands for receptor-binding studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqudLso-a4wR7Vg90H21EOLACvtfcHk0lhc179ZHUgZ7w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SrtbvP&md5=6f102f40b56fd7a13c4fb59b64a3f883</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1517%2F17460441.2011.537322&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F17460441.2011.537322%26sid%3Dliteratum%253Aachs%26aulast%3DLea%26aufirst%3DW.%2BA.%26aulast%3DSimeonov%26aufirst%3DA.%26atitle%3DFluorescence%2520polarization%2520assays%2520in%2520small%2520molecule%2520screening%26jtitle%3DExpert%2520Opin.%2520Drug%2520Discovery%26date%3D2011%26volume%3D6%26spage%3D17%26epage%3D32%26doi%3D10.1517%2F17460441.2011.537322" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Newton, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deiana, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Puleo, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cisneros, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cutrona, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlessinger, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jorgensen, W. L.</span></span> <span> </span><span class="NLM_article-title">JAK2 JH2 fluorescence polarization assay and crystal structures for complexes with three small molecules</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">614</span>â <span class="NLM_lpage">617</span>, <span class="refDoi">Â DOI: 10.1021/acsmedchemlett.7b00154</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.7b00154" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=1%3ACAS%3A528%3ADC%252BC2sXnvFCnsbY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=614-617&author=A.+S.+Newtonauthor=L.+Deianaauthor=D.+E.+Puleoauthor=J.+A.+Cisnerosauthor=K.+J.+Cutronaauthor=J.+Schlessingerauthor=W.+L.+Jorgensen&title=JAK2+JH2+fluorescence+polarization+assay+and+crystal+structures+for+complexes+with+three+small+molecules&doi=10.1021%2Facsmedchemlett.7b00154"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">JAK2 JH2 Fluorescence Polarization Assay and Crystal Structures for Complexes with Three Small Molecules</span></div><div class="casAuthors">Newton, Ana S.; Deiana, Luca; Puleo, David E.; Cisneros, Jose A.; Cutrona, Kara J.; Schlessinger, Joseph; Jorgensen, William L.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">614-617</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A competitive fluorescence polarization (FP) assay is reported for detg. binding affinities of probe mols. with the pseudokinase JAK2 JH2 allosteric site.  The syntheses of the fluorescent 5 and 6 used in the assay are reported as well as Kd results for 10 compds., including JNJ7706621, NVP-BSK805, and filgotinib (GLPG0634).  X-ray crystal structures of JAK2 JH2 in complex with NVP-BSK805, filgotinib, and diaminopyrimidine 8 elucidate the binding poses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQUy01N09zFrVg90H21EOLACvtfcHk0lgEOrUIvtF_Ig"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXnvFCnsbY%253D&md5=e7c6f7c58bc9a9fddf18a72b5df4f964</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.7b00154&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.7b00154%26sid%3Dliteratum%253Aachs%26aulast%3DNewton%26aufirst%3DA.%2BS.%26aulast%3DDeiana%26aufirst%3DL.%26aulast%3DPuleo%26aufirst%3DD.%2BE.%26aulast%3DCisneros%26aufirst%3DJ.%2BA.%26aulast%3DCutrona%26aufirst%3DK.%2BJ.%26aulast%3DSchlessinger%26aufirst%3DJ.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26atitle%3DJAK2%2520JH2%2520fluorescence%2520polarization%2520assay%2520and%2520crystal%2520structures%2520for%2520complexes%2520with%2520three%2520small%2520molecules%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D8%26spage%3D614%26epage%3D617%26doi%3D10.1021%2Facsmedchemlett.7b00154" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McNally, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dekker, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vangrevelinghe, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">ChÃ¨ne, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Machauer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Radimerski, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eck, M. J.</span></span> <span> </span><span class="NLM_article-title">Discovery and structural characterization of ATP-site ligands for the wild-type and V617F-mutant JAK2 pseudokinase domain</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">587</span>â <span class="NLM_lpage">593</span>, <span class="refDoi">Â DOI: 10.1021/acschembio.8b00722</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.8b00722" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjtVSgtro%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2019&pages=587-593&author=R.+McNallyauthor=Q.+Liauthor=K.+Liauthor=C.+Dekkerauthor=E.+Vangrevelingheauthor=M.+Jonesauthor=P.+Ch%C3%A8neauthor=R.+Machauerauthor=T.+Radimerskiauthor=M.+J.+Eck&title=Discovery+and+structural+characterization+of+ATP-site+ligands+for+the+wild-type+and+V617F-mutant+JAK2+pseudokinase+domain&doi=10.1021%2Facschembio.8b00722"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Structural Characterization of ATP-Site Ligands for the Wild-Type and V617F Mutant JAK2 Pseudokinase Domain</span></div><div class="casAuthors">McNally, Randall; Li, Qing; Li, Kunhua; Dekker, Carien; Vangrevelinghe, Eric; Jones, Matthew; Chene, Patrick; Machauer, Rainer; Radimerski, Thomas; Eck, Michael J.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">587-593</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The oncogenic V617F mutation lies in the pseudokinase domain of JAK2, marking it as a potential target for development of compds. that might inhibit the pathogenic activity of the mutant protein.  We used differential scanning fluorometry to identify compds. that bind the JAK2 pseudokinase domain.  Crystal structures of five candidate compds. with the wild-type domain reveal their modes of binding.  Exploration of analogs of screening hit BI-D1870 led to the identification of compd. 2, a 123 nM ligand for the pseudokinase domain.  Interestingly, crystal structures of the V617F domain in complex with two unrelated compds. reveal a conformation that is characteristic of the wild-type domain, rather than that previously obsd. for the V617F mutant.  These structures suggest that certain ATP-site ligands can modulate the V617F allosteric site, thereby providing a mechanistic rationale for targeting the pseudokinase domain and a structural foundation for development of more potent and pseudokinase-selective compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqyQ-36fpAvIrVg90H21EOLACvtfcHk0lgEOrUIvtF_Ig"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjtVSgtro%253D&md5=e8664a0fd8d0bcb5eb69252745b39b5a</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Facschembio.8b00722&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.8b00722%26sid%3Dliteratum%253Aachs%26aulast%3DMcNally%26aufirst%3DR.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DLi%26aufirst%3DK.%26aulast%3DDekker%26aufirst%3DC.%26aulast%3DVangrevelinghe%26aufirst%3DE.%26aulast%3DJones%26aufirst%3DM.%26aulast%3DCh%25C3%25A8ne%26aufirst%3DP.%26aulast%3DMachauer%26aufirst%3DR.%26aulast%3DRadimerski%26aufirst%3DT.%26aulast%3DEck%26aufirst%3DM.%2BJ.%26atitle%3DDiscovery%2520and%2520structural%2520characterization%2520of%2520ATP-site%2520ligands%2520for%2520the%2520wild-type%2520and%2520V617F-mutant%2520JAK2%2520pseudokinase%2520domain%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2019%26volume%3D14%26spage%3D587%26epage%3D593%26doi%3D10.1021%2Facschembio.8b00722" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hron, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jursic, B. S.</span></span> <span> </span><span class="NLM_article-title">Preparation of substituted semicarbazides from corresponding amines and hydrazines via phenyl carbamates</span>. <i>Tetrahedron Lett.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">1540</span>â <span class="NLM_lpage">1543</span>, <span class="refDoi">Â DOI: 10.1016/j.tetlet.2014.01.052</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=10.1016%2Fj.tetlet.2014.01.052" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFyisr4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2014&pages=1540-1543&author=R.+Hronauthor=B.+S.+Jursic&title=Preparation+of+substituted+semicarbazides+from+corresponding+amines+and+hydrazines+via+phenyl+carbamates&doi=10.1016%2Fj.tetlet.2014.01.052"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Preparation of substituted semicarbazides from corresponding amines and hydrazines via phenyl carbamates</span></div><div class="casAuthors">Hron, Rebecca; Jursic, Branko S.</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron Letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1540-1543</span>CODEN:
                <span class="NLM_cas:coden">TELEAY</span>;
        ISSN:<span class="NLM_cas:issn">0040-4039</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A simple synthetic procedure for the conversion of amines and hydrazines into substituted semicarbazides I [R1 = H, Me; R2 = H, Ph, 2,4-(O2N)2C6H3, etc.] was developed.  The initial condensation between the desired amine and Ph chloroformate into Ph carbamate is followed by the addn. of hydrazine under basic conditions.  The reaction is tolerable to a variety of functional groups, with mild conditions and high percent yields.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIO9hp-QRnCLVg90H21EOLACvtfcHk0liALMe7x8I-cg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitFyisr4%253D&md5=36d06f3d0656557576c776c8406aa36a</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.tetlet.2014.01.052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tetlet.2014.01.052%26sid%3Dliteratum%253Aachs%26aulast%3DHron%26aufirst%3DR.%26aulast%3DJursic%26aufirst%3DB.%2BS.%26atitle%3DPreparation%2520of%2520substituted%2520semicarbazides%2520from%2520corresponding%2520amines%2520and%2520hydrazines%2520via%2520phenyl%2520carbamates%26jtitle%3DTetrahedron%2520Lett.%26date%3D2014%26volume%3D55%26spage%3D1540%26epage%3D1543%26doi%3D10.1016%2Fj.tetlet.2014.01.052" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Verrier, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoarau, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsais, F.</span></span> <span> </span><span class="NLM_article-title">Palladium-catalyzed direct (hetero)arylation of ethyl oxazole-4-carboxylate: an efficient access to (hetero)aryloxazoles</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">7383</span>â <span class="NLM_lpage">7386</span>, <span class="refDoi">Â DOI: 10.1021/jo801093n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo801093n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=1%3ACAS%3A528%3ADC%252BD1cXpvVarsLg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2008&pages=7383-7386&author=C.+Verrierauthor=T.+Martinauthor=C.+Hoarauauthor=F.+Marsais&title=Palladium-catalyzed+direct+%28hetero%29arylation+of+ethyl+oxazole-4-carboxylate%3A+an+efficient+access+to+%28hetero%29aryloxazoles&doi=10.1021%2Fjo801093n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Palladium-Catalyzed Direct (Hetero)arylation of Ethyl Oxazole-4-carboxylate: An Efficient Access to (Hetero)aryloxazoles</span></div><div class="casAuthors">Verrier, Cecile; Martin, Thibaut; Hoarau, Christophe; Marsais, Francis</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">7383-7386</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A straightforward route toward 2-(hetero)arylated and 2,5-di(hetero)arylated oxazoles through regiocontrolled palladium-catalyzed direct (hetero)arylation of Et oxazole-4-carboxylate with iodo-, bromo-, and chloro(hetero)aroms. followed by a two-step hydrolysis/decarboxylation sequence was described.  The method was applied here to a neat synthesis of two 2,5-di(hetero)aryloxazole natural products, balsoxin and texaline.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJK_x9guZfj7Vg90H21EOLACvtfcHk0liALMe7x8I-cg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXpvVarsLg%253D&md5=d34b4dd16ec3449962a90f0649b0866d</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjo801093n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo801093n%26sid%3Dliteratum%253Aachs%26aulast%3DVerrier%26aufirst%3DC.%26aulast%3DMartin%26aufirst%3DT.%26aulast%3DHoarau%26aufirst%3DC.%26aulast%3DMarsais%26aufirst%3DF.%26atitle%3DPalladium-catalyzed%2520direct%2520%2528hetero%2529arylation%2520of%2520ethyl%2520oxazole-4-carboxylate%253A%2520an%2520efficient%2520access%2520to%2520%2528hetero%2529aryloxazoles%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2008%26volume%3D73%26spage%3D7383%26epage%3D7386%26doi%3D10.1021%2Fjo801093n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mattsson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DahlstrÃ¶m, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karlsson, S.</span></span> <span> </span><span class="NLM_article-title">A mild hydrolysis of esters mediated by lithium salts</span>. <i>Tetrahedron Lett.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">2497</span>â <span class="NLM_lpage">2499</span>, <span class="refDoi">Â DOI: 10.1016/j.tetlet.2007.02.029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=10.1016%2Fj.tetlet.2007.02.029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=1%3ACAS%3A528%3ADC%252BD2sXis1Ohs7w%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2007&pages=2497-2499&author=S.+Mattssonauthor=M.+Dahlstr%C3%B6mauthor=S.+Karlsson&title=A+mild+hydrolysis+of+esters+mediated+by+lithium+salts&doi=10.1016%2Fj.tetlet.2007.02.029"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">A mild hydrolysis of esters mediated by lithium salts</span></div><div class="casAuthors">Mattsson, Sara; Dahlstroem, Mikael; Karlsson, Staffan</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron Letters</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">2497-2499</span>CODEN:
                <span class="NLM_cas:coden">TELEAY</span>;
        ISSN:<span class="NLM_cas:issn">0040-4039</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">When treated with amine bases such as triethylamine and various lithium salts in wet solvents, esters are efficiently hydrolyzed to the corresponding acids in good yields.  Esters incorporating an Î±- or Î²-heteroatom with respect to the ester carbonyl group are hydrolyzed rapidly even at room temp.  To further demonstrate the usefulness of this method, one example is provided where hydrolysis of acetylated camphorsultam is mediated by LiBr.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlkdhTIjsx0bVg90H21EOLACvtfcHk0liALMe7x8I-cg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXis1Ohs7w%253D&md5=7a88787176f93ccdd555a0ab411b0635</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.tetlet.2007.02.029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tetlet.2007.02.029%26sid%3Dliteratum%253Aachs%26aulast%3DMattsson%26aufirst%3DS.%26aulast%3DDahlstr%25C3%25B6m%26aufirst%3DM.%26aulast%3DKarlsson%26aufirst%3DS.%26atitle%3DA%2520mild%2520hydrolysis%2520of%2520esters%2520mediated%2520by%2520lithium%2520salts%26jtitle%3DTetrahedron%2520Lett.%26date%3D2007%26volume%3D48%26spage%3D2497%26epage%3D2499%26doi%3D10.1016%2Fj.tetlet.2007.02.029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">SchÃ¤rfer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulz-Gasch, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ehrlich, H.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guba, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rarey, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stahl, M.</span></span> <span> </span><span class="NLM_article-title">Torsion angle preferences in druglike chemical space: a comprehensive guide</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">2016</span>â <span class="NLM_lpage">2028</span>, <span class="refDoi">Â DOI: 10.1021/jm3016816</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm3016816" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=1%3ACAS%3A280%3ADC%252BC3szltlGitw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=2016-2028&author=C.+Sch%C3%A4rferauthor=T.+Schulz-Gaschauthor=H.-C.+Ehrlichauthor=W.+Gubaauthor=M.+Rareyauthor=M.+Stahl&title=Torsion+angle+preferences+in+druglike+chemical+space%3A+a+comprehensive+guide&doi=10.1021%2Fjm3016816"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Torsion angle preferences in druglike chemical space: a comprehensive guide</span></div><div class="casAuthors">Scharfer Christin; Schulz-Gasch Tanja; Ehrlich Hans-Christian; Guba Wolfgang; Rarey Matthias; Stahl Martin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of medicinal chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2016-28</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Crystal structure databases offer ample opportunities to derive small molecule conformation preferences, but the derived knowledge is not systematically applied in drug discovery research.  We address this gap by a comprehensive and extendable expert system enabling quick assessment of the probability of a given conformation to occur.  It is based on a hierarchical system of torsion patterns that cover a large part of druglike chemical space.  Each torsion pattern has associated frequency histograms generated from CSD and PDB data and, derived from the histograms, traffic-light rules for frequently observed, rare, and highly unlikely torsion ranges.  Structures imported into the corresponding software are annotated according to these rules.  We present the concept behind the tree of torsion patterns, the design of an intuitive user interface for the management and usage of the torsion library, and we illustrate how the system helps analyze and understand conformation properties of substructures widely used in medicinal chemistry.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR2PDX1V9jtEkEgZCMqtRBifW6udTcc2ebeXKi8gAwgVrntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3szltlGitw%253D%253D&md5=7d0c2f9177cef90f0d7376836a8cefb2</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fjm3016816&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm3016816%26sid%3Dliteratum%253Aachs%26aulast%3DSch%25C3%25A4rfer%26aufirst%3DC.%26aulast%3DSchulz-Gasch%26aufirst%3DT.%26aulast%3DEhrlich%26aufirst%3DH.-C.%26aulast%3DGuba%26aufirst%3DW.%26aulast%3DRarey%26aufirst%3DM.%26aulast%3DStahl%26aufirst%3DM.%26atitle%3DTorsion%2520angle%2520preferences%2520in%2520druglike%2520chemical%2520space%253A%2520a%2520comprehensive%2520guide%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D2016%26epage%3D2028%26doi%3D10.1021%2Fjm3016816" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arjunan, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santhanam, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rani, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohan, S.</span></span> <span> </span><span class="NLM_article-title">Conformational, vibrational, NMR and DFT studies of N-methylacetanilide</span>. <i>Spectrochim. Acta, Part A</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>104</i></span>,  <span class="NLM_fpage">182</span>â <span class="NLM_lpage">196</span>, <span class="refDoi">Â DOI: 10.1016/j.saa.2012.11.037</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=10.1016%2Fj.saa.2012.11.037" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=23266693" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFeit7o%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2013&pages=182-196&author=V.+Arjunanauthor=R.+Santhanamauthor=T.+Raniauthor=H.+Rosiauthor=S.+Mohan&title=Conformational%2C+vibrational%2C+NMR+and+DFT+studies+of+N-methylacetanilide&doi=10.1016%2Fj.saa.2012.11.037"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational, vibrational, NMR and DFT studies of N-methylacetanilide</span></div><div class="casAuthors">Arjunan, V.; Santhanam, R.; Rani, T.; Rosi, H.; Mohan, S.</div><div class="citationInfo"><span class="NLM_cas:title">Spectrochimica Acta, Part A: Molecular and Biomolecular Spectroscopy</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">182-196</span>CODEN:
                <span class="NLM_cas:coden">SAMCAS</span>;
        ISSN:<span class="NLM_cas:issn">1386-1425</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A detailed conformational, vibrational, NMR and DFT studies of N-methylacetanilide have been carried out.  In DFT, B3LYP method have been used with 6-31G(d,p), 6-311++G(d,p) and cc-pVTZ basis sets.  The vibrational frequencies were calcd. resulting in IR and Raman frequencies together with intensities and Raman depolarisation ratios.  The dipole moment derivs. were computed anal.  Owing to the complexity of the mol., the potential energy distributions of the vibrational modes of the compd. are also calcd.  Isoelectronic mol. electrostatic potential surface (MEP) and electron d. surface were examd. 1H and 13C NMR isotropic chem. shifts were calcd. and the assignments made are compared with the exptl. values.  The energies of important MO's of the compd. were also detd. from TD-DFT method.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2BmuWe6H7U7Vg90H21EOLACvtfcHk0lixideBLQpqDA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFeit7o%253D&md5=d44916cfff43da61fa4297c1fc043898</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.saa.2012.11.037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.saa.2012.11.037%26sid%3Dliteratum%253Aachs%26aulast%3DArjunan%26aufirst%3DV.%26aulast%3DSanthanam%26aufirst%3DR.%26aulast%3DRani%26aufirst%3DT.%26aulast%3DRosi%26aufirst%3DH.%26aulast%3DMohan%26aufirst%3DS.%26atitle%3DConformational%252C%2520vibrational%252C%2520NMR%2520and%2520DFT%2520studies%2520of%2520N-methylacetanilide%26jtitle%3DSpectrochim.%2520Acta%252C%2520Part%2520A%26date%3D2013%26volume%3D104%26spage%3D182%26epage%3D196%26doi%3D10.1016%2Fj.saa.2012.11.037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Frisch, M. J.</span>; <span class="NLM_string-name">Trucks, G. W.</span>; <span class="NLM_string-name">Schlegel, H. B.</span>; <span class="NLM_string-name">Scuseria, G. E.</span>; <span class="NLM_string-name">Robb, M. A.</span>; <span class="NLM_string-name">Cheeseman, J. R.</span>; <span class="NLM_string-name">Scalmani, G.</span>; <span class="NLM_string-name">Barone, V.</span>; <span class="NLM_string-name">Petersson, G. A.</span>; <span class="NLM_string-name">Nakatsuji, H.</span>; <span class="NLM_string-name">Li, X.</span>; <span class="NLM_string-name">Caricato, M.</span>; <span class="NLM_string-name">Marenich, A. V.</span>; <span class="NLM_string-name">Bloino, J.</span>; <span class="NLM_string-name">Janesko, B. G.</span>; <span class="NLM_string-name">Gomperts, R.</span>; <span class="NLM_string-name">Mennucci, B.</span>; <span class="NLM_string-name">Hratchian, H. P.</span>; <span class="NLM_string-name">Ortiz, J. V.</span>; <span class="NLM_string-name">Izmaylov, A. F.</span>; <span class="NLM_string-name">Sonnenberg, J. L.</span>; <span class="NLM_string-name">Williams-Young, D.</span>; <span class="NLM_string-name">Ding, F.</span>; <span class="NLM_string-name">Lipparini, F.</span>; <span class="NLM_string-name">Egidi, F.</span>; <span class="NLM_string-name">Goings, J.</span>; <span class="NLM_string-name">Peng, B.</span>; <span class="NLM_string-name">Petrone, A.</span>; <span class="NLM_string-name">Henderson, T.</span>; <span class="NLM_string-name">Ranasinghe, D.</span>; <span class="NLM_string-name">Zakrzewski, V. G.</span>; <span class="NLM_string-name">Gao, J.</span>; <span class="NLM_string-name">Rega, N.</span>; <span class="NLM_string-name">Zheng, G.</span>; <span class="NLM_string-name">Liang, W.</span>; <span class="NLM_string-name">Hada, M.</span>; <span class="NLM_string-name">Ehara, M.</span>; <span class="NLM_string-name">Toyota, K.</span>; <span class="NLM_string-name">Fukuda, R.</span>; <span class="NLM_string-name">Hasegawa, J.</span>; <span class="NLM_string-name">Ishida, M.</span>; <span class="NLM_string-name">Nakajima, T.</span>; <span class="NLM_string-name">Honda, Y.</span>; <span class="NLM_string-name">Kitao, O.</span>; <span class="NLM_string-name">Nakai, H.</span>; <span class="NLM_string-name">Vreven, T.</span>; <span class="NLM_string-name">Throssell, K.</span>; <span class="NLM_string-name">Montgomery, J. A.,  Jr.</span>; <span class="NLM_string-name">Peralta, J. E.</span>; <span class="NLM_string-name">Ogliaro, F.</span>; <span class="NLM_string-name">Bearpark, M. J.</span>; <span class="NLM_string-name">Heyd, J. J.</span>; <span class="NLM_string-name">Brothers, E. N.</span>; <span class="NLM_string-name">Kudin, K. N.</span>; <span class="NLM_string-name">Staroverov, V. N.</span>; <span class="NLM_string-name">Keith, T. A.</span>; <span class="NLM_string-name">Kobayashi, R.</span>; <span class="NLM_string-name">Normand, J.</span>; <span class="NLM_string-name">Raghavachari, K.</span>; <span class="NLM_string-name">Rendell, A. P.</span>; <span class="NLM_string-name">Burant, J. C.</span>; <span class="NLM_string-name">Iyengar, S. S.</span>; <span class="NLM_string-name">Tomasi, J.</span>; <span class="NLM_string-name">Cossi, M.</span>; <span class="NLM_string-name">Millam, J. M.</span>; <span class="NLM_string-name">Klene, M.</span>; <span class="NLM_string-name">Adamo, C.</span>; <span class="NLM_string-name">Cammi, R.</span>; <span class="NLM_string-name">Ochterski, J. W.</span>; <span class="NLM_string-name">Martin, R. L.</span>; <span class="NLM_string-name">Morokuma, K.</span>; <span class="NLM_string-name">Farkas, O.</span>; <span class="NLM_string-name">Foresman, J. B.</span>; <span class="NLM_string-name">Fox, D. J.</span></span> <i>Gaussian 16</i>, Revision A.03, <span class="NLM_publisher-name">Gaussian, Inc.</span>: <span class="NLM_publisher-loc">Wallingford CT</span>, <span class="NLM_year">2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=M.+J.+Frisch&author=G.+W.+Trucks&author=H.+B.+Schlegel&author=G.+E.+Scuseria&author=M.+A.+Robb&author=J.+R.+Cheeseman&author=G.+Scalmani&author=V.+Barone&author=G.+A.+Petersson&author=H.+Nakatsuji&author=X.+Li&author=M.+Caricato&author=A.+V.+Marenich&author=J.+Bloino&author=B.+G.+Janesko&author=R.+Gomperts&author=B.+Mennucci&author=H.+P.+Hratchian&author=J.+V.+Ortiz&author=A.+F.+Izmaylov&author=J.+L.+Sonnenberg&author=D.+Williams-Young&author=F.+Ding&author=F.+Lipparini&author=F.+Egidi&author=J.+Goings&author=B.+Peng&author=A.+Petrone&author=T.+Henderson&author=D.+Ranasinghe&author=V.+G.+Zakrzewski&author=J.+Gao&author=N.+Rega&author=G.+Zheng&author=W.+Liang&author=M.+Hada&author=M.+Ehara&author=K.+Toyota&author=R.+Fukuda&author=J.+Hasegawa&author=M.+Ishida&author=T.+Nakajima&author=Y.+Honda&author=O.+Kitao&author=H.+Nakai&author=T.+Vreven&author=K.+Throssell&author=J.+A.+Montgomery&author=J.+E.+Peralta&author=F.+Ogliaro&author=M.+J.+Bearpark&author=J.+J.+Heyd&author=E.+N.+Brothers&author=K.+N.+Kudin&author=V.+N.+Staroverov&author=T.+A.+Keith&author=R.+Kobayashi&author=J.+Normand&author=K.+Raghavachari&author=A.+P.+Rendell&author=J.+C.+Burant&author=S.+S.+Iyengar&author=J.+Tomasi&author=M.+Cossi&author=J.+M.+Millam&author=M.+Klene&author=C.+Adamo&author=R.+Cammi&author=J.+W.+Ochterski&author=R.+L.+Martin&author=K.+Morokuma&author=O.+Farkas&author=J.+B.+Foresman&author=D.+J.+Fox&title=Gaussian+16"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DFrisch%26aufirst%3DM.%2BJ.%26jtitle%3DGaussian%252016%26pub%3DGaussian%252C%2520Inc%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hansch, C.</span>; <span class="NLM_string-name">Leo, A.</span>; <span class="NLM_string-name">Hoekman, D.</span></span> <i>Exploring QSAR â hydrophobic, electronic, and steric constants</i>, <span class="NLM_publisher-name">American Chemical Society</span>: <span class="NLM_publisher-loc">Washington</span>, <span class="NLM_year">1995</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1995&author=C.+Hansch&author=A.+Leo&author=D.+Hoekman&title=Exploring+QSAR+%E2%80%93+hydrophobic%2C+electronic%2C+and+steric+constants"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DHansch%26aufirst%3DC.%26btitle%3DExploring%2520QSAR%2520%25E2%2580%2593%2520hydrophobic%252C%2520electronic%252C%2520and%2520steric%2520constants%26pub%3DAmerican%2520Chemical%2520Society%26date%3D1995" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jerabek-Willemsen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">AndrÃ©, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wanner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duhr, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baaske, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breitsprecher, D.</span></span> <span> </span><span class="NLM_article-title">MicroScale thermophoresis: interaction analysis and beyond</span>. <i>J. Mol. Struct.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>1077</i></span>,  <span class="NLM_fpage">101</span>â <span class="NLM_lpage">113</span>, <span class="refDoi">Â DOI: 10.1016/j.molstruc.2014.03.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=10.1016%2Fj.molstruc.2014.03.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=1%3ACAS%3A528%3ADC%252BC2cXlt1Wisbo%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1077&publication_year=2014&pages=101-113&author=M.+Jerabek-Willemsenauthor=T.+Andr%C3%A9author=R.+Wannerauthor=H.+M.+Rothauthor=S.+Duhrauthor=P.+Baaskeauthor=D.+Breitsprecher&title=MicroScale+thermophoresis%3A+interaction+analysis+and+beyond&doi=10.1016%2Fj.molstruc.2014.03.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">MicroScale Thermophoresis: Interaction analysis and beyond</span></div><div class="casAuthors">Jerabek-Willemsen, Moran; Andre, Timon; Wanner, Randy; Roth, Heide Marie; Duhr, Stefan; Baaske, Philipp; Breitsprecher, Dennis</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Structure</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">1077</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">101-113</span>CODEN:
                <span class="NLM_cas:coden">JMOSB4</span>;
        ISSN:<span class="NLM_cas:issn">0022-2860</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  MicroScale Thermophoresis (MST) is a powerful technique to quantify biomol. interactions.  It is based on thermophoresis, the directed movement of mols. in a temp. gradient, which strongly depends on a variety of mol. properties such as size, charge, hydration shell or conformation.  Thus, this technique is highly sensitive to virtually any change in mol. properties, allowing for a precise quantification of mol. events independent of the size or nature of the studied specimen.  During a MST expt., a temp. gradient is induced by an IR laser.  The directed movement of mols. through the temp. gradient was detected and quantified using either covalently attached or intrinsic fluorophores.  By combining the precision of fluorescence detection with the variability and sensitivity of thermophoresis, MST provides a flexible, robust and fast way to dissect mol. interactions.  In this review, the authors present recent progress and developments in MST technol. and focus on MST applications beyond std. biomol. interaction studies.  By using different model systems, the authors introduce alternative MST applications - such as detn. of binding stoichiometries and binding modes, anal. of protein unfolding, thermodn. and enzyme kinetics.  In addn., we demonstrate the capability of MST to quantify high-affinity interactions with dissocn. consts. (Kd) in the low picomolar (pM) range as well as protein-protein interactions in pure mammalian cell lysates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUijovMuvKs7Vg90H21EOLACvtfcHk0lhY8fbDW4IsQQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXlt1Wisbo%253D&md5=8a9e1b3c2d10a85550e6d1041c33195e</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.molstruc.2014.03.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molstruc.2014.03.009%26sid%3Dliteratum%253Aachs%26aulast%3DJerabek-Willemsen%26aufirst%3DM.%26aulast%3DAndr%25C3%25A9%26aufirst%3DT.%26aulast%3DWanner%26aufirst%3DR.%26aulast%3DRoth%26aufirst%3DH.%2BM.%26aulast%3DDuhr%26aufirst%3DS.%26aulast%3DBaaske%26aufirst%3DP.%26aulast%3DBreitsprecher%26aufirst%3DD.%26atitle%3DMicroScale%2520thermophoresis%253A%2520interaction%2520analysis%2520and%2520beyond%26jtitle%3DJ.%2520Mol.%2520Struct.%26date%3D2014%26volume%3D1077%26spage%3D101%26epage%3D113%26doi%3D10.1016%2Fj.molstruc.2014.03.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rossi, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, C. W.</span></span> <span> </span><span class="NLM_article-title">Analysis of protein-ligand interactions by fluorescence polarization</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">365</span>â <span class="NLM_lpage">387</span>, <span class="refDoi">Â DOI: 10.1038/nprot.2011.305</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=10.1038%2Fnprot.2011.305" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=21372817" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=1%3ACAS%3A528%3ADC%252BC3MXislentbg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=365-387&author=A.+M.+Rossiauthor=C.+W.+Taylor&title=Analysis+of+protein-ligand+interactions+by+fluorescence+polarization&doi=10.1038%2Fnprot.2011.305"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Analysis of protein-ligand interactions by fluorescence polarization</span></div><div class="casAuthors">Rossi, Ana M.; Taylor, Colin W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">365-387</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Quantification of the assocns. between biomols. is required both to predict and understand the interactions that underpin all biol. activity.  Fluorescence polarization (FP) provides a nondisruptive means of measuring the assocn. of a fluorescent ligand with a larger mol.  We describe an FP assay in which binding of fluorescein-labeled inositol 1,4,5-trisphosphate (IP3) to N-terminal fragments of IP3 receptors can be characterized at different temps. and in competition with other ligands.  The assay allows the std. Gibbs free energy (ÎGÂ°), enthalpy (ÎHÂ°) and entropy (ÎSÂ°) changes of ligand binding to be detd.  The method is applicable to any purified ligand-binding site for which an appropriate fluorescent ligand is available.  FP can be used to measure low-affinity interactions in real time without the use of radioactive materials, it is nondestructive and, with appropriate care, it can resolve ÎHÂ° and ÎSÂ°.  The first part of the protocol, protein prepn., may take several weeks, whereas the FP measurements, once they have been optimized, would normally take 1-6 h.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPmq_JwlOSg7Vg90H21EOLACvtfcHk0lhY8fbDW4IsQQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXislentbg%253D&md5=b78e3bcd884a06cdeb8b090227963d33</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2011.305&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2011.305%26sid%3Dliteratum%253Aachs%26aulast%3DRossi%26aufirst%3DA.%2BM.%26aulast%3DTaylor%26aufirst%3DC.%2BW.%26atitle%3DAnalysis%2520of%2520protein-ligand%2520interactions%2520by%2520fluorescence%2520polarization%26jtitle%3DNat.%2520Protoc.%26date%3D2011%26volume%3D6%26spage%3D365%26epage%3D387%26doi%3D10.1038%2Fnprot.2011.305" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Otwinowski, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minor, W.</span></span> <span> </span><span class="NLM_article-title">Processing of X-ray diffraction data collected in oscillation mode</span>. <i>Methods Enzymol.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>276</i></span>,  <span class="NLM_fpage">307</span>â <span class="NLM_lpage">326</span>, <span class="refDoi">Â DOI: 10.1016/S0076-6879(97)76066-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=10.1016%2FS0076-6879%2897%2976066-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=27754618" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=1997&pages=307-326&author=Z.+Otwinowskiauthor=W.+Minor&title=Processing+of+X-ray+diffraction+data+collected+in+oscillation+mode&doi=10.1016%2FS0076-6879%2897%2976066-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Processing of x-ray diffraction data collected in oscillation mode</span></div><div class="casAuthors">Otwinowski, Zbyszek; Minor, Wladek</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">276</span>
        (<span class="NLM_cas:issue">Macromolecular Crystallography, Part A</span>),
    <span class="NLM_cas:pages">307-326</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Macromol. crystallog. is an iterative process.  Rarely do the first crystals provide all the necessary data to solve the biol. problem being studied.  Each step benefits from experience learned in previous steps.  To monitor the progress, the HKL package provides 2 tools: (1) statistics, both weighted (Ï2) and unweighted (R-merge), are provided, and the Bayesian reasoning and multicomponent error model facilitates obtaining the proper error ests. and (2) visualization of the process plays a double role by helping the operator to confirm that the process of data redn., including the resulting statistics, is correct, and allowing one to evaluate problems for which there are no good statistical criteria.  Visualization also provides confidence that the point of diminishing returns in data collection and redn. has been reached.  At that point, the effort should be directed to solving the structure.  The methods presented here have been applied to solve a large variety of problems, from inorg. mols. with 5 Ã unit cell to rotavirus of 700 Ã diam. crystd. in 700 Ã 1000 Ã 1400 Ã cell.  Overall quality of the method was tested by many researchers by successful application of the programs to MAD structure detns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrA3fvBOHqhhrVg90H21EOLACvtfcHk0lhY8fbDW4IsQQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D&md5=c9536971d4e32cc35352c40fb9368131</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2FS0076-6879%2897%2976066-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0076-6879%252897%252976066-X%26sid%3Dliteratum%253Aachs%26aulast%3DOtwinowski%26aufirst%3DZ.%26aulast%3DMinor%26aufirst%3DW.%26atitle%3DProcessing%2520of%2520X-ray%2520diffraction%2520data%2520collected%2520in%2520oscillation%2520mode%26jtitle%3DMethods%2520Enzymol.%26date%3D1997%26volume%3D276%26spage%3D307%26epage%3D326%26doi%3D10.1016%2FS0076-6879%2897%2976066-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">XDS</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">125</span>â <span class="NLM_lpage">132</span>, <span class="refDoi">Â DOI: 10.1107/S0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&author=W.+Kabsch&title=XDS&doi=10.1107%2FS0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0lhY8fbDW4IsQQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXDS%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2FS0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Argetsinger, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robertson, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koleva, R. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cline, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marto, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myers, M. G.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter-Su, C.</span></span> <span> </span><span class="NLM_article-title">Tyrosines 868, 966, and 972 in the kinase domain of JAK2 Are autophosphorylated and required for maximal JAK2 kinase activity</span>. <i>Mol. Endocrinol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">1062</span>â <span class="NLM_lpage">1076</span>, <span class="refDoi">Â DOI: 10.1210/me.2009-0355</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=10.1210%2Fme.2009-0355" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=20304997" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlslyjsr4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2010&pages=1062-1076&author=L.+S.+Argetsingerauthor=J.+A.+Stuckeyauthor=S.+A.+Robertsonauthor=R.+I.+Kolevaauthor=J.+M.+Clineauthor=J.+A.+Martoauthor=M.+G.+Myersauthor=C.+Carter-Su&title=Tyrosines+868%2C+966%2C+and+972+in+the+kinase+domain+of+JAK2+Are+autophosphorylated+and+required+for+maximal+JAK2+kinase+activity&doi=10.1210%2Fme.2009-0355"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Tyrosines 868, 966, and 972 in the kinase domain of JAK2 are autophosphorylated and required for maximal JAK2 kinase activity</span></div><div class="casAuthors">Argetsinger, Lawrence S.; Stuckey, Jeanne A.; Robertson, Scott A.; Koleva, Rositsa I.; Cline, Joel M.; Marto, Jarrod A.; Myers, Martin G., Jr.; Carter-Su, Christin</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Endocrinology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1062-1076</span>CODEN:
                <span class="NLM_cas:coden">MOENEN</span>;
        ISSN:<span class="NLM_cas:issn">0888-8809</span>.
    
            (<span class="NLM_cas:orgname">Endocrine Society</span>)
        </div><div class="casAbstract">Janus kinase 2 (JAK2) is activated by a majority of cytokine family receptors including receptors for GH, leptin, and erythropoietin.  To identify novel JAK2-regulatory and/or-binding sites, we set out to identify autophosphorylation sites in the kinase domain of JAK2.  Two-dimensional phosphopeptide mapping of in vitro autophosphorylated JAK2 identified tyrosines 868, 966, and 972 as sites of autophosphorylation.  Phosphorylated tyrosines 868 and 972 were also identified by mass spectrometry anal. of JAK2 activated by an erythropoietin-bound chimeric erythropoietin receptor/leptin receptor.  Phosphospecific antibodies suggest that the phosphorylation of all three tyrosines increases in response to GH.  Compared with wild-type JAK2, which is constitutively active when overexpressed, JAK2 lacking tyrosine 868, 966, or 972 has substantially reduced activity.  Coexpression with GH receptor and protein tyrosine phosphatase 1B allowed us to investigate GH-dependent activation of these mutated JAK2s in human embryonic kidney 293T cells.  All three mutated JAK2s are activated by GH, although to a lesser extent than wild-type JAK2.  The three mutated JAK2s also mediate GH activation of signal transducer and activator of transcription 3 (Stat3), signal transducer and activator of transcription 5b (Stat5b) and ERK1, but at reduced levels.  Coexpression with Src-homol. 2B1Î² (SH2B1Î²), like coexpression with GH-bound GH receptor, partially restores the activity of all three JAK2 mutants.  Based on these results and the crystal structure of the JAK2 kinase domain, we hypothesize that small changes in the conformation of the regions of JAK2 surrounding tyrosines 868, 966, and 972 due to e.g. phosphorylation, binding to a ligand-bound cytokine receptor, and/or binding to Src-homol. 2B1, may be essential for JAK2 to assume a maximally active conformation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNZL2jeP64kbVg90H21EOLACvtfcHk0lgxak7Xjt7wcQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlslyjsr4%253D&md5=76b806429b21f6c55893b7699d62b7bf</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1210%2Fme.2009-0355&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1210%252Fme.2009-0355%26sid%3Dliteratum%253Aachs%26aulast%3DArgetsinger%26aufirst%3DL.%2BS.%26aulast%3DStuckey%26aufirst%3DJ.%2BA.%26aulast%3DRobertson%26aufirst%3DS.%2BA.%26aulast%3DKoleva%26aufirst%3DR.%2BI.%26aulast%3DCline%26aufirst%3DJ.%2BM.%26aulast%3DMarto%26aufirst%3DJ.%2BA.%26aulast%3DMyers%26aufirst%3DM.%2BG.%26aulast%3DCarter-Su%26aufirst%3DC.%26atitle%3DTyrosines%2520868%252C%2520966%252C%2520and%2520972%2520in%2520the%2520kinase%2520domain%2520of%2520JAK2%2520Are%2520autophosphorylated%2520and%2520required%2520for%2520maximal%2520JAK2%2520kinase%2520activity%26jtitle%3DMol.%2520Endocrinol.%26date%3D2010%26volume%3D24%26spage%3D1062%26epage%3D1076%26doi%3D10.1210%2Fme.2009-0355" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Prchal-Murphy, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Semper, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lassnig, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wallner, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gausterer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teppner-Klymiuk, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobolak, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MÃ¼ller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolbe, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karaghiosoff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DinnyÃ©s, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">RÃ¼licke, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leitner, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strobl, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MÃ¼ller, M.</span></span> <span> </span><span class="NLM_article-title">TYK2 kinase activity is required for functional type I interferon responses in vivo</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">e39141</span>, <span class="refDoi">Â DOI: 10.1371/journal.pone.0039141</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=10.1371%2Fjournal.pone.0039141" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=22723949" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=1%3ACAS%3A528%3ADC%252BC38Xpt1Wntr8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=e39141&author=M.+Prchal-Murphyauthor=C.+Semperauthor=C.+Lassnigauthor=B.+Wallnerauthor=C.+Gaustererauthor=I.+Teppner-Klymiukauthor=J.+Kobolakauthor=S.+M%C3%BCllerauthor=T.+Kolbeauthor=M.+Karaghiosoffauthor=A.+Dinny%C3%A9sauthor=T.+R%C3%BClickeauthor=N.+R.+Leitnerauthor=B.+Stroblauthor=M.+M%C3%BCller&title=TYK2+kinase+activity+is+required+for+functional+type+I+interferon+responses+in+vivo&doi=10.1371%2Fjournal.pone.0039141"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">TYK2 kinase activity is required for functional type I interferon responses in vivo</span></div><div class="casAuthors">Prchal-Murphy, Michaela; Semper, Christian; Lassnig, Caroline; Wallner, Barbara; Gausterer, Christian; Teppner-Klymiuk, Ingeborg; Kobolak, Julianna; Mueller, Simone; Kolbe, Thomas; Karaghiosoff, Marina; Dinnyes, Andras; Ruelicke, Thomas; Leitner, Nicole R.; Strobl, Birgit; Mueller, Mathias</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">e39141</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Tyrosine kinase 2 (TYK2) is a member of the Janus kinase (JAK) family and is involved in cytokine signalling.  In vitro analyses suggest that TYK2 also has kinase-independent, i.e., non-canonical, functions.  We have generated gene-targeted mice harbouring a mutation in the ATP-binding pocket of the kinase domain.  The Tyk2 kinase-inactive (Tyk2K923E) mice are viable and show no gross abnormalities.  We show that kinase-active TYK2 is required for full-fledged type I interferon-(IFN) induced activation of the transcription factors STAT1-4 and for the in vivo antiviral defense against viruses primarily controlled through type I IFN actions.  In addn., TYK2 kinase activity was found to be required for the protein's stability.  An inhibitory function was only obsd. upon over-expression of TYK2K923E in vitro.  Tyk2K923E mice represent the first model for studying the kinase-independent function of a JAK in vivo and for assessing the consequences of side effects of JAK inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpenRZIOxu3PLVg90H21EOLACvtfcHk0lgxak7Xjt7wcQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xpt1Wntr8%253D&md5=4a833237168d26e520c7ff53b897276c</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0039141&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0039141%26sid%3Dliteratum%253Aachs%26aulast%3DPrchal-Murphy%26aufirst%3DM.%26aulast%3DSemper%26aufirst%3DC.%26aulast%3DLassnig%26aufirst%3DC.%26aulast%3DWallner%26aufirst%3DB.%26aulast%3DGausterer%26aufirst%3DC.%26aulast%3DTeppner-Klymiuk%26aufirst%3DI.%26aulast%3DKobolak%26aufirst%3DJ.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DKolbe%26aufirst%3DT.%26aulast%3DKaraghiosoff%26aufirst%3DM.%26aulast%3DDinny%25C3%25A9s%26aufirst%3DA.%26aulast%3DR%25C3%25BClicke%26aufirst%3DT.%26aulast%3DLeitner%26aufirst%3DN.%2BR.%26aulast%3DStrobl%26aufirst%3DB.%26aulast%3DM%25C3%25BCller%26aufirst%3DM.%26atitle%3DTYK2%2520kinase%2520activity%2520is%2520required%2520for%2520functional%2520type%2520I%2520interferon%2520responses%2520in%2520vivo%26jtitle%3DPLoS%2520One%26date%3D2012%26volume%3D7%26spage%3De39141%26doi%3D10.1371%2Fjournal.pone.0039141" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rasband, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eliceiri, K. W.</span></span> <span> </span><span class="NLM_article-title">NIH Image to ImageJ: 25 Years of image analysis</span>. <i>Nat. Methods</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">671</span>â <span class="NLM_lpage">675</span>, <span class="refDoi">Â DOI: 10.1038/nmeth.2089</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=10.1038%2Fnmeth.2089" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=22930834" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVKntb7P" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2012&pages=671-675&author=C.+A.+Schneiderauthor=W.+S.+Rasbandauthor=K.+W.+Eliceiri&title=NIH+Image+to+ImageJ%3A+25+Years+of+image+analysis&doi=10.1038%2Fnmeth.2089"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">NIH Image to ImageJ: 25 years of image analysis</span></div><div class="casAuthors">Schneider, Caroline A.; Rasband, Wayne S.; Eliceiri, Kevin W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Methods</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">7_part1</span>),
    <span class="NLM_cas:pages">671-675</span>CODEN:
                <span class="NLM_cas:coden">NMAEA3</span>;
        ISSN:<span class="NLM_cas:issn">1548-7091</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">For the past 25 years NIH Image and ImageJ software have been pioneers as open tools for the anal. of scientific images.  We discuss the origins, challenges and solns. of these two programs, and how their history can serve to advise and inform other software projects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGofzHXrdLwDn7Vg90H21EOLACvtfcHk0lgxak7Xjt7wcQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVKntb7P&md5=85ab928cd79f1e2f2351c834c0c600f0</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1038%2Fnmeth.2089&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnmeth.2089%26sid%3Dliteratum%253Aachs%26aulast%3DSchneider%26aufirst%3DC.%2BA.%26aulast%3DRasband%26aufirst%3DW.%2BS.%26aulast%3DEliceiri%26aufirst%3DK.%2BW.%26atitle%3DNIH%2520Image%2520to%2520ImageJ%253A%252025%2520Years%2520of%2520image%2520analysis%26jtitle%3DNat.%2520Methods%26date%3D2012%26volume%3D9%26spage%3D671%26epage%3D675%26doi%3D10.1038%2Fnmeth.2089" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zidar, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macut, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">TomaÅ¡iÄ, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brvar, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MontalvÃ£o, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tammela, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solmajer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterlin
MaÅ¡iÄ, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">IlaÅ¡, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kikelj, D.</span></span> <span> </span><span class="NLM_article-title"><i>N</i>-phenyl-4,5-dibromopyrrolamides and <i>N</i>-phenylindolamides as ATP competitive DNA gyrase B inhibitors: design, synthesis, and evaluation</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">6179</span>â <span class="NLM_lpage">6194</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.5b00775</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00775" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVyitLfL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=6179-6194&author=N.+Zidarauthor=H.+Macutauthor=T.+Toma%C5%A1i%C4%8Dauthor=M.+Brvarauthor=S.+Montalv%C3%A3oauthor=P.+Tammelaauthor=T.+Solmajerauthor=L.+Peterlin%0AMa%C5%A1i%C4%8Dauthor=J.+Ila%C5%A1author=D.+Kikelj&title=N-phenyl-4%2C5-dibromopyrrolamides+and+N-phenylindolamides+as+ATP+competitive+DNA+gyrase+B+inhibitors%3A+design%2C+synthesis%2C+and+evaluation&doi=10.1021%2Facs.jmedchem.5b00775"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">N-Phenyl-4,5-dibromopyrrolamides and N-Phenylindolamides as ATP Competitive DNA Gyrase B Inhibitors: Design, Synthesis, and Evaluation</span></div><div class="casAuthors">Zidar, Nace; Macut, Helena; Tomasic, Tihomir; Brvar, Matjaz; Montalvao, Sofia; Tammela, Paivi; Solmajer, Tom; Peterlin Masic, Lucija; Ilas, Janez; Kikelj, Danijel</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6179-6194</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bacterial DNA gyrase is a well-known and validated target in the design of antibacterial drugs.  However, inhibitors of its ATP binding subunit, DNA gyrase B (GyrB), have so far not reached clin. use.  In the present study, three different series of N-phenyl-4,5-dibromopyrrolamides and N-phenylindolamides were designed and prepd. as potential DNA gyrase B inhibitors.  The IC50 values of compds. on DNA gyrase from Escherichia coli were in the low micromolar range, with the best compd., (4-(4,5-dibromo-1H-pyrrole-2-carboxamido)benzoyl)glycine, displaying an IC50 of 450 nM.  For this compd., a high-resoln. crystal structure in complex with E. coli DNA gyrase B was obtained, revealing details of its binding mode within the active site.  The binding affinities of three compds. with GyrB were addnl. evaluated by surface plasmon resonance, and the results were in good agreement with the detd. enzymic activities.  For the most promising compds., the inhibitory activities against DNA gyrase from Staphylococcus aureus and topoisomerases IV from E. coli and S. aureus were detd.  Antibacterial activities of the most potent compds. of each series were evaluated against two Gram-pos. and two Gram-neg. bacterial strains.  The results obtained in this study provide valuable information on the binding mode and structure-activity relationship of N-phenyl-4,5-dibromopyrrolamides and N-phenylindolamides as promising classes of ATP competitive GyrB inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp53wGgDnJdKbVg90H21EOLACvtfcHk0lia0vDJg-zyJw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVyitLfL&md5=071aa879adb5bdb2f379a10a7b6aac8a</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00775&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00775%26sid%3Dliteratum%253Aachs%26aulast%3DZidar%26aufirst%3DN.%26aulast%3DMacut%26aufirst%3DH.%26aulast%3DToma%25C5%25A1i%25C4%258D%26aufirst%3DT.%26aulast%3DBrvar%26aufirst%3DM.%26aulast%3DMontalv%25C3%25A3o%26aufirst%3DS.%26aulast%3DTammela%26aufirst%3DP.%26aulast%3DSolmajer%26aufirst%3DT.%26aulast%3DPeterlin%2BMa%25C5%25A1i%25C4%258D%26aufirst%3DL.%26aulast%3DIla%25C5%25A1%26aufirst%3DJ.%26aulast%3DKikelj%26aufirst%3DD.%26atitle%3DN-phenyl-4%252C5-dibromopyrrolamides%2520and%2520N-phenylindolamides%2520as%2520ATP%2520competitive%2520DNA%2520gyrase%2520B%2520inhibitors%253A%2520design%252C%2520synthesis%252C%2520and%2520evaluation%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D6179%26epage%3D6194%26doi%3D10.1021%2Facs.jmedchem.5b00775" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Argade, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhamidipati, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carroll, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clough, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keim, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sylvain, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossi, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coquilla, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Issakani, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masuda, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Payan, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, R.</span></span> <span> </span><span class="NLM_article-title">Application of cultured human mast cells (CHMC) for the design and structure-activity relationship of IgE-mediated mast cell activation inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">2117</span>â <span class="NLM_lpage">2121</span>, <span class="refDoi">Â DOI: 10.1016/j.bmcl.2015.03.075</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=10.1016%2Fj.bmcl.2015.03.075" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=25872982" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlvVems7s%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2015&pages=2117-2121&author=A.+Argadeauthor=S.+Bhamidipatiauthor=H.+Liauthor=D.+Carrollauthor=J.+Cloughauthor=H.+Keimauthor=C.+Sylvainauthor=A.+B.+Rossiauthor=C.+Coquillaauthor=S.+D.+Issakaniauthor=E.+S.+Masudaauthor=D.+G.+Payanauthor=R.+Singh&title=Application+of+cultured+human+mast+cells+%28CHMC%29+for+the+design+and+structure-activity+relationship+of+IgE-mediated+mast+cell+activation+inhibitors&doi=10.1016%2Fj.bmcl.2015.03.075"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Application of cultured human mast cells (CHMC) for the design and structure-activity relationship of IgE-mediated mast cell activation inhibitors</span></div><div class="casAuthors">Argade, Ankush; Bhamidipati, Somasekhar; Li, Hui; Carroll, David; Clough, Jeffrey; Keim, Holger; Sylvain, Catherine; Rossi, Alexander B.; Coquilla, Christina; Issakani, Sarkiz D.; Masuda, Esteban S.; Payan, Donald G.; Singh, Rajinder</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2117-2121</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Here we report the optimization of small mol. inhibitors of human mast cell degranulation via anti-IgE-mediated tryptase release following crosslinking and activation of IgE-loaded FcÎµR1 receptors.  The compds. are selective upstream inhibitors of FcÎµR1-dependent human mast cell degranulation and proved to be devoid of activity in downstream ionomycin mediated degranulation.  Structure-activity relationship (SAR) leading to compd. 26 is outlined.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6TS9r3NDMIbVg90H21EOLACvtfcHk0lia0vDJg-zyJw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlvVems7s%253D&md5=933ebe95e6ef28d7f2bff242b9067783</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2015.03.075&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2015.03.075%26sid%3Dliteratum%253Aachs%26aulast%3DArgade%26aufirst%3DA.%26aulast%3DBhamidipati%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DCarroll%26aufirst%3DD.%26aulast%3DClough%26aufirst%3DJ.%26aulast%3DKeim%26aufirst%3DH.%26aulast%3DSylvain%26aufirst%3DC.%26aulast%3DRossi%26aufirst%3DA.%2BB.%26aulast%3DCoquilla%26aufirst%3DC.%26aulast%3DIssakani%26aufirst%3DS.%2BD.%26aulast%3DMasuda%26aufirst%3DE.%2BS.%26aulast%3DPayan%26aufirst%3DD.%2BG.%26aulast%3DSingh%26aufirst%3DR.%26atitle%3DApplication%2520of%2520cultured%2520human%2520mast%2520cells%2520%2528CHMC%2529%2520for%2520the%2520design%2520and%2520structure-activity%2520relationship%2520of%2520IgE-mediated%2520mast%2520cell%2520activation%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D25%26spage%3D2117%26epage%3D2121%26doi%3D10.1016%2Fj.bmcl.2015.03.075" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabro, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lachance, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcoux-Archambault, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perrier, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DorÃ©, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gros, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masson, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gregoire, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ausseil, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheishvili, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laulan, N. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">St-Pierre, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szyf, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arimondo, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gagnon, A.</span></span> <span> </span><span class="NLM_article-title">Preparation of phenylethylbenzamide derivatives as modulators of DNMT3 activity</span>. <i>Medchemcomm</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">1562</span>â <span class="NLM_lpage">1570</span>, <span class="refDoi">Â DOI: 10.1039/c3md00214d</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=10.1039%2Fc3md00214d" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVWgt7%252FI" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=1562-1570&author=A.+Kabroauthor=H.+Lachanceauthor=I.+Marcoux-Archambaultauthor=V.+Perrierauthor=V.+Dor%C3%A9author=C.+Grosauthor=V.+Massonauthor=J.+M.+Gregoireauthor=F.+Ausseilauthor=D.+Cheishviliauthor=N.+B.+Laulanauthor=Y.+St-Pierreauthor=M.+Szyfauthor=P.+B.+Arimondoauthor=A.+Gagnon&title=Preparation+of+phenylethylbenzamide+derivatives+as+modulators+of+DNMT3+activity&doi=10.1039%2Fc3md00214d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Preparation of phenylethylbenzamide derivatives as modulators of DNMT3 activity</span></div><div class="casAuthors">Kabro, Anzhelika; Lachance, Hugo; Marcoux-Archambault, Iris; Perrier, Valerie; Dore, Vicky; Gros, Christina; Masson, Veronique; Gregoire, Jean-Marc; Ausseil, Frederic; Cheishvili, David; Laulan, Nathalie Bibens; St-Pierre, Yves; Szyf, Moshe; Arimondo, Paola B.; Gagnon, Alexandre</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1562-1570</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">DNA-methyltransferases (DNMTs) are a class of epigenetic enzymes that catalyze the transfer of a Me moiety from the Me donor S-adenosyl-l-methionine onto the C5 position of cytosine in DNA.  This process is dysregulated in cancers and leads to the hypermethylation and silencing of tumor suppressor genes.  The development of potent and selective inhibitors of DNMTs is of utmost importance for the discovery of new therapies for the treatment of cancer.  We report herein the synthesis and DNMT inhibitory activity of 29 analogs derived from NSC 319745.  The effect of selected compds. on the methylation level in the MDA-MB-231 human breast cancer cell line was evaluated using a luminometric methylation assay.  Mol. docking studies have been conducted to propose a binding mode for this series.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHH5XzyIf6O7Vg90H21EOLACvtfcHk0lia0vDJg-zyJw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvVWgt7%252FI&md5=4cb82583964a6293e4804b0ec9333009</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1039%2Fc3md00214d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc3md00214d%26sid%3Dliteratum%253Aachs%26aulast%3DKabro%26aufirst%3DA.%26aulast%3DLachance%26aufirst%3DH.%26aulast%3DMarcoux-Archambault%26aufirst%3DI.%26aulast%3DPerrier%26aufirst%3DV.%26aulast%3DDor%25C3%25A9%26aufirst%3DV.%26aulast%3DGros%26aufirst%3DC.%26aulast%3DMasson%26aufirst%3DV.%26aulast%3DGregoire%26aufirst%3DJ.%2BM.%26aulast%3DAusseil%26aufirst%3DF.%26aulast%3DCheishvili%26aufirst%3DD.%26aulast%3DLaulan%26aufirst%3DN.%2BB.%26aulast%3DSt-Pierre%26aufirst%3DY.%26aulast%3DSzyf%26aufirst%3DM.%26aulast%3DArimondo%26aufirst%3DP.%2BB.%26aulast%3DGagnon%26aufirst%3DA.%26atitle%3DPreparation%2520of%2520phenylethylbenzamide%2520derivatives%2520as%2520modulators%2520of%2520DNMT3%2520activity%26jtitle%3DMedchemcomm%26date%3D2013%26volume%3D4%26spage%3D1562%26epage%3D1570%26doi%3D10.1039%2Fc3md00214d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, J.</span></span> <span> </span><span class="NLM_article-title">I<sub>2</sub> -mediated oxidative C-O bond formation for the synthesis of 1,3,4-oxadiazoles from aldehydes and hydrazides</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>78</i></span>,  <span class="NLM_fpage">10337</span>â <span class="NLM_lpage">10343</span>, <span class="refDoi">Â DOI: 10.1021/jo401751h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo401751h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVyrtLbM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=2013&pages=10337-10343&author=W.+Yuauthor=G.+Huangauthor=Y.+Zhangauthor=H.+Liuauthor=L.+Dongauthor=X.+Yuauthor=Y.+Liauthor=J.+Chang&title=I2+-mediated+oxidative+C-O+bond+formation+for+the+synthesis+of+1%2C3%2C4-oxadiazoles+from+aldehydes+and+hydrazides&doi=10.1021%2Fjo401751h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">I2-Mediated Oxidative C-O Bond Formation for the Synthesis of 1,3,4-Oxadiazoles from Aldehydes and Hydrazides</span></div><div class="casAuthors">Yu, Wenquan; Huang, Gang; Zhang, Yueteng; Liu, Hongxu; Dong, Lihong; Yu, Xuejun; Li, Yujiang; Chang, Junbiao</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">78</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">10337-10343</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A practical and transition-metal-free oxidative cyclization of acylhydrazones into 1,3,4-oxadiazoles has been developed by employing stoichiometric mol. iodine in the presence of potassium carbonate.  The conditions of this cyclization reaction also work well with crude acylhydrazone substrates obtained from the condensation of aldehydes and hydrazides.  A series of sym. and asym. 2,5-disubstituted (aryl, alkyl, and/or vinyl) 1,3,4-oxadiazoles can be conveniently generated in an efficient and scalable fashion.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpntjjbgNlRr7Vg90H21EOLACvtfcHk0liIx5LL10pZ1A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVyrtLbM&md5=903d0cdd2372a21ef27b388ef45ff6ac</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fjo401751h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo401751h%26sid%3Dliteratum%253Aachs%26aulast%3DYu%26aufirst%3DW.%26aulast%3DHuang%26aufirst%3DG.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DDong%26aufirst%3DL.%26aulast%3DYu%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DChang%26aufirst%3DJ.%26atitle%3DI2%2520-mediated%2520oxidative%2520C-O%2520bond%2520formation%2520for%2520the%2520synthesis%2520of%25201%252C3%252C4-oxadiazoles%2520from%2520aldehydes%2520and%2520hydrazides%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2013%26volume%3D78%26spage%3D10337%26epage%3D10343%26doi%3D10.1021%2Fjo401751h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Keith, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Apodaca, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tichenor, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pierce, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seierstad, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palmer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Webb, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karbarz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wennerholm, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rizzolio, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rynberg, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaplan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breitenbucher, J. G.</span></span> <span> </span><span class="NLM_article-title">Aryl piperazinyl ureas as inhibitors of fatty acid amide hydrolase (FAAH) in rat, dog, and primate</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">823</span>â <span class="NLM_lpage">827</span>, <span class="refDoi">Â DOI: 10.1021/ml300186g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml300186g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1eksbrN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2012&pages=823-827&author=J.+M.+Keithauthor=R.+Apodacaauthor=M.+Tichenorauthor=W.+Xiaoauthor=W.+Jonesauthor=J.+Pierceauthor=M.+Seierstadauthor=J.+Palmerauthor=M.+Webbauthor=M.+Karbarzauthor=B.+Scottauthor=S.+Wilsonauthor=L.+Luoauthor=M.+Wennerholmauthor=L.+Changauthor=S.+Brownauthor=M.+Rizzolioauthor=R.+Rynbergauthor=S.+Chaplanauthor=J.+G.+Breitenbucher&title=Aryl+piperazinyl+ureas+as+inhibitors+of+fatty+acid+amide+hydrolase+%28FAAH%29+in+rat%2C+dog%2C+and+primate&doi=10.1021%2Fml300186g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Aryl Piperazinyl Ureas as Inhibitors of Fatty Acid Amide Hydrolase (FAAH) in Rat, Dog, and Primate</span></div><div class="casAuthors">Keith, John M.; Apodaca, Rich; Tichenor, Mark; Xiao, Wei; Jones, William; Pierce, Joan; Seierstad, Mark; Palmer, James; Webb, Michael; Karbarz, Mark; Scott, Brian; Wilson, Sandy; Luo, Lin; Wennerholm, Michelle; Chang, Leon; Brown, Sean; Rizzolio, Michele; Rynberg, Raymond; Chaplan, Sandra; Breitenbucher, J. Guy</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">823-827</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of aryl piperazinyl ureas that act as covalent inhibitors of fatty acid amide hydrolase (FAAH) is described.  A potent and selective (does not inhibit FAAH-2) member of this class, JNJ-40355003 (I), was found to elevate the plasma levels of three fatty acid amides: anandamide, oleoyl ethanolamide, and palmitoyl ethanolamide, in the rat, dog, and cynomolgous monkey.  The elevation of the levels of these lipids in the plasma of monkeys suggests that FAAH-2 may not play a significant role in regulating plasma levels of fatty acid ethanolamides in primates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0G60tuqmh8LVg90H21EOLACvtfcHk0liIx5LL10pZ1A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1eksbrN&md5=fbf5fb59244ac006d87a9b80918ae24e</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fml300186g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml300186g%26sid%3Dliteratum%253Aachs%26aulast%3DKeith%26aufirst%3DJ.%2BM.%26aulast%3DApodaca%26aufirst%3DR.%26aulast%3DTichenor%26aufirst%3DM.%26aulast%3DXiao%26aufirst%3DW.%26aulast%3DJones%26aufirst%3DW.%26aulast%3DPierce%26aufirst%3DJ.%26aulast%3DSeierstad%26aufirst%3DM.%26aulast%3DPalmer%26aufirst%3DJ.%26aulast%3DWebb%26aufirst%3DM.%26aulast%3DKarbarz%26aufirst%3DM.%26aulast%3DScott%26aufirst%3DB.%26aulast%3DWilson%26aufirst%3DS.%26aulast%3DLuo%26aufirst%3DL.%26aulast%3DWennerholm%26aufirst%3DM.%26aulast%3DChang%26aufirst%3DL.%26aulast%3DBrown%26aufirst%3DS.%26aulast%3DRizzolio%26aufirst%3DM.%26aulast%3DRynberg%26aufirst%3DR.%26aulast%3DChaplan%26aufirst%3DS.%26aulast%3DBreitenbucher%26aufirst%3DJ.%2BG.%26atitle%3DAryl%2520piperazinyl%2520ureas%2520as%2520inhibitors%2520of%2520fatty%2520acid%2520amide%2520hydrolase%2520%2528FAAH%2529%2520in%2520rat%252C%2520dog%252C%2520and%2520primate%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D3%26spage%3D823%26epage%3D827%26doi%3D10.1021%2Fml300186g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 6 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Alexander D. Wade, <span class="NLM_string-name hlFld-ContribAuthor">David J. Huggins</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of Optimal Ligand Growth Vectors Using an Alchemical Free-Energy Method. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (11)
                                     , 5580-5594. <a href="https://doi.org/10.1021/acs.jcim.0c00610" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c00610</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c00610&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c00610%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DIdentification%252Bof%252BOptimal%252BLigand%252BGrowth%252BVectors%252BUsing%252Ban%252BAlchemical%252BFree-Energy%252BMethod%26aulast%3DWade%26aufirst%3DAlexander%2BD.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D01062020%26date%3D01092020%26date%3D18082020%26volume%3D60%26issue%3D11%26spage%3D5580%26epage%3D5594" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Aysegul Turupcu, Julian Tirado-Rives, <span class="NLM_string-name hlFld-ContribAuthor">William L. Jorgensen</span>. </span><span class="cited-content_cbyCitation_article-title">Explicit Representation of CationâÏ Interactions in Force Fields with 1/r4 Nonbonded Terms. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Theory and Computation</span><span> <strong>2020,</strong> <em>16 </em>
                                    (11)
                                     , 7184-7194. <a href="https://doi.org/10.1021/acs.jctc.0c00847" title="DOI URL">https://doi.org/10.1021/acs.jctc.0c00847</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jctc.0c00847&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jctc.0c00847%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Theory%2520and%2520Computation%26atitle%3DExplicit%252BRepresentation%252Bof%252BCation%2525E2%252588%252592%2525CF%252580%252BInteractions%252Bin%252BForce%252BFields%252Bwith%252B1%25252Fr4%252BNonbonded%252BTerms%26aulast%3DTurupcu%26aufirst%3DAysegul%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D12082020%26date%3D13102020%26volume%3D16%26issue%3D11%26spage%3D7184%26epage%3D7194" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Kara J. Cutrona, Ana S. Newton, Stefan G. Krimmer, Julian Tirado-Rives, <span class="NLM_string-name hlFld-ContribAuthor">William L. Jorgensen</span>. </span><span class="cited-content_cbyCitation_article-title">Metadynamics as a Postprocessing Method for Virtual Screening with Application to the Pseudokinase Domain of JAK2. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (9)
                                     , 4403-4415. <a href="https://doi.org/10.1021/acs.jcim.0c00276" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c00276</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c00276&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c00276%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DMetadynamics%252Bas%252Ba%252BPostprocessing%252BMethod%252Bfor%252BVirtual%252BScreening%252Bwith%252BApplication%252Bto%252Bthe%252BPseudokinase%252BDomain%252Bof%252BJAK2%26aulast%3DCutrona%26aufirst%3DKara%2BJ.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D18032020%26date%3D27052020%26date%3D08052020%26volume%3D60%26issue%3D9%26spage%3D4403%26epage%3D4415" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Juuli  Raivola</span>, <span class="hlFld-ContribAuthor ">Teemu  Haikarainen</span>, <span class="hlFld-ContribAuthor ">Bobin George  Abraham</span>, <span class="hlFld-ContribAuthor ">Olli  Silvennoinen</span>. </span><span class="cited-content_cbyCitation_article-title">Janus Kinases in Leukemia. </span><span class="cited-content_cbyCitation_journal-name">Cancers</span><span> <strong>2021,</strong> <em>13 </em>
                                    (4)
                                     , 800. <a href="https://doi.org/10.3390/cancers13040800" title="DOI URL">https://doi.org/10.3390/cancers13040800</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/cancers13040800&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fcancers13040800%26sid%3Dliteratum%253Aachs%26jtitle%3DCancers%26atitle%3DJanus%252BKinases%252Bin%252BLeukemia%26aulast%3DRaivola%26aufirst%3DJuuli%26date%3D2021%26date%3D2021%26volume%3D13%26issue%3D4%26spage%3D800" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Nayana  Devang</span>, <span class="hlFld-ContribAuthor ">Adyashree  Pani</span>, <span class="hlFld-ContribAuthor ">G.K.  Rajanikant</span>. </span><span class="cited-content_cbyCitation_article-title">Pseudokinases: Prospects for expanding the therapeutic targets armamentarium. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 121-185. <a href="https://doi.org/10.1016/bs.apcsb.2020.09.004" title="DOI URL">https://doi.org/10.1016/bs.apcsb.2020.09.004</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/bs.apcsb.2020.09.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fbs.apcsb.2020.09.004%26sid%3Dliteratum%253Aachs%26atitle%3DPseudokinases%25253A%252BProspects%252Bfor%252Bexpanding%252Bthe%252Btherapeutic%252Btargets%252Barmamentarium%26aulast%3DDevang%26aufirst%3DNayana%26date%3D2021%26spage%3D121%26epage%3D185%26pub%3DElsevier%26atitle%3DProtein%252BKinases%252Bin%252BDrug%252BDiscovery%26date%3D2021%26volume%3D124" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Peter D.  Mace</span>, <span class="hlFld-ContribAuthor ">James M.  Murphy</span>. </span><span class="cited-content_cbyCitation_article-title">Thereâs more to death than life: Noncatalytic functions in kinase and pseudokinase signaling. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biological Chemistry</span><span> <strong>2021,</strong> <em>296 </em>, 100705. <a href="https://doi.org/10.1016/j.jbc.2021.100705" title="DOI URL">https://doi.org/10.1016/j.jbc.2021.100705</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jbc.2021.100705&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jbc.2021.100705%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biological%2520Chemistry%26atitle%3DThere%2525E2%252580%252599s%252Bmore%252Bto%252Bdeath%252Bthan%252Blife%25253A%252BNoncatalytic%252Bfunctions%252Bin%252Bkinase%252Band%252Bpseudokinase%252Bsignaling%26aulast%3DMace%26aufirst%3DPeter%2BD.%26date%3D2021%26volume%3D296%26spage%3D100705" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/medium/jm0c00192_0020.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0020.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00192&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/medium/jm0c00192_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Compound <b>1</b> is the known pan-CDK and pan-JAK kinase inhibitor JNJ7706621. Compounds <b>2</b>â<b>15</b> are JAK2 JH2-binding molecules reported in this work.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00192&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/medium/jm0c00192_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (left) <b>1</b> bound to JAK2 JH2 (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5USZ">5USZ</a>) and (right) <b>1</b> bound to JAK2 JH1 (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5USY">5USY</a>). Hydrogen bonds with <i>r</i><sub>ON</sub> < 3.6 Ã are indicated with dashed lines. Selective binding to JAK2 JH2 is sought by growing the ligands toward the vacant region near T555 and R715.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00192&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/medium/jm0c00192_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Rendering from the crystal structure for ATP bound to JAK2 JH2 (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4FVQ">4FVQ</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00192&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/medium/jm0c00192_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0010.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Compound <b>12</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00192&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) Boc<sub>2</sub>O, Et<sub>3</sub>N, <i>i</i>PrOH, r.t., and 14 h; (b) 10 mol % Pd(OAc)<sub>2</sub>, 20 mol % JohnPhos, Cs<sub>2</sub>CO<sub>3</sub>, dioxane, 110 Â°C, and 19 h; (c) TFA, DCM, 0 Â°C to r.t., and 30 min; (d) NaHCO<sub>3</sub>, H<sub>2</sub>O/THF, 0 Â°C, and 1 h; (e) Et<sub>3</sub>N, dioxane, 110 Â°C, and 50 min; (f) DBN, LiBr, MeCN, 2 vol % H<sub>2</sub>O, r.t., and 68 h.</p></p></figure><figure data-id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/medium/jm0c00192_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Rendering from the 1.90 Ã crystal structure for <b>4</b> bound to JAK2 JH2 (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6OBB">6OBB</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00192&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch2" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/medium/jm0c00192_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0012.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Computed Conformational Preferences (kcal/mol) for Acetanilides</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00192&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/medium/jm0c00192_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Modeled structure illustrating the potential hydrogen bonding between Lys581 and the amide carbonyl oxygen atom and the pyridinyl nitrogen atom for 2-pyridinyl analogs of <b>3</b> and <b>4</b> such as <b>6</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00192&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/medium/jm0c00192_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Rendering from the 1.71 Ã crystal structure for <b>10</b> bound to JAK2 JH2 (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6OBF">6OBF</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00192&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/medium/jm0c00192_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Illustration of the hydrogen bonding in the vicinity of the carboxylate group of <b>10</b> from the 1.71 Ã crystal structure with JAK2 JH2 (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6OBF">6OBF</a>). Red spheres represent oxygen atoms of localized water molecules.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00192&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/medium/jm0c00192_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0008.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Rendering from the 2.03 Ã crystal structure for <b>12</b> bound to JAK2 JH2 (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6OCC">6OCC</a>). Red spheres represent oxygen atoms of localized water molecules.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00192&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/medium/jm0c00192_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0009.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. In vitro [Î³-<sup>32</sup>P]ATP kinase activity assay of immunoprecipitated full-length JAK2. Autoradiography of autophosphorylated JAK2 wild-type and V617F mutant exposed to different concentrations of (A) <b>1</b> and (B) <b>10</b>. Shown is representative data from experiments performed in triplicate.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00192&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch3" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/medium/jm0c00192_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0013.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Syntheses of <b>17a</b>,<b>b</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00192&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) THF, reflux, and 24 h; (b) NaH, THF, reflux, and 24 h; (c) hydrazine, reflux, and 8â19 h.</p></p></figure><figure data-id="sch4" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/medium/jm0c00192_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0014.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Syntheses of <b>18a</b>,<b>b</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00192&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) K<sub>2</sub>CO<sub>3</sub>, MeCN, r.t., and 16.5 h; (b) K<sub>2</sub>CO<sub>3</sub>, acetone, reflux, and 5 h; (c) H<sub>2</sub>, Pd/C, THF/MeOH, and 20 h.</p></p></figure><figure data-id="sch5" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/medium/jm0c00192_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0015.jpeg" id="rightTab-gr14" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of <b>18c</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00192&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) Boc<sub>2</sub>O, Et<sub>3</sub>N, <i>i</i>PrOH, r.t., and 14 h; (b) 10 mol % Pd(OAc)<sub>2</sub>, 20 mol % JohnPhos, Cs<sub>2</sub>CO<sub>3</sub>, dioxane, 110 Â°C, and 19 h; (c) TFA, DCM, 0 Â°C to r.t., and 30 min.</p></p></figure><figure data-id="sch6" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/medium/jm0c00192_0016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0016.jpeg" id="rightTab-gr15" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 6. Synthesis of 1<b>8d</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00192&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) EtOH, reflux, and 10 min; (b) K<sub>2</sub>CO<sub>3</sub>, I<sub>2</sub>, DMSO, 100 Â°C, and 40 min.</p></p></figure><figure data-id="sch7" data-index="16" class="article__inlineFigure"><h2 class="fig-label">Scheme 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/medium/jm0c00192_0017.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0017.jpeg" id="rightTab-gr16" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 7. Syntheses of Phenyl Carbamates <b>19a</b>, <b>c</b>â<b>j</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00192&amp;id=sch7"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) NaHCO<sub>3</sub>, H<sub>2</sub>O/THF, 0 Â°C, and 40 min to 1 h; (b) NaHCO<sub>3</sub>, H<sub>2</sub>O/THF, 0 Â°C to r.t., and 45 h; (c) pyridine, MeCN, r.t., and 2.5 h.</p></p></figure><figure data-id="sch8" data-index="17" class="article__inlineFigure"><h2 class="fig-label">Scheme 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/medium/jm0c00192_0018.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0018.jpeg" id="rightTab-gr17" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 8. Synthesis of <b>2</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00192&amp;id=sch8"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) Cs<sub>2</sub>CO<sub>3</sub>, DMF, r.t., and 16 h.</p></p></figure><figure data-id="sch9" data-index="18" class="article__inlineFigure"><h2 class="fig-label">Scheme 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/medium/jm0c00192_0019.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0019.jpeg" id="rightTab-gr18" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 9. Syntheses of Compounds <b>3</b>, <b>4</b>, and <b>6</b>â<b>15</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0019.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00192&amp;id=sch9"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) Et<sub>3</sub>N, dioxane, 90â110 Â°C, and 35 min to 2.5 h; (b) base, LiBr, MeCN, 2 vol % H<sub>2</sub>O, r.t., and 11â288 h.</p></p></figure><figure data-id="sch10" data-index="19" class="article__inlineFigure"><h2 class="fig-label">Scheme 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/medium/jm0c00192_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0011.jpeg" id="rightTab-gr19" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 10. Synthesis of Compound <b>5</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00192/20200520/images/large/jm0c00192_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00192&amp;id=sch10"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) NaHCO<sub>3</sub>, H<sub>2</sub>O/THF, 0 Â°C, and 1 h; (b) NaH, THF/dioxane, reflux, and 20 h.</p></p></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i88">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_53304" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_53304" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 37 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">OâShea, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holland, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Staudt, L. M.</span></span> <span> </span><span class="NLM_article-title">JAKs and STATs in immunity, immunodeficiency, and cancer</span>. <i>N. Engl. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>368</i></span>,  <span class="NLM_fpage">161</span>â <span class="NLM_lpage">170</span>, <span class="refDoi">Â DOI: 10.1056/NEJMra1202117</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=10.1056%2FNEJMra1202117" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=23301733" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=1%3ACAS%3A528%3ADC%252BC3sXovVyhuw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=368&publication_year=2013&pages=161-170&author=J.+J.+O%E2%80%99Sheaauthor=S.+M.+Hollandauthor=L.+M.+Staudt&title=JAKs+and+STATs+in+immunity%2C+immunodeficiency%2C+and+cancer&doi=10.1056%2FNEJMra1202117"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">JAKs and STATs in immunity, immunodeficiency, and cancer</span></div><div class="casAuthors">O'Shea, John J.; Holland, Steven M.; Staudt, Louis M.</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">368</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">161-170</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">A review highlights interferon-mediated gene regulation has resulted in a new framework for the study of cell signaling and JAKs STATs in immunity, immunodeficiency, and cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqk1n-UgWrlWrVg90H21EOLACvtfcHk0liDxSSU9FVDkg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXovVyhuw%253D%253D&md5=5d87b67f72b721eb0ca9413adf92a7b9</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1056%2FNEJMra1202117&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMra1202117%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Shea%26aufirst%3DJ.%2BJ.%26aulast%3DHolland%26aufirst%3DS.%2BM.%26aulast%3DStaudt%26aufirst%3DL.%2BM.%26atitle%3DJAKs%2520and%2520STATs%2520in%2520immunity%252C%2520immunodeficiency%252C%2520and%2520cancer%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2013%26volume%3D368%26spage%3D161%26epage%3D170%26doi%3D10.1056%2FNEJMra1202117" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Leroy, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Constantinescu, S. N.</span></span> <span> </span><span class="NLM_article-title">Rethinking JAK2 inhibition: towards novel strategies of more specific and versatile Janus kinase inhibition</span>. <i>Leukemia</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">1023</span>â <span class="NLM_lpage">1038</span>, <span class="refDoi">Â DOI: 10.1038/leu.2017.43</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=10.1038%2Fleu.2017.43" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=28119526" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=1%3ACAS%3A528%3ADC%252BC2sXislyms7w%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2017&pages=1023-1038&author=E.+Leroyauthor=S.+N.+Constantinescu&title=Rethinking+JAK2+inhibition%3A+towards+novel+strategies+of+more+specific+and+versatile+Janus+kinase+inhibition&doi=10.1038%2Fleu.2017.43"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Rethinking JAK2 inhibition: towards novel strategies of more specific and versatile janus kinase inhibition</span></div><div class="casAuthors">Leroy, E.; Constantinescu, S. N.</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1023-1038</span>CODEN:
                <span class="NLM_cas:coden">LEUKED</span>;
        ISSN:<span class="NLM_cas:issn">0887-6924</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Janus kinases (JAKs) are required for cytokine receptor signaling.  Since the discovery of the highly prevalent JAK2 V617F mutation in myeloproliferative neoplasms (MPNs), JAK2 became a prime target for inhibition.  Only one approved JAK2 inhibitor exists, with pos., but not curative effects in MPNs, and promising effects in autoimmune diseases and cancer.  On the basis of recent advances in the structural features regulating both normal and mutant JAKs, as well as in small-mol. targeting, we review the current state of JAK2 inhibitor development and present novel avenues of selecting JAK2 inhibitors, with broad and narrow specificities and extend these approaches to other JAKs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLVJS8PwHOlbVg90H21EOLACvtfcHk0liDxSSU9FVDkg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXislyms7w%253D&md5=0dc333d94e27daad1d32110dee26e637</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fleu.2017.43&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fleu.2017.43%26sid%3Dliteratum%253Aachs%26aulast%3DLeroy%26aufirst%3DE.%26aulast%3DConstantinescu%26aufirst%3DS.%2BN.%26atitle%3DRethinking%2520JAK2%2520inhibition%253A%2520towards%2520novel%2520strategies%2520of%2520more%2520specific%2520and%2520versatile%2520Janus%2520kinase%2520inhibition%26jtitle%3DLeukemia%26date%3D2017%26volume%3D31%26spage%3D1023%26epage%3D1038%26doi%3D10.1038%2Fleu.2017.43" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baxter, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">East, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fourouclas, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swanton, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vassiliou, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bench, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boyd, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curtin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erber, W. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, A. R.</span></span> <span> </span><span class="NLM_article-title">Acquired mutation of the tyrosine kinase JAK2 in human myeloproliferative disorders</span>. <i>Lancet</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>365</i></span>,  <span class="NLM_fpage">1054</span>â <span class="NLM_lpage">1061</span>, <span class="refDoi">Â DOI: 10.1016/S0140-6736(05)71142-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=10.1016%2FS0140-6736%2805%2971142-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=15781101" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=1%3ACAS%3A528%3ADC%252BD2MXisFGrs7g%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=365&publication_year=2005&pages=1054-1061&author=E.+J.+Baxterauthor=L.+M.+Scottauthor=P.+J.+Campbellauthor=C.+Eastauthor=N.+Fourouclasauthor=S.+Swantonauthor=G.+S.+Vassiliouauthor=A.+J.+Benchauthor=E.+M.+Boydauthor=N.+Curtinauthor=M.+A.+Scottauthor=W.+N.+Erberauthor=A.+R.+Green&title=Acquired+mutation+of+the+tyrosine+kinase+JAK2+in+human+myeloproliferative+disorders&doi=10.1016%2FS0140-6736%2805%2971142-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Acquired mutation of the tyrosine kinase JAK2 in human myeloproliferative disorders</span></div><div class="casAuthors">Baxter, E. Joanna; Scott, Linda M.; Campbell, Peter J.; East, Clare; Fourouclas, Nasios; Swanton, Soheila; Vassiliou, George S.; Bench, Anthony J.; Boyd, Elaine M.; Curtin, Natasha; Scott, Mike A.; Erber, Wendy N.; Green, Anthony R.</div><div class="citationInfo"><span class="NLM_cas:title">Lancet</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">365</span>
        (<span class="NLM_cas:issue">9464</span>),
    <span class="NLM_cas:pages">1054-1061</span>CODEN:
                <span class="NLM_cas:coden">LANCAO</span>;
        ISSN:<span class="NLM_cas:issn">0140-6736</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Background: Human myeloproliferative disorders form a range of clonal haematol. malignant diseases, the main members of which are polycythemia vera, essential thrombocythemia, and idiopathic myelofibrosis.  The mol. pathogenesis of these disorders is unknown, but tyrosine kinases have been implicated in several related disorders.  We investigated the role of the cytoplasmic tyrosine kinase JAK2 in patients with a myeloproliferative disorder.  Methods: We obtained DNA samples from patients with polycythemia vera, essential thrombocythemia, or idiopathic myelofibrosis.  The coding exons of JAK2 were bidirectionally sequenced from peripheral-blood granulocytes, T cells, or both.  Allele-specific PCR, mol. cytogenetic studies, microsatellite PCR, Affymetrix single nucleotide polymorphism array analyses, and colony assays were undertaken on subgroups of patients.  Findings: A single point mutation (Val617Phe) was identified in JAK2 in 71 (97%) of 73 patients with polycythemia vera, 29 (57%) of 51 with essential thrombocythemia, and eight (50%) of 16 with idiopathic myelofibrosis.  The mutation is acquired, is present in a variable proportion of granulocytes, alters a highly conserved valine present in the neg. regulatory JH2 domain, and is predicted to dysregulate kinase activity.  It was heterozygous in most patients, homozygous in a subset as a result of mitotic recombination, and arose in a multipotent progenitor capable of giving rise to erythroid and myeloid cells.  The mutation was present in all erythropoietin-independent erythroid colonies.  Interpretation: A single acquired mutation of JAK2 was noted in more than half of patients with a myeloproliferative disorder.  Its presence in all erythropoietin-independent erythroid colonies demonstrates a link with growth factor hypersensitivity, a key biol. feature of these disorders.  Relevance to practice: Identification of the Val617Phe JAK2 mutation lays the foundation for new approaches to the diagnosis, classification, and treatment of myeloproliferative disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8cc-kg2leZrVg90H21EOLACvtfcHk0liDxSSU9FVDkg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXisFGrs7g%253D&md5=3c9dc565a1986a47d12e4bccd8c8cc1a</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2FS0140-6736%2805%2971142-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0140-6736%252805%252971142-9%26sid%3Dliteratum%253Aachs%26aulast%3DBaxter%26aufirst%3DE.%2BJ.%26aulast%3DScott%26aufirst%3DL.%2BM.%26aulast%3DCampbell%26aufirst%3DP.%2BJ.%26aulast%3DEast%26aufirst%3DC.%26aulast%3DFourouclas%26aufirst%3DN.%26aulast%3DSwanton%26aufirst%3DS.%26aulast%3DVassiliou%26aufirst%3DG.%2BS.%26aulast%3DBench%26aufirst%3DA.%2BJ.%26aulast%3DBoyd%26aufirst%3DE.%2BM.%26aulast%3DCurtin%26aufirst%3DN.%26aulast%3DScott%26aufirst%3DM.%2BA.%26aulast%3DErber%26aufirst%3DW.%2BN.%26aulast%3DGreen%26aufirst%3DA.%2BR.%26atitle%3DAcquired%2520mutation%2520of%2520the%2520tyrosine%2520kinase%2520JAK2%2520in%2520human%2520myeloproliferative%2520disorders%26jtitle%3DLancet%26date%3D2005%26volume%3D365%26spage%3D1054%26epage%3D1061%26doi%3D10.1016%2FS0140-6736%2805%2971142-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Levine, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wadleigh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cools, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ebert, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wernig, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huntly, B. J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boggon, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wlodarska, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moore, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adelsperger, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabriel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mercher, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DâAndrea, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">FrÃ¶hling, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DÃ¶hner, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marynen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vandenberghe, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mesa, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tefferi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffein, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eck, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sellers, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyerson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golub, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilliland, D. G.</span></span> <span> </span><span class="NLM_article-title">Activating mutation in the tyrosine kinase JAK2 in polycythemia vera, essential thrombocythemia, and myeloid metaplasia with myelofibrosis</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">387</span>â <span class="NLM_lpage">397</span>, <span class="refDoi">Â DOI: 10.1016/j.ccr.2005.03.023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=10.1016%2Fj.ccr.2005.03.023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=15837627" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=1%3ACAS%3A528%3ADC%252BD2MXjvFSku7o%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2005&pages=387-397&author=R.+L.+Levineauthor=M.+Wadleighauthor=J.+Coolsauthor=B.+L.+Ebertauthor=G.+Wernigauthor=B.+J.+P.+Huntlyauthor=T.+J.+Boggonauthor=L.+Wlodarskaauthor=J.+J.+Clarkauthor=S.+Mooreauthor=J.+Adelspergerauthor=S.+Kooauthor=J.+C.+Leeauthor=S.+Gabrielauthor=T.+Mercherauthor=A.+D%E2%80%99Andreaauthor=S.+Fr%C3%B6hlingauthor=K.+D%C3%B6hnerauthor=P.+Marynenauthor=P.+Vandenbergheauthor=R.+A.+Mesaauthor=A.+Tefferiauthor=J.+D.+Griffeinauthor=M.+J.+Eckauthor=W.+R.+Sellersauthor=M.+Meyersonauthor=T.+R.+Golubauthor=S.+J.+Leeauthor=D.+G.+Gilliland&title=Activating+mutation+in+the+tyrosine+kinase+JAK2+in+polycythemia+vera%2C+essential+thrombocythemia%2C+and+myeloid+metaplasia+with+myelofibrosis&doi=10.1016%2Fj.ccr.2005.03.023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Activating mutation in the tyrosine kinase JAK2 in polycythemia vera, essential thrombocythemia, and myeloid metaplasia with myelofibrosis</span></div><div class="casAuthors">Levine, Ross L.; Wadleigh, Martha; Cools, Jan; Ebert, Benjamin L.; Wernig, Gerlinde; Huntly, Brian J. P.; Boggon, Titus J.; Wlodarska, Iwona; Clark, Jennifer J.; Moore, Sandra; Adelsperger, Jennifer; Koo, Sumin; Lee, Jeffrey C.; Gabriel, Stacey; Mercher, Thomas; D'Andrea, Alan; Froehling, Stefan; Doehner, Konstanze; Marynen, Peter; Vandenberghe, Peter; Mesa, Ruben A.; Tefferi, Ayalew; Griffin, James D.; Eck, Michael J.; Sellers, William R.; Meyerson, Matthew; Golub, Todd R.; Lee, Stephanie J.; Gilliland, D. Gary</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">387-397</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Polycythemia vera (PV), essential thrombocythemia (ET), and myeloid metaplasia with myelofibrosis (MMM) are clonal disorders arising from hematopoietic progenitors.  An internet-based protocol was used to collect clin. information and biol. specimens from patients with these diseases.  High-throughput DNA re-sequencing identified a recurrent somatic missense mutation JAK2V617F in granulocyte DNA samples of 121 of 164 PV patients, of which 41 had homozygous and 80 had heterozygous mutations.  Mol. and cytogenetic analyses demonstrated that homozygous mutations were due to duplication of the mutant allele.  JAK2V617F was also identified in granulocyte DNA samples from 37 of 115 ET and 16 of 46 MMM patients, but was not obsd. in 269 normal individuals.  In vitro anal. demonstrated that JAK2V617F is a constitutively active tyrosine kinase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXJNzUFm3Q3bVg90H21EOLACvtfcHk0lhbWFNwl0kzAg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXjvFSku7o%253D&md5=2a3e893867b114e8749cd6f5e55990b7</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2005.03.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2005.03.023%26sid%3Dliteratum%253Aachs%26aulast%3DLevine%26aufirst%3DR.%2BL.%26aulast%3DWadleigh%26aufirst%3DM.%26aulast%3DCools%26aufirst%3DJ.%26aulast%3DEbert%26aufirst%3DB.%2BL.%26aulast%3DWernig%26aufirst%3DG.%26aulast%3DHuntly%26aufirst%3DB.%2BJ.%2BP.%26aulast%3DBoggon%26aufirst%3DT.%2BJ.%26aulast%3DWlodarska%26aufirst%3DL.%26aulast%3DClark%26aufirst%3DJ.%2BJ.%26aulast%3DMoore%26aufirst%3DS.%26aulast%3DAdelsperger%26aufirst%3DJ.%26aulast%3DKoo%26aufirst%3DS.%26aulast%3DLee%26aufirst%3DJ.%2BC.%26aulast%3DGabriel%26aufirst%3DS.%26aulast%3DMercher%26aufirst%3DT.%26aulast%3DD%25E2%2580%2599Andrea%26aufirst%3DA.%26aulast%3DFr%25C3%25B6hling%26aufirst%3DS.%26aulast%3DD%25C3%25B6hner%26aufirst%3DK.%26aulast%3DMarynen%26aufirst%3DP.%26aulast%3DVandenberghe%26aufirst%3DP.%26aulast%3DMesa%26aufirst%3DR.%2BA.%26aulast%3DTefferi%26aufirst%3DA.%26aulast%3DGriffein%26aufirst%3DJ.%2BD.%26aulast%3DEck%26aufirst%3DM.%2BJ.%26aulast%3DSellers%26aufirst%3DW.%2BR.%26aulast%3DMeyerson%26aufirst%3DM.%26aulast%3DGolub%26aufirst%3DT.%2BR.%26aulast%3DLee%26aufirst%3DS.%2BJ.%26aulast%3DGilliland%26aufirst%3DD.%2BG.%26atitle%3DActivating%2520mutation%2520in%2520the%2520tyrosine%2520kinase%2520JAK2%2520in%2520polycythemia%2520vera%252C%2520essential%2520thrombocythemia%252C%2520and%2520myeloid%2520metaplasia%2520with%2520myelofibrosis%26jtitle%3DCancer%2520Cell%26date%3D2005%26volume%3D7%26spage%3D387%26epage%3D397%26doi%3D10.1016%2Fj.ccr.2005.03.023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">HammarÃ©n, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ungureanu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grisouard, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skoda, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silvennoinen, O.</span></span> <span> </span><span class="NLM_article-title">ATP binding to the pseudokinase domain of JAK2 is critical for pathogenic activation</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>112</i></span>,  <span class="NLM_fpage">4642</span>â <span class="NLM_lpage">4647</span>, <span class="refDoi">Â DOI: 10.1073/pnas.1423201112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=10.1073%2Fpnas.1423201112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=25825724" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlsVOntL8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2015&pages=4642-4647&author=H.+M.+Hammar%C3%A9nauthor=D.+Ungureanuauthor=J.+Grisouardauthor=R.+C.+Skodaauthor=S.+R.+Hubbardauthor=O.+Silvennoinen&title=ATP+binding+to+the+pseudokinase+domain+of+JAK2+is+critical+for+pathogenic+activation&doi=10.1073%2Fpnas.1423201112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">ATP binding to the pseudokinase domain of JAK2 is critical for pathogenic activation</span></div><div class="casAuthors">Hammaren, Henrik M.; Ungureanu, Daniela; Grisouard, Jean; Skoda, Radek C.; Hubbard, Stevan R.; Silvennoinen, Olli</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">4642-4647</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Pseudokinases lack conserved motifs typically required for kinase activity.  Nearly half of pseudokinases bind ATP, but only few retain phosphotransfer activity, leaving the functional role of nucleotide binding in most cases unknown.  Janus kinases (JAKs) are nonreceptor tyrosine kinases with a tandem pseudokinase-kinase domain configuration, where the pseudokinase domain (JAK homol. 2, JH2) has important regulatory functions and harbors mutations underlying hematol. and immunol. diseases.  JH2 of JAK1, JAK2, and TYK2 all bind ATP, but the significance of this is unclear.  We characterize the role of nucleotide binding in normal and pathogenic JAK signaling using comprehensive structure-based mutagenesis.  Disruption of JH2 ATP binding in wild-type JAK2 has only minor effects, and in the presence of type I cytokine receptors, the mutations do not affect JAK2 activation.  However, JH2 mutants devoid of ATP binding ameliorate the hyperactivation of JAK2 V617F.  Disrupting ATP binding in JH2 also inhibits the hyperactivity of other pathogenic JAK2 mutants, as well as of JAK1 V658F, and prevents induction of erythrocytosis in a JAK2 V617F myeloproliferative neoplasm mouse model.  Mol. dynamic simulations and thermal-shift anal. indicate that ATP binding stabilizes JH2, with a pronounced effect on the C helix region, which plays a crit. role in pathogenic activation of JAK2.  Taken together, our results suggest that ATP binding to JH2 serves a structural role in JAKs, which is required for aberrant activity of pathogenic JAK mutants.  The inhibitory effect of abrogating JH2 ATP binding in pathogenic JAK mutants may warrant novel therapeutic approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowpDHSf5yGQ7Vg90H21EOLACvtfcHk0lhbWFNwl0kzAg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlsVOntL8%253D&md5=7cb7b90a1978ef02628f9ce9e85622ad</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1423201112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1423201112%26sid%3Dliteratum%253Aachs%26aulast%3DHammar%25C3%25A9n%26aufirst%3DH.%2BM.%26aulast%3DUngureanu%26aufirst%3DD.%26aulast%3DGrisouard%26aufirst%3DJ.%26aulast%3DSkoda%26aufirst%3DR.%2BC.%26aulast%3DHubbard%26aufirst%3DS.%2BR.%26aulast%3DSilvennoinen%26aufirst%3DO.%26atitle%3DATP%2520binding%2520to%2520the%2520pseudokinase%2520domain%2520of%2520JAK2%2520is%2520critical%2520for%2520pathogenic%2520activation%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2015%26volume%3D112%26spage%3D4642%26epage%3D4647%26doi%3D10.1073%2Fpnas.1423201112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kung, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jura, N.</span></span> <span> </span><span class="NLM_article-title">Prospects for pharmacological targeting of pseudokinases</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">501</span>â <span class="NLM_lpage">526</span>, <span class="refDoi">Â DOI: 10.1038/s41573-019-0018-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=10.1038%2Fs41573-019-0018-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=30850748" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=1%3ACAS%3A528%3ADC%252BC1MXms1GjtLw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=501-526&author=J.+E.+Kungauthor=N.+Jura&title=Prospects+for+pharmacological+targeting+of+pseudokinases&doi=10.1038%2Fs41573-019-0018-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Prospects for pharmacological targeting of pseudokinases</span></div><div class="casAuthors">Kung, Jennifer E.; Jura, Natalia</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">501-526</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  Pseudokinases are members of the protein kinase superfamily but signal primarily through noncatalytic mechanisms.  Many pseudokinases contribute to the pathologies of human diseases, yet they remain largely unexplored as drug targets owing to challenges assocd. with modulation of their biol. functions.  Our understanding of the structure and physiol. roles of pseudokinases has improved substantially over the past decade, revealing intriguing similarities between pseudokinases and their catalytically active counterparts.  Pseudokinases often adopt conformations that are analogous to those seen in catalytically active kinases and, in some cases, can also bind metal cations and/or nucleotides.  Several clin. approved kinase inhibitors have been shown to influence the noncatalytic functions of active kinases, providing hope that similar properties in pseudokinases could be pharmacol. regulated.  In this Review, we discuss known roles of pseudokinases in disease, their unique structural features and the progress that has been made towards developing pseudokinase-directed therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUFxQnlHTnn7Vg90H21EOLACvtfcHk0lhWFEIZf4CgWA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXms1GjtLw%253D&md5=f3414770154e8d6449b454f9bc3f19bd</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fs41573-019-0018-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41573-019-0018-3%26sid%3Dliteratum%253Aachs%26aulast%3DKung%26aufirst%3DJ.%2BE.%26aulast%3DJura%26aufirst%3DN.%26atitle%3DProspects%2520for%2520pharmacological%2520targeting%2520of%2520pseudokinases%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2019%26volume%3D18%26spage%3D501%26epage%3D526%26doi%3D10.1038%2Fs41573-019-0018-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Connolly, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wetter, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, W. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emanuel, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gruninger, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuentes-Pesquera, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rugg, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Middleton, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jolliffe, L. K.</span></span> <span> </span><span class="NLM_article-title">1-acyl-1<i>H</i>-[1,2,4]triazole-3,5-diamine analogues as novel and potent anticancer cyclin-dependent kinase inhibitors: synthesis and evaluation of biological activities</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">4208</span>â <span class="NLM_lpage">4211</span>, <span class="refDoi">Â DOI: 10.1021/jm050267e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm050267e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=1%3ACAS%3A528%3ADC%252BD2MXkslSgsbs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=4208-4211&author=R.+Linauthor=P.+J.+Connollyauthor=S.+Huangauthor=S.+K.+Wetterauthor=Y.+Luauthor=W.+V.+Murrayauthor=S.+L.+Emanuelauthor=R.+H.+Gruningerauthor=A.+R.+Fuentes-Pesqueraauthor=C.+A.+Ruggauthor=S.+A.+Middletonauthor=L.+K.+Jolliffe&title=1-acyl-1H-%5B1%2C2%2C4%5Dtriazole-3%2C5-diamine+analogues+as+novel+and+potent+anticancer+cyclin-dependent+kinase+inhibitors%3A+synthesis+and+evaluation+of+biological+activities&doi=10.1021%2Fjm050267e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">1-Acyl-1H-[1,2,4]triazole-3,5-diamine Analogues as Novel and Potent Anticancer Cyclin-Dependent Kinase Inhibitors: Synthesis and Evaluation of Biological Activities</span></div><div class="casAuthors">Lin, Ronghui; Connolly, Peter J.; Huang, Shenlin; Wetter, Steven K.; Lu, Yanhua; Murray, William V.; Emanuel, Stuart L.; Gruninger, Robert H.; Fuentes-Pesquera, Angel R.; Rugg, Catherine A.; Middleton, Steven A.; Jolliffe, Linda K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">4208-4211</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of 1-acyl-1H-[1,2,4]triazole-3,5-diamine analogs, e.g., I, were synthesized as cyclin-dependent kinase (CDK) inhibitors.  These compds. showed potent and selective CDK1 and CDK2 inhibitory activities and inhibited in vitro cellular proliferation in various human tumor cells.  I demonstrated in vivo efficacy in a human melanoma A375 xenograft model in nude mice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrN9p2ArpQa4bVg90H21EOLACvtfcHk0lhWFEIZf4CgWA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXkslSgsbs%253D&md5=6ccbcbc6136bfd61d20c5e369d8fb31a</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Fjm050267e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm050267e%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DR.%26aulast%3DConnolly%26aufirst%3DP.%2BJ.%26aulast%3DHuang%26aufirst%3DS.%26aulast%3DWetter%26aufirst%3DS.%2BK.%26aulast%3DLu%26aufirst%3DY.%26aulast%3DMurray%26aufirst%3DW.%2BV.%26aulast%3DEmanuel%26aufirst%3DS.%2BL.%26aulast%3DGruninger%26aufirst%3DR.%2BH.%26aulast%3DFuentes-Pesquera%26aufirst%3DA.%2BR.%26aulast%3DRugg%26aufirst%3DC.%2BA.%26aulast%3DMiddleton%26aufirst%3DS.%2BA.%26aulast%3DJolliffe%26aufirst%3DL.%2BK.%26atitle%3D1-acyl-1H-%255B1%252C2%252C4%255Dtriazole-3%252C5-diamine%2520analogues%2520as%2520novel%2520and%2520potent%2520anticancer%2520cyclin-dependent%2520kinase%2520inhibitors%253A%2520synthesis%2520and%2520evaluation%2520of%2520biological%2520activities%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D4208%26epage%3D4211%26doi%3D10.1021%2Fjm050267e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Karaman, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrgard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treiber, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallant, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atteridge, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciceri, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edeen, P. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faraoni, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Floyd, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lockhart, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milanov, Z. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morrison, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pallares, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, H. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pritchard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wodicka, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zarrinkar, P. P.</span></span> <span> </span><span class="NLM_article-title">A quantitative analysis of kinase inhibitor selectivity</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">127</span>â <span class="NLM_lpage">132</span>, <span class="refDoi">Â DOI: 10.1038/nbt1358</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=10.1038%2Fnbt1358" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=18183025" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=1%3ACAS%3A528%3ADC%252BD1cXisFGlsQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2008&pages=127-132&author=M.+W.+Karamanauthor=S.+Herrgardauthor=D.+K.+Treiberauthor=P.+Gallantauthor=C.+E.+Atteridgeauthor=B.+T.+Campbellauthor=K.+W.+Chanauthor=P.+Ciceriauthor=M.+I.+Davisauthor=P.+T.+Edeenauthor=R.+Faraoniauthor=M.+Floydauthor=J.+P.+Huntauthor=D.+J.+Lockhartauthor=Z.+V.+Milanovauthor=M.+J.+Morrisonauthor=G.+Pallaresauthor=H.+K.+Patelauthor=S.+Pritchardauthor=L.+M.+Wodickaauthor=P.+P.+Zarrinkar&title=A+quantitative+analysis+of+kinase+inhibitor+selectivity&doi=10.1038%2Fnbt1358"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">A quantitative analysis of kinase inhibitor selectivity</span></div><div class="casAuthors">Karaman, Mazen W.; Herrgard, Sanna; Treiber, Daniel K.; Gallant, Paul; Atteridge, Corey E.; Campbell, Brian T.; Chan, Katrina W.; Ciceri, Pietro; Davis, Mindy I.; Edeen, Philip T.; Faraoni, Raffaella; Floyd, Mark; Hunt, Jeremy P.; Lockhart, Daniel J.; Milanov, Zdravko V.; Morrison, Michael J.; Pallares, Gabriel; Patel, Hitesh K.; Pritchard, Stephanie; Wodicka, Lisa M.; Zarrinkar, Patrick P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">127-132</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Kinase inhibitors are a new class of therapeutics with a propensity to inhibit multiple targets.  The biol. consequences of multikinase activity are poorly defined, and an important step toward understanding the relationship between selectivity, efficacy and safety is the exploration of how inhibitors interact with the human kinome.  The authors present interaction maps for 38 kinase inhibitors across a panel of 317 kinases representing >50% of the predicted human protein kinome.  The data constitute the most comprehensive study of kinase inhibitor selectivity to date and reveal a wide diversity of interaction patterns.  To enable a global anal. of the results, the authors introduce the concept of a selectivity score as a general tool to quantify and differentiate the obsd. interaction patterns.  The authors further investigate the impact of panel size and find that small assay panels do not provide a robust measure of selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiArJSq2_lPrVg90H21EOLACvtfcHk0lhWFEIZf4CgWA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXisFGlsQ%253D%253D&md5=346265d412853ced636ad4128ed8a76f</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnbt1358&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1358%26sid%3Dliteratum%253Aachs%26aulast%3DKaraman%26aufirst%3DM.%2BW.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DGallant%26aufirst%3DP.%26aulast%3DAtteridge%26aufirst%3DC.%2BE.%26aulast%3DCampbell%26aufirst%3DB.%2BT.%26aulast%3DChan%26aufirst%3DK.%2BW.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DDavis%26aufirst%3DM.%2BI.%26aulast%3DEdeen%26aufirst%3DP.%2BT.%26aulast%3DFaraoni%26aufirst%3DR.%26aulast%3DFloyd%26aufirst%3DM.%26aulast%3DHunt%26aufirst%3DJ.%2BP.%26aulast%3DLockhart%26aufirst%3DD.%2BJ.%26aulast%3DMilanov%26aufirst%3DZ.%2BV.%26aulast%3DMorrison%26aufirst%3DM.%2BJ.%26aulast%3DPallares%26aufirst%3DG.%26aulast%3DPatel%26aufirst%3DH.%2BK.%26aulast%3DPritchard%26aufirst%3DS.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26atitle%3DA%2520quantitative%2520analysis%2520of%2520kinase%2520inhibitor%2520selectivity%26jtitle%3DNat.%2520Biotechnol.%26date%3D2008%26volume%3D26%26spage%3D127%26epage%3D132%26doi%3D10.1038%2Fnbt1358" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Puleo, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kucera, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">HammarÃ©n, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ungureanu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Newton, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silvennoinen, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jorgensen, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlessinger, J.</span></span> <span> </span><span class="NLM_article-title">Identification and characterization of JAK2 pseudokinase domain small molecule binders</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">618</span>â <span class="NLM_lpage">621</span>, <span class="refDoi">Â DOI: 10.1021/acsmedchemlett.7b00153</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.7b00153" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=1%3ACAS%3A528%3ADC%252BC2sXnvFClsr4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=618-621&author=D.+E.+Puleoauthor=K.+Kuceraauthor=H.+M.+Hammar%C3%A9nauthor=D.+Ungureanuauthor=A.+S.+Newtonauthor=O.+Silvennoinenauthor=W.+L.+Jorgensenauthor=J.+Schlessinger&title=Identification+and+characterization+of+JAK2+pseudokinase+domain+small+molecule+binders&doi=10.1021%2Facsmedchemlett.7b00153"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and Characterization of JAK2 Pseudokinase Domain Small Molecule Binders</span></div><div class="casAuthors">Puleo, David E.; Kucera, Kaury; Hammaren, Henrik M.; Ungureanu, Daniela; Newton, Ana S.; Silvennoinen, Olli; Jorgensen, William L.; Schlessinger, Joseph</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">618-621</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Janus kinases (JAKs) regulate hematopoiesis via the cytokine-mediated JAK-STAT signaling pathway.  JAKs contain tandem C-terminal pseudokinase (JH2) and tyrosine kinase (JH1) domains.  The JAK2 pseudokinase domain adopts a protein kinase fold and, despite its pseudokinase designation, binds ATP with micromolar affinity.  Recent evidence shows that displacing ATP from the JAK2 JH2 domain alters the hyperactivation state of the oncogenic JAK2 V617F protein while sparing the wild type JAK2 protein.  In this study, small mol. binders of JAK2 JH2 were identified via an in vitro screen.  Top hits were characterized using biophys. and structural approaches.  Development of pseudokinase-selective compds. may offer novel pharmacol. opportunities for treating cancers driven by JAK2 V617F and other oncogenic JAK mutants.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmtcoH5qmUWrVg90H21EOLACvtfcHk0lixHUmkADJw_Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXnvFClsr4%253D&md5=a3f1f6e10a8a1d3b64bb5691d246f5d0</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.7b00153&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.7b00153%26sid%3Dliteratum%253Aachs%26aulast%3DPuleo%26aufirst%3DD.%2BE.%26aulast%3DKucera%26aufirst%3DK.%26aulast%3DHammar%25C3%25A9n%26aufirst%3DH.%2BM.%26aulast%3DUngureanu%26aufirst%3DD.%26aulast%3DNewton%26aufirst%3DA.%2BS.%26aulast%3DSilvennoinen%26aufirst%3DO.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DSchlessinger%26aufirst%3DJ.%26atitle%3DIdentification%2520and%2520characterization%2520of%2520JAK2%2520pseudokinase%2520domain%2520small%2520molecule%2520binders%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D8%26spage%3D618%26epage%3D621%26doi%3D10.1021%2Facsmedchemlett.7b00153" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bandaranayake, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ungureanu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silvennoinen, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, S. R.</span></span> <span> </span><span class="NLM_article-title">Crystal structures of the JAK2 pseudokinase domain and the pathogenic mutant V617F</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">754</span>â <span class="NLM_lpage">759</span>, <span class="refDoi">Â DOI: 10.1038/nsmb.2348</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=10.1038%2Fnsmb.2348" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=22820988" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVKntLbM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2012&pages=754-759&author=R.+M.+Bandaranayakeauthor=D.+Ungureanuauthor=Y.+Shanauthor=D.+E.+Shawauthor=O.+Silvennoinenauthor=S.+R.+Hubbard&title=Crystal+structures+of+the+JAK2+pseudokinase+domain+and+the+pathogenic+mutant+V617F&doi=10.1038%2Fnsmb.2348"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structures of the JAK2 pseudokinase domain and the pathogenic mutant V617F</span></div><div class="casAuthors">Bandaranayake, Rajintha M.; Ungureanu, Daniela; Shan, Yibing; Shaw, David E.; Silvennoinen, Olli; Hubbard, Stevan R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">754-759</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The protein tyrosine kinase JAK2 mediates signaling through numerous cytokine receptors.  JAK2 possesses a pseudokinase domain (JH2) and a tyrosine kinase domain (JH1).  Through unknown mechanisms, JH2 regulates the catalytic activity of JH1, and hyperactivating mutations in the JH2 region of human JAK2 cause myeloproliferative neoplasms (MPNs).  We showed previously that JAK2 JH2 is, in fact, catalytically active.  Here we present crystal structures of human JAK2 JH2, including both wild type and the most prevalent MPN mutant, V617F.  The structures reveal that JH2 adopts the fold of a prototypical protein kinase but binds Mg-ATP noncanonically.  The structural and biochem. data indicate that the V617F mutation rigidifies Î±-helix C in the N lobe of JH2, facilitating trans-phosphorylation of JH1.  The crystal structures of JH2 afford new opportunities for the design of novel JAK2 therapeutics targeting MPNs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGj9DoN3fj37Vg90H21EOLACvtfcHk0lixHUmkADJw_Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVKntLbM&md5=da64398430cd3279e131b2569cdc7391</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.2348&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.2348%26sid%3Dliteratum%253Aachs%26aulast%3DBandaranayake%26aufirst%3DR.%2BM.%26aulast%3DUngureanu%26aufirst%3DD.%26aulast%3DShan%26aufirst%3DY.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26aulast%3DSilvennoinen%26aufirst%3DO.%26aulast%3DHubbard%26aufirst%3DS.%2BR.%26atitle%3DCrystal%2520structures%2520of%2520the%2520JAK2%2520pseudokinase%2520domain%2520and%2520the%2520pathogenic%2520mutant%2520V617F%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2012%26volume%3D19%26spage%3D754%26epage%3D759%26doi%3D10.1038%2Fnsmb.2348" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dusa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mouton, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pecquet, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herman, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Constantinescu, S. N.</span></span> <span> </span><span class="NLM_article-title">JAK2 V617F constitutive activation requires JH2 residue F595: a pseudokinase domain target for specific inhibitors</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>5</i></span>, <span class="NLM_elocation-id">e11157</span> <span class="refDoi">Â DOI: 10.1371/journal.pone.0011157</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=10.1371%2Fjournal.pone.0011157" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=20585391" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=1%3ACAS%3A280%3ADC%252BC3cnivVShsw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2010&author=A.+Dusaauthor=C.+Moutonauthor=C.+Pecquetauthor=M.+Hermanauthor=S.+N.+Constantinescu&title=JAK2+V617F+constitutive+activation+requires+JH2+residue+F595%3A+a+pseudokinase+domain+target+for+specific+inhibitors&doi=10.1371%2Fjournal.pone.0011157"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">JAK2 V617F constitutive activation requires JH2 residue F595: a pseudokinase domain target for specific inhibitors</span></div><div class="casAuthors">Dusa Alexandra; Mouton Celine; Pecquet Christian; Herman Murielle; Constantinescu Stefan N</div><div class="citationInfo"><span class="NLM_cas:title">PloS one</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">e11157</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The JAK2 V617F mutation present in over 95% of Polycythemia Vera patients and in 50% of Essential Thrombocythemia and Primary Myelofibrosis patients renders the kinase constitutively active.  In the absence of a three-dimensional structure for the full-length protein, the mechanism of activation of JAK2 V617F has remained elusive.  In this study, we used functional mutagenesis to investigate the involvement of the JH2 alphaC helix in the constitutive activation of JAK2 V617F.  We show that residue F595, located in the middle of the alphaC helix of JH2, is indispensable for the constitutive activity of JAK2 V617F.  Mutation of F595 to Ala, Lys, Val or Ile significantly decreases the constitutive activity of JAK2 V617F, but F595W and F595Y are able to restore it, implying an aromaticity requirement at position 595.  Substitution of F595 to Ala was also able to decrease the constitutive activity of two other JAK2 mutants, T875N and R683G, as well as JAK2 K539L, albeit to a lower extent.  In contrast, the F595 mutants are activated by erythropoietin-bound EpoR.  We also explored the relationship between the dimeric conformation of EpoR and several JAK2 mutants.  Since residue F595 is crucial to the constitutive activation of JAK2 V617F but not to initiation of JAK2 activation by cytokines, we suggest that small molecules that target the region around this residue might specifically block oncogenic JAK2 and spare JAK2 wild-type.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ-dhwX-6R6Z8KL0FlyGHDofW6udTcc2eax47sz-wVPurntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3cnivVShsw%253D%253D&md5=0b95c5aa9f0644c1f1d94d2a09c858e5</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0011157&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0011157%26sid%3Dliteratum%253Aachs%26aulast%3DDusa%26aufirst%3DA.%26aulast%3DMouton%26aufirst%3DC.%26aulast%3DPecquet%26aufirst%3DC.%26aulast%3DHerman%26aufirst%3DM.%26aulast%3DConstantinescu%26aufirst%3DS.%2BN.%26atitle%3DJAK2%2520V617F%2520constitutive%2520activation%2520requires%2520JH2%2520residue%2520F595%253A%2520a%2520pseudokinase%2520domain%2520target%2520for%2520specific%2520inhibitors%26jtitle%3DPLoS%2520One%26date%3D2010%26volume%3D5%26doi%3D10.1371%2Fjournal.pone.0011157" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jorgensen, W. L.</span></span> <span> </span><span class="NLM_article-title">Efficient drug lead discovery and optimization</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">724</span>â <span class="NLM_lpage">733</span>, <span class="refDoi">Â DOI: 10.1021/ar800236t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ar800236t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjsFentr4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2009&pages=724-733&author=W.+L.+Jorgensen&title=Efficient+drug+lead+discovery+and+optimization&doi=10.1021%2Far800236t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Efficient Drug Lead Discovery and Optimization</span></div><div class="casAuthors">Jorgensen, William L.</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">724-733</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  During the 1980s, advances in the abilities to perform computer simulations of chem. and biomol. systems and to calc. free energy changes led to the expectation that such methodol. would soon show great utility for guiding mol. design.  Important potential applications included design of selective receptors, catalysts, and regulators of biol. function including enzyme inhibitors.  This time also saw the rise of high-throughput screening and combinatorial chem. along with complementary computational methods for de novo design and virtual screening including docking.  These technologies appeared poised to deliver diverse lead compds. for any biol. target.  As with many technol. advances, realization of the expectations required significant addnl. effort and time.  However, as summarized here, striking success has now been achieved for computer-aided drug lead generation and optimization.  De novo design using both mol. growing and docking are illustrated for lead generation, and lead optimization features free energy perturbation calcns. in conjunction with Monte Carlo statistical mechanics simulations for protein-inhibitor complexes in aq. soln.  The specific applications are to the discovery of non-nucleoside inhibitors of HIV reverse transcriptase (HIV-RT) and inhibitors of the binding of the proinflammatory cytokine MIF to its receptor CD74.  A std. protocol is presented that includes scans for possible addns. of small substituents to a mol. core, interchange of heterocycles, and focused optimization of substituents at one site.  Initial leads with activities at low-micromolar concns. have been advanced rapidly to low-nanomolar inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrn0YpWuo1jCLVg90H21EOLACvtfcHk0liApOZKCL0blw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjsFentr4%253D&md5=9641587769701c6541b38e18fa05538a</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Far800236t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Far800236t%26sid%3Dliteratum%253Aachs%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26atitle%3DEfficient%2520drug%2520lead%2520discovery%2520and%2520optimization%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2009%26volume%3D42%26spage%3D724%26epage%3D733%26doi%3D10.1021%2Far800236t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jorgensen, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tirado-Rives, J.</span></span> <span> </span><span class="NLM_article-title">Molecular modeling of organic and biomolecular systems using BOSS and MCPRO</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">1689</span>â <span class="NLM_lpage">1700</span>, <span class="refDoi">Â DOI: 10.1002/jcc.20297</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=10.1002%2Fjcc.20297" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=16200637" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1SlsbbN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2005&pages=1689-1700&author=W.+L.+Jorgensenauthor=J.+Tirado-Rives&title=Molecular+modeling+of+organic+and+biomolecular+systems+using+BOSS+and+MCPRO&doi=10.1002%2Fjcc.20297"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular modeling of organic and biomolecular systems using BOSS and MCPRO</span></div><div class="casAuthors">Jorgensen, William L.; Tirado-Rives, Julian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1689-1700</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">An overview is provided of the capabilities for the current versions of the BOSS and MCPRO programs for mol. modeling of org. and biomol. systems.  Recent applications are noted, particularly for QM/MM studies of org. and enzymic reactions and for protein-ligand binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdrU3aGV4BQLVg90H21EOLACvtfcHk0liApOZKCL0blw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1SlsbbN&md5=e487e9838d39e4b39a5d06a240bfc1e0</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20297&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20297%26sid%3Dliteratum%253Aachs%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DTirado-Rives%26aufirst%3DJ.%26atitle%3DMolecular%2520modeling%2520of%2520organic%2520and%2520biomolecular%2520systems%2520using%2520BOSS%2520and%2520MCPRO%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2005%26volume%3D26%26spage%3D1689%26epage%3D1700%26doi%3D10.1002%2Fjcc.20297" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Robertson, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tirado-Rives, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jorgensen, W. L.</span></span> <span> </span><span class="NLM_article-title">Improved peptide and protein torsional energetics with the OPLS-AA force field</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">3499</span>â <span class="NLM_lpage">3509</span>, <span class="refDoi">Â DOI: 10.1021/acs.jctc.5b00356</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jctc.5b00356" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=1%3ACAS%3A528%3ADC%252BC2MXpsFOlsbc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=3499-3509&author=M.+J.+Robertsonauthor=J.+Tirado-Rivesauthor=W.+L.+Jorgensen&title=Improved+peptide+and+protein+torsional+energetics+with+the+OPLS-AA+force+field&doi=10.1021%2Facs.jctc.5b00356"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Improved Peptide and Protein Torsional Energetics with the OPLS-AA Force Field</span></div><div class="casAuthors">Robertson, Michael J.; Tirado-Rives, Julian; Jorgensen, William L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3499-3509</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The development and validation of new peptide dihedral parameters are reported for the OPLS-AA force field.  High accuracy quantum chem. methods were used to scan Ï, Ï, Ï1, and Ï2 potential energy surfaces for blocked dipeptides.  New Fourier coeffs. for the dihedral angle terms of the OPLS-AA force field were fit to these surfaces, utilizing a Boltzmann-weighted error function and systematically examg. the effects of weighting temp.  To prevent overfitting to the available data, a minimal no. of new residue-specific and peptide-specific torsion terms were developed.  Extensive exptl. soln.-phase and quantum chem. gas-phase benchmarks were used to assess the quality of the new parameters, named OPLS-AA/M, demonstrating significant improvement over previous OPLS-AA force fields.  A Boltzmann weighting temp. of 2000 K was detd. to be optimal for fitting the new Fourier coeffs. for dihedral angle parameters.  Conclusions are drawn from the results for best practices for developing new torsion parameters for protein force fields.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxE-hNg6xKqLVg90H21EOLACvtfcHk0liApOZKCL0blw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXpsFOlsbc%253D&md5=b9836b19acadc1a86369bca744ca9b13</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Facs.jctc.5b00356&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jctc.5b00356%26sid%3Dliteratum%253Aachs%26aulast%3DRobertson%26aufirst%3DM.%2BJ.%26aulast%3DTirado-Rives%26aufirst%3DJ.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26atitle%3DImproved%2520peptide%2520and%2520protein%2520torsional%2520energetics%2520with%2520the%2520OPLS-AA%2520force%2520field%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2015%26volume%3D11%26spage%3D3499%26epage%3D3509%26doi%3D10.1021%2Facs.jctc.5b00356" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jorgensen, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tirado-Rives, J.</span></span> <span> </span><span class="NLM_article-title">Potential energy functions for atomic-level simulations of water and organic and biomolecular systems</span>. <i>Proc. Nat. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>102</i></span>,  <span class="NLM_fpage">6665</span>â <span class="NLM_lpage">6670</span>, <span class="refDoi">Â DOI: 10.1073/pnas.0408037102</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=10.1073%2Fpnas.0408037102" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=15870211" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=1%3ACAS%3A528%3ADC%252BD2MXksVKgt7s%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2005&pages=6665-6670&author=W.+L.+Jorgensenauthor=J.+Tirado-Rives&title=Potential+energy+functions+for+atomic-level+simulations+of+water+and+organic+and+biomolecular+systems&doi=10.1073%2Fpnas.0408037102"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Potential energy functions for atomic-level simulations of water and organic and biomolecular systems</span></div><div class="casAuthors">Jorgensen, William L.; Tirado-Rives, Julian</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">6665-6670</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">A review.  An overview is provided on the development and status of potential energy functions that are used in at.-level statistical mechanics and mol. dynamics simulations of water and of org. and biomol. systems.  Some topics that are considered are the form of force fields, their parameterization and performance, simulations of org. liqs., computation of free energies of hydration, universal extension for org. mols., and choice of at. charges.  The discussion of water models covers some history, performance issues, and special topics such as nuclear quantum effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkn9AFAagzKbVg90H21EOLACvtfcHk0liApOZKCL0blw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXksVKgt7s%253D&md5=633dfc421f1a18cf99ca8cb11d347b3d</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0408037102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0408037102%26sid%3Dliteratum%253Aachs%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DTirado-Rives%26aufirst%3DJ.%26atitle%3DPotential%2520energy%2520functions%2520for%2520atomic-level%2520simulations%2520of%2520water%2520and%2520organic%2520and%2520biomolecular%2520systems%26jtitle%3DProc.%2520Nat.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2005%26volume%3D102%26spage%3D6665%26epage%3D6670%26doi%3D10.1073%2Fpnas.0408037102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lea, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simeonov, A.</span></span> <span> </span><span class="NLM_article-title">Fluorescence polarization assays in small molecule screening</span>. <i>Expert Opin. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">17</span>â <span class="NLM_lpage">32</span>, <span class="refDoi">Â DOI: 10.1517/17460441.2011.537322</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=10.1517%2F17460441.2011.537322" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=22328899" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SrtbvP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=17-32&author=W.+A.+Leaauthor=A.+Simeonov&title=Fluorescence+polarization+assays+in+small+molecule+screening&doi=10.1517%2F17460441.2011.537322"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Fluorescence polarization assays in small molecule screening</span></div><div class="casAuthors">Lea, Wendy A.; Simeonov, Anton</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Discovery</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">17-32</span>CODEN:
                <span class="NLM_cas:coden">EODDBX</span>;
        ISSN:<span class="NLM_cas:issn">1746-0441</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Importance of the field: Fluorescence polarization (FP) is a homogeneous method that allows rapid and quant. anal. of diverse mol. interactions and enzyme activities.  This technique has been widely utilized in clin. and biomedical settings, including the diagnosis of certain diseases and monitoring therapeutic drug levels in body fluids.  Recent developments in the field have been symbolized by the facile adoption of FP in high-throughput screening and small mol. drug discovery of an increasing range of target classes.  Areas covered in this review: The article provides a brief overview of the theor. foundation of FP, followed by updates on recent advancements in its application for various drug target classes, including GPCRs, enzymes and protein-protein interactions.  The strengths and weaknesses of this method, practical considerations in assay design, novel applications and future directions are also discussed.  What the reader will gain: The reader is informed of the most recent advancements and future directions of FP application to small mol. screening.  Take home message: In addn. to its continued utilization in high-throughput screening, FP has expanded into new disease and target areas and has been marked by increased use of labeled small mol. ligands for receptor-binding studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqudLso-a4wR7Vg90H21EOLACvtfcHk0lj04iV47UyAIQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SrtbvP&md5=6f102f40b56fd7a13c4fb59b64a3f883</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1517%2F17460441.2011.537322&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F17460441.2011.537322%26sid%3Dliteratum%253Aachs%26aulast%3DLea%26aufirst%3DW.%2BA.%26aulast%3DSimeonov%26aufirst%3DA.%26atitle%3DFluorescence%2520polarization%2520assays%2520in%2520small%2520molecule%2520screening%26jtitle%3DExpert%2520Opin.%2520Drug%2520Discovery%26date%3D2011%26volume%3D6%26spage%3D17%26epage%3D32%26doi%3D10.1517%2F17460441.2011.537322" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Newton, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deiana, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Puleo, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cisneros, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cutrona, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlessinger, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jorgensen, W. L.</span></span> <span> </span><span class="NLM_article-title">JAK2 JH2 fluorescence polarization assay and crystal structures for complexes with three small molecules</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">614</span>â <span class="NLM_lpage">617</span>, <span class="refDoi">Â DOI: 10.1021/acsmedchemlett.7b00154</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.7b00154" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=1%3ACAS%3A528%3ADC%252BC2sXnvFCnsbY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=614-617&author=A.+S.+Newtonauthor=L.+Deianaauthor=D.+E.+Puleoauthor=J.+A.+Cisnerosauthor=K.+J.+Cutronaauthor=J.+Schlessingerauthor=W.+L.+Jorgensen&title=JAK2+JH2+fluorescence+polarization+assay+and+crystal+structures+for+complexes+with+three+small+molecules&doi=10.1021%2Facsmedchemlett.7b00154"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">JAK2 JH2 Fluorescence Polarization Assay and Crystal Structures for Complexes with Three Small Molecules</span></div><div class="casAuthors">Newton, Ana S.; Deiana, Luca; Puleo, David E.; Cisneros, Jose A.; Cutrona, Kara J.; Schlessinger, Joseph; Jorgensen, William L.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">614-617</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A competitive fluorescence polarization (FP) assay is reported for detg. binding affinities of probe mols. with the pseudokinase JAK2 JH2 allosteric site.  The syntheses of the fluorescent 5 and 6 used in the assay are reported as well as Kd results for 10 compds., including JNJ7706621, NVP-BSK805, and filgotinib (GLPG0634).  X-ray crystal structures of JAK2 JH2 in complex with NVP-BSK805, filgotinib, and diaminopyrimidine 8 elucidate the binding poses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQUy01N09zFrVg90H21EOLACvtfcHk0lj04iV47UyAIQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXnvFCnsbY%253D&md5=e7c6f7c58bc9a9fddf18a72b5df4f964</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.7b00154&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.7b00154%26sid%3Dliteratum%253Aachs%26aulast%3DNewton%26aufirst%3DA.%2BS.%26aulast%3DDeiana%26aufirst%3DL.%26aulast%3DPuleo%26aufirst%3DD.%2BE.%26aulast%3DCisneros%26aufirst%3DJ.%2BA.%26aulast%3DCutrona%26aufirst%3DK.%2BJ.%26aulast%3DSchlessinger%26aufirst%3DJ.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26atitle%3DJAK2%2520JH2%2520fluorescence%2520polarization%2520assay%2520and%2520crystal%2520structures%2520for%2520complexes%2520with%2520three%2520small%2520molecules%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D8%26spage%3D614%26epage%3D617%26doi%3D10.1021%2Facsmedchemlett.7b00154" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McNally, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dekker, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vangrevelinghe, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">ChÃ¨ne, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Machauer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Radimerski, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eck, M. J.</span></span> <span> </span><span class="NLM_article-title">Discovery and structural characterization of ATP-site ligands for the wild-type and V617F-mutant JAK2 pseudokinase domain</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">587</span>â <span class="NLM_lpage">593</span>, <span class="refDoi">Â DOI: 10.1021/acschembio.8b00722</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.8b00722" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjtVSgtro%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2019&pages=587-593&author=R.+McNallyauthor=Q.+Liauthor=K.+Liauthor=C.+Dekkerauthor=E.+Vangrevelingheauthor=M.+Jonesauthor=P.+Ch%C3%A8neauthor=R.+Machauerauthor=T.+Radimerskiauthor=M.+J.+Eck&title=Discovery+and+structural+characterization+of+ATP-site+ligands+for+the+wild-type+and+V617F-mutant+JAK2+pseudokinase+domain&doi=10.1021%2Facschembio.8b00722"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Structural Characterization of ATP-Site Ligands for the Wild-Type and V617F Mutant JAK2 Pseudokinase Domain</span></div><div class="casAuthors">McNally, Randall; Li, Qing; Li, Kunhua; Dekker, Carien; Vangrevelinghe, Eric; Jones, Matthew; Chene, Patrick; Machauer, Rainer; Radimerski, Thomas; Eck, Michael J.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">587-593</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The oncogenic V617F mutation lies in the pseudokinase domain of JAK2, marking it as a potential target for development of compds. that might inhibit the pathogenic activity of the mutant protein.  We used differential scanning fluorometry to identify compds. that bind the JAK2 pseudokinase domain.  Crystal structures of five candidate compds. with the wild-type domain reveal their modes of binding.  Exploration of analogs of screening hit BI-D1870 led to the identification of compd. 2, a 123 nM ligand for the pseudokinase domain.  Interestingly, crystal structures of the V617F domain in complex with two unrelated compds. reveal a conformation that is characteristic of the wild-type domain, rather than that previously obsd. for the V617F mutant.  These structures suggest that certain ATP-site ligands can modulate the V617F allosteric site, thereby providing a mechanistic rationale for targeting the pseudokinase domain and a structural foundation for development of more potent and pseudokinase-selective compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqyQ-36fpAvIrVg90H21EOLACvtfcHk0lj04iV47UyAIQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjtVSgtro%253D&md5=e8664a0fd8d0bcb5eb69252745b39b5a</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Facschembio.8b00722&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.8b00722%26sid%3Dliteratum%253Aachs%26aulast%3DMcNally%26aufirst%3DR.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DLi%26aufirst%3DK.%26aulast%3DDekker%26aufirst%3DC.%26aulast%3DVangrevelinghe%26aufirst%3DE.%26aulast%3DJones%26aufirst%3DM.%26aulast%3DCh%25C3%25A8ne%26aufirst%3DP.%26aulast%3DMachauer%26aufirst%3DR.%26aulast%3DRadimerski%26aufirst%3DT.%26aulast%3DEck%26aufirst%3DM.%2BJ.%26atitle%3DDiscovery%2520and%2520structural%2520characterization%2520of%2520ATP-site%2520ligands%2520for%2520the%2520wild-type%2520and%2520V617F-mutant%2520JAK2%2520pseudokinase%2520domain%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2019%26volume%3D14%26spage%3D587%26epage%3D593%26doi%3D10.1021%2Facschembio.8b00722" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hron, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jursic, B. S.</span></span> <span> </span><span class="NLM_article-title">Preparation of substituted semicarbazides from corresponding amines and hydrazines via phenyl carbamates</span>. <i>Tetrahedron Lett.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">1540</span>â <span class="NLM_lpage">1543</span>, <span class="refDoi">Â DOI: 10.1016/j.tetlet.2014.01.052</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=10.1016%2Fj.tetlet.2014.01.052" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFyisr4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2014&pages=1540-1543&author=R.+Hronauthor=B.+S.+Jursic&title=Preparation+of+substituted+semicarbazides+from+corresponding+amines+and+hydrazines+via+phenyl+carbamates&doi=10.1016%2Fj.tetlet.2014.01.052"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Preparation of substituted semicarbazides from corresponding amines and hydrazines via phenyl carbamates</span></div><div class="casAuthors">Hron, Rebecca; Jursic, Branko S.</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron Letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1540-1543</span>CODEN:
                <span class="NLM_cas:coden">TELEAY</span>;
        ISSN:<span class="NLM_cas:issn">0040-4039</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A simple synthetic procedure for the conversion of amines and hydrazines into substituted semicarbazides I [R1 = H, Me; R2 = H, Ph, 2,4-(O2N)2C6H3, etc.] was developed.  The initial condensation between the desired amine and Ph chloroformate into Ph carbamate is followed by the addn. of hydrazine under basic conditions.  The reaction is tolerable to a variety of functional groups, with mild conditions and high percent yields.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIO9hp-QRnCLVg90H21EOLACvtfcHk0lhkjHM6THRgzA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitFyisr4%253D&md5=36d06f3d0656557576c776c8406aa36a</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.tetlet.2014.01.052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tetlet.2014.01.052%26sid%3Dliteratum%253Aachs%26aulast%3DHron%26aufirst%3DR.%26aulast%3DJursic%26aufirst%3DB.%2BS.%26atitle%3DPreparation%2520of%2520substituted%2520semicarbazides%2520from%2520corresponding%2520amines%2520and%2520hydrazines%2520via%2520phenyl%2520carbamates%26jtitle%3DTetrahedron%2520Lett.%26date%3D2014%26volume%3D55%26spage%3D1540%26epage%3D1543%26doi%3D10.1016%2Fj.tetlet.2014.01.052" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Verrier, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoarau, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsais, F.</span></span> <span> </span><span class="NLM_article-title">Palladium-catalyzed direct (hetero)arylation of ethyl oxazole-4-carboxylate: an efficient access to (hetero)aryloxazoles</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">7383</span>â <span class="NLM_lpage">7386</span>, <span class="refDoi">Â DOI: 10.1021/jo801093n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo801093n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=1%3ACAS%3A528%3ADC%252BD1cXpvVarsLg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2008&pages=7383-7386&author=C.+Verrierauthor=T.+Martinauthor=C.+Hoarauauthor=F.+Marsais&title=Palladium-catalyzed+direct+%28hetero%29arylation+of+ethyl+oxazole-4-carboxylate%3A+an+efficient+access+to+%28hetero%29aryloxazoles&doi=10.1021%2Fjo801093n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Palladium-Catalyzed Direct (Hetero)arylation of Ethyl Oxazole-4-carboxylate: An Efficient Access to (Hetero)aryloxazoles</span></div><div class="casAuthors">Verrier, Cecile; Martin, Thibaut; Hoarau, Christophe; Marsais, Francis</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">7383-7386</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A straightforward route toward 2-(hetero)arylated and 2,5-di(hetero)arylated oxazoles through regiocontrolled palladium-catalyzed direct (hetero)arylation of Et oxazole-4-carboxylate with iodo-, bromo-, and chloro(hetero)aroms. followed by a two-step hydrolysis/decarboxylation sequence was described.  The method was applied here to a neat synthesis of two 2,5-di(hetero)aryloxazole natural products, balsoxin and texaline.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJK_x9guZfj7Vg90H21EOLACvtfcHk0lhkjHM6THRgzA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXpvVarsLg%253D&md5=d34b4dd16ec3449962a90f0649b0866d</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjo801093n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo801093n%26sid%3Dliteratum%253Aachs%26aulast%3DVerrier%26aufirst%3DC.%26aulast%3DMartin%26aufirst%3DT.%26aulast%3DHoarau%26aufirst%3DC.%26aulast%3DMarsais%26aufirst%3DF.%26atitle%3DPalladium-catalyzed%2520direct%2520%2528hetero%2529arylation%2520of%2520ethyl%2520oxazole-4-carboxylate%253A%2520an%2520efficient%2520access%2520to%2520%2528hetero%2529aryloxazoles%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2008%26volume%3D73%26spage%3D7383%26epage%3D7386%26doi%3D10.1021%2Fjo801093n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mattsson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DahlstrÃ¶m, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karlsson, S.</span></span> <span> </span><span class="NLM_article-title">A mild hydrolysis of esters mediated by lithium salts</span>. <i>Tetrahedron Lett.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">2497</span>â <span class="NLM_lpage">2499</span>, <span class="refDoi">Â DOI: 10.1016/j.tetlet.2007.02.029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=10.1016%2Fj.tetlet.2007.02.029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=1%3ACAS%3A528%3ADC%252BD2sXis1Ohs7w%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2007&pages=2497-2499&author=S.+Mattssonauthor=M.+Dahlstr%C3%B6mauthor=S.+Karlsson&title=A+mild+hydrolysis+of+esters+mediated+by+lithium+salts&doi=10.1016%2Fj.tetlet.2007.02.029"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">A mild hydrolysis of esters mediated by lithium salts</span></div><div class="casAuthors">Mattsson, Sara; Dahlstroem, Mikael; Karlsson, Staffan</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron Letters</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">2497-2499</span>CODEN:
                <span class="NLM_cas:coden">TELEAY</span>;
        ISSN:<span class="NLM_cas:issn">0040-4039</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">When treated with amine bases such as triethylamine and various lithium salts in wet solvents, esters are efficiently hydrolyzed to the corresponding acids in good yields.  Esters incorporating an Î±- or Î²-heteroatom with respect to the ester carbonyl group are hydrolyzed rapidly even at room temp.  To further demonstrate the usefulness of this method, one example is provided where hydrolysis of acetylated camphorsultam is mediated by LiBr.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlkdhTIjsx0bVg90H21EOLACvtfcHk0lhkjHM6THRgzA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXis1Ohs7w%253D&md5=7a88787176f93ccdd555a0ab411b0635</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.tetlet.2007.02.029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tetlet.2007.02.029%26sid%3Dliteratum%253Aachs%26aulast%3DMattsson%26aufirst%3DS.%26aulast%3DDahlstr%25C3%25B6m%26aufirst%3DM.%26aulast%3DKarlsson%26aufirst%3DS.%26atitle%3DA%2520mild%2520hydrolysis%2520of%2520esters%2520mediated%2520by%2520lithium%2520salts%26jtitle%3DTetrahedron%2520Lett.%26date%3D2007%26volume%3D48%26spage%3D2497%26epage%3D2499%26doi%3D10.1016%2Fj.tetlet.2007.02.029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">SchÃ¤rfer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulz-Gasch, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ehrlich, H.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guba, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rarey, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stahl, M.</span></span> <span> </span><span class="NLM_article-title">Torsion angle preferences in druglike chemical space: a comprehensive guide</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">2016</span>â <span class="NLM_lpage">2028</span>, <span class="refDoi">Â DOI: 10.1021/jm3016816</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm3016816" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=1%3ACAS%3A280%3ADC%252BC3szltlGitw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=2016-2028&author=C.+Sch%C3%A4rferauthor=T.+Schulz-Gaschauthor=H.-C.+Ehrlichauthor=W.+Gubaauthor=M.+Rareyauthor=M.+Stahl&title=Torsion+angle+preferences+in+druglike+chemical+space%3A+a+comprehensive+guide&doi=10.1021%2Fjm3016816"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Torsion angle preferences in druglike chemical space: a comprehensive guide</span></div><div class="casAuthors">Scharfer Christin; Schulz-Gasch Tanja; Ehrlich Hans-Christian; Guba Wolfgang; Rarey Matthias; Stahl Martin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of medicinal chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2016-28</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Crystal structure databases offer ample opportunities to derive small molecule conformation preferences, but the derived knowledge is not systematically applied in drug discovery research.  We address this gap by a comprehensive and extendable expert system enabling quick assessment of the probability of a given conformation to occur.  It is based on a hierarchical system of torsion patterns that cover a large part of druglike chemical space.  Each torsion pattern has associated frequency histograms generated from CSD and PDB data and, derived from the histograms, traffic-light rules for frequently observed, rare, and highly unlikely torsion ranges.  Structures imported into the corresponding software are annotated according to these rules.  We present the concept behind the tree of torsion patterns, the design of an intuitive user interface for the management and usage of the torsion library, and we illustrate how the system helps analyze and understand conformation properties of substructures widely used in medicinal chemistry.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR2PDX1V9jtEkEgZCMqtRBifW6udTcc2ea9QVUF2k6Cq7ntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3szltlGitw%253D%253D&md5=7d0c2f9177cef90f0d7376836a8cefb2</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fjm3016816&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm3016816%26sid%3Dliteratum%253Aachs%26aulast%3DSch%25C3%25A4rfer%26aufirst%3DC.%26aulast%3DSchulz-Gasch%26aufirst%3DT.%26aulast%3DEhrlich%26aufirst%3DH.-C.%26aulast%3DGuba%26aufirst%3DW.%26aulast%3DRarey%26aufirst%3DM.%26aulast%3DStahl%26aufirst%3DM.%26atitle%3DTorsion%2520angle%2520preferences%2520in%2520druglike%2520chemical%2520space%253A%2520a%2520comprehensive%2520guide%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D2016%26epage%3D2028%26doi%3D10.1021%2Fjm3016816" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arjunan, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santhanam, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rani, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohan, S.</span></span> <span> </span><span class="NLM_article-title">Conformational, vibrational, NMR and DFT studies of N-methylacetanilide</span>. <i>Spectrochim. Acta, Part A</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>104</i></span>,  <span class="NLM_fpage">182</span>â <span class="NLM_lpage">196</span>, <span class="refDoi">Â DOI: 10.1016/j.saa.2012.11.037</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=10.1016%2Fj.saa.2012.11.037" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=23266693" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFeit7o%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2013&pages=182-196&author=V.+Arjunanauthor=R.+Santhanamauthor=T.+Raniauthor=H.+Rosiauthor=S.+Mohan&title=Conformational%2C+vibrational%2C+NMR+and+DFT+studies+of+N-methylacetanilide&doi=10.1016%2Fj.saa.2012.11.037"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational, vibrational, NMR and DFT studies of N-methylacetanilide</span></div><div class="casAuthors">Arjunan, V.; Santhanam, R.; Rani, T.; Rosi, H.; Mohan, S.</div><div class="citationInfo"><span class="NLM_cas:title">Spectrochimica Acta, Part A: Molecular and Biomolecular Spectroscopy</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">182-196</span>CODEN:
                <span class="NLM_cas:coden">SAMCAS</span>;
        ISSN:<span class="NLM_cas:issn">1386-1425</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A detailed conformational, vibrational, NMR and DFT studies of N-methylacetanilide have been carried out.  In DFT, B3LYP method have been used with 6-31G(d,p), 6-311++G(d,p) and cc-pVTZ basis sets.  The vibrational frequencies were calcd. resulting in IR and Raman frequencies together with intensities and Raman depolarisation ratios.  The dipole moment derivs. were computed anal.  Owing to the complexity of the mol., the potential energy distributions of the vibrational modes of the compd. are also calcd.  Isoelectronic mol. electrostatic potential surface (MEP) and electron d. surface were examd. 1H and 13C NMR isotropic chem. shifts were calcd. and the assignments made are compared with the exptl. values.  The energies of important MO's of the compd. were also detd. from TD-DFT method.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2BmuWe6H7U7Vg90H21EOLACvtfcHk0lipUMSHXupBtA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFeit7o%253D&md5=d44916cfff43da61fa4297c1fc043898</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.saa.2012.11.037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.saa.2012.11.037%26sid%3Dliteratum%253Aachs%26aulast%3DArjunan%26aufirst%3DV.%26aulast%3DSanthanam%26aufirst%3DR.%26aulast%3DRani%26aufirst%3DT.%26aulast%3DRosi%26aufirst%3DH.%26aulast%3DMohan%26aufirst%3DS.%26atitle%3DConformational%252C%2520vibrational%252C%2520NMR%2520and%2520DFT%2520studies%2520of%2520N-methylacetanilide%26jtitle%3DSpectrochim.%2520Acta%252C%2520Part%2520A%26date%3D2013%26volume%3D104%26spage%3D182%26epage%3D196%26doi%3D10.1016%2Fj.saa.2012.11.037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Frisch, M. J.</span>; <span class="NLM_string-name">Trucks, G. W.</span>; <span class="NLM_string-name">Schlegel, H. B.</span>; <span class="NLM_string-name">Scuseria, G. E.</span>; <span class="NLM_string-name">Robb, M. A.</span>; <span class="NLM_string-name">Cheeseman, J. R.</span>; <span class="NLM_string-name">Scalmani, G.</span>; <span class="NLM_string-name">Barone, V.</span>; <span class="NLM_string-name">Petersson, G. A.</span>; <span class="NLM_string-name">Nakatsuji, H.</span>; <span class="NLM_string-name">Li, X.</span>; <span class="NLM_string-name">Caricato, M.</span>; <span class="NLM_string-name">Marenich, A. V.</span>; <span class="NLM_string-name">Bloino, J.</span>; <span class="NLM_string-name">Janesko, B. G.</span>; <span class="NLM_string-name">Gomperts, R.</span>; <span class="NLM_string-name">Mennucci, B.</span>; <span class="NLM_string-name">Hratchian, H. P.</span>; <span class="NLM_string-name">Ortiz, J. V.</span>; <span class="NLM_string-name">Izmaylov, A. F.</span>; <span class="NLM_string-name">Sonnenberg, J. L.</span>; <span class="NLM_string-name">Williams-Young, D.</span>; <span class="NLM_string-name">Ding, F.</span>; <span class="NLM_string-name">Lipparini, F.</span>; <span class="NLM_string-name">Egidi, F.</span>; <span class="NLM_string-name">Goings, J.</span>; <span class="NLM_string-name">Peng, B.</span>; <span class="NLM_string-name">Petrone, A.</span>; <span class="NLM_string-name">Henderson, T.</span>; <span class="NLM_string-name">Ranasinghe, D.</span>; <span class="NLM_string-name">Zakrzewski, V. G.</span>; <span class="NLM_string-name">Gao, J.</span>; <span class="NLM_string-name">Rega, N.</span>; <span class="NLM_string-name">Zheng, G.</span>; <span class="NLM_string-name">Liang, W.</span>; <span class="NLM_string-name">Hada, M.</span>; <span class="NLM_string-name">Ehara, M.</span>; <span class="NLM_string-name">Toyota, K.</span>; <span class="NLM_string-name">Fukuda, R.</span>; <span class="NLM_string-name">Hasegawa, J.</span>; <span class="NLM_string-name">Ishida, M.</span>; <span class="NLM_string-name">Nakajima, T.</span>; <span class="NLM_string-name">Honda, Y.</span>; <span class="NLM_string-name">Kitao, O.</span>; <span class="NLM_string-name">Nakai, H.</span>; <span class="NLM_string-name">Vreven, T.</span>; <span class="NLM_string-name">Throssell, K.</span>; <span class="NLM_string-name">Montgomery, J. A.,  Jr.</span>; <span class="NLM_string-name">Peralta, J. E.</span>; <span class="NLM_string-name">Ogliaro, F.</span>; <span class="NLM_string-name">Bearpark, M. J.</span>; <span class="NLM_string-name">Heyd, J. J.</span>; <span class="NLM_string-name">Brothers, E. N.</span>; <span class="NLM_string-name">Kudin, K. N.</span>; <span class="NLM_string-name">Staroverov, V. N.</span>; <span class="NLM_string-name">Keith, T. A.</span>; <span class="NLM_string-name">Kobayashi, R.</span>; <span class="NLM_string-name">Normand, J.</span>; <span class="NLM_string-name">Raghavachari, K.</span>; <span class="NLM_string-name">Rendell, A. P.</span>; <span class="NLM_string-name">Burant, J. C.</span>; <span class="NLM_string-name">Iyengar, S. S.</span>; <span class="NLM_string-name">Tomasi, J.</span>; <span class="NLM_string-name">Cossi, M.</span>; <span class="NLM_string-name">Millam, J. M.</span>; <span class="NLM_string-name">Klene, M.</span>; <span class="NLM_string-name">Adamo, C.</span>; <span class="NLM_string-name">Cammi, R.</span>; <span class="NLM_string-name">Ochterski, J. W.</span>; <span class="NLM_string-name">Martin, R. L.</span>; <span class="NLM_string-name">Morokuma, K.</span>; <span class="NLM_string-name">Farkas, O.</span>; <span class="NLM_string-name">Foresman, J. B.</span>; <span class="NLM_string-name">Fox, D. J.</span></span> <i>Gaussian 16</i>, Revision A.03, <span class="NLM_publisher-name">Gaussian, Inc.</span>: <span class="NLM_publisher-loc">Wallingford CT</span>, <span class="NLM_year">2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=M.+J.+Frisch&author=G.+W.+Trucks&author=H.+B.+Schlegel&author=G.+E.+Scuseria&author=M.+A.+Robb&author=J.+R.+Cheeseman&author=G.+Scalmani&author=V.+Barone&author=G.+A.+Petersson&author=H.+Nakatsuji&author=X.+Li&author=M.+Caricato&author=A.+V.+Marenich&author=J.+Bloino&author=B.+G.+Janesko&author=R.+Gomperts&author=B.+Mennucci&author=H.+P.+Hratchian&author=J.+V.+Ortiz&author=A.+F.+Izmaylov&author=J.+L.+Sonnenberg&author=D.+Williams-Young&author=F.+Ding&author=F.+Lipparini&author=F.+Egidi&author=J.+Goings&author=B.+Peng&author=A.+Petrone&author=T.+Henderson&author=D.+Ranasinghe&author=V.+G.+Zakrzewski&author=J.+Gao&author=N.+Rega&author=G.+Zheng&author=W.+Liang&author=M.+Hada&author=M.+Ehara&author=K.+Toyota&author=R.+Fukuda&author=J.+Hasegawa&author=M.+Ishida&author=T.+Nakajima&author=Y.+Honda&author=O.+Kitao&author=H.+Nakai&author=T.+Vreven&author=K.+Throssell&author=J.+A.+Montgomery&author=J.+E.+Peralta&author=F.+Ogliaro&author=M.+J.+Bearpark&author=J.+J.+Heyd&author=E.+N.+Brothers&author=K.+N.+Kudin&author=V.+N.+Staroverov&author=T.+A.+Keith&author=R.+Kobayashi&author=J.+Normand&author=K.+Raghavachari&author=A.+P.+Rendell&author=J.+C.+Burant&author=S.+S.+Iyengar&author=J.+Tomasi&author=M.+Cossi&author=J.+M.+Millam&author=M.+Klene&author=C.+Adamo&author=R.+Cammi&author=J.+W.+Ochterski&author=R.+L.+Martin&author=K.+Morokuma&author=O.+Farkas&author=J.+B.+Foresman&author=D.+J.+Fox&title=Gaussian+16"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DFrisch%26aufirst%3DM.%2BJ.%26jtitle%3DGaussian%252016%26pub%3DGaussian%252C%2520Inc%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hansch, C.</span>; <span class="NLM_string-name">Leo, A.</span>; <span class="NLM_string-name">Hoekman, D.</span></span> <i>Exploring QSAR â hydrophobic, electronic, and steric constants</i>, <span class="NLM_publisher-name">American Chemical Society</span>: <span class="NLM_publisher-loc">Washington</span>, <span class="NLM_year">1995</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1995&author=C.+Hansch&author=A.+Leo&author=D.+Hoekman&title=Exploring+QSAR+%E2%80%93+hydrophobic%2C+electronic%2C+and+steric+constants"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DHansch%26aufirst%3DC.%26btitle%3DExploring%2520QSAR%2520%25E2%2580%2593%2520hydrophobic%252C%2520electronic%252C%2520and%2520steric%2520constants%26pub%3DAmerican%2520Chemical%2520Society%26date%3D1995" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jerabek-Willemsen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">AndrÃ©, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wanner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duhr, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baaske, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breitsprecher, D.</span></span> <span> </span><span class="NLM_article-title">MicroScale thermophoresis: interaction analysis and beyond</span>. <i>J. Mol. Struct.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>1077</i></span>,  <span class="NLM_fpage">101</span>â <span class="NLM_lpage">113</span>, <span class="refDoi">Â DOI: 10.1016/j.molstruc.2014.03.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=10.1016%2Fj.molstruc.2014.03.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=1%3ACAS%3A528%3ADC%252BC2cXlt1Wisbo%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1077&publication_year=2014&pages=101-113&author=M.+Jerabek-Willemsenauthor=T.+Andr%C3%A9author=R.+Wannerauthor=H.+M.+Rothauthor=S.+Duhrauthor=P.+Baaskeauthor=D.+Breitsprecher&title=MicroScale+thermophoresis%3A+interaction+analysis+and+beyond&doi=10.1016%2Fj.molstruc.2014.03.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">MicroScale Thermophoresis: Interaction analysis and beyond</span></div><div class="casAuthors">Jerabek-Willemsen, Moran; Andre, Timon; Wanner, Randy; Roth, Heide Marie; Duhr, Stefan; Baaske, Philipp; Breitsprecher, Dennis</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Structure</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">1077</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">101-113</span>CODEN:
                <span class="NLM_cas:coden">JMOSB4</span>;
        ISSN:<span class="NLM_cas:issn">0022-2860</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  MicroScale Thermophoresis (MST) is a powerful technique to quantify biomol. interactions.  It is based on thermophoresis, the directed movement of mols. in a temp. gradient, which strongly depends on a variety of mol. properties such as size, charge, hydration shell or conformation.  Thus, this technique is highly sensitive to virtually any change in mol. properties, allowing for a precise quantification of mol. events independent of the size or nature of the studied specimen.  During a MST expt., a temp. gradient is induced by an IR laser.  The directed movement of mols. through the temp. gradient was detected and quantified using either covalently attached or intrinsic fluorophores.  By combining the precision of fluorescence detection with the variability and sensitivity of thermophoresis, MST provides a flexible, robust and fast way to dissect mol. interactions.  In this review, the authors present recent progress and developments in MST technol. and focus on MST applications beyond std. biomol. interaction studies.  By using different model systems, the authors introduce alternative MST applications - such as detn. of binding stoichiometries and binding modes, anal. of protein unfolding, thermodn. and enzyme kinetics.  In addn., we demonstrate the capability of MST to quantify high-affinity interactions with dissocn. consts. (Kd) in the low picomolar (pM) range as well as protein-protein interactions in pure mammalian cell lysates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUijovMuvKs7Vg90H21EOLACvtfcHk0li20eeQg_pNvw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXlt1Wisbo%253D&md5=8a9e1b3c2d10a85550e6d1041c33195e</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.molstruc.2014.03.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molstruc.2014.03.009%26sid%3Dliteratum%253Aachs%26aulast%3DJerabek-Willemsen%26aufirst%3DM.%26aulast%3DAndr%25C3%25A9%26aufirst%3DT.%26aulast%3DWanner%26aufirst%3DR.%26aulast%3DRoth%26aufirst%3DH.%2BM.%26aulast%3DDuhr%26aufirst%3DS.%26aulast%3DBaaske%26aufirst%3DP.%26aulast%3DBreitsprecher%26aufirst%3DD.%26atitle%3DMicroScale%2520thermophoresis%253A%2520interaction%2520analysis%2520and%2520beyond%26jtitle%3DJ.%2520Mol.%2520Struct.%26date%3D2014%26volume%3D1077%26spage%3D101%26epage%3D113%26doi%3D10.1016%2Fj.molstruc.2014.03.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rossi, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, C. W.</span></span> <span> </span><span class="NLM_article-title">Analysis of protein-ligand interactions by fluorescence polarization</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">365</span>â <span class="NLM_lpage">387</span>, <span class="refDoi">Â DOI: 10.1038/nprot.2011.305</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=10.1038%2Fnprot.2011.305" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=21372817" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=1%3ACAS%3A528%3ADC%252BC3MXislentbg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=365-387&author=A.+M.+Rossiauthor=C.+W.+Taylor&title=Analysis+of+protein-ligand+interactions+by+fluorescence+polarization&doi=10.1038%2Fnprot.2011.305"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Analysis of protein-ligand interactions by fluorescence polarization</span></div><div class="casAuthors">Rossi, Ana M.; Taylor, Colin W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">365-387</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Quantification of the assocns. between biomols. is required both to predict and understand the interactions that underpin all biol. activity.  Fluorescence polarization (FP) provides a nondisruptive means of measuring the assocn. of a fluorescent ligand with a larger mol.  We describe an FP assay in which binding of fluorescein-labeled inositol 1,4,5-trisphosphate (IP3) to N-terminal fragments of IP3 receptors can be characterized at different temps. and in competition with other ligands.  The assay allows the std. Gibbs free energy (ÎGÂ°), enthalpy (ÎHÂ°) and entropy (ÎSÂ°) changes of ligand binding to be detd.  The method is applicable to any purified ligand-binding site for which an appropriate fluorescent ligand is available.  FP can be used to measure low-affinity interactions in real time without the use of radioactive materials, it is nondestructive and, with appropriate care, it can resolve ÎHÂ° and ÎSÂ°.  The first part of the protocol, protein prepn., may take several weeks, whereas the FP measurements, once they have been optimized, would normally take 1-6 h.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPmq_JwlOSg7Vg90H21EOLACvtfcHk0li20eeQg_pNvw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXislentbg%253D&md5=b78e3bcd884a06cdeb8b090227963d33</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2011.305&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2011.305%26sid%3Dliteratum%253Aachs%26aulast%3DRossi%26aufirst%3DA.%2BM.%26aulast%3DTaylor%26aufirst%3DC.%2BW.%26atitle%3DAnalysis%2520of%2520protein-ligand%2520interactions%2520by%2520fluorescence%2520polarization%26jtitle%3DNat.%2520Protoc.%26date%3D2011%26volume%3D6%26spage%3D365%26epage%3D387%26doi%3D10.1038%2Fnprot.2011.305" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Otwinowski, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minor, W.</span></span> <span> </span><span class="NLM_article-title">Processing of X-ray diffraction data collected in oscillation mode</span>. <i>Methods Enzymol.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>276</i></span>,  <span class="NLM_fpage">307</span>â <span class="NLM_lpage">326</span>, <span class="refDoi">Â DOI: 10.1016/S0076-6879(97)76066-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=10.1016%2FS0076-6879%2897%2976066-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=27754618" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=1997&pages=307-326&author=Z.+Otwinowskiauthor=W.+Minor&title=Processing+of+X-ray+diffraction+data+collected+in+oscillation+mode&doi=10.1016%2FS0076-6879%2897%2976066-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Processing of x-ray diffraction data collected in oscillation mode</span></div><div class="casAuthors">Otwinowski, Zbyszek; Minor, Wladek</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">276</span>
        (<span class="NLM_cas:issue">Macromolecular Crystallography, Part A</span>),
    <span class="NLM_cas:pages">307-326</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Macromol. crystallog. is an iterative process.  Rarely do the first crystals provide all the necessary data to solve the biol. problem being studied.  Each step benefits from experience learned in previous steps.  To monitor the progress, the HKL package provides 2 tools: (1) statistics, both weighted (Ï2) and unweighted (R-merge), are provided, and the Bayesian reasoning and multicomponent error model facilitates obtaining the proper error ests. and (2) visualization of the process plays a double role by helping the operator to confirm that the process of data redn., including the resulting statistics, is correct, and allowing one to evaluate problems for which there are no good statistical criteria.  Visualization also provides confidence that the point of diminishing returns in data collection and redn. has been reached.  At that point, the effort should be directed to solving the structure.  The methods presented here have been applied to solve a large variety of problems, from inorg. mols. with 5 Ã unit cell to rotavirus of 700 Ã diam. crystd. in 700 Ã 1000 Ã 1400 Ã cell.  Overall quality of the method was tested by many researchers by successful application of the programs to MAD structure detns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrA3fvBOHqhhrVg90H21EOLACvtfcHk0lhfY9US4_lIcg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D&md5=c9536971d4e32cc35352c40fb9368131</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2FS0076-6879%2897%2976066-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0076-6879%252897%252976066-X%26sid%3Dliteratum%253Aachs%26aulast%3DOtwinowski%26aufirst%3DZ.%26aulast%3DMinor%26aufirst%3DW.%26atitle%3DProcessing%2520of%2520X-ray%2520diffraction%2520data%2520collected%2520in%2520oscillation%2520mode%26jtitle%3DMethods%2520Enzymol.%26date%3D1997%26volume%3D276%26spage%3D307%26epage%3D326%26doi%3D10.1016%2FS0076-6879%2897%2976066-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">XDS</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">125</span>â <span class="NLM_lpage">132</span>, <span class="refDoi">Â DOI: 10.1107/S0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&author=W.+Kabsch&title=XDS&doi=10.1107%2FS0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0lhfY9US4_lIcg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXDS%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2FS0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Argetsinger, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robertson, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koleva, R. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cline, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marto, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myers, M. G.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter-Su, C.</span></span> <span> </span><span class="NLM_article-title">Tyrosines 868, 966, and 972 in the kinase domain of JAK2 Are autophosphorylated and required for maximal JAK2 kinase activity</span>. <i>Mol. Endocrinol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">1062</span>â <span class="NLM_lpage">1076</span>, <span class="refDoi">Â DOI: 10.1210/me.2009-0355</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=10.1210%2Fme.2009-0355" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=20304997" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlslyjsr4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2010&pages=1062-1076&author=L.+S.+Argetsingerauthor=J.+A.+Stuckeyauthor=S.+A.+Robertsonauthor=R.+I.+Kolevaauthor=J.+M.+Clineauthor=J.+A.+Martoauthor=M.+G.+Myersauthor=C.+Carter-Su&title=Tyrosines+868%2C+966%2C+and+972+in+the+kinase+domain+of+JAK2+Are+autophosphorylated+and+required+for+maximal+JAK2+kinase+activity&doi=10.1210%2Fme.2009-0355"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Tyrosines 868, 966, and 972 in the kinase domain of JAK2 are autophosphorylated and required for maximal JAK2 kinase activity</span></div><div class="casAuthors">Argetsinger, Lawrence S.; Stuckey, Jeanne A.; Robertson, Scott A.; Koleva, Rositsa I.; Cline, Joel M.; Marto, Jarrod A.; Myers, Martin G., Jr.; Carter-Su, Christin</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Endocrinology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1062-1076</span>CODEN:
                <span class="NLM_cas:coden">MOENEN</span>;
        ISSN:<span class="NLM_cas:issn">0888-8809</span>.
    
            (<span class="NLM_cas:orgname">Endocrine Society</span>)
        </div><div class="casAbstract">Janus kinase 2 (JAK2) is activated by a majority of cytokine family receptors including receptors for GH, leptin, and erythropoietin.  To identify novel JAK2-regulatory and/or-binding sites, we set out to identify autophosphorylation sites in the kinase domain of JAK2.  Two-dimensional phosphopeptide mapping of in vitro autophosphorylated JAK2 identified tyrosines 868, 966, and 972 as sites of autophosphorylation.  Phosphorylated tyrosines 868 and 972 were also identified by mass spectrometry anal. of JAK2 activated by an erythropoietin-bound chimeric erythropoietin receptor/leptin receptor.  Phosphospecific antibodies suggest that the phosphorylation of all three tyrosines increases in response to GH.  Compared with wild-type JAK2, which is constitutively active when overexpressed, JAK2 lacking tyrosine 868, 966, or 972 has substantially reduced activity.  Coexpression with GH receptor and protein tyrosine phosphatase 1B allowed us to investigate GH-dependent activation of these mutated JAK2s in human embryonic kidney 293T cells.  All three mutated JAK2s are activated by GH, although to a lesser extent than wild-type JAK2.  The three mutated JAK2s also mediate GH activation of signal transducer and activator of transcription 3 (Stat3), signal transducer and activator of transcription 5b (Stat5b) and ERK1, but at reduced levels.  Coexpression with Src-homol. 2B1Î² (SH2B1Î²), like coexpression with GH-bound GH receptor, partially restores the activity of all three JAK2 mutants.  Based on these results and the crystal structure of the JAK2 kinase domain, we hypothesize that small changes in the conformation of the regions of JAK2 surrounding tyrosines 868, 966, and 972 due to e.g. phosphorylation, binding to a ligand-bound cytokine receptor, and/or binding to Src-homol. 2B1, may be essential for JAK2 to assume a maximally active conformation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNZL2jeP64kbVg90H21EOLACvtfcHk0lhfY9US4_lIcg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlslyjsr4%253D&md5=76b806429b21f6c55893b7699d62b7bf</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1210%2Fme.2009-0355&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1210%252Fme.2009-0355%26sid%3Dliteratum%253Aachs%26aulast%3DArgetsinger%26aufirst%3DL.%2BS.%26aulast%3DStuckey%26aufirst%3DJ.%2BA.%26aulast%3DRobertson%26aufirst%3DS.%2BA.%26aulast%3DKoleva%26aufirst%3DR.%2BI.%26aulast%3DCline%26aufirst%3DJ.%2BM.%26aulast%3DMarto%26aufirst%3DJ.%2BA.%26aulast%3DMyers%26aufirst%3DM.%2BG.%26aulast%3DCarter-Su%26aufirst%3DC.%26atitle%3DTyrosines%2520868%252C%2520966%252C%2520and%2520972%2520in%2520the%2520kinase%2520domain%2520of%2520JAK2%2520Are%2520autophosphorylated%2520and%2520required%2520for%2520maximal%2520JAK2%2520kinase%2520activity%26jtitle%3DMol.%2520Endocrinol.%26date%3D2010%26volume%3D24%26spage%3D1062%26epage%3D1076%26doi%3D10.1210%2Fme.2009-0355" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Prchal-Murphy, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Semper, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lassnig, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wallner, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gausterer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teppner-Klymiuk, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobolak, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MÃ¼ller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolbe, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karaghiosoff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DinnyÃ©s, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">RÃ¼licke, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leitner, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strobl, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MÃ¼ller, M.</span></span> <span> </span><span class="NLM_article-title">TYK2 kinase activity is required for functional type I interferon responses in vivo</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">e39141</span>, <span class="refDoi">Â DOI: 10.1371/journal.pone.0039141</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=10.1371%2Fjournal.pone.0039141" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=22723949" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=1%3ACAS%3A528%3ADC%252BC38Xpt1Wntr8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=e39141&author=M.+Prchal-Murphyauthor=C.+Semperauthor=C.+Lassnigauthor=B.+Wallnerauthor=C.+Gaustererauthor=I.+Teppner-Klymiukauthor=J.+Kobolakauthor=S.+M%C3%BCllerauthor=T.+Kolbeauthor=M.+Karaghiosoffauthor=A.+Dinny%C3%A9sauthor=T.+R%C3%BClickeauthor=N.+R.+Leitnerauthor=B.+Stroblauthor=M.+M%C3%BCller&title=TYK2+kinase+activity+is+required+for+functional+type+I+interferon+responses+in+vivo&doi=10.1371%2Fjournal.pone.0039141"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">TYK2 kinase activity is required for functional type I interferon responses in vivo</span></div><div class="casAuthors">Prchal-Murphy, Michaela; Semper, Christian; Lassnig, Caroline; Wallner, Barbara; Gausterer, Christian; Teppner-Klymiuk, Ingeborg; Kobolak, Julianna; Mueller, Simone; Kolbe, Thomas; Karaghiosoff, Marina; Dinnyes, Andras; Ruelicke, Thomas; Leitner, Nicole R.; Strobl, Birgit; Mueller, Mathias</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">e39141</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Tyrosine kinase 2 (TYK2) is a member of the Janus kinase (JAK) family and is involved in cytokine signalling.  In vitro analyses suggest that TYK2 also has kinase-independent, i.e., non-canonical, functions.  We have generated gene-targeted mice harbouring a mutation in the ATP-binding pocket of the kinase domain.  The Tyk2 kinase-inactive (Tyk2K923E) mice are viable and show no gross abnormalities.  We show that kinase-active TYK2 is required for full-fledged type I interferon-(IFN) induced activation of the transcription factors STAT1-4 and for the in vivo antiviral defense against viruses primarily controlled through type I IFN actions.  In addn., TYK2 kinase activity was found to be required for the protein's stability.  An inhibitory function was only obsd. upon over-expression of TYK2K923E in vitro.  Tyk2K923E mice represent the first model for studying the kinase-independent function of a JAK in vivo and for assessing the consequences of side effects of JAK inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpenRZIOxu3PLVg90H21EOLACvtfcHk0ljLF2ONc41onA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xpt1Wntr8%253D&md5=4a833237168d26e520c7ff53b897276c</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0039141&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0039141%26sid%3Dliteratum%253Aachs%26aulast%3DPrchal-Murphy%26aufirst%3DM.%26aulast%3DSemper%26aufirst%3DC.%26aulast%3DLassnig%26aufirst%3DC.%26aulast%3DWallner%26aufirst%3DB.%26aulast%3DGausterer%26aufirst%3DC.%26aulast%3DTeppner-Klymiuk%26aufirst%3DI.%26aulast%3DKobolak%26aufirst%3DJ.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DKolbe%26aufirst%3DT.%26aulast%3DKaraghiosoff%26aufirst%3DM.%26aulast%3DDinny%25C3%25A9s%26aufirst%3DA.%26aulast%3DR%25C3%25BClicke%26aufirst%3DT.%26aulast%3DLeitner%26aufirst%3DN.%2BR.%26aulast%3DStrobl%26aufirst%3DB.%26aulast%3DM%25C3%25BCller%26aufirst%3DM.%26atitle%3DTYK2%2520kinase%2520activity%2520is%2520required%2520for%2520functional%2520type%2520I%2520interferon%2520responses%2520in%2520vivo%26jtitle%3DPLoS%2520One%26date%3D2012%26volume%3D7%26spage%3De39141%26doi%3D10.1371%2Fjournal.pone.0039141" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rasband, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eliceiri, K. W.</span></span> <span> </span><span class="NLM_article-title">NIH Image to ImageJ: 25 Years of image analysis</span>. <i>Nat. Methods</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">671</span>â <span class="NLM_lpage">675</span>, <span class="refDoi">Â DOI: 10.1038/nmeth.2089</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=10.1038%2Fnmeth.2089" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=22930834" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVKntb7P" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2012&pages=671-675&author=C.+A.+Schneiderauthor=W.+S.+Rasbandauthor=K.+W.+Eliceiri&title=NIH+Image+to+ImageJ%3A+25+Years+of+image+analysis&doi=10.1038%2Fnmeth.2089"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">NIH Image to ImageJ: 25 years of image analysis</span></div><div class="casAuthors">Schneider, Caroline A.; Rasband, Wayne S.; Eliceiri, Kevin W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Methods</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">7_part1</span>),
    <span class="NLM_cas:pages">671-675</span>CODEN:
                <span class="NLM_cas:coden">NMAEA3</span>;
        ISSN:<span class="NLM_cas:issn">1548-7091</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">For the past 25 years NIH Image and ImageJ software have been pioneers as open tools for the anal. of scientific images.  We discuss the origins, challenges and solns. of these two programs, and how their history can serve to advise and inform other software projects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGofzHXrdLwDn7Vg90H21EOLACvtfcHk0ljLF2ONc41onA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVKntb7P&md5=85ab928cd79f1e2f2351c834c0c600f0</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1038%2Fnmeth.2089&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnmeth.2089%26sid%3Dliteratum%253Aachs%26aulast%3DSchneider%26aufirst%3DC.%2BA.%26aulast%3DRasband%26aufirst%3DW.%2BS.%26aulast%3DEliceiri%26aufirst%3DK.%2BW.%26atitle%3DNIH%2520Image%2520to%2520ImageJ%253A%252025%2520Years%2520of%2520image%2520analysis%26jtitle%3DNat.%2520Methods%26date%3D2012%26volume%3D9%26spage%3D671%26epage%3D675%26doi%3D10.1038%2Fnmeth.2089" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zidar, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macut, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">TomaÅ¡iÄ, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brvar, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MontalvÃ£o, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tammela, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solmajer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterlin
MaÅ¡iÄ, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">IlaÅ¡, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kikelj, D.</span></span> <span> </span><span class="NLM_article-title"><i>N</i>-phenyl-4,5-dibromopyrrolamides and <i>N</i>-phenylindolamides as ATP competitive DNA gyrase B inhibitors: design, synthesis, and evaluation</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">6179</span>â <span class="NLM_lpage">6194</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.5b00775</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00775" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVyitLfL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=6179-6194&author=N.+Zidarauthor=H.+Macutauthor=T.+Toma%C5%A1i%C4%8Dauthor=M.+Brvarauthor=S.+Montalv%C3%A3oauthor=P.+Tammelaauthor=T.+Solmajerauthor=L.+Peterlin%0AMa%C5%A1i%C4%8Dauthor=J.+Ila%C5%A1author=D.+Kikelj&title=N-phenyl-4%2C5-dibromopyrrolamides+and+N-phenylindolamides+as+ATP+competitive+DNA+gyrase+B+inhibitors%3A+design%2C+synthesis%2C+and+evaluation&doi=10.1021%2Facs.jmedchem.5b00775"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">N-Phenyl-4,5-dibromopyrrolamides and N-Phenylindolamides as ATP Competitive DNA Gyrase B Inhibitors: Design, Synthesis, and Evaluation</span></div><div class="casAuthors">Zidar, Nace; Macut, Helena; Tomasic, Tihomir; Brvar, Matjaz; Montalvao, Sofia; Tammela, Paivi; Solmajer, Tom; Peterlin Masic, Lucija; Ilas, Janez; Kikelj, Danijel</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6179-6194</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bacterial DNA gyrase is a well-known and validated target in the design of antibacterial drugs.  However, inhibitors of its ATP binding subunit, DNA gyrase B (GyrB), have so far not reached clin. use.  In the present study, three different series of N-phenyl-4,5-dibromopyrrolamides and N-phenylindolamides were designed and prepd. as potential DNA gyrase B inhibitors.  The IC50 values of compds. on DNA gyrase from Escherichia coli were in the low micromolar range, with the best compd., (4-(4,5-dibromo-1H-pyrrole-2-carboxamido)benzoyl)glycine, displaying an IC50 of 450 nM.  For this compd., a high-resoln. crystal structure in complex with E. coli DNA gyrase B was obtained, revealing details of its binding mode within the active site.  The binding affinities of three compds. with GyrB were addnl. evaluated by surface plasmon resonance, and the results were in good agreement with the detd. enzymic activities.  For the most promising compds., the inhibitory activities against DNA gyrase from Staphylococcus aureus and topoisomerases IV from E. coli and S. aureus were detd.  Antibacterial activities of the most potent compds. of each series were evaluated against two Gram-pos. and two Gram-neg. bacterial strains.  The results obtained in this study provide valuable information on the binding mode and structure-activity relationship of N-phenyl-4,5-dibromopyrrolamides and N-phenylindolamides as promising classes of ATP competitive GyrB inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp53wGgDnJdKbVg90H21EOLACvtfcHk0ljLF2ONc41onA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVyitLfL&md5=071aa879adb5bdb2f379a10a7b6aac8a</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00775&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00775%26sid%3Dliteratum%253Aachs%26aulast%3DZidar%26aufirst%3DN.%26aulast%3DMacut%26aufirst%3DH.%26aulast%3DToma%25C5%25A1i%25C4%258D%26aufirst%3DT.%26aulast%3DBrvar%26aufirst%3DM.%26aulast%3DMontalv%25C3%25A3o%26aufirst%3DS.%26aulast%3DTammela%26aufirst%3DP.%26aulast%3DSolmajer%26aufirst%3DT.%26aulast%3DPeterlin%2BMa%25C5%25A1i%25C4%258D%26aufirst%3DL.%26aulast%3DIla%25C5%25A1%26aufirst%3DJ.%26aulast%3DKikelj%26aufirst%3DD.%26atitle%3DN-phenyl-4%252C5-dibromopyrrolamides%2520and%2520N-phenylindolamides%2520as%2520ATP%2520competitive%2520DNA%2520gyrase%2520B%2520inhibitors%253A%2520design%252C%2520synthesis%252C%2520and%2520evaluation%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D6179%26epage%3D6194%26doi%3D10.1021%2Facs.jmedchem.5b00775" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Argade, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhamidipati, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carroll, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clough, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keim, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sylvain, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossi, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coquilla, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Issakani, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masuda, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Payan, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, R.</span></span> <span> </span><span class="NLM_article-title">Application of cultured human mast cells (CHMC) for the design and structure-activity relationship of IgE-mediated mast cell activation inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">2117</span>â <span class="NLM_lpage">2121</span>, <span class="refDoi">Â DOI: 10.1016/j.bmcl.2015.03.075</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=10.1016%2Fj.bmcl.2015.03.075" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=25872982" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlvVems7s%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2015&pages=2117-2121&author=A.+Argadeauthor=S.+Bhamidipatiauthor=H.+Liauthor=D.+Carrollauthor=J.+Cloughauthor=H.+Keimauthor=C.+Sylvainauthor=A.+B.+Rossiauthor=C.+Coquillaauthor=S.+D.+Issakaniauthor=E.+S.+Masudaauthor=D.+G.+Payanauthor=R.+Singh&title=Application+of+cultured+human+mast+cells+%28CHMC%29+for+the+design+and+structure-activity+relationship+of+IgE-mediated+mast+cell+activation+inhibitors&doi=10.1016%2Fj.bmcl.2015.03.075"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Application of cultured human mast cells (CHMC) for the design and structure-activity relationship of IgE-mediated mast cell activation inhibitors</span></div><div class="casAuthors">Argade, Ankush; Bhamidipati, Somasekhar; Li, Hui; Carroll, David; Clough, Jeffrey; Keim, Holger; Sylvain, Catherine; Rossi, Alexander B.; Coquilla, Christina; Issakani, Sarkiz D.; Masuda, Esteban S.; Payan, Donald G.; Singh, Rajinder</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2117-2121</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Here we report the optimization of small mol. inhibitors of human mast cell degranulation via anti-IgE-mediated tryptase release following crosslinking and activation of IgE-loaded FcÎµR1 receptors.  The compds. are selective upstream inhibitors of FcÎµR1-dependent human mast cell degranulation and proved to be devoid of activity in downstream ionomycin mediated degranulation.  Structure-activity relationship (SAR) leading to compd. 26 is outlined.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6TS9r3NDMIbVg90H21EOLACvtfcHk0lgNxUvptW7lqg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlvVems7s%253D&md5=933ebe95e6ef28d7f2bff242b9067783</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2015.03.075&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2015.03.075%26sid%3Dliteratum%253Aachs%26aulast%3DArgade%26aufirst%3DA.%26aulast%3DBhamidipati%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DCarroll%26aufirst%3DD.%26aulast%3DClough%26aufirst%3DJ.%26aulast%3DKeim%26aufirst%3DH.%26aulast%3DSylvain%26aufirst%3DC.%26aulast%3DRossi%26aufirst%3DA.%2BB.%26aulast%3DCoquilla%26aufirst%3DC.%26aulast%3DIssakani%26aufirst%3DS.%2BD.%26aulast%3DMasuda%26aufirst%3DE.%2BS.%26aulast%3DPayan%26aufirst%3DD.%2BG.%26aulast%3DSingh%26aufirst%3DR.%26atitle%3DApplication%2520of%2520cultured%2520human%2520mast%2520cells%2520%2528CHMC%2529%2520for%2520the%2520design%2520and%2520structure-activity%2520relationship%2520of%2520IgE-mediated%2520mast%2520cell%2520activation%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D25%26spage%3D2117%26epage%3D2121%26doi%3D10.1016%2Fj.bmcl.2015.03.075" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabro, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lachance, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcoux-Archambault, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perrier, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DorÃ©, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gros, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masson, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gregoire, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ausseil, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheishvili, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laulan, N. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">St-Pierre, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szyf, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arimondo, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gagnon, A.</span></span> <span> </span><span class="NLM_article-title">Preparation of phenylethylbenzamide derivatives as modulators of DNMT3 activity</span>. <i>Medchemcomm</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">1562</span>â <span class="NLM_lpage">1570</span>, <span class="refDoi">Â DOI: 10.1039/c3md00214d</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=10.1039%2Fc3md00214d" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVWgt7%252FI" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=1562-1570&author=A.+Kabroauthor=H.+Lachanceauthor=I.+Marcoux-Archambaultauthor=V.+Perrierauthor=V.+Dor%C3%A9author=C.+Grosauthor=V.+Massonauthor=J.+M.+Gregoireauthor=F.+Ausseilauthor=D.+Cheishviliauthor=N.+B.+Laulanauthor=Y.+St-Pierreauthor=M.+Szyfauthor=P.+B.+Arimondoauthor=A.+Gagnon&title=Preparation+of+phenylethylbenzamide+derivatives+as+modulators+of+DNMT3+activity&doi=10.1039%2Fc3md00214d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Preparation of phenylethylbenzamide derivatives as modulators of DNMT3 activity</span></div><div class="casAuthors">Kabro, Anzhelika; Lachance, Hugo; Marcoux-Archambault, Iris; Perrier, Valerie; Dore, Vicky; Gros, Christina; Masson, Veronique; Gregoire, Jean-Marc; Ausseil, Frederic; Cheishvili, David; Laulan, Nathalie Bibens; St-Pierre, Yves; Szyf, Moshe; Arimondo, Paola B.; Gagnon, Alexandre</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1562-1570</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">DNA-methyltransferases (DNMTs) are a class of epigenetic enzymes that catalyze the transfer of a Me moiety from the Me donor S-adenosyl-l-methionine onto the C5 position of cytosine in DNA.  This process is dysregulated in cancers and leads to the hypermethylation and silencing of tumor suppressor genes.  The development of potent and selective inhibitors of DNMTs is of utmost importance for the discovery of new therapies for the treatment of cancer.  We report herein the synthesis and DNMT inhibitory activity of 29 analogs derived from NSC 319745.  The effect of selected compds. on the methylation level in the MDA-MB-231 human breast cancer cell line was evaluated using a luminometric methylation assay.  Mol. docking studies have been conducted to propose a binding mode for this series.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHH5XzyIf6O7Vg90H21EOLACvtfcHk0lgNxUvptW7lqg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvVWgt7%252FI&md5=4cb82583964a6293e4804b0ec9333009</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1039%2Fc3md00214d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc3md00214d%26sid%3Dliteratum%253Aachs%26aulast%3DKabro%26aufirst%3DA.%26aulast%3DLachance%26aufirst%3DH.%26aulast%3DMarcoux-Archambault%26aufirst%3DI.%26aulast%3DPerrier%26aufirst%3DV.%26aulast%3DDor%25C3%25A9%26aufirst%3DV.%26aulast%3DGros%26aufirst%3DC.%26aulast%3DMasson%26aufirst%3DV.%26aulast%3DGregoire%26aufirst%3DJ.%2BM.%26aulast%3DAusseil%26aufirst%3DF.%26aulast%3DCheishvili%26aufirst%3DD.%26aulast%3DLaulan%26aufirst%3DN.%2BB.%26aulast%3DSt-Pierre%26aufirst%3DY.%26aulast%3DSzyf%26aufirst%3DM.%26aulast%3DArimondo%26aufirst%3DP.%2BB.%26aulast%3DGagnon%26aufirst%3DA.%26atitle%3DPreparation%2520of%2520phenylethylbenzamide%2520derivatives%2520as%2520modulators%2520of%2520DNMT3%2520activity%26jtitle%3DMedchemcomm%26date%3D2013%26volume%3D4%26spage%3D1562%26epage%3D1570%26doi%3D10.1039%2Fc3md00214d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, J.</span></span> <span> </span><span class="NLM_article-title">I<sub>2</sub> -mediated oxidative C-O bond formation for the synthesis of 1,3,4-oxadiazoles from aldehydes and hydrazides</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>78</i></span>,  <span class="NLM_fpage">10337</span>â <span class="NLM_lpage">10343</span>, <span class="refDoi">Â DOI: 10.1021/jo401751h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo401751h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVyrtLbM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=2013&pages=10337-10343&author=W.+Yuauthor=G.+Huangauthor=Y.+Zhangauthor=H.+Liuauthor=L.+Dongauthor=X.+Yuauthor=Y.+Liauthor=J.+Chang&title=I2+-mediated+oxidative+C-O+bond+formation+for+the+synthesis+of+1%2C3%2C4-oxadiazoles+from+aldehydes+and+hydrazides&doi=10.1021%2Fjo401751h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">I2-Mediated Oxidative C-O Bond Formation for the Synthesis of 1,3,4-Oxadiazoles from Aldehydes and Hydrazides</span></div><div class="casAuthors">Yu, Wenquan; Huang, Gang; Zhang, Yueteng; Liu, Hongxu; Dong, Lihong; Yu, Xuejun; Li, Yujiang; Chang, Junbiao</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">78</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">10337-10343</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A practical and transition-metal-free oxidative cyclization of acylhydrazones into 1,3,4-oxadiazoles has been developed by employing stoichiometric mol. iodine in the presence of potassium carbonate.  The conditions of this cyclization reaction also work well with crude acylhydrazone substrates obtained from the condensation of aldehydes and hydrazides.  A series of sym. and asym. 2,5-disubstituted (aryl, alkyl, and/or vinyl) 1,3,4-oxadiazoles can be conveniently generated in an efficient and scalable fashion.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpntjjbgNlRr7Vg90H21EOLACvtfcHk0lgNxUvptW7lqg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVyrtLbM&md5=903d0cdd2372a21ef27b388ef45ff6ac</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fjo401751h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo401751h%26sid%3Dliteratum%253Aachs%26aulast%3DYu%26aufirst%3DW.%26aulast%3DHuang%26aufirst%3DG.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DDong%26aufirst%3DL.%26aulast%3DYu%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DChang%26aufirst%3DJ.%26atitle%3DI2%2520-mediated%2520oxidative%2520C-O%2520bond%2520formation%2520for%2520the%2520synthesis%2520of%25201%252C3%252C4-oxadiazoles%2520from%2520aldehydes%2520and%2520hydrazides%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2013%26volume%3D78%26spage%3D10337%26epage%3D10343%26doi%3D10.1021%2Fjo401751h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Keith, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Apodaca, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tichenor, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pierce, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seierstad, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palmer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Webb, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karbarz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wennerholm, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rizzolio, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rynberg, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaplan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breitenbucher, J. G.</span></span> <span> </span><span class="NLM_article-title">Aryl piperazinyl ureas as inhibitors of fatty acid amide hydrolase (FAAH) in rat, dog, and primate</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">823</span>â <span class="NLM_lpage">827</span>, <span class="refDoi">Â DOI: 10.1021/ml300186g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml300186g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1eksbrN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2012&pages=823-827&author=J.+M.+Keithauthor=R.+Apodacaauthor=M.+Tichenorauthor=W.+Xiaoauthor=W.+Jonesauthor=J.+Pierceauthor=M.+Seierstadauthor=J.+Palmerauthor=M.+Webbauthor=M.+Karbarzauthor=B.+Scottauthor=S.+Wilsonauthor=L.+Luoauthor=M.+Wennerholmauthor=L.+Changauthor=S.+Brownauthor=M.+Rizzolioauthor=R.+Rynbergauthor=S.+Chaplanauthor=J.+G.+Breitenbucher&title=Aryl+piperazinyl+ureas+as+inhibitors+of+fatty+acid+amide+hydrolase+%28FAAH%29+in+rat%2C+dog%2C+and+primate&doi=10.1021%2Fml300186g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Aryl Piperazinyl Ureas as Inhibitors of Fatty Acid Amide Hydrolase (FAAH) in Rat, Dog, and Primate</span></div><div class="casAuthors">Keith, John M.; Apodaca, Rich; Tichenor, Mark; Xiao, Wei; Jones, William; Pierce, Joan; Seierstad, Mark; Palmer, James; Webb, Michael; Karbarz, Mark; Scott, Brian; Wilson, Sandy; Luo, Lin; Wennerholm, Michelle; Chang, Leon; Brown, Sean; Rizzolio, Michele; Rynberg, Raymond; Chaplan, Sandra; Breitenbucher, J. Guy</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">823-827</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of aryl piperazinyl ureas that act as covalent inhibitors of fatty acid amide hydrolase (FAAH) is described.  A potent and selective (does not inhibit FAAH-2) member of this class, JNJ-40355003 (I), was found to elevate the plasma levels of three fatty acid amides: anandamide, oleoyl ethanolamide, and palmitoyl ethanolamide, in the rat, dog, and cynomolgous monkey.  The elevation of the levels of these lipids in the plasma of monkeys suggests that FAAH-2 may not play a significant role in regulating plasma levels of fatty acid ethanolamides in primates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0G60tuqmh8LVg90H21EOLACvtfcHk0lgtzPyje1s-xA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1eksbrN&md5=fbf5fb59244ac006d87a9b80918ae24e</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fml300186g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml300186g%26sid%3Dliteratum%253Aachs%26aulast%3DKeith%26aufirst%3DJ.%2BM.%26aulast%3DApodaca%26aufirst%3DR.%26aulast%3DTichenor%26aufirst%3DM.%26aulast%3DXiao%26aufirst%3DW.%26aulast%3DJones%26aufirst%3DW.%26aulast%3DPierce%26aufirst%3DJ.%26aulast%3DSeierstad%26aufirst%3DM.%26aulast%3DPalmer%26aufirst%3DJ.%26aulast%3DWebb%26aufirst%3DM.%26aulast%3DKarbarz%26aufirst%3DM.%26aulast%3DScott%26aufirst%3DB.%26aulast%3DWilson%26aufirst%3DS.%26aulast%3DLuo%26aufirst%3DL.%26aulast%3DWennerholm%26aufirst%3DM.%26aulast%3DChang%26aufirst%3DL.%26aulast%3DBrown%26aufirst%3DS.%26aulast%3DRizzolio%26aufirst%3DM.%26aulast%3DRynberg%26aufirst%3DR.%26aulast%3DChaplan%26aufirst%3DS.%26aulast%3DBreitenbucher%26aufirst%3DJ.%2BG.%26atitle%3DAryl%2520piperazinyl%2520ureas%2520as%2520inhibitors%2520of%2520fatty%2520acid%2520amide%2520hydrolase%2520%2528FAAH%2529%2520in%2520rat%252C%2520dog%252C%2520and%2520primate%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D3%26spage%3D823%26epage%3D827%26doi%3D10.1021%2Fml300186g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5USZ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5USZ','PDB','5USZ'); return false;">PDB: 5USZ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5USY" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5USY','PDB','5USY'); return false;">PDB: 5USY</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4FVQ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4FVQ','PDB','4FVQ'); return false;">PDB: 4FVQ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6OBB" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6OBB','PDB','6OBB'); return false;">PDB: 6OBB</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6OBF" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6OBF','PDB','6OBF'); return false;">PDB: 6OBF</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6OCC" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6OCC','PDB','6OCC'); return false;">PDB: 6OCC</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6OAV" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6OAV','PDB','6OAV'); return false;">PDB: 6OAV</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6OBL" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6OBL','PDB','6OBL'); return false;">PDB: 6OBL</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes2"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i84"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00192">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_88868"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c00192?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00192</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline"><sup>1</sup>H and <sup>13</sup>C NMR spectra for final compounds <b>2</b>â<b>15</b>; IR spectrum for compound <b>14</b>; details for the FP and MST assays; crystallographic data for complexes of <b>3</b>, <b>4</b>, <b>9</b>, <b>10</b>, and <b>12</b> with JAK2 JH2, which have been deposited in the RCSB Protein Data Bank with the PDB ID codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6OAV">6OAV</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6OBB">6OBB</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6OBL">6OBL</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6OBF">6OBF</a>, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6OCC">6OCC</a> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00192/suppl_file/jm0c00192_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings for compounds <b>1</b>â<b>15</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00192/suppl_file/jm0c00192_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00192/suppl_file/jm0c00192_si_001.pdf">jm0c00192_si_001.pdf (2.45 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00192/suppl_file/jm0c00192_si_002.csv">jm0c00192_si_002.csv (1.91 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c00192&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c00192%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-10" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c00192" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                6MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright Â© 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">Youâve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-b425991822a79ada59b3.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-fb88834b824b41b135ad.js"></script>

<script type="text/javascript" src="/wro/ke1q~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6679796fed1024b2","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.6.0","si":10}'></script>
</body>
</html>
