
<html lang="en"     class="pb-page"  data-request-id="277f086a-2f1c-4a25-bc9e-c0595be22be2"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";issue:issue:10.1021/jmcmar.2017.60.issue-22;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;article:article:10.1021/acs.jmedchem.7b00487;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="In Silico Absorption, Distribution, Metabolism, Excretion, and Pharmacokinetics (ADME-PK): Utility and Best Practices. An Industry Perspective from the International Consortium for Innovation through Quality in Pharmaceutical Development" /></meta><meta name="dc.Creator" content="Franco  Lombardo" /></meta><meta name="dc.Creator" content="Prashant V.  Desai" /></meta><meta name="dc.Creator" content="Rieko  Arimoto" /></meta><meta name="dc.Creator" content="Kelly E.  Desino" /></meta><meta name="dc.Creator" content="Holger  Fischer" /></meta><meta name="dc.Creator" content="Christopher E.  Keefer" /></meta><meta name="dc.Creator" content="Carl  Petersson" /></meta><meta name="dc.Creator" content="Susanne  Winiwarter" /></meta><meta name="dc.Creator" content="Fabio  Broccatelli" /></meta><meta name="dc.Description" content="In silico tools to investigate absorption, distribution, metabolism, excretion, and pharmacokinetics (ADME-PK) properties of new chemical entities are an integral part of the current industrial dru..." /></meta><meta name="Description" content="In silico tools to investigate absorption, distribution, metabolism, excretion, and pharmacokinetics (ADME-PK) properties of new chemical entities are an integral part of the current industrial dru..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="June 27, 2017" /></meta><meta name="dc.Type" content="review-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.7b00487" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2017 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.7b00487" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.7b00487" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.7b00487" /></link>
        
    
    

<title>In Silico Absorption, Distribution, Metabolism, Excretion, and Pharmacokinetics (ADME-PK): Utility and Best Practices. An Industry Perspective from the International Consortium for Innovation through Quality in Pharmaceutical Development | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.7b00487" /></meta><meta property="og:title" content="In Silico Absorption, Distribution, Metabolism, Excretion, and Pharmacokinetics (ADME-PK): Utility and Best Practices. An Industry Perspective from the International Consortium for Innovation through Quality in Pharmaceutical Development" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/large/jm-2017-004878_0013.jpeg" /></meta><meta property="og:description" content="In silico tools to investigate absorption, distribution, metabolism, excretion, and pharmacokinetics (ADME-PK) properties of new chemical entities are an integral part of the current industrial drug discovery paradigm. While many companies are active in the field, scientists engaged in this area do not necessarily share the same background and have limited resources when seeking guidance on how to initiate and maintain an in silico ADME-PK infrastructure in an industrial setting. This work summarizes the views of a group of industrial in silico and experimental ADME scientists, participating in the In Silico ADME Working Group, a subgroup of the International Consortium for Innovation through Quality in Pharmaceutical Development (IQ) Drug Metabolism Leadership Group. This overview on the benefits, caveats, and impact of in silico ADME-PK should serve as a resource for medicinal chemists, computational chemists, and DMPK scientists working in drug design to increase their knowledge in the area." /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta><meta name="twitter:description" content="In silico tools to investigate absorption, distribution, metabolism, excretion, and pharmacokinetics (ADME-PK) properties of new chemical entities are an integral part of the current industrial drug discovery paradigm. While many companies are active in the field, scientists engaged in this area do not necessarily share the same background and have limited resources when seeking guidance on how to initiate and maintain an in silico ADME-PK infrastructure in an industrial setting. This work summarizes the views of a group of industrial in silico and experimental ADME scientists, participating in the In Silico ADME Working Group, a subgroup of the International Consortium for Innovation through Quality in Pharmaceutical Development (IQ) Drug Metabolism Leadership Group. This overview on the benefits, caveats, and impact of in silico ADME-PK should serve as a resource for medicinal chemists, computational chemists, and DMPK scientists working in drug design to increase their knowledge in the area." /></meta><meta name="twitter:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/large/jm-2017-004878_0013.jpeg" /></meta><meta name="twitter:title" content="In Silico Absorption, Distribution, Metabolism, Excretion, and Pharmacokinetics (ADME-PK): Utility and Best Practices. An Industry Perspective from the International Consortium for Innovation through Quality in Pharmaceutical Development" /></meta>
        
            <meta property="og:type" content="Article" /></meta>
        

        
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.7b00487"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/ke1q~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-6ee29527b3d8ff00a106.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.7b00487">
            
        





































    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-12/achre4.2021.54.issue-12/20210615/achre4.2021.54.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-6/amrcda.2021.2.issue-6/20210625/amrcda.2021.2.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-6/aabmcb.2021.4.issue-6/20210621/aabmcb.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-6/aaembp.2021.3.issue-6/20210622/aaembp.2021.3.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-6/aaemcq.2021.4.issue-6/20210628/aaemcq.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-25/aamick.2021.13.issue-25/20210630/aamick.2021.13.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-6/aanmf6.2021.4.issue-6/20210625/aanmf6.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-6/aapmcd.2021.3.issue-6/20210611/aapmcd.2021.3.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-6/abseba.2021.7.issue-6/20210614/abseba.2021.7.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-12/accacs.2021.11.issue-12/20210618/accacs.2021.11.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-6/acscii.2021.7.issue-6/20210623/acscii.2021.7.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-6/acbcct.2021.16.issue-6/20210618/acbcct.2021.16.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-3/achsc5.2021.28.issue-3/20210524/achsc5.2021.28.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-12/acncdm.2021.12.issue-12/20210616/acncdm.2021.12.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-6/aesccq.2021.5.issue-6/20210617/aesccq.2021.5.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-6/aelccp.2021.6.issue-6/20210611/aelccp.2021.6.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-6/aeecco.2021.1.issue-6/20210611/aeecco.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-6/aewcaa.2021.1.issue-6/20210611/aewcaa.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-5/afsthl.2021.1.issue-5/20210618/afsthl.2021.1.issue-5.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-6/aidcbc.2021.7.issue-6/20210611/aidcbc.2021.7.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-6/amlccd.2021.10.issue-6/20210615/amlccd.2021.10.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-6/amlcef.2021.3.issue-6/20210607/amlcef.2021.3.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-6/amclct.2021.12.issue-6/20210610/amclct.2021.12.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-6/ancac3.2021.15.issue-6/20210622/ancac3.2021.15.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-25/acsodf.2021.6.issue-25/20210629/acsodf.2021.6.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-6/apchd5.2021.8.issue-6/20210616/apchd5.2021.8.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-6/ascefj.2021.6.issue-6/20210625/ascefj.2021.6.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-25/ascecg.2021.9.issue-25/20210628/ascecg.2021.9.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-6/asbcd6.2021.10.issue-6/20210618/asbcd6.2021.10.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-25/ancham.2021.93.issue-25/20210629/ancham.2021.93.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-25/bichaw.2021.60.issue-25/20210629/bichaw.2021.60.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-6/bcches.2021.32.issue-6/20210616/bcches.2021.32.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-6/bomaf6.2021.22.issue-6/20210614/bomaf6.2021.22.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-24/cgeabj.2021.99.issue-24/20210628-01/cgeabj.2021.99.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-6/crtoec.2021.34.issue-6/20210621/crtoec.2021.34.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-12/chreay.2021.121.issue-12/20210623/chreay.2021.121.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-12/cmatex.2021.33.issue-12/20210622/cmatex.2021.33.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-6/cgdefu.2021.21.issue-6/20210602/cgdefu.2021.21.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-12/enfuem.2021.35.issue-12/20210617/enfuem.2021.35.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-12/esthag.2021.55.issue-12/20210615/esthag.2021.55.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-6/estlcu.2021.8.issue-6/20210608/estlcu.2021.8.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-25/iecred.2021.60.issue-25/20210630/iecred.2021.60.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-12/inocaj.2021.60.issue-12/20210621/inocaj.2021.60.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-6/jaaucr.2021.1.issue-6/20210628/jaaucr.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-25/jafcau.2021.69.issue-25/20210630/jafcau.2021.69.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-6/jceaax.2021.66.issue-6/20210610/jceaax.2021.66.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-6/jceda8.2021.98.issue-6/20210608/jceda8.2021.98.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-6/jcisd8.2021.61.issue-6/20210628/jcisd8.2021.61.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-6/jctcce.2021.17.issue-6/20210608/jctcce.2021.17.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/jmcmar.2021.64.issue-12/20210624/jmcmar.2021.64.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-6/jnprdf.2021.84.issue-6/20210625/jnprdf.2021.84.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-12/joceah.2021.86.issue-12/20210618/joceah.2021.86.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-24/jpcafh.2021.125.issue-24/20210624/jpcafh.2021.125.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-24/jpcbfk.2021.125.issue-24/20210624/jpcbfk.2021.125.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-24/jpccck.2021.125.issue-24/20210624/jpccck.2021.125.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-24/jpclcd.2021.12.issue-24/20210624/jpclcd.2021.12.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-6/jprobs.2021.20.issue-6/20210604/jprobs.2021.20.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-25/jacsat.2021.143.issue-25/20210630/jacsat.2021.143.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-6/jamsef.2021.32.issue-6/20210602/jamsef.2021.32.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-25/langd5.2021.37.issue-25/20210629/langd5.2021.37.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-12/mamobx.2021.54.issue-12/20210622/mamobx.2021.54.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-6/mpohbp.2021.18.issue-6/20210607/mpohbp.2021.18.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-12/nalefd.2021.21.issue-12/20210623/nalefd.2021.21.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-12/orlef7.2021.23.issue-12/20210618/orlef7.2021.23.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-6/oprdfk.2021.25.issue-6/20210618/oprdfk.2021.25.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-12/orgnd7.2021.40.issue-12/20210628/orgnd7.2021.40.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.7b00487&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.7b00487&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.7b00487&amp;href=/doi/10.1021/acs.jmedchem.7b00487" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2017</span><span class="cit-fg-volume">, 60</span><span class="cit-fg-issue">, 22</span><span class="cit-fg-pageRange">, 9097-9113</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/60/22" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><span class="content-navigation__contentType">Perspective</span><a href="/doi/10.1021/acs.jmedchem.7b00965" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title"><i>In Silico</i> Absorption, Distribution, Metabolism, Excretion, and Pharmacokinetics (ADME-PK): Utility and Best Practices. An Industry Perspective from the International Consortium for Innovation through Quality in Pharmaceutical Development</span></h1><h2 class="article_header-subtitle"><span class="hlFld-SubTitle">Miniperspective</span></h2><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Franco++Lombardo">Franco Lombardo</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Prashant+V.++Desai">Prashant V. Desai</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Rieko++Arimoto">Rieko Arimoto</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Kelly+E.++Desino">Kelly E. Desino</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Holger++Fischer">Holger Fischer</a></span><span class="author-xref-symbol author-aff-symbol"><sup>⊥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Christopher+E.++Keefer">Christopher E. Keefer</a></span><span class="author-xref-symbol author-aff-symbol"><sup>#</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Carl++Petersson">Carl Petersson</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∇</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Susanne++Winiwarter">Susanne Winiwarter</a></span><span class="author-xref-symbol author-aff-symbol"><sup>○</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Fabio++Broccatelli">Fabio Broccatelli</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>◆</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0002-0724-9752" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">†</span> <span class="aff-text">Alkermes Inc., 852 Winter Street, Waltham, Massachusetts 02451, United States</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">‡</span> <span class="aff-text">Computational ADME, Drug Disposition, Eli Lilly and Company, Indianapolis, Indiana 46285, United States</span></div><div class="aff-info" id="aff3"><span class="aff-symbol">§</span> <span class="aff-text">Vertex Pharmaceuticals Inc., 50 Northern Avenue, Boston, Massachusetts 02210, United States</span></div><div class="aff-info" id="aff4"><span class="aff-symbol">∥</span> <span class="aff-text">AbbVie, Inc., North Chicago, Illinois 60064, United States</span></div><div class="aff-info" id="aff5"><span class="aff-symbol">⊥</span> <span class="aff-text">Roche Pharmaceutical Research and Early Development, Pharmaceutical Sciences, Innovation Center Basel, F. Hoffmann-La Roche Ltd., 4070 Basel, Switzerland</span></div><div class="aff-info" id="aff6"><span class="aff-symbol">#</span> <span class="aff-text">Pfizer Inc., Groton, Connecticut 06340, United States</span></div><div class="aff-info" id="aff7"><span class="aff-symbol">∇</span> <span class="aff-text">Discovery Drug Disposition, Biopharma, R&D Global Early Development, EMD Serono, Frankfurter Strasse 250 I Postcode D39/001, 64293 Darmstadt, Germany</span></div><div class="aff-info" id="aff8"><span class="aff-symbol">○</span> <span class="aff-text">Drug Safety and Metabolism, AstraZeneca R&D Gothenburg, 431 83 Mölndal, Sweden</span></div><div class="aff-info" id="aff9"><span class="aff-symbol">◆</span> <span class="aff-text">Genentech Inc., South San Francisco, California 94080, United States</span></div><div class="corresp-info"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#036562616a6c2d61716c60606277666f6f6a43646e626a6f2d606c6e"><span class="__cf_email__" data-cfemail="e28483808b8dcc80908d81818396878e8e8ba2858f838b8ecc818d8f">[email protected]</span></a>. Phone: +1-650-467-9522.</div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.7b00487&amp;href=/doi/10.1021%2Facs.jmedchem.7b00487" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2017</span></span><span class="cit-volume">, 60</span><span class="cit-issue">, 22</span><span class="cit-pageRange">, 9097–9113</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">June 13, 2017</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>28 March 2017</li><li><span class="item_label"><b>Published</b> online</span>27 June 2017</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 22 November 2017</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.7b00487" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b00487</a></div><div class="article_header-article-copyright"><strong>Copyright © 2017 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D9097%26pageCount%3D17%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DFranco%2BLombardo%252C%2BPrashant%2BV.%2BDesai%252C%2BRieko%2BArimoto%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D60%26issueNum%3D22%26contentID%3Dacs.jmedchem.7b00487%26title%3DIn%2BSilico%2BAbsorption%252C%2BDistribution%252C%2BMetabolism%252C%2BExcretion%252C%2Band%2BPharmacokinetics%2B%2528ADME-PK%2529%253A%2BUtility%2Band%2BBest%2BPractices.%2BAn%2BIndustry%2BPerspective%2Bfrom%2Bthe%2BInternational%2BConsortium%2Bfor%2BInnovation%2Bthrough%2BQuality%2Bin%2BPharmaceutical%2BDevelopment%26numPages%3D17%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D9113%26publicationDate%3DNovember%2B2017">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.7b00487"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">5729</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">48</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.7b00487" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;In Silico Absorption, Distribution, Metabolism, Excretion, and Pharmacokinetics (ADME-PK): Utility and Best Practices. An Industry Perspective from the International Consortium for Innovation through Quality in Pharmaceutical Development&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Franco&quot;,&quot;last_name&quot;:&quot;Lombardo&quot;},{&quot;first_name&quot;:&quot;Prashant&quot;,&quot;last_name&quot;:&quot;V. Desai&quot;},{&quot;first_name&quot;:&quot;Rieko&quot;,&quot;last_name&quot;:&quot;Arimoto&quot;},{&quot;first_name&quot;:&quot;Kelly&quot;,&quot;last_name&quot;:&quot;E. Desino&quot;},{&quot;first_name&quot;:&quot;Holger&quot;,&quot;last_name&quot;:&quot;Fischer&quot;},{&quot;first_name&quot;:&quot;Christopher&quot;,&quot;last_name&quot;:&quot;E. Keefer&quot;},{&quot;first_name&quot;:&quot;Carl&quot;,&quot;last_name&quot;:&quot;Petersson&quot;},{&quot;first_name&quot;:&quot;Susanne&quot;,&quot;last_name&quot;:&quot;Winiwarter&quot;},{&quot;first_name&quot;:&quot;Fabio&quot;,&quot;last_name&quot;:&quot;Broccatelli&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2017&quot;,&quot;month&quot;:&quot;06&quot;,&quot;day&quot;:&quot;27&quot;,&quot;issue&quot;:&quot;22&quot;,&quot;volume&quot;:&quot;60&quot;,&quot;pages&quot;:&quot;9097-9113&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.7b00487&quot;},&quot;abstract&quot;:&quot;In silico tools to investigate absorption, distribution, metabolism, excretion, and pharmacokinetics (ADME-PK) properties of new chemical entities are an integral part of the current industrial drug discovery paradigm. While many companies are active in the field, scientists engaged in this area do not necessarily share the same background and have limited resources when seeking guidance on how to initiate and maintain an in silico ADME-PK infrastructure in an industrial setting. This work summarizes the views of a group of industrial in silico and experimental ADME scientists, participating in the In Silico ADME Working Group, a subgroup of the International Consortium for Innovation through Quality in Pharmaceutical Development (IQ) Drug Metabolism Leadership Group. This overview on the benefits, caveats, and impact of in silico ADME-PK should serve as a resource for medicinal chemists, computational chemists, and DMPK scientists working in drug design to increase their knowledge in the area.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b00487&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b00487" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b00487&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b00487" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b00487&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b00487" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.7b00487&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b00487&amp;href=/doi/10.1021/acs.jmedchem.7b00487" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.7b00487" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.7b00487" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (4 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b00487&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.7b00487%26sid%3Dliteratum%253Aachs%26pmid%3D28609624%26genre%3Darticle%26aulast%3DLombardo%26date%3D2017%26atitle%3DIn%2BSilico%2BAbsorption%252C%2BDistribution%252C%2BMetabolism%252C%2BExcretion%252C%2Band%2BPharmacokinetics%2B%2528ADME-PK%2529%253A%2BUtility%2Band%2BBest%2BPractices.%2BAn%2BIndustry%2BPerspective%2Bfrom%2Bthe%2BInternational%2BConsortium%2Bfor%2BInnovation%2Bthrough%2BQuality%2Bin%2BPharmaceutical%2BDevelopment%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D60%26issue%3D22%26spage%3D9097%26epage%3D9113%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=292610" title="Antibiotic resistance">Antibiotic resistance</a>,</li><li><a href="/action/doSearch?ConceptID=292031" title="Computational modeling">Computational modeling</a>,</li><li><a href="/action/doSearch?ConceptID=291823" title="Quality management">Quality management</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/60/22" title="Go to Volume 60, Issue 22"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/jmcmar.2017.60.issue-22/20171122/jmcmar.2017.60.issue-22.largecover.jpg" alt="Go to Volume 60, Issue 22"/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/medium/jm-2017-004878_0013.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/large/jm-2017-004878_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00487&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText"><i>In silico</i> tools to investigate absorption, distribution, metabolism, excretion, and pharmacokinetics (ADME-PK) properties of new chemical entities are an integral part of the current industrial drug discovery paradigm. While many companies are active in the field, scientists engaged in this area do not necessarily share the same background and have limited resources when seeking guidance on how to initiate and maintain an <i>in silico</i> ADME-PK infrastructure in an industrial setting. This work summarizes the views of a group of industrial <i>in silico</i> and experimental ADME scientists, participating in the <i>In Silico</i> ADME Working Group, a subgroup of the International Consortium for Innovation through Quality in Pharmaceutical Development (IQ) Drug Metabolism Leadership Group. This overview on the benefits, caveats, and impact of <i>in silico</i> ADME-PK should serve as a resource for medicinal chemists, computational chemists, and DMPK scientists working in drug design to increase their knowledge in the area.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">1 Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_06628" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_06628" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p"><i>In silico</i> ADME-PK (absorption, distribution, metabolism, excretion, and pharmacokinetics) is the use of computer modeling to understand structure–property relationships and to predict DMPK (drug metabolism and pharmacokinetics) properties from compound structure. This is related but distinct from physiologically based pharmacokinetic (PBPK) modeling, which strives to provide accurate predictions of the PK profile of drug candidates. The focus of <i>in silico</i> ADME-PK is to guide the design of novel compounds with superior ADME properties. Most often a QSPR (quantitative structure–property relationship) approach is used to relate a compound’s structure to the property in question (e.g., cell permeability or metabolic clearance) measured in an <i>in vitro</i> assay. As the acronym ADME-PK implies, <i>in vivo</i> properties can also be the target of such efforts.</div><div class="NLM_p">Appropriately applied during the discovery phase, <i>in silico</i> ADME-PK has the potential to enhance the probability of identifying compounds with favorable ADME properties. Importantly, these <i>in silico</i> tools are not always intended to be in lieu of experimental measurements but are intricately connected with the preclinical assay they attempt to predict and are ideally applied by chemists and DMPK scientists alike, presynthetically to guide the next round of synthesis. The intended result of such effort is to accelerate the drug optimization process by reducing the number of cycles of synthesis necessary to identify a quality drug candidate. Additionally <i>in silico</i> ADME tools can be used to prioritize <i>in vitro</i> testing after synthesis.</div><div class="NLM_p">Computational ADME modeling is quite a mature but still evolving field.<a onclick="showRef(event, 'ref1 ref2 ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3 ref4">(1-4)</a><i>In silico</i> ADME tools are routinely applied to drug design. However, scientists engaged in this arena often witness how the use of the <i>in silico</i> ADME tools is largely dependent on the perception of the individuals within the project teams. This skepticism is often the result of lack of clarity on the scope and limitations of the model and of the assay being modeled. For example, one cannot expect a reliable quantitative model of thermodynamic aqueous solubility when we are not aware of the solid form of the material used in generating the solubility data.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a></div><div class="NLM_p">Models, by definition, will provide numbers, and numbers can be misinterpreted if used out of context. This is also true of experimental values, which have associated variability and experimental error and need to be interpreted with caution, especially for some high-throughput screens. In a recent perspective article, Kuhn and coauthors<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> highlight the value of qualitative assessments in their closing notes. These authors invite us to not use absolute values but rather consider a more qualitative approach within the context of the project. They also suggest to “stay close to the experiments”. This should be interpreted, in our view, as a coupling of the use of computed physicochemical, <i>in vitro</i>, and <i>in vivo</i> readouts with a solid understanding of the experimental assay and its limitations. For example, cLogP can easily be calculated by one of several commercially available software packages. The validity of this calculated property for a given chemotype of interest should be validated by submitting a sufficient number of compounds for experimental log <i>P</i>/log <i>D</i> determinations. In other words, “trust but check” to ensure the models are applicable to the structural space of interest; this is a critical consideration that will be further discussed later.</div><div class="NLM_p last">This review discusses key lessons learned by scientists actively working in the <i>in silico</i> ADME-PK field within drug discovery. The present impact will be discussed as well as the importance of paying attention to the purpose and quality of the experimental assay being modeled and the resulting implications in model building, evaluation, and use. The review further suggests individual components necessary to implement a successful <i>in silico</i> ADME-PK modeling culture and infrastructure in industry. Complementary discussions focusing on the technical aspects of building an ADME QSAR infrastructure can be found elsewhere.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> While the work described focuses mainly on QSPR modeling, many of the learning points are transferrable to other technologies such as molecular matched pairs (MMPs) and chemical data mining, which will be discussed briefly before the conclusion of the manuscript. The technologies discussed in this work require availability of hundreds of consistently measured data points for each property target of the modeling effort; this could be a practical limitation for scientists working in small organizations approaching this field.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">2 Impact of in Silico ADME-PK Modeling</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_80220" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_80220" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The aim of <i>in silico</i> ADME-PK modeling is to improve the design of candidate drugs. Experimental end points that are most commonly targeted include metabolic stability, permeability, solubility, CYP inhibition, and transporter-mediated efflux. For these properties most members of our group reported using a “global” model, i.e., including all the available internal data in the training set, as opposed to a “local” model focusing on a specific chemotype. While good models are needed, the mere existence of high quality models does not guarantee success. Models can only be truly impactful if the organization embraces and fosters an <i>in silico</i> culture; this requires a diversified approach involving <i>in silico</i> scientists, DMPK scientists (<i>in vitro</i> and <i>in vivo</i>), chemists, and to some extent management. <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a> reports real life examples where models influenced the quality of compound design organization-wide.<a onclick="showRef(event, 'ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref8 ref9">(8, 9)</a> An <i>in silico</i> solubility model at AstraZeneca and an <i>in silico</i> microsomal stability model at Genentech had been available for years. Nevertheless, true impact could only be seen once usage was enforced through departmental goals in 2004 at AstraZeneca and in 2010 at Genentech. This can be done by creating the expectation that projects facing a ADME challenges should factor <i>in silico</i> ADME models into decision making or provide explanations if they do not. At AstraZeneca this resulted in a 7-fold increase in the number of compounds with good solubility, whereas at Genentech the number of stable compounds increased by 2-fold. The process was slightly different for the third example in <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>: a high-throughput assay for assessing time-dependent inhibition (TDI) of CYP3A4 was made available at Eli Lilly in mid-2009. After proactive enrichment of the TDI data to capture a wide range of structural diversity, a global <i>in silico</i> model was introduced in 2011. Subsequently the ratio of compounds with low vs high potential for CYP3A4 TDI improved by 3-fold (1.4 before 2011 and 4.3 after 2011).</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/medium/jm-2017-004878_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/large/jm-2017-004878_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Changes observed across all projects in microsomal stability at Genentech (GNE), solubility at AstraZeneca (AZ), and CYP3A4 time-dependent inhibition at Eli Lilly (Lilly) after adoption of <i>in silico</i> models.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/large/jm-2017-004878_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00487&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">While high quality of a model is an absolute prerequisite and management may help a model gain momentum, an equally important aspect is continued education and support promoted by a core team composed of the following:<ul class="NLM_list-list_type-bullet"><li><p class="inline"><i>In silico</i> ADME-PK experts with the capability to develop robust models (usually computational chemists or cheminformaticians with a keen interest in understanding ADME properties)</p></li><li><p class="inline"><i>In silico</i> and <i>in vitro</i> (or <i>in vivo</i>) experts monitoring model performance over time as well as experimental data quality</p></li><li><p class="inline">IT and infrastructure experts ensuring model access and ease of use</p></li><li><p class="inline">Early adopters (or “champions”) of the models positioned within the project team that can lead by example</p></li><li><p class="inline">Educators creating awareness by presenting in different forums</p></li><li><p class="inline">Management encouragement of the use of the models</p></li></ul></div><div class="NLM_p">Furthermore, usage and thus the impact of a model depend on the quality of the software infrastructure as well as the ability to promote the model’s interplay and synergy with available <i>in vitro</i> tools. The latter point is very important and is tied to understanding decision drivers during design. In companies adopting a tiered screening strategy, where only compounds that are stable in microsomes (tier 1) will be tested in hepatocytes (tier 2), a good <i>in silico</i> filter for bad actors in the liver microsomes assay can effectively act as a tier 0 assay. In contrast, a model for stability in hepatocytes may be less useful as guidance for the tier 2 assay, since <i>in vitro</i> metabolic stability would already be available during candidate selection for the hepatocytes stability assay. The opposite could be true in a different paradigm where metabolic stability in hepatocytes is used as a tier 1 assay, for example, in a project where phase II metabolism is recognized as important for the metabolic clearance. In essence, the success of the <i>in silico</i> tools is inherently linked to the local company environment.</div><div class="NLM_p">Scientists at AstraZeneca reported that their <i>in silico</i> ADME-Tox modeling infrastructure was utilized to predict at least one property during the design stage for ∼55% of the compounds synthesized between 2011 and 2012.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> While this does not inform on the quality of the predictions, it is a clear indication of their importance in decision-making. Another example of the impact of <i>in silico</i> ADME models was described by scientists at Eli Lilly and Company, based on an integrated approach involving an iterative learning cycle through <i>in silico</i>–<i>in vitro</i>–<i>in vivo</i> assessments.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a></div><div class="NLM_p">In an effort to provide an estimate of the impact of <i>in silico</i> ADME-PK modeling in the industry today, 11 members of the <i>In Silico</i> ADME IQ Working Group answered a survey (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>). The scientists who responded to this survey are industry experts working in some of the major pharmaceutical companies in U.S., Europe, and Asia. The findings of the survey are summarized as follows:<ul class="NLM_list-list_type-bullet"><li><p class="inline"><i>In silico</i> ADME is perceived as an impactful reality in industry; yet, approximately 3 out of 4 of the members think that there is still room for growth.</p></li><li><p class="inline">QSPR predictors are the most established tools. Complementary approaches focused on interpretation and idea generation, such as MMPs and data trend analysis, are also widely adopted.</p></li><li><p class="inline">Structure-based modeling of ADME targets (e.g., CYPs, P-gp, PXR) can be useful but should be considered as a tool to study specific cases.</p></li></ul></div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/medium/jm-2017-004878_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/large/jm-2017-004878_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Results of the survey among industrial scientists involved in <i>in silico</i> ADME.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/large/jm-2017-004878_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00487&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">It is important to point out that <i>in silico</i> predictions of metabolic soft spots and computer aided metabolite identification may be regarded as components of the <i>in silico</i> ADME toolbox but will not be reviewed here. This is an extremely active and impactful area at the interface between <i>in silico</i> and <i>in vitro</i> science but is beyond the scope of this work.</div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">3 Model Building Considerations</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_51268" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_51268" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The necessary starting point for all good and useful models is an in-depth understanding of the property that is being modeled, how that property factors into decision making, and how it is related to the <i>in vivo</i> situation. In addition, model usage often depends on how the experimental data are used, which will also influence the approach. A categorical filter for bad actors will not necessarily need to capture 80% of the experimental data within 2-fold from the prediction, while a continuous model with increased accuracy may be preferred for uses such as establishing an <i>in silico</i> to <i>in vitro</i> correlation around clearance.</div><div class="NLM_p">Some experimental end points lend themselves to being easily modeled as in the case of fraction unbound in human plasma (h-fu<sub>p</sub>) (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>). This parameter is experimentally highly reproducible, leading to well-defined data and thus facilitating robust <i>in silico</i> models. However, protein binding is not a property that warrants optimization<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> but requires an accurate value since it is a cornerstone for understanding pharmacokinetic–pharmacodynamic (PK/PD) relationships. Thus, the impact of such a model will depend very much on whether a confidence can be assigned to any predicted value; an overall high <i>R</i><sup>2</sup> will not be sufficient. Estimating confidence in prediction is not a trivial task; this will be addressed later on in this work.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Summary of <i>in Vitro</i> Variability and <i>in Silico</i> Error in Prediction Based on Genentech Data</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">property</th><th class="colsep0 rowsep0" align="center"><i>R</i><sup>2</sup></th><th class="colsep0 rowsep0" align="center">rmse</th><th class="colsep0 rowsep0" align="center">within 2-fold</th><th class="colsep0 rowsep0" align="center">count</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">h-fu<sub>p</sub>, exptl<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">0.92</td><td class="colsep0 rowsep0" align="left">0.05</td><td class="colsep0 rowsep0" align="left">0.91</td><td class="colsep0 rowsep0" align="left">1240</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">h-fu<sub>p</sub>, calcd<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="left">0.64</td><td class="colsep0 rowsep0" align="left">0.08</td><td class="colsep0 rowsep0" align="left">0.52</td><td class="colsep0 rowsep0" align="left">239</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">efflux ratio<a class="ref internalNav" href="#t1fn3" aria-label="c">c</a> in MDCK-MDR1, exptl<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">0.59</td><td class="colsep0 rowsep0" align="left">17</td><td class="colsep0 rowsep0" align="left">0.77</td><td class="colsep0 rowsep0" align="left">677</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">efflux ratio<a class="ref internalNav" href="#t1fn3" aria-label="c">c</a> in MDCK-MDR1, calcd<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="left">0.22</td><td class="colsep0 rowsep0" align="left">26</td><td class="colsep0 rowsep0" align="left">0.46</td><td class="colsep0 rowsep0" align="left">138</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup>a</sup><p class="last">Based on all the experimental replicates (<i>N</i> = 2) available for the same compound in Genentech’s database, the first measurement (as defined by assay date) is used to predict the second measurement.</p></div><div class="footnote" id="t1fn2"><sup>b</sup><p class="last">Based on prospective validation for Genentech calculated properties since latest model update.</p></div><div class="footnote" id="t1fn3"><sup>c</sup><p class="last">Efflux ratio category is defined adopting a project specific threshold value that is usually between 2 and 10; the positive predictive value for the calculated efflux ratio ranges from 84% (threshold = 2) and 95% (threshold = 10).</p></div></div></div><div class="NLM_p">In contrast, efflux ratio in transfected cell lines, typically associated with quite high experimental variability, can lead to very impactful models despite a relatively poor <i>R</i><sup>2</sup>, when the categorical outcome of the assay is predicted correctly. Madin–Darby canine kidney epithelial cells transfected with MDR1 (MDCK-MDR1) efflux ratio models are of high interest in industry, since it was shown that the experimental value is a good predictor for poor <i>in vivo</i> brain penetration.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> A project targeting the CNS needs to understand the efflux risk at the stage of design. <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a> shows the example of an MDCK-MDR1 efflux ratio model used for categorical predictions at Genentech.</div><div class="NLM_p">Another example is presented in <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a> depicting prospective predictions for the AbbVie MDCK-MDR1 efflux ratio model, which can be confidently used as a filter for compounds with high efflux ratios and thus poor chances of permeating the blood–brain barrier. The model is actually based on two different models using orthogonal descriptor types and is thereby able to estimate the prediction confidence for individual compounds; medium to high confidence predictions (blue and purple dots) rarely appear in the top-left quadrant of the plot; that is, false positives are rarely encountered for predictions associated with good confidence.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/medium/jm-2017-004878_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/large/jm-2017-004878_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Prospective predictions for the AbbVie MDCK-MDR1 efflux ratio model. The compounds are color coded by prediction confidence (purple, high; blue, medium; green, low confidence).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/large/jm-2017-004878_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00487&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In another example authors from Eli Lilly have successfully incorporated a categorical model for P-gp efflux to prioritize compounds for design, synthesis, and testing.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> In this case, given the potential limitations in interpreting numerical values of efflux ratio (see <a class="ref internalNav" href="#sec3_1" aria-label="section 3.1">section 3.1</a> for details), the authors chose to build a categorical model wherein the P-gp efflux category determined in a similar assay (substrates or nonsubstrates) was utilized to train the model. The model was applied prospectively to predict P-gp efflux category along with “confidence” scores associated with each prediction. It was demonstrated that the use of confidence scores improved the accuracy of prediction from 73% to 81%.</div><div id="sec3_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5">3.1 Experimental Data</h3><div class="NLM_p">All modeling efforts require the availability of experimental values, usually measured at one or several occasions under specific, mostly defined conditions. Such data show variability and inherent errors. For example efflux ratio measurements in the MDCK-MDR1 cell line are aimed at understanding whether a compound is substrate of the key efflux transporter P-gp. However, interaction with P-gp may actually be underestimated for compounds with low permeability, which some define as <1 × 10<sup>–6</sup> cm·s<sup>–1</sup> and some define as <5 × 10<sup>–6</sup> cm·s<sup>–1</sup>. In contrast, compounds may be trapped in the cells and their efflux potential underestimated under the static conditions of the assay.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> Understanding these errors, limitations, and data variability is key to understand which data should (or should not) be used for modeling. The popular mantra, “garbage in, garbage out” is a catchy representation of the fact that <i>in silico</i> model can only be as good as the experimental data upon which it is based.</div><div class="NLM_p">In general, one can assume that data from one source only is more likely to be consistent. However, outside industry it may be difficult to obtain data sets from one source that are sufficiently large and diverse. Compilation of multiple literature data sets is a potential approach but is quite time-consuming if done properly. Additionally, one needs to consider that end points known for their experimental reproducibility, such as log <i>P</i>,<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> are more likely to benefit from summarizing literature data than end points with known inherent interlaboratory variability, such as Caco-2 permeability or efflux in transfected MDCK cell lines. Industry data sets, on the other hand, have the advantage of better consistency and higher numbers, and the covered chemical space is usually relevant for the project compounds to be investigated. Nevertheless, such data sets also need to be carefully scrutinized,<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> especially since they are often made up of single determinations, for obvious reasons of speed and cost. Downloading data to train models, without a thorough understanding of the experimental assay and a strong collaboration with the colleagues who have generated the data, should be discouraged.</div><div class="NLM_p">The simplest way to estimate the variability in the experimental data is to use the average standard deviation for the measurements of the set of well-known compounds used during assay validation. Ideally the compounds are rerun at regular intervals and the variability is re-established. An alternative, which additionally enables a continuous check of the variability over time, is to calculate the standard deviations of the control compounds measured during each assay run. Both approaches can be expected to deliver similar results.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> More sophisticated approaches such as minimum discriminant ratios, accounting for standard deviation and number of repeats, can be derived from control compounds and can be used to understand if measurements for two different compounds are truly different.<a onclick="showRef(event, 'ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref15 ref16">(15, 16)</a></div><div class="NLM_p">Also, in industry, data from assays run under slightly different conditions are often combined to expand the data volume. Wenlock and Carlsson<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> discussed which experimental conditions need to be considered when modeling eight DMPK end points and calculated typical standard deviations on the data sets available in the AstraZeneca internal database. Between 350 and 1700 data points were found with at least three repeat measurements for each of the end points, accounting for 5–25% of the total number of compounds. The resulting standard deviations (log scale) fall between 0.11 (human hepatocyte intrinsic clearance, experimental values range from 3 to 300 (μL/min)/10<sup>6</sup> cells) and 0.28 (thermodynamic solubility, measured from solid starting material, values between 0.1 and 1500 μM). Furthermore, the data showed that for some assays (e.g., intrinsic clearance in human liver microsomes, HLM CL<sub>int</sub>) the standard deviations were heteroscedastic,that is, dependent on the actual value. Other <i>in vitro</i> assays such as log <i>D</i> showed a similar standard deviation over the whole value range (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>).</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/medium/jm-2017-004878_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/large/jm-2017-004878_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Scatter and box plots of sample mean and standard deviation for compounds with three or more repeat measurements for log <i>D</i><sub>7.4</sub> and CL<sub>int</sub> in human liver microsomes (HLMs) in the AZ internal database. Box-plots capture the population of the standard deviations values in different discrete intervals (bins).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/large/jm-2017-004878_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00487&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">For <i>in vitro</i> scientists, it may not be surprising that an assay has higher variability close to the assay limit. However, this has important implications in projects where team members may be compelled to push the assays toward these limits. Resulting values may still be of use for a given project but have to be considered with care.<a onclick="showRef(event, 'ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref17 ref18">(17, 18)</a></div><div class="NLM_p last">Wenlock and Carlsson<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> also analyzed how data variability affects the model performance by defining high- and low-quality data sets based on a standard deviation criterion for a compound measured at least twice. The high quality data set was randomly split into a test and training set, whereas all low quality data were used to compile a “low quality training set”. Considering a standard deviation of 0.1, an ideal model could at best result in a root-mean-square error of prediction (RMSEP) of 0.15, taking into account simple error propagation. For a very diverse set of compounds, with data from high-throughput screens and with a great number of compound having <i>N</i> = 1, a standard deviation of 0.11 is generally optimistic and 0.3 (log scale) is a more likely value. Thus, under the optimistic scenario of an ideal model that predicts all data correctly, the RMSEP can be expected to be rather close to 0.42. In reality the models reported by Wenlock and Carlsson achieved RMSEP values between 0.33 and 0.7, indicating that a large fraction of the error in prediction depends on the assay variability. Interestingly, the RMSEP values for the low quality models were not necessarily much higher. This was suggested to be due to the higher number of data points for the low quality data sets, highlighting that <i>in silico</i> ADME modelers need to find the optimal compromise between data quality and data quantity.</div></div><div id="sec3_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6">3.2 Modeling Approach</h3><div class="NLM_p">Modelers own the responsibility of deciding which combination of algorithm and chemical description best fits the purpose. They will have different preferences but will consider the following key parameters: accuracy, speed, interpretability, time of analysis, and risk of overfitting. Most modelers will agree that models need to be accurate in order to be useful. Additionally, usage of a linear model (as opposed to a nonlinear model) may enable easier model interpretation, while the use of an unsupervised learning algorithm could decrease the risk of building a biased model. Concerning the use of descriptors, less is more: if a property can be predicted by a low number of descriptors, these can become guidelines for optimization. Consequently, meaningful and interpretable descriptors are generally preferable.</div><div class="NLM_p">The experience of this working group is that both linear (e.g., principal component analysis, partial least-squares) and nonlinear (e.g., random forest, support vector machine) algorithms can be valuable. Furthermore, it has been observed that for models based on large data sets, as in the case of many <i>in silico</i> ADME models, nonlinear algorithms exploiting a huge number of noninterpretable descriptors often give more accurate predictions. This highlights a fundamental conflict between accuracy and interpretability or risk of model bias. For example, simple physicochemical descriptors such as cLogP and polar surface area (PSA) were useful in multivariate modeling approaches to predict permeability.<a onclick="showRef(event, 'ref19 ref20 ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref19 ref20 ref21 ref22">(19-22)</a> Similarly, most_basic_pKa and PSA were observed to be associated with P-gp efflux categorization.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> There is power in relating these descriptors to a given biological effect, as they are generally ingrained in the minds of most medicinal chemists and pharmaceutical scientists. Other properties that are instead more dependent on direct structural interactions (e.g., binding to proteins) may require descriptors derived from the 3D structure, pharmacophoric information, and/or atomic properties. Ideally, the target of modeling efforts should be to understand how structural modifications affect a property. However, an accurate model able to generate a very rapid prediction (e.g., hERG or microsomal clearance) can be quite helpful to move chemistry into the right direction, even if the model results per se cannot be translated structurally. Depending on the intended use of the tool, both a black box “virtual assay” with superior accuracy and a simple set of rules to guide optimization can be valuable.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> The preference will ultimately be determined by the local company culture and the ability of the modeler to build trust in the tool.</div><div class="NLM_p last">Finally, important aspects of modeling and analyzing data are insights gained that enhance the mechanistic understanding of the property investigated. For example, the well-known Lipinski rule of 5,<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> which pioneered the <i>in silico</i> ADME field, and the solubility forecast index<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> are both routinely used in industry to drive decision making due to their simplicity, despite the lower perceived accuracy compared to QSPR models trained on large high-quality data sets. Simple models are especially appealing because of the resulting clear guidelines for design and, thus, for their potential to educate medicinal chemists. Nevertheless, simple models can only give general trends and may actually be misleading for the individual design problem. Interpretability of more complex models on the compound level has become achievable through, for example, descriptor sensitivity analysis, which enables guidance for specific design questions.</div></div><div id="sec3_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7">3.3 Model Evaluation</h3><div class="NLM_p">Stouch and coauthors discussed the question of why models fail in 2003.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> Among the causes indicated, in the review of case studies of models developed by vendors, scrutiny is placed on the users who may not be sufficiently aware of caveats of the data used in terms of size and type of data set, and relevance to the target properties available in-house.</div><div class="NLM_p">The expectations from the models largely depend on the model developers’ ability to explain, document, and put the model into perspective for the users, which are the colleagues in medicinal chemistry and DMPK. It is not realistic to expect that every user is cross-trained in modeling and statistics and knows the caveats every model will have, so the human factor becomes of utmost importance in not pushing the model beyond the data it was built upon. Our experience is that the time spent in explaining the model performance and in making the user aware of the “domain of applicability” is as important as training and refinement of the models. If the latter activity is something that should be undertaken on a continuous basis, then so should education and guidance to the user.</div><div class="NLM_p">Furthermore, it is important to avoid predictions from an <i>in silico</i> model being directly compared to experimental end points under incorrect mechanistic assumptions. For example, one cannot expect that a compound’s enhanced permeability as a result of an interaction with an uptake transporter can be predicted from a passive permeability model. A case of potential misperception regarding “mispredictions” by a permeability model was presented by Larregieu and Benet<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> who discussed how the outliers in the training set of <i>in silico</i> models should be carefully scrutinized.</div><div class="NLM_p"><i>In silico</i> models are typically evaluated with statistical measures such as <i>R</i><sup>2</sup> and RMSEP for continuous models or accuracy, sensitivity, and specificity for categorical models, based on test sets or via cross-validation of the training set data. The latter should be interpreted with caution to avoid overfitting, and external sets are to be preferred.</div><div class="NLM_p">However, these statistical measures need to be put into the right context to be interpreted correctly.<a onclick="showRef(event, 'ref28 ref29 ref30'); return false;" href="javascript:void(0);" class="ref ref28 ref29 ref30">(28-30)</a> Reports indicate that a truly independent test set, sometimes termed external evaluation set, which is not involved in model development, is essential to evaluate the true predictive power of a QSPR model. In fairly recent work Sheridan and coauthors<a onclick="showRef(event, 'ref31 ref32'); return false;" href="javascript:void(0);" class="ref ref31 ref32">(31, 32)</a> point out that the most widely used cross-validation method is based on random selection, where one randomly extracts data (compounds) from the training set and uses them for internal validation. Additionally, an external validation is often done by selecting a test set from the whole data pool.<a onclick="showRef(event, 'ref33 ref34 ref35 ref36'); return false;" href="javascript:void(0);" class="ref ref33 ref34 ref35 ref36">(33-36)</a> Sheridan points out<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> that the use of the “time-split” method, which is to build a model on data available at a certain time and test the model on data generated later, is the best way to simulate the process of prospective validation. Generally, model training in industry is done with an iterative approach, where the new data are used to test but subsequently also to train the models, in some cases with automated processes.</div><div class="NLM_p">In most cases such an iterative approach, where RMSEP values are determined from the latest new data, i.e., from a temporal test set, gives good confidence in the model. Still, if the end user evaluates model performance based on results for only a small subset of compounds from a given chemical scaffold of interest, the error may differ. At this stage the <i>in silico</i> ADME scientist can work with the project to select additional representative compounds to be tested to increase the validity of the model in the specific chemical space but also to increase confidence in the model. This exercise would result in establishing an <i>in silico</i> to <i>in vitro</i> correlation and could complement analogue efforts to establish <i>in vitro</i> to <i>in vivo</i> correlations resulting in a productive cycle covering all the steps from compound design to <i>in vivo</i> testing.</div><div class="NLM_p">Finally, the examination of outliers can be very revealing and bring out a powerful aspect of modeling. An outlier may highlight an experimental issue worthy of investigating because in reality experimental data may not always be right. When a good working relationship between the modelers and the scientists in the laboratory is established, this can result in a continuous feedback loop by which aberrant data can be highlighted and result in further testing. This helps to improve the quality of the experimental data set and is an additional quality check for the assay laboratories.</div><div class="NLM_p last">The word “synergy” applied to computational and experimental work surfaces in these discussions, and it is recommended that it become a theme, if not yet adopted by scientists in the field. This is as important as, or perhaps even more important than, the choice of a testing scheme for which there is no universal answer. Once again, the models need to be used in context, i.e., for a specific purpose and within the remit of the project team, with the involvement of the computational and cheminformatics scientists, as well as the laboratory scientists who generate the data. If properly used, models will help with asking questions on the experiments, and they will be perhaps even more informative when significant discrepancies may be observed, prompting teams to ask questions.</div></div><div id="sec3_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8">3.4 Applicability Domain and Confidence Metrics</h3><div class="NLM_p last">Confidence and/or applicability domain metrics are crucial elements for chemists and DMPK scientists using the models to understand the utility of a prediction. A confidence metric is usually associated with classification approaches; it addresses the question “What are the odds?” and is generally defined based on the distance from the class cutoff or the degree of agreement among the submodels used in ensemble modeling approaches.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> For regression models this can be provided as a standard error in prediction estimated from the RMSEP. Applicability domain metrics inform on the reliability of the prediction based on the distance from the known chemical space. While different flavors of metrics for prediction confidence and applicability domain have been presented in the past, straightforward approaches (e.g., Tanimoto similarity to training set) may in some cases be considered sufficiently informative and can be easily communicated to teams. More informative algorithms to define applicability domain of QSPR models are based on the sample distance from the variable landscape, which in turn can be estimated using geometrical metrics or probability density distribution functions.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> Depending on the applicability definition, this approach may suffer from shortcomings as the most common modeling techniques transform the variables, whereas the distance from model is calculated via untransformed variables. Hence, it is difficult to ensure that the prediction error is directly related to the distance from the original chemical space. In addition, there is no strict definition of how close to the chemical space an object must be to ensure an accurate prediction. Alternative methods such as the conformal predictor<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> are estimated based on “error models” depending on the conformity with the chemical space occupied by compounds in the training set and the characteristic of the model itself.<a onclick="showRef(event, 'ref39 ref40 ref41'); return false;" href="javascript:void(0);" class="ref ref39 ref40 ref41">(39-41)</a> Once again the ability of the modeler to describe the use and significance of these parameters is of paramount importance to success.</div></div><div id="sec3_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9">3.5 Modeling of <i>in Vivo</i> End Points</h3><div class="NLM_p">The modeling of volume of distribution at steady state (<i>V</i><sub>ss</sub>) and clearance, in human or animal, being targets of different complexity, deserve additional comments. Most authors engaged in this area have focused much more on human than on animal models,<a onclick="showRef(event, 'ref33 ref34 ref36 ref42 ref43'); return false;" href="javascript:void(0);" class="ref ref33 ref34 ref36 ref42 ref43">(33, 34, 36, 42, 43)</a> seemingly seeking a higher return on their effort.</div><div class="NLM_p last">No <i>in silico</i>, <i>in vitro</i>, or <i>in vivo</i> ADME model for predicting human pharmacokinetic properties offers assurance of high accuracy from candidate to candidate. As a corollary, a model that would attempt to predict, from structure only, a full PK profile is too lofty and unattainable at present and for the foreseeable future. Scaling methods based on <i>in vivo</i> data may fare better in most cases, but data analyses and extended modeling efforts<a onclick="showRef(event, 'ref44 ref45 ref46 ref47 ref48 ref49 ref50'); return false;" href="javascript:void(0);" class="ref ref44 ref45 ref46 ref47 ref48 ref49 ref50">(44-50)</a> have shown that, depending on the nature of the compound (basic, neutral, acidic, or zwitterionic) and its properties, different animal scaling approaches will yield different results. A consensus among several approaches, including <i>in silico</i> methods, is probably a safer way to proceed from a risk assessment perspective when predicting properties such as volume of distribution and clearance. This is particularly true for investigators interested in early dose projection during lead optimization (not clinical candidate selection).<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a><i>In vivo</i> testing in higher species requires costly and fairly lengthy cycles that are not generally compatible with rapid design and testing cycles, aside from important considerations related to ethical aspects. For highly permeable compounds <i>V</i><sub>ss</sub> is essentially based on passive diffusion and thus physicochemical determinants such as charge and lipophilicity,<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> which is part of the reason why <i>in silico</i> models (both QSPR and mechanistic) have shown accuracy comparable or superior to predictions based on rodent data.<a onclick="showRef(event, 'ref33 ref52'); return false;" href="javascript:void(0);" class="ref ref33 ref52">(33, 52)</a> However, these models may be less accurate for poorly permeable compounds due to the high importance of active transporters in their disposition. Predicting total clearance in human solely based on molecular structure is a very significant challenge even though several authors have made headway.<a onclick="showRef(event, 'ref34 ref42'); return false;" href="javascript:void(0);" class="ref ref34 ref42">(34, 42)</a></div></div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i10">4 Use of the Models</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_31450" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_31450" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11">4.1 Model Availability: Infrastructural Requirements</h3><div class="NLM_p">The quality of the computational infrastructure is a key feature that will inevitably determine the success of modeling in a company. Quite simply, computational models can be used by nonexperts if they are easy to access, quick to compute (few seconds), and well integrated with other informatics components. Medicinal chemists and DMPK scientists will need to be reminded of calculated DMPK properties through internal presentations but even more importantly through constant exposure. For example, some properties can automatically be calculated for each new compound that is added to the corporate database. Similarly calculations could also be embedded in the design tools, such as platforms for structure-based modeling and repositories for design ideas.<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a></div><div class="NLM_p">The most fertile ground for models to impact design is the microenvironment of a specific drug discovery project. Representatives involved in the design of new compounds, who are aware of past structure–activity relationship (SAR) and current challenges, are the customers and only suitable judges for new design ideas. The focus of scientists designing the computational infrastructure should be on enabling the use of the models during dedicated design meetings and through project workspaces. The latter are dynamic files including all the relevant project data, generally released and analyzed using data mining software such as Spotfire, MedChem Studio, or Vortex.<a onclick="showRef(event, 'ref54 ref55 ref56'); return false;" href="javascript:void(0);" class="ref ref54 ref55 ref56">(54-56)</a> These tools can automatically generate dynamic plots that capture performance of the models for subseries over time and represent a great opportunity to attract chemists’ attention. As an example, <a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a> shows visualization generated by a user-friendly Spotfire-based tool, ADMET_View, implemented at Eli Lilly. The tool is used by project teams to extract, visualize, and analyze <i>in silico</i> and <i>in vitro</i> data for the chemical series of interest. The only information required from the user to generate such a visualization is the list of compound identifiers for a given chemical series and the name of the potency assay for the project. As shown in <a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>a, one of the visualizations provided by this tool helps to evaluate applicability of <i>in silico</i> models for a given series wherein the predicted properties are colored by measured data. For example, in the case of the MDCK permeability category, out of 115 compounds predicted to have “high” permeability 97 were measured to have “high” permeability (colored green) which translates to a positive predictive value of ∼85%. Having assessed the prospective performance of the <i>in silico</i> models for a given chemical series, project teams can then proceed to apply such models for the prioritization of virtual (designed) compounds based on predictions, as shown in <a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>b. Such visualizations also help to assess whether the designed compounds are predicted to offer greater probability of finding compounds with desired properties in a multiparametric approach.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/medium/jm-2017-004878_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/large/jm-2017-004878_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Example of Spotfire visualization to facilitate assessment of applicability of <i>in silico</i> models for a given chemical series and predicted ADME profile of virtual compounds. (a) Predicted vs measured data for three key <i>in vitro</i> ADME properties: rat liver microsomal CL<sub>int,u</sub>, aqueous solubility at pH 6, and MDCK permeability category. For each property, predicted data are presented along the <i>Y</i>-axis and the bars are colored based on the measured data. Number of compounds in each category is labeled for a given bar. (b) Comparison of the above-mentioned key predicted properties for existing compounds and designed (virtual) compounds to facilitate prioritization for synthesis and testing.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/large/jm-2017-004878_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00487&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Ultimately a model will become established only months, or even years, after its release. This is the time required for scientists to familiarize themselves with the new tool. Several members of this group reported that keeping track of all the prospective predictions since the original model release can contribute to success in this area. As an example, Desai et al. reported prospective assessment of successive versions of the P-gp global model to demonstrate a steady performance over several years which resulted in the use of this model as the first screen for P-gp, replacing a high-throughput version of the mechanistic assay.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12">4.2 Use of Calculated DMPK Properties in Drug Discovery Projects</h3><div class="NLM_p">QSPR models can be a great resource for project teams that are in the process of optimizing chemical scaffolds of pharmaceutical interest. <i>In silico</i> DMPK scientists can use their knowledge of the project, of the model, and of the assay to advise team members on how and when the models should be used. Medicinal chemistry is a classical multiparameter optimization process. Yet, this does not imply that at each step of a campaign all the key parameters will concomitantly be optimized. During early stages, the focus is often on understanding potency SAR, but it is not uncommon for concurrent optimization of one or two ADME related issues, particularly if the initial aim of the project is to deliver a tool compound for efficacy or safety proof of concept. In some cases, however, additional tools can help the team temporarily overcome hurdles to establishing proof of concept. Poor solubility could be addressed by enabling formulations, poor permeability could be overcome by means of intraperitoneal dosing, while poor metabolic stability could be improved by predosing pan-CYP inhibitors such as 1-aminobenzotriazole. These ADME related issues may then be revisited at another stage of the program. It is important for modelers to be connected with the project or, at the very least, to understand what DMPK property is actually being optimized at a given moment. Some medicinal chemists may perceive the use of a calculated property as the most cost-effective way to maximize chances of success, while others may perceive it as a limitation to their ability to explore the chemical space of interest. A chemist may in fact argue that the use of <i>in silico</i> tools could result in missed opportunities. While this possibility cannot be excluded, this is a common theme for all models including <i>in vitro</i> and <i>in vivo</i>; for example, using <i>in vitro</i> MDCK permeability to predict intestinal absorption could result in false negatives if uptake transporters are involved. Nevertheless models are needed to support hypothesis driven research, as synthetizing and testing all the possible decorations of a given scaffold not only is practically impossible but would also pose a competitive disadvantage due to time required for completing the lead optimization process. This is of key importance in a landscape where different companies compete for the same target. It is important for <i>in silico</i> ADME scientists to promote models through a nondogmatic approach, since project team members will have different perspectives and past experiences that need to be understood and acknowledged. Claiming that many bad actors could have been avoided after the fact may not be a fruitful way to confront the team. On the contrary the same data points could be an opportunity to build confidence in the tool and to promote its future use. Overall, predictions should be considered as a way to enable informed design decisions, where the term “informed” refers to an objective probabilistic assessment. The use of hard filters and cut-offs in many cases may be in direct contrast with the intention to test hypotheses emerging from <i>in vitro</i> experiments such as metabolite identification and therefore should be disregarded. As the confidence in the <i>in silico</i> tools increases, chemists could be encouraged to synthesize at least a fraction of the new compounds that are predicted to have good DMPK properties, even if SAR and/or structure-based design suggests they may be more risky from a potency standpoint. In fact potency should not be the sole guiding parameter, and we are seeing in more recent times the adoption of multidimensional optimization with dose as the end “deliverable”.<a onclick="showRef(event, 'ref51 ref57 ref58 ref59'); return false;" href="javascript:void(0);" class="ref ref51 ref57 ref58 ref59">(51, 57-59)</a></div><div class="NLM_p">During early stages the priority is generally finding novel promising lead compounds rather than performing a detailed characterization of the space surrounding the front-runner molecules. In this context, models can guide chemistry toward the most promising portion of the accessible chemical space, since during early stages chemists are generally not too worried about losing good compounds due to false negative predictions. When the chemical matter is particularly synthetically tractable, teams might opt for a library enumeration approach, which poses a remarkable opportunity for QSPR DMPK models to influence the selection of the most promising candidates among the thousands that could be synthesized. As teams progress toward later stages, <i>in silico</i> DMPK models become more aware of the scaffold being optimized and therefore more predictive. However, this will not automatically result in increased model usage as the quality requirements for predictions may become much more stringent. Chemists could in fact be rightfully more concerned about losing a good near neighbor than wasting resources in synthesizing a compound with poor properties. Once again in this space there is not a perfect recipe for the use of the ADME models; team members will have to consider a number of problems including selectivity and safety profile, intellectual property coverage, intended modality of administration, and suboptimal DMPK properties.</div><div class="NLM_p">Genentech scientists<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> recently presented real life examples where <i>in silico</i> ADME QSPR models were used to help the team progress toward clinical candidate quality molecules. In each example computational approaches were customized based on the project phase and need and would not have been successful otherwise. It was shown how a JAK-1 project team was initially concerned about establishing potency SAR; therefore models were not extensively used. During later stages the <i>in silico</i> model for metabolic stability was incorporated into the design cycle, resulting in more rapid progression of the chemical series of interest.</div><div class="NLM_p last">Models can encourage synthesis of compounds enriched with favorable ADME properties even if they cannot always explicitly point out a defined molecular feature (group, substituent, dipole moment, etc.) responsible for the improvement. Using such a “black-box” model on many compounds can enable the identification of “clusters” of structures with a higher probability of improving a given ADME property, which will aid prioritization of synthesis. The analysis of these results can provide useful structural hints to the trained eye of a medicinal chemist. The human factor is important here, as it is understandable that once time has been invested in designing a new compound, the person who has conceived the idea might want to pursue that path regardless of the <i>in silico</i> predictions. It is human nature to trust what is “understandable”, and QSPR predictions may not immediately satisfy these requirements.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13">4.3 <i>In Silico</i> ADME: Positioning Relative Organizational Structure</h3><div class="NLM_p last">A further important aspect is where <i>in silico</i> ADME scientists most effectively reside within an organization. This will influence to a large extent the impact of <i>in silico</i> tools on the project. The initial temptation is to answer that these scientists fit within several parts of the organization, which may be true in principle, but that leaves much to be desired. Advice on the positioning of these groups should be a bit more detailed, if only to offer the reader a chance to debate and perhaps disagree. A prescriptive approach could be taken that cheminformatic/computational chemistry groups are ideally located close to, or within, medicinal chemistry departments while the ADME <i>in silico</i> scientists may be located in the DMPK department. We observe that regardless of the location of the <i>in silico</i> ADME expert, it is of key importance that advocates are present in each one of the departments where the customers for the models are located (chemistry, DMPK, and possibly drug safety evaluation).</div></div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i14">5 Knowledge Extraction and Idea Generation Tools</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_86047" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_86047" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Although predictive ADME models significantly guide the design of new molecules, they have the limitation that the structures of the compounds to be predicted have to be preconceived. This is typically done by members of the project team, usually a medicinal or computational chemist. These decisions are usually driven by prior experience, rules of thumb (i.e., decreasing lipophilicity leads to decreased clearance), or other SAR needs for the chemical series.</div><div class="NLM_p">This leads to the question of whether there is an opportunity to use ADME data to assist in the creation of new ideas for the optimization of ADME properties. One approach that has been used recently is the MMP analysis.<a onclick="showRef(event, 'ref60 ref61 ref62 ref63 ref64 ref65'); return false;" href="javascript:void(0);" class="ref ref60 ref61 ref62 ref63 ref64 ref65">(60-65)</a> An MMP is a pair of compounds, typically within a series, where an observed end point difference is attributed to a single structural change. This change can be represented as a chemical transform. If all the pairs that undergo the same chemical transform are summarized, it may be possible to find transforms that lead to an overall improvement in a property. Hence, by utilizing information embedded in existing compounds, the transforms can generate new ideas for improvement of a particular property. MMP analysis is a complementary approach to <i>in silico</i> QSPR modeling in that it is entirely data driven and can generate new hypothetical structures whose properties can be predicted using models to further bolster or refute the hypothesis. <a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a> depicts how MMP idea generation can complement predictive models in compound design.</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/medium/jm-2017-004878_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/large/jm-2017-004878_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Use of MMP data to generate new ideas that can then be predicted with global models to enable informed decision making in design.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/large/jm-2017-004878_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00487&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p"><a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>a shows an example MMP. For a given MMP, <i>all</i> of the changes in biological or physical properties are a result of the structural change. Although there is only one structural change in an MMP, that change can be represented by multiple structural transforms as shown in <a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>b.</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/medium/jm-2017-004878_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/large/jm-2017-004878_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. (a) An example matched molecular pair (MMP). (b) Example transforms for the MMP in (a). R<sub>1</sub> and R<sub>2</sub> refer to any possible chemical group attached to the substructure of interest.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/large/jm-2017-004878_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00487&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">If we group all of the MMPs with a specific transform, we can summarize and analyze the changes in properties to determine if the changes are consistent across different series. <a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a> shows Pfizer HLM CL<sub>int</sub> data for the 5-pyrimidyl → 2-pyrimidyl transform. For this transform, the mean change in log(HLM CL<sub>int</sub>) is −0.29 (50% reduction). 4% of the pairs show an increase in HLM, 45% show a decrease, and 51% show no change (<2-fold). These summaries and analyses can be used to rank-order different transforms to facilitate identification of those that are most likely to help improve a property of interest.</div><figure id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/medium/jm-2017-004878_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/large/jm-2017-004878_0008.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. HLM CL<sub>int</sub> MMP data for the 5-pyrimidyl to 2-pyrimidyl transform. Dashed lines represent ±2-fold change.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/large/jm-2017-004878_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00487&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Although the ability to generate ideas for the improvement of a property is valuable, drug design is often an exercise in the optimization of multiple properties. MMPs can also be useful in this situation by considering more than one end point at a time. For instance, transforms that simultaneously improve clearance, increase permeability, and reduce P-gp liability can be searched for and ranked.</div><div class="NLM_p">Another use for MMP databases is to identify general trends in common fragments that might be unexpected. An example would be the investigation of the changes in properties for all six-membered aromatic rings. <a class="ref internalNav" href="#fig9" aria-label="Figure 9">Figure 9</a> shows a heat map representing the change in HLM clearance, passive permeability, and P-gp efflux for the phenyl → all other six-membered aromatic rings in Pfizer’s database. The data suggest that the best replacement of phenyl for HLM is a 4-pyridazyl, but that replacement may come at the cost of an increase in P-gp efflux.</div><figure id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/medium/jm-2017-004878_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/large/jm-2017-004878_0009.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Mean MMP changes for HLM, passive permeability, and P-gp efflux for phenyl → six-membered aromatic ring transforms. Change values are represented in log<sub>10</sub> units. Colors represent the intensity and the directionality of the change in property (red for increase, blue for decrease).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/large/jm-2017-004878_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00487&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">When using MMP databases and summary information for structural changes, there are a few considerations that one should keep in mind. First, the analysis of transforms makes the assumption that the end point changes are fragment additive, meaning that the change in an end point value will be the same for a given structural change regardless of the rest of the molecule. There are end points where this is largely true (i.e., log <i>D</i>), but this will not always be the case. Another issue is the potential importance of contextualization of a given structural change. This is the local environment of the changing group. For example, the observed change for a transform of a cyano group attached to a phenyl ring may be quite different compared to one attached to an alkyl group. Thus, while contextualization may matter, it is difficult to know a priori in any given situation. Lastly, the analysis of a large set of data via MMP may hide bias due to inclusion of compounds from a single series and may lead to inference that a specific effect to one (or a small number of) series is in fact general. The best advice is to study the underlying MMPs carefully to determine if the overall change is relevant in a given case. Finally, when using MMP transforms, note the statistical significance of the change. Kramer et al. discussed the types of statistical parameters that can be calculated for MMP transforms.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> These measures of significance can be useful for ranking transforms and their likelihood of providing the desired outcome. However, in most MMP databases, there will be a high proportion of transforms that lack the underlying number of MMP examples to be significant. This does not mean that the observed changes of those transforms are not valid, rather that they do not have adequate statistical power. MMP tools and analysis can be very powerful additions to statistical modeling for compound design. From the computational ADME perspective, MMPs enable the move from a passive (model-based) evaluation of ideas to the actual creation of ideas to address a single liability or a combination thereof. These databases can also enable data mining to identify unexpected liabilities in virtual compounds as well as general trends. Future directions in the use of MMPs include transforms based on virtual data<a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(64)</a> and the evolution of matched molecular series (MMS) approaches.<a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(65)</a></div><div class="NLM_p">Alternative methods that borrow elements from both QSPR and MMP technologies, but yet are different, can also be found in the recent literature. <a class="ref internalNav" href="#fig10" aria-label="Figure 10">Figure 10</a> illustrates an implementation where MMP is embedded in R-group table with QSPR model.<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> In this approach, side chains and cores are sequentially fragmented in a way amenable to synthesis and organized into an SAR table. This presentation is intuitive and perhaps the most chemistry-friendly format and therefore could easily fit into the compound-design workflow. All blank cells correspond to virtual compounds generated by the analysis. Predictions for the properties of interest can be made using structural transforms based on MMP database analysis or any QSPR methods suitable for the selected properties. Users can evaluate different end points in parallel to enable a holistic characterization of the profile of the virtual compounds, which will likely be synthetically accessible and therefore aligned with the chemistry plan.</div><figure id="fig10" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/medium/jm-2017-004878_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/large/jm-2017-004878_0010.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. SAR matrix by Gupta-Ostermann and Bajorath,<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a>reproduced from <i>F1000Research</i>. Three series with three compounds each, A–C, D–F, and G–I, are shown with activity (red). Step 1: All compounds are fragmented at a single bond, yielding common core and MMPs within the series. Step 2: Cores are fragmented to obtain core MMPs. The SAR matrix is then generated with rows and columns representing cores and substituents, respectively. Compounds present in the data set are indicated by filled cells.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/large/jm-2017-004878_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00487&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Another approach to stimulate idea generation is to provide a visual representation of the QSPR prediction.<a onclick="showRef(event, 'ref67 ref68'); return false;" href="javascript:void(0);" class="ref ref67 ref68">(67, 68)</a><a class="ref internalNav" href="#fig11" aria-label="Figure 11">Figure 11</a> shows a hERG model example where fingerprint-based descriptors are mapped back on the molecule with the degree of contributions. The visualization identifies the moieties that chemists need to focus on for the property optimization.</div><figure id="fig11" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/medium/jm-2017-004878_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/large/jm-2017-004878_0011.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. Visual representation of hERG inhibition model. Red colors are associated with strong inhibition and blue with absence of inhibition. Orange reflects a weaker association compared to red.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/large/jm-2017-004878_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00487&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec6" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">6 Future Directions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_40006" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_40006" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In terms of new technologies, future opportunities lie with more integration of DMPK science, as well as in de novo design (<a class="ref internalNav" href="#fig12" aria-label="Figure 12">Figure 12</a>). In particular, beyond de novo design, the two most “voted” aspects in our cross-companies survey were early dose predictions and understanding the interplay between <i>in silico</i>, <i>in vitro</i>, and <i>in vivo</i> ADME data. We are of the opinion that robust models can and will assist in considering potential artifacts or lurking and previously unrecognized variables in the experiment. However, we should strive to utilize experimental protocols that (i) are generating consistent data over time, (ii) provide actual and not just boundary values, and (iii) overall generate data with modeling in mind, certainly a cost-related and management question but of the utmost importance and a fruit-bearing investment. Concerning de novo design, although a number of software tools are headed in the direction of suggesting structural modifications,<a onclick="showRef(event, 'ref69 ref70'); return false;" href="javascript:void(0);" class="ref ref69 ref70">(69, 70)</a> there is still still room for improvement to account for a few practical considerations, such as reagent cost and availability, as well as the ability to provide synthetically tractable ideas in line with the current needs of the project on a weekly basis with minimal human intervention, which are barriers to their usage. The aim of this effort should not be replacing the human component from compound selection but rather suggesting minimal structural modifications that may be of interest.</div><figure id="fig12" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 12</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/medium/jm-2017-004878_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/large/jm-2017-004878_0012.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 12. Results of the survey addressing future development in the <i>in silico</i> ADME-PK fields.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/large/jm-2017-004878_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00487&amp;id=fig12"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Drug discovery is a multiparameter optimization (MPO) process, which poses the problem of objectivity when evaluating progress. During lead optimization (or even earlier) it could be beneficial to evolve the optimization process from an arbitrary MPO score (or traffic-light system) to a “dose optimization system”, effectively an MPO approach based on mechanistic premises.<a onclick="showRef(event, 'ref51 ref57 ref58 ref59'); return false;" href="javascript:void(0);" class="ref ref51 ref57 ref58 ref59">(51, 57-59)</a> Each experienced drug discovery scientist is aware of the challenges associated with dose projection, predicting human PK parameters and assessing drivers for the PK/PD relationship.<a onclick="showRef(event, 'ref71'); return false;" href="javascript:void(0);" class="ref ref71">(71)</a> Nevertheless, it would be hard to argue that once <i>in vitro</i> and PK data are available, an arbitrary MPO score constitutes a better way to rank-order compounds in the same chemical series than mechanistic dose projections.<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> In this arena <i>in silico</i> and <i>in vitro</i> scientists and pharmacokineticists should work together to reach an appropriate balance between simple compartmental modeling and more advanced PBPK predictions. Such an effort should focus on creating a tool for rank-ordering similar compounds, not necessarily aiming at providing accurate dose projections. Lastly, as for other technologies discussed, dose projection approaches will only be successful if the effort also emphasizes translatability to the <i>in vivo</i> situation and accessibility and ease of understanding to medicinal chemists.</div></div><div id="sec7" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">7 Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_15067" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_15067" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">This work aims to bring together the wide experience of a group of scientists from several major pharmaceutical companies working in the <i>in silico</i> ADME-PK area. While <i>in silico</i> ADME-PK has been a reality in industry since the introduction of Lipinski’s rule of 5, this is often perceived as an extension of other disciplines such as DMPK or computational chemistry. As a consequence, scientists operating in this area are generally trained in computer science, chemistry, or DMPK science and did not receive formal training on <i>in silico</i> ADME-PK. On one hand, this has promoted an open environment creating an opportunity for people with diverse expertise (i.e., <i>in vitro</i>, <i>in silico</i>, statistics, analytical, PK, structural biology, synthetic chemistry, machine learning, etc.) to collaborate and innovate together. On the other hand, different from the more established PBPK and structure-based design fields, <i>in silico</i> ADME-PK is only now beginning to transform the experience and knowledge assembled through the years into best practices, moving the focus from creating technologies to using technologies, changing the objective from demonstrating potential to demonstrating impact. Implementing <i>in silico</i> tools for ADME-PK in drug discovery to support design leads to a tangible increase in knowledge and reduction in data generation. Collaboration between different departments and ease of access to data and models are key to success in this area. Several companies with well-integrated platforms of models incorporated into the screening cascades to improve the optimization flow for the properties of interest have greatly benefited from this approach.</div><div class="NLM_p last">Drug discovery is an evolving landscape; regulatory agencies are currently promoting and rewarding novel breakthourgh therapies, leading to less explored biology and unfamilar chemical space. In this context companies capable of rapidly progressing initial HTS screening hits to druglike compounds amenable for testing efficacy and safety of novel targets could be the ones that thrive. This is the space where QSPR models, MMP, and other forms of chemical data mining are most effective. It will be important for <i>in silico</i> ADME scientists to consolidate best practices in this area and expand the pool of adopters.</div></div><div class="NLM_back"><div class="NLM_notes" id="NOTES-d7e1531-autogenerated"><p class="first last">The authors declare no competing financial interest.</p></div><div class="article_supporting-info" id=""></div><div class="testing" data-doi="10.1021/acs.jmedchem.7b00487" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_45808" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_45808" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Fabio Broccatelli</span> - <span class="hlFld-Affiliation affiliation">Genentech
Inc., South San Francisco, California 94080, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-0724-9752" title="Orcid link">http://orcid.org/0000-0002-0724-9752</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#95f3f4f7fcfabbf7e7faf6f6f4e1f0f9f9fcd5f2f8f4fcf9bbf6faf8"><span class="__cf_email__" data-cfemail="3b5d5a5952541559495458585a4f5e5757527b5c565a525715585456">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Franco Lombardo</span> - <span class="hlFld-Affiliation affiliation">Alkermes
Inc., 852 Winter Street, Waltham, Massachusetts 02451, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Prashant V. Desai</span> - <span class="hlFld-Affiliation affiliation">Computational
ADME, Drug Disposition, Eli Lilly and Company, Indianapolis, Indiana 46285, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rieko Arimoto</span> - <span class="hlFld-Affiliation affiliation">Vertex
Pharmaceuticals Inc., 50 Northern Avenue, Boston, Massachusetts 02210, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kelly E. Desino</span> - <span class="hlFld-Affiliation affiliation">AbbVie,
Inc., North Chicago, Illinois 60064, United
States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Holger Fischer</span> - <span class="hlFld-Affiliation affiliation">Roche
Pharmaceutical Research and Early Development, Pharmaceutical Sciences,
Innovation Center Basel, F. Hoffmann-La
Roche Ltd., 4070 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Christopher E. Keefer</span> - <span class="hlFld-Affiliation affiliation">Pfizer Inc., Groton, Connecticut 06340, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Carl Petersson</span> - <span class="hlFld-Affiliation affiliation">Discovery
Drug Disposition, Biopharma, R&D Global Early Development, EMD Serono, Frankfurter Strasse 250 I Postcode D39/001, 64293 Darmstadt, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Susanne Winiwarter</span> - <span class="hlFld-Affiliation affiliation">Drug
Safety and Metabolism, AstraZeneca R&D
Gothenburg, 431 83 Mölndal, Sweden</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><p>The authors declare no competing financial interest.<br /></br></p></li></ul></div><div class="bio" rid="ath1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="biographies">Biographies</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_26672" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_26672" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><figure id="ath1" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Franco Lombardo</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00487&amp;id=BIO-d7e1705-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Franco Lombardo</b> earned his M.S. in Pharmaceutical Chemistry at University of Rome, Italy, and his Ph.D. in Organic Chemistry at the University of Minnesota. His industrial work and research and experience span nearly 27 years and several areas, encompassing preformulation work and experimental and modeling work on physicochemical properties and ADME-PK, at Merck, Pfizer in early Development and in Discovery, and at the Novartis Institutes for Biomedical Research. At NIBR, from 2006, his interests and work continued to reside in the areas of ADME-PK and physicochemical properties determination, <i>in vivo</i> and <i>in vitro</i> profiling and modeling, primarily <i>in silico</i> QSP(k)Rs. He joined Biogen in 2014 and, since January 2017, is the head of the DMPK and Bioanalysis Group, at Alkermes, with responsibilities spanning Discovery and Development.</p></figure></div><div class="bio" rid="ath2"><figure id="ath2" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Prashant V. Desai</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00487&amp;id=BIO-d7e1719-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Prashant V. Desai</b> received his B.S. in Pharmacy, M.S. in Organic Synthesis, and Ph.D. in Biophysics and Computational Chemistry from University of Bombay (India). For the past 10 years he has been working with the ADME function in Eli Lilly, leading the computational ADME group. The group’s key focus is to extract knowledge from ADME data using variety of <i>in silico</i> techniques. He also serves as the ADME project leader across various therapeutic areas, the scientific leader for the prelead ADME research, and the co-chair of cross-functional Lead Generation Core Team. His research is mainly focused on interactions with cytochrome P450 enzymes, transporters, and CNS drug discovery. He has contributed to more than 50 papers in peer reviewed journals and book chapters.</p></figure></div><div class="bio" rid="ath3"><figure id="ath3" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Rieko Arimoto</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00487&amp;id=BIO-d7e1727-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Rieko Arimoto</b> received a B.S. in Applied Physics from Osaka University School of Engineering (Osaka, Japan) and a Ph.D. in Biomedical Engineering from Washington University (St. Louis, MO). Her Ph.D work was on modeling of molecular interactions between rhodopsin and transducin. She joined the CADD group at Pfizer (Ann Arbor, MI) as a postdoctoral fellow and focused on computational ADMET modeling works. Currently, she is a Senior Research Scientist in the DMPK department at Vertex Pharmaceuticals (Boston, MA) where she is responsible for supporting research and development projects and continues to work on <i>in silico</i> ADME tools with the modeling group. Her work and research interest spans several areas of ADME, including <i>in vitro</i>/<i>in vivo</i> correlation, human PK prediction, metabolite profile determination, and non-P450 metabolism.</p></figure></div><div class="bio" rid="ath4"><figure id="ath4" data-index="16" class="article__inlineFigure"><h2 class="fig-label">Kelly E. Desino</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00487&amp;id=BIO-d7e1741-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Kelly E. Desino</b> received a B.S. in Biology from the University of Michigan (Ann Arbor, MI) and a Ph.D. in Pharmaceutical Chemistry from the University of Kansas (Lawrence, KS). She is currently the Senior Group Leader of the In Vitro ADME & PK group at AbbVie, Inc. (North Chicago, IL). Her lab is responsible for <i>in vitro</i> ADME and PK support from discovery through NDA/BLA and beyond. She has expertise in assay development, laboratory automation, and data visualization and was instrumental in developing in-house <i>in silico</i> ADME models.</p></figure></div><div class="bio" rid="ath5"><figure id="ath5" data-index="17" class="article__inlineFigure"><h2 class="fig-label">Holger Fischer</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00487&amp;id=BIO-d7e1752-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Holger Fischer</b> received his Ph.D. in Biophysical Chemistry from the University of Basel. After postdoctoral research at the Biocenter in Basel he joined the Molecular Properties group at F. Hoffmann-La Roche in Basel. In this group he was responsible for the development of <i>in vitro</i> assays and <i>in silico</i> tools in the area of ADMETox as well as for strategic concepts in the field of multidimensional compound optimization at Roche which he applied to more than 30 drug discovery projects over 15 years. He currently works as pharmacokinetic, dynamic, and metabolism project leader scientifically supporting projects from discovery until clinical phase II. During his career, he contributed to more than 50 papers in peer reviewed journals and books and he holds three patents on technical inventions.</p></figure></div><div class="bio" rid="ath6"><figure id="ath6" data-index="18" class="article__inlineFigure"><h2 class="fig-label">Christopher E. Keefer</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00487&amp;id=BIO-d7e1763-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Christopher E. Keefer</b> received his B.S. in Chemistry from the University of Florida and his Ph.D. in Synthetic Inorganic Chemistry with a Certificate in Computer Science from North Carolina State University. He is currently Associate Research Fellow and Head of the Computational ADME Group at Pfizer, Inc. (Groton, CT). Prior to Pfizer, he worked at GSK as head of the Methods and Algorithm Development Group in the Cheminformatics Department. His research interests include the development and analysis of <i>in silico</i> statistical models, model confidence metrics, and MMP and MMS data mining for drug discovery.</p></figure></div><div class="bio" rid="ath7"><figure id="ath7" data-index="19" class="article__inlineFigure"><h2 class="fig-label">Carl Petersson</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00487&amp;id=BIO-d7e1771-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Carl Petersson</b> works at Merck KgaA in Darmstadt, Germany, as a scientific director, drug disposition. In this position he is offering strategic, educational, and hands-on support to optimize and predict human ADME properties of small molecules. In addition, he is serving on several multidisciplinary drug discovery teams, in which he is responsible for the pharmacokinetic and drug metabolism contributions. Before joining his current company, he has worked 12 years at the CNS & Pain unit of AstraZeneca in similar roles. Dr. Petersson has a Ph.D. in Organic Chemistry obtained at the Swedish University of Agricultural Sciences and did his postdoctoral research at the University of Alberta, Edmonton, Canada.</p></figure></div><div class="bio" rid="ath8"><figure id="ath8" data-index="20" class="article__inlineFigure"><h2 class="fig-label">Susanne Winiwarter</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00487&amp;id=BIO-d7e1776-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Susanne Winiwarter</b> studied pharmacy at the University of Vienna, Austria, and received her Ph.D. in 1991 for investigations on structure–activity relationships in sandalwood odor molecules using theoretical chemistry and molecular modeling methods. In 2003, after several years of pharmaceutical research based on molecular modeling and QSPR analysis at the University of Tübingen, Germany, and Uppsala University, Sweden, she joined the DMPK department at AstraZeneca in Mölndal, Sweden, and has since then supported many drug discovery projects with data analysis and global or local <i>in silico</i> ADME models. Susanne has also (co)authored a couple of papers on <i>in silico</i> ADME modeling, with special focus on permeability, transporter interactions, and blood–brain partitioning.</p></figure></div><div class="bio" rid="ath9"><figure id="ath9" data-index="21" class="article__inlineFigure"><h2 class="fig-label">Fabio Broccatelli</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00487&amp;id=BIO-d7e1787-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Fabio Broccatelli</b> received his B.S., M.S., and Ph.D. in Chemical Sciences from the University of Perugia working in the Cruciani’s lab. His Ph.D. was collaboration with Les Benet’s lab (UCSF) focusing on <i>in silico</i> ADME-PK. He went on to a postdoc at the Institute of Cancer Research London, working as computational chemist in therapeutic projects. Fabio is currently a Scientist at Genentech where he holds a joint appointment between DMPK and Chemistry; in his role he works as DMPK and Computational Chemistry lead in therapeutic projects, supports the <i>in silico</i> ADME-PK infrastructure, explores the use of <i>in silico</i> tools in IVIVe, early PBPK, and human dose projection. Fabio was awarded with the 2013 AAPS Manuscript Award and has published 13 peer-reviewed articles.</p></figure></div><div class="ack" id="ACK-d7e1528-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i26">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_64325" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_64325" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank all the present and past members of the Working Group that were not actively involved in this publication for the help with some of the ideas and for the stimulating discussions: Arijit Chakravaty, Bernard Faller, Ling He, George Lai, Gerhard Gross, Victor Guarino, Yiding Hu, Brian Mattioni, Karsten Menzel, Tsuyoshi Minematsu, Klara Valko. We thank Marcel Hop and the IQ DMLG for the help and support during the preparation of this manuscript. We are grateful to Nancy Agrawal, Scott Obach, and the IQ DMLG for the critical review of the manuscript and for the valuable insights provided.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i27" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i27"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i28" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i28"> Abbreviations Used</h2><tr><td class="NLM_term">ADME-PK</td><td class="NLM_def"><p class="first last">absorption, distribution, metabolism, excretion, and pharmacokinetics</p></td></tr><tr><td class="NLM_term">AZ</td><td class="NLM_def"><p class="first last">AstraZeneca</p></td></tr><tr><td class="NLM_term">Caco-2</td><td class="NLM_def"><p class="first last">human colon carcinoma cell line</p></td></tr><tr><td class="NLM_term">CL<sub>int</sub></td><td class="NLM_def"><p class="first last">intrinsic clearance</p></td></tr><tr><td class="NLM_term">CL<sub>int,u</sub></td><td class="NLM_def"><p class="first last">unbound intrinsic clearance</p></td></tr><tr><td class="NLM_term">GNE</td><td class="NLM_def"><p class="first last">Genentech</p></td></tr><tr><td class="NLM_term">h-fu<sub>p</sub></td><td class="NLM_def"><p class="first last">fraction unbound in human plasma</p></td></tr><tr><td class="NLM_term">HLM</td><td class="NLM_def"><p class="first last">human liver microsome</p></td></tr><tr><td class="NLM_term">HT</td><td class="NLM_def"><p class="first last">high throughput</p></td></tr><tr><td class="NLM_term">IQ</td><td class="NLM_def"><p class="first last">International Consortium for Innovation through Quality in Pharmaceutical Development</p></td></tr><tr><td class="NLM_term">IT</td><td class="NLM_def"><p class="first last">information technology</p></td></tr><tr><td class="NLM_term">JAK-1</td><td class="NLM_def"><p class="first last">Janus kinase 1</p></td></tr><tr><td class="NLM_term">MDCK</td><td class="NLM_def"><p class="first last">Madin–Darby canine kidney</p></td></tr><tr><td class="NLM_term">MDR1</td><td class="NLM_def"><p class="first last">multidrug resistance gene 1, coding P-gp</p></td></tr><tr><td class="NLM_term">MMP</td><td class="NLM_def"><p class="first last">matched molecular pair</p></td></tr><tr><td class="NLM_term">MMS</td><td class="NLM_def"><p class="first last">matched molecular series</p></td></tr><tr><td class="NLM_term">MPO</td><td class="NLM_def"><p class="first last">multiparameter optimization</p></td></tr><tr><td class="NLM_term">PBPK</td><td class="NLM_def"><p class="first last">physiologically based pharmacokinetics</p></td></tr><tr><td class="NLM_term">Perm</td><td class="NLM_def"><p class="first last">permeability</p></td></tr><tr><td class="NLM_term">PK/PD</td><td class="NLM_def"><p class="first last">pharmacokinetic–pharmacodynamic</p></td></tr><tr><td class="NLM_term">Rat_Mic_Clint,u</td><td class="NLM_def"><p class="first last">unbound intrinsic clearance in rat liver microsomes</p></td></tr><tr><td class="NLM_term">rmse</td><td class="NLM_def"><p class="first last">root mean squared error</p></td></tr><tr><td class="NLM_term">RMSEP</td><td class="NLM_def"><p class="first last">root mean squared error of the prediction</p></td></tr><tr><td class="NLM_term">StDev</td><td class="NLM_def"><p class="first last">standard deviation</p></td></tr><tr><td class="NLM_term">TDI</td><td class="NLM_def"><p class="first last">time-dependent inhibition</p></td></tr><tr><td class="NLM_term"><i>V</i><sub>ss</sub></td><td class="NLM_def"><p class="first last">volume of distribution at steady state</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i29">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_61631" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_61631" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 71 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">van de Waterbeemd, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gifford, E.</span><span> </span><span class="NLM_article-title">ADMET in silico modeling: towards prediction paradise?</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x"> (</span><span class="NLM_issue">3</span><span class="NLM_x">) </span> <span class="NLM_fpage">192</span><span class="NLM_x">–</span> <span class="NLM_lpage">204</span><span class="refDoi"> DOI: 10.1038/nrd1032</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.1038%2Fnrd1032" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2003&pages=192-204&issue=3&author=H.+van+de+Waterbeemdauthor=E.+Gifford&title=ADMET+in+silico+modeling%3A+towards+prediction+paradise%3F&doi=10.1038%2Fnrd1032"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrd1032&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd1032%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2Bde%2BWaterbeemd%26aufirst%3DH.%26aulast%3DGifford%26aufirst%3DE.%26atitle%3DADMET%2520in%2520silico%2520modeling%253A%2520towards%2520prediction%2520paradise%253F%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2003%26volume%3D2%26issue%3D3%26spage%3D192%26epage%3D204%26doi%3D10.1038%2Fnrd1032" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Lombardo, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gifford, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shalaeva, M. Y.</span><span> </span><span class="NLM_article-title"><i>In silico</i> ADME prediction: data, models, facts and myths</span> <span class="citation_source-journal">Mini-Rev. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x"> (</span><span class="NLM_issue">8</span><span class="NLM_x">) </span> <span class="NLM_fpage">861</span><span class="NLM_x">–</span> <span class="NLM_lpage">875</span><span class="refDoi"> DOI: 10.2174/1389557033487629</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.2174%2F1389557033487629" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2003&pages=861-875&issue=8&author=F.+Lombardoauthor=E.+Giffordauthor=M.+Y.+Shalaeva&title=In+silico+ADME+prediction%3A+data%2C+models%2C+facts+and+myths&doi=10.2174%2F1389557033487629"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.2174%2F1389557033487629&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1389557033487629%26sid%3Dliteratum%253Aachs%26aulast%3DLombardo%26aufirst%3DF.%26aulast%3DGifford%26aufirst%3DE.%26aulast%3DShalaeva%26aufirst%3DM.%2BY.%26atitle%3DIn%2520silico%2520ADME%2520prediction%253A%2520data%252C%2520models%252C%2520facts%2520and%2520myths%26jtitle%3DMini-Rev.%2520Med.%2520Chem.%26date%3D2003%26volume%3D3%26issue%3D8%26spage%3D861%26epage%3D875%26doi%3D10.2174%2F1389557033487629" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Gleeson, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hersey, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hannongbua, S.</span><span> </span><span class="NLM_article-title">In silico ADME Models: a general assessment of their utility in drug discovery applications</span> <span class="citation_source-journal">Curr. Top. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x"> (</span><span class="NLM_issue">4</span><span class="NLM_x">) </span> <span class="NLM_fpage">358</span><span class="NLM_x">–</span> <span class="NLM_lpage">381</span><span class="refDoi"> DOI: 10.2174/156802611794480927</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.2174%2F156802611794480927" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=21320065" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BC3MXkslSnur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2011&pages=358-381&issue=4&author=M.+P.+Gleesonauthor=A.+Herseyauthor=S.+Hannongbua&title=In+silico+ADME+Models%3A+a+general+assessment+of+their+utility+in+drug+discovery+applications&doi=10.2174%2F156802611794480927"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">In-silico ADME models: a general assessment of their utility in drug discovery applications</span></div><div class="casAuthors">Gleeson, M. Paul; Hersey, Anne; Hannongbua, Supa</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">358-381</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  ADME prediction is an extremely challenging area as many of the properties we try to predict are a result of multiple physiol. processes.  In this review we consider how in-silico predictions of ADME processes can be used to help bias medicinal chem. into more ideal areas of property space, minimizing the no. of compds. needed to be synthesized to obtain the required biochem./physico-chem. profile.  While such models are not sufficiently accurate to act as a replacement for in-vivo or in-vitro methods, in-silico methods nevertheless can help us to understand the underlying physico-chem. dependencies of the different ADME properties, and thus can give us inspiration on how to optimize them.  Many global in-silico ADME models (i.e generated on large, diverse datasets) have been reported in the literature.  In this paper we selectively review representatives from each distinct class and discuss their relative utility in drug discovery.  For each ADME parameter, we limit our discussion to the most recent, most predictive or most insightful examples in the literature to highlight the current state of the art.  In each case we briefly summarize the different types of models available for each parameter (i.e simple rules, physico-chem. and 3D based QSAR predictions), their overall accuracy and the underlying SAR.  We also discuss the utility of the models as related to lead generation and optimization phases of discovery research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNyakFYvcjtrVg90H21EOLACvtfcHk0ljMJ5OCm11bZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXkslSnur4%253D&md5=5ca56a568815370a3b04833db45c0a9b</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.2174%2F156802611794480927&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F156802611794480927%26sid%3Dliteratum%253Aachs%26aulast%3DGleeson%26aufirst%3DM.%2BP.%26aulast%3DHersey%26aufirst%3DA.%26aulast%3DHannongbua%26aufirst%3DS.%26atitle%3DIn%2520silico%2520ADME%2520Models%253A%2520a%2520general%2520assessment%2520of%2520their%2520utility%2520in%2520drug%2520discovery%2520applications%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2011%26volume%3D11%26issue%3D4%26spage%3D358%26epage%3D381%26doi%3D10.2174%2F156802611794480927" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Cheng, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang, Y.</span><span> </span><span class="NLM_article-title"><i>In silico</i> ADMET prediction: recent advances, current challenges and future trends</span> <span class="citation_source-journal">Curr. Top. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x"> (</span><span class="NLM_issue">11</span><span class="NLM_x">) </span> <span class="NLM_fpage">1273</span><span class="NLM_x">–</span> <span class="NLM_lpage">1283</span><span class="refDoi"> DOI: 10.2174/15680266113139990033</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.2174%2F15680266113139990033" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=23675935" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1WnsbzM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2013&pages=1273-1283&issue=11&author=F.+Chengauthor=W.+Liauthor=G.+Liuauthor=Y.+Tang&title=In+silico+ADMET+prediction%3A+recent+advances%2C+current+challenges+and+future+trends&doi=10.2174%2F15680266113139990033"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">In Silico ADMET Prediction: Recent Advances, Current Challenges and Future Trends</span></div><div class="casAuthors">Cheng, Feixiong; Li, Weihua; Liu, Guixia; Tang, Yun</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1273-1289</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  There are numerous small mol. compds. around us to affect our health, such as drugs, pesticides, food additives, industrial chems., and environmental pollutants.  Over decades, properties related to absorption, distribution, metab., excretion, and toxicity (ADMET) have become one of the most important issues to assess the effects or risks of these compds. on human body.  Recent high-rate drug withdrawals increase the pressure on regulators and pharmaceutical industry to improve preclin. safety testing.  Since in vivo and in vitro evaluations are costly and laborious, in silico techniques have been widely used to est. these properties.  In this review, we would briefly describe the recent advances of in silico ADMET prediction, with emphasis on substructure pattern recognition method that we developed recently.  Challenges and limitations in the area of in silico ADMET prediction were further discussed, such as application domain of models, models validation techniques, and global vs. local models.  At last, several new promising research directions were provided, such as computational systems toxicol. (toxicogenomics), data-integration and meta-decision making systems, which could be used for systemic in silico ADMET prediction in drug discovery and hazard risk assessment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGom7qcGJZNvrbVg90H21EOLACvtfcHk0ljMJ5OCm11bZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1WnsbzM&md5=9edae4014f84c6a0f95d1598a9d4ccc7</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.2174%2F15680266113139990033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F15680266113139990033%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DF.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DG.%26aulast%3DTang%26aufirst%3DY.%26atitle%3DIn%2520silico%2520ADMET%2520prediction%253A%2520recent%2520advances%252C%2520current%2520challenges%2520and%2520future%2520trends%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2013%26volume%3D13%26issue%3D11%26spage%3D1273%26epage%3D1283%26doi%3D10.2174%2F15680266113139990033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Manallack, D. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tehan, B. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gancia, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hudson, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ford, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Livingstone, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whitley, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pitt, W. R.</span><span> </span><span class="NLM_article-title">A consensus neural network-based technique for discriminating soluble and poorly soluble compounds</span> <span class="citation_source-journal">J. Chem. Inf. Comput. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">43</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">674</span><span class="NLM_x">–</span> <span class="NLM_lpage">679</span><span class="refDoi"> DOI: 10.1021/ci0202741</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci0202741" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BD3sXivFWkuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2003&pages=674-679&issue=2&author=D.+T.+Manallackauthor=B.+G.+Tehanauthor=E.+Ganciaauthor=B.+D.+Hudsonauthor=M.+G.+Fordauthor=D.+J.+Livingstoneauthor=D.+C.+Whitleyauthor=W.+R.+Pitt&title=A+consensus+neural+network-based+technique+for+discriminating+soluble+and+poorly+soluble+compounds&doi=10.1021%2Fci0202741"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">A Consensus Neural Network-Based Technique for Discriminating Soluble and Poorly Soluble Compounds</span></div><div class="casAuthors">Manallack, David T.; Tehan, Benjamin G.; Gancia, Emanuela; Hudson, Brian D.; Ford, Martyn G.; Livingstone, David J.; Whitley, David C.; Pitt, Will R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">674-679</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">BCUT [Burden, CAS, and University of Texas] descriptors, defined as eigenvalues of modified connectivity matrixes, have traditionally been applied to drug design tasks such as defining receptor relevant subspaces to assist in compd. selections.  In this paper we present studies of consensus neural networks trained on BCUTs to discriminate compds. with poor aq. soly. from those with reasonable soly.  This level was set at 0.1 mg/mL on advice from drug formulation and drug discovery scientists.  By applying strict criteria to the insoly. predictions, approx. 95% of compds. are classified correctly.  For compds. whose predictions have a lower level of confidence, further parameters are examd. in order to flag those considered to possess unsuitable biopharmaceutical and physicochem. properties.  This approach is not designed to be applied in isolation but is intended to be used as a filter in the selection of screening candidates, compd. purchases, and the application of synthetic priorities to combinatorial libraries.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFfQPsH6Nj77Vg90H21EOLACvtfcHk0lis9nEiu7MEvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXivFWkuw%253D%253D&md5=576c750212ce64c9414401d52cc0156c</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1021%2Fci0202741&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci0202741%26sid%3Dliteratum%253Aachs%26aulast%3DManallack%26aufirst%3DD.%2BT.%26aulast%3DTehan%26aufirst%3DB.%2BG.%26aulast%3DGancia%26aufirst%3DE.%26aulast%3DHudson%26aufirst%3DB.%2BD.%26aulast%3DFord%26aufirst%3DM.%2BG.%26aulast%3DLivingstone%26aufirst%3DD.%2BJ.%26aulast%3DWhitley%26aufirst%3DD.%2BC.%26aulast%3DPitt%26aufirst%3DW.%2BR.%26atitle%3DA%2520consensus%2520neural%2520network-based%2520technique%2520for%2520discriminating%2520soluble%2520and%2520poorly%2520soluble%2520compounds%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D2003%26volume%3D43%26issue%3D2%26spage%3D674%26epage%3D679%26doi%3D10.1021%2Fci0202741" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Kuhn, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guba, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hert, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banner, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bissantz, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ceccarelli, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haap, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Körner, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuglstatter, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lerner, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattei, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neidhart, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pinard, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rudolph, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schulz-Gasch, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woltering, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stahl, M.</span><span> </span><span class="NLM_article-title">A real-world perspective on molecular design</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x"> (</span><span class="NLM_issue">9</span><span class="NLM_x">) </span> <span class="NLM_fpage">4087</span><span class="NLM_x">–</span> <span class="NLM_lpage">4102</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01875</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01875" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BC28XisVyjtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=4087-4102&issue=9&author=B.+Kuhnauthor=W.+Gubaauthor=J.+Hertauthor=D.+Bannerauthor=C.+Bissantzauthor=S.+Ceccarelliauthor=W.+Haapauthor=M.+K%C3%B6rnerauthor=A.+Kuglstatterauthor=A.+Lernerauthor=P.+Matteiauthor=W.+Neidhartauthor=E.+Pinardauthor=M.+G.+Rudolphauthor=T.+Schulz-Gaschauthor=T.+Wolteringauthor=M.+Stahl&title=A+real-world+perspective+on+molecular+design&doi=10.1021%2Facs.jmedchem.5b01875"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">A Real-World Perspective on Molecular Design</span></div><div class="casAuthors">Kuhn, Bernd; Guba, Wolfgang; Hert, Jerome; Banner, David; Bissantz, Caterina; Ceccarelli, Simona; Haap, Wolfgang; Korner, Matthias; Kuglstatter, Andreas; Lerner, Christian; Mattei, Patrizio; Neidhart, Werner; Pinard, Emmanuel; Rudolph, Markus G.; Schulz-Gasch, Tanja; Woltering, Thomas; Stahl, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4087-4102</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We present a series of small mol. drug discovery case studies where computational methods were prospectively employed to impact Roche research projects, with the aim of highlighting those methods that provide real added value.  Our brief accounts encompass a broad range of methods and techniques applied to a variety of enzymes and receptors.  Most of these are based on judicious application of knowledge about mol. conformations and interactions: filling of lipophilic pockets to gain affinity or selectivity, addn. of polar substituents, scaffold hopping, transfer of SAR, conformation anal., and mol. overlays.  A case study of sequence-driven focused screening is presented to illustrate how appropriate preprocessing of information enables effective exploitation of prior knowledge.  We conclude that qual. statements enabling chemists to focus on promising regions of chem. space are often more impactful than quant. prediction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxOLDuNAX5U7Vg90H21EOLACvtfcHk0lis9nEiu7MEvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XisVyjtb0%253D&md5=0b81f2a0e9996c5dae44e61e5488ba79</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01875&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01875%26sid%3Dliteratum%253Aachs%26aulast%3DKuhn%26aufirst%3DB.%26aulast%3DGuba%26aufirst%3DW.%26aulast%3DHert%26aufirst%3DJ.%26aulast%3DBanner%26aufirst%3DD.%26aulast%3DBissantz%26aufirst%3DC.%26aulast%3DCeccarelli%26aufirst%3DS.%26aulast%3DHaap%26aufirst%3DW.%26aulast%3DK%25C3%25B6rner%26aufirst%3DM.%26aulast%3DKuglstatter%26aufirst%3DA.%26aulast%3DLerner%26aufirst%3DA.%26aulast%3DMattei%26aufirst%3DP.%26aulast%3DNeidhart%26aufirst%3DW.%26aulast%3DPinard%26aufirst%3DE.%26aulast%3DRudolph%26aufirst%3DM.%2BG.%26aulast%3DSchulz-Gasch%26aufirst%3DT.%26aulast%3DWoltering%26aufirst%3DT.%26aulast%3DStahl%26aufirst%3DM.%26atitle%3DA%2520real-world%2520perspective%2520on%2520molecular%2520design%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26issue%3D9%26spage%3D4087%26epage%3D4102%26doi%3D10.1021%2Facs.jmedchem.5b01875" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Muegge, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bentzien, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mukherjee, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hughes, R.</span><span> </span><span class="NLM_article-title">Automatically updating predictive modeling workflows support decision-making in drug design</span> <span class="citation_source-journal">Future Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x"> (</span><span class="NLM_issue">14</span><span class="NLM_x">) </span> <span class="NLM_fpage">1779</span><span class="NLM_x">–</span> <span class="NLM_lpage">1796</span><span class="refDoi"> DOI: 10.4155/fmc-2016-0070</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.4155%2Ffmc-2016-0070" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=27584594" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVKhsrrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2016&pages=1779-1796&issue=14&author=I.+Mueggeauthor=J.+Bentzienauthor=P.+Mukherjeeauthor=R.+Hughes&title=Automatically+updating+predictive+modeling+workflows+support+decision-making+in+drug+design&doi=10.4155%2Ffmc-2016-0070"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Automatically updating predictive modeling workflows support decision-making in drug design</span></div><div class="casAuthors">Muegge, Ingo; Bentzien, Joerg; Mukherjee, Prasenjit; Hughes, Robert O.</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">1779-1796</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">Using predictive models for early decision-making in drug discovery has become std. practice.  We suggest that model building needs to be automated with min. input and low tech. maintenance requirements.  Models perform best when tailored to answering specific compd. optimization related questions.  If qual. answers are required, 2-bin classification models are preferred.  Integrating predictive modeling results with structural information stimulates better decision making.  For in silico models supporting rapid structure-activity relationship cycles the performance deteriorates within weeks.  Frequent automated updates of predictive models ensure best predictions.  Consensus between multiple modeling approaches increases the prediction confidence.  Combining qualified and nonqualified data optimally uses all available information.  Dose predictions provide a holistic alternative to multiple individual property predictions for reaching complex decisions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNAKHon4nFZLVg90H21EOLACvtfcHk0lgXvuQ394wcAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVKhsrrJ&md5=0a243c46bf3df9b2b1308d5f5fd9c37d</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.4155%2Ffmc-2016-0070&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc-2016-0070%26sid%3Dliteratum%253Aachs%26aulast%3DMuegge%26aufirst%3DI.%26aulast%3DBentzien%26aufirst%3DJ.%26aulast%3DMukherjee%26aufirst%3DP.%26aulast%3DHughes%26aufirst%3DR.%26atitle%3DAutomatically%2520updating%2520predictive%2520modeling%2520workflows%2520support%2520decision-making%2520in%2520drug%2520design%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2016%26volume%3D8%26issue%3D14%26spage%3D1779%26epage%3D1796%26doi%3D10.4155%2Ffmc-2016-0070" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Cumming, J. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davis, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muresan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haeberlein, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, H.</span><span> </span><span class="NLM_article-title">Chemical predictive modeling to improve compound quality</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x"> (</span><span class="NLM_issue">12</span><span class="NLM_x">) </span> <span class="NLM_fpage">948</span><span class="NLM_x">–</span> <span class="NLM_lpage">962</span><span class="refDoi"> DOI: 10.1038/nrd4128</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.1038%2Fnrd4128" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2013&pages=948-962&issue=12&author=J.+G.+Cummingauthor=A.+M.+Davisauthor=S.+Muresanauthor=M.+Haeberleinauthor=H.+Chen&title=Chemical+predictive+modeling+to+improve+compound+quality&doi=10.1038%2Fnrd4128"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnrd4128&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4128%26sid%3Dliteratum%253Aachs%26aulast%3DCumming%26aufirst%3DJ.%2BG.%26aulast%3DDavis%26aufirst%3DA.%2BM.%26aulast%3DMuresan%26aufirst%3DS.%26aulast%3DHaeberlein%26aufirst%3DM.%26aulast%3DChen%26aufirst%3DH.%26atitle%3DChemical%2520predictive%2520modeling%2520to%2520improve%2520compound%2520quality%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2013%26volume%3D12%26issue%3D12%26spage%3D948%26epage%3D962%26doi%3D10.1038%2Fnrd4128" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Aliagas, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gobbi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heffron, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, M.-L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ortwine, D. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zak, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khojasteh, S. C.</span><span> </span><span class="NLM_article-title">A probabilistic method to report predictions from a human liver microsomes stability QSAR model: a practical tool for drug discovery</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x"> (</span><span class="NLM_issue">4</span><span class="NLM_x">) </span> <span class="NLM_fpage">327</span><span class="NLM_x">–</span> <span class="NLM_lpage">338</span><span class="refDoi"> DOI: 10.1007/s10822-015-9838-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.1007%2Fs10822-015-9838-3" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2015&pages=327-338&issue=4&author=I.+Aliagasauthor=A.+Gobbiauthor=T.+Heffronauthor=M.-L.+Leeauthor=D.+F.+Ortwineauthor=M.+Zakauthor=S.+C.+Khojasteh&title=A+probabilistic+method+to+report+predictions+from+a+human+liver+microsomes+stability+QSAR+model%3A+a+practical+tool+for+drug+discovery&doi=10.1007%2Fs10822-015-9838-3"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1007%2Fs10822-015-9838-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-015-9838-3%26sid%3Dliteratum%253Aachs%26aulast%3DAliagas%26aufirst%3DI.%26aulast%3DGobbi%26aufirst%3DA.%26aulast%3DHeffron%26aufirst%3DT.%26aulast%3DLee%26aufirst%3DM.-L.%26aulast%3DOrtwine%26aufirst%3DD.%2BF.%26aulast%3DZak%26aufirst%3DM.%26aulast%3DKhojasteh%26aufirst%3DS.%2BC.%26atitle%3DA%2520probabilistic%2520method%2520to%2520report%2520predictions%2520from%2520a%2520human%2520liver%2520microsomes%2520stability%2520QSAR%2520model%253A%2520a%2520practical%2520tool%2520for%2520drug%2520discovery%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2015%26volume%3D29%26issue%3D4%26spage%3D327%26epage%3D338%26doi%3D10.1007%2Fs10822-015-9838-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Desai, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sawada, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watson, I. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raub, T. J.</span><span> </span><span class="NLM_article-title">Integration of <i>in silico</i> and <i>in vitro</i> tools for scaffold optimization during drug discovery: predicting P-Glycoprotein efflux</span> <span class="citation_source-journal">Mol. Pharmaceutics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x"> (</span><span class="NLM_issue">4</span><span class="NLM_x">) </span> <span class="NLM_fpage">1249</span><span class="NLM_x">–</span> <span class="NLM_lpage">1261</span><span class="refDoi"> DOI: 10.1021/mp300555n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/mp300555n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1aju7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2013&pages=1249-1261&issue=4&author=P.+V.+Desaiauthor=G.+A.+Sawadaauthor=I.+A.+Watsonauthor=T.+J.+Raub&title=Integration+of+in+silico+and+in+vitro+tools+for+scaffold+optimization+during+drug+discovery%3A+predicting+P-Glycoprotein+efflux&doi=10.1021%2Fmp300555n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Integration of in Silico and in Vitro Tools for Scaffold Optimization during Drug Discovery: Predicting P-Glycoprotein Efflux</span></div><div class="casAuthors">Desai, Prashant V.; Sawada, Geri A.; Watson, Ian A.; Raub, Thomas J.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmaceutics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1249-1261</span>CODEN:
                <span class="NLM_cas:coden">MPOHBP</span>;
        ISSN:<span class="NLM_cas:issn">1543-8384</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In silico tools are regularly utilized for designing and prioritizing compds. to address challenges related to drug metab. and pharmacokinetics (DMPK) during the process of drug discovery.  P-Glycoprotein (P-gp) is a member of the ATP-binding cassette (ABC) transporters with broad substrate specificity that plays a significant role in absorption and distribution of drugs that are P-gp substrates.  As a result, screening for P-gp transport has now become routine in the drug discovery process.  Typically, bidirectional permeability assays are employed to assess in vitro P-gp efflux.  In this article, we use P-gp as an example to illustrate a well-validated methodol. to effectively integrate in silico and in vitro tools to identify and resolve key barriers during the early stages of drug discovery.  A detailed account of development and application of in silico tools such as simple guidelines based on physicochem. properties and more complex quant. structure-activity relationship (QSAR) models is provided.  The tools were developed based on structurally diverse data for more than 2000 compds. generated using a robust P-gp substrate assay over the past several years.  Anal. of physicochem. properties revealed a significantly lower proportion (<10%) of P-gp substrates among the compds. with topol. polar surface area (TPSA) <60 Å2 and the most basic cpKa <8.  In contrast, this proportion of substrates was greater than 75% for compds. with TPSA >60 Å2 and the most basic cpKa >8.  Among the various QSAR models evaluated to predict P-gp efflux, the Bagging model provided optimum prediction performance for prospective validation based on chronol. test sets.  Four sequential versions of the model were built with increasing nos. of compds. to train the models as new data became available.  Except for the first version with the smallest training set, the QSAR models exhibited robust prediction profiles with pos. prediction values (PPV) and neg. prediction values (NPV) exceeding 80%.  The QSAR model demonstrated better concordance with the manual P-gp substrate assay than an automated P-gp substrate screen.  The in silico and the in vitro tools have been effectively integrated during early stages of drug discovery to resolve P-gp-related challenges exemplified by several case studies.  Key learning based on our experience with P-gp can be widely applicable across other DMPK-related challenges.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5Q5Y8HssxH7Vg90H21EOLACvtfcHk0lgXvuQ394wcAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1aju7o%253D&md5=3bd5239b80270c0f4ed85dffb86704ed</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fmp300555n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fmp300555n%26sid%3Dliteratum%253Aachs%26aulast%3DDesai%26aufirst%3DP.%2BV.%26aulast%3DSawada%26aufirst%3DG.%2BA.%26aulast%3DWatson%26aufirst%3DI.%2BA.%26aulast%3DRaub%26aufirst%3DT.%2BJ.%26atitle%3DIntegration%2520of%2520in%2520silico%2520and%2520in%2520vitro%2520tools%2520for%2520scaffold%2520optimization%2520during%2520drug%2520discovery%253A%2520predicting%2520P-Glycoprotein%2520efflux%26jtitle%3DMol.%2520Pharmaceutics%26date%3D2013%26volume%3D10%26issue%3D4%26spage%3D1249%26epage%3D1261%26doi%3D10.1021%2Fmp300555n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Liu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hop, C. E. C. A.</span><span> </span><span class="NLM_article-title">Do we need to optimize plasma protein and tissue binding in drug discovery?</span> <span class="citation_source-journal">Curr. Top. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x"> (</span><span class="NLM_issue">4</span><span class="NLM_x">) </span> <span class="NLM_fpage">450</span><span class="NLM_x">–</span> <span class="NLM_lpage">466</span><span class="refDoi"> DOI: 10.2174/156802611794480918</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.2174%2F156802611794480918" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=21320069" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BC3MXkslSnuro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2011&pages=450-466&issue=4&author=X.+Liuauthor=C.+Chenauthor=C.+E.+C.+A.+Hop&title=Do+we+need+to+optimize+plasma+protein+and+tissue+binding+in+drug+discovery%3F&doi=10.2174%2F156802611794480918"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Do we need to optimize plasma protein and tissue binding in drug discovery?</span></div><div class="casAuthors">Liu, Xingrong; Chen, Cuiping; Hop, Cornelis E. C. A.</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">450-466</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  It is a commonly accepted assumption that only unbound drug mols. are available to interact with their targets.  In order to achieve high unbound plasma drug concn., it seems obvious to design compds. with low plasma protein binding.  Similarly to achieve high unbound tissue concn., we apparently need compds. with low tissue binding.  Our theor. anal. and exptl. data demonstrate that unbound plasma concn. is not detd. by plasma protein binding but by hepatic intrinsic clearance after oral dose, and unbound tissue concn. is not detd. by tissue binding but detd. by unbound plasma concn. and transport properties at the blood-tissue barrier.  Redn. of plasma and tissue protein binding for a compd. will increase the unbound concn. in vitro but may not increase its unbound plasma or tissue concn. in vivo after oral administration.  We conclude that plasma protein and tissue binding are essential parameters to understand pharmacokinetics and pharmacodynamics but they should not be optimized independently in drug discovery.  Instead we should focus on reducing clearance and efflux at the blood-tissue barrier to increase in vivo plasma and tissue unbound concn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFwT9zhofIgLVg90H21EOLACvtfcHk0lh_Ip3SanhVPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXkslSnuro%253D&md5=de19ad9bb14b369795484abe096bf226</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.2174%2F156802611794480918&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F156802611794480918%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DC.%26aulast%3DHop%26aufirst%3DC.%2BE.%2BC.%2BA.%26atitle%3DDo%2520we%2520need%2520to%2520optimize%2520plasma%2520protein%2520and%2520tissue%2520binding%2520in%2520drug%2520discovery%253F%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2011%26volume%3D11%26issue%3D4%26spage%3D450%26epage%3D466%26doi%3D10.2174%2F156802611794480918" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Di, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rong, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, B.</span><span> </span><span class="NLM_article-title">Demystifying brain penetration in central nervous system drug discovery</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">2</span><span class="NLM_x">–</span> <span class="NLM_lpage">12</span><span class="refDoi"> DOI: 10.1021/jm301297f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm301297f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFamtrrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=2-12&issue=1&author=L.+Diauthor=R.+Rongauthor=B.+Feng&title=Demystifying+brain+penetration+in+central+nervous+system+drug+discovery&doi=10.1021%2Fjm301297f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Demystifying Brain Penetration in Central Nervous System Drug Discovery</span></div><div class="casAuthors">Di, Li; Rong, Haojing; Feng, Bo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">2-12</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  This Perspective provides important concepts about the blood-brain barrier (BBB) in drug discovery and how they should be applied effectively in designing successful CNS drugs.  Key parameters for brain penetration are discussed, including unbound brain concn., unbound brain-to-plasma ratio, BBB permeability, fraction unbound in brain and plasma, and transporters.  Results from a retrospective anal. of 32 Pfizer CNS clin. drug candidates are described.  Frequently encountered misconceptions about brain penetration in drug discovery programs are clarified.  Strategies and guidance are provided to enhance or minimize brain exposure for CNS or peripheral targets, resp.  Recommendations for screening methodologies and a cascade in assessing brain penetration potential are presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmurs7cfju37Vg90H21EOLACvtfcHk0lh_Ip3SanhVPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFamtrrJ&md5=8b88e2044bf41898ab40c45e18f2a20d</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Fjm301297f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm301297f%26sid%3Dliteratum%253Aachs%26aulast%3DDi%26aufirst%3DL.%26aulast%3DRong%26aufirst%3DR.%26aulast%3DFeng%26aufirst%3DB.%26atitle%3DDemystifying%2520brain%2520penetration%2520in%2520central%2520nervous%2520system%2520drug%2520discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26issue%3D1%26spage%3D2%26epage%3D12%26doi%3D10.1021%2Fjm301297f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Mannhold, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poda, G. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ostermann, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tetko, I. V.</span><span> </span><span class="NLM_article-title">Calculation of molecular lipophilicity: state-of-the-art and comparison of log P methods on more than 96,000 compounds</span> <span class="citation_source-journal">J. Pharm. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">98</span><span class="NLM_x"> (</span><span class="NLM_issue">3</span><span class="NLM_x">) </span> <span class="NLM_fpage">861</span><span class="NLM_x">–</span> <span class="NLM_lpage">893</span><span class="refDoi"> DOI: 10.1002/jps.21494</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.1002%2Fjps.21494" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2009&pages=861-893&issue=3&author=R.+Mannholdauthor=G.+I.+Podaauthor=C.+Ostermannauthor=I.+V.+Tetko&title=Calculation+of+molecular+lipophilicity%3A+state-of-the-art+and+comparison+of+log+P+methods+on+more+than+96%2C000+compounds&doi=10.1002%2Fjps.21494"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1002%2Fjps.21494&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjps.21494%26sid%3Dliteratum%253Aachs%26aulast%3DMannhold%26aufirst%3DR.%26aulast%3DPoda%26aufirst%3DG.%2BI.%26aulast%3DOstermann%26aufirst%3DC.%26aulast%3DTetko%26aufirst%3DI.%2BV.%26atitle%3DCalculation%2520of%2520molecular%2520lipophilicity%253A%2520state-of-the-art%2520and%2520comparison%2520of%2520log%2520P%2520methods%2520on%2520more%2520than%252096%252C000%2520compounds%26jtitle%3DJ.%2520Pharm.%2520Sci.%26date%3D2009%26volume%3D98%26issue%3D3%26spage%3D861%26epage%3D893%26doi%3D10.1002%2Fjps.21494" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Wenlock, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carlsson, L. A.</span><span> </span><span class="NLM_article-title">How experimental errors influence drug metabolism and pharmacokinetic QSAR/QSPR models</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">125</span><span class="NLM_x">–</span> <span class="NLM_lpage">134</span><span class="refDoi"> DOI: 10.1021/ci500535s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci500535s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFGku7%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=125-134&issue=1&author=M.+C.+Wenlockauthor=L.+A.+Carlsson&title=How+experimental+errors+influence+drug+metabolism+and+pharmacokinetic+QSAR%2FQSPR+models&doi=10.1021%2Fci500535s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">How Experimental Errors Influence Drug Metabolism and Pharmacokinetic QSAR/QSPR Models</span></div><div class="casAuthors">Wenlock, Mark C.; Carlsson, Lars A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">125-134</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We consider the impact of gross, systematic, and random exptl. errors in relation to their impact on the predictive ability of QSAR/QSPR DMPK models used within early drug discovery.  Models whose training sets contain fewer but repeatedly measured data points, with a defined threshold for the random error, resulted in prediction improvements ranging from 3.3% to 23.0% for an external test set, compared to models built from training sets in which the mols. were defined by single measurements.  Similarly, models built on data with low exptl. uncertainty, compared to those built on data with higher exptl. uncertainty, gave prediction improvements ranging from 3.3% to 27.5%.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoy1uwqWjCVt7Vg90H21EOLACvtfcHk0lh_Ip3SanhVPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFGku7%252FF&md5=39f934233f69bd825c17145922771abb</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fci500535s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci500535s%26sid%3Dliteratum%253Aachs%26aulast%3DWenlock%26aufirst%3DM.%2BC.%26aulast%3DCarlsson%26aufirst%3DL.%2BA.%26atitle%3DHow%2520experimental%2520errors%2520influence%2520drug%2520metabolism%2520and%2520pharmacokinetic%2520QSAR%252FQSPR%2520models%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26volume%3D55%26issue%3D1%26spage%3D125%26epage%3D134%26doi%3D10.1021%2Fci500535s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Winiwarter, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Middleton, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Courtney, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lindmark, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Page, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clark, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Landqvist, C.</span><span> </span><span class="NLM_article-title">Time dependent analysis of assay comparability: a novel approach to understand intra- and inter-site variability over time</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x"> (</span><span class="NLM_issue">9</span><span class="NLM_x">) </span> <span class="NLM_fpage">795</span><span class="NLM_x">–</span> <span class="NLM_lpage">807</span><span class="refDoi"> DOI: 10.1007/s10822-015-9836-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.1007%2Fs10822-015-9836-5" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2015&pages=795-807&issue=9&author=S.+Winiwarterauthor=B.+Middletonauthor=B.+Jonesauthor=P.+Courtneyauthor=B.+Lindmarkauthor=K.+M.+Pageauthor=A.+Clarkauthor=C.+Landqvist&title=Time+dependent+analysis+of+assay+comparability%3A+a+novel+approach+to+understand+intra-+and+inter-site+variability+over+time&doi=10.1007%2Fs10822-015-9836-5"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1007%2Fs10822-015-9836-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-015-9836-5%26sid%3Dliteratum%253Aachs%26aulast%3DWiniwarter%26aufirst%3DS.%26aulast%3DMiddleton%26aufirst%3DB.%26aulast%3DJones%26aufirst%3DB.%26aulast%3DCourtney%26aufirst%3DP.%26aulast%3DLindmark%26aufirst%3DB.%26aulast%3DPage%26aufirst%3DK.%2BM.%26aulast%3DClark%26aufirst%3DA.%26aulast%3DLandqvist%26aufirst%3DC.%26atitle%3DTime%2520dependent%2520analysis%2520of%2520assay%2520comparability%253A%2520a%2520novel%2520approach%2520to%2520understand%2520intra-%2520and%2520inter-site%2520variability%2520over%2520time%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2015%26volume%3D29%26issue%3D9%26spage%3D795%26epage%3D807%26doi%3D10.1007%2Fs10822-015-9836-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Kramer, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fuchs, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whitebread, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gedeck, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liedl, K. R.</span><span> </span><span class="NLM_article-title">Matched molecular pair analysis: significance and the impact of experimental uncertainty</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x"> (</span><span class="NLM_issue">9</span><span class="NLM_x">) </span> <span class="NLM_fpage">3786</span><span class="NLM_x">–</span> <span class="NLM_lpage">3802</span><span class="refDoi"> DOI: 10.1021/jm500317a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm500317a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=3786-3802&issue=9&author=C.+Kramerauthor=J.+E.+Fuchsauthor=S.+Whitebreadauthor=P.+Gedeckauthor=K.+R.+Liedl&title=Matched+molecular+pair+analysis%3A+significance+and+the+impact+of+experimental+uncertainty&doi=10.1021%2Fjm500317a"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fjm500317a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm500317a%26sid%3Dliteratum%253Aachs%26aulast%3DKramer%26aufirst%3DC.%26aulast%3DFuchs%26aufirst%3DJ.%2BE.%26aulast%3DWhitebread%26aufirst%3DS.%26aulast%3DGedeck%26aufirst%3DP.%26aulast%3DLiedl%26aufirst%3DK.%2BR.%26atitle%3DMatched%2520molecular%2520pair%2520analysis%253A%2520significance%2520and%2520the%2520impact%2520of%2520experimental%2520uncertainty%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26issue%3D9%26spage%3D3786%26epage%3D3802%26doi%3D10.1021%2Fjm500317a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Segall, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Champness, E. J.</span><span> </span><span class="NLM_article-title">The challenges of making decisions using uncertain data</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x"> (</span><span class="NLM_issue">9</span><span class="NLM_x">) </span> <span class="NLM_fpage">809</span><span class="NLM_x">–</span> <span class="NLM_lpage">816</span><span class="refDoi"> DOI: 10.1007/s10822-015-9855-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.1007%2Fs10822-015-9855-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=26126976" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtV2ktrbI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2015&pages=809-816&issue=9&author=M.+D.+Segallauthor=E.+J.+Champness&title=The+challenges+of+making+decisions+using+uncertain+data&doi=10.1007%2Fs10822-015-9855-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">The challenges of making decisions using uncertain data</span></div><div class="casAuthors">Segall, Matthew D.; Champness, Edmund J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">809-816</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">All of the exptl. compd. data with which we work have significant uncertainties, due to imperfect correlations between exptl. systems and the ultimate in vivo properties of compds. and the inherent variability in exptl. conditions.  When using these data to make decisions, it is essential that these uncertainties are taken into account to avoid making inappropriate decisions in the selection of compds., which can lead to wasted effort and missed opportunities.  In this paper we will consider approaches to rigorously account for uncertainties when selecting between compds. or assessing compds. against a property criterion; first for an individual measurement of a single property and then for multiple measurements of a property for the same compd.  We will then explore how uncertainties in multiple properties can be combined when assessing compds. against a profile of criteria, a process known as multi-parameter optimization.  This guides rigorous decision-making using complex, uncertain data to focus on compds. with the best chance of success, while avoiding missed opportunities by inappropriately rejecting compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3y3PbHH6LibVg90H21EOLACvtfcHk0lh_Aez-OpdNzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtV2ktrbI&md5=9ac2c9e15bf62f3e1b254816041a8484</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1007%2Fs10822-015-9855-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-015-9855-2%26sid%3Dliteratum%253Aachs%26aulast%3DSegall%26aufirst%3DM.%2BD.%26aulast%3DChampness%26aufirst%3DE.%2BJ.%26atitle%3DThe%2520challenges%2520of%2520making%2520decisions%2520using%2520uncertain%2520data%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2015%26volume%3D29%26issue%3D9%26spage%3D809%26epage%3D816%26doi%3D10.1007%2Fs10822-015-9855-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Beck, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seeliger, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kriegl, J. M.</span><span> </span><span class="NLM_article-title">The impact of data integrity on decision making in early lead discovery</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x"> (</span><span class="NLM_issue">9</span><span class="NLM_x">) </span> <span class="NLM_fpage">911</span><span class="NLM_x">–</span> <span class="NLM_lpage">921</span><span class="refDoi"> DOI: 10.1007/s10822-015-9871-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.1007%2Fs10822-015-9871-2" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2015&pages=911-921&issue=9&author=B.+Beckauthor=D.+Seeligerauthor=J.+M.+Kriegl&title=The+impact+of+data+integrity+on+decision+making+in+early+lead+discovery&doi=10.1007%2Fs10822-015-9871-2"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1007%2Fs10822-015-9871-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-015-9871-2%26sid%3Dliteratum%253Aachs%26aulast%3DBeck%26aufirst%3DB.%26aulast%3DSeeliger%26aufirst%3DD.%26aulast%3DKriegl%26aufirst%3DJ.%2BM.%26atitle%3DThe%2520impact%2520of%2520data%2520integrity%2520on%2520decision%2520making%2520in%2520early%2520lead%2520discovery%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2015%26volume%3D29%26issue%3D9%26spage%3D911%26epage%3D921%26doi%3D10.1007%2Fs10822-015-9871-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Winiwarter, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonham, N. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ax, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hallberg, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lennernäs, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karlén, A.</span><span> </span><span class="NLM_article-title">Correlation of human jejunal permeability (<i>in vivo</i>) of drugs with experimentally and theoretically derived parameters. A multivariate data analysis approach</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x"> (</span><span class="NLM_issue">25</span><span class="NLM_x">) </span> <span class="NLM_fpage">4939</span><span class="NLM_x">–</span> <span class="NLM_lpage">4949</span><span class="refDoi"> DOI: 10.1021/jm9810102</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm9810102" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADyaK1cXntFKltLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=1998&pages=4939-4949&issue=25&author=S.+Winiwarterauthor=N.+M.+Bonhamauthor=F.+Axauthor=A.+Hallbergauthor=H.+Lennern%C3%A4sauthor=A.+Karl%C3%A9n&title=Correlation+of+human+jejunal+permeability+%28in+vivo%29+of+drugs+with+experimentally+and+theoretically+derived+parameters.+A+multivariate+data+analysis+approach&doi=10.1021%2Fjm9810102"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Correlation of Human Jejunal Permeability (in Vivo) of Drugs with Experimentally and Theoretically Derived Parameters. A Multivariate Data Analysis Approach</span></div><div class="casAuthors">Winiwarter, Susanne; Bonham, Nicholas M.; Ax, Fredrik; Hallberg, Anders; Lennernaes, Hans; Karlen, Anders</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">4939-4949</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The effective permeability (Peff) in the human jejunum (in vivo) of 22 structurally diverse compds. was correlated with both exptl. detd. lipophilicity values and calcd. mol. descriptors.  The permeability data were previously obtained by using a regional in vivo perfusion system in the proximal jejunum in humans as part of constructing a biopharmaceutical classification system for oral immediate-release products.  PKa, log P, and, where relevant, log Pion values were detd. using the pH-metric technique.  On the basis of these expts., log D values were calcd. at pH 5.5, 6.5, and 7.4.  Multivariate data anal. was used to derive models that correlate passive intestinal permeability to physicochem. descriptors.  The best model obtained, based on 13 passively transcellularly absorbed compds., used the variables HBD (no. of hydrogen bond donors), PSA (polar surface area), and either log D5.5 or log D6.5 (octanol/water distribution coeff. at pH 5.5 and 6.5, resp.).  Statistically good models for predicting human in vivo Peff values were also obtained by using only HBD and PSA or HBD, PSA, and CLOGP.  These models can be used to predict passive intestinal membrane diffusion in humans for compds. that fit within the defined property space.  We used one of the models obtained above to predict the log Peff values for an external validation set consisting of 34 compds.  A good correlation with the absorption data of these compds. was found.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpz3O2C2bSpILVg90H21EOLACvtfcHk0lh_Aez-OpdNzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXntFKltLw%253D&md5=2a7128943e31080e2e839eaacdc51f0f</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fjm9810102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm9810102%26sid%3Dliteratum%253Aachs%26aulast%3DWiniwarter%26aufirst%3DS.%26aulast%3DBonham%26aufirst%3DN.%2BM.%26aulast%3DAx%26aufirst%3DF.%26aulast%3DHallberg%26aufirst%3DA.%26aulast%3DLennern%25C3%25A4s%26aufirst%3DH.%26aulast%3DKarl%25C3%25A9n%26aufirst%3DA.%26atitle%3DCorrelation%2520of%2520human%2520jejunal%2520permeability%2520%2528in%2520vivo%2529%2520of%2520drugs%2520with%2520experimentally%2520and%2520theoretically%2520derived%2520parameters.%2520A%2520multivariate%2520data%2520analysis%2520approach%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1998%26volume%3D41%26issue%3D25%26spage%3D4939%26epage%3D4949%26doi%3D10.1021%2Fjm9810102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Palm, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luthman, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ros, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gråsjö, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Artursson, P.</span><span> </span><span class="NLM_article-title">Effect of molecular charge on intestinal epithelial drug transport: pH-dependent transport of cationic drugs</span> <span class="citation_source-journal">J. Pharmacol. Exp.Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">291</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">435</span><span class="NLM_x">–</span> <span class="NLM_lpage">443</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10525056" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADyaK1MXntFChtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=291&publication_year=1999&pages=435-443&issue=2&author=K.+Palmauthor=K.+Luthmanauthor=J.+Rosauthor=J.+Gr%C3%A5sj%C3%B6author=P.+Artursson&title=Effect+of+molecular+charge+on+intestinal+epithelial+drug+transport%3A+pH-dependent+transport+of+cationic+drugs"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Effect of molecular charge on intestinal epithelial drug transport: pH-dependent transport of cationic drugs</span></div><div class="casAuthors">Palm, Katrin; Luthman, Kristina; Ros, Jenny; Grasjo, Johan; Artursson, Per</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacology and Experimental Therapeutics</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">291</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">435-443</span>CODEN:
                <span class="NLM_cas:coden">JPETAB</span>;
        ISSN:<span class="NLM_cas:issn">0022-3565</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">The aim of this study was to investigate the effect of ionization on drug transport across the intestinal epithelium to include this effect in structure-absorption relationships.  The pH-dependent permeation of one rapidly (alfentanil) and one slowly (cimetidine) transported basic model drug across Caco-2 cell monolayers was investigated.  Both drugs had pKa values in the physiol. pH range.  The permeability coeffs. (Pc) of the model drugs were obtained at varying apical buffer pHs, thus varying the degree of drug ionization (from 5 to 95%).  The relation between Pc and the fraction of the drug in un-ionized form (fu) was analyzed to delineate the permeability coeffs. of the un-ionized (Pc,u) and ionized (Pc,i) forms of the drugs.  Theor. ests. of the pKa values were also calcd. from ionization energies for each model compd.  For both drugs, a linear increase in Pc was obsd. with increasing fu.  Transport of the un-ionized form was 150- and 30-fold more rapid than transport of the ionized form for alfentanil and cimetidine, resp.  However, when fu < 0.1, the contribution of the ionized form was significant.  Because fu is <0.1 over the entire physiol. pH range for a large no. of drugs, these results will have implications on predictions of in vivo intestinal drug absorption both from in vitro studies in cell cultures and from computed structural properties of drug mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpT8-YOYJTmtrVg90H21EOLACvtfcHk0liDhb1FGVD45Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXntFChtr8%253D&md5=c161433bb0305cb8d195107a16982bbc</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPalm%26aufirst%3DK.%26aulast%3DLuthman%26aufirst%3DK.%26aulast%3DRos%26aufirst%3DJ.%26aulast%3DGr%25C3%25A5sj%25C3%25B6%26aufirst%3DJ.%26aulast%3DArtursson%26aufirst%3DP.%26atitle%3DEffect%2520of%2520molecular%2520charge%2520on%2520intestinal%2520epithelial%2520drug%2520transport%253A%2520pH-dependent%2520transport%2520of%2520cationic%2520drugs%26jtitle%3DJ.%2520Pharmacol.%2520Exp.Ther.%26date%3D1999%26volume%3D291%26issue%3D2%26spage%3D435%26epage%3D443" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Clark, D. E.</span><span> </span><span class="NLM_article-title"><i>In silico</i> prediction of blood–brain barrier permeation</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x"> (</span><span class="NLM_issue">20</span><span class="NLM_x">) </span> <span class="NLM_fpage">927</span><span class="NLM_x">–</span> <span class="NLM_lpage">933</span><span class="refDoi"> DOI: 10.1016/S1359-6446(03)02827-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.1016%2FS1359-6446%2803%2902827-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=14554156" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BD3sXnvFSiu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2003&pages=927-933&issue=20&author=D.+E.+Clark&title=In+silico+prediction+of+blood%E2%80%93brain+barrier+permeation&doi=10.1016%2FS1359-6446%2803%2902827-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">In silico prediction of blood-brain barrier permeation</span></div><div class="casAuthors">Clark, David E.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">927-933</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">This review examines the progress that is being made towards the in silico prediction of brain permeation.  Following a brief introduction to the blood-brain barrier, the datasets currently available for in silico modeling are discussed.  Recent developments in in silico models of brain permeation are summarized in the context of the current state of the art in prediction accuracy.  An anal. of recent models is presented, focusing on what such models reveal about the mol. properties that det. brain permeation.  The review concludes by presenting the current key issues in this area of research, noting in particular, the paucity of brain permeation data available for modeling.  Finally, possible future directions are suggested.  The degree to which a mol. penetrates the brain can be crucial in drug discovery.  This review discusses progress towards in silico prediction of this important property.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGroQLUXz_-f0LVg90H21EOLACvtfcHk0liDhb1FGVD45Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXnvFSiu70%253D&md5=bc30c4e968c843a644f1147f90001261</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2FS1359-6446%2803%2902827-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1359-6446%252803%252902827-7%26sid%3Dliteratum%253Aachs%26aulast%3DClark%26aufirst%3DD.%2BE.%26atitle%3DIn%2520silico%2520prediction%2520of%2520blood%25E2%2580%2593brain%2520barrier%2520permeation%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2003%26volume%3D8%26issue%3D20%26spage%3D927%26epage%3D933%26doi%3D10.1016%2FS1359-6446%2803%2902827-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Kelder, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grootenhuis, P. D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bayada, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Delbressine, L. P. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ploemen, J.-P.</span><span> </span><span class="NLM_article-title">Polar molecular surface as a dominating determinant for oral absorption and brain penetration of drugs</span> <span class="citation_source-journal">Pharm. Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x"> (</span><span class="NLM_issue">10</span><span class="NLM_x">) </span> <span class="NLM_fpage">1514</span><span class="NLM_x">–</span> <span class="NLM_lpage">1519</span><span class="refDoi"> DOI: 10.1023/A:1015040217741</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.1023%2FA%3A1015040217741" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=1999&pages=1514-1519&issue=10&author=J.+Kelderauthor=P.+D.+J.+Grootenhuisauthor=D.+M.+Bayadaauthor=L.+P.+C.+Delbressineauthor=J.-P.+Ploemen&title=Polar+molecular+surface+as+a+dominating+determinant+for+oral+absorption+and+brain+penetration+of+drugs&doi=10.1023%2FA%3A1015040217741"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1023%2FA%3A1015040217741&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1015040217741%26sid%3Dliteratum%253Aachs%26aulast%3DKelder%26aufirst%3DJ.%26aulast%3DGrootenhuis%26aufirst%3DP.%2BD.%2BJ.%26aulast%3DBayada%26aufirst%3DD.%2BM.%26aulast%3DDelbressine%26aufirst%3DL.%2BP.%2BC.%26aulast%3DPloemen%26aufirst%3DJ.-P.%26atitle%3DPolar%2520molecular%2520surface%2520as%2520a%2520dominating%2520determinant%2520for%2520oral%2520absorption%2520and%2520brain%2520penetration%2520of%2520drugs%26jtitle%3DPharm.%2520Res.%26date%3D1999%26volume%3D16%26issue%3D10%26spage%3D1514%26epage%3D1519%26doi%3D10.1023%2FA%3A1015040217741" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Wager, T. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hou, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verhoest, P. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Villalobos, A.</span><span> </span><span class="NLM_article-title">A. Moving beyond rules: the development of a central nervous system multiparameter optimization (CNS MPO) approach to enable alignment of druglike properties</span> <span class="citation_source-journal">ACS Chem. Neurosci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x"> (</span><span class="NLM_issue">6</span><span class="NLM_x">) </span> <span class="NLM_fpage">435</span><span class="NLM_x">–</span> <span class="NLM_lpage">449</span><span class="refDoi"> DOI: 10.1021/cn100008c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cn100008c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjvVejtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2010&pages=435-449&issue=6&author=T.+T.+Wagerauthor=X.+Houauthor=P.+R.+Verhoestauthor=A.+Villalobos&title=A.+Moving+beyond+rules%3A+the+development+of+a+central+nervous+system+multiparameter+optimization+%28CNS+MPO%29+approach+to+enable+alignment+of+druglike+properties&doi=10.1021%2Fcn100008c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Moving beyond Rules: The Development of a Central Nervous System Multiparameter Optimization (CNS MPO) Approach To Enable Alignment of Druglike Properties</span></div><div class="casAuthors">Wager, Travis T.; Hou, Xinjun; Verhoest, Patrick R.; Villalobos, Anabella</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Neuroscience</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">435-449</span>CODEN:
                <span class="NLM_cas:coden">ACNCDM</span>;
        ISSN:<span class="NLM_cas:issn">1948-7193</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The interplay among commonly used physicochem. properties in drug design was examd. and utilized to create a prospective design tool focused on the alignment of key druglike attributes.  Using a set of six physicochem. parameters ((a) lipophilicity, calcd. partition coeff. (ClogP); (b) calcd. distribution coeff. at pH = 7.4 (ClogD); (c) mol. wt. (MW); (d) topol. polar surface area (TPSA); (e) no. of hydrogen bond donors (HBD); (f) most basic center (pKa)), a druglikeness central nervous system multiparameter optimization (CNS MPO) algorithm was built and applied to a set of marketed CNS drugs (N = 119) and Pfizer CNS candidates (N = 108), as well as to a large diversity set of Pfizer proprietary compds. (N = 11 303).  The novel CNS MPO algorithm showed that 74% of marketed CNS drugs displayed a high CNS MPO score (MPO desirability score ≥ 4, using a scale of 0-6), in comparison to 60% of the Pfizer CNS candidates.  This anal. suggests that this algorithm could potentially be used to identify compds. with a higher probability of successfully testing hypotheses in the clinic.  In addn., a relationship between an increasing CNS MPO score and alignment of key in vitro attributes of drug discovery (favorable permeability, P-glycoprotein (P-gp) efflux, metabolic stability, and safety) was seen in the marketed CNS drug set, the Pfizer candidate set, and the Pfizer proprietary diversity set.  The CNS MPO scoring function offers advantages over hard cutoffs or utilization of single parameters to optimize structure-activity relationships (SAR) by expanding medicinal chem. design space through a holistic assessment approach.  Based on six physicochem. properties commonly used by medicinal chemists, the CNS MPO function may be used prospectively at the design stage to accelerate the identification of compds. with increased probability of success.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqB3Mvkgm_X5bVg90H21EOLACvtfcHk0liDhb1FGVD45Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjvVejtr8%253D&md5=e94998f65a420be6ee4dec4a987cb983</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Fcn100008c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcn100008c%26sid%3Dliteratum%253Aachs%26aulast%3DWager%26aufirst%3DT.%2BT.%26aulast%3DHou%26aufirst%3DX.%26aulast%3DVerhoest%26aufirst%3DP.%2BR.%26aulast%3DVillalobos%26aufirst%3DA.%26atitle%3DA.%2520Moving%2520beyond%2520rules%253A%2520the%2520development%2520of%2520a%2520central%2520nervous%2520system%2520multiparameter%2520optimization%2520%2528CNS%2520MPO%2529%2520approach%2520to%2520enable%2520alignment%2520of%2520druglike%2520properties%26jtitle%3DACS%2520Chem.%2520Neurosci.%26date%3D2010%26volume%3D1%26issue%3D6%26spage%3D435%26epage%3D449%26doi%3D10.1021%2Fcn100008c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Lipinski, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lombardo, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dominy, B. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feeney, P. J.</span><span> </span><span class="NLM_article-title">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span> <span class="citation_source-journal">Adv. Drug Delivery Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x"> (</span><span class="NLM_issue">1–3</span><span class="NLM_x">) </span> <span class="NLM_fpage">3</span><span class="NLM_x">–</span> <span class="NLM_lpage">25</span><span class="refDoi"> DOI: 10.1016/S0169-409X(96)00423-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.1016%2FS0169-409X%2896%2900423-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADyaK2sXktlKlsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=1997&pages=3-25&issue=1%E2%80%933&author=C.+A.+Lipinskiauthor=F.+Lombardoauthor=B.+W.+Dominyauthor=P.+J.+Feeney&title=Experimental+and+computational+approaches+to+estimate+solubility+and+permeability+in+drug+discovery+and+development+settings&doi=10.1016%2FS0169-409X%2896%2900423-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span></div><div class="casAuthors">Lipinski, Christopher A.; Lombardo, Franco; Dominy, Beryl W.; Feeney, Paul J.</div><div class="citationInfo"><span class="NLM_cas:title">Advanced Drug Delivery Reviews</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">3-25</span>CODEN:
                <span class="NLM_cas:coden">ADDREP</span>;
        ISSN:<span class="NLM_cas:issn">0169-409X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review with 50 refs.  Exptl. and computational approaches to est. soly. and permeability in discovery and development settings are described.  In the discovery setting 'the rule of 5' predicts that poor absorption or permeation is more likely when there are more than 5 H-bond donors, 10 H-bond acceptors, the mol. wt. (MWT) is >500 and the calcd. Log P (CLogP) is >5.  Computational methodol. for the rule-based Moriguchi Log P (MLogP) calcn. is described.  Turbidimetric soly. measurement is described and applied to known drugs.  High throughput screening (HTS) leads tend to have higher MWT and Log P and lower turbidimetric soly. than leads in the pre-HTS era.  In the development setting, soly. calcns. focus on exact value prediction and are difficult because of polymorphism.  Recent work on linear free energy relationships and Log P approaches are critically reviewed.  Useful predictions are possible in closely related analog series when coupled with exptl. thermodn. soly. measurements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGon9gEeZ7cboLVg90H21EOLACvtfcHk0lixdB5jHenTrw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXktlKlsQ%253D%253D&md5=405f70b0594d428f1275e1d56642cd3a</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2FS0169-409X%2896%2900423-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0169-409X%252896%252900423-1%26sid%3Dliteratum%253Aachs%26aulast%3DLipinski%26aufirst%3DC.%2BA.%26aulast%3DLombardo%26aufirst%3DF.%26aulast%3DDominy%26aufirst%3DB.%2BW.%26aulast%3DFeeney%26aufirst%3DP.%2BJ.%26atitle%3DExperimental%2520and%2520computational%2520approaches%2520to%2520estimate%2520solubility%2520and%2520permeability%2520in%2520drug%2520discovery%2520and%2520development%2520settings%26jtitle%3DAdv.%2520Drug%2520Delivery%2520Rev.%26date%3D1997%26volume%3D23%26issue%3D1%25E2%2580%25933%26spage%3D3%26epage%3D25%26doi%3D10.1016%2FS0169-409X%2896%2900423-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Hill, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, R. J.</span><span> </span><span class="NLM_article-title">Getting physical in drug discovery: a contemporary perspective on solubility and hydrophobicity</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x"> (</span><span class="NLM_issue">15–16</span><span class="NLM_x">) </span> <span class="NLM_fpage">648</span><span class="NLM_x">–</span> <span class="NLM_lpage">655</span><span class="refDoi"> DOI: 10.1016/j.drudis.2010.05.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.1016%2Fj.drudis.2010.05.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=20570751" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVeksb3P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2010&pages=648-655&issue=15%E2%80%9316&author=A.+P.+Hillauthor=R.+J.+Young&title=Getting+physical+in+drug+discovery%3A+a+contemporary+perspective+on+solubility+and+hydrophobicity&doi=10.1016%2Fj.drudis.2010.05.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Getting physical in drug discovery: a contemporary perspective on solubility and hydrophobicity</span></div><div class="casAuthors">Hill, Alan P.; Young, Robert J.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">15/16</span>),
    <span class="NLM_cas:pages">648-655</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Suboptimal phys. properties have been identified as a particular shortcoming of compds. in contemporary drug discovery, contributing to high attrition levels.  An anal. of the relationship between hydrophobicity (calcd. and measured) and ∼100 k measured kinetic soly. values has been undertaken.  In line with the General Soly. Equation, ests. of hydrophobicity, particularly ACD c log D pH7.4, give a useful indication of the likely soly. classification of particular mols.  Taking ACD c log D pH7.4 values together with the no. of arom. rings in a given mol. provides enhanced prediction.  The Soly. Forecast Index' (SFI = c log D pH7.4 + #Ar) is proposed as a simple, yet effective, guide to predicting soly.  Moreover, anal. of measured distribution/partition coeff. values highlighted statistically significant shortcomings in the applicability of octanol/water as a model system for hydrophobicity detn. with poorly sol. compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdtQyfNjbFUrVg90H21EOLACvtfcHk0lixdB5jHenTrw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVeksb3P&md5=7b201081c5586a6ea4b7af00d507d462</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2010.05.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2010.05.016%26sid%3Dliteratum%253Aachs%26aulast%3DHill%26aufirst%3DA.%2BP.%26aulast%3DYoung%26aufirst%3DR.%2BJ.%26atitle%3DGetting%2520physical%2520in%2520drug%2520discovery%253A%2520a%2520contemporary%2520perspective%2520on%2520solubility%2520and%2520hydrophobicity%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2010%26volume%3D15%26issue%3D15%25E2%2580%259316%26spage%3D648%26epage%3D655%26doi%3D10.1016%2Fj.drudis.2010.05.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Stouch, T. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kenyon, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X.-Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doweyko, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.</span><span> </span><span class="NLM_article-title"><i>In silico</i> ADME/TOX: why models fail</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">83</span><span class="NLM_x">–</span> <span class="NLM_lpage">92</span><span class="refDoi"> DOI: 10.1023/A:1025358319677</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.1023%2FA%3A1025358319677" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2003&pages=83-92&issue=2&author=T.+R.+Stouchauthor=J.+R.+Kenyonauthor=S.+R.+Johnsonauthor=X.-Q.+Chenauthor=A.+Doweykoauthor=Y.+Li&title=In+silico+ADME%2FTOX%3A+why+models+fail&doi=10.1023%2FA%3A1025358319677"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1023%2FA%3A1025358319677&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1025358319677%26sid%3Dliteratum%253Aachs%26aulast%3DStouch%26aufirst%3DT.%2BR.%26aulast%3DKenyon%26aufirst%3DJ.%2BR.%26aulast%3DJohnson%26aufirst%3DS.%2BR.%26aulast%3DChen%26aufirst%3DX.-Q.%26aulast%3DDoweyko%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DY.%26atitle%3DIn%2520silico%2520ADME%252FTOX%253A%2520why%2520models%2520fail%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2003%26volume%3D17%26issue%3D2%26spage%3D83%26epage%3D92%26doi%3D10.1023%2FA%3A1025358319677" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Larregieu, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benet, L. Z.</span><span> </span><span class="NLM_article-title">Drug Discovery and regulatory considerations for improving <i>in silico</i> and <i>in vitro</i> predictions that use Caco-2 as a surrogate for human intestinal permeability measurements</span> <span class="citation_source-journal">AAPS J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">483</span><span class="NLM_x">–</span> <span class="NLM_lpage">497</span><span class="refDoi"> DOI: 10.1208/s12248-013-9456-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.1208%2Fs12248-013-9456-8" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2013&pages=483-497&issue=2&author=C.+A.+Larregieuauthor=L.+Z.+Benet&title=Drug+Discovery+and+regulatory+considerations+for+improving+in+silico+and+in+vitro+predictions+that+use+Caco-2+as+a+surrogate+for+human+intestinal+permeability+measurements&doi=10.1208%2Fs12248-013-9456-8"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1208%2Fs12248-013-9456-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1208%252Fs12248-013-9456-8%26sid%3Dliteratum%253Aachs%26aulast%3DLarregieu%26aufirst%3DC.%2BA.%26aulast%3DBenet%26aufirst%3DL.%2BZ.%26atitle%3DDrug%2520Discovery%2520and%2520regulatory%2520considerations%2520for%2520improving%2520in%2520silico%2520and%2520in%2520vitro%2520predictions%2520that%2520use%2520Caco-2%2520as%2520a%2520surrogate%2520for%2520human%2520intestinal%2520permeability%2520measurements%26jtitle%3DAAPS%2520J.%26date%3D2013%26volume%3D15%26issue%3D2%26spage%3D483%26epage%3D497%26doi%3D10.1208%2Fs12248-013-9456-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Golbraikh, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tropsha, A.</span><span> </span><span class="NLM_article-title">Beware of q2!</span> <span class="citation_source-journal">J. Mol. Graphics Modell.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x"> (</span><span class="NLM_issue">4</span><span class="NLM_x">) </span> <span class="NLM_fpage">269</span><span class="NLM_x">–</span> <span class="NLM_lpage">276</span><span class="refDoi"> DOI: 10.1016/S1093-3263(01)00123-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.1016%2FS1093-3263%2801%2900123-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=11858635" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BD3MXpt1art7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2002&pages=269-276&issue=4&author=A.+Golbraikhauthor=A.+Tropsha&title=Beware+of+q2%21&doi=10.1016%2FS1093-3263%2801%2900123-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Beware of q2!</span></div><div class="casAuthors">Golbraikh, Alexander; Tropsha, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics & Modelling</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">269-276</span>CODEN:
                <span class="NLM_cas:coden">JMGMFI</span>;
        ISSN:<span class="NLM_cas:issn">1093-3263</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Inc.</span>)
        </div><div class="casAbstract">Validation is a crucial aspect of any quant. structure-activity relation (QSAR) modeling.  This paper examines one of the most popular validation criteria, leave-one-out cross-validated R2 (LOO q2).  Often, a high value of this statistical characteristic (q2>0.5) is considered as a proof of the high predictive ability of the model.  In this paper, we show that this assumption is generally incorrect.  In the case of 3D QSAR, the lack of the correlation between the high LOO q2 and the high predictive ability of a QSAR model has been established earlier [Pharm. Acta Helv. 70 (1995) 149; J. Chemomet. 10 (1996) 95; J. Med. Chem. 41 (1998) 2553].  In this paper, we use two-dimensional (2D) mol. descriptors and k nearest neighbors (kNN) QSAR method for the anal. of several datasets.  No correlation between the values of q2 for the training set and predictive ability for the test set was found for any of the datasets.  Thus, the high value of LOO q2 appears to be the necessary but not the sufficient condition for the model to have a high predictive power.  We argue that this is the general property of QSAR models developed using LOO cross-validation.  We emphasize that the external validation is the only way to establish a reliable QSAR model.  We formulate a set of criteria for evaluation of predictive ability of QSAR models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHZLnUmOwha7Vg90H21EOLACvtfcHk0lh0pfx5vt7tGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXpt1art7o%253D&md5=59d2b3412e43241fee262a8cb61cf963</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2FS1093-3263%2801%2900123-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1093-3263%252801%252900123-1%26sid%3Dliteratum%253Aachs%26aulast%3DGolbraikh%26aufirst%3DA.%26aulast%3DTropsha%26aufirst%3DA.%26atitle%3DBeware%2520of%2520q2%2521%26jtitle%3DJ.%2520Mol.%2520Graphics%2520Modell.%26date%3D2002%26volume%3D20%26issue%3D4%26spage%3D269%26epage%3D276%26doi%3D10.1016%2FS1093-3263%2801%2900123-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Novellino, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fattorusso, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greco, G.</span><span> </span><span class="NLM_article-title">Use of comparative molecular field analysis and cluster analysis in series design</span> <span class="citation_source-journal">Pharm. Acta Helv.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">70</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">149</span><span class="NLM_x">–</span> <span class="NLM_lpage">154</span><span class="refDoi"> DOI: 10.1016/0031-6865(95)00014-Z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.1016%2F0031-6865%2895%2900014-Z" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=1995&pages=149-154&issue=2&author=E.+Novellinoauthor=C.+Fattorussoauthor=G.+Greco&title=Use+of+comparative+molecular+field+analysis+and+cluster+analysis+in+series+design&doi=10.1016%2F0031-6865%2895%2900014-Z"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2F0031-6865%2895%2900014-Z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0031-6865%252895%252900014-Z%26sid%3Dliteratum%253Aachs%26aulast%3DNovellino%26aufirst%3DE.%26aulast%3DFattorusso%26aufirst%3DC.%26aulast%3DGreco%26aufirst%3DG.%26atitle%3DUse%2520of%2520comparative%2520molecular%2520field%2520analysis%2520and%2520cluster%2520analysis%2520in%2520series%2520design%26jtitle%3DPharm.%2520Acta%2520Helv.%26date%3D1995%26volume%3D70%26issue%3D2%26spage%3D149%26epage%3D154%26doi%3D10.1016%2F0031-6865%2895%2900014-Z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Alexander, D. L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tropsha, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winkler, D. A.</span><span> </span><span class="NLM_article-title">Beware of R2: simple, unambiguous assessment of the prediction accuracy of QSAR and QSPR models</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x"> (</span><span class="NLM_issue">7</span><span class="NLM_x">) </span> <span class="NLM_fpage">1316</span><span class="NLM_x">–</span> <span class="NLM_lpage">1322</span><span class="refDoi"> DOI: 10.1021/acs.jcim.5b00206</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.5b00206" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVCmtL%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=1316-1322&issue=7&author=D.+L.+J.+Alexanderauthor=A.+Tropshaauthor=D.+A.+Winkler&title=Beware+of+R2%3A+simple%2C+unambiguous+assessment+of+the+prediction+accuracy+of+QSAR+and+QSPR+models&doi=10.1021%2Facs.jcim.5b00206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Beware of R2: Simple, Unambiguous Assessment of the Prediction Accuracy of QSAR and QSPR Models</span></div><div class="casAuthors">Alexander, D. L. J.; Tropsha, A.; Winkler, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1316-1322</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The statistical metrics used to characterize the external predictivity of a model, i.e., how well it predicts the properties of an independent test set, have proliferated over the past decade.  This paper clarifies some apparent confusion over the use of the coeff. of detn., R2, as a measure of model fit and predictive power in QSAR and QSPR modeling.  R2 (or R2) has been used in various contexts in the literature in conjunction with training and test data for both ordinary linear regression and regression through the origin as well as with linear and nonlinear regression models.  The authors analyze the widely adopted model fit criteria suggested by Golbraikh and Tropsha in a strict statistical manner.  Shortcomings in these criteria are identified, and a clearer and simpler alternative method to characterize model predictivity is provided.  The intent is not to repeat the well-documented arguments for model validation using test data but rather to guide the application of R2 as a model fit statistic.  Examples are used to illustrate both correct and incorrect uses of R2.  Reporting the root-mean-square error or equiv. measures of dispersion, which are typically of more practical importance than R2, is also encouraged, and important challenges in addressing the needs of different categories of users such as computational chemists, exptl. scientists, and regulatory decision support specialists are outlined.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGca7UxDqeXLVg90H21EOLACvtfcHk0lh0pfx5vt7tGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVCmtL%252FO&md5=0022bbacc7686da8cbf509b7898470f9</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.5b00206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.5b00206%26sid%3Dliteratum%253Aachs%26aulast%3DAlexander%26aufirst%3DD.%2BL.%2BJ.%26aulast%3DTropsha%26aufirst%3DA.%26aulast%3DWinkler%26aufirst%3DD.%2BA.%26atitle%3DBeware%2520of%2520R2%253A%2520simple%252C%2520unambiguous%2520assessment%2520of%2520the%2520prediction%2520accuracy%2520of%2520QSAR%2520and%2520QSPR%2520models%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26volume%3D55%26issue%3D7%26spage%3D1316%26epage%3D1322%26doi%3D10.1021%2Facs.jcim.5b00206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Sheridan, R. P.</span><span> </span><span class="NLM_article-title">Time-Split Cross-Validation as a method for estimating the goodness of prospective prediction</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x"> (</span><span class="NLM_issue">4</span><span class="NLM_x">) </span> <span class="NLM_fpage">783</span><span class="NLM_x">–</span> <span class="NLM_lpage">790</span><span class="refDoi"> DOI: 10.1021/ci400084k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci400084k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BC3sXksVyjuro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2013&pages=783-790&issue=4&author=R.+P.+Sheridan&title=Time-Split+Cross-Validation+as+a+method+for+estimating+the+goodness+of+prospective+prediction&doi=10.1021%2Fci400084k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Time-Split Cross-Validation as a Method for Estimating the Goodness of Prospective Prediction.</span></div><div class="casAuthors">Sheridan, Robert P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">783-790</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Cross-validation is a common method to validate a QSAR model.  In cross-validation, some compds. are held out as a test set, while the remaining compds. form a training set.  A model is built from the training set, and the test set compds. are predicted on that model.  The agreement of the predicted and obsd. activity values of the test set (measured by, say, R2) is an est. of the self-consistency of the model and is sometimes taken as an indication of the predictivity of the model.  This est. of predictivity can be optimistic or pessimistic compared to true prospective prediction, depending how compds. in the test set are selected.  Here, we show that time-split selection gives an R2 that is more like that of true prospective prediction than the R2 from random selection (too optimistic) or from our analog of leave-class-out selection (too pessimistic).  Time-split selection should be used in addn. to random selection as a std. for cross-validation in QSAR model building.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoc8SkoRQBbrLVg90H21EOLACvtfcHk0lh0pfx5vt7tGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXksVyjuro%253D&md5=638ca6d5ce1b5fecf4928c3db959d344</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fci400084k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci400084k%26sid%3Dliteratum%253Aachs%26aulast%3DSheridan%26aufirst%3DR.%2BP.%26atitle%3DTime-Split%2520Cross-Validation%2520as%2520a%2520method%2520for%2520estimating%2520the%2520goodness%2520of%2520prospective%2520prediction%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2013%26volume%3D53%26issue%3D4%26spage%3D783%26epage%3D790%26doi%3D10.1021%2Fci400084k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Chen, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sheridan, R. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hornak, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Voigt, J. H.</span><span> </span><span class="NLM_article-title">Comparison of random forest and Pipeline Pilot Naıvë Bayes in prospective QSAR predictions</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x"> (</span><span class="NLM_issue">3</span><span class="NLM_x">) </span> <span class="NLM_fpage">792</span><span class="NLM_x">–</span> <span class="NLM_lpage">803</span><span class="refDoi"> DOI: 10.1021/ci200615h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci200615h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BC38XislajtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=792-803&issue=3&author=B.+Chenauthor=R.+P.+Sheridanauthor=V.+Hornakauthor=J.+H.+Voigt&title=Comparison+of+random+forest+and+Pipeline+Pilot+Na%C4%B1v%C3%AB+Bayes+in+prospective+QSAR+predictions&doi=10.1021%2Fci200615h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of Random Forest and Pipeline Pilot Naive Bayes in Prospective QSAR Predictions</span></div><div class="casAuthors">Chen, Bin; Sheridan, Robert P.; Hornak, Viktor; Voigt, Johannes H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">792-803</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Random forest is currently considered one of the best QSAR methods available in terms of accuracy of prediction.  However, it is computationally intensive.  Naive Bayes is a simple, robust classification method.  The Laplacian-modified Naive Bayes implementation is the preferred QSAR method in the widely used com. chemoinformatics platform Pipeline Pilot.  We made a comparison of the ability of Pipeline Pilot Naive Bayes (PLPNB) and random forest to make accurate predictions on 18 large, diverse inhouse QSAR data sets.  These include on-target and ADME-related activities.  These data sets were set up as classification problems with either binary or multicategory activities.  We used a time-split method of dividing training and test sets, as we feel this is a realistic way of simulating prospective prediction.  PLPNB is computationally efficient.  However, random forest predictions are at least as good and in many cases significantly better than those of PLPNB on our data sets.  PLPNB performs better with ECFP4 and ECFP6 descriptors, which are native to Pipeline Pilot, and more poorly with other descriptors we tried.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopM-mDzNRvcLVg90H21EOLACvtfcHk0lgfaWKysoafYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XislajtbY%253D&md5=63488794c7eebebed1f34c2e6989e104</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fci200615h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci200615h%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DB.%26aulast%3DSheridan%26aufirst%3DR.%2BP.%26aulast%3DHornak%26aufirst%3DV.%26aulast%3DVoigt%26aufirst%3DJ.%2BH.%26atitle%3DComparison%2520of%2520random%2520forest%2520and%2520Pipeline%2520Pilot%2520Na%25C4%25B1v%25C3%25AB%2520Bayes%2520in%2520prospective%2520QSAR%2520predictions%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26issue%3D3%26spage%3D792%26epage%3D803%26doi%3D10.1021%2Fci200615h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Berellini, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Springer, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waters, N. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lombardo, F.</span><span> </span><span class="NLM_article-title"><i>In silico</i> Prediction of volume of distribution in human using linear and nonlinear models on a 669 compound data set</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x"> (</span><span class="NLM_issue">14</span><span class="NLM_x">) </span> <span class="NLM_fpage">4488</span><span class="NLM_x">–</span> <span class="NLM_lpage">4495</span><span class="refDoi"> DOI: 10.1021/jm9004658</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm9004658" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=4488-4495&issue=14&author=G.+Berelliniauthor=C.+Springerauthor=N.+J.+Watersauthor=F.+Lombardo&title=In+silico+Prediction+of+volume+of+distribution+in+human+using+linear+and+nonlinear+models+on+a+669+compound+data+set&doi=10.1021%2Fjm9004658"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fjm9004658&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm9004658%26sid%3Dliteratum%253Aachs%26aulast%3DBerellini%26aufirst%3DG.%26aulast%3DSpringer%26aufirst%3DC.%26aulast%3DWaters%26aufirst%3DN.%2BJ.%26aulast%3DLombardo%26aufirst%3DF.%26atitle%3DIn%2520silico%2520Prediction%2520of%2520volume%2520of%2520distribution%2520in%2520human%2520using%2520linear%2520and%2520nonlinear%2520models%2520on%2520a%2520669%2520compound%2520data%2520set%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26issue%3D14%26spage%3D4488%26epage%3D4495%26doi%3D10.1021%2Fjm9004658" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Lombardo, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Obach, R. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varma, M. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stringer, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berellini, B.</span><span> </span><span class="NLM_article-title">Clearance mechanism assignment and total clearance prediction in human based upon <i>in silico</i> models</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x"> (</span><span class="NLM_issue">10</span><span class="NLM_x">) </span> <span class="NLM_fpage">4397</span><span class="NLM_x">–</span> <span class="NLM_lpage">4405</span><span class="refDoi"> DOI: 10.1021/jm500436v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm500436v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=4397-4405&issue=10&author=F.+Lombardoauthor=R.+S.+Obachauthor=M.+V.+Varmaauthor=R.+Stringerauthor=B.+Berellini&title=Clearance+mechanism+assignment+and+total+clearance+prediction+in+human+based+upon+in+silico+models&doi=10.1021%2Fjm500436v"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fjm500436v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm500436v%26sid%3Dliteratum%253Aachs%26aulast%3DLombardo%26aufirst%3DF.%26aulast%3DObach%26aufirst%3DR.%2BS.%26aulast%3DVarma%26aufirst%3DM.%2BV.%26aulast%3DStringer%26aufirst%3DR.%26aulast%3DBerellini%26aufirst%3DB.%26atitle%3DClearance%2520mechanism%2520assignment%2520and%2520total%2520clearance%2520prediction%2520in%2520human%2520based%2520upon%2520in%2520silico%2520models%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26issue%3D10%26spage%3D4397%26epage%3D4405%26doi%3D10.1021%2Fjm500436v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Martin, T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harten, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muratov, E. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Golbraikh, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tropsha, A.</span><span> </span><span class="NLM_article-title">Does rational selection of training and test sets improve the outcome of QSAR modeling</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x"> (</span><span class="NLM_issue">10</span><span class="NLM_x">) </span> <span class="NLM_fpage">2570</span><span class="NLM_x">–</span> <span class="NLM_lpage">2578</span><span class="refDoi"> DOI: 10.1021/ci300338w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci300338w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVCgtrjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=2570-2578&issue=10&author=T.+M.+Martinauthor=P.+Hartenauthor=D.+M.+Youngauthor=E.+N.+Muratovauthor=A.+Golbraikhauthor=H.+Zhuauthor=A.+Tropsha&title=Does+rational+selection+of+training+and+test+sets+improve+the+outcome+of+QSAR+modeling&doi=10.1021%2Fci300338w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Does Rational Selection of Training and Test Sets Improve the Outcome of QSAR Modeling?</span></div><div class="casAuthors">Martin, Todd M.; Harten, Paul; Young, Douglas M.; Muratov, Eugene N.; Golbraikh, Alexander; Zhu, Hao; Tropsha, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2570-2578</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Prior to using a quant. structure activity relationship (QSAR) model for external predictions, its predictive power should be established and validated.  In the absence of a true external data set, the best way to validate the predictive ability of a model is to perform its statistical external validation.  In statistical external validation, the overall data set is divided into training and test sets.  Commonly, this splitting is performed using random division.  Rational splitting methods can divide data sets into training and test sets in an intelligent fashion.  The purpose of this study was to det. whether rational division methods lead to more predictive models compared to random division.  A special data splitting procedure was used to facilitate the comparison between random and rational division methods.  For each toxicity end point, the overall data set was divided into a modeling set (80% of the overall set) and an external evaluation set (20% of the overall set) using random division.  The modeling set was then subdivided into a training set (80% of the modeling set) and a test set (20% of the modeling set) using rational division methods and by using random division.  The Kennard-Stone, minimal test set dissimilarity, and sphere exclusion algorithms were used as the rational division methods.  The hierarchical clustering, random forest, and k-nearest neighbor (kNN) methods were used to develop QSAR models based on the training sets.  For kNN QSAR, multiple training and test sets were generated, and multiple QSAR models were built.  The results of this study indicate that models based on rational division methods generate better statistical results for the test sets than models based on random division, but the predictive power of both types of models are comparable.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8M643bY3vHbVg90H21EOLACvtfcHk0liPSZLj7pYyBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVCgtrjL&md5=d1afc25036cbaf9bc43e830c57df7f99</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fci300338w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci300338w%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DT.%2BM.%26aulast%3DHarten%26aufirst%3DP.%26aulast%3DYoung%26aufirst%3DD.%2BM.%26aulast%3DMuratov%26aufirst%3DE.%2BN.%26aulast%3DGolbraikh%26aufirst%3DA.%26aulast%3DZhu%26aufirst%3DH.%26aulast%3DTropsha%26aufirst%3DA.%26atitle%3DDoes%2520rational%2520selection%2520of%2520training%2520and%2520test%2520sets%2520improve%2520the%2520outcome%2520of%2520QSAR%2520modeling%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26issue%3D10%26spage%3D2570%26epage%3D2578%26doi%3D10.1021%2Fci300338w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Lombardo, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Obach, R. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shalaeva, M. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, F.</span><span> </span><span class="NLM_article-title">Prediction of human volume of distribution values for neutral and basic drugs. 2. Extended data seta and leave-class-out statistics</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x"> (</span><span class="NLM_issue">5</span><span class="NLM_x">) </span> <span class="NLM_fpage">1242</span><span class="NLM_x">–</span> <span class="NLM_lpage">1250</span><span class="refDoi"> DOI: 10.1021/jm030408h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm030408h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1242-1250&issue=5&author=F.+Lombardoauthor=R.+S.+Obachauthor=M.+Y.+Shalaevaauthor=F.+Gao&title=Prediction+of+human+volume+of+distribution+values+for+neutral+and+basic+drugs.+2.+Extended+data+seta+and+leave-class-out+statistics&doi=10.1021%2Fjm030408h"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fjm030408h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030408h%26sid%3Dliteratum%253Aachs%26aulast%3DLombardo%26aufirst%3DF.%26aulast%3DObach%26aufirst%3DR.%2BS.%26aulast%3DShalaeva%26aufirst%3DM.%2BY.%26aulast%3DGao%26aufirst%3DF.%26atitle%3DPrediction%2520of%2520human%2520volume%2520of%2520distribution%2520values%2520for%2520neutral%2520and%2520basic%2520drugs.%25202.%2520Extended%2520data%2520seta%2520and%2520leave-class-out%2520statistics%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26issue%3D5%26spage%3D1242%26epage%3D1250%26doi%3D10.1021%2Fjm030408h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">Clark, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lawless, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fraczkiewicz, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waldman, M.</span><span> </span><span class="NLM_article-title">Using bet binomials to estimate classification uncertainty for ensemble models</span> <span class="citation_source-journal">J. Cheminf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">34</span><span class="refDoi"> DOI: 10.1186/1758-2946-6-34</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.1186%2F1758-2946-6-34" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVCisbbK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&pages=34&author=R.+D.+Clarkauthor=W.+Liangauthor=A.+C.+Leeauthor=M.+S.+Lawlessauthor=R.+Fraczkiewiczauthor=M.+Waldman&title=Using+bet+binomials+to+estimate+classification+uncertainty+for+ensemble+models&doi=10.1186%2F1758-2946-6-34"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Using beta binomials to estimate classification uncertainty for ensemble models</span></div><div class="casAuthors">Clark, Robert D.; Liang, Wenkel; Lee, Adam C.; Lawless, Michael S.; Fraczkiewicz, Robert; Waldman, Marvin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cheminformatics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">34/1-34/19</span>CODEN:
                <span class="NLM_cas:coden">JCOHB3</span>;
        ISSN:<span class="NLM_cas:issn">1758-2946</span>.
    
            (<span class="NLM_cas:orgname">Chemistry Central Ltd.</span>)
        </div><div class="casAbstract">Background: Quant. structure-activity (QSAR) models have enormous potential for reducing drug discovery and development costs as well as the need for animal testing.  Great strides have been made in estg. their overall reliability, but to fully realize that potential, researchers and regulators need to know how confident they can be in individual predictions.  Results: Submodels in an ensemble model which have been trained on different subsets of a shared training pool represent multiple samples of the model space, and the degree of agreement among them contains information on the reliability of ensemble predictions.  For artificial neural network ensembles (ANNEs) using two different methods for detg. ensemble classification - one using vote tallies and the other averaging individual network outputs - we have found that the distribution of predictions across pos. vote tallies can be reasonably well-modeled as a beta binomial distribution, as can the distribution of errors.  Together, these two distributions can be used to est. the probability that a given predictive classification will be in error.  Large data sets comprised of logP, Ames mutagenicity, and CYP2D6 inhibition data are used to illustrate and validate the method.  The distributions of predictions and errors for the training pool accurately predicted the distribution of predictions and errors for large external validation sets, even when the no. of pos. and neg. examples in the training pool were not balanced.  Moreover, the likelihood of a given compd. being prospectively misclassified as a function of the degree of consensus between networks in the ensemble could in most cases be estd. accurately from the fitted beta binomial distributions for the training pool.  Conclusions: Confidence in an individual predictive classification by an ensemble model can be accurately assessed by examg. the distributions of predictions and errors as a function of the degree of agreement among the constituent submodels.  Further, ensemble uncertainty estn. can often be improved by adjusting the voting or classification threshold based on the parameters of the error distribution.  Finally, the profiles for models whose predictive uncertainty ests. are not reliable provide clues to that effect without the need for comparison to an external test set.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrrouzoA8q6LbVg90H21EOLACvtfcHk0liPSZLj7pYyBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVCisbbK&md5=3037c0a4c637b3c7e7e7322b5c98a308</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1186%2F1758-2946-6-34&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1758-2946-6-34%26sid%3Dliteratum%253Aachs%26aulast%3DClark%26aufirst%3DR.%2BD.%26aulast%3DLiang%26aufirst%3DW.%26aulast%3DLee%26aufirst%3DA.%2BC.%26aulast%3DLawless%26aufirst%3DM.%2BS.%26aulast%3DFraczkiewicz%26aufirst%3DR.%26aulast%3DWaldman%26aufirst%3DM.%26atitle%3DUsing%2520bet%2520binomials%2520to%2520estimate%2520classification%2520uncertainty%2520for%2520ensemble%2520models%26jtitle%3DJ.%2520Cheminf.%26date%3D2014%26volume%3D6%26spage%3D34%26doi%3D10.1186%2F1758-2946-6-34" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Roy, K.; Kar, S.; Das, R. N.</span> <span class="citation_source-book">A Primer on QSAR/QSPR Modeling</span>; <span class="NLM_publisher-name">Springer International Publishing</span>: <span class="NLM_publisher-loc">Zurich</span>,<span class="NLM_x"> </span><span class="NLM_year">2015</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.1007%2F978-3-319-17281-1" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=K.+Roy&author=S.+Kar&author=R.+N.+Das&title=A+Primer+on+QSAR%2FQSPR+Modeling"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1007%2F978-3-319-17281-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-3-319-17281-1%26sid%3Dliteratum%253Aachs%26aulast%3DRoy%26aufirst%3DK.%26btitle%3DA%2520Primer%2520on%2520QSAR%252FQSPR%2520Modeling%26pub%3DSpringer%2520International%2520Publishing%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Norinder, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carlsson, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boyer, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eklund, M.</span><span> </span><span class="NLM_article-title">Introducing conformal prediction in predictive modeling. A transparent and flexible alternative to applicability domain determination</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x"> (</span><span class="NLM_issue">6</span><span class="NLM_x">) </span> <span class="NLM_fpage">1596</span><span class="NLM_x">–</span> <span class="NLM_lpage">1603</span><span class="refDoi"> DOI: 10.1021/ci5001168</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci5001168" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BC2cXntlGitrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=1596-1603&issue=6&author=U.+Norinderauthor=L.+Carlssonauthor=S.+Boyerauthor=M.+Eklund&title=Introducing+conformal+prediction+in+predictive+modeling.+A+transparent+and+flexible+alternative+to+applicability+domain+determination&doi=10.1021%2Fci5001168"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Introducing Conformal Prediction in Predictive Modeling. A Transparent and Flexible Alternative to Applicability Domain Determination</span></div><div class="casAuthors">Norinder, Ulf; Carlsson, Lars; Boyer, Scott; Eklund, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1596-1603</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Conformal prediction is introduced as an alternative approach to domain applicability estn.  The advantages of using conformal prediction are as follows: First, the approach is based on a consistent and well-defined math. framework.  Second, the understanding of the confidence level concept in conformal predictions is straightforward, e.g. a confidence level of 0.8 means that the conformal predictor will commit, at most, 20% errors (i.e., true values outside the assigned prediction range).  Third, the confidence level can be varied depending on the situation where the model is to be applied and the consequences of such changes are readily understandable, i.e. prediction ranges are increased or decreased, and the changes can immediately be inspected.  The authors demonstrate the usefulness of conformal prediction by applying it to 10 publicly available data sets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaF64xrBFUp7Vg90H21EOLACvtfcHk0liPSZLj7pYyBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXntlGitrk%253D&md5=1a4adce403989470336e5ca8a23c8738</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fci5001168&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci5001168%26sid%3Dliteratum%253Aachs%26aulast%3DNorinder%26aufirst%3DU.%26aulast%3DCarlsson%26aufirst%3DL.%26aulast%3DBoyer%26aufirst%3DS.%26aulast%3DEklund%26aufirst%3DM.%26atitle%3DIntroducing%2520conformal%2520prediction%2520in%2520predictive%2520modeling.%2520A%2520transparent%2520and%2520flexible%2520alternative%2520to%2520applicability%2520domain%2520determination%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26issue%3D6%26spage%3D1596%26epage%3D1603%26doi%3D10.1021%2Fci5001168" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">Sheridan, R. P.</span><span> </span><span class="NLM_article-title">The relative importance of domain applicability metrics for estimating prediction errors in QSAR varies with training set diversity</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x"> (</span><span class="NLM_issue">6</span><span class="NLM_x">) </span> <span class="NLM_fpage">1098</span><span class="NLM_x">–</span> <span class="NLM_lpage">1107</span><span class="refDoi"> DOI: 10.1021/acs.jcim.5b00110</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.5b00110" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BC2MXoslKrs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=1098-1107&issue=6&author=R.+P.+Sheridan&title=The+relative+importance+of+domain+applicability+metrics+for+estimating+prediction+errors+in+QSAR+varies+with+training+set+diversity&doi=10.1021%2Facs.jcim.5b00110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">The Relative Importance of Domain Applicability Metrics for Estimating Prediction Errors in QSAR Varies with Training Set Diversity</span></div><div class="casAuthors">Sheridan, Robert P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1098-1107</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In QSAR, a statistical model is generated from a training set of mols. (represented by chem. descriptors) and their biol. activities (an "activity model").  The aim of the field of domain applicability (DA) is to est. the uncertainty of prediction of a specific mol. on a specific activity model.  A no. of DA metrics have been proposed in the literature for this purpose.  A quant. model of the prediction uncertainty (an "error model") can be built using one or more of these metrics.  A previous publication from our lab. (Sheridan, R.P.J. Chem. Inf. Model. 2013, 53, 2837-2850) suggested that QSAR methods such as random forest could be used to build error models by fitting unsigned prediction errors against DA metrics.  The QSAR paradigm contains two useful techniques: descriptor importance can det. which DA metrics are most useful, and cross-validation can be used to tell which subset of DA metrics is sufficient to est. the unsigned errors.  Previously we studied 10 large, diverse data sets and seven DA metrics.  For those data sets for which it is possible to build a significant error model from those seven metrics, only two metrics were sufficient to account for almost all of the information in the error model.  These were TREE_SD (the variation of prediction among random forest trees) and PREDICTED (the predicted activity itself).  In this paper we show that when data sets are less diverse, as for example in QSAR models of mols. in a single chem. series, these two DA metrics become less important in explaining prediction error, and the DA metric SIMILARITYNEAREST1 (the similarity of the mol. being predicted to the closest training set compd.) becomes more important.  Our recommendation is that when the mean pairwise similarity (measured with the Carhart AP descriptor and the Dice similarity index) within a QSAR training set is less than 0.5, one can use only TREE_SD, PREDICTED to form the error model, but otherwise one should use TREE_SD, PREDICTED, SIMILARITYNEAREST1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoimBuihjUGHLVg90H21EOLACvtfcHk0lhcF4FD2H44EQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXoslKrs7g%253D&md5=95b7d9f794f575eebeeb1a8c92b2654c</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.5b00110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.5b00110%26sid%3Dliteratum%253Aachs%26aulast%3DSheridan%26aufirst%3DR.%2BP.%26atitle%3DThe%2520relative%2520importance%2520of%2520domain%2520applicability%2520metrics%2520for%2520estimating%2520prediction%2520errors%2520in%2520QSAR%2520varies%2520with%2520training%2520set%2520diversity%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26volume%3D55%26issue%3D6%26spage%3D1098%26epage%3D1107%26doi%3D10.1021%2Facs.jcim.5b00110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group">Keefer, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kauffman, G. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gupta, R. R.</span><span> </span><span class="NLM_article-title">Interpretable, probability-based confidence metric for continuous quantitative structure-activity relationship models</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">368</span><span class="NLM_x">–</span> <span class="NLM_lpage">383</span><span class="refDoi"> DOI: 10.1021/ci300554t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci300554t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1Cit74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2013&pages=368-383&issue=2&author=C.+E.+Keeferauthor=G.+W.+Kauffmanauthor=R.+R.+Gupta&title=Interpretable%2C+probability-based+confidence+metric+for+continuous+quantitative+structure-activity+relationship+models&doi=10.1021%2Fci300554t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Interpretable, Probability-Based Confidence Metric for Continuous Quantitative Structure-Activity Relationship Models</span></div><div class="casAuthors">Keefer, Christopher E.; Kauffman, Gregory W.; Gupta, Rishi Raj</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">368-383</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A great deal of research has gone into the development of robust confidence in prediction and applicability domain (AD) measures for quant. structure-activity relationship (QSAR) models in recent years.  Much of the attention has historically focused on structural similarity, which can be defined in many forms and flavors.  A concept that is frequently overlooked in the realm of the QSAR applicability domain is how the local activity landscape plays a role in how accurate a prediction is or is not.  In this work, we describe an approach that pairs information about both the chem. similarity and activity landscape of a test compd.'s neighborhood into a single calcd. confidence value.  We also present an approach for converting this value into an interpretable confidence metric that has a simple and informative meaning across data sets.  The approach will be introduced to the reader in the context of models built upon four diverse literature data sets.  The steps we will outline include the definition of similarity used to det. nearest neighbors (NN), how we incorporate the NN activity landscape with a similarity-weighted root-mean-square distance (wRMSD) value, and how that value is then calibrated to generate an intuitive confidence metric for prospective application.  Finally, we will illustrate the prospective performance of the approach on five proprietary models whose predictions and confidence metrics have been tracked for more than a year.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqvcorWVQ5O7Vg90H21EOLACvtfcHk0lhcF4FD2H44EQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1Cit74%253D&md5=14d2b7d42f91b5e4e90fa53b6266a178</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Fci300554t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci300554t%26sid%3Dliteratum%253Aachs%26aulast%3DKeefer%26aufirst%3DC.%2BE.%26aulast%3DKauffman%26aufirst%3DG.%2BW.%26aulast%3DGupta%26aufirst%3DR.%2BR.%26atitle%3DInterpretable%252C%2520probability-based%2520confidence%2520metric%2520for%2520continuous%2520quantitative%2520structure-activity%2520relationship%2520models%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2013%26volume%3D53%26issue%3D2%26spage%3D368%26epage%3D383%26doi%3D10.1021%2Fci300554t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group">Gombar, V. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hall, S. D.</span><span> </span><span class="NLM_article-title">Quantitative structure–activity relationship models of clinical pharmacokinetics: clearance and volume of distribution</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x"> (</span><span class="NLM_issue">4</span><span class="NLM_x">) </span> <span class="NLM_fpage">948</span><span class="NLM_x">–</span> <span class="NLM_lpage">957</span><span class="refDoi"> DOI: 10.1021/ci400001u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci400001u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjt1Wltb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2013&pages=948-957&issue=4&author=V.+K.+Gombarauthor=S.+D.+Hall&title=Quantitative+structure%E2%80%93activity+relationship+models+of+clinical+pharmacokinetics%3A+clearance+and+volume+of+distribution&doi=10.1021%2Fci400001u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Quantitative Structure-Activity Relationship Models of Clinical Pharmacokinetics: Clearance and Volume of Distribution</span></div><div class="casAuthors">Gombar, Vijay K.; Hall, Stephen D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">948-957</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Reliable prediction of two fundamental human pharmacokinetic (PK) parameters, systemic clearance (CL) and apparent vol. of distribution (Vd), det. the size and frequency of drug dosing and are at the heart of drug discovery and development.  Traditionally, estd. CL and Vd are derived from preclin. in vitro and in vivo absorption, distribution, metab., and excretion (ADME) measurements.  In this paper, we report quant. structure-activity relationship (QSAR) models for prediction of systemic CL and steady-state Vd (Vdss) from i.v. (iv) dosing in humans.  These QSAR models avoid uncertainty assocd. with preclin.-to-clin. extrapolation and require two-dimensional structure drawing as the sole input.  The clean, uniform training sets for these models were derived from the compilation published by Obach et al.  Models for CL and Vdss were developed using both a support vector regression (SVR) method and a multiple linear regression (MLR) method.  The SVR models employ a min. of 2048-bit fingerprints developed inhouse as structure quantifiers.  The MLR models, on the other hand, are based on information-rich electro-topol. states of two-atom fragments as descriptors and afford reverse QSAR (RQSAR) anal. to help model-guided, in silico modulation of structures for desired CL and Vdss.  The capability of the models to predict iv CL and Vdss with acceptable accuracy was established by randomly splitting data into training and test sets.  On av., for both CL and Vdss, 75% of test compds. were predicted within 2.5-fold of the value obsd. and 90% of test compds. were within 5.0-fold of the value obsd.  The performance of the final models developed from 525 compds. for CL and 569 compds. for Vdss was evaluated on an external set of 56 compds.  The predictions were either better or comparable to those predicted by other in silico models reported in the literature.  To demonstrate the practical application of the RQSAR approach, the structure of vildagliptin, a high-CL and a high-Vdss compd., is modified based on the at. contributions to its predicted CL and Vdss to propose compds. with lower CL and lower Vdss.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppnw75Jax-0LVg90H21EOLACvtfcHk0lhcF4FD2H44EQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjt1Wltb0%253D&md5=f7d5b6c91e646c37b3eed26730ac087a</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fci400001u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci400001u%26sid%3Dliteratum%253Aachs%26aulast%3DGombar%26aufirst%3DV.%2BK.%26aulast%3DHall%26aufirst%3DS.%2BD.%26atitle%3DQuantitative%2520structure%25E2%2580%2593activity%2520relationship%2520models%2520of%2520clinical%2520pharmacokinetics%253A%2520clearance%2520and%2520volume%2520of%2520distribution%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2013%26volume%3D53%26issue%3D4%26spage%3D948%26epage%3D957%26doi%3D10.1021%2Fci400001u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group">Varma, M. V. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lai, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">El-Kattan, A. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Litchfield, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goosen, T. C.</span><span> </span><span class="NLM_article-title">Physicochemical property space of hepatobiliary transport and computational models for predicting rat biliary excretion</span> <span class="citation_source-journal">Drug Metab. Dispos.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x"> (</span><span class="NLM_issue">8</span><span class="NLM_x">) </span> <span class="NLM_fpage">1527</span><span class="NLM_x">–</span> <span class="NLM_lpage">1537</span><span class="refDoi"> DOI: 10.1124/dmd.112.044628</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.1124%2Fdmd.112.044628" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=1527-1537&issue=8&author=M.+V.+S.+Varmaauthor=G.+Changauthor=Y.+Laiauthor=B.+Fengauthor=A.+F.+El-Kattanauthor=J.+Litchfieldauthor=T.+C.+Goosen&title=Physicochemical+property+space+of+hepatobiliary+transport+and+computational+models+for+predicting+rat+biliary+excretion&doi=10.1124%2Fdmd.112.044628"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1124%2Fdmd.112.044628&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.112.044628%26sid%3Dliteratum%253Aachs%26aulast%3DVarma%26aufirst%3DM.%2BV.%2BS.%26aulast%3DChang%26aufirst%3DG.%26aulast%3DLai%26aufirst%3DY.%26aulast%3DFeng%26aufirst%3DB.%26aulast%3DEl-Kattan%26aufirst%3DA.%2BF.%26aulast%3DLitchfield%26aufirst%3DJ.%26aulast%3DGoosen%26aufirst%3DT.%2BC.%26atitle%3DPhysicochemical%2520property%2520space%2520of%2520hepatobiliary%2520transport%2520and%2520computational%2520models%2520for%2520predicting%2520rat%2520biliary%2520excretion%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2012%26volume%3D40%26issue%3D8%26spage%3D1527%26epage%3D1537%26doi%3D10.1124%2Fdmd.112.044628" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group">Lombardo, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waters, N. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Argikar, U. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dennehy, M. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhan, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gunduz, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harriman, S. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berellini, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liric Rajlic, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Obach, R. S.</span><span> </span><span class="NLM_article-title">Comprehensive assessment of human pharmacokinetic prediction based on <i>in vivo</i> animal pharmacokinetic data, part 1: volume of distribution at steady state</span> <span class="citation_source-journal">J. Clin. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">167</span><span class="NLM_x">–</span> <span class="NLM_lpage">177</span><span class="refDoi"> DOI: 10.1177/0091270012440281</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.1177%2F0091270012440281" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=23436262" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1yns7fL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2013&pages=167-177&issue=2&author=F.+Lombardoauthor=N.+J.+Watersauthor=U.+A.+Argikarauthor=M.+K.+Dennehyauthor=J.+Zhanauthor=M.+Gunduzauthor=S.+P.+Harrimanauthor=G.+Berelliniauthor=I.+Liric+Rajlicauthor=R.+S.+Obach&title=Comprehensive+assessment+of+human+pharmacokinetic+prediction+based+on+in+vivo+animal+pharmacokinetic+data%2C+part+1%3A+volume+of+distribution+at+steady+state&doi=10.1177%2F0091270012440281"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive assessment of human pharmacokinetic prediction based on in vivo animal pharmacokinetic data, part 1: volume of distribution at steady state</span></div><div class="casAuthors">Lombardo, Franco; Waters, Nigel J.; Argikar, Upendra A.; Dennehy, Michelle K.; Zhan, Jenny; Gunduz, Mithat; Harriman, Shawn P.; Berellini, Giuliano; Rajlic, Ivana Liric; Obach, R. Scott</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Pharmacology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">167-177</span>CODEN:
                <span class="NLM_cas:coden">JCPCBR</span>;
        ISSN:<span class="NLM_cas:issn">0091-2700</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">The authors present a comprehensive anal. on the estn. of vol. of distribution at steady state (VDss) in human based on rat, dog, and monkey data on nearly 400 compds. for which there are also assocd. human data.  This data set, to the authors' knowledge, is the largest publicly available, has been carefully compiled from literature reports, and was expanded with some inhouse detns. such as plasma protein binding data.  This work offers a good statistical basis for the evaluation of applicable prediction methods, their accuracy, and some methods-dependent diagnostic tools.  The authors also grouped the compds. according to their charge classes and show the applicability of each method considered to each class, offering further insight into the probability of a successful prediction.  Furthermore, they found that the use of fraction unbound in plasma, to obtain unbound vol. of distribution, is generally detrimental to accuracy of several methods, and they discuss possible reasons.  Overall, the approach using dog and monkey data in the Oie-Tozer equation offers the highest probability of success, with an intrinsic diagnostic tool based on aberrant values (<0 or >1) for the calcd. fraction unbound in tissue.  Alternatively, methods based on dog data (single-species scaling) and rat and dog data (Oie-Tozer equation with 2 species or multiple regression methods) may be considered reasonable approaches while not requiring data in nonhuman primates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozX4w_N8RpP7Vg90H21EOLACvtfcHk0lgPPijjiGuEOQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1yns7fL&md5=446a4a31a0fc877b12f174cdb54c3af1</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1177%2F0091270012440281&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F0091270012440281%26sid%3Dliteratum%253Aachs%26aulast%3DLombardo%26aufirst%3DF.%26aulast%3DWaters%26aufirst%3DN.%2BJ.%26aulast%3DArgikar%26aufirst%3DU.%2BA.%26aulast%3DDennehy%26aufirst%3DM.%2BK.%26aulast%3DZhan%26aufirst%3DJ.%26aulast%3DGunduz%26aufirst%3DM.%26aulast%3DHarriman%26aufirst%3DS.%2BP.%26aulast%3DBerellini%26aufirst%3DG.%26aulast%3DLiric%2BRajlic%26aufirst%3DI.%26aulast%3DObach%26aufirst%3DR.%2BS.%26atitle%3DComprehensive%2520assessment%2520of%2520human%2520pharmacokinetic%2520prediction%2520based%2520on%2520in%2520vivo%2520animal%2520pharmacokinetic%2520data%252C%2520part%25201%253A%2520volume%2520of%2520distribution%2520at%2520steady%2520state%26jtitle%3DJ.%2520Clin.%2520Pharmacol.%26date%3D2013%26volume%3D53%26issue%3D2%26spage%3D167%26epage%3D177%26doi%3D10.1177%2F0091270012440281" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group">Lombardo, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waters, N. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Argikar, U. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dennehy, M. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhan, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gunduz, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harriman, S. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berellini, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liric Rajlic, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Obach, R. S.</span><span> </span><span class="NLM_article-title">Comprehensive assessment of human pharmacokinetic prediction based on <i>in vivo</i> animal pharmacokinetic data, part 2: clearance</span> <span class="citation_source-journal">J. Clin. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">178</span><span class="NLM_x">–</span> <span class="NLM_lpage">191</span><span class="refDoi"> DOI: 10.1177/0091270012440282</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.1177%2F0091270012440282" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=23436263" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1ynsL%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2013&pages=178-191&issue=2&author=F.+Lombardoauthor=N.+J.+Watersauthor=U.+A.+Argikarauthor=M.+K.+Dennehyauthor=J.+Zhanauthor=M.+Gunduzauthor=S.+P.+Harrimanauthor=G.+Berelliniauthor=I.+Liric+Rajlicauthor=R.+S.+Obach&title=Comprehensive+assessment+of+human+pharmacokinetic+prediction+based+on+in+vivo+animal+pharmacokinetic+data%2C+part+2%3A+clearance&doi=10.1177%2F0091270012440282"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive assessment of human pharmacokinetic prediction based on in vivo animal pharmacokinetic data, part 2: clearance</span></div><div class="casAuthors">Lombardo, Franco; Waters, Nigel J.; Argikar, Upendra A.; Dennehy, Michelle K.; Zhan, Jenny; Gunduz, Mithat; Harriman, Shawn P.; Berellini, Giuliano; Rajlic, Ivana Liric; Obach, R. Scott</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Pharmacology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">178-191</span>CODEN:
                <span class="NLM_cas:coden">JCPCBR</span>;
        ISSN:<span class="NLM_cas:issn">0091-2700</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A comprehensive anal. on the prediction of human clearance based on i.v. pharmacokinetic data from rat, dog, and monkey for approx. 400 compds. was undertaken.  This data set has been carefully compiled from literature reports and expanded with some inhouse detns. for plasma protein binding and rat clearance.  To the authors' knowledge, this is the largest publicly available data set.  The present examn. offers a comparison of 37 different methods for prediction of human clearance across compds. of diverse physicochem. properties.  Furthermore, this work demonstrates the application of each prediction method to each charge class of the compds., thus presenting an addnl. dimension to prediction of human pharmacokinetics.  In general, the observations suggest that methods employing monkey clearance values and a method incorporating differences in plasma protein binding between rat and human yield the best overall predictions as suggested by approx. 60% compds. within 2-fold geometric mean-fold error.  Other single-species scaling or proportionality methods incorporating the fraction unbound in the corresponding preclin. species for prediction of free clearance in human were generally unsuccessful.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrD0VWrLLqed7Vg90H21EOLACvtfcHk0lgPPijjiGuEOQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1ynsL%252FK&md5=a87b8d0a0025910b592a3e757fb35d54</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1177%2F0091270012440282&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F0091270012440282%26sid%3Dliteratum%253Aachs%26aulast%3DLombardo%26aufirst%3DF.%26aulast%3DWaters%26aufirst%3DN.%2BJ.%26aulast%3DArgikar%26aufirst%3DU.%2BA.%26aulast%3DDennehy%26aufirst%3DM.%2BK.%26aulast%3DZhan%26aufirst%3DJ.%26aulast%3DGunduz%26aufirst%3DM.%26aulast%3DHarriman%26aufirst%3DS.%2BP.%26aulast%3DBerellini%26aufirst%3DG.%26aulast%3DLiric%2BRajlic%26aufirst%3DI.%26aulast%3DObach%26aufirst%3DR.%2BS.%26atitle%3DComprehensive%2520assessment%2520of%2520human%2520pharmacokinetic%2520prediction%2520based%2520on%2520in%2520vivo%2520animal%2520pharmacokinetic%2520data%252C%2520part%25202%253A%2520clearance%26jtitle%3DJ.%2520Clin.%2520Pharmacol.%26date%3D2013%26volume%3D53%26issue%3D2%26spage%3D178%26epage%3D191%26doi%3D10.1177%2F0091270012440282" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group">Poulin, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Theil, F.-P.</span><span> </span><span class="NLM_article-title">Prediction of pharmacokinetics prior to <i>in vivo</i> studies. 1 Mechanism-based prediction of volume of distribution</span> <span class="citation_source-journal">J. Pharm. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">91</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">129</span><span class="NLM_x">–</span> <span class="NLM_lpage">156</span><span class="refDoi"> DOI: 10.1002/jps.10005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.1002%2Fjps.10005" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=91&publication_year=2002&pages=129-156&issue=1&author=P.+Poulinauthor=F.-P.+Theil&title=Prediction+of+pharmacokinetics+prior+to+in+vivo+studies.+1+Mechanism-based+prediction+of+volume+of+distribution&doi=10.1002%2Fjps.10005"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1002%2Fjps.10005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjps.10005%26sid%3Dliteratum%253Aachs%26aulast%3DPoulin%26aufirst%3DP.%26aulast%3DTheil%26aufirst%3DF.-P.%26atitle%3DPrediction%2520of%2520pharmacokinetics%2520prior%2520to%2520in%2520vivo%2520studies.%25201%2520Mechanism-based%2520prediction%2520of%2520volume%2520of%2520distribution%26jtitle%3DJ.%2520Pharm.%2520Sci.%26date%3D2002%26volume%3D91%26issue%3D1%26spage%3D129%26epage%3D156%26doi%3D10.1002%2Fjps.10005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group">Rodgers, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rowland, M.</span><span> </span><span class="NLM_article-title">Mechanistic approaches to volume of distribution predictions: understanding the process</span> <span class="citation_source-journal">Pharm. Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x"> (</span><span class="NLM_issue">5</span><span class="NLM_x">) </span> <span class="NLM_fpage">918</span><span class="NLM_x">–</span> <span class="NLM_lpage">933</span><span class="refDoi"> DOI: 10.1007/s11095-006-9210-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.1007%2Fs11095-006-9210-3" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2007&pages=918-933&issue=5&author=T.+Rodgersauthor=M.+Rowland&title=Mechanistic+approaches+to+volume+of+distribution+predictions%3A+understanding+the+process&doi=10.1007%2Fs11095-006-9210-3"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1007%2Fs11095-006-9210-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs11095-006-9210-3%26sid%3Dliteratum%253Aachs%26aulast%3DRodgers%26aufirst%3DT.%26aulast%3DRowland%26aufirst%3DM.%26atitle%3DMechanistic%2520approaches%2520to%2520volume%2520of%2520distribution%2520predictions%253A%2520understanding%2520the%2520process%26jtitle%3DPharm.%2520Res.%26date%3D2007%26volume%3D24%26issue%3D5%26spage%3D918%26epage%3D933%26doi%3D10.1007%2Fs11095-006-9210-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group">Ward, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, B. R.</span><span> </span><span class="NLM_article-title">A comprehensive quantitative and qualitative evaluation of extrapolation of intravenous pharmacokinetic parameters from rat, dog, and monkey to human. I. Clearance</span> <span class="citation_source-journal">Drug Metab. Dispos.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x"> (</span><span class="NLM_issue">6</span><span class="NLM_x">) </span> <span class="NLM_fpage">603</span><span class="NLM_x">–</span> <span class="NLM_lpage">611</span><span class="refDoi"> DOI: 10.1124/dmd.32.6.603</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.1124%2Fdmd.32.6.603" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2004&pages=603-611&issue=6&author=K.+W.+Wardauthor=B.+R.+Smith&title=A+comprehensive+quantitative+and+qualitative+evaluation+of+extrapolation+of+intravenous+pharmacokinetic+parameters+from+rat%2C+dog%2C+and+monkey+to+human.+I.+Clearance&doi=10.1124%2Fdmd.32.6.603"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1124%2Fdmd.32.6.603&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.32.6.603%26sid%3Dliteratum%253Aachs%26aulast%3DWard%26aufirst%3DK.%2BW.%26aulast%3DSmith%26aufirst%3DB.%2BR.%26atitle%3DA%2520comprehensive%2520quantitative%2520and%2520qualitative%2520evaluation%2520of%2520extrapolation%2520of%2520intravenous%2520pharmacokinetic%2520parameters%2520from%2520rat%252C%2520dog%252C%2520and%2520monkey%2520to%2520human.%2520I.%2520Clearance%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2004%26volume%3D32%26issue%3D6%26spage%3D603%26epage%3D611%26doi%3D10.1124%2Fdmd.32.6.603" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group">Ward, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, B. R.</span><span> </span><span class="NLM_article-title">A comprehensive quantitative and qualitative evaluation of extrapolation of intravenous pharmacokinetic parameters from rat, dog, and monkey to human. II. Volume of distribution and mean residence time</span> <span class="citation_source-journal">Drug Metab. Dispos.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x"> (</span><span class="NLM_issue">6</span><span class="NLM_x">) </span> <span class="NLM_fpage">612</span><span class="NLM_x">–</span> <span class="NLM_lpage">619</span><span class="refDoi"> DOI: 10.1124/dmd.32.6.612</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.1124%2Fdmd.32.6.612" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2004&pages=612-619&issue=6&author=K.+W.+Wardauthor=B.+R.+Smith&title=A+comprehensive+quantitative+and+qualitative+evaluation+of+extrapolation+of+intravenous+pharmacokinetic+parameters+from+rat%2C+dog%2C+and+monkey+to+human.+II.+Volume+of+distribution+and+mean+residence+time&doi=10.1124%2Fdmd.32.6.612"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1124%2Fdmd.32.6.612&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.32.6.612%26sid%3Dliteratum%253Aachs%26aulast%3DWard%26aufirst%3DK.%2BW.%26aulast%3DSmith%26aufirst%3DB.%2BR.%26atitle%3DA%2520comprehensive%2520quantitative%2520and%2520qualitative%2520evaluation%2520of%2520extrapolation%2520of%2520intravenous%2520pharmacokinetic%2520parameters%2520from%2520rat%252C%2520dog%252C%2520and%2520monkey%2520to%2520human.%2520II.%2520Volume%2520of%2520distribution%2520and%2520mean%2520residence%2520time%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2004%26volume%3D32%26issue%3D6%26spage%3D612%26epage%3D619%26doi%3D10.1124%2Fdmd.32.6.612" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group">Tang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mayersohn, M.</span><span> </span><span class="NLM_article-title">A novel method for prediction of human drug clearance by allometric scaling</span> <span class="citation_source-journal">Drug Metab. Dispos.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x"> (</span><span class="NLM_issue">9</span><span class="NLM_x">) </span> <span class="NLM_fpage">1297</span><span class="NLM_x">–</span> <span class="NLM_lpage">1303</span><span class="refDoi"> DOI: 10.1124/dmd.105.004143</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.1124%2Fdmd.105.004143" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2005&pages=1297-1303&issue=9&author=H.+Tangauthor=M.+Mayersohn&title=A+novel+method+for+prediction+of+human+drug+clearance+by+allometric+scaling&doi=10.1124%2Fdmd.105.004143"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1124%2Fdmd.105.004143&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.105.004143%26sid%3Dliteratum%253Aachs%26aulast%3DTang%26aufirst%3DH.%26aulast%3DMayersohn%26aufirst%3DM.%26atitle%3DA%2520novel%2520method%2520for%2520prediction%2520of%2520human%2520drug%2520clearance%2520by%2520allometric%2520scaling%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2005%26volume%3D33%26issue%3D9%26spage%3D1297%26epage%3D1303%26doi%3D10.1124%2Fdmd.105.004143" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group">Page, K. M.</span><span> </span><span class="NLM_article-title">Validation of early human dose prediction: a key metric for compound progression in drug discovery</span> <span class="citation_source-journal">Mol. Pharmaceutics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">609</span><span class="NLM_x">–</span> <span class="NLM_lpage">620</span><span class="refDoi"> DOI: 10.1021/acs.molpharmaceut.5b00840</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.molpharmaceut.5b00840" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVGrs7%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2016&pages=609-620&issue=2&author=K.+M.+Page&title=Validation+of+early+human+dose+prediction%3A+a+key+metric+for+compound+progression+in+drug+discovery&doi=10.1021%2Facs.molpharmaceut.5b00840"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Validation of Early Human Dose Prediction: A Key Metric for Compound Progression in Drug Discovery</span></div><div class="casAuthors">Page, Ken M.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmaceutics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">609-620</span>CODEN:
                <span class="NLM_cas:coden">MPOHBP</span>;
        ISSN:<span class="NLM_cas:issn">1543-8384</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Human dose prediction is increasingly recognized as an important parameter in Drug Discovery.  Validation of a method using only in vitro and predicted parameters incorporated into a PK model was undertaken by predicting human dose and free Cmax for a no. of marketed drugs and AZ Development compds.  Doses were compared to those most relevant to marketed drugs or to clin. administered doses of AZ compds. normalized either to predicted Cmin or Cmax values.  Av. (AFE) and abs. av. (AAFE) fold-error anal. showed that best predictions were obtained using a QSAR model as the source of Vss, with Fabs set to 1 for acids and 0.5 for all other ion classes; for clearance prediction no binding correction to the well stirred model (WSM) was used for bases, while it was set to Fup/Fup0.5 for all other ion classes.  Using this combination of methods, predicted doses for 45 to 68% of the Cmin- and Cmax-normalized and marketed drug data sets were within 3-fold of the obsd. values, while 82 to 92% of these data sets were within 10-fold.  This method for early human dose prediction is able to rank, identify, and flag risks or optimization opportunities for future development compds. within 10 days of first synthesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoT--CfNq9EGLVg90H21EOLACvtfcHk0lik-v7jJVan9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVGrs7%252FO&md5=612ece6c4e2729ab4694b0ae168e4cba</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Facs.molpharmaceut.5b00840&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.molpharmaceut.5b00840%26sid%3Dliteratum%253Aachs%26aulast%3DPage%26aufirst%3DK.%2BM.%26atitle%3DValidation%2520of%2520early%2520human%2520dose%2520prediction%253A%2520a%2520key%2520metric%2520for%2520compound%2520progression%2520in%2520drug%2520discovery%26jtitle%3DMol.%2520Pharmaceutics%26date%3D2016%26volume%3D13%26issue%3D2%26spage%3D609%26epage%3D620%26doi%3D10.1021%2Facs.molpharmaceut.5b00840" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group">Jones, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rowland, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gibson, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yates, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chien, J. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ring, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adkison, K. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ku, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vuppugalla, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marathe, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fischer, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dutta, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sinha, V. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bjornsson, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lavé, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poulin, P.</span><span> </span><span class="NLM_article-title">PhARMA CPCDC initiative on predictive models or human pharmacokinetics, part 2: Comparative assessment of predictions methods of human volume of distribution</span> <span class="citation_source-journal">J. Pharm. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">100</span><span class="NLM_x"> (</span><span class="NLM_issue">10</span><span class="NLM_x">) </span> <span class="NLM_fpage">4074</span><span class="NLM_x">–</span> <span class="NLM_lpage">4089</span><span class="refDoi"> DOI: 10.1002/jps.22553</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.1002%2Fjps.22553" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=100&publication_year=2011&pages=4074-4089&issue=10&author=R.+D.+Jonesauthor=H.+M.+Jonesauthor=M.+Rowlandauthor=C.+R.+Gibsonauthor=J.+W.+Yatesauthor=J.+Y.+Chienauthor=B.+J.+Ringauthor=K.+K.+Adkisonauthor=M.+S.+Kuauthor=H.+Heauthor=R.+Vuppugallaauthor=P.+Maratheauthor=V.+Fischerauthor=S.+Duttaauthor=V.+K.+Sinhaauthor=T.+Bjornssonauthor=T.+Lav%C3%A9author=P.+Poulin&title=PhARMA+CPCDC+initiative+on+predictive+models+or+human+pharmacokinetics%2C+part+2%3A+Comparative+assessment+of+predictions+methods+of+human+volume+of+distribution&doi=10.1002%2Fjps.22553"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1002%2Fjps.22553&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjps.22553%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DR.%2BD.%26aulast%3DJones%26aufirst%3DH.%2BM.%26aulast%3DRowland%26aufirst%3DM.%26aulast%3DGibson%26aufirst%3DC.%2BR.%26aulast%3DYates%26aufirst%3DJ.%2BW.%26aulast%3DChien%26aufirst%3DJ.%2BY.%26aulast%3DRing%26aufirst%3DB.%2BJ.%26aulast%3DAdkison%26aufirst%3DK.%2BK.%26aulast%3DKu%26aufirst%3DM.%2BS.%26aulast%3DHe%26aufirst%3DH.%26aulast%3DVuppugalla%26aufirst%3DR.%26aulast%3DMarathe%26aufirst%3DP.%26aulast%3DFischer%26aufirst%3DV.%26aulast%3DDutta%26aufirst%3DS.%26aulast%3DSinha%26aufirst%3DV.%2BK.%26aulast%3DBjornsson%26aufirst%3DT.%26aulast%3DLav%25C3%25A9%26aufirst%3DT.%26aulast%3DPoulin%26aufirst%3DP.%26atitle%3DPhARMA%2520CPCDC%2520initiative%2520on%2520predictive%2520models%2520or%2520human%2520pharmacokinetics%252C%2520part%25202%253A%2520Comparative%2520assessment%2520of%2520predictions%2520methods%2520of%2520human%2520volume%2520of%2520distribution%26jtitle%3DJ.%2520Pharm.%2520Sci.%26date%3D2011%26volume%3D100%26issue%3D10%26spage%3D4074%26epage%3D4089%26doi%3D10.1002%2Fjps.22553" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group">Lee, M.-L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aliagas, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dotson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gobbi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heffron, T.</span><span> </span><span class="NLM_article-title">DEGAS: sharing and tracking target compound ideas with external collaborators</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">278</span><span class="NLM_x">–</span> <span class="NLM_lpage">284</span><span class="refDoi"> DOI: 10.1021/ci2003297</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci2003297" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVGru73N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=278-284&issue=2&author=M.-L.+Leeauthor=I.+Aliagasauthor=J.+Dotsonauthor=J.+A.+Fengauthor=A.+Gobbiauthor=T.+Heffron&title=DEGAS%3A+sharing+and+tracking+target+compound+ideas+with+external+collaborators&doi=10.1021%2Fci2003297"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">DEGAS: Sharing and Tracking Target Compound Ideas with External Collaborators</span></div><div class="casAuthors">Lee, Man-Ling; Aliagas, Ignacio; Dotson, Jennafer; Feng, Jianwen A.; Gobbi, Alberto; Heffron, Timothy</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">278-284</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">To minimize the risk of failure in clin. trials, drug discovery teams must propose active and selective clin. candidates with good physicochem. properties.  An addnl. challenge is that today drug discovery is often conducted by teams at different geog. locations.  To improve the collaborative decision making on which compds. to synthesize, we have implemented DEGAS, an application which enables scientists from Genentech and from collaborating external partners to instantly access the same data.  DEGAS was implemented to ensure that only the best target compds. are made and that they are made without duplicate effort.  Physicochem. properties and DMPK model predictions are computed for each compd. to allow the team to make informed decisions when prioritizing.  The synthesis progress can be easily tracked.  While developing DEGAS, ease of use was a particular goal in order to minimize the difficulty of training and supporting remote users.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWoQ8pIpOo0rVg90H21EOLACvtfcHk0lgTLZ52M6yxqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVGru73N&md5=8539de9cc3ea25049905b94774292e0f</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Fci2003297&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci2003297%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DM.-L.%26aulast%3DAliagas%26aufirst%3DI.%26aulast%3DDotson%26aufirst%3DJ.%26aulast%3DFeng%26aufirst%3DJ.%2BA.%26aulast%3DGobbi%26aufirst%3DA.%26aulast%3DHeffron%26aufirst%3DT.%26atitle%3DDEGAS%253A%2520sharing%2520and%2520tracking%2520target%2520compound%2520ideas%2520with%2520external%2520collaborators%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26issue%3D2%26spage%3D278%26epage%3D284%26doi%3D10.1021%2Fci2003297" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span>Spotfire. <a href="http://spotfire.tibco.com/" class="extLink">http://spotfire.tibco.com/</a> (accessed Aug 15,<span class="NLM_x"> </span><span class="NLM_year">2016</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Spotfire.+http%3A%2F%2Fspotfire.tibco.com%2F+%28accessed+Aug+15%2C+2016%29+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span>MedChem Studio. <a href="http://www.simulations-plus.com/Products.aspx?pID=12" class="extLink">http://www.simulations-plus.com/Products.aspx?pID=12</a> (accessed Aug 15,<span class="NLM_x"> </span><span class="NLM_year">2016</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=MedChem+Studio.+http%3A%2F%2Fwww.simulations-plus.com%2FProducts.aspx%3FpID%3D12+%28accessed+Aug+15%2C+2016%29+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span>Vortex. <a href="http://www.dotmatics.com/products/vortex/" class="extLink">http://www.dotmatics.com/products/vortex/</a> (accessed Aug 15,<span class="NLM_x"> </span><span class="NLM_year">2016</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Vortex.+http%3A%2F%2Fwww.dotmatics.com%2Fproducts%2Fvortex%2F+%28accessed+Aug+15%2C+2016%29+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group">Nissink, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Degorce, S.</span><span> </span><span class="NLM_article-title">Analyzing compound and project progress through multi-objective-based compound quality assessment</span> <span class="citation_source-journal">Future Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x"> (</span><span class="NLM_issue">7</span><span class="NLM_x">) </span> <span class="NLM_fpage">753</span><span class="NLM_x">–</span> <span class="NLM_lpage">767</span><span class="refDoi"> DOI: 10.4155/fmc.13.45</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.4155%2Ffmc.13.45" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=23651090" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BC3sXntFyrsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2013&pages=753-767&issue=7&author=J.+W.+Nissinkauthor=S.+Degorce&title=Analyzing+compound+and+project+progress+through+multi-objective-based+compound+quality+assessment&doi=10.4155%2Ffmc.13.45"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Analyzing compound and project progress through multi-objective-based compound quality assessment</span></div><div class="casAuthors">Nissink, J. Willem M.; Degorce, Sebastien</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">753-767</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">Background: Compd.-quality scoring methods designed to evaluate multiple drug properties concurrently are useful to analyze and prioritize output from drug-design efforts.  However, formalized multiparameter optimization approaches are not widely used in drug design.  Methods: We rank mols. synthesized in drug-discovery projects using simple and aggregated desirability functions reflecting medicinal chem. rules'.  Our quality score deals transparently with missing data, a key requirement in drug-hunting projects where data availability is often limited.  We further est. confidence in the interpretation of such a compd.-quality measure.  Conclusion: Scores and assocd. confidences provide systematic insight in the quality of emerging chem. equity.  Tracking quality of synthetic output over time yields valuable insight into the progress of drug-design teams, with potential applications in risk and resource management of a drug portfolio.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4eLt5NGn-G7Vg90H21EOLACvtfcHk0lgTLZ52M6yxqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXntFyrsrg%253D&md5=892624a14de294a7f80cb3fb87bcea53</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.4155%2Ffmc.13.45&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.13.45%26sid%3Dliteratum%253Aachs%26aulast%3DNissink%26aufirst%3DJ.%2BW.%26aulast%3DDegorce%26aufirst%3DS.%26atitle%3DAnalyzing%2520compound%2520and%2520project%2520progress%2520through%2520multi-objective-based%2520compound%2520quality%2520assessment%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2013%26volume%3D5%26issue%3D7%26spage%3D753%26epage%3D767%26doi%3D10.4155%2Ffmc.13.45" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group">Segall, M. D.</span><span> </span><span class="NLM_article-title">Multi-parameter optimization: Identifying high quality compounds with a balance of properties</span> <span class="citation_source-journal">Curr. Pharm. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x"> (</span><span class="NLM_issue">9</span><span class="NLM_x">) </span> <span class="NLM_fpage">1292</span><span class="NLM_x">–</span> <span class="NLM_lpage">1310</span><span class="refDoi"> DOI: 10.2174/138161212799436430</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.2174%2F138161212799436430" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=22316157" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BC38XlvVSqsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2012&pages=1292-1310&issue=9&author=M.+D.+Segall&title=Multi-parameter+optimization%3A+Identifying+high+quality+compounds+with+a+balance+of+properties&doi=10.2174%2F138161212799436430"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Multi-parameter optimization: identifying high quality compounds with a balance of properties</span></div><div class="casAuthors">Segall, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Current Pharmaceutical Design</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1292-1310</span>CODEN:
                <span class="NLM_cas:coden">CPDEFP</span>;
        ISSN:<span class="NLM_cas:issn">1381-6128</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  A successful, efficacious and safe drug must have a balance of properties, including potency against its intended target, appropriate absorption, distribution, metab., and elimination (ADME) properties and an acceptable safety profile.  Achieving this balance of, often conflicting, requirements is a major challenge in drug discovery.  Approaches to simultaneously optimizing many factors in a design are broadly described under the term "multi-parameter optimization" (MPO).  In this review, we will describe how MPO can be applied to efficiently design and select high quality compds. and describe the range of methods that have been employed in drug discovery, including; simple "rules of thumb" such as Lipinski's rule; desirability functions; Pareto optimization; and probabilistic approaches that take into consideration the uncertainty in all drug discovery data due to predictive error and exptl. variability.  We will explore how these methods have been applied to predicted and exptl. data to reduce attrition and improve the productivity of the drug discovery process.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPFOVyg2ehWbVg90H21EOLACvtfcHk0lgTLZ52M6yxqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XlvVSqsbk%253D&md5=43f90ad751fd2c0bc47715f465ef31b3</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.2174%2F138161212799436430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138161212799436430%26sid%3Dliteratum%253Aachs%26aulast%3DSegall%26aufirst%3DM.%2BD.%26atitle%3DMulti-parameter%2520optimization%253A%2520Identifying%2520high%2520quality%2520compounds%2520with%2520a%2520balance%2520of%2520properties%26jtitle%3DCurr.%2520Pharm.%2520Des.%26date%3D2012%26volume%3D18%26issue%3D9%26spage%3D1292%26epage%3D1310%26doi%3D10.2174%2F138161212799436430" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group">Debe, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mamidipaka, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gregg, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Metz, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gupta, R. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muchmore, S. W.</span><span> </span><span class="NLM_article-title">ALOHA: a novel probability fusion approach for scoring multi-parameter drug-likeness during the lead optimization stage of drug discovery</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">27</span><span class="NLM_x"> (</span><span class="NLM_issue">9</span><span class="NLM_x">) </span> <span class="NLM_fpage">771</span><span class="NLM_x">–</span> <span class="NLM_lpage">782</span><span class="refDoi"> DOI: 10.1007/s10822-013-9679-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.1007%2Fs10822-013-9679-x" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2013&pages=771-782&issue=9&author=D.+A.+Debeauthor=R.+B.+Mamidipakaauthor=R.+J.+Greggauthor=J.+T.+Metzauthor=R.+R.+Guptaauthor=S.+W.+Muchmore&title=ALOHA%3A+a+novel+probability+fusion+approach+for+scoring+multi-parameter+drug-likeness+during+the+lead+optimization+stage+of+drug+discovery&doi=10.1007%2Fs10822-013-9679-x"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1007%2Fs10822-013-9679-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-013-9679-x%26sid%3Dliteratum%253Aachs%26aulast%3DDebe%26aufirst%3DD.%2BA.%26aulast%3DMamidipaka%26aufirst%3DR.%2BB.%26aulast%3DGregg%26aufirst%3DR.%2BJ.%26aulast%3DMetz%26aufirst%3DJ.%2BT.%26aulast%3DGupta%26aufirst%3DR.%2BR.%26aulast%3DMuchmore%26aufirst%3DS.%2BW.%26atitle%3DALOHA%253A%2520a%2520novel%2520probability%2520fusion%2520approach%2520for%2520scoring%2520multi-parameter%2520drug-likeness%2520during%2520the%2520lead%2520optimization%2520stage%2520of%2520drug%2520discovery%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2013%26volume%3D27%26issue%3D9%26spage%3D771%26epage%3D782%26doi%3D10.1007%2Fs10822-013-9679-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group">Hussain, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rea, C.</span><span> </span><span class="NLM_article-title">Computationally efficient algorithm to identify matched molecular pairs (MMPs) in large data sets</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x"> (</span><span class="NLM_issue">3</span><span class="NLM_x">) </span> <span class="NLM_fpage">339</span><span class="NLM_x">–</span> <span class="NLM_lpage">348</span><span class="refDoi"> DOI: 10.1021/ci900450m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci900450m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlWltr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=339-348&issue=3&author=J.+Hussainauthor=C.+Rea&title=Computationally+efficient+algorithm+to+identify+matched+molecular+pairs+%28MMPs%29+in+large+data+sets&doi=10.1021%2Fci900450m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Computationally Efficient Algorithm to Identify Matched Molecular Pairs (MMPs) in Large Data Sets</span></div><div class="casAuthors">Hussain, Jameed; Rea, Ceara</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">339-348</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Modern drug discovery organizations generate large vols. of SAR data.  A promising methodol. that can be used to mine this chem. data to identify novel structure-activity relationships is the matched mol. pair (MMP) methodol.  However, before the full potential of the MMP methodol. can be utilized, a MMP identification method that is capable of identifying all MMPs in large chem. data sets on modest computational hardware is required.  In this paper we report an algorithm that is capable of systematically generating all MMPs in chem. data sets.  Addnl., the algorithm is computationally efficient enough to be applied on large data sets.  As an example the algorithm was used to identify the MMPs in the ∼300k NIH MLSMR set.  The algorithm identified ∼5.3 million matched mol. pairs in the set.  These pairs cover ∼2.6 million unique mol. transformations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoF0rN8jaKSjrVg90H21EOLACvtfcHk0lhArpz5iVQ2jw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlWltr4%253D&md5=23964abebc23274f5c36c319bee52bab</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1021%2Fci900450m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci900450m%26sid%3Dliteratum%253Aachs%26aulast%3DHussain%26aufirst%3DJ.%26aulast%3DRea%26aufirst%3DC.%26atitle%3DComputationally%2520efficient%2520algorithm%2520to%2520identify%2520matched%2520molecular%2520pairs%2520%2528MMPs%2529%2520in%2520large%2520data%2520sets%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26issue%3D3%26spage%3D339%26epage%3D348%26doi%3D10.1021%2Fci900450m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group">Papadatos, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alkarouri, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gillet, V. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willett, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kadirkamanathan, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luscombe, C. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bravi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richmond, N. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pickett, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hussain, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pritchard, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cooper, A. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Macdonald, S. J.</span><span> </span><span class="NLM_article-title">Lead optimization using matched molecular pairs: inclusion of contextual information for enhanced prediction of HERG inhibition, solubility, and lipophilicity</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x"> (</span><span class="NLM_issue">10</span><span class="NLM_x">) </span> <span class="NLM_fpage">1872</span><span class="NLM_x">–</span> <span class="NLM_lpage">1886</span><span class="refDoi"> DOI: 10.1021/ci100258p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci100258p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1Whs7jJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=1872-1886&issue=10&author=G.+Papadatosauthor=M.+Alkarouriauthor=V.+J.+Gilletauthor=P.+Willettauthor=V.+Kadirkamanathanauthor=C.+N.+Luscombeauthor=G.+Braviauthor=N.+J.+Richmondauthor=S.+D.+Pickettauthor=J.+Hussainauthor=J.+M.+Pritchardauthor=A.+W.+Cooperauthor=S.+J.+Macdonald&title=Lead+optimization+using+matched+molecular+pairs%3A+inclusion+of+contextual+information+for+enhanced+prediction+of+HERG+inhibition%2C+solubility%2C+and+lipophilicity&doi=10.1021%2Fci100258p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Lead Optimization Using Matched Molecular Pairs: Inclusion of Contextual Information for Enhanced Prediction of hERG Inhibition, Solubility, and Lipophilicity</span></div><div class="casAuthors">Papadatos, George; Alkarouri, Muhammad; Gillet, Valerie J.; Willett, Peter; Kadirkamanathan, Visakan; Luscombe, Christopher N.; Bravi, Gianpaolo; Richmond, Nicola J.; Pickett, Stephen D.; Hussain, Jameed; Pritchard, John M.; Cooper, Anthony W. J.; MacDonald, Simon J. F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1872-1886</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Previous studies of the anal. of mol. matched pairs (MMPs) have often assumed that the effect of a substructural transformation on a mol. property is independent of the context (i.e., the local structural environment in which that transformation occurs).  Expts. with large sets of hERG, soly., and lipophilicity data demonstrate that the inclusion of contextual information can enhance the predictive power of MMP analyses, with significant trends (both pos. and neg.) being identified that are not apparent when using conventional, context-independent approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLZfW5E3jgX7Vg90H21EOLACvtfcHk0lhArpz5iVQ2jw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht1Whs7jJ&md5=464dff86e047cf1b0273571f82786994</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1021%2Fci100258p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100258p%26sid%3Dliteratum%253Aachs%26aulast%3DPapadatos%26aufirst%3DG.%26aulast%3DAlkarouri%26aufirst%3DM.%26aulast%3DGillet%26aufirst%3DV.%2BJ.%26aulast%3DWillett%26aufirst%3DP.%26aulast%3DKadirkamanathan%26aufirst%3DV.%26aulast%3DLuscombe%26aufirst%3DC.%2BN.%26aulast%3DBravi%26aufirst%3DG.%26aulast%3DRichmond%26aufirst%3DN.%2BJ.%26aulast%3DPickett%26aufirst%3DS.%2BD.%26aulast%3DHussain%26aufirst%3DJ.%26aulast%3DPritchard%26aufirst%3DJ.%2BM.%26aulast%3DCooper%26aufirst%3DA.%2BW.%26aulast%3DMacdonald%26aufirst%3DS.%2BJ.%26atitle%3DLead%2520optimization%2520using%2520matched%2520molecular%2520pairs%253A%2520inclusion%2520of%2520contextual%2520information%2520for%2520enhanced%2520prediction%2520of%2520HERG%2520inhibition%252C%2520solubility%252C%2520and%2520lipophilicity%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26issue%3D10%26spage%3D1872%26epage%3D1886%26doi%3D10.1021%2Fci100258p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group">Leach, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, H. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cosgrove, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kenny, P. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruston, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacFaul, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wood, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Colclough, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Law, B.</span><span> </span><span class="NLM_article-title">Matched molecular pairs as a guide in the optimization of pharmaceutical properties; a study of aqueous solubility, plasma protein binding and oral exposure</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x"> (</span><span class="NLM_issue">23</span><span class="NLM_x">) </span> <span class="NLM_fpage">6672</span><span class="NLM_x">–</span> <span class="NLM_lpage">6682</span><span class="refDoi"> DOI: 10.1021/jm0605233</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0605233" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=6672-6682&issue=23&author=A.+G.+Leachauthor=H.+D.+Jonesauthor=D.+A.+Cosgroveauthor=P.+W.+Kennyauthor=L.+Rustonauthor=P.+MacFaulauthor=J.+M.+Woodauthor=N.+Colcloughauthor=B.+Law&title=Matched+molecular+pairs+as+a+guide+in+the+optimization+of+pharmaceutical+properties%3B+a+study+of+aqueous+solubility%2C+plasma+protein+binding+and+oral+exposure&doi=10.1021%2Fjm0605233"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1021%2Fjm0605233&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0605233%26sid%3Dliteratum%253Aachs%26aulast%3DLeach%26aufirst%3DA.%2BG.%26aulast%3DJones%26aufirst%3DH.%2BD.%26aulast%3DCosgrove%26aufirst%3DD.%2BA.%26aulast%3DKenny%26aufirst%3DP.%2BW.%26aulast%3DRuston%26aufirst%3DL.%26aulast%3DMacFaul%26aufirst%3DP.%26aulast%3DWood%26aufirst%3DJ.%2BM.%26aulast%3DColclough%26aufirst%3DN.%26aulast%3DLaw%26aufirst%3DB.%26atitle%3DMatched%2520molecular%2520pairs%2520as%2520a%2520guide%2520in%2520the%2520optimization%2520of%2520pharmaceutical%2520properties%253B%2520a%2520study%2520of%2520aqueous%2520solubility%252C%2520plasma%2520protein%2520binding%2520and%2520oral%2520exposure%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26issue%3D23%26spage%3D6672%26epage%3D6682%26doi%3D10.1021%2Fjm0605233" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group">Warner, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Griffen, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">St-Gallay, S. A.</span><span> </span><span class="NLM_article-title">WizePairZ: A novel algorithm to identify, encode and exploit matched molecular pairs with unspecified cores in medicinal chemistry</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x"> (</span><span class="NLM_issue">8</span><span class="NLM_x">) </span> <span class="NLM_fpage">1350</span><span class="NLM_x">–</span> <span class="NLM_lpage">1357</span><span class="refDoi"> DOI: 10.1021/ci100084s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci100084s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BC3cXpvVSgs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=1350-1357&issue=8&author=D.+J.+Warnerauthor=E.+J.+Griffenauthor=S.+A.+St-Gallay&title=WizePairZ%3A+A+novel+algorithm+to+identify%2C+encode+and+exploit+matched+molecular+pairs+with+unspecified+cores+in+medicinal+chemistry&doi=10.1021%2Fci100084s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">WizePairZ: A Novel Algorithm to Identify, Encode, and Exploit Matched Molecular Pairs with Unspecified Cores in Medicinal Chemistry</span></div><div class="casAuthors">Warner, Daniel J.; Griffen, Edward J.; St-Gallay, Stephen A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1350-1357</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">An algorithm to automatically identify and ext. matched mol. pairs from a collection of compds. has been developed, allowing the learning assocd. with each mol. transformation to be readily exploited in drug discovery projects.  Here, we present the application to an example data set of 11 histone deacetylase inhibitors.  The matched pairs were identified, and corresponding differences in activity and lipophilicity were recorded.  These property differences were assocd. with the chem. transformations encoded in the SMIRKS reaction notation.  The transformations identified a subseries with the optimal balance of these two parameters.  Enumeration of a virtual library of compds. using the extd. transformations identified two addnl. compds. initially excluded from the anal. with an accurate estn. of their biol. activity.  We describe how the WizePairZ system can be used to archive and apply medicinal chem. knowledge from one drug discovery project to another as well as identify common bioisosteres.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpSWafs2qQ4UbVg90H21EOLACvtfcHk0lhNcLQEFUa7ZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXpvVSgs7c%253D&md5=9d83eb14cdb56079429761e79beeb292</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1021%2Fci100084s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100084s%26sid%3Dliteratum%253Aachs%26aulast%3DWarner%26aufirst%3DD.%2BJ.%26aulast%3DGriffen%26aufirst%3DE.%2BJ.%26aulast%3DSt-Gallay%26aufirst%3DS.%2BA.%26atitle%3DWizePairZ%253A%2520A%2520novel%2520algorithm%2520to%2520identify%252C%2520encode%2520and%2520exploit%2520matched%2520molecular%2520pairs%2520with%2520unspecified%2520cores%2520in%2520medicinal%2520chemistry%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26issue%3D8%26spage%3D1350%26epage%3D1357%26doi%3D10.1021%2Fci100084s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group">Sushko, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Novotarskyi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vogt, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abdelaziz, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tetko, I. V.</span><span> </span><span class="NLM_article-title">Prediction-driven matched molecular pairs to interpret QSARs and aid the molecular optimization process</span> <span class="citation_source-journal">J. Cheminf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">48</span><span class="refDoi"> DOI: 10.1186/s13321-014-0048-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.1186%2Fs13321-014-0048-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BC2MXislWhtLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&pages=48&issue=1&author=Y.+Sushkoauthor=S.+Novotarskyiauthor=R.+Kornerauthor=J.+Vogtauthor=A.+Abdelazizauthor=I.+V.+Tetko&title=Prediction-driven+matched+molecular+pairs+to+interpret+QSARs+and+aid+the+molecular+optimization+process&doi=10.1186%2Fs13321-014-0048-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Prediction-driven matched molecular pairs to interpret QSARs and aid the molecular optimization process</span></div><div class="casAuthors">Sushko, Yurii; Novotarskyi, Sergii; Koerner, Robert; Vogt, Joachim; Abdelaziz, Ahmed; Tetko, Igor V.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cheminformatics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">48/1-48/39, 39 pp.</span>CODEN:
                <span class="NLM_cas:coden">JCOHB3</span>;
        ISSN:<span class="NLM_cas:issn">1758-2946</span>.
    
            (<span class="NLM_cas:orgname">Chemistry Central Ltd.</span>)
        </div><div class="casAbstract">Background QSAR is an established and powerful method for cheap in silico assessment of physicochem. properties and biol. activities of chem. compds.  However, QSAR models are rather complex math. constructs that cannot easily be interpreted.  Medicinal chemists would benefit from practical guidance regarding which mols. to synthesize.  Another possible approach is anal. of pairs of very similar mols., so-called matched mol. pairs (MMPs).  Such an approach allows identification of mol. transformations that affect particular activities (e.g. toxicity).  In contrast to QSAR, chem. interpretation of these transformations is straightforward.  Furthermore, such transformations can give medicinal chemists useful hints for the hit-to-lead optimization process.  Results The current study suggests a combination of QSAR and MMP approaches by finding MMP transformations based on QSAR predictions for large chem. datasets.  The study shows that such an approach, referred to as prediction-driven MMP anal., is a useful tool for medicinal chemists, allowing identification of large nos. of "interesting" transformations that can be used to drive the mol. optimization process.  All the methodol. developments have been implemented as software products available online as part of OCHEM.  Conclusions The prediction-driven MMPs methodol. was exemplified by two use cases: modeling of aquatic toxicity and CYP3A4 inhibition.  This approach helped us to interpret QSAR models and allowed identification of a no. of "significant" mol. transformations that affect the desired properties.  This can facilitate drug design as a part of mol. optimization process.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTzctKLS8vxrVg90H21EOLACvtfcHk0lhNcLQEFUa7ZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXislWhtLw%253D&md5=837bda80de79a7621c887de1fed0375a</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1186%2Fs13321-014-0048-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13321-014-0048-0%26sid%3Dliteratum%253Aachs%26aulast%3DSushko%26aufirst%3DY.%26aulast%3DNovotarskyi%26aufirst%3DS.%26aulast%3DKorner%26aufirst%3DR.%26aulast%3DVogt%26aufirst%3DJ.%26aulast%3DAbdelaziz%26aufirst%3DA.%26aulast%3DTetko%26aufirst%3DI.%2BV.%26atitle%3DPrediction-driven%2520matched%2520molecular%2520pairs%2520to%2520interpret%2520QSARs%2520and%2520aid%2520the%2520molecular%2520optimization%2520process%26jtitle%3DJ.%2520Cheminf.%26date%3D2014%26volume%3D6%26issue%3D1%26spage%3D48%26doi%3D10.1186%2Fs13321-014-0048-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group">O’Boyle, N. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bostrom, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sayle, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gill, A.</span><span> </span><span class="NLM_article-title">Using matched molecular series as a predictive tool to optimize biological activity</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x"> (</span><span class="NLM_issue">6</span><span class="NLM_x">) </span> <span class="NLM_fpage">2704</span><span class="NLM_x">–</span> <span class="NLM_lpage">2713</span><span class="refDoi"> DOI: 10.1021/jm500022q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm500022q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=2704-2713&issue=6&author=N.+M.+O%E2%80%99Boyleauthor=J.+Bostromauthor=R.+A.+Sayleauthor=A.+Gill&title=Using+matched+molecular+series+as+a+predictive+tool+to+optimize+biological+activity&doi=10.1021%2Fjm500022q"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1021%2Fjm500022q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm500022q%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Boyle%26aufirst%3DN.%2BM.%26aulast%3DBostrom%26aufirst%3DJ.%26aulast%3DSayle%26aufirst%3DR.%2BA.%26aulast%3DGill%26aufirst%3DA.%26atitle%3DUsing%2520matched%2520molecular%2520series%2520as%2520a%2520predictive%2520tool%2520to%2520optimize%2520biological%2520activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26issue%3D6%26spage%3D2704%26epage%3D2713%26doi%3D10.1021%2Fjm500022q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group">Gupta-Ostermann, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajorath, J.</span><span> </span><span class="NLM_article-title">The “SAR matrix” method and its extensions for applications in medicinal chemistry and chemogenomics</span> <span class="citation_source-journal">F1000Research</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">113</span><span class="refDoi"> DOI: 10.12688/f1000research.4185.2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.12688%2Ff1000research.4185.2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=25383183" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A280%3ADC%252BC2M3ntVeitw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2014&pages=113&author=D.+Gupta-Ostermannauthor=J.+Bajorath&title=The+%E2%80%9CSAR+matrix%E2%80%9D+method+and+its+extensions+for+applications+in+medicinal+chemistry+and+chemogenomics&doi=10.12688%2Ff1000research.4185.2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">The 'SAR Matrix' method and its extensions for applications in medicinal chemistry and chemogenomics</span></div><div class="casAuthors">Gupta-Ostermann Disha; Bajorath Jurgen</div><div class="citationInfo"><span class="NLM_cas:title">F1000Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">113</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">We describe the 'Structure-Activity Relationship (SAR) Matrix' (SARM) methodology that is based upon a special two-step application of the matched molecular pair (MMP) formalism.  The SARM method has originally been designed for the extraction, organization, and visualization of compound series and associated SAR information from compound data sets.  It has been further developed and adapted for other applications including compound design, activity prediction, library extension, and the navigation of multi-target activity spaces.  The SARM approach and its extensions are presented here in context to introduce different types of applications and provide an example for the evolution of a computational methodology in pharmaceutical research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTkxRbz5791CpSWdIdmEou5fW6udTcc2eaGg7uT5JgUpbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2M3ntVeitw%253D%253D&md5=3d96e38612bfbbdf91fd404830c9e886</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.12688%2Ff1000research.4185.2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.12688%252Ff1000research.4185.2%26sid%3Dliteratum%253Aachs%26aulast%3DGupta-Ostermann%26aufirst%3DD.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DThe%2520%25E2%2580%259CSAR%2520matrix%25E2%2580%259D%2520method%2520and%2520its%2520extensions%2520for%2520applications%2520in%2520medicinal%2520chemistry%2520and%2520chemogenomics%26jtitle%3DF1000Research%26date%3D2014%26volume%3D3%26spage%3D113%26doi%3D10.12688%2Ff1000research.4185.2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group">Webb, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hanser, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Howlin, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krause, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vessey, J. D.</span><span> </span><span class="NLM_article-title">Feature combination networks for the interpretation of statistical machine learning models: application to Ames mutagenicity</span> <span class="citation_source-journal">J. Cheminf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">8</span><span class="refDoi"> DOI: 10.1186/1758-2946-6-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.1186%2F1758-2946-6-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BC2MXivFSktro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&pages=8&issue=1&author=S.+J.+Webbauthor=T.+Hanserauthor=B.+Howlinauthor=P.+Krauseauthor=J.+D.+Vessey&title=Feature+combination+networks+for+the+interpretation+of+statistical+machine+learning+models%3A+application+to+Ames+mutagenicity&doi=10.1186%2F1758-2946-6-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Feature combination networks for the interpretation of statistical machine learning models: application to Ames mutagenicity</span></div><div class="casAuthors">Webb, Samuel J.; Hanser, Thierry; Howlin, Brendan; Krause, Paul; Vessey, Jonathan D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cheminformatics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">8/1-8/21, 21 pp.</span>CODEN:
                <span class="NLM_cas:coden">JCOHB3</span>;
        ISSN:<span class="NLM_cas:issn">1758-2946</span>.
    
            (<span class="NLM_cas:orgname">Chemistry Central Ltd.</span>)
        </div><div class="casAbstract">Background: A new algorithm has been developed to enable the interpretation of black box models.  The developed algorithm is agnostic to learning algorithm and open to all structural based descriptors such as fragments, keys and hashed fingerprints.  The algorithm has provided meaningful interpretation of Ames mutagenicity predictions from both random forest and support vector machine models built on a variety of structural fingerprints.  A fragmentation algorithm is utilized to investigate the model's behavior on specific substructures present in the query.  An output is formulated summarising causes of activation and deactivation.  The algorithm is able to identify multiple causes of activation or deactivation in addn. to identifying localised deactivations where the prediction for the query is active overall.  No loss in performance is seen as there is no change in the prediction; the interpretation is produced directly on the model's behavior for the specific query.  Results: Models have been built using multiple learning algorithms including support vector machine and random forest.  The models were built on public Ames mutagenicity data and a variety of fingerprint descriptors were used.  These models produced a good performance in both internal and external validation with accuracies around 82%.  The models were used to evaluate the interpretation algorithm.  Interpretation was revealed that links closely with understood mechanisms for Ames mutagenicity.  Conclusion: This methodol. allows for a greater utilization of the predictions made by black box models and can expedite further study based on the output for a (quant.) structure activity model.  Addnl. the algorithm could be utilized for chem. dataset investigation and knowledge extn./human SAR development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp78W0IxwwolbVg90H21EOLACvtfcHk0lh3uflMpXrX4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXivFSktro%253D&md5=3908e46c4b0d7320d1b105c3dbb0af17</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1186%2F1758-2946-6-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1758-2946-6-8%26sid%3Dliteratum%253Aachs%26aulast%3DWebb%26aufirst%3DS.%2BJ.%26aulast%3DHanser%26aufirst%3DT.%26aulast%3DHowlin%26aufirst%3DB.%26aulast%3DKrause%26aufirst%3DP.%26aulast%3DVessey%26aufirst%3DJ.%2BD.%26atitle%3DFeature%2520combination%2520networks%2520for%2520the%2520interpretation%2520of%2520statistical%2520machine%2520learning%2520models%253A%2520application%2520to%2520Ames%2520mutagenicity%26jtitle%3DJ.%2520Cheminf.%26date%3D2014%26volume%3D6%26issue%3D1%26spage%3D8%26doi%3D10.1186%2F1758-2946-6-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group">Guha, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jurs, P. C.</span><span> </span><span class="NLM_article-title">Interpreting computational neural network QSAR models: a measure of descriptor importance</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x"> (</span><span class="NLM_issue">3</span><span class="NLM_x">) </span> <span class="NLM_fpage">800</span><span class="NLM_x">–</span> <span class="NLM_lpage">806</span><span class="refDoi"> DOI: 10.1021/ci050022a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci050022a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BD2MXisleqs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2005&pages=800-806&issue=3&author=R.+Guhaauthor=P.+C.+Jurs&title=Interpreting+computational+neural+network+QSAR+models%3A+a+measure+of+descriptor+importance&doi=10.1021%2Fci050022a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Interpreting Computational Neural Network QSAR Models: A Measure of Descriptor Importance</span></div><div class="casAuthors">Guha, Rajarshi; Jurs, Peter C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">800-806</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We present a method to measure the relative importance of the descriptors present in a QSAR model developed with a computational neural network (CNN).  The approach is based on a sensitivity anal. of the descriptors.  We tested the method on three published data sets for which linear and CNN models were previously built.  The original work reported interpretations for the linear models, and we compare the results of the new method to the importance of descriptors in the linear models as described by a PLS technique.  The results indicate that the proposed method is able to rank descriptors such that important descriptors in the CNN model correspond to the important descriptors in the linear model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXIOYkme8SaLVg90H21EOLACvtfcHk0lh3uflMpXrX4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXisleqs7k%253D&md5=6aa8c896834301c305ebefa91b0fb82b</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1021%2Fci050022a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci050022a%26sid%3Dliteratum%253Aachs%26aulast%3DGuha%26aufirst%3DR.%26aulast%3DJurs%26aufirst%3DP.%2BC.%26atitle%3DInterpreting%2520computational%2520neural%2520network%2520QSAR%2520models%253A%2520a%2520measure%2520of%2520descriptor%2520importance%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2005%26volume%3D45%26issue%3D3%26spage%3D800%26epage%3D806%26doi%3D10.1021%2Fci050022a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref69"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref69'); return false;" data-citation="" class="refNumLink">69</a></strong><div class="NLM_citation" id="cit69"><span><span class="NLM_contrib-group">Schneider, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fechner, U.</span><span> </span><span class="NLM_article-title">Computer-based de novo design of drug-like molecules</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x"> (</span><span class="NLM_issue">8</span><span class="NLM_x">) </span> <span class="NLM_fpage">649</span><span class="NLM_x">–</span> <span class="NLM_lpage">663</span><span class="refDoi"> DOI: 10.1038/nrd1799</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.1038%2Fnrd1799" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=16056391" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BD2MXmvVOqtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2005&pages=649-663&issue=8&author=G.+Schneiderauthor=U.+Fechner&title=Computer-based+de+novo+design+of+drug-like+molecules&doi=10.1038%2Fnrd1799"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Computer-based de novo design of drug-like molecules</span></div><div class="casAuthors">Schneider, Gisbert; Fechner, Uli</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">649-663</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review with refs.  Ever since the first automated de novo design techniques were conceived only 15 years ago, the computer-based design of hit and lead structure candidates has emerged as a complementary approach to high-throughput screening.  Although many challenges remain, de novo design supports drug discovery projects by generating novel pharmaceutically active agents with desired properties in a cost- and time-efficient manner.  In this review, we outline the various design concepts and highlight current developments in computer-based de novo design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWf2XFrvn_fbVg90H21EOLACvtfcHk0lgyL9LMPfh2aQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXmvVOqtro%253D&md5=a30dbc58ed81e0b7fe3f7d41a668e9ac</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1038%2Fnrd1799&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd1799%26sid%3Dliteratum%253Aachs%26aulast%3DSchneider%26aufirst%3DG.%26aulast%3DFechner%26aufirst%3DU.%26atitle%3DComputer-based%2520de%2520novo%2520design%2520of%2520drug-like%2520molecules%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2005%26volume%3D4%26issue%3D8%26spage%3D649%26epage%3D663%26doi%3D10.1038%2Fnrd1799" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref70"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref70'); return false;" data-citation="" class="refNumLink">70</a></strong><div class="NLM_citation" id="cit70"><span><span class="NLM_contrib-group">Firth, N. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atrash, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blagg, J.</span><span> </span><span class="NLM_article-title">MOARF, an integrated workflow for multiobjective optimization: implementation, synthesis, and biological evaluation</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x"> (</span><span class="NLM_issue">6</span><span class="NLM_x">) </span> <span class="NLM_fpage">1169</span><span class="NLM_x">–</span> <span class="NLM_lpage">1180</span><span class="refDoi"> DOI: 10.1021/acs.jcim.5b00073</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.5b00073" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVShtLvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=1169-1180&issue=6&author=N.+C.+Firthauthor=B.+Atrashauthor=N.+Brownauthor=J.+Blagg&title=MOARF%2C+an+integrated+workflow+for+multiobjective+optimization%3A+implementation%2C+synthesis%2C+and+biological+evaluation&doi=10.1021%2Facs.jcim.5b00073"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">MOARF, an Integrated Workflow for Multiobjective Optimization: Implementation, Synthesis, and Biological Evaluation</span></div><div class="casAuthors">Firth, Nicholas C.; Atrash, Butrus; Brown, Nathan; Blagg, Julian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1169-1180</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We describe the development and application of an integrated, multiobjective optimization workflow (MOARF) for directed medicinal chem. design.  This workflow couples a rule-based mol. fragmentation scheme (SynDiR) with a pharmacophore fingerprint-based fragment replacement algorithm (RATS) to broaden the scope of reconnection options considered in the generation of potential soln. structures.  Solns. are ranked by a multiobjective scoring algorithm comprising ligand-based (shape similarity) biochem. activity predictions as well as physicochem. property calcns.  Application of this iterative workflow to optimization of the CDK2 inhibitor Seliciclib (CYC202, R-roscovitine) generated soln. mols. in desired physicochem. property space.  Synthesis and exptl. evaluation of optimal soln. mols. demonstrates CDK2 biochem. activity and improved human metabolic stability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDa1EOPXpsI7Vg90H21EOLACvtfcHk0lgyL9LMPfh2aQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVShtLvK&md5=0eed904cb0346cb4a521e30998fae979</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.5b00073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.5b00073%26sid%3Dliteratum%253Aachs%26aulast%3DFirth%26aufirst%3DN.%2BC.%26aulast%3DAtrash%26aufirst%3DB.%26aulast%3DBrown%26aufirst%3DN.%26aulast%3DBlagg%26aufirst%3DJ.%26atitle%3DMOARF%252C%2520an%2520integrated%2520workflow%2520for%2520multiobjective%2520optimization%253A%2520implementation%252C%2520synthesis%252C%2520and%2520biological%2520evaluation%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26volume%3D55%26issue%3D6%26spage%3D1169%26epage%3D1180%26doi%3D10.1021%2Facs.jcim.5b00073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref71"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref71'); return false;" data-citation="" class="refNumLink">71</a></strong><div class="NLM_citation" id="cit71"><span><span class="NLM_contrib-group">Gabrielsson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dolgos, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gillberg, P.-G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bredberg, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benthem, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duker, G.</span><span> </span><span class="NLM_article-title">Early intergration of pharmacokinetic and dynamic reasoning is essential for optimal development of lead compounds: strategic considerations</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">358</span><span class="NLM_x">–</span> <span class="NLM_lpage">372</span><span class="refDoi"> DOI: 10.1016/j.drudis.2008.12.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.1016%2Fj.drudis.2008.12.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=19162219" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjvVWgsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2009&pages=358-372&author=J.+Gabrielssonauthor=H.+Dolgosauthor=P.-G.+Gillbergauthor=U.+Bredbergauthor=B.+Benthemauthor=G.+Duker&title=Early+intergration+of+pharmacokinetic+and+dynamic+reasoning+is+essential+for+optimal+development+of+lead+compounds%3A+strategic+considerations&doi=10.1016%2Fj.drudis.2008.12.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Early integration of pharmacokinetic and dynamic reasoning is essential for optimal development of lead compounds: strategic considerations</span></div><div class="casAuthors">Gabrielsson, Johan; Dolgos, Hugues; Gillberg, Per-Goeran; Bredberg, Ulf; Benthem, Bert; Duker, Goeran</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">7/8</span>),
    <span class="NLM_cas:pages">358-372</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The aims of this report are firstly to raise awareness among kineticists and pharmacologists as to why pharmacokinetic-pharmacodynamic (PKPD) integration is essential for target validation (TV), optimizing development of lead compds. (lead generation [LG] and lead optimization [LO]) and scaling these to human.  A related aim is to demonstrate strategic examples of PKPD collaborations that have improved the planning, execution and evaluation of expts. in primary and safety pharmacol.  Examples include design of TV studies, design and data pruning' of PKPD studies in LO, anal. of data with marginal and substantial temporal (time) differences between exposure and response, design of safety pharmacol. studies, assessment of safety margin and assessment of uncertainties in predictions of first dose in human.  PKPD awareness is vital if we are to attempt to relate preclin. results to the acute and long term consequences in humans.  The debate on whether preclin. findings can be translation to the human usage is still engaging scientists across industry, academia and regulatory bodies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXmTuZ4LRZ5bVg90H21EOLACvtfcHk0lgyL9LMPfh2aQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjvVWgsbs%253D&md5=d52632817208dd41fde8e3355ed22bb4</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2008.12.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2008.12.011%26sid%3Dliteratum%253Aachs%26aulast%3DGabrielsson%26aufirst%3DJ.%26aulast%3DDolgos%26aufirst%3DH.%26aulast%3DGillberg%26aufirst%3DP.-G.%26aulast%3DBredberg%26aufirst%3DU.%26aulast%3DBenthem%26aufirst%3DB.%26aulast%3DDuker%26aufirst%3DG.%26atitle%3DEarly%2520intergration%2520of%2520pharmacokinetic%2520and%2520dynamic%2520reasoning%2520is%2520essential%2520for%2520optimal%2520development%2520of%2520lead%2520compounds%253A%2520strategic%2520considerations%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2009%26volume%3D14%26spage%3D358%26epage%3D372%26doi%3D10.1016%2Fj.drudis.2008.12.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68'],'ref69':['cit69'],'ref70':['cit70'],'ref71':['cit71']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 48 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Henriëtte Willems, Stephane De Cesco, <span class="NLM_string-name hlFld-ContribAuthor">Fredrik Svensson</span>. </span><span class="cited-content_cbyCitation_article-title">Computational Chemistry on a Budget: Supporting Drug Discovery with Limited Resources. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (18)
                                     , 10158-10169. <a href="https://doi.org/10.1021/acs.jmedchem.9b02126" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b02126</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b02126&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b02126%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DComputational%252BChemistry%252Bon%252Ba%252BBudget%25253A%252BSupporting%252BDrug%252BDiscovery%252Bwith%252BLimited%252BResources%26aulast%3DWillems%26aufirst%3DHenri%25C3%25ABtte%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D20122019%26date%3D01052020%26date%3D16042020%26volume%3D63%26issue%3D18%26spage%3D10158%26epage%3D10169" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Edward J. Griffen, Alexander G. Dossetter, <span class="NLM_string-name hlFld-ContribAuthor">Andrew G. Leach</span>. </span><span class="cited-content_cbyCitation_article-title">Chemists: AI Is Here; Unite To Get the Benefits. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (16)
                                     , 8695-8704. <a href="https://doi.org/10.1021/acs.jmedchem.0c00163" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00163</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00163&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00163%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DChemists%25253A%252BAI%252BIs%252BHere%25253B%252BUnite%252BTo%252BGet%252Bthe%252BBenefits%26aulast%3DGriffen%26aufirst%3DEdward%2BJ.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D29012020%26date%3D11062020%26date%3D27052020%26volume%3D63%26issue%3D16%26spage%3D8695%26epage%3D8704" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Ulf Norinder, Ola Spjuth, <span class="NLM_string-name hlFld-ContribAuthor">Fredrik Svensson</span>. </span><span class="cited-content_cbyCitation_article-title">Using Predicted Bioactivity Profiles to Improve Predictive Modeling. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (6)
                                     , 2830-2837. <a href="https://doi.org/10.1021/acs.jcim.0c00250" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c00250</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c00250&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c00250%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DUsing%252BPredicted%252BBioactivity%252BProfiles%252Bto%252BImprove%252BPredictive%252BModeling%26aulast%3DNorinder%26aufirst%3DUlf%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D11032020%26date%3D15052020%26date%3D06052020%26volume%3D60%26issue%3D6%26spage%3D2830%26epage%3D2837" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Andy H. Vo, Terry R. Van Vleet, Rishi R. Gupta, Michael J. Liguori, <span class="NLM_string-name hlFld-ContribAuthor">Mohan S. Rao</span>. </span><span class="cited-content_cbyCitation_article-title">An Overview of Machine Learning and Big Data for Drug Toxicity Evaluation. </span><span class="cited-content_cbyCitation_journal-name">Chemical Research in Toxicology</span><span> <strong>2020,</strong> <em>33 </em>
                                    (1)
                                     , 20-37. <a href="https://doi.org/10.1021/acs.chemrestox.9b00227" title="DOI URL">https://doi.org/10.1021/acs.chemrestox.9b00227</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.chemrestox.9b00227&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.chemrestox.9b00227%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Research%2520in%2520Toxicology%26atitle%3DAn%252BOverview%252Bof%252BMachine%252BLearning%252Band%252BBig%252BData%252Bfor%252BDrug%252BToxicity%252BEvaluation%26aulast%3DVo%26aufirst%3DAndy%2BH.%26date%3D2020%26date%3D2019%26date%3D2019%26date%3D31052019%26date%3D22112019%26date%3D18102019%26volume%3D33%26issue%3D1%26spage%3D20%26epage%3D37" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Yadi Zhou, Suntara Cahya, Steven A. Combs, Christos A. Nicolaou, Jibo Wang, Prashant V. Desai, <span class="NLM_string-name hlFld-ContribAuthor">Jie Shen</span>. </span><span class="cited-content_cbyCitation_article-title">Exploring Tunable Hyperparameters for Deep Neural Networks with Industrial ADME Data Sets. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2019,</strong> <em>59 </em>
                                    (3)
                                     , 1005-1016. <a href="https://doi.org/10.1021/acs.jcim.8b00671" title="DOI URL">https://doi.org/10.1021/acs.jcim.8b00671</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.8b00671&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.8b00671%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DExploring%252BTunable%252BHyperparameters%252Bfor%252BDeep%252BNeural%252BNetworks%252Bwith%252BIndustrial%252BADME%252BData%252BSets%26aulast%3DZhou%26aufirst%3DYadi%26date%3D2019%26date%3D2018%26date%3D2018%26date%3D30092018%26date%3D11012019%26date%3D26122018%26volume%3D59%26issue%3D3%26spage%3D1005%26epage%3D1016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Jan Wenzel, Hans Matter, <span class="NLM_string-name hlFld-ContribAuthor">Friedemann Schmidt</span>. </span><span class="cited-content_cbyCitation_article-title">Predictive Multitask Deep Neural Network Models for ADME-Tox Properties: Learning from Large Data Sets. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2019,</strong> <em>59 </em>
                                    (3)
                                     , 1253-1268. <a href="https://doi.org/10.1021/acs.jcim.8b00785" title="DOI URL">https://doi.org/10.1021/acs.jcim.8b00785</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.8b00785&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.8b00785%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DPredictive%252BMultitask%252BDeep%252BNeural%252BNetwork%252BModels%252Bfor%252BADME-Tox%252BProperties%25253A%252BLearning%252Bfrom%252BLarge%252BData%252BSets%26aulast%3DWenzel%26aufirst%3DJan%26date%3D2019%26date%3D2019%26date%3D2018%26date%3D06112018%26date%3D24012019%26date%3D07012019%26volume%3D59%26issue%3D3%26spage%3D1253%26epage%3D1268" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Patrick Hop, Brandon Allgood, <span class="NLM_string-name hlFld-ContribAuthor">Jessen Yu</span>. </span><span class="cited-content_cbyCitation_article-title">Geometric Deep Learning Autonomously Learns Chemical Features That Outperform Those Engineered by Domain Experts. </span><span class="cited-content_cbyCitation_journal-name">Molecular Pharmaceutics</span><span> <strong>2018,</strong> <em>15 </em>
                                    (10)
                                     , 4371-4377. <a href="https://doi.org/10.1021/acs.molpharmaceut.7b01144" title="DOI URL">https://doi.org/10.1021/acs.molpharmaceut.7b01144</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.molpharmaceut.7b01144&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.molpharmaceut.7b01144%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Pharmaceutics%26atitle%3DGeometric%252BDeep%252BLearning%252BAutonomously%252BLearns%252BChemical%252BFeatures%252BThat%252BOutperform%252BThose%252BEngineered%252Bby%252BDomain%252BExperts%26aulast%3DHop%26aufirst%3DPatrick%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D2017%26date%3D2018%26date%3D18122017%26date%3D04062018%26date%3D31052018%26date%3D07072018%26date%3D04062018%26volume%3D15%26issue%3D10%26spage%3D4371%26epage%3D4377" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rahul  Singh</span>, <span class="hlFld-ContribAuthor ">Vijay Kumar  Bhardwaj</span>, <span class="hlFld-ContribAuthor ">Pralay  Das</span>, <span class="hlFld-ContribAuthor ">Rituraj  Purohit</span>. </span><span class="cited-content_cbyCitation_article-title">A computational approach for rational discovery of inhibitors for non-structural protein 1 of SARS-CoV-2. </span><span class="cited-content_cbyCitation_journal-name">Computers in Biology and Medicine</span><span> <strong>2021,</strong> <em>135 </em>, 104555. <a href="https://doi.org/10.1016/j.compbiomed.2021.104555" title="DOI URL">https://doi.org/10.1016/j.compbiomed.2021.104555</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.compbiomed.2021.104555&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.compbiomed.2021.104555%26sid%3Dliteratum%253Aachs%26jtitle%3DComputers%2520in%2520Biology%2520and%2520Medicine%26atitle%3DA%252Bcomputational%252Bapproach%252Bfor%252Brational%252Bdiscovery%252Bof%252Binhibitors%252Bfor%252Bnon-structural%252Bprotein%252B1%252Bof%252BSARS-CoV-2%26aulast%3DSingh%26aufirst%3DRahul%26date%3D2021%26volume%3D135%26spage%3D104555" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yohei  Kosugi</span>, <span class="hlFld-ContribAuthor ">Kunihiko  Mizuno</span>, <span class="hlFld-ContribAuthor ">Cipriano  Santos</span>, <span class="hlFld-ContribAuthor ">Sho  Sato</span>, <span class="hlFld-ContribAuthor ">Natalie  Hosea</span>, <span class="hlFld-ContribAuthor ">Michael  Zientek</span>. </span><span class="cited-content_cbyCitation_article-title">Direct Comparison of the Prediction of the Unbound Brain-to-Plasma Partitioning Utilizing Machine Learning Approach and Mechanistic Neuropharmacokinetic Model. </span><span class="cited-content_cbyCitation_journal-name">The AAPS Journal</span><span> <strong>2021,</strong> <em>23 </em>
                                    (4)
                                     <a href="https://doi.org/10.1208/s12248-021-00604-x" title="DOI URL">https://doi.org/10.1208/s12248-021-00604-x</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1208/s12248-021-00604-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1208%2Fs12248-021-00604-x%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520AAPS%2520Journal%26atitle%3DDirect%252BComparison%252Bof%252Bthe%252BPrediction%252Bof%252Bthe%252BUnbound%252BBrain-to-Plasma%252BPartitioning%252BUtilizing%252BMachine%252BLearning%252BApproach%252Band%252BMechanistic%252BNeuropharmacokinetic%252BModel%26aulast%3DKosugi%26aufirst%3DYohei%26date%3D2021%26date%3D2021%26volume%3D23%26issue%3D4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Esin  Karatas</span>, <span class="hlFld-ContribAuthor ">Egemen  Foto</span>, <span class="hlFld-ContribAuthor ">Tugba  Ertan-Bolelli</span>, <span class="hlFld-ContribAuthor ">Gozde  Yalcin-Ozkat</span>, <span class="hlFld-ContribAuthor ">Serap  Yilmaz</span>, <span class="hlFld-ContribAuthor ">Sanaz  Ataei</span>, <span class="hlFld-ContribAuthor ">Fatma  Zilifdar</span>, <span class="hlFld-ContribAuthor ">Ilkay  Yildiz</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of 5-(or 6)-benzoxazoles and oxazolo[4,5-b]pyridines as novel candidate antitumor agents targeting hTopo IIα. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic Chemistry</span><span> <strong>2021,</strong> <em>112 </em>, 104913. <a href="https://doi.org/10.1016/j.bioorg.2021.104913" title="DOI URL">https://doi.org/10.1016/j.bioorg.2021.104913</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bioorg.2021.104913&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bioorg.2021.104913%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520Chemistry%26atitle%3DDiscovery%252Bof%252B5-%252528or%252B6%252529-benzoxazoles%252Band%252Boxazolo%25255B4%25252C5-b%25255Dpyridines%252Bas%252Bnovel%252Bcandidate%252Bantitumor%252Bagents%252Btargeting%252BhTopo%252BII%2525CE%2525B1%26aulast%3DKaratas%26aufirst%3DEsin%26date%3D2021%26volume%3D112%26spage%3D104913" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Richa  Bharti</span>, <span class="hlFld-ContribAuthor ">Yamini</span>, <span class="hlFld-ContribAuthor ">V.K.  Bhardwaj</span>, <span class="hlFld-ContribAuthor ">C.  Bal Reddy</span>, <span class="hlFld-ContribAuthor ">Rituraj  Purohit</span>, <span class="hlFld-ContribAuthor ">Pralay  Das</span>. </span><span class="cited-content_cbyCitation_article-title">Benzosuberene-sulfone analogues synthesis from Cedrus deodara oil and their therapeutic evaluation by computational analysis to treat type 2 diabetes. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic Chemistry</span><span> <strong>2021,</strong> <em>112 </em>, 104860. <a href="https://doi.org/10.1016/j.bioorg.2021.104860" title="DOI URL">https://doi.org/10.1016/j.bioorg.2021.104860</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bioorg.2021.104860&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bioorg.2021.104860%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520Chemistry%26atitle%3DBenzosuberene-sulfone%252Banalogues%252Bsynthesis%252Bfrom%252BCedrus%252Bdeodara%252Boil%252Band%252Btheir%252Btherapeutic%252Bevaluation%252Bby%252Bcomputational%252Banalysis%252Bto%252Btreat%252Btype%252B2%252Bdiabetes%26aulast%3DBharti%26aufirst%3DRicha%26date%3D2021%26volume%3D112%26spage%3D104860" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Khalid A.  Agha</span>, <span class="hlFld-ContribAuthor ">Tarek S.  Ibrahim</span>, <span class="hlFld-ContribAuthor ">Nehal M.  Elsherbiny</span>, <span class="hlFld-ContribAuthor ">Mohamed  El-Sherbiny</span>, <span class="hlFld-ContribAuthor ">Eatedal H.  Abdel-Aal</span>, <span class="hlFld-ContribAuthor ">Zakaria K.  Abdel-Samii</span>, <span class="hlFld-ContribAuthor ">Nader E.  Abo-Dya</span>. </span><span class="cited-content_cbyCitation_article-title">Design, Synthesis and Pharmacological Screening of Novel Renoprotective Methionine-based Peptidomimetics: Amelioration of Cisplatin-induced nephrotoxicity. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic Chemistry</span><span> <strong>2021,</strong> <em>740 </em>, 105100. <a href="https://doi.org/10.1016/j.bioorg.2021.105100" title="DOI URL">https://doi.org/10.1016/j.bioorg.2021.105100</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bioorg.2021.105100&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bioorg.2021.105100%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520Chemistry%26atitle%3DDesign%25252C%252BSynthesis%252Band%252BPharmacological%252BScreening%252Bof%252BNovel%252BRenoprotective%252BMethionine-based%252BPeptidomimetics%25253A%252BAmelioration%252Bof%252BCisplatin-induced%252Bnephrotoxicity%26aulast%3DAgha%26aufirst%3DKhalid%2BA.%26date%3D2021%26volume%3D740%26spage%3D105100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jatin  Sharma</span>, <span class="hlFld-ContribAuthor ">Vijay  Kumar Bhardwaj</span>, <span class="hlFld-ContribAuthor ">Rituraj  Purohit</span>. </span><span class="cited-content_cbyCitation_article-title">Recognition of distinct chemical molecules as inhibitors for KIT receptor mutants D816H/Y/V: A simulation approach. </span><span class="cited-content_cbyCitation_journal-name">Journal of Molecular Liquids</span><span> <strong>2021,</strong> <em>13 </em>, 116317. <a href="https://doi.org/10.1016/j.molliq.2021.116317" title="DOI URL">https://doi.org/10.1016/j.molliq.2021.116317</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.molliq.2021.116317&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.molliq.2021.116317%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Molecular%2520Liquids%26atitle%3DRecognition%252Bof%252Bdistinct%252Bchemical%252Bmolecules%252Bas%252Binhibitors%252Bfor%252BKIT%252Breceptor%252Bmutants%252BD816H%25252FY%25252FV%25253A%252BA%252Bsimulation%252Bapproach%26aulast%3DSharma%26aufirst%3DJatin%26date%3D2021%26volume%3D13%26spage%3D116317" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alejandro  Varela‐Rial</span>, <span class="hlFld-ContribAuthor ">Maciej  Majewski</span>, <span class="hlFld-ContribAuthor ">Gianni  De Fabritiis</span>. </span><span class="cited-content_cbyCitation_article-title">Structure based virtual screening: Fast and slow. </span><span class="cited-content_cbyCitation_journal-name">WIREs Computational Molecular Science</span><span> <strong>2021,</strong> <em>40 </em><a href="https://doi.org/10.1002/wcms.1544" title="DOI URL">https://doi.org/10.1002/wcms.1544</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/wcms.1544&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fwcms.1544%26sid%3Dliteratum%253Aachs%26jtitle%3DWIREs%2520Computational%2520Molecular%2520Science%26atitle%3DStructure%252Bbased%252Bvirtual%252Bscreening%25253A%252BFast%252Band%252Bslow%26aulast%3DVarela%25E2%2580%2590Rial%26aufirst%3DAlejandro%26date%3D2021%26date%3D2021%26volume%3D40" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sajjad  Ahmad</span>, <span class="hlFld-ContribAuthor ">Yasir  Waheed</span>, <span class="hlFld-ContribAuthor ">Saba  Ismail</span>, <span class="hlFld-ContribAuthor ">Muzammil Hasan  Najmi</span>, <span class="hlFld-ContribAuthor ">Jawad Khaliq  Ansari</span>. </span><span class="cited-content_cbyCitation_article-title">Rational design of potent anti-COVID-19 main protease drugs: An extensive multi-spectrum in silico approach. </span><span class="cited-content_cbyCitation_journal-name">Journal of Molecular Liquids</span><span> <strong>2021,</strong> <em>330 </em>, 115636. <a href="https://doi.org/10.1016/j.molliq.2021.115636" title="DOI URL">https://doi.org/10.1016/j.molliq.2021.115636</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.molliq.2021.115636&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.molliq.2021.115636%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Molecular%2520Liquids%26atitle%3DRational%252Bdesign%252Bof%252Bpotent%252Banti-COVID-19%252Bmain%252Bprotease%252Bdrugs%25253A%252BAn%252Bextensive%252Bmulti-spectrum%252Bin%252Bsilico%252Bapproach%26aulast%3DAhmad%26aufirst%3DSajjad%26date%3D2021%26volume%3D330%26spage%3D115636" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sheng  Guo</span>, <span class="hlFld-ContribAuthor ">Hang  Xie</span>, <span class="hlFld-ContribAuthor ">Yu  Lei</span>, <span class="hlFld-ContribAuthor ">Bin  Liu</span>, <span class="hlFld-ContribAuthor ">Li  Zhang</span>, <span class="hlFld-ContribAuthor ">Yechun  Xu</span>, <span class="hlFld-ContribAuthor ">Zhili  Zuo</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of novel inhibitors against main protease (Mpro) of SARS-CoV-2 via virtual screening and biochemical evaluation. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic Chemistry</span><span> <strong>2021,</strong> <em>110 </em>, 104767. <a href="https://doi.org/10.1016/j.bioorg.2021.104767" title="DOI URL">https://doi.org/10.1016/j.bioorg.2021.104767</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bioorg.2021.104767&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bioorg.2021.104767%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520Chemistry%26atitle%3DDiscovery%252Bof%252Bnovel%252Binhibitors%252Bagainst%252Bmain%252Bprotease%252B%252528Mpro%252529%252Bof%252BSARS-CoV-2%252Bvia%252Bvirtual%252Bscreening%252Band%252Bbiochemical%252Bevaluation%26aulast%3DGuo%26aufirst%3DSheng%26date%3D2021%26volume%3D110%26spage%3D104767" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Edward H.  Kerns</span>, <span class="hlFld-ContribAuthor ">Xun  Zhang</span>, <span class="hlFld-ContribAuthor ">Li  Di</span>. </span><span class="cited-content_cbyCitation_article-title">ADMET
              In
              Vitro
              Profiling – Utility and Applications in Lead Discovery. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 1-31. <a href="https://doi.org/10.1002/0471266949.bmc118.pub2" title="DOI URL">https://doi.org/10.1002/0471266949.bmc118.pub2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/0471266949.bmc118.pub2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F0471266949.bmc118.pub2%26sid%3Dliteratum%253Aachs%26atitle%3DADMET%252BIn%252BVitro%252BProfiling%252B%2525E2%252580%252593%252BUtility%252Band%252BApplications%252Bin%252BLead%252BDiscovery%26aulast%3DKerns%26aufirst%3DEdward%2BH.%26date%3D2021%26date%3D2021%26spage%3D1%26epage%3D31%26pub%3DWiley%26atitle%3DBurger%252527s%252BMedicinal%252BChemistry%252Band%252BDrug%252BDiscovery%26aulast%3DAbraham%26aufirst%3DDonald%2BJ.%26date%3D2003%26date%3D2003%26volume%3D8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mary Alexandra  Schleiff</span>, <span class="hlFld-ContribAuthor ">Deepika  Dhaware</span>, <span class="hlFld-ContribAuthor ">Jasleen K.  Sodhi</span>. </span><span class="cited-content_cbyCitation_article-title">Recent advances in computational metabolite structure predictions and altered metabolic pathways assessment to inform drug development processes. </span><span class="cited-content_cbyCitation_journal-name">Drug Metabolism Reviews</span><span> <strong>2021,</strong> <em>53 </em>
                                    (2)
                                     , 173-187. <a href="https://doi.org/10.1080/03602532.2021.1910292" title="DOI URL">https://doi.org/10.1080/03602532.2021.1910292</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/03602532.2021.1910292&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F03602532.2021.1910292%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Metabolism%2520Reviews%26atitle%3DRecent%252Badvances%252Bin%252Bcomputational%252Bmetabolite%252Bstructure%252Bpredictions%252Band%252Baltered%252Bmetabolic%252Bpathways%252Bassessment%252Bto%252Binform%252Bdrug%252Bdevelopment%252Bprocesses%26aulast%3DSchleiff%26aufirst%3DMary%2BAlexandra%26date%3D2021%26date%3D2021%26volume%3D53%26issue%3D2%26spage%3D173%26epage%3D187" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ioana  Oprisiu</span>, <span class="hlFld-ContribAuthor ">Susanne  Winiwarter</span>. </span><span class="cited-content_cbyCitation_article-title">In Silico ADME Modeling. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 208-222. <a href="https://doi.org/10.1016/B978-0-12-801238-3.11532-6" title="DOI URL">https://doi.org/10.1016/B978-0-12-801238-3.11532-6</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-801238-3.11532-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-801238-3.11532-6%26sid%3Dliteratum%253Aachs%26atitle%3DIn%252BSilico%252BADME%252BModeling%26aulast%3DOprisiu%26aufirst%3DIoana%26date%3D2021%26spage%3D208%26epage%3D222%26pub%3DElsevier%26atitle%3DSystems%252BMedicine%26date%3D2021%26volume%3D280" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Vijay Kumar  Bhardwaj</span>, <span class="hlFld-ContribAuthor ">Rahul  Singh</span>, <span class="hlFld-ContribAuthor ">Pralay  Das</span>, <span class="hlFld-ContribAuthor ">Rituraj  Purohit</span>. </span><span class="cited-content_cbyCitation_article-title">Evaluation of acridinedione analogs as potential SARS-CoV-2 main protease inhibitors and their comparison with repurposed anti-viral drugs. </span><span class="cited-content_cbyCitation_journal-name">Computers in Biology and Medicine</span><span> <strong>2021,</strong> <em>128 </em>, 104117. <a href="https://doi.org/10.1016/j.compbiomed.2020.104117" title="DOI URL">https://doi.org/10.1016/j.compbiomed.2020.104117</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.compbiomed.2020.104117&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.compbiomed.2020.104117%26sid%3Dliteratum%253Aachs%26jtitle%3DComputers%2520in%2520Biology%2520and%2520Medicine%26atitle%3DEvaluation%252Bof%252Bacridinedione%252Banalogs%252Bas%252Bpotential%252BSARS-CoV-2%252Bmain%252Bprotease%252Binhibitors%252Band%252Btheir%252Bcomparison%252Bwith%252Brepurposed%252Banti-viral%252Bdrugs%26aulast%3DBhardwaj%26aufirst%3DVijay%2BKumar%26date%3D2021%26volume%3D128%26spage%3D104117" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rahul  Singh</span>, <span class="hlFld-ContribAuthor ">Vijay Kumar  Bhardwaj</span>, <span class="hlFld-ContribAuthor ">Jatin  Sharma</span>, <span class="hlFld-ContribAuthor ">Pralay  Das</span>, <span class="hlFld-ContribAuthor ">Rituraj  Purohit</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery and in silico evaluation of aminoarylbenzosuberene molecules as novel checkpoint kinase 1 inhibitor determinants. </span><span class="cited-content_cbyCitation_journal-name">Genomics</span><span> <strong>2021,</strong> <em>113 </em>
                                    (1)
                                     , 707-715. <a href="https://doi.org/10.1016/j.ygeno.2020.10.001" title="DOI URL">https://doi.org/10.1016/j.ygeno.2020.10.001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ygeno.2020.10.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ygeno.2020.10.001%26sid%3Dliteratum%253Aachs%26jtitle%3DGenomics%26atitle%3DDiscovery%252Band%252Bin%252Bsilico%252Bevaluation%252Bof%252Baminoarylbenzosuberene%252Bmolecules%252Bas%252Bnovel%252Bcheckpoint%252Bkinase%252B1%252Binhibitor%252Bdeterminants%26aulast%3DSingh%26aufirst%3DRahul%26date%3D2021%26volume%3D113%26issue%3D1%26spage%3D707%26epage%3D715" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Camila  Cardoso Santos</span>, <span class="hlFld-ContribAuthor ">Marcos  Meuser Batista</span>, <span class="hlFld-ContribAuthor ">Asma  Inam Ullah</span>, <span class="hlFld-ContribAuthor ">Tummala  Rama Krishna Reddy</span>, <span class="hlFld-ContribAuthor ">Maria de Nazaré Correia  Soeiro</span>. </span><span class="cited-content_cbyCitation_article-title">Drug screening using shape-based virtual screening and
              in vitro
              experimental models of cutaneous Leishmaniasis. </span><span class="cited-content_cbyCitation_journal-name">Parasitology</span><span> <strong>2021,</strong> <em>148 </em>
                                    (1)
                                     , 98-104. <a href="https://doi.org/10.1017/S0031182020001900" title="DOI URL">https://doi.org/10.1017/S0031182020001900</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1017/S0031182020001900&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1017%2FS0031182020001900%26sid%3Dliteratum%253Aachs%26jtitle%3DParasitology%26atitle%3DDrug%252Bscreening%252Busing%252Bshape-based%252Bvirtual%252Bscreening%252Band%252Bin%252Bvitro%252Bexperimental%252Bmodels%252Bof%252Bcutaneous%252BLeishmaniasis%26aulast%3DCardoso%2BSantos%26aufirst%3DCamila%26date%3D2021%26date%3D2020%26volume%3D148%26issue%3D1%26spage%3D98%26epage%3D104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Stephanie  Wilt</span>, <span class="hlFld-ContribAuthor ">Sean  Kodani</span>, <span class="hlFld-ContribAuthor ">Thanh N.H.  Le</span>, <span class="hlFld-ContribAuthor ">Lato  Nguyen</span>, <span class="hlFld-ContribAuthor ">Nghi  Vo</span>, <span class="hlFld-ContribAuthor ">Tanya  Ly</span>, <span class="hlFld-ContribAuthor ">Mark  Rodriguez</span>, <span class="hlFld-ContribAuthor ">Paula K.  Hudson</span>, <span class="hlFld-ContribAuthor ">Christophe  Morisseau</span>, <span class="hlFld-ContribAuthor ">Bruce D.  Hammock</span>, <span class="hlFld-ContribAuthor ">Stevan  Pecic</span>. </span><span class="cited-content_cbyCitation_article-title">Development of multitarget inhibitors for the treatment of pain: Design, synthesis, biological evaluation and molecular modeling studies. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic Chemistry</span><span> <strong>2020,</strong> <em>103 </em>, 104165. <a href="https://doi.org/10.1016/j.bioorg.2020.104165" title="DOI URL">https://doi.org/10.1016/j.bioorg.2020.104165</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bioorg.2020.104165&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bioorg.2020.104165%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520Chemistry%26atitle%3DDevelopment%252Bof%252Bmultitarget%252Binhibitors%252Bfor%252Bthe%252Btreatment%252Bof%252Bpain%25253A%252BDesign%25252C%252Bsynthesis%25252C%252Bbiological%252Bevaluation%252Band%252Bmolecular%252Bmodeling%252Bstudies%26aulast%3DWilt%26aufirst%3DStephanie%26date%3D2020%26volume%3D103%26spage%3D104165" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Andreas H.  Göller</span>, <span class="hlFld-ContribAuthor ">Lara  Kuhnke</span>, <span class="hlFld-ContribAuthor ">Floriane  Montanari</span>, <span class="hlFld-ContribAuthor ">Anne  Bonin</span>, <span class="hlFld-ContribAuthor ">Sebastian  Schneckener</span>, <span class="hlFld-ContribAuthor ">Antonius  ter Laak</span>, <span class="hlFld-ContribAuthor ">Jörg  Wichard</span>, <span class="hlFld-ContribAuthor ">Mario  Lobell</span>, <span class="hlFld-ContribAuthor ">Alexander  Hillisch</span>. </span><span class="cited-content_cbyCitation_article-title">Bayer’s in silico ADMET platform: a journey of machine learning over the past two decades. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today</span><span> <strong>2020,</strong> <em>25 </em>
                                    (9)
                                     , 1702-1709. <a href="https://doi.org/10.1016/j.drudis.2020.07.001" title="DOI URL">https://doi.org/10.1016/j.drudis.2020.07.001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.drudis.2020.07.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.drudis.2020.07.001%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%26atitle%3DBayer%2525E2%252580%252599s%252Bin%252Bsilico%252BADMET%252Bplatform%25253A%252Ba%252Bjourney%252Bof%252Bmachine%252Blearning%252Bover%252Bthe%252Bpast%252Btwo%252Bdecades%26aulast%3DG%25C3%25B6ller%26aufirst%3DAndreas%2BH.%26date%3D2020%26volume%3D25%26issue%3D9%26spage%3D1702%26epage%3D1709" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Moustafa T.  Gabr</span>, <span class="hlFld-ContribAuthor ">Sefa  Celik</span>, <span class="hlFld-ContribAuthor ">Sevim  Akyuz</span>, <span class="hlFld-ContribAuthor ">Aysen E.  Ozel</span>. </span><span class="cited-content_cbyCitation_article-title">Biological evaluation and pharmacokinetic profiling of a coumarin-benzothiazole hybrid as a new scaffold for human gliomas. </span><span class="cited-content_cbyCitation_journal-name">Medicine in Drug Discovery</span><span> <strong>2020,</strong> <em>7 </em>, 100012. <a href="https://doi.org/10.1016/j.medidd.2020.100012" title="DOI URL">https://doi.org/10.1016/j.medidd.2020.100012</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.medidd.2020.100012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.medidd.2020.100012%26sid%3Dliteratum%253Aachs%26jtitle%3DMedicine%2520in%2520Drug%2520Discovery%26atitle%3DBiological%252Bevaluation%252Band%252Bpharmacokinetic%252Bprofiling%252Bof%252Ba%252Bcoumarin-benzothiazole%252Bhybrid%252Bas%252Ba%252Bnew%252Bscaffold%252Bfor%252Bhuman%252Bgliomas%26aulast%3DGabr%26aufirst%3DMoustafa%2BT.%26date%3D2020%26volume%3D7%26spage%3D100012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">John Oludele  Olanlokun</span>, <span class="hlFld-ContribAuthor ">Fisayo A.  Olotu</span>, <span class="hlFld-ContribAuthor ">Oyebode Thomas  Idowu</span>, <span class="hlFld-ContribAuthor ">Clement  Agoni</span>, <span class="hlFld-ContribAuthor ">Moses Oluwole  David</span>, <span class="hlFld-ContribAuthor ">Mahmoud  Soliman</span>, <span class="hlFld-ContribAuthor ">Olufunso Olabode  Olorunsogo</span>. </span><span class="cited-content_cbyCitation_article-title">In vitro, in silico studies of newly isolated tetrahydro-4-(7-hydroxy-10-methoxy-6, 14-dimethyl-15-m-tolylpentadec-13-enyl) pyran-2-one and isobutyryl acetate compounds from Alstonia boonei stem bark. </span><span class="cited-content_cbyCitation_journal-name">Journal of Molecular Structure</span><span> <strong>2020,</strong> <em>1216 </em>, 128225. <a href="https://doi.org/10.1016/j.molstruc.2020.128225" title="DOI URL">https://doi.org/10.1016/j.molstruc.2020.128225</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.molstruc.2020.128225&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.molstruc.2020.128225%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Molecular%2520Structure%26atitle%3DIn%2525C2%2525A0vitro%25252C%252Bin%252Bsilico%252Bstudies%252Bof%252Bnewly%252Bisolated%252Btetrahydro-4-%2525287-hydroxy-10-methoxy-6%25252C%252B14-dimethyl-15-m-tolylpentadec-13-enyl%252529%252Bpyran-2-one%252Band%252Bisobutyryl%252Bacetate%252Bcompounds%252Bfrom%252BAlstonia%252Bboonei%252Bstem%252Bbark%26aulast%3DOlanlokun%26aufirst%3DJohn%2BOludele%26date%3D2020%26volume%3D1216%26spage%3D128225" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ahmad  Junaid</span>, <span class="hlFld-ContribAuthor ">Felicia Phei Lin  Lim</span>, <span class="hlFld-ContribAuthor ">Edward R. T.  Tiekink</span>, <span class="hlFld-ContribAuthor ">Anton V.  Dolzhenko</span>. </span><span class="cited-content_cbyCitation_article-title">Design, synthesis, and biological evaluation of new 6,
              N
              2
              -diaryl-1,3,5-triazine-2,4-diamines as anticancer agents selectively targeting triple negative breast cancer cells. </span><span class="cited-content_cbyCitation_journal-name">RSC Advances</span><span> <strong>2020,</strong> <em>10 </em>
                                    (43)
                                     , 25517-25528. <a href="https://doi.org/10.1039/D0RA04970K" title="DOI URL">https://doi.org/10.1039/D0RA04970K</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0RA04970K&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0RA04970K%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Advances%26atitle%3DDesign%25252C%252Bsynthesis%25252C%252Band%252Bbiological%252Bevaluation%252Bof%252Bnew%252B6%25252C%252BN%252B2%252B-diaryl-1%25252C3%25252C5-triazine-2%25252C4-diamines%252Bas%252Banticancer%252Bagents%252Bselectively%252Btargeting%252Btriple%252Bnegative%252Bbreast%252Bcancer%252Bcells%26aulast%3DJunaid%26aufirst%3DAhmad%26date%3D2020%26date%3D2020%26volume%3D10%26issue%3D43%26spage%3D25517%26epage%3D25528" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Laura E.  Russell</span>, <span class="hlFld-ContribAuthor ">Mary Alexandra  Schleiff</span>, <span class="hlFld-ContribAuthor ">Eric  Gonzalez</span>, <span class="hlFld-ContribAuthor ">Aaron G.  Bart</span>, <span class="hlFld-ContribAuthor ">Fabio  Broccatelli</span>, <span class="hlFld-ContribAuthor ">Jessica H.  Hartman</span>, <span class="hlFld-ContribAuthor ">W. Griffith  Humphreys</span>, <span class="hlFld-ContribAuthor ">Volker M.  Lauschke</span>, <span class="hlFld-ContribAuthor ">Iain  Martin</span>, <span class="hlFld-ContribAuthor ">Chukwunonso  Nwabufo</span>, <span class="hlFld-ContribAuthor ">Bhagwat  Prasad</span>, <span class="hlFld-ContribAuthor ">Emily E.  Scott</span>, <span class="hlFld-ContribAuthor ">Matthew  Segall</span>, <span class="hlFld-ContribAuthor ">Ryan  Takahashi</span>, <span class="hlFld-ContribAuthor ">Mitchell E.  Taub</span>, <span class="hlFld-ContribAuthor ">Jasleen K.  Sodhi</span>. </span><span class="cited-content_cbyCitation_article-title">Advances in the study of drug metabolism – symposium report of the 12th Meeting of the International Society for the Study of Xenobiotics (ISSX). </span><span class="cited-content_cbyCitation_journal-name">Drug Metabolism Reviews</span><span> <strong>2020,</strong> <em>52 </em>
                                    (3)
                                     , 395-407. <a href="https://doi.org/10.1080/03602532.2020.1765793" title="DOI URL">https://doi.org/10.1080/03602532.2020.1765793</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/03602532.2020.1765793&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F03602532.2020.1765793%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Metabolism%2520Reviews%26atitle%3DAdvances%252Bin%252Bthe%252Bstudy%252Bof%252Bdrug%252Bmetabolism%252B%2525E2%252580%252593%252Bsymposium%252Breport%252Bof%252Bthe%252B12th%252BMeeting%252Bof%252Bthe%252BInternational%252BSociety%252Bfor%252Bthe%252BStudy%252Bof%252BXenobiotics%252B%252528ISSX%252529%26aulast%3DRussell%26aufirst%3DLaura%2BE.%26date%3D2020%26date%3D2020%26volume%3D52%26issue%3D3%26spage%3D395%26epage%3D407" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Carina  Cantrill</span>, <span class="hlFld-ContribAuthor ">Prasoon  Chaturvedi</span>, <span class="hlFld-ContribAuthor ">Caroline  Rynn</span>, <span class="hlFld-ContribAuthor ">Jeannine  Petrig Schaffland</span>, <span class="hlFld-ContribAuthor ">Isabelle  Walter</span>, <span class="hlFld-ContribAuthor ">Matthias B.  Wittwer</span>. </span><span class="cited-content_cbyCitation_article-title">Fundamental aspects of DMPK optimization of targeted protein degraders. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today</span><span> <strong>2020,</strong> <em>25 </em>
                                    (6)
                                     , 969-982. <a href="https://doi.org/10.1016/j.drudis.2020.03.012" title="DOI URL">https://doi.org/10.1016/j.drudis.2020.03.012</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.drudis.2020.03.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.drudis.2020.03.012%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%26atitle%3DFundamental%252Baspects%252Bof%252BDMPK%252Boptimization%252Bof%252Btargeted%252Bprotein%252Bdegraders%26aulast%3DCantrill%26aufirst%3DCarina%26date%3D2020%26volume%3D25%26issue%3D6%26spage%3D969%26epage%3D982" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Michael  Davies</span>, <span class="hlFld-ContribAuthor ">Rhys D.O.  Jones</span>, <span class="hlFld-ContribAuthor ">Ken  Grime</span>, <span class="hlFld-ContribAuthor ">Rasmus  Jansson-Löfmark</span>, <span class="hlFld-ContribAuthor ">Adrian J.  Fretland</span>, <span class="hlFld-ContribAuthor ">Susanne  Winiwarter</span>, <span class="hlFld-ContribAuthor ">Paul  Morgan</span>, <span class="hlFld-ContribAuthor ">Dermot F.  McGinnity</span>. </span><span class="cited-content_cbyCitation_article-title">Improving the Accuracy of Predicted Human Pharmacokinetics: Lessons Learned from the AstraZeneca Drug Pipeline Over Two Decades. </span><span class="cited-content_cbyCitation_journal-name">Trends in Pharmacological Sciences</span><span> <strong>2020,</strong> <em>41 </em>
                                    (6)
                                     , 390-408. <a href="https://doi.org/10.1016/j.tips.2020.03.004" title="DOI URL">https://doi.org/10.1016/j.tips.2020.03.004</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.tips.2020.03.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.tips.2020.03.004%26sid%3Dliteratum%253Aachs%26jtitle%3DTrends%2520in%2520Pharmacological%2520Sciences%26atitle%3DImproving%252Bthe%252BAccuracy%252Bof%252BPredicted%252BHuman%252BPharmacokinetics%25253A%252BLessons%252BLearned%252Bfrom%252Bthe%252BAstraZeneca%252BDrug%252BPipeline%252BOver%252BTwo%252BDecades%26aulast%3DDavies%26aufirst%3DMichael%26date%3D2020%26volume%3D41%26issue%3D6%26spage%3D390%26epage%3D408" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Neha  Maharao</span>, <span class="hlFld-ContribAuthor ">Victor  Antontsev</span>, <span class="hlFld-ContribAuthor ">Matthew  Wright</span>, <span class="hlFld-ContribAuthor ">Jyotika  Varshney</span>. </span><span class="cited-content_cbyCitation_article-title">Entering the era of computationally driven drug development. </span><span class="cited-content_cbyCitation_journal-name">Drug Metabolism Reviews</span><span> <strong>2020,</strong> <em>52 </em>
                                    (2)
                                     , 283-298. <a href="https://doi.org/10.1080/03602532.2020.1726944" title="DOI URL">https://doi.org/10.1080/03602532.2020.1726944</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/03602532.2020.1726944&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F03602532.2020.1726944%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Metabolism%2520Reviews%26atitle%3DEntering%252Bthe%252Bera%252Bof%252Bcomputationally%252Bdriven%252Bdrug%252Bdevelopment%26aulast%3DMaharao%26aufirst%3DNeha%26date%3D2020%26date%3D2020%26volume%3D52%26issue%3D2%26spage%3D283%26epage%3D298" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rob C.  van Wijk</span>, <span class="hlFld-ContribAuthor ">Rami  Ayoun Alsoud</span>, <span class="hlFld-ContribAuthor ">Hans  Lennernäs</span>, <span class="hlFld-ContribAuthor ">Ulrika S. H.  Simonsson</span>. </span><span class="cited-content_cbyCitation_article-title">Model-Informed Drug Discovery and Development Strategy for the Rapid Development of Anti-Tuberculosis Drug Combinations. </span><span class="cited-content_cbyCitation_journal-name">Applied Sciences</span><span> <strong>2020,</strong> <em>10 </em>
                                    (7)
                                     , 2376. <a href="https://doi.org/10.3390/app10072376" title="DOI URL">https://doi.org/10.3390/app10072376</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/app10072376&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fapp10072376%26sid%3Dliteratum%253Aachs%26jtitle%3DApplied%2520Sciences%26atitle%3DModel-Informed%252BDrug%252BDiscovery%252Band%252BDevelopment%252BStrategy%252Bfor%252Bthe%252BRapid%252BDevelopment%252Bof%252BAnti-Tuberculosis%252BDrug%252BCombinations%26aulast%3Dvan%2BWijk%26aufirst%3DRob%2BC.%26date%3D2020%26date%3D2020%26volume%3D10%26issue%3D7%26spage%3D2376" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mohd Ishtiaq  Anasir</span>, <span class="hlFld-ContribAuthor ">Babu  Ramanathan</span>, <span class="hlFld-ContribAuthor ">Chit Laa  Poh</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-Based Design of Antivirals against Envelope Glycoprotein of Dengue Virus. </span><span class="cited-content_cbyCitation_journal-name">Viruses</span><span> <strong>2020,</strong> <em>12 </em>
                                    (4)
                                     , 367. <a href="https://doi.org/10.3390/v12040367" title="DOI URL">https://doi.org/10.3390/v12040367</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/v12040367&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fv12040367%26sid%3Dliteratum%253Aachs%26jtitle%3DViruses%26atitle%3DStructure-Based%252BDesign%252Bof%252BAntivirals%252Bagainst%252BEnvelope%252BGlycoprotein%252Bof%252BDengue%252BVirus%26aulast%3DAnasir%26aufirst%3DMohd%2BIshtiaq%26date%3D2020%26date%3D2020%26volume%3D12%26issue%3D4%26spage%3D367" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Arne  Krüger</span>, <span class="hlFld-ContribAuthor ">Vinicius  Gonçalves Maltarollo</span>, <span class="hlFld-ContribAuthor ">Carsten  Wrenger</span>, <span class="hlFld-ContribAuthor ">Thales  Kronenberger</span>. </span><span class="cited-content_cbyCitation_article-title">ADME Profiling in Drug Discovery and a New Path Paved on Silica. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,<a href="https://doi.org/10.5772/intechopen.86174" title="DOI URL">https://doi.org/10.5772/intechopen.86174</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.5772/intechopen.86174&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.5772%2Fintechopen.86174%26sid%3Dliteratum%253Aachs%26atitle%3DADME%252BProfiling%252Bin%252BDrug%252BDiscovery%252Band%252Ba%252BNew%252BPath%252BPaved%252Bon%252BSilica%26aulast%3DKr%25C3%25BCger%26aufirst%3DArne%26date%3D2020%26date%3D2020%26pub%3DIntechOpen%26atitle%3DDrug%252BDiscovery%252Band%252BDevelopment%252B-%252BNew%252BAdvances%26aulast%3DGaitonde%26aufirst%3DVishwanath%26date%3D2020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Carmela  Bonaccorso</span>, <span class="hlFld-ContribAuthor ">Irina  Naletova</span>, <span class="hlFld-ContribAuthor ">Cristina  Satriano</span>, <span class="hlFld-ContribAuthor ">Giorgia  Spampinato</span>, <span class="hlFld-ContribAuthor ">Vincenza  Barresi</span>, <span class="hlFld-ContribAuthor ">Cosimo G.  Fortuna</span>. </span><span class="cited-content_cbyCitation_article-title">New Di(heteroaryl)ethenes as Apoptotic Anti‐proliferative Agents Towards Breast Cancer: Design, One‐Pot Synthesis and In Vitro Evaluation. </span><span class="cited-content_cbyCitation_journal-name">ChemistrySelect</span><span> <strong>2020,</strong> <em>5 </em>
                                    (8)
                                     , 2581-2587. <a href="https://doi.org/10.1002/slct.201903502" title="DOI URL">https://doi.org/10.1002/slct.201903502</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/slct.201903502&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fslct.201903502%26sid%3Dliteratum%253Aachs%26jtitle%3DChemistrySelect%26atitle%3DNew%252BDi%252528heteroaryl%252529ethenes%252Bas%252BApoptotic%252BAnti%2525E2%252580%252590proliferative%252BAgents%252BTowards%252BBreast%252BCancer%25253A%252BDesign%25252C%252BOne%2525E2%252580%252590Pot%252BSynthesis%252Band%252BIn%252BVitro%252BEvaluation%26aulast%3DBonaccorso%26aufirst%3DCarmela%26date%3D2020%26date%3D2020%26volume%3D5%26issue%3D8%26spage%3D2581%26epage%3D2587" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Anshul  Gupta</span>. </span><span class="cited-content_cbyCitation_article-title">Toxicokinetics, pharmacokinetics, and absorption, distribution, metabolism, and excretion. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,, 483-488. <a href="https://doi.org/10.1016/B978-0-12-813724-6.00044-X" title="DOI URL">https://doi.org/10.1016/B978-0-12-813724-6.00044-X</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-813724-6.00044-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-813724-6.00044-X%26sid%3Dliteratum%253Aachs%26atitle%3DToxicokinetics%25252C%252Bpharmacokinetics%25252C%252Band%252Babsorption%25252C%252Bdistribution%25252C%252Bmetabolism%25252C%252Band%252Bexcretion%26aulast%3DGupta%26aufirst%3DAnshul%26date%3D2020%26spage%3D483%26epage%3D488%26pub%3DElsevier%26atitle%3DInformation%252BResources%252Bin%252BToxicology%26date%3D2020%26volume%3D13" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jie  Shen</span>, <span class="hlFld-ContribAuthor ">Christos A.  Nicolaou</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular property prediction: recent trends in the era of artificial intelligence. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today: Technologies</span><span> <strong>2019,</strong> <em>32-33 </em>, 29-36. <a href="https://doi.org/10.1016/j.ddtec.2020.05.001" title="DOI URL">https://doi.org/10.1016/j.ddtec.2020.05.001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ddtec.2020.05.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ddtec.2020.05.001%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technologies%26atitle%3DMolecular%252Bproperty%252Bprediction%25253A%252Brecent%252Btrends%252Bin%252Bthe%252Bera%252Bof%252Bartificial%252Bintelligence%26aulast%3DShen%26aufirst%3DJie%26date%3D2019%26volume%3D32-33%26spage%3D29%26epage%3D36" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jeremy A.  Leonard</span>, <span class="hlFld-ContribAuthor ">Yu-Mei  Tan</span>. </span><span class="cited-content_cbyCitation_article-title">Tiered approaches for screening and prioritizing chemicals through integration of pharmacokinetics and exposure information with in vitro dose-response data. </span><span class="cited-content_cbyCitation_journal-name">Computational Toxicology</span><span> <strong>2019,</strong> <em>12 </em>, 100101. <a href="https://doi.org/10.1016/j.comtox.2019.100101" title="DOI URL">https://doi.org/10.1016/j.comtox.2019.100101</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.comtox.2019.100101&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.comtox.2019.100101%26sid%3Dliteratum%253Aachs%26jtitle%3DComputational%2520Toxicology%26atitle%3DTiered%252Bapproaches%252Bfor%252Bscreening%252Band%252Bprioritizing%252Bchemicals%252Bthrough%252Bintegration%252Bof%252Bpharmacokinetics%252Band%252Bexposure%252Binformation%252Bwith%252Bin%252Bvitro%252Bdose-response%252Bdata%26aulast%3DLeonard%26aufirst%3DJeremy%2BA.%26date%3D2019%26volume%3D12%26spage%3D100101" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shenaz B.  Bunally</span>, <span class="hlFld-ContribAuthor ">Christopher N.  Luscombe</span>, <span class="hlFld-ContribAuthor ">Robert J.  Young</span>. </span><span class="cited-content_cbyCitation_article-title">Using Physicochemical Measurements to Influence Better Compound Design. </span><span class="cited-content_cbyCitation_journal-name">SLAS DISCOVERY: Advancing the Science of Drug Discovery</span><span> <strong>2019,</strong> <em>24 </em>
                                    (8)
                                     , 791-801. <a href="https://doi.org/10.1177/2472555219859845" title="DOI URL">https://doi.org/10.1177/2472555219859845</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1177/2472555219859845&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1177%2F2472555219859845%26sid%3Dliteratum%253Aachs%26jtitle%3DSLAS%2520DISCOVERY%253A%2520Advancing%2520the%2520Science%2520of%2520Drug%2520Discovery%26atitle%3DUsing%252BPhysicochemical%252BMeasurements%252Bto%252BInfluence%252BBetter%252BCompound%252BDesign%26aulast%3DBunally%26aufirst%3DShenaz%2BB.%26date%3D2019%26date%3D2019%26volume%3D24%26issue%3D8%26spage%3D791%26epage%3D801" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Fabian Pitter  Steinmetz</span>, <span class="hlFld-ContribAuthor ">Carl  Petersson</span>, <span class="hlFld-ContribAuthor ">Ugo  Zanelli</span>, <span class="hlFld-ContribAuthor ">Paul  Czodrowski</span>. </span><span class="cited-content_cbyCitation_article-title">Building
              In Silico
              Models to Trigger Retesting: A Strategy on How to Use Predictive Models to Identify Potentially Incorrect
              In Vitro
              Intrinsic Clearance Results. </span><span class="cited-content_cbyCitation_journal-name">Applied In Vitro Toxicology</span><span> <strong>2019,</strong> <em>5 </em>
                                    (2)
                                     , 86-91. <a href="https://doi.org/10.1089/aivt.2018.0018" title="DOI URL">https://doi.org/10.1089/aivt.2018.0018</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1089/aivt.2018.0018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1089%2Faivt.2018.0018%26sid%3Dliteratum%253Aachs%26jtitle%3DApplied%2520In%2520Vitro%2520Toxicology%26atitle%3DBuilding%252BIn%252BSilico%252BModels%252Bto%252BTrigger%252BRetesting%25253A%252BA%252BStrategy%252Bon%252BHow%252Bto%252BUse%252BPredictive%252BModels%252Bto%252BIdentify%252BPotentially%252BIncorrect%252BIn%252BVitro%252BIntrinsic%252BClearance%252BResults%26aulast%3DSteinmetz%26aufirst%3DFabian%2BPitter%26date%3D2019%26volume%3D5%26issue%3D2%26spage%3D86%26epage%3D91" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Luca  Docci</span>, <span class="hlFld-ContribAuthor ">Neil  Parrott</span>, <span class="hlFld-ContribAuthor ">Stephan  Krähenbühl</span>, <span class="hlFld-ContribAuthor ">Stephen  Fowler</span>. </span><span class="cited-content_cbyCitation_article-title">Application of New Cellular and Microphysiological Systems to Drug Metabolism Optimization and Their Positioning Respective to In Silico Tools. </span><span class="cited-content_cbyCitation_journal-name">SLAS DISCOVERY: Advancing the Science of Drug Discovery</span><span> <strong>2019,</strong> <em>24 </em>
                                    (5)
                                     , 523-536. <a href="https://doi.org/10.1177/2472555219831407" title="DOI URL">https://doi.org/10.1177/2472555219831407</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1177/2472555219831407&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1177%2F2472555219831407%26sid%3Dliteratum%253Aachs%26jtitle%3DSLAS%2520DISCOVERY%253A%2520Advancing%2520the%2520Science%2520of%2520Drug%2520Discovery%26atitle%3DApplication%252Bof%252BNew%252BCellular%252Band%252BMicrophysiological%252BSystems%252Bto%252BDrug%252BMetabolism%252BOptimization%252Band%252BTheir%252BPositioning%252BRespective%252Bto%252BIn%252BSilico%252BTools%26aulast%3DDocci%26aufirst%3DLuca%26date%3D2019%26date%3D2019%26volume%3D24%26issue%3D5%26spage%3D523%26epage%3D536" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rafael  Balaña-Fouce</span>, <span class="hlFld-ContribAuthor ">M. Yolanda  Pérez Pertejo</span>, <span class="hlFld-ContribAuthor ">Bárbara  Domínguez-Asenjo</span>, <span class="hlFld-ContribAuthor ">Camino  Gutiérrez-Corbo</span>, <span class="hlFld-ContribAuthor ">Rosa M.  Reguera</span>. </span><span class="cited-content_cbyCitation_article-title">Walking a tightrope: drug discovery in visceral leishmaniasis. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today</span><span> <strong>2019,</strong> <em>24 </em>
                                    (5)
                                     , 1209-1216. <a href="https://doi.org/10.1016/j.drudis.2019.03.007" title="DOI URL">https://doi.org/10.1016/j.drudis.2019.03.007</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.drudis.2019.03.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.drudis.2019.03.007%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%26atitle%3DWalking%252Ba%252Btightrope%25253A%252Bdrug%252Bdiscovery%252Bin%252Bvisceral%252Bleishmaniasis%26aulast%3DBala%25C3%25B1a-Fouce%26aufirst%3DRafael%26date%3D2019%26volume%3D24%26issue%3D5%26spage%3D1209%26epage%3D1216" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Muhammad Jawad  Nasim</span>, <span class="hlFld-ContribAuthor ">Karolina  Witek</span>, <span class="hlFld-ContribAuthor ">Annamária  Kincses</span>, <span class="hlFld-ContribAuthor ">Ahmad Yaman  Abdin</span>, <span class="hlFld-ContribAuthor ">Ewa  Żesławska</span>, <span class="hlFld-ContribAuthor ">Małgorzata Anna  Marć</span>, <span class="hlFld-ContribAuthor ">Márió  Gajdács</span>, <span class="hlFld-ContribAuthor ">Gabriella  Spengler</span>, <span class="hlFld-ContribAuthor ">Wojciech  Nitek</span>, <span class="hlFld-ContribAuthor ">Gniewomir  Latacz</span>, <span class="hlFld-ContribAuthor ">Elżbieta  Karczewska</span>, <span class="hlFld-ContribAuthor ">Katarzyna  Kieć-Kononowicz</span>, <span class="hlFld-ContribAuthor ">Jadwiga  Handzlik</span>, <span class="hlFld-ContribAuthor ">Claus  Jacob</span>. </span><span class="cited-content_cbyCitation_article-title">Pronounced activity of aromatic selenocyanates against multidrug resistant ESKAPE bacteria. </span><span class="cited-content_cbyCitation_journal-name">New Journal of Chemistry</span><span> <strong>2019,</strong> <em>43 </em>
                                    (15)
                                     , 6021-6031. <a href="https://doi.org/10.1039/C9NJ00563C" title="DOI URL">https://doi.org/10.1039/C9NJ00563C</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C9NJ00563C&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC9NJ00563C%26sid%3Dliteratum%253Aachs%26jtitle%3DNew%2520Journal%2520of%2520Chemistry%26atitle%3DPronounced%252Bactivity%252Bof%252Baromatic%252Bselenocyanates%252Bagainst%252Bmultidrug%252Bresistant%252BESKAPE%252Bbacteria%26aulast%3DNasim%26aufirst%3DMuhammad%2BJawad%26date%3D2019%26date%3D2019%26volume%3D43%26issue%3D15%26spage%3D6021%26epage%3D6031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Fabio  Broccatelli</span>, <span class="hlFld-ContribAuthor ">Cornelis  E.C.A Hop</span>, <span class="hlFld-ContribAuthor ">Matthew  Wright</span>. </span><span class="cited-content_cbyCitation_article-title">Strategies to optimize drug half-life in lead candidate identification. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Drug Discovery</span><span> <strong>2019,</strong> <em>14 </em>
                                    (3)
                                     , 221-230. <a href="https://doi.org/10.1080/17460441.2019.1569625" title="DOI URL">https://doi.org/10.1080/17460441.2019.1569625</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/17460441.2019.1569625&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F17460441.2019.1569625%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Drug%2520Discovery%26atitle%3DStrategies%252Bto%252Boptimize%252Bdrug%252Bhalf-life%252Bin%252Blead%252Bcandidate%252Bidentification%26aulast%3DBroccatelli%26aufirst%3DFabio%26date%3D2019%26date%3D2019%26volume%3D14%26issue%3D3%26spage%3D221%26epage%3D230" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jie  Shen</span>, <span class="hlFld-ContribAuthor ">Ying  Yang</span>, <span class="hlFld-ContribAuthor ">Howard  Broughton</span>, <span class="hlFld-ContribAuthor ">Ian A  Watson</span>, <span class="hlFld-ContribAuthor ">Prashant V  Desai</span>. </span><span class="cited-content_cbyCitation_article-title">High-throughput hydrogen bond strength calculation and its applications in optimizing drug ADME properties. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2019,</strong> <em>11 </em>
                                    (6)
                                     , 511-524. <a href="https://doi.org/10.4155/fmc-2018-0470" title="DOI URL">https://doi.org/10.4155/fmc-2018-0470</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc-2018-0470&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc-2018-0470%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DHigh-throughput%252Bhydrogen%252Bbond%252Bstrength%252Bcalculation%252Band%252Bits%252Bapplications%252Bin%252Boptimizing%252Bdrug%252BADME%252Bproperties%26aulast%3DShen%26aufirst%3DJie%26date%3D2019%26volume%3D11%26issue%3D6%26spage%3D511%26epage%3D524" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Emilio S.  Petito</span>, <span class="hlFld-ContribAuthor ">David J.R.  Foster</span>, <span class="hlFld-ContribAuthor ">Michael B.  Ward</span>, <span class="hlFld-ContribAuthor ">Matthew J.  Sykes</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular Modeling Approaches for the Prediction of Selected Pharmacokinetic Properties. </span><span class="cited-content_cbyCitation_journal-name">Current Topics in Medicinal Chemistry</span><span> <strong>2019,</strong> <em>18 </em>
                                    (26)
                                     , 2230-2238. <a href="https://doi.org/10.2174/1568026619666181220105726" title="DOI URL">https://doi.org/10.2174/1568026619666181220105726</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2174/1568026619666181220105726&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2174%2F1568026619666181220105726%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Topics%2520in%2520Medicinal%2520Chemistry%26atitle%3DMolecular%252BModeling%252BApproaches%252Bfor%252Bthe%252BPrediction%252Bof%252BSelected%252BPharmacokinetic%252BProperties%26aulast%3DPetito%26aufirst%3DEmilio%2BS.%26date%3D2019%26volume%3D18%26issue%3D26%26spage%3D2230%26epage%3D2238" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Franco  Lombardo</span>, <span class="hlFld-ContribAuthor ">Giuliano  Berellini</span>, <span class="hlFld-ContribAuthor ">R. Scott  Obach</span>. </span><span class="cited-content_cbyCitation_article-title">Trend Analysis of a Database of Intravenous Pharmacokinetic Parameters in Humans for 1352 Drug Compounds. </span><span class="cited-content_cbyCitation_journal-name">Drug Metabolism and Disposition</span><span> <strong>2018,</strong> <em>46 </em>
                                    (11)
                                     , 1466-1477. <a href="https://doi.org/10.1124/dmd.118.082966" title="DOI URL">https://doi.org/10.1124/dmd.118.082966</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1124/dmd.118.082966&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1124%2Fdmd.118.082966%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Metabolism%2520and%2520Disposition%26atitle%3DTrend%252BAnalysis%252Bof%252Ba%252BDatabase%252Bof%252BIntravenous%252BPharmacokinetic%252BParameters%252Bin%252BHumans%252Bfor%252B1352%252BDrug%252BCompounds%26aulast%3DLombardo%26aufirst%3DFranco%26date%3D2018%26date%3D2018%26volume%3D46%26issue%3D11%26spage%3D1466%26epage%3D1477" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Edward J.  Griffen</span>, <span class="hlFld-ContribAuthor ">Alexander G.  Dossetter</span>, <span class="hlFld-ContribAuthor ">Andrew G.  Leach</span>, <span class="hlFld-ContribAuthor ">Shane  Montague</span>. </span><span class="cited-content_cbyCitation_article-title">Can we accelerate medicinal chemistry by augmenting the chemist with Big Data and artificial intelligence?. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today</span><span> <strong>2018,</strong> <em>23 </em>
                                    (7)
                                     , 1373-1384. <a href="https://doi.org/10.1016/j.drudis.2018.03.011" title="DOI URL">https://doi.org/10.1016/j.drudis.2018.03.011</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.drudis.2018.03.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.drudis.2018.03.011%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%26atitle%3DCan%252Bwe%252Baccelerate%252Bmedicinal%252Bchemistry%252Bby%252Baugmenting%252Bthe%252Bchemist%252Bwith%252BBig%252BData%252Band%252Bartificial%252Bintelligence%25253F%26aulast%3DGriffen%26aufirst%3DEdward%2BJ.%26date%3D2018%26volume%3D23%26issue%3D7%26spage%3D1373%26epage%3D1384" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/medium/jm-2017-004878_0013.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/large/jm-2017-004878_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00487&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/medium/jm-2017-004878_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/large/jm-2017-004878_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Changes observed across all projects in microsomal stability at Genentech (GNE), solubility at AstraZeneca (AZ), and CYP3A4 time-dependent inhibition at Eli Lilly (Lilly) after adoption of <i>in silico</i> models.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/large/jm-2017-004878_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00487&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/medium/jm-2017-004878_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/large/jm-2017-004878_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Results of the survey among industrial scientists involved in <i>in silico</i> ADME.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/large/jm-2017-004878_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00487&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/medium/jm-2017-004878_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/large/jm-2017-004878_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Prospective predictions for the AbbVie MDCK-MDR1 efflux ratio model. The compounds are color coded by prediction confidence (purple, high; blue, medium; green, low confidence).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/large/jm-2017-004878_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00487&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/medium/jm-2017-004878_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/large/jm-2017-004878_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Scatter and box plots of sample mean and standard deviation for compounds with three or more repeat measurements for log <i>D</i><sub>7.4</sub> and CL<sub>int</sub> in human liver microsomes (HLMs) in the AZ internal database. Box-plots capture the population of the standard deviations values in different discrete intervals (bins).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/large/jm-2017-004878_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00487&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/medium/jm-2017-004878_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/large/jm-2017-004878_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Example of Spotfire visualization to facilitate assessment of applicability of <i>in silico</i> models for a given chemical series and predicted ADME profile of virtual compounds. (a) Predicted vs measured data for three key <i>in vitro</i> ADME properties: rat liver microsomal CL<sub>int,u</sub>, aqueous solubility at pH 6, and MDCK permeability category. For each property, predicted data are presented along the <i>Y</i>-axis and the bars are colored based on the measured data. Number of compounds in each category is labeled for a given bar. (b) Comparison of the above-mentioned key predicted properties for existing compounds and designed (virtual) compounds to facilitate prioritization for synthesis and testing.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/large/jm-2017-004878_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00487&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/medium/jm-2017-004878_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/large/jm-2017-004878_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Use of MMP data to generate new ideas that can then be predicted with global models to enable informed decision making in design.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/large/jm-2017-004878_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00487&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/medium/jm-2017-004878_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/large/jm-2017-004878_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. (a) An example matched molecular pair (MMP). (b) Example transforms for the MMP in (a). R<sub>1</sub> and R<sub>2</sub> refer to any possible chemical group attached to the substructure of interest.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/large/jm-2017-004878_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00487&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/medium/jm-2017-004878_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/large/jm-2017-004878_0008.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. HLM CL<sub>int</sub> MMP data for the 5-pyrimidyl to 2-pyrimidyl transform. Dashed lines represent ±2-fold change.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/large/jm-2017-004878_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00487&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/medium/jm-2017-004878_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/large/jm-2017-004878_0009.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Mean MMP changes for HLM, passive permeability, and P-gp efflux for phenyl → six-membered aromatic ring transforms. Change values are represented in log<sub>10</sub> units. Colors represent the intensity and the directionality of the change in property (red for increase, blue for decrease).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/large/jm-2017-004878_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00487&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig10" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/medium/jm-2017-004878_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/large/jm-2017-004878_0010.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. SAR matrix by Gupta-Ostermann and Bajorath,<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a>reproduced from <i>F1000Research</i>. Three series with three compounds each, A–C, D–F, and G–I, are shown with activity (red). Step 1: All compounds are fragmented at a single bond, yielding common core and MMPs within the series. Step 2: Cores are fragmented to obtain core MMPs. The SAR matrix is then generated with rows and columns representing cores and substituents, respectively. Compounds present in the data set are indicated by filled cells.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/large/jm-2017-004878_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00487&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig11" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/medium/jm-2017-004878_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/large/jm-2017-004878_0011.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. Visual representation of hERG inhibition model. Red colors are associated with strong inhibition and blue with absence of inhibition. Orange reflects a weaker association compared to red.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/large/jm-2017-004878_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00487&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig12" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 12</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/medium/jm-2017-004878_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/large/jm-2017-004878_0012.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 12. Results of the survey addressing future development in the <i>in silico</i> ADME-PK fields.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b00487/20171117/images/large/jm-2017-004878_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00487&amp;id=fig12"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i29">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_50154" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_50154" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 71 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">van de Waterbeemd, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gifford, E.</span><span> </span><span class="NLM_article-title">ADMET in silico modeling: towards prediction paradise?</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x"> (</span><span class="NLM_issue">3</span><span class="NLM_x">) </span> <span class="NLM_fpage">192</span><span class="NLM_x">–</span> <span class="NLM_lpage">204</span><span class="refDoi"> DOI: 10.1038/nrd1032</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.1038%2Fnrd1032" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2003&pages=192-204&issue=3&author=H.+van+de+Waterbeemdauthor=E.+Gifford&title=ADMET+in+silico+modeling%3A+towards+prediction+paradise%3F&doi=10.1038%2Fnrd1032"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrd1032&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd1032%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2Bde%2BWaterbeemd%26aufirst%3DH.%26aulast%3DGifford%26aufirst%3DE.%26atitle%3DADMET%2520in%2520silico%2520modeling%253A%2520towards%2520prediction%2520paradise%253F%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2003%26volume%3D2%26issue%3D3%26spage%3D192%26epage%3D204%26doi%3D10.1038%2Fnrd1032" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Lombardo, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gifford, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shalaeva, M. Y.</span><span> </span><span class="NLM_article-title"><i>In silico</i> ADME prediction: data, models, facts and myths</span> <span class="citation_source-journal">Mini-Rev. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x"> (</span><span class="NLM_issue">8</span><span class="NLM_x">) </span> <span class="NLM_fpage">861</span><span class="NLM_x">–</span> <span class="NLM_lpage">875</span><span class="refDoi"> DOI: 10.2174/1389557033487629</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.2174%2F1389557033487629" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2003&pages=861-875&issue=8&author=F.+Lombardoauthor=E.+Giffordauthor=M.+Y.+Shalaeva&title=In+silico+ADME+prediction%3A+data%2C+models%2C+facts+and+myths&doi=10.2174%2F1389557033487629"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.2174%2F1389557033487629&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1389557033487629%26sid%3Dliteratum%253Aachs%26aulast%3DLombardo%26aufirst%3DF.%26aulast%3DGifford%26aufirst%3DE.%26aulast%3DShalaeva%26aufirst%3DM.%2BY.%26atitle%3DIn%2520silico%2520ADME%2520prediction%253A%2520data%252C%2520models%252C%2520facts%2520and%2520myths%26jtitle%3DMini-Rev.%2520Med.%2520Chem.%26date%3D2003%26volume%3D3%26issue%3D8%26spage%3D861%26epage%3D875%26doi%3D10.2174%2F1389557033487629" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Gleeson, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hersey, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hannongbua, S.</span><span> </span><span class="NLM_article-title">In silico ADME Models: a general assessment of their utility in drug discovery applications</span> <span class="citation_source-journal">Curr. Top. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x"> (</span><span class="NLM_issue">4</span><span class="NLM_x">) </span> <span class="NLM_fpage">358</span><span class="NLM_x">–</span> <span class="NLM_lpage">381</span><span class="refDoi"> DOI: 10.2174/156802611794480927</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.2174%2F156802611794480927" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=21320065" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BC3MXkslSnur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2011&pages=358-381&issue=4&author=M.+P.+Gleesonauthor=A.+Herseyauthor=S.+Hannongbua&title=In+silico+ADME+Models%3A+a+general+assessment+of+their+utility+in+drug+discovery+applications&doi=10.2174%2F156802611794480927"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">In-silico ADME models: a general assessment of their utility in drug discovery applications</span></div><div class="casAuthors">Gleeson, M. Paul; Hersey, Anne; Hannongbua, Supa</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">358-381</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  ADME prediction is an extremely challenging area as many of the properties we try to predict are a result of multiple physiol. processes.  In this review we consider how in-silico predictions of ADME processes can be used to help bias medicinal chem. into more ideal areas of property space, minimizing the no. of compds. needed to be synthesized to obtain the required biochem./physico-chem. profile.  While such models are not sufficiently accurate to act as a replacement for in-vivo or in-vitro methods, in-silico methods nevertheless can help us to understand the underlying physico-chem. dependencies of the different ADME properties, and thus can give us inspiration on how to optimize them.  Many global in-silico ADME models (i.e generated on large, diverse datasets) have been reported in the literature.  In this paper we selectively review representatives from each distinct class and discuss their relative utility in drug discovery.  For each ADME parameter, we limit our discussion to the most recent, most predictive or most insightful examples in the literature to highlight the current state of the art.  In each case we briefly summarize the different types of models available for each parameter (i.e simple rules, physico-chem. and 3D based QSAR predictions), their overall accuracy and the underlying SAR.  We also discuss the utility of the models as related to lead generation and optimization phases of discovery research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNyakFYvcjtrVg90H21EOLACvtfcHk0ljpn4VwP-fI5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXkslSnur4%253D&md5=5ca56a568815370a3b04833db45c0a9b</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.2174%2F156802611794480927&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F156802611794480927%26sid%3Dliteratum%253Aachs%26aulast%3DGleeson%26aufirst%3DM.%2BP.%26aulast%3DHersey%26aufirst%3DA.%26aulast%3DHannongbua%26aufirst%3DS.%26atitle%3DIn%2520silico%2520ADME%2520Models%253A%2520a%2520general%2520assessment%2520of%2520their%2520utility%2520in%2520drug%2520discovery%2520applications%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2011%26volume%3D11%26issue%3D4%26spage%3D358%26epage%3D381%26doi%3D10.2174%2F156802611794480927" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Cheng, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang, Y.</span><span> </span><span class="NLM_article-title"><i>In silico</i> ADMET prediction: recent advances, current challenges and future trends</span> <span class="citation_source-journal">Curr. Top. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x"> (</span><span class="NLM_issue">11</span><span class="NLM_x">) </span> <span class="NLM_fpage">1273</span><span class="NLM_x">–</span> <span class="NLM_lpage">1283</span><span class="refDoi"> DOI: 10.2174/15680266113139990033</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.2174%2F15680266113139990033" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=23675935" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1WnsbzM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2013&pages=1273-1283&issue=11&author=F.+Chengauthor=W.+Liauthor=G.+Liuauthor=Y.+Tang&title=In+silico+ADMET+prediction%3A+recent+advances%2C+current+challenges+and+future+trends&doi=10.2174%2F15680266113139990033"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">In Silico ADMET Prediction: Recent Advances, Current Challenges and Future Trends</span></div><div class="casAuthors">Cheng, Feixiong; Li, Weihua; Liu, Guixia; Tang, Yun</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1273-1289</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  There are numerous small mol. compds. around us to affect our health, such as drugs, pesticides, food additives, industrial chems., and environmental pollutants.  Over decades, properties related to absorption, distribution, metab., excretion, and toxicity (ADMET) have become one of the most important issues to assess the effects or risks of these compds. on human body.  Recent high-rate drug withdrawals increase the pressure on regulators and pharmaceutical industry to improve preclin. safety testing.  Since in vivo and in vitro evaluations are costly and laborious, in silico techniques have been widely used to est. these properties.  In this review, we would briefly describe the recent advances of in silico ADMET prediction, with emphasis on substructure pattern recognition method that we developed recently.  Challenges and limitations in the area of in silico ADMET prediction were further discussed, such as application domain of models, models validation techniques, and global vs. local models.  At last, several new promising research directions were provided, such as computational systems toxicol. (toxicogenomics), data-integration and meta-decision making systems, which could be used for systemic in silico ADMET prediction in drug discovery and hazard risk assessment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGom7qcGJZNvrbVg90H21EOLACvtfcHk0ljpn4VwP-fI5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1WnsbzM&md5=9edae4014f84c6a0f95d1598a9d4ccc7</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.2174%2F15680266113139990033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F15680266113139990033%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DF.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DG.%26aulast%3DTang%26aufirst%3DY.%26atitle%3DIn%2520silico%2520ADMET%2520prediction%253A%2520recent%2520advances%252C%2520current%2520challenges%2520and%2520future%2520trends%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2013%26volume%3D13%26issue%3D11%26spage%3D1273%26epage%3D1283%26doi%3D10.2174%2F15680266113139990033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Manallack, D. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tehan, B. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gancia, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hudson, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ford, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Livingstone, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whitley, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pitt, W. R.</span><span> </span><span class="NLM_article-title">A consensus neural network-based technique for discriminating soluble and poorly soluble compounds</span> <span class="citation_source-journal">J. Chem. Inf. Comput. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">43</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">674</span><span class="NLM_x">–</span> <span class="NLM_lpage">679</span><span class="refDoi"> DOI: 10.1021/ci0202741</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci0202741" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BD3sXivFWkuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2003&pages=674-679&issue=2&author=D.+T.+Manallackauthor=B.+G.+Tehanauthor=E.+Ganciaauthor=B.+D.+Hudsonauthor=M.+G.+Fordauthor=D.+J.+Livingstoneauthor=D.+C.+Whitleyauthor=W.+R.+Pitt&title=A+consensus+neural+network-based+technique+for+discriminating+soluble+and+poorly+soluble+compounds&doi=10.1021%2Fci0202741"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">A Consensus Neural Network-Based Technique for Discriminating Soluble and Poorly Soluble Compounds</span></div><div class="casAuthors">Manallack, David T.; Tehan, Benjamin G.; Gancia, Emanuela; Hudson, Brian D.; Ford, Martyn G.; Livingstone, David J.; Whitley, David C.; Pitt, Will R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">674-679</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">BCUT [Burden, CAS, and University of Texas] descriptors, defined as eigenvalues of modified connectivity matrixes, have traditionally been applied to drug design tasks such as defining receptor relevant subspaces to assist in compd. selections.  In this paper we present studies of consensus neural networks trained on BCUTs to discriminate compds. with poor aq. soly. from those with reasonable soly.  This level was set at 0.1 mg/mL on advice from drug formulation and drug discovery scientists.  By applying strict criteria to the insoly. predictions, approx. 95% of compds. are classified correctly.  For compds. whose predictions have a lower level of confidence, further parameters are examd. in order to flag those considered to possess unsuitable biopharmaceutical and physicochem. properties.  This approach is not designed to be applied in isolation but is intended to be used as a filter in the selection of screening candidates, compd. purchases, and the application of synthetic priorities to combinatorial libraries.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFfQPsH6Nj77Vg90H21EOLACvtfcHk0ljpn4VwP-fI5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXivFWkuw%253D%253D&md5=576c750212ce64c9414401d52cc0156c</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1021%2Fci0202741&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci0202741%26sid%3Dliteratum%253Aachs%26aulast%3DManallack%26aufirst%3DD.%2BT.%26aulast%3DTehan%26aufirst%3DB.%2BG.%26aulast%3DGancia%26aufirst%3DE.%26aulast%3DHudson%26aufirst%3DB.%2BD.%26aulast%3DFord%26aufirst%3DM.%2BG.%26aulast%3DLivingstone%26aufirst%3DD.%2BJ.%26aulast%3DWhitley%26aufirst%3DD.%2BC.%26aulast%3DPitt%26aufirst%3DW.%2BR.%26atitle%3DA%2520consensus%2520neural%2520network-based%2520technique%2520for%2520discriminating%2520soluble%2520and%2520poorly%2520soluble%2520compounds%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D2003%26volume%3D43%26issue%3D2%26spage%3D674%26epage%3D679%26doi%3D10.1021%2Fci0202741" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Kuhn, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guba, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hert, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banner, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bissantz, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ceccarelli, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haap, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Körner, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuglstatter, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lerner, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattei, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neidhart, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pinard, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rudolph, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schulz-Gasch, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woltering, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stahl, M.</span><span> </span><span class="NLM_article-title">A real-world perspective on molecular design</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x"> (</span><span class="NLM_issue">9</span><span class="NLM_x">) </span> <span class="NLM_fpage">4087</span><span class="NLM_x">–</span> <span class="NLM_lpage">4102</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01875</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01875" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BC28XisVyjtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=4087-4102&issue=9&author=B.+Kuhnauthor=W.+Gubaauthor=J.+Hertauthor=D.+Bannerauthor=C.+Bissantzauthor=S.+Ceccarelliauthor=W.+Haapauthor=M.+K%C3%B6rnerauthor=A.+Kuglstatterauthor=A.+Lernerauthor=P.+Matteiauthor=W.+Neidhartauthor=E.+Pinardauthor=M.+G.+Rudolphauthor=T.+Schulz-Gaschauthor=T.+Wolteringauthor=M.+Stahl&title=A+real-world+perspective+on+molecular+design&doi=10.1021%2Facs.jmedchem.5b01875"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">A Real-World Perspective on Molecular Design</span></div><div class="casAuthors">Kuhn, Bernd; Guba, Wolfgang; Hert, Jerome; Banner, David; Bissantz, Caterina; Ceccarelli, Simona; Haap, Wolfgang; Korner, Matthias; Kuglstatter, Andreas; Lerner, Christian; Mattei, Patrizio; Neidhart, Werner; Pinard, Emmanuel; Rudolph, Markus G.; Schulz-Gasch, Tanja; Woltering, Thomas; Stahl, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4087-4102</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We present a series of small mol. drug discovery case studies where computational methods were prospectively employed to impact Roche research projects, with the aim of highlighting those methods that provide real added value.  Our brief accounts encompass a broad range of methods and techniques applied to a variety of enzymes and receptors.  Most of these are based on judicious application of knowledge about mol. conformations and interactions: filling of lipophilic pockets to gain affinity or selectivity, addn. of polar substituents, scaffold hopping, transfer of SAR, conformation anal., and mol. overlays.  A case study of sequence-driven focused screening is presented to illustrate how appropriate preprocessing of information enables effective exploitation of prior knowledge.  We conclude that qual. statements enabling chemists to focus on promising regions of chem. space are often more impactful than quant. prediction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxOLDuNAX5U7Vg90H21EOLACvtfcHk0ljBA8exbMUU0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XisVyjtb0%253D&md5=0b81f2a0e9996c5dae44e61e5488ba79</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01875&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01875%26sid%3Dliteratum%253Aachs%26aulast%3DKuhn%26aufirst%3DB.%26aulast%3DGuba%26aufirst%3DW.%26aulast%3DHert%26aufirst%3DJ.%26aulast%3DBanner%26aufirst%3DD.%26aulast%3DBissantz%26aufirst%3DC.%26aulast%3DCeccarelli%26aufirst%3DS.%26aulast%3DHaap%26aufirst%3DW.%26aulast%3DK%25C3%25B6rner%26aufirst%3DM.%26aulast%3DKuglstatter%26aufirst%3DA.%26aulast%3DLerner%26aufirst%3DA.%26aulast%3DMattei%26aufirst%3DP.%26aulast%3DNeidhart%26aufirst%3DW.%26aulast%3DPinard%26aufirst%3DE.%26aulast%3DRudolph%26aufirst%3DM.%2BG.%26aulast%3DSchulz-Gasch%26aufirst%3DT.%26aulast%3DWoltering%26aufirst%3DT.%26aulast%3DStahl%26aufirst%3DM.%26atitle%3DA%2520real-world%2520perspective%2520on%2520molecular%2520design%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26issue%3D9%26spage%3D4087%26epage%3D4102%26doi%3D10.1021%2Facs.jmedchem.5b01875" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Muegge, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bentzien, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mukherjee, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hughes, R.</span><span> </span><span class="NLM_article-title">Automatically updating predictive modeling workflows support decision-making in drug design</span> <span class="citation_source-journal">Future Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x"> (</span><span class="NLM_issue">14</span><span class="NLM_x">) </span> <span class="NLM_fpage">1779</span><span class="NLM_x">–</span> <span class="NLM_lpage">1796</span><span class="refDoi"> DOI: 10.4155/fmc-2016-0070</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.4155%2Ffmc-2016-0070" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=27584594" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVKhsrrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2016&pages=1779-1796&issue=14&author=I.+Mueggeauthor=J.+Bentzienauthor=P.+Mukherjeeauthor=R.+Hughes&title=Automatically+updating+predictive+modeling+workflows+support+decision-making+in+drug+design&doi=10.4155%2Ffmc-2016-0070"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Automatically updating predictive modeling workflows support decision-making in drug design</span></div><div class="casAuthors">Muegge, Ingo; Bentzien, Joerg; Mukherjee, Prasenjit; Hughes, Robert O.</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">1779-1796</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">Using predictive models for early decision-making in drug discovery has become std. practice.  We suggest that model building needs to be automated with min. input and low tech. maintenance requirements.  Models perform best when tailored to answering specific compd. optimization related questions.  If qual. answers are required, 2-bin classification models are preferred.  Integrating predictive modeling results with structural information stimulates better decision making.  For in silico models supporting rapid structure-activity relationship cycles the performance deteriorates within weeks.  Frequent automated updates of predictive models ensure best predictions.  Consensus between multiple modeling approaches increases the prediction confidence.  Combining qualified and nonqualified data optimally uses all available information.  Dose predictions provide a holistic alternative to multiple individual property predictions for reaching complex decisions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNAKHon4nFZLVg90H21EOLACvtfcHk0ljBA8exbMUU0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVKhsrrJ&md5=0a243c46bf3df9b2b1308d5f5fd9c37d</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.4155%2Ffmc-2016-0070&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc-2016-0070%26sid%3Dliteratum%253Aachs%26aulast%3DMuegge%26aufirst%3DI.%26aulast%3DBentzien%26aufirst%3DJ.%26aulast%3DMukherjee%26aufirst%3DP.%26aulast%3DHughes%26aufirst%3DR.%26atitle%3DAutomatically%2520updating%2520predictive%2520modeling%2520workflows%2520support%2520decision-making%2520in%2520drug%2520design%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2016%26volume%3D8%26issue%3D14%26spage%3D1779%26epage%3D1796%26doi%3D10.4155%2Ffmc-2016-0070" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Cumming, J. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davis, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muresan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haeberlein, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, H.</span><span> </span><span class="NLM_article-title">Chemical predictive modeling to improve compound quality</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x"> (</span><span class="NLM_issue">12</span><span class="NLM_x">) </span> <span class="NLM_fpage">948</span><span class="NLM_x">–</span> <span class="NLM_lpage">962</span><span class="refDoi"> DOI: 10.1038/nrd4128</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.1038%2Fnrd4128" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2013&pages=948-962&issue=12&author=J.+G.+Cummingauthor=A.+M.+Davisauthor=S.+Muresanauthor=M.+Haeberleinauthor=H.+Chen&title=Chemical+predictive+modeling+to+improve+compound+quality&doi=10.1038%2Fnrd4128"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnrd4128&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4128%26sid%3Dliteratum%253Aachs%26aulast%3DCumming%26aufirst%3DJ.%2BG.%26aulast%3DDavis%26aufirst%3DA.%2BM.%26aulast%3DMuresan%26aufirst%3DS.%26aulast%3DHaeberlein%26aufirst%3DM.%26aulast%3DChen%26aufirst%3DH.%26atitle%3DChemical%2520predictive%2520modeling%2520to%2520improve%2520compound%2520quality%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2013%26volume%3D12%26issue%3D12%26spage%3D948%26epage%3D962%26doi%3D10.1038%2Fnrd4128" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Aliagas, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gobbi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heffron, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, M.-L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ortwine, D. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zak, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khojasteh, S. C.</span><span> </span><span class="NLM_article-title">A probabilistic method to report predictions from a human liver microsomes stability QSAR model: a practical tool for drug discovery</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x"> (</span><span class="NLM_issue">4</span><span class="NLM_x">) </span> <span class="NLM_fpage">327</span><span class="NLM_x">–</span> <span class="NLM_lpage">338</span><span class="refDoi"> DOI: 10.1007/s10822-015-9838-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.1007%2Fs10822-015-9838-3" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2015&pages=327-338&issue=4&author=I.+Aliagasauthor=A.+Gobbiauthor=T.+Heffronauthor=M.-L.+Leeauthor=D.+F.+Ortwineauthor=M.+Zakauthor=S.+C.+Khojasteh&title=A+probabilistic+method+to+report+predictions+from+a+human+liver+microsomes+stability+QSAR+model%3A+a+practical+tool+for+drug+discovery&doi=10.1007%2Fs10822-015-9838-3"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1007%2Fs10822-015-9838-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-015-9838-3%26sid%3Dliteratum%253Aachs%26aulast%3DAliagas%26aufirst%3DI.%26aulast%3DGobbi%26aufirst%3DA.%26aulast%3DHeffron%26aufirst%3DT.%26aulast%3DLee%26aufirst%3DM.-L.%26aulast%3DOrtwine%26aufirst%3DD.%2BF.%26aulast%3DZak%26aufirst%3DM.%26aulast%3DKhojasteh%26aufirst%3DS.%2BC.%26atitle%3DA%2520probabilistic%2520method%2520to%2520report%2520predictions%2520from%2520a%2520human%2520liver%2520microsomes%2520stability%2520QSAR%2520model%253A%2520a%2520practical%2520tool%2520for%2520drug%2520discovery%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2015%26volume%3D29%26issue%3D4%26spage%3D327%26epage%3D338%26doi%3D10.1007%2Fs10822-015-9838-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Desai, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sawada, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watson, I. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raub, T. J.</span><span> </span><span class="NLM_article-title">Integration of <i>in silico</i> and <i>in vitro</i> tools for scaffold optimization during drug discovery: predicting P-Glycoprotein efflux</span> <span class="citation_source-journal">Mol. Pharmaceutics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x"> (</span><span class="NLM_issue">4</span><span class="NLM_x">) </span> <span class="NLM_fpage">1249</span><span class="NLM_x">–</span> <span class="NLM_lpage">1261</span><span class="refDoi"> DOI: 10.1021/mp300555n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/mp300555n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1aju7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2013&pages=1249-1261&issue=4&author=P.+V.+Desaiauthor=G.+A.+Sawadaauthor=I.+A.+Watsonauthor=T.+J.+Raub&title=Integration+of+in+silico+and+in+vitro+tools+for+scaffold+optimization+during+drug+discovery%3A+predicting+P-Glycoprotein+efflux&doi=10.1021%2Fmp300555n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Integration of in Silico and in Vitro Tools for Scaffold Optimization during Drug Discovery: Predicting P-Glycoprotein Efflux</span></div><div class="casAuthors">Desai, Prashant V.; Sawada, Geri A.; Watson, Ian A.; Raub, Thomas J.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmaceutics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1249-1261</span>CODEN:
                <span class="NLM_cas:coden">MPOHBP</span>;
        ISSN:<span class="NLM_cas:issn">1543-8384</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In silico tools are regularly utilized for designing and prioritizing compds. to address challenges related to drug metab. and pharmacokinetics (DMPK) during the process of drug discovery.  P-Glycoprotein (P-gp) is a member of the ATP-binding cassette (ABC) transporters with broad substrate specificity that plays a significant role in absorption and distribution of drugs that are P-gp substrates.  As a result, screening for P-gp transport has now become routine in the drug discovery process.  Typically, bidirectional permeability assays are employed to assess in vitro P-gp efflux.  In this article, we use P-gp as an example to illustrate a well-validated methodol. to effectively integrate in silico and in vitro tools to identify and resolve key barriers during the early stages of drug discovery.  A detailed account of development and application of in silico tools such as simple guidelines based on physicochem. properties and more complex quant. structure-activity relationship (QSAR) models is provided.  The tools were developed based on structurally diverse data for more than 2000 compds. generated using a robust P-gp substrate assay over the past several years.  Anal. of physicochem. properties revealed a significantly lower proportion (<10%) of P-gp substrates among the compds. with topol. polar surface area (TPSA) <60 Å2 and the most basic cpKa <8.  In contrast, this proportion of substrates was greater than 75% for compds. with TPSA >60 Å2 and the most basic cpKa >8.  Among the various QSAR models evaluated to predict P-gp efflux, the Bagging model provided optimum prediction performance for prospective validation based on chronol. test sets.  Four sequential versions of the model were built with increasing nos. of compds. to train the models as new data became available.  Except for the first version with the smallest training set, the QSAR models exhibited robust prediction profiles with pos. prediction values (PPV) and neg. prediction values (NPV) exceeding 80%.  The QSAR model demonstrated better concordance with the manual P-gp substrate assay than an automated P-gp substrate screen.  The in silico and the in vitro tools have been effectively integrated during early stages of drug discovery to resolve P-gp-related challenges exemplified by several case studies.  Key learning based on our experience with P-gp can be widely applicable across other DMPK-related challenges.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5Q5Y8HssxH7Vg90H21EOLACvtfcHk0limluL4-oMF_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1aju7o%253D&md5=3bd5239b80270c0f4ed85dffb86704ed</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fmp300555n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fmp300555n%26sid%3Dliteratum%253Aachs%26aulast%3DDesai%26aufirst%3DP.%2BV.%26aulast%3DSawada%26aufirst%3DG.%2BA.%26aulast%3DWatson%26aufirst%3DI.%2BA.%26aulast%3DRaub%26aufirst%3DT.%2BJ.%26atitle%3DIntegration%2520of%2520in%2520silico%2520and%2520in%2520vitro%2520tools%2520for%2520scaffold%2520optimization%2520during%2520drug%2520discovery%253A%2520predicting%2520P-Glycoprotein%2520efflux%26jtitle%3DMol.%2520Pharmaceutics%26date%3D2013%26volume%3D10%26issue%3D4%26spage%3D1249%26epage%3D1261%26doi%3D10.1021%2Fmp300555n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Liu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hop, C. E. C. A.</span><span> </span><span class="NLM_article-title">Do we need to optimize plasma protein and tissue binding in drug discovery?</span> <span class="citation_source-journal">Curr. Top. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x"> (</span><span class="NLM_issue">4</span><span class="NLM_x">) </span> <span class="NLM_fpage">450</span><span class="NLM_x">–</span> <span class="NLM_lpage">466</span><span class="refDoi"> DOI: 10.2174/156802611794480918</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.2174%2F156802611794480918" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=21320069" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BC3MXkslSnuro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2011&pages=450-466&issue=4&author=X.+Liuauthor=C.+Chenauthor=C.+E.+C.+A.+Hop&title=Do+we+need+to+optimize+plasma+protein+and+tissue+binding+in+drug+discovery%3F&doi=10.2174%2F156802611794480918"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Do we need to optimize plasma protein and tissue binding in drug discovery?</span></div><div class="casAuthors">Liu, Xingrong; Chen, Cuiping; Hop, Cornelis E. C. A.</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">450-466</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  It is a commonly accepted assumption that only unbound drug mols. are available to interact with their targets.  In order to achieve high unbound plasma drug concn., it seems obvious to design compds. with low plasma protein binding.  Similarly to achieve high unbound tissue concn., we apparently need compds. with low tissue binding.  Our theor. anal. and exptl. data demonstrate that unbound plasma concn. is not detd. by plasma protein binding but by hepatic intrinsic clearance after oral dose, and unbound tissue concn. is not detd. by tissue binding but detd. by unbound plasma concn. and transport properties at the blood-tissue barrier.  Redn. of plasma and tissue protein binding for a compd. will increase the unbound concn. in vitro but may not increase its unbound plasma or tissue concn. in vivo after oral administration.  We conclude that plasma protein and tissue binding are essential parameters to understand pharmacokinetics and pharmacodynamics but they should not be optimized independently in drug discovery.  Instead we should focus on reducing clearance and efflux at the blood-tissue barrier to increase in vivo plasma and tissue unbound concn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFwT9zhofIgLVg90H21EOLACvtfcHk0limluL4-oMF_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXkslSnuro%253D&md5=de19ad9bb14b369795484abe096bf226</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.2174%2F156802611794480918&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F156802611794480918%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DC.%26aulast%3DHop%26aufirst%3DC.%2BE.%2BC.%2BA.%26atitle%3DDo%2520we%2520need%2520to%2520optimize%2520plasma%2520protein%2520and%2520tissue%2520binding%2520in%2520drug%2520discovery%253F%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2011%26volume%3D11%26issue%3D4%26spage%3D450%26epage%3D466%26doi%3D10.2174%2F156802611794480918" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Di, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rong, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, B.</span><span> </span><span class="NLM_article-title">Demystifying brain penetration in central nervous system drug discovery</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">2</span><span class="NLM_x">–</span> <span class="NLM_lpage">12</span><span class="refDoi"> DOI: 10.1021/jm301297f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm301297f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFamtrrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=2-12&issue=1&author=L.+Diauthor=R.+Rongauthor=B.+Feng&title=Demystifying+brain+penetration+in+central+nervous+system+drug+discovery&doi=10.1021%2Fjm301297f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Demystifying Brain Penetration in Central Nervous System Drug Discovery</span></div><div class="casAuthors">Di, Li; Rong, Haojing; Feng, Bo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">2-12</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  This Perspective provides important concepts about the blood-brain barrier (BBB) in drug discovery and how they should be applied effectively in designing successful CNS drugs.  Key parameters for brain penetration are discussed, including unbound brain concn., unbound brain-to-plasma ratio, BBB permeability, fraction unbound in brain and plasma, and transporters.  Results from a retrospective anal. of 32 Pfizer CNS clin. drug candidates are described.  Frequently encountered misconceptions about brain penetration in drug discovery programs are clarified.  Strategies and guidance are provided to enhance or minimize brain exposure for CNS or peripheral targets, resp.  Recommendations for screening methodologies and a cascade in assessing brain penetration potential are presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmurs7cfju37Vg90H21EOLACvtfcHk0limluL4-oMF_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFamtrrJ&md5=8b88e2044bf41898ab40c45e18f2a20d</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Fjm301297f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm301297f%26sid%3Dliteratum%253Aachs%26aulast%3DDi%26aufirst%3DL.%26aulast%3DRong%26aufirst%3DR.%26aulast%3DFeng%26aufirst%3DB.%26atitle%3DDemystifying%2520brain%2520penetration%2520in%2520central%2520nervous%2520system%2520drug%2520discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26issue%3D1%26spage%3D2%26epage%3D12%26doi%3D10.1021%2Fjm301297f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Mannhold, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poda, G. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ostermann, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tetko, I. V.</span><span> </span><span class="NLM_article-title">Calculation of molecular lipophilicity: state-of-the-art and comparison of log P methods on more than 96,000 compounds</span> <span class="citation_source-journal">J. Pharm. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">98</span><span class="NLM_x"> (</span><span class="NLM_issue">3</span><span class="NLM_x">) </span> <span class="NLM_fpage">861</span><span class="NLM_x">–</span> <span class="NLM_lpage">893</span><span class="refDoi"> DOI: 10.1002/jps.21494</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.1002%2Fjps.21494" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2009&pages=861-893&issue=3&author=R.+Mannholdauthor=G.+I.+Podaauthor=C.+Ostermannauthor=I.+V.+Tetko&title=Calculation+of+molecular+lipophilicity%3A+state-of-the-art+and+comparison+of+log+P+methods+on+more+than+96%2C000+compounds&doi=10.1002%2Fjps.21494"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1002%2Fjps.21494&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjps.21494%26sid%3Dliteratum%253Aachs%26aulast%3DMannhold%26aufirst%3DR.%26aulast%3DPoda%26aufirst%3DG.%2BI.%26aulast%3DOstermann%26aufirst%3DC.%26aulast%3DTetko%26aufirst%3DI.%2BV.%26atitle%3DCalculation%2520of%2520molecular%2520lipophilicity%253A%2520state-of-the-art%2520and%2520comparison%2520of%2520log%2520P%2520methods%2520on%2520more%2520than%252096%252C000%2520compounds%26jtitle%3DJ.%2520Pharm.%2520Sci.%26date%3D2009%26volume%3D98%26issue%3D3%26spage%3D861%26epage%3D893%26doi%3D10.1002%2Fjps.21494" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Wenlock, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carlsson, L. A.</span><span> </span><span class="NLM_article-title">How experimental errors influence drug metabolism and pharmacokinetic QSAR/QSPR models</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">125</span><span class="NLM_x">–</span> <span class="NLM_lpage">134</span><span class="refDoi"> DOI: 10.1021/ci500535s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci500535s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFGku7%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=125-134&issue=1&author=M.+C.+Wenlockauthor=L.+A.+Carlsson&title=How+experimental+errors+influence+drug+metabolism+and+pharmacokinetic+QSAR%2FQSPR+models&doi=10.1021%2Fci500535s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">How Experimental Errors Influence Drug Metabolism and Pharmacokinetic QSAR/QSPR Models</span></div><div class="casAuthors">Wenlock, Mark C.; Carlsson, Lars A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">125-134</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We consider the impact of gross, systematic, and random exptl. errors in relation to their impact on the predictive ability of QSAR/QSPR DMPK models used within early drug discovery.  Models whose training sets contain fewer but repeatedly measured data points, with a defined threshold for the random error, resulted in prediction improvements ranging from 3.3% to 23.0% for an external test set, compared to models built from training sets in which the mols. were defined by single measurements.  Similarly, models built on data with low exptl. uncertainty, compared to those built on data with higher exptl. uncertainty, gave prediction improvements ranging from 3.3% to 27.5%.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoy1uwqWjCVt7Vg90H21EOLACvtfcHk0ljvXDLitN9JoA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFGku7%252FF&md5=39f934233f69bd825c17145922771abb</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fci500535s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci500535s%26sid%3Dliteratum%253Aachs%26aulast%3DWenlock%26aufirst%3DM.%2BC.%26aulast%3DCarlsson%26aufirst%3DL.%2BA.%26atitle%3DHow%2520experimental%2520errors%2520influence%2520drug%2520metabolism%2520and%2520pharmacokinetic%2520QSAR%252FQSPR%2520models%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26volume%3D55%26issue%3D1%26spage%3D125%26epage%3D134%26doi%3D10.1021%2Fci500535s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Winiwarter, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Middleton, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Courtney, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lindmark, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Page, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clark, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Landqvist, C.</span><span> </span><span class="NLM_article-title">Time dependent analysis of assay comparability: a novel approach to understand intra- and inter-site variability over time</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x"> (</span><span class="NLM_issue">9</span><span class="NLM_x">) </span> <span class="NLM_fpage">795</span><span class="NLM_x">–</span> <span class="NLM_lpage">807</span><span class="refDoi"> DOI: 10.1007/s10822-015-9836-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.1007%2Fs10822-015-9836-5" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2015&pages=795-807&issue=9&author=S.+Winiwarterauthor=B.+Middletonauthor=B.+Jonesauthor=P.+Courtneyauthor=B.+Lindmarkauthor=K.+M.+Pageauthor=A.+Clarkauthor=C.+Landqvist&title=Time+dependent+analysis+of+assay+comparability%3A+a+novel+approach+to+understand+intra-+and+inter-site+variability+over+time&doi=10.1007%2Fs10822-015-9836-5"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1007%2Fs10822-015-9836-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-015-9836-5%26sid%3Dliteratum%253Aachs%26aulast%3DWiniwarter%26aufirst%3DS.%26aulast%3DMiddleton%26aufirst%3DB.%26aulast%3DJones%26aufirst%3DB.%26aulast%3DCourtney%26aufirst%3DP.%26aulast%3DLindmark%26aufirst%3DB.%26aulast%3DPage%26aufirst%3DK.%2BM.%26aulast%3DClark%26aufirst%3DA.%26aulast%3DLandqvist%26aufirst%3DC.%26atitle%3DTime%2520dependent%2520analysis%2520of%2520assay%2520comparability%253A%2520a%2520novel%2520approach%2520to%2520understand%2520intra-%2520and%2520inter-site%2520variability%2520over%2520time%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2015%26volume%3D29%26issue%3D9%26spage%3D795%26epage%3D807%26doi%3D10.1007%2Fs10822-015-9836-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Kramer, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fuchs, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whitebread, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gedeck, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liedl, K. R.</span><span> </span><span class="NLM_article-title">Matched molecular pair analysis: significance and the impact of experimental uncertainty</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x"> (</span><span class="NLM_issue">9</span><span class="NLM_x">) </span> <span class="NLM_fpage">3786</span><span class="NLM_x">–</span> <span class="NLM_lpage">3802</span><span class="refDoi"> DOI: 10.1021/jm500317a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm500317a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=3786-3802&issue=9&author=C.+Kramerauthor=J.+E.+Fuchsauthor=S.+Whitebreadauthor=P.+Gedeckauthor=K.+R.+Liedl&title=Matched+molecular+pair+analysis%3A+significance+and+the+impact+of+experimental+uncertainty&doi=10.1021%2Fjm500317a"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fjm500317a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm500317a%26sid%3Dliteratum%253Aachs%26aulast%3DKramer%26aufirst%3DC.%26aulast%3DFuchs%26aufirst%3DJ.%2BE.%26aulast%3DWhitebread%26aufirst%3DS.%26aulast%3DGedeck%26aufirst%3DP.%26aulast%3DLiedl%26aufirst%3DK.%2BR.%26atitle%3DMatched%2520molecular%2520pair%2520analysis%253A%2520significance%2520and%2520the%2520impact%2520of%2520experimental%2520uncertainty%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26issue%3D9%26spage%3D3786%26epage%3D3802%26doi%3D10.1021%2Fjm500317a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Segall, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Champness, E. J.</span><span> </span><span class="NLM_article-title">The challenges of making decisions using uncertain data</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x"> (</span><span class="NLM_issue">9</span><span class="NLM_x">) </span> <span class="NLM_fpage">809</span><span class="NLM_x">–</span> <span class="NLM_lpage">816</span><span class="refDoi"> DOI: 10.1007/s10822-015-9855-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.1007%2Fs10822-015-9855-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=26126976" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtV2ktrbI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2015&pages=809-816&issue=9&author=M.+D.+Segallauthor=E.+J.+Champness&title=The+challenges+of+making+decisions+using+uncertain+data&doi=10.1007%2Fs10822-015-9855-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">The challenges of making decisions using uncertain data</span></div><div class="casAuthors">Segall, Matthew D.; Champness, Edmund J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">809-816</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">All of the exptl. compd. data with which we work have significant uncertainties, due to imperfect correlations between exptl. systems and the ultimate in vivo properties of compds. and the inherent variability in exptl. conditions.  When using these data to make decisions, it is essential that these uncertainties are taken into account to avoid making inappropriate decisions in the selection of compds., which can lead to wasted effort and missed opportunities.  In this paper we will consider approaches to rigorously account for uncertainties when selecting between compds. or assessing compds. against a property criterion; first for an individual measurement of a single property and then for multiple measurements of a property for the same compd.  We will then explore how uncertainties in multiple properties can be combined when assessing compds. against a profile of criteria, a process known as multi-parameter optimization.  This guides rigorous decision-making using complex, uncertain data to focus on compds. with the best chance of success, while avoiding missed opportunities by inappropriately rejecting compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3y3PbHH6LibVg90H21EOLACvtfcHk0lhyMExrdfcCRg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtV2ktrbI&md5=9ac2c9e15bf62f3e1b254816041a8484</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1007%2Fs10822-015-9855-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-015-9855-2%26sid%3Dliteratum%253Aachs%26aulast%3DSegall%26aufirst%3DM.%2BD.%26aulast%3DChampness%26aufirst%3DE.%2BJ.%26atitle%3DThe%2520challenges%2520of%2520making%2520decisions%2520using%2520uncertain%2520data%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2015%26volume%3D29%26issue%3D9%26spage%3D809%26epage%3D816%26doi%3D10.1007%2Fs10822-015-9855-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Beck, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seeliger, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kriegl, J. M.</span><span> </span><span class="NLM_article-title">The impact of data integrity on decision making in early lead discovery</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x"> (</span><span class="NLM_issue">9</span><span class="NLM_x">) </span> <span class="NLM_fpage">911</span><span class="NLM_x">–</span> <span class="NLM_lpage">921</span><span class="refDoi"> DOI: 10.1007/s10822-015-9871-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.1007%2Fs10822-015-9871-2" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2015&pages=911-921&issue=9&author=B.+Beckauthor=D.+Seeligerauthor=J.+M.+Kriegl&title=The+impact+of+data+integrity+on+decision+making+in+early+lead+discovery&doi=10.1007%2Fs10822-015-9871-2"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1007%2Fs10822-015-9871-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-015-9871-2%26sid%3Dliteratum%253Aachs%26aulast%3DBeck%26aufirst%3DB.%26aulast%3DSeeliger%26aufirst%3DD.%26aulast%3DKriegl%26aufirst%3DJ.%2BM.%26atitle%3DThe%2520impact%2520of%2520data%2520integrity%2520on%2520decision%2520making%2520in%2520early%2520lead%2520discovery%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2015%26volume%3D29%26issue%3D9%26spage%3D911%26epage%3D921%26doi%3D10.1007%2Fs10822-015-9871-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Winiwarter, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonham, N. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ax, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hallberg, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lennernäs, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karlén, A.</span><span> </span><span class="NLM_article-title">Correlation of human jejunal permeability (<i>in vivo</i>) of drugs with experimentally and theoretically derived parameters. A multivariate data analysis approach</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x"> (</span><span class="NLM_issue">25</span><span class="NLM_x">) </span> <span class="NLM_fpage">4939</span><span class="NLM_x">–</span> <span class="NLM_lpage">4949</span><span class="refDoi"> DOI: 10.1021/jm9810102</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm9810102" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADyaK1cXntFKltLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=1998&pages=4939-4949&issue=25&author=S.+Winiwarterauthor=N.+M.+Bonhamauthor=F.+Axauthor=A.+Hallbergauthor=H.+Lennern%C3%A4sauthor=A.+Karl%C3%A9n&title=Correlation+of+human+jejunal+permeability+%28in+vivo%29+of+drugs+with+experimentally+and+theoretically+derived+parameters.+A+multivariate+data+analysis+approach&doi=10.1021%2Fjm9810102"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Correlation of Human Jejunal Permeability (in Vivo) of Drugs with Experimentally and Theoretically Derived Parameters. A Multivariate Data Analysis Approach</span></div><div class="casAuthors">Winiwarter, Susanne; Bonham, Nicholas M.; Ax, Fredrik; Hallberg, Anders; Lennernaes, Hans; Karlen, Anders</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">4939-4949</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The effective permeability (Peff) in the human jejunum (in vivo) of 22 structurally diverse compds. was correlated with both exptl. detd. lipophilicity values and calcd. mol. descriptors.  The permeability data were previously obtained by using a regional in vivo perfusion system in the proximal jejunum in humans as part of constructing a biopharmaceutical classification system for oral immediate-release products.  PKa, log P, and, where relevant, log Pion values were detd. using the pH-metric technique.  On the basis of these expts., log D values were calcd. at pH 5.5, 6.5, and 7.4.  Multivariate data anal. was used to derive models that correlate passive intestinal permeability to physicochem. descriptors.  The best model obtained, based on 13 passively transcellularly absorbed compds., used the variables HBD (no. of hydrogen bond donors), PSA (polar surface area), and either log D5.5 or log D6.5 (octanol/water distribution coeff. at pH 5.5 and 6.5, resp.).  Statistically good models for predicting human in vivo Peff values were also obtained by using only HBD and PSA or HBD, PSA, and CLOGP.  These models can be used to predict passive intestinal membrane diffusion in humans for compds. that fit within the defined property space.  We used one of the models obtained above to predict the log Peff values for an external validation set consisting of 34 compds.  A good correlation with the absorption data of these compds. was found.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpz3O2C2bSpILVg90H21EOLACvtfcHk0lhyMExrdfcCRg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXntFKltLw%253D&md5=2a7128943e31080e2e839eaacdc51f0f</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fjm9810102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm9810102%26sid%3Dliteratum%253Aachs%26aulast%3DWiniwarter%26aufirst%3DS.%26aulast%3DBonham%26aufirst%3DN.%2BM.%26aulast%3DAx%26aufirst%3DF.%26aulast%3DHallberg%26aufirst%3DA.%26aulast%3DLennern%25C3%25A4s%26aufirst%3DH.%26aulast%3DKarl%25C3%25A9n%26aufirst%3DA.%26atitle%3DCorrelation%2520of%2520human%2520jejunal%2520permeability%2520%2528in%2520vivo%2529%2520of%2520drugs%2520with%2520experimentally%2520and%2520theoretically%2520derived%2520parameters.%2520A%2520multivariate%2520data%2520analysis%2520approach%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1998%26volume%3D41%26issue%3D25%26spage%3D4939%26epage%3D4949%26doi%3D10.1021%2Fjm9810102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Palm, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luthman, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ros, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gråsjö, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Artursson, P.</span><span> </span><span class="NLM_article-title">Effect of molecular charge on intestinal epithelial drug transport: pH-dependent transport of cationic drugs</span> <span class="citation_source-journal">J. Pharmacol. Exp.Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">291</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">435</span><span class="NLM_x">–</span> <span class="NLM_lpage">443</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10525056" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADyaK1MXntFChtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=291&publication_year=1999&pages=435-443&issue=2&author=K.+Palmauthor=K.+Luthmanauthor=J.+Rosauthor=J.+Gr%C3%A5sj%C3%B6author=P.+Artursson&title=Effect+of+molecular+charge+on+intestinal+epithelial+drug+transport%3A+pH-dependent+transport+of+cationic+drugs"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Effect of molecular charge on intestinal epithelial drug transport: pH-dependent transport of cationic drugs</span></div><div class="casAuthors">Palm, Katrin; Luthman, Kristina; Ros, Jenny; Grasjo, Johan; Artursson, Per</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacology and Experimental Therapeutics</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">291</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">435-443</span>CODEN:
                <span class="NLM_cas:coden">JPETAB</span>;
        ISSN:<span class="NLM_cas:issn">0022-3565</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">The aim of this study was to investigate the effect of ionization on drug transport across the intestinal epithelium to include this effect in structure-absorption relationships.  The pH-dependent permeation of one rapidly (alfentanil) and one slowly (cimetidine) transported basic model drug across Caco-2 cell monolayers was investigated.  Both drugs had pKa values in the physiol. pH range.  The permeability coeffs. (Pc) of the model drugs were obtained at varying apical buffer pHs, thus varying the degree of drug ionization (from 5 to 95%).  The relation between Pc and the fraction of the drug in un-ionized form (fu) was analyzed to delineate the permeability coeffs. of the un-ionized (Pc,u) and ionized (Pc,i) forms of the drugs.  Theor. ests. of the pKa values were also calcd. from ionization energies for each model compd.  For both drugs, a linear increase in Pc was obsd. with increasing fu.  Transport of the un-ionized form was 150- and 30-fold more rapid than transport of the ionized form for alfentanil and cimetidine, resp.  However, when fu < 0.1, the contribution of the ionized form was significant.  Because fu is <0.1 over the entire physiol. pH range for a large no. of drugs, these results will have implications on predictions of in vivo intestinal drug absorption both from in vitro studies in cell cultures and from computed structural properties of drug mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpT8-YOYJTmtrVg90H21EOLACvtfcHk0lhyMExrdfcCRg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXntFChtr8%253D&md5=c161433bb0305cb8d195107a16982bbc</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPalm%26aufirst%3DK.%26aulast%3DLuthman%26aufirst%3DK.%26aulast%3DRos%26aufirst%3DJ.%26aulast%3DGr%25C3%25A5sj%25C3%25B6%26aufirst%3DJ.%26aulast%3DArtursson%26aufirst%3DP.%26atitle%3DEffect%2520of%2520molecular%2520charge%2520on%2520intestinal%2520epithelial%2520drug%2520transport%253A%2520pH-dependent%2520transport%2520of%2520cationic%2520drugs%26jtitle%3DJ.%2520Pharmacol.%2520Exp.Ther.%26date%3D1999%26volume%3D291%26issue%3D2%26spage%3D435%26epage%3D443" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Clark, D. E.</span><span> </span><span class="NLM_article-title"><i>In silico</i> prediction of blood–brain barrier permeation</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x"> (</span><span class="NLM_issue">20</span><span class="NLM_x">) </span> <span class="NLM_fpage">927</span><span class="NLM_x">–</span> <span class="NLM_lpage">933</span><span class="refDoi"> DOI: 10.1016/S1359-6446(03)02827-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.1016%2FS1359-6446%2803%2902827-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=14554156" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BD3sXnvFSiu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2003&pages=927-933&issue=20&author=D.+E.+Clark&title=In+silico+prediction+of+blood%E2%80%93brain+barrier+permeation&doi=10.1016%2FS1359-6446%2803%2902827-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">In silico prediction of blood-brain barrier permeation</span></div><div class="casAuthors">Clark, David E.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">927-933</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">This review examines the progress that is being made towards the in silico prediction of brain permeation.  Following a brief introduction to the blood-brain barrier, the datasets currently available for in silico modeling are discussed.  Recent developments in in silico models of brain permeation are summarized in the context of the current state of the art in prediction accuracy.  An anal. of recent models is presented, focusing on what such models reveal about the mol. properties that det. brain permeation.  The review concludes by presenting the current key issues in this area of research, noting in particular, the paucity of brain permeation data available for modeling.  Finally, possible future directions are suggested.  The degree to which a mol. penetrates the brain can be crucial in drug discovery.  This review discusses progress towards in silico prediction of this important property.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGroQLUXz_-f0LVg90H21EOLACvtfcHk0ljUnNyE7oZV4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXnvFSiu70%253D&md5=bc30c4e968c843a644f1147f90001261</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2FS1359-6446%2803%2902827-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1359-6446%252803%252902827-7%26sid%3Dliteratum%253Aachs%26aulast%3DClark%26aufirst%3DD.%2BE.%26atitle%3DIn%2520silico%2520prediction%2520of%2520blood%25E2%2580%2593brain%2520barrier%2520permeation%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2003%26volume%3D8%26issue%3D20%26spage%3D927%26epage%3D933%26doi%3D10.1016%2FS1359-6446%2803%2902827-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Kelder, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grootenhuis, P. D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bayada, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Delbressine, L. P. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ploemen, J.-P.</span><span> </span><span class="NLM_article-title">Polar molecular surface as a dominating determinant for oral absorption and brain penetration of drugs</span> <span class="citation_source-journal">Pharm. Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x"> (</span><span class="NLM_issue">10</span><span class="NLM_x">) </span> <span class="NLM_fpage">1514</span><span class="NLM_x">–</span> <span class="NLM_lpage">1519</span><span class="refDoi"> DOI: 10.1023/A:1015040217741</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.1023%2FA%3A1015040217741" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=1999&pages=1514-1519&issue=10&author=J.+Kelderauthor=P.+D.+J.+Grootenhuisauthor=D.+M.+Bayadaauthor=L.+P.+C.+Delbressineauthor=J.-P.+Ploemen&title=Polar+molecular+surface+as+a+dominating+determinant+for+oral+absorption+and+brain+penetration+of+drugs&doi=10.1023%2FA%3A1015040217741"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1023%2FA%3A1015040217741&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1015040217741%26sid%3Dliteratum%253Aachs%26aulast%3DKelder%26aufirst%3DJ.%26aulast%3DGrootenhuis%26aufirst%3DP.%2BD.%2BJ.%26aulast%3DBayada%26aufirst%3DD.%2BM.%26aulast%3DDelbressine%26aufirst%3DL.%2BP.%2BC.%26aulast%3DPloemen%26aufirst%3DJ.-P.%26atitle%3DPolar%2520molecular%2520surface%2520as%2520a%2520dominating%2520determinant%2520for%2520oral%2520absorption%2520and%2520brain%2520penetration%2520of%2520drugs%26jtitle%3DPharm.%2520Res.%26date%3D1999%26volume%3D16%26issue%3D10%26spage%3D1514%26epage%3D1519%26doi%3D10.1023%2FA%3A1015040217741" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Wager, T. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hou, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verhoest, P. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Villalobos, A.</span><span> </span><span class="NLM_article-title">A. Moving beyond rules: the development of a central nervous system multiparameter optimization (CNS MPO) approach to enable alignment of druglike properties</span> <span class="citation_source-journal">ACS Chem. Neurosci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x"> (</span><span class="NLM_issue">6</span><span class="NLM_x">) </span> <span class="NLM_fpage">435</span><span class="NLM_x">–</span> <span class="NLM_lpage">449</span><span class="refDoi"> DOI: 10.1021/cn100008c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cn100008c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjvVejtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2010&pages=435-449&issue=6&author=T.+T.+Wagerauthor=X.+Houauthor=P.+R.+Verhoestauthor=A.+Villalobos&title=A.+Moving+beyond+rules%3A+the+development+of+a+central+nervous+system+multiparameter+optimization+%28CNS+MPO%29+approach+to+enable+alignment+of+druglike+properties&doi=10.1021%2Fcn100008c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Moving beyond Rules: The Development of a Central Nervous System Multiparameter Optimization (CNS MPO) Approach To Enable Alignment of Druglike Properties</span></div><div class="casAuthors">Wager, Travis T.; Hou, Xinjun; Verhoest, Patrick R.; Villalobos, Anabella</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Neuroscience</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">435-449</span>CODEN:
                <span class="NLM_cas:coden">ACNCDM</span>;
        ISSN:<span class="NLM_cas:issn">1948-7193</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The interplay among commonly used physicochem. properties in drug design was examd. and utilized to create a prospective design tool focused on the alignment of key druglike attributes.  Using a set of six physicochem. parameters ((a) lipophilicity, calcd. partition coeff. (ClogP); (b) calcd. distribution coeff. at pH = 7.4 (ClogD); (c) mol. wt. (MW); (d) topol. polar surface area (TPSA); (e) no. of hydrogen bond donors (HBD); (f) most basic center (pKa)), a druglikeness central nervous system multiparameter optimization (CNS MPO) algorithm was built and applied to a set of marketed CNS drugs (N = 119) and Pfizer CNS candidates (N = 108), as well as to a large diversity set of Pfizer proprietary compds. (N = 11 303).  The novel CNS MPO algorithm showed that 74% of marketed CNS drugs displayed a high CNS MPO score (MPO desirability score ≥ 4, using a scale of 0-6), in comparison to 60% of the Pfizer CNS candidates.  This anal. suggests that this algorithm could potentially be used to identify compds. with a higher probability of successfully testing hypotheses in the clinic.  In addn., a relationship between an increasing CNS MPO score and alignment of key in vitro attributes of drug discovery (favorable permeability, P-glycoprotein (P-gp) efflux, metabolic stability, and safety) was seen in the marketed CNS drug set, the Pfizer candidate set, and the Pfizer proprietary diversity set.  The CNS MPO scoring function offers advantages over hard cutoffs or utilization of single parameters to optimize structure-activity relationships (SAR) by expanding medicinal chem. design space through a holistic assessment approach.  Based on six physicochem. properties commonly used by medicinal chemists, the CNS MPO function may be used prospectively at the design stage to accelerate the identification of compds. with increased probability of success.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqB3Mvkgm_X5bVg90H21EOLACvtfcHk0ljUnNyE7oZV4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjvVejtr8%253D&md5=e94998f65a420be6ee4dec4a987cb983</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Fcn100008c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcn100008c%26sid%3Dliteratum%253Aachs%26aulast%3DWager%26aufirst%3DT.%2BT.%26aulast%3DHou%26aufirst%3DX.%26aulast%3DVerhoest%26aufirst%3DP.%2BR.%26aulast%3DVillalobos%26aufirst%3DA.%26atitle%3DA.%2520Moving%2520beyond%2520rules%253A%2520the%2520development%2520of%2520a%2520central%2520nervous%2520system%2520multiparameter%2520optimization%2520%2528CNS%2520MPO%2529%2520approach%2520to%2520enable%2520alignment%2520of%2520druglike%2520properties%26jtitle%3DACS%2520Chem.%2520Neurosci.%26date%3D2010%26volume%3D1%26issue%3D6%26spage%3D435%26epage%3D449%26doi%3D10.1021%2Fcn100008c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Lipinski, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lombardo, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dominy, B. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feeney, P. J.</span><span> </span><span class="NLM_article-title">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span> <span class="citation_source-journal">Adv. Drug Delivery Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x"> (</span><span class="NLM_issue">1–3</span><span class="NLM_x">) </span> <span class="NLM_fpage">3</span><span class="NLM_x">–</span> <span class="NLM_lpage">25</span><span class="refDoi"> DOI: 10.1016/S0169-409X(96)00423-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.1016%2FS0169-409X%2896%2900423-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADyaK2sXktlKlsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=1997&pages=3-25&issue=1%E2%80%933&author=C.+A.+Lipinskiauthor=F.+Lombardoauthor=B.+W.+Dominyauthor=P.+J.+Feeney&title=Experimental+and+computational+approaches+to+estimate+solubility+and+permeability+in+drug+discovery+and+development+settings&doi=10.1016%2FS0169-409X%2896%2900423-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span></div><div class="casAuthors">Lipinski, Christopher A.; Lombardo, Franco; Dominy, Beryl W.; Feeney, Paul J.</div><div class="citationInfo"><span class="NLM_cas:title">Advanced Drug Delivery Reviews</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">3-25</span>CODEN:
                <span class="NLM_cas:coden">ADDREP</span>;
        ISSN:<span class="NLM_cas:issn">0169-409X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review with 50 refs.  Exptl. and computational approaches to est. soly. and permeability in discovery and development settings are described.  In the discovery setting 'the rule of 5' predicts that poor absorption or permeation is more likely when there are more than 5 H-bond donors, 10 H-bond acceptors, the mol. wt. (MWT) is >500 and the calcd. Log P (CLogP) is >5.  Computational methodol. for the rule-based Moriguchi Log P (MLogP) calcn. is described.  Turbidimetric soly. measurement is described and applied to known drugs.  High throughput screening (HTS) leads tend to have higher MWT and Log P and lower turbidimetric soly. than leads in the pre-HTS era.  In the development setting, soly. calcns. focus on exact value prediction and are difficult because of polymorphism.  Recent work on linear free energy relationships and Log P approaches are critically reviewed.  Useful predictions are possible in closely related analog series when coupled with exptl. thermodn. soly. measurements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGon9gEeZ7cboLVg90H21EOLACvtfcHk0ljUnNyE7oZV4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXktlKlsQ%253D%253D&md5=405f70b0594d428f1275e1d56642cd3a</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2FS0169-409X%2896%2900423-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0169-409X%252896%252900423-1%26sid%3Dliteratum%253Aachs%26aulast%3DLipinski%26aufirst%3DC.%2BA.%26aulast%3DLombardo%26aufirst%3DF.%26aulast%3DDominy%26aufirst%3DB.%2BW.%26aulast%3DFeeney%26aufirst%3DP.%2BJ.%26atitle%3DExperimental%2520and%2520computational%2520approaches%2520to%2520estimate%2520solubility%2520and%2520permeability%2520in%2520drug%2520discovery%2520and%2520development%2520settings%26jtitle%3DAdv.%2520Drug%2520Delivery%2520Rev.%26date%3D1997%26volume%3D23%26issue%3D1%25E2%2580%25933%26spage%3D3%26epage%3D25%26doi%3D10.1016%2FS0169-409X%2896%2900423-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Hill, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, R. J.</span><span> </span><span class="NLM_article-title">Getting physical in drug discovery: a contemporary perspective on solubility and hydrophobicity</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x"> (</span><span class="NLM_issue">15–16</span><span class="NLM_x">) </span> <span class="NLM_fpage">648</span><span class="NLM_x">–</span> <span class="NLM_lpage">655</span><span class="refDoi"> DOI: 10.1016/j.drudis.2010.05.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.1016%2Fj.drudis.2010.05.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=20570751" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVeksb3P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2010&pages=648-655&issue=15%E2%80%9316&author=A.+P.+Hillauthor=R.+J.+Young&title=Getting+physical+in+drug+discovery%3A+a+contemporary+perspective+on+solubility+and+hydrophobicity&doi=10.1016%2Fj.drudis.2010.05.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Getting physical in drug discovery: a contemporary perspective on solubility and hydrophobicity</span></div><div class="casAuthors">Hill, Alan P.; Young, Robert J.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">15/16</span>),
    <span class="NLM_cas:pages">648-655</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Suboptimal phys. properties have been identified as a particular shortcoming of compds. in contemporary drug discovery, contributing to high attrition levels.  An anal. of the relationship between hydrophobicity (calcd. and measured) and ∼100 k measured kinetic soly. values has been undertaken.  In line with the General Soly. Equation, ests. of hydrophobicity, particularly ACD c log D pH7.4, give a useful indication of the likely soly. classification of particular mols.  Taking ACD c log D pH7.4 values together with the no. of arom. rings in a given mol. provides enhanced prediction.  The Soly. Forecast Index' (SFI = c log D pH7.4 + #Ar) is proposed as a simple, yet effective, guide to predicting soly.  Moreover, anal. of measured distribution/partition coeff. values highlighted statistically significant shortcomings in the applicability of octanol/water as a model system for hydrophobicity detn. with poorly sol. compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdtQyfNjbFUrVg90H21EOLACvtfcHk0lhhSvwG42bKZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVeksb3P&md5=7b201081c5586a6ea4b7af00d507d462</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2010.05.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2010.05.016%26sid%3Dliteratum%253Aachs%26aulast%3DHill%26aufirst%3DA.%2BP.%26aulast%3DYoung%26aufirst%3DR.%2BJ.%26atitle%3DGetting%2520physical%2520in%2520drug%2520discovery%253A%2520a%2520contemporary%2520perspective%2520on%2520solubility%2520and%2520hydrophobicity%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2010%26volume%3D15%26issue%3D15%25E2%2580%259316%26spage%3D648%26epage%3D655%26doi%3D10.1016%2Fj.drudis.2010.05.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Stouch, T. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kenyon, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X.-Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doweyko, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.</span><span> </span><span class="NLM_article-title"><i>In silico</i> ADME/TOX: why models fail</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">83</span><span class="NLM_x">–</span> <span class="NLM_lpage">92</span><span class="refDoi"> DOI: 10.1023/A:1025358319677</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.1023%2FA%3A1025358319677" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2003&pages=83-92&issue=2&author=T.+R.+Stouchauthor=J.+R.+Kenyonauthor=S.+R.+Johnsonauthor=X.-Q.+Chenauthor=A.+Doweykoauthor=Y.+Li&title=In+silico+ADME%2FTOX%3A+why+models+fail&doi=10.1023%2FA%3A1025358319677"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1023%2FA%3A1025358319677&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1025358319677%26sid%3Dliteratum%253Aachs%26aulast%3DStouch%26aufirst%3DT.%2BR.%26aulast%3DKenyon%26aufirst%3DJ.%2BR.%26aulast%3DJohnson%26aufirst%3DS.%2BR.%26aulast%3DChen%26aufirst%3DX.-Q.%26aulast%3DDoweyko%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DY.%26atitle%3DIn%2520silico%2520ADME%252FTOX%253A%2520why%2520models%2520fail%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2003%26volume%3D17%26issue%3D2%26spage%3D83%26epage%3D92%26doi%3D10.1023%2FA%3A1025358319677" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Larregieu, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benet, L. Z.</span><span> </span><span class="NLM_article-title">Drug Discovery and regulatory considerations for improving <i>in silico</i> and <i>in vitro</i> predictions that use Caco-2 as a surrogate for human intestinal permeability measurements</span> <span class="citation_source-journal">AAPS J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">483</span><span class="NLM_x">–</span> <span class="NLM_lpage">497</span><span class="refDoi"> DOI: 10.1208/s12248-013-9456-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.1208%2Fs12248-013-9456-8" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2013&pages=483-497&issue=2&author=C.+A.+Larregieuauthor=L.+Z.+Benet&title=Drug+Discovery+and+regulatory+considerations+for+improving+in+silico+and+in+vitro+predictions+that+use+Caco-2+as+a+surrogate+for+human+intestinal+permeability+measurements&doi=10.1208%2Fs12248-013-9456-8"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1208%2Fs12248-013-9456-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1208%252Fs12248-013-9456-8%26sid%3Dliteratum%253Aachs%26aulast%3DLarregieu%26aufirst%3DC.%2BA.%26aulast%3DBenet%26aufirst%3DL.%2BZ.%26atitle%3DDrug%2520Discovery%2520and%2520regulatory%2520considerations%2520for%2520improving%2520in%2520silico%2520and%2520in%2520vitro%2520predictions%2520that%2520use%2520Caco-2%2520as%2520a%2520surrogate%2520for%2520human%2520intestinal%2520permeability%2520measurements%26jtitle%3DAAPS%2520J.%26date%3D2013%26volume%3D15%26issue%3D2%26spage%3D483%26epage%3D497%26doi%3D10.1208%2Fs12248-013-9456-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Golbraikh, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tropsha, A.</span><span> </span><span class="NLM_article-title">Beware of q2!</span> <span class="citation_source-journal">J. Mol. Graphics Modell.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x"> (</span><span class="NLM_issue">4</span><span class="NLM_x">) </span> <span class="NLM_fpage">269</span><span class="NLM_x">–</span> <span class="NLM_lpage">276</span><span class="refDoi"> DOI: 10.1016/S1093-3263(01)00123-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.1016%2FS1093-3263%2801%2900123-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=11858635" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BD3MXpt1art7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2002&pages=269-276&issue=4&author=A.+Golbraikhauthor=A.+Tropsha&title=Beware+of+q2%21&doi=10.1016%2FS1093-3263%2801%2900123-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Beware of q2!</span></div><div class="casAuthors">Golbraikh, Alexander; Tropsha, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics & Modelling</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">269-276</span>CODEN:
                <span class="NLM_cas:coden">JMGMFI</span>;
        ISSN:<span class="NLM_cas:issn">1093-3263</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Inc.</span>)
        </div><div class="casAbstract">Validation is a crucial aspect of any quant. structure-activity relation (QSAR) modeling.  This paper examines one of the most popular validation criteria, leave-one-out cross-validated R2 (LOO q2).  Often, a high value of this statistical characteristic (q2>0.5) is considered as a proof of the high predictive ability of the model.  In this paper, we show that this assumption is generally incorrect.  In the case of 3D QSAR, the lack of the correlation between the high LOO q2 and the high predictive ability of a QSAR model has been established earlier [Pharm. Acta Helv. 70 (1995) 149; J. Chemomet. 10 (1996) 95; J. Med. Chem. 41 (1998) 2553].  In this paper, we use two-dimensional (2D) mol. descriptors and k nearest neighbors (kNN) QSAR method for the anal. of several datasets.  No correlation between the values of q2 for the training set and predictive ability for the test set was found for any of the datasets.  Thus, the high value of LOO q2 appears to be the necessary but not the sufficient condition for the model to have a high predictive power.  We argue that this is the general property of QSAR models developed using LOO cross-validation.  We emphasize that the external validation is the only way to establish a reliable QSAR model.  We formulate a set of criteria for evaluation of predictive ability of QSAR models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHZLnUmOwha7Vg90H21EOLACvtfcHk0lhhSvwG42bKZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXpt1art7o%253D&md5=59d2b3412e43241fee262a8cb61cf963</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2FS1093-3263%2801%2900123-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1093-3263%252801%252900123-1%26sid%3Dliteratum%253Aachs%26aulast%3DGolbraikh%26aufirst%3DA.%26aulast%3DTropsha%26aufirst%3DA.%26atitle%3DBeware%2520of%2520q2%2521%26jtitle%3DJ.%2520Mol.%2520Graphics%2520Modell.%26date%3D2002%26volume%3D20%26issue%3D4%26spage%3D269%26epage%3D276%26doi%3D10.1016%2FS1093-3263%2801%2900123-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Novellino, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fattorusso, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greco, G.</span><span> </span><span class="NLM_article-title">Use of comparative molecular field analysis and cluster analysis in series design</span> <span class="citation_source-journal">Pharm. Acta Helv.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">70</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">149</span><span class="NLM_x">–</span> <span class="NLM_lpage">154</span><span class="refDoi"> DOI: 10.1016/0031-6865(95)00014-Z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.1016%2F0031-6865%2895%2900014-Z" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=1995&pages=149-154&issue=2&author=E.+Novellinoauthor=C.+Fattorussoauthor=G.+Greco&title=Use+of+comparative+molecular+field+analysis+and+cluster+analysis+in+series+design&doi=10.1016%2F0031-6865%2895%2900014-Z"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2F0031-6865%2895%2900014-Z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0031-6865%252895%252900014-Z%26sid%3Dliteratum%253Aachs%26aulast%3DNovellino%26aufirst%3DE.%26aulast%3DFattorusso%26aufirst%3DC.%26aulast%3DGreco%26aufirst%3DG.%26atitle%3DUse%2520of%2520comparative%2520molecular%2520field%2520analysis%2520and%2520cluster%2520analysis%2520in%2520series%2520design%26jtitle%3DPharm.%2520Acta%2520Helv.%26date%3D1995%26volume%3D70%26issue%3D2%26spage%3D149%26epage%3D154%26doi%3D10.1016%2F0031-6865%2895%2900014-Z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Alexander, D. L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tropsha, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winkler, D. A.</span><span> </span><span class="NLM_article-title">Beware of R2: simple, unambiguous assessment of the prediction accuracy of QSAR and QSPR models</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x"> (</span><span class="NLM_issue">7</span><span class="NLM_x">) </span> <span class="NLM_fpage">1316</span><span class="NLM_x">–</span> <span class="NLM_lpage">1322</span><span class="refDoi"> DOI: 10.1021/acs.jcim.5b00206</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.5b00206" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVCmtL%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=1316-1322&issue=7&author=D.+L.+J.+Alexanderauthor=A.+Tropshaauthor=D.+A.+Winkler&title=Beware+of+R2%3A+simple%2C+unambiguous+assessment+of+the+prediction+accuracy+of+QSAR+and+QSPR+models&doi=10.1021%2Facs.jcim.5b00206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Beware of R2: Simple, Unambiguous Assessment of the Prediction Accuracy of QSAR and QSPR Models</span></div><div class="casAuthors">Alexander, D. L. J.; Tropsha, A.; Winkler, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1316-1322</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The statistical metrics used to characterize the external predictivity of a model, i.e., how well it predicts the properties of an independent test set, have proliferated over the past decade.  This paper clarifies some apparent confusion over the use of the coeff. of detn., R2, as a measure of model fit and predictive power in QSAR and QSPR modeling.  R2 (or R2) has been used in various contexts in the literature in conjunction with training and test data for both ordinary linear regression and regression through the origin as well as with linear and nonlinear regression models.  The authors analyze the widely adopted model fit criteria suggested by Golbraikh and Tropsha in a strict statistical manner.  Shortcomings in these criteria are identified, and a clearer and simpler alternative method to characterize model predictivity is provided.  The intent is not to repeat the well-documented arguments for model validation using test data but rather to guide the application of R2 as a model fit statistic.  Examples are used to illustrate both correct and incorrect uses of R2.  Reporting the root-mean-square error or equiv. measures of dispersion, which are typically of more practical importance than R2, is also encouraged, and important challenges in addressing the needs of different categories of users such as computational chemists, exptl. scientists, and regulatory decision support specialists are outlined.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGca7UxDqeXLVg90H21EOLACvtfcHk0liyCf8St-8sug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVCmtL%252FO&md5=0022bbacc7686da8cbf509b7898470f9</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.5b00206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.5b00206%26sid%3Dliteratum%253Aachs%26aulast%3DAlexander%26aufirst%3DD.%2BL.%2BJ.%26aulast%3DTropsha%26aufirst%3DA.%26aulast%3DWinkler%26aufirst%3DD.%2BA.%26atitle%3DBeware%2520of%2520R2%253A%2520simple%252C%2520unambiguous%2520assessment%2520of%2520the%2520prediction%2520accuracy%2520of%2520QSAR%2520and%2520QSPR%2520models%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26volume%3D55%26issue%3D7%26spage%3D1316%26epage%3D1322%26doi%3D10.1021%2Facs.jcim.5b00206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Sheridan, R. P.</span><span> </span><span class="NLM_article-title">Time-Split Cross-Validation as a method for estimating the goodness of prospective prediction</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x"> (</span><span class="NLM_issue">4</span><span class="NLM_x">) </span> <span class="NLM_fpage">783</span><span class="NLM_x">–</span> <span class="NLM_lpage">790</span><span class="refDoi"> DOI: 10.1021/ci400084k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci400084k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BC3sXksVyjuro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2013&pages=783-790&issue=4&author=R.+P.+Sheridan&title=Time-Split+Cross-Validation+as+a+method+for+estimating+the+goodness+of+prospective+prediction&doi=10.1021%2Fci400084k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Time-Split Cross-Validation as a Method for Estimating the Goodness of Prospective Prediction.</span></div><div class="casAuthors">Sheridan, Robert P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">783-790</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Cross-validation is a common method to validate a QSAR model.  In cross-validation, some compds. are held out as a test set, while the remaining compds. form a training set.  A model is built from the training set, and the test set compds. are predicted on that model.  The agreement of the predicted and obsd. activity values of the test set (measured by, say, R2) is an est. of the self-consistency of the model and is sometimes taken as an indication of the predictivity of the model.  This est. of predictivity can be optimistic or pessimistic compared to true prospective prediction, depending how compds. in the test set are selected.  Here, we show that time-split selection gives an R2 that is more like that of true prospective prediction than the R2 from random selection (too optimistic) or from our analog of leave-class-out selection (too pessimistic).  Time-split selection should be used in addn. to random selection as a std. for cross-validation in QSAR model building.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoc8SkoRQBbrLVg90H21EOLACvtfcHk0liyCf8St-8sug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXksVyjuro%253D&md5=638ca6d5ce1b5fecf4928c3db959d344</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fci400084k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci400084k%26sid%3Dliteratum%253Aachs%26aulast%3DSheridan%26aufirst%3DR.%2BP.%26atitle%3DTime-Split%2520Cross-Validation%2520as%2520a%2520method%2520for%2520estimating%2520the%2520goodness%2520of%2520prospective%2520prediction%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2013%26volume%3D53%26issue%3D4%26spage%3D783%26epage%3D790%26doi%3D10.1021%2Fci400084k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Chen, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sheridan, R. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hornak, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Voigt, J. H.</span><span> </span><span class="NLM_article-title">Comparison of random forest and Pipeline Pilot Naıvë Bayes in prospective QSAR predictions</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x"> (</span><span class="NLM_issue">3</span><span class="NLM_x">) </span> <span class="NLM_fpage">792</span><span class="NLM_x">–</span> <span class="NLM_lpage">803</span><span class="refDoi"> DOI: 10.1021/ci200615h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci200615h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BC38XislajtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=792-803&issue=3&author=B.+Chenauthor=R.+P.+Sheridanauthor=V.+Hornakauthor=J.+H.+Voigt&title=Comparison+of+random+forest+and+Pipeline+Pilot+Na%C4%B1v%C3%AB+Bayes+in+prospective+QSAR+predictions&doi=10.1021%2Fci200615h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of Random Forest and Pipeline Pilot Naive Bayes in Prospective QSAR Predictions</span></div><div class="casAuthors">Chen, Bin; Sheridan, Robert P.; Hornak, Viktor; Voigt, Johannes H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">792-803</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Random forest is currently considered one of the best QSAR methods available in terms of accuracy of prediction.  However, it is computationally intensive.  Naive Bayes is a simple, robust classification method.  The Laplacian-modified Naive Bayes implementation is the preferred QSAR method in the widely used com. chemoinformatics platform Pipeline Pilot.  We made a comparison of the ability of Pipeline Pilot Naive Bayes (PLPNB) and random forest to make accurate predictions on 18 large, diverse inhouse QSAR data sets.  These include on-target and ADME-related activities.  These data sets were set up as classification problems with either binary or multicategory activities.  We used a time-split method of dividing training and test sets, as we feel this is a realistic way of simulating prospective prediction.  PLPNB is computationally efficient.  However, random forest predictions are at least as good and in many cases significantly better than those of PLPNB on our data sets.  PLPNB performs better with ECFP4 and ECFP6 descriptors, which are native to Pipeline Pilot, and more poorly with other descriptors we tried.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopM-mDzNRvcLVg90H21EOLACvtfcHk0liyCf8St-8sug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XislajtbY%253D&md5=63488794c7eebebed1f34c2e6989e104</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fci200615h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci200615h%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DB.%26aulast%3DSheridan%26aufirst%3DR.%2BP.%26aulast%3DHornak%26aufirst%3DV.%26aulast%3DVoigt%26aufirst%3DJ.%2BH.%26atitle%3DComparison%2520of%2520random%2520forest%2520and%2520Pipeline%2520Pilot%2520Na%25C4%25B1v%25C3%25AB%2520Bayes%2520in%2520prospective%2520QSAR%2520predictions%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26issue%3D3%26spage%3D792%26epage%3D803%26doi%3D10.1021%2Fci200615h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Berellini, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Springer, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waters, N. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lombardo, F.</span><span> </span><span class="NLM_article-title"><i>In silico</i> Prediction of volume of distribution in human using linear and nonlinear models on a 669 compound data set</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x"> (</span><span class="NLM_issue">14</span><span class="NLM_x">) </span> <span class="NLM_fpage">4488</span><span class="NLM_x">–</span> <span class="NLM_lpage">4495</span><span class="refDoi"> DOI: 10.1021/jm9004658</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm9004658" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=4488-4495&issue=14&author=G.+Berelliniauthor=C.+Springerauthor=N.+J.+Watersauthor=F.+Lombardo&title=In+silico+Prediction+of+volume+of+distribution+in+human+using+linear+and+nonlinear+models+on+a+669+compound+data+set&doi=10.1021%2Fjm9004658"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fjm9004658&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm9004658%26sid%3Dliteratum%253Aachs%26aulast%3DBerellini%26aufirst%3DG.%26aulast%3DSpringer%26aufirst%3DC.%26aulast%3DWaters%26aufirst%3DN.%2BJ.%26aulast%3DLombardo%26aufirst%3DF.%26atitle%3DIn%2520silico%2520Prediction%2520of%2520volume%2520of%2520distribution%2520in%2520human%2520using%2520linear%2520and%2520nonlinear%2520models%2520on%2520a%2520669%2520compound%2520data%2520set%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26issue%3D14%26spage%3D4488%26epage%3D4495%26doi%3D10.1021%2Fjm9004658" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Lombardo, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Obach, R. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varma, M. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stringer, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berellini, B.</span><span> </span><span class="NLM_article-title">Clearance mechanism assignment and total clearance prediction in human based upon <i>in silico</i> models</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x"> (</span><span class="NLM_issue">10</span><span class="NLM_x">) </span> <span class="NLM_fpage">4397</span><span class="NLM_x">–</span> <span class="NLM_lpage">4405</span><span class="refDoi"> DOI: 10.1021/jm500436v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm500436v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=4397-4405&issue=10&author=F.+Lombardoauthor=R.+S.+Obachauthor=M.+V.+Varmaauthor=R.+Stringerauthor=B.+Berellini&title=Clearance+mechanism+assignment+and+total+clearance+prediction+in+human+based+upon+in+silico+models&doi=10.1021%2Fjm500436v"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fjm500436v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm500436v%26sid%3Dliteratum%253Aachs%26aulast%3DLombardo%26aufirst%3DF.%26aulast%3DObach%26aufirst%3DR.%2BS.%26aulast%3DVarma%26aufirst%3DM.%2BV.%26aulast%3DStringer%26aufirst%3DR.%26aulast%3DBerellini%26aufirst%3DB.%26atitle%3DClearance%2520mechanism%2520assignment%2520and%2520total%2520clearance%2520prediction%2520in%2520human%2520based%2520upon%2520in%2520silico%2520models%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26issue%3D10%26spage%3D4397%26epage%3D4405%26doi%3D10.1021%2Fjm500436v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Martin, T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harten, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muratov, E. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Golbraikh, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tropsha, A.</span><span> </span><span class="NLM_article-title">Does rational selection of training and test sets improve the outcome of QSAR modeling</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x"> (</span><span class="NLM_issue">10</span><span class="NLM_x">) </span> <span class="NLM_fpage">2570</span><span class="NLM_x">–</span> <span class="NLM_lpage">2578</span><span class="refDoi"> DOI: 10.1021/ci300338w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci300338w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVCgtrjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=2570-2578&issue=10&author=T.+M.+Martinauthor=P.+Hartenauthor=D.+M.+Youngauthor=E.+N.+Muratovauthor=A.+Golbraikhauthor=H.+Zhuauthor=A.+Tropsha&title=Does+rational+selection+of+training+and+test+sets+improve+the+outcome+of+QSAR+modeling&doi=10.1021%2Fci300338w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Does Rational Selection of Training and Test Sets Improve the Outcome of QSAR Modeling?</span></div><div class="casAuthors">Martin, Todd M.; Harten, Paul; Young, Douglas M.; Muratov, Eugene N.; Golbraikh, Alexander; Zhu, Hao; Tropsha, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2570-2578</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Prior to using a quant. structure activity relationship (QSAR) model for external predictions, its predictive power should be established and validated.  In the absence of a true external data set, the best way to validate the predictive ability of a model is to perform its statistical external validation.  In statistical external validation, the overall data set is divided into training and test sets.  Commonly, this splitting is performed using random division.  Rational splitting methods can divide data sets into training and test sets in an intelligent fashion.  The purpose of this study was to det. whether rational division methods lead to more predictive models compared to random division.  A special data splitting procedure was used to facilitate the comparison between random and rational division methods.  For each toxicity end point, the overall data set was divided into a modeling set (80% of the overall set) and an external evaluation set (20% of the overall set) using random division.  The modeling set was then subdivided into a training set (80% of the modeling set) and a test set (20% of the modeling set) using rational division methods and by using random division.  The Kennard-Stone, minimal test set dissimilarity, and sphere exclusion algorithms were used as the rational division methods.  The hierarchical clustering, random forest, and k-nearest neighbor (kNN) methods were used to develop QSAR models based on the training sets.  For kNN QSAR, multiple training and test sets were generated, and multiple QSAR models were built.  The results of this study indicate that models based on rational division methods generate better statistical results for the test sets than models based on random division, but the predictive power of both types of models are comparable.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8M643bY3vHbVg90H21EOLACvtfcHk0lgQlIjNrmW7sw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVCgtrjL&md5=d1afc25036cbaf9bc43e830c57df7f99</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fci300338w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci300338w%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DT.%2BM.%26aulast%3DHarten%26aufirst%3DP.%26aulast%3DYoung%26aufirst%3DD.%2BM.%26aulast%3DMuratov%26aufirst%3DE.%2BN.%26aulast%3DGolbraikh%26aufirst%3DA.%26aulast%3DZhu%26aufirst%3DH.%26aulast%3DTropsha%26aufirst%3DA.%26atitle%3DDoes%2520rational%2520selection%2520of%2520training%2520and%2520test%2520sets%2520improve%2520the%2520outcome%2520of%2520QSAR%2520modeling%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26issue%3D10%26spage%3D2570%26epage%3D2578%26doi%3D10.1021%2Fci300338w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Lombardo, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Obach, R. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shalaeva, M. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, F.</span><span> </span><span class="NLM_article-title">Prediction of human volume of distribution values for neutral and basic drugs. 2. Extended data seta and leave-class-out statistics</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x"> (</span><span class="NLM_issue">5</span><span class="NLM_x">) </span> <span class="NLM_fpage">1242</span><span class="NLM_x">–</span> <span class="NLM_lpage">1250</span><span class="refDoi"> DOI: 10.1021/jm030408h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm030408h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1242-1250&issue=5&author=F.+Lombardoauthor=R.+S.+Obachauthor=M.+Y.+Shalaevaauthor=F.+Gao&title=Prediction+of+human+volume+of+distribution+values+for+neutral+and+basic+drugs.+2.+Extended+data+seta+and+leave-class-out+statistics&doi=10.1021%2Fjm030408h"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fjm030408h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030408h%26sid%3Dliteratum%253Aachs%26aulast%3DLombardo%26aufirst%3DF.%26aulast%3DObach%26aufirst%3DR.%2BS.%26aulast%3DShalaeva%26aufirst%3DM.%2BY.%26aulast%3DGao%26aufirst%3DF.%26atitle%3DPrediction%2520of%2520human%2520volume%2520of%2520distribution%2520values%2520for%2520neutral%2520and%2520basic%2520drugs.%25202.%2520Extended%2520data%2520seta%2520and%2520leave-class-out%2520statistics%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26issue%3D5%26spage%3D1242%26epage%3D1250%26doi%3D10.1021%2Fjm030408h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">Clark, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lawless, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fraczkiewicz, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waldman, M.</span><span> </span><span class="NLM_article-title">Using bet binomials to estimate classification uncertainty for ensemble models</span> <span class="citation_source-journal">J. Cheminf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">34</span><span class="refDoi"> DOI: 10.1186/1758-2946-6-34</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.1186%2F1758-2946-6-34" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVCisbbK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&pages=34&author=R.+D.+Clarkauthor=W.+Liangauthor=A.+C.+Leeauthor=M.+S.+Lawlessauthor=R.+Fraczkiewiczauthor=M.+Waldman&title=Using+bet+binomials+to+estimate+classification+uncertainty+for+ensemble+models&doi=10.1186%2F1758-2946-6-34"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Using beta binomials to estimate classification uncertainty for ensemble models</span></div><div class="casAuthors">Clark, Robert D.; Liang, Wenkel; Lee, Adam C.; Lawless, Michael S.; Fraczkiewicz, Robert; Waldman, Marvin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cheminformatics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">34/1-34/19</span>CODEN:
                <span class="NLM_cas:coden">JCOHB3</span>;
        ISSN:<span class="NLM_cas:issn">1758-2946</span>.
    
            (<span class="NLM_cas:orgname">Chemistry Central Ltd.</span>)
        </div><div class="casAbstract">Background: Quant. structure-activity (QSAR) models have enormous potential for reducing drug discovery and development costs as well as the need for animal testing.  Great strides have been made in estg. their overall reliability, but to fully realize that potential, researchers and regulators need to know how confident they can be in individual predictions.  Results: Submodels in an ensemble model which have been trained on different subsets of a shared training pool represent multiple samples of the model space, and the degree of agreement among them contains information on the reliability of ensemble predictions.  For artificial neural network ensembles (ANNEs) using two different methods for detg. ensemble classification - one using vote tallies and the other averaging individual network outputs - we have found that the distribution of predictions across pos. vote tallies can be reasonably well-modeled as a beta binomial distribution, as can the distribution of errors.  Together, these two distributions can be used to est. the probability that a given predictive classification will be in error.  Large data sets comprised of logP, Ames mutagenicity, and CYP2D6 inhibition data are used to illustrate and validate the method.  The distributions of predictions and errors for the training pool accurately predicted the distribution of predictions and errors for large external validation sets, even when the no. of pos. and neg. examples in the training pool were not balanced.  Moreover, the likelihood of a given compd. being prospectively misclassified as a function of the degree of consensus between networks in the ensemble could in most cases be estd. accurately from the fitted beta binomial distributions for the training pool.  Conclusions: Confidence in an individual predictive classification by an ensemble model can be accurately assessed by examg. the distributions of predictions and errors as a function of the degree of agreement among the constituent submodels.  Further, ensemble uncertainty estn. can often be improved by adjusting the voting or classification threshold based on the parameters of the error distribution.  Finally, the profiles for models whose predictive uncertainty ests. are not reliable provide clues to that effect without the need for comparison to an external test set.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrrouzoA8q6LbVg90H21EOLACvtfcHk0lgQlIjNrmW7sw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVCisbbK&md5=3037c0a4c637b3c7e7e7322b5c98a308</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1186%2F1758-2946-6-34&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1758-2946-6-34%26sid%3Dliteratum%253Aachs%26aulast%3DClark%26aufirst%3DR.%2BD.%26aulast%3DLiang%26aufirst%3DW.%26aulast%3DLee%26aufirst%3DA.%2BC.%26aulast%3DLawless%26aufirst%3DM.%2BS.%26aulast%3DFraczkiewicz%26aufirst%3DR.%26aulast%3DWaldman%26aufirst%3DM.%26atitle%3DUsing%2520bet%2520binomials%2520to%2520estimate%2520classification%2520uncertainty%2520for%2520ensemble%2520models%26jtitle%3DJ.%2520Cheminf.%26date%3D2014%26volume%3D6%26spage%3D34%26doi%3D10.1186%2F1758-2946-6-34" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Roy, K.; Kar, S.; Das, R. N.</span> <span class="citation_source-book">A Primer on QSAR/QSPR Modeling</span>; <span class="NLM_publisher-name">Springer International Publishing</span>: <span class="NLM_publisher-loc">Zurich</span>,<span class="NLM_x"> </span><span class="NLM_year">2015</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.1007%2F978-3-319-17281-1" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=K.+Roy&author=S.+Kar&author=R.+N.+Das&title=A+Primer+on+QSAR%2FQSPR+Modeling"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1007%2F978-3-319-17281-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-3-319-17281-1%26sid%3Dliteratum%253Aachs%26aulast%3DRoy%26aufirst%3DK.%26btitle%3DA%2520Primer%2520on%2520QSAR%252FQSPR%2520Modeling%26pub%3DSpringer%2520International%2520Publishing%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Norinder, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carlsson, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boyer, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eklund, M.</span><span> </span><span class="NLM_article-title">Introducing conformal prediction in predictive modeling. A transparent and flexible alternative to applicability domain determination</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x"> (</span><span class="NLM_issue">6</span><span class="NLM_x">) </span> <span class="NLM_fpage">1596</span><span class="NLM_x">–</span> <span class="NLM_lpage">1603</span><span class="refDoi"> DOI: 10.1021/ci5001168</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci5001168" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BC2cXntlGitrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=1596-1603&issue=6&author=U.+Norinderauthor=L.+Carlssonauthor=S.+Boyerauthor=M.+Eklund&title=Introducing+conformal+prediction+in+predictive+modeling.+A+transparent+and+flexible+alternative+to+applicability+domain+determination&doi=10.1021%2Fci5001168"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Introducing Conformal Prediction in Predictive Modeling. A Transparent and Flexible Alternative to Applicability Domain Determination</span></div><div class="casAuthors">Norinder, Ulf; Carlsson, Lars; Boyer, Scott; Eklund, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1596-1603</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Conformal prediction is introduced as an alternative approach to domain applicability estn.  The advantages of using conformal prediction are as follows: First, the approach is based on a consistent and well-defined math. framework.  Second, the understanding of the confidence level concept in conformal predictions is straightforward, e.g. a confidence level of 0.8 means that the conformal predictor will commit, at most, 20% errors (i.e., true values outside the assigned prediction range).  Third, the confidence level can be varied depending on the situation where the model is to be applied and the consequences of such changes are readily understandable, i.e. prediction ranges are increased or decreased, and the changes can immediately be inspected.  The authors demonstrate the usefulness of conformal prediction by applying it to 10 publicly available data sets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaF64xrBFUp7Vg90H21EOLACvtfcHk0lj0QXap2_oQAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXntlGitrk%253D&md5=1a4adce403989470336e5ca8a23c8738</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fci5001168&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci5001168%26sid%3Dliteratum%253Aachs%26aulast%3DNorinder%26aufirst%3DU.%26aulast%3DCarlsson%26aufirst%3DL.%26aulast%3DBoyer%26aufirst%3DS.%26aulast%3DEklund%26aufirst%3DM.%26atitle%3DIntroducing%2520conformal%2520prediction%2520in%2520predictive%2520modeling.%2520A%2520transparent%2520and%2520flexible%2520alternative%2520to%2520applicability%2520domain%2520determination%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26issue%3D6%26spage%3D1596%26epage%3D1603%26doi%3D10.1021%2Fci5001168" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">Sheridan, R. P.</span><span> </span><span class="NLM_article-title">The relative importance of domain applicability metrics for estimating prediction errors in QSAR varies with training set diversity</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x"> (</span><span class="NLM_issue">6</span><span class="NLM_x">) </span> <span class="NLM_fpage">1098</span><span class="NLM_x">–</span> <span class="NLM_lpage">1107</span><span class="refDoi"> DOI: 10.1021/acs.jcim.5b00110</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.5b00110" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BC2MXoslKrs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=1098-1107&issue=6&author=R.+P.+Sheridan&title=The+relative+importance+of+domain+applicability+metrics+for+estimating+prediction+errors+in+QSAR+varies+with+training+set+diversity&doi=10.1021%2Facs.jcim.5b00110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">The Relative Importance of Domain Applicability Metrics for Estimating Prediction Errors in QSAR Varies with Training Set Diversity</span></div><div class="casAuthors">Sheridan, Robert P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1098-1107</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In QSAR, a statistical model is generated from a training set of mols. (represented by chem. descriptors) and their biol. activities (an "activity model").  The aim of the field of domain applicability (DA) is to est. the uncertainty of prediction of a specific mol. on a specific activity model.  A no. of DA metrics have been proposed in the literature for this purpose.  A quant. model of the prediction uncertainty (an "error model") can be built using one or more of these metrics.  A previous publication from our lab. (Sheridan, R.P.J. Chem. Inf. Model. 2013, 53, 2837-2850) suggested that QSAR methods such as random forest could be used to build error models by fitting unsigned prediction errors against DA metrics.  The QSAR paradigm contains two useful techniques: descriptor importance can det. which DA metrics are most useful, and cross-validation can be used to tell which subset of DA metrics is sufficient to est. the unsigned errors.  Previously we studied 10 large, diverse data sets and seven DA metrics.  For those data sets for which it is possible to build a significant error model from those seven metrics, only two metrics were sufficient to account for almost all of the information in the error model.  These were TREE_SD (the variation of prediction among random forest trees) and PREDICTED (the predicted activity itself).  In this paper we show that when data sets are less diverse, as for example in QSAR models of mols. in a single chem. series, these two DA metrics become less important in explaining prediction error, and the DA metric SIMILARITYNEAREST1 (the similarity of the mol. being predicted to the closest training set compd.) becomes more important.  Our recommendation is that when the mean pairwise similarity (measured with the Carhart AP descriptor and the Dice similarity index) within a QSAR training set is less than 0.5, one can use only TREE_SD, PREDICTED to form the error model, but otherwise one should use TREE_SD, PREDICTED, SIMILARITYNEAREST1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoimBuihjUGHLVg90H21EOLACvtfcHk0lj0QXap2_oQAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXoslKrs7g%253D&md5=95b7d9f794f575eebeeb1a8c92b2654c</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.5b00110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.5b00110%26sid%3Dliteratum%253Aachs%26aulast%3DSheridan%26aufirst%3DR.%2BP.%26atitle%3DThe%2520relative%2520importance%2520of%2520domain%2520applicability%2520metrics%2520for%2520estimating%2520prediction%2520errors%2520in%2520QSAR%2520varies%2520with%2520training%2520set%2520diversity%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26volume%3D55%26issue%3D6%26spage%3D1098%26epage%3D1107%26doi%3D10.1021%2Facs.jcim.5b00110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group">Keefer, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kauffman, G. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gupta, R. R.</span><span> </span><span class="NLM_article-title">Interpretable, probability-based confidence metric for continuous quantitative structure-activity relationship models</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">368</span><span class="NLM_x">–</span> <span class="NLM_lpage">383</span><span class="refDoi"> DOI: 10.1021/ci300554t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci300554t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1Cit74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2013&pages=368-383&issue=2&author=C.+E.+Keeferauthor=G.+W.+Kauffmanauthor=R.+R.+Gupta&title=Interpretable%2C+probability-based+confidence+metric+for+continuous+quantitative+structure-activity+relationship+models&doi=10.1021%2Fci300554t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Interpretable, Probability-Based Confidence Metric for Continuous Quantitative Structure-Activity Relationship Models</span></div><div class="casAuthors">Keefer, Christopher E.; Kauffman, Gregory W.; Gupta, Rishi Raj</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">368-383</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A great deal of research has gone into the development of robust confidence in prediction and applicability domain (AD) measures for quant. structure-activity relationship (QSAR) models in recent years.  Much of the attention has historically focused on structural similarity, which can be defined in many forms and flavors.  A concept that is frequently overlooked in the realm of the QSAR applicability domain is how the local activity landscape plays a role in how accurate a prediction is or is not.  In this work, we describe an approach that pairs information about both the chem. similarity and activity landscape of a test compd.'s neighborhood into a single calcd. confidence value.  We also present an approach for converting this value into an interpretable confidence metric that has a simple and informative meaning across data sets.  The approach will be introduced to the reader in the context of models built upon four diverse literature data sets.  The steps we will outline include the definition of similarity used to det. nearest neighbors (NN), how we incorporate the NN activity landscape with a similarity-weighted root-mean-square distance (wRMSD) value, and how that value is then calibrated to generate an intuitive confidence metric for prospective application.  Finally, we will illustrate the prospective performance of the approach on five proprietary models whose predictions and confidence metrics have been tracked for more than a year.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqvcorWVQ5O7Vg90H21EOLACvtfcHk0lj0QXap2_oQAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1Cit74%253D&md5=14d2b7d42f91b5e4e90fa53b6266a178</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Fci300554t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci300554t%26sid%3Dliteratum%253Aachs%26aulast%3DKeefer%26aufirst%3DC.%2BE.%26aulast%3DKauffman%26aufirst%3DG.%2BW.%26aulast%3DGupta%26aufirst%3DR.%2BR.%26atitle%3DInterpretable%252C%2520probability-based%2520confidence%2520metric%2520for%2520continuous%2520quantitative%2520structure-activity%2520relationship%2520models%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2013%26volume%3D53%26issue%3D2%26spage%3D368%26epage%3D383%26doi%3D10.1021%2Fci300554t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group">Gombar, V. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hall, S. D.</span><span> </span><span class="NLM_article-title">Quantitative structure–activity relationship models of clinical pharmacokinetics: clearance and volume of distribution</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x"> (</span><span class="NLM_issue">4</span><span class="NLM_x">) </span> <span class="NLM_fpage">948</span><span class="NLM_x">–</span> <span class="NLM_lpage">957</span><span class="refDoi"> DOI: 10.1021/ci400001u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci400001u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjt1Wltb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2013&pages=948-957&issue=4&author=V.+K.+Gombarauthor=S.+D.+Hall&title=Quantitative+structure%E2%80%93activity+relationship+models+of+clinical+pharmacokinetics%3A+clearance+and+volume+of+distribution&doi=10.1021%2Fci400001u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Quantitative Structure-Activity Relationship Models of Clinical Pharmacokinetics: Clearance and Volume of Distribution</span></div><div class="casAuthors">Gombar, Vijay K.; Hall, Stephen D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">948-957</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Reliable prediction of two fundamental human pharmacokinetic (PK) parameters, systemic clearance (CL) and apparent vol. of distribution (Vd), det. the size and frequency of drug dosing and are at the heart of drug discovery and development.  Traditionally, estd. CL and Vd are derived from preclin. in vitro and in vivo absorption, distribution, metab., and excretion (ADME) measurements.  In this paper, we report quant. structure-activity relationship (QSAR) models for prediction of systemic CL and steady-state Vd (Vdss) from i.v. (iv) dosing in humans.  These QSAR models avoid uncertainty assocd. with preclin.-to-clin. extrapolation and require two-dimensional structure drawing as the sole input.  The clean, uniform training sets for these models were derived from the compilation published by Obach et al.  Models for CL and Vdss were developed using both a support vector regression (SVR) method and a multiple linear regression (MLR) method.  The SVR models employ a min. of 2048-bit fingerprints developed inhouse as structure quantifiers.  The MLR models, on the other hand, are based on information-rich electro-topol. states of two-atom fragments as descriptors and afford reverse QSAR (RQSAR) anal. to help model-guided, in silico modulation of structures for desired CL and Vdss.  The capability of the models to predict iv CL and Vdss with acceptable accuracy was established by randomly splitting data into training and test sets.  On av., for both CL and Vdss, 75% of test compds. were predicted within 2.5-fold of the value obsd. and 90% of test compds. were within 5.0-fold of the value obsd.  The performance of the final models developed from 525 compds. for CL and 569 compds. for Vdss was evaluated on an external set of 56 compds.  The predictions were either better or comparable to those predicted by other in silico models reported in the literature.  To demonstrate the practical application of the RQSAR approach, the structure of vildagliptin, a high-CL and a high-Vdss compd., is modified based on the at. contributions to its predicted CL and Vdss to propose compds. with lower CL and lower Vdss.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppnw75Jax-0LVg90H21EOLACvtfcHk0lhRFclQVJ5TtQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjt1Wltb0%253D&md5=f7d5b6c91e646c37b3eed26730ac087a</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fci400001u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci400001u%26sid%3Dliteratum%253Aachs%26aulast%3DGombar%26aufirst%3DV.%2BK.%26aulast%3DHall%26aufirst%3DS.%2BD.%26atitle%3DQuantitative%2520structure%25E2%2580%2593activity%2520relationship%2520models%2520of%2520clinical%2520pharmacokinetics%253A%2520clearance%2520and%2520volume%2520of%2520distribution%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2013%26volume%3D53%26issue%3D4%26spage%3D948%26epage%3D957%26doi%3D10.1021%2Fci400001u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group">Varma, M. V. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lai, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">El-Kattan, A. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Litchfield, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goosen, T. C.</span><span> </span><span class="NLM_article-title">Physicochemical property space of hepatobiliary transport and computational models for predicting rat biliary excretion</span> <span class="citation_source-journal">Drug Metab. Dispos.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x"> (</span><span class="NLM_issue">8</span><span class="NLM_x">) </span> <span class="NLM_fpage">1527</span><span class="NLM_x">–</span> <span class="NLM_lpage">1537</span><span class="refDoi"> DOI: 10.1124/dmd.112.044628</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.1124%2Fdmd.112.044628" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=1527-1537&issue=8&author=M.+V.+S.+Varmaauthor=G.+Changauthor=Y.+Laiauthor=B.+Fengauthor=A.+F.+El-Kattanauthor=J.+Litchfieldauthor=T.+C.+Goosen&title=Physicochemical+property+space+of+hepatobiliary+transport+and+computational+models+for+predicting+rat+biliary+excretion&doi=10.1124%2Fdmd.112.044628"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1124%2Fdmd.112.044628&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.112.044628%26sid%3Dliteratum%253Aachs%26aulast%3DVarma%26aufirst%3DM.%2BV.%2BS.%26aulast%3DChang%26aufirst%3DG.%26aulast%3DLai%26aufirst%3DY.%26aulast%3DFeng%26aufirst%3DB.%26aulast%3DEl-Kattan%26aufirst%3DA.%2BF.%26aulast%3DLitchfield%26aufirst%3DJ.%26aulast%3DGoosen%26aufirst%3DT.%2BC.%26atitle%3DPhysicochemical%2520property%2520space%2520of%2520hepatobiliary%2520transport%2520and%2520computational%2520models%2520for%2520predicting%2520rat%2520biliary%2520excretion%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2012%26volume%3D40%26issue%3D8%26spage%3D1527%26epage%3D1537%26doi%3D10.1124%2Fdmd.112.044628" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group">Lombardo, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waters, N. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Argikar, U. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dennehy, M. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhan, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gunduz, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harriman, S. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berellini, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liric Rajlic, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Obach, R. S.</span><span> </span><span class="NLM_article-title">Comprehensive assessment of human pharmacokinetic prediction based on <i>in vivo</i> animal pharmacokinetic data, part 1: volume of distribution at steady state</span> <span class="citation_source-journal">J. Clin. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">167</span><span class="NLM_x">–</span> <span class="NLM_lpage">177</span><span class="refDoi"> DOI: 10.1177/0091270012440281</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.1177%2F0091270012440281" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=23436262" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1yns7fL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2013&pages=167-177&issue=2&author=F.+Lombardoauthor=N.+J.+Watersauthor=U.+A.+Argikarauthor=M.+K.+Dennehyauthor=J.+Zhanauthor=M.+Gunduzauthor=S.+P.+Harrimanauthor=G.+Berelliniauthor=I.+Liric+Rajlicauthor=R.+S.+Obach&title=Comprehensive+assessment+of+human+pharmacokinetic+prediction+based+on+in+vivo+animal+pharmacokinetic+data%2C+part+1%3A+volume+of+distribution+at+steady+state&doi=10.1177%2F0091270012440281"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive assessment of human pharmacokinetic prediction based on in vivo animal pharmacokinetic data, part 1: volume of distribution at steady state</span></div><div class="casAuthors">Lombardo, Franco; Waters, Nigel J.; Argikar, Upendra A.; Dennehy, Michelle K.; Zhan, Jenny; Gunduz, Mithat; Harriman, Shawn P.; Berellini, Giuliano; Rajlic, Ivana Liric; Obach, R. Scott</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Pharmacology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">167-177</span>CODEN:
                <span class="NLM_cas:coden">JCPCBR</span>;
        ISSN:<span class="NLM_cas:issn">0091-2700</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">The authors present a comprehensive anal. on the estn. of vol. of distribution at steady state (VDss) in human based on rat, dog, and monkey data on nearly 400 compds. for which there are also assocd. human data.  This data set, to the authors' knowledge, is the largest publicly available, has been carefully compiled from literature reports, and was expanded with some inhouse detns. such as plasma protein binding data.  This work offers a good statistical basis for the evaluation of applicable prediction methods, their accuracy, and some methods-dependent diagnostic tools.  The authors also grouped the compds. according to their charge classes and show the applicability of each method considered to each class, offering further insight into the probability of a successful prediction.  Furthermore, they found that the use of fraction unbound in plasma, to obtain unbound vol. of distribution, is generally detrimental to accuracy of several methods, and they discuss possible reasons.  Overall, the approach using dog and monkey data in the Oie-Tozer equation offers the highest probability of success, with an intrinsic diagnostic tool based on aberrant values (<0 or >1) for the calcd. fraction unbound in tissue.  Alternatively, methods based on dog data (single-species scaling) and rat and dog data (Oie-Tozer equation with 2 species or multiple regression methods) may be considered reasonable approaches while not requiring data in nonhuman primates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozX4w_N8RpP7Vg90H21EOLACvtfcHk0lhRFclQVJ5TtQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1yns7fL&md5=446a4a31a0fc877b12f174cdb54c3af1</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1177%2F0091270012440281&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F0091270012440281%26sid%3Dliteratum%253Aachs%26aulast%3DLombardo%26aufirst%3DF.%26aulast%3DWaters%26aufirst%3DN.%2BJ.%26aulast%3DArgikar%26aufirst%3DU.%2BA.%26aulast%3DDennehy%26aufirst%3DM.%2BK.%26aulast%3DZhan%26aufirst%3DJ.%26aulast%3DGunduz%26aufirst%3DM.%26aulast%3DHarriman%26aufirst%3DS.%2BP.%26aulast%3DBerellini%26aufirst%3DG.%26aulast%3DLiric%2BRajlic%26aufirst%3DI.%26aulast%3DObach%26aufirst%3DR.%2BS.%26atitle%3DComprehensive%2520assessment%2520of%2520human%2520pharmacokinetic%2520prediction%2520based%2520on%2520in%2520vivo%2520animal%2520pharmacokinetic%2520data%252C%2520part%25201%253A%2520volume%2520of%2520distribution%2520at%2520steady%2520state%26jtitle%3DJ.%2520Clin.%2520Pharmacol.%26date%3D2013%26volume%3D53%26issue%3D2%26spage%3D167%26epage%3D177%26doi%3D10.1177%2F0091270012440281" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group">Lombardo, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waters, N. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Argikar, U. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dennehy, M. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhan, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gunduz, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harriman, S. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berellini, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liric Rajlic, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Obach, R. S.</span><span> </span><span class="NLM_article-title">Comprehensive assessment of human pharmacokinetic prediction based on <i>in vivo</i> animal pharmacokinetic data, part 2: clearance</span> <span class="citation_source-journal">J. Clin. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">178</span><span class="NLM_x">–</span> <span class="NLM_lpage">191</span><span class="refDoi"> DOI: 10.1177/0091270012440282</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.1177%2F0091270012440282" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=23436263" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1ynsL%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2013&pages=178-191&issue=2&author=F.+Lombardoauthor=N.+J.+Watersauthor=U.+A.+Argikarauthor=M.+K.+Dennehyauthor=J.+Zhanauthor=M.+Gunduzauthor=S.+P.+Harrimanauthor=G.+Berelliniauthor=I.+Liric+Rajlicauthor=R.+S.+Obach&title=Comprehensive+assessment+of+human+pharmacokinetic+prediction+based+on+in+vivo+animal+pharmacokinetic+data%2C+part+2%3A+clearance&doi=10.1177%2F0091270012440282"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive assessment of human pharmacokinetic prediction based on in vivo animal pharmacokinetic data, part 2: clearance</span></div><div class="casAuthors">Lombardo, Franco; Waters, Nigel J.; Argikar, Upendra A.; Dennehy, Michelle K.; Zhan, Jenny; Gunduz, Mithat; Harriman, Shawn P.; Berellini, Giuliano; Rajlic, Ivana Liric; Obach, R. Scott</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Pharmacology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">178-191</span>CODEN:
                <span class="NLM_cas:coden">JCPCBR</span>;
        ISSN:<span class="NLM_cas:issn">0091-2700</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A comprehensive anal. on the prediction of human clearance based on i.v. pharmacokinetic data from rat, dog, and monkey for approx. 400 compds. was undertaken.  This data set has been carefully compiled from literature reports and expanded with some inhouse detns. for plasma protein binding and rat clearance.  To the authors' knowledge, this is the largest publicly available data set.  The present examn. offers a comparison of 37 different methods for prediction of human clearance across compds. of diverse physicochem. properties.  Furthermore, this work demonstrates the application of each prediction method to each charge class of the compds., thus presenting an addnl. dimension to prediction of human pharmacokinetics.  In general, the observations suggest that methods employing monkey clearance values and a method incorporating differences in plasma protein binding between rat and human yield the best overall predictions as suggested by approx. 60% compds. within 2-fold geometric mean-fold error.  Other single-species scaling or proportionality methods incorporating the fraction unbound in the corresponding preclin. species for prediction of free clearance in human were generally unsuccessful.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrD0VWrLLqed7Vg90H21EOLACvtfcHk0lgM4IULgAf-sQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1ynsL%252FK&md5=a87b8d0a0025910b592a3e757fb35d54</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1177%2F0091270012440282&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F0091270012440282%26sid%3Dliteratum%253Aachs%26aulast%3DLombardo%26aufirst%3DF.%26aulast%3DWaters%26aufirst%3DN.%2BJ.%26aulast%3DArgikar%26aufirst%3DU.%2BA.%26aulast%3DDennehy%26aufirst%3DM.%2BK.%26aulast%3DZhan%26aufirst%3DJ.%26aulast%3DGunduz%26aufirst%3DM.%26aulast%3DHarriman%26aufirst%3DS.%2BP.%26aulast%3DBerellini%26aufirst%3DG.%26aulast%3DLiric%2BRajlic%26aufirst%3DI.%26aulast%3DObach%26aufirst%3DR.%2BS.%26atitle%3DComprehensive%2520assessment%2520of%2520human%2520pharmacokinetic%2520prediction%2520based%2520on%2520in%2520vivo%2520animal%2520pharmacokinetic%2520data%252C%2520part%25202%253A%2520clearance%26jtitle%3DJ.%2520Clin.%2520Pharmacol.%26date%3D2013%26volume%3D53%26issue%3D2%26spage%3D178%26epage%3D191%26doi%3D10.1177%2F0091270012440282" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group">Poulin, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Theil, F.-P.</span><span> </span><span class="NLM_article-title">Prediction of pharmacokinetics prior to <i>in vivo</i> studies. 1 Mechanism-based prediction of volume of distribution</span> <span class="citation_source-journal">J. Pharm. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">91</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">129</span><span class="NLM_x">–</span> <span class="NLM_lpage">156</span><span class="refDoi"> DOI: 10.1002/jps.10005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.1002%2Fjps.10005" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=91&publication_year=2002&pages=129-156&issue=1&author=P.+Poulinauthor=F.-P.+Theil&title=Prediction+of+pharmacokinetics+prior+to+in+vivo+studies.+1+Mechanism-based+prediction+of+volume+of+distribution&doi=10.1002%2Fjps.10005"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1002%2Fjps.10005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjps.10005%26sid%3Dliteratum%253Aachs%26aulast%3DPoulin%26aufirst%3DP.%26aulast%3DTheil%26aufirst%3DF.-P.%26atitle%3DPrediction%2520of%2520pharmacokinetics%2520prior%2520to%2520in%2520vivo%2520studies.%25201%2520Mechanism-based%2520prediction%2520of%2520volume%2520of%2520distribution%26jtitle%3DJ.%2520Pharm.%2520Sci.%26date%3D2002%26volume%3D91%26issue%3D1%26spage%3D129%26epage%3D156%26doi%3D10.1002%2Fjps.10005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group">Rodgers, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rowland, M.</span><span> </span><span class="NLM_article-title">Mechanistic approaches to volume of distribution predictions: understanding the process</span> <span class="citation_source-journal">Pharm. Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x"> (</span><span class="NLM_issue">5</span><span class="NLM_x">) </span> <span class="NLM_fpage">918</span><span class="NLM_x">–</span> <span class="NLM_lpage">933</span><span class="refDoi"> DOI: 10.1007/s11095-006-9210-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.1007%2Fs11095-006-9210-3" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2007&pages=918-933&issue=5&author=T.+Rodgersauthor=M.+Rowland&title=Mechanistic+approaches+to+volume+of+distribution+predictions%3A+understanding+the+process&doi=10.1007%2Fs11095-006-9210-3"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1007%2Fs11095-006-9210-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs11095-006-9210-3%26sid%3Dliteratum%253Aachs%26aulast%3DRodgers%26aufirst%3DT.%26aulast%3DRowland%26aufirst%3DM.%26atitle%3DMechanistic%2520approaches%2520to%2520volume%2520of%2520distribution%2520predictions%253A%2520understanding%2520the%2520process%26jtitle%3DPharm.%2520Res.%26date%3D2007%26volume%3D24%26issue%3D5%26spage%3D918%26epage%3D933%26doi%3D10.1007%2Fs11095-006-9210-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group">Ward, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, B. R.</span><span> </span><span class="NLM_article-title">A comprehensive quantitative and qualitative evaluation of extrapolation of intravenous pharmacokinetic parameters from rat, dog, and monkey to human. I. Clearance</span> <span class="citation_source-journal">Drug Metab. Dispos.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x"> (</span><span class="NLM_issue">6</span><span class="NLM_x">) </span> <span class="NLM_fpage">603</span><span class="NLM_x">–</span> <span class="NLM_lpage">611</span><span class="refDoi"> DOI: 10.1124/dmd.32.6.603</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.1124%2Fdmd.32.6.603" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2004&pages=603-611&issue=6&author=K.+W.+Wardauthor=B.+R.+Smith&title=A+comprehensive+quantitative+and+qualitative+evaluation+of+extrapolation+of+intravenous+pharmacokinetic+parameters+from+rat%2C+dog%2C+and+monkey+to+human.+I.+Clearance&doi=10.1124%2Fdmd.32.6.603"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1124%2Fdmd.32.6.603&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.32.6.603%26sid%3Dliteratum%253Aachs%26aulast%3DWard%26aufirst%3DK.%2BW.%26aulast%3DSmith%26aufirst%3DB.%2BR.%26atitle%3DA%2520comprehensive%2520quantitative%2520and%2520qualitative%2520evaluation%2520of%2520extrapolation%2520of%2520intravenous%2520pharmacokinetic%2520parameters%2520from%2520rat%252C%2520dog%252C%2520and%2520monkey%2520to%2520human.%2520I.%2520Clearance%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2004%26volume%3D32%26issue%3D6%26spage%3D603%26epage%3D611%26doi%3D10.1124%2Fdmd.32.6.603" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group">Ward, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, B. R.</span><span> </span><span class="NLM_article-title">A comprehensive quantitative and qualitative evaluation of extrapolation of intravenous pharmacokinetic parameters from rat, dog, and monkey to human. II. Volume of distribution and mean residence time</span> <span class="citation_source-journal">Drug Metab. Dispos.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x"> (</span><span class="NLM_issue">6</span><span class="NLM_x">) </span> <span class="NLM_fpage">612</span><span class="NLM_x">–</span> <span class="NLM_lpage">619</span><span class="refDoi"> DOI: 10.1124/dmd.32.6.612</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.1124%2Fdmd.32.6.612" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2004&pages=612-619&issue=6&author=K.+W.+Wardauthor=B.+R.+Smith&title=A+comprehensive+quantitative+and+qualitative+evaluation+of+extrapolation+of+intravenous+pharmacokinetic+parameters+from+rat%2C+dog%2C+and+monkey+to+human.+II.+Volume+of+distribution+and+mean+residence+time&doi=10.1124%2Fdmd.32.6.612"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1124%2Fdmd.32.6.612&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.32.6.612%26sid%3Dliteratum%253Aachs%26aulast%3DWard%26aufirst%3DK.%2BW.%26aulast%3DSmith%26aufirst%3DB.%2BR.%26atitle%3DA%2520comprehensive%2520quantitative%2520and%2520qualitative%2520evaluation%2520of%2520extrapolation%2520of%2520intravenous%2520pharmacokinetic%2520parameters%2520from%2520rat%252C%2520dog%252C%2520and%2520monkey%2520to%2520human.%2520II.%2520Volume%2520of%2520distribution%2520and%2520mean%2520residence%2520time%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2004%26volume%3D32%26issue%3D6%26spage%3D612%26epage%3D619%26doi%3D10.1124%2Fdmd.32.6.612" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group">Tang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mayersohn, M.</span><span> </span><span class="NLM_article-title">A novel method for prediction of human drug clearance by allometric scaling</span> <span class="citation_source-journal">Drug Metab. Dispos.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x"> (</span><span class="NLM_issue">9</span><span class="NLM_x">) </span> <span class="NLM_fpage">1297</span><span class="NLM_x">–</span> <span class="NLM_lpage">1303</span><span class="refDoi"> DOI: 10.1124/dmd.105.004143</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.1124%2Fdmd.105.004143" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2005&pages=1297-1303&issue=9&author=H.+Tangauthor=M.+Mayersohn&title=A+novel+method+for+prediction+of+human+drug+clearance+by+allometric+scaling&doi=10.1124%2Fdmd.105.004143"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1124%2Fdmd.105.004143&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.105.004143%26sid%3Dliteratum%253Aachs%26aulast%3DTang%26aufirst%3DH.%26aulast%3DMayersohn%26aufirst%3DM.%26atitle%3DA%2520novel%2520method%2520for%2520prediction%2520of%2520human%2520drug%2520clearance%2520by%2520allometric%2520scaling%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2005%26volume%3D33%26issue%3D9%26spage%3D1297%26epage%3D1303%26doi%3D10.1124%2Fdmd.105.004143" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group">Page, K. M.</span><span> </span><span class="NLM_article-title">Validation of early human dose prediction: a key metric for compound progression in drug discovery</span> <span class="citation_source-journal">Mol. Pharmaceutics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">609</span><span class="NLM_x">–</span> <span class="NLM_lpage">620</span><span class="refDoi"> DOI: 10.1021/acs.molpharmaceut.5b00840</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.molpharmaceut.5b00840" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVGrs7%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2016&pages=609-620&issue=2&author=K.+M.+Page&title=Validation+of+early+human+dose+prediction%3A+a+key+metric+for+compound+progression+in+drug+discovery&doi=10.1021%2Facs.molpharmaceut.5b00840"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Validation of Early Human Dose Prediction: A Key Metric for Compound Progression in Drug Discovery</span></div><div class="casAuthors">Page, Ken M.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmaceutics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">609-620</span>CODEN:
                <span class="NLM_cas:coden">MPOHBP</span>;
        ISSN:<span class="NLM_cas:issn">1543-8384</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Human dose prediction is increasingly recognized as an important parameter in Drug Discovery.  Validation of a method using only in vitro and predicted parameters incorporated into a PK model was undertaken by predicting human dose and free Cmax for a no. of marketed drugs and AZ Development compds.  Doses were compared to those most relevant to marketed drugs or to clin. administered doses of AZ compds. normalized either to predicted Cmin or Cmax values.  Av. (AFE) and abs. av. (AAFE) fold-error anal. showed that best predictions were obtained using a QSAR model as the source of Vss, with Fabs set to 1 for acids and 0.5 for all other ion classes; for clearance prediction no binding correction to the well stirred model (WSM) was used for bases, while it was set to Fup/Fup0.5 for all other ion classes.  Using this combination of methods, predicted doses for 45 to 68% of the Cmin- and Cmax-normalized and marketed drug data sets were within 3-fold of the obsd. values, while 82 to 92% of these data sets were within 10-fold.  This method for early human dose prediction is able to rank, identify, and flag risks or optimization opportunities for future development compds. within 10 days of first synthesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoT--CfNq9EGLVg90H21EOLACvtfcHk0lgxTjNDpEw4lg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVGrs7%252FO&md5=612ece6c4e2729ab4694b0ae168e4cba</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Facs.molpharmaceut.5b00840&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.molpharmaceut.5b00840%26sid%3Dliteratum%253Aachs%26aulast%3DPage%26aufirst%3DK.%2BM.%26atitle%3DValidation%2520of%2520early%2520human%2520dose%2520prediction%253A%2520a%2520key%2520metric%2520for%2520compound%2520progression%2520in%2520drug%2520discovery%26jtitle%3DMol.%2520Pharmaceutics%26date%3D2016%26volume%3D13%26issue%3D2%26spage%3D609%26epage%3D620%26doi%3D10.1021%2Facs.molpharmaceut.5b00840" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group">Jones, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rowland, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gibson, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yates, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chien, J. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ring, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adkison, K. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ku, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vuppugalla, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marathe, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fischer, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dutta, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sinha, V. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bjornsson, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lavé, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poulin, P.</span><span> </span><span class="NLM_article-title">PhARMA CPCDC initiative on predictive models or human pharmacokinetics, part 2: Comparative assessment of predictions methods of human volume of distribution</span> <span class="citation_source-journal">J. Pharm. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">100</span><span class="NLM_x"> (</span><span class="NLM_issue">10</span><span class="NLM_x">) </span> <span class="NLM_fpage">4074</span><span class="NLM_x">–</span> <span class="NLM_lpage">4089</span><span class="refDoi"> DOI: 10.1002/jps.22553</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.1002%2Fjps.22553" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=100&publication_year=2011&pages=4074-4089&issue=10&author=R.+D.+Jonesauthor=H.+M.+Jonesauthor=M.+Rowlandauthor=C.+R.+Gibsonauthor=J.+W.+Yatesauthor=J.+Y.+Chienauthor=B.+J.+Ringauthor=K.+K.+Adkisonauthor=M.+S.+Kuauthor=H.+Heauthor=R.+Vuppugallaauthor=P.+Maratheauthor=V.+Fischerauthor=S.+Duttaauthor=V.+K.+Sinhaauthor=T.+Bjornssonauthor=T.+Lav%C3%A9author=P.+Poulin&title=PhARMA+CPCDC+initiative+on+predictive+models+or+human+pharmacokinetics%2C+part+2%3A+Comparative+assessment+of+predictions+methods+of+human+volume+of+distribution&doi=10.1002%2Fjps.22553"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1002%2Fjps.22553&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjps.22553%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DR.%2BD.%26aulast%3DJones%26aufirst%3DH.%2BM.%26aulast%3DRowland%26aufirst%3DM.%26aulast%3DGibson%26aufirst%3DC.%2BR.%26aulast%3DYates%26aufirst%3DJ.%2BW.%26aulast%3DChien%26aufirst%3DJ.%2BY.%26aulast%3DRing%26aufirst%3DB.%2BJ.%26aulast%3DAdkison%26aufirst%3DK.%2BK.%26aulast%3DKu%26aufirst%3DM.%2BS.%26aulast%3DHe%26aufirst%3DH.%26aulast%3DVuppugalla%26aufirst%3DR.%26aulast%3DMarathe%26aufirst%3DP.%26aulast%3DFischer%26aufirst%3DV.%26aulast%3DDutta%26aufirst%3DS.%26aulast%3DSinha%26aufirst%3DV.%2BK.%26aulast%3DBjornsson%26aufirst%3DT.%26aulast%3DLav%25C3%25A9%26aufirst%3DT.%26aulast%3DPoulin%26aufirst%3DP.%26atitle%3DPhARMA%2520CPCDC%2520initiative%2520on%2520predictive%2520models%2520or%2520human%2520pharmacokinetics%252C%2520part%25202%253A%2520Comparative%2520assessment%2520of%2520predictions%2520methods%2520of%2520human%2520volume%2520of%2520distribution%26jtitle%3DJ.%2520Pharm.%2520Sci.%26date%3D2011%26volume%3D100%26issue%3D10%26spage%3D4074%26epage%3D4089%26doi%3D10.1002%2Fjps.22553" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group">Lee, M.-L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aliagas, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dotson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gobbi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heffron, T.</span><span> </span><span class="NLM_article-title">DEGAS: sharing and tracking target compound ideas with external collaborators</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">278</span><span class="NLM_x">–</span> <span class="NLM_lpage">284</span><span class="refDoi"> DOI: 10.1021/ci2003297</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci2003297" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVGru73N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=278-284&issue=2&author=M.-L.+Leeauthor=I.+Aliagasauthor=J.+Dotsonauthor=J.+A.+Fengauthor=A.+Gobbiauthor=T.+Heffron&title=DEGAS%3A+sharing+and+tracking+target+compound+ideas+with+external+collaborators&doi=10.1021%2Fci2003297"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">DEGAS: Sharing and Tracking Target Compound Ideas with External Collaborators</span></div><div class="casAuthors">Lee, Man-Ling; Aliagas, Ignacio; Dotson, Jennafer; Feng, Jianwen A.; Gobbi, Alberto; Heffron, Timothy</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">278-284</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">To minimize the risk of failure in clin. trials, drug discovery teams must propose active and selective clin. candidates with good physicochem. properties.  An addnl. challenge is that today drug discovery is often conducted by teams at different geog. locations.  To improve the collaborative decision making on which compds. to synthesize, we have implemented DEGAS, an application which enables scientists from Genentech and from collaborating external partners to instantly access the same data.  DEGAS was implemented to ensure that only the best target compds. are made and that they are made without duplicate effort.  Physicochem. properties and DMPK model predictions are computed for each compd. to allow the team to make informed decisions when prioritizing.  The synthesis progress can be easily tracked.  While developing DEGAS, ease of use was a particular goal in order to minimize the difficulty of training and supporting remote users.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWoQ8pIpOo0rVg90H21EOLACvtfcHk0lgxTjNDpEw4lg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVGru73N&md5=8539de9cc3ea25049905b94774292e0f</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Fci2003297&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci2003297%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DM.-L.%26aulast%3DAliagas%26aufirst%3DI.%26aulast%3DDotson%26aufirst%3DJ.%26aulast%3DFeng%26aufirst%3DJ.%2BA.%26aulast%3DGobbi%26aufirst%3DA.%26aulast%3DHeffron%26aufirst%3DT.%26atitle%3DDEGAS%253A%2520sharing%2520and%2520tracking%2520target%2520compound%2520ideas%2520with%2520external%2520collaborators%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26issue%3D2%26spage%3D278%26epage%3D284%26doi%3D10.1021%2Fci2003297" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span>Spotfire. <a href="http://spotfire.tibco.com/" class="extLink">http://spotfire.tibco.com/</a> (accessed Aug 15,<span class="NLM_x"> </span><span class="NLM_year">2016</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Spotfire.+http%3A%2F%2Fspotfire.tibco.com%2F+%28accessed+Aug+15%2C+2016%29+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span>MedChem Studio. <a href="http://www.simulations-plus.com/Products.aspx?pID=12" class="extLink">http://www.simulations-plus.com/Products.aspx?pID=12</a> (accessed Aug 15,<span class="NLM_x"> </span><span class="NLM_year">2016</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=MedChem+Studio.+http%3A%2F%2Fwww.simulations-plus.com%2FProducts.aspx%3FpID%3D12+%28accessed+Aug+15%2C+2016%29+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span>Vortex. <a href="http://www.dotmatics.com/products/vortex/" class="extLink">http://www.dotmatics.com/products/vortex/</a> (accessed Aug 15,<span class="NLM_x"> </span><span class="NLM_year">2016</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Vortex.+http%3A%2F%2Fwww.dotmatics.com%2Fproducts%2Fvortex%2F+%28accessed+Aug+15%2C+2016%29+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group">Nissink, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Degorce, S.</span><span> </span><span class="NLM_article-title">Analyzing compound and project progress through multi-objective-based compound quality assessment</span> <span class="citation_source-journal">Future Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x"> (</span><span class="NLM_issue">7</span><span class="NLM_x">) </span> <span class="NLM_fpage">753</span><span class="NLM_x">–</span> <span class="NLM_lpage">767</span><span class="refDoi"> DOI: 10.4155/fmc.13.45</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.4155%2Ffmc.13.45" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=23651090" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BC3sXntFyrsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2013&pages=753-767&issue=7&author=J.+W.+Nissinkauthor=S.+Degorce&title=Analyzing+compound+and+project+progress+through+multi-objective-based+compound+quality+assessment&doi=10.4155%2Ffmc.13.45"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Analyzing compound and project progress through multi-objective-based compound quality assessment</span></div><div class="casAuthors">Nissink, J. Willem M.; Degorce, Sebastien</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">753-767</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">Background: Compd.-quality scoring methods designed to evaluate multiple drug properties concurrently are useful to analyze and prioritize output from drug-design efforts.  However, formalized multiparameter optimization approaches are not widely used in drug design.  Methods: We rank mols. synthesized in drug-discovery projects using simple and aggregated desirability functions reflecting medicinal chem. rules'.  Our quality score deals transparently with missing data, a key requirement in drug-hunting projects where data availability is often limited.  We further est. confidence in the interpretation of such a compd.-quality measure.  Conclusion: Scores and assocd. confidences provide systematic insight in the quality of emerging chem. equity.  Tracking quality of synthetic output over time yields valuable insight into the progress of drug-design teams, with potential applications in risk and resource management of a drug portfolio.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4eLt5NGn-G7Vg90H21EOLACvtfcHk0lgnUd-sWcj58Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXntFyrsrg%253D&md5=892624a14de294a7f80cb3fb87bcea53</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.4155%2Ffmc.13.45&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.13.45%26sid%3Dliteratum%253Aachs%26aulast%3DNissink%26aufirst%3DJ.%2BW.%26aulast%3DDegorce%26aufirst%3DS.%26atitle%3DAnalyzing%2520compound%2520and%2520project%2520progress%2520through%2520multi-objective-based%2520compound%2520quality%2520assessment%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2013%26volume%3D5%26issue%3D7%26spage%3D753%26epage%3D767%26doi%3D10.4155%2Ffmc.13.45" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group">Segall, M. D.</span><span> </span><span class="NLM_article-title">Multi-parameter optimization: Identifying high quality compounds with a balance of properties</span> <span class="citation_source-journal">Curr. Pharm. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x"> (</span><span class="NLM_issue">9</span><span class="NLM_x">) </span> <span class="NLM_fpage">1292</span><span class="NLM_x">–</span> <span class="NLM_lpage">1310</span><span class="refDoi"> DOI: 10.2174/138161212799436430</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.2174%2F138161212799436430" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=22316157" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BC38XlvVSqsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2012&pages=1292-1310&issue=9&author=M.+D.+Segall&title=Multi-parameter+optimization%3A+Identifying+high+quality+compounds+with+a+balance+of+properties&doi=10.2174%2F138161212799436430"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Multi-parameter optimization: identifying high quality compounds with a balance of properties</span></div><div class="casAuthors">Segall, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Current Pharmaceutical Design</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1292-1310</span>CODEN:
                <span class="NLM_cas:coden">CPDEFP</span>;
        ISSN:<span class="NLM_cas:issn">1381-6128</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  A successful, efficacious and safe drug must have a balance of properties, including potency against its intended target, appropriate absorption, distribution, metab., and elimination (ADME) properties and an acceptable safety profile.  Achieving this balance of, often conflicting, requirements is a major challenge in drug discovery.  Approaches to simultaneously optimizing many factors in a design are broadly described under the term "multi-parameter optimization" (MPO).  In this review, we will describe how MPO can be applied to efficiently design and select high quality compds. and describe the range of methods that have been employed in drug discovery, including; simple "rules of thumb" such as Lipinski's rule; desirability functions; Pareto optimization; and probabilistic approaches that take into consideration the uncertainty in all drug discovery data due to predictive error and exptl. variability.  We will explore how these methods have been applied to predicted and exptl. data to reduce attrition and improve the productivity of the drug discovery process.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPFOVyg2ehWbVg90H21EOLACvtfcHk0lgnUd-sWcj58Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XlvVSqsbk%253D&md5=43f90ad751fd2c0bc47715f465ef31b3</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.2174%2F138161212799436430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138161212799436430%26sid%3Dliteratum%253Aachs%26aulast%3DSegall%26aufirst%3DM.%2BD.%26atitle%3DMulti-parameter%2520optimization%253A%2520Identifying%2520high%2520quality%2520compounds%2520with%2520a%2520balance%2520of%2520properties%26jtitle%3DCurr.%2520Pharm.%2520Des.%26date%3D2012%26volume%3D18%26issue%3D9%26spage%3D1292%26epage%3D1310%26doi%3D10.2174%2F138161212799436430" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group">Debe, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mamidipaka, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gregg, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Metz, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gupta, R. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muchmore, S. W.</span><span> </span><span class="NLM_article-title">ALOHA: a novel probability fusion approach for scoring multi-parameter drug-likeness during the lead optimization stage of drug discovery</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">27</span><span class="NLM_x"> (</span><span class="NLM_issue">9</span><span class="NLM_x">) </span> <span class="NLM_fpage">771</span><span class="NLM_x">–</span> <span class="NLM_lpage">782</span><span class="refDoi"> DOI: 10.1007/s10822-013-9679-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.1007%2Fs10822-013-9679-x" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2013&pages=771-782&issue=9&author=D.+A.+Debeauthor=R.+B.+Mamidipakaauthor=R.+J.+Greggauthor=J.+T.+Metzauthor=R.+R.+Guptaauthor=S.+W.+Muchmore&title=ALOHA%3A+a+novel+probability+fusion+approach+for+scoring+multi-parameter+drug-likeness+during+the+lead+optimization+stage+of+drug+discovery&doi=10.1007%2Fs10822-013-9679-x"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1007%2Fs10822-013-9679-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-013-9679-x%26sid%3Dliteratum%253Aachs%26aulast%3DDebe%26aufirst%3DD.%2BA.%26aulast%3DMamidipaka%26aufirst%3DR.%2BB.%26aulast%3DGregg%26aufirst%3DR.%2BJ.%26aulast%3DMetz%26aufirst%3DJ.%2BT.%26aulast%3DGupta%26aufirst%3DR.%2BR.%26aulast%3DMuchmore%26aufirst%3DS.%2BW.%26atitle%3DALOHA%253A%2520a%2520novel%2520probability%2520fusion%2520approach%2520for%2520scoring%2520multi-parameter%2520drug-likeness%2520during%2520the%2520lead%2520optimization%2520stage%2520of%2520drug%2520discovery%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2013%26volume%3D27%26issue%3D9%26spage%3D771%26epage%3D782%26doi%3D10.1007%2Fs10822-013-9679-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group">Hussain, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rea, C.</span><span> </span><span class="NLM_article-title">Computationally efficient algorithm to identify matched molecular pairs (MMPs) in large data sets</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x"> (</span><span class="NLM_issue">3</span><span class="NLM_x">) </span> <span class="NLM_fpage">339</span><span class="NLM_x">–</span> <span class="NLM_lpage">348</span><span class="refDoi"> DOI: 10.1021/ci900450m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci900450m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlWltr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=339-348&issue=3&author=J.+Hussainauthor=C.+Rea&title=Computationally+efficient+algorithm+to+identify+matched+molecular+pairs+%28MMPs%29+in+large+data+sets&doi=10.1021%2Fci900450m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Computationally Efficient Algorithm to Identify Matched Molecular Pairs (MMPs) in Large Data Sets</span></div><div class="casAuthors">Hussain, Jameed; Rea, Ceara</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">339-348</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Modern drug discovery organizations generate large vols. of SAR data.  A promising methodol. that can be used to mine this chem. data to identify novel structure-activity relationships is the matched mol. pair (MMP) methodol.  However, before the full potential of the MMP methodol. can be utilized, a MMP identification method that is capable of identifying all MMPs in large chem. data sets on modest computational hardware is required.  In this paper we report an algorithm that is capable of systematically generating all MMPs in chem. data sets.  Addnl., the algorithm is computationally efficient enough to be applied on large data sets.  As an example the algorithm was used to identify the MMPs in the ∼300k NIH MLSMR set.  The algorithm identified ∼5.3 million matched mol. pairs in the set.  These pairs cover ∼2.6 million unique mol. transformations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoF0rN8jaKSjrVg90H21EOLACvtfcHk0lgnUd-sWcj58Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlWltr4%253D&md5=23964abebc23274f5c36c319bee52bab</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1021%2Fci900450m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci900450m%26sid%3Dliteratum%253Aachs%26aulast%3DHussain%26aufirst%3DJ.%26aulast%3DRea%26aufirst%3DC.%26atitle%3DComputationally%2520efficient%2520algorithm%2520to%2520identify%2520matched%2520molecular%2520pairs%2520%2528MMPs%2529%2520in%2520large%2520data%2520sets%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26issue%3D3%26spage%3D339%26epage%3D348%26doi%3D10.1021%2Fci900450m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group">Papadatos, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alkarouri, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gillet, V. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willett, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kadirkamanathan, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luscombe, C. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bravi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richmond, N. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pickett, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hussain, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pritchard, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cooper, A. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Macdonald, S. J.</span><span> </span><span class="NLM_article-title">Lead optimization using matched molecular pairs: inclusion of contextual information for enhanced prediction of HERG inhibition, solubility, and lipophilicity</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x"> (</span><span class="NLM_issue">10</span><span class="NLM_x">) </span> <span class="NLM_fpage">1872</span><span class="NLM_x">–</span> <span class="NLM_lpage">1886</span><span class="refDoi"> DOI: 10.1021/ci100258p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci100258p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1Whs7jJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=1872-1886&issue=10&author=G.+Papadatosauthor=M.+Alkarouriauthor=V.+J.+Gilletauthor=P.+Willettauthor=V.+Kadirkamanathanauthor=C.+N.+Luscombeauthor=G.+Braviauthor=N.+J.+Richmondauthor=S.+D.+Pickettauthor=J.+Hussainauthor=J.+M.+Pritchardauthor=A.+W.+Cooperauthor=S.+J.+Macdonald&title=Lead+optimization+using+matched+molecular+pairs%3A+inclusion+of+contextual+information+for+enhanced+prediction+of+HERG+inhibition%2C+solubility%2C+and+lipophilicity&doi=10.1021%2Fci100258p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Lead Optimization Using Matched Molecular Pairs: Inclusion of Contextual Information for Enhanced Prediction of hERG Inhibition, Solubility, and Lipophilicity</span></div><div class="casAuthors">Papadatos, George; Alkarouri, Muhammad; Gillet, Valerie J.; Willett, Peter; Kadirkamanathan, Visakan; Luscombe, Christopher N.; Bravi, Gianpaolo; Richmond, Nicola J.; Pickett, Stephen D.; Hussain, Jameed; Pritchard, John M.; Cooper, Anthony W. J.; MacDonald, Simon J. F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1872-1886</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Previous studies of the anal. of mol. matched pairs (MMPs) have often assumed that the effect of a substructural transformation on a mol. property is independent of the context (i.e., the local structural environment in which that transformation occurs).  Expts. with large sets of hERG, soly., and lipophilicity data demonstrate that the inclusion of contextual information can enhance the predictive power of MMP analyses, with significant trends (both pos. and neg.) being identified that are not apparent when using conventional, context-independent approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLZfW5E3jgX7Vg90H21EOLACvtfcHk0ljxUclbGHD0nA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht1Whs7jJ&md5=464dff86e047cf1b0273571f82786994</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1021%2Fci100258p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100258p%26sid%3Dliteratum%253Aachs%26aulast%3DPapadatos%26aufirst%3DG.%26aulast%3DAlkarouri%26aufirst%3DM.%26aulast%3DGillet%26aufirst%3DV.%2BJ.%26aulast%3DWillett%26aufirst%3DP.%26aulast%3DKadirkamanathan%26aufirst%3DV.%26aulast%3DLuscombe%26aufirst%3DC.%2BN.%26aulast%3DBravi%26aufirst%3DG.%26aulast%3DRichmond%26aufirst%3DN.%2BJ.%26aulast%3DPickett%26aufirst%3DS.%2BD.%26aulast%3DHussain%26aufirst%3DJ.%26aulast%3DPritchard%26aufirst%3DJ.%2BM.%26aulast%3DCooper%26aufirst%3DA.%2BW.%26aulast%3DMacdonald%26aufirst%3DS.%2BJ.%26atitle%3DLead%2520optimization%2520using%2520matched%2520molecular%2520pairs%253A%2520inclusion%2520of%2520contextual%2520information%2520for%2520enhanced%2520prediction%2520of%2520HERG%2520inhibition%252C%2520solubility%252C%2520and%2520lipophilicity%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26issue%3D10%26spage%3D1872%26epage%3D1886%26doi%3D10.1021%2Fci100258p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group">Leach, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, H. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cosgrove, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kenny, P. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruston, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacFaul, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wood, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Colclough, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Law, B.</span><span> </span><span class="NLM_article-title">Matched molecular pairs as a guide in the optimization of pharmaceutical properties; a study of aqueous solubility, plasma protein binding and oral exposure</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x"> (</span><span class="NLM_issue">23</span><span class="NLM_x">) </span> <span class="NLM_fpage">6672</span><span class="NLM_x">–</span> <span class="NLM_lpage">6682</span><span class="refDoi"> DOI: 10.1021/jm0605233</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0605233" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=6672-6682&issue=23&author=A.+G.+Leachauthor=H.+D.+Jonesauthor=D.+A.+Cosgroveauthor=P.+W.+Kennyauthor=L.+Rustonauthor=P.+MacFaulauthor=J.+M.+Woodauthor=N.+Colcloughauthor=B.+Law&title=Matched+molecular+pairs+as+a+guide+in+the+optimization+of+pharmaceutical+properties%3B+a+study+of+aqueous+solubility%2C+plasma+protein+binding+and+oral+exposure&doi=10.1021%2Fjm0605233"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1021%2Fjm0605233&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0605233%26sid%3Dliteratum%253Aachs%26aulast%3DLeach%26aufirst%3DA.%2BG.%26aulast%3DJones%26aufirst%3DH.%2BD.%26aulast%3DCosgrove%26aufirst%3DD.%2BA.%26aulast%3DKenny%26aufirst%3DP.%2BW.%26aulast%3DRuston%26aufirst%3DL.%26aulast%3DMacFaul%26aufirst%3DP.%26aulast%3DWood%26aufirst%3DJ.%2BM.%26aulast%3DColclough%26aufirst%3DN.%26aulast%3DLaw%26aufirst%3DB.%26atitle%3DMatched%2520molecular%2520pairs%2520as%2520a%2520guide%2520in%2520the%2520optimization%2520of%2520pharmaceutical%2520properties%253B%2520a%2520study%2520of%2520aqueous%2520solubility%252C%2520plasma%2520protein%2520binding%2520and%2520oral%2520exposure%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26issue%3D23%26spage%3D6672%26epage%3D6682%26doi%3D10.1021%2Fjm0605233" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group">Warner, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Griffen, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">St-Gallay, S. A.</span><span> </span><span class="NLM_article-title">WizePairZ: A novel algorithm to identify, encode and exploit matched molecular pairs with unspecified cores in medicinal chemistry</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x"> (</span><span class="NLM_issue">8</span><span class="NLM_x">) </span> <span class="NLM_fpage">1350</span><span class="NLM_x">–</span> <span class="NLM_lpage">1357</span><span class="refDoi"> DOI: 10.1021/ci100084s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci100084s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BC3cXpvVSgs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=1350-1357&issue=8&author=D.+J.+Warnerauthor=E.+J.+Griffenauthor=S.+A.+St-Gallay&title=WizePairZ%3A+A+novel+algorithm+to+identify%2C+encode+and+exploit+matched+molecular+pairs+with+unspecified+cores+in+medicinal+chemistry&doi=10.1021%2Fci100084s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">WizePairZ: A Novel Algorithm to Identify, Encode, and Exploit Matched Molecular Pairs with Unspecified Cores in Medicinal Chemistry</span></div><div class="casAuthors">Warner, Daniel J.; Griffen, Edward J.; St-Gallay, Stephen A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1350-1357</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">An algorithm to automatically identify and ext. matched mol. pairs from a collection of compds. has been developed, allowing the learning assocd. with each mol. transformation to be readily exploited in drug discovery projects.  Here, we present the application to an example data set of 11 histone deacetylase inhibitors.  The matched pairs were identified, and corresponding differences in activity and lipophilicity were recorded.  These property differences were assocd. with the chem. transformations encoded in the SMIRKS reaction notation.  The transformations identified a subseries with the optimal balance of these two parameters.  Enumeration of a virtual library of compds. using the extd. transformations identified two addnl. compds. initially excluded from the anal. with an accurate estn. of their biol. activity.  We describe how the WizePairZ system can be used to archive and apply medicinal chem. knowledge from one drug discovery project to another as well as identify common bioisosteres.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpSWafs2qQ4UbVg90H21EOLACvtfcHk0ljxUclbGHD0nA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXpvVSgs7c%253D&md5=9d83eb14cdb56079429761e79beeb292</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1021%2Fci100084s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100084s%26sid%3Dliteratum%253Aachs%26aulast%3DWarner%26aufirst%3DD.%2BJ.%26aulast%3DGriffen%26aufirst%3DE.%2BJ.%26aulast%3DSt-Gallay%26aufirst%3DS.%2BA.%26atitle%3DWizePairZ%253A%2520A%2520novel%2520algorithm%2520to%2520identify%252C%2520encode%2520and%2520exploit%2520matched%2520molecular%2520pairs%2520with%2520unspecified%2520cores%2520in%2520medicinal%2520chemistry%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26issue%3D8%26spage%3D1350%26epage%3D1357%26doi%3D10.1021%2Fci100084s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group">Sushko, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Novotarskyi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vogt, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abdelaziz, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tetko, I. V.</span><span> </span><span class="NLM_article-title">Prediction-driven matched molecular pairs to interpret QSARs and aid the molecular optimization process</span> <span class="citation_source-journal">J. Cheminf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">48</span><span class="refDoi"> DOI: 10.1186/s13321-014-0048-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.1186%2Fs13321-014-0048-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BC2MXislWhtLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&pages=48&issue=1&author=Y.+Sushkoauthor=S.+Novotarskyiauthor=R.+Kornerauthor=J.+Vogtauthor=A.+Abdelazizauthor=I.+V.+Tetko&title=Prediction-driven+matched+molecular+pairs+to+interpret+QSARs+and+aid+the+molecular+optimization+process&doi=10.1186%2Fs13321-014-0048-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Prediction-driven matched molecular pairs to interpret QSARs and aid the molecular optimization process</span></div><div class="casAuthors">Sushko, Yurii; Novotarskyi, Sergii; Koerner, Robert; Vogt, Joachim; Abdelaziz, Ahmed; Tetko, Igor V.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cheminformatics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">48/1-48/39, 39 pp.</span>CODEN:
                <span class="NLM_cas:coden">JCOHB3</span>;
        ISSN:<span class="NLM_cas:issn">1758-2946</span>.
    
            (<span class="NLM_cas:orgname">Chemistry Central Ltd.</span>)
        </div><div class="casAbstract">Background QSAR is an established and powerful method for cheap in silico assessment of physicochem. properties and biol. activities of chem. compds.  However, QSAR models are rather complex math. constructs that cannot easily be interpreted.  Medicinal chemists would benefit from practical guidance regarding which mols. to synthesize.  Another possible approach is anal. of pairs of very similar mols., so-called matched mol. pairs (MMPs).  Such an approach allows identification of mol. transformations that affect particular activities (e.g. toxicity).  In contrast to QSAR, chem. interpretation of these transformations is straightforward.  Furthermore, such transformations can give medicinal chemists useful hints for the hit-to-lead optimization process.  Results The current study suggests a combination of QSAR and MMP approaches by finding MMP transformations based on QSAR predictions for large chem. datasets.  The study shows that such an approach, referred to as prediction-driven MMP anal., is a useful tool for medicinal chemists, allowing identification of large nos. of "interesting" transformations that can be used to drive the mol. optimization process.  All the methodol. developments have been implemented as software products available online as part of OCHEM.  Conclusions The prediction-driven MMPs methodol. was exemplified by two use cases: modeling of aquatic toxicity and CYP3A4 inhibition.  This approach helped us to interpret QSAR models and allowed identification of a no. of "significant" mol. transformations that affect the desired properties.  This can facilitate drug design as a part of mol. optimization process.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTzctKLS8vxrVg90H21EOLACvtfcHk0ljzR_JCBomfLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXislWhtLw%253D&md5=837bda80de79a7621c887de1fed0375a</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1186%2Fs13321-014-0048-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13321-014-0048-0%26sid%3Dliteratum%253Aachs%26aulast%3DSushko%26aufirst%3DY.%26aulast%3DNovotarskyi%26aufirst%3DS.%26aulast%3DKorner%26aufirst%3DR.%26aulast%3DVogt%26aufirst%3DJ.%26aulast%3DAbdelaziz%26aufirst%3DA.%26aulast%3DTetko%26aufirst%3DI.%2BV.%26atitle%3DPrediction-driven%2520matched%2520molecular%2520pairs%2520to%2520interpret%2520QSARs%2520and%2520aid%2520the%2520molecular%2520optimization%2520process%26jtitle%3DJ.%2520Cheminf.%26date%3D2014%26volume%3D6%26issue%3D1%26spage%3D48%26doi%3D10.1186%2Fs13321-014-0048-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group">O’Boyle, N. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bostrom, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sayle, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gill, A.</span><span> </span><span class="NLM_article-title">Using matched molecular series as a predictive tool to optimize biological activity</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x"> (</span><span class="NLM_issue">6</span><span class="NLM_x">) </span> <span class="NLM_fpage">2704</span><span class="NLM_x">–</span> <span class="NLM_lpage">2713</span><span class="refDoi"> DOI: 10.1021/jm500022q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm500022q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=2704-2713&issue=6&author=N.+M.+O%E2%80%99Boyleauthor=J.+Bostromauthor=R.+A.+Sayleauthor=A.+Gill&title=Using+matched+molecular+series+as+a+predictive+tool+to+optimize+biological+activity&doi=10.1021%2Fjm500022q"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1021%2Fjm500022q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm500022q%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Boyle%26aufirst%3DN.%2BM.%26aulast%3DBostrom%26aufirst%3DJ.%26aulast%3DSayle%26aufirst%3DR.%2BA.%26aulast%3DGill%26aufirst%3DA.%26atitle%3DUsing%2520matched%2520molecular%2520series%2520as%2520a%2520predictive%2520tool%2520to%2520optimize%2520biological%2520activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26issue%3D6%26spage%3D2704%26epage%3D2713%26doi%3D10.1021%2Fjm500022q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group">Gupta-Ostermann, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajorath, J.</span><span> </span><span class="NLM_article-title">The “SAR matrix” method and its extensions for applications in medicinal chemistry and chemogenomics</span> <span class="citation_source-journal">F1000Research</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">113</span><span class="refDoi"> DOI: 10.12688/f1000research.4185.2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.12688%2Ff1000research.4185.2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=25383183" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A280%3ADC%252BC2M3ntVeitw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2014&pages=113&author=D.+Gupta-Ostermannauthor=J.+Bajorath&title=The+%E2%80%9CSAR+matrix%E2%80%9D+method+and+its+extensions+for+applications+in+medicinal+chemistry+and+chemogenomics&doi=10.12688%2Ff1000research.4185.2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">The 'SAR Matrix' method and its extensions for applications in medicinal chemistry and chemogenomics</span></div><div class="casAuthors">Gupta-Ostermann Disha; Bajorath Jurgen</div><div class="citationInfo"><span class="NLM_cas:title">F1000Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">113</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">We describe the 'Structure-Activity Relationship (SAR) Matrix' (SARM) methodology that is based upon a special two-step application of the matched molecular pair (MMP) formalism.  The SARM method has originally been designed for the extraction, organization, and visualization of compound series and associated SAR information from compound data sets.  It has been further developed and adapted for other applications including compound design, activity prediction, library extension, and the navigation of multi-target activity spaces.  The SARM approach and its extensions are presented here in context to introduce different types of applications and provide an example for the evolution of a computational methodology in pharmaceutical research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTkxRbz5791CpSWdIdmEou5fW6udTcc2ebzR3qIaqYyELntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2M3ntVeitw%253D%253D&md5=3d96e38612bfbbdf91fd404830c9e886</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.12688%2Ff1000research.4185.2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.12688%252Ff1000research.4185.2%26sid%3Dliteratum%253Aachs%26aulast%3DGupta-Ostermann%26aufirst%3DD.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DThe%2520%25E2%2580%259CSAR%2520matrix%25E2%2580%259D%2520method%2520and%2520its%2520extensions%2520for%2520applications%2520in%2520medicinal%2520chemistry%2520and%2520chemogenomics%26jtitle%3DF1000Research%26date%3D2014%26volume%3D3%26spage%3D113%26doi%3D10.12688%2Ff1000research.4185.2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group">Webb, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hanser, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Howlin, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krause, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vessey, J. D.</span><span> </span><span class="NLM_article-title">Feature combination networks for the interpretation of statistical machine learning models: application to Ames mutagenicity</span> <span class="citation_source-journal">J. Cheminf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">8</span><span class="refDoi"> DOI: 10.1186/1758-2946-6-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.1186%2F1758-2946-6-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BC2MXivFSktro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&pages=8&issue=1&author=S.+J.+Webbauthor=T.+Hanserauthor=B.+Howlinauthor=P.+Krauseauthor=J.+D.+Vessey&title=Feature+combination+networks+for+the+interpretation+of+statistical+machine+learning+models%3A+application+to+Ames+mutagenicity&doi=10.1186%2F1758-2946-6-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Feature combination networks for the interpretation of statistical machine learning models: application to Ames mutagenicity</span></div><div class="casAuthors">Webb, Samuel J.; Hanser, Thierry; Howlin, Brendan; Krause, Paul; Vessey, Jonathan D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cheminformatics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">8/1-8/21, 21 pp.</span>CODEN:
                <span class="NLM_cas:coden">JCOHB3</span>;
        ISSN:<span class="NLM_cas:issn">1758-2946</span>.
    
            (<span class="NLM_cas:orgname">Chemistry Central Ltd.</span>)
        </div><div class="casAbstract">Background: A new algorithm has been developed to enable the interpretation of black box models.  The developed algorithm is agnostic to learning algorithm and open to all structural based descriptors such as fragments, keys and hashed fingerprints.  The algorithm has provided meaningful interpretation of Ames mutagenicity predictions from both random forest and support vector machine models built on a variety of structural fingerprints.  A fragmentation algorithm is utilized to investigate the model's behavior on specific substructures present in the query.  An output is formulated summarising causes of activation and deactivation.  The algorithm is able to identify multiple causes of activation or deactivation in addn. to identifying localised deactivations where the prediction for the query is active overall.  No loss in performance is seen as there is no change in the prediction; the interpretation is produced directly on the model's behavior for the specific query.  Results: Models have been built using multiple learning algorithms including support vector machine and random forest.  The models were built on public Ames mutagenicity data and a variety of fingerprint descriptors were used.  These models produced a good performance in both internal and external validation with accuracies around 82%.  The models were used to evaluate the interpretation algorithm.  Interpretation was revealed that links closely with understood mechanisms for Ames mutagenicity.  Conclusion: This methodol. allows for a greater utilization of the predictions made by black box models and can expedite further study based on the output for a (quant.) structure activity model.  Addnl. the algorithm could be utilized for chem. dataset investigation and knowledge extn./human SAR development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp78W0IxwwolbVg90H21EOLACvtfcHk0ljzR_JCBomfLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXivFSktro%253D&md5=3908e46c4b0d7320d1b105c3dbb0af17</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1186%2F1758-2946-6-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1758-2946-6-8%26sid%3Dliteratum%253Aachs%26aulast%3DWebb%26aufirst%3DS.%2BJ.%26aulast%3DHanser%26aufirst%3DT.%26aulast%3DHowlin%26aufirst%3DB.%26aulast%3DKrause%26aufirst%3DP.%26aulast%3DVessey%26aufirst%3DJ.%2BD.%26atitle%3DFeature%2520combination%2520networks%2520for%2520the%2520interpretation%2520of%2520statistical%2520machine%2520learning%2520models%253A%2520application%2520to%2520Ames%2520mutagenicity%26jtitle%3DJ.%2520Cheminf.%26date%3D2014%26volume%3D6%26issue%3D1%26spage%3D8%26doi%3D10.1186%2F1758-2946-6-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group">Guha, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jurs, P. C.</span><span> </span><span class="NLM_article-title">Interpreting computational neural network QSAR models: a measure of descriptor importance</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x"> (</span><span class="NLM_issue">3</span><span class="NLM_x">) </span> <span class="NLM_fpage">800</span><span class="NLM_x">–</span> <span class="NLM_lpage">806</span><span class="refDoi"> DOI: 10.1021/ci050022a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci050022a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BD2MXisleqs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2005&pages=800-806&issue=3&author=R.+Guhaauthor=P.+C.+Jurs&title=Interpreting+computational+neural+network+QSAR+models%3A+a+measure+of+descriptor+importance&doi=10.1021%2Fci050022a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Interpreting Computational Neural Network QSAR Models: A Measure of Descriptor Importance</span></div><div class="casAuthors">Guha, Rajarshi; Jurs, Peter C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">800-806</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We present a method to measure the relative importance of the descriptors present in a QSAR model developed with a computational neural network (CNN).  The approach is based on a sensitivity anal. of the descriptors.  We tested the method on three published data sets for which linear and CNN models were previously built.  The original work reported interpretations for the linear models, and we compare the results of the new method to the importance of descriptors in the linear models as described by a PLS technique.  The results indicate that the proposed method is able to rank descriptors such that important descriptors in the CNN model correspond to the important descriptors in the linear model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXIOYkme8SaLVg90H21EOLACvtfcHk0lihgdfLE181xg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXisleqs7k%253D&md5=6aa8c896834301c305ebefa91b0fb82b</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1021%2Fci050022a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci050022a%26sid%3Dliteratum%253Aachs%26aulast%3DGuha%26aufirst%3DR.%26aulast%3DJurs%26aufirst%3DP.%2BC.%26atitle%3DInterpreting%2520computational%2520neural%2520network%2520QSAR%2520models%253A%2520a%2520measure%2520of%2520descriptor%2520importance%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2005%26volume%3D45%26issue%3D3%26spage%3D800%26epage%3D806%26doi%3D10.1021%2Fci050022a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref69"><div class="reference"><strong class="refLabel"><a href="#ref69" class="rightTabRefNumLink">69</a></strong><div class="NLM_citation" id="rightTab-cit69"><span><span class="NLM_contrib-group">Schneider, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fechner, U.</span><span> </span><span class="NLM_article-title">Computer-based de novo design of drug-like molecules</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x"> (</span><span class="NLM_issue">8</span><span class="NLM_x">) </span> <span class="NLM_fpage">649</span><span class="NLM_x">–</span> <span class="NLM_lpage">663</span><span class="refDoi"> DOI: 10.1038/nrd1799</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.1038%2Fnrd1799" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=16056391" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BD2MXmvVOqtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2005&pages=649-663&issue=8&author=G.+Schneiderauthor=U.+Fechner&title=Computer-based+de+novo+design+of+drug-like+molecules&doi=10.1038%2Fnrd1799"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Computer-based de novo design of drug-like molecules</span></div><div class="casAuthors">Schneider, Gisbert; Fechner, Uli</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">649-663</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review with refs.  Ever since the first automated de novo design techniques were conceived only 15 years ago, the computer-based design of hit and lead structure candidates has emerged as a complementary approach to high-throughput screening.  Although many challenges remain, de novo design supports drug discovery projects by generating novel pharmaceutically active agents with desired properties in a cost- and time-efficient manner.  In this review, we outline the various design concepts and highlight current developments in computer-based de novo design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWf2XFrvn_fbVg90H21EOLACvtfcHk0lihgdfLE181xg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXmvVOqtro%253D&md5=a30dbc58ed81e0b7fe3f7d41a668e9ac</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1038%2Fnrd1799&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd1799%26sid%3Dliteratum%253Aachs%26aulast%3DSchneider%26aufirst%3DG.%26aulast%3DFechner%26aufirst%3DU.%26atitle%3DComputer-based%2520de%2520novo%2520design%2520of%2520drug-like%2520molecules%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2005%26volume%3D4%26issue%3D8%26spage%3D649%26epage%3D663%26doi%3D10.1038%2Fnrd1799" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref70"><div class="reference"><strong class="refLabel"><a href="#ref70" class="rightTabRefNumLink">70</a></strong><div class="NLM_citation" id="rightTab-cit70"><span><span class="NLM_contrib-group">Firth, N. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atrash, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blagg, J.</span><span> </span><span class="NLM_article-title">MOARF, an integrated workflow for multiobjective optimization: implementation, synthesis, and biological evaluation</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x"> (</span><span class="NLM_issue">6</span><span class="NLM_x">) </span> <span class="NLM_fpage">1169</span><span class="NLM_x">–</span> <span class="NLM_lpage">1180</span><span class="refDoi"> DOI: 10.1021/acs.jcim.5b00073</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.5b00073" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVShtLvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=1169-1180&issue=6&author=N.+C.+Firthauthor=B.+Atrashauthor=N.+Brownauthor=J.+Blagg&title=MOARF%2C+an+integrated+workflow+for+multiobjective+optimization%3A+implementation%2C+synthesis%2C+and+biological+evaluation&doi=10.1021%2Facs.jcim.5b00073"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">MOARF, an Integrated Workflow for Multiobjective Optimization: Implementation, Synthesis, and Biological Evaluation</span></div><div class="casAuthors">Firth, Nicholas C.; Atrash, Butrus; Brown, Nathan; Blagg, Julian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1169-1180</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We describe the development and application of an integrated, multiobjective optimization workflow (MOARF) for directed medicinal chem. design.  This workflow couples a rule-based mol. fragmentation scheme (SynDiR) with a pharmacophore fingerprint-based fragment replacement algorithm (RATS) to broaden the scope of reconnection options considered in the generation of potential soln. structures.  Solns. are ranked by a multiobjective scoring algorithm comprising ligand-based (shape similarity) biochem. activity predictions as well as physicochem. property calcns.  Application of this iterative workflow to optimization of the CDK2 inhibitor Seliciclib (CYC202, R-roscovitine) generated soln. mols. in desired physicochem. property space.  Synthesis and exptl. evaluation of optimal soln. mols. demonstrates CDK2 biochem. activity and improved human metabolic stability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDa1EOPXpsI7Vg90H21EOLACvtfcHk0lihgdfLE181xg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVShtLvK&md5=0eed904cb0346cb4a521e30998fae979</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.5b00073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.5b00073%26sid%3Dliteratum%253Aachs%26aulast%3DFirth%26aufirst%3DN.%2BC.%26aulast%3DAtrash%26aufirst%3DB.%26aulast%3DBrown%26aufirst%3DN.%26aulast%3DBlagg%26aufirst%3DJ.%26atitle%3DMOARF%252C%2520an%2520integrated%2520workflow%2520for%2520multiobjective%2520optimization%253A%2520implementation%252C%2520synthesis%252C%2520and%2520biological%2520evaluation%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26volume%3D55%26issue%3D6%26spage%3D1169%26epage%3D1180%26doi%3D10.1021%2Facs.jcim.5b00073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref71"><div class="reference"><strong class="refLabel"><a href="#ref71" class="rightTabRefNumLink">71</a></strong><div class="NLM_citation" id="rightTab-cit71"><span><span class="NLM_contrib-group">Gabrielsson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dolgos, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gillberg, P.-G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bredberg, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benthem, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duker, G.</span><span> </span><span class="NLM_article-title">Early intergration of pharmacokinetic and dynamic reasoning is essential for optimal development of lead compounds: strategic considerations</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">358</span><span class="NLM_x">–</span> <span class="NLM_lpage">372</span><span class="refDoi"> DOI: 10.1016/j.drudis.2008.12.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=10.1016%2Fj.drudis.2008.12.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=19162219" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjvVWgsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2009&pages=358-372&author=J.+Gabrielssonauthor=H.+Dolgosauthor=P.-G.+Gillbergauthor=U.+Bredbergauthor=B.+Benthemauthor=G.+Duker&title=Early+intergration+of+pharmacokinetic+and+dynamic+reasoning+is+essential+for+optimal+development+of+lead+compounds%3A+strategic+considerations&doi=10.1016%2Fj.drudis.2008.12.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Early integration of pharmacokinetic and dynamic reasoning is essential for optimal development of lead compounds: strategic considerations</span></div><div class="casAuthors">Gabrielsson, Johan; Dolgos, Hugues; Gillberg, Per-Goeran; Bredberg, Ulf; Benthem, Bert; Duker, Goeran</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">7/8</span>),
    <span class="NLM_cas:pages">358-372</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The aims of this report are firstly to raise awareness among kineticists and pharmacologists as to why pharmacokinetic-pharmacodynamic (PKPD) integration is essential for target validation (TV), optimizing development of lead compds. (lead generation [LG] and lead optimization [LO]) and scaling these to human.  A related aim is to demonstrate strategic examples of PKPD collaborations that have improved the planning, execution and evaluation of expts. in primary and safety pharmacol.  Examples include design of TV studies, design and data pruning' of PKPD studies in LO, anal. of data with marginal and substantial temporal (time) differences between exposure and response, design of safety pharmacol. studies, assessment of safety margin and assessment of uncertainties in predictions of first dose in human.  PKPD awareness is vital if we are to attempt to relate preclin. results to the acute and long term consequences in humans.  The debate on whether preclin. findings can be translation to the human usage is still engaging scientists across industry, academia and regulatory bodies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXmTuZ4LRZ5bVg90H21EOLACvtfcHk0ljxyU3luZlOIQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjvVWgsbs%253D&md5=d52632817208dd41fde8e3355ed22bb4</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2008.12.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2008.12.011%26sid%3Dliteratum%253Aachs%26aulast%3DGabrielsson%26aufirst%3DJ.%26aulast%3DDolgos%26aufirst%3DH.%26aulast%3DGillberg%26aufirst%3DP.-G.%26aulast%3DBredberg%26aufirst%3DU.%26aulast%3DBenthem%26aufirst%3DB.%26aulast%3DDuker%26aufirst%3DG.%26atitle%3DEarly%2520intergration%2520of%2520pharmacokinetic%2520and%2520dynamic%2520reasoning%2520is%2520essential%2520for%2520optimal%2520development%2520of%2520lead%2520compounds%253A%2520strategic%2520considerations%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2009%26volume%3D14%26spage%3D358%26epage%3D372%26doi%3D10.1016%2Fj.drudis.2008.12.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.7b00487&amp;pbContext=%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2017.60.issue-22%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.7b00487%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.7b00487" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                4MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-b425991822a79ada59b3.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-fb88834b824b41b135ad.js"></script>

<script type="text/javascript" src="/wro/ke1q~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"667988070a21d93a","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.6.0","si":10}'></script>
</body>
</html>
