
<html lang="en"     class="pb-page"  data-request-id="aef71125-2e0f-4b14-a214-722ee7083338"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;article:article:10.1021/acs.jmedchem.0c01026;issue:issue:10.1021/jmcmar.2021.64.issue-2;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Discovery and Biological Evaluation of N-Methyl-pyrrolo[2,3-b]pyridine-5-carboxamide Derivatives as JAK1-Selective Inhibitors" /></meta><meta name="dc.Creator" content="Eunsun  Park" /></meta><meta name="dc.Creator" content="Sun Joo  Lee" /></meta><meta name="dc.Creator" content="Heegyum  Moon" /></meta><meta name="dc.Creator" content="Jongmi  Park" /></meta><meta name="dc.Creator" content="Hyeonho  Jeon" /></meta><meta name="dc.Creator" content="Ji Sun  Hwang" /></meta><meta name="dc.Creator" content="Hayoung  Hwang" /></meta><meta name="dc.Creator" content="Ki Bum  Hong" /></meta><meta name="dc.Creator" content="Seung-Hee  Han" /></meta><meta name="dc.Creator" content="Sun  Choi" /></meta><meta name="dc.Creator" content="Soosung  Kang" /></meta><meta name="dc.Description" content="Janus kinase 1 (JAK1) plays a key role in most cytokine-mediated inflammatory and autoimmune responses through JAK/STAT signaling; thus, JAK1 inhibition is a promising therapeutic strategy for seve..." /></meta><meta name="Description" content="Janus kinase 1 (JAK1) plays a key role in most cytokine-mediated inflammatory and autoimmune responses through JAK/STAT signaling; thus, JAK1 inhibition is a promising therapeutic strategy for seve..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="January 11, 2021" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c01026" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2021 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01026" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c01026" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01026" /></link>
        
    
    

<title>Discovery and Biological Evaluation of N-Methyl-pyrrolo[2,3-b]pyridine-5-carboxamide Derivatives as JAK1-Selective Inhibitors | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c01026" /></meta><meta property="og:title" content="Discovery and Biological Evaluation of N-Methyl-pyrrolo[2,3-b]pyridine-5-carboxamide Derivatives as JAK1-Selective Inhibitors" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01026/20210121/images/large/jm0c01026_0017.jpeg" /></meta><meta property="og:description" content="Janus kinase 1 (JAK1) plays a key role in most cytokine-mediated inflammatory and autoimmune responses through JAK/STAT signaling; thus, JAK1 inhibition is a promising therapeutic strategy for several diseases. Analysis of the binding modes of current JAK inhibitors to JAK isoforms allowed the design of N-alkyl-substituted 1-H-pyrrolo[2,3-b] pyridine carboxamide as a JAK1-selective scaffold, and the synthesis of various methyl amide derivatives provided 4-((cis-1-(4-chlorobenzyl)-2-methylpiperidin-4-yl)amino)-N-methyl-1H-pyrrolo[2,3-b]pyridine-5-carboxamide (31g) as a potent JAK1-selective inhibitor. In particular, the (S,S)-enantiomer of 31g (38a) exhibited excellent potency for JAK1 and selectivity over JAK2, JAK3, and TYK2. On investigating the effect of 31g on hepatic fibrosis, it was found that it reduces the proliferation and fibrogenic gene expression of TGF-β-induced hepatic stellate cells (HSCs). Specifically, 31g significantly inhibited TGF-β-induced migration of HSCs at 0.25 μM in wound-healing assays." /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta><meta name="twitter:description" content="Janus kinase 1 (JAK1) plays a key role in most cytokine-mediated inflammatory and autoimmune responses through JAK/STAT signaling; thus, JAK1 inhibition is a promising therapeutic strategy for several diseases. Analysis of the binding modes of current JAK inhibitors to JAK isoforms allowed the design of N-alkyl-substituted 1-H-pyrrolo[2,3-b] pyridine carboxamide as a JAK1-selective scaffold, and the synthesis of various methyl amide derivatives provided 4-((cis-1-(4-chlorobenzyl)-2-methylpiperidin-4-yl)amino)-N-methyl-1H-pyrrolo[2,3-b]pyridine-5-carboxamide (31g) as a potent JAK1-selective inhibitor. In particular, the (S,S)-enantiomer of 31g (38a) exhibited excellent potency for JAK1 and selectivity over JAK2, JAK3, and TYK2. On investigating the effect of 31g on hepatic fibrosis, it was found that it reduces the proliferation and fibrogenic gene expression of TGF-β-induced hepatic stellate cells (HSCs). Specifically, 31g significantly inhibited TGF-β-induced migration of HSCs at 0.25 μM in wound-healing assays." /></meta><meta name="twitter:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01026/20210121/images/large/jm0c01026_0017.jpeg" /></meta><meta name="twitter:title" content="Discovery and Biological Evaluation of N-Methyl-pyrrolo[2,3-b]pyridine-5-carboxamide Derivatives as JAK1-Selective Inhibitors" /></meta>
        
            <meta property="og:type" content="Article" /></meta>
        

        
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c01026"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/ke1q~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-6ee29527b3d8ff00a106.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01026">
            
        





































    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-12/achre4.2021.54.issue-12/20210615/achre4.2021.54.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-6/amrcda.2021.2.issue-6/20210625/amrcda.2021.2.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-6/aabmcb.2021.4.issue-6/20210621/aabmcb.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-6/aaembp.2021.3.issue-6/20210622/aaembp.2021.3.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-6/aaemcq.2021.4.issue-6/20210628/aaemcq.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-25/aamick.2021.13.issue-25/20210630/aamick.2021.13.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-6/aanmf6.2021.4.issue-6/20210625/aanmf6.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-6/aapmcd.2021.3.issue-6/20210611/aapmcd.2021.3.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-6/abseba.2021.7.issue-6/20210614/abseba.2021.7.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-12/accacs.2021.11.issue-12/20210618/accacs.2021.11.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-6/acscii.2021.7.issue-6/20210623/acscii.2021.7.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-6/acbcct.2021.16.issue-6/20210618/acbcct.2021.16.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-3/achsc5.2021.28.issue-3/20210524/achsc5.2021.28.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-12/acncdm.2021.12.issue-12/20210616/acncdm.2021.12.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-6/aesccq.2021.5.issue-6/20210617/aesccq.2021.5.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-6/aelccp.2021.6.issue-6/20210611/aelccp.2021.6.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-6/aeecco.2021.1.issue-6/20210611/aeecco.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-6/aewcaa.2021.1.issue-6/20210611/aewcaa.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-5/afsthl.2021.1.issue-5/20210618/afsthl.2021.1.issue-5.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-6/aidcbc.2021.7.issue-6/20210611/aidcbc.2021.7.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-6/amlccd.2021.10.issue-6/20210615/amlccd.2021.10.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-6/amlcef.2021.3.issue-6/20210607/amlcef.2021.3.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-6/amclct.2021.12.issue-6/20210610/amclct.2021.12.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-6/ancac3.2021.15.issue-6/20210622/ancac3.2021.15.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-25/acsodf.2021.6.issue-25/20210629/acsodf.2021.6.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-6/apchd5.2021.8.issue-6/20210616/apchd5.2021.8.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-6/ascefj.2021.6.issue-6/20210625/ascefj.2021.6.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-25/ascecg.2021.9.issue-25/20210628/ascecg.2021.9.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-6/asbcd6.2021.10.issue-6/20210618/asbcd6.2021.10.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-25/ancham.2021.93.issue-25/20210629/ancham.2021.93.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-25/bichaw.2021.60.issue-25/20210629/bichaw.2021.60.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-6/bcches.2021.32.issue-6/20210616/bcches.2021.32.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-6/bomaf6.2021.22.issue-6/20210614/bomaf6.2021.22.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-24/cgeabj.2021.99.issue-24/20210628-01/cgeabj.2021.99.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-6/crtoec.2021.34.issue-6/20210621/crtoec.2021.34.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-12/chreay.2021.121.issue-12/20210623/chreay.2021.121.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-12/cmatex.2021.33.issue-12/20210622/cmatex.2021.33.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-6/cgdefu.2021.21.issue-6/20210602/cgdefu.2021.21.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-12/enfuem.2021.35.issue-12/20210617/enfuem.2021.35.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-12/esthag.2021.55.issue-12/20210615/esthag.2021.55.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-6/estlcu.2021.8.issue-6/20210608/estlcu.2021.8.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-25/iecred.2021.60.issue-25/20210630/iecred.2021.60.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-12/inocaj.2021.60.issue-12/20210621/inocaj.2021.60.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-6/jaaucr.2021.1.issue-6/20210628/jaaucr.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-25/jafcau.2021.69.issue-25/20210630/jafcau.2021.69.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-6/jceaax.2021.66.issue-6/20210610/jceaax.2021.66.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-6/jceda8.2021.98.issue-6/20210608/jceda8.2021.98.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-6/jcisd8.2021.61.issue-6/20210628/jcisd8.2021.61.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-6/jctcce.2021.17.issue-6/20210608/jctcce.2021.17.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/jmcmar.2021.64.issue-12/20210624/jmcmar.2021.64.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-6/jnprdf.2021.84.issue-6/20210625/jnprdf.2021.84.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-12/joceah.2021.86.issue-12/20210618/joceah.2021.86.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-24/jpcafh.2021.125.issue-24/20210624/jpcafh.2021.125.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-24/jpcbfk.2021.125.issue-24/20210624/jpcbfk.2021.125.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-24/jpccck.2021.125.issue-24/20210624/jpccck.2021.125.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-24/jpclcd.2021.12.issue-24/20210624/jpclcd.2021.12.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-6/jprobs.2021.20.issue-6/20210604/jprobs.2021.20.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-25/jacsat.2021.143.issue-25/20210630/jacsat.2021.143.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-6/jamsef.2021.32.issue-6/20210602/jamsef.2021.32.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-25/langd5.2021.37.issue-25/20210629/langd5.2021.37.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-12/mamobx.2021.54.issue-12/20210622/mamobx.2021.54.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-6/mpohbp.2021.18.issue-6/20210607/mpohbp.2021.18.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-12/nalefd.2021.21.issue-12/20210623/nalefd.2021.21.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-12/orlef7.2021.23.issue-12/20210618/orlef7.2021.23.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-6/oprdfk.2021.25.issue-6/20210618/oprdfk.2021.25.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-12/orgnd7.2021.40.issue-12/20210628/orgnd7.2021.40.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c01026&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c01026&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c01026&amp;href=/doi/10.1021/acs.jmedchem.0c01026" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2021</span><span class="cit-fg-volume">, 64</span><span class="cit-fg-issue">, 2</span><span class="cit-fg-pageRange">, 958-979</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/64/2" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c01133" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c01450" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery and Biological Evaluation of <i>N</i>-Methyl-pyrrolo[2,3-<i>b</i>]pyridine-5-carboxamide Derivatives as JAK1-Selective Inhibitors</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Eunsun Park</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Eunsun Park</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">College of Pharmacy and Graduate School of Pharmaceutical Sciences, Ewha Womans University, Seoul 03760, Republic of Korea</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Eunsun++Park">Eunsun Park</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Sun Joo Lee</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sun Joo Lee</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">New Drug Development Center, Daegu-Gyeongbuk Medical Innovation Foundation (DGMIF), Daegu 41061, Republic of Korea</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Sun+Joo++Lee">Sun Joo Lee</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Heegyum Moon</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Heegyum Moon</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">New Drug Development Center, Daegu-Gyeongbuk Medical Innovation Foundation (DGMIF), Daegu 41061, Republic of Korea</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Heegyum++Moon">Heegyum Moon</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jongmi Park</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jongmi Park</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">College of Pharmacy and Graduate School of Pharmaceutical Sciences, Ewha Womans University, Seoul 03760, Republic of Korea</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jongmi++Park">Jongmi Park</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Hyeonho Jeon</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hyeonho Jeon</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">New Drug Development Center, Daegu-Gyeongbuk Medical Innovation Foundation (DGMIF), Daegu 41061, Republic of Korea</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hyeonho++Jeon">Hyeonho Jeon</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ji Sun Hwang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ji Sun Hwang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">New Drug Development Center, Daegu-Gyeongbuk Medical Innovation Foundation (DGMIF), Daegu 41061, Republic of Korea</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ji+Sun++Hwang">Ji Sun Hwang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Hayoung Hwang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hayoung Hwang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">New Drug Development Center, Daegu-Gyeongbuk Medical Innovation Foundation (DGMIF), Daegu 41061, Republic of Korea</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hayoung++Hwang">Hayoung Hwang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ki Bum Hong</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ki Bum Hong</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">New Drug Development Center, Daegu-Gyeongbuk Medical Innovation Foundation (DGMIF), Daegu 41061, Republic of Korea</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ki+Bum++Hong">Ki Bum Hong</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-4853-2263" title="Orcid link">http://orcid.org/0000-0002-4853-2263</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Seung-Hee Han</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Seung-Hee Han</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Central Research Laboratory, KOREA PHARMA Co. Ltd, jeyakgongdan 3-gil, Hyangnam-eup, Hwaseong-si, Gyeonggi-do 18622, Republic of Korea</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Seung-Hee++Han">Seung-Hee Han</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Sun Choi</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sun Choi</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">College of Pharmacy and Graduate School of Pharmaceutical Sciences, Ewha Womans University, Seoul 03760, Republic of Korea</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Sun++Choi">Sun Choi</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-7669-7954" title="Orcid link">http://orcid.org/0000-0002-7669-7954</a></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Soosung Kang</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Soosung Kang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">College of Pharmacy and Graduate School of Pharmaceutical Sciences, Ewha Womans University, Seoul 03760, Republic of Korea</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#ddaeaeb6bcb3ba9db8aab5bcf3bcbef3b6af"><span class="__cf_email__" data-cfemail="91e2e2faf0fff6d1f4e6f9f0bff0f2bffae3">[email protected]</span></a>. Phone: 82-2-3277-6619.</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Soosung++Kang">Soosung Kang</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-7016-2417" title="Orcid link">http://orcid.org/0000-0001-7016-2417</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01026&amp;href=/doi/10.1021%2Facs.jmedchem.0c01026" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2021</span></span><span class="cit-volume">, 64</span><span class="cit-issue">, 2</span><span class="cit-pageRange">, 958–979</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">January 11, 2021</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>16 June 2020</li><li><span class="item_label"><b>Published</b> online</span>11 January 2021</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 28 January 2021</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c01026" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01026</a></div><div class="article_header-article-copyright"><strong>Copyright © 2021 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D958%26pageCount%3D22%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DEunsun%2BPark%252C%2BSun%2BJoo%2BLee%252C%2BHeegyum%2BMoon%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D64%26issueNum%3D2%26contentID%3Dacs.jmedchem.0c01026%26title%3DDiscovery%2Band%2BBiological%2BEvaluation%2Bof%2BN-Methyl-pyrrolo%255B2%252C3-b%255Dpyridine-5-carboxamide%2BDerivatives%2Bas%2BJAK1-Selective%2BInhibitors%26numPages%3D22%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D979%26publicationDate%3DJanuary%2B2021">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c01026"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1766</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val">-</div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c01026" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery and Biological Evaluation of N-Methyl-pyrrolo[2,3-b]pyridine-5-carboxamide Derivatives as JAK1-Selective Inhibitors&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Eunsun&quot;,&quot;last_name&quot;:&quot;Park&quot;},{&quot;first_name&quot;:&quot;Sun&quot;,&quot;last_name&quot;:&quot;Joo Lee&quot;},{&quot;first_name&quot;:&quot;Heegyum&quot;,&quot;last_name&quot;:&quot;Moon&quot;},{&quot;first_name&quot;:&quot;Jongmi&quot;,&quot;last_name&quot;:&quot;Park&quot;},{&quot;first_name&quot;:&quot;Hyeonho&quot;,&quot;last_name&quot;:&quot;Jeon&quot;},{&quot;first_name&quot;:&quot;Ji&quot;,&quot;last_name&quot;:&quot;Sun Hwang&quot;},{&quot;first_name&quot;:&quot;Hayoung&quot;,&quot;last_name&quot;:&quot;Hwang&quot;},{&quot;first_name&quot;:&quot;Ki&quot;,&quot;last_name&quot;:&quot;Bum Hong&quot;},{&quot;first_name&quot;:&quot;Seung-Hee&quot;,&quot;last_name&quot;:&quot;Han&quot;},{&quot;first_name&quot;:&quot;Sun&quot;,&quot;last_name&quot;:&quot;Choi&quot;},{&quot;first_name&quot;:&quot;Soosung&quot;,&quot;last_name&quot;:&quot;Kang&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2021&quot;,&quot;month&quot;:&quot;01&quot;,&quot;day&quot;:&quot;11&quot;,&quot;issue&quot;:&quot;2&quot;,&quot;volume&quot;:&quot;64&quot;,&quot;pages&quot;:&quot;958-979&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c01026&quot;},&quot;abstract&quot;:&quot;Janus kinase 1 (JAK1) plays a key role in most cytokine-mediated inflammatory and autoimmune responses through JAK/STAT signaling; thus, JAK1 inhibition is a promising therapeutic strategy for several diseases. Analysis of the binding modes of current JAK inhibitors to JAK isoforms allowed the design of N-alkyl-substituted 1-H-pyrrolo[2,3-b] pyridine carboxamide as a JAK1-selective scaffold, and the synthesis of various methyl amide derivatives provided 4-((cis-1-(4-chlorobenzyl)-2-methylpiperidin-4-yl)amino)-N-methyl-1H-pyrrolo[2,3-b]pyridine-5-carboxamide (31g) as a potent JAK1-selective inhibitor. In particular, the (S,S)-enantiomer of 31g (38a) exhibited excellent potency for JAK1 and selectivity over JAK2, JAK3, and TYK2. On investigating the effect of 31g on hepatic fibrosis, it was found that it reduces the proliferation and fibrogenic gene expression of TGF-β-induced hepatic stellate cells (HSCs). Specifically, 31g significantly inhibited TGF-β-induced migration of HSCs at 0.25 μM in wound-healing ass&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01026&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01026" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01026&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01026" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01026&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01026" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01026&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01026&amp;href=/doi/10.1021/acs.jmedchem.0c01026" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c01026" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c01026" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (5 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c01026%26sid%3Dliteratum%253Aachs%26pmid%3D33428419%26genre%3Darticle%26aulast%3DPark%26date%3D2021%26atitle%3DDiscovery%2Band%2BBiological%2BEvaluation%2Bof%2BN-Methyl-pyrrolo%255B2%252C3-b%255Dpyridine-5-carboxamide%2BDerivatives%2Bas%2BJAK1-Selective%2BInhibitors%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D64%26issue%3D2%26spage%3D958%26epage%3D979%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=290658" title="Amides">Amides</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=292349" title="Alkyls">Alkyls</a>,</li><li><a href="/action/doSearch?ConceptID=291711" title="Selectivity">Selectivity</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/64/2" title="Go to Volume 64, Issue 2"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/jmcmar.2021.64.issue-2/20210128/jmcmar.2021.64.issue-2.largecover.jpg" alt="Go to Volume 64, Issue 2"/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01026/20210121/images/medium/jm0c01026_0017.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01026/20210121/images/large/jm0c01026_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01026&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Janus kinase 1 (JAK1) plays a key role in most cytokine-mediated inflammatory and autoimmune responses through JAK/STAT signaling; thus, JAK1 inhibition is a promising therapeutic strategy for several diseases. Analysis of the binding modes of current JAK inhibitors to JAK isoforms allowed the design of <i>N</i>-alkyl-substituted 1-<i>H</i>-pyrrolo[2,3-<i>b</i>] pyridine carboxamide as a JAK1-selective scaffold, and the synthesis of various methyl amide derivatives provided 4-((<i>cis</i>-1-(4-chlorobenzyl)-2-methylpiperidin-4-yl)amino)-<i>N</i>-methyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-5-carboxamide (<b>31g</b>) as a potent JAK1-selective inhibitor. In particular, the (<i>S</i>,<i>S)</i>-enantiomer of <b>31g</b> (<b>38a</b>) exhibited excellent potency for JAK1 and selectivity over JAK2, JAK3, and TYK2. On investigating the effect of <b>31g</b> on hepatic fibrosis, it was found that it reduces the proliferation and fibrogenic gene expression of TGF-β-induced hepatic stellate cells (HSCs). Specifically, <b>31g</b> significantly inhibited TGF-β-induced migration of HSCs at 0.25 μM in wound-healing assays.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_40871" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_40871" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Janus kinases (JAKs), consisting of four homology members (JAK1, JAK2, JAK3, and TYK2), are a family of tyrosine kinases that transduce cytokine-mediated signals through the JAK/STAT pathway.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> The JAK/STAT signaling pathway controls inflammation-related gene expression through cytokines such as IFN and gp130.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> Each cytokine binds to its specific receptor, leading to receptor dimerization and subsequent phosphorylation of JAK dimers. STATs are the phosphorylated downstream of JAKs, and these activated STAT dimers induce proinflammatory gene transcription inside the nucleus.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> As JAK/STAT signaling is generally involved in inflammation and autoimmune responses,<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> researchers have actively developed JAK inhibitors for the treatment of inflammation and autoimmune-related diseases such as inflammatory bowel disease (IBD),<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> rheumatoid arthritis (RA),<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> and atopic dermatitis.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> Recent studies have indicated the potential of JAK inhibitors in the treatment of hepatic fibrosis.<a onclick="showRef(event, 'ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref8 ref9">(8,9)</a> The connective tissue growth factor (CTGF) is constantly expressed in activated hepatic stellate cells (HSCs) and modulates extracellular matrix production downstream of TGF-β in the fibrotic liver.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> CTGF production follows the integration of canonical SMAD3 and STAT3 signaling. The SMAD-independent JAK1/STAT pathway is essential for the expression of TGF-β target genes in HSCs; thus, the cooperation between the JAK1-STAT3 and SMAD pathways is critical for the functions of TGF-β in liver fibrosis.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a></div><div class="NLM_p">Although several JAK inhibitors such as tofacitinib (<b>1</b>), baricitinib (<b>2</b>), and ruxolitinib (<b>3</b>)(<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) are currently approved by the FDA or in clinical trials,<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> these inhibitors are pan-JAK inhibitors or dual inhibitors.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> Pan-inhibitors can have various side effects.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> The pan-JAK inhibitor tofacitinib is found to be associated with anemia and an increased risk of infections,<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> in some human patients, which may be attributed to its ability to induce lymphopenia<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> via JAK2 inhibition. JAK2 homodimers are only involved initially in the signaling pathway of erythropoietin (EPO), a red blood cell growth factor. If pan-JAK inhibitors that block the EPO pathway are used for treating inflammatory and autoimmune diseases, there may be side effects such as the inhibition of hematopoiesis and myeloid differentiation.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> Various cytokine receptors are differentially associated with each JAK homodimer or heterodimer; thus, the selective inhibition of a single-JAK member can allow specific control of biological functions related to that JAK without affecting other JAK-dependent signals. Among the JAK family members, JAK1 is the only isoform that can form a heterodimer with all three JAKs.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> JAK1 regulates the function of T-cells and B-cells in signaling triggered by cytokines such as γc (IL-2, IL-4, IL-7, IL-9, and IL-21),<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> gp130 (IL-6, IL-11, LIF, and OSM),<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> IRF, and IL-10-like cytokines.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> The dysfunction of JAK1 may lead to severe inflammatory and autoimmune diseases.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> Therefore, many researchers have recently focused on the development of JAK1-selective inhibitors,<a onclick="showRef(event, 'ref23 ref24 ref25'); return false;" href="javascript:void(0);" class="ref ref23 ref24 ref25">(23−25)</a> including filgotinib (<b>4</b>)<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> and upadacitinib (<b>5</b>)<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>).</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01026/20210121/images/medium/jm0c01026_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01026/20210121/images/large/jm0c01026_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Representative JAK inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01026/20210121/images/large/jm0c01026_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01026&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">JAKs are composed of kinase, pseudo-kinase, SH2, and FERM domains, ranging from 120 to 140 kDa. The kinase domain contains an ATP-binding site that regulates the activation of JAK via phosphorylation and acts as the main drug-binding site.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> More specifically, the catalytic site of the kinase domain is composed of the hinge region, nontransferable phosphate-glycine-rich P-loop, and A-loop containing the DFG (Asp-Phe-Gly) motif in the ATP-binding site.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> The hinge region exhibits a bidentate interaction with the adenosine of ATP, which is an important drug target for the control of kinase activity. The design of a JAK-selective inhibitor is challenging because only a few residues are different in the ATP-binding sites of the four JAKs. To develop JAK1-selective inhibitors, we first determined the key differences in the residues of JAK proteins in the 3D structural view and referred to well-known pan-JAK inhibitors and JAK1 inhibitors for the design of JAK1-specific inhibitors. After identifying <i>N</i>-methyl-pyrrolo[2,3-<i>b</i>]pyridine-5-carboxamide as a JAK1-selective scaffold, we optimized the hit compound and developed a highly potent and selective JAK1 inhibitor using enzyme- and cell-based assays. After investigating the basic <i>in vitro</i> ADME properties and kinase panel selectivity, the inhibitor was evaluated for inhibiting hepatic fibrosis.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_41554" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_41554" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Design of JAK1-Selective Inhibitors</h3><div class="NLM_p">To develop JAK1-selective inhibitors, we referred to the crystal structures of the well-known JAK1 inhibitors, tofacitinib (<b>1</b>) and filgotinib (<b>4</b>), which have characteristic binding modes in the ATP-binding site in JAK JH1 domains. As a pan-inhibitor of JAKs, tofacitinib has co-crystal structures with all four JAK family proteins (JAK1: 3EYG, JAK2: 3FUP, JAK3: 3LXK, TYK2: 3LXN).<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> The JAK1-selective inhibitor filgotinib has a co-crystal structure with JAK2 (5UT5).<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> Although the protein structure overlay showed that the drug-binding sites of all four JAK proteins are almost identical, we found that several amino acids near the drug-binding site are different from each other (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>a). As the most distinguishing residue, a hinge residue (Phe958) of JAK1 is displaced to Tyr at JAK2, JAK3, and TYK2. The hydroxyl group of Tyr in the hinge-binding region of JAKs, except JAK1, could allow a ligand-specific polar interaction. We can utilize this relatively nonpolar environment near the hinge-binding region of JAK1 for selective inhibition. In comparison with tofacitinib, the characteristic functional group of filgotinib, a JAK1-selective inhibitor, is the cyclopropyl amide in the hinge-binding area (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>b). The cyclopropyl amide of filgotinib occupies the relatively nonpolar hinge-binding site to form a stable structure, thus contributing to JAK1 selectivity.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01026/20210121/images/medium/jm0c01026_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01026/20210121/images/large/jm0c01026_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (a) Alignment of ATP-binding site of tofacitinib (cyan)–JAK1 (gray) complex (3EYG) with filgotinib (purple)–JAK2 (light pink) complex (3FUP), highlighting different residues. (b) Schematic alignment of docking result of filgotinib with JAK1 and overlay with tofacitinib. (c) Three chosen areas for modification on the suggested new molecule.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01026/20210121/images/large/jm0c01026_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01026&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">The 3-oxo-3-(piperidin-1-yl)propanenitrile of tofacitinib and 4-benzylthiomorpholine 1,1-dioxide of filgotinib stretch toward the middle of the P- and A-loops. It was demonstrated that introducing small hydrophobic structures improved the potency for JAKs by positioning between the P- and A-loops.<a onclick="showRef(event, 'ref32 ref33'); return false;" href="javascript:void(0);" class="ref ref32 ref33">(32,33)</a> As filgotinib is selective for JAK1 over JAK2, its conformation may serve as a more effective guideline for the design of JAK1-selective inhibitors. We decided to adapt the amide functional group of filgotinib to the pyrrolo[2,3-<i>b</i>]pyridine scaffold of tofacitinib with the hybridization of the lipophilic tail of tofacitinib to occupy the hydrophobic binding pocket near the P-loop of JAK1 and synthesize various <i>N</i>-alkyl-pyrrolo[2,3-<i>b</i>]pyridine-5-carboxamide derivatives. Although a research group used a similar hinge binder to obtain a JAK3-selective inhibitor,<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> they did not consider both the hinge region and P-loop hydrophobic pocket, which are considered as hot spots for JAK1 selectivity and potency.<a onclick="showRef(event, 'ref26 ref27'); return false;" href="javascript:void(0);" class="ref ref26 ref27">(26,27)</a></div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Chemistry</h3><div class="NLM_p">As shown in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>, the hinge binding building blocks <i>N</i>-alkyl-substituted 1-<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine carboxamides (<b>7a–7d</b>) were synthesized by the carbonyldiimidazole-mediated amide coupling of 4-chloro-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-5-carboxylic acid. The other building blocks (<b>13a–13g</b>, <b>14a–14e</b>, <b>15a–15e</b>, and <b>16a–16c</b>) for the hydrophobic pocket were prepared from various Boc-protected amines by <i>N</i>-benzylation using K<sub>2</sub>CO<sub>3</sub> as the base or purchased from commercial sources. Microwave-assisted nucleophilic aromatic substitution at the C4-position of <b>7a–7d</b> with various amines prepared <i>in situ</i> via HCl/MeOH treatment of Boc-protected intermediates <b>13a–13g</b>, <b>14a–14e</b>, <b>15a–15e</b>, and <b>16a–16c</b> was performed to obtain various <i>N</i>-alkyl-pyrrolo[2,3-<i>b</i>]pyridine-5-carboxamide derivatives <b>17a–17m</b>, <b>18a–18e</b>, <b>19a–19e</b>, and <b>20a–20c</b> (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>). Alternatively, compounds <b>7a–7d</b> were subjected to a nucleophilic aromatic substitution reaction with various Boc-protected amines to produce intermediates <b>25</b>, <b>26</b>, <b>27a–27c</b>, <b>28a</b>, and <b>28b</b> (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>). <i>In situ</i> Boc-deprotection of <b>25</b>, <b>26</b>, <b>27a–27c</b>, <b>28a</b>, and <b>28b</b> with HCl/MeOH and subsequent reductive amination with various benzaldehydes afforded the desired final compounds <b>20d–20f</b>, <b>30a–30f</b>, and <b>31a–31l</b>, and <b>32a–32c</b> at a high yield.</div><figure id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01026/20210121/images/medium/jm0c01026_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01026/20210121/images/large/jm0c01026_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Syntheses of Intermediates <b>7a–7d</b>, <b>13a–13g</b>, <b>14a–14e</b>, <b>15a–15e</b>, and <b>16a–16c</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01026/20210121/images/large/jm0c01026_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01026&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="sch2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01026/20210121/images/medium/jm0c01026_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01026/20210121/images/large/jm0c01026_0008.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Syntheses of Products <b>17a–17m</b>, <b>18a–18e</b>, <b>19a–19e</b>, and <b>20a–20c</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01026/20210121/images/large/jm0c01026_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01026&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="sch3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01026/20210121/images/medium/jm0c01026_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01026/20210121/images/large/jm0c01026_0009.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Syntheses of Products <b>20d</b>, <b>20e</b>, <b>30a–30f</b>, <b>31a–31l</b>, and <b>32a–32c</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01026/20210121/images/large/jm0c01026_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01026&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Additionally, each (<i>R</i>,<i>R</i>)- and (<i>S</i>,<i>S</i>)-enantiomers of <b>31g</b> were synthesized from optically pure <b>36a</b> and <b>39b</b> prepared via reductive amination of each enantiomer of <i>N</i>-Boc-2-methyl-4-piperidone with (<i>S</i>)- or (<i>R</i>)-1-phenylethan-1-amine, following hydrogenolytic debenzylation (<a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>). Then, <b>36a</b> and <b>36b</b> underwent a nucleophilic aromatic substitution reaction with <b>7b</b> to produce <b>37a</b> and <b>37b</b>. Boc-deprotection and subsequent reductive amination with 4-chlorobenzaldehyde afforded the desired optically pure (2<i>S</i>,4<i>S</i>)-isomer of <b>31g</b> (<b>38a</b>) and (2<i>R</i>, 4<i>R</i>)-isomer of <b>31g</b> (<b>38b</b>)(<a class="ref internalNav" href="#sch5" aria-label="Scheme 5">Scheme 5</a>).</div><figure id="sch4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01026/20210121/images/medium/jm0c01026_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01026/20210121/images/large/jm0c01026_0010.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Enantioselective Syntheses of 4-Amino-2-methylpiperidine Intermediates <b>36a</b> and <b>36b</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01026/20210121/images/large/jm0c01026_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01026&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="sch5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01026/20210121/images/medium/jm0c01026_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01026/20210121/images/large/jm0c01026_0011.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Syntheses of <b>31g</b> Enantiomers <b>38a</b> and <b>38b</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01026/20210121/images/large/jm0c01026_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01026&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Structure–Activity Relationship</h3><div class="NLM_p">We clarified the appropriate polarity, size, and shape of inhibitors in A, B, and C by conducting structure–activity relationship (SAR) studies (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>c). First, we identified the most suitable R<sub>1</sub> carboxamide (A) by introducing primary, methyl, cyclopropyl, and cyclopentyl amides after fixing B to piperidine and C to benzyl or 3-methoxybenzyl (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). As mentioned in the introduction, this region was regarded as a hot spot for JAK1-selective inhibition via interaction with the JAK1-specific residue Phe958. Compounds <b>17c</b> and <b>17e</b> containing methyl (R<sub>1</sub>) were potent inhibitors for JAK1 (IC<sub>50</sub> = 65 nM and 18 nM, respectively). Notably, these methyl amide compounds demonstrated >9-fold selectivity over JAK2, JAK3, and TYK2. However, compound <b>17a</b>, a primary amide, exhibited activity similar to that of methyl amide <b>17c</b> in JAK1 inhibition but was reported to have a similar binding affinity for JAK2 or even improved binding affinity for JAK3.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a></div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. IC<sub>50</sub>s of <b>17a–17m</b><a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01026/20210121/images/medium/jm0c01026_0012.gif" alt="" id="gr11" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01026/20210121/images/medium/jm0c01026_0013.gif" alt="" id="gr12" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">All JAK family <i>in vitro</i> kinase assays were performed using LanthaScreen.</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">IC<sub>50</sub> values are the average of duplicate experiments.</p></div><div class="footnote" id="t1fn3"><sup><sup>c</sup></sup><p class="last">Values are calculated using GraphPad Prism 6.</p></div><div class="footnote" id="t1fn4"><sup><sup>d</sup></sup><p class="last">The data came from a reference.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a></p></div></div><div></div></div><div class="NLM_p">On comparing methyl amides (<b>17c</b>, <b>17e</b>) with cyclopropyl amides (<b>17i</b>, <b>17j</b>) and cyclopentyl amides (<b>17 k</b>, <b>17l</b>), the bulkier amides tended to have reduced binding affinity for JAK1. <b>17i</b>, which contains cyclopropyl (R<sub>1</sub>), had an IC<sub>50</sub> value of 114 nM against JAK1 with only 2-fold selectivity over JAK2, and <b>17k</b>, which contains cyclopentyl (R<sub>1</sub>), had a much weaker binding affinity for JAK1 (>10,000 nM). These results suggest that the amide may be a JAK isoform selectivity switch due to differential interactions in the hinge region of JAKs and that methyl amide may be the most effective in improving the potency and selectivity for JAK1.</div><div class="NLM_p">To determine the influence of the size and shape of cyclic amine groups (B), piperidin-4-ylmethyl (<b>18a–18e</b>), pyrrolidine (<b>19a–19e</b>), 8-azabicyclo[3.2.1]octane (<b>20a–20f</b>), azepane (<b>30a–30f</b>), and methyl- (<b>31a–31l</b>) or fluorine- (<b>32a–32c</b>)-substituted piperidine derivatives were applied to B with a MeO-, Cl-, and CN-substituted benzyl ring (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Tables <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>). When the piperidin-4-ylmethyl derivatives <b>18a–18e</b> and 3-aminopyrrolidine derivatives <b>19a–19e</b> displayed various IC<sub>50</sub> for JAK1 depending on the benzyl side chains (60–827 nM), both 8-azabicyclo[3.2.1]octan-3-amine derivatives <b>20a–20g</b> and 4-aminoazepane derivatives 3<b>0a–30e</b>, which are bulkier than other scaffolds, exhibited slightly improved inhibitory activity toward JAK1 with the modest isoform selectivity in general.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. IC<sub>50</sub>s of <b>18a–18e</b>, <b>19a–19e</b>, <b>20a–20e</b>, and <b>30a–30f</b><a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01026/20210121/images/medium/jm0c01026_0014.gif" alt="" id="gr13" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="4" align="center" char=".">IC<sub>50</sub> (nM)<a class="ref internalNav" href="#t2fn2" aria-label="b">b</a><a class="ref internalNav" href="#t2fn3" aria-label="c">c</a></th><th class="rowsep1 colsep0" colspan="3" align="center" char=".">IC<sub>50</sub> ratio</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center">R</th><th class="colsep0 rowsep0" align="center" char=".">JAK1</th><th class="colsep0 rowsep0" align="center" char=".">JAK2</th><th class="colsep0 rowsep0" align="center" char=".">JAK3</th><th class="colsep0 rowsep0" align="center" char=".">TYK2</th><th class="colsep0 rowsep0" align="center" char=".">JAK2/JAK1</th><th class="colsep0 rowsep0" align="center" char=".">JAK3/JAK1</th><th class="colsep0 rowsep0" align="center" char=".">TYK2/JAK1</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18a</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="char" char=".">215</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18b</b></td><td class="colsep0 rowsep0" align="left">3-Cl</td><td class="colsep0 rowsep0" align="char" char=".">243</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18c</b></td><td class="colsep0 rowsep0" align="left">4-Cl</td><td class="colsep0 rowsep0" align="char" char=".">72.2</td><td class="colsep0 rowsep0" align="char" char=".">712</td><td class="colsep0 rowsep0" align="char" char=".">426</td><td class="colsep0 rowsep0" align="char" char=".">48.9</td><td class="colsep0 rowsep0" align="char" char=".">9.9</td><td class="colsep0 rowsep0" align="char" char=".">5.9</td><td class="colsep0 rowsep0" align="char" char=".">0.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18d</b></td><td class="colsep0 rowsep0" align="left">3-OMe</td><td class="colsep0 rowsep0" align="char" char=".">132</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18e</b></td><td class="colsep0 rowsep0" align="left">4-OMe</td><td class="colsep0 rowsep0" align="char" char=".">93.4</td><td class="colsep0 rowsep0" align="char" char=".">1080</td><td class="colsep0 rowsep0" align="char" char=".">576</td><td class="colsep0 rowsep0" align="char" char=".">200</td><td class="colsep0 rowsep0" align="char" char=".">11.6</td><td class="colsep0 rowsep0" align="char" char=".">6.1</td><td class="colsep0 rowsep0" align="char" char=".">2.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>19a</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="char" char=".">400</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>19b</b></td><td class="colsep0 rowsep0" align="left">3-Cl</td><td class="colsep0 rowsep0" align="char" char=".">60.0</td><td class="colsep0 rowsep0" align="char" char=".">514</td><td class="colsep0 rowsep0" align="char" char=".">200</td><td class="colsep0 rowsep0" align="char" char=".">597</td><td class="colsep0 rowsep0" align="char" char=".">8.6</td><td class="colsep0 rowsep0" align="char" char=".">3.3</td><td class="colsep0 rowsep0" align="char" char=".">10.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>19c</b></td><td class="colsep0 rowsep0" align="left">4-Cl</td><td class="colsep0 rowsep0" align="char" char=".">80.1</td><td class="colsep0 rowsep0" align="char" char=".">503.8</td><td class="colsep0 rowsep0" align="char" char=".">375.6</td><td class="colsep0 rowsep0" align="char" char=".">657.7</td><td class="colsep0 rowsep0" align="char" char=".">6.3</td><td class="colsep0 rowsep0" align="char" char=".">4.7</td><td class="colsep0 rowsep0" align="char" char=".">8.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>19d</b></td><td class="colsep0 rowsep0" align="left">3-OMe</td><td class="colsep0 rowsep0" align="char" char=".">827</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>19e</b></td><td class="colsep0 rowsep0" align="left">4-OMe</td><td class="colsep0 rowsep0" align="char" char=".">149</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20a</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="char" char=".">74.9</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20b</b></td><td class="colsep0 rowsep0" align="left">3-Cl</td><td class="colsep0 rowsep0" align="char" char=".">40.8</td><td class="colsep0 rowsep0" align="char" char=".">153</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char=".">292</td><td class="colsep0 rowsep0" align="char" char=".">3.8</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20c</b></td><td class="colsep0 rowsep0" align="left">4-Cl</td><td class="colsep0 rowsep0" align="char" char=".">17.4</td><td class="colsep0 rowsep0" align="char" char=".">2590</td><td class="colsep0 rowsep0" align="char" char=".">47.5</td><td class="colsep0 rowsep0" align="char" char=".">134</td><td class="colsep0 rowsep0" align="char" char=".">148</td><td class="colsep0 rowsep0" align="char" char=".">2.7</td><td class="colsep0 rowsep0" align="char" char=".">7.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20d</b></td><td class="colsep0 rowsep0" align="left">3-OMe</td><td class="colsep0 rowsep0" align="char" char=".">46.1</td><td class="colsep0 rowsep0" align="char" char=".">89.9</td><td class="colsep0 rowsep0" align="char" char=".">225</td><td class="colsep0 rowsep0" align="char" char=".">88.2</td><td class="colsep0 rowsep0" align="char" char=".">2.0</td><td class="colsep0 rowsep0" align="char" char=".">4.9</td><td class="colsep0 rowsep0" align="char" char=".">1.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20e</b></td><td class="colsep0 rowsep0" align="left">4-OMe</td><td class="colsep0 rowsep0" align="char" char=".">33.8</td><td class="colsep0 rowsep0" align="char" char=".">162</td><td class="colsep0 rowsep0" align="char" char=".">391</td><td class="colsep0 rowsep0" align="char" char=".">79.1</td><td class="colsep0 rowsep0" align="char" char=".">4.9</td><td class="colsep0 rowsep0" align="char" char=".">11.6</td><td class="colsep0 rowsep0" align="char" char=".">2.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20f</b></td><td class="colsep0 rowsep0" align="left">4-CN</td><td class="colsep0 rowsep0" align="char" char=".">15.3</td><td class="colsep0 rowsep0" align="char" char=".">86.6</td><td class="colsep0 rowsep0" align="char" char=".">78.3</td><td class="colsep0 rowsep0" align="char" char=".">96.1</td><td class="colsep0 rowsep0" align="char" char=".">5.7</td><td class="colsep0 rowsep0" align="char" char=".">5.1</td><td class="colsep0 rowsep0" align="char" char=".">6.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>30a</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="char" char=".">37.7</td><td class="colsep0 rowsep0" align="char" char=".">89.3</td><td class="colsep0 rowsep0" align="char" char=".">341</td><td class="colsep0 rowsep0" align="char" char=".">276</td><td class="colsep0 rowsep0" align="char" char=".">2.4</td><td class="colsep0 rowsep0" align="char" char=".">9.1</td><td class="colsep0 rowsep0" align="char" char=".">7.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>30b</b></td><td class="colsep0 rowsep0" align="left">3-Cl</td><td class="colsep0 rowsep0" align="char" char=".">9.43</td><td class="colsep0 rowsep0" align="char" char=".">57.7</td><td class="colsep0 rowsep0" align="char" char=".">62.5</td><td class="colsep0 rowsep0" align="char" char=".">59.8</td><td class="colsep0 rowsep0" align="char" char=".">6.1</td><td class="colsep0 rowsep0" align="char" char=".">6.6</td><td class="colsep0 rowsep0" align="char" char=".">6.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>30c</b></td><td class="colsep0 rowsep0" align="left">4-Cl</td><td class="colsep0 rowsep0" align="char" char=".">12.0</td><td class="colsep0 rowsep0" align="char" char=".">44.3</td><td class="colsep0 rowsep0" align="char" char=".">41.3</td><td class="colsep0 rowsep0" align="char" char=".">49.4</td><td class="colsep0 rowsep0" align="char" char=".">3.7</td><td class="colsep0 rowsep0" align="char" char=".">3.4</td><td class="colsep0 rowsep0" align="char" char=".">4.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>30d</b></td><td class="colsep0 rowsep0" align="left">3-OMe</td><td class="colsep0 rowsep0" align="char" char=".">8.61</td><td class="colsep0 rowsep0" align="char" char=".">4.34</td><td class="colsep0 rowsep0" align="char" char=".">72.8</td><td class="colsep0 rowsep0" align="char" char=".">42.7</td><td class="colsep0 rowsep0" align="char" char=".">0.5</td><td class="colsep0 rowsep0" align="char" char=".">8.4</td><td class="colsep0 rowsep0" align="char" char=".">5.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>30e</b></td><td class="colsep0 rowsep0" align="left">4-OMe</td><td class="colsep0 rowsep0" align="char" char=".">30.5</td><td class="colsep0 rowsep0" align="char" char=".">175</td><td class="colsep0 rowsep0" align="char" char=".">283</td><td class="colsep0 rowsep0" align="char" char=".">51.2</td><td class="colsep0 rowsep0" align="char" char=".">5.7</td><td class="colsep0 rowsep0" align="char" char=".">9.3</td><td class="colsep0 rowsep0" align="char" char=".">1.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>30f</b></td><td class="colsep0 rowsep0" align="left">4-CN</td><td class="colsep0 rowsep0" align="char" char=".">32.7</td><td class="colsep0 rowsep0" align="char" char=".">150</td><td class="colsep0 rowsep0" align="char" char=".">343</td><td class="colsep0 rowsep0" align="char" char=".">120</td><td class="colsep0 rowsep0" align="char" char=".">4.6</td><td class="colsep0 rowsep0" align="char" char=".">10.5</td><td class="colsep0 rowsep0" align="char" char=".">3.4</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">All JAK family <i>in vitro</i> kinase assays were performed using LanthaScreen.</p></div><div class="footnote" id="t2fn2"><sup><sup>b</sup></sup><p class="last">IC<sub>50</sub> values are the average of duplicate experiments.</p></div><div class="footnote" id="t2fn3"><sup><sup>c</sup></sup><p class="last">Values are calculated using GraphPad Prism 6.</p></div></div></div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. IC<sub>50</sub>s of <b>31a–31l</b> and <b>32a–32c</b><a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01026/20210121/images/medium/jm0c01026_0015.gif" alt="" id="gr14" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="4" align="center" char=".">IC<sub>50</sub> (nM)<a class="ref internalNav" href="#t3fn2" aria-label="b">b</a><a class="ref internalNav" href="#t3fn3" aria-label="c">c</a></th><th class="rowsep1 colsep0" colspan="3" align="center" char=".">IC<sub>50</sub> ratio</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center">R<sub>1</sub></th><th class="colsep0 rowsep0" align="center">R<sub>2</sub></th><th class="colsep0 rowsep0" align="center" char=".">JAK1</th><th class="colsep0 rowsep0" align="center" char=".">JAK2</th><th class="colsep0 rowsep0" align="center" char=".">JAK3</th><th class="colsep0 rowsep0" align="center" char=".">TYK2</th><th class="colsep0 rowsep0" align="center" char=".">JAK2/JAK1</th><th class="colsep0 rowsep0" align="center" char=".">JAK3/JAK1</th><th class="colsep0 rowsep0" align="center" char=".">TYK2/JAK1</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>31a</b></td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="char" char=".">10.6</td><td class="colsep0 rowsep0" align="char" char=".">179</td><td class="colsep0 rowsep0" align="char" char=".">25.0</td><td class="colsep0 rowsep0" align="char" char=".">276</td><td class="colsep0 rowsep0" align="char" char=".">16.9</td><td class="colsep0 rowsep0" align="char" char=".">2.4</td><td class="colsep0 rowsep0" align="char" char=".">26.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>31b</b></td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">3-Cl</td><td class="colsep0 rowsep0" align="char" char=".">1.12</td><td class="colsep0 rowsep0" align="char" char=".">28.5</td><td class="colsep0 rowsep0" align="char" char=".">19.6</td><td class="colsep0 rowsep0" align="char" char=".">61.7</td><td class="colsep0 rowsep0" align="char" char=".">25.4</td><td class="colsep0 rowsep0" align="char" char=".">17.5</td><td class="colsep0 rowsep0" align="char" char=".">55.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>31c</b></td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">4-Cl</td><td class="colsep0 rowsep0" align="char" char=".">5.67</td><td class="colsep0 rowsep0" align="char" char=".">245</td><td class="colsep0 rowsep0" align="char" char=".">24.5</td><td class="colsep0 rowsep0" align="char" char=".">153</td><td class="colsep0 rowsep0" align="char" char=".">43.2</td><td class="colsep0 rowsep0" align="char" char=".">4.3</td><td class="colsep0 rowsep0" align="char" char=".">27.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>31d</b></td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">3-OMe</td><td class="colsep0 rowsep0" align="char" char=".">3.36</td><td class="colsep0 rowsep0" align="char" char=".">88.5</td><td class="colsep0 rowsep0" align="char" char=".">17.1</td><td class="colsep0 rowsep0" align="char" char=".">155</td><td class="colsep0 rowsep0" align="char" char=".">26.3</td><td class="colsep0 rowsep0" align="char" char=".">5.1</td><td class="colsep0 rowsep0" align="char" char=".">46.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>31e</b></td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">4-OMe</td><td class="colsep0 rowsep0" align="char" char=".">27.6</td><td class="colsep0 rowsep0" align="char" char=".">817</td><td class="colsep0 rowsep0" align="char" char=".">64.1</td><td class="colsep0 rowsep0" align="char" char=".">504</td><td class="colsep0 rowsep0" align="char" char=".">29.6</td><td class="colsep0 rowsep0" align="char" char=".">2.3</td><td class="colsep0 rowsep0" align="char" char=".">18.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>31f</b></td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">3-Cl</td><td class="colsep0 rowsep0" align="char" char=".">3.13</td><td class="colsep0 rowsep0" align="char" char=".">8.83</td><td class="colsep0 rowsep0" align="char" char=".">36.2</td><td class="colsep0 rowsep0" align="char" char=".">48.6</td><td class="colsep0 rowsep0" align="char" char=".">2.8</td><td class="colsep0 rowsep0" align="char" char=".">11.6</td><td class="colsep0 rowsep0" align="char" char=".">15.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>31g</b></td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">4-Cl</td><td class="colsep0 rowsep0" align="char" char=".">0.199</td><td class="colsep0 rowsep0" align="char" char=".">3.70</td><td class="colsep0 rowsep0" align="char" char=".">3.36</td><td class="colsep0 rowsep0" align="char" char=".">8.50</td><td class="colsep0 rowsep0" align="char" char=".">18.6</td><td class="colsep0 rowsep0" align="char" char=".">17.1</td><td class="colsep0 rowsep0" align="char" char=".">42.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>31h</b></td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">3-OMe</td><td class="colsep0 rowsep0" align="char" char=".">12.9</td><td class="colsep0 rowsep0" align="char" char=".">28.4</td><td class="colsep0 rowsep0" align="char" char=".">27.6</td><td class="colsep0 rowsep0" align="char" char=".">79.5</td><td class="colsep0 rowsep0" align="char" char=".">2.2</td><td class="colsep0 rowsep0" align="char" char=".">2.1</td><td class="colsep0 rowsep0" align="char" char=".">6.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>31i</b></td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">4-OMe</td><td class="colsep0 rowsep0" align="char" char=".">23.8</td><td class="colsep0 rowsep0" align="char" char=".">63.2</td><td class="colsep0 rowsep0" align="char" char=".">106</td><td class="colsep0 rowsep0" align="char" char=".">161</td><td class="colsep0 rowsep0" align="char" char=".">2.7</td><td class="colsep0 rowsep0" align="char" char=".">4.5</td><td class="colsep0 rowsep0" align="char" char=".">6.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>31j</b></td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">4-CN</td><td class="colsep0 rowsep0" align="char" char=".">1.93</td><td class="colsep0 rowsep0" align="char" char=".">4.46</td><td class="colsep0 rowsep0" align="char" char=".">2.41</td><td class="colsep0 rowsep0" align="char" char=".">6.73</td><td class="colsep0 rowsep0" align="char" char=".">2.3</td><td class="colsep0 rowsep0" align="char" char=".">1.3</td><td class="colsep0 rowsep0" align="char" char=".">3.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>31k</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">3-Cl</td><td class="colsep0 rowsep0" align="char" char=".">6.87</td><td class="colsep0 rowsep0" align="char" char=".">93.8</td><td class="colsep0 rowsep0" align="char" char=".">9.94</td><td class="colsep0 rowsep0" align="char" char=".">48.4</td><td class="colsep0 rowsep0" align="char" char=".">13.7</td><td class="colsep0 rowsep0" align="char" char=".">1.5</td><td class="colsep0 rowsep0" align="char" char=".">7.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>31l</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">3-OMe</td><td class="colsep0 rowsep0" align="char" char=".">17.2</td><td class="colsep0 rowsep0" align="char" char=".">57.6</td><td class="colsep0 rowsep0" align="char" char=".">8.84</td><td class="colsep0 rowsep0" align="char" char=".">157</td><td class="colsep0 rowsep0" align="char" char=".">3.4</td><td class="colsep0 rowsep0" align="char" char=".">0.5</td><td class="colsep0 rowsep0" align="char" char=".">9.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>32a</b></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char=".">17.0</td><td class="colsep0 rowsep0" align="char" char=".">28.0</td><td class="colsep0 rowsep0" align="char" char=".">9.24</td><td class="colsep0 rowsep0" align="char" char=".">308</td><td class="colsep0 rowsep0" align="char" char=".">1.7</td><td class="colsep0 rowsep0" align="char" char=".">0.6</td><td class="colsep0 rowsep0" align="char" char=".">18.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>32b</b></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char=".">7.86</td><td class="colsep0 rowsep0" align="char" char=".">319</td><td class="colsep0 rowsep0" align="char" char=".">314</td><td class="colsep0 rowsep0" align="char" char=".">696</td><td class="colsep0 rowsep0" align="char" char=".">40.6</td><td class="colsep0 rowsep0" align="char" char=".">40.3</td><td class="colsep0 rowsep0" align="char" char=".">88.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>32c</b></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char=".">0.769</td><td class="colsep0 rowsep0" align="char" char=".">10.9</td><td class="colsep0 rowsep0" align="char" char=".">88.7</td><td class="colsep0 rowsep0" align="char" char=".">116</td><td class="colsep0 rowsep0" align="char" char=".">14.1</td><td class="colsep0 rowsep0" align="char" char=".">115</td><td class="colsep0 rowsep0" align="char" char=".">150</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">All JAK family <i>in vitro</i> kinase assays were performed using LanthaScreen.</p></div><div class="footnote" id="t3fn2"><sup><sup>b</sup></sup><p class="last">IC<sub>50</sub> values are the average of duplicate experiments.</p></div><div class="footnote" id="t3fn3"><sup><sup>c</sup></sup><p class="last">Values are calculated using GraphPad Prism 6.</p></div></div></div><div class="NLM_p">The binding affinity for JAK1 was markedly improved when a <i>trans</i>-methyl (<b>31a–31e</b>) or <i>cis</i>-methyl (<b>31f–31j</b>) group was introduced to the C-2 position of piperidine of B (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). The SAR of these molecules synthesized from commercially available (±)-<i>cis</i>- or (±)-<i>trans</i>-2-methyl-4-amino-piperidine is hard to interpret, and further structural optimization would be helpful to clarify stereo-correlated potency and selectivity; nevertheless, it is clear that the “magic methyl” effect exists.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> The methyl substituent on piperidine occupies the tiny hydrophobic pocket around the middle of the ATP-binding pocket of JAK1, according to the docking analysis (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>c). The methyl group on the piperidine of tofacitinib also occupies this binding pocket (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>b). When <i>trans</i>-2-methyl-4-amino-piperidine was used (<b>31b–31e</b>), compounds Cl or OMe substituted at the meta position of the benzyl ring (<b>31b</b>, <b>31d</b>) were more potent than the compounds substituted at the para position (<b>31c</b>, <b>31e</b>). Interestingly, all of these compounds (<b>31b–31e</b>) demonstrated excellent selectivity over JAK2 and TYK2, exhibiting >25-fold selectivity over JAK2 and >18-fold selectivity over TYK2. Substituents at the benzyl ring had considerable effects on the selectivity for JAK1 over JAK2. In the case of <i>cis</i>-2-methyl-4-amino-piperidine derivatives (<b>31f–3j</b>), the Cl substituents (<b>31f</b> and <b>31g</b>) and CN substituent (<b>31j</b>) instead of a methoxy substituent (<b>31h</b> and <b>31i</b>) resulted in improved potency for JAK1 (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). Notably, compound <b>31g</b> inhibited JAK1 at the subnanomolar level (IC<sub>50</sub> = 0.199 nM), with 19-, 17-, and 43-fold selectivity over JAK2, JAK3, and TYK2, respectively. On introducing fluorine at the C-3 position of piperidine (<b>32a–32c</b>, <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>), the <i>cis</i>-isomer with <i>m</i>-OMe-benzyl (<b>32c</b>) exhibited strong potency for JAK1 at the subnanomolar level (IC<sub>50</sub> = 0.77 nM). This compound demonstrated good selectivity over JAK2 (14-fold) and excellent selectivity over JAK3 (115-fold) and TYK2 (150-fold). The primary carboxamide compounds <b>31k</b> and <b>31l</b> were also cross-evaluated to determine the effect of R<sub>1</sub> carboxamides; on comparing with other methyl amide compounds, they were less potent and less selective for JAK1. Additional asymmetrical synthesis of each enantiomer of <b>31g</b> and inhibitory activity assay confirmed that (<i>S</i>,<i>S</i>)-isomer (<b>38a</b>) is the eutomer of the racemic <b>31g</b> (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). IC<sub>50</sub> of <b>38a</b> for JAK1 is 0.145 nM with 20-, 16-, and 56-fold selectivity over JAK2, JAK3, and TYK2, respectively. However, its enantiomer <b>38b</b> displayed IC<sub>50</sub> of 71 nM for JAK1.</div><figure id="fig3" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01026/20210121/images/medium/jm0c01026_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01026/20210121/images/large/jm0c01026_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. (a) Docking pose generated induced fit showing <b>38a</b> (orange) in complex with JAK1 (gray, 3EYG). (b) Overlay of tofacitinib and <b>38a</b> binding poses with JAK1. (c) Surface view of the methyl binding site on the middle lipophilic binding pocket.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01026/20210121/images/large/jm0c01026_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01026&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. IC<sub>50</sub>s of <b>31g</b>, <b>32a–32c</b>, <b>38a</b>, and <b>38b</b><a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01026/20210121/images/medium/jm0c01026_0016.gif" alt="" id="gr15" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">All JAK family <i>in vitro</i> kinase assays were performed using LanthaScreen.</p></div></div><div></div></div><div class="NLM_p">Docking analyses of <b>38a</b> and <b>38b</b> with JAKs were performed using the induced fit docking module<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> in Schrödinger software (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a> and Supplementary <a href="/doi/suppl/10.1021/acs.jmedchem.0c01026/suppl_file/jm0c01026_si_001.pdf" class="ext-link">Figures S1 and S2</a>) to analyze the binding mode and explain JAK1 specificity. The location of the benzyl ring in the <b>38a</b> (<i>S</i>,<i>S</i>)- and <b>38b</b> (<i>R</i>,<i>R</i>)-isomers were different. However, the nitrogens of pyrrolopyridine in both <b>38a</b> and <b>38b</b> were found to form two hydrogen bonds with the carbonyl backbone of E957 and NH of L959 in the hinge region of JAK1, likewise interacting with the carbonyl of E930 and NH of L932 in JAK2. The methylcarboxamide at the C-1 position of pyrrolopyridine in <b>38a</b> appeared to improve selectivity for JAK1 over other JAKs due to localization near the lipophilic hinge region of JAK1. Methyl carboxamide had no steric effect on the JAK1 hinge region in this binding conformation. However, the H-bond between the NH of methyl carboxamide and hydroxyl of Tyr in the hinge region of other JAKs induced conformational changes, resulting in the displacement of the molecule from the hinge region. The NH<sup>+</sup> of piperidine in <b>38a</b> was observed to interact with the carbonyl of R1007 in JAK1, improving the potency for JAKs.</div><div class="NLM_p last">In comparison with the P-loop in the ATP-binding site of other JAKs, the P-loop of JAK1 was barely displaced for both <b>31g</b> isomers, <b>38a</b> and <b>38b</b>, because the NH of H885 in the P-loop could interact with the carbonyl of D1003 in the A-loop of JAK1, maintaining the stability of the gap between the P-loop and the A-loop. Taken together, the introduction of alkyl amides and cyclic amines of the appropriate size could improve the binding affinity for JAK1 through increased hydrophobic interactions between JAK1 and ligands; in other JAKs, the P-loop could be pushed out by a bulky hydrophobic structure, leading to a decrease in the binding affinity. In comparison with simple piperidine, the methyl group at the C-2 position of piperidine of <b>38a</b> was located inside the small hydrophobic pocket between the P- and A-loops, forming additional hydrophobic interactions.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Cell-Based Assay of the Potent Inhibitors</h3><div class="NLM_p">The inhibitory activity (GI<sub>50</sub>) of compounds <b>20c</b>, <b>30b</b>, <b>31b</b>, <b>31g</b>, <b>32c</b>, <b>38a</b>, and <b>38b</b>, which are JAK1 representative inhibitors, was examined in JAK1, JAK2, JAK3, or TYK2 gene-transformed IL-3-dependent murine hematopoietic cells (Ba/F3)(<a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>). Most compounds resulted in an approximately 100-fold reduced activity of JAK1 in the cell-based assay compared with the <i>in vitro</i> kinase inhibition assay. Compound <b>38a</b>, the most potent JAK1 inhibitor in this study, had a GI<sub>50</sub> value of 6.4 nM, which was 15-fold more potent than that of tofacitinib (GI<sub>50</sub> = 95.3 nM). Moreover, <b>38a</b> demonstrated a more than 60-fold selectivity for JAK1 over other JAKs.</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Cell-Based Assays of the Inhibitors of JAKs</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="4" align="center" char=".">GI<sub>50</sub> (μM)</th><th class="rowsep1 colsep0" colspan="3" align="center" char=".">GI<sub>50</sub> ratio</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center" char=".">JAK1</th><th class="colsep0 rowsep0" align="center" char=".">JAK2</th><th class="colsep0 rowsep0" align="center" char=".">JAK3</th><th class="colsep0 rowsep0" align="center" char=".">TYK2</th><th class="colsep0 rowsep0" align="center" char=".">JAK2/JAK1</th><th class="colsep0 rowsep0" align="center" char=".">JAK3/JAK1</th><th class="colsep0 rowsep0" align="center" char=".">TYK2/JAK1</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BBT594</td><td class="colsep0 rowsep0" align="char" char=".">0.129</td><td class="colsep0 rowsep0" align="char" char=".">0.409</td><td class="colsep0 rowsep0" align="char" char=".">0.451</td><td class="colsep0 rowsep0" align="char" char=".">0.150</td><td class="colsep0 rowsep0" align="char" char=".">3.2</td><td class="colsep0 rowsep0" align="char" char=".">3.5</td><td class="colsep0 rowsep0" align="char" char=".">1.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">tofacitinib</td><td class="colsep0 rowsep0" align="char" char=".">0.0953</td><td class="colsep0 rowsep0" align="char" char=".">1.157</td><td class="colsep0 rowsep0" align="char" char=".">0.503</td><td class="colsep0 rowsep0" align="char" char=".">1.524</td><td class="colsep0 rowsep0" align="char" char=".">12.1</td><td class="colsep0 rowsep0" align="char" char=".">5.3</td><td class="colsep0 rowsep0" align="char" char=".">16.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">filgotinib</td><td class="colsep0 rowsep0" align="char" char=".">0.798</td><td class="colsep0 rowsep0" align="char" char=".">>10</td><td class="colsep0 rowsep0" align="char" char=".">>10</td><td class="colsep0 rowsep0" align="char" char=".">7.191</td><td class="colsep0 rowsep0" align="char" char=".">>13</td><td class="colsep0 rowsep0" align="char" char=".">>13</td><td class="colsep0 rowsep0" align="char" char=".">9.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20c</b></td><td class="colsep0 rowsep0" align="char" char=".">1.795</td><td class="colsep0 rowsep0" align="char" char=".">3.953</td><td class="colsep0 rowsep0" align="char" char=".">3.895</td><td class="colsep0 rowsep0" align="char" char=".">1.714</td><td class="colsep0 rowsep0" align="char" char=".">2.2</td><td class="colsep0 rowsep0" align="char" char=".">2.2</td><td class="colsep0 rowsep0" align="char" char=".">1.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>30b</b></td><td class="colsep0 rowsep0" align="char" char=".">0.476</td><td class="colsep0 rowsep0" align="char" char=".">7.688</td><td class="colsep0 rowsep0" align="char" char=".">>10</td><td class="colsep0 rowsep0" align="char" char=".">>10</td><td class="colsep0 rowsep0" align="char" char=".">16.2</td><td class="colsep0 rowsep0" align="char" char=".">>21</td><td class="colsep0 rowsep0" align="char" char=".">>21</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>31b</b></td><td class="colsep0 rowsep0" align="char" char=".">0.232</td><td class="colsep0 rowsep0" align="char" char=".">3.414</td><td class="colsep0 rowsep0" align="char" char=".">>10</td><td class="colsep0 rowsep0" align="char" char=".">4.752</td><td class="colsep0 rowsep0" align="char" char=".">14.7</td><td class="colsep0 rowsep0" align="char" char=".">>43</td><td class="colsep0 rowsep0" align="char" char=".">20.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>32c</b></td><td class="colsep0 rowsep0" align="char" char=".">0.214</td><td class="colsep0 rowsep0" align="char" char=".">3.181</td><td class="colsep0 rowsep0" align="char" char=".">>10</td><td class="colsep0 rowsep0" align="char" char=".">9.654</td><td class="colsep0 rowsep0" align="char" char=".">14.8</td><td class="colsep0 rowsep0" align="char" char=".">>46</td><td class="colsep0 rowsep0" align="char" char=".">45.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>31g</b></td><td class="colsep0 rowsep0" align="char" char=".">0.0105</td><td class="colsep0 rowsep0" align="char" char=".">0.500</td><td class="colsep0 rowsep0" align="char" char=".">1.357</td><td class="colsep0 rowsep0" align="char" char=".">0.655</td><td class="colsep0 rowsep0" align="char" char=".">47.5</td><td class="colsep0 rowsep0" align="char" char=".">128.9</td><td class="colsep0 rowsep0" align="char" char=".">62.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>38a</b></td><td class="colsep0 rowsep0" align="char" char=".">0.0064</td><td class="colsep0 rowsep0" align="char" char=".">0.382</td><td class="colsep0 rowsep0" align="char" char=".">1.141</td><td class="colsep0 rowsep0" align="char" char=".">0.475</td><td class="colsep0 rowsep0" align="char" char=".">59.9</td><td class="colsep0 rowsep0" align="char" char=".">179.0</td><td class="colsep0 rowsep0" align="char" char=".">74.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>38b</b></td><td class="colsep0 rowsep0" align="char" char=".">2.951</td><td class="colsep0 rowsep0" align="char" char=".">>10</td><td class="colsep0 rowsep0" align="char" char=".">>10</td><td class="colsep0 rowsep0" align="char" char=".">>10</td><td class="colsep0 rowsep0" align="char" char=".">>3</td><td class="colsep0 rowsep0" align="char" char=".">>3</td><td class="colsep0 rowsep0" align="char" char=".">>3</td></tr></tbody></table></div></div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> <i>In vitro</i> ADME and Kinase Panel Assay</h3><div class="NLM_p">Additionally, basic <i>in vitro</i> ADME assays<a onclick="showRef(event, 'ref37 ref38 ref39 ref40'); return false;" href="javascript:void(0);" class="ref ref37 ref38 ref39 ref40">(37−40)</a> such as human liver microsomal stability, CYP inhibition, and MDCK-MDR1 permeability were performed with <b>31g</b> (<a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a>). When liver microsomal stability of <b>31g</b> was evaluated using human, rat, and mouse microsomes, the compound was stable in human microsomes and moderately stable in mouse microsomes. The compound showed no or weak inhibition for all five major CYP isomers at 10 μM and was highly stable in human and rat plasma at 10 μM, even over 120 min. Furthermore, the compound exhibited moderate permeability with an acceptable efflux ratio in the MDCK-MDR1 permeability assay.</div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. <i>In vitro</i> ADME Properties of <b>31g</b></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="rowsep1 colsep0" colspan="3" align="center">liver microsomal stability (% remaining, 30 min, at 10 μM)</th><th class="rowsep1 colsep0" colspan="5" align="center">CYP inhibition (% of control activity at 10 μM)</th><th class="rowsep1 colsep0" colspan="2" align="center">plasma stability (% remaining, 2 h, at 10 μM)</th><th class="rowsep1 colsep0" colspan="3" align="center">MDCK permeability (10<sup>–5</sup> cm/sec) at 10 μM</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">human</th><th class="colsep0 rowsep0" align="center">rat</th><th class="colsep0 rowsep0" align="center">mouse</th><th class="colsep0 rowsep0" align="center">3A4</th><th class="colsep0 rowsep0" align="center">2D6</th><th class="colsep0 rowsep0" align="center">2C9</th><th class="colsep0 rowsep0" align="center">2C19</th><th class="colsep0 rowsep0" align="center">1A2</th><th class="colsep0 rowsep0" align="center">human</th><th class="colsep0 rowsep0" align="center">rat</th><th class="colsep0 rowsep0" align="center">A to B</th><th class="colsep0 rowsep0" align="center">B to A</th><th class="colsep0 rowsep0" align="center">ER</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">80.6</td><td class="colsep0 rowsep0" align="left">11.6</td><td class="colsep0 rowsep0" align="left">28.2</td><td class="colsep0 rowsep0" align="left">92.9</td><td class="colsep0 rowsep0" align="left">77.1</td><td class="colsep0 rowsep0" align="left">70.0</td><td class="colsep0 rowsep0" align="left">75.7</td><td class="colsep0 rowsep0" align="left">74.8</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">92.6</td><td class="colsep0 rowsep0" align="left">0.19</td><td class="colsep0 rowsep0" align="left">0.37</td><td class="colsep0 rowsep0" align="left">1.95</td></tr></tbody></table></div></div><div class="NLM_p">Considering that kinase inhibitors targeting ATP-binding sites may inhibit other kinases, a kinase panel assay of <b>31g</b> was performed using 78 representative kinases (<a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">Table <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">7</a></a>).<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> These kinases belonged to kinase subfamilies such as TK, TKL, CMGC, MISC, lipid/atypical, CAMK, AGC, and STE. In this <i>in vitro</i> kinase panel assay, we used 1 μM <b>31g</b> at the <i>K</i><sub>m</sub> concentration of ATP, and the kinase activity was expressed as a percentage of residual activity of each enzyme in the presence of <b>31g</b> (<a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">Table <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">7</a></a>). Compound <b>31g</b> showed strong interaction with Flt1 belonging to the TK family and moderate interaction with Aurora-A in the MISC family, but the inhibition of other kinases did not occur at 1 μM <b>31g</b>. An additional IC<sub>50</sub> evaluation of <b>31g</b> for Flt1 was 106 nM. This finding implies that <b>31g</b> has >800-fold selectivity over Flt1 and, thus, is a highly selective JAK1 inhibitor.</div><div class="NLM_table-wrap" id="tbl7"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 7. Kinase Panel Assay of <b>31g</b><a class="ref internalNav" href="#t7fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">kinase Species</th><th class="colsep0 rowsep0" align="center">name</th><th class="colsep0 rowsep0" align="center">% activity</th><th class="colsep0 rowsep0" align="center">kinase species</th><th class="colsep0 rowsep0" align="center">name</th><th class="colsep0 rowsep0" align="center">% activity</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="27" align="left">TK</td><td class="colsep0 rowsep0" align="left">Abl</td><td class="colsep0 rowsep0" align="left">26</td><td class="colsep0 rowsep0" rowspan="11" align="left">CMGC</td><td class="colsep0 rowsep0" align="left">CDK1/cyclinB</td><td class="colsep0 rowsep0" align="left">84</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ALK</td><td class="colsep0 rowsep0" align="left">72</td><td class="colsep0 rowsep0" align="left">CDK2/cyclinA</td><td class="colsep0 rowsep0" align="left">77</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Blk</td><td class="colsep0 rowsep0" align="left">55</td><td class="colsep0 rowsep0" align="left">CDK6/cyclinD3</td><td class="colsep0 rowsep0" align="left">85</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Bmx</td><td class="colsep0 rowsep0" align="left">78</td><td class="colsep0 rowsep0" align="left">CDK7/cyclinH/MAT1</td><td class="colsep0 rowsep0" align="left">85</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BTK</td><td class="colsep0 rowsep0" align="left">114</td><td class="colsep0 rowsep0" align="left">CDK9/cyclin T1</td><td class="colsep0 rowsep0" align="left">85</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CSK</td><td class="colsep0 rowsep0" align="left">101</td><td class="colsep0 rowsep0" align="left">CK1γ1</td><td class="colsep0 rowsep0" align="left">100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">EGFR</td><td class="colsep0 rowsep0" align="left">88</td><td class="colsep0 rowsep0" align="left">CK2α2</td><td class="colsep0 rowsep0" align="left">91</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">EphA5</td><td class="colsep0 rowsep0" align="left">85</td><td class="colsep0 rowsep0" align="left">DYRK3</td><td class="colsep0 rowsep0" align="left">94</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">EphB4</td><td class="colsep0 rowsep0" align="left">92</td><td class="colsep0 rowsep0" align="left">GSK3β</td><td class="colsep0 rowsep0" align="left">76</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">FGFR1</td><td class="colsep0 rowsep0" align="left">34</td><td class="colsep0 rowsep0" align="left">SAPK2a</td><td class="colsep0 rowsep0" align="left">95</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">FGFR4</td><td class="colsep0 rowsep0" align="left">85</td><td class="colsep0 rowsep0" align="left">SRPK1</td><td class="colsep0 rowsep0" align="left">95</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Flt1</td><td class="colsep0 rowsep0" align="left">–1</td><td class="colsep0 rowsep0" rowspan="5" align="left">lipid/atypical</td><td class="colsep0 rowsep0" align="left">mTOR</td><td class="colsep0 rowsep0" align="left">97</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Fyn</td><td class="colsep0 rowsep0" align="left">63</td><td class="colsep0 rowsep0" align="left">PI3(p110b/p85a)</td><td class="colsep0 rowsep0" align="left">103</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">IGF-1R</td><td class="colsep0 rowsep0" align="left">59</td><td class="colsep0 rowsep0" align="left">PI3(p120g)</td><td class="colsep0 rowsep0" align="left">99</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Itk</td><td class="colsep0 rowsep0" align="left">67</td><td class="colsep0 rowsep0" align="left">PI3(p110d/p85a)</td><td class="colsep0 rowsep0" align="left">94</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">JAK1</td><td class="colsep0 rowsep0" align="left">–7</td><td class="colsep0 rowsep0" align="left">PI3(p110a/p85a)</td><td class="colsep0 rowsep0" align="left">95</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">JAK2</td><td class="colsep0 rowsep0" align="left">0</td><td class="colsep0 rowsep0" rowspan="8" align="left">CAMK</td><td class="colsep0 rowsep0" align="left">AMPKα1</td><td class="colsep0 rowsep0" align="left">52</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">JAK3</td><td class="colsep0 rowsep0" align="left">–1</td><td class="colsep0 rowsep0" align="left">CaMKI</td><td class="colsep0 rowsep0" align="left">75</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">KDR</td><td class="colsep0 rowsep0" align="left">17</td><td class="colsep0 rowsep0" align="left">CHK1</td><td class="colsep0 rowsep0" align="left">75</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Lck</td><td class="colsep0 rowsep0" align="left">31</td><td class="colsep0 rowsep0" align="left">DRAK1</td><td class="colsep0 rowsep0" align="left">79</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Lyn</td><td class="colsep0 rowsep0" align="left">26</td><td class="colsep0 rowsep0" align="left">MAPKAP-K2</td><td class="colsep0 rowsep0" align="left">97</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">LTK</td><td class="colsep0 rowsep0" align="left">73</td><td class="colsep0 rowsep0" align="left">Mnk2</td><td class="colsep0 rowsep0" align="left">88</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Met</td><td class="colsep0 rowsep0" align="left">79</td><td class="colsep0 rowsep0" align="left">Pim-1</td><td class="colsep0 rowsep0" align="left">85</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PDGFRβ</td><td class="colsep0 rowsep0" align="left">68</td><td class="colsep0 rowsep0" align="left">PRAK</td><td class="colsep0 rowsep0" align="left">97</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Rse</td><td class="colsep0 rowsep0" align="left">89</td><td class="colsep0 rowsep0" rowspan="9" align="left">AGC</td><td class="colsep0 rowsep0" align="left">MSK2</td><td class="colsep0 rowsep0" align="left">70</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Src(1-530)</td><td class="colsep0 rowsep0" align="left">82</td><td class="colsep0 rowsep0" align="left">p70S6K</td><td class="colsep0 rowsep0" align="left">60</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">TYK2</td><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left">PKA</td><td class="colsep0 rowsep0" align="left">76</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="8" align="left">MISC</td><td class="colsep0 rowsep0" align="left">Aurora-A</td><td class="colsep0 rowsep0" align="left">10</td><td class="colsep0 rowsep0" align="left">PKBα</td><td class="colsep0 rowsep0" align="left">89</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CLIK1</td><td class="colsep0 rowsep0" align="left">52</td><td class="colsep0 rowsep0" align="left">PKCα</td><td class="colsep0 rowsep0" align="left">100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">eEF-2K</td><td class="colsep0 rowsep0" align="left">73</td><td class="colsep0 rowsep0" align="left">PKCθ</td><td class="colsep0 rowsep0" align="left">68</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">IKKα</td><td class="colsep0 rowsep0" align="left">84</td><td class="colsep0 rowsep0" align="left">PKG1α</td><td class="colsep0 rowsep0" align="left">77</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">NEK2</td><td class="colsep0 rowsep0" align="left">88</td><td class="colsep0 rowsep0" align="left">ROCK-I</td><td class="colsep0 rowsep0" align="left">88</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Plk3</td><td class="colsep0 rowsep0" align="left">109</td><td class="colsep0 rowsep0" align="left">Rsk1</td><td class="colsep0 rowsep0" align="left">62</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">STK32A</td><td class="colsep0 rowsep0" align="left">83</td><td class="colsep0 rowsep0" rowspan="5" align="left">STE</td><td class="colsep0 rowsep0" align="left">ASK1</td><td class="colsep0 rowsep0" align="left">39</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">STK32C</td><td class="colsep0 rowsep0" align="left">104</td><td class="colsep0 rowsep0" align="left">LOK</td><td class="colsep0 rowsep0" align="left">62</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="4" align="left">TKL</td><td class="colsep0 rowsep0" align="left">c-RAF</td><td class="colsep0 rowsep0" align="left">81</td><td class="colsep0 rowsep0" align="left">MEK1</td><td class="colsep0 rowsep0" align="left">94</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">IRAK4</td><td class="colsep0 rowsep0" align="left">85</td><td class="colsep0 rowsep0" align="left">MST1</td><td class="colsep0 rowsep0" align="left">57</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MLK1</td><td class="colsep0 rowsep0" align="left">35</td><td class="colsep0 rowsep0" align="left">PAK2</td><td class="colsep0 rowsep0" align="left">87</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">TAK1</td><td class="colsep0 rowsep0" align="left">88</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t7fn1"><div class="footnote" id="t7fn1"><sup><sup>a</sup></sup><p class="last">Kinase panel assays were performed in the presence of <b>31g</b> (1 μM) and ATP (<i>K</i><sub>m</sub> concentration) using Eurofins KinaseProfiler. % activity represents percentage of remaining kinase activity.</p></div></div></div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Application to Fibrosis Inhibition</h3><div class="NLM_p last">Recent studies on the pro-fibrotic effects of TGF-β show that the signal transducer and activator of transcription 3 (STAT3) is implicated in fibrotic diseases.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> STAT3 signaling is hyperactivated in systemic sclerosis in a TGF-β-dependent manner, and the inhibition of the signal could ameliorate skin fibrosis in experimental mouse models. Moreover, it has been reported that the JAK1/STAT3 pathway is directly involved in TGF-β-induced liver fibrosis,<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> an abnormal response to liver injury. Considering that JAK1 inhibitors might be potential candidates for treating fibrotic disease, we examined the biological effects of <b>31g</b> in a hepatic fibrosis model.</div></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Effect of <b>31g</b> on the Proliferation of Human LX-2 Cells</h3><div class="NLM_p">The major cell types in hepatic fibrosis are HSCs, hepatocytes, and macrophages. Quiescent HSCs are activated to transform into myofibroblast-like cells in response to chronic injury. For determining the effects of <b>31g</b> on liver fibrosis, LX-2 human hepatic stellate cells were selected to examine HSC activation through proliferation, fibrogenic gene expression, and migration. Nintedanib (an FDA-approved drug for hepatic fibrosis) and <b>31g</b> showed similar inhibitory activity toward LX-2 cell proliferation induced by TGF-β. (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>a,b) The inhibition of LX-2 cell growth was more effective than with tofacitinib or filgotinib when nintedanib or <b>31g</b> was incubated with TGF-β for 72 h (Supplementary <a href="/doi/suppl/10.1021/acs.jmedchem.0c01026/suppl_file/jm0c01026_si_001.pdf" class="ext-link">Figure S3a</a>). For determining the potential toxicity of <b>31g</b>, cytotoxicity evaluation was performed using a cell-impermeable cyanine dimer nucleic acid stain (YOYO1) that binds to dsDNA in human stellate cells. As shown in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>c,d, <b>31g</b> had relatively low toxicity to LX-2 cells compared to nintedanib in normal media.</div><figure id="fig4" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01026/20210121/images/medium/jm0c01026_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01026/20210121/images/large/jm0c01026_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (a) Proliferation of LX2 induced by TGF-β was investigated with CCK8. The inhibition effects of nintedanib and <b>31g</b> on TGF-β induced proliferation of LX-2 were observed in a time-dependent manner. (b) Under the same conditions, growth inhibition of nintedanib and <b>31g</b> was observed in a dose-dependent manner to measure GI<sub>50</sub>. (c) Cell impermeable YOYO1 was used to assess real-time cytotoxicity. YOYO1 fluorescence images were acquired by increase treatment with nintedanib in LX-2 cells compared to <b>31g</b>. (d) YOYO1 images were analyzed by an Incucyte imaging FLR in a time-dependent manner. The IC<sub>50</sub> value was calculated through the Imaging FLR.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01026/20210121/images/large/jm0c01026_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01026&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> <b>31g</b> Ameliorates TGF-β-Induced Fibrotic Gene Expression and Migration in Human LX-2 Cells</h3><div class="NLM_p">As TGF-β is a potent pro-fibrotic cytokine that regulates the genes involved in liver fibrosis,<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> TGF-β-induced HSCs differentiate into myofibroblasts and secrete extracellular matrix (ECM) proteins. Changes in fibrotic gene parameters were measured to investigate the effects of <b>31g</b> on myoblastic differentiation in HSC activation. As shown in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>a, TGF-β-induced HSCs produced α-smooth muscle actin (α-SMA), collagen type l α1 (Col1A1), and tissue inhibitors of metalloproteinases 1 (TIMP1), while treatment with <b>31g</b> at a concentration of 500 nM reduced the production of α-SMA, Col1A1, and TIMP1 in HSCs. Especially, immunofluorescence staining and intensity analysis by Harmony software 3.1 revealed that α-SMA reduced with the treatment of <b>31g</b> (250 nM) in LX-2 cells (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>b). In liver fibrosis, HSCs migrate to tissue injury sites during fibrogenesis and differentiate into contractile myofibroblasts that promote liver stiffness. Examination of HSC migration in wound-healing assays revealed that 250 nM <b>31g</b> significantly inhibited TGF-β-induced migration of human HSCs, as shown in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>c. Wound healing was observed in a confluent and serum-starved state to exclude the proliferation effects. There was no difference even during proliferation at concentrations with migration inhibitory effects (0.25, 0.5, and 1 μM). The inhibition of HSC migration with <b>31g</b> was more effective than with tofacitinib or filgotinib when the drugs (500 nM) were incubated with TGF-β (10 ng/mL)(Supplementary <a href="/doi/suppl/10.1021/acs.jmedchem.0c01026/suppl_file/jm0c01026_si_001.pdf" class="ext-link">Figure S3b</a>). These results demonstrated the potential of <b>31g</b> as a therapeutic compound for preventing HSC activation.</div><figure id="fig5" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01026/20210121/images/medium/jm0c01026_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01026/20210121/images/large/jm0c01026_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (a) Gene expression levels of fibrotic parameters measured in LX-2 incubation with TGF-β treated without or with <b>31g</b> (500 nM). (b) left, <b>31g</b> (250 nM) ameliorated TGF-β-induced α-SMA stress fiber formation. Immunofluorescence analysis of α-SMA (green) and Hoechst 33258 (blue) in LX-2 cells treated with 10 ng/mL TGF-β for 72 h. Scale bar 200 μm. right, Fluorescence intensity was analyzed by phenotypic percentage of the population with constant GFP intensity value using Harmony 3.1. (c) Evaluation of the wound healing/migration inhibition effect of <b>31g</b> based on an <i>in vitro</i> scratch wound assay using LX-2 cells. Scratch wound healing/migration induced by TGF-β in LX-2 cells treated with media alone (0.5% FBS) and indicated concentrations of <b>31g</b>. Red line: initial 0 h, purple line: after 24 h. (d) <b>31g</b> inhibits the JAK-STAT signal, but not nintedanib (classical inhibitor of hepatic fibrosis). Immunobit cell-based immunoassays of pSTAT3 and pSTAT5 were analyzed in TF1 cells with nintedanib and <b>31g</b>. (e) TGF-β-induced STAT3 phosphorylation in LX-2 levels reduced with <b>31g</b> (250 nM) treatment. Band intensity was analyzed by Image studio with ODYSSEY from LI-COR.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01026/20210121/images/large/jm0c01026_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01026&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> JAK1/STAT3 Pathway Induced in Human LX-2 Cells by TGF-β Was Reduced by <b>31g</b></h3><div class="NLM_p last">Immunobit cell-based immune analysis showed that <b>31g</b> inhibited JAK1/STAT3 signaling in HSCs in a dose-dependent manner. However, nintedanib did not affect the JAK/STAT pathway (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>d). In the same experiment, the well-known JAK1-selective inhibitor filgotinib also reduced pSTAT3 and pSTAT5; however, it was less effective than <b>31g</b>. The results indicated that both nintedanib and <b>31g</b> ameliorate hepatic fibrosis (reduced gene expression of fibrotic parameters and migration) but function through different pathways.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> Therefore, <b>31g</b> may be a potential candidate for the treatment of hepatic fibrosis alone or as a concomitant agent with synergistic effects due to binding with other pathways with low toxicity.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i25">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_20601" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_20601" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In summary, an analysis of the structural characteristics of the well-known JAK1 inhibitors and their binding modes to JAKs was used to select <i>N</i>-methyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-5-carboxamide as a potent and selective scaffold for JAK1 inhibition. Optimization of the middle lipophilic pocket binder with the terminal substituted benzyl led to the identification of 4-((<i>cis</i>-1-(4-chlorobenzyl)-2-methylpiperidin-4-yl)amino)-<i>N</i>-methyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-5-carboxamide (<b>31g</b>) as a highly potent and selective JAK1 inhibitor with an IC<sub>50</sub> value of 0.199 nM.</div><div class="NLM_p last">Additional asymmetrical synthesis of each enantiomer of <b>31g</b> and inhibitory activity assay confirmed that (<i>S</i>,<i>S</i>)-isomer (<b>38a</b>) is the eutomer of the racemic <b>31g</b>. IC<sub>50</sub> of <b>38a</b> for JAK1 was 0.145 nM with 20-, 16-, and 56-fold selectivity over JAK2, JAK3, and TYK2, respectively, which was enhanced in the Ba/F3 cell-based assay. The GI<sub>50</sub> value against JAK1 was 6.4 nM, and the selectivity over JAK2, JAK3, and TYK2 were more than 60-fold. In the kinase panel assay that included 78 representative kinases from across the kinome, there were no kinases hit by <b>31g</b> at the tested concentration of 1 μM, except for FLT1 (IC<sub>50</sub> = 106 nM), indicating a high degree of kinase selectivity. Compound <b>31g</b> reduced the proliferation, fibrogenic gene expression, and migration of TGF-β-induced LX-2 cells. In particular, <b>31g</b> significantly inhibited the TGF-β-induced migration of HSCs at 0.25 μM in wound-healing assays. JAK1-selective inhibition may be an effective strategy to treat various fibrosis and inflammation-related disorders.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i26">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_07616" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_07616" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> JAK Enzyme IC<sub>50</sub> Assay</h3><div class="NLM_p last">JAK family <i>in vitro</i> kinase assays were followed according to the “optimization of LanthaScreen kinase assay for JAKs family” from Thermofisher. Recombinant JAK1 (Promega, PV4744), JAK2 (Promega, PV4210), JAK3 (Promega, PV3855), and TYK2 (Promega PV4790) were adapted. GFP-STAT1 (Promega, PV5211) was used as a substrate for kinase, and the phosphorylation degree was measured by the TR-FRET method using LanthaScreen Tb-anti-pSTAT1(pTyr701)(Promega, PV4844). Kinase reactions were performed in 384-well low-volume plates in duplicate. Assays were conducted for 1 h with candidate compounds, kinase, and substrate. Followed by EDTA, kinase quench buffer (Promega, P2825), and LanthaScreen Tb-anti-pSTAT1(pTyr701) in TR-FRET, dilution buffer was added and incubated for 30 min. TR-FRET emission ratios were generated by a Synergy Neo2 plate reader (Biotek, Winooski, VT) using the LanthaScreen filters, and IC<sub>50</sub> was calculated by GraphPad Prism 6.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> JAK Cell-Based Selectivity Assay</h3><div class="NLM_p last">Ba/F3-JAK1, JAK2, JAK3, and TYK2 cells were derived from the IL-3-dependent murine hematopoietic cell line (Ba/F3) purchased from Applied Biologic Materials Inc. (Abm). Transfected genes (JAK1, JAK2, JAK3, and TYK2) were identified by sequencing and immunoblotting. Stably transfected Ba/F3 cells were maintained in a PriGrowll (Abm, TM999) supplemented with puromycin. Ba/F3-JAK1, JAK2, JAK3, and TYK2 cells (1 × 10<sup>4</sup>/well) were seeded in a 96-well plate, and candidate compounds were treated at 10 concentrations over 3-fold serial dilutions (0–10 μM) for 72 h in a CO<sub>2</sub> incubator. CCK8 solutions (Dojindo, CK04-20) were added to each well of the plate and incubated for 2 h in the CO<sub>2</sub> incubator. Absorbance was measured at 450 nm by a Synergy Neo2 plate reader (BioTek, Winnoskin, VT), and GI<sub>50</sub> was calculated by GraphPad Prism 6.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> Kinase Profiling</h3><div class="NLM_p last">Activities of 78 kinases were measured by the Kinase Profiler service, Eurofins Pharma Discovery Services. For more details, refer to https://www.eurofinsdiscoveryservices.com/cms/cms-content/services/in-vitro-assays/kinases/kinase-profiler/.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> Cell Culture</h3><div class="NLM_p last">LX-2 human hepatic stellate cell (HSC) lines were purchased from EMD Millipore Corporation. LX-2 cells were cultured in DMEM High Glucose (Millipore cat. no. ES009-B) supplemented with 2% fetal bovine serum (FBS)(HyClone, Nepean, ON, Canada) and 1× glutamine (Millipore cat. no. TMS-002-C). For all experiments, LX-2 cells were seeded at 2 or 7 × 10<sup>3</sup> cells/well in a 96-well plate, 2 × 10<sup>5</sup> cells/well in a 6-well plate. For HSC activation, LX-2 cells were incubated with cell media containing 0.5% FBS for 24 h before TGF-β 5 or 10 ng treatment. Cell growth was measured using a cell counting kit-8 (Dojindo Molecular Technologies).</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> Quantitative Real-Time RT-PCR</h3><div class="NLM_p last">After stimulation, total RNA from the LX-2 cells was extracted using a Hybrid RTM (GeneAll). cDNAs were generated with a high-capacity cDNA reverse transcription kit (ThermoFisher) and amplified using the FAST SYBRTM Green Master kit (Applied Biosystems, Warrington, UK) on a StepOnePlus Real-Time PCR system (Applied Biosystems). Relative expression of the profibrotic genes α-SMA, COL1A1, and TIMP-I was measured. The primer sequences were as follows: human α-SMA (forward: CTGTTCCAGCCATCCTTCAT, reverse: CGGCTTCATCGTATTCCTGT), human Col1α1 (forward: GAACATCACCTACCACTGCA, reverse: GTTGGGATGGAGGGAGTTTA), human TIMP1 (forward: GAACATCACCTACCACTGCA, reverse: GTTGGGATGGAGGGAGTTTA). GAPDH was used as an internal control, and a specific reaction was confirmed by melting curve analysis.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> Immunofluorescence</h3><div class="NLM_p last">LX-2 cells were plated in a 96-well plate at a density of 2 × 10<sup>3</sup> cells/well and incubated at 37 °C for 24 h. Subsequently, the cells were treated with compounds for 72 h of stimulation in TGF-β (Peprotech, cat.: 100-21, 5 or 10 ng) conditions. The cells were fixed with 1% paraformaldehyde (PFA) for 1 h, permeabilized with 0.1% Triton X-100, and incubated with αSMA antibody (ab197240, SIGMA A5228) in zymogen Ab diluent solution for 1 day at 4 °C. After adding Alexa Fluor 488-conjugated anti-rabbit secondary antibody (Molecular Probes) for 2 h at room temperature, the nuclei were counterstained with Hoechst33258 (Invitrogen, H3569). The images were acquired by Operetta (PerkinElmer). The fluorescence intensity of images was analyzed by Harmony software 3.1.2.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> Immunobit Cell-Based Immunoassay</h3><div class="NLM_p last">TF1 cells were plated in a white 96-well plate at a density of 2.5 × 10<sup>5</sup> cells/well and incubated in 10% RPMI1640 with 10% FBS for 16 h. For JAK1/STAT3, the same amounts of TF1 cells were plated in a white 96-well plate and incubated in RPMI1640 containing 0.1% FBS for 16 h. Cells were pretreated with the indicated concentration of the compound for 30 min and incubated with IL-6 for 30 min. For JAK2/STAT5, the cells were pretreated with the indicated concentration of the compound for 1 h and incubated with IL-3 for 20 min. The cells were lysed with cell lysis buffer containing 0.1% digitonin for 20 min. Subsequently, the antibody mix (pSTAT3/STAT3 or pSTAT5/STAT5, NanoBiT detecting antibodies) was added to cell lysates. After incubating the plates at 23 °C for 90 min, the NanoGlo reagent was added. Luminescence values were measured using Synergy Neo2, and the acquired data was calculated via GraphPad Prism 6.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> Cytotoxicity Assay</h3><div class="NLM_p last">SV40 MES 13 cells at a density of 1 × 10<sup>3</sup> cells/well were plated in 96-well plates overnight. The medium was replaced with a fresh medium containing the indicated concentrations of the compound, 50 nM YOYO-1 cytotoxicity reagent (Essen Biosciences), a fluorescent cell-impermeant cyanine dimer, and a nucleic acid-staining dye. The fluorescence of YOYO-1 was monitored over 48 h and quantified according to the manufacturer’s instructions.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> Scratch Wound Assay</h3><div class="NLM_p last">LX-2 cells at a density of 1 × 10<sup>4</sup> cells/well were plated overnight in Essen 96-well ImageLock microplates. Wound areas were automatically created using Essen 96-well WoundMaker. The medium was replaced with a 0.5% starvation medium for 24 h. The next day, the medium was exchanged with a fresh 0.5% medium containing the indicated concentrations of the compound for 24 h. The confluence of cells within the wound region was analyzed by IncuCyte software.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> Computational Chemistry</h3><div class="NLM_p last">All the docking studies were performed on Schrodinger-Maestro v12.1. The proteins used in the docking studies were obtained from the PDB data bank (PDB: 3EYG, 3FUP, 3LXK, and 3LXN). The proteins were prepared through the process that corrected missing side chains and loops, removed waters, and restrained minimization by using OPLS3e. Inhibitors went through LigPrep, which made all forms at pH 2–7. Induced fit docking was performed under the following conditions: picking tofacitinib, a ligand of the protein structures, as the center of the grid box, giving an H-bond at the hinge region, and optimizing side chains within 10 Å of ligand poses.</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> Synthetic Methods and Molecular Characterization</h3><div class="NLM_p">All chemical reagents were purchased from Sigma-Aldrich, TCI, Ark-pharm, Combi-Blocks, and Alfa-Aesar. The chirality test of <i>cis</i>-2-methylaminopiperidine derivatives was conducted by using the Mosher’s reagent, turning out to be a (<i>R</i>,<i>R</i>)- and (<i>S</i>,<i>S</i>)-racemic mixture. A Biotage 356007 synthesizer was used for microwave-assisted reaction. Flash column chromatography was performed using a Combiflash NextGen 300+ with an appropriate prepacked silica gel cartage. High-resolution mass spectra were recorded on an Agilent 6230 TOF LC/MS instrument. <sup>1</sup>H and <sup>13</sup>C NMR spectra were recorded on an Agilent DD2 400 MHz FT-NMR instrument at 400 and 100 MHz, respectively. Chemical shifts are expressed in δ values using TMS as a reference standard. The purity of final compounds were determined by HPLC (Shimadzu LCMS-2020, Thunder ST C18, 3 μm, 150 × 3.0 mm, 40 °C, UV 254 nm, flow rate = 0.8 mL/min). The gradient was used with 5% of ACN in water (0.1% FA) to 80% of ACN. All final compounds were confirmed to be >95% pure by HPLC analysis. Enantiopurity was measured by chiral HPLC (Agilent 1100 HPLC/DAD, Chiralpak IG-3, 3 μm, 150 × 4.6 mm, room temperature, UV 310 nm, flow rate = 1.0 mL/min, and mobile phase, diethyl amine/Hex/EtOH/MeOH = 0.04/50/40/10).</div><div id="sec4_11_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> 4-Chloro-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-5-carboxamide (<b>7a</b>)</h4><div class="NLM_p last">To a solution of 4-chloro-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-5-carboxylic acid (1.0 mmol) in DMF (20 mL) was added CDI (1.05 mmol) under N<sub>2</sub> gas at room temperature. After stirring for 1 h, ammonia solution in THF (5.0 mmol) was added slowly and the reaction mixture was stirred for 1 h at room temperature. Solvents were removed by using the rotary evaporator, and the resulting mixture was diluted with ethyl acetate. The precipitate was collected by vacuum filtration, washed with water to remove the imidazole, and dried under vacuum. The resulting amorphous powder was used to the next step without further purification. Yield: 71%; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.44 (s, 1H), 7.49 (d, <i>J</i> = 3.4 Hz, 1H), 6.47 (d, <i>J</i> = 3.4 Hz, 1H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 168.76, 148.24, 145.29, 132.33, 128.10, 127.23, 119.13, 99.24; MS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>8</sub>H<sub>7</sub>ClN<sub>3</sub>O [M + H]<sup>+</sup> 196.03, found 196.90.</div></div><div id="sec4_11_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> 4-Chloro-<i>N</i>-methyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-5-carboxamide (<b>7b</b>)</h4><div class="NLM_p last"><b>7b</b> was synthesized in 87% yield according to the same procedure as <b>7a</b> using methylamine; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.15 (s, 1H), 8.37 (q, <i>J</i> = 4.7 Hz, 1H), 8.24 (s, 1H), 7.65 (d, <i>J</i> = 3.5 Hz, 1H), 6.56 (d, <i>J</i> = 3.4 Hz, 1H), 2.80 (d, <i>J</i> = 4.6 Hz, 3H); <sup>13</sup>C NMR (100 MHz, chloroform-<i>d</i>) δ 168.9, 149.4, 143.1, 134.3, 128.5, 124.5, 120.7, 100.6, 26.9; MS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>9</sub>H<sub>9</sub>ClN<sub>3</sub>O [M + H]<sup>+</sup> 210.04, found 210.00.</div></div><div id="sec4_11_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> 4-Chloro-<i>N</i>-cyclopropyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-5-carboxamide (<b>7c</b>)</h4><div class="NLM_p last"><b>7c</b> was synthesized in 62% yield according to the same procedure as <b>7a</b> using cyclopropylamine; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.18 (s, 1H), 8.53 (d, <i>J</i> = 4.4 Hz, 1H), 8.21 (s, 1H), 7.65 (d, <i>J</i> = 3.5 Hz, 1H), 6.56 (d, <i>J</i> = 3.5 Hz, 1H), 2.91–2.80 (m, 1H), 0.76–0.66 (m, 2H), 0.60–0.51 (m, 2H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.01, 148.94, 142.63, 132.06, 128.81, 124.79, 118.93, 99.29, 23.39, 6.25; MS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>11</sub>H<sub>11</sub>ClN<sub>3</sub>O [M + H]<sup>+</sup> 236.06, found 235.95.</div></div><div id="sec4_11_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> 4-Chloro-<i>N</i>-cyclopentyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-5-carboxamide (<b>7d</b>)</h4><div class="NLM_p last"><b>7d</b> was synthesized in 32% yield according to the same procedure as <b>7a</b> using cyclopentylamine; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.44 (d, <i>J</i> = 7.2 Hz, 1H), 8.20 (s, 1H), 7.65 (d, <i>J</i> = 3.4 Hz, 1H), 6.56 (d, <i>J</i> = 3.4 Hz, 1H), 4.27–4.17 (m, 1H), 1.96–1.82 (m, 2H), 1.78–1.61 (m, 3H), 1.59–1.42 (m, 5H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.27, 148.90, 142.64, 131.96, 128.74, 125.29, 118.89, 99.19, 51.43, 32.64, 24.03; MS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>13</sub>H<sub>15</sub>ClN<sub>3</sub>O [M + H]<sup>+</sup> 264.09, found 263.95.</div></div><div id="sec4_11_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> <i>tert</i>-Butyl (1-Phenylpiperidin-4-yl)carbamate (<b>13a</b>) and <i>tert</i>-Butyl (1-Benzylpiperidin-4-yl)carbamate (<b>13b</b>)</h4><div class="NLM_p last"><b>13a</b> and <b>13b</b> were purchased.</div></div><div id="sec4_11_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> <i>tert</i>-Butyl (1-(2-Methoxybenzyl)piperidin-4-yl)carbamate (<b>13c</b>)</h4><div class="NLM_p last">To a solution of <i>tert</i>-butyl piperidin-4-ylcarbamate (<b>9</b>, 1.3 mmol) in DMF (5 mL) were added K<sub>2</sub>CO<sub>3</sub> (5.2 mmol) and 2-methoxybenzyl bromide (2.6 mmol). After stirring for 1 h at 40 °C, the mixture was poured into Et<sub>2</sub>O (40 mL), washed with brine (40 mL × 2), dried over MgSO<sub>4</sub>, and the solvent was removed <i>in vacuo</i>. The residue was purified by silica gel chromatography (30% ethyl acetate in <i>n</i>-hexane elution) to give the target product <b>13c</b>. Yield: 93%; <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 7.31–7.25 (m, 2H), 6.98 (d, <i>J</i> = 8.2 Hz, 1H), 6.93 (t, <i>J</i> = 7.5 Hz, 1H), 3.84 (s, 3H), 3.59 (s, 2H), 3.33–3.27 (m, 1H), 2.95–2.88 (m, 2H), 2.23–2.13 (m, 2H), 1.87–1.79 (m, 2H), 1.55–1.45 (m, 2H), 1.44 (s, 9H).</div></div><div id="sec4_11_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> <i>tert</i>-Butyl (1-(3-Methoxy)piperidin-4-yl)carbamate (<b>13d</b>)</h4><div class="NLM_p last"><b>13d</b> was synthesized in 74% yield according to the same procedure as <b>13c</b> using <b>9</b> (1.3 mmol) and 3-methoxybenzyl bromide (2.6 mmol); Yield: 74%; <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 7.24 (t, <i>J</i> = 7.6 Hz, 1H), 6.94–6.88 (m, 2H), 6.84 (dd, <i>J</i> = 2.4, 8.4 Hz, 1H), 3.80 (S, 3H), 3.50 (s, 2H), 3.39–3.32 (m, 1H), 2.90–2.83 (m, 2H), 2.16–2.07 (m, 2H), 1.88–1.81 (m, 2H), 1.55–1.46 (m, 2H), 1.45 (s, 9H).</div></div><div id="sec4_11_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> <i>tert</i>-Butyl (1-(4-Methoxybenzyl)piperidin-4-yl)carbamate (<b>13e</b>)</h4><div class="NLM_p last"><b>13e</b> was synthesized in 81% yield according to the same procedure as <b>13c</b> using <b>9</b> (1.3 mmol) and 4-methoxybenzyl bromide (2.6 mmol); <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 7.24 (d, <i>J</i> = 8.6 Hz, 2H), 6.89 (d, <i>J</i> = 8.6 Hz, 2H), 3.80 (s, 3H), 3.47 (s, 2H), 3.38–3.28 (m, 1H), 2.90–2.82 (m, 2H), 2.14–2.05 (m, 2H), 1.88–1.80 (m, 2H), 1.53–1.44 (m, 2H), 1.44 (s, 9H).</div></div><div id="sec4_11_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> <i>tert</i>-Butyl (1-(3-Cyanobenzyl)piperidin-4-yl)carbamate (<b>13f</b>)</h4><div class="NLM_p last"><b>13f</b> was synthesized in 78% yield according to the same procedure as <b>13c</b> using <b>9</b> (1.3 mmol) and 3-(bromomethyl)benzonitrile (2.6 mmol); <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 7.73 (bs, 1H), 7.68–7.63 (m, 1H), 7.53 (t, <i>J</i> = 7.6 Hz, 1H), 3.58 (s, 2H), 3.40–3.32 (m, 1H), 2.87–2.80 (m, 2H), 2.19–2.10 (m, 2H), 1.89–1.91 (m, 2H), 1.55–1.46 (m, 2H), 1.45 (s, 9H).</div></div><div id="sec4_11_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> <i>tert</i>-Butyl (1-(3,4-Dimethoxybenzyl)piperidin-4-yl)carbamate (<b>13g</b>)</h4><div class="NLM_p last"><b>13g</b> was synthesized in 55% yield according to the same procedure as <b>13c</b> using <b>9</b> (1.3 mmol) and 3,4-dimethoxybenzyl bromide (2.6 mmol); <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 6.87 (bs, 1H), 6.84–6.77 (m, 2H), 3.89 (s, 3H), 3.87 (s, 3H), 3.53–3.42 (m, 1H), 3.42 (s, 2H), 2.83–2.74 (m, 2H), 2.11–2.00 (m, 2H), 1.94–1.86 (m, 2H), 1.45–1.34 (m, 2H), 1.44 (s, 9H).</div></div><div id="sec4_11_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> <i>tert</i>-Butyl ((1-Benzylpiperidin-4-yl)methyl)carbamate (<b>14a</b>)</h4><div class="NLM_p last"><b>14a</b> was synthesized in 53% yield according to the same procedure as <b>13c</b> using <i>tert</i>-butyl (piperidin-4-ylmethyl)carbamate (<b>10</b>, 1.3 mmol) and benzyl bromide (2.6 mmol); <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 7.30 (d, <i>J</i> = 4.4 Hz, 4H), 7.23 (dd, <i>J</i> = 5.9, 2.8 Hz, 1H), 4.58 (s, 1H), 3.48 (s, 2H), 3.01 (t, <i>J</i> = 6.2 Hz, 2H), 2.88 (d, <i>J</i> = 11.6 Hz, 2H), 1.99–1.89 (m, 2H), 1.68–1.60 (m, 2H), 1.43 (s, 9H), 1.34–1.20 (m, 2H).</div></div><div id="sec4_11_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> <i>tert</i>-Butyl ((1-(3-Chlorobenzyl)piperidin-4-yl)methyl)carbamate (<b>14b</b>)</h4><div class="NLM_p last"><b>14b</b> was synthesized in 79% yield according to the same procedure as <b>13c</b> using <i>tert</i>-butyl (piperidin-4-ylmethyl)carbamate (<b>10</b>, 1.3 mmol) and 3-chlorobenzyl bromide (2.6 mmol); <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 7.32 (dt, <i>J</i> = 2.2, 1.0 Hz, 1H), 7.26 (s, 1H), 7.24–7.16 (m, 3H), 4.57 (s, 1H), 3.45 (s, 2H), 3.02 (t, <i>J</i> = 6.4 Hz, 2H), 2.85 (d, <i>J</i> = 11.3 Hz, 2H), 1.94 (td, <i>J</i> = 11.6, 2.5 Hz, 2H), 1.65 (d, <i>J</i> = 12.9 Hz, 2H), 1.57 (s, 1H), 1.44 (s, 9H), 1.32–1.21 (m, 2H).</div></div><div id="sec4_11_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> <i>tert</i>-Butyl ((1-(4-Chlorobenzyl)piperidin-4-yl)methyl)carbamate (<b>14c</b>)</h4><div class="NLM_p last"><b>14c</b> was synthesized in 34% yield according to the same procedure as <b>13c</b> using <i>tert</i>-butyl (piperidin-4-ylmethyl)carbamate (<b>10</b>, 1.3 mmol) and 4-chlorobenzyl bromide (2.6 mmol); <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 7.27 (d, <i>J</i> = 6.1 Hz, 2H), 7.23 (d, <i>J</i> = 8.5 Hz, 2H), 4.57 (s, 1H), 3.44 (s, 2H), 3.01 (t, <i>J</i> = 6.4 Hz, 2H), 2.84 (d, <i>J</i> = 11.3 Hz, 2H), 1.97–1.88 (m, 2H), 1.65 (d, <i>J</i> = 12.5 Hz, 2H), 1.56 (s, 1H), 1.43 (s, 9H), 1.31–1.19 (m, 2H).</div></div><div id="sec4_11_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> <i>tert</i>-Butyl ((1-(3-Methoxybenzyl)piperidin-4-yl)methyl)carbamate (<b>14d</b>)</h4><div class="NLM_p last"><b>14d</b> was synthesized in 77% yield according to the same procedure as <b>13c</b> using <i>tert</i>-butyl (piperidin-4-ylmethyl)carbamate (<b>10</b>, 1.3 mmol) and 3-methoxybenzyl bromide (2.6 mmol); <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 7.21 (t, <i>J</i> = 8.0 Hz, 1H), 6.91–6.87 (m, 2H), 6.80–6.77 (m, 1H), 4.57 (s, 1H), 3.81 (s, 3H), 3.46 (s, 2H), 3.01 (t, <i>J</i> = 6.4 Hz, 2H), 2.88 (d, <i>J</i> = 11.4 Hz, 2H), 1.98–1.89 (m, 2H), 1.64 (d, <i>J</i> = 12.8 Hz, 2H), 1.57 (s, 1H), 1.43 (s, 9H), 1.32–1.21 (m, 2H).</div></div><div id="sec4_11_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> <i>tert</i>-Butyl ((1-(4-Methoxybenzyl)piperidin-4-yl)methyl)carbamate (<b>14e</b>)</h4><div class="NLM_p last"><b>14e</b> was synthesized in 58% yield according to the same procedure as <b>13c</b> using <i>tert</i>-butyl (piperidin-4-ylmethyl)carbamate (<b>10</b>, 1.3 mmol) and 4-methoxybenzyl bromide (2.6 mmol); <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 7.23–7.18 (m, 2H), 6.87–6.82 (m, 2H), 4.56 (s, 1H), 3.80 (s, 3H), 3.42 (s, 2H), 3.01 (t, <i>J</i> = 6.4 Hz, 2H), 2.87 (d, <i>J</i> = 11.0 Hz, 2H), 1.91 (dd, <i>J</i> = 12.5, 10.0 Hz, 2H), 1.64 (d, <i>J</i> = 12.8 Hz, 2H), 1.56 (s, 1H), 1.43 (s, 9H), 1.31–1.19 (m, 2H).</div></div><div id="sec4_11_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> <i>tert</i>-Butyl (1-Benzylpyrrolidin-3-yl)carbamate (<b>15a</b>)</h4><div class="NLM_p last"><b>15a</b> was synthesized in 68% yield according to the same procedure as <b>13c</b> using <i>tert</i>-butyl pyrrolidin-3-ylcarbamate (<b>11</b>, 1.3 mmol) and benzyl bromide (2.6 mmol); <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 7.31 (d, <i>J</i> = 3.6 Hz, 4H), 7.24 (dd, <i>J</i> = 5.5, 3.0 Hz, 1H), 4.82 (s, 1H), 4.16 (s, 1H), 3.59 (s, 2H), 2.77 (s, 1H), 2.67–2.57 (m, 1H), 2.53 (s, 1H), 2.37–2.20 (m, 2H), 1.60–1.53 (m, 1H), 1.43 (s, 9H).</div></div><div id="sec4_11_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> <i>tert</i>-Butyl (1-(3-Chlorobenzyl)pyrrolidin-3-yl)carbamate (<b>15b</b>)</h4><div class="NLM_p last"><b>15b</b> was synthesized in 96% yield according to the same procedure as <b>13c</b> using <i>tert</i>-butyl pyrrolidin-3-ylcarbamate (<b>11</b>, 1.3 mmol) and 3-chlorobenzyl bromide (2.6 mmol); <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 7.32 (s, 1H), 7.25–7.20 (m, 2H), 7.18 (t, <i>J</i> = 4.1 Hz, 1H), 4.80 (s, 1H), 4.17 (s, 1H), 3.62–3.50 (m, 2H), 2.77 (s, 1H), 2.61 (t, <i>J</i> = 8.1 Hz, 1H), 2.53 (s, 1H), 2.35–2.19 (m, 2H), 1.58 (d, <i>J</i> = 9.1 Hz, 1H), 1.44 (s, 9H).</div></div><div id="sec4_11_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> <i>tert</i>-Butyl (1-(4-Chlorobenzyl)pyrrolidin-3-yl)carbamate (<b>15c</b>)</h4><div class="NLM_p last"><b>15c</b> was synthesized in 32% yield according to the same procedure as <b>13c</b> using <i>tert</i>-butyl pyrrolidin-3-ylcarbamate (<b>11</b>, 1.3 mmol) and 4-chlorobenzyl bromide (2.6 mmol); <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 7.30–7.28 (m, 1H), 7.27 (d, <i>J</i> = 2.2 Hz, 1H), 7.24 (d, <i>J</i> = 2.4 Hz, 1H), 7.23–7.22 (m, 1H), 4.79 (s, 1H), 4.16 (s, 1H), 3.55 (s, 2H), 2.75 (s, 1H), 2.59 (dd, <i>J</i> = 9.7, 6.4 Hz, 1H), 2.50 (d, <i>J</i> = 9.6 Hz, 1H), 2.34–2.16 (m, 2H), 1.66–1.53 (m, 1H), 1.43 (s, 9H).</div></div><div id="sec4_11_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> <i>tert</i>-Butyl (1-(3-Methoxybenzyl)pyrrolidin-3-yl)carbamate (<b>15d</b>)</h4><div class="NLM_p last"><b>15d</b> was synthesized in 63% yield according to the same procedure as <b>13c</b> using <i>tert</i>-butyl pyrrolidin-3-ylcarbamate (<b>11</b>, 1.3 mmol) and 3-methoxybenzyl bromide (2.6 mmol); 1H NMR (400 MHz, chloroform-<i>d</i>) δ 7.25–7.19 (m, 1H), 6.91–6.86 (m, 2H), 6.81–6.77 (m, 1H), 4.82 (s, 1H), 4.15 (d, <i>J</i> = 13.1 Hz, 1H), 3.81 (s, 3H), 3.57 (s, 2H), 2.78 (s, 1H), 2.60 (t, <i>J</i> = 8.1 Hz, 1H), 2.52 (d, <i>J</i> = 9.7 Hz, 1H), 2.37–2.18 (m, 2H), 1.67–1.53 (m, 1H), 1.43 (s, 9H).</div></div><div id="sec4_11_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> <i>tert</i>-Butyl (1-(4-Methoxybenzyl)pyrrolidin-3-yl)carbamate (<b>15e</b>)</h4><div class="NLM_p last"><b>15e</b> was synthesized in 58% yield according to the same procedure as <b>13c</b> using <i>tert</i>-butyl pyrrolidin-3-ylcarbamate (<b>11</b>, 1.3 mmol) and 4-methoxybenzyl bromide (2.6 mmol); <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 7.23–7.19 (m, 2H), 6.87–6.83 (m, 2H), 4.80 (s, 1H), 4.15 (s, 1H), 3.80 (s, 3H), 3.53 (s, 2H), 2.76 (d, <i>J</i> = 11.3 Hz, 1H), 2.63–2.55 (m, 1H), 2.49 (d, <i>J</i> = 9.7 Hz, 1H), 2.34–2.19 (m, 2H), 1.72–1.49 (m, 1H), 1.43 (s, 9H).</div></div><div id="sec4_11_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> <i>tert</i>-Butyl (8-Benzyl-8-azabicyclo[3.2.1]octan-3-yl)carbamate (<b>16a</b>)</h4><div class="NLM_p last">Purchased.</div></div><div id="sec4_11_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> <i>tert</i>-Butyl (8-(3-Chlorobenzyl)-8-azabicyclo[3.2.1]octan-3-yl)carbamate (<b>16b</b>)</h4><div class="NLM_p last"><b>16b</b> was synthesized in 55% yield according to the same procedure as <b>13c</b> using <i>tert-</i>butyl (8-azabicyclo[3.2.1]octan-3-yl)carbamate (<b>12</b>, 1.3 mmol) and 3-chlorobenzyl bromide (2.6 mmol); <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 7.39 (m, 1H), 7.24–7.19 (m, 3H), 4.32 (s, 1H), 3.80 (s, 1H), 3.50 (s, 2H), 3.17 (p, <i>J</i> = 3.0 Hz, 2H), 2.05–1.95 (m, 2H), 1.86–1.78 (m, 2H), 1.70 (d, <i>J</i> = 8.1 Hz, 2H), 1.53–1.46 (m, 2H), 1.44 (s, 9H).</div></div><div id="sec4_11_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> <i>tert</i>-Butyl (8-(4-Chlorobenzyl)-8-azabicyclo[3.2.1]octan-3-yl)carbamate (<b>16c</b>)</h4><div class="NLM_p last"><b>16c</b> was synthesized in 69% yield according to the same procedure as <b>13c</b> using <i>tert-</i>butyl (8-azabicyclo[3.2.1]octan-3-yl)carbamate (<b>12</b>, 1.3 mmol) and 4-chlorobenzyl bromide (2.6 mmol); <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 7.31–7.27 (m, 4H), 4.35–4.27 (m, 1H), 3.81 (d, <i>J</i> = 14.8 Hz, 1H), 3.48 (s, 2H), 3.17 (q, <i>J</i> = 3.7, 3.2 Hz, 2H), 2.00 (m, 2H), 1.85–1.77 (m, 2H), 1.74–1.65 (m, 2H), 1.47 (dd, <i>J</i> = 12.3, 2.7 Hz, 2H), 1.43 (s, 9H).</div></div><div id="sec4_11_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61"> 4-((1-Benzylpiperidin-4-yl)amino)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-5-carboxamide (<b>17a</b>)</h4><div class="NLM_p last">HCl (4 N) in dioxane (2 mL) was added to the solution of <b>13b</b> (2.0 mmol) in MeOH, and the mixtures were stirred for 1 h at room temperature for the Boc deprotection <i>in situ</i>. The mixture was dried by a rotary evaporator, and the produced residue was further dried on a high-vacuum condition. To a solution of the upper dried residue in NMP (2 mL) were added DIEA (10 equiv) and <b>7a</b> (0.5 mmol) and then stirred for 5 h at 180 °C under microwave irradiation. The resulting mixture was diluted with ethyl acetate, washed with brine, and dried over MgSO<sub>4</sub>. After the solvent was removed <i>in vacuo</i>, the residue was purified by silica gel chromatography (dichloromethane/MeOH = 10:1 elution) to give the titled product; Yield: 63%; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.45 (s, 1H), 9.70 (d, <i>J</i> = 7.9 Hz, 1H), 8.36 (s, 1H), 7.33 (s, 2H), 7.32 (s, 3H), 7.29–7.22 (m, 1H), 7.16–7.09 (m, 1H), 6.52–6.44 (m, 1H), 4.11 (q, <i>J</i> = 5.2 Hz, 1H), 3.99 (s, 1H), 3.17 (d, <i>J</i> = 5.2 Hz, 2H), 2.71 (s, 2H), 2.28 (s, 2H), 2.01 (d, <i>J</i> = 10.6 Hz, 2H), 1.52 (q, <i>J</i> = 9.1 Hz, 2H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 172.3, 150.4, 149.5, 145.6, 138.4, 128.9, 128.2, 127.0, 121.7, 104.5, 101.9, 101.8, 62.1, 51.0, 49.0, 32.7; HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>20</sub>H<sub>24</sub>N<sub>5</sub>O [M + H]<sup>+</sup> 350.1975, found 350.1980.</div></div><div id="sec4_11_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i62" class="anchor-spacer"></div><h4 class="article-section__title" id="_i62"> 4-((1-(3-Methoxybenzyl)piperidin-4-yl)amino)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-5-carboxamide (<b>17b</b>)</h4><div class="NLM_p last"><b>17b</b> was synthesized in 86% yield according to the same procedure as <b>17a</b> using <b>7a</b> (0.5 mmol) and <b>13d</b> (2.0 mmol); <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 7.83 (d, <i>J</i> = 6.3 Hz, 1H), 7.34–7.29 (m, 1H), 7.15 (d, <i>J</i> = 3.6 Hz, 1H), 7.04–6.94 (m, 3H), 6.79 (d, <i>J</i> = 3.6 Hz, 1H), 6.61 (d, <i>J</i> = 6.9 Hz, 1H), 4.24 (s, 2H), 4.05–3.90 (m, 1H), 3.74 (s, 3H), 3.58–3.47 (m, 2H), 3.19–3.08 (m, 1H), 2.28–2.16 (m, 2H), 2.01–1.81 (m, 2H); <sup>13</sup>C NMR (100 MHz, methanol-<i>d</i><sub>4</sub>) δ 170.8, 149.8, 148.9, 143.2, 137.9, 136.9, 130.3, 128.2, 128.1, 126.6, 121.2, 105.2, 103.6, 102.6, 77.3, 35.9, 51.3, 32.2, 26.4, 21.3; HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>21</sub>H<sub>26</sub>N<sub>5</sub>O<sub>2</sub> [M + H]<sup>+</sup> 380.2081, found 380.2084.</div></div><div id="sec4_11_26" class="NLM_sec NLM_sec_level_3"><div id="ac_i63" class="anchor-spacer"></div><h4 class="article-section__title" id="_i63"> 4-((1-Benzylpiperidin-4-yl)amino)-<i>N</i>-methyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-5-carboxamide (<b>17c</b>)</h4><div class="NLM_p last"><b>17c</b> was synthesized in 87% yield according to the same procedure as <b>17a</b> using <b>7b</b> (0.5 mmol) and <b>13b</b> (2.0 mmol); <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 8.25 (s, 1H), 7.51–7.36 (m, 5H), 7.26 (d, <i>J</i> = 3.6 Hz, 1H), 6.84 (d, <i>J</i> = 3.7 Hz, 1H), 4.45–4.33 (m, 1H), 4.29 (s, 2H), 3.61–3.46 (m, 2H), 3.32–3.20 (m, 2H), 2.79 (s, 3H), 2.45–2.27 (m, 2H), 2.22 (br. s, 1H), 1.91–1.74 (m, 2H); <sup>13</sup>C NMR (100 MHz, methanol-<i>d</i><sub>4</sub>) δ 161.4, 161.0, 138.6, 134.5, 131.0, 130.0, 129.1, 128.8, 123.9, 118.0, 115.1, 103.7, 60.3, 50.6, 47.9, 29.6, 25.4; HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>21</sub>H<sub>26</sub>N<sub>5</sub>O [M + H]<sup>+</sup> 364.2132, found 364.2136.</div></div><div id="sec4_11_27" class="NLM_sec NLM_sec_level_3"><div id="ac_i64" class="anchor-spacer"></div><h4 class="article-section__title" id="_i64"> 4-((1-(2-Methoxybenzyl)piperidin-4-yl)amino)-<i>N</i>-methyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-5-carboxamide (<b>17d</b>)</h4><div class="NLM_p last"><b>17d</b> was synthesized in 20% yield according to the same procedure as <b>17a</b> using <b>7b</b> (0.5 mmol) and <b>13c</b> (2.0 mmol); <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 8.21 (s, 1H), 7.38–7.28 (m, 2H), 7.13 (d, <i>J</i> = 3.4 Hz, 1H), 7.04–6.93 (m, 2H), 6.61 (d, <i>J</i> = 3.6 Hz, 1H), 4.20–4.08 (m, 1H), 3.87 (s, 3H), 3.75 (s, 2H), 3.05–2.94 (m, 2H), 2.89 (s, 3H), 2.62–2.50 (m, 2H), 2.22–2.12 (<i>m</i>, 2H), 1.80–1.69 (m, 2H); <sup>13</sup>C NMR (100 MHz, methanol-<i>d</i><sub>4</sub>) δ 171.3, 158.2, 149.4, 149.4, 143.6, 131.4, 129.0, 123.7, 121.6, 112.0, 110.4, 105.2, 103.5, 101.9, 55.6, 54.4, 51.0, 31.8, 31.4, 25.2; HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>22</sub>H<sub>28</sub>N<sub>5</sub>O<sub>2</sub> [M + H]<sup>+</sup> 394.2238, found 394.2240.</div></div><div id="sec4_11_28" class="NLM_sec NLM_sec_level_3"><div id="ac_i65" class="anchor-spacer"></div><h4 class="article-section__title" id="_i65"> 4-((1-(3-Methoxybenzyl)piperidin-4-yl)amino)-<i>N</i>-methyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-5-carboxamide (<b>17e</b>)</h4><div class="NLM_p last"><b>17e</b> was synthesized in 27% yield according to the same procedure as <b>17a</b> using <b>7b</b> (0.5 mmol) and <b>13d</b> (2.0 mmol); <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.44 (d, <i>J</i> = 7.9 Hz, 1H), 8.28 (s, 1H), 8.22 (d, <i>J</i> = 4.4 Hz, 1H), 7.23 (t, <i>J</i> = 8.0 Hz, 1H), 7.15–7.12 (m, 1H), 6.92–6.87 (m, 2H), 6.81 (d, <i>J</i> = 7.1 Hz, 1H), 6.48 (bs, 1H), 4.02–3.92 (m, 1H), 3.75 (s, 3H), 3.47 (s, 2H), 2.78–2.66 (m, 2H), 2.74 (d, <i>J</i> = 4.3 Hz, 3H), 2.30–2.22 (m, 2H), 2.05–1.97 (m, 2H), 1.58–1.47 (m, 2H); <sup>13</sup>C NMR (100 MHz, methanol-<i>d</i><sub>4</sub>) δ 168.0, 160.4, 152.1, 138.6, 134.5, 130.2, 130.1, 130.1, 123.9, 122.9, 121.5, 116.2, 115.5, 104.88, 103.7, 60.3, 54.5, 50.7, 29.6, 25.4; HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>22</sub>H<sub>28</sub>N<sub>5</sub>O<sub>2</sub> [M + H]<sup>+</sup> 394.2238, found 394.2242.</div></div><div id="sec4_11_29" class="NLM_sec NLM_sec_level_3"><div id="ac_i66" class="anchor-spacer"></div><h4 class="article-section__title" id="_i66"> 4-((1-(4-Methoxybenzyl)piperidin-4-yl)amino)-<i>N</i>-methyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-5-carboxamide (<b>17f</b>)</h4><div class="NLM_p last"><b>17f</b> was synthesized in 21% yield according to the same procedure as <b>17a</b> using <b>7b</b>(0.5 mmol) and <b>13e</b> (2.0 mmol); <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 8.23 (s, 1H), 7.27–7.23 (m, 3H), 7.03 (d, <i>J</i> = 3.5 Hz, 1H), 6.87 (d, <i>J</i> = 8.6 Hz, 2H), 6.53 (d, <i>J</i> = 3.5 Hz, 1H), 4.05–3.95 (m, 1H), 3.81 (s, 3H), 3.50 (s, 2H), 2.98 (s, 3H), 2.89–2.77 (m, 2H), 2.32–2.22 (m, 2H), 2.15–2.08 (m, 2H), 1.83–1.68 (m, 2H); <sup>13</sup>C NMR (100 MHz, chloroform-<i>d</i>) δ 170.7, 158.7, 149.9, 149.8, 143.4, 130.3, 121.0, 113.6, 105.1, 103.5, 102.8, 62.4, 55.3, 51.6, 32.8, 32.1, 26.7, 26.5; HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>22</sub>H<sub>28</sub>N<sub>5</sub>O<sub>2</sub> [M + H]<sup>+</sup> 394.2238, found 394.2243.</div></div><div id="sec4_11_30" class="NLM_sec NLM_sec_level_3"><div id="ac_i67" class="anchor-spacer"></div><h4 class="article-section__title" id="_i67"> 4-((1-(3,4-Dimethoxybenzyl)piperidin-4-yl)amino)-<i>N</i>-methyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-5-carboxamide (<b>17g</b>)</h4><div class="NLM_p last"><b>17g</b> was synthesized in 36% yield according to the same procedure as <b>17a</b> using <b>7b</b> (0.5 mmol) and <b>13g</b> (2.0 mmol); <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 8.36 (bs, 1H), 7.40–7.35 (m, 1H), 7.19–7.05 (m, 4H), 6.99–6.92 (m, 1H), 4.58–4.47 (m, 1H), 4.33 (br. s, 2H), 3.90 (s, 3H), 3.87 (s, 3H), 3.69–3.61 (m, 2H), 3.38–3.27 (m, 2H), 2.91 (s, 3H), 2.53–2.44 (m, 2H), 2.02–1.87 (m, 2H); <sup>13</sup>C NMR (100 MHz, methanol-<i>d</i><sub>4</sub>) δ 167.9, 150.7, 149.5, 138.5, 134.4, 124.1, 123.9, 121.1, 117.9, 115.0, 113.9, 111.5, 106.4, 104.9, 103.7, 60.2, 55.1, 55.0, 50.4, 29.6, 25.4; HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>23</sub>H<sub>30</sub>N<sub>5</sub>O<sub>3</sub> [M + H]<sup>+</sup> 424.2343, found 424.2347.</div></div><div id="sec4_11_31" class="NLM_sec NLM_sec_level_3"><div id="ac_i68" class="anchor-spacer"></div><h4 class="article-section__title" id="_i68"> 4-((1-(3-Cyanobenzyl)piperidin-4-yl)amino)-<i>N</i>-methyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-5-carboxamide (<b>17h</b>)</h4><div class="NLM_p last"><b>17h</b> was synthesized in 17% yield according to the same procedure as <b>17a</b> using <b>7b</b> (0.5 mmol) and <b>13f</b> (2.0 mmol); <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 8.38 (s, 1H), 7.99 (bs, 1H), 7.93–7.89 (m, 2H), 7.73 (t, <i>J</i> = 8.0 Hz, 1H), 7.38 (d, <i>J</i> = 4.0 Hz, 1H), 6.96 (d, <i>J</i> = 4.0 Hz, 1H), 4.62–4.52 (m, 1H), 4.50 (s, 2H), 3.67–3.50 (m, 2H), 3.50–3.38 (m, 2H), 2.93 (s, 3H), 2.52–2.40 (m, 2H), 2.18–1.90 (m, 2H); <sup>13</sup>C NMR (100 MHz, methanol-<i>d</i><sub>4</sub>) δ 168.1, 152.0, 138.9, 135.7, 135.7, 134.8, 134.7, 133.5, 130.6, 123.8, 117.5, 113.2, 106.5, 104.9, 103.7, 58.9, 48.4, 29.3, 25.4; HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>22</sub>H<sub>25</sub>N<sub>6</sub>O [M + H]<sup>+</sup> 389.2084, found 389.2084.</div></div><div id="sec4_11_32" class="NLM_sec NLM_sec_level_3"><div id="ac_i69" class="anchor-spacer"></div><h4 class="article-section__title" id="_i69"> <i>N</i>-Cyclopropyl-4-((1-benzylpiperidin-4-yl)amino)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-5-carboxamide (<b>17i</b>)</h4><div class="NLM_p last"><b>17i</b> was synthesized in 11% yield according to the same procedure as <b>17a</b> using <b>7c</b> (0.5 mmol) and <b>13b</b> (2.0 mmol); <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 9.34 (bs, 1H), 9.28 (d, <i>J</i> = 7.8 Hz, 1H), 8.12 (s, 1H), 7.38–7.30 (m, 4H), 7.30–7.27 (m, 1H), 7.03 (d, <i>J</i> = 3.6 Hz, 1H), 6.55 (d, <i>J</i> = 3.7 Hz, 1H), 6.18 (s, 1H), 4.08–3.91 (m, 1H), 3.56 (s, 2H), 2.94–2.77 (m, 2H), 2.34–2.20 (m, 2H), 2.16–2.06 (m, 2H), 2.08–1.97 (m, 1H), 1.81–1.71 (m, 2H), 0.92–0.85 (m, 2H), 0.67–0.59 (m, 2H); HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>23</sub>H<sub>28</sub>N<sub>5</sub>O [M + H]<sup>+</sup> 390.2288, found 390.2293.</div></div><div id="sec4_11_33" class="NLM_sec NLM_sec_level_3"><div id="ac_i70" class="anchor-spacer"></div><h4 class="article-section__title" id="_i70"> <i>N</i>-Cyclopropyl-4-((1-(3-methoxybenzyl)piperidin-4-yl)amino)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-5-carboxamide (<b>17j</b>)</h4><div class="NLM_p last"><b>17j</b> was synthesized in 20% yield according to the same procedure as <b>17a</b> using <b>7c</b> (0.5 mmol) and <b>13d</b> (2.0 mmol); <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 8.23 (bs, 1H), 7.37–7.29 (m, 1H), 7.05–6.93 (m, 3H), 6.82 (bs, 1H), 4.48–4.31 (m, 1H), 4.25 (bs, 2H), 3.60–3.48 (m, 2H), 3.30–3.20 (m, 2H), 2.84–2.70 (m, 1H), 2.43–2.31 (m, 2H), 1.90–1.73 (m, 2H), 0.80–0.69 (m, 2H), 0.60–0.49 (m, 2H); HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>24</sub>H<sub>30</sub>N<sub>5</sub>O<sub>2</sub> [M + H]<sup>+</sup> 420.2394, found 420.2403.</div></div><div id="sec4_11_34" class="NLM_sec NLM_sec_level_3"><div id="ac_i71" class="anchor-spacer"></div><h4 class="article-section__title" id="_i71"> 4-((1-Benzylpiperidin-4-yl)amino)-<i>N</i>-cyclopentyl-1<i>H</i>-pyrrolo[2,3-b]pyridine-5-carboxamide (<b>17k</b>)</h4><div class="NLM_p last"><b>17k</b> was synthesized in 23% yield according to the same procedure as <b>17a</b> using <b>7d</b> (0.5 mmol) and <b>13b</b> (2.0 mmol); <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 8.39 (s, 1H), 7.61–7.52 (m, 5H), 7.40 (d, <i>J</i> = 4.0 Hz, 1H), 6.97 (d, <i>J</i> = 4.0 Hz, 1H), 4.57–4.48 (m, 1H), 4.34 (s, 2H), 4.34–4.26 (m, 1H), 3.72–3.62 (m, 2H), 3.50–3.40 (m, 2H), 2.56–2.44 (m, 2H), 2.11–2.02 (m, 2H), 1.99–1.89 (m, 2H), 1.88–1.76 (m, 2H), 1.76–1.54 (m, 4H); HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>25</sub>H<sub>32</sub>N<sub>5</sub>O [M + H]<sup>+</sup> 418.2601, found 418.2602.</div></div><div id="sec4_11_35" class="NLM_sec NLM_sec_level_3"><div id="ac_i72" class="anchor-spacer"></div><h4 class="article-section__title" id="_i72"> <i>N</i>-Cyclopentyl-4-((1-(3-methoxybenzyl)piperidin-4-yl)amino)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-5-carboxamide (<b>17l</b>)</h4><div class="NLM_p last"><b>17l</b> was synthesized in 11% yield according to the same procedure as <b>17a</b> using <b>7d</b> (0.5 mmol) and <b>13d</b> (2.0 mmol); <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 8.40 (bs, 1H), 7.49–7.40 (m, 1H), 7.38 (d, <i>J</i> = 4.0 Hz, 1H), 7.18–7.09 (m, 3H), 6.95 (bs, 1H), 4.58–4.45 (m, 1H), 4.37 (s, 2H), 4.34–4.26 (m, 1H), 3.70–3.60 (m, 2H), 3.40–3.30 (m, 2H), 2.57–2.41 (m, 2H), 2.11–2.00 (m, 4H), 2.00–1.88 (m, 2H), 1.74–1.56 (m, 4H); HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>26</sub>H<sub>34</sub>N<sub>5</sub>O<sub>2</sub> [M + H]<sup>+</sup> 448.2707, found 448.2713.</div></div><div id="sec4_11_36" class="NLM_sec NLM_sec_level_3"><div id="ac_i73" class="anchor-spacer"></div><h4 class="article-section__title" id="_i73"> <i>N</i>-Methyl-4-((1-phenylpiperidin-4-yl)amino)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-5-carboxamide (<b>17m</b>)</h4><div class="NLM_p last"><b>17m</b> was synthesized in 30% yield according to the same procedure as <b>17a</b> using <b>7b</b> (0.5 mmol) and <b>13a</b> (2.0 mmol); <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 8.28 (s, 1H), 7.26 (t, <i>J</i> = 8.7 Hz, 2H), 7.15 (d, <i>J</i> = 3.7 Hz, 1H), 7.05 (d, <i>J</i> = 7.9 Hz, 2H), 6.86 (t, <i>J</i> = 7.3 Hz, 1H), 6.69 (d, <i>J</i> = 3.7 Hz, 1H), 4.30–4.21 (m, 1H), 3.67–3.58 (m, 2H), 3.37 (s, 2H), 3.13–3.05 (m, 2H), 2.89 (s, 3H), 2.32–2.24 (m, 2H), 1.88–1.77 (m, 2H); <sup>13</sup>C NMR (100 MHz, methanol-<i>d</i><sub>4</sub>) δ 171.3, 151.5, 149.5, 149.4, 143.6, 128.7, 121.6, 119.7, 116.8, 105.3, 103.5, 102.0, 60.1, 49.5, 32.4, 25.2; MS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>20</sub>H<sub>24</sub>N<sub>5</sub>O [M + H]<sup>+</sup> 350.1975, found 350.1979.</div></div><div id="sec4_11_37" class="NLM_sec NLM_sec_level_3"><div id="ac_i74" class="anchor-spacer"></div><h4 class="article-section__title" id="_i74"> 4-(((1-Benzylpiperidin-4-yl)methyl)amino)-<i>N</i>-methyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-5-carboxamide (<b>18a</b>)</h4><div class="NLM_p last"><b>18a</b> was synthesized in 31% yield according to the same procedure as <b>17a</b> using <b>7b</b> and <b>14a</b>; <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 8.17 (s, 1H), 7.37–7.25 (m, 5H), 7.08 (d, <i>J</i> = 3.7 Hz, 1H), 6.69 (d, <i>J</i> = 3.7 Hz, 1H), 5.49 (s, 1H), 3.65–3.57 (m, 4H), 3.03 (d, <i>J</i> = 11.9 Hz, 2H), 2.86 (s, 3H), 2.17 (t, <i>J</i> = 11.6 Hz, 2H), 1.92 (d, <i>J</i> = 12.9 Hz, 2H), 1.84–1.70 (m, 1H), 1.49–1.37 (m, 2H); <sup>13</sup>C NMR (100 MHz, methanol-<i>d</i><sub>4</sub>) δ 171.7, 151.4, 149.7, 144.1, 136.1, 130.4, 128.8, 128.2, 121.6, 106.4, 103.9, 103.0, 63.4, 53.4, 50.7, 36.6, 29.7, 26.6; HRMS calcd for C<sub>22</sub>H<sub>28</sub>N<sub>5</sub>O [M + H]<sup>+</sup> 378.2288, found 378.2293.</div></div><div id="sec4_11_38" class="NLM_sec NLM_sec_level_3"><div id="ac_i75" class="anchor-spacer"></div><h4 class="article-section__title" id="_i75"> 4-(((1-(3-Chlorobenzyl)piperidin-4-yl)methyl)amino)-<i>N</i>-methyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-5-carboxamide (<b>18b</b>)</h4><div class="NLM_p last"><b>18b</b> was synthesized in 52% yield according to the same procedure as <b>17a</b> using <b>7b</b> and <b>14b</b>; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.43 (s, 1H), 9.38 (t, <i>J</i> = 5.4 Hz, 1H), 8.27 (s, 1H), 8.19 (d, <i>J</i> = 4.7 Hz, 1H), 7.40–7.24 (m, 4H), 7.10 (dd, <i>J</i> = 3.6, 2.4 Hz, 1H), 6.60 (dd, <i>J</i> = 3.6, 1.9 Hz, 1H), 3.55–3.45 (m, 4H), 2.84 (d, <i>J</i> = 11.1 Hz, 2H), 2.73 (d, <i>J</i> = 4.4 Hz, 3H), 1.97 (s, 2H), 1.80–1.73 (m, 2H), 1.61 (s, 1H), 1.32 (q, <i>J</i> = 10.8 Hz, 2H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 170.2, 150.2, 150.1, 144.4, 141.0, 132.9, 130.0, 128.4, 127.4, 126.9, 121.3, 105.1, 102.5, 101.9, 61.3, 52.7, 49.7, 36.0, 29.4, 26.0; HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>22</sub>H<sub>27</sub>ClN<sub>5</sub>O [M + H]<sup>+</sup> 412.1899, found 412.1901.</div></div><div id="sec4_11_39" class="NLM_sec NLM_sec_level_3"><div id="ac_i76" class="anchor-spacer"></div><h4 class="article-section__title" id="_i76"> 4-(((1-(4-Chlorobenzyl)piperidin-4-yl)methyl)amino)-<i>N</i>-methyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-5-carboxamide (<b>18c</b>)</h4><div class="NLM_p last"><b>18c</b> was synthesized in 19% yield according to the same procedure as <b>17a</b> using <b>7b</b> and <b>14c</b>; <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 8.18 (s, 1H), 7.39 (s, 4H), 7.09 (d, <i>J</i> = 3.6 Hz, 1H), 6.70 (d, <i>J</i> = 3.7 Hz, 1H), 3.84–3.77 (m, 2H), 3.63 (d, <i>J</i> = 6.7 Hz, 2H), 3.20–3.11 (m, 2H), 2.87 (s, 3H), 2.43 (s, 2H), 2.02–1.95 (m, 2H), 1.85 (s, 1H), 1.55–1.43 (m, 2H); <sup>13</sup>C NMR (100 MHz, methanol-<i>d</i><sub>4</sub>) δ 171.1, 151.4, 148.3, 142.9, 134.6, 133.0, 131.9, 129.1, 121.7, 106.4, 103.9, 103.0, 61.7, 53.0, 50.3, 35.9, 29.0, 26.5; HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>22</sub>H<sub>27</sub>ClN<sub>5</sub>O [M + H]<sup>+</sup> 412.1899, found 412.1901.</div></div><div id="sec4_11_40" class="NLM_sec NLM_sec_level_3"><div id="ac_i77" class="anchor-spacer"></div><h4 class="article-section__title" id="_i77"> 4-(((1-(3-Methoxybenzyl)piperidin-4-yl)methyl)amino)-<i>N</i>-methyl-1<i>H</i>-pyrrolo[2,3-b]pyridine-5-carboxamide (<b>18d</b>)</h4><div class="NLM_p last"><b>18d</b> was synthesized in 17% yield according to the same procedure as <b>17a</b> using <b>7b</b> and <b>14d</b>; <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 8.18 (s, 1H), 7.25 (dd, <i>J</i> = 8.2, 7.5 Hz, 1H), 7.08 (d, <i>J</i> = 3.7 Hz, 1H), 6.96–6.85 (m, 3H), 6.69 (d, <i>J</i> = 3.7 Hz, 1H), 3.80 (s, 3H), 3.64–3.58 (m, 4H), 3.05 (d, <i>J</i> = 11.7 Hz, 2H), 2.86 (s, 3H), 2.22 (t, <i>J</i> = 11.9 Hz, 2H), 1.93 (d, <i>J</i> = 13.2 Hz, 2H), 1.84–1.72 (m, 1H), 1.45 (dd, <i>J</i> = 12.5, 3.8 Hz, 2H); <sup>13</sup>C NMR (100 MHz, methanol-<i>d</i><sub>4</sub>) δ 171.5, 160.1, 151.3, 149.5, 143.9, 137.5, 129.7, 122.6, 121.6, 115.7, 113.7, 106.4, 103.9, 103.0, 63.2, 55.5, 53.4, 50.6, 36.4, 29.6, 26.5; HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>23</sub>H<sub>30</sub>N<sub>5</sub>O<sub>2</sub> [M + H]<sup>+</sup> 408.2394, found 408.2392.</div></div><div id="sec4_11_41" class="NLM_sec NLM_sec_level_3"><div id="ac_i78" class="anchor-spacer"></div><h4 class="article-section__title" id="_i78"> 4-(((1-(4-Methoxybenzyl)piperidin-4-yl)methyl)amino)-<i>N</i>-methyl-1<i>H</i>-pyrrolo[2,3-b]pyridine-5-carboxamide (<b>18e</b>)</h4><div class="NLM_p last"><b>18e</b> was synthesized in 40% yield according to the same procedure as <b>17a</b> using <b>7b</b> and <b>14e</b>; <sup>1</sup>H NMR (400 MHz, methanol-<i>d<sub>4</sub></i>) δ 8.13 (s, 1H), 7.25–7.20 (m, 2H), 7.09 (d, <i>J</i> = 3.6 Hz, 1H), 6.89–6.84 (m, 2H), 6.61 (d, <i>J</i> = 3.7 Hz, 1H), 3.69 (s, 3H), 3.57 (s, 2H), 3.46 (d, <i>J</i> = 6.5 Hz, 2H), 2.92 (d, <i>J</i> = 11.3 Hz, 2H), 2.70 (s, 3H), 2.16 (s, 2H), 1.76 (d, <i>J</i> = 13.1 Hz, 2H), 1.64 (s, 1H), 1.30 (q, <i>J</i> = 12.3 Hz, 2H); <sup>13</sup>C NMR (100 MHz, methanol-<i>d</i><sub>4</sub>) δ 172.0, 160.4, 151.5, 150.0, 144.4, 132.2, 126.8, 122.0, 114.5, 106.6, 104.1, 103.0, 62.4, 55.6, 53.2, 50.6, 36.4, 29.4, 26.6; HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>23</sub>H<sub>30</sub>N<sub>5</sub>O<sub>2</sub> [M + H]<sup>+</sup> 408.2394, found 408.2392.</div></div><div id="sec4_11_42" class="NLM_sec NLM_sec_level_3"><div id="ac_i79" class="anchor-spacer"></div><h4 class="article-section__title" id="_i79"> 4-((1-Benzylpyrrolidin-3-yl)amino)-<i>N</i>-methyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-5-carboxamide (<b>19a</b>)</h4><div class="NLM_p last"><b>19a</b> was synthesized in 16% yield according to the same procedure as <b>17a</b> using <b>7b</b> and <b>15a</b>; <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 8.18 (s, 1H), 7.40–7.23 (m, 5H), 7.10 (d, <i>J</i> = 3.7 Hz, 1H), 6.64 (d, <i>J</i> = 3.7 Hz, 1H), 4.77–4.70 (m, 1H), 3.74 (d, <i>J</i> = 1.7 Hz, 2H), 3.05 (dd, <i>J</i> = 10.1, 6.5 Hz, 1H), 2.88 (s, 3H), 2.86 (d, <i>J</i> = 10.0 Hz, 1H), 2.72–2.63 (m, 2H), 2.51–2.41 (m, 1H), 1.90–1.79 (m, 1H); <sup>13</sup>C NMR (100 MHz, methanol-<i>d</i><sub>4</sub>) δ 171.9, 150.4, 149.9, 144.3, 138.1, 129.7, 129.0, 128.1, 122.3, 106.3, 104.5, 102.6, 62.2, 60.8, 53.4, 53.3, 34.2, 26.6; HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>20</sub>H<sub>24</sub>N<sub>5</sub>O [M + H]<sup>+</sup> 350.1975, found 350.1979.</div></div><div id="sec4_11_43" class="NLM_sec NLM_sec_level_3"><div id="ac_i80" class="anchor-spacer"></div><h4 class="article-section__title" id="_i80"> 4-((1-(3-Chlorobenzyl)pyrrolidin-3-yl)amino)-<i>N</i>-methyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-5-carboxamide (<b>19b</b>)</h4><div class="NLM_p last"><b>19b</b> was synthesized in 12% yield according to the same procedure as <b>17a</b> using <b>7b</b> and <b>15b</b>; <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 8.19 (s, 1H), 7.43 (q, <i>J</i> = 1.4 Hz, 1H), 7.32–7.29 (m, 2H), 7.28–7.24 (m, 1H), 7.10 (d, <i>J</i> = 3.7 Hz, 1H), 6.65 (d, <i>J</i> = 3.7 Hz, 1H), 4.75–4.69 (m, 1H), 3.72 (s, 2H), 3.02–2.95 (m, 1H), 2.89 (s, 3H), 2.88–2.84 (m, 1H), 2.73–2.68 (m, 1H), 2.62 (td, <i>J</i> = 8.8, 6.1 Hz, 1H), 2.50–2.41 (m, 1H), 1.90–1.80 (m, 1H); <sup>13</sup>C NMR (100 MHz, methanol-<i>d</i><sub>4</sub>) δ 171.3, 150.1, 149.3, 143.7, 140.6, 134.5, 130.0, 129.2, 127.7, 127.3, 121.8, 105.9, 104.1, 102.5, 62.0, 59.9, 53.1, 53.0, 34.1, 26.5; HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>20</sub>H<sub>23</sub>ClN<sub>5</sub>O [M + H]<sup>+</sup> 384.1586, found 384.1588.</div></div><div id="sec4_11_44" class="NLM_sec NLM_sec_level_3"><div id="ac_i81" class="anchor-spacer"></div><h4 class="article-section__title" id="_i81"> 4-((1-(4-Chlorobenzyl)pyrrolidin-3-yl)amino)-<i>N</i>-methyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-5-carboxamide (<b>19c</b>)</h4><div class="NLM_p last"><b>19c</b> was synthesized in 34% yield according to the same procedure as <b>17a</b> using <b>7b</b> and <b>15c</b>; <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 8.18 (s, 1H), 7.40–7.30 (m, 4H), 7.10 (d, <i>J</i> = 3.7 Hz, 1H), 6.64 (d, <i>J</i> = 3.7 Hz, 1H), 4.76–4.69 (m, 1H), 3.71 (s, 2H), 2.99 (dd, <i>J</i> = 10.0, 6.4 Hz, 1H), 2.91–2.83 (m, 1H), 2.89 (s, 1H), 2.72–2.66 (m, 1H), 2.66–2.57 (m, 1H), 2.51–2.40 (m, 1H), 1.89–1.81 (m, 1H); <sup>13</sup>C NMR (100 MHz, methanol-<i>d</i><sub>4</sub>) δ 171.1, 150.0, 149.2, 143.6, 136.9, 133.1, 130.5, 128.6, 121.6, 105.8, 103.9, 102.4, 61.9, 59.6, 53.0, 52.9, 34.0, 26.4; HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>20</sub>H<sub>23</sub>ClN<sub>5</sub>O [M + H]<sup>+</sup> 384.1586, found 384.1588.</div></div><div id="sec4_11_45" class="NLM_sec NLM_sec_level_3"><div id="ac_i82" class="anchor-spacer"></div><h4 class="article-section__title" id="_i82"> 4-((1-(3-Methoxybenzyl)pyrrolidin-3-yl)amino)-<i>N</i>-methyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-5-carboxamide (<b>19d</b>)</h4><div class="NLM_p last"><b>19d</b> was synthesized in 34% yield according to the same procedure as <b>17a</b> using <b>7b</b> and <b>15d</b>; <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 8.18 (s, 1H), 7.25–7.19 (m, 1H), 7.10 (d, <i>J</i> = 3.6 Hz, 1H), 6.97 (dd, <i>J</i> = 2.6, 1.4 Hz, 1H), 6.95–6.92 (m, 1H), 6.86–6.77 (m, 1H), 6.65 (d, <i>J</i> = 3.7 Hz, 1H), 4.77–4.69 (m, 1H), 3.78 (s, 3H), 3.75–3.64 (m, 2H), 3.01 (dd, <i>J</i> = 10.1, 6.3 Hz, 1H), 2.93–2.83 (m, 1H), 2.88 (s, 1H), 2.72–2.62 (m, 2H), 2.52–2.41 (m, 1H), 1.90–1.79 (m, 1H); <sup>13</sup>C NMR (100 MHz, methanol-<i>d</i><sub>4</sub>) δ 171.0, 159.8, 150.0, 149.1, 143.4, 139.6, 129.5, 121.6, 121.5, 114.4, 113.1, 105.7, 103.9, 102.4, 61.8, 60.3, 55.3, 52.9, 33.9, 26.4; HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>21</sub>H<sub>26</sub>N<sub>5</sub>O<sub>2</sub> [M + H]<sup>+</sup> 380.2081, found 380.2076.</div></div><div id="sec4_11_46" class="NLM_sec NLM_sec_level_3"><div id="ac_i83" class="anchor-spacer"></div><h4 class="article-section__title" id="_i83"> 4-((1-(4-Methoxybenzyl)pyrrolidin-3-yl)amino)-<i>N</i>-methyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-5-carboxamide (<b>19e</b>)</h4><div class="NLM_p last"><b>19e</b> was synthesized in 19% yield according to the same procedure as <b>17a</b> using <b>7b</b> and <b>15e</b>; <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 8.19 (s, 1H), 7.33–7.26 (m, 2H), 7.11 (d, <i>J</i> = 3.7 Hz, 1H), 6.91–6.86 (m, 2H), 6.64 (d, <i>J</i> = 3.7 Hz, 1H), 4.77–4.69 (m, 1H), 3.78 (s, 3H), 3.74 (s, 2H), 3.12 (dd, <i>J</i> = 10.4, 6.5 Hz, 1H), 2.96–2.89 (m, 1H), 2.88 (s, 3H), 2.79–2.71 (m, 2H), 2.53–2.43 (m, 1H), 1.92–1.81 (m, 1H); <sup>13</sup>C NMR (100 MHz, methanol-<i>d</i><sub>4</sub>) δ 171.4, 159.5, 150.1, 149.4, 143.8, 130.8, 122.0, 114.2, 106.0, 104.2, 102.4, 61.6, 59.8, 55.5, 53.0, 52.9, 33.9, 26.5; HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>21</sub>H<sub>26</sub>N<sub>5</sub>O<sub>2</sub> [M + H]<sup>+</sup> 380.2081, found 380.2079.</div></div><div id="sec4_11_47" class="NLM_sec NLM_sec_level_3"><div id="ac_i84" class="anchor-spacer"></div><h4 class="article-section__title" id="_i84"> 4-((8-Benzyl-8-azabicyclo[3.2.1]octan-3-yl)amino)-<i>N</i>-methyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-5-carboxamide (<b>20a</b>)</h4><div class="NLM_p last"><b>20a</b> was synthesized in 37% yield according to the same procedure as <b>17a</b> using <b>7b</b> and <b>16a</b>; <sup>1</sup>H NMR (400 MHz, methanol-<i>d<sub>4</sub></i>) δ 8.17 (s, 1H), 7.46–7.42 (m, 2H), 7.39–7.32 (m, 2H), 7.32–7.26 (m, 1H), 7.15 (d, <i>J</i> = 3.6 Hz, 1H), 6.64 (d, <i>J</i> = 3.7 Hz, 1H), 4.47–4.36 (m, 1H), 3.73 (s, 2H), 3.42 (s, 2H), 2.87 (s, 3H), 2.30–2.21 (m, 2H), 2.14–2.07 (m, 2H), 1.95–1.88 (m, 2H), 1.77 (t, <i>J</i> = 11.2 Hz, 2H); <sup>13</sup>C NMR (100 MHz, methanol-<i>d</i><sub>4</sub>) δ 170.8, 150.0, 149.1, 143.4, 139.1, 128.8, 128.4, 127.1, 121.4, 105.2, 103.9, 102.5, 58.6, 55.4, 45.5, 38.4, 26.9, 26.4; HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>23</sub>H<sub>28</sub>N<sub>5</sub>O [M + H]<sup>+</sup> 390.2288, found 390.2293.</div></div><div id="sec4_11_48" class="NLM_sec NLM_sec_level_3"><div id="ac_i85" class="anchor-spacer"></div><h4 class="article-section__title" id="_i85"> 4-((8-(3-Chlorobenzyl)-8-azabicyclo[3.2.1]octan-3-yl)amino)-<i>N</i>-methyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-5-carboxamide (<b>20b</b>)</h4><div class="NLM_p last"><b>20b</b> was synthesized in 52% yield according to the same procedure as <b>17a</b> using <b>7b</b> and <b>16b</b>; <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 8.16 (s, 1H), 7.49 (s, 1H), 7.37–7.25 (m, 3H), 7.14 (d, <i>J</i> = 3.6 Hz, 1H), 6.65 (d, <i>J</i> = 3.7 Hz, 1H), 4.45–4.34 (m, 1H), 3.66 (s, 2H), 2.87 (s, 3H), 2.26–2.17 (m, 2H), 2.14–2.06 (m, 2H), 1.93–1.84 (m, 2H), 1.75 (t, <i>J</i> = 11.0 Hz, 2H); <sup>13</sup>C NMR (100 MHz, methanol-<i>d</i><sub>4</sub>) δ 171.7, 150.3, 149.7, 144.2, 141.7, 134.6, 130.0, 129.2, 127.6, 127.4, 122.0, 105.7, 104.3, 102.6, 59.1, 55.4, 45.7, 38.9, 27.1, 26.5; HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>23</sub>H<sub>27</sub>ClN<sub>5</sub>O [M + H]<sup>+</sup> 424.1899, found 424.1903.</div></div><div id="sec4_11_49" class="NLM_sec NLM_sec_level_3"><div id="ac_i86" class="anchor-spacer"></div><h4 class="article-section__title" id="_i86"> 4-((8-(4-Chlorobenzyl)-8-azabicyclo[3.2.1]octan-3-yl)amino)-<i>N</i>-methyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-5-carboxamide (<b>20c</b>)</h4><div class="NLM_p last"><b>20c</b> was synthesized in 39% yield according to the same procedure as <b>17a</b> using <b>7b</b> and <b>16c</b>; <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 8.17 (s, 1H), 7.46–7.35 (m, 4H), 7.15 (d, <i>J</i> = 3.6 Hz, 1H), 6.65 (d, <i>J</i> = 3.7 Hz, 1H), 4.42 (m, 1H), 3.73 (s, 2H), 3.43 (s, 2H), 2.86 (s, 3H), 2.29–2.22 (m, 2H), 2.17–2.11 (m, 2H), 1.94 (d, <i>J</i> = 7.8 Hz, 2H), 1.76 (t, <i>J</i> = 12.1 Hz, 2H); <sup>13</sup>C NMR (100 MHz, methanol-<i>d</i><sub>4</sub>) δ 171.6, 150.3, 149.6, 144.0, 137.5, 133.4, 130.7, 128.9, 122.1, 105.7, 104.3, 102.6, 59.1, 55.0, 45.7, 38.7, 27.0, 26.5; HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>23</sub>H<sub>27</sub>ClN<sub>5</sub>O [M + H]<sup>+</sup> 424.1899, found 424.1903.</div></div><div id="sec4_11_50" class="NLM_sec NLM_sec_level_3"><div id="ac_i87" class="anchor-spacer"></div><h4 class="article-section__title" id="_i87"> 4-((8-(3-Methoxybenzyl)-8-azabicyclo[3.2.1]octan-3-yl)amino)-<i>N</i>-methyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-5-carboxamide (<b>20d</b>)</h4><div class="NLM_p last">To a solution of <b>25</b> (0.25 mmol) and 3-methoxybenzaldehyde (0.5 mmol) in DCM (5 mL) were added triethylamine (0.5 mmol) and sodium triacetoxyborohydride (0.5 mmol). After stirring for 24 h at the room temperature, the mixture was diluted with ethyl acetate, washed with brine, and dried over MgSO<sub>4</sub>. After the solvent was removed <i>in vacuo</i>, the residue was purified by silica gel flash chromatography (dichloromethane/MeOH = 10:1 elution) to give the titled product; Yield: 59% (endo/exo = 1:1); NMR result of endo and exo mixture <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 9.62 (d, <i>J</i> = 7.4 Hz, 1H), 9.41 (s, 1H), 9.33 (s, 1H), 9.01 (d, <i>J</i> = 8.1 Hz, 1H), 8.20 (s, 1H), 8.19 (s, 1H), 7.25–7.21 (m, 2H), 7.07–6.96 (m, 6H), 6.80 (d, <i>J</i> = 2.5 Hz, 1H), 6.78 (s, 1H), 6.64 (d, <i>J</i> = 3.7 Hz, 1H), 6.51 (d, <i>J</i> = 3.6 Hz, 1H), 6.02 (s, 2H), 4.33 (q, <i>J</i> = 6.4 Hz, 2H), 3.83 (s, 3H), 3.82 (s, 3H), 3.60 (s, 2H), 3.58 (s, 2H), 3.30 (s, 2H), 3.24 (s, 2H), 2.99 (t, <i>J</i> = 5.0 Hz, 6H), 2.34 (dd, <i>J</i> = 8.0, 4.0 Hz, 2H), 2.11–2.00 (m, 4H), 1.93–1.74 (m, 10H); <sup>13</sup>C NMR (100 MHz, chloroform-<i>d</i>) δ 170.9, 159.7, 150.0, 149.7, 149.5, 149.4, 143.7, 143.6, 141.4, 141.1, 129.3, 129.2, 121.3, 121.2, 121.0, 120.9, 114.3, 114.0, 112.6, 112.4, 105.2, 105.1, 103.8, 103.3, 103.0, 102.6, 58.6, 58.1, 55.6, 55.3, 45.6, 44.7, 38.7, 37.7, 26.9, 25.8; HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>24</sub>H<sub>30</sub>N<sub>5</sub>O<sub>2</sub> [M + H]<sup>+</sup> 420.2394, found 420.2395.</div></div><div id="sec4_11_51" class="NLM_sec NLM_sec_level_3"><div id="ac_i88" class="anchor-spacer"></div><h4 class="article-section__title" id="_i88"> 4-((8-(4-Methoxybenzyl)-8-azabicyclo[3.2.1]octan-3-yl)amino)-<i>N</i>-methyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-5-carboxamide (<b>20e</b>)</h4><div class="NLM_p last"><b>20e</b> was synthesized in 71% (endo/exo = 1:1) yield according to the same procedure as <b>20d</b> using <b>25</b> and 4-methoxybenzaldehyde; the NMR result of the endo and exo mixture: <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 9.62 (d, <i>J</i> = 7.4 Hz, 1H), 9.60 (s, 1H), 9.57 (s, 1H), 9.00 (d, <i>J</i> = 8.4 Hz, 1H), 8.19 (d, <i>J</i> = 4.5 Hz, 2H), 7.30 (d, <i>J</i> = 8.6 Hz, 4H), 7.06 (d, <i>J</i> = 3.6 Hz, 1H), 7.00 (d, <i>J</i> = 3.5 Hz, 1H), 6.87 (dd, <i>J</i> = 8.6, 1.9 Hz, 4H), 6.63 (d, <i>J</i> = 3.7 Hz, 1H), 6.50 (d, <i>J</i> = 3.6 Hz, 1H), 6.03 (s, 1H), 4.32 (d, <i>J</i> = 6.8 Hz, 2H), 3.81 (s, 6H), 3.55 (s, 2H), 3.1 (s, 2H), 3.28 (s, 2H), 3.23 (s, 2H), 2.99 (dd, <i>J</i> = 6.5, 4.5 Hz, 6H), 2.31 (ddd, <i>J</i> = 14.2, 6.3, 3.6 Hz, 2H), 2.12 (d, <i>J</i> = 12.0 Hz, 6H), 2.02 (dd, <i>J</i> = 7.9, 3.6 Hz, 2H), 1.90 (d, <i>J</i> = 13.7 Hz, 2H), 1.92–1.77 (m, 6H); <sup>13</sup>C NMR (100 MHz, chloroform-<i>d</i>) δ 171.2, 171.1, 158.9, 158.8, 150.0, 149.9, 149.5, 149.4, 143.8, 131.4, 130.9, 130.3, 130.1, 121.4, 121.0, 113.9, 113.8, 105.3, 105.2, 103.9, 103.3, 103.0, 102.5, 58.3, 57.8, 55.4, 54.8, 45.6, 44.8, 38.5, 37.6, 26.9, 26.5, 26.4, 25.8; HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>24</sub>H<sub>30</sub>N<sub>5</sub>O<sub>2</sub> [M + H]<sup>+</sup> 420.2394, found 420.2410.</div></div><div id="sec4_11_52" class="NLM_sec NLM_sec_level_3"><div id="ac_i89" class="anchor-spacer"></div><h4 class="article-section__title" id="_i89"> 4-((8-(4-Cyanobenzyl)-8-azabicyclo[3.2.1]octan-3-yl)amino)-<i>N</i>-methyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-5-carboxamide (<b>20f</b>)</h4><div class="NLM_p last"><b>20f</b> was synthesized in 68% (endo/exo = 1:1) yield according to the same procedure as <b>20d</b> using <b>25</b> and 4-formylbenzonitrile; the NMR result of the endo and exo mixture: <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 9.69 (d, <i>J</i> = 7.5 Hz, 1H), 9.10 (d, <i>J</i> = 7.9 Hz, 1H), 8.23 (s, 1H), 8.22 (s, 1H), 7.62 (d, <i>J</i> = 8.1 Hz, 4H), 7.54 (d, <i>J</i> = 7.2 Hz, 4H), 7.05 (d, <i>J</i> = 3.6 Hz, 1H), 7.01–6.92 (m, 1H), 6.59 (d, <i>J</i> = 3.5 Hz, 1H), 6.47 (d, <i>J</i> = 3.5 Hz, 1H), 6.24 (s, 2H), 4.32 (q, <i>J</i> = 6.5 Hz, 2H), 3.67 (s, 2H), 3.63 (s, 2H), 3.25 (s, 2H), 3.20 (s, 2H), 3.00 (dd, <i>J</i> = 4.8, 3.3 Hz, 6H), 2.37–2.26 (m, 2H), 2.22–2.09 (m, 6H), 2.05 (dd, <i>J</i> = 7.9, 3.3 Hz, 2H), 1.94 (d, <i>J</i> = 13.4 Hz, 2H), 1.81 (dd, <i>J</i> = 15.6, 9.4 Hz, 5H); <sup>13</sup>C NMR (100 MHz, methanol-<i>d</i><sub>4</sub>) δ 171.1, 171.0, 150.1, 149.7, 149.1, 148.9, 145.9, 143.5, 143.4, 132.3, 132.2, 129.4, 129.2, 121.5, 121.2, 119.1, 119.0, 110.7, 110.4, 105.3, 105.2, 103.9, 103.3, 102.9, 102.4, 58.9, 58.5, 56.3, 55.4, 45.4, 44.5, 38.8, 37.6, 26.9, 26.5, 26.4, 25.8. HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>24</sub>H<sub>27</sub>N<sub>6</sub>O [M + H]<sup>+</sup> 415.2241, found 415.2245.</div></div><div id="sec4_11_53" class="NLM_sec NLM_sec_level_3"><div id="ac_i90" class="anchor-spacer"></div><h4 class="article-section__title" id="_i90"> 4-((8-Azabicyclo[3.2.1]octan-3-yl)amino)-<i>N</i>-methyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-5-carboxamide Hydrochloride (<b>25</b>)</h4><div class="NLM_p last">To a solution of <b>21</b> (2.0 mmol) in NMP (2 mL) were added DIEA (2.0 mmol) and 4-chloro-<i>N</i>-methyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-5-carboxamide (1.0 mmol) and then stirred for 5 h at 180 °C under microwave irradiation. The resulting mixture was diluted with ethyl acetate, washed with brine, and dried over MgSO<sub>4</sub>. After the solvent was removed <i>in vacuo</i>, the residue was purified by silica gel chromatography (dichloromethane/MeOH = 10:1 elution) to give the protected and unprotected products as a mixture. HCl (4 N) in dioxane (2 mL) was added to the solution of the mixture in MeOH, and the mixtures were stirred for 1 h at room temperature for the Boc deprotection <i>in situ</i>. After evaporation and additional drying on a high-vacuum condition, the product <b>25</b> was used at next step without further purification. Yield: 51% (endo/exo = 1:1); <sup>1</sup>H NMR (400 MHz, deuterium oxide) δ 8.29 (s, 1H), 8.23 (s, 1H), 7.39 (d, <i>J</i> = 3.7 Hz, 1H), 7.34 (d, <i>J</i> = 3.7 Hz, 1H), 6.86 (d, <i>J</i> = 3.7 Hz, 1H), 6.77 (d, <i>J</i> = 3.7 Hz, 1H), 4.72–4.61 (m, 2H), 4.29 (s, 2H), 4.23 (s, 2H), 2.95 (d, <i>J</i> = 3.1 Hz, 6H), 2.60 (dd, <i>J</i> = 10.4, 3.8 Hz, 2H), 2.48 (d, <i>J</i> = 13.7 Hz, 2H), 2.40–2.21 (m, 10H), 2.01 (t, <i>J</i> = 11.8 Hz, 2H); <sup>13</sup>C NMR (100 MHz, deuterium oxide) δ 171.1, 170.9, 153.7, 153.6, 140.3, 140.1, 136.6, 136.4, 126.6, 126.3, 108.6, 108.6, 108.6, 108.0, 107.0, 106.5, 106.2, 57.4, 56.1, 46.9, 46.0, 37.6, 35.7, 28.9, 28.8, 28.0, 27.4; HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>16</sub>H<sub>22</sub>N<sub>5</sub>O [M + H]<sup>+</sup> 300.1819, found 300.1821.</div></div><div id="sec4_11_54" class="NLM_sec NLM_sec_level_3"><div id="ac_i91" class="anchor-spacer"></div><h4 class="article-section__title" id="_i91"> 4-(Azepan-4-ylamino)-<i>N</i>-methyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-5-carboxamide Hydrochloride (<b>26</b>)</h4><div class="NLM_p last"><b>26</b> was synthesized in 38% yield according to the same procedure as <b>25</b> using <b>7b</b> and <b>21</b>; <sup>1</sup>H NMR (400 MHz, deuterium oxide) δ 8.25–8.21 (m, 1H), 7.38–7.34 (m, 1H), 6.85 (d, <i>J</i> = 3.6 Hz, 1H), 4.61–4.51 (m, 1H), 3.51 (dd, <i>J</i> = 13.5, 6.9 Hz, 1H), 3.46–3.33 (m, 3H), 2.97–2.92 (m, 3H), 2.54–2.36 (m, 2H), 2.21–2.07 (m, 2H), 2.06–1.84 (m, 2H); <sup>13</sup>C NMR (100 MHz, deuterium oxide) δ 168.6, 150.9, 137.8, 133.9, 123.6, 106.1, 105.4, 103.8, 52.3, 46.0, 41.0, 32.8, 30.7, 26.2, 19.9; HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>15</sub>H<sub>22</sub>N<sub>5</sub>O [M + H]<sup>+</sup> 288.1819, found 288.1821.</div></div><div id="sec4_11_55" class="NLM_sec NLM_sec_level_3"><div id="ac_i92" class="anchor-spacer"></div><h4 class="article-section__title" id="_i92"> <i>N</i>-Methyl-4-((<i>trans</i>-2-methylpiperidin-4-yl)amino)-1<i>H</i>-pyrrolo[2,3-b]pyridine-5-carboxamide Hydrochloride (<b>27a</b>)</h4><div class="NLM_p last"><b>27a</b> was synthesized in 38% yield according to the same procedure as <b>25</b> using <b>7b</b> and <b>23a</b>; <sup>1</sup>H NMR (400 MHz, deuterium oxide) δ 8.10 (d, <i>J</i> = 0.9 Hz, 1H), 7.19 (dd, <i>J</i> = 3.7, 0.8 Hz, 1H), 6.51 (dd, <i>J</i> = 3.8, 0.9 Hz, 1H), 4.43–4.37 (m, 1H), 3.51–3.41 (m, 1H), 3.42–3.35 (m, 1H), 3.23–3.11 (m, 1H), 2.90 (d, <i>J</i> = 0.8 Hz, 3H), 2.15–2.00 (m, 2H), 1.99–1.88 (m, 1H), 1.48–1.43 (m, 1H), 1.35 (d, <i>J</i> = 6.5 Hz, 3H); <sup>13</sup>C NMR (100 MHz, deuterium oxide) δ 171.4, 148.6, 148.5, 143.5, 122.4, 104.9, 103.1, 101.7, 47.6, 44.2, 39.0, 34.6, 27.5, 26.7, 26.0; HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>15</sub>H<sub>22</sub>N<sub>5</sub>O [M + H]<sup>+</sup> 288.1819, found 288.1821.</div></div><div id="sec4_11_56" class="NLM_sec NLM_sec_level_3"><div id="ac_i93" class="anchor-spacer"></div><h4 class="article-section__title" id="_i93"> <i>N</i>-Methyl-4-((<i>cis</i>-2-methylpiperidin-4-yl)amino)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-5-carboxamide Hydrochloride (<b>27b</b>)</h4><div class="NLM_p last"><b>27b</b> was synthesized in 47% yield according to the same procedure as <b>25</b> using <b>7b</b> and <b>23b</b>; <sup>1</sup>H NMR (400 MHz, deuterium oxide) δ 8.26–8.22 (m, 1H), 7.39–7.33 (m, 1H), 6.87 (d, <i>J</i> = 3.3 Hz, 1H), 4.58–4.49 (m, 1H), 3.67–3.60 (m, 1H), 3.60–3.53 (m, 1H), 3.35–3.26 (m, 1H), 2.95–2.92 (m, 3H), 2.50 (t, <i>J</i> = 13.3 Hz, 2H), 1.93–1.81 (m, 1H), 1.78–1.66 (m, 1H), 1.43 (d, <i>J</i> = 6.1 Hz, 3H); <sup>13</sup>C NMR (100 MHz, deuterium oxide) δ 168.5, 151.0, 137.9, 133.9, 123.9, 106.1, 105.6, 103.7, 51.6, 48.7, 42.5, 36.4, 28.3, 26.2, 18.2; HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>15</sub>H<sub>22</sub>N<sub>5</sub>O [M + H]<sup>+</sup> 288.1819, found 288.1821.</div></div><div id="sec4_11_57" class="NLM_sec NLM_sec_level_3"><div id="ac_i94" class="anchor-spacer"></div><h4 class="article-section__title" id="_i94"> 4-((<i>cis</i>-2-Methylpiperidin-4-yl)amino)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-5-carboxamide Hydrochloride (<b>27c</b>)</h4><div class="NLM_p last"><b>27c</b> was synthesized in 38% yield according to the same procedure as <b>25</b> using <b>7a</b> and <b>23b</b>; <sup>1</sup>H NMR (400 MHz, deuterium oxide) δ 8.22 (s, 1H), 7.21 (d, <i>J</i> = 3.7 Hz, 1H), 6.70 (d, <i>J</i> = 3.7 Hz, 1H), 4.35 (t, <i>J</i> = 11.4, 3.9 Hz, 1H), 3.52–3.45 (m, 1H), 3.45–3.37 (m, 1H), 3.19–3.09 (m, 1H), 2.40–2.29 (m, 2H), 1.80–1.65 (m, 1H), 1.65–1.51 (m, 1H), 1.29 (d, <i>J</i> = 6.5 Hz, 3H); <sup>13</sup>C NMR (100 MHz, deuterium oxide) δ 171.2, 151.6, 138.2, 135.2, 123.7, 106.2, 104.2, 103.9, 51.6, 48.7, 42.5, 36.5, 28.3, 18.2; HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>14</sub>H<sub>20</sub>N<sub>5</sub>O [M + H]<sup>+</sup> 274.1662, found 274.1663.</div></div><div id="sec4_11_58" class="NLM_sec NLM_sec_level_3"><div id="ac_i95" class="anchor-spacer"></div><h4 class="article-section__title" id="_i95"> 4-((<i>trans</i>-3-Fluoropiperidin-4-yl)amino)-<i>N</i>-methyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-5-carboxamide Hydrochloride (<b>28a</b>)</h4><div class="NLM_p last"><b>28a</b> was synthesized in 27% yield according to the same procedure as <b>25</b> using <b>7b</b> and <b>24a</b>; <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 8.46 (s, 1H), 7.40 (d, <i>J</i> = 3.7 Hz, 1H), 7.01 (d, <i>J</i> = 3.7 Hz, 1H), 5.17–5.06 (m, 1H), 5.04–4.94 (m, 1H), 3.63–3.57 (m, 2H), 2.93 (s, 3H), 2.61–2.49 (m, 2H), 2.17–2.06 (m, 2H); <sup>13</sup>C NMR (100 MHz, methanol-<i>d</i><sub>4</sub>) δ 169.3, 154.3, 139.9, 135.9, 125.5, 108.4, 106.5, 105.2, 87.3 (d, <i>J</i><sub>C–F</sub> = 181.0 Hz), 51.8 (d, <i>J</i><sub>C–F</sub> = 23.8 Hz), 44.8 (d, <i>J</i><sub>C–F</sub> = 26.3 Hz), 41.6, 26.8, 26.2; MS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>14</sub>H<sub>19</sub>FN<sub>5</sub>O [M + H]<sup>+</sup> 292.16, found 292.10.</div></div><div id="sec4_11_59" class="NLM_sec NLM_sec_level_3"><div id="ac_i96" class="anchor-spacer"></div><h4 class="article-section__title" id="_i96"> 4-((<i>cis</i>-3-Fluoropiperidin-4-yl)amino)-<i>N</i>-methyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-5-carboxamide Hydrochloride (<b>28b</b>)</h4><div class="NLM_p last"><b>28b</b> was synthesized in 79% yield according to the same procedure as <b>25</b> using <b>7b</b> and <b>24b</b>; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.52 (s, 1H), 9.59 (d, <i>J</i> = 8.7 Hz, 1H), 8.31 (s, 1H), 8.23 (d, <i>J</i> = 4.4 Hz, 1H), 7.16 (d, <i>J</i> = 3.5 Hz, 1H), 6.53 (d, <i>J</i> = 3.6 Hz, 1H), 4.71 (d, <i>J</i> = 50.5 Hz, 1H), 4.34–4.14 (m, 1H), 3.12 (t, <i>J</i> = 11.4 Hz, 1H), 2.90 (d, <i>J</i> = 14.9 Hz, 1H), 2.84–2.76 (m, 1H), 2.74 (d, <i>J</i> = 4.4 Hz, 3H), 2.65 (t, <i>J</i> = 12.0 Hz, 1H), 1.86–1.75 (m, 1H), 1.67–1.51 (m, 1H), 1.43–1.33 (m, 1H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 170.0, 150.3, 148.5, 144.5, 122.0, 104.6, 103.1, 101.5, 88.9 (d, <i>J</i><sub>C–F</sub> = 173.6 Hz), 51.7 (d, <i>J</i><sub>C–F</sub> = 18.1 Hz), 48.2 (d, <i>J</i><sub>C–F</sub> = 20.5 Hz), 29.6, 28.1, 26.1; MS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>14</sub>H<sub>19</sub>FN<sub>5</sub>O [M + H]<sup>+</sup> 292.16, found 292.10.</div></div><div id="sec4_11_60" class="NLM_sec NLM_sec_level_3"><div id="ac_i97" class="anchor-spacer"></div><h4 class="article-section__title" id="_i97"> 4-((1-Benzylazepan-4-yl)amino)-<i>N</i>-methyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-5-carboxamide (<b>30a</b>)</h4><div class="NLM_p last"><b>30a</b> was synthesized in 76% yield according to the same procedure as <b>20d</b> using <b>26</b> and benzaldehyde; <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 8.17 (s, 1H), 7.43–7.27 (m, 5H), 7.09 (d, <i>J</i> = 3.7 Hz, 1H), 6.62 (d, <i>J</i> = 3.7 Hz, 1H), 4.45–4.37 (m, 1H), 3.81 (s, 2H), 2.97–2.87 (m, 2H), 2.87 (s, 3H), 2.88–2.78 (m, 2H), 2.27–2.14 (m, 2H), 1.94–1.84 (m, 2H), 1.80 (dd, <i>J</i> = 11.1, 7.4 Hz, 2H); <sup>13</sup>C NMR (100 MHz, methanol-<i>d</i><sub>4</sub>) δ 163.2, 141.3, 141.1, 135.6, 121.3, 121.0, 120.7, 120.1, 119.4, 118.6, 113.4, 97.2, 95.5, 93.8, 53.9, 46.9, 44.0, 41.9, 26.3, 25.3, 17.2, 14.6; HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>22</sub>H<sub>28</sub>N<sub>5</sub>O [M + H]<sup>+</sup> 378.2288, found 378.2292.</div></div><div id="sec4_11_61" class="NLM_sec NLM_sec_level_3"><div id="ac_i98" class="anchor-spacer"></div><h4 class="article-section__title" id="_i98"> 4-((1-(3-Chlorobenzyl)azepan-4-yl)amino)-<i>N</i>-methyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-5-carboxamide (<b>30b</b>)</h4><div class="NLM_p last"><b>30b</b> was synthesized in 56% yield according to the same procedure as <b>20d</b> using <b>26</b> and 3-chlorobenzaldehyde; <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 8.17 (s, 1H), 7.44 (d, <i>J</i> = 1.1 Hz, 1H), 7.33–7.29 (m, 2H), 7.29–7.23 (m, 1H), 7.09 (d, <i>J</i> = 3.6 Hz, 1H), 6.62 (d, <i>J</i> = 3.8 Hz, 1H), 4.45–4.38 (m, 1H), 3.72–3.62 (m, 2H), 2.88 (s, 3H), 2.83–2.75 (m, 1H), 2.75–2.67 (m, 3H), 2.23–2.12 (m, 2H), 1.91–1.71 (m, 4H); <sup>13</sup>C NMR (100 MHz, methanol-<i>d</i><sub>4</sub>) δ 171.1, 149.7, 149.2, 143.6, 141.4, 134.2, 129.6, 129.0, 127.3, 127.2, 121.3, 105.4, 103.7, 102.7, 62.5, 55.8, 52.7, 50.9, 35.3, 34.8, 26.4, 24.5. HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>22</sub>H<sub>27</sub>ClN<sub>5</sub>O [M + H]<sup>+</sup> 412.1899, found 412.1903.</div></div><div id="sec4_11_62" class="NLM_sec NLM_sec_level_3"><div id="ac_i99" class="anchor-spacer"></div><h4 class="article-section__title" id="_i99"> 4-((1-(4-Chlorobenzyl)azepan-4-yl)amino)-<i>N</i>-methyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-5-carboxamide (<b>30c</b>)</h4><div class="NLM_p last"><b>30c</b> was synthesized in 58% yield according to the same procedure as <b>20d</b> using <b>26</b> and 4-chlorobenzaldehyde; <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 8.17 (s, 1H), 7.41–7.30 (m, 4H), 7.09 (d, <i>J</i> = 3.7 Hz, 1H), 6.61 (d, <i>J</i> = 3.7 Hz, 1H), 4.47–4.35 (m, 1H), 3.69 (d, <i>J</i> = 2.0 Hz, 2H), 2.88 (s, 3H), 2.85–2.77 (m, 1H), 2.77–2.69 (m, 3H), 2.22–2.13 (m, 2H), 1.92–1.70 (m, 4H); <sup>13</sup>C NMR (100 MHz, methanol-<i>d</i><sub>4</sub>) δ 171.3, 149.9, 149.2, 143.7, 137.4, 133.1, 130.7, 128.7, 121.6, 105.7, 103.9, 102.8, 62.3, 55.9, 52.9, 50.8, 35.1, 34.9, 26.5, 24.3; HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>22</sub>H<sub>27</sub>ClN<sub>5</sub>O [M + H]<sup>+</sup> 412.1899, found 412.1904.</div></div><div id="sec4_11_63" class="NLM_sec NLM_sec_level_3"><div id="ac_i100" class="anchor-spacer"></div><h4 class="article-section__title" id="_i100"> 4-((1-(3-Methoxybenzyl)azepan-4-yl)amino)-<i>N</i>-methyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-5-carboxamide (<b>30d</b>)</h4><div class="NLM_p last"><b>30d</b> was synthesized in 74% yield according to the same procedure as <b>20d</b> using <b>26</b> and 3-methoxybenzaldehyde; <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 9.36 (d, <i>J</i> = 7.9 Hz, 1H), 8.27 (s, 1H), 7.22 (d, <i>J</i> = 7.9 Hz, 1H), 7.02–6.86 (m, 3H), 6.80 (dd, <i>J</i> = 8.2, 1.9 Hz, 1H), 6.47 (d, <i>J</i> = 3.0 Hz, 1H), 4.28 (dd, <i>J</i> = 8.0, 4.0 Hz, 1H), 3.82 (s, 3H), 3.70–3.60 (m, 2H), 2.99 (d, <i>J</i> = 4.7 Hz, 3H), 2.83–2.64 (m, 4H), 2.29–2.05 (m, 2H), 1.98–1.77 (m, 3H), 1.77–1.61 (m, 1H); <sup>13</sup>C NMR (100 MHz, chloroform-<i>d</i>) δ 170.7, 159.7, 149.6, 148.5, 147.4, 142.9, 129.4, 121.6, 121.3, 114.6, 113.2, 105.4, 103.6, 102.6, 62.5, 55.5, 55.2, 52.3, 50.2, 34.6, 34.0, 26.3, 23.5; HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>23</sub>H<sub>30</sub>N<sub>5</sub>O<sub>2</sub> [M + H]<sup>+</sup> 408.2394, found 408.2390.</div></div><div id="sec4_11_64" class="NLM_sec NLM_sec_level_3"><div id="ac_i101" class="anchor-spacer"></div><h4 class="article-section__title" id="_i101"> 4-((1-(4-Methoxybenzyl)azepan-4-yl)amino)-<i>N</i>-methyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-5-carboxamide (<b>30e</b>)</h4><div class="NLM_p last"><b>30e</b> was synthesized in 74% yield according to the same procedure as <b>20d</b> using <b>26</b> and 4-methoxybenzaldehyde; <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 9.38 (d, <i>J</i> = 7.9 Hz, 1H), 8.28 (s, 1H), 7.34 (d, <i>J</i> = 8.5 Hz, 2H), 6.93 (d, <i>J</i> = 3.3 Hz, 1H), 6.91–6.85 (m, 2H), 6.46 (d, <i>J</i> = 3.0 Hz, 1H), 4.30 (s, 1H), 3.81 (s, 3H), 3.74 (s, 2H), 3.00 (d, <i>J</i> = 4.7 Hz, 3H), 2.98–2.80 (m, 2H), 2.74 (dd, <i>J</i> = 8.9, 4.4 Hz, 1H), 2.31–2.15 (m, 1H), 1.98–1.75 (m, 4H); <sup>13</sup>C NMR (100 MHz, methanol-<i>d</i><sub>4</sub>) δ 170.6, 159.3, 149.6, 149.4, 148.5, 142.8, 131.0, 121.4, 113.9, 105.3, 103.5, 102.4, 61.5, 55.1, 55.0, 51.9, 34.4, 32.9, 26.4, 26.2, 22.6; HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>23</sub>H<sub>30</sub>N<sub>5</sub>O<sub>2</sub> [M + H]<sup>+</sup> 408.2394, found 408.2390.</div></div><div id="sec4_11_65" class="NLM_sec NLM_sec_level_3"><div id="ac_i102" class="anchor-spacer"></div><h4 class="article-section__title" id="_i102"> 4-((1-(4-Cyanobenzyl)aze5pan-4-yl)amino)-<i>N</i>-methyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-5-carboxamide (<b>30f</b>)</h4><div class="NLM_p last"><b>30f</b> was synthesized in 65% yield according to the same procedure as <b>20d</b> using <b>26</b> and 4-formylbenzonitrile; <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 9.32 (d, <i>J</i> = 8.2 Hz, 1H), 8.24 (s, 1H), 7.64–7.49 (m, 4H), 7.00 (d, <i>J</i> = 3.5 Hz, 1H), 6.54 (d, <i>J</i> = 3.7 Hz, 1H), 4.39–4.28 (m, 1H), 3.71 (s, 2H), 2.99 (d, <i>J</i> = 4.8 Hz, 3H), 2.79–2.61 (m, 4H), 2.21–2.09 (m, 2H), 1.98–1.80 (m, 3H), 1.78–1.65 (m, 1H); <sup>13</sup>C NMR (100 MHz, methanol-<i>d</i><sub>4</sub>) δ 170.7, 149.6, 148.6, 145.4, 143.0, 132.1, 129.3, 121.1, 118.9, 110.4, 105.3, 103.5, 102.5, 62.5, 55.7, 52.6, 51.0, 35.4, 34.3, 26.3, 24.5; HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>23</sub>H<sub>27</sub>N<sub>6</sub>O [M + H]<sup>+</sup> 403.2241, found 403.2246.</div></div><div id="sec4_11_66" class="NLM_sec NLM_sec_level_3"><div id="ac_i103" class="anchor-spacer"></div><h4 class="article-section__title" id="_i103"> 4-((<i>trans</i>-1-Benzyl-2-methylpiperidin-4-yl)amino)-<i>N</i>-methyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-5-carboxamide (<b>31a</b>)</h4><div class="NLM_p last"><b>31a</b> was synthesized in 49% yield according to the same procedure as <b>20d</b> using <b>27a</b> and benzaldehyde; <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 9.97 (s, 1H), 9.47 (d, <i>J</i> = 7.8 Hz, 1H), 8.24 (s, 1H), 7.41–7.29 (m, 4H), 7.25–7.23 (m, 1H), 7.00 (d, <i>J</i> = 3.6 Hz, 1H), 6.54–6.50 (m, 1H), 6.41 (s, 1H), 4.33 (s, 1H), 4.02 (d, <i>J</i> = 13.5 Hz, 1H), 3.36 (d, <i>J</i> = 13.5 Hz, 1H), 3.01 (d, <i>J</i> = 4.7 Hz, 3H), 2.96–2.88 (m, 1H), 2.73–2.67 (m, 1H), 2.50–2.42 (m, 1H), 1.94 (d, <i>J</i> = 5.7 Hz, 3H), 1.84–1.75 (m, 1H), 1.23 (d, <i>J</i> = 6.3 Hz, 3H); <sup>13</sup>C NMR (100 MHz, methanol-<i>d</i><sub>4</sub>) δ 171.8, 150.2, 149.7, 144.2, 137.2, 130.2, 128.9, 128.1, 122.1, 106.0, 104.1, 102.9, 58.7, 53.4, 46.7, 46.7, 39.1, 31.1, 26.6, 17.8; HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>22</sub>H<sub>28</sub>N<sub>5</sub>O [M + H]<sup>+</sup> 378.2288, found 378.2295.</div></div><div id="sec4_11_67" class="NLM_sec NLM_sec_level_3"><div id="ac_i104" class="anchor-spacer"></div><h4 class="article-section__title" id="_i104"> 4-((<i>trans</i>-1-(3-Chlorobenzyl)-2-methylpiperidin-4-yl)amino)-<i>N</i>-methyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-5-carboxamide (<b>31b</b>)</h4><div class="NLM_p last"><b>31b</b> was synthesized in 45% yield according to the same procedure as <b>20d</b> using <b>27a</b> and 3-clorobenzaldehyde; <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 9.42 (d, <i>J</i> = 7.8 Hz, 2H), 8.22 (s, 1H), 7.38 (d, <i>J</i> = 2.1 Hz, 1H), 7.24–7.18 (m, 3H), 7.03 (d, <i>J</i> = 3.6 Hz, 1H), 6.55 (d, <i>J</i> = 3.7 Hz, 1H), 6.22 (s, 1H), 4.34 (s, 1H), 3.94 (d, <i>J</i> = 13.7 Hz, 1H), 3.32 (d, <i>J</i> = 13.7 Hz, 1H), 3.01 (d, <i>J</i> = 4.8 Hz, 3H), 2.92–2.85 (m, 1H), 2.71–2.63 (m, 1H), 2.48–2.40 (m, 1H), 1.87–1.77 (m, 4H), 1.19 (d, <i>J</i> = 6.3 Hz, 3H); <sup>13</sup>C NMR (100 MHz, methanol-<i>d</i><sub>4</sub>) δ 172.2, 150.5, 150.0, 144.5, 141.3, 135.0, 130.4, 130.1, 128.3, 128.1, 122.4, 106.3, 104.4, 103.3, 58.7, 53.3, 47.3, 47.1, 39.9, 32.0, 26.9, 18.0; HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>22</sub>H<sub>27</sub>ClN<sub>5</sub>O [M + H]<sup>+</sup> 412.1899, found 412.1903.</div></div><div id="sec4_11_68" class="NLM_sec NLM_sec_level_3"><div id="ac_i105" class="anchor-spacer"></div><h4 class="article-section__title" id="_i105"> 4-((<i>trans</i>-1-(4-Chlorobenzyl)-2-methylpiperidin-4-yl)amino)-<i>N</i>-methyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-5-carboxamide (<b>31c</b>)</h4><div class="NLM_p last"><b>31c</b> was synthesized in 57% yield according to the same procedure as <b>20d</b> using <b>27a</b> and 4-clorobenzaldehyde; <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 9.80 (s, 1H), 9.44 (d, <i>J</i> = 7.8 Hz, 1H), 8.22 (s, 1H), 7.32–7.27 (m, 4H), 7.03 (d, <i>J</i> = 3.6 Hz, 1H), 6.54 (d, <i>J</i> = 3.7 Hz, 1H), 6.25 (s, 1H), 4.33 (s, 1H), 3.94 (d, <i>J</i> = 13.6 Hz, 1H), 3.32 (d, <i>J</i> = 13.6 Hz, 1H), 3.00 (d, <i>J</i> = 4.7 Hz, 3H), 2.92–2.84 (m, 1H), 2.69–2.62 (m, 1H), 2.46–2.38 (m, 1H), 1.95 (dd, <i>J</i> = 12.6, 3.9 Hz, 3H), 1.80 (d, <i>J</i> = 11.2 Hz, 1H), 1.19 (d, <i>J</i> = 6.3 Hz, 3H); <sup>13</sup>C NMR (100 MHz, methanol-<i>d</i><sub>4</sub>) δ 171.5, 150.1, 149.4, 143.9, 136.8, 133.3, 131.2, 128.8, 121.9, 105.8, 103.9, 102.9, 58.0, 52.8, 46.8, 46.6, 39.4, 31.5, 26.6, 17.7; HRMS calcd for C<sub>22</sub>H<sub>27</sub>ClN<sub>5</sub>O [M + H]<sup>+</sup> 412.1899, found 412.1903.</div></div><div id="sec4_11_69" class="NLM_sec NLM_sec_level_3"><div id="ac_i106" class="anchor-spacer"></div><h4 class="article-section__title" id="_i106"> 4-((<i>trans</i>-1-(3-Methoxybenzyl)-2-methylpiperidin-4-yl)amino)-<i>N</i>-methyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-5-carboxamide (<b>31d</b>)</h4><div class="NLM_p last"><b>31d</b> was synthesized in 56% yield according to the same procedure as <b>20d</b> using <b>27a</b> and 3-methoxybenzaldehyde; <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 9.98 (s, 1H), 9.47 (d, <i>J</i> = 7.8 Hz, 1H), 8.23 (s, 1H), 7.23 (t, <i>J</i> = 7.8 Hz, 1H), 7.01 (d, <i>J</i> = 3.7 Hz, 1H), 6.99–6.92 (m, 2H), 6.81–6.77 (m, 1H), 6.53 (d, <i>J</i> = 3.7 Hz, 1H), 6.34 (s, 1H), 4.33 (s, 1H), 3.99 (d, <i>J</i> = 13.4 Hz, 1H), 3.82 (s, 3H), 3.35 (d, <i>J</i> = 13.4 Hz, 1H) 3.00 (d, <i>J</i> = 4.7 Hz, 3H), 2.97–2.91 (m, 1H), 2.76–2.69 (m, 1H), 2.52–2.44 (m, 1H), 2.05–2.00 (m, 1H), 1.95 (t, <i>J</i> = 5.3 Hz, 2H), 1.84–1.75 (m, 1H), 1.22 (d, <i>J</i> = 6.4 Hz, 3H); <sup>13</sup>C NMR (100 MHz, methanol-<i>d</i><sub>4</sub>) δ 171.6, 160.1, 150.1, 149.4, 143.9, 138.9, 129.7, 122.4, 121.9, 115.4, 113.5, 105.9, 104.0, 102.9, 58.7, 55.5, 53.3, 49.9, 46.7, 46.6, 39.1, 31.1, 17.6; HRMS calcd for C<sub>23</sub>H<sub>30</sub>N<sub>5</sub>O<sub>2</sub> [M + H]<sup>+</sup> 408.2394, found 408.2399.</div></div><div id="sec4_11_70" class="NLM_sec NLM_sec_level_3"><div id="ac_i107" class="anchor-spacer"></div><h4 class="article-section__title" id="_i107"> 4-((<i>trans</i>-1-(4-Methoxybenzyl)-2-methylpiperidin-4-yl)amino)-<i>N</i>-methyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-5-carboxamide (<b>31e</b>)</h4><div class="NLM_p last"><b>31e</b> was synthesized in 49% yield according to the same procedure as <b>20d</b> using <b>27a</b> and 4-methoxybenzaldehyde; <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 10.20 (s, 1H), 9.54 (d, <i>J</i> = 7.8 Hz, 1H), 8.23 (s, 1H), 7.31 (d, <i>J</i> = 8.4 Hz, 2H), 7.01 (d, <i>J</i> = 3.6 Hz, 1H), 6.89–6.85 (m, 2H), 6.50 (d, <i>J</i> = 3.7 Hz, 2H), 4.34 (s, 1H), 4.06 (d, <i>J</i> = 13.1 Hz, 1H), 3.81 (s, 3H), 3.36 (d, <i>J</i> = 13.1 Hz, 1H), 3.02–2.95 (m, 3H), 2.77 (d, <i>J</i> = 12.3 Hz, 1H), 2.49 (t, <i>J</i> = 10.9 Hz, 1H), 2.00 (dd, <i>J</i> = 28.9, 10.6 Hz, 3H), 1.83 (d, <i>J</i> = 13.3 Hz, 2H), 1.29 (d, <i>J</i> = 6.3 Hz, 3H); <sup>13</sup>C NMR (100 MHz, methanol-<i>d</i><sub>4</sub>) δ 171.5, 159.9, 150.0, 149.4, 143.8, 131.8, 127.1, 122.2, 114.4, 105.9, 104.0, 102.7, 57.6, 55.5, 53.7, 49.9, 46.3, 38.5, 30.3, 26.6, 17.6; HRMS calcd for C<sub>23</sub>H<sub>30</sub>N<sub>5</sub>O<sub>2</sub> [M + H]<sup>+</sup> 408.2394, found 408.2401.</div></div><div id="sec4_11_71" class="NLM_sec NLM_sec_level_3"><div id="ac_i108" class="anchor-spacer"></div><h4 class="article-section__title" id="_i108"> 4-((<i>cis</i>-1-(3-Chlorobenzyl)-2-methylpiperidin-4-yl)amino)-<i>N</i>-methyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-5-carboxamide (<b>31f</b>)</h4><div class="NLM_p last"><b>31f</b> was synthesized in 65% yield according to the same procedure as <b>20d</b> using <b>27b</b> and 3-clorobenzaldehyde; <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 9.08 (d, <i>J</i> = 7.9 Hz, 1H), 8.18 (s, 1H), 7.33 (s, 1H), 7.22–7.14 (m, 3H), 7.02 (d, <i>J</i> = 3.7 Hz, 1H), 6.52 (d, <i>J</i> = 3.7 Hz, 1H), 6.05 (s, 1H), 4.08 (d, <i>J</i> = 13.7 Hz, 1H), 4.00–3.87 (m, 1H), 3.10 (d, <i>J</i> = 13.7 Hz, 1H), 2.96 (d, <i>J</i> = 4.8 Hz, 3H), 2.90–2.82 (m, 1H), 2.46–2.36 (m, 1H), 2.15 (dd, <i>J</i> = 12.8, 2.4 Hz, 1H), 2.12–2.04 (m, 2H), 1.58 (dd, <i>J</i> = 11.5, 3.8 Hz, 2H), 1.21 (d, <i>J</i> = 6.1 Hz, 3H); <sup>13</sup>C NMR (100 MHz, methanol-<i>d</i><sub>4</sub>) δ 170.9, 149.8, 149.3, 143.6, 140.9, 134.2, 129.5, 129.1, 127.3, 127.2, 121.3, 105.2, 103.6, 102.6, 57.8, 57.7, 57.1, 56.0, 51.5, 51.0, 42.0, 33.2, 26.3, 20.9; HRMS calcd for C<sub>22</sub>H<sub>27</sub>ClN<sub>5</sub>O [M + H]<sup>+</sup> 412.1899, found 412.1904.</div></div><div id="sec4_11_72" class="NLM_sec NLM_sec_level_3"><div id="ac_i109" class="anchor-spacer"></div><h4 class="article-section__title" id="_i109"> 4-((<i>cis-</i>1-(4-Chlorobenzyl)-2-methylpiperidin-4-yl)amino)-<i>N</i>-methyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-5-carboxamide (<b>31g</b>)</h4><div class="NLM_p last"><b>31g</b> was synthesized in 63% yield according to the same procedure as <b>20d</b> using <b>27b</b> and 4-clorobenzaldehyde; <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 9.38 (s, 1H), 9.09 (d, <i>J</i> = 7.9 Hz, 1H), 8.20 (s, 1H), 7.31–7.27 (m, 4H), 7.03 (d, <i>J</i> = 3.7 Hz, 1H), 6.53 (d, <i>J</i> = 3.7 Hz, 1H), 6.11–5.99 (m, 1H), 4.08 (d, <i>J</i> = 13.7 Hz, 1H), 4.00–3.86 (m, 1H), 3.12 (d, <i>J</i> = 13.7 Hz, 1H), 2.98 (d, <i>J</i> = 4.8 Hz, 3H), 2.91–2.82 (m, 1H), 2.48–2.36 (m, 1H), 2.19–2.12 (m, 1H), 2.12–2.04 (m, 2H), 1.59–1.45 (m, 2H), 1.23 (d, <i>J</i> = 6.1 Hz, 3H); <sup>13</sup>C NMR (100 MHz, methanol-<i>d</i><sub>4</sub>) δ 171.7, 150.1, 149.9, 144.3, 137.1, 133.3, 131.2, 128.8, 122.0, 105.8, 104.1, 102.8, 57.3, 56.5, 51.8, 51.4, 42.5, 33.7, 26.5, 21.1; HRMS calcd for C<sub>22</sub>H<sub>27</sub>ClN<sub>5</sub>O [M + H]<sup>+</sup> 412.1903, found 412.1899.</div></div><div id="sec4_11_73" class="NLM_sec NLM_sec_level_3"><div id="ac_i110" class="anchor-spacer"></div><h4 class="article-section__title" id="_i110"> 4-((<i>cis</i>-1-(3-Methoxybenzyl)-2-methylpiperidin-4-yl)amino)-<i>N</i>-methyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-5-carboxamide (<b>31h</b>)</h4><div class="NLM_p last"><b>31h</b> was synthesized in 66% yield according to the same procedure as <b>20d</b> using <b>27b</b> and 3-methoxybenzaldehyde; <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 9.66 (s, 1H), 9.09 (d, <i>J</i> = 7.9 Hz, 1H), 8.20 (s, 1H), 7.22 (d, <i>J</i> = 8.0 Hz, 1H), 7.03 (d, <i>J</i> = 3.6 Hz, 1H), 6.93–6.90 (m, 2H), 6.82–6.77 (m, 1H), 6.53 (d, <i>J</i> = 3.7 Hz, 1H), 6.09 (s, 1H), 4.11 (d, <i>J</i> = 13.6 Hz, 1H), 3.99–3.88 (m, 1H), 3.82 (s, 3H), 3.36 (d, <i>J</i> = 13.6 Hz, 1H), 2.98 (d, <i>J</i> = 4.8 Hz, 3H), 2.95–2.84 (m, 1H), 2.49–2.39 (m, 1H), 2.20–2.04 (m, 3H), 1.66–1.46 (m, 2H), 1.24 (d, <i>J</i> = 6.1 Hz, 3H); <sup>13</sup>C NMR (100 MHz, chloroform-<i>d</i>) δ 170.9, 158.7, 149.7, 149.3, 143.5, 130.7, 129.7, 121.2, 113.6, 105.2, 103.6, 102.6, 56.7, 55.7, 55.2, 51.1, 51.0, 41.9, 33.1, 26.3, 20.8; HRMS calcd for C<sub>23</sub>H<sub>30</sub>N<sub>5</sub>O<sub>2</sub> [M + H]<sup>+</sup> 408.2394, found 408.2403.</div></div><div id="sec4_11_74" class="NLM_sec NLM_sec_level_3"><div id="ac_i111" class="anchor-spacer"></div><h4 class="article-section__title" id="_i111"> 4-((<i>cis</i>-1-(4-Methoxybenzyl)-2-methylpiperidin-4-yl)amino)-<i>N</i>-methyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-5-carboxamide (<b>31i</b>)</h4><div class="NLM_p last"><b>31i</b> was synthesized in 63% yield according to the same procedure as <b>20d</b> using <b>27b</b> and 4-methoxybenzaldehyde; <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 9.66 (s, 1H), 9.07 (d, <i>J</i> = 7.9 Hz, 1H), 8.20 (s, 1H), 7.26–7.22 (m, 2H), 7.03 (d, <i>J</i> = 3.6 Hz, 1H), 6.88–6.84 (m, 2H), 6.52 (d, <i>J</i> = 3.7 Hz, 1H), 6.06 (s, 1H), 4.07 (d, <i>J</i> = 13.3 Hz, 1H), 3.97–3.86 (m, 1H), 3.81 (s, 3H), 3.15 (d, <i>J</i> = 13.3 Hz, 1H), 2.97 (d, <i>J</i> = 4.8 Hz, 3H), 2.94–2.88 (m, 1H), 2.47–2.35 (m, 1H), 2.20–2.13 (m, 1H), 2.11–2.02 (m, 2H), 1.63–1.46 (m, 2H), 1.26 (d, <i>J</i> = 6.1 Hz, 3H); <sup>13</sup>C NMR (100 MHz, chloroform-<i>d</i>) δ 170.9, 158.8, 149.8, 149.4, 143.6, 130.7, 129.8, 121.3, 113.7, 105.2, 103.7, 102.7, 57.5, 55.9, 55.2, 51.4, 51.0, 42.0, 33.2, 26.3, 20.9; HRMS calcd for C<sub>23</sub>H<sub>30</sub>N<sub>5</sub>O<sub>2</sub> [M + H]<sup>+</sup> 408.2394, found 408.2394.</div></div><div id="sec4_11_75" class="NLM_sec NLM_sec_level_3"><div id="ac_i112" class="anchor-spacer"></div><h4 class="article-section__title" id="_i112"> 4-((<i>cis</i>-1-(4-Cyanobenzyl)-2-methylpiperidin-4-yl)amino)-<i>N</i>-methyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-5-carboxamide (<b>31j</b>)</h4><div class="NLM_p last"><b>31j</b> was synthesized in 91% yield according to the same procedure as <b>20d</b> using <b>27b</b> and 4-formylbenzonitrile; <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 9.67 (s, 1H), 9.12 (d, <i>J</i> = 7.9 Hz, 1H), 8.21 (s, 1H), 7.63–7.44 (m, 4H), 7.04 (d, <i>J</i> = 3.7 Hz, 1H), 6.53 (d, <i>J</i> = 3.7 Hz, 1H), 6.09 (s, 1H), 4.16 (d, <i>J</i> = 14.4 Hz, 1H), 4.01–3.91 (m, 1H), 3.32 (d, <i>J</i> = 14.4 Hz, 1H), 2.98 (d, <i>J</i> = 4.8 Hz, 3H), 2.89–2.77 (m, 1H), 2.51–2.42 (m, 1H), 2.24–2.04 (m, 3H), 1.62–1.55 (m, 1H), 1.55–1.46 (m, 1H), 1.20 (d, <i>J</i> = 6.1 Hz, 3H); <sup>13</sup>C NMR (100 MHz, chloroform-<i>d</i>) δ 170.8, 149.8, 149.3, 145.5, 143.5, 132.1, 129.4, 121.3, 119.0, 110.5, 105.2, 103.6, 102.5, 57.3, 56.2, 51.9, 51.0, 42.1, 33.3, 26.4, 21.1; HRMS calcd for C<sub>23</sub>H<sub>27</sub>N<sub>6</sub>O [M + H]<sup>+</sup> 403.2241, found 403.2247.</div></div><div id="sec4_11_76" class="NLM_sec NLM_sec_level_3"><div id="ac_i113" class="anchor-spacer"></div><h4 class="article-section__title" id="_i113"> 4-((<i>cis</i>-1-(3-Chlorobenzyl)-2-methylpiperidin-4-yl)amino)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-5-carboxamide (<b>31k</b>)</h4><div class="NLM_p last"><b>31k</b> was synthesized in 56% yield according to the same procedure as <b>20d</b> using <b>27c</b> and 3-chlorobenzaldehyde; <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 9.86–9.68 (m, 1H), 9.32 (d, <i>J</i> = 8.0 Hz, 1H), 8.25 (s, 1H), 7.23 (d, <i>J</i> = 8.0 Hz, 1H), 7.04 (d, <i>J</i> = 3.6 Hz, 1H), 6.93 (d, <i>J</i> = 2.2 Hz, 2H), 6.80 (dd, <i>J</i> = 7.9, 2.2 Hz, 1H), 6.55 (d, <i>J</i> = 3.7 Hz, 1H), 5.65 (s, 2H), 4.12 (d, <i>J</i> = 13.5 Hz, 1H), 4.02–3.90 (m, 1H), 3.82 (s, 3H), 3.14 (d, <i>J</i> = 13.5 Hz, 1H), 2.97–2.90 (m, 1H), 2.50–2.38 (m, 1H), 2.17 (d, <i>J</i> = 13.0 Hz, 1H), 2.11–2.05 (m, 2H), 1.67–1.47 (m, 2H), 1.26 (d, <i>J</i> = 6.0 Hz, 3H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 172.1, 150.4, 149.4, 145.6, 142.5, 132.9, 130.0, 128.1, 127.2, 126.6, 121.7, 104.4, 101.7, 99.0, 56.1, 55.2, 51.1, 50.2, 42.4, 33.4, 21.0; HRMS calcd for C<sub>21</sub>H<sub>25</sub>ClN<sub>5</sub>O [M + H]<sup>+</sup> 398.1742, found 398.1746.</div></div><div id="sec4_11_77" class="NLM_sec NLM_sec_level_3"><div id="ac_i114" class="anchor-spacer"></div><h4 class="article-section__title" id="_i114"> 4-((<i>cis</i>-1-(3-Methoxybenzyl)-2-methylpiperidin-4-yl)amino)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-5-carboxamide (<b>31l</b>)</h4><div class="NLM_p last"><b>31l</b> was synthesized in 36% yield according to the same procedure as <b>20d</b> using <b>27c</b> and 4-methoxybenzaldehyde; <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 9.74 (d, <i>J</i> = 2.6 Hz, 1H), 9.32 (d, <i>J</i> = 8.0 Hz, 1H), 8.25 (s, 1H), 7.23 (d, <i>J</i> = 8.0 Hz, 1H), 7.04 (d, <i>J</i> = 3.6 Hz, 1H), 6.93 (d, <i>J</i> = 2.2 Hz, 2H), 6.80 (dd, <i>J</i> = 7.9, 2.2 Hz, 1H), 6.55 (d, <i>J</i> = 3.7 Hz, 1H), 5.65 (s, 2H), 4.12 (d, <i>J</i> = 13.5 Hz, 1H), 4.01–3.89 (m, 1H), 3.82 (s, 3H), 3.14 (d, <i>J</i> = 13.5 Hz, 1H), 2.97–2.90 (m, 1H), 2.50–2.39 (m, 1H), 2.17 (d, <i>J</i> = 13.0 Hz, 1H), 2.13–2.06 (m, 2H), 1.66–1.46 (m, 2H), 1.26 (d, <i>J</i> = 6.0 Hz, 3H); <sup>13</sup>C NMR (100 MHz, chloroform-<i>d</i>) δ 173.0, 159.6, 150.4, 149.2, 144.6, 139.8, 129.2, 121.8, 121.2, 115.0, 112.4, 105.1, 102.8, 101.8, 57.5, 55.9, 55.2, 51.3, 51.0, 41.9, 33.1, 20.8; HRMS calcd for C<sub>22</sub>H<sub>28</sub>N<sub>5</sub>O<sub>2</sub> [M + H]<sup>+</sup> 394.2238, found 394.2239.</div></div><div id="sec4_11_78" class="NLM_sec NLM_sec_level_3"><div id="ac_i115" class="anchor-spacer"></div><h4 class="article-section__title" id="_i115"> 4-((<i>trans</i>-1-(3-Chlorobenzyl)-3-fluoropiperidin-4-yl)amino)-<i>N</i>-methyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-5-carboxamide (<b>32a</b>)</h4><div class="NLM_p last"><b>32a</b> was synthesized in 25% yield according to the same procedure as <b>20d</b> using <b>28a</b> and 3-chlorobenzaldehyde; <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 9.38 (d, <i>J</i> = 8.2 Hz, 1H), 8.24 (s, 1H), 7.34 (s, 1H), 7.26–7.18 (m, 3H), 7.06 (d, <i>J</i> = 3.7 Hz, 1H), 6.59 (t, <i>J</i> = 3.3 Hz, 1H), 6.28 (s, 1H), 4.77–4.57 (m, 1H), 4.31–4.19 (m, 1H), 3.57 (s, 2H), 3.09–3.00 (m, 1H), 2.98 (d, <i>J</i> = 4.8 Hz, 3H), 2.78–2.67 (m, 1H), 2.51 (dt, <i>J</i> = 11.2, 7.2 Hz, 1H), 2.38 (t, <i>J</i> = 9.4 Hz, 1H), 2.28 (dd, <i>J</i> = 8.8, 3.5 Hz, 1H), 1.82–1.68 (m, 1H); <sup>13</sup>C NMR (100 MHz, chloroform-<i>d</i>) δ 170.8, 150.2, 149.3, 143.3, 139.8, 134.3, 129.7, 129.0, 127.5, 127.1, 121.6, 105.5, 104.0, 102.3, 90.2 (d, <i>J</i><sub>C–F</sub> = 182.0 Hz), 61.8, 55.1 (d, <i>J</i><sub>C–F</sub> = 23.9 Hz), 53.5 (d, <i>J</i><sub>C–F</sub> = 23.7 Hz), 50.2, 29.8, 26.4; HRMS calcd for C<sub>21</sub>H<sub>24</sub>ClFN<sub>5</sub>O [M + H]<sup>+</sup> 416.1648, found 416.1648.</div></div><div id="sec4_11_79" class="NLM_sec NLM_sec_level_3"><div id="ac_i116" class="anchor-spacer"></div><h4 class="article-section__title" id="_i116"> 4-((<i>trans</i>-1-(3-Methoxybenzyl)-3-fluoropiperidin-4-yl)amino)-<i>N</i>-methyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-5-carboxamide (<b>32b</b>)</h4><div class="NLM_p last"><b>32b</b> was synthesized in 61% yield according to the same procedure as <b>20d</b> using <b>28a</b> and 3-methoxybenzaldehyde; <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 11.40 (s, 1H), 9.36 (d, <i>J</i> = 8.2 Hz, 1H), 8.24 (d, <i>J</i> = 5.6 Hz, 1H), 7.28–7.17 (m, 1H), 7.08–6.96 (m, 1H), 6.94–6.87 (m, 2H), 6.81 (dd, <i>J</i> = 8.9, 1.8 Hz, 1H), 6.56 (d, <i>J</i> = 3.0 Hz, 1H), 6.30 (s, 1H), 4.77–4.55 (m, 1H), 4.34–4.15 (m, 1H), 3.81 (s, 3H), 3.57 (s, 2H), 3.10 (td, <i>J</i> = 15.5, 13.4, 6.8 Hz, 1H), 2.97 (d, <i>J</i> = 4.7 Hz, 3H), 2.82–2.75 (m, 1H), 2.47 (q, <i>J</i> = 7.1 Hz, 1H), 2.35 (t, <i>J</i> = 9.5 Hz, 1H), 2.26 (d, <i>J</i> = 5.3 Hz, 1H), 1.85–1.68 (m, 1H); <sup>13</sup>C NMR (100 MHz, chloroform-<i>d</i>) δ 170.79, 159.75, 150.30, 143.17, 143.02, 139.55, 129.31, 121.50, 121.23, 114.21, 112.85, 105.42, 103.94, 102.41, 90.68 (d, <i>J</i><sub>C–F</sub> = 180.1 Hz), 62.31, 55.50 (d, <i>J</i><sub>C–F</sub> = 24.1 Hz), 55.23, 54.16 (d, <i>J</i><sub>C–F</sub> = 22.8 Hz), 50.53, 30.27, 26.65; HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>22</sub>H<sub>27</sub>FN<sub>5</sub>O<sub>2</sub> [M + H]<sup>+</sup> 412.2143, found 412.2139.</div></div><div id="sec4_11_80" class="NLM_sec NLM_sec_level_3"><div id="ac_i117" class="anchor-spacer"></div><h4 class="article-section__title" id="_i117"> 4-((<i>cis</i>-3-Fluoro-1-(3-methoxybenzyl)piperidin-4-yl)amino)-<i>N</i>-methyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-5-carboxamide (<b>32c</b>)</h4><div class="NLM_p last"><b>32c</b> was synthesized in 40% yield according to the same procedure as <b>20d</b> using <b>28b</b> and 3-methoxybenzaldehyde; <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 9.51 (d, <i>J</i> = 8.3 Hz, 1H), 8.21 (s, 1H), 7.23 (d, <i>J</i> = 7.8 Hz, 1H), 7.06 (d, <i>J</i> = 3.7 Hz, 1H), 6.97–6.89 (m, 2H), 6.82 (dd, <i>J</i> = 7.9, 2.2 Hz, 1H), 6.46 (d, <i>J</i> = 3.7 Hz, 1H), 6.10 (s, 1H), 4.87 (d, <i>J</i> = 48.9 Hz, 1H), 3.82 (s, 3H), 3.62 (s, 2H), 3.21 (s, 1H), 3.00 (d, <i>J</i> = 4.8 Hz, 3H), 2.95–2.87 (m, 1H), 2.53–2.37 (m, 1H), 2.38–2.28 (m, 1H), 2.13 (d, <i>J</i> = 16.4 Hz, 2H), 2.04 (s, 1H); <sup>13</sup>C NMR (100 MHz, methanol-<i>d</i><sub>4</sub>) δ 171.7, 160.4, 150.1, 149.9, 144.2, 139.0, 129.9, 122.6, 122.3, 115.3, 113.6, 106.1, 104.9, 102.3, 88.9 (d, <i>J</i><sub>C–F</sub> = 179.0 Hz), 62.7, 55.5 (d, <i>J</i><sub>C–F</sub> = 18.4 Hz), 53.0 (d, <i>J</i><sub>C–F</sub> = 19.3 Hz), 51.6, 30.2, 28.3, 26.6; HRMS calcd for C<sub>22</sub>H<sub>27</sub>FN<sub>5</sub>O<sub>2</sub> [M + H]<sup>+</sup> 412.2143, found 412.2145.</div></div><div id="sec4_11_81" class="NLM_sec NLM_sec_level_3"><div id="ac_i118" class="anchor-spacer"></div><h4 class="article-section__title" id="_i118"> <i>tert</i>-Butyl (2<i>S</i>,4<i>S</i>)-2-Methyl-4-(((<i>S</i>)-1-phenylethyl)amino)piperidine-1-carboxylate (<b>35a</b>)</h4><div class="NLM_p last">To a solution of <i>tert</i>-butyl (<i>S</i>)-2-methyl-4-oxopiperidine-1-carboxylate (<b>33a</b>; 2.35 mmol) and (<i>S</i>)-1-phenylethan-1-amine (<b>34a</b>; 4.7 mmol) in DCM (10 mL) were added acetic acid and triacetoxyborohydride (4.7 mmol) and then stirred overnight at the room temperature. The mixture was diluted with ethyl acetate, washed with brine, and dried over MgSO<sub>4</sub>. After the solvent was removed <i>in vacuo</i>, the residue was purified by silica gel chromatography (hexane/EtOAc = 4:1 elution) to give the titled products (<b>35a</b>); Yield: 88.2%; <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 7.34–7.25 (m, 4H), 7.25–7.19 (m, 1H), 4.03 (h, <i>J</i> = 6.6 Hz, 1H), 3.83 (q, <i>J</i> = 6.6 Hz, 1H), 3.64 (ddd, <i>J</i> = 13.5, 5.5, 3.0 Hz, 1H), 3.18 (ddd, <i>J</i> = 13.5, 11.9, 4.0 Hz, 1H), 2.73 (dt, <i>J</i> = 9.3, 4.7 Hz, 1H), 1.77 (ddd, <i>J</i> = 13.9, 6.3, 3.9 Hz, 1H), 1.68 (ddt, <i>J</i> = 13.4, 11.0, 5.4 Hz, 1H), 1.53–1.45 (m, 1H), 1.43 (s, 9H), 1.34 (d, <i>J</i> = 1.4 Hz, 3H), 1.32 (d, <i>J</i> = 1.6 Hz, 3H), 1.30–1.25 (m, 1H); <sup>13</sup>C NMR (100 MHz, chloroform-<i>d</i>) δ 155.0, 146.1, 128.5, 126.8, 126.4, 79.0, 55.5, 48.4, 46.9, 35.1, 34.2, 31.4, 28.5, 24.9, 19.6; MS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>19</sub>H<sub>31</sub>N2O<sub>2</sub> [M + H]<sup>+</sup> 319.24, found 319.15.</div></div><div id="sec4_11_82" class="NLM_sec NLM_sec_level_3"><div id="ac_i119" class="anchor-spacer"></div><h4 class="article-section__title" id="_i119"> <i>tert</i>-Butyl (2<i>R</i>,4<i>R</i>)-2-Methyl-4-(((<i>R</i>)-1-phenylethyl)amino)piperidine-1-carboxylate (<b>35b</b>)</h4><div class="NLM_p last"><b>35b</b> was synthesized in 86% yield according to the same procedure as <b>35a</b>, from <i>tert</i>-butyl (<i>R</i>)-2-methyl-4-oxopiperidine-1-carboxylate (<b>33b</b>; 2.35 mmol) and (<i>R</i>)-1-phenylethan-1-amine (<b>34b</b>; 4.7 mmol); <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 7.35–7.18 (m, 5H), 4.03 (h, <i>J</i> = 6.6 Hz, 1H), 3.83 (q, <i>J</i> = 6.6 Hz, 1H), 3.64 (ddd, <i>J</i> = 13.5, 5.5, 3.0 Hz, 1H), 3.18 (ddd, <i>J</i> = 13.5, 11.9, 4.0 Hz, 1H), 2.73 (dt, <i>J</i> = 9.4, 4.7 Hz, 1H), 1.77 (ddd, <i>J</i> = 13.9, 6.3, 3.9 Hz, 1H), 1.68 (ddt, <i>J</i> = 13.4, 11.1, 5.4 Hz, 1H), 1.51–1.45 (m, 1H), 1.43 (s, 9H), 1.34 (d, <i>J</i> = 1.5 Hz, 3H), 1.32 (d, <i>J</i> = 1.6 Hz, 3H), 1.31–1.26 (m, 1H); <sup>13</sup>C NMR (100 MHz, chloroform-<i>d</i>) δ 155.0, 146.1, 128.5, 126.8, 126.3, 79.0, 55.5, 48.4, 46.9, 35.1, 34.2, 31.3, 28.5, 24.9, 19.5; MS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>19</sub>H<sub>31</sub>N2O<sub>2</sub> [M + H]<sup>+</sup> 319.24, found 319.15.</div></div><div id="sec4_11_83" class="NLM_sec NLM_sec_level_3"><div id="ac_i120" class="anchor-spacer"></div><h4 class="article-section__title" id="_i120"> <i>tert</i>-Butyl (2<i>S</i>,4<i>S</i>)-4-Amino-2-methylpiperidine-1-carboxylate (<b>36a</b>)</h4><div class="NLM_p last">To a mixture of <b>35a</b> (1.0 mmol) and 10% palladium hydroxide (20 wt % on carbon) in methanol and ethanol (1:1) was added acetic acid (1 equiv) dropwise, and the mixture was hydrogenated with a five-layered balloon of hydrogen at 50 °C for 2 h. The reaction mixture was filtered through Celite and washed Celite with methanol. The combined filtrate was evaporated to dryness to obtain the titled compound. Yield: 93%; <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 3.96–3.86 (m, 1H), 3.69 (ddd, <i>J</i> = 14.2, 6.6, 3.9 Hz, 1H), 3.39–3.32 (m, 2H), 2.16 (ddt, <i>J</i> = 19.6, 11.3, 4.4 Hz, 1H), 2.06 (dt, <i>J</i> = 13.3, 5.0 Hz, 1H), 1.66–1.58 (m, 1H), 1.58–1.48 (m, 1H), 1.46 (s, 9H), 1.27 (d, <i>J</i> = 6.6 Hz, 3H); <sup>13</sup>C NMR (100 MHz, methanol-<i>d</i><sub>4</sub>) δ 155.2, 80.0, 48.4, 45.3, 36.4, 33.3, 27.6, 27.3, 18.5; MS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>11</sub>H<sub>23</sub>N<sub>2</sub>O<sub>2</sub> [M + H]<sup>+</sup> 215.18, found 215.20; confirmed chirality with the (<i>S</i>)-mosher’s reagent; data were attached on the <a href="/doi/suppl/10.1021/acs.jmedchem.0c01026/suppl_file/jm0c01026_si_001.pdf" class="ext-link">Supporting Information</a>.</div></div><div id="sec4_11_84" class="NLM_sec NLM_sec_level_3"><div id="ac_i121" class="anchor-spacer"></div><h4 class="article-section__title" id="_i121"> <i>tert</i>-Butyl (2<i>R</i>,4<i>R</i>)-4-Amino-2-methylpiperidine-1-carboxylate (<b>36b</b>)</h4><div class="NLM_p last"><b>36b</b> was synthesized in 96% yield according to the same procedure as <b>36a</b>, from <b>35a</b> (1.0 mmol); <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 3.96–3.84 (m, 1H), 3.69 (ddd, <i>J</i> = 14.2, 6.6, 3.9 Hz, 1H), 3.39–3.32 (m, 2H), 2.17 (ddt, <i>J</i> = 13.9, 9.1, 6.9 Hz, 1H), 2.07 (dt, <i>J</i> = 13.2, 4.7 Hz, 1H), 1.69–1.52 (m, 2H), 1.46 (s, 9H), 1.28 (d, <i>J</i> = 6.6 Hz, 3H); <sup>13</sup>C NMR (100 MHz, methanol-<i>d</i><sub>4</sub>) δ 155.1, 80.0, 48.4, 45.4, 36.5, 33.2, 27.5, 27.3, 18.5; MS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>11</sub>H<sub>23</sub>N<sub>2</sub>O<sub>2</sub> [M + H]<sup>+</sup> 215.18, found 215.15; confirmed chirality with the (<i>S</i>)-mosher’s reagent; data were attached on the <a href="/doi/suppl/10.1021/acs.jmedchem.0c01026/suppl_file/jm0c01026_si_001.pdf" class="ext-link">Supporting Information</a>.</div></div><div id="sec4_11_85" class="NLM_sec NLM_sec_level_3"><div id="ac_i122" class="anchor-spacer"></div><h4 class="article-section__title" id="_i122"> <i>N</i>-Methyl-4-(((2<i>S</i>,4<i>S</i>)-2-methylpiperidin-4-yl)amino)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-5-carboxamide Hydrochloride (<b>37a</b>)</h4><div class="NLM_p last"><b>37a</b> was synthesized in 26% yield according to the same procedure as <b>25</b> using <b>7b</b> and <b>36a</b>; <sup>1</sup>H NMR (400 MHz, deuterium oxide) δ 8.19 (s, 1H), 7.33 (d, <i>J</i> = 3.7 Hz, 1H), 6.82 (d, <i>J</i> = 3.8 Hz, 1H), 4.53–4.42 (m, 1H), 3.65–3.46 (m, 3H), 3.27 (td, <i>J</i> = 13.4, 2.7 Hz, 1H), 2.90 (s, 3H), 2.53–2.35 (m, 2H), 1.82 (qd, <i>J</i> = 13.9, 4.3 Hz, 1H), 1.75–1.60 (m, 1H), 1.39 (d, <i>J</i> = 6.5 Hz, 3H); <sup>13</sup>C NMR (100 MHz, deuterium oxide) δ 169.4, 151.7, 138.8, 134.7, 124.5, 106.9, 106.7, 104.4, 52.3, 49.4, 43.2, 37.1, 29.0, 26.9, 18.9; MS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>15</sub>H<sub>22</sub>N<sub>5</sub>O [M + H]<sup>+</sup> 288.18, found 288.15.</div></div><div id="sec4_11_86" class="NLM_sec NLM_sec_level_3"><div id="ac_i123" class="anchor-spacer"></div><h4 class="article-section__title" id="_i123"> <i>N</i>-Methyl-4-(((2<i>R</i>,4<i>R</i>)-2-methylpiperidin-4-yl)amino)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-5-carboxamide Hydrochloride (<b>37b</b>)</h4><div class="NLM_p last"><b>37b</b> was synthesized in 62% yield according to the same procedure as <b>25</b> using <b>7b</b> and <b>36b</b>; <sup>1</sup>H NMR (400 MHz, deuterium oxide) δ 8.11 (s, 1H), 7.29 (d, <i>J</i> = 3.6 Hz, 1H), 6.71 (d, <i>J</i> = 3.7 Hz, 1H), 4.39–4.25 (m, 1H), 3.65–3.48 (m, 2H), 3.27 (td, <i>J</i> = 13.3, 2.6 Hz, 1H), 2.90 (s, 3H), 2.48–2.32 (m, 2H), 1.78 (qd, <i>J</i> = 14.0, 4.2 Hz, 1H), 1.64 (q, <i>J</i> = 12.1 Hz, 1H), 1.41 (d, <i>J</i> = 6.5 Hz, 3H); <sup>13</sup>C NMR (100 MHz, deuterium oxide) δ 169.0, 150.6, 139.7, 135.5, 123.7, 106.0, 105.3, 103.4, 51.7, 48.6, 42.6, 36.7, 28.5, 26.2, 18.3; MS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>15</sub>H<sub>22</sub>N<sub>5</sub>O [M + H]<sup>+</sup> 288.18, found 288.15.</div></div><div id="sec4_11_87" class="NLM_sec NLM_sec_level_3"><div id="ac_i124" class="anchor-spacer"></div><h4 class="article-section__title" id="_i124"> 4-(((2<i>S</i>,4<i>S</i>)-1-(4-Chlorobenzyl)-2-methylpiperidin-4-yl)amino)-<i>N</i>-methyl-1<i>H</i>-pyrrolo[2,3-b]pyridine-5-carboxamide (<b>38a</b>)</h4><div class="NLM_p last"><b>38a</b> was synthesized in 29% yield according to the same procedure as <b>20d</b> using <b>37a</b> and 4-chlorobenzaldehyde; <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 9.13 (d, <i>J</i> = 7.8 Hz, 1H), 8.21 (s, 1H), 7.31–7.23 (m, 4H), 7.02 (d, <i>J</i> = 3.4 Hz, 1H), 6.48 (d, <i>J</i> = 3.6 Hz, 1H), 6.33 (s, 1H), 4.07 (d, <i>J</i> = 13.6 Hz, 1H), 3.92 (dq, <i>J</i> = 11.4, 7.7, 5.6 Hz, 1H), 3.12 (d, <i>J</i> = 13.6 Hz, 1H), 2.96 (d, <i>J</i> = 4.6 Hz, 3H), 2.90–2.77 (m, 1H), 2.42 (ddd, <i>J</i> = 10.7, 6.0, 2.2 Hz, 1H), 2.15 (d, <i>J</i> = 12.2 Hz, 1H), 2.11–2.02 (m, 2H), 1.63–1.44 (m, 2H), 1.22 (d, <i>J</i> = 6.0 Hz, 3H); <sup>13</sup>C NMR (100 MHz, chloroform-<i>d</i>) δ 170.7, 150.1, 149.7, 143.0, 138.0, 132.6, 130.3, 128.5, 121.3, 105.3, 103.7, 102.9, 57.0, 56.1, 51.7, 51.4, 42.4, 33.6, 26.7, 21.3; HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>22</sub>H<sub>27</sub>ClN<sub>5</sub>O [M + H]<sup>+</sup> 412.1899, found 412.1902; enantiomeric purity: 99.9% <i>ee</i>.</div></div><div id="sec4_11_88" class="NLM_sec NLM_sec_level_3"><div id="ac_i125" class="anchor-spacer"></div><h4 class="article-section__title" id="_i125"> 4-(((2<i>R</i>,4<i>R</i>)-1-(4-Chlorobenzyl)-2-methylpiperidin-4-yl)amino)-<i>N</i>-methyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-5-carboxamide (<b>38b</b>)</h4><div class="NLM_p last"><b>38b</b> was synthesized in 26% yield according to the same procedure as <b>20d</b> using <b>37b</b> and 4-chlorobenzaldehyde; <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 10.50 (s, 1H), 9.08 (d, <i>J</i> = 7.9 Hz, 1H), 8.21 (s, 1H), 7.33–7.24 (m, 4H), 7.05 (d, <i>J</i> = 3.6 Hz, 1H), 6.53 (d, <i>J</i> = 3.7 Hz, 1H), 6.17–6.00 (m, 1H), 3.94 (dp, <i>J</i> = 11.5, 3.8 Hz, 1H), 3.60 (dd, <i>J</i> = 382.4, 13.6 Hz, 2H), 2.97 (d, <i>J</i> = 4.8 Hz, 3H), 2.90–2.78 (m, 1H), 2.43 (ddd, <i>J</i> = 11.0, 6.1, 2.5 Hz, 2H), 2.20–2.13 (m, 1H), 2.13–2.04 (m, 2H), 1.63–1.45 (m, 2H), 1.23 (d, <i>J</i> = 6.1 Hz, 3H); <sup>13</sup>C NMR (100 MHz, chloroform-<i>d</i>) δ 170.9, 150.6, 150.0, 143.8, 138.1, 132.6, 130.3, 128.5, 121.1, 105.2, 103.0, 57.0, 56.1, 51.7, 51.4, 42.5, 33.6, 26.7, 21.4; HRMS (ESI, <i>m</i>/<i>z</i>) calculated for C<sub>22</sub>H<sub>27</sub>ClN<sub>5</sub>O [M + H]<sup>+</sup> 412.1899, found 412.1904; enantiomeric purity: 99.6% <i>ee</i>.</div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i126"><a href="/doi/suppl/10.1021/acs.jmedchem.0c01026" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_47326" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_47326" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c01026?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01026</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Chirality confirmation of <b>36a</b> and <b>36b</b> with a chiral derivatizing agent, (<i>S</i>)-mosher’s reagent; HPLC spectra of <b>20c</b>, <b>30b–30c</b>, <b>31b</b>, <b>30d</b>, <b>30g</b>, <b>30j</b>, and <b>32b–32c</b>; chiral HPLC spectra of <b>38a</b> and <b>38b</b>; NMR spectra of final compounds; and docking poses of <b>31g</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01026/suppl_file/jm0c01026_si_001.pdf" class="ext-link">PDF</a>)</p><p class="last">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01026/suppl_file/jm0c01026_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01026/suppl_file/jm0c01026_si_001.pdf">jm0c01026_si_001.pdf (6.35 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01026/suppl_file/jm0c01026_si_002.csv">jm0c01026_si_002.csv (4.53 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.0c01026" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_10581" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_10581" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Soosung Kang</span> - <span class="hlFld-Affiliation affiliation">College
of Pharmacy and Graduate School of Pharmaceutical Sciences, Ewha Womans University, Seoul 03760, Republic
of Korea</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-7016-2417" title="Orcid link">http://orcid.org/0000-0001-7016-2417</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#3c4f4f575d525b7c594b545d125d5f12574e"><span class="__cf_email__" data-cfemail="d3a0a0b8b2bdb493b6a4bbb2fdb2b0fdb8a1">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Eunsun Park</span> - <span class="hlFld-Affiliation affiliation">College
of Pharmacy and Graduate School of Pharmaceutical Sciences, Ewha Womans University, Seoul 03760, Republic
of Korea</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sun Joo Lee</span> - <span class="hlFld-Affiliation affiliation">New
Drug Development Center, Daegu-Gyeongbuk
Medical Innovation Foundation (DGMIF), Daegu 41061, Republic
of Korea</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Heegyum Moon</span> - <span class="hlFld-Affiliation affiliation">New
Drug Development Center, Daegu-Gyeongbuk
Medical Innovation Foundation (DGMIF), Daegu 41061, Republic
of Korea</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jongmi Park</span> - <span class="hlFld-Affiliation affiliation">College
of Pharmacy and Graduate School of Pharmaceutical Sciences, Ewha Womans University, Seoul 03760, Republic
of Korea</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hyeonho Jeon</span> - <span class="hlFld-Affiliation affiliation">New
Drug Development Center, Daegu-Gyeongbuk
Medical Innovation Foundation (DGMIF), Daegu 41061, Republic
of Korea</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ji Sun Hwang</span> - <span class="hlFld-Affiliation affiliation">New
Drug Development Center, Daegu-Gyeongbuk
Medical Innovation Foundation (DGMIF), Daegu 41061, Republic
of Korea</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hayoung Hwang</span> - <span class="hlFld-Affiliation affiliation">New
Drug Development Center, Daegu-Gyeongbuk
Medical Innovation Foundation (DGMIF), Daegu 41061, Republic
of Korea</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ki Bum Hong</span> - <span class="hlFld-Affiliation affiliation">New
Drug Development Center, Daegu-Gyeongbuk
Medical Innovation Foundation (DGMIF), Daegu 41061, Republic
of Korea</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-4853-2263" title="Orcid link">http://orcid.org/0000-0002-4853-2263</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Seung-Hee Han</span> - <span class="hlFld-Affiliation affiliation">Central
Research Laboratory, KOREA PHARMA Co. Ltd, jeyakgongdan 3-gil, Hyangnam-eup, Hwaseong-si, Gyeonggi-do 18622, Republic of Korea</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sun Choi</span> - <span class="hlFld-Affiliation affiliation">College
of Pharmacy and Graduate School of Pharmaceutical Sciences, Ewha Womans University, Seoul 03760, Republic
of Korea</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-7669-7954" title="Orcid link">http://orcid.org/0000-0002-7669-7954</a></span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>E.P. and S.J.L. contributed equally. The manuscript was written through contributions of all authors. All authors have given approval to the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><p>The authors declare no competing financial interest.<br /></br></p></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i128">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_74770" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_74770" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">S.K. thanks National Research Foundation of Korea grant funded by MSIT (NRF - 2019M3E5D4065251, 2018R1A5A2025286, and 2019R1A2C2004142). We are grateful to KOREA PHARMA Co. Ltd. for their research grant.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">JAK</td><td class="NLM_def"><p class="first last">Janus kinase</p></td></tr><tr><td class="NLM_term">TYK</td><td class="NLM_def"><p class="first last">tyrosine kinase</p></td></tr><tr><td class="NLM_term">STAT</td><td class="NLM_def"><p class="first last">signal transducer and transcription</p></td></tr><tr><td class="NLM_term">IFN</td><td class="NLM_def"><p class="first last">interferon</p></td></tr><tr><td class="NLM_term">gp 130</td><td class="NLM_def"><p class="first last">glycoprotein 130</p></td></tr><tr><td class="NLM_term">IBD</td><td class="NLM_def"><p class="first last">inflammatory bowel disease</p></td></tr><tr><td class="NLM_term">RA</td><td class="NLM_def"><p class="first last">rheumatoid arthritis</p></td></tr><tr><td class="NLM_term">CTGF</td><td class="NLM_def"><p class="first last">connective tissue growth factor</p></td></tr><tr><td class="NLM_term">HSC</td><td class="NLM_def"><p class="first last">hepatic stellate cell</p></td></tr><tr><td class="NLM_term">TGF</td><td class="NLM_def"><p class="first last">Transforming growth factor</p></td></tr><tr><td class="NLM_term">IL</td><td class="NLM_def"><p class="first last">interleukin</p></td></tr><tr><td class="NLM_term">ATP</td><td class="NLM_def"><p class="first last">adenosine triphosphate</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr><tr><td class="NLM_term">MDCK</td><td class="NLM_def"><p class="first last">madin–darby canine kidney cell</p></td></tr><tr><td class="NLM_term">MRD</td><td class="NLM_def"><p class="first last">multidrug resistance proteins</p></td></tr><tr><td class="NLM_term">TK</td><td class="NLM_def"><p class="first last">tyrosine kinase</p></td></tr><tr><td class="NLM_term">TKL</td><td class="NLM_def"><p class="first last">tyrosine kinase-like</p></td></tr><tr><td class="NLM_term">ECM</td><td class="NLM_def"><p class="first last">extracellular matrix</p></td></tr><tr><td class="NLM_term">α-SMA</td><td class="NLM_def"><p class="first last">α-smooth muscle actin</p></td></tr><tr><td class="NLM_term">Col1A1</td><td class="NLM_def"><p class="first last">collagen type l α1</p></td></tr><tr><td class="NLM_term">TIMP1</td><td class="NLM_def"><p class="first last">tissue inhibitors of metalloproteinases1.</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i130">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_40107" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_40107" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 42 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kanno, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villarino, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gadina, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Shea, J. J.</span></span> <span> </span><span class="NLM_article-title">JAK inhibition as a therapeutic strategy for immune and inflammatory diseases</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">843</span>– <span class="NLM_lpage">862</span>, <span class="refDoi"> DOI: 10.1038/nrd.2017.201</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=10.1038%2Fnrd.2017.201" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=29104284" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslGqsLnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=843-862&author=D.+M.+Schwartzauthor=Y.+Kannoauthor=A.+Villarinoauthor=M.+Wardauthor=M.+Gadinaauthor=J.+J.+O%E2%80%99Shea&title=JAK+inhibition+as+a+therapeutic+strategy+for+immune+and+inflammatory+diseases&doi=10.1038%2Fnrd.2017.201"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">JAK inhibition as a therapeutic strategy for immune and inflammatory diseases</span></div><div class="casAuthors">Schwartz, Daniella M.; Kanno, Yuka; Villarino, Alejandro; Ward, Michael; Gadina, Massimo; O'Shea, John J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">843-862</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  The discovery of cytokines as key drivers of immune-mediated diseases has spurred efforts to target their assocd. signalling pathways.  Janus kinases (JAKs) are essential signalling mediators downstream of many pro-inflammatory cytokines, and small-mol. inhibitors of JAKs (jakinibs) have gained traction as safe and efficacious options for the treatment of inflammation-driven pathologies such as rheumatoid arthritis, psoriasis and inflammatory bowel disease.  Building on the clin. success of first-generation jakinibs, second-generation compds. that claim to be more selective are currently undergoing development and proceeding to clin. trials.  However, important questions remain about the advantages and limitations of improved JAK selectivity, optimal routes and dosing regimens and how best to identify patients who will benefit from jakinibs.  This Review discusses the biol. of jakinibs from a translational perspective, focusing on recent insights from clin. trials, the development of novel agents and the use of jakinibs in a spectrum of immune and inflammatory diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9TNpGT01l8rVg90H21EOLACvtfcHk0lg4ph-U2rsfBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslGqsLnO&md5=4607422f83f94c0c7194935d6d460c31</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2017.201&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2017.201%26sid%3Dliteratum%253Aachs%26aulast%3DSchwartz%26aufirst%3DD.%2BM.%26aulast%3DKanno%26aufirst%3DY.%26aulast%3DVillarino%26aufirst%3DA.%26aulast%3DWard%26aufirst%3DM.%26aulast%3DGadina%26aufirst%3DM.%26aulast%3DO%25E2%2580%2599Shea%26aufirst%3DJ.%2BJ.%26atitle%3DJAK%2520inhibition%2520as%2520a%2520therapeutic%2520strategy%2520for%2520immune%2520and%2520inflammatory%2520diseases%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26spage%3D843%26epage%3D862%26doi%3D10.1038%2Fnrd.2017.201" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">O’Shea, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villarino, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gadina, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McInnes, I. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laurence, A.</span></span> <span> </span><span class="NLM_article-title">The JAK-STAT pathway: impact on human disease and therapeutic intervention</span>. <i>Annu. Rev. Med.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">311</span>– <span class="NLM_lpage">328</span>, <span class="refDoi"> DOI: 10.1146/annurev-med-051113-024537</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=10.1146%2Fannurev-med-051113-024537" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=25587654" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjvFyhsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2015&pages=311-328&author=J.+J.+O%E2%80%99Sheaauthor=D.+M.+Schwartzauthor=A.+V.+Villarinoauthor=M.+Gadinaauthor=I.+B.+McInnesauthor=A.+Laurence&title=The+JAK-STAT+pathway%3A+impact+on+human+disease+and+therapeutic+intervention&doi=10.1146%2Fannurev-med-051113-024537"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">The JAK-STAT pathway: impact on human disease and therapeutic intervention</span></div><div class="casAuthors">O'Shea, John J.; Schwartz, Daniella M.; Villarino, Alejandro V.; Gadina, Massimo; McInnes, Iain B.; Laurence, Arian</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Medicine</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">311-328</span>CODEN:
                <span class="NLM_cas:coden">ARMCAH</span>;
        ISSN:<span class="NLM_cas:issn">0066-4219</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">A review.  The Janus kinase (JAK)-signal transducer of activators of transcription (STAT) pathway is now recognized as an evolutionarily conserved signaling pathway employed by diverse cytokines, interferons, growth factors, and related mols.  This pathway provides an elegant and remarkably straightforward mechanism whereby extracellular factors control gene expression.  It thus serves as a fundamental paradigm for how cells sense environmental cues and interpret these signals to regulate cell growth and differentiation.  Genetic mutations and polymorphisms are functionally relevant to a variety of human diseases, esp. cancer and immune-related conditions.  The clin. relevance of the pathway has been confirmed by the emergence of a new class of therapeutics that targets JAKs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMxvQCRiTbL7Vg90H21EOLACvtfcHk0lhNhzFRvzBtEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjvFyhsLY%253D&md5=19182154f1a046df039eb4a37b66bf78</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1146%2Fannurev-med-051113-024537&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-med-051113-024537%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Shea%26aufirst%3DJ.%2BJ.%26aulast%3DSchwartz%26aufirst%3DD.%2BM.%26aulast%3DVillarino%26aufirst%3DA.%2BV.%26aulast%3DGadina%26aufirst%3DM.%26aulast%3DMcInnes%26aufirst%3DI.%2BB.%26aulast%3DLaurence%26aufirst%3DA.%26atitle%3DThe%2520JAK-STAT%2520pathway%253A%2520impact%2520on%2520human%2520disease%2520and%2520therapeutic%2520intervention%26jtitle%3DAnnu.%2520Rev.%2520Med.%26date%3D2015%26volume%3D66%26spage%3D311%26epage%3D328%26doi%3D10.1146%2Fannurev-med-051113-024537" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yamaoka, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saharinen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pesu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holt, V. E. T.,  III</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silvennoinen, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Shea, J. J.</span></span> <span> </span><span class="NLM_article-title">The janus kinases (jaks)</span>. <i>Genome Biol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">253</span>, <span class="refDoi"> DOI: 10.1186/gb-2004-5-12-253</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=10.1186%2Fgb-2004-5-12-253" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=15575979" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A280%3ADC%252BD2cngtlKktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2004&pages=253&author=K.+Yamaokaauthor=P.+Saharinenauthor=M.+Pesuauthor=V.+E.+T.+Holtauthor=O.+Silvennoinenauthor=J.+J.+O%E2%80%99Shea&title=The+janus+kinases+%28jaks%29&doi=10.1186%2Fgb-2004-5-12-253"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The Janus kinases (Jaks)</span></div><div class="casAuthors">Yamaoka Kunihiro; Saharinen Pipsa; Pesu Marko; Holt Vance E T 3rd; Silvennoinen Olli; O'Shea John J</div><div class="citationInfo"><span class="NLM_cas:title">Genome biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">253</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The Janus kinase (Jak) family is one of ten recognized families of non-receptor tyrosine kinases.  Mammals have four members of this family, Jak1, Jak2, Jak3 and Tyrosine kinase 2 (Tyk2).  Birds, fish and insects also have Jaks.  Each protein has a kinase domain and a catalytically inactive pseudo-kinase domain, and they each bind cytokine receptors through amino-terminal FERM (Band-4.1, ezrin, radixin, moesin) domains.  Upon binding of cytokines to their receptors, Jaks are activated and phosphorylate the receptors, creating docking sites for signaling molecules, especially members of the signal transducer and activator of transcription (Stat) family.  Mutations of the Drosophila Jak (Hopscotch) have revealed developmental defects, and constitutive activation of Jaks in flies and humans is associated with leukemia-like syndromes.  Through the generation of Jak-deficient cell lines and gene-targeted mice, the essential, nonredundant functions of Jaks in cytokine signaling have been established.  Importantly, deficiency of Jak3 is the basis of human autosomal recessive severe combined immunodeficiency (SCID); accordingly, a selective Jak3 inhibitor has been developed, forming a new class of immunosuppressive drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS0BwDh7EYTsV1C8OJBvdfbfW6udTcc2ebi3EzGJrrpx7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2cngtlKktg%253D%253D&md5=4f8335c37dc8c20245e771224c1d5b12</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1186%2Fgb-2004-5-12-253&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fgb-2004-5-12-253%26sid%3Dliteratum%253Aachs%26aulast%3DYamaoka%26aufirst%3DK.%26aulast%3DSaharinen%26aufirst%3DP.%26aulast%3DPesu%26aufirst%3DM.%26aulast%3DHolt%26aufirst%3DV.%2BE.%2BT.%26aulast%3DSilvennoinen%26aufirst%3DO.%26aulast%3DO%25E2%2580%2599Shea%26aufirst%3DJ.%2BJ.%26atitle%3DThe%2520janus%2520kinases%2520%2528jaks%2529%26jtitle%3DGenome%2520Biol.%26date%3D2004%26volume%3D5%26spage%3D253%26doi%3D10.1186%2Fgb-2004-5-12-253" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Banerjee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biehl, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gadina, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasni, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, D. M.</span></span> <span> </span><span class="NLM_article-title">JAK–STAT signaling as a target for inflammatory and autoimmune diseases: current and future prospects</span>. <i>Drugs</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>77</i></span>,  <span class="NLM_fpage">521</span>– <span class="NLM_lpage">546</span>, <span class="refDoi"> DOI: 10.1007/s40265-017-0701-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=10.1007%2Fs40265-017-0701-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=28255960" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjs12qsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2017&pages=521-546&author=S.+Banerjeeauthor=A.+Biehlauthor=M.+Gadinaauthor=S.+Hasniauthor=D.+M.+Schwartz&title=JAK%E2%80%93STAT+signaling+as+a+target+for+inflammatory+and+autoimmune+diseases%3A+current+and+future+prospects&doi=10.1007%2Fs40265-017-0701-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">JAK-STAT Signaling as a Target for Inflammatory and Autoimmune Diseases: Current and Future Prospects</span></div><div class="casAuthors">Banerjee, Shubhasree; Biehl, Ann; Gadina, Massimo; Hasni, Sarfaraz; Schwartz, Daniella M.</div><div class="citationInfo"><span class="NLM_cas:title">Drugs</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">77</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">521-546</span>CODEN:
                <span class="NLM_cas:coden">DRUGAY</span>;
        ISSN:<span class="NLM_cas:issn">0012-6667</span>.
    
            (<span class="NLM_cas:orgname">Springer International Publishing AG</span>)
        </div><div class="casAbstract">The Janus kinase/signal transduction and activator of transcription (JAK-STAT) signaling pathway is implicated in the pathogenesis of inflammatory and autoimmune diseases including rheumatoid arthritis, psoriasis, and inflammatory bowel disease.  Many cytokines involved in the pathogenesis of autoimmune and inflammatory diseases use JAKs and STATs to transduce intracellular signals.  Mutations in JAK and STAT genes cause a no. of immunodeficiency syndromes, and polymorphisms in these genes are assocd. with autoimmune diseases.  The success of small-mol. JAK inhibitors (Jakinibs) in the treatment of rheumatol. disease demonstrates that intracellular signaling pathways can be targeted therapeutically to treat autoimmunity.  Tofacitinib, the first rheumatol. Jakinib, is US Food and Drug Administration (FDA) approved for rheumatoid arthritis and is currently under investigation for other autoimmune diseases.  Many other Jakinibs are in preclin. development or in various phases of clin. trials.  This review describes the JAK-STAT pathway, outlines its role in autoimmunity, and explains the rationale/pre-clin. evidence for targeting JAK-STAT signaling.  The safety and clin. efficacy of the Jakinibs are reviewed, starting with the FDA-approved Jakinib tofacitinib, and continuing on to next-generation Jakinibs.  Recent and ongoing studies are emphasized, with a focus on emerging indications for JAK inhibition and novel mechanisms of JAK-STAT signaling blockade.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGXi4XmOlzP7Vg90H21EOLACvtfcHk0lhNhzFRvzBtEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjs12qsbg%253D&md5=9d06038330083ff3d7f4516578987524</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1007%2Fs40265-017-0701-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs40265-017-0701-9%26sid%3Dliteratum%253Aachs%26aulast%3DBanerjee%26aufirst%3DS.%26aulast%3DBiehl%26aufirst%3DA.%26aulast%3DGadina%26aufirst%3DM.%26aulast%3DHasni%26aufirst%3DS.%26aulast%3DSchwartz%26aufirst%3DD.%2BM.%26atitle%3DJAK%25E2%2580%2593STAT%2520signaling%2520as%2520a%2520target%2520for%2520inflammatory%2520and%2520autoimmune%2520diseases%253A%2520current%2520and%2520future%2520prospects%26jtitle%3DDrugs%26date%3D2017%26volume%3D77%26spage%3D521%26epage%3D546%26doi%3D10.1007%2Fs40265-017-0701-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Danese, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grisham, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hodge, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Telliez, J.-B.</span></span> <span> </span><span class="NLM_article-title">JAK inhibition using tofacitinib for inflammatory bowel disease treatment: a hub for multiple inflammatory cytokines</span>. <i>Am. J. Physiol. Gastrointest. Liver Physiol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>310</i></span>,  <span class="NLM_fpage">G155</span>– <span class="NLM_lpage">G162</span>, <span class="refDoi"> DOI: 10.1152/ajpgi.00311.2015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=10.1152%2Fajpgi.00311.2015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=26608188" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=310&publication_year=2016&pages=G155-G162&author=S.+Daneseauthor=M.+Grishamauthor=J.+Hodgeauthor=J.-B.+Telliez&title=JAK+inhibition+using+tofacitinib+for+inflammatory+bowel+disease+treatment%3A+a+hub+for+multiple+inflammatory+cytokines&doi=10.1152%2Fajpgi.00311.2015"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1152%2Fajpgi.00311.2015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1152%252Fajpgi.00311.2015%26sid%3Dliteratum%253Aachs%26aulast%3DDanese%26aufirst%3DS.%26aulast%3DGrisham%26aufirst%3DM.%26aulast%3DHodge%26aufirst%3DJ.%26aulast%3DTelliez%26aufirst%3DJ.-B.%26atitle%3DJAK%2520inhibition%2520using%2520tofacitinib%2520for%2520inflammatory%2520bowel%2520disease%2520treatment%253A%2520a%2520hub%2520for%2520multiple%2520inflammatory%2520cytokines%26jtitle%3DAm.%2520J.%2520Physiol.%2520Gastrointest.%2520Liver%2520Physiol.%26date%3D2016%26volume%3D310%26spage%3DG155%26epage%3DG162%26doi%3D10.1152%2Fajpgi.00311.2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Spergel, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mertzman, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kempson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stachura, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haque, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lippy, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galella, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pitt, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fura, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gillooly, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McIntyre, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tokarski, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sack, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, P. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrish, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nadler, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salter-Cid, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schieven, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wrobleski, S. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pitts, W. J.</span></span> <span> </span><span class="NLM_article-title">Discovery of a JAK1/3 inhibitor and use of a prodrug to demonstrate efficacy in a model of rheumatoid arthritis</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">306</span>– <span class="NLM_lpage">311</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.8b00508</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.8b00508" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjtVSgtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=306-311&author=S.+H.+Spergelauthor=M.+E.+Mertzmanauthor=J.+Kempsonauthor=J.+Guoauthor=S.+Stachuraauthor=L.+Haqueauthor=J.+S.+Lippyauthor=R.+F.+Zhangauthor=M.+Galellaauthor=S.+Pittauthor=G.+Shenauthor=A.+Furaauthor=K.+Gilloolyauthor=K.+W.+McIntyreauthor=V.+Tangauthor=J.+Tokarskiauthor=J.+S.+Sackauthor=J.+Khanauthor=P.+H.+Carterauthor=J.+C.+Barrishauthor=S.+G.+Nadlerauthor=L.+M.+Salter-Cidauthor=G.+L.+Schievenauthor=S.+T.+Wrobleskiauthor=W.+J.+Pitts&title=Discovery+of+a+JAK1%2F3+inhibitor+and+use+of+a+prodrug+to+demonstrate+efficacy+in+a+model+of+rheumatoid+arthritis&doi=10.1021%2Facsmedchemlett.8b00508"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a JAK1/3 Inhibitor and Use of a Prodrug To Demonstrate Efficacy in a Model of Rheumatoid Arthritis</span></div><div class="casAuthors">Spergel, Steven H.; Mertzman, Michael E.; Kempson, James; Guo, Junqing; Stachura, Sylwia; Haque, Lauren; Lippy, Jonathan S.; Zhang, Rosemary F.; Galella, Michael; Pitt, Sidney; Shen, Guoxiang; Fura, Aberra; Gillooly, Kathleen; McIntyre, Kim W.; Tang, Vicky; Tokarski, John; Sack, John S.; Khan, Javed; Carter, Percy H.; Barrish, Joel C.; Nadler, Steven G.; Salter-Cid, Luisa M.; Schieven, Gary L.; Wrobleski, Stephen T.; Pitts, William J.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">306-311</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The four members of the Janus family of nonreceptor tyrosine kinases play a significant role in immune function.  The JAK family kinase inhibitor, tofacitinib 1, has been approved in the United States for use in rheumatoid arthritis (RA) patients.  A no. of JAK inhibitors with a variety of JAK family selectivity profiles are currently in clin. trials.  Our goal was to identify inhibitors that were functionally selective for JAK1 and JAK3.  Compd. 22 was prepd. with the desired functional selectivity profile, but it suffered from poor absorption related to phys. properties.  Use of the phosphate prodrug 32 enabled progression to a murine collagen induced arthritis (CIA) model.  The demonstration of a robust efficacy in the CIA model suggests that use of phosphate prodrugs may resolve issues with progressing this chemotype for the treatment of autoimmune diseases such as RA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQKLujiDlteLVg90H21EOLACvtfcHk0lgA9qmh4vmZIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjtVSgtr4%253D&md5=7562490a531e325dc7404dc8e64d090b</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.8b00508&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.8b00508%26sid%3Dliteratum%253Aachs%26aulast%3DSpergel%26aufirst%3DS.%2BH.%26aulast%3DMertzman%26aufirst%3DM.%2BE.%26aulast%3DKempson%26aufirst%3DJ.%26aulast%3DGuo%26aufirst%3DJ.%26aulast%3DStachura%26aufirst%3DS.%26aulast%3DHaque%26aufirst%3DL.%26aulast%3DLippy%26aufirst%3DJ.%2BS.%26aulast%3DZhang%26aufirst%3DR.%2BF.%26aulast%3DGalella%26aufirst%3DM.%26aulast%3DPitt%26aufirst%3DS.%26aulast%3DShen%26aufirst%3DG.%26aulast%3DFura%26aufirst%3DA.%26aulast%3DGillooly%26aufirst%3DK.%26aulast%3DMcIntyre%26aufirst%3DK.%2BW.%26aulast%3DTang%26aufirst%3DV.%26aulast%3DTokarski%26aufirst%3DJ.%26aulast%3DSack%26aufirst%3DJ.%2BS.%26aulast%3DKhan%26aufirst%3DJ.%26aulast%3DCarter%26aufirst%3DP.%2BH.%26aulast%3DBarrish%26aufirst%3DJ.%2BC.%26aulast%3DNadler%26aufirst%3DS.%2BG.%26aulast%3DSalter-Cid%26aufirst%3DL.%2BM.%26aulast%3DSchieven%26aufirst%3DG.%2BL.%26aulast%3DWrobleski%26aufirst%3DS.%2BT.%26aulast%3DPitts%26aufirst%3DW.%2BJ.%26atitle%3DDiscovery%2520of%2520a%2520JAK1%252F3%2520inhibitor%2520and%2520use%2520of%2520a%2520prodrug%2520to%2520demonstrate%2520efficacy%2520in%2520a%2520model%2520of%2520rheumatoid%2520arthritis%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2019%26volume%3D10%26spage%3D306%26epage%3D311%26doi%3D10.1021%2Facsmedchemlett.8b00508" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Villarino, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kanno, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferdinand, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Shea, J. J.</span></span> <span> </span><span class="NLM_article-title">Mechanisms of JAK/STAT signaling in immunity and disease</span>. <i>J. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>194</i></span>,  <span class="NLM_fpage">21</span>– <span class="NLM_lpage">27</span>, <span class="refDoi"> DOI: 10.4049/jimmunol.1401867</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=10.4049%2Fjimmunol.1401867" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=25527793" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFWls7rF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=194&publication_year=2015&pages=21-27&author=A.+V.+Villarinoauthor=Y.+Kannoauthor=J.+R.+Ferdinandauthor=J.+J.+O%E2%80%99Shea&title=Mechanisms+of+JAK%2FSTAT+signaling+in+immunity+and+disease&doi=10.4049%2Fjimmunol.1401867"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanisms of Jak/STAT Signaling in Immunity and Disease</span></div><div class="casAuthors">Villarino, Alejandro V.; Kanno, Yuka; Ferdinand, John R.; O'Shea, John J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Immunology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">194</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">21-27</span>CODEN:
                <span class="NLM_cas:coden">JOIMA3</span>;
        ISSN:<span class="NLM_cas:issn">0022-1767</span>.
    
            (<span class="NLM_cas:orgname">American Association of Immunologists</span>)
        </div><div class="casAbstract">A review.  More than two decades ago, expts. on the antiviral mechanisms of IFNs led to the discovery of JAKs and their downstream effectors, the STAT proteins.  This pathway has since become a paradigm for membrane-to-nucleus signaling and explains how a broad range of sol. factors, including cytokines and hormones, mediate their diverse functions.  Jak/STAT research has not only impacted basic science, particularly in the context of intercellular communication and cell-extrinsic control of gene expression, it also has become a prototype for transition from bench to bedside, culminating in the development and clin. implementation of pathway-specific therapeutics.  This brief review synthesizes our current understanding of Jak/STAT biol. while taking stock of the lessons learned and the challenges that lie ahead.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotOsFFpF-ChLVg90H21EOLACvtfcHk0lhPA6vjO5TV9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitFWls7rF&md5=777f0f64db038ca8a7f17554931e0655</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.4049%2Fjimmunol.1401867&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4049%252Fjimmunol.1401867%26sid%3Dliteratum%253Aachs%26aulast%3DVillarino%26aufirst%3DA.%2BV.%26aulast%3DKanno%26aufirst%3DY.%26aulast%3DFerdinand%26aufirst%3DJ.%2BR.%26aulast%3DO%25E2%2580%2599Shea%26aufirst%3DJ.%2BJ.%26atitle%3DMechanisms%2520of%2520JAK%252FSTAT%2520signaling%2520in%2520immunity%2520and%2520disease%26jtitle%3DJ.%2520Immunol.%26date%3D2015%26volume%3D194%26spage%3D21%26epage%3D27%26doi%3D10.4049%2Fjimmunol.1401867" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, X.</span></span> <span> </span><span class="NLM_article-title">Transforming growth factor β1 promotes migration and invasion in HepG2 cells: Epithelial-to-mesenchymal transition via JAK/STAT3 signaling</span>. <i>Int. J. Mol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">129</span>– <span class="NLM_lpage">136</span>, <span class="refDoi"> DOI: 10.3892/ijmm.2017.3228</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=10.3892%2Fijmm.2017.3228" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=29115395" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVyqtbfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2018&pages=129-136&author=X.+L.+Linauthor=M.+Liuauthor=Y.+Liuauthor=H.+Huauthor=Y.+Panauthor=W.+Zouauthor=X.+Fanauthor=X.+Hu&title=Transforming+growth+factor+%CE%B21+promotes+migration+and+invasion+in+HepG2+cells%3A+Epithelial-to-mesenchymal+transition+via+JAK%2FSTAT3+signaling&doi=10.3892%2Fijmm.2017.3228"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Transforming growth factor β1 promotes migration and invasion in HepG2 cells: epithelial-to-mesenchymal transition via JAK/STAT3 signaling</span></div><div class="casAuthors">Lin, Xiao-Long; Liu, Mihua; Liu, Yuanbo; Hu, Huijun; Pan, Yongquan; Zou, Weiwen; Fan, Xiaojuan; Hu, Xuemei</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Molecular Medicine</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">129-136</span>CODEN:
                <span class="NLM_cas:coden">IJMMFG</span>;
        ISSN:<span class="NLM_cas:issn">1791-244X</span>.
    
            (<span class="NLM_cas:orgname">Spandidos Publications Ltd.</span>)
        </div><div class="casAbstract">Transforming growth factor β1 (TGFβ1) is a cytokine with multiple functions.  TGFβ1 significantly induces migration and invasion of liver cancer cells.  However, the mol. mechanisms underlying this effect remain unclear.  Epithelial-to-mesenchymal transition (EMT) is crucial for the development of invasion and metastasis in human cancers.  The aim of the present study was to det. whether TGFβ1-induced EMT promoted migration and invasion in HepG2 cells.  The underlying mechanism and the effect of EMT on HepG2 cells were also investigated.  The results demonstrated that TGFβ1 may i nduce E MT t o p romote migration and invasion of HepG2 cells, and this effect depends on activation of the Janus kinase/signal transducer and activator of transcription 3 (JAK/STAT3) signaling pathway.  JAK/STAT3 signaling is involved in human malignancies, including lung cancer, and is implicated in cell transformation, tumorigenicity, EMT and metastasis.  In the present study, TGFβ1 also activated JAK/STAT3 signaling in HepG2 cells and promoted Twist expression, but these events were abolished by treatment with the STAT3 inhibitor AG490.  Addnl., Twist siRNA blocked TGFβ1-induced EMT.  Thus, TGFβ1 was shown to induce EMT, thereby promoting the migration and invasion of HepG2 cells via JAK/STAT3/Twist signalling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOVmKjh2cCjrVg90H21EOLACvtfcHk0lhPA6vjO5TV9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVyqtbfK&md5=c65d44bd68d4cb4e608c7fd39093e92d</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.3892%2Fijmm.2017.3228&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3892%252Fijmm.2017.3228%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DX.%2BL.%26aulast%3DLiu%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DHu%26aufirst%3DH.%26aulast%3DPan%26aufirst%3DY.%26aulast%3DZou%26aufirst%3DW.%26aulast%3DFan%26aufirst%3DX.%26aulast%3DHu%26aufirst%3DX.%26atitle%3DTransforming%2520growth%2520factor%2520%25CE%25B21%2520promotes%2520migration%2520and%2520invasion%2520in%2520HepG2%2520cells%253A%2520Epithelial-to-mesenchymal%2520transition%2520via%2520JAK%252FSTAT3%2520signaling%26jtitle%3DInt.%2520J.%2520Mol.%2520Med.%26date%3D2018%26volume%3D41%26spage%3D129%26epage%3D136%26doi%3D10.3892%2Fijmm.2017.3228" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gu, Y.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, W.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, W.</span></span> <span> </span><span class="NLM_article-title">Targeted blockade of JAK/STAT3 signaling inhibits proliferation, migration and collagen production as well as inducing the apoptosis of hepatic stellate cells</span>. <i>Int. J. Mol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">903</span>– <span class="NLM_lpage">911</span>, <span class="refDoi"> DOI: 10.3892/ijmm.2016.2692</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=10.3892%2Fijmm.2016.2692" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=27460897" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFOqurvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2016&pages=903-911&author=Y.-J.+Guauthor=W.-Y.+Sunauthor=S.+Zhangauthor=X.+R.+Liauthor=W.+Wei&title=Targeted+blockade+of+JAK%2FSTAT3+signaling+inhibits+proliferation%2C+migration+and+collagen+production+as+well+as+inducing+the+apoptosis+of+hepatic+stellate+cells&doi=10.3892%2Fijmm.2016.2692"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted blockade of JAK/STAT3 signaling inhibits proliferation, migration and collagen production as well as inducing the apoptosis of hepatic stellate cells</span></div><div class="casAuthors">Gu, Yuan-Jing; Sun, Wu-Yi; Zhang, Sen; Li, Xin-Ran; Wei, Wei</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Molecular Medicine</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">903-911</span>CODEN:
                <span class="NLM_cas:coden">IJMMFG</span>;
        ISSN:<span class="NLM_cas:issn">1791-244X</span>.
    
            (<span class="NLM_cas:orgname">Spandidos Publications Ltd.</span>)
        </div><div class="casAbstract">Protein tyrosine kinases belonging to the Janus kinase (JAK) family are assocd. with many cytokine receptors, which, on ligand binding, regulate important cellular functions such as proliferation, apoptosis and differentiation.  The protective effects of JAK inhibitors on fibrotic diseases such as myelofibrosis and bone marrow fibrosis have been demonstrated in previous studies.  The JAK inhibitor SHR0302 is a synthetic mol. that potently inhibits all members of the JAK family, particularly JAK1.  However, its effect on hepatic fibrosis has not been investigated to date, to the best of our knowledge.  In the present study, the effects of SHR0302 on the activation, proliferation, migration and apoptosis of hepatic stellate cells (HSCs) as well as HSC collagen prodn. were investigated.  Our data demonstrated that treatment with SHR0302 (10-9-10-5 mol/l) exerted an inhibitory effect on the activation, proliferation and migration of HSCs.  In addn., the expression of collagen I and collagen III were significantly decreased following treatment with SHR0302.  Furthermore, SHR0302 induced the apoptosis of HSCs, which was demonstrated by Annexin V/PI staining.  SHR0302 significantly increased the activation of caspase-3 and Bax in HSCs whereas it decreased the expression of Bcl-2.  SHR0302 also inhibited the activation of Akt signaling pathway.  The pharmacol. inhibition of the JAK1/signal transducer and activator of transcription (STAT)3 pathway led to the disruption of functions essential for HSC growth.  Taken together, these findings provide evidence that SHR0302 may have the potential to alleviate hepatic fibrosis by targeting HSC functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrU0x8DPYNMILVg90H21EOLACvtfcHk0lgzTPVnxIhtTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFOqurvP&md5=4fd8d79529c979366f4593e3d8205c9e</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.3892%2Fijmm.2016.2692&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3892%252Fijmm.2016.2692%26sid%3Dliteratum%253Aachs%26aulast%3DGu%26aufirst%3DY.-J.%26aulast%3DSun%26aufirst%3DW.-Y.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DX.%2BR.%26aulast%3DWei%26aufirst%3DW.%26atitle%3DTargeted%2520blockade%2520of%2520JAK%252FSTAT3%2520signaling%2520inhibits%2520proliferation%252C%2520migration%2520and%2520collagen%2520production%2520as%2520well%2520as%2520inducing%2520the%2520apoptosis%2520of%2520hepatic%2520stellate%2520cells%26jtitle%3DInt.%2520J.%2520Mol.%2520Med.%26date%3D2016%26volume%3D38%26spage%3D903%26epage%3D911%26doi%3D10.3892%2Fijmm.2016.2692" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nadalin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koenigsrainer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dooley, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">ten Dijke, P.</span></span> <span> </span><span class="NLM_article-title">Transforming growth factor-β (TGF-β)-mediated connective tissue growth factor (CTGF) expression in hepatic stellate cells requires Stat3 signaling Activity</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>288</i></span>,  <span class="NLM_fpage">30708</span>– <span class="NLM_lpage">30719</span>, <span class="refDoi"> DOI: 10.1074/jbc.M113.478685</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=10.1074%2Fjbc.M113.478685" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=24005672" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1Cru7%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=288&publication_year=2013&pages=30708-30719&author=Y.+Liuauthor=H.+Liuauthor=C.+Meyerauthor=J.+Liauthor=S.+Nadalinauthor=A.+Koenigsrainerauthor=H.+Wengauthor=S.+Dooleyauthor=P.+ten+Dijke&title=Transforming+growth+factor-%CE%B2+%28TGF-%CE%B2%29-mediated+connective+tissue+growth+factor+%28CTGF%29+expression+in+hepatic+stellate+cells+requires+Stat3+signaling+Activity&doi=10.1074%2Fjbc.M113.478685"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Transforming Growth Factor-β (TGF-β)-mediated Connective Tissue Growth Factor (CTGF) Expression in Hepatic Stellate Cells Requires Stat3 Signaling Activation</span></div><div class="casAuthors">Liu, Yan; Liu, Heng; Meyer, Christoph; Li, Jun; Nadalin, Silvio; Koenigsrainer, Alfred; Weng, Honglei; Dooley, Steven; ten Dijke, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">288</span>
        (<span class="NLM_cas:issue">42</span>),
    <span class="NLM_cas:pages">30708-30719</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">In fibrotic liver, connective tissue growth factor (CTGF) is constantly expressed in activated hepatic stellate cells (HSCs) and acts downstream of TGF-β to modulate extracellular matrix prodn.  Distinct from other cell types in which Smad signaling plays major role in regulating CTGF prodn., TGF-β stimulated CTGF expression in activated HSCs is only in part dependent on Smad3.  Other signaling mols. like MAPKs and PI3Ks may also participate in this process, and the underlying mechanisms have yet to be clarified.  In this study, we report involvement of Stat3 activation in modulating CTGF prodn. upon TGF-β challenge in activated HSCs.  Stat3 is phosphorylated via JAK1 and acts as a crit. ALK5 (activin receptor-like kinase 5) downstream signaling mol. to mediate CTGF expression.  This process requires de novo gene transcription and is addnl. modulated by MEK1/2, JNK, and PI3K pathways.  Cell-specific knockdown of Smad3 partially decreases CTGF prodn., whereas it has no significant influence on Stat3 activation.  The total CTGF prodn. induced by TGF-β in activated HSCs is therefore, to a large extent, dependent on the balance and integration of the canonical Smad3 and Stat3 signaling pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8flotEXW6yLVg90H21EOLACvtfcHk0lgzTPVnxIhtTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1Cru7%252FJ&md5=c6d61d93df1ad6fbd5ab67a4ac801d09</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M113.478685&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M113.478685%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DMeyer%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DNadalin%26aufirst%3DS.%26aulast%3DKoenigsrainer%26aufirst%3DA.%26aulast%3DWeng%26aufirst%3DH.%26aulast%3DDooley%26aufirst%3DS.%26aulast%3Dten%2BDijke%26aufirst%3DP.%26atitle%3DTransforming%2520growth%2520factor-%25CE%25B2%2520%2528TGF-%25CE%25B2%2529-mediated%2520connective%2520tissue%2520growth%2520factor%2520%2528CTGF%2529%2520expression%2520in%2520hepatic%2520stellate%2520cells%2520requires%2520Stat3%2520signaling%2520Activity%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2013%26volume%3D288%26spage%3D30708%26epage%3D30719%26doi%3D10.1074%2Fjbc.M113.478685" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tang, L.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heller, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, L.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y. E.</span></span> <span> </span><span class="NLM_article-title">Transforming Growth Factor-β (TGF-β) Directly Activates the JAK1-STAT3 Axis to Induce Hepatic Fibrosis in Coordination with the SMAD Pathway</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>292</i></span>,  <span class="NLM_fpage">4302</span>– <span class="NLM_lpage">4312</span>, <span class="refDoi"> DOI: 10.1074/jbc.M116.773085</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=10.1074%2Fjbc.M116.773085" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=28154170" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A528%3ADC%252BC2sXktFSltLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=292&publication_year=2017&pages=4302-4312&author=L.-Y.+Tangauthor=M.+Hellerauthor=Z.+Mengauthor=L.-R.+Yuauthor=Y.+Tangauthor=M.+Zhouauthor=Y.+E.+Zhang&title=Transforming+Growth+Factor-%CE%B2+%28TGF-%CE%B2%29+Directly+Activates+the+JAK1-STAT3+Axis+to+Induce+Hepatic+Fibrosis+in+Coordination+with+the+SMAD+Pathway&doi=10.1074%2Fjbc.M116.773085"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Transforming Growth Factor-β (TGF-β) Directly Activates the JAK1-STAT3 Axis to Induce Hepatic Fibrosis in Coordination with the SMAD Pathway</span></div><div class="casAuthors">Tang, Liu-Ya; Heller, Mary; Meng, Zhaojing; Yu, Li-Rong; Tang, Yi; Zhou, Ming; Zhang, Ying E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">292</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4302-4312</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Transforming growth factor-β (TGF-β) signals through both SMAD and non-SMAD pathways to elicit a wide array of biol. effects.  Existing data have shown the assocn. and coordination between STATs and SMADs in mediating TGF-β functions in hepatic cells, but it is not clear how STATs are activated under these circumstances.  Here, we report that JAK1 is a constitutive TGFβRI binding protein and is absolutely required for phosphorylation of STATs in a SMAD-independent manner within minutes of TGF-β stimulation.  Following the activation of SMADs, TGF-β also induces a second phase of STAT phosphorylation that requires SMADs, de novo protein synthesis, and contribution from JAK1.  Our global gene expression profiling indicates that the non-SMAD JAK1/STAT pathway is essential for the expression of a subset of TGF-β target genes in hepatic stellate cells, and the cooperation between the JAK1-STAT3 and SMAD pathways is crit. to the roles of TGF-β in liver fibrosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptJ4wllzrSp7Vg90H21EOLACvtfcHk0lgzTPVnxIhtTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXktFSltLo%253D&md5=fefeed5ed048ba85372ea4fc30619e47</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M116.773085&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M116.773085%26sid%3Dliteratum%253Aachs%26aulast%3DTang%26aufirst%3DL.-Y.%26aulast%3DHeller%26aufirst%3DM.%26aulast%3DMeng%26aufirst%3DZ.%26aulast%3DYu%26aufirst%3DL.-R.%26aulast%3DTang%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DY.%2BE.%26atitle%3DTransforming%2520Growth%2520Factor-%25CE%25B2%2520%2528TGF-%25CE%25B2%2529%2520Directly%2520Activates%2520the%2520JAK1-STAT3%2520Axis%2520to%2520Induce%2520Hepatic%2520Fibrosis%2520in%2520Coordination%2520with%2520the%2520SMAD%2520Pathway%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2017%26volume%3D292%26spage%3D4302%26epage%3D4312%26doi%3D10.1074%2Fjbc.M116.773085" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nielsen, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clausen, M. H.</span></span> <span> </span><span class="NLM_article-title">FDA-approved small-molecule kinase inhibitors</span>. <i>Trends Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">422</span>– <span class="NLM_lpage">439</span>, <span class="refDoi"> DOI: 10.1016/j.tips.2015.04.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=10.1016%2Fj.tips.2015.04.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=25975227" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A528%3ADC%252BC2MXns1KisLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2015&pages=422-439&author=P.+Wuauthor=T.+E.+Nielsenauthor=M.+H.+Clausen&title=FDA-approved+small-molecule+kinase+inhibitors&doi=10.1016%2Fj.tips.2015.04.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">FDA-approved small-molecule kinase inhibitors</span></div><div class="casAuthors">Wu, Peng; Nielsen, Thomas E.; Clausen, Mads H.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">422-439</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Kinases have emerged as one of the most intensively pursued targets in current pharmacol. research, esp. for cancer, due to their crit. roles in cellular signaling.  To date, the US FDA has approved 28 small-mol. kinase inhibitors, half of which were approved in the past 3 years.  While the clin. data of these approved mols. are widely presented and structure-activity relationship (SAR) has been reported for individual mols., an updated review that analyzes all approved mols. and summarizes current achievements and trends in the field has yet to be found.  Here we present all approved small-mol. kinase inhibitors with an emphasis on binding mechanism and structural features, summarize current challenges, and discuss future directions in this field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrudyvM3eO09bVg90H21EOLACvtfcHk0lhYkTStMCg22A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXns1KisLg%253D&md5=a6aa3dbb11e6fe3001fa5fa03a4c950a</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2015.04.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2015.04.005%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DP.%26aulast%3DNielsen%26aufirst%3DT.%2BE.%26aulast%3DClausen%26aufirst%3DM.%2BH.%26atitle%3DFDA-approved%2520small-molecule%2520kinase%2520inhibitors%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2015%26volume%3D36%26spage%3D422%26epage%3D439%26doi%3D10.1016%2Fj.tips.2015.04.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Verden, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dimbil, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kyle, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overstreet, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, K. B.</span></span> <span> </span><span class="NLM_article-title">Analysis of spontaneous postmarket case reports submitted to the FDA regarding thromboembolic adverse events and JAK inhibitors</span>. <i>Drug Saf.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">357</span>– <span class="NLM_lpage">361</span>, <span class="refDoi"> DOI: 10.1007/s40264-017-0622-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=10.1007%2Fs40264-017-0622-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=29196988" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFektbnL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2018&pages=357-361&author=A.+Verdenauthor=M.+Dimbilauthor=R.+Kyleauthor=B.+Overstreetauthor=K.+B.+Hoffman&title=Analysis+of+spontaneous+postmarket+case+reports+submitted+to+the+FDA+regarding+thromboembolic+adverse+events+and+JAK+inhibitors&doi=10.1007%2Fs40264-017-0622-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Analysis of Spontaneous Postmarket Case Reports Submitted to the FDA Regarding Thromboembolic Adverse Events and JAK Inhibitors</span></div><div class="casAuthors">Verden, Abril; Dimbil, Mo; Kyle, Robert; Overstreet, Brian; Hoffman, Keith B.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Safety</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">357-361</span>CODEN:
                <span class="NLM_cas:coden">DRSAEA</span>;
        ISSN:<span class="NLM_cas:issn">0114-5916</span>.
    
            (<span class="NLM_cas:orgname">Springer International Publishing AG</span>)
        </div><div class="casAbstract">Introduction: The Janus kinase (JAK) inhibitor baricitinib is approved in Europe and Japan for the treatment of rheumatoid arthritis.  In Apr. 2017, the US FDA expressed concern about thromboembolic events (deep venous thrombosis [DVT] and pulmonary embolism [PE]) obsd. in placebo-controlled clin. trials of baricitinib.  The European and Japanese labels for baricitinib were recently updated to include a precaution related to potential thromboembolic events in patients at risk.  Given that the FDA-approved drugs tofacitinib and ruxolitinib are in the same class, we conducted a safety review of the FDA's Adverse Event Reporting System (FAERS) to assess postmarketing reporting rates for related thromboembolic risks.  Methods: Adverse event (AE) data for tofacitinib, tofacitinib extended-release (XR), and ruxolitinib were obtained from the FAERS.  Reporting odds ratios (RORs) and the R package 'PhViD' to est. the empirical Bayesian geometric mean (EBGM) were used to detect AEs with higher-than-expected reporting rates within the FAERS.  Results: We did not find evidence in the FAERS for elevated reporting rates for DVT and PE across the three JAK inhibitors we analyzed.  However, multiple drug-AE combinations relating to thromboembolic events had both RORs and EBGM values above 1, indicating a trend toward higher-than-expected reporting rates.  For pulmonary thrombosis, the ROR values for ruxolitinib, tofacitinib, and tofacitinib XR were 1.46 (95% confidence interval [CI] 0.76-2.80), 2.46 (1.55-3.91), and 2.48 (0.80-7.71), resp., while the EBGM values were 1.25 (0.70), 2.46 (1.64), and 1.56 (0.57), resp.  Ruxolitinib had ROR values of 4.08 (2.25-7.38) and 1.22 (0.97-1.53) for portal vein thrombosis and thrombosis, resp.  The EBGM values for the same drug-AE combinations were 3.04 (1.79) and 1.16 (0.96).  Conclusions: Our safety review of postmarketing FAERS reports assocd. with three FDA-approved JAK inhibitors did not find elevated reporting rates for DVT and PE specifically.  However, the FAERS data indicated that pulmonary thrombosis may potentially be a class-wide issue for JAK inhibitors.  Portal vein thrombosis may also be a potential risk for ruxolitinib.  While these FAERS data add to a growing body of evidence that JAK inhibitors may be contraindicated in patients at risk of thromboembolic events, the data need to be confirmed by future AE reporting trends, anal. of electronic health records, and/or future clin. trials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrB4-lwzE2yXLVg90H21EOLACvtfcHk0lhYkTStMCg22A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFektbnL&md5=c4679fffffff7f34b35b5ba34d88817b</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1007%2Fs40264-017-0622-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs40264-017-0622-2%26sid%3Dliteratum%253Aachs%26aulast%3DVerden%26aufirst%3DA.%26aulast%3DDimbil%26aufirst%3DM.%26aulast%3DKyle%26aufirst%3DR.%26aulast%3DOverstreet%26aufirst%3DB.%26aulast%3DHoffman%26aufirst%3DK.%2BB.%26atitle%3DAnalysis%2520of%2520spontaneous%2520postmarket%2520case%2520reports%2520submitted%2520to%2520the%2520FDA%2520regarding%2520thromboembolic%2520adverse%2520events%2520and%2520JAK%2520inhibitors%26jtitle%3DDrug%2520Saf.%26date%3D2018%26volume%3D41%26spage%3D357%26epage%3D361%26doi%3D10.1007%2Fs40264-017-0622-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, Z. C.</span></span> <span> </span><span class="NLM_article-title">Adverse drug events associated with 5mg versus 10mg tofacitinib (Janus kinase inhibitor) twice daily for the treatment of autoimmune diseases: A systematic review and meta-analysis of randomized controlled trials</span>. <i>Clin. Rheumatol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">523</span>– <span class="NLM_lpage">534</span>, <span class="refDoi"> DOI: 10.1007/s10067-018-4299-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=10.1007%2Fs10067-018-4299-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=30242639" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A280%3ADC%252BB3czhsFeruw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2019&pages=523-534&author=F.+Huangauthor=Z.+C.+Luo&title=Adverse+drug+events+associated+with+5mg+versus+10mg+tofacitinib+%28Janus+kinase+inhibitor%29+twice+daily+for+the+treatment+of+autoimmune+diseases%3A+A+systematic+review+and+meta-analysis+of+randomized+controlled+trials&doi=10.1007%2Fs10067-018-4299-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Adverse drug events associated with 5mg versus 10mg Tofacitinib (Janus kinase inhibitor) twice daily for the treatment of autoimmune diseases: A systematic review and meta-analysis of randomized controlled trials</span></div><div class="casAuthors">Huang Feng; Luo Zu-Chun</div><div class="citationInfo"><span class="NLM_cas:title">Clinical rheumatology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">523-534</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Several recently published clinical trials have shown tofacitinib to be effective in the treatment of autoimmune diseases.  This drug is commonly prescribed either in a 5-mg or in a10-mg dosage twice daily.  In this review, we aimed to systematically compare the adverse drug events which were observed with 5 mg versus 10 mg tofacitinib for the treatment of autoimmune diseases.  MEDLINE, EMBASE, the Cochrane library, and www.ClinicalTrials.gov were searched (from March to April 2018) for suitable English publications (published before April 2018).  The inclusion criteria were as follows: randomized controlled trials, autoimmune disorders (rheumatic arthritis, psoriatic arthritis, moderate to severe psoriasis, and ankylosing spondylitis), and comparison of adverse drug events associated with 5 mg versus 10 mg tofacitinib.  This study had follow-up time periods of 3 months and ≥ 6 months.  Statistical analysis was carried out by RevMan 5.3 whereby risk ratios (RRs) and 95% confidence intervals (CIs) were generated.  A total number of 4287 participants were included (2144 versus 2143 participants who received 5 mg and 10 mg tofacitinib twice daily respectively).  The results showed that at 3 months, similar risks of adverse drug events, serious adverse events, and adverse events leading to drug discontinuation were observed with 5 mg versus 10 mg tofacitinib (RR 1.04, 95% CI 0.98-1.10; P = 0.17, I(2) = 0%; RR 1.06, 95% CI 0.77-1.48; P = 0.71, I(2) = 0%; and RR 1.06, 95% CI 0.78-1.43; P = 0.73, I(2) = 32%, respectively).  The other outcomes including serious infection events, adjudicated herpes zoster infection, adjudicated opportunistic infection, mild and severe neutropenia, malignancies, and adjudicated major adverse cardiovascular events were also similarly manifested.  However, a decreased level of hemoglobin significantly favored 5 mg tofacitinib (RR 1.75, 95% CI 1.19-2.58; P = 0.005, I(2) = 49%).  Even at a follow-up time period of ≥ 6 months, adverse drug events, serious adverse events, adverse drug events leading to drug discontinuation, and serious infection were still similarly observed.  According to this current review, both dosages of tofacitinib were safe to use.  Even if similar adverse drug events were observed with 5 mg versus 10 mg tofacitinib twice daily for the treatment of autoimmune disorders, anemia was more prominent with 10 mg tofacitinib at a 3 month follow-up.  Nevertheless, future studies based on a larger population size with longer follow-up time periods should further be considered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSx8tJmFzzB7z0SWnHPWbmmfW6udTcc2ebFFvI86-131Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3czhsFeruw%253D%253D&md5=4df34f92cb4e4ed0f090e9c50eed6c11</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1007%2Fs10067-018-4299-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10067-018-4299-4%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DF.%26aulast%3DLuo%26aufirst%3DZ.%2BC.%26atitle%3DAdverse%2520drug%2520events%2520associated%2520with%25205mg%2520versus%252010mg%2520tofacitinib%2520%2528Janus%2520kinase%2520inhibitor%2529%2520twice%2520daily%2520for%2520the%2520treatment%2520of%2520autoimmune%2520diseases%253A%2520A%2520systematic%2520review%2520and%2520meta-analysis%2520of%2520randomized%2520controlled%2520trials%26jtitle%3DClin.%2520Rheumatol.%26date%3D2019%26volume%3D38%26spage%3D523%26epage%3D534%26doi%3D10.1007%2Fs10067-018-4299-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hall, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nash, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rischmueller, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bossingham, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bird, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cook, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witcombe, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soma, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwok, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thirunavukkarasu, K.</span></span> <span> </span><span class="NLM_article-title">Tofacitinib, an oral Janus kinase inhibitor: Pooled efficacy and safety analyses in an australian rheumatoid arthritis population</span>. <i>Rheumatol Ther.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">383</span>– <span class="NLM_lpage">401</span>, <span class="refDoi"> DOI: 10.1007/s40744-018-0118-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=10.1007%2Fs40744-018-0118-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=29949132" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A280%3ADC%252BB3c%252FisVGgsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2018&pages=383-401&author=S.+Hallauthor=P.+Nashauthor=M.+Rischmuellerauthor=D.+Bossinghamauthor=P.+Birdauthor=N.+Cookauthor=D.+Witcombeauthor=K.+Somaauthor=K.+Kwokauthor=K.+Thirunavukkarasu&title=Tofacitinib%2C+an+oral+Janus+kinase+inhibitor%3A+Pooled+efficacy+and+safety+analyses+in+an+australian+rheumatoid+arthritis+population&doi=10.1007%2Fs40744-018-0118-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Tofacitinib, an Oral Janus Kinase Inhibitor: Pooled Efficacy and Safety Analyses in an Australian Rheumatoid Arthritis Population</span></div><div class="casAuthors">Hall Stephen; Nash Peter; Rischmueller Maureen; Bossingham David; Bird Paul; Cook Nicola; Witcombe David; Thirunavukkarasu Krishan; Soma Koshika; Kwok Kenneth</div><div class="citationInfo"><span class="NLM_cas:title">Rheumatology and therapy</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">383-401</span>
        ISSN:<span class="NLM_cas:issn">2198-6576</span>.
    </div><div class="casAbstract">INTRODUCTION:  In Australia, there is an unmet need for improved treatments for rheumatoid arthritis (RA).  Tofacitinib is an oral Janus kinase inhibitor for the treatment of RA.  To provide an overview of key study outcomes for tofacitinib in Australian patients, we analyzed the efficacy and safety of tofacitinib in the Australian subpopulation of global RA phase III and long-term extension (LTE) studies.  METHODS:  Data were pooled from the Australian subpopulation of four phase III studies and one LTE study (database not locked at cut-off date: January 2016).  Patients in the phase III studies received tofacitinib 5 or 10 mg twice daily (BID), placebo (advancing to tofacitinib at months 3 or 6), or adalimumab, with background methotrexate or conventional synthetic disease-modifying antirheumatic drugs.  Patients in the LTE study received tofacitinib 5 or 10 mg BID.  Efficacy endpoints were American College of Rheumatology (ACR) 20/50/70 response rates, and change from baseline in the Disease Activity Score in 28 joints, erythrocyte sedimentation rate [DAS28-4(ESR)] and Health Assessment Questionnaire-Disability Index (HAQ-DI) scores.  Safety endpoints included incidence of adverse events (AEs), serious AEs, and discontinuations due to AEs.  AEs of special interest and laboratory parameters were analyzed in the LTE study.  RESULTS:  Across phase III studies (N = 100), ACR response rates and improvements in DAS28-4(ESR) and HAQ-DI scores were numerically greater with tofacitinib vs. placebo at month 3, and increased until month 12.  The results were sustained in the LTE study (N = 99) after 60 months' observation.  In general, the efficacy and safety profiles of tofacitinib were similar to those of the global RA population.  CONCLUSIONS:  In Australian patients with RA, tofacitinib therapy demonstrated sustained efficacy and consistent safety over ≥ 60 months' treatment.  FUNDING:  Pfizer Inc.  TRIAL REGISTRATION NUMBERS (ALL CLINICALTRIALS.GOV): NCT00960440; NCT00847613; NCT00856544; NCT00853385; NCT00413699.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTF0DbmWrmTD0H3Rlp_k91zfW6udTcc2eYn8P1S2cJS6Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c%252FisVGgsg%253D%253D&md5=1d77e5c9389fe82bb292917e6b575712</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1007%2Fs40744-018-0118-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs40744-018-0118-2%26sid%3Dliteratum%253Aachs%26aulast%3DHall%26aufirst%3DS.%26aulast%3DNash%26aufirst%3DP.%26aulast%3DRischmueller%26aufirst%3DM.%26aulast%3DBossingham%26aufirst%3DD.%26aulast%3DBird%26aufirst%3DP.%26aulast%3DCook%26aufirst%3DN.%26aulast%3DWitcombe%26aufirst%3DD.%26aulast%3DSoma%26aufirst%3DK.%26aulast%3DKwok%26aufirst%3DK.%26aulast%3DThirunavukkarasu%26aufirst%3DK.%26atitle%3DTofacitinib%252C%2520an%2520oral%2520Janus%2520kinase%2520inhibitor%253A%2520Pooled%2520efficacy%2520and%2520safety%2520analyses%2520in%2520an%2520australian%2520rheumatoid%2520arthritis%2520population%26jtitle%3DRheumatol%2520Ther.%26date%3D2018%26volume%3D5%26spage%3D383%26epage%3D401%26doi%3D10.1007%2Fs40744-018-0118-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">van
Vollenhoven, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riese, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krishnaswami, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawabata, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fosser, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rottinghaus, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamba, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwillich, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradley, J.</span></span> <span> </span><span class="NLM_article-title">Relationship between lymphocyte count and risk of infection in rheumatoid arthritis patients treated with tofacitinib</span>. <i>Ann. Rheumatoid Dis.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>72</i></span>,  <span class="NLM_fpage">250</span>– <span class="NLM_lpage">251</span>, <span class="refDoi"> DOI: 10.1136/annrheumdis-2013-eular.780</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=10.1136%2Fannrheumdis-2013-eular.780" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=23065732" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2013&pages=250-251&author=R.+van%0AVollenhovenauthor=R.+Rieseauthor=S.+Krishnaswamiauthor=T.+Kawabataauthor=C.+Fosserauthor=S.+Rottinghausauthor=M.+Lambaauthor=S.+H.+Zwillichauthor=J.+Bradley&title=Relationship+between+lymphocyte+count+and+risk+of+infection+in+rheumatoid+arthritis+patients+treated+with+tofacitinib&doi=10.1136%2Fannrheumdis-2013-eular.780"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1136%2Fannrheumdis-2013-eular.780&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1136%252Fannrheumdis-2013-eular.780%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2BVollenhoven%26aufirst%3DR.%26aulast%3DRiese%26aufirst%3DR.%26aulast%3DKrishnaswami%26aufirst%3DS.%26aulast%3DKawabata%26aufirst%3DT.%26aulast%3DFosser%26aufirst%3DC.%26aulast%3DRottinghaus%26aufirst%3DS.%26aulast%3DLamba%26aufirst%3DM.%26aulast%3DZwillich%26aufirst%3DS.%2BH.%26aulast%3DBradley%26aufirst%3DJ.%26atitle%3DRelationship%2520between%2520lymphocyte%2520count%2520and%2520risk%2520of%2520infection%2520in%2520rheumatoid%2520arthritis%2520patients%2520treated%2520with%2520tofacitinib%26jtitle%3DAnn.%2520Rheumatoid%2520Dis.%26date%3D2013%26volume%3D72%26spage%3D250%26epage%3D251%26doi%3D10.1136%2Fannrheumdis-2013-eular.780" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Akada, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akada, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hutchison, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakamoto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, K. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohi, G.</span></span> <span> </span><span class="NLM_article-title">Critical role of Jak2 in the maintenance and function of adult hematopoietic stem cells</span>. <i>Stem Cells</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">1878</span>– <span class="NLM_lpage">1889</span>, <span class="refDoi"> DOI: 10.1002/stem.1711</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=10.1002%2Fstem.1711" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=24677703" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVSqsrvL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2014&pages=1878-1889&author=H.+Akadaauthor=S.+Akadaauthor=R.+E.+Hutchisonauthor=K.+Sakamotoauthor=K.+U.+Wagnerauthor=G.+Mohi&title=Critical+role+of+Jak2+in+the+maintenance+and+function+of+adult+hematopoietic+stem+cells&doi=10.1002%2Fstem.1711"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Critical role of Jak2 in the maintenance and function of adult hematopoietic stem cells</span></div><div class="casAuthors">Akada, Hajime; Akada, Saeko; Hutchison, Robert E.; Sakamoto, Kazuhito; Wagner, Kay-Uwe; Mohi, Golam</div><div class="citationInfo"><span class="NLM_cas:title">Stem Cells (Durham, NC, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1878-1889</span>CODEN:
                <span class="NLM_cas:coden">STCEEJ</span>;
        ISSN:<span class="NLM_cas:issn">1066-5099</span>.
    
            (<span class="NLM_cas:orgname">AlphaMed Press</span>)
        </div><div class="casAbstract">Jak2, a member of the Janus kinase family of nonreceptor protein tyrosine kinases, is activated in response to a variety of cytokines, and functions in survival and proliferation of cells.  An activating JAK2V617F mutation has been found in most patients with myeloproliferative neoplasms, and patients treated with Jak2 inhibitors show significant hematopoietic toxicities.  However, the role of Jak2 in adult hematopoietic stem cells (HSCs) has not been clearly elucidated.  Using a conditional Jak2 knockout allele, we have found that Jak2 deletion results in rapid loss of HSCs/progenitors leading to bone marrow failure and early lethality in adult mice.  Jak2 deficiency causes marked impairment in HSC function, and the mutant HSCs are severely defective in reconstituting hematopoiesis in recipient animals.  Jak2 deficiency also causes significant apoptosis and loss of quiescence in HSC-enriched LSK (Lin-Sca-1+c-Kit+) cells.  Jak2-deficient LSK cells exhibit elevated reactive oxygen species levels and enhanced p38 MAPK activation.  Mutant LSK cells also show defective Stat5, Erk, and Akt activation in response to thrombopoietin and stem cell factor.  Gene expression anal. reveals significant downregulation of genes related to HSC quiescence and self-renewal in Jak2-deficient LSK cells.  These data suggest that Jak2 plays a crit. role in the maintenance and function of adult HSCs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrrritPwMPsTLVg90H21EOLACvtfcHk0lh1oKX9LbdRmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVSqsrvL&md5=5d8a4051c521f096754fe559c41bb878</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1002%2Fstem.1711&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fstem.1711%26sid%3Dliteratum%253Aachs%26aulast%3DAkada%26aufirst%3DH.%26aulast%3DAkada%26aufirst%3DS.%26aulast%3DHutchison%26aufirst%3DR.%2BE.%26aulast%3DSakamoto%26aufirst%3DK.%26aulast%3DWagner%26aufirst%3DK.%2BU.%26aulast%3DMohi%26aufirst%3DG.%26atitle%3DCritical%2520role%2520of%2520Jak2%2520in%2520the%2520maintenance%2520and%2520function%2520of%2520adult%2520hematopoietic%2520stem%2520cells%26jtitle%3DStem%2520Cells%26date%3D2014%26volume%3D32%26spage%3D1878%26epage%3D1889%26doi%3D10.1002%2Fstem.1711" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vazquez, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaila, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strohbach, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trzupek, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flanagan, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitton-Fry, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">TenBrink, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnold, E. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Basak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heasley, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langille, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parikh, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffin, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casavant, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duclos, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fenwick, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caspers, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dowty, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banker, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hegen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Symanowicz, P. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jussif, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Telliez, J.-B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Unwalla, R.</span></span> <span> </span><span class="NLM_article-title">Identification of <i>N</i>-{cis-3-[methyl(7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-4-yl)amino]cyclobutyl}propane-1-sulfonamide (PF-04965842): A selective JAK1 clinical candidate for the treatment of autoimmune diseases</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">1130</span>– <span class="NLM_lpage">1152</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01598</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01598" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A528%3ADC%252BC1cXit1Wnsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=1130-1152&author=M.+L.+Vazquezauthor=N.+Kailaauthor=J.+W.+Strohbachauthor=J.+D.+Trzupekauthor=M.+F.+Brownauthor=M.+E.+Flanaganauthor=M.+J.+Mitton-Fryauthor=T.+A.+Johnsonauthor=R.+E.+TenBrinkauthor=E.+P.+Arnoldauthor=A.+Basakauthor=S.+E.+Heasleyauthor=S.+Kwonauthor=J.+Langilleauthor=M.+D.+Parikhauthor=S.+H.+Griffinauthor=J.+M.+Casavantauthor=B.+A.+Duclosauthor=A.+E.+Fenwickauthor=T.+M.+Harrisauthor=S.+Hanauthor=N.+Caspersauthor=M.+E.+Dowtyauthor=X.+Yangauthor=M.+E.+Bankerauthor=M.+Hegenauthor=P.+T.+Symanowiczauthor=L.+Liauthor=L.+Wangauthor=T.+H.+Linauthor=J.+Jussifauthor=J.+D.+Clarkauthor=J.-B.+Telliezauthor=R.+P.+Robinsonauthor=R.+Unwalla&title=Identification+of+N-%7Bcis-3-%5Bmethyl%287H-pyrrolo%5B2%2C3-d%5Dpyrimidin-4-yl%29amino%5Dcyclobutyl%7Dpropane-1-sulfonamide+%28PF-04965842%29%3A+A+selective+JAK1+clinical+candidate+for+the+treatment+of+autoimmune+diseases&doi=10.1021%2Facs.jmedchem.7b01598"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of N-{cis-3-[Methyl(7H-pyrrolo[2,3-d]pyrimidin-4-yl)amino]cyclobutyl}propane-1-sulfonamide (PF-04965842): A Selective JAK1 Clinical Candidate for the Treatment of Autoimmune Diseases</span></div><div class="casAuthors">Vazquez, Michael L.; Kaila, Neelu; Strohbach, Joseph W.; Trzupek, John D.; Brown, Matthew F.; Flanagan, Mark E.; Mitton-Fry, Mark J.; Johnson, Timothy A.; TenBrink, Ruth E.; Arnold, Eric P.; Basak, Arindrajit; Heasley, Steven E.; Kwon, Soojin; Langille, Jonathan; Parikh, Mihir D.; Griffin, Sarah H.; Casavant, Jeffrey M.; Duclos, Brian A.; Fenwick, Ashley E.; Harris, Thomas M.; Han, Seungil; Caspers, Nicole; Dowty, Martin E.; Yang, Xin; Banker, Mary Ellen; Hegen, Martin; Symanowicz, Peter T.; Li, Li; Wang, Lu; Lin, Tsung H.; Jussif, Jason; Clark, James D.; Telliez, Jean-Baptiste; Robinson, Ralph P.; Unwalla, Ray</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1130-1152</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Janus kinases (JAKs) are intracellular tyrosine kinases that mediate the signaling of numerous cytokines and growth factors involved in the regulation of immunity, inflammation, and hematopoiesis.  As JAK1 pairs with JAK2, JAK3, and TYK2, a JAK1-selective inhibitor would be expected to inhibit many cytokines involved in inflammation and immune function while avoiding inhibition of the JAK2 homodimer regulating erythropoietin and thrombopoietin signaling.  The authors' efforts began with tofacitinib, an oral JAK inhibitor approved for the treatment of rheumatoid arthritis.  Through modification of the 3-aminopiperidine linker in tofacitinib, the authors discovered highly selective JAK1 inhibitors with nanomolar potency in a human whole blood assay.  Improvements in JAK1 potency and selectivity were achieved via structural modifications suggested by x-ray crystallog. anal.  After demonstrating efficacy in a rat adjuvant-induced arthritis (rAIA) model, PF-04965842 (25) was nominated as a clin. candidate for the treatment of JAK1-mediated autoimmune diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNdiJFIHI-YLVg90H21EOLACvtfcHk0lh12sfGjdxSFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXit1Wnsw%253D%253D&md5=45c6d1b0eee7ab9d5dfc8ea168607a8b</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01598&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01598%26sid%3Dliteratum%253Aachs%26aulast%3DVazquez%26aufirst%3DM.%2BL.%26aulast%3DKaila%26aufirst%3DN.%26aulast%3DStrohbach%26aufirst%3DJ.%2BW.%26aulast%3DTrzupek%26aufirst%3DJ.%2BD.%26aulast%3DBrown%26aufirst%3DM.%2BF.%26aulast%3DFlanagan%26aufirst%3DM.%2BE.%26aulast%3DMitton-Fry%26aufirst%3DM.%2BJ.%26aulast%3DJohnson%26aufirst%3DT.%2BA.%26aulast%3DTenBrink%26aufirst%3DR.%2BE.%26aulast%3DArnold%26aufirst%3DE.%2BP.%26aulast%3DBasak%26aufirst%3DA.%26aulast%3DHeasley%26aufirst%3DS.%2BE.%26aulast%3DKwon%26aufirst%3DS.%26aulast%3DLangille%26aufirst%3DJ.%26aulast%3DParikh%26aufirst%3DM.%2BD.%26aulast%3DGriffin%26aufirst%3DS.%2BH.%26aulast%3DCasavant%26aufirst%3DJ.%2BM.%26aulast%3DDuclos%26aufirst%3DB.%2BA.%26aulast%3DFenwick%26aufirst%3DA.%2BE.%26aulast%3DHarris%26aufirst%3DT.%2BM.%26aulast%3DHan%26aufirst%3DS.%26aulast%3DCaspers%26aufirst%3DN.%26aulast%3DDowty%26aufirst%3DM.%2BE.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DBanker%26aufirst%3DM.%2BE.%26aulast%3DHegen%26aufirst%3DM.%26aulast%3DSymanowicz%26aufirst%3DP.%2BT.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DLin%26aufirst%3DT.%2BH.%26aulast%3DJussif%26aufirst%3DJ.%26aulast%3DClark%26aufirst%3DJ.%2BD.%26aulast%3DTelliez%26aufirst%3DJ.-B.%26aulast%3DRobinson%26aufirst%3DR.%2BP.%26aulast%3DUnwalla%26aufirst%3DR.%26atitle%3DIdentification%2520of%2520N-%257Bcis-3-%255Bmethyl%25287H-pyrrolo%255B2%252C3-d%255Dpyrimidin-4-yl%2529amino%255Dcyclobutyl%257Dpropane-1-sulfonamide%2520%2528PF-04965842%2529%253A%2520A%2520selective%2520JAK1%2520clinical%2520candidate%2520for%2520the%2520treatment%2520of%2520autoimmune%2520diseases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D1130%26epage%3D1152%26doi%3D10.1021%2Facs.jmedchem.7b01598" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Spolski, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gromer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leonard, W. J.</span></span> <span> </span><span class="NLM_article-title">The γ c family of cytokines: fine-tuning signals from IL-2 and IL-21 in the regulation of the immune response</span>. <i>F1000Research</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">1872</span>, <span class="refDoi"> DOI: 10.12688/f1000research.12202.1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=10.12688%2Ff1000research.12202.1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=29123649" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisFSru73L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2017&pages=1872&author=R.+Spolskiauthor=D.+Gromerauthor=W.+J.+Leonard&title=The+%CE%B3+c+family+of+cytokines%3A+fine-tuning+signals+from+IL-2+and+IL-21+in+the+regulation+of+the+immune+response&doi=10.12688%2Ff1000research.12202.1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">The γc family of cytokines: fine-tuning signals from IL-2 and IL-21 in the regulation of the immune response [version 1; referees: 3 approved]</span></div><div class="casAuthors">Spolski, Rosanne; Gromer, Daniel; Leonard, Warren J.</div><div class="citationInfo"><span class="NLM_cas:title">F1000Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1872/1-1872/12</span>CODEN:
                <span class="NLM_cas:coden">FRESJL</span>;
        ISSN:<span class="NLM_cas:issn">2046-1402</span>.
    
            (<span class="NLM_cas:orgname">F1000 Research Ltd.</span>)
        </div><div class="casAbstract">Interleukin (IL)-2, IL-4, IL-7, IL-9, IL-15, and IL-21 form a family of cytokines based on the sharing of a receptor component, the common cytokine receptor γ chain, γc, which is encoded by the gene mutated in humans with X-linked severe combined immunodeficiency (XSCID).  Together, these cytokines play crit. roles in lymphoid development, differentiation, growth, and survival as well as mediating effector function.  Here, we provide an overview of the main actions of members of this cytokine family but then primarily focus on IL-2 and IL-21, discussing their dynamic interplay and contributions to a fine-tuned immune response.  Moreover, we discuss the therapeutic utility of modulating their actions, particularly for autoimmunity and cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpg5g2WamKEyrVg90H21EOLACvtfcHk0lh12sfGjdxSFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisFSru73L&md5=18f97a3267d5cd8c30370dcb606dd51e</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.12688%2Ff1000research.12202.1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.12688%252Ff1000research.12202.1%26sid%3Dliteratum%253Aachs%26aulast%3DSpolski%26aufirst%3DR.%26aulast%3DGromer%26aufirst%3DD.%26aulast%3DLeonard%26aufirst%3DW.%2BJ.%26atitle%3DThe%2520%25CE%25B3%2520c%2520family%2520of%2520cytokines%253A%2520fine-tuning%2520signals%2520from%2520IL-2%2520and%2520IL-21%2520in%2520the%2520regulation%2520of%2520the%2520immune%2520response%26jtitle%3DF1000Research%26date%3D2017%26volume%3D6%26spage%3D1872%26doi%3D10.12688%2Ff1000research.12202.1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">White, U. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stephens, J. M.</span></span> <span> </span><span class="NLM_article-title">The gp130 receptor cytokine family: regulators of adipocyte development and function</span>. <i>Curr. Pharm. Des.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">340</span>– <span class="NLM_lpage">346</span>, <span class="refDoi"> DOI: 10.2174/138161211795164202</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=10.2174%2F138161211795164202" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=21375496" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A528%3ADC%252BC3MXlsVKlsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2011&pages=340-346&author=U.+A.+Whiteauthor=J.+M.+Stephens&title=The+gp130+receptor+cytokine+family%3A+regulators+of+adipocyte+development+and+function&doi=10.2174%2F138161211795164202"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">The gp130 receptor cytokine family: regulators of adipocyte development and function</span></div><div class="casAuthors">White, Ursula A.; Stephens, Jacqueline M.</div><div class="citationInfo"><span class="NLM_cas:title">Current Pharmaceutical Design</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">340-346</span>CODEN:
                <span class="NLM_cas:coden">CPDEFP</span>;
        ISSN:<span class="NLM_cas:issn">1381-6128</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Gp130 cytokines are involved in the regulation of numerous biol. processes, including hematopoiesis, immune response, inflammation, cardiovascular action, and neuronal survival.  These cytokines share glycoprotein 130 as a common signal transducer in their receptor complex and typically activate STAT3.  Most gp130 cytokines have paracrine or endocrine actions, and their levels can be measured in circulation in rodents and humans.  In recent years, various labs. have conducted studies to demonstrate that gp130 cytokines can modulate adipocyte development and function.  Therefore, these studies suggest that some gp130 cytokines may be viable anti-obesity therapeutics.  In this review, we will summarize the reported effects of gp130 cytokines on adipocyte differentiation and adipocyte function.  In addn., the modulation of gp130 cytokines in conditions of obesity, insulin resistance, and Type 2 diabetes will be presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprllah-s4Q2rVg90H21EOLACvtfcHk0lh12sfGjdxSFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXlsVKlsbo%253D&md5=8cb1820e42f890178365ad102df33a14</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.2174%2F138161211795164202&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138161211795164202%26sid%3Dliteratum%253Aachs%26aulast%3DWhite%26aufirst%3DU.%2BA.%26aulast%3DStephens%26aufirst%3DJ.%2BM.%26atitle%3DThe%2520gp130%2520receptor%2520cytokine%2520family%253A%2520regulators%2520of%2520adipocyte%2520development%2520and%2520function%26jtitle%3DCurr.%2520Pharm.%2520Des.%26date%3D2011%26volume%3D17%26spage%3D340%26epage%3D346%26doi%3D10.2174%2F138161211795164202" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonelli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gadina, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’shea, J. J.</span></span> <span> </span><span class="NLM_article-title">Type I/II cytokines, JAKs, and new strategies for treating autoimmune diseases</span>. <i>Nat. Rev. Rheumatol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">25</span>– <span class="NLM_lpage">36</span>, <span class="refDoi"> DOI: 10.1038/nrrheum.2015.167</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=10.1038%2Fnrrheum.2015.167" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=26633291" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvF2ks7vJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=25-36&author=D.+M.+Schwartzauthor=M.+Bonelliauthor=M.+Gadinaauthor=J.+J.+O%E2%80%99shea&title=Type+I%2FII+cytokines%2C+JAKs%2C+and+new+strategies+for+treating+autoimmune+diseases&doi=10.1038%2Fnrrheum.2015.167"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Type I/II cytokines, JAKs, and new strategies for treating autoimmune diseases</span></div><div class="casAuthors">Schwartz, Daniella M.; Bonelli, Michael; Gadina, Massimo; O'Shea, John J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Rheumatology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">25-36</span>CODEN:
                <span class="NLM_cas:coden">NRRACB</span>;
        ISSN:<span class="NLM_cas:issn">1759-4790</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Cytokines are major drivers of autoimmunity, and biol. agents targeting cytokines have revolutionized the treatment of immune-mediated diseases.  Despite the effectiveness of these drugs, they do not induce complete remission in all patients, prompting the development of alternative strategies - including targeting of intracellular signal transduction pathways downstream of cytokines.  Many cytokines that bind type I and type II cytokine receptors are crit. regulators of immune-mediated diseases and employ the Janus kinase (JAK) and signal transducer and activator of transcription (STAT) pathway to exert their effect.  Pharmacol. inhibition of JAKs blocks the actions of type I/II cytokines, and within the past 3 years therapeutic JAK inhibitors, or Jakinibs, have become available to rheumatologists.  Jakinibs have proven effective for the treatment of rheumatoid arthritis and other inflammatory diseases.  Adverse effects of these agents are largely related to their mode of action and include infections and hyperlipidemia.  Jakinibs are currently being investigated for a no. of new indications, and second-generation selective Jakinibs are being developed and tested.  Targeting STATs could be a future avenue for the treatment of rheumatol. diseases, although substantial challenges remain.  Nonetheless, the ability to therapeutically target intracellular signalling pathways has already created a new paradigm for the treatment of rheumatol. disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUdr1Dh92dnLVg90H21EOLACvtfcHk0liquSPYUlQQdA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvF2ks7vJ&md5=e06f7ab6424f1fbdbf481c75fa9ebd23</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fnrrheum.2015.167&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrrheum.2015.167%26sid%3Dliteratum%253Aachs%26aulast%3DSchwartz%26aufirst%3DD.%2BM.%26aulast%3DBonelli%26aufirst%3DM.%26aulast%3DGadina%26aufirst%3DM.%26aulast%3DO%25E2%2580%2599shea%26aufirst%3DJ.%2BJ.%26atitle%3DType%2520I%252FII%2520cytokines%252C%2520JAKs%252C%2520and%2520new%2520strategies%2520for%2520treating%2520autoimmune%2520diseases%26jtitle%3DNat.%2520Rev.%2520Rheumatol.%26date%3D2016%26volume%3D12%26spage%3D25%26epage%3D36%26doi%3D10.1038%2Fnrrheum.2015.167" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roskoski, R.,  Jr.</span></span> <span> </span><span class="NLM_article-title">Janus kinase (JAK) inhibitors in the treatment of inflammatory and neoplastic diseases</span>. <i>Pharmacol. Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>111</i></span>,  <span class="NLM_fpage">784</span>– <span class="NLM_lpage">803</span>, <span class="refDoi"> DOI: 10.1016/j.phrs.2016.07.038</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=10.1016%2Fj.phrs.2016.07.038" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=27473820" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht12jurzJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2016&pages=784-803&author=R.+Roskoski&title=Janus+kinase+%28JAK%29+inhibitors+in+the+treatment+of+inflammatory+and+neoplastic+diseases&doi=10.1016%2Fj.phrs.2016.07.038"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Janus kinase (JAK) inhibitors in the treatment of inflammatory and neoplastic diseases</span></div><div class="casAuthors">Roskoski, Robert, Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">784-803</span>CODEN:
                <span class="NLM_cas:coden">PHMREP</span>;
        ISSN:<span class="NLM_cas:issn">1043-6618</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The Janus kinase (JAK) family of non-receptor protein-tyrosine kinases consists of JAK1, JAK2, JAK3, and TYK2 (tyrosine kinase-2).  Each of these proteins contains a JAK homol. pseudokinase (JH2) domain that regulates the adjacent protein kinase domain (JH1).  JAK1/2 and TYK2 are ubiquitously expressed whereas JAK3 is found predominantly in hematopoietic cells.  The Janus kinase family is regulated by numerous cytokines including interleukins, interferons, and hormones such as erythropoietin, thrombopoietin, and growth hormone.  Ligand binding to cytokine and hormone receptors leads to the activation of assocd. Janus kinases, which then mediate the phosphorylation of the receptors.  The SH2 domain of STATs (signal transducers and activators of transcription) binds to the receptor phosphotyrosines thereby promoting STAT phosphorylation by the Janus kinases and consequent activation.  STAT dimers are translocated to the nucleus where they participate in the regulation of the expression of thousands of proteins.  JAK-STAT dysregulation results in autoimmune disorders such as rheumatoid arthritis, ulcerative colitis, and Crohn disease.  JAK-STAT dysregulation also plays a role in the pathogenesis of myelofibrosis, polycythemia vera, and other myeloproliferative illnesses.  An activating JAK2 V617F mutation occurs in 95% of people with polycythemia vera and in a lower percentage of people with other neoplasms.  JAK1/3 signaling participates in the pathogenesis of inflammatory afflictions while JAK1/2 signaling participates in the development of several malignancies including leukemias and lymphomas as well as myeloproliferative neoplasms.  Tofacitinib is a pan-JAK inhibitor that is approved by the FDA for the treatment of rheumatoid arthritis and ruxolitinib is a JAK1/2 inhibitor that is approved for the treatment of polycythemia vera and myelofibrosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGov32m4UjOrv7Vg90H21EOLACvtfcHk0liquSPYUlQQdA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht12jurzJ&md5=a45fba1bb110eff4d5c1b31b78c1b3ac</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.phrs.2016.07.038&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.phrs.2016.07.038%26sid%3Dliteratum%253Aachs%26aulast%3DRoskoski%26aufirst%3DR.%26atitle%3DJanus%2520kinase%2520%2528JAK%2529%2520inhibitors%2520in%2520the%2520treatment%2520of%2520inflammatory%2520and%2520neoplastic%2520diseases%26jtitle%3DPharmacol.%2520Res.%26date%3D2016%26volume%3D111%26spage%3D784%26epage%3D803%26doi%3D10.1016%2Fj.phrs.2016.07.038" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Parmentier, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Voss, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graff, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Argiriadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friedman, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Camp, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Padley, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">George, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hyland, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosebraugh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wishart, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, J. A.</span></span> <span> </span><span class="NLM_article-title"><i>In vitro</i> and <i>in vivo</i> characterization of the JAK1 selectivity of upadacitinib (ABT-494)</span>. <i>BMC Rheumatol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">23</span>, <span class="refDoi"> DOI: 10.1186/s41927-018-0031-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=10.1186%2Fs41927-018-0031-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=30886973" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A280%3ADC%252BB3cbls1ymuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2018&pages=23&author=J.+M.+Parmentierauthor=J.+Vossauthor=C.+Graffauthor=A.+Schwartzauthor=M.+Argiriadiauthor=M.+Friedmanauthor=S.+H.+Campauthor=J.+R.+Padleyauthor=S.+J.+Georgeauthor=D.+Hylandauthor=M.+Rosebraughauthor=N.+Wishartauthor=L.+Olsonauthor=J.+A.+Long&title=In+vitro+and+in+vivo+characterization+of+the+JAK1+selectivity+of+upadacitinib+%28ABT-494%29&doi=10.1186%2Fs41927-018-0031-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">In vitro and in vivo characterization of the JAK1 selectivity of upadacitinib (ABT-494)</span></div><div class="casAuthors">Parmentier Julie M; Voss Jeff; Graff Candace; Schwartz Annette; Argiriadi Maria; Friedman Michael; George Jonathan S; Hyland Deborah; Wishart Neil; Olson Lisa; Long Andrew J; Camp Heidi S; Padley Robert J; Rosebraugh Matthew</div><div class="citationInfo"><span class="NLM_cas:title">BMC rheumatology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">23</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Background:  Anti-cytokine therapies such as adalimumab, tocilizumab, and the small molecule JAK inhibitor tofacitinib have proven that cytokines and their subsequent downstream signaling processes are important in the pathogenesis of rheumatoid arthritis.  Tofacitinib, a pan-JAK inhibitor, is the first approved JAK inhibitor for the treatment of RA and has been shown to be effective in managing disease.  However, in phase 2 dose-ranging studies tofacitinib was associated with dose-limiting tolerability and safety issues such as anemia.  Upadacitinib (ABT-494) is a selective JAK1 inhibitor that was engineered to address the hypothesis that greater JAK1 selectivity over other JAK family members will translate into a more favorable benefit:risk profile.  Upadacitinib selectively targets JAK1 dependent disease drivers such as IL-6 and IFNγ, while reducing effects on reticulocytes and natural killer (NK) cells, which potentially contributed to the tolerability issues of tofacitinib.  Methods:  Structure-based hypotheses were used to design the JAK1 selective inhibitor upadacitinib.  JAK family selectivity was defined with in vitro assays including biochemical assessments, engineered cell lines, and cytokine stimulation.  In vivo selectivity was defined by the efficacy of upadacitinib and tofacitinib in a rat adjuvant induced arthritis model, activity on reticulocyte deployment, and effect on circulating NK cells.  The translation of the preclinical JAK1 selectivity was assessed in healthy volunteers using ex vivo stimulation with JAK-dependent cytokines.  Results:  Here, we show the structural basis for the JAK1 selectivity of upadacitinib, along with the in vitro JAK family selectivity profile and subsequent in vivo physiological consequences.  Upadacitinib is ~ 60 fold selective for JAK1 over JAK2, and > 100 fold selective over JAK3 in cellular assays.  While both upadacitinib and tofacitinib demonstrated efficacy in a rat model of arthritis, the increased selectivity of upadacitinib for JAK1 resulted in a reduced effect on reticulocyte deployment and NK cell depletion relative to efficacy.  Ex vivo pharmacodynamic data obtained from Phase I healthy volunteers confirmed the JAK1 selectivity of upadactinib in a clinical setting.  Conclusions:  The data presented here highlight the JAK1 selectivity of upadacinitinib and supports its use as an effective therapy for the treatment of RA with the potential for an improved benefit:risk profile.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRTVFFxllNWaxfKb6ELK8t0fW6udTcc2ebQqL-9TMXcWLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3cbls1ymuw%253D%253D&md5=5ca2dcb18a98362567d2193e858cc800</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1186%2Fs41927-018-0031-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs41927-018-0031-x%26sid%3Dliteratum%253Aachs%26aulast%3DParmentier%26aufirst%3DJ.%2BM.%26aulast%3DVoss%26aufirst%3DJ.%26aulast%3DGraff%26aufirst%3DC.%26aulast%3DSchwartz%26aufirst%3DA.%26aulast%3DArgiriadi%26aufirst%3DM.%26aulast%3DFriedman%26aufirst%3DM.%26aulast%3DCamp%26aufirst%3DS.%2BH.%26aulast%3DPadley%26aufirst%3DJ.%2BR.%26aulast%3DGeorge%26aufirst%3DS.%2BJ.%26aulast%3DHyland%26aufirst%3DD.%26aulast%3DRosebraugh%26aufirst%3DM.%26aulast%3DWishart%26aufirst%3DN.%26aulast%3DOlson%26aufirst%3DL.%26aulast%3DLong%26aufirst%3DJ.%2BA.%26atitle%3DIn%2520vitro%2520and%2520in%2520vivo%2520characterization%2520of%2520the%2520JAK1%2520selectivity%2520of%2520upadacitinib%2520%2528ABT-494%2529%26jtitle%3DBMC%2520Rheumatol.%26date%3D2018%26volume%3D2%26spage%3D23%26doi%3D10.1186%2Fs41927-018-0031-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span> <span> </span><span class="NLM_article-title">Discovery and evaluation of ZT55, a novel highly-selective tyrosine kinase inhibitor of JAK2<sup>V617F</sup> against myeloproliferative neoplasms</span>. <i>J. Exp. Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">49</span>, <span class="refDoi"> DOI: 10.1186/s13046-019-1062-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=10.1186%2Fs13046-019-1062-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=30717771" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A280%3ADC%252BB3cjovVyksg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2019&pages=49&author=M.+Huauthor=C.+Xuauthor=C.+Yangauthor=H.+Zuoauthor=C.+Chenauthor=D.+Zhangauthor=G.+Shiauthor=W.+Wangauthor=J.+Shiauthor=T.+Zhang&title=Discovery+and+evaluation+of+ZT55%2C+a+novel+highly-selective+tyrosine+kinase+inhibitor+of+JAK2V617F+against+myeloproliferative+neoplasms&doi=10.1186%2Fs13046-019-1062-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and evaluation of ZT55, a novel highly-selective tyrosine kinase inhibitor of JAK2(V617F) against myeloproliferative neoplasms</span></div><div class="casAuthors">Hu Min; Xu Chengbo; Chen Chengjuan; Shi Gaona; Wang Wenjie; Shi Jiangong; Zhang Tiantai; Yang Chao; Zuo Hongli; Zhang Dan; Zhang Tiantai</div><div class="citationInfo"><span class="NLM_cas:title">Journal of experimental & clinical cancer research : CR</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">49</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  The JAK2-STAT signaling pathway plays a critical role in myeloproliferative neoplasms (MPN).  An activating mutation in JAK2 (V617F) is present in ~ 95% of polycythemia vera, essential thrombocythemia, and primary myelofibrosis cases.  This study aims to explore the selective JAK2(V617F) inhibitor, evaluate the efficacy and possible mechanism of ZT55 on MPN.  METHODS:  HTRF assays were conducted to evaluate the selective inhibition of ZT55 for JAKs.  Cell apoptosis, proliferation, and cycle arrest assays were performed to examine the effect of ZT55 on HEL cell line with JAK2(V617F) mutation in vitro.  Western analysis was used to monitor the expression and activity of proteins on JAK2/STAT pathway.  A mice xenograft model was established to evaluate the antitumor efficacy of ZT55 in vivo.  Peripheral blood samples from patients with the JAK2(V617F) mutation were collected to estimate the effect of ZT55 on erythroid colony formation by colony-forming assay.  RESULTS:  We found that ZT55 showed a selective inhibition of a 0.031 μM IC50 value against JAK2.  It exhibited potent effects on the cellular JAK-STAT pathway, inhibiting tyrosine phosphorylation in JAK2(V617F) and downstream STAT3/5 transcription factors.  ZT55 inhibited the proliferation of the JAK2(V617F)-expressing HEL cell line, leading to cell cycle arrest at the G2/M phase and induction of caspase-dependent apoptosis.  Notably, ZT55 also significantly suppressed the growth of HEL xenograft tumors in vivo.  Further evaluation indicated that ZT55 blocked erythroid colony formation of peripheral blood hematopoietic progenitors from patients carrying the JAK2(V617F) mutation.  CONCLUSION:  These results suggest that ZT55 is a highly-selective JAK2 inhibitor that can induce apoptosis of human erythroleukemia cells by inhibiting the JAK2-STAT signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTBu63j-Oqd92hHGZei4OW9fW6udTcc2eaXWnnOyNGA-Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3cjovVyksg%253D%253D&md5=a65f4b36f9cacf7547c25d16f881132f</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1186%2Fs13046-019-1062-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13046-019-1062-x%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DM.%26aulast%3DXu%26aufirst%3DC.%26aulast%3DYang%26aufirst%3DC.%26aulast%3DZuo%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DShi%26aufirst%3DG.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DT.%26atitle%3DDiscovery%2520and%2520evaluation%2520of%2520ZT55%252C%2520a%2520novel%2520highly-selective%2520tyrosine%2520kinase%2520inhibitor%2520of%2520JAK2V617F%2520against%2520myeloproliferative%2520neoplasms%26jtitle%3DJ.%2520Exp.%2520Clin.%2520Cancer%2520Res.%26date%3D2019%26volume%3D38%26spage%3D49%26doi%3D10.1186%2Fs13046-019-1062-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thorarensen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dowty, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banker, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Juba, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jussif, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vincent, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czerwinski, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casimiro-Garcia, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Unwalla, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trujillo, I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balbo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Che, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilbert, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayward, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montgomery, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hegen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coe, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langille, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajdos, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chrencik, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Telliez, J.-B.</span></span> <span> </span><span class="NLM_article-title">Design of a Janus kinase 3 (JAK3) specific inhibitor 1-((2<i>S</i>, 5<i>R</i>)-5-((7<i>H</i>-Pyrrolo[2,3-<i>d</i>] pyrimidin-4-yl) amino)-2-methylpiperidin-1-yl) prop-2-en-1-one (PF-06651600) allowing for the interrogation of JAK3 signaling in humans</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">1971</span>– <span class="NLM_lpage">1993</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01694</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01694" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVykt7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=1971-1993&author=A.+Thorarensenauthor=M.+E.+Dowtyauthor=M.+E.+Bankerauthor=B.+Jubaauthor=J.+Jussifauthor=T.+Linauthor=F.+Vincentauthor=M.+R.+Czerwinskiauthor=A.+Casimiro-Garciaauthor=R.+Unwallaauthor=I.+J.+Trujilloauthor=S.+Liangauthor=P.+Balboauthor=Y.+Cheauthor=M.+A.+Gilbertauthor=F.+M.+Brownauthor=M.+Haywardauthor=J.+Montgomeryauthor=L.+Leungauthor=X.+Yangauthor=M.+Hegenauthor=J.+Coeauthor=J.+Langilleauthor=F.+Vajdosauthor=J.+Chrencikauthor=J.-B.+Telliez&title=Design+of+a+Janus+kinase+3+%28JAK3%29+specific+inhibitor+1-%28%282S%2C+5R%29-5-%28%287H-Pyrrolo%5B2%2C3-d%5D+pyrimidin-4-yl%29+amino%29-2-methylpiperidin-1-yl%29+prop-2-en-1-one+%28PF-06651600%29+allowing+for+the+interrogation+of+JAK3+signaling+in+humans&doi=10.1021%2Facs.jmedchem.6b01694"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Design of a Janus Kinase 3 (JAK3) Specific Inhibitor 1-((2S,5R)-5-((7H-Pyrrolo[2,3-d]pyrimidin-4-yl)amino)-2-methylpiperidin-1-yl)prop-2-en-1-one (PF-06651600) Allowing for the Interrogation of JAK3 Signaling in Humans</span></div><div class="casAuthors">Thorarensen, Atli; Dowty, Martin E.; Banker, Mary Ellen; Juba, Brian; Jussif, Jason; Lin, Tsung; Vincent, Fabien; Czerwinski, Robert M.; Casimiro-Garcia, Agustin; Unwalla, Ray; Trujillo, John I.; Liang, Sidney; Balbo, Paul; Che, Ye; Gilbert, Adam M.; Brown, Matthew F.; Hayward, Matthew; Montgomery, Justin; Leung, Louis; Yang, Xin; Soucy, Sarah; Hegen, Martin; Coe, Jotham; Langille, Jonathan; Vajdos, Felix; Chrencik, Jill; Telliez, Jean-Baptiste</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1971-1993</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Significant work has been dedicated to the discovery of JAK kinase inhibitors resulting in several compds. entering clin. development and two FDA approved NMEs.  However, despite significant effort during the past two decades, identification of highly selective JAK3 inhibitors has eluded the scientific community.  A significant effort within the research organization has resulted in the identification of the first orally active JAK3 specific inhibitor, which achieves JAK isoform specificity through covalent interaction with a unique JAK3 residue Cys-909.  The relatively rapid resynthesis rate of the JAK3 enzyme presented a unique challenge in the design of covalent inhibitors with appropriate pharmacodynamics properties coupled with limited unwanted off-target reactivity.  This effort resulted in the identification of PF-06651600 I, a potent and low clearance compd. with demonstrated in vivo efficacy.  The favorable efficacy and safety profile of this specific JAK3 inhibitor I led to its evaluation in several human clin. studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodS6VFIL07nrVg90H21EOLACvtfcHk0lgXnoNNsEijvg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVykt7c%253D&md5=d1ab749203fae07f89463f97518e3db8</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01694&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01694%26sid%3Dliteratum%253Aachs%26aulast%3DThorarensen%26aufirst%3DA.%26aulast%3DDowty%26aufirst%3DM.%2BE.%26aulast%3DBanker%26aufirst%3DM.%2BE.%26aulast%3DJuba%26aufirst%3DB.%26aulast%3DJussif%26aufirst%3DJ.%26aulast%3DLin%26aufirst%3DT.%26aulast%3DVincent%26aufirst%3DF.%26aulast%3DCzerwinski%26aufirst%3DM.%2BR.%26aulast%3DCasimiro-Garcia%26aufirst%3DA.%26aulast%3DUnwalla%26aufirst%3DR.%26aulast%3DTrujillo%26aufirst%3DI.%2BJ.%26aulast%3DLiang%26aufirst%3DS.%26aulast%3DBalbo%26aufirst%3DP.%26aulast%3DChe%26aufirst%3DY.%26aulast%3DGilbert%26aufirst%3DM.%2BA.%26aulast%3DBrown%26aufirst%3DF.%2BM.%26aulast%3DHayward%26aufirst%3DM.%26aulast%3DMontgomery%26aufirst%3DJ.%26aulast%3DLeung%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DHegen%26aufirst%3DM.%26aulast%3DCoe%26aufirst%3DJ.%26aulast%3DLangille%26aufirst%3DJ.%26aulast%3DVajdos%26aufirst%3DF.%26aulast%3DChrencik%26aufirst%3DJ.%26aulast%3DTelliez%26aufirst%3DJ.-B.%26atitle%3DDesign%2520of%2520a%2520Janus%2520kinase%25203%2520%2528JAK3%2529%2520specific%2520inhibitor%25201-%2528%25282S%252C%25205R%2529-5-%2528%25287H-Pyrrolo%255B2%252C3-d%255D%2520pyrimidin-4-yl%2529%2520amino%2529-2-methylpiperidin-1-yl%2529%2520prop-2-en-1-one%2520%2528PF-06651600%2529%2520allowing%2520for%2520the%2520interrogation%2520of%2520JAK3%2520signaling%2520in%2520humans%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D1971%26epage%3D1993%26doi%3D10.1021%2Facs.jmedchem.6b01694" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Menet, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fletcher, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Lommen, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geney, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blanc, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smits, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jouannigot, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deprez, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der
Aar, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clement-Lacroix, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lepescheux, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galien, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vayssiere, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nelles, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christophe, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brys, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uhring, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciesielski, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Rompaey, L.</span></span> <span> </span><span class="NLM_article-title">Triazolopyridines as selective JAK1 inhibitors: From hit identification to GLPG0634</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">9323</span>– <span class="NLM_lpage">9342</span>, <span class="refDoi"> DOI: 10.1021/jm501262q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501262q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVGls7nN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=9323-9342&author=C.+J.+Menetauthor=S.+R.+Fletcherauthor=G.+Van+Lommenauthor=R.+Geneyauthor=J.+Blancauthor=K.+Smitsauthor=N.+Jouannigotauthor=P.+Deprezauthor=E.+M.+van+der%0AAarauthor=P.+Clement-Lacroixauthor=L.+Lepescheuxauthor=R.+Galienauthor=B.+Vayssiereauthor=L.+Nellesauthor=T.+Christopheauthor=R.+Brysauthor=M.+Uhringauthor=F.+Ciesielskiauthor=L.+Van+Rompaey&title=Triazolopyridines+as+selective+JAK1+inhibitors%3A+From+hit+identification+to+GLPG0634&doi=10.1021%2Fjm501262q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Triazolopyridines as Selective JAK1 Inhibitors: From Hit Identification to GLPG0634</span></div><div class="casAuthors">Menet, Christel J.; Fletcher, Stephen R.; Van Lommen, Guy; Geney, Raphael; Blanc, Javier; Smits, Koen; Jouannigot, Nolwenn; Deprez, Pierre; van der Aar, Ellen M.; Clement-Lacroix, Philippe; Lepescheux, Lien; Galien, Rene; Vayssiere, Beatrice; Nelles, Luc; Christophe, Thierry; Brys, Reginald; Uhring, Muriel; Ciesielski, Fabrice; Van Rompaey, Luc</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">9323-9342</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Janus kinases (JAK1, JAK2, JAK3, and TYK2) are involved in the signaling of multiple cytokines important in cellular function.  Blockade of the JAK-STAT pathway with a small mol. has been shown to provide therapeutic immunomodulation.  Having identified JAK1 as a possible new target for arthritis at Galapagos, the compd. library was screened against JAK1, resulting in the identification of a triazolopyridine-based series of inhibitors represented by I.  Optimization within this chem. series led to identification of II (GLPG0634, filgotinib), a selective JAK1 inhibitor currently in phase 2B development for RA and phase 2A development for Crohn's disease (CD).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBHELw4zQ5QLVg90H21EOLACvtfcHk0lgXnoNNsEijvg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVGls7nN&md5=756f687dc30e6fd34004deb672fe8b23</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fjm501262q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501262q%26sid%3Dliteratum%253Aachs%26aulast%3DMenet%26aufirst%3DC.%2BJ.%26aulast%3DFletcher%26aufirst%3DS.%2BR.%26aulast%3DVan%2BLommen%26aufirst%3DG.%26aulast%3DGeney%26aufirst%3DR.%26aulast%3DBlanc%26aufirst%3DJ.%26aulast%3DSmits%26aufirst%3DK.%26aulast%3DJouannigot%26aufirst%3DN.%26aulast%3DDeprez%26aufirst%3DP.%26aulast%3Dvan%2Bder%2BAar%26aufirst%3DE.%2BM.%26aulast%3DClement-Lacroix%26aufirst%3DP.%26aulast%3DLepescheux%26aufirst%3DL.%26aulast%3DGalien%26aufirst%3DR.%26aulast%3DVayssiere%26aufirst%3DB.%26aulast%3DNelles%26aufirst%3DL.%26aulast%3DChristophe%26aufirst%3DT.%26aulast%3DBrys%26aufirst%3DR.%26aulast%3DUhring%26aufirst%3DM.%26aulast%3DCiesielski%26aufirst%3DF.%26aulast%3DVan%2BRompaey%26aufirst%3DL.%26atitle%3DTriazolopyridines%2520as%2520selective%2520JAK1%2520inhibitors%253A%2520From%2520hit%2520identification%2520to%2520GLPG0634%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D9323%26epage%3D9342%26doi%3D10.1021%2Fjm501262q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Genovese, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smolen, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinblatt, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burmester, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meerwein, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Camp, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Othman, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pangan, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jungerwirth, S.</span></span> <span> </span><span class="NLM_article-title">Efficacy and safety of ABT-494, a selective JAK-1 inhibitor, in a phase IIb study in patients with rheumatoid arthritis and an inadequate response to methotrexate</span>. <i>Arthritis Rheumatol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>68</i></span>,  <span class="NLM_fpage">2857</span>– <span class="NLM_lpage">2866</span>, <span class="refDoi"> DOI: 10.1002/art.39808</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=10.1002%2Fart.39808" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=27390150" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFGgsL3I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2016&pages=2857-2866&author=M.+C.+Genoveseauthor=J.+S.+Smolenauthor=M.+E.+Weinblattauthor=G.+R.+Burmesterauthor=S.+Meerweinauthor=H.+L.+Campauthor=L.+Wangauthor=A.+A.+Othmanauthor=N.+Khanauthor=A.+L.+Panganauthor=S.+Jungerwirth&title=Efficacy+and+safety+of+ABT-494%2C+a+selective+JAK-1+inhibitor%2C+in+a+phase+IIb+study+in+patients+with+rheumatoid+arthritis+and+an+inadequate+response+to+methotrexate&doi=10.1002%2Fart.39808"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Efficacy and Safety of ABT-494, a Selective JAK-1 Inhibitor, in a Phase IIb Study in Patients With Rheumatoid Arthritis and an Inadequate Response to Methotrexate</span></div><div class="casAuthors">Genovese, Mark C.; Smolen, Josef S.; Weinblatt, Michael E.; Burmester, Gerd R.; Meerwein, Sebastian; Camp, Heidi S.; Wang, Li; Othman, Ahmed A.; Khan, Nasser; Pangan, Aileen L.; Jungerwirth, Steven</div><div class="citationInfo"><span class="NLM_cas:title">Arthritis & Rheumatology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2857-2866</span>CODEN:
                <span class="NLM_cas:coden">ARRHBO</span>;
        ISSN:<span class="NLM_cas:issn">2326-5191</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Objective : To evaluate the efficacy and safety of ABT-494, a selective JAK-1 inhibitor, in patients with moderate-to-severe rheumatoid arthritis (RA) and an inadequate response to methotrexate (MTX).  Methods : Three hundred RA patients receiving stable doses of MTX were randomly assigned equally to receive immediate-release ABT-494 at 3, 6, 12, or 18 mg twice daily, 24 mg once daily, or placebo for 12 wk.  The primary efficacy end point was the proportion of patients meeting the American College of Rheumatol. 20% improvement criteria (achieving an ACR20 response) at week 12, as detd. using the last observation carried forward method.  Results : At week 12, the proportion of ACR20 responses was higher with ABT-494 (62%, 68%, 80%, 64%, and 76% for the 3, 6, 12, 18, and 24 mg doses, resp.) than with placebo (46%) (using nonresponder imputation) (P < 0.05 for the 6, 12, and 24 mg doses).  There was a significant dose-response relationship among all ABT-494 doses (P < 0.001).  The proportions of patients achieving ACR50 and ACR70 responses were significantly higher for all ABT-494 doses (except the 12 mg dose for the ACR70 response) than for placebo, as were changes in the Disease Activity Score in 28 joints using the C-reactive protein level (DAS28-CRP).  Rapid improvement was demonstrated by significant differences in ACR20 response rates and changes in the DAS28-CRP for all doses compared with placebo at week 2 (the first postbaseline visit).  The incidence of adverse events was similar across groups; most were mild, and infections were the most frequent.  One serious infection (community-acquired pneumonia) occurred with ABT-494 at 12 mg.  There were dose-dependent increases in high-d. lipoprotein (HDL) and low-d. lipoprotein (LDL) cholesterol, but the LDL cholesterol:HDL cholesterol ratios were unchanged through week 12.  Mean Hb levels remained stable at lower doses, but decreases were obsd. at higher doses.  Conclusion : This study evaluated a broad range of doses of ABT-494 in RA patients with an inadequate response to MTX.  ABT-494 demonstrated efficacy, with a safety and tolerability profile similar to that of other JAK inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrc0MsK02mgIbVg90H21EOLACvtfcHk0lig06SAj2gQHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFGgsL3I&md5=5d4bf383496b2153fede781caf6ba0c7</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1002%2Fart.39808&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fart.39808%26sid%3Dliteratum%253Aachs%26aulast%3DGenovese%26aufirst%3DM.%2BC.%26aulast%3DSmolen%26aufirst%3DJ.%2BS.%26aulast%3DWeinblatt%26aufirst%3DM.%2BE.%26aulast%3DBurmester%26aufirst%3DG.%2BR.%26aulast%3DMeerwein%26aufirst%3DS.%26aulast%3DCamp%26aufirst%3DH.%2BL.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DOthman%26aufirst%3DA.%2BA.%26aulast%3DKhan%26aufirst%3DN.%26aulast%3DPangan%26aufirst%3DA.%2BL.%26aulast%3DJungerwirth%26aufirst%3DS.%26atitle%3DEfficacy%2520and%2520safety%2520of%2520ABT-494%252C%2520a%2520selective%2520JAK-1%2520inhibitor%252C%2520in%2520a%2520phase%2520IIb%2520study%2520in%2520patients%2520with%2520rheumatoid%2520arthritis%2520and%2520an%2520inadequate%2520response%2520to%2520methotrexate%26jtitle%3DArthritis%2520Rheumatol.%26date%3D2016%26volume%3D68%26spage%3D2857%26epage%3D2866%26doi%3D10.1002%2Fart.39808" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Williams, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamert, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walden, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilks, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fantino, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossjohn, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucet, I. S.</span></span> <span> </span><span class="NLM_article-title">Dissecting specificity in the Janus kinases: the structures of JAK-specific inhibitors complexed to the JAK1 and JAK2 protein tyrosine kinase domains</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>387</i></span>,  <span class="NLM_fpage">219</span>– <span class="NLM_lpage">232</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2009.01.041</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=10.1016%2Fj.jmb.2009.01.041" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=19361440" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A528%3ADC%252BD1MXivVegtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=387&publication_year=2009&pages=219-232&author=N.+K.+Williamsauthor=R.+S.+Bamertauthor=O.+Patelauthor=C.+Wangauthor=P.+M.+Waldenauthor=A.+F.+Wilksauthor=E.+Fantinoauthor=J.+Rossjohnauthor=I.+S.+Lucet&title=Dissecting+specificity+in+the+Janus+kinases%3A+the+structures+of+JAK-specific+inhibitors+complexed+to+the+JAK1+and+JAK2+protein+tyrosine+kinase+domains&doi=10.1016%2Fj.jmb.2009.01.041"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Dissecting Specificity in the Janus Kinases: The Structures of JAK-Specific Inhibitors Complexed to the JAK1 and JAK2 Protein Tyrosine Kinase Domains</span></div><div class="casAuthors">Williams, Neal K.; Bamert, Rebecca S.; Patel, Onisha; Wang, Christina; Walden, Patricia M.; Wilks, Andrew F.; Fantino, Emmanuelle; Rossjohn, Jamie; Lucet, Isabelle S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">387</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">219-232</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The Janus kinases (JAKs) are a pivotal family of protein tyrosine kinases (PTKs) that play prominent roles in numerous cytokine signaling pathways, with aberrant JAK activity assocd. with a variety of hematopoietic malignancies, cardiovascular diseases and immune-related disorders.  Whereas the structures of the JAK2 and JAK3 PTK domains have been detd., the structure of the JAK1 PTK domain is unknown.  Here, we report the high-resoln. crystal structures of the "active form" of the JAK1 PTK domain in complex with two JAK inhibitors, a tetracyclic pyridone 2-t-butyl-9-fluoro-3,6-dihydro-7H-benz[h]-imidaz[4,5-f]isoquinoline-7-one (CMP6) and (3R,4R)-3-[4-methyl-3-[N-methyl-N-(7H-pyrrolo[2,3-d]pyrimidin-4-yl)amino]piperidin-1-yl]-3-oxopropionitrile (CP-690,550), and compare them with the corresponding JAK2 PTK inhibitor complexes.  Both inhibitors bound in a similar manner to JAK1, namely buried deep within a constricted ATP-binding site, thereby providing a basis for the potent inhibition of JAK1.  As expected, the mode of inhibitor binding in JAK1 was very similar to that obsd. in JAK2, highlighting the challenges in developing JAK-specific inhibitors that target the ATP-binding site.  Nevertheless, differences surrounding the JAK1 and JAK2 ATP-binding sites were apparent, thereby providing a platform for the rational design of JAK2- and JAK1-specific inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQDLkMkCgiLbVg90H21EOLACvtfcHk0lig06SAj2gQHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXivVegtbo%253D&md5=79957755d04323057b5acf7e9d2c56ca</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2009.01.041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2009.01.041%26sid%3Dliteratum%253Aachs%26aulast%3DWilliams%26aufirst%3DN.%2BK.%26aulast%3DBamert%26aufirst%3DR.%2BS.%26aulast%3DPatel%26aufirst%3DO.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DWalden%26aufirst%3DP.%2BM.%26aulast%3DWilks%26aufirst%3DA.%2BF.%26aulast%3DFantino%26aufirst%3DE.%26aulast%3DRossjohn%26aufirst%3DJ.%26aulast%3DLucet%26aufirst%3DI.%2BS.%26atitle%3DDissecting%2520specificity%2520in%2520the%2520Janus%2520kinases%253A%2520the%2520structures%2520of%2520JAK-specific%2520inhibitors%2520complexed%2520to%2520the%2520JAK1%2520and%2520JAK2%2520protein%2520tyrosine%2520kinase%2520domains%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2009%26volume%3D387%26spage%3D219%26epage%3D232%26doi%3D10.1016%2Fj.jmb.2009.01.041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alicea-Velazquez, N. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boggon, T. J.</span></span> <span> </span><span class="NLM_article-title">The use of structural biology in Janus kinase targeted drug discovery</span>. <i>Curr. Drug Targets</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">546</span>– <span class="NLM_lpage">555</span>, <span class="refDoi"> DOI: 10.2174/138945011794751528</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=10.2174%2F138945011794751528" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=21126226" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktVOkurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=546-555&author=N.+L.+Alicea-Velazquezauthor=T.+J.+Boggon&title=The+use+of+structural+biology+in+Janus+kinase+targeted+drug+discovery&doi=10.2174%2F138945011794751528"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">The use of structural biology in Janus kinase targeted drug discovery</span></div><div class="casAuthors">Alicea-Velazquez, Nilda L.; Boggon, Titus J.</div><div class="citationInfo"><span class="NLM_cas:title">Current Drug Targets</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">546-555</span>CODEN:
                <span class="NLM_cas:coden">CDTUAU</span>;
        ISSN:<span class="NLM_cas:issn">1389-4501</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  The Janus kinases (or Jak kinases) mediate cytokine and growth factor signal transduction.  Acquired or inherited Jak mutations can result in dysregulation of Jak-mediated signal transduction and can be crit. to disease acquisition in neoplasias including acute myeloid, acute lymphoblastic and acute megakaryoblastic leukemias, and in rare X-linked severe combined immunodeficiency.  The discovery of an acquired Jak2 point mutation, V617F, in significant nos. of patients with classical myeloproliferative disorders has increased the interest in development of Jak2-specific tyrosine kinase inhibitors and consequently there are now over 20 publically available structures of Jak kinase domains that describe all four family members, Jak1, Jak2, Jak3, and Tyk2.  Here we review the recent advances in understanding the druggable structure and function of the Jak family, with a focus on the structural biol. of the Jak kinase domain.  We will discuss how these advances impact the development of Jak-targeted therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJwEtMK-lOu7Vg90H21EOLACvtfcHk0licH98RoIWeCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktVOkurg%253D&md5=5e76fc7eda3ffe580dc4e28044a88c2c</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.2174%2F138945011794751528&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138945011794751528%26sid%3Dliteratum%253Aachs%26aulast%3DAlicea-Velazquez%26aufirst%3DN.%2BL.%26aulast%3DBoggon%26aufirst%3DT.%2BJ.%26atitle%3DThe%2520use%2520of%2520structural%2520biology%2520in%2520Janus%2520kinase%2520targeted%2520drug%2520discovery%26jtitle%3DCurr.%2520Drug%2520Targets%26date%3D2011%26volume%3D12%26spage%3D546%26epage%3D555%26doi%3D10.2174%2F138945011794751528" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chrencik, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patny, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, I. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korniski, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emmons, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinberg, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gormley, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Day, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirsch, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiefer, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leone, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sommers, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, C. -H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobsen, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tenbrink, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomasselli, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benson, E. T.</span></span> <span> </span><span class="NLM_article-title">Structural and thermodynamic characterization of the TYK2 and JAK3 kinase domains in complex with CP-690550 and CMP-6</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>400</i></span>,  <span class="NLM_fpage">413</span>– <span class="NLM_lpage">433</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2010.05.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=10.1016%2Fj.jmb.2010.05.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=20478313" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A528%3ADC%252BC3cXos1ylt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=400&publication_year=2010&pages=413-433&author=J.+E.+Chrencikauthor=A.+Patnyauthor=I.+K.+Leungauthor=B.+Korniskiauthor=T.+L.+Emmonsauthor=T.+Hallauthor=A.+R.+Weinbergauthor=A.+J.+Gormleyauthor=M.+J.+Williamsauthor=E.+J.+Dayauthor=L.+J.+Hirschauthor=R.+J.+Kieferauthor=W.+J.+Leoneauthor=D.+H.+Fischerauthor=D.+C.+Sommersauthor=C.+-H.+Huangauthor=J.+E.+Jacobsenauthor=E.+R.+Tenbrinkauthor=G.+A.+Tomasselliauthor=E.+T.+Benson&title=Structural+and+thermodynamic+characterization+of+the+TYK2+and+JAK3+kinase+domains+in+complex+with+CP-690550+and+CMP-6&doi=10.1016%2Fj.jmb.2010.05.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Structural and Thermodynamic Characterization of the TYK2 and JAK3 Kinase Domains in Complex with CP-690550 and CMP-6</span></div><div class="casAuthors">Chrencik, Jill E.; Patny, Akshay; Leung, Iris K.; Korniski, Brian; Emmons, Thomas L.; Hall, Troii; Weinberg, Robin A.; Gormley, Jennifer A.; Williams, Jennifer M.; Day, Jacqueline E.; Hirsch, Jeffrey L.; Kiefer, James R.; Leone, Joseph W.; Fischer, H. David; Sommers, Cynthia D.; Huang, Horng-Chih; Jacobsen, E. J.; Tenbrink, Ruth E.; Tomasselli, Alfredo G.; Benson, Timothy E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">400</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">413-433</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Janus kinases (JAKs) are crit. regulators of cytokine pathways and attractive targets of therapeutic value in both inflammatory and myeloproliferative diseases.  Although the crystal structures of active JAK1 and JAK2 kinase domains have been reported recently with the clin. compd. CP-690550, the structures of both TYK2 and JAK3 with CP-690550 have remained outstanding.  Here, we report the crystal structures of TYK2, a first in class structure, and JAK3 in complex with PAN-JAK inhibitors CP-690550 ((3R,4R)-3-[4-methyl-3-[N-methyl-N-(7H-pyrrolo[2,3-d]pyrimidin-4-yl)amino]piperidin-1-yl]-3-oxopropionitrile) and CMP-6 (tetracyclic pyridone 2-t-butyl-9-fluoro-3,6-dihydro-7H-benz[h]-imidaz[4,5-f]isoquinoline-7-one), both of which bind in the ATP-binding cavities of both JAK isoenzymes in orientations similar to that obsd. in crystal structures of JAK1 and JAK2.  Addnl., a complete thermodn. characterization of JAK/CP-690550 complex formation was completed by isothermal titrn. calorimetry, indicating the crit. role of the nitrile group from the CP-690550 compd.  Finally, computational anal. using WaterMap further highlights the crit. positioning of the CP-690550 nitrile group in the displacement of an unfavorable water mol. beneath the glycine-rich loop.  Taken together, the data emphasize the outstanding properties of the kinome-selective JAK inhibitor CP-690550, as well as the challenges in obtaining JAK isoenzyme-selective inhibitors due to the overall structural and sequence similarities between the TYK2, JAK1, JAK2 and JAK3 isoenzymes.  Nevertheless, subtle amino acid variations of residues lining the ligand-binding cavity of the JAK enzymes, as well as the global positioning of the glycine-rich loop, might provide the initial clues to obtaining JAK-isoenzyme selective inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5dnGkvq0jPbVg90H21EOLACvtfcHk0licH98RoIWeCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXos1ylt7g%253D&md5=1b1efc3d6f913143dc9113e4ac1c4c60</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2010.05.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2010.05.020%26sid%3Dliteratum%253Aachs%26aulast%3DChrencik%26aufirst%3DJ.%2BE.%26aulast%3DPatny%26aufirst%3DA.%26aulast%3DLeung%26aufirst%3DI.%2BK.%26aulast%3DKorniski%26aufirst%3DB.%26aulast%3DEmmons%26aufirst%3DT.%2BL.%26aulast%3DHall%26aufirst%3DT.%26aulast%3DWeinberg%26aufirst%3DA.%2BR.%26aulast%3DGormley%26aufirst%3DA.%2BJ.%26aulast%3DWilliams%26aufirst%3DM.%2BJ.%26aulast%3DDay%26aufirst%3DE.%2BJ.%26aulast%3DHirsch%26aufirst%3DL.%2BJ.%26aulast%3DKiefer%26aufirst%3DR.%2BJ.%26aulast%3DLeone%26aufirst%3DW.%2BJ.%26aulast%3DFischer%26aufirst%3DD.%2BH.%26aulast%3DSommers%26aufirst%3DD.%2BC.%26aulast%3DHuang%26aufirst%3DC.%2B-H.%26aulast%3DJacobsen%26aufirst%3DJ.%2BE.%26aulast%3DTenbrink%26aufirst%3DE.%2BR.%26aulast%3DTomasselli%26aufirst%3DG.%2BA.%26aulast%3DBenson%26aufirst%3DE.%2BT.%26atitle%3DStructural%2520and%2520thermodynamic%2520characterization%2520of%2520the%2520TYK2%2520and%2520JAK3%2520kinase%2520domains%2520in%2520complex%2520with%2520CP-690550%2520and%2520CMP-6%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2010%26volume%3D400%26spage%3D413%26epage%3D433%26doi%3D10.1016%2Fj.jmb.2010.05.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Newton, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deiana, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Puleo, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cisneros, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cutrona, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlessinger, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jorgensen, W. L.</span></span> <span> </span><span class="NLM_article-title">JAK2 JH2 fluorescence polarization assay and crystal structures for complexes with three small molecules</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">614</span>– <span class="NLM_lpage">617</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.7b00154</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.7b00154" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A528%3ADC%252BC2sXnvFCnsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=614-617&author=A.+S.+Newtonauthor=L.+Deianaauthor=D.+E.+Puleoauthor=J.+A.+Cisnerosauthor=K.+J.+Cutronaauthor=J.+Schlessingerauthor=W.+L.+Jorgensen&title=JAK2+JH2+fluorescence+polarization+assay+and+crystal+structures+for+complexes+with+three+small+molecules&doi=10.1021%2Facsmedchemlett.7b00154"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">JAK2 JH2 Fluorescence Polarization Assay and Crystal Structures for Complexes with Three Small Molecules</span></div><div class="casAuthors">Newton, Ana S.; Deiana, Luca; Puleo, David E.; Cisneros, Jose A.; Cutrona, Kara J.; Schlessinger, Joseph; Jorgensen, William L.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">614-617</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A competitive fluorescence polarization (FP) assay is reported for detg. binding affinities of probe mols. with the pseudokinase JAK2 JH2 allosteric site.  The syntheses of the fluorescent 5 and 6 used in the assay are reported as well as Kd results for 10 compds., including JNJ7706621, NVP-BSK805, and filgotinib (GLPG0634).  X-ray crystal structures of JAK2 JH2 in complex with NVP-BSK805, filgotinib, and diaminopyrimidine 8 elucidate the binding poses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQUy01N09zFrVg90H21EOLACvtfcHk0licH98RoIWeCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXnvFCnsbY%253D&md5=e7c6f7c58bc9a9fddf18a72b5df4f964</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.7b00154&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.7b00154%26sid%3Dliteratum%253Aachs%26aulast%3DNewton%26aufirst%3DA.%2BS.%26aulast%3DDeiana%26aufirst%3DL.%26aulast%3DPuleo%26aufirst%3DD.%2BE.%26aulast%3DCisneros%26aufirst%3DJ.%2BA.%26aulast%3DCutrona%26aufirst%3DK.%2BJ.%26aulast%3DSchlessinger%26aufirst%3DJ.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26atitle%3DJAK2%2520JH2%2520fluorescence%2520polarization%2520assay%2520and%2520crystal%2520structures%2520for%2520complexes%2520with%2520three%2520small%2520molecules%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D8%26spage%3D614%26epage%3D617%26doi%3D10.1021%2Facsmedchemlett.7b00154" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Flanagan, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blumenkopf, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brissette, W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casavant, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shang-Poa, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doty, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elliott, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fisher, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kent, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kudlacz, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lillie, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magnuson, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCurdy, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munchhof, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perry, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sawyer, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strelevitz, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Subramanyam, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whipple, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Changelian, P. S.</span></span> <span> </span><span class="NLM_article-title">Discovery of CP-690,550: a potent and selective Janus kinase (JAK) inhibitor for the treatment of autoimmune diseases and organ transplant rejection</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">8468</span>– <span class="NLM_lpage">8484</span>, <span class="refDoi"> DOI: 10.1021/jm1004286</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm1004286" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVOgt7vP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=8468-8484&author=M.+E.+Flanaganauthor=T.+A.+Blumenkopfauthor=W.+H.+Brissetteauthor=M.+F.+Brownauthor=J.+M.+Casavantauthor=C.+Shang-Poaauthor=J.+L.+Dotyauthor=E.+A.+Elliottauthor=M.+B.+Fisherauthor=M.+Hinesauthor=C.+Kentauthor=E.+M.+Kudlaczauthor=B.+M.+Lillieauthor=K.+S.+Magnusonauthor=S.+P.+McCurdyauthor=M.+J.+Munchhofauthor=B.+D.+Perryauthor=P.+S.+Sawyerauthor=T.+J.+Strelevitzauthor=C.+Subramanyamauthor=J.+Sunauthor=D.+A.+Whippleauthor=P.+S.+Changelian&title=Discovery+of+CP-690%2C550%3A+a+potent+and+selective+Janus+kinase+%28JAK%29+inhibitor+for+the+treatment+of+autoimmune+diseases+and+organ+transplant+rejection&doi=10.1021%2Fjm1004286"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of CP-690,550: A Potent and Selective Janus Kinase (JAK) Inhibitor for the Treatment of Autoimmune Diseases and Organ Transplant Rejection</span></div><div class="casAuthors">Flanagan, Mark E.; Blumenkopf, Todd A.; Brissette, William H.; Brown, Matthew F.; Casavant, Jeffrey M.; Chang, Shang-Poa; Doty, Jonathan L.; Elliott, Eileen A.; Fisher, Michael B.; Hines, Michael; Kent, Craig; Kudlacz, Elizabeth M.; Lillie, Brett M.; Magnuson, Kelly S.; McCurdy, Sandra P.; Munchhof, Michael J.; Perry, Bret D.; Sawyer, Perry S.; Strelevitz, Timothy J.; Subramanyam, Chakrapani; Sun, Jianmin; Whipple, David A.; Changelian, Paul S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">8468-8484</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">There is a crit. need for safer and more convenient treatments for organ transplant rejection and autoimmune disorders such as rheumatoid arthritis.  Janus tyrosine kinases (JAK1, JAK3) are expressed in lymphoid cells and are involved in the signaling of multiple cytokines important for various T cell functions.  Blockade of the JAK1/JAK3-STAT pathway with a small mol. was anticipated to provide therapeutic immunosuppression/immunomodulation.  The Pfizer compd. library was screened against the catalytic domain of JAK3 resulting in the identification of a pyrrolopyrimidine-based series of inhibitors represented by the hexahydrocarbazolyl pyrrolopyrimidine CP-352,664.  Synthetic analogs of CP-352,664 were screened against the JAK enzymes and evaluated in an IL-2 induced T cell blast proliferation assay.  Select compds. were evaluated in rodent efficacy models of allograft rejection and destructive inflammatory arthritis.  Optimization within this chem. series led to identification of the cyanoacetylpiperidinylamino pyrrolopyrimidine CP-690,550, a potential first-in-class JAK inhibitor for treatment of autoimmune diseases and organ transplant rejection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbFS_8DSHkDbVg90H21EOLACvtfcHk0lilqOwfJ7SNrw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVOgt7vP&md5=c1d31ce42ab3f1c1f3029c6f4f48df1a</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fjm1004286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm1004286%26sid%3Dliteratum%253Aachs%26aulast%3DFlanagan%26aufirst%3DM.%2BE.%26aulast%3DBlumenkopf%26aufirst%3DT.%2BA.%26aulast%3DBrissette%26aufirst%3DW.%2BH.%26aulast%3DBrown%26aufirst%3DM.%2BF.%26aulast%3DCasavant%26aufirst%3DJ.%2BM.%26aulast%3DShang-Poa%26aufirst%3DC.%26aulast%3DDoty%26aufirst%3DJ.%2BL.%26aulast%3DElliott%26aufirst%3DE.%2BA.%26aulast%3DFisher%26aufirst%3DM.%2BB.%26aulast%3DHines%26aufirst%3DM.%26aulast%3DKent%26aufirst%3DC.%26aulast%3DKudlacz%26aufirst%3DE.%2BM.%26aulast%3DLillie%26aufirst%3DB.%2BM.%26aulast%3DMagnuson%26aufirst%3DK.%2BS.%26aulast%3DMcCurdy%26aufirst%3DS.%2BP.%26aulast%3DMunchhof%26aufirst%3DM.%2BJ.%26aulast%3DPerry%26aufirst%3DB.%2BD.%26aulast%3DSawyer%26aufirst%3DP.%2BS.%26aulast%3DStrelevitz%26aufirst%3DT.%2BJ.%26aulast%3DSubramanyam%26aufirst%3DC.%26aulast%3DSun%26aufirst%3DJ.%26aulast%3DWhipple%26aufirst%3DD.%2BA.%26aulast%3DChangelian%26aufirst%3DP.%2BS.%26atitle%3DDiscovery%2520of%2520CP-690%252C550%253A%2520a%2520potent%2520and%2520selective%2520Janus%2520kinase%2520%2528JAK%2529%2520inhibitor%2520for%2520the%2520treatment%2520of%2520autoimmune%2520diseases%2520and%2520organ%2520transplant%2520rejection%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D8468%26epage%3D8484%26doi%3D10.1021%2Fjm1004286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Noji, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hara, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamanaka, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maeda, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hori, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Obika, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inoue, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hase, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orita, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adachi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanimoto, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oki, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kimoto, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogawa, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Negoro, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hashimoto, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shiozaki, M.</span></span> <span> </span><span class="NLM_article-title">Discovery of a Janus kinase inhibitor bearing a highly three-dimensional spiro scaffold: JTE-052 (delgocitinib) as a new dermatological agent to treat inflammatory skin disorders</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">7163</span>– <span class="NLM_lpage">7185</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00450</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00450" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtFWmsLrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=7163-7185&author=S.+Nojiauthor=Y.+Haraauthor=T.+Miuraauthor=H.+Yamanakaauthor=K.+Maedaauthor=A.+Horiauthor=H.+Yamamotoauthor=S.+Obikaauthor=M.+Inoueauthor=Y.+Haseauthor=T.+Oritaauthor=S.+Doiauthor=T.+Adachiauthor=A.+Tanimotoauthor=C.+Okiauthor=Y.+Kimotoauthor=Y.+Ogawaauthor=T.+Negoroauthor=H.+Hashimotoauthor=M.+Shiozaki&title=Discovery+of+a+Janus+kinase+inhibitor+bearing+a+highly+three-dimensional+spiro+scaffold%3A+JTE-052+%28delgocitinib%29+as+a+new+dermatological+agent+to+treat+inflammatory+skin+disorders&doi=10.1021%2Facs.jmedchem.0c00450"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Janus Kinase Inhibitor Bearing a Highly Three-Dimensional Spiro Scaffold: JTE-052 (Delgocitinib) as a New Dermatological Agent to Treat Inflammatory Skin Disorders</span></div><div class="casAuthors">Noji, Satoru; Hara, Yoshinori; Miura, Tomoya; Yamanaka, Hiroshi; Maeda, Katsuya; Hori, Akimi; Yamamoto, Hiroshi; Obika, Shingo; Inoue, Masafumi; Hase, Yasunori; Orita, Takuya; Doi, Satoki; Adachi, Tsuyoshi; Tanimoto, Atsuo; Oki, Chika; Kimoto, Yukari; Ogawa, Yoshihiro; Negoro, Tamotsu; Hashimoto, Hiromasa; Shiozaki, Makoto</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">7163-7185</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Dermatol. disorders such as atopic dermatitis arise from genetic and environmental causes and are complex and multifactorial in nature.  Among possible risk factors, aberrant immunol. reactions are one of the leading etiologies.  Immunosuppressive agents including topical steroids are common treatments for these disorders.  Despite their reliability in clin. settings, topical steroids display side effects, typified by skin thinning.  Accordingly, there is a need for alternate effective and well-tolerated therapies.  As part of our efforts to investigate new immunomodulators, we have developed a series of JAK inhibitors, which incorporate novel three-dimensional spiro motifs and unexpectedly possess both excellent physicochem. properties and antidermatitis efficacy in the animal models.  One of these compds., JTE-052 (ent-60), also known as delgocitinib, has been shown to be effective and well-tolerated in human clin. trials and has recently been approved in Japan for the treatment of atopic dermatitis as the first drug among Janus kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqa4KSm53XHzLVg90H21EOLACvtfcHk0lilqOwfJ7SNrw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtFWmsLrM&md5=f4eecf28446abb441f1436c41d8cb0a6</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00450&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00450%26sid%3Dliteratum%253Aachs%26aulast%3DNoji%26aufirst%3DS.%26aulast%3DHara%26aufirst%3DY.%26aulast%3DMiura%26aufirst%3DT.%26aulast%3DYamanaka%26aufirst%3DH.%26aulast%3DMaeda%26aufirst%3DK.%26aulast%3DHori%26aufirst%3DA.%26aulast%3DYamamoto%26aufirst%3DH.%26aulast%3DObika%26aufirst%3DS.%26aulast%3DInoue%26aufirst%3DM.%26aulast%3DHase%26aufirst%3DY.%26aulast%3DOrita%26aufirst%3DT.%26aulast%3DDoi%26aufirst%3DS.%26aulast%3DAdachi%26aufirst%3DT.%26aulast%3DTanimoto%26aufirst%3DA.%26aulast%3DOki%26aufirst%3DC.%26aulast%3DKimoto%26aufirst%3DY.%26aulast%3DOgawa%26aufirst%3DY.%26aulast%3DNegoro%26aufirst%3DT.%26aulast%3DHashimoto%26aufirst%3DH.%26aulast%3DShiozaki%26aufirst%3DM.%26atitle%3DDiscovery%2520of%2520a%2520Janus%2520kinase%2520inhibitor%2520bearing%2520a%2520highly%2520three-dimensional%2520spiro%2520scaffold%253A%2520JTE-052%2520%2528delgocitinib%2529%2520as%2520a%2520new%2520dermatological%2520agent%2520to%2520treat%2520inflammatory%2520skin%2520disorders%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D7163%26epage%3D7185%26doi%3D10.1021%2Facs.jmedchem.0c00450" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nakajima, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inoue, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakai, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mukoyoshi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamaguchi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatanaka, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sasaki, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kunikawa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Usuda, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moritomo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Higashi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inami, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shirakam, S.</span></span> <span> </span><span class="NLM_article-title">Synthesis and evaluation of novel 1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-5-carboxamide derivatives as potent and orally efficacious immune modulators targeting</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">4871</span>– <span class="NLM_lpage">4883</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2015.05.034</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=10.1016%2Fj.bmc.2015.05.034" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=26071372" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A528%3ADC%252BC2MXptFOktrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2015&pages=4871-4883&author=Y.+Nakajimaauthor=T.+Inoueauthor=K.+Nakaiauthor=K.+Mukoyoshiauthor=H.+Hamaguchiauthor=K.+Hatanakaauthor=H.+Sasakiauthor=A.+Tanakaauthor=F.+Takahashiauthor=S.+Kunikawaauthor=H.+Usudaauthor=A.+Moritomoauthor=Y.+Higashiauthor=M.+Inamiauthor=S.+Shirakam&title=Synthesis+and+evaluation+of+novel+1H-pyrrolo%5B2%2C3-b%5Dpyridine-5-carboxamide+derivatives+as+potent+and+orally+efficacious+immune+modulators+targeting&doi=10.1016%2Fj.bmc.2015.05.034"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and evaluation of novel 1H-pyrrolo[2,3-b]pyridine-5-carboxamide derivatives as potent and orally efficacious immunomodulators targeting JAK3</span></div><div class="casAuthors">Nakajima, Yutaka; Inoue, Takayuki; Nakai, Kazuo; Mukoyoshi, Koichiro; Hamaguchi, Hisao; Hatanaka, Keiko; Sasaki, Hiroshi; Tanaka, Akira; Takahashi, Fumie; Kunikawa, Shigeki; Usuda, Hiroyuki; Moritomo, Ayako; Higashi, Yasuyuki; Inami, Masamichi; Shirakami, Shohei</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">4871-4883</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Janus kinases (JAKs) regulate various inflammatory and immune responses and are targets for the treatment of inflammatory and immune diseases.  As a novel class of immunomodulators targeting JAK3, 1H-pyrrolo[2,3-b]pyridine-5-carboxamide derivs. are promising candidates for treating such diseases.  In chem. modification of lead compd. (I), the substitution of a cycloalkyl ring for an N-cyanopyridylpiperidine in C4-position was effective for increasing JAK3 inhibitory activity.  In addn., modulation of phys. properties such as mol. lipophilicity and basicity was important for reducing human ether-a-go-go-related gene (hERG) inhibitory activity.  The authors' optimization study gave compd. 31 (4-[[(3S,4R)-1-(5-cyanopyridin-2-yl)-3-fluoropiperidin-4-yl]amino]-1H-pyrrolo[2,3-b]pyridine-5-carboxamide), which exhibited potent JAK3 inhibitory activity as well as weak hERG inhibitory activity.  In cellular assay, 31 exhibited potent immunomodulating effect on IL-2-stimulated T cell proliferation.  In a pharmacokinetic study, good metabolic stability and oral bioavailability of 31 were achieved in rats, dogs, and monkeys.  Further, 31 prolonged graft survival in an in vivo rat heterotopic cardiac transplant model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqc-PiU8OQJ_bVg90H21EOLACvtfcHk0li4JLgsiMdm4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXptFOktrc%253D&md5=378ff172c2374ea40d784c59f1e192d7</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2015.05.034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2015.05.034%26sid%3Dliteratum%253Aachs%26aulast%3DNakajima%26aufirst%3DY.%26aulast%3DInoue%26aufirst%3DT.%26aulast%3DNakai%26aufirst%3DK.%26aulast%3DMukoyoshi%26aufirst%3DK.%26aulast%3DHamaguchi%26aufirst%3DH.%26aulast%3DHatanaka%26aufirst%3DK.%26aulast%3DSasaki%26aufirst%3DH.%26aulast%3DTanaka%26aufirst%3DA.%26aulast%3DTakahashi%26aufirst%3DF.%26aulast%3DKunikawa%26aufirst%3DS.%26aulast%3DUsuda%26aufirst%3DH.%26aulast%3DMoritomo%26aufirst%3DA.%26aulast%3DHigashi%26aufirst%3DY.%26aulast%3DInami%26aufirst%3DM.%26aulast%3DShirakam%26aufirst%3DS.%26atitle%3DSynthesis%2520and%2520evaluation%2520of%2520novel%25201H-pyrrolo%255B2%252C3-b%255Dpyridine-5-carboxamide%2520derivatives%2520as%2520potent%2520and%2520orally%2520efficacious%2520immune%2520modulators%2520targeting%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2015%26volume%3D23%26spage%3D4871%26epage%3D4883%26doi%3D10.1016%2Fj.bmc.2015.05.034" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schönherr, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cernak, T.</span></span> <span> </span><span class="NLM_article-title">Profound methyl effects in drug discovery and a call for new C-H methylation reactions</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">12256</span>– <span class="NLM_lpage">12267</span>, <span class="refDoi"> DOI: 10.1002/anie.201303207</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=10.1002%2Fanie.201303207" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A280%3ADC%252BC2c%252FptVahuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2013&pages=12256-12267&author=H.+Sch%C3%B6nherrauthor=T.+Cernak&title=Profound+methyl+effects+in+drug+discovery+and+a+call+for+new+C-H+methylation+reactions&doi=10.1002%2Fanie.201303207"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Profound methyl effects in drug discovery and a call for new C-H methylation reactions</span></div><div class="casAuthors">Schonherr Heike; Cernak Tim</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie (International ed. in English)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">47</span>),
    <span class="NLM_cas:pages">12256-67</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The methyl group is one of the most commonly occurring carbon fragments in small-molecule drugs.  This simplest alkyl fragment appears in more than 67 % of the top-selling drugs of 2011 and can modulate both the biological and physical properties of a molecule.  This Review focuses on so-called magic methyl effects on binding potency, where the seemingly mundane change of C H to C Me improves the IC50  value of a drug candidate more than 100-fold.  This discussion is followed by a survey of recent advances in synthetic chemistry that allow the direct methylation of C(sp(2) ) H and C(sp(3) ) H bonds.  It is our hope that the relevance of the meager methyl group to drug discovery as presented herein will inspire reports on new C H methylation reactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTyoLZDDgvsMh0LsK7uFuY7fW6udTcc2ebrf_49KN_t_7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2c%252FptVahuw%253D%253D&md5=ef08f80e55b2c530b49e4306da024902</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1002%2Fanie.201303207&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201303207%26sid%3Dliteratum%253Aachs%26aulast%3DSch%25C3%25B6nherr%26aufirst%3DH.%26aulast%3DCernak%26aufirst%3DT.%26atitle%3DProfound%2520methyl%2520effects%2520in%2520drug%2520discovery%2520and%2520a%2520call%2520for%2520new%2520C-H%2520methylation%2520reactions%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2013%26volume%3D52%26spage%3D12256%26epage%3D12267%26doi%3D10.1002%2Fanie.201303207" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sherman, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Day, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farid, R.</span></span> <span> </span><span class="NLM_article-title">Novel procedure for modeling ligand/receptor induced fit effects</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">534</span>– <span class="NLM_lpage">553</span>, <span class="refDoi"> DOI: 10.1021/jm050540c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm050540c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlCgsr7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=534-553&author=W.+Shermanauthor=T.+Dayauthor=M.+P.+Jacobsonauthor=R.+A.+Friesnerauthor=R.+Farid&title=Novel+procedure+for+modeling+ligand%2Freceptor+induced+fit+effects&doi=10.1021%2Fjm050540c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Procedure for Modeling Ligand/Receptor Induced Fit Effects</span></div><div class="casAuthors">Sherman, Woody; Day, Tyler; Jacobson, Matthew P.; Friesner, Richard A.; Farid, Ramy</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">534-553</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We present a novel protein-ligand docking method that accurately accounts for both ligand and receptor flexibility by iteratively combining rigid receptor docking (Glide) with protein structure prediction (Prime) techniques.  While traditional rigid-receptor docking methods are useful when the receptor structure does not change substantially upon ligand binding, success is limited when the protein must be "induced" into the correct binding conformation for a given ligand.  We provide an in-depth description of our novel methodol. and present results for 21 pharmaceutically relevant examples.  Traditional rigid-receptor docking for these 21 cases yields an av. RMSD of 5.5 Å.  The av. ligand RMSD for docking to a flexible receptor for the 21 pairs is 1.4 Å; the RMSD is ≤1.8 Å for 18 of the cases.  For the three cases with RMSDs greater than 1.8 Å, the core of the ligand is properly docked and all key protein/ligand interactions are captured.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqm3E6ctnPGSrVg90H21EOLACvtfcHk0li4JLgsiMdm4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtlCgsr7I&md5=388811ead5cee1fd460951263de486cb</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fjm050540c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm050540c%26sid%3Dliteratum%253Aachs%26aulast%3DSherman%26aufirst%3DW.%26aulast%3DDay%26aufirst%3DT.%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DFarid%26aufirst%3DR.%26atitle%3DNovel%2520procedure%2520for%2520modeling%2520ligand%252Freceptor%2520induced%2520fit%2520effects%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D534%26epage%3D553%26doi%3D10.1021%2Fjm050540c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Di, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerns, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourassa, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huryn, D. M.</span></span> <span> </span><span class="NLM_article-title">Experimental design on single-time-point high-throughput microsomal stability assay</span>. <i>J. Pharm. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>93</i></span>,  <span class="NLM_fpage">1537</span>– <span class="NLM_lpage">1544</span>, <span class="refDoi"> DOI: 10.1002/jps.20076</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=10.1002%2Fjps.20076" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=15124211" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A528%3ADC%252BD2cXks12hsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=2004&pages=1537-1544&author=L.+Diauthor=E.+H.+Kernsauthor=N.+Gaoauthor=S.+Q.+Liauthor=Y.+Huangauthor=J.+L.+Bourassaauthor=D.+M.+Huryn&title=Experimental+design+on+single-time-point+high-throughput+microsomal+stability+assay&doi=10.1002%2Fjps.20076"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Experimental design on single-time-point high-throughput microsomal stability assay</span></div><div class="casAuthors">Di, Li; Kerns, Edward H.; Gao, Ning; Li, Susan Q.; Huang, Youping; Bourassa, Jim L.; Huryn, Donna M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmaceutical Sciences</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1537-1544</span>CODEN:
                <span class="NLM_cas:coden">JPMSAE</span>;
        ISSN:<span class="NLM_cas:issn">0022-3549</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">An exptl. design for a single-time-point microsomal stability assay was evaluated as compared with multiple-time-point studies.  Results obtained from single-time-point expts. are in excellent agreement with those from multiple time points.  First-order reaction kinetics revealed rapid changes of predicted half-life from percent remaining of the parent compd. at the inflection points, suggesting a max. predictive limit for half-life.  Selection of the incubation time in single-time-point assays is important to obtain balanced information for stable and unstable compds.  A short incubation time (e.g., 5 min) is most useful for differentiating between unstable compds., which is beneficial to direct the synthetic efforts in projects with poor metabolic stability.  A long incubation time (e.g., 30 min) is more applicable to a compd. series with high metabolic stability.  For screening purposes, a moderate incubation time (e.g., 15 min) is recommended to achieve good resoln. and a sufficiently high max. predictive limit for half-life.  This study suggests that a single-time-point assay is sufficient for ranking compds. in early drug discovery.  It increases throughput and reduces turnaround time and cost.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdHPH7tWzrIrVg90H21EOLACvtfcHk0lhHSkLxGuSSYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXks12hsr4%253D&md5=45b3bfb054ad93e18655b988fdbbe89b</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1002%2Fjps.20076&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjps.20076%26sid%3Dliteratum%253Aachs%26aulast%3DDi%26aufirst%3DL.%26aulast%3DKerns%26aufirst%3DE.%2BH.%26aulast%3DGao%26aufirst%3DN.%26aulast%3DLi%26aufirst%3DS.%2BQ.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DBourassa%26aufirst%3DJ.%2BL.%26aulast%3DHuryn%26aufirst%3DD.%2BM.%26atitle%3DExperimental%2520design%2520on%2520single-time-point%2520high-throughput%2520microsomal%2520stability%2520assay%26jtitle%3DJ.%2520Pharm.%2520Sci.%26date%3D2004%26volume%3D93%26spage%3D1537%26epage%3D1544%26doi%3D10.1002%2Fjps.20076" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cha, I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shon, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shin, J. G.</span></span> <span> </span><span class="NLM_article-title">High-throughput screening of inhibitory potential of nine cytochrome P450 enzymes in vitro using liquid chromatography/tandem mass spectrometry</span>. <i>Rapid Commun. Mass Spectrom.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">2651</span>– <span class="NLM_lpage">2658</span>, <span class="refDoi"> DOI: 10.1002/rcm.2110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=10.1002%2Frcm.2110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=16124035" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtVKntr%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2005&pages=2651-2658&author=M.+J.+Kimauthor=H.+Kimauthor=I.+J.+Chaauthor=J.+S.+Parkauthor=J.+H.+Shonauthor=K.+H.+Liuauthor=J.+G.+Shin&title=High-throughput+screening+of+inhibitory+potential+of+nine+cytochrome+P450+enzymes+in+vitro+using+liquid+chromatography%2Ftandem+mass+spectrometry&doi=10.1002%2Frcm.2110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">High-throughput screening of inhibitory potential of nine cytochrome P450 enzymes in vitro using liquid chromatography/tandem mass spectrometry</span></div><div class="casAuthors">Kim, Min-Jung; Kim, Hyojin; Cha, In-June; Park, Jang-Su; Shon, Ji-Hong; Liu, Kwang-Hyeon; Shin, Jae-Gook</div><div class="citationInfo"><span class="NLM_cas:title">Rapid Communications in Mass Spectrometry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">2651-2658</span>CODEN:
                <span class="NLM_cas:coden">RCMSEF</span>;
        ISSN:<span class="NLM_cas:issn">0951-4198</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons Ltd.</span>)
        </div><div class="casAbstract">The early detection of potential drug-drug interactions is an important issue of drug discovery that has led to the development of high-throughput screening (HTS) methods for potential drug interactions.  We developed a HTS method for potential interactions of inhibitory drugs for nine human P 450 enzymes using cocktail incubation and tandem mass spectrometry in vitro.  This new method involves incubation of two cocktail doses and single cassette anal.  The two cocktail doses in vitro were developed to minimize solvent effects and mutual drug interactions among substrates: cocktail A was composed of phenacetin for CYP1A2, coumarin for CYP2A6, paclitaxel for CYP2C8, S-mephenytoin for CYP2C19, dextromethorphan for CYP2D6, and midazolam for CYP3A4; and cocktail B was composed of three substrates including bupropion for CYP2B6, tolbutamide for CYP2C9, and chlorzoxazone for CYP2E1.  In the incubation study of these cocktails, the reaction mixts. were pooled and simultaneously analyzed using liq. chromatog./tandem mass spectrometry employing a fast gradient.  The method was validated by comparing the inhibition data obtained from the incubation of each individual probe substrate alone with data from the new method.  The IC50 value of each inhibitor in the cocktail agreed well with that of the individual probe drug as well as with values previously reported in the literature.  As a HTS method for potential interactions of the inhibition of these nine P 450 enzymes, this new method will be useful in the drug discovery process and for the mechanistic understanding of drug interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLoIA81FWoWrVg90H21EOLACvtfcHk0lhHSkLxGuSSYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtVKntr%252FJ&md5=38747a0def46d55e0e6223ac234772cd</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1002%2Frcm.2110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Frcm.2110%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DM.%2BJ.%26aulast%3DKim%26aufirst%3DH.%26aulast%3DCha%26aufirst%3DI.%2BJ.%26aulast%3DPark%26aufirst%3DJ.%2BS.%26aulast%3DShon%26aufirst%3DJ.%2BH.%26aulast%3DLiu%26aufirst%3DK.%2BH.%26aulast%3DShin%26aufirst%3DJ.%2BG.%26atitle%3DHigh-throughput%2520screening%2520of%2520inhibitory%2520potential%2520of%2520nine%2520cytochrome%2520P450%2520enzymes%2520in%2520vitro%2520using%2520liquid%2520chromatography%252Ftandem%2520mass%2520spectrometry%26jtitle%3DRapid%2520Commun.%2520Mass%2520Spectrom.%26date%3D2005%26volume%3D19%26spage%3D2651%26epage%3D2658%26doi%3D10.1002%2Frcm.2110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Di, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerns, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span> <span> </span><span class="NLM_article-title">Development and application of high throughput plasma stability assay for drug discovery</span>. <i>Int. J. Pharm.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>297</i></span>,  <span class="NLM_fpage">110</span>– <span class="NLM_lpage">119</span>, <span class="refDoi"> DOI: 10.1016/j.ijpharm.2005.03.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=10.1016%2Fj.ijpharm.2005.03.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=15876500" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A528%3ADC%252BD2MXkt1ansrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=297&publication_year=2005&pages=110-119&author=L.+Diauthor=E.+H.+Kernsauthor=Y.+Hongauthor=H.+Chen&title=Development+and+application+of+high+throughput+plasma+stability+assay+for+drug+discovery&doi=10.1016%2Fj.ijpharm.2005.03.022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Development and application of high throughput plasma stability assay for drug discovery</span></div><div class="casAuthors">Di, Li; Kerns, Edward H.; Hong, Yan; Chen, Hong</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Pharmaceutics</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">297</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">110-119</span>CODEN:
                <span class="NLM_cas:coden">IJPHDE</span>;
        ISSN:<span class="NLM_cas:issn">0378-5173</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Plasma stability plays an important role in drug discovery and development.  Unstable compds. tend to have rapid clearance and short half-life, resulting in poor in vivo performance.  This paper examines the variables that affect the plasma stability assay results, including substrate concn., %DMSO, plasma concn., enzyme activity upon incubation and batch variation.  The results show that plasma stability can accommodate a wide range of exptl. conditions.  Relatively minor differences in results are produced with major differences in conditions.  Significant batch-to-batch variations were obsd. for rat plasma.  We selected the following conditions: 1 μM substrate concn., 2.5% DMSO, and 50% diln. of plasma in pH 7.4 buffer.  Plasma stability can be used as a diagnostic assay when compds. are unexpectedly rapidly cleared, as a special assay when structural classes contain groups that may be susceptible to plasma enzyme hydrolysis, or as general screen for compds. if resources are available.  Plasma stability assay has many applications in drug discovery: to alert teams to labile structural motifs, to prioritize compds. for in vivo studies and to screen prodrugs and antedrugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZUoDVyphu6bVg90H21EOLACvtfcHk0lhHSkLxGuSSYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXkt1ansrg%253D&md5=cee758567f2d75838b28817a1040f855</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2Fj.ijpharm.2005.03.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ijpharm.2005.03.022%26sid%3Dliteratum%253Aachs%26aulast%3DDi%26aufirst%3DL.%26aulast%3DKerns%26aufirst%3DE.%2BH.%26aulast%3DHong%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DH.%26atitle%3DDevelopment%2520and%2520application%2520of%2520high%2520throughput%2520plasma%2520stability%2520assay%2520for%2520drug%2520discovery%26jtitle%3DInt.%2520J.%2520Pharm.%26date%3D2005%26volume%3D297%26spage%3D110%26epage%3D119%26doi%3D10.1016%2Fj.ijpharm.2005.03.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Irvine, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lockhart, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tolan, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Selick, H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grove, J. R.</span></span> <span> </span><span class="NLM_article-title">MDCK (Madin–Darby canine kidney) cells: a tool for membrane permeability screening</span>. <i>J. Pharm. Sci.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>88</i></span>,  <span class="NLM_fpage">28</span>– <span class="NLM_lpage">33</span>, <span class="refDoi"> DOI: 10.1021/js9803205</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=10.1021%2Fjs9803205" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=9874698" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A528%3ADyaK1cXns1ylurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=88&publication_year=1999&pages=28-33&author=J.+D.+Irvineauthor=L.+Takahashiauthor=K.+Lockhartauthor=J.+Cheongauthor=J.+W.+Tolanauthor=H.+E.+Selickauthor=J.+R.+Grove&title=MDCK+%28Madin%E2%80%93Darby+canine+kidney%29+cells%3A+a+tool+for+membrane+permeability+screening&doi=10.1021%2Fjs9803205"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">MDCK (Madin-Darby canine kidney) cells: a tool for membrane permeability screening</span></div><div class="casAuthors">Irvine, Jennifer D.; Takahashi, Lori; Lockhart, Karen; Cheong, Jonathan; Tolan, John W.; Selick, H. E.; Grove, J. Russell</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmaceutical Sciences</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">88</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">28-33</span>CODEN:
                <span class="NLM_cas:coden">JPMSAE</span>;
        ISSN:<span class="NLM_cas:issn">0022-3549</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The goal of this work was to investigate the use of MDCK (Madin-Darby canine kidney) cells as a possible tool for assessing the membrane permeability properties of early drug discovery compds.  Apparent permeability (Papp) values of 55 compds. with known human absorption values were detd. using MDCK cell monolayers.  For comparison, Papp values of the same compds. were also detd. using Caco-2 cells, a well-characterized in vitro model of intestinal drug absorption.  Monolayers were grown on 0.4-μm Transwell-COL membrane culture inserts. MDCK cells were seeded at high d. and cultured for 3 days, and Caco-2 cells were cultured under std. conditions for 21 to 25 days.  Compds. were tested using 100 μM donor solns. in transport medium (pH 7.4) contg. 1% DMSO.  The Papp values in MDCK cells correlated well with those in Caco-2 cells (r2 = 0.79). Spearman's rank correlation coeff. for MDCK Papp and human absorption was 0.58 compared with 0.54 for Caco-2 Papp and human absorption.  These results indicate that MDCK cells may be a useful tool for rapid membrane permeability screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolDvo0FQMg7bVg90H21EOLACvtfcHk0lhHSkLxGuSSYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXns1ylurk%253D&md5=ea421117433fa4067578476edde9444e</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fjs9803205&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjs9803205%26sid%3Dliteratum%253Aachs%26aulast%3DIrvine%26aufirst%3DJ.%2BD.%26aulast%3DTakahashi%26aufirst%3DL.%26aulast%3DLockhart%26aufirst%3DK.%26aulast%3DCheong%26aufirst%3DJ.%26aulast%3DTolan%26aufirst%3DJ.%2BW.%26aulast%3DSelick%26aufirst%3DH.%2BE.%26aulast%3DGrove%26aufirst%3DJ.%2BR.%26atitle%3DMDCK%2520%2528Madin%25E2%2580%2593Darby%2520canine%2520kidney%2529%2520cells%253A%2520a%2520tool%2520for%2520membrane%2520permeability%2520screening%26jtitle%3DJ.%2520Pharm.%2520Sci.%26date%3D1999%26volume%3D88%26spage%3D28%26epage%3D33%26doi%3D10.1021%2Fjs9803205" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, H.</span></span> <span> </span><span class="NLM_article-title">Protein kinase profiling assays: a technology review</span>. <i>Drug Discovery Today Technol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">8</span>, <span class="refDoi"> DOI: 10.1016/j.ddtec.2015.10.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=10.1016%2Fj.ddtec.2015.10.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=26723886" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A280%3ADC%252BC28rms1Wisw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2015&pages=1-8&author=Y.+Wangauthor=H.+Ma&title=Protein+kinase+profiling+assays%3A+a+technology+review&doi=10.1016%2Fj.ddtec.2015.10.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Protein kinase profiling assays: a technology review</span></div><div class="casAuthors">Wang Yuren; Ma Haiching</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today. Technologies</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1-8</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Protein kinases have become one of the most intensively pursued classes of drug targets for many diseases such as cancers and inflammatory diseases.  Kinase profiling work seeks to understand general selectivity trends of lead compounds across the kinome, which help with target selection, compound prioritization, and potential implications in toxicity.  Under the current drug discovery process, screening of compounds against comprehensive panels of kinases and their mutants has become the standard approach.  Many screening assays and technologies which are compatible for high-throughput screening (HTS) against kinases have been extensively pursued and developed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ2PU1XOycdfMve_zCu79I8fW6udTcc2eYkxWNO7dF8erntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28rms1Wisw%253D%253D&md5=89bc86678797ea211e5a2e4b6a7da709</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2015.10.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2015.10.007%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DH.%26atitle%3DProtein%2520kinase%2520profiling%2520assays%253A%2520a%2520technology%2520review%26jtitle%3DDrug%2520Discovery%2520Today%2520Technol.%26date%3D2015%26volume%3D18%26spage%3D1%26epage%3D8%26doi%3D10.1016%2Fj.ddtec.2015.10.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chakraborty, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Šumová, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mallano, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Distler, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergmann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ludolph, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horch, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gelse, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramming, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Distler, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schett, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Šenolt, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Distler, J. H. W.</span></span> <span> </span><span class="NLM_article-title">Activation of STAT3 integrates common profibrotic pathways to promote fibroblast activation and tissue fibrosTransforming Growth Factor-β (TGF-β) Directly Activates the JAK1-STAT3 Axis to Induce Hepatic Fibrosis in Coordination with the SMAD Pathwayis</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1130</span>, <span class="refDoi"> DOI: 10.1038/s41467-017-01236-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=10.1038%2Fs41467-017-01236-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=29066712" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A280%3ADC%252BC1M7ktFGqsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=1130&author=D.+Chakrabortyauthor=B.+%C5%A0umov%C3%A1author=T.+Mallanoauthor=C.-W.+Chenauthor=A.+Distlerauthor=C.+Bergmannauthor=I.+Ludolphauthor=R.+E.+Horchauthor=K.+Gelseauthor=A.+Rammingauthor=O.+Distlerauthor=G.+Schettauthor=L.+%C5%A0enoltauthor=J.+H.+W.+Distler&title=Activation+of+STAT3+integrates+common+profibrotic+pathways+to+promote+fibroblast+activation+and+tissue+fibrosTransforming+Growth+Factor-%CE%B2+%28TGF-%CE%B2%29+Directly+Activates+the+JAK1-STAT3+Axis+to+Induce+Hepatic+Fibrosis+in+Coordination+with+the+SMAD+Pathwayis&doi=10.1038%2Fs41467-017-01236-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Activation of STAT3 integrates common profibrotic pathways to promote fibroblast activation and tissue fibrosis</span></div><div class="casAuthors">Chakraborty Debomita; Sumova Barbora; Mallano Tatjana; Chen Chih-Wei; Distler Alfiya; Bergmann Christina; Ramming Andreas; Schett Georg; Distler Jorg H W; Sumova Barbora; Senolt Ladislav; Ludolph Ingo; Horch Raymund E; Gelse Kolja; Distler Oliver</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1130</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Signal transducer and activator of transcription 3 (STAT3) is phosphorylated by various kinases, several of which have been implicated in aberrant fibroblast activation in fibrotic diseases including systemic sclerosis (SSc).  Here we show that profibrotic signals converge on STAT3 and that STAT3 may be an important molecular checkpoint for tissue fibrosis.  STAT3 signaling is hyperactivated in SSc in a TGFβ-dependent manner.  Expression profiling and functional studies in vitro and in vivo demonstrate that STAT3 activation is mediated by the combined action of JAK, SRC, c-ABL, and JNK kinases.  STAT3-deficient fibroblasts are less sensitive to the pro-fibrotic effects of TGFβ.  Fibroblast-specific knockout of STAT3, or its pharmacological inhibition, ameliorate skin fibrosis in experimental mouse models.  STAT3 thus integrates several profibrotic signals and might be a core mediator of fibrosis.  Considering that several STAT3 inhibitors are currently tested in clinical trials, STAT3 might be a candidate for molecular targeted therapies of SSc.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT4MyXAypwc0ORS7coPn-RtfW6udTcc2eYkxWNO7dF8erntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M7ktFGqsg%253D%253D&md5=7bf14601b39a53147071c7a160636831</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1038%2Fs41467-017-01236-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-017-01236-6%26sid%3Dliteratum%253Aachs%26aulast%3DChakraborty%26aufirst%3DD.%26aulast%3D%25C5%25A0umov%25C3%25A1%26aufirst%3DB.%26aulast%3DMallano%26aufirst%3DT.%26aulast%3DChen%26aufirst%3DC.-W.%26aulast%3DDistler%26aufirst%3DA.%26aulast%3DBergmann%26aufirst%3DC.%26aulast%3DLudolph%26aufirst%3DI.%26aulast%3DHorch%26aufirst%3DR.%2BE.%26aulast%3DGelse%26aufirst%3DK.%26aulast%3DRamming%26aufirst%3DA.%26aulast%3DDistler%26aufirst%3DO.%26aulast%3DSchett%26aufirst%3DG.%26aulast%3D%25C5%25A0enolt%26aufirst%3DL.%26aulast%3DDistler%26aufirst%3DJ.%2BH.%2BW.%26atitle%3DActivation%2520of%2520STAT3%2520integrates%2520common%2520profibrotic%2520pathways%2520to%2520promote%2520fibroblast%2520activation%2520and%2520tissue%2520fibrosTransforming%2520Growth%2520Factor-%25CE%25B2%2520%2528TGF-%25CE%25B2%2529%2520Directly%2520Activates%2520the%2520JAK1-STAT3%2520Axis%2520to%2520Induce%2520Hepatic%2520Fibrosis%2520in%2520Coordination%2520with%2520the%2520SMAD%2520Pathwayis%26jtitle%3DNat.%2520Commun.%26date%3D2017%26volume%3D8%26spage%3D1130%26doi%3D10.1038%2Fs41467-017-01236-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article has not yet been cited by other publications.</p></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01026/20210121/images/medium/jm0c01026_0017.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01026/20210121/images/large/jm0c01026_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01026&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01026/20210121/images/medium/jm0c01026_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01026/20210121/images/large/jm0c01026_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Representative JAK inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01026/20210121/images/large/jm0c01026_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01026&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01026/20210121/images/medium/jm0c01026_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01026/20210121/images/large/jm0c01026_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (a) Alignment of ATP-binding site of tofacitinib (cyan)–JAK1 (gray) complex (3EYG) with filgotinib (purple)–JAK2 (light pink) complex (3FUP), highlighting different residues. (b) Schematic alignment of docking result of filgotinib with JAK1 and overlay with tofacitinib. (c) Three chosen areas for modification on the suggested new molecule.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01026/20210121/images/large/jm0c01026_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01026&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01026/20210121/images/medium/jm0c01026_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01026/20210121/images/large/jm0c01026_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Syntheses of Intermediates <b>7a–7d</b>, <b>13a–13g</b>, <b>14a–14e</b>, <b>15a–15e</b>, and <b>16a–16c</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01026/20210121/images/large/jm0c01026_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01026&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01026/20210121/images/medium/jm0c01026_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01026/20210121/images/large/jm0c01026_0008.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Syntheses of Products <b>17a–17m</b>, <b>18a–18e</b>, <b>19a–19e</b>, and <b>20a–20c</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01026/20210121/images/large/jm0c01026_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01026&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01026/20210121/images/medium/jm0c01026_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01026/20210121/images/large/jm0c01026_0009.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Syntheses of Products <b>20d</b>, <b>20e</b>, <b>30a–30f</b>, <b>31a–31l</b>, and <b>32a–32c</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01026/20210121/images/large/jm0c01026_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01026&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01026/20210121/images/medium/jm0c01026_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01026/20210121/images/large/jm0c01026_0010.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Enantioselective Syntheses of 4-Amino-2-methylpiperidine Intermediates <b>36a</b> and <b>36b</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01026/20210121/images/large/jm0c01026_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01026&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01026/20210121/images/medium/jm0c01026_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01026/20210121/images/large/jm0c01026_0011.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Syntheses of <b>31g</b> Enantiomers <b>38a</b> and <b>38b</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01026/20210121/images/large/jm0c01026_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01026&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01026/20210121/images/medium/jm0c01026_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01026/20210121/images/large/jm0c01026_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. (a) Docking pose generated induced fit showing <b>38a</b> (orange) in complex with JAK1 (gray, 3EYG). (b) Overlay of tofacitinib and <b>38a</b> binding poses with JAK1. (c) Surface view of the methyl binding site on the middle lipophilic binding pocket.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01026/20210121/images/large/jm0c01026_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01026&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01026/20210121/images/medium/jm0c01026_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01026/20210121/images/large/jm0c01026_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (a) Proliferation of LX2 induced by TGF-β was investigated with CCK8. The inhibition effects of nintedanib and <b>31g</b> on TGF-β induced proliferation of LX-2 were observed in a time-dependent manner. (b) Under the same conditions, growth inhibition of nintedanib and <b>31g</b> was observed in a dose-dependent manner to measure GI<sub>50</sub>. (c) Cell impermeable YOYO1 was used to assess real-time cytotoxicity. YOYO1 fluorescence images were acquired by increase treatment with nintedanib in LX-2 cells compared to <b>31g</b>. (d) YOYO1 images were analyzed by an Incucyte imaging FLR in a time-dependent manner. The IC<sub>50</sub> value was calculated through the Imaging FLR.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01026/20210121/images/large/jm0c01026_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01026&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01026/20210121/images/medium/jm0c01026_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01026/20210121/images/large/jm0c01026_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (a) Gene expression levels of fibrotic parameters measured in LX-2 incubation with TGF-β treated without or with <b>31g</b> (500 nM). (b) left, <b>31g</b> (250 nM) ameliorated TGF-β-induced α-SMA stress fiber formation. Immunofluorescence analysis of α-SMA (green) and Hoechst 33258 (blue) in LX-2 cells treated with 10 ng/mL TGF-β for 72 h. Scale bar 200 μm. right, Fluorescence intensity was analyzed by phenotypic percentage of the population with constant GFP intensity value using Harmony 3.1. (c) Evaluation of the wound healing/migration inhibition effect of <b>31g</b> based on an <i>in vitro</i> scratch wound assay using LX-2 cells. Scratch wound healing/migration induced by TGF-β in LX-2 cells treated with media alone (0.5% FBS) and indicated concentrations of <b>31g</b>. Red line: initial 0 h, purple line: after 24 h. (d) <b>31g</b> inhibits the JAK-STAT signal, but not nintedanib (classical inhibitor of hepatic fibrosis). Immunobit cell-based immunoassays of pSTAT3 and pSTAT5 were analyzed in TF1 cells with nintedanib and <b>31g</b>. (e) TGF-β-induced STAT3 phosphorylation in LX-2 levels reduced with <b>31g</b> (250 nM) treatment. Band intensity was analyzed by Image studio with ODYSSEY from LI-COR.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01026/20210121/images/large/jm0c01026_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01026&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i130">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_08424" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_08424" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 42 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kanno, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villarino, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gadina, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Shea, J. J.</span></span> <span> </span><span class="NLM_article-title">JAK inhibition as a therapeutic strategy for immune and inflammatory diseases</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">843</span>– <span class="NLM_lpage">862</span>, <span class="refDoi"> DOI: 10.1038/nrd.2017.201</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=10.1038%2Fnrd.2017.201" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=29104284" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslGqsLnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=843-862&author=D.+M.+Schwartzauthor=Y.+Kannoauthor=A.+Villarinoauthor=M.+Wardauthor=M.+Gadinaauthor=J.+J.+O%E2%80%99Shea&title=JAK+inhibition+as+a+therapeutic+strategy+for+immune+and+inflammatory+diseases&doi=10.1038%2Fnrd.2017.201"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">JAK inhibition as a therapeutic strategy for immune and inflammatory diseases</span></div><div class="casAuthors">Schwartz, Daniella M.; Kanno, Yuka; Villarino, Alejandro; Ward, Michael; Gadina, Massimo; O'Shea, John J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">843-862</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  The discovery of cytokines as key drivers of immune-mediated diseases has spurred efforts to target their assocd. signalling pathways.  Janus kinases (JAKs) are essential signalling mediators downstream of many pro-inflammatory cytokines, and small-mol. inhibitors of JAKs (jakinibs) have gained traction as safe and efficacious options for the treatment of inflammation-driven pathologies such as rheumatoid arthritis, psoriasis and inflammatory bowel disease.  Building on the clin. success of first-generation jakinibs, second-generation compds. that claim to be more selective are currently undergoing development and proceeding to clin. trials.  However, important questions remain about the advantages and limitations of improved JAK selectivity, optimal routes and dosing regimens and how best to identify patients who will benefit from jakinibs.  This Review discusses the biol. of jakinibs from a translational perspective, focusing on recent insights from clin. trials, the development of novel agents and the use of jakinibs in a spectrum of immune and inflammatory diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9TNpGT01l8rVg90H21EOLACvtfcHk0li4ID4U_61-nA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslGqsLnO&md5=4607422f83f94c0c7194935d6d460c31</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2017.201&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2017.201%26sid%3Dliteratum%253Aachs%26aulast%3DSchwartz%26aufirst%3DD.%2BM.%26aulast%3DKanno%26aufirst%3DY.%26aulast%3DVillarino%26aufirst%3DA.%26aulast%3DWard%26aufirst%3DM.%26aulast%3DGadina%26aufirst%3DM.%26aulast%3DO%25E2%2580%2599Shea%26aufirst%3DJ.%2BJ.%26atitle%3DJAK%2520inhibition%2520as%2520a%2520therapeutic%2520strategy%2520for%2520immune%2520and%2520inflammatory%2520diseases%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26spage%3D843%26epage%3D862%26doi%3D10.1038%2Fnrd.2017.201" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">O’Shea, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villarino, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gadina, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McInnes, I. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laurence, A.</span></span> <span> </span><span class="NLM_article-title">The JAK-STAT pathway: impact on human disease and therapeutic intervention</span>. <i>Annu. Rev. Med.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">311</span>– <span class="NLM_lpage">328</span>, <span class="refDoi"> DOI: 10.1146/annurev-med-051113-024537</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=10.1146%2Fannurev-med-051113-024537" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=25587654" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjvFyhsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2015&pages=311-328&author=J.+J.+O%E2%80%99Sheaauthor=D.+M.+Schwartzauthor=A.+V.+Villarinoauthor=M.+Gadinaauthor=I.+B.+McInnesauthor=A.+Laurence&title=The+JAK-STAT+pathway%3A+impact+on+human+disease+and+therapeutic+intervention&doi=10.1146%2Fannurev-med-051113-024537"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">The JAK-STAT pathway: impact on human disease and therapeutic intervention</span></div><div class="casAuthors">O'Shea, John J.; Schwartz, Daniella M.; Villarino, Alejandro V.; Gadina, Massimo; McInnes, Iain B.; Laurence, Arian</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Medicine</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">311-328</span>CODEN:
                <span class="NLM_cas:coden">ARMCAH</span>;
        ISSN:<span class="NLM_cas:issn">0066-4219</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">A review.  The Janus kinase (JAK)-signal transducer of activators of transcription (STAT) pathway is now recognized as an evolutionarily conserved signaling pathway employed by diverse cytokines, interferons, growth factors, and related mols.  This pathway provides an elegant and remarkably straightforward mechanism whereby extracellular factors control gene expression.  It thus serves as a fundamental paradigm for how cells sense environmental cues and interpret these signals to regulate cell growth and differentiation.  Genetic mutations and polymorphisms are functionally relevant to a variety of human diseases, esp. cancer and immune-related conditions.  The clin. relevance of the pathway has been confirmed by the emergence of a new class of therapeutics that targets JAKs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMxvQCRiTbL7Vg90H21EOLACvtfcHk0li4ID4U_61-nA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjvFyhsLY%253D&md5=19182154f1a046df039eb4a37b66bf78</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1146%2Fannurev-med-051113-024537&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-med-051113-024537%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Shea%26aufirst%3DJ.%2BJ.%26aulast%3DSchwartz%26aufirst%3DD.%2BM.%26aulast%3DVillarino%26aufirst%3DA.%2BV.%26aulast%3DGadina%26aufirst%3DM.%26aulast%3DMcInnes%26aufirst%3DI.%2BB.%26aulast%3DLaurence%26aufirst%3DA.%26atitle%3DThe%2520JAK-STAT%2520pathway%253A%2520impact%2520on%2520human%2520disease%2520and%2520therapeutic%2520intervention%26jtitle%3DAnnu.%2520Rev.%2520Med.%26date%3D2015%26volume%3D66%26spage%3D311%26epage%3D328%26doi%3D10.1146%2Fannurev-med-051113-024537" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yamaoka, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saharinen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pesu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holt, V. E. T.,  III</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silvennoinen, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Shea, J. J.</span></span> <span> </span><span class="NLM_article-title">The janus kinases (jaks)</span>. <i>Genome Biol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">253</span>, <span class="refDoi"> DOI: 10.1186/gb-2004-5-12-253</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=10.1186%2Fgb-2004-5-12-253" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=15575979" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A280%3ADC%252BD2cngtlKktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2004&pages=253&author=K.+Yamaokaauthor=P.+Saharinenauthor=M.+Pesuauthor=V.+E.+T.+Holtauthor=O.+Silvennoinenauthor=J.+J.+O%E2%80%99Shea&title=The+janus+kinases+%28jaks%29&doi=10.1186%2Fgb-2004-5-12-253"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The Janus kinases (Jaks)</span></div><div class="casAuthors">Yamaoka Kunihiro; Saharinen Pipsa; Pesu Marko; Holt Vance E T 3rd; Silvennoinen Olli; O'Shea John J</div><div class="citationInfo"><span class="NLM_cas:title">Genome biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">253</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The Janus kinase (Jak) family is one of ten recognized families of non-receptor tyrosine kinases.  Mammals have four members of this family, Jak1, Jak2, Jak3 and Tyrosine kinase 2 (Tyk2).  Birds, fish and insects also have Jaks.  Each protein has a kinase domain and a catalytically inactive pseudo-kinase domain, and they each bind cytokine receptors through amino-terminal FERM (Band-4.1, ezrin, radixin, moesin) domains.  Upon binding of cytokines to their receptors, Jaks are activated and phosphorylate the receptors, creating docking sites for signaling molecules, especially members of the signal transducer and activator of transcription (Stat) family.  Mutations of the Drosophila Jak (Hopscotch) have revealed developmental defects, and constitutive activation of Jaks in flies and humans is associated with leukemia-like syndromes.  Through the generation of Jak-deficient cell lines and gene-targeted mice, the essential, nonredundant functions of Jaks in cytokine signaling have been established.  Importantly, deficiency of Jak3 is the basis of human autosomal recessive severe combined immunodeficiency (SCID); accordingly, a selective Jak3 inhibitor has been developed, forming a new class of immunosuppressive drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS0BwDh7EYTsV1C8OJBvdfbfW6udTcc2eY0jhdDZNafRrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2cngtlKktg%253D%253D&md5=4f8335c37dc8c20245e771224c1d5b12</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1186%2Fgb-2004-5-12-253&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fgb-2004-5-12-253%26sid%3Dliteratum%253Aachs%26aulast%3DYamaoka%26aufirst%3DK.%26aulast%3DSaharinen%26aufirst%3DP.%26aulast%3DPesu%26aufirst%3DM.%26aulast%3DHolt%26aufirst%3DV.%2BE.%2BT.%26aulast%3DSilvennoinen%26aufirst%3DO.%26aulast%3DO%25E2%2580%2599Shea%26aufirst%3DJ.%2BJ.%26atitle%3DThe%2520janus%2520kinases%2520%2528jaks%2529%26jtitle%3DGenome%2520Biol.%26date%3D2004%26volume%3D5%26spage%3D253%26doi%3D10.1186%2Fgb-2004-5-12-253" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Banerjee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biehl, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gadina, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasni, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, D. M.</span></span> <span> </span><span class="NLM_article-title">JAK–STAT signaling as a target for inflammatory and autoimmune diseases: current and future prospects</span>. <i>Drugs</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>77</i></span>,  <span class="NLM_fpage">521</span>– <span class="NLM_lpage">546</span>, <span class="refDoi"> DOI: 10.1007/s40265-017-0701-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=10.1007%2Fs40265-017-0701-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=28255960" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjs12qsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2017&pages=521-546&author=S.+Banerjeeauthor=A.+Biehlauthor=M.+Gadinaauthor=S.+Hasniauthor=D.+M.+Schwartz&title=JAK%E2%80%93STAT+signaling+as+a+target+for+inflammatory+and+autoimmune+diseases%3A+current+and+future+prospects&doi=10.1007%2Fs40265-017-0701-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">JAK-STAT Signaling as a Target for Inflammatory and Autoimmune Diseases: Current and Future Prospects</span></div><div class="casAuthors">Banerjee, Shubhasree; Biehl, Ann; Gadina, Massimo; Hasni, Sarfaraz; Schwartz, Daniella M.</div><div class="citationInfo"><span class="NLM_cas:title">Drugs</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">77</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">521-546</span>CODEN:
                <span class="NLM_cas:coden">DRUGAY</span>;
        ISSN:<span class="NLM_cas:issn">0012-6667</span>.
    
            (<span class="NLM_cas:orgname">Springer International Publishing AG</span>)
        </div><div class="casAbstract">The Janus kinase/signal transduction and activator of transcription (JAK-STAT) signaling pathway is implicated in the pathogenesis of inflammatory and autoimmune diseases including rheumatoid arthritis, psoriasis, and inflammatory bowel disease.  Many cytokines involved in the pathogenesis of autoimmune and inflammatory diseases use JAKs and STATs to transduce intracellular signals.  Mutations in JAK and STAT genes cause a no. of immunodeficiency syndromes, and polymorphisms in these genes are assocd. with autoimmune diseases.  The success of small-mol. JAK inhibitors (Jakinibs) in the treatment of rheumatol. disease demonstrates that intracellular signaling pathways can be targeted therapeutically to treat autoimmunity.  Tofacitinib, the first rheumatol. Jakinib, is US Food and Drug Administration (FDA) approved for rheumatoid arthritis and is currently under investigation for other autoimmune diseases.  Many other Jakinibs are in preclin. development or in various phases of clin. trials.  This review describes the JAK-STAT pathway, outlines its role in autoimmunity, and explains the rationale/pre-clin. evidence for targeting JAK-STAT signaling.  The safety and clin. efficacy of the Jakinibs are reviewed, starting with the FDA-approved Jakinib tofacitinib, and continuing on to next-generation Jakinibs.  Recent and ongoing studies are emphasized, with a focus on emerging indications for JAK inhibition and novel mechanisms of JAK-STAT signaling blockade.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGXi4XmOlzP7Vg90H21EOLACvtfcHk0ljtmVFPkR7qEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjs12qsbg%253D&md5=9d06038330083ff3d7f4516578987524</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1007%2Fs40265-017-0701-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs40265-017-0701-9%26sid%3Dliteratum%253Aachs%26aulast%3DBanerjee%26aufirst%3DS.%26aulast%3DBiehl%26aufirst%3DA.%26aulast%3DGadina%26aufirst%3DM.%26aulast%3DHasni%26aufirst%3DS.%26aulast%3DSchwartz%26aufirst%3DD.%2BM.%26atitle%3DJAK%25E2%2580%2593STAT%2520signaling%2520as%2520a%2520target%2520for%2520inflammatory%2520and%2520autoimmune%2520diseases%253A%2520current%2520and%2520future%2520prospects%26jtitle%3DDrugs%26date%3D2017%26volume%3D77%26spage%3D521%26epage%3D546%26doi%3D10.1007%2Fs40265-017-0701-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Danese, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grisham, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hodge, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Telliez, J.-B.</span></span> <span> </span><span class="NLM_article-title">JAK inhibition using tofacitinib for inflammatory bowel disease treatment: a hub for multiple inflammatory cytokines</span>. <i>Am. J. Physiol. Gastrointest. Liver Physiol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>310</i></span>,  <span class="NLM_fpage">G155</span>– <span class="NLM_lpage">G162</span>, <span class="refDoi"> DOI: 10.1152/ajpgi.00311.2015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=10.1152%2Fajpgi.00311.2015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=26608188" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=310&publication_year=2016&pages=G155-G162&author=S.+Daneseauthor=M.+Grishamauthor=J.+Hodgeauthor=J.-B.+Telliez&title=JAK+inhibition+using+tofacitinib+for+inflammatory+bowel+disease+treatment%3A+a+hub+for+multiple+inflammatory+cytokines&doi=10.1152%2Fajpgi.00311.2015"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1152%2Fajpgi.00311.2015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1152%252Fajpgi.00311.2015%26sid%3Dliteratum%253Aachs%26aulast%3DDanese%26aufirst%3DS.%26aulast%3DGrisham%26aufirst%3DM.%26aulast%3DHodge%26aufirst%3DJ.%26aulast%3DTelliez%26aufirst%3DJ.-B.%26atitle%3DJAK%2520inhibition%2520using%2520tofacitinib%2520for%2520inflammatory%2520bowel%2520disease%2520treatment%253A%2520a%2520hub%2520for%2520multiple%2520inflammatory%2520cytokines%26jtitle%3DAm.%2520J.%2520Physiol.%2520Gastrointest.%2520Liver%2520Physiol.%26date%3D2016%26volume%3D310%26spage%3DG155%26epage%3DG162%26doi%3D10.1152%2Fajpgi.00311.2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Spergel, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mertzman, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kempson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stachura, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haque, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lippy, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galella, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pitt, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fura, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gillooly, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McIntyre, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tokarski, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sack, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, P. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrish, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nadler, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salter-Cid, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schieven, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wrobleski, S. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pitts, W. J.</span></span> <span> </span><span class="NLM_article-title">Discovery of a JAK1/3 inhibitor and use of a prodrug to demonstrate efficacy in a model of rheumatoid arthritis</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">306</span>– <span class="NLM_lpage">311</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.8b00508</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.8b00508" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjtVSgtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=306-311&author=S.+H.+Spergelauthor=M.+E.+Mertzmanauthor=J.+Kempsonauthor=J.+Guoauthor=S.+Stachuraauthor=L.+Haqueauthor=J.+S.+Lippyauthor=R.+F.+Zhangauthor=M.+Galellaauthor=S.+Pittauthor=G.+Shenauthor=A.+Furaauthor=K.+Gilloolyauthor=K.+W.+McIntyreauthor=V.+Tangauthor=J.+Tokarskiauthor=J.+S.+Sackauthor=J.+Khanauthor=P.+H.+Carterauthor=J.+C.+Barrishauthor=S.+G.+Nadlerauthor=L.+M.+Salter-Cidauthor=G.+L.+Schievenauthor=S.+T.+Wrobleskiauthor=W.+J.+Pitts&title=Discovery+of+a+JAK1%2F3+inhibitor+and+use+of+a+prodrug+to+demonstrate+efficacy+in+a+model+of+rheumatoid+arthritis&doi=10.1021%2Facsmedchemlett.8b00508"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a JAK1/3 Inhibitor and Use of a Prodrug To Demonstrate Efficacy in a Model of Rheumatoid Arthritis</span></div><div class="casAuthors">Spergel, Steven H.; Mertzman, Michael E.; Kempson, James; Guo, Junqing; Stachura, Sylwia; Haque, Lauren; Lippy, Jonathan S.; Zhang, Rosemary F.; Galella, Michael; Pitt, Sidney; Shen, Guoxiang; Fura, Aberra; Gillooly, Kathleen; McIntyre, Kim W.; Tang, Vicky; Tokarski, John; Sack, John S.; Khan, Javed; Carter, Percy H.; Barrish, Joel C.; Nadler, Steven G.; Salter-Cid, Luisa M.; Schieven, Gary L.; Wrobleski, Stephen T.; Pitts, William J.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">306-311</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The four members of the Janus family of nonreceptor tyrosine kinases play a significant role in immune function.  The JAK family kinase inhibitor, tofacitinib 1, has been approved in the United States for use in rheumatoid arthritis (RA) patients.  A no. of JAK inhibitors with a variety of JAK family selectivity profiles are currently in clin. trials.  Our goal was to identify inhibitors that were functionally selective for JAK1 and JAK3.  Compd. 22 was prepd. with the desired functional selectivity profile, but it suffered from poor absorption related to phys. properties.  Use of the phosphate prodrug 32 enabled progression to a murine collagen induced arthritis (CIA) model.  The demonstration of a robust efficacy in the CIA model suggests that use of phosphate prodrugs may resolve issues with progressing this chemotype for the treatment of autoimmune diseases such as RA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQKLujiDlteLVg90H21EOLACvtfcHk0ljtmVFPkR7qEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjtVSgtr4%253D&md5=7562490a531e325dc7404dc8e64d090b</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.8b00508&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.8b00508%26sid%3Dliteratum%253Aachs%26aulast%3DSpergel%26aufirst%3DS.%2BH.%26aulast%3DMertzman%26aufirst%3DM.%2BE.%26aulast%3DKempson%26aufirst%3DJ.%26aulast%3DGuo%26aufirst%3DJ.%26aulast%3DStachura%26aufirst%3DS.%26aulast%3DHaque%26aufirst%3DL.%26aulast%3DLippy%26aufirst%3DJ.%2BS.%26aulast%3DZhang%26aufirst%3DR.%2BF.%26aulast%3DGalella%26aufirst%3DM.%26aulast%3DPitt%26aufirst%3DS.%26aulast%3DShen%26aufirst%3DG.%26aulast%3DFura%26aufirst%3DA.%26aulast%3DGillooly%26aufirst%3DK.%26aulast%3DMcIntyre%26aufirst%3DK.%2BW.%26aulast%3DTang%26aufirst%3DV.%26aulast%3DTokarski%26aufirst%3DJ.%26aulast%3DSack%26aufirst%3DJ.%2BS.%26aulast%3DKhan%26aufirst%3DJ.%26aulast%3DCarter%26aufirst%3DP.%2BH.%26aulast%3DBarrish%26aufirst%3DJ.%2BC.%26aulast%3DNadler%26aufirst%3DS.%2BG.%26aulast%3DSalter-Cid%26aufirst%3DL.%2BM.%26aulast%3DSchieven%26aufirst%3DG.%2BL.%26aulast%3DWrobleski%26aufirst%3DS.%2BT.%26aulast%3DPitts%26aufirst%3DW.%2BJ.%26atitle%3DDiscovery%2520of%2520a%2520JAK1%252F3%2520inhibitor%2520and%2520use%2520of%2520a%2520prodrug%2520to%2520demonstrate%2520efficacy%2520in%2520a%2520model%2520of%2520rheumatoid%2520arthritis%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2019%26volume%3D10%26spage%3D306%26epage%3D311%26doi%3D10.1021%2Facsmedchemlett.8b00508" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Villarino, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kanno, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferdinand, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Shea, J. J.</span></span> <span> </span><span class="NLM_article-title">Mechanisms of JAK/STAT signaling in immunity and disease</span>. <i>J. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>194</i></span>,  <span class="NLM_fpage">21</span>– <span class="NLM_lpage">27</span>, <span class="refDoi"> DOI: 10.4049/jimmunol.1401867</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=10.4049%2Fjimmunol.1401867" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=25527793" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFWls7rF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=194&publication_year=2015&pages=21-27&author=A.+V.+Villarinoauthor=Y.+Kannoauthor=J.+R.+Ferdinandauthor=J.+J.+O%E2%80%99Shea&title=Mechanisms+of+JAK%2FSTAT+signaling+in+immunity+and+disease&doi=10.4049%2Fjimmunol.1401867"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanisms of Jak/STAT Signaling in Immunity and Disease</span></div><div class="casAuthors">Villarino, Alejandro V.; Kanno, Yuka; Ferdinand, John R.; O'Shea, John J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Immunology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">194</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">21-27</span>CODEN:
                <span class="NLM_cas:coden">JOIMA3</span>;
        ISSN:<span class="NLM_cas:issn">0022-1767</span>.
    
            (<span class="NLM_cas:orgname">American Association of Immunologists</span>)
        </div><div class="casAbstract">A review.  More than two decades ago, expts. on the antiviral mechanisms of IFNs led to the discovery of JAKs and their downstream effectors, the STAT proteins.  This pathway has since become a paradigm for membrane-to-nucleus signaling and explains how a broad range of sol. factors, including cytokines and hormones, mediate their diverse functions.  Jak/STAT research has not only impacted basic science, particularly in the context of intercellular communication and cell-extrinsic control of gene expression, it also has become a prototype for transition from bench to bedside, culminating in the development and clin. implementation of pathway-specific therapeutics.  This brief review synthesizes our current understanding of Jak/STAT biol. while taking stock of the lessons learned and the challenges that lie ahead.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotOsFFpF-ChLVg90H21EOLACvtfcHk0limctzGL2_umg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitFWls7rF&md5=777f0f64db038ca8a7f17554931e0655</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.4049%2Fjimmunol.1401867&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4049%252Fjimmunol.1401867%26sid%3Dliteratum%253Aachs%26aulast%3DVillarino%26aufirst%3DA.%2BV.%26aulast%3DKanno%26aufirst%3DY.%26aulast%3DFerdinand%26aufirst%3DJ.%2BR.%26aulast%3DO%25E2%2580%2599Shea%26aufirst%3DJ.%2BJ.%26atitle%3DMechanisms%2520of%2520JAK%252FSTAT%2520signaling%2520in%2520immunity%2520and%2520disease%26jtitle%3DJ.%2520Immunol.%26date%3D2015%26volume%3D194%26spage%3D21%26epage%3D27%26doi%3D10.4049%2Fjimmunol.1401867" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, X.</span></span> <span> </span><span class="NLM_article-title">Transforming growth factor β1 promotes migration and invasion in HepG2 cells: Epithelial-to-mesenchymal transition via JAK/STAT3 signaling</span>. <i>Int. J. Mol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">129</span>– <span class="NLM_lpage">136</span>, <span class="refDoi"> DOI: 10.3892/ijmm.2017.3228</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=10.3892%2Fijmm.2017.3228" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=29115395" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVyqtbfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2018&pages=129-136&author=X.+L.+Linauthor=M.+Liuauthor=Y.+Liuauthor=H.+Huauthor=Y.+Panauthor=W.+Zouauthor=X.+Fanauthor=X.+Hu&title=Transforming+growth+factor+%CE%B21+promotes+migration+and+invasion+in+HepG2+cells%3A+Epithelial-to-mesenchymal+transition+via+JAK%2FSTAT3+signaling&doi=10.3892%2Fijmm.2017.3228"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Transforming growth factor β1 promotes migration and invasion in HepG2 cells: epithelial-to-mesenchymal transition via JAK/STAT3 signaling</span></div><div class="casAuthors">Lin, Xiao-Long; Liu, Mihua; Liu, Yuanbo; Hu, Huijun; Pan, Yongquan; Zou, Weiwen; Fan, Xiaojuan; Hu, Xuemei</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Molecular Medicine</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">129-136</span>CODEN:
                <span class="NLM_cas:coden">IJMMFG</span>;
        ISSN:<span class="NLM_cas:issn">1791-244X</span>.
    
            (<span class="NLM_cas:orgname">Spandidos Publications Ltd.</span>)
        </div><div class="casAbstract">Transforming growth factor β1 (TGFβ1) is a cytokine with multiple functions.  TGFβ1 significantly induces migration and invasion of liver cancer cells.  However, the mol. mechanisms underlying this effect remain unclear.  Epithelial-to-mesenchymal transition (EMT) is crucial for the development of invasion and metastasis in human cancers.  The aim of the present study was to det. whether TGFβ1-induced EMT promoted migration and invasion in HepG2 cells.  The underlying mechanism and the effect of EMT on HepG2 cells were also investigated.  The results demonstrated that TGFβ1 may i nduce E MT t o p romote migration and invasion of HepG2 cells, and this effect depends on activation of the Janus kinase/signal transducer and activator of transcription 3 (JAK/STAT3) signaling pathway.  JAK/STAT3 signaling is involved in human malignancies, including lung cancer, and is implicated in cell transformation, tumorigenicity, EMT and metastasis.  In the present study, TGFβ1 also activated JAK/STAT3 signaling in HepG2 cells and promoted Twist expression, but these events were abolished by treatment with the STAT3 inhibitor AG490.  Addnl., Twist siRNA blocked TGFβ1-induced EMT.  Thus, TGFβ1 was shown to induce EMT, thereby promoting the migration and invasion of HepG2 cells via JAK/STAT3/Twist signalling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOVmKjh2cCjrVg90H21EOLACvtfcHk0lijx4n292oi_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVyqtbfK&md5=c65d44bd68d4cb4e608c7fd39093e92d</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.3892%2Fijmm.2017.3228&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3892%252Fijmm.2017.3228%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DX.%2BL.%26aulast%3DLiu%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DHu%26aufirst%3DH.%26aulast%3DPan%26aufirst%3DY.%26aulast%3DZou%26aufirst%3DW.%26aulast%3DFan%26aufirst%3DX.%26aulast%3DHu%26aufirst%3DX.%26atitle%3DTransforming%2520growth%2520factor%2520%25CE%25B21%2520promotes%2520migration%2520and%2520invasion%2520in%2520HepG2%2520cells%253A%2520Epithelial-to-mesenchymal%2520transition%2520via%2520JAK%252FSTAT3%2520signaling%26jtitle%3DInt.%2520J.%2520Mol.%2520Med.%26date%3D2018%26volume%3D41%26spage%3D129%26epage%3D136%26doi%3D10.3892%2Fijmm.2017.3228" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gu, Y.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, W.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, W.</span></span> <span> </span><span class="NLM_article-title">Targeted blockade of JAK/STAT3 signaling inhibits proliferation, migration and collagen production as well as inducing the apoptosis of hepatic stellate cells</span>. <i>Int. J. Mol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">903</span>– <span class="NLM_lpage">911</span>, <span class="refDoi"> DOI: 10.3892/ijmm.2016.2692</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=10.3892%2Fijmm.2016.2692" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=27460897" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFOqurvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2016&pages=903-911&author=Y.-J.+Guauthor=W.-Y.+Sunauthor=S.+Zhangauthor=X.+R.+Liauthor=W.+Wei&title=Targeted+blockade+of+JAK%2FSTAT3+signaling+inhibits+proliferation%2C+migration+and+collagen+production+as+well+as+inducing+the+apoptosis+of+hepatic+stellate+cells&doi=10.3892%2Fijmm.2016.2692"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted blockade of JAK/STAT3 signaling inhibits proliferation, migration and collagen production as well as inducing the apoptosis of hepatic stellate cells</span></div><div class="casAuthors">Gu, Yuan-Jing; Sun, Wu-Yi; Zhang, Sen; Li, Xin-Ran; Wei, Wei</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Molecular Medicine</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">903-911</span>CODEN:
                <span class="NLM_cas:coden">IJMMFG</span>;
        ISSN:<span class="NLM_cas:issn">1791-244X</span>.
    
            (<span class="NLM_cas:orgname">Spandidos Publications Ltd.</span>)
        </div><div class="casAbstract">Protein tyrosine kinases belonging to the Janus kinase (JAK) family are assocd. with many cytokine receptors, which, on ligand binding, regulate important cellular functions such as proliferation, apoptosis and differentiation.  The protective effects of JAK inhibitors on fibrotic diseases such as myelofibrosis and bone marrow fibrosis have been demonstrated in previous studies.  The JAK inhibitor SHR0302 is a synthetic mol. that potently inhibits all members of the JAK family, particularly JAK1.  However, its effect on hepatic fibrosis has not been investigated to date, to the best of our knowledge.  In the present study, the effects of SHR0302 on the activation, proliferation, migration and apoptosis of hepatic stellate cells (HSCs) as well as HSC collagen prodn. were investigated.  Our data demonstrated that treatment with SHR0302 (10-9-10-5 mol/l) exerted an inhibitory effect on the activation, proliferation and migration of HSCs.  In addn., the expression of collagen I and collagen III were significantly decreased following treatment with SHR0302.  Furthermore, SHR0302 induced the apoptosis of HSCs, which was demonstrated by Annexin V/PI staining.  SHR0302 significantly increased the activation of caspase-3 and Bax in HSCs whereas it decreased the expression of Bcl-2.  SHR0302 also inhibited the activation of Akt signaling pathway.  The pharmacol. inhibition of the JAK1/signal transducer and activator of transcription (STAT)3 pathway led to the disruption of functions essential for HSC growth.  Taken together, these findings provide evidence that SHR0302 may have the potential to alleviate hepatic fibrosis by targeting HSC functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrU0x8DPYNMILVg90H21EOLACvtfcHk0lijx4n292oi_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFOqurvP&md5=4fd8d79529c979366f4593e3d8205c9e</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.3892%2Fijmm.2016.2692&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3892%252Fijmm.2016.2692%26sid%3Dliteratum%253Aachs%26aulast%3DGu%26aufirst%3DY.-J.%26aulast%3DSun%26aufirst%3DW.-Y.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DX.%2BR.%26aulast%3DWei%26aufirst%3DW.%26atitle%3DTargeted%2520blockade%2520of%2520JAK%252FSTAT3%2520signaling%2520inhibits%2520proliferation%252C%2520migration%2520and%2520collagen%2520production%2520as%2520well%2520as%2520inducing%2520the%2520apoptosis%2520of%2520hepatic%2520stellate%2520cells%26jtitle%3DInt.%2520J.%2520Mol.%2520Med.%26date%3D2016%26volume%3D38%26spage%3D903%26epage%3D911%26doi%3D10.3892%2Fijmm.2016.2692" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nadalin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koenigsrainer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dooley, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">ten Dijke, P.</span></span> <span> </span><span class="NLM_article-title">Transforming growth factor-β (TGF-β)-mediated connective tissue growth factor (CTGF) expression in hepatic stellate cells requires Stat3 signaling Activity</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>288</i></span>,  <span class="NLM_fpage">30708</span>– <span class="NLM_lpage">30719</span>, <span class="refDoi"> DOI: 10.1074/jbc.M113.478685</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=10.1074%2Fjbc.M113.478685" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=24005672" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1Cru7%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=288&publication_year=2013&pages=30708-30719&author=Y.+Liuauthor=H.+Liuauthor=C.+Meyerauthor=J.+Liauthor=S.+Nadalinauthor=A.+Koenigsrainerauthor=H.+Wengauthor=S.+Dooleyauthor=P.+ten+Dijke&title=Transforming+growth+factor-%CE%B2+%28TGF-%CE%B2%29-mediated+connective+tissue+growth+factor+%28CTGF%29+expression+in+hepatic+stellate+cells+requires+Stat3+signaling+Activity&doi=10.1074%2Fjbc.M113.478685"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Transforming Growth Factor-β (TGF-β)-mediated Connective Tissue Growth Factor (CTGF) Expression in Hepatic Stellate Cells Requires Stat3 Signaling Activation</span></div><div class="casAuthors">Liu, Yan; Liu, Heng; Meyer, Christoph; Li, Jun; Nadalin, Silvio; Koenigsrainer, Alfred; Weng, Honglei; Dooley, Steven; ten Dijke, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">288</span>
        (<span class="NLM_cas:issue">42</span>),
    <span class="NLM_cas:pages">30708-30719</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">In fibrotic liver, connective tissue growth factor (CTGF) is constantly expressed in activated hepatic stellate cells (HSCs) and acts downstream of TGF-β to modulate extracellular matrix prodn.  Distinct from other cell types in which Smad signaling plays major role in regulating CTGF prodn., TGF-β stimulated CTGF expression in activated HSCs is only in part dependent on Smad3.  Other signaling mols. like MAPKs and PI3Ks may also participate in this process, and the underlying mechanisms have yet to be clarified.  In this study, we report involvement of Stat3 activation in modulating CTGF prodn. upon TGF-β challenge in activated HSCs.  Stat3 is phosphorylated via JAK1 and acts as a crit. ALK5 (activin receptor-like kinase 5) downstream signaling mol. to mediate CTGF expression.  This process requires de novo gene transcription and is addnl. modulated by MEK1/2, JNK, and PI3K pathways.  Cell-specific knockdown of Smad3 partially decreases CTGF prodn., whereas it has no significant influence on Stat3 activation.  The total CTGF prodn. induced by TGF-β in activated HSCs is therefore, to a large extent, dependent on the balance and integration of the canonical Smad3 and Stat3 signaling pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8flotEXW6yLVg90H21EOLACvtfcHk0lijx4n292oi_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1Cru7%252FJ&md5=c6d61d93df1ad6fbd5ab67a4ac801d09</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M113.478685&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M113.478685%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DMeyer%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DNadalin%26aufirst%3DS.%26aulast%3DKoenigsrainer%26aufirst%3DA.%26aulast%3DWeng%26aufirst%3DH.%26aulast%3DDooley%26aufirst%3DS.%26aulast%3Dten%2BDijke%26aufirst%3DP.%26atitle%3DTransforming%2520growth%2520factor-%25CE%25B2%2520%2528TGF-%25CE%25B2%2529-mediated%2520connective%2520tissue%2520growth%2520factor%2520%2528CTGF%2529%2520expression%2520in%2520hepatic%2520stellate%2520cells%2520requires%2520Stat3%2520signaling%2520Activity%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2013%26volume%3D288%26spage%3D30708%26epage%3D30719%26doi%3D10.1074%2Fjbc.M113.478685" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tang, L.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heller, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, L.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y. E.</span></span> <span> </span><span class="NLM_article-title">Transforming Growth Factor-β (TGF-β) Directly Activates the JAK1-STAT3 Axis to Induce Hepatic Fibrosis in Coordination with the SMAD Pathway</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>292</i></span>,  <span class="NLM_fpage">4302</span>– <span class="NLM_lpage">4312</span>, <span class="refDoi"> DOI: 10.1074/jbc.M116.773085</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=10.1074%2Fjbc.M116.773085" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=28154170" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A528%3ADC%252BC2sXktFSltLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=292&publication_year=2017&pages=4302-4312&author=L.-Y.+Tangauthor=M.+Hellerauthor=Z.+Mengauthor=L.-R.+Yuauthor=Y.+Tangauthor=M.+Zhouauthor=Y.+E.+Zhang&title=Transforming+Growth+Factor-%CE%B2+%28TGF-%CE%B2%29+Directly+Activates+the+JAK1-STAT3+Axis+to+Induce+Hepatic+Fibrosis+in+Coordination+with+the+SMAD+Pathway&doi=10.1074%2Fjbc.M116.773085"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Transforming Growth Factor-β (TGF-β) Directly Activates the JAK1-STAT3 Axis to Induce Hepatic Fibrosis in Coordination with the SMAD Pathway</span></div><div class="casAuthors">Tang, Liu-Ya; Heller, Mary; Meng, Zhaojing; Yu, Li-Rong; Tang, Yi; Zhou, Ming; Zhang, Ying E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">292</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4302-4312</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Transforming growth factor-β (TGF-β) signals through both SMAD and non-SMAD pathways to elicit a wide array of biol. effects.  Existing data have shown the assocn. and coordination between STATs and SMADs in mediating TGF-β functions in hepatic cells, but it is not clear how STATs are activated under these circumstances.  Here, we report that JAK1 is a constitutive TGFβRI binding protein and is absolutely required for phosphorylation of STATs in a SMAD-independent manner within minutes of TGF-β stimulation.  Following the activation of SMADs, TGF-β also induces a second phase of STAT phosphorylation that requires SMADs, de novo protein synthesis, and contribution from JAK1.  Our global gene expression profiling indicates that the non-SMAD JAK1/STAT pathway is essential for the expression of a subset of TGF-β target genes in hepatic stellate cells, and the cooperation between the JAK1-STAT3 and SMAD pathways is crit. to the roles of TGF-β in liver fibrosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptJ4wllzrSp7Vg90H21EOLACvtfcHk0lhTMvaqzTj0xg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXktFSltLo%253D&md5=fefeed5ed048ba85372ea4fc30619e47</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M116.773085&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M116.773085%26sid%3Dliteratum%253Aachs%26aulast%3DTang%26aufirst%3DL.-Y.%26aulast%3DHeller%26aufirst%3DM.%26aulast%3DMeng%26aufirst%3DZ.%26aulast%3DYu%26aufirst%3DL.-R.%26aulast%3DTang%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DY.%2BE.%26atitle%3DTransforming%2520Growth%2520Factor-%25CE%25B2%2520%2528TGF-%25CE%25B2%2529%2520Directly%2520Activates%2520the%2520JAK1-STAT3%2520Axis%2520to%2520Induce%2520Hepatic%2520Fibrosis%2520in%2520Coordination%2520with%2520the%2520SMAD%2520Pathway%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2017%26volume%3D292%26spage%3D4302%26epage%3D4312%26doi%3D10.1074%2Fjbc.M116.773085" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nielsen, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clausen, M. H.</span></span> <span> </span><span class="NLM_article-title">FDA-approved small-molecule kinase inhibitors</span>. <i>Trends Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">422</span>– <span class="NLM_lpage">439</span>, <span class="refDoi"> DOI: 10.1016/j.tips.2015.04.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=10.1016%2Fj.tips.2015.04.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=25975227" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A528%3ADC%252BC2MXns1KisLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2015&pages=422-439&author=P.+Wuauthor=T.+E.+Nielsenauthor=M.+H.+Clausen&title=FDA-approved+small-molecule+kinase+inhibitors&doi=10.1016%2Fj.tips.2015.04.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">FDA-approved small-molecule kinase inhibitors</span></div><div class="casAuthors">Wu, Peng; Nielsen, Thomas E.; Clausen, Mads H.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">422-439</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Kinases have emerged as one of the most intensively pursued targets in current pharmacol. research, esp. for cancer, due to their crit. roles in cellular signaling.  To date, the US FDA has approved 28 small-mol. kinase inhibitors, half of which were approved in the past 3 years.  While the clin. data of these approved mols. are widely presented and structure-activity relationship (SAR) has been reported for individual mols., an updated review that analyzes all approved mols. and summarizes current achievements and trends in the field has yet to be found.  Here we present all approved small-mol. kinase inhibitors with an emphasis on binding mechanism and structural features, summarize current challenges, and discuss future directions in this field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrudyvM3eO09bVg90H21EOLACvtfcHk0lhTMvaqzTj0xg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXns1KisLg%253D&md5=a6aa3dbb11e6fe3001fa5fa03a4c950a</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2015.04.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2015.04.005%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DP.%26aulast%3DNielsen%26aufirst%3DT.%2BE.%26aulast%3DClausen%26aufirst%3DM.%2BH.%26atitle%3DFDA-approved%2520small-molecule%2520kinase%2520inhibitors%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2015%26volume%3D36%26spage%3D422%26epage%3D439%26doi%3D10.1016%2Fj.tips.2015.04.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Verden, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dimbil, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kyle, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overstreet, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, K. B.</span></span> <span> </span><span class="NLM_article-title">Analysis of spontaneous postmarket case reports submitted to the FDA regarding thromboembolic adverse events and JAK inhibitors</span>. <i>Drug Saf.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">357</span>– <span class="NLM_lpage">361</span>, <span class="refDoi"> DOI: 10.1007/s40264-017-0622-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=10.1007%2Fs40264-017-0622-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=29196988" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFektbnL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2018&pages=357-361&author=A.+Verdenauthor=M.+Dimbilauthor=R.+Kyleauthor=B.+Overstreetauthor=K.+B.+Hoffman&title=Analysis+of+spontaneous+postmarket+case+reports+submitted+to+the+FDA+regarding+thromboembolic+adverse+events+and+JAK+inhibitors&doi=10.1007%2Fs40264-017-0622-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Analysis of Spontaneous Postmarket Case Reports Submitted to the FDA Regarding Thromboembolic Adverse Events and JAK Inhibitors</span></div><div class="casAuthors">Verden, Abril; Dimbil, Mo; Kyle, Robert; Overstreet, Brian; Hoffman, Keith B.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Safety</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">357-361</span>CODEN:
                <span class="NLM_cas:coden">DRSAEA</span>;
        ISSN:<span class="NLM_cas:issn">0114-5916</span>.
    
            (<span class="NLM_cas:orgname">Springer International Publishing AG</span>)
        </div><div class="casAbstract">Introduction: The Janus kinase (JAK) inhibitor baricitinib is approved in Europe and Japan for the treatment of rheumatoid arthritis.  In Apr. 2017, the US FDA expressed concern about thromboembolic events (deep venous thrombosis [DVT] and pulmonary embolism [PE]) obsd. in placebo-controlled clin. trials of baricitinib.  The European and Japanese labels for baricitinib were recently updated to include a precaution related to potential thromboembolic events in patients at risk.  Given that the FDA-approved drugs tofacitinib and ruxolitinib are in the same class, we conducted a safety review of the FDA's Adverse Event Reporting System (FAERS) to assess postmarketing reporting rates for related thromboembolic risks.  Methods: Adverse event (AE) data for tofacitinib, tofacitinib extended-release (XR), and ruxolitinib were obtained from the FAERS.  Reporting odds ratios (RORs) and the R package 'PhViD' to est. the empirical Bayesian geometric mean (EBGM) were used to detect AEs with higher-than-expected reporting rates within the FAERS.  Results: We did not find evidence in the FAERS for elevated reporting rates for DVT and PE across the three JAK inhibitors we analyzed.  However, multiple drug-AE combinations relating to thromboembolic events had both RORs and EBGM values above 1, indicating a trend toward higher-than-expected reporting rates.  For pulmonary thrombosis, the ROR values for ruxolitinib, tofacitinib, and tofacitinib XR were 1.46 (95% confidence interval [CI] 0.76-2.80), 2.46 (1.55-3.91), and 2.48 (0.80-7.71), resp., while the EBGM values were 1.25 (0.70), 2.46 (1.64), and 1.56 (0.57), resp.  Ruxolitinib had ROR values of 4.08 (2.25-7.38) and 1.22 (0.97-1.53) for portal vein thrombosis and thrombosis, resp.  The EBGM values for the same drug-AE combinations were 3.04 (1.79) and 1.16 (0.96).  Conclusions: Our safety review of postmarketing FAERS reports assocd. with three FDA-approved JAK inhibitors did not find elevated reporting rates for DVT and PE specifically.  However, the FAERS data indicated that pulmonary thrombosis may potentially be a class-wide issue for JAK inhibitors.  Portal vein thrombosis may also be a potential risk for ruxolitinib.  While these FAERS data add to a growing body of evidence that JAK inhibitors may be contraindicated in patients at risk of thromboembolic events, the data need to be confirmed by future AE reporting trends, anal. of electronic health records, and/or future clin. trials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrB4-lwzE2yXLVg90H21EOLACvtfcHk0lhTMvaqzTj0xg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFektbnL&md5=c4679fffffff7f34b35b5ba34d88817b</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1007%2Fs40264-017-0622-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs40264-017-0622-2%26sid%3Dliteratum%253Aachs%26aulast%3DVerden%26aufirst%3DA.%26aulast%3DDimbil%26aufirst%3DM.%26aulast%3DKyle%26aufirst%3DR.%26aulast%3DOverstreet%26aufirst%3DB.%26aulast%3DHoffman%26aufirst%3DK.%2BB.%26atitle%3DAnalysis%2520of%2520spontaneous%2520postmarket%2520case%2520reports%2520submitted%2520to%2520the%2520FDA%2520regarding%2520thromboembolic%2520adverse%2520events%2520and%2520JAK%2520inhibitors%26jtitle%3DDrug%2520Saf.%26date%3D2018%26volume%3D41%26spage%3D357%26epage%3D361%26doi%3D10.1007%2Fs40264-017-0622-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, Z. C.</span></span> <span> </span><span class="NLM_article-title">Adverse drug events associated with 5mg versus 10mg tofacitinib (Janus kinase inhibitor) twice daily for the treatment of autoimmune diseases: A systematic review and meta-analysis of randomized controlled trials</span>. <i>Clin. Rheumatol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">523</span>– <span class="NLM_lpage">534</span>, <span class="refDoi"> DOI: 10.1007/s10067-018-4299-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=10.1007%2Fs10067-018-4299-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=30242639" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A280%3ADC%252BB3czhsFeruw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2019&pages=523-534&author=F.+Huangauthor=Z.+C.+Luo&title=Adverse+drug+events+associated+with+5mg+versus+10mg+tofacitinib+%28Janus+kinase+inhibitor%29+twice+daily+for+the+treatment+of+autoimmune+diseases%3A+A+systematic+review+and+meta-analysis+of+randomized+controlled+trials&doi=10.1007%2Fs10067-018-4299-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Adverse drug events associated with 5mg versus 10mg Tofacitinib (Janus kinase inhibitor) twice daily for the treatment of autoimmune diseases: A systematic review and meta-analysis of randomized controlled trials</span></div><div class="casAuthors">Huang Feng; Luo Zu-Chun</div><div class="citationInfo"><span class="NLM_cas:title">Clinical rheumatology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">523-534</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Several recently published clinical trials have shown tofacitinib to be effective in the treatment of autoimmune diseases.  This drug is commonly prescribed either in a 5-mg or in a10-mg dosage twice daily.  In this review, we aimed to systematically compare the adverse drug events which were observed with 5 mg versus 10 mg tofacitinib for the treatment of autoimmune diseases.  MEDLINE, EMBASE, the Cochrane library, and www.ClinicalTrials.gov were searched (from March to April 2018) for suitable English publications (published before April 2018).  The inclusion criteria were as follows: randomized controlled trials, autoimmune disorders (rheumatic arthritis, psoriatic arthritis, moderate to severe psoriasis, and ankylosing spondylitis), and comparison of adverse drug events associated with 5 mg versus 10 mg tofacitinib.  This study had follow-up time periods of 3 months and ≥ 6 months.  Statistical analysis was carried out by RevMan 5.3 whereby risk ratios (RRs) and 95% confidence intervals (CIs) were generated.  A total number of 4287 participants were included (2144 versus 2143 participants who received 5 mg and 10 mg tofacitinib twice daily respectively).  The results showed that at 3 months, similar risks of adverse drug events, serious adverse events, and adverse events leading to drug discontinuation were observed with 5 mg versus 10 mg tofacitinib (RR 1.04, 95% CI 0.98-1.10; P = 0.17, I(2) = 0%; RR 1.06, 95% CI 0.77-1.48; P = 0.71, I(2) = 0%; and RR 1.06, 95% CI 0.78-1.43; P = 0.73, I(2) = 32%, respectively).  The other outcomes including serious infection events, adjudicated herpes zoster infection, adjudicated opportunistic infection, mild and severe neutropenia, malignancies, and adjudicated major adverse cardiovascular events were also similarly manifested.  However, a decreased level of hemoglobin significantly favored 5 mg tofacitinib (RR 1.75, 95% CI 1.19-2.58; P = 0.005, I(2) = 49%).  Even at a follow-up time period of ≥ 6 months, adverse drug events, serious adverse events, adverse drug events leading to drug discontinuation, and serious infection were still similarly observed.  According to this current review, both dosages of tofacitinib were safe to use.  Even if similar adverse drug events were observed with 5 mg versus 10 mg tofacitinib twice daily for the treatment of autoimmune disorders, anemia was more prominent with 10 mg tofacitinib at a 3 month follow-up.  Nevertheless, future studies based on a larger population size with longer follow-up time periods should further be considered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSx8tJmFzzB7z0SWnHPWbmmfW6udTcc2eaTeqKkJZd-bLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3czhsFeruw%253D%253D&md5=4df34f92cb4e4ed0f090e9c50eed6c11</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1007%2Fs10067-018-4299-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10067-018-4299-4%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DF.%26aulast%3DLuo%26aufirst%3DZ.%2BC.%26atitle%3DAdverse%2520drug%2520events%2520associated%2520with%25205mg%2520versus%252010mg%2520tofacitinib%2520%2528Janus%2520kinase%2520inhibitor%2529%2520twice%2520daily%2520for%2520the%2520treatment%2520of%2520autoimmune%2520diseases%253A%2520A%2520systematic%2520review%2520and%2520meta-analysis%2520of%2520randomized%2520controlled%2520trials%26jtitle%3DClin.%2520Rheumatol.%26date%3D2019%26volume%3D38%26spage%3D523%26epage%3D534%26doi%3D10.1007%2Fs10067-018-4299-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hall, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nash, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rischmueller, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bossingham, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bird, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cook, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witcombe, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soma, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwok, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thirunavukkarasu, K.</span></span> <span> </span><span class="NLM_article-title">Tofacitinib, an oral Janus kinase inhibitor: Pooled efficacy and safety analyses in an australian rheumatoid arthritis population</span>. <i>Rheumatol Ther.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">383</span>– <span class="NLM_lpage">401</span>, <span class="refDoi"> DOI: 10.1007/s40744-018-0118-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=10.1007%2Fs40744-018-0118-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=29949132" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A280%3ADC%252BB3c%252FisVGgsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2018&pages=383-401&author=S.+Hallauthor=P.+Nashauthor=M.+Rischmuellerauthor=D.+Bossinghamauthor=P.+Birdauthor=N.+Cookauthor=D.+Witcombeauthor=K.+Somaauthor=K.+Kwokauthor=K.+Thirunavukkarasu&title=Tofacitinib%2C+an+oral+Janus+kinase+inhibitor%3A+Pooled+efficacy+and+safety+analyses+in+an+australian+rheumatoid+arthritis+population&doi=10.1007%2Fs40744-018-0118-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Tofacitinib, an Oral Janus Kinase Inhibitor: Pooled Efficacy and Safety Analyses in an Australian Rheumatoid Arthritis Population</span></div><div class="casAuthors">Hall Stephen; Nash Peter; Rischmueller Maureen; Bossingham David; Bird Paul; Cook Nicola; Witcombe David; Thirunavukkarasu Krishan; Soma Koshika; Kwok Kenneth</div><div class="citationInfo"><span class="NLM_cas:title">Rheumatology and therapy</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">383-401</span>
        ISSN:<span class="NLM_cas:issn">2198-6576</span>.
    </div><div class="casAbstract">INTRODUCTION:  In Australia, there is an unmet need for improved treatments for rheumatoid arthritis (RA).  Tofacitinib is an oral Janus kinase inhibitor for the treatment of RA.  To provide an overview of key study outcomes for tofacitinib in Australian patients, we analyzed the efficacy and safety of tofacitinib in the Australian subpopulation of global RA phase III and long-term extension (LTE) studies.  METHODS:  Data were pooled from the Australian subpopulation of four phase III studies and one LTE study (database not locked at cut-off date: January 2016).  Patients in the phase III studies received tofacitinib 5 or 10 mg twice daily (BID), placebo (advancing to tofacitinib at months 3 or 6), or adalimumab, with background methotrexate or conventional synthetic disease-modifying antirheumatic drugs.  Patients in the LTE study received tofacitinib 5 or 10 mg BID.  Efficacy endpoints were American College of Rheumatology (ACR) 20/50/70 response rates, and change from baseline in the Disease Activity Score in 28 joints, erythrocyte sedimentation rate [DAS28-4(ESR)] and Health Assessment Questionnaire-Disability Index (HAQ-DI) scores.  Safety endpoints included incidence of adverse events (AEs), serious AEs, and discontinuations due to AEs.  AEs of special interest and laboratory parameters were analyzed in the LTE study.  RESULTS:  Across phase III studies (N = 100), ACR response rates and improvements in DAS28-4(ESR) and HAQ-DI scores were numerically greater with tofacitinib vs. placebo at month 3, and increased until month 12.  The results were sustained in the LTE study (N = 99) after 60 months' observation.  In general, the efficacy and safety profiles of tofacitinib were similar to those of the global RA population.  CONCLUSIONS:  In Australian patients with RA, tofacitinib therapy demonstrated sustained efficacy and consistent safety over ≥ 60 months' treatment.  FUNDING:  Pfizer Inc.  TRIAL REGISTRATION NUMBERS (ALL CLINICALTRIALS.GOV): NCT00960440; NCT00847613; NCT00856544; NCT00853385; NCT00413699.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTF0DbmWrmTD0H3Rlp_k91zfW6udTcc2eaTeqKkJZd-bLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c%252FisVGgsg%253D%253D&md5=1d77e5c9389fe82bb292917e6b575712</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1007%2Fs40744-018-0118-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs40744-018-0118-2%26sid%3Dliteratum%253Aachs%26aulast%3DHall%26aufirst%3DS.%26aulast%3DNash%26aufirst%3DP.%26aulast%3DRischmueller%26aufirst%3DM.%26aulast%3DBossingham%26aufirst%3DD.%26aulast%3DBird%26aufirst%3DP.%26aulast%3DCook%26aufirst%3DN.%26aulast%3DWitcombe%26aufirst%3DD.%26aulast%3DSoma%26aufirst%3DK.%26aulast%3DKwok%26aufirst%3DK.%26aulast%3DThirunavukkarasu%26aufirst%3DK.%26atitle%3DTofacitinib%252C%2520an%2520oral%2520Janus%2520kinase%2520inhibitor%253A%2520Pooled%2520efficacy%2520and%2520safety%2520analyses%2520in%2520an%2520australian%2520rheumatoid%2520arthritis%2520population%26jtitle%3DRheumatol%2520Ther.%26date%3D2018%26volume%3D5%26spage%3D383%26epage%3D401%26doi%3D10.1007%2Fs40744-018-0118-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">van
Vollenhoven, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riese, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krishnaswami, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawabata, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fosser, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rottinghaus, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamba, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwillich, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradley, J.</span></span> <span> </span><span class="NLM_article-title">Relationship between lymphocyte count and risk of infection in rheumatoid arthritis patients treated with tofacitinib</span>. <i>Ann. Rheumatoid Dis.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>72</i></span>,  <span class="NLM_fpage">250</span>– <span class="NLM_lpage">251</span>, <span class="refDoi"> DOI: 10.1136/annrheumdis-2013-eular.780</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=10.1136%2Fannrheumdis-2013-eular.780" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=23065732" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2013&pages=250-251&author=R.+van%0AVollenhovenauthor=R.+Rieseauthor=S.+Krishnaswamiauthor=T.+Kawabataauthor=C.+Fosserauthor=S.+Rottinghausauthor=M.+Lambaauthor=S.+H.+Zwillichauthor=J.+Bradley&title=Relationship+between+lymphocyte+count+and+risk+of+infection+in+rheumatoid+arthritis+patients+treated+with+tofacitinib&doi=10.1136%2Fannrheumdis-2013-eular.780"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1136%2Fannrheumdis-2013-eular.780&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1136%252Fannrheumdis-2013-eular.780%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2BVollenhoven%26aufirst%3DR.%26aulast%3DRiese%26aufirst%3DR.%26aulast%3DKrishnaswami%26aufirst%3DS.%26aulast%3DKawabata%26aufirst%3DT.%26aulast%3DFosser%26aufirst%3DC.%26aulast%3DRottinghaus%26aufirst%3DS.%26aulast%3DLamba%26aufirst%3DM.%26aulast%3DZwillich%26aufirst%3DS.%2BH.%26aulast%3DBradley%26aufirst%3DJ.%26atitle%3DRelationship%2520between%2520lymphocyte%2520count%2520and%2520risk%2520of%2520infection%2520in%2520rheumatoid%2520arthritis%2520patients%2520treated%2520with%2520tofacitinib%26jtitle%3DAnn.%2520Rheumatoid%2520Dis.%26date%3D2013%26volume%3D72%26spage%3D250%26epage%3D251%26doi%3D10.1136%2Fannrheumdis-2013-eular.780" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Akada, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akada, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hutchison, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakamoto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, K. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohi, G.</span></span> <span> </span><span class="NLM_article-title">Critical role of Jak2 in the maintenance and function of adult hematopoietic stem cells</span>. <i>Stem Cells</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">1878</span>– <span class="NLM_lpage">1889</span>, <span class="refDoi"> DOI: 10.1002/stem.1711</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=10.1002%2Fstem.1711" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=24677703" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVSqsrvL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2014&pages=1878-1889&author=H.+Akadaauthor=S.+Akadaauthor=R.+E.+Hutchisonauthor=K.+Sakamotoauthor=K.+U.+Wagnerauthor=G.+Mohi&title=Critical+role+of+Jak2+in+the+maintenance+and+function+of+adult+hematopoietic+stem+cells&doi=10.1002%2Fstem.1711"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Critical role of Jak2 in the maintenance and function of adult hematopoietic stem cells</span></div><div class="casAuthors">Akada, Hajime; Akada, Saeko; Hutchison, Robert E.; Sakamoto, Kazuhito; Wagner, Kay-Uwe; Mohi, Golam</div><div class="citationInfo"><span class="NLM_cas:title">Stem Cells (Durham, NC, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1878-1889</span>CODEN:
                <span class="NLM_cas:coden">STCEEJ</span>;
        ISSN:<span class="NLM_cas:issn">1066-5099</span>.
    
            (<span class="NLM_cas:orgname">AlphaMed Press</span>)
        </div><div class="casAbstract">Jak2, a member of the Janus kinase family of nonreceptor protein tyrosine kinases, is activated in response to a variety of cytokines, and functions in survival and proliferation of cells.  An activating JAK2V617F mutation has been found in most patients with myeloproliferative neoplasms, and patients treated with Jak2 inhibitors show significant hematopoietic toxicities.  However, the role of Jak2 in adult hematopoietic stem cells (HSCs) has not been clearly elucidated.  Using a conditional Jak2 knockout allele, we have found that Jak2 deletion results in rapid loss of HSCs/progenitors leading to bone marrow failure and early lethality in adult mice.  Jak2 deficiency causes marked impairment in HSC function, and the mutant HSCs are severely defective in reconstituting hematopoiesis in recipient animals.  Jak2 deficiency also causes significant apoptosis and loss of quiescence in HSC-enriched LSK (Lin-Sca-1+c-Kit+) cells.  Jak2-deficient LSK cells exhibit elevated reactive oxygen species levels and enhanced p38 MAPK activation.  Mutant LSK cells also show defective Stat5, Erk, and Akt activation in response to thrombopoietin and stem cell factor.  Gene expression anal. reveals significant downregulation of genes related to HSC quiescence and self-renewal in Jak2-deficient LSK cells.  These data suggest that Jak2 plays a crit. role in the maintenance and function of adult HSCs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrrritPwMPsTLVg90H21EOLACvtfcHk0lhtJmp7OHj9oA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVSqsrvL&md5=5d8a4051c521f096754fe559c41bb878</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1002%2Fstem.1711&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fstem.1711%26sid%3Dliteratum%253Aachs%26aulast%3DAkada%26aufirst%3DH.%26aulast%3DAkada%26aufirst%3DS.%26aulast%3DHutchison%26aufirst%3DR.%2BE.%26aulast%3DSakamoto%26aufirst%3DK.%26aulast%3DWagner%26aufirst%3DK.%2BU.%26aulast%3DMohi%26aufirst%3DG.%26atitle%3DCritical%2520role%2520of%2520Jak2%2520in%2520the%2520maintenance%2520and%2520function%2520of%2520adult%2520hematopoietic%2520stem%2520cells%26jtitle%3DStem%2520Cells%26date%3D2014%26volume%3D32%26spage%3D1878%26epage%3D1889%26doi%3D10.1002%2Fstem.1711" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vazquez, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaila, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strohbach, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trzupek, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flanagan, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitton-Fry, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">TenBrink, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnold, E. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Basak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heasley, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langille, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parikh, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffin, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casavant, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duclos, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fenwick, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caspers, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dowty, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banker, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hegen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Symanowicz, P. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jussif, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Telliez, J.-B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Unwalla, R.</span></span> <span> </span><span class="NLM_article-title">Identification of <i>N</i>-{cis-3-[methyl(7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-4-yl)amino]cyclobutyl}propane-1-sulfonamide (PF-04965842): A selective JAK1 clinical candidate for the treatment of autoimmune diseases</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">1130</span>– <span class="NLM_lpage">1152</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01598</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01598" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A528%3ADC%252BC1cXit1Wnsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=1130-1152&author=M.+L.+Vazquezauthor=N.+Kailaauthor=J.+W.+Strohbachauthor=J.+D.+Trzupekauthor=M.+F.+Brownauthor=M.+E.+Flanaganauthor=M.+J.+Mitton-Fryauthor=T.+A.+Johnsonauthor=R.+E.+TenBrinkauthor=E.+P.+Arnoldauthor=A.+Basakauthor=S.+E.+Heasleyauthor=S.+Kwonauthor=J.+Langilleauthor=M.+D.+Parikhauthor=S.+H.+Griffinauthor=J.+M.+Casavantauthor=B.+A.+Duclosauthor=A.+E.+Fenwickauthor=T.+M.+Harrisauthor=S.+Hanauthor=N.+Caspersauthor=M.+E.+Dowtyauthor=X.+Yangauthor=M.+E.+Bankerauthor=M.+Hegenauthor=P.+T.+Symanowiczauthor=L.+Liauthor=L.+Wangauthor=T.+H.+Linauthor=J.+Jussifauthor=J.+D.+Clarkauthor=J.-B.+Telliezauthor=R.+P.+Robinsonauthor=R.+Unwalla&title=Identification+of+N-%7Bcis-3-%5Bmethyl%287H-pyrrolo%5B2%2C3-d%5Dpyrimidin-4-yl%29amino%5Dcyclobutyl%7Dpropane-1-sulfonamide+%28PF-04965842%29%3A+A+selective+JAK1+clinical+candidate+for+the+treatment+of+autoimmune+diseases&doi=10.1021%2Facs.jmedchem.7b01598"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of N-{cis-3-[Methyl(7H-pyrrolo[2,3-d]pyrimidin-4-yl)amino]cyclobutyl}propane-1-sulfonamide (PF-04965842): A Selective JAK1 Clinical Candidate for the Treatment of Autoimmune Diseases</span></div><div class="casAuthors">Vazquez, Michael L.; Kaila, Neelu; Strohbach, Joseph W.; Trzupek, John D.; Brown, Matthew F.; Flanagan, Mark E.; Mitton-Fry, Mark J.; Johnson, Timothy A.; TenBrink, Ruth E.; Arnold, Eric P.; Basak, Arindrajit; Heasley, Steven E.; Kwon, Soojin; Langille, Jonathan; Parikh, Mihir D.; Griffin, Sarah H.; Casavant, Jeffrey M.; Duclos, Brian A.; Fenwick, Ashley E.; Harris, Thomas M.; Han, Seungil; Caspers, Nicole; Dowty, Martin E.; Yang, Xin; Banker, Mary Ellen; Hegen, Martin; Symanowicz, Peter T.; Li, Li; Wang, Lu; Lin, Tsung H.; Jussif, Jason; Clark, James D.; Telliez, Jean-Baptiste; Robinson, Ralph P.; Unwalla, Ray</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1130-1152</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Janus kinases (JAKs) are intracellular tyrosine kinases that mediate the signaling of numerous cytokines and growth factors involved in the regulation of immunity, inflammation, and hematopoiesis.  As JAK1 pairs with JAK2, JAK3, and TYK2, a JAK1-selective inhibitor would be expected to inhibit many cytokines involved in inflammation and immune function while avoiding inhibition of the JAK2 homodimer regulating erythropoietin and thrombopoietin signaling.  The authors' efforts began with tofacitinib, an oral JAK inhibitor approved for the treatment of rheumatoid arthritis.  Through modification of the 3-aminopiperidine linker in tofacitinib, the authors discovered highly selective JAK1 inhibitors with nanomolar potency in a human whole blood assay.  Improvements in JAK1 potency and selectivity were achieved via structural modifications suggested by x-ray crystallog. anal.  After demonstrating efficacy in a rat adjuvant-induced arthritis (rAIA) model, PF-04965842 (25) was nominated as a clin. candidate for the treatment of JAK1-mediated autoimmune diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNdiJFIHI-YLVg90H21EOLACvtfcHk0ljVTZvR5hLffA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXit1Wnsw%253D%253D&md5=45c6d1b0eee7ab9d5dfc8ea168607a8b</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01598&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01598%26sid%3Dliteratum%253Aachs%26aulast%3DVazquez%26aufirst%3DM.%2BL.%26aulast%3DKaila%26aufirst%3DN.%26aulast%3DStrohbach%26aufirst%3DJ.%2BW.%26aulast%3DTrzupek%26aufirst%3DJ.%2BD.%26aulast%3DBrown%26aufirst%3DM.%2BF.%26aulast%3DFlanagan%26aufirst%3DM.%2BE.%26aulast%3DMitton-Fry%26aufirst%3DM.%2BJ.%26aulast%3DJohnson%26aufirst%3DT.%2BA.%26aulast%3DTenBrink%26aufirst%3DR.%2BE.%26aulast%3DArnold%26aufirst%3DE.%2BP.%26aulast%3DBasak%26aufirst%3DA.%26aulast%3DHeasley%26aufirst%3DS.%2BE.%26aulast%3DKwon%26aufirst%3DS.%26aulast%3DLangille%26aufirst%3DJ.%26aulast%3DParikh%26aufirst%3DM.%2BD.%26aulast%3DGriffin%26aufirst%3DS.%2BH.%26aulast%3DCasavant%26aufirst%3DJ.%2BM.%26aulast%3DDuclos%26aufirst%3DB.%2BA.%26aulast%3DFenwick%26aufirst%3DA.%2BE.%26aulast%3DHarris%26aufirst%3DT.%2BM.%26aulast%3DHan%26aufirst%3DS.%26aulast%3DCaspers%26aufirst%3DN.%26aulast%3DDowty%26aufirst%3DM.%2BE.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DBanker%26aufirst%3DM.%2BE.%26aulast%3DHegen%26aufirst%3DM.%26aulast%3DSymanowicz%26aufirst%3DP.%2BT.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DLin%26aufirst%3DT.%2BH.%26aulast%3DJussif%26aufirst%3DJ.%26aulast%3DClark%26aufirst%3DJ.%2BD.%26aulast%3DTelliez%26aufirst%3DJ.-B.%26aulast%3DRobinson%26aufirst%3DR.%2BP.%26aulast%3DUnwalla%26aufirst%3DR.%26atitle%3DIdentification%2520of%2520N-%257Bcis-3-%255Bmethyl%25287H-pyrrolo%255B2%252C3-d%255Dpyrimidin-4-yl%2529amino%255Dcyclobutyl%257Dpropane-1-sulfonamide%2520%2528PF-04965842%2529%253A%2520A%2520selective%2520JAK1%2520clinical%2520candidate%2520for%2520the%2520treatment%2520of%2520autoimmune%2520diseases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D1130%26epage%3D1152%26doi%3D10.1021%2Facs.jmedchem.7b01598" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Spolski, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gromer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leonard, W. J.</span></span> <span> </span><span class="NLM_article-title">The γ c family of cytokines: fine-tuning signals from IL-2 and IL-21 in the regulation of the immune response</span>. <i>F1000Research</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">1872</span>, <span class="refDoi"> DOI: 10.12688/f1000research.12202.1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=10.12688%2Ff1000research.12202.1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=29123649" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisFSru73L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2017&pages=1872&author=R.+Spolskiauthor=D.+Gromerauthor=W.+J.+Leonard&title=The+%CE%B3+c+family+of+cytokines%3A+fine-tuning+signals+from+IL-2+and+IL-21+in+the+regulation+of+the+immune+response&doi=10.12688%2Ff1000research.12202.1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">The γc family of cytokines: fine-tuning signals from IL-2 and IL-21 in the regulation of the immune response [version 1; referees: 3 approved]</span></div><div class="casAuthors">Spolski, Rosanne; Gromer, Daniel; Leonard, Warren J.</div><div class="citationInfo"><span class="NLM_cas:title">F1000Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1872/1-1872/12</span>CODEN:
                <span class="NLM_cas:coden">FRESJL</span>;
        ISSN:<span class="NLM_cas:issn">2046-1402</span>.
    
            (<span class="NLM_cas:orgname">F1000 Research Ltd.</span>)
        </div><div class="casAbstract">Interleukin (IL)-2, IL-4, IL-7, IL-9, IL-15, and IL-21 form a family of cytokines based on the sharing of a receptor component, the common cytokine receptor γ chain, γc, which is encoded by the gene mutated in humans with X-linked severe combined immunodeficiency (XSCID).  Together, these cytokines play crit. roles in lymphoid development, differentiation, growth, and survival as well as mediating effector function.  Here, we provide an overview of the main actions of members of this cytokine family but then primarily focus on IL-2 and IL-21, discussing their dynamic interplay and contributions to a fine-tuned immune response.  Moreover, we discuss the therapeutic utility of modulating their actions, particularly for autoimmunity and cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpg5g2WamKEyrVg90H21EOLACvtfcHk0ljVTZvR5hLffA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisFSru73L&md5=18f97a3267d5cd8c30370dcb606dd51e</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.12688%2Ff1000research.12202.1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.12688%252Ff1000research.12202.1%26sid%3Dliteratum%253Aachs%26aulast%3DSpolski%26aufirst%3DR.%26aulast%3DGromer%26aufirst%3DD.%26aulast%3DLeonard%26aufirst%3DW.%2BJ.%26atitle%3DThe%2520%25CE%25B3%2520c%2520family%2520of%2520cytokines%253A%2520fine-tuning%2520signals%2520from%2520IL-2%2520and%2520IL-21%2520in%2520the%2520regulation%2520of%2520the%2520immune%2520response%26jtitle%3DF1000Research%26date%3D2017%26volume%3D6%26spage%3D1872%26doi%3D10.12688%2Ff1000research.12202.1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">White, U. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stephens, J. M.</span></span> <span> </span><span class="NLM_article-title">The gp130 receptor cytokine family: regulators of adipocyte development and function</span>. <i>Curr. Pharm. Des.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">340</span>– <span class="NLM_lpage">346</span>, <span class="refDoi"> DOI: 10.2174/138161211795164202</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=10.2174%2F138161211795164202" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=21375496" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A528%3ADC%252BC3MXlsVKlsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2011&pages=340-346&author=U.+A.+Whiteauthor=J.+M.+Stephens&title=The+gp130+receptor+cytokine+family%3A+regulators+of+adipocyte+development+and+function&doi=10.2174%2F138161211795164202"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">The gp130 receptor cytokine family: regulators of adipocyte development and function</span></div><div class="casAuthors">White, Ursula A.; Stephens, Jacqueline M.</div><div class="citationInfo"><span class="NLM_cas:title">Current Pharmaceutical Design</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">340-346</span>CODEN:
                <span class="NLM_cas:coden">CPDEFP</span>;
        ISSN:<span class="NLM_cas:issn">1381-6128</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Gp130 cytokines are involved in the regulation of numerous biol. processes, including hematopoiesis, immune response, inflammation, cardiovascular action, and neuronal survival.  These cytokines share glycoprotein 130 as a common signal transducer in their receptor complex and typically activate STAT3.  Most gp130 cytokines have paracrine or endocrine actions, and their levels can be measured in circulation in rodents and humans.  In recent years, various labs. have conducted studies to demonstrate that gp130 cytokines can modulate adipocyte development and function.  Therefore, these studies suggest that some gp130 cytokines may be viable anti-obesity therapeutics.  In this review, we will summarize the reported effects of gp130 cytokines on adipocyte differentiation and adipocyte function.  In addn., the modulation of gp130 cytokines in conditions of obesity, insulin resistance, and Type 2 diabetes will be presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprllah-s4Q2rVg90H21EOLACvtfcHk0lg3BKHaINHEcw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXlsVKlsbo%253D&md5=8cb1820e42f890178365ad102df33a14</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.2174%2F138161211795164202&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138161211795164202%26sid%3Dliteratum%253Aachs%26aulast%3DWhite%26aufirst%3DU.%2BA.%26aulast%3DStephens%26aufirst%3DJ.%2BM.%26atitle%3DThe%2520gp130%2520receptor%2520cytokine%2520family%253A%2520regulators%2520of%2520adipocyte%2520development%2520and%2520function%26jtitle%3DCurr.%2520Pharm.%2520Des.%26date%3D2011%26volume%3D17%26spage%3D340%26epage%3D346%26doi%3D10.2174%2F138161211795164202" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonelli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gadina, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’shea, J. J.</span></span> <span> </span><span class="NLM_article-title">Type I/II cytokines, JAKs, and new strategies for treating autoimmune diseases</span>. <i>Nat. Rev. Rheumatol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">25</span>– <span class="NLM_lpage">36</span>, <span class="refDoi"> DOI: 10.1038/nrrheum.2015.167</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=10.1038%2Fnrrheum.2015.167" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=26633291" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvF2ks7vJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=25-36&author=D.+M.+Schwartzauthor=M.+Bonelliauthor=M.+Gadinaauthor=J.+J.+O%E2%80%99shea&title=Type+I%2FII+cytokines%2C+JAKs%2C+and+new+strategies+for+treating+autoimmune+diseases&doi=10.1038%2Fnrrheum.2015.167"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Type I/II cytokines, JAKs, and new strategies for treating autoimmune diseases</span></div><div class="casAuthors">Schwartz, Daniella M.; Bonelli, Michael; Gadina, Massimo; O'Shea, John J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Rheumatology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">25-36</span>CODEN:
                <span class="NLM_cas:coden">NRRACB</span>;
        ISSN:<span class="NLM_cas:issn">1759-4790</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Cytokines are major drivers of autoimmunity, and biol. agents targeting cytokines have revolutionized the treatment of immune-mediated diseases.  Despite the effectiveness of these drugs, they do not induce complete remission in all patients, prompting the development of alternative strategies - including targeting of intracellular signal transduction pathways downstream of cytokines.  Many cytokines that bind type I and type II cytokine receptors are crit. regulators of immune-mediated diseases and employ the Janus kinase (JAK) and signal transducer and activator of transcription (STAT) pathway to exert their effect.  Pharmacol. inhibition of JAKs blocks the actions of type I/II cytokines, and within the past 3 years therapeutic JAK inhibitors, or Jakinibs, have become available to rheumatologists.  Jakinibs have proven effective for the treatment of rheumatoid arthritis and other inflammatory diseases.  Adverse effects of these agents are largely related to their mode of action and include infections and hyperlipidemia.  Jakinibs are currently being investigated for a no. of new indications, and second-generation selective Jakinibs are being developed and tested.  Targeting STATs could be a future avenue for the treatment of rheumatol. diseases, although substantial challenges remain.  Nonetheless, the ability to therapeutically target intracellular signalling pathways has already created a new paradigm for the treatment of rheumatol. disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUdr1Dh92dnLVg90H21EOLACvtfcHk0lg3BKHaINHEcw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvF2ks7vJ&md5=e06f7ab6424f1fbdbf481c75fa9ebd23</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fnrrheum.2015.167&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrrheum.2015.167%26sid%3Dliteratum%253Aachs%26aulast%3DSchwartz%26aufirst%3DD.%2BM.%26aulast%3DBonelli%26aufirst%3DM.%26aulast%3DGadina%26aufirst%3DM.%26aulast%3DO%25E2%2580%2599shea%26aufirst%3DJ.%2BJ.%26atitle%3DType%2520I%252FII%2520cytokines%252C%2520JAKs%252C%2520and%2520new%2520strategies%2520for%2520treating%2520autoimmune%2520diseases%26jtitle%3DNat.%2520Rev.%2520Rheumatol.%26date%3D2016%26volume%3D12%26spage%3D25%26epage%3D36%26doi%3D10.1038%2Fnrrheum.2015.167" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roskoski, R.,  Jr.</span></span> <span> </span><span class="NLM_article-title">Janus kinase (JAK) inhibitors in the treatment of inflammatory and neoplastic diseases</span>. <i>Pharmacol. Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>111</i></span>,  <span class="NLM_fpage">784</span>– <span class="NLM_lpage">803</span>, <span class="refDoi"> DOI: 10.1016/j.phrs.2016.07.038</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=10.1016%2Fj.phrs.2016.07.038" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=27473820" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht12jurzJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2016&pages=784-803&author=R.+Roskoski&title=Janus+kinase+%28JAK%29+inhibitors+in+the+treatment+of+inflammatory+and+neoplastic+diseases&doi=10.1016%2Fj.phrs.2016.07.038"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Janus kinase (JAK) inhibitors in the treatment of inflammatory and neoplastic diseases</span></div><div class="casAuthors">Roskoski, Robert, Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">784-803</span>CODEN:
                <span class="NLM_cas:coden">PHMREP</span>;
        ISSN:<span class="NLM_cas:issn">1043-6618</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The Janus kinase (JAK) family of non-receptor protein-tyrosine kinases consists of JAK1, JAK2, JAK3, and TYK2 (tyrosine kinase-2).  Each of these proteins contains a JAK homol. pseudokinase (JH2) domain that regulates the adjacent protein kinase domain (JH1).  JAK1/2 and TYK2 are ubiquitously expressed whereas JAK3 is found predominantly in hematopoietic cells.  The Janus kinase family is regulated by numerous cytokines including interleukins, interferons, and hormones such as erythropoietin, thrombopoietin, and growth hormone.  Ligand binding to cytokine and hormone receptors leads to the activation of assocd. Janus kinases, which then mediate the phosphorylation of the receptors.  The SH2 domain of STATs (signal transducers and activators of transcription) binds to the receptor phosphotyrosines thereby promoting STAT phosphorylation by the Janus kinases and consequent activation.  STAT dimers are translocated to the nucleus where they participate in the regulation of the expression of thousands of proteins.  JAK-STAT dysregulation results in autoimmune disorders such as rheumatoid arthritis, ulcerative colitis, and Crohn disease.  JAK-STAT dysregulation also plays a role in the pathogenesis of myelofibrosis, polycythemia vera, and other myeloproliferative illnesses.  An activating JAK2 V617F mutation occurs in 95% of people with polycythemia vera and in a lower percentage of people with other neoplasms.  JAK1/3 signaling participates in the pathogenesis of inflammatory afflictions while JAK1/2 signaling participates in the development of several malignancies including leukemias and lymphomas as well as myeloproliferative neoplasms.  Tofacitinib is a pan-JAK inhibitor that is approved by the FDA for the treatment of rheumatoid arthritis and ruxolitinib is a JAK1/2 inhibitor that is approved for the treatment of polycythemia vera and myelofibrosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGov32m4UjOrv7Vg90H21EOLACvtfcHk0lg3BKHaINHEcw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht12jurzJ&md5=a45fba1bb110eff4d5c1b31b78c1b3ac</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.phrs.2016.07.038&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.phrs.2016.07.038%26sid%3Dliteratum%253Aachs%26aulast%3DRoskoski%26aufirst%3DR.%26atitle%3DJanus%2520kinase%2520%2528JAK%2529%2520inhibitors%2520in%2520the%2520treatment%2520of%2520inflammatory%2520and%2520neoplastic%2520diseases%26jtitle%3DPharmacol.%2520Res.%26date%3D2016%26volume%3D111%26spage%3D784%26epage%3D803%26doi%3D10.1016%2Fj.phrs.2016.07.038" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Parmentier, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Voss, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graff, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Argiriadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friedman, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Camp, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Padley, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">George, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hyland, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosebraugh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wishart, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, J. A.</span></span> <span> </span><span class="NLM_article-title"><i>In vitro</i> and <i>in vivo</i> characterization of the JAK1 selectivity of upadacitinib (ABT-494)</span>. <i>BMC Rheumatol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">23</span>, <span class="refDoi"> DOI: 10.1186/s41927-018-0031-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=10.1186%2Fs41927-018-0031-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=30886973" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A280%3ADC%252BB3cbls1ymuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2018&pages=23&author=J.+M.+Parmentierauthor=J.+Vossauthor=C.+Graffauthor=A.+Schwartzauthor=M.+Argiriadiauthor=M.+Friedmanauthor=S.+H.+Campauthor=J.+R.+Padleyauthor=S.+J.+Georgeauthor=D.+Hylandauthor=M.+Rosebraughauthor=N.+Wishartauthor=L.+Olsonauthor=J.+A.+Long&title=In+vitro+and+in+vivo+characterization+of+the+JAK1+selectivity+of+upadacitinib+%28ABT-494%29&doi=10.1186%2Fs41927-018-0031-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">In vitro and in vivo characterization of the JAK1 selectivity of upadacitinib (ABT-494)</span></div><div class="casAuthors">Parmentier Julie M; Voss Jeff; Graff Candace; Schwartz Annette; Argiriadi Maria; Friedman Michael; George Jonathan S; Hyland Deborah; Wishart Neil; Olson Lisa; Long Andrew J; Camp Heidi S; Padley Robert J; Rosebraugh Matthew</div><div class="citationInfo"><span class="NLM_cas:title">BMC rheumatology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">23</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Background:  Anti-cytokine therapies such as adalimumab, tocilizumab, and the small molecule JAK inhibitor tofacitinib have proven that cytokines and their subsequent downstream signaling processes are important in the pathogenesis of rheumatoid arthritis.  Tofacitinib, a pan-JAK inhibitor, is the first approved JAK inhibitor for the treatment of RA and has been shown to be effective in managing disease.  However, in phase 2 dose-ranging studies tofacitinib was associated with dose-limiting tolerability and safety issues such as anemia.  Upadacitinib (ABT-494) is a selective JAK1 inhibitor that was engineered to address the hypothesis that greater JAK1 selectivity over other JAK family members will translate into a more favorable benefit:risk profile.  Upadacitinib selectively targets JAK1 dependent disease drivers such as IL-6 and IFNγ, while reducing effects on reticulocytes and natural killer (NK) cells, which potentially contributed to the tolerability issues of tofacitinib.  Methods:  Structure-based hypotheses were used to design the JAK1 selective inhibitor upadacitinib.  JAK family selectivity was defined with in vitro assays including biochemical assessments, engineered cell lines, and cytokine stimulation.  In vivo selectivity was defined by the efficacy of upadacitinib and tofacitinib in a rat adjuvant induced arthritis model, activity on reticulocyte deployment, and effect on circulating NK cells.  The translation of the preclinical JAK1 selectivity was assessed in healthy volunteers using ex vivo stimulation with JAK-dependent cytokines.  Results:  Here, we show the structural basis for the JAK1 selectivity of upadacitinib, along with the in vitro JAK family selectivity profile and subsequent in vivo physiological consequences.  Upadacitinib is ~ 60 fold selective for JAK1 over JAK2, and > 100 fold selective over JAK3 in cellular assays.  While both upadacitinib and tofacitinib demonstrated efficacy in a rat model of arthritis, the increased selectivity of upadacitinib for JAK1 resulted in a reduced effect on reticulocyte deployment and NK cell depletion relative to efficacy.  Ex vivo pharmacodynamic data obtained from Phase I healthy volunteers confirmed the JAK1 selectivity of upadactinib in a clinical setting.  Conclusions:  The data presented here highlight the JAK1 selectivity of upadacinitinib and supports its use as an effective therapy for the treatment of RA with the potential for an improved benefit:risk profile.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRTVFFxllNWaxfKb6ELK8t0fW6udTcc2eZ5T2xGWxQY5rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3cbls1ymuw%253D%253D&md5=5ca2dcb18a98362567d2193e858cc800</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1186%2Fs41927-018-0031-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs41927-018-0031-x%26sid%3Dliteratum%253Aachs%26aulast%3DParmentier%26aufirst%3DJ.%2BM.%26aulast%3DVoss%26aufirst%3DJ.%26aulast%3DGraff%26aufirst%3DC.%26aulast%3DSchwartz%26aufirst%3DA.%26aulast%3DArgiriadi%26aufirst%3DM.%26aulast%3DFriedman%26aufirst%3DM.%26aulast%3DCamp%26aufirst%3DS.%2BH.%26aulast%3DPadley%26aufirst%3DJ.%2BR.%26aulast%3DGeorge%26aufirst%3DS.%2BJ.%26aulast%3DHyland%26aufirst%3DD.%26aulast%3DRosebraugh%26aufirst%3DM.%26aulast%3DWishart%26aufirst%3DN.%26aulast%3DOlson%26aufirst%3DL.%26aulast%3DLong%26aufirst%3DJ.%2BA.%26atitle%3DIn%2520vitro%2520and%2520in%2520vivo%2520characterization%2520of%2520the%2520JAK1%2520selectivity%2520of%2520upadacitinib%2520%2528ABT-494%2529%26jtitle%3DBMC%2520Rheumatol.%26date%3D2018%26volume%3D2%26spage%3D23%26doi%3D10.1186%2Fs41927-018-0031-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span> <span> </span><span class="NLM_article-title">Discovery and evaluation of ZT55, a novel highly-selective tyrosine kinase inhibitor of JAK2<sup>V617F</sup> against myeloproliferative neoplasms</span>. <i>J. Exp. Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">49</span>, <span class="refDoi"> DOI: 10.1186/s13046-019-1062-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=10.1186%2Fs13046-019-1062-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=30717771" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A280%3ADC%252BB3cjovVyksg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2019&pages=49&author=M.+Huauthor=C.+Xuauthor=C.+Yangauthor=H.+Zuoauthor=C.+Chenauthor=D.+Zhangauthor=G.+Shiauthor=W.+Wangauthor=J.+Shiauthor=T.+Zhang&title=Discovery+and+evaluation+of+ZT55%2C+a+novel+highly-selective+tyrosine+kinase+inhibitor+of+JAK2V617F+against+myeloproliferative+neoplasms&doi=10.1186%2Fs13046-019-1062-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and evaluation of ZT55, a novel highly-selective tyrosine kinase inhibitor of JAK2(V617F) against myeloproliferative neoplasms</span></div><div class="casAuthors">Hu Min; Xu Chengbo; Chen Chengjuan; Shi Gaona; Wang Wenjie; Shi Jiangong; Zhang Tiantai; Yang Chao; Zuo Hongli; Zhang Dan; Zhang Tiantai</div><div class="citationInfo"><span class="NLM_cas:title">Journal of experimental & clinical cancer research : CR</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">49</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  The JAK2-STAT signaling pathway plays a critical role in myeloproliferative neoplasms (MPN).  An activating mutation in JAK2 (V617F) is present in ~ 95% of polycythemia vera, essential thrombocythemia, and primary myelofibrosis cases.  This study aims to explore the selective JAK2(V617F) inhibitor, evaluate the efficacy and possible mechanism of ZT55 on MPN.  METHODS:  HTRF assays were conducted to evaluate the selective inhibition of ZT55 for JAKs.  Cell apoptosis, proliferation, and cycle arrest assays were performed to examine the effect of ZT55 on HEL cell line with JAK2(V617F) mutation in vitro.  Western analysis was used to monitor the expression and activity of proteins on JAK2/STAT pathway.  A mice xenograft model was established to evaluate the antitumor efficacy of ZT55 in vivo.  Peripheral blood samples from patients with the JAK2(V617F) mutation were collected to estimate the effect of ZT55 on erythroid colony formation by colony-forming assay.  RESULTS:  We found that ZT55 showed a selective inhibition of a 0.031 μM IC50 value against JAK2.  It exhibited potent effects on the cellular JAK-STAT pathway, inhibiting tyrosine phosphorylation in JAK2(V617F) and downstream STAT3/5 transcription factors.  ZT55 inhibited the proliferation of the JAK2(V617F)-expressing HEL cell line, leading to cell cycle arrest at the G2/M phase and induction of caspase-dependent apoptosis.  Notably, ZT55 also significantly suppressed the growth of HEL xenograft tumors in vivo.  Further evaluation indicated that ZT55 blocked erythroid colony formation of peripheral blood hematopoietic progenitors from patients carrying the JAK2(V617F) mutation.  CONCLUSION:  These results suggest that ZT55 is a highly-selective JAK2 inhibitor that can induce apoptosis of human erythroleukemia cells by inhibiting the JAK2-STAT signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTBu63j-Oqd92hHGZei4OW9fW6udTcc2eZ5T2xGWxQY5rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3cjovVyksg%253D%253D&md5=a65f4b36f9cacf7547c25d16f881132f</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1186%2Fs13046-019-1062-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13046-019-1062-x%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DM.%26aulast%3DXu%26aufirst%3DC.%26aulast%3DYang%26aufirst%3DC.%26aulast%3DZuo%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DShi%26aufirst%3DG.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DT.%26atitle%3DDiscovery%2520and%2520evaluation%2520of%2520ZT55%252C%2520a%2520novel%2520highly-selective%2520tyrosine%2520kinase%2520inhibitor%2520of%2520JAK2V617F%2520against%2520myeloproliferative%2520neoplasms%26jtitle%3DJ.%2520Exp.%2520Clin.%2520Cancer%2520Res.%26date%3D2019%26volume%3D38%26spage%3D49%26doi%3D10.1186%2Fs13046-019-1062-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thorarensen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dowty, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banker, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Juba, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jussif, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vincent, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czerwinski, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casimiro-Garcia, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Unwalla, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trujillo, I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balbo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Che, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilbert, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayward, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montgomery, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hegen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coe, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langille, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajdos, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chrencik, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Telliez, J.-B.</span></span> <span> </span><span class="NLM_article-title">Design of a Janus kinase 3 (JAK3) specific inhibitor 1-((2<i>S</i>, 5<i>R</i>)-5-((7<i>H</i>-Pyrrolo[2,3-<i>d</i>] pyrimidin-4-yl) amino)-2-methylpiperidin-1-yl) prop-2-en-1-one (PF-06651600) allowing for the interrogation of JAK3 signaling in humans</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">1971</span>– <span class="NLM_lpage">1993</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01694</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01694" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVykt7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=1971-1993&author=A.+Thorarensenauthor=M.+E.+Dowtyauthor=M.+E.+Bankerauthor=B.+Jubaauthor=J.+Jussifauthor=T.+Linauthor=F.+Vincentauthor=M.+R.+Czerwinskiauthor=A.+Casimiro-Garciaauthor=R.+Unwallaauthor=I.+J.+Trujilloauthor=S.+Liangauthor=P.+Balboauthor=Y.+Cheauthor=M.+A.+Gilbertauthor=F.+M.+Brownauthor=M.+Haywardauthor=J.+Montgomeryauthor=L.+Leungauthor=X.+Yangauthor=M.+Hegenauthor=J.+Coeauthor=J.+Langilleauthor=F.+Vajdosauthor=J.+Chrencikauthor=J.-B.+Telliez&title=Design+of+a+Janus+kinase+3+%28JAK3%29+specific+inhibitor+1-%28%282S%2C+5R%29-5-%28%287H-Pyrrolo%5B2%2C3-d%5D+pyrimidin-4-yl%29+amino%29-2-methylpiperidin-1-yl%29+prop-2-en-1-one+%28PF-06651600%29+allowing+for+the+interrogation+of+JAK3+signaling+in+humans&doi=10.1021%2Facs.jmedchem.6b01694"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Design of a Janus Kinase 3 (JAK3) Specific Inhibitor 1-((2S,5R)-5-((7H-Pyrrolo[2,3-d]pyrimidin-4-yl)amino)-2-methylpiperidin-1-yl)prop-2-en-1-one (PF-06651600) Allowing for the Interrogation of JAK3 Signaling in Humans</span></div><div class="casAuthors">Thorarensen, Atli; Dowty, Martin E.; Banker, Mary Ellen; Juba, Brian; Jussif, Jason; Lin, Tsung; Vincent, Fabien; Czerwinski, Robert M.; Casimiro-Garcia, Agustin; Unwalla, Ray; Trujillo, John I.; Liang, Sidney; Balbo, Paul; Che, Ye; Gilbert, Adam M.; Brown, Matthew F.; Hayward, Matthew; Montgomery, Justin; Leung, Louis; Yang, Xin; Soucy, Sarah; Hegen, Martin; Coe, Jotham; Langille, Jonathan; Vajdos, Felix; Chrencik, Jill; Telliez, Jean-Baptiste</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1971-1993</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Significant work has been dedicated to the discovery of JAK kinase inhibitors resulting in several compds. entering clin. development and two FDA approved NMEs.  However, despite significant effort during the past two decades, identification of highly selective JAK3 inhibitors has eluded the scientific community.  A significant effort within the research organization has resulted in the identification of the first orally active JAK3 specific inhibitor, which achieves JAK isoform specificity through covalent interaction with a unique JAK3 residue Cys-909.  The relatively rapid resynthesis rate of the JAK3 enzyme presented a unique challenge in the design of covalent inhibitors with appropriate pharmacodynamics properties coupled with limited unwanted off-target reactivity.  This effort resulted in the identification of PF-06651600 I, a potent and low clearance compd. with demonstrated in vivo efficacy.  The favorable efficacy and safety profile of this specific JAK3 inhibitor I led to its evaluation in several human clin. studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodS6VFIL07nrVg90H21EOLACvtfcHk0lib7iUzS7nPmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVykt7c%253D&md5=d1ab749203fae07f89463f97518e3db8</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01694&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01694%26sid%3Dliteratum%253Aachs%26aulast%3DThorarensen%26aufirst%3DA.%26aulast%3DDowty%26aufirst%3DM.%2BE.%26aulast%3DBanker%26aufirst%3DM.%2BE.%26aulast%3DJuba%26aufirst%3DB.%26aulast%3DJussif%26aufirst%3DJ.%26aulast%3DLin%26aufirst%3DT.%26aulast%3DVincent%26aufirst%3DF.%26aulast%3DCzerwinski%26aufirst%3DM.%2BR.%26aulast%3DCasimiro-Garcia%26aufirst%3DA.%26aulast%3DUnwalla%26aufirst%3DR.%26aulast%3DTrujillo%26aufirst%3DI.%2BJ.%26aulast%3DLiang%26aufirst%3DS.%26aulast%3DBalbo%26aufirst%3DP.%26aulast%3DChe%26aufirst%3DY.%26aulast%3DGilbert%26aufirst%3DM.%2BA.%26aulast%3DBrown%26aufirst%3DF.%2BM.%26aulast%3DHayward%26aufirst%3DM.%26aulast%3DMontgomery%26aufirst%3DJ.%26aulast%3DLeung%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DHegen%26aufirst%3DM.%26aulast%3DCoe%26aufirst%3DJ.%26aulast%3DLangille%26aufirst%3DJ.%26aulast%3DVajdos%26aufirst%3DF.%26aulast%3DChrencik%26aufirst%3DJ.%26aulast%3DTelliez%26aufirst%3DJ.-B.%26atitle%3DDesign%2520of%2520a%2520Janus%2520kinase%25203%2520%2528JAK3%2529%2520specific%2520inhibitor%25201-%2528%25282S%252C%25205R%2529-5-%2528%25287H-Pyrrolo%255B2%252C3-d%255D%2520pyrimidin-4-yl%2529%2520amino%2529-2-methylpiperidin-1-yl%2529%2520prop-2-en-1-one%2520%2528PF-06651600%2529%2520allowing%2520for%2520the%2520interrogation%2520of%2520JAK3%2520signaling%2520in%2520humans%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D1971%26epage%3D1993%26doi%3D10.1021%2Facs.jmedchem.6b01694" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Menet, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fletcher, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Lommen, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geney, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blanc, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smits, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jouannigot, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deprez, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der
Aar, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clement-Lacroix, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lepescheux, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galien, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vayssiere, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nelles, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christophe, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brys, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uhring, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciesielski, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Rompaey, L.</span></span> <span> </span><span class="NLM_article-title">Triazolopyridines as selective JAK1 inhibitors: From hit identification to GLPG0634</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">9323</span>– <span class="NLM_lpage">9342</span>, <span class="refDoi"> DOI: 10.1021/jm501262q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501262q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVGls7nN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=9323-9342&author=C.+J.+Menetauthor=S.+R.+Fletcherauthor=G.+Van+Lommenauthor=R.+Geneyauthor=J.+Blancauthor=K.+Smitsauthor=N.+Jouannigotauthor=P.+Deprezauthor=E.+M.+van+der%0AAarauthor=P.+Clement-Lacroixauthor=L.+Lepescheuxauthor=R.+Galienauthor=B.+Vayssiereauthor=L.+Nellesauthor=T.+Christopheauthor=R.+Brysauthor=M.+Uhringauthor=F.+Ciesielskiauthor=L.+Van+Rompaey&title=Triazolopyridines+as+selective+JAK1+inhibitors%3A+From+hit+identification+to+GLPG0634&doi=10.1021%2Fjm501262q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Triazolopyridines as Selective JAK1 Inhibitors: From Hit Identification to GLPG0634</span></div><div class="casAuthors">Menet, Christel J.; Fletcher, Stephen R.; Van Lommen, Guy; Geney, Raphael; Blanc, Javier; Smits, Koen; Jouannigot, Nolwenn; Deprez, Pierre; van der Aar, Ellen M.; Clement-Lacroix, Philippe; Lepescheux, Lien; Galien, Rene; Vayssiere, Beatrice; Nelles, Luc; Christophe, Thierry; Brys, Reginald; Uhring, Muriel; Ciesielski, Fabrice; Van Rompaey, Luc</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">9323-9342</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Janus kinases (JAK1, JAK2, JAK3, and TYK2) are involved in the signaling of multiple cytokines important in cellular function.  Blockade of the JAK-STAT pathway with a small mol. has been shown to provide therapeutic immunomodulation.  Having identified JAK1 as a possible new target for arthritis at Galapagos, the compd. library was screened against JAK1, resulting in the identification of a triazolopyridine-based series of inhibitors represented by I.  Optimization within this chem. series led to identification of II (GLPG0634, filgotinib), a selective JAK1 inhibitor currently in phase 2B development for RA and phase 2A development for Crohn's disease (CD).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBHELw4zQ5QLVg90H21EOLACvtfcHk0ljb0VUiERanOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVGls7nN&md5=756f687dc30e6fd34004deb672fe8b23</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fjm501262q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501262q%26sid%3Dliteratum%253Aachs%26aulast%3DMenet%26aufirst%3DC.%2BJ.%26aulast%3DFletcher%26aufirst%3DS.%2BR.%26aulast%3DVan%2BLommen%26aufirst%3DG.%26aulast%3DGeney%26aufirst%3DR.%26aulast%3DBlanc%26aufirst%3DJ.%26aulast%3DSmits%26aufirst%3DK.%26aulast%3DJouannigot%26aufirst%3DN.%26aulast%3DDeprez%26aufirst%3DP.%26aulast%3Dvan%2Bder%2BAar%26aufirst%3DE.%2BM.%26aulast%3DClement-Lacroix%26aufirst%3DP.%26aulast%3DLepescheux%26aufirst%3DL.%26aulast%3DGalien%26aufirst%3DR.%26aulast%3DVayssiere%26aufirst%3DB.%26aulast%3DNelles%26aufirst%3DL.%26aulast%3DChristophe%26aufirst%3DT.%26aulast%3DBrys%26aufirst%3DR.%26aulast%3DUhring%26aufirst%3DM.%26aulast%3DCiesielski%26aufirst%3DF.%26aulast%3DVan%2BRompaey%26aufirst%3DL.%26atitle%3DTriazolopyridines%2520as%2520selective%2520JAK1%2520inhibitors%253A%2520From%2520hit%2520identification%2520to%2520GLPG0634%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D9323%26epage%3D9342%26doi%3D10.1021%2Fjm501262q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Genovese, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smolen, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinblatt, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burmester, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meerwein, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Camp, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Othman, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pangan, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jungerwirth, S.</span></span> <span> </span><span class="NLM_article-title">Efficacy and safety of ABT-494, a selective JAK-1 inhibitor, in a phase IIb study in patients with rheumatoid arthritis and an inadequate response to methotrexate</span>. <i>Arthritis Rheumatol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>68</i></span>,  <span class="NLM_fpage">2857</span>– <span class="NLM_lpage">2866</span>, <span class="refDoi"> DOI: 10.1002/art.39808</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=10.1002%2Fart.39808" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=27390150" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFGgsL3I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2016&pages=2857-2866&author=M.+C.+Genoveseauthor=J.+S.+Smolenauthor=M.+E.+Weinblattauthor=G.+R.+Burmesterauthor=S.+Meerweinauthor=H.+L.+Campauthor=L.+Wangauthor=A.+A.+Othmanauthor=N.+Khanauthor=A.+L.+Panganauthor=S.+Jungerwirth&title=Efficacy+and+safety+of+ABT-494%2C+a+selective+JAK-1+inhibitor%2C+in+a+phase+IIb+study+in+patients+with+rheumatoid+arthritis+and+an+inadequate+response+to+methotrexate&doi=10.1002%2Fart.39808"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Efficacy and Safety of ABT-494, a Selective JAK-1 Inhibitor, in a Phase IIb Study in Patients With Rheumatoid Arthritis and an Inadequate Response to Methotrexate</span></div><div class="casAuthors">Genovese, Mark C.; Smolen, Josef S.; Weinblatt, Michael E.; Burmester, Gerd R.; Meerwein, Sebastian; Camp, Heidi S.; Wang, Li; Othman, Ahmed A.; Khan, Nasser; Pangan, Aileen L.; Jungerwirth, Steven</div><div class="citationInfo"><span class="NLM_cas:title">Arthritis & Rheumatology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2857-2866</span>CODEN:
                <span class="NLM_cas:coden">ARRHBO</span>;
        ISSN:<span class="NLM_cas:issn">2326-5191</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Objective : To evaluate the efficacy and safety of ABT-494, a selective JAK-1 inhibitor, in patients with moderate-to-severe rheumatoid arthritis (RA) and an inadequate response to methotrexate (MTX).  Methods : Three hundred RA patients receiving stable doses of MTX were randomly assigned equally to receive immediate-release ABT-494 at 3, 6, 12, or 18 mg twice daily, 24 mg once daily, or placebo for 12 wk.  The primary efficacy end point was the proportion of patients meeting the American College of Rheumatol. 20% improvement criteria (achieving an ACR20 response) at week 12, as detd. using the last observation carried forward method.  Results : At week 12, the proportion of ACR20 responses was higher with ABT-494 (62%, 68%, 80%, 64%, and 76% for the 3, 6, 12, 18, and 24 mg doses, resp.) than with placebo (46%) (using nonresponder imputation) (P < 0.05 for the 6, 12, and 24 mg doses).  There was a significant dose-response relationship among all ABT-494 doses (P < 0.001).  The proportions of patients achieving ACR50 and ACR70 responses were significantly higher for all ABT-494 doses (except the 12 mg dose for the ACR70 response) than for placebo, as were changes in the Disease Activity Score in 28 joints using the C-reactive protein level (DAS28-CRP).  Rapid improvement was demonstrated by significant differences in ACR20 response rates and changes in the DAS28-CRP for all doses compared with placebo at week 2 (the first postbaseline visit).  The incidence of adverse events was similar across groups; most were mild, and infections were the most frequent.  One serious infection (community-acquired pneumonia) occurred with ABT-494 at 12 mg.  There were dose-dependent increases in high-d. lipoprotein (HDL) and low-d. lipoprotein (LDL) cholesterol, but the LDL cholesterol:HDL cholesterol ratios were unchanged through week 12.  Mean Hb levels remained stable at lower doses, but decreases were obsd. at higher doses.  Conclusion : This study evaluated a broad range of doses of ABT-494 in RA patients with an inadequate response to MTX.  ABT-494 demonstrated efficacy, with a safety and tolerability profile similar to that of other JAK inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrc0MsK02mgIbVg90H21EOLACvtfcHk0ljb0VUiERanOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFGgsL3I&md5=5d4bf383496b2153fede781caf6ba0c7</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1002%2Fart.39808&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fart.39808%26sid%3Dliteratum%253Aachs%26aulast%3DGenovese%26aufirst%3DM.%2BC.%26aulast%3DSmolen%26aufirst%3DJ.%2BS.%26aulast%3DWeinblatt%26aufirst%3DM.%2BE.%26aulast%3DBurmester%26aufirst%3DG.%2BR.%26aulast%3DMeerwein%26aufirst%3DS.%26aulast%3DCamp%26aufirst%3DH.%2BL.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DOthman%26aufirst%3DA.%2BA.%26aulast%3DKhan%26aufirst%3DN.%26aulast%3DPangan%26aufirst%3DA.%2BL.%26aulast%3DJungerwirth%26aufirst%3DS.%26atitle%3DEfficacy%2520and%2520safety%2520of%2520ABT-494%252C%2520a%2520selective%2520JAK-1%2520inhibitor%252C%2520in%2520a%2520phase%2520IIb%2520study%2520in%2520patients%2520with%2520rheumatoid%2520arthritis%2520and%2520an%2520inadequate%2520response%2520to%2520methotrexate%26jtitle%3DArthritis%2520Rheumatol.%26date%3D2016%26volume%3D68%26spage%3D2857%26epage%3D2866%26doi%3D10.1002%2Fart.39808" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Williams, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamert, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walden, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilks, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fantino, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossjohn, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucet, I. S.</span></span> <span> </span><span class="NLM_article-title">Dissecting specificity in the Janus kinases: the structures of JAK-specific inhibitors complexed to the JAK1 and JAK2 protein tyrosine kinase domains</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>387</i></span>,  <span class="NLM_fpage">219</span>– <span class="NLM_lpage">232</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2009.01.041</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=10.1016%2Fj.jmb.2009.01.041" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=19361440" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A528%3ADC%252BD1MXivVegtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=387&publication_year=2009&pages=219-232&author=N.+K.+Williamsauthor=R.+S.+Bamertauthor=O.+Patelauthor=C.+Wangauthor=P.+M.+Waldenauthor=A.+F.+Wilksauthor=E.+Fantinoauthor=J.+Rossjohnauthor=I.+S.+Lucet&title=Dissecting+specificity+in+the+Janus+kinases%3A+the+structures+of+JAK-specific+inhibitors+complexed+to+the+JAK1+and+JAK2+protein+tyrosine+kinase+domains&doi=10.1016%2Fj.jmb.2009.01.041"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Dissecting Specificity in the Janus Kinases: The Structures of JAK-Specific Inhibitors Complexed to the JAK1 and JAK2 Protein Tyrosine Kinase Domains</span></div><div class="casAuthors">Williams, Neal K.; Bamert, Rebecca S.; Patel, Onisha; Wang, Christina; Walden, Patricia M.; Wilks, Andrew F.; Fantino, Emmanuelle; Rossjohn, Jamie; Lucet, Isabelle S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">387</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">219-232</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The Janus kinases (JAKs) are a pivotal family of protein tyrosine kinases (PTKs) that play prominent roles in numerous cytokine signaling pathways, with aberrant JAK activity assocd. with a variety of hematopoietic malignancies, cardiovascular diseases and immune-related disorders.  Whereas the structures of the JAK2 and JAK3 PTK domains have been detd., the structure of the JAK1 PTK domain is unknown.  Here, we report the high-resoln. crystal structures of the "active form" of the JAK1 PTK domain in complex with two JAK inhibitors, a tetracyclic pyridone 2-t-butyl-9-fluoro-3,6-dihydro-7H-benz[h]-imidaz[4,5-f]isoquinoline-7-one (CMP6) and (3R,4R)-3-[4-methyl-3-[N-methyl-N-(7H-pyrrolo[2,3-d]pyrimidin-4-yl)amino]piperidin-1-yl]-3-oxopropionitrile (CP-690,550), and compare them with the corresponding JAK2 PTK inhibitor complexes.  Both inhibitors bound in a similar manner to JAK1, namely buried deep within a constricted ATP-binding site, thereby providing a basis for the potent inhibition of JAK1.  As expected, the mode of inhibitor binding in JAK1 was very similar to that obsd. in JAK2, highlighting the challenges in developing JAK-specific inhibitors that target the ATP-binding site.  Nevertheless, differences surrounding the JAK1 and JAK2 ATP-binding sites were apparent, thereby providing a platform for the rational design of JAK2- and JAK1-specific inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQDLkMkCgiLbVg90H21EOLACvtfcHk0lhCi2kxDNxT9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXivVegtbo%253D&md5=79957755d04323057b5acf7e9d2c56ca</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2009.01.041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2009.01.041%26sid%3Dliteratum%253Aachs%26aulast%3DWilliams%26aufirst%3DN.%2BK.%26aulast%3DBamert%26aufirst%3DR.%2BS.%26aulast%3DPatel%26aufirst%3DO.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DWalden%26aufirst%3DP.%2BM.%26aulast%3DWilks%26aufirst%3DA.%2BF.%26aulast%3DFantino%26aufirst%3DE.%26aulast%3DRossjohn%26aufirst%3DJ.%26aulast%3DLucet%26aufirst%3DI.%2BS.%26atitle%3DDissecting%2520specificity%2520in%2520the%2520Janus%2520kinases%253A%2520the%2520structures%2520of%2520JAK-specific%2520inhibitors%2520complexed%2520to%2520the%2520JAK1%2520and%2520JAK2%2520protein%2520tyrosine%2520kinase%2520domains%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2009%26volume%3D387%26spage%3D219%26epage%3D232%26doi%3D10.1016%2Fj.jmb.2009.01.041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alicea-Velazquez, N. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boggon, T. J.</span></span> <span> </span><span class="NLM_article-title">The use of structural biology in Janus kinase targeted drug discovery</span>. <i>Curr. Drug Targets</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">546</span>– <span class="NLM_lpage">555</span>, <span class="refDoi"> DOI: 10.2174/138945011794751528</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=10.2174%2F138945011794751528" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=21126226" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktVOkurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=546-555&author=N.+L.+Alicea-Velazquezauthor=T.+J.+Boggon&title=The+use+of+structural+biology+in+Janus+kinase+targeted+drug+discovery&doi=10.2174%2F138945011794751528"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">The use of structural biology in Janus kinase targeted drug discovery</span></div><div class="casAuthors">Alicea-Velazquez, Nilda L.; Boggon, Titus J.</div><div class="citationInfo"><span class="NLM_cas:title">Current Drug Targets</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">546-555</span>CODEN:
                <span class="NLM_cas:coden">CDTUAU</span>;
        ISSN:<span class="NLM_cas:issn">1389-4501</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  The Janus kinases (or Jak kinases) mediate cytokine and growth factor signal transduction.  Acquired or inherited Jak mutations can result in dysregulation of Jak-mediated signal transduction and can be crit. to disease acquisition in neoplasias including acute myeloid, acute lymphoblastic and acute megakaryoblastic leukemias, and in rare X-linked severe combined immunodeficiency.  The discovery of an acquired Jak2 point mutation, V617F, in significant nos. of patients with classical myeloproliferative disorders has increased the interest in development of Jak2-specific tyrosine kinase inhibitors and consequently there are now over 20 publically available structures of Jak kinase domains that describe all four family members, Jak1, Jak2, Jak3, and Tyk2.  Here we review the recent advances in understanding the druggable structure and function of the Jak family, with a focus on the structural biol. of the Jak kinase domain.  We will discuss how these advances impact the development of Jak-targeted therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJwEtMK-lOu7Vg90H21EOLACvtfcHk0lhCi2kxDNxT9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktVOkurg%253D&md5=5e76fc7eda3ffe580dc4e28044a88c2c</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.2174%2F138945011794751528&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138945011794751528%26sid%3Dliteratum%253Aachs%26aulast%3DAlicea-Velazquez%26aufirst%3DN.%2BL.%26aulast%3DBoggon%26aufirst%3DT.%2BJ.%26atitle%3DThe%2520use%2520of%2520structural%2520biology%2520in%2520Janus%2520kinase%2520targeted%2520drug%2520discovery%26jtitle%3DCurr.%2520Drug%2520Targets%26date%3D2011%26volume%3D12%26spage%3D546%26epage%3D555%26doi%3D10.2174%2F138945011794751528" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chrencik, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patny, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, I. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korniski, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emmons, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinberg, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gormley, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Day, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirsch, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiefer, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leone, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sommers, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, C. -H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobsen, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tenbrink, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomasselli, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benson, E. T.</span></span> <span> </span><span class="NLM_article-title">Structural and thermodynamic characterization of the TYK2 and JAK3 kinase domains in complex with CP-690550 and CMP-6</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>400</i></span>,  <span class="NLM_fpage">413</span>– <span class="NLM_lpage">433</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2010.05.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=10.1016%2Fj.jmb.2010.05.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=20478313" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A528%3ADC%252BC3cXos1ylt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=400&publication_year=2010&pages=413-433&author=J.+E.+Chrencikauthor=A.+Patnyauthor=I.+K.+Leungauthor=B.+Korniskiauthor=T.+L.+Emmonsauthor=T.+Hallauthor=A.+R.+Weinbergauthor=A.+J.+Gormleyauthor=M.+J.+Williamsauthor=E.+J.+Dayauthor=L.+J.+Hirschauthor=R.+J.+Kieferauthor=W.+J.+Leoneauthor=D.+H.+Fischerauthor=D.+C.+Sommersauthor=C.+-H.+Huangauthor=J.+E.+Jacobsenauthor=E.+R.+Tenbrinkauthor=G.+A.+Tomasselliauthor=E.+T.+Benson&title=Structural+and+thermodynamic+characterization+of+the+TYK2+and+JAK3+kinase+domains+in+complex+with+CP-690550+and+CMP-6&doi=10.1016%2Fj.jmb.2010.05.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Structural and Thermodynamic Characterization of the TYK2 and JAK3 Kinase Domains in Complex with CP-690550 and CMP-6</span></div><div class="casAuthors">Chrencik, Jill E.; Patny, Akshay; Leung, Iris K.; Korniski, Brian; Emmons, Thomas L.; Hall, Troii; Weinberg, Robin A.; Gormley, Jennifer A.; Williams, Jennifer M.; Day, Jacqueline E.; Hirsch, Jeffrey L.; Kiefer, James R.; Leone, Joseph W.; Fischer, H. David; Sommers, Cynthia D.; Huang, Horng-Chih; Jacobsen, E. J.; Tenbrink, Ruth E.; Tomasselli, Alfredo G.; Benson, Timothy E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">400</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">413-433</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Janus kinases (JAKs) are crit. regulators of cytokine pathways and attractive targets of therapeutic value in both inflammatory and myeloproliferative diseases.  Although the crystal structures of active JAK1 and JAK2 kinase domains have been reported recently with the clin. compd. CP-690550, the structures of both TYK2 and JAK3 with CP-690550 have remained outstanding.  Here, we report the crystal structures of TYK2, a first in class structure, and JAK3 in complex with PAN-JAK inhibitors CP-690550 ((3R,4R)-3-[4-methyl-3-[N-methyl-N-(7H-pyrrolo[2,3-d]pyrimidin-4-yl)amino]piperidin-1-yl]-3-oxopropionitrile) and CMP-6 (tetracyclic pyridone 2-t-butyl-9-fluoro-3,6-dihydro-7H-benz[h]-imidaz[4,5-f]isoquinoline-7-one), both of which bind in the ATP-binding cavities of both JAK isoenzymes in orientations similar to that obsd. in crystal structures of JAK1 and JAK2.  Addnl., a complete thermodn. characterization of JAK/CP-690550 complex formation was completed by isothermal titrn. calorimetry, indicating the crit. role of the nitrile group from the CP-690550 compd.  Finally, computational anal. using WaterMap further highlights the crit. positioning of the CP-690550 nitrile group in the displacement of an unfavorable water mol. beneath the glycine-rich loop.  Taken together, the data emphasize the outstanding properties of the kinome-selective JAK inhibitor CP-690550, as well as the challenges in obtaining JAK isoenzyme-selective inhibitors due to the overall structural and sequence similarities between the TYK2, JAK1, JAK2 and JAK3 isoenzymes.  Nevertheless, subtle amino acid variations of residues lining the ligand-binding cavity of the JAK enzymes, as well as the global positioning of the glycine-rich loop, might provide the initial clues to obtaining JAK-isoenzyme selective inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5dnGkvq0jPbVg90H21EOLACvtfcHk0lhCi2kxDNxT9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXos1ylt7g%253D&md5=1b1efc3d6f913143dc9113e4ac1c4c60</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2010.05.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2010.05.020%26sid%3Dliteratum%253Aachs%26aulast%3DChrencik%26aufirst%3DJ.%2BE.%26aulast%3DPatny%26aufirst%3DA.%26aulast%3DLeung%26aufirst%3DI.%2BK.%26aulast%3DKorniski%26aufirst%3DB.%26aulast%3DEmmons%26aufirst%3DT.%2BL.%26aulast%3DHall%26aufirst%3DT.%26aulast%3DWeinberg%26aufirst%3DA.%2BR.%26aulast%3DGormley%26aufirst%3DA.%2BJ.%26aulast%3DWilliams%26aufirst%3DM.%2BJ.%26aulast%3DDay%26aufirst%3DE.%2BJ.%26aulast%3DHirsch%26aufirst%3DL.%2BJ.%26aulast%3DKiefer%26aufirst%3DR.%2BJ.%26aulast%3DLeone%26aufirst%3DW.%2BJ.%26aulast%3DFischer%26aufirst%3DD.%2BH.%26aulast%3DSommers%26aufirst%3DD.%2BC.%26aulast%3DHuang%26aufirst%3DC.%2B-H.%26aulast%3DJacobsen%26aufirst%3DJ.%2BE.%26aulast%3DTenbrink%26aufirst%3DE.%2BR.%26aulast%3DTomasselli%26aufirst%3DG.%2BA.%26aulast%3DBenson%26aufirst%3DE.%2BT.%26atitle%3DStructural%2520and%2520thermodynamic%2520characterization%2520of%2520the%2520TYK2%2520and%2520JAK3%2520kinase%2520domains%2520in%2520complex%2520with%2520CP-690550%2520and%2520CMP-6%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2010%26volume%3D400%26spage%3D413%26epage%3D433%26doi%3D10.1016%2Fj.jmb.2010.05.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Newton, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deiana, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Puleo, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cisneros, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cutrona, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlessinger, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jorgensen, W. L.</span></span> <span> </span><span class="NLM_article-title">JAK2 JH2 fluorescence polarization assay and crystal structures for complexes with three small molecules</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">614</span>– <span class="NLM_lpage">617</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.7b00154</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.7b00154" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A528%3ADC%252BC2sXnvFCnsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=614-617&author=A.+S.+Newtonauthor=L.+Deianaauthor=D.+E.+Puleoauthor=J.+A.+Cisnerosauthor=K.+J.+Cutronaauthor=J.+Schlessingerauthor=W.+L.+Jorgensen&title=JAK2+JH2+fluorescence+polarization+assay+and+crystal+structures+for+complexes+with+three+small+molecules&doi=10.1021%2Facsmedchemlett.7b00154"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">JAK2 JH2 Fluorescence Polarization Assay and Crystal Structures for Complexes with Three Small Molecules</span></div><div class="casAuthors">Newton, Ana S.; Deiana, Luca; Puleo, David E.; Cisneros, Jose A.; Cutrona, Kara J.; Schlessinger, Joseph; Jorgensen, William L.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">614-617</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A competitive fluorescence polarization (FP) assay is reported for detg. binding affinities of probe mols. with the pseudokinase JAK2 JH2 allosteric site.  The syntheses of the fluorescent 5 and 6 used in the assay are reported as well as Kd results for 10 compds., including JNJ7706621, NVP-BSK805, and filgotinib (GLPG0634).  X-ray crystal structures of JAK2 JH2 in complex with NVP-BSK805, filgotinib, and diaminopyrimidine 8 elucidate the binding poses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQUy01N09zFrVg90H21EOLACvtfcHk0li5Za-ibLsZsA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXnvFCnsbY%253D&md5=e7c6f7c58bc9a9fddf18a72b5df4f964</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.7b00154&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.7b00154%26sid%3Dliteratum%253Aachs%26aulast%3DNewton%26aufirst%3DA.%2BS.%26aulast%3DDeiana%26aufirst%3DL.%26aulast%3DPuleo%26aufirst%3DD.%2BE.%26aulast%3DCisneros%26aufirst%3DJ.%2BA.%26aulast%3DCutrona%26aufirst%3DK.%2BJ.%26aulast%3DSchlessinger%26aufirst%3DJ.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26atitle%3DJAK2%2520JH2%2520fluorescence%2520polarization%2520assay%2520and%2520crystal%2520structures%2520for%2520complexes%2520with%2520three%2520small%2520molecules%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D8%26spage%3D614%26epage%3D617%26doi%3D10.1021%2Facsmedchemlett.7b00154" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Flanagan, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blumenkopf, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brissette, W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casavant, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shang-Poa, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doty, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elliott, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fisher, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kent, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kudlacz, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lillie, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magnuson, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCurdy, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munchhof, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perry, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sawyer, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strelevitz, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Subramanyam, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whipple, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Changelian, P. S.</span></span> <span> </span><span class="NLM_article-title">Discovery of CP-690,550: a potent and selective Janus kinase (JAK) inhibitor for the treatment of autoimmune diseases and organ transplant rejection</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">8468</span>– <span class="NLM_lpage">8484</span>, <span class="refDoi"> DOI: 10.1021/jm1004286</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm1004286" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVOgt7vP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=8468-8484&author=M.+E.+Flanaganauthor=T.+A.+Blumenkopfauthor=W.+H.+Brissetteauthor=M.+F.+Brownauthor=J.+M.+Casavantauthor=C.+Shang-Poaauthor=J.+L.+Dotyauthor=E.+A.+Elliottauthor=M.+B.+Fisherauthor=M.+Hinesauthor=C.+Kentauthor=E.+M.+Kudlaczauthor=B.+M.+Lillieauthor=K.+S.+Magnusonauthor=S.+P.+McCurdyauthor=M.+J.+Munchhofauthor=B.+D.+Perryauthor=P.+S.+Sawyerauthor=T.+J.+Strelevitzauthor=C.+Subramanyamauthor=J.+Sunauthor=D.+A.+Whippleauthor=P.+S.+Changelian&title=Discovery+of+CP-690%2C550%3A+a+potent+and+selective+Janus+kinase+%28JAK%29+inhibitor+for+the+treatment+of+autoimmune+diseases+and+organ+transplant+rejection&doi=10.1021%2Fjm1004286"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of CP-690,550: A Potent and Selective Janus Kinase (JAK) Inhibitor for the Treatment of Autoimmune Diseases and Organ Transplant Rejection</span></div><div class="casAuthors">Flanagan, Mark E.; Blumenkopf, Todd A.; Brissette, William H.; Brown, Matthew F.; Casavant, Jeffrey M.; Chang, Shang-Poa; Doty, Jonathan L.; Elliott, Eileen A.; Fisher, Michael B.; Hines, Michael; Kent, Craig; Kudlacz, Elizabeth M.; Lillie, Brett M.; Magnuson, Kelly S.; McCurdy, Sandra P.; Munchhof, Michael J.; Perry, Bret D.; Sawyer, Perry S.; Strelevitz, Timothy J.; Subramanyam, Chakrapani; Sun, Jianmin; Whipple, David A.; Changelian, Paul S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">8468-8484</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">There is a crit. need for safer and more convenient treatments for organ transplant rejection and autoimmune disorders such as rheumatoid arthritis.  Janus tyrosine kinases (JAK1, JAK3) are expressed in lymphoid cells and are involved in the signaling of multiple cytokines important for various T cell functions.  Blockade of the JAK1/JAK3-STAT pathway with a small mol. was anticipated to provide therapeutic immunosuppression/immunomodulation.  The Pfizer compd. library was screened against the catalytic domain of JAK3 resulting in the identification of a pyrrolopyrimidine-based series of inhibitors represented by the hexahydrocarbazolyl pyrrolopyrimidine CP-352,664.  Synthetic analogs of CP-352,664 were screened against the JAK enzymes and evaluated in an IL-2 induced T cell blast proliferation assay.  Select compds. were evaluated in rodent efficacy models of allograft rejection and destructive inflammatory arthritis.  Optimization within this chem. series led to identification of the cyanoacetylpiperidinylamino pyrrolopyrimidine CP-690,550, a potential first-in-class JAK inhibitor for treatment of autoimmune diseases and organ transplant rejection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbFS_8DSHkDbVg90H21EOLACvtfcHk0li5Za-ibLsZsA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVOgt7vP&md5=c1d31ce42ab3f1c1f3029c6f4f48df1a</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fjm1004286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm1004286%26sid%3Dliteratum%253Aachs%26aulast%3DFlanagan%26aufirst%3DM.%2BE.%26aulast%3DBlumenkopf%26aufirst%3DT.%2BA.%26aulast%3DBrissette%26aufirst%3DW.%2BH.%26aulast%3DBrown%26aufirst%3DM.%2BF.%26aulast%3DCasavant%26aufirst%3DJ.%2BM.%26aulast%3DShang-Poa%26aufirst%3DC.%26aulast%3DDoty%26aufirst%3DJ.%2BL.%26aulast%3DElliott%26aufirst%3DE.%2BA.%26aulast%3DFisher%26aufirst%3DM.%2BB.%26aulast%3DHines%26aufirst%3DM.%26aulast%3DKent%26aufirst%3DC.%26aulast%3DKudlacz%26aufirst%3DE.%2BM.%26aulast%3DLillie%26aufirst%3DB.%2BM.%26aulast%3DMagnuson%26aufirst%3DK.%2BS.%26aulast%3DMcCurdy%26aufirst%3DS.%2BP.%26aulast%3DMunchhof%26aufirst%3DM.%2BJ.%26aulast%3DPerry%26aufirst%3DB.%2BD.%26aulast%3DSawyer%26aufirst%3DP.%2BS.%26aulast%3DStrelevitz%26aufirst%3DT.%2BJ.%26aulast%3DSubramanyam%26aufirst%3DC.%26aulast%3DSun%26aufirst%3DJ.%26aulast%3DWhipple%26aufirst%3DD.%2BA.%26aulast%3DChangelian%26aufirst%3DP.%2BS.%26atitle%3DDiscovery%2520of%2520CP-690%252C550%253A%2520a%2520potent%2520and%2520selective%2520Janus%2520kinase%2520%2528JAK%2529%2520inhibitor%2520for%2520the%2520treatment%2520of%2520autoimmune%2520diseases%2520and%2520organ%2520transplant%2520rejection%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D8468%26epage%3D8484%26doi%3D10.1021%2Fjm1004286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Noji, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hara, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamanaka, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maeda, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hori, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Obika, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inoue, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hase, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orita, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adachi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanimoto, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oki, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kimoto, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogawa, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Negoro, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hashimoto, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shiozaki, M.</span></span> <span> </span><span class="NLM_article-title">Discovery of a Janus kinase inhibitor bearing a highly three-dimensional spiro scaffold: JTE-052 (delgocitinib) as a new dermatological agent to treat inflammatory skin disorders</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">7163</span>– <span class="NLM_lpage">7185</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00450</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00450" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtFWmsLrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=7163-7185&author=S.+Nojiauthor=Y.+Haraauthor=T.+Miuraauthor=H.+Yamanakaauthor=K.+Maedaauthor=A.+Horiauthor=H.+Yamamotoauthor=S.+Obikaauthor=M.+Inoueauthor=Y.+Haseauthor=T.+Oritaauthor=S.+Doiauthor=T.+Adachiauthor=A.+Tanimotoauthor=C.+Okiauthor=Y.+Kimotoauthor=Y.+Ogawaauthor=T.+Negoroauthor=H.+Hashimotoauthor=M.+Shiozaki&title=Discovery+of+a+Janus+kinase+inhibitor+bearing+a+highly+three-dimensional+spiro+scaffold%3A+JTE-052+%28delgocitinib%29+as+a+new+dermatological+agent+to+treat+inflammatory+skin+disorders&doi=10.1021%2Facs.jmedchem.0c00450"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Janus Kinase Inhibitor Bearing a Highly Three-Dimensional Spiro Scaffold: JTE-052 (Delgocitinib) as a New Dermatological Agent to Treat Inflammatory Skin Disorders</span></div><div class="casAuthors">Noji, Satoru; Hara, Yoshinori; Miura, Tomoya; Yamanaka, Hiroshi; Maeda, Katsuya; Hori, Akimi; Yamamoto, Hiroshi; Obika, Shingo; Inoue, Masafumi; Hase, Yasunori; Orita, Takuya; Doi, Satoki; Adachi, Tsuyoshi; Tanimoto, Atsuo; Oki, Chika; Kimoto, Yukari; Ogawa, Yoshihiro; Negoro, Tamotsu; Hashimoto, Hiromasa; Shiozaki, Makoto</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">7163-7185</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Dermatol. disorders such as atopic dermatitis arise from genetic and environmental causes and are complex and multifactorial in nature.  Among possible risk factors, aberrant immunol. reactions are one of the leading etiologies.  Immunosuppressive agents including topical steroids are common treatments for these disorders.  Despite their reliability in clin. settings, topical steroids display side effects, typified by skin thinning.  Accordingly, there is a need for alternate effective and well-tolerated therapies.  As part of our efforts to investigate new immunomodulators, we have developed a series of JAK inhibitors, which incorporate novel three-dimensional spiro motifs and unexpectedly possess both excellent physicochem. properties and antidermatitis efficacy in the animal models.  One of these compds., JTE-052 (ent-60), also known as delgocitinib, has been shown to be effective and well-tolerated in human clin. trials and has recently been approved in Japan for the treatment of atopic dermatitis as the first drug among Janus kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqa4KSm53XHzLVg90H21EOLACvtfcHk0ljT-nro0W1WuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtFWmsLrM&md5=f4eecf28446abb441f1436c41d8cb0a6</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00450&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00450%26sid%3Dliteratum%253Aachs%26aulast%3DNoji%26aufirst%3DS.%26aulast%3DHara%26aufirst%3DY.%26aulast%3DMiura%26aufirst%3DT.%26aulast%3DYamanaka%26aufirst%3DH.%26aulast%3DMaeda%26aufirst%3DK.%26aulast%3DHori%26aufirst%3DA.%26aulast%3DYamamoto%26aufirst%3DH.%26aulast%3DObika%26aufirst%3DS.%26aulast%3DInoue%26aufirst%3DM.%26aulast%3DHase%26aufirst%3DY.%26aulast%3DOrita%26aufirst%3DT.%26aulast%3DDoi%26aufirst%3DS.%26aulast%3DAdachi%26aufirst%3DT.%26aulast%3DTanimoto%26aufirst%3DA.%26aulast%3DOki%26aufirst%3DC.%26aulast%3DKimoto%26aufirst%3DY.%26aulast%3DOgawa%26aufirst%3DY.%26aulast%3DNegoro%26aufirst%3DT.%26aulast%3DHashimoto%26aufirst%3DH.%26aulast%3DShiozaki%26aufirst%3DM.%26atitle%3DDiscovery%2520of%2520a%2520Janus%2520kinase%2520inhibitor%2520bearing%2520a%2520highly%2520three-dimensional%2520spiro%2520scaffold%253A%2520JTE-052%2520%2528delgocitinib%2529%2520as%2520a%2520new%2520dermatological%2520agent%2520to%2520treat%2520inflammatory%2520skin%2520disorders%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D7163%26epage%3D7185%26doi%3D10.1021%2Facs.jmedchem.0c00450" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nakajima, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inoue, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakai, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mukoyoshi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamaguchi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatanaka, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sasaki, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kunikawa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Usuda, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moritomo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Higashi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inami, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shirakam, S.</span></span> <span> </span><span class="NLM_article-title">Synthesis and evaluation of novel 1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-5-carboxamide derivatives as potent and orally efficacious immune modulators targeting</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">4871</span>– <span class="NLM_lpage">4883</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2015.05.034</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=10.1016%2Fj.bmc.2015.05.034" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=26071372" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A528%3ADC%252BC2MXptFOktrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2015&pages=4871-4883&author=Y.+Nakajimaauthor=T.+Inoueauthor=K.+Nakaiauthor=K.+Mukoyoshiauthor=H.+Hamaguchiauthor=K.+Hatanakaauthor=H.+Sasakiauthor=A.+Tanakaauthor=F.+Takahashiauthor=S.+Kunikawaauthor=H.+Usudaauthor=A.+Moritomoauthor=Y.+Higashiauthor=M.+Inamiauthor=S.+Shirakam&title=Synthesis+and+evaluation+of+novel+1H-pyrrolo%5B2%2C3-b%5Dpyridine-5-carboxamide+derivatives+as+potent+and+orally+efficacious+immune+modulators+targeting&doi=10.1016%2Fj.bmc.2015.05.034"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and evaluation of novel 1H-pyrrolo[2,3-b]pyridine-5-carboxamide derivatives as potent and orally efficacious immunomodulators targeting JAK3</span></div><div class="casAuthors">Nakajima, Yutaka; Inoue, Takayuki; Nakai, Kazuo; Mukoyoshi, Koichiro; Hamaguchi, Hisao; Hatanaka, Keiko; Sasaki, Hiroshi; Tanaka, Akira; Takahashi, Fumie; Kunikawa, Shigeki; Usuda, Hiroyuki; Moritomo, Ayako; Higashi, Yasuyuki; Inami, Masamichi; Shirakami, Shohei</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">4871-4883</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Janus kinases (JAKs) regulate various inflammatory and immune responses and are targets for the treatment of inflammatory and immune diseases.  As a novel class of immunomodulators targeting JAK3, 1H-pyrrolo[2,3-b]pyridine-5-carboxamide derivs. are promising candidates for treating such diseases.  In chem. modification of lead compd. (I), the substitution of a cycloalkyl ring for an N-cyanopyridylpiperidine in C4-position was effective for increasing JAK3 inhibitory activity.  In addn., modulation of phys. properties such as mol. lipophilicity and basicity was important for reducing human ether-a-go-go-related gene (hERG) inhibitory activity.  The authors' optimization study gave compd. 31 (4-[[(3S,4R)-1-(5-cyanopyridin-2-yl)-3-fluoropiperidin-4-yl]amino]-1H-pyrrolo[2,3-b]pyridine-5-carboxamide), which exhibited potent JAK3 inhibitory activity as well as weak hERG inhibitory activity.  In cellular assay, 31 exhibited potent immunomodulating effect on IL-2-stimulated T cell proliferation.  In a pharmacokinetic study, good metabolic stability and oral bioavailability of 31 were achieved in rats, dogs, and monkeys.  Further, 31 prolonged graft survival in an in vivo rat heterotopic cardiac transplant model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqc-PiU8OQJ_bVg90H21EOLACvtfcHk0ljT-nro0W1WuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXptFOktrc%253D&md5=378ff172c2374ea40d784c59f1e192d7</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2015.05.034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2015.05.034%26sid%3Dliteratum%253Aachs%26aulast%3DNakajima%26aufirst%3DY.%26aulast%3DInoue%26aufirst%3DT.%26aulast%3DNakai%26aufirst%3DK.%26aulast%3DMukoyoshi%26aufirst%3DK.%26aulast%3DHamaguchi%26aufirst%3DH.%26aulast%3DHatanaka%26aufirst%3DK.%26aulast%3DSasaki%26aufirst%3DH.%26aulast%3DTanaka%26aufirst%3DA.%26aulast%3DTakahashi%26aufirst%3DF.%26aulast%3DKunikawa%26aufirst%3DS.%26aulast%3DUsuda%26aufirst%3DH.%26aulast%3DMoritomo%26aufirst%3DA.%26aulast%3DHigashi%26aufirst%3DY.%26aulast%3DInami%26aufirst%3DM.%26aulast%3DShirakam%26aufirst%3DS.%26atitle%3DSynthesis%2520and%2520evaluation%2520of%2520novel%25201H-pyrrolo%255B2%252C3-b%255Dpyridine-5-carboxamide%2520derivatives%2520as%2520potent%2520and%2520orally%2520efficacious%2520immune%2520modulators%2520targeting%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2015%26volume%3D23%26spage%3D4871%26epage%3D4883%26doi%3D10.1016%2Fj.bmc.2015.05.034" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schönherr, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cernak, T.</span></span> <span> </span><span class="NLM_article-title">Profound methyl effects in drug discovery and a call for new C-H methylation reactions</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">12256</span>– <span class="NLM_lpage">12267</span>, <span class="refDoi"> DOI: 10.1002/anie.201303207</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=10.1002%2Fanie.201303207" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A280%3ADC%252BC2c%252FptVahuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2013&pages=12256-12267&author=H.+Sch%C3%B6nherrauthor=T.+Cernak&title=Profound+methyl+effects+in+drug+discovery+and+a+call+for+new+C-H+methylation+reactions&doi=10.1002%2Fanie.201303207"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Profound methyl effects in drug discovery and a call for new C-H methylation reactions</span></div><div class="casAuthors">Schonherr Heike; Cernak Tim</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie (International ed. in English)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">47</span>),
    <span class="NLM_cas:pages">12256-67</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The methyl group is one of the most commonly occurring carbon fragments in small-molecule drugs.  This simplest alkyl fragment appears in more than 67 % of the top-selling drugs of 2011 and can modulate both the biological and physical properties of a molecule.  This Review focuses on so-called magic methyl effects on binding potency, where the seemingly mundane change of C H to C Me improves the IC50  value of a drug candidate more than 100-fold.  This discussion is followed by a survey of recent advances in synthetic chemistry that allow the direct methylation of C(sp(2) ) H and C(sp(3) ) H bonds.  It is our hope that the relevance of the meager methyl group to drug discovery as presented herein will inspire reports on new C H methylation reactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTyoLZDDgvsMh0LsK7uFuY7fW6udTcc2ebLOujub890Mbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2c%252FptVahuw%253D%253D&md5=ef08f80e55b2c530b49e4306da024902</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1002%2Fanie.201303207&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201303207%26sid%3Dliteratum%253Aachs%26aulast%3DSch%25C3%25B6nherr%26aufirst%3DH.%26aulast%3DCernak%26aufirst%3DT.%26atitle%3DProfound%2520methyl%2520effects%2520in%2520drug%2520discovery%2520and%2520a%2520call%2520for%2520new%2520C-H%2520methylation%2520reactions%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2013%26volume%3D52%26spage%3D12256%26epage%3D12267%26doi%3D10.1002%2Fanie.201303207" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sherman, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Day, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farid, R.</span></span> <span> </span><span class="NLM_article-title">Novel procedure for modeling ligand/receptor induced fit effects</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">534</span>– <span class="NLM_lpage">553</span>, <span class="refDoi"> DOI: 10.1021/jm050540c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm050540c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlCgsr7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=534-553&author=W.+Shermanauthor=T.+Dayauthor=M.+P.+Jacobsonauthor=R.+A.+Friesnerauthor=R.+Farid&title=Novel+procedure+for+modeling+ligand%2Freceptor+induced+fit+effects&doi=10.1021%2Fjm050540c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Procedure for Modeling Ligand/Receptor Induced Fit Effects</span></div><div class="casAuthors">Sherman, Woody; Day, Tyler; Jacobson, Matthew P.; Friesner, Richard A.; Farid, Ramy</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">534-553</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We present a novel protein-ligand docking method that accurately accounts for both ligand and receptor flexibility by iteratively combining rigid receptor docking (Glide) with protein structure prediction (Prime) techniques.  While traditional rigid-receptor docking methods are useful when the receptor structure does not change substantially upon ligand binding, success is limited when the protein must be "induced" into the correct binding conformation for a given ligand.  We provide an in-depth description of our novel methodol. and present results for 21 pharmaceutically relevant examples.  Traditional rigid-receptor docking for these 21 cases yields an av. RMSD of 5.5 Å.  The av. ligand RMSD for docking to a flexible receptor for the 21 pairs is 1.4 Å; the RMSD is ≤1.8 Å for 18 of the cases.  For the three cases with RMSDs greater than 1.8 Å, the core of the ligand is properly docked and all key protein/ligand interactions are captured.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqm3E6ctnPGSrVg90H21EOLACvtfcHk0ljzMpj1Te9V5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtlCgsr7I&md5=388811ead5cee1fd460951263de486cb</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fjm050540c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm050540c%26sid%3Dliteratum%253Aachs%26aulast%3DSherman%26aufirst%3DW.%26aulast%3DDay%26aufirst%3DT.%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DFarid%26aufirst%3DR.%26atitle%3DNovel%2520procedure%2520for%2520modeling%2520ligand%252Freceptor%2520induced%2520fit%2520effects%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D534%26epage%3D553%26doi%3D10.1021%2Fjm050540c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Di, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerns, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourassa, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huryn, D. M.</span></span> <span> </span><span class="NLM_article-title">Experimental design on single-time-point high-throughput microsomal stability assay</span>. <i>J. Pharm. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>93</i></span>,  <span class="NLM_fpage">1537</span>– <span class="NLM_lpage">1544</span>, <span class="refDoi"> DOI: 10.1002/jps.20076</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=10.1002%2Fjps.20076" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=15124211" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A528%3ADC%252BD2cXks12hsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=2004&pages=1537-1544&author=L.+Diauthor=E.+H.+Kernsauthor=N.+Gaoauthor=S.+Q.+Liauthor=Y.+Huangauthor=J.+L.+Bourassaauthor=D.+M.+Huryn&title=Experimental+design+on+single-time-point+high-throughput+microsomal+stability+assay&doi=10.1002%2Fjps.20076"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Experimental design on single-time-point high-throughput microsomal stability assay</span></div><div class="casAuthors">Di, Li; Kerns, Edward H.; Gao, Ning; Li, Susan Q.; Huang, Youping; Bourassa, Jim L.; Huryn, Donna M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmaceutical Sciences</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1537-1544</span>CODEN:
                <span class="NLM_cas:coden">JPMSAE</span>;
        ISSN:<span class="NLM_cas:issn">0022-3549</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">An exptl. design for a single-time-point microsomal stability assay was evaluated as compared with multiple-time-point studies.  Results obtained from single-time-point expts. are in excellent agreement with those from multiple time points.  First-order reaction kinetics revealed rapid changes of predicted half-life from percent remaining of the parent compd. at the inflection points, suggesting a max. predictive limit for half-life.  Selection of the incubation time in single-time-point assays is important to obtain balanced information for stable and unstable compds.  A short incubation time (e.g., 5 min) is most useful for differentiating between unstable compds., which is beneficial to direct the synthetic efforts in projects with poor metabolic stability.  A long incubation time (e.g., 30 min) is more applicable to a compd. series with high metabolic stability.  For screening purposes, a moderate incubation time (e.g., 15 min) is recommended to achieve good resoln. and a sufficiently high max. predictive limit for half-life.  This study suggests that a single-time-point assay is sufficient for ranking compds. in early drug discovery.  It increases throughput and reduces turnaround time and cost.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdHPH7tWzrIrVg90H21EOLACvtfcHk0ljzMpj1Te9V5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXks12hsr4%253D&md5=45b3bfb054ad93e18655b988fdbbe89b</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1002%2Fjps.20076&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjps.20076%26sid%3Dliteratum%253Aachs%26aulast%3DDi%26aufirst%3DL.%26aulast%3DKerns%26aufirst%3DE.%2BH.%26aulast%3DGao%26aufirst%3DN.%26aulast%3DLi%26aufirst%3DS.%2BQ.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DBourassa%26aufirst%3DJ.%2BL.%26aulast%3DHuryn%26aufirst%3DD.%2BM.%26atitle%3DExperimental%2520design%2520on%2520single-time-point%2520high-throughput%2520microsomal%2520stability%2520assay%26jtitle%3DJ.%2520Pharm.%2520Sci.%26date%3D2004%26volume%3D93%26spage%3D1537%26epage%3D1544%26doi%3D10.1002%2Fjps.20076" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cha, I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shon, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shin, J. G.</span></span> <span> </span><span class="NLM_article-title">High-throughput screening of inhibitory potential of nine cytochrome P450 enzymes in vitro using liquid chromatography/tandem mass spectrometry</span>. <i>Rapid Commun. Mass Spectrom.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">2651</span>– <span class="NLM_lpage">2658</span>, <span class="refDoi"> DOI: 10.1002/rcm.2110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=10.1002%2Frcm.2110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=16124035" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtVKntr%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2005&pages=2651-2658&author=M.+J.+Kimauthor=H.+Kimauthor=I.+J.+Chaauthor=J.+S.+Parkauthor=J.+H.+Shonauthor=K.+H.+Liuauthor=J.+G.+Shin&title=High-throughput+screening+of+inhibitory+potential+of+nine+cytochrome+P450+enzymes+in+vitro+using+liquid+chromatography%2Ftandem+mass+spectrometry&doi=10.1002%2Frcm.2110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">High-throughput screening of inhibitory potential of nine cytochrome P450 enzymes in vitro using liquid chromatography/tandem mass spectrometry</span></div><div class="casAuthors">Kim, Min-Jung; Kim, Hyojin; Cha, In-June; Park, Jang-Su; Shon, Ji-Hong; Liu, Kwang-Hyeon; Shin, Jae-Gook</div><div class="citationInfo"><span class="NLM_cas:title">Rapid Communications in Mass Spectrometry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">2651-2658</span>CODEN:
                <span class="NLM_cas:coden">RCMSEF</span>;
        ISSN:<span class="NLM_cas:issn">0951-4198</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons Ltd.</span>)
        </div><div class="casAbstract">The early detection of potential drug-drug interactions is an important issue of drug discovery that has led to the development of high-throughput screening (HTS) methods for potential drug interactions.  We developed a HTS method for potential interactions of inhibitory drugs for nine human P 450 enzymes using cocktail incubation and tandem mass spectrometry in vitro.  This new method involves incubation of two cocktail doses and single cassette anal.  The two cocktail doses in vitro were developed to minimize solvent effects and mutual drug interactions among substrates: cocktail A was composed of phenacetin for CYP1A2, coumarin for CYP2A6, paclitaxel for CYP2C8, S-mephenytoin for CYP2C19, dextromethorphan for CYP2D6, and midazolam for CYP3A4; and cocktail B was composed of three substrates including bupropion for CYP2B6, tolbutamide for CYP2C9, and chlorzoxazone for CYP2E1.  In the incubation study of these cocktails, the reaction mixts. were pooled and simultaneously analyzed using liq. chromatog./tandem mass spectrometry employing a fast gradient.  The method was validated by comparing the inhibition data obtained from the incubation of each individual probe substrate alone with data from the new method.  The IC50 value of each inhibitor in the cocktail agreed well with that of the individual probe drug as well as with values previously reported in the literature.  As a HTS method for potential interactions of the inhibition of these nine P 450 enzymes, this new method will be useful in the drug discovery process and for the mechanistic understanding of drug interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLoIA81FWoWrVg90H21EOLACvtfcHk0ljPRY80LbbnGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtVKntr%252FJ&md5=38747a0def46d55e0e6223ac234772cd</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1002%2Frcm.2110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Frcm.2110%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DM.%2BJ.%26aulast%3DKim%26aufirst%3DH.%26aulast%3DCha%26aufirst%3DI.%2BJ.%26aulast%3DPark%26aufirst%3DJ.%2BS.%26aulast%3DShon%26aufirst%3DJ.%2BH.%26aulast%3DLiu%26aufirst%3DK.%2BH.%26aulast%3DShin%26aufirst%3DJ.%2BG.%26atitle%3DHigh-throughput%2520screening%2520of%2520inhibitory%2520potential%2520of%2520nine%2520cytochrome%2520P450%2520enzymes%2520in%2520vitro%2520using%2520liquid%2520chromatography%252Ftandem%2520mass%2520spectrometry%26jtitle%3DRapid%2520Commun.%2520Mass%2520Spectrom.%26date%3D2005%26volume%3D19%26spage%3D2651%26epage%3D2658%26doi%3D10.1002%2Frcm.2110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Di, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerns, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span> <span> </span><span class="NLM_article-title">Development and application of high throughput plasma stability assay for drug discovery</span>. <i>Int. J. Pharm.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>297</i></span>,  <span class="NLM_fpage">110</span>– <span class="NLM_lpage">119</span>, <span class="refDoi"> DOI: 10.1016/j.ijpharm.2005.03.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=10.1016%2Fj.ijpharm.2005.03.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=15876500" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A528%3ADC%252BD2MXkt1ansrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=297&publication_year=2005&pages=110-119&author=L.+Diauthor=E.+H.+Kernsauthor=Y.+Hongauthor=H.+Chen&title=Development+and+application+of+high+throughput+plasma+stability+assay+for+drug+discovery&doi=10.1016%2Fj.ijpharm.2005.03.022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Development and application of high throughput plasma stability assay for drug discovery</span></div><div class="casAuthors">Di, Li; Kerns, Edward H.; Hong, Yan; Chen, Hong</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Pharmaceutics</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">297</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">110-119</span>CODEN:
                <span class="NLM_cas:coden">IJPHDE</span>;
        ISSN:<span class="NLM_cas:issn">0378-5173</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Plasma stability plays an important role in drug discovery and development.  Unstable compds. tend to have rapid clearance and short half-life, resulting in poor in vivo performance.  This paper examines the variables that affect the plasma stability assay results, including substrate concn., %DMSO, plasma concn., enzyme activity upon incubation and batch variation.  The results show that plasma stability can accommodate a wide range of exptl. conditions.  Relatively minor differences in results are produced with major differences in conditions.  Significant batch-to-batch variations were obsd. for rat plasma.  We selected the following conditions: 1 μM substrate concn., 2.5% DMSO, and 50% diln. of plasma in pH 7.4 buffer.  Plasma stability can be used as a diagnostic assay when compds. are unexpectedly rapidly cleared, as a special assay when structural classes contain groups that may be susceptible to plasma enzyme hydrolysis, or as general screen for compds. if resources are available.  Plasma stability assay has many applications in drug discovery: to alert teams to labile structural motifs, to prioritize compds. for in vivo studies and to screen prodrugs and antedrugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZUoDVyphu6bVg90H21EOLACvtfcHk0ljPRY80LbbnGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXkt1ansrg%253D&md5=cee758567f2d75838b28817a1040f855</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2Fj.ijpharm.2005.03.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ijpharm.2005.03.022%26sid%3Dliteratum%253Aachs%26aulast%3DDi%26aufirst%3DL.%26aulast%3DKerns%26aufirst%3DE.%2BH.%26aulast%3DHong%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DH.%26atitle%3DDevelopment%2520and%2520application%2520of%2520high%2520throughput%2520plasma%2520stability%2520assay%2520for%2520drug%2520discovery%26jtitle%3DInt.%2520J.%2520Pharm.%26date%3D2005%26volume%3D297%26spage%3D110%26epage%3D119%26doi%3D10.1016%2Fj.ijpharm.2005.03.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Irvine, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lockhart, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tolan, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Selick, H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grove, J. R.</span></span> <span> </span><span class="NLM_article-title">MDCK (Madin–Darby canine kidney) cells: a tool for membrane permeability screening</span>. <i>J. Pharm. Sci.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>88</i></span>,  <span class="NLM_fpage">28</span>– <span class="NLM_lpage">33</span>, <span class="refDoi"> DOI: 10.1021/js9803205</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=10.1021%2Fjs9803205" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=9874698" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A528%3ADyaK1cXns1ylurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=88&publication_year=1999&pages=28-33&author=J.+D.+Irvineauthor=L.+Takahashiauthor=K.+Lockhartauthor=J.+Cheongauthor=J.+W.+Tolanauthor=H.+E.+Selickauthor=J.+R.+Grove&title=MDCK+%28Madin%E2%80%93Darby+canine+kidney%29+cells%3A+a+tool+for+membrane+permeability+screening&doi=10.1021%2Fjs9803205"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">MDCK (Madin-Darby canine kidney) cells: a tool for membrane permeability screening</span></div><div class="casAuthors">Irvine, Jennifer D.; Takahashi, Lori; Lockhart, Karen; Cheong, Jonathan; Tolan, John W.; Selick, H. E.; Grove, J. Russell</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmaceutical Sciences</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">88</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">28-33</span>CODEN:
                <span class="NLM_cas:coden">JPMSAE</span>;
        ISSN:<span class="NLM_cas:issn">0022-3549</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The goal of this work was to investigate the use of MDCK (Madin-Darby canine kidney) cells as a possible tool for assessing the membrane permeability properties of early drug discovery compds.  Apparent permeability (Papp) values of 55 compds. with known human absorption values were detd. using MDCK cell monolayers.  For comparison, Papp values of the same compds. were also detd. using Caco-2 cells, a well-characterized in vitro model of intestinal drug absorption.  Monolayers were grown on 0.4-μm Transwell-COL membrane culture inserts. MDCK cells were seeded at high d. and cultured for 3 days, and Caco-2 cells were cultured under std. conditions for 21 to 25 days.  Compds. were tested using 100 μM donor solns. in transport medium (pH 7.4) contg. 1% DMSO.  The Papp values in MDCK cells correlated well with those in Caco-2 cells (r2 = 0.79). Spearman's rank correlation coeff. for MDCK Papp and human absorption was 0.58 compared with 0.54 for Caco-2 Papp and human absorption.  These results indicate that MDCK cells may be a useful tool for rapid membrane permeability screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolDvo0FQMg7bVg90H21EOLACvtfcHk0ljPRY80LbbnGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXns1ylurk%253D&md5=ea421117433fa4067578476edde9444e</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fjs9803205&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjs9803205%26sid%3Dliteratum%253Aachs%26aulast%3DIrvine%26aufirst%3DJ.%2BD.%26aulast%3DTakahashi%26aufirst%3DL.%26aulast%3DLockhart%26aufirst%3DK.%26aulast%3DCheong%26aufirst%3DJ.%26aulast%3DTolan%26aufirst%3DJ.%2BW.%26aulast%3DSelick%26aufirst%3DH.%2BE.%26aulast%3DGrove%26aufirst%3DJ.%2BR.%26atitle%3DMDCK%2520%2528Madin%25E2%2580%2593Darby%2520canine%2520kidney%2529%2520cells%253A%2520a%2520tool%2520for%2520membrane%2520permeability%2520screening%26jtitle%3DJ.%2520Pharm.%2520Sci.%26date%3D1999%26volume%3D88%26spage%3D28%26epage%3D33%26doi%3D10.1021%2Fjs9803205" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, H.</span></span> <span> </span><span class="NLM_article-title">Protein kinase profiling assays: a technology review</span>. <i>Drug Discovery Today Technol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">8</span>, <span class="refDoi"> DOI: 10.1016/j.ddtec.2015.10.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=10.1016%2Fj.ddtec.2015.10.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=26723886" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A280%3ADC%252BC28rms1Wisw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2015&pages=1-8&author=Y.+Wangauthor=H.+Ma&title=Protein+kinase+profiling+assays%3A+a+technology+review&doi=10.1016%2Fj.ddtec.2015.10.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Protein kinase profiling assays: a technology review</span></div><div class="casAuthors">Wang Yuren; Ma Haiching</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today. Technologies</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1-8</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Protein kinases have become one of the most intensively pursued classes of drug targets for many diseases such as cancers and inflammatory diseases.  Kinase profiling work seeks to understand general selectivity trends of lead compounds across the kinome, which help with target selection, compound prioritization, and potential implications in toxicity.  Under the current drug discovery process, screening of compounds against comprehensive panels of kinases and their mutants has become the standard approach.  Many screening assays and technologies which are compatible for high-throughput screening (HTS) against kinases have been extensively pursued and developed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ2PU1XOycdfMve_zCu79I8fW6udTcc2eZmVs-PaGLK3bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28rms1Wisw%253D%253D&md5=89bc86678797ea211e5a2e4b6a7da709</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2015.10.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2015.10.007%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DH.%26atitle%3DProtein%2520kinase%2520profiling%2520assays%253A%2520a%2520technology%2520review%26jtitle%3DDrug%2520Discovery%2520Today%2520Technol.%26date%3D2015%26volume%3D18%26spage%3D1%26epage%3D8%26doi%3D10.1016%2Fj.ddtec.2015.10.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chakraborty, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Šumová, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mallano, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Distler, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergmann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ludolph, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horch, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gelse, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramming, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Distler, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schett, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Šenolt, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Distler, J. H. W.</span></span> <span> </span><span class="NLM_article-title">Activation of STAT3 integrates common profibrotic pathways to promote fibroblast activation and tissue fibrosTransforming Growth Factor-β (TGF-β) Directly Activates the JAK1-STAT3 Axis to Induce Hepatic Fibrosis in Coordination with the SMAD Pathwayis</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1130</span>, <span class="refDoi"> DOI: 10.1038/s41467-017-01236-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=10.1038%2Fs41467-017-01236-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=29066712" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;key=1%3ACAS%3A280%3ADC%252BC1M7ktFGqsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=1130&author=D.+Chakrabortyauthor=B.+%C5%A0umov%C3%A1author=T.+Mallanoauthor=C.-W.+Chenauthor=A.+Distlerauthor=C.+Bergmannauthor=I.+Ludolphauthor=R.+E.+Horchauthor=K.+Gelseauthor=A.+Rammingauthor=O.+Distlerauthor=G.+Schettauthor=L.+%C5%A0enoltauthor=J.+H.+W.+Distler&title=Activation+of+STAT3+integrates+common+profibrotic+pathways+to+promote+fibroblast+activation+and+tissue+fibrosTransforming+Growth+Factor-%CE%B2+%28TGF-%CE%B2%29+Directly+Activates+the+JAK1-STAT3+Axis+to+Induce+Hepatic+Fibrosis+in+Coordination+with+the+SMAD+Pathwayis&doi=10.1038%2Fs41467-017-01236-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Activation of STAT3 integrates common profibrotic pathways to promote fibroblast activation and tissue fibrosis</span></div><div class="casAuthors">Chakraborty Debomita; Sumova Barbora; Mallano Tatjana; Chen Chih-Wei; Distler Alfiya; Bergmann Christina; Ramming Andreas; Schett Georg; Distler Jorg H W; Sumova Barbora; Senolt Ladislav; Ludolph Ingo; Horch Raymund E; Gelse Kolja; Distler Oliver</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1130</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Signal transducer and activator of transcription 3 (STAT3) is phosphorylated by various kinases, several of which have been implicated in aberrant fibroblast activation in fibrotic diseases including systemic sclerosis (SSc).  Here we show that profibrotic signals converge on STAT3 and that STAT3 may be an important molecular checkpoint for tissue fibrosis.  STAT3 signaling is hyperactivated in SSc in a TGFβ-dependent manner.  Expression profiling and functional studies in vitro and in vivo demonstrate that STAT3 activation is mediated by the combined action of JAK, SRC, c-ABL, and JNK kinases.  STAT3-deficient fibroblasts are less sensitive to the pro-fibrotic effects of TGFβ.  Fibroblast-specific knockout of STAT3, or its pharmacological inhibition, ameliorate skin fibrosis in experimental mouse models.  STAT3 thus integrates several profibrotic signals and might be a core mediator of fibrosis.  Considering that several STAT3 inhibitors are currently tested in clinical trials, STAT3 might be a candidate for molecular targeted therapies of SSc.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT4MyXAypwc0ORS7coPn-RtfW6udTcc2eamAvFJs6ChMbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M7ktFGqsg%253D%253D&md5=7bf14601b39a53147071c7a160636831</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1038%2Fs41467-017-01236-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-017-01236-6%26sid%3Dliteratum%253Aachs%26aulast%3DChakraborty%26aufirst%3DD.%26aulast%3D%25C5%25A0umov%25C3%25A1%26aufirst%3DB.%26aulast%3DMallano%26aufirst%3DT.%26aulast%3DChen%26aufirst%3DC.-W.%26aulast%3DDistler%26aufirst%3DA.%26aulast%3DBergmann%26aufirst%3DC.%26aulast%3DLudolph%26aufirst%3DI.%26aulast%3DHorch%26aufirst%3DR.%2BE.%26aulast%3DGelse%26aufirst%3DK.%26aulast%3DRamming%26aufirst%3DA.%26aulast%3DDistler%26aufirst%3DO.%26aulast%3DSchett%26aufirst%3DG.%26aulast%3D%25C5%25A0enolt%26aufirst%3DL.%26aulast%3DDistler%26aufirst%3DJ.%2BH.%2BW.%26atitle%3DActivation%2520of%2520STAT3%2520integrates%2520common%2520profibrotic%2520pathways%2520to%2520promote%2520fibroblast%2520activation%2520and%2520tissue%2520fibrosTransforming%2520Growth%2520Factor-%25CE%25B2%2520%2528TGF-%25CE%25B2%2529%2520Directly%2520Activates%2520the%2520JAK1-STAT3%2520Axis%2520to%2520Induce%2520Hepatic%2520Fibrosis%2520in%2520Coordination%2520with%2520the%2520SMAD%2520Pathwayis%26jtitle%3DNat.%2520Commun.%26date%3D2017%26volume%3D8%26spage%3D1130%26doi%3D10.1038%2Fs41467-017-01236-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i126"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01026">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_41452"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c01026?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01026</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Chirality confirmation of <b>36a</b> and <b>36b</b> with a chiral derivatizing agent, (<i>S</i>)-mosher’s reagent; HPLC spectra of <b>20c</b>, <b>30b–30c</b>, <b>31b</b>, <b>30d</b>, <b>30g</b>, <b>30j</b>, and <b>32b–32c</b>; chiral HPLC spectra of <b>38a</b> and <b>38b</b>; NMR spectra of final compounds; and docking poses of <b>31g</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01026/suppl_file/jm0c01026_si_001.pdf">PDF</a>)</p><p class="last">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01026/suppl_file/jm0c01026_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01026/suppl_file/jm0c01026_si_001.pdf">jm0c01026_si_001.pdf (6.35 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01026/suppl_file/jm0c01026_si_002.csv">jm0c01026_si_002.csv (4.53 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c01026&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c01026%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2021.64.issue-2%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c01026" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                5MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-b425991822a79ada59b3.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-fb88834b824b41b135ad.js"></script>

<script type="text/javascript" src="/wro/ke1q~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"66797b10afccd18f","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.6.0","si":10}'></script>
</body>
</html>
